<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/1/239" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/1/239.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/1/239.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/1/239.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/1/239.atom" />
<meta name="HW.pisa" content="ggg;8/1/239" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Natural Variation in SER1 and ENA6 Underlie Condition-Specific Growth Defects in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.117.300392" />
<meta name="DC.Date" content="2018-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Sirr et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Despite their ubiquitous use in laboratory strains, naturally occurring loss-of-function mutations in genes encoding core metabolic enzymes are relatively rare in wild isolates of Saccharomyces cerevisiae . Here, we identify a naturally occurring serine auxotrophy in a sake brewing strain from Japan. Through a cross with a honey wine (white tecc) brewing strain from Ethiopia, we map the minimal medium growth defect to [SER1][1] , which encodes 3-phosphoserine aminotransferase and is orthologous to the human disease gene, PSAT1 . To investigate the impact of this polymorphism under conditions of abundant external nutrients, we examine growth in rich medium alone or with additional stresses, including the drugs caffeine and rapamycin and relatively high concentrations of copper, salt, and ethanol. Consistent with studies that found widespread effects of different auxotrophies on RNA expression patterns in rich media, we find that the [SER1][1] loss-of-function allele dominates the quantitative trait locus (QTL) landscape under many of these conditions, with a notable exacerbation of the effect in the presence of rapamycin and caffeine. We also identify a major-effect QTL associated with growth on salt that maps to the gene encoding the sodium exporter, [ENA6][2] . We demonstrate that the salt phenotype is largely driven by variation in the [ENA6][2] promoter, which harbors a deletion that removes binding sites for the [Mig1][3] and [Nrg1][4] transcriptional repressors. Thus, our results identify natural variation associated with both coding and regulatory regions of the genome that underlie strong growth phenotypes.

 [1]: http://www.yeastgenome.org/locus/S000005710/overview
 [2]: http://www.yeastgenome.org/locus/S000130715/overview
 [3]: http://www.yeastgenome.org/locus/S000003003/overview
 [4]: http://www.yeastgenome.org/locus/S000002450/overview" />
<meta name="DC.Contributor" content="Amy Sirr" />
<meta name="DC.Contributor" content="Adrian C. Scott" />
<meta name="DC.Contributor" content="Gareth A. Cromie" />
<meta name="DC.Contributor" content="Catherine L. Ludlow" />
<meta name="DC.Contributor" content="Vida Ahyong" />
<meta name="DC.Contributor" content="Trey S. Morgan" />
<meta name="DC.Contributor" content="Teresa Gilbert" />
<meta name="DC.Contributor" content="Aimée M. Dudley" />
<meta name="article:published_time" content="2018-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Natural Variation in SER1 and ENA6 Underlie Condition-Specific Growth Defects in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Despite their ubiquitous use in laboratory strains, naturally occurring loss-of-function mutations in genes encoding core metabolic enzymes are relatively rare in wild isolates of &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;. Here, we identify a naturally occurring serine auxotrophy in a sake brewing strain from Japan. Through a cross with a honey wine (white tecc) brewing strain from Ethiopia, we map the minimal medium growth defect to &lt;i&gt;SER1&lt;/i&gt;, which encodes 3-phosphoserine aminotransferase and is orthologous to the human disease gene, &lt;i&gt;PSAT1&lt;/i&gt;. To investigate the impact of this polymorphism under conditions of abundant external nutrients, we examine growth in rich medium alone or with additional stresses, including the drugs caffeine and rapamycin and relatively high concentrations of copper, salt, and ethanol. Consistent with studies that found widespread effects of different auxotrophies on RNA expression patterns in rich media, we find that the &lt;i&gt;SER1&lt;/i&gt; loss-of-function allele dominates the quantitative trait locus (QTL) landscape under many of these conditions, with a notable exacerbation of the effect in the presence of rapamycin and caffeine. We also identify a major-effect QTL associated with growth on salt that maps to the gene encoding the sodium exporter, &lt;i&gt;ENA6&lt;/i&gt;. We demonstrate that the salt phenotype is largely driven by variation in the &lt;i&gt;ENA6&lt;/i&gt; promoter, which harbors a deletion that removes binding sites for the Mig1 and Nrg1 transcriptional repressors. Thus, our results identify natural variation associated with both coding and regulatory regions of the genome that underlie strong growth phenotypes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="ggg;8/1/239" />
<meta name="citation_id" content="8/1/239" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/1/239" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/1/239.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/1/239.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/1/239.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300392" />
<meta name="citation_pmid" content="29138237" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="239" />
<meta name="citation_lastpage" content="251" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Amy Sirr" />
<meta name="citation_author_institution" content="Pacific Northwest Research Institute, Seattle, Washington 98122" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5186-9692" />
<meta name="citation_author" content="Adrian C. Scott" />
<meta name="citation_author_institution" content="Pacific Northwest Research Institute, Seattle, Washington 98122" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4511-0898" />
<meta name="citation_author" content="Gareth A. Cromie" />
<meta name="citation_author_institution" content="Pacific Northwest Research Institute, Seattle, Washington 98122" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5265-1970" />
<meta name="citation_author" content="Catherine L. Ludlow" />
<meta name="citation_author_institution" content="Pacific Northwest Research Institute, Seattle, Washington 98122" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9682-7851" />
<meta name="citation_author" content="Vida Ahyong" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, University of California, Berkeley, California 94720" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8632-9629" />
<meta name="citation_author" content="Trey S. Morgan" />
<meta name="citation_author_institution" content="Pacific Northwest Research Institute, Seattle, Washington 98122" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9299-8562" />
<meta name="citation_author" content="Teresa Gilbert" />
<meta name="citation_author" content="Aimée M. Dudley" />
<meta name="citation_author_institution" content="Pacific Northwest Research Institute, Seattle, Washington 98122" />
<meta name="citation_author_email" content="aimee.dudley@gmail.com" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3644-0625" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=R. Acuna-Hidalgo;citation_author=D. Schanze;citation_author=A. Kariminejad;citation_author=A. Nordgren;citation_author=M. H. Kariminejad;citation_title=Neu-Laxova syndrome is a heterogeneous metabolic disorder caused by defects in enzymes of the L-serine biosynthesis pathway.;citation_pages=285-293;citation_volume=95;citation_year=2014;citation_pmid=25152457;citation_doi=10.1016/j.ajhg.2014.07.012" />
<meta name="citation_reference" content="citation_journal_title=DNA Res.;citation_author=T. Akao;citation_author=I. Yashiro;citation_author=A. Hosoyama;citation_author=H. Kitagaki;citation_author=H. Horikawa;citation_title=Whole-genome sequencing of sake yeast Saccharomyces cerevisiae Kyokai no. 7.;citation_volume=18;citation_year=2011;citation_pmid=21900213;citation_doi=10.1093/dnares/dsr029" />
<meta name="citation_reference" content="citation_journal_title=Nat. Microbiol.;citation_author=M. T. Alam;citation_author=A. Zelezniak;citation_author=M. Mülleder;citation_author=P. Shliaha;citation_author=R. Schwarz;citation_title=The metabolic background is a global player in Saccharomyces gene expression epistasis.;citation_pages=15030;citation_volume=1;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=The FASEB Journal;citation_journal_abbrev=FASEB J.;citation_author=D. R. Appling;citation_title=Compartmentation of folate-mediated one-carbon metabolism in eukaryotes;citation_pages=2645-2651;citation_volume=5;citation_year=1991;citation_issue=12;citation_pmid=1916088" />
<meta name="citation_reference" content="citation_journal_abbrev=Food Microbiol;citation_author=B. Bahiru;citation_author=T. Mehari;citation_author=M. Ashenafi;citation_title=Yeast and lactic acid flora of tej, an indigenous Ethiopian honey wine: variations within and between production units.;citation_pages=277-282;citation_volume=23;citation_year=2006;citation_issue=3;citation_pmid=16943014" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=D. Bates;citation_author=M. Mächler;citation_author=B. Bolker;citation_author=S. Walker;citation_title=Fitting linear mixed-effects models Using lme4.;citation_pages=1-48;citation_volume=67;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=T. Beck;citation_author=A. Schmidt;citation_author=M. N. Hall;citation_title=Starvation induces vacuolar targeting and degradation of the tryptophan permease in yeast.;citation_volume=146;citation_year=1999;citation_pmid=10491387;citation_doi=10.1083/jcb.146.6.1227" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. S. Bloom;citation_author=I. M. Ehrenreich;citation_author=W. T. Loo;citation_author=T. L. Lite;citation_author=L. Kruglyak;citation_title=Finding the sources of missing heritability in a yeast cross.;citation_pages=234-237;citation_volume=494;citation_year=2013;citation_pmid=23376951;citation_doi=10.1038/nature11867" />
<meta name="citation_reference" content="citation_journal_title=Eur. J. Paediatr. Neurol.;citation_author=A. Brassier;citation_author=V. Valayannopoulos;citation_author=N. Bahi-Buisson;citation_author=E. Wiame;citation_author=L. Hubert;citation_title=Two new cases of serine deficiency disorders treated with l-serine.;citation_pages=53-60;citation_volume=20;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=K. Campbell;citation_author=J. Vowinckel;citation_author=M. Mülleder;citation_author=S. Malmsheimer;citation_author=N. Lawrence;citation_title=Self-establishing communities enable cooperative metabolite exchange in a eukaryote.;citation_volume=4;citation_year=2015;citation_pmid=26499891;citation_doi=10.7554/eLife.09943" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=K. E. Christensen;citation_author=R. E. MacKenzie;citation_title=Mitochondrial one-carbon metabolism is adapted to the specific needs of yeast, plants and mammals.;citation_pages=595-605;citation_volume=28;citation_year=2006;citation_issue=6;citation_pmid=16700064;citation_doi=10.1002/bies.20420" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Rev.;citation_author=M. Conrad;citation_author=J. Schothorst;citation_author=H. N. Kankipati;citation_author=G. Van Zeebroeck;citation_author=M. Rubio-Texeira;citation_title=Nutrient sensing and signaling in the yeast Saccharomyces cerevisiae.;citation_volume=38;citation_year=2014;citation_pmid=24483210;citation_doi=10.1111/1574-6976.12065" />
<meta name="citation_reference" content="citation_title=G3;citation_author=G. A. Cromie;citation_author=K. E. Hyma;citation_author=C. L. Ludlow;citation_author=C. Garmendia-Torres;citation_author=T. L. Gilbert;citation_title=Genomic sequence diversity and population structure of Saccharomyces cerevisiae assessed by RAD-seq.;citation_pages=2163-2171;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=G. A. Cromie;citation_author=Z. Tan;citation_author=M. Hays;citation_author=E. W. Jeffery;citation_author=A. M. Dudley;citation_title=Dissecting gene expression changes accompanying a ploidy-based phenotypic switch.;citation_volume=7;citation_year=2017;citation_pmid=27836908;citation_doi=10.1534/g3.116.036160" />
<meta name="citation_reference" content="citation_title=G3;citation_author=G. A. Cromie;citation_author=Z. Tan;citation_author=M. Hays;citation_author=A. Sirr;citation_author=E. W. Jeffery;citation_title=Transcriptional profiling of biofilm regulators identified by an overexpression screen in Saccharomyces cerevisiae.;citation_volume=7;citation_year=2017;citation_pmid=28673928;citation_doi=10.1534/g3.117.042440" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=P. Daran-Lapujade;citation_author=J. M. Daran;citation_author=M. A. Luttik;citation_author=M. J. Almering;citation_author=J. T. Pronk;citation_title=An atypical PMR2 locus is responsible for hypersensitivity to sodium and lithium cations in the laboratory strain Saccharomyces cerevisiae CEN.PK113–7D.;citation_volume=9;citation_year=2009;citation_pmid=19519766;citation_doi=10.1111/j.1567-1364.2009.00530.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of inherited metabolic disease;citation_journal_abbrev=J Inherit Metab Dis;citation_author=T. J. de Koning;citation_author=M. Duran;citation_author=L. Van Maldergem;citation_author=M. Pineda;citation_author=L. Dorland;citation_title=Congenital microcephaly and seizures due to 3-phosphoglycerate dehydrogenase deficiency: outcome of treatment with amino acids.;citation_pages=119-125;citation_volume=25;citation_year=2002;citation_issue=2;citation_pmid=12118526;citation_doi=10.1023/A:1015624726822" />
<meta name="citation_reference" content="citation_journal_title=Lancet;citation_journal_abbrev=Lancet;citation_author=T. J. de Koning;citation_author=L. W. Klomp;citation_author=A. C. van Oppen;citation_author=F. A. Beemer;citation_author=L. Dorland;citation_title=Prenatal and early postnatal treatment in 3-phosphoglycerate-dehydrogenase deficiency.;citation_pages=2221-2222;citation_volume=364;citation_year=2004;citation_issue=9452;citation_pmid=15610810;citation_doi=10.1016/S0140-6736(04)17596-X" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=G. M. DeNicola;citation_author=P. H. Chen;citation_author=E. Mullarky;citation_author=J. A. Sudderth;citation_author=Z. Hu;citation_title=Erratum: NRF2 regulates serine biosynthesis in non-small cell lung cancer.;citation_pages=473;citation_volume=48;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. W. Doniger;citation_author=H. S. Kim;citation_author=D. Swain;citation_author=D. Corcuera;citation_author=M. Williams;citation_title=A catalog of neutral and deleterious polymorphism in yeast.;citation_pages=e1000183-e1000183;citation_volume=4;citation_year=2008;citation_issue=8;citation_pmid=18769710;citation_doi=10.1371/journal.pgen.1000183" />
<meta name="citation_reference" content="citation_journal_title=Cell Metab.;citation_author=G. S. Ducker;citation_author=J. D. Rabinowitz;citation_title=One-carbon metabolism in health and disease.;citation_pages=27-42;citation_volume=25;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Clin. Perinatol.;citation_author=A. W. El-Hattab;citation_title=Inborn errors of metabolism.;citation_pages=413-439;citation_volume=42;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Inherit. Metab. Dis.;citation_author=A. W. El-Hattab;citation_author=R. Shaheen;citation_author=J. Hertecant;citation_author=H. I. Galadari;citation_author=B. S. Albaqawi;citation_title=On the phenotypic spectrum of serine biosynthesis defects.;citation_pages=373-381;citation_volume=39;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. C. Fay;citation_author=J. A. Benavides;citation_title=Evidence for domesticated and wild populations of Saccharomyces cerevisiae.;citation_pages=66-71;citation_volume=1;citation_year=2005;citation_issue=1;citation_pmid=16103919;citation_doi=10.1371/journal.pgen.0010066" />
<meta name="citation_reference" content="citation_journal_title=Cell Metab.;citation_author=D. C. Goberdhan;citation_author=C. Wilson;citation_author=A. L. Harris;citation_title=Amino acid sensing by mTORC1: intracellular transporters mark the spot.;citation_pages=580-589;citation_volume=23;citation_year=2016;citation_doi=10.1016/j.cmet.2016.03.013" />
<meta name="citation_reference" content="citation_journal_title=Environmental Microbiology (Print);citation_journal_abbrev=Environmental Microbiology (Print);citation_author=M. R. Goddard;citation_author=N. Anfang;citation_author=R. Tang;citation_author=R. C. Gardner;citation_author=C. Jun;citation_title=A distinct population of Saccharomyces cerevisiae in New Zealand: evidence for local dispersal by insects and human-aided global dispersal in oak barrels.;citation_pages=63-73;citation_volume=12;citation_year=2010;citation_issue=1;citation_pmid=19691498;citation_doi=10.1111/j.1462-2920.2009.02035.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Goffeau;citation_author=B. G. Barrell;citation_author=H. Bussey;citation_author=R. W. Davis;citation_author=B. Dujon;citation_title=Life with 6000 Genes;citation_pages=546-567;citation_volume=274;citation_year=1996;citation_issue=5287;citation_pmid=8849441;citation_doi=10.1126/science.274.5287.546" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=A. L. Goldstein;citation_author=J. H. McCusker;citation_title=Three new dominant drug resistance cassettes for gene disruption in Saccharomyces cerevisiae.;citation_pages=1541-1553;citation_volume=15;citation_year=1999;citation_issue=14;citation_pmid=10514571;citation_doi=10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=A. Gonzalez;citation_author=M. N. Hall;citation_title=Nutrient sensing and TOR signaling in yeast and mammals.;citation_volume=36;citation_year=2017;citation_pmid=28096180;citation_doi=10.15252/embj.201696010" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=R. Haro;citation_author=B. Garciadeblas;citation_author=A. Rodriguez-Navarro;citation_title=A novel P-type ATPase from yeast involved in sodium transport.;citation_pages=189-191;citation_volume=291;citation_year=1991;citation_issue=2;citation_pmid=1657642;citation_doi=10.1016/0014-5793(91)81280-L" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=C. E. Hart;citation_author=V. Race;citation_author=Y. Achouri;citation_author=E. Wiame;citation_author=M. Sharrard;citation_title=Phosphoserine aminotransferase deficiency: a novel disorder of the serine biosynthesis pathway.;citation_pages=931-937;citation_volume=80;citation_year=2007;citation_issue=5;citation_pmid=17436247;citation_doi=10.1086/517888" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=S. Hashimoto;citation_author=M. Ogura;citation_author=K. Aritomi;citation_author=H. Hoshida;citation_author=Y. Nishizawa;citation_title=Isolation of Auxotrophic Mutants of Diploid Industrial Yeast Strains after UV Mutagenesis;citation_pages=312-319;citation_volume=71;citation_year=2005;citation_issue=1;citation_pmid=15640203;citation_doi=10.1128/AEM.71.1.312-319.2005" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Heitman;citation_author=N. R. Movva;citation_author=M. N. Hall;citation_title=Targets for cell cycle arrest by the immunosuppressant rapamycin in yeast;citation_pages=905-909;citation_volume=253;citation_year=1991;citation_issue=5022;citation_pmid=1715094;citation_doi=10.1126/science.1715094" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=G. Hester;citation_author=W. Stark;citation_author=M. Moser;citation_author=J. Kallen;citation_author=Z. Markovic-Housley;citation_title=Crystal structure of phosphoserine aminotransferase from Escherichia coli at 2.3 A resolution: comparison of the unligated enzyme and a complex with alpha-methyl-l-glutamate.;citation_pages=829-850;citation_volume=286;citation_year=1999;citation_issue=3;citation_pmid=10024454;citation_doi=10.1006/jmbi.1998.2506" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. E. Hillenmeyer;citation_author=E. Fung;citation_author=J. Wildenhain;citation_author=S. E. Pierce;citation_author=S. Hoon;citation_title=The Chemical Genomic Portrait of Yeast: Uncovering a Phenotype for All Genes;citation_pages=362-365;citation_volume=320;citation_year=2008;citation_issue=5874;citation_pmid=18420932;citation_doi=10.1126/science.1150021" />
<meta name="citation_reference" content="citation_journal_title=Cell Rep.;citation_author=J. Hou;citation_author=A. Sigwalt;citation_author=T. Fournier;citation_author=D. Pflieger;citation_author=J. Peter;citation_title=The hidden complexity of Mendelian traits across natural yeast populations.;citation_pages=1106-1114;citation_volume=16;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=S. C. Kalhan;citation_author=R. W. Hanson;citation_title=Resurgence of serine: an often neglected but indispensable amino Acid.;citation_volume=287;citation_year=2012;citation_pmid=22566694;citation_doi=10.1074/jbc.R112.357194" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. S. Kim;citation_author=J. C. Fay;citation_title=Genetic variation in the cysteine biosynthesis pathway causes sensitivity to pharmacological compounds.;citation_volume=104;citation_year=2007;citation_pmid=18042712;citation_doi=10.1073/pnas.0708194104" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. Kruglyak;citation_author=E. S. Lander;citation_title=A nonparametric approach for mapping quantitative trait loci.;citation_pages=1421-1428;citation_volume=139;citation_year=1995;citation_issue=3;citation_pmid=7768449" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=T. M. Lamb;citation_author=A. P. Mitchell;citation_title=The Transcription Factor Rim101p Governs Ion Tolerance and Cell Differentiation by Direct Repression of the Regulatory Genes NRG1 and SMP1 in Saccharomyces cerevisiae;citation_pages=677-686;citation_volume=23;citation_year=2003;citation_issue=2;citation_pmid=12509465;citation_doi=10.1128/MCB.23.2.677-686.2003" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. S. Lander;citation_author=D. Botstein;citation_title=Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.;citation_pages=185-199;citation_volume=121;citation_year=1989;citation_issue=1;citation_pmid=2563713" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=B. Lanpher;citation_author=N. Brunetti-Pierri;citation_author=B. Lee;citation_title=Inborn errors of metabolism: the flux from Mendelian to complex diseases.;citation_pages=449-460;citation_volume=7;citation_year=2006;citation_issue=6;citation_pmid=16708072" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=J. C. Lee;citation_author=A. Tsoi;citation_author=G. D. Kornfeld;citation_author=I. W. Dawes;citation_title=Cellular responses to L-serine in Saccharomyces cerevisiae: roles of general amino acid control, compartmentalization, and aspartate synthesis.;citation_volume=13;citation_year=2013;citation_pmid=23837815;citation_doi=10.1111/1567-1364.12063" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAMtools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. Liti;citation_author=D. B. Barton;citation_author=E. J. Louis;citation_title=Sequence diversity, reproductive isolation and species concepts in Saccharomyces.;citation_volume=174;citation_year=2006;citation_pmid=16951060;citation_doi=10.1534/genetics.106.062166" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Cancer;citation_author=J. W. Locasale;citation_title=Serine, glycine and one-carbon units: cancer metabolism in full circle.;citation_pages=572-583;citation_volume=13;citation_year=2013;citation_pmid=23822983;citation_doi=10.1038/nrc3557" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=R. Loewith;citation_author=E. Jacinto;citation_author=S. Wullschleger;citation_author=A. Lorberg;citation_author=J. L. Crespo;citation_title=Two TOR complexes, only one of which is rapamycin sensitive, have distinct roles in cell growth control.;citation_pages=457-468;citation_volume=10;citation_year=2002;citation_issue=3;citation_pmid=12408816;citation_doi=10.1016/S1097-2765(02)00636-6" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=C. L. Ludlow;citation_author=A. C. Scott;citation_author=G. A. Cromie;citation_author=E. W. Jeffery;citation_author=A. Sirr;citation_title=High-throughput tetrad analysis.;citation_pages=671-675;citation_volume=10;citation_year=2013;citation_pmid=23666411;citation_doi=10.1038/nmeth.2479" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=C. L. Ludlow;citation_author=G. A. Cromie;citation_author=C. Garmendia-Torres;citation_author=A. Sirr;citation_author=M. Hays;citation_title=Independent origins of yeast associated with coffee and cacao fermentation.;citation_pages=965-971;citation_volume=26;citation_year=2016;citation_pmid=27020745;citation_doi=10.1016/j.cub.2016.02.012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. L. Lutfiyya;citation_author=V. R. Iyer;citation_author=J. DeRisi;citation_author=M. J. DeVit;citation_author=P. O. Brown;citation_title=Characterization of Three Related Glucose Repressors and Genes They Regulate in Saccharomyces cerevisiae;citation_pages=1377-1391;citation_volume=150;citation_year=1998;citation_issue=4;citation_pmid=9832517" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=K. Melcher;citation_author=K. D. Entian;citation_title=Genetic analysis of serine biosynthesis and glucose repression in yeast.;citation_pages=295-300;citation_volume=21;citation_year=1992;citation_issue=4-5;citation_pmid=1326413;citation_doi=10.1007/BF00351686" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=K. Melcher;citation_author=M. Rose;citation_author=M. Kunzler;citation_author=G. H. Braus;citation_author=K. D. Entian;citation_title=Molecular analysis of the yeast SER1 gene encoding 3-phosphoserine aminotransferase: regulation by general control and serine repression.;citation_pages=501-508;citation_volume=27;citation_year=1995;citation_issue=6;citation_pmid=7553933;citation_doi=10.1007/BF00314439" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=W. Palm;citation_author=C. B. Thompson;citation_title=Nutrient acquisition strategies of mammalian cells.;citation_pages=234-242;citation_volume=546;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=A. B. Parsons;citation_author=R. L. Brost;citation_author=H. Ding;citation_author=Z. Li;citation_author=C. Zhang;citation_title=Integration of chemical-genetic and genetic interaction data links bioactive compounds to cellular target pathways.;citation_pages=62-69;citation_volume=22;citation_year=2004;citation_issue=1;citation_pmid=14661025;citation_doi=10.1038/nbt919" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. Platara;citation_author=A. Ruiz;citation_author=R. Serrano;citation_author=A. Palomino;citation_author=F. Moreno;citation_title=The transcriptional response of the yeast Na(+)-ATPase ENA1 gene to alkaline stress involves three main signaling pathways.;citation_volume=281;citation_year=2006;citation_pmid=17023428;citation_doi=10.1074/jbc.M606483200" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. Possemato;citation_author=K. M. Marks;citation_author=Y. D. Shaul;citation_author=M. E. Pacold;citation_author=D. Kim;citation_title=Functional genomics reveal that the serine synthesis pathway is essential in breast cancer.;citation_pages=346-350;citation_volume=476;citation_year=2011;citation_issue=7360;citation_pmid=21760589;citation_doi=10.1038/nature10350" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. Proft;citation_author=R. Serrano;citation_title=Repressors and Upstream Repressing Sequences of the Stress-Regulated ENA1 Gene in Saccharomyces cerevisiae: bZIP Protein Sko1p Confers HOG-Dependent Osmotic Regulation;citation_pages=537-546;citation_volume=19;citation_year=1999;citation_issue=1;citation_pmid=9858577;citation_doi=10.1128/MCB.19.1.537" />
<meta name="citation_reference" content="citation_journal_title=Applied and Environmental Microbiology;citation_journal_abbrev=Appl. Environ. Microbiol.;citation_author=J. T. Pronk;citation_title=Auxotrophic Yeast Strains in Fundamental and Applied Research;citation_pages=2095-2100;citation_volume=68;citation_year=2002;citation_issue=5;citation_pmid=11976076;citation_doi=10.1128/AEM.68.5.2095-2100.2002" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Ronald;citation_author=J. M. Akey;citation_title=The evolution of gene expression QTL in Saccharomyces cerevisiae.;citation_pages=e678-e678;citation_volume=2;citation_year=2007;citation_issue=7;citation_pmid=17668057;citation_doi=10.1371/journal.pone.0000678" />
<meta name="citation_reference" content="citation_title=Methods in Yeast Genetics: A Laboratory Course Manual.;citation_author=M. D. Rose;citation_author=F. M. Winston;citation_author=P. Hieter;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=H. K. Rudolph;citation_author=A. Antebi;citation_author=G. R. Fink;citation_author=C. M. Buckley;citation_author=T. E. Dorman;citation_title=The yeast secretory pathway is perturbed by mutations in PMR1, a member of a Ca2+ ATPase family.;citation_pages=133-145;citation_volume=58;citation_year=1989;citation_issue=1;citation_pmid=2526682;citation_doi=10.1016/0092-8674(89)90410-8" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=A. Ruiz;citation_author=J. Arino;citation_title=Function and regulation of the Saccharomyces cerevisiae ENA sodium ATPase system.;citation_volume=6;citation_year=2007;citation_pmid=17951516;citation_doi=10.1128/EC.00337-07" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Schacherer;citation_author=J. A. Shapiro;citation_author=D. M. Ruderfer;citation_author=L. Kruglyak;citation_title=Comprehensive polymorphism survey elucidates population structure of Saccharomyces cerevisiae.;citation_pages=342-345;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212320;citation_doi=10.1038/nature07670" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=A. Schmidt;citation_author=T. Beck;citation_author=A. Koller;citation_author=J. Kunz;citation_author=M. N. Hall;citation_title=The TOR nutrient signalling pathway phosphorylates NPR1 and inhibits turnover of the tryptophan permease;citation_pages=6924-6931;citation_volume=17;citation_year=1998;citation_issue=23;citation_pmid=9843498;citation_doi=10.1093/emboj/17.23.6924" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=C. A. Schneider;citation_author=W. S. Rasband;citation_author=K. W. Eliceiri;citation_title=NIH Image to ImageJ: 25 years of image analysis.;citation_pages=671-675;citation_volume=9;citation_year=2012;citation_issue=7;citation_pmid=22930834;citation_doi=10.1038/nmeth.2089" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=R. Serrano;citation_author=A. Ruiz;citation_author=D. Bernal;citation_author=J. R. Chambers;citation_author=J. Arino;citation_title=The transcriptional response to alkaline pH in Saccharomyces cerevisiae: evidence for calcium-mediated signalling.;citation_pages=1319-1333;citation_volume=46;citation_year=2002;citation_issue=5;citation_pmid=12453218;citation_doi=10.1046/j.1365-2958.2002.03246.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Sirr;citation_author=G. A. Cromie;citation_author=E. W. Jeffery;citation_author=T. L. Gilbert;citation_author=C. L. Ludlow;citation_title=Allelic variation, aneuploidy, and nongenetic mechanisms suppress a monogenic trait in yeast.;citation_volume=199;citation_year=2015;citation_pmid=25398792;citation_doi=10.1534/genetics.114.170563" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. K. Strope;citation_author=D. A. Skelly;citation_author=S. G. Kozmin;citation_author=G. Mahadevan;citation_author=E. A. Stone;citation_title=The 100-genomes strains, an S. cerevisiae resource that illuminates its natural phenotypic and genotypic variation and emergence as an opportunistic pathogen.;citation_volume=25;citation_year=2015;citation_pmid=25840857;citation_doi=10.1101/gr.185538.114" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and metabolism;citation_journal_abbrev=Mol Genet Metab;citation_author=L. Tabatabaie;citation_author=L. W. Klomp;citation_author=R. Berger;citation_author=T. J. de Koning;citation_title=L-serine synthesis in the central nervous system: a review on serine deficiency disorders.;citation_pages=256-262;citation_volume=99;citation_year=2010;citation_issue=3;citation_pmid=19963421;citation_doi=10.1016/j.ymgme.2009.10.012" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Tan;citation_author=M. Hays;citation_author=G. A. Cromie;citation_author=E. W. Jeffery;citation_author=A. C. Scott;citation_title=Aneuploidy underlies a multicellular phenotypic switch.;citation_volume=110;citation_year=2013;citation_pmid=23812752;citation_doi=10.1073/pnas.1301047110" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. H. Tong;citation_author=M. Evangelista;citation_author=A. B. Parsons;citation_author=H. Xu;citation_author=G. D. Bader;citation_title=Systematic Genetic Analysis with Ordered Arrays of Yeast Deletion Mutants;citation_pages=2364-2368;citation_volume=294;citation_year=2001;citation_issue=5550;citation_pmid=11743205;citation_doi=10.1126/science.1065810" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=R. Ulane;citation_author=M. Ogur;citation_title=Genetic and Physiological Control of Serine and Glycine Biosynthesis in Saccharomyces;citation_pages=34-43;citation_volume=109;citation_year=1972;citation_issue=1;citation_pmid=4333378" />
<meta name="citation_reference" content="citation_journal_title=JAMA Pediatr.;citation_author=H. J. Vernon;citation_title=Inborn errors of metabolism: advances in diagnosis and therapy.;citation_pages=778-782;citation_volume=169;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=V. K. Vyas;citation_author=S. Kuchin;citation_author=M. Carlson;citation_title=Interaction of the Repressors Nrg1 and Nrg2 With the Snf1 Protein Kinase in Saccharomyces cerevisiae;citation_pages=563-572;citation_volume=158;citation_year=2001;citation_issue=2;citation_pmid=11404322" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Warringer;citation_author=E. Zorgo;citation_author=F. A. Cubillos;citation_author=A. Zia;citation_author=A. Gjuvsland;citation_title=Trait variation in yeast is defined by population history.;citation_pages=e1002111-e1002111;citation_volume=7;citation_year=2011;citation_issue=6;citation_pmid=21698134;citation_doi=10.1371/journal.pgen.1002111" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=J. Wieland;citation_author=A. M. Nitsche;citation_author=J. Strayle;citation_author=H. Steiner;citation_author=H. K. Rudolph;citation_title=The PMR2 gene cluster encodes functionally distinct isoforms of a putative Na+ pump in the yeast plasma membrane.;citation_pages=3870-3882;citation_volume=14;citation_year=1995;citation_pmid=7664728" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Wilkening;citation_author=G. Lin;citation_author=E. S. Fritsch;citation_author=M. M. Tekkedil;citation_author=S. Anders;citation_title=An evaluation of high-throughput approaches to QTL mapping in Saccharomyces cerevisiae.;citation_volume=196;citation_year=2014;citation_pmid=24374355;citation_doi=10.1534/genetics.113.160291" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=F. Winston;citation_author=C. Dollard;citation_author=S. L. Ricupero-Hovasse;citation_title=Construction of a set of convenient Saccharomyces cerevisiae strains that are isogenic to S288C.;citation_pages=53-55;citation_volume=11;citation_year=1995;citation_issue=1;citation_pmid=7762301;citation_doi=10.1002/yea.320110107" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. A. Winzeler;citation_author=D. D. Shoemaker;citation_author=A. Astromoff;citation_author=H. Liang;citation_author=K. Anderson;citation_title=Functional Characterization of the S. cerevisiae Genome by Gene Deletion and Parallel Analysis;citation_pages=901-906;citation_volume=285;citation_year=1999;citation_issue=5429;citation_pmid=10436161;citation_doi=10.1126/science.285.5429.901" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Natural Variation in SER1 and ENA6 Underlie Condition-Specific Growth Defects in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="twitter:description" content="Despite their ubiquitous use in laboratory strains, naturally occurring loss-of-function mutations in genes encoding core metabolic enzymes are relatively rare in wild isolates of Saccharomyces cerevisiae . Here, we identify a naturally occurring serine auxotrophy in a sake brewing strain from Japan. Through a cross with a honey wine (white tecc) brewing strain from Ethiopia, we map the minimal medium growth defect to [SER1][1] , which encodes 3-phosphoserine aminotransferase and is orthologous to the human disease gene, PSAT1 . To investigate the impact of this polymorphism under conditions of abundant external nutrients, we examine growth in rich medium alone or with additional stresses, including the drugs caffeine and rapamycin and relatively high concentrations of copper, salt, and ethanol. Consistent with studies that found widespread effects of different auxotrophies on RNA expression patterns in rich media, we find that the [SER1][1] loss-of-function allele dominates the quantitative trait locus (QTL) landscape under many of these conditions, with a notable exacerbation of the effect in the presence of rapamycin and caffeine. We also identify a major-effect QTL associated with growth on salt that maps to the gene encoding the sodium exporter, [ENA6][2] . We demonstrate that the salt phenotype is largely driven by variation in the [ENA6][2] promoter, which harbors a deletion that removes binding sites for the [Mig1][3] and [Nrg1][4] transcriptional repressors. Thus, our results identify natural variation associated with both coding and regulatory regions of the genome that underlie strong growth phenotypes.

 [1]: http://www.yeastgenome.org/locus/S000005710/overview
 [2]: http://www.yeastgenome.org/locus/S000130715/overview
 [3]: http://www.yeastgenome.org/locus/S000003003/overview
 [4]: http://www.yeastgenome.org/locus/S000002450/overview" />
<meta name="og-title" property="og:title" content="Natural Variation in SER1 and ENA6 Underlie Condition-Specific Growth Defects in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/1/239" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Despite their ubiquitous use in laboratory strains, naturally occurring loss-of-function mutations in genes encoding core metabolic enzymes are relatively rare in wild isolates of Saccharomyces cerevisiae . Here, we identify a naturally occurring serine auxotrophy in a sake brewing strain from Japan. Through a cross with a honey wine (white tecc) brewing strain from Ethiopia, we map the minimal medium growth defect to [SER1][1] , which encodes 3-phosphoserine aminotransferase and is orthologous to the human disease gene, PSAT1 . To investigate the impact of this polymorphism under conditions of abundant external nutrients, we examine growth in rich medium alone or with additional stresses, including the drugs caffeine and rapamycin and relatively high concentrations of copper, salt, and ethanol. Consistent with studies that found widespread effects of different auxotrophies on RNA expression patterns in rich media, we find that the [SER1][1] loss-of-function allele dominates the quantitative trait locus (QTL) landscape under many of these conditions, with a notable exacerbation of the effect in the presence of rapamycin and caffeine. We also identify a major-effect QTL associated with growth on salt that maps to the gene encoding the sodium exporter, [ENA6][2] . We demonstrate that the salt phenotype is largely driven by variation in the [ENA6][2] promoter, which harbors a deletion that removes binding sites for the [Mig1][3] and [Nrg1][4] transcriptional repressors. Thus, our results identify natural variation associated with both coding and regulatory regions of the genome that underlie strong growth phenotypes.

 [1]: http://www.yeastgenome.org/locus/S000005710/overview
 [2]: http://www.yeastgenome.org/locus/S000130715/overview
 [3]: http://www.yeastgenome.org/locus/S000003003/overview
 [4]: http://www.yeastgenome.org/locus/S000002450/overview" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
    <title>Natural Variation in SER1 and ENA6 Underlie Condition-Specific Growth Defects in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"QzqYPt40MlSsmfWI_zcqLbOP4UQ1Bd13jedG0AcCips","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487835","apath":"\/ggg\/8\/1\/239.atom","pisa":"ggg;8\/1\/239","ac":{"\/ggg\/8\/1\/239.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/1\/239.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/1\/239"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/1\/239"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/1\/239":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022HLpH\\\\\\\/2hbau+lFZerL0roUjmpRGA3FXJ32e5O7NtogUPGdqqxDbYFYPD4dlj4O5DSs0CGcUkooWMlkjx62qfL6kYQ9njIiLGJYoZxhS0ylt8j0Wico9tH7kg1lqIbtQ9HOIDHpiJSHsepMHmetGd5S5fQyW31YcsqIkmOHYqKl\\\\\\\/2p688q9Zz5oEiBMtXNhE1fUds91aCRRt1qolKVoOjkIX7O8WpSX+Re2+K\\\\\\\/qDQx2g58J1nJMdjMZ12kKvD4Vyct8iqnLQJK+jnWWqaPczwfQTouUjyzYQtupfNH2LJ5t2Kf4sDZYsbWEjACJ83Kx14byL1dxgnz7vgicWzOj0827JeBwGL0jZJxEOsPnpLvukYuq+uXFATmwrBdwTSpPlwFf3Uc721TAGAWbb17NfoZGegAmvYo4hqrYNT\\\\\\\/qXFpYcNRmWz9\\\\\\\/goZPquBAfohOpBexeN7Xqn8EiYPcSzjAsCjszqgbhwqbj7\\\\\\\/4K1Rw\\\\\\\/oramZrF+gXgVcS3SaqrRlWf19RwcmOlaaJmyjtiizyfxB+dzERPHEEeoHq56\\\\\\\/85EyZhyzMRcu6uG4\\\\\\\/h4MztGR9Yvhwr\\\\\\\/PjrdTRVIMQ5nK6fS\\\\\\\/r6tj6FMOf4VdgiWBT9pXvlZnlFsXtlATMNIoEKLDRMy\\\\\\\/tvPAWpmPEKoULQ2EQgZ34UaSbdpog23G2jin0bpgeTqbh5Y0vrywDgJ773icmEGXgPIQhlsJ9OscyZGe0AC2BTDGARoY19zutSREz6dthy3FlbaupfmtLIlRz6XSsb95JBdoxlAetGguIuSzBbEREMmDZEuFInDTZu8Mpf\\\\\\\/YA2\\\\\\\/OjAIm879ux7E3VMBgWyEPgNt2Sl+aNP6heuXaAHWF00sHsouKI+To3ephFYXwi7Jvh0QmyXGEUWcBQbrQ0ofviWykLzEzuwY2\\\\\\\/3GTR5nK\\\\\\\/xgtsadMJneL95uuCDRHZPdQh74\\\\\\\/B\\\\\\\/XhN2ZTF80ZPTqaq\\\\\\\/xwi3t0JM2Iv9CIbqgmKJPpwbkgZUm+pPYoSLp7hXX4jYMcpY1HhH4SPqqEuWrcRrsbT4+tmDGMed1AtODDAoMGTSq8LXIZL93zPa66tpjmhJiS0ki258z\\\\\\\/09av0r4c5vPSAKv6ItSo83g7X7uJFpjBhjxgpsExFwB51TQn3v3g+BV0SYYlHUiPHmdVTUJv5ZL9Yb3b4UzmRM4x9klB8xP\\\\\\\/tERuoGlMkdNJNsqBQdPhliwwESXLrmwS80KqnHZ647aKw9C1qpfHEALI6V9+cUi+rheyTYD4X99UgxZl0Fe7MY+nAP4\\\\\\\/+kq7S6QEHETq9\\\u0022,\\\u0022iv\\\u0022:\\\u0022E0v4ohyz0Qhv82Wwfa9lyA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222774450c6dd46041d1e568c1d9c97e96506c3404faf99c08c65bef66e9377056\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227xOg6kg2oJb+qvLe9m3AGcrbiyeUwTaf9QJeO\\\\\\\/f8ZFIhrUYHOhrDLFg5IfQ0bt\\\\\\\/VPXjC7gnyDRFEugrCrw7Tnno9SFX2\\\\\\\/lJeEOooPztNLevbHi\\\\\\\/YxLi7FW50mjDIuzNCz\\\\\\\/FaAfmeBwWCUtzvkgvn+49b38RhaI\\\\\\\/KfK4kDCVnjv\\\\\\\/m6VHRGYtNkvPsx6CJlcrCSmp6uBuMlSlpE+lzj3+wIb0m3opfL0QtSjqYoVp\\\\\\\/JV515pSKLSQyMCtIc3kNKxmJtQ8fMcEeTOIIdxRG\\\\\\\/IeNp1MXsm6quqEMuWGSGL1TA7OdwdMqZiRQGxCqkLc0JlibyvYzG6akcJYfXKSsFECIaizI5rTD9qoq\\\\\\\/zKMEyQBmGXZ7JvPzht41dk2JFIBbPO7g\\\\\\\/TsMvOj6\\\\\\\/Nc2mmiy+c8mXg6WEek4uEfjeYs+T9cKS55SGcIQxvwFzI7\\\\\\\/bEB6Fyfd0ARXwriLbRs7pBgz+O0lP3g+EsxaUy8uoOlqMWvLcPOPcFLjKh8V1P2WcJNyW3yteXJm+LTibWrru6M12esLvbrWNrZmvx\\\\\\\/LTQxBZmX1Dl8cUyHx7\\\\\\\/3cjy8V9rvXziM8XoIRm31MbxWcuS8sj+x7AbXI62zZ2yceXjjtIY\\\\\\\/9+fCT1NoCbAhzbKvGfIVAuwlIJZjJNLRUr0G+BaMF\\\\\\\/KTP5ljxediB5vERWnPQp\\\\\\\/jr3zlEXCN6cZ2EJhJhw6KqrUUEu\\\\\\\/rBKcYmVymzRoDJN4rRk4zqljexk2O0JWX0WdzgA7JCZqqcHWu5Ow8GW+hh11jbL31anzIvT11o9AlkyLfMHcneexwHbVJy74H18qliSD9hxTIs4DK+TpB0vMyF5Gmh7koFuelrXwK3+iblQ2+v4UREi+8+l\\\\\\\/9yuNNt6apDlN2yynluActQC1mLBASwsd5IFV64Y4ccjvpq5jXL6QjMf1TwgBGuiRmKlkB+UnAIRqRPSWdIcb9xdJbZX0HANdQt3Ndc+LFXi2l69fJawzaBf7dA23olPtr\\\\\\\/6pMOW7epu0ZJru\\\\\\\/Tl3MACa0LcDmrPq5hDwdo\\\\\\\/dLv\\\\\\\/tiY4EWpI2zxYTdiqY5GPWgHf7eWR95sVEHlU8Eg3OLHYBGV4mrtG+NX\\\\\\\/n1SF2MWC1BdkU7WHZlysLomEZs36sy60yPXbKoK6CfK9uHw0PlG5Cbqiz27sn7BIbLJQzjkAeHGVyo3+ONVS1UYQ6y\\\\\\\/0s1M7XDF5MVUekHP7zlMiqJauv2e+X0jnakdZ\\\\\\\/mQbiJl6xUZVyqf\\\\\\\/di+DYBWYCkuh45jT6ciN6Xa05wrIqbLLIG2UgS+ee5pydldvMG00IK6C\\\\\\\/9uo3J5rvGjNC1I28Q2XJm6\\\\\\\/qt4w0taBDd40Z8nWA44nVxUYA4by3TL2z4nOLYSHRX7HnsxpAOGQZ\\\\\\\/G0ZsRWo2FHqSv7wb0MeaSFQLNPw8Ncbf\\\\\\\/lSZnkx6UuNp3ygbPb1LW0\\\\\\\/uTKmqezaJPhYTS\\\\\\\/h5Qq\\\\\\\/m+xZv27DpfLjO7TLY1hWF+\\\\\\\/pw7WTVbSQkwTzwVrBhdjBMk9gAKewgvwQYGMQvVyHEeU7D2DE4E1HzPVtsJMXQrvBnyFcwUu98qHv7YGtez3X1HLePpedy33zk3q1neNRymyKPlfgn\\\\\\\/qy6lpHLNSUVDSfj5ziGv\\\\\\\/l\\\\\\\/992JRh1cBu3+5ZIh5h1iiNi+gAIpiEb0eDIpn3Zagse4WC9d5+cvHvKZ0O\\\\\\\/xNRTttCaQwc0FKuGl5dFHu+8IhI0f9+8co9L2x3OX0kCtbbBcycAF3tjQgUb99qO2033tXvFz4gz7JDUr96WF7VicixWRzw71VNXQeDQ\\\\\\\/gcK+mD7y0IBazeU+3Wi26qnc4y1fj1TSOC3ix0C1sm0JulnVzjAOvnU4Jv4IavTyfsezArPWsZJhPZ9TqVBpU0+5P\\\\\\\/tv\\\\\\\/sfNs\\\\\\\/3OPAYz00lzVjbzLtYqJKwOF4deHvpdoGhaDOfvVZldTPFJGXm0tnlaigUsYyaRUSs+LrLF1AWMFqqY9NN9CO+K6kpIxKavUPNwYGsdl0ACRh\\\\\\\/YIEygJR94iWJD7YWvuSrp8C0XhK83T5O+MOhS7Ao8m6iUo4skcV8MUQLGkEBlmBcLII620H+GxPkGCpD5K62CqKG8TqsXbvIPUQRmhE2l71734oc\\\\\\\/RUc1xQI\\\\\\\/\\\\\\\/l+DM4yQmo+ikr0VRzeBQyClTHgSz17U3fofE2wgRgAOlp7Zh69EBN+ygLl6FdbdD7uTAk58awynf0YKsG+qBU5SaC\\\\\\\/va\\\\\\\/yvw4MGVUds2MmEGXZchdQddr1oU2Jk\\\\\\\/PSPHDhBoJAHRiFZMRaUNsx34jUbPDky3GYHrsHZ9rcLslkMF1Y80SIJNxQaoWNVW+QsHMVbEG9wjJYup1xpz8SSGhSVDYaC6nsvVOZYyR66k5cWbyxclGXNbZMDHQs6J01yNZ5cSJeVL3QVLD4drAVULPFuRpFUjp809JHe1fxBLyUhohQ+sEf4bduAyctoI34g2ImkLBUbKzfTPXYTptW278nwenQ6dFMwkNbgg4oCPjAXra21ni6lJMCbNQKwDDJWownWNHPawMbzBf3JQtbH+47ZbIFgBWKcMqGqgGbAvy3o0x7vMF2sX\\\\\\\/pSuMjWVG+JttS31I24URwexrRsI9ACaVjJDRRfZG8vA2314SABJ4KeWnxQl9IulG+5JXA\\\\\\\/4TTH4KxVKfpqhrcidkrhf85mySsZEv0DegpcGmbqEI7GJnrQAUgo6nMl8E7mHx0V2S4cZMff\\\\\\\/4HoEEiLmEwWvmGNYAo9mURYpeNBhwhfNtHvtoLBBnyVga7OeYaa9JJ2RVjmmvb6W92Nq7sdOPotfW3IxyEl2hWSBBnChFP7V0gAhCdskvqvmKMSvTOgAS6M0bxPAJts1cMBFMsl6sYaGxly3XaLXyI9625faVoYIfItzItuC+Q1SGYosp8\\\\\\\/eE9QivnWkdt1RXKkjXuZfcsofzHTXU8X3fDS\\\\\\\/55sp3hk2BnahWqk44fJCGgCt0xHsiizvHoDDABQlPCxU2XWGPrEbYu5mxTb3qau4XyQYYHEz9Updwe7z355dzXRzS2U\\\\\\\/XDQMAD9MuWuBceR\\\\\\\/wFQNJQ2S5q6h4TMD57s1jFcQebyAX+gTYJyPPMgbNVkkjrC8y9F\\\\\\\/Pt1IwxZpWbgM6civSTAGl\\\\\\\/LGKCdp1pWvcvD5YtgeC5R3olaNrh+nXVDm3lvIsDMCrnHNZm3wI6oel8q12TFPM19dsSm\\\\\\\/+VY5n2iaqP75HQxNoAxowheEq5\\\\\\\/acIUPR11ewjfIRlFuKGDCzt7JxOlft8wQPwsPjz33Ur2IbtzMZ\\\\\\\/rX3nq4CHBDZHqHdudqGyeqoEapf3ks9kr6kWmdOZM7mvo1QKxynsNi+CkBh1mAMqAW6ol\\\\\\\/ObSHNEM8GVRz\\\\\\\/Lv6buGNc4oXuBJ\\\\\\\/hrOahldubPvvbAHP5j3yXA3sJKIyU7As8T0YBCzSn5vqiMwcWxRF8o80ixCBZWJdGtOhuzRvbuTc5dOtFUZ5UDLmYBzDRA4IVkZolRCbso6TdxuH\\\\\\\/cPWVPti3A8l6XBpt8I3GVjDXe4gwAvVBI2t3eXRkAUlhnu+JjexWIbtIQvbYzDQO1DuKA93QGgNj7KuzVF50K4keXLKaZyNFrp00Muy4zu0SodGlIPiw84eDPQVsyxWuKIMkKRM5wWo+gx\\\\\\\/e+46E5KJ9B5ayC+sAaGOtYFFAnVYzX9xb+jYw9AvuifJuw+E\\\\\\\/K4GpSuEh3dwPL0UGJ1wCXwDtsRIdtFFQmXhrX8XMFcJ2MbFXluTjhB3pvVg6WRN13JtPM5v\\\\\\\/DwFNkZkVGzDvuDxcmuhjMbe4TRLynK1fmSCUvjFCCDf4Agjh+CFoKbsCvQzYOGgFeGBxVS+gyT61H9Kn2+MP8nU7mlRwhr\\\\\\\/DEpu99N7W8qIw3Y7TcTL8eOcTLPl7Wg57Tm3hntarf6liqNksz9wGW6OBUienOGky45SGfUXcRMDRvfr9BTEE1WpfpPnlx6EJfV0CAvX9O50WthvXVvf8dLY1bo+BGcYFMXSC1zla3E5gyJjC+Z9zBXYrGFcLcUoey+0MnfbCrnItzBWtWJiWVcuuDpe9r3BawZHLMRxUjlKQHZDWUlE3eINOKgw4ECQS2nnlp8bhPI20SMnjClapfRaIVBO8W+w1zsAqw8iP8XG\\\\\\\/Q2o3TzVNxdLKWvNgxA\\\\\\\/eGnCzL587jYvg1D0gb6nrkEHGW1sh2mRzQYxJi4s7WweZ2JJaNQaORB2K+Bm0cFIredE1SDNSgm7RSC5BHGSnJrsA0xpvaOdDX3zf53nTDk+APrlkset9WDF5jh3OtoP0smdcRDTd0Ym+60nBzcjUHxWB8ZkDUlhfWMJiAs76YJjBtlqtcIiVP2cmB2IEk8Um44DZsvX3RYFDIRDqgRH2fWmTaaec2EIDjoHHSkt0JHQxRnFJccOZ5irVXq9n\\\\\\\/lri9yvRWsBXdqS8adYXyw\\\\\\\/oQzDWeiIVvygd3m119sD3aGVWVAlCPth+UsxTJ\\\\\\\/GR79agfr49AfiwKfHHKEVeXB9IBFPWcMD9t1+etavaVaaVERZhfvvW4w4hhkIw2mjlCI+0+3GanADN0WoeKxtcMIiTt+T19yfGE36mLuNMXPFFUzZv5mFd+JU7QOTumwSmpy2AzXJjcYgCDKhSCOeLCxoZCr7oHFOIHIry\\\\\\\/L0SYfTbgmSJx2QnFVSMsmc9W8fnSVxu6oWjfOIdt\\\\\\\/8Mvr3R4SlQuk5QQ85H+AOufBQzCSyDHw6vBXkraEHy+iF8yTZFvrVtlzqJgIibBFJICjYP49LMquHQZ\\\\\\\/pIKH9udQjJa7Vosgu9heTJxtFUkuQpz0fbwBTHZMlCue9I4p9gkXbqR+jKlR1PjEvCTykBO04Gy6x94pO2bQhvGsi8JtQm10YJs6Bocs3izG2rvtuotpX\\\\\\\/XP3u7RDPvZLiuHeQG+4\\\\\\\/XkjCqutASioomXNpYv8+iRkjHmdRmFCb6TQ0j2seIBu5cIDClIDYb9rFJrGhWn1EszZ9WttROwvhC1bNjMue8OULr1lJ5Vke6\\\\\\\/CBCEzawzVtYFQLAgU8m1HHBPqkQqN\\\\\\\/pfgOcMMKKRG71AAY4TdG06ZBf6dn6pP+YYjJb2ybVlYN2pnD+vWDug2zDCDP6QY7gKJ4pEF1Eo3wPoJSgKx9QVPjXTH8+qf0nkjxcijif26b0NzfPAL3o32pFDgN0cdhkiFgtT+FhRk30CiRL2fhooFOSgeBwcFp1jz5SBgQCZ2OjUHYnxFxqhvmBKSesyoZXhvyUv4842\\\\\\\/zpKCsMjXRIpnmwp1Fg+xximhkSv33VNtpchO\\\\\\\/Mbc+Vv75fB2kuxJD9IUdGs8lOBbFhe\\\\\\\/rYwIoVtQhvQBEYEm+ALCriUigFcEv3L\\\\\\\/WLF00MC0lBgevBUuokiJnNsyONRQrgSMoZqYlMy6sHtFRW5B5EbL\\\\\\\/85dSnK800HxPnPkVR1MqsAsQbE167mhZBWPRw7\\\\\\\/z7zXGfx6hhOSNGSC8KBCdo79tMJkIvMyYMOE8qHkRwibRkBIdytxaMn+E2DUIfELCmD5jfzIUPZEJdAF2\\\\\\\/b255p6vVRFZnKyVZ36zoR2nHnUKzVB\\\\\\\/IwAac6g3ofdeJYowpz4DLyQYeHRR4+Vwt9qS2hRHoYcu3CU70B\\\\\\\/IVMeMmw3\\\\\\\/FkJen3qvTypasoZTIykkbONIcObN18s9MtA\\\\\\\/qnwDSEIbkGSiDC8IlRB0iewUvIBfF26PZH8j1f\\\\\\\/22WhZt5eMGgX1gIf6fzdeGlSzZZL9i02K3lyFkvu5n1BxbVo06thOGevfqW9lOLKRsfcDsa+2W9xU+GFN43NqzW\\\\\\\/SYRUemKO5GyDAXjbWK4ipL8HjFM3lanMBH\\\\\\\/Sz31tqwAwPdHpioL2H5Js42XW5RTXpaO7hWZcHilCr205Js89LyZT98RT58iWOCxpJ70Ix8u5ceGaXLQeKBNfBzSDfSPy2mzNhYwGRee8MhRwwSWR+Bmk6nWIczMEyCaXNJ3CxiK7I867uUvU7S7xb17x6Z8cvFA9SY7T+jfYcFyntMtIXbV382s4H5pyDNCH7BZ3ysArVfr9t0atO8G4pnRHpEOeXD5v7nYkRtFxP9B\\\\\\\/ifCK5nZG5RzlIbZh67rf8rRBVuaqSv\\\\\\\/swiVkEPT+BWDRC3sKVN2ugcIVkiXdT6oIh2xtjwhbGeYr5P0LC1jgodqu4RibQvoFf9nsMsBAL+vqWcdYxF2+9Kqi7B8xcSIKDXaU\\\\\\\/+RiePWlvQluwaoU1VpS2hTRIyml8nLsC+bE9eB7YJZyR50W916DYQPx\\\\\\\/ecINOgnWSu6tbOBJe9gZHyrP3zUeurMSz9S7Exa5DpdNh2rrMr8Md+MlZXrpjzNmNxm0vcuKfP0EbER\\\\\\\/sks3IScybleMhHtqTXUO3NEb5U62ueIK9IRWchAHKw\\\\\\\/LYUxvDnQqWo8t9Huz2JTY0tqE12yKx2p0k+5bRoZPMQJJqNvsCBFQKF6GVrdlIKJ5b7Sp7D+NqWrE85gb\\\\\\\/Z4pejNMp\\\\\\\/gLPISO\\\\\\\/iwl+RmyKS4aB1u3nRVR2s0mmsd4IucsDcRpohXtc299YeH+52z9ONbcGJRKHFzGCPsegpIxwQk4jHL+lpB6Y4+C353pyZnXSoucAqtJk6agPr5geUkSGQiTqCwK4aae3VuxwpGzqvRBCEG8MwMXfJHf83QFf1p4jov7afnA39ixueT1tGedm6n\\\\\\\/r\\\\\\\/G9oUVHatZwnjWaNR+BMsV\\\\\\\/ndWhCsGOQmj6AKWGWm9skQFC4LBD\\\\\\\/7PDo\\\\\\\/qN8wwmlt2fd83yRoLXY2ZasZva1hr7vKNwbFcYQyDNgN8BFKO0IuTtytrI1U75jurKesa\\\\\\\/HHs7up05BScG9CL6IU7fges3b8jsndW\\\\\\\/36MfLf7MmEAiMMI8yHwe1YB1tJcfOYjzQIJJ547YKw1cqw7tYHjDvDhYIWGU8WRNOhwzd8HUrQa+dEgKsU3TGmwINzn5WghSO\\\\\\\/R3BqLZe8ePoS8bhLG55UA1Vtnu58q8qYIrIMtVZVPy7uWHlysdua98qDdJ+zGFpU0ix3lw0Wv\\\\\\\/Fy47M22jFPw+VFS0IyoC0QUFjgPAhUEDY3AfEio8BvMqWu28uH8WzKEgH47aGUtW4BewOa7Y6nQg0Qh4ZTtvrzSh5g8OncIiiLLxYBsl40axAoYx8nVyFtmBsyauyD85s\\\\\\\/0PHVFql+GhdGEMZpejevhP30dUiu6A0faV6FCTbJ4wqhk\\\\\\\/4FpU20phf9Q\\\\\\\/47NNba7wBR004PPMUBruPcAUswxpv\\\\\\\/SS12ErthuQCe3B9OUEp2JVTqHQk9lWIF\\\\\\\/QTn\\\\\\\/ngR7hAns9DbbgWRv0ZDOt6V26fmLqfoNA1d6ypzDmMDWjY2\\\\\\\/aLCHB2Yz1eRT3x2hAmWWANpqB0t1WWyoRKPM\\\\\\\/w3ENtTcrlgsz\\\\\\\/ImO7ygXcTzt5TE\\\\\\\/TyW\\\\\\\/6f7QIRxs1CLi\\\\\\\/tZBCY63YfMUXtWmlDC1mx8qveFGTGlTRjYCBY5viq5PCkuW41eu3yyulnN9YhQ4RiRW1efsSTnFt9\\\\\\\/q5wiRICSo1fnUV868hfxF8SBfioBWiFCpQ2loe97VyKyi2iG35yvctR+DqU6UP+DsLFF0zpxq20UJO3VBL1n+mV9U6EPE3mnSs2B2BOS+iD6X6qDjMH8ai4s14XjTUuaTYQigdGeI2C6n93UaxRKRCFGO6IwbJsr0cHWXnlWgRMa7HwwLwdXiDtLHZfjp5cfKjYgdSsXvEB9j3akW3D1OrW1APWuWGjjcTdTnbSmLozqkZfoirMEwkkReVBXR8oFimPlQF8awNK\\\\\\\/b8LJfrIziINfXkfqaz6552JD5m619n1E4vzHSfghVJjN322HyGgdJiRMdwFK5cPCUzJ9ajrq1S7FQZVlQo8p8QuT4W6BmL5CHPxQPAViVWlIP83oKrc0SeSEFQ8fE+klp5ba8VIN2SMiB\\\\\\\/yFuO0XoaFfqgObmWd\\\\\\\/KrO82KXO3sefPe9V9WMkqtL6Pk+Ib+y2V640M7Ldxm1qwloH10Y6hdWjGKCuRraJSJkKfaIn+NjaruNXhzvD3dcMa89SosUAhZLTUxpYyW2IdgiLpsjMDf10fnqso0iFXbc7u6GSo5Iia4IYaPYFnadc7T3Kv0EAKs\\\\\\\/MSob1Y8Zms6lTGxg22G9G+lE4\\\\\\\/a9\\\\\\\/0bCJ4D+26f3SVs2gOIGGcoeKxPxgwjg\\\\\\\/Dgo06cSuoQDsi2Q8ULwqzXR4aAWwpEUW9ldvB9xxY8sdRqzuIP4Wf3v6oq7pn9hKDEkZ3r8OEtBSg3ljcRlYAinfO7vmJr1GKn4nZMMpDGHum0+yIlcsErOP2JcGAZ1gWF+lNN0qvRqtBqbXHGg\\\\\\\/YlGDvrpslrE7XZjnwd3puGiRvT54\\\\\\\/dRJ9d3\\\\\\\/tKnlYM5p64myz\\\\\\\/0Sc+pAlHOilRacPEUkwldv8yufJu0NRiC6t+znWbvysBM5pRRLg+uIOH7aGLYfusZugT9EyvxqXzbb5XnxgYqn+2YjS3QE\\\\\\\/\\\\\\\/gEPUYdjxU8AlGhnsWFrDkTaAi+Wbot+A1G+OpYUesWRt+5B3syAvZvnWgIU3lQWSfg\\\\\\\/2H0FkNRKyS4mvP6shrozwBLird6Hqtw9nqeiHCTjuALG3m7Q8DwN\\\\\\\/eliPb\\\\\\\/70SXxA5uCrETMoFZmGIfviOAMmGlQqeEtz+Wn2pxPm2SDEGZXhS2ptaUzxUtEfN8+72wkzl6MzYnhXZREUs64Z4brFGVNInyN+w6zVBPsXVHvpIBPpFBMqfCNO3xmL4oEI6AiSObCYbXxBHuvYmAVKQYwXM9dZPdv5L+U52SCjWR7LFy0wCNeAk0KZfZdG1DmBvvlj2g35tlhED5wKxepkIkLyOKAWro1J2G\\\\\\\/uxFMplE9x5lbTDI+wYgUjUozUwSGNPox8Zl4gjZgtKUv0WhKT+NDsy0vB5rWwGf\\\\\\\/oiJDvWKD3tNrdSbiQjx4xb6DqFnEnDsJw6IVG+51CjRmBv36N1JibfmcMSt4TVF8GX1GZdUKNVnpZI8oXIM7ulgxbxlaIaQvJE7rsMl45S9XoBh4hgs\\\\\\\/FLaUo5gH\\\\\\\/N6GN047GYHWqUKg3fUfqwq8DmmYpqPqd5\\\\\\\/JlypHYn\\\\\\\/KIFFZS9FMoU8pStBKMv3VMEIbYdemRprPnUAezYtG5u2QEJuFkNYNeLHa8ivhq4p\\\\\\\/OM2YWERATbwll38VEype\\\\\\\/NJQR5clgTZPdNfcTgfCc8WgI6xjHcoyW0Qu5018x1iCbw2HHoNkI9SARSQEj6KDIB3Qdr8wHyveIvY6BxX\\\\\\\/TblIk4nBVSBzTjICzRF0\\\\\\\/gaskl1+rFgFhU4FAeXh1MWfOpzRfa2+A9mN6Ezmoid7ecWKrf8qN1E5i3QFCOcXplmlr0BxiNoke54OdpxsNIB8uDQU\\\\\\\/IcjOu6KQAz7rqi9xfIu01c2I4O2LwJ7jJTa1+PIEgxoXsk+2AJQ8nkw7VrbBloyLdI0poCsXyPh\\\\\\\/8EzA8EJ0iO26HV9Qc0W6r+Q\\\\\\\/iO9pE7Aow9yPBvJ70j2qoD5\\\\\\\/n1oTHkOoPcekd\\\\\\\/ppzKqnA895i1Sf\\\\\\\/ovS0hf0Cy6R9Cjy+kyqf76+4GH5gfthX7mxfeyZdyHTKPYD1M5cHsGQ2yFKvxsMePgm+OW6GnvtRMUYoIHppfPkZLlJ28EGKyTbg0hpfHqy8p1U3e3fycc\\\\\\\/QXZrBFBrDP9RX2B2a3vrJK4qi9ko9AshWNfVGgYa29fazRl3VZCzBRH\\\\\\\/JAzqTKIu2tUL8C12Kf6RsTUJWJesHR1Q2fbXOuGHBOpGOhmLL\\\\\\\/WmDbdB1m\\\\\\\/2PUKJGZyfoCMm9Cf99RR4vpxGytnh919yoAT51zm1L1UFlIpMaIBHX6kndu+Zprjy6\\\\\\\/STw8dSkxDWRTtARCQQ1pWuIwhLkYO8cjRxHYnn0KQUK189fOlyNN6hZSJXkBbQ5njxGHwuAMeuW6Sn1lzNaz\\\\\\\/9FvR+VknZ+keGSpBDBuQMiOd6If+Px86ghsuPaRBlNouCxtyIIkK\\\\\\\/0XTJnP8YNnIsfLajKoPb+c6PvbhL79D0X72bkKsIbnIZrxqBKKnpb00F5OyOcWBW9gx\\\\\\\/NCx0G2SvLv5S7bNVFkIq3e9gTuxtrOsX7svUF91nAf6RphyH2KOjUPg1qDmZleQ8fCn38n43\\\\\\\/9GJxg3f5z4LtRzvzVn6Fvc1HiNfCHJqlliAP6RWTkrmq9heenwH9MewTQ\\\\\\\/LGimq6M0cfGFjlQ0eT6XwO1RlL3+H+3rQw+HeGr4md3IIzky35+9Uc30wKCa+AVcKUQV5MUXTYc2m0hp26LPGhQcAkTMtPCrZqDca0wPVw6q17ZKuGLBsSc8WwsxUhOdhowRSPxDAvSQ6A7HPYwH0DCwBQUOl9inQd2GNeY3m1XqI2jcnPntkIhaZf4ZpZCvBZplVWGHET2VKYfBZWqp2SkWmfuOY2qzSnWpoHv4Sks2Qwnz5kAxqLBqy\\\\\\\/5TY3d2boBrGGvgoAYUR3HsES3e5yUBiamfYPxMwkKP1+Dp5DBw91J5YrXi4oHNsw9kqjborxcAN3TP+q90yOz8M+0d6uxMkeHQl1FXWeRoVlLYZJDYnnEBHvVoYBH6FtIQbRDswugOUQfO39uF4hl4MyHqg0YNJkK8T8Wqq0j0ux\\\\\\\/B8lzjKVm9zLr2gCtjKv+WoalzOGdpJi5YcPN2EDV7cQCyTAgx42sRi2T53Q9EoG49ZKahoNQn6XjDzw3MChVjVUMSeAx5wlJMl4+4nxI7SpmA04SeYfL9+0oG+gp+4raAM9KVyPo9F4TblCMbTUDrViSFHKcPaLlV9BdKWi1Hsqxp9qSVoeER7woJFDdM9B951\\\\\\\/w2ybwv3laGhyQ33YEU5v1jCtugDRty20GplC5K46EFgp6XzEctRoczJ8RvOFSJp50Q0S\\\\\\\/HY0zTt+qhhbl\\\\\\\/iCec4oCy4H+OQoP9itsSEX7fW4uSpbjcpNE6vFegBptdmJ+kc7C1Y\\\\\\\/aU6e5xNPZAx9MPtIESbhNGfIatx3OUWFPEL9cBb9QJK+dVJQV1SQSnDAxnQXLL++UgEHuO7hSuIDFodbctHy\\\\\\\/GUHyxCHUNCiem1sAwRK0QT8TOHzf+s\\\\\\\/Hlz2FCi8Sg2Zzba8uPcpvfMv4vEXUX336tSOgCcSdSoSkR3p9hUPfNaWi1mCz3aj0NDm9KubVaRWjVy4dfQ8IFHJUBIE4t15LH0r2+wN4AOFw0AbKX8KC8D4ZJ\\\\\\\/vz2kfdJyoNhAjxAs6SLYYLgiWnDILeChrEVHcBGArTvA++2vUg6T+rIM0AUpmMEsg59ipreM8JL29tO0Hzfh5j9U7+YnI40j0NGk5+jyG2PNYBInwpqE8nj3vFE2fRWqTQdmtA3rswH\\\\\\\/gwgen8\\\\\\\/GFFtgJWEp0rfYzv6QBitfG+Vdo4oUplz\\\\\\\/prpVOdr7eMvzw3XPx2km5+iT75PaREjPqSmQ0ahnPjj1S\\\\\\\/eBdiezMlzbj97z68\\\\\\\/vWkDuRIwWzmdmPu+D9I1VlXX+OI1rkBaRmv5rWaw90UrdIhWw9q2jgfRDdFegGukGBV12NXjH7Ibs+hHL2\\\\\\\/c3S+XqbMn3xWjZb1cQGqsnaNHhL4yO4iAxGnWxOoV76sdMBOKRpr296cL2tgR96PK8ug0KxM\\\\\\\/PrCJZnEUWxSoXp4UqNb5g3wemD+uRMBGThsE1Es5zjh4I\\\\\\\/VtzYfvpYwy0sKAvrNyjupsIW3Xf5fGqbzr822Y8uJ36UTkf8jnJmEGfcsFoZ3Un5lRwwWoPdrL6flB19e4DQbDqoN0hBO9lHGsP7j5n+Kp8ayLfoWksqE6ShhN34ElJjLblrvJ6d+MMKMmzkDUJ4tr\\\\\\\/G\\\\\\\/b\\\\\\\/\\\\\\\/t3jIdYtBsA8e7rnYKpydffC08hCc+of7f7J9NFr7areit7HSJwJvUafql+53aFQ0KeNjni6dfIrsysPYYqhyx6bAZlIavqmV5gQuwrho94v+V0U0ZS9RwsUkSDXFWF2AxG6qMI9FM7cO6B1Q1o41CBmYeewpHlN5sCGamw5xHp9z5yeFCK0d8NF4Nly0+y3cUnCHPGzGfu81AS9gSXY3UE1wV2\\\\\\\/LIPakYAoTDj+2xjhhUbTeMi4KueQT1AVDqee\\\\\\\/IaLyeWiEFPgRcVoGnDfxLq6rUEwSOjojbGAZWvXIIpa3k1OqJOQYiRLSWenxOudiVgFhj4OwtjVZt0Fd2wX2AP1O\\\\\\\/OnM2H404tTv2nh1J8P\\\\\\\/QTTU4i5ufU1lgYU7b6Y8hL75HZPYnTzoA4GBNXva3XrUTBlagk2iotLxFeSGAgMq8+ErU2ceTtG7iF+\\\\\\\/hkWckVZ0LKEqgVmJt50+9HqvRLLZELe+QO3kkOBulF69laZdAX3kXsyhVXqFlGHhtWvxBleuz6vR8EdZUawBWd6lMPk6mw20yTgBSZ4wm6WYqUDUZUCX8M\\\\\\\/FkA28MHnxPjq8Y5BcelEbkoLPzMmm\\\\\\\/Yxc3+2WaIyv2lhGrli1YJWv4c7zDlf553cYM7muCNmcsNDVsDnkIAIUKOBGQX8ewXvYaWc1Bz0W43+7JbU31zjp9lDVgE9huA76p75AY5G4TzKfbggMHAqPsKw9tD21\\\\\\\/LyYmcC0jEQTTkQYkPU1eO1uLVYjS0h1kdlWwYZOs5+qKW2LnKidUqlMaSivWmUNJvDHmQT0BtbKjPb5GHA6uyofdjxGX22ph\\\\\\\/ooVMh456CyQ6ndvcyN69\\\\\\\/DV+lvyI4rqTiCwr76E8D8oz8b3cmEbUZ5P0w6Fmfo7YrhYW\\\\\\\/3GjnPdJpzQAXY6JNuB\\\\\\\/3N+JlX2Zg\\\\\\\/B\\\\\\\/mw\\\\\\\/7IITwzYd5pgq2nP4wm0e+c4Zkn7G4OJ5+l7fdwnlpEgZPWVpU9bjxliTO5Pk7\\\\\\\/+8r0PXqkjj0sn+xQ0BEQK4+ZGRCbatsHleQiUPlhQ9i7mVOxzvG7tzzNvhBe7MhHJ5k1kmVMORGgRGJmEQRIz5uUWlM7iG312PnsSgZtN0DwloGwFpWR8sBW2qBoaeNEspuaLD68eW321JqdWeyqz06dH+1aKyhMwg+rS7+G19nU\\\\\\\/7vA\\\\\\\/nrkLB330QZjfbXPtbBYz83H50d51UM7Ruyrpch7DS1Xd8ak755tdRD81pocEaHF9ZraPj70hNTbgtkZ23rd5FWE0CoXqWM7KiDjSXN89HNBIZb44rTJyG0+izAoUk1kbQitX79zS48QRVeea4Odn1+L5hLwkCcY1c2s9DDeV8HtU6TdneRI2tbSwUkF72CYbEGV7Ryja8DNnuGSoUfpWc0RUkZNYpbdl1i130ntRaYW+JiSB7qWiazrdkDikixa1bDZhiRRFqrE1LUMS4s+8f1lAsxeR1aE7db50p287SknAST24qsDf58TgBt7ppL5OXbtHW65gd6G6CuoNj7MvvDbqkUvIDWZuI1jhiquBKYTg3uURGQSFvKc73PZJEgtS13j5KdGmK6tAv7DUtcTT\\\\\\\/DwVJKQPGT0ESRTWA5W6EUAqpf1EUaK0ktWJg7oyOr4FUpGHZFvGvYOwikkJcZ\\\\\\\/TaVAgG4xIDEPiKbI6sCmVCb8fUDM0KEMxAlSsqmtJAmlfWff\\\\\\\/qpWGG7G09VEDXjEhihCgO\\\\\\\/4xtEMDe2O8NTOc+1TPLKsXwzuxabenwYfYfUO2hUy9SIA8l\\\\\\\/yqFtGZHbiZrgoTCDcsFrXTyf8Fx14\\\\\\\/b0iekPy0G0sDv4daDI5FATRD5nL5F5GDzuAaSFpVv0RZppXnelr3UhtBwhTIs3GrDz\\\\\\\/Cd0JkhFWzBpV5VNI5nLOI2wC\\\\\\\/9VsNDbqksn8tND0IQ+rrzpIg7sErqT\\\\\\\/Qf9u1JleRHJaWqDnZ3+pKRQz+DwP88HUQT3v11I+ziqKLh4Kj8scdKt60FpVarM+wnLB2EL+7apSX66d5yqlq7dXsnVzSY4kgMCx+Fs1aa5hTObvS+HuIWPJqX7VLJPp68qJdYd1MeDoiXnZWsj6B3XlKd2XNeF7exFLeyekJrJjFx1BQFT49PiG7fFKV2QRmnKEKN\\\\\\\/1sovexHKScEey3cfOJCc6m9ywwtFF4wWGFIftabLBFje2tghgHHqLTJx6kZn198JYOxog+HRWglObkXz8Uu09cY2o0\\\\\\\/Hrds+o5JtKpFWcZOWHsSvXLvu2zpv8jLy7Xp+0Hao1YY6\\\\\\\/at27zBIAn8PMfy6lNsYHgOBpskQWetqkqnpsYSGlG8P8HlL+ptMnNDX8L5ejzEbYUWJqpFwR6\\\\\\\/WmJkSWJ2JbmchAbIF2O6ODBuFnITzvbGepA3kMUjsEDl67DI26bd39RbrCnAs9GSzWSHBZjridttU5VAhZwx5XgyELKJUANZwWKCArQ38PuenCzV1ggcrtA4lQT9tMo9S9w0FPusWdz18Wjbz9jtvbH6uPeaAMdJjBBM+e9KHADiIhi6Ho2NmitexVLRV1MDIQywx1sq9sCTgxtOw\\\\\\\/xh7T9mfBzMdrscbVbe+heH6z+WlIojg6AG23ghHUxgB+sXdLLucgra0m185JDM6aZKH4dZIDAU4MewOeFA5XNxIeZ3fsspv3v7X4KwtlflAs228ptuKQcRI7Tp6m5HJGmipzo0HE\\\\\\\/ztnppLc2XVXAuJ+FGeRixW1Mh5m\\\\\\\/z55GJ+sGWhjbMlwEVaPncm1JT78VLpr0B4aShnRfg9LH1sC4m8HNIPxcZ6uFxbPbgTnZR49Qrg1pHRrcd6fyEAGyKuL5VoibGoWvzwwVPXyzesrc7q33qxY6eUAQH3Mh\\\\\\\/jidxGAb9kEoF\\\\\\\/lj3V4cb97xI2MKo4vFg9rOY\\\\\\\/T\\\\\\\/PO6aHXxeuZ7wIEDyl9507PzsbZ3ma8SoOW\\\\\\\/lvUS8zF2c2mj\\\\\\\/iblnKifnSokxfzOyHrul3vSGWWapel8rw+h1LGBo1sQTL1ay4p1JVXp91DQzIuMqkG5JrZLKdHhJ+9YkEN1fYYDWcLsWsnONGAPCuMwYcEuLUIaxf5WmqqksfvZfvBxJuI4zBTLk4omszxcEIY9y3gnUhOn0YU6UDKUn00Ki+is0AUUPFhFDL2ilKhT687c+j4vQxmlowiAIS7nwYdJP6dxM\\\\\\\/pYkFqcKZj4w1PsU6IV2\\\\\\\/1opOVl7DPq8Kq8exh2c4O5QMz9rNPElH3rNpCSziPx4H\\\\\\\/t7PHWo7ooYyB7Sw0vomk4juHXcEGIRgXfTJY5EuO5OXoYojdwxH\\\\\\\/xJIGTBbWxPW\\\\\\\/\\\\\\\/HJNFshTg4or\\\\\\\/CoA6C8mSWmgj9ZrxBPkMEfDbdfpK+qx65T0fnya4mj12kqqgl2Z3vEUt6zMDaeNM3hj9y\\\\\\\/L5oD8UFZx6iTnyYbj9tAYv+iaEwuaJ1G4G\\\\\\\/IfNOuccZ2samm6fhk\\\\\\\/p9mxXp5dtoi3ekkoh3YqdLZ1exsiK5sUMEVNr+jnoCAQCuu9rCEzUoUMMqxXGM5kcB1U7pSq7\\\\\\\/UZwOqn2+aJo6MaM457\\\\\\\/yH47ehvVP7wZFOs2VhOis5wba74VQyPLl5C8H+Orj5CeGdmr6WawqhkqajlEcLupivgJQQMVoMeA1z8tvM0ulD\\\\\\\/JozMsYI7fecXGmDbGbEYvSGgMZZdI9dcS0ibj+Taf6ei0V2Bb9DEcNHpiULfwi9KGVIhO28kuiXwuJDa0WhfbV6c5G1r4IuzRz4RmYl2egEwjd6LGuk8GqyBZxMEuOcQ+3gQ48bdRTXBNuT8UEhEPlNzI\\\\\\\/hPKWqx2vWflezGLKSW4dGsnZBTlDEvDnnKCz\\\\\\\/MbID5df6ojF0AX7CXYf\\\\\\\/uUvhipwU312\\\\\\\/v\\\\\\\/MtnJvFRmOoOmXsLnDcGawMHWPFdCyO1uxzms6X1b9JBe4kmd6FnMas7cczDyMlLI3iAAA5XncrDFcn06QHRtdtniQohvQceI\\\\\\\/vVAjrtQrenTlL8dWbRJl3k0bFaFCIjMZITqmMOeqFT7M2wfmWlck9y1ebGyTNICze+6ec9t10AN5sTvtPdc7ZScM2mOFW0QozlJtBkS6cmbKVN5f10n1kF+cKB7h8NNBMAKqMQJy8nOJAMBl8cfCx99Xc7QxThZZjy0n\\\\\\\/0fjE5wQHqLWcZYnYitA4bCQw125Eku\\\\\\\/R+K4DjFg+L5gPl2fgYtSHY4C4tByDpzoonK\\\\\\\/B0AN6vf2MqN9Hv4xT6GdJlBXdKwedrUZY9XqD0uPt53EC7jdFIyUhZ6D8iLKGkk8BWcPNrk1zm3fClsn3hwvwsBfSbb30z95pE70BvTLB3w4+m9GbRlK88MqGZ0Y6nEKzD9MnuwkFmDDR6paZziSPy5YWH2jUh\\\\\\\/yU8X1SQVETdE7aCFiCnO\\\\\\\/zRx4+Df7W6EaIfqDRL0fOQ5efv\\\\\\\/ONJPbUM1RxrufGktRR1Ieto8utrnMQne1BksjCTUtk1xWX028fPPegSB5snb2f1qjGzafxaNJPz3lAK4snpej2EaEFI5ZPlybrK4nuZ\\\\\\\/PgQijHwEMHPsyVf2+VGhN8plDnCjQDzYscBkt\\\\\\\/NeJQ89BxYvmSVX8gi7b0uVofSvgPD9t9pc9kMMrYxj2ogYF4C+vzRHur1OaZjudn7BYsM0wmM1lAK9BlskAMvLz5StG8fdyNg+ucolNUx6M1A9ox5DkErCCo87C4yyYSnwnz0RxEYKcGo4r2mxph7\\\\\\\/6PildjM8bbm5st1Ucbvmpm\\\\\\\/6QaWMCw1c9GnjRFca\\\\\\\/KtCeSXOF44zs84K2f2A0QwIrmk6iwtf\\\\\\\/bZ6qttWibEhu\\\\\\\/YUt49A9IoqNpLJwllFBGvV+DrUMGhaG1LqbqOr4aBi1UiASa1BRyC8nY6w15fD09E2TrZAxeIDtrQFSnl6sGjWtVBqOqMDx1XN9noBkkhEcdOq+g2jfhnizKl8lulPooeSn\\\\\\\/7tC2OxEZ\\\\\\\/WAnnVQOi53kF9g4lTlJvZZGnFLdBDITkRnXOisw0GqKSgKMNitR\\\\\\\/x\\\\\\\/Q3SrmDX4WYDUtG\\\\\\\/6TpuWDZ5ABKnpEDjTfBe5eXRvWM3xymX\\\\\\\/MG+ADETdmJ50mud6Ahr26+dUH8\\\\\\\/wBVxmWTGSXg8kgk04C7Kj\\\\\\\/kIcFKc94Qp4NypL4h7i9Ugli5EkKPdp0xHFVHHSMNBHY\\\\\\\/devYk+2hxv5WfBLvyisEkNGEsa9Ag8Q3KcZco1LGBgYgZ2ld+s4GYhsxyaWSyQzHA8F2eztl5aKVcCuXtiSfgcpQi\\\\\\\/vblrOG6NsnvT2cF1Oa9erJOJbZLdz5697I6iJZ6Tpdb8r6lYFvgxv2dc7X7BhxIYNuq19hI2R7EcjBm6DVkISkS7eS1qiCMJn7dxEy0MwQGJPO2zIsjqrTy8v8KS8VZOMqJK\\\\\\\/lYSyuXsFxFVtIFJEjEtdh\\\\\\\/nWOrR7VG8QI+vIEd6aJnUFt8DE+7gEnl1odO4gSKO\\\\\\\/5TGTO1a5uvvZCZi1gF75JHwU7S0yAPiRmQobAC9kIqOc+xGIh89N0cYOM+8NBuWYr\\\\\\\/UCj+SEi1S4rJ8C9ZYP1Q4DO1WKIqcEkFMJRueczvYUWLJAJYCfkX5vBnaCsFcIZ846Ab4ljXQNrP7AmfSnn2zFIRpeMZKROtmzSJqDV4sTnWjAI6tba8sW15vFvspMpBIgilp2Lif5DQUNun\\\\\\\/qC4z3G1bZgsoiMqEmbnaeg9f9dR8pDSG5TlwQIHOWmYrSQG\\\\\\\/vyXlbNJp5PGJrRBW\\\\\\\/9YKNGznI2dpmeQKeoWQYELKIFe8dX1jWQKZ+VIiIjYvQjnbNrw\\\\\\\/RVTm121vjw91f9FtZBL2TKlzuKbqZOa8N0rPfSyxhBM3derNzcErbkN0QUeQ6LdDn0ZetJ2oQ6IYp4soqoVyyAojAUBjCyPG+KvIzTxePcLqxDHUdC4QDQg0Pn2u6Hs5r\\\\\\\/8GOphlb\\\\\\\/g9Kn5QRy+7BBNbor3zKfZxPToYp\\\\\\\/fulp78iNU8EIi7VUpc3iYOP6EUSn04ba2T7ur\\\\\\\/lz42nEIeQe5cSqkjh7aRtn7IDsHdKgncd7wIhh6G0SB4GG4lKoszX\\\\\\\/uvXGstKVi7qahFvzh3FoFNRfIgStHzxRGkU\\\\\\\/lXtqGfNczAATsKsgmxUJMslcEsq+QeJrJbQOMHtNj24MgXrsLaStArquSw2NeOHBrCNykesIDW5QI+7JtvDFizDRS\\\\\\\/SCComN7e98BO6DKWC\\\\\\\/CrFWd91pteoBkCvcZ3FRq22L3tTxZ4GEdUMa3lBq+mG6NIA8nVAXwrqktffyXvBTrz17rF8Kah+asdf6IWNK7MQOL1kcAP17akx1r9VDqNI8caPRIagGZRkET3g8SzKW+9spPrF7r2soDDKLzUocu85PgqCMy4KRfytEfqdqqhEgBDHVKtoAW7420MTVjyHfoF9Bc9fdmBDNNpJ6JP4LjgPP73KhbElNrVewF4umsOE2s9UBrwDdmyqEbqtJRTfaXT7s7qIXa5GNu4fHtDtPJbxHnyMnSr9LoIe64E+kjNXj6OGKhKSRHVtu+eUExHQcfXEFSjevq6le5PMdwFOceqzOLaEWUH+jF5ZvnMMiC6XiQqI7qOBlXBf0qWvIpIEAOyrRLSFngCLDmqcr83GcJBV2KwXiREohfNYkKgUouYpSIrv\\\\\\\/MaPkIB34yEqYzrvjeAOTaaxfnzAPfKSGm5iwODOUAZBpiNQdk+mb7Z0ZN01D0mqSwi31PJBC5TTdDMcmrGVyaceuWAsWxBTJDBME\\\\\\\/CUTxvd2h7zSNQgp1fgISoHMgMDUGDxOcxm+DqKe5cwkYBqH1PWeWJ2l78qU\\\\\\\/Vz6Xan4ZvJcNBPvgPqnEQYzPEFq26N6rRCwM0d3Kt9SgGDNGYPkEfx3b\\\\\\\/dKhPhlElBjSB+CcEFJl0+YTTPYbDtNNZHJU666zu\\\\\\\/Ypw8wFbEZx9HgsVsm7NGbDYZqa6Xn2oa5f5q3tnTO6yLS8FZ\\\\\\\/6cYvbTWdpqUsRplv7xPAQIJv17POLmejKAth2SfRmIqN1RSwiIZRmMShK7nMBm0XZeMCBH5pVRWc0H0yScIPl3jG3dBkTylAz7As\\\\\\\/gi+Dc1N\\\\\\\/vdqlnT02A7Dj8MqT5DGEuIx60faltvjqopKeBeGavPXDEd48eIAJ70bFpXmCY1CViRO9jSuAgPh1sw6y9svyyNF9kQp\\\\\\\/hl3EBhm19aJo3imzaak5JaeMndhHPhT3ytDpntFxjc0uOlx4ScR6CewgpBJ\\\\\\\/00muEOT\\\\\\\/zahifV4DqkY8a\\\\\\\/6tnSywu47VVbqJhzWzjbqIqq7PersnKoV30509l3TxXu0HMZJcKlDBFNLU+emGL5f0IPiPALO3W0wKAYzynkb9l0Azfs5gTqOWfi0cfL34gLQHtYjjd\\\\\\\/jQSExI9o1ZY7eqQYfOhY9dR4QtAV9uOYTTDtBdZ+nflVhc8TN5SKfYSEyXrNPLzzLqtMquNYc57Q+ckUNuho4V9Lo8qrxOQXFCmbHLg+ovVgUGqlqxyxJ5HUduc\\\\\\\/zBqTmZaKKLQ3wlxzhir17Tz3Sq6lBiBwl1cEC0KGDn9c5NCOyv6+SXd4dqzByZ93BhwVi8W60u3U\\\\\\\/EOkEUExblVj8YgJPo89TVhFUWZ0oCQ\\\\\\\/wcd5uNYucDiVI838sZYTveYSZaUQXXbmXhevhI7zfJ3epzmX3aDw8CerMZjS6uG939YumYeIhE6Z8LmwT47vlSKJrvSL3YN8EaMfNbfNjLoR+EV4ImO8Rjfr81lTcfwMZsNBgzxj52dTG3eaLEOtfLRkli4LZdMLFS95JO1hidYljqlpa5NfEsrsysrv+1nhVjzBM8sk3TMiXggXYLK9m7avFyoHLo0wO8m\\\\\\\/oe3fVF9ht5LY5tC7pKVBr\\\\\\\/wbQutOW1xNmqQDdIbwqZvIh40Wwfrnmv9IsG1HjN29KqjR6j+TSgOj2bJAPZjBOo4bc+oKxkmMIcrkvVv8f1u\\\\\\\/xSQyhyi3c0l5BxkQpAJn4SYJa4\\\\\\\/Roiji74UReZKXFMujnVd2G6pprDHC3HlIwfy1cp+\\\\\\\/gJYPxQ1\\\\\\\/S9sL5P9e80a+NlszKqJv0C6Wdr66ZY692GYa32aERW+83SRLd3qGYOm2ro0kUNm\\\\\\\/0V92\\\\\\\/0iffi4CpPsIrI8KrB7bkGwKVCoQ7DdqkaFCs9gXpkHMj9vXyTiiN17HAWKFzTjDH9JLsStXG2o+hkzM9kkA1l8hclLJINoPJxSgWNP5Tj4otAZkt\\\\\\\/Jb4+0FXcKfI501NlqlNeTiyKArUctPKQ8a\\\\\\\/LHFVBvOExANoOLixSZJZx2J1ERvxxCsH2T\\\\\\\/iSTy3quirUgpJqXeqbfAQ5xsDu9wQ5Lk7rnKXhi\\\\\\\/qw+pKggKs90XTa2\\\\\\\/BwyXT5X5eYZITdlxaVofG7YUzFFGFcOKPFwSIWMS+eK8Y88FFlI2Zyd8n6irjY2pz1UVwTkp0dpRU+h+kx6qaY\\\\\\\/98ykZm8wbvoEOkLBgMKiCz42fqk7e5OIFza0a4qN+MP5m7JtrCMHfp3sm4V3bhUzYiKsw4M49MwLT67PURjh47\\\\\\\/YRf+vUd3IyJ9g39\\\\\\\/5eUAzZP03kAoe4dr4uqHCiXxflwDAXMppNAahEB0zxvdk05K6grL+ZmDC6E\\\\\\\/2GlTdYfsHgPtfV0QHAaWhSz+pzdresXYwbzo0++cFl559GYH\\\\\\\/SxEcrOZJB1XY3gbxa\\\\\\\/pA0z1VImQwJz8D5vmNZR+IuUWOFCbPgR73PQY+JTeIEcqVnBQ6Rpov39VUZCpz5tzqmUvw3om2lO3VNAVcToYRM7XQFeVdsLaEru5kxZU6\\\\\\\/5qbB3n9JeU5uKfQYJmh\\\\\\\/GndHkntS9qQIZxN\\\\\\\/obQKqf8t6jGMmzYtGZjLksn4OIcGtkzlIXoI7nNph9H9iCuIuFec\\\\\\\/+nbjoZRyjYl3vim1qw\\\\\\\/1R+CkwQVYqKt\\\\\\\/2idlyNC991MRk2n2ghcyuSDlkYjv9Q0HCdO\\\\\\\/6aTNWnYHW1FHq9BkZr97NUyYUMFsCmSHQSuj+Jap1d9an+PMlxfooEizqydKxdFyjxMX4KrKIxIAdwtflXYNSqaqH4bJf00VhwRcnQP1lHTn5qabx5thE2rjvweWreCztzP+vQhxe17bRMHXZGM\\\\\\\/2gKs+idelO310tgkbpBi1jradmPSp6F8Q5LHnPsJNS2UydkxnZYAzmkHa47RHtlebp1aZhb0AhF5LLISlhyiWHDNVTiG4smopb1qpcGewsm7B0+LevQVlwoSIu2Y\\\\\\\/7fZd3tfOBrkE26H0PUXhTOHHdlq4eU3lkP3CMQByeBSif5DI3VamxMS6ji+ty6hRvrpcpQISV3eic4iuidXmSZtebTWG+HrizHAB2ihwlUPrL95KNadwIFDOV8+yJsMOaTM338ZPZ+JAY1gygTi7NWmk\\\\\\\/eQkobMfwqWSZVw86+MyOaG4SAh\\\\\\\/uZF7DZORWHhLeYUL5GGk+RwL6Z9E2Z5Y9igPDz1Dz8wBSaByk4hKXm4507v6o3qrFXIb57lDrCMd5bXR5YaibRYvhfMM92xAy1L4Qntensj\\\\\\\/+hnBc6Dtc2mETr\\\\\\\/uob9jy380QZRDGnHmQIEfgruZx5ydFqWUYD4Vq9MjI\\\\\\\/A2yTf2sLuyHViznlUbjEBlA4gWeztkoKI5sJPDwj0unw1dcixQ5ipm0L4I8m880gT2BDkBMV5duM90bSUsoYP7OiaQl4gAbA8bTvBHjHPBqI9doVS1qVDfpYTX2XTvSTW6h4yCFQg71KbOFWSuCPAhOuWps1n7SUFMFYotcdUjkpdC6CKQydahNcDeScfT9AVpTUsgDvnLcRXnz9M3q5\\\\\\\/3ldHd9vPTNxpGO78a3ntAITuaIhT80NMc7glQJQaz3nD5A+\\\\\\\/dPgNALOpT1jkqZSkv0+UL7wPczbIVdeYEZoemcAJWzsACAXnZuC1ZAUo6Dbre0BniWhB3vJgormC3U2iCwWQeZ93lGatvRUhej5I00X5pHSCk\\\\\\\/AgFR1kShe2gOTxoAopz4rMaxayytOoiX9GjFOy\\\\\\\/\\\\\\\/rM0B+Ks4qCBjrFofx2scDiqUfaeR\\\\\\\/yXStdMw7Euj74SpWYQaN4nyBWrIx7UbSY28iL2vEEHcUdXJKLrEWeJGKmPZUPd38AUFsVe9QDuJVB6rIgQOoyfE9rsQl+pLYC6CKimGJSuh7R1LrxFm9sNSVYT5w7XZKZpk7jlxdkli9Wb4kxjLfM2McJONvt59gs1FNUNVPjZUNTQTPG2HAvat\\\\\\\/KKSB19TylU5c\\\\\\\/aLMTXr5oBI0C3+QzkwzMxtSSHbeKz5yAZWcVRAoJ+hFFpwZzsTqpbXCUv5OOAEjjLjZiTNWoK3Z\\\\\\\/pqKsr9KjcdMJdIpCvjxZyh7aDExJbvjCGud2\\\\\\\/ZV8uraajzlLn1qKjYmEqUSymd8mlYoaW5MFVIgzpALAlVdt\\\\\\\/Cv0xx1Dvgso+gOKiYs+euh+o3yvhlV1OxWE9BBhoTRYqy3cD7kKS+lpxeE1ukyIAYz5XSJ58pB81FqHMi4G32y9gDPgzyXlyqK6ml9Kdwz6rUExcvOrxmKvLUk0\\\\\\\/gDL73iUoDNtbZGJOVPux4Bz\\\\\\\/j\\\\\\\/ROd4FXowtSrMqesSxtk6UP7RRVXx8LfNjNcpkyL+cJaWC9pd1lfj2L8VBhKvM08\\\\\\\/NBHz71PNOALLATRvhb3+F8vVLJSdTtIxC+G69f\\\\\\\/dKywfmBFe7O2V7BCm\\\\\\\/oo6xcER6mrlvDbdOzwIlnrOf94nLysXR\\\\\\\/huaoH7anHhHpa8mWxTgUyhi3IrvjbKsIXAb\\\\\\\/hEDw3MRvLHwIMckchcxa8uf2bGZlmMFJYkom+o4MazPufbdB\\\\\\\/OkDPHoqwNhGs8oUI6eIXlnYso\\\\\\\/9hOJv\\\\\\\/zKfMyJE+dUpujUR23Ktt2wU+yUrD5nPTPxqzo3x72k4UJIBP+0P1Aqq7duubdBfv1lH+54MwBJQyIfbb0dOUnDn1NdsBTMfIcEWNsJLGxP1RB5vxVnJ8sJON\\\\\\\/5jwAKZztFoK155qhNowXYHy4gnkzPtISPexvTrckijWrpuXoAlDplikUsm1GFFDH2kuwSlkVUNTJQg5+52AacsNFknt5weqDKP8ExMOOEqupjwq10YLwCMbdRC2vsZsKtq+lFt\\\\\\\/3R9BBxkHKkqMH3Ha1SrT3IKXdho\\\\\\\/v\\\\\\\/wu2\\\\\\\/GDn+VGfiC58g+XNxm7ISkaseBtKyOtVFv0SEO1yc7l2e7UeHi5cerVNh0otDcVAit4qCDItOqFBTn675NOO+ocW5wlZvxBkofK0rFpVT8HLYJlX8HpljtQdEYcfj41Uy1q0wvfkxuMAYPe9GjNjSTgIPQsLvJFJfbzvvROV4CYbehUQtyHlSsDdNHqN8fzO4WOlPOqubHhmouKe+jr4SrgTfOVP\\\\\\\/43xpKNYqp9SVrA0qwoxg7wR1tFmxqlXdICXhV8yVk9WEzj\\\\\\\/svHSBaXQLMKJbLl2gSwTIgGC\\\\\\\/q7hAoHql+IeVm4TslD88RMmwflpClSkuDI1bVsQpHiDZ1Z8z7UInAEw52bAtW7MVKYahOnhBfzmzul74\\\\\\\/UTSxdmQqLKcmaRfd0m4KwbGj6G+YKfLGtbpixKr\\\\\\\/n0VhrZfcfxS8+6Dggru8CUJtbMSp\\\\\\\/Qz7JoyOy1Ve3MNYGZGrAUgyRNvBE7u7DX7ucatUTW+DloI5QuONDmY4zfJQ1XTKPr\\\\\\\/5EWMQ4CSaka3lRMckdIdGrvl1PL6D9njNThluL9VSrSi3yn5Fn7y09MBL84neLbG5g\\\\\\\/bp1czOTEOxg1pzJPDuXCXSIaaQRg1U65IHibfeY3F+q54rvyUfBmrYXuM3U\\\\\\\/\\\\\\\/NqrSTxf33LYrwTh7oGUCwAC7NsW1LHiVyNUdYRHhWyaNYTbeAmT3XJ5PsJkO4GYkauxR\\\\\\\/U7mxbzEfqNsdL6dnYuLeXOgXzAefNrbiNdowCA08ssU1Gjg7k7UoZrccFAJCtRbYg11DCZiak8kfJdf2+TAFLf1GLkuqxJn0X2F77AlkNQqK2HJhZKNPXtjuCunB8SZMw26ZT6yMl3RVXYuYMKWjtVBaIl8xm0+IV0AQSyjfwsYRTK2E2x9ne\\\\\\\/1jeKBpjq52OSBG5dtAFgaotPc6r75HRGhKav2JyXC4750Re74JHtP7oVt1IJi8j0JICIpSHEsdcZCje01yei3inQqVkMDgjre1kOqq0AVcp0Nul8uxfKQMpM7fVRT2GIU+AJKO3WXk0h6neLASaf6JZ7VOD5PBduTj\\\\\\\/ushAoF7m5FMr0HudqjiMIOV5A1Gr7Re2QqrTTkCM5q04dQoq9I5gZXEuDMN0f\\\\\\\/dxvMP0NN87Ere0h1\\\\\\\/9IxZGIKrjBVZCCoBPsGFpx1IXHHzRqMPgu2B9cQRlWTvvet\\\\\\\/VZpMnulAmTXEOrFnTyfGhDyjqZPw0tegWU8SHtuNwd+xTJ4Cf2cOEBjh8HfT1Fnu1mlG6jvDQH9fqZmTtlxyzpuBixmg4NGn5Gxn95Uia1yxL5iRGJwMBHPr2tKjAjOfRhFhrpZO9VFELU9VQeA+UmjptruvEwQog5tjNvUBGlFkgDToZOPkMXGUQ8E8cosSl\\\\\\\/KRbY+uke3Q+aAHQPhMrUmsyswZzWy5bnC3d4P9cUdQ++7ylJ2w6EnPbx+AW2Ju3qfCp0piLkeBpjN1\\\\\\\/Bcp6Lmm\\\\\\\/+WsO+6LF+ssarJNG4oo8rwUV8jI+Q5\\\\\\\/cyCAT2J8ckbjs+BTEmZ4S5PMFwTcfX3jJWm5bYXQ4ZnTHazTVzOSi207vuEkhQ2mDP5aBJURr\\\\\\\/Y5neS1iNUyHjDkUuW3C+BNw+BC1VzWuUDoO\\\\\\\/lrraEAVnijx+kQVqBN8DBn8B\\\\\\\/6su2kbcyEjtEieOGyAvDlPVUkippMgO7NBaxTrd8UB4NotL6JBQhRTYN5wq47iZPPkTMpO1NEuzPDxcefOejbwI3X6L8OjwqtsFGI7wgzpi4emq0RqW3G+kNWrAPyH4aLI2hcsYUaEPD1Q6YXKth9T9d0tM4bCJ5GGUBMkYD4BBf8+Dd03ydMMDMp31AAQAM+sTm8zzp3hJf61mx73s9pQ9TFGN8323toLhOFfnLIFHGr+YgYme0DOpEUTvMIbEwxvv3rfGfOTG1BDZmWipEccdMZgQBdA1WoWIlZ9920FP6NER0qOsL54gyUcaCyLUZ8ikpajwhOYx9EQkr+UFjvm0V9UllZqguHQtmVbRBxbKmjXfRO3GlpekuNm7plub0xo5wONB+MwuxdiL1WGFnt6YtQPGNrMM\\\\\\\/88pv3JWdmdLYskbufUbFUHiR2tjIPgLCJjSYBnDrsazhLkua9B65CoxVQwM3IWga8c7T7a6hiBklw\\\\\\\/IYN5dyljOtRAMCnOEALF\\\\\\\/DfJD8NHY3mZ6N7fV\\\\\\\/ZtKyZSxQXvrQJHpPt0IixDBqgLPC7xGniqzptyd4J3v0M0YjD0H3cKrv8KxHGWfzSWU4BwDuHukQe4diZqtMOjXooutHsN6LDzTh28w5eaRSMWchh17E3xNhqYU4YHpDJ1SEnejAmi\\\\\\\/kDXNbTTDQ\\\\\\\/CYyQtZ\\\\\\\/iDA9bEW4+x4D4Pfr0c3OPxOlVKX98mbIw61xrStS6Dr4Cxz1kdd7Mo2UPKa7NJIk9\\\\\\\/iuLG2s4rdMeFfKkxhdHBOPZujoU3ZKTOmTsysPLABp810ZY3cMBClYvmOvpANwwIvRNwHgQMMqL41IWTDbQu+rXEvDIcX6tjn5fX20anZg9OOIwEi1mVZx4rDymSKLrRsxENDw4EvQD86vkisgA8Gf9rt\\\\\\\/dlz4CQtVNx7BIj+tv91nzSgeM73cO\\\\\\\/ZP4cjldjwa8vFHm9nyQyOKFmmhmJ4u3kLaRG1Sd4YyWoKg+Gp\\\\\\\/dz+bdSdBWUSGmXg6Cp90OgGE1ZuyXdStAInmPPoOTEoewgjhuobq+AdfOrQqdy2\\\\\\\/OWDI75jRIz3wKcSTTHL78FKwUWCfwzFBiImYATdTDj7JiNL6V9s34Z130KuSIFZjtNoVI4NSjWFM1SW07qN5P41MDCf7AfWE+0EJ4lR0LPbKqctTQzHAb9nhsw\\\\\\\/1NoFgBLncnTld3b6PDODpf4SSTLEkaHSpqHF0n4\\\\\\\/Fyi1DKrbs5roIYnDodJsB26Y6BI8RM5bPP+9nhe0N+Lccyi7yxFHSoVTsi8ewQzTcBKU8YTxIhbmV8Lv2dMDNGG4cirSKoVHBae8VJ+awdav64tKiYjNIDwrCH0E32rmZZewgHvd82ieXKAqrSji\\\\\\\/Sbxc3jVY8naKaZwj942KQPszbx83n6w65FVtptogJk3JUE4qYophd07X3EnNTSkN2AkgMUbegZ2pvRVB87D3cELW7TBFQaoyw\\\\\\\/eHY1QcpxeXWyH4a23iYHapYVMRU1EUTLc1PKsX4dYbfQ274jdg6svQJwDLIk0HYofZ5C0l\\\\\\\/hzwZJQMuydhrNhEtUht4CXjlzHaKjBYx0QwFtnuTk35fNaaOh1uVTUL7sPJPIPCwJ4f+Odr0d8LXkmcNzD3oFYJ6ziAZXiiJ+oZVegiZj9Ozx8cpEVCEf0HbEggB+13NIcLEbjU94OOgbKERnxeJhkWg8SJH8\\\\\\\/s81iTFtUicHttJcq1cPN03Kvfebx8CMTL\\\\\\\/HsuPzJ6jDAj5i4mSMgUf30V4ekgcDwvRtVi7YuRtSEFB3QlKXd+al0x9NSnn2zJLAk1ZFZ4oXxCpGXTnrc50kW+Q3k9Y1CnNn9\\\\\\\/lCsQYY+nvdEkiS9JTsj6uAdQ5HAlwQAFQW4lwxqt1kEB\\\\\\\/9CMFwo9aDjHOpMTaemWPJi9OVVsx8ukQOswnMjnHc4fBvFY511Grtb7fDDo7dVQpjv8aVEBw6JuUB192wi81RFLU\\\\\\\/UD1aNcFs7olcj57CPLspqHxs4LnTF0Wegctopyxxeteg1313uELYcHkLSgHYETSjd1jif808N39aRN9fVDqAycf6N3zyfdJX4H+hgoOoLVCHdvzfQKefijXzEAC1NoW+U338FzLmaKjCxlUcza9HJ6nYfIHoygT5U69k509OSrYP0l57NU+gH+VJZlA0vp71eUvvxXiMC+DSifIABh7brWF1gAopiEcIFUNKWfYtmYWG31U1zQfpk8xA4FevjfWW+uyUrs9eeXTWRkYJYlYAZQbrtk9H6JmQ4fcWzqreJ3+Ul5b\\\\\\\/uuBrVOafxeFbthtsiDC9kAcE\\\\\\\/gUwywqQhj5dupPIBOMlA3oEstcWf4iIgOpcWsPXC35v5YT8XF3jNrE\\\\\\\/vRxcVKLVmmX7VhRcJlqN7LGt35PltvUEgMyH5dZnc7Xe0nBca3ACCr2BFKWhzoKEARoXDJGEk0tcFQ\\\\\\\/qs3K1JlyAVcU5Z+UyZtdAkovZ1X+msjKfnaVwciuCcPfgPvghP39xc7bcEmgl5YMeB0r0\\\\\\\/ZVP5qmnvM1mINo1aUHUj78MsFoyQLZtO58Jj2Pxhrlz2nCUNyzlNrLJbDLl3gUCEnqRgBiDyR0TcVMJLHh8sgi+94fjO7JvjOZ0YhqnTURNwE1tsFJAdGM4xc18FfuLnsIj7tA3GEL7bMK7g3yGSGUR2LE0w7xNBpBPFQNKTub4zKXj\\\\\\\/dwmd7TL5\\\\\\\/cqT7ZcIV1S1UuZo4RO6rEGx3KlvH+dDgAG5qfzKxWx\\\\\\\/P2jW9Amf9c8oSTOXbC9KCo0A16MT15yqMJGVbnVYCIiWTvR6SEKuMICEWfC2kHSSD2TTEIPdvPvqTe\\\\\\\/9OXX5m1GbmvED1cEddZst6PTy6VuCjAiSy9QDfNosCF0Jl6AlyENblA8GxmUojj7X71F1\\\\\\\/Z+46cDUItI1eig9SM6hkSHUcV4LBnUUgcaCJsK4NsQ3KF\\\\\\\/vCcwy90IOCPEkfbuFdqP1mHjgrvVeR22IE5XqQ9EoG6RIlLoM6aW40cNGBKsnHvfvBcpF7C3Kt9wCZ4JyDlOt912PXRtMJJ\\\\\\\/7uqo6FJAy4BzvLT39ISbQHyPyA6tKTG0bHtF2BFSUIN6pztHsqOczo\\\\\\\/lCeHsm+rLoyTpOVtydmudy+jEwwRrmXp8pxp4oqMDAM3aK13F+SpqLzEXyDzJ0Cbk+o\\\\\\\/4LdXV15W\\\\\\\/ozspcgj7762PxmQWSzmseBGClGKM3jejjQJdcOP60MSge22r2+ChEOYqbtEq8J9j6buQfowRWrT\\\\\\\/i2BI1xOJ4nKhz9umy2FYqC8VFRsMdpoUd9DYtGiYnQM\\\\\\\/XM9+b0m7RD3EmJppskd8opGoRAHrqwDdK\\\\\\\/e1gKyv6fNTZwszktXWDOeKaCRdZ6fArnuBrrk9dJYWAYESGzDpp\\\\\\\/6VNE6VM4+tGRLfBhnu0jKQkuLQnlTsGr2PU07B1L9oJ681WRdt6Zj58prAUSniY5wk7+2HX+XF\\\\\\\/mOtB1JAq\\\\\\\/Den4LBrs+FEZfkeeJBW1fLmk1\\\\\\\/oKjhz+LSFs+uIaXkULdxfWiWM1ctMFCxTXKFPwIjkYSZJpoVcRNLf3\\\\\\\/PX8VlXwKMCZnKzjmqn9QdS\\\\\\\/lsT\\\\\\\/eKJfQ8i6jUAp8UfpKpjK\\\\\\\/tSl4lzLe3KNSFiHkwNbksaxDF\\\\\\\/Q7cL5qn7hwOo7p\\\\\\\/84EiMYcw78w45xElHi6x+ejU5GL36bDdUS4WGbG+2gASuQDByVooUlL9DliHCWC08z6\\\\\\\/g4boeuaUVhXfPigLtpRz1DbjYW43Rp3jwdbpodOFbpsroqyHipK2PGdn1wjh19O6c2RdttxTP0YiuCKgkfX1WB+OAZC+z\\\\\\\/eOHLEfxEJw\\\\\\\/S9SEayyfvV8VxQJp0Dr4U5oUFVWCENqI3AT7Z2G7rqR7BvxwpTqNhOQC2Jgt\\\\\\\/a9NTEtig==\\\u0022,\\\u0022iv\\\u0022:\\\u0022+Bav5wFnMHQr4MNrOmbQfQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223ed9e8760339a26779450e1cc88dac793a04dec1be759fe6584718db6976cb59\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022w1QM7ATPzoh4EgAfjE2yrJpNolD6FuQuffm5IVbj7IGRvNTdAolX0nbgFEc0GiiHdy13uQgoI\\\\\\\/WmryDARZasfIPE0UyaETLjD47VLMkkR5syXAsXBj\\\\\\\/3fJ3ReV1E4VWyZai3tN11kaCbnZkbhMRKDjJ7nF1ccyICeOWVXC319vfKbHZpvX3LzShpdepeqfyFB3dlSC80Vz8nQTlorvqGMaAvPd+ZVz9Sd8IIKotON93KV6qQlRafBFg+eMHzXYBPpz+wMES3balkNtpJgbISAsk3JKnpo9eD1vp\\\\\\\/Kc4KvW8U21RHwRnACDXwdg6Wqd3iTsgEkwpciuXbAobMjjIeSWGnj60u0ypcxPcnB9QpyXbon9WkoLu9Cpekm27QEXMy3DslC3MXyXSNVFj9HIHQ0elItp9p3L6\\\\\\\/gnxgKtQPoQfZYrWwjH7wuQ0A1HqWXy6c09zaW83NyDmcJRWgO9MQJDuYtOiEt0SDx3AX6kmTJqT2a7F1BBSHb5OeMy\\\\\\\/dsdzSHZJavUk8w9kwGjLR9IbRTO6bLFph5u+2DdfYLi9UFSyJW2LPNWafTVtQiKPusGfcD\\\\\\\/8\\\\\\\/rn73pycYYHjBDfOuSVozLfmxslQ3F1NBdGMwldpO9bgBQw+5fyt7MCayWGpsjEfbn\\\\\\\/jx\\\\\\\/\\\\\\\/ykazSgtT\\\\\\\/Qj8YotPSOuZlaWgmbEijNMiFP7n1MvXNG3agjwfMxYaZ5j4osxx4AwDvSwe1ZOQHzL1D+RVPG5h+\\\\\\\/TuRe34urvNWh56f5x3nsZiCAZxpi2ke4Jr0mgAIuE\\\\\\\/GtZDFa\\\\\\\/ct9mgnWfxydxasVDBNW3LAwmKFHgaDGKK72+uAfJitRFcMJ2ad20ylpgYUW7Et\\\\\\\/\\\\\\\/MCp3VbYqkN9aaTZCWgyCd6Mnexuo8hq98mImQpiINW6n+vGTzHNXrH+Qzo33wpUkYxyiyDjV1pu1DAl++7dKELJtGj1Y2+guadOmoDgptG\\\\\\\/xMGgHnBBwiIjPe9Se6RUOu4aWBdPcjJNZdZAo0+06e4SYuxKA2dl+giHk8xCim2IRFtdPC9gabBCqBWBv0vWqKJ1amUUL0FGrQ0QqozSin8HMFs6kKBbnv0Ufi+wKccfbQJFc3f\\\\\\\/TdasxLeram9wcJuAcs3l6uHEXekLgHtsQp6QV\\\\\\\/qc1iAw2KDdPASRDYGlOHbXliYjzqI\\\\\\\/Vhr1VzlmDa5k8BJkAh3JQDKFWCXH3+1jZBTfNy1tynLYt8M62xAQyp6+3SxhDImfqydmrzTKNTo0z9XCN79gq3qoNIKO6\\\\\\\/6IbPUfod4XhMz6dl3jA\\\\\\\/HJjUpnwX4iz4T2lyHAjY8mAswUl8Ct+PM7dlJfPw3Ql5X\\\\\\\/Pv9\\\\\\\/e8190wKwcybmrI+jz3oSD5iEoJCeompSYzQAm2ISrUUqoW\\\\\\\/ErhzHm2XvNRGcquSXnIYZtu06yWrxlvFfKPinDj7whtFD2w0R45RZD0Lpct72BkYvrLRHlUbnLXcGUUelBzaCfo0bbPWBDOyxV7r23qmM\\\\\\\/mtlMjXGvzc2YYBIXyruTZakKhZ+mjU+W2Grg\\\\\\\/D9UzSMzP6bgS9bPVhIfZ4uxJaThCV37eKrbra9Gyo\\\\\\\/r4lpzXg4CC2Q7Q47Z7j9rXHR3BsrQor3F+Ra1X5YwGAZLv8Hlne9yAbrTI\\\\\\\/6HYBxKnnRXfCgP7RNfMYZDkx1HLBBQTokc0CTZ5+NDmzuBB3Qors1DF5Zl3VWg+H5VNP\\\\\\\/H7nUPwUlwNO+U2QoOsCkDaas4aG7cWR4Ufr1K0SCBmhqxx0kklSPNC14jpzxAcLGuy6KM4jRkA\\\\\\\/5HEsXPqs4oFs\\\\\\\/Yo+4sk3piuS\\\\\\\/mK1yEDHRGgwchco+qyg4T6SVaI2Kh+cdelaCUH8Qc3Dgk4I23KWUoM0s9b+mVXV\\\\\\\/LxQJWWJTlXcHza20ofLGqhsNw85kVQDjY7\\\\\\\/Vi3EpIWEoKh+xqgP78nzTwlTwglcoo6nN9YVftLYXtmTGiaW3WSyQi9WTC2V91Zbiccw2fMFOAXSevMjHUmGgpgyBb8X88JUvBzx5e0rojaHGixJn8TAPcGHcUQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022kJyFfZaPrU9kZHSsdFH6yQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022537f7a2e850d61c8c6bef7b85b3fac418220a85464a290820205d75a6b67aa2b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487835 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/239" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-5lKOL5azItF3179jEILJGTejGbx7VR-JcDPKgz7aX0U" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/239" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YCz1M0IlwA2l3bSUKcCeHWpBVNXvmw6NEo1oZ8b2pIk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/231" title="Egg Case Silk Gene Sequences from Argiope Spiders: Evidence for Multiple Loci and a Loss of Function Between Paralogs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/253" title="Rice TSV3 Encoding Obg-Like GTPase Protein Is Essential for Chloroplast Development During the Early Leaf Stage Under Cold Stress" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487835" id="node487835" data-pisa="ggg;8/1/239" data-pisa-master="ggg;g3.117.300392" data-apath="/ggg/8/1/239.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Natural Variation in <em>SER1</em> and <em>ENA6</em> Underlie Condition-Specific Growth Defects in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5186-9692" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amy Sirr</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4511-0898" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Adrian C. Scott</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5265-1970" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gareth A. Cromie</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-9682-7851" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Catherine L. Ludlow</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-8632-9629" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vida Ahyong</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-9299-8562" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Trey S. Morgan</span>, <span class="highwire-citation-author" data-delta="6">Teresa Gilbert</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aimée M. Dudley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">239-251;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300392 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487835" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amy Sirr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BSirr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sirr%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmy%2BSirr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5186-9692" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amy Sirr</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Adrian C. Scott </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adrian%2BC.%2BScott%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scott%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAdrian%2BC.%2BScott%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4511-0898" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Adrian C. Scott</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Gareth A. Cromie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gareth%2BA.%2BCromie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cromie%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGareth%2BA.%2BCromie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5265-1970" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gareth A. Cromie</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Catherine L. Ludlow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BL.%2BLudlow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ludlow%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACatherine%2BL.%2BLudlow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9682-7851" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Catherine L. Ludlow</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Vida Ahyong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Cellular Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vida%2BAhyong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ahyong%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVida%2BAhyong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8632-9629" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vida Ahyong</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Trey S. Morgan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Trey%2BS.%2BMorgan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morgan%20TS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATrey%2BS.%2BMorgan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9299-8562" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Trey S. Morgan</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Teresa Gilbert </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Teresa%2BGilbert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gilbert%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATeresa%2BGilbert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aimée M. Dudley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aim%C3%A9e%2BM.%2BDudley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dudley%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAim%25C3%25A9e%2BM.%2BDudley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aimée M. Dudley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/1/239" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487835" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487835/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/239.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487835" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487835/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/239.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487835" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487835/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/239.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487835" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487835/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/1/239.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487835" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487835/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Despite their ubiquitous use in laboratory strains, naturally occurring loss-of-function mutations in genes encoding core metabolic enzymes are relatively rare in wild isolates of <em>Saccharomyces cerevisiae</em>. Here, we identify a naturally occurring serine auxotrophy in a sake brewing strain from Japan. Through a cross with a honey wine (white tecc) brewing strain from Ethiopia, we map the minimal medium growth defect to <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em>, which encodes 3-phosphoserine aminotransferase and is orthologous to the human disease gene, <em>PSAT1</em>. To investigate the impact of this polymorphism under conditions of abundant external nutrients, we examine growth in rich medium alone or with additional stresses, including the drugs caffeine and rapamycin and relatively high concentrations of copper, salt, and ethanol. Consistent with studies that found widespread effects of different auxotrophies on RNA expression patterns in rich media, we find that the <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> loss-of-function allele dominates the quantitative trait locus (QTL) landscape under many of these conditions, with a notable exacerbation of the effect in the presence of rapamycin and caffeine. We also identify a major-effect QTL associated with growth on salt that maps to the gene encoding the sodium exporter, <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em>. We demonstrate that the salt phenotype is largely driven by variation in the <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> promoter, which harbors a deletion that removes binding sites for the <a href="http://www.yeastgenome.org/locus/S000003003/overview">Mig1</a> and <a href="http://www.yeastgenome.org/locus/S000002450/overview">Nrg1</a> transcriptional repressors. Thus, our results identify natural variation associated with both coding and regulatory regions of the genome that underlie strong growth phenotypes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li><li class="kwd"><a href="/keyword/natural-variation" class="hw-term hw-article-keyword hw-article-keyword-natural-variation" rel="nofollow">natural variation</a></li><li class="kwd"><a href="/keyword/qtl-mapping" class="hw-term hw-article-keyword hw-article-keyword-qtl-mapping" rel="nofollow">QTL mapping</a></li><li class="kwd"><a href="/keyword/serine-metabolism" class="hw-term hw-article-keyword hw-article-keyword-serine-metabolism" rel="nofollow">serine metabolism</a></li><li class="kwd"><a href="/keyword/salt-stress" class="hw-term hw-article-keyword hw-article-keyword-salt-stress" rel="nofollow">salt stress</a></li></ul><p id="p-3">The biological consequences of perturbing a metabolic pathway can be severe. In humans, polymorphisms in genes that encode core metabolic enzymes can cause rare, monogenic diseases known collectively as inborn errors of metabolism. These disorders generally result from loss-of-function mutations within the coding region of metabolic genes that lead to either a toxic buildup of precursor metabolites or deficiencies in the reaction end products (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lanpher <em>et al.</em> 2006</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">El-Hattab 2015</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Vernon 2015</a>). While often fatal if untreated, in many cases timely diagnosis followed by appropriate dietary restrictions or supplementation can greatly reduce or even eliminate symptoms in these patients (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Vernon 2015</a>). Serine deficiency disorders provide one such example. Disruption of any one of the three enzymes (3-phosphoglycerate dehydrogenase, phosphoserine aminotransferase, and phosphoserine phosphatase) in the phosphorylated serine biosynthesis pathway causes severe neurodevelopmental disorders that present with congenital microcephaly, intractable seizures, and psychomotor retardation (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Acuna-Hidalgo <em>et al.</em> 2014</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">El-Hattab <em>et al.</em> 2016</a>). However, case studies have shown that morbidity and mortality may be reduced and even eliminated with serine supplementation (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">de Koning <em>et al.</em> 2002</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">2004</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hart <em>et al.</em> 2007</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brassier <em>et al.</em> 2016</a>). In one such case, prenatal diagnosis of 3- phosphoglycerate dehydrogenase deficiency followed by L-serine supplementation to the mother early in pregnancy (week 27) and to the patient from birth onward completely prevented the onset of neurological symptoms (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">de Koning <em>et al.</em> 2004</a>).</p><p id="p-4">In the model organism <em>Saccharomyces cerevisiae</em>, gene deletions or loss-of-function mutations in amino acid or nucleobase biosynthetic enzymes are widely used as genetic markers and tools for genetic engineering. Accordingly, standard growth conditions include abundant supplies of the nutrients required to support the robust growth of both prototrophic and auxotrophic strains (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Rose <em>et al.</em> 1990</a>). Whether yeast cells synthesize a metabolite or import it from the surrounding environment depends on many factors, including the interplay between the nutrient sensors monitoring the metabolic status of the cell and signaling cascades that regulate metabolic pathways (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Conrad <em>et al.</em> 2014</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Palm and Thompson 2017</a>). When a biosynthetic pathway is disrupted and no secondary route exists, cells depend entirely on exogenous supplies of the missing metabolite. However, nutritional supplementation does not always fully compensate for impaired biosynthesis (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Pronk 2002</a>), and auxotrophies have been shown to have widespread effects on the transcriptional profile of a cell, even when grown in rich conditions (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Ronald and Akey 2007</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Alam <em>et al.</em> 2016</a>).</p><p id="p-5">Because naturally occurring auxotrophies in <em>S. cerevisiae</em> are thought to be rare, the influence of auxotrophies on phenotypic traits has largely been investigated in laboratory strains. However, a number of groups have begun to assemble and characterize collections of natural isolates, thereby sampling the natural variation within the species across a wide range of habitats (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Fay and Benavides 2005</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Schacherer <em>et al.</em> 2009</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Goddard <em>et al.</em> 2010</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cromie <em>et al.</em> 2013</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Strope <em>et al.</em> 2015</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Ludlow <em>et al.</em> 2016</a>). One powerful statistical method for studying the functional consequences of natural variation is quantitative trait locus (QTL) mapping (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lander and Botstein 1989</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kruglyak and Lander 1995</a>). This large-scale mapping approach looks for statistical associations between genotype and phenotype information from relatively large numbers of recombinant progeny, with the goal of identifying common regions of the genome associated with a trait.</p><p id="p-6">Here, we describe a cross between two yeast strains isolated from alcoholic beverages, but with genetically and geographically distinct lineages (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Fay and Benavides 2005</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Liti <em>et al.</em> 2006</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Cromie <em>et al.</em> 2013</a>): an auxotrophic strain isolated from Japanese sake and a prototrophic strain isolated from Ethiopian honey wine. Ethiopian honey wine (white tecc) is traditionally made in small batches and fermented by spontaneous cultures with batch-to-batch variation in microbial content (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bahiru <em>et al.</em> 2006</a>). In contrast, the industrialized process of sake brewing relies on inoculations with defined starter cultures of <em>S. cerevisiae</em> that have been selected for many traits, including ethanol tolerance, flavor production, and foaming characteristics (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akao <em>et al.</em> 2011</a>). While the <em>S. cerevisiae</em> strains used in the production of sake are generally prototrophic diploids, chemical and UV mutagenesis methods have been used to isolate auxotrophic haploids for strain construction purposes (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hashimoto <em>et al.</em> 2005</a>). Approximately 5% of the progeny of this cross exhibit a range of complex colony morphologies on standard growth conditions (YPD, 2% glucose), and two of the progeny strains from this cross have been used by our laboratory to investigate traits relevant to biofilm formation (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Tan <em>et al.</em> 2013</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cromie <em>et al.</em> 2017a</a>,<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">b</a>).</p><p id="p-7">In this study, we measure the growth of several hundred haploid, recombinant progeny of this cross and use genetic mapping to identify <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> as the causative gene underlying the minimal medium growth defect. We also find that <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em>, which encodes an enzyme in the phosphorylated serine biosynthesis pathway, dominates the QTL landscape even in the presence of abundant external nutrients. We identify a second major-effect QTL associated with growth on salt and map it to the sodium exporter, <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em>. We demonstrate that, despite significant variation within the coding region, the growth defect in the presence of salt is largely the result of polymorphisms in the <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> promoter.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Yeast growth and genetic manipulation</h3><p id="p-8">Unless noted, standard media and methods were used for the growth and manipulation of yeast (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Rose <em>et al.</em> 1990</a>). The <em>S. cerevisiae</em> strains used in this study are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The haploid parental strains of the yeast cross were derived from the UC5 (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Fay and Benavides 2005</a>) sake brewing strain from Japan (hereafter the “sake” genetic background) and the DBVPG1853 (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Liti <em>et al.</em> 2006</a>) white tecc brewing strain from Ethiopia (hereafter the “tecc” genetic background). Following mating of the sake (YO486) and tecc (YO502) parental strains and sporulation of the heterozygous diploid (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Tong <em>et al.</em> 2001</a>), a total of 552 recombinant progeny were isolated by manual tetrad dissection on YPD agar plates.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487949/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487949/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1 <em>S</em>.</span> <span class="caption-title"><em>cerevisiae</em> strains used in this study</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Yeast strain construction</h3><p id="p-9">Haploid strains for the sake and tecc cross were generated as follows. First, UC5 and DBVPG1853 were transformed with an <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">ho</a></em>Δ<em>0</em>::<em>hphMX6</em> cassette (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Sirr <em>et al.</em> 2015</a>) to produce a sake heterothallic MAT<strong>a</strong> haploid (YO607) and a tecc MAT<strong>a</strong>/α diploid that was hemizygous at the <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> locus (YO601). Next, YO601 was sporulated and a MATα heterothallic haploid resistant to hygromycin (YO560) was selected from hand-dissected tetrads. The final strains used in the cross (YO486 and YO502) were generated by transforming YO607 with <em><a href="http://www.yeastgenome.org/locus/S000002930/overview">SPS2</a>:EGFP:kanMX4</em> and YO560 with <em><a href="http://www.yeastgenome.org/locus/S000002930/overview">SPS2</a>:EGFP:natMX4</em> as previously described (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Sirr <em>et al.</em> 2015</a>). Since the original UC5 isolate (YO300) harbored a nonfunctional <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> gene (C. L. Ludlow and A. M. Dudley, unpublished results), it was subsequently used as the sake genetic background strain for some strain construction and growth assays.</p><p id="p-10"><em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> was deleted from the sake and tecc genetic backgrounds as follows. First, the drug marker of the <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">ser1</a></em>Δ<em>0</em>::<em>kanMX</em> allele in the yeast deletion collection (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Winzeler <em>et al.</em> 1999</a>) was switched from <em>kanMX</em> to <em>natMX</em> by transformation as previously described (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Goldstein and McCusker 1999</a>). The resulting <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">ser1</a></em>Δ<em>0</em>::<em>natMX</em> cassette was then PCR amplified using primers SER1_del_F and SER1_del_R (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS1.xlsx">Table S1</a>) and transformed into YO300 (sake) and YO560 (tecc).</p><p id="p-11">To generate <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> allele-replacement and control strains, integrating plasmids harboring alleles of <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS2.xlsx">Table S2</a>) were constructed using the New England BioLabs HiFi DNA Assembly Kit. Primers (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS3.xlsx">Table S3</a>) were designed using the NEBuilder Assembly Tool (<a href="http://nebuilder.neb.com/">http://nebuilder.neb.com/</a>). Plasmid pAS11-<a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-S contains the sake <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> allele (<em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-S</em>) which consists of the sake <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> open reading frame, 95 bp upstream of the start codon, 350 bp downstream of the stop codon, and the <em>kanMX4</em> drug marker followed by 240 bp of homology to a region downstream of <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> (to direct integration). The <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-S</em> allele [PCR amplified from YO300 using primers pUC19_SER1_F and SER1_R (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS3.xlsx">Table S3</a>)] and the <em>kanMX4</em> cassette gene fragments [either synthesized by GENEWIZ or PCR amplified using primers TEF_promoter_F and TEF_terminator_R (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS3.xlsx">Table S3</a>)] were cloned into a <em>Bam</em>HI/<em>Pst</em>I-digested pUC19 vector backbone. Plasmid pAS12-<a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-T was assembled in the same way, except using the tecc allele of <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> [<em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-T</em>, PCR amplified from YO560 using primers pUC19_SER1_F and SER1_R (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS3.xlsx">Table S3</a>)]. <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-S:kanMX4</em> was liberated from pAS11-<a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-S by <em>Sna</em>BI/<em>Sap</em>I double digestion and transformed into both the sake genetic background (to generate <em>kanMX</em>-tagged control strains) and into the tecc genetic background (to generate allele-replacement strains). <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-T:kanMX4</em> was liberated from pAS12-<a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-T by <em>Sna</em>BI/<em>Sap</em>I double digestion and transformed into both the tecc genetic background (to generate <em>kanMX</em>-tagged control strains) and into the sake genetic background (to generate allele-replacement strains) (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Transformants harboring the correct integration of the <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>:kanMX4</em> cassette were identified by PCR and Sanger sequencing (GENEWIZ).</p><p id="p-12">Sake and tecc <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>:natMX6</em> control strains (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) were constructed by transforming <em>natMX6</em> cassettes PCR amplified with primers ENA_300_tag_F or ENA_560_tag_F and ENA_tag_R (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS1.xlsx">Table S1</a>) into YO300 and YO560, respectively. The <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> deletion strains were constructed using <em>kanMX4</em> cassettes PCR amplified with primers ENA_del_F and ENA_del_R (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS1.xlsx">Table S1</a>) and transformed into YO300 and YO560.</p><p id="p-13">For the <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> allele swaps, the tecc <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> allele (<em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>-T</em>), including 234 bp upstream of the start codon, the complete open reading frame, and 349 bp downstream of the stop codon was PCR amplified from the <em>kanMX</em>-tagged control strain YO2432 using primers ENA_swap_F and ENA_swap_R (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS1.xlsx">Table S1</a>) and transformed into the sake <em><a href="https://www.yeastgenome.org/locus/S000130715">ena6</a></em>Δ strain YO2463. Integration of the tecc <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>:natMX6</em> cassette into the sake background was confirmed by PCR and Sanger sequencing. Different points of integration resulted in two allele variants in the sake genetic background. In isolate YO2524, the <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> coding region is identical to tecc <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em>, but the promoter region is identical to the sake genetic background (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS4.xlsx">Table S4</a>). We refer to this allele as <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>-pS-T</em>. The <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> coding regions of isolates YO2525 and YO2526 are also identical to the tecc <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em>, but these strains have tecc SNPs present in the first ∼234 bp of the promoter (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS4.xlsx">Table S4</a>), resulting in a sake/tecc hybrid promoter. We refer to this allele as <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>-pS1-T</em>.</p><p id="p-14">The reciprocal swap of the sake <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> allele into the tecc background was constructed in the same fashion. The sake <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> allele (<em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>-S</em>) was PCR amplified from the <em>kanMX</em>-tagged control strain YO2402 using primers ENA_swap_F and ENA_swap_R (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS1.xlsx">Table S1</a>) and transformed into the tecc <em><a href="https://www.yeastgenome.org/locus/S000130715">ena6</a></em>Δ strain YO2465. Proper integration of the sake <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>:natMX6</em> cassette into the tecc background was confirmed by PCR and Sanger sequencing. We refer to this allele as <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>-pT-S</em>.</p><p id="p-15">For allele replacements of the <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> promoter region, a longer segment containing the sake <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> upstream repressor sequence (URS) (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Serrano <em>et al.</em> 2002</a>) and sake/tecc hybrid URS region was integrated into the tecc genetic background as follows. Primers ENA_swap_F2 and ENA_swap_R (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS1.xlsx">Table S1</a>) were used to PCR amplify the entire <em>natMX6</em>-tagged <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>-pS-T</em> allele from YO2524. This allele, containing 610 bp of the sake promoter region and the tecc <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> open reading frame, was then transformed into the tecc <em><a href="https://www.yeastgenome.org/locus/S000130715">ena6</a></em>Δ strain YO2465. Additionally, the same primers were used to PCR amplify the <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>-pS1-T</em> allele from YO2525, which produced a sake/tecc hybrid promoter. This allele consists of the tecc promoter sequence for the first ∼234 bp upstream of the <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> start, followed by 376 bp of the sake promoter. These cassettes were integrated into the tecc <em><a href="https://www.yeastgenome.org/locus/S000130715">ena6</a></em>Δ strain YO2465. Tecc <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> strains with the extended sake promoter allele (<em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>-pS-T)</em> and tecc <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> strains with the extended sake/tecc hybrid promoter allele (<em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>-pS1-T)</em> were PCR and sequence confirmed (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS4.xlsx">Table S4</a>).</p></div><div id="sec-4" class="subsection"><h3>Yeast growth assays</h3><p id="p-16">Cross progeny strain growth was assayed on different media (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) as follows. Strains arrayed in a 96-well format were grown overnight at 30° in liquid YPD. For all conditions except SD, strains were replica pinned onto solid media in 250-mm square bioassay trays (Teknova, Hollister, CA) at a density of six well plates per bioassay tray using a QP Expression robot (Molecular Devices, Sunnyvale, CA). For growth on SD, strains were manually pinned onto 127 × 85 mm agar plates. Strains were grown at 30° and imaged once daily for 7 d.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487942/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487942/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Solid media used in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-18">Growth of strains harboring deletions or allele replacements of the <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> candidate genes was assayed by arraying YPD cultures grown overnight at 30° into a checkerboard pattern, such that the pinning density was 48 evenly spaced strains per 127 × 85 mm agar plate. Plates were grown at 30° for 3 d and photographed daily. Growth was measured as patch area extracted from the images using a custom script (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/FileS2.zip">File S2</a>) for ImageJ (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Schneider <em>et al.</em> 2012</a>). Briefly, for each image, the plate corners were manually identified and patch locations were interpolated from the plate layout. Then, the plate corners and patch edges were enhanced using a variance filter. The edges were converted to a binary mask and filled, and the area of each filled region [in square pixels (px<sup>2</sup>)] was used as the patch area. Patch areas were converted from px<sup>2</sup> to mm<sup>2</sup>, and the area in mm<sup>2</sup> for each day was used for QTL mapping.</p></div><div id="sec-5" class="subsection"><h3>Genotyping</h3><p id="p-19">Parental strains of the cross were whole genome sequenced using 2 × 36 bp (tecc) and 2 × 80 bp (sake) paired-end reads on an Illumina GA II. Genomic DNA was isolated using a YeaStar Genomic DNA Extraction Kit (Zymo Research). DNA sequencing libraries were prepared using the Paired-End Sequencing Kit (Illumina) following the manufacturer’s instructions. Reads were aligned to the S288c reference genome (R64-1-1) using BWA (v5.8) (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Li and Durbin 2009</a>), allowing six mismatches and using quality trimming (threshold of Phred = 20). SAMtools (v0.1.18) (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Li <em>et al.</em> 2009</a>) was then used to generate a pileup file for each parental strain, using the -C 50 and -q 20 parameters. At each position in the reference genome, the most common base was identified for each strain from the pileup files, with the criterion that the most commonly observed base must be at least five times more frequent than the second most frequent. Insertions and deletions were excluded. Positions that differed between YO486 and YO502 were then identified, allowing construction of a SNP table (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS5.txt">Table S5</a>) that defined potential genotyping markers.</p><p id="p-20">Progeny strains were genotyped using RAD-seq (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ludlow <em>et al.</em> 2013</a>) and the reads were aligned as above. SAMtools (v0.1.8) (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Li <em>et al.</em> 2009</a>) was then used to generate a pileup file for each progeny strain. To produce a strain-by-marker genotype table, at each position identified as polymorphic between the two parents in the parental SNP table, the counts of the two parental bases were recorded from each pileup file. This resulted in 5848 unique marker positions. Next, strains with poor coverage were dropped by removing any strains which had reads assigned to &lt;750 marker positions. At each marker position, the remaining strains were assigned the genotype of whichever parent was most frequently represented in the strain’s reads, so long as the most prevalent parent’s reads were at least five times greater than those corresponding to the other parent. Finally, markers were dropped if they had reads in &lt;5% of the strains, or if the ratio of sake calls to tecc calls deviated more than twofold from a 1:1 ratio. After this filtering, there were 457 strains and 537 markers.</p></div><div id="sec-6" class="subsection"><h3>Detecting aneuploidy</h3><p id="p-21">The relative chromosome copy numbers were calculated as the proportion of reads in each strain aligned to that chromosome, normalized by the median proportion for that chromosome across all strains (based on the assumption that most strains are euploid). Because half of the progeny strains were disomic for chromosome I, the proportion of reads aligned to chromosome I was normalized to the mean coverage over all strains divided by 1.5. Finally, each strain was normalized such that the median relative chromosome copy number in that strain was equal to one. Strains with a chromosome I coverage value between 1.5 and 2.5 were classed as chromosome I disomes, while strains with a value between 0.5 and 1.5 were classed as euploids/haploids. Strains with values outside the range of 0.5–2.5 were assumed to reflect sequencing errors. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS6.csv">Table S6</a> lists the estimated ploidy of each chromosome in each strain, and strains with a value of one for all chromosomes are considered euploid.</p></div><div id="sec-7" class="subsection"><h3>QTL mapping</h3><p id="p-22">QTL were identified using the R/qtl package (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Broman <em>et al.</em> 2003</a>). Since some of the conditions substantially restricted growth for a subset of the strains, resulting in nonnormal phenotype distributions, a nonparametric model was used. The 552 progeny strains from the cross were filtered by genotyping quality (as described above) and were additionally filtered by the drug resistance from the marker linked to the <em><a href="http://www.yeastgenome.org/locus/S000002930/overview">SPS2</a></em> locus. Specifically, 25 strains which grew larger than 5 mm<sup>2</sup> after 2 d on both YPD + Kan and YPD + Nat were removed. Also, four strains were removed because they exhibited no growth on YPD after 4 d. Finally, 16 strains were removed because their genotypes were extremely similar (&gt;90% identical) to another strain. As a result, a final set of 412 strains was used for QTL mapping. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS7.csv">Table S7</a> lists the phenotypes and genotypes for all 412 progeny strains as well as the two parents. Significance thresholds for QTL were determined by permutation tests (<em>n</em> = 10,000; α = 0.01) in R/qtl. Unless noted, all LOD scores were calculated based on phenotype data acquired on day 4 of growth at 30°.</p><p id="p-23">The non-normal phenotype distributions precluded a stepwise approach based on linear model fitting for uncovering secondary QTL. As an alternative, we repeated the QTL analysis after splitting the population based on the major-effect QTL genotype. For each phenotype, the marker with the highest LOD score was selected and subsets were created for strains with either the sake allele or the tecc allele at that marker. Strains with no allele call were ignored. A single-locus scan was run on each subset, and significance thresholds were calculated by permutation tests for each subset (<em>n</em> = 1000, α = 0.01).</p></div><div id="sec-8" class="subsection"><h3>Quantile normalization for interaction tests</h3><p id="p-24">Similarities in the QTL mapping results for YPD and three of the YPD-based conditions led us to test whether there was any additional effect of the perturbation beyond the growth defects in YPD. For each of these conditions, we performed two-factor ANOVA using patch area as the response. The two factors were the chr15_643999 allele and the presence or absence of the medium additive (copper, caffeine, or rapamycin). The phenotype distributions on caffeine and rapamycin were bimodal with a distinct floor comprised of strains with no growth. By splitting the population based on the chr15_643999 allele (as in the above secondary QTL analysis) we found that each subset approximated a normal distribution with a floor. We inferred values for strains near this floor by carrying out quantile normalization (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS8.csv">Table S8</a>) to a normal distribution with parameters estimated from linear regression against the portion of each subpopulation that was well separated from the floor (growth &gt;5 mm<sup>2</sup>).</p></div><div id="sec-9" class="subsection"><h3>Heritability and genetic variance calculations</h3><p id="p-25">We estimated the broad-sense heritability as <em>H</em><sup>2</sup> = 1−(σ<sup>2</sup><sub>e</sub>/σ<sup>2</sup><sub>s</sub>), where σ<sup>2</sup><sub>s</sub> is the variance of the population of segregant means, and σ<sup>2</sup><sub>e</sub> is the variance of the sampling distribution of these means. We calculated σ<sup>2</sup><sub>e</sub> using the random effects ANOVA approach of <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bloom <em>et al.</em> (2013)</a>. First, the pooled within-group (segregant) variance σ<sup>2</sup><sub>w</sub> was calculated using the lmer command in R (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bates <em>et al.</em> 2015</a>), fitting the random effects model to all segregant individual phenotype measurements where the measurement was &gt;5 mm<sup>2</sup> and there were at least two such measurements from the segregant. We then calculated σ<sup>2</sup><sub>e</sub> as (1/<em>n</em> × σ<sup>2</sup><sub>w</sub>) where <em>n</em> is the mean number of phenotype measurements per segregant.</p><p id="p-26">To estimate genetic variance accounted for by the effect of a genetic locus, we normalized the phenotypes of each of the two subpopulations determined by the alleles of that locus to have means of zero. We then calculated the variance of the normalized segregant means σ<sup>2</sup><sub>ns</sub> and the genetic variance accounted for as σ<sup>2</sup><sub>s</sub>−σ<sup>2</sup><sub>ns</sub>.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-27">All strains and plasmids are available upon request. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/FileS1.docx">File S1</a> contains a description of all supplemental files. Whole genome sequencing data for the sake (YO486) and tecc (YO502) parental strains has been deposited in the Sequence Read Archive under accession numbers PRJEB68822 and PRJEB22489, respectively. RAD-seq data for the progeny has been deposited in the Sequence Read Archive under accession number PRJEB22560. The script used to extract patch areas from images is included as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/FileS2.zip">File S2</a>. The R scripts to create the R/qtl cross object and perform analysis, including quantile normalization and heritability calculations, are included with necessary data files as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/FileS3.zip">File S3</a>.</p></div></div><div class="section results" id="sec-11"><h2 class="">Results</h2><div id="sec-12" class="subsection"><h3>Coding polymorphism in SER1 underlies the sake auxotrophy</h3><p id="p-28">Because naturally occurring auxotrophies are relatively rare in wild isolates of <em>S. cerevisiae</em>, we sought to identify the gene(s) underlying the failure of the Japanese UC5 sake strain to grow on minimal medium and the relationship of this trait to other phenotypes by meiotic mapping. We crossed modified versions (<em>Materials and Methods</em>) of UC5 (<a id="xref-ref-25-5" class="xref-bibr" href="#ref-25">Fay and Benavides 2005</a>) to a genetically distant, prototrophic, Ethiopian white tecc brewing strain DBVPG1853 (<a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Liti <em>et al.</em> 2006</a>) and isolated the haploid progeny of 138 three- or four-spore viable tetrads. To determine the gene(s) underlying the sake auxotrophy, we measured growth of the parental strains and 412 progeny on minimal medium (SD) and performed QTL analysis. Our results showed a QTL peak (LOD 45.4) on chromosome XV (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS9.xlsx">Table S9</a>) that was strongly associated with the minimal medium growth phenotype (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). In addition, the distribution of progeny growth on SD (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>) was consistent with the monogenic segregation of this trait (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hou <em>et al.</em> 2016</a>). Examining the functional annotation of the genes in this region suggested a strong candidate gene, <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em>, which encodes a highly conserved 3-phosphoserine aminotransferase required for the biosynthesis of serine (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Ulane and Ogur 1972</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Melcher and Entian 1992</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Melcher <em>et al.</em> 1995</a>). The sake parent of the cross harbors a nonsynonymous coding allele (G78R) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS2.xlsx">Table S2</a>) that is unique among the strains currently listed in the yeast genome database [SGD project; <a href="http://yeastgenome.org/cgi-bin/FUNGI/alignment.pl?locus=YOR184W">http://yeastgenome.org/cgi-bin/FUNGI/alignment.pl?locus=YOR184W</a>]. This polymorphism alters an amino acid in a highly conserved region of the protein near the cofactor (pyridoxal phosphate) binding pocket that is essential for enzyme function (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hester <em>et al.</em> 1999</a>) and is the only polymorphism within the <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> coding region that differs from the amino acid sequence of the reference strain or the tecc strain background.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/239/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="SER1 is the quantitative trait gene associated with the UC5 auxotrophy. (A) QTL mapping of 412 recombinant progeny grown on minimal medium (SD) identified a single highly significant peak (LOD = 45.4) at the marker at 643,999 bp on chromosome XV. The horizontal line is the significance threshold determined by permutation tests (n = 10,000; &#x3B1; = 0.01) in R/qtl. The LOD scores of other statistically significant markers are presented in Table S9. (B) Growth of the progeny strains on SD as measured by patch area and split into subsets based on genotype at 643,999 bp on chromosome XV (tecc allele in red; sake allele in blue). Horizontal lines denote the median value for each group of strains. The growth value of the corresponding parent strain is indicated by a red or blue box. (C) Histogram of progeny strain growth on minimal media, which shows a bimodal pattern. The red line at 9 mm2 indicates the approximate minimum between the two growth classes, with 201 strains below that threshold and 211 above. (D) Growth of strains on SD with and without the addition of 1 mM serine, including the parent strains of the cross (tecc and sake), both strains with the SER1 open reading frame deleted (ser1&#x394;), and two independent isolates of each strain harboring allele replacements of SER1. Strains were photographed after 3 d of growth at 30&#xB0;. Each strain was assayed at least four times across two replicate plates for each condition and a representative example is shown. Additional replicates are shown in Figure S1. Plate images were cropped without resizing or additional image manipulation to create the composite image." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-437829365" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;SER1&lt;/em&gt; is the quantitative trait gene associated with the UC5 auxotrophy. (A) QTL mapping of 412 recombinant progeny grown on minimal medium (SD) identified a single highly significant peak (LOD = 45.4) at the marker at 643,999 bp on chromosome XV. The horizontal line is the significance threshold determined by permutation tests (&lt;em&gt;n&lt;/em&gt; = 10,000; &#x3B1; = 0.01) in R/qtl. The LOD scores of other statistically significant markers are presented in Table S9. (B) Growth of the progeny strains on SD as measured by patch area and split into subsets based on genotype at 643,999 bp on chromosome XV (tecc allele in red; sake allele in blue). Horizontal lines denote the median value for each group of strains. The growth value of the corresponding parent strain is indicated by a red or blue box. (C) Histogram of progeny strain growth on minimal media, which shows a bimodal pattern. The red line at 9 mm&lt;sup&gt;2&lt;/sup&gt; indicates the approximate minimum between the two growth classes, with 201 strains below that threshold and 211 above. (D) Growth of strains on SD with and without the addition of 1 mM serine, including the parent strains of the cross (tecc and sake), both strains with the &lt;em&gt;SER1&lt;/em&gt; open reading frame deleted (&lt;em&gt;ser1&lt;/em&gt;&#x394;), and two independent isolates of each strain harboring allele replacements of &lt;em&gt;SER1&lt;/em&gt;. Strains were photographed after 3 d of growth at 30&#xB0;. Each strain was assayed at least four times across two replicate plates for each condition and a representative example is shown. Additional replicates are shown in Figure S1. Plate images were cropped without resizing or additional image manipulation to create the composite image.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/239/F1.medium.gif" width="440" height="402"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/1/239/F1.medium.gif" width="440" height="402"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/239/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/239/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487999" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-29" class="first-child"><em>SER1</em> is the quantitative trait gene associated with the UC5 auxotrophy. (A) QTL mapping of 412 recombinant progeny grown on minimal medium (SD) identified a single highly significant peak (LOD = 45.4) at the marker at 643,999 bp on chromosome XV. The horizontal line is the significance threshold determined by permutation tests (<em>n</em> = 10,000; α = 0.01) in R/qtl. The LOD scores of other statistically significant markers are presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS9.xlsx">Table S9</a>. (B) Growth of the progeny strains on SD as measured by patch area and split into subsets based on genotype at 643,999 bp on chromosome XV (tecc allele in red; sake allele in blue). Horizontal lines denote the median value for each group of strains. The growth value of the corresponding parent strain is indicated by a red or blue box. (C) Histogram of progeny strain growth on minimal media, which shows a bimodal pattern. The red line at 9 mm<sup>2</sup> indicates the approximate minimum between the two growth classes, with 201 strains below that threshold and 211 above. (D) Growth of strains on SD with and without the addition of 1 mM serine, including the parent strains of the cross (tecc and sake), both strains with the <em>SER1</em> open reading frame deleted (<em>ser1</em>Δ), and two independent isolates of each strain harboring allele replacements of <em>SER1</em>. Strains were photographed after 3 d of growth at 30°. Each strain was assayed at least four times across two replicate plates for each condition and a representative example is shown. Additional replicates are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/FigureS1.tif">Figure S1</a>. Plate images were cropped without resizing or additional image manipulation to create the composite image.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Consistent with the hypothesis that the sake background harbored a defect in serine biosynthesis, supplementation of the minimal medium with 1 mM serine restored growth of the sake parental strain (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1D</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/FigureS1.tif">Figure S1</a>). To test whether <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> was in fact the causative gene underlying the sake strain auxotrophy, we constructed a series of allele-replacement strains and assayed their growth on minimal medium in the presence and absence of exogenous serine. The results (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1D</a>) show that, in either genetic background, the tecc allele of <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> (<em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-T</em>) is required for growth on minimal medium and the sake allele (<em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-S</em>) confers a serine auxotrophy. Furthermore, the comparable growth defects of <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-S</em> and <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">ser1</a></em>Δ in the absence of serine (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1D</a>) are consistent with the hypothesis that the G78R amino acid substitution confers a loss-of-function phenotype.</p></div><div id="sec-13" class="subsection"><h3>Major-effect QTL common to several conditions</h3><p id="p-31">To explore the influence of this naturally occurring polymorphism on other traits, we examined the effects of additional stresses, including the addition of the drugs caffeine and rapamycin and relatively high concentrations of copper, salt, and ethanol to yeast-rich medium (YPD, 2% glucose). In four of the six conditions (the rich medium control, copper, caffeine, and rapamycin), a single-locus scan revealed a highly significant QTL peak in the same region of chromosome XV that included <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The two remaining conditions, ethanol and salt, had modestly significant QTL peaks at this locus, but shared a stronger peak on chromosome IV (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS9.xlsx">Table S9</a>). The ethanol condition also showed another significant peak on chromosome IX (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS9.xlsx">Table S9</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/239/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QTL mapping reveals a major QTL peak common to several conditions. QTL mapping of 412 recombinant progeny (left) and the distribution of the growth of the progeny strains split into subsets based on genotype (right) at the marker of the strongest QTL peak (*) for that condition on (A) YPD (LOD = 36.2; marker 643,999 bp on chromosome XV), (B) YPD + copper (LOD = 36.0; marker 643,999 bp on chromosome XV), (C) YPD + caffeine (LOD = 41.0; marker 643,999 bp on chromosome XV), (D) YPD + rapamycin (LOD = 40.2; marker 643,999 bp on chromosome XV), (E) YPD + ethanol (LOD = 10.2; marker 409,532 bp on chromosome IX), and (F) YPD + sodium (LOD = 54.8; marker 527,628 bp on chromosome IV). The horizontal line in QTL plots represents the significance threshold determined by permutation tests (n = 10,000; &#x3B1; = 0.01) in R/qtl. The LOD scores of other statistically significant markers are presented in Table S9. Progeny strain growth on each condition was measured by patch area and split into subsets based on genotype (tecc allele in red; sake allele in blue) at the marker with the highest LOD score. Horizontal lines denote the median value for each group of strains. The growth value of the corresponding parent strain is indicated by a red or blue box." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-437829365" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;QTL mapping reveals a major QTL peak common to several conditions. QTL mapping of 412 recombinant progeny (left) and the distribution of the growth of the progeny strains split into subsets based on genotype (right) at the marker of the strongest QTL peak (*) for that condition on (A) YPD (LOD = 36.2; marker 643,999 bp on chromosome XV), (B) YPD + copper (LOD = 36.0; marker 643,999 bp on chromosome XV), (C) YPD + caffeine (LOD = 41.0; marker 643,999 bp on chromosome XV), (D) YPD + rapamycin (LOD = 40.2; marker 643,999 bp on chromosome XV), (E) YPD + ethanol (LOD = 10.2; marker 409,532 bp on chromosome IX), and (F) YPD + sodium (LOD = 54.8; marker 527,628 bp on chromosome IV). The horizontal line in QTL plots represents the significance threshold determined by permutation tests (&lt;em&gt;n&lt;/em&gt; = 10,000; &#x3B1; = 0.01) in R/qtl. The LOD scores of other statistically significant markers are presented in Table S9. Progeny strain growth on each condition was measured by patch area and split into subsets based on genotype (tecc allele in red; sake allele in blue) at the marker with the highest LOD score. Horizontal lines denote the median value for each group of strains. The growth value of the corresponding parent strain is indicated by a red or blue box.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/239/F2.medium.gif" width="330" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/1/239/F2.medium.gif" width="330" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/239/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/239/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487995" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-32" class="first-child">QTL mapping reveals a major QTL peak common to several conditions. QTL mapping of 412 recombinant progeny (left) and the distribution of the growth of the progeny strains split into subsets based on genotype (right) at the marker of the strongest QTL peak (*) for that condition on (A) YPD (LOD = 36.2; marker 643,999 bp on chromosome XV), (B) YPD + copper (LOD = 36.0; marker 643,999 bp on chromosome XV), (C) YPD + caffeine (LOD = 41.0; marker 643,999 bp on chromosome XV), (D) YPD + rapamycin (LOD = 40.2; marker 643,999 bp on chromosome XV), (E) YPD + ethanol (LOD = 10.2; marker 409,532 bp on chromosome IX), and (F) YPD + sodium (LOD = 54.8; marker 527,628 bp on chromosome IV). The horizontal line in QTL plots represents the significance threshold determined by permutation tests (<em>n</em> = 10,000; α = 0.01) in R/qtl. The LOD scores of other statistically significant markers are presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS9.xlsx">Table S9</a>. Progeny strain growth on each condition was measured by patch area and split into subsets based on genotype (tecc allele in red; sake allele in blue) at the marker with the highest LOD score. Horizontal lines denote the median value for each group of strains. The growth value of the corresponding parent strain is indicated by a red or blue box.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Three conditions that consist of YPD plus an additive (copper, caffeine, or rapamycin) had LOD profiles that were nearly indistinguishable from YPD, raising the possibility that the phenotypic differences between the progeny merely reflected the growth differences in YPD. We examined this possibility in two ways: first by comparing the magnitude and direction of effect of the sake and tecc alleles in YPD <em>vs.</em> the other conditions, and second by statistically testing whether there was an interaction between the peak XV QTL marker and each condition. We first defined two subpopulations based on the allele at the peak XV QTL marker. Because of a clear floor effect in the progeny strains on rapamycin and caffeine, particularly in the sake subpopulation, we first performed quantile normalization on the two subpopulations (<em>Materials and Methods</em>).</p><p id="p-34">After normalizing the rapamycin and caffeine phenotype values, we estimated the levels of broad-sense heritability and found them to be very high in all four of these YPD-based traits (YPD = 0.95, copper = 0.97, rapamycin = 0.99, and caffeine = 0.99). The proportion of the genetic variance explained by the <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> locus varied considerably across the four conditions, but was substantial in each (YPD = 0.23, copper = 0.26, rapamycin = 0.51, and caffeine = 0.48). Therefore, in all four of these conditions, growth is a highly heritable trait with the phenotype of any strain substantially determined by the allele of <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> that it carries.</p><p id="p-35">The copper condition is a clear example of a case in which the added stress had relatively little impact on the growth of the progeny beyond that of YPD alone. The growth of the progeny on YPD <em>vs.</em> copper (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/FigureS2.tif">Figure S2</a>) shows a strong correlation (Pearson’s correlation coefficient = 0.945). Furthermore, the difference in mean growth between the subpopulations on YPD was 9 mm<sup>2</sup>, which is very similar to the 10.5 mm<sup>2</sup> on copper (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/FigureS3.tif">Figure S3</a>). In contrast, the difference in mean growth between the subpopulations was 18.7 mm<sup>2</sup> on rapamycin and 12.4 mm<sup>2</sup> on caffeine. To test for an interaction between <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> and the growth medium in each of these three conditions, we carried out a two-factor ANOVA with an interaction term. Significant interactions were identified for rapamycin (<em>P</em> = 5.3 × 10<sup>−19</sup>) and caffeine (<em>P</em> = 3.7 × 10<sup>−5</sup>), but not for copper (<em>P</em> = 0.069). Taken together, our results are consistent with the hypothesis that factors regulated by the target of rapamycin (TOR) pathway contribute to the relatively robust growth of the auxotrophic sake strain in the presence of abundant nutrients.</p><p id="p-36">Although <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> accounted for a substantial proportion of the genetic variance in all four of the YPD-based growth conditions, the fact that substantial residual genetic variance remained after accounting for <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> (YPD = 36.0, copper = 47.2, rapamycin = 81.9, and caffeine = 38.4; variance units are phenotype units squared, <em>i.e.</em>, mm<sup>4</sup>) suggested the existence of additional QTL. Because the large-effect size of <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> could mask detection of weaker secondary QTL, we split the populations based on the major-effect QTL and performed QTL analysis on each of the subpopulations (<em>Materials and Methods</em>). Similarly, for the ethanol and sodium conditions, the population was split on the major chromosome IX and IV QTL, respectively. This analysis identified several minor-effect QTL in the split populations that, with the exception of the rapamycin condition, were in the same locations as primary peaks identified in other conditions (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). For example, the secondary QTL peak linked to growth on YPD was located at the same position on chromosome IV as the major-effect peak observed on salt. Conversely, on salt, the secondary QTL peak linked to growth was located in the same region of chromosome XV as the major-effect peak observed in YPD. Thus, the only additional QTL uncovered by this stepwise analysis was a peak on chromosome X centered over <em><a href="http://www.yeastgenome.org/locus/S000003827/overview">TOR1</a></em>, a core component of the TOR complex 1 (TORC1) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Heitman <em>et al.</em> 1991</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Loewith <em>et al.</em> 2002</a>), identified in the rapamycin condition (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). While we did not characterize the gene(s) underlying this QTL further, we note that the sake allele of <em><a href="http://www.yeastgenome.org/locus/S000003827/overview">TOR1</a></em> harbors several nonsynonymous amino acid substitutions at highly conserved residues in the HEAT repeats or FAT regions of the protein (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS10.xlsx">Table S10</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/239/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QTL mapping after fixing the major effect locus reveals minor-effect QTL. To identify loci that might be obscured by the major-effect locus in each condition, the cross population was split based on the genotype at the peak marker (denoted by * in Figure 2) and mapping was applied to each subset separately (blue = sake; red = tecc). Significance thresholds (red and blue horizontal lines) at a nominal &#x3B1; = 0.01 with Bonferroni correction (n = 6 conditions &#xD7; 2 subsets = 12) were calculated separately for each subset by permutation (n = 1000)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-437829365" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;QTL mapping after fixing the major effect locus reveals minor-effect QTL. To identify loci that might be obscured by the major-effect locus in each condition, the cross population was split based on the genotype at the peak marker (denoted by * in Figure 2) and mapping was applied to each subset separately (blue = sake; red = tecc). Significance thresholds (red and blue horizontal lines) at a nominal &#x3B1; = 0.01 with Bonferroni correction (&lt;em&gt;n&lt;/em&gt; = 6 conditions &#xD7; 2 subsets = 12) were calculated separately for each subset by permutation (&lt;em&gt;n&lt;/em&gt; = 1000).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/239/F3.medium.gif" width="440" height="391"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/1/239/F3.medium.gif" width="440" height="391"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/239/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/239/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487898" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-37" class="first-child">QTL mapping after fixing the major effect locus reveals minor-effect QTL. To identify loci that might be obscured by the major-effect locus in each condition, the cross population was split based on the genotype at the peak marker (denoted by * in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>) and mapping was applied to each subset separately (blue = sake; red = tecc). Significance thresholds (red and blue horizontal lines) at a nominal α = 0.01 with Bonferroni correction (<em>n</em> = 6 conditions × 2 subsets = 12) were calculated separately for each subset by permutation (<em>n</em> = 1000).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>SER1 is the causative gene underlying growth defects in several rich media conditions</h3><p id="p-38">To test whether <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> was also the causative gene underlying the QTL peaks in rich media conditions, we assayed the growth of the <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">ser1</a></em>Δ and allele-replacement strains in YPD, caffeine, and rapamycin (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Our results demonstrated that the robust growth of the tecc genetic background was dependent on the presence of a functional <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> allele, with reduced or even no growth observed in a <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">ser1</a></em>Δ or a <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-S</em> allele replacement on YPD alone (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>) or in the presence of rapamycin or caffeine (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). These results support the hypothesis that <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-S</em> is a loss-of-function allele and that the tecc genetic background provides a reliable readout of <a href="http://www.yeastgenome.org/locus/S000005710/overview">Ser1</a> activity. Results in the sake background were more complicated. In the presence of rapamycin, a comparable amount of growth was seen in the sake parental strain and a <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">ser1</a></em>Δ, but robust growth was conferred by the <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-T</em> allele replacement (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>). Thus, <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em> is the causative gene underlying the growth defect in rapamycin. However, in both YPD alone and in the presence of caffeine, the sake parental strain exhibits stronger growth than would be predicted from the growth of progeny strains harboring the sake allele at that locus (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). One possible explanation for this transgressive segregation pattern is the presence of compensatory adaptation(s) to <a href="http://www.yeastgenome.org/locus/S000005710/overview">Ser1</a> deficiency in the sake strain. However, the absence of modifier QTL peaks in the YPD or caffeine conditions (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>) suggests that this trait may be highly complex, unlinked to the Mendelian segregation patterns of the nuclear chromosomes, or involve one or more nongenetic factors.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/239/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Allele swaps of SER1. (A) A comparison of patch area on YPD after 3 d of growth. Error bars are SE. (B) Representative example images of growth of these same strains on YPD + 0.1 &#xB5;g/ml rapamycin, YPD +10 mM caffeine, and YPD. Strains include the parent strains of the cross (tecc and sake), both strains with the SER1 open reading frame deleted (ser1&#x394;), and two independent isolates of each strain harboring allele replacements of SER1. Strains were photographed after 3 d of growth at 30&#xB0;. Plate images were cropped without resizing or additional image manipulation to create the composite image presented. In both (A) and (B), each strain was assayed at least four times across two replicate plates for each condition." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-437829365" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Allele swaps of &lt;em&gt;SER1&lt;/em&gt;. (A) A comparison of patch area on YPD after 3 d of growth. Error bars are SE. (B) Representative example images of growth of these same strains on YPD + 0.1 &#xB5;g/ml rapamycin, YPD +10 mM caffeine, and YPD. Strains include the parent strains of the cross (tecc and sake), both strains with the &lt;em&gt;SER1&lt;/em&gt; open reading frame deleted (&lt;em&gt;ser1&lt;/em&gt;&#x394;), and two independent isolates of each strain harboring allele replacements of &lt;em&gt;SER1&lt;/em&gt;. Strains were photographed after 3 d of growth at 30&#xB0;. Plate images were cropped without resizing or additional image manipulation to create the composite image presented. In both (A) and (B), each strain was assayed at least four times across two replicate plates for each condition.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/239/F4.medium.gif" width="415" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/1/239/F4.medium.gif" width="415" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/239/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/239/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488055" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-39" class="first-child">Allele swaps of <em>SER1</em>. (A) A comparison of patch area on YPD after 3 d of growth. Error bars are SE. (B) Representative example images of growth of these same strains on YPD + 0.1 µg/ml rapamycin, YPD +10 mM caffeine, and YPD. Strains include the parent strains of the cross (tecc and sake), both strains with the <em>SER1</em> open reading frame deleted (<em>ser1</em>Δ), and two independent isolates of each strain harboring allele replacements of <em>SER1</em>. Strains were photographed after 3 d of growth at 30°. Plate images were cropped without resizing or additional image manipulation to create the composite image presented. In both (A) and (B), each strain was assayed at least four times across two replicate plates for each condition.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Natural variation in the ENA6 promoter improves growth in salt</h3><p id="p-40">Robust growth on salt was associated with the inheritance of a genomic region on chromosome IV from the sake parent (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>). Within the support interval of this peak, we identified a promising candidate, the <em>ENA</em> locus, which has been previously linked to sodium and lithium growth defects (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kim and Fay 2007</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Warringer <em>et al.</em> 2011</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Wilkening <em>et al.</em> 2014</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Hou <em>et al.</em> 2016</a>). Different strain backgrounds harbor one or more genes (<em><a href="http://www.yeastgenome.org/locus/S000002447/overview">ENA1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002446/overview">ENA2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002445/overview">ENA5</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em>) at this locus that encode a yeast P-type ATPase sodium and lithium exporter(s) (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Rudolph <em>et al.</em> 1989</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Haro <em>et al.</em> 1991</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Wieland <em>et al.</em> 1995</a>). The parental strains of our cross each bear a single <em>ENA</em> gene that is homologous to the ancestral <em>S. cerevisiae <a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Daran-Lapujade <em>et al.</em> 2009</a>; <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Strope <em>et al.</em> 2015</a>), although the tecc allele of <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> shares homology with the hybrid <em><a href="http://www.yeastgenome.org/locus/S000002447/overview">ENA1</a></em> allele over the last ∼465 bp of the gene (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS11.xlsx">Table S11</a>). Hereafter, we refer to the sake and tecc <em>ENA</em> alleles as <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>-S</em> and <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>-T</em>, respectively.</p><p id="p-41">To test whether <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> was in fact the causative gene underlying the QTL associated with growth in the salt stress condition, we constructed a series of deletion and allele-replacement strains and assayed their growth in the presence of high salt. Consistent with previous studies (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Haro <em>et al.</em> 1991</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Daran-Lapujade <em>et al.</em> 2009</a>; <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Warringer <em>et al.</em> 2011</a>), deletion of the <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> open reading frame from the tecc genetic background significantly reduced growth on salt (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6B</a>). Surprisingly, despite the presence of numerous nonsynonymous coding polymorphisms between the two alleles (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS11.xlsx">Table S11</a>), replacement of the <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>-T</em> open reading frame with the sake coding sequence (<em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>-pT-S</em>) did not restore growth on salt to the levels seen in the sake strain (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>).</p><p id="p-42">Given the previously observed relationship between salt tolerance and <em>ENA</em> expression levels (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Daran-Lapujade <em>et al.</em> 2009</a>; <a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Warringer <em>et al.</em> 2011</a>), we examined the sequence of the <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> promoter regions. <em><a href="http://www.yeastgenome.org/locus/S000002447/overview">ENA1</a></em> is constitutively repressed by numerous pathways including the HOG, glucose, calcineurin, and <a href="http://www.yeastgenome.org/locus/S000001019/overview">Rim101</a> pathways (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Ruiz and Arino 2007</a>). The zinc finger binding proteins <a href="http://www.yeastgenome.org/locus/S000003003/overview">Mig1</a> and <a href="http://www.yeastgenome.org/locus/S000003177/overview">Mig2</a> play a primary role in the glucose repression pathway (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Lutfiyya <em>et al.</em> 1998</a>), and deletion of <em><a href="http://www.yeastgenome.org/locus/S000003003/overview">MIG1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000003177/overview">MIG2</a></em> have additive effects on <em><a href="http://www.yeastgenome.org/locus/S000002447/overview">ENA1</a></em> expression levels (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Proft and Serrano 1999</a>). <em><a href="http://www.yeastgenome.org/locus/S000002447/overview">ENA1</a></em> is also repressed by <a href="http://www.yeastgenome.org/locus/S000002450/overview">Nrg1</a>, a protein regulated by both the glucose repression pathway (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Vyas <em>et al.</em> 2001</a>) and the <a href="http://www.yeastgenome.org/locus/S000001019/overview">Rim101</a> pathway in response to high salt and high pH (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Platara <em>et al.</em> 2006</a>). Similar to the effect of deleting <em><a href="http://www.yeastgenome.org/locus/S000003003/overview">MIG1</a></em>, deletion of <em><a href="http://www.yeastgenome.org/locus/S000002450/overview">NRG1</a></em> is associated with the increased transcription of <em><a href="http://www.yeastgenome.org/locus/S000002447/overview">ENA1</a></em> even in nonstress conditions (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Lamb and Mitchell 2003</a>). Interestingly, <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>-S</em> contains a 33-bp deletion (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>) in a previously defined URS of the alkaline responsive region (ARR2) (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Serrano <em>et al.</em> 2002</a>) that deletes the <a href="http://www.yeastgenome.org/locus/S000003003/overview">Mig1</a> and <a href="http://www.yeastgenome.org/locus/S000002450/overview">Nrg1</a> binding sites (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Proft and Serrano 1999</a>; <a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Serrano <em>et al.</em> 2002</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Platara <em>et al.</em> 2006</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Ruiz and Arino 2007</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/239/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The ARR2 sequence within the ENA6 promoter. Boxes indicate binding sites for repressor proteins Nrg1, Mig1, Mig2, and Sko1. Under standard growth conditions, ENA6 is constitutively repressed at multiple locations within this region; however, a 33-bp deletion in the sake background (underlined in red) disrupts Nrg1 and Mig1, Mig2 binding sites. Figure modified from Serrano et al. (2002) and Ruiz and Arino (2007)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-437829365" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The ARR2 sequence within the &lt;em&gt;ENA6&lt;/em&gt; promoter. Boxes indicate binding sites for repressor proteins Nrg1, Mig1, Mig2, and Sko1. Under standard growth conditions, &lt;em&gt;ENA6&lt;/em&gt; is constitutively repressed at multiple locations within this region; however, a 33-bp deletion in the sake background (underlined in red) disrupts Nrg1 and Mig1, Mig2 binding sites. Figure modified from Serrano &lt;em&gt;et al.&lt;/em&gt; (2002) and Ruiz and Arino (2007).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/239/F5.medium.gif" width="440" height="229"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/1/239/F5.medium.gif" width="440" height="229"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/239/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/239/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487882" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-43" class="first-child">The ARR2 sequence within the <em>ENA6</em> promoter. Boxes indicate binding sites for repressor proteins Nrg1, Mig1, Mig2, and Sko1. Under standard growth conditions, <em>ENA6</em> is constitutively repressed at multiple locations within this region; however, a 33-bp deletion in the sake background (underlined in red) disrupts Nrg1 and Mig1, Mig2 binding sites. Figure modified from <a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">Serrano <em>et al.</em> (2002)</a> and <a id="xref-ref-65-3" class="xref-bibr" href="#ref-65">Ruiz and Arino (2007)</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">To test the effect of the promoter polymorphisms, we constructed a series of strains harboring the tecc <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> open reading frame under the transcriptional control of the sake <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> promoter and assayed their growth on high salt. Interestingly, alleles that replaced the tecc <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> promoter with the sake promoter sequence (<em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>-pS-T</em> and <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a>-pS1-T</em>) rescued growth on high concentrations of salt (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/FigureS4.tif">Figure S4</a>), demonstrating that the <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> locus is the causative QTL involved in salt sensitivity in the tecc background and that this phenotype is strongly influenced by variation in the <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> promoter region. Because extended homology between the allele swap cassette and the genomic sequence permitted recombination at different points across the promoter and produced some sake/tecc promoter hybrids (<em>Materials and Methods</em>), we were able to compare the effects of alleles that replaced only a portion of the promoter against the effects of replacing the entire 610-bp region upstream of the start codon (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS4.xlsx">Table S4</a>). Strains harboring sake <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> SNPs from −610 to −234 bp exhibited only slight differences in growth on high salt relative to strains with the entire 610-bp sake <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> promoter region. These results further narrowed the causal regulatory region of <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> to the large 33-bp deletion that removed the <a href="http://www.yeastgenome.org/locus/S000003003/overview">Mig1</a> and <a href="http://www.yeastgenome.org/locus/S000002450/overview">Nrg1</a> binding sites and seven other SNPs, including a 4-bp difference in the TA repeat length of the TATA box.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/239/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Polymorphisms in the ENA6 promoter confer differential sensitivity to high salt. (A) A depiction of the promoter region of the sake (blue) and tecc (gray) ENA6 alleles tested in this study compared to the reference strain (purple). The sake genetic background has a 33-bp deletion in the ARR2 URS which deletes binding sites for the Mig1, Mig2, and Nrg1 repressors. Representative images of additional tecc and sake background strains are shown in Figure S4. (B) The parental strains, a tecc ena6&#x394;, and three different allele substitution strains in the tecc background were grown on YPD (&#x25B4;) and YPD + 0.5 M salt (&#x25CF;). The median of each group is indicated with a line. Each condition included 12 technical replicates (four replicates per plate and three plates per condition). In addition, all modified strains (black) had two biological replicates except tecc ENA6-pS-T, which had one biological replicate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-437829365" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Polymorphisms in the &lt;em&gt;ENA6&lt;/em&gt; promoter confer differential sensitivity to high salt. (A) A depiction of the promoter region of the sake (blue) and tecc (gray) &lt;em&gt;ENA6&lt;/em&gt; alleles tested in this study compared to the reference strain (purple). The sake genetic background has a 33-bp deletion in the ARR2 URS which deletes binding sites for the Mig1, Mig2, and Nrg1 repressors. Representative images of additional tecc and sake background strains are shown in Figure S4. (B) The parental strains, a tecc &lt;em&gt;ena6&lt;/em&gt;&#x394;, and three different allele substitution strains in the tecc background were grown on YPD (&#x25B4;) and YPD + 0.5 M salt (&#x25CF;). The median of each group is indicated with a line. Each condition included 12 technical replicates (four replicates per plate and three plates per condition). In addition, all modified strains (black) had two biological replicates except tecc ENA6-pS-T, which had one biological replicate.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/239/F6.medium.gif" width="440" height="376"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/1/239/F6.medium.gif" width="440" height="376"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/239/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/239/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488071" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-45" class="first-child">Polymorphisms in the <em>ENA6</em> promoter confer differential sensitivity to high salt. (A) A depiction of the promoter region of the sake (blue) and tecc (gray) <em>ENA6</em> alleles tested in this study compared to the reference strain (purple). The sake genetic background has a 33-bp deletion in the ARR2 URS which deletes binding sites for the Mig1, Mig2, and Nrg1 repressors. Representative images of additional tecc and sake background strains are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/FigureS4.tif">Figure S4</a>. (B) The parental strains, a tecc <em>ena6</em>Δ, and three different allele substitution strains in the tecc background were grown on YPD (▴) and YPD + 0.5 M salt (●). The median of each group is indicated with a line. Each condition included 12 technical replicates (four replicates per plate and three plates per condition). In addition, all modified strains (black) had two biological replicates except tecc ENA6-pS-T, which had one biological replicate.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Small, but condition-specific effects of aneuploidy</h3><p id="p-46">RAD-seq genotyping of the progeny (<em>Materials and Methods</em>) uncovered an extra copy of chromosome I from the sake parental strain that segregated at 2:2 through the cross, but with an overrepresentation of chromosome I heterozygosity in the aneuploid progeny (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS6.csv">Table S6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1/TableS12.csv">Table S12</a>). To assess whether the chromosome I disomy affected growth, we compared growth of the euploid and aneuploid strains in each condition. Because some of the conditions had nonnormal phenotype distributions, we used a nonparametric statistical test (Mann–Whitney <em>U</em>-test). We found no significant difference between the two populations on YPD, copper, salt, rapamycin, and SD (α = 0.01 with Bonferroni correction <em>n</em> = 7). Statistically significant differences were found in ethanol (<em>P</em> = 0.0012) and caffeine (<em>P</em> = 0.0004). The mean difference in growth between chromosome I monosomes and disomes were 1.05 mm<sup>2</sup> on ethanol and 2.54 mm<sup>2</sup> on caffeine. Based on these results, we conclude that the chromosome I disomy had a small, but condition-specific effect on the progeny of this cross.</p></div></div><div class="section discussion" id="sec-17"><h2 class="">Discussion</h2><p id="p-47">Our study uncovered a rare polymorphism in the yeast ortholog of the second enzyme in the serine biosynthesis pathway, phosphoserine aminotransferase (<em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em>). Serine has roles in numerous metabolic pathways linked to cellular growth and proliferation. It is a main donor of one-carbon units fueling the folate cycle, methionine cycle, and transsulfuration pathway; and the serine biosynthesis pathway is interconnected with glucose metabolism (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Appling 1991</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Christensen and MacKenzie 2006</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kalhan and Hanson 2012</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lee <em>et al.</em> 2013</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ducker and Rabinowitz 2017</a>). The genes encoding enzymes involved in serine biosynthesis, such as 3-phosphoserine aminotransferase, are highly conserved from yeast (<em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a></em>) to human (<em>PSAT1</em>). This includes the <a href="http://www.yeastgenome.org/locus/S000005710/overview">Ser1</a> residue G78 (G77 of Psat1) responsible for the loss-of-function mutation studied here. Overexpression of serine pathway enzymes in neoplastic cells is thought to drive oncogenesis in a number of human cancers (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Possemato <em>et al.</em> 2011</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Locasale 2013</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">DeNicola <em>et al.</em> 2016</a>). Loss-of-function mutations in these enzymes are linked to a rare class of inborn errors of metabolism known as serine deficiency disorders in which the spectrum of phenotypes is directly associated with the bioavailability of serine (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Tabatabaie <em>et al.</em> 2010</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">El-Hattab <em>et al.</em> 2016</a>). Given that <em>S. cerevisiae</em> preferentially imports amino acids when nutrients are available (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Campbell <em>et al.</em> 2015</a>), it is perhaps surprising that a serine auxotrophy would have such a large effect in rich media conditions. However, our results with this naturally occurring auxotrophy’s effect on growth are consistent with the widespread effects of engineered gene deletions on transcriptional profiles of cells grown in rich conditions (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Ronald and Akey 2007</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Alam <em>et al.</em> 2016</a>), and thus underscore the need to carefully consider the use of auxotrophic markers in genetic analysis.</p><p id="p-48">The sake loss-of-function allele <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-S</em> also confers a growth defect in the parents and progeny of this cross when grown in the presence of rapamycin, the immunosuppressive and antiproliferation drug which blocks cell cycle progression by inhibiting TORC1 (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Heitman <em>et al.</em> 1991</a>). Conserved across eukaryotes, TORC1 is the hub of a complex signaling network that integrates nutrient availability with cell growth and proliferation (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Loewith <em>et al.</em> 2002</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Goberdhan <em>et al.</em> 2016</a>). Our results are consistent with the strong growth defects of the reference strain (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Winston <em>et al.</em> 1995</a>) observed in high throughput screens of the MAT<strong>a</strong> haploid deletion collection (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Parsons <em>et al.</em> 2004</a>) and homozygous diploid deletion collection (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hillenmeyer <em>et al.</em> 2008</a>). One possible explanation for the strong interaction between the <em><a href="http://www.yeastgenome.org/locus/S000005710/overview">SER1</a>-S</em> allele and rapamycin is the significant decrease in amino acid transport by yeast in response to rapamycin (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Schmidt <em>et al.</em> 1998</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Beck <em>et al.</em> 1999</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Gonzalez and Hall 2017</a>).</p><p id="p-49">Our results also identified <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> as the causative gene underlying growth differences in the presence of high salt. The <em>ENA</em> locus is highly polymorphic in <em>S. cerevisiae</em>. Because it is the site of a recent introgression of sequence from <em>S. paradoxus</em>, many strain backgrounds, including the S288c-derived reference strain background (<a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Winston <em>et al.</em> 1995</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Goffeau <em>et al.</em> 1996</a>), harbor multiple copies of hybrid <em>S. cerevisiae <a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> and the <em>S. paradoxus ENA</em> genes (<a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Wieland <em>et al.</em> 1995</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Doniger <em>et al.</em> 2008</a>; <a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Strope <em>et al.</em> 2015</a>). Strains with multiple <em>ENA</em> genes at this locus (<em>e.g.</em>, <em><a href="http://www.yeastgenome.org/locus/S000002447/overview">ENA1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002446/overview">ENA2</a></em>, and <em><a href="http://www.yeastgenome.org/locus/S000002445/overview">ENA5</a></em>) exhibit higher salt tolerance than strains with a single copy of <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em>, although overexpression of <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> can rescue growth on high concentrations of salt (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Daran-Lapujade <em>et al.</em> 2009</a>; <a id="xref-ref-78-4" class="xref-bibr" href="#ref-78">Warringer <em>et al.</em> 2011</a>; <a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Wilkening <em>et al.</em> 2014</a>). Here, we identify a naturally occurring polymorphism in the <em><a href="http://www.yeastgenome.org/locus/S000130715/overview">ENA6</a></em> promoter, in which growth in the presence of high salt is increased in large part due to the deletion of a transcriptional repressor element. Thus, natural isolates of <em>S. cerevisiae</em> have evolved several different mechanisms of increasing expression of the ENA transporters under specific environmental conditions.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-50">We thank Patrick May and Barak Cohen for helpful discussions and Justin Fay for providing strains. This work was funded by National Institutes of Health/National Institute of General Medical Sciences grant R01 GM089978 (to A.M.D.) and by a strategic partnership between the Institute for Systems Biology and the University of Luxembourg.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-51">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300392/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-53"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-08-23"><span class="received-label">Received </span>August 23, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-11-09"><span class="accepted-label">Accepted </span>November 9, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Sirr <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.1" data-doi="10.1016/j.ajhg.2014.07.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Acuna-Hidalgo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schanze</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kariminejad</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordgren</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kariminejad</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Neu-Laxova syndrome is a heterogeneous metabolic disorder caused by defects in enzymes of the L-serine biosynthesis pathway.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">293</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D95%26rft.spage%253D285%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2014.07.012%26rft_id%253Dinfo%253Apmid%252F25152457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2014.07.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25152457&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.2" data-doi="10.1093/dnares/dsr029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yashiro</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hosoyama</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitagaki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horikawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Whole-genome sequencing of sake yeast Saccharomyces cerevisiae Kyokai no. 7.</span> <abbr class="cit-jnl-abbrev">DNA Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fdnares%252Fdsr029%26rft_id%253Dinfo%253Apmid%252F21900213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/dnares/dsr029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21900213&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297990700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alam</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zelezniak</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mülleder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shliaha</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The metabolic background is a global player in Saccharomyces gene expression epistasis.</span> <abbr class="cit-jnl-abbrev">Nat. Microbiol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">15030</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMicrobiol.%26rft.volume%253D1%26rft.spage%253D15030%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Appling</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Compartmentation of folate-mediated one-carbon metabolism in eukaryotes.</span> <abbr class="cit-jnl-abbrev">FASEB J.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2645</span>–<span class="cit-lpage">2651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BFASEB%2BJournal%26rft.stitle%253DFASEB%2BJ.%26rft.aulast%253DAppling%26rft.auinit1%253DD.%2BR.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253D2645%26rft.epage%253D2651%26rft.atitle%253DCompartmentation%2Bof%2Bfolate-mediated%2Bone-carbon%2Bmetabolism%2Bin%2Beukaryotes%26rft_id%253Dinfo%253Apmid%252F1916088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZmFzZWJqIjtzOjU6InJlc2lkIjtzOjk6IjUvMTIvMjY0NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMS8yMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bahiru</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mehari</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashenafi</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Yeast and lactic acid flora of tej, an indigenous Ethiopian honey wine: variations within and between production units.</span> <abbr class="cit-jnl-abbrev">Food Microbiol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFood%2BMicrobiol%26rft.aulast%253DBahiru%26rft.auinit1%253DB.%26rft.volume%253D23%26rft.issue%253D3%26rft.spage%253D277%26rft.epage%253D282%26rft.atitle%253DYeast%2Band%2Blactic%2Bacid%2Bflora%2Bof%2Btej%252C%2Ban%2Bindigenous%2BEthiopian%2Bhoney%2Bwine%253A%2Bvariations%2Bwithin%2Band%2Bbetween%2Bproduction%2Bunits.%26rft_id%253Dinfo%253Apmid%252F16943014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16943014&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mächler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fitting linear mixed-effects models Using lme4.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D67%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.7" data-doi="10.1083/jcb.146.6.1227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beck</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">M. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Starvation induces vacuolar targeting and degradation of the tryptophan permease in yeast.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">146</span>: <span class="cit-fpage">1227</span>–<span class="cit-lpage">1238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.146.6.1227%26rft_id%253Dinfo%253Apmid%252F10491387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNDYvNi8xMjI3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzIzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.8" data-doi="10.1038/nature11867"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bloom</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loo</span>  <span class="cit-name-given-names">W. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lite</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Finding the sources of missing heritability in a yeast cross.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">494</span>: <span class="cit-fpage">234</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D494%26rft.spage%253D234%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11867%26rft_id%253Dinfo%253Apmid%252F23376951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11867&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23376951&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000315137700039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brassier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valayannopoulos</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bahi-Buisson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiame</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubert</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Two new cases of serine deficiency disorders treated with l-serine.</span> <abbr class="cit-jnl-abbrev">Eur. J. Paediatr. Neurol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEur.%2BJ.%2BPaediatr.%2BNeurol.%26rft.volume%253D20%26rft.spage%253D53%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.10" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12724300&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182634800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.11" data-doi="10.7554/eLife.09943"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vowinckel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mülleder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malmsheimer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Self-establishing communities enable cooperative metabolite exchange in a eukaryote.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e09943</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.09943%26rft_id%253Dinfo%253Apmid%252F26499891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZWxpZmUiO3M6NToicmVzaWQiO3M6MTA6IjQvMC9lMDk5NDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.12" data-doi="10.1002/bies.20420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacKenzie</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Mitochondrial one-carbon metabolism is adapted to the specific needs of yeast, plants and mammals.</span> <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%2B%253A%2B%2Bnews%2Band%2Breviews%2Bin%2Bmolecular%252C%2Bcellular%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DBioessays%26rft.aulast%253DChristensen%26rft.auinit1%253DK.%2BE.%26rft.volume%253D28%26rft.issue%253D6%26rft.spage%253D595%26rft.epage%253D605%26rft.atitle%253DMitochondrial%2Bone-carbon%2Bmetabolism%2Bis%2Badapted%2Bto%2Bthe%2Bspecific%2Bneeds%2Bof%2Byeast%252C%2Bplants%2Band%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.20420%26rft_id%253Dinfo%253Apmid%252F16700064%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.20420&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16700064&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238111100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.13" data-doi="10.1111/1574-6976.12065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schothorst</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kankipati</span>  <span class="cit-name-given-names">H. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Zeebroeck</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubio-Texeira</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Nutrient sensing and signaling in the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">254</span>–<span class="cit-lpage">299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1574-6976.12065%26rft_id%253Dinfo%253Apmid%252F24483210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1574-6976.12065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24483210&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000332394200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.239.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cromie</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyma</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ludlow</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garmendia-Torres</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic sequence diversity and population structure of Saccharomyces cerevisiae assessed by RAD-seq.</span> <span class="cit-source">G3</span> <span class="cit-vol">3</span>: <span class="cit-fpage">2163</span>–<span class="cit-lpage">2171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D3%26rft.spage%253D2163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.15" data-doi="10.1534/g3.116.036160"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cromie</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hays</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffery</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">a</span> <span class="cit-article-title">Dissecting gene expression changes accompanying a ploidy-based phenotypic switch.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.036160%26rft_id%253Dinfo%253Apmid%252F27836908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMS8yMzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.239.16" data-doi="10.1534/g3.117.042440"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cromie</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hays</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sirr</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffery</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">b</span> <span class="cit-article-title">Transcriptional profiling of biofilm regulators identified by an overexpression screen in Saccharomyces cerevisiae.</span> <span class="cit-source">G3</span> <span class="cit-vol">7</span>: <span class="cit-fpage">2845</span>–<span class="cit-lpage">2854</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.042440%26rft_id%253Dinfo%253Apmid%252F28673928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvOC8yODQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzIzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.17" data-doi="10.1111/j.1567-1364.2009.00530.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daran-Lapujade</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daran</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luttik</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Almering</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pronk</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">An atypical PMR2 locus is responsible for hypersensitivity to sodium and lithium cations in the laboratory strain Saccharomyces cerevisiae CEN.PK113–7D.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">792</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2009.00530.x%26rft_id%253Dinfo%253Apmid%252F19519766%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1567-1364.2009.00530.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19519766&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.18" data-doi="10.1023/A:1015624726822"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duran</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Maldergem</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pineda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorland</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Congenital microcephaly and seizures due to 3-phosphoglycerate dehydrogenase deficiency: outcome of treatment with amino acids.</span> <abbr class="cit-jnl-abbrev">J. Inherit. Metab. Dis.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">119</span>–<span class="cit-lpage">125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Binherited%2Bmetabolic%2Bdisease%26rft.stitle%253DJ%2BInherit%2BMetab%2BDis%26rft.aulast%253DDe%2BKoning%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D25%26rft.issue%253D2%26rft.spage%253D119%26rft.epage%253D125%26rft.atitle%253DCongenital%2Bmicrocephaly%2Band%2Bseizures%2Bdue%2Bto%2B3-phosphoglycerate%2Bdehydrogenase%2Bdeficiency%253A%2Boutcome%2Bof%2Btreatment%2Bwith%2Bamino%2Bacids.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1015624726822%26rft_id%253Dinfo%253Apmid%252F12118526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1015624726822&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12118526&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175800500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.19" data-doi="10.1016/S0140-6736(04)17596-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klomp</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Oppen</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beemer</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorland</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Prenatal and early postnatal treatment in 3-phosphoglycerate-dehydrogenase deficiency.</span> <abbr class="cit-jnl-abbrev">Lancet</abbr> <span class="cit-vol">364</span>: <span class="cit-fpage">2221</span>–<span class="cit-lpage">2222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLancet%26rft.stitle%253DLancet%26rft.aulast%253Dde%2BKoning%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D364%26rft.issue%253D9452%26rft.spage%253D2221%26rft.epage%253D2222%26rft.atitle%253DPrenatal%2Band%2Bearly%2Bpostnatal%2Btreatment%2Bin%2B3-phosphoglycerate-dehydrogenase%2Bdeficiency.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0140-6736%252804%252917596-X%26rft_id%253Dinfo%253Apmid%252F15610810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0140-6736(04)17596-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15610810&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225799700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DeNicola</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullarky</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sudderth</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Erratum: NRF2 regulates serine biosynthesis in non-small cell lung cancer.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">473</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.21" data-doi="10.1371/journal.pgen.1000183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doniger</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swain</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corcuera</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A catalog of neutral and deleterious polymorphism in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDoniger%26rft.auinit1%253DS.%2BW.%26rft.volume%253D4%26rft.issue%253D8%26rft.spage%253De1000183%26rft.epage%253De1000183%26rft.atitle%253DA%2Bcatalog%2Bof%2Bneutral%2Band%2Bdeleterious%2Bpolymorphism%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000183%26rft_id%253Dinfo%253Apmid%252F18769710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18769710&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ducker</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabinowitz</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">One-carbon metabolism in health and disease.</span> <abbr class="cit-jnl-abbrev">Cell Metab.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BMetab.%26rft.volume%253D25%26rft.spage%253D27%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Hattab</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Inborn errors of metabolism.</span> <abbr class="cit-jnl-abbrev">Clin. Perinatol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">413</span>–<span class="cit-lpage">439</span>, <span class="cit-lpage">x</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BPerinatol.%26rft.volume%253D42%26rft.spage%253D413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Hattab</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaheen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hertecant</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galadari</span>  <span class="cit-name-given-names">H. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albaqawi</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">On the phenotypic spectrum of serine biosynthesis defects.</span> <abbr class="cit-jnl-abbrev">J. Inherit. Metab. Dis.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInherit.%2BMetab.%2BDis.%26rft.volume%253D39%26rft.spage%253D373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.25" data-doi="10.1371/journal.pgen.0010066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benavides</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evidence for domesticated and wild populations of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">66</span>–<span class="cit-lpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D1%26rft.issue%253D1%26rft.spage%253D66%26rft.epage%253D71%26rft.atitle%253DEvidence%2Bfor%2Bdomesticated%2Band%2Bwild%2Bpopulations%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0010066%26rft_id%253Dinfo%253Apmid%252F16103919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0010066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16103919&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234295900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.26" data-doi="10.1016/j.cmet.2016.03.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goberdhan</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Amino acid sensing by mTORC1: intracellular transporters mark the spot.</span> <abbr class="cit-jnl-abbrev">Cell Metab.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">580</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BMetab.%26rft.volume%253D23%26rft.spage%253D580%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cmet.2016.03.013%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cmet.2016.03.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.27" data-doi="10.1111/j.1462-2920.2009.02035.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anfang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jun</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A distinct population of Saccharomyces cerevisiae in New Zealand: evidence for local dispersal by insects and human-aided global dispersal in oak barrels.</span> <abbr class="cit-jnl-abbrev">Environ. Microbiol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">63</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DEnvironmental%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BR.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D63%26rft.epage%253D73%26rft.atitle%253DA%2Bdistinct%2Bpopulation%2Bof%2BSaccharomyces%2Bcerevisiae%2Bin%2BNew%2BZealand%253A%2Bevidence%2Bfor%2Blocal%2Bdispersal%2Bby%2Binsects%2Band%2Bhuman-aided%2Bglobal%2Bdispersal%2Bin%2Boak%2Bbarrels.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1462-2920.2009.02035.x%26rft_id%253Dinfo%253Apmid%252F19691498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1462-2920.2009.02035.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19691498&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274234200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.28" data-doi="10.1126/science.274.5287.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goffeau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrell</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussey</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dujon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Life with 6000 genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">546</span>, <span class="cit-fpage">563</span>–<span class="cit-lpage">547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGoffeau%26rft.auinit1%253DA.%26rft.volume%253D274%26rft.issue%253D5287%26rft.spage%253D546%26rft.epage%253D567%26rft.atitle%253DLife%2Bwith%2B6000%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.274.5287.546%26rft_id%253Dinfo%253Apmid%252F8849441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNzQvNTI4Ny81NDYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.29" data-doi="10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Three new dominant drug resistance cassettes for gene disruption in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1541</span>–<span class="cit-lpage">1553</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DGoldstein%26rft.auinit1%253DA.%2BL.%26rft.volume%253D15%26rft.issue%253D14%26rft.spage%253D1541%26rft.epage%253D1553%26rft.atitle%253DThree%2Bnew%2Bdominant%2Bdrug%2Bresistance%2Bcassettes%2Bfor%2Bgene%2Bdisruption%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199910%252915%253A14%253C1541%253A%253AAID-YEA476%253E3.0.CO%253B2-K%26rft_id%253Dinfo%253Apmid%252F10514571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199910)15:14&lt;1541::AID-YEA476&gt;3.0.CO;2-K&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10514571&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083104100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.30" data-doi="10.15252/embj.201696010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">M. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Nutrient sensing and TOR signaling in yeast and mammals.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fembj.201696010%26rft_id%253Dinfo%253Apmid%252F28096180%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo4OiIzNi80LzM5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMS8yMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.31" data-doi="10.1016/0014-5793(91)81280-L"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haro</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garciadeblas</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez-Navarro</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">A novel P-type ATPase from yeast involved in sodium transport.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">291</span>: <span class="cit-fpage">189</span>–<span class="cit-lpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DHaro%26rft.auinit1%253DR.%26rft.volume%253D291%26rft.issue%253D2%26rft.spage%253D189%26rft.epage%253D191%26rft.atitle%253DA%2Bnovel%2BP-type%2BATPase%2Bfrom%2Byeast%2Binvolved%2Bin%2Bsodium%2Btransport.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0014-5793%252891%252981280-L%26rft_id%253Dinfo%253Apmid%252F1657642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0014-5793(91)81280-L&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1657642&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991GN10000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.32" data-doi="10.1086/517888"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hart</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Race</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Achouri</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiame</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharrard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Phosphoserine aminotransferase deficiency: a novel disorder of the serine biosynthesis pathway.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">931</span>–<span class="cit-lpage">937</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DHart%26rft.auinit1%253DC.%2BE.%26rft.volume%253D80%26rft.issue%253D5%26rft.spage%253D931%26rft.epage%253D937%26rft.atitle%253DPhosphoserine%2Baminotransferase%2Bdeficiency%253A%2Ba%2Bnovel%2Bdisorder%2Bof%2Bthe%2Bserine%2Bbiosynthesis%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F517888%26rft_id%253Dinfo%253Apmid%252F17436247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/517888&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17436247&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.33" data-doi="10.1128/AEM.71.1.312-319.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hashimoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aritomi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoshida</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishizawa</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Isolation of auxotrophic mutants of diploid industrial yeast strains after UV mutagenesis.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">312</span>–<span class="cit-lpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DHashimoto%26rft.auinit1%253DS.%26rft.volume%253D71%26rft.issue%253D1%26rft.spage%253D312%26rft.epage%253D319%26rft.atitle%253DIsolation%2Bof%2BAuxotrophic%2BMutants%2Bof%2BDiploid%2BIndustrial%2BYeast%2BStrains%2Bafter%2BUV%2BMutagenesis%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.71.1.312-319.2005%26rft_id%253Dinfo%253Apmid%252F15640203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjg6IjcxLzEvMzEyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzIzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.34" data-doi="10.1126/science.1715094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heitman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Movva</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">M. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Targets for cell cycle arrest by the immunosuppressant rapamycin in yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">253</span>: <span class="cit-fpage">905</span>–<span class="cit-lpage">909</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHeitman%26rft.auinit1%253DJ.%26rft.volume%253D253%26rft.issue%253D5022%26rft.spage%253D905%26rft.epage%253D909%26rft.atitle%253DTargets%2Bfor%2Bcell%2Bcycle%2Barrest%2Bby%2Bthe%2Bimmunosuppressant%2Brapamycin%2Bin%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1715094%26rft_id%253Dinfo%253Apmid%252F1715094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNTMvNTAyMi85MDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.35" data-doi="10.1006/jmbi.1998.2506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hester</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kallen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Markovic-Housley</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Crystal structure of phosphoserine aminotransferase from Escherichia coli at 2.3 A resolution: comparison of the unligated enzyme and a complex with alpha-methyl-l-glutamate.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">829</span>–<span class="cit-lpage">850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DHester%26rft.auinit1%253DG.%26rft.volume%253D286%26rft.issue%253D3%26rft.spage%253D829%26rft.epage%253D850%26rft.atitle%253DCrystal%2Bstructure%2Bof%2Bphosphoserine%2Baminotransferase%2Bfrom%2BEscherichia%2Bcoli%2Bat%2B2.3%2BA%2Bresolution%253A%2Bcomparison%2Bof%2Bthe%2Bunligated%2Benzyme%2Band%2Ba%2Bcomplex%2Bwith%2Balpha-methyl-l-glutamate.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1998.2506%26rft_id%253Dinfo%253Apmid%252F10024454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1998.2506&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10024454&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.36" data-doi="10.1126/science.1150021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hillenmeyer</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wildenhain</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pierce</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The chemical genomic portrait of yeast: uncovering a phenotype for all genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">365</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHillenmeyer%26rft.auinit1%253DM.%2BE.%26rft.volume%253D320%26rft.issue%253D5874%26rft.spage%253D362%26rft.epage%253D365%26rft.atitle%253DThe%2BChemical%2BGenomic%2BPortrait%2Bof%2BYeast%253A%2BUncovering%2Ba%2BPhenotype%2Bfor%2BAll%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1150021%26rft_id%253Dinfo%253Apmid%252F18420932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjAvNTg3NC8zNjIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sigwalt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fournier</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pflieger</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The hidden complexity of Mendelian traits across natural yeast populations.</span> <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1106</span>–<span class="cit-lpage">1114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D16%26rft.spage%253D1106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.38" data-doi="10.1074/jbc.R112.357194"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalhan</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanson</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Resurgence of serine: an often neglected but indispensable amino Acid.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">19786</span>–<span class="cit-lpage">19791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.R112.357194%26rft_id%253Dinfo%253Apmid%252F22566694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODcvMjQvMTk3ODYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.39" data-doi="10.1073/pnas.0708194104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic variation in the cysteine biosynthesis pathway causes sensitivity to pharmacological compounds.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">19387</span>–<span class="cit-lpage">19391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0708194104%26rft_id%253Dinfo%253Apmid%252F18042712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzQ5LzE5Mzg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzIzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">A nonparametric approach for mapping quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">1421</span>–<span class="cit-lpage">1428</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKruglyak%26rft.auinit1%253DL.%26rft.volume%253D139%26rft.issue%253D3%26rft.spage%253D1421%26rft.epage%253D1428%26rft.atitle%253DA%2Bnonparametric%2Bapproach%2Bfor%2Bmapping%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Apmid%252F7768449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzOS8zLzE0MjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.41" data-doi="10.1128/MCB.23.2.677-686.2003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamb</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The transcription factor Rim101p governs ion tolerance and cell differentiation by direct repression of the regulatory genes NRG1 and SMP1 in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">677</span>–<span class="cit-lpage">686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DLamb%26rft.auinit1%253DT.%2BM.%26rft.volume%253D23%26rft.issue%253D2%26rft.spage%253D677%26rft.epage%253D686%26rft.atitle%253DThe%2BTranscription%2BFactor%2BRim101p%2BGoverns%2BIon%2BTolerance%2Band%2BCell%2BDifferentiation%2Bby%2BDirect%2BRepression%2Bof%2Bthe%2BRegulatory%2BGenes%2BNRG1%2Band%2BSMP1%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.23.2.677-686.2003%26rft_id%253Dinfo%253Apmid%252F12509465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjIzLzIvNjc3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzIzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Mapping mendelian factors underlying quantitative traits using RFLP linkage maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D199%26rft.atitle%253DMapping%2Bmendelian%2Bfactors%2Bunderlying%2Bquantitative%2Btraits%2Busing%2BRFLP%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Apmid%252F2563713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIxLzEvMTg1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzIzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lanpher</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunetti-Pierri</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Inborn errors of metabolism: the flux from Mendelian to complex diseases.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DLanpher%26rft.auinit1%253DB.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253D449%26rft.epage%253D460%26rft.atitle%253DInborn%2Berrors%2Bof%2Bmetabolism%253A%2Bthe%2Bflux%2Bfrom%2BMendelian%2Bto%2Bcomplex%2Bdiseases.%26rft_id%253Dinfo%253Apmid%252F16708072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16708072&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238377000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.44" data-doi="10.1111/1567-1364.12063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsoi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kornfeld</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawes</span>  <span class="cit-name-given-names">I. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Cellular responses to L-serine in Saccharomyces cerevisiae: roles of general amino acid control, compartmentalization, and aspartate synthesis.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">618</span>–<span class="cit-lpage">634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1567-1364.12063%26rft_id%253Dinfo%253Apmid%252F23837815%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1567-1364.12063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23837815&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.45" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.46" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAMtools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.47" data-doi="10.1534/genetics.106.062166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Sequence diversity, reproductive isolation and species concepts in Saccharomyces.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">839</span>–<span class="cit-lpage">850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.062166%26rft_id%253Dinfo%253Apmid%252F16951060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc0LzIvODM5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzIzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.48" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.49" data-doi="10.1038/nrc3557"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Locasale</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Serine, glycine and one-carbon units: cancer metabolism in full circle.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Cancer</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">572</span>–<span class="cit-lpage">583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BCancer%26rft.volume%253D13%26rft.spage%253D572%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrc3557%26rft_id%253Dinfo%253Apmid%252F23822983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrc3557&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23822983&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000322158200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.50" data-doi="10.1016/S1097-2765(02)00636-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loewith</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacinto</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wullschleger</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crespo</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Two TOR complexes, only one of which is rapamycin sensitive, have distinct roles in cell growth control.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">457</span>–<span class="cit-lpage">468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DLoewith%26rft.auinit1%253DR.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D457%26rft.epage%253D468%26rft.atitle%253DTwo%2BTOR%2Bcomplexes%252C%2Bonly%2Bone%2Bof%2Bwhich%2Bis%2Brapamycin%2Bsensitive%252C%2Bhave%2Bdistinct%2Broles%2Bin%2Bcell%2Bgrowth%2Bcontrol.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252802%252900636-6%26rft_id%253Dinfo%253Apmid%252F12408816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(02)00636-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12408816&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178330900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.51" data-doi="10.1038/nmeth.2479"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ludlow</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cromie</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffery</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sirr</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-throughput tetrad analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">671</span>–<span class="cit-lpage">675</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D671%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2479%26rft_id%253Dinfo%253Apmid%252F23666411%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2479&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23666411&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.52" data-doi="10.1016/j.cub.2016.02.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ludlow</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cromie</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garmendia-Torres</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sirr</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hays</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Independent origins of yeast associated with coffee and cacao fermentation.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">965</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D26%26rft.spage%253D965%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2016.02.012%26rft_id%253Dinfo%253Apmid%252F27020745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2016.02.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27020745&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lutfiyya</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iyer</span>  <span class="cit-name-given-names">V. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeRisi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeVit</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Characterization of three related glucose repressors and genes they regulate in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">1377</span>–<span class="cit-lpage">1391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLutfiyya%26rft.auinit1%253DL.%2BL.%26rft.volume%253D150%26rft.issue%253D4%26rft.spage%253D1377%26rft.epage%253D1391%26rft.atitle%253DCharacterization%2Bof%2BThree%2BRelated%2BGlucose%2BRepressors%2Band%2BGenes%2BThey%2BRegulate%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F9832517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1MC80LzEzNzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.54" data-doi="10.1007/BF00351686"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melcher</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Entian</span>  <span class="cit-name-given-names">K. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Genetic analysis of serine biosynthesis and glucose repression in yeast.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DMelcher%26rft.auinit1%253DK.%26rft.volume%253D21%26rft.issue%253D4-5%26rft.spage%253D295%26rft.epage%253D300%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bserine%2Bbiosynthesis%2Band%2Bglucose%2Brepression%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00351686%26rft_id%253Dinfo%253Apmid%252F1326413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00351686&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1326413&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.55" data-doi="10.1007/BF00314439"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melcher</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunzler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braus</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Entian</span>  <span class="cit-name-given-names">K. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Molecular analysis of the yeast SER1 gene encoding 3-phosphoserine aminotransferase: regulation by general control and serine repression.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">508</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DMelcher%26rft.auinit1%253DK.%26rft.volume%253D27%26rft.issue%253D6%26rft.spage%253D501%26rft.epage%253D508%26rft.atitle%253DMolecular%2Banalysis%2Bof%2Bthe%2Byeast%2BSER1%2Bgene%2Bencoding%2B3-phosphoserine%2Baminotransferase%253A%2Bregulation%2Bby%2Bgeneral%2Bcontrol%2Band%2Bserine%2Brepression.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00314439%26rft_id%253Dinfo%253Apmid%252F7553933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00314439&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7553933&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RB41600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palm</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">C. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Nutrient acquisition strategies of mammalian cells.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">546</span>: <span class="cit-fpage">234</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D546%26rft.spage%253D234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.57" data-doi="10.1038/nbt919"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brost</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Integration of chemical-genetic and genetic interaction data links bioactive compounds to cellular target pathways.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">62</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DParsons%26rft.auinit1%253DA.%2BB.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D62%26rft.epage%253D69%26rft.atitle%253DIntegration%2Bof%2Bchemical-genetic%2Band%2Bgenetic%2Binteraction%2Bdata%2Blinks%2Bbioactive%2Bcompounds%2Bto%2Bcellular%2Btarget%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt919%26rft_id%253Dinfo%253Apmid%252F14661025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt919&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14661025&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187745400028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.58" data-doi="10.1074/jbc.M606483200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Platara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruiz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serrano</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palomino</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The transcriptional response of the yeast Na(+)-ATPase ENA1 gene to alkaline stress involves three main signaling pathways.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">36632</span>–<span class="cit-lpage">36642</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M606483200%26rft_id%253Dinfo%253Apmid%252F17023428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODEvNDgvMzY2MzIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.59" data-doi="10.1038/nature10350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Possemato</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marks</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaul</span>  <span class="cit-name-given-names">Y. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pacold</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Functional genomics reveal that the serine synthesis pathway is essential in breast cancer.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">476</span>: <span class="cit-fpage">346</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPossemato%26rft.auinit1%253DR.%26rft.volume%253D476%26rft.issue%253D7360%26rft.spage%253D346%26rft.epage%253D350%26rft.atitle%253DFunctional%2Bgenomics%2Breveal%2Bthat%2Bthe%2Bserine%2Bsynthesis%2Bpathway%2Bis%2Bessential%2Bin%2Bbreast%2Bcancer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10350%26rft_id%253Dinfo%253Apmid%252F21760589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10350&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21760589&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294206500041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.60" data-doi="10.1128/MCB.19.1.537"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Proft</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serrano</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Repressors and upstream repressing sequences of the stress-regulated ENA1 gene in Saccharomyces cerevisiae: bZIP protein Sko1p confers HOG-dependent osmotic regulation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DProft%26rft.auinit1%253DM.%26rft.volume%253D19%26rft.issue%253D1%26rft.spage%253D537%26rft.epage%253D546%26rft.atitle%253DRepressors%2Band%2BUpstream%2BRepressing%2BSequences%2Bof%2Bthe%2BStress-Regulated%2BENA1%2BGene%2Bin%2BSaccharomyces%2Bcerevisiae%253A%2BbZIP%2BProtein%2BSko1p%2BConfers%2BHOG-Dependent%2BOsmotic%2BRegulation%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.19.1.537%26rft_id%253Dinfo%253Apmid%252F9858577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvNTM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzIzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.61" data-doi="10.1128/AEM.68.5.2095-2100.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pronk</span>  <span class="cit-name-given-names">J. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Auxotrophic yeast strains in fundamental and applied research.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">2095</span>–<span class="cit-lpage">2100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2Band%2BEnvironmental%2BMicrobiology%26rft.stitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft.aulast%253DPronk%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D68%26rft.issue%253D5%26rft.spage%253D2095%26rft.epage%253D2100%26rft.atitle%253DAuxotrophic%2BYeast%2BStrains%2Bin%2BFundamental%2Band%2BApplied%2BResearch%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.68.5.2095-2100.2002%26rft_id%253Dinfo%253Apmid%252F11976076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6IjY4LzUvMjA5NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMS8yMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.62" data-doi="10.1371/journal.pone.0000678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ronald</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akey</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The evolution of gene expression QTL in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e678</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DRonald%26rft.auinit1%253DJ.%26rft.volume%253D2%26rft.issue%253D7%26rft.spage%253De678%26rft.epage%253De678%26rft.atitle%253DThe%2Bevolution%2Bof%2Bgene%2Bexpression%2BQTL%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0000678%26rft_id%253Dinfo%253Apmid%252F17668057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0000678&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17668057&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.239.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hieter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source"><em>Methods in Yeast Genetics: A Laboratory Course Manual</em>.</span> <span class="cit-publ-name">Cold Spring Harbor Laboratory Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.64" data-doi="10.1016/0092-8674(89)90410-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rudolph</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antebi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckley</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorman</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">The yeast secretory pathway is perturbed by mutations in PMR1, a member of a Ca2+ ATPase family.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DRudolph%26rft.auinit1%253DH.%2BK.%26rft.volume%253D58%26rft.issue%253D1%26rft.spage%253D133%26rft.epage%253D145%26rft.atitle%253DThe%2Byeast%2Bsecretory%2Bpathway%2Bis%2Bperturbed%2Bby%2Bmutations%2Bin%2BPMR1%252C%2Ba%2Bmember%2Bof%2Ba%2BCa2%252B%2BATPase%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252889%252990410-8%26rft_id%253Dinfo%253Apmid%252F2526682%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(89)90410-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2526682&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AG04600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.65" data-doi="10.1128/EC.00337-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruiz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arino</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Function and regulation of the Saccharomyces cerevisiae ENA sodium ATPase system.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2175</span>–<span class="cit-lpage">2183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00337-07%26rft_id%253Dinfo%253Apmid%252F17951516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo5OiI2LzEyLzIxNzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.66" data-doi="10.1038/nature07670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruderfer</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comprehensive polymorphism survey elucidates population structure of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchacherer%26rft.auinit1%253DJ.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D342%26rft.epage%253D345%26rft.atitle%253DComprehensive%2Bpolymorphism%2Bsurvey%2Belucidates%2Bpopulation%2Bstructure%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07670%26rft_id%253Dinfo%253Apmid%252F19212320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212320&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.67" data-doi="10.1093/emboj/17.23.6924"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beck</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koller</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">M. N.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The TOR nutrient signalling pathway phosphorylates NPR1 and inhibits turnover of the tryptophan permease.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">6924</span>–<span class="cit-lpage">6931</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DSchmidt%26rft.auinit1%253DA.%26rft.volume%253D17%26rft.issue%253D23%26rft.spage%253D6924%26rft.epage%253D6931%26rft.atitle%253DThe%2BTOR%2Bnutrient%2Bsignalling%2Bpathway%2Bphosphorylates%2BNPR1%2Band%2Binhibits%2Bturnover%2Bof%2Bthe%2Btryptophan%2Bpermease%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F17.23.6924%26rft_id%253Dinfo%253Apmid%252F9843498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMTcvMjMvNjkyNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMS8yMzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.68" data-doi="10.1038/nmeth.2089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasband</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eliceiri</span>  <span class="cit-name-given-names">K. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">NIH image to ImageJ: 25 years of image analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">671</span>–<span class="cit-lpage">675</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSchneider%26rft.auinit1%253DC.%2BA.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D671%26rft.epage%253D675%26rft.atitle%253DNIH%2BImage%2Bto%2BImageJ%253A%2B25%2Byears%2Bof%2Bimage%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2089%26rft_id%253Dinfo%253Apmid%252F22930834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22930834&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305942200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.69" data-doi="10.1046/j.1365-2958.2002.03246.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serrano</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruiz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernal</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chambers</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arino</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The transcriptional response to alkaline pH in Saccharomyces cerevisiae: evidence for calcium-mediated signalling.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1319</span>–<span class="cit-lpage">1333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DSerrano%26rft.auinit1%253DR.%26rft.volume%253D46%26rft.issue%253D5%26rft.spage%253D1319%26rft.epage%253D1333%26rft.atitle%253DThe%2Btranscriptional%2Bresponse%2Bto%2Balkaline%2BpH%2Bin%2BSaccharomyces%2Bcerevisiae%253A%2Bevidence%2Bfor%2Bcalcium-mediated%2Bsignalling.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2002.03246.x%26rft_id%253Dinfo%253Apmid%252F12453218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2002.03246.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12453218&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.70" data-doi="10.1534/genetics.114.170563"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sirr</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cromie</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffery</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ludlow</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Allelic variation, aneuploidy, and nongenetic mechanisms suppress a monogenic trait in yeast.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">262</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.170563%26rft_id%253Dinfo%253Apmid%252F25398792%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzEvMjQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzIzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.71" data-doi="10.1101/gr.185538.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strope</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skelly</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozmin</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahadevan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The 100-genomes strains, an S. cerevisiae resource that illuminates its natural phenotypic and genotypic variation and emergence as an opportunistic pathogen.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">762</span>–<span class="cit-lpage">774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.185538.114%26rft_id%253Dinfo%253Apmid%252F25840857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzUvNzYyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzIzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.72" data-doi="10.1016/j.ymgme.2009.10.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tabatabaie</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klomp</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">L-serine synthesis in the central nervous system: a review on serine deficiency disorders.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Metab.</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">256</span>–<span class="cit-lpage">262</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bmetabolism%26rft.stitle%253DMol%2BGenet%2BMetab%26rft.aulast%253DTabatabaie%26rft.auinit1%253DL.%26rft.volume%253D99%26rft.issue%253D3%26rft.spage%253D256%26rft.epage%253D262%26rft.atitle%253DL-serine%2Bsynthesis%2Bin%2Bthe%2Bcentral%2Bnervous%2Bsystem%253A%2Ba%2Breview%2Bon%2Bserine%2Bdeficiency%2Bdisorders.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymgme.2009.10.012%26rft_id%253Dinfo%253Apmid%252F19963421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymgme.2009.10.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19963421&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.73" data-doi="10.1073/pnas.1301047110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hays</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cromie</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffery</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Aneuploidy underlies a multicellular phenotypic switch.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">12367</span>–<span class="cit-lpage">12372</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1301047110%26rft_id%253Dinfo%253Apmid%252F23812752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzMwLzEyMzY3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzIzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.74" data-doi="10.1126/science.1065810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evangelista</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsons</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bader</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Systematic genetic analysis with ordered arrays of yeast deletion mutants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>: <span class="cit-fpage">2364</span>–<span class="cit-lpage">2368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTong%26rft.auinit1%253DA.%2BH.%2BY.%26rft.volume%253D294%26rft.issue%253D5550%26rft.spage%253D2364%26rft.epage%253D2368%26rft.atitle%253DSystematic%2BGenetic%2BAnalysis%2Bwith%2BOrdered%2BArrays%2Bof%2BYeast%2BDeletion%2BMutants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1065810%26rft_id%253Dinfo%253Apmid%252F11743205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTQvNTU1MC8yMzY0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzIzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ulane</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogur</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Genetic and physiological control of serine and glycine biosynthesis in Saccharomyces.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">34</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DUlane%26rft.auinit1%253DR.%26rft.volume%253D109%26rft.issue%253D1%26rft.spage%253D34%26rft.epage%253D43%26rft.atitle%253DGenetic%2Band%2BPhysiological%2BControl%2Bof%2BSerine%2Band%2BGlycine%2BBiosynthesis%2Bin%2BSaccharomyces%26rft_id%253Dinfo%253Apmid%252F4333378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6ODoiMTA5LzEvMzQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vernon</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Inborn errors of metabolism: advances in diagnosis and therapy.</span> <abbr class="cit-jnl-abbrev">JAMA Pediatr.</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">778</span>–<span class="cit-lpage">782</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJAMA%2BPediatr.%26rft.volume%253D169%26rft.spage%253D778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vyas</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuchin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Interaction of the repressors Nrg1 and Nrg2 with the Snf1 protein kinase in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DVyas%26rft.auinit1%253DV.%2BK.%26rft.volume%253D158%26rft.issue%253D2%26rft.spage%253D563%26rft.epage%253D572%26rft.atitle%253DInteraction%2Bof%2Bthe%2BRepressors%2BNrg1%2Band%2BNrg2%2BWith%2Bthe%2BSnf1%2BProtein%2BKinase%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F11404322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU4LzIvNTYzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzIzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.78" data-doi="10.1371/journal.pgen.1002111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zorgo</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zia</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gjuvsland</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Trait variation in yeast is defined by population history.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWarringer%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De1002111%26rft.epage%253De1002111%26rft.atitle%253DTrait%2Bvariation%2Bin%2Byeast%2Bis%2Bdefined%2Bby%2Bpopulation%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002111%26rft_id%253Dinfo%253Apmid%252F21698134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21698134&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.79"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wieland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nitsche</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strayle</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steiner</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudolph</span>  <span class="cit-name-given-names">H. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">The PMR2 gene cluster encodes functionally distinct isoforms of a putative Na+ pump in the yeast plasma membrane.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">3870</span>–<span class="cit-lpage">3882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft.volume%253D14%26rft.spage%253D3870%26rft_id%253Dinfo%253Apmid%252F7664728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7664728&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RQ66900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.80" data-doi="10.1534/genetics.113.160291"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilkening</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritsch</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tekkedil</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An evaluation of high-throughput approaches to QTL mapping in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.160291%26rft_id%253Dinfo%253Apmid%252F24374355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk2LzMvODUzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzIzOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.81" data-doi="10.1002/yea.320110107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dollard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ricupero-Hovasse</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Construction of a set of convenient Saccharomyces cerevisiae strains that are isogenic to S288C.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DWinston%26rft.auinit1%253DF.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D53%26rft.epage%253D55%26rft.atitle%253DConstruction%2Bof%2Ba%2Bset%2Bof%2Bconvenient%2BSaccharomyces%2Bcerevisiae%2Bstrains%2Bthat%2Bare%2Bisogenic%2Bto%2BS288C.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320110107%26rft_id%253Dinfo%253Apmid%252F7762301%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320110107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7762301&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F239.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QF61400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.239.82" data-doi="10.1126/science.285.5429.901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Astromoff</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Functional characterization of the S. cerevisiae genome by gene deletion and parallel analysis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">901</span>–<span class="cit-lpage">906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWinzeler%26rft.auinit1%253DE.%2BA.%26rft.volume%253D285%26rft.issue%253D5429%26rft.spage%253D901%26rft.epage%253D906%26rft.atitle%253DFunctional%2BCharacterization%2Bof%2Bthe%2BS.%2Bcerevisiae%2BGenome%2Bby%2BGene%2BDeletion%2Band%2BParallel%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.285.5429.901%26rft_id%253Dinfo%253Apmid%252F10436161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvNTQyOS85MDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMjM5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/1/239.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/231" title="Egg Case Silk Gene Sequences from Argiope Spiders: Evidence for Multiple Loci and a Loss of Function Between Paralogs" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/253" title="Rice TSV3 Encoding Obg-Like GTPase Protein Is Essential for Chloroplast Development During the Early Leaf Stage Under Cold Stress" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/1"><span>Volume 8 Issue 1, January 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/1.cover-source.jpg?itok=vyoqxep2" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/239" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VJvjlTP86Uc8BSDCJMlxqm1qFk4BGVdViJTl5LmyoZw" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/1/239" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/1/239" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/1/239" class="active" data-icon-position="" data-hide-link-title="0">Natural Variation in SER1 and ENA6 Underlie Condition-Specific Growth Defects in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487835" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-834yiUmcld3qa63rTeEF5TKTbkfCeDljrXJbcJGWr58" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/239.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/1/239" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-sGa4AWLpq3qIQWm-KRn6-_uwe1jYLp96AXx8TTu8nus" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/1/239" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/239.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487835" id="node487835--2" data-pisa="ggg;8/1/239" data-pisa-master="ggg;g3.117.300392" data-apath="/ggg/8/1/239.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Natural Variation in <em>SER1</em> and <em>ENA6</em> Underlie Condition-Specific Growth Defects in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5186-9692" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amy Sirr</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4511-0898" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Adrian C. Scott</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5265-1970" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gareth A. Cromie</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-9682-7851" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Catherine L. Ludlow</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-8632-9629" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vida Ahyong</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-9299-8562" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Trey S. Morgan</span>, <span class="highwire-citation-author" data-delta="6">Teresa Gilbert</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aimée M. Dudley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">239-251;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300392 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487835--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amy Sirr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BSirr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sirr%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmy%2BSirr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5186-9692" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amy Sirr</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Adrian C. Scott </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adrian%2BC.%2BScott%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scott%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAdrian%2BC.%2BScott%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4511-0898" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Adrian C. Scott</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Gareth A. Cromie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gareth%2BA.%2BCromie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cromie%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGareth%2BA.%2BCromie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5265-1970" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gareth A. Cromie</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Catherine L. Ludlow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BL.%2BLudlow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ludlow%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACatherine%2BL.%2BLudlow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9682-7851" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Catherine L. Ludlow</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Vida Ahyong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Cellular Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vida%2BAhyong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ahyong%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVida%2BAhyong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8632-9629" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vida Ahyong</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Trey S. Morgan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Trey%2BS.%2BMorgan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morgan%20TS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATrey%2BS.%2BMorgan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9299-8562" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Trey S. Morgan</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Teresa Gilbert </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Teresa%2BGilbert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gilbert%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATeresa%2BGilbert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aimée M. Dudley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aim%C3%A9e%2BM.%2BDudley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dudley%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAim%25C3%25A9e%2BM.%2BDudley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aimée M. Dudley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/1/239" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/1/239&amp;title=Natural%20Variation%20in%20SER1%20and%20ENA6%20Underlie%20Condition-Specific%20Growth%20Defects%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/1/239&amp;title=Natural%20Variation%20in%20SER1%20and%20ENA6%20Underlie%20Condition-Specific%20Growth%20Defects%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/1/239&amp;title=Natural%20Variation%20in%20SER1%20and%20ENA6%20Underlie%20Condition-Specific%20Growth%20Defects%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/1/239&amp;text=Natural%20Variation%20in%20SER1%20and%20ENA6%20Underlie%20Condition-Specific%20Growth%20Defects%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/1/239&amp;title=Natural%20Variation%20in%20SER1%20and%20ENA6%20Underlie%20Condition-Specific%20Growth%20Defects%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/1/239&amp;t=Natural%20Variation%20in%20SER1%20and%20ENA6%20Underlie%20Condition-Specific%20Growth%20Defects%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/1/239&amp;title=Natural%20Variation%20in%20SER1%20and%20ENA6%20Underlie%20Condition-Specific%20Growth%20Defects%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/1/239&amp;title=Natural%20Variation%20in%20SER1%20and%20ENA6%20Underlie%20Condition-Specific%20Growth%20Defects%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487835/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487835" id="node487835--3" data-pisa="ggg;8/1/239" data-pisa-master="ggg;g3.117.300392" data-apath="/ggg/8/1/239.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Natural Variation in <em>SER1</em> and <em>ENA6</em> Underlie Condition-Specific Growth Defects in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-5186-9692" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amy Sirr</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-4511-0898" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Adrian C. Scott</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5265-1970" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gareth A. Cromie</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-9682-7851" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Catherine L. Ludlow</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-8632-9629" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vida Ahyong</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-9299-8562" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Trey S. Morgan</span>, <span class="highwire-citation-author" data-delta="6">Teresa Gilbert</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aimée M. Dudley</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">239-251;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300392 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487835--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Amy Sirr </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amy%2BSirr%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sirr%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmy%2BSirr%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5186-9692" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amy Sirr</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Adrian C. Scott </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Adrian%2BC.%2BScott%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Scott%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAdrian%2BC.%2BScott%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4511-0898" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Adrian C. Scott</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Gareth A. Cromie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gareth%2BA.%2BCromie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cromie%20GA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGareth%2BA.%2BCromie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5265-1970" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gareth A. Cromie</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Catherine L. Ludlow </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Catherine%2BL.%2BLudlow%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ludlow%20CL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACatherine%2BL.%2BLudlow%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9682-7851" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Catherine L. Ludlow</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Vida Ahyong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular and Cellular Biology, University of California, Berkeley, California 94720</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vida%2BAhyong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ahyong%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVida%2BAhyong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8632-9629" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vida Ahyong</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Trey S. Morgan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Trey%2BS.%2BMorgan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morgan%20TS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATrey%2BS.%2BMorgan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9299-8562" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Trey S. Morgan</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Teresa Gilbert </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Teresa%2BGilbert%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gilbert%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATeresa%2BGilbert%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aimée M. Dudley </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Pacific Northwest Research Institute, Seattle, Washington 98122</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aim%C3%A9e%2BM.%2BDudley%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dudley%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAim%25C3%25A9e%2BM.%2BDudley%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3644-0625" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aimée M. Dudley</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:aimee.dudley@gmail.com" class="" data-icon-position="" data-hide-link-title="0">aimee.dudley@gmail.com</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=239&amp;rft.epage=251&amp;rft.atitle=Natural%20Variation%20in%20SER1%20and%20ENA6%20Underlie%20Condition-Specific%20Growth%20Defects%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=8&amp;rft.issue=1&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Sirr%2C+Amy&amp;rft.au=Scott%2C+Adrian+C.&amp;rft.au=Cromie%2C+Gareth+A.&amp;rft.au=Ludlow%2C+Catherine+L.&amp;rft.au=Ahyong%2C+Vida&amp;rft.au=Morgan%2C+Trey+S.&amp;rft.au=Gilbert%2C+Teresa&amp;rft.au=Dudley%2C+Aim%C3%A9e+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487835/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487835/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487835/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487835/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487835/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487835/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487835/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487835/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487835/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487835/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487835/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487835/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/1/239" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/1/239.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/1/239.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/1/239.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/1/239.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
