<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/1/1" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/1/1.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/1/1.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/1/1.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/1/1.atom" />
<meta name="HW.pisa" content="ggg;8/1/1" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Reference Assembly and Annotation of the Pyrenophora teres f. teres Isolate 0-1" />
<meta name="DC.Identifier" content="10.1534/g3.117.300196" />
<meta name="DC.Date" content="2018-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Wyatt et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Pyrenophora teres f. teres , the causal agent of net form net blotch (NFNB) of barley, is a destructive pathogen in barley-growing regions throughout the world. Typical yield losses due to NFNB range from 10 to 40%; however, complete loss has been observed on highly susceptible barley lines where environmental conditions favor the pathogen. Currently, genomic resources for this economically important pathogen are limited to a fragmented draft genome assembly and annotation, with limited RNA support of the P. teres f. teres isolate 0-1. This research presents an updated 0-1 reference assembly facilitated by long-read sequencing and scaffolding with the assistance of genetic linkage maps. Additionally, genome annotation was mediated by RNAseq analysis using three infection time points and a pure culture sample, resulting in 11,541 high-confidence gene models. The 0-1 genome assembly and annotation presented here now contains the majority of the repetitive content of the genome. Analysis of the 0-1 genome revealed classic characteristics of a “two-speed” genome, being compartmentalized into GC-equilibrated and AT-rich compartments. The assembly of repetitive AT-rich regions will be important for future investigation of genes known as effectors, which often reside in close proximity to repetitive regions. These effectors are responsible for manipulation of the host defense during infection. This updated P. teres f. teres isolate 0-1 reference genome assembly and annotation provides a robust resource for the examination of the barley– P. teres f. teres host–pathogen coevolution." />
<meta name="DC.Contributor" content="Nathan A. Wyatt" />
<meta name="DC.Contributor" content="Jonathan K. Richards" />
<meta name="DC.Contributor" content="Robert S. Brueggeman" />
<meta name="DC.Contributor" content="Timothy L. Friesen" />
<meta name="article:published_time" content="2018-01-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="Reference Assembly and Annotation of the Pyrenophora teres f. teres Isolate 0-1" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;&lt;i&gt;Pyrenophora teres&lt;/i&gt; f. &lt;i&gt;teres&lt;/i&gt;, the causal agent of net form net blotch (NFNB) of barley, is a destructive pathogen in barley-growing regions throughout the world. Typical yield losses due to NFNB range from 10 to 40%; however, complete loss has been observed on highly susceptible barley lines where environmental conditions favor the pathogen. Currently, genomic resources for this economically important pathogen are limited to a fragmented draft genome assembly and annotation, with limited RNA support of the &lt;i&gt;P. teres&lt;/i&gt; f. &lt;i&gt;teres&lt;/i&gt; isolate 0-1. This research presents an updated 0-1 reference assembly facilitated by long-read sequencing and scaffolding with the assistance of genetic linkage maps. Additionally, genome annotation was mediated by RNAseq analysis using three infection time points and a pure culture sample, resulting in 11,541 high-confidence gene models. The 0-1 genome assembly and annotation presented here now contains the majority of the repetitive content of the genome. Analysis of the 0-1 genome revealed classic characteristics of a “two-speed” genome, being compartmentalized into GC-equilibrated and AT-rich compartments. The assembly of repetitive AT-rich regions will be important for future investigation of genes known as effectors, which often reside in close proximity to repetitive regions. These effectors are responsible for manipulation of the host defense during infection. This updated &lt;i&gt;P. teres&lt;/i&gt; f. &lt;i&gt;teres&lt;/i&gt; isolate 0-1 reference genome assembly and annotation provides a robust resource for the examination of the barley–&lt;i&gt;P. teres&lt;/i&gt; f. &lt;i&gt;teres&lt;/i&gt; host–pathogen coevolution.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="ggg;8/1/1" />
<meta name="citation_id" content="8/1/1" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/1/1" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/1/1.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/1/1.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/1/1.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300196" />
<meta name="citation_pmid" content="29167271" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="1" />
<meta name="citation_lastpage" content="8" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Nathan A. Wyatt" />
<meta name="citation_author_institution" content="Genomics and Bioinformatics Program, North Dakota State University, Fargo, North Dakota 58102" />
<meta name="citation_author_institution" content="Department of Plant Pathology, North Dakota State University, Fargo, North Dakota 58102" />
<meta name="citation_author" content="Jonathan K. Richards" />
<meta name="citation_author_institution" content="Department of Plant Pathology, North Dakota State University, Fargo, North Dakota 58102" />
<meta name="citation_author" content="Robert S. Brueggeman" />
<meta name="citation_author_institution" content="Genomics and Bioinformatics Program, North Dakota State University, Fargo, North Dakota 58102" />
<meta name="citation_author_institution" content="Department of Plant Pathology, North Dakota State University, Fargo, North Dakota 58102" />
<meta name="citation_author" content="Timothy L. Friesen" />
<meta name="citation_author_institution" content="Genomics and Bioinformatics Program, North Dakota State University, Fargo, North Dakota 58102" />
<meta name="citation_author_institution" content="Department of Plant Pathology, North Dakota State University, Fargo, North Dakota 58102" />
<meta name="citation_author_institution" content="Cereal Crops Research Unit, Red River Valley Agricultural Research Center, United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Fargo, North Dakota 58102" />
<meta name="citation_author_email" content="timothy.friesen@ars.usda.gov" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5634-2200" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. Abu Qamar;citation_author=Z. H. Liu;citation_author=J. D. Faris;citation_author=S. Chao;citation_author=M. C. Edwards;citation_title=A region of barley chromosome 6H harbors multiple major genes associated with net type net blotch resistance.;citation_pages=1261-1270;citation_volume=117;citation_year=2008;citation_issue=8;citation_pmid=18712341;citation_doi=10.1007/s00122-008-0860-x" />
<meta name="citation_reference" content="citation_title=FastQC: A Quality Control Tool for High Throughput Sequence Data;citation_author=S. Andrews;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Mob. DNA;citation_author=W. Bao;citation_author=K. K. Kojima;citation_author=O. Kohany;citation_title=2015 Repbase update, a database of repetitive elements in eukaryotic genomes.;citation_pages=11;citation_volume=6;citation_year=2015;citation_pmid=26045719;citation_doi=10.1186/s13100-015-0041-9" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=S. A. Carlsen;citation_author=A. Neupane;citation_author=N. A. Wyatt;citation_author=J. K. Richards;citation_author=J. D. Faris;citation_title=Characterizing the Pyrenophora teres f. maculata–barley interaction using pathogen genetics.;citation_volume=7;citation_year=2017;citation_pmid=28659291;citation_doi=10.1534/g3.117.043265" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. T. Chisholm;citation_author=G. Coaker;citation_author=B. Day;citation_author=B. J. Staskawicz;citation_title=Host-microbe interactions: shaping the evolution of the plant immune response.;citation_pages=803-814;citation_volume=124;citation_year=2006;citation_issue=4;citation_pmid=16497589;citation_doi=10.1016/j.cell.2006.02.008" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. A. Dean;citation_author=N. J. Talbot;citation_author=D. J. Ebbole;citation_author=M. L. Farman;citation_author=T. K. Mitchell;citation_title=The genome sequence of the rice blast fungus Magnaporthe grisea.;citation_pages=980-986;citation_volume=434;citation_year=2005;citation_issue=7036;citation_pmid=15846337;citation_doi=10.1038/nature03449" />
<meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Bioinformatics;citation_author=A. L. Delcher;citation_author=S. L. Salzberg;citation_author=A. M. Phillippy;citation_title=Using MUMmer to identify similar regions in large sequence sets.;citation_pages=Unit103;citation_volume=Chapter 10;citation_year=2003;citation_pmid=18428693" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=M. Derbyshire;citation_author=M. Denton-Giles;citation_author=D. Hegedus;citation_author=S. Seifbarghy;citation_author=J. Rollins;citation_title=The complete genome sequence of the phytopathogenic fungus Sclerotinia sclerotiorum reveals insights into the genome architecture of broad host range pathogens.;citation_pages=593-618;citation_volume=9;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Genet. Dev.;citation_author=S. Dong;citation_author=S. Raffaele;citation_author=S. Kamoun;citation_title=The two-speed genomes of filamentous pathogens: waltz with plants.;citation_pages=57-65;citation_volume=35;citation_year=2015;citation_pmid=26451981;citation_doi=10.1016/j.gde.2015.09.001" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=R. Ekblom;citation_author=J. B. Wolf;citation_title=A field guide to whole‐genome sequencing, assembly and annotation.;citation_pages=1026-1042;citation_volume=7;citation_year=2014;citation_pmid=25553065;citation_doi=10.1111/eva.12178" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. R. Ellwood;citation_author=Z. Liu;citation_author=R. A. Syme;citation_author=Z. Lai;citation_author=J. K. Hane;citation_title=A first genome assembly of the barley fungal pathogen Pyrenophora teres f. teres.;citation_pages=R109-R109;citation_volume=11;citation_year=2010;citation_issue=11;citation_pmid=21067574" />
<meta name="citation_reference" content="citation_journal_title=MBio;citation_author=L. Faino;citation_author=M. F. Seidl;citation_author=E. Datema;citation_author=G. C. van den Berg;citation_author=A. Janssen;citation_title=Single-molecule real-time sequencing combined with optical mapping yields completely finished fungal genome.;citation_pages=e00936-e15;citation_volume=6;citation_year=2015;citation_pmid=26286689" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Faino;citation_author=M. F. Seidl;citation_author=X. Shi-Kunne;citation_author=M. Pauper;citation_author=G. C. van den Berg;citation_title=Transposons passively and actively contribute to evolution of the two-speed genome of a fungal pathogen.;citation_volume=26;citation_year=2016;citation_pmid=27325116;citation_doi=10.1101/gr.204974.116" />
<meta name="citation_reference" content="citation_journal_title=Microbiol. Mol. Biol. Rev.;citation_author=M. Franceschetti;citation_author=A. Maqbool;citation_author=M. J. Jiménez-Dalmaroni;citation_author=H. G. Pennington;citation_author=S. Kamoun;citation_title=Effectors of filamentous plant pathogens: commonalities amid diversity.;citation_pages=e00066-e16;citation_volume=81;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. E. Galagan;citation_author=S. E. Calvo;citation_author=K. A. Borkovich;citation_author=E. U. Selker;citation_author=N. D. Read;citation_title=The genome sequence of the filamentous fungus Neurospora crassa.;citation_pages=859-868;citation_volume=422;citation_year=2003;citation_issue=6934;citation_pmid=12712197;citation_doi=10.1038/nature01554" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=S. Goodwin;citation_author=J. D. McPherson;citation_author=W. R. McCombie;citation_title=Coming of age: ten years of next-generation sequencing technologies.;citation_pages=333-351;citation_volume=17;citation_year=2016;citation_pmid=27184599;citation_doi=10.1038/nrg.2016.49" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=J. K. Hane;citation_author=R. P. Oliver;citation_title=RIPCAL: a tool for alignment-based analysis of repeat-induced point mutations in fungal genomic sequences.;citation_pages=478-478;citation_volume=9;citation_year=2008;citation_pmid=19014496;citation_doi=10.1186/1471-2105-9-478" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=C. Holt;citation_author=M. Yandell;citation_title=MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.;citation_pages=491-491;citation_volume=12;citation_year=2011;citation_pmid=22192575;citation_doi=10.1186/1471-2105-12-491" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=A. Idnurm;citation_author=B. J. Howlett;citation_title=Analysis of loss of pathogenicity mutants reveals that repeat-induced point mutations can occur in the Dothideomycete Leptosphaeria maculans.;citation_pages=31-37;citation_volume=39;citation_year=2003;citation_issue=1;citation_pmid=12742061;citation_doi=10.1016/S1087-1845(02)00588-1" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. D. Jones;citation_author=J. L. Dangl;citation_title=The plant immune system.;citation_pages=323-329;citation_volume=444;citation_year=2006;citation_issue=7117;citation_pmid=17108957;citation_doi=10.1038/nature05286" />
<meta name="citation_reference" content="citation_title=Fungal Genet. Biol.;citation_author=V. M. Koladia;citation_author=J. K. Richards;citation_author=N. A. Wyatt;citation_author=J. D. Faris;citation_author=R. S. Brueggeman;citation_title=Genetic analysis of virulence in the Pyrenophora teres f. teres population BB25× FGOH04Ptt-21.;citation_pages=12-19;citation_volume=107;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=V. M. Koladia;citation_author=J. D. Faris;citation_author=J. K. Richards;citation_author=R. S. Brueggeman;citation_author=S. Chao;citation_title=Genetic analysis of net form net blotch resistance in barley lines CIho5791 and Tifang against a global collection of P. teres f. teres isolates.;citation_pages=163-173;citation_volume=130;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Genome Res;citation_author=S. Koren;citation_author=B. P. Walenz;citation_author=K. Berlin;citation_author=J. R. Miller;citation_author=N. H. Bergman;citation_title=Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation.;citation_volume=27;citation_year=2017;citation_pmid=28298431;citation_doi=10.1101/gr.215087.116" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=I. Korf;citation_title=Gene finding in novel genomes.;citation_pages=59-59;citation_volume=5;citation_year=2004;citation_pmid=15144565;citation_doi=10.1186/1471-2105-5-59" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=Z. Lai;citation_author=J. D. Faris;citation_author=J. J. Weiland;citation_author=B. J. Steffenson;citation_author=T. L. Friesen;citation_title=Genetic mapping of Pyrenophora teres f. teres genes conferring avirulence on barley.;citation_pages=323-329;citation_volume=44;citation_year=2007;citation_issue=5;citation_pmid=17224286" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant Pathol.;citation_author=B. Laurent;citation_author=C. Palaiokostas;citation_author=C. Spataro;citation_author=M. Moinard;citation_author=E. Zehraoui;citation_title=High‐resolution mapping of the recombination landscape of the phytopathogen Fusarium graminearum suggests two‐speed genome evolution.;citation_year=2017;citation_doi=10.1111/mpp.12524" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Plant Pathol;citation_author=Z. Liu;citation_author=S. R. Ellwood;citation_author=R. P. Oliver;citation_author=T. L. Friesen;citation_title=Pyrenophora teres: profile of an increasingly damaging barley pathogen.;citation_pages=1-19;citation_volume=12;citation_year=2011;citation_issue=1;citation_pmid=21118345;citation_doi=10.1111/j.1364-3703.2010.00649.x" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant Pathol.;citation_author=Z. Liu;citation_author=D. J. Holmes;citation_author=J. D. Faris;citation_author=S. Chao;citation_author=R. S. Brueggeman;citation_title=Necrotrophic effector triggered susceptibility (NETS) underlies the barley–Pyrenophora teres f. teres interaction specific to chromosome 6H.;citation_pages=188-200;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=Z. H. Liu;citation_author=J. D. Faris;citation_author=M. C. Edwards;citation_author=T. L. Friesen;citation_title=Development of expressed sequence tags (EST)-based markers for genomic analysis of a barley 6H region harboring multiple net form net blotch resistance genes.;citation_pages=41-52;citation_volume=3;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Plant Biol.;citation_author=L. Lo Presti;citation_author=D. Lanver;citation_author=G. Schweizer;citation_author=S. Tanaka;citation_author=L. Liang;citation_title=Fungal effectors and plant susceptibility.;citation_pages=513-545;citation_volume=66;citation_year=2015;citation_pmid=25923844;citation_doi=10.1146/annurev-arplant-043014-114623" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=M. Lorieux;citation_title=MapDisto: fast and efficient computation of genetic linkage maps.;citation_pages=1231-1235;citation_volume=30;citation_year=2012;citation_doi=10.1007/s11032-012-9706-y" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=V. A. Manning;citation_author=I. Pandelova;citation_author=B. Dhillon;citation_author=L. J. Wilhelm;citation_author=S. B. Goodwin;citation_title=Comparative genomics of a plant-pathogenic fungus, Pyrenophora tritici-repentis, reveals transduplication and the impact of repeat elements on pathogenicity and population divergence.;citation_pages=41-63;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=J. Prod. Agric.;citation_author=D. E. Mathre;citation_author=G. D. Kushnak;citation_author=J. M. Martin;citation_author=W. E. Grey;citation_author=R. H. Johnston;citation_title=Effect of residue management on barley production in the presence of net blotch disease.;citation_pages=323-326;citation_volume=10;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=M. Pertea;citation_author=D. Kim;citation_author=G. M. Pertea;citation_author=J. T. Leek;citation_author=S. L. Salzberg;citation_title=Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie and Ballgown.;citation_pages=1650-1667;citation_volume=11;citation_year=2016;citation_pmid=27560171;citation_doi=10.1038/nprot.2016.095" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=T. N. Petersen;citation_author=S. Brunak;citation_author=G. von Heijne;citation_author=H. Nielsen;citation_title=SignalP 4.0: discriminating signal peptides from transmembrane regions.;citation_pages=785-786;citation_volume=8;citation_year=2011;citation_issue=10;citation_pmid=21959131;citation_doi=10.1038/nmeth.1701" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. R. Quinlan;citation_author=I. M. Hall;citation_title=BEDTools: a flexible suite of utilities for comparing genomic features.;citation_volume=26;citation_year=2010;citation_pmid=20110278;citation_doi=10.1093/bioinformatics/btq033" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Raffaele;citation_author=R. A. Farrer;citation_author=L. M. Cano;citation_author=D. J. Studholme;citation_author=D. MacLean;citation_title=Genome Evolution Following Host Jumps in the Irish Potato Famine Pathogen Lineage;citation_pages=1540-1543;citation_volume=330;citation_year=2010;citation_issue=6010;citation_pmid=21148391;citation_doi=10.1126/science.1193070" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. K. Richards;citation_author=T. L. Friesen;citation_author=R. S. Brueggeman;citation_title=Association mapping utilizing diverse barley lines reveals net form net blotch seedling resistance/susceptibility loci.;citation_pages=915-927;citation_volume=130;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=T. Rouxel;citation_author=M. H. Balesdent;citation_title=Life, death and rebirth of avirulence effectors in a fungal pathogen of Brassica crops, Leptosphaeria maculans.;citation_pages=526-532;citation_volume=214;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Biol.;citation_author=R. A. Shjerve;citation_author=J. D. Faris;citation_author=R. S. Brueggeman;citation_author=C. Yan;citation_author=Y. Zhu;citation_title=Evaluation of a Pyrenophora teres f. teres mapping population reveals multiple independent interactions with a region of barley chromosome 6H.;citation_pages=104-112;citation_volume=70;citation_year=2014;citation_pmid=25093269;citation_doi=10.1016/j.fgb.2014.07.012" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_title=Gene Silencing in Higher Plants and Related Phenomena in Other Eukaryotes;citation_title=Genetic and epigenetic inactivation of repetitive sequences in Neurospora crassa: RIP, DNA methylation, and quelling;citation_author=M. J. Singer;citation_author=E. U. Selker;citation_pages=165-177;citation_year=1995" />
<meta name="citation_reference" content="citation_author=A. F. A. Smit;citation_author=R. Hubley;citation_year=2008–2015" />
<meta name="citation_reference" content="citation_author=A. F. A. Smit;citation_author=R. Hubley;citation_author=P. Green;citation_year=2013–2015" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=J. Sperschneider;citation_author=D. M. Gardiner;citation_author=P. N. Dodds;citation_author=F. Tini;citation_author=L. Covarelli;citation_title=EffectorP: predicting fungal effector proteins from secretomes using machine learning.;citation_pages=743-761;citation_volume=210;citation_year=2016;citation_pmid=26680733;citation_doi=10.1111/nph.13794" />
<meta name="citation_reference" content="citation_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Stanke;citation_author=B. Morgenstern;citation_title=AUGUSTUS: a web server for gene prediction in eukaryotes that allows user-defined constraints;citation_pages=W465-W467;citation_volume=33;citation_year=2005;citation_issue=suppl_2;citation_pmid=15980513;citation_doi=10.1093/nar/gki458" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=R. A. Syme;citation_author=K. C. Tan;citation_author=J. K. Hane;citation_author=K. Dodhia;citation_author=T. Stoll;citation_title=Comprehensive annotation of the Parastagonospora nodorum reference genome using next-generation genomics, transcriptomics and proteogenomics.;citation_pages=e0147221;citation_volume=11;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=H. Tang;citation_author=X. Zhang;citation_author=C. Miao;citation_author=J. Zhang;citation_author=R. Ming;citation_title=ALLMAPS: robust scaffold ordering based on multiple maps.;citation_pages=3;citation_volume=16;citation_year=2015;citation_pmid=25583564;citation_doi=10.1186/s13059-014-0573-1" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=V. Ter-Hovhannisyan;citation_author=A. Lomsadze;citation_author=Y. O. Chernoff;citation_author=M. Borodovsky;citation_title=Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training.;citation_volume=18;citation_year=2008;citation_pmid=18757608;citation_doi=10.1101/gr.081612.108" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=A. C. Testa;citation_author=R. P. Oliver;citation_author=J. K. Hane;citation_title=OcculterCut: a comprehensive survey of AT-rich regions in fungal genomes.;citation_volume=8;citation_year=2016;citation_pmid=27289099;citation_doi=10.1093/gbe/evw121" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Biol.;citation_author=B. P. Thomma;citation_author=M. F. Seidl;citation_author=X. Shi-Kunne;citation_author=D. E. Cook;citation_author=M. D. Bolton;citation_title=Mind the gap; seven reasons to close fragmented genome assemblies.;citation_pages=24-30;citation_volume=90;citation_year=2016;citation_doi=10.1016/j.fgb.2015.08.010" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant Pathol.;citation_author=J. A. Van Kan;citation_author=J. H. Stassen;citation_author=A. Mosbach;citation_author=T. A. Van Der Lee;citation_author=L. Faino;citation_title=A gapless genome sequence of the fungus Botrytis cinerea.;citation_pages=75-89;citation_volume=18;citation_year=2017;citation_doi=10.1111/mpp.12384" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=B. J. Walker;citation_author=T. Abeel;citation_author=T. Shea;citation_author=M. Priest;citation_author=A. Abouelliel;citation_title=Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement.;citation_pages=e112963;citation_volume=9;citation_year=2014;citation_pmid=25409509;citation_doi=10.1371/journal.pone.0112963" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=J. J. Weiland;citation_author=B. J. Steffenson;citation_author=R. D. Cartwright;citation_author=R. K. Webster;citation_title=Identification of Molecular Genetic Markers in Pyrenophora teres f. teres Associated with Low Virulence on 'Harbin' Barley.;citation_pages=176-181;citation_volume=89;citation_year=1999;citation_issue=2;citation_pmid=18944793;citation_doi=10.1094/PHYTO.1999.89.2.176" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Reference Assembly and Annotation of the Pyrenophora teres f. teres Isolate 0-1" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="twitter:description" content="Pyrenophora teres f. teres , the causal agent of net form net blotch (NFNB) of barley, is a destructive pathogen in barley-growing regions throughout the world. Typical yield losses due to NFNB range from 10 to 40%; however, complete loss has been observed on highly susceptible barley lines where environmental conditions favor the pathogen. Currently, genomic resources for this economically important pathogen are limited to a fragmented draft genome assembly and annotation, with limited RNA support of the P. teres f. teres isolate 0-1. This research presents an updated 0-1 reference assembly facilitated by long-read sequencing and scaffolding with the assistance of genetic linkage maps. Additionally, genome annotation was mediated by RNAseq analysis using three infection time points and a pure culture sample, resulting in 11,541 high-confidence gene models. The 0-1 genome assembly and annotation presented here now contains the majority of the repetitive content of the genome. Analysis of the 0-1 genome revealed classic characteristics of a “two-speed” genome, being compartmentalized into GC-equilibrated and AT-rich compartments. The assembly of repetitive AT-rich regions will be important for future investigation of genes known as effectors, which often reside in close proximity to repetitive regions. These effectors are responsible for manipulation of the host defense during infection. This updated P. teres f. teres isolate 0-1 reference genome assembly and annotation provides a robust resource for the examination of the barley– P. teres f. teres host–pathogen coevolution." />
<meta name="og-title" property="og:title" content="Reference Assembly and Annotation of the Pyrenophora teres f. teres Isolate 0-1" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/1/1" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Pyrenophora teres f. teres , the causal agent of net form net blotch (NFNB) of barley, is a destructive pathogen in barley-growing regions throughout the world. Typical yield losses due to NFNB range from 10 to 40%; however, complete loss has been observed on highly susceptible barley lines where environmental conditions favor the pathogen. Currently, genomic resources for this economically important pathogen are limited to a fragmented draft genome assembly and annotation, with limited RNA support of the P. teres f. teres isolate 0-1. This research presents an updated 0-1 reference assembly facilitated by long-read sequencing and scaffolding with the assistance of genetic linkage maps. Additionally, genome annotation was mediated by RNAseq analysis using three infection time points and a pure culture sample, resulting in 11,541 high-confidence gene models. The 0-1 genome assembly and annotation presented here now contains the majority of the repetitive content of the genome. Analysis of the 0-1 genome revealed classic characteristics of a “two-speed” genome, being compartmentalized into GC-equilibrated and AT-rich compartments. The assembly of repetitive AT-rich regions will be important for future investigation of genes known as effectors, which often reside in close proximity to repetitive regions. These effectors are responsible for manipulation of the host defense during infection. This updated P. teres f. teres isolate 0-1 reference genome assembly and annotation provides a robust resource for the examination of the barley– P. teres f. teres host–pathogen coevolution." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
    <title>Reference Assembly and Annotation of the Pyrenophora teres f. teres Isolate 0-1 | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"TGdu0O1XKKpjD8F6AGBG6E3oJDXCPtkc_MvDfZSp88Q","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487831","apath":"\/ggg\/8\/1\/1.atom","pisa":"ggg;8\/1\/1","ac":{"\/ggg\/8\/1\/1.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/1\/1.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/1\/1"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/1\/1"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/1\/1":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022u0m\\\\\\\/mglA4KpNDF8EUiFcaT+Q4E8p2LN4DWKQ5GB4ZLEkoqqIxsYamV8Ei07QSAiJRXVNt7r826RG\\\\\\\/GCcJMwx\\\\\\\/b+LjGiu8MgF1KhxITKXSa3aiJgo3E0YX3hqT4SInp7OpqbnDwW7uwlYsc384mimaoHtjquQW0OXJb0faDmmNZ4uyzdZIoQAoB9t8MSAqQJMU3oPaLtTQwwQdg+r0d75INrrSC6pDpmYHk+WQFO4Vjj+UDS7eBwkNP3AJJ6Jx\\\\\\\/Hqoh4qt+4cWbhd+KogzazLGDn8a6oJpxP5y2R1iC7Qhq5PpapWje6znCuCFy\\\\\\\/0KJObyXqpEJHaOVVM+QI3vKoJKFYl7T9lCVk7r7JXPBP+dXy2vwm0Ybe75I5PDezOISsO1YhEJ2ldwyrYsM7YGlV5VY8V35ZTMAQKrPwdbqDBiJnd0r1fw10bO6gEHMjEkcJE47Oot+UmuH9Cm4WlIlpjgkljawJJ7xY+mRL6fX9M\\\\\\\/lrtAcMcCZrh7pNO5UcvYy149xkB0czlaxXgluKgQ8cPF+MDDuOUmEgeYDBJTht3uDffb74qpt2P14XPcUUYzPBMHlhb2NXnRXyYiahudGcCbnywv9A5h\\\\\\\/cKMTJxUgbYTN0+dzCp1vhZO7ZYk8YODDRgult0o6kak683v\\\\\\\/VyIDqQXFU0QjLDN7JHnEeebtYGysZ6Lj2xkkBKLmsepPyiTbSPHzWgoWD1y7r9mfY7DieK1Iati+c2N6poo4PX6wcVXr2eRpMqlLTiFBaEWPFMeWxXodCiymgZpmvfJwNpP9EeGejFhTirk9kPrxvk0dz3kxT3ihF+DemtDfXLry6mN2SbGr4R6mo075UfGGXU1XSTI0+smxWYkE\\\\\\\/QosMQ6+7VdrIEvWnKcY5U4xXLKz1ov4LJyhe5rXH0XNbW+SLh+ZAxypqOLZIEFWZICwhB+CXhfB2OkxklakEo3HG3uwZj\\\\\\\/+25EXBRkx4YRl\\\\\\\/jR9E63a23pkAVDxVhYQJfJfUxnzwf6kS4V1G1P+O5tMd41fTXeaspPmXbnBGqhWYEfvKF0AnCz\\\\\\\/qAtqh7iTufHc1p3CLHz3TEOXB2wYyeD7fszJkkINCDZflYSuFz68rvF8u9p9TcfSI9O\\\\\\\/TBhW1nKuPV0J0HdAOCUvGpNb6zZUUMLLVSGtl7O4YKaz8oTmA9VYrJ26eZiVr6\\\\\\\/qs2ghGGqjDGWKJGHYLCMal52vXqqOMPS4LstOqB\\\\\\\/PGs5Li3rV6IBPNQkAteeGTQjir8yTE4tlREfBF1T0unfTl88aj1vV586RhPlYZx\\\u0022,\\\u0022iv\\\u0022:\\\u0022r4N0RzIButxAejv1SdHQbg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221fa759307845f2f16020878d6b3c71b3324c327fa777743a4f5324f0c8d1f1d3\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022gcX5N3Zla\\\\\\\/DCroPjcwArpm6djmL+iuhNSexr1yPyN9b1rsPVylmgTgvvwH5HpOMuTCfWgS8QuhyEu+ggxbSx5PnvHSj\\\\\\\/jCzR0LXygCXBXwVprk3ZWq74pxBz+dher70bNMOLv5C\\\\\\\/KESsezEKIv9K8ffqgoWwhRY3qE1UX9Ml5IBO5CfoRcVkGhdFVl5QIa60BIp1MU7Pt88WkkV7t3wX5YyfSTy+9q1exnERG2EJ2D\\\\\\\/\\\\\\\/g5\\\\\\\/0SmdKauRWVyXEGwvfZxJWYnXjgf8WbNdP2hM7a7R2GyDRfKlDVNBtf\\\\\\\/BM6ZW2ogm+gFEUBsaf6VMTJqKDzn8Bb4UfPYcNgqV37TWBM20BMrnpTpzGuTEW1+8TT2y18vCoUk\\\\\\\/Py+\\\\\\\/nWZVGApug2jCAsNb0I+nlx7HGdVd6skKTRpPcZF2g\\\\\\\/Bzzfy1Nn62GXH24fnd6EsL6\\\\\\\/AW8jJEeva92NiDNnCO\\\\\\\/vbSNqXTUEErdQ+2hfA7U3Wxm0zJ829FWWT9m3KCKn0tjADEyul4zV2bl5c9Ag4MY7+ktgFFNkiQ7E68WOKdgB7Gtm+AoefqjCRVdTJUJOn65Y7JVqMV64WkuKObyXpMoGs+pgZkY6\\\\\\\/to2DF+ecdQw4ZzgP4dT90sSTiHWYcueU6fQ6inKh\\\\\\\/JOl9tk4AEPkYIZCXhY\\\\\\\/aJjPN3Izf6AfGvjZRL34aHLNIq6EhCYrixfw9MWPIm9XPCJyhY\\\\\\\/Ib0ZW45e74Xv0EPdS843Q6vkqbs\\\\\\\/ztJcYmEXWGDKe1oNt4GNSjV8VWe7NoLsXEr3QNyeeldDDnePzCtrg0vpYiS5BZ+m32xuiKdlgS8Sh+\\\\\\\/eU\\\\\\\/6NLdFq2nA5gZCvRoDqgBmJpQhn+ilrOMvnpPOQ0eSGE3ZtEwAco4NORCgGsXmNsMhPXNR2Lr77qFXLathX26faLgNXMLi\\\\\\\/OlAbYBISUMjyZ\\\\\\\/raBvkAoimTBFk9d820D238AWyfKlyQ+NwBpXq6LD43rfntU5M3j9Ko8qkUyl6i\\\\\\\/\\\\\\\/ccScsuPQXoGPm2Br+zFDY8NTzfuyOOmdsyFGBLdH5qIfxnOC7NkFryxge3VjmvLOWbll\\\\\\\/aMlFJSH3sZVEqAPgWmzpt\\\\\\\/Cjeu3\\\\\\\/7pv0qE72uFYhIkhfE7WVNYyE3InYRuNYqXWCCemUlHHOo0qaHL106+CxKZvPowvgbOYh4uWqqtwsAuWaWwe4jvZ7aWn2XNS7Zu7Q+wk6E0Umc3l+KvZnEkFHc0gE0H2hSuJoh4K8rKrv3USdiMBE4nnysxtxu0YL2cyjTpkT0DpUdELpsFmHfVzJ5SaSIjS5Mjb39rVhRlx41FbRQ9rXWQdQRRimBQEGToK+3Lpg9ncpTUisdkUJa81utRCXfiQnd1k2igovTBVcZ7\\\\\\\/BS4mYIKnd+5MCUQuA3kl\\\\\\\/a46Ldfts6xphQiWlBbHFAdrYxtnHD5c+Ef5E7ZTTNKj5ii6\\\\\\\/+mP\\\\\\\/eM4e9kLm06pmPQ2AwH3mgOQcnuTijESBp4x1CcdRZR5iRXWT2LidaxFKLg9stlqwxqSjIavYFuyBsOxW\\\\\\\/4ie2KJpkjPXW9ZnD20M\\\\\\\/mqTNeuprB7S4UYNVCItZwnqfyFOrC4qXHCTWKEtp7f+hn9ASCN7wjNQE5GYN1ZnQP4tNV5lCC8lCAbjBuSxbbvU4G51GjPIhvrkv5IYSaEal0vG0LshjlQgtuX1l22knFjpr6\\\\\\\/6HEK+2k8\\\\\\\/ezziViLtAxrQg3TD5GWRvElVepdDxzY+WqR+VCepts6j\\\\\\\/7bKfpcGbGQOAsUZImS9PSDFEaDP08AZC1UTDKaWj2VF3TjhS\\\\\\\/A22Ray3YYMMHUBjobpxCF6hDysy9rUNWB29dKUI6TbSB548FTKkqvWgWF6RtrJRCSyCclQkkiY7uCxGMftKKMAhVPKN0\\\\\\\/pdyksWfSAEpipTRvkoFlzvUvJ1EMa+u17wTgEGVlZLa34YAoC68zejZtkYp4FkwO81MuCYvkuBUJJKEEIb9t9yvt1UT8mNEVyk5ZhnNWTyUjVx5ljkwnOgQ2574PTmUKGl67m1gWYAKHIup65horYKXVzHfpvTcQJdw2MLXC7WUrTpkIegR\\\\\\\/fxFKrC7DWSrKD8PyCaqrw3u5oSDNhFhrbu\\\\\\\/kSrBn6FkC7ouY7XDQC0Y5mdmb\\\\\\\/AvdQVgQoFyFU8UXVqtG5vRgWemrTQ0IQIHRg0F6pq1XsCaYem8cdx5krTFCVmcfvE5d5b2rXBfF7S5s8XYwN5GGOiFM1XArtfYolvdlSSGNZavEHjKhKefBDmlwJTk+LeSlJRraJRv2zMqrHBinrDff9tebI4lJ\\\\\\\/HbjC\\\\\\\/7wvLV02uSio1DcJghnGEJV3JMajbMz5gs3iJL3gpwtCcXii5BCi5CKMZTVSE4+2j1oXzMFPe+m5ZNAXxuPdbxoofVu7RO1owab9x33pqJ9CzkSJ40uOWehTv1Kkig4rh3Mci38ocQmFJedBSvQWZRhgMGRlWoObIw329i\\\\\\\/uOU\\\\\\\/Ira6N+a4NN70GOjiBrnzY7WiMGvYUeo1T1TQ1L9AK+tIWthXOoYWHq5cPzb\\\\\\\/KEAS3q5CtI98lrzAz+JuToL\\\\\\\/WECyOgUOwdu8EYW3lmNSVuG8d1EiT+2nR8KAapl61ypRmHKpwwDzdLW0zt+7ng2tixKqmzdV6hL3Atu+H1kcU2P4TYhjLTrdB7k+joTUve1UKayAJNDnpQzvn\\\\\\\/+NJ9bouewOb0LsPVUdPFy1OOHKLFTBUURVXvuJiQhd4m03IOq2CtiBGTwzJVyfIXzbck+m+sG6gFMO7VwAdM5blZdHkF00usEt7TzzA4mAzwB8g7wbQh\\\\\\\/d6oMHjzfpMgUqAI2icF\\\\\\\/k+BNd\\\\\\\/8hKV9O625NtWS4BGVle6HAbuetZ0+lTfbURtECjspmwjUTQGNJ0LiG7o7JJ0yL6bFGqZ1oqxn1ZhqnFJJ0fynZX2qZ1exY7mv5krKnH7NAvYq7S2QpzU5c0kC4F0BkcVgM1VPVzOwbvgI9ziQ7ejJE0L15Gb7kZHmCB4pzGcuyxH+v4gaGtarzuffwFhWHw4LuB73lFudz\\\\\\\/YCTLjO7OnV2l2T4\\\\\\\/6J9nW6i6v93dY1yJv\\\\\\\/+t4x2uy\\\\\\\/F6kADUFCZlc2zFwj\\\\\\\/+G+1tnFu7K3Qok9OEVRche1UWlADN1ky\\\\\\\/UzTzDtsFXbiCr8gHPgXKxkrShjNKbIhAFjnWivYorE8mZ6LD\\\\\\\/ydC6OmB8+D7VNUm1TjuH+qJPG2t+q5s\\\\\\\/co9PhrlUk00h7fQcvgNM7MX8hsSd5eH\\\\\\\/8EoIkDxk8kpk+hdl+GQkYGupq\\\\\\\/EQR1BI7LYD7lt80L0zFKxb0rhrSAyyy6KOY4ODyywv7ue\\\\\\\/B42b8LXBu8I77O7BOudjlsFZoiSufX0s11umPvYXmsMuWREAnKE78DMlSBKvnwueR6\\\\\\\/EzTF1744VNz9DjqCqLbbHZ\\\\\\\/OJZPutq+tMvAiYWPLxhoO2dhFh1b0VHemEmkn7iFcC90MUXzJN0R6zsxex5MUYh6P3Wu6vCC3ucomPZY1PJBv9KOQ3fGU0FpjSJR4nAMXndXGZQre2SmdKyq4vojmLxR0DE5M1gYk+97GRLxZy+SS8luyL2fC6ujiE7\\\\\\\/ccEq09POkRPCUMaV2d+sQl+u29zpZFSj\\\\\\\/4wOhQR+hdgnxqmmKDdcbxxxLO6Hl11ZGjzmcj9NX3VBhaZWn8JkXLmAjGLYOBjuDfP1Is+QZF5nWuGsN9ao5SiG4kMr\\\\\\\/nyG1WxvuDO60BX\\\\\\\/1m\\\\\\\/KAUkSi\\\\\\\/51xCmzN9p56HeZGtWrnaizTDgRT\\\\\\\/SrX1ohwY5lfskhYmgCxh6b6wYyELktOP9KtnpEBS01SHDG+mVNC1WgyJtck33bp\\\\\\\/BWBoNzR1K4FDuihfRGT6DYM4aR1SApRTvhHqZurF0JGTFas6eRbwpZPdIK5Y3xt2kuM6NJIyMi8SgP8VbAySSG98xikhOuIiAmN6jyHlhaHjUk3s5MLbdhp4RTj4VVkrZ1KSDFs64ZEnBzS20ZpkNvNhHbDi27Bdniu6ln7b8jEbI5fle5ZET5GmRsfSzS86c5Xaj1gNGGFkfq+7CXnFx7QraNVe61L2I2LlKhL\\\\\\\/JbhCpzCv68ztuUWQVvgo0QE7DQ1QJ8zingmrWzxWlrbrzrIz\\\\\\\/oAc6u1\\\\\\\/9Ika5QHuDzt4B1akYD7nG4V6V\\\\\\\/\\\\\\\/+wFZKBoNDcCfQeXwbhyJdiBuaRcm\\\\\\\/pDgsHCowo103AHiOiE1FlPbSs+QqBOvV+MFQKdMLPSDBo+sxrYQs4PW7dHs2SrzFTu2X7OMC0Bqx7qGDrLeUQcPRgQX8Xi\\\\\\\/bwSU7UPK2adAxNVfAHexnJBMjS63vZULs3kJRW7moqi\\\\\\\/2nTwh+1O\\\\\\\/ew9zm9bUkKb\\\\\\\/nCprhVgr9dfqv5n1ICACsfQk27xMY1QV1rTb9mBiVFsyX24eH8i1+mNkMHyWae8S48tHgZrjDrTf7ErzwxkalWKiSUb3rYIHRg1255FuuLOPpIxgnWQ34tm6rmE8PQ+\\\\\\\/faLEQKT1+yLj4WZmip09jmQbVBjt+2Mz9CPR0pm\\\\\\\/iYc0G4kp\\\\\\\/YpqEKkkbSBcP3InOf2LRodbpHh48uioBRBmZE8CszqCShBZxWXN3HAYtvbsRY9IINAGCEUh0adVIPS0CqybSy1xuMaZHXb6lir996KCJPjnGp1PTrijpt4JNrHxW+n1\\\\\\\/ViXHwmnQQIXnFHpe0Mta875BfG1Z6XcMTdrsWP8JhrocsAKJM\\\\\\\/z3whJz3xifvLq7Z7WRU\\\\\\\/P8tUBAUc8X3HrJ27D9p8pxjlaNrL\\\\\\\/ydRxkVFF0GOmMsAO0tFkPNJZRL\\\\\\\/Cd86jdXHTkwa4tOAkqY097\\\\\\\/fOaVmqEt76P3LoD0yZytugO62WXvD5nE4NVK0qWf3oxkF8+TjpsCFtDpZ9xnmJ+ripJKrGeQc6c93CzurAcr8zmo6oYOGN\\\\\\\/Hl3z5RrDlXfo5PS6ZZZX2wgJZIz9Xc\\\\\\\/bcUzmbfmZtTaDR\\\\\\\/5cnJ9tTxgQQR4BAFGaNDIGdsGaoAbuHaNDKYhi0zuUMZMAtSJUjDy\\\\\\\/MY6T+t2avjc0gS5ZIQt6GAHCexlj5JMORSS3AiqkC9bW4HOkdJ7qp1vnWp7VCqrEUrwU8l+bEPo6C4WCfMq3nVLeI6lExDBXNEaZOG6NnGE4lXP+IGGXz5AAMAZ2NQ6vt1ezq2bfmBxOyibuZK51VB7d6m3x0ZYRgKpZDRjfuz65xCMWz1EqIMbaza7zEG79hfwqqmRVL4Hgq4b7pOevxX80K7d+bEp+uucBCrqemIIIRQ5ThBCmLw55wQ6CWShlweF9mJRew6qxcbK9YY9kKDKLeGcdM7rbwavgkX+x4ymgH6EW6Mj0wUI7jv6e6aHmW2R1f1mO4E9lnE7CqlgXYI0m4Obi51lJTpbJLVwu77K4DxS1oXSZqdKMKEmyrjfrMqze0+W2to1B\\\\\\\/itITXzIF+vJeSBr4bRTFc6cGtotA+wbjhVuo6nzVp67WCun8\\\\\\\/UtFozBgmLyLL9SUhVV1WjgjZruIf78QZYPE7sy8ecRiGIYovFKZS11R729Ur444kw7cZucf4YcFHKbdxd9wKF9XRbFjsLsqZymFOyhenubZkx95jLa7vQbF5p+IteRW4iG5+7wS3VqzMAM8MwtJxQfcxRUF651S4iRvU8CPYFRMeZK6FD8fh9J8qAX8bO0tumLzTmfYXQATq0ngR0nLg6e5EU3Lvrwt4RKnZkviXGn9+UiFXK2PfmRU1ifrUcqsDcWB7iayM30308cNLTflQmEQWPYmFR01V9CCImuTnLkQENvOcdl79UMC6kI3zo13mOfHc3qfFPmMcgjW+4EC5IuwoUzJBZT+m4rt221KP+XUarWo+zlKKelx36b8VNT3x3RjC5rBDWhQ47VOaKrKs4iVRyYbGUkCz8woJTlKYZYgi62Rb0g15qzz6L7rwA+\\\\\\\/xRlN8+ofBJqYUQVfmIFU9X2apfgYa9MrVCM8U2j2kqKwGaIF5oytTMa8UBZNdShEKQLAE9h5+1xKr7h\\\\\\\/sK63kLvtNcbUA3NGMxGBxq5AgNUg6\\\\\\\/nJu\\\\\\\/7W2z1elLeofSVMqUQLO0KYkSh1CEWdCYRvqLB2ySxuq4btVjFXrZutsLUC4CViJHP4wbNQLerNnKxHHsgG3ADwNTItLe0TrEJ4onjIuGrviy0BzuZQ1ikFGcpG0WbiSQT0+ZSmcJf9bcu8u26DPiwxbnAbMfL5E1gzwz3QRz3gY5y2c7nJZU+dlKI14v2aznnkan0ZxjsR93fRdEF9ZwtcL67QjWRToyk8+H1BNN1DG+46H+x\\\\\\\/Mfw3TF0dkRAuYdX7qXM+wB9P\\\\\\\/lW4JCecGxKuGxnRTZ7zM2Em+H6ce9Lsb9oR8iVNLHoDubb1OQLV7L7Ar1+ZRw1akH677ZpVzWEfeJ5BOqyObiedUCdS5kUrkB2j\\\\\\\/iQccHuhgE0DDa70pz163MLRxSrDLLFfE3yVQjHJqD3PS+aFAa+vxSRbB2O3PornT2bCfcRGLAM2EsCgHA2Uh0HeFCu145xIfkVEVkJTiuUR6EwuzhI8kTaf2W8G8x\\\\\\\/IkMY8VXi4udcHd231cGOLRQhKJ1ce3zoDG8fq3F9ia8lGaj8VHdTN9SSHIb3aZHYGmr332S7SJQf\\\\\\\/dxmnKok7ILHmviK964ulPiPQVDxTJm3\\\\\\\/2AwAUjKp5uFgW9NhfURNV1ZdJXRgWxK1jWfU+aCLsZLVcJfuGyq\\\\\\\/QKxOWBivZXbLUR+7U7O24qYw1PJMIIRsByw\\\\\\\/wrsaU3cWcdsb5bGJnIHQcddOpVVbvjLGtb3q7+RsKTfuqP3vqSCuimM1yGufXGu220k9ZSC7W6k8Y5EnPc0Hx8gn\\\\\\\/CDozsCXlrBRZrneU9V\\\\\\\/29zLN2Eb3CPR5c9UGgoy0y31UdixM924RmmnTZEXibMXjUCX9oaR2G4GJ+l\\\\\\\/UzY8oiw\\\\\\\/BUCc9bw1DYc4T3t1cNZqXqVtCx+BUM\\\\\\\/4QtfkR63zIhAv8KVIBS\\\\\\\/NEPKTb777LMRNYEWeRpwUZM+h1jb12IZJ2hTRO1ggJa+ypr1LPePvYUAWwAqg9NIfovAUbv\\\\\\\/vwaM3z5ICAkoICjgfsLN+eNBY1YkAsKuCFfYHjgGx4cFfH+qTXOaZ+40zGYnMOiNRlV5DL6U7N2YIoJ00o6XbLaz56f18hVRoNuBKKF1Q2JkUlfs\\\\\\\/F1QT99YnAyC+0pApZm8jrJUCabD4xXGMLGVNBaT+v3ifZ11Z2n7Z4mFDdp\\\\\\\/vmWFwOTCKEPSylabsNvvpg4ENY7fyagDL5mWvkp8r2A81p1iOTEBapLktx9u9vW1fOkxoee8RB\\\\\\\/JkZX6Po9jJ\\\\\\\/5ED\\\\\\\/0bUAtv03LqcLrOD0Y+gMBtZ15BGIYpDil9ugTKSc2icOPbshvQb863+FwetCwTYXpJwDFDxVZWPnHLdPZiETTdTQ+eoqWvT73u\\\\\\\/eVDcGwWoGMLD9+hXrBmiMrxy4MXs7bPKzH4LUT\\\\\\\/bTtT09P6ASgR0zgEWqpXG3\\\\\\\/Oqtmy+gr2R6NyOZSyx0ppqmMwJA\\\\\\\/JoqwW75p3yW8J5aT\\\\\\\/b3iVs\\\\\\\/AeF6O6La8UjKpecZd4NDxNqjzEuSJV8AdWOXnmJyS3Ph8ZxEKe6HoH4dymSA1M0IPTuy6BtfQkA\\\\\\\/kXfTB70c\\\\\\\/S\\\\\\\/ItLU1KsfGV7Jtfaur6rwhnorj9oU9LOQPnXfnOAmiwShV+9KNcZRAp6CqGJhHqxGeHYVK\\\\\\\/fh9IPQVO8+RsL551krJbv30XfB11m6ZC\\\\\\\/dn3Q5dkUOv9SSwGJi7Pg\\\\\\\/Yk4qcYOt6HaovT1oUz45sbVmKmrWYN6nUuKoSIzScxPqeh+vDfgOeoE\\\\\\\/yK4DKEMZBSgikSU7Co3DPnaomJMo0ydNMCG3Bu8PdLu0uyPD+txl49FqaUUuMQt\\\\\\\/eY6ifpdEqi3dwesPVxWOnvK9d07a7nM15n5Pjhz\\\\\\\/OOjddGRaKDp49YUPRMw8KiGX3yDauucNQogw4CBCSprlGNEzOlc3jo3\\\\\\\/GENbq1h9aV8IDunuRKohmqa+P5zdQfHQjfjk1BpA+qY+2+eSVJULWTOKqljeVX97Bh4roGIvwXCmJ5rog7YNaz+tIrAP9OVqQe7FB\\\\\\\/S3ivOJn9Vms1CgP\\\\\\\/p+19cto\\\\\\\/a51DO2X9wtbTRTYhyoNP7AF0WwZsP8REPRYk\\\\\\\/b3q+4+3B7qqFGpv8jVmjGeqsjPP4e4+mLRs9e\\\\\\\/obLUBAjakyLnneYPM\\\\\\\/7+tSxlZGBw7Kc1KwNAxbQahXe\\\\\\\/Pr8C4PBPcyPzTwh4LvEms7yOHm8\\\\\\\/HvGsDF2+iwUqQ93RdVi2mQWFHEVqyojf0wjUevCua19cqL\\\\\\\/GlJ1A3BUJPY6IDuves8MeKJCEA\\\\\\\/5GNH5IsWaUqaz1qK+5cWeeh72TXPi4JkGqDtMMMvmVgYuwqgN0gL0e+ErxNwrQlOI+ETedTFVTN2VGuaOCkyfjJpev4FwpWPaOjuPOIvdRji9vAKPXE+Q6HCqh2Ce\\\\\\\/bAhXxrISpDIoktFQERuhS4deSaYHGG99R8sKubqAHytMJkUpV8ywpo+AR1sexZPbJjfgvhifIXh79IDvjraJ3GWkLe2mVf5QAQsI8OXQ82HouhYXRRvlZ6RyPqLzQ5\\\\\\\/AhBW42OIwEqAS78C+u4Pku0rT6BUJQejkciEyDDLg57roooAO7NJJJ7c2wmGXirqaqJGzDaNOdtJqcPIlXprod9y2anDWRZnq8WOApQDYB+uVJFLlbBL\\\\\\\/LdT020C0LCD+0WK2bDP7R7vMo8Rf3nWb+aNJIhS0IQBeKEzp4uxpjPKSrtgC6aFlUQ3Gyv5rm5IBEIAk6z3oneYG6Fxltuk7Xh6ywWLZDx3oAP90nR78p++jzlhcqzklBti7G04NSsNcua2vs1JLVaZq1HbXvqqoojJQzjOiOna8UVRcrmJiNb\\\\\\\/DZkj8D5a8XcgT+kuYL97T0sVrIcYFtHvGwnXFt8TyBzbWMJXYthCHBd4kRcvtBw0ongfkZnzQYJORLn61QNjOD5wE1KJJ1STIk8UV0+dToIX\\\\\\\/gHvAFoWzGc12b7oDQubON6ro6UjO5+gUPitoTuy53KfqMaQPkylCka1LMyZOIxhiTvKc+NMiMUBRhYyrhi7ROfjd2JyNHhpbtCWs+BEJk0W\\\\\\\/EEYHyufUgdJm96YUflT14TqQCFgqveaR1yGW\\\\\\\/u3ehaKI1n8u5qx9OJAZIClK37HG5j7W\\\\\\\/4NxPUpXQzaif0t3jxw87z4kw8C+NrY1PBVUceKOoIO+4xYBTHL0276xhaWSR5pnDew3pSjyDdj13jni7YhzO7Wdus8J3kjN1rMpfq4IHqQ2oWz3ckZwUHJ4nFmMNHuwH+IMcnIXBK\\\\\\\/Vp9BoF8bKQm1Os56EgqdSryj\\\\\\\/zFXCcM\\\\\\\/pshHFWFTDFvgqlnCYJcLV6TMZliy2iQYtYlj1Nanw2M1FitqjB3aduIDGrhffrNgKvjx5XB2\\\\\\\/Zbxa3Pe6Tsf4Rfyw+5qJYLLS0UvjCdhOLyCO3tq66U3plW5JkHlSrzp9MdFIj6jCf7ticE9lOvMugJGkiLhIqAG\\\\\\\/iP3usVD4h\\\\\\\/Fl6AXAzhyDn1ehYSGZLuLqnKGLxeVPxVqhdjoeJvMl0Vcu41xGgYDXHYd1uB5h+j3+kTVBjNBOifjernpUO+rYHA1jWPnZc9yToRdbHtapJV4E2\\\\\\\/KWvHlFzwBN9QZPuKojl6DxdjgHtRzoqflDrIg8HCS4OVnruwPYcEpYEsAUKTZCCX5bIZsQRAzp7TDdTWDdXR3jzZmOUQYfxpN2+wQu00cA5iIl3ldcRC\\\\\\\/VYfi9G0mGd+lFuiEcJ9xcjTCH4gmqD0kROvgLKEdO0Sts221ybljvQSLBBRwMap3MZUUEp8wfKlElbBjLRnK6itHPu6urN9RdwS9eKL7Av93jRdZMvcWk4oLUZyDj6k9b1FbPnwBRHvAwBXDOChik9RwsKMsNVT8W767T1qjbqrua2G3SyY7nx6EbKz4MeOqSDe\\\\\\\/OPOe8aFuhXl+1GNZWIt7HJLlkRXyyBXbvXofx9Zk7lPsJdLoNBQS82sEENr1WUwnM9vXm+UdXnZ3sFWSBwMB6IHbG1Hehu4YCUOba4U0BDGD9L4p8xjJgg5rdgBjbradHGA\\\\\\\/50igBtUJtwMJ4SoiO4Xz4TkARDWB5N38FvVlN8PU4YgCxmFkjTvOZBhSZeNlv\\\\\\\/SK5BNA5NOk0GeNcvsSOw\\\\\\\/fNRxu2gS1ew1tvG7aXgOFcDBKmjDSpzVWTmiiCvSWUYkERQRc71PMOMlZcq1OV9ifFklXlL6ub7PGQTM6AW\\\\\\\/wXPSdZk8kwnwdiW15zcWerFZCZzmnXP9pDGupQSCfQjQl2itIWqF7ADZN68HhHU3ZKZ5RpSPTboInTVeq9Up+tQIZGJp\\\\\\\/8pnT80KiuJ0Wj0lA4B9R7jpGxDWlbJzGKlYLwRrOsS1FKNAE7x1ctfVB1vdo6+VKtKBecMI6C4ZaY0igVGLrH39GlC4jyV2kGScHa3BobarNLygDWwy9zA71p4l1hyLPprcWP9v8b3S2dadEFNGOBO9cl9sjFt09rCNlIh39Co+rBYFiXJQ+TiHY1iKGryG1bhhZG0GhAB7LPdU+Uxqu4NozsuSo2Sz\\\\\\\/aho6ncGGew\\\\\\\/bxUq63sycbf6jSDZbVXq0mo3v8ikDEN\\\\\\\/uGEix8JXTHJ1z0g1yVY1y1XSK8aNrpAapGYeXgkyKSx89C7pn+fG3CYCkVAST+XAhlCBadpNA5FYTBrqHxWsatYhWHohiypaAfJaIROwbGqE2XUjzcsuN2kqrZkvK38hmd5yXcRbbRJAngRXzRpFLqYUr0IUKelBeDAm1HvHUDyDAZ7kkE7vvU9+5W9Kv65jBvCVYcY72b2eT8b5gsNtRFoyngQiyZvT5P7Xcgs2uIUVnl3ZT0gohNTI2\\\\\\\/W6Y2v9eHw1Fu8iE8IgVWWWSjYP8kx7ycga2O\\\\\\\/ZlH0ubUSukhR4jyUHFTsVla4EjiuNxDUSsbc9QAPVDTWWINnpdBSBtRreNLpa\\\\\\\/0xbKCHprIDrU+H6xyQOfno\\\\\\\/bwWrugNX6w2Xf4d+LjbXs7AFE2TVGD43zj+ocgpTQXIZRp31yxDWwLrlLjevVEnPdW99HiR+t+nPISOwJO3aAICczMJYrnHE4CwWlgDH33p1wsJZjR4rJL\\\\\\\/DpZGljRd\\\\\\\/ll1utIoWMtaCynXI1Lo4lhshpyCu4fBHQPZdYrQriVKDD03NjUi4d4Psya0SXrMpLfHn+hozeHoH3X7e9BbWkKsuejF\\\\\\\/cL+sUf0vtcnQq+vHVqV9EkeLkHxE3Xv8Pn6slYuY7egFwVOIXxG9WNAkZfvrhbyRuh2jWT8ma6iHnv\\\\\\\/LaHm\\\\\\\/4\\\\\\\/L+ykjnEI3FWUnaeh+ygnclaY5FvmZCHfLA2VPKbeLv0RK4vGmItATnbNZGATg5umEWUDAV95svy3bit8mfeMvuwXlY9Hk6KtG7e+VHhH1fgZ92Ai7yReJNLFR3hxu9IXsko9kyMlsioh8JH\\\\\\\/pZMeCnBQ0nObp3ArxHFT5gJUVN8GRTGGlny2ky+1Dc2bQs4+pneYiJ0J6ZWaoVPeMmshHj5pNb1M9YRub4sWgRgY1fXELHtmIWsEA+B1tfDAlFvPjtzh2el3zg+hrai+F\\\\\\\/1v57BD1hwvZ0229gRItn4KKUWBgwPXmWt\\\\\\\/K3Ri9XO6s7QDkSObXlKf43tSv2Co2rFcXU14Rn\\\\\\\/yTQGu1g1mISzMmOrkOCM5oRrtAoiTB79HElSEm4TfcBuAiVa9rNC8kh0pA4zTr1Q7l0TtS\\\\\\\/LDN+Bon35naiIm+wyG3sMKP9X3oi5nSvI1BbuzhWSRPGU9C+07ikz3fc4O0gbtAOpk6oWq6YzlxkJjBDakUl\\\\\\\/TQJYKIlvk9KV7TfLqaHQPUepC5jRtyMXStfQw74fu4jXrUWq9fiCfSsfuJHVZVsnlEIhjFlHQV233PbTr4I6m5ulrcwmGXnki91i5e7+X7c1n0v6\\\\\\\/KlIX7SyZJlLPqphJlateSMVUFr3hbG85uDd8MKPwWB0r8mkohMCDZwKqYzPzZrlfijpHh8pc1N9K9PMpdFMUTruTDSb6bPghFp6uJW67zx5gS+5hyCJZ6NwfSI5D6cYs5Z0stmaddflZdPKzisRrG27NOGEvfSDV9bQ+P\\\\\\\/HLzqRUY5jrN71MBeums6OC2stJlJnk1dV+OMtWv3FwwITC2gckOa1sRrF0GDpRh2h2mOdVemQLmcvi5ZK8HnAmk\\\\\\\/rAF2dq9OHFL\\\\\\\/Ln7oCjZy\\\\\\\/bl+AVZl\\\\\\\/md3\\\\\\\/ajt9PmVMpXS\\\\\\\/2E3KD6DGym5toHYJ5zZWwTLMRqIkwTh8MzMGqe89qmY9w7uqpr6Bq+iIPCCZrHFolZ0+sW9EblJLvX2ovlv3DoJwmsMwTLV4+58TuZCe0hmbMKR\\\\\\\/AVbrvQda11pI1r+mc79iYhOvF796POMcaUngZTBPHnZ4hHNYWLpfDTYRPpppmloU9zm5PLBP0GIH\\\\\\\/64XCSiPy\\\\\\\/O3\\\\\\\/BuMtJ9hnrHxWVmpRVs8l453GghDrrqbWP1IfpeSBvqvoPEhe3pSTh1kr0\\\\\\\/8ofzKtUd8Wo8tfsMJzCaL6cCFeIIR5poqY0X1nADxVnWv5b0ijuUWb0J4a7wOCQlvmjB2DSK4F7NOrcLCGWfAel08ickSBKKFUqu6fr7qfjll9JwraJm6Pfzb09Dam9E5z4meA3Akv7Bk3nmCm0pjlSJixBmoqiG+1r3QAVDfMIimn+ZKulx5YOZ6mXQ5yOkagHPejTJtpYYa7r7HpYimJ208MGIEsPADsUZZ9gyRW0TqrjLdoxLkG7EBkJ\\\\\\\/X+0zsYOE1KUmwJCmIg767ws9gRucy6Q7osrHVtWEHYLsIEbRxdXYC6BdTfwSlzXGajUw0MdXJ1U2fOaO7+1Q4TueTMKfRvFJ1xc0Ky8TaRLFGT7aTWUpqps1rdCrD6kw2BB0RO8RSXkhlZeZi8GcFWQTv0KxtH23e4u7b8+Pe8ky3nmyyFeuzIZIsLVsOGQXkTtsqb46ZTj5A+E12HipMOmEGLCR17k1YUHOBEJSC6HFeQDAzt7wBiNtlrig5XjqdJoOTiPeOm7MpEtM9eahfnhL6mc16PQ3DB6BgC9\\\\\\\/HcPVrYDP85CF9tD2hhKofZKyBqVPWJuu3CCEPvT4SIgg5VWWuClDsL8jlNqvozyrODu2Pf4I2EchkC+wkOHp1By5pebB51WE5tRlr5Hk5vFj2M6gLJLI5gIiqIjOjmUkx005z7xhNrTejAwUpYEuqK9xHSPLeNOYs2SboWY49r4ls64i0GpUGwjHXKjthcYMI9lDtPl8SeAaE\\\\\\\/DkWn97J45uDrhcqdoA\\\\\\\/Lio0+TzQRrpyBnBTymS7P\\\\\\\/deFsHZgMpUchoALOO+irIs7KIScYYJlvUveueyJ7RxEn8n7yrFvwTcVXOUfq\\\\\\\/TokLUfI63quimaLsb0luiXQv9eKHirKvmxWKYQENZK6ii4Wm0g68dnkVSdwW8x1vwlSwu7ibxz2re1240yxGfQZUozVaoAHrFxX3RFzjiiG7LLIjrNb5yy6EMEgUEG8vfd72vkG4eUjxMHe2CG0iGdYtmoFWOTPVM7FEYK3UQtb5XYirzj\\\\\\\/grVzPC8f7rO6Hy90Y4Pr5eRZed4gaqbyWuJ5MLCxSu7FzJPGyoA0U5jgBSokcKU72YBc8vju1iLOSRuY0Yzp4\\\\\\\/ZCyvIBtuieHQUziJvbLRHU6ifhGuq4+TO9+WjfH4GSNBeUDbiFSWAUat1fibA+XzBrCH9JFcc1Ajt29NtkMyoUm+edLADqB\\\\\\\/GoCgr096bIAeLiL+50FnEZzxSl30fcRAL3dS5uyyD6LB3gMEd9HWJ\\\\\\\/C2HzzirlU1RVLqzI5z7SbfkFV\\\\\\\/AQNRO2ffY0LRe8cmJAtCm9g8f75JRy\\\\\\\/zuWDy+g3vDi7\\\\\\\/JRuqGRSfREECqT+o7zdyZgShyHbtDhoPwl4Thg3Fa+tvesnoi+vdDw+0Lkrfxr382759wSGLbFoUaKhvh61Q4mXwc1Kz1RxXs6gOwLnCjA3BzSBzoOitsEQPVkWbVPWH\\\\\\\/6bYZwlUvLGNJ1SMXj009iBitlGMKOscjVFKsyM\\\\\\\/LUD2pLrKIqrZbeyzZRTYh9xXf5f+8fKP\\\\\\\/Jaw4RCuvMTL17zeO\\\\\\\/gDnQC2MVkS81bj3FCZfZZFTYXXO3oOXVvnlqX10E\\\\\\\/\\\\\\\/mJQVa1MsxbeglBbhGeqXRx7blPsvYkz8GEC4+CLx7Y+48ZG7YfDTQHFl5bcn2DFcQYcfvvtPThUiVH0CPGCbT2W\\\\\\\/nGCw2fIcvjNr7riluakbwJZtMxPiOhARfioTM4GMSdFsWJlS6MgXjev2pPz4biwTUH1K9xBehXXpiHsMeobfo4GIuz+rC955q9X9iHplG53MWD0YkbRY+vWncAaNG7THbKt4eSQrXr7N2HvP+xaYDysH\\\\\\\/HFacZKEazBkGog8QnYS7KCnbjh\\\\\\\/+Hw5IhWd4fFDykvGKKpFyyByeSIFHKPsw0Vwi2cSrHYB9h\\\\\\\/GT22bhqvosaT+QwsJo67liUlKYH+HbB59hzwp6WB4OKVwIQuAT7IslsHhtoG81iUgvvxzmf2xetd0j7FD+7JyLJmm18XByb78sIX4C2ZHHgdYISjuvZm82jTjN3BUqK2DYzK6MylX4c7aMJKl9yAxgpNZXdSPfW\\\\\\\/MinN2Ml6JJHi4TWU7DGgvM0rswZitqPqfcfy2IDL7oprWVtU1nGrcoejrXlFhivJyGDmptKLnPX4Fn68mlB2UCRqLVE5P7W4WoAVxvHU9wFMNJqaUj\\\\\\\/cXeZY7KpAcJlNvq2r0FVtZFhqQHXkdoaatypNtDP8Xune8AMXv1Q+kUOK+vc0eOK88XgzP5Dq3RIv8YXEYEgi9iSSukQtxFmnHCpeMgFqNlVoHvsIstX6kXfX150Npuj\\\\\\\/a0k7C4iGwFxv2xlfpoRD8k3L\\\\\\\/mZCbwFgLNqBCBty0fzFkI3RTZ5FFYoTCmj9rH8HSeny+Z9MYnNcnzczO9Xm229W+05MrSwTPMXoL+Jmezmpv+Nc6f3XBLFiUYP4uS2Mor1YyKaBnkgGAPftAQa7Le1VPJs88B3dzz5r7oT\\\\\\\/rQm0JXtQ9ZLU0kFKTN8+odf3O3xPRqL+O5cCK6q++3JAzs6HjS7yycoHz0sOKLy58QE0bZMKMtamPIzFy86eUeHTuljNw4vaxhbGgb6Q9Xio6ng26sokRjLeLIpgHWOtK8tFxOPoQNbxxSvZuPDfG38Z7xSl8ggDu\\\\\\\/Kwmz7wIXh4nLJDP83tx3o308PjbeDc2xPUCWrUr4mSLlqhf0TG1dQQ+P3bH3a\\\\\\\/5GMhJFwL34fwSZmKNEl3UAkBrRWf9oGO29kkHnQCGllBhSkRX5pD8\\\\\\\/KVRgoeQx9ewX8Yh\\\\\\\/pP+1L2Yd6SJp\\\\\\\/2\\\\\\\/KbvCK11SIvjxDs0F4VDpzn8LWdl12GsZz2bOxvZHtHjUPZTzKuIRbUjh2JuIYR1D1rToxVPwQ5hwqZgV68y2UEtImDGi3ITzBi9u3waUM\\\\\\\/X5jVKAIC9ZlOkBajZ2xTttZ8HzMD0Pxm9DpSCL+tA6pldSrNqWpF0uaeefoIlPQLFHHA\\\\\\\/uDLhxvCO9to7+D9S8z4YrCxgxeGFA\\\\\\\/Afy5NCzdxSSZttzx7FwAFldFcsY1FYB7SpWnQGnGqKbeKGptQGDvv74DauFkwdIPEpIeBMGJj6wz9p58iFUia05NtdsJjlYoMFQ4R7CmoD8+1geuKisMKngXZZaj8Wf5XI2isWOsuzgucEjs+sQ613qCN\\\\\\\/2+GZM9ehk706b1Kp5x8ka8Mkjt8n+x9lj6UBkFI7RAcKw6LhQ5XP6Ch0ky40lOMe1kimSE6J02u3QgphX5j7l2a8lNB\\\\\\\/t+owL\\\\\\\/zk4Bn7x\\\\\\\/GHbLD2AXp4TONjASgugDyUHLoluq9CpwSrGakphtgKFnvyFRd2vBaEEqNYjsJTyy5b1S6Gr9VJFkmT7h2NDZ1A8zGitu8JKWcZ0vIk6fzddVq9vIOKcYwKZ7AI6yOMbN8lI7uOdTCw2XbR9DOHk+A+1d9dm4Zjnf\\\\\\\/3eUO3Btf+nKx+sQqaaFHDZQa2ofmeB+G2xjzHwx49WpUDNvV8\\\\\\\/zP0YOOasl\\\\\\\/f7WCXBf\\\\\\\/VIP3D3vmMrjBjVNKhQqkKVa1TedE5gqoDkzCf8yOB3HxLNyMYW0Nd9fCJsl1xjcxP0bXGdB7Hb3MY5pGrJJLiYy2bisk7FrO74o7O\\\\\\\/X7RBwYAlZa4JplicumzSDSD4G0YbHDB6rbdtXYEjjSqTftXk+YjAVJkXnVAxGWOb+aIMhhVosTDeTGBTsq9LFg4bLqPoaCWIuD1sNLefFxoajo\\\\\\\/1D78\\\\\\\/xDsRHrn2sYs\\\\\\\/lVFaCa\\\\\\\/ES5JxCFUwZBTB3rskWSgnbcSj6NsrAGtJeELc5IA0NIRg68auubHlYSxeuRgJXp55Va2N5uWWD+b4yPg+Xb7UU3FcaTJgU8QCeWKP0DdiyjBRzeTWsvUEPIKuXbVWfJ8K3WL+mhA\\\\\\\/Nz5xQt2wTHmZ6EhVMhd6coc7rhVkwD04Ht7hWn4P6nsgbDZdF4fqTD1wb8iWKms2dqQJCbnjnbUP9Gpx5jVQ0mETekj\\\\\\\/iHd1AEGCt29GuSWasmIuvp\\\\\\\/gJGEcJdWUkNNvaGmF6welvlzPJ6G1E5L1U2YyX145W2xItigr8dcCCeI1kvTyr44AeX7Incv8EI0ELd3T3AVRz9mE\\\\\\\/OvSWTCRtFGqTejFqn9srW0m8Ot1sn+1hrrKV3ZbpMHMKxaNfs3nlXiCcyK1RMepwdxspKQYrlTHlzNfJeFJSUaejBKfx+4R5yG3Uy+jJ\\\\\\\/AwMNxY5uIhDLQDPGRwYk672KbwSxdNfg3YOb5pAs3taJRqkiWssDrPxkEKB7sn0K2iicJ51UvJwUeVgyGW9uBek0KqUEz9UTnmA3oCxu9FX0TjCOij+CTCJ6itkcNpyvBILJTWab8IGJADBmF96+iKqDHUgF2SirlIGOlqq34OXE+evSTrMSYlJE+tmUkdjH+7JuXD04ARu49xTOptDDXoJd8Gk6MPWy9VylWYSqMX3sDguFOh7AI0t\\\\\\\/mA64prxLv0hahUlTuIiQ2+anLyU6TFEvQj8qoIHHEZQSHfn1Bnh3NWTe5maHR0y4ccDNEU2LZ+pIVhhltUFq0uzs+4qKItYU5QUVC2YcmMlBkzENQS3BbgS4GmcvfwKEiThrfMJ+fKeiS4zyQ8UOPOjDEe0CucF+40SNFNl7sHfrRfu194c+LRKpS5dpm6avzSjRbOy5fPczkMcgTb4ochg7SrcVI6juxfK9N48h1wRBO8nx\\\\\\\/+\\\\\\\/KJVj+IPJqr0AV89ummR+dkgbmHG+qkbkzDZ6+tMcbjbmZvJeiH5L+8z08HgjhmGk6z\\\\\\\/ukgChUYATjPCJbXhtp\\\\\\\/ReE3fN928fk6hghRPJ18R4iWitwbEXa0lxQHwseM1eQHYCVz8z63qwLyrt7ShxyqHZQBfi9AYtV\\\\\\\/tq+nEPzK12vRqXiKPWbnWJ4fefwZgiFWn6gGPVDCNrNV\\\\\\\/n\\\\\\\/5x8H7IxGl7Ewgs5f8zu3c+fmtE3F4bfLezV92Dh0ZONmY5t8xdLEqQplR95FyvbJU1Zbb5aHVlLYQYa70syPlcqRAMV6SbT0tx8NvU3qK7\\\\\\\/v89GvdTRJRc8P954O+WHw9h5xcsrCVQPcmecYvaihXqn13du\\\\\\\/A8bdzaPhlzUdtxtOJQlDFHYVB3TNTuQ1CsTRgmewub2Q9jZ+oDAtRW2sY\\\\\\\/ksohZxVgGngpgkY6CnY2otdvPmbniAw4xLlljEZgpQMAnNnu8knVus5G1jRcPUPt9Nfxl4Xft9d2YxAbXZ7JKl0kUF6Tag+wo+PrIYVRDXkIC2NyhprdtsJxCtIU2rM+ShPsnWGlcr+Qa6oOD7Bo726qlMIw6DB0QEynWN8keTDKC2po3t6lLPZmb9qKjcgLQPjxfRPxindEdlxw43\\\\\\\/UjyVgWvaFno18USt01CBCQKWjT0o9PUv8b8sT0M3a4ccS9QA9WdSzY4mEmJjwcEreI\\\\\\\/DE9tBd\\\\\\\/IhCQfloD1OgV+KyMyDQ0SxQxGDpyZvA+SdKZ2xsP3yCNryi9koAKEt0DUIyCDBmcxhZgs4YeC9CC5yhy+gjIEtx2yrOnAQfv1phsV1\\\\\\\/r7dJfL2GAjhyeHKytAylYYl7rIVCNKEQul6bHpekUIWJQYFjqjkkiLB2dXjwH5jXDc1\\\\\\\/dx5ghVacYCuju3FPUq5a2cml+xBnY3FCPgXetLgAQ1UEH+FvyCUieIOlT7PFUW8dojQVkwGDE7JoM6\\\\\\\/SWC2UikPeimYvvU8pzw\\\\\\\/\\\\\\\/j201LY5kxjpvYoYFGVzIkCb12fWNC1nZuntPK2Rf7dgywhnecWB9xQvTTsSIUMp525iX9t9uQQGFNYlp75WiSTL5hH0BFBsPJ6nqq3KrZwkOCphDfj8oxK2yEtq5abVPSbx0FOzmyKZWSab2M4NajBveV85ejKqonbqpUIk+VXnIqaZkHbRJMmqddtsIRkIX3ZakpVg7l3\\\\\\\/WiEHRfXl0M0hUPEwYf3Zo4ibFs\\\\\\\/YGdxE2yrk7qNRHK4NOdpGOMvIkr9r\\\\\\\/jPd9RNGSb\\\\\\\/wBEi46ifshzmUNOt0VhIy1gI60DRh0SfB8l9ow78yGCKx7MnB2lh12ojRmTDfld5VzPDkVCf+hoe77nzNYazLMrmDg6tJS2FvqCS6U5YIGG4lnH8e4I4M627uFenY6JXbkdklFEpjNjV\\\\\\\/ohqY+9S6aKZOAnG3nM9BLMgw6DOrdTHgVzl\\\\\\\/8qmPyYQEO3JhIZ7woVE3fAyFCFEF7+7fk0ulw7yTTnfOG3LYQgVkytIfH\\\\\\\/n8z1NdmMuanz9r0MssQzREqoQ4Oe0LP\\\\\\\/JPa7BN1jz1z5oihGlmryt3u156\\\\\\\/M26hfNKFEGiOhf08qtrWdZOv5woE2ujtRuePi3LFTdJJu9oJArLHUOcorvuh0dY4z85QYUSLWosKQTIDmftXZESIkBYwLgaOIzYIOPbVhGACGSpDdpncQyTpk19llGMDByUTm8JERGOcVGz1pDGGuoq+JP6CJxnVDpp80k8Q9E\\\\\\\/d3lmlmc7H02eefv3RGBR4z3j3L6eEPthO4WzhqHTIBR9E15JrgQTq7t2KNTdL1L2xqLIPPOwHgpjj\\\\\\\/JAe+oIay+D\\\\\\\/giFUhPGwcxaMuidy9Qg7EDJujTBl\\\\\\\/TBHl80DHa\\\\\\\/FPKoxX22HQB6s7+PDiL7IxDLkdKzFISyK3cAxMaLtNn9GicHnES\\\\\\\/v2X6QT1Odfe9HDi9Ys1AI4kiN9eYkhtYSPPlxYe0vIn06NgP+YPUfy3e9mdZWP\\\\\\\/Vi4HMWeFQPHfmvGMPMOaws8UYmRFowibnpreXT9H2YnJRsY1aQaI7ZCfkh9nXUeB0AsphVoh7NSxctadufFnctZ2QBKq+IDKXznu1zpWdaUWvVVgIOYbanJER1jNuJQv2VymTax7LQiy7AFW8motySjA6QMVLIxFJlp28H2dRWkE3ywm6u6U9bWkKkE1UMpY4+cqOmqjCM9iP9vnni3Tcv3aaX9TOYXkWrFSUnH6yVpBNOSSKzW+He27eE3olapWBhSs9k8NO4Iy8DNwy651KxnVrdnzmsKsSPUDFiGbPNif+YyzqC5laGXl6B\\\\\\\/PxtZBlCZQeLvY8SJzl8Bipaw7wNDMiuhz\\\\\\\/akiKTLRnsB7W5hTIdKCS5V0yerSFC4xRUPoOXYowlJLSeeVLE18aOAxggu9LUwM0rDGWGCYm81VWOMWTEcwoYbrZz1sFGA0t+23jEs5pJYcDtPUGa2Iuug8qXmaDKR5HBldXnjxxX7XDUToJmuSFhzRxYlcT2ooZOP8hFlg6PSokSppMHhsKhfoijiprhUZm37CK89jVxu+YUJB6nH8XQVJPMxDnKRB3LSl8i1m2r6p8SAMHDNsisifNaaW4tHssX2SxILe1niMMRHiWRTMLj4R6ZbRw\\\\\\\/NY1XOfil3yVSX\\\\\\\/nQUJhgZDc2u0dK7cXL8blQvuiRizXbvT4r8n\\\\\\\/RHa1dlTCZuhApX3ySrtl0YEDyHz3qnknniyJUNuWAVpnXjLJ2bdYiVLCUic8clOmgoKoCpyY0uKXiiLYEJSOAimldBVOv3j9NwYazyncIAUlHwaIQ6rYcEVLIPSeDt+6IhyGjpXu0xYJrwhTHWUBa5PNqDgAAvBhZiIlYYbi\\\\\\\/fOT\\\\\\\/29U+EnjMk+0lKt+QhQ0EcpdWqtpSzZeOuxfXx1kyNY7LwccRAMs7K1Vw4TjBzee4tFkA6b\\\\\\\/y6rGcyuPNWwikSP\\\\\\\/OXlc+\\\\\\\/p5jH9Jm4gpV9GnrjkW8igNoo+3lowmfyv322Z++VK7Uby7bON4Y4zJj24UwgsL3lzNZvFQDAA9Dnjhqf6i20VDr15nBl6uRiXoGZciWDuDbJTnqpSdP0LYOQuizcpfxhmLrb9o\\\\\\\/MMtPFzWZjrIv1oxmRxG4fEwfIR1LJ7V3ef+umfYJWvCbs9cmLO36\\\\\\\/HVg8JxcSWBOW8LxCNqvHDfRtyBxc56Ykfa3xMvJxP4hBSaQoPs+HTKamwRYGGWYEi1EbLEyEjHf03XC9BLz7mwpBWzEraI3dFUmwXZD4XIhWn70ZSKtSsGSVDj+xxyrQUBsX9PUkGK4fa4dxa4q299IZM7+UTPfdCY8fvfmScCg9bMrvGO8+fUlV3JExaVSOWKJVXTVEGntE9a23b+UQaEdULiHgFC6Web7r2evswRacvi9Iv43X1ok9GgfFtSjSYZ5l50Ujm3qXGIZJhAKSCXuTnBxmmWNOcfKSnAH1fNvieij79hmGQDyWiNX6LtxQAwdFq2VHqRZdGlqA\\\\\\\/5AxJNaucMgf3F25IJaISWwpJ87\\\\\\\/5wZqvP2jMbocGg8ndCmW3LOfOhV8bogL\\\\\\\/xBqOxy9YwNWKh638TBuO4GYaF5iU5HGJYrqEFWbpzJ5veupTz9ibClEKTzibVUVzqcvZ40F0jx4kyvVYlkLAGBs5DJP21BHJZQXMuUnq4t+dEllTvsfupTM9Fl27mLn3gUv\\\\\\\/D4zVCeCeKmS6e2x6u0dHvS+AvTlhEzoJfNdnQVrNkVAl2\\\\\\\/x4ciPj2EgeQgKwApeuPMweT66qbCYaE6ow3wv3KxLUBGX6WR6rYFhXCnfj2k7a0RWRUYqI8WZpHvP4+sP7KDY3FmmOCudQm2UMAudnMljNRKBYQEVXKOPRNY18sYdcfCX3HVjVRBsiG10ynv73VRDFHjivNmmq9rFt8XZeWrsFiE5Q5eEb1PMlAKT2xy9gtt+SPyNW3s8HbTE6FI0a0q10c8MbzEwfJ4H2FHCycXpDUuIGxFkyRec54GfKRscvNOeZYvj42ibUkA2r21fPzzwUhbMPQIAmhFDXV3LuH4aONYgzj1Id9v97TfvfSTdS+CoiM1bjmt5USJg\\\\\\\/tpxHIIsOS\\\\\\\/TCxut5h\\\\\\\/dzpSE9QL9uwXXhLsM4YLtP2yJUWRIexnazI1nhW3NJseG3+SKLlUonJ\\\\\\\/43L2yfeyHvYtblHfoMGz14tjIjuHUsW5zEVrenGYbb+enNpBP2XwX3Mg0oiqZtd+TbJrnoPOnekrE9Y1s8JLsPQJZJC0ADrAU5VTCPZlxlJOeJVgkD7EtknSq2j4YbhUmNuVQED7TVORzb8ykujmOg6Nu\\\\\\\/OEqWb91jfuiiA2G5Zt9RmR1PKaSC3iElMtTfqkU4jh54gkdRfd+BQ+EG\\\\\\\/tCuX8DJIEBfYDYIiqrT8Y7s4KFBreLiDQiBn+8oElvkbDnqA7Jpc0+XYOlyWbs2LDKJaTM3kYZ3\\\\\\\/ScrapQ5FEgxoiHvJErzTKytnMgV4HQiPSgFV8B2huO+5RAII+3kYCOtReLStY9pn749Ms0t5bSgego3Ats8EkTtwSOoSm9eL6eW\\\\\\\/1S+mf5+mfKMYmER0UKpUq7IL6YWVka3Tko4bYc89R1Qx5GMgFaKCreiv64TSJdFhq0e5gOpiMY2sJRs\\\\\\\/bt0YtJs8t68crp7iA0zepgE2Tix806wL0RouGbwIKOGwOnKFsBERLjH3RhsX7qLJ1sMGTJsPr2I8clzbd0GUlkqgAxq\\\\\\\/olQXTFt0XxM39FdFAP64ibiPoYAQB4QB5PqQCa+20Mg5SHKLtBihIYUJtYB7uFx+PhqscS1Hu0yflEaICiZohO3JHXvkYdUxg7A0zlzVook8\\\\\\\/EtAOdlD6PIllAd2ioGpBahqVQOY086uL0zhBHFtEwVUdz5lGO35n+u60gfBLTLnSyZgo36OHlMT4SeowDk2bE1aOZHaI1h4\\\\\\\/wDDo27eb2ohfiwr2d\\\\\\\/4F47FJhbU8ctd4AQs4pDtEkgjX0BOkTKMMSr0s8WZWnBuBU\\\\\\\/llm\\\\\\\/+ACiLS8VlDR+ZhxLQEPaaoPjhlghgo\\\\\\\/kjHqpoLeaR7LkHyt6VI+mvFNEhScQesO6MR9Ai1ZCsygRG3dsSfsphOOZdQ7J7x++iuWKXpU0iFNChxebM4kY4GGSkhtyaH0fsJVaKLy1Cw9BiPuh621XpMMwpSEnIUzQ9UiUZeu5cQY5eu49qU0xKMiSgEepai1GZMDJ0iegHb\\\\\\\/c\\\\\\\/tfMOxkHf3SqGuQsepIhDaa5ayKnNI5qxl+l4yrTgtKCa0qQHdDJOgN1PHKu9WJ9QaCdxeMwvnWNjq25xTo6yUiOLjLj2BkiYW8w4baxmKnCoBu8KW\\\\\\\/vBu+fgBqw6Cc\\\\\\\/+9YYsu1l9wy3dV5UpOXnl2b7TF4i+WCJzPZapyXW4jAZMey\\\\\\\/wGXEsu7ERMkb1KBm\\\\\\\/a7Gu20LvoZBFLRxOE7uBko2SNmT4aZRPDZDaT0TbIf8m\\\\\\\/6oMCQngcbVSFNTh2+t1YqMBd3cgItVrPPeIdCbcSxEb7ql\\\\\\\/HMK5\\\\\\\/zNMPDo2sB5uR\\\\\\\/7AcFRwyz5kHJVawru7yNSY6ey8ldXcdDWOomtbO1J3LSDXF8KqZcT3+9\\\\\\\/AdLMg0Tniqtvtc4q6jdh6jbxaxGeCohYqTA1GOGlL+wLNhR5Xox9vlnAtUFUsgxlAh\\\\\\\/D\\\\\\\/iFBTPM9rIBdI4ysuirjhIO+Yi2W7R3h1tRz+6KvWM0r5xj7JK1UqukLj+kq1z3i8KNLF5n8YFgR0dduCxChnkGW+SS9e69qWOT3xeSO1ZJEXwD7EbZelhO\\\\\\\/YbMEItTvEHnMQWZzskda83lIMN9KEeZEQcYzaI\\\\\\\/pcu\\\\\\\/RnagtX5SqP70ZdqoEvgGHUJw847p9P97tG5fEALmatsdU+5C9TScOlSxN7KD0TTJsS\\\\\\\/7nrEgaBvCQOUtfz3ZOzj\\\\\\\/NfwELI+U4qxWRkgKzbI0NXF\\\\\\\/PFEAthDQkKksCjV1FAWru1Uf\\\\\\\/0yfSqI6uJHx9YjM+bhWtBoOnkN75rh+8CiZ7X++A93cGWyxa4ua2U1srRc0xOPAmkAE0i7xlSZ8Eyr7laA1AyM5KBuo9kPNNsM6WSsBZESDYupXo8VrukI1EJs9OME6kJcp6nl7TJx+Mpkjmblz7WeQCmt12Rfqa\\\\\\\/+z6qpykUHIa8KMLnUoUzcJNMwJLcmO7lN1IsE\\\\\\\/t0WbZRkbsTKvodSU6aNL8r\\\\\\\/HWcn3mspxqoYts78Bd8vrR9+hs3zDSCE\\\\\\\/drjPTnHkyw+UUbvNZpvg4KLwcged3a1Xer8ys13vtFoesOOu91uSTgkRQKgZ7EbwXft4\\\\\\\/bpCnK3zVdJ8P3mVxlcFKiwRDjYJyAB11eydz5q8qHluE\\\\\\\/EKgyz403m4yXyNLUv9GhjvLgd2k8tTZUw6t15sixIJi4VUyT8lyJDYX6e2CTxbzyLPjB8l9REXBcGjkXKZSfcV7Bvk0jVkQP0LmW\\\\\\\/BzMTnvbb9ZoTKtrkKTZYfCu2ifLxoWxT1vD7Jdvl7VUH6iwsOKtAmg+0akgVdYvc7TDLniZdycftBQjf74tVJ3qe\\\\\\\/nNGrg+2CiTUOcC5DYyDL1uZAdSYwe7gojbA36wmNtTtSCavBit8BllQlVTFXkLTn4OLcXta8VIYs6wpjL60XRH5o0iFBGP3DwM4k++uemhI5ko44XlIz7zK\\\\\\\/\\\\\\\/K\\\\\\\/Ycrd7+Fllt1fZ0WbAlXB7bZamUxHvuBwZH3jzWEMgdqQkCn9qfX0xzcKHJSMvbSAiMrcX+BcniZ+VRmbXrG+g1ntwsHY44067kLuKaXAVjZwUbRo6DUqAwlpj\\\\\\\/BvqZRA9bM5HsLMNGZi2PjmyUWQGdtmF6ngS\\\\\\\/+lkKhpirgHN7SRG2fR+hBgGago1ZU9M7nr5F48S0p6iY1zBtB7Up03zfq962MTicW3QVO\\\\\\\/2c1mzMVxW9uBm6jvAHpAUPt\\\\\\\/ShcWbPx1sRT+goWz24K5X6PgapQzI3DPE+SwFU866kXDeWO0lrFZO\\\\\\\/TaKLNRea\\\\\\\/HkBUYhTopvkaX\\\\\\\/fEkqcmosYBQhJy8bJYRGLOfcZ+KdYgkNNCI73lrwkAJQdlQ2uBeAar53pHFntNTKk\\\\\\\/7jhhhGG2Rf8j9T47PiTPhIOravsNPYAukpTXIcpYmgoX7hFLen6Gjbn98ymtKEOMRrcKhNnPyh7UJKgbrj9SVuvT5WFBdoR+UHL30B3UtL7bEi1t14PW1NAfze\\\\\\\/jaoW977ojhBvuKrCgxToZQvLHKJPZNgPO\\\\\\\/9dPQGhm7RuXp\\\\\\\/DD+pxXTTI9uLz0j2mJpmk4ahw+B2K1j3E3PX2b3\\\\\\\/vkVqwiVetGs8LLHK7+2mAZtpokV4t2JwBDPcI8NMBr2\\\\\\\/+ltnkK1uiTXhvJ1WRqv+hzAfVVpIleJhwu9JJHXfh8E+OvX6M4L5PAo+v2xvhnmqy82HCQsxwhaSb+hEYGSgOxxLYbfHdBcrBFqxh2MOU0Ykp2fn2Gfft3nOGq0\\\\\\\/T0Nddr3CaEg8V6JqSNqxN4i2u2FG9AE3\\\\\\\/rdm37sfeGoZHplxLJixL1rPTb60jCMObYNbH81+82dxVRAfrXiqXtq4r3w3KBh0OmQ+1rHvpuWgvYTaqeoYchTn7eJPatzpkt74CiDhAxC2IU6aE8PZU35F3KPcSKUWT01FYnDDc\\\\\\\/PbE9RVgyQtSlEa5zwHT7wqwv9pEyhf8QeBDsrdLpLgq\\\\\\\/g8BvSSqPQkzSFQlTFOzG5dUh5tIU\\\\\\\/sFuxDeDQipUFyD6c9UEQidkJyWEpTB28TonyEMbaNxfBqH+5tCI\\\\\\\/d6t7PLS1FbCs\\\\\\\/uZnX5R\\\\\\\/ge13S1jlrkm11mjljZsCPf1SbkaKvIilA3yK\\\\\\\/29AlmhsFejgzEMcH54L4oO9VFMguJMcpmxH+LeQ4sevSpJQto5C1oSjolBYbSzO1k2GbYQavDy1Rrc3ztX5mHx6WEduhG9E4CMX41O8\\\\\\\/6X6UWZrMGiBPU2CDaKmF3Gxcnqy5NBkU3qEBvQaXMoWRJ3eFu2KEstxV8jAkc06mJ\\\\\\\/nC+Y6t6aYfyeIzPXWAs7iKwrtx\\\\\\\/nGKXWyUhusXi\\\\\\\/m1YBLsXa7S8BEBZJsyXmcLVP94JmhhPmpSbhpgcdhx3PM\\\\\\\/7j\\\\\\\/ihbImO+WPQszDcqIAclX8yHbLYy2D61cxKZvtZejdta8eeI3MOuQ4Xbv+27X8uE8TcJFErBLyMXVrRhSGaxeNklFY8XA9\\\\\\\/K\\\\\\\/4EZPI08jUYPHQw04ZNd0jTNgCTxTcwqQCRCT3Fs6POwEvoCIVzlz3pdo5CnuBlNPElvOUwcd\\\\\\\/t4sheJ2glW0c5ZqsEw+\\\\\\\/bflimH\\\\\\\/PE4jQw81OsGsf74Gyw\\\\\\\/acsSrSK+k9nq3sQRkeRPlCQSOAGwjmWTNvo8PgnwKYT+kVKE\\\\\\\/335O\\\\\\\/VmGIL+e8fW4pEyXi+KylEtdR6eAVYN+MdgxBxCJslEHdz3ywayB8NXnA9\\\\\\\/3r6krMKaoDIPnSEhUaQcz1eHcJqAJE5iwY4Ywl0JHBDNDSzOxt5eUlaXdKv\\\\\\\/qZeJVo0\\\\\\\/EPHzrgY6HRZCQNt50xN3yURsMkFh54nmdS8P0lq5j3G0GhjLJ\\\\\\\/v7NVeLUzL6HXGWQQUHWtj2eBrc5Oh9\\\\\\\/B0b7VhL\\\\\\\/HGjl+EUeyWKEC\\\\\\\/5AAzQtdnWBqs3Nmf8gX2T8+a+3+uFQ2orwr0jWcWF0g7r0U4Ssp85HH46uLOlVxMQxUhFO4QMUazA8G3LcLxNckht6Wu6SVOjSkWvVtWshK9imUQ+AP5IU\\\\\\\/sSc4DOWkGtnYmHbXQNjaiPo2UruRcgUmfPIhrPFSCc8F8FPIJRDLwUz0YDh\\\\\\\/Dy7h6OZsbzdw1PUU4yiTXtzhgDZeP6WvO0W3kyyRxFFART0tQzSCsRBy\\\\\\\/+QsNQIwILrepb9IcxelD27yh21Rw1aiC4xrg5NUQV4a8ucbdeEsYL32lORzMgSS8cUB7IrJ+pWrb19kpIQ0xX8idz1QuwWlw\\\\\\\/aonbRzAcSPW+c6vgMQcrbDYLodv1NMrmjT0Be3CDkgv7v3KC8e0w4kIxbKyQ\\\\\\\/pxHSmPXvuru5rv4Sc99Z5gm5+EDJzPJv8GeLFeBCjud66qbD9s2yjGYRlYpt2fsaCCFx1j6hF21aFfvhrozr7li7lSxyFsjdBAHDaVeOXQJpol8aN9X6gh1WtKUD79WbZOvdXM3nfVa05FG34NhR9ha+WepSJrUFXkMPH19rv9mq9N\\\\\\\/U4RDVgx++45NAgEasgKc4RNBgU78ErIG4rRzgkjEOBzWuhMkh465bldDlXUT3aTfEqHjBg2fK\\\\\\\/64zXYL\\\\\\\/+ijd30Xcfdc1ml4SAy2M706O2zHqyHU8EWGMpHdbEvo6+Eb6XLyqTbNKfYsgY8tKKbKOsCaxW+2RFo1TbkZcEOi\\\\\\\/fBzb11+G24kkTDV\\\\\\\/GIKVAP+\\\\\\\/uNtbRQaz2Tua6PwmAopD8N3C9tmEd2lohpQqhOEF\\\\\\\/1PsjVnkVhxpdp1v32Gol0b0kcVtcjK57J3d5Fxk8qZ+yB6iOlQjWwe2wkOGPtj0mld4JLj6LJzUUZ\\\\\\\/LXSn0elENTEPTX5r+IwVXRkptOXKYTMg8SsfFjeINSBfz4cX8J5sotvBnNvi751RkefcSDiQMdjTtBoLzbIatImxFfb5z5Dnf\\\\\\\/gl19gV1xCFD\\\\\\\/MZ\\\\\\\/jHT99bTwFbALJ6zd\\\\\\\/67maC1q6t0X7tt\\\\\\\/jeLxP8RXizghOvdSy7cLhm67zODtUCrJ4tK75DnY2cbd1qNKCvslD9xiZ+Z0m5OCd6y2+YNQiwNAyL60Ba1tIaWGw\\\\\\\/QHL1tQa90Ullt8o0xhUcNrRycfKJ7rNAEIBZVu+Tl5n7HJuu05K84lXQwitLnrVm1Knj03wCDiInifcez\\\\\\\/QJ2sxjXoiBT34dkK2lDl3Ajyya\\\\\\\/mJBMptVhtWwHIoIfqKF+NBD+vpqOE15a8utK23jftPZQd2Q5qsRK6Pg4V6suI6DRW5ZldA0lzp28scH9vRgxRs56fOEyTpI0A\\\\\\\/4sKqV62tH4ThZn0Uw5NyqMpyvyPVqTUfb\\\\\\\/rDw24LD8wC2Gu6shVJlCz59tN4NVWtb\\\\\\\/YYOWbQryqO\\\\\\\/7PNkenCoTFKzu4qaVBN5pyfMSTjiKOj4ouSa7vAEwkmwlTa+wZ9NmiOjgHiCdF7Bkg3QJCJoG0Sf171mLID6OMg+1LCmlilB0FALWODEhoTg8BFZvU0pHp18iNhosnQR5CJc2H5ezYlUsv0RGBZBk8zU6t3jUSqyrVGKaZHdW6x3Im1pmGJYLp0hDvvKqSE63p7vBtcG2fbtcTVLSebI\\\\\\\/se3e9b21IqwVN86CcHSKwpJPgtWHa5TjfKzQZVudnTdWNyrIGRLD5VPuKItLCkI\\\\\\\/rcAjlFdlCvt84AXZpVz61OrT1jwLXEt2i0mOnE3uNiigc1xrrBe0ODdFPjZMuuWFjanqnolcnNq2sFiG4LxOQIG37iebjtsUpJiaYiEwni+rtc8r4PkDp6rydQ5l8IV2KVQXLABSkCXaD\\\\\\\/mcCULBJPnzi9nWE6zxjyjbrPlAl86xuzqwZLhWr2kKrO08WPIhvZpYhzQi0dpPCcp45O9WP7xJeQYHGTmb1lrejKvNu\\\\\\\/1r5IO8lZlQ3LwnUv4MVEDewp4v2bAulHx9zkThHzkFBF7zJEcVMiAeU22+VGsk1+kMJiIu+sY8sWZV1NRV5cYg7uO6Zp1jKJGvqbGVbzeqKxQvk4bD9YmBNWmPwxuuC9Kf2LAcZIJ06tIt5QZ3M3K1OpuM2LRp6PDM\\\\\\\/RznriRygFNRJUeo9tjzRl8fz\\\\\\\/1Lxx6am16VATHPkGJ3wWJGlQsTKG2sXPR0plyeVh147Tkkpk\\\\\\\/rwVUx+PjsqNJnOS1nC9Sd+TNSkst6Dh2gYE3kvopzo7ULpc45Av9Pv4+QHTyB71W58D0CiDfrxx1Pw4biodXVBGm+FfldxFqu\\\\\\\/1MI8WRHOIG66C7ITdMRuTlQrtB9zZlJI8yHeTyV6+TygHpKAFGBpElLva55jT2PfW0PfbZE1B3H42dzEAlFn+MBD5wQLIiB8aUDyVDmJcVu\\\\\\\/HHyX+9UzuOl9IsgDZW2JsG\\\\\\\/Pz62ZcBqz7Irwo0BeU+l52xtrTeOeoNdhjq1dnn7W+uHEA0V811lflcCq0RtpVqVP56vh9EALLBwHy5p236r8dQ0oEciZx\\\\\\\/YVY8xjPPOhKBaXqO5v2t9iHon3Vs3XCsaphtOg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022YaaruQLZNOKCdXp102Dhmg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002267b907430c535a337621ffbaed5f999d4fa9794af36cfe5345f33be10b7aaf46\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022y7Hcq1fkX9GyakdzmrOGmVPgrneopUJ6WQGbOD\\\\\\\/9fvcEpJ8bFIuDTKQXSLWVePnw2zk18z7yHADamdsR2bYOrHQ6iT3aJ5EV7PdVPffeq5eBvqk58jFMnl94+XaK8AxZgeRvz3pfDE6e42to418TVXFv3vgqWRubkvvQMK+D5zquJSb12hFpoGqcA+Lkg4jHR7BdaFq0k8C8h2yyTfX\\\\\\\/piacNnoACmLICJHro3uHqHx4X\\\\\\\/vi4ioRa6bOyZL\\\\\\\/fjdgV76oLVqyJrsYICEKVg4zu4ZE4CIvaj7H+lH0jqGqkgAiVJnwXwLrUYE8crsqdRxotvmBo5wUvT2aqPAdFVBQSufeQ2SUvctNTR9gm16eD2kldgbbcsMDwvEJF6ZsQQbSExcIiDhrhr1kX5Fx\\\\\\\/IngzsYaTKZPmweumF\\\\\\\/53lYFyL\\\\\\\/ihZ9tQO8DrOi1XiGS\\\\\\\/W89\\\\\\\/NLURWS0QJCjX3vNKzAIBDDSm4iNw6qIgrJQunUw8e+QPBmKI3WikHY3ZQyiIaKqwDpDC7u4rrYlorhkxK0MscFIb+3mrIAocXcBbM7Bb24wSXOzfTjl\\\\\\\/pmwcjpb4TVsowZLowXJuUdTKiydEOU6oOWpeqI2lXPj6+uCWzYVlCw93GgsmNVCx\\\\\\\/HHywDeYW0YhfWYyK9AD35bvXcSOf8aF+pJ3wh8lNSDPdkHZlnyEaAsyOQn1n\\\\\\\/HJLhP9249BzaGgvR0xvxFNPtdn1UyEJGeQ6JQB09QPjdjy\\\\\\\/tXBZ6H2DmLKKQbZUBU7APJSYDHb3csSIgPVYwRjI4v5WVXNhjgswWNzON+wBnDETIBStupoCp41Ojy7XSgLO63vrzqbGMngTuxAKZaEVwT5eSvALbU5LnDWzEPWla9pnNC85FrXwe+kc9PBxcWNB2SUP0BKkUR5DJof+2RzLBxiitRYxqR3odAV7EqwwwemiN8I+Fi4OWwIvYEqMqDiQivwZDHMwopBnBphCXtQUjhsG228bb90spj6s+D4YfmI9UEw7pGpdGMLQoAaz3SkgfRssZmkcjW\\\\\\\/Io6Sy0per1ceFx\\\\\\\/ZYzS1TUD7fX6biLy8Y2\\\\\\\/6cD8T6v6h1BZAhEh0nLSC87rpOqD\\\\\\\/rEg1DVtZGuUAgrzkHuChpwdaVd9aZoG4JyeBDgxKyrdnk89\\\\\\\/hD0yM1wAisVabv1LKMIW6qfhcFIUjKpsGbLlVu0v6fpQ5igo7W4yDSTcoUegJ4tZxL\\\\\\\/0OkUOxgXqjPRNtX6PU0JudOUmqofGkBKj70yoU\\\\\\\/nntUmLRAwmhloqmsQ6MrITP+tINnbv2BG7j\\\\\\\/dXPFCm5P5tgkLdpFsS09DJXWyhkDnqFBe6oFmcC+YycIuOSADcNc0oExxViET36MpYk\\\\\\\/uVpdkDPYM6GpO87\\\\\\\/lFpeOFH\\\\\\\/dec81vPsdyrlGhABjjgv4R7VQJ7UAH3pgJmVFDwpZu2PsOkqLY2Y4rV4MPoCj+utv0t2I4J4DpbsiMg7WULwkn7uarJCxc3Use6BCpnhL\\\\\\\/ng4qG1L+TX2EyKcdy9ZFulZ+EB48RtTtHNyaHuIELxYnFFBS2r6z+hs+K0bJSwpukPCLYp4cF4W7JUpupjNMFzHsMjSsoy9WHZBdx3aAOZUkRQ\\\\\\\/8Cu+xqcQEY3S9A\\\\\\\/fFMWJkoYfuG7+v9TpqKYU3ZnWJ0sXANK3ZWEln4vMhYTHJ3RcP0pOJMLrZPIyL6Bo96bSjkjikqt+kRM+lGS3CyjKm8x4e2GdIziCJoYdqQM5JmZAYWrTM5AX5SNxH5b6ukdoPW8NHIUNWAkI1SObRPyjInyI+xAoiAZBHLYwqRnR08B51aijGeSYVpq7eeNzwKQm2+Q07q+kPZoUXBIiFj79nUmh7YhXz9E5jCFuie3OpDgrIG89HDoVl0rx3iURwD+rrPnqv721rr2B0PTgz24GJDGLNFqG4AnOcJ+00uayjqmkQWeTdlHhJpJCCmJBT\\\\\\\/7pBC1UbzxCcxnhuHm+Rg2nLT1uiHhMb9\\\\\\\/\\\\\\\/vvWmcygefqF18l5vqlPqRLVS\\\\\\\/9phKZ0h9ZArSAIVDTs\\\\\\\/cQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022PFJq2WGIT+1Xxkvw\\\\\\\/Kdwfg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e09d964e9d6c3f70a4763122ae749358f46f4655ba51b8aa709a78ee8756da4e\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487831 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/1" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-2ZX5iPM5IuheYlyOiDhFWdyWhVVOurGpoBBXZzMZA6E" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/1" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qa6rQqVAes-Ie6XVek5iG1wyo_q9PIC_TMIcIZtnXoY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/12/4019" title="G3 Reviewer Index 2017" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/9" title="Mapping Second Chromosome Mutations to Defined Genomic Regions in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487831" id="node487831" data-pisa="ggg;8/1/1" data-pisa-master="ggg;g3.117.300196" data-apath="/ggg/8/1/1.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Reference Assembly and Annotation of the <em>Pyrenophora teres</em> f. <em>teres</em> Isolate 0-1</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nathan A. Wyatt</span>, <span class="highwire-citation-author" data-delta="1">Jonathan K. Richards</span>, <span class="highwire-citation-author" data-delta="2">Robert S. Brueggeman</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5634-2200" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Timothy L. Friesen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1-8;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300196 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487831" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nathan A. Wyatt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genomics and Bioinformatics Program, North Dakota State University, Fargo, North Dakota 58102</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, North Dakota State University, Fargo, North Dakota 58102</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathan%2BA.%2BWyatt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wyatt%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANathan%2BA.%2BWyatt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jonathan K. Richards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, North Dakota State University, Fargo, North Dakota 58102</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BK.%2BRichards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Richards%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BK.%2BRichards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Robert S. Brueggeman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genomics and Bioinformatics Program, North Dakota State University, Fargo, North Dakota 58102</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, North Dakota State University, Fargo, North Dakota 58102</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BS.%2BBrueggeman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brueggeman%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BS.%2BBrueggeman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Timothy L. Friesen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genomics and Bioinformatics Program, North Dakota State University, Fargo, North Dakota 58102</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, North Dakota State University, Fargo, North Dakota 58102</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cereal Crops Research Unit, Red River Valley Agricultural Research Center, United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Fargo, North Dakota 58102</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BL.%2BFriesen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friesen%20TL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimothy%2BL.%2BFriesen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5634-2200" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timothy L. Friesen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:timothy.friesen@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">timothy.friesen@ars.usda.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/1/1" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487831" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487831/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/1.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487831" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487831/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/1.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487831" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487831/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/1.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487831" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487831/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/1/1.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487831" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487831/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2"><em>Pyrenophora teres</em> f. <em>teres</em>, the causal agent of net form net blotch (NFNB) of barley, is a destructive pathogen in barley-growing regions throughout the world. Typical yield losses due to NFNB range from 10 to 40%; however, complete loss has been observed on highly susceptible barley lines where environmental conditions favor the pathogen. Currently, genomic resources for this economically important pathogen are limited to a fragmented draft genome assembly and annotation, with limited RNA support of the <em>P. teres</em> f. <em>teres</em> isolate 0-1. This research presents an updated 0-1 reference assembly facilitated by long-read sequencing and scaffolding with the assistance of genetic linkage maps. Additionally, genome annotation was mediated by RNAseq analysis using three infection time points and a pure culture sample, resulting in 11,541 high-confidence gene models. The 0-1 genome assembly and annotation presented here now contains the majority of the repetitive content of the genome. Analysis of the 0-1 genome revealed classic characteristics of a “two-speed” genome, being compartmentalized into GC-equilibrated and AT-rich compartments. The assembly of repetitive AT-rich regions will be important for future investigation of genes known as effectors, which often reside in close proximity to repetitive regions. These effectors are responsible for manipulation of the host defense during infection. This updated <em>P. teres</em> f. <em>teres</em> isolate 0-1 reference genome assembly and annotation provides a robust resource for the examination of the barley–<em>P. teres</em> f. <em>teres</em> host–pathogen coevolution.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/pyrenophora-teres-f-teres" class="hw-term hw-article-keyword hw-article-keyword-pyrenophora-teres-f-teres" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Pyrenophora teres</em> f. <em xmlns:default="http://www.w3.org/1999/xhtml">teres</em></a></li><li class="kwd"><a href="/keyword/genome-sequencing" class="hw-term hw-article-keyword hw-article-keyword-genome-sequencing" rel="nofollow">genome sequencing</a></li><li class="kwd"><a href="/keyword/rnaseq" class="hw-term hw-article-keyword hw-article-keyword-rnaseq" rel="nofollow">RNAseq</a></li><li class="kwd"><a href="/keyword/pacbio" class="hw-term hw-article-keyword hw-article-keyword-pacbio" rel="nofollow">PacBio</a></li><li class="kwd"><a href="/keyword/barley" class="hw-term hw-article-keyword hw-article-keyword-barley" rel="nofollow">barley</a></li><li class="kwd"><a href="/keyword/genome-report" class="hw-term hw-article-keyword hw-article-keyword-genome-report" rel="nofollow">genome report</a></li></ul><p id="p-3">Net form net blotch (NFNB) of barley (<em>Hordeum vulgare</em>) is caused by the fungal pathogen <em>Pyrenophora teres</em> f. <em>teres</em>. Globally, NFNB results in regular yield losses of between 10 and 40% with the potential for complete losses in environmental settings favorable to the pathogen, namely, susceptible cultivars with high sustained humidity and the absence of fungicides (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mathre <em>et al.</em> 1997</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Liu <em>et al.</em> 2011</a>). Several studies have investigated the genetics of this host–pathogen interaction, utilizing biparental mapping populations of both the host and pathogen as well as genome-wide association studies (GWAS) in the host (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Liu <em>et al.</em> 2011</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Shjerve <em>et al.</em> 2014</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Carlsen <em>et al.</em> 2017</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Koladia <em>et al.</em> 2017a</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Richards <em>et al.</em> 2017</a>). These studies have been critical in developing hypothetical models for this pathosystem. These models have proposed that the <em>P. teres</em> f. <em>teres</em>-barley interaction involves the production of effectors that are involved in manipulating the host to gain an advantage (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Koladia <em>et al.</em> 2017a</a>), and that some of these effectors may be recognized by dominant resistance genes (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Koladia <em>et al.</em> 2017b</a>), showing hallmarks of an effector triggered susceptibility/effector triggered immunity type model as described by <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chisholm <em>et al.</em> (2006)</a> and <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jones and Dangl (2006)</a>. <em>P. teres</em> f. <em>teres</em> has also been shown to produces necrotrophic effectors (NE) that are involved in NE-triggered susceptibility (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Liu <em>et al.</em> 2015</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Shjerve <em>et al.</em> 2014</a>) when recognized by dominant host susceptibility genes (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abu Qamar <em>et al.</em> 2008</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Liu <em>et al.</em> 2010</a>). Together, these studies indicate a complex interaction where selection pressure has been placed on the pathogen to produce different types of effectors to manipulate its host.</p><p id="p-4">Currently, 1084 fungal genomes have been sequenced and deposited at the National Center for Biotechnology Information (NCBI) (as of August 2017, source: <a href="http://www.ncbi.nlm.nih.gov/genome/browse/">http://www.ncbi.nlm.nih.gov/genome/browse/</a>), and many of these genomes remain fragmented. The genomic regions responsible for this fragmentation are typically repetitive regions, which are common among fungal “two-speed” genomes, a term used for genomes that have a distinct bipartite architecture made up of typical gene-rich regions and repeat-rich, gene-poor regions thought to be hotbeds for genome evolution (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dong <em>et al.</em> 2015</a>). Genome assemblies that rely on short-read technologies struggle to span these repetitive regions as the repeats are often collapsed (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Ekblom and Wolf 2014</a>). Effector genes involved in the manipulation of the plant defenses cluster proximal to, and within, these repeat regions (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Thomma <em>et al.</em> 2016</a>), and, therefore, it is critical that these regions be accounted for in any genome assembly of a plant pathogen being used for effector discovery.</p><p id="p-5">The current reference isolate of <em>P. teres</em> f. <em>teres</em> is the Canadian isolate 0-1. The original draft genome sequence of 0-1 was reported in 2010, and used paired-end Illumina sequencing at roughly 20× coverage, resulting in an assembly with a total size of 41.95 Mb (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ellwood <em>et al.</em> 2010</a>). This assembly contained 11,799 gene models, providing useful tools for the interrogation of the genome; however, the genome still remained fragmented at 6412 contigs with an N50 of only 30,037 bp. The fragmented nature of this assembly presented obstacles to the study of the genome of <em>P. teres</em> f. <em>teres</em>, especially in regards to map based cloning using biparental mapping populations.</p><p id="p-6">New long-read sequencing technologies [<em>e.g.</em>, Pacific Biosciences (PacBio) single molecule real-time (SMRT) sequencing platform] are capable of producing reads up to 60 kb (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Goodwin <em>et al.</em> 2016</a>), and, currently, two fungal plant pathogens have been fully sequenced using these long read technologies (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Faino <em>et al.</em> 2015</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Van Kan <em>et al.</em> 2017</a>). These results demonstrate the utility of long-read technologies in the production of better assemblies of fungal plant pathogens.</p><p id="p-7">Here, we present an updated, reference quality genome assembly and annotation for the <em>P. teres</em> f. <em>teres</em> isolate 0-1. In order to update the reference assembly and annotation, we sequenced a total of 14 SMRT cells at the National Center for Genome Resources (NCGR), and conducted RNAseq using both <em>in vitro</em> and <em>in planta</em> samples. The updated 0-1 assembly is currently in 86 contigs, with a total genomic content of 46.5 Mb, with a high-confidence annotation set of 11,541 gene models. This updated assembly and annotation presents a useful tool for the genomic interrogation of <em>P. teres</em> f. <em>teres</em>, and, specifically, the investigation of the secretome and effectorome.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Biological materials and high-molecular-weight DNA extraction</h3><p id="p-8"><em>P. teres</em> f. <em>teres</em> isolate 0-1 is a Canadian isolate collected from Ontario (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Weiland <em>et al.</em> 1999</a>). Fungal tissue for DNA extraction was obtained in a manner similar to that reported by <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Shjerve <em>et al.</em> (2014)</a>. Briefly, a single dried fungal plug was placed on a V8-PDA plate and allowed to grow for 5 d, after which the culture underwent a 24-hr light and dark cycle to induce sporulation. Spores were then inoculated into Fries medium [5 g (NH<sub>4</sub>)<sub>2</sub>C<sub>4</sub>H<sub>4</sub>O<sub>6</sub>, 1 g NH<sub>4</sub>NO<sub>3</sub>, 0.5 g MgSO<sub>4</sub>•7H<sub>2</sub>O, 1.3 g KH<sub>2</sub>PO<sub>4</sub>, 5.48 g K<sub>2</sub>HPO<sub>4</sub>•3H<sub>2</sub>O, 30 g sucrose, 1 g yeast extract, 2 ml trace element stock solution (167 mg LiCl, 107 mg CuCl•H<sub>2</sub>O, 34 mg H<sub>2</sub>MoO<sub>4</sub>, 72 mg MnCl<sub>2</sub>•4H<sub>2</sub>O, 80 mg CoCl<sub>2</sub>•4H<sub>2</sub>O, ddH<sub>2</sub>O to 1 liter)] and allowed to grow for 5 d. Five-day cultures were blended and inoculated into new Fries medium, and allowed to grow for another 24 hr before harvesting.</p><p id="p-9">Harvested tissue was ground to a fine powder under liquid nitrogen with a mortar and pestle, and then placed in a 50 ml conical tube. Next, 25 ml of Qiagen RLT buffer and 150 µl RNAse A at 20 mg/ml was added to the 50 ml conical tube containing fungal tissue. The mixture was homogenized by pipetting and vortexing until well mixed, and incubated at 65° for 45 min, mixing at 15-min intervals. The 50 ml conical tube was centrifuged at 3166 × <em>g</em> for 15 min and the resulting supernatant was split between two Oakridge tubes. Equal volumes of 25:24:1 phenol:chloroform:isoamyl alcohol was added to each tube and mixed gently and thoroughly by rocking. After mixing, tubes were centrifuged in a fixed angle rotor for 20 min at 13,000 × <em>g</em> at room temperature. The aqueous layer was drawn off, so as not to disturb the middle phase, and placed in a new 50 ml conical tube; 0.4 volumes of sodium acetate and an equal volume of isopropyl alcohol were added to the 50 ml tube, and the solution was mixed and incubated at room temperature for 30 min to precipitate the DNA. DNA was removed from the 50 ml conical tube using a glass hook and placed in a clean weigh boat and subsequently rinsed twice using 2–5 ml of freshly prepared 70% ethanol. Ethanol was then pulled off via pipetting and the DNA was moved to a 5 ml tube and placed in a lyophilizer for 30 min to dry. DNA was rehydrated with 1 ml of molecular biology grade water, and incubated at 4° overnight. DNA was quantified using a Qubit (ThermoFischer Scientific), and sample concentration was adjusted to 1 µg/ml.</p></div><div id="sec-3" class="subsection"><h3>Genomic sequencing and de novo assembly</h3><p id="p-10">Genomic DNA was shipped on dry ice to NCGR (Santa Fe, NM) for library preparation and sequencing. Whole-genome shotgun sequencing was performed at NCGR using the PacBio RSII instrument with a 20 kb size selected library and current P6-C4 chemistry. A total of 14 SMRT cells was sequenced for <em>P. teres</em> f. <em>teres</em> isolate 0-1.</p><p id="p-11">Raw reads in the form of FASTQ files were input into the Canu assembler (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Koren <em>et al.</em> 2017</a>) under default parameters for correction, trimming, and assembly with a genome size estimate of 42 Mb. A second iteration of genome assembly was then done with the same parameters, but a larger genome estimate of 46.5 Mb based on the first draft assembly. Pilon v1.21 (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Walker <em>et al.</em> 2014</a>) was used to polish the 0-1 assembly to improve local base calling accuracy. Pilon takes the reference assembly FASTA file and a BAM file of the aligned reads as an input to identify miscalled bases, small indels, or large structural misassemblies for correction, and outputs a new FASTA file containing the polished genome assembly.</p></div><div id="sec-4" class="subsection"><h3>Genetic mapping and genome scaffolding</h3><p id="p-12">A genetic linkage map was created from the biparental population 0–1 × 15A consisting of 120 progeny isolates, 78 isolates were obtained from <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lai <em>et al.</em> (2007)</a>, and an additional 42 isolates were obtained from another 0–1 × 15A cross following the methods described in <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Koladia <em>et al.</em> (2017a)</a>. Single nucleotide polymorphic (SNP) markers were identified following a RAD-GBS pipeline also described in <a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Koladia <em>et al.</em> (2017a)</a>. A total of 10 progeny isolates were dropped from the analysis due to large amounts of missing data (&gt;75%), and five isolates were dropped after being identified as parental clones, bringing the total to 105 progeny isolates. Linkage mapping was conducted in MapDisto v1.7.9 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Lorieux 2012</a>) as described in <a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Koladia <em>et al.</em> (2017a)</a> using a LOD threshold of 5.0.</p><p id="p-13"><em>P. teres</em> f. <em>teres</em> isolate 0-1 assembled contigs were scaffolded using ALLMAPS (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Tang <em>et al.</em> 2015</a>). ALLMAPS takes assembled contigs and generates scaffolds based on coordinates from genetic linkage maps, optical maps, or syntenic maps. To scaffold the 0-1 assembly, linkage maps from the biparental populations 0–1 × 15A and the recently published FGOH04-Ptt21 × BB25 population (<a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Koladia <em>et al.</em> 2017a</a>) were input into ALLMAPS and merged to a single coordinate BED file. The merged coordinate BED file was input with the 0-1 genome FASTA file for scaffolding under default parameters within ALLMAPS.</p></div><div id="sec-5" class="subsection"><h3>RNA sequencing and assembly</h3><p id="p-14">Cultures and inoculum of <em>P. teres</em> f. <em>teres</em> isolate 0-1 were prepared as previously described (<a id="xref-ref-22-7" class="xref-bibr" href="#ref-22">Koladia <em>et al.</em> 2017a</a>). Seeds of barley line “Tifang” were sown into a 96-conetainer rack with a border of Tradition barley and grown under greenhouse conditions for ∼2 wk. For each RNAseq sample time point, five individual containers containing two seedlings each were selected for inoculation. The second fully extended leaf of each seedling was taped flat to a 24 × 30 cm plastic surface so as to provide a flat surface to evenly coat sample leaves with inoculum. Following inoculation, plants were placed into a lighted mist chamber for 24 hr with 100% relative humidity. After 24 hr, plants were moved to a growth chamber with a temperature of 21° and a 12-hr photoperiod. Samples were collected by punching circular leaf discs from predesignated regions of the leaf using a sterile hole punch. Each leaf was punched a total of five times equating to 50 tissue samples per collected time point. Tissue was immediately flash frozen in liquid nitrogen and stored at −80° until RNA extraction. Liquid cultures of isolate 0-1 were prepared by incubating collected fungal spores in 75 ml of Fries medium (<a id="xref-ref-22-8" class="xref-bibr" href="#ref-22">Koladia <em>et al.</em> 2017a</a>) for 5 d. Tissue was harvested, rinsed, flash frozen in liquid nitrogen, and stored at −80° until RNA extraction. RNA sequencing was done using <em>in planta</em> time points of 48, 72, and 96 hr postinoculation, and a sample from pure culture. Both the pure culture and <em>in planta</em> samples were collected in three replicates. mRNA from each sample was extracted using the mRNA Direct Kit (ThermoFisher Scientific) following the manufacturer’s protocol. RNAseq library preparation was done with the Illumina Truseq v.3 kit following the manufacturer’s protocol. Quality and fragment size distribution of the prepared libraries were examined using an Agilent DNA chip on a bioanalyzer (Agilent, Santa Clara, CA). Libraries were sequenced on an Illumina Nextseq at the USDA-ARS Small Grains Genotyping Center (Fargo, ND) to produce 150 bp single-end reads.</p><p id="p-15">The output of the sequencing run was parsed with the open source program bcl2fastq2 (Illumina) and reads were input to FastQC for quality inspection (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andrews 2011</a>). Trimmomatic was used to trim raw reads using the parameters HEADCROP:15, ILLUMINACLIP:2:30:10 with Illumina adapter and index sequences provided, and SLIDINGWINDOW:4:15 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bolger <em>et al.</em> 2014</a>). Trimmed reads were aligned to the 0-1 genome sequence using HISAT, and aligned reads were assembled and analyzed using StringTie following the protocol laid out in <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Pertea <em>et al.</em> (2016)</a>. Briefly, reads were aligned to the genome using HISATv2 with the option “max-intronlen=3000,” which is suggested for fungal genomes (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ter-Hovhannisyan <em>et al.</em> 2008</a>). Aligned reads were output in SAM format, and converted to sorted BAM files. These BAM files were input into StringTie for assembly of transcripts (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Pertea <em>et al.</em> 2016</a>).</p></div><div id="sec-6" class="subsection"><h3>Genome annotation</h3><p id="p-16">Gene models were determined using the Maker2 pipeline (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Holt and Yandell 2011</a>). The Maker2 pipeline incorporates multiple sources of evidence, and leverages these to create the most accurate gene models possible. <em>Ab initio</em> annotations were provided through Augustus with the model organisms <em>Neurospora crassa</em> training set (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Stanke and Morgenstern 2005</a>) and Genemark-ES v.2 (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Ter-Hovhannisyan <em>et al.</em> 2008</a>), which contains a self-training algorithm. Assembled RNAseq transcripts were also provided to the pipeline as a GFF3 file along with external protein evidence from the closely related species <em>Pyrenophora tritici-repentis</em> (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Manning <em>et al.</em> 2013</a>) and the current NCBI <em>P. teres</em> f. <em>teres</em> 0-1 annotation (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Ellwood <em>et al.</em> 2010</a>) in fasta format. Options within the Maker2 pipeline that were used in the first iteration of annotation were “est2genome=1” and “protein2genome=1,” which instruct the pipeline to use evidence from RNAseq data and BLAST results of supplied proteins to build gene models. These gene models were then used to train the <em>ab initio</em> annotation program SNAP (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Korf 2004</a>), and the pipeline was rerun with the addition of a SNAP training file specific to the 0-1 genome. The output from this second Maker2 pipeline run was then used to retrain SNAP a second time, and subsequently rerun to further refine gene models (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Holt and Yandell 2011</a>). To evaluate the quality of the updated 0-1 assembly gene models, RNAseq transcript coverage was calculated using BEDtools “coverage” (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Quinlan and Hall 2010</a>) with Maker2 gene models and aligned transcripts input in BED format. RNAseq evidence for a gene model was defined as having &gt;50% transcript coverage of a gene model.</p><p id="p-17">To evaluate the completeness of the assembly’s annotated gene models, the program BUSCO was applied to the updated 0-1 genome assembly and annotation. BUSCO utilizes sets of core genes in taxon-specific databases to evaluate the relative completeness of the assembly and annotation. For the purposes of assessing the 0-1 genome, the BUSCO database for Ascomycota was used, as it was the most specific data set in the BUSCO databases relating to <em>P. teres</em> f. <em>teres</em>. The Ascomycota data set contains 1315 genes curated from 75 different Ascomycota species (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Simão <em>et al.</em> 2015</a>). To compare the completeness of the 0-1 updated assembly, three other closely related species were downloaded from NCBI and evaluated with BUSCO; <em>P. tritici repentis Pt-1C-BFP</em> (ASM14998v1), <em>Parastagonospora nodorum SN15</em> (ASM14691v2), and <em>Leptosphaeria maculans JN3</em> (ASM23037v1).</p><p id="p-18">RepeatModeler v1.0.11 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Smit <em>et al.</em> 2013–2015</a>) was used to <em>de novo</em> annotate repetitive elements in the genome in order to create a custom <em>P. teres</em> f. <em>teres</em> repeat library. The RepeatModeler <em>P. teres</em> f. <em>teres</em> repeat library was input into RepeatMasker (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Smit and Hubley 2008-2015</a> alongside the current release of Repbase (v22.10) (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bao <em>et al.</em> 2015</a>), to soft mask identified repetitive elements and output a final annotation of repetitive elements identified in the newly assembled 0-1 <em>P. teres</em> f. <em>teres</em> genome. The “buildSummary.pl” RepeatMasker script was applied to gather summary statistics for downstream analysis of repetitive elements.</p></div><div id="sec-7" class="subsection"><h3>Secretome, effectorome, GC content structure, and repetitive analysis</h3><p id="p-19">Secreted proteins were identified using SignalP v4 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Petersen <em>et al.</em> 2011</a>) and output as mature proteins, lacking the signal sequence. Mature secreted proteins were input into EffectorP v1 (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sperschneider <em>et al.</em> 2016</a>) to identify putative effectors in the updated 0-1 annotation set.</p><p id="p-20">OcculterCut v1 (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Testa <em>et al.</em> 2016</a>) is a tool used to identify GC content patterns in genomes. OcculterCut outputs a number of useful statistics, which include the average sizes of GC-rich and poor regions, as well as average gene densities within each region. OcculterCut was run with the updated 0-1 genome input as a FASTA file, the updated 0-1 annotation in GFF3 format, and default run parameters. RIPCAL (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hane and Oliver 2008</a>) was implemented to scan for evidence of repeat-induced point mutations (RIP) within the repetitive content of the 0-1 genome. Repeat family sequences of &gt;400 bp of the five most common repeat families were subjected to RIPCAL to determine RIP dominance. The TpA/ApT RIP index and the (CpA+TpG)/(ApC+GpT) RIP index were additionally computed and compared to the RIP indices of a randomly parsed set of sequences from the 0-1 genome. Random sequences of similar size were parsed using the custom Perl script supplied in <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Derbyshire <em>et al.</em> (2017)</a> for a total of 50 sequences.</p></div><div id="sec-8" class="subsection"><h3>Whole-genome alignment</h3><p id="p-21">Contigs from the first assembly of <em>P. teres</em> f. <em>teres</em> isolate 0-1 (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Ellwood <em>et al.</em> 2010</a>) were aligned to the newly assembled reference with the alignment program “nucmer” within the MUMmer v3.0 package (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Delcher <em>et al.</em> 2003</a>) using the “mum” option to compute maximal unique matches in the references and query sequences. Alignments were converted to a BED file for downstream analysis. Genome coverage for the 12 reference 0-1 scaffolds were calculated using Bedtools v2.26.0 “coverage” (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Quinlan and Hall 2010</a>) to output a percent coverage of each scaffold.</p><p id="p-22">Genomic windows consisting of 1000 bp of the reference 0-1 assembly were calculated using Bedtools “makewindows” (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Quinlan and Hall 2010</a>) in order to compare coverage statistics relative to repeat regions of the genome. Genomic regions containing low- to no-coverage were identified as having &lt;200 bp of overlap within a 1000 bp genomic region. These low- to no-coverage regions were then compared to regions of the genome harboring repetitive element using Bedtools v2.26.0 “coverage” (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Quinlan and Hall 2010</a>). Genomic regions containing low- to no-coverage, and also having 50% of the region covered by a repetitive element, were output.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-23">Sequence and annotation data are available at NCBI GenBank under BioProject PRJNA392275. Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300196/-/DC1/FileS1.pdf">File S1</a> contains figures representing marker positions along scaffolded <em>P. teres</em> f. <em>teres</em> isolate 0-1 assembly contigs. Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300196/-/DC1/FileS1.pdf">File S1</a> contains RIPCAL outputs of RIP dominance observed in the five most numerous repeat families annotated in the 0-1 assembly. Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300196/-/DC1/FileS1.pdf">File S1</a> contains summary data of RepeatModeler annotations. Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300196/-/DC1/FileS1.pdf">File S1</a> contains calculated RIP indices of the five move numerous repeat families and a randomly parsed set of genomic sequences.</p></div></div><div class="section results-discussion" id="sec-10"><h2 class="">Results and Discussion</h2><div id="sec-11" class="subsection"><h3>Sequencing and de novo genome assembly</h3><p id="p-24">PacBio SMRT sequencing of <em>P. teres</em> f. <em>teres</em> isolate 0-1 generated a total of 1,148,507 reads from the 14 SMRT cells sequenced, with an average read length of 8051 bp. A total of 9,246,774,161 bp were obtained, equating to ∼200× coverage of the 0-1 genome.</p><p id="p-25">Assembling this data using the Canu assembler yielded a fairly contiguous assembly, with 85 total contigs and one contig representing the mitochondrial genome (86 total contigs). The total size of the assembly was ∼46.5 Mb, with an N50 of 1,730,401 bp, and an L50 of 11 contigs. This assembly provides a drastic improvement from the previous assembly based on the quality metrics summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The use of long-read technology allowed for the assembly of low complexity, repeat dense regions, which are difficult to assemble using short-read technologies. The increased resolution of the genome will aid in the investigation of evolutionarily active regions, which are often repeat-rich and harbor important genes related to pathogen adaptation.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487913/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487913/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Updated 0-1 assembly summary statistics compared to previous 0-1 assembly</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Genetic mapping and genome scaffolding</h3><p id="p-31">Ion Torrent RAD-GBS sequencing generated reads for parental isolates 0-1 and 15A, and the 120 progeny. Reads were aligned to the updated <em>P. teres</em> f. <em>teres</em> 0-1 genome, and a total of 284 SNP markers was identified. SNPs were input into MapDisto v1.7.9 following a filtering process for genetic mapping. A total of 17 linkage groups (LGs) was obtained, with sizes that ranged from 9.69 to 159.44 cM, and totaling a genetic distance of 987.36 cM.</p><p id="p-32">The 0-1 × 15A genetic map appears to be at low resolution by comparison to the FGOH04 × BB25 genetic map consisting of 16 LGs with 685 SNP markers (370 nonredundant markers) spanning a genetic distance of 1905.81 cM (<a id="xref-ref-22-9" class="xref-bibr" href="#ref-22">Koladia <em>et al.</em> 2017a</a>). To increase the resolution of the genetic map used for scaffolding the updated 0-1 assembly, the 0-1 × 15A SNPs and the FGOH04 × BB25 SNPs were combined into a single coordinate BED file using ALLMAPS (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Tang <em>et al.</em> 2015</a>).</p><p id="p-33">Genome scaffolding using ALLMAPS (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Tang <em>et al.</em> 2015</a>) was accomplished with the combined 0-1 × 15A and FGOH04 × BB25 genetic maps (<a id="xref-ref-22-10" class="xref-bibr" href="#ref-22">Koladia <em>et al.</em> 2017a</a>) input as a single merged coordinate BED file and the updated 0-1 genome assembly in FASTA format. Scaffolding resulted in 12 scaffolds containing 43 of the 85 contigs in the updated 0-1 assembly, and represents 91.8% of the total base pairs in the 0-1 assembly (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Marker density across the 12 scaffolds equaled 15.3 markers per Mbp, with 652 of the markers uniquely anchored in the scaffolds (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Unscaffolded contigs ranged in size from ∼28 to ∼367 kb, and represented 8.2% of the 0-1 genome. ALLMAPS scaffolding statistics are summarized in Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300196/-/DC1/FileS1.pdf">File S1</a> and <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> contains graphics depicting marker locations across the scaffolded assembly.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488063/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488063/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">ALLMAPS genome scaffolding statistics</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Genome annotation and assessment</h3><p id="p-38">Genome annotation via the Maker2 pipeline yielded 11,541 genes or pseudogenes (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Evidence for the updated 0-1 gene annotations were derived from either imported protein sequences from the current genome annotation of 0-1 (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Ellwood <em>et al.</em> 2010</a>), or from the closely related species <em>P. tritici-repentis</em> (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Manning <em>et al.</em> 2013</a>), as well as <em>ab initio</em> annotations. RNAseq evidence was present for 72.9% (8414) of the gene annotations. illustrating the high level of confidence for these gene models. Many of the gene models without RNAseq evidence are likely to be involved in the saprotrophic stage of the <em>P. teres</em> f. <em>teres</em> life cycle due to the samples being collected only from culture and early <em>in planta</em> time points.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488051/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488051/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Gene annotation summary statistics</span><div class="sb-div caption-clear"></div></div></div><p id="p-41">BUSCO is a tool for evaluating genome and annotation completeness based on the presence of core genes in curated taxon-specific databases (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Simão <em>et al.</em> 2015</a>). For the purposes of assessing the completeness of the updated 0-1 reference genome, BUSCO was run utilizing the Ascomycota core gene set, which is comprised of a total of 1315 core genes. Results from running BUSCO on the updated 0-1 genome resulted in 97.8% of genes being present from the core Ascomycota gene set. Of the genes present relating to the core Ascomycota gene set, 97.7% were complete, 0.8% were fragmented, and 1.6% were missing. These results compare well with four other previously sequenced Dothideomycetes including <em>P. tritici-repentis Pt-1C-BFP</em> (97.6%), <em>P. nodorum SN15</em> (97.4%), and <em>L. maculans JN3</em> (97.6%) (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487919/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487919/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">BUSCO analysis on assembly and annotations</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Functional analysis and evidence of a two-speed genome</h3><p id="p-46">The host–pathogen interaction is directly modulated by a suite of pathogen-secreted proteins known as effectors (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Franceschetti <em>et al.</em> 2017</a>). Effectors work to modulate plant cell physiology to facilitate pathogen infection. SignalP v4 (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Petersen <em>et al.</em> 2011</a>) predicted 1002 secretion signals from the 11,541 <em>P. teres</em> f. <em>teres</em> isolate 0-1 annotated genes (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>), representing the secretome of 0-1. Mature amino acid sequences (lacking secretion signals) were input into EffectorP v1 (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Sperschneider <em>et al.</em> 2016</a>) to further differentiate the effectorome from within the predicted secretome, resulting in a total of 167 proteins predicted to be effectors (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). These predicted effectors are likely to be important in the barley–<em>P. teres</em> f. <em>teres</em> interaction.</p><p id="p-47">Fungal plant pathogens have been shown to have bipartite compartmentalized genomes comprised of gene-rich, repeat-sparse, regions and gene-sparse, repeat-rich regions. This genomic architecture represents the “two-speed genome” model, which has been observed in a number of plant pathogens (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Dong <em>et al.</em> 2015</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Faino <em>et al.</em> 2016</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Laurent <em>et al.</em> 2017</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rouxel and Balesdent 2017</a>). Genes in close proximity to repeat-rich and gene-sparse regions have been shown to undergo higher rates of positive selection, indicating these compartments are evolutionarily active (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Raffaele <em>et al.</em> 2010</a>). A fungal genome defense mechanism against duplication events known as RIP may be a contributing factor in the development of AT-rich genome regions. Evolution rates of genes in close proximity to AT-rich regions could be increased through the aid of RIP (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lo Presti <em>et al.</em> 2015</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Testa <em>et al.</em> 2016</a>). The program OcculterCut v1 (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Testa <em>et al.</em> 2016</a>) was used to examine the overall GC content of the genome. The output of this analysis resulted in two distinct genome categories: one with high GC content (41–100%) and the other with low GC content (0–41%) (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). The high GC content portion constituted ∼75% of the genomic content, with an average gene density of 306 genes/Mb, in contrast to the low GC content region, which constitutes 24.9% of the genome with an average gene density of 72.6 genes/Mb (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>). This clear genomic segmentation is a classic representation of the two-speed fungal genome often seen in plant pathogens (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Dong <em>et al.</em> 2015</a>; <a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Testa <em>et al.</em> 2016</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Thomma <em>et al.</em> 2016</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/1/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="OcculterCut v1 GC% plot of the P. teres f. teres 0-1 genome. A bimodal distribution is observed in which genome segments fall into the high GC% category (41&#x2013;100%) or the low GC% category (0&#x2013;41%). The OcculterCut calculated cut off for distinguishing low and high GC% content in 0-1 is 41%." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1711237162" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;OcculterCut v1 GC% plot of the &lt;em&gt;P. teres&lt;/em&gt; f. &lt;em&gt;teres&lt;/em&gt; 0-1 genome. A bimodal distribution is observed in which genome segments fall into the high GC% category (41&#x2013;100%) or the low GC% category (0&#x2013;41%). The OcculterCut calculated cut off for distinguishing low and high GC% content in 0-1 is 41%.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/1/F1.medium.gif" width="440" height="307"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/1/1/F1.medium.gif" width="440" height="307"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/1/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/1/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487884" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-48" class="first-child">OcculterCut v1 GC% plot of the <em>P. teres</em> f. <em>teres</em> 0-1 genome. A bimodal distribution is observed in which genome segments fall into the high GC% category (41–100%) or the low GC% category (0–41%). The OcculterCut calculated cut off for distinguishing low and high GC% content in 0-1 is 41%.</p><div class="sb-div caption-clear"></div></div></div><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488066/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488066/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">OcculterCut analysis of <em>P. teres</em> f. <em>teres</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-54">The OcculterCut analysis supports the repetitive analysis output through the RepeatModeler/RepeatMasker pipeline. This repetitive analysis identified 26.7% of the 0-1 genome as being interspersed repeat elements, and an additional 5.0% of simple repeats. This equates to roughly 32% of the genome being comprised of repetitive elements—a greater number compared to the closely related species <em>P. tritici-repentis</em> (16.7%) and <em>P. nodorum</em> (4.52%) (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Manning <em>et al.</em> 2013</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Syme <em>et al.</em> 2016</a>). The most numerous repetitive element classes annotated were the LTR-Gypsy elements. comprising 9.28% of the genome. and DNA-TcMar-Fot1 elements. at 5.38% of the genome. with an additional 7.81% of the genome belonging to unclassified transposable elements (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300196/-/DC1/FileS1.pdf">File S1</a>).</p><p id="p-55">RIP is a genomic defense mechanism against transposons that has been identified in a number of fungal species (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Singer and Selker 1995</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Dean <em>et al.</em> 2005</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Idnurm and Howlett 2003</a>; <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Manning <em>et al.</em> 2013</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Syme <em>et al.</em> 2016</a>). RIP involves C:G nucleotide transitions to T:A nucleotides, and affects sequences with ∼80% identity over at least 400 bp in length, creating a bias toward TpA dinucleotides over CpA dinucleotides in RIP-affected areas (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hane and Oliver 2008</a>). RIPCAL RIP indices were calculated for the top five annotated repeat families (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300196/-/DC1/FileS1.pdf">File S1</a>), and compared to a set of randomly extracted DNA sequences of the same size range of the 0-1 genome. RIP indices that show evidence of RIP were defined as values of TpA/ApT &gt; 2.0 and/or (CpA+TpG)/(ApC+GpT) &lt; 0.7 (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Galagan <em>et al.</em> 2003</a>). Using these criteria, all five repeat families show evidence of RIP with (CpA+TpG)/(ApC+GpT) &lt; 0.7, but none of the five repeat families show evidence of RIP with TpA/ApT &gt; 2.0 (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300196/-/DC1/FileS1.pdf">File S1</a>). RIPCAL alignment “degenerative consensus” analysis of the five repeat families indicated a RIP dominance of CpT to TpA transitions (TpG to TpA in the reverse complement) and CpT to TpT transitions (TpG to TpT in the reverse complement) (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300196/-/DC1/FileS1.pdf">File S1</a>). Given the indication of RIP-affected sequences from only one of the common indices, and the high degree of homology observed between members of the five repeat families examined, it would seem that RIP is not an efficient process in the <em>P. teres</em> f. <em>teres</em> isolate 0-1 genome. This is further supported by the increased repetitive content of the 0-1 genome relative to closely related species, and reflects similar results observed in <em>P. tritici-repentis</em> (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Manning <em>et al.</em> 2013</a>), which concluded that, if RIP is functional, the efficiency is low.</p></div><div id="sec-15" class="subsection"><h3>Whole-genome alignment</h3><p id="p-56">Using a combination of MUMmer v3.0 (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Delcher <em>et al.</em> 2003</a>) and Bedtools v2.26.0 (<a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Quinlan and Hall 2010</a>), whole genome alignments were calculated and compared between the first draft assembly of <em>P. teres</em> f. <em>teres</em> isolate 0-1 and the newly assembled reference genome of 0-1. Alignments between the first draft assembly and the 12 reference scaffolds resulted in coverages ranging from 60.5 to 80.1% for the 12 reference scaffolds, equating to an average of 27.2% missing sequence between the first draft assembly and the current 12 reference scaffolds. This amount of missing data correlates well with the amount of repetitive elements detected in the genome (32%), and, in fact, 67.1% of the 27.2% missing sequence of the first draft assembly contains an annotated repeat element in the new reference genome of isolate 0-1 (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300196/-/DC1/FileS1.pdf">File S1</a>).</p><p id="p-57">The correlation between missing sequence and repetitive elements adds support to previous observations that short-read technologies struggle to span low complexity regions (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Ekblom and Wolf 2014</a>), and highlights the usefulness of long-read technologies such as PacBio (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Goodwin <em>et al.</em> 2016</a>). With regards to host–pathogen interactions, long-read technologies will aid in understanding effector genes, which can be difficult to identify as they are known to associate with low complexity repeat regions (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Thomma <em>et al.</em> 2016</a>). Long-read technologies present the best method for sequencing and assembling the genomes of fungal plant pathogen species with the goal of understanding the host–pathogen interactions.</p></div><div id="sec-16" class="subsection"><h3>Conclusion</h3><p id="p-58">Here, we present an updated assembly and annotation of the barley pathogenic fungus <em>P. teres</em> f. <em>teres</em> reference isolate 0-1. This improved assembly and annotation provides a higher resolution assembly of the <em>P. teres</em> f. <em>teres</em> 0-1 reference genome and annotation, which now includes a large proportion of the repetitive content within the genome that have been shown to be of evolutionary importance to plant pathogens (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Dong <em>et al.</em> 2015</a>). This data set will be particularly useful in investigating effector genes that have been reported to reside in, and proximal to, evolutionarily active repetitive regions of the genome.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-59">The authors thank Danielle Holmes for technical assistance and expertise, and Dr. Zhaohui Liu for comments on the manuscript. Research was supported by funding from The North Dakota Barley Council. Mention of trade names or commercial products in this publication is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the United States Department of Agriculture (USDA). USDA is an equal opportunity provider and employer.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-21"><p id="p-60">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300196/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300196/-/DC1</a>.</p></li><li class="fn" id="fn-22"><p id="p-61"><em>Communicating editor: S. Scofield</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-08-24"><span class="received-label">Received </span>August 24, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-11-20"><span class="accepted-label">Accepted </span>November 20, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Wyatt <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.1" data-doi="10.1007/s00122-008-0860-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abu Qamar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A region of barley chromosome 6H harbors multiple major genes associated with net type net blotch resistance</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">1261</span>–<span class="cit-lpage">1270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAbu%2BQamar%26rft.auinit1%253DM.%26rft.volume%253D117%26rft.issue%253D8%26rft.spage%253D1261%26rft.epage%253D1270%26rft.atitle%253DA%2Bregion%2Bof%2Bbarley%2Bchromosome%2B6H%2Bharbors%2Bmultiple%2Bmajor%2Bgenes%2Bassociated%2Bwith%2Bnet%2Btype%2Bnet%2Bblotch%2Bresistance.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0860-x%26rft_id%253Dinfo%253Apmid%252F18712341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0860-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18712341&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260526700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.1.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andrews</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source">FastQC: A Quality Control Tool for High Throughput Sequence Data</span>. <span class="cit-publ-name">Babraham Institute</span>, <span class="cit-publ-loc">Cambridge, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.3" data-doi="10.1186/s13100-015-0041-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kojima</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohany</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">2015 Repbase update, a database of repetitive elements in eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Mob. DNA</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMob.%2BDNA%26rft.volume%253D6%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13100-015-0041-9%26rft_id%253Dinfo%253Apmid%252F26045719%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13100-015-0041-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26045719&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.4" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.5" data-doi="10.1534/g3.117.043265"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carlsen</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neupane</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wyatt</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Characterizing the <em>Pyrenophora teres</em> f. <em>maculata</em>–barley interaction using pathogen genetics.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">2615</span>–<span class="cit-lpage">2626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.043265%26rft_id%253Dinfo%253Apmid%252F28659291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvOC8yNjE1IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvOC8xLzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.6" data-doi="10.1016/j.cell.2006.02.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chisholm</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coaker</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Day</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staskawicz</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Host–microbe interactions: shaping the evolution of the plant immune response.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">814</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DChisholm%26rft.auinit1%253DS.%2BT.%26rft.volume%253D124%26rft.issue%253D4%26rft.spage%253D803%26rft.epage%253D814%26rft.atitle%253DHost-microbe%2Binteractions%253A%2Bshaping%2Bthe%2Bevolution%2Bof%2Bthe%2Bplant%2Bimmune%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.02.008%26rft_id%253Dinfo%253Apmid%252F16497589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.02.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16497589&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237240900022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.7" data-doi="10.1038/nature03449"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talbot</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebbole</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farman</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The genome sequence of the rice blast fungus <em>Magnaporthe grisea</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">434</span>: <span class="cit-fpage">980</span>–<span class="cit-lpage">986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDean%26rft.auinit1%253DR.%2BA.%26rft.volume%253D434%26rft.issue%253D7036%26rft.spage%253D980%26rft.epage%253D986%26rft.atitle%253DThe%2Bgenome%2Bsequence%2Bof%2Bthe%2Brice%2Bblast%2Bfungus%2BMagnaporthe%2Bgrisea.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature03449%26rft_id%253Dinfo%253Apmid%252F15846337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature03449&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15846337&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228524600029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Delcher</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillippy</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Using MUMmer to identify similar regions in large sequence sets.</span> <abbr class="cit-jnl-abbrev">Curr. Protoc. Bioinformatics</abbr> <span class="cit-vol">Chapter 10</span>: <span class="cit-fpage">Unit 10.3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BBioinformatics%26rft.volume%253D10%26rft.spage%253DUnit103%26rft_id%253Dinfo%253Apmid%252F18428693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18428693&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Derbyshire</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denton-Giles</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegedus</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seifbarghy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rollins</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The complete genome sequence of the phytopathogenic fungus <em>Sclerotinia sclerotiorum</em> reveals insights into the genome architecture of broad host range pathogens.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">618</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft.volume%253D9%26rft.spage%253D593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.10" data-doi="10.1016/j.gde.2015.09.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raffaele</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamoun</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The two-speed genomes of filamentous pathogens: waltz with plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D35%26rft.spage%253D57%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2015.09.001%26rft_id%253Dinfo%253Apmid%252F26451981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2015.09.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26451981&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.11" data-doi="10.1111/eva.12178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ekblom</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A field guide to whole‐genome sequencing, assembly and annotation.</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1026</span>–<span class="cit-lpage">1042</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D7%26rft.spage%253D1026%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12178%26rft_id%253Dinfo%253Apmid%252F25553065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25553065&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ellwood</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Syme</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A first genome assembly of the barley fungal pathogen <em>Pyrenophora teres</em> f. <em>teres</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DEllwood%26rft.auinit1%253DS.%2BR.%26rft.volume%253D11%26rft.issue%253D11%26rft.spage%253DR109%26rft.epage%253DR109%26rft.atitle%253DA%2Bfirst%2Bgenome%2Bassembly%2Bof%2Bthe%2Bbarley%2Bfungal%2Bpathogen%2BPyrenophora%2Bteres%2Bf.%2Bteres.%26rft_id%253Dinfo%253Apmid%252F21067574%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21067574&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faino</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seidl</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Datema</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Berg</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janssen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Single-molecule real-time sequencing combined with optical mapping yields completely finished fungal genome.</span> <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e00936</span>–<span class="cit-lpage">e15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D6%26rft.spage%253De00936%26rft_id%253Dinfo%253Apmid%252F26286689%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26286689&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.14" data-doi="10.1101/gr.204974.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faino</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seidl</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi-Kunne</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pauper</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Berg</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Transposons passively and actively contribute to evolution of the two-speed genome of a fungal pathogen.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1091</span>–<span class="cit-lpage">1100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.204974.116%26rft_id%253Dinfo%253Apmid%252F27325116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjI2LzgvMTA5MSI7czo0OiJhdG9tIjtzOjE1OiIvZ2dnLzgvMS8xLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Franceschetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maqbool</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiménez-Dalmaroni</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pennington</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamoun</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Effectors of filamentous plant pathogens: commonalities amid diversity.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">e00066</span>–<span class="cit-lpage">e16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.volume%253D81%26rft.spage%253De00066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.16" data-doi="10.1038/nature01554"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galagan</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calvo</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borkovich</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Read</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The genome sequence of the filamentous fungus <em>Neurospora crassa</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">422</span>: <span class="cit-fpage">859</span>–<span class="cit-lpage">868</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGalagan%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D422%26rft.issue%253D6934%26rft.spage%253D859%26rft.epage%253D868%26rft.atitle%253DThe%2Bgenome%2Bsequence%2Bof%2Bthe%2Bfilamentous%2Bfungus%2BNeurospora%2Bcrassa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01554%26rft_id%253Dinfo%253Apmid%252F12712197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01554&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12712197&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182432600046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.17" data-doi="10.1038/nrg.2016.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McPherson</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCombie</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Coming of age: ten years of next-generation sequencing technologies.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D17%26rft.spage%253D333%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2016.49%26rft_id%253Dinfo%253Apmid%252F27184599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg.2016.49&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27184599&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.18" data-doi="10.1186/1471-2105-9-478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">RIPCAL: a tool for alignment-based analysis of repeat-induced point mutations in fungal genomic sequences.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">478</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHane%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D9%26rft.spage%253D478%26rft.epage%253D478%26rft.atitle%253DRIPCAL%253A%2Ba%2Btool%2Bfor%2Balignment-based%2Banalysis%2Bof%2Brepeat-induced%2Bpoint%2Bmutations%2Bin%2Bfungal%2Bgenomic%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-478%26rft_id%253Dinfo%253Apmid%252F19014496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19014496&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.19" data-doi="10.1186/1471-2105-12-491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yandell</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHolt%26rft.auinit1%253DC.%26rft.volume%253D12%26rft.spage%253D491%26rft.epage%253D491%26rft.atitle%253DMAKER2%253A%2Ban%2Bannotation%2Bpipeline%2Band%2Bgenome-database%2Bmanagement%2Btool%2Bfor%2Bsecond-generation%2Bgenome%2Bprojects.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-491%26rft_id%253Dinfo%253Apmid%252F22192575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22192575&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.20" data-doi="10.1016/S1087-1845(02)00588-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Idnurm</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howlett</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Analysis of loss of pathogenicity mutants reveals that repeat-induced point mutations can occur in the Dothideomycete <em>Leptosphaeria maculans</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DIdnurm%26rft.auinit1%253DA.%26rft.volume%253D39%26rft.issue%253D1%26rft.spage%253D31%26rft.epage%253D37%26rft.atitle%253DAnalysis%2Bof%2Bloss%2Bof%2Bpathogenicity%2Bmutants%2Breveals%2Bthat%2Brepeat-induced%2Bpoint%2Bmutations%2Bcan%2Boccur%2Bin%2Bthe%2BDothideomycete%2BLeptosphaeria%2Bmaculans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1087-1845%252802%252900588-1%26rft_id%253Dinfo%253Apmid%252F12742061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1087-1845(02)00588-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12742061&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.21" data-doi="10.1038/nature05286"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dangl</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The plant immune system.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">444</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJones%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D444%26rft.issue%253D7117%26rft.spage%253D323%26rft.epage%253D329%26rft.atitle%253DThe%2Bplant%2Bimmune%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05286%26rft_id%253Dinfo%253Apmid%252F17108957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05286&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17108957&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242018300039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-other" id="cit-8.1.1.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koladia</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wyatt</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brueggeman</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">a</span> <span class="cit-article-title">Genetic analysis of virulence in the <em>Pyrenophora teres</em> f. <em>teres</em> population BB25× FGOH04Ptt-21.</span> <span class="cit-source">Fungal Genet. Biol.</span> <span class="cit-vol">107</span>: <span class="cit-fpage">12</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D107%26rft.spage%253D12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koladia</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brueggeman</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span><span class="cit-comment">b</span> <span class="cit-article-title">Genetic analysis of net form net blotch resistance in barley lines CIho5791 and Tifang against a global collection of <em>P. teres</em> f. <em>teres</em> isolates.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.24" data-doi="10.1101/gr.215087.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koren</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walenz</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergman</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation.</span> <abbr class="cit-jnl-abbrev">Genome Res</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">722</span>–<span class="cit-lpage">736</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.215087.116%26rft_id%253Dinfo%253Apmid%252F28298431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI3LzUvNzIyIjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvOC8xLzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.25" data-doi="10.1186/1471-2105-5-59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Gene finding in novel genomes.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DKorf%26rft.auinit1%253DI.%26rft.volume%253D5%26rft.spage%253D59%26rft.epage%253D59%26rft.atitle%253DGene%2Bfinding%2Bin%2Bnovel%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-5-59%26rft_id%253Dinfo%253Apmid%252F15144565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-5-59&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15144565&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiland</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steffenson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic mapping of <em>Pyrenophora teres</em> f. <em>teres</em> genes conferring avirulence on barley.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DLai%26rft.auinit1%253DZ.%26rft.volume%253D44%26rft.issue%253D5%26rft.spage%253D323%26rft.epage%253D329%26rft.atitle%253DGenetic%2Bmapping%2Bof%2BPyrenophora%2Bteres%2Bf.%2Bteres%2Bgenes%2Bconferring%2Bavirulence%2Bon%2Bbarley.%26rft_id%253Dinfo%253Apmid%252F17224286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17224286&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.27" data-doi="10.1111/mpp.12524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laurent</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spataro</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moinard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zehraoui</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">High‐resolution mapping of the recombination landscape of the phytopathogen <em>Fusarium graminearum</em> suggests two‐speed genome evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> .</span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/mpp.12524</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%2BPathol.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmpp.12524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mpp.12524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.28" data-doi="10.1111/j.1364-3703.2010.00649.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellwood</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>Pyrenophora teres</em>: profile of an increasingly damaging barley pathogen.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BPlant%2BPathol%26rft.aulast%253DLiu%26rft.auinit1%253DZ.%26rft.volume%253D12%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D19%26rft.atitle%253DPyrenophora%2Bteres%253A%2Bprofile%2Bof%2Ban%2Bincreasingly%2Bdamaging%2Bbarley%2Bpathogen.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1364-3703.2010.00649.x%26rft_id%253Dinfo%253Apmid%252F21118345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1364-3703.2010.00649.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21118345&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brueggeman</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Necrotrophic effector triggered susceptibility (NETS) underlies the barley–<em>Pyrenophora teres</em> f. <em>teres</em> interaction specific to chromosome 6H.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">188</span>–<span class="cit-lpage">200</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%2BPathol.%26rft.volume%253D16%26rft.spage%253D188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Development of expressed sequence tags (EST)-based markers for genomic analysis of a barley 6H region harboring multiple net form net blotch resistance genes.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">52</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D3%26rft.spage%253D41%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.31" data-doi="10.1146/annurev-arplant-043014-114623"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo Presti</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanver</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schweizer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fungal effectors and plant susceptibility.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPlant%2BBiol.%26rft.volume%253D66%26rft.spage%253D513%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-arplant-043014-114623%26rft_id%253Dinfo%253Apmid%252F25923844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-arplant-043014-114623&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25923844&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.32" data-doi="10.1007/s11032-012-9706-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorieux</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">MapDisto: fast and efficient computation of genetic linkage maps.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1231</span>–<span class="cit-lpage">1235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D30%26rft.spage%253D1231%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-012-9706-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-012-9706-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manning</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pandelova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilhelm</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodwin</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparative genomics of a plant-pathogenic fungus, <em>Pyrenophora tritici</em>-<em>repentis</em>, reveals transduplication and the impact of repeat elements on pathogenicity and population divergence.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D3%26rft.spage%253D41%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathre</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kushnak</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grey</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Effect of residue management on barley production in the presence of net blotch disease.</span> <abbr class="cit-jnl-abbrev">J. Prod. Agric.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BProd.%2BAgric.%26rft.volume%253D10%26rft.spage%253D323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.35" data-doi="10.1038/nprot.2016.095"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leek</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie and Ballgown.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1650</span>–<span class="cit-lpage">1667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D11%26rft.spage%253D1650%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2016.095%26rft_id%253Dinfo%253Apmid%252F27560171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2016.095&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27560171&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.36" data-doi="10.1038/nmeth.1701"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">von Heijne</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SignalP 4.0: discriminating signal peptides from transmembrane regions.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">785</span>–<span class="cit-lpage">786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DPetersen%26rft.auinit1%253DT.%2BN.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D785%26rft.epage%253D786%26rft.atitle%253DSignalP%2B4.0%253A%2Bdiscriminating%2Bsignal%2Bpeptides%2Bfrom%2Btransmembrane%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1701%26rft_id%253Dinfo%253Apmid%252F21959131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1701&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21959131&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295358000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.37" data-doi="10.1093/bioinformatics/btq033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinlan</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">I. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BEDTools: a flexible suite of utilities for comparing genomic features.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">841</span>–<span class="cit-lpage">842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq033%26rft_id%253Dinfo%253Apmid%252F20110278%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20110278&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275243500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.38" data-doi="10.1126/science.1193070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raffaele</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farrer</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cano</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Studholme</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacLean</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome evolution following host jumps in the Irish potato famine pathogen lineage.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">330</span>: <span class="cit-fpage">1540</span>–<span class="cit-lpage">1543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRaffaele%26rft.auinit1%253DS.%26rft.volume%253D330%26rft.issue%253D6010%26rft.spage%253D1540%26rft.epage%253D1543%26rft.atitle%253DGenome%2BEvolution%2BFollowing%2BHost%2BJumps%2Bin%2Bthe%2BIrish%2BPotato%2BFamine%2BPathogen%2BLineage%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1193070%26rft_id%253Dinfo%253Apmid%252F21148391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzAvNjAxMC8xNTQwIjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvOC8xLzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friesen</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brueggeman</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Association mapping utilizing diverse barley lines reveals net form net blotch seedling resistance/susceptibility loci.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">915</span>–<span class="cit-lpage">927</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rouxel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balesdent</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Life, death and rebirth of avirulence effectors in a fungal pathogen of Brassica crops, <em>Leptosphaeria maculans</em>.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">214</span>: <span class="cit-fpage">526</span>–<span class="cit-lpage">532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D214%26rft.spage%253D526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.41" data-doi="10.1016/j.fgb.2014.07.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shjerve</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faris</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brueggeman</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evaluation of a <em>Pyrenophora teres</em> f. <em>teres</em> mapping population reveals multiple independent interactions with a region of barley chromosome 6H.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">104</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D70%26rft.spage%253D104%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2014.07.012%26rft_id%253Dinfo%253Apmid%252F25093269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2014.07.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25093269&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.42" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioannidis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kriventseva</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zdobnov</span>  <span class="cit-name-given-names">E. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.1.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selker</span>  <span class="cit-name-given-names">E. U.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Genetic and epigenetic inactivation of repetitive sequences in Neurospora crassa: RIP, DNA methylation, and quelling</span>, pp. <span class="cit-fpage">165</span>–<span class="cit-lpage">177</span> in <span class="cit-source">Gene Silencing in Higher Plants and Related Phenomena in Other Eukaryotes</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Berlin</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-web" id="cit-8.1.1.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smit</span>  <span class="cit-name-given-names">A. F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubley</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008–2015</span> RepeatModeler Open-1.0. Available at: <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>. Accessed: October 30, 2017.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-web" id="cit-8.1.1.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smit</span>  <span class="cit-name-given-names">A. F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubley</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013–2015</span> RepeatMasker Open-4.0. Available at: <a href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>. Accessed: October 30, 2017</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.46" data-doi="10.1111/nph.13794"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sperschneider</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardiner</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dodds</span>  <span class="cit-name-given-names">P. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tini</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Covarelli</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">EffectorP: predicting fungal effector proteins from secretomes using machine learning.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">761</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D210%26rft.spage%253D743%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13794%26rft_id%253Dinfo%253Apmid%252F26680733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13794&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26680733&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-other" id="cit-8.1.1.47" data-doi="10.1093/nar/gki458"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgenstern</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">AUGUSTUS: a web server for gene prediction in eukaryotes that allows user-defined constraints.</span> <span class="cit-source">Nucleic acids Res.</span> <span class="cit-vol">33</span>: <span class="cit-fpage">W465</span>–<span class="cit-lpage">W467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStanke%26rft.auinit1%253DM.%26rft.volume%253D33%26rft.issue%253Dsuppl_2%26rft.spage%253DW465%26rft.epage%253DW467%26rft.atitle%253DAUGUSTUS%253A%2Ba%2Bweb%2Bserver%2Bfor%2Bgene%2Bprediction%2Bin%2Beukaryotes%2Bthat%2Ballows%2Buser-defined%2Bconstraints%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki458%26rft_id%253Dinfo%253Apmid%252F15980513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki458&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15980513&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230271400094&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Syme</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dodhia</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoll</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Comprehensive annotation of the <em>Parastagonospora nodorum</em> reference genome using next-generation genomics, transcriptomics and proteogenomics.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0147221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0147221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.49" data-doi="10.1186/s13059-014-0573-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miao</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ming</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">ALLMAPS: robust scaffold ordering based on multiple maps.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0573-1%26rft_id%253Dinfo%253Apmid%252F25583564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0573-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25583564&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.50" data-doi="10.1101/gr.081612.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ter-Hovhannisyan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lomsadze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chernoff</span>  <span class="cit-name-given-names">Y. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borodovsky</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1979</span>–<span class="cit-lpage">1990</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.081612.108%26rft_id%253Dinfo%253Apmid%252F18757608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMi8xOTc5IjtzOjQ6ImF0b20iO3M6MTU6Ii9nZ2cvOC8xLzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.51" data-doi="10.1093/gbe/evw121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Testa</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hane</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">OcculterCut: a comprehensive survey of AT-rich regions in fungal genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2044</span>–<span class="cit-lpage">2064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevw121%26rft_id%253Dinfo%253Apmid%252F27289099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evw121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27289099&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.52" data-doi="10.1016/j.fgb.2015.08.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomma</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seidl</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi-Kunne</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolton</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Mind the gap; seven reasons to close fragmented genome assemblies.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D90%26rft.spage%253D24%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2015.08.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2015.08.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.53" data-doi="10.1111/mpp.12384"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Kan</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stassen</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mosbach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Der Lee</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faino</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A gapless genome sequence of the fungus <em>Botrytis cinerea</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%2BPathol.%26rft.volume%253D18%26rft.spage%253D75%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmpp.12384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mpp.12384&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.54" data-doi="10.1371/journal.pone.0112963"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abeel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shea</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Priest</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abouelliel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e112963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112963%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112963%26rft_id%253Dinfo%253Apmid%252F25409509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112963&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25409509&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.1.55" data-doi="10.1094/PHYTO.1999.89.2.176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weiland</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steffenson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cartwright</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webster</span>  <span class="cit-name-given-names">R. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Identification of molecular genetic markers in <em>Pyrenophora teres</em> f. <em>teres</em> associated with low virulence on ‘Harbin’barley.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">176</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DWeiland%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D89%26rft.issue%253D2%26rft.spage%253D176%26rft.epage%253D181%26rft.atitle%253DIdentification%2Bof%2BMolecular%2BGenetic%2BMarkers%2Bin%2BPyrenophora%2Bteres%2Bf.%2Bteres%2BAssociated%2Bwith%2BLow%2BVirulence%2Bon%2B%2527Harbin%2527%2BBarley.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO.1999.89.2.176%26rft_id%253Dinfo%253Apmid%252F18944793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO.1999.89.2.176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18944793&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F1.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078392100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/1/1.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/7/12/4019" title="G3 Reviewer Index 2017" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/9" title="Mapping Second Chromosome Mutations to Defined Genomic Regions in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/1"><span>Volume 8 Issue 1, January 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/1.cover-source.jpg?itok=vyoqxep2" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/1" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IXASKz5XbgLBjB-GS2naOHUZW4hiD22jDn5TEnfH8Rc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/1/1" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/1/1" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/1/1" class="active" data-icon-position="" data-hide-link-title="0">Reference Assembly and Annotation of the Pyrenophora teres f. teres Isolate 0-1</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487831" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-S5gLOrhivkmLmOmG-ldRUVSJqhkQJTVVTZFi68VBxsE" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/1.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/1/1" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-OwERU3iPsfFeCze0OtiZQbGHFjL-vCX5sYaqAK95olY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/1/1" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/1.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487831" id="node487831--2" data-pisa="ggg;8/1/1" data-pisa-master="ggg;g3.117.300196" data-apath="/ggg/8/1/1.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Reference Assembly and Annotation of the <em>Pyrenophora teres</em> f. <em>teres</em> Isolate 0-1</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nathan A. Wyatt</span>, <span class="highwire-citation-author" data-delta="1">Jonathan K. Richards</span>, <span class="highwire-citation-author" data-delta="2">Robert S. Brueggeman</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5634-2200" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Timothy L. Friesen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1-8;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300196 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487831--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nathan A. Wyatt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genomics and Bioinformatics Program, North Dakota State University, Fargo, North Dakota 58102</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, North Dakota State University, Fargo, North Dakota 58102</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathan%2BA.%2BWyatt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wyatt%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANathan%2BA.%2BWyatt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jonathan K. Richards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, North Dakota State University, Fargo, North Dakota 58102</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BK.%2BRichards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Richards%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BK.%2BRichards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Robert S. Brueggeman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genomics and Bioinformatics Program, North Dakota State University, Fargo, North Dakota 58102</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, North Dakota State University, Fargo, North Dakota 58102</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BS.%2BBrueggeman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brueggeman%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BS.%2BBrueggeman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Timothy L. Friesen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genomics and Bioinformatics Program, North Dakota State University, Fargo, North Dakota 58102</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, North Dakota State University, Fargo, North Dakota 58102</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cereal Crops Research Unit, Red River Valley Agricultural Research Center, United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Fargo, North Dakota 58102</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BL.%2BFriesen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friesen%20TL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimothy%2BL.%2BFriesen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5634-2200" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timothy L. Friesen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:timothy.friesen@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">timothy.friesen@ars.usda.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/1/1" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/1/1&amp;title=Reference%20Assembly%20and%20Annotation%20of%20the%20Pyrenophora%20teres%20f.%20teres%20Isolate%200-1" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/1/1&amp;title=Reference%20Assembly%20and%20Annotation%20of%20the%20Pyrenophora%20teres%20f.%20teres%20Isolate%200-1" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/1/1&amp;title=Reference%20Assembly%20and%20Annotation%20of%20the%20Pyrenophora%20teres%20f.%20teres%20Isolate%200-1" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/1/1&amp;text=Reference%20Assembly%20and%20Annotation%20of%20the%20Pyrenophora%20teres%20f.%20teres%20Isolate%200-1" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/1/1&amp;title=Reference%20Assembly%20and%20Annotation%20of%20the%20Pyrenophora%20teres%20f.%20teres%20Isolate%200-1" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/1/1&amp;t=Reference%20Assembly%20and%20Annotation%20of%20the%20Pyrenophora%20teres%20f.%20teres%20Isolate%200-1" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/1/1&amp;title=Reference%20Assembly%20and%20Annotation%20of%20the%20Pyrenophora%20teres%20f.%20teres%20Isolate%200-1" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/1/1&amp;title=Reference%20Assembly%20and%20Annotation%20of%20the%20Pyrenophora%20teres%20f.%20teres%20Isolate%200-1" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487831/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487831" id="node487831--3" data-pisa="ggg;8/1/1" data-pisa-master="ggg;g3.117.300196" data-apath="/ggg/8/1/1.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Reference Assembly and Annotation of the <em>Pyrenophora teres</em> f. <em>teres</em> Isolate 0-1</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Nathan A. Wyatt</span>, <span class="highwire-citation-author" data-delta="1">Jonathan K. Richards</span>, <span class="highwire-citation-author" data-delta="2">Robert S. Brueggeman</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-5634-2200" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Timothy L. Friesen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1-8;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300196 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487831--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Nathan A. Wyatt </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genomics and Bioinformatics Program, North Dakota State University, Fargo, North Dakota 58102</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, North Dakota State University, Fargo, North Dakota 58102</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nathan%2BA.%2BWyatt%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wyatt%20NA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANathan%2BA.%2BWyatt%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jonathan K. Richards </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, North Dakota State University, Fargo, North Dakota 58102</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jonathan%2BK.%2BRichards%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Richards%20JK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJonathan%2BK.%2BRichards%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Robert S. Brueggeman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genomics and Bioinformatics Program, North Dakota State University, Fargo, North Dakota 58102</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, North Dakota State University, Fargo, North Dakota 58102</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BS.%2BBrueggeman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brueggeman%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BS.%2BBrueggeman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Timothy L. Friesen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genomics and Bioinformatics Program, North Dakota State University, Fargo, North Dakota 58102</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Plant Pathology, North Dakota State University, Fargo, North Dakota 58102</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Cereal Crops Research Unit, Red River Valley Agricultural Research Center, United States Department of Agriculture-Agricultural Research Service (USDA-ARS), Fargo, North Dakota 58102</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BL.%2BFriesen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Friesen%20TL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimothy%2BL.%2BFriesen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5634-2200" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timothy L. Friesen</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:timothy.friesen@ars.usda.gov" class="" data-icon-position="" data-hide-link-title="0">timothy.friesen@ars.usda.gov</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1&amp;rft.epage=8&amp;rft.atitle=Reference%20Assembly%20and%20Annotation%20of%20the%20Pyrenophora%20teres%20f.%20teres%20Isolate%200-1&amp;rft.volume=8&amp;rft.issue=1&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wyatt%2C+Nathan+A.&amp;rft.au=Richards%2C+Jonathan+K.&amp;rft.au=Brueggeman%2C+Robert+S.&amp;rft.au=Friesen%2C+Timothy+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487831/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487831/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487831/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487831/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487831/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487831/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487831/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487831/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487831/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487831/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487831/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487831/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/1/1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/1/1.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/1/1.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/1/1.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/1/1.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
