<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/1/315" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/1/315.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/1/315.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/1/315.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/1/315.atom" />
<meta name="HW.pisa" content="ggg;8/1/315" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Mapping of Decay Factor–mRNA Interactions in Yeast Identifies Nutrient-Responsive Transcripts as Targets of the Deadenylase Ccr4" />
<meta name="DC.Identifier" content="10.1534/g3.117.300415" />
<meta name="DC.Date" content="2018-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Miller et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The [Ccr4][1] (carbon catabolite repression 4)-Not complex is a major regulator of stress responses that controls gene expression at multiple levels, from transcription to mRNA decay. [Ccr4][1], a “core” subunit of the complex, is the main cytoplasmic deadenylase in Saccharomyces cerevisiae ; however, its mRNA targets have not been mapped on a genome-wide scale. Here, we describe a genome-wide approach, RNA immunoprecipitation (RIP) high-throughput sequencing (RIP-seq), to identify the RNAs bound to [Ccr4][1], and two proteins that associate with it, [Dhh1][2] and [Puf5][3]. All three proteins were preferentially bound to lowly abundant mRNAs, most often at the 3′ end of the transcript. Furthermore, [Ccr4][1], [Dhh1][2], and [Puf5][3] are recruited to mRNAs that are targeted by other RNA-binding proteins that promote decay and mRNA transport, and inhibit translation. Although [Ccr4][1]-Not regulates mRNA transcription and decay, [Ccr4][1] recruitment to mRNAs correlates better with decay rates, suggesting it imparts greater control over transcript abundance through decay. [Ccr4][1]-enriched mRNAs are refractory to control by the other deadenylase complex in yeast, [Pan2][4]/[3][5], suggesting a division of labor between these deadenylation complexes. Finally, [Ccr4][1] and [Dhh1][2] associate with mRNAs whose abundance increases during nutrient starvation, and those that fluctuate during metabolic and oxygen consumption cycles, which explains the known genetic connections between these factors and nutrient utilization and stress pathways.

 [1]: http://www.yeastgenome.org/locus/S000000019/overview
 [2]: http://www.yeastgenome.org/locus/S000002319/overview
 [3]: http://www.yeastgenome.org/locus/S000003146/overview
 [4]: http://www.yeastgenome.org/locus/S000003062/overview
 [5]: https://www.yeastgenome.org/locus/S000001508" />
<meta name="DC.Contributor" content="Jason E. Miller" />
<meta name="DC.Contributor" content="Liye Zhang" />
<meta name="DC.Contributor" content="Haoyang Jiang" />
<meta name="DC.Contributor" content="Yunfei Li" />
<meta name="DC.Contributor" content="B. Franklin Pugh" />
<meta name="DC.Contributor" content="Joseph C. Reese" />
<meta name="article:published_time" content="2018-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome-Wide Mapping of Decay Factor–mRNA Interactions in Yeast Identifies Nutrient-Responsive Transcripts as Targets of the Deadenylase Ccr4" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The Ccr4 (carbon catabolite repression 4)-Not complex is a major regulator of stress responses that controls gene expression at multiple levels, from transcription to mRNA decay. Ccr4, a “core” subunit of the complex, is the main cytoplasmic deadenylase in &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;; however, its mRNA targets have not been mapped on a genome-wide scale. Here, we describe a genome-wide approach, RNA immunoprecipitation (RIP) high-throughput sequencing (RIP-seq), to identify the RNAs bound to Ccr4, and two proteins that associate with it, Dhh1 and Puf5. All three proteins were preferentially bound to lowly abundant mRNAs, most often at the 3′ end of the transcript. Furthermore, Ccr4, Dhh1, and Puf5 are recruited to mRNAs that are targeted by other RNA-binding proteins that promote decay and mRNA transport, and inhibit translation. Although Ccr4-Not regulates mRNA transcription and decay, Ccr4 recruitment to mRNAs correlates better with decay rates, suggesting it imparts greater control over transcript abundance through decay. Ccr4-enriched mRNAs are refractory to control by the other deadenylase complex in yeast, Pan2/3, suggesting a division of labor between these deadenylation complexes. Finally, Ccr4 and Dhh1 associate with mRNAs whose abundance increases during nutrient starvation, and those that fluctuate during metabolic and oxygen consumption cycles, which explains the known genetic connections between these factors and nutrient utilization and stress pathways.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="ggg;8/1/315" />
<meta name="citation_id" content="8/1/315" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/1/315" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/1/315.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/1/315.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/1/315.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300415" />
<meta name="citation_pmid" content="29158339" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="315" />
<meta name="citation_lastpage" content="330" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jason E. Miller" />
<meta name="citation_author_institution" content="Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author_institution" content="Center for RNA Molecular Biology, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author" content="Liye Zhang" />
<meta name="citation_author_institution" content="Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author_institution" content="Center for Comparative Genomics and Bioinformatics, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author" content="Haoyang Jiang" />
<meta name="citation_author_institution" content="Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author_institution" content="Center for RNA Molecular Biology, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author" content="Yunfei Li" />
<meta name="citation_author_institution" content="Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author_institution" content="Center for Comparative Genomics and Bioinformatics, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author" content="B. Franklin Pugh" />
<meta name="citation_author_institution" content="Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author_institution" content="Center for Comparative Genomics and Bioinformatics, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author" content="Joseph C. Reese" />
<meta name="citation_author_institution" content="Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author_institution" content="Center for RNA Molecular Biology, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802" />
<meta name="citation_author_email" content="Jcr8@psu.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3578-4266" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=N. Alhusaini;citation_author=J. Coller;citation_title=The deadenylase components Not2p, Not3p, and Not5p promote mRNA decapping.;citation_volume=22;citation_year=2016;citation_pmid=26952104;citation_doi=10.1261/rna.054742.115" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. A. Arribere;citation_author=J. A. Doudna;citation_author=W. V. Gilbert;citation_title=Reconsidering movement of eukaryotic mRNAs between polysomes and P bodies.;citation_pages=745-758;citation_volume=44;citation_year=2011;citation_issue=5;citation_pmid=22152478;citation_doi=10.1016/j.molcel.2011.09.019" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=N. Azzouz;citation_author=O. O. Panasenko;citation_author=C. Deluen;citation_author=J. Hsieh;citation_author=G. Theiler;citation_title=Specific roles for the Ccr4-Not complex subunits in expression of the genome.;citation_volume=15;citation_year=2009;citation_pmid=19155328;citation_doi=10.1261/rna.1348209" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=V. Babbarwal;citation_author=J. Fu;citation_author=J. C. Reese;citation_title=The Rpb4/7 module of RNA polymerase II is required for carbon catabolite repressor protein 4-negative on TATA (Ccr4-not) complex to promote elongation.;citation_volume=289;citation_year=2014;citation_pmid=25315781;citation_doi=10.1074/jbc.C114.601088" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=J. Basquin;citation_author=V. V. Roudko;citation_author=M. Rode;citation_author=C. Basquin;citation_author=B. Seraphin;citation_title=Architecture of the nuclease module of the yeast ccr4-not complex: the not1-caf1-ccr4 interaction.;citation_pages=207-218;citation_volume=48;citation_year=2012;citation_issue=2;citation_pmid=22959269;citation_doi=10.1016/j.molcel.2012.08.014" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=Y. Benjamini;citation_author=Y. Hochberg;citation_title=Controlling the false discovery rate - a practical and powerful approach to multiple testing.;citation_pages=289-300;citation_volume=57;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Bergkessel;citation_author=J. C. Reese;citation_title=An Essential Role for the Saccharomyces cerevisiae DEAD-Box Helicase DHH1 in G1/S DNA-Damage Checkpoint Recovery;citation_pages=21-33;citation_volume=167;citation_year=2004;citation_issue=1;citation_pmid=15166134;citation_doi=10.1534/genetics.167.1.21" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. I. Boyle;citation_author=S. Weng;citation_author=J. Gollub;citation_author=H. Jin;citation_author=D. Botstein;citation_title=GO:TermFinder–open source software for accessing Gene Ontology information and finding significantly enriched Gene Ontology terms associated with a list of genes.;citation_volume=20;citation_year=2004;citation_pmid=15297299;citation_doi=10.1093/bioinformatics/bth456" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=P. H. Bradley;citation_author=M. J. Brauer;citation_author=J. D. Rabinowitz;citation_author=O. G. Troyanskaya;citation_title=Coordinated concentration changes of transcripts and metabolites in Saccharomyces cerevisiae.;citation_pages=e1000270-e1000270;citation_volume=5;citation_year=2009;citation_issue=1;citation_pmid=19180179;citation_doi=10.1371/journal.pcbi.1000270" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. E. Brown;citation_author=A. B. Sachs;citation_title=Poly(A) Tail Length Control in Saccharomyces cerevisiae Occurs by Message-Specific Deadenylation;citation_pages=6548-6559;citation_volume=18;citation_year=1998;citation_issue=11;citation_pmid=9774670" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. E. Brown;citation_author=S. Z. Tarun;citation_author=R. Boeck;citation_author=A. B. Sachs;citation_title=PAN3 encodes a subunit of the Pab1p-dependent poly(A) nuclease in Saccharomyces cerevisiae;citation_pages=5744-5753;citation_volume=16;citation_year=1996;citation_issue=10;citation_pmid=8816488;citation_doi=10.1128/MCB.16.10.5744" />
<meta name="citation_reference" content="citation_journal_title=Annual review of cell and developmental biology;citation_journal_abbrev=Annu Rev Cell Dev Biol;citation_author=L. Cai;citation_author=B. P. Tu;citation_title=Driving the cell cycle through metabolism.;citation_pages=59-87;citation_volume=28;citation_year=2012;citation_pmid=22578140;citation_doi=10.1146/annurev-cellbio-092910-154010" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=G. A. Cary;citation_author=D. B. Vinh;citation_author=P. May;citation_author=R. Kuestner;citation_author=A. M. Dudley;citation_title=Proteomic analysis of Dhh1 complexes reveals a role for Hsp40 Chaperone Ydj1 in yeast P-body assembly.;citation_pages=2497-2511;citation_volume=5;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=J. Chen;citation_author=Y. C. Chiang;citation_author=C. L. Denis;citation_title=CCR4, a 3'-5' poly(A) RNA and ssDNA exonuclease, is the catalytic component of the cytoplasmic deadenylase;citation_pages=1414-1426;citation_volume=21;citation_year=2002;citation_issue=6;citation_pmid=11889047;citation_doi=10.1093/emboj/21.6.1414" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=Y. Chen;citation_author=A. Boland;citation_author=D. Kuzuoglu-Ozturk;citation_author=P. Bawankar;citation_author=B. Loh;citation_title=A DDX6-CNOT1 complex and W-binding pockets in CNOT9 reveal direct links between miRNA target recognition and silencing.;citation_pages=737-750;citation_volume=54;citation_year=2014;citation_pmid=24768540;citation_doi=10.1016/j.molcel.2014.03.034" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=Z. Cheng;citation_author=J. Coller;citation_author=R. Parker;citation_author=H. Song;citation_title=Crystal structure and functional analysis of DEAD-box protein Dhh1p.;citation_volume=11;citation_year=2005;citation_pmid=15987810;citation_doi=10.1261/rna.2920905" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=J. J. Chritton;citation_author=M. Wickens;citation_title=Translational repression by PUF proteins in vitro.;citation_volume=16;citation_year=2010;citation_pmid=20427513;citation_doi=10.1261/rna.2070110" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=M. A. Collart;citation_title=Global control of gene expression in yeast by the Ccr4-Not complex.;citation_pages=1-16;citation_volume=313;citation_year=2003;citation_pmid=12957374;citation_doi=10.1016/S0378-1119(03)00672-3" />
<meta name="citation_reference" content="citation_journal_title=Wiley Interdiscip. Rev. RNA;citation_author=M. A. Collart;citation_title=The Ccr4-Not complex is a key regulator of eukaryotic gene expression.;citation_pages=438-454;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Coller;citation_author=R. Parker;citation_title=General translational repression by activators of mRNA decapping.;citation_pages=875-886;citation_volume=122;citation_year=2005;citation_issue=6;citation_pmid=16179257;citation_doi=10.1016/j.cell.2005.07.012" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_journal_abbrev=RNA;citation_author=J. M. Coller;citation_author=M. Tucker;citation_author=U. Sheth;citation_author=M. A. Valencia-Sanchez;citation_author=R. Parker;citation_title=The DEAD box helicase, Dhh1p, functions in mRNA decapping and interacts with both the decapping and deadenylase complexes;citation_pages=1717-1727;citation_volume=7;citation_year=2001;citation_issue=12;citation_pmid=11780629;citation_doi=10.1017/S135583820101994X" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=Y. Cui;citation_author=D. B. Ramnarain;citation_author=Y. C. Chiang;citation_author=L. H. Ding;citation_author=J. S. McMahon;citation_title=Genome wide expression analysis of the CCR4-NOT complex indicates that it consists of three modules with the NOT module controlling SAGA-responsive genes.;citation_pages=323-337;citation_volume=279;citation_year=2008;citation_issue=4;citation_pmid=18214544;citation_doi=10.1007/s00438-007-0314-1" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. David;citation_author=W. Huber;citation_author=M. Granovskaia;citation_author=J. Toedling;citation_author=C. J. Palm;citation_title=A high-resolution map of transcription in the yeast genome.;citation_volume=103;citation_year=2006;citation_pmid=16569694;citation_doi=10.1073/pnas.0601091103" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. Deluen;citation_author=N. James;citation_author=L. Maillet;citation_author=M. Molinete;citation_author=G. Theiler;citation_title=The Ccr4-Not Complex and yTAF1 (yTafII130p/yTafII145p) Show Physical and Functional Interactions;citation_pages=6735-6749;citation_volume=22;citation_year=2002;citation_issue=19;citation_pmid=12215531;citation_doi=10.1128/MCB.22.19.6735-6749.2002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. L. Denis;citation_title=IDENTIFICATION OF NEW GENES INVOLVED IN THE REGULATION OF YEAST ALCOHOL DEHYDROGENASE II;citation_pages=833-844;citation_volume=108;citation_year=1984;citation_issue=4;citation_pmid=6392016" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=A. Dutta;citation_author=S. Zheng;citation_author=D. Jain;citation_author=C. E. Cameron;citation_author=J. C. Reese;citation_title=Intermolecular interactions within the abundant DEAD-box protein Dhh1 regulate its activity in vivo.;citation_volume=286;citation_year=2011;citation_pmid=21642421;citation_doi=10.1074/jbc.M111.220251" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell. Biol.;citation_author=A. Dutta;citation_author=V. Babbarwal;citation_author=J. Fu;citation_author=D. Brunke-Reese;citation_author=D. M. Libert;citation_title=Ccr4-Not and TFIIS function cooperatively to rescue arrested RNA polymerase II.;citation_volume=35;citation_year=2015;citation_pmid=25776559;citation_doi=10.1128/MCB.00044-15" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=O. Elemento;citation_author=N. Slonim;citation_author=S. Tavazoie;citation_title=A universal framework for regulatory element discovery across all genomes and data types.;citation_pages=337-350;citation_volume=28;citation_year=2007;citation_issue=2;citation_pmid=17964271;citation_doi=10.1016/j.molcel.2007.09.027" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=P. Eser;citation_author=C. Demel;citation_author=K. C. Maier;citation_author=B. Schwalb;citation_author=N. Pirkl;citation_title=Periodic mRNA synthesis and degradation co-operate during cell cycle gene expression;citation_pages=717-717;citation_volume=10;citation_year=2014;citation_issue=1;citation_pmid=24489117;citation_doi=10.1002/msb.134886" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=A. Eulalio;citation_author=I. Behm-Ansmant;citation_author=E. Izaurralde;citation_title=P bodies: at the crossroads of post-transcriptional pathways.;citation_pages=9-22;citation_volume=8;citation_year=2007;citation_issue=1;citation_pmid=17183357;citation_doi=10.1038/nrm2080" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Faraji;citation_author=Y. Hu;citation_author=H. H. Yang;citation_author=M. P. Lee;citation_author=G. S. Winkler;citation_title=Post-transcriptional control of tumor cell autonomous metastatic potential by CCR4-NOT deadenylase CNOT7.;citation_pages=e1005820;citation_volume=12;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=N. Fischer;citation_author=K. Weis;citation_title=The DEAD box protein Dhh1 stimulates the decapping enzyme Dcp1;citation_pages=2788-2797;citation_volume=21;citation_year=2002;citation_issue=11;citation_pmid=12032091;citation_doi=10.1093/emboj/21.11.2788" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=A. P. Gasch;citation_author=P. T. Spellman;citation_author=C. M. Kao;citation_author=O. Carmel-Harel;citation_author=M. B. Eisen;citation_title=Genomic Expression Programs in the Response of Yeast Cells to Environmental Changes;citation_pages=4241-4257;citation_volume=11;citation_year=2000;citation_issue=12;citation_pmid=11102521;citation_doi=10.1091/mbc.11.12.4241" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. V. Gerashchenko;citation_author=A. V. Lobanov;citation_author=V. N. Gladyshev;citation_title=Genome-wide ribosome profiling reveals complex translational regulation in response to oxidative stress.;citation_volume=109;citation_year=2012;citation_pmid=23045643;citation_doi=10.1073/pnas.1120799109" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=A. P. Gerber;citation_author=D. Herschlag;citation_author=P. O. Brown;citation_title=Extensive association of functionally and cytotopically related mRNAs with Puf family RNA-binding proteins in yeast.;citation_pages=E79-E79;citation_volume=2;citation_year=2004;citation_issue=3;citation_pmid=15024427;citation_doi=10.1371/journal.pbio.0020079" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Ghaemmaghami;citation_author=W. K. Huh;citation_author=K. Bower;citation_author=R. W. Howson;citation_author=A. Belle;citation_title=Global analysis of protein expression in yeast.;citation_pages=737-741;citation_volume=425;citation_year=2003;citation_issue=6959;citation_pmid=14562106;citation_doi=10.1038/nature02046" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=A. C. Goldstrohm;citation_author=M. Wickens;citation_title=Multifunctional deadenylase complexes diversify mRNA control.;citation_pages=337-344;citation_volume=9;citation_year=2008;citation_issue=4;citation_pmid=18334997;citation_doi=10.1038/nrm2370" />
<meta name="citation_reference" content="citation_journal_title=Nature structural &amp; molecular biology;citation_journal_abbrev=Nat Struct Mol Biol;citation_author=A. C. Goldstrohm;citation_author=B. A. Hook;citation_author=D. J. Seay;citation_author=M. Wickens;citation_title=PUF proteins bind Pop2p to regulate messenger RNAs.;citation_pages=533-539;citation_volume=13;citation_year=2006;citation_issue=6;citation_pmid=16715093;citation_doi=10.1038/nsmb1100" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=A. C. Goldstrohm;citation_author=D. J. Seay;citation_author=B. A. Hook;citation_author=M. Wickens;citation_title=PUF protein-mediated deadenylation is catalyzed by Ccr4p.;citation_volume=282;citation_year=2007;citation_pmid=17090538;citation_doi=10.1074/jbc.M609413200" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=J. Grigull;citation_author=S. Mnaimneh;citation_author=J. Pootoolal;citation_author=M. D. Robinson;citation_author=T. R. Hughes;citation_title=Genome-Wide Analysis of mRNA Stability Using Transcription Inhibitors and Microarrays Reveals Posttranscriptional Control of Ribosome Biogenesis Factors;citation_pages=5534-5547;citation_volume=24;citation_year=2004;citation_issue=12;citation_pmid=15169913;citation_doi=10.1128/MCB.24.12.5534-5547.2004" />
<meta name="citation_reference" content="citation_journal_title=Cell Rep.;citation_author=I. Gupta;citation_author=Z. Villanyi;citation_author=S. Kassem;citation_author=C. Hughes;citation_author=O. O. Panasenko;citation_title=Translational capacity of a cell is determined during transcription elongation via the Ccr4-Not complex.;citation_pages=1782-1794;citation_volume=15;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Biochim. Biophys. Acta;citation_author=G. Haimovich;citation_author=M. Choder;citation_author=R. H. Singer;citation_author=T. Trcek;citation_title=The fate of the messenger is pre-determined: a new model for regulation of gene expression.;citation_pages=643-653;citation_volume=1829;citation_year=2013;citation_doi=10.1016/j.bbagrm.2013.01.004" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=G. Haimovich;citation_author=D. A. Medina;citation_author=S. Z. Causse;citation_author=M. Garber;citation_author=G. Millan-Zambrano;citation_title=Gene expression is circular: factors for mRNA degradation also foster mRNA synthesis.;citation_pages=1000-1011;citation_volume=153;citation_year=2013;citation_pmid=23706738;citation_doi=10.1016/j.cell.2013.05.012" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Hata;citation_author=H. Mitsui;citation_author=H. Liu;citation_author=Y. Bai;citation_author=C. L. Denis;citation_title=Dhh1p, a Putative RNA Helicase, Associates with the General Transcription Factors Pop2p and Ccr4p from Saccharomyces cerevisiae;citation_pages=571-580;citation_volume=148;citation_year=1998;citation_issue=2;citation_pmid=9504907" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=R. E. Hector;citation_author=K. R. Nykamp;citation_author=S. Dheur;citation_author=J. T. Anderson;citation_author=P. J. Non;citation_title=Dual requirement for yeast hnRNP Nab2p in mRNA poly(A) tail length control and nuclear export;citation_pages=1800-1810;citation_volume=21;citation_year=2002;citation_issue=7;citation_pmid=11927564;citation_doi=10.1093/emboj/21.7.1800" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=S. Heinz;citation_author=C. Benner;citation_author=N. Spann;citation_author=E. Bertolino;citation_author=Y. C. Lin;citation_title=Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities.;citation_pages=576-589;citation_volume=38;citation_year=2010;citation_issue=4;citation_pmid=20513432;citation_doi=10.1016/j.molcel.2010.05.004" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=D. J. Hogan;citation_author=D. P. Riordan;citation_author=A. P. Gerber;citation_author=D. Herschlag;citation_author=P. O. Brown;citation_title=Diverse RNA-binding proteins interact with functionally related sets of RNAs, suggesting an extensive regulatory system.;citation_pages=e255-e255;citation_volume=6;citation_year=2008;citation_issue=10;citation_pmid=18959479;citation_doi=10.1371/journal.pbio.0060255" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=B. A. Hook;citation_author=A. C. Goldstrohm;citation_author=D. J. Seay;citation_author=M. Wickens;citation_title=Two yeast PUF proteins negatively regulate a single mRNA.;citation_volume=282;citation_year=2007;citation_pmid=17389596;citation_doi=10.1074/jbc.M611253200" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=W. Hu;citation_author=T. J. Sweet;citation_author=S. Chamnongpol;citation_author=K. E. Baker;citation_author=J. Coller;citation_title=Co-translational mRNA decay in Saccharomyces cerevisiae.;citation_pages=225-229;citation_volume=461;citation_year=2009;citation_issue=7261;citation_pmid=19701183;citation_doi=10.1038/nature08265" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=N. T. Ingolia;citation_author=S. Ghaemmaghami;citation_author=J. R. Newman;citation_author=J. S. Weissman;citation_title=Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling.;citation_volume=324;citation_year=2009;citation_pmid=19213877;citation_doi=10.1126/science.1168978" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=W. Ito;citation_author=X. Li;citation_author=K. Irie;citation_author=T. Mizuno;citation_author=K. Irie;citation_title=RNA-binding protein Khd1 and Ccr4 deadenylase play overlapping roles in the cell wall integrity pathway in Saccharomyces cerevisiae.;citation_volume=10;citation_year=2011;citation_pmid=21873511;citation_doi=10.1128/EC.05181-11" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. D. Keene;citation_title=RNA regulons: coordination of post-transcriptional events.;citation_pages=533-543;citation_volume=8;citation_year=2007;citation_issue=7;citation_pmid=17572691;citation_doi=10.1038/nrg2111" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=S. C. Kerr;citation_author=N. Azzouz;citation_author=S. M. Fuchs;citation_author=M. A. Collart;citation_author=B. D. Strahl;citation_title=The Ccr4-Not complex interacts with the mRNA export machinery.;citation_pages=e18302-e18302;citation_volume=6;citation_year=2011;citation_issue=3;citation_pmid=21464899;citation_doi=10.1371/journal.pone.0018302" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. A. Kruk;citation_author=A. Dutta;citation_author=J. Fu;citation_author=D. S. Gilmour;citation_author=J. C. Reese;citation_title=The multifunctional Ccr4-Not complex directly promotes transcription elongation;citation_pages=581-593;citation_volume=25;citation_year=2011;citation_issue=6;citation_pmid=21406554;citation_doi=10.1101/gad.2020911" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. N. Laribee;citation_author=A. Hosni-Ahmed;citation_author=J. J. Workman;citation_author=H. Chen;citation_title=Ccr4-not regulates RNA polymerase I transcription and couples nutrient signaling to the control of ribosomal RNA biogenesis.;citation_pages=e1005113;citation_volume=11;citation_year=2015;citation_pmid=25815716;citation_doi=10.1371/journal.pgen.1005113" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. C. Lau;citation_author=K. W. Mulder;citation_author=A. B. Brenkman;citation_author=S. Mohammed;citation_author=N. J. van den Broek;citation_title=Phosphorylation of Not4p functions parallel to BUR2 to regulate resistance to cellular stresses in Saccharomyces cerevisiae.;citation_pages=e9864-e9864;citation_volume=5;citation_year=2010;citation_issue=4;citation_pmid=20386698;citation_doi=10.1371/journal.pone.0009864" />
<meta name="citation_reference" content="citation_journal_title=Cell Rep.;citation_author=C. D. Lee;citation_author=B. P. Tu;citation_title=Glucose-regulated phosphorylation of the PUF protein Puf3 regulates the translational fate of its bound mRNAs and association with RNA granules.;citation_pages=1638-1650;citation_volume=11;citation_year=2015;citation_pmid=26051939;citation_doi=10.1016/j.celrep.2015.05.014" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=D. Lee;citation_author=T. Ohn;citation_author=Y. C. Chiang;citation_author=G. Quigley;citation_author=G. Yao;citation_title=PUF3 acceleration of deadenylation in vivo can operate independently of CCR4 activity, possibly involving effects on the PAB1-mRNP structure.;citation_pages=562-575;citation_volume=399;citation_year=2010;citation_issue=4;citation_pmid=20435044;citation_doi=10.1016/j.jmb.2010.04.034" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=M. S. Lee;citation_author=M. Henry;citation_author=P. A. Silver;citation_title=A protein that shuttles between the nucleus and the cytoplasm is an important mediator of RNA export.;citation_pages=1233-1246;citation_volume=10;citation_year=1996;citation_issue=10;citation_pmid=8675010;citation_doi=10.1101/gad.10.10.1233" />
<meta name="citation_reference" content="citation_journal_title=Trends Cell Biol.;citation_author=S. R. Lee;citation_author=J. Lykke-Andersen;citation_title=Emerging roles for ribonucleoprotein modification and remodeling in controlling RNA fate.;citation_pages=504-510;citation_volume=23;citation_year=2013;citation_pmid=23756094;citation_doi=10.1016/j.tcb.2013.05.001" />
<meta name="citation_reference" content="citation_journal_title=Cancer biology &amp; therapy;citation_journal_abbrev=Cancer Biol Ther;citation_author=F. Lin;citation_author=R. Wang;citation_author=J. J. Shen;citation_author=X. Wang;citation_author=P. Gao;citation_title=Knockdown of RCK/p54 expression by RNAi inhibits proliferation of human colorectal cancer cells in vitro and in vivo.;citation_pages=1669-1676;citation_volume=7;citation_year=2008;citation_issue=10;citation_pmid=18769115" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=M. S. Longtine;citation_author=A. McKenzie;citation_author=D. J. Demarini;citation_author=N. G. Shah;citation_author=A. Wach;citation_title=Additional modules for versatile and economical PCR-based gene deletion and modification in Saccharomyces cerevisiae.;citation_pages=953-961;citation_volume=14;citation_year=1998;citation_issue=10;citation_pmid=9717241;citation_doi=10.1002/(SICI)1097-0061(199807)14:10&lt;953::AID-YEA293&gt;3.0.CO;2-U" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=H. Maekawa;citation_author=T. Nakagawa;citation_author=Y. Uno;citation_author=K. Kitamura;citation_author=C. Shimoda;citation_title=The ste13+ gene encoding a putative RNA helicase is essential for nitrogen starvation-induced G1 arrest and initiation of sexual development in the fission yeast Schizosaccharomyces pombe.;citation_pages=456-464;citation_volume=244;citation_year=1994;citation_issue=5;citation_pmid=8078473;citation_doi=10.1007/BF00583896" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=L. Maillet;citation_author=M. A. Collart;citation_title=Interaction between Not1p, a component of the Ccr4-not complex, a global regulator of transcription, and Dhh1p, a putative RNA helicase.;citation_volume=277;citation_year=2002;citation_pmid=11696541;citation_doi=10.1074/jbc.M107979200" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=K. A. Marfatia;citation_author=E. B. Crafton;citation_author=D. M. Green;citation_author=A. H. Corbett;citation_title=Domain analysis of the Saccharomyces cerevisiae heterogeneous nuclear ribonucleoprotein, Nab2p. Dissecting the requirements for Nab2p-facilitated poly(A) RNA export.;citation_volume=278;citation_year=2003;citation_pmid=12496292;citation_doi=10.1074/jbc.M207571200" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=H. Mathys;citation_author=J. Basquin;citation_author=S. Ozgur;citation_author=M. Czarnocki-Cieciura;citation_author=F. Bonneau;citation_title=Structural and biochemical insights to the role of the CCR4-NOT complex and DDX6 ATPase in microRNA repression.;citation_pages=751-765;citation_volume=54;citation_year=2014;citation_pmid=24768538;citation_doi=10.1016/j.molcel.2014.03.036" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=L. Micallef;citation_author=P. Rodgers;citation_title=eulerAPE: drawing area-proportional 3-Venn diagrams using ellipses.;citation_pages=e101717;citation_volume=9;citation_year=2014;citation_pmid=25032825;citation_doi=10.1371/journal.pone.0101717" />
<meta name="citation_reference" content="citation_journal_title=Critical Reviews in Biochemistry and Molecular Biology;citation_journal_abbrev=CRIT REV BIOCHEM MOL BIOL;citation_author=J. E. Miller;citation_author=J. C. Reese;citation_title=Ccr4-Not complex: the control freak of eukaryotic cells.;citation_pages=315-333;citation_volume=47;citation_year=2012;citation_issue=4;citation_pmid=22416820;citation_doi=10.3109/10409238.2012.667214" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. A. Miller;citation_author=J. Russo;citation_author=A. D. Fischer;citation_author=F. A. Lopez Leban;citation_author=W. M. Olivas;citation_title=Carbon source-dependent alteration of Puf3p activity mediates rapid changes in the stabilities of mRNAs involved in mitochondrial function.;citation_volume=42;citation_year=2014;citation_pmid=24371272;citation_doi=10.1093/nar/gkt1346" />
<meta name="citation_reference" content="citation_journal_title=Nat. Struct. Mol. Biol.;citation_author=S. F. Mitchell;citation_author=S. Jain;citation_author=M. She;citation_author=R. Parker;citation_title=Global analysis of yeast mRNPs.;citation_pages=127-133;citation_volume=20;citation_year=2013;citation_pmid=23222640;citation_doi=10.1038/nsmb.2468" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. M. Molina-Navarro;citation_author=L. Castells-Roca;citation_author=G. Belli;citation_author=J. Garcia-Martinez;citation_author=J. Marin-Navarro;citation_title=Comprehensive transcriptional analysis of the oxidative response in yeast.;citation_volume=283;citation_year=2008;citation_pmid=18424442;citation_doi=10.1074/jbc.M800295200" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=H. Moriya;citation_author=Y. Shimizu-Yoshida;citation_author=A. Omori;citation_author=S. Iwashita;citation_author=M. Katoh;citation_title=Yak1p, a DYRK family kinase, translocates to the nucleus and phosphorylates yeast Pop2p in response to a glucose signal;citation_pages=1217-1228;citation_volume=15;citation_year=2001;citation_issue=10;citation_pmid=11358866;citation_doi=10.1101/gad.884001" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=U. Nagalakshmi;citation_author=Z. Wang;citation_author=K. Waern;citation_author=C. Shou;citation_author=D. Raha;citation_title=The transcriptional landscape of the yeast genome defined by RNA sequencing.;citation_volume=320;citation_year=2008;citation_pmid=18451266;citation_doi=10.1126/science.1158441" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Neph;citation_author=M. S. Kuehn;citation_author=A. P. Reynolds;citation_author=E. Haugen;citation_author=R. E. Thurman;citation_title=BEDOPS: high-performance genomic feature operations.;citation_volume=28;citation_year=2012;citation_pmid=22576172;citation_doi=10.1093/bioinformatics/bts277" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=T. Nissan;citation_author=P. Rajyaguru;citation_author=M. She;citation_author=H. Song;citation_author=R. Parker;citation_title=Decapping activators in Saccharomyces cerevisiae act by multiple mechanisms.;citation_pages=773-783;citation_volume=39;citation_year=2010;citation_issue=5;citation_pmid=20832728;citation_doi=10.1016/j.molcel.2010.08.025" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=R. Parker;citation_author=U. Sheth;citation_title=P bodies and the control of mRNA translation and degradation.;citation_pages=635-646;citation_volume=25;citation_year=2007;citation_issue=5;citation_pmid=17349952;citation_doi=10.1016/j.molcel.2007.02.011" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=V. Pelechano;citation_author=W. Wei;citation_author=L. M. Steinmetz;citation_title=Widespread Co-translational RNA decay reveals ribosome dynamics.;citation_pages=1400-1412;citation_volume=161;citation_year=2015;citation_pmid=26046441;citation_doi=10.1016/j.cell.2015.05.008" />
<meta name="citation_reference" content="citation_title=EMBO J.;citation_author=S. Preissler;citation_author=J. Reuther;citation_author=M. Koch;citation_author=A. Scior;citation_author=M. Bruderek;citation_title=Not4-dependent translational repression is important for cellular protein homeostasis in yeast.;citation_volume=34;citation_year=2015;citation_pmid=25971775;citation_doi=10.15252/embj.201490194" />
<meta name="citation_reference" content="citation_journal_title=Biochim. Biophys. Acta;citation_author=V. Presnyak;citation_author=J. Coller;citation_title=The DHH1/RCKp54 family of helicases: an ancient family of proteins that promote translational silencing.;citation_pages=817-823;citation_volume=1829;citation_year=2013;citation_doi=10.1016/j.bbagrm.2013.03.006" />
<meta name="citation_reference" content="citation_journal_title=Trends in cell biology;citation_journal_abbrev=Trends Cell Biol;citation_author=T. Quenault;citation_author=T. Lithgow;citation_author=A. Traven;citation_title=PUF proteins: repression, activation and mRNA localization.;citation_pages=104-112;citation_volume=21;citation_year=2011;citation_issue=2;citation_pmid=21115348;citation_doi=10.1016/j.tcb.2010.09.013" />
<meta name="citation_reference" content="citation_title=Cell;citation_author=A. Radhakrishnan;citation_author=Y. H. Chen;citation_author=S. Martin;citation_author=N. Alhusaini;citation_author=R. Green;citation_title=The DEAD-Box protein Dhh1p couples mRNA decay and translation by monitoring codon optimality.;citation_pages=122-132.e9;citation_volume=167;citation_year=2016;citation_doi=10.1016/j.cell.2016.08.053" />
<meta name="citation_reference" content="citation_journal_title=Biochim. Biophys. Acta;citation_author=J. C. Reese;citation_title=The control of elongation by the yeast Ccr4-not complex.;citation_pages=127-133;citation_volume=1829;citation_year=2013;citation_doi=10.1016/j.bbagrm.2012.09.001" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=L. M. Rendl;citation_author=M. A. Bieman;citation_author=C. A. Smibert;citation_title=S. cerevisiae Vts1p induces deadenylation-dependent transcript degradation and interacts with the Ccr4p-Pop2p-Not deadenylase complex.;citation_volume=14;citation_year=2008;citation_pmid=18469165;citation_doi=10.1261/rna.955508" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=K. J. Riley;citation_author=J. A. Steitz;citation_title=The “Observer Effect” in genome-wide surveys of protein-RNA interactions.;citation_pages=601-604;citation_volume=49;citation_year=2013;citation_pmid=23438856;citation_doi=10.1016/j.molcel.2013.01.030" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. T. Robinson;citation_author=H. Thorvaldsdottir;citation_author=W. Winckler;citation_author=M. Guttman;citation_author=E. S. Lander;citation_title=Integrative genomics viewer.;citation_pages=24-26;citation_volume=29;citation_year=2011;citation_issue=1;citation_pmid=21221095;citation_doi=10.1038/nbt.1754" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. D. Robinson;citation_author=D. J. McCarthy;citation_author=G. K. Smyth;citation_title=edgeR: a bioconductor package for differential expression analysis of digital gene expression data.;citation_volume=26;citation_year=2010;citation_pmid=19910308;citation_doi=10.1093/bioinformatics/btp616" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=S. M. Rumble;citation_author=P. Lacroute;citation_author=A. V. Dalca;citation_author=M. Fiume;citation_author=A. Sidow;citation_title=SHRiMP: accurate mapping of short color-space reads.;citation_pages=e1000386-e1000386;citation_volume=5;citation_year=2009;citation_issue=5;citation_pmid=19461883;citation_doi=10.1371/journal.pcbi.1000386" />
<meta name="citation_reference" content="citation_journal_title=Trends in biochemical sciences;citation_journal_abbrev=Trends Biochem Sci;citation_author=M. Schmid;citation_author=T. H. Jensen;citation_title=The exosome: a multipurpose RNA-decay machine.;citation_pages=501-510;citation_volume=33;citation_year=2008;citation_issue=10;citation_pmid=18786828;citation_doi=10.1016/j.tibs.2008.07.003" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Sun;citation_author=B. Schwalb;citation_author=D. Schulz;citation_author=N. Pirkl;citation_author=S. Etzold;citation_title=Comparative dynamic transcriptome analysis (cDTA) reveals mutual feedback between mRNA synthesis and degradation.;citation_volume=22;citation_year=2012;citation_pmid=22466169;citation_doi=10.1101/gr.130161.111" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=M. Sun;citation_author=B. Schwalb;citation_author=N. Pirkl;citation_author=K. C. Maier;citation_author=A. Schenk;citation_title=Global analysis of eukaryotic mRNA degradation reveals Xrn1-dependent buffering of transcript levels.;citation_pages=52-62;citation_volume=52;citation_year=2013;citation_pmid=24119399;citation_doi=10.1016/j.molcel.2013.09.010" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=T. Sweet;citation_author=C. Kovalak;citation_author=J. Coller;citation_title=The DEAD-box protein Dhh1 promotes decapping by slowing ribosome movement.;citation_pages=e1001342-e1001342;citation_volume=10;citation_year=2012;citation_issue=6;citation_pmid=22719226;citation_doi=10.1371/journal.pbio.1001342" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=N. Talarek;citation_author=E. Cameroni;citation_author=M. Jaquenoud;citation_author=X. Luo;citation_author=S. Bontron;citation_title=Initiation of the TORC1-regulated G0 program requires Igo1/2, which license specific mRNAs to evade degradation via the 5'-3' mRNA decay pathway.;citation_pages=345-355;citation_volume=38;citation_year=2010;citation_issue=3;citation_pmid=20471941;citation_doi=10.1016/j.molcel.2010.02.039" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=K. Tarassov;citation_author=V. Messier;citation_author=C. R. Landry;citation_author=S. Radinovic;citation_author=M. M. Serna Molina;citation_title=An in vivo map of the yeast protein interactome.;citation_volume=320;citation_year=2008;citation_pmid=18467557;citation_doi=10.1126/science.1153878" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=D. Teixeira;citation_author=U. Sheth;citation_author=M. A. Valencia-Sanchez;citation_author=M. Brengues;citation_author=R. Parker;citation_title=Processing bodies require RNA for assembly and contain nontranslating mRNAs.;citation_volume=11;citation_year=2005;citation_pmid=15703442;citation_doi=10.1261/rna.7258505" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=H. Thorvaldsdottir;citation_author=J. T. Robinson;citation_author=J. P. Mesirov;citation_title=Integrative genomics viewer (IGV): high-performance genomics data visualization and exploration.;citation_volume=14;citation_year=2013;citation_pmid=22517427;citation_doi=10.1093/bib/bbs017" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=B. P. Tu;citation_author=A. Kudlicki;citation_author=M. Rowicka;citation_author=S. L. McKnight;citation_title=Logic of the yeast metabolic cycle: temporal compartmentalization of cellular processes.;citation_volume=310;citation_year=2005;citation_pmid=16254148;citation_doi=10.1126/science.1120499" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Tucker;citation_author=M. A. Valencia-Sanchez;citation_author=R. R. Staples;citation_author=J. Chen;citation_author=C. L. Denis;citation_title=The transcription factor associated Ccr4 and Caf1 proteins are components of the major cytoplasmic mRNA deadenylase in Saccharomyces cerevisiae.;citation_pages=377-386;citation_volume=104;citation_year=2001;citation_issue=3;citation_pmid=11239395;citation_doi=10.1016/S0092-8674(01)00225-2" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=M. Tucker;citation_author=R. R. Staples;citation_author=M. A. Valencia-Sanchez;citation_author=D. Muhlrad;citation_author=R. Parker;citation_title=Ccr4p is the catalytic subunit of a Ccr4p/Pop2p/Notp mRNA deadenylase complex in Saccharomyces cerevisiae;citation_pages=1427-1436;citation_volume=21;citation_year=2002;citation_issue=6;citation_pmid=11889048;citation_doi=10.1093/emboj/21.6.1427" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=B. J. Venters;citation_author=S. Wachi;citation_author=T. N. Mavrich;citation_author=B. E. Andersen;citation_author=P. Jena;citation_title=A comprehensive genomic binding map of gene and chromatin regulatory proteins in Saccharomyces.;citation_pages=480-492;citation_volume=41;citation_year=2011;citation_issue=4;citation_pmid=21329885;citation_doi=10.1016/j.molcel.2011.01.015" />
<meta name="citation_reference" content="citation_journal_title=Biochim. Biophys. Acta;citation_author=E. Wahle;citation_author=G. S. Winkler;citation_title=RNA decay machines: deadenylation by the Ccr4-not and Pan2-Pan3 complexes.;citation_pages=561-570;citation_volume=1829;citation_year=2013;citation_pmid=23337855;citation_doi=10.1016/j.bbagrm.2013.01.003" />
<meta name="citation_reference" content="citation_journal_title=The Journal of surgical research;citation_journal_abbrev=J Surg Res;citation_author=T. J. Westmoreland;citation_author=J. A. Olson;citation_author=W. Y. Saito;citation_author=G. Huper;citation_author=J. R. Marks;citation_title=Dhh1 regulates the G1/S-checkpoint following DNA damage or BRCA1 expression in yeast.;citation_pages=62-73;citation_volume=113;citation_year=2003;citation_issue=1;citation_pmid=12943812;citation_doi=10.1016/S0022-4804(03)00155-0" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=T. J. Westmoreland;citation_author=J. R. Marks;citation_author=J. A. Olson;citation_author=E. M. Thompson;citation_author=M. A. Resnick;citation_title=Cell Cycle Progression in G1 and S Phases Is CCR4 Dependent following Ionizing Radiation or Replication Stress in Saccharomyces cerevisiae;citation_pages=430-446;citation_volume=3;citation_year=2004;citation_issue=2;citation_pmid=15075273;citation_doi=10.1128/EC.3.2.430-446.2004" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=D. Wilinski;citation_author=C. Qiu;citation_author=C. P. Lapointe;citation_author=M. Nevil;citation_author=Z. T. Campbell;citation_title=RNA regulatory networks diversified through curvature of the PUF protein scaffold.;citation_pages=8213;citation_volume=6;citation_year=2015;citation_pmid=26364903;citation_doi=10.1038/ncomms9213" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Cell Biology;citation_journal_abbrev=JCB;citation_author=S. M. Wilson;citation_author=K. V. Datar;citation_author=M. R. Paddy;citation_author=J. R. Swedlow;citation_author=M. S. Swanson;citation_title=Characterization of nuclear polyadenylated RNA-binding proteins in Saccharomyces cerevisiae;citation_pages=1173-1184;citation_volume=127;citation_year=1994;citation_issue=5;citation_pmid=7962083;citation_doi=10.1083/jcb.127.5.1173" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Mapping of Decay Factor–mRNA Interactions in Yeast Identifies Nutrient-Responsive Transcripts as Targets of the Deadenylase Ccr4" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="twitter:description" content="The [Ccr4][1] (carbon catabolite repression 4)-Not complex is a major regulator of stress responses that controls gene expression at multiple levels, from transcription to mRNA decay. [Ccr4][1], a “core” subunit of the complex, is the main cytoplasmic deadenylase in Saccharomyces cerevisiae ; however, its mRNA targets have not been mapped on a genome-wide scale. Here, we describe a genome-wide approach, RNA immunoprecipitation (RIP) high-throughput sequencing (RIP-seq), to identify the RNAs bound to [Ccr4][1], and two proteins that associate with it, [Dhh1][2] and [Puf5][3]. All three proteins were preferentially bound to lowly abundant mRNAs, most often at the 3′ end of the transcript. Furthermore, [Ccr4][1], [Dhh1][2], and [Puf5][3] are recruited to mRNAs that are targeted by other RNA-binding proteins that promote decay and mRNA transport, and inhibit translation. Although [Ccr4][1]-Not regulates mRNA transcription and decay, [Ccr4][1] recruitment to mRNAs correlates better with decay rates, suggesting it imparts greater control over transcript abundance through decay. [Ccr4][1]-enriched mRNAs are refractory to control by the other deadenylase complex in yeast, [Pan2][4]/[3][5], suggesting a division of labor between these deadenylation complexes. Finally, [Ccr4][1] and [Dhh1][2] associate with mRNAs whose abundance increases during nutrient starvation, and those that fluctuate during metabolic and oxygen consumption cycles, which explains the known genetic connections between these factors and nutrient utilization and stress pathways.

 [1]: http://www.yeastgenome.org/locus/S000000019/overview
 [2]: http://www.yeastgenome.org/locus/S000002319/overview
 [3]: http://www.yeastgenome.org/locus/S000003146/overview
 [4]: http://www.yeastgenome.org/locus/S000003062/overview
 [5]: https://www.yeastgenome.org/locus/S000001508" />
<meta name="og-title" property="og:title" content="Genome-Wide Mapping of Decay Factor–mRNA Interactions in Yeast Identifies Nutrient-Responsive Transcripts as Targets of the Deadenylase Ccr4" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/1/315" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The [Ccr4][1] (carbon catabolite repression 4)-Not complex is a major regulator of stress responses that controls gene expression at multiple levels, from transcription to mRNA decay. [Ccr4][1], a “core” subunit of the complex, is the main cytoplasmic deadenylase in Saccharomyces cerevisiae ; however, its mRNA targets have not been mapped on a genome-wide scale. Here, we describe a genome-wide approach, RNA immunoprecipitation (RIP) high-throughput sequencing (RIP-seq), to identify the RNAs bound to [Ccr4][1], and two proteins that associate with it, [Dhh1][2] and [Puf5][3]. All three proteins were preferentially bound to lowly abundant mRNAs, most often at the 3′ end of the transcript. Furthermore, [Ccr4][1], [Dhh1][2], and [Puf5][3] are recruited to mRNAs that are targeted by other RNA-binding proteins that promote decay and mRNA transport, and inhibit translation. Although [Ccr4][1]-Not regulates mRNA transcription and decay, [Ccr4][1] recruitment to mRNAs correlates better with decay rates, suggesting it imparts greater control over transcript abundance through decay. [Ccr4][1]-enriched mRNAs are refractory to control by the other deadenylase complex in yeast, [Pan2][4]/[3][5], suggesting a division of labor between these deadenylation complexes. Finally, [Ccr4][1] and [Dhh1][2] associate with mRNAs whose abundance increases during nutrient starvation, and those that fluctuate during metabolic and oxygen consumption cycles, which explains the known genetic connections between these factors and nutrient utilization and stress pathways.

 [1]: http://www.yeastgenome.org/locus/S000000019/overview
 [2]: http://www.yeastgenome.org/locus/S000002319/overview
 [3]: http://www.yeastgenome.org/locus/S000003146/overview
 [4]: http://www.yeastgenome.org/locus/S000003062/overview
 [5]: https://www.yeastgenome.org/locus/S000001508" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
    <title>Genome-Wide Mapping of Decay Factor–mRNA Interactions in Yeast Identifies Nutrient-Responsive Transcripts as Targets of the Deadenylase Ccr4 | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"SDMY4l3xkr5hyJO1A_OFGrUEWcnJXp4UbnmFSpRlJTw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487822","apath":"\/ggg\/8\/1\/315.atom","pisa":"ggg;8\/1\/315","ac":{"\/ggg\/8\/1\/315.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/1\/315.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/1\/315"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/1\/315"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/1\/315.abstract":true,"\/content\/8\/1\/315":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SB66BjCJHLIw4+FkXVDFWFTV6Ct2uyPLJKiLXMqC8JAguQTfKTvoFkhn9rn0p+ITd0v4y1kEfwZiFXC6OqXyfezKmQvN64nI+95h3NvN5pG4uyjjPmvgUh2b6om1PvnBpg3Cf\\\\\\\/pra1RhYbN6\\\\\\\/RbqcOBYO\\\\\\\/+NMUox7TtGfkQ+gRs05jtAWCeAlXhltfKXaqgeGAEBUB7GvcI\\\\\\\/D23J5KGaWhLirlMxMubcD4IheJ\\\\\\\/7JyXU5XMVFNPOvSAkfCExxhsD+h95oH0IswMJin9Tm8deKn\\\\\\\/eQargWWZdMrs25nkEXijRkfNGBSazM06Lh6n0CUA0FHxvch+Z\\\\\\\/qJE3pF7Jfsf1MLRAVDhXseOm5l7CAv5jxH9KmBQdNGLbE4cTccKegeQg9moABg23rHLXHgL7Q4bEB\\\\\\\/ETWIv9ySvjTlZPNGh7+sMjVNu3wtZqBFODGdTfOP+aWZlvbuuYo2S0yK9ImWim1YXr6q11GMUOCB5x8DBE\\\\\\\/HGW5BZDNksJuNrysXhTFtWQhBCQpSYhTSdW+h7cl3X8Fps063fe4nYQcA2AlmFVgchG4OM5mtoRylau6ZW1dGfKSNRpc\\\\\\\/yoZUfPRs7K7r55tVFCczUjIk+MgcTJFEz\\\\\\\/8VJShBIjEQRWaZAEWWFhldbiosxZYQmeP7A9QK9gPy95sLjTGUBS38zEIuQKm69x33Rj3YC6ZCmUpToOvVI7ANstVpxrwvCDGYrf5VhHQv7yunyNiNbt8DZH7rCZVcdqN6U1XuknRhn\\\\\\\/xwrS2rDBy8Lsvbw9JZu0H6jPEKBRriA3RbRIzpVfnN6ToTla4+ttj4BdU2DjEjuInXmYyHiNtoLoFWCDBY4JQpcR6kU8g8ENWHBMZ0aS96xReO150PbB7K3RVpVJ5o23M2bNaX59sfpymyBpMffR3MBdyLxMwvp0yGiM+2kurRKaqdbUhjxcdlll2vExf9rtjgf6k4XlnskFROGSTOkOoK2woX0O+5Ij6oZ\\\\\\\/jUnB7VouIeNl8sTS0CkIzINUOstRT\\\\\\\/DSAI6Wu\\\\\\\/W129AvCYoTGXugq91ba0HSWb3NEA9e3ivblWFOMKDfGePE+faFNtbors\\\\\\\/k9mbYLgBBEs1uNvpFjqgic0sBqkrnRy6pK2f9J0LnvCB8QaYHPFBDhzPUXPufffBkpyqVUz6gwHj\\\\\\\/zSo+f9t9OzcrESCqSSzeEU9Y+1ahhDsgztJo7pEQEzN1\\\\\\\/q3\\\\\\\/qSyJ\\\\\\\/85CZVLTMmKSwQ1HW7maiDPR5zAKoh27x6yk\\\\\\\/0PsbyLp46l6xN\\\\\\\/WyBNZzB8JQcLe4X\\\\\\\/108e\\\u0022,\\\u0022iv\\\u0022:\\\u0022bf3qvqppoCCRTiqquv92kw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e8385164d8e62a852fd42b88995c3f7b9480aa7fb88ede17f624d716f5b96fae\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xg14eiCTgbazXq\\\\\\\/38RrCDhrJQZQUXmB9S4Z\\\\\\\/RAAEkpepqzIyVtK5kQ8CUvP\\\\\\\/oCuzY5RXrxK3llEYwj\\\\\\\/OA8kIsuVUJ2ixAsRy33NoHPsgTJ4Rmq1rMjy3W0D\\\\\\\/5LhRkY7bGL7s90BmmC+jlvhqlFeSpvviHWfbW+swppLhcUAy2MLZsy5nAEbR6nrJscvuOln5YnGClBmXUMqM3eJwELh8iw8qBS0riTdFGI4X14evNZNtyx1NL8G+XOQfwWFbieirVcdyqhuWEg6nuIjtPjTRVgi5KSLU6IeGdpEbO9l2ERUwi0oDo7f70n77ewKYVtrgKXAMNuCiRUZzQqUHeTkWNxu0X7F\\\\\\\/gVFJ1+FXFWmacy41dcPA4KcPk\\\\\\\/LELcLJ+F+BFSbstADZu\\\\\\\/G8tJ\\\\\\\/JiFkNzkw\\\\\\\/zM+1MIxuAKFrQBGNrmx+cbrtj4WXhWddPkFhXXah2Oks0pzG2fDg00a4XkqekqCmHRbVYTcQPeBnPxm3rNVn7euR1smr8mwf6ZNfZMDCG6SV\\\\\\\/hRSmXeXiDDbPAF1cNbBw8ARaTxkLoS5KYLSQS1hovLsCoeews4SEXUS6RgUyRuBQQFHUlKZb7tjvzMcb0a8TLqgY3rHK0sEI64cB12w2lLdogTP+MZfkZb5WWmOAPpRhsWT7iE85FuAPmtuFkY56JGnUh7ag5aQezbv1lUUQc5EbOkqmfpLJ8Rl0M+oRnyOX8lRA+r3fc2KcI9TLOuZxKtiPy\\\\\\\/w\\\\\\\/HxWnaj5udpv0P6w7NlkiX\\\\\\\/L25CtQ1ziuZNzMuLQ1B0P0o3cK6SUfyS7dSoKU6v1\\\\\\\/KOgBBoQ6pKPiH7H6jthdt\\\\\\\/Ok0wGo\\\\\\\/MSl3kG8uxgC9FKw8JsOuRXS1pU3GddGAjgXJw2wO46gV6RZ8G43BGV0GbNvDanJCOzgog0mVtxlqNJUAgb6PLu9FybZVX3tYvn3zram+ZPWEClNa56eOgDBb2cHZfSV7THM+7L1lGJQTAche7s0xVKmOE1EYNT+PDv9lrQ9RMuwaqK7sdbmFaFCKciFsa0XhtIGDPVy3wtvEIR6XFPygZVVCK61tetlcCVAaB2SPT\\\\\\\/Y+IlNN0eXeOn1YaS88EwrxQEt8tEfZk1tzvQiJaP+W5H1cgMDxoqP4\\\\\\\/7h1HJTYjKHD97Tl\\\\\\\/+g1KOADlcuTi40yD2ZKeDGFYwdgsCM4d0g4bFlH\\\\\\\/ii8sw6mRGlOgpwNcCzEfH8x3WAIt+moM\\\\\\\/kv45DUYF48J0SiR8Ethpl5GisO\\\\\\\/hIIYveBARp+\\\\\\\/OiaB+8fnFOZe8V0D80q1UYeKESXGZ+vhnMJxqL3d4LgcXTeCG9HwvASJ6AgikhhYOb9YGzWes85eqJunMS8r5xo2UFQlYAsFD44OF+ipAiownnD\\\\\\\/+f3Qbz0Cx36TXI6srk9S+UpTzTVOr+Mx2Rd6uOhWQ712duysBSlubsSWwZNIdQh1Aw8fEIVndS3\\\\\\\/9mwgwdOpOHYf+4InLTsVBAqLq+\\\\\\\/UjTFyLflsHKTMyytAnX49aob\\\\\\\/+o9cMFJBAceXEOLrsh1dl7CTxEstE7jAD4XKyIg0b8jAOl50nFliH4xi+y\\\\\\\/TmP9xBFuUWDbkX4OzMHIDBw1I4KfEoiQ2\\\\\\\/eCuWXmSNnb07N5\\\\\\\/db3Iv5ci7qLTeSyDWHFItfJvulfAP7lEL5mi8oW17b8lhGxYT3\\\\\\\/JoskUMisLgMhgwRPjE6fvqwzrEIL7IPmWdParPD9XjBKp7soZsJ5Y4CfhEMn3kt8fd0+mIcwR6rAc1EaVk9VHDnSEgMCnQE706Ts+Hi8iR1rOVbl30rgoruyfMG4ej0yPFE0Im1WQG94gzvnXg1Aaf8Eb\\\\\\\/KrvGbPhN48u+yNPvTHg1vrAaavdIlzTH7suORWq8hwfa2fIM1qAkxpM4sv9nUMdht+VJS5DodOq7UG46I1EnqEnTwU1atejnPAL+0+ItL+kyn0SWFUMQSObDJHAzxLf\\\\\\\/KRXbLdTwS9oiL2VQqU4d92BBWlPhI7OwTr8JCzeoVffo8DbPSoh94BGMf7slkZQLixF1zo2Df3jlIrg0odQYzHqPuC+\\\\\\\/KTEYya7ycmkgct4W5f\\\\\\\/g1dquOslD\\\\\\\/FMwb6a33bEGDENVBCgvdpCuA3Gd7fUbSIx3S0hEEznxv5aSCnifYtx3WNs6rN6MwdOmFFgNWa4CKLuNwvHQiSCD9y4SeFbdlvbIG9DJAobfF8l5bnXYznBNkGZY+I0YvALZ4NHXjXTISUUiheVWyUYDKwY1z7T\\\\\\\/\\\\\\\/pfkTJPjHiZDPVxRdxrnUc4Cn4SZCXaSQ855gXzaGYe8ul2whe10f+2nAOB+dQaF6LX7arkqsLVG3X3+7bwcxZcSVxuQbpyyWugbzILczft03zIs\\\\\\\/CLCVG3jJ1MQ9BATPkQOH1AesLRrlY2BAg3qgnAmvpIFLZko3VM8pzvo77+s0fkKUbng82MIi6Bzi7dyDtSZmIU4GKT\\\\\\\/Wey9A3jf+NOicOobhg4XC6ewhkrwZ8gP9hnqPkzd2\\\\\\\/MFRqvVdnyAQ5X9xCF3RKdfekIwsvHqxUkixEIjHp29JsfgoBn1QoEcDRO7970Lo0D5Ti7QsI5\\\\\\\/CeQ9Kov+nZSACJ\\\\\\\/J4B2iVA1nuOFjjjXAanOnSgnFSmq3+QOI+KdRMiIjGhglA7jRg+dJroAiO\\\\\\\/tXE2vW08P6d3MCMm45OYm03S7g8g\\\\\\\/TYg+HxOZJjHvUUsxrUSMyQXnO4+72DKgbCsJtSA+YMmYmn\\\\\\\/mgS3o2K4aKNwV\\\\\\\/kDXZFnzT1ZkGpYVtSadSeb5ZZUY81SjMYhDSyxjnz7DHn9g2mtidBYBMuwrCXSaFFi\\\\\\\/6v\\\\\\\/HtMqEITrzF3IceUMfyrHJWdty7VnBmGkd6S6WPqmWlUSNy6S\\\\\\\/yumgpK3+nYva1x6WuKgXYECP2WGnn5903ly0SkmDM0cyDM1\\\\\\\/gfpQkmfN96ivzrBbMdpKnSrjaNoxBFaF2tuJc0t7xGO3gunSYf6\\\\\\\/giCBuzj+XccN4GUhDrb8w61ODkHknrlY8Vgu1VHnzRr0Ev5hbV7gGNWObbomesmLnIuUcW5hjbhk1nnyJ3CoAeeW1YouzYBY6vvXG\\\\\\\/lW87RKzj0Gj2+YjEgz9\\\\\\\/KJ6S8k\\\\\\\/cqvHyaT1I\\\\\\\/tAnN4Fuc1juITnn9GUuMwUN5ifPp8h9OnAFG+sQrHZ67oApcnwWeK+kwarYY+UWIWUvkF7\\\\\\\/td23S+lkiR\\\\\\\/QWE2HUVJ03+Cpd1TY4hi4oovmpzzzZFZlBQ2WYHQ1zgFsUx3HvFm3+W8p1OgcdAej6BMB6xGgN4\\\\\\\/SSGWyUtxcWYeYSfJyKb3xJBVi8mEAiUc1JuxygICcOiNVAB6HG7j82Fuv1vMnouvwYLEGUA8F2dmpNqYCwjHUEmHZ4pz\\\\\\\/rl+fQ2IzYTnuq1KFhvboWMN1YQWKltbyHVKgu51J6C3lF6rYkKacTfY8X+gkTmb5KIX7KvfHoaA\\\\\\\/6vUbjq1G+pWnrsyq1DCtH1J+6FUCRjAOnzqD24FLDUrS8pZ4rk8zvboE3I2nASJ43S98\\\\\\\/FYHN0ZX0nW2QkhLh1sRKIsoF2QjxTfIS39twfn4tVo7RVpya6QoEtYch8Pxb1YaAkzSQuzD8O5KbM21uE4HbhEScBzvG\\\\\\\/11SbJy50b6Ky8QQfVdhtBf1J2QZ9dw8gQsIbmrSvbhDR\\\\\\\/wv+IVPkizpghrZyCwo\\\\\\\/OvUOh1EL0+fZqfj65agKwAVsP+QCHwSRVg4FdOTrXbu0WKZ0iPNXhbJamT3ur7ld+pCyR0XA4RgmTstSGbjZ2B5\\\\\\\/cbxAKFJoWs2iYk3H6eTDt6+jOsPqZZfwgqyXbuUt45UJgQCDSfHvdR8wKpZjACjjh2sFNlcrL3tX81hJxJU1A\\\\\\\/OQmvnIZ6QqHN\\\\\\\/z8QDQWn1D6Pk\\\\\\\/YFn7t0gT7J0Qm09nHWRe5sewdeR1TWDciyyYXQ4UhThyBbwvCiNPydYwIDROZPh1kZMLS6lMW9DEauBHIyXZRnLNjeUF5wpWYrcM6wygkYQqVs+Wok8JD1BbtWw5MWTeI7Kg3bjN4cWW2EVOabZOLIkpUE\\\\\\\/ZmrSMBn5SBWIuelnFp8gTltmy3Q9WIPOOQUmfGN+kAfL0xpR\\\\\\\/ysGC9pj23ADpS1oSDAHkVnz0tuU6z2k5UbFkOlzkLlyfuC8bkg74DHbxgehw\\\\\\\/FO6n7gE4zsOMZjMV7MQZVjjN0xMC3+rG3wbHAFfsIzFIUbrgLKECxKvtcQFwYNf7NpHvWG6LmgdC4SfNX23z7l7gqL6OqPshRbLd4FnY3Bx9Aqvzx\\\\\\\/NXECtMXDeNO2zuy\\\\\\\/n7NGYrCv+jWw7gmva7wd5SI9e7AbRTGKDlbD3+MIsLT1obaZSLtkjZBm3tji5N0bUlu5aj2NJhl4X7JrDuwJLPbxSEkvUKbI5+XK2YJvZvzHVE+I0fcSUHX9GynAIGpaBnmCGjEeGgFV8FMJ7sUrmhbRewCrrL88jnt4jyNccpFwGj9G9NdBP6fOQGSTB5Xm7tYXH\\\\\\\/W1oFHrk0jbCVqDmpgqIZXl4xQ9J\\\\\\\/201xB6sb7iGQhGERsDRFpfjoSwSTfw4Mwlh\\\\\\\/UpGA+KgjHdlb\\\\\\\/HT2SzsMcy+uFQnyr73LKcPjDBgBwutWDkvYAoaQNc05mrR\\\\\\\/v0+qp9fHVm78GjFQJ1wNbfZR5ezcCylDK982jpfmGOfKYyIDbMDqdrzyHYg0u\\\\\\\/v0NfCsyF++t1f+dMjfKHQTaXA5jr9U5UzSsKWfNMbNqNmlpY9pTnr8AyOQyo7npbtMkrZ05OOb2ZTyxHgGLLTzT++YtC0RIfG3jUJPpNm1v+bNssW1YnVRt52ADkO7AusLLW2AN5RHjDyo2sT\\\\\\\/MdC8wnzQuqHVo3twZywyoU2zXjxT+Ob1ZKJLPvxk+Si+AjZoDrUMF\\\\\\\/k470O99MFuEsMTZ44XZftDZyX7o+eAcejfZJK6EByl08POQUDiRefogQGcxDbPSHrnJ3XdD\\\\\\\/ee9jFATRCd+tCytSXkTGKjAP4C4AHGZv3KmDeMtvTwktksM2XgtDhvixdV1FvUrNV\\\\\\\/HNHlEuvw3hXkNm1nJT5jp1qAptI0JZ\\\\\\\/0QZyz0ggAwbpwE\\\\\\\/azO35GE7yCaHo\\\\\\\/tKGJMeoxLdjWjQuxcfueKcSQCKqwPj+OLprHaDAJF6VjM9N28GxklNH6D2E0woKkbNb9crHZrMoUpMeFiyxCIzHFUUiQpGBpBNeLXJh0gyik\\\\\\\/DfPnubt2ZQ8yin3cJiLVgt3pxciAw7nKzZHSdRuE195VX9btVfKKR3L65wy1frZB6UW0ffR7tVoQX6H1nzxF6tddoj5w4wKWMyK2abf1zy3tlK0amvPlGdAgyztuW69fFFB6y7nsAXpOxd7NCmn+0lZ+7u7WwvIUxwOmw78Fhs7wmJY+gpBHEVMnX2J0+2KXBlyGH1SVp3azn6vxmJ52fJbYfGuIArXjc2b7+ksBHuZTiy3+7XddRHGezVHq7synAQD+R0sCeAj9H4xldXbInx0KIGI54FFAhNblzlxkpTt\\\\\\\/M4ElbWWyb+U9WpMqfeMc1cFou\\\\\\\/pLbQtCTWeP9hGgtc4j2t4DLGwWmmaAI3uuEcI\\\\\\\/17\\\\\\\/5n03y5Mvk7yYxAeaghFAS8F4HtOzXxvnnVVICP3b5+8gmwqMSkHs8IlI9SzWIXgYB\\\\\\\/jOerntYZ6tecwT1AItGKXX9LVkJ2Orh7fbrZMSLtAxQiPL6w2pcSZbiDEPAiBGax5ceNc4n1ul1O03NqKGYQgcf2mGjlk1ZQSMu4ICD5Qyk9bKWVsIiglLbtivQWIvwfUPuAARjqcycOnOrNbwvODEC6caQ9LvIHTG3H+Qf16XkxvcJKTyCcNEXivJeHadkVc6tpctMnqJtCa4+Bf1eQcmaSGVxK94ZI7y98jhWpiFmXIHqX5NtwsyhuGAWt8e+wWOhNlDS8rfSev1nePN8Va8\\\\\\\/\\\\\\\/KXxAB5kWuEmkFWZ2SSylPmLm\\\\\\\/MFXnIi\\\\\\\/lbbEsfI3PxhzdF7xwBDqCACAZVMBkLLCYK\\\\\\\/SdWo4OAmf7ZqHJoXYQ5KsHbJre9ALDj5TJOY2VKu3u4qjP0zebeYSAjKjReGpQXPFWd5Pewk6ezik+PQV\\\\\\\/70qTsZm7fgXs5sbol8L5w26zQ8\\\\\\\/+yAx1W+wjqIjfnAG8l52OzrW8rx0W8zicLezwYS3yBBlHRyV5NjPP9zKrPcrcVa5zVkhNGanV4lNNoDTtoImvN\\\\\\\/m+Fx+RUGmGjmmRnEhgArQxmYinFX+Z36afhU0a1DrodMKAXtgiocHIrcVH5jjYCRdv6bUMeORfp33Q5\\\\\\\/ZIVw7LngTp1twV0hDzO9SRq57SQd05LZuygKxxPKxfS\\\\\\\/r5TUyXBsaiD3icVM6bMGxV9qVmeA3l3Zk8QXNusnfnE+Co\\\\\\\/OjiZ3YTTP5GyjnqGhGDo4FWlJ2J\\\\\\\/70hR6I8h4o0ZOwqWtdR1aEcImd23xfkPLSENIdsKZU0gFkoktReOY5cPwgyRn9ljv8D3IaBcG0\\\\\\\/ks5vG\\\\\\\/u+nzA61CIeRb+UfHSwwAcVsj5UshdQiryA2TghD2kRXht8Sum1E\\\\\\\/N6O140gK93aEOrJxfgSbyCm+Y+5\\\\\\\/tKoAxUIpGFEoHLbkQLOa5KEXtiHgb2+ReMPCRnZupUQIZeFDP6rH2+0hVFrKCZRRwc9k9ObUg+\\\\\\\/2KKgeLWiEnwCYDZlT3YFSx7NeXxKzgNF8vVLLl3WHjSmH3+ICUL2UJi5vmBRDm4BQ1cFky8hOMlEInq3GKkelYtMPfoW5prgAyUQnBVsjamKTnNsVBTytAPaf0kUAtWn9fwoI71WfQs4SJcWvelLRaWj0gl1RNyVhqLikI6PgOP2MjeEKERoMJa5\\\\\\\/MFRSuVKqTxs+QXK8vAYlzTFU9RV\\\\\\\/eu3cZorLojfUQTc5RlSaLDMYbr5pg8cgY7bNd3PvQWogSKJCQQIVc98C0THxAqTZtVSr+QOFfjO1Je8XJke5Gil\\\\\\\/nyPiLI1D39QPk7HgtocW2xpMuX2eYu81UeVe2nRGuDtEUqmQ1nNhtgTreQTmadzCpjXYkBEjX\\\\\\\/J3VQNWhBrHfybhQvOqNzpQxgKHWDbw0ppkTcyvwRR3BVRua2bCLBOTh7N3YyPWFS1BwNHSQVhhv4Y5RVBPP4080D10CV29eVuKZueoIaNyrlPa8SPHKTEw09RVupaJTI2kDXijEfdk8hrbtIFI2h4gGjMNaaQSuMueUS46UuluSxcljJ2w73gupJ2xYSCjEzpf+\\\\\\\/cVxaWxFOnuBOTnJ6s8cR9SMpv4DJn9pDO1Q+HWyoXVbwcwdspH\\\\\\\/fjYi9hI2lmqvOrZ1SUh9oo57CXPVjs0VymMOjr35elMNjW\\\\\\\/+QRtlsCQL\\\\\\\/9V4x24bw3BHkNdcrUWnP2jL1de6QB6sxf+8TNWWFE68HYUiox7lv8\\\\\\\/fQPzJc4PtShV1oBfbnYPZoZnyfpWrKit1I29IgAoXVCwrqW\\\\\\\/qR0Y8WCfojJ1D8zQP1VKOfR+D07jnPL07WANahH+bKwaPXNzbKH6VcsxPWG\\\\\\\/Od5ghXvStGvP1tTBVasqCQE5Q4bPAotCJalErVlob7\\\\\\\/PVDo+7QxRFePBCBxWwYuzuFrL0CrbrTMHZ1jnqC5i3VacSuC6RokK3l8gR4i4gXV6uLkm0RY6T9cGNxniiB5u2P28L6cXW\\\\\\\/y\\\\\\\/Ia8yDu2Gc8VX46NiL7UaS8NdJvsq46ULO7cB9hSmdGSD7MIOgnYQw6FsqhcJUs5qyZQ02vESgb2k7fjRNuBOduPGWDYSGPBQZgGxslKWg+12pFdZaiXimrADODedJAxHXA8\\\\\\\/NPM1mhVsS9blQWlQgZ34vvtI8Vz3RCni4eoJgha4e+H8us4JbrvZHqAobPYz6dgDyXoi+V7m5KJ9UD2t6ErydK29Pw96Up9pTFp3AlhfqmxMg8jvW9xxsd2SnILk5YRpWTIwWX7ak4NwxM+TfGo9uvf8f57tRPg8EAjgqVDLz6BmtFhfxy9HwPOyjEQAgumgrtqHVRRecewlH1EDl0+GCAlo3S6ZlokfRxXTG5vVWOAGRlVY5EDyuKaaezFVNkwmbSvTlFJq7IScmnYtYKFf9Ru3zjyn65rChsuByOcPRtWmOuwjDSJzU2TXGCc3DxCay\\\\\\\/R7ntiz3x++CQvdX25noJjjwUpScDnG5ZBQINOfjR5lZb\\\\\\\/MM+jHCxqUMKieTYPk0opsy\\\\\\\/DIuog2X0PQZOaLfRLfn9bQvqc2tMhQefWwftPOBcs+OOHLcquyy4KAMSaQGRxTXLWsUXlICc03Q1ckCX\\\\\\\/lmHtEgr3\\\\\\\/nMhW46GM1FOG12d4xUcsixd2QB4yZyNc05M8oCen3Wu7seGI4qRoF+mQLCBdLQSA5HQWVBZ92e5jaJcFjjfBQkQ8J69is6CJ1S7w2HpPyD3pPzpUo2aZZ\\\\\\\/i3WuUXg8jwmOQKq88kXO3bgZgQw2pGEhoNWndoyu1p6MraP2CFkKWE5nV58h9Qa0c9WRzTKzgtf+HVhmAx+c5zcrDZNqN1ylUxoOWEeeFwwuvhGjBneg4uEV1kSQX9jjZAniiqM9JyY+rzg5EQrMLWR1Ob4NWfKw4u4\\\\\\\/EqnEZAQJrS6XR2lsCj2PiiAzgDIMxyLIvgaHj69ukhjxcx673jWV6uzSi9SBXFpKftYGwHsu82HCFFSmYrFR2xiFrSook9SYfcBq3zqsvvg88PYKd35EAEPBLN\\\\\\\/wWNDGlBqzIrTOXyRDB7PAlukO9BpuPRQq1Qo4BA39xcLSI8mv6mexljpUxaaPggflG42lgpPiz\\\\\\\/oGQPrHxWS1OiCMBMGg3yQvSKZjZ2auqUUeIpC895MMM4U4h\\\\\\\/BpPcmeAfl0TpAdjLVwBRlQgIBvTGYk\\\\\\\/59gFpziHTDk7g2X8U8L5G4Tt\\\\\\\/vO6wy5FT316\\\\\\\/sXl8t0xO\\\\\\\/2\\\\\\\/QK1ZApNoea0enrwwl+zSGyVeVYI05gVsmbg5pq6c8CV9\\\\\\\/m8ofhMKiqw5aBmQFMVhtimnXd2VvOIWkai1MF05JUCS0trGjLGm+mjphCVyUmVs9AJmCSAFfbwvOLrKScGgB6UpKl78AuK9YfZGahYS4ieEEmyxu65PuA+JoeZJrq0fpsN1fuoRy72S+61ju9tKBVFESgDkRqrwTi7pXaw9UxbYnzw+i3yV66Va2vtXfZEvvy+KjFFXV9e3FkNkxfUf52RiY2EUjTFcZaeEkQe\\\\\\\/bOESgpQBKbBid4RgH4EjkiSmZT6aZhFtF83hTL9UZyjdUvecuLOWCnQGT3OBND3zduBAzOmhm5UTI3d8JKlLTp3f80tfakK4mvMnMS68LRkJTJ4DeqkOe5JP7JHMesULSkWooI41KgCbhdJMHwz2dCh1C8rzHEWm7NQzIqEdioq3btUxSQoWEBa4\\\\\\\/kixIq5rhkyi89p\\\\\\\/0xptdGP\\\\\\\/+NdLzCi7OgULRkM+XPZ6FQqljUeDMB3bOq4rV6fWlEWLQXfqzcUz6DW8bppArx+vv5IXGkllxXDih4nsfEefYGWSyEUjRvVEMidTysFZE2GDkJAmU5a4kHga5WIqLizrxoGILjyVw05+OR2sAkukmgwjzP2heA5pMc+2rr3H6+ubwMqjIXByeQlRUK3CZUm5hydbhaORKQp3Tc7dXyBPqNBsUA81WUbemATaQOq3xyNOCKa8i3EcxR4dVXRtkzLU6MrMY9sH7MyUDkdzfw4ImgV3sSFE30\\\\\\\/jWAZdElvY2BMg8btfXH6JVNzE6hkejpVcrhU7H4bg+Mv53cTjH8Zg\\\\\\\/r7GnU2MOm9ZnFiSL40SyBdaK6CqphQX7m+JlYBmPiXrb+Z3puJaBFhiAzWibIaqfmGPQy+GfMpb0PvGBp5Pp24oBSpxAMNJ3iEIqYSAtNuXo2H07XXowbNhlD5TNwUgMBXJg1PqpSDMyui4IiB1ELMsyAWvSfyrPOPNpLi6dk1qjpyMPhw4R9cNJW0mcPZFzwTePvkssqpPrYAwno9aa\\\\\\\/sxyfJpGANnbWSVSj8Hve5t2BHX9LzPkKEW8oQWOEo76TKs\\\\\\\/zapjhJrro+v39G4CBER8qqmGtKxxlyxXCFUJqq9E87XqZC\\\\\\\/kg8ilaDnfJbL\\\\\\\/Z36VkO0xWzULOhgi3soIPuttrQjzKTCL02tEU5r1rJL4orONp1en1bJTNaPm41C1v7cdBT4ukiLNMigunm8vQ2ymbdnS0gyB52OfQYp2mMPIbu\\\\\\\/ecaDcLrTe7NfbEfkfF\\\\\\\/OYTfMDWdd9VamMAFSeoa6ianmyYhNbwKpxsZMOxMTwB6Qq6XtLbgNVICwjwu+kF6wqlVotj93vhMiu9O6rgkZuBFln55gGDotrw5eSjbQPMYQZ\\\\\\\/YUEf37CW\\\\\\\/UEifqNlb2BEkUsgd9ig1xuiUwUQmXDLJvSpBgjBlFQgjx8rmzr7e6UsQo4x02tJgoD7tpZVGV\\\\\\\/0TR3dbq8Daw5wgN7gVlozdxWhPbSYDh0zjfv7IQOIDZo7jX1djL3pqmnXaxiTuDx1FwNWO7FYAsg1fiDdTVkjmYjQcbnQ+uljVjhcx53y0\\\\\\\/MaFddBoNQSLa+JQnXRuA9t9THWb5YueAyGruyHs3hTNcmddHAd530rmARnQ7gaizl67DddtFWcaXHKRGaP76UfuB8pGWyvjOsWYxjmlDvccT9UbCl8M1PwPCwd37usH2dIJHXy5QgVuSzTGV5XAAyrxGMAXiQNVqbNM\\\\\\\/tehjLv+ar6mPozKCxxgQWkfNp\\\\\\\/n6Qnaw1P6gUCuFzqTkzi0Ub5uz7oXWVrb\\\\\\\/e4EYtUS2nfLCYg6vSAdJyLPVkyQV0kvMVIKBxRjfCDzGnq1c5ojnmNvM3YLn\\\\\\\/u61pXCv805yCcQLZ5eVjJ+F1K\\\\\\\/0MZzA\\\\\\\/WY3XGe3+1tGRfm4mSeOVxAsS\\\\\\\/Q92mGnbkmTauKyUs9B7Js\\\\\\\/meaetXNpsjGHwLuTqbOVhHmpuHsEi6Jsj5nNHwlgutKdH3TAp9loyZqXDUhbl7tY5gDOlFfJkc4OhQTkej1x7PcpPjx\\\\\\\/GgXeiMksl8\\\\\\\/2xrMmPxgnJ9fsCfj+JfgppwWFfAwbyc1qiNkowkXppqbgvOYquYRvbD4Kih9o+a\\\\\\\/2DWUJpZKb0VSiEiw00IhbXO8cKObGp2UvXm8aS2n1BsVbEV2fcMAS9fBPyTmOrRNMAH+vnmcfOpIQCI2MYi0hcEUVBoJmhGmAfkVtaexpxp0Y7NzKe5rcJFR844X7zzwFhX8fV6yB0n3A+LbRl7g4YebZmwEf4uNT6pS8PPP6M3OgYnCSQOuy3CZB6rvbskhBWw1t2QEcgNcTRbWVh1hqsu4AttoESTNNIUEuZtPAUD0xiycUK3RgYpChvEX5vd8PCvLO4H89WrN6A+ZFDNseGcQ4D3wMVk4CajQQLqS41T\\\\\\\/T\\\\\\\/l32jcdglYHZptl\\\\\\\/uIu9u2sJGKomqmS0vRmN6YDdE084R3+eQUJMUaeGn0YQh\\\\\\\/vlFmkWvEjxhFu18FqyiEIQpA2KE8zUTmPfl\\\\\\\/b5J88XSdqAnwUCq\\\\\\\/5yb3FFxlLetZcrfqIUNnQ2WgiKrlnZIa5F4xxFcmuY9u5HPZOEnIUyutt3Cn80aKMcmxs146MCFT8Wz49sZMrwU54oe\\\\\\\/EHiV4zmAeZpdJJ6g\\\\\\\/aurqqX+99aSVh110+0Q31Dsm\\\\\\\/CJb3pS2KriKzf2lEZUAg9Zo8+3aB6s\\\\\\\/fKRaOkIki\\\\\\\/YadS6oxkhKXOCxR65u7Upd9T+zAH5\\\\\\\/WPF8ygRoKyZuR4nURZEmWMvUq9s0kwUbgu44i67A4zHl3e7owzmgxtmdmPQ3HhOSBMktm0INE\\\\\\\/P0oM+hJVFBNLLQq7jguk9e0lyhQ+JRf0Y7yklE5Yoc66jCZZH9s27gfgTbOCIhgANwr5I6TWEAsA96EAaVXEpoDPhkFuaZbtiSPcuOZjYgJLUydQocbknp7bSu3fgl8yQyagAyGm5FF9B0r05eMGZxn8u0Q+pgBWaj4Mpukl1GRQ8jr5t1wiV6e3dZDHg\\\\\\\/n3IPA\\\\\\\/EhD\\\\\\\/q6BcC\\\\\\\/tz6mA\\\\\\\/QWX1oPvkxHnVNfPGQvDtMcrf04wXtDP3R0ECXr1mQp4w7x2En8T00IKksKJMWs1V3B+93kR3QKPIZ9EFc90ZaViwEKAIOJREvtpPVCRxqOlr73TtNTURSskNkD00ulLBlfB+BNg2ehTL16Mpo65ujaxMrL6zJ7KC+B8k8fnyy4SEyh\\\\\\\/vlxKM9Csm8dw3vLlSO5EXSDnfeMphqp7OiQpdPM3a884wslOwT4ECelONMpTTxEpJ1Xb3VY\\\\\\\/MBK2r4o92pgxOLekjFYDNkfhiAxLabZPpoohF0uPUc5tMMPUk5+mwxMl2W77khxGY7KlCjJX6jacoWg9xt6KJGDc9Kiga4bVvNGXoxjvcZwymA6964+9ERRZcWz5bbtbKs3cW3KQpFky2dqyNXT51lqDsdC4WVG6khvT+s6QxrWozzxqIblXsyChrdQxmxaDY53OYsvnsYJk9Ut6sABqjNNrO78XndThmI2BWbNMJ16nHuAwfDeR3VbMshLtU6G01JNum0\\\\\\\/TjXzhrSTkeo2FpoahtdSyujDqi2gwu8+wWYNDICEwSv7U5S4ytYapMphFUp5VUGrHSaL3xef+31g6v2vkwWn0SyYQTYN5nSbAf3GuMVs9NKSScU1GzCE2h1BvJ1x8uBrx3qwqqa3kkWE\\\\\\\/VsEdNV90xK40xOzZDxiaYRYFqLMfDxD1yHlptwY0fKUQjl0MZNi71Hyt8N38CXQR6FmXIkwQ7JZMhE81TmjXdjmu3DXFhfux+2qHnlF2fj9\\\\\\\/0rPXoqaH0fqZWuQn1fLlzL922fmwHl3YYr2QvUXSJSkyqV0qLytnS71m8BjhZ60eSdduA9AJ3KyvxeVp69k4hNvGQjYxiP\\\\\\\/8vf7Bau9OLXQTOdZqIW8WgIu3tsfWc24y\\\\\\\/YavgE2KguyH6jIGgoMCAjLVqdel+aVhSYaifoU1lo4Nk0GuoT9NgpbO1ZNG7NfZ+6i94mjCMe3LxrklHAkM\\\\\\\/ccZ01yzei9B1cXMseOYnq1kBly+WNt7Q6MjhWvhPoGmIp4GlT23FkwdiNS0w7WJ6Xm6ooPL+OOw6PGsYmXK0cBMeufnsOYEhafybSAQITUr5JRH5M\\\\\\\/B7eXtnK1QGtp681p\\\\\\\/svlN1Wfr+us+oNHfpugT0Bw6lsBweCQmEm\\\\\\\/eoZ77qc4gv9vAOg90x0b0jqfuPxr3Rlx6P490kI9kDDuGx5sqVSgms8CWfcACSihVGsiFGUe9\\\\\\\/cxFcgc7zeCA254aH06jXWIiLKiJK3zl7vkb6SxCVBxnm8MzDkpGMw5bMeZjeZ6BQeV6a7xDSzvjcxmBT9QskzI9bwvqHp3YeVN2ZqcFDqR\\\\\\\/cx0KGZfSNlIk3X41kJZib9NbZHF6oxHWvQy0nSm4i25akyIiORk+N5Ijsg+wmnaWmPOqxjdZy5TP7QDE1wVULKkEpVUcY55ovHuPdd8ipnPNHcD9gd3dn\\\\\\\/YmRJIOdDsYLTx\\\\\\\/C72khmXAQI9OOkEoslG97WZYxEK7GQD2qqRkl3n76hGU0W9SHRNiw1Z4fpeuRbp77cnVDvh6dSl9YyPqcp2Eaji8APbmAu8sjkKTru4Gq7zz3FgXdtAqcHg6hhVkBC1dOQl4ddhhZ\\\\\\\/1Ea3mMqS5No\\\\\\\/gBLptN39wabI2cwHp3u3DIW\\\\\\\/87O9O+mLXBczM3UC3YmF3XoA6rnyBv0Nig48B\\\\\\\/utam9vAOpd\\\\\\\/pA18nT9ydjZpkQ5rjej6CS1lOuM\\\\\\\/OhesaOHLVYzm6WuMCP7Em6jdHtSIL9TTQeKBXgoPaB3SLjJl6pkwMaJrXJQlxZ+iOrCtyAW1wOtyYiQ2\\\\\\\/6uwsIc2VjbeCjZBfrLbjcuXGwnh7TXaVTgw+6m5fGbRamMq\\\\\\\/rwBgcHfP\\\\\\\/QVje9TT6uSHVHvbygS+Mho7yT+636Mh5QRLpId92hDeMnZBIVhAoFhW1wlf2SRetgvdz9G9lsLRtsl7dQvSSiWzcPJbGIZ3I9VyErctYE4VhfVa\\\\\\\/8CxCIiIaji2qJ4ZxH34i9d\\\\\\\/K2t1XKCdmGP4KQnC6pkA6pcfIv3ilAWwhWMQa5EO5j49IVG2VK2eEJqRBEVPFmsHsbwC0BODyTxgfwJKzekT+fOaAvs1KVQktKmUpbQR0iAtF2c38qox9na1dtsRikUsQaRDyt0p1vFYZTUORZuwHR5O4oFZxSB+jzXhvbBObzZmBtuLQ\\\\\\\/2AQnjvk99OCUaKHc88fuL5Rp1yB8MS+hCxvIu8NMkuevWK8fv4rRAXE7mWbn2R8ndZtsPyEXDRnFh7XKxiGRM\\\\\\\/lVYa5XOk3S7P1\\\\\\\/dHN96Ng9bdq2H5AegJEZLnH5QI0aSWi4dYEEkAoFtEb9\\\\\\\/teU\\\\\\\/SBZmJrp4waWd5XN2Ik68wyyiWfB8GV9U6Wc024tnS8rUt1F8TrPK5+Bv+Gw+7JLZGjSVBoEaHjMRm8\\\\\\\/MquwKLSFej\\\\\\\/Wz6E97lkdmxVT4qt+nU8evChuauWFLkHneR2+zoEes8L8Vd2wKAYGPPaahKJmWNzLDSdbrICqO4sUGuEsLA3loK\\\\\\\/KESs6Ni8zdoz34bO5PGNH3hbCh0UKDOKYRF7EIQgjGguPPd07Uawc8XwRxVTNDlenC9\\\\\\\/ygU8w8X7dksUkiDJDiuMMmD48HcN4GEF4fDx8DHHx8p1xlC5O0SyEWsjWtyqj3b8W8vhrrEH5AxL0qLmOd4rftfRTdemjjHvwqnqInTK1bteNhucDeNuAdT9WED70kaG3VFL3RkBWpbD2GwsuhPnSrULp307fd+3ZLM8fnN2xXoXAFX03Ry76f0xE6vWucxkrWy457fcrOD1Pjq0UTIV+Qa3KoQFZ5qXqckuxxJlyrviEFtMq7bcdX3l6cktthfmNKfUpYMCVIMQEm6HMV1pim0Lwu7CCeAMKQb6R5MBf4a\\\\\\\/DJvANd\\\\\\\/QN2lfoBDjN5n+XP92rIVMm9EwrMOF\\\\\\\/+mGA4nwcKZeLGNxX0e4eWe9HSwv2i1Ho73Rj6D1ehm8ZmR7KK3ROvRzX27hD6c5QMDkQ1whgGgO01KpqEzzuhYjWS9RruOpLqmKh0B1YH2u\\\\\\\/tWmIbEpQWeAh9Z9kggGPpWQyS4Zagb31CQJXEGYy2ixE1uU8Fg\\\\\\\/MeAmqeNZomuJPfOhNafcM48Hu5MD2Sl65yIq2yJGpcIq0Mj7SNWLQFsvB8w1DFrMlAGIyFOhjfzLEBaBYIYEHjcFyrCo2JYxNKirMY36b\\\\\\\/nx\\\\\\\/xDmRds3pZNVIGjVwiw\\\\\\\/JBEnhtE54I5Ao5MCfAbu7FllYmXrICJZqhzL076544xLIo\\\\\\\/Q+EQAhsv7ILJ5qo2LTFPPoAPmVlfiqYzqM12hk3b+bprIzyh4sOr2Y033BbhSKS8Wj+42NoZqGKgJuIdhOGXUfPDlv+fgapG1YAUu06u5TU7+F1dHIstU9ZFgxh++QOQfDcA5dbTq6Z6dITD9iAlVjhSE8q0Zv6poA\\\\\\\/Ic6bDmu31A6zKo+OrOMiTnxOQbFD2ZMkCD\\\\\\\/\\\\\\\/oL17FD8LZme2aDhauUGXwd4lTJNOD5UG2sCh1VNn9BwY1vsdggb5VlDKLIw21vvHZF4ZA\\\\\\\/8fHvyDjZpQ7LWkD3+kJKUcMC15UPjIyqxyJsze4t57IXelebtH3BD34x9pgu5Iu2DcYgNvV7j7NIW1HTnNTB3l2bdU8P8Ha2I012a2NgBolKT75e5v0jS1UGgGnukMyPWlliPuC7QXWS3+pH52idB9addW7zbJcxQeH9IrbRcdQVLKW5972NK6kJGuOkDekk0E8XatxUAfP9ZMWxL1YXCWKnzdL2Erryh\\\\\\\/FhyDBNcNb2umxAm2VKwcSdinKcRlRBqdOUw5\\\\\\\/rV4MJNT4N4ZC0b2Nw2+BC7l1iqDchXeCoHixx5lM3wPuMzdSPNFVrNcXAkpdGHux51ANyHvcWyu8DNA\\\\\\\/D5wcJVM3nyQlVnh8+SS4yG7h9VlcU3Gp7EAKKuej7oVDAA0vgicqEDIF0uH0xjsLiO79ujOZctBVIum+5OklpfWaW9f3CBQyvVbrMXOYzpc2eLUC9x\\\\\\\/d4mjhy9BSdl3C1P+iixFvn9RusK2SKcchZ+VpLshM1MNn4eb+n5sp0Ppf8CUmXeUfP6\\\\\\\/5VUj0t3eTper6bCjF1M8lEMX4SvFPJxW0kFRo\\\\\\\/Ypy2cFjxHLCuUOeIfNEI31VSI9CMtItIS8DfYCiiD096JlF3GRa0SCI78XFL4E9CFhWPLnYpN6zRqImU38c23Q\\\\\\\/3Cyb6VjXjRl1whzZOO54S\\\\\\\/yonq1PiLtTgq9HrE+4YlWN\\\\\\\/klXfQqqpjcbPiKaC3O39d4i4ulaSgaJcQPqvESFUPKpuud41SXIUQMh4km3GRFB77YCGWKr0k2qjL1dd2YKPzg9oxyM\\\\\\\/17lBrvlLQ7qAcH8ymofLZcKwSWJWDqazyniK9xxKXL09ikBbBRvX7fvqsywtmSstxCQAYKSY4X3lMVyH4xSffxneDUWj52jW2CgmmBK8OxumPqPhFwbHZSkGafLe2+WORTjJ4nd7RMCJDIdMDoSfIjTahL68Bj6INpVyqD8O71M5s7g4gejbo849qy0KWNM0lH6zXw+qkaIRwgGcI5qQaT1jT3Jv3oziJtcz\\\\\\\/tX6lKl\\\\\\\/\\\\\\\/FVSUnj1pLUdF0DKVV6JCWzE7gKmtx3IHHDcEji\\\\\\\/iZEi9A8rFRB+5zzHsCV6j\\\\\\\/h\\\\\\\/SOmXgvScZCHQj+yz2TwFaYFBp14y0INiL5aalujaDX9tDKycp20TnC8+NsfdSGuATm6GvOZNRLNMCNpLQw5ZOqf8Xng9E\\\\\\\/gp8zIPrvdFgM9T9c\\\\\\\/2C+waPNMBQpjwyp+++K3UZsWFXht8snQEP\\\\\\\/70SmP+ikxISxkEr4Dcp0HlgEhXf9\\\\\\\/beaUGh5AVgq4uOINjOgcr7YFnlccIn0DkWh\\\\\\\/uVRozURytl7iWNSRvJdr4Zm9IBtTGV3BMjlMmQTr8fwYAfR\\\\\\\/BWeObFFifV4wdhULJK7P2LI6OtRvj1Hqwlcj78vqsvhu05diqnJRnrCx2uuhlPrVmnIiT0CZ56\\\\\\\/8TnDEORipLl7QLaLYcK+GqmTt3c71vgkRVjmUTA8pEAVH583mhcFY8nqYsOCyXqLiUCgsYt5yTod6m37s0xOeaS9n2jyrXQn8bhXe256p37IlfiaoY8S7DXeqoH6+pmw4806SBEYkZjou1IQTlHsiAzDyxYpqomntPyD+lAUaQwW3h2zTztTsGPZSdtwOQX0RmRoqMhddvsNIf2OuyR92mB6tb+E4qnFlsOo5wNGsq4Gtve6eipnUTIRa\\\\\\\/NDwQafCgSPaC6HM0v4vwdQmMACegcP53P3GBCbDeOu53XVJIORaux8bYl8Bzez3G385sTELnFiLd1hlsWq\\\\\\\/903CgwbOf\\\\\\\/tRbXQj8grjXpZH2vBVcJNNebVfntvI62MAbieteIVd8va+siw5vFjce\\\\\\\/JqAAAlDXHWGWajEugWq4keGIbDGqnt0lSFMn1rsSZTZwf0QPn5WKFogygGdXRf73NRuBqdDfYOC8M3yDrGMc17IPA+iNCXzlJ95zigufyBHbGxOShvzGEPaZBo0JB4Ko+AnC3PkOomqvrlVT5FRw1KGDgRkmR7FsXgGAnq+yHgsv8wjpUeMvY\\\\\\\/MsiJKZlFMeerPt5BgGU2qSd3EmuuaZG4erLiTeWnaQ5xWJ2X3keyiBhAU0jLI5e6KU58uyDVplHkiTkmSL+0uOyGEZV7NmEj62\\\\\\\/Yq2kVJhsDHPszyT+o6gRxX3iwmTlsUimuoIAne5x14qBbH+M0rYQdCdJWE+FkvjOb+cVMKw5M9WhIc2BjUwz\\\\\\\/9XRlAIK7PreRhpbB9s1lxomNGwncwsxxKp\\\\\\\/C0uRHPizG3l1Z8gMcDyWOZuyHZ3NQSvL+pQl5HYtS6pp0qlJyzcPiYf72\\\\\\\/TW626BHfBmAFUtsEhfWNZFXGxyycWbwdnJbax3OTwJFqF4EgObBAEWGhUCAPyuz8mYllt29oZndM+I7zlavhsyb8JOIddId5X8IVtGL+ELitR+1bntQk948ZA8tTQawUWFD5V00O64pfVK1kdEHz5LoFNjnaLyjGFrRATXLB9b4Pr\\\\\\\/b1gFQZE6vqfcTNY3PPMEHx2KJeApaqbmsbIrvS7mMvIq46AjRfFfHY7U2N3nEY0QJJz8Tqj8TRhdiLq0tssgZvpibyJAwf+QTNhpq+kXczUTyyIYcYat38w5+9DVD\\\\\\\/3yeBhORNGKP9oZ1TwV\\\\\\\/kC8Ge6cK7Dc34M0p4UpT+\\\\\\\/nJB946g6\\\\\\\/G98hN89y2YPpNZ+OhYnk1vaX4jZAxQfGnczggDRotOv8\\\\\\\/fJVXCD1jpDF7S1ZkpigyIinCQW73CPNMtK4bKK6z2JZaJPN26Eupfd7IFoK45J8WccUbIKbzNw\\\\\\\/2C+gMPbFpSx\\\\\\\/FTLKLxOF1cE8miq6tvIbltHOWIOlRUZGMjyRHZpDJ3jfPw9eGMtffXnl7LiyudB0S\\\\\\\/DMRkV5LZuH0wGLilzrds0nkZZ+GE60PmVHC\\\\\\\/K72w5jxoFLRdaFTXjfl\\\\\\\/5KmsBjw5JQHkxyeDWqjSYyqU1IiwWW7ynp+vf9gnayZdBVDlXvMgSn81FIINxmLW+S9qDjCbUAfkhE2Vjdik223X8g+DZ8V0yNGtrSx\\\\\\\/g8XnSaXJhNfYEJbXrvMs\\\\\\\/7DjdA9D3wPpBQGiNja\\\\\\\/u4ow3COIB7fBnqhLspmsDKPxL4v6OQ9ukwgqiKqXcvmID1CxEO3ztSa1p9bt8SZErmB\\\\\\\/XssVYi+6yzo0Vi\\\\\\\/jD50SEFdCiGh8\\\\\\\/2AQxMAM1AXP32Egn7\\\\\\\/7ivzFLuFkr6jtTluE8bRPek+knNcN+MJpoT12Ro1q\\\\\\\/OGPwlh6rCekVwBC\\\\\\\/ZG+3bv4mQHerRh6VeSmsGtxCrcigpiVTbNgnKpYgCmMW6l06qbKgBa1f8r4Q1FeGXsVdAjkMG1u+V3dgbu00TkfgJmWKMw\\\\\\\/SdYImSfsACFfFvVXsGzv1Pfg0pRkfJTP09PbMDYt43AyCYHEweL5+uNe8yJ4CQfI3aZ01Uj2cgJAURXq2HdSRnJbnMT4fET9aDtSgmWPlCNjnnxoocDwxu3TkVzMXfSMX85P9foRlhaefVlOF2\\\\\\\/rAv13LaIldSXP216B13KFWtpyuGeMWh+Yj5H52dTFmyLBnTNM6JZT6nfZUcURWlYbCeARW5VXtKEjrOCNc9kMgQnlje6FVY0p0d+OM7168ma9f+q+2XIWvFozoWzB1mBJpnl1CnL2niYqfEO7g2PLJJGr8P3JyPb9Gzn5bZIVUWXwnLUTDsTsvZXl5Ly9aevJIUMHJdZGgAB\\\\\\\/tdaHIGa\\\\\\\/LEHrmZwOBUcc2HbppW\\\\\\\/dcdE9beSHy2IFYgTlgacl4x4QzAUTrFBcXK1NvffnB996gGAr9F4krqg0ZYjZaHhX42G8RrNpqmCOt12584ip7YSLE6TV5zX56MxlgKrjX2Lftuok0\\\\\\\/BTm0ay73p7Q1+TS2Vqn+RAPxrx2S1YAioted5zlQ7fcbCl2uhizTpFYHr3m8C2sldYBgHB6gEgMimCjTD\\\\\\\/w5Wl8obfBUqK7bi5rLJ66lJVQsq+3hFCyMVJQlbg3bi4PheO\\\\\\\/+KFrWmCTdboL20EPvYiP5BO7ZAfPvK5IwVAQvQ4ianvZpgOsj+L+kPghibQTS4eriAnPbXC\\\\\\\/4L219tXJqaSO8x2WOXdybzezqzoVlhECaeJVdrPSjOUABv5xhM9aIDYZTDnp1b4yMOdGOhnU2dXu\\\\\\\/MMrkh9\\\\\\\/cBUnr5einUAWUJQICFbXSAreICifEAWPTeEcYNUofy8Mnev3metuVPdCsCF39wUbPjMLqIbrAhQEkZph8c2ACSo4kS\\\\\\\/woStUia3+lm9\\\\\\\/CX6ET74trDXnMRJuLksswrSu9F4KVVZtaQMvn8aY\\\\\\\/VtPEd\\\\\\\/tUXkvtj21Lwi3tvckq3Pzj6Z37+2bHuFn496pv1yNgI0pJ+nt0BxWWjGY58oon02JRUWfjhpRkhf+N2yhaNc3T5qEO2d1+Q0NJy1wv6wSkEXprjWDD7YekCuUWfgKQ8fxjOE\\\\\\\/kSltjmt51kMzD0ElHnqt2DdbcND\\\\\\\/xCFBHI5DE5T\\\\\\\/wGgrQ7KpZT7ftPH\\\\\\\/RFuDxKb0OY9lJvz11dGzGPHd6OvJroIVrPlx9+N2aYKMPN1cUh0P84lhdPVARMPnCmhvxxBv5Cwc8U3s8v28wPTnSvadVE1pFpCTyF7otvuo7ipQcCX\\\\\\\/8ibJUx1JMHMju8SFHZkqsfFPqZj80pUbOh5KaOcBrGTnWsc0CpyEcVgJJA38RKyea8wjVZ3qYZxekjsr9vGjEnbvpQnOO6MN5LQxhixC0GwXw6l7\\\\\\\/EtUsuwhW6\\\\\\\/MUfT4QWmZuO9Mj6NPj0t88qq3njxdyN6+Xo8j0+i+YivBiqRYNBFkYmdO5L+6i8s3wO4BAI5wdH6m5z8mjSsMXvGBJaqOIMKBc5XblTYvA8XcARzI96OpFg0NbOKRM9jQEFzODemmWIauqFzT6y6ynhYEvy5oftxU8jJ9H0dSGUM0Plk6VI1PEVzhqUFBt0d4pgi\\\\\\\/XG4owuyODckr\\\\\\\/nbIbv4zclRa+Fwp9xArrmnyytUdMynTOab7F5qompA4j8qxf4tPBzJW3q7RTIFAs1HDjNj4xq8MIvppzt4CD3EZJhadKuw6f67+9xzZ99oaw6vh+qFPA\\\\\\\/vwX2HJzdEaJLkPnUdr5QLLvjYM+a1V8grAzoEUiHNgC9UinReNmUs5aK32ebs5NwbhAzS+c21zVPNalmSenKC8o0q1bs1AGtXBg6XqiJYlmRNKtlXn5xf1p\\\\\\\/Xepti4OpFAvyLXGdbsT\\\\\\\/Gq+w4TFh8icrRhmETPWI3\\\\\\\/oZWiN7pSiCT8uNElj0FAcVfGgewt65lIB12NUg6774vWtPlBnu1oAcDBnRrvV8dbs68PixdXb898qV4c48egU5dO2NYApddIvRalV2f\\\\\\\/wc2NMw3KZlZeiutdUFKZk1rUhEf06KWZ+22ZhpLuZaFJSNvMKTl\\\\\\\/XAwIdiKFj+011HXanva0RBToNndg5NjXwZJmOUghe7bxxaB16aYWX5OVrqpAwfi\\\\\\\/IoVzhPN+B3NdkS0J58\\\\\\\/At9d0l6StXiyaXWIIxh2mNFGuQpFgT922KYT4dO+5z9cRAxWuuQbD+24qu\\\\\\\/otPJzBA7qzBwXzVMFdLBXrehCYvZkiIe27eyXYrjwVbOxiTYw27GD8unopd4kkKllpeN73Mt7LdTfwva4mlVbiO3E8rqbzwUcuTu3LDzl62xIy6ewXv+fxd6+KOcSKhAYykxT0IWuF0fz2PGzN9mFCt49aC6MBh65CxNCbh4gm9M1WbsNY2zM1EYhgyZbQGRwc7lSt3qEfRF0L2FRd9iJZMpI9vEe3fQeJApifCW2PfQubiVdfdu+dfA3Dfp5Pg01\\\\\\\/PqQJJEHWt7Sm1uj6j\\\\\\\/ripFXT7C6oKE3RfDzbzu5aSdkWTYHE0e63tgdM\\\\\\\/qdOWz9SDx+Pql1+nlazk61HBN4HrM4RISQPeNaM0UA6CxfEd51yoYcf2dMY0VFS3gQ9Hjk\\\\\\\/4UPvqNqXZSyaUyN7vEXiR0343ZeSup\\\\\\\/Mqkz6qwjPr8K6iDFXH99Am9vwxwxOJlc247PZhMpdMR20WbFuMNbF1P+0kksGTE0idclCslWFbIhUsZU6HRinDodYlm7HAGbRF61Plb8UNUI7ZAXElOoVDc5PMjYSr4cYED1Kdkv1QyBj7ba\\\\\\\/T58\\\\\\\/SIOlK10QDhrvTsoaX2TTqyDVrg6Tyx+tXAePPufPNGXhk14fd0M4icJXtEzRpdvoP+raps3\\\\\\\/jl3q8oSSjqsWk483nAX0c5F3nqbaIVafW+CyQfC3FLJ6xJ0DGnRkf3Aw1hYwUdfuvJhfHky0vj6kCN7bCkK9nYWGbm\\\\\\\/IE3ieJgwfT5kUq7PEVrmNze6YBdIYBOv\\\\\\\/So7\\\\\\\/BxWYkgkhhQeBCDsGYTwrGjghE3qfg3CoZDVcvioaZvJjVKeJEEkeFoeIv8\\\\\\\/CtQoinxvzYSjIuulc3wx3qCt6945\\\\\\\/xtnAEV2XtXKuv\\\\\\\/07djR\\\\\\\/1kMhyn7EDQsxHfZkvbhQ44tPXymTETzZP7hXjX5tR3+TtjI7RRGhGa3zkXSsmBUgWnOhrTs9YlXRRxmVdaLFuALJCKrVJqSYs5FkHW2c0ayZIVw+YUySF+PQnpmV7+hqS7suD4og55Jm1VFRCsLRknxdnGAKid64LYtNQ9gniiHkYJnauExfFTYerct3PgEDoskyifLR6evNWGnqBIUUcut4Ad+ef3fMVfPyJkjyI4T4SDX+fu+D465FmcJtMqZb59oeTD3Q+HFnvQPo5BDX19QXUa9Q5hzkDplOwMWsMfaQcXwUTrmo9OOrLBDXnalm+EiHt20IKPiBdw32frBs+Lv9b9l49+vpaoybrG\\\\\\\/1oEWUvKjKCLPqWAUvuZk1+Fo53333k8BS0avY7nNHi34906Qs4KzWyJux73HrArjioL\\\\\\\/WBljp4wZqVnA8QnPC8n1vLk9Llfa7iLqFtRlfrnBsNSJsWuMF7O2ksKbGq\\\\\\\/RVTze867dovx3I5LrH6iXQxRglx6HFKgc0aSUUoXdH0Hbq3p8Gf\\\\\\\/K\\\\\\\/Aq4Mx0k3r9PMKe5rOSOcjTtw358uFNCKl9nTjX6rLRBAoqtwZeVIu1+6+maMq20aPK42oQSKnJcrQUowvWLLjxeW3pb75l9yjtSj2IxlEfCHvlu8TNRECt7RBiBhBd+clV013VzmIixlpriyelPk3mQcIgrzT39Bh6iKbebjDV7kliu\\\\\\\/iBAIiC9NILlt5Z7Rb63iah166YY7UqvVftoyvOvjc5c2s0zzTz2+DLd1zBbpNlIx3O5nr3hG9iwXjsT8daPAEeHKFk+s+rw1X1K4LJczwvk+6HokmUOnE5DculfhkzyOrNMSodRZ586aY9+gsOB0o5hibT+PePGYnV\\\\\\\/yPML0kXrZVgaWT9bgdq\\\\\\\/Te3xZLC16M5og3iQN7TZL+KG3Vc2oToXGTbeN+Pd6wQ9XbJtE2znnc38swd4kPYrILDr8ySzh55U0DHbJqjF3Z+TZ5HHbuyayoLjn6TJtHwz2jVvFIgqO5klajXpMpozmpzNwzCj6MusOUkn0Q1RBBp45dBLNsU1NvKUpFCGPf3Frn3owfh7w1fcmDaaZp87xhV\\\\\\\/Y\\\\\\\/UTc33wrULik2LyHmtvYS0fnxh+L29ryfLJO842mFwIn+QyEvzjFwgBegVqVyrnbF+J+VISoHL24QoI5beo1S6+Rwlz2buioJqlnTPDmWqA6lERPDIQ78QrMQTY5uOK5u1jTan085Jxf7dd2CbKzhGIP34KwNE3XCaKsx2tjbSRGYpsOq9y6jtMCHn1+rxNrF6nwA3KNFR8I3yahlQ0y9Y9+Y3XDpAWGrh2Mf9LR9axTzCHuCQabjVRnyAdKrQhu5rRmtqCCAKXuhGyMTq2BOhLtI1sYNsdVUPtWTA3d88IzIDCiOb5D4f6F5qakC+hWBlwiaPdTvh8PctnrYZFnU2IzbGudwsE6hX\\\\\\\/EpiM0XFgvoEWxYIa84PU5I1Y8Vi0Wg7FcGwGE6DSSmY\\\\\\\/g\\\\\\\/CtEYbmIO6bqz\\\\\\\/tvINXDeXE74sdlRQSRMoSTAltjPPu8yZJ\\\\\\\/3368sjJVB371eS6hySugbTJ6V3KZ+\\\\\\\/6j\\\\\\\/OzuA8prLuScu2GYv55fkTt7IGKDS32o2p4DsfqPkRtOJxRHlkYB1yLFOX+cxOsU+UsglnMUrIPheQMPj0aED9okvKIFBoFqcsA2Diz59reh+LO4zexAc1bX3dWUvGSD8w8j\\\\\\\/1EufIk6R1rzUzVbO7sbNn01IxWCYTwIg7PDfnfPONVAT7yyYyxbr3jxHj08Gl6wR+xGE+uho0BaEsqDQfRrd+o+AquvhCzJGN8EAkb+IcN7xLMXTYeQ9TLK8XU\\\\\\\/h7hlGq+3CFcuMYUfrO3oVyriC267MDfro7mY5X6eaogoquzEeY0yU\\\\\\\/QKPd4W9EHt29EZZhpFQZ3e7NKmjFwbP6KV4PiX2uNy8et28jKB3OP0bENCeT2oJBAG1Fp8c9HBM9Zu3UmlICmoAgIpW\\\\\\\/+tzeJyM7W9bzsPSTp9TNvF6WHJNwaVyMgHHENYSaABkkWCJgMho2euL\\\\\\\/pNSYP56+eP\\\\\\\/uevTJh8p7vpscLGnu9DJFz88ArNBPmTw0lMquY24Nw3pyz2Y5y+R3F6H1xOBVz91Dkt3sCd7J\\\\\\\/abGbw5HRvpswwk4ue0oymSJFZQHWPNx1oL+LEOM+NH7fJ28ZHTPUPmeiRDn51W2hQyQ3rcNfourgFHREzV13RBoL2uZQnqpRiSGGLIjFFRx5k2jxAD8vc\\\\\\\/ZjRCm1NEAqDG4n+jQM\\\\\\\/a1rsc53r6kbkvKOonrYfDKxCvOKWMnb\\\\\\\/qftkhokPyloA1vZYiC9iMPildZc3tTAtWMlNpV\\\\\\\/mZGEmEgSyjGGur1Y\\\\\\\/jtYVsYKfo7tB0Q2o5A4yRI5bhICaPqHruqhs2bcCfm0bt88r8mTW+XtXCB\\\\\\\/rSHLVYZKKLyMHFX3nUMckNwmj7ObWXgD3AbRowcEA9L1xlamSoo6+cRtWQOiSLvoY6Ay5fIbQZI20itqYnFZV4eAzpY9EkOkuTxo20iu1ZD9nFlmeUCwKN0ISiCGSRMJW4mN5K7uSusoyQiUrmj47MvlL4uUxyfoxl2z0Vuwk6NVR\\\\\\\/8atNf9DPJ2MS4ghYxxIgEqQuPHDiYL3NNEoMaDOngr+S7xv3pRFrpG7IS2xHI7GmjgkFip9tkc0sh2A\\\\\\\/QtMWjCEaz3FHDMXtDYgrO4le4f4btDF\\\\\\\/kwfYVVqV3X2m5QKVctQwH5M+2k+n0JuqcO6Qc03hBC26hOdnb1XanPv\\\\\\\/U2UywpnrKXFHekSv0VzQXR7k+JjwMjmMbmSpK78I40AiGUQVy\\\\\\\/RVgCjTBOFu4VdTTUbTgyNk72hKxKfd+TPNbREn9MWUii4ouA253gQPhjAWyedetyYhj4CN5zSqVSiCjSUWzNCkv1IXrlq+RiIG3VlIBaFzLOl2bSY8u8o+7fmNfYa98JJZ8z3nDV56\\\\\\\/OeSyE9sJ7463DOHOULWT4ZwzAprRtU4jGu5G6dQNWiBzVNNrLFG65mqxf6IPolIObdBXnZS\\\\\\\/xuelbW4lSxaBVPT9dtKCQbJJ9P2OVBWHDLEVD4q0tXk7iEJN1Reqf2m1IUO7zRETSkRI227xmUHndXgiBiFQzLGKiFj98WIlEwQtEeAPPaX2vnwmXvoYAoUmErF629maPoH9sY6hS4UNPyzVgi3PJqshDEl+D4H1V4d+O5vtcKoIRcUVd4Ok7ckYOPzD7TTDlPOdAb3xp+nFoXmemZb0IbKxVrmKWppRpZgsz+6DLvMSs1bk1IUzKc4nB7nHMqZCbdKCvWm5Qsg5psXNh+XaPT+fVqYVunJ96eL6pOjLamUzQ8D3CXdZUmfzzmJehTZAE2bVdbYQKz++6UxvfICUPt1xj1o+529KXcadqI2qZr3FQm6bKrDbCzqxcROdbJJXfOyT2U3me46bLrBCNWrWtHoPSg7XZIcuGLY7f\\\\\\\/X5eqqZIIYOAj9ARcnukRL7WvyQ02J1xXRKwgzQIcRHXiFBKPSnH84MdUBY0huHS1CIgrkDCjbDfs5FTuKvUC1AT+X+ag+GtOs39NIlCZiAPZDvOUXi3gUjeI9bQcgO2UP94lqsG8WBGi2uOcmw2kfn0b0vJUkUxZWQP0jcuC41WJwBFrgTC4rjxLyZwKWIwhPffxInC\\\\\\\/ECoAItp4d+\\\\\\\/xmkGTKaz+CU0ZfncYW9R885chZHk4i1E+0Ol2ldXefjYD3I9i3Lf4aEQrTWB0r5bi+S8kTwg\\\\\\\/IbayEy1NkH+4v9+EnbNtzzhwG6cDARTWtTazEV0GiknCTU30RP+QDXLVAqmXl0XneMZJUZ9iY92YVEcADVHjvtiDo2oRRHRPKh5JGUwZMCzglWidZsSTVov4mLsEfO77SD8Jfuj8CpS9Jey4+kHQqpe5s4sDnD+M34T28rI7HHPE0950yllQZhZzTtIV1+uv9Avg2n7LuyhG7Y2uSzT+camJpDWNWBg0KyVLZtuQ4I+iR3TVaKLjnzCUWMe0Qq1QQOM2wd51LE5qhdeHOY5lp05uNRKxZRFjAstq2qfHQip5KuybsUZ\\\\\\\/\\\\\\\/wuKySxkxYWl1xagEp94HF6Y6Lalal+kIYtjR1nmM21j1RMk+\\\\\\\/4vmx+wiNruzWlk8YbCj5cdG0t7ZP1voPFU5IgP2pjSs04dStZ9joGF7+zetqUP9pYQXyzwXuxF1pihYHwjGausfwMKS7HN\\\\\\\/7z47pkhL8PHs2mXbJ+\\\\\\\/g1+9Kpzn9LWvsRXaHkXRLc7KxB1pPpwXrGbH6b6Eshm6\\\\\\\/pD7aHVy7713foF45uKTFD6m+d2JzDuzG3metD5SVWsHHxggrMOXxjpx\\\\\\\/4e78ggqulyLBGZcWSU86kaPZ\\\\\\\/hU5O4hbvzVcDv8E5wtyBp6AlVM\\\\\\\/oEmNnLaU7FLzqNR9ha0vMLUxOpeYftt77xfIPobe2+rZfzHHFo3OBv0yFC\\\\\\\/qDZ2O1drlu13cnHs8SBEqk3mB0ZHT5ycjRKWVHSsGWt7DygzXZ0D5qzxlqeONydYO8HaPULOIUb29Z\\\\\\\/zsA3Agj2hKs\\\\\\\/kveDEn9\\\\\\\/tJ29dx0ChjZtmDwedrxF6S32eAt1RS9vj0TsIZGK67QRLygdDXBlBcG0N8+doWii41F\\\\\\\/0OG8nNEXXnSJ1tFPtE\\\\\\\/evUEjz8jhoxWTio6+3eyUGZylzoORB3nY0BV6nMsWo471UPMr9QRAPPRnFV73mCc04pWzGlpYeHPTU+0kzu2MBkNSWXmkMBsFPW55lWhbukTQC6g3Y01JksQbeQaZHIJyqEK6N9pmrWwvDD9N31ju3YSEtMW0l\\\\\\\/7cID\\\\\\\/iGIzjtD22vhMxaMMpngBZYfz7DG9\\\\\\\/HPTjTkSYUrmOkGz12ukyvHyyg5Pi4q9fWPyB+eQMXtn2kPapmMekxRbIl8Wu0v1VY7Xh0CyYX\\\\\\\/9k9S61Qp\\\\\\\/scB+IyOrNG9Vd89kwG0swarvPzxaASwBozGFJlBkcAmxvcMBL+tw14PD3PCYyX+EBXAC4loCt2wfQXusiFk3XoKlAJPJ\\\\\\\/4yMU7\\\\\\\/ocZr6SkB50LZvfOzSEuZXAjOuBYHRohDuKgVWv8NtsJDskUWMv8fn27SVY3cEtPaeD0IovkcWZF134LHITBydHu7XA\\\\\\\/5+LzEH8kMAfbqPSxQqNW+aeBj6OEAVZ3aOQlwtAxY0pith8Y4sBX8dG\\\\\\\/k1TiPMeMe0ItzTh\\\\\\\/eCL3LBGVAb\\\\\\\/NVIXz0jU5UMs6PYuq389rhG\\\\\\\/7VTB1zZmt+DPfcrtWdpK3HY7i9ENOrzCTA7eoXmtS+wo7C5+535IB+Iu1JUDe0Qo2r1IfHVWiDRIGnj+WezIcwpXmIpA7b4poTmcd7Po8FuMCQCpGwM5TfKBIH9Ku\\\\\\\/ns6+2HPt0o3ZMdwSGZQPJEbQch2qMg6eA3HUqwPy9vW5PqMNlyjVwAVs7jLWJj2mhirJfOG67LBwY\\\\\\\/0k7GtjXhYObap\\\\\\\/Me3UBnfgCZGOW8eK0WWY3Vc+\\\\\\\/kp8k5DyB3Ntm8XRfQ6VOsKZjRjxILhErWvHn8q6hTtBwkbG1gRxD053+UkEkiHDdcAKJ9dy\\\\\\\/DcG78eHV+f\\\\\\\/NAxThoUOBA59vVqZw27nj0Q+WEYG6wMoKxnUOzD4PYODieY6RzBhGURPgL5bN7rXDSwHbDsU+0ItiJzfcnEbx5knMv9QDngKYLtV8ROLVft8Q0MuT2u5aT3PNshSUwojcFHEU5zHb0pJxFTVq6PD98\\\\\\\/ONqK+jxNctLHCrKpbPwpC9Mnzr9Sj34HNqnksfIZPZfoyX26bU3\\\\\\\/GIvwfPNuVw819jz\\\\\\\/fSs+B4mcf08fr6TyB\\\\\\\/lo7JO2SZV8j9+WIl7lDvdq5o4iaypNXFSbyJnWK02kF6uG9xHN0aC3RuznuIxbF\\\\\\\/efLbiyPpeFfi\\\\\\\/Wd7r+fTGDeyl4Z9mveiTKXCY5AQEaoW8siGcTUZn0uE836Tppoh8vYJZK\\\\\\\/u12oAKeaf4lpTKJ993vff7fBsVC2PEMslP4j8+W0OjzqgQZTv08VZ9Lz\\\\\\\/wWHKhHUv\\\\\\\/g58JDNadH0wG3ghDvazIqX1QwAFN1xXe1RV8\\\\\\\/8sYP5LMKB6ivVEWt5k3u6wI+o86KJKLk\\\\\\\/1nIadLySJbt\\\\\\\/8JXBm3TSE5s0M5MRFL7LRTRVyybAk5YRHClnl7F3vNRC1VRNu1SrckTTr048prtVXil32U7oNAxO3MbfCRs64NV1GrsgZpzZbfuAeXuw6WSH\\\\\\\/TcuvUc7s20b\\\\\\\/7R5re7Jk4owQaeh0vZU7YFS3HVcsEnVdcawO\\\\\\\/Xbtvpk+vCeS6qTaPiisqUxyzRblADGTDmawai0FL1RP\\\\\\\/JDgvJV4g8B\\\\\\\/iblTbnAQyVkY5GyiIt03suJNPjQi7057BTZscPI4+m38bnMnbNSkC5puUTQYJg7F2xAgMUeBsLNuVKvyiOsJ+iJCvX5xEIO4wjd2PaOPhNKtONuhU4GT2b80R+TdXOtZnH3ZP1oQEip7sL1HKoCvUhaulx6E0S2lmwKRx2AjAeVt\\\u0022,\\\u0022iv\\\u0022:\\\u0022XOdlTpgu33t2v8MxDSo7Ig==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a91818506788b7a3200fd37375b3d1dcb5bb9b26c097ae6c5299da3ed55874e8\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wiNrcnJGKasFa5L1p8NeisZSdyzb9fJyLWESKKNM7EjU09JdyDT\\\\\\\/Um3GXj+koeMsAcb8OmDG2W6ugh7xIaoidQOSBXRbgmRm+9A5VfNXujDF4VjlWYUw\\\\\\\/cUmv2cRQpUqvcbE4PJKfuhb7G+jvK7k0sqlcIYzcLscL3kwU5YgtF8VQXYm45mR6\\\\\\\/HbEFiIncDHhOC0Pq6ZFULAcpu7GerqDD7DEujMF97nbuh2XJwD5sJh4CE5eIGCsvfU+av2oZ1xd9zV1ETqHi0yyJucON7abwIy6xiWIcFL7pAI404\\\\\\\/SYP7u\\\\\\\/fJib\\\\\\\/reDYLX1GsRpLHsktMzlaWsREgH+SRXU\\\\\\\/vutHCIbj9jXwe\\\\\\\/fZ8ybSu8rps3sDCPn\\\\\\\/5bjW4JNHG\\\\\\\/Q37znTfdDD7G53uoHi5B\\\\\\\/fcOgvWE7PhW0DPZUAxDfzw6odmjIr0NUZUg92mrdywEY3fPvTvzKNeTz6LdwMHuNata1cSoml4l8lkgTY1sANao7+46tQ8xJiDHcV2YietdyeXvOhUC5g1wOHrPUtrEd5TEpWwXrgS726rFRkM6OsPKUI0FGpzL+R6RbsH6ai5sGnnoRqTKOmNsyMCzOoz8QPQlZ3Z8qk919lF178Rc+biBO2NVlm0zH6S\\\\\\\/PK56ribFJq8cylYbXxDBWhcGOlbFceurPCm6zfz\\\\\\\/k5EcIUe5ZdLiOYqqQ6lGVMzyCxnGpQLVaqBG7aVusI9Q9cqXtvLsKBheD+lHNKnnZ+jXZ\\\\\\\/3OIOBQ+ytOnxX014VTX6+ZBPntPAPgUufgChr+t487KdB0T24xKa3ryuXbMmHWeCY6fgovGvldlo7TLgf5NainFyIHaacff+3X8J4gwhJdQn4SI6qt9kyG3ru4XJhJ6zpXvBrxeyLdQS44NKPtsa2vmJtoOe5BGdynYXT97sVKyvcpVLzV0q0gkBCS+Uq2H8ciYGWQ0bLM5WW7wdFACSYx9+SroBDVdgthj2CYGSQZkkCtzzCPiLznCTcK1yz45O4HIDiJGaTdV4IqdztkXEMj2o9\\\\\\\/m41Hnk9rWcssuScGQmW4kn4+hOwfhfqp7MnVZENSZBcQpTzSxNhobmhI4pEcN+JCa+ro+k8QCFdaCG1dnXh58BIk4GjnBj5RjYJq75mYkfDwKEb4xwvwPUuTmNGCoCWf1dYjRCbnxlLxZl5oY4tdDN4R3ih9lqScVRsX6fTwvHOdS92R5k4oYq70DGVIZM6orBMq45xawZeACNrqdqir1MvIgW67vFRpqC1IC6DlPVayDPeG0pA9YnSLwnnjT7b2bDLN4tiGUFATPPNGhSqHn47PILH8e7dT24zuEh0WttG8yfRTHkb3E1tUkRszBzucQ\\\\\\\/EkUK+cbxDQ5t43v4GkfxxjCxolXErDjsWc+RQOOOEvwA34HZxGjqoslDzNOPVZha4H0eeTQJPAK4hX9dwKB+goRJQNhGplNDe6pTuuLMzn7\\\\\\\/bpxSfNDWA9j4SQ4IzMLt\\\\\\\/5VGwGXyZp9xIqTuZ75VTn\\\\\\\/t3n7b0IHaXurYW+TGh+GraOZRdhIyhMoyk+x9lz+gTfa0yLNXKn2YneNrVRP8gR49H26Q6pXJ1AGp8cODDesJ0RJEfgc7229NPhZqG5zkGXDvxo+fu9MBzv227YF9dHwTmI6sOGqJN91ILOW0fHUg+0F181tCh8ZBW10insFtu6oLkTVI4Yk0alK1E+YiST9ShDBpBCfLR++qDIk3tboIsGQjyKYtkm0jkzNdmwcTkKYoP4eUS9fcxPbhqca\\\\\\\/xF6avqRcW+HCqpdypReffxXTOd9Pojv7CjeKH4MS8\\\\\\\/Vcpp46NPt+zGOu0rRA12rLqSBz33v+J\\\\\\\/h22jdGU5c\\\\\\\/ZtyO1W+fTzajsbevpYfWIE1Jmzy3RwWvY9sGG56cw1ccfsLpy7QVQ4qSS6moVnB16Cun45klZ33B5A8P83tbCl2OtnpVUwmFH08R6GUmGgVgZwO0q8NM49xojrcK86jFS\\\\\\\/LF+L92AKxKFmkbN8S+O5xpkK3h4Z1ouNnm7KWpuP2JQIElBfymrr6hmb0LpvUpSrxM=\\\u0022,\\\u0022iv\\\u0022:\\\u0022Akz0IUmU6S3quGBA9QVgTQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022948e79a550757f3f4e67a9789c7245eeff6e084492fe62765a873844f2a15a09\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487822 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/315" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fOiyRSIufGWOoCLZEmGTzbmpXl0hKnEqtU11--30r5Q" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/315" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-3BihnVAocd8xcW7BKTqPNUUwEHRiD0aVwAH9MzyJJWk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/303" title="A Deconvolution Protocol for ChIP-Seq Reveals Analogous Enhancer Structures on the Mouse and Human Ribosomal RNA Genes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/331" title="Mating Design and Genetic Structure of a Multi-Parent Advanced Generation Intercross (MAGIC) Population of Sorghum (Sorghum bicolor (L.) Moench)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487822" id="node487822" data-pisa="ggg;8/1/315" data-pisa-master="ggg;g3.117.300415" data-apath="/ggg/8/1/315.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Mapping of Decay Factor–mRNA Interactions in Yeast Identifies Nutrient-Responsive Transcripts as Targets of the Deadenylase Ccr4</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jason E. Miller</span>, <span class="highwire-citation-author" data-delta="1">Liye Zhang</span>, <span class="highwire-citation-author" data-delta="2">Haoyang Jiang</span>, <span class="highwire-citation-author" data-delta="3">Yunfei Li</span>, <span class="highwire-citation-author" data-delta="4">B. Franklin Pugh</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-3578-4266" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joseph C. Reese</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">315-330;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300415 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487822" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jason E. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for RNA Molecular Biology, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BE.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BE.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Liye Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Comparative Genomics and Bioinformatics, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liye%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALiye%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Haoyang Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for RNA Molecular Biology, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haoyang%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaoyang%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yunfei Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Comparative Genomics and Bioinformatics, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yunfei%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYunfei%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">B. Franklin Pugh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Comparative Genomics and Bioinformatics, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BFranklin%2BPugh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pugh%20BF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AB.%2BFranklin%2BPugh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Joseph C. Reese </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for RNA Molecular Biology, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BC.%2BReese%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reese%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BC.%2BReese%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3578-4266" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joseph C. Reese</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Jcr8@psu.edu" class="" data-icon-position="" data-hide-link-title="0">Jcr8@psu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/1/315" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487822" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487822/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/315.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487822" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487822/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/315.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487822" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487822/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/315.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487822" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487822/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/1/315.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487822" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487822/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">The <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> (carbon catabolite repression 4)-Not complex is a major regulator of stress responses that controls gene expression at multiple levels, from transcription to mRNA decay. <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, a “core” subunit of the complex, is the main cytoplasmic deadenylase in <em>Saccharomyces cerevisiae</em>; however, its mRNA targets have not been mapped on a genome-wide scale. Here, we describe a genome-wide approach, RNA immunoprecipitation (RIP) high-throughput sequencing (RIP-seq), to identify the RNAs bound to <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, and two proteins that associate with it, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> and <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a>. All three proteins were preferentially bound to lowly abundant mRNAs, most often at the 3′ end of the transcript. Furthermore, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, and <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> are recruited to mRNAs that are targeted by other RNA-binding proteins that promote decay and mRNA transport, and inhibit translation. Although <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not regulates mRNA transcription and decay, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> recruitment to mRNAs correlates better with decay rates, suggesting it imparts greater control over transcript abundance through decay. <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-enriched mRNAs are refractory to control by the other deadenylase complex in yeast, <a href="http://www.yeastgenome.org/locus/S000003062/overview">Pan2</a>/<a href="https://www.yeastgenome.org/locus/S000001508">3</a>, suggesting a division of labor between these deadenylation complexes. Finally, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> associate with mRNAs whose abundance increases during nutrient starvation, and those that fluctuate during metabolic and oxygen consumption cycles, which explains the known genetic connections between these factors and nutrient utilization and stress pathways.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/ccr4-not" class="hw-term hw-article-keyword hw-article-keyword-ccr4-not" rel="nofollow">Ccr4-Not</a></li><li class="kwd"><a href="/keyword/mrna-decay" class="hw-term hw-article-keyword hw-article-keyword-mrna-decay" rel="nofollow">mRNA decay</a></li><li class="kwd"><a href="/keyword/dhh1-rna" class="hw-term hw-article-keyword hw-article-keyword-dhh1-rna" rel="nofollow">Dhh1 RNA</a></li><li class="kwd"><a href="/keyword/immunoprecipitation-seq" class="hw-term hw-article-keyword hw-article-keyword-immunoprecipitation-seq" rel="nofollow">immunoprecipitation-seq</a></li></ul><p id="p-6">An important aspect of gene regulation is the proper control of mRNA levels, which is the product of both synthesis and decay. Although the earliest studies placed the greatest attention on how mRNA levels are controlled through transcription, interest in the role of mRNA decay is increasing. In yeast, the canonical mode of degradation starts with the shortening of the poly(A) tail by <a href="http://www.yeastgenome.org/locus/S000003062/overview">Pan2</a>/<a href="https://www.yeastgenome.org/locus/S000001508">3p</a> and the main cytoplasmic deadenylase, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4p</a>, which is part of the <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not complex (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Brown <em>et al.</em> 1996</a>; <a id="xref-ref-97-1" class="xref-bibr" href="#ref-97">Tucker <em>et al.</em> 2001</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chen <em>et al.</em> 2002</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Goldstrohm and Wickens 2008</a>; <a id="xref-ref-100-1" class="xref-bibr" href="#ref-100">Wahle and Winkler 2013</a>). Deadenylation is followed by the removal of the 5′ cap by <a href="http://www.yeastgenome.org/locus/S000005509/overview">Dcp1</a>/<a href="https://www.yeastgenome.org/locus/S000005062">2p</a>, which leads to exonucleolytic cleavage of the mRNA in the 5′ to 3′ direction by <a href="http://www.yeastgenome.org/locus/S000003141/overview">Xrn1p</a> (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Eulalio <em>et al.</em> 2007</a>). Alternatively, 3′ to 5′ decay is catalyzed by the exosome complex in a regulated manner (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Schmid and Jensen 2008</a>). Additional proteins bind specific regions on the mRNA, such as the 3′ UTR (untranslated region), to mediate decay and the localization of the mRNA (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Goldstrohm and Wickens 2008</a>). Remarkably, even among the most well-studied factors like <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4p</a>, it remains unclear how and what regulates their recruitment to mRNAs across the transcriptome.</p><p id="p-7"><a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not is composed of nine conserved core subunits that form a 0.9–1.2 MDa protein complex: <a href="http://www.yeastgenome.org/locus/S000000689/overview">Not1</a> through <a href="http://www.yeastgenome.org/locus/S000006276/overview">Not5</a>, <a href="http://www.yeastgenome.org/locus/S000005335/overview">Caf1</a>/<a href="http://www.yeastgenome.org/locus/S000005335/overview">Pop2</a>, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, <a href="http://www.yeastgenome.org/locus/S000005232/overview">Caf40</a>, and <a href="http://www.yeastgenome.org/locus/S000003366/overview">Caf130</a> (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Miller and Reese 2012</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Collart 2016</a>). The <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not complex interacts with a number of other proteins, such as the RNA-binding proteins (RBPs) <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> and the RNA helicase <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Hata <em>et al.</em> 1998</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Coller <em>et al.</em> 2001</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Maillet and Collart 2002</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Goldstrohm <em>et al.</em> 2006</a>; <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Tarassov <em>et al.</em> 2008</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alhusaini and Coller 2016</a>). <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> is a member of the Fem-binding family of proteins (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Quenault <em>et al.</em> 2011</a>) that binds to a motif in the 3′ UTR of mRNAs and enhances deadenylation by <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Goldstrohm <em>et al.</em> 2006</a>, <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">2007</a>). Recent crystallography studies verified the interaction between <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> and the N-terminus of <a href="http://www.yeastgenome.org/locus/S000000689/overview">Not1</a> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chen <em>et al.</em> 2014</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Mathys <em>et al.</em> 2014</a>). Although <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> binds to <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not, it is much more abundant, and additionally inhibits translation and promotes the decapping of mRNAs (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Ghaemmaghami <em>et al.</em> 2003</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Coller and Parker 2005</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Sweet <em>et al.</em> 2012</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Radhakrishnan <em>et al.</em> 2016</a>). Although these three proteins physically and genetically interact with each other, they play distinct functions in mRNA regulation (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Miller and Reese 2012</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Presnyak and Coller 2013</a>); thus, it is unclear to what extent their mRNA targets overlap.</p><p id="p-8">One of the features of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not that makes it an intriguing complex to study is that it regulates both transcription and mRNA degradation (<a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Miller and Reese 2012</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Collart 2016</a>). The deletion of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not subunits affects the abundance, decay, and synthesis of many mRNAs (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Grigull <em>et al.</em> 2004</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Cui <em>et al.</em> 2008</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Molina-Navarro <em>et al.</em> 2008</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Sun <em>et al.</em> 2012</a>, <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">2013</a>). However, the interpretation of gene expression changes in deletion mutants, while valuable, can be influenced by complex genetic interactions and secondary effects. In the aforementioned studies, it remains unknown which changes in mRNA expression are direct and which result from perturbed mRNA synthesis or decay. Therefore, identifying the direct mRNA targets of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not could shed light on the molecular underpinnings behind how RNA abundance, decay, and synthesis are controlled across the transcriptome.</p><p id="p-9">Here, a modified RIP-seq procedure was used to identify transcripts associated with <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, and <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a>. In addition to showing that these factors are recruited to many of the same mRNAs, the analysis of the mRNA targets suggests that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> imparts its greatest influence on gene regulation at the level of decay <em>vs.</em> synthesis. Our study has illuminated the interplay between the two cytoplasmic deadenylases in mRNA decay. Additionally, the recruitment of these three factors negatively correlated with ribosome density, suggesting new insights into the relationship between mRNA decay and translation. Finally, we show that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> bind mRNAs that fluctuate during the yeast metabolic cycle (YMC), suggesting a role for post-transcriptional regulation in reshaping the transcriptome in response to changing environmental conditions.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strain construction</h3><p id="p-10">All strains were constructed in the BY4741 background using published protocols (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Longtine <em>et al.</em> 1998</a>). A list of strains is contained in Supplemental Material, Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>.</p></div><div id="sec-3" class="subsection"><h3>RIP</h3><p id="p-11">RIP-seq was based upon a previous version of the procedure (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Dutta <em>et al.</em> 2011</a>), but incorporating changes for high-throughput sequencing of mRNAs. Cells were grown in 1 L of YPAD (2% peptone, 1% yeast extract, 0.02 mg/ml adenine sulfate, and 2% dextrose) at 30° to an OD<sub>600</sub> ∼0.9, and then formaldehyde was added to 1% (v/v) for 15 min. Cross-linking was quenched with glycine (final = 136 mM) for 5 min. Subsequent steps used buffers prepared with diethyl polycarbonate-treated (DEPC) water. Cells were harvested and washed in ice-cold STE (10 mM Tris-HCl pH 8, 1 mM EDTA, 50 mM NaCl, 0.5 mM PMSF, and 1 mM benzamidine-HCl) and frozen at −80°. Cells were resuspended in FA-lysis buffer (50 mM HEPES/KOH pH 7.5, 150 mM NaCl, 1% Triton X-100, and 0.1% sodium deoxycholate) containing protease inhibitors (2 μg/ml leupeptin, 3 μg/ml aprotinin, 2 μg/ml pepstatin A, 1 μg/ml chymostatin, 1 mM benzamidine-HCl, and 0.5 mM PMSF). Cells were disrupted by vortexing in the presence of glass beads for 45 min at 4°, and then 200 µl FA-lysis buffer was added to each tube and mixed for 30 sec, twice. Cell lysate was then transferred to two 15 ml polystyrene tubes (∼1.8 ml lysate in each), sonicated with two 30 sec pulses using a Bioruptor sonicator (Diagenode, Philadelphia, PA), and then clarified by centrifugation twice at 4° at 14,000 rpm. Protein content was measured by Bradford assay (Bio-Rad) using BSA as a standard. Samples with &gt; 7 mg/ml protein concentration were used in RIP.</p><p id="p-12">Whole-cell extract (2.5 ml) was diluted with an equal volume of FA-lysis buffer (containing protease inhibitors). MgCl<sub>2</sub> and CaCl<sub>2</sub> were added to concentrations of 25 and 5 mM, respectively, and then RNase-free DNase I (Worthington, Lakewood, NJ) was added to 174 units per ml. The samples were incubated for 90 min at 30°. EDTA was added to 50 mM and the samples were placed on ice. Samples were spun for 10 min at 14,000 rpm at 4°, and then 1 ml of supernatant was transferred to four tubes containing 15 µl of protein A–sepharose (GE Healthcare) slurry containing 3.5 µl 9E10 (anti-myc) monoclonal antibody from ascites fluid (Biolegend). In experiments where RT-qPCR of RNA was used for validation, Protein A beads bound with 7 µl of purified 9E10 antibody were used. The samples were incubated overnight at 4° with rotation. Beads were washed three times with FA-lysis buffer, two times with FA-wash buffer 2 (50 mM HEPES/KOH pH 7.5, 1 mM EDTA, 1% Triton X-100, 0.1% sodium deoxycholate, 0.5 M NaCl, 0.5 M PMSF, and 1 mM benzamidine-HCl), three times with FA-wash buffer 3 (0.25 M LiCl, 1% NP-40, 1% sodium deoxycholate, 1 mM EDTA, 10 mM Tris-HCl pH 8.0, 0.5 M PMSF, and 1 mM benzamidine-HCl), and two times with TE buffer (10 mM Tris-HCl and 1 mM EDTA, pH 8). RNA was eluted off the beads at 65° for 20 min in 450 µl elution buffer [25 mM Tris-HCl (pH 7.5), 1 mM EDTA, 0.2 M NaCl, and 0.5% SDS]. To reverse cross-links, proteinase K was added to 70 μg/ml and incubated at 65° for 5 hr. Input samples (100 µl of extract) were supplemented with 350 μl elution buffer, 10 μl 10% SDS, and then incubated at 65° for 5 hr. The RNA was purified using an acid–phenol (pH 4.8)/chloroform (1:1) extraction followed by ethanol precipitation in the presence of 20 μg glycogen. Samples were resuspended in DEPC water, and then treated with DNase once again. RNA was extracted with acid–phenol (pH 4.8)/chloroform/isoamyl-alcohol (25:24:1) and ethanol-precipitated in the presence of 10 μg glycogen. The pellet was resuspended in DEPC water and the nucleic acids quantified using a Nanodrop instrument.</p></div><div id="sec-4" class="subsection"><h3>Library construction, sequencing, and read mapping</h3><p id="p-13">Each sample was performed in biological triplicate except for the input, which was performed in duplicate. First, 0.5 μg of RNA from each replicate was incubated with RNase III (Ambion cat#2290) for 10 min at 37° to fragment the RNA and prepare the ends for linker addition. RNA was then concentrated using a concentration module (Invitrogen). The size and quality of the RNA was examined on a 2100 Agilent Bioanalyzer. cDNA libraries were then prepared using the SOLiD Total RNA-Seq Kit (PN 4445374). In brief, adapters were ligated onto the fragmented RNA and reverse transcribed using the SOLiD Total RNA kit. After two rounds of purification and size selection using the Agencourt AMPure XP Reagent, the library was PCR amplified and then purified using the Invitrogen PureLink PCR Micro Kit. Yield and size of amplified DNA was assessed using a 2100 Agilent Bioanalyzer. Each amplified DNA library was then clonally amplified in an emulsion PCR reaction. RIP and total RNA samples were sequenced on a SOLiD 4 platform.</p><p id="p-14">All samples were downloaded separately by barcode and reads were mapped to the 2007 <em>Saccharomyces cerevisiae</em> reference genome (sg7) using SHRiMP version 2.2.2 (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Rumble <em>et al.</em> 2009</a>). Reads were trimmed 15 bp from the 3′ end as a quality control measure. During mapping, SHRiMP2 calculated a score for each read based upon mismatches, and reads with &lt; 90% of the maximum possible score were filtered out (a 90% threshold is similar to allowing for three mismatches).</p></div><div id="sec-5" class="subsection"><h3>Composite plots</h3><p id="p-15">Python scripts were used to create composite plots. The purpose of the scripts can be conceptually described as follows. First, the reads were aligned to a TSS (Transcription Start Site) or a TTS (Transcription Termination Site) from a published source (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">David <em>et al.</em> 2006</a>), in a strand-specific manner. The read density was summarized around TSS or TTS mRNAs, then corrected for differences in total uniquely mappable reads so that the final output would be average read density per 100 million reads. Finally, the data were binned into 15 bp bins and smoothed using a 6 bp sliding window average.</p></div><div id="sec-6" class="subsection"><h3>Enrichment calculation</h3><p id="p-16">See Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a> for a diagram of the enrichment calculation. The reads per gene were tabulated after aligning reads to TSSs and TTSs identified from a published source (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">David <em>et al.</em> 2006</a>). Counts per gene were used to measure reproducibility. The abundant reads from ribosomal RNAs were filtered out prior to measuring the reproducibility between the samples. To calculate enrichment, two of three replicates were processed at a time in all three combinations (<em>i.e.</em>, rep 1 <em>vs.</em> rep 2, rep 2 <em>vs.</em> rep 3, and rep 1 <em>vs.</em> rep 3) then averaged if the false discovery rate (FDR) was &lt; 1%. Enrichment was calculated in edgeR for each gene by dividing reads in the immunoprecipitation (IP) sample by reads in the mock IP, while controlling for differences in library size among samples, (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Robinson <em>et al.</em> 2010</a>). One count was added to all genes using the prior.count argument in order to avoid infinite log<sub>2</sub> values resulting from 0 reads in the mock IP. Log<sub>2</sub> and p-values were calculated using the edgeR exacTest (assuming the Poisson model) and the Benjamini–Hochberg method was used to correct p-values for multiple testing (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Benjamini and Hochberg 1995</a>). To identify the regions of the mRNA bound by each factor, enrichment within the 5′ UTR, coding region, and 3′ UTR, reads were counted in a strand-specific manner using BEDOPS (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Neph <em>et al.</em> 2012</a>) and published gene annotations (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Nagalakshmi <em>et al.</em> 2008</a>). Following this step, enrichment was calculated in the same manner as for the whole RNA, as described in Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>. Enrichment was also calculated across the length of mRNAs based on gene length as follows. Each replicate was first split into reads that aligned to the first-third, middle-third, and last-third of each gene before calculating enrichment. Following this step, enrichment was calculated in the same manner as for the whole RNA. For intronic analysis, HOMER was used to count reads per gene or per intron (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Heinz <em>et al.</em> 2010</a>). Enrichment was calculated separately for intronic regions using the pipeline described in Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>, with the exception of using the saccer2 reference genome. A twofold enrichment cutoff was used to select for enriched intronic regions.</p></div><div id="sec-7" class="subsection"><h3>Gene ontology (GO) terms analysis</h3><p id="p-17">For GO terms analysis, the percent ranks for enrichment values were first calculated using the =PERCENTRANK() function in Microsoft Excel. The genes among the top 20th percentile of enrichment values along with a list of all genes (“background population”) were submitted to an online GO term finder (go.princeton.edu; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Boyle <em>et al.</em> 2004</a>), after which p-values were corrected using the Bonferroni method. The following terms were not included in the figures representing the GO terms output due to their redundant nature: “regulation of cellular process,” “regulation of biological process,” “biological regulation,” and “biological_process.” The 232 mRNAs connected to “organelle organization” in the <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> GO Term Finder output were submitted to GO Slim Mapper using default settings (<a href="http://www.yeastgenome.org/cgi-bin/GO/goSlimMapper.pl">http://www.yeastgenome.org/cgi-bin/GO/goSlimMapper.pl</a>).</p></div><div id="sec-8" class="subsection"><h3>Motif analysis</h3><p id="p-18">Sequence motifs were identified using FIRE (finding informative regulatory elements), which was performed either locally or on the iGET website (<a href="https://iget.c2b2.columbia.edu">https://iget.c2b2.columbia.edu</a>) using default settings to analyze “discrete” or “continuous” data on <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a>, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, and <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> unstressed enrichment values using 6, 7, and 8 bins, respectively (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Elemento <em>et al.</em> 2007</a>).</p></div><div id="sec-9" class="subsection"><h3>Statistical and visual analysis</h3><p id="p-19">Correlations were calculated in R using the cor, cor.test, or corr.test (from the “psych” package) functions. Heatscatter graphs were made using the R package “LSD.” Circles and ellipses for Venn Diagrams were made using the program eulerAPE (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Micallef and Rodgers 2014</a>) or using the package “VennDiagram” in R. Other graphs were generated using base R graphics or in Microsoft Excel. Screen shots were generated in IGV after adjusting tag count for differences in library size (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Robinson <em>et al.</em> 2011</a>; <a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Thorvaldsdottir <em>et al.</em> 2013</a>). Average reads per kilobase of transcript per million reads mapped (RPKM) was calculated by first adding 0.5 to each gene so as to not eventually take the log of zero, then calculating RPKM for each input separately, and finally taking the average of the two inputs.</p><div id="sec-10" class="subsection"><h4>External data sources:</h4><p id="p-20">The Comparative Dynamic Transcriptome Analysis (cDTA) data (<a id="xref-ref-89-2" class="xref-bibr" href="#ref-89">Sun <em>et al.</em> 2012</a>, <a id="xref-ref-90-2" class="xref-bibr" href="#ref-90">2013</a>) can be downloaded from the Cramer laboratory website <a href="http://www.cramer.genzentrum.lmu.de/movies/">http://www.cramer.genzentrum.lmu.de/movies/</a>, or can be found in the supplemental information associated with the respective publications. The RIP-Chip (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Gerber <em>et al.</em> 2004</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Hogan <em>et al.</em> 2008</a>), Ribo-seq (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Gerashchenko <em>et al.</em> 2012</a>), <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> CLIP-seq (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Mitchell <em>et al.</em> 2013</a>), <a href="http://www.yeastgenome.org/locus/S000000689/overview">Not1</a> RIP-seq (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Gupta <em>et al.</em> 2016</a>), gene run-on (GRO-Chip) (<a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Molina-Navarro <em>et al.</em> 2008</a>), and genome-wide ChIP (<a id="xref-ref-99-1" class="xref-bibr" href="#ref-99">Venters <em>et al.</em> 2011</a>) data sets were downloaded from their respective online supplemental files associated with the publication. Gene expression array data (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bradley <em>et al.</em> 2009</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arribere <em>et al.</em> 2011</a>) was also found in the online supplemental files associated with their respective publication. The <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Gasch <em>et al.</em> (2000</a>) microarray data were located online at <a href="http://downloads.yeastgenome.org/published_datasets/Expression_connection_data/">http://downloads.yeastgenome.org/published_datasets/Expression_connection_data/</a>. YMC gene expression values can be found at <a href="http://moment.utmb.edu/cgi-bin/dload.cgi">http://moment.utmb.edu/cgi-bin/dload.cgi</a>, while processed data were obtained via E-mail (see <em>Acknowledgments</em>).</p></div><div id="sec-11" class="subsection"><h4>RT-qPCR verification:</h4><p id="p-21">RNA was isolated from biological triplicate (cells were grown overnight in separate flasks before harvesting) samples. Equivalent amounts of RNA from input and IP samples (∼350 ng) were used to produce cDNA using random hexadeoxynucleotide primers ([final] = 0.01 μg/μl, Promega #C1181) and AMV reverse transcriptase (0.4 unit/µl) for 1 hr at 42° in a 15 μl volume. Real-time PCR was performed using Quanta PerfeCTa SYBR Green SuperMix in a 96-well plate (in technical triplicate) in an Applied Biosystems StepOnePlus machine. For each input and IP, the average Ct value was calculated from the technical triplicates. Then the %IP was calculated using the equation 2^(Ct<sub>Input(avg)</sub>−Ct<sub>IP(avg)</sub>). The average and SD of three biological replicates was reported using the AVERAGE and STDEV functions in Microsoft Excel. See <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a> for a list of primers.</p></div></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-22">The data from this study will be deposited with the Gene Expression Omnibus, accession no. GSE72366.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Factors involved in different aspects of mRNA decay specifically target the 3′ UTR of the same collection of mRNAs</h3><p id="p-23">RNA CLIP procedures are often used to map RNA–protein interactions, but the harvesting, processing, and UV cross-linking steps can introduce stress and change the RNA–protein interaction landscape (<a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Mitchell <em>et al.</em> 2013</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Riley and Steitz 2013</a>). We used formaldehyde (FA) cross-linking of cells in culture, which rapidly traps protein–nucleic acid interactions in their physiological state with minimal-to-no stress response. All three proteins under study here contact RNA directly (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Chen <em>et al.</em> 2002</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cheng <em>et al.</em> 2005</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Goldstrohm <em>et al.</em> 2006</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Dutta <em>et al.</em> 2011</a>), but FA can potentially cross-link a protein to RNA indirectly via another protein that is in direct contact with the RNA. Percentages of cross-linking efficiency are protein-dependent and hard to estimate <em>in vivo</em>, but it is widely accepted that a protein is cross-linked to a nucleic acid only a fraction of the time; therefore, “double-hit” cross-linking is expected to occur at a low frequency. Furthermore, even if the recruitment of a protein to an RNA is through a bridging protein, this in itself is desirable and informative because it indicates that it could regulate the fate of that RNA. Finally, RNA isolation and library construction used in this procedure do not target the poly(A) tail; therefore, potential technical artifacts caused by variations in the poly(A) tail length of messages under the control of the decay machinery are reduced.</p><p id="p-24">We identified RNAs associated with <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, and <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> as representatives of the mRNA decay machinery because they play different roles in the regulation of mRNAs. <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> is the major mRNA deadenylase; thus, its binding can estimate the decay potential of transcripts in the same way that the binding of RNA polymerase II to a gene is used to determine its transcription level. <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> was chosen for study, as opposed to <a href="http://www.yeastgenome.org/locus/S000003062/overview">Pan2</a>/<a href="https://www.yeastgenome.org/locus/S000001508">3</a>, because <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> plays a more prominent role in regulating poly(A) tail length and degradation rates (<a id="xref-ref-97-2" class="xref-bibr" href="#ref-97">Tucker <em>et al.</em> 2001</a>; <a id="xref-ref-90-3" class="xref-bibr" href="#ref-90">Sun <em>et al.</em> 2013</a>). Until this study, the recruitment of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> to mRNAs has not been reported on a transcriptome-wide scale in <em>S. cerevisiae</em>.</p><p id="p-25">RNAs isolated from the immunoprecipitates of extracts from cells containing a myc epitope-tagged version of each protein (<em>e.g.</em>, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> IP), and from extracts of untagged cells (from here on referred to as UT IP), were identified by high-throughput sequencing (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). The assays were carried out in triplicate and the reproducibility among all samples was very high (<em>R</em><sup>2</sup> values &gt; 0.93) (Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS1.xlsx">File S1</a>). Enrichment was calculated using the workflow described in the <em>Materials and Methods</em> and as illustrated in Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>. The vast majority (&gt; 95%) of the transcripts enriched by each protein were mRNAs (Figure S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS6.pdf">File S6</a>). We compared the enrichment values of each decay factor to mRNA abundance (<a id="xref-ref-89-3" class="xref-bibr" href="#ref-89">Sun <em>et al.</em> 2012</a>) to address the question of whether these proteins are recruited to mRNAs (targeted) or if the enrichment is simply a reflection of their relative abundance in the cell (nonspecific). The mRNA enrichment to all three proteins displayed a negative correlation with abundance (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>), with the <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> data showing the strongest anticorrelation (−0.41 Pearson correlation). The negative correlation between RIP-seq enrichment and abundance was observed using the input RNA-seq samples from this study and other published measurements of mRNA abundance (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>). Thus, these observations are robust and applicable to multiple data sets. Enrichment of a transcript did not correlate with its length on a global scale either (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>). Thus, the data suggest that these proteins are recruited to specific mRNAs, rather than associating with them indiscriminately based on abundance or length.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/315/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of Ccr4, Dhh1, and Puf5 RNA targets. (A) An illustration of the RNA immunoprecipitation high-throughput sequencing (RIP-seq) procedure (see Materials and Methods for details). (B) Heat scatter plots of RIP-seq enrichment values correlated with mRNA abundance levels (mRNA/cell) on a log2 scale [measured in Sun et al. (2012)]. The R values within the graphs represent Pearson correlation values. The color refers to the relative concentration of data points in a single area, i.e., red refers to greater density and blue to lower density. The dotted black trend line represents the least squares regression." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1743458345" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identification of Ccr4, Dhh1, and Puf5 RNA targets. (A) An illustration of the RNA immunoprecipitation high-throughput sequencing (RIP-seq) procedure (see &lt;em&gt;Materials and Methods&lt;/em&gt; for details). (B) Heat scatter plots of RIP-seq enrichment values correlated with mRNA abundance levels (mRNA/cell) on a log&lt;sub&gt;2&lt;/sub&gt; scale [measured in Sun &lt;em&gt;et al.&lt;/em&gt; (2012)]. The &lt;em&gt;R&lt;/em&gt; values within the graphs represent Pearson correlation values. The color refers to the relative concentration of data points in a single area, &lt;em&gt;i.e.&lt;/em&gt;, red refers to greater density and blue to lower density. The dotted black trend line represents the least squares regression.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/315/F1.medium.gif" width="440" height="228"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/1/315/F1.medium.gif" width="440" height="228"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/315/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/315/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487869" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-26" class="first-child">Identification of Ccr4, Dhh1, and Puf5 RNA targets. (A) An illustration of the RNA immunoprecipitation high-throughput sequencing (RIP-seq) procedure (see <em>Materials and Methods</em> for details). (B) Heat scatter plots of RIP-seq enrichment values correlated with mRNA abundance levels (mRNA/cell) on a log<sub>2</sub> scale [measured in <a id="xref-ref-89-4" class="xref-bibr" href="#ref-89">Sun <em>et al.</em> (2012)</a>]. The <em>R</em> values within the graphs represent Pearson correlation values. The color refers to the relative concentration of data points in a single area, <em>i.e.</em>, red refers to greater density and blue to lower density. The dotted black trend line represents the least squares regression.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27"><a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, and <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> display physical and genetic interactions with each other; however, they play different roles in mRNA decay. For example, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> deadenylates the poly(A) tail located at the 3′ end, and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> regulates translation and decapping occurring at the 5′ end of mRNAs (<a id="xref-ref-97-3" class="xref-bibr" href="#ref-97">Tucker <em>et al.</em> 2001</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Fischer and Weis 2002</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Coller and Parker 2005</a>). Furthermore, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> is 15 and 34 times more abundant than either <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> or <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a>, respectively (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Ghaemmaghami <em>et al.</em> 2003</a>); thus, it was unclear how well the targets of these proteins would overlap and if they would be found at the same location on mRNAs. A pairwise comparison of the enrichment values of RNAs bound to each protein revealed a high level of correlation (<em>R</em> &gt; 0.6, <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> and Figure S3, A–C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>). Additionally, we validated several targets using an RT-PCR approach, which revealed a good-to-strong correlation between RIP-seq values and those obtained by RT-PCR (Figure S4, A and B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/315/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="RNA immunoprecipitation high-throughput sequencing (RIP-seq) reveals that Ccr4, Dhh1, and Puf5 target the same collection of mRNAs (A). Pearson correlations of pairwise RIP-seq sample enrichment values. (B) Venn diagram of targets of Puf5 (green), Dhh1 (red), and Ccr4 (blue) with enrichment values &gt; 1 log2 in two of the three biological replicates using a false discover rate &lt; 1%. (C) Composite plots of mRNA sequencing read density of all mRNAs with a length of &gt; 400 nucleotides (nt) (n = 4565). The reads were aligned relative to the TSS [transcription Start Site (left)] and TTS [Transcript Termination Site (Right)] in nt. The TSS and TTS were identified from another study (David et al. 2006). Tag count per gene was normalized to 100 million reads to account for differences in library sizes. (D) The enrichment of each factor was calculated over the 5&#x2032; untranslated region (UTR), coding sequence (CDS), and 3&#x2032; UTR of mRNAs. The number of mRNAs (&gt; 400 nt) with more than fourfold enrichment is displayed on the y-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1743458345" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;RNA immunoprecipitation high-throughput sequencing (RIP-seq) reveals that Ccr4, Dhh1, and Puf5 target the same collection of mRNAs (A). Pearson correlations of pairwise RIP-seq sample enrichment values. (B) Venn diagram of targets of Puf5 (green), Dhh1 (red), and Ccr4 (blue) with enrichment values &gt; 1 log&lt;sub&gt;2&lt;/sub&gt; in two of the three biological replicates using a false discover rate &lt; 1%. (C) Composite plots of mRNA sequencing read density of all mRNAs with a length of &gt; 400 nucleotides (nt) (&lt;em&gt;n&lt;/em&gt; = 4565). The reads were aligned relative to the TSS [transcription Start Site (left)] and TTS [Transcript Termination Site (Right)] in nt. The TSS and TTS were identified from another study (David &lt;em&gt;et al.&lt;/em&gt; 2006). Tag count per gene was normalized to 100 million reads to account for differences in library sizes. (D) The enrichment of each factor was calculated over the 5&#x2032; untranslated region (UTR), coding sequence (CDS), and 3&#x2032; UTR of mRNAs. The number of mRNAs (&gt; 400 nt) with more than fourfold enrichment is displayed on the &lt;em&gt;y&lt;/em&gt;-axis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/315/F2.medium.gif" width="440" height="260"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/1/315/F2.medium.gif" width="440" height="260"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/315/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/315/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487888" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">RNA immunoprecipitation high-throughput sequencing (RIP-seq) reveals that Ccr4, Dhh1, and Puf5 target the same collection of mRNAs (A). Pearson correlations of pairwise RIP-seq sample enrichment values. (B) Venn diagram of targets of Puf5 (green), Dhh1 (red), and Ccr4 (blue) with enrichment values &gt; 1 log<sub>2</sub> in two of the three biological replicates using a false discover rate &lt; 1%. (C) Composite plots of mRNA sequencing read density of all mRNAs with a length of &gt; 400 nucleotides (nt) (<em>n</em> = 4565). The reads were aligned relative to the TSS [transcription Start Site (left)] and TTS [Transcript Termination Site (Right)] in nt. The TSS and TTS were identified from another study (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">David <em>et al.</em> 2006</a>). Tag count per gene was normalized to 100 million reads to account for differences in library sizes. (D) The enrichment of each factor was calculated over the 5′ untranslated region (UTR), coding sequence (CDS), and 3′ UTR of mRNAs. The number of mRNAs (&gt; 400 nt) with more than fourfold enrichment is displayed on the <em>y</em>-axis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">Next, we examined the overlap of RNA targets of the three factors with each other. For this purpose, we identified RNAs that were enriched at least twofold (&gt; 1 log<sub>2</sub>) with an FDR &lt; 1% and present in at least two of the three replicates. Using these criteria, 3157, 2262, and 1241 RNAs were found to be associated with <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, and <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a>, respectively (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). There was a strong overlap among the targets of these three proteins. Interestingly, most mRNAs bound by <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> or <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> were also targets of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). These results suggest that the physical interactions between these proteins that have been identified in previous studies reflect their coregulation of mRNAs <em>in vivo</em>. Furthermore, the overlap of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> targets is consistent with studies that have shown that <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> recruits the <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not complex to the 3′ UTR of the <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> mRNA (<a id="xref-ref-38-4" class="xref-bibr" href="#ref-38">Goldstrohm <em>et al.</em> 2006</a>, <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">2007</a>). Moreover, motifs (H<span class="underline">UGUA</span>NH<span class="underline">A</span>D) with likeness (underlined portion is similar) to both <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> (<span class="underline">UGUA</span>AY<span class="underline">A</span>WUA) and the related RBP <a href="http://www.yeastgenome.org/locus/S000002982/overview">Puf4</a> (WH<span class="underline">UGUA</span>HA<span class="underline">W</span>UA) were overrepresented among the RNAs associated with all three proteins (p-value &lt; 10<sup>−4</sup>, Figure S3D in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>) (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Gerber <em>et al.</em> 2004</a>). The discovery of motifs similar to both Puf4- and Puf5-binding sites in the enriched mRNAs may be explained by the ability of <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> to adapt to variations in the RNA motif and recognize related sequences (<a id="xref-ref-103-1" class="xref-bibr" href="#ref-103">Wilinski <em>et al.</em> 2015</a>). Although <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> was detected with the vast majority of mRNAs associated with <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> and/or <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a>, an additional ∼1000 mRNAs were exclusive to the <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> data set. These messages were depleted of the <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> and/or <a href="http://www.yeastgenome.org/locus/S000002982/overview">Puf4</a> motifs (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a> and Figure S3D in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>, p-value &lt; 10<sup>−5</sup>), and <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> is likely recruited to this group of mRNAs by other RBPs (see below).</p><p id="p-30">The 5′ and 3′ UTRs of mRNAs contain sequences that control the translation, localization, and decay of mRNAs (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Keene 2007</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Hogan <em>et al.</em> 2008</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Lee and Lykke-Andersen 2013</a>). RIP sample preparation required a sonication step to solubilize cross-linked RNA–protein complexes, resulting in the shearing of RNAs to ∼200–600 nucleotides (nt). Thus, information on the spatial location of these proteins on mRNAs can be obtained. Composite plots were generated by aligning the sequencing reads to the TSS and the transcript TTS identified from a published high-resolution mapping of the transcriptome (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">David <em>et al.</em> 2006</a>). A strong enrichment of <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> was observed over the 3′ end of mRNAs (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>), which agrees with studies showing that <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> binds to the 3′ UTR of mRNAs (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Gerber <em>et al.</em> 2004</a>; <a id="xref-ref-38-5" class="xref-bibr" href="#ref-38">Goldstrohm <em>et al.</em> 2006</a>). Where <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> binds on an mRNA has not been examined on a global scale, but its function in deadenylation predicts that it would bind near the poly(A) tail. The mapping of reads from the <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> IP revealed that it is preferentially recruited to 3′ ends of mRNAs (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2C</a>). Likewise, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> showed a stronger enrichment to the 3′ end of mRNAs. While the cross-linking of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> was strongest at the 3′ end of mRNAs, reads above background were also observed in the middle and 5′ end. The detection of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> in the body of the mRNA is unlikely to result from incompletely fractionated mRNAs during sample preparation, because all three samples were processed identically and equivalent enrichment of <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> along the length of the mRNA was not observed. While it was unexpected that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> would be enriched at both ends of mRNAs, our results with <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> agree with a CLIP-seq study conducted in stressed cells, which observed it cross-linking to both the 5′ and 3′ ends of transcripts (<a id="xref-ref-70-3" class="xref-bibr" href="#ref-70">Mitchell <em>et al.</em> 2013</a>). The cross-linking of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> at multiple locations on the transcript could result from the packaging of mRNAs into mRNPs, and the juxtaposition of the 5′ and 3′ ends of the RNA caused by associations between factors binding to the 5′ and 3′ ends. Indeed, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> interacts with both <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not and factors associated with the 5′ cap of mRNAs (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Hata <em>et al.</em> 1998</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Maillet and Collart 2002</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Nissan <em>et al.</em> 2010</a>).</p><p id="p-31">Composite plots can be biased by a large number of reads from a few mRNAs or can underestimate the extent of recruitment to a smaller number of transcripts. Therefore, we calculated the enrichment of each factor over the 5′ UTR, the coding sequence (CDS), and the 3′ UTR of mRNAs with a length &gt; 400 nt. The frequency (number of mRNAs) that showed strong enrichment over these regions was calculated and displayed in <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2D</a>. Similar to what was observed in the composite plots, the results indicate that each protein was associated predominantly with the 3′ UTR of mRNAs, and that <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> and <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> were detected at the 5′ UTR and CDS of more mRNAs than <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a>. An alternative way to analyze this is to separate the mRNAs into thirds, based on lengths, and calculate the number of RNAs where enrichment was detected at the 5′ ends, middle, or 3′ ends of the mRNA. Here too, the results indicate that each protein predominantly associated with the 3′ ends of the mRNA (Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS3.pdf">File S3</a>). The recruitment of <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a>, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, and <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> to the 3′ ends of mRNAs was observed at individual mRNAs (Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>).</p><p id="p-32">This is the first study to map mRNAs bound by <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>; however, targets of <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> have been identified using RIP-Chip and CLIP-seq methods. We next compared our <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> and <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> targets to published data sets. A native RIP-CHIP procedure conducted on <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> identified far fewer RNA targets than we did (only 224; <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Gerber <em>et al.</em> 2004</a>), which may be explained by a lack of a cross-linking step and/or a high background of native RIP procedures. Nonetheless, we identified many of the same targets, and a significant positive correlation was observed between the RIP-Chip study and our own data (Figure S7, A and B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>).</p><p id="p-33">The number of transcripts bound to <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> in our study differs significantly from two other studies that used native (no cross-linking) conditions. When native RIP-Chip and RIP-seq methods were employed, &lt; 80 Dhh1-enriched RNAs were detected (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Hogan <em>et al.</em> 2008</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cary <em>et al.</em> 2015</a>). Considering the high abundance of <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> and its known functions in mRNA regulation, one can surmise that omitting a cross-linking step greatly underestimated the number of targets. <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> targets were identified using CLIP-seq in cells undergoing glucose deprivation stress (<a id="xref-ref-70-4" class="xref-bibr" href="#ref-70">Mitchell <em>et al.</em> 2013</a>). This study identified 299 high- and 1838 low-confidence mRNAs (twofold cutoff and 2% FDR). A comparison of the targets identified in our study (2262 targets) to those identified in the CLIP-seq study (1838 targets) found relatively little overlap between the two data sets (548 targets; Figure S7C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>). Since the CLIP-seq study was conducted in cells deprived of a carbon source (stressed), and ours was carried out in rich media, the relatively weak overlap between the two studies is not surprising, considering that glucose starvation causes strong processing body (p-body) formation and presumably widespread remodeling of RNA–protein interactions (<a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Teixeira <em>et al.</em> 2005</a>; <a id="xref-ref-70-5" class="xref-bibr" href="#ref-70">Mitchell <em>et al.</em> 2013</a>).</p></div><div id="sec-15" class="subsection"><h3>The mRNA decay machinery shares a regulatory network with other RBPs</h3><p id="p-34">Decay factors are recruited to mRNAs by RBPs that recognize sequence motifs located within the 3′ UTR (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Goldstrohm and Wickens 2008</a>; <a id="xref-ref-80-2" class="xref-bibr" href="#ref-80">Quenault <em>et al.</em> 2011</a>). Information on how <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> are recruited to mRNAs can be obtained by identifying sequence motifs in the bound mRNAs and by identifying overlap with publicly available RNA–protein interaction data sets. A search for sequence motifs among our highly enriched <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> targets using FIRE (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Elemento <em>et al.</em> 2007</a>) identified a motif resembling the <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> motif, as defined by the FIRE program (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>, p-value = 1.3 × 10<sup>−12</sup>). Identification of the <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> motif in our data set, and the correlation between our targets and those identified in other studies (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a> and Figure S7, A and B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>), validate our RIP-seq procedure. <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not is not known to display sequence-specific RNA binding, but instead is recruited to mRNAs via RBPs. The overlap of mRNA targets of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> suggests that <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> is a candidate (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2, A and B</a>), and we also discovered that the binding motif for <a href="http://www.yeastgenome.org/locus/S000003936/overview">Puf3</a> was overrepresented in the most highly enriched <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> targets (p-value = 6.5 × 10<sup>−4</sup>, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). It should be noted that the <a href="http://www.yeastgenome.org/locus/S000003936/overview">Puf3</a> motif emerging from the FIRE analysis of our data is a more restricted version than the motif identified in another study (<a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Gerber <em>et al.</em> 2004</a>). Finding the <a href="http://www.yeastgenome.org/locus/S000003936/overview">Puf3</a> motif overrepresented in <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> targets is in good agreement with a study suggesting that <a href="http://www.yeastgenome.org/locus/S000003936/overview">Puf3</a> interacts with <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> to regulate deadenylation of <em><a href="http://www.yeastgenome.org/locus/S000003932/overview">COX17</a></em> mRNA (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Lee <em>et al.</em> 2010</a>). This, and our own data, suggests that <a href="http://www.yeastgenome.org/locus/S000003936/overview">Puf3</a> plays a significant role in recruiting <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> to mRNAs across the genome. Neither our nor the published CLIP-seq study identified a motif significantly enriched in <a href="https://www.yeastgenome.org/locus/S000002319">Dhh1</a>-bound transcripts (<a id="xref-ref-70-6" class="xref-bibr" href="#ref-70">Mitchell <em>et al.</em> 2013</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/315/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The decay machinery is directed to mRNAs through a network of RNA-binding proteins (RBPs). Enrichment values were submitted to FIRE (finding informative regulatory elements; Elemento et al. 2007) using default setting options for Puf5 (A) and Ccr4 (B). Values were binned into six and eight bins, respectively. The black and red heatmap represents the distribution of enrichment values contained within each bin (see range of enrichment values to the left in red font). Yellow bins represent groups of genes that have a significant overrepresentation of the motif, while blue represent bins where the motif is significantly underrepresented. See Elemento et al. (2007) for a detailed description of how the optimized motifs and location were attained. (C) Pearson correlations between RNA immunoprecipitation high-throughput sequencing (RIP-seq) enrichment values and enrichment values of multiple RBP targets from another study (Hogan et al. 2008). Multiple testing correction was performed and only correlations that met a false discover rate &lt; 1% threshold are included." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1743458345" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The decay machinery is directed to mRNAs through a network of RNA-binding proteins (RBPs). Enrichment values were submitted to FIRE (finding informative regulatory elements; Elemento &lt;em&gt;et al.&lt;/em&gt; 2007) using default setting options for Puf5 (A) and Ccr4 (B). Values were binned into six and eight bins, respectively. The black and red heatmap represents the distribution of enrichment values contained within each bin (see range of enrichment values to the left in red font). Yellow bins represent groups of genes that have a significant overrepresentation of the motif, while blue represent bins where the motif is significantly underrepresented. See Elemento &lt;em&gt;et al.&lt;/em&gt; (2007) for a detailed description of how the optimized motifs and location were attained. (C) Pearson correlations between RNA immunoprecipitation high-throughput sequencing (RIP-seq) enrichment values and enrichment values of multiple RBP targets from another study (Hogan &lt;em&gt;et al.&lt;/em&gt; 2008). Multiple testing correction was performed and only correlations that met a false discover rate &lt; 1% threshold are included.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/315/F3.medium.gif" width="440" height="219"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/1/315/F3.medium.gif" width="440" height="219"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/315/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/315/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488067" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-35" class="first-child">The decay machinery is directed to mRNAs through a network of RNA-binding proteins (RBPs). Enrichment values were submitted to FIRE (finding informative regulatory elements; <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Elemento <em>et al.</em> 2007</a>) using default setting options for Puf5 (A) and Ccr4 (B). Values were binned into six and eight bins, respectively. The black and red heatmap represents the distribution of enrichment values contained within each bin (see range of enrichment values to the left in red font). Yellow bins represent groups of genes that have a significant overrepresentation of the motif, while blue represent bins where the motif is significantly underrepresented. See <a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Elemento <em>et al.</em> (2007)</a> for a detailed description of how the optimized motifs and location were attained. (C) Pearson correlations between RNA immunoprecipitation high-throughput sequencing (RIP-seq) enrichment values and enrichment values of multiple RBP targets from another study (<a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Hogan <em>et al.</em> 2008</a>). Multiple testing correction was performed and only correlations that met a false discover rate &lt; 1% threshold are included.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">To find other RBPs that coregulate targets of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, and <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a>, correlations were calculated between the enrichment values from this study and those of other RIP-Chip studies (<a id="xref-ref-47-5" class="xref-bibr" href="#ref-47">Hogan <em>et al.</em> 2008</a>) (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>). Generally, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, and <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> enrichment positively correlated with targets of sequence-specific RBPs that have been implicated in mRNA decay or translational repression, including <a href="http://www.yeastgenome.org/locus/S000005886/overview">Vts1</a>, <a href="http://www.yeastgenome.org/locus/S000002982/overview">Puf4</a>, and <a href="http://www.yeastgenome.org/locus/S000000128/overview">Khd1</a>. <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, and <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> have genetic and physical interactions with <a href="http://www.yeastgenome.org/locus/S000002982/overview">Puf4</a> and <a href="http://www.yeastgenome.org/locus/S000005886/overview">Vts1</a> (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Hook <em>et al.</em> 2007</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Rendl <em>et al.</em> 2008</a>; <a id="xref-ref-93-2" class="xref-bibr" href="#ref-93">Tarassov <em>et al.</em> 2008</a>), and <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000000128/overview">Khd1</a> regulate the mRNAs of <em><a href="http://www.yeastgenome.org/locus/S000002399/overview">LRG1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000004363/overview">ROM2</a></em> [also identified in this work (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS2.xlsx">File S2</a>)] (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ito <em>et al.</em> 2011</a>). Furthermore, there was a strong correlation between the mRNA targets identified here and those bound to <a href="http://www.yeastgenome.org/locus/S000003090/overview">Nab2</a> and <a href="http://www.yeastgenome.org/locus/S000002840/overview">Npl3</a>, two nuclear poly(A)-binding proteins connected to mRNA export (<a id="xref-ref-104-1" class="xref-bibr" href="#ref-104">Wilson <em>et al.</em> 1994</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Lee <em>et al.</em> 1996</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Hector <em>et al.</em> 2002</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Marfatia <em>et al.</em> 2003</a>). Physical interactions between <a href="http://www.yeastgenome.org/locus/S000003090/overview">Nab2</a> and <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not complexes have been identified, suggesting that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not regulates some aspect of mRNA transport (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Kerr <em>et al.</em> 2011</a>). <a href="http://www.yeastgenome.org/locus/S000003090/overview">Nab2</a> binds poly(A) tails, and while thought to protect mRNAs from degradation, may aid in the recruitment of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not to mRNAs in the nucleus. In contrast, enrichment of the decay factors examined here negatively correlated with the targets of factors involved in ribosomal biogenesis (<em>i.e</em>., <a href="http://www.yeastgenome.org/locus/S000004165/overview">Cbf5</a>, <a href="http://www.yeastgenome.org/locus/S000004187/overview">Nop56</a>, and <a href="http://www.yeastgenome.org/locus/S000003934/overview">Sof1</a>).</p></div><div id="sec-16" class="subsection"><h3>Ccr4-Not directly regulates both mRNA synthesis and decay, but identification of its targets suggests that its role in decay predominates in gene expression control</h3><p id="p-37">The function of a gene is often inferred from gene expression profiling of mutants. However, interpretation of this type of data is difficult if the gene of interest has multiple functions in the cell. For example, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> associate with RNA polymerase II and regulate transcription at the initiation and elongation stages of mRNA synthesis, and also function in decay (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Deluen <em>et al.</em> 2002</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Kruk <em>et al.</em> 2011</a>; <a id="xref-ref-68-4" class="xref-bibr" href="#ref-68">Miller and Reese 2012</a>). Deleting <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not subunits result in both increased and decreased expression of many genes (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Cui <em>et al.</em> 2008</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Azzouz <em>et al.</em> 2009</a>), but which effects result from changes in transcription <em>vs.</em> decay are not known.</p><p id="p-38">RIP-seq enrichment data were compared to the decay and synthesis rates of mRNAs determined by cDTA, which measures these parameters with minimal perturbations by metabolic labeling of mRNAs (<a id="xref-ref-89-5" class="xref-bibr" href="#ref-89">Sun <em>et al.</em> 2012</a>). <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> enrichment showed a positive correlation with mRNA decay rates (<em>R</em> = 0.43 and 0.38, respectively) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>), which is consistent with their role in mRNA decay (<a id="xref-ref-97-4" class="xref-bibr" href="#ref-97">Tucker <em>et al.</em> 2001</a>, <a id="xref-ref-98-1" class="xref-bibr" href="#ref-98">2002</a>). A positive correlation with decay rates, albeit not as strong, was observed also for <a href="https://www.yeastgenome.org/locus/S000003146">Puf5</a>-enriched mRNAs (<em>R</em> = 0.28). When the correlation between enrichment and decay rate was calculated using ranks (Spearman), the correlation was even stronger [0.53 (<a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>), 0.47 (<a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>), and 0.34 (<a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a>)]. In contrast, correlations between the RIP-seq data from all three proteins and mRNA synthesis rates were weaker, and showed a modest negative-to-no correlation (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/315/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Recruitment of Ccr4, Dhh1, and Puf5 correlates with decay and not synthesis rates. Pair-wise comparisons of enrichment values (y-axis) vs. decay rates (1/min) and synthesis rates (mRNA/Cell/Cell Cycle, Log2) from Sun et al. (2012) and ribosome footprints [reads per kilobase of transcript per million reads mapped (RPKM), Log2] from Gerashchenko et al. (2012). Numbers within the graphs represent Pearson correlation values. The dotted black trend line represents the least squares line of regression." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1743458345" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Recruitment of Ccr4, Dhh1, and Puf5 correlates with decay and not synthesis rates. Pair-wise comparisons of enrichment values (&lt;em&gt;y&lt;/em&gt;-axis) &lt;em&gt;vs.&lt;/em&gt; decay rates (1/min) and synthesis rates (mRNA/Cell/Cell Cycle, Log&lt;sub&gt;2&lt;/sub&gt;) from Sun &lt;em&gt;et al.&lt;/em&gt; (2012) and ribosome footprints [reads per kilobase of transcript per million reads mapped (RPKM), Log&lt;sub&gt;2&lt;/sub&gt;] from Gerashchenko &lt;em&gt;et al.&lt;/em&gt; (2012). Numbers within the graphs represent Pearson correlation values. The dotted black trend line represents the least squares line of regression.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/315/F4.medium.gif" width="440" height="385"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/1/315/F4.medium.gif" width="440" height="385"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/315/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/315/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487969" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-39" class="first-child">Recruitment of Ccr4, Dhh1, and Puf5 correlates with decay and not synthesis rates. Pair-wise comparisons of enrichment values (<em>y</em>-axis) <em>vs.</em> decay rates (1/min) and synthesis rates (mRNA/Cell/Cell Cycle, Log<sub>2</sub>) from <a id="xref-ref-89-6" class="xref-bibr" href="#ref-89">Sun <em>et al.</em> (2012)</a> and ribosome footprints [reads per kilobase of transcript per million reads mapped (RPKM), Log<sub>2</sub>] from <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Gerashchenko <em>et al.</em> (2012)</a>. Numbers within the graphs represent Pearson correlation values. The dotted black trend line represents the least squares line of regression.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">The synthesis rates measured by cDTA analysis were extrapolated from measured abundance and turnover rates (<a id="xref-ref-89-7" class="xref-bibr" href="#ref-89">Sun <em>et al.</em> 2012</a>). GRO-Chip analysis has been performed in yeast, which directly measures the transcription frequency across the genome (<a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Molina-Navarro <em>et al.</em> 2008</a>). To further verify that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> recruitment negatively correlated with mRNA synthesis rates, we examined the correlation between <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> recruitment and mRNA synthesis rates measured by GRO-Chip and found a similar anticorrelation (Pearson = −0.25, Figure S8 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>). Therefore, using two different estimates of mRNA synthesis, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> recruitment to an mRNA negatively correlated with its synthesis rate.</p><p id="p-41">Since <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> associate with elongating RNAPII (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Kruk <em>et al.</em> 2011</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Dutta <em>et al.</em> 2015</a>), we considered that the recruitment that we detected may be linked to the process of transcription. To examine this possibility, we first looked for enrichment of these factors within intronic sequences. Yeast has 282 genes with introns (<a href="http://intron.ucsc.edu/yeast4.1/">http://intron.ucsc.edu/yeast4.1/</a>), and less than nine of these mRNAs showed enrichment of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, or <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> over both the exon and intron (Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>). Additionally, neither <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> (<em>R</em> = −0.03) nor <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> (<em>R</em> = −0.10) recruitment to mRNAs correlated with a published genome-wide ChIP-seq experiment that mapped the association of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not subunits with genes (<a id="xref-ref-99-2" class="xref-bibr" href="#ref-99">Venters <em>et al.</em> 2011</a>). Hence, few RNA–protein interactions detected here are likely to occur cotranscriptionally, and they are more likely to occur on cytoplasmic RNAs. Taken together, these results suggest that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not recruitment to mRNAs reflects its function in decay as opposed to synthesis.</p><p id="p-42">A recent report suggested that the synthesis of ribosomal protein mRNAs may be regulated by <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not in a Not5-dependent manner during stress (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Gupta <em>et al.</em> 2016</a>). However, we did not find a strong correlation with synthesis across the genome. The differences may be related to the subunits examined (NOTs <em>vs.</em> <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>), the RNA enrichment strategy (native <em>vs.</em> cross-linked), or the positive correlation with transcription being restricted to a particular class of genes (see <em>Discussion</em>).</p></div><div id="sec-17" class="subsection"><h3>mRNA decay factor recruitment anticorrelates with ribosome occupancy</h3><p id="p-43">A long-standing model posits that mRNAs are either partitioned into the translatable or nontranslatable pool, with the former category encompassing mRNAs that are associated with ribosomes and undergoing translation (<a id="xref-ref-94-2" class="xref-bibr" href="#ref-94">Teixeira <em>et al.</em> 2005</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Parker and Sheth 2007</a>). We next explored the relationship between decay factor recruitment and the translation of mRNAs measured by Ribo-seq (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ingolia <em>et al.</em> 2009</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Gerashchenko <em>et al.</em> 2012</a>). <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, and <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> recruitment displayed an anticorrelation with ribosome footprint data (R = −0.52, −0.42 and −0.33, respectively) (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). Thus, mRNAs bound to decay factors, especially <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, are less likely to be translated at a high rate. This observation supports the long-standing model that deadenylation and translation are in competition (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Parker and Sheth 2007</a>), and is also consistent with the role of <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> in translation repression (<a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Presnyak and Coller 2013</a>).</p></div><div id="sec-18" class="subsection"><h3>Identification of Ccr4-associated mRNAs suggests that the two deadenylase complexes in yeast regulate different groups of mRNAs</h3><p id="p-44"><a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000003062/overview">Pan2</a>/<a href="https://www.yeastgenome.org/locus/S000001508">3</a> are the main mRNA deadenylases in yeast, and two models have been proposed to explain how they cooperate in the decay of messages. The first model suggests that <a href="http://www.yeastgenome.org/locus/S000003062/overview">Pan2</a>/<a href="https://www.yeastgenome.org/locus/S000001508">3</a> initiates poly(A) shortening, which is then followed by complete deadenylation by <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Brown and Sachs 1998</a>; <a id="xref-ref-97-5" class="xref-bibr" href="#ref-97">Tucker <em>et al.</em> 2001</a>; <a id="xref-ref-100-2" class="xref-bibr" href="#ref-100">Wahle and Winkler 2013</a>). The second model, based on a study that found no correlation between the changes in decay rates of mRNAs in a <em><a href="http://www.yeastgenome.org/locus/S000000019/overview">ccr4</a></em>∆ <em>vs.</em> a <em><a href="http://www.yeastgenome.org/locus/S000003062/overview">pan2</a></em>∆ mutant, suggests that they each target different collections of mRNAs (<a id="xref-ref-90-4" class="xref-bibr" href="#ref-90">Sun <em>et al.</em> 2013</a>). Little is known about the interplay between these two complexes on the transcriptome; thus, we used the <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> RIP-seq data to shed light on the contributions of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000003062/overview">Pan2</a>/<a href="https://www.yeastgenome.org/locus/S000001508">3</a> to mRNA decay.</p><p id="p-45">The <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-enriched mRNAs were divided equally into five percentile ranks, and boxplots were created using the relative abundance values in a <em><a href="http://www.yeastgenome.org/locus/S000000019/overview">ccr4</a></em>∆ mutant <em>vs.</em> a wild-type strain (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). This plot illustrates that the most highly enriched mRNAs increased in abundance in a <em><a href="http://www.yeastgenome.org/locus/S000000019/overview">ccr4</a></em>∆ deletion (top 20%, p-value &lt; 2.2 × 10<sup>−16</sup>), and the lowly (essentially no recruitment) enriched mRNAs actually decreased in abundance (bottom 20%, p-value &lt; 2.2 × 10<sup>−16</sup>). Although almost all mRNAs had decreased decay rates in the <em><a href="http://www.yeastgenome.org/locus/S000000019/overview">ccr4</a></em>∆ mutant, mRNAs that were highly enriched by <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> had the largest decrease in their decay rate (p-value &lt; 2.2 × 10<sup>−16</sup>, <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>), and those that were lowly recruited had the smallest decrease in their decay rate (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>). Similar correlations were also observed between <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> enrichment and changes in abundance and decay in a <em><a href="http://www.yeastgenome.org/locus/S000002319/overview">dhh1</a></em>∆ mutant (Figure S9, A and B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/315/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of Ccr4 targets suggests a division of labor between the two cytoplasmic deadenylases. (A) Ccr4 RNA immunoprecipitation high-throughput sequencing (RIP-seq) enrichment values were equally divided into five groups based upon their percent rank (i.e., 80&#x2013;100 is the top 20th percentile). Each boxplot represents the log2 change in abundance in a ccr4&#x2206; strain relative to a wild-type strain (Sun et al. 2013). Statistical significance was calculated using a Wilcoxon rank sum test with continuity correction. The red line indicates the median of all values. (B) The same as (A) but the boxplots represent the log2 change in decay rates in a ccr4&#x2206; strain relative to wild-type. (C) The same as (A) but the boxplots represent the log2 change in expression in a pan2&#x2206; strain relative to wild-type. (D) The same as (A) but the boxplots represent the log2 change in decay rates in a pan2&#x2206; strain relative to wild-type. In order to clearly visualize the boxplots in (A), (C), and (D), two outlier data points were removed from the y-axis. * p-value &lt; 0.001, ** p-value &lt; 0.0001, and *** p-value &lt; 2.2 &#xD7; 10&#x2212;16." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1743458345" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identification of Ccr4 targets suggests a division of labor between the two cytoplasmic deadenylases. (A) Ccr4 RNA immunoprecipitation high-throughput sequencing (RIP-seq) enrichment values were equally divided into five groups based upon their percent rank (&lt;em&gt;i.e.&lt;/em&gt;, 80&#x2013;100 is the top 20th percentile). Each boxplot represents the log2 change in abundance in a &lt;em&gt;ccr4&lt;/em&gt;&#x2206; strain relative to a wild-type strain (Sun &lt;em&gt;et al.&lt;/em&gt; 2013). Statistical significance was calculated using a Wilcoxon rank sum test with continuity correction. The red line indicates the median of all values. (B) The same as (A) but the boxplots represent the log2 change in decay rates in a &lt;em&gt;ccr4&lt;/em&gt;&#x2206; strain relative to wild-type. (C) The same as (A) but the boxplots represent the log2 change in expression in a &lt;em&gt;pan2&lt;/em&gt;&#x2206; strain relative to wild-type. (D) The same as (A) but the boxplots represent the log2 change in decay rates in a &lt;em&gt;pan2&lt;/em&gt;&#x2206; strain relative to wild-type. In order to clearly visualize the boxplots in (A), (C), and (D), two outlier data points were removed from the &lt;em&gt;y&lt;/em&gt;-axis. * p-value &lt; 0.001, ** p-value &lt; 0.0001, and *** p-value &lt; 2.2 &#xD7; 10&lt;sup&gt;&#x2212;16&lt;/sup&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/315/F5.medium.gif" width="404" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/1/315/F5.medium.gif" width="404" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/315/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/315/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487996" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-46" class="first-child">Identification of Ccr4 targets suggests a division of labor between the two cytoplasmic deadenylases. (A) Ccr4 RNA immunoprecipitation high-throughput sequencing (RIP-seq) enrichment values were equally divided into five groups based upon their percent rank (<em>i.e.</em>, 80–100 is the top 20th percentile). Each boxplot represents the log2 change in abundance in a <em>ccr4</em>∆ strain relative to a wild-type strain (<a id="xref-ref-90-5" class="xref-bibr" href="#ref-90">Sun <em>et al.</em> 2013</a>). Statistical significance was calculated using a Wilcoxon rank sum test with continuity correction. The red line indicates the median of all values. (B) The same as (A) but the boxplots represent the log2 change in decay rates in a <em>ccr4</em>∆ strain relative to wild-type. (C) The same as (A) but the boxplots represent the log2 change in expression in a <em>pan2</em>∆ strain relative to wild-type. (D) The same as (A) but the boxplots represent the log2 change in decay rates in a <em>pan2</em>∆ strain relative to wild-type. In order to clearly visualize the boxplots in (A), (C), and (D), two outlier data points were removed from the <em>y</em>-axis. * p-value &lt; 0.001, ** p-value &lt; 0.0001, and *** p-value &lt; 2.2 × 10<sup>−16</sup>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">mRNAs that were stabilized in a <em><a href="http://www.yeastgenome.org/locus/S000003062/overview">pan2</a></em>∆ mutant were either not strongly affected or were turned over more rapidly in a <em><a href="http://www.yeastgenome.org/locus/S000000019/overview">ccr4</a></em>∆ strain (<a id="xref-ref-90-6" class="xref-bibr" href="#ref-90">Sun <em>et al.</em> 2013</a>), suggesting that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000003062/overview">Pan2</a>/<a href="https://www.yeastgenome.org/locus/S000001508">3</a> may regulate different mRNAs. Consistent with this, we found that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-enriched transcripts displayed reduced abundance (p-value &lt; 2.2 × 10<sup>−16</sup>, <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>) and increased decay rates (p-value &lt; 0.001, <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5D</a>) in the <em><a href="http://www.yeastgenome.org/locus/S000003062/overview">pan2</a></em>∆ mutant. Also, mRNAs that were lowly enriched increased in abundance (p-value &lt; 2.2 × 10<sup>−16</sup>, <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5C</a>) and displayed reduced decay rates (p-value &lt; 0.001, <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5D</a>) in the <em><a href="http://www.yeastgenome.org/locus/S000003062/overview">pan2</a></em>∆ mutant. In other words, the abundance and stability of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-bound mRNAs are most affected by the deletion of <em><a href="http://www.yeastgenome.org/locus/S000000019/overview">CCR4</a></em>, while the lowly enriched targets are more strongly affected by the deletion of <em><a href="http://www.yeastgenome.org/locus/S000003062/overview">PAN2</a></em>. The decrease in abundance of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> targets in the <em><a href="http://www.yeastgenome.org/locus/S000003062/overview">pan2</a></em>∆ mutant is interesting. The cause of this is not known but it suggests that, in the absence of <a href="http://www.yeastgenome.org/locus/S000003062/overview">Pan2</a>, either <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> compensates for its loss by increasing its activity or another redundant pathway, such as the exosome, becomes more prevalent.</p></div><div id="sec-19" class="subsection"><h3>Ccr4 and Dhh1 bind mRNAs that respond to the metabolic state of the cell</h3><p id="p-48">To identify the pathways regulated by <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, and <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a>, lists of the most highly enriched mRNAs (top 20%) were submitted to GO terms analysis. The <a href="https://www.yeastgenome.org/locus/S000003146">Puf5</a>-bound mRNAs had terms associated with transcription and RNA biogenesis overrepresented, among others (Figure S10 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>). This is consistent with a previous native RIP-Chip study, which also revealed that <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> targets transcriptional and chromatin regulators (<a id="xref-ref-35-6" class="xref-bibr" href="#ref-35">Gerber <em>et al.</em> 2004</a>). The mRNAs that showed the highest level of recruitment to <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> code for proteins involved in many of the same processes, such as RNA metabolism, nitrogen compound metabolism, transcription, and several biosynthetic processes (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6, A and B</a>). The identification of many of the same GO terms in the <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> data sets is consistent with the high degree of overlap between the mRNA targets of these two proteins (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2, A and B</a>). <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> was also recruited to 232 mRNAs important for organelle organization processes (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). This collection of mRNAs was further analyzed using GO Slim Mapper to narrow down the organelle(s), which revealed that 26% of the messages were specifically connected to mitochondrial organization (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS4.xlsx">File S4</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/315/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ccr4 and Dhh1 are recruited to mRNAs involved in metabolic processes (A). Gene ontology (GO) terms analysis of the top 20th percentile of Ccr4-enriched RNAs (N = 846 genes). (B) The same as (A), except that the top 20th percentile of Dhh1-enriched RNAs (N = 773 genes) was analyzed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1743458345" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ccr4 and Dhh1 are recruited to mRNAs involved in metabolic processes (A). Gene ontology (GO) terms analysis of the top 20th percentile of Ccr4-enriched RNAs (&lt;em&gt;N&lt;/em&gt; = 846 genes). (B) The same as (A), except that the top 20th percentile of Dhh1-enriched RNAs (&lt;em&gt;N&lt;/em&gt; = 773 genes) was analyzed.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/315/F6.medium.gif" width="440" height="187"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/1/315/F6.medium.gif" width="440" height="187"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/315/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/315/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487929" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-49" class="first-child">Ccr4 and Dhh1 are recruited to mRNAs involved in metabolic processes (A). Gene ontology (GO) terms analysis of the top 20th percentile of Ccr4-enriched RNAs (<em>N</em> = 846 genes). (B) The same as (A), except that the top 20th percentile of Dhh1-enriched RNAs (<em>N</em> = 773 genes) was analyzed.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">Since mRNAs involved in multiple metabolic processes were identified in the RIP-seq data sets, we examined the correlation between our recruitment data and gene expression changes during metabolic stress (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bradley <em>et al.</em> 2009</a>). We found that the most highly enriched mRNAs (top 10th percentile) bound to <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> overlapped significantly with genes that are induced during nitrogen and carbon starvation (Figure S11, A and B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>; p-values from &lt; 0.001 to &lt; 1 × 10<sup>−14</sup>). In order to establish if this relationship reached beyond the top 10th percentile of targets, we calculated the correlation between RIP-seq data and the changes in gene expression over time during carbon and nitrogen starvation (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> and Figures S11 and S12 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>). <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> showed positive correlations between its enrichment values and gene expression levels during carbon starvation (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a>). Furthermore, the Pearson correlation values increased as cells progressed deeper into the starvation program, which most likely results from the accumulation of more genes with expression changes over time (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Bradley <em>et al.</em> 2009</a>). Although <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> enrichment did not correlate with gene expression changes during nutritional stress as strongly as <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> did, the same trends between recruitment and gene expression changes were still observed (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7B</a> and Figures S11 and S12 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/315/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Ccr4 and Dhh1 are recruited to mRNAs that respond to nutrient and metabolic signals. (A) Pearson correlation values were calculated between Ccr4 RNA immunoprecipitation high-throughput sequencing (RIP-seq) enrichment and expression values from the carbon depletion experiment in Bradley et al. (2009). Each column represents a different time point after carbon depletion relative to the unstressed condition. (B) The same as in (A), except that Dhh1 RIP-seq enrichment values were used. (C and D) Scatter plots comparing RIP-seq enrichment to the change in decay rates in a deletion strain relative to wild-type. The plots for Ccr4 (C) and Dhh1 (D) are shown. The color of each data point represents how the expression changed at the 240 min time point after carbon depletion from Bradley et al. (2009). Genes that were upregulated (log2 expression &gt; 1), downregulated (log2 expression &lt; &#x2212;1), or unchanged were colored red, blue, and gray, respectively. The dashed black line trend line represents the least squares line of regression. (E) A &#x3C7;2 test was performed between the highly enriched Ccr4 targets (top 10th percentile) against the upregulated genes [top 10th percentile of genes (red area)] and downregulated genes [bottom 10th percentile (blue area)] from Tu et al. (2005). The periods when oxygen consumption was increasing (Inc) or decreasing (Dec) is displayed above the graph." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1743458345" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Ccr4 and Dhh1 are recruited to mRNAs that respond to nutrient and metabolic signals. (A) Pearson correlation values were calculated between Ccr4 RNA immunoprecipitation high-throughput sequencing (RIP-seq) enrichment and expression values from the carbon depletion experiment in Bradley &lt;em&gt;et al.&lt;/em&gt; (2009). Each column represents a different time point after carbon depletion relative to the unstressed condition. (B) The same as in (A), except that Dhh1 RIP-seq enrichment values were used. (C and D) Scatter plots comparing RIP-seq enrichment to the change in decay rates in a deletion strain relative to wild-type. The plots for Ccr4 (C) and Dhh1 (D) are shown. The color of each data point represents how the expression changed at the 240 min time point after carbon depletion from Bradley &lt;em&gt;et al.&lt;/em&gt; (2009). Genes that were upregulated (log2 expression &gt; 1), downregulated (log2 expression &lt; &#x2212;1), or unchanged were colored red, blue, and gray, respectively. The dashed black line trend line represents the least squares line of regression. (E) A &#x3C7;&lt;sup&gt;2&lt;/sup&gt; test was performed between the highly enriched Ccr4 targets (top 10th percentile) against the upregulated genes [top 10th percentile of genes (red area)] and downregulated genes [bottom 10th percentile (blue area)] from Tu &lt;em&gt;et al.&lt;/em&gt; (2005). The periods when oxygen consumption was increasing (Inc) or decreasing (Dec) is displayed above the graph.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/315/F7.medium.gif" width="321" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/1/315/F7.medium.gif" width="321" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/315/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/315/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487868" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-51" class="first-child">Ccr4 and Dhh1 are recruited to mRNAs that respond to nutrient and metabolic signals. (A) Pearson correlation values were calculated between Ccr4 RNA immunoprecipitation high-throughput sequencing (RIP-seq) enrichment and expression values from the carbon depletion experiment in <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Bradley <em>et al.</em> (2009)</a>. Each column represents a different time point after carbon depletion relative to the unstressed condition. (B) The same as in (A), except that Dhh1 RIP-seq enrichment values were used. (C and D) Scatter plots comparing RIP-seq enrichment to the change in decay rates in a deletion strain relative to wild-type. The plots for Ccr4 (C) and Dhh1 (D) are shown. The color of each data point represents how the expression changed at the 240 min time point after carbon depletion from <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Bradley <em>et al.</em> (2009)</a>. Genes that were upregulated (log2 expression &gt; 1), downregulated (log2 expression &lt; −1), or unchanged were colored red, blue, and gray, respectively. The dashed black line trend line represents the least squares line of regression. (E) A χ<sup>2</sup> test was performed between the highly enriched Ccr4 targets (top 10th percentile) against the upregulated genes [top 10th percentile of genes (red area)] and downregulated genes [bottom 10th percentile (blue area)] from <a id="xref-ref-96-1" class="xref-bibr" href="#ref-96">Tu <em>et al.</em> (2005)</a>. The periods when oxygen consumption was increasing (Inc) or decreasing (Dec) is displayed above the graph.</p><div class="sb-div caption-clear"></div></div></div><p id="p-52">We found that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> recruitment correlated with changes in gene expression during carbon source deprivation stress. We next attempted to determine how many of these differentially expressed mRNAs are dependent upon <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, and to what extent the correlations are driven by changes in the decay of these messages. We constructed scatter plots that displayed mRNA enrichment (<em>y</em>-axis) <em>vs.</em> changes in decay rates (<em>x</em>-axis) in the mutants. Each point was then colored red, blue, or gray to indicate if the mRNA expression increased, decreased, or remained unchanged in stressed cells, respectively. The plot shows that the upregulated mRNAs were predominantly those that showed the highest <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> recruitment and the largest decrease in decay rate in the <em><a href="http://www.yeastgenome.org/locus/S000000019/overview">ccr4</a></em>∆ mutant (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7C</a>, <em>R</em> = −0.31, p-value &lt; 2.2e−16). Conversely, the genes that were downregulated more often had transcripts that displayed a modest increase in decay rate and were less enriched with <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> compared to upregulated genes. The same trend was observed for <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7D</a>), but to a lesser extent (<em>R</em> = −0.16). Finally, repeating the same analysis in cells undergoing nitrogen depletion stress also revealed the same pattern (Figure S12 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>). In summary, the genes that are differentially expressed during metabolic stress are regulated during unstressed conditions by <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, which likely involves changes in mRNA stability.</p><p id="p-53">Methods of carbon and nitrogen depletion differ between studies, as do other variables. For example, the study by <a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Bradley <em>et al.</em> (2009</a>), which was used as the data set for our analysis, depleted nutrients by transferring cells on membranes between different media conditions (in this case, nitrogen base plates replete of depleted of nutrients). To demonstrate that our conclusions are robust and not unique to any depletion technique or condition, we repeated our analysis using data sets from two other studies measuring gene expression changes during nutrient limitations (Figure S11C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>). One study that measured the response to nitrogen depletion shifted cells from liquid culture in nitrogen-containing to nitrogen-depleted medium (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Gasch <em>et al.</em> 2000</a>). Another measured changes in gene expression by shifting cells in rich (YPD) liquid media to medium lacking dextrose (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Arribere <em>et al.</em> 2011</a>). The latter condition is most like those used to grow cells in this work. Repeating the analysis using these other data sets lead to the same conclusion: the RIP-seq data correlated with the changes in gene expression during carbon and nitrogen starvation (Figure S11C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>). Thus, we found similar correlations between the RIP-seq targets and changes in gene expression from three different studies, suggesting that the correlations we identified are robust and not unique to any one depletion protocol, data set, or strain used to generate the expression data.</p><p id="p-54">The identification of many mRNA targets associated with metabolic processes prompted us to investigate the relationship between mRNA decay machinery recruitment and changes in gene expression during the YMC. The YMC is a process in which budding yeast cells oscillate between the storage (reductive, nonrespiratory phase) and consumption (oxidative, respiration) of metabolites, which produces waves of gene expression that correlate with bursts of O<sub>2</sub> utilization (<a id="xref-ref-96-2" class="xref-bibr" href="#ref-96">Tu <em>et al.</em> 2005</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cai and Tu 2012</a>). The overlap between the most highly enriched <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> targets (top 10%) and mRNAs that were highly or lowly expressed during the YMC was calculated, a χ<sup>2</sup> test was performed, and the p-values of the overlap were plotted <em>vs.</em> time (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7E</a>). The top <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> targets significantly overlapped (p-value &lt; 10<sup>−4</sup>) with the most highly expressed genes when cells decreased their O<sub>2</sub> consumption during the nonrespiratory phase (<a id="xref-fig-7-7" class="xref-fig" href="#F7">Figure 7E</a>, see time points 2, 14–15, and 26–28). Strikingly, as cells progressed out of a nonrespiratory phase and into the respiratory phase (increasing O<sub>2</sub> consumption), the top <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> targets significantly overlapped (p-value &lt; 10<sup>−4</sup>) with the strongly repressed genes (bottom 10%) (<a id="xref-fig-7-8" class="xref-fig" href="#F7">Figure 7E</a>, see time points 8–12, 19–24, 32, and 34–36). These patterns suggest that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> may target specific transcripts in response to metabolic signals that arise from a change in the redox or metabolic state of the cell. Comparing the overlap between <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> targets and fluctuations in gene expression produced similar trends, although the level of overlap, based on p-values, was less extensive than those observed for <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-associated transcripts (Figure S12A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>). In contrast, there was no significant overlap between <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> targets and the genes fluctuating during the YMC (Figure S12B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>). Interestingly, the <a href="http://www.yeastgenome.org/locus/S000003936/overview">Puf3</a> motif was overrepresented both among the top <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> targets identified in our study (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a>) and among the genes that showed peak expression during the transition from increased to decreased oxygen consumption (<a id="xref-ref-96-3" class="xref-bibr" href="#ref-96">Tu <em>et al.</em> 2005</a>). This suggests that <a href="http://www.yeastgenome.org/locus/S000003936/overview">Puf3</a>, a protein shown to recruit <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not to a single mRNA encoding a protein involved in regulating mitochondrial respiration, <a href="http://www.yeastgenome.org/locus/S000003932/overview">COX17</a>, (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Lee <em>et al.</em> 2010</a>), regulates the association of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not with many transcripts during the YMC.</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-55">The integration of mRNA expression data (<em>e.g.</em>, microarrays) with transcription factor recruitment throughout the genome (ChIP-Chip or ChIP-seq) has been an important advance in our understanding the mechanisms of transcription regulation. For instance, correlating the binding of RNA polymerase II to genes with changes in mRNA levels has identified direct effects and provided deeper mechanistic insights into the process of gene transcription. In contrast to transcriptional regulators, which have been studied extensively, much less is known about the relationship between the binding of mRNA regulatory factors and the fate of mRNAs on a global scale. Our study identified the targets of three characterized mRNA decay factors, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, and <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a>, and these data were used to address their roles in the control of gene expression.</p><div id="sec-21" class="subsection"><h3>Identifying mRNA targets of Ccr4 provides insights into its role in transcription and decay</h3><p id="p-56">The <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not complex regulates mRNA levels by directly regulating synthesis in the nucleus and mRNA decay in the cytoplasm (<a id="xref-ref-68-5" class="xref-bibr" href="#ref-68">Miller and Reese 2012</a>). Expression profiling in <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not mutants has been performed (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Cui <em>et al.</em> 2008</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Azzouz <em>et al.</em> 2009</a>; <a id="xref-ref-90-7" class="xref-bibr" href="#ref-90">Sun <em>et al.</em> 2013</a>), but disentangling direct <em>vs.</em> indirect effects and attributing a change in mRNA levels to altered synthesis or decay is not trivial without a global map of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>–mRNA interactions. We utilized existing genome-wide mRNA synthesis and decay rate data to illustrate that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> recruitment, and that of its binding partners <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> and <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a>, correlated better with decay rather than synthesis. In fact, the highly enriched mRNAs displayed a lower synthesis rate overall. Considering that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not directly promotes elongation <em>in vitro</em> (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Kruk <em>et al.</em> 2011</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Babbarwal <em>et al.</em> 2014</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Dutta <em>et al.</em> 2015</a>), cross-links to active genes (<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">Kruk <em>et al.</em> 2011</a>; <a id="xref-ref-99-3" class="xref-bibr" href="#ref-99">Venters <em>et al.</em> 2011</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Gupta <em>et al.</em> 2016</a>), and was first identified as a transcription factor (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Collart 2003</a>; <a id="xref-ref-68-6" class="xref-bibr" href="#ref-68">Miller and Reese 2012</a>), its anticorrelation to synthesis rates was surprising on the surface. However, an explanation for this observation is that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not is utilized more at lowly synthesized genes that are under greater elongation control.</p><p id="p-57"><a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> is the main cytoplasmic deadenylase in eukaryotes and plays a global role in decay. Therefore, it might be expected that it is bound to nearly every mRNA. We found this not to be the case. There are a number of factors, some technical and others biological, which can explain this. Technically, the stringent 1% FDR filtering and imposition of a twofold cut off would exclude some mRNAs from the data set. A biological explanation is redundancy with the other deadenylase, <a href="http://www.yeastgenome.org/locus/S000003062/overview">Pan2</a>. Comparison of the level of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> recruitment to the changes in gene expression in <em><a href="http://www.yeastgenome.org/locus/S000000019/overview">ccr4</a></em>∆ and <em><a href="http://www.yeastgenome.org/locus/S000003062/overview">pan2</a></em>∆ mutants showed that mRNAs that are not/lowly bound to <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> are more strongly affected by the deletion of <em><a href="http://www.yeastgenome.org/locus/S000003062/overview">PAN2</a></em> (<a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>). This is highly suggestive of redundancy between these two deadenylases. There may be a contribution from deadenylation-independent decay pathways as well, such as exosome-mediated decay (<a id="xref-ref-88-2" class="xref-bibr" href="#ref-88">Schmid and Jensen 2008</a>).</p></div><div id="sec-22" class="subsection"><h3>Analysis of decay factor targets does not support a widespread imprinting mechanism as a major mode of action to regulate mRNA levels</h3><p id="p-58">As stated above, decay factors such as <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, <a href="http://www.yeastgenome.org/locus/S000003141/overview">Xrn1</a>, <a href="http://www.yeastgenome.org/locus/S000003660/overview">Lsm1</a>, and <a href="http://www.yeastgenome.org/locus/S000005062/overview">Dcp2</a> cross-link to the promoters and open reading frames of highly transcribed genes (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Lin <em>et al.</em> 2008</a>; <a id="xref-ref-54-5" class="xref-bibr" href="#ref-54">Kruk <em>et al.</em> 2011</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Haimovich <em>et al.</em> 2013b</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Faraji <em>et al.</em> 2016</a>). These findings, along with others, have led to speculation that mRNAs could be imprinted with decay factors during transcription in the nucleus, where they mark mRNAs for post-transcriptional regulation and decay in the cytoplasm (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Haimovich <em>et al.</em> 2013a</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Reese 2013</a>). <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not is a good candidate to carry out this function, since it participates in both the synthesis and destruction of mRNAs (<a id="xref-ref-68-7" class="xref-bibr" href="#ref-68">Miller and Reese 2012</a>; <a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">Reese 2013</a>; <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Collart 2016</a>; <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Gupta <em>et al.</em> 2016</a>). If the imprinting of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> onto mRNAs during transcription was an obligate step in the targeting of mRNAs for regulation in the cytoplasm, one might expect that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> would be loaded onto the vast majority of newly synthesized RNAs proportional to their synthesis; thus, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> recruitment would correlate positively with transcription rates. As discussed above, this was not the case. These results do not rule out the possibility that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not is loaded onto mRNAs cotranscriptionally and escorts them out into the cytoplasm, but once in the cytoplasm its binding to mRNAs may be subject to dynamic regulation driven by normal cellular cues and/or redistribution to specific mRNAs by sequence-specific mRNA-binding proteins.</p><p id="p-59">A recent paper described a genome-wide map of <a href="http://www.yeastgenome.org/locus/S000000689/overview">Not1</a> targets (<a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">Gupta <em>et al.</em> 2016</a>). The authors of that paper concluded that mRNAs are imprinted with <a href="http://www.yeastgenome.org/locus/S000000689/overview">Not1</a> in the nucleus, which affects the ultimate translation of the mRNA. This appears to apply to a subset of genes, such as those encoding ribosomal proteins. We found some similarities between the <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> targets described here and <a href="http://www.yeastgenome.org/locus/S000000689/overview">Not1</a> targets identified in the other study. For example, both <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000000689/overview">Not1</a> associated with mRNAs with a <a href="http://www.yeastgenome.org/locus/S000003936/overview">Puf3</a> motif and were bound by other RBPs known to genetically or physically interact with <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not (<a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">Gupta <em>et al.</em> 2016</a>). Nonetheless, the correlation between the <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000000689/overview">Not1</a> RIP-seq data was weak (Spearman’s <em>R</em> = 0.11, Figure S13A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>), and our <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> recruitment data more strongly correlated with global mRNA decay rates (<em>R</em> = 0.53) than the <a href="http://www.yeastgenome.org/locus/S000000689/overview">Not1</a> data did (<em>R</em> = 0.06) (see <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a> and Figure S13B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>). Additionally, we found a smaller number of overlapping (562 mRNAs) targets than anticipated, considering that the two proteins reside in the same complex and results indicate that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> must bind to <a href="http://www.yeastgenome.org/locus/S000000689/overview">Not1</a> to carryout mRNA decay (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Basquin <em>et al.</em> 2012</a>). Having said that, technical differences between the studies may explain this. Most notably, we used rapid formaldehyde cross-linking and detergents in the isolation step, which allows for more stringent determination of targets, and would prevent the redistribution of RBPs and inhibit degradation of mRNAs during isolation (<a id="xref-ref-84-2" class="xref-bibr" href="#ref-84">Riley and Steitz 2013</a>). As we have found to be the case for <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, studies not using a cross-linking step (<a id="xref-ref-47-6" class="xref-bibr" href="#ref-47">Hogan <em>et al.</em> 2008</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Cary <em>et al.</em> 2015</a>) identified far fewer targets than those that did [ours and that of <a id="xref-ref-70-7" class="xref-bibr" href="#ref-70">Mitchell <em>et al.</em> (2013</a>)]. Another significant difference is the proteins examined. We examined <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, proteins that form the nuclease module of the complex. It is possible that <a href="http://www.yeastgenome.org/locus/S000000689/overview">Not1</a> has a different behavior than the nuclease submodule subunits analyzed here. <a id="xref-ref-41-7" class="xref-bibr" href="#ref-41">Gupta <em>et al.</em> (2016</a>) demonstrated by immunofluorescence that <a href="http://www.yeastgenome.org/locus/S000000689/overview">Not1</a> staining is largely nuclear, while multiple studies have shown that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> are predominantly cytoplasmic (<a id="xref-ref-97-6" class="xref-bibr" href="#ref-97">Tucker <em>et al.</em> 2001</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Fischer and Weis 2002</a>; <a id="xref-ref-70-8" class="xref-bibr" href="#ref-70">Mitchell <em>et al.</em> 2013</a>). Furthermore, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> (and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>) bind the N-terminus of <a href="http://www.yeastgenome.org/locus/S000000689/overview">Not1</a> and display different genetic interactions than those that bind to the C-terminus of <a href="http://www.yeastgenome.org/locus/S000000689/overview">Not1</a>, such as <a href="http://www.yeastgenome.org/locus/S000006276/overview">Not5</a> (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Collart 2003</a>). Since there is the potential for heterogeneity among <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not complexes in yeast and metazoans [for review see <a id="xref-ref-68-8" class="xref-bibr" href="#ref-68">Miller and Reese (2012)</a>], it is possible that separate pools of <a href="http://www.yeastgenome.org/locus/S000000689/overview">Not1</a> (with <a href="http://www.yeastgenome.org/locus/S000006276/overview">Not5</a> and others) are dedicated to global decay and imprinting.</p></div><div id="sec-23" class="subsection"><h3>mRNAs highly enriched with decay factors are less likely to have high ribosome occupancy</h3><p id="p-60">A long-standing model for the fate of mRNAs is that they are partitioned into either the translatable or nontranslatable pool, with the latter category encompassing mRNAs that are stored in mRNPs or undergoing degradation (<a id="xref-ref-94-3" class="xref-bibr" href="#ref-94">Teixeira <em>et al.</em> 2005</a>; <a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Parker and Sheth 2007</a>). However, recently, the lines have become blurred. The processes of translation and decay is more intertwined than was initially thought, and some mRNA decay factors have also been implicated in regulating translation (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Coller and Parker 2005</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Chritton and Wickens 2010</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Pelechano <em>et al.</em> 2015</a>; <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Collart 2016</a>; <a id="xref-ref-41-8" class="xref-bibr" href="#ref-41">Gupta <em>et al.</em> 2016</a>). In addition, decay factors and 5′ to 3′ decay intermediates have been detected among polyribosomes (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Hu <em>et al.</em> 2009</a>; <a id="xref-ref-91-2" class="xref-bibr" href="#ref-91">Sweet <em>et al.</em> 2012</a>; <a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Pelechano <em>et al.</em> 2015</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Preissler <em>et al.</em> 2015</a>), suggesting that RNAs undergoing translation can be subjected to decay.</p><p id="p-61">We found that decay factor recruitment negatively correlated with ribosome density. If deadenylation mostly occurred on polyribosomes, it would be expected that the recruitment of the major initiator of decay, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, would show a positive correlation with ribosome density. This was not the case. Furthermore, the mRNA targets of the decay factors analyzed here correlated with those targets of multiple proteins involved in decay and translational repression, and not those that promote translation. The relationship between decay factor recruitment and ribosome occupancy described here, and the correlations between the deadenylase and other repressors of translation, supports a model that deadenylation and translation of transcripts are in competition. It is important to note that our results do not eliminate the existence of cotranslational decay pathways. Additional studies are required to fully understand the relationship between translation and decay.</p></div><div id="sec-24" class="subsection"><h3>Targeting of Ccr4 and Dhh1 to nutrient-regulated transcripts suggests that mRNA decay and/or translational repression are important for cells to respond to nutrient availability and environmental conditions</h3><p id="p-62"><em><a href="http://www.yeastgenome.org/locus/S000000019/overview">CCR4</a></em> was initially identified as a regulator of <em><a href="http://www.yeastgenome.org/locus/S000004918/overview">ADH2</a></em> and other nonfermentative growth genes (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Denis 1984</a>). This discovery provided the first evidence that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> plays an important role in the cell’s ability to adjust to carbon sources. Genetic studies have revealed that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> are required for cells to survive nutrient starvation (<a id="xref-ref-101-1" class="xref-bibr" href="#ref-101">Westmoreland <em>et al.</em> 2003</a>, <a id="xref-ref-102-1" class="xref-bibr" href="#ref-102">2004</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bergkessel and Reese 2004</a>), and the <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> homolog from <em>Schizosaccharomyces pombe</em>, ste13+, regulates N<sub>2</sub> starvation-induced entry into meiosis (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Maekawa <em>et al.</em> 1994</a>). Furthermore, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> have been tied to the TOR pathway (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Talarek <em>et al.</em> 2010</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Laribee <em>et al.</em> 2015</a>). Thus, there is previous genetic evidence that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not is an important regulator of the nutrient response. Our work has added to these observations and provided additional insights into how <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> regulate gene expression in response to nutrient levels. <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> associate with mRNAs that are normally repressed in rich media and induced by nutrient starvation. Significantly, these mRNAs are under the tightest control of these factors at the level of decay (<em>i.e.</em>, highly enriched and stabilized by deletion of the gene) (<a id="xref-fig-7-9" class="xref-fig" href="#F7">Figure 7, C and D</a>). This suggests a logical model whereby <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> are recruited to these mRNAs when nutrients are rich to repress their expression by mediating the decay of these messages. However, as stated in other parts of this manuscript, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not regulates gene expression at the level of transcription, binds transcription factors, and is recruited to active genes. Therefore, its functions in the nutritional response are unlikely to be restricted to mRNA decay. However, analysis presented here, and that of others, presents a solid case that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> are important for the cell to control nutrient-regulated genes, and that this involves regulation of the decay of the messages.</p><p id="p-63">Typically, experiments are carried out under exaggerated conditions (<em>e.g.</em>, nutrient-rich <em>vs.</em> complete starvation), comparing two “static” conditions to amplify effects. Although contrasting nutrient-rich and -poor conditions is useful, understanding how yeast behave during the YMC has been suggested as a better way to capture the reprograming of metabolic pathways (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Cai and Tu 2012</a>). Thus, the more interesting result of our mapping study is the uncovering of the previously unknown relationship between <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> recruitment and the fluctuations of mRNAs during the YMC. We found that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> are highly recruited to mRNAs that are repressed as cells move from the nonrespiratory phase into the respiratory phase (increasing O<sub>2</sub> consumption). As mentioned in other sections of this manuscript, <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not is targeted to specific mRNAs via RBPs. The <a href="http://www.yeastgenome.org/locus/S000003936/overview">Puf3</a> motif is overrepresented in the <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-bound mRNAs (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>) and also in those that fluctuate during the yeast YMC (<a id="xref-ref-96-4" class="xref-bibr" href="#ref-96">Tu <em>et al.</em> 2005</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Lee and Tu 2015</a>). <a href="http://www.yeastgenome.org/locus/S000003936/overview">Puf3</a> immunoprecipitates with <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, suggesting that a physical interaction between <a href="http://www.yeastgenome.org/locus/S000003936/overview">Puf3</a> and <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not is important for deadenylase recruitment (<a id="xref-ref-58-3" class="xref-bibr" href="#ref-58">Lee <em>et al.</em> 2010</a>). Although <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> are bound to messages undergoing nutrient control, levels of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>, <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a>, or <a href="http://www.yeastgenome.org/locus/S000003146/overview">Puf5</a> proteins do not change under nutrient starvation conditions or when cells are grown in nonfermentable carbon sources (Figure S14A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>). Whether nutrient signals affect the recruitment of <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not to mRNAs or regulate its activity remains to be tested. <a href="http://www.yeastgenome.org/locus/S000000689/overview">Not1</a>, <a href="http://www.yeastgenome.org/locus/S000000870/overview">Not4</a>, and <a href="http://www.yeastgenome.org/locus/S000005335/overview">Caf1</a> are phosphoproteins, and <a href="http://www.yeastgenome.org/locus/S000000870/overview">Not4</a> becomes phosphorylated in stressed cells (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Lau <em>et al.</em> 2010</a>) and <a href="http://www.yeastgenome.org/locus/S000005335/overview">Caf1</a> undergoes phosphorylation in carbon-starved cells (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Moriya <em>et al.</em> 2001</a>). We noticed that the mobility of <a href="http://www.yeastgenome.org/locus/S000000870/overview">Not4</a>, a subunit of the complex, changed when cells were starved for carbon or grown in nonfermentable carbon sources (Figure S14B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1/FileS5.pdf">File S5</a>). <a href="http://www.yeastgenome.org/locus/S000003936/overview">Puf3p</a> phosphorylation during the YMC and glucose starvation correlated with increased accumulation and decreased turnover of <a href="https://www.yeastgenome.org/locus/S000003936">Puf3</a>-targeted mRNAs (<a id="xref-ref-96-5" class="xref-bibr" href="#ref-96">Tu <em>et al.</em> 2005</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Miller <em>et al.</em> 2014</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Lee and Tu 2015</a>). Taken together, these observations raise the intriguing possibility that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a>-Not, and/or the proteins that recruit it to messages, are regulated by the nutritional status of the cell. Testing this hypothesis is worthy of comprehensive future studies, which are well beyond the scope of the work reported here.</p><p id="p-64">The association of decay factors with specific messages in response to metabolic fluxes may be required to rapidly “clear” the cell of these mRNAs, allowing the cell to transition between phases of fermentation and respiration. In addition, recruitment of decay factors could sharpen peaks of gene expression to impart tighter temporal control over the YMC, similar to how changes in the decay and synthesis rates of cell cycle-regulated messages are important during the mitotic cell cycle (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Eser <em>et al.</em> 2014</a>). Previous genetic studies indicate that <a href="http://www.yeastgenome.org/locus/S000000019/overview">Ccr4</a> and <a href="http://www.yeastgenome.org/locus/S000002319/overview">Dhh1</a> are required for cells to survive stresses that affect the cell cycle (<a id="xref-ref-101-2" class="xref-bibr" href="#ref-101">Westmoreland <em>et al.</em> 2003</a>, <a id="xref-ref-102-2" class="xref-bibr" href="#ref-102">2004</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bergkessel and Reese 2004</a>). Thus, the regulated targeting of decay factors to transcripts is likely to be important for cells to respond to and recover from stressful conditions.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-65">We thank the members of the Center for Eukaryotic Gene Regulation and the Center for RNA Molecular Biology for advice and support. SOLiD sequencing was performed at the Penn State Huck Institutes of the Life Science Genomics Core Facility at University Park. We acknowledge Craig Praul for advice on library preparation and DNA sequencing, Zhenhai Zhang for sharing a computational pipeline that the read alignment and composite plot code would be based upon, Björn Schwalb and Patrick Cramer for providing cDTA data, and Benjamin Tu for providing processed YMC data. This work was supported by National Institutes of Health grants GM-58672 (J.C.R.) and ES-013768 (B.F.P.).</p><p id="p-66">Author contributions: J.E.M. and J.C.R. conceived the study and cowrote the manuscript. J.E.M. and H.J. conducted the experiments. J.E.M. analyzed the data. Y.L., L.Z., and B.F.P. conducted read mapping and construction of part of the bioinformatics pipeline.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-67">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300415/-/DC1</a>.</p></li><li class="fn" id="fn-5"><p id="p-68"><em>Communicating editor: B. Gregory</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-08-03"><span class="received-label">Received </span>August 3, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-11-15"><span class="accepted-label">Accepted </span>November 15, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Miller <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.1" data-doi="10.1261/rna.054742.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alhusaini</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coller</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The deadenylase components Not2p, Not3p, and Not5p promote mRNA decapping.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">709</span>–<span class="cit-lpage">721</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.054742.115%26rft_id%253Dinfo%253Apmid%252F26952104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjIyLzUvNzA5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.2" data-doi="10.1016/j.molcel.2011.09.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arribere</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doudna</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">W. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Reconsidering movement of eukaryotic mRNAs between polysomes and P bodies.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">758</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DArribere%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D44%26rft.issue%253D5%26rft.spage%253D745%26rft.epage%253D758%26rft.atitle%253DReconsidering%2Bmovement%2Bof%2Beukaryotic%2BmRNAs%2Bbetween%2Bpolysomes%2Band%2BP%2Bbodies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2011.09.019%26rft_id%253Dinfo%253Apmid%252F22152478%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2011.09.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22152478&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000297962600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.3" data-doi="10.1261/rna.1348209"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Azzouz</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panasenko</span>  <span class="cit-name-given-names">O. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deluen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hsieh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theiler</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Specific roles for the Ccr4-Not complex subunits in expression of the genome.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.1348209%26rft_id%253Dinfo%253Apmid%252F19155328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjE1LzMvMzc3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.4" data-doi="10.1074/jbc.C114.601088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Babbarwal</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reese</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Rpb4/7 module of RNA polymerase II is required for carbon catabolite repressor protein 4-negative on TATA (Ccr4-not) complex to promote elongation.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">33125</span>–<span class="cit-lpage">33130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.C114.601088%26rft_id%253Dinfo%253Apmid%252F25315781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODkvNDgvMzMxMjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.5" data-doi="10.1016/j.molcel.2012.08.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Basquin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roudko</span>  <span class="cit-name-given-names">V. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rode</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basquin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seraphin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Architecture of the nuclease module of the yeast Ccr4-not complex: the Not1-Caf1-Ccr4 interaction.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DBasquin%26rft.auinit1%253DJ.%26rft.volume%253D48%26rft.issue%253D2%26rft.spage%253D207%26rft.epage%253D218%26rft.atitle%253DArchitecture%2Bof%2Bthe%2Bnuclease%2Bmodule%2Bof%2Bthe%2Byeast%2Bccr4-not%2Bcomplex%253A%2Bthe%2Bnot1-caf1-ccr4%2Binteraction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.08.014%26rft_id%253Dinfo%253Apmid%252F22959269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.08.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22959269&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310423800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benjamini</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hochberg</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Controlling the false discovery rate - a practical and powerful approach to multiple testing.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D57%26rft.spage%253D289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.7" data-doi="10.1534/genetics.167.1.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergkessel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reese</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">An essential role for the Saccharomyces cerevisiae DEAD-box helicase DHH1 in G1/S DNA-damage checkpoint recovery.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">21</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBergkessel%26rft.auinit1%253DM.%26rft.volume%253D167%26rft.issue%253D1%26rft.spage%253D21%26rft.epage%253D33%26rft.atitle%253DAn%2BEssential%2BRole%2Bfor%2Bthe%2BSaccharomyces%2Bcerevisiae%2BDEAD-Box%2BHelicase%2BDHH1%2Bin%2BG1%252FS%2BDNA-Damage%2BCheckpoint%2BRecovery%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.167.1.21%26rft_id%253Dinfo%253Apmid%252F15166134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTY3LzEvMjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.8" data-doi="10.1093/bioinformatics/bth456"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">E. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gollub</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">GO:TermFinder–open source software for accessing Gene Ontology information and finding significantly enriched Gene Ontology terms associated with a list of genes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3710</span>–<span class="cit-lpage">3715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth456%26rft_id%253Dinfo%253Apmid%252F15297299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bth456&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15297299&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225786600064&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.9" data-doi="10.1371/journal.pcbi.1000270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradley</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brauer</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabinowitz</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troyanskaya</span>  <span class="cit-name-given-names">O. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Coordinated concentration changes of transcripts and metabolites in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">PLoS Comput. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DBradley%26rft.auinit1%253DP.%2BH.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253De1000270%26rft.epage%253De1000270%26rft.atitle%253DCoordinated%2Bconcentration%2Bchanges%2Bof%2Btranscripts%2Band%2Bmetabolites%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000270%26rft_id%253Dinfo%253Apmid%252F19180179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19180179&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sachs</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Poly(A) tail length control in Saccharomyces cerevisiae occurs by message-specific deadenylation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">6548</span>–<span class="cit-lpage">6559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBrown%26rft.auinit1%253DC.%2BE.%26rft.volume%253D18%26rft.issue%253D11%26rft.spage%253D6548%26rft.epage%253D6559%26rft.atitle%253DPoly%2528A%2529%2BTail%2BLength%2BControl%2Bin%2BSaccharomyces%2Bcerevisiae%2BOccurs%2Bby%2BMessage-Specific%2BDeadenylation%26rft_id%253Dinfo%253Apmid%252F9774670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS82NTQ4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.11" data-doi="10.1128/MCB.16.10.5744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarun</span>  <span class="cit-name-given-names">S. Z.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boeck</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sachs</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">PAN3 encodes a subunit of the Pab1p-dependent poly(A) nuclease in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">5744</span>–<span class="cit-lpage">5753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBrown%26rft.auinit1%253DC.%2BE.%26rft.volume%253D16%26rft.issue%253D10%26rft.spage%253D5744%26rft.epage%253D5753%26rft.atitle%253DPAN3%2Bencodes%2Ba%2Bsubunit%2Bof%2Bthe%2BPab1p-dependent%2Bpoly%2528A%2529%2Bnuclease%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.16.10.5744%26rft_id%253Dinfo%253Apmid%252F8816488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIxNi8xMC81NzQ0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.12" data-doi="10.1146/annurev-cellbio-092910-154010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tu</span>  <span class="cit-name-given-names">B. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Driving the cell cycle through metabolism.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Cell Dev. Biol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bcell%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BCell%2BDev%2BBiol%26rft.aulast%253DCai%26rft.auinit1%253DL.%26rft.volume%253D28%26rft.spage%253D59%26rft.epage%253D87%26rft.atitle%253DDriving%2Bthe%2Bcell%2Bcycle%2Bthrough%2Bmetabolism.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-cellbio-092910-154010%26rft_id%253Dinfo%253Apmid%252F22578140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-cellbio-092910-154010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22578140&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cary</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinh</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuestner</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Proteomic analysis of Dhh1 complexes reveals a role for Hsp40 Chaperone Ydj1 in yeast P-body assembly.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2497</span>–<span class="cit-lpage">2511</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D5%26rft.spage%253D2497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.14" data-doi="10.1093/emboj/21.6.1414"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denis</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">CCR4, a 3′-5′ poly(A) RNA and ssDNA exonuclease, is the catalytic component of the cytoplasmic deadenylase.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1414</span>–<span class="cit-lpage">1426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DChen%26rft.auinit1%253DJ.%26rft.volume%253D21%26rft.issue%253D6%26rft.spage%253D1414%26rft.epage%253D1426%26rft.atitle%253DCCR4%252C%2Ba%2B3%2527-5%2527%2Bpoly%2528A%2529%2BRNA%2Band%2BssDNA%2Bexonuclease%252C%2Bis%2Bthe%2Bcatalytic%2Bcomponent%2Bof%2Bthe%2Bcytoplasmic%2Bdeadenylase%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F21.6.1414%26rft_id%253Dinfo%253Apmid%252F11889047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIyMS82LzE0MTQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.15" data-doi="10.1016/j.molcel.2014.03.034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boland</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuzuoglu-Ozturk</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bawankar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loh</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A DDX6-CNOT1 complex and W-binding pockets in CNOT9 reveal direct links between miRNA target recognition and silencing.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">750</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D54%26rft.spage%253D737%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2014.03.034%26rft_id%253Dinfo%253Apmid%252F24768540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2014.03.034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24768540&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340626100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.16" data-doi="10.1261/rna.2920905"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Crystal structure and functional analysis of DEAD-box protein Dhh1p.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1258</span>–<span class="cit-lpage">1270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2920905%26rft_id%253Dinfo%253Apmid%252F15987810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjExLzgvMTI1OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMS8zMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.17" data-doi="10.1261/rna.2070110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chritton</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wickens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Translational repression by PUF proteins in vitro.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1217</span>–<span class="cit-lpage">1225</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2070110%26rft_id%253Dinfo%253Apmid%252F20427513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjE2LzYvMTIxNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMS8zMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.18" data-doi="10.1016/S0378-1119(03)00672-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collart</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global control of gene expression in yeast by the Ccr4-Not complex.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">313</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DCollart%26rft.auinit1%253DM.%2BA.%26rft.volume%253D313%26rft.spage%253D1%26rft.epage%253D16%26rft.atitle%253DGlobal%2Bcontrol%2Bof%2Bgene%2Bexpression%2Bin%2Byeast%2Bby%2Bthe%2BCcr4-Not%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252803%252900672-3%26rft_id%253Dinfo%253Apmid%252F12957374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(03)00672-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12957374&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185443300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collart</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The Ccr4-Not complex is a key regulator of eukaryotic gene expression.</span> <abbr class="cit-jnl-abbrev">Wiley Interdiscip. Rev. RNA</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdiscip.%2BRev.%2BRNA%26rft.volume%253D7%26rft.spage%253D438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.20" data-doi="10.1016/j.cell.2005.07.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">General translational repression by activators of mRNA decapping.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">875</span>–<span class="cit-lpage">886</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DColler%26rft.auinit1%253DJ.%26rft.volume%253D122%26rft.issue%253D6%26rft.spage%253D875%26rft.epage%253D886%26rft.atitle%253DGeneral%2Btranslational%2Brepression%2Bby%2Bactivators%2Bof%2BmRNA%2Bdecapping.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2005.07.012%26rft_id%253Dinfo%253Apmid%252F16179257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2005.07.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16179257&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232130200012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.21" data-doi="10.1017/S135583820101994X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coller</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheth</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valencia-Sanchez</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The DEAD box helicase, Dhh1p, functions in mRNA decapping and interacts with both the decapping and deadenylase complexes.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1717</span>–<span class="cit-lpage">1727</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft.stitle%253DRNA%26rft.aulast%253DColler%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253D1717%26rft.epage%253D1727%26rft.atitle%253DThe%2BDEAD%2Bbox%2Bhelicase%252C%2BDhh1p%252C%2Bfunctions%2Bin%2BmRNA%2Bdecapping%2Band%2Binteracts%2Bwith%2Bboth%2Bthe%2Bdecapping%2Band%2Bdeadenylase%2Bcomplexes%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS135583820101994X%26rft_id%253Dinfo%253Apmid%252F11780629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjcvMTIvMTcxNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMS8zMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.22" data-doi="10.1007/s00438-007-0314-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cui</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramnarain</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMahon</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome wide expression analysis of the CCR4-NOT complex indicates that it consists of three modules with the NOT module controlling SAGA-responsive genes.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">279</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DCui%26rft.auinit1%253DY.%26rft.volume%253D279%26rft.issue%253D4%26rft.spage%253D323%26rft.epage%253D337%26rft.atitle%253DGenome%2Bwide%2Bexpression%2Banalysis%2Bof%2Bthe%2BCCR4-NOT%2Bcomplex%2Bindicates%2Bthat%2Bit%2Bconsists%2Bof%2Bthree%2Bmodules%2Bwith%2Bthe%2BNOT%2Bmodule%2Bcontrolling%2BSAGA-responsive%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-007-0314-1%26rft_id%253Dinfo%253Apmid%252F18214544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-007-0314-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18214544&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.23" data-doi="10.1073/pnas.0601091103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">David</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Granovskaia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toedling</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palm</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A high-resolution map of transcription in the yeast genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">5320</span>–<span class="cit-lpage">5325</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0601091103%26rft_id%253Dinfo%253Apmid%252F16569694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzE0LzUzMjAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.24" data-doi="10.1128/MCB.22.19.6735-6749.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deluen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maillet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molinete</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Theiler</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The Ccr4-not complex and yTAF1 (yTaf(II)130p/yTaf(II)145p) show physical and functional interactions.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">6735</span>–<span class="cit-lpage">6749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DDeluen%26rft.auinit1%253DC.%26rft.volume%253D22%26rft.issue%253D19%26rft.spage%253D6735%26rft.epage%253D6749%26rft.atitle%253DThe%2BCcr4-Not%2BComplex%2Band%2ByTAF1%2B%2528yTafII130p%252FyTafII145p%2529%2BShow%2BPhysical%2Band%2BFunctional%2BInteractions%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.22.19.6735-6749.2002%26rft_id%253Dinfo%253Apmid%252F12215531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xOS82NzM1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denis</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Identification of new genes involved in the regulation of yeast alcohol dehydrogenase II.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">833</span>–<span class="cit-lpage">844</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDenis%26rft.auinit1%253DC.%2BL.%26rft.volume%253D108%26rft.issue%253D4%26rft.spage%253D833%26rft.epage%253D844%26rft.atitle%253DIDENTIFICATION%2BOF%2BNEW%2BGENES%2BINVOLVED%2BIN%2BTHE%2BREGULATION%2BOF%2BYEAST%2BALCOHOL%2BDEHYDROGENASE%2BII%26rft_id%253Dinfo%253Apmid%252F6392016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA4LzQvODMzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.26" data-doi="10.1074/jbc.M111.220251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dutta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cameron</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reese</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Intermolecular interactions within the abundant DEAD-box protein Dhh1 regulate its activity in vivo.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">27454</span>–<span class="cit-lpage">27470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M111.220251%26rft_id%253Dinfo%253Apmid%252F21642421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODYvMzEvMjc0NTQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.27" data-doi="10.1128/MCB.00044-15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dutta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babbarwal</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunke-Reese</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Libert</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ccr4-Not and TFIIS function cooperatively to rescue arrested RNA polymerase II.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">1915</span>–<span class="cit-lpage">1925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.00044-15%26rft_id%253Dinfo%253Apmid%252F25776559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIzNS8xMS8xOTE1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.28" data-doi="10.1016/j.molcel.2007.09.027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elemento</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slonim</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavazoie</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A universal framework for regulatory element discovery across all genomes and data types.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DElemento%26rft.auinit1%253DO.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D337%26rft.epage%253D350%26rft.atitle%253DA%2Buniversal%2Bframework%2Bfor%2Bregulatory%2Belement%2Bdiscovery%2Bacross%2Ball%2Bgenomes%2Band%2Bdata%2Btypes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2007.09.027%26rft_id%253Dinfo%253Apmid%252F17964271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2007.09.027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17964271&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250739800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.29" data-doi="10.1002/msb.134886"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eser</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maier</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwalb</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pirkl</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Periodic mRNA synthesis and degradation co-operate during cell cycle gene expression.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">717</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DEser%26rft.auinit1%253DP.%26rft.volume%253D10%26rft.issue%253D1%26rft.spage%253D717%26rft.epage%253D717%26rft.atitle%253DPeriodic%2BmRNA%2Bsynthesis%2Band%2Bdegradation%2Bco-operate%2Bduring%2Bcell%2Bcycle%2Bgene%2Bexpression%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fmsb.134886%26rft_id%253Dinfo%253Apmid%252F24489117%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjg6IjEwLzEvNzE3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.30" data-doi="10.1038/nrm2080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eulalio</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Behm-Ansmant</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Izaurralde</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">P bodies: at the crossroads of post-transcriptional pathways.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DEulalio%26rft.auinit1%253DA.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D22%26rft.atitle%253DP%2Bbodies%253A%2Bat%2Bthe%2Bcrossroads%2Bof%2Bpost-transcriptional%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2080%26rft_id%253Dinfo%253Apmid%252F17183357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm2080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17183357&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243005100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Faraji</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winkler</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Post-transcriptional control of tumor cell autonomous metastatic potential by CCR4-NOT deadenylase CNOT7.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">e1005820</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D12%26rft.spage%253De1005820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.32" data-doi="10.1093/emboj/21.11.2788"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weis</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The DEAD box protein Dhh1 stimulates the decapping enzyme Dcp1.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2788</span>–<span class="cit-lpage">2797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DFischer%26rft.auinit1%253DN.%26rft.volume%253D21%26rft.issue%253D11%26rft.spage%253D2788%26rft.epage%253D2797%26rft.atitle%253DThe%2BDEAD%2Bbox%2Bprotein%2BDhh1%2Bstimulates%2Bthe%2Bdecapping%2Benzyme%2BDcp1%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F21.11.2788%26rft_id%253Dinfo%253Apmid%252F12032091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMjEvMTEvMjc4OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMS8zMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.33" data-doi="10.1091/mbc.11.12.4241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gasch</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spellman</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmel-Harel</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genomic expression programs in the response of yeast cells to environmental changes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">4241</span>–<span class="cit-lpage">4257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DGasch%26rft.auinit1%253DA.%2BP.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D4241%26rft.epage%253D4257%26rft.atitle%253DGenomic%2BExpression%2BPrograms%2Bin%2Bthe%2BResponse%2Bof%2BYeast%2BCells%2Bto%2BEnvironmental%2BChanges%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.11.12.4241%26rft_id%253Dinfo%253Apmid%252F11102521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMi80MjQxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.34" data-doi="10.1073/pnas.1120799109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerashchenko</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobanov</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gladyshev</span>  <span class="cit-name-given-names">V. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide ribosome profiling reveals complex translational regulation in response to oxidative stress.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">17394</span>–<span class="cit-lpage">17399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1120799109%26rft_id%253Dinfo%253Apmid%252F23045643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQzLzE3Mzk0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.35" data-doi="10.1371/journal.pbio.0020079"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerber</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herschlag</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Extensive association of functionally and cytotopically related mRNAs with Puf family RNA-binding proteins in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">E79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DGerber%26rft.auinit1%253DA.%2BP.%26rft.volume%253D2%26rft.issue%253D3%26rft.spage%253DE79%26rft.epage%253DE79%26rft.atitle%253DExtensive%2Bassociation%2Bof%2Bfunctionally%2Band%2Bcytotopically%2Brelated%2BmRNAs%2Bwith%2BPuf%2Bfamily%2BRNA-binding%2Bproteins%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0020079%26rft_id%253Dinfo%253Apmid%252F15024427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0020079&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15024427&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.36" data-doi="10.1038/nature02046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huh</span>  <span class="cit-name-given-names">W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Global analysis of protein expression in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">741</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGhaemmaghami%26rft.auinit1%253DS.%26rft.volume%253D425%26rft.issue%253D6959%26rft.spage%253D737%26rft.epage%253D741%26rft.atitle%253DGlobal%2Banalysis%2Bof%2Bprotein%2Bexpression%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02046%26rft_id%253Dinfo%253Apmid%252F14562106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14562106&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185924500046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.37" data-doi="10.1038/nrm2370"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldstrohm</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wickens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Multifunctional deadenylase complexes diversify mRNA control.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DGoldstrohm%26rft.auinit1%253DA.%2BC.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D337%26rft.epage%253D344%26rft.atitle%253DMultifunctional%2Bdeadenylase%2Bcomplexes%2Bdiversify%2BmRNA%2Bcontrol.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2370%26rft_id%253Dinfo%253Apmid%252F18334997%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm2370&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18334997&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254274300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.38" data-doi="10.1038/nsmb1100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldstrohm</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hook</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seay</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wickens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">PUF proteins bind Pop2p to regulate messenger RNAs.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">533</span>–<span class="cit-lpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bstructural%2B%2526%2Bmolecular%2Bbiology%26rft.stitle%253DNat%2BStruct%2BMol%2BBiol%26rft.aulast%253DGoldstrohm%26rft.auinit1%253DA.%2BC.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D533%26rft.epage%253D539%26rft.atitle%253DPUF%2Bproteins%2Bbind%2BPop2p%2Bto%2Bregulate%2Bmessenger%2BRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb1100%26rft_id%253Dinfo%253Apmid%252F16715093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb1100&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16715093&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238122500018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.39" data-doi="10.1074/jbc.M609413200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldstrohm</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seay</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hook</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wickens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PUF protein-mediated deadenylation is catalyzed by Ccr4p.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M609413200%26rft_id%253Dinfo%253Apmid%252F17090538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjk6IjI4Mi8xLzEwOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMS8zMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.40" data-doi="10.1128/MCB.24.12.5534-5547.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grigull</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mnaimneh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pootoolal</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genome-wide analysis of mRNA stability using transcription inhibitors and microarrays reveals posttranscriptional control of ribosome biogenesis factors.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">5534</span>–<span class="cit-lpage">5547</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DGrigull%26rft.auinit1%253DJ.%26rft.volume%253D24%26rft.issue%253D12%26rft.spage%253D5534%26rft.epage%253D5547%26rft.atitle%253DGenome-Wide%2BAnalysis%2Bof%2BmRNA%2BStability%2BUsing%2BTranscription%2BInhibitors%2Band%2BMicroarrays%2BReveals%2BPosttranscriptional%2BControl%2Bof%2BRibosome%2BBiogenesis%2BFactors%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.12.5534-5547.2004%26rft_id%253Dinfo%253Apmid%252F15169913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMi81NTM0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanyi</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassem</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Panasenko</span>  <span class="cit-name-given-names">O. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Translational capacity of a cell is determined during transcription elongation via the Ccr4-Not complex.</span> <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1782</span>–<span class="cit-lpage">1794</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D15%26rft.spage%253D1782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.42" data-doi="10.1016/j.bbagrm.2013.01.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haimovich</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singer</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trcek</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">The fate of the messenger is pre-determined: a new model for regulation of gene expression.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1829</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">653</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim.%2BBiophys.%2BActa%26rft.volume%253D1829%26rft.spage%253D643%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbagrm.2013.01.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbagrm.2013.01.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.43" data-doi="10.1016/j.cell.2013.05.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haimovich</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Medina</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Causse</span>  <span class="cit-name-given-names">S. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garber</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millan-Zambrano</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Gene expression is circular: factors for mRNA degradation also foster mRNA synthesis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">1000</span>–<span class="cit-lpage">1011</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D153%26rft.spage%253D1000%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.05.012%26rft_id%253Dinfo%253Apmid%252F23706738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.05.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23706738&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319456800009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitsui</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denis</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Dhh1p, a putative RNA helicase, associates with the general transcription factors Pop2p and Ccr4p from Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">571</span>–<span class="cit-lpage">579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHata%26rft.auinit1%253DH.%26rft.volume%253D148%26rft.issue%253D2%26rft.spage%253D571%26rft.epage%253D580%26rft.atitle%253DDhh1p%252C%2Ba%2BPutative%2BRNA%2BHelicase%252C%2BAssociates%2Bwith%2Bthe%2BGeneral%2BTranscription%2BFactors%2BPop2p%2Band%2BCcr4p%2Bfrom%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F9504907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ4LzIvNTcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.45" data-doi="10.1093/emboj/21.7.1800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hector</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nykamp</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dheur</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Non</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Dual requirement for yeast hnRNP Nab2p in mRNA poly(A) tail length control and nuclear export.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1800</span>–<span class="cit-lpage">1810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DHector%26rft.auinit1%253DR.%2BE.%26rft.volume%253D21%26rft.issue%253D7%26rft.spage%253D1800%26rft.epage%253D1810%26rft.atitle%253DDual%2Brequirement%2Bfor%2Byeast%2BhnRNP%2BNab2p%2Bin%2BmRNA%2Bpoly%2528A%2529%2Btail%2Blength%2Bcontrol%2Band%2Bnuclear%2Bexport%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F21.7.1800%26rft_id%253Dinfo%253Apmid%252F11927564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIyMS83LzE4MDAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.46" data-doi="10.1016/j.molcel.2010.05.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heinz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benner</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spann</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bertolino</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">576</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DHeinz%26rft.auinit1%253DS.%26rft.volume%253D38%26rft.issue%253D4%26rft.spage%253D576%26rft.epage%253D589%26rft.atitle%253DSimple%2Bcombinations%2Bof%2Blineage-determining%2Btranscription%2Bfactors%2Bprime%2Bcis-regulatory%2Belements%2Brequired%2Bfor%2Bmacrophage%2Band%2BB%2Bcell%2Bidentities.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.05.004%26rft_id%253Dinfo%253Apmid%252F20513432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.05.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20513432&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278448100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.47" data-doi="10.1371/journal.pbio.0060255"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hogan</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riordan</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerber</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herschlag</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Diverse RNA-binding proteins interact with functionally related sets of RNAs, suggesting an extensive regulatory system.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DHogan%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D6%26rft.issue%253D10%26rft.spage%253De255%26rft.epage%253De255%26rft.atitle%253DDiverse%2BRNA-binding%2Bproteins%2Binteract%2Bwith%2Bfunctionally%2Brelated%2Bsets%2Bof%2BRNAs%252C%2Bsuggesting%2Ban%2Bextensive%2Bregulatory%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060255%26rft_id%253Dinfo%253Apmid%252F18959479%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060255&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18959479&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.48" data-doi="10.1074/jbc.M611253200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hook</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstrohm</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seay</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wickens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Two yeast PUF proteins negatively regulate a single mRNA.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">15430</span>–<span class="cit-lpage">15438</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M611253200%26rft_id%253Dinfo%253Apmid%252F17389596%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODIvMjEvMTU0MzAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.49" data-doi="10.1038/nature08265"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sweet</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamnongpol</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coller</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Co-translational mRNA decay in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHu%26rft.auinit1%253DW.%26rft.volume%253D461%26rft.issue%253D7261%26rft.spage%253D225%26rft.epage%253D229%26rft.atitle%253DCo-translational%2BmRNA%2Bdecay%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08265%26rft_id%253Dinfo%253Apmid%252F19701183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08265&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19701183&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269654600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.50" data-doi="10.1126/science.1168978"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingolia</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghaemmaghami</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newman</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">218</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1168978%26rft_id%253Dinfo%253Apmid%252F19213877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNC8yMTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.51" data-doi="10.1128/EC.05181-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ito</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irie</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mizuno</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irie</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RNA-binding protein Khd1 and Ccr4 deadenylase play overlapping roles in the cell wall integrity pathway in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1340</span>–<span class="cit-lpage">1347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.05181-11%26rft_id%253Dinfo%253Apmid%252F21873511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czoxMDoiMTAvMTAvMTM0MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMS8zMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.52" data-doi="10.1038/nrg2111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keene</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">RNA regulons: coordination of post-transcriptional events.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">533</span>–<span class="cit-lpage">543</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DKeene%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253D533%26rft.epage%253D543%26rft.atitle%253DRNA%2Bregulons%253A%2Bcoordination%2Bof%2Bpost-transcriptional%2Bevents.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2111%26rft_id%253Dinfo%253Apmid%252F17572691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2111&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17572691&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247344800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.53" data-doi="10.1371/journal.pone.0018302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kerr</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Azzouz</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchs</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collart</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strahl</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The Ccr4-Not complex interacts with the mRNA export machinery.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e18302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DKerr%26rft.auinit1%253DS.%2BC.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253De18302%26rft.epage%253De18302%26rft.atitle%253DThe%2BCcr4-Not%2Bcomplex%2Binteracts%2Bwith%2Bthe%2BmRNA%2Bexport%2Bmachinery.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0018302%26rft_id%253Dinfo%253Apmid%252F21464899%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0018302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21464899&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.54" data-doi="10.1101/gad.2020911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kruk</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dutta</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reese</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The multifunctional Ccr4-Not complex directly promotes transcription elongation.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">581</span>–<span class="cit-lpage">593</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DKruk%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D25%26rft.issue%253D6%26rft.spage%253D581%26rft.epage%253D593%26rft.atitle%253DThe%2Bmultifunctional%2BCcr4-Not%2Bcomplex%2Bdirectly%2Bpromotes%2Btranscription%2Belongation%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.2020911%26rft_id%253Dinfo%253Apmid%252F21406554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMjUvNi81ODEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.55" data-doi="10.1371/journal.pgen.1005113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laribee</span>  <span class="cit-name-given-names">R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hosni-Ahmed</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Workman</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ccr4-not regulates RNA polymerase I transcription and couples nutrient signaling to the control of ribosomal RNA biogenesis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e1005113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005113%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005113%26rft_id%253Dinfo%253Apmid%252F25815716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25815716&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.56" data-doi="10.1371/journal.pone.0009864"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lau</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulder</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenkman</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohammed</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Broek</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Phosphorylation of Not4p functions parallel to BUR2 to regulate resistance to cellular stresses in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e9864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLau%26rft.auinit1%253DN.%2BC.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253De9864%26rft.epage%253De9864%26rft.atitle%253DPhosphorylation%2Bof%2BNot4p%2Bfunctions%2Bparallel%2Bto%2BBUR2%2Bto%2Bregulate%2Bresistance%2Bto%2Bcellular%2Bstresses%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0009864%26rft_id%253Dinfo%253Apmid%252F20386698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0009864&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20386698&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.57" data-doi="10.1016/j.celrep.2015.05.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tu</span>  <span class="cit-name-given-names">B. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Glucose-regulated phosphorylation of the PUF protein Puf3 regulates the translational fate of its bound mRNAs and association with RNA granules.</span> <abbr class="cit-jnl-abbrev">Cell Rep.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1638</span>–<span class="cit-lpage">1650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BRep.%26rft.volume%253D11%26rft.spage%253D1638%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2015.05.014%26rft_id%253Dinfo%253Apmid%252F26051939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2015.05.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26051939&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.58" data-doi="10.1016/j.jmb.2010.04.034"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohn</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quigley</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">PUF3 acceleration of deadenylation in vivo can operate independently of CCR4 activity, possibly involving effects on the PAB1-mRNP structure.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">399</span>: <span class="cit-fpage">562</span>–<span class="cit-lpage">575</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DLee%26rft.auinit1%253DD.%26rft.volume%253D399%26rft.issue%253D4%26rft.spage%253D562%26rft.epage%253D575%26rft.atitle%253DPUF3%2Bacceleration%2Bof%2Bdeadenylation%2Bin%2Bvivo%2Bcan%2Boperate%2Bindependently%2Bof%2BCCR4%2Bactivity%252C%2Bpossibly%2Binvolving%2Beffects%2Bon%2Bthe%2BPAB1-mRNP%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2010.04.034%26rft_id%253Dinfo%253Apmid%252F20435044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2010.04.034&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20435044&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.59" data-doi="10.1101/gad.10.10.1233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silver</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">A protein that shuttles between the nucleus and the cytoplasm is an important mediator of RNA export.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1233</span>–<span class="cit-lpage">1246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DLee%26rft.auinit1%253DM%2BS%26rft.volume%253D10%26rft.issue%253D10%26rft.spage%253D1233%26rft.epage%253D1246%26rft.atitle%253DA%2Bprotein%2Bthat%2Bshuttles%2Bbetween%2Bthe%2Bnucleus%2Band%2Bthe%2Bcytoplasm%2Bis%2Ban%2Bimportant%2Bmediator%2Bof%2BRNA%2Bexport.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.10.10.1233%26rft_id%253Dinfo%253Apmid%252F8675010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjEwLzEwLzEyMzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.60" data-doi="10.1016/j.tcb.2013.05.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lykke-Andersen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Emerging roles for ribonucleoprotein modification and remodeling in controlling RNA fate.</span> <abbr class="cit-jnl-abbrev">Trends Cell Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">504</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BCell%2BBiol.%26rft.volume%253D23%26rft.spage%253D504%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tcb.2013.05.001%26rft_id%253Dinfo%253Apmid%252F23756094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tcb.2013.05.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23756094&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Knockdown of RCK/p54 expression by RNAi inhibits proliferation of human colorectal cancer cells in vitro and in vivo.</span> <abbr class="cit-jnl-abbrev">Cancer Biol. Ther.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1669</span>–<span class="cit-lpage">1676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2Bbiology%2B%2526%2Btherapy%26rft.stitle%253DCancer%2BBiol%2BTher%26rft.aulast%253DLin%26rft.auinit1%253DF.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253D1669%26rft.epage%253D1676%26rft.atitle%253DKnockdown%2Bof%2BRCK%252Fp54%2Bexpression%2Bby%2BRNAi%2Binhibits%2Bproliferation%2Bof%2Bhuman%2Bcolorectal%2Bcancer%2Bcells%2Bin%2Bvitro%2Band%2Bin%2Bvivo.%26rft_id%253Dinfo%253Apmid%252F18769115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18769115&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.62" data-doi="10.1002/(SICI)1097-0061(199807)14:10&lt;953::AID-YEA293&gt;3.0.CO;2-U"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Longtine</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKenzie</span>  <span class="cit-name-given-names">A.</span>  <span class="cit-name-suffix">III</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demarini</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wach</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Additional modules for versatile and economical PCR-based gene deletion and modification in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">953</span>–<span class="cit-lpage">961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DLongtine%26rft.auinit1%253DM.%2BS.%26rft.volume%253D14%26rft.issue%253D10%26rft.spage%253D953%26rft.epage%253D961%26rft.atitle%253DAdditional%2Bmodules%2Bfor%2Bversatile%2Band%2Beconomical%2BPCR-based%2Bgene%2Bdeletion%2Band%2Bmodification%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F%2528SICI%25291097-0061%2528199807%252914%253A10%253C953%253A%253AAID-YEA293%253E3.0.CO%253B2-U%26rft_id%253Dinfo%253Apmid%252F9717241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/(SICI)1097-0061(199807)14:10&lt;953::AID-YEA293&gt;3.0.CO;2-U&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9717241&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075005100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.63" data-doi="10.1007/BF00583896"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maekawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakagawa</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uno</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimoda</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The ste13+ gene encoding a putative RNA helicase is essential for nitrogen starvation-induced G1 arrest and initiation of sexual development in the fission yeast Schizosaccharomyces pombe.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">244</span>: <span class="cit-fpage">456</span>–<span class="cit-lpage">464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DMaekawa%26rft.auinit1%253DH.%26rft.volume%253D244%26rft.issue%253D5%26rft.spage%253D456%26rft.epage%253D464%26rft.atitle%253DThe%2Bste13%252B%2Bgene%2Bencoding%2Ba%2Bputative%2BRNA%2Bhelicase%2Bis%2Bessential%2Bfor%2Bnitrogen%2Bstarvation-induced%2BG1%2Barrest%2Band%2Binitiation%2Bof%2Bsexual%2Bdevelopment%2Bin%2Bthe%2Bfission%2Byeast%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00583896%26rft_id%253Dinfo%253Apmid%252F8078473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00583896&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8078473&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PF27400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.64" data-doi="10.1074/jbc.M107979200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maillet</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collart</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Interaction between Not1p, a component of the Ccr4-not complex, a global regulator of transcription, and Dhh1p, a putative RNA helicase.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">2835</span>–<span class="cit-lpage">2842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M107979200%26rft_id%253Dinfo%253Apmid%252F11696541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzcvNC8yODM1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.65" data-doi="10.1074/jbc.M207571200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marfatia</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crafton</span>  <span class="cit-name-given-names">E. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Corbett</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Domain analysis of the Saccharomyces cerevisiae heterogeneous nuclear ribonucleoprotein, Nab2p. Dissecting the requirements for Nab2p-facilitated poly(A) RNA export.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">278</span>: <span class="cit-fpage">6731</span>–<span class="cit-lpage">6740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M207571200%26rft_id%253Dinfo%253Apmid%252F12496292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyNzgvOS82NzMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.66" data-doi="10.1016/j.molcel.2014.03.036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mathys</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basquin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozgur</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czarnocki-Cieciura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonneau</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Structural and biochemical insights to the role of the CCR4-NOT complex and DDX6 ATPase in microRNA repression.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">751</span>–<span class="cit-lpage">765</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D54%26rft.spage%253D751%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2014.03.036%26rft_id%253Dinfo%253Apmid%252F24768538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2014.03.036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24768538&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340626100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.67" data-doi="10.1371/journal.pone.0101717"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Micallef</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodgers</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">eulerAPE: drawing area-proportional 3-Venn diagrams using ellipses.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e101717</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De101717%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0101717%26rft_id%253Dinfo%253Apmid%252F25032825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0101717&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25032825&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.68" data-doi="10.3109/10409238.2012.667214"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reese</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Ccr4-Not complex: the control freak of eukaryotic cells.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Biochem. Mol. Biol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCritical%2BReviews%2Bin%2BBiochemistry%2Band%2BMolecular%2BBiology%26rft.stitle%253DCRIT%2BREV%2BBIOCHEM%2BMOL%2BBIOL%26rft.aulast%253DMiller%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D47%26rft.issue%253D4%26rft.spage%253D315%26rft.epage%253D333%26rft.atitle%253DCcr4-Not%2Bcomplex%253A%2Bthe%2Bcontrol%2Bfreak%2Bof%2Beukaryotic%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.3109%252F10409238.2012.667214%26rft_id%253Dinfo%253Apmid%252F22416820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3109/10409238.2012.667214&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22416820&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.69" data-doi="10.1093/nar/gkt1346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez Leban</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olivas</span>  <span class="cit-name-given-names">W. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Carbon source-dependent alteration of Puf3p activity mediates rapid changes in the stabilities of mRNAs involved in mitochondrial function.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">3954</span>–<span class="cit-lpage">3970</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1346%26rft_id%253Dinfo%253Apmid%252F24371272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24371272&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.70" data-doi="10.1038/nsmb.2468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">She</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Global analysis of yeast mRNPs.</span> <abbr class="cit-jnl-abbrev">Nat. Struct. Mol. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BStruct.%2BMol.%2BBiol.%26rft.volume%253D20%26rft.spage%253D127%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnsmb.2468%26rft_id%253Dinfo%253Apmid%252F23222640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nsmb.2468&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23222640&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.71" data-doi="10.1074/jbc.M800295200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Molina-Navarro</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Castells-Roca</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belli</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Martinez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marin-Navarro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Comprehensive transcriptional analysis of the oxidative response in yeast.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">283</span>: <span class="cit-fpage">17908</span>–<span class="cit-lpage">17918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M800295200%26rft_id%253Dinfo%253Apmid%252F18424442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODMvMjYvMTc5MDgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.72" data-doi="10.1101/gad.884001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moriya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimizu-Yoshida</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Omori</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwashita</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Yak1p, a DYRK family kinase, translocates to the nucleus and phosphorylates yeast Pop2p in response to a glucose signal.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1217</span>–<span class="cit-lpage">1228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DMoriya%26rft.auinit1%253DH.%26rft.volume%253D15%26rft.issue%253D10%26rft.spage%253D1217%26rft.epage%253D1228%26rft.atitle%253DYak1p%252C%2Ba%2BDYRK%2Bfamily%2Bkinase%252C%2Btranslocates%2Bto%2Bthe%2Bnucleus%2Band%2Bphosphorylates%2Byeast%2BPop2p%2Bin%2Bresponse%2Bto%2Ba%2Bglucose%2Bsignal%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.884001%26rft_id%253Dinfo%253Apmid%252F11358866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE1LzEwLzEyMTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.73" data-doi="10.1126/science.1158441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagalakshmi</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waern</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shou</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raha</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The transcriptional landscape of the yeast genome defined by RNA sequencing.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">1344</span>–<span class="cit-lpage">1349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1158441%26rft_id%253Dinfo%253Apmid%252F18451266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4MS8xMzQ0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.74" data-doi="10.1093/bioinformatics/bts277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neph</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuehn</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haugen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thurman</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">BEDOPS: high-performance genomic feature operations.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1919</span>–<span class="cit-lpage">1920</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts277%26rft_id%253Dinfo%253Apmid%252F22576172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts277&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22576172&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306136100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.75" data-doi="10.1016/j.molcel.2010.08.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nissan</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajyaguru</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">She</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Decapping activators in Saccharomyces cerevisiae act by multiple mechanisms.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">783</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DNissan%26rft.auinit1%253DT.%26rft.volume%253D39%26rft.issue%253D5%26rft.spage%253D773%26rft.epage%253D783%26rft.atitle%253DDecapping%2Bactivators%2Bin%2BSaccharomyces%2Bcerevisiae%2Bact%2Bby%2Bmultiple%2Bmechanisms.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.08.025%26rft_id%253Dinfo%253Apmid%252F20832728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.08.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20832728&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281941000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.76" data-doi="10.1016/j.molcel.2007.02.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheth</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">P bodies and the control of mRNA translation and degradation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">646</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DParker%26rft.auinit1%253DR.%26rft.volume%253D25%26rft.issue%253D5%26rft.spage%253D635%26rft.epage%253D646%26rft.atitle%253DP%2Bbodies%2Band%2Bthe%2Bcontrol%2Bof%2BmRNA%2Btranslation%2Band%2Bdegradation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2007.02.011%26rft_id%253Dinfo%253Apmid%252F17349952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2007.02.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17349952&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245127400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.77" data-doi="10.1016/j.cell.2015.05.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pelechano</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Widespread Co-translational RNA decay reveals ribosome dynamics.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">1400</span>–<span class="cit-lpage">1412</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D161%26rft.spage%253D1400%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.05.008%26rft_id%253Dinfo%253Apmid%252F26046441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.05.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26046441&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-other" id="cit-8.1.315.78" data-doi="10.15252/embj.201490194"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Preissler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reuther</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scior</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruderek</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Not4-dependent translational repression is important for cellular protein homeostasis in yeast.</span> <span class="cit-source">EMBO J.</span> <span class="cit-vol">34</span>: <span class="cit-fpage">1905</span>–<span class="cit-lpage">1924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fembj.201490194%26rft_id%253Dinfo%253Apmid%252F25971775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMzQvMTQvMTkwNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMS8zMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.79" data-doi="10.1016/j.bbagrm.2013.03.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Presnyak</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coller</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The DHH1/RCKp54 family of helicases: an ancient family of proteins that promote translational silencing.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1829</span>: <span class="cit-fpage">817</span>–<span class="cit-lpage">823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim.%2BBiophys.%2BActa%26rft.volume%253D1829%26rft.spage%253D817%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbagrm.2013.03.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbagrm.2013.03.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.80" data-doi="10.1016/j.tcb.2010.09.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quenault</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lithgow</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Traven</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">PUF proteins: repression, activation and mRNA localization.</span> <abbr class="cit-jnl-abbrev">Trends Cell Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">104</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bcell%2Bbiology%26rft.stitle%253DTrends%2BCell%2BBiol%26rft.aulast%253DQuenault%26rft.auinit1%253DT.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D104%26rft.epage%253D112%26rft.atitle%253DPUF%2Bproteins%253A%2Brepression%252C%2Bactivation%2Band%2BmRNA%2Blocalization.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tcb.2010.09.013%26rft_id%253Dinfo%253Apmid%252F21115348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tcb.2010.09.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21115348&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287544600005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-other" id="cit-8.1.315.81" data-doi="10.1016/j.cell.2016.08.053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Radhakrishnan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alhusaini</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The DEAD-Box protein Dhh1p couples mRNA decay and translation by monitoring codon optimality.</span> <span class="cit-source">Cell</span> <span class="cit-vol">167</span>: <span class="cit-fpage">122</span>–<span class="cit-lpage">132.e9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D167%26rft.spage%253D122%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.08.053%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.08.053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.82" data-doi="10.1016/j.bbagrm.2012.09.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reese</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The control of elongation by the yeast Ccr4-not complex.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1829</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim.%2BBiophys.%2BActa%26rft.volume%253D1829%26rft.spage%253D127%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbagrm.2012.09.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbagrm.2012.09.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.83" data-doi="10.1261/rna.955508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rendl</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bieman</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smibert</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">S. cerevisiae Vts1p induces deadenylation-dependent transcript degradation and interacts with the Ccr4p-Pop2p-Not deadenylase complex.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1328</span>–<span class="cit-lpage">1336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.955508%26rft_id%253Dinfo%253Apmid%252F18469165%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjE0LzcvMTMyOCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMS8zMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.84" data-doi="10.1016/j.molcel.2013.01.030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riley</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steitz</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The “Observer Effect” in genome-wide surveys of protein-RNA interactions.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">601</span>–<span class="cit-lpage">604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D49%26rft.spage%253D601%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2013.01.030%26rft_id%253Dinfo%253Apmid%252F23438856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2013.01.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23438856&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000315555200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.85" data-doi="10.1038/nbt.1754"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winckler</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guttman</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative genomics viewer.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRobinson%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D29%26rft.issue%253D1%26rft.spage%253D24%26rft.epage%253D26%26rft.atitle%253DIntegrative%2Bgenomics%2Bviewer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1754%26rft_id%253Dinfo%253Apmid%252F21221095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1754&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21221095&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286048900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.86" data-doi="10.1093/bioinformatics/btp616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">edgeR: a bioconductor package for differential expression analysis of digital gene expression data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp616%26rft_id%253Dinfo%253Apmid%252F19910308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19910308&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273116100025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.87" data-doi="10.1371/journal.pcbi.1000386"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rumble</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacroute</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalca</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fiume</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sidow</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">SHRiMP: accurate mapping of short color-space reads.</span> <abbr class="cit-jnl-abbrev">PLoS Comput. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DRumble%26rft.auinit1%253DS.%2BM.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253De1000386%26rft.epage%253De1000386%26rft.atitle%253DSHRiMP%253A%2Baccurate%2Bmapping%2Bof%2Bshort%2Bcolor-space%2Breads.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000386%26rft_id%253Dinfo%253Apmid%252F19461883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000386&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19461883&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.88" data-doi="10.1016/j.tibs.2008.07.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmid</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The exosome: a multipurpose RNA-decay machine.</span> <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bbiochemical%2Bsciences%26rft.stitle%253DTrends%2BBiochem%2BSci%26rft.aulast%253DSchmid%26rft.auinit1%253DM.%26rft.volume%253D33%26rft.issue%253D10%26rft.spage%253D501%26rft.epage%253D510%26rft.atitle%253DThe%2Bexosome%253A%2Ba%2Bmultipurpose%2BRNA-decay%2Bmachine.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibs.2008.07.003%26rft_id%253Dinfo%253Apmid%252F18786828%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibs.2008.07.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18786828&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260552800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.89" data-doi="10.1101/gr.130161.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwalb</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pirkl</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etzold</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparative dynamic transcriptome analysis (cDTA) reveals mutual feedback between mRNA synthesis and degradation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1350</span>–<span class="cit-lpage">1359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.130161.111%26rft_id%253Dinfo%253Apmid%252F22466169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzcvMTM1MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMS8zMTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.90" data-doi="10.1016/j.molcel.2013.09.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwalb</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pirkl</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maier</span>  <span class="cit-name-given-names">K. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schenk</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Global analysis of eukaryotic mRNA degradation reveals Xrn1-dependent buffering of transcript levels.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D52%26rft.spage%253D52%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2013.09.010%26rft_id%253Dinfo%253Apmid%252F24119399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2013.09.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24119399&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326195800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.91" data-doi="10.1371/journal.pbio.1001342"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sweet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kovalak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coller</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The DEAD-box protein Dhh1 promotes decapping by slowing ribosome movement.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1001342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DSweet%26rft.auinit1%253DT.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253De1001342%26rft.epage%253De1001342%26rft.atitle%253DThe%2BDEAD-box%2Bprotein%2BDhh1%2Bpromotes%2Bdecapping%2Bby%2Bslowing%2Bribosome%2Bmovement.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001342%26rft_id%253Dinfo%253Apmid%252F22719226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001342&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22719226&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.92" data-doi="10.1016/j.molcel.2010.02.039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Talarek</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cameroni</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaquenoud</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bontron</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Initiation of the TORC1-regulated G0 program requires Igo1/2, which license specific mRNAs to evade degradation via the 5′-3′ mRNA decay pathway.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">345</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DTalarek%26rft.auinit1%253DN.%26rft.volume%253D38%26rft.issue%253D3%26rft.spage%253D345%26rft.epage%253D355%26rft.atitle%253DInitiation%2Bof%2Bthe%2BTORC1-regulated%2BG0%2Bprogram%2Brequires%2BIgo1%252F2%252C%2Bwhich%2Blicense%2Bspecific%2BmRNAs%2Bto%2Bevade%2Bdegradation%2Bvia%2Bthe%2B5%2527-3%2527%2BmRNA%2Bdecay%2Bpathway.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.02.039%26rft_id%253Dinfo%253Apmid%252F20471941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.02.039&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20471941&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277818400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.93" data-doi="10.1126/science.1153878"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tarassov</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messier</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landry</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radinovic</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serna Molina</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">An in vivo map of the yeast protein interactome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">1465</span>–<span class="cit-lpage">1470</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1153878%26rft_id%253Dinfo%253Apmid%252F18467557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4Mi8xNDY1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.94" data-doi="10.1261/rna.7258505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teixeira</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheth</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valencia-Sanchez</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brengues</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Processing bodies require RNA for assembly and contain nontranslating mRNAs.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.7258505%26rft_id%253Dinfo%253Apmid%252F15703442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjExLzQvMzcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.95" data-doi="10.1093/bib/bbs017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thorvaldsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mesirov</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Integrative genomics viewer (IGV): high-performance genomics data visualization and exploration.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">178</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbs017%26rft_id%253Dinfo%253Apmid%252F22517427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bib/bbs017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22517427&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-96-1" title="View reference  in text" id="ref-96">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.96" data-doi="10.1126/science.1120499"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tu</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kudlicki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowicka</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKnight</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Logic of the yeast metabolic cycle: temporal compartmentalization of cellular processes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">310</span>: <span class="cit-fpage">1152</span>–<span class="cit-lpage">1158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1120499%26rft_id%253Dinfo%253Apmid%252F16254148%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTAvNTc1MS8xMTUyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-97-1" title="View reference  in text" id="ref-97">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.97" data-doi="10.1016/S0092-8674(01)00225-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valencia-Sanchez</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staples</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denis</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The transcription factor associated Ccr4 and Caf1 proteins are components of the major cytoplasmic mRNA deadenylase in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">377</span>–<span class="cit-lpage">386</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DTucker%26rft.auinit1%253DM.%26rft.volume%253D104%26rft.issue%253D3%26rft.spage%253D377%26rft.epage%253D386%26rft.atitle%253DThe%2Btranscription%2Bfactor%2Bassociated%2BCcr4%2Band%2BCaf1%2Bproteins%2Bare%2Bcomponents%2Bof%2Bthe%2Bmajor%2Bcytoplasmic%2BmRNA%2Bdeadenylase%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252900225-2%26rft_id%253Dinfo%253Apmid%252F11239395%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)00225-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11239395&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167003100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-98-1" title="View reference  in text" id="ref-98">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.98" data-doi="10.1093/emboj/21.6.1427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staples</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valencia-Sanchez</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muhlrad</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parker</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Ccr4p is the catalytic subunit of a Ccr4p/Pop2p/Notp mRNA deadenylase complex in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1427</span>–<span class="cit-lpage">1436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DTucker%26rft.auinit1%253DM.%26rft.volume%253D21%26rft.issue%253D6%26rft.spage%253D1427%26rft.epage%253D1436%26rft.atitle%253DCcr4p%2Bis%2Bthe%2Bcatalytic%2Bsubunit%2Bof%2Ba%2BCcr4p%252FPop2p%252FNotp%2BmRNA%2Bdeadenylase%2Bcomplex%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1093%252Femboj%252F21.6.1427%26rft_id%253Dinfo%253Apmid%252F11889048%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo5OiIyMS82LzE0MjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMzE1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-99-1" title="View reference  in text" id="ref-99">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.99" data-doi="10.1016/j.molcel.2011.01.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venters</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wachi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mavrich</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jena</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A comprehensive genomic binding map of gene and chromatin regulatory proteins in Saccharomyces.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">480</span>–<span class="cit-lpage">492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DVenters%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D480%26rft.epage%253D492%26rft.atitle%253DA%2Bcomprehensive%2Bgenomic%2Bbinding%2Bmap%2Bof%2Bgene%2Band%2Bchromatin%2Bregulatory%2Bproteins%2Bin%2BSaccharomyces.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2011.01.015%26rft_id%253Dinfo%253Apmid%252F21329885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2011.01.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21329885&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287773300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-100-1" title="View reference  in text" id="ref-100">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.100" data-doi="10.1016/j.bbagrm.2013.01.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wahle</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winkler</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RNA decay machines: deadenylation by the Ccr4-not and Pan2-Pan3 complexes.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1829</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim.%2BBiophys.%2BActa%26rft.volume%253D1829%26rft.spage%253D561%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbagrm.2013.01.003%26rft_id%253Dinfo%253Apmid%252F23337855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbagrm.2013.01.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23337855&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-101-1" title="View reference  in text" id="ref-101">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.101" data-doi="10.1016/S0022-4804(03)00155-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westmoreland</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saito</span>  <span class="cit-name-given-names">W. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huper</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marks</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Dhh1 regulates the G1/S-checkpoint following DNA damage or BRCA1 expression in yeast.</span> <abbr class="cit-jnl-abbrev">J. Surg. Res.</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">62</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2Bsurgical%2Bresearch%26rft.stitle%253DJ%2BSurg%2BRes%26rft.aulast%253DWestmoreland%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D113%26rft.issue%253D1%26rft.spage%253D62%26rft.epage%253D73%26rft.atitle%253DDhh1%2Bregulates%2Bthe%2BG1%252FS-checkpoint%2Bfollowing%2BDNA%2Bdamage%2Bor%2BBRCA1%2Bexpression%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-4804%252803%252900155-0%26rft_id%253Dinfo%253Apmid%252F12943812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-4804(03)00155-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12943812&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185080300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-102-1" title="View reference  in text" id="ref-102">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.102" data-doi="10.1128/EC.3.2.430-446.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westmoreland</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marks</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">J. A.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resnick</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Cell cycle progression in G1 and S phases is CCR4 dependent following ionizing radiation or replication stress in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">430</span>–<span class="cit-lpage">446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DWestmoreland%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D430%26rft.epage%253D446%26rft.atitle%253DCell%2BCycle%2BProgression%2Bin%2BG1%2Band%2BS%2BPhases%2BIs%2BCCR4%2BDependent%2Bfollowing%2BIonizing%2BRadiation%2Bor%2BReplication%2BStress%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.3.2.430-446.2004%26rft_id%253Dinfo%253Apmid%252F15075273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiIzLzIvNDMwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-103-1" title="View reference  in text" id="ref-103">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.103" data-doi="10.1038/ncomms9213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilinski</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lapointe</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nevil</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">Z. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">RNA regulatory networks diversified through curvature of the PUF protein scaffold.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">8213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D6%26rft.spage%253D8213%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms9213%26rft_id%253Dinfo%253Apmid%252F26364903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms9213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26364903&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F315.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-104-1" title="View reference  in text" id="ref-104">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.315.104" data-doi="10.1083/jcb.127.5.1173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Datar</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paddy</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swedlow</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Characterization of nuclear polyadenylated RNA-binding proteins in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1173</span>–<span class="cit-lpage">1184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BCell%2BBiology%26rft.stitle%253DJCB%26rft.aulast%253DWilson%26rft.auinit1%253DS.%2BM.%26rft.volume%253D127%26rft.issue%253D5%26rft.spage%253D1173%26rft.epage%253D1184%26rft.atitle%253DCharacterization%2Bof%2Bnuclear%2Bpolyadenylated%2BRNA-binding%2Bproteins%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.127.5.1173%26rft_id%253Dinfo%253Apmid%252F7962083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjEwOiIxMjcvNS8xMTczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMxNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/1/315.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/303" title="A Deconvolution Protocol for ChIP-Seq Reveals Analogous Enhancer Structures on the Mouse and Human Ribosomal RNA Genes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/331" title="Mating Design and Genetic Structure of a Multi-Parent Advanced Generation Intercross (MAGIC) Population of Sorghum (Sorghum bicolor (L.) Moench)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/1"><span>Volume 8 Issue 1, January 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/1.cover-source.jpg?itok=vyoqxep2" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/315.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-49wpkqsz8ae4ZzqtVTcrhDNuNIIJGreaBZO-jxndkd0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/1/315" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/1/315" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/1/315" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Mapping of Decay Factor–mRNA Interactions in Yeast Identifies Nutrient-Responsive Transcripts as Targets of the Deadenylase Ccr4</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487822" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-wSkO3IlBJoRh8o_Z1X2IjrJ3iAQF__4nQNTku7ezxZA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/315.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/1/315" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-c6ZIhQddNab3euKh0BCfZdMHwx63zkbrXxAshYP8p0I" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/1/315" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/315.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487822" id="node487822--2" data-pisa="ggg;8/1/315" data-pisa-master="ggg;g3.117.300415" data-apath="/ggg/8/1/315.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Mapping of Decay Factor–mRNA Interactions in Yeast Identifies Nutrient-Responsive Transcripts as Targets of the Deadenylase Ccr4</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jason E. Miller</span>, <span class="highwire-citation-author" data-delta="1">Liye Zhang</span>, <span class="highwire-citation-author" data-delta="2">Haoyang Jiang</span>, <span class="highwire-citation-author" data-delta="3">Yunfei Li</span>, <span class="highwire-citation-author" data-delta="4">B. Franklin Pugh</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-3578-4266" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joseph C. Reese</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">315-330;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300415 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487822--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jason E. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for RNA Molecular Biology, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BE.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BE.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Liye Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Comparative Genomics and Bioinformatics, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liye%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALiye%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Haoyang Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for RNA Molecular Biology, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haoyang%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaoyang%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yunfei Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Comparative Genomics and Bioinformatics, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yunfei%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYunfei%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">B. Franklin Pugh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Comparative Genomics and Bioinformatics, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BFranklin%2BPugh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pugh%20BF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AB.%2BFranklin%2BPugh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Joseph C. Reese </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for RNA Molecular Biology, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BC.%2BReese%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reese%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BC.%2BReese%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3578-4266" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joseph C. Reese</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Jcr8@psu.edu" class="" data-icon-position="" data-hide-link-title="0">Jcr8@psu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/1/315" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/1/315&amp;title=Genome-Wide%20Mapping%20of%20Decay%20Factor%E2%80%93mRNA%20Interactions%20in%20Yeast%20Identifies%20Nutrient-Responsive%20Transcripts%20as%20Targets%20of%20the%20Deadenylase%20Ccr4" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/1/315&amp;title=Genome-Wide%20Mapping%20of%20Decay%20Factor%E2%80%93mRNA%20Interactions%20in%20Yeast%20Identifies%20Nutrient-Responsive%20Transcripts%20as%20Targets%20of%20the%20Deadenylase%20Ccr4" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/1/315&amp;title=Genome-Wide%20Mapping%20of%20Decay%20Factor%E2%80%93mRNA%20Interactions%20in%20Yeast%20Identifies%20Nutrient-Responsive%20Transcripts%20as%20Targets%20of%20the%20Deadenylase%20Ccr4" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/1/315&amp;text=Genome-Wide%20Mapping%20of%20Decay%20Factor%E2%80%93mRNA%20Interactions%20in%20Yeast%20Identifies%20Nutrient-Responsive%20Transcripts%20as%20Targets%20of%20the%20Deadenylase%20Ccr4" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/1/315&amp;title=Genome-Wide%20Mapping%20of%20Decay%20Factor%E2%80%93mRNA%20Interactions%20in%20Yeast%20Identifies%20Nutrient-Responsive%20Transcripts%20as%20Targets%20of%20the%20Deadenylase%20Ccr4" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/1/315&amp;t=Genome-Wide%20Mapping%20of%20Decay%20Factor%E2%80%93mRNA%20Interactions%20in%20Yeast%20Identifies%20Nutrient-Responsive%20Transcripts%20as%20Targets%20of%20the%20Deadenylase%20Ccr4" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/1/315&amp;title=Genome-Wide%20Mapping%20of%20Decay%20Factor%E2%80%93mRNA%20Interactions%20in%20Yeast%20Identifies%20Nutrient-Responsive%20Transcripts%20as%20Targets%20of%20the%20Deadenylase%20Ccr4" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/1/315&amp;title=Genome-Wide%20Mapping%20of%20Decay%20Factor%E2%80%93mRNA%20Interactions%20in%20Yeast%20Identifies%20Nutrient-Responsive%20Transcripts%20as%20Targets%20of%20the%20Deadenylase%20Ccr4" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487822/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487822" id="node487822--3" data-pisa="ggg;8/1/315" data-pisa-master="ggg;g3.117.300415" data-apath="/ggg/8/1/315.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Mapping of Decay Factor–mRNA Interactions in Yeast Identifies Nutrient-Responsive Transcripts as Targets of the Deadenylase Ccr4</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jason E. Miller</span>, <span class="highwire-citation-author" data-delta="1">Liye Zhang</span>, <span class="highwire-citation-author" data-delta="2">Haoyang Jiang</span>, <span class="highwire-citation-author" data-delta="3">Yunfei Li</span>, <span class="highwire-citation-author" data-delta="4">B. Franklin Pugh</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-3578-4266" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joseph C. Reese</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">315-330;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300415 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487822--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jason E. Miller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for RNA Molecular Biology, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BE.%2BMiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Miller%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BE.%2BMiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Liye Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Comparative Genomics and Bioinformatics, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liye%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALiye%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Haoyang Jiang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for RNA Molecular Biology, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Haoyang%2BJiang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jiang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHaoyang%2BJiang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Yunfei Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Comparative Genomics and Bioinformatics, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yunfei%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYunfei%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">B. Franklin Pugh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Comparative Genomics and Bioinformatics, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=B.%2BFranklin%2BPugh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pugh%20BF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AB.%2BFranklin%2BPugh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Joseph C. Reese </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for RNA Molecular Biology, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joseph%2BC.%2BReese%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reese%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoseph%2BC.%2BReese%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-3578-4266" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joseph C. Reese</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Jcr8@psu.edu" class="" data-icon-position="" data-hide-link-title="0">Jcr8@psu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=315&amp;rft.epage=330&amp;rft.atitle=Genome-Wide%20Mapping%20of%20Decay%20Factor%E2%80%93mRNA%20Interactions%20in%20Yeast%20Identifies%20Nutrient-Responsive%20Transcripts%20as%20Targets%20of%20the%20Deadenylase%20Ccr4&amp;rft.volume=8&amp;rft.issue=1&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Miller%2C+Jason+E.&amp;rft.au=Zhang%2C+Liye&amp;rft.au=Jiang%2C+Haoyang&amp;rft.au=Li%2C+Yunfei&amp;rft.au=Pugh%2C+B.+Franklin&amp;rft.au=Reese%2C+Joseph+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487822/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487822/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487822/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487822/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487822/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487822/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487822/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487822/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487822/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487822/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487822/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487822/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/1/315" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/1/315.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/1/315.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/1/315.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/1/315.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
