<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/1/331" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/1/331.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/1/331.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/1/331.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/1/331/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/1/331.atom" />
<meta name="HW.pisa" content="ggg;8/1/331" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Mating Design and Genetic Structure of a Multi-Parent Advanced Generation Intercross (MAGIC) Population of Sorghum (Sorghum bicolor (L.) Moench)" />
<meta name="DC.Identifier" content="10.1534/g3.117.300248" />
<meta name="DC.Date" content="2018-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Ongom, Ejeta. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Multi-parent advanced generation intercross (MAGIC) populations are powerful next-generation mapping resources. We describe here the mating design and structure of the first MAGIC population in sorghum, and test its utility for mapping. The population was developed by intercrossing 19 diverse founder lines through a series of paired crosses with a genetic male sterile (MS) source, followed by 10 generations of random mating. At the final stage of random mating, 1000 random fertile plants in the population were identified and subjected to six generations of selfing to produce 1000 immortal MAGIC inbred lines. The development of this sorghum MAGIC population took over 15 yr. Genotyping-by-sequencing (GBS) of a subset of 200 MAGIC lines identified 79,728 SNPs, spanning high gene-rich regions. Proportion of SNPs per chromosome ranged from 6 to 15%. Structure analyses produced no evidence of population stratification, portraying the desirability of this population for genome-wide association studies (GWAS). The 19 founders formed three clusters, each with considerable genetic diversity. Further analysis showed that 73% of founder alleles segregated in the MAGIC population. Linkage disequilibrium (LD) patterns depicted the MAGIC population to be highly recombined, with LD decaying to r 2 ![Formula][1]&lt;/img&gt; 0.2 at 40 kb and down to r 2 ![Formula][2]&lt;/img&gt; 0.1 at 220 kb. GWAS detected two known plant height genes, DWARF1 (chromosome 9) and DWARF3 (chromosome 7), and a potentially new plant height quantitative trait locus (QTL) ( QTL-6 ) on chromosome 6. The MAGIC population was found to be rich in allelic content with high fragmentation of its genome, making it fit for both gene mapping and effective marker-assisted breeding.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif" />
<meta name="DC.Contributor" content="Patrick O. Ongom" />
<meta name="DC.Contributor" content="Gebisa Ejeta" />
<meta name="article:published_time" content="2018-01-01" />
<meta name="article:section" content="Multiparental Populations" />
<meta name="citation_title" content="Mating Design and Genetic Structure of a Multi-Parent Advanced Generation Intercross (MAGIC) Population of Sorghum (Sorghum bicolor (L.) Moench)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Multi-parent advanced generation intercross (MAGIC) populations are powerful next-generation mapping resources. We describe here the mating design and structure of the first MAGIC population in sorghum, and test its utility for mapping. The population was developed by intercrossing 19 diverse founder lines through a series of paired crosses with a genetic male sterile (MS) source, followed by 10 generations of random mating. At the final stage of random mating, 1000 random fertile plants in the population were identified and subjected to six generations of selfing to produce 1000 immortal MAGIC inbred lines. The development of this sorghum MAGIC population took over 15 yr. Genotyping-by-sequencing (GBS) of a subset of 200 MAGIC lines identified 79,728 SNPs, spanning high gene-rich regions. Proportion of SNPs per chromosome ranged from 6 to 15%. Structure analyses produced no evidence of population stratification, portraying the desirability of this population for genome-wide association studies (GWAS). The 19 founders formed three clusters, each with considerable genetic diversity. Further analysis showed that 73% of founder alleles segregated in the MAGIC population. Linkage disequilibrium (LD) patterns depicted the MAGIC population to be highly recombined, with LD decaying to &lt;i&gt;r&lt;/i&gt;&lt;sup&gt;2&lt;/sup&gt;&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me1&quot; overflow=&quot;scroll&quot;&gt;&lt;mo&gt;≤&lt;/mo&gt;&lt;/math&gt; 0.2 at 40 kb and down to &lt;i&gt;r&lt;/i&gt;&lt;sup&gt;2&lt;/sup&gt;&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;me2&quot; overflow=&quot;scroll&quot;&gt;&lt;mo&gt;≤&lt;/mo&gt;&lt;/math&gt; 0.1 at 220 kb. GWAS detected two known plant height genes, &lt;i&gt;DWARF1&lt;/i&gt; (chromosome 9) and &lt;i&gt;DWARF3&lt;/i&gt; (chromosome 7), and a potentially new plant height quantitative trait locus (QTL) (&lt;i&gt;QTL-6&lt;/i&gt;) on chromosome 6. The MAGIC population was found to be rich in allelic content with high fragmentation of its genome, making it fit for both gene mapping and effective marker-assisted breeding.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="ggg;8/1/331" />
<meta name="citation_id" content="8/1/331" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/1/331" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/1/331.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/1/331.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/1/331.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300248" />
<meta name="citation_pmid" content="29150594" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Multiparental Populations" />
<meta name="citation_firstpage" content="331" />
<meta name="citation_lastpage" content="341" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Patrick O. Ongom" />
<meta name="citation_author_institution" content="Department of Agronomy, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author" content="Gebisa Ejeta" />
<meta name="citation_author_institution" content="Department of Agronomy, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author_institution" content="Center for Global Food Security, Purdue University, West Lafayette, Indiana 47907" />
<meta name="citation_author_email" content="gejeta@purdue.edu" />
<meta name="citation_reference" content="citation_journal_title=Journal of genetics;citation_journal_abbrev=J Genet;citation_author=D. B. Apotikar;citation_author=D. Venkateswarlu;citation_author=R. B. Ghorade;citation_author=R. M. Wadaskar;citation_author=J. V. Patil;citation_title=Mapping of shoot fly tolerance loci in sorghum using SSR markers.;citation_pages=59-66;citation_volume=90;citation_year=2011;citation_issue=1;citation_pmid=21677382" />
<meta name="citation_reference" content="citation_journal_title=Rice (N. Y.);citation_author=N. Bandillo;citation_author=C. Raghavan;citation_author=P. A. Muyco;citation_author=M. A. L. Sevilla;citation_author=I. T. Lobina;citation_title=Multi-parent advanced generation inter-cross (MAGIC) populations in rice: progress and potential for genetics research and breeding.;citation_pages=1-15;citation_volume=6;citation_year=2013;citation_pmid=24280096;citation_doi=10.1186/1939-8433-6-1" />
<meta name="citation_reference" content="citation_journal_title=Maize Genet. Coop. News Lett.;citation_author=W. Beavis;citation_author=M. Lee;citation_author=D. Grant;citation_author=A. Hallauer;citation_title=The influence of random mating on recombination among RFLP loci.;citation_pages=52-53;citation_volume=66;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. J. Brown;citation_author=W. L. Rooney;citation_author=C. Franks;citation_author=S. Kresovich;citation_title=Efficient mapping of plant height quantitative trait loci in a sorghum association population with introgressed dwarfing genes.;citation_volume=180;citation_year=2008;citation_pmid=18757942;citation_doi=10.1534/genetics.108.092239" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=C. Cavanagh;citation_author=M. Morell;citation_author=M. Ian;citation_author=P. Wayne;citation_title=From mutations to MAGIC: resources for gene discovery, validation and delivery in crop plants.;citation_pages=215-221;citation_volume=11;citation_year=2008;citation_issue=2;citation_pmid=18295532;citation_doi=10.1016/j.pbi.2008.01.002" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=B. C. Y. Collard;citation_author=M. Z. Z. Jahufer;citation_author=J. B. Brouwer;citation_author=E. C. K. Pang;citation_title=An introduction to markers, quantitative trait loci (QTL) mapping and marker-assisted selection for crop improvement: the basic concepts.;citation_pages=169-196;citation_volume=142;citation_year=2005;citation_doi=10.1007/s10681-005-1681-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Darvasi;citation_author=M. Soller;citation_title=Advanced intercross lines, an experimental population for fine genetic mapping.;citation_pages=1199-1207;citation_volume=141;citation_year=1995;citation_issue=3;citation_pmid=8582624" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Dell’Acqua;citation_author=D. Gatti;citation_author=G. Pea;citation_title=Genetic properties of the MAGIC maize population: a new platform for high definition QTL mapping in Zea mays.;citation_pages=1-23;citation_volume=16;citation_year=2015;citation_pmid=25583448;citation_doi=10.1186/s13059-014-0572-2" />
<meta name="citation_reference" content="citation_title=Sorghum in Seventies;citation_title=Techniques and methods for more efficient population improvement in sorghum;citation_author=S. Eberhart;citation_pages=197-213;citation_year=1972" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. Evans;citation_author=R. F. McCormick;citation_author=D. Morishige;citation_author=S. N. Olson;citation_author=B. Weers;citation_title=Extensive variation in the density and distribution of DNA polymorphism in sorghum genomes.;citation_pages=e79192;citation_volume=8;citation_year=2013;citation_pmid=24265758;citation_doi=10.1371/journal.pone.0079192" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=F. Feltus;citation_author=G. Hart;citation_author=K. Schertz;citation_author=A. Casa;citation_author=S. Kresovich;citation_title=Alignment of genetic maps and QTLs between inter- and intra-specific sorghum populations.;citation_pages=1295-1305;citation_volume=112;citation_year=2006;citation_issue=7;citation_pmid=16491426;citation_doi=10.1007/s00122-006-0232-3" />
<meta name="citation_reference" content="citation_journal_title=Trans. R. Soc. Edinb.;citation_author=R. A. Fisher;citation_title=The correlation between relatives on the supposition of Mendelian inheritance.;citation_pages=399-433;citation_volume=52;citation_year=1919;citation_doi=10.1017/S0080456800012163" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=P. Gupta;citation_author=S. Rustgi;citation_author=P. Kulwal;citation_title=Linkage disequilibrium and association studies in higher plants: present status and future prospects.;citation_pages=461-485;citation_volume=57;citation_year=2005;citation_issue=4;citation_pmid=15821975;citation_doi=10.1007/s11103-005-0257-z" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. T. Hamblin;citation_author=S. E. Mitchell;citation_author=G. M. White;citation_author=J. Gallego;citation_author=R. Kukatla;citation_title=Comparative Population Genetics of the Panicoid Grasses: Sequence Polymorphism, Linkage Disequilibrium and Selection in a Diverse Sample of Sorghum bicolor;citation_pages=471-483;citation_volume=167;citation_year=2004;citation_issue=1;citation_pmid=15166170;citation_doi=10.1534/genetics.167.1.471" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=H. Harlan;citation_author=M. Martini;citation_title=A Composite Hybrid Mixture;citation_pages=487-490;citation_volume=21;citation_year=1929;citation_issue=4" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=B. I. G. Haussmann;citation_author=D. E. Hess;citation_author=G. O. Omanya;citation_author=R. T. Folkertsma;citation_author=B. V. S. Reddy;citation_title=Genomic regions influencing resistance to the parasitic weed Striga hermonthica in two recombinant inbred populations of sorghum.;citation_pages=1005-1016;citation_volume=109;citation_year=2004;citation_issue=5;citation_pmid=15160241;citation_doi=10.1007/s00122-004-1706-9" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=R. H. Higgins;citation_author=C. S. Thurber;citation_author=I. Assaranurak;citation_author=P. J. Brown;citation_title=Multiparental mapping of plant height and flowering time QTL in partially isogenic sorghum families.;citation_pages=1593-1602;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. Hilley;citation_author=S. Truong;citation_author=S. Olson;citation_author=D. Morishige;citation_author=J. Mullet;citation_title=Identification of Dw1, a regulator of sorghum stem internode length.;citation_pages=e0151271;citation_volume=11;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=J. Holland;citation_title=MAGIC maize: a new resource for plant genetics.;citation_pages=163;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=B. E. Huang;citation_author=A. W. George;citation_author=K. L. Forrest;citation_author=A. Kilian;citation_author=M. J. Hayden;citation_title=A multiparent advanced generation inter-cross population for genetic analysis in wheat.;citation_pages=826-839;citation_volume=10;citation_year=2012;citation_issue=7;citation_pmid=22594629;citation_doi=10.1111/j.1467-7652.2012.00702.x" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=B. E. Huang;citation_author=K. L. Verbyla;citation_author=A. P. Verbyla;citation_author=C. Raghavan;citation_author=V. K. Singh;citation_title=MAGIC populations in crops: current status and future prospects.;citation_pages=999-1017;citation_volume=128;citation_year=2015;citation_pmid=25855139;citation_doi=10.1007/s00122-015-2506-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Kang;citation_author=N. Zaitlen;citation_author=C. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient Control of Population Structure in Model Organism Association Mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=R. Klein;citation_author=J. Mullet;citation_author=D. Jordan;citation_author=F. Miller;citation_author=W. Rooney;citation_title=The effect of tropical sorghum conversion and inbred development on genome diversity as revealed by high-resolution genotyping.;citation_pages=S12-S26;citation_volume=48;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. Knoll;citation_author=N. Gunaratna;citation_author=G. Ejeta;citation_title=QTL analysis of early-season cold tolerance in sorghum.;citation_pages=577-587;citation_volume=116;citation_year=2008;citation_issue=4;citation_pmid=18097644" />
<meta name="citation_reference" content="citation_title=Genomics and Breeding for Climate-Resilient Crops;citation_author=C. Kole;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=P. X. Kover;citation_author=W. Valdar;citation_author=J. Trakalo;citation_author=N. Scarcelli;citation_author=I. M. Ehrenreich;citation_title=A Multiparent Advanced Generation Inter-Cross to fine-map quantitative traits in Arabidopsis thaliana.;citation_pages=e1000551-e1000551;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19593375;citation_doi=10.1371/journal.pgen.1000551" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=E. Mace;citation_author=D. Jordan;citation_title=Integrating sorghum whole genome sequence information with a compendium of sorghum QTL studies reveals uneven distribution of QTL and of gene-rich regions with significant implications for crop improvement.;citation_pages=169-191;citation_volume=123;citation_year=2011;citation_issue=1;citation_pmid=21484332;citation_doi=10.1007/s00122-011-1575-y" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=I. J. Mackay;citation_author=P. Bansept-Basler;citation_author=T. Barber;citation_author=A. R. Bentley;citation_author=J. Cockram;citation_title=An eight-parent multiparent advanced generation inter-cross population for winter-sown wheat: creation, properties, and validation.;citation_pages=1603-1610;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_title=Sorghum in Seventies;citation_title=Objectives and approaches to grain and forage sorghum improvement in the Americas;citation_author=A. Maunder;citation_pages=60-100;citation_year=1972" />
<meta name="citation_reference" content="citation_title=Commentary on R.A. Fisher’s Paper on the Correlation Between Relatives on the Supposition of Mendelian Inheritance;citation_author=P. A. P. Moran;citation_author=C. A. B. Smith;citation_author=R. A. F. Sir;citation_year=1966" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. P. Morris;citation_author=P. Ramu;citation_author=S. P. Deshpande;citation_author=C. T. Hash;citation_author=T. Shah;citation_title=Population genomic and genome-wide association studies of agroclimatic traits in sorghum.;citation_volume=110;citation_year=2013;citation_pmid=23267105;citation_doi=10.1073/pnas.1215985110" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. S. Multani;citation_author=S. P. Briggs;citation_author=M. A. Chamberlin;citation_author=J. J. Blakeslee;citation_author=A. S. Murphy;citation_title=Loss of an MDR Transporter in Compact Stalks of Maize br2 and Sorghum dw3 Mutants;citation_pages=81-84;citation_volume=302;citation_year=2003;citation_issue=5642;citation_pmid=14526073;citation_doi=10.1126/science.1086072" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Nordborg;citation_author=D. Weigel;citation_title=Next-generation genetics in plants.;citation_pages=720-723;citation_volume=456;citation_year=2008;citation_issue=7223;citation_pmid=19079047;citation_doi=10.1038/nature07629" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=R. K. Pasam;citation_author=R. Sharma;citation_author=M. Malosetti;citation_author=F. A. van Eeuwijk;citation_author=G. Haseneyer;citation_title=Genome-wide association studies for agronomical traits in a world wide spring barley collection.;citation_pages=16-16;citation_volume=12;citation_year=2012;citation_pmid=22284310;citation_doi=10.1186/1471-2229-12-16" />
<meta name="citation_reference" content="citation_journal_abbrev=Int J Plant Genomics;citation_author=A. H. Paterson;citation_title=Genomics of sorghum.;citation_pages=362451-362451;citation_volume=2008;citation_year=2008;citation_pmid=18483564" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. H. Paterson;citation_author=J. E. Bowers;citation_author=R. Bruggmann;citation_author=I. Dubchak;citation_author=J. Grimwood;citation_title=The Sorghum bicolor genome and the diversification of grasses.;citation_pages=551-556;citation_volume=457;citation_year=2009;citation_issue=7229;citation_pmid=19189423;citation_doi=10.1038/nature07723" />
<meta name="citation_reference" content="citation_journal_title=JAMA;citation_journal_abbrev=JAMA;citation_author=T. A. Pearson;citation_author=T. A. Manolio;citation_title=How to Interpret a Genome-wide Association Study;citation_pages=1335-1344;citation_volume=299;citation_year=2008;citation_issue=11;citation_pmid=18349094;citation_doi=10.1001/jama.299.11.1335" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=J. Quinby;citation_author=R. Karper;citation_title=Inheritance of Height in Sorghum;citation_pages=211-216;citation_volume=46;citation_year=1954;citation_issue=5;citation_doi=10.2134/agronj1954.00021962004600050007x" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=J. A. Rafalski;citation_title=Association genetics in crop improvement.;citation_pages=174-180;citation_volume=13;citation_year=2010;citation_issue=2;citation_pmid=20089441;citation_doi=10.1016/j.pbi.2009.12.004" />
<meta name="citation_reference" content="citation_journal_title=Journal of genetics;citation_journal_abbrev=J Genet;citation_author=S. Rakshit;citation_author=A. Rakshit;citation_author=J. V. Patil;citation_title=Multiparent intercross populations in analysis of quantitative traits.;citation_pages=111-117;citation_volume=91;citation_year=2012;citation_issue=1;citation_pmid=22546834;citation_doi=10.1007/s12041-012-0144-8" />
<meta name="citation_reference" content="citation_journal_title=PKV Res. J.;citation_author=B. V. S. Reddy;citation_author=J. W. Stenhouse;citation_title=Sorghum improvement for semi-arid tropics region: past, current and future research thrusts in Asia.;citation_pages=155-169;citation_volume=18;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed. Rev.;citation_author=B. V. S. Reddy;citation_author=S. Ramesh;citation_author=R. Ortiz;citation_title=Genetic and cytoplasmic-nuclear male sterility in sorghum.;citation_pages=139-172;citation_volume=25;citation_year=2005" />
<meta name="citation_reference" content="citation_title=Technical and Institutional Options for sorghum Grain Mold Management : Proceedings of an International Consultation;citation_title=Potential of marker-assisted selection for improving grain mold resistance in sorghum;citation_author=W. Rooney;citation_author=R. Klein;citation_pages=183-194;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=C. A. Suneson;citation_title=An Evolutionary Plant Breeding Method;citation_pages=188-191;citation_volume=48;citation_year=1956;citation_issue=4" />
<meta name="citation_reference" content="citation_author=M. E. Tabangin;citation_author=J. G. Woo;citation_author=L. J. Martin;citation_year=2009" />
<meta name="citation_reference" content="citation_title=Population Genetics and Microevolutionary Theory;citation_author=A. R. Templeton;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=A haplotype map of the human genome.;citation_pages=1299-1320;citation_volume=437;citation_year=2005;citation_issue=7063;citation_pmid=16255080;citation_doi=10.1038/nature04226" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=C. S. Thurber;citation_author=J. M. Ma;citation_author=R. H. Higgins;citation_author=P. J. Brown;citation_title=Retrospective genomic analysis of sorghum adaptation to temperate-zone grain production.;citation_pages=1-12;citation_volume=14;citation_year=2013;citation_doi=10.1186/gb-2013-14-1-r1" />
<meta name="citation_reference" content="citation_journal_title=Moench. Mol. Breed.;citation_author=M. R. Tuinstra;citation_author=E. M. Grote;citation_author=P. B. Goldsbrough;citation_author=G. Ejeta;citation_title=Genetic analysis of post-flowering drought tolerance and components of grain development in Sorghum bicolor (L.);citation_pages=439-448;citation_volume=3;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=H. D. Upadhyaya;citation_author=Y.-H. Wang;citation_author=S. Sharma;citation_author=S. Singh;citation_title=Association mapping of height and maturity across five environments using the sorghum mini core collection.;citation_pages=471-479;citation_volume=55;citation_year=2012;citation_issue=6;citation_pmid=22680231" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. P. Verbyla;citation_author=C. R. Cavanagh;citation_author=K. L. Verbyla;citation_title=Whole-genome analysis of multienvironment or multitrait QTL in MAGIC.;citation_pages=1569-1584;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=R. K. Vogler;citation_author=G. Ejeta;citation_author=L. G. Butler;citation_title=Inheritance of Low Production of Striga Germination Stimulant in Sorghum;citation_pages=1185-1191;citation_volume=36;citation_year=1996;citation_issue=5" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. Voorman;citation_author=T. Lumley;citation_author=B. McKnight;citation_author=K. Rice;citation_title=Behavior of QQ-plots and genomic control in studies of gene-environment interaction.;citation_pages=e19416-e19416;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21589913;citation_doi=10.1371/journal.pone.0019416" />
<meta name="citation_reference" content="citation_journal_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_author=S. R. Wilson;citation_title=A note on assortative mating, linkage and genotypic frequencies.;citation_pages=129-130;citation_volume=42;citation_year=1978;citation_issue=1;citation_pmid=686680" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=M. Yamaguchi;citation_author=H. Fujimoto;citation_author=K. Hirano;citation_title=Sorghum Dw1, an agronomically important gene for lodging resistance, encodes a novel protein involved in cell proliferation.;citation_pages=28366;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Z. Zhang;citation_author=E. Ersoz;citation_author=C.-Q. Lai;citation_author=R. J. Todhunter;citation_author=H. K. Tiwari;citation_title=Mixed linear model approach adapted for genome-wide association studies.;citation_pages=355-360;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208535;citation_doi=10.1038/ng.546" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Mating Design and Genetic Structure of a Multi-Parent Advanced Generation Intercross (MAGIC) Population of Sorghum (Sorghum bicolor (L.) Moench)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="twitter:description" content="Multi-parent advanced generation intercross (MAGIC) populations are powerful next-generation mapping resources. We describe here the mating design and structure of the first MAGIC population in sorghum, and test its utility for mapping. The population was developed by intercrossing 19 diverse founder lines through a series of paired crosses with a genetic male sterile (MS) source, followed by 10 generations of random mating. At the final stage of random mating, 1000 random fertile plants in the population were identified and subjected to six generations of selfing to produce 1000 immortal MAGIC inbred lines. The development of this sorghum MAGIC population took over 15 yr. Genotyping-by-sequencing (GBS) of a subset of 200 MAGIC lines identified 79,728 SNPs, spanning high gene-rich regions. Proportion of SNPs per chromosome ranged from 6 to 15%. Structure analyses produced no evidence of population stratification, portraying the desirability of this population for genome-wide association studies (GWAS). The 19 founders formed three clusters, each with considerable genetic diversity. Further analysis showed that 73% of founder alleles segregated in the MAGIC population. Linkage disequilibrium (LD) patterns depicted the MAGIC population to be highly recombined, with LD decaying to r 2 ![Formula][1]&lt;/img&gt; 0.2 at 40 kb and down to r 2 ![Formula][2]&lt;/img&gt; 0.1 at 220 kb. GWAS detected two known plant height genes, DWARF1 (chromosome 9) and DWARF3 (chromosome 7), and a potentially new plant height quantitative trait locus (QTL) ( QTL-6 ) on chromosome 6. The MAGIC population was found to be rich in allelic content with high fragmentation of its genome, making it fit for both gene mapping and effective marker-assisted breeding.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif" />
<meta name="og-title" property="og:title" content="Mating Design and Genetic Structure of a Multi-Parent Advanced Generation Intercross (MAGIC) Population of Sorghum (Sorghum bicolor (L.) Moench)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/1/331" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Multi-parent advanced generation intercross (MAGIC) populations are powerful next-generation mapping resources. We describe here the mating design and structure of the first MAGIC population in sorghum, and test its utility for mapping. The population was developed by intercrossing 19 diverse founder lines through a series of paired crosses with a genetic male sterile (MS) source, followed by 10 generations of random mating. At the final stage of random mating, 1000 random fertile plants in the population were identified and subjected to six generations of selfing to produce 1000 immortal MAGIC inbred lines. The development of this sorghum MAGIC population took over 15 yr. Genotyping-by-sequencing (GBS) of a subset of 200 MAGIC lines identified 79,728 SNPs, spanning high gene-rich regions. Proportion of SNPs per chromosome ranged from 6 to 15%. Structure analyses produced no evidence of population stratification, portraying the desirability of this population for genome-wide association studies (GWAS). The 19 founders formed three clusters, each with considerable genetic diversity. Further analysis showed that 73% of founder alleles segregated in the MAGIC population. Linkage disequilibrium (LD) patterns depicted the MAGIC population to be highly recombined, with LD decaying to r 2 ![Formula][1]&lt;/img&gt; 0.2 at 40 kb and down to r 2 ![Formula][2]&lt;/img&gt; 0.1 at 220 kb. GWAS detected two known plant height genes, DWARF1 (chromosome 9) and DWARF3 (chromosome 7), and a potentially new plant height quantitative trait locus (QTL) ( QTL-6 ) on chromosome 6. The MAGIC population was found to be rich in allelic content with high fragmentation of its genome, making it fit for both gene mapping and effective marker-assisted breeding.

 [1]: /embed/mml-math-1.gif
 [2]: /embed/mml-math-2.gif" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
    <title>Mating Design and Genetic Structure of a Multi-Parent Advanced Generation Intercross (MAGIC) Population of Sorghum (Sorghum bicolor (L.) Moench) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"kO2s0H0_f5MP-1Ttb2KCL_3GgwlUYeuHIt6Aan1DYTY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487832","apath":"\/ggg\/8\/1\/331.atom","pisa":"ggg;8\/1\/331","ac":{"\/ggg\/8\/1\/331.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/1\/331.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/1\/331"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/1\/331"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/1\/331":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PMa0cihakU78ERLC2tANHZW4\\\\\\\/ThjF1hZOu5FgO4T+CDwLharKenSa5tdcpAFyIewZyazaiHkFJmqkIC\\\\\\\/XV0I8DudOGa3ZrVkYHh+mKfVc6r\\\\\\\/bKWxxjHgXncUVGyVmr4EytC6Q4TLPSeDtJ95plDS9+AsKG06j2Pfe1eGG8XFZJO4Ot2pPEz4xHasd9R6AmDRUnN7GcOYD0\\\\\\\/sHpmptaLt8lgLPkSVpxljQQFR0D3m0CQC8ExmbmlB2\\\\\\\/nSDJPBWFjRD3AQ3jMHVSvjVfI5yFigTnubV9Z230U1ZHAjnofa2SYIwEghS1W17P7VQuU\\\\\\\/1GS\\\\\\\/QeuTT6BCEkFOsb7MUDK52Vdp\\\\\\\/qXQeVJchCFgFCbI6Jd8tVdrlKjSEH+6T2t7TdAXP2L2ItHcjzq9ffCu\\\\\\\/pD5qXA71wM1JGa2Mt+rfUZjjOI7vUalfGcaY8YuIQxd7iwR7sC+sk6bqdBSdUHDdDJ7kzqd11s47ZDONOunef4\\\\\\\/3QyFHsxnEYqMBTILuUIfTnDOZKpq2sVRfU3BWrdFAAQ6oxVOtIqC9ffhMfESJHUUhBwzUdKsNSEJQOkMMD7KKQiZB9MupdTWqVJmICDaDg0hPM5G3iLQqt4cQyXAQRTDfSwu+ZxvZEpc6HPntFAfgB3unuZU1WGVqhXOcBVGkLH5OWPa0ZNOr4CMqv1D7xKVp9LeaDmf7p3HAAQX45eP8i\\\\\\\/imUpSBf6RSjqHeDPdSwXJ4hUHlepFmzbRUz\\\\\\\/EM\\\\\\\/hlYaUzZD77Mrexj2wsddIu0nO5lQZGGd60W58bTG6jDaKNx0Pn3CF9c9JPKc2iPx9ikz7mrSwpg3qfl5NG5uWHRCi6h5oq24x9L5WZ1rh8HXLzXxN1Wx7CeOCrr60JuPhLwiAFvbmO6O6HOmhXbYkiZsGPSqabitOElabm\\\\\\\/ZK0ojU2bG56MjiHEXxwgUtVaGvvDa3Np7xW6BB2sOSd+iCCvzz6TYUl2GbU4G\\\\\\\/X\\\\\\\/O3M3N9aoMts63uLnxiMSazMZ60tP+1CWlcpwHl\\\\\\\/iTG2UpaSkyIHKh3+g7JyvV6SYoA2aLLegdfv6waO0kHexFTrPB9erv2Jdmkljhceagk2BxyNVNgik\\\\\\\/c7Fpa0srDkH9yPMGa9wlMDwcWeAtdosmnDzTrpCtexRCwEFP9T6jFtYQM5HWGRIGaMt1IZeQCtVNttw6W2gEFskle3Uv7qH6r7PQijVRbvarJ\\\\\\\/yhuHi76HP7yZ\\\\\\\/nvnlFBJ5irQLNDii5JzSikakMlq8C6JR8TfEUewCbR8mYzI1W+TsHtok8\\\\\\\/lVJqF7zCk\\\u0022,\\\u0022iv\\\u0022:\\\u0022NmTImmdoyS6XY7bZqlFw7w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d7e25434b1c896063ea709050c60984926b02a8694019c5f2679cb10b99b9d12\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PpYWOCF2FghwvaAMeOwcjq9pZxAcbQiDDFcwOTjoINmwHLNarnqAiejd+kgEsz5MPBKfqof6elSZG4grCMHwcxAqPtIsncyM2j5qSvKEHXWsaFyN\\\\\\\/C9WlTn2nkwjlv0kvBYxrk8Oc2TmIzZvPIo7PO0+uQYkqfFqE7Cxwct\\\\\\\/P61tJ\\\\\\\/k8SBx2VpWVVwFqPN+1ZnKk222nzDzfMedNsw4ZinXBzCkGbFgUXpW84GA5IJcJBwrRWB\\\\\\\/Wk3naHDsTKmLzm0dfMewlIt69mdprpYKiVU\\\\\\\/teA4Pro6MQW+SClgk513d1HzjhwkUkotEzmdprgrIcwj35za4tMJ0lB3LeIEmuXgsVI9b7KSFmBdBYak4wjNOU48Jgl9bIPyjJUuZwA\\\\\\\/nqoO8WxIhcDiF8P+MArjKZ+Wj7A\\\\\\\/QqRMSnV0MjWX\\\\\\\/hwF2bp+E92VI04gv9QYl6R0vFddmxg+i9XTn4nBUrrJ+Z5Ul2DYfnrnQHHbzssWBtJEwjdVbGe7TnWXOpK6kHsbrf0tlFT0sfpFifOnIXKH9HmVFzcfzOGZABiFsEuEzuZ+MoeC78\\\\\\\/E3RFoN3e4pYpvuUsaLsoNSDeX7GKKfvScKPDnyOHaneTCNHxqlFwJQ\\\\\\\/rj0JXmDg3TE4WL\\\\\\\/+nr3TaORqwN5WuFKBfj2R3qHGlRp2lkhywnME4F73gP2WmekZXm7\\\\\\\/eM0SQuDVERWpGCmzBixeLNfv3E6pOgrBmpSRr1YdIbR2EvAhllwXrGc9ig2WNJMmHCe0doRk0iUnOnegPfMIGeKEJ2htpJaxF391KmdxYQrIx0J7+na2xmtiznvFRscLoE110+O\\\\\\\/ilk\\\\\\\/YGF8zAotEcix1Ip\\\\\\\/OkssytqnbUbxF+iV7ZWS1lBSYIXK6hplPyRSlOQ3Jq876QeBnQJkerQxZbyIBLQJ7PGcUzol9dIdn1MbxmdLOkLbASsdqrte2yojFtSX\\\\\\\/p7Mhf8NocsiKbfS6OsUs7Sxyc\\\\\\\/YW8LGBDVkCsyN+0AS555PaJdE8rNqyPQsIf+1\\\\\\\/Ifau10K1IYgQ75eeB0exyf2wLwqi7H3Gk3Rk3ogi0TcTZYwXaDNJw6C+LSKtVtsikgrUVygquTagFF\\\\\\\/aGufQmV7+rjOYD0jvLwumeZclfSNhK+Q2kFws0UPN9cPOFGrfyQq6y1\\\\\\\/94h0APnLIJ+fGX85Swhyf4kXPCNwr8gH3ZlRNIaPLOdHRJz7M4YE4YUawr4TlJnWK6H0JGwp5YQVtmYMswA5\\\\\\\/NPHQL9g4waQTdQ+grizMSkge6kCIEH8cPRmzl6ILNT9PIKd24FadwSxUyVqMA9ddvai9OuVWbUZoCaKqcpxGjVS29aNV+8AeKcThJ2iAmrACSxTmh6dHsVlvK0IT7wF3LNsNncD7IOopYiApS7FZkKh4lCR4YBe2Yxw0ONzpB7UyXyMa34D7finMWJcTM0h9+8\\\\\\\/oyuUMRbVAzduCMrjXDiPPV2eGRtKeFv34uTh1YM3fG6czCk2dsOw4Vs586NWJ+16ud6YbicR2niZ\\\\\\\/uaJsA2WFFQ+hzTYCp9tWSWOnoqxcbewI6hSSDcrEh34OFLjFQtvcqb6gSZXR9747KLWQmEHocH\\\\\\\/1C5vjWsX+\\\\\\\/IEukcr6CfLH9JTaNbjIeVeZaCq49h6CJIJC\\\\\\\/J2YjD0Ki5EPzlsl+X0OMjUblJLCJscPRMzU81a1PveDzH4x1EyCiFsizZLmh10j7KYpjsTpq1Mcvc09LuEnjoIf21SKT99NqnGCgkjoqkegaHk0ftTZqizw125HnSbizLpcZUpkBpLwen5EHT4jBYrAjsx0P42UeJA6AFJ7M\\\\\\\/7V3cIfOUrZfjKHow9afTXvRwOb\\\\\\\/z2U8kpezTl2rlhxbvgO3nkclXKaSNJsPzw9PhWLOD8077BpqhmsalRVn6tOzJl0Hw6hOCQHrQGKyWT0W0+THrVhLrtl9LsDMstV1eyk5M3oE6Zivnwfp8okchKajTacRAua8UG0d\\\\\\\/1Jv8r8nl3667FY2ODhbEl1KcApRkuuqqrAZ1hZqrV4sUsUjQizOLZHF8J4yvedf8xv9uZ+S2qHURDF3HsYiXAZoD10dm+LG\\\\\\\/XXOG630GZwZ2obI1xJ+orcD00oUtbqCvWWcdVWcQGtDE3xXhS3GExoJlI28rJybOflaUz8XLovQOn3e0DR1GjgHmjeLpEQagjtfW438gL5Emc4OytcG\\\\\\\/2eB0qL3pz3D5izxoz6C8gSIc1I\\\\\\\/jsKnCZh2ESlp+aSX846jaMRgZX2DnfchA94KK4lOOQE6fm24DP5lUBA4s\\\\\\\/DZkmpvdnAKAt2\\\\\\\/AyZ5kcqyJpRQ0lS+Nat6uGJUGV5uYqtOQKLv0oNWJhWT2oxPAgYEkG2UKehR6X6U+KJAHE92MyBHC7L1qoDUocdi0ZnUcSnsayC7F6Xxn+5E6T0k0Mlxs01wki+4M\\\\\\\/QUXy6DHTmBD4OcxA+SCfXcU4Hp\\\\\\\/D89IJLbMzrBoz9JhsOwsaGs7nVJjKwr\\\\\\\/m2y\\\\\\\/savyMCJZwC1wK\\\\\\\/zNB1Gf4YwL9TVdfpKA31ACz1BLzn0Rk5AZSZXZYaZ4breqnSyZYlcoADdWOjmnEY50QbVbZozX+Jlt0xvtR88WGpsfezp0j4XWwhM2ULH+IdQrdsf2fHWt0adhCFGghuqLUIPgzbzA2Svpxap\\\\\\\/DTljpJEDDDDxvA4mXArBmjY2+IQQ+7kHHTSrwTkwcABNPVGQarR8V1VeKyg1prwjdSvZx2UBqCCKU2SU6yVdxP6YFupnieXZofEhikjN\\\\\\\/5r0O4yWYlMqok2zS1GdtR4xP4MvnDznELDwstBfkSn6K81KuDuCZ9WP88Fh7L6Kgrndy5XQclMJpMYmtw6eD47rQvUtGAQSU2teJL1s4Q7y7ek5UEpCFI9MjEOweQL5\\\\\\\/jaSBcMnhrVbHGESEWjeGIYkXJokbP4ShREUrChdD2CufRapcs+dPcL8ToRlSt\\\\\\\/KO6DdFQpz8JARpvPtxox8JWVaBmic\\\\\\\/LLSPQxo6n1GfHg5HEdAyFBXCQ8vpeLssArYyuKshZQuRj9NTCL5UG6DD0+j2Iq7Zq8R8pAPM2TuAS4nItQUiO5WkZMOY6kGpXRtUZT\\\\\\\/knoW6TS89S53fsXMsYZFROafoTnkxGrUv\\\\\\\/W+i+JQsPMHRD5bet8mVZCA7Z45RZYAA4z+xznKmeWouOMPqyBK8NLVZKD1A7stMcFZZiIMjEqCvjG3XvgCFABZDzyvkZbx5giWvVGq4h1RUbUrwTg8cU4MdRn8GIJQ6Xap7ejHjTkTbKt6j2aXR0Z6qKtLFrCrR2NebN+qZrvacKT\\\\\\\/vxiseD3B3qX5oEmOEriJvikGcNfEL7Z\\\\\\\/wCs+CFDeLOkjgSoFS8v6QkQ09WDekLoGVZKJPsOvEYwudlpt4e5mCt05LA\\\\\\\/QkzB5R7Q+GcCMVMhqlzet6dlGvN6rfQ7rmDBOxU1SOK\\\\\\\/uOzzRIwF\\\\\\\/9MAH39mLpDHwKCG3zJJ6pyOJhACCA\\\\\\\/5olPi4RpL+LKGqQCGx6csAAFTbTqvdlc64v+7YiTf43llb7TKKOt1wKrAHwhB5Up90h\\\\\\\/0wfzzXygggAB5zLwPlsuJ0umLU7HrtyaoAOQSdBqmeUoNFP9UjIrPplDDAaRChPAENnZ8GkNdhCOa6z4AkBa6pJDozabOgBVYOIOyEJkoRoYAaPeFTZLwaPVgYRswLmMh1nnLnUsATIr3DkSYrpdyMScd\\\\\\\/dTSk7EeDUhZwE9XsRCC\\\\\\\/CUFr21HAANY2bkYFLGQTVSOuBMtRAX4cM7fCVDiEAOiI\\\\\\\/eXEJdhsAfGt\\\\\\\/i\\\\\\\/dJUDWpTK0JEkXzDn7JguJv9KduhLj1OoLKVW1UpdKXiqMidRH1MG1R+5HnZoSHM8vqyFJxPtqPAnpQ1F0XPSGSXajykR3lI39xB0Ccx2PAEwijMx7IxlGUPxMw327cUqgOqMjAHw3lOeQs7YRO9h5IRdZk2dRq056QIrNzzxF6t7T3OFzB0oE3xKqd17ldCQjlIpZECDor8C8xd2mxw\\\\\\\/VrbukHK0Wwv7k69yIRZIBmMmj77WAbcLFeUk2vS8jbTxCA7CWSOIN77G7B\\\\\\\/vmUjscVCxM0ZMVvI1KWZky6SN8EOqADS7fnzTec4EXWYD8q\\\\\\\/5T6bNyvvhAGHrDfwbVplsvnfEhLozegOy3e7A0QpggBzNC7FQZd0wrvDAgc37\\\\\\\/iuhD\\\\\\\/qb1FRM0vpDFCPXT45IsbaDfAgna25KRmOVao3lTpD3oZnMlZvlUud6\\\\\\\/Ni+e1qhhoLzaxd4QIkdjMWBnjeVTAT3zDaEAbN1e6d+S6N7Av5l7DJIASbH0o7ZJM+rcRlTTTG1T5E4cB6FmyyOAlL1eff5PRcDZLQ4ti0iUot6f\\\\\\\/YYy55psaH9PdUGGE9+ynkXiXDFeJ3C12dNJZIHyMBTQc\\\\\\\/vqYp0LJfX9Mt+RIZ20otXJrsWpFUB5wJaNyvRTzdXot4NTHTcIIzp1K7knDbkotfyFAbhTmj+6NUw+EKhp4smyIzVV7RONwESXEXR4zlBzlXjUByQGBid9cITkvTZOzpI123c17SnPr57o7NPKCTzXVECn16TrYvVUS+A\\\\\\\/LDEo65qgg6Y6iZaDGu9uFOwfN3WBk7kiXodzOJ6lZaQg0JvcE+haRvBbtDtFJXfuoKHTQ8SA+zXxHSJq6UP5bLIUZbQYh87xfS8WXkCIvnHEGtuS+Msmtiq2XyGcdwypbreFmg\\\\\\\/yKNsUnsasHcWv9ulTolnLN52IgL7JCGi7YkZcGPUMXx1zdMHVGJlrGlWdjtRzp2e8rBu7KRCCiCPy7dEpyShiqPEbB4BbyFPPs8d83sAq+y7ljnJKCVUhR2tfpnDD0zgq2C9q7WrhGpc1RgKgmTvt67ilOVL7L96kZADZXCOjVqWwZX4VfdyLtvaUNcqpCwwcM6mILmiEVI1tQsZi\\\\\\\/YVNbghXTyh0IcbXCiAabTXUHrI+M8DQStxumMRAtb1F3QPZroUurwbpFHCfxUBE15WTPkCqNYG3JQEwBkvIp8nPuGx6IzYKLOEfr4I1xNev+Cuwyo9kQ2djvnhoDALbrYI21ZEPh5i2Z85+IFjkYGcrkR46ZLL5W9\\\\\\\/b2PMVqd7RsbnDFDDTt6rxVS5FJqCjPTe+WnzmVQiIs+vY\\\\\\\/k1+fWFhvCz2LJT8Mtq6CFMCtzvHLbqDjjZvbQWBgpEzAFBI4fc3ZaVOkqeeBax9+v8Sc\\\\\\\/P6jY4jNNqpTPSTP08\\\\\\\/oR0WPkTPw1ZcdA0u7ABaQy8cXI7Fqnm7jqZL9Y3PQXl8HxlyP3DigBeJPV7kDRC3ss+6wPUznYkz67B\\\\\\\/LV0XxcUU3F3laf65\\\\\\\/6qRBokrAVRPhd+7Pik+yEgvvfYhnnewXgjRgvWP1rL7FRMqcZSTfmiw\\\\\\\/jAo1yY2P5aqf7y7CfJ3LdEXRFs2Af00tQW\\\\\\\/DQHYDtPqz2l\\\\\\\/HMysxarGfEocO+gOa348RWV7Xx5BGuOyodlf97\\\\\\\/NzBH4+c+pF8S4H2s0OU3kpKC4S+oJFFdxrDlyRzt66Y6pbfZORe4SJZq038p\\\\\\\/+K6EdqikIfRffnAuEU+5MMoz6iWR4HcB7Pt5F+eW\\\\\\\/YNLes4O7PstC5U7uDp3Bqe34HlyM5Dxlz6si8XTXgPedS5I9vMNAeQ1EzDUWyXN0cFVaViwwBOfy9t8fERBlL5SLBnWWLsVmngoCx8vJg7BbBPGWjKB3A0CgPjFeuahm1Jvn72sGnF2r0SfGEZ\\\\\\\/n2SdB2XwJNpPrgDQurJiryReqtsnmv1FpuY5vpD8rNXzur5qX5Jozux571mmippLH8Er7\\\\\\\/68INfCcLVa9r7FVj+xEyL8s5do7YeaD51pGE+cFdFAWU\\\\\\\/a5rn85TBbl7evMgK8vprhAwfcmwB35kxfZ1of8CigFfdVpkmXivHT3PutOB7ErC9ZKqxe2yR+I+BI18CW8r0GvGvbPIj5l9RGKjIcsYrc8ZYPo+tbxwS5NSA0Km6u2PGeXFeUT\\\\\\\/MuujBz\\\\\\\/6pebR3fB8+kwhPFjrry1xPC\\\\\\\/3JdNTWPoe\\\\\\\/9Bx9qxv7Ciq4d1FnX9y5e+T4rcnRW9wGNXJKZDL1K6rVMZDkdzKmbwLcorIyjaL+w+plnq28Dd0X7SWarWNAkw14ub3hOMIvAdvFkrGOMqsPwN71tLF4H2EJFgvtfEprePDjJ8G5mjCrqq9MB96NRKE3Rt4f5CwK78WWPwwxkCxWBDovOh3SbO7kOUQke9eZhRoTJDIHcIpheN6qGJb04fDLStluTdd4nU2lkiwt56e8Pl8+hq\\\\\\\/SjdTG4Y1I8zHJADWPQBojjtyEYGXkPk7VkhiwhAtzE6M638fcz2imPAZEvcu7N3yRYT7fYRyh02dotFO0+UEsE1rr8WeLPtk9nOdXIQ2JQOAmhLSuKg6ppudkYZzI240kbyO6Yxk1v+BHZ5zSmXasF3YAH+cSnqEAJ9g6z51cOC6U51\\\\\\\/DyQSQrgZKmcEYO9NYfMpGtndJ2lggKShq6Vpwla2UDkP2XCw6sATEgX1Em0Vx9L7LmXZ8IQihDlWjk4h3rEGOcvEtbXff35Z+iJWYcXRiGPkIgI0ZIJz+MPktjrIhD9iaGhUH2lrUnACUswZuZMwDHtnOfaZ9eTbXZlmHXC5h6iTFuz1p7xg3SZ037O4lB8UYvVEkTgbvBOu0WNbxIE0RS577TkijjaHRbDqYRvyVoSpYIEIia8mgxZbnIRqBf\\\\\\\/ZUvcfJnBXrWa6lUGC4Ca\\\\\\\/CHmLAEoFPy+dVEejH0HvkSM2HXQ5a00gZRQhvSS9Lf0v95afMM\\\\\\\/npafuTfVxuvdshl49uLK7HNdt7ZEbLmiDvzo1BGCLIl2p2sphWCvaNeiQMo8kvvDjAxqs8nZjLa\\\\\\\/cmHUGB0v0gOd7lg4roB\\\\\\\/PzUGupGuJByyagZJzHvNFvi+Ahjw\\\\\\\/IPmfyR5B6KoWuM2tJ9JN1GujcZpmC0GPdt1AO2WdUTkImZXDKE5IDkwTkGhEYFzRv0+UJQuTTMNsM11C9YSetNQHay9G9KaCu9r0CyXxavWq88o0tXqQ9sHiyMEEXz5QcGSlQaYtiB+Ga3B2RypVytG4Tl3ysKhBOuRY8SLZloNK+TALplr6So7jx26zDafNjS8n00wnwlJ\\\\\\\/cpAjXChkObUSFO\\\\\\\/9MSvgbLKDwF7w5KvfRLhmyV0tgFOQQz1+3WNelEUGbDuYcBek6hhuQZuUjdSgK5ITNntTg\\\\\\\/UcTlZ3+a5jSNQ2i0t\\\\\\\/Lfnl\\\\\\\/m0Py\\\\\\\/opEUM1a7D8CR3TR9yZJc8sVa+tg64gRhsx7iEpWkHFSlTfot7Bb4XAXvYmsBpqsLNMfoe6yVXiNX3I9BNTEYWx2qWfpK8XbtA8A9BAcviMpsQueCzXFR1I40qFwoKes+hx7iUiavyS\\\\\\\/qYr0shgO9mdTGjw4c4abgi+mX2ezKTCHTtlHic1fft7HrL6P\\\\\\\/xY+YkCcOny3TY2TVru+i8nTpaacv7ts+WKweq2uLzHcShd3QfLQcfMial2TItT\\\\\\\/LRFq4oa1c+cut1U7s3js3YsKDZ7oS8GY1kU4w0ilCH9asQCGOJe4fVrxC5SiiYmrdkax9JD7ITBfyOt+Y\\\\\\\/w1btTS0By8K39p3EZZjzcdw5XX483UYKyuQoFAHnlU\\\\\\\/AkHik0Bgj58F\\\\\\\/c5G8bHg+paPMTEaEqMYYR4acHyX8Yro4k4fjwOyPMu9urPWsAJsV1ymOWz79VwTl6ZaAS1PZ53T7H5ouwDF0g1BxcfvqsEykzsWo7XLL37enxwz\\\\\\\/G2szPJDnDwl6sDbxk4TehCaHeMkOF6g9ToncwiOwGdb2+b4h+RaAr0JUNb+1xp\\\\\\\/pbcrKcvWB3JsA4jwSCWsOwpwzM4SqfgG3V3IIFSmzHmlQfMh6i1k6bkPmzIj6hICw4R3PvyThkSHrXN8Q5g\\\\\\\/hDf1M9mIq0S0xlyTpllLQl+4SQEovoQGwxPURU8V\\\\\\\/rtsr25mkUIuexJy2Hb+\\\\\\\/5Ik3IAjxLYgNKLJZXKnlBghat+HEU+VpBZqSFdx+rF6J43HUUstokFnGrAwzwx1VgNf+AU5OLluLsa5ZuSW7uIGAYJEE7wEcFMQQqGp0axKb7l5DxoPHLXhHm8XVUf349NWD5MYeSIC\\\\\\\/ybRQ58EsgoOu9T3N9fPOXCLw0G+RbAVJGLS7\\\\\\\/hsDmJvOej3eBDX4+ybU1XlyApEUjVDGVtJx8SdnEB7aQ5FlGat2ZYl3X8qslPKMGhThYF5v5ruLrPRXCsD6dRG0xiwCSp3ZTquj1Oliz2AiBvEwV7nYj8AwltT9QtlYzWMvInnEzGYjLioybryiKjbJmT9uLRfdee\\\\\\\/zUqsh78+upRiN++AyAtiIT12C4bAh61lbojn4DU1SHbs+FSi+esiTeclJ0mu9XZH+5XhDK7bGEV7WHKcm6G+ugQ+9fe9KTft2fMvQcPm4\\\\\\\/0EO7tQesaJxq7tauqGLWW\\\\\\\/KsRRHcek1FrwlrVwqMG254HO1L1Mp1356xabP6xbQodr77pmKUaiHjzHss2cv1mVlFHWwuhKtGw2EaWXOJZM\\\\\\\/sNGSs1xJJT03LyItjF+8+haXwdr\\\\\\\/0M6Uo6ijpAkNDjdQaLiFtv4x0CtZCgXEeIwoEysiQhVXJDz\\\\\\\/zMuHRqEUPecpMFjcBWEtvRDjQKOAhF1A6fTuUac11c17ZSvigKBTYlNvo83vB4tA6EANKrPq\\\\\\\/i+X3IWJgd1jQOJ\\\\\\\/Ephody2lHhQiu6zPVjw4qaIo8hSMYDC6hrK24WHYYVnDa0L40X8VGOfD57Cu3ekXrPgeX9vUSpKPrI6kCJcMrOCNTLQ1ms\\\\\\\/OH3jToK2G77WvnLL0z2\\\\\\\/fEngg9zm+xv9UmurW\\\\\\\/6sN0uYu6uuAm6cqDIuM5wh7GRFqio400BFzwo\\\\\\\/X+eUu7O5z6Ml9vGTMZFd\\\\\\\/GfRbqxjMY3ZpWsDGezqI+ws7cb7ucSkC+rTBvSJRaJykZEscgaHBxgUww6hj9iQ\\\\\\\/s0jAmrfCDwQRSx\\\\\\\/rqIHyvv5zOb0TGh\\\\\\\/8ZjxW4A+\\\\\\\/MKKnKaT8zNnAE\\\\\\\/5Eac6iFzugFcWsqejFthjWJkpHOExa8IOf5Rc2cLL\\\\\\\/rpnncFoe38MU9g\\\\\\\/VnfqsC3wCHbas1TGU3KmDE9e+BZvTfzBjmRX\\\\\\\/xQU4b9U6VnkW0n7Zl3Vgm1fIMauUBqben0HZWaryhqwO7Ng+gamwbBGZ35KS2h7dRl3Xr\\\\\\\/l6fQRMjhd8iOaO6oWZBNNCSeie9aO3ayINblQztalFe2Arj6Gz0UBczZhkfZUyvLGcoEWOW7s9FzBwn1Kzysz88DqEqVT3kvRMpPCg4s\\\\\\\/69EPhmHJQicXgDpZc\\\\\\\/fL\\\\\\\/d5GQb9iErPVlIE4vQiVDHGPbR92YYj0YknEOi42vXDOSsQwayaVGs6a1hs\\\\\\\/CT8j8s8WWSkUrHeB3eLqmiDg1vUaZDbrblP7oTsxN8n4fC6ULsiUKSVoSNifQhhUza4D0mUyr3cntF40OQwoM4sMfj3EjeX8ZrD7jvNFykeJDUEoz+GI6Ze3vh9eXzasmKP+NcXV5pQEv4031oNP+BBuWH6Gyt8LQNUEaEHs+TuwC+A4KrSmax++sk2rVKtEmXRx3xRVIK6tkiX4FZgI9kWyyIZiWLecLsPtYmcroUWorx5+27kgUaGI2CdDRmH8oXCjZrSH8cY9X0va7hKvdI7aQoEUsBl4tHLG7K8njQc\\\\\\\/jtorI9CGV0tc5QtP5hJ5LtF8SAnaGw2E63fQtAP9N3c71OPT9fDqHdTWD1J6PeT\\\\\\\/iaQa2+ukV5Jl\\\\\\\/cV+WS3TCC+BecQOV0rBLe\\\\\\\/OmZ6LhyiA28hGynVhOlI87csI\\\\\\\/wEr+\\\\\\\/hdwKEPEx3GN0UMBVPWg\\\\\\\/AvjUDA2N1Ux4byxaMzETCz39if6uLDX\\\\\\\/LKGsg7P+Uc6wTCIbWvyqpG7rtxSC8zoT+p6W2vT18DYs7ePvCuQVDETHyaRseOalWgr++elmIt+d8HQXMf16uXVDlnrhKEWE+KGuX6qbqKWiD\\\\\\\/oKkuAA2E0+lcM3ONKm5PwM0i5A7JyJWErynbFoYNgnMUkP\\\\\\\/lyXLiRvLX\\\\\\\/W2RPbgJk2AAnoOCobTYsJucf8I4EqGDMlCZ3m6KZoJW2SxeB+xpL0IaTg8M0dL5B9oizeMs8KBYuf1xQ10NRqqTCS58xM2DzJJn4+RtB15n5gXWFzteO7HQslIu9huY6F2awWLb+ugcYhuxzMya1SKdnGRp9VrB\\\\\\\/6zsTk0OnR9HmUJM313j2SRU7I2mtkt5zd7zECcTbB8HOrKKZPIhISkKKnXeczqE3i3RCc\\\\\\\/V0hjpCq+BSq0MrkN3bVoarMDaaiLjnEDnAoHEEmZjIhJ5BEhhoXqJ7HfO9fYT18ocGeGqq8AmSgd4LfQFWoQhj5wbjGA00c5uktorHx7y9OYEre79rQ7pB3Mrq2Fap7TFkdQqQ2xSRhYYuvnXBD4XOIHMj0GAT98jdvy8H1g6HPksdC5vJEExh\\\\\\\/8CzO1HBIqpifJCOGzz1No4o8ZeR9auYjyM+ztgNhUw0eAZvSTNS6bBwwGI8QZ+jhvkrb42cDeewco4hG3ljHnbzOB4G0wdSqM7+Gl+W+FgP5KEVfWaZBIuLxc9vG+lEl4NnYAQgCNokp+9nZP+uTksurk2pjz0g+AaaFIirBzhN4k2t0llz1O1+znc2gXrhxxuQzu7WOuQbm8++a8ME3SnhQEjoNXB6+SqWeWdBmSKgaj1YKsCDca8XpLK8cBb+VuI4cwu1z2OicpTybvknFJTrq0gtLjMuD+OrofH\\\\\\\/MgLUOk0xVBT3qNV6mmxS31uBF4JdL56+++L+1KUheUQ30djaYMV+mv7gJ4MAX2XYxYSDF0WQOl94W\\\\\\\/hss894trwEVGLxDA3o88h+0UCAhqAMI7kIW6lw2cj31CgpAtANRIQDgaC3gcep1XRMR\\\\\\\/gEsl+Fn+pZhhQ3Mb3Wpr6SdMLotZLGiR4WF+xe3qdsoUS+kfgxKGJECSK3j3p9rHKRk1JKhTOaYXBm4MNN4ah7rEjFxS1FVUBDOGSdlmWsrISgyhzhjMSBRElZmJR4HvY4RxWDgk\\\\\\\/MuOnhjh3FqYObEICDDC84SxUs2uF6jgXSTyeRhIDSUW1vzjnJoMCJO\\\\\\\/mUG8hvReCk\\\\\\\/NGFysf1s4WvXPCrVHpT8JjHStJ5ObgEEMIrKaF7VprMrA9PfsjvjhQkD6o\\\\\\\/3kFJMv2+d6PIhXMPMZ+7itxORE1A1R76n1byaL+SPjT5HmTrk88jk09bJB+CVOi73M63giiqsHr7BnDHZKKPO44VcB5zdLIdvJIHpMIybITtrKsebS7JNujC+f\\\\\\\/1MCxBbh9Rq7astWER0Lpo2+kT5gsQrq9wpKa58FEnWgNg2WEuzSewq805hH4jIeQRPrR6ix8hzYeUCKUvNuKzA4nhPAcnKp8vQFJb7GDXPIxBRJuOLmDKsBLD1ztbkixn9VjD4UfFlBjTEqDWjYv2rBVavDCOljhupTNjjHojY0h5Cwd2vlSKVQH2mUMqyrWC18WLXvM4nb\\\\\\\/r4Fn3QjCcInXbptUFCk\\\\\\\/a6bG88kvRVkVhC\\\\\\\/iFOhj0s+SVXQVkUGx9kWlJLf2Ehgr9FjgRx4KH\\\\\\\/TZrbVo978gDr7BBJFhd1qZ0KT96oqoV9JHC7oc5CFjYU3nLwNFa+hsHhzivqfd+1K6MVD8DouHySvG7WwAlKPpIu1h1SAesZPWP8NLub6bT65EvrF96HlvudkKE3DROFqiM\\\\\\\/cn3prGRiyqbBLqHziVB3\\\\\\\/RBRnn8TyZYOxg22UuoTDzcGKy0hYmQRsiebEObjyxQdn6pd7z4mHrh9kbLnNDjr3df0bJSKlAiflToiKihbGa0i2ZipYaQZoplqogeSsQc6Y3GdH6aXij+mW74vCQMT4F8ticc1dcqKsa4LThXkTIe0K7pP17TCvY2pa37Abfd6It1v2XvW8HqlRuL69ozgmsHUtJfirRY2QkRw2OGaJlRIjd1g9+xtQ7v6X5nrOw\\\\\\\/Aew6pMmBTulI6QqngpnAU1hsEJuPq6s\\\\\\\/kHQ58xFK05HUMNRpQrSoIpHrKiIhQgkRwyLLcAhzehWtBXo1kDTBFJAeYT597MgPMcO+S37xXOq5b7gPSsJPn8GwKjIsEgrnAT\\\\\\\/qAnwp8zZMANdavs4qlDT\\\\\\\/Szrv5yNZEBcRyPSA+oaxBOFoIMGk+cEbnQs+HuqYO1Y7Thpq5ElBxVK2MAV+9mjJ3+Dze5XZe8fCqCTMk3UICkgzTGUvT5thJ6crCGBcDkARpnZPbnt3Yb2TmcYePcT51Mz3NAt2iM0mQnBBGwv08L0xWbm73YeFv\\\\\\\/kpFu0xJXnD7aVJLiosxgLSiGCv85odxZJZIJ7oaQ2N1nRNVKDBnGB4v+cKM20IPoLAs5YmvgQZjxrkeHMKj\\\\\\\/4cHYk\\\\\\\/clmUjxWx04QcCt8ov+l16Z+REXTBUfmVzBxUiKb6r6joPFVwW8rNid0JOEM7tfG32b+tu\\\\\\\/+oa\\\\\\\/5bHOxsssy8AsS37jqEbKfPfyPx43AVV+YJKXkjCn8G6w8OYx4ZGuh7HkC3Mj39RUBFmvx\\\\\\\/ydcWvDu0vs3B69v4fAa5mjEFqy993qPueHbKg5xuuq1dz19uGlmIRLFviBaVphB1N9\\\\\\\/W2kjwNTosx7\\\\\\\/x+81\\\\\\\/MmG4igg8r7WWt4pnrO81oJ+BLSOIfhhfqeWbbj3mLbxn1hJ5rmck60FIv+ZGYwye7YYzumADDQJWThcWOanPArPwILJ0+GLyOkUSlsU4ZWsmYd5xv2IXY6uAJs8RAUtiqVvlMGJ7\\\\\\\/5wlLmkJArB4rDsAPd6EBQHYfkmXl6MxBAGX9bj6FTFohbEScUzHPw\\\\\\\/HhdpFPwaBJ\\\\\\\/b4kLngjS4L1TX5KCQKdksa9tDV90HNzl6HwXBv8KRzlCQBiZmC+3bjK1jq0ovlb+oXb9K4gxs9nDYT+wp+tF7SO\\\\\\\/sIB1DLb9j6dShd7mVCOq+nfq\\\\\\\/ajLoHqgY9X4pTSs2zP5JExohUa5rP1iUvSL54csoaDKd2MU36AtsMa5XOKcVgpHT+mUsxubIOKvlSRSES+gqyHGWKEVNVefRSXPmHGDI7uIk3S7CbRv+Rw\\\\\\\/KUjLCuM2A8tMIObJbX6MmpEAnXgGU+fOMmSJgkhsh43\\\\\\\/7EeSUxaZpr5nQoZKo9dnAfJmU307fHfSncXIafQ\\\\\\\/EgQjjY8YzL5CNUxJva1StJc1t4gDQhORzdPsnH4b21fpzI5Gj7dG16GuWVKo8MlpD8PeTyVt3gfZ9\\\\\\\/AEVY\\\\\\\/VAj+U+7I9NsKQjNbX4h1u9rJWi4pbhuUV0ctJ2VAfI9zumzttDCw+dplwpr35pfIJzkT8rdEwwGmP550YTDqq\\\\\\\/f1N4HXSh8G6mIxnM6IeEtePmMZzEpdLInc89g6ZXDYdaHxuEucwAKkp41iwsKoKyi1UAjT4M6ddUs6gJysyoyyMs+5RXLa0\\\\\\\/xcA\\\\\\\/qlJuMVscwdSb3AY8Fz48Ba3ziTTpG8Vy0Oncjhl5mmZgFxkApmbvEKdAJmKXU3Z3tW27Rogkw9aKJTUxQtW134JXel63j7lzq+UqHuheZPNbmlc1w6tz13ZMMUzCi8cwAXXWyzKtD7fgo+30wu2z3uIBZP0nr8qUMPBnnrZgoT+lzw+jn9FDFtD5aQK2whq1ZXbHbgZfqrr8do4oaLSdSMfb3VheNlDcfF3MVulqzKHffEKgFbna5jPPo8uvRJJOd7xMoMYVlsQntFzYqfoxALRmT9K\\\\\\\/mMmo7vibMt\\\\\\\/umcwTo\\\\\\\/AFmuox3iXHjomakIp2k\\\\\\\/CJqcebBbRZykNtIILXRRWUq5nbukN0jGbqLc46+p0TO3puoOY8p67znRN856Xp53F3NPdouUVN\\\\\\\/q\\\\\\\/n0GrEraejADvhY83tyaCCotoJD9Q5sG6wRi4djA7bvImtlXS44YpaSwA3Gx2AsZJy+N0W27bx+1hZtoDQJTBP9eZnqFYdRGDlrEQc6WmifL7zbfZQ9DIvDi49PW6AjyFXLAQIkSldpE\\\\\\\/4rYxBx9SsfgbPhH4AzrIwBHd3WL2dP6ow9C2rEDOS6abv3+LfhEGd2O9J89S8+xsSj62VcgTtfAKjgZHOHKvgJiHe\\\\\\\/fHNX34NNkV31vr8ockFjlM+W1w\\\\\\\/0MkZMtG6WT3\\\\\\\/e6UHYXAdlHO4Ty0IW6UejyYZrBPcK7dW59OdETUf2qe6Qjt\\\\\\\/VPfjEkM\\\\\\\/YkEH7HkDm2WjU+ORRsepr7TxE3d33LEIdSAXWDjMuMnp\\\\\\\/RbIeGQmX6qNOUeiAFmuaHPYQnfxv2IYcDYbXe2S0TbBaaG0G7zMfj4qB2cCJh+hKJ7J5B7IjcRENVKy22Xbk7BhjToigv12twIeMyuab2uibUe7qQkKI7IdEI5P7xPmQtiruje+s72i40N2dZS7ssehTx6MIlOVGWZfawpo8VDdk13z6KTeoPKrNudnSo+1xaRhlBX7aHO1lf9zjzUU5yT8OdAU5i5IEkr4KXD9JJhbIw+7CBlICwrKpk2teV\\\\\\\/LRdE0KLvaV5h9IXBTazbNUtDwSXeZrmgyZjZ2CaWk8rffz2BaknlwNoKIJ5638QM3Ab8A6UTKCcikfyy\\\\\\\/g53NclE0gkiLx4ZUWbJZZul0vPGgLODbNkZeVMQa83VPEHOn8I9IZ4mZeiCNCotn3ysIPmV7\\\\\\\/fjHznml6WhArnYsW2ArzKYN88do6mhvoMFkXroAgGCqJYI5v98hAEwvHiSlAlVsQx6UP02JYSCnsEgjP3Sb423v7zR2XgNRunNg8T7gdnb0YdHEZUbHlDj\\\\\\\/aipUm5N3bzr343q3MgTc9LmEgXzpKhEb4K+VBMhYeelmXFdSDeqVjN0Qo3b68+1Ri4n4YxtS7CdxaeS7daf\\\\\\\/UL4xtsK9wiZayB7weuNgx1giUBUZu3+Y9Po+VkuwmmPVCpMrvN3NMkWWQhruFaHWdUgtWbxL4PUTvtfmoBKqoG0FtKKrqNcqIiwO9kPns7TiBtz2hXNvSZ4PRlkuORtWkjd8S\\\\\\\/WKPQjTDX+X3cFRjekXlpTOddRwGGE6hwkIiInLw2Fc1N0nHgZWamTPw1y3JA46i1PsNb0ClPq88NUX9813i+sHvRytGV0QVwuhP1bzQqtm7YKdGHvqmPt6dR1a2WJ+6nPhO8om5iWw0I6NNQdqIyiTmz3lZTF0SNP8dMhhmC3PbhPlWTJIwOYVmjz0+ITNlGACYAfsfTrcY2sW2hbhzMvmZV4GXBUkyjPG293DmhHtOH9sgMcJheUaH6AFhUViryhWC6+ua4mPIOJQmJkfnugQIK0pIfIPnJcZlMHB4eCgFFP\\\\\\\/8cqmbTPNeuVrjHffp28XQpAqMXIjRWc6WYNjFrUAO9gPifN9oes+y6mKt6VHKdrMo\\\\\\\/+MS8QTZk+Adjb9t8+m4O6nia8uhaLUGwS54A9Tj2hPxq9YEw4r4s+51oAV5TDfJeAk7UXj4Qcw1tMhtR9uYd8Um\\\\\\\/tuqN4nWxypLCWKpsnPblNXMgINtsNwB2OrsRphKxlj4Kq6i85QDrCXB+e\\\\\\\/xjdG\\\\\\\/ne0Yky9U+ceP54CJngMq1\\\\\\\/EGOlkN0J8AkN4rZjXGrPCG\\\\\\\/Qq9LuEfdI02h5AGsUKDRUFve9MOV2YT8vTVyQPXcv7nGoJgZmJoNdKEnX0y1Il1LoeyZXzOeZaKUw4bZ675ZVjvP\\\\\\\/juFpOxpdFHoMnmbb\\\\\\\/6O\\\\\\\/+5g6nBAjz7DPuM8rDYcpRvrtvmtAFjdqO2OMH0OYwjS27+FYHnrJppvdinZ2+AggDFt4lnNvNFleSUFmXhRAsV\\\\\\\/QTMJRWbdMtflv4mLdfh\\\\\\\/ij7NjugDtqZT7dJGu0KvM2lH3r+OGSp1BJaJk4avkFieSrGqB+3RsDpkC0yiT0PRp1\\\\\\\/YDgxxG23btDkjZf3TOKi0P0LsOjGVO+D6jYinsERdh4X1lcAY2AXQwxqf13qNOR5vCSSVcwIn1U9oVJbnYgtpi++hhTMSLiZ8RrRCuRZly1RIbkGJNutFtgl+eoRVuRb4mYosFhlPRhGUcisQ8d5FTwfYguIdmqm9y8oepls4Lejv0o23JFvV0edB3MIyWBfILKy7eguY3HjbmfyvQRMwVNfsEsxE\\\\\\\/K5WK1pJ5r577bJjc5t6uGR89gK6DXpO0+90rcPOlKDcU5Rgp66OnWWbligEZxgnvFkFMRZO1HgkzDJp4bdCDQ5zTwVECrz5bIWkHJPUr9q01\\\\\\\/UJ2E2uQk6xGp+WM7OFtYc+xfw3PgDKR8uwcaQDkkLkP\\\\\\\/HimQKyZqq03r\\\\\\\/kTZ4dFSXIlg2M0jpfXOrQm6N\\\\\\\/QQlG1JavkFnvrD0Ay3bay1jb1epRK3y0u5CJ7KHgPA+hqnUkCS3r7wlO40Pl1bNBqlXUeprTVnEKIpE7+t48z7reU\\\\\\\/HAieRb1IQ6CVfb82F5O+6Hg7D4SBshB8L1xX1T+WS6HyPkEfnrVFkIr\\\\\\\/w7DH7vW\\\\\\\/1oMSvTORS0b23JCAO0MEUQEfOwxSO94Pg+wUcpxT0zQVoYru0SHQzkRAz2f45wixQSH7wOgpSY5JA4XvGM4yaQRGI1+G1LIsV3\\\\\\\/hZL0b4iWmBN5ulTG7Ovyb8bVeWeNyrdS\\\\\\\/aD219xHViCqnmwG6rTCbaXmhu7agFBlsNH+h7xMvypOJKABixZyYlibyTDEro3lXwJGZt8xyo3BX3cWrgG0tyftATbtsVRNG3XjuZQsjdRQTxa8QLw4ixcETyk0C\\\\\\\/cTrfq1faUb\\\\\\\/dZxrI66igV5sDTO8BxrqDbHqYRcR1gIPiYLz42\\\\\\\/vM5toaoUuTo04bitJRQfsLgfIvc6zdW+v0fewCXJHPzqssqXljiVe+u3iP9TcIXJV56GzASm+T7qYy9B6USl8uOrYA1ZWMAjwPLoN5Waim4\\\\\\\/Y+mcuUUOeL6UrGcslJQFYE713om5rbBOArnkC8Rd9b1ONCgup1DzCw77TLeZaSdeWIqiz7NLEEZT5gz4nC\\\\\\\/vPKZNaFW\\\\\\\/MVgH13k8vrM73WUttUPQyRyIOnDsbY3cHfZkU28vjcWjNKeeIcBOcUCWEbsWv\\\\\\\/2r1E9jOxxeGirYrl1WM\\\\\\\/EylVDLpIBGUltBuwXpCqYByCHMUvleqGIQvQ9MImlB\\\\\\\/eEkQKtzRbKkeZnSzh3WioiriXh1kgoP+cDcq4O9FqcOU5acHdNAYDLZNNRx9bPOS94pJo3WGbHPlD5s13UWuIv7uvE8WUp5B7WmPOeY7X+6iymODqFefQcJT0Uv3QufngAn6Q9cBKY2AEVsSeKCLk8icqmvK6SFD3ET3yfZymNAnoqXBL9+4iKkr\\\\\\\/WXZNNbyfDbMs74RTnTesXfigWLUJRGpD1zb3yAGuAY19PB6mtyPNxLVZOMYvYEBVAGiZpQ814wLNHI8no8Az2mYfIogS1yCmDn2tdpN+zXl3UaepylWHQN1mMH1sYYLWtik2+rYm5QzqiuBdIWar464bLYSlLZwWZ5JYsJ5MlzE8qDs2V4wrZwbGYBQszwQVxHkcPHV7sqfjxWMVeR6ThloS2Yil7W0ZO28U79\\\\\\\/+d090TFlu1z4151mk9A4B34J+dB2A86JefO+OEAc6Yxd6i8M35jazAgT4xKPZsC2AqRTOEtPMjAK0\\\\\\\/sES9wc6zmq8gwoS4CS7l1HPdKStboyZKiMfWtGlCejlo9WjcQ0d2xh3wsZ7Dy\\\\\\\/cczMlKQYJ4n+E8P9Psnu33NHQbWGr0sCyyVAloOTGmI55SrYswsZKOKBh7AHfX2gDJfu1zEuAMkVhZW4Yq946\\\\\\\/qiHnkxXoiuEEfD7lbMneX0Af5tubTWqKWVF6VvceEVIeFw33A1lu3Jdj9TJBpN\\\\\\\/MK657pPGzLyH8jZrVjh0xrUx0T5WbN+rqRv91r9CxcEGOZ9YHHOTscRxxXYGaxk0nLQB0pWW9hjkCwbXdGTvB9wxurjegFCwzY9cEZBmsZJFTdKHkfsyZSN1hbtyqrtUhOUmE969KRfnra1tGlBibzYgB4NOBPLgNaGJhU0lqoBOab7F9EVYdY9MQSaocPuCYd+DpSw6sI8\\\\\\\/E9afEhixEVojQqg6biabEOTbpuuEZ+2SEcs+moIp5nXq4FgqpNDxsz9AK66hPJew6OeqYTtQiOgFxhcMDhNgmuCiv0DbXwXhAJi5VYEv81x3ZNm93TlQAjhylr4JARhbMCEbjJF6o06hvqsMpOxdHxUcoI7G8BaMLxtAfkAsKIWMb9bTuvx6yOgXQOTvgtSrylymKj7lIAaiqRiiRMdeULzuUU4KtSOKz2O\\\\\\\/erNSbuYXkTI9kZCZR0sz6R1gq\\\\\\\/PsrzjaJKYrKzOSQ2RQUTWNYknfP78nwHJXIW1M9358eTqPkaGxqD+BLU\\\\\\\/DhKJsRIau6PHBC9hY\\\\\\\/YCMB5LiOmG\\\\\\\/IBRACzwT\\\\\\\/r1crX\\\\\\\/RTjo+QgVAKirWAZAqUBmCawrGt9ZNQaKhW\\\\\\\/td5bejbYafkX0yxj3vAPD0LSN3EUSSffqrqERCE1Y6bfI+RuzcCVsWjEUb5Ao6AkrRT7SI39Gt9F3fXkiA6C71Cge4Qc1zJ3iD8O\\\\\\\/zrCrM1\\\\\\\/XZnZkBqW0KA8BqnFLuudxpH6SzOrtRnrYDnPE4vxB2ZIGFFxll5gJjWyWz8UIRO59h8xYzqOf5Ldz5cdd6xrOLeladAZ5WS5a4\\\\\\\/PNF8eioXDruAPrlYKcSOiR4LR\\\\\\\/RAFAsW+bPb33brMScSczETrxobERmboCT3ZmVA2zyuGtdX6LJ5dxZKA5heBADux79viS2lGWYSSfJqbMwtRf4TB2rFctVnqt+cJJ2Ehy69Hr2NmMW0TM6pLsW0Z1J8AqLPSJRxy41\\\\\\\/x1cSAEcc9Ey4sy2MW0X6+eqo7EGR+2HAqsTHuHnqXGlGzJUJ5F4tzK5QLjwQGjN0iSui\\\\\\\/oGUIkoBdJoD0g1IfIrMVmq5mCmThkCRjCJetqYhawGgr0cOSH4ovQJwO7SQdQfXjXSZxS2sWF8IFWOJZKayxtWikKkj+dOLQkLzjiXLgCfdeBXYDCuAHXk0+RZ5hVe25n46INBtv2t6bWqnJ8U2WH0W\\\\\\\/sweKOSfvoo9q25BdkQR3wHeixexz8bfejrPhQ56+Yh1FbI4\\\\\\\/VqkX\\\\\\\/cave0xhb4i3NvoLWTT7pIb5NpZJARyKGmKZLySvdsVR8A4Nt8Zqfyef0wpaH4IqxViqMk9dVe5J7iVf3AGWMDT8FOtdhhcNgkDoEn8xE\\\\\\\/Agt+4a77TYKjaGkP1jUjPbd0BKY2XPYiKa9tcPhh8Kk6UoA6RLKuUzwHdMQzIOJS0i5Uqjg3EPviM+aUg4GXt57mhNkGxRKQEGE1rw90h9igeSj6KPWQqVB9h1XTrV+f0AVCVm6kk0RTHnimsIgkV7tkC5X5h3SueqCxvZjRJzWzCBk917QI8ZcCZtzm4GWkI4ZRs7wKY7mt510dgaKnectqW0e0GfD3zhLq3Nvk+sD4FZue0JiRb9tLcYUu41tPLfGUBnxifvpP1sRwoZdDuRVFdV09C1+9OoTjfTOFcUpgvs\\\\\\\/S2ckmC59+NXYQlyu4d40gEKJQtc52Kc2SRjPa2a7HwPA9j+SVeAKxPjVItlk\\\\\\\/RhTjPPkCX\\\\\\\/Az282zKFXPxGWDVxM2fnc\\\\\\\/FPRC1p0zu2zgY6OLFgeyFUVpWdOa4W0gx5OdrlgaC8VLO88EMu+XeV2qaA35Qx4CSTqpIGYcOVbpHhfHvO\\\\\\\/KIwS\\\\\\\/0FJ0YKmXB9jn8l22DpYW9g4ZtZBz0C0oeDXgOJHWVyDtjYn116ix1LtXN6ltC8EkdUfaoBYs8x6sz2BOvfcr0r5DVGQ16xarWwaezL5kfzDOP80XA0W9o1NGGCIHKrEh5iuY7EKP\\\\\\\/MU5ms8ETVUnE13d69Uy5+gC2\\\\\\\/OBifFQczP7zU6lXXs9B60tLzkFtWGj7R+5ZzcfygPQIfvCa2MBNQwgdPHcVbW3dqJwz1RqonFcBgwwuhQ4\\\\\\\/RdMvhJ5pLCw+9hfJX7Bic3\\\\\\\/C5Gi1wUIRT9NTaW6WKsnEB0N5dYmhfpBu7hiZaz6yBQES4vYiLfvZLl3O0syejszLaHy6s46gYBEniF4WvS8kCrv41Mv9eN2scQbugvgu\\\\\\\/49EumCOjgyotk\\\\\\\/wctIONrOR6Dgfc7Ra9eCPrElegGMJdTp2htW4j+GBrFf+FWPmUhalJcaqh5slZa08dOw\\\\\\\/7yk08EHL2P9E1XHx+vowW2iDALTQsshs2vfB8g18JT6lUXs7rE0K9mJGA4KtlU5QVIACATp7U8nDAkA+B\\\\\\\/vXirr3DrwtBPebNeW2E1CLWRk0u7iNurbkNJwGnTN5x3rDePS7rfLIlpoKfbmtfEGdTiDd\\\\\\\/AfpESuDcXOSBAuE4w1VERmtADvIrovZeCzy4v7E4AvzyuXFmEY7rby02D6OJIZobxjEQrV2Edauloqzlu5bJaeUSnOQjP0SrE\\\\\\\/85Y8UjxqjWREtSam6zetgWvi16m6slcwUxkvnGigPTmNJ2fsn1yngI\\\\\\\/rO1dLAkAdwx7IAVHPwmX3BYc6WbANg1te89QG9dLmsaDFb58DN\\\\\\\/wT9Fq2ANpYuvcvAjEQ+\\\\\\\/kBWcOiFJ3Y9DorTvtIEvAgB\\\\\\\/Vx7Mr19n5YIZoCBl0Dlh9PXIA7r\\\\\\\/a1raekklBcCKpDs6M8ppPf\\\\\\\/+vnVsMnR+Jxl6fvgpo5lx1BMdzmUk6Vr0eekiX0vzhxd84xdGrH+2jF28RGLoYLpPCY1Yiq3ACNiVHqeEDWWNF4gRurFe6vZoPbvU5W2TrMWe4khCTPTlCmdWDjq6hLI\\\\\\\/M\\\\\\\/xCl6jWd5gNlHXhZ64ZmlIRnOX0N4srgCWRDZi9LQvpoz3g9Bcc1PSZX4jpeA75SQRzYEFVMVmbZDqDRRHK5NWPCUA\\\\\\\/kGlMw34KgxBTHq2NyiPmFL5j2bK3WnXGnAzNnrjXiwFSebzWlv4X62gzjcEtc5qUvnT2JPCcM+PHUsfmvhKBLGzlaBI9nL9C0Mf8I5F2yQYHIs6wcLw3HWF7rBKG++Ywc+xDX8DAJYdhq05ZcAEpn6ZQVS4stmyrIbRy\\\\\\\/\\\\\\\/YgpU5Hh\\\\\\\/7sAZtGdRyIB1fL6FGrvAsyOT3sW5NP+09XkMN\\\\\\\/uMBRMsd6xUIulVEb+s+EVLgygLEwNYrK3boqMDcEfqeSdOJ5pVLjZxyiF+KZ3JrY7VG77Cb9k8kvJLU8QlXc5TAr2p76PkPUz6friS1PEQxhzc2\\\\\\\/pGDmtvypa+Ay1lFQKbYq5NqRWZ07yiBeXneOcMNG63N5Etv5yPAxewMi5IeKKFQDngglD17YSfZ5LuKYfQ0pQ4DyzQWmuu0Sr1PyO2MwlmCl+BCJ8OP0DhD\\\\\\\/1s2jw3kabVEYoFBtO7lN0NsZRgTq28oQvaXupxMFnLf7G65Srs2+IJFMvRRp\\\\\\\/EvLjxurUm0ptiZK05TE+Lyji2u42ROXUTIKYql\\\\\\\/Gne5kbTNQgQitRbMV+QzO2lkB9zOMgwhpYV+zqCKRj7M3ehw5CdzbmHfZM5VjJ4\\\\\\\/8TsJmO\\\\\\\/uj335DyaM1L3MbMy2M9VDkZjyCLBVOdSFsULpHxqO5yCX4hL7ecL2Uos08Q4grAh6wZG4w4oQbdK\\\\\\\/9QoGb8CDlXF+fKo3tvN5pZsKx1R8++XxWac8clVQJz0CTPSLdmcwfEB7W1DOv770B3vVw7peI++l5VAHsoNBpxkJ3BIJjvLflV1vAAi7Sq0MoX+Hhsea33QL9znM43LdUTnxopdmJ2n2YiAhLk60eg9KY9TSxHqPRf2fqwRnYK55wISw0bGOIWTtHIlTHJIHDcn5WCN8lZ2oee2IjOrcRdhaB5hQsXE9YmNSTEcOSzeVCs9kSjs9fT5I43xZ06vMgyNH8bC\\\\\\\/YmkW+rKebusthCOEJXsZyzdDSejQHBNGUiB9djGbzrgQTPsEROqRLVfSL1+Kv3ddZk4H6X5IE82NU10b43yblYWLEu6eCigbyLQ5T0EQt+HuunfEpBsVDRZTZXNWW+jfAIZlwFErHhOn+gFfeul3BlMYD5HxwwkRgy3cUuMqGCv4JFrqqWmFSuEsU1MTXJlERrEadDrhyO20O0pmVIEefxRxDGcShtXUYZz54AAnRtLD021A1Z8x5xDC8uTcJ2KFKrQUQ8Up+xyVOorEV7+ixYRlSLwr1BTOtACk2L0o1rNppjw9+qGaghp28QBO3x+kOi\\\\\\\/Druw09+G9\\\\\\\/asD\\\\\\\/+VrDDwMaX82Pj3UcE6LpT\\\\\\\/UL9uHK+FCAzqJwKZ42jYAQCRDfsrlcuVwAkXW8XY9iNZlyyqVde\\\\\\\/yRwB5WdRt6aKakQ\\\\\\\/fY46Jzt8ncpAUhMPgMv9OMOGZuOTIps0gxfF3V+BoixxhV3tZXPUbLJLje1ZSCJm0vc2XocC2QNJHK+uN9hyjRJVk3m897L\\\\\\\/DGOGrBKB4H4ILJQH\\\\\\\/AV+jCnveNpMuIV85U5R5nA+qyhjUfBFUBzAdI7uezZjCoxWn\\\\\\\/2Jn5navC7\\\\\\\/t2xFg5QYhjp3I27acBPGltKFsWEbanNqQL8IyYjD\\\\\\\/Ifr+9qEhSi2t6Wdtvbq5D1MT8fjl94seYscXL0YyYAWgc+PGBOsotz4RjY73X5LNE+dnyIZQA8pYu4Ztcu1l9wTLFaRMwSO1iUO\\\\\\\/jqtWz3H\\\\\\\/eu0WLvm8t2SAWYbBPuHzCGopOvz6rtgt0h3aFlcEfVLTVKVl7dUPDgVY08F6STVEuM5RjH77gx7efbIr292pxrC5uMizkUN+i7PoVfVPrPMLKhf6nLEj3XWQ97ig9KXLh63D+OYbC9NWJ8xj6oCtTMKYSSLtKebijukTh9YTDP6AGkrzBAUJUt\\\\\\\/HeU43R4ntUn9cHp\\\\\\\/mdD\\\\\\\/1jUHFLRzsOlmYQTNKu32ZPYJszKa1bQUKtmuBwoA0TP1lU8SW92kFpQHic+TlvrUiExVd805hYJXtmBF2nPxR63H9v2zGnCKw7tQ7ZyFeC9ZHb6RieiWLFajHJhFV3hz6cZAXvtUme1JUbGSMKc8dQI6ZEgjKUjJxxLeXoM6Z8SS8QOpFl7dbgthzgHmO810l2DwFPso4y3GsX3JRikAB\\\\\\\/OPr4\\\\\\\/dW\\\\\\\/SfnINP3ajGrB911aa1Pw0neaTWzjm+rdDZkye5AbjY7yxBCtyWbiIjRwn5QA7pAlJV4MRrwnQK39nDeoUquU\\\\\\\/jgodAjZnWmeXHp+Js8TCgf8FaRbvFd6pzz5C+yd5jsGAWXla6sJt9LWOImRSS1UUay2eySSwJT4d6p38UrjuyY6g0NpkzmekFj8R+U6d8i3M4Dg24f\\\\\\\/3KWZ+l9D4vZ6Cb9\\\\\\\/MCIY\\\\\\\/eVt8fR5a8866Up16ZYcF9hK4mSUrEE0eKCE7\\\\\\\/FNsDvM2UFUs3aYJfVdmLkPTyjVwBFiM3OVeKzpIabizpcK7Jts2a5\\\\\\\/KoQW\\\\\\\/e6QnrL3nO9U\\\\\\\/3TNUxiD0nd8vT+H+Z1y+VzEBvNP8pgRz0bDfTVKTiUnP\\\\\\\/Ag1aV55DKEUSNlD9Dj6FWNGWnBlPmeUZh4T\\\\\\\/5v0WVhNudUJXzb10s7\\\\\\\/ihY9r0GbG3zTVqVYRVAFat20znHpxIXgsuBUXyUTzGGCJpD7x0LQrFLOAJal\\\\\\\/7kyBvd7rns6sMaw1Uila13KmrRJuwESG1I1ja350kbCiPgr5kuliOmLzccUOmiTITKP1b9kgiIYx1UMt99bJGWWzrLOXYm2qygDjWR+mW59+O3ULVpOvYQOrhL2AVpziOoBCOBiGjYHpuIhYbayJSyx4\\\\\\\/Aso6hhIc1Swwzv9ucsE4jran3X6RjCMeikty6pi0aeSlqGP1NimxZVLrrdd299zwIxCZbnbziBG\\\\\\\/czBENpaIhWf8TDe93eaNPe8TTjhvk9bUWmfspGisuGw\\\\\\\/WGFsyJRSYdq9riVFH4CCFYmfKAILsPIAaR8JT0lgT4Ixy5VsE1ZPW5zPbDl\\\\\\\/5mrSnxqYFU32F2au8L5Y17qyyuwoBPjyHwtp\\\\\\\/sMoZ4abFWOBD8sbMsSvgd5hcHbjVhPnAX6PB\\\\\\\/padKijj53weh11AQNq626eSbJl1N3qdXshGyNxEmnPSbx3r3di\\\\\\\/LG8q0If\\\\\\\/xv4KqYh6FNdgucfkovLsjlYdzblnnYGYBPCpp1McJNgh0sVuJymRGA9TqSbJAH8LDmCuw8rydbQiWvq5TmQcSrpEU7FrmjEQ2ZNEk4f20SsYjJZ5NnFAbFFJMAfuKKNWb3WDovUBv861O8qltVf1VpB0mM50gXyf\\\\\\\/EJkpUWWZboJ73Mm\\\\\\\/hKCsAf7e59aky5A0PgFZno\\\\\\\/8\\\\\\\/0P7i9aJA9OwQG\\\\\\\/mpTNA85RZvE0AU2bZXHlQQmggJVRBq+SqmWOMdUvC3eO8OLwuLWrUO7Yf2jmqgXpRBwafe7ycXUPLrcm9Mx9LIhKQ6a\\\\\\\/PVCUrbN51Xp7rhCS3tlecjKvamLL6Meutw3asfiHIKA+XTLTeuXUlXMzcgibszTnEMmhFsJUpCZB0LADvV4iBGNWPpjKjK2szH50ZdQZ1Gcjhgv38M3G1od1HfOCdRmjvE12RPFZMnH5HMWgPKMTqf5eLZ7DoupBQo\\\\\\\/+REHuvJeAwx+9\\\\\\\/ntbed25rHfTIA7GhIj+lrZODQ3BlskDk4RpQ9hWzXBVu1D+H+tc1Tw1Zjb6sRgazIH3UudAcnJqY9UvcszaarXLua2bwXR2V6obZlW0KpOGlWlFq1bR5v2ba2m4cy94LP8o5j7FGhOXIk5M7ao7gwYYGdJ5M7iCHakiNKdBI3Cmev1TMadE\\\\\\\/ihN8bwGEGoyJ9kVWKKmZOylIAi6phpyDRzrd0mZMh6DYf4sFv++HAUSeeGetZDiZG6vwkmFg2cvLpUut9H2e12ViCjdMP1fV3mgJT3SA2\\\\\\\/cVtCJg+SYn+X56PZs0QnceaiSBlSWYdjYexn+FzAjt4vMgatydAn5cMm2i4bvj72t2c34fwLVPTKtiyhe0fS20sLRClsTV0B4L6oxDa9wx1MP+jxqZKnjdasbjDIdUlqXPGyfE+fsvce4OFfTlv3TSZoPME6JnfL6BBqHPIcBEmI9fKH0J1P8cePDhbZyb\\\\\\\/HRIho+\\\\\\\/Sn3rHFAMOxryVH44li1X3WL1Uvkoh4wmkjFJQJn7h7KOUTYXFJSfd2N6cGL4\\\\\\\/GucqbFBwHi7zbZ467jh0jW2cv8xIur8gaj3MdqN6MP9isxyX8BDFkTP6l3LMzsVaZpJomtNiEh9dpkYoEEh\\\\\\\/aIWtp6301b\\\\\\\/ibvQtjKuHccGIRsxaEqIkn1LW6lsHQjQMC8D5vije8s7JZ3ALLd3+0kr5TGXETTequtKP5AgZUjIIHhzC\\\\\\\/j7Ri2NNHwKNmkR2hfsIUlbfCLdBs+zx8Od3xaKdXtp6xEmuULunVgteSlwg8QdFJDAlQrn2Nz4IlgUs0jrlQ1iFsGSJwHCGuyUOBkKyrAd+kF6kq\\\\\\\/2Cq\\\\\\\/TIZxXT3Z0NsVTck4cNEQTItIcNiEYqbpP740vzxm6LNTmXZZ2GyWzLXPIUsLp7vnpTZNv7Jzlfd3XtgG8GZ\\\\\\\/Cg3MCAnDs4wHNL8tvXV9oBekq8g659KNrMn0pjBOBOOCk2ViCgiIKGfcIxaOn7SlAVKuysBU4PI1FODAjWTOt\\\\\\\/arXT+fsRl\\\\\\\/S24dyz8CkYzA7a9pJ86b2F9hhxLHXchm33+sPHQeLblKtwlU6wY09Hbc7SGZuLq6Y+eWilgCt3eGtZBzx7mCe1w5MbIRV838+atcJ4RdZSEC+rypq5rLp6w5SqJ7a5Op6U\\\\\\\/xzKXIhRo\\\\\\\/FcFzM8LyBgHwrbucBl4Sfwa2j55K2rr2irUxpr6ybi\\\\\\\/9wSuzgzmLtrrftLWl5sf074jfIci0c2PENs70r3j1JgQ4CKvDi8mYZQAD8Aha9frqUOv4ZA\\\\\\\/0QK0I37hTXnlIrskzXipkNSD40KZvYsV1xDAklCro6gSdG8Is4bDaVK8BDkCb5YdrJjo3riJK+KYzB3cKELYWtt8uhd\\\\\\\/nN92G6LJg\\\\\\\/tUi413TmzOJq9SrWliVSCw2or1Y7OeR3hIOI3WcWz1jJIRxh74rjd8PTKhVUEnE5GmdTa\\\\\\\/kBO3y874yoEwniVDyDpJkPvkuq7K5\\\\\\\/A0d3TTpm8LyV7RPxfK9lwCqlFSQjRo5aeSTG1Ltd4eTV\\\\\\\/wy4jl801ynmNri5aP819isZTE5jewQVmuQw1hG1IUEc6l03FrnI3Wb1Ia2wJBpu1wPx\\\\\\\/4SGHXHL\\\\\\\/0iC\\\\\\\/iFEnZmddNQhDCR5DhMF42vr6SY94mCvg0ESZWfUjGkLEIwM5n6hAnJ588b8LcuASqXVO531I+wP7MCZhN1U6RHuQN4lUn9Yk+cNLocUnv5B\\\\\\\/79bkUeSXR3tkkmYsf28YFX5j76Iiq9wYAa62NyHuuV6Yv5E++qMw7JKVfDfIjXE4rZqve0in8dvFEwTfsCXZhJGVpdAY93cxoG9yM8QOnQ91eaTLo7Lz9uDGcx1JWDpduNPq2oiE9pSBDMo6PNruhYkQ7vZDb9CFIuMQtElDScfxRA0sgDmaWVkatu7kc2Agc\\\\\\\/wTpKqpYDM9tAVHm9hMnZ3sTTzF2TZxwFnARPZzxPao1WPYctOX7amoiFMqoYmCJ6OvaIeAYw0hGzvKVa\\\\\\\/QkSvCVct4d4VBzYrMpbrXrtvY5CQ1YFXJzl1L8oqOz23VXbhhCpOymSnPgYOlNJRgacPx+S2jamOGIxvi+nOJ8PWHFOe\\\\\\\/hASjhGfF6LU3bssPVh5VqMDT5NiNXIsftXUFB+XlUSPRHHIDOs+aHqRI9Yo+P4BMe+5+1lHkcbipvvs1wjHst3VdYC\\\\\\\/\\\\\\\/BxZlZdDvii7m4rIVD0Mm1kGTBBpm7g7Lw\\\\\\\/O8+jOXoHb8R1Syq8tsg0OrwmznRlg00ToAO6IN7INbOQakgoG7oJh9Fx7mslsd\\\\\\\/hEdTizjf+AyE3H49Vh4gw+t1T+smZXU9f1QhDNMko4kO4lwEIDDuEnOyGdIs5iJLySXYBHbKt9JcmSFkIVi62FA95RwQy04IgegBH3d+HgGnTfKer8VqUoGAqzV7V2efiRJMKrpq3NfR2K5\\\\\\\/wZpL\\\\\\\/pI6c1oB1gJqeOEJWkXK5\\\\\\\/wXatgT6wsl8wo7k44BTpELE\\\\\\\/dxboLzPnWy8cFKdwVhUZfXFQwTCVQlD0WDJcBdYrijecdBr8VaiI4+Sx89ESf7JFu3+zhtZCSDeBjZcOKZrAwcVVo6SpLi+Y7zZhXKiHy77VKk2xPTLXggo346NX+3C7Biq7aFim0ZfX+uIxJrabPzk27\\\\\\\/eeYr6JoMoB9YpXs1cHWC\\\\\\\/78w8ihikUWcLCQNJfNgpaffnBshRiRF+e3V24gVE+uNR8c77rcooa4RWoyYH3H2Z6R4VjTdKTVNYm1WUYCAIzaN9FgM3yIjrpW8s+TLhQqDVpVnuoek8mNmnRYEsfawe8frHQM0gPLIgAL5LcfWV0rQ+etiNLZM+gv506RDNZ3hq+i74yAqkZmNNTnGkPAqcRl7+Sl1r6YFJxkoC\\\\\\\/2jWpxuZq903PaNXtYmWGcr7CT1vPkaWWfAVqtAMY4lttnIqSN0DK0zTh3voD0mPekCfBsz9dMfcjVHGAtlnCwz9wgGOvp\\\\\\\/0lNHDqBaQeI7q8uLCAI9GqjyHDI4hvqF3BgM0\\\\\\\/BbJOLClMyJz+MYs7pgl6LqEVNmuNdlyrl8Qgz\\\\\\\/\\\\\\\/G8ze8luJwRhT3nml6cgT9my6uMkVQ11exW\\\\\\\/ypETsGlcLyAPQT+poSkny\\\\\\\/A1Tq2ZPvOi2gngRebn2M8rBEUSshUkXSC8tZASlukiN48InvZDVBVSBRWPpTClhRY2fwyINEl4dRzt3PCtmxJMc0qf0U+ciAwC4ArCCx01YvvkZO+W6uHMT8lN+4gjjGp1YsAn+rMosm6YmQcrXUJO3dGldwJOZc1YR+2VBeAShHrKqObNrFkDTzqjmy5NhbxUTRqjzEmMP6YYp6j0\\\\\\\/2Ujphv7XXrTpPVrI4QpfJSsau7j5HC09XUO9DSWWWYxFq+6NBNftDaCc3Tql1OVl+3UpXdq031OpMnY1cbVq24hrqL9h7t8aiCIRRQZNMOPepu0Dr1pScEXMFHiJF4+Vaw8no+zUUYtqxJFf0Q7vul0jfNj5tsPCUSvCxW+WotBAbJ7DN2hxaakI2a\\\\\\\/wk97neisdaU7dGi3sjyBQApS8XGeW\\\\\\\/7VXWVgrIASZB2KDexkJjBZZZm+cqxEZBicpIvrY\\\\\\\/7QcwsLyIBZpY83hPHfXp4zXXEJBIQfBYp2d6xHMd7dckXipSkn2MSA9D0FCO\\\\\\\/uYbtZatp7sm+X0gELhvmaAJTqYCs1DlG+Yru+vgfkhDcMiGEQuBq\\\\\\\/4rLU33rDXM\\\\\\\/U1gOp5tcSGSWZSIpjTiJC3oQU92y\\\\\\\/XPKVo+jcwnjctg9ee\\\\\\\/G+h0dKWDSLyDb+OkEdu+0qCeWC\\\\\\\/DDW8J\\\\\\\/dh37wGyle2088Qt8R1B2ac5F4aS5ag0dAyRoOrBbC3a5CBGphd10U1BRX0UZXsex5YmRDQYoZwFbZ+ecJ0tN0jIGIAW7LakVzHXaubPg4lCmObqCPQHIlE9wo6BK7BdMOKjOoRhBecG7tsLDIrJTaEi4gv\\\\\\\/naLr89II8CCn+q2yTSkPReNE+U\\\\\\\/56Moi1b7hKLA5UJi81VZvuLGGUfr\\\u0022,\\\u0022iv\\\u0022:\\\u0022kf0x0NyiDbyPcV+nopLPRA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e0a9e263ae0eef2ba77eb1b301b8cab59800e9d299ac03da49fdb2c3418c5cad\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022IQRrAcZU6iVz2lqZIM5J3lEaN0qignQ7QwCSASh967FcCP7pfdOwjWOtcJMIj5NVGPKLKDTl9jv0t8dX\\\\\\\/4ZEy\\\\\\\/NXkEgNzd\\\\\\\/2\\\\\\\/yLwHt7j3zHK5iEGw9hjlwVhZ91G\\\\\\\/qNtG+mWQbLT4hRttoa9aMKCcMCb9UIBNr0eU7SALzbF4uXGZNMglQxLc2k\\\\\\\/kpgqIGaXMGaYYhTKS6C3mL8ZqfPq5I\\\\\\\/O8hIjbWFxSYYcV2IjES38EH0lP+58mJhQw6xJQ+2cM5CIjtECS75vDZHWRvXRPHM9lVq+jfpH0YxNPWfsDUtHWRtF\\\\\\\/WKI5UyOWicv9wJ96oOktuLJWk9ageIBV7ZB4guLWt\\\\\\\/bbQEUkqZeJ4NC8Yj88UC94cn+anmqOLZLJQT93OSqWilpgG9AEWN1HuXjtrZdTrJ+1BBYPt9zaJuifFaIRSAXANUbDLHvyGjzjnoBFxHHB\\\\\\\/vDJ\\\\\\\/ceuFUdvvXpZmxIEni+OqdFRiY2+80Xs0DR1G4PnYpAxVauVb8u5wU2C0hKPZTtlsOYRlHYI2Uxr1kORtMWHzFPle4Ol84U6Ts4p5wbqTjEFqBY6A5E\\\\\\\/SA8ECdPuQ1CjxE0b1nGY2smdDB9b47VwdzRwM10DM4WHL2DVY47dIQ1gRFAl5ncmBIY0b1Kjm8jQIClW2AQxGkYniYeEzTmL073nkgvwXjYR9tQxYlxTcV+nFD\\\\\\\/wJA+ep2DLjaZJhqe9yqu8DZEz5MpbGqwVFTyMPBOUC3fQsUc5XZhzQiX4IvrY\\\\\\\/RUJC7CCaRJTkOa7KLYk3wyWR0\\\\\\\/kw68IF9Jmd\\\\\\\/h7qWBqKLkrqsGjgnL7KyxN3L8GGS3CHwTlni5MWg5+MLQF8tjRUVKg+otqk3qIu4nRKrSQs\\\\\\\/0gy6KPL1JtMlaqFaWo1DuY\\\\\\\/2mRb3GaAQiGkWD\\\\\\\/LNja4Egby2POFp2R\\\\\\\/TVcZH1mC0mQzV9WIXUJk2P7b6uhlne234hQ+5cXe3Z6U1b4CKXxrPX8UNwMlt2\\\\\\\/tXtXKS1DpMnrTeaIq4dvnUcfgV\\\\\\\/40pEQhucUUEIFkXJNs3CNsBIN8TxY39NGdNoljG52EDPxCyC+xzF+w5eDBojLlQrUtrnuc4syY73YW0rHcB0EgWvQkbUrJPGzSFDoe6uPwa8trK8g9QkWyVxYZmSsqGsNYF2nyqm4YDuLmNoubntia621iO4Q8IC6qBebDg5TsIUn4U53xMLWHWXgnwnJkJT099\\\\\\\/RPWsbZJxxLz5y36oc2BqiaE8wSdLYJGuOY4EhxL83AlHWX1X\\\\\\\/DuRNZ3z1k44yqMuzd23sT8X6y0Jjm\\\\\\\/HF8orDntFkiC6Z5Z7IFgY5ARpOnwDryIYXyEHgfMggQHRxbp4MFR9zjopzHeOxK5C6P6QJ9W4OPbrJeR9JH9TykE8Khbf9H6njvzKX0ICZg\\\\\\\/cQK\\\\\\\/bghHuyuFOW\\\\\\\/Lc40kg4xRaSTbXL8Wl1OCH3S9+l+flGMOQRHnWV\\\\\\\/KfDih8D8JvcZEl7iGmTMCFbK1w3lntGj4pGf5IcFgIXPrDHdwtlWZ23irC5nnxGH1ayGmYTHb5pFlWnloiIRVZZ1RWeTEcnwirseQjmWaXWhx4WYSiZDntseS\\\\\\\/K1ONw9cFYMgsyts12lsdnk2J8vtsdV+83UF+QeuiUA8W2xyEemdI93s2rnOQrqxO02DBJD7FvYwa3aMA2R\\\\\\\/UhgxlpqT69kaje68nSrAGiLQ8sRVsPqNKpzJFNz2xY\\\\\\\/Z3PkUIAYJ6kX5R+pXKEjWwnpVgQzFqmrYAQDUoejLLEBThParTWmThat+xE4q0AAYGytppkdKPPNqOSjVlFb0VkhhTFoq2SK2D74t7\\\\\\\/3pus7tHC3bqPr2IAoMeEmzdgVMwTCMbQtrAA7BsAH+HUW+Oq4DDEDRTjbHX\\\\\\\/HflY5eW1vyvngggOvBsPRBL3FroAWTov04EdNr3LLDjbyRd3ApnQSQGky6Th75T\\\\\\\/0dCr7ouxvkKIuOgYuPpIwpwOAIPoF3Ga892vIMYPeuuHuJHRE6Q2oWFjqlcUnvT6+CXiJw1a06enWDkgKiwAX60usq46UIGFBMiVeg=\\\u0022,\\\u0022iv\\\u0022:\\\u0022A6uj1q\\\\\\\/8zlDfagjnGmWUFw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022964a0c37f0b16e696be2d1f87ea704a78615f9a4c8d60860224729399bb6a15b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487832 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-multiparental-populations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/331" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Cq7ayhG909nGdQe69ta7Tlk5Nl8AI8fdSbYK7YrHs8c" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/331" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Dv4uT54NIO9Jq0SwBWfpaFdAagFZvPSlVDSgIKto6w8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/315" title="Genome-Wide Mapping of Decay Factor–mRNA Interactions in Yeast Identifies Nutrient-Responsive Transcripts as Targets of the Deadenylase Ccr4" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/343" title="Caenorhabditis elegans DBL-1/BMP Regulates Lipid Accumulation via Interaction with Insulin Signaling" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487832" id="node487832" data-pisa="ggg;8/1/331" data-pisa-master="ggg;g3.117.300248" data-apath="/ggg/8/1/331.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mating Design and Genetic Structure of a Multi-Parent Advanced Generation Intercross (MAGIC) Population of Sorghum (<em>Sorghum bicolor</em> (L.) Moench)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Patrick O. Ongom</span> and <span class="highwire-citation-author" data-delta="1">Gebisa Ejeta</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">331-341;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300248 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487832" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Patrick O. Ongom </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BO.%2BOngom%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ongom%20PO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BO.%2BOngom%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gebisa Ejeta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Purdue University, West Lafayette, Indiana 47907</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Global Food Security, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gebisa%2BEjeta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ejeta%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGebisa%2BEjeta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gejeta@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">gejeta@purdue.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/1/331" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487832" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487832/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/331.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487832" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487832/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/331.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487832" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487832/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/331.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487832" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487832/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/1/331.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487832" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487832/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Multi-parent advanced generation intercross (MAGIC) populations are powerful next-generation mapping resources. We describe here the mating design and structure of the first MAGIC population in sorghum, and test its utility for mapping. The population was developed by intercrossing 19 diverse founder lines through a series of paired crosses with a genetic male sterile (MS) source, followed by 10 generations of random mating. At the final stage of random mating, 1000 random fertile plants in the population were identified and subjected to six generations of selfing to produce 1000 immortal MAGIC inbred lines. The development of this sorghum MAGIC population took over 15 yr. Genotyping-by-sequencing (GBS) of a subset of 200 MAGIC lines identified 79,728 SNPs, spanning high gene-rich regions. Proportion of SNPs per chromosome ranged from 6 to 15%. Structure analyses produced no evidence of population stratification, portraying the desirability of this population for genome-wide association studies (GWAS). The 19 founders formed three clusters, each with considerable genetic diversity. Further analysis showed that 73% of founder alleles segregated in the MAGIC population. Linkage disequilibrium (LD) patterns depicted the MAGIC population to be highly recombined, with LD decaying to <em>r</em><sup>2</sup> <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/331/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/331/embed/mml-math-1.gif"/></noscript></span></span> 0.2 at 40 kb and down to <em>r</em><sup>2</sup> <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/331/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/331/embed/mml-math-2.gif"/></noscript></span></span> 0.1 at 220 kb. GWAS detected two known plant height genes, <em>DWARF1</em> (chromosome 9) and <em>DWARF3</em> (chromosome 7), and a potentially new plant height quantitative trait locus (QTL) (<em>QTL-6</em>) on chromosome 6. The MAGIC population was found to be rich in allelic content with high fragmentation of its genome, making it fit for both gene mapping and effective marker-assisted breeding.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/multi-parent-advanced-generation-intercrosses-magic" class="hw-term hw-article-keyword hw-article-keyword-multi-parent-advanced-generation-intercrosses-magic" rel="nofollow">multi-parent advanced generation intercrosses (MAGIC)</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing-gbs" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing-gbs" rel="nofollow">genotyping-by-sequencing (GBS)</a></li><li class="kwd"><a href="/keyword/population-structure" class="hw-term hw-article-keyword hw-article-keyword-population-structure" rel="nofollow">population structure</a></li><li class="kwd"><a href="/keyword/single-nucleotide-polymorphisms-snps-0" class="hw-term hw-article-keyword hw-article-keyword-single-nucleotide-polymorphisms-snps" rel="nofollow">single nucleotide polymorphisms (SNPs)</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium-ld" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium-ld" rel="nofollow">linkage disequilibrium (LD)</a></li><li class="kwd"><a href="/keyword/mpp" class="hw-term hw-article-keyword hw-article-keyword-mpp" rel="nofollow">MPP</a></li></ul><p id="p-3">Quantitative traits in crop plants form the predominant plant characteristics of agronomic and economic importance, including such traits as crop productivity, environmental adaptation, and tolerance to drought, as well as resistance or immunity to certain pests and diseases. The inheritance of these traits is often complex, with patterns of phenotypic expression determined by an intricate series of interactions between multiple genes and environmental factors. Conventionally, allelic sources to improve these traits in crops are sought from naturally occurring variation (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kover <em>et al.</em> 2009</a>), but the genetic underpinning of natural variation in these traits often appears ambiguous, partly due to lack of appropriate core genetic resources to properly exploit such variation. The more common way to dissect QTL in crop species has been through use of populations derived from biparental crosses, including F<sub>2</sub>, backcross, recombinant inbred lines, backcross inbred lines, double haploids, and near isogenic lines (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Collard <em>et al.</em> 2005</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Rakshit <em>et al.</em> 2012</a>). Immortal inbred lines from biparental populations have been used for genetic studies and mapping of QTL linked to key traits in many crops, including sorghum, where putative loci for drought resistance (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Tuinstra <em>et al.</em> 1997</a>), cold tolerance (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Knoll <em>et al.</em> 2008</a>), <em>Striga</em> resistance (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Vogler <em>et al.</em> 1996</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Haussmann <em>et al.</em> 2004</a>), and disease and insect resistances (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Apotikar <em>et al.</em> 2011</a>) have been identified.</p><p id="p-4">With advances in genomic technologies that allow high-throughput genotyping capabilities in crops at a continuously reduced cost, alternative QTL mapping resources have recently emerged. These new advances have immensely increased the larger marker density required for QTL studies, thrusting plant scientists into the genomic era and embracing GWAS as a tool to dissect QTL underlying complex traits as well as to identify functional genes, which initially have been used only in human populations. The potential utility of GWAS for more precise mapping of novel genes in crop plants has been demonstrated by a number of studies (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gupta <em>et al.</em> 2005</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Nordborg and Weigel 2008</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Rafalski 2010</a>). The mapping resources that have commonly been used for GWAS are accessions of landrace genetic resources or breeding lines that have been extensively phenotyped over years in most breeding programs. These collections have provided more cost-effective mapping panels that are diverse and have had high historical recombination that is suitable for GWAS. Furthermore, the faster LD decays in natural accessions make it possible to map QTL with higher resolution (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Nordborg and Weigel 2008</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Kover <em>et al.</em> 2009</a>). However, natural accessions tend to be highly stratified, requiring the use of complex statistical models to account for the false association signals that result in GWAS conducted using them.</p><p id="p-5">More recently, novel MAGIC populations have been advanced as next-generation mapping resources to address the major limitations of existing mapping populations (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bandillo <em>et al.</em> 2013</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Mackay <em>et al.</em> 2014</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Verbyla <em>et al.</em> 2014</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Holland 2015</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Huang <em>et al.</em> 2015</a>). The mouse collaborative cross was the first MAGIC population to be developed before the approach was proposed for QTL mapping of complex traits in a crop plant (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cavanagh <em>et al.</em> 2008</a>). The MAGIC design concept is similar to that of advanced intercross lines (AILs) proposed by <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Darvasi and Soller (1995)</a>, except that AILs are still derived from a two-parent cross; however, each generation is sequentially and randomly intercrossed until advanced intercross generations are attained. MAGIC populations are generally created by intercrossing multiple founder lines over several generations. Using multiple founders contributes more allelic diversity than that captured in typical biparental mapping populations. Multiple cycles of intercrossing result in greater opportunities for recombination, thus providing greater precision in QTL location. MAGIC and MAGIC-like populations have now been made available in a range of plant and crop species, including <em>Arabidopsis</em> (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Kover <em>et al.</em> 2009</a>), wheat (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Huang <em>et al.</em> 2012</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Mackay <em>et al.</em> 2014</a>), rice (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Bandillo <em>et al.</em> 2013</a>), and maize (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Dell’Acqua <em>et al.</em> 2015</a>).</p><p id="p-6">Multi-parent crosses have a long history in conventional plant breeding, especially in outcrossing crops where crosses are easy to make (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Harlan and Martini 1929</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Suneson 1956</a>). In self-pollinated species like sorghum, multi-parent crosses gained usage in 1960s after the discovery of genetic male sterility (GMS) systems, which facilitated the adoption of recurrent selection (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Eberhart 1972</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Reddy and Stenhouse 1994</a>). It is with the capacity to generate large populations more efficiently and recent tools in high-throughput genomic technologies, as well as in advanced statistical capabilities, that multi-parent populations have been employed in genetic mapping. Although MAGIC populations are useful in genetic studies, they present some unique challenges. The crossing schemes of the currently available MAGIC populations pose a potential intermating bias that can result in assortative mating other than the assumed random mating. Depending on the number of founders, intermating may become cumbersome as family sizes of the parent crosses increase along the pedigree. This complexity increases the chances of mating between individuals with similar genotypes, causing deviations from random mating expectation, a phenomenon termed assortative mating (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Templeton 2006</a>). Hence, it is often advised that staggered planting and/or planting of the same families multiple times should be avoided during MAGIC development, because these practices would promote assortative mating (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kole 2013</a>). Assortative mating would introduce the rise of subgroupings within the MAGIC population, distorting LD and causing spurious associations (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fisher 1919</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Moran <em>et al.</em> 1966</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wilson 1978</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Garnier-Géré and Chikhi 2013</a>), and bringing about the false associations often encountered in natural populations. Times required to establish MAGIC populations can be long, given the number of founders involved, making the process logistically challenging and labor intensive. In sorghum, the use of GMS in the construction of a MAGIC population significantly reduces the challenges described above.</p><p id="p-7">The objectives of this study were as follows: (i) to describe the construction and mating design of the first sorghum MAGIC population facilitated with the aid of GMS in a base population, (ii) to construct a genome-wide single nucleotide polymorphism map and, utilizing these SNPs, to dissect the structure of the MAGIC population, and (iii) to evaluate the efficiency of the SNP panel and the potential of the sorghum MAGIC population for conducting GWAS using plant height, a trait that is well studied in sorghum, with some of the governing loci that have already been cloned.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-8">The MAGIC population was constructed from 19 founder sorghum lines that had previously been characterized for a broad range of useful agronomic traits, including early maturity, broad adaptation, yield potential, drought tolerance, and grain quality, as well as pest and disease resistance. The 19 founders were intermated and recombined with the aid of a GMS system that facilitated random mating for several cycles to produce 1000 random inbreds, through further selfing through a single-seed descent (SSD) method of inbreeding. A subset of 200 S<sub>7</sub> inbred lines were pooled from the MAGIC population for use in this study. The 200 S<sub>7</sub> inbreds were phenotyped for plant height at the Agronomy Center for Research and Education, Purdue University, during the summer of 2013 and 2014. Tissue from each genotype were collected, lyophilized, and pulverized for DNA extraction. Thirty-microliter aliquots of DNA at concentrations of 50–100 ng/µl in 96-well plates were sent to the genomic facility at Cornell University for genotyping using a high-throughput GBS platform. The reference genome was indexed and alignment generated with the software package BWA Version: 0.7.8-r455 (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Li and Durbin 2009</a>). Of the 4,122,598 tags used, 2,763,236 (67.0%) were aligned to unique positions, while 457,904 (11.1%) were aligned to multiple positions, and 901,458 (21.9%) could not be aligned. Chromosome-wide density plots for SNPs and the reference sequence genes were generated to gauge genomic coverage of the discovered SNPs. To determine the proportion of founder alleles that were captured in the MAGIC subset, the MAGIC and founder SNPs were filtered separately, allowing minor allele frequency (MAF) &gt; 0.01 and 10% missing data, and the number of polymorphic SNPs in both panels for each chromosome were counted. The pattern of genetic structure of the MAGIC population was assessed using principal component analysis (PCA) and neighbor-joining tree analyses. PCA scores were generated in TASSEL and exported to Microsoft Excel, where visualization graphics were produced. To evaluate the effectiveness of the sorghum MAGIC population for use in gene mapping, data on plant height was subjected to GWAS using three different models.</p><div id="sec-2" class="subsection"><h3>Data availability</h3><p id="p-9">The MAGIC lines and founders are maintained by the sorghum breeding program at Purdue University and are available upon request. Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS1.docx">File S1</a> contains supplemental figures and tables cited in this article. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS2.txt">File S2</a> contains SNP ID numbers and locations. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS3.txt">File S3</a> contains genotypes for each MAGIC line used for GWAS analysis. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS4.txt">File S4</a> contains genotypes for each Founder line. The GBS raw sequence data are available at the GenBank, and have been assigned accession number PRJNA417037. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS5.txt">File S5</a> contains plant height phenotypic data used for GWAS analysis. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS6.docx">File S6</a> contains full details of the materials and methods employed in this study.</p></div></div><div class="section results" id="sec-3"><h2 class="">Results</h2><div id="sec-4" class="subsection"><h3>Development of MAGIC population</h3><p id="p-10">This sorghum MAGIC population was created using 19 diverse founder lines (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) intercrossed randomly through a GMS system, resulting in a population built to suit association studies for complex traits. The GMS in sorghum is conditioned by a nuclear recessive gene that segregates in the population, allowing the creation of random mating populations. Each of the 19 founder lines were crossed to 10 random genetic MS plants in the <em>ms<sub>3</sub></em> source population. The MS source was an open-pollinated population segregating for male sterility, and not a highly inbred genotype. The 190 random MS plants to which the 19 founder lines were crossed (10 × 19) collectively represented the base-population, but unfortunately could not be saved for sequencing. This meant that we could only approximate the estimates of the allelic make-up of the MAGIC population. Each of the resultant F<sub>1</sub> hybrids were allowed to self-pollinate to generate F<sub>2</sub> seeds. Equal amounts of F<sub>2</sub> seeds were pooled from each cross pair and bulked to create the new random mating base population (RM<sub>0</sub>). The RM<sub>0</sub> population, segregating for the <em>ms<sub>3</sub></em> gene, was planted on an ∼0.25 acre field that first year, and annually for the subsequent 10 seasons, to allow it to intermate randomly. Each year, 1000–1500 random sterile plants in the large intermating population were identified at bloom stage when they could easily be discerned and tagged. Only seeds that set on tagged MS plants, known to be random-mated, were harvested and bulked each generation to form subsequent generations of the population (RM<sub>n</sub>). The sterile plants are easy to identify because they do not shed pollen and have characteristic scaly, whitish or less yellowish, and nonplump anthers (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Reddy <em>et al.</em> 2005</a>). In the tenth season, random mating was stopped, and the resulting RM<sub>9</sub> MAGIC population was planted so as to randomly sample 1000 fertile S<sub>0</sub> plants to start inbreeding. For the next six seasons, each of the 1000 S<sub>0</sub> plants were self-pollinated and advanced using the SSD method. At the end of the selfing generation, resultant progenies were row-planted in the field so that 1000 S<sub>7</sub> plant rows were bulked, resulting in 1000 random S<sub>7</sub> inbred lines. From this set, 200 random S<sub>7</sub> inbreds were drawn randomly to create a MAGIC subset, used in this study to dissect the structure of the MAGIC population. The mating scheme for the sorghum MAGIC population, described above, is presented in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The MAGIC inbreds are currently being maintained by the sorghum breeding program at Purdue University and may be accessed on request.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487849/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487849/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Founder lines of the sorghum MAGIC population</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/331/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mating design of the first multi-parent advanced generation intercross (MAGIC) population of sorghum, developed with the aid of genetic male sterility. In this figure, ms3 refers to the male sterile sorghum lines used to introduce sterility in the MAGIC population, RM0 to RM9 refers to cycles of random mating, S0 to S7 refers to generations of selfing, and SSD refers to single-seed descent. Initial pairwise hybridization made between 19 founding parents and 10 random sterile plants. Progenies were bulked and random mated for nine cycles, 1000 random lines were generated, and a subset of 200 has been genotyped. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1719179735" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Mating design of the first multi-parent advanced generation intercross (MAGIC) population of sorghum, developed with the aid of genetic male sterility. In this figure, &lt;em&gt;ms&lt;sub&gt;3&lt;/sub&gt;&lt;/em&gt; refers to the male sterile sorghum lines used to introduce sterility in the MAGIC population, RM&lt;sub&gt;0&lt;/sub&gt; to RM&lt;sub&gt;9&lt;/sub&gt; refers to cycles of random mating, S&lt;sub&gt;0&lt;/sub&gt; to S&lt;sub&gt;7&lt;/sub&gt; refers to generations of selfing, and SSD refers to single-seed descent. Initial pairwise hybridization made between 19 founding parents and 10 random sterile plants. Progenies were bulked and random mated for nine cycles, 1000 random lines were generated, and a subset of 200 has been genotyped. &lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/331/F1.medium.gif" width="440" height="238"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/1/331/F1.medium.gif" width="440" height="238"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/331/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/331/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488001" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Mating design of the first multi-parent advanced generation intercross (MAGIC) population of sorghum, developed with the aid of genetic male sterility. In this figure, <em>ms<sub>3</sub></em> refers to the male sterile sorghum lines used to introduce sterility in the MAGIC population, RM<sub>0</sub> to RM<sub>9</sub> refers to cycles of random mating, S<sub>0</sub> to S<sub>7</sub> refers to generations of selfing, and SSD refers to single-seed descent. Initial pairwise hybridization made between 19 founding parents and 10 random sterile plants. Progenies were bulked and random mated for nine cycles, 1000 random lines were generated, and a subset of 200 has been genotyped. </p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Genomic features of MAGIC population</h3><p id="p-12">Analysis of the GBS data resulted in a total of 79,728 SNPs (excluding those that mapped to super contigs), distributed across the entire sorghum genome and at an average spacing of 9 kb. The distribution of SNPs varied within and among chromosomes. Chromosome 1 showed the highest density of SNPs (11,966 SNPs), with chromosome 7 being the least (5178 SNPS) dense (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS1.docx">File S1</a>). Chromosome-wide SNPs and gene density analyses revealed that the MAGIC SNPs were higher in regions with high gene density and vice versa, indicating the richness of SNPs in the MAGIC population and their potential usefulness in tagging genes of interest (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS1.docx">File S1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/331/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosome-wide distribution of single nucleotide polymorphisms (SNPs) and reference sequence genes. Each of the 10 sorghum chromosomes (chr) is presented with SNP density (in gray) and a corresponding gene density (in blue) plotted below it. Chromosomes 5&#x2013;10 are shown in Figure S1 in File S1. A summary plot of the number of SNPs per chromosome is presented at the bottom right hand corner of this figure. Pos, position." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1719179735" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Chromosome-wide distribution of single nucleotide polymorphisms (SNPs) and reference sequence genes. Each of the 10 sorghum chromosomes (chr) is presented with SNP density (in gray) and a corresponding gene density (in blue) plotted below it. Chromosomes 5&#x2013;10 are shown in Figure S1 in File S1. A summary plot of the number of SNPs per chromosome is presented at the bottom right hand corner of this figure. Pos, position.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/331/F2.medium.gif" width="396" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/1/331/F2.medium.gif" width="396" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/331/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/331/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487906" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-13" class="first-child">Chromosome-wide distribution of single nucleotide polymorphisms (SNPs) and reference sequence genes. Each of the 10 sorghum chromosomes (chr) is presented with SNP density (in gray) and a corresponding gene density (in blue) plotted below it. Chromosomes 5–10 are shown in Figure S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS1.docx">File S1</a>. A summary plot of the number of SNPs per chromosome is presented at the bottom right hand corner of this figure. Pos, position.</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">The distribution of heterozygotes, MAF, and missing data are presented in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS1.docx">File S1</a>. The average proportions of heterozygotes were 0.039 and 0.044 in the MAGIC subset and founders, respectively (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, a and b</a>). Mean MAF was 0.15 in the MAGIC subset and 0.16 among the founders, while the mean proportion of missing data were 0.076 and 0.064 in the MAGIC and the founders, respectively (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, a and b</a>). MAF is useful in assessing the efficiency of the markers in a mapping panel, as shown previously (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pasam <em>et al.</em> 2012</a>). Our results showed the mean MAF in the MAGIC panel to vary from 0.12 on chromosome 4 to 0.18 on chromosome 6. About 36% of the MAGIC SNP markers had MAF &lt; 0.05 (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3a</a>), suggesting that a greater percentage (64%) of the SNPs had high MAF and with sufficient polymorphism required for GWAS.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/331/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pattern of genetic features in a subset of sorghum multi-parent advanced generation intercross (MAGIC) population and founder lines. (a) Chromosome-wide distribution of mean minor allele frequency (MAF), mean proportion of heterozygotes, and missing data in the MAGIC subset. The embedded bar chart in (a) depicts the percentage proportion of MAGIC SNPs corresponding to bins or categories of MAF. (b) Chromosome-wide distribution of mean MAF, mean proportion of heterozygotes, and missing data among the 19 founder lines. In (a and b), summary statistics are presented in the embedded text boxes. Nr., number; Pos, position." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1719179735" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Pattern of genetic features in a subset of sorghum multi-parent advanced generation intercross (MAGIC) population and founder lines. (a) Chromosome-wide distribution of mean minor allele frequency (MAF), mean proportion of heterozygotes, and missing data in the MAGIC subset. The embedded bar chart in (a) depicts the percentage proportion of MAGIC SNPs corresponding to bins or categories of MAF. (b) Chromosome-wide distribution of mean MAF, mean proportion of heterozygotes, and missing data among the 19 founder lines. In (a and b), summary statistics are presented in the embedded text boxes. Nr., number; Pos, position.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/331/F3.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/1/331/F3.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/331/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/331/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487985" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-15" class="first-child">Pattern of genetic features in a subset of sorghum multi-parent advanced generation intercross (MAGIC) population and founder lines. (a) Chromosome-wide distribution of mean minor allele frequency (MAF), mean proportion of heterozygotes, and missing data in the MAGIC subset. The embedded bar chart in (a) depicts the percentage proportion of MAGIC SNPs corresponding to bins or categories of MAF. (b) Chromosome-wide distribution of mean MAF, mean proportion of heterozygotes, and missing data among the 19 founder lines. In (a and b), summary statistics are presented in the embedded text boxes. Nr., number; Pos, position.</p><div class="sb-div caption-clear"></div></div></div><p id="p-16">We further estimated the proportion of polymorphic alleles that were shared between the 200 subsets of the MAGIC population and the founders, by first filtering the data to allow &gt; 0.01 MAF with &lt; 10% missing data. A total of 48,327 and 48,545 SNPs were polymorphic among the MAGIC subset and founders, respectively, while 35,507 SNPs segregated in both (Figure S2, a–c in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS1.docx">File S1</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). We expressed the number of shared SNPs relative to the total number of SNPs in the founders to determine the proportion of founder SNPs captured in the MAGIC subset. The proportion of founder alleles segregating within the MAGIC subset varied among chromosomes from a minimum ratio of 0.69 on chromosome 10 to a maximum of 0.79 on chromosome 5, and with a mean of 0.73 (Figure S2, a–c in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS1.docx">File S1</a> and <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487953/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487953/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Chromosome-wide distribution of polymorphic founder alleles that segregated in the MAGIC population</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-6" class="subsection"><h3>MAGIC population structure</h3><p id="p-18">Population stratification was assessed using multiple statistical approaches including PCA, neighbor-joining tree, and STRUCTURE analyses. The PCA results are presented in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. The Scree plot showed ∼14 principal components (PCs) before beginning to flatten (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4a</a>), with these first 14 PCs together explaining ∼21% of total variation in the population. That many PCs were required to account for only 21% of total variation in the population indicated that each of the individual PCs explained small, and about the same amount of, variation, suggesting an absence of unique groupings within the population. Because the PC values were about the same, we clustered the population using the first two PCs, which together explained 5% of the total variation. PCA did not show any grouping pattern among the 200 individuals in the MAGIC subset, yet the founders were clustered into three major groups (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4b</a>). Neighbor-joining tree analysis revealed a similar pattern to that observed from PCA (Figure S3, a and b in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS1.docx">File S1</a>). When we compared the MAGIC structure to that of the sorghum association panel (SAP), which has been reported in a previous GWAS (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Morris <em>et al.</em> 2013</a>), as an example of a structured population (Figure S3, c and d in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS1.docx">File S1</a>), the difference was remarkably high, with the SAP displaying five distinctive clusters (Figure S3d in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS1.docx">File S1</a>), in clear contrast to the MAGIC showing a more uniform distribution of individuals in the population. Further attempts to infer clusters in the MAGIC subset using STUCTURE software (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Pritchard <em>et al.</em> 2000</a>) yielded no evidence of population structure. A plot of mean posterior probability distribution [lnP(D)] for each run against the number of inferred clusters showed lnP(D) to decrease with an increase in K (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6a</a>, Figure S4a, and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS1.docx">File S1</a>), an outcome exactly opposite to that often seen in structured populations. The highest values of lnP(D) were between K = 1 and K = 4, and fluctuated thereafter while descending (Figure S4a and Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS1.docx">File S1</a>). If there was true population stratification, we would expect small lnP(D) for lower values of K, and this would increase steadily and plateau for larger Ks (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Pritchard <em>et al.</em> 2000</a>). The bar plot graphics of STRUCTURE revealed no complete assignment of any individual in the MAGIC subset to a specific group (Figure S4b in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS1.docx">File S1</a>). Overall, PCA, neighbor-joining tree, and STRUCTURE analyses generated consistent evidence for no subgrouping in the sorghum MAGIC population.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/331/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PC analysis of the sorghum MAGIC population. (a) Scree plot of the PCs (x-axis) and their contribution to variance (the eigenvalues) y-axis). (b) Axes plot of the first two PCs for the MAGIC subset (black dots), including the founders (red dots labeled with founder names). The MAGIC subset appears to form a single group surrounded by three major clusters of founders (black circles around groups of red dots). chr, chromosome; MAGIC, multi-parent advanced generation intercross; PC, principal component." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1719179735" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;PC analysis of the sorghum MAGIC population. (a) Scree plot of the PCs (&lt;em&gt;x&lt;/em&gt;-axis) and their contribution to variance (the eigenvalues) &lt;em&gt;y&lt;/em&gt;-axis). (b) Axes plot of the first two PCs for the MAGIC subset (black dots), including the founders (red dots labeled with founder names). The MAGIC subset appears to form a single group surrounded by three major clusters of founders (black circles around groups of red dots). chr, chromosome; MAGIC, multi-parent advanced generation intercross; PC, principal component.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/331/F4.medium.gif" width="378" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/1/331/F4.medium.gif" width="378" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/331/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/331/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488056" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-19" class="first-child">PC analysis of the sorghum MAGIC population. (a) Scree plot of the PCs (<em>x</em>-axis) and their contribution to variance (the eigenvalues) <em>y</em>-axis). (b) Axes plot of the first two PCs for the MAGIC subset (black dots), including the founders (red dots labeled with founder names). The MAGIC subset appears to form a single group surrounded by three major clusters of founders (black circles around groups of red dots). chr, chromosome; MAGIC, multi-parent advanced generation intercross; PC, principal component.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>LD analysis</h3><p id="p-20">LD analysis was conducted to explore the genome landscape of the MAGIC population for association studies. The pattern of LD across the entire genome revealed several haplotype blocks harboring SNPs that are in strong LD (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5, a and b</a>). These blocks are surrounded by regions that have undergone recombination and are therefore not in LD. The P-values associated with the <em>r</em><sup>2</sup> or D´ (measures of LD), clearly showed the patterns of block fragmentation that are spread across the genome (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5, a and b</a>). Such patterns offer great opportunity for fine mapping, since the SNPs that persistently remain in strong LD despite the opportunity for recombination tend to be tightly associated with the genes of interest.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/331/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LD patterns. (a and b) LD heatmaps of a segment of the MAGIC genome based on (a) r2 and (b) D&#x2019;, with r2 and D&#x2019; shown above diagonals while P-values are below diagonals. Each cell represents the comparison of two pairs of marker sites with the color codes shown for the presence of significant LD. (c) LD decay rate for each chromosome. The top right insert depicts the mean genome-wide decay rate. The vertical dashed lines intersect the decay curve at r2 = 0.2 and r2 = 0.1, corresponding to the distances of 40 and 220 kb, respectively, the LD values below which the pairwise linkage between loci dissipates. chr, chromosome; LD, linkage disequilibrium; MAF, minor allele frequency; MAGIC, multi-parent advanced generation intercross; SNP, single nucleotide polymorphism." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1719179735" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;LD patterns. (a and b) LD heatmaps of a segment of the MAGIC genome based on (a) &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; and (b) D&#x2019;, with &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; and D&#x2019; shown above diagonals while P-values are below diagonals. Each cell represents the comparison of two pairs of marker sites with the color codes shown for the presence of significant LD. (c) LD decay rate for each chromosome. The top right insert depicts the mean genome-wide decay rate. The vertical dashed lines intersect the decay curve at &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; = 0.2 and &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; = 0.1, corresponding to the distances of 40 and 220 kb, respectively, the LD values below which the pairwise linkage between loci dissipates. chr, chromosome; LD, linkage disequilibrium; MAF, minor allele frequency; MAGIC, multi-parent advanced generation intercross; SNP, single nucleotide polymorphism.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/331/F5.medium.gif" width="403" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/1/331/F5.medium.gif" width="403" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/331/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/331/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488044" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-21" class="first-child">LD patterns. (a and b) LD heatmaps of a segment of the MAGIC genome based on (a) <em>r</em><sup>2</sup> and (b) D’, with <em>r</em><sup>2</sup> and D’ shown above diagonals while P-values are below diagonals. Each cell represents the comparison of two pairs of marker sites with the color codes shown for the presence of significant LD. (c) LD decay rate for each chromosome. The top right insert depicts the mean genome-wide decay rate. The vertical dashed lines intersect the decay curve at <em>r</em><sup>2</sup> = 0.2 and <em>r</em><sup>2</sup> = 0.1, corresponding to the distances of 40 and 220 kb, respectively, the LD values below which the pairwise linkage between loci dissipates. chr, chromosome; LD, linkage disequilibrium; MAF, minor allele frequency; MAGIC, multi-parent advanced generation intercross; SNP, single nucleotide polymorphism.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">To estimate the size of the LD blocks or the LD decay rate in the MAGIC population, we plotted the <em>r</em><sup>2</sup> values against physical distance (kb) for each chromosome (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5c</a> and Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS1.docx">File S1</a>). We set the LD decay rate at <em>r</em><sup>2</sup> threshold values of 0.2 and 0.1. These plots provided estimates of LD decay to be at 40 kb when <em>r</em><sup>2</sup> ≤ 0.2 and 220 kb when <em>r</em><sup>2</sup> ≤ 0.1, close to the values reported for outcrossing species or highly diverse populations (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Morris <em>et al.</em> 2013</a>). This implied that that the genomes of MAGIC-derived lines are highly recombined, and therefore well suited for genome-wide gene mapping.</p></div><div id="sec-8" class="subsection"><h3>Mapping plant height genes in sorghum</h3><p id="p-23">To verify the suitability of our new sorghum MAGIC population for gene mapping, as predicted from our genetic analysis, we conducted genome-wide association analysis for plant height. This phenotype is a well-studied trait in sorghum, and four major dwarfing genes controlling this trait have been previously described (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brown <em>et al.</em> 2008</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Morris <em>et al.</em> 2013</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Thurber <em>et al.</em> 2013</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Higgins <em>et al.</em> 2014</a>). Phenotypic dissection and depiction of genetic variability from a 2 yr plant height data set is summarized in Figure S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS1.docx">File S1</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. Genotypic differences, year-to-year variation and the interaction between genotype and year were highly significant (P ≤ 0.001). The genotypic variance component accounted for most of the phenotypic variation, that is, 75% (2013), 75% (2014), and 65% (2013 and 2014 combined). Repeatability for plant height was ∼85%, which reflected that the trait measurements were fairly consistent between replications as well as across years, an indication of high predictability and, hence, high heritability.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487963/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487963/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">ANOVA depicting magnitude of genetic variation for plant height among the 200 subset of sorghum MAGIC population</span><div class="sb-div caption-clear"></div></div></div><p id="p-27">Genome-wide association analysis based on the GLM (Q model) revealed three significant [−log 10(P) <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/331/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/331/embed/mml-math-3.gif"/></noscript></span></span> 4.8] association signals for plant height on chromosomes 6, 7, and 9 (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6a</a>). These signals spanned genomic regions that harbor two known plant height genes, <em>DWARF1</em> on chromosome 9 and <em>DWARF3</em> on chromosome 7 (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Feltus <em>et al.</em> 2006</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Brown <em>et al.</em> 2008</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Upadhyaya <em>et al.</em> 2012</a>; <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Morris <em>et al.</em> 2013</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Thurber <em>et al.</em> 2013</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Higgins <em>et al.</em> 2014</a>), and a potentially new plant height locus (<em>QTL-6</em>) on chromosome 6 (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6, b–d</a>). Remapping of these plant height genes signified the potential utility of the sorghum MAGIC panel in gene discovery. Examination of the Q-Q plots from the three GWAS models (GLM-Naïve, GLM-Q, and MLM-Q+K), used in this study to evaluate the effect of these models on gene mapping in the MAGIC population, showed that the Q-Q plot for the Naïve and Q models were very similar, with no serious departure from expectation (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6, e and f</a>), with power to detect the three plant height genes. On the other hand, the Q+K model [mixed linear model (MLM)] showed a better fit than the other two models (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6g</a>), although the signal for the <em>QTL-6</em> locus on chromosome 6 could not reach the genome-wide significance threshold (Figure S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1/FileS1.docx">File S1</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/331/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide scan for plant height loci. (a) Manhattan plot based on Q-model shows a strong association signal in the plant height gene regions. (b&#x2013;d) Highlight of peaks for the two known (DWARF1 and DWARF3) and one new (QTL-6) dwarfing genes: (b) QTL-6, (c) DWARF3, and (d) DWARF1. The vertical dashed lines show the physical positions of the height genes. (e&#x2013;g) Q-Q plots for the three models: (e) na&#xEF;ve model, (f) Q-model, and (g) Q+K-model. Significant threshold (horizontal line) in (a) was determined based on the average linkage disequilibrium (LD) decay extent in the multi-parent advanced generation intercross (MAGIC) population as: Number of independent tests = reference genome size (730 Mb)/MAGIC LD extent (220 kb); the threshold is then obtained as [0.05/(730 Mb/0.22 Mb)] = 1.50685 &#xD7; 10&#x2212;5, equivalent to &#x2212;log10(P) of 4.8." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1719179735" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome-wide scan for plant height loci. (a) Manhattan plot based on Q-model shows a strong association signal in the plant height gene regions. (b&#x2013;d) Highlight of peaks for the two known (&lt;em&gt;DWARF1 and DWARF3&lt;/em&gt;) and one new (&lt;em&gt;QTL-6&lt;/em&gt;) dwarfing genes: (b) &lt;em&gt;QTL-6&lt;/em&gt;, (c) &lt;em&gt;DWARF3&lt;/em&gt;, and (d) &lt;em&gt;DWARF1&lt;/em&gt;. The vertical dashed lines show the physical positions of the height genes. (e&#x2013;g) Q-Q plots for the three models: (e) na&#xEF;ve model, (f) Q-model, and (g) Q+K-model. Significant threshold (horizontal line) in (a) was determined based on the average linkage disequilibrium (LD) decay extent in the multi-parent advanced generation intercross (MAGIC) population as: Number of independent tests = reference genome size (730 Mb)/MAGIC LD extent (220 kb); the threshold is then obtained as [0.05/(730 Mb/0.22 Mb)] = 1.50685 &#xD7; 10&lt;sup&gt;&#x2212;5&lt;/sup&gt;, equivalent to &#x2212;log10(P) of 4.8.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/331/F6.medium.gif" width="430" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/1/331/F6.medium.gif" width="430" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/331/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/331/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487925" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-28" class="first-child">Genome-wide scan for plant height loci. (a) Manhattan plot based on Q-model shows a strong association signal in the plant height gene regions. (b–d) Highlight of peaks for the two known (<em>DWARF1 and DWARF3</em>) and one new (<em>QTL-6</em>) dwarfing genes: (b) <em>QTL-6</em>, (c) <em>DWARF3</em>, and (d) <em>DWARF1</em>. The vertical dashed lines show the physical positions of the height genes. (e–g) Q-Q plots for the three models: (e) naïve model, (f) Q-model, and (g) Q+K-model. Significant threshold (horizontal line) in (a) was determined based on the average linkage disequilibrium (LD) decay extent in the multi-parent advanced generation intercross (MAGIC) population as: Number of independent tests = reference genome size (730 Mb)/MAGIC LD extent (220 kb); the threshold is then obtained as [0.05/(730 Mb/0.22 Mb)] = 1.50685 × 10<sup>−5</sup>, equivalent to −log10(P) of 4.8.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-9"><h2 class="">Discussion</h2><p id="p-29">In crops, the potential for high-resolution mapping has not been fully exploited, partly because little progress has been made in designing mapping resources to match the rapid advances in genotyping technologies. Early development of novel genetic resource panels in maize (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Dell’Acqua <em>et al.</em> 2015</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Holland 2015</a>) and rice (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Bandillo <em>et al.</em> 2013</a>) has helped to initiate many genomic studies that have led to the discovery of genes and gene functions in these crops. In sorghum, QTL are still being mapped based on traditional biparental mapping populations with limited allelic combinations and recombination, although random mating populations have long been used in recurrent selection programs (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Eberhart 1972</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Maunder 1972</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Reddy and Stenhouse 1994</a>). In the past, the complexity of the genetics in such broad-base populations limited their usage as a gene mapping resource. More recently, the development of high-throughput sequencing technologies, and greater statistical power, have generated new interest and attention in creating unique mapping populations to take advantage of the new genomic tools. MAGIC populations are the latest mapping resources with the potential to overcome some of the limitations of biparental populations.</p><p id="p-30">This MAGIC population, the first described in sorghum, is rich in multiple allelic combinations from its diverse founders, making it a good mapping resource and a breeding pool for cultivar development. The sorghum population derives its uniqueness in the fine structure brought about due to the multiple cycles of random mating, with the aid of GMS in its crossing scheme. GMS in sorghum is conditioned by a single nuclear recessive allele in homozygous condition, designated with a series of alleles as <em>ms<sub>1</sub></em>–<em>ms<sub>7</sub></em> (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rooney and Klein 2000</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Reddy <em>et al.</em> 2005</a>). Male sterility induced by the <em>ms<sub>3</sub></em> and <em>ms<sub>7</sub></em> alleles are very stable and have been widely used in population improvement (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Reddy and Stenhouse 1994</a>; <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Reddy <em>et al.</em> 2005</a>). An advantage of this system is that, once introduced in the population, the <em>ms</em> gene facilitates intercrossing readily among founding parents in a population that is planted in isolation, without the need for the maintenance of complex pedigrees. Multiple cycles of random mating can therefore be achieved, thereby speeding up the process a bit more and eliminating tedious manual emasculation and crossing. More importantly, the risk of assortative mating (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Templeton 2006</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kole 2013</a>) is significantly reduced, owing to intermating that is facilitated by wind pollination through secondary and tertiary growths (tiller plants) that allow random mating among different maturity groups, compared to previously reported MAGIC populations in other crops that relied on mechanical crossing schemes other than GMS.</p><p id="p-31">The sorghum MAGIC SNP panel consists of 79,000 SNPs spanning the entire genome, most of which are in densely populated gene rich regions, with few around pericentromeric regions (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). Sorghum chromosomes are known to contain large pericentromeric regions spanning ∼50% of the genome, and these regions are characterized by low gene density and very low rates of recombination (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Evans <em>et al.</em> 2013</a>). In terms of efficiency, the sorghum MAGIC SNP panel showed variable distribution of MAFs per chromosome, the average MAF being 0.15. In most cases, MAF ≤ 0.05 is considered appropriate for any SNP mapping panel (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">The International HapMap Consortium 2005</a>). MAF estimation is important in GWAS because the least common alleles in the population are difficult to detect with low statistical power and, if detected, are very prone to false-positive associations (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tabangin <em>et al.</em> 2009</a>). It would require impracticably large population sizes to precisely detect associations of minor alleles with traits. For this reason, SNPs with MAF ≤ 0.05 are often excluded from GWAS analysis. For our sorghum MAGIC SNP panel, excluding 36% of SNPs with MAF &lt; 0.05, leaves ∼51 K informative SNPs spanning the genome at an average spacing of 14 kb, which is sufficient and efficient for GWAS. However, mean residual heterozygosity (0.039) in the sorghum MAGIC population was higher than expected for S<sub>7</sub> inbreds, and not comparable to the values observed in a maize MAGIC population (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Dell’Acqua <em>et al.</em> 2015</a>). This higher level of heterozygosity may reflect contamination and poor genotyping integrity, suggesting perhaps that this SNP panel should be filtered to reduce the percentage of heterozygous loci. The proportion of founder alleles that segregated in the MAGIC population was 73%, a somewhat lower value compared to that observed in maize (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Dell’Acqua <em>et al.</em> 2015</a>). This may have been caused by unaccounted alleles from the random plants from the MS base population germplasm to which the founder lines were intercrossed, given that alleles from that source were not tracked, but evenly distributed in the resultant MAGIC population.</p><p id="p-32">The results of this study provided evidence for the absence of a population structure, which is consistent with other MAGIC populations that have been previously described in other crop plants (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Cavanagh <em>et al.</em> 2008</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Huang <em>et al.</em> 2012</a>; <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Bandillo <em>et al.</em> 2013</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Mackay <em>et al.</em> 2014</a>; <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Dell’Acqua <em>et al.</em> 2015</a>). Population structure affects GWAS results as it is the major source of false associations, and it is the reason for the development of complex statistical models like the MLM and efficient mixed-model association (EMMA) algorithms to try to account for the problem (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kang <em>et al.</em> 2008</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Zhang <em>et al.</em> 2010</a>). Unfortunately, these models, while they are useful in reducing the problem, do not completely resolve the basic problem with structured populations. With nonstructured populations, the burden of complex modeling is eliminated and association signals are usually real.</p><p id="p-33">Random mating increases the frequency of detectable recombinants in the population (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beavis <em>et al.</em> 1992</a>) and enhances LD decay. Our results showed a generally low LD, the maximum genome-wide average having an <em>r</em><sup>2</sup> value of 0.4. This LD diminished quickly in our population to 0.2 when two loci were 40 kb apart, and down to 0.1 when the loci were 220 kb apart, a decay rate that is comparable to those observed in natural populations (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Morris <em>et al.</em> 2013</a>). The faster LD decay achieved in our population through the multiple RM cycles was made possible by random mating through the use of the GMS system. Faster LD decay rate means that the haplotype blocks are smaller due to enhanced recombination, and that the loci that remain in LD at this point are truly linked. This is the origin of the high-resolution mapping commonly implied in association studies.</p><p id="p-34">By remapping some of the previously identified plant height genes—<em>DWARF1</em> (chromosome 9), <em>DWARF2</em> (chromosome 6), and <em>DWARF3</em> (chromosome 7) (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6, b–d</a>)—we show the potential of the sorghum MAGIC population as a powerful mapping resource. An early genetic study found four loci affecting sorghum height: <em>DWARF1</em>, <em>DWARF2</em>, <em>DWARF3</em>, and <em>DWARF4</em> (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Quinby and Karper 1954</a>). The <em>DWARF3</em> gene lying within 58.56–58.57 Mbp on chromosome 7 has been cloned and it encodes an auxin efflux carrier, PGP19 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Multani <em>et al.</em> 2003</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Higgins <em>et al.</em> 2014</a>). We mapped a plant height QTL within a 56–58 Mbp region on chromosome 7 (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6c</a>). Recently, <em>DWARF1</em>, located within 55–57 Mbp on chromosome 9, was cloned and found to encode a putative membrane protein of unknown function that is highly conserved in plants (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hilley <em>et al.</em> 2016</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Yamaguchi <em>et al.</em> 2016</a>). The present study mapped one plant height QTL within the same region of 56–57 Mbp (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6d</a>). Additionally, <em>DWARF2</em> has been found to be associated with a QTL on chromosome 6, but this QTL has not yet been cloned (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Feltus <em>et al.</em> 2006</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Klein <em>et al.</em> 2008</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Mace and Jordan 2011</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Upadhyaya <em>et al.</em> 2012</a>; <a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Morris <em>et al.</em> 2013</a>). These studies suggested that <em>DWARF2</em> is positioned within 43–45 Mbp on chromosome 6, but our QTL on the same chromosome is positioned within 56–57 Mbp, &gt; 10 Mbp away, indicating the new plant height QTL (<em>QTL-6</em>) that we detected on chromosome 6. On the other hand, <em>DWARF4</em> has not been genetically mapped conclusively to a linkage group, although some reports indicate a probable location on chromosome 9 (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Brown <em>et al.</em> 2008</a>).</p><p id="p-35">We further examined the Q-Q plots of three commonly used GWAS models that are based on GLM and MLM algorisms to evaluate spurious associations that could emanate from residual undetected structure in our MAGIC panel. All three models (Naïve, Q-model, and Q+K-model) did not show any extreme deviation from expectations. In GWAS, early departure from the diagonal identity line of a Q-Q plot usually suggests that the assumed distribution is not correct and is a signal for the presence of false associations (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pearson and Manolio 2008</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Voorman <em>et al.</em> 2011</a>). The Q+K-model showed a generally more reduced statistical significance than the other two models, with the association peak for the <em>QTL-6</em> locus on chromosome 6 not reaching a genome-wide statistical threshold.</p><p id="p-36">In summary, this sorghum MAGIC population provides unique opportunities. It has highly dense genome-wide polymorphic alleles spanning gene-rich regions. Recombination, enhanced through random mating, has improved its mapping resolution, showing no evidence of population stratification. Its utility for gene mapping has been demonstrated by remapping two previously identified plant height loci on chromosomes 7 and 9, and a new height QTL on chromosome 6. The results reported suggest that the new sorghum MAGIC population is a useful resource for the further exploitation of traits whose variation is captured in the population. Highly inbred and genetically diverse, the immortal lines that make up the population would serve as potential sources of germplasm for improved sorghum lines and hybrids. The sorghum crop is endowed with rich genetic diversity. With robust adaptive traits and a small genome size of 730 Mbp (∼25% the size of maize or sugarcane) that is fully sequenced (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Paterson <em>et al.</em> 2009</a>), sorghum is an attractive model for genomic studies of C4 grasses. The crop is related to many cereals including maize, such that new innovations, including dissection of causal variants in its genome, may enhance the use of syntenic knowledge with relative ease in homologous gene discoveries among related species (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hamblin <em>et al.</em> 2004</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Paterson 2008</a>). With more investment in further genetic studies in the crop, the sorghum genome could be an asset for the wider application of genomic tools toward enhancing crop productivity, as well as tolerance to biotic and abiotic stresses.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-37">We acknowledge the genotyping service provided by the Buckler Laboratory at Cornell University. The authors thank Terry Lemming for managing field activities during the development of the sorghum MAGIC population, and Patrick Rich, Daniel Gobena, and Adedayo Adeyanju for shared insights on sorghum research. This study was partially funded by the Sorghum and Millet Innovation Lab project, through a grant from the United States Agency for International Development. </p><p id="p-38">Author contributions: P.O.O. carried out all experiments and analyzed data. G.E. guided this study. P.O.O. and G.E. wrote paper. The authors declare no competing financial interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-39">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300248/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-40"><em>Communicating editor: K. Devos</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-09-07"><span class="received-label">Received </span>September 7, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-11-15"><span class="accepted-label">Accepted </span>November 15, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Ongom, Ejeta</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Apotikar</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venkateswarlu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghorade</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wadaskar</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patil</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mapping of shoot fly tolerance loci in sorghum using SSR markers.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bgenetics%26rft.stitle%253DJ%2BGenet%26rft.aulast%253DApotikar%26rft.auinit1%253DD.%2BB.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D59%26rft.epage%253D66%26rft.atitle%253DMapping%2Bof%2Bshoot%2Bfly%2Btolerance%2Bloci%2Bin%2Bsorghum%2Busing%2BSSR%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F21677382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21677382&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.2" data-doi="10.1186/1939-8433-6-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bandillo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muyco</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sevilla</span>  <span class="cit-name-given-names">M. A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lobina</span>  <span class="cit-name-given-names">I. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Multi-parent advanced generation inter-cross (MAGIC) populations in rice: progress and potential for genetics research and breeding.</span> <abbr class="cit-jnl-abbrev">Rice (N. Y.)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRice%2B%2528N.%2BY.%2529%26rft.volume%253D6%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1939-8433-6-1%26rft_id%253Dinfo%253Apmid%252F24280096%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1939-8433-6-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24280096&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beavis</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grant</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hallauer</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The influence of random mating on recombination among RFLP loci.</span> <abbr class="cit-jnl-abbrev">Maize Genet. Coop. News Lett.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMaize%2BGenet.%2BCoop.%2BNews%2BLett.%26rft.volume%253D66%26rft.spage%253D52%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.4" data-doi="10.1534/genetics.108.092239"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooney</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franks</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient mapping of plant height quantitative trait loci in a sorghum association population with introgressed dwarfing genes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.092239%26rft_id%253Dinfo%253Apmid%252F18757942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNjI5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.5" data-doi="10.1016/j.pbi.2008.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wayne</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">From mutations to MAGIC: resources for gene discovery, validation and delivery in crop plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">215</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DBeyer%26rft.auinit1%253DP.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D215%26rft.epage%253D221%26rft.atitle%253DFrom%2Bmutations%2Bto%2BMAGIC%253A%2Bresources%2Bfor%2Bgene%2Bdiscovery%252C%2Bvalidation%2Band%2Bdelivery%2Bin%2Bcrop%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2008.01.002%26rft_id%253Dinfo%253Apmid%252F18295532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2008.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18295532&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255623300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.6" data-doi="10.1007/s10681-005-1681-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collard</span>  <span class="cit-name-given-names">B. C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jahufer</span>  <span class="cit-name-given-names">M. Z. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brouwer</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pang</span>  <span class="cit-name-given-names">E. C. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">An introduction to markers, quantitative trait loci (QTL) mapping and marker-assisted selection for crop improvement: the basic concepts.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D142%26rft.spage%253D169%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-005-1681-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-005-1681-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000228875800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darvasi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soller</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Advanced intercross lines, an experimental population for fine genetic mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">1199</span>–<span class="cit-lpage">1207</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DDarvasi%26rft.auinit1%253DA.%26rft.volume%253D141%26rft.issue%253D3%26rft.spage%253D1199%26rft.epage%253D1207%26rft.atitle%253DAdvanced%2Bintercross%2Blines%252C%2Ban%2Bexperimental%2Bpopulation%2Bfor%2Bfine%2Bgenetic%2Bmapping.%26rft_id%253Dinfo%253Apmid%252F8582624%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0MS8zLzExOTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMzMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.8" data-doi="10.1186/s13059-014-0572-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dell’Acqua</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gatti</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pea</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genetic properties of the MAGIC maize population: a new platform for high definition QTL mapping in Zea mays.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-014-0572-2%26rft_id%253Dinfo%253Apmid%252F25583448%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-014-0572-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25583448&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.331.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eberhart</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Techniques and methods for more efficient population improvement in sorghum</span>, pp. <span class="cit-fpage">197</span>–<span class="cit-lpage">213</span> in <span class="cit-source">Sorghum in Seventies</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">N.</span></span>, <span class="cit-ed"><span class="cit-name-surname">House</span>  <span class="cit-name-given-names">L.</span></span>. <span class="cit-publ-name">Oxford and IBH Publishing Co.</span>, <span class="cit-publ-loc">New Delhi, India</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.10" data-doi="10.1371/journal.pone.0079192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCormick</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morishige</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weers</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Extensive variation in the density and distribution of DNA polymorphism in sorghum genomes.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e79192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De79192%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0079192%26rft_id%253Dinfo%253Apmid%252F24265758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0079192&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24265758&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.11" data-doi="10.1007/s00122-006-0232-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feltus</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hart</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schertz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kresovich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Alignment of genetic maps and QTLs between inter-and intra-specific sorghum populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">1295</span>–<span class="cit-lpage">1305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DFeltus%26rft.auinit1%253DF.%2BA.%26rft.volume%253D112%26rft.issue%253D7%26rft.spage%253D1295%26rft.epage%253D1305%26rft.atitle%253DAlignment%2Bof%2Bgenetic%2Bmaps%2Band%2BQTLs%2Bbetween%2Binter-%2Band%2Bintra-specific%2Bsorghum%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-006-0232-3%26rft_id%253Dinfo%253Apmid%252F16491426%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-006-0232-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16491426&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236585600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.12" data-doi="10.1017/S0080456800012163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1919</span> <span class="cit-article-title">The correlation between relatives on the supposition of Mendelian inheritance.</span> <abbr class="cit-jnl-abbrev">Trans. R. Soc. Edinb.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BR.%2BSoc.%2BEdinb.%26rft.volume%253D52%26rft.spage%253D399%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0080456800012163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0080456800012163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.331.13"><div class="cit-metadata unstructured">Garnier-Géré, P., and L. Chikhi, 2013 Population subdivision, Hardy–Weinberg equilibrium and the Wahlund effect. eLS DOI: 10.1002/9780470015902.a0005446.pub3.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.14" data-doi="10.1007/s11103-005-0257-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rustgi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulwal</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Linkage disequilibrium and association studies in higher plants: present status and future prospects.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DGupta%26rft.auinit1%253DP.%2BK.%26rft.volume%253D57%26rft.issue%253D4%26rft.spage%253D461%26rft.epage%253D485%26rft.atitle%253DLinkage%2Bdisequilibrium%2Band%2Bassociation%2Bstudies%2Bin%2Bhigher%2Bplants%253A%2Bpresent%2Bstatus%2Band%2Bfuture%2Bprospects.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-005-0257-z%26rft_id%253Dinfo%253Apmid%252F15821975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-005-0257-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15821975&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228284900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.15" data-doi="10.1534/genetics.167.1.471"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallego</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kukatla</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Comparative population genetics of the panicoid grasses: sequence polymorphism, linkage disequilibrium and selection in a diverse sample of Sorghum bicolor.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHamblin%26rft.auinit1%253DM.%2BT.%26rft.volume%253D167%26rft.issue%253D1%26rft.spage%253D471%26rft.epage%253D483%26rft.atitle%253DComparative%2BPopulation%2BGenetics%2Bof%2Bthe%2BPanicoid%2BGrasses%253A%2BSequence%2BPolymorphism%252C%2BLinkage%2BDisequilibrium%2Band%2BSelection%2Bin%2Ba%2BDiverse%2BSample%2Bof%2BSorghum%2Bbicolor%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.167.1.471%26rft_id%253Dinfo%253Apmid%252F15166170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY3LzEvNDcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harlan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martini</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1929</span> <span class="cit-article-title">A composite hybrid mixture.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">487</span>–<span class="cit-lpage">490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DHarlan%26rft.auinit1%253DH.%2BV.%26rft.volume%253D21%26rft.issue%253D4%26rft.spage%253D487%26rft.epage%253D490%26rft.atitle%253DA%2BComposite%2BHybrid%2BMixture%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.17" data-doi="10.1007/s00122-004-1706-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haussmann</span>  <span class="cit-name-given-names">B. I. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hess</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Omanya</span>  <span class="cit-name-given-names">G. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Folkertsma</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">B. V. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genomic regions influencing resistance to the parasitic weed Striga hermonthica in two recombinant inbred populations of sorghum.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">1005</span>–<span class="cit-lpage">1016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DHaussmann%26rft.auinit1%253DB.%2BI.%26rft.volume%253D109%26rft.issue%253D5%26rft.spage%253D1005%26rft.epage%253D1016%26rft.atitle%253DGenomic%2Bregions%2Binfluencing%2Bresistance%2Bto%2Bthe%2Bparasitic%2Bweed%2BStriga%2Bhermonthica%2Bin%2Btwo%2Brecombinant%2Binbred%2Bpopulations%2Bof%2Bsorghum.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1706-9%26rft_id%253Dinfo%253Apmid%252F15160241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1706-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15160241&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224528300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thurber</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Assaranurak</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Multiparental mapping of plant height and flowering time QTL in partially isogenic sorghum families.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1593</span>–<span class="cit-lpage">1602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D1593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hilley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Truong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morishige</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Identification of Dw1, a regulator of sorghum stem internode length.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">e0151271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0151271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">MAGIC maize: a new resource for plant genetics.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">163</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.21" data-doi="10.1111/j.1467-7652.2012.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">George</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayden</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A multiparent advanced generation inter-cross population for genetic analysis in wheat.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">826</span>–<span class="cit-lpage">839</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DHuang%26rft.auinit1%253DB.%2BE.%26rft.volume%253D10%26rft.issue%253D7%26rft.spage%253D826%26rft.epage%253D839%26rft.atitle%253DA%2Bmultiparent%2Badvanced%2Bgeneration%2Binter-cross%2Bpopulation%2Bfor%2Bgenetic%2Banalysis%2Bin%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2012.00702.x%26rft_id%253Dinfo%253Apmid%252F22594629%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2012.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22594629&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307051700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.22" data-doi="10.1007/s00122-015-2506-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">V. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">MAGIC populations in crops: current status and future prospects.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">999</span>–<span class="cit-lpage">1017</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D999%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-015-2506-0%26rft_id%253Dinfo%253Apmid%252F25855139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-015-2506-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25855139&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.23" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient control of population structure in model organism association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMzMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooney</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The effect of tropical sorghum conversion and inbred development on genome diversity as revealed by high-resolution genotyping.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">S12</span>–<span class="cit-lpage">S26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D48%26rft.spage%253DS12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knoll</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunaratna</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ejeta</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">QTL analysis of early-season cold tolerance in sorghum.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">577</span>–<span class="cit-lpage">587</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D116%26rft.issue%253D4%26rft.spage%253D577%26rft.epage%253D587%26rft.atitle%253DQTL%2Banalysis%2Bof%2Bearly-season%2Bcold%2Btolerance%2Bin%2Bsorghum.%26rft_id%253Dinfo%253Apmid%252F18097644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18097644&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.331.26"><div class="cit-metadata"><ol class="cit-ed-list"><li><span class="cit-ed"><span class="cit-name-surname">Kole</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, (Editor), <span class="cit-pub-date">2013</span> <span class="cit-source">Genomics and Breeding for Climate-Resilient Crops</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.27" data-doi="10.1371/journal.pgen.1000551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kover</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trakalo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scarcelli</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrenreich</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A multiparent advanced generation inter-cross to fine-map quantitative traits in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKover%26rft.auinit1%253DP.%2BX.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000551%26rft.epage%253De1000551%26rft.atitle%253DA%2BMultiparent%2BAdvanced%2BGeneration%2BInter-Cross%2Bto%2Bfine-map%2Bquantitative%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000551%26rft_id%253Dinfo%253Apmid%252F19593375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19593375&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.28" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.29" data-doi="10.1007/s00122-011-1575-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mace</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrating sorghum whole genome sequence information with a compendium of sorghum QTL studies reveals uneven distribution of QTL and of gene-rich regions with significant implications for crop improvement.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMace%26rft.auinit1%253DE.%2BS.%26rft.volume%253D123%26rft.issue%253D1%26rft.spage%253D169%26rft.epage%253D191%26rft.atitle%253DIntegrating%2Bsorghum%2Bwhole%2Bgenome%2Bsequence%2Binformation%2Bwith%2Ba%2Bcompendium%2Bof%2Bsorghum%2BQTL%2Bstudies%2Breveals%2Buneven%2Bdistribution%2Bof%2BQTL%2Band%2Bof%2Bgene-rich%2Bregions%2Bwith%2Bsignificant%2Bimplications%2Bfor%2Bcrop%2Bimprovement.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1575-y%26rft_id%253Dinfo%253Apmid%252F21484332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1575-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21484332&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bansept-Basler</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barber</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockram</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An eight-parent multiparent advanced generation inter-cross population for winter-sown wheat: creation, properties, and validation.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1603</span>–<span class="cit-lpage">1610</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D1603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.331.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maunder</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Objectives and approaches to grain and forage sorghum improvement in the Americas</span>, pp. <span class="cit-fpage">60</span>–<span class="cit-lpage">100</span> in <span class="cit-source">Sorghum in Seventies</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">N. G. P</span></span>, <span class="cit-ed"><span class="cit-name-surname">House</span>  <span class="cit-name-given-names">L. R</span></span>. <span class="cit-publ-name">Oxford &amp; IBH Publ. Co.</span>, <span class="cit-publ-loc">New Delhi, India</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.331.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">P. A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sir</span>  <span class="cit-name-given-names">R. A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-source">Commentary on R.A. Fisher’s Paper on the Correlation Between Relatives on the Supposition of Mendelian Inheritance</span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.33" data-doi="10.1073/pnas.1215985110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hash</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population genomic and genome-wide association studies of agroclimatic traits in sorghum.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">453</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1215985110%26rft_id%253Dinfo%253Apmid%252F23267105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTAvMi80NTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMzMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.34" data-doi="10.1126/science.1086072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Multani</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chamberlin</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blakeslee</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Loss of an MDR transporter in compact stalks of maize br2 and sorghum dw3 mutants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">302</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMultani%26rft.auinit1%253DD.%2BS.%26rft.volume%253D302%26rft.issue%253D5642%26rft.spage%253D81%26rft.epage%253D84%26rft.atitle%253DLoss%2Bof%2Ban%2BMDR%2BTransporter%2Bin%2BCompact%2BStalks%2Bof%2BMaize%2Bbr2%2Band%2BSorghum%2Bdw3%2BMutants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1086072%26rft_id%253Dinfo%253Apmid%252F14526073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIzMDIvNTY0Mi84MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzgvMS8zMzEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.35" data-doi="10.1038/nature07629"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Next-generation genetics in plants.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">720</span>–<span class="cit-lpage">723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DNordborg%26rft.auinit1%253DM.%26rft.volume%253D456%26rft.issue%253D7223%26rft.spage%253D720%26rft.epage%253D723%26rft.atitle%253DNext-generation%2Bgenetics%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07629%26rft_id%253Dinfo%253Apmid%252F19079047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07629&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19079047&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261559900034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.36" data-doi="10.1186/1471-2229-12-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pasam</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malosetti</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haseneyer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association studies for agronomical traits in a world wide spring barley collection.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DPasam%26rft.auinit1%253DR.%2BK.%26rft.volume%253D12%26rft.spage%253D16%26rft.epage%253D16%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bfor%2Bagronomical%2Btraits%2Bin%2Ba%2Bworld%2Bwide%2Bspring%2Bbarley%2Bcollection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-12-16%26rft_id%253Dinfo%253Apmid%252F22284310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-12-16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22284310&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genomics of sorghum.</span> <abbr class="cit-jnl-abbrev">Int. J. Plant Genomics</abbr> <span class="cit-vol">2008</span>: <span class="cit-fpage">362451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DInt%2BJ%2BPlant%2BGenomics%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D2008%26rft.spage%253D362451%26rft.epage%253D362451%26rft.atitle%253DGenomics%2Bof%2Bsorghum.%26rft_id%253Dinfo%253Apmid%252F18483564%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18483564&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.38" data-doi="10.1038/nature07723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruggmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubchak</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimwood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The Sorghum bicolor genome and the diversification of grasses.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D457%26rft.issue%253D7229%26rft.spage%253D551%26rft.epage%253D556%26rft.atitle%253DThe%2BSorghum%2Bbicolor%2Bgenome%2Band%2Bthe%2Bdiversification%2Bof%2Bgrasses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07723%26rft_id%253Dinfo%253Apmid%252F19189423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07723&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19189423&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262852200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.39" data-doi="10.1001/jama.299.11.1335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manolio</span>  <span class="cit-name-given-names">T. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">How to interpret a genome-wide association study.</span> <abbr class="cit-jnl-abbrev">JAMA</abbr> <span class="cit-vol">299</span>: <span class="cit-fpage">1335</span>–<span class="cit-lpage">1344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJAMA%26rft.stitle%253DJAMA%26rft.aulast%253DPearson%26rft.auinit1%253DT.%2BA.%26rft.volume%253D299%26rft.issue%253D11%26rft.spage%253D1335%26rft.epage%253D1344%26rft.atitle%253DHow%2Bto%2BInterpret%2Ba%2BGenome-wide%2BAssociation%2BStudy%26rft_id%253Dinfo%253Adoi%252F10.1001%252Fjama.299.11.1335%26rft_id%253Dinfo%253Apmid%252F18349094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1001/jama.299.11.1335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18349094&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254079100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.40" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzMzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.41" data-doi="10.2134/agronj1954.00021962004600050007x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quinby</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karper</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">Inheritance of height in sorghum.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DQuinby%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D46%26rft.issue%253D5%26rft.spage%253D211%26rft.epage%253D216%26rft.atitle%253DInheritance%2Bof%2BHeight%2Bin%2BSorghum%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj1954.00021962004600050007x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj1954.00021962004600050007x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1954UC65000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.42" data-doi="10.1016/j.pbi.2009.12.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rafalski</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Association genetics in crop improvement.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DRafalski%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D13%26rft.issue%253D2%26rft.spage%253D174%26rft.epage%253D180%26rft.atitle%253DAssociation%2Bgenetics%2Bin%2Bcrop%2Bimprovement.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2009.12.004%26rft_id%253Dinfo%253Apmid%252F20089441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2009.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20089441&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278303200010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.43" data-doi="10.1007/s12041-012-0144-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rakshit</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rakshit</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patil</span>  <span class="cit-name-given-names">J. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multiparent intercross populations in analysis of quantitative traits.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bgenetics%26rft.stitle%253DJ%2BGenet%26rft.aulast%253DRakshit%26rft.auinit1%253DS.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D111%26rft.epage%253D117%26rft.atitle%253DMultiparent%2Bintercross%2Bpopulations%2Bin%2Banalysis%2Bof%2Bquantitative%2Btraits.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12041-012-0144-8%26rft_id%253Dinfo%253Apmid%252F22546834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12041-012-0144-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22546834&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305207700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">B. V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stenhouse</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Sorghum improvement for semi-arid tropics region: past, current and future research thrusts in Asia.</span> <abbr class="cit-jnl-abbrev">PKV Res. J.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPKV%2BRes.%2BJ.%26rft.volume%253D18%26rft.spage%253D155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reddy</span>  <span class="cit-name-given-names">B. V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramesh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic and cytoplasmic-nuclear male sterility in sorghum.</span> <abbr class="cit-jnl-abbrev">Plant Breed. Rev.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%2BRev.%26rft.volume%253D25%26rft.spage%253D139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.331.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rooney</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Potential of marker-assisted selection for improving grain mold resistance in sorghum</span>, pp. <span class="cit-fpage">183</span>–<span class="cit-lpage">194</span> in <span class="cit-source">Technical and Institutional Options for sorghum Grain Mold Management : Proceedings of an International Consultation</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Chandrashekar</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Bandyopadhyay</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">A. J.</span></span>. <span class="cit-publ-name">International Crops Research Institute for the Semi-arid Tropics</span>, <span class="cit-publ-loc">Patancheru, India</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suneson</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1956</span> <span class="cit-article-title">An evolutionary plant breeding method.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">188</span>–<span class="cit-lpage">191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DSuneson%26rft.auinit1%253DC.%2BA.%26rft.volume%253D48%26rft.issue%253D4%26rft.spage%253D188%26rft.epage%253D191%26rft.atitle%253DAn%2BEvolutionary%2BPlant%2BBreeding%2BMethod%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-confproc" id="cit-8.1.331.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tabangin</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">L. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> The effect of minor allele frequency on the likelihood of obtaining false positives. <span class="cit-publ-name">B</span><span class="cit-publ-loc">MC Proc</span>. 3: S41.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.331.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Templeton</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source">Population Genetics and Microevolutionary Theory</span>. <span class="cit-publ-name">John Wiley &amp; Sons, Inc.</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.50" data-doi="10.1038/nature04226"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The International HapMap Consortium</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A haplotype map of the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">1299</span>–<span class="cit-lpage">1320</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%26rft.volume%253D437%26rft.issue%253D7063%26rft.spage%253D1299%26rft.epage%253D1320%26rft.atitle%253DA%2Bhaplotype%2Bmap%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04226%26rft_id%253Dinfo%253Apmid%252F16255080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04226&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16255080&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232829100044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.51" data-doi="10.1186/gb-2013-14-1-r1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thurber</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Retrospective genomic analysis of sorghum adaptation to temperate-zone grain production.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-1-r1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-1-r1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuinstra</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grote</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldsbrough</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ejeta</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Genetic analysis of post-flowering drought tolerance and components of grain development in Sorghum bicolor (L.)</span>. <abbr class="cit-jnl-abbrev">Moench. Mol. Breed.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMoench.%2BMol.%2BBreed.%26rft.volume%253D3%26rft.spage%253D439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Association mapping of height and maturity across five environments using the sorghum mini core collection.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">471</span>–<span class="cit-lpage">479</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DUpadhyaya%26rft.auinit1%253DH.%2BD.%26rft.volume%253D55%26rft.issue%253D6%26rft.spage%253D471%26rft.epage%253D479%26rft.atitle%253DAssociation%2Bmapping%2Bof%2Bheight%2Band%2Bmaturity%2Bacross%2Bfive%2Benvironments%2Busing%2Bthe%2Bsorghum%2Bmini%2Bcore%2Bcollection.%26rft_id%253Dinfo%253Apmid%252F22680231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22680231&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verbyla</span>  <span class="cit-name-given-names">K. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Whole-genome analysis of multienvironment or multitrait QTL in MAGIC.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1569</span>–<span class="cit-lpage">1584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D1569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vogler</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ejeta</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">L. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Inheritance of low production of striga germination stimulant in sorghum.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1185</span>–<span class="cit-lpage">1191</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DVogler%26rft.auinit1%253DR.%2BK.%26rft.volume%253D36%26rft.issue%253D5%26rft.spage%253D1185%26rft.epage%253D1191%26rft.atitle%253DInheritance%2Bof%2BLow%2BProduction%2Bof%2BStriga%2BGermination%2BStimulant%2Bin%2BSorghum%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1996VK30900020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.56" data-doi="10.1371/journal.pone.0019416"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lumley</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKnight</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Behavior of QQ-plots and genomic control in studies of gene-environment interaction.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19416</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DVoorman%26rft.auinit1%253DA.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19416%26rft.epage%253De19416%26rft.atitle%253DBehavior%2Bof%2BQQ-plots%2Band%2Bgenomic%2Bcontrol%2Bin%2Bstudies%2Bof%2Bgene-environment%2Binteraction.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019416%26rft_id%253Dinfo%253Apmid%252F21589913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019416&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21589913&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">A note on assortative mating, linkage and genotypic frequencies.</span> <abbr class="cit-jnl-abbrev">Ann. Hum. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DWilson%26rft.auinit1%253DS.%2BR.%26rft.volume%253D42%26rft.issue%253D1%26rft.spage%253D129%26rft.epage%253D130%26rft.atitle%253DA%2Bnote%2Bon%2Bassortative%2Bmating%252C%2Blinkage%2Band%2Bgenotypic%2Bfrequencies.%26rft_id%253Dinfo%253Apmid%252F686680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=686680&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamaguchi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujimoto</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirano</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Sorghum Dw1, an agronomically important gene for lodging resistance, encodes a novel protein involved in cell proliferation.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">28366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D28366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.331.59" data-doi="10.1038/ng.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">C.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todhunter</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiwari</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mixed linear model approach adapted for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D355%26rft.epage%253D360%26rft.atitle%253DMixed%2Blinear%2Bmodel%2Bapproach%2Badapted%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.546%26rft_id%253Dinfo%253Apmid%252F20208535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208535&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F331.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/1/331.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/315" title="Genome-Wide Mapping of Decay Factor–mRNA Interactions in Yeast Identifies Nutrient-Responsive Transcripts as Targets of the Deadenylase Ccr4" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/343" title="Caenorhabditis elegans DBL-1/BMP Regulates Lipid Accumulation via Interaction with Insulin Signaling" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/1"><span>Volume 8 Issue 1, January 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/1.cover-source.jpg?itok=vyoqxep2" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/331" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-wKbD9GvpcsWSVYbVylDLUU38anPkwqRatRZg_SYUl_4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/multiparental-populations" class="" data-icon-position="" data-hide-link-title="0">Multiparental Populations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/1/331" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/1/331" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/1/331" class="active" data-icon-position="" data-hide-link-title="0">Mating Design and Genetic Structure of a Multi-Parent Advanced Generation Intercross (MAGIC) Population of Sorghum (Sorghum bicolor (L.) Moench)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487832" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-kbjcHu56GCrnaGzwEya-oKfAfF4jwYpWunano2HPd3g" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/331.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/1/331" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-DRr8KQyv6W_nvqGgUZZCfQ_dfErDk5UzxMg1vJnG8ic" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/1/331" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/331.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487832" id="node487832--2" data-pisa="ggg;8/1/331" data-pisa-master="ggg;g3.117.300248" data-apath="/ggg/8/1/331.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mating Design and Genetic Structure of a Multi-Parent Advanced Generation Intercross (MAGIC) Population of Sorghum (<em>Sorghum bicolor</em> (L.) Moench)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Patrick O. Ongom</span> and <span class="highwire-citation-author" data-delta="1">Gebisa Ejeta</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">331-341;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300248 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487832--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Patrick O. Ongom </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BO.%2BOngom%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ongom%20PO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BO.%2BOngom%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gebisa Ejeta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Purdue University, West Lafayette, Indiana 47907</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Global Food Security, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gebisa%2BEjeta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ejeta%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGebisa%2BEjeta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gejeta@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">gejeta@purdue.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/1/331" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/1/331&amp;title=Mating%20Design%20and%20Genetic%20Structure%20of%20a%20Multi-Parent%20Advanced%20Generation%20Intercross%20%28MAGIC%29%20Population%20of%20Sorghum%20%28Sorghum%20bicolor%20%28L.%29%20Moench%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/1/331&amp;title=Mating%20Design%20and%20Genetic%20Structure%20of%20a%20Multi-Parent%20Advanced%20Generation%20Intercross%20%28MAGIC%29%20Population%20of%20Sorghum%20%28Sorghum%20bicolor%20%28L.%29%20Moench%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/1/331&amp;title=Mating%20Design%20and%20Genetic%20Structure%20of%20a%20Multi-Parent%20Advanced%20Generation%20Intercross%20%28MAGIC%29%20Population%20of%20Sorghum%20%28Sorghum%20bicolor%20%28L.%29%20Moench%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/1/331&amp;text=Mating%20Design%20and%20Genetic%20Structure%20of%20a%20Multi-Parent%20Advanced%20Generation%20Intercross%20%28MAGIC%29%20Population%20of%20Sorghum%20%28Sorghum%20bicolor%20%28L.%29%20Moench%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/1/331&amp;title=Mating%20Design%20and%20Genetic%20Structure%20of%20a%20Multi-Parent%20Advanced%20Generation%20Intercross%20%28MAGIC%29%20Population%20of%20Sorghum%20%28Sorghum%20bicolor%20%28L.%29%20Moench%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/1/331&amp;t=Mating%20Design%20and%20Genetic%20Structure%20of%20a%20Multi-Parent%20Advanced%20Generation%20Intercross%20%28MAGIC%29%20Population%20of%20Sorghum%20%28Sorghum%20bicolor%20%28L.%29%20Moench%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/1/331&amp;title=Mating%20Design%20and%20Genetic%20Structure%20of%20a%20Multi-Parent%20Advanced%20Generation%20Intercross%20%28MAGIC%29%20Population%20of%20Sorghum%20%28Sorghum%20bicolor%20%28L.%29%20Moench%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/1/331&amp;title=Mating%20Design%20and%20Genetic%20Structure%20of%20a%20Multi-Parent%20Advanced%20Generation%20Intercross%20%28MAGIC%29%20Population%20of%20Sorghum%20%28Sorghum%20bicolor%20%28L.%29%20Moench%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487832/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487832" id="node487832--3" data-pisa="ggg;8/1/331" data-pisa-master="ggg;g3.117.300248" data-apath="/ggg/8/1/331.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mating Design and Genetic Structure of a Multi-Parent Advanced Generation Intercross (MAGIC) Population of Sorghum (<em>Sorghum bicolor</em> (L.) Moench)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Patrick O. Ongom</span> and <span class="highwire-citation-author" data-delta="1">Gebisa Ejeta</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">331-341;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300248 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487832--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Patrick O. Ongom </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BO.%2BOngom%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ongom%20PO&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BO.%2BOngom%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gebisa Ejeta </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy, Purdue University, West Lafayette, Indiana 47907</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Center for Global Food Security, Purdue University, West Lafayette, Indiana 47907</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gebisa%2BEjeta%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ejeta%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGebisa%2BEjeta%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gejeta@purdue.edu" class="" data-icon-position="" data-hide-link-title="0">gejeta@purdue.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=331&amp;rft.epage=341&amp;rft.atitle=Mating%20Design%20and%20Genetic%20Structure%20of%20a%20Multi-Parent%20Advanced%20Generation%20Intercross%20%28MAGIC%29%20Population%20of%20Sorghum%20%28Sorghum%20bicolor%20%28L.%29%20Moench%29&amp;rft.volume=8&amp;rft.issue=1&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Ongom%2C+Patrick+O.&amp;rft.au=Ejeta%2C+Gebisa"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487832/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487832/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487832/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487832/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487832/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487832/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487832/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487832/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487832/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487832/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487832/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487832/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495847" id="node495847" data-pisa="ggg;10/3/951" data-pisa-master="ggg;g3.119.400763" data-apath="/ggg/10/3/951.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/951" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genome-Wide Association Study in Two Cohorts from a Multi-generational Mouse Advanced Intercross Line Highlights the Difficulty of Replication Due to Study-Specific Heterogeneity</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="494983" id="node494983" data-pisa="ggg;9/12/4045" data-pisa-master="ggg;g3.119.400353" data-apath="/ggg/9/12/4045.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/12/4045" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Genetic Architecture of Chilling Tolerance in Sorghum Dissected with a Nested Association Mapping Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493460" id="node493460" data-pisa="ggg;9/7/2317" data-pisa-master="ggg;g3.119.400098" data-apath="/ggg/9/7/2317.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/9/7/2317" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Testing Pleiotropy <em>vs.</em> Separate QTL in Multiparental Populations</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Multiparental%20Populations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Multiparental Populations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/1/331" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/1/331.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/1/331.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/1/331.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/1/331.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
