<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/1/113" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/1/113.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/1/113.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/1/113.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/1/113/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/1/113.atom" />
<meta name="HW.pisa" content="ggg;8/1/113" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Which Individuals To Choose To Update the Reference Population? Minimizing the Loss of Genetic Diversity in Animal Genomic Selection Programs" />
<meta name="DC.Identifier" content="10.1534/g3.117.1117" />
<meta name="DC.Date" content="2018-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Eynard et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection (GS) is commonly used in livestock and increasingly in plant breeding. Relying on phenotypes and genotypes of a reference population, GS allows performance prediction for young individuals having only genotypes. This is expected to achieve fast high genetic gain but with a potential loss of genetic diversity. Existing methods to conserve genetic diversity depend mostly on the choice of the breeding individuals. In this study, we propose a modification of the reference population composition to mitigate diversity loss. Since the high cost of phenotyping is the limiting factor for GS, our findings are of major economic interest. This study aims to answer the following questions: how would decisions on the reference population affect the breeding population, and how to best select individuals to update the reference population and balance maximizing genetic gain and minimizing loss of genetic diversity? We investigated three updating strategies for the reference population: random, truncation, and optimal contribution (OC) strategies. OC maximizes genetic merit for a fixed loss of genetic diversity. A French Montbéliarde dairy cattle population with 50K SNP chip genotypes and simulations over 10 generations were used to compare these different strategies using milk production as the trait of interest. Candidates were selected to update the reference population. Prediction bias and both genetic merit and diversity were measured. Changes in the reference population composition slightly affected the breeding population. Optimal contribution strategy appeared to be an acceptable compromise to maintain both genetic gain and diversity in the reference and the breeding populations." />
<meta name="DC.Contributor" content="Sonia E. Eynard" />
<meta name="DC.Contributor" content="Pascal Croiseau" />
<meta name="DC.Contributor" content="Denis Laloë" />
<meta name="DC.Contributor" content="Sebastien Fritz" />
<meta name="DC.Contributor" content="Mario P. L. Calus" />
<meta name="DC.Contributor" content="Gwendal Restoux" />
<meta name="article:published_time" content="2018-01-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Which Individuals To Choose To Update the Reference Population? Minimizing the Loss of Genetic Diversity in Animal Genomic Selection Programs" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection (GS) is commonly used in livestock and increasingly in plant breeding. Relying on phenotypes and genotypes of a reference population, GS allows performance prediction for young individuals having only genotypes. This is expected to achieve fast high genetic gain but with a potential loss of genetic diversity. Existing methods to conserve genetic diversity depend mostly on the choice of the breeding individuals. In this study, we propose a modification of the reference population composition to mitigate diversity loss. Since the high cost of phenotyping is the limiting factor for GS, our findings are of major economic interest. This study aims to answer the following questions: how would decisions on the reference population affect the breeding population, and how to best select individuals to update the reference population and balance maximizing genetic gain and minimizing loss of genetic diversity? We investigated three updating strategies for the reference population: random, truncation, and optimal contribution (OC) strategies. OC maximizes genetic merit for a fixed loss of genetic diversity. A French Montbéliarde dairy cattle population with 50K SNP chip genotypes and simulations over 10 generations were used to compare these different strategies using milk production as the trait of interest. Candidates were selected to update the reference population. Prediction bias and both genetic merit and diversity were measured. Changes in the reference population composition slightly affected the breeding population. Optimal contribution strategy appeared to be an acceptable compromise to maintain both genetic gain and diversity in the reference and the breeding populations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="ggg;8/1/113" />
<meta name="citation_id" content="8/1/113" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/1/113" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/1/113.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/1/113.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/1/113.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.1117" />
<meta name="citation_pmid" content="29133511" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="113" />
<meta name="citation_lastpage" content="121" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sonia E. Eynard" />
<meta name="citation_author_institution" content="Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France" />
<meta name="citation_author_institution" content="Animal Breeding and Genomics Centre, Wageningen University &amp; Research, 6700 AH Wageningen, The Netherlands" />
<meta name="citation_author_institution" content="Centre for Genetic Resources, Wageningen University &amp; Research, the Netherlands, 6700 AA Wageningen, The Netherlands" />
<meta name="citation_author_email" content="sonia.eynard@wur.nl" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8609-5869" />
<meta name="citation_author" content="Pascal Croiseau" />
<meta name="citation_author_institution" content="Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France" />
<meta name="citation_author" content="Denis Laloë" />
<meta name="citation_author_institution" content="Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France" />
<meta name="citation_author" content="Sebastien Fritz" />
<meta name="citation_author_institution" content="Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France" />
<meta name="citation_author_institution" content="Allice, 75595 Paris Cedex 12, France" />
<meta name="citation_author" content="Mario P. L. Calus" />
<meta name="citation_author_institution" content="Animal Breeding and Genomics Centre, Wageningen University &amp; Research, 6700 AH Wageningen, The Netherlands" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3213-704X" />
<meta name="citation_author" content="Gwendal Restoux" />
<meta name="citation_author_institution" content="Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=F. W. Allendorf;citation_author=P. A. Hohenlohe;citation_author=G. Luikart;citation_title=Genomics and the future of conservation genetics.;citation_pages=697-709;citation_volume=11;citation_year=2010;citation_issue=10;citation_pmid=20847747;citation_doi=10.1038/nrg2844" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Bartholomé;citation_author=J. Van Heerwaarden;citation_author=F. Isik;citation_author=C. Boury;citation_author=M. Vidal;citation_title=Performance of genomic prediction within and across generations in maritime pine.;citation_pages=604;citation_volume=17;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=J. W. M. Bastiaansen;citation_author=A. Coster;citation_author=M. P. L. Calus;citation_author=J. A. M. van Arendonk;citation_author=H. Bovenhuis;citation_title=Association between polymorphism in the FTO gene and growth and carcass traits in pig crosses.;citation_pages=13-13;citation_volume=44;citation_year=2012;citation_pmid=22510482" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=D. Bates;citation_author=M. Machler;citation_author=B. M. Bolker;citation_author=S. C. Walker;citation_title=Fitting linear mixed-effects models using lme4.;citation_pages=1-48;citation_volume=67;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=D. Boichard;citation_author=V. Ducrocq;citation_author=S. Fritz;citation_title=Sustainable dairy cattle selection in the genomic era.;citation_pages=135-143;citation_volume=132;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. R. Browning;citation_author=B. L. Browning;citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.;citation_pages=1084-1097;citation_volume=81;citation_year=2007;citation_issue=5;citation_pmid=17924348;citation_doi=10.1086/521987" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=M. W. Bruford;citation_author=C. Ginja;citation_author=I. Hoffmann;citation_author=S. Joost;citation_author=P. Orozco-terWengel;citation_title=Prospects and challenges for the conservation of farm animal genomic resources, 2015–2025.;citation_pages=314;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_abbrev=Animal;citation_author=M. P. L. Calus;citation_title=Genomic breeding value prediction: methods and procedures.;citation_pages=157-164;citation_volume=4;citation_year=2010;citation_issue=2;citation_pmid=22443868;citation_doi=10.1017/S1751731109991352" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. P. L. Calus;citation_author=R. F. Veerkamp;citation_title=Accuracy of multi-trait genomic selection using different methods.;citation_pages=26-26;citation_volume=43;citation_year=2011;citation_pmid=21729282;citation_doi=10.1186/1297-9686-43-26" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=M. P. L. Calus;citation_author=A. C. Bouwman;citation_author=C. Schrooten;citation_author=R. F. Veerkamp;citation_title=Efficient genomic prediction based on whole-genome sequence data using split-and-merge Bayesian variable selection.;citation_pages=19;citation_volume=48;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=A. S. Clark;citation_author=B. P. Kinghorn;citation_author=J. M. Hickey;citation_author=J. H. J. Van der Werf;citation_title=The effect of genomic information on optimal contribution selection in livestock breeding programs.;citation_pages=44;citation_volume=45;citation_year=2013;citation_pmid=24171942;citation_doi=10.1186/1297-9686-45-44" />
<meta name="citation_reference" content="citation_journal_title=Rencontres Recherches Ruminants;citation_author=J. J. Colleau;citation_author=S. Fritz;citation_author=F. Guillaume;citation_author=A. Baur;citation_author=D. Dupassieux;citation_title=Simulating the potential of genomic selection in dairy cattle breeding.;citation_pages=419;citation_volume=16;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=P. Croiseau;citation_author=A. Legarra;citation_author=F. Guillaume;citation_author=S. Fritz;citation_author=A. Baur;citation_title=Fine tuning genomic evaluations in dairy cattle through SNP pre-selection with the Elastic-Net algorithm.;citation_pages=409-417;citation_volume=93;citation_year=2011;citation_issue=6;citation_pmid=22189606;citation_doi=10.1017/S0016672311000358" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. D. Daetwyler;citation_author=R. Pong-Wong;citation_author=B. Villanueva;citation_author=J. A. Woolliams;citation_title=The impact of genetic architecture on genome-wide evaluation methods.;citation_volume=185;citation_year=2010;citation_pmid=20407128;citation_doi=10.1534/genetics.110.116855" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=C. Danchin-Burge;citation_author=G. Leroy;citation_author=M. Brochard;citation_author=S. Moureaux;citation_author=E. Verrier;citation_title=Evolution of the genetic variability of eight French dairy cattle breeds assessed by pedigree analysis.;citation_pages=206-217;citation_volume=129;citation_year=2012;citation_issue=3;citation_pmid=22583325" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. De Beukelaer;citation_author=Y. Badke;citation_author=V. Fack;citation_author=G. De Meyer;citation_title=Moving beyond managing realized genomic relationship in long-term genomic selection.;citation_volume=206;citation_year=2017;citation_pmid=28381589;citation_doi=10.1534/genetics.116.194449" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=M. A. R. de Cara;citation_author=B. Villanueva;citation_author=M. A. Toro;citation_author=J. Fernandez;citation_title=Using genomic tools to maintain diversity and fitness in conservation programmes.;citation_pages=6091-6099;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12560" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Druet;citation_author=M. Georges;citation_title=A hidden Markov model combining linkage and linkage disequilibrium information for haplotype reconstruction and quantitative trait locus fine mapping.;citation_volume=184;citation_year=2010;citation_pmid=20008575;citation_doi=10.1534/genetics.109.108431" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=H. Eding;citation_author=T. H. E. Meuwissen;citation_title=Marker-based estimates of between and within population kinships for the conservation of genetic diversity.;citation_pages=141-159;citation_volume=118;citation_year=2001;citation_doi=10.1046/j.1439-0388.2001.00290.x" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=K. A. Engelsma;citation_author=R. F. Veerkamp;citation_author=M. P. L. Calus;citation_author=J. J. Windig;citation_title=Consequences for diversity when prioritizing animals for conservation with pedigree or genomic information.;citation_pages=473-481;citation_volume=128;citation_year=2011;citation_issue=6;citation_pmid=22059581" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=K. A. Engelsma;citation_author=R. F. Veerkamp;citation_author=M. P. L. Calus;citation_author=P. Bijma;citation_author=J. J. Windig;citation_title=Pedigree- and marker-based methods in the estimation of genetic diversity in small groups of Holstein cattle.;citation_pages=195-205;citation_volume=129;citation_year=2012;citation_issue=3;citation_pmid=22583324" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=S. E. Eynard;citation_author=J. J. Windig;citation_author=G. Leroy;citation_author=R. van Binsbergen;citation_author=M. P. L. Calus;citation_title=The effect of rare alleles on estimated genomic relationships from whole genome sequence data.;citation_pages=12;citation_volume=16;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=S. E. Eynard;citation_author=J. J. Windig;citation_author=S. J. Hiemstra;citation_author=M. P. L. Calus;citation_title=Whole-genome sequence data uncover loss of genetic diversity due to selection.;citation_pages=33;citation_volume=48;citation_year=2016" />
<meta name="citation_reference" content="citation_title=Introduction to Quantative Genetics;citation_author=D. S. Falconer;citation_author=T. F. C. Mackay;citation_year=1996" />
<meta name="citation_reference" content="citation_title=Inbreeding and Brood Stock Management;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=W. F. Fikse;citation_author=G. Banos;citation_title=Weighting Factors of Sire Daughter Information in International Genetic Evaluations;citation_pages=1759-1767;citation_volume=84;citation_year=2001;citation_issue=7;citation_pmid=11467826" />
<meta name="citation_reference" content="citation_title=An R Companion to Applied Regression;citation_author=J. Fox;citation_author=S. Weisberg;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=M. E. Goddard;citation_title=Genomic selection: prediction of accuracy and maximisation of long term response.;citation_pages=245-257;citation_volume=136;citation_year=2009;citation_pmid=18704696;citation_doi=10.1007/s10709-008-9308-0" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=M. Henryon;citation_author=P. Berg;citation_author=A. C. Sørensen;citation_title=Invited review: animal-breeding schemes using genomic information need breeding plans designed to maximise long-term genetic gains.;citation_pages=38-47;citation_volume=166;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=N. Heslot;citation_author=J. Rutkoski;citation_author=J. Poland;citation_author=J. L. Jannink;citation_author=M. E. Sorrells;citation_title=Impact of marker ascertainment bias on genomic selection accuracy and estimates of genetic diversity.;citation_pages=e74612;citation_volume=8;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=C. Hoze;citation_author=M. N. Fouilloux;citation_author=E. Venot;citation_author=F. Guillaume;citation_author=R. Dassonneville;citation_title=High-density marker imputation accuracy in sixteen French cattle breeds.;citation_pages=33;citation_volume=45;citation_year=2013;citation_pmid=24004563" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=J. Isidro;citation_author=J. L. Jannink;citation_author=D. Akdemir;citation_author=J. Poland;citation_author=N. Heslot;citation_title=Training set optimization under population structure in genomic selection.;citation_pages=145-158;citation_volume=128;citation_year=2015;citation_pmid=25367380;citation_doi=10.1007/s00122-014-2418-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=J. L. Jannink;citation_title=Dynamics of long-term genomic selection.;citation_pages=35-35;citation_volume=42;citation_year=2010;citation_pmid=20712894;citation_doi=10.1186/1297-9686-42-35" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. S. Khatkar;citation_author=G. Moser;citation_author=B. J. Hayes;citation_author=H. W. Raadsma;citation_title=Optimizing Illumina next-generation sequencing library preparation for extremely AT-biased genomes.;citation_pages=1-1;citation_volume=13;citation_year=2012;citation_pmid=22214261;citation_doi=10.1186/1471-2164-13-1" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=S. Konig;citation_author=H. Simianer;citation_author=A. Willam;citation_title=Economic evaluation of genomic breeding programs;citation_pages=382-391;citation_volume=92;citation_year=2009;citation_issue=1;citation_pmid=19109296;citation_doi=10.3168/jds.2008-1310" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. Laloe;citation_title=Precision and information in linear-models of genetic evaluation.;citation_pages=557-576;citation_volume=25;citation_year=1993;citation_doi=10.1051/gse:19930604" />
<meta name="citation_reference" content="citation_author=A. Legarra;citation_author=A. Ricard;citation_author=O. Filangi;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=R. V. Lenth;citation_title=Least-squares means: the R package lsmeans.;citation_pages=1-33;citation_volume=69;citation_year=2016;citation_doi=10.18637/jss.v069.i01" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=G. Leroy;citation_author=C. Danchin-Burge;citation_author=E. Verrier;citation_title=Impact of the use of cryobank samples in a selected cattle breed: a simulation study.;citation_pages=36-36;citation_volume=43;citation_year=2011;citation_pmid=22047533" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=Y. Li;citation_author=H. N. Kadarmideen;citation_author=J. C. M. Dekkers;citation_title=Selection on multiple QTL with control of gene diversity and inbreeding for long-term benefit.;citation_pages=320-329;citation_volume=125;citation_year=2008;citation_issue=5;citation_pmid=18803787;citation_doi=10.1111/j.1439-0388.2007.00717.x" />
<meta name="citation_reference" content="citation_journal_title=Animal;citation_author=M. S. Lund;citation_author=I. van den Berg;citation_author=P. Ma;citation_author=R. F. Brøndum;citation_author=G. Su;citation_title=How to improve genomic predictions in small dairy cattle populations.;citation_pages=1042-1049;citation_volume=10;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Anim. Biosci.;citation_author=T. Meuwissen;citation_author=B. Hayes;citation_author=M. Goddard;citation_title=Accelerating improvement of livestock with genomic selection.;citation_pages=221-237;citation_volume=1;citation_year=2013;citation_pmid=25387018;citation_doi=10.1146/annurev-animal-031412-103705" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=T. H. E. Meuwissen;citation_title=Maximizing the response of selection with a predefined rate of inbreeding;citation_pages=934-940;citation_volume=75;citation_year=1997;citation_issue=4;citation_pmid=9110204" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Livest. Prod. Sci.;citation_author=R. A. Mrode;citation_author=G. J. Swanson;citation_title=Calculating cow and daughter yield deviations and partitioning of genetic evaluations under a random regression model.;citation_pages=253-260;citation_volume=86;citation_year=2004;citation_doi=10.1016/j.livprodsci.2003.09.001" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=A. Nejati-Javaremi;citation_author=C. Smith;citation_author=J. P. Gibson;citation_title=Effect of total allelic relationship on accuracy of evaluation and response to selection;citation_pages=1738-1745;citation_volume=75;citation_year=1997;citation_issue=7;citation_pmid=9222829" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=G. Ni;citation_author=D. Cavero;citation_author=A. Fangmann;citation_author=M. Erbe;citation_author=H. Simianer;citation_title=Whole-genome sequence-based genomic prediction in laying chickens with different genomic relationship matrices to account for genetic architecture.;citation_pages=8;citation_volume=49;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Anim. Prod. Sci.;citation_author=J. E. Pryce;citation_author=H. D. Daetwyler;citation_title=Designing dairy cattle breeding schemes under genomic selection: a review of international research.;citation_pages=107-114;citation_volume=52;citation_year=2012;citation_doi=10.1071/AN11098" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. Pszczola;citation_author=H. A. Mulder;citation_author=M. P. L. Calus;citation_title=Effect of enlarging the reference population with (un)genotyped animals on the accuracy of genomic selection in dairy cattle.;citation_pages=431-441;citation_volume=94;citation_year=2011;citation_issue=1;citation_pmid=21183054;citation_doi=10.3168/jds.2009-2840" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. Pszczola;citation_author=T. Strabel;citation_author=H. A. Mulder;citation_author=M. P. L. Calus;citation_title=Reliability of direct genomic values for animals with different relationships within and to the reference population.;citation_pages=389-400;citation_volume=95;citation_year=2012;citation_issue=1;citation_pmid=22192218;citation_doi=10.3168/jds.2011-4338" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=M. Pszczola;citation_author=T. Strabel;citation_author=J. A. M. van Arendonk;citation_author=M. P. L. Calus;citation_title=The impact of genotyping different groups of animals on accuracy when moving from traditional to genomic selection.;citation_pages=5412-5421;citation_volume=95;citation_year=2012;citation_issue=9;citation_pmid=22916948" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Rincent;citation_author=D. Laloe;citation_author=S. Nicolas;citation_author=T. Altmann;citation_author=D. Brunel;citation_title=Maximizing the reliability of genomic selection by optimizing the calibration set of reference individuals: comparison of methods in two diverse groups of maize inbreds (Zea mays L.).;citation_volume=192;citation_year=2012;citation_pmid=22865733;citation_doi=10.1534/genetics.112.141473" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Sargolzaei;citation_author=F. S. Schenkel;citation_title=QMSim: a large-scale genome simulator for livestock.;citation_volume=25;citation_year=2009;citation_pmid=19176551;citation_doi=10.1093/bioinformatics/btp045" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. Sargolzaei;citation_author=H. Iwaisaki;citation_author=J. J. Colleau;citation_title=A fast algorithm for computing inbreeding coefficients in large populations.;citation_pages=325-331;citation_volume=122;citation_year=2005;citation_issue=5;citation_pmid=16191041;citation_doi=10.1111/j.1439-0388.2005.00538.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=A. K. Sonesson;citation_author=J. A. Woolliams;citation_author=T. H. Meuwissen;citation_title=Genomic selection requires genomic control of inbreeding.;citation_pages=27-27;citation_volume=44;citation_year=2012;citation_issue=1;citation_pmid=22898324;citation_doi=10.1186/1297-9686-44-27" />
<meta name="citation_reference" content="citation_journal_title=Reprod. Domest. Anim.;citation_author=K. F. Stock;citation_author=R. Reents;citation_title=Genomic selection: status in different species and challenges for breeding.;citation_pages=2-10;citation_volume=48;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Livest. Sci.;citation_author=M. A. Toro;citation_author=J. Fernandez;citation_author=A. Caballero;citation_title=Molecular characterization of breeds and its use in conservation.;citation_pages=174-195;citation_volume=120;citation_year=2009;citation_doi=10.1016/j.livsci.2008.07.003" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=R. van Binsbergen;citation_author=M. P. L. Calus;citation_author=M. C. A. M. Bink;citation_author=F. A. van Eeuwijk;citation_author=C. Schrooten;citation_title=Genomic prediction using imputed whole-genome sequence data in Holstein Friesian cattle.;citation_pages=71;citation_volume=47;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=I. van den Berg;citation_author=D. Boichard;citation_author=M. S. Lund;citation_title=Sequence variants selected from a multibreed GWAS can improve the reliability of genomic predictions in dairy cattle.;citation_pages=83;citation_volume=48;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. A. Yang;citation_author=B. Benyamin;citation_author=B. P. McEvoy;citation_author=S. Gordon;citation_author=A. K. Henders;citation_title=Common SNPs explain a large proportion of the heritability for human height.;citation_pages=565-569;citation_volume=42;citation_year=2010;citation_issue=7;citation_pmid=20562875;citation_doi=10.1038/ng.608" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Which Individuals To Choose To Update the Reference Population? Minimizing the Loss of Genetic Diversity in Animal Genomic Selection Programs" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection (GS) is commonly used in livestock and increasingly in plant breeding. Relying on phenotypes and genotypes of a reference population, GS allows performance prediction for young individuals having only genotypes. This is expected to achieve fast high genetic gain but with a potential loss of genetic diversity. Existing methods to conserve genetic diversity depend mostly on the choice of the breeding individuals. In this study, we propose a modification of the reference population composition to mitigate diversity loss. Since the high cost of phenotyping is the limiting factor for GS, our findings are of major economic interest. This study aims to answer the following questions: how would decisions on the reference population affect the breeding population, and how to best select individuals to update the reference population and balance maximizing genetic gain and minimizing loss of genetic diversity? We investigated three updating strategies for the reference population: random, truncation, and optimal contribution (OC) strategies. OC maximizes genetic merit for a fixed loss of genetic diversity. A French Montbéliarde dairy cattle population with 50K SNP chip genotypes and simulations over 10 generations were used to compare these different strategies using milk production as the trait of interest. Candidates were selected to update the reference population. Prediction bias and both genetic merit and diversity were measured. Changes in the reference population composition slightly affected the breeding population. Optimal contribution strategy appeared to be an acceptable compromise to maintain both genetic gain and diversity in the reference and the breeding populations." />
<meta name="og-title" property="og:title" content="Which Individuals To Choose To Update the Reference Population? Minimizing the Loss of Genetic Diversity in Animal Genomic Selection Programs" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/1/113" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection (GS) is commonly used in livestock and increasingly in plant breeding. Relying on phenotypes and genotypes of a reference population, GS allows performance prediction for young individuals having only genotypes. This is expected to achieve fast high genetic gain but with a potential loss of genetic diversity. Existing methods to conserve genetic diversity depend mostly on the choice of the breeding individuals. In this study, we propose a modification of the reference population composition to mitigate diversity loss. Since the high cost of phenotyping is the limiting factor for GS, our findings are of major economic interest. This study aims to answer the following questions: how would decisions on the reference population affect the breeding population, and how to best select individuals to update the reference population and balance maximizing genetic gain and minimizing loss of genetic diversity? We investigated three updating strategies for the reference population: random, truncation, and optimal contribution (OC) strategies. OC maximizes genetic merit for a fixed loss of genetic diversity. A French Montbéliarde dairy cattle population with 50K SNP chip genotypes and simulations over 10 generations were used to compare these different strategies using milk production as the trait of interest. Candidates were selected to update the reference population. Prediction bias and both genetic merit and diversity were measured. Changes in the reference population composition slightly affected the breeding population. Optimal contribution strategy appeared to be an acceptable compromise to maintain both genetic gain and diversity in the reference and the breeding populations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
    <title>Which Individuals To Choose To Update the Reference Population? Minimizing the Loss of Genetic Diversity in Animal Genomic Selection Programs | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"aBqlZ6wLqGOxWjdCapFtXDOOl0FFyPgWebJIPaVausw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487828","apath":"\/ggg\/8\/1\/113.atom","pisa":"ggg;8\/1\/113","ac":{"\/ggg\/8\/1\/113.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/1\/113.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/1\/113"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/1\/113"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/1\/113":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022QjTY6hRibRsHALiKqXY+PnvA3tJuuDXnbcL3+kzVlS1q6cxDdsPNtdciq6YyiIXtyHfYJ\\\\\\\/MA5PD8rTeR4BKyLSOqicnxqP9em2gpjIWm3ip54vnyvPtjw38LFVSUbv1+v2b0uhwsxXHgDCOkwyyjhuMf6B0IctX3eQaj4nMd1QU\\\\\\\/bgZKnfgS9hGDOqpsKJzC3tWoUEFJTXxYBwXXKR++EQF57G+nwYwUVRo+cVKzG1SkwN7tgoHqXUloFH6dkxnUluADwyyJza\\\\\\\/1lYhgurS9RtK0\\\\\\\/z5tlLf40xBGVpNVnw1VAdElT7BpboK+vT0wXyAvmPrOiGfL5PnhHehZShlRBJ2zg63MPSTJCgNcjq7POUCv8AbVPSBAXSTQks8trh5DUuXhP80FzcEYTusKS9yKw3gFRSpW1tYxZUPimaCmfBZ4R8nDjv10LEJ5k2Ox12FdUrh+8t0xdzPZJjCNweFXeyUb4PJDhPFqV\\\\\\\/RVd+adOZumDWqNKtN\\\\\\\/VWGwuvMeLkF7E9vkAwZZ+TXtXn1m4vUn5v0KkT7rtmd7r2zmo9rBTalkqX692o2\\\\\\\/RPap9p4T544xtXBr4SGeGwlyDQXDcTGlUx1kK+QA+dIqvpMvB9Jrb06bpFVyS\\\\\\\/ahGZS8059Gv3ZLs\\\\\\\/HY5aQ8FYs6CyVqUGeohq0AsFvttYBUKl3W8AKqqW3+SnS0\\\\\\\/jVlgxcCILpcm7jxIC7k4yujc9mS5dwHT+PTWgYeMBCtXavN9v+wChNu5tpWIWezLjXb+53\\\\\\\/tw0YTtOtiC+I5tKiG1j2G7ljX\\\\\\\/yKpjkCrr2sCfw9lBBMgxUIC9XaoYwoEtwxt+OCdGmhDM+XKpmDGz\\\\\\\/8Gyi06aaaAJBUWG9mj1BrtCwB5fTCTcDXsJ\\\\\\\/y2YHy67bXWkT6DL\\\\\\\/u7\\\\\\\/92xIW8F\\\\\\\/tFIJtMbhIrVUNLRXYo2e39aYAvn9CzRRrZuDLVp+B9gCWdu+vukdBA2RuGZ0ScUyeqovZav7fy4e7HV3A+Ql1aQYdlghx3UC8BeZ9P3MUjTQL52NiKOw6BiihoD8DcQ0L7bCx7Cvegi\\\\\\\/E1Y\\\\\\\/dyZePj6wy7MQXmK1+E9S4\\\\\\\/LvSyqYFZXg\\\\\\\/t\\\\\\\/D9keZUievUQuNt6ZYKwPz6q8WcG63ADi+bx1FuVfbqDAYWlHDPTuyJsHkkhjnEw1ndvf5fYulStLQvMls0zPSrsNCel4Tl\\\\\\\/r4qYYbO9aoqBRq9rqJtmysjTP91+uNCR128wqhSscpW2UVWQvvD94jHUmAhLXgmeUxdRTXA60qPI8nAfKgCMDlkyGMgvWsaRnBbsIkL\\\\\\\/\\\u0022,\\\u0022iv\\\u0022:\\\u00220zOpii7ehUSmF6kyB329YQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a82cf5fe0a54ce6224df095bf9cdc32a1b53af448a797457f7913cf0399c59b5\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227jhQOXJ6AydVpwTkHkdNyiBA958ql14rI+4xhp4HjvKm1LCiXC4vqHLgDcD1aPrBuHj0bcNaZwJMMtSKSR2sTk2WiUh0YUCECH0c\\\\\\\/ja7QRJ1ZTdAaEeQP7C9gRzF\\\\\\\/tlBnVeqDXkzYxGTNcvY\\\\\\\/1+vBidqstKYcWR2IDI5YzDW4qJi6I8jcpjEtRX7PSMF3g\\\\\\\/FCpl0kucHVOHswj7b\\\\\\\/qN8\\\\\\\/wB\\\\\\\/u6YHMmVM1flWmVlVJdzRcGrX9ltbO8O+UerA8AAmSprzla4QAu7lLlqxebtryi96zOECouPaSWAcyxPyIekl7lZUoeCdqe7xajKWqDDgSlQJ3hon1nP27PrAN59uWBlbnvpNk8B\\\\\\\/H6TSdwCYW3xNyHtlFcTpIKOD7GDnFJ0B18sw52UIAEIF6xUQ04Iwe0DTu\\\\\\\/bCzVajB82n9e3bfJbdjL5g4Srk6\\\\\\\/Q2khTlYSXFiv0evrr+bytRI8XSKZhkgem2y3VGAfERQfYNBm\\\\\\\/cH9LbOZs9XlkhhjX4Og0sD9D+4kKC6tmxCuogBtwphzjN0H1CLwycnN1rriEje\\\\\\\/SSaco2O0k0vtWk7NPdwjl3SBracoGTD\\\\\\\/syur\\\\\\\/vXx+XFNBCFickpWrueZvQ9dic56lx5\\\\\\\/HGqe6Cj6Zvt\\\\\\\/zDq\\\\\\\/BSfLGSN8Lhl0b7ASEddBqgX6LlsdCW8ODyOyCsSn3\\\\\\\/ZOkYQeKmUNr2+5CgeKlhsLQ6T88u6qzV9c2bLXqsDyYAs2XfEcm8VjwQ1KFCwPGhdSWEa1MZMlTywy19uTINWi+SpapW+YMbuFPcfXT0f5qA7U4HiO8R2PH\\\\\\\/zeAlPGnKBe1vGBir+Ro\\\\\\\/eQeictec0BL+ByY\\\\\\\/reQFsgqx+mGSoeSB6rJxlqOwuGQTlwGIfWGwG7ST7W4QdvU6XxkZ0ojr24gWIWGc\\\\\\\/9xSUVPwVY\\\\\\\/\\\\\\\/63uxRDreUf+O1ywc86ykurx5vv\\\\\\\/5xQwqxWSj6XiuG7UVZU3aT\\\\\\\/6sGSkitRfDORWDLD\\\\\\\/OcpgfxBfKIo1n8Pv4BZDJgURHPGDwslMCWpQ1bqCvPFpuxWppLXCBqspAiLYFwnfntLpsAjyJ6mMP8cvM0EPP67XclROCqq+HEQixUtpAM529NTlPM65RidYv+VsLaMeW9lWNS85GXIVlxx6ojKwBYdBXhSmqonUgrPMTYsuZIjYHfHwDGSV6Ms48dd7oJQQPNNG3PBozGK7Uw3X2kFhBUOC5vhe88bGxlUyQWNC7UKFxDwq7De7OemZ9BGrYEDS\\\\\\\/JnZi+IopD7i0\\\\\\\/hK86r9tYJyV+HlqKzPK0cUI+saOJgzXTzK8OkBK2IlfgFF3h88dJn1PWdPZOqB5okl6n1G5KBBotJP\\\\\\\/vQZAUtK0Ym3jKRjaZAr4gTrBhvrrcIUuREcgrMj+\\\\\\\/h4Q\\\\\\\/zPJwNYm6AAbe5Lkk6jDsbh+fbZIgsqeCfbQwmppBpVMTMh0RjYxAMTjzj0z7a\\\\\\\/fQF3uHekZmmZwDdN1JhKaVBCAfntG1Unn1aEljhgB0ubirR+hyVUccuzJsWZbmAYZQE2lpdwTJ5RjIb2qy8pjvr6rgbDRmUR2p2xUuXUWbilWBejZHsJbzv\\\\\\\/nYzgakIF2Vz1TIwgKtvOQnKnbn0er\\\\\\\/9ytOxSCcUkOODrVtn6U9K67vA3b7CdeptHGGaDm1nAp+8j9GcXg\\\\\\\/7TZFmdc\\\\\\\/M6ahzYVhnEGhVsY3hANA9yFOyMgQlB83mxCzEzd1Hm6\\\\\\\/HyJpxxaMo54wYVa6Y+eNYtV7mn6NqOSWJgqseXMtfkCp6LK+tk6GFPBKaMdoqRK1arUQ2zwckYv4hNCsynX05tRuvUVzPfu6DKgbE9AX6Cvr31sy3tLQqX9atdAke7JILmTbkDFeAUSiBDe31Jhp8AwKXzApLRWSzIeZwtEQxzmNjyH6loeLJkMTkVA1dRSJPs9khdxZOCd8kYF0xx4F7NmJbp8+TZ+4y5FdcJtUQWKSHQLZMhkCLeqFu\\\\\\\/h\\\\\\\/ytP4qsvZ+OS2jgUeJXMjv7ZTaSvYPUNKm2sOq+Xze+NFIqjjsrQE7ksgR8U4\\\\\\\/kYWLVqZzpor8caBZKuqoTfAPxq8D+XsRTk1sdjiVPQvqiQZOTvTxRNBepc6iSEcPc6Ju2Sp7FPrPPVtuKTsg3wGMTRjYc\\\\\\\/7YFKMv4222oRoN7iE6taO1MAPSIZpt878rj8cOCTgI9LKuh3Ywgj9ARaJ2HBmic47H0og47kFPg\\\\\\\/p0rHJ8ze3BhqT2ihkX40knriONTmghh3uBJ1k0s6aK2NFOf3mQetHC4kl3CYiHejNojog4je66lf8hUbNGdw7BwWMNmnvS3YWhNjhCt1FVkJx2z9LqOKeQUIWNBU+VKsUHFhp1fBANAYfBAGQlB2crutPopACoeus5ltG8TQK9tkXMc2K0GCzJK3n6YXmJnc6VYAkSbj5YXVsbMUYigbnVw9UpTEWyGOFsdjxyVidISqSXBmL052m0ifMg9W+77de0cjjpte0CGkB7CNTdKcRoLLclsRjfSpP8h1aRdijxO+hr+5nVi+6QwlMGD+jmOtE7D7gNJxDlhoJL1VYVlOwLQfxcFmOtEw97SqWt\\\\\\\/4S6a3XkO1HbAgLKF78lqTuyf8\\\\\\\/ywAWm9UcfWr+zhpqsVVkoEBSYF6effEMHc\\\\\\\/5FXsPBiziB2GtxHgb00cLEi\\\\\\\/yGk82auxe+qgVY52VvXKchIID7tVNw0AQWs0FU9sW\\\\\\\/iFfgYefYxjeuSFuzt4Dz+AZB1GGLIgx+ue1yuGCl2Zck3o97LB95ZGH0uKEH8yBvUPPeSYeMJs6Ui3uO8rsUY6tFbxKh3Yct7GIfEV7agCdSqxfjyx3nitqFE8pxe3vK\\\\\\\/9cHHRp5V2muRp1YBF17OIDfdUsv2Jt5gTbT4\\\\\\\/65372Q4sGIGjn2wUcwqFks6tW0cgQ6EFIaHxt\\\\\\\/WaMxVR5K3A5KUSHJ50\\\\\\\/nCMBT8YHWi4KO34QzyX0HSIa96g1YrhPJm7lMd9FFY1aVOZXkVArtx4eJ2s49zQkZxYOyz301zYkRLyTG9AdgMkQcpz6RvrqwKQBVKzPTIMIZVYrAHGCYV+ZnCVzJvs+62cgEgqbGD4JZ9rPcpUdnIcaKE4PAAY\\\\\\\/ja1vEJU2j+VSYCPHfNfTKM5q1d8F8hIdmbhrBXpYoXRB01p0jfitydfJrDFfzVax7XkqdMAn5K7skYop8vawAdhLfvSUnCZChCQEn\\\\\\\/hewBDflBvcjw6L4ONCy+8YvWbr51sV8D2uF8HxNBA\\\\\\\/iF4JgK7Er2m\\\\\\\/ytNpzCNnH5HgL2McJXDVpCCdnekPl\\\\\\\/KacVuInd4D5u2tQWbtAScCqagYsmuJVEN5XjSQXQhX4z42Q2YF1jwoca2oAOIWt3kkQNZMlu2KfOJrLCzP7Yh8bRqHrJEYpmYA\\\\\\\/\\\\\\\/SkzS5\\\\\\\/EmGjUlj\\\\\\\/RgOOntzJdJp1EnIMdwgGS9YGKVGbkNx1UI1XqbjN6NI1Ac+Spb7r9N11yxuLZAdCZi3EPxRroKaJFgDQ43Y5pIWjgnv91lW4TTQYGuvb3j6Ql3eIraLzRd9XgUb87WfxOqy+KbkPpwkb8dULB\\\\\\\/HbhQzHBgHsE9l9r+lz1cBYzHjY7Y\\\\\\\/\\\\\\\/Erwj+LhaxK\\\\\\\/jW4l7J\\\\\\\/EtUQuDgBVB190mJvjDKvH7RdwL1k0z\\\\\\\/kyi8s0\\\\\\\/S\\\\\\\/84VEjhhVpIwIn+sr6VbeHRfZOrosi8XGhJbV03Td+0OdGNdFsU3lKqFkSv+ijVkmINwg5hvA38Y4yG63pDomVVwrx26ZYZme7pu2ATr2rBO4iHnZXD36JTXVcFQopQMzDxRdFlG1EWs+v+q\\\\\\\/BS1HU+QNponwtozjXm43O5nPISx6if834RXATJC4SyWO3R9TiruEiLf+g4Ga5ksPsO5jVoVj2mMhjJ0GwvpuuBuBKTb+FDasvK5cRx6yUICj0PZYD\\\\\\\/PX\\\\\\\/w9RjnJtfF9+OQIAB+TPmdX44xDjMVN2eSIP7kPGZsRxMg6v93sXnBWA+YtyabalkUKm0hPiEAdPWC0j\\\\\\\/pDtZt6oTagyESprNw0ScDn9njcQDBDfx0j0v9gy0g9Vx9xMYZ+Kkc0z7mv+Pgov\\\\\\\/11LTX7iiGQmFuAPWJY5hKdci92GF5yl9ornE+sgBeBCZq8YynWaWgPc70rRdXvZOvmBwg514QTk\\\\\\\/t3jfD\\\\\\\/BFDNkopljI7Nh\\\\\\\/YGyVaDYNOF8FvXnSC7puTvB\\\\\\\/t760KfDDE09ueMwoNy\\\\\\\/s4mFkl\\\\\\\/K7wq3b51anL9y68CI5RDQv7w7B43i\\\\\\\/LtujNZ6XnFZYPSJJ6X9Vp2zewvaKgS7SVOTNWBsTlSFpOa1mHjnn9Quw+Le1TbCvl0pj4yFu17vpgLIpnVkPn1P\\\\\\\/lFjzeKW4UJYFEVyUPkzgxXXsYM3b96p1qH11s6beRsMm768AgSU\\\\\\\/sNu+EPUHXGvQ7bJQHd\\\\\\\/gv9E5XmGoXiof9J3RepKHfrsVFTyabxT4uyib2YL0NYO6I8UE\\\\\\\/T0KauH0jQGLus09aJEWXMx+G2PH+BX4uLmCIK7AMQZ+W2vSlprI5BMCeWSs7zm2CUdwEPJZRv6Zsiv4WeLKPhOYmMp+eXA600CK\\\\\\\/hSoSoBXu+wUNvj\\\\\\\/BY3JE9M8rvfqlvrooo139OdzLy2gsRhukHOCs\\\\\\\/+Y1u6xgqUCF+uG4DEJJcJUKylW6P9h6uT\\\\\\\/Yd08wg0HWBXBvlaBJSssq5yy3yY9jly5d6wViYexhwUS\\\\\\\/AuMc6xz3ATkiFux9\\\\\\\/RvwCcs9DZ5rZweSgVqjuA6gOSeIOV7i6yQARPz1eNX4k3HGuzophJ9iztnLRp+2xV3sXKQaswLWaI0utMZKzQAnqqQCWxe2H+5\\\\\\\/kXYAYDVb033mJ5XKIrWWH65X9fuFp3O9lCYdyZ\\\\\\\/t3LCXY82RLQuOgTHQjxGZAixhj8XJMHnIrlw+lGG3FmkXcMhHwZP7Vx8pfyh7lBPGvEJLCPx6AXhcywAyUPGzBxDoXTGiGDOs9en9T723H5MR1NFzP6TA8f8nIPjx8CBd+HDgpYgh+BrR\\\\\\\/sLjCE4UecPDbXVRI8UOfXAcHY2PFXWGMX0foVWIM4a2VnIakWnShAVAnje9czFoyq52F4DErbjBeMIs421gdbBlwDj0FXF5As3NAAqbOQr2odCN1S1Rb0JrBHYkmOAqukwJgVPhBdqhAhIwul6gaBFMFL2uulrC1lwo3Aa+2ubkgJ5raqeIFRadwuJFT8IQ5eswk+Hc+Uy4ntp1772nJwF9xeh5alruJRkm0HabHjPFiHn4lUXBsr1EvR0kGkJDX91GpxjpWefIZTNYP\\\\\\\/6daGEqxHAa6nbOujq70eMmhEFRa6MvkGsZojpNRL0LoDrBfAz8INOTQ472RMAXzkR\\\\\\\/gx9HspHGXXm41PSPKSmHRI73NrzYu3tF1lNbsCcPAUbVBEvv4rLLNftShV+Dk27LZaF2JN72ER5XhHKFMTGNZhEvcS\\\\\\\/kEv4cZzfWexCBCLnlMcBtCMCfKhXYSS84wazJZuwBfgSitZjqSUAjvQg5xLqFE7FybqxChY5p8hidMkZjFqGGSih7QnxJdrHWg1MP+lojvFEzK9vOkS\\\\\\\/SSGRj7L+wNEiUrYciWfhwr7MB+PbU\\\\\\\/BWA5jm0tkJJZyS7be4ctimRGtOjXzDqJkCkOwcm+bDsIeojz27bVr7MxiK3FB5ZqQsoxvkV8TNr+3cc5+cWYU\\\\\\\/w3A7kdA50ALWl4cVlc7QnVt9FjS\\\\\\\/8mddps3nzBTFdXD0TWtldmXNkCR17gBChdCOP\\\\\\\/3ehxmYhHSqEF1E4Nm42GKUHlbl42By+sc1yrCLOztGeKnkDdHxfsqxnttRxan+Qj520vjAwLKO0SH78RV3tCAXRykm9xRVHiokOLYaxzTWnJ4x7wAfEV6Er\\\\\\\/YK4ZalN16QG8YI6YER6srD+zE23BDhBYXhhosMkQtBdL4kunZqAYlyBxfezmuuC\\\\\\\/irEt4Zt+Lg\\\\\\\/kj2xfDoccgbGfpms5Xx3HQKHMqc+EXA+b7KzPyx9UKY3rhFU\\\\\\\/MIzz4uCuexXiYKpSQOrwlf0Mh\\\\\\\/8ahJNjuI8ObnybF5GN2\\\\\\\/5Q15yKRu62IoMVe126Imdqh0\\\\\\\/1xatoAhd0XeMxE+0w2RuwIgh8XbsQh\\\\\\\/ztzcpVT66tENRERJxleu0p0OR0c01DgWc0i12jJ73BKL9lsk73zA+AHaRpYqOJeMLCkMmlmTk+dhIguqhGOSvZj4zqfe4m5KnvdbZ3GPxtYqMKBGEb43r5J1bBBYKGllDCyd7IeXGIQnBMYBtHoFBGAQS\\\\\\\/R78adKAd\\\\\\\/396KhCmTRXCcOUDktaDotxUV9llkVdpZx7R+oj6qFSTSzBJp6idtgpsZjjnj0J4+TfRYCDEfpPekg9p4fWNVwHD9NqEDhvkPVkBfpqc2kvItijFnQ+G2PBgWl0Ze7UtSaBuYHWQ+Q6mCtp4Y6PNufGrHcwBjq8PuQHNOe01i3SyRU\\\\\\\/LJ16h5rYCFscZUlFX3yJGzI7UFWdFB2zFH3dgRFGlKi0I35G44R6J4yPSazj\\\\\\\/yBsZaBihItTG2s0W8sYaqSmqv9GevBYKLc2ub0r2WGzU2yJS9I\\\\\\\/lSy\\\\\\\/pAxwahBAfHODV9leRc5a2rzUvwitHYGzDzquPJSkZ8W1\\\\\\\/BxH38Nj9mHmwMHbxAfDk06cCngYrKUSA6YCEvu9ZdaW8JFpGoj39eW31+DcNCfTnMjtH0Bl9bUypglYVx\\\\\\\/qq\\\\\\\/DkhNYx247+Whx4PWgULtoVzYuh3iTgZT73bLKDST2hjNEvHhOJEscEfC7pf+d209BJg6lwREoCzKUv8XLKItBG4sktgrD7Hhc\\\\\\\/gEv4FHy\\\\\\\/OWhBkSsto32m\\\\\\\/R5MdUcdtDfipTJXzMp5VfXnW3SAJKg8HGpyCCtrW6ZmULn+va8nqaYuWzwiiATles9UeatTDoOHuyTCvvBVMXe8i+qnGFpHdyv29nNbX4qpEoQSghrnReOXCBW19iTFl9VVHqhfS0R07UNHaM180NMnmKNMykVFwoguT7e8+mv16IoQvQ+Sm7QQthSMWNiDM95Y2GBPchrjLmeEo2bGFyvepAN+P0IX8\\\\\\\/X9SMQF+5Y+PpHwHSGp6OvJ09wzEw6f6IJtf80tFIdqI0pS6JA6cHm\\\\\\\/0qtZ3OxBT+HHz+1MU6tm9PRE3YODMKfuizcr1FxlHYXwJ0jiaIb0OWwVxCFBx\\\\\\\/IoBxuwnpvLm\\\\\\\/w+E5AwOXdK+S\\\\\\\/5yKSi+F6QBOrUGkyZVfvXoWOP9Jy1nlhfQ7g5p4M18jnWeEdIqn5rLOCCa9zOWH+Bc3hUXmeC8vEcwghykpP63N+1FxZC9B+lItGRCJG5i7cW5FG8nlZZOOsW9YdpXtZWvvMQjVasj+DvzbPAWj+dAenUXQP\\\\\\\/UNinREFU8pOJLigaWv7D2b6r3w61pkrht5NGrJepq7R1Gow18fr4FOpZG3XrQMt80RXm1JcraraY0YHugzGf6Qv6GsunP5nVE3p\\\\\\\/aOlmU+P0cal87rzLCIBcRqyRwPtuDzTZnjsI4rdKBjSyIcgZO7ebyDNEqcMnqaQxMStyxEZ8x30\\\\\\\/zWiwfhN\\\\\\\/xuwJ5NniZojiQC7s6VSNZJGW3li0BOlp6IuYFAjChruSKd5fgjbE0GR+R9XNebGkdpJTYCeiYTcCGknVo0gxFFkpiK+hxwCR0nKcdQlArCG9Fy+yjI\\\\\\\/jzFfCtSD0tzBb8DF\\\\\\\/GwC+euvMojZh4WZaAYYQGfznwmzqEYWD1O\\\\\\\/+ek3YxqKzqsRkv7Lt\\\\\\\/KeFQ4F84yKEgvYHwKlkwixz85XD\\\\\\\/uvtn7mrAJrVRL3h5OVGFf1pDnu9Kc2SW54K9QVxsmujHYLd8AsDuddIqCeMKHgsS5MbwlsfWBOHWnjH+Mk0Uwv3wM7tRx7uam8++Q78lvdJgh2oLqKDRRM5p635VtYIDmFR86mR8N8Ef8g5X9NjFOJADjFRAEcwoChnqC\\\\\\\/mOjSH\\\\\\\/YamGc85SWZBz\\\\\\\/yJ\\\\\\\/Mu\\\\\\\/E8iWxTsBBx0fXRDswsWyNX1AFgHemr9Il7zjG2sw1ChQ45v7ZyhQO44UfDau2ovDmzD8vaRIvrvJVahMZBDEiVxDwlKkATK8ARzenDLK+8csGnWQRtEMgmLVBgD1R5d+vGXRQb3wliQV2rP0\\\\\\\/MxozFx\\\\\\\/Pv+pFD4PGqD9Fp3\\\\\\\/RPrKkJdueaQV67BqcmtFX\\\\\\\/STUmjK3EoqQ6YNUR8TKYYUG+s6zAGO4oIKoWc\\\\\\\/uoFqbMBl1jIYYRGH1fMwWbHT6NDEE4+nS3M36V3is1h2ad3DxnT2ZtbG32CxtGZbZM+8v+CYMTdXRovvTb1aqQX3Ms0HtWopcgyJ2NL\\\\\\\/7fs6sQbdXvZpMq6ZC8lhwq8wFzM8bUK+2tMFKmoITlsuVQmECMlPX162w3QBeFKEP7OrBWclKd8oA5AbJsjQ10zo6I6rDT2hazLDxC0X\\\\\\\/w1\\\\\\\/+M3BAPRkdep+OeK7n9pdWRX2gzBnQRgcPmLgfMYJeZdL2ylypwDXzxfDmGpBHcQG8Hdk3CI6\\\\\\\/JD7jaiQ0pw+ZPdE89emt+KrdgPn96VvyKrzZ41v8x1x8nQrx9b43bJ61wXXikC1ooCKYDgAQ+\\\\\\\/cHm4pnt\\\\\\\/Pt8Untwan6EH\\\\\\\/U0FECD6ycCzN4Ujw+s10lr9KsdKut+J52MZhqUi2luaDidZY8ueXsNuxshLJBa7B10AKBVPreFjmkrGQIK+V5L0vo1FYHdgVbgOZHK1oPxtgBWpz12EugRVXD08AQuGEP9fb+KNumisDIw8kqdtPNoy47gMIm3azlv0qsAswt+rPx\\\\\\\/IOvc6n\\\\\\\/203uKc775AIUaiEJFVGpLD7CaqRms8TXsqDBF\\\\\\\/EDRG3ufbZvYtm8Mtgct5AC1D1Iso0eOsr\\\\\\\/S9SaeGzzKySaASlz44RSKHzVHBjqGimkthgN0fOy7gLlycXxCt51Sz+5vGjx30yyp3acKcKYUihU3YQl3yh1wSWX0L19lKoLCucHFMk0VgsLol84N1inldgvUnMemL4PrWylX\\\\\\\/Ydtk4PlF6ZTOSYwxwI1KtYOrOdhDsmHU3BHBGbvQRCJW\\\\\\\/LrOdkXGlMIxvi4QGPUw6fC\\\\\\\/g2NDnwWY2lUnNZkc7SKMVZ8K42Tgf1fmOdx8vhsGdhy+8k+bc98vdZ4OrZeMzaKajYJ+U82QjeWHPnN+BD3B+f01iR\\\\\\\/Rdnavr9Gud1COu6EA8sDYIWRh1Bi1UkMSKqw6xxWO9TUwH3KdYfK3AHOOKgl\\\\\\\/JbpoikpShr4Uf4FWF5w4GQgpbieU89FLcAc62HcqhSVgJI2NLxdJ\\\\\\\/btca5KiYj2Ieix8jk7h2gCV+aMYZVzFvN7vh2Yj5mdunzVMuBl\\\\\\\/vY\\\\\\\/ms8n9ChgCDIRuG5amYOd6IO45vHexfNny9\\\\\\\/JNzrotOPuaucaqR\\\\\\\/SmS8zGGVqMDMuDSxKP3Mk2UAtUrfJmA+9bAPbsR3sYkYZ+zHkxQTaWWs9nKuA+4ppMUpDPJ6WCSy4\\\\\\\/WMCXJZNUfXB6srVquRON7L9oM\\\\\\\/aTeftVCo+VveRMWBpBrcaYoqN2WVoFEQvHcU\\\\\\\/6n8mZlmbHrcRgBn4iiI+5UPmu0XQbpEtWF7sPf8xE\\\\\\\/tMhBOvElqBHZOnZhua1w2T6ItCyqSkSyvm4jpLWqmYwYKIXzJFB00GYlSI2BfLK+rwvosB699UhWWgRlBlJ4oxBFTLDvEC6N4A\\\\\\\/2LdrGWNHpJ8sVObXdYlWTNovOacWeVzf\\\\\\\/se3f58hfogXOwn2N4o+8ZnNmbEGgNHt4K9w+b51117qk0VElFWDuK4kmcjWf0R5RPJnSH49wal\\\\\\\/UHg2VYRMLglnF33zE2U5+ocJ0VdQbqcDu0wDLJkfv9fiyabIzJbM9CgUuqooq0xLyonXfjMMw2bMRsEgry0r5vtqew\\\\\\\/wqBY6rRmuUupTxq2P9vOQTpojEOxjNJYgyMok+XiK\\\\\\\/UWbEALCifMKrcYOA1gvrGCxD0zXwT7DsYZQjZYzC36TwQ09Zpz6AOdE5FIWErTLZU1vRiJMIOnwXo90q3RFqjT88gRB8\\\\\\\/0xRB1CFk5+qBZ+xdz5eSLrQurlsRE2kigzfa+MNwn01SscXlKn08eFB+Nryizjlko\\\\\\\/6738O+uNy0yBpfikXB3DgERJupZVGPAnIwSmJjGn1vBo3PjUWbKv2VMPFGdudDeI6upf+qbCvSBtBm12GxI8J07HKm1EY2srcVIvU1pMxh7M5\\\\\\\/FOgNA7fwD7n9zBXxrAF5ydmq4PMjFN2wrQtz5L31RLAcGjp9gG5Y40X8B0oUPQw4ghn78tBJi9\\\\\\\/XEqr7wxqczhgcTzAYHN1QMtNWg7JN112EhNCnN8Q2wwxqzGbzwzx4vUOfhEOapcCpunevZqZhBHZ2fZDF\\\\\\\/a0CdEx739\\\\\\\/YS2moa4O1VUkiodPqQ\\\\\\\/8xpDT1aNiABviljV8Bah+m2DME9fhHWJ9g\\\\\\\/HXrW5QZdHiL8UdqIL1t+WqewBF1CuHf2mzaGZsYlFJNmmFGT0RqgcStVm74fEYoG9pgLyrnOtNrVjDuTSWFnyZrdc7WUwCZNrImd54aE+FbOiGDWScumezCArsyyqod5nhDsNVxkhoPcFoqAiVXzyySGmEIiQ6qGI5\\\\\\\/wlWFE1RQHW8hVWycvZ2AcBKHC+qM4pmWfvCS5zAhMOKT7qbTPeH9pQrBEjzbxcYcg3kHLPpm\\\\\\\/kte4l+gjZ+hieATuhdJ0h6L2Ce5CGndW85FnQWdAEfOHgrZp1VT9wGAJVvRctFAXuMjuVfuDkaaicgfcaijBkDtxPHHntKoUVISHV6GCPfBJs8ahMnuhid1kYaRyTZi5cJvWrMIoDQq6mtT1SOTx9hmSlRgfSXgdvKxEDQX2cIMDksF9xolWQT45clKHTXJRQmt1uGi1uYrOU2SpzRPKEaMcfDU78+COyirMsFqLoddESZxlZHjtMF2F0J6vQidwP5rYGKAvqY5vOwODFsZ5uGrF+L\\\\\\\/eHFC81j1LR4i7ZMMorZfC0GnQnQiG\\\\\\\/TlQdqSbLgvZ9f\\\\\\\/WxaDSOCvO\\\\\\\/6kKgll2nOZwQLE7d1dmsW9UJUx5LZzWYDDt5g+ymumPEgJI8YzuxV49XMS4Oiw\\\\\\\/9i7mbWrdTewtamMENCCgoB2jQnRjpE3GQNKYFJMpqhYRle3nFxU9GN\\\\\\\/lpevrskaaZySaL3UysyRnkB9RQOUEkTTauAPX55ARbrWdi74nsyOl+LntLJyT+AczqvjccpQ1QH2aMC4qNgD7GJ9xj+QPb95bVE8epftPprGn\\\\\\\/YXphwGaL2tRvRcrhPy0cnCO6YpIJtYolutwPRnycjMt+YaAZT8szZnds4gE7lxFrJTDLyz52Hjx2hnNRfy+4IebQha4xPMPNuPd2mgyS7\\\\\\\/R5Rx3q\\\\\\\/L1E8sk9zm9Mwt1MAw7Z84jQr670Ki5QMdw9Ynv+ftYj5UNSdPkSqA53jurl6\\\\\\\/6VeehSTgRIgpU2LVMB7dYJV8GLC\\\\\\\/P3LID\\\\\\\/d0KP9qZxODT4OKD+H6WDojjoaegiE5Ue8uIyL\\\\\\\/GEuqteRpYTBg7GE8hOw3RUn1yNIJmggRI+ZfTPAVxT2ZDO8oPl710c2dyJR4BxbLSRhQVy43G4Bm3WRyKVhUxS2NfgcVbAJFXRNZ9lAYVKQzH2wM1AmPBqp89Jc12\\\\\\\/vp5zCV4V1d9kdH+GDkUS9OQG2iRQZ9HRs8QvKOJBJJT55lKut4Sbs26ncC3bZdaMfrnuM\\\\\\\/kVeLWJRdhKaSrwqGDbItthQL9HNFJpXu3uYjXotn\\\\\\\/OuqQN\\\\\\\/aTNQLKuyRwKaAxcl6JaUnqJmAL6rg\\\\\\\/fg4pIyVmmageNrS2MXdyo7cREn5eA+HuyRGq8Jf5RXBbWJkfqyOu0Cp+CYcdq\\\\\\\/BvBIVE6S4gPiizqbOYg8TZNRrZ9vonPPu94QiW+IL3P6WOxaQTJZheFB8YBNg44zplY7+0VSX7B4PbuGeU6zvrfE8tDMIWoSpiDp\\\\\\\/oJyVtQksZJaARcFlNI+1e\\\\\\\/Y0TACTxudP5TVa4NujcMht3V8weUQfImZznkSqFL6PH8wZu\\\\\\\/gnkXlpOF0ZRsCK7jFm24d5URQPhYPYNclOK3AgTIo5BL3baO87\\\\\\\/9wohEhpG947CdEDeSdv+3JDFN7IwL2SMnjBJA6vS4UyKR5BcAyT5sPrgjjf7VlwibqTScXkqfeoc1fSlLb3d\\\\\\\/OKj6mSNAw1A6rdgUudr4mzXBXyBF75NZuVUryzRYQzSOIUL+hTGWfAb6XgsT7EGXcUg4pSbYPnL5L8GOk6HZV+QhCBbzHcseiuJ+mScO41aG9JxQhr7BF5oW+sjzXiod7r3uR53Qm9C+5rNZ0lsM+z+xlBjTXUHU2AkG3cFpQ\\\\\\\/dN984ReIzPhJ3PqeHllQTu\\\\\\\/obrncFfDjpXJX66xOnUtzkY1G4ta8gH2h+vNWvs96luOXB4YYs8c0LjA\\\\\\\/gwTqzkJx+cGoKvLXOYRDcJYyVnZP\\\\\\\/zVSlTT0U9BNnkbSxo3aBuTC0SSf+8myOALleFiYByHUwgf3ogZoLac+RZLFW6wgkg4OSh1jzqpVPqFUxg8mnONcElex\\\\\\\/sqqIa0ig1P2aieZZMXEM0GzyqpUnuSxSl8jYuI5Sp23ZG0Zx+U2LO3nN4gPs6lBDqm7+rlzGRK3drlE00lxf9esz36HHlo9kx7U0D9S5+quqiHp8yW+I\\\\\\\/91pbivJ2ctIVq7Bl\\\\\\\/KJ41CzSL4ZBf1mlR\\\\\\\/xKscAUdljy3359Uu4xEXAjfIXUCD7xksaS0IbCBOxOTBhG5gb56cNhGD1dLnvps1EDYPxgbWAVVZKWfKJXTfr4VvqRIX5XyzSY5xgwdEaWl3Hr9K1WHbcq1pjCvhAYLn0Ioz8fAANiA5XMdXxxtEc9tgV0+QJVFHAO4VmzBanNRDofcFycIVlBTK30SUxszPwE6XJcxgn5xzbs1eJE9UmCrSuH6Q1AE8ujBv\\\\\\\/DPUVXDuVhHkcUv3dOFXcT3XgUNF2iacacxCgK8eLiOPHrF79UnWgg5WQ5PT2HcAyPVRKy2Rfict2XdVRBS\\\\\\\/Y629BMzbqbbX2kQVzAUMzkVCGrRNmnX44wnKe4IKlu6RISW50ZYaOAeC+hMJEGAtAjLvB3NLK5zokZ0ZdT6k9rS\\\\\\\/XcqYzhmoOLYXWHSmHJRfwIQdF69U3Lx7ITqIqby3DnBmBzjOqsfhTxOYZPL3L\\\\\\\/nKAa1eUUWobB1AcO2hKUjRNvOkdhBEWimJBYfDM3FknkzhO8ZPHJ9PChd8Q67CZYk9Smq7OKDG+GfhBXH54miBEqZbhREx4KUTUPpQmo16VgdO0JBNAq+pagUifi\\\\\\\/19Y8c8qV3x7gf55azUGt+Sa\\\\\\\/GfFQmdPzHTZd9K2EH69cizyyuvgvWtWB1Pn6OWNMZjkf66H9gyKTvkArOhB92+pcVms2\\\\\\\/WitKKKe2Mh50QMzquSggX\\\\\\\/yycpjKr92FMm6BE8MW2wH7tJicNp657fzVIgw6BsrDV6P\\\\\\\/M9tDlwYPzktnTPt3flhTINzx5qGmH7zoYfv5NAop4E2C7NshQJoOTERJoxhIZ9Hgt9WKtGhBIfPwb6Hua53HgIR7Aq5fuD+Vzv5PKMHgJZnsAdUQFcRax1wr3hSbFn2Swkv1cFAJP5Y01IFla+l6Gy1gPOHKrUovLUjNvzQsMgFsktT4sASOFTtoDePS7w51lWfAl9vanbA+13y1JFxjrckjABLr4Pe5ALXOpBGtFl5T+Lz9jUlbsTBxugVJOWXaX7qmOCxHhmh2UJIuQ\\\\\\\/YTQjGmmWysyLfUX6PPmUg84l4P7TCVhXYR+obgE8a0w\\\\\\\/92v9YgIeGApUUXaI7Jlu8VeqJ07qK1LfznXRDyHzP4VysadAeJuFWEOpswJNzdLcjmG1ZYhVTLMve9vCyJpI2VyNafoJ+HqvxiB7e4TtBikZYYwz3x1W4QGNhhgX4Mw9m7MfPKpLZNjKrj1CF0HFhF0J6pR7ss20YOReucINyWIQj6czSlJHTwU\\\\\\\/\\\\\\\/CSKsWbtuk2BN3KaaGNcmRJotmhFrCW496hyGX6VuFKe4MPIbN2nxW6x2T5Y4P7D5soExio3e8xiQHQdjxRonvyf4bCZ0WUOTGlU96+9VnGw6Qn1Se+fp0UoBhASm1k9dOM4MMvwH8fNBQF68bS6wxHFKv0HIwS6AFtmHpWB6HB6UhQYFW130z\\\\\\\/y4SRfCaH1Wk1d+6daUrSKQ3uCOYqm4Vn5CMg8T2Nnw\\\\\\\/Yoz7c\\\\\\\/oNHlMLVgDZIBWIushylL2fhA\\\\\\\/PDFN0bmye4u5Z8mx4OkHlqMxs7DdBuomG4jtSZnkTpOtSwZaermalrIHjEhLodLPztF0a0ZFxcYE3U0hSKzwCJ79wkzBoF+LoFP008pysn8YOd3zR4JGSLTWlvZNlQDgYWjyp7TxzW2ll9Lt\\\\\\\/1IMFtiM+pfNW7mWSBrHJlG\\\\\\\/qo+5lQJF9VM6oHLRT+BZcUmIQMYUOw3bFvzmQxmoYxDyU154TWway0slDXcmIwTqT8Y3D6YTwBcQBqlBbsDOpJh8+xQClZ800uf4fo0Uwr0kwNpcULME4kn0TDY3ODZ7P3b2SQ9Aj3wQgS4Lmzu+8nqM\\\\\\\/SlkohDJLkJtWJ1V7obSO64ee6SD9PKnghxZ6qKEV\\\\\\\/DmwnNR8nWDDE8qkagt0VVJiJYnN8ReF\\\\\\\/7ZocTFagn20I4TvlkGUv+LemZGloEt2zv8ccYaTovmQYP17lL09wN\\\\\\\/TFTs5oNLMUvqPyBu9l0guI0ymf1K0WisY4QcPH5TFgN66Pe\\\\\\\/C\\\\\\\/LQF\\\\\\\/O2ZBmPk68Sl425U4MNsu1FdAmCsmUM9+RE4xh5ZaSw81pBwOOtADZw00cjmM+Z7ZM8tAtV\\\\\\\/+\\\\\\\/Ba6iPMAGHaWqI3TxbrQ7SbIzIiitBr5M4EX+EIqrF2aRedBUwzFqjRkpaRISpttdr3OnGt3rKvZ97uqfRcWIaX0GLaG3ozwYPArkaNpw6pCxbLQuMjeJ3F9DFQeYHGxENbgbGS70A49w5QaS7OqP7aAFXRSpxyMfNS2wt7teUqreBI27BXUszDGihAYO1TUa9u1k75dXCZJjsedG11OVejVXJNbWgSAE8ueKSbCBaIQTS6fAv19eqRHPGIlTKYlNSFTQcs4m+JRTGc7\\\\\\\/72G0+YutzaBG5wypjaZP1DXlEg7DmV9ec\\\\\\\/+yrrkPa\\\\\\\/aFEGU5HOH7MCs31WxJIwAI1zW2YGRTF\\\\\\\/gRjhPKJRM1hAdGGLYacM5X0inrXycbBd+YKON4TtquKnqN1kjMZD\\\\\\\/HgACBLIGavQWAAiC2+yKP4VeCfQgeQHtAavC3XU5zWBrF\\\\\\\/ndk2ndHIpxe3BEE3VsRw8ycnbEH5F3MfbfyZ3Ec5RPTBOw70AsNK5UmU78Be22ogJ6OvW7rvmt\\\\\\\/NfOR\\\\\\\/y1blvVoKyFAX5i+lzBDThBK1snpaDVNt9Vj4F2nBdRD3PxLNAPb+7Ne3yP1OLgSBaC0XVHhU6jzddKNAz5HszQG4Hg4QYz9DlbAmQRQ+154bQuPwRZ3J9hIy\\\\\\\/tOf8p2xrHCXUPt5\\\\\\\/YlOT0nJ8Q1dH\\\\\\\/+MsEU4rWZDxmTajdHLqCc\\\\\\\/P0h0YsxhG1gsUyK8taC05LINk6Td6qarWRAohso\\\\\\\/HIoFhyNHAk2mQ385i1oHiG48HN3EVU2FKrR2Sj2W6X\\\\\\\/bBUEEwvuNLvx18qquxIsNXXGfY91NWh8i4DFqzCusWNfcuaBC9zDDn1lNpW9RZ55oGJXUWVzSQEsYbdknIyi0Ruysql1BQRFJj4c14rEaFtXFlw8AXBX8J2SqO5x0IUYw+Q55Jkwpg2RpNgJWfb1mQkk0B5XPfD3facPlHh7e6Dv6HX2aEYH1O9tL9Geau8FghHimZcuYnsd9cI\\\\\\\/IQoMVnjGKXukIJttp9EEkFc02mP4iDoaa9wi5co7429TdFjH9dFOtblQI31ryzUQvC8EYACP2Ok1xPISdWC+Dg4MTax4kqDugN69IxgJ5hsvKAUblSYqiblahrQPfnI5N8Sy4NRJ3DM9yk3NlvuQrRKii29IJ\\\\\\\/X2JZNxaH0eGsLnUmrjNt1WPcpOj5zYtO4XtPkzn0P0niehyciLu3i8mHMiGzNKWNZ+MDh\\\\\\\/3\\\\\\\/JKyCFoVMLTyqucWH+Gi5kvv412M8dYLx\\\\\\\/i5E3s69bCafEE6g2or1g6lwrjskTA+RYmGZ\\\\\\\/72NA\\\\\\\/c\\\\\\\/dpGyzMhTZr2z6q7PrHzhN\\\\\\\/P9U0yqy8BeC85qH32OsizfuLceFXoeIr2KyBh2f0LwIII1J6ts9rbz+fO7jON+K4MjD1CuoW5Dm6iVjyvqlQysEZf0AqFv33thsVXrJJsPFbnY\\\\\\\/8nvSkvKHjuMlzj3Jt25UKjq4+PXR7gMf8XfoKRYL0QFyih6yDrVUDUB\\\\\\\/p0Lee2JwqVoZMgP7OF+T9aFbWH6KjjGUS6arn2LCZ5UMjdaJYhZJwB5XmsQyLk+zi0vraP0MU0opNJmna\\\\\\\/vfz2tPIubMjJxXe32oNQruzJYi46JlMC2GNclMA5tAmuFvfsagqJ2226oxuq5fbPEJ5z36PZtscqVdNVsPOlIChA6AeNbhoLUl4OUuXXwBodRLrUiwNdGOyA2Zxxxu3Sc3kz+gwVTifYh6eDlF4xxZ0Ev52usWk0RljDGN+i2YMTog1Dcz2YKcxO1KaUCx6h8id\\\\\\\/8qd\\\\\\\/5q94I\\\\\\\/H0fpiIZHMuru3xMUcLXkRrNeU0LYaA4rp9dwMQc\\\\\\\/Zsa1p\\\\\\\/RpNx6YOTIY7IZnL6l\\\\\\\/KB9kJWSHM7IcK2NZx9yUn5dshGL2x39jlnRBSbjNFK\\\\\\\/CWjSM3Kx1LR9WBBDYDUl2TDi9a9HZDOmTe+3oi6zAY1F7Ef\\\\\\\/PQVxhlrjWTVwcRnPNlqJuNn9USYC101FkybgCDePbSb03KQmZB0VSnzkYOzA5r+uePlzcqAKruqs59tHWiKKd3ETlBAS\\\\\\\/3yUMSSVULTHwtWEP+utV8zUbhHnISWebwMTvuFetMNzpOcuIU6\\\\\\\/xGQcSAtnGS5pQ9FwnATDpjSic\\\\\\\/OnqPL+79pstPGuUD78Cr3+z+AsAYseCCoEf7ojN+BZb+6g021PYC22\\\\\\\/Uh0GSPHDfy2CEBp+X5Ka5+FTang87dM8SAhjpJSeOI7o3KSQlPhrujAs+yuPWp3rqBd3176emldav1e08zKQXaFIQ\\\\\\\/48cKuIM+6MQvUqkLiIDoDnErAHlct\\\\\\\/PXEgi3tVmBHJMYPGhGK1HeMLdIrY2kkslFHjzDIOPLRUMzmTkK4wb14ZN8rWq4nTEfRoC+gY5D9\\\\\\\/HYetE0hmjArtHyqxa4Cipr6zF0hrIOpMITC3RhHbhb3aDeQ1gKfRF9Sb3cjrd2ebbnD4MpkDw9JtaFYDYartmxko9dpDNNbURerl+A0+D+3DrR+imt\\\\\\\/r4H3ydqEUF4mJLFmgI4lWtQbApBLB+swGrXkBne23t3kD2uKOEy4PJrAtueAlUuPhouUbh7bGqMO43cZpppTPLOZbGw4CwOc2LQCFndzmL7gvUF0Y+FjWHsJXIgnK\\\\\\\/cdb1GhbrlUCAvDM4F0gCLWzK8kbkqdhkFFZUKoRayYNBzj266Fwe4i27q+lxBzCLV41GRYqFf7wIkvI9+XPlYA7wxNWB32CqyQ6Jhwe+u2dfBo4NfuCjV+7h9Cqr8G9Y4eGy6l3VAFMaho25hgH53Zi0TSvpkC8syZfrdS5GPqb3mxOkwZ9SWRRO4+gls9bN8o7TXNwEoBym9+hhJRoPyYBVsUbWriU8PEvHKlWA02bfj0Y2KaT9FrqxS\\\\\\\/WkvHLKO8DmfZz7D4KwN4erzvuu3nDOHE8ByarEi1tQ\\\\\\\/WqhlPc603lbItWh8VpMT7152tie+skQg2cWLlqO+d7bzqZPMKfcCsa+dMwNAAPRrs8PzIWhHgqVNkIg0kKYq0Ig7Sy2D4G\\\\\\\/oXJUZAYOIdeKAoEQeF2vJgGeKW2WtGWHAjxWCJkveGUv8X3899aujsy\\\\\\\/x1HCG4UKy20N8dwOqeFZSwYB+DXkgZ\\\\\\\/3bedPZ8S0bMQ71cAFL1OtHsHbWy4e2IZCn\\\\\\\/TjpCEPS3zylJnRhTqEEGl7h6kIcWkOcLN9D6L4loMrUgKdw\\\\\\\/6qyf\\\\\\\/E8ubhHOQKd8oQbYdL9yAAFDFASJ3F+YXopP7YG6uzOntljLmyfAsaNrFcpd6VHp+o2A\\\\\\\/afXmXTQ6UFsWShljV9658D4U7jvBAaJCXRPc5Mf\\\\\\\/ytH4m25QzUtdKzO\\\\\\\/7K23QWGQ0P6QL3CUFaar1f8sqgyaesyg8h2Jk1jX\\\\\\\/16TgrlZXk7MJUfXxWhUKj5+\\\\\\\/J53A8PAwSwq8tPvtKazTgXFq06c9jMD9rmW+U\\\\\\\/G+wcetq0xpwdziwJqh8iuOlRpTsICkwJDZcE2NskckuYNPSFALVu\\\\\\\/zFbM\\\\\\\/BKvBFbEJoO0nH039ZRugdMP63mVWoSwGYFld6NCaF9QXy\\\\\\\/Q9hCbBPEVgAunsd8h1NQDdONBnjiX8w6IJeWIAFv7Cntj8YOB51ZCXa8F3EZaXJboNCfxGnxGyBNzEi4g6xvf\\\\\\\/eoxclLbvNhtt82+ng09C2jkQ\\\\\\\/Hf\\\\\\\/j2OrCSBiA4DUA3Lukh+Bq642N8+H1QPg9X9nc1qr14hDdR4gjIAkYzNtEK+V8zwY3joM6nwbrDLF8IK8keWvx9ldOcwBb0V4DdDdQBlE4hxd1lhPJmXHra3hmkrahwP4yqnTCmZ+8RFVXC4zMH7JUl\\\\\\\/kpHnn16eFQHA36C6sf0Vv3joCISn8VFJsXQxtj\\\\\\\/WdgD60vIOQhIS2yckiyfQjxlN1xdQkKC\\\\\\\/Sg6MbFCQO1SbnWTMCf7HEscEVtHhd3d59QVi\\\\\\\/RYZhGkAk6WZqQEE7UM6NnbvtlMAPsuLG3hGHdwDb2PNs0\\\\\\\/smfBoOxZoMRqZMVaDcOzET8HEVBmaKJLhU3dTMMLEXtOyAFk\\\\\\\/Opf\\\\\\\/eiuK4+HWM\\\\\\\/xqcQ17qp8lrYOJseNdRsRzcw2AKrUgFp8bfsWhZMvqa5LCidvvZCJIDC2Ty29M2l20f\\\\\\\/sA7sLEuze6oy8A57LSjFxw5xwgCavFXKHDbdu0BuKQp30HVY2OuZt9k7KlACj86Wzd+X8s9YYG3papOoYG41wl5X5NppCD6d2BVKNlf8om4acAx07Hl6fbVJ5mPIHvc35RNVjinsiSdfkn3cfIzZ9GMhHe6TKgtq6F0zSqUCnmpZlVz5xLErcFn7okZCn136\\\\\\\/zNk8NPiSNZIGXBFPRF13dwzBibnuuX5gkJJyXzX0A64bByBWTQiTPXhvP+gIHtqX0quMZh+4HZ3gyRr3zyX16boYV7I7CzrtrPVzeyS844jsi3JqdtdKh6Pebpp5hR\\\\\\\/sZvMLiUIeZ3nJbJ63TT+rKgi62tuNZCgTfl1G58oRxFt+iHzARBrGhAl0SEFixxYoNBKngteyS7ML67yp6CUlFJsNG+B17cBjR1UabWVbu4XcGDEKHYyzqzB6ctz6jazbnH3DNU8R5DEKam6I3T\\\\\\\/NOjMt20QlfHXFAHHBnL8IUia94x6zQmLH+SkmciwyRLLannA5IfcI\\\\\\\/gJ4AJTvYR2WBxN2VcZcYfHOjTtmOeW5Xy5PrRonKkRQ\\\\\\\/mleGmWfGNiDC1wQc6XZpO\\\\\\\/SDj0C0QoCtJerC4\\\\\\\/R8KIZtXL+NtBcn4SqKTh5ohpF3XpAzGKwIY8jz2EjlwGaKPiwyXGMdXroXkY8mkxo\\\\\\\/ug+mzFnjUZXO15LukkV2eHuTuvKLpqWR3UaftYakuepLzaeHy2B4jnbe+6BEFAcikck4DgyXgBFiki5U\\\\\\\/iWGV+MHMy4YjTAg4N9Y98OCSV21VMkKgxcAFPqq89O6DaNWu8TMK8Y8\\\\\\\/TQ6HK3ueMgiqetbALXNON87U70l2BOImIyQ\\\\\\\/a+0piYMzYJEGXG7QllBTMoY7cTNrC\\\\\\\/FjpTSbKrg1Z+8w+pVZUEFu2YjHqcCIwpf9Fd+vCR68\\\\\\\/EtbLzQW1lyBu+Aawo91\\\\\\\/G4HER7zJAZm44GkRIDJtgI5faXtYG4E6ZwkrDr5+H2JXT2vdHwTzupHrqKsnBpUzNpmvqYH432Wog6jTLN+SrEquv6JWqO58+NXDeGrdQbhDinbj2tYviaMBMTSZRpJcdE+8mEquGimeW+i4+G8LvoeiA\\\\\\\/PTdKDNKZ6qtMrJpu7qco3BP6ahm5DHlivUTYqoMkUvd7ZJtUg5mvPmHrryvNCBZnUTKqWzEFC7+XEkFPRmhDLAm0pPplGceG8TgBtGckp9t4bBc2WQ2EfwYlehQyR1Gry7d6o6ZkrwPqOY8TU1ZQ+++fsm6AO02nap+sCP0k5WyRyyJjO2yXeF2nADyT8aB2QCFnXnpUYcrqtKpCqzvh43vhtTKNWDHPxjAq35pUDjW5AvkW41Vrg86Di7FvJXIdVrZZa9h8g9l8DK6CMcQA98rkNIHRQwvKlKzLlAoP1Zp5KXKJF2G8WEobTK3GX0QPICf+rnRaj49DdLRnB2HnSEHAEG7X8otDzPPaUGkqM32QTwzuN+ZpABkbfB0buDPMDitFCk\\\\\\\/cSnQkW5ELwmFYhkobZ1P+oOXtLAWs+P4pnk+vwLPzK4qa1muBNCLy68ikZntBchJ7fWExXiDEFopGvpvAeHSwxy+iaGrg21zofsSLD7nV2WtDxwQJ7OFrP8suaeFOHgLRhq0LfeQOfXev\\\\\\\/avD5ALkVRkBOMksNy+h22MQKjthWvv+euCzpxtI1XmBYamjoRKMdSLiS4UVfeZVTPtIvWnZOy+mcY+mQE78Hy4DZebOdlwR7a18Epk+ZGM0efhV8T5X+aSSqUsfKVFVUundahncmoizni2A3CA8zOtSGjas+c+SXqbdNEQ6ESyELrjfXL0sRTms\\\\\\\/7+Ct5doSpFEP5mnYP8UMrKHe6ZkkQFah4QbVXQOO7q7CT5wO27QuoqKshxA47\\\\\\\/H9xkYeJtEuzhlAXosuB3QI\\\\\\\/PV8qW90TNnbyK0S0MbCrDWZjIwrAnYKMexZDoTgddaBJWcjwKgTPAMVlblvQ4HC8SoxOQmlom0V3uJUm6AWE\\\\\\\/4l\\\\\\\/aJ1qjmykuPWbKbprIHGA\\\\\\\/QyllY52Q030GntA2zwDRY+Rh2S8E29NvVjTqVIk\\\\\\\/X9CE58wOvHR3Ofdan+YaTUVZq4bEbshonQail+YolylzjP7hPkZ0Osl9mkBd3kl8z2VjJw7jBka21M3AKhPyfeHpcxNMSLSc6oRl1t0XcZ2SWpx6yaPUBHwjs91GmsVFmeRX7JUoJ9X+kqawGwfeSgYnhTZEyo7Xarsu\\\\\\\/bDma2vN65LagEWxjruj3vLRrrOb2FjPm+KK3zJ95X4uhQA1x83VWnoTzLt8QHPAGGCwZ+yMuAeJJx3qcXt4GJBCJX9vwRsJftxFalwRylStJxETTSLabyIhVmNtchmY+eJOmq0uIBCKo2CjRZj3TY0FdIIJpA7K6CmP1BukT0Emfi2LUbqdg+SSjRGk\\\\\\\/TFJYvN1d68\\\\\\\/+X9+HbGEaUOhF+KHJ2a0S9cSvKlzRcbVxQrJFp\\\\\\\/HKp7Sb1fi4d4+GEyExsemdcTMATrVaqW6AtSVnX61nuNlDGEwC9WckW47jL5aVSRZRJAJCqmaFNU\\\\\\\/rHMH0IgFA3jt1h7IunCWUao\\\\\\\/ZpjVPtpRC8FceaRwVNt\\\\\\\/wweyh1AeJydY\\\\\\\/Gm7jTMUjN7J9tNy4BqQ3L\\\\\\\/gg+hyQIOW2nRNxNXEwC2ocVQkzBFCL1KoUd3t3sAM5Fy+ep0IaX5F7DmLlRrSuCtlKmNDeSMu3zL9jwQ6Zgocfq\\\\\\\/\\\\\\\/GAgR4dNEsaH5VtWEqS25IDunyjDOl5p2MtjGu24sD94Ty78LWdP2d4VHRXY6dpZOsjxe1TqkPYO\\\\\\\/zm98adW5KjtVn0hK0pG+uILCnH974juTu2wncDOvvN+L\\\\\\\/DMZitTZ\\\\\\\/wKvz57eqoLZs6y+v50+sOEQTzvoDm6jy2fYTTeY9romvyrVXnLEJJXNCVUGCburILR4hWiZKU8zamhWSY42hg8UukDa0t2X41iP87FI9rKX8nUkBJMb2Gu74SGw18+9Hg3IGWyB7fktaFBOe7rwgqYfidP7w69KVSDiqUCp4A8khKFQQTaS+clCRUtqwF\\\\\\\/DzQ5YEmtLMe4gB+J+Ocz\\\\\\\/5AS0HBeV+JEjC1BPPnCYro4o12K9MEzUw36b8sVMFjRsvJHbldjFgXpy5\\\\\\\/Ry8YSlb4wf4ZWhG2PouqlaHkri6ipuj9TOBn27FhSGRL0Q76PObcZC0kXuFIw5Raev1OjtThrXvuW5bwD+XqmbIc9f8p8kH1LKQNc8QeVq1O3jlHqbRYASxqJ5I2QI0LztzL\\\\\\\/YO85gTkclsyhFWLkJok8mF6+pxu7z\\\\\\\/RlUgugdiYVzJZojhx90xGaIiEzvIS8s393nt\\\\\\\/5VyeW9veJNKyWOEJxLw\\\\\\\/p3bkofbBV8oR+1E3pQgFLr5NiCV8B1sUzU3fhRs9eHmRIMeDa3CdXI3tQp7jOHiE28UeBHd3oV0e\\\\\\\/pMJWdZVPYYYi\\\\\\\/tDxeVLTKbi35QhE7cEJalwdX8z1G\\\\\\\/Bss3R1ijfHrg+o9rYWTU63W8FnKfSLd9IjxaflVa7Zs1L2MzCYCaagO+zozgicEW33bvIBW3Zoy64VQm5gPJiuQw+nixlf5CYiWYzV1qEclh0hUSa7w6OE4lNe7m7104L8\\\\\\\/BRg9itnq+w5kM4073skSBZi2IYI+JXqapA5Y73AWFc+qwKe6uB3bCklu0BcIV3mauvc0ieQooeAvcdBPaT8tx5bVskOf7ffKanlJq7v9\\\\\\\/rFvXi5dRDWiv\\\\\\\/988gOnR+qgVxuMZMxkmuL2rS5wsN4\\\\\\\/xRqqj7o\\\\\\\/vqWRURXqFzYjTVeR8JBBAeG32fLSRidDDiD32MT4NA4ccpcDi0uU29UnjzXAUi19dRofkD+EVZaBvkdSIN\\\\\\\/yvxgNqqneJ6ylT77CqCaDWiOqJOblbr4CLL6BLLRswHVXze7k\\\\\\\/85tLKGMikgTXBZIfp2X\\\\\\\/nPOBDnQ7IxlYk+GKmHGWinbLygvBdJk3LGuikqj086nXu40dqV+rqway1YRqFHbDFOxwTlWx\\\\\\\/0zRUP63TeuDIvgrBgWA+GJAy75A3xS+f9ox0wngpcVGQjZwFmX8Rs7S3VFPitEgBP8rb21aVfKW4qovgBpawt53t7HpXqKELB4HmdU5D3oSKbHevUhwIPet2v2HEb\\\\\\\/IRUQaD8oB5N352XcRqmEWZHw0Er0jaDbpWbRLXUXV1AVX1oYfw\\\\\\\/9T+M6izTHj3rUpoHXmdWDQCt\\\\\\\/UZPaX6mKk9dJDffn0XJVixMhQpxBhOXGf\\\\\\\/nPEo+D2WJB\\\\\\\/TbkDCwxNuXHjSsHTyDa6tBFd5Sx1YDf421MeAVFHohoHzPiKVWsPm8kpQuenj4dftbEFWLNKzcRzUJbN1gDX4+ZYGwznnnWR46aBiXNlgYyhZNlolsquoNYt5FJ7AfuHCUDaHZS+5LAqen7XEStbVtM8yerqOdHmEPCcpMcfqX4TYexrkrOOaw+78fNJ2kpZTEzKZFm5PjQHHJyWG2Db6Dx9hZHlmbqyZtm5LLvdI6TFPi6wbP4YSnNUTNkSXG9dJ5bNh1xNZva718BnuFcEgduoHtTh6sTOhdLyUDUPAe+sjsn+AMg3kUIANNfWjuRo3s39vZ+ELVMGP0rLf9DewBhGib8RL3\\\\\\\/WSGAgiq8Xguo08K34jPlPPcl6S43I\\\\\\\/Ds5tJivUxW9V6Uw\\\\\\\/rYT5nv7cV366lcCZOE2urH11Rhojx9AjLRSR+AqsbNrZzxnWMfVL1D2nWCKs4\\\\\\\/dMOumcsQubvfpKhzqJHwToO+i2YjhnY57eqcr8w8d\\\\\\\/kipZ3pw9FpdgeWNfDncCvN\\\\\\\/2df+DBsQhsEg5cVyQTo7NeY7zOZdQttBH4sPmsSa8d5gYMIyc0IQ6bFqS84DIoAiHaAB0z5MXYpTmGu45FIZSuj4e8+PRCjLw7N+1gPwtOnc6HtrinctyRC2xBL+icggJ+520nEfgOZapYsyU6fWCpqj0LgUjFFAKSjRnNc55UTal3z8xNaHR727FGxbeYcvbM4qjbvmJFck9CcDyRqrT7RYTGs5g22YeXWDs4qxFJhLLqJ5LYa5UxUNLUU8U2eyvnVEBzEv48rkusBF8E19wBkqzePsTjjgE\\\\\\\/MWBL7PjPkzw4kOti9E24XSfRx0ZXJk9AJWJzEd4AnXic+HEtVO\\\\\\\/r3bjxM3U+jDQoc589EfL5fkkbS\\\\\\\/+WSBhRhHgfoO67EWm4mZ+WkWLCzn9uN2+8S6BIhHLKbM7AECjDla\\\\\\\/9RW4ly\\\\\\\/EiIJ7YH382tP5Pv7vqMUS+tvRSW7idyjjWy9HN6KJAZI6QuR6zKBHBbXmxJvW9WfWb3xJF+OZxCwE1KPa9ZsfP9N\\\\\\\/7xuQoYtBAXavYvp\\\\\\\/MvFP17mrVA26viY8RA0VDsdggDMQX6G7n+cqPzVesGr9rqNPOn1w6qhqmzbl4JEA2+4kiKIMcv6FvuXGumar9mW6hLthqhywqcvRp429k4F515WqYoN5vThkiYQCWEAmYeehxEuCZrQqUZpiVAFzCzeB+3t5OeFQaCCWsc9VSZK\\\\\\\/nQBwM4uPEhBxJw91QlZrV7AN9p\\\\\\\/4QoozgtlG5c6Q4ooBVQv+u0sW+05sI2WQOjqTTkJaLSJSgr1EjBCnmhnN96IsKSXTQ1llB5svrjSd\\\\\\\/tDKUZlGcQV6eGqgTw4ynqvdTzBLTff0krrVsLzyxJY2uBRiRM\\\\\\\/Qmw6tYiX+H5m9jxe\\\\\\\/hsbKbt\\\\\\\/+31sylIwQWyp7a7mpiXtj16GpZh44ywanuDXI5PpxChULSCm6sHm1fPA1nlSeuVrLId7WlON4KXyBJ9nIgzjybBLTTJYavix7yO13YcDFBG+kYLrhBMm61yw0yCdpPpliQnCMkqJgkr0bVAg3fOEpI+jAkDfxBXYtJHVS6y0PvoR9HvmyHsgbglRVJUbCA+Oi\\\\\\\/D8hVswn+mNuWaDIS16g6wahAncKrNWqf7YSkA6oC5XWVjbSkXTkYE3qBcFEcDTn9wjh5WUOu5BxGuHNLqJSTtTvMRvAFFgeZx7SlaBN8ZbCK+fe6p3BFPWijZYLtBcKvTILCXFFhNEa1LBYxWv+Xc2zypSuV\\\\\\\/SKDi6SmrBeGVG4p\\\\\\\/rigzMEXf\\\\\\\/CljP5UTiv54Lw2E7OqPVVG1E0QwYqr5lhj8ytIEvpS0PAcCIzMbTp0VosCAheNNrtp1+BoyxLcFgrJUuS1eMwGq7zuu46IP7umvgb6eG6\\\\\\\/FsJbviyMX5ZHG0CqpGGCHgYbLcea298PSVn64YxAjtnNEIuDO8f6tN6xFmJKwthgD8InQJC1OCq0JiEIzygMVAURoJbI1gzamSGj6I9euJdqxob\\\\\\\/ImGRi31qJRSSHNJN4k+lHU\\\\\\\/QFLcZ\\\\\\\/h982oevGfp7uYb9E4ShkrcAlnB40TNc5BR6Ut5Sm+jebvXIdMb5E7tr8ekXd8OXt6cK5ehA1\\\\\\\/aMxGmAY2JNjcWBl4tA\\\\\\\/gbNQHqusUt\\\\\\\/vPfUHhnHXG7ubNPbqU1nW5VU5iHH2akX7a8pacrSyDPouc\\\\\\\/qOEvYAGtvki7uKhZ\\\\\\\/bqPCXuN7TVlzNgwkqffK7TzwaUdmrv6WvbUogSeMKl1sGgW1MC7MHthGQEOo9P19HPlCOZlAbjPDnkJAjJTJL8VLnj9UzjjJzoFEt+Kp5hl9oNspjoK1nU\\\\\\\/FcYPJkPMV8aV69c\\\\\\\/unBs1jHArBCt4jFb1AagkMnhwoVOlwwJtwDVo672xcXYZYbvpPskaRGJmtEagEblQyDGZHzyR1rdQd4kjPBlw6lEHahDBCboAeAjSFSnI3b3KQEBdPRKOv9WRc356KKnsIhitt2l0qH2OclXwztR2r3hZNdwpySHU+t42z6DAd6lDzmHCIup6D\\\\\\\/m8koK+cGA06PfL4J45u\\\\\\\/oMmIFoveKmIWeCQ4gVmPmu1P6bTagOI7mdJObpmSoW3FArNjVsFkZzEQTri8okjN0bAJp\\\\\\\/RiCBEr0ZJ8CohW\\\\\\\/PAbucxoErri3aR3L2WpzQ3PpTr2MijYJphRNDmq67p21imuS0dKDnJYUgyH6Ft\\\\\\\/gu\\\\\\\/LrYOX0Mrj04ejvl1DvNXFb9a6OQWCYP3iSdUuJngZwGFJAnAQKsaXRTjRgRN7Nr9oDeKYH7wHUStsDDEl4dns4wUosN6JM+LWg5yzdG6wdQ4hizymHVwPrZpXWhwvRSOe6NpRRkxiPZA5Lda35Ff1rJdw1TYEYvlMA7i7yBcdTEwwvg7ujEbzmicspZUVJek74hnvwh45\\\\\\\/oz6ytFIbT4aetUSeLqztIf6ANy2FtJXQ3KC0Bta3nYNmIUn9xXQebGux4rwdenIU5ea43fFkKpF+zbXLL4+ev43im+e082YQxCdXaWPbnVVdLwtD2KCeN+Ww0RQtIAtATb1pueKqY1UN68DwTQmGECr6Tq5pc+zyYZdbiCmL2qxjljclmYgxNEt1ediD3tKCfe0xeYGqcRUHA3hC8LSMrX6\\\\\\\/N0zkgAhW7+BtuF1bJf1400T63fs59zjZPGrj\\\\\\\/y2hcgiCdXrsZicmbuv5\\\\\\\/JFt8woWAGPIv\\\\\\\/R4iVxXf92lSFzzi0q6CQ8fZLFRIpCiTPGFRXSrSO+u4DO\\\\\\\/xzqcA\\\\\\\/euVWk6Mj48yZvyHI1G5ZyqAi6JsYMc7fh30z0FClOWCsFlg2AIjgXrNPtX8jCoeaWaOzZRiAbjuhT142LdXo+CsFrZufNF7mn+DsV1IFokV8JFtMG0LGXHMceqyMsuF+qB7\\\\\\\/ccgRjNewbpRYEfMVeTNZv+o1PkY97Bld6+eOHcr7Vpw6TGWxW+ePDWQRjYnqLlnKhhkjJ28fuv9gyhUV8iPycGmyT3zwVoZQyt75L2\\\\\\\/ams6AK+Zk\\\\\\\/q7gH7O3I3zYnSNL25x0tVjxosz6\\\\\\\/zh35dvzQAyZugVtVJfvlU00qHaruwOOUG3TaNhcg90rdDo\\\\\\\/ppVBG1SElKFJghy5Wqp8sWNO9Rae27BT3oaSUNvLuuSgfM7DO3CVvcdd0\\\\\\\/zOReeAVzj60UHXDR\\\\\\\/Hv8X4NhDdb+eCNC4+rAQBToTWsbJy9C\\\\\\\/FPORwkd+EURu0DTORg2BAabeUlk3WPogPoBjY+ZktjNb8VQVtzGy7SiVBErbRuH29HU+UaFD2Gly+rrn9oV0qH3kB+KeDbjGQRx49yW0XaRRB4xa8GIrDieZaf\\\\\\\/yVU7bq0V1JMCkE1WCDoLPkr9kRsv7YSuBjnnuVPEtpgYM\\\\\\\/PgiE44cnF4wtg\\\\\\\/p3tvHIs\\\\\\\/XWoH2\\\\\\\/MJn7JYs6RZCj1YMIJI4BdLax8cuV3zUcfKgs399bxK7y2ZEwfk6uXl88dgssLjqzXr9pzypMY5YykyQf+MQKFoS\\\\\\\/hdyUHNLK5DYCKpS9TWOoiUE8vNS6mdi1KAW9GtAvPvwJOccr+4g4FbCC7hHZf0k1ot0rpTpK7IwBfdXhcovMD0PpjSljpOdUAgFu4DSMlp8XvDrLB07RO\\\\\\\/5AXco7RlSGq9x3m0pveAG59HtQZR5D0SPNe0cTHUmOSxMlco42iCXG2bK2bqm\\\\\\\/gmwtKFFI8AHNOBYqEeagR62L02p33TIQ\\\\\\\/2CJlB0rMnbQdfomDXnEPudbuM4nVKr9agZ6TsJU3UALm\\\\\\\/ZlZQKPNJ\\\\\\\/EYOwLuhi2uzdiygZdjtdDINB4HH88qneiZPIkwdiQ\\\\\\\/5pjm2k6w41rFxXBP4Nz+8rH\\\\\\\/J1jdSrGukEnj+L1KJEb5hIqkx\\\\\\\/IylpITe2p4DRofjGgdOqh5KogqqMp9P11tgksoJueeMNNucchkncAxUNBTdxfO2TFSUPARHD1LFcKWBXgjMiBWnWO2xauU13NoGQeV8I\\\\\\\/JlRRknx41ocp9uCOCpYyf7xwNphBUuWNYSpoon2AcVQPpPLvINHyMk8VMqZXAypwKdQEznlJmYUrV1lV7sHfNVpcYRW009ce\\\\\\\/ONUHy2c26gVbFLQsgCDTnkAN3aCoN7si3ZVI\\\\\\\/lxmeBsEfBW6ZrJ7jA400aTbpUGLJUkF+SXe0WcSENlO6E+TM1+78ckrZJkV6h6x5zAL7Xs0qMF23iAYNHN2H66AE8vlA3CQihqqIxFAsQRxyWlD26dfPBZNHRubRFnqaGFZLd5T68yOfpChRAEVbXpAMOOkZ\\\\\\\/h4K8GlP87YrjyxRmbjI20NRQozve2cn\\\\\\\/t3VlTiCn1hP1cnJmg4lfBiz3Vvq6KufUgsBG8\\\\\\\/zQTh9uaPFRgdlIUMzYZgYeqqytCBbtTNc9XIfzC1iRDrnXzNMAI\\\\\\\/5p8AxU5V47w+KKxdilZltdF1B04C7VtH7XUPHhXQrg\\\\\\\/zvcge0939YT\\\\\\\/ObrYvB+ubOSuv1Am2mfdd4J8V2NBsIzdqSWrZcUukccGSkMEU+7Z0TJm4akFSK6KrlfCDtJlcYvfGTynfpIx2OIUfP0BufIyBe82RYEA\\\\\\\/0xkLzqLsMakFttUY7nzQ2NDqtUkBW+24nQ6lZ5P+oz3vGTTALoegbhR4gY0QDWJSNXr8q\\\\\\\/NDklZQclfmfCqt18tOjLlZP87G+UIHKIt1wNTMbv\\\\\\\/Z\\\\\\\/8etyjx0fsfXqWrdUGOSozR23tfPpL6O+YF3ChmrP0knktG2TQ9ymm0vSCZFwgeq0BVLSq1Wo34bDbCOk2WCWXvFDr5vKu2Myj0PG0XGvzTdgZHVVJlUE0+xdcYQiTtKaEzD4sFwER1N+\\\u0022,\\\u0022iv\\\u0022:\\\u0022olqVXPpdhBccUyFHBF3lxw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222fbda7bdbd5ed22c2b23bc88d4c3a9b567bfa48058ddf6ccc98bd566ee2d951b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UJvJ3UpBx47nhPoXQdqHhWCz7BizK6vAYc5EwWvlXcQ+88rE707XtvLl\\\\\\\/US7PGjJjvWUlHcgQwbMf3ZcMjdBo+apxcnEb8a7XMpxajh8zYEFLwOC9EQER\\\\\\\/ng02dmt8w5cOMW0MY5U7XD\\\\\\\/zcEHM\\\\\\\/p\\\\\\\/unkFk7JnuMdETpeDWoOQ9xeYdmfqgyTtE9aKtvDBLPDEUMDy3FN6JDQrIiumnWCwsLWs7TXDwHicXQ2F3i4VYe6TdKtWpi8h4Vlr5N+21QzfTX6juOtgFVY+72TVwvLveVR0aCaSaAHhuzr29afgV4+m7u4n55QfxujbQMaQMsD+nmMxQh\\\\\\\/G4Yk51GDeuzHCDRIA8eLeDyezhX0XkB8yxUEv5dfukZoJ+xfhmsReCbD1jnZGwYGKe9\\\\\\\/EesyFaeoU04K80z48oGBL6NFWbjWOlBronZC9rT1WdCwd3NGZuaTEWidGcXmzrnpv7pP7+EbSBLXLv7nHhDLo8\\\\\\\/lyE9ic6clADEl1VW9gRNJ8tD0M4q1eT\\\\\\\/65Vpg5RDNCunqeKAICm\\\\\\\/SmT8Sj5o09X\\\\\\\/+XeYpQpnd5sv1HsC+h+1a3HFzMwHKB9owmAXyXf6WyKuKZwZHEdabnM9dnhCeTUcfRpizf06+9mJiGVnnSVX91Ps4+SFnMQKoub+vC0hoEgx3mn3FNlokyXHLtOnW9PFaWxE3pNZh8O4dYRiSSMm5vcwxcJTCaJIH+U+oO7ARd3lZ33aYZSXUismSueQfwD5uNFjZIOVSv861idJAMhAgI0sqxAvC422eB3BwoWzrKQNSAp7KbZa+KmsfGIvtSzpwFykSs3SI1ilimKERfrpdzWicAMSnniC41PF10HovPJnkTovEKGN3\\\\\\\/gqBBbayoPe\\\\\\\/VrAW\\\\\\\/ZmsTx4ZpDJ8pEl\\\\\\\/925\\\\\\\/X4WyIrpZMVkNDrXMGE7sIpVlSHK57kSD6ujLwMhOANp5c\\\\\\\/JWxsU+FOVa0+LAkOEgqjKbCpgLcvD+k2CIUpbueXTfCDKAzE9nnD57MFAzEb\\\\\\\/ZDF1mxk5YlNTyOEE5FdgTGD48v5hb6kvcrwlXrH3AAQt1cgCPZrrwaiEgiChwovJMeC6rZj++TDz+hjPPmxzrBV+tRH1kkxjjlax3GGEcK28fVTdYO2ZGulJqmTr\\\\\\\/orrKorlPgIc9H96U3zXZN09+0syczopDTuX5ZrOQogMFZ6A8lgEgqp\\\\\\\/BYF+gu06rskVpqrIo2C+ceOWoM7+zAGXuuMHoBkKcx138k\\\\\\\/BwObuOdKoRf5ayLV5k1bjAwTYZbTX2ZC4A6uCuS0d7Ue5uPDl79C4kJ8TFY07TwUmUzgUbWfwtlbPfJt3zHmP1Ofl\\\\\\\/lM7xke9fyoCWJn\\\\\\\/dPI4vefb\\\\\\\/6l+liOevuMXY920AOVNwmPWWmUSuKcXXk20voArww3ikImZXmmfzuaHRzSl9tMKHyKtvaCuwLepY9M2NA5\\\\\\\/DmAZQ2CAM8fUpMmof1crvtWFoouFzf9MiS5wqhHSjv6aTXg9dzokS5CBkzflA47aBjzeLvBwR0SBo65yP0Di3lc6wP2TEq3IDpFEOffOcwU2kWbDerH2UIcBl1JKXVHtlxrhflMZOzptH9k1XZIWDmAJiTiOZPhICtqGr6+bltyWv3HfzIePhD4iiJWb3LggIM8wvEW6hhPL9s\\\\\\\/MT\\\\\\\/mYYIaHf\\\\\\\/C+5VOCO5466q\\\\\\\/YxGLwCxde7vY3QSqQCRgrXXhJCpjrRyrs\\\\\\\/XGQxaaGnkgfVXb4FgGufUUjIbA9cwWrvjqs6UAVjos1aswaQ+\\\\\\\/lypAQ0lOtz2Sa1CqXMhXnOsLrQubFX6FPqlLxFdaZyeEoKx02t7x4eQMC6AZDt9ZBp8IUZN792+12D0\\\\\\\/I6npKfZnRJ\\\\\\\/wwX6b17Du8SS1mlhb8fXMySPkOink2fIiFmGeSYg25cWQLQfcmsWdNDX241ZKYGBBguMLm213e1apt7z0XuKHv+SUt+PeNiou270cfhkf4oQ75JulkUrRf36WHY72OAk8cSZvcgZ\\\\\\\/5IK3TJFo8ka4xLxz+179s93l1RZ99anMdTA6ToDp14orq2ZNWWp8Qb8waLcq0=\\\u0022,\\\u0022iv\\\u0022:\\\u0022nkdb9j2\\\\\\\/+nkx2tAsI\\\\\\\/0+4A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f3c0f2068bc861213e65b5407d8ae65185039a839a2cb29660c47f9e9764a87f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487828 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/113" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CPCxYdW67vPgxgnlS7V_7M-3Y45KRrH-rjQsn5dXVbA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/113" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OEiOKkEvhtpMwquJR3TOhwkXw7xFMV_wTcLWYHtHIzk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/105" title="Analysis of Copy Number Variants on Chromosome 21 in Down Syndrome-Associated Congenital Heart Defects" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/123" title="Modeling the Evolution of Female Meiotic Drive in Maize" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487828" id="node487828" data-pisa="ggg;8/1/113" data-pisa-master="ggg;g3.117.1117" data-apath="/ggg/8/1/113.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Which Individuals To Choose To Update the Reference Population? Minimizing the Loss of Genetic Diversity in Animal Genomic Selection Programs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8609-5869" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sonia E. Eynard</span>, <span class="highwire-citation-author" data-delta="1">Pascal Croiseau</span>, <span class="highwire-citation-author" data-delta="2">Denis Laloë</span>, <span class="highwire-citation-author" data-delta="3">Sebastien Fritz</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3213-704X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mario P. L. Calus</span> and <span class="highwire-citation-author" data-delta="5">Gwendal Restoux</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">113-121;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.1117 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487828" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sonia E. Eynard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics Centre, Wageningen University & Research, 6700 AH Wageningen, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre for Genetic Resources, Wageningen University & Research, the Netherlands, 6700 AA Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sonia%2BE.%2BEynard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eynard%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASonia%2BE.%2BEynard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8609-5869" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sonia E. Eynard</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sonia.eynard@wur.nl" class="" data-icon-position="" data-hide-link-title="0">sonia.eynard@wur.nl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pascal Croiseau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pascal%2BCroiseau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Croiseau%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APascal%2BCroiseau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Denis Laloë </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Denis%2BLalo%C3%AB%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lalo%C3%AB%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADenis%2BLalo%25C3%25AB%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sebastien Fritz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Allice, 75595 Paris Cedex 12, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastien%2BFritz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fritz%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastien%2BFritz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Mario P. L. Calus </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics Centre, Wageningen University & Research, 6700 AH Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mario%2BP.%2BL.%2BCalus%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Calus%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMario%2BP.%2BL.%2BCalus%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3213-704X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mario P. L. Calus</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gwendal Restoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gwendal%2BRestoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Restoux%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGwendal%2BRestoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/1/113" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487828" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487828/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/113.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487828" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487828/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/113.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487828" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487828/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/113.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487828" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487828/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/1/113.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487828" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487828/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomic selection (GS) is commonly used in livestock and increasingly in plant breeding. Relying on phenotypes and genotypes of a reference population, GS allows performance prediction for young individuals having only genotypes. This is expected to achieve fast high genetic gain but with a potential loss of genetic diversity. Existing methods to conserve genetic diversity depend mostly on the choice of the breeding individuals. In this study, we propose a modification of the reference population composition to mitigate diversity loss. Since the high cost of phenotyping is the limiting factor for GS, our findings are of major economic interest. This study aims to answer the following questions: how would decisions on the reference population affect the breeding population, and how to best select individuals to update the reference population and balance maximizing genetic gain and minimizing loss of genetic diversity? We investigated three updating strategies for the reference population: random, truncation, and optimal contribution (OC) strategies. OC maximizes genetic merit for a fixed loss of genetic diversity. A French Montbéliarde dairy cattle population with 50K SNP chip genotypes and simulations over 10 generations were used to compare these different strategies using milk production as the trait of interest. Candidates were selected to update the reference population. Prediction bias and both genetic merit and diversity were measured. Changes in the reference population composition slightly affected the breeding population. Optimal contribution strategy appeared to be an acceptable compromise to maintain both genetic gain and diversity in the reference and the breeding populations.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/genetic-diversity" class="hw-term hw-article-keyword hw-article-keyword-genetic-diversity" rel="nofollow">genetic diversity</a></li><li class="kwd"><a href="/keyword/reference-population" class="hw-term hw-article-keyword hw-article-keyword-reference-population" rel="nofollow">reference population</a></li><li class="kwd"><a href="/keyword/optimal-contribution" class="hw-term hw-article-keyword hw-article-keyword-optimal-contribution" rel="nofollow">optimal contribution</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared data resources</a></li></ul><p id="p-3">The development of genomic selection (GS), as described by <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Meuwissen <em>et al.</em> (2001)</a>, is the most important recent innovation in animal breeding. In livestock breeding, GS comprises the estimation of genomic estimated breeding values (GEBVs) and the actual selection of individuals with only genotypes available, <em>e.g.</em>, young individuals that are candidates for selection, based on these GEBVs (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1117/-/DC1/FigureS1.pdf">Figure S1</a>). A reference population, composed of individuals with known phenotypes and genotypes based on many markers across the genome, is used to set up prediction equations and infer GEBVs of selection candidates. The main advantages of GS, compared to the traditional methods based on phenotype and pedigree, are that generation intervals are reduced since phenotypes of mature progenies are no longer needed to perform genetic evaluation. Second, selection can still be performed with the same accuracy as classical selection. Lastly, it allows selection for new traits that are difficult and costly to record (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Calus and Veerkamp 2011</a>). Despite the confirmed advantages, most of the knowledge on the long-term impact of GS is based on simulation studies [<em>e.g.</em>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Colleau <em>et al.</em> (2009)</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Jannink (2010)</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bastiaansen <em>et al.</em> (2012)</a>, and <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Clark <em>et al.</em> (2013)</a>] and many questions remain concerning its use. In particular about the design of the reference population: how many individuals are needed (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Pszczola <em>et al.</em> 2011</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Khatkar <em>et al.</em> 2012</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Pryce and Daetwyler 2012</a>), how often should marker effects be reestimated (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Calus 2010</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Heslot <em>et al.</em> 2013</a>), how closely related should individuals in the reference population be to the selection candidates (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Pszczola <em>et al.</em> 2012a</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Meuwissen <em>et al.</em> 2013</a>), and which individuals should be used to update the reference population (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Rincent <em>et al.</em> 2012</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Isidro <em>et al.</em> 2015</a>)?</p><p id="p-4">Many livestock breeds have high inbreeding rates and low genetic diversity as a result of intensive selection (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Leroy <em>et al.</em> 2011</a>). Limited genetic diversity restricts the potential long-term genetic gain of the populations (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Li <em>et al.</em> 2008</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Goddard 2009</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Jannink 2010</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Engelsma <em>et al.</em> 2012</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Liu 2013</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Henryon <em>et al.</em> 2014</a>) and reduces their ability to respond to new challenges (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Toro <em>et al.</em> 2009</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allendorf <em>et al.</em> 2010</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Stock and Reents 2013</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bruford <em>et al.</em> 2015</a>). To allow for long-term maintenance, individuals representing the overall population’s diversity need to be used for breeding (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Rincent <em>et al.</em> 2012</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Heslot <em>et al.</em> 2013</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Isidro <em>et al.</em> 2015</a>). Different strategies have been previously suggested: (1) limiting the number of offspring per male to avoid the sire “star system” (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Danchin-Burge <em>et al.</em> 2012</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Boichard <em>et al.</em> 2015</a>), (2) distinguishing individuals according to the marker variation they carry and giving extra weight to the low-frequency favorable markers (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Jannink 2010</a>), or (3) choose individuals to represent the highest overall population diversity (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Meuwissen 1997</a>; <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Rincent <em>et al.</em> 2012</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Heslot <em>et al.</em> 2013</a>). One of the available methods developed for such a goal is the optimal contribution (OC) strategy, as defined by <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Meuwissen (1997)</a>. The OC strategy can be used to simultaneously conserve genetic diversity and achieve genetic gain by minimizing the relationships between the individuals (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Engelsma <em>et al.</em> 2011</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Sonesson <em>et al.</em> 2012</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Clark <em>et al.</em> 2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">de Cara <em>et al.</em> 2013</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Eynard <em>et al.</em> 2016</a>). The effectiveness of these methods relies on the final choice of the breeding individuals. In the case of dairy cattle, such strategies to conserve overall population genetic diversity may be insufficiently used in the context of competitive economical markets promoting the use of elite reproducers. Methods implicitly driving selection toward both genetic gain and the maintenance of genetic diversity may be an alternative. With the design of the reference population there is the potential to modify the breeding population by changing the genetic evaluation.</p><p id="p-5">In this study, we addressed the following question: how does one choose individuals to update the reference population of a GS scheme in order to balance genetic gain and genetic diversity? We anticipate that changes in the composition of the reference population will be associated with changes in the breeding population due to adjustments of the prediction equations for GS. To test this hypothesis, we compared three different strategies (random, truncation, and OC) to select individuals for the update of the reference population. Using a real data set of French dairy cattle (Montbéliarde), we focused on the effect of updating strategies on the population of selected candidates. Using simulations, we inferred the long-term effect of these updating strategies on the breeding population. For both real and simulated data sets, updating strategies were evaluated in terms of genetic merit, genetic diversity, and performances of GS.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Real data set</h3><p id="p-6">A population of 14,052 individuals from the French Montbéliarde dairy cattle breed, 2459 males and 11,593 females, born between 1969 and 2011 was available for the analysis. The complete pedigree record contained 50,852 individuals born from the 1940s until 2011. All individuals had, at the very least, complete pedigree records for their parental generation with a maximum of seven complete generations. The generation equivalents [sum of the proportion of known ancestors in all available generations (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Maignel <em>et al.</em> 1996</a>)] ranged from two to nine. For all individuals 50K SNP genotypes were available. Males were genotyped using the BovineSNP50 v2 BeadChip (Illumina) and females were genotyped using the 10K SNP chip (Illumina) and subsequently imputed, by <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hoze <em>et al.</em> (2013)</a>, to the BovineSNP50 v2 BeadChip using the BEAGLE software (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Browning and Browning 2007</a>). The software DAGPHASE (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Druet and Georges 2010</a>) was used for phasing. Subsequent quality control steps were required for each SNP: (i) a minimum call rate &gt; 90%, (ii) nondeparture from Hardy–Weinberg equilibrium (p-values &lt; 10<sup>−4</sup>), and (iii) MAF &gt; 1%, to minimize potential genotyping errors. The final genotype data comprised 43,801 markers genotyped on the 29 autosomes. In this study, we focused on milk yield having heritability of 0.3, a genetic variance of 423,390 kg<sup>2</sup>, and a residual variance of 987,910 kg<sup>2</sup>. Milk yield was measured as the corrected milk yield for the females with, on average, 1.66 records per female. For the progeny-tested males, milk yield was measured as daughter yield deviation, reflecting the average milk yield of their daughters adjusted for fixed and nongenetic random effects and the additive genetic value of their dam (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Mrode and Swanson 2004</a>). Weights used for male records were defined as effective daughters’ contribution (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Fikse and Banos 2001</a>) and were on average 26.21. The data set was divided into three groups according to individuals’ birth years. The first group included 5969 individuals (2325 males and 3644 females) born between 1969 and 2007 and was used as the initial reference population for GS (<em>A</em><sub>1</sub>). The second group included 3791 individuals (134 males and 3657 females) born in 2008 and 2009, and those individuals were considered to be available to be added to the updated reference population (<em>A</em><sub>2</sub>). The third group included 4292 individuals (all females) born in 2010 and 2011, and was used for validation of the GS (<em>V</em>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1117/-/DC1/FigureS2.pdf">Figure S2</a>).</p></div><div id="sec-3" class="subsection"><h3>Simulation process</h3><p id="p-7">We simulated a population with characteristics similar to a domestic cattle population and a trait similar to milk yield. An ancestral population of 1000 males and 1000 females that had undergone selection based on estimated breeding values (EBVs) estimated from a best linear unbiased prediction (BLUP) method was used as the starting point of our simulations. Next, 10 more generations of selection and breeding were simulated. In every generation, the 150 males and 500 females from the previous generations with the highest GEBVs were selected to produce the next generation <em>n</em> + 1 (a selection rate of 0.6 for the males, of 1 for the females from the generation <em>n</em>, and of 0.5 for the females from the generation <em>n</em> − 1). Males could reproduce for one generation while females could produce offspring in multiple generations assuming that their GEBVs were high enough. We assumed that selection excluded them from the population after 2 yr. Each female produced one offspring per generation and the sex ratio in the offspring generation was 0.5 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The simulated design is simpler than what occurs in a real breeding scheme. Simulations were performed using QMSim (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Sargolzaei and Schenkel 2009</a>). Details of the simulation process are provided in Supplemental Material (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1117/-/DC1/FileS1.docx">File S1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/113/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulation design. This figure represents the scheme used for simulations. The highlighted boxes represent the population under consideration. The green arrows inform on the selection decision. G, genotype; GEBV, genomic estimated breeding value; P, phenotype; Pred. P, predicted phenotype." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1839432961" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulation design. This figure represents the scheme used for simulations. The highlighted boxes represent the population under consideration. The green arrows inform on the selection decision. G, genotype; GEBV, genomic estimated breeding value; P, phenotype; Pred. P, predicted phenotype.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/113/F1.medium.gif" width="440" height="380"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/1/113/F1.medium.gif" width="440" height="380"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/113/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/113/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487958" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Simulation design. This figure represents the scheme used for simulations. The highlighted boxes represent the population under consideration. The green arrows inform on the selection decision. G, genotype; GEBV, genomic estimated breeding value; P, phenotype; Pred. P, predicted phenotype.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Genomic best linear unbiased prediction (GBLUP)</h3><p id="p-9">To investigate the impact of an update to the reference population on GS in terms of subsequent predicted GEBVs, we used both real and simulated data sets. The real data set allowed us to study the impact of reference population updating strategies on the choice of breeding individuals for the next generation only. Simulations were used to study the impact on the breeding population over multiple generations. GEBVs were predicted by a GBLUP model fitted with GS3 software (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Legarra <em>et al.</em> 2011</a>). For the GBLUP model (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Croiseau <em>et al.</em> 2011</a>): (i) the estimated relationship matrix was calculated according to the <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">VanRaden (2008)</a> equation <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-1.gif"/></noscript></span></span>, where <strong>Z</strong> is the genotype matrix and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-2.gif"/></noscript></span></span> the allele frequency of marker <em>i</em>, (ii) the variance components for this trait were the ones used in the routine evaluation in France and were fixed in the model, and (iii) only random effects were fitted as the phenotypes used were already corrected for fixed effects and nongenetic random effects.</p></div><div id="sec-5" class="subsection"><h3>Reference population update</h3><p id="p-10">Three updating strategies were compared: (1) selection at random (Random) repeated 100 independent times, (2) truncation selection based on the highest GEBVs (Sel), and (3) selection to simultaneously maximize the genetic diversity and the genetic merit of the group of selected individuals (SelDiv) using the OC strategy and the Gencont program (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Meuwissen 1997</a>). The genetic merit of a set of selected individuals is the average breeding value (BV) of the selected individuals. The rate of inbreeding (∆F) between the current and next generation is estimated from the average genomic relationships of selected individuals. The OC method identifies a set of individuals with maximum genetic merit with the restriction that the expected rate of inbreeding is no &gt; 1%, as recommended by the <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">FAO (1998)</a>. If the given constraint of 1% cannot be met because of population structure, then the choice of individuals is made to minimize the rate of inbreeding and genetic merit is effectively not considered. The SelDiv strategy used genomic relationships, computed as similarities that count the number of identical alleles, averaged across loci between two individuals (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Nejati-Javaremi <em>et al.</em> 1997</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Eding and Meuwissen 2001</a>):<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-3.gif"/></noscript></span></span>where <em>N</em> is the number of markers and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-4.gif"/></noscript></span></span> is the estimated relationship between individual <em>j</em> and <em>k</em> across all markers. At each marker, <em>i</em>, <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-5.gif"/></noscript></span></span> is the individual variant genotype coded as 0, 1, or 2. Note that computing these relationships using the methods described by <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">VanRaden (2008)</a> and <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Yang <em>et al.</em> (2010)</a>, assuming allele frequencies of 0.5 for all loci, yields exactly the same result. This relationship matrix has been shown to reduce the loss of overall genetic diversity better than other relationship matrices when applying the OC strategy (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Eynard <em>et al.</em> 2016</a>).</p><div id="sec-6" class="subsection"><h4>Update of reference population in real data sets:</h4><p id="p-11">The initial reference population (<em>A</em><sub>1</sub>) was used to predict GEBVs of the individuals in the candidates’ population (<em>A</em><sub>2</sub>). Using these GEBVs and the relationships between individuals in <em>A</em><sub>1</sub> and <em>A</em><sub>2</sub>, we selected subgroups of individuals to build updated reference populations (<em>A</em><sub>1+2</sub>) For all strategies (Random, Sel, and SelDiv), the initial reference population (<em>A</em><sub>1</sub>) of 5969 individuals was updated with 100, 200, 500, 1000, or 2000 new individuals, which represented adding ∼1.5, 3, 8, 15, and 30% to the initial reference population, respectively. The updated reference populations (<em>A</em><sub>1+2</sub>) were used to predict GEBVs of the candidates’ group <em>V</em>. Based on their GEBVs, the top 100 individuals from <em>V</em> were selected as breeding populations, <em>V<sub>sel</sub></em>. A detailed review of all results is available in Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1117/-/DC1/FileS2.docx">File S2</a>.</p></div><div id="sec-7" class="subsection"><h4>Update of reference population in simulated data sets:</h4><p id="p-12">The initial reference population (<em>A</em><sub>1</sub>) consisted of 1000 males from the ancestral individuals and was updated every generation by adding 150 individuals, males and/or females, selected based on one of the proposed strategies (Random, Sel, and SelDiv). The size of the reference population therefore rose from 1000 in the first generation to 2350 individuals in the 10th generation. In each generation, the reference population was updated based on GEBVs from the candidates’ population, and subsequently used for prediction of GEBVs of the simulated offspring. Therefore, individuals in the reference population could be included as part of the breeding population provided that they had been selected for breeding based on their GEBVs. The whole simulation and updating process was replicated 50 times for each strategy.</p></div></div><div id="sec-8" class="subsection"><h3>Evaluation of updating strategies</h3><p id="p-13">To compare the different updating strategies, several parameters were evaluated for the selected candidates’ population (<em>V<sub>sel</sub></em>, top 100 individuals) in the real data set and for the breeding population in the simulated data set. Those parameters included: (i) the response to selection, (ii) the genetic diversity, (iii) prediction bias, and (iv) the effective population size of the reference population. Response to selection was measured as the change in average BV. Genetic diversity was measured as: (i) observed heterozygosity and (ii) the inbreeding coefficient obtained from pedigree following the <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Sargolzaei <em>et al.</em> (2005)</a> algorithm. The bias of GEBV was measured by the absolute standardized prediction errors for the BV as follows:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-6.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-7.gif"/></noscript></span></span> is the GEBV of the individual <em>k</em>, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-8.gif"/></noscript></span></span> is the BV (based on multiple records in the real data set or given by the simulations in the form of a true BV) of the individual <em>k</em>, and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-9.gif"/></noscript></span></span> is the true BV SD of the population under scrutiny <em>i</em>. The effective population size of the reference population, <em>N<sub>e</sub></em>, was also estimated following the classical formula derived from the inbreeding coefficient definition (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Falconer and Mackay 1996</a>):<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-10.gif"/></noscript></span>,</span>with <em>f<sub>t</sub></em> representing the mean inbreeding coefficient of the population in the <em>t</em>th generation.</p><p id="p-14">The effects of the different updating strategies on BV, heterozygosity, inbreeding, and prediction bias were tested using linear models implemented in R and the <em>lme4</em> package (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bates <em>et al.</em> 2015</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">R Core Team 2016</a>), considering the random strategy as the null hypothesis distribution. When dealing with heterozygosity or inbreeding, an arcsine-square root transformation was applied to ensure the applicability of linear models. The effects of strategy and the size of the update were tested using a type II ANOVA [R package <em>car</em> (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Fox and Weisberg 2011</a>)]. Coefficients of change throughout generations were compared using least square means for qualitative variables and least square trends to compare regression slopes for quantitative variables [R package <em>lsmeans</em> (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Lenth 2016</a>)].</p><p id="p-15">For the real data set, linear models were applied on the candidates’ populations as follows,<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-11.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-12.gif"/></noscript></span></span> is the variable measured on individual <em>k</em>, for strategy <em>i</em> (Random, Sel, or SelDiv), when adding <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-13.gif"/></noscript></span></span> number of individuals added to the reference population, fitted here as a qualitative effect (100, 200, 500, 1000, or 2000). <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-14.gif"/></noscript></span></span> is the regression coefficient on the ratio <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-15.gif"/></noscript></span></span>of the reference population (with <em>N</em> the census population size) and <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-16.gif"/></noscript></span></span>is the Gaussian residual. For simulated data sets, we focused on the breeding and offspring populations using the following mixed effects models,<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-17.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-18.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-19.gif"/></noscript></span></span> is the variable measured on individual <em>k</em>, for the strategy <em>i</em>, in generation <em>l</em> of simulation, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-20.gif"/></noscript></span></span> the regression coefficient on the generation number for strategy <em>i</em>, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-21.gif"/></noscript></span></span> is the interaction effect of method with generation, and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-22.gif"/></noscript></span></span> was the random effect of the simulation where <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-23.gif"/></noscript></span></span> represented the data variability among simulation replicates and <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-24.gif"/></noscript></span></span> the gaussian residuals. The ratio <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-25.gif"/></noscript></span></span> of the reference population was used in the model to account for the effect of the change in reference population size through time while accounting for a parallel growth of census population size. This allows one to distinguish between the increases in size over time from the cumulative effect due to consecutive population changes over the 10 generations.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-16">Genetic information (in the form of a <strong>G</strong>-matrix), pedigree (for the individuals under scrutiny), and BV for the trait of interest are available for the real data set, as well as the script allowing the production of the simulated data sets and documents describing each file for real and simulated data sets on the following depository: doi.org/10.5281/zenodo.1066566.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Effect of updating strategy on selected candidates (real data sets)</h3><div id="sec-12" class="subsection"><h4>Genetic merit of the selected candidates:</h4><p id="p-17">Individual BVs in <em>V<sub>sel</sub></em> exhibited large variability and ranged from 461 to 5674. Average BV of <em>V<sub>sel</sub></em> populations, across all combinations of strategies and the size of updates, ranged from 3153.56 to 3185.63 (± 5.21), thus revealing limited variation in genetic gain between different strategies to update the reference population. Even though none of these differences were significant, genetic merit tended to increase when increasing the size of the group used to update the reference population.</p></div><div id="sec-13" class="subsection"><h4>Genetic diversity of the selected candidates:</h4><p id="p-18">Individuals’ inbreeding ranged from 0.02 to 0.11. Over all combinations of strategies and size of updates, per <em>V<sub>sel</sub></em>, the inbreeding coefficients were all on average 0.05 (± 1.14 × 10<sup>−4</sup>) and not significantly different from each other. Individuals’ heterozygosity ranged from 0.28 to 0.33, and average populations’ heterozygosities were all close to the mean value of 0.31 (± 5.65 × 10<sup>−5</sup>) and not significantly different across scenarios.</p></div><div id="sec-14" class="subsection"><h4>Precision of GEBV prediction procedure:</h4><p id="p-19">The prediction bias of GEBVs of the full candidates’ population, <em>V</em>, ranged from 0.00 to 7.73, indicating substantial disparity in how well individuals’ GEBVs are predicted. Across all combinations of strategies and size of updates, average absolute bias of GEBV ranged from 1.05 to 1.08 (± 0.01) without any significant difference among them (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487986/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487986/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Descriptive statistics of the four variables analyzed at group level, for the different strategies and sizes of update in the real data set</span><div class="sb-div caption-clear"></div></div></div><p id="p-21">Overall, no significant differences could be observed between the three tested strategies when considering the top 100 candidates for selection.</p></div></div><div id="sec-15" class="subsection"><h3>Long-term effect of updating strategy on breeding population (simulated data sets)</h3><div id="sec-16" class="subsection"><h4>Genetic merit of the breeding population:</h4><p id="p-22">The average BV of the breeding population always increased from one generation to the next. Despite the fact that strategy significantly affected the realized genetic merit (all p-values &lt; 10<sup>−5</sup>, Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1117/-/DC1/FileS2.docx">File S2</a>), the actual differences between the Sel, SelDiv, and Random strategies were very modest (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1117/-/DC1/FileS2.docx">File S2</a>, and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/113/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evolution of genetic merit, performance of genomic selection, and genetic diversity over 10 generations of simulations for different update strategies. The four plots represent the average genetic merit of the breeding populations (top left), average prediction bias of genomic estimated breeding values of the offspring populations (top right), the average inbreeding (bottom left), and the average heterozygosity (bottom right) of the breeding populations over 10 generations of selection. For the three update strategies Random (gray circle), Sel (magenta triangles), and SelDiv (blue squares), the average values and SE are represented. Random, selection at random repeated 100 independent times; Sel, truncation selection based on the highest genomic estimated breeding values; SelDiv, selection to simultaneously maximize the genetic diversity and the genetic merit of the group of selected individuals using the optimal contribution strategy and the Gencont program." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1839432961" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Evolution of genetic merit, performance of genomic selection, and genetic diversity over 10 generations of simulations for different update strategies. The four plots represent the average genetic merit of the breeding populations (top left), average prediction bias of genomic estimated breeding values of the offspring populations (top right), the average inbreeding (bottom left), and the average heterozygosity (bottom right) of the breeding populations over 10 generations of selection. For the three update strategies Random (gray circle), Sel (magenta triangles), and SelDiv (blue squares), the average values and SE are represented. Random, selection at random repeated 100 independent times; Sel, truncation selection based on the highest genomic estimated breeding values; SelDiv, selection to simultaneously maximize the genetic diversity and the genetic merit of the group of selected individuals using the optimal contribution strategy and the Gencont program.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/113/F2.medium.gif" width="440" height="429"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/1/113/F2.medium.gif" width="440" height="429"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/113/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/113/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488037" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Evolution of genetic merit, performance of genomic selection, and genetic diversity over 10 generations of simulations for different update strategies. The four plots represent the average genetic merit of the breeding populations (top left), average prediction bias of genomic estimated breeding values of the offspring populations (top right), the average inbreeding (bottom left), and the average heterozygosity (bottom right) of the breeding populations over 10 generations of selection. For the three update strategies Random (gray circle), Sel (magenta triangles), and SelDiv (blue squares), the average values and SE are represented. Random, selection at random repeated 100 independent times; Sel, truncation selection based on the highest genomic estimated breeding values; SelDiv, selection to simultaneously maximize the genetic diversity and the genetic merit of the group of selected individuals using the optimal contribution strategy and the Gencont program.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487855/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487855/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Trends of changes throughout the 10 generations of simulation for each of the three updating strategies and four variables</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h4>Genetic diversity of the breeding population:</h4><p id="p-25">Whatever the strategy, the inbreeding coefficient increased from one generation to the next. Despite large SE (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), the increase in inbreeding coefficients throughout the 10 generations appeared to be significantly slower for SelDiv than for the two other strategies (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). Inbreeding level was significantly associated with both generation number and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-26.gif"/></noscript></span></span> (p-values &lt; 10<sup>−16</sup>, Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1117/-/DC1/FileS2.docx">File S2</a>). Both an increase in generation number and a decrease in <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-27.gif"/></noscript></span></span> was associated with an increase of the average population inbreeding. After the fourth generation, the SelDiv strategy resulted in higher heterozygosity than the Sel or Random strategies (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>) due to a slower decrease over generations (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). All the parameters—strategy (p-value = 1.12 × 10<sup>−2</sup>), <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-28.gif"/></noscript></span></span> (p-value = 1.26 × 10<sup>−6</sup>), generation number, and the interaction between strategy and generation (both with p-values &lt; 10<sup>−16</sup>)—significantly affected the heterozygosity (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1117/-/DC1/FileS2.docx">File S2</a>). The effect of <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-29.gif"/></noscript></span></span> was positive; an increase in <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-30.gif"/></noscript></span></span> caused an increase in average heterozygosity of the population. Average heterozygosity decreased from one generation to the next faster for the Random and Sel strategies than for SelDiv.</p></div><div id="sec-18" class="subsection"><h4>Precision of GEBV prediction procedure:</h4><p id="p-26">For all generations, on average the Sel strategy, and even more the SelDiv strategy, resulted in lower prediction bias of the offspring’s GEBVs than the Random strategy (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1117/-/DC1/FileS2.docx">File S2</a>). The parameters strategy, generation number, interaction between strategy and generation, and <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-31.gif"/></noscript></span></span> significantly affected prediction bias, with p-values &lt; 10<sup>−10</sup> (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1117/-/DC1/FileS2.docx">File S2</a>). The Random, Sel, and SelDiv strategies were significantly different from each other (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). A shift was observed at the fourth generation, with the Random strategy having the largest bias, whereas the SelDiv strategy had the lowest bias (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). Despite the apparently chaotic behavior of this variable, prediction bias tended to increase over time faster for the Random and Sel strategies than SelDiv. The small effect of <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/113/embed/mml-math-32.gif"/></noscript></span></span> on the prediction bias is presumably due to the decline in relationships between reference and candidate populations through time, as a result of the constant addition of new individuals without the removal of older ones.</p><p id="p-27">To summarize, the results above show that different strategies to update the reference population have a significant, but small, impact on the breeding population. The SelDi<em>v</em> strategy resulted in slightly higher genetic diversity in the breeding population accompanied by a minor impact on the genetic gain and lower long-term prediction bias.</p></div></div></div><div class="section discussion" id="sec-19"><h2 class="">Discussion</h2><p id="p-28">In this study, we compared the impact of different strategies to update the reference population in a GS framework on the genetic merit and diversity of the resulting breeding population. Optimizing the updating strategy is especially important in artificial selection based on the genotypes of individuals at an early age. This is because phenotyping is the limiting factor due to the time and money investment for the rearing of the individuals (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Colleau <em>et al.</em> 2009</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Konig <em>et al.</em> 2009</a>). It is also relevant when both phenotypes and genotypes are available, but only a fraction can be included in the reference population, for example, when designing a core collection in plant breeding (<a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Rincent <em>et al.</em> 2012</a>; <a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Isidro <em>et al.</em> 2015</a>). In GS, reference population design and breeding decisions are linked through GEBVs of selection candidates. Our hypothesis was that the choice of individuals in building the reference population might impact the GEBVs of selection candidates and, consequently, the breeding population, both in terms of genetic gain and diversity.</p><div id="sec-20" class="subsection"><h3>Long-term impact of updating strategy on the breeding population</h3><p id="p-29">Analysis based on a single generation in the real data set did not show significant differences between the three proposed updating strategies; however, analysis based on simulated data sets over 10 generations did show significant effects of the updating strategy on the breeding populations over time. A small beneficial response of the truncation strategy was observed for genetic merit, while the OC strategy performed best at conserving genetic diversity.</p><p id="p-30">A recent study by <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">De Beukelaer <em>et al.</em> (2017)</a> focused on the similar question of how to balance genetic gain and genetic diversity conservation in populations under selection. The authors used simulations to compare established selection strategies: GS including OC (GOCS) and GS weighting for rare alleles (GSW) for long-term genetic diversity conservation in plant breeding. Even though both GOCS and GSW outperformed GS for long-term genetic gain, they were not successful in controlling inbreeding rate and loss of rare variants in the breeding population. These authors proposed two new strategies combining an index-based method and expected heterozygosity or rare allele frequencies as alternatives outperforming GS, GOCS, and GSW in balancing genetic gain and diversity. These methods require further investigation to confirm their benefits in practice but could be of potential interest to answer the questions we raised in this study.</p><p id="p-31">Approaches proposed in plant breeding to design reference populations representing population structure and diversity (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Laloe 1993</a>; <a id="xref-ref-55-5" class="xref-bibr" href="#ref-55">Rincent <em>et al.</em> 2012</a>; <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Isidro <em>et al.</em> 2015</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bartholomé <em>et al.</em> 2016</a>) could also be alternatives in the context of animal breeding. In fact, the current concerns of how to best design reference populations by targeting only relevant individuals is also now of interest in animal breeding due to the increasing availability of individual information both for phenotypes and genotypes. The data on livestock reference populations are now far more comprehensive and should enable choices regarding which individuals should be present in reference populations to take place. Therefore, methods used in plant breeding, mostly to design core collections, may be of interest to animal breeders.</p></div><div id="sec-21" class="subsection"><h3>Potential implications for animal breeding</h3><p id="p-32">In practice, breeding decisions are mainly based on the genetic merit of individuals. This is because breeders’ incomes come from production. This phenomenon is putting small breeds in a difficult situation, in a market mostly dominated by mainstream breeds, because of their limited population size, high inbreeding rates, and lower fitness potential (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Toro <em>et al.</em> 2009</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Allendorf <em>et al.</em> 2010</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Pryce and Daetwyler 2012</a>). Livestock breeding has to balance the conservation of genetic diversity against genetic gain. Within GS, the adoption of alternative selection strategies, such as OC, is not common in practice. Acting on the reference population to directly mitigate the loss of genetic diversity of the breeding population, while only marginally affecting the genetic gain over generations, is a promising way to incorporate genetic diversity into breeding programs. Indeed, current methods to cope with the loss of genetic diversity mainly deal with the choice of which individuals to keep in the breeding population according to their estimated performances. On the one hand, direct selection of breeding individuals has the advantage of having a strong impact on both the level of genetic diversity and genetic gain, depending on the method used. On the other hand, it relies on the choice of the breeders and is thus not systematic. Here, we propose an integrated method to cope with genetic diversity at the genetic evaluation level, making it systematically incorporated. Thus, even if its impact on the conservation of genetic diversity is weaker than direct choices in the short-term, it potentially has a more consistent impact on a long-term basis. We expect that in the ideal case of operating on both the reference and breeding population, the effect observed would be further amplified and thus have an important impact on genetic diversity conservation.</p></div><div id="sec-22" class="subsection"><h3>Limitations and perspectives of the study</h3><p id="p-33">The 50K SNP chip is routinely used in GS because of its low cost and fair performance for genetic gain. Several studies cautioned that the accuracy of prediction in GS when using whole-genome sequencing (WGS) was, at best, marginally higher than of the SNP chips (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">van Binsbergen <em>et al.</em> 2015</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Calus <em>et al.</em> 2016</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lund <em>et al.</em> 2016</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">van den Berg <em>et al.</em> 2016</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Ni <em>et al.</em> 2017</a>). Still, we can hypothesize that using WGS or genotypes of higher density could favor larger differences in genetic diversity conserved between the described scenarios. This may be especially true in the case of rare variant sites, since they are underrepresented in the SNP chip compared to WGS (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Eynard <em>et al.</em> 2015</a>, <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">2016</a>). Using WGS could enable the OC strategy to better conserve rare variants during updates of the reference population.</p><p id="p-34">Prediction bias appeared to be smaller in the case of the OC strategy compared to the other two strategies. Increasing the genetic diversity of the reference population increased our representation of the overall population diversity and seemed to lead to slightly more accurate overall prediction. This is potentially thanks to an improved prediction of “outsider” variants. Additionally, particular attention should be paid to how many and which individuals should be removed. In fact, bias was first reduced by the addition of specifically selected individuals (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Pszczola <em>et al.</em> 2012b</a>). However, after some generations, adding individuals elevated the prediction bias. This is probably due to the lack of a relationship between the old individuals of the reference population and the candidates for selection. There is a need for further investigation in order to give recommendations as to the best updating strategy for reference populations, accounting for the addition and removal of individuals. Finally, our study is based on milk production, a trait of major interest in current livestock, with moderate heritability (0.3) that is similar to the those of composite index traits that represent the entire breeding goal. An important question is how results would change when the heritability is lower, because GS is especially appealing for low-heritability traits. Using a lower heritability, while leaving the reference population size unchanged, would have yielded lower prediction accuracies and also smaller differences between scenarios. A lower accuracy means that more emphasis is put on information from relatives, such that the EBV of relatives becomes more correlated and thus selected individuals are more likely to be related. This would result in conserving less genetic diversity and more inbreeding depression. Increasing the size of the reference population could counteract these effects of a low-heritability trait, because it would increase the accuracy (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Daetwyler <em>et al.</em> 2010</a>). This is provided that increasing the reference population is possible given, for example, the size of the actual population.</p></div><div id="sec-23" class="subsection"><h3>Conclusions</h3><p id="p-35">The aim of this study was to investigate ways to reduce the loss of genetic diversity in GS breeding programs. The choice of individuals to be phenotyped and/or added to the reference population appeared to modestly impact the genetic gain and genetic diversity of the breeding population. The use of the OC strategy, taking into account both the relationships and performances of the individuals, to update the reference population: (i) allowed for better conservation of genetic diversity in the breeding population, (ii) predicted more accurate BV, and (iii) had only minor repercussions on the genetic gain. The results of this study support the use of the OC strategy as a way to update the reference population, especially for breeds in need of diversity conservation wanting to implement long-term GS programs. Making changes in the composition of the reference population impacted the breeding population characteristics and enabled the incorporation of genetic diversity in GS without revising farmers’ practices.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-36">The authors thank V. Ducrocq for discussions and suggestions given on the analysis, J. Vandenplas for his help with the programming, and R. Rincent for discussions on the methodology; the editor, J. B. Holland, and the reviewers for their comments and contributions to the improvement of the manuscript; and the projects VALOGENE, CARTOFINE, AMASGEN and LACTOSCAN, funded by the French National Agency for Research, and APIS-GENE for producing the data. S.E.E. benefited from a grant from the European Commission, within the framework of the Erasmus Mundus joint doctorate European Graduate School in Animal Breeding and Genetics, cofunded by the Dutch Ministry of Economic Affairs (grant KB-21-004-003).</p><p id="p-37">Author contributions: G.R., D.L., P.C., and S.E.E. designed the study. S.F. provided the data. P.C. provided analytical tools. S.E.E. and G.R. performed the statistical analysis. S.E.E., G.R., and M.P.L.C. drafted the manuscript. G.R., D.L., P.C., M.P.L.C., and S.E.E. contributed to the interpretation of results. G.R., D.L., P.C., M.P.L.C., and S.F. contributed to the discussion and commented on the manuscript. All authors read and approved the manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-38">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1117/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.1117/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-39"><em>Communicating editor: J. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-07-20"><span class="received-label">Received </span>July 20, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-10-31"><span class="accepted-label">Accepted </span>October 31, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Eynard <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.1" data-doi="10.1038/nrg2844"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allendorf</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luikart</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomics and the future of conservation genetics.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">697</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DAllendorf%26rft.auinit1%253DF.%2BW.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253D697%26rft.epage%253D709%26rft.atitle%253DGenomics%2Band%2Bthe%2Bfuture%2Bof%2Bconservation%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2844%26rft_id%253Dinfo%253Apmid%252F20847747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2844&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20847747&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bartholomé</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Heerwaarden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isik</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boury</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Performance of genomic prediction within and across generations in maritime pine.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">604</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D17%26rft.spage%253D604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bastiaansen</span>  <span class="cit-name-given-names">J. W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coster</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Arendonk</span>  <span class="cit-name-given-names">J. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bovenhuis</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Long-term response to genomic selection: effects of estimation method and reference population structure for different genetic architectures.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DDvorakova%26rft.auinit1%253DV.%26rft.volume%253D44%26rft.spage%253D13%26rft.epage%253D13%26rft.atitle%253DAssociation%2Bbetween%2Bpolymorphism%2Bin%2Bthe%2BFTO%2Bgene%2Band%2Bgrowth%2Band%2Bcarcass%2Btraits%2Bin%2Bpig%2Bcrosses.%26rft_id%253Dinfo%253Apmid%252F22510482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22510482&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Machler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolker</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">S. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fitting linear mixed-effects models using lme4.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D67%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boichard</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ducrocq</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Sustainable dairy cattle selection in the genomic era.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">143</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D132%26rft.spage%253D135%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.6" data-doi="10.1086/521987"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1084</span>–<span class="cit-lpage">1097</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DS.%2BR.%26rft.volume%253D81%26rft.issue%253D5%26rft.spage%253D1084%26rft.epage%253D1097%26rft.atitle%253DRapid%2Band%2Baccurate%2Bhaplotype%2Bphasing%2Band%2Bmissing-data%2Binference%2Bfor%2Bwhole-genome%2Bassociation%2Bstudies%2Bby%2Buse%2Bof%2Blocalized%2Bhaplotype%2Bclustering.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F521987%26rft_id%253Dinfo%253Apmid%252F17924348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/521987&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17924348&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250480900018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bruford</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ginja</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joost</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Orozco-terWengel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Prospects and challenges for the conservation of farm animal genomic resources, 2015–2025.</span> <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">314</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D6%26rft.spage%253D314%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.8" data-doi="10.1017/S1751731109991352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic breeding value prediction: methods and procedures.</span> <abbr class="cit-jnl-abbrev">Animal</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DAnimal%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D157%26rft.epage%253D164%26rft.atitle%253DGenomic%2Bbreeding%2Bvalue%2Bprediction%253A%2Bmethods%2Band%2Bprocedures.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS1751731109991352%26rft_id%253Dinfo%253Apmid%252F22443868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S1751731109991352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22443868&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274461500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.9" data-doi="10.1186/1297-9686-43-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of multi-trait genomic selection using different methods.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%26rft.volume%253D43%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DAccuracy%2Bof%2Bmulti-trait%2Bgenomic%2Bselection%2Busing%2Bdifferent%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-26%26rft_id%253Dinfo%253Apmid%252F21729282%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21729282&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouwman</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrooten</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Efficient genomic prediction based on whole-genome sequence data using split-and-merge Bayesian variable selection.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D19%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.11" data-doi="10.1186/1297-9686-45-44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kinghorn</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van der Werf</span>  <span class="cit-name-given-names">J. H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The effect of genomic information on optimal contribution selection in livestock breeding programs.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D45%26rft.spage%253D44%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-45-44%26rft_id%253Dinfo%253Apmid%252F24171942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-45-44&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24171942&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colleau</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dupassieux</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Simulating the potential of genomic selection in dairy cattle breeding.</span> <abbr class="cit-jnl-abbrev">Rencontres Recherches Ruminants</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRencontres%2BRecherches%2BRuminants%26rft.volume%253D16%26rft.spage%253D419%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.13" data-doi="10.1017/S0016672311000358"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Croiseau</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fritz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fine tuning genomic evaluations in dairy cattle through SNP pre-selection with the Elastic-Net algorithm.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DCroiseau%26rft.auinit1%253DP.%26rft.volume%253D93%26rft.issue%253D6%26rft.spage%253D409%26rft.epage%253D417%26rft.atitle%253DFine%2Btuning%2Bgenomic%2Bevaluations%2Bin%2Bdairy%2Bcattle%2Bthrough%2BSNP%2Bpre-selection%2Bwith%2Bthe%2BElastic-Net%2Balgorithm.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672311000358%26rft_id%253Dinfo%253Apmid%252F22189606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672311000358&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22189606&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.14" data-doi="10.1534/genetics.110.116855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The impact of genetic architecture on genome-wide evaluation methods.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1021</span>–<span class="cit-lpage">1031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116855%26rft_id%253Dinfo%253Apmid%252F20407128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS8zLzEwMjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danchin-Burge</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leroy</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brochard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moureaux</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verrier</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution of the genetic variability of eight French dairy cattle breeds assessed by pedigree analysis.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">206</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DDanchin-Burge%26rft.auinit1%253DC.%26rft.volume%253D129%26rft.issue%253D3%26rft.spage%253D206%26rft.epage%253D217%26rft.atitle%253DEvolution%2Bof%2Bthe%2Bgenetic%2Bvariability%2Bof%2Beight%2BFrench%2Bdairy%2Bcattle%2Bbreeds%2Bassessed%2Bby%2Bpedigree%2Banalysis.%26rft_id%253Dinfo%253Apmid%252F22583325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22583325&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.16" data-doi="10.1534/genetics.116.194449"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Beukelaer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badke</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fack</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Meyer</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Moving beyond managing realized genomic relationship in long-term genomic selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">206</span>: <span class="cit-fpage">1127</span>–<span class="cit-lpage">1138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.194449%26rft_id%253Dinfo%253Apmid%252F28381589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwNi8yLzExMjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.17" data-doi="10.1111/mec.12560"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Cara</span>  <span class="cit-name-given-names">M. A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Using genomic tools to maintain diversity and fitness in conservation programmes.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">6091</span>–<span class="cit-lpage">6099</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D6091%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12560&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.18" data-doi="10.1534/genetics.109.108431"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Druet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Georges</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A hidden Markov model combining linkage and linkage disequilibrium information for haplotype reconstruction and quantitative trait locus fine mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">798</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.108431%26rft_id%253Dinfo%253Apmid%252F20008575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg0LzMvNzg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzExMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.19" data-doi="10.1046/j.1439-0388.2001.00290.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eding</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Marker-based estimates of between and within population kinships for the conservation of genetic diversity.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">118</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D118%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1439-0388.2001.00290.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1439-0388.2001.00290.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000169434900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engelsma</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Windig</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Consequences for diversity when prioritizing animals for conservation with pedigree or genomic information.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">473</span>–<span class="cit-lpage">481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DEngelsma%26rft.auinit1%253DK.%2BA.%26rft.volume%253D128%26rft.issue%253D6%26rft.spage%253D473%26rft.epage%253D481%26rft.atitle%253DConsequences%2Bfor%2Bdiversity%2Bwhen%2Bprioritizing%2Banimals%2Bfor%2Bconservation%2Bwith%2Bpedigree%2Bor%2Bgenomic%2Binformation.%26rft_id%253Dinfo%253Apmid%252F22059581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22059581&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Engelsma</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veerkamp</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bijma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Windig</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Pedigree- and marker-based methods in the estimation of genetic diversity in small groups of Holstein cattle.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">205</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DEngelsma%26rft.auinit1%253DK.%2BA.%26rft.volume%253D129%26rft.issue%253D3%26rft.spage%253D195%26rft.epage%253D205%26rft.atitle%253DPedigree-%2Band%2Bmarker-based%2Bmethods%2Bin%2Bthe%2Bestimation%2Bof%2Bgenetic%2Bdiversity%2Bin%2Bsmall%2Bgroups%2Bof%2BHolstein%2Bcattle.%26rft_id%253Dinfo%253Apmid%252F22583324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22583324&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eynard</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Windig</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leroy</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Binsbergen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The effect of rare alleles on estimated genomic relationships from whole genome sequence data.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D16%26rft.spage%253D12%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eynard</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Windig</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiemstra</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Whole-genome sequence data uncover loss of genetic diversity due to selection.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.113.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source"><em>Introduction to Quantative Genetics</em></span>, <span class="cit-edition">Ed. 4</span>. <span class="cit-publ-name">Longman Scientific &amp; Technical</span>, <span class="cit-publ-loc">Harlow, England</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.113.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">FAO</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source"><em>Inbreeding and Brood Stock Management</em></span>. <span class="cit-publ-name">Electronic Publishing Policy and Support Branch, Communication Division FAO</span>, <span class="cit-publ-loc">Rome</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fikse</span>  <span class="cit-name-given-names">W. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Weighting factors of sire daughter information in international genetic evaluations.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">1759</span>–<span class="cit-lpage">1767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DFikse%26rft.auinit1%253DW.%2BF.%26rft.volume%253D84%26rft.issue%253D7%26rft.spage%253D1759%26rft.epage%253D1767%26rft.atitle%253DWeighting%2BFactors%2Bof%2BSire%2BDaughter%2BInformation%2Bin%2BInternational%2BGenetic%2BEvaluations%26rft_id%253Dinfo%253Apmid%252F11467826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11467826&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.113.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weisberg</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-source"><em>An R Companion to Applied Regression</em></span>. <span class="cit-publ-name">Sage</span>, <span class="cit-publ-loc">Thousand Oaks, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.28" data-doi="10.1007/s10709-008-9308-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection: prediction of accuracy and maximisation of long term response.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D136%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-008-9308-0%26rft_id%253Dinfo%253Apmid%252F18704696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-008-9308-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18704696&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265818900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henryon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sørensen</span>  <span class="cit-name-given-names">A. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Invited review: animal-breeding schemes using genomic information need breeding plans designed to maximise long-term genetic gains.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">38</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D166%26rft.spage%253D38%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Impact of marker ascertainment bias on genomic selection accuracy and estimates of genetic diversity.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e74612</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De74612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoze</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fouilloux</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venot</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guillaume</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dassonneville</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-density marker imputation accuracy in sixteen French cattle breeds.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D45%26rft.spage%253D33%26rft_id%253Dinfo%253Apmid%252F24004563%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24004563&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.32" data-doi="10.1007/s00122-014-2418-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isidro</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Training set optimization under population structure in genomic selection.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">145</span>–<span class="cit-lpage">158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D128%26rft.spage%253D145%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2418-4%26rft_id%253Dinfo%253Apmid%252F25367380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2418-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25367380&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.33" data-doi="10.1186/1297-9686-42-35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Dynamics of long-term genomic selection.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DJannink%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D42%26rft.spage%253D35%26rft.epage%253D35%26rft.atitle%253DDynamics%2Bof%2Blong-term%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-35%26rft_id%253Dinfo%253Apmid%252F20712894%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-35&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20712894&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.34" data-doi="10.1186/1471-2164-13-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khatkar</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moser</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raadsma</span>  <span class="cit-name-given-names">H. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Strategies and utility of imputed SNP genotypes for genomic analysis in dairy cattle.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DOyola%26rft.auinit1%253DS.%2BO.%26rft.volume%253D13%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DOptimizing%2BIllumina%2Bnext-generation%2Bsequencing%2Blibrary%2Bpreparation%2Bfor%2Bextremely%2BAT-biased%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-1%26rft_id%253Dinfo%253Apmid%252F22214261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22214261&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.35" data-doi="10.3168/jds.2008-1310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Konig</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simianer</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willam</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Economic evaluation of genomic breeding programs.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">382</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DKonig%26rft.auinit1%253DS.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D382%26rft.epage%253D391%26rft.atitle%253DEconomic%2Bevaluation%2Bof%2Bgenomic%2Bbreeding%2Bprograms%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2008-1310%26rft_id%253Dinfo%253Apmid%252F19109296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2008-1310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19109296&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.36" data-doi="10.1051/gse:19930604"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laloe</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Precision and information in linear-models of genetic evaluation.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D25%26rft.spage%253D557%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19930604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19930604&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-other" id="cit-8.1.113.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ricard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filangi</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> GS3: Genomic Selection, Gibbs Sampling, Gauss-Seidel (and BayesCπ). Available at <a href="http://snp.toulouse.inra.fr/%E2%88%BCalegarra/">http://snp.toulouse.inra.fr/∼alegarra/</a>. Accessed: November 10, 2016.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.38" data-doi="10.18637/jss.v069.i01"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lenth</span>  <span class="cit-name-given-names">R. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Least-squares means: the R package lsmeans.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D69%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.18637%252Fjss.v069.i01%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.18637/jss.v069.i01&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leroy</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danchin-Burge</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verrier</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Impact of the use of cryobank samples in a selected cattle breed: a simulation study.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DLeroy%26rft.auinit1%253DG.%26rft.volume%253D43%26rft.spage%253D36%26rft.epage%253D36%26rft.atitle%253DImpact%2Bof%2Bthe%2Buse%2Bof%2Bcryobank%2Bsamples%2Bin%2Ba%2Bselected%2Bcattle%2Bbreed%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Apmid%252F22047533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22047533&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.40" data-doi="10.1111/j.1439-0388.2007.00717.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadarmideen</span>  <span class="cit-name-given-names">H. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J. C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Selection on multiple QTL with control of gene diversity and inbreeding for long-term benefit.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">320</span>–<span class="cit-lpage">329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DLi%26rft.auinit1%253DY.%26rft.volume%253D125%26rft.issue%253D5%26rft.spage%253D320%26rft.epage%253D329%26rft.atitle%253DSelection%2Bon%2Bmultiple%2BQTL%2Bwith%2Bcontrol%2Bof%2Bgene%2Bdiversity%2Band%2Binbreeding%2Bfor%2Blong-term%2Bbenefit.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00717.x%26rft_id%253Dinfo%253Apmid%252F18803787%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00717.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18803787&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259309000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.113.41"><div class="cit-metadata unstructured">Liu, H., 2013 Application of dense marker genotypes for long-term genetic gain in animal breeding schemes. Ph.D. Thesis, Aarhus University, Aarhus, Denmark.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Berg</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brøndum</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">How to improve genomic predictions in small dairy cattle populations.</span> <abbr class="cit-jnl-abbrev">Animal</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1042</span>–<span class="cit-lpage">1049</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%26rft.volume%253D10%26rft.spage%253D1042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.113.43"><div class="cit-metadata unstructured">Maignel, L., D. Boichard, and E. Verrier, 1996 Genetic variability of French dairy breeds estimated from pedigree information. Proceedings of the Open Session of the Interbull Annual Meeting, Veldhoven, The Netherlands, Vol. 14, pp. 6.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.44" data-doi="10.1146/annurev-animal-031412-103705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accelerating improvement of livestock with genomic selection.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Anim. Biosci.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BAnim.%2BBiosci.%26rft.volume%253D1%26rft.spage%253D221%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-animal-031412-103705%26rft_id%253Dinfo%253Apmid%252F25387018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-animal-031412-103705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25387018&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Maximizing the response of selection with a predefined rate of inbreeding.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">934</span>–<span class="cit-lpage">940</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%26rft.volume%253D75%26rft.issue%253D4%26rft.spage%253D934%26rft.epage%253D940%26rft.atitle%253DMaximizing%2Bthe%2Bresponse%2Bof%2Bselection%2Bwith%2Ba%2Bpredefined%2Brate%2Bof%2Binbreeding%26rft_id%253Dinfo%253Apmid%252F9110204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9110204&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WU20500008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMTEzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.47" data-doi="10.1016/j.livprodsci.2003.09.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mrode</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swanson</span>  <span class="cit-name-given-names">G. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Calculating cow and daughter yield deviations and partitioning of genetic evaluations under a random regression model.</span> <abbr class="cit-jnl-abbrev">Livest. Prod. Sci.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BProd.%2BSci.%26rft.volume%253D86%26rft.spage%253D253%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livprodsci.2003.09.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livprodsci.2003.09.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nejati-Javaremi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Effect of total allelic relationship on accuracy of evaluation and response to selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">1738</span>–<span class="cit-lpage">1745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DNejati-Javaremi%26rft.auinit1%253DA.%26rft.volume%253D75%26rft.issue%253D7%26rft.spage%253D1738%26rft.epage%253D1745%26rft.atitle%253DEffect%2Bof%2Btotal%2Ballelic%2Brelationship%2Bon%2Baccuracy%2Bof%2Bevaluation%2Band%2Bresponse%2Bto%2Bselection%26rft_id%253Dinfo%253Apmid%252F9222829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9222829&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XH54200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cavero</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fangmann</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erbe</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simianer</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Whole-genome sequence-based genomic prediction in laying chickens with different genomic relationship matrices to account for genetic architecture.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D49%26rft.spage%253D8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.50" data-doi="10.1071/AN11098"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pryce</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Designing dairy cattle breeding schemes under genomic selection: a review of international research.</span> <abbr class="cit-jnl-abbrev">Anim. Prod. Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BProd.%2BSci.%26rft.volume%253D52%26rft.spage%253D107%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAN11098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AN11098&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000301064800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.51" data-doi="10.3168/jds.2009-2840"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pszczola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulder</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Effect of enlarging the reference population with (un)genotyped animals on the accuracy of genomic selection in dairy cattle.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DPszczola%26rft.auinit1%253DM.%26rft.volume%253D94%26rft.issue%253D1%26rft.spage%253D431%26rft.epage%253D441%26rft.atitle%253DEffect%2Bof%2Benlarging%2Bthe%2Breference%2Bpopulation%2Bwith%2B%2528un%2529genotyped%2Banimals%2Bon%2Bthe%2Baccuracy%2Bof%2Bgenomic%2Bselection%2Bin%2Bdairy%2Bcattle.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2840%26rft_id%253Dinfo%253Apmid%252F21183054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2840&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21183054&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.52" data-doi="10.3168/jds.2011-4338"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pszczola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strabel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mulder</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Reliability of direct genomic values for animals with different relationships within and to the reference population.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">400</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DPszczola%26rft.auinit1%253DM.%26rft.volume%253D95%26rft.issue%253D1%26rft.spage%253D389%26rft.epage%253D400%26rft.atitle%253DReliability%2Bof%2Bdirect%2Bgenomic%2Bvalues%2Bfor%2Banimals%2Bwith%2Bdifferent%2Brelationships%2Bwithin%2Band%2Bto%2Bthe%2Breference%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2011-4338%26rft_id%253Dinfo%253Apmid%252F22192218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2011-4338&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22192218&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pszczola</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strabel</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Arendonk</span>  <span class="cit-name-given-names">J. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">The impact of genotyping different groups of animals on accuracy when moving from traditional to genomic selection.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">5412</span>–<span class="cit-lpage">5421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DPszczola%26rft.auinit1%253DM.%26rft.volume%253D95%26rft.issue%253D9%26rft.spage%253D5412%26rft.epage%253D5421%26rft.atitle%253DThe%2Bimpact%2Bof%2Bgenotyping%2Bdifferent%2Bgroups%2Bof%2Banimals%2Bon%2Baccuracy%2Bwhen%2Bmoving%2Bfrom%2Btraditional%2Bto%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Apmid%252F22916948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22916948&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.113.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-source"><em>R: A Language and Environment for Statistical Computing</em></span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.55" data-doi="10.1534/genetics.112.141473"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rincent</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laloe</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maximizing the reliability of genomic selection by optimizing the calibration set of reference individuals: comparison of methods in two diverse groups of maize inbreds (<em>Zea mays L</em>.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141473%26rft_id%253Dinfo%253Apmid%252F22865733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNzE1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzExMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.56" data-doi="10.1093/bioinformatics/btp045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sargolzaei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schenkel</span>  <span class="cit-name-given-names">F. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">QMSim: a large-scale genome simulator for livestock.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">680</span>–<span class="cit-lpage">681</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp045%26rft_id%253Dinfo%253Apmid%252F19176551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19176551&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.57" data-doi="10.1111/j.1439-0388.2005.00538.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sargolzaei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iwaisaki</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Colleau</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A fast algorithm for computing inbreeding coefficients in large populations.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DSargolzaei%26rft.auinit1%253DM.%26rft.volume%253D122%26rft.issue%253D5%26rft.spage%253D325%26rft.epage%253D331%26rft.atitle%253DA%2Bfast%2Balgorithm%2Bfor%2Bcomputing%2Binbreeding%2Bcoefficients%2Bin%2Blarge%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2005.00538.x%26rft_id%253Dinfo%253Apmid%252F16191041%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2005.00538.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16191041&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.58" data-doi="10.1186/1297-9686-44-27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonesson</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woolliams</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection requires genomic control of inbreeding.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DSonesson%26rft.auinit1%253DA.%2BK.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D27%26rft.atitle%253DGenomic%2Bselection%2Brequires%2Bgenomic%2Bcontrol%2Bof%2Binbreeding.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-44-27%26rft_id%253Dinfo%253Apmid%252F22898324%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-44-27&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22898324&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stock</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reents</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic selection: status in different species and challenges for breeding.</span> <abbr class="cit-jnl-abbrev">Reprod. Domest. Anim.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">2</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DReprod.%2BDomest.%2BAnim.%26rft.volume%253D48%26rft.spage%253D2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.60" data-doi="10.1016/j.livsci.2008.07.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caballero</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Molecular characterization of breeds and its use in conservation.</span> <abbr class="cit-jnl-abbrev">Livest. Sci.</abbr> <span class="cit-vol">120</span>: <span class="cit-fpage">174</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLivest.%2BSci.%26rft.volume%253D120%26rft.spage%253D174%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.livsci.2008.07.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.livsci.2008.07.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263322600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Binsbergen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bink</span>  <span class="cit-name-given-names">M. C. A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeuwijk</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schrooten</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic prediction using imputed whole-genome sequence data in Holstein Friesian cattle.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D47%26rft.spage%253D71%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.62"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van den Berg</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boichard</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Sequence variants selected from a multibreed GWAS can improve the reliability of genomic predictions in dairy cattle.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D48%26rft.spage%253D83%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.63" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.113.64" data-doi="10.1038/ng.608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benyamin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEvoy</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henders</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Common SNPs explain a large proportion of the heritability for human height.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D7%26rft.spage%253D565%26rft.epage%253D569%26rft.atitle%253DCommon%2BSNPs%2Bexplain%2Ba%2Blarge%2Bproportion%2Bof%2Bthe%2Bheritability%2Bfor%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.608%26rft_id%253Dinfo%253Apmid%252F20562875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20562875&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F113.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279242400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/1/113.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/105" title="Analysis of Copy Number Variants on Chromosome 21 in Down Syndrome-Associated Congenital Heart Defects" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/123" title="Modeling the Evolution of Female Meiotic Drive in Maize" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/1"><span>Volume 8 Issue 1, January 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/1.cover-source.jpg?itok=vyoqxep2" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/113" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tK9WI_x2eqs0I_R4Z8h0WJGOB5nwFn9CIDxWbOfMuRg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/1/113" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/1/113" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/1/113" class="active" data-icon-position="" data-hide-link-title="0">Which Individuals To Choose To Update the Reference Population? Minimizing the Loss of Genetic Diversity in Animal Genomic Selection Programs</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487828" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-VsnZe1n4nxanr9Z79t6ETHZ9FJta3akJuNsPtGdlsYM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/113.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/1/113" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-euKmNubxnxUTQ5Imk55BToA7DOAJaWRR_SgZxVgDN8E" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/1/113" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/113.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487828" id="node487828--2" data-pisa="ggg;8/1/113" data-pisa-master="ggg;g3.117.1117" data-apath="/ggg/8/1/113.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Which Individuals To Choose To Update the Reference Population? Minimizing the Loss of Genetic Diversity in Animal Genomic Selection Programs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8609-5869" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sonia E. Eynard</span>, <span class="highwire-citation-author" data-delta="1">Pascal Croiseau</span>, <span class="highwire-citation-author" data-delta="2">Denis Laloë</span>, <span class="highwire-citation-author" data-delta="3">Sebastien Fritz</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3213-704X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mario P. L. Calus</span> and <span class="highwire-citation-author" data-delta="5">Gwendal Restoux</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">113-121;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.1117 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487828--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sonia E. Eynard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics Centre, Wageningen University & Research, 6700 AH Wageningen, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre for Genetic Resources, Wageningen University & Research, the Netherlands, 6700 AA Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sonia%2BE.%2BEynard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eynard%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASonia%2BE.%2BEynard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8609-5869" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sonia E. Eynard</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sonia.eynard@wur.nl" class="" data-icon-position="" data-hide-link-title="0">sonia.eynard@wur.nl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pascal Croiseau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pascal%2BCroiseau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Croiseau%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APascal%2BCroiseau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Denis Laloë </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Denis%2BLalo%C3%AB%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lalo%C3%AB%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADenis%2BLalo%25C3%25AB%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sebastien Fritz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Allice, 75595 Paris Cedex 12, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastien%2BFritz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fritz%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastien%2BFritz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Mario P. L. Calus </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics Centre, Wageningen University & Research, 6700 AH Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mario%2BP.%2BL.%2BCalus%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Calus%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMario%2BP.%2BL.%2BCalus%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3213-704X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mario P. L. Calus</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gwendal Restoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gwendal%2BRestoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Restoux%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGwendal%2BRestoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/1/113" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/1/113&amp;title=Which%20Individuals%20To%20Choose%20To%20Update%20the%20Reference%20Population%3F%20Minimizing%20the%20Loss%20of%20Genetic%20Diversity%20in%20Animal%20Genomic%20Selection%20Programs" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/1/113&amp;title=Which%20Individuals%20To%20Choose%20To%20Update%20the%20Reference%20Population%3F%20Minimizing%20the%20Loss%20of%20Genetic%20Diversity%20in%20Animal%20Genomic%20Selection%20Programs" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/1/113&amp;title=Which%20Individuals%20To%20Choose%20To%20Update%20the%20Reference%20Population%3F%20Minimizing%20the%20Loss%20of%20Genetic%20Diversity%20in%20Animal%20Genomic%20Selection%20Programs" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/1/113&amp;text=Which%20Individuals%20To%20Choose%20To%20Update%20the%20Reference%20Population%3F%20Minimizing%20the%20Loss%20of%20Genetic%20Diversity%20in%20Animal%20Genomic%20Selection%20Programs" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/1/113&amp;title=Which%20Individuals%20To%20Choose%20To%20Update%20the%20Reference%20Population%3F%20Minimizing%20the%20Loss%20of%20Genetic%20Diversity%20in%20Animal%20Genomic%20Selection%20Programs" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/1/113&amp;t=Which%20Individuals%20To%20Choose%20To%20Update%20the%20Reference%20Population%3F%20Minimizing%20the%20Loss%20of%20Genetic%20Diversity%20in%20Animal%20Genomic%20Selection%20Programs" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/1/113&amp;title=Which%20Individuals%20To%20Choose%20To%20Update%20the%20Reference%20Population%3F%20Minimizing%20the%20Loss%20of%20Genetic%20Diversity%20in%20Animal%20Genomic%20Selection%20Programs" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/1/113&amp;title=Which%20Individuals%20To%20Choose%20To%20Update%20the%20Reference%20Population%3F%20Minimizing%20the%20Loss%20of%20Genetic%20Diversity%20in%20Animal%20Genomic%20Selection%20Programs" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487828/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487828" id="node487828--3" data-pisa="ggg;8/1/113" data-pisa-master="ggg;g3.117.1117" data-apath="/ggg/8/1/113.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Which Individuals To Choose To Update the Reference Population? Minimizing the Loss of Genetic Diversity in Animal Genomic Selection Programs</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-8609-5869" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sonia E. Eynard</span>, <span class="highwire-citation-author" data-delta="1">Pascal Croiseau</span>, <span class="highwire-citation-author" data-delta="2">Denis Laloë</span>, <span class="highwire-citation-author" data-delta="3">Sebastien Fritz</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-3213-704X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mario P. L. Calus</span> and <span class="highwire-citation-author" data-delta="5">Gwendal Restoux</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">113-121;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.1117 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487828--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sonia E. Eynard </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics Centre, Wageningen University & Research, 6700 AH Wageningen, The Netherlands</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Centre for Genetic Resources, Wageningen University & Research, the Netherlands, 6700 AA Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sonia%2BE.%2BEynard%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eynard%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASonia%2BE.%2BEynard%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-8609-5869" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sonia E. Eynard</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:sonia.eynard@wur.nl" class="" data-icon-position="" data-hide-link-title="0">sonia.eynard@wur.nl</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Pascal Croiseau </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pascal%2BCroiseau%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Croiseau%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APascal%2BCroiseau%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Denis Laloë </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Denis%2BLalo%C3%AB%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lalo%C3%AB%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADenis%2BLalo%25C3%25AB%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sebastien Fritz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Allice, 75595 Paris Cedex 12, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sebastien%2BFritz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fritz%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASebastien%2BFritz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Mario P. L. Calus </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Animal Breeding and Genomics Centre, Wageningen University & Research, 6700 AH Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mario%2BP.%2BL.%2BCalus%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Calus%20MP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMario%2BP.%2BL.%2BCalus%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3213-704X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mario P. L. Calus</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Gwendal Restoux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Génétique Animale et Biologie Intégrative (GABI), Institut National de la Recherche Agronomique (INRA), AgroParisTech, Université Paris-Saclay, 78350 Jouy en Josas, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gwendal%2BRestoux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Restoux%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGwendal%2BRestoux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=113&amp;rft.epage=121&amp;rft.atitle=Which%20Individuals%20To%20Choose%20To%20Update%20the%20Reference%20Population%3F%20Minimizing%20the%20Loss%20of%20Genetic%20Diversity%20in%20Animal%20Genomic%20Selection%20Programs&amp;rft.volume=8&amp;rft.issue=1&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Eynard%2C+Sonia+E.&amp;rft.au=Croiseau%2C+Pascal&amp;rft.au=Lalo%C3%AB%2C+Denis&amp;rft.au=Fritz%2C+Sebastien&amp;rft.au=Calus%2C+Mario+P.+L.&amp;rft.au=Restoux%2C+Gwendal"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487828/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487828/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487828/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487828/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487828/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487828/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487828/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487828/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487828/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487828/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487828/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487828/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/1/113" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-19" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/1/113.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/1/113.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/1/113.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/1/113.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
