<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/1/79" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/1/79.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/1/79.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/1/79.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/1/79.atom" />
<meta name="HW.pisa" content="ggg;8/1/79" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Large-Scale Low-Cost NGS Library Preparation Using a Robust Tn5 Purification and Tagmentation Protocol" />
<meta name="DC.Identifier" content="10.1534/g3.117.300257" />
<meta name="DC.Date" content="2018-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Hennig et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Efficient preparation of high-quality sequencing libraries that well represent the biological sample is a key step for using next-generation sequencing in research. Tn5 enables fast, robust, and highly efficient processing of limited input material while scaling to the parallel processing of hundreds of samples. Here, we present a robust Tn5 transposase purification strategy based on an N-terminal His6-Sumo3 tag. We demonstrate that libraries prepared with our in-house Tn5 are of the same quality as those processed with a commercially available kit (Nextera XT), while they dramatically reduce the cost of large-scale experiments. We introduce improved purification strategies for two versions of the Tn5 enzyme. The first version carries the previously reported point mutations E54K and L372P, and stably produces libraries of constant fragment size distribution, even if the Tn5-to-input molecule ratio varies. The second Tn5 construct carries an additional point mutation (R27S) in the DNA-binding domain. This construct allows for adjustment of the fragment size distribution based on enzyme concentration during tagmentation, a feature that opens new opportunities for use of Tn5 in customized experimental designs. We demonstrate the versatility of our Tn5 enzymes in different experimental settings, including a novel single-cell polyadenylation site mapping protocol as well as ultralow input DNA sequencing." />
<meta name="DC.Contributor" content="Bianca P. Hennig" />
<meta name="DC.Contributor" content="Lars Velten" />
<meta name="DC.Contributor" content="Ines Racke" />
<meta name="DC.Contributor" content="Chelsea Szu Tu" />
<meta name="DC.Contributor" content="Matthias Thoms" />
<meta name="DC.Contributor" content="Vladimir Rybin" />
<meta name="DC.Contributor" content="Hüseyin Besir" />
<meta name="DC.Contributor" content="Kim Remans" />
<meta name="DC.Contributor" content="Lars M. Steinmetz" />
<meta name="article:published_time" content="2018-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Large-Scale Low-Cost NGS Library Preparation Using a Robust Tn5 Purification and Tagmentation Protocol" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Efficient preparation of high-quality sequencing libraries that well represent the biological sample is a key step for using next-generation sequencing in research. Tn5 enables fast, robust, and highly efficient processing of limited input material while scaling to the parallel processing of hundreds of samples. Here, we present a robust Tn5 transposase purification strategy based on an N-terminal His&lt;sub&gt;6&lt;/sub&gt;-Sumo3 tag. We demonstrate that libraries prepared with our in-house Tn5 are of the same quality as those processed with a commercially available kit (Nextera XT), while they dramatically reduce the cost of large-scale experiments. We introduce improved purification strategies for two versions of the Tn5 enzyme. The first version carries the previously reported point mutations E54K and L372P, and stably produces libraries of constant fragment size distribution, even if the Tn5-to-input molecule ratio varies. The second Tn5 construct carries an additional point mutation (R27S) in the DNA-binding domain. This construct allows for adjustment of the fragment size distribution based on enzyme concentration during tagmentation, a feature that opens new opportunities for use of Tn5 in customized experimental designs. We demonstrate the versatility of our Tn5 enzymes in different experimental settings, including a novel single-cell polyadenylation site mapping protocol as well as ultralow input DNA sequencing.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="ggg;8/1/79" />
<meta name="citation_id" content="8/1/79" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/1/79" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/1/79.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/1/79.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/1/79.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300257" />
<meta name="citation_pmid" content="29118030" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="79" />
<meta name="citation_lastpage" content="89" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Bianca P. Hennig" />
<meta name="citation_author_institution" content="Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg 69117, Germany" />
<meta name="citation_author" content="Lars Velten" />
<meta name="citation_author_institution" content="Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg 69117, Germany" />
<meta name="citation_author" content="Ines Racke" />
<meta name="citation_author_institution" content="Protein Expression and Purification Core Facility, European Molecular Biology Laboratory, Heidelberg 69117, Germany" />
<meta name="citation_author" content="Chelsea Szu Tu" />
<meta name="citation_author_institution" content="Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg 69117, Germany" />
<meta name="citation_author" content="Matthias Thoms" />
<meta name="citation_author_institution" content="Biochemistry Center Heidelberg, Heidelberg University, 69120, Germany" />
<meta name="citation_author" content="Vladimir Rybin" />
<meta name="citation_author_institution" content="Protein Expression and Purification Core Facility, European Molecular Biology Laboratory, Heidelberg 69117, Germany" />
<meta name="citation_author" content="Hüseyin Besir" />
<meta name="citation_author_institution" content="Protein Expression and Purification Core Facility, European Molecular Biology Laboratory, Heidelberg 69117, Germany" />
<meta name="citation_author" content="Kim Remans" />
<meta name="citation_author_institution" content="Protein Expression and Purification Core Facility, European Molecular Biology Laboratory, Heidelberg 69117, Germany" />
<meta name="citation_author_email" content="kim.remans@embl.de" />
<meta name="citation_author_email" content="lars.steinmetz@embl.de" />
<meta name="citation_author" content="Lars M. Steinmetz" />
<meta name="citation_author_institution" content="Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg 69117, Germany" />
<meta name="citation_author_institution" content="Department of Genetics, Stanford University School of Medicine, California 94305" />
<meta name="citation_author_institution" content="Stanford Genome Technology Center, Stanford University School of Medicine, Palo Alto, California 94304" />
<meta name="citation_author_email" content="kim.remans@embl.de" />
<meta name="citation_author_email" content="lars.steinmetz@embl.de" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. Adey;citation_author=H. G. Morrison;citation_author=X. Asan;citation_author=J. O. Xun;citation_author=. Kitzman;citation_title=Rapid, low-input, low-bias construction of shotgun fragment libraries by high-density in vitro transposition.;citation_pages=R119-R119;citation_volume=11;citation_year=2010;citation_issue=12;citation_pmid=21143862;citation_doi=10.1186/gb-2010-11-12-r119" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Anders;citation_author=P. T. Pyl;citation_author=W. Huber;citation_title=HTSeq–a Python framework to work with high-throughput sequencing data.;citation_volume=31;citation_year=2015;citation_pmid=25260700;citation_doi=10.1093/bioinformatics/btu638" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=E. A. Ashley;citation_title=Towards precision medicine.;citation_pages=507-522;citation_volume=17;citation_year=2016;citation_pmid=27528417;citation_doi=10.1038/nrg.2016.86" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=J. D. Buenrostro;citation_author=B. Wu;citation_author=U. M. Litzenburger;citation_author=D. Ruff;citation_author=M. L. Gonzales;citation_title=Single-cell chromatin accessibility reveals principles of regulatory variation.;citation_pages=486-490;citation_volume=523;citation_year=2015;citation_pmid=26083756;citation_doi=10.1038/nature14590" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. R. Davies;citation_author=I. Y. Goryshin;citation_author=W. S. Reznikoff;citation_author=I. Rayment;citation_title=Three-Dimensional Structure of the Tn5 Synaptic Complex Transposition Intermediate;citation_pages=77-85;citation_volume=289;citation_year=2000;citation_issue=5476;citation_pmid=10884228;citation_doi=10.1126/science.289.5476.77" />
<meta name="citation_reference" content="citation_journal_title=Pharmacogenomics;citation_author=E. D. Esplin;citation_author=L. Oei;citation_author=M. P. Snyder;citation_title=Personalized sequencing and the future of medicine: discovery, diagnosis and defeat of disease.;citation_pages=1771-1790;citation_volume=15;citation_year=2014;citation_pmid=25493570" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. Giladi;citation_author=I. Amit;citation_title=Immunology, one cell at a time.;citation_pages=27-29;citation_volume=547;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=I. Y. Goryshin;citation_author=W. S. Reznikoff;citation_title=Tn5 in Vitro Transposition;citation_pages=7367-7374;citation_volume=273;citation_year=1998;citation_issue=13;citation_pmid=9516433;citation_doi=10.1074/jbc.273.13.7367" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=I. Y. Goryshin;citation_author=J. A. Miller;citation_author=Y. V. Kil;citation_author=V. A. Lanzov;citation_author=W. S. Reznikoff;citation_title=Tn5/IS50 target recognition;citation_pages=10716-10721;citation_volume=95;citation_year=1998;citation_issue=18;citation_pmid=9724770;citation_doi=10.1073/pnas.95.18.10716" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=D. Grun;citation_author=A. Lyubimova;citation_author=L. Kester;citation_author=K. Wiebrands;citation_author=O. Basak;citation_title=Single-cell messenger RNA sequencing reveals rare intestinal cell types.;citation_pages=251-255;citation_volume=525;citation_year=2015;citation_pmid=26287467;citation_doi=10.1038/nature14966" />
<meta name="citation_reference" content="citation_journal_title=bioRxiv;citation_title=On the widespread and critical impact of systematic bias and batch effects in single-cell RNA-Seq data.;citation_author=S. C. Hicks;citation_author=M. Teng;citation_author=R. A. Irizarry;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. A. Jaitin;citation_author=E. Kenigsberg;citation_author=H. Keren-Shaul;citation_author=N. Elefant;citation_author=F. Paul;citation_title=Massively Parallel Single-Cell RNA-Seq for Marker-Free Decomposition of Tissues into Cell Types;citation_pages=776-779;citation_volume=343;citation_year=2014;citation_issue=6172;citation_pmid=24531970;citation_doi=10.1126/science.1247651" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. C. Johnson;citation_author=W. S. Reznikoff;citation_title=DNA sequences at the ends of transposon Tn5 required for transposition.;citation_pages=280-282;citation_volume=304;citation_year=1983;citation_issue=5923;citation_pmid=6306482;citation_doi=10.1038/304280a0" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=E. Z. Macosko;citation_author=A. Basu;citation_author=R. Satija;citation_author=J. Nemesh;citation_author=K. Shekhar;citation_title=Highly parallel genome-wide expression profiling of individual cells using nanoliter droplets.;citation_pages=1202-1214;citation_volume=161;citation_year=2015;citation_pmid=26000488;citation_doi=10.1016/j.cell.2015.05.002" />
<meta name="citation_reference" content="citation_journal_title=BMC Microbiology;citation_journal_abbrev=BMC Microbiology;citation_author=E. Martínez-García;citation_author=B. Calles;citation_author=M. Arévalo-Rodríguez;citation_author=V. de Lorenzo;citation_title=pBAM1: an all-synthetic genetic tool for analysis and construction of complex bacterial phenotypes.;citation_pages=38-38;citation_volume=11;citation_year=2011;citation_pmid=21342504;citation_doi=10.1186/1471-2180-11-38" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=T. Nagano;citation_author=Y. Lubling;citation_author=T. J. Stevens;citation_author=S. Schoenfelder;citation_author=E. Yaffe;citation_title=Single-cell Hi-C reveals cell-to-cell variability in chromosome structure.;citation_pages=59-64;citation_volume=502;citation_year=2013;citation_pmid=24067610;citation_doi=10.1038/nature12593" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=F. Paul;citation_author=Y. Arkin;citation_author=A. Giladi;citation_author=D. A. Jaitin;citation_author=E. Kenigsberg;citation_title=Transcriptional heterogeneity and lineage commitment in myeloid progenitors.;citation_pages=1663-1677;citation_volume=163;citation_year=2015;citation_pmid=26627738;citation_doi=10.1016/j.cell.2015.11.013" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=S. Picelli;citation_author=A. K. Bjorklund;citation_author=O. R. Faridani;citation_author=S. Sagasser;citation_author=G. Winberg;citation_title=Smart-seq2 for sensitive full-length transcriptome profiling in single cells.;citation_pages=1096-1098;citation_volume=10;citation_year=2013;citation_pmid=24056875;citation_doi=10.1038/nmeth.2639" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Picelli;citation_author=A. K. Björklund;citation_author=B. Reinius;citation_author=S. Sagasser;citation_author=G. Winberg;citation_title=Tn5 transposase and tagmentation procedures for massively scaled sequencing projects.;citation_volume=24;citation_year=2014;citation_pmid=25079858;citation_doi=10.1101/gr.177881.114" />
<meta name="citation_reference" content="citation_journal_title=Nat. Protoc.;citation_author=S. Picelli;citation_author=O. R. Faridani;citation_author=A. K. Björklund;citation_author=G. Winberg;citation_author=S. Sagasser;citation_title=Full-length RNA-seq from single cells using Smart-seq2.;citation_pages=171-181;citation_volume=9;citation_year=2014;citation_pmid=24385147;citation_doi=10.1038/nprot.2014.006" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=D. Ramskold;citation_author=S. Luo;citation_author=Y. C. Wang;citation_author=R. Li;citation_author=Q. Deng;citation_title=Full-length mRNA-Seq from single-cell levels of RNA and individual circulating tumor cells.;citation_pages=777-782;citation_volume=30;citation_year=2012;citation_issue=8;citation_pmid=22820318;citation_doi=10.1038/nbt.2282" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=W. S. Reznikoff;citation_title=Transposon Tn5.;citation_pages=269-286;citation_volume=42;citation_year=2008;citation_pmid=18680433;citation_doi=10.1146/annurev.genet.42.110807.091656" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=C. Sasakawa;citation_author=G. F. Carle;citation_author=D. E. Berg;citation_title=Sequences essential for transposition at the termini of IS50.;citation_pages=7293-7297;citation_volume=80;citation_year=1983;citation_issue=23;citation_pmid=6316355;citation_doi=10.1073/pnas.80.23.7293" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=Y. Shevchenko;citation_author=G. G. Bouffard;citation_author=Y. S. N. Butterfield;citation_author=R. W. Blakesley;citation_author=J. L. Hartley;citation_title=Systematic sequencing of cDNA clones using the transposon Tn5;citation_pages=2469-2477;citation_volume=30;citation_year=2002;citation_issue=11;citation_pmid=12034835;citation_doi=10.1093/nar/30.11.2469" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=L. Velten;citation_author=S. Anders;citation_author=A. Pekowska;citation_author=A. I. Jarvelin;citation_author=W. Huber;citation_title=Single-cell polyadenylation site mapping reveals 3' isoform choice variability;citation_pages=812-812;citation_volume=11;citation_year=2015;citation_issue=6;citation_pmid=26040288;citation_doi=10.15252/msb.20156198" />
<meta name="citation_reference" content="citation_journal_title=Nat. Cell Biol.;citation_author=L. Velten;citation_author=S. F. Haas;citation_author=S. Raffel;citation_author=S. Blaszkiewicz;citation_author=S. Islam;citation_title=Human haematopoietic stem cell lineage commitment is a continuous process.;citation_pages=271-281;citation_volume=19;citation_year=2017;citation_doi=10.1038/ncb3493" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=M. D. Weinreich;citation_author=A. Gasch;citation_author=W. S. Reznikoff;citation_title=Evidence that the cis preference of the Tn5 transposase is caused by nonproductive multimerization.;citation_pages=2363-2374;citation_volume=8;citation_year=1994;citation_issue=19;citation_pmid=7958902;citation_doi=10.1101/gad.8.19.2363" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=S. Nacu;citation_title=Fast and SNP-tolerant detection of complex variants and splicing in short reads.;citation_volume=26;citation_year=2010;citation_pmid=20147302;citation_doi=10.1093/bioinformatics/btq057" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. Zhou;citation_author=W. S. Reznikoff;citation_title=Tn5 transposase mutants that alter DNA binding specificity.;citation_pages=362-373;citation_volume=271;citation_year=1997;citation_issue=3;citation_pmid=9268665;citation_doi=10.1006/jmbi.1997.1188" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. Zhou;citation_author=A. Bhasin;citation_author=W. S. Reznikoff;citation_title=Molecular genetic analysis of transposase-end DNA sequence recognition: cooperativity of three adjacent base-pairs in specific interaction with a mutant Tn5 transposase.;citation_pages=913-925;citation_volume=276;citation_year=1998;citation_issue=5;citation_pmid=9566196;citation_doi=10.1006/jmbi.1997.1579" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Large-Scale Low-Cost NGS Library Preparation Using a Robust Tn5 Purification and Tagmentation Protocol" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="twitter:description" content="Efficient preparation of high-quality sequencing libraries that well represent the biological sample is a key step for using next-generation sequencing in research. Tn5 enables fast, robust, and highly efficient processing of limited input material while scaling to the parallel processing of hundreds of samples. Here, we present a robust Tn5 transposase purification strategy based on an N-terminal His6-Sumo3 tag. We demonstrate that libraries prepared with our in-house Tn5 are of the same quality as those processed with a commercially available kit (Nextera XT), while they dramatically reduce the cost of large-scale experiments. We introduce improved purification strategies for two versions of the Tn5 enzyme. The first version carries the previously reported point mutations E54K and L372P, and stably produces libraries of constant fragment size distribution, even if the Tn5-to-input molecule ratio varies. The second Tn5 construct carries an additional point mutation (R27S) in the DNA-binding domain. This construct allows for adjustment of the fragment size distribution based on enzyme concentration during tagmentation, a feature that opens new opportunities for use of Tn5 in customized experimental designs. We demonstrate the versatility of our Tn5 enzymes in different experimental settings, including a novel single-cell polyadenylation site mapping protocol as well as ultralow input DNA sequencing." />
<meta name="og-title" property="og:title" content="Large-Scale Low-Cost NGS Library Preparation Using a Robust Tn5 Purification and Tagmentation Protocol" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/1/79" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Efficient preparation of high-quality sequencing libraries that well represent the biological sample is a key step for using next-generation sequencing in research. Tn5 enables fast, robust, and highly efficient processing of limited input material while scaling to the parallel processing of hundreds of samples. Here, we present a robust Tn5 transposase purification strategy based on an N-terminal His6-Sumo3 tag. We demonstrate that libraries prepared with our in-house Tn5 are of the same quality as those processed with a commercially available kit (Nextera XT), while they dramatically reduce the cost of large-scale experiments. We introduce improved purification strategies for two versions of the Tn5 enzyme. The first version carries the previously reported point mutations E54K and L372P, and stably produces libraries of constant fragment size distribution, even if the Tn5-to-input molecule ratio varies. The second Tn5 construct carries an additional point mutation (R27S) in the DNA-binding domain. This construct allows for adjustment of the fragment size distribution based on enzyme concentration during tagmentation, a feature that opens new opportunities for use of Tn5 in customized experimental designs. We demonstrate the versatility of our Tn5 enzymes in different experimental settings, including a novel single-cell polyadenylation site mapping protocol as well as ultralow input DNA sequencing." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
    <title>Large-Scale Low-Cost NGS Library Preparation Using a Robust Tn5 Purification and Tagmentation Protocol | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__MRoKQICmX-Ek5ZIyBlwXzUtbKmCmIVqNnsCLjO1o65o__abNV7fJaqFhsjvAynloQDGCCstgzs17fQnVdDEt9-NA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"aqNzJ4LtjIUVOjed9dTeMYzQue__I-bkrXbWIuqfos8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487813","apath":"\/ggg\/8\/1\/79.atom","pisa":"ggg;8\/1\/79","ac":{"\/ggg\/8\/1\/79.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/1\/79.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/1\/79"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/1\/79"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/1\/79.supplemental":true,"\/content\/8\/1\/79":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022fKN9bi5wvpkeqDIOlgMpVj1Q0I0EpRiFnSfUn8V90fpGCs4qhUjQgwOF5E75qsW2FKiZe2jiZuPR7z\\\\\\\/fKAZ\\\\\\\/d9VfpLKEctdTkoCzv7DZJb9CJWGOxKHcS8SEJDwlEBp3nzXfwcol5Urw\\\\\\\/Lw94dUmyM+NL5ugwXZVOOt2zFlAstqrTTht38r6YOasSLUqmoPHAfHk4ydAvptd+yorPDVQfdMuSUuySLkPR3BAHIOtKCYyG9KC8OmJbtKqhZoI\\\\\\\/BEMaS+wrTeThY212VNzqEBoD64byQVz8fKqlS4wNdf0v0Kiofuy49sERdtbObOPEb5WsHFdMbIzozvrFj4cceRI0m8ehZKQ1PWGGx11jha75cR4\\\\\\\/+y8bGOrmWBnFpQU8BUXlIC6pJW0ZqSMWLKe+hWcnNjhL30hKNlfx\\\\\\\/xmxN6EJW253RTCVlLiEBbHY++Xu3d6+2M5X2P3rFlVMbJJSwfn4sYoi\\\\\\\/mbZHVGoMDI7dcC\\\\\\\/aIc6Yhz10Uuu7571tQexLB+ZarIy\\\\\\\/4Pvu9dwxCgMQ\\\\\\\/VNDkDhRyfIWbe74qF46byKITgm4roQh8Uw24kgj6yGmngfAOQ95Zy6EEp2b7OW1Zp1WwR1ZmaT6BNiyseHH3hJybTCnC8p0p+BH1xfaTJOPdecwea\\\\\\\/rNoF2wn+O\\\\\\\/vkw+440cZZSe065WQTaNgxXBAJFA6779hzCuuNf9Iyio2GL56\\\\\\\/kprpbJkPP90O+eRAmmM\\\\\\\/QFc1l9Jr2iE\\\\\\\/g4TJd3KU23CNBbPsY25UWTaMv0fiCfwhrrKgrEjy6PIiewg26ZpTAr5nTI209Xo2xRnld\\\\\\\/tZuViUW1VqkeHslnuUcq2+Xf6eWQyXrVJPU\\\\\\\/XVIPVIg1OWRhO2WIzV5a6hkvf6MK5Fch7JJVr+KJlbpK\\\\\\\/Q0dCqAJUZtT3ZnJuglk8DbWgEEDUywZFzWZ0Souer2xl5CRhMGFzzheqAf\\\\\\\/dIq6ALG37rViKC5hhhz90VWb8hAVjXVaKwawaO\\\\\\\/h\\\\\\\/wN1ezsegV2uNf1mGjIp3uiMwUl4Nvcw5hlj2XmGIzCFVX8j2Q3wwX2okVHSqHSMqysyPVNJBL+Me0ENJioJy3KcI2UFjIpYnb4qjvXs5EMP65BEMzVwhgxOBY3ZHQIVLYtZZp1SDP0qe53ODRJMKV7GbmRKHFCqBsRSAfuEQFnPWkbFUb3Hwpzinldnqse4JOu0HTLYaDMCVPBivs+4rTviZD47Q0N+nQZJF3WZUaR3GaZO15s6BbX6K8vL4oXwBrNdPIIsPvfXu\\\\\\\/NQ1dVH4wlGQ4prDpPQtO6IU\\\u0022,\\\u0022iv\\\u0022:\\\u0022FodrccU\\\\\\\/ffcpqN+W4H2FwA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225b6b7c96895f50c3e38da5bf902cb9e2749fb15f562440abd67f50e7796ee023\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022N0ufGQtnzK4Kx04t2WY38Qy7YhOn1ljgK9h1GO591w0ar0rXhiAmXR2rWqmssmvw\\\\\\\/lQRBgJP1+4NEriYj+vAkRQjabBFqmLsTsNQACr8fcZ2Tbl9OAcZtFaZgHlqNcRd\\\\\\\/5geM9A+Is6bf+ToKPEPoSVvDQxq5o2No\\\\\\\/ClN7mHrXAWlS2NeBaT5UcbDOCSv4kNOa5TttoFdSsu7aBUWFmhSiJjzqfGxfgyMc7bwkMnvN4GvwoHGEyV3\\\\\\\/2HQmV+yRH14EUDNk7Jgtu6Z0RTSj09GLi+VAhHy22cRyd8Wa6stMHvhhe2QyTMaV0DHosFxcSM4DJCPH13ULsFFAmPolbTKf69fXxRpQt9hVhTAyQLrZ0wTZ38SbCEVbBqZP1PlVXHzzxULLs+ybq9X4jEFt9CjMV0wM+8Q3pilpcPid3T2VtrZA8Cz1jRojbmkgKV3UyVzah2haOh+LgE29qHyPfc1Ho37aDtubq5qk4mD++ZGRaZp9PVEnr5u+RnibmD9lmLrKcXzhQWL9VdOV2X8SDPRggRaY55aYzkVoQ6MtCoxjuAzjOckpBXdnCl24Gs8oL7DQvYYASKQF8RlGTc01pdu4BOoFRdvd76raK8r1nDo7eeEqUGWxttr2QpsGnzkAsvjYxEJ2r8VXT7OxZy6Mj4nDjm12aWEBQinGv+LsoQHSLxC3csXL2d6+6l+oXy\\\\\\\/jfdcVEgpTOHrz2nJmlHu6Ii8ZpqP10eMz5Hr2Al2Qyo\\\\\\\/6kGFeEyo0+4b3XTAKRCkMoQFtmquDvxuNvHuBdAndQ8Vpv3RbLEoPKHVpKJ\\\\\\\/cKVm3XK63\\\\\\\/NaD1QT\\\\\\\/qOYb9N5nYJK\\\\\\\/N+GB1A8cR4n\\\\\\\/2W3L3kDi\\\\\\\/45Acg6oLLGX8swQmCKZxvG\\\\\\\/6EITVR5QCUs6tnkH2kAjj23T9Zajlfii\\\\\\\/5JgsxJBzQlq9DA+juFzMkqjckNwl7W5Uf7A62fhelpF0xeATmU8NHHwK49csENiGw89u5am4yFtJURgyjYWT+ZKyi+2AKCDInNo0R7bbo95KNGN5KIqgPiyAHR9+5vmoEaTzvVOBdTImwVIX2oMIqqI\\\\\\\/nZ3n9Go8OynZR25jhf3uufTIZQjzIAbg3WnbSThK15yMq9p3f1VBfZPaENVEka8N3O6KUDJVu0LdpoHkA3BIvfsAX9NOltpwVMB1fRIqlqsGePbIGNlP0XI9BTWhmYVTDVvA0q3Dg5KhX0Il8naUhn0dc8pE6Hf2XHX053cq5eJ0xOIZoCuMINDH84Pfla62EKODxE3TtfzqbOIeZu6C5nDeCAKm\\\\\\\/0e82OSSKOMAjdlCgTdSwKflzxgWVGB8d3le8H31MPqu3nEiBWXGcF\\\\\\\/lpedgwqnCXaz28ORsQEDBz2KuCcYe9kDTJ6j0IHlinYtNNO3I4SfexPneqX0y8rpI5j0I+vNRiFt8sD\\\\\\\/JzTGbUdLUFOIEwu0JXvnUmyp63iHbfTR34oD3FXlsZeE9OB3fVkNKlm9jS6I6OcjXGZquREuNzd0m0vo6hwveFOqkvOpn\\\\\\\/4oJWyWJTLh7Qz\\\\\\\/6ioPY5pFnXieBH4NCMmWzpOOVU6f9++IGGvfH3ZPS1waDtpOtqFpp9KLftfjMDzHkhWzPlfokJapi6T4Olu74PL7YScs50G4nHnDVu0ip4jFkwjuk3bOxjcaLDLDOJAdFEwTKihMPwoyPxRoYLiZrSyuMMMq6PZ9ZJhXXaVgLDPONmC+e++0bPOeC0G65UWpNhzdy1DndhKjlm6KqVZ\\\\\\\/mqTlR7UQHkZTrxxx2DckOG8ozgtbWKxYT\\\\\\\/6SQO\\\\\\\/J7TCEZSKsLptIEfHC2rALzJ6QC3OhB5mLmIkPdSQyWFMG7DjPOl9L\\\\\\\/V9GTUhUWI\\\\\\\/wbbJaWXIKolPjuZTKcXSLw4zbFIuYEXvIelzYPILKNPA3f8vN9QkPMavpYOVprVzjni0uT0DoB83TppaapVM+V439VCury5WN2sHCyprYIQcovNJZkAIjSyuj14TWN1+Qjo4015UBoJhZNMchyc\\\\\\\/P2nZbdNK+XdAUkIziwBzfjWQMEGyjjtUBGnZPbhRSlW4AiB34AgR6rZkQ5M1rus59j1RacmPhOYcFKqtl\\\\\\\/7JAJralkq\\\\\\\/haUxpPIonEY58p0Aa8Tk8h1szZfo3Ez0IQs\\\\\\\/5ahvfvrm42OwCLf0Gr1fAlnSH\\\\\\\/lnJtSq7NgKYhTaijN7p1C5tAHjm4JYlZzkoec0mSQCVIPaN+bI+0LOXWSzYeK7pACkr6q4ZtrINgiY3Mu\\\\\\\/zpsD\\\\\\\/rFalfR3CaUDck3Wi7nFwp0fXPnYUQ7JMix1G9++KLCsZQwGV8fRuc8Mm4QuF+BS9yY7fxT0tMU+1Yd7NYeqSa1KY7bnBX+jH15ZHAa8\\\\\\\/x7x40GVOGw6y6fsmGSqlF\\\\\\\/h\\\\\\\/zfEFOv2IeY5VpOLtflJ0wUU\\\\\\\/2Xx8\\\\\\\/K0WBlH7fJ9Jk21QNHXvBeGjU67wIepxspff5Ejax7P8r6uMOTaH4o1S+BMhnlHaT6YZAE2s+61jwvOqnZ\\\\\\\/Im5ek\\\\\\\/FPgzwiDBEa4pxxkV3E\\\\\\\/YD6Sjxz8O5OvYzDhVwIjhnJt5\\\\\\\/3P9AUhHmRFwwCuE+Bl5yNc66\\\\\\\/CXBqzy+OHyoY3VjiCcFr\\\\\\\/VY24PnOuSNiWxAmSH1yQCzCZnQDL4e\\\\\\\/9ZLVgbdGhUMy0yCedu8D1Wr4+TNck8jRmYXr1gVBauU5dlcqcXcF8zwQ7M+uUx3ztFcxp3ghFFmns6rYq1v1dE5bWY0oA88NJRMNs4niQ5bikBkHf9zFTO3v5rf1gCQY7iBqa3oWDtafLDxEDPo8OZw8DUfkVPiMAreIEweFOOoXhbi72HudbIp9V2081JL9QHyam\\\\\\\/ToyxWEmX8ki3pQLnfpt1wovPPT1z5BnZBwe5l8eVppzaTdQhEUpQPKQYyZGE3frxUW\\\\\\\/3KAT0xrrOOOUb0wgD8CJXuCMyFZfigE6M0u0ZrazpS3w+RJFz\\\\\\\/Db2WuPhx1cX4eVCFxvJIXE34TL5EDKDbafKjHfT9Z1HnOpnoeC9s+fyus6Kafc2VkjpgA3wcZqXdgokvmKGOMOI\\\\\\\/MFvSpf51NBlHvcHwpNw2La8LKGpWs4OD6fa5chl55CWywX8dQL9I5oU6IiGrnoGjqamEdiJ2eLxiCSJwPeFBRnUWMb9n8CXDFw0\\\\\\\/i8oWTcFjs\\\\\\\/S\\\\\\\/gNxYD60AoRM0FpNGd7FZiQ9zEc+VhYj9ffIauPVzB3fTR0IxaAo\\\\\\\/QTJHIlyJ2oIM6YY1vDj6dB7lBpMhWZO6b77dpt15slj9M36X+s6+Ujs3Xi\\\\\\\/xrtDb\\\\\\\/H+kinSt4d2ScyobgsLNh7ifqPqwLsSgymSGKSN3lLuco1ADhBRRWCppBhFNAjq\\\\\\\/dwLQFY2qYv2MzV8JWeZyVjhzlLt6S+DYDFQ2A5HgUagMov11TPy0YY17I+LIQJRfNluW\\\\\\\/PHPGFhXtW+aVWbRPPZ5kVz4n5nj3wHRVQhHgt5So0ohnMIfPlMqpnWXrKx5ZIsHAydRCD6QiH1FG1CzAR8AAX8ev0bpFS524fz5EHpyEK+CwNyCg9VoicudePGnGrHI6kMTs1kgX416G3jyeEiuM1fFG2WWeLn2fwGmg4\\\\\\\/7L36bJhJ\\\\\\\/zoyxOBm3AXVcLfuuMIPW4SVNFcsktw0W0sdGAcM52ROOQ8N\\\\\\\/JjceoKWUFy2+AjE8cgu\\\\\\\/J\\\\\\\/bJ7el7SqBDiLkTvxJMFI+SOmO4Y3yjmKuHU5hgaLZnZNxztJJ8m53aAwyEBxH3oAGwzg65NpwnB4rYYDJq+bqecZu+QH7LnufZwDhdmgGHPC3VSeuuCXAVSBfhMKIKcjQDHPMd5vLFTB3KJsRpV2KrZ33mCT3h66gumoX\\\\\\\/BxcJQjTOjAqFMelhNrECmUkXXD1GLhyHl9nGfSbC664A2pVXGu6ZQtNd2BW7tsWAuSdrSHPO524Ef+Sby7V+IRem7vjK7aG1ePcD7PLMUTUBV81ZURRVyeGE7saqiEfuLPF3\\\\\\\/9\\\\\\\/MHb8hYWQodeoOo\\\\\\\/mm\\\\\\\/GXs86qLU6v0Lc7mHQTwBP0gH\\\\\\\/HPN5XhDOwn\\\\\\\/0k0Y6VwUQG3TdneIdXUpbVSUQtSbpst7RQzsM+TdO4iY+Iy3WGJLXRwUStv1vQTE4uO65Cf3QohSc45d4hRbnsLDJxUXSOYTW7oBHpSnOMQNp\\\\\\\/YD\\\\\\\/uLwiI+p3ZO32xAyAzYy9tKEAkbKUSQ5kGq8UY9l1jPoj+2d\\\\\\\/Ag2LbrTFpQQ8HodVg1PNn\\\\\\\/IeAlZ6IGYwuM97fqcx9s2\\\\\\\/6b1+sIvJd9aGGtfIWK2bVJrf\\\\\\\/30eMrY2TuzpPuBOQyf\\\\\\\/jk99X7GPig\\\\\\\/d+XbFUZ+cJ3ENKcXRBWRXLG0pbyuLajM5C\\\\\\\/\\\\\\\/7aR\\\\\\\/Nkm32xcbpKQoricm7II96vIfJcDMNSbL0EuAIdRbjbGzQxnEYiX\\\\\\\/eLwPE0SW9dmXPtpAvbB6TnKYYt1mfZw0xkCnZgCLxaVjM26wB2TwB6lgQCeTGMUR5a40+GMPB5HCfGk0sOO\\\\\\\/d7sZ0+Hvx6WzbG9nNO1ThgO0RnBRduoEqQZWd+l7XYEbRuPnh61q85bYQDLHa5OIl9BYWK0MSuO5c4o5YWs8FzM7HWgBGn5vHYQNegegYqH4ezE7yYbpafQCwHJY9ArM9+2UYeRuCI4ipamX31\\\\\\\/KUfztgd+B1AdH0o1AgK0fpcOwQSdBumhZaQJu9et1fhUc369Oj6bwIiB0l7BXefmQeGKknXNiFWXsw\\\\\\\/eGuXufzR50JE3pKI4K3OZx80Yd2iVqE2k9qwxRKkhEsimRcTQD\\\\\\\/JjFHMZKpNzdtZgSu6yDX3jL96ebilZ9+\\\\\\\/oYyY3rJLYDy9vCwZRH5gXJfrMfsGvj5Bu\\\\\\\/Op4ir7klZqf0IrstjN1O51EBkOraeM85Nqoa9+rJFMLl0d7HKCXMgrEOWhKfOXlTyc+mhhIP5QaKe7lk1NIG4v2bQr1y9jv3TC1BQJfwIaGzpJyj+r9T8duYSN3xTeZ5NcW39O42Y60f1QL233vQ+TGWgPLsmmjtFchIvKFhA9qV+wixwfFp2UO0+FtXMkCPDAsuhG88dZ0Y1VXcJ23SuJSXuABu6BBn9x6Mf2DVqmU0W\\\\\\\/0hh61r5CWRKbdI8Chm7T08iAz9gztgJgZVTayZkbOha3zZ+dV28l6MZX9faEA4YDjQ6rNRJKjiK0T7EsMGqfxcMX9I4HtDed3cQ0jD7Bk1f4UEKENilryS5U0AgM06cp\\\\\\\/8AHZ0VIeSjKYfFHLFUA5jEjLQM\\\\\\\/wED8h\\\\\\\/i9W4Eokc8emvPa1fUuyfwUahb\\\\\\\/dgyZzIgpwyeO0sglYLYoz2TDukVxwpPoW\\\\\\\/J\\\\\\\/kAMQKDqSjUuC0EI90i6giZb0aHVlLDjsrGLX3LwFiPe4YpNERVKEQDQ6+u7BVYDbmGAniySaEIQbSIhiKS42i49OECjz5DbQhzBerKDtM6MAROnd7WhvAmePTajBH5x\\\\\\\/t5ILS8IPGF1IYb6iARXg0QeorgCrO9\\\\\\\/5KNem\\\\\\\/1EqXbvBKepSYCR3qAcGirHKr2CDfwHt3Ueb5cVUSZ1eAFbBvhaNuoOhHym1Aboe7piJB7E2fOnP4tKdt3ikq166+\\\\\\\/9DVkCa4vqyE3q2\\\\\\\/ybd1B0zMOtqjrNZ89lAy8Tfyt0DLOfD0oSGR+YVYNH0m2U3Yr3AB5MEHoCbXfDvLFc51HF3ULvstt3SYKkJpgmojqrb\\\\\\\/1BvBkBSb\\\\\\\/y31\\\\\\\/bQMyVTHTmXAHzqZI\\\\\\\/DYBOVc+A4W1cMTML1RAkJxatMAsLOOt6hDJg5pOZd1X2P5FMBNqqvM7dIJtN2GJ3t5Hk6AMKM9QZJO2WLleiZ1vB4Mv2BZix5BHzVhf0kcNPTC6CxJKhj7NnFaRO7UfvUxU5D8plXm90ynTUftKVSVZ6T1C\\\\\\\/cRD2Z\\\\\\\/5K6htg6MdG9LfvMULNES\\\\\\\/8x6V1ehUCM7Vu0vQC6NS8iZuuzhAucRHwn3moZkEqeANN2ApgiTeXQ4QIjoXAbXV8yYOqPydRoG92cn5R2MDLvKVsqmr2h6opFG0\\\\\\\/9hepPKlgvxWGXT\\\\\\\/GU8bt+gf9ZSB\\\\\\\/UYpEPuUW\\\\\\\/e2VfKwOxqIRLuLfnA4DuyzFQYwIFCe2x6ZYep1DTvOAiRBdoZhZhkgRrj\\\\\\\/+\\\\\\\/ePcE9xH1ScUCpolTUxnLwMwntwVDJDWa5WNp+KeSOK+2oBJmJFeByRi6rpriB8LUPJVel64x9x1j04rC0D+Rs\\\\\\\/vcs34N8aAIzDpo7I6ELiiiVnXpNrQCtjmerIjPsgwK0lCDatiNWlb96R1FjXxrbrkkf176+OHTiq3vVlylecoprXaTey3yb7+DJy1V5SOGq+Yx7WcxOvIjtAuPNUx8JC5hfK\\\\\\\/bZo6p3m9\\\\\\\/+VWHu5piBeebe+OlyQ7XpNuVOSOVu+UCRp3EkmXPR2xuLGJRA7ma8vgUbx2GB+sdBy8byGjTRLNudCRK9E+JbM\\\\\\\/yKPR1MCpMGq4\\\\\\\/Y15aNdPBd8D9PCFY7OMJja0DHQsdtHqPW2t\\\\\\\/h9Gu4Wn9AcG2TDZH7fONNmSLv6iVr5tg2P7kq6Kczmx5HyjPQQ9HC4gHr+KalXJSPH6hcRV3084SXKxy9Qb6JTuNZ4hEPmkCr7K+In06TncfdW9AaRA+nZ6Ckglycsuv40DavhP5e4TqSww5rvJP0Y6jplQhef2VmAEfdzrbOeQQZqhm1U6GuALbadRTahhyPpnchwYxOKv1BhoIYCOvm86U0+wgjoqtAWFHewBiJ2N4LlplGv6w\\\\\\\/g+BjOTc0yXpuyZg1OkJuRPNYe5qqgRkfmeE7wIWSTkMDjB5ls6RcCPBddeOD1MWeIzdJ4E07AlS1Udra93YyBIKrtl93CXt+77RrFAkjk\\\\\\\/thLFuN2ViHixbg+fdQjga1O1\\\\\\\/ssUn8uXV\\\\\\\/mzsBUm4Gg\\\\\\\/BvoRbwUoXmJ6h9rvUMiQxvXbmS2zsQstrZqOYFDcW+RtF\\\\\\\/5RD8evZI9biL\\\\\\\/5LM8cs305r32zrWuDOESXbbt2\\\\\\\/08yTFsmKi\\\\\\\/QHvPHxfAp+E6tZ2CLepR2pVGkqU\\\\\\\/5lWd4yfxFj5zFTQRX9F4tkJ4L595jHS+WFhF60nH+uC4PdSBy72PTipBeFQYZq98Kb\\\\\\\/8UMrHjQmB7WbyzANbq0jIzYuT0caVARXUAxqHWhCE00Fh1BugcxuO4WSucNV+12ZBAlYDqES\\\\\\\/uImkFlRqK6ZunDWqAI0QGLnWTyH45WkNSHp9ee5E+CuXpxSlvc6CqDln6LF1omzV2SG68\\\\\\\/xKv37lpfGYjeW7tRTAImreJtPTCyIcN94V5WtIbUzlmRFmp3VIhUIbN3yhegkQslLvHz4fgIpFbKk4g9Bxp\\\\\\\/E6MBb9\\\\\\\/IVTQou1wICdADbbkMSywk+yKHqfKYUhSUrODntT0g1uFXX2OH3AQ5JUTTyBrToDAKlKtbNXxlJU\\\\\\\/NcWFfrIKknp3iJmB+zzk6bWuP6IVzE+edYFdXU+uSD0fZwwTuzL\\\\\\\/6u25TWb\\\\\\\/jWlVKl2nnPjcce7BGx3tjWzGIAsInfdytdyXa3ZDXBoCoeNl\\\\\\\/JnC4xcDg1jMr9kQekfGcL2+dtqB08SFcVB39ytn2TMppxKzGibZ6zc0OZ8OoA7ai7e2mHLnou7sBKIvarH6PEQsXYE85HD9kXn+SXRnkLvhMMVcCIPg1AtHWsKBMoHFZczZaqDY7bJX29m6Iua16rnIx\\\\\\\/mfy8WXsNQURla98M1uR5pxrj6soJ21cGaJtvbahwynFQR2dOqB5VtyglARjrw8968zBwsPgv86EYrQBENTH85Wxa3P4\\\\\\\/ZRTcXUjywyNo0s\\\\\\\/LmKFT39pqQSq3DfpjizQLMnpsG\\\\\\\/SVoBuB58fWiWiI\\\\\\\/W2Zd3VnDsMm8PDu76ui335ic8cEOtQjZDAPR2CHn4n6DYaHSqKH571Bf3Khy3wKZOOn5qDFCt3ktNKNFlU7U49WGisjip1qYRxQTgEY6hWXSq+JxpSePYL0GoAwt87YI8\\\\\\\/EReo+06tUAF9yCXfqrCVPZR4ZcDAop7ML9wi0K08A4E4tAhCfdFN76vRGl0MXH6g42HrckeQzH48ygzU5v4FWxzHLy11olXT+PdV7QLhT3O1ddsEEs1Z\\\\\\\/GiOlOX0HLncxIOqGLz1ZTfOqeoDebpaiFdEOUaV2KUQ14dUScAMQ3sw1YYLdsXYD4B3Bl3JLAwHk3w765l+O3KSf7BSoe6APh1eI79LM5eTT97IJFMZ2PAOm9nrv6fGjvDRNM9iCBPxuBY0E7Uqq1jP2yfdnMNNMbF76\\\\\\\/8J3bzvCKHvrXl08vJREs53mJ67zGrAQDz4WilVastMK1WhpjwHq28cuGU3rTWC1d+HpSTUodricKiSeXUa2xwo3xDjAi4QEPFi2h12E99lsfeLB3lOd7GBrST7QuPb638PWPD2L2fy6J7ZFf21JB2OG\\\\\\\/qDNRc9t6mnA4fhHYtao9A6ldU08NRgtbcm5DQKbN+x7crhSwjQWuR8WR8qoGFoxaIyNP8IKHZAhW80Dz\\\\\\\/rLCUNInBqWs5B5tWQ9SOn1pZusLL1gKTtjXLS+rNpsm72zFAJzCpg6Ntd\\\\\\\/jvcD6j8f9ePaNORdA00n4VlMAKCP5f3F\\\\\\\/P3QjDHNWCglvYDoqjyVYSRqGk2Oznjtcv8BbOlvOJFeTFe1wnLqV08RjKLPtaS\\\\\\\/IaMi3cs0WD78pSmFiNEy0yTixBvbtmvshvgoMo3cQ8nHhhTnpBWN5d5EJw5TJWt0g2ZBIRMN5McYcmU08Ymr87fepRnOpf29t6W4ju4ixqkEtkiXqmw+4xq27tYF2zBtbQOqdlKlC5IfaB2o7X5rGTrAUlIrUCUjrHfPuCkjvFu\\\\\\\/GkWzQQE\\\\\\\/xMsOGKsJxJo3nCm0a2DXh0yUlW\\\\\\\/gaajnwRFz+swlPTF0ZJu564R3HL5oflu9GrW5S7Uqn5rOowLyVrdM5Zrryg23RbQ72gHK1ZDJhjR7Hf+SX2YoUBp387W+6YNxvfEXWZC9K1WTC0pL3d58rQdnWZNGC\\\\\\\/1cSAu34vXA6m3aPv3HJ+qp0L4S+AjW\\\\\\\/93aSGZbRmK33hhr58eo3P1o1XVJSdLeaOPfqFHRzaJWXw8LLKC79ZPjTUtPR4UEL34Vpc+IPmB5OkWHRuSACM+tPL4R+k9Ay4WsKc8Fwco521MowNPRdQ4wPYYKdKz4schPixHgN7+\\\\\\\/frH5eJjAFz7sBi3ZiL7VakMpfaYh\\\\\\\/GFGoWKQu82abgBgYIgYRChRKa6\\\\\\\/zKWW2i1sctcQy1pfPUxTKFF\\\\\\\/IS3uX8ZxUilFI8JMpZDqLu7A\\\\\\\/+pquYfRr+IwqFUZd7Y4f8lVrDM27plOcQMZwX3ac5jJALzcYo0pZdOyO+ZjTdo0\\\\\\\/mZBaxCr8iHRKiQVRcEVLpCn6ekPvoJz3jhMh4hSZTMXu1JEwHlTcsrns44Yr37lal1xsthCAkpWcYCHM+kygERfVv7cKm9NimTgM1S54rrEhAoDEBr8\\\\\\\/sk99oceo40VvlVNwtmQk1vl8zrpi\\\\\\\/dFlTHoNCLnSMZEW6vV976od9OuIvd8A1PhZi95kIUEaCcqCRAyUV\\\\\\\/yIirAruFu6w99ntRdAawqHIFftrAPQ6sUlXI\\\\\\\/Q4DIfCgyRxk4t9fwurnr+1Eatb3w62CSQPAK8HvAKiq\\\\\\\/1yk6jOTYKbWBCdu9fwWasIe02IGbnFcgO9pMHyeU2t\\\\\\\/w9hH+pf3W7Kv1gX\\\\\\\/BgZ9skGEQ1VmVELq61gP3AfTvq6siPVD+sgW6XhuU6EZhUbrsC5noVD8I+se+JNINCrP+OUPcdvUqcCohQHzeTr4EBiXGJDivhT042XJRlW0sf4cFx3KUrau5X90bxf+VJ0zwIrORpxc51nszG+YG3yb62MQw2svncHfakhTrNbFJAV3xYW9WoSVWvIALYlY8RurZGeeoM8nXLiOqqobgZZncmvWItn\\\\\\\/GBUl6MH51LJxVG3I2Rsq1hKMHn1lQTpQsPisHiDqIIpnUHEUGOnb8bVXZLKPHGVE94D6IVit3nIHw9Dd\\\\\\\/H2ev1G+zcFmIPrKImneEvKLUbVJjyqS1pyiRz6uhDoFvcRy18s1aDbgBJM7AsnmbdcQSLMmeWMXdXLvzC0x4Q9bd+Ttl2LgMT7q8BNXiYi4ncYUgl6turk6TsxGoVpdkUHz2MzmhzHR2mMoXucOnXPnOCoDJ35eGkuHpuQGzJopjvbd40OnMkBlez\\\\\\\/1MkqW3vwjGHfzISuElJnSVwUyy2N7esaOKTmM+eVIN\\\\\\\/t1x7LbFGlXBIn2sIpb4mDC\\\\\\\/FLdvu6QBmwk0qB45ztlioUdLabWwYPsErOZfTmBC6O3DJ67fKRVDH3TD1fObzibRHoUpBrtVav5VvBdoacOnLQsAhHel2vbk6yEy9nzxxrZqjk2Dka5eaU5lmiw8iqgr6sFaQAHl+8Hd+nfy+tI94jmQE2yfQzDyGPAoRGp3mwFbExRMZFkQGqWqnT0DIX53hJnKiHb1f8\\\\\\\/O\\\\\\\/RrevrRTdcsfZr7S4cGkqSkoOMUUXBjXMX5Ni1jAp9sl1M8YEbnsVVFrmStLzDypAj26uUgIaCXbjhbGnk0wS4YqknwodC67uyNbx6br41MZZK3ZABnL5W+jFi5qAejsLWeogtM+YZrg5RWdWjYfG1E681MgigsUmTIp13qj2HuovNlX95aF8i3gb36LETFAcnZZ51akQf2Xmnm1WHes2ktbvo0uJ5UlKEEv0U1PZnOWdU9WrbwiNwDFGrnGFpgmoJPRJlyV23clPzfwNjpx0z5lB4qHb5FQRdVK+CY2bP7LcyYMmDu2gFpNWUXqw4IS5C554zQWIMz5qwQfmu4lGa5KU0jqnd5DG4bOA+pEZvnIsY2mGmm28kAAXml5aOxhZc292iqgcNE4ebSmgOvFuFDbUe37zG++Kv0H88+eiYlTx+h+WTp0DcoSWfaBE82mSnKQqsbI9z0v21XylmlLkuhOEZv2MX2kg4S7wXwySdThumZHX5UmiTCdjDdsLt1dxIBDVidENSL7dZlu4izLfadSiCIfu4SAmAjqYKNmZ+YNVvT\\\\\\\/zJ7T8GXnWHwYPP\\\\\\\/SCOBawSY7PkZwVhr5UPn8Jxm4tbg6RCU37OSNUD+wSakCW1ORKoGDdYZJsS1LLHDFR72a4IMYUNEzt4u8lfyW9BUGwr1+qtOAP5UV2Kdp6lOsFg7TggSi2\\\\\\\/530+BwHNBqac72MKQYwIIZU51kNo8LwN2Oud57SlYoZ4Ab98LuHTFAUr9Aiafs46fe+Cfqa5xeriE0aaMaTDx6Ui5bvBq4rCZOPe4bXSTCTKFY9u0MWF9dCNlnjSdTkvw0veuLcbDrNuNQfQBGdAPAvQbMIB2a54NuZNKA4oivJ\\\\\\\/E58iD9gUmjK0xKC27CJsodHctrjePrw48LE\\\\\\\/oT9qEeH9C7DVm4G9sm1ghM\\\\\\\/HnCsdHKpGq5Hj30WqKwVgnUyC66Ud1t5z9SJaftG68ptrZ4bDQplxsAAKTI4eoWsPECBHOPWYoxxSWriSnHo5kIO8+jt1F5mIGQ9KEslZlIDpNEEtdSsLgvYShKMgpb04jjxOQ+1uFrx6BnfvxAHNoMnTKpZNN5HoZIo1km4Rt8tIoPIvOkNQUAGg6ixvo2iLZV8Q\\\\\\\/nvZ1\\\\\\\/z3VgE9zM9TT8AYklU4WbyNt7FKQBH8OOCIwZREa1Lg3ZlhnUc6mzaFyeZwoczjNkU2ZFLwhXCnxKdYdTNJLA85AE79LuaAmV4J52J75DqlJgMK4V5Km3GtLaJQ3W7lmnLDRcGNJdj4e8jbW+8iI8Y3cwBa9kXJWc18cSz7qkxQqHCu4hFbFLdKz4pr3Minr\\\\\\\/onZ4FjGpWmKy9tWDz2h+cUBwgH2OnQNELExnFGGoutBaZfYa3jg56uQnqkWncQt8aFTEDU3C89xBgNnn7IL9\\\\\\\/MLKtRykrPGz8PM58azvgI\\\\\\\/In8nkcIFSRnuDzXTgKXyAXQMji9TqMI6ESBoWRVsboJl2PfnNyCDwXLKqiikBxX3z7Yz931NAzJOJcNQ5CwYmwqsBMhBNf01WSwN9Idhp7fA3vW9U\\\\\\\/+rG63ujWWzLh4M3vLBfuIy6FOgVnSfjV+kNGBvDEazUK54nJEZ3XwiHFJQnNiOb3dBIgNrgwrq9gJS35Fs8OvrnAkOMiEPVU0xfTQELaJBHg89Ecp5YhJel504uV\\\\\\\/CaLdz0F\\\\\\\/UNHuAtqoW+5CoJFjymZ2l84BVbksk9eelrVWWo\\\\\\\/Fcda5G8Uxqo4ea6YTjDh5eizZ59gDIEOUqnPL+IXmpYGQ6n01pLJydU6WNmrouM8YUU25TsH2akJVKNr1uqQZaZcZLdQJw8IkR38YCwzf81YzJOtN\\\\\\\/eku\\\\\\\/Z164Kjkb8sjpcjAnEwtZusgPfBE05wigqJinsK4WTszBlJTAs\\\\\\\/Z7++eqrJ+SjIY9mJ6+S818ij9iBSsvBuhCcsNAekXKKy1JyC79CwJoCZmvTQFLRsyATxZUR4hbk5S\\\\\\\/Zlk6qo7yepTgMGpDfR2mYQcNdsNDMCpUNbEtCua1EIYcHCjY0V+27WPCxh\\\\\\\/9mayicvzEa8wgHv7qMfceJfO3bVU4b1qF2euS2NwHt5gu5RtPM\\\\\\\/es3YqxL\\\\\\\/7QYy7CM1va46xRaw56WnUGfYRDPkDYvb5hqccQqYx\\\\\\\/+JoYWm3xhvTFSdOzmThqf9XZeXqaBbKybXn9FBlMop12aIYxUGm3Y0hqXO0SsTEBaY6szzCMxANcF0MM0zJYzSafsUJrBKuHf+WtNnwBBXvZHm7p+J4jw\\\\\\\/nRkD0cujXUHQAPL4bxvssRuugjwEw6k6myMCIU0\\\\\\\/VGqFigIoKqMeiA4cs0hxN5THH5VMzHkBjp+9t2X8BIvULpf8QgCYC1AazsZ1MtjCTmdhYnzzTZyczBvJeJZtEVzFeKNmJAhhUOJfdM1\\\\\\\/IvYXZl0irOTJwc318\\\\\\\/204hmnwiJekYeOTmr1nYEhxWfEjJ+CNeF41nM3fk580g8FtM7EdexrMENgRJoDqAc0riY6P+j8WSfEkityc7kjrs3IX5TW7FAgNK6a47VWeS5F1MaXjwYXO3b+dgaZk99GJCj4HvdYRchUmxz961GcuHkZZAtNJ2gack5aHVRpOhFf451hC9j5GhpnM2Pm0DYFJTW7Mle8KRDjnZnsm5JLMffUrZRJkvVPubIqE1E7asRaJCrA+r+gaCiNHLc41AW7mSpyXJHAV\\\\\\\/6hManHiEZEbgdfPpWKbGloDEWwSQ9Y32XqKJ3\\\\\\\/1QQdnrcv\\\\\\\/f+q8OMSvSiueIs47u2dTUpuTvWW4cGuVAkorQosylbiN8b6y9NIZGCvYOVfjnFKYQC6QV5T2TG+WZEiqjkEjEI8gVHYnEZ1DvFQHxDEX\\\\\\\/+YeJg4exQ8dAqPrv3+y\\\\\\\/LtYaPJtBhWKBOgwRNmktU08Jv4P7+RorXEELyz74gNtwfZxHqVQrGE8bo48VpEGu08aBFFJoxaTxlMBJd5+WQ\\\\\\\/uTPVkqTygi1oZLHQ+bV0nPRP5eW+yTf1lCTK2T62HtC3X5Ue2wf1K\\\\\\\/F9WSYbfGVYwqqXyOiMh9ZY1TmhD2YlfkxpgC21ESB3mv9Qv7CMJZ0ZgoSShCWcCwqv5L+vhESj9mLPbAcxmObDAoCL4xNPmIcbO+QYEJIQOOC\\\\\\\/1vXkHcFsVWjaZPgz831R1GTK7W\\\\\\\/kcWfk4IYR6V72keHRxu6jfKCnmCiOnRlgrhMoWCaDf6Sjfb4sL4ycEG2NQDO+iXafmfMh4ahBlCPZjUDJ85DhJteyXNZU0Ux4vutCTXQBi1yOLtjyaK\\\\\\\/NIAKzVFn+QCXOqNylpvXEqZNi8jm9feFylqO072dUeddJlCUEI3j+nL+h1Eo5OMPaWm\\\\\\\/pt10ufRzsZGtZZ6Hjjquz6Oh5PkiIHDXGIFWRetzWyjvfdDJL4IfHnF0zgGs0ONWCM3ItuhGRGRfsaDuOHCmx9Cs1tdOVTZHBGugsEHMC12bNqi88HcicB+J3+0UVslnSS\\\\\\\/ss53Xb71hoWUE2GN\\\\\\\/JE55OVNTpad14ZPtBHq5af+kjC1KPhD5hmczL8ObrhYn4Fi+jHshRtBeaBkYjk7uVdrpdLBm3jpuosXGn51t8urNwSGtv\\\\\\\/KIcFQZKOdG0lRitoXBrM9e8HpEwMyyiVNmZPOzlnBhhAmrQWp9lMmL1jILBIkUbde95rYqdBIEsMxydayzVYDRN6xh4wpaiWdGU1YU8RyPQ\\\\\\\/orxYXytKLNESngu0lSe9\\\\\\\/OBK+ys2bRw9ExYGuQZnlZi6mds6nXSLhNtNK0t7PRtxcHPVxbBpw3TSrmWm38JTbnPaY0AbPoiEhGAKv9hVobMNJ99mR\\\\\\\/K4ECVAeUHWljZCfG0AMpWNW8hD+2aAWGZB4mOrmvucZ5eMc90axPWpOiPW0IcsglpakZ60HZOW+WXNJgtBv0n2dFP5UuR\\\\\\\/dv60HqKHXS3CQdm\\\\\\\/n6KemvpVRd+w7S4PZImMsDmNKjtIc64edJkrYuV8tMCUqaWSc\\\\\\\/vyJzYuTsOP5R2xhgfwRWxAjYKusL6xi0oWE2d506slaS639U8rq8fjgEx1SPIBqeJEpzzfhgV3Rh07KcUJ400cqrIdDQQRwtYNUl7QCZE5+M0zGnnMvw68kn9QWsVckHUBm2WpmaP\\\\\\\/\\\\\\\/Qi73ZGy5Vxch8AF3JlB+pimarxXkSA6Bp7QwH5HQ1loRt7RYTU3HN\\\\\\\/5o1yid9ZZOHQ6gJR2tju5dObXx8mmBWK8dVZiQXvVI8sXCJ68Amrc7bgYcRlFDj4UG7V8cMeYIqFgJuQcjp2DZXAaHySt+GBq4pGL0uW3UpP4WxH7xYQoSUYpHtgSkJN+N+QQ5lwj96dmIf3+nRxJXp7eZ\\\\\\\/oQP4UrXJV+wDoje4WIo6xdOm4rpODJnDzHYmlpR4Bi4ileTanlay1KM0RLTwmQr4mRayZIsMSSuCH+ZhxqXNnqL7N5p6oGwcZUAVjDIkP1ZjGObXO8zNMpF8qlK1Z3IZwhZODQJAA4KV90amEFHbJ9pRicIVaqF7FDMjPt3ObKB5oSKvsAn9tzPQPtlh7JfA29oLOYyrmrjrdhY\\\\\\\/yAYRinnpRKyk1pi6Yl5vnmhiTpAvOoMvoptWuQzqdIOdbaEFIEx6GDZ3b8yZh+LDRwWdGpHHkS5LdNN4Zcay2dEeHC9eh0VGqVnJyvvVu4XeiLJ924JiKS4PbtyL4FxKYabIeVb5fSSnVA7QJkgMhJnZQT8sSZvd\\\\\\\/3q3UVz7k6f0YcQmIXykZIeYC9f6FzIq0kLLXDtg2vqOU4x6EaYhrggxzaWM1cH9UvAIHby0spPSDCu8l8bBsG2Vp1bCumq\\\\\\\/7sWZz7Y5BLr4lXn2n6ljCanudUhJx11cQf0jQIeVMZ04tdqdgPj+wLmPEhNav1jO+XE\\\\\\\/lSo1OiQJ7cext0ahZCRTVOStaHE5TKS2uFNpx\\\\\\\/sNTEhUnj7ayRgSI1CGI4xYvSEyftV34PImDXhAqmV5eDxama4AZZnbRGgNRSbXOGHxyI8CQmLI5Yo\\\\\\\/JvMe3m8JRnxPBMvt5TQelnfpr8P7iJAnh1RbuLEvwOfV37ADMr9xEdDVeB9\\\\\\\/Ejfvu2HswBv5mzWRZVyKc0tO83+7HdWYeV4XHieyf19WcOfNvN04bD5R660IQZjSIT3usJB1uuCED62BbFLtKO0LCxqqx7kajBPH3j3Q1VWs8CW3LU7NGBzHVOzc8wqwI1fs36pHxlOd1+azwk62px0FnYlwxa\\\\\\\/W8FAYE03eqc6A\\\\\\\/3EQ0X7pK+fLxewerCKhR\\\\\\\/bYH0JGs\\\\\\\/N1FSs+mF+RH8P1ovJ9OYEkdPeqQCIcYibF3XUz+sUu\\\\\\\/0vKANFcUnic11dEmNrlI0hqkSidoW1iu7CtqMOdc4DnCpAfgwYLJE64obBWTch4\\\\\\\/blfzlc1xobnVfqBbzwv0bgUkMyOzmMRKpqxvW3HL2MoGO0\\\\\\\/x8YTV\\\\\\\/LEIad6ZrkAK8cTOGSYL\\\\\\\/+2lj4UwExyD1Md7HtJObPsZbdJTbgP2Dc3ghp+TLDOUP2HGmw6RAcuVj1H5wyrUl5OnShoLtcjOkJVTs3N2EVUak+vF1ohNDgBtUrlmwv4Tnu0G8JwnNB87zzfHs9bi4DtCNEtNruhm466PpvmhJ5bckNrQGupYpEo7X3JINXBJ4yukBXrdi7VEKkogplygh1tqZBh12H\\\\\\\/GUUGGKLb+LlRVvd1Mx0RomFBeOrR6GqBAErGnjukoQx3TKou8sfy7ieNHaqzW2nNMNIXEXsiMSiBMdQpRGPhlfvPEwb0USMF9iwoAgkDMv\\\\\\\/Rns+KEuLEtvYGK8gc+Bj+hkWYjlIhFpZw9jvJwmQkBhPCjnD1TtoJvTnYXOxpfAW3iYw219gaA1puRwkHJeywNDA1S5wdmOkKhnzuGYCHhgBvZZYNfd6\\\\\\\/dj9KyvxplRLI9t5OoKRtg7fJLvkZxXnw8HUJDHzGnaItp5nUTzlaBvn6NfRSNts+W7x2bqRtad7ezqgnI7BGcJYuIpF4PzuIC4ycT4St2iGuQnlaM3hhTGGw9S60GRX13L\\\\\\\/RO9wBGpY24IQu5hdAipLd52+9GH31d9aX9Mo4nCh1dkw9EjcEEWulP6IZVtgKDHwthJk8EOgS7J96MDm7RnCuUHVtEZSy5rNt3Ls1\\\\\\\/rto5FvRWnxYq07+sENvQjvWTqALN\\\\\\\/LtaTuDublxRLlssnBd12KP665iJMvHrc8igPXFwFUF4t2+q135w3+GBdAT1GPL2+0jtMuWg5M6mgHOBMIJDIy6I217Ci2lRxkXNLBu7FV+e05isMbcK6WeYLx1Rpls4KOV1LOyTqBHD4G+ytYKCrEJ1WHmcveXICAXwjB+Xgm7p6uQVwjmeum74fjGBwZ3lL2Iu8EeD3TIdRmwvSEcNmiHrhZ2cLDX0d5uKyWFvtjarWqbunDnaLYDs5Hi5MhhdLK0PAqDraky6+pnaSNXbboMu\\\\\\\/ramJnTj+zEfmW72L3JsTnDLlmpAssYsr5M8qpC7K7MSmcZcvj0iO9J70mbjJ+Zx3hIhUDrUcZIipTYMFzFrEGErnwwJx067PHEyERz95GJRkYP1+7nQp3OQla+9ATxTIudvHnxYBowIjFpYYwHBCrG2I1QpKL6rTMfFeDcEycQG8ECBn9WPWeM\\\\\\\/y2jNfki1FxGAFSgjsBBaoAWJhsYY0iSKuT5AHx9gAtEQ5lT1YvWTY+aGqD9JspdXcR\\\\\\\/2ama9dW7ndIH6zsZrEJu8IzG3U96FWklnSITmd3VobwGnxsjyi464fZBAir7RTrlEE4kqq2ze\\\\\\\/AakjR4hzWGJw1fuD5KFWORk4dDarQPBdOH7zQTeh4a9hHbyCN8zENehSg1JtLucpcFUanRgHAWxP7TebGXL+Pd05rqb4Twwq72A9OuX0vakoDYbrmg+NVjEgEx7aPIk0q66l6CqZ8HDm7IOTBvTH\\\\\\\/UfI0LSbTL2ef3GY6+FYmLxA6uRAr2JkKi4Nyegebl+cKwuJ4\\\\\\\/UYZGq7Lt3ONkE09ZxpR54ZvZO8SjMW3Dh1KWyGz7QlzXP+e1YOrfOzQK7h7QyW4zPSllZD4od0ymbJTp8TUjONqGHeZg3xjjgmy5CSSvF6DxfG6xlVXHF5\\\\\\\/opozT1HfvICvqBfsqxE6qkoJRDCi4TyObDACyC3NHWI3dc5dVniHxdVcu\\\\\\\/nDuc0wIiQ87lGzmOtVhjmYEcvuG9EHVDxobGT8l4msvGWGPBliCjMh66p7qjfO3Kdnm6yxJy8jluI166xxlQdTjRq0mmV5W4jfowrZeIwB2FjrrsjIuOVMqhyppC7MKZZ4W0rd5h14cns0r1PPgKjGdGV\\\\\\\/mSH8qL1ALl0MS3t7PQ22HKjkRyzYuy4QYSgVgdqvXLoyB6KJ3UkAqQyq3bD1R87V1Ou7rKzw6jCyaJDZG1CUjHVLKmXgdloOd6pn8fZ03dAEqghUYqWhKUzOguakmIUfF3RWSOaJBBIWtswLVNDcyzmVh7hojspgh+Ny40j45QH\\\\\\\/0hH0vTvc8LIvx33fHtRV6Rcqs4SP4Idun744tHa2hbq8oJHFPnswl4xmoeiZ2yH7wwGnfgYPelXfWwP+O78nhGbl96zJoaw1B48HIVcPi\\\\\\\/3wZgrPnos3RgqYef7Pb+7jSACYoO9X+tHALHImazW2oNmxfOiV9PxfqgUTLR2PgOywhI0zTcftJNAup+iWBZ3BU5AV2eBk38GXkcFTpkf03NMMliQuyiZHeDvHZ56BAQaGbn7xWKvMzOjc4M1TRULH8ty0yU92B7E7wSfapC1FOTgciF\\\\\\\/oFe2VxyF70eJbU80xW1n6ISjL0zCgfp\\\\\\\/cdPwCda9Pq4Ux1gJ5PO+SEKKAeWnQnrH4ETO1zydi6oNbiHFghhjDDw9Q89ZpjQEEMEjvuYGA0XK8yctxg6Owb9D8MpjsJD42ejxdUt+YjPg4C\\\\\\\/QbJoW1qP6+6Geg0tvffakiMnavGWY2PUw19vtvhMZBnAg3L7sNMjQ7De8TsNKsDZ4euInl8VFfXd7xPBVW7xkbIA86tFFbky3Ia8ghpoLxmyUVrjJAMHtvuZzteKS0+htDvC9p1\\\\\\\/f7eavTfbCMeZDyTzZ3OcdsFLF5vA4IJFj3h3CooDle+ceEQJAacrMyEXtgnbUoDiQk6qf1N8lZt1BVT+zlD3wsgocVRSGAhMJPgVDeATCgs\\\\\\\/SbcoDe5zhPzNV6FkW5RiJMHhM1S48rP+B1glPARrY4ayRxYrDSyJ3rBSM1YdRvv9Wx\\\\\\\/Twz3RwYxgZ6KlO2Eb0RHzzdDGd03b40\\\\\\\/kmDj572rUgeUUCA3Gp5Ae40B0Wr9xSCx7h2YZW2miu0N0vMGqMLPSeodBNpFOr0dsxyBXyh1XOJjTvlQlWcDA6ZUYIF6JLmNc5Zh44pWZCSVH0Ht75q8L6VG\\\\\\\/2mS\\\\\\\/NVfhf0V1KS3d2sxY\\\\\\\/TrKmv7ENgNbg9ef60wUmC1XkvTBdaEW23J4sV53P2LBOyTW8PivHttG7Lv8xIRmpR\\\\\\\/ezkZ+2W60lZW0iICqOLFHXVmjtKvRESEN2\\\\\\\/IoXUSfSwTz7A5g7rNvCM31CWKqoBCQjsTUJsshe2d5CZn01XrShevBEHic0Weivai7G2Y3YRbCZ4TGJOIEVLUDrWwFvpJEQyImCRL7CoZt0J7Xtp4HNfMxswAnXxK0yLzOEouVMxyYGIG98LpbiqEcM4nR5unQihPkdJCYHIfPrapZAqSvImALTKtLHdL4OoWrQEiEUHD7u3HlUSmWDx67xyFjxmW8pw2rc9mBxf4CB\\\\\\\/70ULNN2W144EWA5+rrCevH33JOcomPDxO7VRU0Y8ZKvM3841Pk+RRs8MlJuNybJCqLVNmbf09kR\\\\\\\/\\\\\\\/oD2nlf8w65esI\\\\\\\/5bkyV1mJj1TOuVW7uVftuPZhB57HRbuMxtQmvgctRW2NAFzB6WcWkJioyV\\\\\\\/ljqi1kyztIp3kqkQOmw4WNdW87Igj6b6v\\\\\\\/7dZ7CoAw7s9zBU12EiomeDj964uO8NuBSAlSzYpxlahnqg2\\\\\\\/UH6UnOrXVs\\\\\\\/bCy7cE\\\\\\\/bTlKvKvRSBFVl3mYijt4GNs6mRxAR6+zq8kGpQAFlzazejLpIaGNk2zGka8bd2MXFh\\\\\\\/nwwFEtOPfBRgn3LSdFaFQS6VxNqEagwi4Lpp\\\\\\\/rPTzTm3hsKMun7Fisbq0wGW4OWC+vycNgbddAWY4sExYmsN3LyZNZSgsJUy95KgogWAbqKO5BF9HNY0V6ixRqHgfI9AIbKC0m5hrtAMvkN91+XPY+SqoGt2UFL4\\\\\\\/OkTQdVz4wzH3Y6SGr6iZLtYZdzQRnarTlhA\\\\\\\/GgfW8i+5UQ6MCekLWn2pp42it51NKnDWvhu6x7k3TM\\\\\\\/p7+ZqzyGgdjmMUbdDTQkKXetFUXsoIU674idmJEE8LbkQnxqGW9I+l6bXslvJ6JSfwMeQ85M8kmGX4KSG\\\\\\\/uqCCJX6BpBmgT0FEpQrPNtNA09ubSjqO9pCbXPklmSqS0t6g3WLznpwRmOMtd9MwNg8vKX+X\\\\\\\/YOdzeLySNMtoDt1dPiPTDwFnZKvdjSmLjKPyp8dzBuJjnjsMAdHaN4dDFl4pa\\\\\\\/w6HI1tP0pio6l6Xnwqd7NPPSRL9u\\\\\\\/NP1SaAKT\\\\\\\/4e9nzen56TWMETZASpOBYBHQ\\\\\\\/z6KQTNC5T1V0AC+3jKkMPnjRaoCdh7ImPPdqInFn5GIM3REos5yogiEjFcBzMLioTMGaPwr2G2dDRly5M0X+tSXRY0VPWKiLLomTJcesjgFB13pXoCbOpqHFLbm9vhuLOY3OMeMFEm01cjI0nrxnZK8zs8B1tvFkmv0GJw1DJtzSpmoHSVc7ydTsqQDlsluftNP6vJq77QMqSK9Elw\\\\\\\/h+WVpiZ6SVnR19o70XyZj8c66wLUJ4wSyOfGcokujg9Nc6In6GPmuK6M2n1ONCzsN8za8+LTHQ6mky92g+ZVy1OE\\\\\\\/PvQNAm85BaQErVn4kwj7oddLtfkgcB11YT422dicvuhxVdjVhxazI3QfvKs1\\\\\\\/DkdtmOfo0mWhgNNgn9yzr1myAI+RkZ7cFpvtlgtaESXAuDXiwz+GahCCg3bTYs1wup1nw7LxX8ykZfjVqtQKaBhIVJGUVP36JSxzVU5Kpb4I2716tGsVBP5zD7ikcAZsuk4QryQAqWa3X5m4v2i3PYFoNLAKqwVz4lW5Vi8PY4ViVbq0+5GyWkufquBhs0Ep6D0OZTs2Z485x7W4R++HtUg4rtx3tLBlp0qjJwHdjSHrAN32jqE+P5jH8ZJxkitFV3ARKjI\\\\\\\/1iXBjo\\\\\\\/ToIw65QOnT9rXT0YYmaI4jFKrqR\\\\\\\/q7dN00i+e4swiVN8UXyr8CaBu2pNH4q5LA1+Im+mREOH7roEVGaGdbSxySVualqcLgrQPyYyXIEDOKT+StNg20XMpBvKAtzpgdgtKNyrSSSVeewB0nXHNpUqzdYmDmMcAmIWkdcNZuBc2+IrSUPsKe850BJ8FFlQNIx6MJp6hDBzOG5tYZl3narhrFIw1gB5TDGlPh8eHDnH1UKgo8dIfst9IiOskgjr65P8qXtZoMnsASXKYpdn0PVcJfFhhesq\\\\\\\/AFZqTtLG8p1WIQKTTWnUsbHeQDeZgOkp2Ti7+VMXX5qsWopuwwJ6XriiWHsfNf6vCbsdmCQ+R24kyqPc3KWrW8I6g\\\\\\\/yUbWDI0lUoDzDJ2TEK25+wwU2Vr+kAht1bY6v3uIE+eVOllNLYXxJiTpMk7DikvDXrN+neqAgUct0VfzT6DIliimz511rFbiPIVOJoM9pvmnVem+tfFPr0REjlLoqDEkOO4gZXFtomyevFVTyqgI5mzWpMFJP3ff814NX7YO8+DBE89DRBR4U+C5BIb1kNPDrqWx2Tggzs5N3Rdc6GlEGq8XSfpxfSYWOqSJJ8348EGqwNjGIwTKOukxORdsVeWZnQDRZoGk8cWHKUdFC0GkHeBHPNyOKP41ti+tVbX4BD8hkyTYutbxS8z793PPb298NnzWu8rX4gqu87o4G8bdWkZ7VU0cKvQrSHzXONqnX7ypHuwo\\\\\\\/kRezVERlvJi2OwQvK+DgRtZHmQCc6f1jmxMj96Lf3D+jf2fwwIHWyy2EzJvb3CBbt4XDNZcrXk6tJZidnmacwLTPSKmGwLcpP14poepDWix\\\\\\\/opSrltfdsl6051SqSwaSC0r1jU2LZn40e6Dcb1RZ4pGwWqoW9lAq+J2M7Kw5hTG\\\\\\\/1dxIGA6jHDsCRHiR3JWuJ+Yrww\\\\\\\/0kSOVeHNJwAkURIxIBYrNuOofcjF+5m+OSwGk7h\\\\\\\/H2VEAvZvfhpmIFR\\\\\\\/vT\\\\\\\/wUKXixeZiD9qZPD7Iq5e8Oe4DM5qZl6xp41N5HBoYLGHuzh77zyZ3KhcHCrT3uZcKDdGI2EgXPzP6Ou0vDFgF4mJ0fdYcy\\\\\\\/4eOCItfbBFSbdJ4mE9gVKDnoa5IqXz8u5t4acjr17H1Bn0t7PQAmOnB6W855YGYcJq1kV+VRSNMwGDDeXxOJhfXFpNKFejm51DicQW7B\\\\\\\/H3gT5inx2DFw9Vtgw4XI8u1BE2gLYJnB\\\\\\\/9Y8WKkzx5QB\\\\\\\/TjGWpPKidGNZlVZ5elvEOvYTW1kBp++kuhyoJBc1JoIC9Kuaw7vsu\\\\\\\/NrFsjIserY2VIuStNKNsU9NjEuEa134agrgn3FdDkcVz\\\\\\\/mVj+MsSPRp9qtBpKxJ9mJACTy9VvkLa\\\\\\\/4k6MlWzu\\\\\\\/F5ug9qQMY0SSRoNhk1aNR9ihAFA\\\\\\\/6lF84G0lLoqoI0BpPqS4Lsn74JP3WOaOjnTUMGclaScJcnwfhii6jvA1OP4UEY2XaNSpwMDsnKKEflGWai17IBahUfTITuN9qFOMNHMZ7Am6fSJ7Y9N3yW3l6+m6BAFhdcGemVXL2oygcomyzGZgZrnsGAsZ5wJATP75ugP0jOEyUF7kvErmSMP00muMhYixWPtlCDqyhmqU5tl2meptUPEE80lQCYzjCXIxRSjDeBUfJZP5sYm50dJnxBqShoJvnzt24Ce7DgEmyqpwflFROXeoE\\\\\\\/TOLtUzTunrxjXmJuOmINLe23wVemL+rj3X0Zzhy+iJVE\\\\\\\/YPoPTLkTtX6UwsFECv1YajssxGintaDWdxPjV3VRgaT7VzJhKLkBT3lavTeiK7pP0DBRlLV+QinFnrBO7QbJoS7kKj2DWub466jj4jQLF6gsneQJkPq1iq8n7S0y9\\\\\\\/APdwvyGwU+O+7OaW4q3EgDXF2IgdzrPLi47O7ggytV5wsdN+mpOpRsy35k+7FzKY9yN6B2dL2mvH+DQKoIeR6MyrihgHkQho2xPO8s1hb7DF6SbIYlP38mGhYTCY\\\\\\\/x9LQd\\\\\\\/W\\\\\\\/LGvZlorOgSBaxu8jz3xzp+F4ATB3Ln7AQIKgqPruVeRr7aGebCb2ACI9xowfB6dpawIZNN+azTGzoilfLnuWlpA\\\\\\\/kR8cGTKzBoqwhsTmNUsG3OM97dzNtGS\\\\\\\/UDFiWE029LxQ9wwnwob0XWdD71koscaQIA5FKgkbPBKTGIbt1v6taWo2HhmJdPBZmnb2xYUTFKafoM5SErk4LZOz1qeAOa48IS7WWVAniZJr9kpYaI\\\\\\\/0JiO66EK\\\\\\\/2jM+bGHW0C\\\\\\\/gHP1KOvU2+VMhG9Pn5IfsFVBQL5qUQ7NX1BXMbE0bEdmBF99QlxJ5vGjGsWggw4Ur9mjH82ZOzX4qLwzzAG33buN7hmFGtdwIYa5lkZbUx\\\\\\\/2Hzugv9LUvt1+UcUYoaxfVMGxM7VXLmdU9ZXl62U1fScLRrQ9bprHpoPK3iVs5cxPd0Gn\\\\\\\/OGegHKR9CSJP\\\\\\\/qwsEbE6u6oWN6pCR4IdPbFQ5apjlM1S1bdS4tmtclOf5GruIseU8NmbyziPPNwtN0MP80nLiqkY09mIh1+Rlmab8xd0MWq9+lVi569zcSacaaKxSM1Fja78Da0mtjrsfCLBzaQpieF6UsRXJfABgQRDoNbC0FsUJvteDEGSrr9gYl7Z3776mEnED0gTvoXQH+1OO4bhWTdFd8Q+7bq7teVxq\\\\\\\/lnrxVTZNtkWher6Jc59JjYB4M70KfUu2G9GDb0kLcIhWZoEzhsb2Rn1s02oSJOCTLVBtkVkSDeQJTySjRBUMmyIeiahGPJnsOyU8ocqRBAhSBV+MnjhMOO4EluqHzqyzMd45cff2riNPVMwWJ\\\\\\\/TItb6Pf3zT9UBdVO\\\\\\\/OgllOm+r8W0gT4J71WkUvroKejpQsciBG9P7r4CZefMjsCMvOawuozixN4pD7+zHRz0sOVoWYwMt92K0C+hZ2Bad6jdOM5g1GjAi\\\\\\\/I0tzUttkbZCtHxHy4szyDJ6AmIvTpBofQrJS1QxfhvIeH1N7faxTUmfdHzc+HfrmkiEs1VqplAHYOgCqTIXaO+bdzx7lA7J\\\\\\\/Rj9CltV1lq5aajrKx4C8u\\\\\\\/jjhuKRhZo0R97hrxhp2IOb88BZG+CXflIUb+wmEd3J2nBI\\\\\\\/jujwzf03Ld8y0rtlH8TFNsQHOHeU6KXmM0w8tGtqPH3cVaRJExO8VumYIQdNnnE6u526zNJIJ6dxWUwIfpQT1MkXemF\\\\\\\/8sG61q5Ye2Fw8Mqc+QIX5iooh6ZHnXmqIwcWrfmxAzi84mL2LBR0Hvnv6HC6vdyikmfJ7VW2GseMnx\\\\\\\/7u5xq1hNIURs3YwtDwHp2\\\\\\\/tbjF2LK420dn0PNXcJc5HVTXblarUoz8XNtAmEMxmdLCyIrSgC0CpRZtgmOZKXXOYERSH\\\\\\\/NtESuyFhnMkyfU\\\\\\\/7MHCYv2Xu1XJbaoyqw2nwiiDNOflQLNDchhox\\\\\\\/mHc\\\\\\\/eIlHbzIv+lxcIWkZOcChCwLZsl9\\\\\\\/RmzMHfJpoiGZL01+BHxXWt9V3RsoQhUE76pmWxayrLyWXNasT1QBBFHLasNwggOFGJoXEFO0Xz7XQsdc9c+P5u8mJv1G+u\\\\\\\/kX0UnZVf8VmDQDVBMrlnWgl+7dkhuZHXrWsbun+xRWpC07Ec0o7GJcpO39xMzU9bz5QWQ1u0hIXVqyrfzOjPEfP+YI\\\\\\\/CZTmnMzitUt\\\\\\\/B\\\\\\\/3iel3ma5OOE+ktW1l9rXk1IPQ+yI7lYiaDLFa78HBwNbmAnmKgx\\\\\\\/GLMVSWjht2GsNRn0h5gxwOAvNKl8cNLCmv6V\\\\\\\/zrdl4xALo6nFURYnQGBCqLGZ7wwy6U5ScSdvVoOyxb\\\\\\\/HisOQ2BrgpCV+\\\\\\\/tMc\\\\\\\/YuYLfYH2nZvk9P+TYnhSO7KbG17RgO6LWZMVfqHbBJPrOK4mqV7Z\\\\\\\/hA3NmYC7h1OqN9sYLSLtjmYsljbTVzH2OZBDHicpyrrX9VOn8NHGpcga1Jyfr22atx+kGscMTPX4+mro2u1MCNINNK95uV0gIgWxhmevefXSGcT3iD2mzbcQ5A2YacYpS3ZHo8w2e31yNqG4fl1nkF5P\\\\\\\/dPPdM2uDH6aeX45bulrNATNEO1Jnnogy39nQjqteJ7+rwqGSEBraWGvOMHzS4gvnd3lfnXamTELQfhMQfL2le9cklvJrZ0yRxQJwyDVVbdMZKs2o6c2jqaQxFDN29T\\\\\\\/gWpcmoMPxxjB48\\\\\\\/HxF+hZ547\\\\\\\/mARniWwXApT9W559Honm2dtCvOd9YIwrVdH+t5pTZwLcxA9SVtOrjdNdDLjo8LW53OtQlyQfQwEXjTj1cMvPxOsrxqNAmS+PjSnBR6DsmEi4BHDd90obYN1oBYBvuMTsPll4d1xF79fNQYzhjyvak\\\\\\\/3+\\\\\\\/g642Tg1cThwx4kOnbyxAE2Tt5UbHARyGZyCroyJpciz\\\\\\\/3s1q4lSd\\\\\\\/7lIAL5zofROuL2tR3JNtAKgie3XhK85a6wLkO1WoJKIy8LOwWuR9MscuinmV1djUDkiuw3j30v1c6V9bxyxcPW9yxFBuzKQZ2pq6ZQWPu8vLWiMGG7FfjcDSiFe6fiCypRrVf7fQXPDCEtwweMnMiX7JIiJLnjc+jkgl6S4qLlUvVEzQEpXU00pny5yCOGG6rguAYQIOdY7T8rX8mgyqONBdqbuSNNhnIWYk8ppbEDKqTl21YK2+JrNtcvq7HgTuYVztd2zcipQqsziZCS30\\\\\\\/Y5PwsSgxnu7+qJL2r7nQbFzGZAh5dPQxZC0OUj6tYeKoHL58mKMandKYj9hrYjzjj7mT95moGtS2HIE0Tph7uVGrxEUeajhU363GpLt4kTFjRU9WGN62THnhAcjBmwbowsjCE6VPSHu903C2kYcmdd6VbKOil1t4JClEv0cI2uNXbbjTX\\\\\\\/RO+2hCcbCOTat036PYNUaXrl0hKawEQb4H6ki20VWcLvbj3jocGbkXNiK9nPjzpQu0AbmR68bqwSxuqG3Bj53CCop8EcfJ6U9aEHPutwzoXlk0p\\\\\\\/J7W4mvny8GwEPydJ4Vg4uacoseRb1ZIJdiuCH1lTb\\\\\\\/rY0UIcqxR9fTKkzSJ841wHxPb9917pIRvk1INmdfGspUv9v0c9QApBz45Y2l1zHjmH7mh7oPq9tjSelHnLsbUBAmuAv8E7KSf6BFe+4i5+HL8tuzUyPvNuIzrG8d6348XFaL0tRhxmqZ+G0t6TfuQNxERhC3Q6zk\\\\\\\/MEy1yL+mVFy+HyLk2UKAY4uqrnTVDszmvZgGJXSFYtztF53nf\\\\\\\/n4T6bYyofGfJmLm3u7hFAySUgkxR6EP+x7qn05EdYsWOX37aYM2hZ81JoW3YigARyzFmWaE9I26Wlp6iyfcBVr2gZqSMZ11RJ3OtWlGsTwnFcNzJjrW3Pd49u5FZwEzNAeqQ53vqJjpku3LRscSHMTLb4D\\\\\\\/gxbiSWgWartVOtzo576OA4HtQp30tqmAzPKj48sjKhEMnT0EEEEgDiD5tmYXIv9aZJIY12LsK5ShZtyknT8uKRdiqqhXag4CitxxQkRYTem2BCaL3lXXrJ0XUC7UnyKHcJGD3brvfwKunV3ajyxZXx3bBrAVWKprz\\\\\\\/V\\\\\\\/ZHfMKbx1QKtLYSa7mdNBHcoG5LMfqF7414UbPrqFGTfpxWkS3kdIOKkYJvJqsb9aT01ODIEj5aKe2XrVEnfhAgvZ\\\\\\\/sVex2qpUU2jJNVro8s\\\\\\\/PfK1Oc2EKFblU4XphToVAFZY3g\\\\\\\/FhE4FVcQtFqtQUaYD1uwj0WL0JeEDeG0PQDeHkUMz36b7seDdCR6a8EbGPXRHriPHO\\\\\\\/F2fq3dL6YOw17SnrAfb2e9\\\\\\\/PrIvTOiuK1DA8S3eV556wyUbO9AMbl9AUHwZiIg6Bb\\\\\\\/l7THgTF3Qfl2u2XRD63d4UPvIgTH2tGoThS9O3eh4c3ubtkgOIM2Ff3Mqx7HjhqTOap2ahw3jnS7t63fJpLXMMv1d5qB8SbGo4rCd6y4HHAkVxC8c2bHKrgzDBjl4WOqtsdAseHlnidk7lHb8h+pgGoOUVixg1wZyr87jQlwkI\\\\\\\/ZdZRAlUOyJb4F1IzoLFiQrgCb7JQyp\\\\\\\/FI7M4flRbnUGjO8WzfGMMAGMLSmUWZSQyXnRcBCMgDrPn5RDwk06DaKQbCprjmY5roa6zY5KXj7IFij4b1DKPP4rUBiixzlzmOtstAlyKMwFUayrV9i\\\\\\\/OmbvRIizkAJrDnVVzKooLQUZJAeOvo8QQIL\\\\\\\/citwfzbTLIGka+bfxTauQfgqWI5ksa3aTtIntoX8RkTBCaPSWda+sPa1rP1hXyxGaQTc82zYt1uuuE+R2HSVBUwm8qH3HGk9SoOReV75JzDbDEe8SM\\\\\\\/aE4Dj97ACM\\\\\\\/rpZTyL8CT\\\\\\\/Ot9hw7C9QT39hPMe2QaLAw3yzn5OYwNABkYjDmX5Q6vCETLOuTnHNy4dyG2V4kRyg0DSQnxd8YFtBOExY0ZDCOzWNHOvyfd1moleSsjlKYUR\\\\\\\/SCPggNFSRzZhlExqH5sZ\\\\\\\/CyxkbL5JcI75xDiSqTb8J+\\\\\\\/53W+96h21AT05Y2ZEt4ndyS+prJRuzee5GhG8xMwWGZNvVPDOYPx8ApcqWzDcnQ6xXyDvEN1s4Y\\\\\\\/Pm4Mud1Rtb8EEXLMCzjHaW6N5SAYjI5JrHHbIY00aP8gU8I8ynyoZb89zALZRG8iVvCbUMRC58s7Trknnlt8Fuei5r8k36j7ZEUZ5EYGTPMUKsaUclHNjj5qF55ZKiqTKnnrvelt2WNwr9I1p1AUlT27xg8QINrFucReiZ37DqxVSH6k3AGQTX1V48Tq5shlLTwyWCtNIUWP3mfblnzpljBzQ2NxddOEsTAatE3ITi3\\\\\\\/mcQvjy9XPtQfkfo8elN2BenDUjX5LrU8Xzi8IZfdduI5APppk2HfTAql3JPAQrmVnuwsjd0vyf8x5kV\\\\\\\/Ci6Ch+TTpmzoIhjt\\\\\\\/s+Po9Lt7XZB+m0q+6\\\\\\\/b5VuaTQFDxkfkNaUqs83yioNKfsUVac\\\\\\\/YdPIWELqQqCgEMPomTw3iXE6AsJRQrJWSS\\\\\\\/PIMJ5caFDXO9qHzOmt3ZE9HicwjUTM5kxOozyvIXDf3anaeau+s6wkcKItTbM8scIxTfoYtDtDDzFLsMDJvnW7zi1tjXR8cvaXWxMIxnXaAmcjY\\\\\\\/6T9HKbxsPfPBM5gAKqbLY5o01coQ\\\\\\\/dyYy1qSuycPUWI8hb30PeDOfdDEUvhl1+uvtTDiSS3YsqeVyLlG9v0GVv6BP15buSkKV2lXkyXaw+IvsLJZegRaRJ6JXay3ncSEtaVka5XaTw1KJP21J8Ukgue2luB\\\\\\\/Lem1el3JkSz5zYndEqSWCPZdNCOQLBynE0ngoBXxDH+NNKnp8oklkNlO6+tRHrlTNNVYDfZvL2cN2BhODo6USzvx9Ey2bwQmx4c2YD\\\\\\\/SfzHJtz3OAsG+YDoqtc+kyYrWYU2A+efVI9KbaOvtUhgmd5bcBzQDVFOjn4+LJ5AvDFDvB4NMNz0j+m25QvpXEup7dAi4QqrmHqyaOjx5bicPIzU36+jliy1BJVWycR\\\\\\\/cozmwQsXMSZlfVZua8r2CicPOnAi2poWIOmcL9KgD7Y7ucPcmq1t4zx\\\\\\\/rpI1N+Zp22B8pcqVGwvKY8Zpf0niPDsxH7sC6FO7j2VaTUBdWjUvYqFmZXEdBZgrpZctz0fgyhNdxIUOWwS60EcmRDBvbngYLvVpPyupmAfm9+CobHzVyFtlQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00221caVId+b8piwxysB2PH9DQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022261a485a4dbfe744852663bd80bf088b79e4e4988d0d4423d9917a7c7c0081ac\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022I60oHP1UQkWX0K6mmMmVF3XR2++SpYYHGEiWEMkaWZoaTbMBgTJJeexloyj7sIKOZimOgLICbZCWJ07L17EqF1f20\\\\\\\/hNgkEH6uT8T8TkgE7NGjebQKL+SiioPOwloJsNmqvUCqHcf0pWYSJZ6WwWsDPqYoQq80O2nKCmyAV537CFh7QTUJlFrBSF+zim+rTeouNAiuwwPOjRQBksLVgbAJsYPMg0PVXOcZmbSpXGd6QiAfP6ttcl0Kjf8fR62XSnRrEiVLmWkkkqV6olJ6x7JZotDjKZJq9+kr+773eazGBfjW3oFUixjh47kvBwfvMIS8nudKlbzEFpVCXkmZT3MSWNzW7YMBFyKVUz+Gv2frTN2+wuxcB0X0S2bVingilNCeMWXGswU1xxjB4RxIGvtMJx7biJYluwbVEDt\\\\\\\/zo74L1K0umilYj4\\\\\\\/I8rs0b0dtNBsyxv8\\\\\\\/hnJGYRo8yUAslkcU8oKFoATqcHQ935Ayx94QmFT4QHXMqgKIvCHLSACoGUW\\\\\\\/hzBWy9V\\\\\\\/PuVGTxlxxedruAH1IfP1hEZBqBtUyJLwSTdrh+cvoLCRHLw1RREpu5AGptpqWZcnlAzb\\\\\\\/8ZfDreR38sSvU\\\\\\\/HAEG8T1VPcDipOXaSGZe1r0WHrAAE23SYkYx\\\\\\\/kFru00JM8Kh+\\\\\\\/T2RmcohpPzbDBaON4FK4ISt4GPiJNVoci9ikDU5Qg9o8zcql5nBElcjwEz57JTBvfK76b9LCYTOfz\\\\\\\/BOpV8klIGghpHY+PTVGrTyEzWJAR7mArMlLH7+R1fwt+5sj7YNUZ+q+VL8y3KF1c2MaaNUhO4z0nWAVE0ljZAyqX\\\\\\\/V59\\\\\\\/S0vCUaQ7vgMEyQF7kBbeq5Z4xWY+GZ\\\\\\\/Bl\\\\\\\/tWO3nD\\\\\\\/jwWGrzitqkq0F3SEBuElLITRMCtfu1BqlohAxqXQhUsDVfiJ306\\\\\\\/8jY54u6J1icVBYhaHQ\\\\\\\/bh74UNm+sxE6FuxOA+E2YaOkFOk8kVgQdaFn\\\\\\\/1ZG4IS792cgvyWYnHPzqFsINjVuOiH01LFxHaJ\\\\\\\/I0\\\\\\\/sB3LS0aFf4dBFjUXbGUsCil\\\\\\\/a+bCk3etQGW6KQQJlaf5ueC6ZOeazsHD12U7z6na5mHr7rPNgAhdtNq+ORLKFoxjXMyvPe3bYRWoHeBnAOvmCCD7ltjxvYOIzW6jQMFOvY+IfRlllJQVTkYz0eakSOueRbe9WyV5rrp\\\\\\\/Y6qa5pxK0H9ebMqaYkmdCcma8Xw7pJGBdefke4kAoimROBKjFEpQycOESUphb7gJiqVEIem\\\\\\\/TtQaWksIXV7hZhy1GPAf7QPbaWUAOvloDESQINUluviEGWXG3+9GtpWZiYbGtHwDTK0PVU+tqUuC5oM+eF0w5ziEX48sgQOLJJZJIhrOhf93jVbVaUipsEd6I5942UxvOJ8Iypc4bowHMy9m9taNh\\\\\\\/1PTTS8vVPX3yWlUGMVBv6+f42wFtYgBddWM66RpWvUhKuXIiV3N\\\\\\\/iFlzAMLWpgzB\\\\\\\/dQyAPXn+sFjtHis+38AQ8uVYv5TupVJDH2elKOEf7LRmYQ6nUwKe4LkWtBqnF\\\\\\\/kEHWXM3ConFqpZ8dxg7HcV6AUyc0vcK5NEAR8vJY0JyS51y\\\\\\\/MGPQIsLqP+WfLjEIMUds3183OKtHmkTAE7iMalSRwZ6ECz2JeQP73bWJOjBx5bQjiKM1u3TNr+g3ZriTkhLSIm3uVk2hyKGCNBfGR2lsTLjDvw+rGeW1r1Wt7YLKg3rLnlhcucwkSRlonJAsiYtseHtSCbtOGTRigGy8Pcmy5CfJeXjBqhrfFuhngi3ajwfGz0qaXxJU3L8gi28dUBWcw+aip56vOwlNLru63SB5Ta1YP5qlsTpST8fsXNj5HASPF5iL\\\\\\\/pqVVxqD47iDvQ0D9QN8RhydMgiYOFgT89TG2HbVG4aS60kWMtY9vTjDXbC5owUGPMDfaqysqIHf3Ps8CaYygQQ+XzKb0FXmuV7OZp0+nIxSWCylmCT1LkHuW02lp0WJFS0blUVB6U7sr1sBO39ZeJ4dnHvSGLydPwCpNCw==\\\u0022,\\\u0022iv\\\u0022:\\\u00227V4aIMGnMMYWmK16e8RbrQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002280ccf3581c616efe5f5fc1d1ac7b06e4f83b425f4f48a98d3e8f9b90dcc86474\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487813 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/79" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iGj3rckUk-l5eq2JSRvbEVBlZZ8882qK-SmLb52Y4Lg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/79" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pEHTSXalf-BZph2_SBC9rjyzgolvM2leXvkfIq8sf98" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/63" title="Genome-Wide Screen for Saccharomyces cerevisiae Genes Contributing to Opportunistic Pathogenicity in an Invertebrate Model Host" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/91" title="A Resource for the Allele-Specific Analysis of DNA Methylation at Multiple Genomically Imprinted Loci in Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487813" id="node487813" data-pisa="ggg;8/1/79" data-pisa-master="ggg;g3.117.300257" data-apath="/ggg/8/1/79.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Large-Scale Low-Cost NGS Library Preparation Using a Robust Tn5 Purification and Tagmentation Protocol</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bianca P. Hennig</span>, <span class="highwire-citation-author" data-delta="1">Lars Velten</span>, <span class="highwire-citation-author" data-delta="2">Ines Racke</span>, <span class="highwire-citation-author" data-delta="3">Chelsea Szu Tu</span>, <span class="highwire-citation-author" data-delta="4">Matthias Thoms</span>, <span class="highwire-citation-author" data-delta="5">Vladimir Rybin</span>, <span class="highwire-citation-author" data-delta="6">Hüseyin Besir</span>, <span class="highwire-citation-author" data-delta="7">Kim Remans</span> and <span class="highwire-citation-author" data-delta="8">Lars M. Steinmetz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">79-89;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300257 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487813" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bianca P. Hennig </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bianca%2BP.%2BHennig%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hennig%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABianca%2BP.%2BHennig%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lars Velten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BVelten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Velten%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALars%2BVelten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ines Racke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Protein Expression and Purification Core Facility, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ines%2BRacke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Racke%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AInes%2BRacke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chelsea Szu Tu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chelsea%2BSzu%2BTu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tu%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChelsea%2BSzu%2BTu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Matthias Thoms </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biochemistry Center Heidelberg, Heidelberg University, 69120, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthias%2BThoms%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thoms%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthias%2BThoms%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Vladimir Rybin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Protein Expression and Purification Core Facility, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vladimir%2BRybin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rybin%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVladimir%2BRybin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Hüseyin Besir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Protein Expression and Purification Core Facility, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H%C3%BCseyin%2BBesir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Besir%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH%25C3%25BCseyin%2BBesir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kim Remans </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Protein Expression and Purification Core Facility, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kim%2BRemans%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Remans%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKim%2BRemans%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kim.remans@embl.de" class="" data-icon-position="" data-hide-link-title="0">kim.remans@embl.de</a>
<a href="mailto:lars.steinmetz@embl.de" class="" data-icon-position="" data-hide-link-title="0">lars.steinmetz@embl.de</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Lars M. Steinmetz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, Stanford University School of Medicine, California 94305</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Stanford Genome Technology Center, Stanford University School of Medicine, Palo Alto, California 94304</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BM.%2BSteinmetz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steinmetz%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALars%2BM.%2BSteinmetz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kim.remans@embl.de" class="" data-icon-position="" data-hide-link-title="0">kim.remans@embl.de</a>
<a href="mailto:lars.steinmetz@embl.de" class="" data-icon-position="" data-hide-link-title="0">lars.steinmetz@embl.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/1/79" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487813" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487813/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/79.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487813" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487813/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/79.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487813" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487813/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/79.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487813" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487813/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/1/79.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487813" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487813/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Efficient preparation of high-quality sequencing libraries that well represent the biological sample is a key step for using next-generation sequencing in research. Tn5 enables fast, robust, and highly efficient processing of limited input material while scaling to the parallel processing of hundreds of samples. Here, we present a robust Tn5 transposase purification strategy based on an N-terminal His<sub>6</sub>-Sumo3 tag. We demonstrate that libraries prepared with our in-house Tn5 are of the same quality as those processed with a commercially available kit (Nextera XT), while they dramatically reduce the cost of large-scale experiments. We introduce improved purification strategies for two versions of the Tn5 enzyme. The first version carries the previously reported point mutations E54K and L372P, and stably produces libraries of constant fragment size distribution, even if the Tn5-to-input molecule ratio varies. The second Tn5 construct carries an additional point mutation (R27S) in the DNA-binding domain. This construct allows for adjustment of the fragment size distribution based on enzyme concentration during tagmentation, a feature that opens new opportunities for use of Tn5 in customized experimental designs. We demonstrate the versatility of our Tn5 enzymes in different experimental settings, including a novel single-cell polyadenylation site mapping protocol as well as ultralow input DNA sequencing.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/tn5" class="hw-term hw-article-keyword hw-article-keyword-tn5" rel="nofollow">Tn5</a></li><li class="kwd"><a href="/keyword/tagmentation" class="hw-term hw-article-keyword hw-article-keyword-tagmentation" rel="nofollow">tagmentation</a></li><li class="kwd"><a href="/keyword/single-cell-0" class="hw-term hw-article-keyword hw-article-keyword-single-cell" rel="nofollow">single cell</a></li></ul><p id="p-3">Next generation sequencing (NGS) methods have revolutionized biology by providing detailed insights into genome organization, the regulation of gene expression, and genetic variability. However, recent studies have revealed that studying the transcriptomes and epigenomes of entire cell populations can mask tissue complexity. In this respect, single-cell approaches have led to the discovery and characterization of multiple new cell types. In parallel, clinicians are calling for a paradigm shift toward personalized medicine based on information from each patient’s genome (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Esplin <em>et al.</em> 2014</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Giladi and Amit 2017</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ashley 2016</a>). However, both developments necessitate the profiling of hundreds to thousands of samples (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Velten <em>et al.</em> 2017</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Buenrostro <em>et al.</em> 2015</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ramskold <em>et al.</em> 2012</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Jaitin <em>et al.</em> 2014</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Paul <em>et al.</em> 2015</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Nagano <em>et al.</em> 2013</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Grun <em>et al.</em> 2015</a>).</p><p id="p-4">The most rapid and scalable NGS library preparation strategy available to date is based on a hyperactive version of the Tn5 transposase (Illumina; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adey <em>et al.</em> 2010</a>). The corresponding wild-type transposable element is present in bacteria and contains two nearly identical but inverted elements, IS50L and IS50R, with IS50R encoding the Tn5 transposase [reviewed in <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Reznikoff (2008)</a>]. Tn5 catalyzes the translocation of transposable elements by a “cut-and-paste” mechanism that relies on a Tn5-homodimer binding to the outside ends (OE), which are two identical 19 bp inverted repeat sequences present at the outer sides of IS50 elements (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Johnson and Reznikoff 1983</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Sasakawa <em>et al.</em> 1983</a>). Notably, the target integration locus is selected with only very modest genomic sequence motif requirements, <em>i.e.</em>, a mild tendency toward GC-rich regions (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Shevchenko <em>et al.</em> 2002</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Reznikoff 2008</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Goryshin <em>et al.</em> 1998</a>).</p><p id="p-5">The efficiency of this process and the activity of the Tn5 transposase are regulated on many levels, resulting in a rather low transposition frequency <em>in vivo</em>. However, extensive studies over many years have resulted in the generation of a hyperactive version of Tn5 by introducing three point mutations—E54K, M56A, and L372P—where both the N-terminal M56A as well as the C-terminal L372P mutations reduce inhibitory effects on the activity of Tn5 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Goryshin and Reznikoff 1998</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Weinreich <em>et al.</em> 1994</a>). In parallel, the E54K mutation increases the DNA-binding efficiency of Tn5 to the OE (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Zhou and Reznikoff 1997</a>). Furthermore, Zhou and colleagues have reported increased binding efficiency of the E54K mutant to a mosaic sequence of the natural transposon OE and inside end elements, the mosaic end (ME) element, <em>in vitro</em> (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Zhou <em>et al.</em> 1998</a>).</p><p id="p-6">Its relatively simple mode of action raised interest in the scientific community in using Tn5 as a molecular tool for NGS library preparation. Illumina developed the tagmentation protocol, in which a modified Tn5 enzyme cuts double-stranded DNA and concurrently ligates the linker sequences that are required for Illumina sequencing to both ends. Due to its high efficiency, the low input concentration needed, and its limited hands-on time, this tagmentation protocol not only meets the requirements of NGS library construction, but also for the processing of thousands of samples in parallel (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Adey <em>et al.</em> 2010</a>).</p><p id="p-7">Recently, the Sandberg laboratory published a protocol describing the purification of hyperactive Tn5 and subsequent tagmentation-based library preparation (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Picelli <em>et al.</em> 2014a</a>). In this study, the authors used a Tn5 construct containing the E54K and L372P mutations previously described by <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Martínez-García <em>et al.</em> (2011)</a>. They loaded the enzyme with a synthetic oligonucleotide containing the double-stranded 19-bp ME sequence and a single-stranded 5′ overhang. Tn5 bound to the 19 bp ME region of the oligonucleotide and ligated it to the input DNA on both sides of the cut site. The single-stranded 5′ ends served as templates for index adapter primers during PCR-based NGS library preparation. When using this protocol, we encountered difficulties in purifying the Tn5-intein-CBD fusion construct and failed to reproducibly obtain libraries for NGS. We therefore attempted to improve the Tn5 purification procedure.</p><p id="p-8">Here, we report a revised and improved Tn5 purification protocol that is reproducible across institutes. Our robust tagmentation protocol uses solely homemade or inexpensive commercially available reagents and reduces the costs of tagmentation-based library generation by up to 70-fold compared to the commercially available kit. We have validated the quality of the NGS libraries processed with our in-house Tn5 enzymes and demonstrated their suitability for various experimental designs, including single-cell full-length RNA-seq, Drop-Seq (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Macosko <em>et al.</em> 2015</a>), and ultralow input sequencing. We also report a novel single-cell polyadenylation site mapping protocol based on custom tagmentation linkers. By comparing two Tn5 molecules that reproducibly generate high-quality NGS libraries, but vary in their characteristic features, we pave the way toward further development of novel applications for Tn5.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-9">Detailed protocols are provided in the Supplemental Material, Supplemental Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a>. All oligonucleotides used in this study were ordered at HPLC grade from Sigma Aldrich, Germany. Index oligonucleotide sequences were adapted from Illumina: Oligonucleotide sequences © 2007–2013 Illumina, Inc. All rights reserved.</p><div id="sec-2" class="subsection"><h3>Subcloning of the hyperactive Tn5 allele into pETM11-Sumo3</h3><p id="p-10">The pTXB1 plasmid carrying the Tn5-intein-CBD fusion construct with the hyperactive Tn5 allele containing the E54K and L372P mutations was a generous gift from the Rickard Sandberg lab. This plasmid was used as template to PCR-amplify the hyperactive Tn5 allele. We replaced the catalytic cysteine of the GyrA intein protein after the terminal Tn5 isoleucine, introduced by <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Picelli <em>et al.</em> (2014a)</a>, with the stop codon TAA. The PCR product was <em>Bam</em>HI- and <em>Hin</em>dIII-digested and cloned into the pETM11-Sumo3 vector (European Molecular Biology Laboratory [EMBL], available upon request). This cloning strategy introduced an additional serine residue between the Sumo3 protein and the Tn5 construct, and resulted in a His<sub>6</sub>-Sumo3-Tn5 fusion protein. The coding sequence of the Tn5 transposase was validated via Sanger sequencing.</p></div><div id="sec-3" class="subsection"><h3>Expression and purification of homemade Tn5</h3><p id="p-11">The pETM11-Sumo3-Tn5 plasmid was transformed into <em>Escherichia coli</em> BL21(DE3) codon + RIL cells (Stratagene). Cells were grown in LB supplemented with kanamycin and chloramphenicol at 37° until OD<sub>600</sub> ∼0.5. The temperature was then lowered to 18° and expression of Tn5 was induced by the addition of 0.2 mM IPTG. The cells were then grown overnight at 18° and harvested by centrifugation. The cell pellet was resuspended in running buffer (20 mM HEPES-NaOH pH 7.2, 800 mM NaCl, 20 mM imidazole, 1 mM EDTA, 2 mM DTT, and 10% glycerol) supplemented with cOmplete protease inhibitors (Roche) and lysed via sonication. Polyethyleneimine (PEI) pH 7.2 was added dropwise to a final concentration of 1% to remove nucleic acids. The cleared lysate was loaded onto a cOmplete His-tag purification column (Roche) and the His<sub>6</sub>-Sumo3-Tn5 was eluted in running buffer containing 300 mM imidazole. To remove the fusion tag, His<sub>6</sub>-tagged SenP2 protease was added to the elution fractions. The sample was digested overnight at 4° while being dialyzed back to running buffer. The next morning, the dialyzed sample was loaded again onto a cOmplete His-tag purification column (Roche) and the untagged Tn5 was collected in the flow through. The concentrated flow through was then loaded onto a Superdex200 Increase 10/300 GL column (GE Healthcare) equilibrated with 50 mM Tris pH 7.5, 800 mM NaCl, 0.2 mM EDTA, 2 mM DTT, and 10% glycerol. Elution fractions corresponding to the Tn5 dimer peak were pooled and aliquoted. The identity of both Tn5 variants was verified by mass spectrometry (Figure S1 and Supplemental Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a>). Final Tn5 samples were stored at −20° in 25 mM Tris pH 7.5, 800 mM NaCl, 0.1 mM EDTA, 1 mM DTT, and 50% glycerol.</p></div><div id="sec-4" class="subsection"><h3>Isothermal titration calorimetry (ITC) experiments</h3><p id="p-12">Binding of linker DNA to Tn5<sub>E54K,L372P</sub> and Tn5<sub>R27S,E54K,L372P</sub> was studied in ITC buffer (50 mM Tris pH 7.5, 800 mM NaCl, 0.1 mM EDTA, and 5% glycerol) using a MicroCal ITC<sub>200</sub> System (MicroCal). All proteins and DNA samples were dialyzed overnight against ITC buffer at 4°. Titration was performed at 25° by stepwise addition of 100 µM DNA solution to the 4 µM protein solution in the cell. ITC data were corrected for dilution heat and analyzed using the MicroCal Origin software package.</p></div><div id="sec-5" class="subsection"><h3>Loading of in-house-produced Tn5</h3><p id="p-13">Tn5 was loaded with the previously reported linker oligonucleotides Tn5ME-A/Tn5MErev and Tn5ME-B/Tn5MErev (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Picelli <em>et al.</em> 2014a</a>) by mixing equal amounts of each linker with Tn5 and incubating at 23° for 30–60 min. The loaded Tn5 was ready to use without removal of unbound linkers and dilutions were performed in nuclease-free water. Alternatively, loaded Tn5 can be supplemented with glycerol at a final concentration of 50% and stored at −20° for several days.</p></div><div id="sec-6" class="subsection"><h3>Tagmentation-based library preparation for duplex index full-length RNA sequencing using in-house-produced Tn5</h3><p id="p-14">Tagmentation was performed in 10 mM Tris-HCl pH 7.5 (Sigma Aldrich), 10 mM MgCl<sub>2</sub> (Sigma Aldrich), and 25% dimethylformamide (DMF) (Sigma Aldrich) using 100–200 pg cDNA. Excellent fragmentation of cDNA was achieved using Tn5 at a concentration of 20–40 ng/μl. Reactions were incubated at 55° for 3 min in a preheated thermocycler and either stopped with 0.2% SDS or heat inactivation (see Figure S6A and Supplemental Methods in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a>).</p><p id="p-15">PCR amplification was performed using the KAPA HiFi HotStart polymerase or KAPA HiFi HotStart ReadyMix (see Figure S6B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a>). Residual dNTPs, oligonucleotides, and polymerase were removed by subsequent bead purification using 1 vol of AMPureXP beads (Beckmann) following the manufacturer’s instructions. Final libraries were eluted with 10 μl nuclease-free water.</p><p id="p-16">Tagmentation of genomic DNA was performed as described above. Note that the number of PCR cycles was increased when processing input material ≤0.5 pg/μl.</p></div><div id="sec-7" class="subsection"><h3>Single-cell polyadenylation site mapping</h3><div id="sec-8" class="subsection"><h4>cDNA preparation:</h4><p id="p-17">In this study, we performed all tagmentation experiments with cDNA obtained from extracted HeLa RNA processed with the previously described Smart-seq2 protocol (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Picelli <em>et al.</em> 2013</a>, <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">2014b</a>) with the following modifications: we modified the oligo dT primer by introducing a four-nucleotide end identifier and an eight-nucleotide specific cell barcode. The end identifier was introduced only in the oligo dT but not in the template-switching primer (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Picelli <em>et al.</em> 2014b</a>), and thus allowed for specific amplification of the 3′ end of the cDNA without interfering with the common primer sequence present in both the oligo dT and template switching primer, which are needed for amplification of the cDNA library.</p><p id="p-18">cDNA was processed from 15 pg extracted HeLa RNA, mimicking the RNA amount of a single cell. The amplified cDNA was purified using 1 vol of AMPureXP beads following the manufacturer’s instructions, then eluted in 15 μl nuclease-free water and stored at −20°. cDNA was quantified using the Qubit HS dsDNA (Invitrogen) and the size distribution and quality of the cDNA were analyzed with an Agilent High Sensitivity DNA kit and an Agilent 2100 Bioanalyzer following the manufacturer’s instructions.</p></div><div id="sec-9" class="subsection"><h4>Tagmentation:</h4><p id="p-19">To obtain 3′ RNA-specific libraries, we essentially used the above-described Tn5 loading and tagmentation protocol with the following changes: undiluted Tn5 stock was loaded exclusively with the Tn5ME-B/Tn5MErev linker. Subsequent PCR enrichment was performed using KAPA HiFi HotStart polymerase (Kapa Biosystems) supplemented with TMAC, i7 adapter index primer, and a custom PE1.Smart-seq2 primer. Importantly, we skipped the gap-filling step in the PCR program to prevent amplification of tagmented cDNA from the gene-coding region.</p></div></div><div id="sec-10" class="subsection"><h3>Tagmentation using the Nextera XT DNA library preparation kit</h3><p id="p-20">Tagmentation using the Nextera XT DNA library preparation kit (Illumina) was performed following the manufacturer’s instructions. Purification of samples was performed using an identical volume of AMPureXP beads, according to the manufacturer’s instructions.</p></div><div id="sec-11" class="subsection"><h3>Multiplexing and NGS of tagmented samples</h3><p id="p-21">Samples were multiplexed for sequencing by mixing equal concentrations of each library. The resulting volume was determined and an identical volume of AMPureXP beads was added. Purification was performed in accordance with the manufacturer’s instructions and the multiplexed library was eluted in 50 μl nuclease-free water. For size selection, 0.6 vol of AMPureXP were added to the sample and incubated at room temperature for 5 min, followed by separation of large fragments (beads) and small fragments (supernatant) using a magnet. The supernatant was transferred into a new tube (beads were discarded), and mixed with 0.2 vol of AMPureXP (with respect to the supernatant). Purification of the multiplexed library was performed according to the manufacturer’s instructions. The final library was eluted in 12 μl nuclease-free water. Samples were quantified using Qubit HS dsDNA. Quality control was performed using an Agilent 2100 Bioanalyzer and High Sensitivity DNA kit.</p><p id="p-22">All libraries were sequenced on NextSeq platforms. Full-length RNA libraries were sequenced using the 75 bp paired end option with 75 cycles each from both the P5 and P7 sites using Illumina sequencing primers. Libraries were sequenced with either 75 cycles each from each end of the fragment (resulting in sequencing reads of 75 nt) or as an asymmetric run with all cycles from P7 (resulting in sequencing reads of 150 nt). Using the double indexing strategy, we obtained information about both i5 and i7 Illumina indices (each index has a length of 8 nt).</p><p id="p-23">To map polyadenylation sites in single cells, 3′ RNA libraries were sequenced using the 150 bp paired end option as an asymmetric run. Read 1 with 14 cycles resulted in 14 nt long sequencing reads providing the cell barcode (6 nt) and UMI (8 nt, information not used), and was performed from the P5 site using a custom sequencing primer [poly(A)-seq primer, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a>]. Read 2 was performed from the P7 site using the Illumina P7 sequencing primer with the remaining ∼290 cycles to identify the gene, resulting in sequencing reads of ∼290 nt. Read 2 was supposed to reach the polyadenylation site but was excluded from the analysis if the polyadenylation site was missing. A dedicated index read provided the i7 barcode (8 nt) and was used for demultiplexing into sample batches.</p><p id="p-24">At least two technical replicates were sequenced per condition.</p></div><div id="sec-12" class="subsection"><h3>Data analysis</h3><p id="p-25">Several batches of cDNA obtained from independent Smart-seq2 replicates were used in this study. All comparisons between Tn5 variants, Tn5 batches, and treatments were done on data obtained from an identical cDNA batch for each figure, unless stated differently.</p><p id="p-26">For libraries created with the double indexing strategy, raw sequencing data were processed as described (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Velten <em>et al.</em> 2017</a>). In short, reads were demultiplexed using index read information and sequencing reads extending into the poly(A) tail were trimmed. Subsequently, all reads were aligned to the hg19 version of the human genome using GSNAP (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Wu and Nacu 2010</a>), and reads falling on annotated exons were counted using HTSeq (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anders <em>et al.</em> 2015</a>). For polyadenylation site mapping libraries, data were processed as described (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Velten <em>et al.</em> 2017</a>), with small modifications. In short, samples were demultiplexed using the i7 index read and the first six bases of the first read. The first read was then discarded and all further processing was done on the second read. Reads extending into the poly(A) tail were trimmed and aligned to the hg19 version of the human genome using GSNAP, and alignments were filtered as previously described. We then created a table of read counts per genomic coordinate using a custom script based on HTSeq and call polyadenylation sites, as previously described.</p><p id="p-27">All correlation coefficients shown are Pearson’s product–moment correlation coefficients. Clustering was performed using a correlation-based distance metric and complete linkage. The number of genes detected in NGS libraries was based on a minimum of five reads per barcode.</p><p id="p-28">All scripts used for data analysis are available on GitHub (<a href="https://git.embl.de/velten/misc-pub/blob/master/HennigEtAl.R">https://git.embl.de/velten/misc-pub/blob/master/HennigEtAl.R</a>).</p></div><div id="sec-13" class="subsection"><h3>Data availability</h3><p id="p-29">The pETM11-Sumo3-Tn5 plasmids carrying either the Tn5<sub>E54K,L372P</sub> or Tn5<sub>R27S,E54K,L372P</sub> allele are available upon request. The plasmid expressing the His<sub>6</sub>-tagged SenP2 protease is available upon request.</p><p id="p-30">The data discussed in this manuscript have been deposited in NCBI’s Gene Expression Omnibus (GEO) and are accessible through GEO Series accession number GSE101520 (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE101520">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE101520</a>).</p><p id="p-31"><a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a> includes Figures S1–S6, Table S1 which lists all primers used in this study, detailed protocols for mass spectrometry measurements, expression and purification of the Tn5 enzymes, as well as loading of the Tn5, tagmentation reaction, and subsequent PCR enrichment.</p></div></div><div class="section results" id="sec-14"><h2 class="">Results</h2><div id="sec-15" class="subsection"><h3>Expression and purification of Tn5 transposase</h3><p id="p-32">In a previous study, Tn5 transposase was expressed from the pTXB1-Tn5 plasmid and purified via a C-terminal intein-CBP tag (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Picelli <em>et al.</em> 2014a</a>). We aimed to produce the Tn5 in-house following this protocol; however, this purification procedure proved to be intricate and we only obtained very low yields of the enzyme. Furthermore, the enzyme showed low activity and we failed to reproducibly obtain libraries when processing cDNA with this transposase. Therefore, we developed a new Tn5 purification strategy using the N-terminal His<sub>6</sub>-Sumo3 tag, which is a more common and straight forward protein purification method than using the intein tag. This is a very common and straight forward protein purification method that is likely to work robustly in most biochemistry laboratories. The hyperactive Tn5 allele carrying the E54K and L372P mutations, further referred to as Tn5<sub>E54K,L372P</sub>, was PCR-amplified from the pTXB1-Tn5 plasmid (<a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Picelli <em>et al.</em> 2014a</a>) and subcloned into the pETM11-Sumo3 vector (EMBL), resulting in the His<sub>6</sub>-Sumo3-Tn5 fusion construct (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). The Tn5 sequence was validated via Sanger sequencing. One of the clones carried an additional point mutation in the DNA-binding domain that resulted in an amino acid substitution from arginine to serine (R27S), referred to as Tn5<sub>R27S,E54K,L372P</sub> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>). It is likely that this substitution was introduced during PCR amplification. The arginine is located in the DNA-binding domain of the Tn5 transposase and is involved in the binding of the linker oligonucleotide (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Davies <em>et al.</em> 2000</a>). Since the arginine to serine mutation leads to a loss of the positive charge, we hypothesized that the mutation might have an influence on the enzyme’s activity. In the following sections, we will compare the performance of both Tn5 enzymes during tagmentation-based library preparation.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/79/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Purification of the His6-Sumo3-Tn5 construct. (A) Plasmid map of pETM11-Sumo3-Tn5 encoding the His6-Sumo3-Tn5 fusion protein. The locations of the point mutations in the Tn5 coding region (R27S, E54K, and L372P) are indicated. (B) SDS-PAGE analysis of the two His6-Sumo3 purified Tn5 proteins after size exclusion chromatography. The single band runs at a molecular weight of &#x223C;53 kDa, corresponding to the Tn5 monomer. (C) Size exclusion chromatography (Superdex200 Increase 10/300 GL) profile of the purified Tn5R27S,E54K,L372P transposase. The Tn5 dimer peak elutes at 12.9 ml, while the Tn5 aggregate elutes at 8.7 ml. (D) Size exclusion chromatography (Superdex200 Increase 10/300 GL) profile of the purified Tn5E54K,L372P transposase. The Tn5 dimer peak is the main peak and elutes at 13.2 ml. MW, molecular weight; SDS-PAGE, sodium dodecyl sulfate-polyacrylamide gel electrophoresis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-783931227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Purification of the His&lt;sub&gt;6&lt;/sub&gt;-Sumo3-Tn5 construct. (A) Plasmid map of pETM11-Sumo3-Tn5 encoding the His&lt;sub&gt;6&lt;/sub&gt;-Sumo3-Tn5 fusion protein. The locations of the point mutations in the Tn5 coding region (R27S, E54K, and L372P) are indicated. (B) SDS-PAGE analysis of the two His&lt;sub&gt;6&lt;/sub&gt;-Sumo3 purified Tn5 proteins after size exclusion chromatography. The single band runs at a molecular weight of &#x223C;53 kDa, corresponding to the Tn5 monomer. (C) Size exclusion chromatography (Superdex200 Increase 10/300 GL) profile of the purified Tn5&lt;sub&gt;R27S,E54K,L372P&lt;/sub&gt; transposase. The Tn5 dimer peak elutes at 12.9 ml, while the Tn5 aggregate elutes at 8.7 ml. (D) Size exclusion chromatography (Superdex200 Increase 10/300 GL) profile of the purified Tn5&lt;sub&gt;E54K,L372P&lt;/sub&gt; transposase. The Tn5 dimer peak is the main peak and elutes at 13.2 ml. MW, molecular weight; SDS-PAGE, sodium dodecyl sulfate-polyacrylamide gel electrophoresis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/79/F1.medium.gif" width="440" height="363"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/1/79/F1.medium.gif" width="440" height="363"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/79/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/79/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488054" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-33" class="first-child">Purification of the His<sub>6</sub>-Sumo3-Tn5 construct. (A) Plasmid map of pETM11-Sumo3-Tn5 encoding the His<sub>6</sub>-Sumo3-Tn5 fusion protein. The locations of the point mutations in the Tn5 coding region (R27S, E54K, and L372P) are indicated. (B) SDS-PAGE analysis of the two His<sub>6</sub>-Sumo3 purified Tn5 proteins after size exclusion chromatography. The single band runs at a molecular weight of ∼53 kDa, corresponding to the Tn5 monomer. (C) Size exclusion chromatography (Superdex200 Increase 10/300 GL) profile of the purified Tn5<sub>R27S,E54K,L372P</sub> transposase. The Tn5 dimer peak elutes at 12.9 ml, while the Tn5 aggregate elutes at 8.7 ml. (D) Size exclusion chromatography (Superdex200 Increase 10/300 GL) profile of the purified Tn5<sub>E54K,L372P</sub> transposase. The Tn5 dimer peak is the main peak and elutes at 13.2 ml. MW, molecular weight; SDS-PAGE, sodium dodecyl sulfate-polyacrylamide gel electrophoresis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-34">The His<sub>6</sub>-Sumo3-Tn5 fusion protein was expressed in <em>E. coli</em> BL21(DE3) codon + RIL cells. <em>E. coli</em> DNA bound to the transposase was removed via PEI precipitation. After His-tag purification and cleaving of the His<sub>6</sub>-Sumo3 fusion tag, the untagged Tn5 was loaded on a size exclusion chromatography column to obtain a pure Tn5 dimer (Tn5 monomer ∼53 kDa, <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1, B–D</a>) and stored at −20°.</p></div><div id="sec-16" class="subsection"><h3>Initial activity tests with the homemade Tn5 enzymes</h3><p id="p-35"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a> illustrates the tagmentation-based library preparation workflow using purified Tn5 constructs. Prior to tagmentation, the Tn5 enzyme is loaded with linker oligonucleotides containing the double-stranded 19 bp ME sequence and a single-stranded 5′ overhang (<a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Picelli <em>et al.</em> 2014a</a>). In a first attempt to test the activity of Tn5<sub>E54K,L372P</sub> and Tn5<sub>R27S,E54K,L372P</sub>, we performed tagmentation reactions with reagents supplied in the Illumina Nextera XT DNA library preparation kit following the manufacturer’s instructions, but substituting the Nextera ATM enzyme with one of the two loaded in-house-produced Tn5 versions. As we did not know the activity range of these transposases, we performed several tagmentation reactions simultaneously with different dilutions of Tn5 ranging from 3 to 30 ng/μl, while keeping the cDNA concentrations constant (150–200 pg per reaction). Using Tn5<sub>E54K,L372P</sub>, we obtained libraries with an average fragment size of roughly 340 bp with all Tn5 concentrations used (Figure S2A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a>). Fragment size distribution was similar when tagmenting with 30 ng/μl Tn5<sub>R27S,E54K,L372P</sub>, while higher dilutions of the enzyme with constant input cDNA concentrations resulted in larger fragment sizes (Figure S2B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/79/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="NGS-library preparation using the homemade Tn5 constructs. (A) Workflow of Tn5 loading, cDNA tagmentation, and subsequent NGS library preparation for duplex index (Illumina i5/i7) full-length cDNA sequencing. Tn5 molecules are shown as gray hexamers. The double-stranded part of the linker oligonucleotide, the mosaic element, is shown in gray with a yellow circle depicting the phosphorylated 3&#x2032; end. The 5&#x2032; overhangs as templates for the i5 or i7 index adapter primers are shown in red or blue, respectively. cDNA is shown as two parallel black lines with a 3&#x2032; poly(A) tail. The synthesis of the 5&#x2032; overhang complementary strand (gap-filling step during PCR amplification) is depicted as a dotted arrow. i5 index adapter primer is shown in dark blue, while i7 index adapter primer is shown in orange. Fragments that are lost during library preparation are transparent. (B) Bioanalyzer traces of NGS libraries processed with different concentrations of the in-house-produced Tn5R27S,E54K,L372P using only homemade or inexpensive commercially available reagents for tagmentation and subsequent PCR reaction. Following the tagmentation protocol presented here, fragmentation of the cDNA works best when using Tn5R27S,E54K,L372P at a concentration of 30 ng/&#x3BC;l. (C) Heat scatter plot showing the correlation of read counts between libraries processed with either in-house-produced Tn5E54K,L372P or Tn5R27S,E54K,L372P. Data of three technical replicates per condition were pooled for this analysis. (D) Heat scatter plot showing the correlation of gene counts between libraries processed using either in-house-produced Tn5R27S,E54K,L372P and the protocol presented here or the Nextera XT DNA library preparation kit following the manufacturer&#x2019;s instructions. Data of three technical replicates per condition were pooled for this analysis. (E) Pairwise correlation of read counts between three technical replicates (samples processed from the same cDNA on the same day) when using homemade Tn5R27S,E54K,L372P and the tagmentation protocol presented here. The Pearson correlation of r = 0.99 between all samples demonstrates high reproducibility of both the enzyme and the protocol. (F) Heat map analysis of gene counts in technical replicates processed from the same cDNA but on different days. The color code indicates the Pearson correlation between samples (see legend on the right side). NGS, next-generation sequencing; PCR, polymerase chain reaction; SDS, sodium dodecyl sulfate." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-783931227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;NGS-library preparation using the homemade Tn5 constructs. (A) Workflow of Tn5 loading, cDNA tagmentation, and subsequent NGS library preparation for duplex index (Illumina i5/i7) full-length cDNA sequencing. Tn5 molecules are shown as gray hexamers. The double-stranded part of the linker oligonucleotide, the mosaic element, is shown in gray with a yellow circle depicting the phosphorylated 3&#x2032; end. The 5&#x2032; overhangs as templates for the i5 or i7 index adapter primers are shown in red or blue, respectively. cDNA is shown as two parallel black lines with a 3&#x2032; poly(A) tail. The synthesis of the 5&#x2032; overhang complementary strand (gap-filling step during PCR amplification) is depicted as a dotted arrow. i5 index adapter primer is shown in dark blue, while i7 index adapter primer is shown in orange. Fragments that are lost during library preparation are transparent. (B) Bioanalyzer traces of NGS libraries processed with different concentrations of the in-house-produced Tn5&lt;sub&gt;R27S,E54K,L372P&lt;/sub&gt; using only homemade or inexpensive commercially available reagents for tagmentation and subsequent PCR reaction. Following the tagmentation protocol presented here, fragmentation of the cDNA works best when using Tn5&lt;sub&gt;R27S,E54K,L372P&lt;/sub&gt; at a concentration of 30 ng/&#x3BC;l. (C) Heat scatter plot showing the correlation of read counts between libraries processed with either in-house-produced Tn5&lt;sub&gt;E54K,L372P&lt;/sub&gt; or Tn5&lt;sub&gt;R27S,E54K,L372P&lt;/sub&gt;. Data of three technical replicates per condition were pooled for this analysis. (D) Heat scatter plot showing the correlation of gene counts between libraries processed using either in-house-produced Tn5&lt;sub&gt;R27S,E54K,L372P&lt;/sub&gt; and the protocol presented here or the Nextera XT DNA library preparation kit following the manufacturer&#x2019;s instructions. Data of three technical replicates per condition were pooled for this analysis. (E) Pairwise correlation of read counts between three technical replicates (samples processed from the same cDNA on the same day) when using homemade Tn5&lt;sub&gt;R27S,E54K,L372P&lt;/sub&gt; and the tagmentation protocol presented here. The Pearson correlation of &lt;em&gt;r&lt;/em&gt; = 0.99 between all samples demonstrates high reproducibility of both the enzyme and the protocol. (F) Heat map analysis of gene counts in technical replicates processed from the same cDNA but on different days. The color code indicates the Pearson correlation between samples (see legend on the right side). NGS, next-generation sequencing; PCR, polymerase chain reaction; SDS, sodium dodecyl sulfate.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/79/F2.medium.gif" width="351" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/1/79/F2.medium.gif" width="351" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/79/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/79/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487846" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-36" class="first-child">NGS-library preparation using the homemade Tn5 constructs. (A) Workflow of Tn5 loading, cDNA tagmentation, and subsequent NGS library preparation for duplex index (Illumina i5/i7) full-length cDNA sequencing. Tn5 molecules are shown as gray hexamers. The double-stranded part of the linker oligonucleotide, the mosaic element, is shown in gray with a yellow circle depicting the phosphorylated 3′ end. The 5′ overhangs as templates for the i5 or i7 index adapter primers are shown in red or blue, respectively. cDNA is shown as two parallel black lines with a 3′ poly(A) tail. The synthesis of the 5′ overhang complementary strand (gap-filling step during PCR amplification) is depicted as a dotted arrow. i5 index adapter primer is shown in dark blue, while i7 index adapter primer is shown in orange. Fragments that are lost during library preparation are transparent. (B) Bioanalyzer traces of NGS libraries processed with different concentrations of the in-house-produced Tn5<sub>R27S,E54K,L372P</sub> using only homemade or inexpensive commercially available reagents for tagmentation and subsequent PCR reaction. Following the tagmentation protocol presented here, fragmentation of the cDNA works best when using Tn5<sub>R27S,E54K,L372P</sub> at a concentration of 30 ng/μl. (C) Heat scatter plot showing the correlation of read counts between libraries processed with either in-house-produced Tn5<sub>E54K,L372P</sub> or Tn5<sub>R27S,E54K,L372P</sub>. Data of three technical replicates per condition were pooled for this analysis. (D) Heat scatter plot showing the correlation of gene counts between libraries processed using either in-house-produced Tn5<sub>R27S,E54K,L372P</sub> and the protocol presented here or the Nextera XT DNA library preparation kit following the manufacturer’s instructions. Data of three technical replicates per condition were pooled for this analysis. (E) Pairwise correlation of read counts between three technical replicates (samples processed from the same cDNA on the same day) when using homemade Tn5<sub>R27S,E54K,L372P</sub> and the tagmentation protocol presented here. The Pearson correlation of <em>r</em> = 0.99 between all samples demonstrates high reproducibility of both the enzyme and the protocol. (F) Heat map analysis of gene counts in technical replicates processed from the same cDNA but on different days. The color code indicates the Pearson correlation between samples (see legend on the right side). NGS, next-generation sequencing; PCR, polymerase chain reaction; SDS, sodium dodecyl sulfate.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">Overall, this initial experiment demonstrated that both purified Tn5 constructs are active and produce libraries suitable for sequencing from small amounts of input material. Interestingly, the hyperactive Tn5 version carrying the additional R27S mutation showed higher sensitivity toward the ratio of Tn5-to-cDNA molecules than the regular hyperactive Tn5<sub>E54K,L372P</sub> enzyme.</p></div><div id="sec-17" class="subsection"><h3>A robust tagmentation-based NGS library preparation protocol</h3><p id="p-38">In order to develop a tagmentation protocol that is completely independent of the Nextera XT DNA library preparation kit, we substituted all Nextera reagents with homemade or inexpensive commercial reagents in a stepwise manner. We found that a Tris-based tagmentation buffer at a final concentration of 10 mM Tris-HCl, 10 mM MgCl<sub>2</sub>, and 25% DMF resulted in robust and reproducible tagmentation of input cDNA. We used SDS inactivation and KAPA HiFi HotStart DNA polymerase (Kapa Biosystems) for subsequent PCR enrichment of the tagmented input cDNA to generate NGS libraries.</p><p id="p-39">Using these conditions, we observed stable fragment size distribution of cDNA with a wide range of tested Tn5<sub>E54K,L372P</sub> concentrations (3–30 ng/μl, Figure S3A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a>). For Tn5<sub>R27S,E54K,L372P</sub>, the ideal library fragment size distribution was achieved using the enzyme at a concentration of 30 ng/μl, while higher dilutions of the enzyme increased the average fragment size (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). These observations were in agreement with the results of the initial experiment described above (Figure S2B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a>). The bigger fragments obtained when using the Nextera XT DNA library preparation buffer instead of the homemade Tris-based buffer for tagmentation with the Tn5<sub>R27S,E54K,L372P</sub> enzyme are likely to result from faint differences in the ratio between cDNA input and enzyme (compare also <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3D</a>). Importantly, libraries prepared with either of the transposases were of high quality and showed excellent Pearson correlations of <em>r</em> = 0.99 when processed from the same input cDNA (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>). We also compared the performance of our in-house Tn5 transposases with the Nextera XT DNA library preparation kit and found that NGS libraries using the same input cDNA processed with either the Tn5<sub>R27S,E54K,L372P</sub> or the commercially available kit correlated very well with a Pearson’s correlation coefficient of <em>r</em> = 0.99 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a> and Figure S3B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/79/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="NGS library preparation using different batches of in-house-produced Tn5 and reproducibility of the Tn5 purification protocol across institutes. (A) Left panel: comparison of two technical replicates for each of the three Tn5 batches. The color code indicates the Pearson correlation between samples (see legend on the right side). Right panel: pairwise correlation of read counts between NGS-libraries processed with three different Tn5 batches. Two technical replicates were pooled for each condition. (B) SDS-PAGE analysis of His6-Sumo3 Tn5 enzyme after size exclusion chromatography, purified at BZH. The single band runs at a molecular weight of &#x223C;53 kDa, corresponding to the Tn5 monomer. (C) Size exclusion chromatography (Superdex200 10/300 GL) profile of Tn5R27S,E54K,L372P transposase purified at BZH. The Tn5 dimer peak elutes at 13.88 ml, while the Tn5 aggregate elutes at 8.93 ml. (D) Bioanalyzer traces of NGS-libraries processed with different concentrations of Tn5R27S,E54K,L372P produced at BZH (see Figure 2B for libraries processed with Tn5 produced at EMBL). BZH, Heidelberg University Biochemistry Center; EMBL, European Molecular Biology Laboratory; MW, molecular weight; NGS, next-generation sequencing; SDS-PAGE, sodium dodecyl sulfate-polyacrylamide gel electrophoresis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-783931227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;NGS library preparation using different batches of in-house-produced Tn5 and reproducibility of the Tn5 purification protocol across institutes. (A) Left panel: comparison of two technical replicates for each of the three Tn5 batches. The color code indicates the Pearson correlation between samples (see legend on the right side). Right panel: pairwise correlation of read counts between NGS-libraries processed with three different Tn5 batches. Two technical replicates were pooled for each condition. (B) SDS-PAGE analysis of His&lt;sub&gt;6&lt;/sub&gt;-Sumo3 Tn5 enzyme after size exclusion chromatography, purified at BZH. The single band runs at a molecular weight of &#x223C;53 kDa, corresponding to the Tn5 monomer. (C) Size exclusion chromatography (Superdex200 10/300 GL) profile of Tn5&lt;sub&gt;R27S,E54K,L372P&lt;/sub&gt; transposase purified at BZH. The Tn5 dimer peak elutes at 13.88 ml, while the Tn5 aggregate elutes at 8.93 ml. (D) Bioanalyzer traces of NGS-libraries processed with different concentrations of Tn5&lt;sub&gt;R27S,E54K,L372P&lt;/sub&gt; produced at BZH (see Figure 2B for libraries processed with Tn5 produced at EMBL). BZH, Heidelberg University Biochemistry Center; EMBL, European Molecular Biology Laboratory; MW, molecular weight; NGS, next-generation sequencing; SDS-PAGE, sodium dodecyl sulfate-polyacrylamide gel electrophoresis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/79/F3.medium.gif" width="440" height="389"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/1/79/F3.medium.gif" width="440" height="389"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/79/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/79/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487850" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-40" class="first-child">NGS library preparation using different batches of in-house-produced Tn5 and reproducibility of the Tn5 purification protocol across institutes. (A) Left panel: comparison of two technical replicates for each of the three Tn5 batches. The color code indicates the Pearson correlation between samples (see legend on the right side). Right panel: pairwise correlation of read counts between NGS-libraries processed with three different Tn5 batches. Two technical replicates were pooled for each condition. (B) SDS-PAGE analysis of His<sub>6</sub>-Sumo3 Tn5 enzyme after size exclusion chromatography, purified at BZH. The single band runs at a molecular weight of ∼53 kDa, corresponding to the Tn5 monomer. (C) Size exclusion chromatography (Superdex200 10/300 GL) profile of Tn5<sub>R27S,E54K,L372P</sub> transposase purified at BZH. The Tn5 dimer peak elutes at 13.88 ml, while the Tn5 aggregate elutes at 8.93 ml. (D) Bioanalyzer traces of NGS-libraries processed with different concentrations of Tn5<sub>R27S,E54K,L372P</sub> produced at BZH (see <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2B</a> for libraries processed with Tn5 produced at EMBL). BZH, Heidelberg University Biochemistry Center; EMBL, European Molecular Biology Laboratory; MW, molecular weight; NGS, next-generation sequencing; SDS-PAGE, sodium dodecyl sulfate-polyacrylamide gel electrophoresis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">Using the Tn5<sub>R27S,E54K,L372P</sub>-processed libraries as an example, more detailed evaluation of the sequencing data demonstrated good raw read quality with a high percentage of reads uniquely aligning to the human genome (median across all experiments 87% of reads, first quartile: 86% and third quartile: 88%). On average, 9594 genes were detected per sample (median: 9740, first quartile: 9470, and third quartile: 9760), which is remarkable considering the limited amount of input material of ∼150–200 pg amplified cDNA processed from as little as 15 pg HeLa RNA [<em>e.g.</em>, see <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hicks <em>et al.</em> (2015)</a>]. We further validated the reproducibility of the gene expression analysis for both Tn5 variants and the Nextera XT DNA library preparation kit (Figure S3C in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a>). We obtained good coverage over the entire gene-coding region (Figure S3D in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a>) consistent with the full-length cDNA library preparation protocol. Importantly, we obtained excellent correlations between samples processed in the same experiment with a Pearson’s correlation coefficient of <em>r</em> = 0.99 (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2E</a>). We detected slightly lower correlation of detected genes when comparing samples prepared on different days, known as batch effects. However, the correlation coefficients between samples always remained above 0.88, and typically above 0.96 (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2F</a>), demonstrating high reproducibility between libraries processed on different days.</p><p id="p-42">Taken together, our data demonstrate that we have developed a robust protocol for tagmentation of low cDNA amounts (<em>e.g.</em>, from single cells) and that both of our homemade Tn5 transposases work equally well for NGS library preparation. As library fragment size distribution only correlated with the tested Tn5-to-input ratios for Tn5<sub>R27S,E54K,L372P</sub>, we performed all following experiments with this enzyme and used fragment size distribution as an indicator for the activity of the enzyme, thereby inferring the quality and reproducibility of Tn5 purification and tagmentation reactions.</p></div><div id="sec-18" class="subsection"><h3>Reproducibility of the His<sub>6</sub>-Sumo3-Tn5 purification</h3><p id="p-43">In the course of this study we produced three independent Tn5<sub>R27S,E54K,L372P</sub> batches. Stock aliquots were stored for up to 14 months at −20° without any loss of activity or library quality (<em>r</em> = 0.99, Figure S3E in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a>). To compare the activity of the different Tn5<sub>R27S,E54K,L372P</sub> purifications, we performed tagmentation experiments using the same input cDNA with a range of Tn5<sub>R27S,E54K,L372P</sub> dilutions from all three batches. All batches showed comparable activity, as seen in the typical fragment size distribution patterns when using different concentrations of the Tn5 enzyme (3–30 ng/μl). Sequencing of the resulting libraries revealed pairwise Pearson correlations of <em>r</em> &gt; 0.985 between samples processed with different Tn5 batches (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>), demonstrating that all three batches of in-house-produced Tn5 were of equal quality.</p><p id="p-44">As we aimed to develop a Tn5 purification protocol that can be used across institutes, we passed our pETM11-Sumo3-Tn5<sub>R27S,E54K,L372P</sub> plasmid on to a group at the Heidelberg University Biochemistry Center (BZH), who then independently repeated our expression and purification protocol. Importantly, following our protocol, the purification of the Tn5<sub>R27S,E54K,L372P</sub> construct worked well in their hands (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, B and C</a>). The activity and functionality of the Tn5 protein generated at BZH was then compared to the Tn5 protein purified at EMBL. We performed tagmentation on the same input cDNA using the EMBL- or BZH-purified Tn5 enzymes at a wide range of concentrations. Bioanalyzer profiles revealed comparable fragment size distribution of the NGS libraries at Tn5 dilutions ranging from 3 to 30 ng/μl (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3D</a>), demonstrating high reproducibility of the expression and purification of the His<sub>6</sub>-Sumo3-Tn5 construct across institutes.</p></div><div id="sec-19" class="subsection"><h3>3′ RNA Seq protocol using the Tn5<sub>R27S,E54K,L372P</sub> enzyme</h3><p id="p-45">Thus far, we have demonstrated that our homemade Tn5 enzymes can be used to generate high-quality NGS libraries, and that our purification strategy using the His<sub>6</sub>-Sumo3 fusion tag is robust and reproducible across institutes. An important advantage of our in-house Tn5 is the possibility to load the enzyme with any desired linker oligonucleotide, allowing for customized experimental design. We leveraged this feature to develop a method suitable for the mapping of polyadenylation sites in single cells. Recent single-cell studies have revealed that cellular heterogeneity does not only involve varying levels of gene expression but also differential choice of polyadenylation sites in different cells, resulting in 3′ isoform heterogeneity between cells (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Velten <em>et al.</em> 2015</a>). Such approaches provide new insights into biological diversity at the cellular level; however, the underlying protocols are often time-consuming and costly.</p><p id="p-46">We modified the established Smart-seq2 protocol (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Picelli <em>et al.</em> 2014b</a>) by introducing small changes during cDNA processing that allow for specific amplification of only 3′ ends after customized tagmentation with our homemade Tn5. We adapted the oligo dT primer used for mRNA priming and included a 6 nt cell barcode as well as a 4 nt end identifier (“CCAA”). The cell barcode allows for pooling of samples before tagmentation and significantly reduces hands-on time, while the end identifier allows for specific amplification of 3′ ends. mRNA priming and cDNA synthesis were performed as described in <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Picelli <em>et al.</em> (2013)</a> and tagmentation was performed as described above with a few important modifications (see <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a> for a schematic workflow): Tn5 was loaded solely with one linker oligonucleotide (Tn5ME-B/Tn5MErev), followed by tagmentation of cDNA using the above-described reaction conditions. In the subsequent PCR, we used the regular i7 adapter index primer but substituted the i5 adapter index primer with a customized one that binds specifically to the end identifier at the 3′ end of the cDNA. To avoid amplification from fragments containing Tn5ME-B adapters at both sites, we skipped the gap-filling step. Without this initial step, the i7 adapter index primer cannot bind the tagmented cDNA, while the customized adapter primer binds its template sequence at the 3′ end of the cDNA. The new, PCR-amplified strand now contains the template sequence for the i7 index adapter so that both primers can bind in the following PCR cycles, resulting in a library suitable for NGS.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/79/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Polyadenylation site mapping in single cells using a customized tagmentation protocol. (A) Workflow of Tn5 loading, cDNA tagmentation, and subsequent NGS library preparation for the mapping of polyadenylation sites in single cells. Tn5 molecules are shown as gray hexamers. The double-stranded part of the linker oligo, the mosaic element, is shown in gray with a yellow circle depicting the phosphorylated 3&#x2032; end. The 5&#x2032; overhang template for the i7 index adapter primer is shown in blue. cDNA is shown as two parallel black lines with a 3&#x2032; poly(A) tail followed by a 6 nt cell barcode (light green) and the template sequence for the customized P5 adapter primer (green, introduced during the priming of the mRNA). In contrast to the duplex index full-length NGS libraries, Tn5 is loaded solely with the linker oligonucleotide for the i7 index adapter primer (blue, i7 index is dark blue). In the subsequent PCR, the initial gap-filling step is omitted so that only the customized i5 adapter primer (dark green) can bind to the cDNA fragments, resulting in amplification from the 3&#x2032; end of the cDNA during the first PCR cycle. In the following cycles, PCR enrichment only works for terminal cDNA fragments harboring both adaptors (corresponding to the RNA 3&#x2032; end) while the other fragments are lost during library preparation (shown as transparent). (B) Schematic representation of the percentage of reads mapping to the annotated TTS or within the surrounding 50 bp. (C) Distribution of poly(A) reads mapping to different genomic features: TTS, poly(A) signal maps to the TTS of a gene &#xB1; 10 or &#xB1; 200 bp (TTS10 or TTS200, respectively), poly(A) maps to the terminal exon (term.Exon), poly(A) signal found DS of the TTS (TTS-DS), poly(A) signal is found in the exon before the terminal exon (US-exon), antisense, or intergenic. (D) Pairwise correlation of gene counts between three technical replicates when using in-house-produced Tn5R27S,E54K,L372P and the tagmentation protocol for the mapping of polyadenylation sites in single cells. DS, downstream; NGS, next-generation sequencing; PCR, polymerase chain reaction; SDS, sodium dodecyl sulfate; TTS, transcription termination site; US, upstream." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-783931227" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Polyadenylation site mapping in single cells using a customized tagmentation protocol. (A) Workflow of Tn5 loading, cDNA tagmentation, and subsequent NGS library preparation for the mapping of polyadenylation sites in single cells. Tn5 molecules are shown as gray hexamers. The double-stranded part of the linker oligo, the mosaic element, is shown in gray with a yellow circle depicting the phosphorylated 3&#x2032; end. The 5&#x2032; overhang template for the i7 index adapter primer is shown in blue. cDNA is shown as two parallel black lines with a 3&#x2032; poly(A) tail followed by a 6 nt cell barcode (light green) and the template sequence for the customized P5 adapter primer (green, introduced during the priming of the mRNA). In contrast to the duplex index full-length NGS libraries, Tn5 is loaded solely with the linker oligonucleotide for the i7 index adapter primer (blue, i7 index is dark blue). In the subsequent PCR, the initial gap-filling step is omitted so that only the customized i5 adapter primer (dark green) can bind to the cDNA fragments, resulting in amplification from the 3&#x2032; end of the cDNA during the first PCR cycle. In the following cycles, PCR enrichment only works for terminal cDNA fragments harboring both adaptors (corresponding to the RNA 3&#x2032; end) while the other fragments are lost during library preparation (shown as transparent). (B) Schematic representation of the percentage of reads mapping to the annotated TTS or within the surrounding 50 bp. (C) Distribution of poly(A) reads mapping to different genomic features: TTS, poly(A) signal maps to the TTS of a gene &#xB1; 10 or &#xB1; 200 bp (TTS10 or TTS200, respectively), poly(A) maps to the terminal exon (term.Exon), poly(A) signal found DS of the TTS (TTS-DS), poly(A) signal is found in the exon before the terminal exon (US-exon), antisense, or intergenic. (D) Pairwise correlation of gene counts between three technical replicates when using in-house-produced Tn5&lt;sub&gt;R27S,E54K,L372P&lt;/sub&gt; and the tagmentation protocol for the mapping of polyadenylation sites in single cells. DS, downstream; NGS, next-generation sequencing; PCR, polymerase chain reaction; SDS, sodium dodecyl sulfate; TTS, transcription termination site; US, upstream.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/79/F4.medium.gif" width="440" height="342"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/1/79/F4.medium.gif" width="440" height="342"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/79/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/79/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487984" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-47" class="first-child">Polyadenylation site mapping in single cells using a customized tagmentation protocol. (A) Workflow of Tn5 loading, cDNA tagmentation, and subsequent NGS library preparation for the mapping of polyadenylation sites in single cells. Tn5 molecules are shown as gray hexamers. The double-stranded part of the linker oligo, the mosaic element, is shown in gray with a yellow circle depicting the phosphorylated 3′ end. The 5′ overhang template for the i7 index adapter primer is shown in blue. cDNA is shown as two parallel black lines with a 3′ poly(A) tail followed by a 6 nt cell barcode (light green) and the template sequence for the customized P5 adapter primer (green, introduced during the priming of the mRNA). In contrast to the duplex index full-length NGS libraries, Tn5 is loaded solely with the linker oligonucleotide for the i7 index adapter primer (blue, i7 index is dark blue). In the subsequent PCR, the initial gap-filling step is omitted so that only the customized i5 adapter primer (dark green) can bind to the cDNA fragments, resulting in amplification from the 3′ end of the cDNA during the first PCR cycle. In the following cycles, PCR enrichment only works for terminal cDNA fragments harboring both adaptors (corresponding to the RNA 3′ end) while the other fragments are lost during library preparation (shown as transparent). (B) Schematic representation of the percentage of reads mapping to the annotated TTS or within the surrounding 50 bp. (C) Distribution of poly(A) reads mapping to different genomic features: TTS, poly(A) signal maps to the TTS of a gene ± 10 or ± 200 bp (TTS10 or TTS200, respectively), poly(A) maps to the terminal exon (term.Exon), poly(A) signal found DS of the TTS (TTS-DS), poly(A) signal is found in the exon before the terminal exon (US-exon), antisense, or intergenic. (D) Pairwise correlation of gene counts between three technical replicates when using in-house-produced Tn5<sub>R27S,E54K,L372P</sub> and the tagmentation protocol for the mapping of polyadenylation sites in single cells. DS, downstream; NGS, next-generation sequencing; PCR, polymerase chain reaction; SDS, sodium dodecyl sulfate; TTS, transcription termination site; US, upstream.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">We found that 82% of sequencing reads end in poly(A) tails, a much higher fraction than we reported using our previous single-cell polyadenylation site mapping protocol, BATSeq (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Velten <em>et al.</em> 2015</a>). As described for BATSeq, we then applied further filters to remove short reads, ambiguous alignments, and possible sites of mispriming, yielding 57.3% of reads aligning to high-confidence poly(A) sites (opposed to only 24.3% in BATSeq). On average, 6606 genes were detected per sample (median: 6603, first quartile: 6384, and third quartile: 6833), representing the biological sample well.</p><p id="p-49">As in BATSeq, poly(A) sites identified in single cells mostly corresponded to annotated transcription termination sites (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, B and C</a>). The correlation of genes detected between samples processed in the same experiment was good, although slightly lower than in the standard full-length cDNA preparation data described above (<em>r</em> = 0.93, <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4D</a>). Based on these results, we conclude that the modified Smart-seq2 protocol is suitable to successfully process 3′ RNA sequencing libraries with little hands-on time. Notably, the preloaded Nextera ATM enzyme cannot be used in this experimental design as every second cDNA fragment would contain the i5 linker sequence and thereby become lost during PCR amplification. Thus, using the in-house-produced Tn5, NGS library preparation can easily be adjusted and experimental design benefits from customized Tn5 loading strategies.</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-50">Here, we present a robust strategy for the purification of Tn5 transposase and a tagmentation-based NGS library preparation protocol. Tagmentation using the His<sub>6</sub>-Sumo3 purified Tn5 results in high-quality NGS libraries with Pearson’s correlation coefficients of 0.99 between samples processed on the same day, and a performance equal to that of the Nextera XT DNA library preparation kit. Importantly, we demonstrate high reproducibility of Tn5 expression and purification in-house and across institutes.</p><p id="p-51">Protein purification via the His<sub>6</sub>-tag is one of the most common strategies in biochemistry as it is robust and easy-to-handle. Consequently, we assume that the required equipment and expertise is available in most biochemistry laboratories, making purification of the Tn5 enzyme straightforward. By contrast, purification of intein-CBD fusion proteins, as described in the published Tn5-intein-CBD purification protocol (<a id="xref-ref-19-7" class="xref-bibr" href="#ref-19">Picelli <em>et al.</em> 2014a</a>), requires specific chitin beads, which are not commonly found in most biochemistry laboratories. Furthermore, the elution of the protein of interest via on-column cleavage using thiol reagents can take several days and needs to be optimized on a case-to-case basis. The only nonstandard step in our purification protocol is the cleavage of the Sumo3-tag using the SenP2 protease. We will therefore provide the plasmid expressing the His<sub>6</sub>-tagged SenP2 protease upon request. The size exclusion chromatography included in the final step of our purification strategy allows for robust quality control and reproducibility between batches as the dimer peak is exclusively isolated for tagmentation. Any protein aggregates or degradation products are excluded and a pure Tn5 dimer is obtained for every batch of in-house-produced Tn5, ensuring high reproducibility of the enzyme’s activity range in downstream approaches.</p><p id="p-52">In this study, we present two alternative His<sub>6</sub>-Sumo3 purified Tn5 transposases: Tn5<sub>E54K,L372P</sub> and Tn5<sub>R27S,E54K,L372P</sub>. Both enzymes show remarkable differences in their behavior. Tn5<sub>E54K,L372P</sub> stably produces fragments with similar size distributions over a wide range of Tn5 concentrations (Tn5-to-input molecule ratio), while Tn5<sub>R27S,E54K,L372P</sub> carries an additional point mutation (R27S) that is associated with different fragment sizes depending on the Tn5 concentration used. The crystal structure of the Tn5 synaptic complex shows that arginine 27 is involved in the binding of the ME sequence in the linker oligonucleotide that is loaded onto Tn5 (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Davies <em>et al.</em> 2000</a>). More precisely, arginine 27 forms salt bridges with phosphates in the DNA backbone of the ME sequence. Since the amino acid substitution from arginine to serine neutralizes the positive charge, we hypothesized that the Tn5<sub>R27S,E54K,L372P</sub> might have a lower affinity to the linker oligonucleotide. To experimentally verify this hypothesis, we performed ITC experiments in which linker oligonucleotide was titrated to either Tn5<sub>E54K,L372P</sub> or Tn5<sub>R27S,E54K,L372P</sub>. The results of the ITC experiments showed that the equilibrium dissociation constant K<sub>d</sub> for Tn5<sub>E54K,L372P</sub> is 0.50 µM, while the K<sub>d</sub> for Tn5<sub>R27S,E54K,L372P</sub> is only 12.9 µM (Figure S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a>). The difference in binding affinity might in turn affect the synaptic complex assembly and its functionality. We find this feature intriguing as it makes it possible to customize the fragment length of NGS libraries. The sensitivity of this Tn5 variant for the Tn5-to-input ratio provides further opportunities to develop novel Tn5 applications.</p><p id="p-53">We leveraged the flexible enzyme-loading capacity of our in-house Tn5 to develop a novel method for polyadenylation site mapping in single cells. By loading only one linker oligonucleotide onto the Tn5 enzyme, we ensured that (after tagmentation) all 3′ end fragments of the cDNA contained the correct template sequence for PCR-based library preparation. In contrast, the preloaded Nextera ATM enzyme would introduce both linkers, resulting in a loss of ∼50% of all fragments during PCR amplification. We believe that single-cell approaches with limited input material will benefit from our new method as the comparatively high number of genes detected recapitulates biological samples more accurately.</p><p id="p-54">To date, we have successfully used the in-house-produced Tn5<sub>R27S,E54K,L372P</sub> transposase and our tagmentation-based library preparation protocol to investigate the differentiation of hematopoietic stem cells based on the transcriptomes of single cells isolated from bone marrow (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Velten <em>et al.</em> 2017</a>). In addition, our homemade Tn5 variants can be used to generate NGS libraries following the previously published Drop-Seq (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Macosko <em>et al.</em> 2015</a>) method (see Figure S5A in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a>). Furthermore, NGS libraries can be created from as little as 0.06 pg of genomic DNA by decreasing the Tn5 concentration to 0.6 ng/μl and increasing the number of PCR cycles from 12 to 21, without requiring any further protocol adjustments (Figure S5B in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1/FileS1.pdf">File S1</a>). In conclusion, our Tn5 purification and tagmentation protocol can be used reliably for various experimental set-ups. It presents new options for the field due to its low cost and the high flexibility in experimental design.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-55">The pTXB1 plasmid carrying the Tn5-intein-CBD fusion construct was a generous gift from Simone Picelli and Rickard Sandberg. We thank Paul Collier and the EMBL Genomic Core Facility for fruitful discussions and their help during initial tests with the in-house-produced Tn5. We thank Allan Jones for proofreading and Lior David for critical reading of the manuscript. Mass spectrometry measurements to confirm the Tn5 enzymes were performed by the EMBL Proteomics Core Facility. This work was supported by a European Research Council Advanced Investigator grant (grant number AdG-294542 to L.M.S.). B.P.H. was supported by a fellowship from the EMBL Interdisciplinary Postdoc (EIPOD) program under Marie Sklodowska-Curie Actions COFUND (grant number 291772).</p><p id="p-56">Author contributions: L.M.S., K.R., and B.P.H. designed the study. H.B. designed the His<sub>6</sub>-Sumo3-Tn5 fusion construct and provided input to the Tn5 purification strategy. I.R. performed all Tn5 purifications at EMBL Heidelberg. M.T. purified the Tn5 from the pETM11-Sumo-Tn5 plasmid at BZH. V.R. performed isothermal titration calorimetry measurements. B.P.H. and C.S.T. processed cDNA from isolated HeLa RNA using the Smart-seq2 method and performed tagmentation experiments. L.V. performed data analysis. B.P.H. wrote the manuscript. All authors read and edited the manuscript. The authors declare no conflict of interest.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-57">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300257/-/DC1</a>.</p></li><li class="fn" id="fn-2"><p id="p-58"><em>Communicating editor: B. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-09-11"><span class="received-label">Received </span>September 11, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-10-30"><span class="accepted-label">Accepted </span>October 30, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Hennig <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.1" data-doi="10.1186/gb-2010-11-12-r119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adey</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrison</span>  <span class="cit-name-given-names">H. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asan</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xun</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitzman</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rapid, low-input, low-bias construction of shotgun fragment libraries by high-density in vitro transposition.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DAdey%26rft.auinit1%253DA.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253DR119%26rft.epage%253DR119%26rft.atitle%253DRapid%252C%2Blow-input%252C%2Blow-bias%2Bconstruction%2Bof%2Bshotgun%2Bfragment%2Blibraries%2Bby%2Bhigh-density%2Bin%2Bvitro%2Btransposition.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-12-r119%26rft_id%253Dinfo%253Apmid%252F21143862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-12-r119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21143862&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.2" data-doi="10.1093/bioinformatics/btu638"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyl</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">HTSeq–a Python framework to work with high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu638%26rft_id%253Dinfo%253Apmid%252F25260700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu638&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25260700&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000347832300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.3" data-doi="10.1038/nrg.2016.86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashley</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Towards precision medicine.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">507</span>–<span class="cit-lpage">522</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D17%26rft.spage%253D507%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg.2016.86%26rft_id%253Dinfo%253Apmid%252F27528417%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg.2016.86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27528417&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.4" data-doi="10.1038/nature14590"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buenrostro</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Litzenburger</span>  <span class="cit-name-given-names">U. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruff</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzales</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Single-cell chromatin accessibility reveals principles of regulatory variation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">523</span>: <span class="cit-fpage">486</span>–<span class="cit-lpage">490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D523%26rft.spage%253D486%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14590%26rft_id%253Dinfo%253Apmid%252F26083756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14590&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26083756&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.5" data-doi="10.1126/science.289.5476.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goryshin</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reznikoff</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rayment</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Three-dimensional structure of the Tn5 synaptic complex transposition intermediate.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">85</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDavies%26rft.auinit1%253DD.%2BR.%26rft.volume%253D289%26rft.issue%253D5476%26rft.spage%253D77%26rft.epage%253D85%26rft.atitle%253DThree-Dimensional%2BStructure%2Bof%2Bthe%2BTn5%2BSynaptic%2BComplex%2BTransposition%2BIntermediate%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.289.5476.77%26rft_id%253Dinfo%253Apmid%252F10884228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIyODkvNTQ3Ni83NyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzgvMS83OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Esplin</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oei</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Personalized sequencing and the future of medicine: discovery, diagnosis and defeat of disease.</span> <abbr class="cit-jnl-abbrev">Pharmacogenomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1771</span>–<span class="cit-lpage">1790</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPharmacogenomics%26rft.volume%253D15%26rft.spage%253D1771%26rft_id%253Dinfo%253Apmid%252F25493570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25493570&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giladi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amit</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Immunology, one cell at a time.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">547</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D547%26rft.spage%253D27%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.8" data-doi="10.1074/jbc.273.13.7367"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goryshin</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reznikoff</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Tn5 <em>in vitro</em> transposition.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">273</span>: <span class="cit-fpage">7367</span>–<span class="cit-lpage">7374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DGoryshin%26rft.auinit1%253DI.%2BY.%26rft.volume%253D273%26rft.issue%253D13%26rft.spage%253D7367%26rft.epage%253D7374%26rft.atitle%253DTn5%2Bin%2BVitro%2BTransposition%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.273.13.7367%26rft_id%253Dinfo%253Apmid%252F9516433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNzMvMTMvNzM2NyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzgvMS83OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.9" data-doi="10.1073/pnas.95.18.10716"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goryshin</span>  <span class="cit-name-given-names">I. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kil</span>  <span class="cit-name-given-names">Y. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanzov</span>  <span class="cit-name-given-names">V. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reznikoff</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Tn5/IS50 target recognition.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">10716</span>–<span class="cit-lpage">10721</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DGoryshin%26rft.auinit1%253DI.%2BY.%26rft.volume%253D95%26rft.issue%253D18%26rft.spage%253D10716%26rft.epage%253D10721%26rft.atitle%253DTn5%252FIS50%2Btarget%2Brecognition%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.95.18.10716%26rft_id%253Dinfo%253Apmid%252F9724770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTUvMTgvMTA3MTYiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy84LzEvNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.10" data-doi="10.1038/nature14966"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grun</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyubimova</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kester</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiebrands</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basak</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Single-cell messenger RNA sequencing reveals rare intestinal cell types.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">525</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D525%26rft.spage%253D251%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14966%26rft_id%253Dinfo%253Apmid%252F26287467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14966&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26287467&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hicks</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teng</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irizarry</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">On the widespread and critical impact of systematic bias and batch effects in single-cell RNA-Seq data.</span> <abbr class="cit-jnl-abbrev">bioRxiv</abbr> Available at: <a href="https://pdfs.semanticscholar.org/aa12/953bdc4cbc2c01f9b1618c6d2818f5cb37c6.pdf">https://pdfs.semanticscholar.org/aa12/953bdc4cbc2c01f9b1618c6d2818f5cb37c6.pdf</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.12" data-doi="10.1126/science.1247651"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jaitin</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenigsberg</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keren-Shaul</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elefant</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Massively parallel single-cell RNA-seq for marker-free decomposition of tissues into cell types.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">343</span>: <span class="cit-fpage">776</span>–<span class="cit-lpage">779</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DJaitin%26rft.auinit1%253DD.%2BA.%26rft.volume%253D343%26rft.issue%253D6172%26rft.spage%253D776%26rft.epage%253D779%26rft.atitle%253DMassively%2BParallel%2BSingle-Cell%2BRNA-Seq%2Bfor%2BMarker-Free%2BDecomposition%2Bof%2BTissues%2Binto%2BCell%2BTypes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1247651%26rft_id%253Dinfo%253Apmid%252F24531970%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDMvNjE3Mi83NzYiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy84LzEvNzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.13" data-doi="10.1038/304280a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reznikoff</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">DNA sequences at the ends of transposon Tn5 required for transposition.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">304</span>: <span class="cit-fpage">280</span>–<span class="cit-lpage">282</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJohnson%26rft.auinit1%253DR.%2BC.%26rft.volume%253D304%26rft.issue%253D5923%26rft.spage%253D280%26rft.epage%253D282%26rft.atitle%253DDNA%2Bsequences%2Bat%2Bthe%2Bends%2Bof%2Btransposon%2BTn5%2Brequired%2Bfor%2Btransposition.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F304280a0%26rft_id%253Dinfo%253Apmid%252F6306482%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/304280a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6306482&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.14" data-doi="10.1016/j.cell.2015.05.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Macosko</span>  <span class="cit-name-given-names">E. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Satija</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nemesh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shekhar</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Highly parallel genome-wide expression profiling of individual cells using nanoliter droplets.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">1202</span>–<span class="cit-lpage">1214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D161%26rft.spage%253D1202%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.05.002%26rft_id%253Dinfo%253Apmid%252F26000488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.05.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26000488&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.15" data-doi="10.1186/1471-2180-11-38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martínez-García</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calles</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arévalo-Rodríguez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Lorenzo</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">pBAM1: an all-synthetic genetic tool for analysis and construction of complex bacterial phenotypes.</span> <abbr class="cit-jnl-abbrev">BMC Microbiol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BMicrobiology%26rft.stitle%253DBMC%2BMicrobiology%26rft.aulast%253DMartinez-Garcia%26rft.auinit1%253DE.%26rft.volume%253D11%26rft.spage%253D38%26rft.epage%253D38%26rft.atitle%253DpBAM1%253A%2Ban%2Ball-synthetic%2Bgenetic%2Btool%2Bfor%2Banalysis%2Band%2Bconstruction%2Bof%2Bcomplex%2Bbacterial%2Bphenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2180-11-38%26rft_id%253Dinfo%253Apmid%252F21342504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2180-11-38&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21342504&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.16" data-doi="10.1038/nature12593"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagano</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lubling</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schoenfelder</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yaffe</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Single-cell Hi-C reveals cell-to-cell variability in chromosome structure.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">502</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D502%26rft.spage%253D59%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12593%26rft_id%253Dinfo%253Apmid%252F24067610%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12593&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24067610&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325106000030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.17" data-doi="10.1016/j.cell.2015.11.013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paul</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arkin</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giladi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jaitin</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kenigsberg</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Transcriptional heterogeneity and lineage commitment in myeloid progenitors.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">1663</span>–<span class="cit-lpage">1677</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D163%26rft.spage%253D1663%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2015.11.013%26rft_id%253Dinfo%253Apmid%252F26627738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2015.11.013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26627738&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.18" data-doi="10.1038/nmeth.2639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Picelli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bjorklund</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faridani</span>  <span class="cit-name-given-names">O. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sagasser</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winberg</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Smart-seq2 for sensitive full-length transcriptome profiling in single cells.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1096</span>–<span class="cit-lpage">1098</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D1096%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2639%26rft_id%253Dinfo%253Apmid%252F24056875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2639&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24056875&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326507600024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.19" data-doi="10.1101/gr.177881.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Picelli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Björklund</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinius</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sagasser</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winberg</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Tn5 transposase and tagmentation procedures for massively scaled sequencing projects.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">2033</span>–<span class="cit-lpage">2040</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.177881.114%26rft_id%253Dinfo%253Apmid%252F25079858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMi8yMDMzIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.20" data-doi="10.1038/nprot.2014.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Picelli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faridani</span>  <span class="cit-name-given-names">O. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Björklund</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winberg</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sagasser</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Full-length RNA-seq from single cells using Smart-seq2.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BProtoc.%26rft.volume%253D9%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2014.006%26rft_id%253Dinfo%253Apmid%252F24385147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2014.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24385147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.21" data-doi="10.1038/nbt.2282"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramskold</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Full-length mRNA-Seq from single-cell levels of RNA and individual circulating tumor cells.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">777</span>–<span class="cit-lpage">782</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DRamskold%26rft.auinit1%253DD.%26rft.volume%253D30%26rft.issue%253D8%26rft.spage%253D777%26rft.epage%253D782%26rft.atitle%253DFull-length%2BmRNA-Seq%2Bfrom%2Bsingle-cell%2Blevels%2Bof%2BRNA%2Band%2Bindividual%2Bcirculating%2Btumor%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2282%26rft_id%253Dinfo%253Apmid%252F22820318%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2282&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22820318&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.22" data-doi="10.1146/annurev.genet.42.110807.091656"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reznikoff</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Transposon Tn5.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">269</span>–<span class="cit-lpage">286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DReznikoff%26rft.auinit1%253DW.%2BS.%26rft.volume%253D42%26rft.spage%253D269%26rft.epage%253D286%26rft.atitle%253DTransposon%2BTn5.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.42.110807.091656%26rft_id%253Dinfo%253Apmid%252F18680433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.42.110807.091656&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18680433&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261767000013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.23" data-doi="10.1073/pnas.80.23.7293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sasakawa</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carle</span>  <span class="cit-name-given-names">G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berg</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Sequences essential for transposition at the termini of IS50.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">7293</span>–<span class="cit-lpage">7297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSasakawa%26rft.auinit1%253DC.%26rft.volume%253D80%26rft.issue%253D23%26rft.spage%253D7293%26rft.epage%253D7297%26rft.atitle%253DSequences%2Bessential%2Bfor%2Btransposition%2Bat%2Bthe%2Btermini%2Bof%2BIS50.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.80.23.7293%26rft_id%253Dinfo%253Apmid%252F6316355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODAvMjMvNzI5MyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzgvMS83OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.24" data-doi="10.1093/nar/30.11.2469"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shevchenko</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouffard</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butterfield</span>  <span class="cit-name-given-names">Y. S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blakesley</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartley</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Systematic sequencing of cDNA clones using the transposon Tn5.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2469</span>–<span class="cit-lpage">2477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DShevchenko%26rft.auinit1%253DY.%26rft.volume%253D30%26rft.issue%253D11%26rft.spage%253D2469%26rft.epage%253D2477%26rft.atitle%253DSystematic%2Bsequencing%2Bof%2BcDNA%2Bclones%2Busing%2Bthe%2Btransposon%2BTn5%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F30.11.2469%26rft_id%253Dinfo%253Apmid%252F12034835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/30.11.2469&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12034835&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176256500024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.25" data-doi="10.15252/msb.20156198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Velten</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pekowska</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarvelin</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Single-cell polyadenylation site mapping reveals 3′ isoform choice variability.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">812</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DVelten%26rft.auinit1%253DL.%26rft.volume%253D11%26rft.issue%253D6%26rft.spage%253D812%26rft.epage%253D812%26rft.atitle%253DSingle-cell%2Bpolyadenylation%2Bsite%2Bmapping%2Breveals%2B3%2527%2Bisoform%2Bchoice%2Bvariability%26rft_id%253Dinfo%253Adoi%252F10.15252%252Fmsb.20156198%26rft_id%253Dinfo%253Apmid%252F26040288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjg6IjExLzYvODEyIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.26" data-doi="10.1038/ncb3493"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Velten</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raffel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blaszkiewicz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Islam</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Human haematopoietic stem cell lineage commitment is a continuous process.</span> <abbr class="cit-jnl-abbrev">Nat. Cell Biol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">271</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCell%2BBiol.%26rft.volume%253D19%26rft.spage%253D271%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncb3493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncb3493&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.27" data-doi="10.1101/gad.8.19.2363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weinreich</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasch</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reznikoff</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Evidence that the cis preference of the Tn5 transposase is caused by nonproductive multimerization.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2363</span>–<span class="cit-lpage">2374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DWeinreich%26rft.auinit1%253DM%2BD%26rft.volume%253D8%26rft.issue%253D19%26rft.spage%253D2363%26rft.epage%253D2374%26rft.atitle%253DEvidence%2Bthat%2Bthe%2Bcis%2Bpreference%2Bof%2Bthe%2BTn5%2Btransposase%2Bis%2Bcaused%2Bby%2Bnonproductive%2Bmultimerization.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.8.19.2363%26rft_id%253Dinfo%253Apmid%252F7958902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiOC8xOS8yMzYzIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.28" data-doi="10.1093/bioinformatics/btq057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nacu</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fast and SNP-tolerant detection of complex variants and splicing in short reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">873</span>–<span class="cit-lpage">881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq057%26rft_id%253Dinfo%253Apmid%252F20147302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20147302&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276045800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.29" data-doi="10.1006/jmbi.1997.1188"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reznikoff</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Tn5 transposase mutants that alter DNA binding specificity.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">271</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DZhou%26rft.auinit1%253DM.%26rft.volume%253D271%26rft.issue%253D3%26rft.spage%253D362%26rft.epage%253D373%26rft.atitle%253DTn5%2Btransposase%2Bmutants%2Bthat%2Balter%2BDNA%2Bbinding%2Bspecificity.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1997.1188%26rft_id%253Dinfo%253Apmid%252F9268665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1997.1188&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9268665&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XT42100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.79.30" data-doi="10.1006/jmbi.1997.1579"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhasin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reznikoff</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Molecular genetic analysis of transposase-end DNA sequence recognition: cooperativity of three adjacent base-pairs in specific interaction with a mutant Tn5 transposase.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">276</span>: <span class="cit-fpage">913</span>–<span class="cit-lpage">925</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DZhou%26rft.auinit1%253DM.%26rft.volume%253D276%26rft.issue%253D5%26rft.spage%253D913%26rft.epage%253D925%26rft.atitle%253DMolecular%2Bgenetic%2Banalysis%2Bof%2Btransposase-end%2BDNA%2Bsequence%2Brecognition%253A%2Bcooperativity%2Bof%2Bthree%2Badjacent%2Bbase-pairs%2Bin%2Bspecific%2Binteraction%2Bwith%2Ba%2Bmutant%2BTn5%2Btransposase.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1997.1579%26rft_id%253Dinfo%253Apmid%252F9566196%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1997.1579&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9566196&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F79.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000072824000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/1/79.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/63" title="Genome-Wide Screen for Saccharomyces cerevisiae Genes Contributing to Opportunistic Pathogenicity in an Invertebrate Model Host" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/91" title="A Resource for the Allele-Specific Analysis of DNA Methylation at Multiple Genomically Imprinted Loci in Mice" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/1"><span>Volume 8 Issue 1, January 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/1.cover-source.jpg?itok=vyoqxep2" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/79.supplemental" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tBhEjuvE-0xGMQbw4gq_ejpojIZar7F2dSixU6cKhPA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/1/79" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/1/79" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/1/79" class="active" data-icon-position="" data-hide-link-title="0">Large-Scale Low-Cost NGS Library Preparation Using a Robust Tn5 Purification and Tagmentation Protocol</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487813" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-st2P9Bn7G8Uohut3mBmwuiZ5B7JkjCXkTDKtRWrJb60" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/79.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/1/79" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-TEP6KP8HJ_zSgPK5m9Irhloap1z3a1jZCm0MtQJ0b6I" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/1/79" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/79.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487813" id="node487813--2" data-pisa="ggg;8/1/79" data-pisa-master="ggg;g3.117.300257" data-apath="/ggg/8/1/79.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Large-Scale Low-Cost NGS Library Preparation Using a Robust Tn5 Purification and Tagmentation Protocol</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bianca P. Hennig</span>, <span class="highwire-citation-author" data-delta="1">Lars Velten</span>, <span class="highwire-citation-author" data-delta="2">Ines Racke</span>, <span class="highwire-citation-author" data-delta="3">Chelsea Szu Tu</span>, <span class="highwire-citation-author" data-delta="4">Matthias Thoms</span>, <span class="highwire-citation-author" data-delta="5">Vladimir Rybin</span>, <span class="highwire-citation-author" data-delta="6">Hüseyin Besir</span>, <span class="highwire-citation-author" data-delta="7">Kim Remans</span> and <span class="highwire-citation-author" data-delta="8">Lars M. Steinmetz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">79-89;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300257 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487813--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bianca P. Hennig </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bianca%2BP.%2BHennig%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hennig%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABianca%2BP.%2BHennig%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lars Velten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BVelten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Velten%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALars%2BVelten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ines Racke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Protein Expression and Purification Core Facility, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ines%2BRacke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Racke%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AInes%2BRacke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chelsea Szu Tu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chelsea%2BSzu%2BTu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tu%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChelsea%2BSzu%2BTu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Matthias Thoms </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biochemistry Center Heidelberg, Heidelberg University, 69120, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthias%2BThoms%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thoms%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthias%2BThoms%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Vladimir Rybin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Protein Expression and Purification Core Facility, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vladimir%2BRybin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rybin%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVladimir%2BRybin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Hüseyin Besir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Protein Expression and Purification Core Facility, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H%C3%BCseyin%2BBesir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Besir%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH%25C3%25BCseyin%2BBesir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kim Remans </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Protein Expression and Purification Core Facility, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kim%2BRemans%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Remans%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKim%2BRemans%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kim.remans@embl.de" class="" data-icon-position="" data-hide-link-title="0">kim.remans@embl.de</a>
<a href="mailto:lars.steinmetz@embl.de" class="" data-icon-position="" data-hide-link-title="0">lars.steinmetz@embl.de</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Lars M. Steinmetz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, Stanford University School of Medicine, California 94305</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Stanford Genome Technology Center, Stanford University School of Medicine, Palo Alto, California 94304</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BM.%2BSteinmetz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steinmetz%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALars%2BM.%2BSteinmetz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kim.remans@embl.de" class="" data-icon-position="" data-hide-link-title="0">kim.remans@embl.de</a>
<a href="mailto:lars.steinmetz@embl.de" class="" data-icon-position="" data-hide-link-title="0">lars.steinmetz@embl.de</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/1/79" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/1/79&amp;title=Large-Scale%20Low-Cost%20NGS%20Library%20Preparation%20Using%20a%20Robust%20Tn5%20Purification%20and%20Tagmentation%20Protocol" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/1/79&amp;title=Large-Scale%20Low-Cost%20NGS%20Library%20Preparation%20Using%20a%20Robust%20Tn5%20Purification%20and%20Tagmentation%20Protocol" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/1/79&amp;title=Large-Scale%20Low-Cost%20NGS%20Library%20Preparation%20Using%20a%20Robust%20Tn5%20Purification%20and%20Tagmentation%20Protocol" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/1/79&amp;text=Large-Scale%20Low-Cost%20NGS%20Library%20Preparation%20Using%20a%20Robust%20Tn5%20Purification%20and%20Tagmentation%20Protocol" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/1/79&amp;title=Large-Scale%20Low-Cost%20NGS%20Library%20Preparation%20Using%20a%20Robust%20Tn5%20Purification%20and%20Tagmentation%20Protocol" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/1/79&amp;t=Large-Scale%20Low-Cost%20NGS%20Library%20Preparation%20Using%20a%20Robust%20Tn5%20Purification%20and%20Tagmentation%20Protocol" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/1/79&amp;title=Large-Scale%20Low-Cost%20NGS%20Library%20Preparation%20Using%20a%20Robust%20Tn5%20Purification%20and%20Tagmentation%20Protocol" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/1/79&amp;title=Large-Scale%20Low-Cost%20NGS%20Library%20Preparation%20Using%20a%20Robust%20Tn5%20Purification%20and%20Tagmentation%20Protocol" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487813/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487813" id="node487813--3" data-pisa="ggg;8/1/79" data-pisa-master="ggg;g3.117.300257" data-apath="/ggg/8/1/79.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Large-Scale Low-Cost NGS Library Preparation Using a Robust Tn5 Purification and Tagmentation Protocol</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bianca P. Hennig</span>, <span class="highwire-citation-author" data-delta="1">Lars Velten</span>, <span class="highwire-citation-author" data-delta="2">Ines Racke</span>, <span class="highwire-citation-author" data-delta="3">Chelsea Szu Tu</span>, <span class="highwire-citation-author" data-delta="4">Matthias Thoms</span>, <span class="highwire-citation-author" data-delta="5">Vladimir Rybin</span>, <span class="highwire-citation-author" data-delta="6">Hüseyin Besir</span>, <span class="highwire-citation-author" data-delta="7">Kim Remans</span> and <span class="highwire-citation-author" data-delta="8">Lars M. Steinmetz</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">79-89;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300257 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487813--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bianca P. Hennig </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bianca%2BP.%2BHennig%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hennig%20BP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABianca%2BP.%2BHennig%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lars Velten </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BVelten%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Velten%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALars%2BVelten%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Ines Racke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Protein Expression and Purification Core Facility, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ines%2BRacke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Racke%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AInes%2BRacke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chelsea Szu Tu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chelsea%2BSzu%2BTu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tu%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChelsea%2BSzu%2BTu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Matthias Thoms </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Biochemistry Center Heidelberg, Heidelberg University, 69120, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthias%2BThoms%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thoms%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthias%2BThoms%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Vladimir Rybin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Protein Expression and Purification Core Facility, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vladimir%2BRybin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rybin%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVladimir%2BRybin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Hüseyin Besir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Protein Expression and Purification Core Facility, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=H%C3%BCseyin%2BBesir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Besir%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AH%25C3%25BCseyin%2BBesir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kim Remans </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Protein Expression and Purification Core Facility, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kim%2BRemans%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Remans%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKim%2BRemans%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kim.remans@embl.de" class="" data-icon-position="" data-hide-link-title="0">kim.remans@embl.de</a>
<a href="mailto:lars.steinmetz@embl.de" class="" data-icon-position="" data-hide-link-title="0">lars.steinmetz@embl.de</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Lars M. Steinmetz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genome Biology Unit, European Molecular Biology Laboratory, Heidelberg 69117, Germany</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, Stanford University School of Medicine, California 94305</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Stanford Genome Technology Center, Stanford University School of Medicine, Palo Alto, California 94304</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BM.%2BSteinmetz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Steinmetz%20LM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALars%2BM.%2BSteinmetz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:kim.remans@embl.de" class="" data-icon-position="" data-hide-link-title="0">kim.remans@embl.de</a>
<a href="mailto:lars.steinmetz@embl.de" class="" data-icon-position="" data-hide-link-title="0">lars.steinmetz@embl.de</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=79&amp;rft.epage=89&amp;rft.atitle=Large-Scale%20Low-Cost%20NGS%20Library%20Preparation%20Using%20a%20Robust%20Tn5%20Purification%20and%20Tagmentation%20Protocol&amp;rft.volume=8&amp;rft.issue=1&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hennig%2C+Bianca+P.&amp;rft.au=Velten%2C+Lars&amp;rft.au=Racke%2C+Ines&amp;rft.au=Tu%2C+Chelsea+Szu&amp;rft.au=Thoms%2C+Matthias&amp;rft.au=Rybin%2C+Vladimir&amp;rft.au=Besir%2C+H%C3%BCseyin&amp;rft.au=Remans%2C+Kim&amp;rft.au=Steinmetz%2C+Lars+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487813/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487813/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487813/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487813/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487813/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487813/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487813/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487813/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487813/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487813/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487813/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487813/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/1/79" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/1/79.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/1/79.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/1/79.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/1/79.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
