<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/1/63" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/1/63.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/1/63.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/1/63.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/1/63/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/1/63.atom" />
<meta name="HW.pisa" content="ggg;8/1/63" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Screen for Saccharomyces cerevisiae Genes Contributing to Opportunistic Pathogenicity in an Invertebrate Model Host" />
<meta name="DC.Identifier" content="10.1534/g3.117.300245" />
<meta name="DC.Date" content="2018-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Phadke et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Environmental opportunistic pathogens can exploit vulnerable hosts through expression of traits selected for in their natural environments. Pathogenicity is itself a complicated trait underpinned by multiple complex traits, such as thermotolerance, morphology, and stress response. The baker’s yeast, Saccharomyces cerevisiae , is a species with broad environmental tolerance that has been increasingly reported as an opportunistic pathogen of humans. Here we leveraged the genetic resources available in yeast and a model insect species, the greater waxmoth Galleria mellonella , to provide a genome-wide analysis of pathogenicity factors. Using serial passaging experiments of genetically marked wild-type strains, a hybrid strain was identified as the most fit genotype across all replicates. To dissect the genetic basis for pathogenicity in the hybrid isolate, bulk segregant analysis was performed which revealed eight quantitative trait loci significantly differing between the two bulks with alleles from both parents contributing to pathogenicity. A second passaging experiment with a library of deletion mutants for most yeast genes identified a large number of mutations whose relative fitness differed in vivo vs. in vitro , including mutations in genes controlling cell wall integrity, mitochondrial function, and tyrosine metabolism. Yeast is presumably subjected to a massive assault by the innate insect immune system that leads to melanization of the host and to a large bottleneck in yeast population size. Our data support that resistance to the innate immune response of the insect is key to survival in the host and identifies shared genetic mechanisms between S. cerevisiae and other opportunistic fungal pathogens." />
<meta name="DC.Contributor" content="Sujal S. Phadke" />
<meta name="DC.Contributor" content="Calum J. Maclean" />
<meta name="DC.Contributor" content="Serena Y. Zhao" />
<meta name="DC.Contributor" content="Emmi A. Mueller" />
<meta name="DC.Contributor" content="Lucas A. Michelotti" />
<meta name="DC.Contributor" content="Kaitlyn L. Norman" />
<meta name="DC.Contributor" content="Anuj Kumar" />
<meta name="DC.Contributor" content="Timothy Y. James" />
<meta name="article:published_time" content="2018-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome-Wide Screen for Saccharomyces cerevisiae Genes Contributing to Opportunistic Pathogenicity in an Invertebrate Model Host" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Environmental opportunistic pathogens can exploit vulnerable hosts through expression of traits selected for in their natural environments. Pathogenicity is itself a complicated trait underpinned by multiple complex traits, such as thermotolerance, morphology, and stress response. The baker’s yeast, &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;, is a species with broad environmental tolerance that has been increasingly reported as an opportunistic pathogen of humans. Here we leveraged the genetic resources available in yeast and a model insect species, the greater waxmoth &lt;i&gt;Galleria mellonella&lt;/i&gt;, to provide a genome-wide analysis of pathogenicity factors. Using serial passaging experiments of genetically marked wild-type strains, a hybrid strain was identified as the most fit genotype across all replicates. To dissect the genetic basis for pathogenicity in the hybrid isolate, bulk segregant analysis was performed which revealed eight quantitative trait loci significantly differing between the two bulks with alleles from both parents contributing to pathogenicity. A second passaging experiment with a library of deletion mutants for most yeast genes identified a large number of mutations whose relative fitness differed &lt;i&gt;in vivo&lt;/i&gt;&lt;i&gt;vs.&lt;/i&gt;&lt;i&gt;in vitro&lt;/i&gt;, including mutations in genes controlling cell wall integrity, mitochondrial function, and tyrosine metabolism. Yeast is presumably subjected to a massive assault by the innate insect immune system that leads to melanization of the host and to a large bottleneck in yeast population size. Our data support that resistance to the innate immune response of the insect is key to survival in the host and identifies shared genetic mechanisms between &lt;i&gt;S. cerevisiae&lt;/i&gt; and other opportunistic fungal pathogens.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="ggg;8/1/63" />
<meta name="citation_id" content="8/1/63" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/1/63" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/1/63.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/1/63.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/1/63.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300245" />
<meta name="citation_pmid" content="29122853" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="63" />
<meta name="citation_lastpage" content="78" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Sujal S. Phadke" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109" />
<meta name="citation_author" content="Calum J. Maclean" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109" />
<meta name="citation_author" content="Serena Y. Zhao" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109" />
<meta name="citation_author" content="Emmi A. Mueller" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7805-8785" />
<meta name="citation_author" content="Lucas A. Michelotti" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109" />
<meta name="citation_author" content="Kaitlyn L. Norman" />
<meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, University of Michigan, Ann Arbor, Michigan 48109" />
<meta name="citation_author" content="Anuj Kumar" />
<meta name="citation_author_institution" content="Department of Molecular, Cellular, and Developmental Biology, University of Michigan, Ann Arbor, Michigan 48109" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-8923-297X" />
<meta name="citation_author" content="Timothy Y. James" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109" />
<meta name="citation_author_email" content="tyjames@umich.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1123-5986" />
<meta name="citation_reference" content="citation_journal_title=Fungal Genet. Biol.;citation_author=J. A. Alspaugh;citation_title=Virulence mechanisms and Cryptococcus neoformans pathogenesis.;citation_pages=55-58;citation_volume=78;citation_year=2015;citation_doi=10.1016/j.fgb.2014.09.004" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=C. R. Bartholomew;citation_author=C. F. J. Hardy;citation_title=p21-activated kinases Cla4 and Ste20 regulate vacuole inheritance in Saccharomyces cerevisiae.;citation_volume=8;citation_year=2009;citation_pmid=19218422;citation_doi=10.1128/EC.00111-08" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Bergström;citation_author=J. T. Simpson;citation_author=F. Salinas;citation_author=B. Barre;citation_author=L. Parts;citation_title=A high-definition view of functional genetic variation from natural yeast genomes.;citation_volume=31;citation_year=2014;citation_pmid=24425782;citation_doi=10.1093/molbev/msu037" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data.;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=FEMS Immunol. Med. Microbiol.;citation_author=M. Brennan;citation_author=D. Y. Thomas;citation_author=M. Whiteway;citation_author=K. Kavanagh;citation_title=Correlation between virulence of Candida albicans mutants in mice and Galleria mellonella larvae.;citation_volume=34;citation_year=2002;citation_pmid=12381467;citation_doi=10.1111/j.1574-695X.2002.tb00617.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=S. Brunke;citation_author=K. Seider;citation_author=D. Fischer;citation_author=I. D. Jacobsen;citation_author=L. Kasper;citation_title=One small step for a yeast - microevolution within macrophages renders Candida glabrata hypervirulent due to a single point mutation.;citation_pages=18;citation_volume=10;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=A. M. Calcagno;citation_author=E. Bignell;citation_author=T. R. Rogers;citation_author=M. Canedo;citation_author=F. A. Muhlschlegel;citation_title=Candida glabrata Ste20 is involved in maintaining cell wall integrity and adaptation to hypertonic stress, and is required for wild-type levels of virulence.;citation_pages=557-568;citation_volume=21;citation_year=2004;citation_issue=7;citation_pmid=15164359;citation_doi=10.1002/yea.1125" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=R. Calderone;citation_author=D. M. Li;citation_author=A. Traven;citation_title=System-level impact of mitochondria on fungal virulence: to metabolism and beyond.;citation_pages=11;citation_volume=15;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Infection and Immunity;citation_journal_abbrev=Infect. Immun.;citation_author=A. Casadevall;citation_author=L. A. Pirofski;citation_title=Host-Pathogen Interactions: Redefining the Basic Concepts of Virulence and Pathogenicity;citation_pages=3703-3713;citation_volume=67;citation_year=1999;citation_issue=8;citation_pmid=10417127" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=A. Casadevall;citation_author=L. A. Pirofski;citation_title=Accidental virulence, cryptic pathogenesis, Martians, lost hosts, and the pathogenicity of environmental microbes.;citation_volume=6;citation_year=2007;citation_pmid=17951515;citation_doi=10.1128/EC.00308-07" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in microbiology;citation_journal_abbrev=Curr Opin Microbiol;citation_author=A. Casadevall;citation_author=J. N. Steenbergen;citation_author=J. D. Nosanchuk;citation_title='Ready made' virulence and 'dual use' virulence factors in pathogenic environmental fungi--the Cryptococcus neoformans paradigm.;citation_pages=332-337;citation_volume=6;citation_year=2003;citation_issue=4;citation_pmid=12941400;citation_doi=10.1016/S1369-5274(03)00082-1" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=B. L. Chin;citation_author=M. A. Frizzell;citation_author=W. E. Timberlake;citation_author=G. R. Fink;citation_title=FASTER MT: isolation of pure populations of a and α ascospores from Saccharomyces cerevisiae.;citation_pages=449-452;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. K. Christians;citation_author=M. S. Cheema;citation_author=I. A. Vergara;citation_author=C. A. Watt;citation_author=L. J. Pinto;citation_title=Quantitative trait locus (QTL) mapping reveals a role for unstudied genes in Aspergillus virulence.;citation_pages=12;citation_volume=6;citation_year=2011;citation_doi=10.1371/journal.pone.0021746" />
<meta name="citation_reference" content="citation_journal_title=J. Infect. Dis.;citation_author=K. V. Clemons;citation_author=J. H. McCusker;citation_author=R. W. Davis;citation_author=D. A. Stevens;citation_title=Comparative pathogenesis of clinical and nonclinical isolates of Saccharomyces cerevisiae.;citation_volume=169;citation_year=1994;citation_pmid=8133102;citation_doi=10.1093/infdis/169.4.859" />
<meta name="citation_reference" content="citation_journal_title=FEMS Immunol. Med. Microbiol.;citation_author=G. Cotter;citation_author=S. Doyle;citation_author=K. Kavanagh;citation_title=Development of an insect model for the in vivo pathogenicity testing of yeasts.;citation_volume=27;citation_year=2000;citation_pmid=10640612;citation_doi=10.1111/j.1574-695X.2000.tb01427.x" />
<meta name="citation_reference" content="citation_journal_title=Antonie van Leeuwenhoek;citation_journal_abbrev=Antonie van Leeuwenhoek;citation_author=R. de Llanos;citation_author=M. T. Fernández-Espinar;citation_author=A. Querol;citation_title=A comparison of clinical and food Saccharomyces cerevisiae isolates on the basis of potential virulence factors.;citation_pages=221-231;citation_volume=90;citation_year=2006;citation_issue=3;citation_pmid=16871421;citation_doi=10.1007/s10482-006-9077-7" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=R. de Llanos;citation_author=C. Hernández-Haro;citation_author=E. Barrio;citation_author=A. Querol;citation_author=M. T. Fernández-Espinar;citation_title=Differences in activation of MAP kinases and variability in the polyglutamine tract of Slt2 in clinical and non-clinical isolates of Saccharomyces cerevisiae.;citation_pages=549-561;citation_volume=27;citation_year=2010;citation_issue=8;citation_pmid=20586115" />
<meta name="citation_reference" content="citation_journal_title=J. Infect. Dis.;citation_author=A. Desalermos;citation_author=X. J. Tan;citation_author=R. Rajamuthiah;citation_author=M. Arvanitis;citation_author=Y. Wang;citation_title=A multi-host approach for the systematic analysis of virulence factors in Cryptococcus neoformans.;citation_volume=211;citation_year=2015;citation_pmid=25114160;citation_doi=10.1093/infdis/jiu441" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Diezmann;citation_author=F. S. Dietrich;citation_title=Oxidative stress survival in a clinical Saccharomyces cerevisiae isolate is influenced by a major quantitative trait nucleotide.;citation_volume=188;citation_year=2011;citation_pmid=21515583;citation_doi=10.1534/genetics.111.128256" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Dobson;citation_author=K. D. Lafferty;citation_author=A. M. Kuris;citation_author=R. F. Hechinger;citation_author=W. Jetz;citation_title=Homage to Linnaeus: how many parasites? How many hosts?;citation_volume=105;citation_year=2008;citation_pmid=18695218;citation_doi=10.1073/pnas.0803232105" />
<meta name="citation_reference" content="citation_title=Evolution in Health and Disease;citation_title=The evolution and expression of virulence;citation_author=D. Ebert;citation_author=J. J. Bull;citation_pages=153-167;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=M. D. Edwards;citation_author=D. K. Gifford;citation_title=Convergent evolution in structural elements of proteins investigated using cross profile analysis.;citation_pages=11-11;citation_volume=13;citation_year=2012;citation_pmid=22244085" />
<meta name="citation_reference" content="citation_journal_title=Clin. Infect. Dis.;citation_author=A. Enache-Angoulvant;citation_author=C. Hennequin;citation_title=Invasive Saccharomyces infection: a comprehensive review.;citation_volume=41;citation_year=2005;citation_pmid=16267727;citation_doi=10.1086/497832" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Falcon;citation_author=R. Gentleman;citation_title=Using GOstats to test gene lists for GO term association.;citation_volume=23;citation_year=2007;citation_pmid=17098774;citation_doi=10.1093/bioinformatics/btl567" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Firacative;citation_author=S. Duan;citation_author=W. Meyer;citation_title=Galleria mellonella model identifies highly virulent strains among all major molecular types of Cryptococcus gattii.;citation_pages=12;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0108788" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. A. Fraser;citation_author=S. S. Giles;citation_author=E. C. Wenink;citation_author=S. G. Geunes-Boyer;citation_author=J. R. Wright;citation_title=Same-sex mating and the origin of the Vancouver Island Cryptococcus gattii outbreak.;citation_pages=1360-1364;citation_volume=437;citation_year=2005;citation_issue=7063;citation_pmid=16222245;citation_doi=10.1038/nature04220" />
<meta name="citation_reference" content="citation_journal_title=Virulence;citation_author=B. B. Fuchs;citation_author=E. O’Brien;citation_author=J. B. El Khoury;citation_author=E. Mylonakis;citation_title=Methods for using Galleria mellonella as a model host to study fungal pathogenesis.;citation_pages=475-482;citation_volume=1;citation_year=2010;citation_pmid=21178491;citation_doi=10.4161/viru.1.6.12985" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. A. Gale;citation_author=C. M. Bendel;citation_author=M. McClellan;citation_author=M. Hauser;citation_author=J. M. Becker;citation_title=Linkage of Adhesion, Filamentous Growth, and Virulence in Candida albicans to a Single Gene, INT1;citation_pages=1355-1358;citation_volume=279;citation_year=1998;citation_issue=5355;citation_pmid=9478896;citation_doi=10.1126/science.279.5355.1355" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Rev.;citation_author=J. M. Gancedo;citation_title=Control of pseudohyphae formation in Saccharomyces cerevisiae.;citation_volume=25;citation_year=2001;citation_pmid=11152942;citation_doi=10.1111/j.1574-6976.2001.tb00573.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Giaever;citation_author=A. M. Chu;citation_author=L. Ni;citation_author=C. Connelly;citation_author=L. Riles;citation_title=Functional profiling of the Saccharomyces cerevisiae genome.;citation_pages=387-391;citation_volume=418;citation_year=2002;citation_issue=6896;citation_pmid=12140549;citation_doi=10.1038/nature00935" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. J. Gimeno;citation_author=P. O. Ljungdahl;citation_author=C. A. Styles;citation_author=G. R. Fink;citation_title=Unipolar cell divisions in the yeast S. cerevisiae lead to filamentous growth: regulation by starvation and RAS.;citation_pages=1077-1090;citation_volume=68;citation_year=1992;citation_issue=6;citation_pmid=1547504;citation_doi=10.1016/0092-8674(92)90079-R" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. L. Goldstein;citation_author=J. H. McCusker;citation_title=Development of Saccharomyces cerevisiae as a Model Pathogen: A System for the Genetic Identification of Gene Products Required for Survival in the Mammalian Host Environment;citation_pages=499-513;citation_volume=159;citation_year=2001;citation_issue=2;citation_pmid=11606528" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=P. H. Hsu;citation_author=P. C. Chiang;citation_author=C. H. Liu;citation_author=Y. W. Chang;citation_title=Characterization of cell wall proteins in Saccharomyces cerevisiae clinical isolates elucidates Hsp150p in virulence.;citation_pages=18;citation_volume=10;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=C. Huxley;citation_author=E. D. Green;citation_author=I. Dunham;citation_title=Rapid assessment of S. cerevisiae mating type by PCR.;citation_pages=236-236;citation_volume=6;citation_year=1990;citation_issue=8;citation_pmid=2238077;citation_doi=10.1016/0168-9525(90)90190-H" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=C. Jain;citation_author=M. J. Yun;citation_author=S. M. Politz;citation_author=R. P. Rao;citation_title=A pathogenesis assay using Saccharomyces cerevisiae and Caenorhabditis elegans reveals novel roles for yeast AP-1, Yap1, and host dual oxidase BLI-3 in fungal pathogenesis.;citation_volume=8;citation_year=2009;citation_pmid=19502579;citation_doi=10.1128/EC.00367-08" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=R. Jin;citation_author=C. J. Dobry;citation_author=P. J. McCown;citation_author=A. Kumar;citation_title=Large-scale analysis of yeast filamentous growth by systematic gene disruption and overexpression.;citation_volume=19;citation_year=2008;citation_pmid=17989363;citation_doi=10.1091/mbc.E07-05-0519" />
<meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Rev.;citation_author=K. Kavanagh;citation_author=E. P. Reeves;citation_title=Exploiting the potential of insects for in vivo pathogenicity testing of microbial pathogens.;citation_volume=28;citation_year=2004;citation_pmid=14975532;citation_doi=10.1016/j.femsre.2003.09.002" />
<meta name="citation_reference" content="citation_journal_title=Eukaryot. Cell;citation_author=J. M. Kingsbury;citation_author=J. H. McCusker;citation_title=Fungal homoserine kinase (thr1Δ) mutants are attenuated in virulence and die rapidly upon threonine starvation and serum incubation.;citation_volume=9;citation_year=2010;citation_pmid=20305003;citation_doi=10.1128/EC.00045-10" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=J. M. Kingsbury;citation_author=A. L. Goldstein;citation_author=J. H. McCusker;citation_title=Role of Nitrogen and Carbon Transport, Regulation, and Metabolism Genes for Saccharomyces cerevisiae Survival In Vivo;citation_pages=816-824;citation_volume=5;citation_year=2006;citation_issue=5;citation_pmid=16682459;citation_doi=10.1128/EC.5.5.816-824.2006" />
<meta name="citation_reference" content="citation_journal_title=FEMS Yeast Res.;citation_author=T. D. Klingberg;citation_author=U. Lesnik;citation_author=N. Arneborg;citation_author=P. Raspor;citation_author=L. Jespersen;citation_title=Comparison of Saccharomyces cerevisiae strains of clinical and nonclinical origin by molecular typing and determination of putative virulence traits.;citation_volume=8;citation_year=2008;citation_pmid=18355272;citation_doi=10.1111/j.1567-1364.2008.00365.x" />
<meta name="citation_reference" content="citation_journal_title=MBio;citation_author=C. H. Kowalski;citation_author=S. R. Beattie;citation_author=K. K. Fuller;citation_author=E. A. McGurk;citation_author=Y. W. Tang;citation_title=Heterogeneity among isolates reveals that fitness in low oxygen correlates with Aspergillus fumigatus virulence.;citation_pages=13;citation_volume=7;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=P. R. Kraus;citation_author=D. S. Fox;citation_author=G. M. Cox;citation_author=J. Heitman;citation_title=The Cryptococcus neoformans MAP kinase Mpk1 regulates cell integrity in response to antifungal drugs and loss of calcineurin function.;citation_pages=1377-1387;citation_volume=48;citation_year=2003;citation_issue=5;citation_pmid=12787363;citation_doi=10.1046/j.1365-2958.2003.03508.x" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. Leberer;citation_author=D. Harcus;citation_author=I. D. Broadbent;citation_author=K. L. Clark;citation_author=D. Dignard;citation_title=Signal transduction through homologs of the Ste20p and Ste7p protein kinases can trigger hyphal formation in the pathogenic fungus Candida albicans;citation_pages=13217-13222;citation_volume=93;citation_year=1996;citation_issue=23;citation_pmid=8917571;citation_doi=10.1073/pnas.93.23.13217" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=K. B. Lengeler;citation_author=R. C. Davidson;citation_author=C. D’Souza;citation_author=T. Harashima;citation_author=W. C. Shen;citation_title=Signal Transduction Cascades Regulating Fungal Development and Virulence;citation_pages=746-785;citation_volume=64;citation_year=2000;citation_issue=4;citation_pmid=11104818;citation_doi=10.1128/MMBR.64.4.746-785.2000" />
<meta name="citation_reference" content="citation_title=arXiv;citation_title=Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM.;citation_author=H. Li;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Liti;citation_author=D. M. Carter;citation_author=A. M. Moses;citation_author=J. Warringer;citation_author=L. Parts;citation_title=Population genomics of domestic and wild yeasts.;citation_pages=337-341;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212322;citation_doi=10.1038/nature07743" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. P. Liu;citation_author=C. A. Styles;citation_author=G. R. Fink;citation_title=Saccharomyces cerevisiae S288C Has a Mutation in FL08, a Gene Required for Filamentous Growth;citation_pages=967-978;citation_volume=144;citation_year=1996;citation_issue=3;citation_pmid=8913742" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Llopis;citation_author=C. Hernández-Haro;citation_author=L. Monteoliva;citation_author=A. Querol;citation_author=M. Molina;citation_title=Pathogenic potential of Saccharomyces strains isolated from dietary supplements.;citation_pages=21;citation_volume=9;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Virulence;citation_author=H. S. Ma;citation_author=R. C. May;citation_title=The fatal fungal outbreak on Vancouver Island is characterized by enhanced intracellular parasitism driven by mitochondrial regulation.;citation_pages=197-201;citation_volume=1;citation_year=2010;citation_pmid=21178442;citation_doi=10.4161/viru.1.3.11053" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=C. J. Maclean;citation_author=B. P. H. Metzger;citation_author=J.-R. Yang;citation_author=W.-C. Ho;citation_author=B. Moyers;citation_title=Deciphering the genic basis of yeast fitness variation by simultaneous forward and reverse genetics.;citation_pages=2486-2502;citation_volume=34;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Infection and Immunity;citation_journal_abbrev=Infect. Immun.;citation_author=J. H. McCusker;citation_author=K. V. Clemons;citation_author=D. A. Stevens;citation_author=R. W. Davis;citation_title=Saccharomyces cerevisiae virulence phenotype as determined with CD-1 mice is associated with the ability to grow at 42 degrees C and form pseudohyphae.;citation_pages=5447-5455;citation_volume=62;citation_year=1994;citation_issue=12;citation_pmid=7960125" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=L. A. H. Muller;citation_author=J. H. McCusker;citation_title=Microsatellite analysis of genetic diversity among clinical and nonclinical Saccharomyces cerevisiae isolates suggests heterozygote advantage in clinical environments.;citation_pages=2779-2786;citation_volume=18;citation_year=2009;citation_issue=13;citation_pmid=19457175;citation_doi=10.1111/j.1365-294X.2009.04234.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=L. A. H. Muller;citation_author=J. E. Lucas;citation_author=D. R. Georgianna;citation_author=J. H. McCusker;citation_title=Genome-wide association analysis of clinical vs. nonclinical origin provides insights into Saccharomyces cerevisiae pathogenesis.;citation_pages=4085-4097;citation_volume=20;citation_year=2011;citation_issue=19;citation_pmid=21880084;citation_doi=10.1111/j.1365-294X.2011.05225.x" />
<meta name="citation_reference" content="citation_journal_title=Enzyme Microb. Technol.;citation_author=A. Murphy;citation_author=K. Kavanagh;citation_title=Emergence of Saccharomyces cerevisiae as a human pathogen. Implications for biotechnology.;citation_pages=551-557;citation_volume=25;citation_year=1999;citation_doi=10.1016/S0141-0229(99)00086-1" />
<meta name="citation_reference" content="citation_journal_title=Infection and Immunity;citation_journal_abbrev=Infect. Immun.;citation_author=E. Mylonakis;citation_author=R. Moreno;citation_author=J. B. El Khoury;citation_author=A. Idnurm;citation_author=J. Heitman;citation_title=Galleria mellonella as a Model System To Study Cryptococcus neoformans Pathogenesis;citation_pages=3842-3850;citation_volume=73;citation_year=2005;citation_issue=7;citation_pmid=15972469;citation_doi=10.1128/IAI.73.7.3842-3850.2005" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=G. Y. Navarro-Velasco;citation_author=R. C. Prados-Rosales;citation_author=A. Ortiz-Urquiza;citation_author=E. Quesada-Moraga;citation_author=A. Di Pietro;citation_title=Galleria mellonella as model host for the trans-kingdom pathogen Fusarium oxysporum.;citation_pages=1124-1129;citation_volume=48;citation_year=2011;citation_issue=12;citation_pmid=21907298;citation_doi=10.1016/j.fgb.2011.08.004" />
<meta name="citation_reference" content="citation_journal_title=Infection and Immunity;citation_journal_abbrev=Infect. Immun.;citation_author=J. B. Neilson;citation_author=M. H. Ivey;citation_author=G. S. Bulmer;citation_title=Cryptococcus neoformans: pseudohyphal forms surviving culture with Acanthamoeba polyphaga.;citation_pages=262-266;citation_volume=20;citation_year=1978;citation_issue=1;citation_pmid=352931" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=F. C. Odds;citation_title=Pathogenic fungi in the 21st century.;citation_pages=200-201;citation_volume=8;citation_year=2000;citation_issue=5;citation_pmid=10785633;citation_doi=10.1016/S0966-842X(00)01752-2" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=L. Parts;citation_author=F. A. Cubillos;citation_author=J. Warringer;citation_author=K. Jain;citation_author=F. Salinas;citation_title=Revealing the genetic structure of a trait by sequencing a population under selection.;citation_volume=21;citation_year=2011;citation_pmid=21422276;citation_doi=10.1101/gr.116731.110" />
<meta name="citation_reference" content="citation_journal_title=Clinical microbiology and infection :  the official publication of the European Society of Clinical Microbiology and Infectious Diseases;citation_journal_abbrev=Clin Microbiol Infect;citation_author=M. Richardson;citation_author=C. Lass-Flörl;citation_title=Limiting the attributable mortality of nosocomial infection and multidrug resistance in intensive care units.;citation_pages=5-13;citation_volume=14;citation_year=2008;citation_issue=1;citation_pmid=17944969;citation_doi=10.1111/j.1469-0691.2008.01978.x" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=R. L. Roberts;citation_author=G. R. Fink;citation_title=Elements of a single MAP kinase cascade in Saccharomyces cerevisiae mediate two developmental programs in the same cell type: mating and invasive growth.;citation_pages=2974-2985;citation_volume=8;citation_year=1994;citation_issue=24;citation_pmid=8001818;citation_doi=10.1101/gad.8.24.2974" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. D. Robinson;citation_author=D. J. McCarthy;citation_author=G. K. Smyth;citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.;citation_volume=26;citation_year=2010;citation_pmid=19910308;citation_doi=10.1093/bioinformatics/btp616" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=B. Rockmill;citation_author=E. J. Lambie;citation_author=G. S. Roeder;citation_title=Spore enrichment.;citation_pages=146-149;citation_volume=194;citation_year=1991;citation_pmid=2005784;citation_doi=10.1016/S1046-2023(05)80215-4" />
<meta name="citation_reference" content="citation_journal_title=Trends in microbiology;citation_journal_abbrev=Trends Microbiol;citation_author=E. Román;citation_author=D. M. Arana;citation_author=C. Nombela;citation_author=R. Alonso-Monge;citation_author=J. Pla;citation_title=MAP kinase pathways as regulators of fungal virulence.;citation_pages=181-190;citation_volume=15;citation_year=2007;citation_issue=4;citation_pmid=17321137;citation_doi=10.1016/j.tim.2007.02.001" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Schacherer;citation_author=J. A. Shapiro;citation_author=D. M. Ruderfer;citation_author=L. Kruglyak;citation_title=Comprehensive polymorphism survey elucidates population structure of Saccharomyces cerevisiae.;citation_pages=342-345;citation_volume=458;citation_year=2009;citation_issue=7236;citation_pmid=19212320;citation_doi=10.1038/nature07670" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Sellis;citation_author=D. J. Kvitek;citation_author=B. Dunn;citation_author=G. Sherlock;citation_author=D. A. Petrov;citation_title=Heterozygote advantage is a common outcome of adaptation in Saccharomyces cerevisiae.;citation_volume=203;citation_year=2016;citation_pmid=27194750;citation_doi=10.1534/genetics.115.185165" />
<meta name="citation_reference" content="citation_journal_title=Med. Mycol.;citation_author=J. L. Slater;citation_author=L. Gregson;citation_author=D. W. Denning;citation_author=P. A. Warn;citation_title=Pathogenicity of Aspergillus fumigatus mutants assessed in Galleria mellonella matches that in mice.;citation_volume=49;citation_year=2011;citation_pmid=20950221;citation_doi=10.3109/13693786.2010.523852" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. M. Smith;citation_author=L. E. Heisler;citation_author=R. P. St Onge;citation_author=E. Farias-Hesson;citation_author=I. M. Wallace;citation_title=Highly-multiplexed barcode sequencing: an efficient method for parallel analysis of pooled samples.;citation_pages=7;citation_volume=38;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. M. Steinmetz;citation_author=H. Sinha;citation_author=D. R. Richards;citation_author=J. I. Spiegelman;citation_author=P. J. Oefner;citation_title=Dissecting the architecture of a quantitative trait locus in yeast.;citation_pages=326-330;citation_volume=416;citation_year=2002;citation_issue=6878;citation_pmid=11907579;citation_doi=10.1038/416326a" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. K. Strope;citation_author=D. A. Skelly;citation_author=S. G. Kozmin;citation_author=G. Mahadevan;citation_author=E. A. Stone;citation_title=The 100-genomes strains, an S. cerevisiae resource that illuminates its natural phenotypic and genotypic variation and emergence as an opportunistic pathogen.;citation_volume=25;citation_year=2015;citation_pmid=25840857;citation_doi=10.1101/gr.185538.114" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=A. B. Sunshine;citation_author=C. Payen;citation_author=G. T. Ong;citation_author=I. Liachko;citation_author=K. M. Tan;citation_title=The fitness consequences of aneuploidy are driven by condition-dependent gene effects.;citation_pages=34;citation_volume=13;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Annual review of microbiology;citation_journal_abbrev=Annu Rev Microbiol;citation_author=J. A. H. van Burik;citation_author=P. T. Magee;citation_title=ASPECTS OF FUNGAL PATHOGENESIS IN HUMANS;citation_pages=743-772;citation_volume=55;citation_year=2001;citation_issue=1;citation_pmid=11544373;citation_doi=10.1146/annurev.micro.55.1.743" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. A. Vogan;citation_author=J. Khankhet;citation_author=H. Samarasinghe;citation_author=J. Xu;citation_title=Identification of QTLs associated with virulence related traits and drug resistance in Cryptococcus neoformans.;citation_pages=2745-2759;citation_volume=6;citation_year=2016;citation_doi=10.1534/g3.116.029595" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. Walter;citation_author=F. Sanchez-Cabo;citation_author=M. Ricote;citation_title=GOplot: an R package for visually combining expression data with functional analysis.;citation_volume=31;citation_year=2015;citation_pmid=25964631;citation_doi=10.1093/bioinformatics/btv300" />
<meta name="citation_reference" content="citation_journal_title=Eukaryotic Cell;citation_journal_abbrev=Eukaryot Cell;citation_author=P. Wang;citation_author=C. B. Nichols;citation_author=M. B. Lengeler;citation_author=M. E. Cardenas;citation_author=G. M. Cox;citation_title=Mating-Type-Specific and Nonspecific PAK Kinases Play Shared and Divergent Roles in Cryptococcus neoformans;citation_pages=257-272;citation_volume=1;citation_year=2002;citation_issue=2;citation_pmid=12455960;citation_doi=10.1128/EC.1.2.257-272.2002" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=Q. M. Wang;citation_author=W. Q. Liu;citation_author=G. Liti;citation_author=S. A. Wang;citation_author=F. Y. Bai;citation_title=Surprisingly diverged populations of Saccharomyces cerevisiae in natural environments remote from human activity.;citation_pages=5404-5417;citation_volume=21;citation_year=2012;citation_issue=22;citation_pmid=22913817;citation_doi=10.1111/j.1365-294X.2012.05732.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=A. Wartenberg;citation_author=J. Linde;citation_author=R. Martin;citation_author=M. Schreiner;citation_author=F. Horn;citation_title=Microevolution of Candida albicans in macrophages restores filamentation in a nonfilamentous mutant.;citation_pages=18;citation_volume=10;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=R. T. Wheeler;citation_author=G. R. Fink;citation_title=A drug-sensitive genetic network masks fungi from the immune system.;citation_pages=328-339;citation_volume=2;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. T. Wheeler;citation_author=M. Kupiec;citation_author=P. Magnelli;citation_author=C. Abeijon;citation_author=G. R. Fink;citation_title=A Saccharomyces cerevisiae mutant with increased virulence.;citation_volume=100;citation_year=2003;citation_pmid=12589024;citation_doi=10.1073/pnas.0437995100" />
<meta name="citation_reference" content="citation_title=ggplot2: Elegant Graphics for Data Analysis.;citation_author=H. Wickham;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. A. Winzeler;citation_author=D. D. Shoemaker;citation_author=A. Astromoff;citation_author=H. Liang;citation_author=K. Anderson;citation_title=Functional Characterization of the S. cerevisiae Genome by Gene Deletion and Parallel Analysis;citation_pages=901-906;citation_volume=285;citation_year=1999;citation_issue=5429;citation_pmid=10436161;citation_doi=10.1126/science.285.5429.901" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=J. R. Xu;citation_title=Map kinases in fungal pathogens.;citation_pages=137-152;citation_volume=31;citation_year=2000;citation_issue=3;citation_pmid=11273677;citation_doi=10.1006/fgbi.2000.1237" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Microbiol.;citation_author=O. Zaragoza;citation_author=K. Nielsen;citation_title=Titan cells in Cryptococcus neoformans: cells with a giant impact.;citation_pages=409-413;citation_volume=16;citation_year=2013;citation_pmid=23588027;citation_doi=10.1016/j.mib.2013.03.006" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. R. Zerbino;citation_author=E. Birney;citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs.;citation_volume=18;citation_year=2008;citation_pmid=18349386;citation_doi=10.1101/gr.074492.107" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=Y. O. Zhu;citation_author=G. Sherlock;citation_author=D. A. Petrov;citation_title=Whole genome analysis of 132 clinical Saccharomyces cerevisiae strains reveals extensive ploidy variation.;citation_pages=2421-2434;citation_volume=6;citation_year=2016" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Screen for Saccharomyces cerevisiae Genes Contributing to Opportunistic Pathogenicity in an Invertebrate Model Host" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="twitter:description" content="Environmental opportunistic pathogens can exploit vulnerable hosts through expression of traits selected for in their natural environments. Pathogenicity is itself a complicated trait underpinned by multiple complex traits, such as thermotolerance, morphology, and stress response. The baker’s yeast, Saccharomyces cerevisiae , is a species with broad environmental tolerance that has been increasingly reported as an opportunistic pathogen of humans. Here we leveraged the genetic resources available in yeast and a model insect species, the greater waxmoth Galleria mellonella , to provide a genome-wide analysis of pathogenicity factors. Using serial passaging experiments of genetically marked wild-type strains, a hybrid strain was identified as the most fit genotype across all replicates. To dissect the genetic basis for pathogenicity in the hybrid isolate, bulk segregant analysis was performed which revealed eight quantitative trait loci significantly differing between the two bulks with alleles from both parents contributing to pathogenicity. A second passaging experiment with a library of deletion mutants for most yeast genes identified a large number of mutations whose relative fitness differed in vivo vs. in vitro , including mutations in genes controlling cell wall integrity, mitochondrial function, and tyrosine metabolism. Yeast is presumably subjected to a massive assault by the innate insect immune system that leads to melanization of the host and to a large bottleneck in yeast population size. Our data support that resistance to the innate immune response of the insect is key to survival in the host and identifies shared genetic mechanisms between S. cerevisiae and other opportunistic fungal pathogens." />
<meta name="og-title" property="og:title" content="Genome-Wide Screen for Saccharomyces cerevisiae Genes Contributing to Opportunistic Pathogenicity in an Invertebrate Model Host" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/1/63" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Environmental opportunistic pathogens can exploit vulnerable hosts through expression of traits selected for in their natural environments. Pathogenicity is itself a complicated trait underpinned by multiple complex traits, such as thermotolerance, morphology, and stress response. The baker’s yeast, Saccharomyces cerevisiae , is a species with broad environmental tolerance that has been increasingly reported as an opportunistic pathogen of humans. Here we leveraged the genetic resources available in yeast and a model insect species, the greater waxmoth Galleria mellonella , to provide a genome-wide analysis of pathogenicity factors. Using serial passaging experiments of genetically marked wild-type strains, a hybrid strain was identified as the most fit genotype across all replicates. To dissect the genetic basis for pathogenicity in the hybrid isolate, bulk segregant analysis was performed which revealed eight quantitative trait loci significantly differing between the two bulks with alleles from both parents contributing to pathogenicity. A second passaging experiment with a library of deletion mutants for most yeast genes identified a large number of mutations whose relative fitness differed in vivo vs. in vitro , including mutations in genes controlling cell wall integrity, mitochondrial function, and tyrosine metabolism. Yeast is presumably subjected to a massive assault by the innate insect immune system that leads to melanization of the host and to a large bottleneck in yeast population size. Our data support that resistance to the innate immune response of the insect is key to survival in the host and identifies shared genetic mechanisms between S. cerevisiae and other opportunistic fungal pathogens." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
    <title>Genome-Wide Screen for Saccharomyces cerevisiae Genes Contributing to Opportunistic Pathogenicity in an Invertebrate Model Host | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"CxVn_vHM1pcj0fvliGce7ax7b-xhsxzIcKQ-5zduKcE","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487839","apath":"\/ggg\/8\/1\/63.atom","pisa":"ggg;8\/1\/63","ac":{"\/ggg\/8\/1\/63.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/1\/63.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/1\/63"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/1\/63"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/1\/63.figures-only":true,"\/content\/8\/1\/63":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022G+0KHLuJRPqjYYOlN6dEVYCNFZMd\\\\\\\/cW0EqJmh+tI1YBNF6z0wkCS69UOBWV8tcHufmUL6XTYaDEDU7\\\\\\\/kgVywefT\\\\\\\/mL\\\\\\\/2QmmUCrajri2BKJfBRG5Vc8viAMsGoRvnnvAr7Js2VUeorZflS9WCSWmLQRddjNJqXXC7a4OA1e0lepcLHhQ8bi8ebuvHVBGVWfZHXHCaI9ZMKEyjKewpSd6K0KLJ7GNtyLryVRbyW1QULxNm569mOC0yYbDQcNYCG8S9tOoi7lQDE9UuCupTJpZIXQKN9lahCqfVcJ1+FebH+deO99eZQQkcL7bRCmYwaeUezd+viWOFP\\\\\\\/iRNy5\\\\\\\/G4rNNIqElZX8JcbiGIX02mQlNeieLwJXkMuqqXbw0VE3cZDg3Gf0XtAPqoIkqkiEzUpI4QGvHPzfmirNZPQSyoYBDKXUQEELJntoTc+OBwLmywD+2sx7CP3Pb+AZpDZ0P4GDh4Kxe6F83pL9xf2j93R\\\\\\\/B0er+T1QATy3\\\\\\\/pP5LWzwo9dHhd8oyPKxVhoo0NMWqxd8NirZAN81e8PPLgRWt\\\\\\\/kCCYOFmf4qjzjMXw2iBeCPT4R+0eJmSIaESPCjmj2cPRFh8gLRMsGWAqw02A7\\\\\\\/DjAlJShuucSDJjrvsJcXYBtvmncZtu1e1Z3\\\\\\\/F7QTaGKu8JLEHrU8Pf96sOMJL\\\\\\\/S0AyTaJLRwhDiL1dVzANytWr0oX4p1mzjUsqxu13OYjq68aypa3AZzRW5qdIETFcXC8DZbS1o9SanW14S8a0LTVfBYZgiNMqM7wOTvCS1rGNY+wD1SQbTdggIYQjvMCKaHS+c+HGqyPIRCY3VmLahkpVNbhs8m3X5CWN1Vcx0pW7T8uhUrDQ78X0qa4H5435I6xALQNVQr5si9zBJQQjhVAOcDk9cp+N+rCfW2xkm9X74n2oBqeVfU6qADJuMaXJfIPAU\\\\\\\/ZfVmwTdodj8azl2o4tw64Di4XeetlKmazHVDyic6IWUfsYNuOkt7CMxzG3eBJv2zNqrP8Lrl5+EM1cGAGDLLPXWOMmwa0f1JVNnjslyVTbZjK4BQWZXVE6db8Klv9VbYw2wByQqDNaDimL7qu\\\\\\\/bQOoBajV0vJ+1C5GNwasBFrhVtv+jxJXAW1+CiITD6+UwRSDCkVtkKwaSj6zw0po68eSNJTmCfC7NsW2zoUCGD8vncZU\\\\\\\/G84uDJ6dAWXNEs1Ur8pltW5mCPAo7upG9LibM\\\\\\\/151zahcNAr\\\\\\\/34LxFKP4ToCLpmHF1IiJa6scyrYtZwFHa9yzVtc13E3RA+JeQ4fvhYeJ\\\u0022,\\\u0022iv\\\u0022:\\\u0022LpkZ1o\\\\\\\/TYOrzWLX8\\\\\\\/OjLAA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b8de9cc2c179084ca03f0adb2cc290683f7474a717ef754cf2ae429056aaa08f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qsvd6pQ1BOzWssdjF5SvdIMgj3oGFEyAc413N4BhVTsag4tR1pfQ2B1sq7tLD4iHCpl1xZffA7TyjoArlSut+SQ6mc7kHoKuXRZQEhSNBz+a7p6\\\\\\\/zOk\\\\\\\/AMWkat7iArwCAYe8yW9J2Lds9P\\\\\\\/I2oVOH8b\\\\\\\/0IfPPbBeFqPqFBOpgzhAtG\\\\\\\/0Gi21tKYFMKT7M5oIXgqIzKrwuYGNP382WYVcDr7G3fibKVkq0VNaFXqTaQjTjpJ99mZvdusKL5vTJEEsMRaA5YV3uOkRRQAGt+0zUD3D6Jalj8OxeezdFj9wXbP85uEm6FuCBNPJ39Qvt0KXw89DMTzyGOQZFb8irz066WGHpr8umthUngEVSNx+BdhymI8QOHRkpJkQ6U0ud6GOISIQU3wqFoB\\\\\\\/cf4ZyTLXsQE1am9qC9XebLwKAfE8aXNkavKf0S3m21XsDunWWjG8OUG\\\\\\\/fTZSiSsf\\\\\\\/THKZ6PzIGEwcZOfphLyLlse\\\\\\\/vuPHsjrUKRpSnmH35EXnhZZ9piczsuiTPNVZNLdDV6ckAYyVBZ6SRv4ldhNcfak6O+SsSGTZobGlqWraVh0evdm7UZHUXRbmtYgZIN5+TBBlTqFXSPsnBcJiWWS+XBfqg\\\\\\\/3rQ5EQKp86KYer\\\\\\\/VeKgQ5CX3e+S6Hxb4z6gMGVttnC3bTUzBtZ59fDDlS4dQUojjyqXso97XaeRMi51ROOOvRyw+TbZv4\\\\\\\/Q+VyEEaIlql7j2qJNaW7Yj3pLIwGPOQV92xq1mlEfu69e3mSAsvvQt+lz46MqAXcvlr1YJhsEWdUMZa0WudzNhiI4N1eGy7+y2p\\\\\\\/\\\\\\\/HvmLPQLeZyeMt0jtxpA3ebrju9XhxDKgvLEpzKUT9pTPLHcdEYtrQQFORyol8PQNj4y9YGzWV1IVDsO+ts0ISfQ0bLzA8SFvLRiQ9txVsdOEvmIWGPR6EQYbtunnSaEhC\\\\\\\/xsVhxFTl+xjtG7YeiamDEzXCstYsWarZYE72ZyGpFuD6AsSl3KuW40FIjlyFArqQlbj3psEnhAS11UT8a23U7J2O\\\\\\\/fsjQN6o7z0aB5pA2VEa3Zr7J7pIoLdH2yGfGdL7\\\\\\\/ouUXM6F0O4Bno+sNxIM6CkKUvkP3lZWUnRaKoq08G\\\\\\\/0zKXe7jWATzDGfrCGfmXzjknwbBJbJaLZz1tdDNoPROy08bWiqDcoOhZIraDquMFy5gVxdvv6Mx+wySA2p\\\\\\\/EtVNr4fBfF4Q\\\\\\\/YblnZid4xjuBXTOwZQDuFF1TKpByCgu34bIFGMY8S4Tv\\\\\\\/2Gs\\\\\\\/+rzNcM5YPvKO8QjqWjZtfum\\\\\\\/0vgZ3Q2\\\\\\\/9mNWifYDTi5Ty36tF9WExNX9XcC59grdYT7oGCmUWWx9YqbF\\\\\\\/7D\\\\\\\/g1LalE\\\\\\\/8IXhrAlBrFk9DoO\\\\\\\/FiD6bvpdQofdZnfmpyb7fj56YypdJ4qRcy4xL2s08HQbImHKUte47hlQIdHuQ+TE0AuV3EqheTiwnDgQUdWtsuHWLCDAGarqqmlssg25PNLENjqZXosk30HT9gclyqV6CjCNmn7qQXiMkgMmEBt2M9pIS7Z\\\\\\\/Jnv3e2kT+GIUECQk1Y4was0O9el63DoG3cvH8QRZvu309wjmEZZ4oVH8f5VtwoRhnp0KXeEy0KOs40XTbEo+jz2wIdCbRB6xZLcCHz7UrL5fvAoBXK4TRarrSEqt7RbxuaA87Z7QslUIuFKHc7AXCx6Edmgh3yXHgkmQK749dV418vgusF\\\\\\\/01b9TyYoyCbOnGB5kC080PgLf9wRXOSq1fZgkbFqZTXCvAU\\\\\\\/70Ey37YulS9pSqB3aT41EHOffLNB0B7ztjNV5zuHjBbvD4GWDJUZ5o1Axf6lw5Rh5+YiqjXGg9xW4SDYs4Qoz1V86sq3Lks1P11LB9C4q0XBDLo6ApzRl9Ns6YcR3NpYiDlb33J39U36bdJDSaZrKjNdKjI9G9Ik0Cm5S0XouNgdlvxVo39PSWJYkXknrDtzU8ZV3mo5nPu1KCUSZvkZBg6GfiB\\\\\\\/QizPAXK8ehZXk4pJQGcjY+\\\\\\\/WpsHBt2uDGK1u5GWeiVl4SIU\\\\\\\/D5ghgIyJ6JzLnRv9lLJ11cT75spJHjnkt70OA4+jPEKyAPrYiIPijQARFoBdYewxj8sunreXy0xhCKp\\\\\\\/eiM5N7eyySPdTtQ\\\\\\\/cvlVHRnhOAlx8sgHpvl2X3vM1hAEFDXHLeVtjiySJlcD3jYdMTf0U1iA+h9wXcPpZH2CAfik9mPVy8te+9KMHiEgihhpD5VOl88pB9Md9QsXn3aauN4uv0eIbJlJmZ6wfRybsYC7QIjGzfyiR2wLWrBXxZ\\\\\\\/u9jgnKT144pDC8jR7fFTeUXH4VoWZfMiR4KkC2f1f5N\\\\\\\/T\\\\\\\/xddVCTPmLNRU49GdYp6uc7YhkcACRWGbvdNE9aEAbaf7BLu6lH7n5AdbEIUgQ\\\\\\\/LjsTPdVZ5\\\\\\\/7y2i03GuWhera5PlV4R\\\\\\\/u9ItGHGieFhM5IfDygYFq43MJNKbSSfBIBQUE64EgTp1QVQin56OJgYybeSMEDScAbsIftr1Nquw++SkUunqLjEQGd0EmTKrJOO3+m1PSsK8K3peYq0RqL6fZjQxQe63kMl5uRsJ9Q4UTjO0WmFDx9Z5cJBDSuxFytyPVazDArEco8Q+7vzcKxKBOccpiCG5fqcIi9G\\\\\\\/AMUh59F2uhN8LIM7ouprzP1qgwSPNETQT+9M9hstLUQd4ViJWLPfvPmXvknI4kzB48GNpKiTOnFskh9D3DjGb9p69nGfOJDd7W63wePYu3oZg2x7\\\\\\\/hrDXYCWAnRjn6jrrZWEP3tPinWs3xpc517Gc8qVGT9J33pX2ARqT9MUQjf5CPAhkaRDBVXhk3NRt7pFnGUv0bnpTuRjP0cnGhaAt3oCfoB\\\\\\\/sliTcaKzIHSMpCmduJrYrNJLKZ\\\\\\\/c29+ZD+IJXO9dCN\\\\\\\/lSGfBOzW9npPye+D4A6TkuInR5Q4AevyaBeJD1FbOV6wp\\\\\\\/2bTGPANFMDkdIEnIPqq7Wn\\\\\\\/\\\\\\\/YJeCns7eFfwisw8klsMEC3dXO6RjduFrCTsKDICPvnunsyDhDqluLb7+U3mzvffMvlJyBPvfbBpaivemv7adXW8R07kbGL83EWYV8FK++8UxuGpO8izncutH19hvdXnrh09hVgKQbuDuVgkr2d9B98Nh4L5Aay4BkW168\\\\\\\/Ebri2RcO6XSYyzHUGyZz\\\\\\\/qdG\\\\\\\/ywVTsPtAP2cWqLNYO6l+dCoIq5VXvLpQjGp4aF59KQruWz5MkSsRnusg2I201Damqfs+HPGTWhe+P3gJUoPvtf3nv64zo697p54hWQtBSSbjY6S9GHdIY5NRaFkNMJH\\\\\\\/AHuCOfzrQfkfaGE+R79NZm+3zowmH+aEPUjP5zVCJK1vj\\\\\\\/IhWYyzslydfkPBMKWbOGsXYLiWTQTE0h9UKuCmlP721YZRcVOEVnTbAar6fCXaZASu\\\\\\\/yKakrr0RoqzD3IEVrUokpI5v7R3zar38nV9dlbzbuOaaZRriGD36WtdpbSXRomH7hhkOM42MW5yjZXq\\\\\\\/GeITHFvRI8rRamf\\\\\\\/HJEJbr2sRH5eOI6hwxpCuGvI4jO3HvAcl7+4iIh0Im9Y4ZRiUzmVikSnP0wGsxQX+wAMYI1d9LSrjwObGwaqYEMWlsrDCXuUvFif4+2O4vxvE7QcYgCyi\\\\\\\/\\\\\\\/KQgNTSvqzRCzs29QtcOGY9yrPujcSei2y0FSMgjnW4IMnkes+serf+n878+Zh+VC+x+uqN\\\\\\\/Uu2HNL4d4myfHL7MYPC0n1cuLjP08E63MIcvNx52XhJ3tLIfxcf\\\\\\\/aYDcxBKlhnd3DxwFiT0Gfxl367ZOkyGRw+gczQzMDmbieqCbentavj+ZjnrVWsMq1Qwi1tzS7VgpTjQeUKAnaCICuKrl\\\\\\\/NguQVOaA\\\\\\\/H8QTKFZ2j0TWQTMKQj4H5ZXlZDHlM2uNGsRxPjpZSHqf+hNJPtYGmNAAypjC5y4tQP7lBQSaM5KzZNaZc7pxSbR4Rj3wyGRv56PL3qd3B3CvDxXXHiMs0nqG\\\\\\\/OyeGyGPGHASvbFe3NfEwqznFnkoY6ZzinU9PHRbGBBuQpgD+Dp8yRlSoZc0iCLlkp1J0FbIQmBXDwUNOt5M2qj2RzzoPpltEqXt3WQtBY1K5zwKsLSwOY56gthYXpuKbPpCP7Z9vKtF3Yp74spU\\\\\\\/JpJdlfOm+lS6UQJkJaIhg29Yg6m64GZlGi2iIyP3TIhJUPYyBT4+W9xvbByaja32BZOAMmBPvt9LNF3r+1DI3kbnB1k58g+M9uExaBlohyLDqGgsm1RqQCV0MSBnfjQdyy6Xj0UpOb+0YlU\\\\\\\/\\\\\\\/Oec1ASyM5\\\\\\\/5sBnkFJOzFJpecvdbNlrzLv5+LdG1PbG8TkWkzATou9I51Q8qCcKlv0m36mLRjZ+C2so75DeeiJhCSakM\\\\\\\/h88kYyOh004loxXxmKcXyBaNcm3Lp+wuik7j\\\\\\\/2ebjZM5AfAkbhYgHhGtmAdKbWmhYDSpKVxhz+k\\\\\\\/LA\\\\\\\/equCybootYB3SzLIQ\\\\\\\/ny20lVqhyrzhqivjG1G5kEDs33ANM65kVTPTiXTIa2o2vtT2l2I\\\\\\\/pw3bghOIigKzUv9BD6Z\\\\\\\/9dgT8qS1zdt3aa7VZTsbIIbnG0+fQ60KUt9SFBW423xoMyK6osvnp8KCiakPwg0PLzX5\\\\\\\/WpqFUpFDsMNO6T6rU3FuE9\\\\\\\/FJCdaJSvw40fgtg5P54mEjJmGX5oBGb3PjynNZb09VIun6NunNJ8OST59Cz\\\\\\\/ShqM\\\\\\\/JLkxN2uDld6N1ZxFK9u2JixEcV1mJ6vpmIalRAjsUYrFd+heeyMaK1wcT3DuZ58egT5iHZj6a+iy9cqy03AzFXpBu+trAs7c+ApPvkuw6g9wn9xtQ92TXre4kd3RcirjLULNC1pPUha36CvKQNff1hhdl3\\\\\\\/fDSAO5FOPlfRSRWJlF4lDfKaioPgP++OK5cZuyCFkB\\\\\\\/XuvnHKGFc3NRudvwOaJmMm73ueLFWprewovxs6SbNRVzCa9zYwPY+78Ro2NWBASfshHdO727plA2Kd\\\\\\\/KPFWjT5pEsb\\\\\\\/IRwOYI6zs72YWqVtNxerblGj8QSCF+05TQZVbLfbd9DC3Bsfi8AqanCQPh9H0LtwY8lwQcIol8f\\\\\\\/2uEYo70DyRKpYeJP0fTSuxdA1ATweVGt4+rY+cP2zpFVE4uk\\\\\\\/x6egzFmjpkwx7ngagT7Ft1a3GhJH1t+PDO2y7kaKkTG3ogn0q+3Mo4zsKM6oZwv+lNCLeb+4RDXuqvPPSmFvZaAf6TAI+lCcm3zE4AL36rbl0PvfneHEWq+IyEFy\\\\\\\/DVbpShR2x0eld9V1aLF7b0UpIlaVHSd3jKEwfMybP+FRS5vedlLO75aKLbyUix16PA2dkAPO9Cn8x3L1YzlDyM7l3B2zwMVskb6FYmmCl4ltOYaw+96e8FmbgN8BfUAOTKCgJvcavdkTsEJl9q3Jij4n8A0lzto2Hb5NapVBWS87fBFaX1fWQ7li28\\\\\\\/JPRIUQyh8+UCW3S8n6GDSjJ+sYjTQeMAg7wgBfkBaQf5fu2DBmBxxrMYoyPfgGBMV359Qhq0FnMX+N\\\\\\\/Q6yR50T3c\\\\\\\/Bct3Z3Wx9vtAX7MY4UzhD+8CLewSz67YuJKnHqahsNHdyKSRpvkbrD8NLYJyqDgPzyBWHhbq6nkWSvOmwSw+uq68tBOVld\\\\\\\/IiLWQRJPrlUxEonPaYhliRjU6Bn4ovEfrM4RE0F\\\\\\\/ALIuCLk+LRIMVyECwFB\\\\\\\/+zn9o3qJy8PUlWGEzAPBckTZgMbhPVOdUJZjI7Lfuwj5T9sxGmuz02U1gCDEOLPQIshdw6NIhSXaWCB4e+19iAUHwyl\\\\\\\/OFdGlsgEePZYDZ3oPvMAPDDJvdGg1dmOuCy6wicMh9xJ1IEVBpKVJ\\\\\\\/HE08WazEnFIy\\\\\\\/hJzurS47JrJQn5P5H6Tw7rxW5bW2FNN8aCZnywgD\\\\\\\/bGRih5EDtTDwVAQv4STNjKazbn0XRulhrlXWGicKzOg9bBClR3gHoq+FUTYXv6ltWwm1QE6g7UulcgALVyshc8wb1q3yxSPViET\\\\\\\/P6B+PNDlWR4aBc\\\\\\\/gPRUP\\\\\\\/NkricDBJrmyLklU8DQmgXFXerVNDappmV76h6HPQguBLVg8Tyzj3wwcwAGhKIqC9K5AwNBqxmMDGQuAvAn\\\\\\\/YVVVcrfr+l1kSozt3d5dbuGfo4sH74MQW361W8yPlvvu8zC0hIJ82Y1KORnKtgR9sjkREoi2yftX7eq1XApbsEf2PPKbSJhCVaKvOKLD\\\\\\\/5\\\\\\\/e\\\\\\\/h2VCwW3hEVty4IeE2vbGxuB4my3LcVBVgEcMHrYmnemGmaw9cxlCHeVVEZ49HnI5jbXqt9PtkW9dYECFkyd71+d7mPRC5xjmobpYes4zOM0YDOJSELhraF5yuZLEoYYJN3PYRLe3kDORkFjCMZq8U2PlzGMsLN5mKnnSIKRXeY1vE\\\\\\\/yRFx6JScL1YzPBAZr8PgQw8HlZaavX+Pnvlj5+4fPrTE4SIFsweY1Cwi+sJWD069eNYgKAK0W\\\\\\\/xarl51XVbcJI\\\\\\\/xplEABXLbCPTRqRl8gSLqSr7V2Fi8BIYvt\\\\\\\/3EboirZs4V916CG6YQhkU3fbWfJ2gA7dfUvBXuybuelJMqKrM2DyP2aYj0Yk7C5unNYNBJ2QYNIEXCB3U9l4B+uK82eYC1fD\\\\\\\/hkw8zb32t9JRUvoupsq2qDThQqCjnSLndVks99ur4ue3QnC78IRnLXl2g+YHXRbKWOVo74P9tlfnbT+gmUbgmFEhioZ7QU+sTle1vzfwBapmg4Q3zHHjwm5RJ45wXclQVFL5+0xa8xSeTw8DM6dRQaoI408H\\\\\\\/2kvDaCbJ2JrZhaapdYCJXPP8ExqD6KQDsWFVktFqwTJjEzWRAhptOZS8b9PAd1qRBNuewnSCl\\\\\\\/mINEwTrMiQTBTV8C19pyPbNspdQYVlF3TInP+y1M8t2ZMlw1I5KLwn4hOw4lohYD\\\\\\\/zUlMr7vXD9MqZhtYWPtATq0K7BqBq7kV\\\\\\\/OaMZxVUBOeIOur2PKssBX5XddLAADwE8jGUmTxiKUSDOjx7SFMUqdCE5AZdZUXpX0x+QF2JkeVS4+KmIXXvC6QM2E9TZ4tDdR9hz0c16UkNvmaxKDxe+iun6dsNvZvcDeM8x5MdjcekTXJ3ODWvYQl\\\\\\\/sOtmT6FjiMMDLD9TGGgReRDK3Fst1LXVgFApaFMl1KYfGn6jYnMspZOaQE8KbgrVxcxhPfgWTFol+vGQnRFBHj4zhVXNvhHFIL9RA38XiBHNvyVDfzTrlMMJCac\\\\\\\/D5H\\\\\\\/Zg+2p2PjMHZ7vdzo0Up596XGK8tX05VobyVHTENMmYs40eq+i4Or819xDE+7cn3DfcouoynF24E2hebM9kXniWveIsfaDnlKEZ13nlnIUEsaTl9ftWr\\\\\\\/6VY4HDqYglnTn7xC2t1HrcFqFASsQU4HbO0F4EBsl+pdRED4BY8QG5ygX+omqz+dM2UUHfM9xkBb9vCuN95Nuxhu3d+Tq6K6vV04eBOCs2A38eeRjzWhNm3wgFp\\\\\\\/yX9yxSbxds\\\\\\\/Ppu+ceEObvZXEBZAJC6Vg7PkFi3su7jRHa+8ovqqayWS9opUPUk1Em1gs3zHFb8P0+9TKtPnamNLSkcFA42S9TK7knJBHuuaz2YibwokqWCgyewMmad4TDfCqw0ihn1d\\\\\\\/qFES+4CPiI740Twjwh16VVI5FHNBVszf1tMdIx7a\\\\\\\/24dSA0TwPlFu9p98VUfj646SUsfARTy3pOc99wBHB8GO2+5uIJFgNJW74vqP7Gtmw7b5HvPBB+\\\\\\\/9Hh5FIVjxvaDg1d\\\\\\\/71AVPEFr8NAeW+DQ6YWDiAU0ixcvsTvJevDLBONhnfzCNwiHtnhUAhMV1+ZX2Tfzb1XlIo3RsziZBlG+QmBzKzw9RkmcEVxgK8k4Efs8euGrfRUlJPFnjwBTn5lsSUD3SlePTBEPdivMGrFXxxpvQ0urXshesw4EyHOyjf5MnkvkS5PXIVl\\\\\\\/yL7r23pbUFU4VoAaKDNhY3jYW3h9BCavgTR2ka3ExNME9hvaz6fXtLGwrzMOgacgLe+WggWHe6vBEpjdPchknTJUvB1Nf\\\\\\\/WLJLZifWPAK\\\\\\\/Ca8Xq9sU2vwnIUlcbIG4JHHnpe8oQ3\\\\\\\/yuvX\\\\\\\/rBmueDR3wawKrVS\\\\\\\/L9xwh0Xc\\\\\\\/7jOXiJWNz6xL6qqsG194HsyloseJV2X6tEsxWa2lL7JPOrM8RuLdTLCsWDd+eekTMKBspLriXTBrek5AfD1wJ5TQj318ucWd9n5eHbvRYMWzyaLAAIGmnQTPjAFlDXc5shhA9J4DpZB6pBDGw7X6kY487h+YbBJHdHwPnPGG7+nHM4Z1BdCtzxBj67rDdkprsxstWxkD9\\\\\\\/In7jiHmzOp\\\\\\\/I5htaatoEW12DQW\\\\\\\/gT+BhDg\\\\\\\/Q6Blaf4sAXcsuDyrjkB86bhqaIKg3QEv3G1HvM5I9ZZYChbyjw\\\\\\\/Znc+8bK+\\\\\\\/lBAm5TcJTQw1eTIBiptAWkI78kU1TJqHBGiFrkcp7cWGSwIo26pSdidQYznHdLxMHzWNSzyqDnbnDF2DE7JQCEDgPX+FyzsRNOVqEkLja0VBY4EZDL7CMbcTsh84sUr+Qq2BKxU2boW4ksxXolf5CArjm5yVuaIaqMJiAutaemhX6R1DUh9xdH5PzPuC8pFcej2nb3cbC\\\\\\\/YAG1Ez6S40YkIxkEDxDdWd2NhkTWkbqktI241RUkqPAnDgRGmVBG8KyiK7As5ML6ijUgXpcHxRtwmbRrMRyKY6IY4JCam+BUE8mVzSfQKY1p+BMlxuKx8ZJw3yd3pOsX87d3lzoC27sYhBg7ipkWpAcKEEoQ3uTyrXBJvVNFoBGmz8qdp7VE3Lcn\\\\\\\/n0FiMJX+9fub9574kbBdghpeuO2eoIQW5ctArUEr1gOgBncjuTfkUcC2Ew1oK9hckloJf1M5Ad2vQQV45r1uR8mtEkqSXF+TnwncCaQw5PS+GLAqeS\\\\\\\/WeqbYYwc8bWULtchmYcXR4S0tydjfEVux3TZ6Ng\\\\\\\/7DG1yX7Vn3nQUktjo3KdpX7eCCd8newTOKfEfzwNSf2cAfzo0Wg7GU+cdNOAFmAMbTDfIXEMPbHKRmU18g7Xwxqr0QIuK5OLsiDAp9S01urIcD\\\\\\\/cVQX0XncC73Gx7fytJfNDRt6XJaNOGArYhZnyCFldO86q9V6b4YwVwjacEC19Az\\\\\\\/ttvrNTdOIsvnu5D9bZvi3QE3TjsTXRvwqQIDt9oV5MgZqSi0PZ7czRvSoUvY3Zw9iPLevv54IUES0MLzOItxctyoSdsthcC579AX0RcdNjjGJM0k3sKeOmaAFhhK1sjGFd4evI\\\\\\\/Hiq7AUGE8nzJfxvDbfmokek1EsAtvaoLQoKtaukKTciavph1N\\\\\\\/OOetyr25FVMIT7+q839WyesfCsj9sXOUQxCGIpkLwptWb2dgphktcU4bjYDB1xntNUal4hsb2IGQCX2VBrgOtEWiJHyooZGg\\\\\\\/Uq+w6vDlFrFxkzKPpDYneHQK5s1Smz2vlzMzsSzOr1Mt6R0Z0LHY6tCEPsu6+77AkpAY3EPTgizgKZLdiYyJG\\\\\\\/H8PfmM1Ozvn0F5DfPwKZ3kviMETVUNOdu7DtpvzOldrcm1oG0gag0aHjYqaJwpg5mc6ZLCuKB4O2mzUprGXHfsYGX5+9DG2FsAOopEtbC2ejrIeUtFxELqli3nxadTC0m+0MewTUXuqUvQHR+6j9TKM1aMT4dt5BDS3+YrpNYwEPiyClMUqzUQI+d2IV+irwBIy\\\\\\\/TO1F8r89+1XVznxgsEKa3koja1JNPFMXOY\\\\\\\/\\\\\\\/Lei7t51q+7X4Mw0HCV5UompqZ9HhqWkj4dXvXAH+EId7qbPM1JNAA8rgzP9EPd7\\\\\\\/GwfLQvL64YJXEylhIcvO\\\\\\\/hBEbZz44p5l0B1jmoJtRqSCnFyGP4UrSjh4AmxmxqLRsetnIjLw1zklFT6Tep+j2\\\\\\\/J+Q6XrG4qAbZsUQyYSYzmAn\\\\\\\/lIYgvysYat08Qyzfjvrzkk3vH\\\\\\\/Q3bq5XTzHXwoZydqo1QMlykuobcdGii\\\\\\\/ISTYs387M+MZF\\\\\\\/wEPqZz5agcqX6s28UKiKTnJ\\\\\\\/wWC6FUvA6t6cw+Dw4aPAK2ZlA4jEwj5qeeEtvIwIPs\\\\\\\/q3h1858TsAsoEy9BDEqGkJqHJEo6hRAqQw\\\\\\\/vBatAmQFqiXVBAimI1rHwM3RY1mWVX\\\\\\\/ACE1DBNitSx2CPkdV2Uxr21LuKP5yFY9fHWsHxPgmPJouygFpOw5lNM1m7pTcw7XJ8ylNz+8kwmL6nGVcLc2WEKgxcKzTg71h4GhKqS03q\\\\\\\/YZRhxXruRPrR5s5acgL0bDhJGBkvDyAKhyd8F9f+gKEMu9BRt97JuRyvpAmNYyjG5avPvVQcVTTCe1tuo2EEzdp6r630eqeTiE5xzpcXFr3D5bK2W9T5f\\\\\\\/8+ja4z7XpQZQUGF9DUAJgeuc+Tcjg23McUcCfbOwkY\\\\\\\/I4PPRg2Fpy1psduI6+ji9d4Hy8jzgEEWqyIhzAdWQlSGWzC2mKY7j8uzklGi\\\\\\\/HKcXMaWulvx183LB6s09vPQISVQwzqmH+lf3wuqF0j5EL++hBNpCZSNmWQ9rRRXOQrVAC+KP2+TlU+\\\\\\\/a8tIlt0pPjgxo5tM0Wt+jrRPNJK\\\\\\\/WD9dJuY33\\\\\\\/zZ1dySow3+8X1P8Crm0duIEMCtDT7+kEVjrCGb7wPLtHEki4fHWvbYdxbLOXgExog2qxRdO1VN7pdFf1v8dRPuiKCuLPsueinddX07nT0FxLtbW6g\\\\\\\/LXnnkxc12DLL5gBlK3wHAIFu0WH\\\\\\\/JYnAKEMZJrxccBoEpwV5QAc1PSGxXL7CM75wI+LPyMsrfnQsumuv8NmHDMDfjs07RSJDQ8pDCbShb3jFHCfc2I\\\\\\\/6rAjOyWXiTgIlaEnaT\\\\\\\/ZMTRYqz63EsydLZJ9m0tu5XhTdsx1LSnz2DLta6C4fXNICEnOfn1KORJ5n\\\\\\\/eMXvpDYRqHb\\\\\\\/niuv38SZDW2AK61UxsC3\\\\\\\/ZSd5WHDtJi6BRXyu8RuZ0x1XvzetDqo9z21W0\\\\\\\/hjEwiQUoiebgbluvoj6mM\\\\\\\/Lx8Gg0+jFGrHe6K3Y9FIgv4DhE8Qu\\\\\\\/mqzU5xRFxEniHwEzPoTTn9mBt8S8m5FTWpM7kDVID0nkhZXiG\\\\\\\/ldO39EFyIOn4Tox9e3m1qXTxhohiKVeAQEP3Osg0PMRVoTLe82VqnSgM\\\\\\\/+wyiK30I6kPi48ZlHtG5D1g7aCcseWW4jCAWnxzcyqdMfCxIxyZNL2awWmwfS5Ql+iqeB7v8d7UAH4xL44zruDrzwB8\\\\\\\/u9j1DOJW1ALxV\\\\\\\/gL8W8CX86Dca5gfJdW84i\\\\\\\/xqEJXarKOvrjLcam\\\\\\\/mXrcWizblpNyYukp3zG3tOyfFciaDoIsUroXlUADWveJKF64kBbJU4guBYgAO7UL5dW8mswRItJYpnyuJ0t6qemUfBnlCI4VlPoDRKibS6yVAu5UaApd6uiG4IstZk\\\\\\\/AichBwxW+t3S5srY8nEa9K0N+mldq0ubOiPtk0BffufmXawE6eDBu5uiCiPPoxR6zP382SSm6TGzID3TcwrJJXBg6Hcix8fDjURrNwwC+X6qnyFWS6Omc6v0DbqUoTYTF5bAwDN4TL+OzNcB4uljoVxttbzhRymKo+SHbgmey7wfDGzD00i1o0WwWe84dGOnAVdYfUmMyiALUyByvvA8GCnhJMFqzUmFRnU\\\\\\\/t7b0Nl5H\\\\\\\/SFvJ0QIo2IVlEolRR\\\\\\\/dzGO6YnVaFFfOQB6gtvqni1rzpuYY+aU3tiZk4i7q+hGsmZ5s4T6Z929nb8gk\\\\\\\/fXTf2xPr5S\\\\\\\/vUr9x1XmQWQUIH4teEvnC6kfpnWGE2vDwPK4PbR9qH7mQLiRAWCdIKSgAmsRCuzyNV3tdBdrZ2G9ugeS1o9BM628oJNWgNfChieDA593wAVr+6KMUG7\\\\\\\/Kylb0A8HYir85Psyxmb9\\\\\\\/qBE5Jph6ZDvRa+MOPwFhbR5jcVCBtcGQK7C89aDog\\\\\\\/s9M2ezTGEClFdAh+uS5G92iCOLZpTmoFM13qh8UfSnX8C6tA6sfX4Bpw+KMmFKxO\\\\\\\/E3J1fXaqx38sVc5RsEXqkGlFzX3Hch8Y7wz8KqdnV8uTEyz4a9eYks\\\\\\\/77OBvN4kfTrhUChRx3pFbeAabnP\\\\\\\/MVYTNu94aDESOxTimWTyGku69UwyXE20RGLF6jh6H+79Hiy8LEGAgDGHsQMyJj5OoL+8lCYpJGafekhjFS70vgJJCT0\\\\\\\/cbjmo97SPIiEcneR8bZDGoZw7rroVsC00XOkEo6iIwmxEV25JJag+h4TGbr61JPbyoN+AFMshAH4z\\\\\\\/prw2gHb7JQC1XgZttpwyaG50Mf5JO6fwWDGNzz00wsQxP\\\\\\\/PUo+qi7c\\\\\\\/TBAH3ggRXo45paYfXh0Qk8JreUJ6H3lRKZKs8fFVvIyrjf37OJ9lMzBtljVI0gmOXTfSihvFrKWMokWpzGbiiH8+VOn4DqoGnt7tGityrDYLUnkN2LXmvHxsU9+z+7kzSFqJdvlMVSavnjMwE1iUtxdhlskWgsTw5dyaCeMaMumi7dZUanZvUfi7\\\\\\\/8yk0YawIQWbFPrzjXVRERFQSAcHPSL5xk3jGERx166C3NO5EFSKMtF5AN2N7UVDpBsDyIEcu3JutIFDGXZJMB5mGsj3Ivu1RDJGczYHJBojFGXt69xM8SlHx\\\\\\\/xoCowL3ngpLPXTecrAPDUUOxAdTBP0u+AeWsd5RhbPP3xCV8CMxygcTCTZRorYtxDJXSIYIvkCmNmYEvKlU5NEAhHET9qw20gumdHNVmGS2aAcSUlLkjk1TvchDVEfr9iuAuL5zExK8\\\\\\\/U49SQrKU8LxCToG\\\\\\\/p0ykhCQKlN+poSwuloQ1JzRv7qtVtkMleF1OLG3y\\\\\\\/9Cp4YPQxvUEUl9F\\\\\\\/2Ee4TGgc1y+MoWd87Ldx8dPvFyI5PZLstR8BB4CAhxUb0OT0Ny3QtdQsxoDQ+SIKZ6X05EiIxjdHsavo\\\\\\\/sVDgPVPDGnTREGT3ETj5ynRT8MMbsPI6gpdR4kfMBHK\\\\\\\/sn\\\\\\\/I3+nNSCSR09lDgJ7b+HGQCbXoPzw83h+15Ssifk8pErEsygZMgHhm1UP77Yc0evbj2B\\\\\\\/YBevqxiV5F27RxKQm9B\\\\\\\/PpqL316IhlArkSM67GKGFsT0ucZ5M2RGAWxcKV7pMpAPtojBtiIJI28t1tIDVwxHgqFoDUIPSRpUvqzKO\\\\\\\/Qxp\\\\\\\/5oASlJy1gWJ1HE6MykbbrqF3kObH3qXVlSGhQKpy089MO1jKEYrCCJjmz4hKRTv4WTQ\\\\\\\/xXt5v1QemQR+Rt2ar2nuyE63\\\\\\\/27Dpa+fIva3wULZbfqqHi8pa7zgF1aIh9obuGZfjE0mxqMfWqy9aa4QM6euuCrSmISJVn3dVkveg1elKgOBN6HaxUlIxflRXd0m\\\\\\\/Dl75OvjdT25QYcsGvQ5B9CXDjLKEWROLRx38jkRLKK4JcIJ8XrUzk34Xs173LnOj9mVIubCAcmX94s+XoX\\\\\\\/Eiw7UkgJ5KdSXAPTW3nFkzT7cbH9FuVf+nJiGVonEgiK73TrrX57b8V0Ki11X9mmfI\\\\\\\/1lvZm5XVjc+xw0AzDfcdn0ffpAslken1XGGnlqisbCe29jOoS7LmuxpL6ux6Ur7AoYtW2U4u+fquQqTZK8PGjvNtjiuQyJelITaHuaLwNeFTkesi5+KxgJ5Mdf8V4E5+xWFHE1TiLlV+12j1CV3XhrH+AY6+7B6DhCSJ4IE\\\\\\\/fDB6tjkHILuheSrX9wOUWAh39kVfTJ8BQLkbbpGhqC0en4jDaUqfvEXZJRtnUMKM+4XBsKN0y5ZmEYbckS\\\\\\\/3px+ILum4v8jl63\\\\\\\/q\\\\\\\/ygHfbBUbdqixCyEelqP4F5ij\\\\\\\/JmisshvXH0aMLwZJIQ\\\\\\\/jY4OoTRXR\\\\\\\/iZ8kTT+bP\\\\\\\/OVcvNIi+n9vSTv+cABBfyzml\\\\\\\/ciDIc8n78oOrmfk3JaypEsGRRtGcHW7IX1aYFKgofaDuR6LsZGK5GX8TKdL+PN+re2j74aINGG+9x3IJFb8ZnVYEza84lU8qqIeoxogGkuU8zDG5vN8GQx4BSfeAJu16ku5mnEKPg7MfzIWcC4+UADbmoqSz6ID1d4IZ0YQoB5DRjapYqFkWWwLy1cifT5OxXr\\\\\\\/q3GGNdcgiRgbfUKw5GZbhpxcDYPMlJAcJrFG94A9DB4f3ShxNQzoTlHomzEhsTIRRhs97w5bAtR\\\\\\\/FB3k8\\\\\\\/awTdF4wV1WjgxXHMbSKFQhl750aqJDBfWa7UgOFKMsZuHLEK3K4RoXLXFBfgW8Gx6cyKBwKrFlv5bwSgXFvvImZAcimbHVn+q3Qtc9Q7eChjc\\\\\\\/9TIKnfJ5Ck1cIn3WbMn6khjg9LCDJciw2tt5aYc7\\\\\\\/Y4FFzR9vGl+Bd2k6m3Xmdi0dim2k4Fg7298rfeO7KmtSDDxzHCV\\\\\\\/9y\\\\\\\/Kgjvkr6kv7RxUkpNZqS3I9KaobEP6HNVJxAMHk+c3tg95F81lrgA8hyEODL5XX6UL7yrTTvMV6MxwJ6\\\\\\\/mrC05Ix2mPym6j5IihCkugXyCKroj00wbt4lD8ojBtjZdQtXw\\\\\\\/RSq1rRMTnjmnudhwAcyen9vnRUfKeECx47wUajmZcLvvXZhNcQEp7+k3klTuIvz2gaAZGbEMlDCb\\\\\\\/AqNmP6wrsxgSMMr8EeUWEh2BL+u500gAFI8YmkjCM+EOA7WEFKZToBbotlN8mkxo6ubkjiYO2eMwnwnSuwylPWEbAivo3oLaFe3quRm+MVtC9Ai1RbpttmC4RZiWNl9VqD32uB7\\\\\\\/GV9EmNtXTTjtrZYWXduhfPQCclEV0zmDCLiRQD5lvhdSq9HjF+t1JXWPiB75VoAddS8R0\\\\\\\/CqzE1Y1bwaGd22VHDGNIbpr7p9Euj3EmzWNfD0t5lXaP4bWRNgffJKuBSnuRFWrVZ1LTvao8XumCm5LXJvGMv4FO2ztGGp0iBGPDZpXEpf8dYaNxdOpTOP+eDK+MBbSMjNEOnW3iRSyy4U2R9M9JYza\\\\\\\/Wk\\\\\\\/EeL2uUWPHOdP9B7Y+pVl+LYMJsLB73fKXm7E7u+l9N3KUfOvazl+OFGxF77VhikGTqXNAVWg1k0mJtmXwi1eJg5YyKrBeYf80TwO0\\\\\\\/Td5J4XY+TXpyP3Pci\\\\\\\/mH3MJsSR7Bgk2bnSkLk3Ail4lnhh2yBPw3D3phbZC7bQk+EERnuMKEfdC7Q+HNDdI7OPoio8bwSHJcOOjryp0wINQmh57PL2a3qxwoNfFFQkc71VbXguvfeLpicK4PQ6JSC3GfCSZaWWvWsgL6rgjFPyvq9+wlSquilWL4UVXSympPdYUaRS7LhL7dvoDdE4x4cvLeX+UMCCyA9oGCNwaySsyw3z3vbZBO7xvmuuIngVzzTB4se2Ze6HsJFkI9\\\\\\\/u3VQ42\\\\\\\/HVhcRaJlEjJXE+CBWifrapm4bEWaD33WygJqGpCHAMdUfE4wCi5b4Ldlk9RIpAV80ibnPVvMW+UmLyknkrbaLeG8IxiNXR1dD40N3KfhwElMJBelTcG+Eo656gk4YQCz5glNhQN5NqGRBg1wIQViU63TfolWkfeXIc2fno2vkKlpajtTyDh4xdQxbj14ry5W6v26HmtUcyx8Wcj+Ud1BWRRM+ga4G8AqIgy2egQRaadhpBSNKiHLtdbdJeE8ZvFtdhWPSGcyf0Ut4B01pmJAZKNOnGsZycGchKzbkzWUfdcg7UOJJhVbwwB9Mx2CPr3Ym2XgOCYRpsNbSlnNybZ\\\\\\\/iN54k3TpW4Yf\\\\\\\/X7hQD\\\\\\\/WRdvCO0Pfa5AKDT3j0tavpO7\\\\\\\/scXrplUX74ocMu8ndeBIHndQLThmYwCvQ607VjDgbX1Z1qkjELMGpwkGAkS1o5RphofXXEaRepCHh0sUQA7mLBJ3XYwxZxL\\\\\\\/+pRqOKMsqQQ6rC1WBUx1b+QzKphtBPcX1yoLLEurLy9Pa5q895e1N+iGRfZsCMmCZxAeYkKyqshNoghqI8k7XVePchi8DHdheEbXl30gK5WTjMNNLzzS1d9nyM+bS5L35azwht9R1L\\\\\\\/CcFT5ymvltJUg2Ta7GDcv3iYPFlc9pdOMXOqQQffq9oUkm\\\\\\\/vRnNGI\\\\\\\/N7pzEpii+Um3Hk\\\\\\\/UU7TXDpoLQbpyziOD+zarp644RdVJfro6TnHRb1457elKkwbSfHVU1Kt34t4x69Pg\\\\\\\/vpzQw6gCcdUJAbr5jfaEQIjf+nrmCbTlgSiurN524YLOrmN8XebYWw\\\\\\\/oNxizwTBEqMEsidqsmJZjicBC9GfHRdhhjDfe\\\\\\\/HnzkRdbDZCK3xux1xGhxGofPdDKnyn3wJknBDSnioDexTzPGvSvrARYGerxNTqhDKWGDqs3yzWOMWjq43UZTOZcG+X1EhUGiT3CMSV8Aar3SDY8ECA\\\\\\\/6Lx\\\\\\\/MOJzCFvbJAAjLwL41ugyNq5wTLG+Es1VSYcpkNxM5TaJ6P4Q2ZQOkyY7WTYXi3JJEfa\\\\\\\/L7Dn6Gv8MRN0gygmj78u8u3YGAC8taFvNAsHNKcZpqvJvWqCGo1X7iYQ2OVj2KUtxqSuPug0h8NenJBvUmiVa1bRvbB+7Nz\\\\\\\/190uQMJX17wrXQyPHSEKA0o4zX+7ov9AkGYZ8wUUN0RDzGfv8+daDJFvL7tFqU1rHKBlqkP5pI9YQ7SA0j00npqKD0MAFKS2Cr77fjXXhklnrpfZ9AsmQ70uxB4rDmMtkVKyEYXgNk2kAjzRDs+w3NjB9xHQuc7+HnjWT\\\\\\\/\\\\\\\/nhMOUr32PcWYU\\\\\\\/vrX0Z\\\\\\\/CEPl8M69XnS113gTbNe1egoB\\\\\\\/WuYLyFT2D0j\\\\\\\/GUaxf2C7iWRfpwlLdijgB7qiGai4lm\\\\\\\/XEzWtz5lYkTrCdON+sqSCjE8hw9v0BMdkf9BZCmu4IuEpK0MN8M65Qip5zWTST0CFnvPltAr7zMQl\\\\\\\/sbNdW3gaFOaR92y28fZcRMsIHlYj\\\\\\\/m+vUNTpYYuj0c0j+LqwQ05VjSaRjFwKrVoJ\\\\\\\/kTXT\\\\\\\/cohNIuk+LKqWNDsad\\\\\\\/xNpVB2MT+D6d6UeAriYeEq\\\\\\\/MopfEtWr89aNZCTCZyc2clRsNCgu4HQfZiYXSwYZTq8d\\\\\\\/vcO+\\\\\\\/WGZ9U9KdCHs\\\\\\\/70H1mEMFsdYS5mw2QilZiGLttTkhLT2sX4nrvqnUWhcOc7OE3M8a2pAadIgi4bxUrPnvo5PfjbnQxM+lh7zAFFAyOUO7fnRO0CoWJMY5XVuSuXpSC+fxBEM3yiRqBEhrhjnw16sUcPDCQi9MAHE4zGMMiTuLNpYZR8cTAtq89FyT+6zl1PI1QrmY4Y\\\\\\\/bguEO+RTjsvbT1he6zweGbZeJhKAQ1Nw8ZqCoP\\\\\\\/2TrObeV1HGSggkY5qw2+Ae4aBQfckKqY7NIbVk\\\\\\\/M421eB5sZRknPGFo1Jv+8BFqPnWFLDoXShQjPjyALDY72z7gzslb1ekH\\\\\\\/l5\\\\\\\/i9sJvwmGSfDWCUusAl7OwBYi5UcyfYCIUow80P8DLnSpwgx+Likc7a6VdkbsrGCeMXBMszyx4FcAI\\\\\\\/KtJuw96i\\\\\\\/RK8RUpXfIW8G70boAm0kAwRNxaG3dGs2KJ33+cxvPZ\\\\\\\/\\\\\\\/6xygAf9MHPixPVsVYg0CbuxT70F4Dv\\\\\\\/+1yBkx+Nxoc\\\\\\\/akaoCfYiHV0ZjfAUTlywmCtFgLO0XmqvNKXpV6HE4l+cwtO59pYp6zP2gVHzyZSLCUzjfaKi8fUsEPkplKQ2T2kaeoGXOa9u8O7ldCEnG26LvcR3k14+QEfb66duvmFKc0pbRjHoCWnz\\\\\\\/8q\\\\\\\/Kf6Fbt7nKRrXycMbs3QuVOTP9ylJa1VlGcasoKpDTEfSGq7ArlAin90vqo+5KMM4U0pEMsB3bkJbLClAcCC+amsKp3f153\\\\\\\/MXCIyB2EO82Ezohpv3vXdjBx5nU2QMouqIiDG8UlXp7bkz6C8vfRT+w29PjDZOuJ5e0US\\\\\\\/xPRiZk\\\\\\\/9\\\\\\\/TCY3\\\\\\\/NzIWE5FrRl8ajafs1Ru6PvCZyt5CvYLfQCH7PabptkWB63VXdg4VsGRcFNFuGTTrLZFcM4+t5w3+T+gxR3LNyh+dWOCvXHXoXBZe4Ufqfv7PRzOpWwGthypy4U5ysQgv9AcbyU+aooQGIL+nrqEk\\\\\\\/SYQgRi4w0OKwye3rJ8DZt17VoUjdhogsIPXQxm6Asg9\\\\\\\/abR9o53bEELSbIzrwTSBAGnHA9s0FwvhTSXvtQsMwdSFhJEumsSleymnoScSFXmB2wt+HNU5XkHuegU2obRrZ6lvi9C1Gp3aVCK3CQYCbj\\\\\\\/C+pnoZ9pKBDaIHFGfYQl1ZSRJc\\\\\\\/W+f6nUBndXaa5\\\\\\\/BHow9DF\\\\\\\/2M+8CC842jrnR4JYxZf1VU5U5U8h4ETmUAm998qA\\\\\\\/g468PR5SxOozHRX++5RYi0Z6OO7xFr76I+o9lKgQt0CR\\\\\\\/N9pV0cNqghsYKIeBmai04RPECxWCSEnfU6o0UVZGU+g4V\\\\\\\/xySoKNDP1d02GMUS8fgoWoQYUplRlZpQs3pb4I\\\\\\\/71tXfgEl6zLfTwfeo06ZfoRxDU5r7Ba6W75oODAQ1JkUeE6ghKqP60N6fqerc3vxYr1TBqQCwSkctXmbBxOa7oiydOCkkjv3mPY4J\\\\\\\/z04b0\\\\\\\/iNbF8OKV81DtvP1P8y1SnmHDFju9pxOQMPwaSyxy4MJvjReXfjP+B\\\\\\\/bCcoALGdgcRuqPFUvCY6STpU+yhV\\\\\\\/bNTCkgeUQIICzZWD+J+Z6Xc\\\\\\\/ULxqDdqQaeN45MQDzns8jbfXDf\\\\\\\/gmOE7ksyZfm11+UlEWy5x05GLxzZxGnV7VLBhdiQlcYuN7igYP3vNdNHFPTQTJU9bfPadPduV8asXp6gCuI28zzV4nyFrf7T2DVAeHVloqrhOe2ksrwsa2Fp9kTOkrSlecBbMbmFxISIjIOHUZ0+DaWZBml8wg7TNcu+WKvqz0uiFYb4HwUMOagyU3ixRiGXDleGsG75l0wploO5+L0HHwTxYNZ4MfsKudDrQ0CDTFi3v1OIXI6a\\\\\\\/mN9ccKEqrIXqV90fsd1tmjXiVZAzCO+XJ6nQZrJo2jmXRjC1prn8SM7c6d+mrFsPLe6TmBko+DvC5MI\\\\\\\/nK6HN+XO4rZ+B2bkpaQHACvjBp9qWvtSSc0Qg0s9ErqiZrCNSWEeFeOT2zjE9Hs4xfdRd0v7favlyU19Ia6UdZrFf4atf52IeOx8r6W7n8FQENFxJ2BGBMswRWV0lBtf05Wcd6XBnBatWcYM\\\\\\\/NLRil\\\\\\\/yzXdOMfXXfkNwoo4UZvoszQYiRvXfHwkk6Ia4KyoR4HV2YI9Xp38RQXIuuzKcndOsCtU864hAa7unfo15GwNTAWbXQxws0+rWAu4EfMbL2cfhbNdimz8IuDtVPzDVxqQOHDQ1lUWrEuLhyCtmnm8rHkko3axEPD5+W4WBgS2hnioEu5T6TgzO0QRDCOjlFRE4\\\\\\\/umm5c3EKd+ZoXq1WUWRBTLHi5pDq8TrX5sl+HlxAtztdmCBL2UttkZxwWx9fpCRFZdvpGMBIujsUnt1hrHePtLo6au1zndj+rrHTDL\\\\\\\/gv4VcsE+6cPhT\\\\\\\/Pccxb2ThcJVWzbuYl1GFdabf1M9FGUB7B\\\\\\\/UMK+Za7jYdeqbN78SRGwguGGiNqe92tiOezc\\\\\\\/H8pYt\\\\\\\/+vLUItaqH6i\\\\\\\/L6s7gsf0HuD+UmhARpeTb3yg6oktiQ8Fd045DIR3Knto08Bj6D7uDsI19DgZOKG5ykYxiqdIXQ2P6S0USjnSY3pF01H\\\\\\\/f2hBTFwEkB66XLLfqIV9jdsUNAszJ135HIamKWQXmfPxJhEXNtdGvGkckgZpM53OoXxqHb9izROD0qNXlnQe+rTcpmK\\\\\\\/P2fGaUJ17AE3elVe\\\\\\\/Q2qlgtjJOPxkcER4yRTSnMUVvdtPMcp8RR1iSaagnSKx+JcV7CFYhIJmOdLTm7vfMQZOmt0YADQK33a+mEAuTKWxMKpqgPf2AzfVxi0JT4TCSIhqYTz8sNfUj9cYErjS8SKhWV2g\\\\\\\/hjtA3gFtl98UvEuNNnakIlyP+OiwKelWmQjoc6dIEUURuWXV25YsA2sZozyxp2XxeXA5vJ7PKmb+fb0D8f0Xb0m4cZHWIO0MAZkt2IzJTkBDGWZB\\\\\\\/KuWVCwAN1UK7SmdwaDf1oa6XcpAsnF1\\\\\\\/EZSQa9ENAdE8rWB4WAcVvDAav0pdYE7gTEJXBigqJDiVJeAOVQMBQVe7wkuVzVV1wkliXTySYdGnIj0344kB0+\\\\\\\/ZiU4RHhq2H973szMD\\\\\\\/LcW120N2tD9AiRMgKeUY908Y6lzJ6LOUXGs5\\\\\\\/wc0eCJZqwCDY+N9RECIVbUWiNoz1w\\\\\\\/Uaw0JS3KQEpIH7yvnQV8BlYxNP9AFTlQzWpELgyhSVgH1aVDDRWOKHc+G+4dCFjMc0mDcaUP\\\\\\\/gk7t6mGJk248yMSKp9WOt7ettNBdTk8V6JBTkMP00DqLvNV+iQwRJ36BdLHjt+ARHmQN09PMEnm6Mv2Kb0nVfQpFI+8JLXp67HBNWPQ3BGk+0MobMCyC90GdLVkkl\\\\\\\/OWQwAEyqVsYHSeCgS\\\\\\\/fSMaNnZcj1tRbjzbYgTCFrI1TUNNOc+h2VkIv6Xp6sjwKE+ErOeZT1SHo3W\\\\\\\/phzF4kaN+jJi0d587vuZ3Nmtmtyxj\\\\\\\/h9\\\\\\\/W02vzHIki3HyUnWokJ4bzuC63OjmL\\\\\\\/xd9RVxqauHje4hP5oUH5PIf4xcg09RhyBzTsgiIagfqj11ZYN3rGYuVqUZhLoH6nhahY7t9xyaXQtiiyiLe+rdQtfADrA0LBEWMK1GwnhvGvB\\\\\\\/0vzCf\\\\\\\/HIbHXVtSHQiva8BQzDDXS2mxBM\\\\\\\/HR1cJK8kY\\\\\\\/\\\\\\\/kFLvigHJcEJP9E7i6B4NxiJn2ohi5G4nlApTB+KWNxonFbSuQcPbsyKI29gGgo8QAW6FTmWATFutceaIYtYr4M4xOMiKU4uYcEg7UDpaJYp3Iz\\\\\\\/57tVoHrnt+\\\\\\\/PPynpplcfHarwixqA2w7s0wabBtBSqVNP2jgNswGnIRSm5Z\\\\\\\/UsjQYjiq4+B1KJZxWW+V7UPpY4L0lOba2bqt74jX37HPVyxdthTTRtcnSv4dy+vUjD21ZBzeOsiiaOtbN6bBuXT\\\\\\\/\\\\\\\/5reEDae4WKtruqb9v342LXXg0A\\\\\\\/t4epZkO5G1po4sT+5LBWcM6peOKTjA1oL0Fo6lcNhRXVbKN9UhHxEQrj7dOvecXa8W3Tx7C26xpodT872YZ+iuswHRrYnSQxXqJRMVf1pKhT+KUqB1BXUAiiXfzYc0\\\\\\\/kfXvFPazTsTJyhI\\\\\\\/IgSLJgP982uU4NqDoMcEztJ3UsKpJEPBGytXALXu2sZB5coC\\\\\\\/iTLTIBm5bq4v9k0kIMYlaSUEo3vZ35U\\\\\\\/uIVoeO3K12scrzxaLcHcZfIHXTE3Jf6pJiSmVo1FTgco6Fum61INoSaHdMilq15gY0R4T3cymkHJXAYC7PTzpIzvF8LwU52MHLR29X7DZuN6yT4UFjI60a3W+5gaFXgJQ5mjDP9dbcg3TLDM54C0d\\\\\\\/KaS4Mtz\\\\\\\/g66ryGD32i4Zr\\\\\\\/fcdP2nKZG6PsJ4ElApJ+306KKrSk5IKouAKU7uetX7XyffDRCnqaSY0nRDJG9aHJwcmosYSnkr6hL8dTnpsO\\\\\\\/jBprQaPDXH3JyOK\\\\\\\/yZelW8Wy4YMPooliOAM+Czbmed6DKWmlsqXmU1fLgGO4Jf8wYeoUxZHIrjqzHdvbJm7Sx6s5I5FDNLpa\\\\\\\/EjhZ2IOHMp4hOklrfEaV0b62Tn03o46KUTttGau9LkIk2rRJfUnC\\\\\\\/M+P\\\\\\\/sExHEkNSfp3v9wztwBjwTJLUn7aD6el7u0wHCR9hFw5Dz0cRDtIuFJCZwBzGVw5YA7M2vNPyyxaCHtNhppcOWuZt2iKr+mZaQADsjhe9idBCfBor8eeg5iF8OZIwzWi8u5HdNOq6FvIOxvZqKMkURRRhsr7Fud7k0STzRpewP21ha9FtEnrOOm0p3nZkkmhYBjrAncuLRq+D0V7\\\\\\\/HONd5jd92n+wuDR0nbrzaYLqFYMbBSw0lflr9gSJ1hmjuAoNbw2C4ARrxKCSPm3YwrHQSS2xgkNrHCXf2lBCGLenFu\\\\\\\/9QIymFFFk+BjqvX1XHZnA\\\\\\\/GAoYuMFrxy2J0iCkoHljrbIWsFzGbCkJHH0gXo3JfQpr4B2nv2W8mZgNurKXoUjR1nIe7mdWlKZT8d7HjBMU0agjqD4617Fy56RML\\\\\\\/eDAQYClbkdpOqyMHllnlrVj2Z4\\\\\\\/VWqGmtDHXvyI2rvE2QoqTMJ\\\\\\\/ECu0RIfubJoK\\\\\\\/JFBL2cH2kyJf2nEgqODBpiEoNgbrcJaGGoudphJ4246fiZFF11IWV5\\\\\\\/bLUlZ\\\\\\\/\\\\\\\/k+u4ZChj50IMk9D3Z6EhTc0xVyHqMrvzKfGcWdxxO5xSBOsCVhK8lgA2+UIiskGG0dgIyNjHklGf0Mp1meN+eWe+6fSYkuJ0GJULJUgFg3166HhWaCnO+Bn72qMLi5CiizgB\\\\\\\/TSNF4ne5B7OgwtkXyUlh1cYXFO9sn\\\\\\\/QiEWp5I9+DR2OqqSjjBd8Rasf3CkZcTZYHO4pYBAM2+Ed8ZIl81lKQVBAmyvpQ0SwaoX4ZdaQQiRP7lTYjiZLrvLXUJUQZlr7VNb2JwOkUSUlh0KrevTmpLYdHyyko8lIrSforE1GTZhPPk6E3dJQRpp2XiHVSdYVS7X3CDNZyfAtL3lkzdsirJ8aKyEkvfi4\\\\\\\/g9kjKrT1rKee3HtJgeNeleHJ3pi3xslDX34G2BlcPnLNHR6mE2\\\\\\\/YlQmYZ5ORZ0UOsY1lR54gTMtq2gS0tAZYygP67nAvlb53+S9DYuZ4GjcXZJuwk41bXG6b6PUqLJP8r\\\\\\\/fwz7uyeYEB+nvn\\\\\\\/fHWpF4IAuBYdilrNv+V3EpFVUT\\\\\\\/\\\\\\\/v13ZvKCw+Oqts0FOtPxfaYuEuW2\\\\\\\/jpqIoXOrytIRO7YYhIXqagb7pGeuFRMxEvUYa\\\\\\\/eAl59Rq6VrJFBCt+dTZr7G\\\\\\\/lWPePtLeXlAtve1qj9eonQVflZpYr7YoIvWLTUeAMSbhXvZJGdgVmIqSpNiskC9nz\\\\\\\/L2ef5uuzYNZWaFI9DE6j\\\\\\\/XhgstNzZr+BaI6FwesIyrl5PRedjE26Mi\\\\\\\/DIeqS3Z6z+T\\\\\\\/slRab99rVPykv6OAeDNWx7xN1FiEXEOIZ3qxK15AWWMnotPfcNgxOJmfeivLrolquY3+TUr2cTy39JeFIygj5Uxtn34NwFrj6R8nIZZhOx13woeVyV6WEOxxgHSEXoja1ZLRPT4y4PAph2NNNKJA7GMA4lIC2jmzo7emFQC0RRuQ+VgKpGnso1lLl7kj5FM0OS39KNOXTTpBNwvRaSb5arhTziTeYbISh6GsXUg0Zbz4zJztauJUaV7a6CQ0CUpt8MTxGKzs4tRW+f+8YKeMxxdy9LWUCZUNfjOH5hWGHtra5LwxRepXImxKWf8dJc7CcJdsvNdZOXCIiS5N0Nc7Jrm3sKa7AX3f2tcNf00Sh5kp7rmxxwGxhgZ2RFlkz7+DeG3ug0sAGPRptPzqf+2gmQ5WDMn3v2e4cU1o7zfjYjwmZzOLmNHc35cnTwX8E0Hu1yskS6oFtg6wjQdbHTk9NzrTWKighCnQtsM4rndc3Rv2EI40Zzyv8x3RDDhIvxN3U0venkqZUnP5tj9k8uv5JvGP0GGX87hP\\\\\\\/8Ig5HovT5qcXMhpH0Cc3SAViAm8RkkuMwERFMKMZGrYO9jxPvH8vf9mZrEJfNZgn0u2grvc0AxvdWfk6M2G+y00Q\\\\\\\/t6tXZfz9StFf9SlC+4UYe8LEWTbzHGFXxdLDL9omJtnnMkoPLlIIWCLgh0EVvZZkBCloBUFPe7Qy+5GYrc2so+3Fbq9CQgUzHyYn48Q39+iuqoGQA6TbgHUwvQ2tmbJhQcgCeOVMZt53HtoCq\\\\\\\/e\\\\\\\/J0T+X\\\\\\\/4Os7lRZSx8HpRobX175qyqLwsUwj9ySBWDivpR0+TNcxof0xYnVzEW\\\\\\\/A7zD5yTtaek0IbQtpSvlXZoodTTN+L8z8Uw95EFoyq78qdXJ7hRuURrXJe1FEffev2Tnvy1FwX+\\\\\\\/tC8kW4pSmWHlD\\\\\\\/51FrOiW3gihgK\\\\\\\/SudN1jOXAUqqNNU3IpStTw+dITKea9DLln\\\\\\\/9LbCNju\\\\\\\/7EDMmDzRncppFjldAOPzP0CqhyBeCpqEwbxcemgWyvhHnRhluT8Xx75UbQ2QNNJyoGQinmW4b4yNvH\\\\\\\/hxxzOlQUUQeySfJ\\\\\\\/uAG4NsYhL6p453FeRgxDsZHsuFMxGskK9jCA6PzuvhgEmjCqy6ty5sb9vAwLWGrt9S2Piw8OQxaP\\\\\\\/wVhswoGjWSEDVLV1t0nQBhQcbhlyHZNPmr6xo5BPQSAb8aLcVasjRs+\\\\\\\/TrFUdHWR4E10NnDNzDdEigTdJehQi3oWZ1l31R8V45x7vFf+wy03n\\\\\\\/EGrGL\\\\\\\/SxYbD0DfrDZbP+D+jdu5XJHuz8TWUEovmKRfqa+9siXaaQcROlDhFiShTXS+qrYMvPdgUHnufXOpAQCIzjNmGgBh12DxG\\\\\\\/qmgZIQqrHpfgvaB6cOw6l8hkBxGT6PduRPJ9mhyIsHHQZLKMOPSE8ivpHpymikYdtNkhsOfyS0cDlDhXyDm5lKBMqHQjV6JBrB+f3KNDbyYTtRhtcverqBBIpPblfmNIa4RBvqmJcqrfZUTOAao151ybPeAxf6s6MteYZXUgldpTcsoEd7k8qmB6BOAnDna3xfRMDZRvD3BtCpskS13zrfxSLuXf\\\\\\\/29b88hgrWkP6UHnurR3yi6Xrth3q+oEmGUf93OvEs5t0v0trkijtNKsIGaGFBUBODkCux6VhCjlm2hHfOHVfQrAZcQrtqZepHreHpi4eb6ShR23WDCUlm2\\\\\\\/L+yPkM6GCeL0zAvYQ5AVviymsEik9mmyEVTGVgDsI4RJtPziGGCeQWLD2DVaxi0rGfq+hP7CvUn3zjmFHAwh3VBw6E5aFqIvDJVRBk88wviYuq6UPfMbL06uRbR9XeE\\\\\\\/KYfleOkrX\\\\\\\/rBGNnho3x16GdH+SpH5qzpeZUAvT2\\\\\\\/gvvo80kgyD0Eb0f7hMR6iTEMT35swRoDkwZ3rw6CE+nLcgL5VRVAW5p8nqTiY7YOLMZZVzT+FuFkdjXktZHp39GHSPdZNGARgvZ12mTtzixCoJ1OIYdfHHtYjydcd90187FTSsedpAIc4Y\\\\\\\/TGYrfV2GpLwHHTa+6E3MdB0AvqYiAlnwgduEMpspewO9OxN0uSpxLBzOlz4sXn6vadz7ZpxkitEK1IqqGMe51OJADq9zi13S3Sf0rb+PPGbPbPJM\\\\\\\/owqY5fXNi5NbvJ5neKhqT4IVG2vXGMAf7HAdUeezDSnSpRqx3G\\\\\\\/vvRP\\\\\\\/Xr0v9WdYX9d\\\\\\\/OMU97n3UnNBoPqd\\\\\\\/hA1eNevS9F\\\\\\\/9mwkD1R6P4vZctTkFY06ceJJ2aHvGtlImmutOjZxJwJEqFTXDOnANyBSExUyYfoO\\\\\\\/KChcPB9dCPrWlLoW3ljBvPa+gklaO4G0fTKFg5PLDmnU\\\\\\\/wP6OBw9pKGMXvJR2VcJVu\\\\\\\/LmQR8AEPaER5w+Xf7XDYA2RjhlZK18txLjDVS+tCnliYuY\\\\\\\/y97Rd9yh\\\\\\\/fOw+dD9uXuGxY2gbNb7s3SLpA3YKG2aQKpfqUJjg8LYYMhWmfBnrun\\\\\\\/gSkdinwNHH7xLSJgTZ2ylQ+lmrZg8tkiyKdl9J5njejADYyjACPaQbhkg\\\\\\\/duxlWyUkyYa8Yrn20zC8VyD5zhiI\\\\\\\/MiHgP0bH+0AHBC\\\\\\\/KoxouywoPY\\\\\\\/7pE8b+fZLOC\\\\\\\/XREx2pttFTQZz\\\\\\\/xeuTmgCJzKSRiJndQ50GE+NkYmT68d8G0fNZq8kftzhpJRS7qfEU9btLFWjRA1v+0u\\\\\\\/OBGtKKWET3Hub42XH5QMLNhGhDNXkfaTRZ6qOWcKa94XMRBtB01uJvH0G7BlHd+wjNBzOxnge\\\\\\\/6BNwnRDi776E8pCT6j6vzxSJ9tj4sU9rAsr0EX9T+2y\\\\\\\/vEcRp9+CLBSpyideHjj0oqaNnAZ\\\\\\\/n7gd49F9SjRhnw8+szNcwoE4FMv+vt39mL6CE3R\\\\\\\/G2hSn6bauGSPu8hx1y8ZC2zgD47PvwTnBJ3uWBny3mrWQWIiqyNZP1f15yaB3bL1cxs6DVYvbjgP7652MVgS+rzhCTMbPSg\\\\\\\/XaIaxk3QSDvRt5d7yfAyfedQUp8nCnpW9zhKpo2N\\\\\\\/SA2dHQZMsnwaTtUdpeXlGeXFAC\\\\\\\/PW7t1HLACmgadWzY\\\\\\\/5Icd5QJ2m0iorkjAmAAjCypGwLkE2wekBDN8GpW2NZGAbbD8b9NC+QzzTtTD5NGiJeT2L17RTFjuCRIIpVF1AxLzdZzalXXR2f7CYPMGgGfUTFlvfOVmchovE5c75WL71api5lN3O21kct31gwERxcf\\\\\\\/pi6\\\\\\\/TCrQC8OdmLF7bFp9++fTMEwkxfno85YTGHiBDEy68Gbp2oa59x4zFT7wgRlCupUxPUkPvH6JH7lMKj5VNJDuB\\\\\\\/KoQqeQonszo8JuDS1zzSecsX27Cui6A6VVbq\\\\\\\/f0PMRKNfJ5bF\\\\\\\/c0W1Aoj6Ft6Qt7bz36tEw2BGtK6ARjH5YSBdjreyPORpcYFBObBDQOYISD0p4Mwce4pez2nsMgno8JgilOP4EdW5aUW5YxRDi9378cfBw3ugwWpYS3IZOT7FVo\\\\\\\/Uhy72G66Kr\\\\\\\/MmbkrRLjL6xPXXeth4qPA3Srx\\\\\\\/s+G\\\\\\\/JMmRYL1h6Wn2HyL\\\\\\\/pGCVtrxSsT9KbSIBtGSgL36EHgYyhWD4Dl5Z5TS6Ybf7nptswY3Bt3l\\\\\\\/dFeQqUx\\\\\\\/FNf\\\\\\\/1lqxLEp2BvKlun9iWRaKmwEco\\\\\\\/L9weOMP1QZzRJCVmmOT\\\\\\\/uuLSIrJDIZUr2pyk\\\\\\\/8UcFeJyPI0i8BeCaBhqW2mhxKhhwdQ5AsGfkrdhZ+ZJjIhB0KKF7Yg0W9sGdVl\\\\\\\/hRpHKcR+gchM619s6MfEFCFs047Ty7jeausQ2Ndh+9GefUAKcvhCjiqMKPkD9BEm263UIivl4gP2s89kOPIPKGQXA0jZLsScWWapVAX4vufIMmv0BF2z56mcNpauo5U\\\\\\\/YyUBJcPbnH5rU8avRuBIefQgQdvEPhOZLB0GPrII8G5i7Oc6kJoIe73z9XofajlmMhCyy4LJ7Y66B2mQbhO2f5c\\\\\\\/OLQOOPIQyC11SAodlkzmvxapjHNwQaWFlUUowuXNACEdkeCfeRWMviEirogAcEkNh93qo\\\\\\\/vVTZ6AOcKNizu2fveg0NhOp2IURvBsGJnO8sNL5kZ+10Rqlev7wqzxR\\\\\\\/v6zcTrkNrpv+jjw1Y4mOh0SY3sgJKuTxggL2w5r4NgOWVoISlKlzVYWmGyWuzNzHNIKuRef9Kjvl32ClfG4R1dCSTY5pN7S2tkP6qkTz0kIc2QuQk\\\\\\\/jrWmRCyovx9SCKRWvMS3c414R2Gxkkf8hiPpOBvTWgcrbHQiOKleFHSXYATz1au53iM8S1zR0bKh6mr7bywy97XXGj7wCsB6gmbY+tfYUeuilII+uNdGBb6NKJ8KYOvhBDnF+jWGu5EhDZYUCuGeD9rzkoQXt1o684jIf2TGHcFuxcT48ozNYaHp63BADvKGV6LRyAIJMaCEpEkmsRHQQxDs8TKCmG9ZrtIlkfj29EcQp6t08KAbFnHbA2DTqQ6XKV8b3lD+hAutos176tiosJibvG6pO1oL3QlOpdGeLdYMZMqzUulYHTOUC+Zhn0Bt\\\\\\\/GUzIQ6EtFh6qnIr3W2l8aezURCcnhAd86yqBbCspBPS0rrYYkCsHW+lGEQePQ0\\\\\\\/pSc4kdyMm0aUjjYk794QQnNDojSoTihIWNPzKwXtEZzVA+3lTYBeUdkcrerBZ8ThYTC\\\\\\\/0bRwP1kHDTCvqKnZeo4ydkyrRnmGkLKbLcn1\\\\\\\/k9uf5fXgysLtSBJjYqpqobyND+XrparjXXoY6MaldVFZAL9mQTnbIelO0y4rLOQqU4jCOv9oaB8qLke9wyoQucX5M0JA75aIdHY4joQZQPU63hch99zH4etZquT+lh9Xz2vCN6Ss4ZvznRN60iVGK2OzOz+IVmM19P3eyzRLB1puhWADWl4N2kmi\\\\\\\/P5UlAEmRwQe\\\\\\\/d+gmQW5tdTldzfXFV4JMqy\\\\\\\/19e2gZM\\\\\\\/5z7KBE3w+54wo+PJ2PdBqasYGfICT2YPC9V2fV2XMI75aViPOV17Si6\\\\\\\/j\\\\\\\/c44zO5+TzOI3hQ9e2NL6tl+v0JhwmpRaSfJoGHBeJ44nhxsB6pQThZWCPxCQ8NHlAhlWLPxwdW+YZOZan7sMged1eheuEXcexMbaCsXvhY7zOaI0u3gFRARiFPKdlz47eAHkj4YvJ8o0BA6Z1LwbmK6k6s+HpHO\\\\\\\/3LE1qrQMzWksK\\\\\\\/avQYjE4N\\\\\\\/tDO+utBY7Q\\\\\\\/AyDOqgnGaWIUGBK2+P3YzqWarIjUbSOTZ88nLnusfmzq6WlPHWH6cRG18Di2cW4KvSjRdQlRlXN7+M2fvNzLsppU\\\\\\\/yRp+bVeEIin41QnYd8Cy3ixrju+8CYugm4i2UnnFmz2rBjR\\\\\\\/osie+tKIAuL4MqJkdabwRqT\\\\\\\/WpcJBH00U7FF3c1cCpU8ZK4LwOUDfA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022XGGMhJCNPwfGVbAzUs4OxA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227338121e2b2ddfbf038a71f738e2b0d18d9c67dbd0fed7eac966c6e69ded6a3e\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rNsrrzLRQHBTD1UZK7kqepRywy20eo6XpDAmYbhPJ0srsq\\\\\\\/1gxUAOltIfJ+KcD235HnE+Gz+VUOQStHkY\\\\\\\/pZwPpv42ud\\\\\\\/U\\\\\\\/Auhf\\\\\\\/4i4HqXYaVPguUy\\\\\\\/iMW\\\\\\\/hw\\\\\\\/2BUCXfQNzazlIVptXS52QhOZCW3z+MNrWgoZFSktvcB+tdfLHPVPqUHovV+ZyDKOjnd9XgnfOQ8HtsbuWsHvyPnB1y99x6fi0i+Lo418jlykwikPUSWo2gySsN1pvznKefq+RNvrddKtMD2xYat784hb71KwxKiS6\\\\\\\/R4wqGGY8CAABbFQRJr+PDVVDXubeggTNjsSr\\\\\\\/HzzF05rxugj9fIHToxugat7dRHiXiHbVRc0hrtsfBWfOuZJSnyaFrvFvgXbq20rkWgbiJDZwTpetAUFxfSDp9+uqB\\\\\\\/N4og2eh5rkW\\\\\\\/ue0rIgvky0hvAFQTS+njf3876aeLxaXK+v4jefuWacCDslz1VF4AzF4f+kb8BteWYqZWCHp2w6GotZ9aN3IFNYUGn6Yx7b0ShqiV5hXEoJ0ELipZdwSm2IQ2f0xu5+1bt1QP7B06xffeh0VOy1J4n+DftidTWqQPIwyvE4qwK\\\\\\\/\\\\\\\/9eRiHOGaIsqFa4oXRsbqddNAPHBDWCJoJ0ecMiFyy0HzVegbGi19UhmbUj2WCmn+7cPWnwew4I0CDQlF1N136rDqMGV+jdwjg5\\\\\\\/5XJVO9qfyVOYfNNWnAdTePTNKaBf7FKcZWy4JoMDXaqRsm8dFB3UNsw63JIhGqNSGoKZLDieuooUyhzv0wyMednRbyfNCL0VDC2LS8DnpgPm\\\\\\\/4vzqIa20yO+6CajZ\\\\\\\/+v5TzZfAJ\\\\\\\/xxKSiti8d+zjyTjszjBao+cia2ALhR5MIfr\\\\\\\/pn0S9Rduxe7JSFcmN8LJPB4ggW+ZWk7aI9xe8q6+Wc8e9lL\\\\\\\/B8BWrkB\\\\\\\/8VM9nh6SZtLO+s30HnF5iz9vSndF2aROWimIyqrdLnZ\\\\\\\/o8+baJId9NXU38eouF547oejRUZ4MXxYhipDb0ceyt\\\\\\\/kPO8Veglk4h+vere2OkQLpYfKzYKjYgdQyGJwG1fbx6z4RjuBRq2I0tXltxGRTQqnUGcF5KPXJhexJx8R\\\\\\\/Yubf5Y8MUgAUu8AfyEondnLZOP2OZzXQSejZjZIXXo8UJUu3EJAJv3g\\\\\\\/1pPwnAOdYf9PoZk\\\\\\\/dnV7RdjvNba7VyrIylbprXsKsE6KXsiGTV\\\\\\\/p5SlcJdUIo6ktHllogq0OTpfRYTYxqZ1akNn9sySFiVjo3bh42jaVS+A586oz\\\\\\\/i0z+DaxVF+EzU9IZrp3fCzwOieKI9edDxYyJE2lHSLu0t80MSYumaiYT\\\\\\\/DGFOHQwaBiauViBuUvSqONGIEV0SfdFfMZ\\\\\\\/FIyNaTA\\\\\\\/w33JB9N11fMsGoafFoTTShd2TDtd8PIPRBap\\\\\\\/fmgnriCgdeAgoSNVmc9ur3yKp+hLJsnpD7d+5v+FhaHxiz7A6S6dC6gurbW3Kww9ANoPIZ5IxrpBHOalupa+A1esHgk0Hq6Spq9r\\\\\\\/1442ZU37WJ40rfXaVEiSMH41Db8lxrlg9Go9gSr6PDRbs9oAli56yLeR\\\\\\\/WZcjrqbGA7YYIrKFxs5lsUMzd2s1tiJaYAM7m2F5SbJZs6nSeQ9Mn3hK4M6tzZh4HvXlO\\\\\\\/fEm\\\\\\\/tA\\\\\\\/L\\\\\\\/mdCm0\\\\\\\/4sghc6zt7AKZrKaXiK5d+h6VZeGLiGI+8KlB8TH2RWHrp23NPN3VFT\\\\\\\/uzgmXD7eoQw7fs6rWfcA5JgYUdAEdWj1YgaSYnCcR58jSiRifDJjPSRYeF8DugivtrnLm0T1QVcTZNAf1F8oC3phf2tRmVdibRbo+GaHeRAnV5yUch5noPnhklffGXifAzOG9Ja\\\\\\\/cv0AjHj9NsZB7sDYQGYipf4wBXCQGQ7fdoEHArKPPAKmWWrcmx9wTKfbOy1xHmD4KEEX9vitxw0bfERY1qCDLodwOmndEvDi3lahWU+IduEcUvNVUfkGwN2ayJ9rOrj+pImEdon\\\\\\\/9GVXeYI7TGkJYkWsrUUTnbHb16T+\\\\\\\/joWCQpyIBPQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00226hqnQx5mCt5Z13abUWMn4A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022069463434957b228b6756a745e6fcae0c0c85a2806d10ec12647dd32e59b9271\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487839 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/63" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-o9bt3_mi014g_nSBXjsVuzB6sNbq_5Bg0e8ZXl-gIX4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/63" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Fb0Mx3jYLF8Con4FFMkYWRKHIxL5gMIb0nZjNb2La8M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/53" title="Improving Genomic Prediction in Cassava Field Experiments Using Spatial Analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/79" title="Large-Scale Low-Cost NGS Library Preparation Using a Robust Tn5 Purification and Tagmentation Protocol" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487839" id="node487839" data-pisa="ggg;8/1/63" data-pisa-master="ggg;g3.117.300245" data-apath="/ggg/8/1/63.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Screen for <em>Saccharomyces cerevisiae</em> Genes Contributing to Opportunistic Pathogenicity in an Invertebrate Model Host</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sujal S. Phadke</span>, <span class="highwire-citation-author" data-delta="1">Calum J. Maclean</span>, <span class="highwire-citation-author" data-delta="2">Serena Y. Zhao</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7805-8785" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Emmi A. Mueller</span>, <span class="highwire-citation-author" data-delta="4">Lucas A. Michelotti</span>, <span class="highwire-citation-author" data-delta="5">Kaitlyn L. Norman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-8923-297X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Anuj Kumar</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-1123-5986" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Timothy Y. James</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">63-78;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300245 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487839" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sujal S. Phadke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sujal%2BS.%2BPhadke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Phadke%20SS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASujal%2BS.%2BPhadke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Calum J. Maclean </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Calum%2BJ.%2BMaclean%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maclean%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACalum%2BJ.%2BMaclean%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Serena Y. Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Serena%2BY.%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20SY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASerena%2BY.%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Emmi A. Mueller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emmi%2BA.%2BMueller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mueller%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEmmi%2BA.%2BMueller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7805-8785" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Emmi A. Mueller</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Lucas A. Michelotti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucas%2BA.%2BMichelotti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michelotti%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALucas%2BA.%2BMichelotti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kaitlyn L. Norman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kaitlyn%2BL.%2BNorman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Norman%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaitlyn%2BL.%2BNorman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anuj Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anuj%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnuj%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8923-297X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Anuj Kumar</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Timothy Y. James </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BY.%2BJames%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=James%20TY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimothy%2BY.%2BJames%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1123-5986" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timothy Y. James</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tyjames@umich.edu" class="" data-icon-position="" data-hide-link-title="0">tyjames@umich.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/1/63" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487839" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487839/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/63.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487839" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487839/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/63.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487839" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487839/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/63.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487839" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487839/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/1/63.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487839" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487839/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">Environmental opportunistic pathogens can exploit vulnerable hosts through expression of traits selected for in their natural environments. Pathogenicity is itself a complicated trait underpinned by multiple complex traits, such as thermotolerance, morphology, and stress response. The baker’s yeast, <em>Saccharomyces cerevisiae</em>, is a species with broad environmental tolerance that has been increasingly reported as an opportunistic pathogen of humans. Here we leveraged the genetic resources available in yeast and a model insect species, the greater waxmoth <em>Galleria mellonella</em>, to provide a genome-wide analysis of pathogenicity factors. Using serial passaging experiments of genetically marked wild-type strains, a hybrid strain was identified as the most fit genotype across all replicates. To dissect the genetic basis for pathogenicity in the hybrid isolate, bulk segregant analysis was performed which revealed eight quantitative trait loci significantly differing between the two bulks with alleles from both parents contributing to pathogenicity. A second passaging experiment with a library of deletion mutants for most yeast genes identified a large number of mutations whose relative fitness differed <em>in vivo</em> <em>vs.</em> <em>in vitro</em>, including mutations in genes controlling cell wall integrity, mitochondrial function, and tyrosine metabolism. Yeast is presumably subjected to a massive assault by the innate insect immune system that leads to melanization of the host and to a large bottleneck in yeast population size. Our data support that resistance to the innate immune response of the insect is key to survival in the host and identifies shared genetic mechanisms between <em>S. cerevisiae</em> and other opportunistic fungal pathogens.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/virulence" class="hw-term hw-article-keyword hw-article-keyword-virulence" rel="nofollow">virulence</a></li><li class="kwd"><a href="/keyword/barseq" class="hw-term hw-article-keyword hw-article-keyword-barseq" rel="nofollow">barseq</a></li><li class="kwd"><a href="/keyword/pseudohyphae" class="hw-term hw-article-keyword hw-article-keyword-pseudohyphae" rel="nofollow">pseudohyphae</a></li><li class="kwd"><a href="/keyword/hybrid" class="hw-term hw-article-keyword hw-article-keyword-hybrid" rel="nofollow">hybrid</a></li><li class="kwd"><a href="/keyword/bulk-segregant-analysis" class="hw-term hw-article-keyword hw-article-keyword-bulk-segregant-analysis" rel="nofollow">bulk segregant analysis</a></li></ul><p id="p-6">Understanding the evolution of pathogenicity (ability to infect a host) and virulence (degree of host damage) is critical to understanding the biology of nearly half of all life forms (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Dobson <em>et al.</em> 2008</a>) and managing the spread and treatment of infectious diseases. Natural selection on pathogenicity and virulence traits vary greatly depending on the parasite life cycle. For example, opportunistic pathogens—those that cause damage to a host dependent upon environment and host condition (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Casadevall and Pirofski 1999</a>)—include those whose primary niche is outside of the host, such as saprobic fungi. The observation of variable pathogenicity across fungi is enigmatic, nonetheless, the coincidental selection or accidental virulence hypotheses can account for the virulence of environmental opportunistic pathogens as a by-product of selection for traits in their natural habitats (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Casadevall and Pirofski 2007</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ebert and Bull 2008</a>). Despite the accidental nature of the virulence, insights into what traits are pathogenicity factors in environmental opportunistic pathogens can be obtained using genetic and comparative approaches.</p><p id="p-7">Fungi represent a large subset of all pathogens, with many species causing opportunistic infections in animals including humans. Consequently, the genetic machinery underlying pathogenicity has been actively studied in many fungi of which species of <em>Candida</em>, <em>Cryptococcus</em>, and <em>Aspergillus</em> have been the leading models due to the high number of infection cases associated with them (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Odds 2000</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Richardson and Lass-Flörl 2008</a>). These studies show that opportunistic fungal pathogens of humans possess a set of factors that allow them to function as pathogens. Pathogenicity or virulence factors for fungi causing mycoses are known to include: the ability to grow above 37°, adhesion, cell wall integrity, melanin formation, iron scavenging, and mitochondrial function to name a few (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">van Burik and Magee 2001</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alspaugh 2015</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Calderone <em>et al.</em> 2015</a>). Morphological factors are also important: hypha and pseudohyphal forms are associated with penetration, yeast forms with dissemination and migration, and extraordinary large forms, so-called titan cells, with macrophage evasion (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Zaragoza and Nielsen 2013</a>). Major insights into the pathogenicity of environmental opportunistic fungi followed from the appreciation that some of the pathogenicity factors are likely the product of selection to avoid predation by amoebae and nematodes in the wild (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Casadevall <em>et al.</em> 2003</a>).</p><p id="p-8">Most insights into the genetics of opportunistic fungi have been generated by using the leading clinical fungal species. However, these systems, while biologically relevant, are not ideal model systems for genetic investigation. For example, <em>Candida albicans</em> does not undergo a standard sexual cycle, and targeted genetic transformation is challenging in both <em>Aspergillus</em> and <em>Cryptococcus</em>. Moreover, because there is known variation in virulence across isolates of a single species (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Fraser <em>et al.</em> 2005</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kowalski <em>et al.</em> 2016</a>), leveraging this genetic variation would provide a major source of natural variation to determine the mechanisms and evolution of pathogenicity in opportunistic fungi. In this study, we leveraged the power and genetic variation of a leading model fungus, <em>Saccharomyces cerevisiae</em>, to address whether its genetic resources can be used to identify genes of relevance to virulence in opportunistic fungi.</p><p id="p-9"><em>S. cerevisiae</em>, known as brewer’s or baker’s yeast, is common in the environment on tree bark and rotting wood as well as a commensal in the human gastrointestinal tract (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Wang <em>et al.</em> 2012</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Murphy and Kavanagh 1999</a>), leading to frequent opportunities to function as a pathogen. Indeed, <em>S. cerevisiae</em> has been shown to occasionally cause invasive, disseminated infections in immuno-compromised individuals as well as vaginitis in nonpredisposed individuals (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Enache-Angoulvant and Hennequin 2005</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Murphy and Kavanagh 1999</a>). Surprisingly, despite its utility as a genetic model, <em>S. cerevisiae</em> has been underutilized in empirical studies on evolution of virulence and relatively little is known about the genetics underlying its pathogenicity (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Goldstein and McCusker 2001</a>). Clinical isolates of <em>S. cerevisiae</em> are reported to show certain phenotypes including the ability to grow invasively at high temperatures (&gt;39°), copper resistance and pseudohyphal growth, suggesting that specific alleles underlying these phenotypes contribute to fitness of <em>S. cerevisiae</em> <em>in vivo</em> (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">McCusker <em>et al.</em> 1994</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Strope <em>et al.</em> 2015</a>). Other studies, however, have not demonstrated a clear division of clinical isolates of <em>S. cerevisiae</em> either genetically or phenotypically (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Klingberg <em>et al.</em> 2008</a>), suggesting that the genetics behind opportunistic pathogenicity in <em>S. cerevisiae</em> is complex.</p><p id="p-10">Here, we combine the genetic tools of <em>S. cerevisiae</em> with an invertebrate infection model to understand variation in pathogenicity within the species and to identify its underlying genetic basis. We screened a diverse pool of genotypes of <em>S. cerevisiae</em> for their ability to survive and grow inside the larvae of the wax moth, <em>Galleria mellonella</em>, which is a model host often used in studies of microbial pathogenesis (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kavanagh and Reeves 2004</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Mylonakis <em>et al.</em> 2005</a>). Our goals were to identify genotypes that outcompeted others <em>in vivo</em> and to test whether clinical isolates generally have high pathogen potential. Because <em>S. cerevisiae</em> has not been documented to be naturally associated with <em>G. mellonella</em>, our choice of host represents a novel and challenging environment in which the fitness of the genotypes does not reflect historical adaptation to the host but instead selects for pathogenicity factors of a generalist nature. We also performed bulk segregant analysis (BSA) (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Parts <em>et al.</em> 2011</a>) to determine the number of quantitative trait loci influencing pathogenicity. Lastly, we conducted an additional passaging study using the library of mutants in the <em>S. cerevisiae</em> deletion collection. This approach allowed us to test the contribution of most genes in the <em>S. cerevisiae</em> genome to pathogenicity separately in a controlled genetic background and to identify types of genes that are required during infection of the insect host. By leveraging natural and engineered variation in a model fungus, we were able to describe the genetic architecture of pathogenicity toward a model host that draws deep parallels to other opportunistic fungi.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains</h3><p id="p-11">Strains used in virulence and quantitative trait locus (QTL) mapping experiments are shown in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1/TableS1.pdf">Table S1</a>. The 43 diploid strains derived from wild-type isolates used for serial passaging experiment I are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1/TableS2.pdf">Table S2</a>. Each of these strains was constructed using methods described previously (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Maclean <em>et al.</em> 2017</a>). Briefly, wild isolates (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Schacherer <em>et al.</em> 2009</a>) were barcoded by replacing the <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> locus with a cassette containing two unique 20 bp barcodes (uptags and downtags) and a g418 resistance marker allele <em>KANMX4</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1/TableS2.pdf">Table S2</a>). These heterozygous diploids were sporulated on potassium acetate media, and <em>MATa</em> and <em>MATα</em> G418 ho::<em>KANMX4</em> colonies were obtained and mating type determined using a PCR assay (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Huxley <em>et al.</em> 1990</a>) with three primers: OSP37 (5′-TGGGCAGTTTACCTTTACGG), OSP38 (5′-TGTCTTCTCTGCTCGCTGAA), and OSP39 (5′-GCAAAGCCTTAATTCCAAGG). The drug resistant marker of <em>MATa</em> strains was switched to hygromycin B, and <em>MATa</em> ho::<em>HYG</em> and <em>MATα</em> ho::<em>KANMX4</em> strains were mated to form diploids. Experimental passaging experiment II utilized a second subset of strains from the 4828 haploid strains of <em>MATα</em> from the <em>S. cerevisiae</em> deletion collection (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Winzeler <em>et al.</em> 1999</a>). Each strain in this set has a unique open reading frame replaced with a cassette containing two distinct 20 bp barcodes flanking a g418 resistance marker allele (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Giaever <em>et al.</em> 2002</a>).</p></div><div id="sec-3" class="subsection"><h3>G. mellonella infection protocol</h3><p id="p-12"><em>G. mellonella</em>, the greater waxmoth, is a model host for studying fungal pathogenesis (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Fuchs <em>et al.</em> 2010</a>). We maintained a healthy growing colony of the host maintained at 30° on a diet containing ∼400 g of Gerber multigrain baby formula, 100 ml corn syrup, 100 ml glycerine, 50 ml tap water, and 1–2 ml of the antimicrobial propionic acid. In cases when the colony did not provide enough hosts, we used waxworms obtained from the first available supplier of fishing baits. Inoculations were performed by injection into the final posterior left proleg of last instar larvae using cells in 10 μl H<sub>2</sub>O, after washing by pelleting cells, resuspending in H<sub>2</sub>O, and repelleting. Sham inoculations using H<sub>2</sub>O were used to control for background variability in larval response to trauma, excluding results when control animals died. All inoculated larvae were stored at room temperature (20–22°) under ambient lighting.</p></div><div id="sec-4" class="subsection"><h3>In vivo generation time estimates</h3><p id="p-13">We used a haploid strain (SSP253) derived from the wild diploid clinical isolate 322134S (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1/TableS1.pdf">Table S1</a>) to estimate the generation time of <em>S. cerevisiae</em> <em>in vivo</em>. We grew the frozen stock of the strain on YPD (Yeast extract Peptone Dextrose) agar containing g418 (200 μg/ml) and transferred cells to 3 ml of liquid YPD medium for overnight growth at 30°, with shaking at 200 rpm. The next day, we washed the overnight culture, adjusted the density to 10<sup>9</sup>/ml with sterile water, and used it to inject 25 larvae, each with 10 μl. We crushed three larvae immediately in 30 ml sterile water after injection and subsequently every 4 hr for 48 hr to recover yeast cells. At every time point, the crushed extract was serially diluted and plated on YPD agar containing ampicillin (100 μg/ml), chloramphenicol (17 μg/ml), and g418 (200 μg/ml). We counted the colony forming units after 48 hr of incubation at 30° to estimate the density recovered from the host at each time point. These data were used to plot a growth curve to estimate the generation time assuming exponential growth after the initial postinjection bottleneck (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1/FigureS1.pdf">Figure S1</a>; time points 16, 24, 32, and 40 hr).</p></div><div id="sec-5" class="subsection"><h3>Serial passaging experiment I</h3><p id="p-14">We grew a consortium of 43 wild strains in two separate conditions: in YPD medium (<em>in vitro</em>) and injection into waxmoth larvae (<em>in vivo</em>).</p><div id="sec-6" class="subsection"><h4>Inoculum preparation:</h4><p id="p-15">Inoculum was prepared after reviving the strains from frozen stocks by inoculating YPD agar containing g418 (200 μg/ml) followed by overnight growth at 30°. The colonies were used to initiate cultures in 3 ml liquid YPD media that were grown overnight at 30° with shaking. The density of individual strains was measured using a hemocytometer and adjusted to 10<sup>8</sup> cells/ml prior to mixing them in equal volumes and adjusting to the final density of 10<sup>9</sup>/ml using sterile water. Eight independent mixtures were made for each replicate founding population each of which was used to found both an <em>in vitro</em> and <em>in vivo</em> population. Frozen stocks of the initial inocula were made in 20% glycerol and stored at −80°.</p></div><div id="sec-7" class="subsection"><h4>In vivo passaging:</h4><p id="p-16">We initiated eight replicate populations of five larvae. Each larva was injected with 10<sup>7</sup> cells. Five larvae were injected with sterile water as a control. After 48 hr, yeast cells in each population were collected by crushing the five larvae together in a mortar and pestle and resuspending the extract in 50 ml sterile water, followed by filtration through sterile nylon (40 μm) to separate the yeast cells from the larval tissue. After filtration, 2 ml of the extract was mixed with 8 ml liquid YPD medium containing g418 (200 μg/ml), ampicillin (100 μg/ml), and chloramphenicol (17 μg/ml), and grown overnight at 30° with shaking at 200 rpm. The next day, the cultures were pelleted and washed with sterile water and their density adjusted to 10<sup>9</sup>/ml using hemocytometer counts prior to injecting for the next passage. We carried out a total of 10 passages. We also estimated the density of yeasts recovered from each replicate population after each <em>in vivo</em> passage. For this, we diluted the filtered extract serially with sterile water and plated on YPD agar containing drugs at above concentrations. The density of recovered cells per larva was estimated using the number of colony forming units obtained after 48 hr of incubation at 30°.</p></div><div id="sec-8" class="subsection"><h4>In vitro passaging:</h4><p id="p-17">We used YPD liquid media containing the drugs at the concentration mentioned above for serial passages <em>in vitro</em>. Each of the eight replicate populations was initiated by transferring 200 μl of the inoculum at the density of 10<sup>9</sup>/ml to a total of 1 ml medium. Each subsequent passage was initiated by transferring 200 μl culture from the previous passage after adjusting the density to 10<sup>9</sup>/ml to a total of 1 ml fresh medium. The populations were grown overnight at 30° with shaking. The <em>in vitro</em> passaging was also conducted for a total of 10 transfers or ∼40 generations.</p></div></div><div id="sec-9" class="subsection"><h3>Serial passaging experiment II</h3><p id="p-18">We grew a subset of the <em>MATα</em> strains of the <em>S. cerevisiae</em> deletion collection in both <em>in vivo</em> and <em>in vitro</em> conditions as described for passaging experiment I. Procedures for inoculum preparation and passaging were identical unless listed below.</p><div id="sec-10" class="subsection"><h4>Inoculum preparation:</h4><p id="p-19">Colonies obtained from growing the frozen stocks were used to initiate cultures in 1 ml liquid YPD media in 96-deep-well plates and incubated overnight at 30° with shaking at 200 rpm. Densities of strains in the deletion set were not equalized, but instead were mixed using equal volumes and adjusting to the final density of 10<sup>9</sup>/ml using sterile water.</p></div><div id="sec-11" class="subsection"><h4>In vivo passaging:</h4><p id="p-20">After filtration of larval extracts, 20 ml of the extract was mixed with 30 ml liquid YPD containing antibiotics and grown overnight at 30° with shaking at 200 rpm. We carried out a total of three passages.</p></div><div id="sec-12" class="subsection"><h4>In vitro passaging:</h4><p id="p-21">The populations were initiated by transferring 2 ml of the initial populations to a total of 5 ml medium. The populations were then transferred for 10 passages at the same rate, to reflect the <em>in vivo</em> passaging portion of the experiment.</p></div></div><div id="sec-13" class="subsection"><h3>Illumina sequencing of barcodes</h3><p id="p-22">We used a Qiagen Pure Yeast Gene kit to extract DNA for sequencing the 20 bp unique barcodes. For passaging experiment I, DNA was extracted from the initial inocula and from replicate populations after the first, fifth, and 10th <em>in vitro</em> and <em>in vivo</em> passages. For passaging experiment II, DNA was extracted from the first, fifth, and 10th passages <em>in vitro</em> and the first, second, and third passages <em>in vivo</em> in addition to the initial inocula. To obtain enough high-quality DNA for the passages <em>in vivo</em>, we used the cultures obtained after overnight growth of the filtrates collected after the respective passages.</p><p id="p-23">We prepared one Illumina library for each replicate population at each passage in each environment using a barseq (barcode sequencing) approach similar to <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Smith <em>et al.</em> (2010)</a>, by amplifying the uptag barcodes to obtain ∼50 ng/μl of 186 bp PCR product using Ex Taq DNA polymerase (Takara). The primers fused the Illumina adapters to the amplification primers flanking the uptags, OSP12: 5′-AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCTNNNNNNGATGTCCACGAGGTCTCT, and BSeqR: 5′-CAAGCAGAAGACGGCATACGAGATXXXXXXGTGACTGGAGTTCAGACGTGTGCTCTTCCGATCTGTCGACCTGCAGCGTACG, where <em>N</em> is a mixture of all bases and X is a unique 6 bp sequence used for multiplexing. The products were purified using a Qiagen PCR purification kit before sequencing using a multiplexing approach through Illumina Hiseq-2000 50 bp single-end platform at the University of Michigan DNA Sequencing Core. The libraries, <em>i.e.</em>, replicate populations, from the two experiments were divided across three Illumina lanes. Each library was sequenced to an average total depth of 5.08 × 10<sup>6</sup> reads (range, 1.62–9.72 × 10<sup>6</sup>). On average, we obtained ∼4,200,000 reads per library that were assignable to genotypes for both passaging experiments, with a mean of 9.5 × 10<sup>4</sup> reads per genotype for experiment I and 1.0 × 10<sup>3</sup> reads per genotype for experiment II.</p></div><div id="sec-14" class="subsection"><h3>Analysis of barseq data from serial passaging experiment I</h3><p id="p-24">We used a previously described script (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Heisler 2012</a>) to calculate frequencies of strain-specific barcodes from reads obtained through Illumina sequencing to estimate relative fitness of the strains <em>in vivo</em> and <em>in vitro</em>. Specifically, for each library corresponding to a replicate population at a given time point and environment, we trimmed each 50 bp Illumina single-end read by removing the first 24 and last eight nucleotides and retained the remaining 20 bp corresponding to barcode sequences. This 20 bp region was aligned with barcodes in the strain-specific barcode map (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1/TableS2.pdf">Table S2</a>) allowing a maximum of two mismatches to account for the inherent errors during DNA sequencing. Alignments with a score above 90 (on a scale of 0–100) were used to assign the corresponding sequence read to a unique strain. We discovered that one strain was inadvertently outcrossed instead of selfed as planned (SSP81). This strain appeared to be a mating product of two strains: 322134S, a clinical strain from Newcastle, UK and YPS606, an oak bark strain from Pennsylvania. In order to separate the barcodes from SSP81 from YPS606, we confirmed that by the end of the serial passaging experiment, all of the clones were doubly barcoded by colony PCR of &gt;190 clones. Using the ratio of the two barcodes, we estimated that each barcode (YPS06) = 1.75 × barcode (322134S). Correcting for this allowed us to approximately separate SSP81 from YPS606 in both <em>in vivo</em> and <em>in vitro</em> passages. Where estimated sequences of YPS606 were negative due to the correction, a value of zero was used. On average, we were able to assign ∼82% reads per library. The total number of reads obtained for a strain in a given library was used in calculation of frequencies for strains that had &gt;100 reads assigned to them in the initial inoculum.</p><p id="p-25">The fitness of the strains in specific environments was approximated using fold-change (<em>F</em>) between initial and after the first passage using the formula:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/63/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/63/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <em>S<sub>n</sub></em> and <em>S<sub>n</sub></em><sub>−1</sub>, are the total number of reads assigned to strain <em>S</em> at the <em>n</em>th and previous passage, respectively, and <em>T<sub>n</sub></em> and <em>T<sub>n</sub></em><sub>−1</sub> are the total number of reads across all strains in the corresponding libraries. The effect of isolation source, <em>e.g.</em>, habitat, on <em>F</em> was tested using ANOVA for both environments.</p></div><div id="sec-15" class="subsection"><h3>Analysis of barseq data from serial passaging experiment II</h3><p id="p-26">The libraries from the deletion collection were demultiplexed using barcodes downloaded from the <em>Saccharomyces</em> Genome Database. Barcodes found &lt;100 times in any of the initial eight populations were eliminated, leaving a total of 4110 genotypes for analysis. We calculated <em>F</em> for each mutant as above, and used these values to test for correlation between time points using Pearson’s <em>r</em>. In order to find mutant genotypes whose relative fitness differed between <em>in vitro</em> and <em>in vivo</em> conditions, we tested for differential abundance between the sets of replicate populations using an exact test with a negative binomial model with dispersion as implemented in edgeR (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Robinson <em>et al.</em> 2010</a>). We adjusted <em>P</em>-values using the Benjamini–Hochberg method to control the false discovery rate at 5%. The large number of mutant genotypes of significantly different fitness were characterized functionally for differences in gene ontologies using the standard hypergeometric test of the GOStats package (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Falcon and Gentleman 2007</a>). The universe of possible terms was derived from the 4110 genotypes we successfully recovered from the initial populations, and the exploratory gene sets were those that were either significantly more fit in the <em>in vivo</em> or in the <em>in vitro</em> passages. The output of GOStats was plotted with the aid of the GOplot package (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Walter <em>et al.</em> 2015</a>).</p></div><div id="sec-16" class="subsection"><h3>Pseudohyphal filamentation assays</h3><p id="p-27">Strains were grown overnight in 5 ml liquid YPD. For assays of surface filamentation, a dilution of the overnight culture containing roughly 100 cells was spread on synthetic low ammonium dextrose agar medium (0.17% yeast nitrogen base without amino acids or ammonium sulfate, 2% dextrose, and 50 μM ammonium sulfate). Plates were incubated for 5 d at 30° and single colonies were imaged. Pseudohyphal filaments are evident circling the colony periphery. Conditions of low nitrogen were used to induce surface-spread filamentation, as similar conditions have been found to induce pseudohyphal growth efficiently in the Σ1278b strain of <em>S. cerevisiae</em>.</p></div><div id="sec-17" class="subsection"><h3>Virulence estimation</h3><p id="p-28">We measured relative pathogenicity of individual strains for two features. Virulence was estimated using host mortality caused by individual strains following injection, and the relative ability to survive and proliferate in the host was measured using the relative proportion of the injected yeast recovered after 48 hr from inside the host in pairwise competition with another strain.</p><div id="sec-18" class="subsection"><h4>Host mortality:</h4><p id="p-29">Frozen stocks of the haploid strains derived from the wild isolates YPS606, 322134S, and YPS128 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1/TableS1.pdf">Table S1</a>) were grown separately at 30° on YPD agar plates containing g418 (200 μg/ml). For each strain, inoculum was prepared by overnight growth with shaking at 30° in 3 ml liquid YPD containing g418 at the same concentration. The cells were washed and the density was adjusted to 10<sup>9</sup>/ml using sterile water after hemocytometry. This inoculum was used to inject 10 larvae per strain with 10 μl per larva. The number of dead larvae was recorded for 7 d to estimate percent mortality by each strain. A population of 10 larvae, each injected with sterile water was used as a control.</p></div><div id="sec-19" class="subsection"><h4>Yeast recovery following two strain coinjection:</h4><p id="p-30">We used haploid strains of YPS606, 322134S, and YPS128 and L-1528 genetic background to estimate virulence by pairwise competition (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1/TableS1.pdf">Table S1</a>). The strains were sporulated for 3 d in minimal media at room temperature to isolate haploid progeny of <em>MATα</em>. The ploidy of the progeny was verified using the PCR approach as described above. Each of these haploids contained a g418 resistance marker allele at the <em><a href="http://www.yeastgenome.org/locus/S000002386/overview">HO</a></em> locus, which was replaced with an allele conferring resistance to nourseothricin (<em>NAT</em>) to create nearly isogenic pairs of wild strains that differ only by the drug marker (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1/TableS1.pdf">Table S1</a>).</p><p id="p-31">The eight haploid strains (four wild genetic backgrounds × two drug markers) were grown overnight by shaking at 30° in 5 ml liquid YPD containing g418 (200 μg/ml) or NAT (100 μg/ml). The cultures were washed with sterile water the next day and the densities were adjusted to 10<sup>9</sup>/ml. Equal volumes of strains were mixed with each other to create pairwise combinations in which the two strains in a pair represented a distinct genetic background of their wild parental isolate and a distinct drug resistance marker. The initial density of each strain in the mixture was obtained by plating the pairwise mixture separately on YPD agar containing ampicillin (100 μg/ml) and chloramphenicol (17 μg/ml) along with either g418 (200 μg/ml) or NAT (100 μg/ml), prior to injection. The number of colony forming units 48 hr after incubation at 30° were used to estimate initial density per milliliter.</p><p id="p-32">Three larvae (10 μl/larvae) were injected per pairwise combination per replicate and data were obtained from a total of four replicates. For each replicate, the yeast cells were recovered by crushing the three larvae into 30 ml sterile water 48 hr after injection. The extract was serially diluted and plated on the same agar media as described above to obtain colony forming units for each strain in the pair based on the drug marker. The number of colony forming units was counted after 48 hr of incubation at 30° to estimate relative recovery of the strains per ml and per larva. For each pair, differences between the final and initial relative density of each strain out of total were compared with the initial relative density of the strain in the given replicate.</p></div></div><div id="sec-20" class="subsection"><h3>BSA</h3><p id="p-33">To investigate the genetic basis of differences in virulence and survival <em>in vivo</em> between 322134S and YPS128, we performed an analysis of quantitative trait loci using the BSA approach. A clone of <em>MATa</em> in the YPS128 background (SSP96) was isolated and tagged with a cassette containing a hygromycin resistance allele and an RFP marker linked to the <em>MATa</em> allele at the mating type locus (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chin <em>et al.</em> 2012</a>). The derived strain (SSP264) was crossed with a clone of <em>MATα</em> isolated from the background of 322134S (SSP253). The resulting diploids were identified using PCR as detailed above. The diploids were sporulated as explained above and spores were separated using zymolyase followed by sonication (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Rockmill <em>et al.</em> 1991</a>). An Aria II flow cytometer was used to separate the RFP-labeled spores of <em>MATa</em> from the unlabeled <em>MATα</em> spores to obtain a nearly pure haploid population in which all progeny were <em>MATa</em> and unable to mate among each other. Approximately 10<sup>5</sup> RFP-labeled spores from the sorting were plated on YPD agar plates containing hygromycin (200 μg/ml) at 30° for 48 hr, after which they were collected into sterile water at the density of 10<sup>9</sup> cells/ml. This initial inoculum was used to initiate serial passages <em>in vivo</em> and <em>in vitro</em>.</p><p id="p-34">For <em>in vivo</em> passages, 10 μl of the inoculum from each cross was injected into each of five larvae and three serial passages <em>in vivo</em> were conducted using a single replicate and a similar strategy as explained above (see the section <em>Serial passaging experiment I</em>). Briefly, the host larvae were crushed in 50 ml sterile water 48 hr after injection, and the extract was filtered using a sterile 40 μm nylon filter. Yeast cells from 2 ml of the filtrate was combined with 8 ml of YPD containing hygromycin (200 μg/ml) and antibacterial drugs at the same concentration as previously mentioned, and grown overnight with shaking at 200 rpm at 30°. The next day, the cells were washed, adjusted to 10<sup>9</sup> cells/ml, and 10 μl was injected into five new larvae for the next passage. The overnight growth in YPD after the third passage in the host was washed with sterile water and used to extract DNA using a Pure Gene Yeast kit. Genomic DNA libraries were prepared by fragmenting DNA using a Covaris ultrasonicator and a TruSeq (Illumina) library kit, and sequenced on one lane of an Illumina HiSeq-2000 platform to obtain 1.28 × 10<sup>8</sup> 50 bp single-end reads. Sequencing of haploid parents of the BSA cross between 322134S and YPS128 was performed using the Nextera XT kit (Illumina) with sequencing on a portion of a HiSeq-4000 lane with paired end 150 bp reads yielding 7.6 × 10<sup>6</sup> average read pairs per strain.</p><p id="p-35">We also carried out three serial passages <em>in vitro</em>. Specifically, 1 ml of initial inoculum of germinated spores from each cross was added to 9 ml liquid YPD containing hygromycin (200 μg /ml) and incubated overnight at 30° with shaking at 200 rpm. Subsequent passages involved addition of 9 ml fresh medium to 1 ml of the overnight grown populations every 24 hr for 3 d. Genomic DNA libraries were prepared as described above for populations obtained after the third passage <em>in vitro</em> as well as the initial inocula.</p></div><div id="sec-21" class="subsection"><h3>QTL analysis</h3><p id="p-36">We combined the BSA initial, <em>in vitro</em>, and <em>in vivo</em> passages with raw sequence reads of SSP96 and SSP265 to identify segregating SNPs. The data were trimmed using trimmomatic 0.33 with settings (HEADCROP:6 LEADING:33 TRAILING:27 SLIDINGWINDOW:4:28 MINLEN:30) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bolger <em>et al.</em> 2014</a>). Reads were aligned to S288c with bwa-mem (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Li 2013</a>). We then used the standard workflow for SNP identification using GATK v 3.6 (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">McKenna <em>et al.</em> 2010</a>). We used the vcf file of <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bergström <em>et al.</em> (2014)</a> to recalibrate bases using GATK’s BaseRecalibrator. After SNP calling yielded 88,306 SNPs, we used HaplotypeCaller and then filtered the SNPs with VariantFiltration with the following parameters: –filterExpression “QD &lt; 5 || FS &gt; 50.0 || MQ &lt; 50.0 || MQRankSum &lt; −12.5 || ReadPosRankSum &lt; −8.0.” This reduced the data set to 80,593 SNPs. The parental haploid genomes were assembled after trimming using trimmomatic with commands as above (except HEADCROP:4 TRAILING:26 MINLEN:50) using velvet 1.2.10 (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Zerbino and Birney 2008</a>), with hash length optimized using VelvetOptimizer.pl. Because we had the initial population and the parental genomes, we were able to further filter the data to only include those in which the two parents were predicted to be homozygous for different genotypes and for which the initial population was polymorphic, but with an allele frequency of major allele frequency ≤0.85. We also filtered SNPs of Phred-scaled quality &lt;5000 or if any of the BSA pools (initial, <em>in vitro</em>, or <em>in vivo</em>) had a depth of &lt;30 reads. This yielded 44,153 SNPs for analysis.</p><p id="p-37">Estimates of allele frequency of each pool and logarithm of the odds (LOD) scores for QTL between <em>in vivo</em> and <em>in vitro</em> pools were estimated using MULTI POOL (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Edwards and Gifford 2012</a>) using contrast mode. We used <em>n</em> = 1000 (number of individuals per pool), assuming a 99% bottleneck of the population during passaging, although this number likely differs between <em>in vitro</em> and <em>in vivo</em> conditions. MULTI POOL allows the 90% confidence window for each QTL (here considered a LOD score &gt;5) to be estimated as well as the most likely QTL position, using a Bayesian framework that simultaneously incorporates all linked SNPs. Allele frequencies for each pool were estimated by running each pool against itself in contrast mode. BSA observed and estimated SNP frequencies were plotted using the ggplot2 package of R (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Wickham 2009</a>).</p></div><div id="sec-22" class="subsection"><h3>Data availability</h3><p id="p-38">Strains are available upon request. Code for figures and data files have been archived at <a href="https://github.com/Michigan-Mycology/Saccharomyces-barseq">https://github.com/Michigan-Mycology/Saccharomyces-barseq</a>. Raw sequence data has been deposited into NCBI’s SRA archive under BioProject: PRJNA390197, with the accession numbers SRR5684247–SRR5684364.</p></div></div><div class="section results" id="sec-23"><h2 class="">Results</h2><div id="sec-24" class="subsection"><h3>Saccharomyces cerevisiae kills G. mellonella and multiplies in vivo</h3><p id="p-39">We observed that larvae injected with a fatal dose of <em>S. cerevisiae</em> turned a black color (consistent with melanization) within 30 min postinjection as opposed to worms injected with PBS or H<sub>2</sub>O (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). The survivorship of the larvae postinjection was observed to be dosage-dependent (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). Despite the consistent negative effects of the <em>S. cerevisiae</em> inoculation on the larvae, we observed that a large proportion (∼99%) of the yeast cells were killed within 1 hr by the host, based on recovery of cells immediately determined postinjection. However, the cells were able to divide within the host following the bottleneck as evidenced by the estimated doubling time of 8 hr (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1/FigureS1.pdf">Figure S1</a>). Although the negative effects on larvae were consistent, in general the estimates of relative virulence of strains using host survivorship measured on pools of larvae inoculated with a single strain were difficult to repeat. To overcome these challenges, we used coinfection experiments that eliminate many of the variables, presumably due to intrahost individual and environmental factors that we are unable to control. In order to maintain a large population size after infection to facilitate passaging and minimize genetic drift, larvae were inoculated with an overdose (10<sup>7</sup> cells per larva), deceased larvae crushed after 48 hr, the yeast multiplied overnight by addition of YPD medium plus antibiotics to the extract, and finally, the yeast cells were harvested, washed, and adjusted to the appropriate density for the next round of injection. Over passages, we observed an increased accumulation of melanin in the larval extract which was negatively correlated with recovery of cells.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/63/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="S. cerevisiae causes significant mortality to G. mellonella larvae. (A) At 4 hr after injection of 5 &#xD7; 106 cells of BY4722 in 10 &#x3BC;l PBS into 10 Galleria sixth instar larvae, animals show evidence of melanization and morbidity (right dish), whereas controls injected with 10 &#x3BC;l PBS appear healthy. (B) Survivorship for 72 hr is significantly different (P &lt; 0.05) among treatments (PBS control, 2.5 &#xD7; 106 cells per animal, and 5 &#xD7; 106, Cox proportional hazards test), Error bars indicate SD across three replicate populations of 10 larvae per treatment." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-159690276" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;S. cerevisiae&lt;/em&gt; causes significant mortality to &lt;em&gt;G. mellonella&lt;/em&gt; larvae. (A) At 4 hr after injection of 5 &#xD7; 10&lt;sup&gt;6&lt;/sup&gt; cells of BY4722 in 10 &#x3BC;l PBS into 10 &lt;em&gt;Galleria&lt;/em&gt; sixth instar larvae, animals show evidence of melanization and morbidity (right dish), whereas controls injected with 10 &#x3BC;l PBS appear healthy. (B) Survivorship for 72 hr is significantly different (&lt;em&gt;P&lt;/em&gt; &lt; 0.05) among treatments (PBS control, 2.5 &#xD7; 10&lt;sup&gt;6&lt;/sup&gt; cells per animal, and 5 &#xD7; 10&lt;sup&gt;6&lt;/sup&gt;, Cox proportional hazards test)&lt;sub&gt;,&lt;/sub&gt; Error bars indicate SD across three replicate populations of 10 larvae per treatment.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/63/F1.medium.gif" width="440" height="99"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/1/63/F1.medium.gif" width="440" height="99"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/63/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/63/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487948" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1 </span> <p id="p-40" class="first-child"><em>S. cerevisiae</em> causes significant mortality to <em>G. mellonella</em> larvae. (A) At 4 hr after injection of 5 × 10<sup>6</sup> cells of BY4722 in 10 μl PBS into 10 <em>Galleria</em> sixth instar larvae, animals show evidence of melanization and morbidity (right dish), whereas controls injected with 10 μl PBS appear healthy. (B) Survivorship for 72 hr is significantly different (<em>P</em> &lt; 0.05) among treatments (PBS control, 2.5 × 10<sup>6</sup> cells per animal, and 5 × 10<sup>6</sup>, Cox proportional hazards test)<sub>,</sub> Error bars indicate SD across three replicate populations of 10 larvae per treatment.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-25" class="subsection"><h3>Fitness of Saccharomyces cerevisiae genotypes differs in vivo and in vitro</h3><p id="p-41">Serial passaging experiment I used a consortium of wild <em>S. cerevisiae</em> genotypes passaged through two distinct environments: <em>in vivo</em>, using a model host, <em>G. mellonella</em>; and <em>in vitro</em>, using a standard yeast growth medium, YPD. We estimated fitness of genotypes in both environments using their relative frequency, which was obtained from sequencing genotype-specific barcodes from our initial populations as well as after specific passages. Of 43 barcoded genotypes used in passaging experiment I, 10 represent those derived from clinical isolates, whereas the rest represent isolates from various <em>S. cerevisiae</em> habitats and one represents a laboratory hybrid (which was inadvertently included) between a clinical and oak bark strain (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Liti <em>et al.</em> 2009</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Maclean <em>et al.</em> 2017</a>). We hypothesized that one or more clinical isolates will likely have higher fitness in the novel host as compared with nonclinical genotypes, if our model host provides some of the same selective pressures that the pathogen can experience in a mammalian host. To test these hypotheses, we analyzed how relative frequencies of the genotypes changed in replicate populations after the first, fifth, and 10th passage in each environment.</p><p id="p-42">During the course of 10 passages <em>in vitro</em> during experiment I, the frequencies of the 43 genotypes changed dramatically and consistently across replicate populations (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). After a single passage, the initial diversity was not greatly changed, but some strains, such as CLIB413, noticeably increased in frequency. However, by passage 5, three strains emerged as dominant (YJM269, YPS128, and YPS606), and by passage 10 these same strains were even more dominant, comprising &gt;89% of the populations. The <em>in vivo</em> passaging of experiment I showed a more rapid loss of genotype diversity (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). After a single passage, strain CLIB413 showed a dramatic increase (mean 35% across populations), but by passage 5, this genotype was reduced to a low frequency (mean 3%). At passage 5, the hybrid strain (SSP81) represented most (&gt;94%) of the barcodes recovered, and by passage 10, strain SSP81 represented nearly all of the reads excepting some uncertainty due to mathematical error correction for the duplicated barcode in the hybrid and pure strain. The hybrid strain SSP81 is a cross between 322134S, a clinical isolate from Newcastle, UK, and YPS606, an oak bark isolate from the USA, and was inadvertently included in our pool. Below we perform additional experiments to determine the genetic contributions leading to the differential success of SSP81 in the larval habitat.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/63/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Change in relative frequencies of the 43 genotypes in serial passaging experiment I. Each stacked bar plot represents the proportion of the barcode sequences recovered that could be assigned to one of the strains. In the top panel, the frequencies of the eight initial populations are shown. These samples were used to start both in vitro and in vivo passages." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-159690276" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Change in relative frequencies of the 43 genotypes in serial passaging experiment I. Each stacked bar plot represents the proportion of the barcode sequences recovered that could be assigned to one of the strains. In the top panel, the frequencies of the eight initial populations are shown. These samples were used to start both &lt;em&gt;in vitro&lt;/em&gt; and &lt;em&gt;in vivo&lt;/em&gt; passages.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/63/F2.medium.gif" width="302" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/1/63/F2.medium.gif" width="302" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/63/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/63/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487874" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-43" class="first-child">Change in relative frequencies of the 43 genotypes in serial passaging experiment I. Each stacked bar plot represents the proportion of the barcode sequences recovered that could be assigned to one of the strains. In the top panel, the frequencies of the eight initial populations are shown. These samples were used to start both <em>in vitro</em> and <em>in vivo</em> passages.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">In order to test whether clinical isolates performed better than isolates from other habitats, we used fitness as assessed after the first passage <em>in vivo</em> and <em>in vitro</em> (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). There was no correlation of relative fitness of genotypes across the two environments (<em>r</em> = 0.15, <em>P</em> = 0.33). The source of isolation was not a significant factor (one-way ANOVA excluding SSP81, S288c, and DBVPG6765) determining fitness in either <em>in vitro</em> ([<em>F</em>(2,37) = 0.30, <em>P</em> = 0.75]) or <em>in vivo</em> environments ([<em>F</em>(2,37) = 0.91, <em>P</em> = 0.41]).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/63/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Success in in vivo and in vitro habitats in passaging experiment I is not correlated with source of isolation. In the top and bottom panels are shown the fitness of the 43 strains measured after a single passage grouped by isolation source in the in vitro and in vivo environments, respectively. Fitness is measured as the fold change from the initial to after the first passage. Boxes indicate 25th&#x2013;75th percentiles (IQR), and whiskers extend to the points within 1.5&#xD7; the IQR. Outlier points are shown as separate dots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-159690276" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Success in &lt;em&gt;in vivo&lt;/em&gt; and &lt;em&gt;in vitro&lt;/em&gt; habitats in passaging experiment I is not correlated with source of isolation. In the top and bottom panels are shown the fitness of the 43 strains measured after a single passage grouped by isolation source in the &lt;em&gt;in vitro&lt;/em&gt; and &lt;em&gt;in vivo&lt;/em&gt; environments, respectively. Fitness is measured as the fold change from the initial to after the first passage. Boxes indicate 25th&#x2013;75th percentiles (IQR), and whiskers extend to the points within 1.5&#xD7; the IQR. Outlier points are shown as separate dots.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/63/F3.medium.gif" width="440" height="328"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/1/63/F3.medium.gif" width="440" height="328"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/63/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/63/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487914" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-45" class="first-child">Success in <em>in vivo</em> and <em>in vitro</em> habitats in passaging experiment I is not correlated with source of isolation. In the top and bottom panels are shown the fitness of the 43 strains measured after a single passage grouped by isolation source in the <em>in vitro</em> and <em>in vivo</em> environments, respectively. Fitness is measured as the fold change from the initial to after the first passage. Boxes indicate 25th–75th percentiles (IQR), and whiskers extend to the points within 1.5× the IQR. Outlier points are shown as separate dots.</p><div class="sb-div caption-clear"></div></div></div><p id="p-46">Pseudohyphal growth is strongly linked to pathogenicity and virulence in several fungi, including the opportunistic human fungal pathogens <em>C. albicans</em> and <em>Cryptococcus neoformans</em> (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Wang <em>et al.</em> 2002</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Leberer <em>et al.</em> 1996</a>). Thus, pseudohyphal formation may also be involved in persistence or host evasion in the case of <em>G. mellonella</em>. We therefore tested the ability of our wild strains to undergo pseudohyphal growth according to standard methods (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Gimeno <em>et al.</em> 1992</a>). As indicated in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>, the strains were cultured under conditions of nitrogen limitation to induce surface-spread filamentation. We then correlated the ability of these strains to form pseudohyphal filaments with <em>in vivo</em> fitness measured in serial passaging experiment I. These results showed that filamentation (<em>P</em> = 0.03, Mann–Whitney <em>U</em>-test) in our assays was positively associated with pathogenicity in <em>G. mellonella</em> (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/63/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The ability of strains to grow filamentously influences fitness in G. mellonella. (A) Representative photos of strains without or with evidence of filamentous growth on SLAD media after 5 d and photographed at low magnification. (B) Comparison of fitness measured as fold-change (F) in G. mellonella with the ability to grow filamentously. Horizontal lines show means and whiskers show SD. Differences between means were significant (P = 0.03) tested using Mann&#x2013;Whitney U-test (used due to unequal variance between classes)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-159690276" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The ability of strains to grow filamentously influences fitness in &lt;em&gt;G. mellonella&lt;/em&gt;. (A) Representative photos of strains without or with evidence of filamentous growth on SLAD media after 5 d and photographed at low magnification. (B) Comparison of fitness measured as fold-change (&lt;em&gt;F&lt;/em&gt;) in &lt;em&gt;G. mellonella&lt;/em&gt; with the ability to grow filamentously. Horizontal lines show means and whiskers show SD. Differences between means were significant (&lt;em&gt;P&lt;/em&gt; = 0.03) tested using Mann&#x2013;Whitney &lt;em&gt;U&lt;/em&gt;-test (used due to unequal variance between classes).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/63/F4.medium.gif" width="237" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/1/63/F4.medium.gif" width="237" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/63/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/63/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487873" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-47" class="first-child">The ability of strains to grow filamentously influences fitness in <em>G. mellonella</em>. (A) Representative photos of strains without or with evidence of filamentous growth on SLAD media after 5 d and photographed at low magnification. (B) Comparison of fitness measured as fold-change (<em>F</em>) in <em>G. mellonella</em> with the ability to grow filamentously. Horizontal lines show means and whiskers show SD. Differences between means were significant (<em>P</em> = 0.03) tested using Mann–Whitney <em>U</em>-test (used due to unequal variance between classes).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-26" class="subsection"><h3>Genetic analysis of in vivo fitness in the hybrid strain</h3><p id="p-48">Passaging experiment I identified strain SSP81 as being specifically fit in the <em>in vivo</em> habitat. To understand the genetic components of pathogenicity in this hybrid isolate, we evaluated pathogenicity of the two haploid parental components (322134S and YPS606) using pairwise interactions by coinjection, followed by estimating recovery of the two genotypes using selective media. We also included comparisons with one strain YPS128, which was only found to perform well <em>in vitro</em>, and strain L-1528, which performed poorly <em>in vivo</em> and <em>in vitro</em>. The results confirmed the relative fitness of the genotypes <em>in vivo</em>, with strain 322134S outperforming the other three genotypes, and significantly so against L-1528 and YPS128 (<em>t</em>(6) = 5.7, <em>P</em> = 0.001 and <em>t</em>(6) = 4.2, <em>P</em> = 0.006, respectively) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1/FigureS2.pdf">Figure S2</a>). In three of the four replicate coinjections with 322134S and YPS606, strain 322134S outcompeted YPS606, but the fitness was not significantly different over the four replicates. Strain YPS606 also did not perform better than YPS128 and L-1528 <em>in vivo</em>, suggesting overall that the 322134S genotype has higher pathogenicity than YPS606. To support these data, we performed virulence experiments estimating the survivorship of larvae postinoculation with single strains. Despite the high variation among replicate populations in this assay, there were significant differences among strains, with clinical isolate 322134S consistently causing significantly higher host mortality (one-way ANOVA, <em>F</em>(2,8) = 9.2, <em>P</em> = 0.02), killing an average 90% of the hosts in a week compared with the YPS606, which caused an average of only 50% mortality (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1/FigureS3.pdf">Figure S3</a>). There was no significant difference between the host mortality caused by strains 322134S and YPS128 or between YPS606 and YPS128 (Tukey’s HSD).</p><p id="p-49">We then used BSA to identify QTL potentially related to <em>in vivo</em> fitness differences among these strains. Haploid strains 322134S and YPS128 were crossed, the diploid strain (SSP267) sporulated, and ∼10<sup>5</sup> meiotic progeny isolated and passaged independently both <em>in vitr</em>o and <em>in vivo</em>. We insured that only haploid cells were present by tagging the <em>MATa</em> locus with a red fluorescent protein and isolation of red fluorescing spores by flow cytometry. After passaging, the two passaged populations and the initial population before passaging were subjected to whole genome sequencing to reveal QTL indicating differential selection in the two conditions. We considered only SNPs that were predicted to be polymorphic based on the genome sequences of the parents and that had a major allele frequency &lt;0.85 in the initial population. This resulted in 44,153 validated SNPs segregating at an average depth of 132X for the initial population, 135X for the <em>in vitro</em> passaged, and 83X for the <em>in vivo</em> passaged.</p><p id="p-50">The allele frequencies in the bulk selected by <em>in vivo</em> passaging changed more than the <em>in vitro</em> only passaging. The mean frequency of the alleles from the 322134S parent was 0.510 initially. This changed to a mean of 0.513 in the YPD selected pool <em>vs.</em> a mean of 0.497 in the larvae selected pool. Overall, the size of the genomic regions differing from equal segregation, <em>i.e.</em>, 0.5, were large, occasionally as big as some of the smaller chromosomes, suggesting that our population sizes were considerably lower than the number of spores with which we initiated our populations (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Chromosome III, which contains the mating type locus, showed very biased segregation toward the YPS128 parent, showing that the isolation of <em>MATa</em> spores by flow cytometry was successful. Other genomic regions, such as chromosome V, also showed biased segregation of both populations from 0.5. As opposed to the mating type chromosome, this selection was imposed after the passaging as evidenced by deviation from the initial population frequencies.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/63/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Bulk segregant analysis identifies eight QTL regions differentially affecting survivorship in vitro vs. in vivo. Each panel indicates a different chromosome with the top portion the frequency of the allele from the 322134S parent and the lower portion the LOD score. Black dashed line indicates the estimated allele frequency of the initial pool before passaging. Green dots indicate estimated SNP after three passages in vivo, and the green line indicates the estimated frequency of the bulk population. The red dots and lines indicate the corresponding values after three in vitro passages. Eight regions surpassing LOD 5.0 are labeled as QTL and discussed in the text." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-159690276" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Bulk segregant analysis identifies eight QTL regions differentially affecting survivorship &lt;em&gt;in vitro&lt;/em&gt; &lt;em&gt;vs.&lt;/em&gt; &lt;em&gt;in vivo&lt;/em&gt;. Each panel indicates a different chromosome with the top portion the frequency of the allele from the 322134S parent and the lower portion the LOD score. Black dashed line indicates the estimated allele frequency of the initial pool before passaging. Green dots indicate estimated SNP after three passages &lt;em&gt;in vivo&lt;/em&gt;, and the green line indicates the estimated frequency of the bulk population. The red dots and lines indicate the corresponding values after three &lt;em&gt;in vitro&lt;/em&gt; passages. Eight regions surpassing LOD 5.0 are labeled as QTL and discussed in the text.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/63/F5.medium.gif" width="407" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/1/63/F5.medium.gif" width="407" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/63/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/63/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488047" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-51" class="first-child">Bulk segregant analysis identifies eight QTL regions differentially affecting survivorship <em>in vitro</em> <em>vs.</em> <em>in vivo</em>. Each panel indicates a different chromosome with the top portion the frequency of the allele from the 322134S parent and the lower portion the LOD score. Black dashed line indicates the estimated allele frequency of the initial pool before passaging. Green dots indicate estimated SNP after three passages <em>in vivo</em>, and the green line indicates the estimated frequency of the bulk population. The red dots and lines indicate the corresponding values after three <em>in vitro</em> passages. Eight regions surpassing LOD 5.0 are labeled as QTL and discussed in the text.</p><div class="sb-div caption-clear"></div></div></div><p id="p-52">Eight regions showed allele frequency differences between <em>in vitro</em> and <em>in vivo</em> populations with a LOD score &gt;5.0. The QTL fell into two main groups. In the first group both <em>in vitro</em> and <em>in vivo</em> populations differed from the initial in the same direction (QTL1, QTL2, QTL5, QTL7; <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). This pattern supports a favored allele that is beneficial in both environments. The second group are those in which only one population differed from the initial (QTL3, QTL4, QTL6, QTL8). The expectation was that the favored allele in the <em>in vivo</em> population would be derived from the 322134S parent, but this was only the case for QTL4 and QTL8. The strongest QTL effect sizes were ones in which the alleles from the YPS128 parent were favored (QTL1, QTL5, QTL6).</p><p id="p-53">The MULTI POOL method estimates the QTL region with the maximum likelihood (sublocalized best location), but also produces confidence intervals for each QTL. The 90% credible intervals for the eight QTL ranged from 24.4 kp (QTL6) to 969.1 kb (QTL8). The 90% credible intervals comprising QTL contained a minimum of 12 genes to over 100 genes. The genes closest to each of the QTL best locations were <em><a href="http://www.yeastgenome.org/locus/S000000361/overview">ICS2</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000001889/overview">SEC4</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000003046/overview">DBP3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000000999/overview">STE20</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004239/overview">YEF3</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004598/overview">COX14</a></em>, <a href="http://www.yeastgenome.org/locus/S000005039/overview">YNL095C</a>, and <em><a href="http://www.yeastgenome.org/locus/S000005504/overview">NOP8</a></em>, respectively for QTL 1-8. QTL4 is of particular interest as it is the only QTL which shows an increase of the <em>in vivo</em> population for the 322134S allele above the <em>in vitro</em> and initial (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>), and the gene nearest the QTL’s best location, <em><a href="http://www.yeastgenome.org/locus/S000000999/overview">STE20</a></em>, is widely implicated in virulence in other opportunistic fungi (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Calcagno <em>et al.</em> 2004</a>; <a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Wang <em>et al.</em> 2002</a>). However, the QTL4 50% credible region contains seven genes in a span of 21.8 kb. This region contains 132 SNPs and seven gapped regions differing between 322134S and YPS128. All of the genes had SNPs, and most genes had amino acid replacing polymorphisms, making it impossible to determine the alleles underlying this QTL with the current data set.</p></div><div id="sec-27" class="subsection"><h3>Identification of genes required for infection of G. mellonella</h3><p id="p-54">We repeated our serial infection strategy using the <em>S. cerevisiae</em> haploid deletion collection to identify genes important for survival and growth inside our host. Each strain lacks a distinct open reading frame containing a nonessential gene (<a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Winzeler <em>et al.</em> 1999</a>). We created eight replicate populations by pooling cultures of these strains together and passaged each population serially 10 times <em>in vitro</em> and three times <em>in vivo</em>. For each replicate population <em>in vivo</em>, we estimated the density of the yeast recovered from the host after every passage and found the density to decrease below ∼10<sup>4</sup> per individual host for most replicate populations after four passages. Hence, we restricted the experiment <em>in vivo</em> to three passages to avoid a population bottleneck before the ultimate passage. Populations were subjected to barseq after the first, fifth, and 10th passage <em>in vitro</em> and after the first, second, and third passages <em>in vivo</em>. Barcodes of genes not found at &gt;100 counts in the initial populations for each replicate were eliminated, leaving 4110 deletion mutant genotypes that were recovered at high enough abundance to quantify.</p><p id="p-55">Most of the deletion mutants showed consistent patterns across the eight replicates. Moreover, the fitness of the mutants (<em>F</em>) was correlated across time and environments (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1/FigureS4.pdf">Figure S4</a>). Most mutants showed a consistent decrease over passages in both environments with a smaller number increasing over time (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1/FigureS5.pdf">Figure S5</a>). After the first passage, only one mutant, <a href="http://www.yeastgenome.org/locus/S000003883/overview">YJR122W</a>, was lost from all replicate populations <em>in vivo</em>, whereas no strain disappeared completely <em>in vitro</em>. A total of ∼11% of strains (483 out of 4110) disappeared completely by the last passage (all eight replicate populations) from either environment, including 205 <em>in vivo</em> and 400 <em>in vitro</em>, out of which 122 were common between environments. We tested for differential abundance after final passages <em>in vitro</em> <em>vs.</em> <em>in vivo</em> and found 2061 genes of significantly different frequency in the two treatments (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>), with 1357 of higher abundance <em>in vivo</em> <em>vs.</em> 704 higher <em>in vitro</em>. Correlation in log fold-change among genes between the two environments was highly significant (<em>P</em> = 1 × 10<sup>−9</sup>) but weak (<em>r</em> = 0.1).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/63/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Volcano plot of mean fold change between in vivo vs. in vitro passages of serial passaging experiment II. Green points indicate significant P-values corrected using a false-discovery rate of 0.05 (Benjamini&#x2013;Hochberg), and red points are nonsignificant after correction." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-159690276" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Volcano plot of mean fold change between &lt;em&gt;in vivo&lt;/em&gt; &lt;em&gt;vs.&lt;/em&gt; &lt;em&gt;in vitro&lt;/em&gt; passages of serial passaging experiment II. Green points indicate significant &lt;em&gt;P&lt;/em&gt;-values corrected using a false-discovery rate of 0.05 (Benjamini&#x2013;Hochberg), and red points are nonsignificant after correction.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/63/F6.medium.gif" width="303" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/1/63/F6.medium.gif" width="303" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/63/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/63/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487979" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-56" class="first-child">Volcano plot of mean fold change between <em>in vivo</em> <em>vs.</em> <em>in vitro</em> passages of serial passaging experiment II. Green points indicate significant <em>P</em>-values corrected using a false-discovery rate of 0.05 (Benjamini–Hochberg), and red points are nonsignificant after correction.</p><div class="sb-div caption-clear"></div></div></div><p id="p-57">To investigate the underlying biological differences between the differential fitness of these large sets of genes, we analyzed the sets for enrichment or depletion of relevant gene ontology (GO) terms. We analyzed GO terms in the set of genes with significantly higher fitness <em>in vivo</em> than <em>in vitro</em>, and found 49 terms significantly enriched and 13 terms significantly depleted (<em>P</em> &lt; 0.001) in the <em>in vivo</em> set (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). Most of the enriched terms are involved in metabolic processes such as DNA replication, translation, and chromosomal segregation, whereas the majority of the depleted terms (eight out of 13) related to mitochondria. We also analyzed the genes that had significantly higher fitness <em>in vitro</em> than <em>in vivo</em> (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>). There were 23 terms enriched in the <em>in vitro</em> set and 17 depleted (<em>P</em> &lt; 0.001). There was an enrichment of cell wall and peroxisomal genes and a depletion of genes involved in DNA replication, translation, and repair.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/63/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Summary of GO terms enriched and depleted in sets of deletion mutants with significantly higher fitness in vivo (A) or in vitro (B). Size of circles is proportional to the number of genes with that GO term. Values on the x-axis indicate log odds ratio of terms being represented by genes in the selected set vs. the universe of possible genes. For complete data of significant terms see Table S3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-159690276" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Summary of GO terms enriched and depleted in sets of deletion mutants with significantly higher fitness &lt;em&gt;in vivo&lt;/em&gt; (A) or &lt;em&gt;in vitro&lt;/em&gt; (B). Size of circles is proportional to the number of genes with that GO term. Values on the &lt;em&gt;x&lt;/em&gt;-axis indicate log odds ratio of terms being represented by genes in the selected set &lt;em&gt;vs.&lt;/em&gt; the universe of possible genes. For complete data of significant terms see Table S3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/63/F7.medium.gif" width="355" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/1/63/F7.medium.gif" width="355" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/63/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/63/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487981" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-58" class="first-child">Summary of GO terms enriched and depleted in sets of deletion mutants with significantly higher fitness <em>in vivo</em> (A) or <em>in vitro</em> (B). Size of circles is proportional to the number of genes with that GO term. Values on the <em>x</em>-axis indicate log odds ratio of terms being represented by genes in the selected set <em>vs.</em> the universe of possible genes. For complete data of significant terms see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1/TableS3.pdf">Table S3</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-59">We augmented this approach by considering the specific genes that had the greatest differences between <em>in vivo</em> and <em>in vitro</em> passages. The following types of patterns were considered as suggestive of candidates for pathogenicity genes: set 1, genes that were lost only from the <em>in vivo</em> passaging (positive regulators of pathogenicity); set 2, genes that were only lost from the <em>in vitro</em> passaging (negative regulators of pathogenicity); and set 3, genes that were of greatest differential fitness in the two environments. The top 10 genes (with respect to significant differences) for each of the three patterns are listed in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. The functions of these sets were similar to those observed in the GO term enrichment analysis, but provide specific candidates for pathogenicity genes. Set 1 contained a cell wall integrity gene <em><a href="http://www.yeastgenome.org/locus/S000001072/overview">SLT2</a></em> and six mitochondrial genes. Set 2 was enriched for DNA repair and ribosomal genes. Set 3 contained two genes involved in tyrosine metabolism as well as a gene encoding a cell wall mannoprotein, <em><a href="http://www.yeastgenome.org/locus/S000002942/overview">FIT1</a></em>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487905/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487905/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Genes from serial passaging experiment II with the greatest differential fitness</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-28"><h2 class="">Discussion</h2><p id="p-61">Here we tested the ability of a model fungus to behave as a pathogen by injecting it in large doses into living insect larvae to analyze genetic variation in pathogenicity. We found ample evidence of genetic variation in pathogenicity as evidenced by differential recovery from hosts of particular genotypes, differential mortality caused by strains, and variation at loci that explain survival in the larval environment. At the onset of this experiment, our working hypothesis was that <em>S. cerevisiae</em> would not show virulence in <em>G. mellonella</em> based on previous results showing <em>Candida</em> species to be effective at killing <em>G. mellonella</em> but not <em>S. cerevisiae</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cotter <em>et al.</em> 2000</a>). However, we found significant mortality at a similar dosage (2.5 × 10<sup>6</sup> cells per larva; <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>) used by <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Cotter <em>et al.</em> (2000)</a>, and thus differences between our experiments and theirs may be due to particular yeast strains tested. This is particularly likely in this case, because <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Cotter <em>et al.</em> (2000)</a> used only a single yeast strain (JJ1A), which was also auxotrophic (<em>arg</em><sup>−</sup>, <em>thr</em><sup>−</sup>), and threonine auxotrophs are known to have severely attenuated virulence (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Kingsbury and McCusker 2010</a>).</p><p id="p-62">The results from the first passaging experiment with the wild strains showed a striking determinism in equilibrium genotype frequencies across populations, suggesting the habitats provided a selective regime that greatly outweighed genetic drift (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). We did not find that clinical strains had a higher pathogenic potential than strains from other environments. Clinical <em>S. cerevisiae</em> are associated with growth at 42° and the ability to form pseudohyphae (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">McCusker <em>et al.</em> 1994</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Muller <em>et al.</em> 2011</a>). However, both phenotypic and genotypic distinction of <em>S. cerevisiae</em> clinical isolates is debated (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Muller and McCusker 2009</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Klingberg <em>et al.</em> 2008</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">de Llanos <em>et al.</em> 2006</a>). In a mouse infection model, clinical strains were in general more virulent than nonclinical strains (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Clemons <em>et al.</em> 1994</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Llopis <em>et al.</em> 2014</a>). Although clinical strains were not able to survive better <em>in vivo</em> in <em>G. mellonella</em>, we found that the ability of strains to form pseudohyphae was correlated with yeast survival in <em>G. mellonella</em> (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>), and this provides a commonality with <em>S. cerevisiae</em> virulence in mouse. Our genetic screens also suggested genes known to be involved in virulence in other opportunistic fungal pathogens of humans, such as the MAPK pathway and cell wall integrity genes, which have a role in pathogenicity in other opportunistic fungi (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lengeler <em>et al.</em> 2000</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Xu 2000</a>). Our hypothesis is that the diversity of <em>S. cerevisiae</em> ecological roles selects for traits such as pseudohyphal formation and cell wall integrity, which provide resistance to competitors and predators, selection on which preadapts the fungus to function as an opportunistic pathogen. The fact that this study discovered similar genetic mechanisms underlying pathogenicity in <em>S. cerevisiae</em> and other opportunistic fungi signals that there are common traits that link these fungi and gives hope that broad-ranging therapies may be discovered.</p><p id="p-63">Our screen identified a number of genes associated with persistence during larval passaging, which are candidate pathogenicity factors in <em>S. cerevisiae</em> (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). We identified both genes that are similar to those known from other fungal pathogen species as well as novel genes that generate hypotheses for further testing of their contribution to pathogenicity. Here we discuss three types of genes further that have important parallels with other opportunistic fungi: genes involved in cell wall integrity and filamentation, mitochondrial genes, and genes involved in tyrosine metabolism.</p><p id="p-64">We identified that pseudohyphal formation, cell wall integrity, and the genes that control these processes are associated with fitness <em>in vivo</em>. First, we found that there was an association between filamentation and fitness <em>in vivo</em> (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). Filamentous growth is linked to adherence to epithelial cells which promotes biofilm formation and could prevent clearing by phagocytosis (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gale <em>et al.</em> 1998</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Román <em>et al.</em> 2007</a>). In both <em>C. neoformans</em> and <em>Candida</em> spp., cells that form pseudohyphae are able to escape phagocytosis by amoebae and macrophages, respectively (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Neilson <em>et al.</em> 1978</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brunke <em>et al.</em> 2014</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Wartenberg <em>et al.</em> 2014</a>). A similar process could be occurring in the hemolymph of <em>G. mellonella</em> between yeast and phagocytic hemocytes. Second, we determined that the gene closest to the QTL on chromosome VII was <em><a href="http://www.yeastgenome.org/locus/S000000999/overview">STE20</a></em>, a p21-activated signal transducing kinase that is a known pathogenicity factor in other opportunistic fungi (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Calcagno <em>et al.</em> 2004</a>; <a id="xref-ref-77-3" class="xref-bibr" href="#ref-77">Wang <em>et al.</em> 2002</a>), and in <em>S. cerevisiae</em> functions during mating pheromone stimulation, controls bud site selection, pseudohyphal formation, and vacuole inheritance (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gancedo 2001</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Roberts and Fink 1994</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bartholomew and Hardy 2009</a>). Unfortunately, our data were unable to conclusively determine whether the actual gene(s) underlying the effect was <em><a href="http://www.yeastgenome.org/locus/S000000999/overview">STE20</a></em>, and further studies are needed. Components of MAP kinase signaling pathway in pathogenic fungi, such as <a href="http://www.yeastgenome.org/locus/S000000999/overview">Ste20</a>, play diverse regulatory roles in multiple processes such as biofilm formation, stress response, and cell wall remodeling, which may also provide resistance to innate insect immunity (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Román <em>et al.</em> 2007</a>). In particular, cell wall integrity appears to be critical for virulence in a number of fungal pathogens (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Hsu <em>et al.</em> 2015</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Calcagno <em>et al.</em> 2004</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kraus <em>et al.</em> 2003</a>). Our screen with the deletion collection identified at least two specific genes involved in cell wall integrity (<em><a href="http://www.yeastgenome.org/locus/S000001072/overview">SLT2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000002942/overview">FIT1</a></em>; <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>) and also suggested that genes involved in structural constituents of the cell wall (GO: 0005199) were more likely to have low fitness <em>in vivo</em> (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>). In fact, the <em><a href="http://www.yeastgenome.org/locus/S000001072/overview">slt2</a></em>Δ mutant was identified as having the most significant fitness differential between <em>in vivo</em> and <em>in vitro</em>, and was lost across all <em>in vivo</em> passages. <a href="http://www.yeastgenome.org/locus/S000001072/overview">Slt2</a> is also a MAPK signaling gene that has a role in cell wall integrity following response to stress and virulence (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Kraus <em>et al.</em> 2003</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">de Llanos <em>et al.</em> 2010</a>). The <a href="http://www.yeastgenome.org/locus/S000001072/overview">Slt2</a> protein has previously been shown to be involved in maintaining the mannoprotein coat of the <em>S. cerevisiae</em> cell wall such that β-1,3 glucans are not exposed (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Wheeler and Fink 2006</a>). The masking of this fungal-specific antigen is likely essential for evading recognition by β-1,3 glucan binding protein in insects, which recognizes fungal glucan and activates the pathway of pathogen immobilization by melanin (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Kavanagh and Reeves 2004</a>). In summary, our genetic screens have identified proteins involved in cell wall integrity and morphogenesis that likely function in avoidance of the innate insect immune system.</p><p id="p-65">The second group of genes we observed with important implication for <em>S. cerevisiae</em> pathology are genes of mitochondrial function. Our genetic screen with the yeast deletion collection identified multiple mitochondrial genes as being more vital for persistence in the host than growth in YPD (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7</a> and <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Mitochondrial genes play numerous functions beyond aerobic respiration that may be related to pathogenicity, such as control of filamentation, cell wall integrity, and response to hypoxic conditions and oxidative stress (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Calderone <em>et al.</em> 2015</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Ma and May 2010</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Jin <em>et al.</em> 2008</a>), and previous studies have similarly shown that respiration deficient mutants have reduced ability to survive <em>in vivo</em> using a mouse model (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Goldstein and McCusker 2001</a>). Because of the diverse functions of mitochondria, the exact mechanism(s) that links our genes to the ability to survive <em>in vivo</em> are unclear, although multiple hypotheses, such as energy efficiency and resistance to reactive oxygen species, have been proposed to explain why <em>S. cerevisiae</em> mitochondrial mutants are deficient for growth in mouse models (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Goldstein and McCusker 2001</a>).</p><p id="p-66">Our final cohesive group of pathogenicity genes are those involved in cellular aromatic compound metabolism (GO: 0006725). Deletion mutants associated with this GO term were more likely to be have higher fitness <em>in vivo</em> <em>vs.</em> <em>in vitro</em>, suggesting that these compounds are less important for pathogenicity. However, two of the genes in this class were among the two genes with the greatest differential fitness disadvantage <em>in vivo</em> (<em><a href="http://www.yeastgenome.org/locus/S000000370/overview">TYR1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000006264/overview">ARO7</a></em>; <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). Both genes are part of the phenylalanine/tyrosine biosynthesis pathway and are involved in conversions to or from the tyrosine precursor prephenate. Genotypes of <em>S. cerevisiae</em> that are auxotrophic for amino acids are known to have attenuated virulence in mice, although it depends on which amino acid deficiency (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Kingsbury and McCusker 2010</a>; <a id="xref-ref-32-4" class="xref-bibr" href="#ref-32">Goldstein and McCusker 2001</a>). In mice, aromatic amino acids must be synthesized by the pathogen, providing target pathways for antifungal therapies (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kingsbury <em>et al.</em> 2006</a>). It may be that the poor <em><a href="http://www.yeastgenome.org/locus/S000000370/overview">tyr1</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000006264/overview">aro7</a></em>Δ mutant fitness in <em>G. mellonella</em> derives from low hemolymph concentrations of tyrosine, phenylalanine, or the production of a toxic intermediate. However, we speculate that tyrosine <em>in vivo</em> concentration may be particularly low during the rapid production melanization response because DOPA-melanin is produced from tyrosine precursors (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>).</p><p id="p-67">Beyond specific classes of genes, our approach provides insight into the architecture of genetic variation of pathogenicity through genetic mapping. Together, these data show that virulence is controlled by a large number of genes, and that there are variants segregating in natural populations that have large effect, <em>e.g.</em>, QTL. BSA identified four QTL that showed the same allele was favored in both <em>in vitro</em> and <em>in vivo</em> environments. Because <em>in vivo</em> passaging also includes a growth step in rich media, it is impossible to distinguish whether these are due to selection for or against alleles of a very general fitness effect or whether there is selection specific to the growth in rich media. BSA also identified three genomic regions in which one parental allele was favored only in the larval passaged pool, and only one of these was from the clinical parent, QTL 4 (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). The fact that loci from both parents significantly contribute to survival in the larval environment may explain the observation of heterosis we observed in the hybrid strain.</p><p id="p-68"> The fact that clinical strains of <em>S. cerevisiae</em> do not form a genetic group (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Muller and McCusker 2009</a>) reinforces the notion that virulence or pathogenicity is a complex trait. At least one other study has shown that virulence in an opportunistic pathogen, <em>Aspergillus nidulans</em>, measured by inoculation of <em>G. mellonella</em> is controlled by multiple genes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Christians <em>et al.</em> 2011</a>), which is consistent with a complex genetic architecture underlying virulence in opportunistic fungi. The complexity of virulence as a trait is magnified by the quantitative nature of traits that underlie it, such as thermotolerance, melanin production, and resistance to oxidative stress (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Steinmetz <em>et al.</em> 2002</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Vogan <em>et al.</em> 2016</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Diezmann and Dietrich 2011</a>).</p><p id="p-69">Our study was conducted on strain pools of varying ploidy. Specifically, the wild strains of the first serial passaging were diploid, and the strains from all other experiments were haploid <em>S. cerevisiae</em> is predominantly diploid in the wild, and a large survey of 144 clinical strains has shown that clinical strains are also predominantly diploid, with the remainder roughly equally distributed among haploid, triploid, or tetraploid (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Zhu <em>et al.</em> 2016</a>). The general difference in pathogenic potential between haploid and diploid <em>S. cerevisiae</em> is uncertain, but it is known that there are clear differences in pseudohyphal formation between haploids and diploids that may lead to differences in pathogenic potential (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Liu <em>et al.</em> 1996</a>). Because of the nature of the experiments we wished to conduct, we were required to utilize haploid strains, which is the ploidy of most of the genotypes used in previous studies on pathogenicity in <em>S. cerevisiae</em> (<a id="xref-ref-32-5" class="xref-bibr" href="#ref-32">Goldstein and McCusker 2001</a>). Although we did not perform extensive comparisons, the fitness measured using diploids in our serial passaging experiment I (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>) matched well with the coinfection experiments we conducted using pairs of haploid genotypes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1/FigureS2.pdf">Figure S2</a>).</p><p id="p-70">The increased use of invertebrate hosts to study opportunistic fungi requires pathogenicity factors to supersede host, environment, and even fungal species. To varying degrees, fungal pathogenicity factors have been correlated between mouse, <em>G. mellonella</em>, and <em>C. elegans</em> models (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Desalermos <em>et al.</em> 2015</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Navarro-Velasco <em>et al.</em> 2011</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brennan <em>et al.</em> 2002</a>). In both invertebrate and vertebrate host models, common virulence factors are genes controlling filamentation, cell wall integrity, melanin production, siderophores, and intracellular signaling (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Desalermos <em>et al.</em> 2015</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Navarro-Velasco <em>et al.</em> 2011</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Firacative <em>et al.</em> 2014</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Slater <em>et al.</em> 2011</a>). There are strong similarities between the innate immunity of insects and mammals and their similar response to fungal infection (<a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Kavanagh and Reeves 2004</a>). However, <em>G. mellonella</em> presents important differences from mammals such as being ectothermic, having an open circulatory system, and absence of adaptive immunity. Despite these obvious differences, common virulence factors, and higher virulence of particular clinical strains in <em>G. mellonella</em> for at least two species (<a id="xref-ref-15-4" class="xref-bibr" href="#ref-15">Cotter <em>et al.</em> 2000</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Mylonakis <em>et al.</em> 2005</a>) suggests that it is an appropriate model host to learn insights into yeast pathogenicity. If our <em>in vivo</em> fitness analyses provide a good model for understanding pathogenicity genes, then the pathogenicity of genotypes should be correlated regardless of host. Comparison of our results with known pathogenicity factors revealed that <em><a href="http://www.yeastgenome.org/locus/S000002701/overview">SSD1</a></em> (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Wheeler <em>et al.</em> 2003</a>), <em><a href="http://www.yeastgenome.org/locus/S000005857/overview">MIP1</a></em> (<a id="xref-ref-32-6" class="xref-bibr" href="#ref-32">Goldstein and McCusker 2001</a>), and <em><a href="http://www.yeastgenome.org/locus/S000001067/overview">THR1</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000000649/overview">THR4</a></em> (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Kingsbury and McCusker 2010</a>) deletion mutants showed similar results with deletion mutants having significantly lower pathogenicity. In contrast, we found the opposite pattern with <em><a href="http://www.yeastgenome.org/locus/S000005654/overview">ADE2</a></em> and <em><a href="http://www.yeastgenome.org/locus/S000004915/overview">ADE4</a></em> mutants, which are severely attenuated in pathogenicity in mouse (<a id="xref-ref-32-7" class="xref-bibr" href="#ref-32">Goldstein and McCusker 2001</a>), but which had a higher relative fitness in <em>G. mellonella</em> than <em>in vitro</em>. More recently, there has been development of a nematode pathogenicity model for <em>S. cerevisiae</em> using the host <em>Caenorhabditis elegans</em> (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jain <em>et al.</em> 2009</a>). The three genes involved in resistance to reactive oxygen species that were found by the authors to be required to trigger a virulence phenotype (<em><a href="http://www.yeastgenome.org/locus/S000003865/overview">SOD1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000004466/overview">YAP1</a></em>, <em><a href="http://www.yeastgenome.org/locus/S000002831/overview">YAP2</a></em>) were not observed to be detrimental to fitness <em>in vivo</em> in <em>G. mellonella</em>. Instead, <em><a href="http://www.yeastgenome.org/locus/S000003865/overview">SOD1</a></em> deletion mutants had very low fitness in both habitats, <em><a href="http://www.yeastgenome.org/locus/S000002831/overview">YAP2</a></em> had slightly higher fitness <em>in vivo</em>, and <em><a href="http://www.yeastgenome.org/locus/S000004466/overview">YAP1</a></em> had a much higher relative fitness <em>in vivo</em> (41st highest of 4110). Major differences between our results and the other model hosts are not only to do with the experimental system, but also the genome-wide approach that we used, allowing us to find the genes of highest fitness effect rather than choosing candidate genes.</p><p id="p-71">Our experimental system also imposed limitations due to the <em>in vivo</em> passaging strategy. In between each round of injection, pools were grown overnight in YPD to generate enough cells for the next round of injection. This was dictated by the reduction of the initial population size to ∼1% right after injection, presumably due to the action of the innate insect immunity. Both the bottlenecks imposed by the host and low spore germination led to a lower population size in our BSA that limited the mapping resolution we were able to achieve. Indeed, the mapping resolution does not allow us to identify specific genes underlying the QTL. Growth of the populations in YPD in between passages effectively creates a constantly changing environment with somewhat unpredictable outcomes through selection: antagonistic pleiotropy may limit the efficacy of selection, genotypes with higher plasticity may be favored, or balancing selection and heterosis may be favored. This latter hypothesis is particularly interesting with respect to our observation that the wild strain that was a laboratory-created hybrid with high heterozygosity was the clear winner in each of the eight replicate <em>in vivo</em> passages of serial passaging experiment I. The fact that one of the parents was a clinical strain and the other highly fit in the YPD passages may suggest that the hybrid has heterozygote advantage by combining alleles that perform well <em>in vivo</em> and <em>in vitro</em>, which was experienced during the larval passaging. There is an uncertain but significant association with clinical strains of <em>S. cerevisiae</em> and heterozygosity, with clinical strains showing roughly 50% more heterozygosity than strains from other habitats (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Muller and McCusker 2009</a>). This could possibly reflect fitness tradeoffs of mutations that provide an advantage inside the host, but whose expression bear a cost outside the host. Such fitness tradeoffs are well known across a number of yeast traits (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Sunshine <em>et al.</em> 2015</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Sellis <em>et al.</em> 2016</a>).</p><p id="p-72">In this study we have leveraged the tools of yeast genetics to study pathogenicity across the genome to identify a large number of candidate genes involved in survival inside an insect host. The data suggest that the host/fungus combination is an adequate model for studying pathogenicity properties of opportunistic fungi because the identification of genes in our screens overlaps with well-known virulence factors in other fungi. The approaches we employed could be readily extended to other species of fungi and other invertebrate hosts. Because environmental opportunistic fungi generally do not transmit from host to host, our passaging regime does not contradict what is observed in nature and allows persistent selection to identify alleles that function as pathogenicity factors. The most significant factors we uncovered generally point to mechanisms that would lead to resistance or evasion of innate immunity. This seems logical, given that 99% of cells were cleared immediately after injection and the strong melanization response by the host (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1A</a>). Because commonalities are found between virulence of environmental fungi in insects, mammals, and even protists, the data suggest that the primary virulence factors function at the level of innate immunity. As related pathogenicity mechanisms are detected across fungal species, they provide clues to development of novel therapies.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-73">We thank James Tan, John Recknagel, and Ellen James for technical assistance in the laboratory. We also thank Brian Metzger and Fabian Duveau for helpful discussions and advice on methods. Jianzhi Zhang generously provided the wild type strains. This work was made possible by a grant to T.Y.J. from the National Institutes of Health, National Institute of Allergy and Infectious Diseases (5R21AI105167-02). A.K. was supported by grants 1R01-A1098450-01A1 and R21-A1-114837 from the National Institutes of Health.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-2"><a class="rev-xref" href="#xref-fn-2-1">↵</a><span class="fn-label">3</span> Department of Integrative Biology, University of Texas at Austin, Austin, Texas 78712</p></li><li class="fn" id="fn-3"><p id="p-3"><a class="rev-xref" href="#xref-fn-3-1">↵</a><span class="fn-label">4</span> Department of Biology, Indiana University, Bloomington, Indiana 47405</p></li><li class="fn" id="fn-5"><p id="p-74">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300245/-/DC1</a>.</p></li><li class="fn" id="fn-6"><p id="p-75"><em>Communicating editor: B. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-09-06"><span class="received-label">Received </span>September 6, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-10-14"><span class="accepted-label">Accepted </span>October 14, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Phadke <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.1" data-doi="10.1016/j.fgb.2014.09.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alspaugh</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Virulence mechanisms and <em>Cryptococcus neoformans</em> pathogenesis.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2BGenet.%2BBiol.%26rft.volume%253D78%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2014.09.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2014.09.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.2" data-doi="10.1128/EC.00111-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bartholomew</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hardy</span>  <span class="cit-name-given-names">C. F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">p21-activated kinases Cla4 and Ste20 regulate vacuole inheritance in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">560</span>–<span class="cit-lpage">572</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00111-08%26rft_id%253Dinfo%253Apmid%252F19218422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI4LzQvNTYwIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.3" data-doi="10.1093/molbev/msu037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergström</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barre</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A high-definition view of functional genetic variation from natural yeast genomes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">872</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu037%26rft_id%253Dinfo%253Apmid%252F24425782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24425782&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333283200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.4" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usadel</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.5" data-doi="10.1111/j.1574-695X.2002.tb00617.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brennan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">D. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whiteway</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kavanagh</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Correlation between virulence of <em>Candida albicans</em> mutants in mice and <em>Galleria mellonella</em> larvae.</span> <abbr class="cit-jnl-abbrev">FEMS Immunol. Med. Microbiol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">153</span>–<span class="cit-lpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BImmunol.%2BMed.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-695X.2002.tb00617.x%26rft_id%253Dinfo%253Apmid%252F12381467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-695X.2002.tb00617.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12381467&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brunke</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seider</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fischer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobsen</span>  <span class="cit-name-given-names">I. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasper</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">One small step for a yeast - microevolution within macrophages renders <em>Candida glabrata</em> hypervirulent due to a single point mutation.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D10%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.7" data-doi="10.1002/yea.1125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calcagno</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bignell</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Canedo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muhlschlegel</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title"><em>Candida glabrata Ste20</em> is involved in maintaining cell wall integrity and adaptation to hypertonic stress, and is required for wild-type levels of virulence.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">557</span>–<span class="cit-lpage">568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DCalcagno%26rft.auinit1%253DA.%2BM.%26rft.volume%253D21%26rft.issue%253D7%26rft.spage%253D557%26rft.epage%253D568%26rft.atitle%253DCandida%2Bglabrata%2BSte20%2Bis%2Binvolved%2Bin%2Bmaintaining%2Bcell%2Bwall%2Bintegrity%2Band%2Badaptation%2Bto%2Bhypertonic%2Bstress%252C%2Band%2Bis%2Brequired%2Bfor%2Bwild-type%2Blevels%2Bof%2Bvirulence.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1125%26rft_id%253Dinfo%253Apmid%252F15164359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15164359&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221567300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calderone</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Traven</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">System-level impact of mitochondria on fungal virulence: to metabolism and beyond.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft.volume%253D15%26rft.spage%253D11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casadevall</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pirofski</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Host-pathogen interactions: redefining the basic concepts of virulence and pathogenicity.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">3703</span>–<span class="cit-lpage">3713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DCasadevall%26rft.auinit1%253DA.%26rft.volume%253D67%26rft.issue%253D8%26rft.spage%253D3703%26rft.epage%253D3713%26rft.atitle%253DHost-Pathogen%2BInteractions%253A%2BRedefining%2Bthe%2BBasic%2BConcepts%2Bof%2BVirulence%2Band%2BPathogenicity%26rft_id%253Dinfo%253Apmid%252F10417127%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjk6IjY3LzgvMzcwMyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzgvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.10" data-doi="10.1128/EC.00308-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casadevall</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pirofski</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Accidental virulence, cryptic pathogenesis, Martians, lost hosts, and the pathogenicity of environmental microbes.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2169</span>–<span class="cit-lpage">2174</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00308-07%26rft_id%253Dinfo%253Apmid%252F17951515%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo5OiI2LzEyLzIxNjkiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy84LzEvNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.11" data-doi="10.1016/S1369-5274(03)00082-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casadevall</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steenbergen</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nosanchuk</span>  <span class="cit-name-given-names">J. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">‘Ready made’ virulence and ‘dual use’ virulence factors in pathogenic environmental fungi—the <em>Cryptococcus neoformans</em> paradigm.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Microbiol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">332</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bmicrobiology%26rft.stitle%253DCurr%2BOpin%2BMicrobiol%26rft.aulast%253DCasadevall%26rft.auinit1%253DA.%26rft.volume%253D6%26rft.issue%253D4%26rft.spage%253D332%26rft.epage%253D337%26rft.atitle%253D%2527Ready%2Bmade%2527%2Bvirulence%2Band%2B%2527dual%2Buse%2527%2Bvirulence%2Bfactors%2Bin%2Bpathogenic%2Benvironmental%2Bfungi--the%2BCryptococcus%2Bneoformans%2Bparadigm.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1369-5274%252803%252900082-1%26rft_id%253Dinfo%253Apmid%252F12941400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1369-5274(03)00082-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12941400&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185173000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frizzell</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Timberlake</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">FASTER MT: isolation of pure populations of a and α ascospores from <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D449%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.13" data-doi="10.1371/journal.pone.0021746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christians</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheema</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vergara</span>  <span class="cit-name-given-names">I. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watt</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinto</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantitative trait locus (QTL) mapping reveals a role for unstudied genes in <em>Aspergillus</em> virulence.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D6%26rft.spage%253D12%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0021746%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0021746&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.14" data-doi="10.1093/infdis/169.4.859"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clemons</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Comparative pathogenesis of clinical and nonclinical isolates of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">J. Infect. Dis.</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">859</span>–<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInfect.%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Finfdis%252F169.4.859%26rft_id%253Dinfo%253Apmid%252F8133102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/infdis/169.4.859&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8133102&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NP11100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.15" data-doi="10.1111/j.1574-695X.2000.tb01427.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cotter</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kavanagh</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Development of an insect model for the <em>in vivo</em> pathogenicity testing of yeasts.</span> <abbr class="cit-jnl-abbrev">FEMS Immunol. Med. Microbiol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BImmunol.%2BMed.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-695X.2000.tb01427.x%26rft_id%253Dinfo%253Apmid%252F10640612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-695X.2000.tb01427.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10640612&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.16" data-doi="10.1007/s10482-006-9077-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Llanos</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández-Espinar</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Querol</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A comparison of clinical and food <em>Saccharomyces cerevisiae</em> isolates on the basis of potential virulence factors.</span> <abbr class="cit-jnl-abbrev">Antonie van Leeuwenhoek</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAntonie%2Bvan%2BLeeuwenhoek%26rft.stitle%253DAntonie%2Bvan%2BLeeuwenhoek%26rft.aulast%253Dde%2BLlanos%26rft.auinit1%253DR.%26rft.volume%253D90%26rft.issue%253D3%26rft.spage%253D221%26rft.epage%253D231%26rft.atitle%253DA%2Bcomparison%2Bof%2Bclinical%2Band%2Bfood%2BSaccharomyces%2Bcerevisiae%2Bisolates%2Bon%2Bthe%2Bbasis%2Bof%2Bpotential%2Bvirulence%2Bfactors.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10482-006-9077-7%26rft_id%253Dinfo%253Apmid%252F16871421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10482-006-9077-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16871421&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000241175400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Llanos</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernández-Haro</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrio</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Querol</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández-Espinar</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Differences in activation of MAP kinases and variability in the polyglutamine tract of Slt2 in clinical and non-clinical isolates of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">549</span>–<span class="cit-lpage">561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253Dde%2BLlanos%26rft.auinit1%253DR.%26rft.volume%253D27%26rft.issue%253D8%26rft.spage%253D549%26rft.epage%253D561%26rft.atitle%253DDifferences%2Bin%2Bactivation%2Bof%2BMAP%2Bkinases%2Band%2Bvariability%2Bin%2Bthe%2Bpolyglutamine%2Btract%2Bof%2BSlt2%2Bin%2Bclinical%2Band%2Bnon-clinical%2Bisolates%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F20586115%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20586115&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.18" data-doi="10.1093/infdis/jiu441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Desalermos</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">X. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajamuthiah</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arvanitis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A multi-host approach for the systematic analysis of virulence factors in <em>Cryptococcus neoformans</em>.</span> <abbr class="cit-jnl-abbrev">J. Infect. Dis.</abbr> <span class="cit-vol">211</span>: <span class="cit-fpage">298</span>–<span class="cit-lpage">305</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BInfect.%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Finfdis%252Fjiu441%26rft_id%253Dinfo%253Apmid%252F25114160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/infdis/jiu441&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25114160&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.19" data-doi="10.1534/genetics.111.128256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diezmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dietrich</span>  <span class="cit-name-given-names">F. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Oxidative stress survival in a clinical <em>Saccharomyces cerevisiae</em> isolate is influenced by a major quantitative trait nucleotide.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">709</span>–<span class="cit-lpage">722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.128256%26rft_id%253Dinfo%253Apmid%252F21515583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzMvNzA5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.20" data-doi="10.1073/pnas.0803232105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lafferty</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuris</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hechinger</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jetz</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Homage to Linnaeus: how many parasites? How many hosts?</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">11482</span>–<span class="cit-lpage">11489</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0803232105%26rft_id%253Dinfo%253Apmid%252F18695218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoyMjoiMTA1L1N1cHBsZW1lbnRfMS8xMTQ4MiI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzgvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.63.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ebert</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bull</span>  <span class="cit-name-given-names">J. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The evolution and expression of virulence</span>, pp. <span class="cit-fpage">153</span>–<span class="cit-lpage">167</span> in <span class="cit-source">Evolution in Health and Disease</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Stearns</span>  <span class="cit-name-given-names">S. C.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Koella</span>  <span class="cit-name-given-names">J. C.</span></span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gifford</span>  <span class="cit-name-given-names">D. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-resolution genetic mapping with pooled sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DTomii%26rft.auinit1%253DK.%26rft.volume%253D13%26rft.spage%253D11%26rft.epage%253D11%26rft.atitle%253DConvergent%2Bevolution%2Bin%2Bstructural%2Belements%2Bof%2Bproteins%2Binvestigated%2Busing%2Bcross%2Bprofile%2Banalysis.%26rft_id%253Dinfo%253Apmid%252F22244085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22244085&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.23" data-doi="10.1086/497832"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Enache-Angoulvant</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hennequin</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Invasive <em>Saccharomyces</em> infection: a comprehensive review.</span> <abbr class="cit-jnl-abbrev">Clin. Infect. Dis.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">1559</span>–<span class="cit-lpage">1568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClin.%2BInfect.%2BDis.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F497832%26rft_id%253Dinfo%253Apmid%252F16267727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/497832&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16267727&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233018600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.24" data-doi="10.1093/bioinformatics/btl567"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falcon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gentleman</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Using GOstats to test gene lists for GO term association.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl567%26rft_id%253Dinfo%253Apmid%252F17098774%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btl567&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17098774&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243992400060&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.25" data-doi="10.1371/journal.pone.0108788"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Firacative</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title"><em>Galleria mellonella</em> model identifies highly virulent strains among all major molecular types of <em>Cryptococcus gattii</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253D12%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0108788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0108788&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.26" data-doi="10.1038/nature04220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fraser</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giles</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wenink</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geunes-Boyer</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Same-sex mating and the origin of the Vancouver Island <em>Cryptococcus gattii</em> outbreak.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">437</span>: <span class="cit-fpage">1360</span>–<span class="cit-lpage">1364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFraser%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D437%26rft.issue%253D7063%26rft.spage%253D1360%26rft.epage%253D1364%26rft.atitle%253DSame-sex%2Bmating%2Band%2Bthe%2Borigin%2Bof%2Bthe%2BVancouver%2BIsland%2BCryptococcus%2Bgattii%2Boutbreak.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04220%26rft_id%253Dinfo%253Apmid%252F16222245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16222245&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232829100055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.27" data-doi="10.4161/viru.1.6.12985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fuchs</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Brien</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El Khoury</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mylonakis</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Methods for using <em>Galleria mellonella</em> as a model host to study fungal pathogenesis.</span> <abbr class="cit-jnl-abbrev">Virulence</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">475</span>–<span class="cit-lpage">482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirulence%26rft.volume%253D1%26rft.spage%253D475%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fviru.1.6.12985%26rft_id%253Dinfo%253Apmid%252F21178491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/viru.1.6.12985&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21178491&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292521800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.28" data-doi="10.1126/science.279.5355.1355"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gale</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bendel</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McClellan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hauser</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Linkage of adhesion, filamentous growth, and virulence in <em>Candida albicans</em> to a single gene, <em>INT1</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">279</span>: <span class="cit-fpage">1355</span>–<span class="cit-lpage">1358</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGale%26rft.auinit1%253DC.%2BA.%26rft.volume%253D279%26rft.issue%253D5355%26rft.spage%253D1355%26rft.epage%253D1358%26rft.atitle%253DLinkage%2Bof%2BAdhesion%252C%2BFilamentous%2BGrowth%252C%2Band%2BVirulence%2Bin%2BCandida%2Balbicans%2Bto%2Ba%2BSingle%2BGene%252C%2BINT1%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.279.5355.1355%26rft_id%253Dinfo%253Apmid%252F9478896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzkvNTM1NS8xMzU1IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.29" data-doi="10.1111/j.1574-6976.2001.tb00573.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gancedo</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Control of pseudohyphae formation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1574-6976.2001.tb00573.x%26rft_id%253Dinfo%253Apmid%252F11152942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1574-6976.2001.tb00573.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11152942&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000166463400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.30" data-doi="10.1038/nature00935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riles</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional profiling of the <em>Saccharomyces cerevisiae</em> genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.31" data-doi="10.1016/0092-8674(92)90079-R"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gimeno</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ljungdahl</span>  <span class="cit-name-given-names">P. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styles</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Unipolar cell divisions in the yeast <em>Saccharomyces cerevisiae</em> lead to filamentous growth: regulation by starvation and RAS.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">1077</span>–<span class="cit-lpage">1090</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DGimeno%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D68%26rft.issue%253D6%26rft.spage%253D1077%26rft.epage%253D1090%26rft.atitle%253DUnipolar%2Bcell%2Bdivisions%2Bin%2Bthe%2Byeast%2BS.%2Bcerevisiae%2Blead%2Bto%2Bfilamentous%2Bgrowth%253A%2Bregulation%2Bby%2Bstarvation%2Band%2BRAS.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252892%252990079-R%26rft_id%253Dinfo%253Apmid%252F1547504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(92)90079-R&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1547504&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HK67400011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Development of <em>Saccharomyces cerevisae</em> as a model pathogen: a system for the genetic identification of gene products required for survival in the mammalian host environment.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">513</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGoldstein%26rft.auinit1%253DA.%2BL.%26rft.volume%253D159%26rft.issue%253D2%26rft.spage%253D499%26rft.epage%253D513%26rft.atitle%253DDevelopment%2Bof%2BSaccharomyces%2Bcerevisiae%2Bas%2Ba%2BModel%2BPathogen%253A%2BA%2BSystem%2Bfor%2Bthe%2BGenetic%2BIdentification%2Bof%2BGene%2BProducts%2BRequired%2Bfor%2BSurvival%2Bin%2Bthe%2BMammalian%2BHost%2BEnvironment%26rft_id%253Dinfo%253Apmid%252F11606528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU5LzIvNDk5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-web" id="cit-8.1.63.33"><div class="cit-metadata unstructured">Heisler, L., 2012 BarSeq: identification and counting of short molecular barcodes in HT-Seq data. Available at: <a href="http://chemogenomics.med.utoronto.ca/supplemental/barseq_analysis/files.php">http://chemogenomics.med.utoronto.ca/supplemental/barseq_analysis/files.php</a>.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hsu</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">Y. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Characterization of cell wall proteins in <em>Saccharomyces cerevisiae</em> clinical isolates elucidates Hsp150p in virulence.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.35" data-doi="10.1016/0168-9525(90)90190-H"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huxley</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Green</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Rapid assessment of <em>S. cerevisiae</em> mating type by PCR.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DHuxley%26rft.auinit1%253DC.%26rft.volume%253D6%26rft.issue%253D8%26rft.spage%253D236%26rft.epage%253D236%26rft.atitle%253DRapid%2Bassessment%2Bof%2BS.%2Bcerevisiae%2Bmating%2Btype%2Bby%2BPCR.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0168-9525%252890%252990190-H%26rft_id%253Dinfo%253Apmid%252F2238077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0168-9525(90)90190-H&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2238077&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990DT54000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.36" data-doi="10.1128/EC.00367-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yun</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Politz</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">R. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A pathogenesis assay using <em>Saccharomyces cerevisiae</em> and <em>Caenorhabditis elegans</em> reveals novel roles for yeast AP-1, Yap1, and host dual oxidase BLI-3 in fungal pathogenesis.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1218</span>–<span class="cit-lpage">1227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00367-08%26rft_id%253Dinfo%253Apmid%252F19502579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo4OiI4LzgvMTIxOCI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzgvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.37" data-doi="10.1091/mbc.E07-05-0519"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dobry</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Large-scale analysis of yeast filamentous growth by systematic gene disruption and overexpression.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">284</span>–<span class="cit-lpage">296</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E07-05-0519%26rft_id%253Dinfo%253Apmid%252F17989363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE5LzEvMjg0IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.38" data-doi="10.1016/j.femsre.2003.09.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kavanagh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reeves</span>  <span class="cit-name-given-names">E. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Exploiting the potential of insects for <em>in vivo</em> pathogenicity testing of microbial pathogens.</span> <abbr class="cit-jnl-abbrev">FEMS Microbiol. Rev.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BMicrobiol.%2BRev.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.femsre.2003.09.002%26rft_id%253Dinfo%253Apmid%252F14975532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.femsre.2003.09.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14975532&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000189242400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.39" data-doi="10.1128/EC.00045-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kingsbury</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Fungal homoserine kinase (<em>thr1Δ</em>) mutants are attenuated in virulence and die rapidly upon threonine starvation and serum incubation.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">729</span>–<span class="cit-lpage">737</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryot.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.00045-10%26rft_id%253Dinfo%253Apmid%252F20305003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI5LzUvNzI5IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.40" data-doi="10.1128/EC.5.5.816-824.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kingsbury</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Role of nitrogen and carbon transport, regulation, and metabolism genes for <em>Saccharomyces cerevisiae</em> survival i<em>n vivo</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">816</span>–<span class="cit-lpage">824</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DKingsbury%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D5%26rft.issue%253D5%26rft.spage%253D816%26rft.epage%253D824%26rft.atitle%253DRole%2Bof%2BNitrogen%2Band%2BCarbon%2BTransport%252C%2BRegulation%252C%2Band%2BMetabolism%2BGenes%2Bfor%2BSaccharomyces%2Bcerevisiae%2BSurvival%2BIn%2BVivo%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.5.5.816-824.2006%26rft_id%253Dinfo%253Apmid%252F16682459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiI1LzUvODE2IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.41" data-doi="10.1111/j.1567-1364.2008.00365.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Klingberg</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lesnik</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arneborg</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raspor</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jespersen</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Comparison of <em>Saccharomyces cerevisiae</em> strains of clinical and nonclinical origin by molecular typing and determination of putative virulence traits.</span> <abbr class="cit-jnl-abbrev">FEMS Yeast Res.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">631</span>–<span class="cit-lpage">640</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEMS%2BYeast%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1567-1364.2008.00365.x%26rft_id%253Dinfo%253Apmid%252F18355272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1567-1364.2008.00365.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18355272&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255831700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kowalski</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beattie</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuller</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGurk</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">Y. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Heterogeneity among isolates reveals that fitness in low oxygen correlates with <em>Aspergillus fumigatus</em> virulence.</span> <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D7%26rft.spage%253D13%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.43" data-doi="10.1046/j.1365-2958.2003.03508.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kraus</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heitman</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The <em>Cryptococcus neoformans</em> MAP kinase Mpk1 regulates cell integrity in response to antifungal drugs and loss of calcineurin function.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1377</span>–<span class="cit-lpage">1387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DKraus%26rft.auinit1%253DP.%2BR.%26rft.volume%253D48%26rft.issue%253D5%26rft.spage%253D1377%26rft.epage%253D1387%26rft.atitle%253DThe%2BCryptococcus%2Bneoformans%2BMAP%2Bkinase%2BMpk1%2Bregulates%2Bcell%2Bintegrity%2Bin%2Bresponse%2Bto%2Bantifungal%2Bdrugs%2Band%2Bloss%2Bof%2Bcalcineurin%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2958.2003.03508.x%26rft_id%253Dinfo%253Apmid%252F12787363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2958.2003.03508.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12787363&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.44" data-doi="10.1073/pnas.93.23.13217"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leberer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harcus</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broadbent</span>  <span class="cit-name-given-names">I. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dignard</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Signal transduction through homologs of the Ste20p and Ste7p protein kinases can trigger hyphal formation in the pathogenic fungus <em>Candida albicans</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">13217</span>–<span class="cit-lpage">13222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DLeberer%26rft.auinit1%253DE.%26rft.volume%253D93%26rft.issue%253D23%26rft.spage%253D13217%26rft.epage%253D13222%26rft.atitle%253DSignal%2Btransduction%2Bthrough%2Bhomologs%2Bof%2Bthe%2BSte20p%2Band%2BSte7p%2Bprotein%2Bkinases%2Bcan%2Btrigger%2Bhyphal%2Bformation%2Bin%2Bthe%2Bpathogenic%2Bfungus%2BCandida%2Balbicans%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.93.23.13217%26rft_id%253Dinfo%253Apmid%252F8917571%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTMvMjMvMTMyMTciO3M6NDoiYXRvbSI7czoxNjoiL2dnZy84LzEvNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.45" data-doi="10.1128/MMBR.64.4.746-785.2000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lengeler</span>  <span class="cit-name-given-names">K. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’Souza</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harashima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Signal transduction cascades regulating fungal development and virulence.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">746</span>–<span class="cit-lpage">785</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DLengeler%26rft.auinit1%253DK.%2BB.%26rft.volume%253D64%26rft.issue%253D4%26rft.spage%253D746%26rft.epage%253D785%26rft.atitle%253DSignal%2BTransduction%2BCascades%2BRegulating%2BFungal%2BDevelopment%2Band%2BVirulence%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.64.4.746-785.2000%26rft_id%253Dinfo%253Apmid%252F11104818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI2NC80Lzc0NiI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzgvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-web" id="cit-8.1.63.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM.</span> <span class="cit-source">arXiv</span> Available at: <a href="https://arxiv.org/abs/1303.3997">https://arxiv.org/abs/1303.3997</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.47" data-doi="10.1038/nature07743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genomics of domestic and wild yeasts.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">337</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLiti%26rft.auinit1%253DG.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D337%26rft.epage%253D341%26rft.atitle%253DPopulation%2Bgenomics%2Bof%2Bdomestic%2Band%2Bwild%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07743%26rft_id%253Dinfo%253Apmid%252F19212322%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212322&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600041&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Styles</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> S288C has a mutation in FL08 a gene required for filamentous growth.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">967</span>–<span class="cit-lpage">978</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLiu%26rft.auinit1%253DH.%26rft.volume%253D144%26rft.issue%253D3%26rft.spage%253D967%26rft.epage%253D978%26rft.atitle%253DSaccharomyces%2Bcerevisiae%2BS288C%2BHas%2Ba%2BMutation%2Bin%2BFL08%252C%2Ba%2BGene%2BRequired%2Bfor%2BFilamentous%2BGrowth%26rft_id%253Dinfo%253Apmid%252F8913742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ0LzMvOTY3IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Llopis</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernández-Haro</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monteoliva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Querol</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molina</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pathogenic potential of <em>Saccharomyces</em> strains isolated from dietary supplements.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253D21%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.50" data-doi="10.4161/viru.1.3.11053"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The fatal fungal outbreak on Vancouver Island is characterized by enhanced intracellular parasitism driven by mitochondrial regulation.</span> <abbr class="cit-jnl-abbrev">Virulence</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirulence%26rft.volume%253D1%26rft.spage%253D197%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fviru.1.3.11053%26rft_id%253Dinfo%253Apmid%252F21178442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/viru.1.3.11053&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21178442&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292469200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maclean</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metzger</span>  <span class="cit-name-given-names">B. P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.-R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">W.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moyers</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Deciphering the genic basis of yeast fitness variation by simultaneous forward and reverse genetics.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">2486</span>–<span class="cit-lpage">2502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft.volume%253D34%26rft.spage%253D2486%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clemons</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title"><em>Saccharomyces cerevisiae</em> virulence phenotype as determined with CD-1 mice is associated with the ability to grow at 42°C and form pseudohyphae.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">5447</span>–<span class="cit-lpage">5455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DMcCusker%26rft.auinit1%253DJ%2BH%26rft.volume%253D62%26rft.issue%253D12%26rft.spage%253D5447%26rft.epage%253D5455%26rft.atitle%253DSaccharomyces%2Bcerevisiae%2Bvirulence%2Bphenotype%2Bas%2Bdetermined%2Bwith%2BCD-1%2Bmice%2Bis%2Bassociated%2Bwith%2Bthe%2Bability%2Bto%2Bgrow%2Bat%2B42%2Bdegrees%2BC%2Band%2Bform%2Bpseudohyphae.%26rft_id%253Dinfo%253Apmid%252F7960125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjEwOiI2Mi8xMi81NDQ3IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.53" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzgvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.54" data-doi="10.1111/j.1365-294X.2009.04234.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">L. A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Microsatellite analysis of genetic diversity among clinical and nonclinical <em>Saccharomyces cerevisiae</em> isolates suggests heterozygote advantage in clinical environments.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2779</span>–<span class="cit-lpage">2786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMuller%26rft.auinit1%253DL.%2BA.%26rft.volume%253D18%26rft.issue%253D13%26rft.spage%253D2779%26rft.epage%253D2786%26rft.atitle%253DMicrosatellite%2Banalysis%2Bof%2Bgenetic%2Bdiversity%2Bamong%2Bclinical%2Band%2Bnonclinical%2BSaccharomyces%2Bcerevisiae%2Bisolates%2Bsuggests%2Bheterozygote%2Badvantage%2Bin%2Bclinical%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2009.04234.x%26rft_id%253Dinfo%253Apmid%252F19457175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2009.04234.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19457175&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.55" data-doi="10.1111/j.1365-294X.2011.05225.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">L. A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lucas</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Georgianna</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCusker</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association analysis of clinical <em>vs.</em> nonclinical origin provides insights into <em>Saccharomyces cerevisiae</em> pathogenesis.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">4085</span>–<span class="cit-lpage">4097</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DMuller%26rft.auinit1%253DL.%2BA.%26rft.volume%253D20%26rft.issue%253D19%26rft.spage%253D4085%26rft.epage%253D4097%26rft.atitle%253DGenome-wide%2Bassociation%2Banalysis%2Bof%2Bclinical%2Bvs.%2Bnonclinical%2Borigin%2Bprovides%2Binsights%2Binto%2BSaccharomyces%2Bcerevisiae%2Bpathogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2011.05225.x%26rft_id%253Dinfo%253Apmid%252F21880084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2011.05225.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21880084&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.56" data-doi="10.1016/S0141-0229(99)00086-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kavanagh</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Emergence of <em>Saccharomyces cerevisiae</em> as a human pathogen. Implications for biotechnology.</span> <abbr class="cit-jnl-abbrev">Enzyme Microb. Technol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnzyme%2BMicrob.%2BTechnol.%26rft.volume%253D25%26rft.spage%253D551%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0141-0229%252899%252900086-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0141-0229(99)00086-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000082754000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.57" data-doi="10.1128/IAI.73.7.3842-3850.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mylonakis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El Khoury</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Idnurm</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heitman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title"><em>Galleria mellonella</em> as a model system to study <em>Cryptococcus neoformans</em> pathogenesis.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">3842</span>–<span class="cit-lpage">3850</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DMylonakis%26rft.auinit1%253DE.%26rft.volume%253D73%26rft.issue%253D7%26rft.spage%253D3842%26rft.epage%253D3850%26rft.atitle%253DGalleria%2Bmellonella%2Bas%2Ba%2BModel%2BSystem%2BTo%2BStudy%2BCryptococcus%2Bneoformans%2BPathogenesis%26rft_id%253Dinfo%253Adoi%252F10.1128%252FIAI.73.7.3842-3850.2005%26rft_id%253Dinfo%253Apmid%252F15972469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjk6IjczLzcvMzg0MiI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzgvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.58" data-doi="10.1016/j.fgb.2011.08.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Navarro-Velasco</span>  <span class="cit-name-given-names">G. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prados-Rosales</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ortiz-Urquiza</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quesada-Moraga</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Di Pietro</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>Galleria mellonella</em> as model host for the trans-kingdom pathogen <em>Fusarium oxysporum</em>.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">1124</span>–<span class="cit-lpage">1129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DNavarro-Velasco%26rft.auinit1%253DG.%2BY.%26rft.volume%253D48%26rft.issue%253D12%26rft.spage%253D1124%26rft.epage%253D1129%26rft.atitle%253DGalleria%2Bmellonella%2Bas%2Bmodel%2Bhost%2Bfor%2Bthe%2Btrans-kingdom%2Bpathogen%2BFusarium%2Boxysporum.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2011.08.004%26rft_id%253Dinfo%253Apmid%252F21907298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2011.08.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21907298&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neilson</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ivey</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bulmer</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title"><em>Cryptococcus neoformans</em>: pseudohyphal forms surviving culture with <em>Acanthamoeba polyphaga</em>.</span> <abbr class="cit-jnl-abbrev">Infect. Immun.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">262</span>–<span class="cit-lpage">266</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInfection%2Band%2BImmunity%26rft.stitle%253DInfect.%2BImmun.%26rft.aulast%253DNeilson%26rft.auinit1%253DJ%2BB%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D262%26rft.epage%253D266%26rft.atitle%253DCryptococcus%2Bneoformans%253A%2Bpseudohyphal%2Bforms%2Bsurviving%2Bculture%2Bwith%2BAcanthamoeba%2Bpolyphaga.%26rft_id%253Dinfo%253Apmid%252F352931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaWFpIjtzOjU6InJlc2lkIjtzOjg6IjIwLzEvMjYyIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.60" data-doi="10.1016/S0966-842X(00)01752-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Odds</span>  <span class="cit-name-given-names">F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Pathogenic fungi in the 21st century.</span> <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">200</span>–<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DOdds%26rft.auinit1%253DF.%2BC.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253D200%26rft.epage%253D201%26rft.atitle%253DPathogenic%2Bfungi%2Bin%2Bthe%2B21st%2Bcentury.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0966-842X%252800%252901752-2%26rft_id%253Dinfo%253Apmid%252F10785633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0966-842X(00)01752-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10785633&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087036100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.61" data-doi="10.1101/gr.116731.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parts</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cubillos</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warringer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salinas</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Revealing the genetic structure of a trait by sequencing a population under selection.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1131</span>–<span class="cit-lpage">1138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.116731.110%26rft_id%253Dinfo%253Apmid%252F21422276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzcvMTEzMSI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzgvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.62" data-doi="10.1111/j.1469-0691.2008.01978.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lass-Flörl</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Changing epidemiology of systemic fungal infections.</span> <abbr class="cit-jnl-abbrev">Clin. Microbiol. Infect.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DClinical%2Bmicrobiology%2Band%2Binfection%2B%253A%2B%2Bthe%2Bofficial%2Bpublication%2Bof%2Bthe%2BEuropean%2BSociety%2Bof%2BClinical%2BMicrobiology%2Band%2BInfectious%2BDiseases%26rft.stitle%253DClin%2BMicrobiol%2BInfect%26rft.aulast%253DBlot%26rft.auinit1%253DS.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D5%26rft.epage%253D13%26rft.atitle%253DLimiting%2Bthe%2Battributable%2Bmortality%2Bof%2Bnosocomial%2Binfection%2Band%2Bmultidrug%2Bresistance%2Bin%2Bintensive%2Bcare%2Bunits.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-0691.2008.01978.x%26rft_id%253Dinfo%253Apmid%252F17944969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-0691.2008.01978.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17944969&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251562700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.63" data-doi="10.1101/gad.8.24.2974"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Elements of a single map kinase cascade in <em>Saccharomyces cerevisiae</em> mediate two developmental programs in the same cell-type: mating and invasive growth.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2974</span>–<span class="cit-lpage">2985</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DRoberts%26rft.auinit1%253DR%2BL%26rft.volume%253D8%26rft.issue%253D24%26rft.spage%253D2974%26rft.epage%253D2985%26rft.atitle%253DElements%2Bof%2Ba%2Bsingle%2BMAP%2Bkinase%2Bcascade%2Bin%2BSaccharomyces%2Bcerevisiae%2Bmediate%2Btwo%2Bdevelopmental%2Bprograms%2Bin%2Bthe%2Bsame%2Bcell%2Btype%253A%2Bmating%2Band%2Binvasive%2Bgrowth.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.8.24.2974%26rft_id%253Dinfo%253Apmid%252F8001818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiOC8yNC8yOTc0IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.64" data-doi="10.1093/bioinformatics/btp616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp616%26rft_id%253Dinfo%253Apmid%252F19910308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19910308&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273116100025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.65" data-doi="10.1016/S1046-2023(05)80215-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambie</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Spore enrichment.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DRockmill%26rft.auinit1%253DB.%26rft.volume%253D194%26rft.spage%253D146%26rft.epage%253D149%26rft.atitle%253DSpore%2Benrichment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1046-2023%252805%252980215-4%26rft_id%253Dinfo%253Apmid%252F2005784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1046-2023(05)80215-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2005784&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FN84200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.66" data-doi="10.1016/j.tim.2007.02.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Román</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arana</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nombela</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alonso-Monge</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pla</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">MAP kinase pathways as regulators of fungal virulence.</span> <abbr class="cit-jnl-abbrev">Trends Microbiol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bmicrobiology%26rft.stitle%253DTrends%2BMicrobiol%26rft.aulast%253DRoman%26rft.auinit1%253DE.%26rft.volume%253D15%26rft.issue%253D4%26rft.spage%253D181%26rft.epage%253D190%26rft.atitle%253DMAP%2Bkinase%2Bpathways%2Bas%2Bregulators%2Bof%2Bfungal%2Bvirulence.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tim.2007.02.001%26rft_id%253Dinfo%253Apmid%252F17321137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tim.2007.02.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17321137&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246093700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.67" data-doi="10.1038/nature07670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schacherer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapiro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruderfer</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comprehensive polymorphism survey elucidates population structure of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">458</span>: <span class="cit-fpage">342</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSchacherer%26rft.auinit1%253DJ.%26rft.volume%253D458%26rft.issue%253D7236%26rft.spage%253D342%26rft.epage%253D345%26rft.atitle%253DComprehensive%2Bpolymorphism%2Bsurvey%2Belucidates%2Bpopulation%2Bstructure%2Bof%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07670%26rft_id%253Dinfo%253Apmid%252F19212320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212320&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264285600042&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.68" data-doi="10.1534/genetics.115.185165"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sellis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kvitek</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunn</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Heterozygote advantage is a common outcome of adaptation in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">203</span>: <span class="cit-fpage">1401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.185165%26rft_id%253Dinfo%253Apmid%252F27194750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMy8zLzE0MDEiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy84LzEvNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.69" data-doi="10.3109/13693786.2010.523852"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denning</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warn</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Pathogenicity of <em>Aspergillus fumigatus</em> mutants assessed in <em>Galleria mellonella</em> matches that in mice.</span> <abbr class="cit-jnl-abbrev">Med. Mycol.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">S107</span>–<span class="cit-lpage">S113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMed.%2BMycol.%26rft_id%253Dinfo%253Adoi%252F10.3109%252F13693786.2010.523852%26rft_id%253Dinfo%253Apmid%252F20950221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3109/13693786.2010.523852&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20950221&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heisler</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">St Onge</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farias-Hesson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Highly-multiplexed barcode sequencing: an efficient method for parallel analysis of pooled samples.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D38%26rft.spage%253D7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.71" data-doi="10.1038/416326a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spiegelman</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oefner</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Dissecting the architecture of a quantitative trait locus in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">416</span>: <span class="cit-fpage">326</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSteinmetz%26rft.auinit1%253DL.%2BM.%26rft.volume%253D416%26rft.issue%253D6878%26rft.spage%253D326%26rft.epage%253D330%26rft.atitle%253DDissecting%2Bthe%2Barchitecture%2Bof%2Ba%2Bquantitative%2Btrait%2Blocus%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F416326a%26rft_id%253Dinfo%253Apmid%252F11907579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/416326a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11907579&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174482200044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.72" data-doi="10.1101/gr.185538.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strope</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skelly</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kozmin</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahadevan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The 100-genomes strains, an <em>S. cerevisiae</em> resource that illuminates its natural phenotypic and genotypic variation and emergence as an opportunistic pathogen.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">762</span>–<span class="cit-lpage">774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.185538.114%26rft_id%253Dinfo%253Apmid%252F25840857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI1LzUvNzYyIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sunshine</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ong</span>  <span class="cit-name-given-names">G. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liachko</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The fitness consequences of aneuploidy are driven by condition-dependent gene effects.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D13%26rft.spage%253D34%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.74" data-doi="10.1146/annurev.micro.55.1.743"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Burik</span>  <span class="cit-name-given-names">J. A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magee</span>  <span class="cit-name-given-names">P. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Aspects of fungal pathogenesis in humans.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Microbiol.</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmicrobiology%26rft.stitle%253DAnnu%2BRev%2BMicrobiol%26rft.aulast%253Dvan%2BBurik%26rft.auinit1%253DJ.-A.%2BH.%26rft.volume%253D55%26rft.issue%253D1%26rft.spage%253D743%26rft.epage%253D772%26rft.atitle%253DASPECTS%2BOF%2BFUNGAL%2BPATHOGENESIS%2BIN%2BHUMANS%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.micro.55.1.743%26rft_id%253Dinfo%253Apmid%252F11544373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.micro.55.1.743&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11544373&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171732600028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.75" data-doi="10.1534/g3.116.029595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vogan</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khankhet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samarasinghe</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Identification of QTLs associated with virulence related traits and drug resistance in <em>Cryptococcus neoformans</em>.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2745</span>–<span class="cit-lpage">2759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D2745%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.029595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1534/g3.116.029595&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.76" data-doi="10.1093/bioinformatics/btv300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Cabo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ricote</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">GOplot: an R package for visually combining expression data with functional analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">2912</span>–<span class="cit-lpage">2914</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv300%26rft_id%253Dinfo%253Apmid%252F25964631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv300&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25964631&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.77" data-doi="10.1128/EC.1.2.257-272.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lengeler</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardenas</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Mating-type-specific and nonspecific PAK kinases play shared and divergent roles in <em>Cryptococcus neoformans</em>.</span> <abbr class="cit-jnl-abbrev">Eukaryot. Cell</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEukaryotic%2BCell%26rft.stitle%253DEukaryot%2BCell%26rft.aulast%253DWang%26rft.auinit1%253DP.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253D257%26rft.epage%253D272%26rft.atitle%253DMating-Type-Specific%2Band%2BNonspecific%2BPAK%2BKinases%2BPlay%2BShared%2Band%2BDivergent%2BRoles%2Bin%2BCryptococcus%2Bneoformans%26rft_id%253Dinfo%253Adoi%252F10.1128%252FEC.1.2.257-272.2002%26rft_id%253Dinfo%253Apmid%252F12455960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZXVrY2VsbCI7czo1OiJyZXNpZCI7czo3OiIxLzIvMjU3IjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.78" data-doi="10.1111/j.1365-294X.2012.05732.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">W. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">F. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Surprisingly diverged populations of <em>Saccharomyces cerevisiae</em> in natural environments remote from human activity.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">5404</span>–<span class="cit-lpage">5417</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DWang%26rft.auinit1%253DQ.%2BM.%26rft.volume%253D21%26rft.issue%253D22%26rft.spage%253D5404%26rft.epage%253D5417%26rft.atitle%253DSurprisingly%2Bdiverged%2Bpopulations%2Bof%2BSaccharomyces%2Bcerevisiae%2Bin%2Bnatural%2Benvironments%2Bremote%2Bfrom%2Bhuman%2Bactivity.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2012.05732.x%26rft_id%253Dinfo%253Apmid%252F22913817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2012.05732.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22913817&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310583800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.79"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wartenberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linde</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schreiner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horn</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Microevolution of <em>Candida albicans</em> in macrophages restores filamentation in a nonfilamentous mutant.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D10%26rft.spage%253D18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A drug-sensitive genetic network masks fungi from the immune system.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">328</span>–<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D2%26rft.spage%253D328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000202894400009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.81" data-doi="10.1073/pnas.0437995100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kupiec</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magnelli</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abeijon</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A <em>Saccharomyces cerevisiae</em> mutant with increased virulence.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">2766</span>–<span class="cit-lpage">2770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0437995100%26rft_id%253Dinfo%253Apmid%252F12589024%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzUvMjc2NiI7czo0OiJhdG9tIjtzOjE2OiIvZ2dnLzgvMS82My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.63.82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wickham</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source"><em>ggplot2: Elegant Graphics for Data Analysis</em>.</span> <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.83" data-doi="10.1126/science.285.5429.901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoemaker</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Astromoff</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Functional characterization of the <em>S. cerevisiae</em> genome by gene deletion and parallel analysis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">285</span>: <span class="cit-fpage">901</span>–<span class="cit-lpage">906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWinzeler%26rft.auinit1%253DE.%2BA.%26rft.volume%253D285%26rft.issue%253D5429%26rft.spage%253D901%26rft.epage%253D906%26rft.atitle%253DFunctional%2BCharacterization%2Bof%2Bthe%2BS.%2Bcerevisiae%2BGenome%2Bby%2BGene%2BDeletion%2Band%2BParallel%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.285.5429.901%26rft_id%253Dinfo%253Apmid%252F10436161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODUvNTQyOS85MDEiO3M6NDoiYXRvbSI7czoxNjoiL2dnZy84LzEvNjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.84" data-doi="10.1006/fgbi.2000.1237"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">MAP kinases in fungal pathogens.</span> <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">152</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DXu%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D31%26rft.issue%253D3%26rft.spage%253D137%26rft.epage%253D152%26rft.atitle%253DMap%2Bkinases%2Bin%2Bfungal%2Bpathogens.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ffgbi.2000.1237%26rft_id%253Dinfo%253Apmid%252F11273677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/fgbi.2000.1237&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11273677&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167271900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.85" data-doi="10.1016/j.mib.2013.03.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zaragoza</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Titan cells in <em>Cryptococcus neoformans</em>: cells with a giant impact.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Microbiol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BMicrobiol.%26rft.volume%253D16%26rft.spage%253D409%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.mib.2013.03.006%26rft_id%253Dinfo%253Apmid%252F23588027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.mib.2013.03.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23588027&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F63.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.86" data-doi="10.1101/gr.074492.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zerbino</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graphs.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.074492.107%26rft_id%253Dinfo%253Apmid%252F18349386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzUvODIxIjtzOjQ6ImF0b20iO3M6MTY6Ii9nZ2cvOC8xLzYzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.63.87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Whole genome analysis of 132 clinical <em>Saccharomyces cerevisiae</em> strains reveals extensive ploidy variation.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2421</span>–<span class="cit-lpage">2434</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D2421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/1/63.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/53" title="Improving Genomic Prediction in Cassava Field Experiments Using Spatial Analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/79" title="Large-Scale Low-Cost NGS Library Preparation Using a Robust Tn5 Purification and Tagmentation Protocol" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/1"><span>Volume 8 Issue 1, January 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/1.cover-source.jpg?itok=vyoqxep2" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/63.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-x_XoZpN2Vuwo6XDq_eZVQ160-bao9Rv1GBw-vgUzZIM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/1/63" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/1/63" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/1/63" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Screen for Saccharomyces cerevisiae Genes Contributing to Opportunistic Pathogenicity in an Invertebrate Model Host</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487839" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-C1A5spjEhK25xA0NRmvBrFhNCkKC0YwPLGDpALSEsJ4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/63.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/1/63" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-31zSGepO-_sRQvU2ZXeWOcIFUhg7fAmW_4Z4fkrdcIw" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/1/63" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/63.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487839" id="node487839--2" data-pisa="ggg;8/1/63" data-pisa-master="ggg;g3.117.300245" data-apath="/ggg/8/1/63.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Screen for <em>Saccharomyces cerevisiae</em> Genes Contributing to Opportunistic Pathogenicity in an Invertebrate Model Host</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sujal S. Phadke</span>, <span class="highwire-citation-author" data-delta="1">Calum J. Maclean</span>, <span class="highwire-citation-author" data-delta="2">Serena Y. Zhao</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7805-8785" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Emmi A. Mueller</span>, <span class="highwire-citation-author" data-delta="4">Lucas A. Michelotti</span>, <span class="highwire-citation-author" data-delta="5">Kaitlyn L. Norman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-8923-297X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Anuj Kumar</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-1123-5986" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Timothy Y. James</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">63-78;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300245 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487839--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sujal S. Phadke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sujal%2BS.%2BPhadke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Phadke%20SS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASujal%2BS.%2BPhadke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Calum J. Maclean </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Calum%2BJ.%2BMaclean%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maclean%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACalum%2BJ.%2BMaclean%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Serena Y. Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Serena%2BY.%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20SY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASerena%2BY.%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Emmi A. Mueller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emmi%2BA.%2BMueller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mueller%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEmmi%2BA.%2BMueller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7805-8785" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Emmi A. Mueller</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Lucas A. Michelotti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucas%2BA.%2BMichelotti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michelotti%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALucas%2BA.%2BMichelotti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kaitlyn L. Norman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kaitlyn%2BL.%2BNorman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Norman%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaitlyn%2BL.%2BNorman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anuj Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anuj%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnuj%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8923-297X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Anuj Kumar</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Timothy Y. James </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BY.%2BJames%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=James%20TY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimothy%2BY.%2BJames%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1123-5986" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timothy Y. James</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tyjames@umich.edu" class="" data-icon-position="" data-hide-link-title="0">tyjames@umich.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/1/63" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/1/63&amp;title=Genome-Wide%20Screen%20for%20Saccharomyces%20cerevisiae%20Genes%20Contributing%20to%20Opportunistic%20Pathogenicity%20in%20an%20Invertebrate%20Model%20Host" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/1/63&amp;title=Genome-Wide%20Screen%20for%20Saccharomyces%20cerevisiae%20Genes%20Contributing%20to%20Opportunistic%20Pathogenicity%20in%20an%20Invertebrate%20Model%20Host" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/1/63&amp;title=Genome-Wide%20Screen%20for%20Saccharomyces%20cerevisiae%20Genes%20Contributing%20to%20Opportunistic%20Pathogenicity%20in%20an%20Invertebrate%20Model%20Host" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/1/63&amp;text=Genome-Wide%20Screen%20for%20Saccharomyces%20cerevisiae%20Genes%20Contributing%20to%20Opportunistic%20Pathogenicity%20in%20an%20Invertebrate%20Model%20Host" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/1/63&amp;title=Genome-Wide%20Screen%20for%20Saccharomyces%20cerevisiae%20Genes%20Contributing%20to%20Opportunistic%20Pathogenicity%20in%20an%20Invertebrate%20Model%20Host" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/1/63&amp;t=Genome-Wide%20Screen%20for%20Saccharomyces%20cerevisiae%20Genes%20Contributing%20to%20Opportunistic%20Pathogenicity%20in%20an%20Invertebrate%20Model%20Host" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/1/63&amp;title=Genome-Wide%20Screen%20for%20Saccharomyces%20cerevisiae%20Genes%20Contributing%20to%20Opportunistic%20Pathogenicity%20in%20an%20Invertebrate%20Model%20Host" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/1/63&amp;title=Genome-Wide%20Screen%20for%20Saccharomyces%20cerevisiae%20Genes%20Contributing%20to%20Opportunistic%20Pathogenicity%20in%20an%20Invertebrate%20Model%20Host" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487839/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487839" id="node487839--3" data-pisa="ggg;8/1/63" data-pisa-master="ggg;g3.117.300245" data-apath="/ggg/8/1/63.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Screen for <em>Saccharomyces cerevisiae</em> Genes Contributing to Opportunistic Pathogenicity in an Invertebrate Model Host</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Sujal S. Phadke</span>, <span class="highwire-citation-author" data-delta="1">Calum J. Maclean</span>, <span class="highwire-citation-author" data-delta="2">Serena Y. Zhao</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7805-8785" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Emmi A. Mueller</span>, <span class="highwire-citation-author" data-delta="4">Lucas A. Michelotti</span>, <span class="highwire-citation-author" data-delta="5">Kaitlyn L. Norman</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-8923-297X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Anuj Kumar</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-1123-5986" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Timothy Y. James</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">63-78;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300245 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487839--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Sujal S. Phadke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sujal%2BS.%2BPhadke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Phadke%20SS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASujal%2BS.%2BPhadke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Calum J. Maclean </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Calum%2BJ.%2BMaclean%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maclean%20CJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACalum%2BJ.%2BMaclean%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Serena Y. Zhao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Serena%2BY.%2BZhao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhao%20SY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASerena%2BY.%2BZhao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Emmi A. Mueller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Emmi%2BA.%2BMueller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mueller%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AEmmi%2BA.%2BMueller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7805-8785" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Emmi A. Mueller</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Lucas A. Michelotti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucas%2BA.%2BMichelotti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Michelotti%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALucas%2BA.%2BMichelotti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Kaitlyn L. Norman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kaitlyn%2BL.%2BNorman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Norman%20KL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKaitlyn%2BL.%2BNorman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Anuj Kumar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Molecular, Cellular, and Developmental Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Anuj%2BKumar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kumar%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAnuj%2BKumar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-8923-297X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Anuj Kumar</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Timothy Y. James </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, Michigan 48109</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BY.%2BJames%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=James%20TY&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimothy%2BY.%2BJames%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1123-5986" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timothy Y. James</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tyjames@umich.edu" class="" data-icon-position="" data-hide-link-title="0">tyjames@umich.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=63&amp;rft.epage=78&amp;rft.atitle=Genome-Wide%20Screen%20for%20Saccharomyces%20cerevisiae%20Genes%20Contributing%20to%20Opportunistic%20Pathogenicity%20in%20an%20Invertebrate%20Model%20Host&amp;rft.volume=8&amp;rft.issue=1&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Phadke%2C+Sujal+S.&amp;rft.au=Maclean%2C+Calum+J.&amp;rft.au=Zhao%2C+Serena+Y.&amp;rft.au=Mueller%2C+Emmi+A.&amp;rft.au=Michelotti%2C+Lucas+A.&amp;rft.au=Norman%2C+Kaitlyn+L.&amp;rft.au=Kumar%2C+Anuj&amp;rft.au=James%2C+Timothy+Y."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487839/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487839/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487839/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487839/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487839/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487839/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487839/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487839/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487839/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487839/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487839/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487839/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/1/63" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-28" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/1/63.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/1/63.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/1/63.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/1/63.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
