<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/1/131" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/1/131.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/1/131.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/1/131.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/8/1/131/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/1/131.atom" />
<meta name="HW.pisa" content="ggg;8/1/131" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Prediction of Multiple-Trait and Multiple-Environment Genomic Data Using Recommender Systems" />
<meta name="DC.Identifier" content="10.1534/g3.117.300309" />
<meta name="DC.Date" content="2018-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Montesinos-Lopez et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In genomic-enabled prediction, the task of improving the accuracy of the prediction of lines in environments is difficult because the available information is generally sparse and usually has low correlations between traits. In current genomic selection, although researchers have a large amount of information and appropriate statistical models to process it, there is still limited computing efficiency to do so. Although some statistical models are usually mathematically elegant, many of them are also computationally inefficient, and they are impractical for many traits, lines, environments, and years because they need to sample from huge normal multivariate distributions. For these reasons, this study explores two recommender systems: item-based collaborative filtering (IBCF) and the matrix factorization algorithm (MF) in the context of multiple traits and multiple environments. The IBCF and MF methods were compared with two conventional methods on simulated and real data. Results of the simulated and real data sets show that the IBCF technique was slightly better in terms of prediction accuracy than the two conventional methods and the MF method when the correlation was moderately high. The IBCF technique is very attractive because it produces good predictions when there is high correlation between items (environment–trait combinations) and its implementation is computationally feasible, which can be useful for plant breeders who deal with very large data sets." />
<meta name="DC.Contributor" content="Osval A. Montesinos-López" />
<meta name="DC.Contributor" content="Abelardo Montesinos-López" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="José C. Montesinos-López" />
<meta name="DC.Contributor" content="David Mota-Sanchez" />
<meta name="DC.Contributor" content="Fermín Estrada-González" />
<meta name="DC.Contributor" content="Jussi Gillberg" />
<meta name="DC.Contributor" content="Ravi Singh" />
<meta name="DC.Contributor" content="Suchismita Mondal" />
<meta name="DC.Contributor" content="Philomin Juliana" />
<meta name="article:published_time" content="2018-01-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Prediction of Multiple-Trait and Multiple-Environment Genomic Data Using Recommender Systems" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In genomic-enabled prediction, the task of improving the accuracy of the prediction of lines in environments is difficult because the available information is generally sparse and usually has low correlations between traits. In current genomic selection, although researchers have a large amount of information and appropriate statistical models to process it, there is still limited computing efficiency to do so. Although some statistical models are usually mathematically elegant, many of them are also computationally inefficient, and they are impractical for many traits, lines, environments, and years because they need to sample from huge normal multivariate distributions. For these reasons, this study explores two recommender systems: item-based collaborative filtering (IBCF) and the matrix factorization algorithm (MF) in the context of multiple traits and multiple environments. The IBCF and MF methods were compared with two conventional methods on simulated and real data. Results of the simulated and real data sets show that the IBCF technique was slightly better in terms of prediction accuracy than the two conventional methods and the MF method when the correlation was moderately high. The IBCF technique is very attractive because it produces good predictions when there is high correlation between items (environment–trait combinations) and its implementation is computationally feasible, which can be useful for plant breeders who deal with very large data sets.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="ggg;8/1/131" />
<meta name="citation_id" content="8/1/131" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/1/131" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/1/131.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/1/131.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/1/131.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300309" />
<meta name="citation_pmid" content="29097376" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="131" />
<meta name="citation_lastpage" content="147" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Osval A. Montesinos-López" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, 28040 Colima, México" />
<meta name="citation_author_email" content="oamontes1@ucol.mx" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author" content="Abelardo Montesinos-López" />
<meta name="citation_author_institution" content="Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430 Jalisco, México" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600 México City, México" />
<meta name="citation_author_email" content="oamontes1@ucol.mx" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9429-5855" />
<meta name="citation_author" content="José C. Montesinos-López" />
<meta name="citation_author_institution" content="Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), 36240 Guanajuato, México" />
<meta name="citation_author" content="David Mota-Sanchez" />
<meta name="citation_author_institution" content="Department of Entomology, Michigan State University, East Lancing, Michigan 48824" />
<meta name="citation_author" content="Fermín Estrada-González" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, 28040 Colima, México" />
<meta name="citation_author" content="Jussi Gillberg" />
<meta name="citation_author_institution" content="Department of Computer Science, Aalto University, FI-00076, Finland" />
<meta name="citation_author" content="Ravi Singh" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600 México City, México" />
<meta name="citation_author" content="Suchismita Mondal" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600 México City, México" />
<meta name="citation_author" content="Philomin Juliana" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600 México City, México" />
<meta name="citation_reference" content="citation_author=L. Coba;citation_author=M. Zanker;citation_author=P. Symeonidis;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=G3;citation_author=J. Crossa;citation_author=Y. Beyene;citation_author=S. Kassa;citation_author=P. Pérez;citation_author=J. M. Hickey;citation_title=Genomic prediction in maize breeding populations with genotyping-by-sequencing.;citation_pages=1903-1926;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. E. Goddard;citation_author=B. J. Hayes;citation_title=Genomic selection.;citation_pages=323-330;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076469;citation_doi=10.1111/j.1439-0388.2007.00702.x" />
<meta name="citation_reference" content="citation_title=IEEE Internet Comput.;citation_author=G. Linden;citation_author=B. Smith;citation_author=J. York;citation_title=Amazon.com recommendations: item-to-item collaborative filtering.;citation_pages=76-80;citation_volume=7;citation_year=2003;citation_doi=10.1109/MIC.2003.1167344" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=O. A. Montesinos-López;citation_author=A. Montesinos-López;citation_author=J. Crossa;citation_author=F. H. Toledo;citation_author=O. Pérez-Hernández;citation_title=A genomic Bayesian multi-trait and multi-environment model.;citation_pages=2725-2744;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=O. A. Montesinos-López;citation_author=A. Montesinos-López;citation_author=J. Crossa;citation_author=F. H. Toledo;citation_author=J. C. Montesinos-López;citation_title=A Bayesian Poisson-lognormal model for count data for multiple-trait multiple-environment genomic-enabled prediction.;citation_volume=7;citation_year=2017;citation_pmid=28364037;citation_doi=10.1534/g3.117.039974" />
<meta name="citation_reference" content="citation_journal_title=Recommender Systems Handbook;citation_title=Introduction to recommender systems handbook;citation_author=F. Ricci;citation_author=L. Rokach;citation_author=B. Shapira;citation_pages=1-35;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. Rutkoski;citation_author=J. Poland;citation_author=S. Mondal;citation_author=E. Autrique;citation_author=L. G. Pérez;citation_title=Canopy temperature and vegetation indices from high-throughput phenotyping improve accuracy of pedigree and genomic selection for grain yield in wheat.;citation_pages=2799-2808;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_author=B. Sarwar;citation_author=G. Karypis;citation_author=J. Konstan;citation_author=J. Riedl;citation_pages=285-295;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Prediction of Multiple-Trait and Multiple-Environment Genomic Data Using Recommender Systems" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="twitter:description" content="In genomic-enabled prediction, the task of improving the accuracy of the prediction of lines in environments is difficult because the available information is generally sparse and usually has low correlations between traits. In current genomic selection, although researchers have a large amount of information and appropriate statistical models to process it, there is still limited computing efficiency to do so. Although some statistical models are usually mathematically elegant, many of them are also computationally inefficient, and they are impractical for many traits, lines, environments, and years because they need to sample from huge normal multivariate distributions. For these reasons, this study explores two recommender systems: item-based collaborative filtering (IBCF) and the matrix factorization algorithm (MF) in the context of multiple traits and multiple environments. The IBCF and MF methods were compared with two conventional methods on simulated and real data. Results of the simulated and real data sets show that the IBCF technique was slightly better in terms of prediction accuracy than the two conventional methods and the MF method when the correlation was moderately high. The IBCF technique is very attractive because it produces good predictions when there is high correlation between items (environment–trait combinations) and its implementation is computationally feasible, which can be useful for plant breeders who deal with very large data sets." />
<meta name="og-title" property="og:title" content="Prediction of Multiple-Trait and Multiple-Environment Genomic Data Using Recommender Systems" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/1/131" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In genomic-enabled prediction, the task of improving the accuracy of the prediction of lines in environments is difficult because the available information is generally sparse and usually has low correlations between traits. In current genomic selection, although researchers have a large amount of information and appropriate statistical models to process it, there is still limited computing efficiency to do so. Although some statistical models are usually mathematically elegant, many of them are also computationally inefficient, and they are impractical for many traits, lines, environments, and years because they need to sample from huge normal multivariate distributions. For these reasons, this study explores two recommender systems: item-based collaborative filtering (IBCF) and the matrix factorization algorithm (MF) in the context of multiple traits and multiple environments. The IBCF and MF methods were compared with two conventional methods on simulated and real data. Results of the simulated and real data sets show that the IBCF technique was slightly better in terms of prediction accuracy than the two conventional methods and the MF method when the correlation was moderately high. The IBCF technique is very attractive because it produces good predictions when there is high correlation between items (environment–trait combinations) and its implementation is computationally feasible, which can be useful for plant breeders who deal with very large data sets." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
    <title>Prediction of Multiple-Trait and Multiple-Environment Genomic Data Using Recommender Systems | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__HLkFhTtFYKr9UM35ZkeLEJf0AA-RJsk2jDIDlTr7P6E__db-Yi0lrH1NHADEOMB-20Dvr8Qs6F9pwOIMQF9BHr1w__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__EjT-6YMPvgLl8LSJ78FRw-tAFtIqvvC0dKPDt3xx2Ec__naWIXWhSBwaYF2DqFTeJJxk8gTTEkdfCwGZ2o8_HgIw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__WoUUhFHu9IZRJChlxfU_hFiUTBhKim997BgtM6rQv_0__WtsG9gDKkyL0Zcl619hpmtR4jbV5vkG5UnoAmrjf3jM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"FObx1I0iABSG66yAMJvDefrbDEp5Yqm97LwYXhrlySI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"misc\/ui\/jquery.ui.accordion.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487825","apath":"\/ggg\/8\/1\/131.atom","pisa":"ggg;8\/1\/131","ac":{"\/ggg\/8\/1\/131.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/1\/131.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/1\/131"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/1\/131"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/1\/131":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wcvRF\\\\\\\/w+ofk2IlwQF4t7pdiQTQvf7iUTJUJ8v4eqqolN9ft1XCgdxrMKOTGBtvOAb6aNureJw\\\\\\\/hlkQ5JjAW4pwOie1OFh2a2r8bXh+7VJHvSDdDwefKqI66j98M1z7Rp1LbXiilUaWByfv2OMmv3c543MzE+6\\\\\\\/woLHKQVFSJmNPiySktew9txipSDbxpFOdVbzPxFv7bDZzPstEmp22FIwFzVbcWzQO7as4muw2RDlXr+8vRuIYw8+1g0k+AcR01W92c7HGBNKNgYqpi+Q\\\\\\\/B4KKYCYB8FX9b\\\\\\\/Fl3EiCOjAWM6zCgS\\\\\\\/4VzEPg\\\\\\\/JxifDYJZ3iTq3QdFJjAP5jbBNd5D7RAuzbmagMXefZmHgG1o1gEnbVIA84U+4pIKWKGybBjbRVUyXFYaqQE9MmEgU1MmzFnh7nQraeMX+B\\\\\\\/k+rbgpqsC79XQuQigSwd11L5yIbAXqJAIugRH6cxeONWz5wFxe3YQLxGS8nlhxjHkSLn1uI5dWfd0ZplNLZe\\\\\\\/1woiO8KkCp19nRxXCh0eVGB1a2dMQvLkPkZvFocqxptem1eJTFtq+z3mM0Ec6QiF\\\\\\\/PVSYFqcPwf5I1f\\\\\\\/pBmCvAHQsEKyrD0yp\\\\\\\/1huTWNFEqDbjL\\\\\\\/voPgHlEge\\\\\\\/XB\\\\\\\/eebq86uAjHRDO31cXSjtwK9VtKevaRcml9iYlQLDgUWEVcn0uTbCnjgYKrdgdEwCSsY3OCw\\\\\\\/l6ivW+YshUESkozTn6BwoymGA5kCmta1O21whn7yKj5XzlH9r4E6WuPQSBr78zXOWsCD3r2las5qavmULyakqdCQyfpdYZX0WMwHGIANKP0PmQUII+\\\\\\\/9nl9TOgrY+NFsXbwksluuT8Hrm3ksyNPZlNF7g1+MYyJA4lCpYfbx6tAZ7C2hucabSaxMj8QbLWTZsiX7hqOqwYNT1T0CBuVdkrWpLg3KBhTotvbr5dzTd0SBmQVR935XXPWPGSaWKES6uaefZciPG39SDzCD197UYBExhNWHBMxblcAa5iLCDHc9RqN8wtreEjACL4x7BsdDd78kktlQV3T1Ypf+MPfMwoLb4cKcOiVEXdm70itaFO+Zlqs+QGilIClJ6JWpBwG77UqldBsUIp36UakGmq2Qlwx39aybULiRMk6KzF83ACkmL0Irf1dNfSknRQH+9UjkT0+0WCXh6adXQPSPbPyfWglBdfHmzrMlv+ohUGb1PBm5s4TgFy\\\\\\\/iq7BynoRyRKzi9Z94ypTVyBBaiyzApA3vOSq1lMPW7Vj06CYl8f4g5I8P+xvTUlis+46Ze0kJ5hMTFc\\\u0022,\\\u0022iv\\\u0022:\\\u0022foxBf1QIbjPncH4F5I5YXA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229bc8ca2e69820647ac07c876f177716cfae4b3dc075c271d722e98a583cbd7de\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022xuOF5GEqZ8aqnon8f2VGFLGWrDrc1YLFteuR9VhjMLaf3qmPA7nuVCkRrYZNc3\\\\\\\/ItCWkmG2DvfrOgVP3E\\\\\\\/Jsa59jqNaydPevQiJI3VMaEISbdclGgvV7pEuPwCcov5KHKU2Sfp0kgAAdXHHqTGldka3chPQcBiwSoLf4dLlnrSrHqg1dDYyKm4VMl2a32KMs+RwTfjrErgAilyarDb6UOc59ao3nh\\\\\\\/h6KtxZWGOdRhwD+PVPr2GA8CZscdFhP58QXL8nCPm1YQ0ZbIxvWYC0kV3QDRVYeB+bjAH8BKzEwQS\\\\\\\/VmVaDAObge4RNfV4gTk7Wu7NW+wa8OjNINFOmszAo3NBvsubiZeSYmehrgWK21r3\\\\\\\/OI01nqrIXg7OXESuuhfdq+NqPd0Dx7hX5DFt22fJnFFX5749JDKYe3NHL0aNKJqHSOlwpQJSs6lJmK5HYB61Bojx\\\\\\\/JJDB2YgVWA7TUjaZq86yamqSUUlLoPHv\\\\\\\/U2wi88AkmQK9Nnf0OQB8m91XCtih\\\\\\\/oIoCaw4r0gFxwKlRt0p7rwuD11w6y+Jreh1qyMjRnfOtdLgrA68PZH\\\\\\\/mx1y2eltR9ufPYT\\\\\\\/ckO8f4\\\\\\\/wJn4AoqC6p4Hx0r7WAA3UGihr7rWRcrph6062E6ehf2RKa21TphPHkKiFUCzvQp1kTDH6La1h5o0jQpC3OsIloi2\\\\\\\/uLSFf+xVH27fh29GQ8reLXZFHfgjBuoSWJFCDcvigZIIfQUT8J7WniladPiL0PiMpof83Wu6CLWOjjWwVJdwKuEoBOItFH7xAojJ725yUvdFhGlJuLOYsNpkt+kBFR9vQlPwO83FpibKtUGrC46JCwpIr3wd6AcKI1V4PjptyPHLQ1iJilKxurCl8KdP7mux4FSnXCYzxKLnpt2RzCWkRi7GrHk5gLXY9CqZ+47FFjFUzFWcCKwttPYluHf2du2fqIZSQ9qUpJZmnf4n1tkpw+lzMfguxt0+\\\\\\\/e1jKy5auA5Z5fSJN0dN3b38+NMuWi3Na+gtX3cDcv14x7ivXSyGtaYfsigwYkkZOl8puUYdCrHuoFxNNC2EgB8Z7y8SqWvlrpyDq9I6B8nrb5elXHsRtXaLRSULiReniSTf1HqZMoG7LdtuXuC\\\\\\\/ST0aVjXXVdypwR0uGKrt2eYJmB6zLsYFDFVLKihFC2wGD7giysyuJgcxm6ujukPuWuuJp\\\\\\\/AVptoKHcIzK0VZQarjzVe+32e\\\\\\\/yGwscPd7m2FITdjMH6ngwZyURGttqZDIsk7iCn1DVrx8cFrMNrJ4mgvEpq\\\\\\\/C\\\\\\\/vdFk\\\\\\\/dF5MQWZIXl2YttVsDsxmK\\\\\\\/6VgQgzXIyTSi072APXxLyGsHO+eNLPtD2kaCa+JAI+u8hLKqulBZNUxuESVDK7Jf6iwS5gAPjyI2c8SeJTpRlJVK13XelI+LGiGMw0rOTRLb0oMpl79\\\\\\\/gpwmTXX5YykzrUvFr7XsOg49Y1mn4+bww3jCP5l\\\\\\\/FtAcXNjo\\\\\\\/SoPzWqvTJvxrHc8MzbDZNZnpIfm0CWTUCOkdRT6TrWJACsZU10cajOd1s46O7GgIXFEJ99LCrFkMHEeselVISZTRmNfk1giY5VCoCkyu1dyxfYYBO+1nqKHDO3G6pJ5aB6sr5p86e2jVKMqSLSyQBAlQwHBesM4wYyAouMuqmG1uE+PJvXg7FoZJFvI+RZ5YZoE5kjLWi7kDr7IXkcfGtjifBJX0dObqRnxUznTdgwaK9Hjx2MZ6sf4mPK2xoc2Ub7SA+NBLkd108k8fl0f3j8lzfx50KGt5JycKb5M37BizQSLXh5QkmLSVf5lgQdazCxv9xRnZ3CD8ZWj08BZB0Sengrh252nQBSo\\\\\\\/3sXeNhR9UBvIUEGdGIYj7MaqExQHZh3wJkAOJA4kzuaQ3vr72nNG+ZZ4tf3vvpG2V2vzYuJTeV+IiZFObeSzU43Mq0pF5ZkjQVA5TY4ldujxBzWP9YzzVO\\\\\\\/JA6KxSOLBN6JY\\\\\\\/qdZOtkN\\\\\\\/ZuTIg33pLxv+BIke20aQ+FmzzJxM16h+MH0k+LSYA6gsnqI\\\\\\\/xxbFegbPxcoCntKMgWLaHfzcn+gUEpFOX66x9hzsuf9K71Cyuk6rf3sif00wLy7I2H4NkHLmx43TkFR6Yw+DRL\\\\\\\/sJo4JWMyteT\\\\\\\/Mf+H\\\\\\\/UfKa567Oi8zt9+r5DtNEc1pZZIyOxMuwPyLcUhbeIhbwQhAb\\\\\\\/orHAqcJHgjqP5jz8SaRqhjwQEYjTnpqfwYA\\\\\\\/mY80wHQJEQitV+CuGtin6MNfcLTMobKiQSpW+yiTGa7SG9lDgncHWWUdkPiCACXzsSj33d6fhAoRp07CN4dv99wL5yloYQoavcKNp2ubQqfC0R6ggitGOq2coPx+ptXr+HVWidt6G1uRKNpyTBm9NkyUmB8hF6uNPi4g6SzaFRu5lQKc1x4hb8Bpwj4dHinbeHhkJcjvgPJA43XmDQNmSbZYUypfs42GX7hjKG6WY6QG7zTlhKxZ4NXbdZguRjKIQn6g3toSk0Ku7xsG4TBf1nHNexf5rCrRHNERKPgxrGmjHs11kKILsr8IUXfBJxPEYnV24hDSKEZFr4hgpWnQN\\\\\\\/YLx3D+SYM6vQtmodvpOVDDtO6cqdsrMqZhGp1VhjIpgHCyzlgC7WRrPapmT45N0zqkRtrIkNGuj0JizJzcjFJ3rH6pF3J6r5kJtMHaqJ6UZE0UTrtKvSDsNvAlTlWmQ6zeW6uJ24o4k52+6da39zfZVdjOEYQPS1BUa8onDdJgnAT7Ert+WclaO+A6XkzRASysP3VUtDQQogZGlIQb49PsxlDQsWEmZVli+8ZzzOK3AwImTLJ59m5DR5EvyugZkjPIzbgdeMR1ZVfI9Jp7M3cQLBo3LIm\\\\\\\/fmhNIw0\\\\\\\/fHEjA035vXag0VSNQCrgNDr6JcNNyoz0Mm+j9Qb63PK8Y8FmyxU0+jfRtielcirDCO+Ja7aOYVY1wmJnlsAjdTc7idcgylW7DTj0CLkuCtRaQfOTTMcaDZ84NX8EHNc1pSUeMhMGGT96bzv\\\\\\\/szIsAdV9\\\\\\\/ntMEdcWDEyNNdrG3MtidfACh086dDEzVCbKvMynRWhzCLaKUwX25obzaHCptPVJiPLTaE5AaJQOS7TpsQI4TpAI9p37nLO\\\\\\\/lUED5yhfCymIu6Yn5GtrOpCywXbNX5AG9fX2mfVGdnS79Gvhz9e9IHykrB5brGiS5mvQ8zRdcEsyebPUWNYUlUm5HK4d4qkOemlKdBzG\\\\\\\/8jMp0qB44q3YwYoJI55OtbXrQ4FUwZY7OWgh7sB6s4AQ6\\\\\\\/vPXN2y5XqIzBDgirVfDQ8NDK0Rhr+usaAb6\\\\\\\/2A+gc4+YnLE+1N8+o+DFqeR5Igk6\\\\\\\/6Bw3X5InaO7o4VE1vLUpgybTGM0\\\\\\\/dspu\\\\\\\/bLvgQjBfs1ZlnMWBC7sfQQXRRLNHxL2h4+BWng6cn17YO89aMsiadzhem5YWtFLd8Sdh\\\\\\\/3vyx8CwRIbV19UD\\\\\\\/gaO3eXtLRPoL\\\\\\\/m3YAzz0MbwbyMvIATDzEHdv9lKxbtJmfZbFBo+9Zz8HHperXE2cjjOEgA29ow\\\\\\\/xMbuEG22X+5Zz8Jbs9uELwN0ylfRyW\\\\\\\/C2HAa\\\\\\\/pNLCLGIpo7aELmx4HlWosMvVP\\\\\\\/jb+1srVXxdkPCtwuhzGZus6mI0pKu4oLeknB6NkiDLcEBzyVsTfsY4U49tZtjvd5dcU7lZOOT+4TKmMB7jb62q\\\\\\\/2aJXL2u2gX+Vr+OkLHnVYGLbl7p426gfLRaXeHdhFKhk2jYg\\\\\\\/d8kiBQqvlXpSXwqiAaAo6M4eINZgThktfbGv\\\\\\\/XFaxkLKDTcWzQHmZsZMMyNTboNZJpP+IeGBXRJTcyVP7UgNh3k1CN0aJBGfPb1T+V2+K329LMV42UymzasFp\\\\\\\/nLFi9NHD5BtP19ZNsK6+gbUTJCzc0PZ4\\\\\\\/uHYRgieC5YVcxIpARqyaJ6Y5gWl4nfG7su73rVcy3UGSE72j\\\\\\\/RgDYro26hjXdUQNOJy2EwFekHLT\\\\\\\/KMelwJd9VDUXxIggioCtK8gr1Er4dHEY0aeG9NFWncJ0cL0LlduKtswYBx5wIY0w1nn5tAgbWxtbSr7lvaYqFqPSpp05VpSsLnb93fBjKktLId9VgRsZ7XTnr4ccO4iUcOit+DFdAIDw8YtUHH5dFZOiiCAlZZGBSMJvEQ0YOHDxeIdCwV2AZIffXJof1eFc5dt8IRDUTktL\\\\\\\/ZjUp4IYyquEFUO+FqyERKeDbj8BdD\\\\\\\/KSQQeGv\\\\\\\/QXIglIwXzKXYEGm314ImPBQWq7OfTuGXa3d+1FWgJTmpKl00pru0rReu6uinEEbH2tCBcQua1ibPSc4TkKNZXsLADUDA8YZR8mOrW3DOFlWxfHJQ5JSUPpJOE4ByJPCrGsRC4000WyktHgWhSMyjcx5nXnCKwJZQzXNJ396YgDdMCuZc\\\\\\\/8vf1aQ5axE9pGnInWrt0LNlLbeIUzgbn3Kjt54JMeqbHiLHszmpA7qtedU4qbOkg0f3o7LUkE1hxFyZLPiXxVOarW\\\\\\\/2TmFR6WC0YJm\\\\\\\/dVbszg8IewypqSKUxi9NI8CdJ3azGWvnz1OQBGkwpe8siLxroIV8l\\\\\\\/xdc9\\\\\\\/pGhjJQde1wfsS0e5PnM9+2IfmzozZQ+z7XCNcLNXVy2v+RsmwNh8Z7CED3Q5ytPZPTFiSNq9ndQN4xoS9Vx+7M2tALOXX\\\\\\\/ieeh2ISQh0Vnnrh0kV4XY2MyvOj\\\\\\\/fNp5wR2QT6OQH8TV8uLn0\\\\\\\/0xgx6UpLF0MS4uc2fUiOpHpP91acc9BRWGBlJ4BTpa6hFFTIJkxOp+czMv6e09vyKVH9nuUDdXBdoktYbxpvlF7qrmEtHyUI53TXKyBSVMrvpuqkoYoZLfW\\\\\\\/3RATWTm0EyhPI70Q3j1\\\\\\\/0reLvnm9wFpv1sXAkwd+\\\\\\\/Mv5wjoh0cHhnj89+kGKY205XIQpI67L+wPnRc5eBtgZum38\\\\\\\/DY3amPrCcMgDrzkPamnh3YamOu263TFyGFVpoS4QoAaZSY\\\\\\\/5VCx97K0VxapB+1UiSEnBmpLRNDf7VnYteF2mzJr6GwgF\\\\\\\/BsmNrdgCq2XQL8JMgVLqd4Msv\\\\\\\/jDKpYn+y8Uwo8dHfQCX924Bf2rPavRc2X4afBIeDk+O577QDWUkUtgIanuY8I3lzF8Zp1MThU4OY4vKDAU\\\\\\\/WiufWCP3U4DIJ5LDpzIGDpEnzYbwvGo9Osg3jCt6v+FZ+Nx+VfYzhJIu4lFCpBaTwyAfernCprTxC0CG3gd8heNhBFd1i+G1nFx6TexoNysuVYUT6HN1BOutw8Lzx8K9D4\\\\\\\/XIvHdS8foexPLmU0POAfdCCSZQTiaE03nsxzqXboajCUDjWfgopK9XEC2reiRhm2H2bTQkZCbKTpKgKBL64B\\\\\\\/T2OBHvHJfn\\\\\\\/4Gf7fSRnChy8isS0aDeqYplMOnEy\\\\\\\/8JIdtAaQ\\\\\\\/GsH+C+UFKlgjqYYxHjSz2aa6Is43U9pTNu49zxwfmRFpNS2\\\\\\\/FNtUIQWQy03P0U2NumC4dXb6pXQm3qtoBKmeG9E4KxZLYOGutwRYcWlGhWh8GzcEHkvf1x1czPQKfdtl8nShfazMQiWEVqi1iL4\\\\\\\/TAJZsQQku9JAZYdeJ91LzPFzHb7GziAhAlrXukF\\\\\\\/gIa8zfaDt1UhdEE0PLlaP\\\\\\\/\\\\\\\/ADsv31z8e\\\\\\\/2\\\\\\\/anKPV7n874RLegRPAuVWRJcmAAAItg6zHCqbAA9jF2\\\\\\\/VTCLDCwBmLK3J4CFPlcVloo+5PxtJ6UpAfC9JA19BxDYliHqdG1HTj9KAotmRPpTFL5PEqrOmR01SnMBJp8npLJ8BPJ\\\\\\\/6NT66dM+r9ka6ijuf8H4R5CEb7nMEJQk5QU5U6UQ+vLhLlMuePbjZunrvCunQAJuZqHhwACaQZRSDnIvIai1Paa5Eu0igdQrMH7HQAZBX4HzQeTRho2sNSninACGRPfw7sPHNIOYPNRvNiQ0DRb+oShRg3EWdARBdttEUrRcw8VMTVPMS0rIuPh+o3ppxfUpt\\\\\\\/robLfKgMuwktQlpyNfaJG\\\\\\\/hHqv360dATRzZwOZQTf\\\\\\\/aqhWjXe55kx9OtLPbrnWSnfIhNw+IHNEsr0kuUcOHq\\\\\\\/IgiiNOf03JI2hirovbZOqA8FbCtmNtwzG+8IgAWUGYFJ8iUEdaMz\\\\\\\/4rInhTezStu6IbsSQsd\\\\\\\/RTGWwMXjm9xRntMcvG2Mu2ckGvYioXT9dykvSz+54LEkMcPQ7eouZsxNyl\\\\\\\/mUbqD+EXb0mnWAbiMcE0BHifqFiW+dIhCgTz5Wu8y3p7RFCudutLtInS3p7Ffm8hPn6TZgHCsPBrQVOgrD813zyP8tqcM7PzyL0Ytjt0vc0Z+Ap7sjwKpycUVmFmgvA2rg+gww8rF4dWY0xlyQiaAMC2hjKzF5QiZ85+p1YtrX5bcisoW+\\\\\\\/3nNAw\\\\\\\/8z47bfVa8laugVJ2qhQVFLtcLcesb8ZNZYf96lVzgX8OXK0t9zuavxDfymgklU27uvbg66eVzsXXW3cecmCNCzG+dx3YY450TqhkiPSB9j3vR8nIGLl03ghsPQPB0HDcZqGqvKgGYBPA6YBYNlQwFx9vbvd1LgI8wj4M7aiN3pUmqEaRI9IY6XnG\\\\\\\/tY84H14sQYSntxOWn24I+oLHqZTVB3UiZV8qJdpojesEiEQ8E0vLdLdS+rs2sP69+FrVAzDm9IQ6kAqfFxIvfcvEPtYuWVyOw1Lm5IUnVDyimNyL6\\\\\\\/yLKsp3zMNNm9MKu1+i0MtoznguAvg+s6s13VxVWYGIk0L9CMHIv8sJ7Owjq++mgQqHVqRgZRYvACKUJaSad1PZR\\\\\\\/N1P16SKqVGxOVIa47GgSUwXuJM2hCWxPIte+j2hb1CY\\\\\\\/Tr2Ln6kET1\\\\\\\/M+Oc6nL9XCMpyPLDEjO2HvHvbb4moEV7Fgt72\\\\\\\/q0GsKIRJu3j\\\\\\\/rtN69rlwEAzrS7h13AqfXqXnJ4IGoXlXKA01DZNJ6vltdwBtnZmpUj5KkF06UcZgqKWT66w2qjc6lugR1gJO5O75fENczYHNHfnnCwERPB7Jf\\\\\\\/hvK3sx9Ihv\\\\\\\/9bgl7QioW3POi3ZZ3BqyXlOJtQ1YeXaK\\\\\\\/mUz0bTNWc8EyowAMMXatUWsJ40ndTvRdtvpkBwNo6knuUk3LDhsWoJs\\\\\\\/fuEWpO0EHytIlkwHRWE1Ziw0F3c09HrQ00jWbxZEif34tSM3cXErv2uvgm1bBliFDP711BxxEjyghqN3Sb5pkaPizPw8Kpwpsaw5h5XJs+h82f0NvLnN8Aa4\\\\\\\/QrvllNeY2MVIrSY1nX76HPK63IL9jLOyIBXXSkysB8rBRwWbeKSgkqU0\\\\\\\/Op8X4fISzNmrDEZtJ5BHSQDpqIx3w2DW\\\\\\\/jUprwpxWVCo5gGGNAdhKnatAamQN9G8fWcf8KzsuDDd7kN4ojKdAJiSl6CSdcWgNqssnId3\\\\\\\/AUZWkwrIe63ICNm3dfFNveXSNmrKHYHnkQuMgr4ZGGTq\\\\\\\/Dr8bLH3cflGbaPeglXFGRXXRoDt\\\\\\\/lKQhbx0\\\\\\\/9Q0RKvPZvlrU\\\\\\\/3It2697Mdu4mSEEWBo+HdMNSg1HLMrvcbk0VKiTFSQ7\\\\\\\/hAqssy68pTCagx1efGVP2jVLS2JOY55ierxPx66U6Rv9A\\\\\\\/8\\\\\\\/1jjkZhBj58KSSPBlRk974gSa6UyXZLq08jt3OqFlIuRGxzgXIJRk5ZNuk+uP+EFDfMoargRVtS7U9SGsfCFzk4J8wQMPlAAbTBPcErSMWFSsKtuUabZkENrCWbMOPi84moYhkGpfFPQ6d0GV8sZgjqXnjXYHzs0Uz+J4+ZSYBCAI2zUG5EnhpeKbvfsBjPYgSATRhvU6SDUkLRzR1Uq4MFFwd0SJMFo3MBZ0a6pVFXLFVL5p6o0yXSahZp2kHXJIOtL76UQIVExS6F0VxBE5PAGlXvxtBrHVqMD7AC4oUWw5nCKcoFMLo97PmbS\\\\\\\/tm7qYYC\\\\\\\/1UDYrLlFB\\\\\\\/vJH2q16k3jCfkWJiNpaIGSB1MW+aBkT85OMIhCcDGkRMy2K4nt\\\\\\\/s5+9Y\\\\\\\/30D2h9kIWksY+dwSZKTbUAvT4sw6zF\\\\\\\/VMCJQ7Id5R+nKRFQ61RoV\\\\\\\/6S+pEpep0qepB0EJQcXIkXBcFHHlENRqj\\\\\\\/y4NTfIDTW2XxBQBGyQayXeKzd\\\\\\\/U9d0neApo8emItm\\\\\\\/ikJcZ6F4PeJdewDSy2QRv8ycN+Q2UFapkqTlt+aNKDdBNS1JCGAa3Z7vrf9yWACffObVtqEyhYECn1Uh2qHZMd30KDBwTSWK0c1NPsHXTxTbTxwxW7kFs\\\\\\\/Aket152cx8GdHSi4CPv5uHevErEfQ1KDEudEvelbap7FtIsMgV1d1xsZgY8KT6bqMp6tXjgbJmwfHIUwo8Y6nO654927hs+bQ0NP1PeLon3RO3WCLFq0jS81JwsUF7\\\\\\\/Xfml+ej\\\\\\\/egSlhIGBcFTM\\\\\\\/l4zywMsQohz\\\\\\\/DavAw34kOdLhHO8vGJfJg6VkbPg4QcX+2GS4rzTfQosMfTBDhe8X7z\\\\\\\/AADV2Hmust\\\\\\\/eXdndxFuFu0WyirAxVK2lOjerTk+4DmWHSIalNgOFDJMoLl8kn7WJOVYjcddC\\\\\\\/WNR4A6EQA0BBOx0Rr6WeJP0+dxPDkasm8avLOr26ocfymB3lW1Xy9DV\\\\\\\/sUtcxD196lecHSehEVwPNN8MOR3sUCKKPchGy71uBBmTuY+yuyUyqe7kcQvTEuEq4Pll2MQrH1X9S3u1QIawyFjU3ZzZ16W3P+kPlnKy+Z6+MfG8m3hi4MXW3tiXFI5XeUY8\\\\\\\/1yR5xx2VHRapQFtz8Aep9qtIa21WGx\\\\\\\/\\\\\\\/IXho1CRo8FgLBE4FIcqxVJUZQ7C8TSBMBUtISfkYNto0TtDJmVDdCUb3WiTC\\\\\\\/ipsfUWpe18LpOf9vFiWcys+CRrksD72PnJQ4uj74p5dtgK1XdlMxN\\\\\\\/wDrqwsI5ndA87NVwcHljvCAlrkVnmHUra0m8APqMmSMf8Ob84s1pRPEHrPkR6aAx6dfMrDufCITQcj6pVQTHQVboxxon0xLLKyKchdR7a9yRjqwmczCTWiP5XfLPJLPdH8x5nZMIm6X1l0EKjvffMM8LhewxoCkFvyt0HXTKx8NyTzCQLNlz8jCJ3rpZzYIHe17lNQPTcbo1jONT4pDLsk5z2wHxgakh4q+pgKUI0EhSzrUS8wDxCKBrX8oCNuGl+yTQI7AXcF3UjKCwYBHLWwOAWD9YK3uJjdgIfriXRawKPfw2Ni0aLG8g\\\\\\\/vuyDBsfj3beUpoM97BLaAdKzd520gJPOfe5Wer2wBOSLahEfeXhPV1LpUcvBnmBDr6MWbWGS2U5GiJ8FfInhengOK7UH1sRCr+wZCf8rGcMUodSh+4ljLYx4BhRbPyd9+lXqefWvd4t6UU8+nchOO1VglWWULQyfhE6W3c6u1tsyJVrUYpNv+ahUtD+7VOO5uRXNdp8Uqao2\\\\\\\/UPryilPm+tI6HtcSdAVeLrpZbv7HQr1JSGEy283MM0RqC5WNUgcNbc5JLvV7+Gj83vomK05qEERHNkuhrr+aFDRbg9mdTi6Rr\\\\\\\/3bPHE542RkLeZ+jzeufhWBl5f\\\\\\\/VP\\\\\\\/nX\\\\\\\/m+eZGtW+vqeCAKqvKxEnyooKv6RqbmCZNTxDWcFIUYtiGdq4X1j2OY9kWtYZfWWI+IVMCwJEu9r\\\\\\\/qgSbEvua4OmwXDL1HUA4Wl\\\\\\\/RfDBu7AJqeoAFWjwaEuu5orQYUip7uWUcblkS2VtIqkdtVhy5tsW3OpAlfax\\\\\\\/mpUvuhiS114eNVnUHFSep3igVuAf0TXd1DE8RiakCnA7qJ2UayBggtk\\\\\\\/C0CBNp3uoxfCskb\\\\\\\/vRjsNJOKa0PldS\\\\\\\/0jwweLboxsdu4eT3qSbw0T6AFqJQ3YOYFfjimS+ZB2x83AqlHl5ltDBrnH2W9S2NUoOVS+vcObwrBI1XrfaNZL8ECEBh7jgRENI6rEhqte+ykss3KhVdaTyUcIog1ZLeSu2bpow9Jhms2F2PtVk9IkHsytxi3faO7Ch8423KYnJejEPK0dVExx0cleXRCpDf8czjcW4S5xfU8zW8ggO9tDScAQvISfqoImixKc8lAnNDjbtjJd34NohcBQqLacqPkzvGyP0+EWrsIKPgOhcXoH0XPpfD9RDQ85gBxJOTkknARvNw7nVxRt6P0egWDEuOtFLxdCGJAR+qy9F5CW4euFmKej2Qdtwx45bLRcz7Mq1YGkvuBJ80Bt5xN229OwYlq1RUlWu24rk8bRaQyrR8BK0DdKIU7lPreoTl7POSnMurqP\\\\\\\/q8lqdZUuantMpPrBu3ca1mIDrwNRKCE3yCpzxEMB4LiNC3a8JzhT\\\\\\\/OieBOsvpZBkWtM5PLxuaYOV1pylbkbHq20i4KveUksJyeuvJYax\\\\\\\/jBGUhEQM177BSx31N7gXUvfYld20WHJiqNWeatMdQwxDPS2BblLp7HlQ8jTlHGfgRhLGt9stz0OiBJUiebuTnQ7H\\\\\\\/v5fi5\\\\\\\/zN9YjdOKAsz1orY+f4T+uP3PA26OfaOeucR8l5Mfjb5scIa79bgKBqQy+FDW7gm4kD9LQoGvVvZ2lClIThjB7np4qpSM\\\\\\\/UaIf9AT+g7utR\\\\\\\/cVBm4tAdikeCJlCWZ7Rl4Hr9iSF60IgNbZXuxBTP4loqCSU+B6s8zMWclk7z4Sf5fXF0YWxDKU\\\\\\\/GFFOqdDvHEu8Wuwsh+gZS0UoyYikyerm9llQNLerhZN3uDFsxx5XhEnuS4BHhU0ekZrlRBBxnxJhvgTCYR4zuR4KSWPuRprGwWtXgNAyNVPgzVz6wMbVL3oHiP+Qqz5kv\\\\\\\/PBxgpF6zT0gAzCDt0xVh8lBOkDB8CqWcAfTehcnDYOkIbO8gSZjV8qaPyfZtI3\\\\\\\/3p1AWsEQTsEeeOpPAhtENfu5PI2WAkTu1U3nNhC\\\\\\\/J0j\\\\\\\/xIlF+ZJSKK63nU+QeDOiCXegwUNw6326QPUUwJmmazi7E0\\\\\\\/ef6CY55VQDIFHtVyc0ZppxEMLJNcjBVnEPEKAkxInvHORz8HrmOzjpfESwIbsKiz+ZfMAUkqCdMJgJQ3unU48oQ+aGmGxleNs1X1PVPrhGjeRWLV+TFZ+SwowKogKkvQD\\\\\\\/XJECsBaB3y5GJ1B5uCsTISFuZZt5iL6MRWdxbi1qoZsaizoqL38ql6HeLUQrJIyWZqodgN\\\\\\\/CN386QUHm2Tg3295gKzBXEMDGXwyhhQ7VtS\\\\\\\/0uZe1hARWHWjntWwik9QswSj6FFjCuNOgzigccNdYujb+D7s+2fJp435kZEbjWWsrRviYxR\\\\\\\/kw3jibvCdFVHiAG4It1DsdThGKYAS\\\\\\\/+I62rFuby+XvMTFW0MEZRA6HtZsh3mPBfBzrqcD\\\\\\\/t1XfdWogxASedEWRVmnZ7wbuxOnSNgwEOBWK1eYmzkznWyXovlt90hDmL9wa2oVq7M1gr20ORbK2aVmXJu+o6YZskQc92CpgJLyOiUuQ7w8wbAhh3PmAEsAK2senfOwh6xYW+oJFsnaKrF7yoo9AjFY7VR+U722IJ\\\\\\\/doxc5x2wDKt1GZnaIv2kczM4V+4v8l5wpWIy01ictYVr9ySgnjrts74tWtnrUoXxTVJBuXJBgoM3k9Pdk5CW3rGSlNjc9eWLMkYegkrQIstUZ80o3s2bC8wNCyvE9\\\\\\\/hqsMHyhYpxXkxBGVLp+6ayZ7oemHWdRHB9k+iLWZNvqlGGaApTWwwCssB1AOimWgh0DGbqyLifvwUS5djNB2mj6QDCs+GfIhAUgam2dTmuSY7BC+PTG0LfZi+gNWYkd44LlFjmZOJ4KOCB6YOVGzFt5yDGajYf+\\\\\\\/fOwCoMgSMEtG0VhX7PkbKC9apxkcfKtcpmw\\\\\\\/RchSHhI6k2fvcCKKtGGvp1adojN9hWK0t7\\\\\\\/ZITqvFv0YlJYpVItec9I6exwkQ\\\\\\\/XigZio1tUV26i0sIzyoSbKX+kz7azU2tlER+6\\\\\\\/MfrJi1BHrCIfLrar0Jfi4Ebbbdp9hNdOnR0TmJp0rDAjszJrdN05\\\\\\\/VDfwb\\\\\\\/YVfpXhXppFMUj4QU0QT4SirMIRJaCqo+GGttfEfnfhxjqBb8g8LESdJ1DtbA3+kcoUtvUeBe9XIX4km2VbnIMlxyg1jQ2OKc09OsuFqQ+XHiOktDvd9drSLh41gQuA4uxa34k8hxpjdsynTxSJADuVRpnRhA\\\\\\\/r6r0UjNWRPM1DiHish3cS9eFM1g7sSGkUWR715Tvm3Y5R1ZG8OpA16nQEMVQeW5TJcgBbJo9VgelkY7gBnoIv7tlWrQy+V\\\\\\\/dIFpS4NAzjx\\\\\\\/xbsjfi4VuFduc02YMI1XvIbQgz5\\\\\\\/gvUbyDENZWpbtveLkDqQtwYri3S4WjVa6qnYR3ts+HYFpRm\\\\\\\/M\\\\\\\/Jos8MSPar5oNwiTCx5YblMXXIQIyqBe4713XPEEIZEeEGhK9PikzR8s+RC3bj+q\\\\\\\/jbhUbAY044xoaxViy8n92zTJN4jxbctQPsQQv7n6cQ6O+Efk9rltbIuX06OKLhx96JRDO2ZX4VxfPib1FBpcwsRQxLbP0M2jR1Ks2lYOaN2OgBPj73ma7fY2y7r9kaQWf8I4feLaZ3a4AM+XaKH8GpCnJGBWGbW6NqVXmiLoBh4TXD4KLE5y6S14AH4kc7QssACYvbq0PyZ0dp9qL3HSGlPOIhgqDIT05aX6H7iqBbBMMODxPymWNVpptUJy2A\\\\\\\/6zYp6z+9ukqlPPn0sPp4P2wvoTb1RYZa+IL64aa+WNG2htEa+F75KLGgUPExKRemHnqN69WYYlT0VQPG3iuVWDPJ1xz2wMnlE3wXewnZkWpgSU\\\\\\\/nJNTKOFMdSoOYLlNSNDB2o8ZU8jEExrJSU\\\\\\\/8jxyYtA9DJWy4t\\\\\\\/x7R44\\\\\\\/CxBOJhX241ivW5AB\\\\\\\/KBmGrfEMYCO9758ij4QtqZbCs7PPvkxa8Hft3ywXpzMeLPkDlBXR\\\\\\\/q9xMf1Ohi8+\\\\\\\/bODjKnIMWoMV9pmLa\\\\\\\/\\\\\\\/VUFv6WrcdZ8lYs33o8p+bGvPFZKDEPEWMhYMGKf5hx5YcGjebCQJRmt0F9sx1l\\\\\\\/WTxRMhYMpwYQ7SsKpj9KNL1BlnS12VVsRpknARUvlmT5jvTaAIxVPI8h3pTsgFQbO1jkadgwAiPfbD0FUI88hfTTxMYlxjq8orQfjYRBH3kUGqC5RGLABftVzzL\\\\\\\/tNRmX0oGbXx7SFkTCzX+NBgUC6ZkJaxdqEUPR\\\\\\\/xu8XPIb5yfrtCqZ+NmcGPPzqKWvPHR\\\\\\\/2kfAM1RGqNmZlAql5h7kDU95D\\\\\\\/z6jh6WYqfgiL\\\\\\\/PGIEp3f1mDkoQIJO\\\\\\\/ebLXuJLBKAKYV2xjdzWrAsNnS5N+NNxQkGYKTYytcRMJVrxW7gBFTpXdOlHy9TbOkgrVuinyfou0FQJ6SFoTH5wEtU3phBJH2ykUBsLDmtW7OcmPDAC16gxnKlAS\\\\\\\/FjN5HAdqYTPlekLBf2344FVhMrSxw7XCk0tzdITsRCR1XoPwhK2RR21P2xRluOdbAnglYoNzYveSsMBzrHwcVKFb7HIPHMJU80cSn46aj0c4DFSq7jDbNjl1vFerw1Vh\\\\\\\/2aQ0a45+CwQoBk653CU3JcERwBhlXByqKOqzlCnGXLiqJm4AitcFk4KerU\\\\\\\/BZNdgHFgIy8thOVmTKv1thqeqRONqOrsWEe0jyxT028vaYZIax1gzvd5V1keC8VMmNqfwLq5GVATWhAeO+3nNrmIY6ahSAXbPr8CMtCgrzw6wWWQxnv4kaOf3Z5B\\\\\\\/DA34Lhtq7d0gA9EMTOeQikGGyUHVrPC4zpUlIDVPjz\\\\\\\/lXakS\\\\\\\/Cr9DgCFcxU3nzAjt\\\\\\\/MpFDO8mTZVWepdzYjvfDdiUcYV3YOL2fMhiPzVHRZqTFcrBXRL1fwmlKImcEnKXv8ZwFzq6iOy2K+gx6WpwIO0\\\\\\\/PSdvTx350MmZgugwi048N\\\\\\\/XmX3S92xbuc5I8292dGcBLEXDNJVcMJmyF\\\\\\\/25rkK2\\\\\\\/yeNu\\\\\\\/JT7Ld7R7y0Xnx8elH9Y4RVo1LfE60FpWHeRd7Ij+ruXi64xcMtEBb1VnHRvHjNBQcHW1b6aJ69lNXuWJmeF1X3aP0DWoMXmNKWyq18WFJ3YlrHbvKI4ioRuw9nmBTocrmLsB9eiAF+\\\\\\\/fYyJBsA9taRjBwr6wKzdGRDaUOp4QMRPAqGYwy70tXGaop9R6Bikwlk7AcFqeTqpF9RL6BFQWkEGa43a5koVy\\\\\\\/mTkPhb6VGsT3UqeqyBzRXlYG3SZBJ2DviVFaTMQu+aUqnKtejzgW7VYT7xKGN9U2AS8BSGDO05dTbfpKthovmilJnqc2Ls+aFUwVJk1LbflcOvTC5VQkVB1JXahETslvUMwLN7ZSa2wdubJXKTMl2LUuuACdp2inx24xL+5eAwTzb2nenntlK5Pd3NqcveGOCOJLLymV9XUBlj67KjlyUY6xxbBp9ysq0q5l7G2tsInSlo2XWnB1r9WOHkIfoCDzm2JfuSEcbDgdwmrQM7F1QiUzue4npmIJ9MMwu1yRItQrxANqYM9TtEYd14L7IDZOb6fNTzIvTUg5CXsrWNDaTeVUNPSU6YcP3oCkbR9IGvsOTw9B35tAZQfnTLYulDekt7ulNBvWiauX35WlGWXQ+nur76no2Y\\\\\\\/refCJmjz2bQEScvOpQ5fHEMgwX2hpoOZw+9SSfB+E+AnMYQRCupAdUWEpINDjmu6i+Rn7UkYiBsA3WLqEB7UxFX9P0uaINd6VZK4kXcPIn4ketD0VswLbtpk7g9OM9XWyR3gEg94JL8qISXqydnxQvtS9jrNtdrqIraw8nNxZdVWM\\\\\\\/XKCvheLprSIa4\\\\\\\/GGK51v4iCXpniePUM2bF9l9mWTBaX+ZOeM56oHZmuUkzct\\\\\\\/XZn99rRrIsfd3vs2sy66iv5GZf7pKZbffy9TGWiIbLTFnfctwzdID5+7UPKlXF0FNWqhPmMWH5WHxsa3vynaADGXOKIfDVVwt3Zli+Go6Wl9TEUMeQJ5eNW1KoklIEwSIIlanV0ROOnKJSVfAQ5FhKtnRmymyt2rFs2XnTELv1Q68UQIS4x1BnGXRet0HmKDfZaBHrP8F1JQ2gHZO0ougWvGQDOIgl78XtmO4udVoWfyaGmJhWD6ZxgfP5qDpTOe6e6T\\\\\\\/WOaLpSyLFsNxX8PW6agHdJBAd3gZQ6vx3VXI3FSLQmnEIeMyh8gthjRrXwEdOqOIN9smUgr\\\\\\\/6g1r2sp9i+jQ1FuE\\\\\\\/rwc7kvK9VmNav+XMcEFQV2kMUEl6Nog2bBqaSQfVw+9GTv+LbVLkWDb+bN977\\\\\\\/miyld79Q7HG7wxyG5Wg0k1WDiMEHsDxseWRjKZa6VO6urjixeIiaKKKYHp15S68L++nymE2Q4W46hcXewiEVppLOvQwkEDbHFrnEfC6jLeRCcPOmK114VXWTgNhmlsgvIFSwxIpLuEFa6gNerOuyg1hhEgJRAN18MKss1thFnOO6HXv0ggM\\\\\\\/a7Kco2iCpa0TkloIWU4+4+e5WtzXmtrBbp9W4QmiJxs5\\\\\\\/k8SfmeHUzQ23SYJpofkrKtwopZmnbQHkPRxR0MpEi3b46QWJNOpoC283EnConmlTtZ2Vsbkb2sU8HVyvTwnpRe1jzP2TFFOK5Yr6vq7Jwpl94unxUu1jVB1CQoCE\\\\\\\/5DfIJgNKkvruHhNn1qFHu4NkKhOIVPotqELPegxQ7p7AU0oRDY8VW9UTWus9Umow+nBwX4vIzcXzd8qNm77KKGqIG7HsPWID4P8y+TTXawUC2HcCEUIX78Fgak\\\\\\\/FJ67+4W3K2meuwTOYSvYqZAfQizl\\\\\\\/kigGwcwigXlpSw0uaLo+jxU\\\\\\\/sOsBm8L3lDSthPEiQlu\\\\\\\/h\\\\\\\/cr\\\\\\\/cj1x23xZkdnsXwQO1ZSOtk1eX\\\\\\\/RkAwV48FbWUSQKXD\\\\\\\/+3+8JvzAIU2UmnuVv2xD6ygSUQFXwcJnQDjQftufx2VML4g3PuuCrPh8tqGazyb4oqv8AmrL\\\\\\\/Lp7yv6JNGuTYVsajwWCd9abBjQUhz2VqYTWMQAgS1RAPbjBpkoUlCubwKtwPGXrv54mm3TwROj3ir0P+Dk3Sf384ceLb5PxzhS4eRDGcNdOpg\\\\\\\/qwXBRhK+QlIBxcWVTNoB6Nq8FkQEeKTbgjhd56OeVtETUnP2oPGpNrQOr6hTdYaYrLTwwkwX3MV6dzfh+1hah6ByLfw2GuAoDTJ4xtsWnMwecLZg+y0mrpGwPF5ZgJLMtjchQxhnmr5gULIQlM46o67JXDiuILOtaBiB0CuuibGt3AjNkggu9uHLx1fuQ+s1q9Gej9uTNXP8HbmMuL3yFaBbWh9uWFhQkjafra2mX6yFM7+Qwk\\\\\\\/wKAWDt0Ut6naVpkRnMMNIefYfYoix8DYvDRSZYDWjg2JXw1W0HOW72cWtjcibmLSJAkFWT2ZTm2BjbUfv2\\\\\\\/ZXVHgGCbgS4ZJUvdvIehyimgrQgFNp+vdCbxIqfSe5ZmDwI9k1BQ0leQPtS+ZaBdaas+D+heuS30nSpNM7Px89i7jiKkSRdTX7nVFqL7KCb6q8vEv394bjtf\\\\\\\/9IF\\\\\\\/Q48FAq3\\\\\\\/G8UZyz9aKkGrQKIAcEeKXZi\\\\\\\/OIQLJjJyvxOgAgbSkkkurcCipRgb6h9OqEiMwDMm4Knd8t+qLeImOWZiPe5rcWLr7vx60YPKsADnq7RrGzEF1A6SCY1ew1wMxXhAzNVtiCtmBKh3Al7DrUEizyNsM4\\\\\\\/tBDOoDJadADpRtZ\\\\\\\/EVphpb6mmN78Zl9+9OXoK3GDBaf696n+Ph1tMzzSYzlVRvpD9Z7bj1LaiCmjj85NML1dHi8\\\\\\\/9H\\\\\\\/teQkHNlUtJFuaUG4UeTqJCy2V\\\\\\\/JduzECTWH7FlnMUsKubUHvFjlRgfwl8sWXakGapaI0YxCDfO5hnCoONfV0aLUJtUq1J5\\\\\\\/02qZNOx+2nhHWLaUwO94E\\\\\\\/kcj1J9h+NorRMT+Pc9p1wxNEKuxzkkHNPqeTIUqsWby+mVU0FRvVPBRMX7YbEBmujvXvP9rYX79RG1O\\\\\\\/+uxRHBtUkj5vQC5GIeyFNGejwsdOCK0ERguY98GsXANPNdVh528py4xLNcV8g6TbatOKCTNQnQ5B0drvTipS7CFdxheEPaxXN4lNm5b+tnTiJI95\\\\\\\/f64Nbkgn718p+Hs8Yt0g90P5fCtzpzUJWC4kiN+s7YMpc\\\\\\\/8K+mUPlmbbS1KDEdTTatxLMpNwnFynsejbk0\\\\\\\/\\\\\\\/rA1D+NGibRxUj8FW5KlqCWLC7HL9cnR5hBYQ\\\\\\\/A7\\\\\\\/OilDEQ1rgjJykH8JgBrQjzzcTaZuTAfjAwEbARI7qVxgZlVZBGqS3zS1PXi5gqYmo5FAgPIK\\\\\\\/2hH+eoK3pqApM2iaKlaWvU0jnkhAPqZkXg9wK2n9pefnp417P4+rdubH83ZkBGjigSR1eSewPvqUx0lphDG15XQ1o3lvMUkRzRh3Mq9McsDF\\\\\\\/yEeisv5AHEXKjvD2RjctrTH\\\\\\\/WUq35OcsFih0smm5k3BTEJz9fA\\\\\\\/YZ0THkR2W\\\\\\\/DWV8oPt9dyZPuyV5BNn3bhYzic5g70A8FxGUJ5DXyeGeKWf16yvc3rbf5Rp9u59POzixEn+Z+9i0tHRtaqKDmsFDXTQeUPrLA\\\\\\\/DgNHw9jBRyqLGpEdny9m4YyXC7Tnjup5ALEG4knsC5Tl6y8SVaOd7HEhtst7sqW5pe+0\\\\\\\/jfF1+TE14rZVfv7XFq9MrB6NhNtwSNlUUq0VZhAamAZA3Qywlalhv2fVwmfB+5NlH9hbyCq4cvy3ACu8yqqGhYdxfQwzLmKmh8BRaQsWZ5RkS1uOOCJ2zC90qgn9LTAg\\\\\\\/SYYv\\\\\\\/l4sQ35\\\\\\\/fbqo6e047m4aWihgQfo1FLn6yikurjzQp\\\\\\\/O0FgFad1sT1pHuOWAOpL21T2YI5O92Jh5thZyAF8bckiQjyeuGb8u+jAHz+KY5Y1lkT+1IZLrK6opcbZccvcg90pe6HyklPLfxk\\\\\\\/bBTY3iyGJbBr+5XjylqpKPko3jtStDrCDqich+OgmAbepPPqnkDkTMkA4vDZi8jU1vs6NAAOaro4ZFnxtZM6bIaYfxL1KziUzfgLPh+PuNmQdCt8nmeDKU9YupefxGu\\\\\\\/XCVuwiyw1iPKHqKMzMnpvlc\\\\\\\/hKdmKxjKruXTyiu6VVk9AWC13ZXhP9qw5G9c3L9kANV1B1UVbLoSSVlQoUnVesAG+\\\\\\\/QqzIhQYZBkD1xzNabexfLMXWFfcB1iF\\\\\\\/X0o9RUFWUfmsedt\\\\\\\/\\\\\\\/tWGxffYrniP7LdbAqqFI6UeCsd8HfYcBXlA20lfLzKA5iYaPF9MuD3RWSiIvQ3iDQHTd86Dea2MaUQ1eTZ5H3t3b2GhyNY\\\\\\\/omWZR0mDavBPczXnfQealhdjcvJ47+KbI93f8wEVZjSIv\\\\\\\/hWTp6D7xcaBWSQoAQwPjn14D7R2KcPMKnTu3EOpmt2u6ZP6Xxg6WeXytPWr0F3NOVJuvqQHEeuXTIeIn1515cdnfz2oCjSOZW9\\\\\\\/aQfqDPjVck+ZHpupZK41cqZ4U5lYNtUbsLZTwz8Cc\\\\\\\/k+Swb2bxWUtsgD\\\\\\\/\\\\\\\/sb6CuM2OhvwRvJVO\\\\\\\/FCF7ehZ+5x8Tq0SkvNha5LjBmkyqzh7sLcEIxyAX5R0b4y9BD1vbCip1yomkZ6ENPZ2Qxn6pUApTC9UGTFPp2mG45nkDwrn4qXtdiFfOFzJxKH\\\\\\\/8ZVOP2+3WoX8X+ZurqvC8LBBZSiCTAOE+hLFr6ZXwaxxGhRCp7\\\\\\\/awliNi8\\\\\\\/\\\\\\\/BcOfTeLPa2L\\\\\\\/ct2+4SeEt2bw1OuU8PBlfceA4\\\\\\\/sbZH2Q\\\\\\\/Fgaw3uiGcBPQhZsZ+MKbPOr\\\\\\\/lcXq1rUXatOhq6GvvLhNwfAzY0c7tBg5FAYSC6Otlyt0dizcjsbRNbXKNcgzoSP7oO+KJ9gnj6TjcWZIeEGPFoQuC+lZtjroSKZaeBaNDInVrXCL+N+OKmKAxjHmBBM02cmkySjt+MbSB1c\\\\\\\/Vsw5vGuB7Res2SnsfhWJnHeoXvzG9O0DA4qnX5vzO6Wc0SzeMHUNw0nKaONFFIcZdRr6+ze8R2yZLJFvvTt68Ths1J993xtaXPcbGNKBwNAsSotdgPTDavoWjXTtaZghnwW8JApSlEehWOXbOXrQY9PQ13Pt8GaQxedbbKHXZF+wXOtPz8eRs+KcWh0aMYgOVG4UvdDqhcsyUynekPKhmpmH+o5K\\\\\\\/qvbPEfnOq1ikgI47RL16gBAFzf1COJjZ7mkekImg4DNvz13uZ59gtM5VoBD1m457OcK4K3vByuVv75CpEO2W+YnVdX1eaE5OzdBx40cWqF21ZEokl\\\\\\\/LTTawwYbPWcYdHbmlyoJpn0BaRTQu8WCxyAvBHIhLrD\\\\\\\/6Z\\\\\\\/52dmqRfDwfJdxNdVdlPr5\\\\\\\/RdsRKrraB94m789D2wON1E8xnzWOpkMFYIUhs9E+8\\\\\\\/gyRZG+PdTwCZNkY+dWRx7BZtJnTZn7SkpJI9WY6nvCG5dZB7xdsxb1IB431\\\\\\\/ZcIm\\\\\\\/ObB6FgGllD+g0Z0mU7UN+CH+YudwjYvUM4NewJg+HLhpkSJeCCwLNZ8SbWwV+wcLIBiKZ\\\\\\\/rMqCj0TItQvBO2yOkPZr6qNazxsdzQjfMgjpzp6tyrWDPvMShlZnfbEFuxddhh83zZ4rj+YJ4mM9KSDjzUHRc1MxbPDp+sA2Xqs9XxOS6mBpp32rU4C4\\\\\\\/UKb0v2Jq4A2TQCYkXISYWwefEa3uuu1QTzN2QuVPyD5h+Yt1aG1T7FcB0g\\\\\\\/MLNtlEcjmvAxcR++6oUXQrRFc15LT\\\\\\\/9Pe6kFasZ2twVnbrYcvT7FJYG1EJg1KCkGfZIIhaEMCDpAGlffdyDLZgsq7NRUbWqWwTuP5Y4COx3zOjwQ9266FPQmfF1GqurWIrTMpaXjU12LZvqWkUbJZa0oEVBcCd6jhI\\\\\\\/Vi1LJKiIF9YdcAGIgRnxUvMKdU6zL7Hg6fv2KkIoMA1pbkPzFtaOSCn58gnkNoZ+F9SWcjpDw46XFZYoOj+iAVDho9028IZTP1bpCO1PzbId4qj\\\\\\\/x0m\\\\\\\/Ht98eh9NSx7gggsYbcDRDDETSWpRHt8UsYqDyV7lLobkbo80s87lKKWnSvV\\\\\\\/DvZzxiQMwCn+8uzZj30rI0\\\\\\\/de\\\\\\\/OjJFioP+SelcfgZ7QldfGNaVUmSw9lN7wH069eJ0v5oiPr4MOknmrxfm\\\\\\\/0AO+OID5LW9Ownd3pjqiTEwqmhlq6rVDY+gYOf4p1rkmqt4x2edG1f0q56kC7qSp2nk1Omt+JjOHSMuL3RQ+8V1SwwxnobIFk29Bk5Xor2iCbbC\\\\\\\/HhpJQsy9tPVljzXblRiZ7uLqOSL6aJw7ae78bHjfvgch4IDHDpEqugJQtevQ9dibm7c21UCaATpZq2+A\\\\\\\/is9jCPtJ+oZOt67cdGsBGfU5v+AUnZ31z8lKGNaxABhIB1Z4kCJNJTzo9NNVa2rLOmP\\\\\\\/o2+oJCzhN\\\\\\\/C6mTPtcq++qfxz+cnkofAGl0vH6Cl+f58AVxmQSVVKhSTHKN1ZDmJmk2zyu9Jbcbwz9wCTDHac1aK+d1YV7au0bCHKt+vVO5JJcJLbORrEEadFK2vYyoyz9cxmRh4i8TvhNOJdPo\\\\\\\/fwb18afY5od4QQZ+oAGbdMudRVOIAaZeoGQKFOaM2PeW7JJ0F7DnGRb7hwggTMlBQN1ErUHR6jV9wC7Nb\\\\\\\/bpDNbcUHFIG1AHM6rdisBkAR9MZfZ6iXGqox8gWmV81lQize6s2gahjv3ThHlBjHVPEuuSSoxIZIgNxVl9Z6Gof97\\\\\\\/xLPJJAD3yBf9Std5f0mCUSBWPVsKCCgaRz0H+ciIwt8Ln0OP5CbXTmiTr9bJ3flSDJIaK0jTAVWxC0OnY+3O7HhO3O4\\\\\\\/UdjrOdfFFs+WQUTZpRwzAwnpNXajvb7zjc\\\\\\\/SgFh+9Jq92sIj2zVFi+4L2DXPfJQoiXM2UZZAZ4onKOYIx0wNWyH0yMu+SSQwjfI5YHSrMRDuaHRapmAJYdAppf3se1Bc0IEemIjfKWaRs2b4dg+Iac4dUtjUkag1UvwMQhcsvDbkJlLfChsA5zqklfBwMCJqRqFZhbsvEC+sqkhqHkoONxcxAzsqMqRcpsZLGq6ngxlZMM+YRzGAz3aDfJVNQc69Fn5uFb2zCItI10QTvdXeO+58dr91uSrt\\\\\\\/yDFpH4SuOSglS2uR34uK1zmS5nNTco75NKP\\\\\\\/pY\\\\\\\/Elj2JFAZAkUY0493g\\\\\\\/Bh0UDc1QodvHfnMlYArtDKVHFEhbXdfCnCDUtam1Io9jIGASrjCJUEYvzPCAPwFcU3zwn1jwoCh+Kwfe+VvQ+WEaQ7uRWgo1SvfwA5ZsLbcxZ81U0f6zV94wPDEY6vCH5ex\\\\\\\/rnnBbVIE\\\\\\\/HsH8VQZ+sG5ZW4Fe7zT55yLfGsGlG\\\\\\\/YAMDnwLjH8xdoI98Or1DZKaAYCXC4ub86A4DCBQtANXUpw\\\\\\\/RJuTl9CXmKBWeWxyukc7QvSmqkBlk+c2x24ou0ahVjPqq9v3W1lScoX7U+v5OiVOnFJVbS3NkbDm2fMYkfPUfTZ8dbY11VZxIewMnhwxNK0cQmpy+4c4ZI4mlsUs5bn0yJdAA0E8OqP8s+cc7kgPHpZldjFkwy+7I5eVGjDWDoO1Gm43VOw8e2JsXHHihFS9brr8Yow7biQUXlNufRScnz4+zX7viaMQqaFOGXXyQ3uWbM9z7SqpFW6usSfy4AgCU12g60Z8uJJkcBz080wTghYWTCHfoEvcet\\\\\\\/o8AoMcptH5t6\\\\\\\/yjz+xzpmPmtJB5SO2L5Un\\\\\\\/0Xb8hNdXNIHlWN1htDNSzeiqiB2x52lBK4BY9fF0goiCX5JrQGKKbrEocUaTNumIgZx4Y4tiImQbJRIJtEr0WAsLZPvWmK88+5Tu30e6GAbGlMUdH08QE\\\\\\\/NJjhrayHdq5hObev3pS13LVhB71KDKZqwKraL1qSY+TsfgUzfVUrZWzSZ7KSAsBJ0oxPXkeACVVP+qfp+ZS5zNnnR+SSD0P\\\\\\\/8RyJHE3X1e4Emkl+izpc8KAsxnBIeiA6Yv9Og1HTubBQoN8gikMns5OQAqtIzOy5KOruOSGBs28vVyUdUzH21lsYtNQOqAQe8UiVhRQmZwyx7lp37sevJ2mlJpdtISNktm9+wu38nk5aMilE2TRQN84EaiYZNmmOLyK2yYg\\\\\\\/ytagAZ+gFjb\\\\\\\/+KREV7o77tvMWSSzzjI1n9ImWDYp0SUFN+iN9NeSwlmkzz0j80lNODxVmVRDMcFz6x4OaeE5Uw0KkrTN8uCjb\\\\\\\/xerIEX\\\\\\\/3\\\\\\\/BwK1LzmwOZSLkPqeJoAMdMld1mhlLmyKozxz+IxvT6+aTCNL2tpRAVzC2YBIYRVTD4TngQGKeSKbIpKe29vX3YviY7Clnluww7jPrPUPry\\\\\\\/Yl1UxDQxBDO\\\\\\\/uYxhPF7Xtr\\\\\\\/9mCWp9n01WCshr+eZXwezfhhgzFRe7GJlkk58MxZPtgWUisBFuiuvBZrVGk4PlGzKmfeYY05ys162xOnwzxTlwfv8yNIATT\\\\\\\/zzJ2Zsr4gOisKzUzuK\\\\\\\/VHcSP9kKWJKsGLqPRMAGOylaObAA2pii+KbHi88rdLDckhq0SAlPUmdaJGFFACm\\\\\\\/F9v\\\\\\\/jQZGb+5nqM\\\\\\\/kOUIm6XznsT53y+DXQKcSykMeSaBceY2ZsUGK8M6vE\\\\\\\/8iuYMlePKVJ6WI8fs4lwFkSxm7lMJgDAYltvgNanBtO70QzfU3lW\\\\\\\/7FzPVMHEfPY\\\\\\\/AjPVqshxtRf+VT97OTyfvY0NRcOkKAj9RKAvYAxIfwYxn0AC3p52ecUBXX744r6\\\\\\\/ug5qxI152FD6Ks7bcx7t22yB9jGKfIahm9Td9l484gxVoaRoM30IQgs+FBe9URgSIE9Sal5FZ3CtuRWBjbVQuUlQE5mnPks64YglSA5Ws0mE6uEhRlhmM8hXQvUWdwZMcsdazBPzzqorWjPClZjeU68L4L6JOkj9etcXzmfpXf9Md0pAs2d5lyHDn7d2b5Kgxc7t+g9VZ8sT7ruQv4WSXUH+egYqsrRL1inVftfzBoYCY1eYSfIsjkZL4QXPqpLdJPm\\\\\\\/XdAE5kYfYMY\\\\\\\/8yOlnWxBibaC386AD1\\\\\\\/oXLgFg\\\\\\\/cU7h0030Cbfc+HokrmjrOKaM6E7E75mcAWYiGO8aYupQHfkoi5e3SMDWal\\\\\\\/Nd2H9O5frGGHySxc9ksbVLef\\\\\\\/yiX8ZF7jglBuZ7tnuLEdVVoey9+VV4na+IVOp\\\\\\\/8GJxaFSOm1+nxo38qTj6Qyh4UDq2\\\\\\\/8yIrruXXEsFGQ0SzhV6bu6e\\\\\\\/LzDfRLktanxAGTBuT4Ex9hsTXOanoFzK4lwPmRllplE1eIdR00eOFWs+ll9YMf\\\\\\\/wtaLsFLL3Mwyky6bWlxiL5s9opgXGaMhaFtOacVZGjtHWEnomXOGo33XRa8kF4Qo+3NXHPUNyz6Kd2s0kX14JGNhMsn3J8IoktEPxDHbFsH+stzbMn\\\\\\\/10WUU\\\\\\\/SxBZI8KuO4Mv4uwLA79DF8p1DVUjoEz1iZ3+X3qfXGTiAvAuKSaQGVpAWrSwZ00iyRxxkd+DlV95ZyQOaURpmJR\\\\\\\/ZAE5XtOXs7pJ90MxutFikxv65SWOAAXc12FSynpBs\\\\\\\/FfYVgmJ\\\\\\\/n9AxUdSn8f7F7NrEpn8h6g6hYLKaJPGe5AnUnagOF4B2Yi0Wdx+nLRhxWtpgEGMikw6P53EExrGWr+uFzHGAiafKeC1\\\\\\\/zBRS+M9JYpReBmZT27efup1anPkWF97G66wWX\\\\\\\/I8d25GO70UJwb85Dcs1pB7r95svZ98xcfUgqx\\\\\\\/yJeIb05eT58n\\\\\\\/vsQiPgNTndwpZR6yHDgkVbvraEEOmFttxBDFPuv2n2EaJZdkOQyys3yrK6j\\\\\\\/\\\\\\\/ZKc6+xcMagyDP+8oP2Xvp6kCxHky2cyu\\\\\\\/4+DBP0\\\\\\\/AsBgXiibZ0lPlCkyJlyTUiZifk\\\\\\\/9ukQ0GWh5CiKTo8QQrbMGA9QDpSRS\\\\\\\/KzsG57DNqQsKpynM87Xv\\\\\\\/QKEc829Z4AVOIsEg7OcGuetdUjvC7uJMUBQiTdkC75h9hpDfSqAhcdRaTUL7RMnlMPlwnm3EYrY\\\\\\\/gQvONCyeNKcFDCqbtVxG\\\\\\\/7yoy8UnmLeqNx5PK27Rk\\\\\\\/+x8P3SSsxZey+JhRPgICpuPxTjmru10OHuXRm7wSGQNLvzvxJRz+i2E9pIkU9XosfWg369KWCo+LAWAT1SnNadSBAKc3vI1GwTNOs+ysgLhRN9fkj8d5ReroFGqMTc+qhzl4TwJMoVCGzJx6slwbvC6xp4muY5uyhZT649OU2mHGu8YvHlGCn3dEiNypzI8y6zr6Mw\\\\\\\/YMSr1\\\\\\\/qTX\\\\\\\/BF6zsG66Nd+HSlPXYqfCxGvrNPGX9KtPdBrtBY2ujFHHvs47pGqM3V\\\\\\\/j\\\\\\\/b4Lj+28dQucioDA\\\\\\\/YWxLaA6BzrXCqA8A5S\\\\\\\/YDr1zwcIkHQ9QVS8TX5B\\\\\\\/9VOPUPr2AQtKw39O+ZOYiM1GwTlx1lwCBTPxoPjIq52bDu1DgHxHfBeGalTxvbeT\\\\\\\/C0KKldzB\\\\\\\/aTOyOrdApCsNC8ZDLV8\\\\\\\/UWWLTKsGd9Oq\\\\\\\/T1DglQWMbOHADBcEAhGhlvwwkTnBqtYigj21as8dnaV0mThLdo3P9xLGxo9C34gUM0W8\\\\\\\/qepa9rfT4aUS6TydTnUc0qHv7YmdFgTaponE6iO5rjN\\\\\\\/\\\\\\\/HSZ7fGMUZmEnu1lqNsqxHt235Z3bfWqIB+rtEE+Vx3ABGvm7Zc8M7WpG7PQ5HFFi4i0Gb9AR\\\\\\\/OLSk3YD1npAzNIp20kbUf7S3sM340yBfQ45FLc2VV7dTg4e5lghadkeX++LQL96H9P6tDFNUQ9WmQ573ynhiYD6lmnR0BbcjcHpRt4Yi\\\\\\\/A4RPq8r57oDcDrfyVmgi28Kmz4Ol1eIwLfRdATPH7dFmi1Xzi83dzMxtVIOLLeeuCysw0bxMACiIrul3XJQ30qc8gPd4bz13O1+95QKNsL7S\\\\\\\/VywdMysnKjw4TnlIeO50YTyMLUXScLP3Felbi\\\\\\\/7qO\\\\\\\/3u9qkPoG0Wh9Tr82TLN0EQHX446WWpDihIljo1B6DNBoAfiDY9Zr0LhRl1\\\\\\\/yHzyKPbxmwNMIb3+lITyhiYCJDOesGCh3icsfQ4+ittUzdeC+\\\\\\\/0bBm14vDUvckOO2cTY5GyK2oA4N6x3IHf6RO0kC0UcMahCujCJxR29hAZFBaOG57GAerGoQH07LanYoJ+zNun0DCZDYiri00hvWPs9EHXjU\\\\\\\/Dm2m4EBvqg2Rn8nkhGCofKPHgFFWmnmC0AVAEojexgwpCM1bUl\\\\\\\/zQ75qmFCU\\\\\\\/\\\\\\\/FNQu9SPjON4U9o99BvoQpwIEdMJrEhETeO4\\\\\\\/CaBOkaOSwCw23V4Mc1x4ncWhEm0\\\\\\\/2fgBsV7a4j1sMNnEbGEZwYCIYUUssHJchfNpUxbRxJBh\\\\\\\/N49j2JcFI7kuVx\\\\\\\/6yxE5YDb3R\\\\\\\/gDwEjuS\\\\\\\/UA06vrsrg8qng2ztbeQQyZOMOTK3utrdIgUIproAxTd388dDm501RM4d8n\\\\\\\/2DnlIsM2nhkGIXgwnGA4fELG8XKZp73zhgrjPSPOHUE6B3HYbjW0rTzZRTswM006dncw0pyEkjVqfidZYTMgyFrKu5iD5PaD9FHtgxvPpwZUNC1SPUz4SXLN+nOJvhZhhlUc3NP\\\\\\\/+qiffZY5I3W4RuCH2o7l6mdol0\\\\\\\/giTa7+YA5cAERxL1b9TXmK3kiiPj04kZk71ak7TPkUwOgViHUOeGb+70oHecKdabkHuQF6R0kcOkFUfiO2b84\\\\\\\/oCBLBqMMlfxmtNAtFI2P3k1oFpq2sKU99SuBXMKStU+BV0nGp0OuhnU5ZozbOhvC5OeHinCtB5mygpp\\\\\\\/Av+GK\\\\\\\/S5r2ZEQRzAbOypMl9fm+OsoGEREJ8EyvhLw7Q4ljZ2YIvkxTPrBLE3H93xhjVn0FBnGhyITJXKAt8RYq8eGDC7ksD\\\\\\\/8NVS4+fBOMm2he+P5x4GYkiUnwogZkktRBdZ1elm0DwpfpNSszwmEsoIAnVPfDl9872rJ7Pso2XKqlaI1oD94RXtRpCTKl3Uym3EQ2+Pj3ft2+1eLVINKSc95UAm3LEIzqcZkhwV9iXkIp+VOxuYKqCdUnzN7rwMqnRfXf7HJyHZSpkPD7Ms0RA69ebHWisoTbLzgDJT17G10Z0fjQ5+onJHN2ErpKZaXXt88qSfPp2Wcpycll+qj2bun7MmvL2zq+96nNhQLmzMLPzkpjdjwIuQkZjwQ998vgNuMB3+RJJUuzY+iPUwkghEUAUwEZUD0yFW4jpa0T2f9NG1GcovQZhu1Bb0usSP1ljnztGVasVoP9X9V+J9zcbHwYfNnrqluHr+5f1Rj6+keLdSgMz2YVC61vKuYY\\\\\\\/87OvKsAT6EgSaNZboX7fnTsx8g29r9VbhLLHfNsAAGFJK8Oeih4tuN\\\\\\\/80UufUBlZLPqpw6BwWjLIrLhyZ3F5YxVmGmbRp2+6muJ7rhmW1zXaTg5XRXc8I8pQitFo8GtJqAkLWcCMWp5e+m5fgDpO4ioDXngsIa3qi3SU2zNOj2CW0b5ecSf7Pj1WKRSu\\\\\\\/WEtwU2Ift58ZaPAJBUuLse72M6RnCmrEezlK58BfrdyEtl7I0NExTjWQaaZOKrl18iKrlKDYPG6MRTH2x1BqYlpveuq7Sino387JZD+JGWHZpQGlaY8pujbCPqr+MKQfXh+isjkAtdH4PouhScYjHAMoWtjBNz0\\\\\\\/PN5QYOlkzX7U5XNTlGMVBHzhNFvzAFr+oiMdSw3LSQw5LMlQC2+sKb2DYG87a9\\\\\\\/7jGsbCZ1zF+PHiTNWehazSKYu89rGcpgL4OujQF9vLooKAHXgPJ8njRD6Bcq\\\\\\\/kTBzPeaQtojHabTNiTgiVhUQj4+L8gOp\\\\\\\/rBcBv9PfcwcUzIsKbTT1rqEAoyG+kQM+hjaEXouornSTfJWhjotDgwu7iSGLNJXvvP7oqVnzKxPi+fX6KG0cHDJhQxvfRoldXB4ZheHDsInlIL3O7IbK9SDxfbkiEkSB76wbOCXwO9Zdafz3vvuRlO82MbczsNSB9ghsx60PPfV1QUkc5eT4Cj5EP5clb\\\\\\\/0msJC7IDHJGkrBlN6DUNgXICQgE7PmIwpWGnqLdFCDffbzQvzJWSsyDYqmx1RSrPwZMuljMsOajcgi5+fpT3CM6j1UJmMh6Qh3P49BIpbPtvejG6LmC6KBV2ny91xShDa5FUauBuXd\\\\\\\/Ffgx9aklMHCMlkvIJ8lwY5\\\\\\\/lMANAcw\\\\\\\/cQCOCTDIZaDDsUScxbXR4o0ecCTHwkPzdDqjsLAlHWaYsc1QdJS\\\\\\\/YHW2iV3kU8oQVgfeJJSz\\\\\\\/PZVZ5Zags\\\\\\\/e1ypAn3HQgTqZqUEU23o7oxekZE8OVUeC1g0XEnmImpZE5gRRoBETwmlpWzRaadnXl4dT6x1I8sVtufSgMCAiXiMThdfH+j5dhht9POMQzgwwZBI9JHbgTBiqIhBaqNZlMATSOGkamQEmNTDc3rRY5uveusojnFwAghConhWVBbx5yDyWRzkOCAKi5Adrdcsnr9n8D0k3lAo5rwszNJ\\\\\\\/KS\\\\\\\/t2yM027YKdgWAgWaws+VGfnnX8Jgja7LbxAX5IJyntLyYhtWyjKgs6C9o39gMOO0AZfJUsfpd0W51Oet2VmBdmG5z8qIQLhIBYXKyisuRpzaMPPdgrKg8MecfthsnckhOUchUkkqNebQJSCgsd7eyIqXR9IRyGUluri0vzkvqT5nfH+CIXhGgaQUGAhaYPKn4+r0ENFxQVPA9GMPG9d1m+nUdS6fNL7OAfGwh6jfJG49kf4XAqBPCEHHnZ3u0m9A5Xyodg6twQatjxpr+LTHXY1C\\\\\\\/Z\\\\\\\/\\\\\\\/psz1jX\\\\\\\/evJtRcrjjA9hNJ4jwZoLgCSnIfSUJUJnHtwk0sSUcwHTcYd9WgqHyR9QRQfqx+OG9B6ndDg2DaQp5LqPc3wk+yGLchztMH0zNbT+CPh2xYoSHGQZPC6ABjlZOAR8DoCAb4ZTvn5R\\\\\\\/pmXHhZ4d2QqM3cZ2Y7et6bzafQQZS7X8AySPNJ7B4fLkBm5AK1fudJGlr0LX5+iY\\\\\\\/hIc6FtfaYKO3FxT3VwTGfUUgfJ1Um2Ut+WHOrbYTCfaN4zWabWB5wBgz+hKtEF\\\\\\\/w6vSMxRFc4JkD06RNbM92tclaLid\\\\\\\/7x7cwLVoP4hmT5n+TF+KEUOaUrFuXclx5L+v2Q+F4U8z3k+uW+216uduBnOQsk4evBVSHJLEylG61da3CiIIlGeqTro05t34QrT1M7ULMznEMsiA0h2o7n1mUrXL96lYEir6IdxD4QbR+VwRk2+0vpoWDHINQ9Z4gpYMJDLKNPV1APYQptgLvVJwPUItbxi7QFjR+iOF0xhMbjFP3jT31UNATVZUF+SrFpO60E\\\\\\\/uGZx2TjK6UmYPQi+QjYppjVKk8z1oByg2fqFrqerSBxbcXECYHmBUSLyXhQrey5Sxen1Una5I8sZqpU2YZ2fxFviBZHEVlC0joW68\\\\\\\/WySxfKh2ood55+kd3sYZa8muK56KZ93g3e9daRNRRe7+Ub2mJiEISyeBNta95pby4EF4gZnUfhQdFkiSZjda2mgE8SDxMvQow6KpH3FYsUHA==\\\u0022,\\\u0022iv\\\u0022:\\\u00226JWq566nV5Me6O\\\\\\\/G0T91Ow==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220eb2e4efb44ed635465f0e5a0c0f5eeb06dd39e50c7fa38086fbbbaa137ed233\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220BNlevG14y77fayZ8gF3k4B+Ag4yYXM6HhAJxjylG7fZaUApv\\\\\\\/Q98u1OmLOuwpBNyFkjr1lbW23lPARdvuZM9qiyxaWoyL2Q\\\\\\\/UA3LeVwVbK\\\\\\\/gVroCyOlSpoVcc5PWdyB\\\\\\\/cqexkjMwVpDMfB15d7nXa8UqZTx\\\\\\\/\\\\\\\/CMtg+Z3bQPUWy2iqemVx0nuDWBT9i7zgWybXZp3VANAnWgXq1XNSpC6PeWxFY+pzuApuW7BIxUcTIokFUcgsScWrZDjflaCQE2\\\\\\\/ymjydTgnsTDSoOgwjxayx4n2KyP10TZZdhTb3H1EEy+vjvQ6du\\\\\\\/UrYhLtjTicpX0wR2TNVZ6aU6uH8KTy9Wbq7I6JVCkN\\\\\\\/xB6\\\\\\\/6Fv8DBglArRjWHBjuvjkDvE1\\\\\\\/8voarojgYRA+R0PcluZkmSsOJJMcdK\\\\\\\/GHjeAnavj0uSTvJe49TknU7M1mIcsDnfQcbX1KshvZ2ATom3avAxWIxG61dLystMjJVoxSrL\\\\\\\/ROX8lk\\\\\\\/q4kmkmOBJgKaSRI\\\\\\\/C2v+0mpCbZ2WEP9ouzHq+Spq2\\\\\\\/CiMDthT7mti0Zu6vTrFc02KjE1ejxaX5ipW01Awd8hQxciXDOOKIWBEjuaYoSFiaec349AT02ZmjMZKd5HSCJlpG2sNWBp2C+H0Jwq5BANJbh2PUBPsnoSeH0PkuwZEnwTGfkc\\\\\\\/JcVNWNtkk7dLA9djCisVkYyQuYUCSldIH0+1BRT2lzxi7Yg3yr8hUspvPW8QfUHzt7i95hMW9yw1Q3LWqq6DAbWjbXDEt9oSHWhTpSn6auCgN6e7Wt4z6Fhfj2NcQPMumdX8SvrCjmsbScZ3k1Me1aTBOxFyfHt6GG6Jo98BKfWCPSDQuZVvkP\\\\\\\/8tyiSyorVD5ncwYU2\\\\\\\/cDZ+fbT1XA9noiq8jQf7F8t4\\\\\\\/EtPuc9tTmKZ3YIuFpH1n5l7lmjDn8jd35FhZHE6Z6jexsjzcWANW7KWHjea9hc\\\\\\\/R3bX194oyYY8\\\\\\\/1j885V0fs7oOrK5cj\\\\\\\/g9G6RLB4NOCvr6zUw6opvbgaspzNK6uidCPkEzP48GBvZoDPypeaOcDrlRjp4E7oG50YOkxu8sNw5iaoCrkVdwpHLF61gjQkgDCim05tx4u\\\\\\\/ky1E5w8yTk2woRrkYcMvDfT18WWYaHvFKhXI18naqsMAO2Gi0ZcnCavG9zh51GNxOatS86CD2B+uZV7AKrulb4oGfq2yUDluBrhkYcvmyCH9PhQE+NXaXDVrWASWQCbLJtziVgUz4pZkJtQEecmRlZcuzbmBLIkTfMvR8y9mEVwFjOXU3mAt8ZDYeI\\\\\\\/sdq4SDVUdpcWoXwzHBl3vyHmd71bXQ55YkFlhU6SBUkxYKOzdDckBd215G+DZa0RqdLvZNWKL3ANHK8ttid5d3kfuoYzTfKr2fA0\\\\\\\/GJtirVGVf1Hb646mP0nfWDUCKWsFuw3liwgXOTRuxM\\\\\\\/Qe3aigu3O4mHDEx3aelB8DUTYb5UNjCx+asHQsCNloO6clJ1qjxp8sLshhj2GeXvrVegclIqOD0qQA8nvCaMySgRUyC+6SQalazwVCFxVtrhPuQRbvG6NFty2Ea3NNZyDFf042njnYZ6aqTuyJAkJh5XP62b7pSSu2WY6SLyCLPpgFOUrZkvhayoJxMqsViHeMgfbxL205suLx3poZjTddLVATO3p6mfrvvYX0M7GjVnC86qgaOiUJy0E6Nnkl+gcPYTXpzACSVSRaedG1P1VkwXDVar58+buw4H1ckOpkkdOfKdsdkEkKGG1J5tQ8N8e15BcR0i\\\\\\\/pBA8Y9kcs2pHTjxG9IfvdZGM05VZkI7+7mnOv79SdPVS2Q5s9zff4rEYC1DeJgwisRZ5ZQ02T6XYf9sJ5sjVUH0Kt2oXA3wnh6YPbxAks8bL9p7s\\\\\\\/E\\\\\\\/KIFY+i7kD3pVzGc4nSINgUWBWsmS3XIZ9syDevv2Wo4jJLM8k9Mp\\\\\\\/uc+NrmxAx902877xPNQXUSzDcShhI8O1DiUWWLRETXouE0lRpN6k\\\\\\\/Eow\\\\\\\/4bDmqnQx0\\\\\\\/lKWAf6nuHjSdGMw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZxeEMIHGOxPjwReDt6aTcA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221520c3458acfe007ddfc803d47974fea13adcd11238dee0804c17e7bf73b4fb7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487825 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/131" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ACEzeT2UKT3ZYTtPQy41wuGeU_JPUxQTUji0L21PShg" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/131" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-s_376zftxasHIG3sJxbF6BXhCB_4m7XJ88jNyM1bwyU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/123" title="Modeling the Evolution of Female Meiotic Drive in Maize" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/149" title="Severe Fertility Effects of sheepish Sperm Caused by Failure To Enter Female Sperm Storage Organs in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487825" id="node487825" data-pisa="ggg;8/1/131" data-pisa-master="ggg;g3.117.300309" data-apath="/ggg/8/1/131.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Prediction of Multiple-Trait and Multiple-Environment Genomic Data Using Recommender Systems</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">José C. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="4">David Mota-Sanchez</span>, <span class="highwire-citation-author" data-delta="5">Fermín Estrada-González</span>, <span class="highwire-citation-author" data-delta="6">Jussi Gillberg</span>, <span class="highwire-citation-author" data-delta="7">Ravi Singh</span>, <span class="highwire-citation-author" data-delta="8">Suchismita Mondal</span> and <span class="highwire-citation-author" data-delta="9">Philomin Juliana</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">131-147;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300309 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487825" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040 Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:oamontes1@ucol.mx" class="" data-icon-position="" data-hide-link-title="0">oamontes1@ucol.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430 Jalisco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600 México City, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:oamontes1@ucol.mx" class="" data-icon-position="" data-hide-link-title="0">oamontes1@ucol.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">José C. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), 36240 Guanajuato, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BC.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BC.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David Mota-Sanchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Entomology, Michigan State University, East Lancing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BMota-Sanchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mota-Sanchez%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BMota-Sanchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Fermín Estrada-González </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040 Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ferm%C3%ADn%2BEstrada-Gonz%C3%A1lez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Estrada-Gonz%C3%A1lez%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFerm%25C3%25ADn%2BEstrada-Gonz%25C3%25A1lez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jussi Gillberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Computer Science, Aalto University, FI-00076, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jussi%2BGillberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gillberg%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJussi%2BGillberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600 México City, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Suchismita Mondal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600 México City, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suchismita%2BMondal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mondal%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuchismita%2BMondal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Philomin Juliana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600 México City, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philomin%2BJuliana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juliana%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilomin%2BJuliana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/1/131" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487825" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487825/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/131.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487825" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487825/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/131.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487825" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487825/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/1/131.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487825" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487825/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">In genomic-enabled prediction, the task of improving the accuracy of the prediction of lines in environments is difficult because the available information is generally sparse and usually has low correlations between traits. In current genomic selection, although researchers have a large amount of information and appropriate statistical models to process it, there is still limited computing efficiency to do so. Although some statistical models are usually mathematically elegant, many of them are also computationally inefficient, and they are impractical for many traits, lines, environments, and years because they need to sample from huge normal multivariate distributions. For these reasons, this study explores two recommender systems: item-based collaborative filtering (IBCF) and the matrix factorization algorithm (MF) in the context of multiple traits and multiple environments. The IBCF and MF methods were compared with two conventional methods on simulated and real data. Results of the simulated and real data sets show that the IBCF technique was slightly better in terms of prediction accuracy than the two conventional methods and the MF method when the correlation was moderately high. The IBCF technique is very attractive because it produces good predictions when there is high correlation between items (environment–trait combinations) and its implementation is computationally feasible, which can be useful for plant breeders who deal with very large data sets.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genomic-information" class="hw-term hw-article-keyword hw-article-keyword-genomic-information" rel="nofollow">genomic information</a></li><li class="kwd"><a href="/keyword/item-based-collaborative-filtering" class="hw-term hw-article-keyword hw-article-keyword-item-based-collaborative-filtering" rel="nofollow">item-based collaborative filtering</a></li><li class="kwd"><a href="/keyword/matrix-factorization" class="hw-term hw-article-keyword hw-article-keyword-matrix-factorization" rel="nofollow">matrix factorization</a></li><li class="kwd"><a href="/keyword/multi-trait" class="hw-term hw-article-keyword hw-article-keyword-multi-trait" rel="nofollow">multi-trait</a></li><li class="kwd"><a href="/keyword/genotype" class="hw-term hw-article-keyword hw-article-keyword-genotype" rel="nofollow">genotype</a></li><li class="kwd"><a href="/keyword/environment-interaction" class="hw-term hw-article-keyword hw-article-keyword-environment-interaction" rel="nofollow">environment interaction</a></li><li class="kwd"><a href="/keyword/prediction-accuracy" class="hw-term hw-article-keyword hw-article-keyword-prediction-accuracy" rel="nofollow">prediction accuracy</a></li><li class="kwd"><a href="/keyword/collaborative-filtering" class="hw-term hw-article-keyword hw-article-keyword-collaborative-filtering" rel="nofollow">collaborative filtering</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">Genomic Selection</a></li></ul><p id="p-3">Plant breeding programs aim to improve multiple traits and develop new, higher yielding varieties that are disease and drought resistant and regionally adapted to different environments and growing conditions. However, to reach that challenging goal, plant breeders have been looking for new tools to improve the selection of candidate genotypes in early stages. Genomic selection (GS) proposed by <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Meuwissen <em>et al.</em> (2001)</a> is one of the most powerful tools used by breeders and is now revolutionizing plant breeding. One of the fundamental features of GS is the use of high-density markers. Rather than seeking to identify individual loci that are significantly associated with a trait, GS uses all marker data simultaneously as performance predictors of a line, and this consequently delivers predictions that are more accurate. Selection under this approach is based on GS predictions, potentially leading to more rapid genetic gains at a lower cost. Also, GS allows marker data to be combined with phenotypic and pedigree data (when available) in an attempt to increase the accuracy of the prediction of breeding and genotypic values (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Goddard and Hayes 2007</a>).</p><p id="p-4">Although there is empirical evidence showing that GS is a powerful tool for selecting candidate genotypes in early stages, there is still the need for novel statistical models for selecting multiple correlated traits evaluated in multiple environments. Recent examples of publications on this type of models for multiple traits and multiple environments are those of <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Montesinos-López <em>et al.</em> (2016</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">2017)</a> for continuous and count data. However, although these proposed models are mathematically elegant, they are computational very limited; as the number of genotypes, environments, and traits increases, these models cannot be implemented because it is extremely difficult to sample from large multivariate normal distributions. For the abovementioned reasons, statisticians and plant scientists are open and looking for better prediction alternatives in the context of data for multiple traits and multiple environments. However, the need to increase prediction accuracy is not exclusive to GS; it is present in many other areas like marketing, e-commerce, finance, and biology. Each area has made a significant effort to increase predictions; however, in general, few cases have been successful. This reminds us that making predictions is usually very challenging since the target inference is to predict unobserved quantities at the time of the inferences.</p><p id="p-5">In this article, we will explore two methods that are very popular in recommender systems. A recommender system is a subclass of an information filtering system that seeks to predict the “rating” or “preference” that a user would give to an item (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Ricci <em>et al.</em> 2011</a>). Recommender systems have become increasingly popular in recent years, and are utilized in a variety of areas including movies, music, news, books, research articles, search queries, social tags, and products in general. In this research, we will implement the following two recommender systems methods in the context of GS: item-based collaborative filtering (IBCF) and the matrix factorization method. The first method (IBCF) belongs to the collaborative filtering techniques, which are a set of prediction methods that have been significantly successful in building recommender systems in various settings (<em>e.g.</em>, marketing, e-commerce, <em>etc</em>.). The basis of this technique is to use the known preferences of a group of users to make recommendations or predictions of the unknown preferences of other users.</p><p id="p-6">The second method (matrix factorization) belongs to the latent factor models, and tries to explain the ratings by characterizing both items and users on a reduced number of factors that approximate the rating matrix. The matrix factorization method is considered one of the most successful of the latent factor models. In its basic form, matrix factorization characterizes both items and users by vectors of factors inferred from item rating patterns. High correspondence between item and user factors leads to a recommendation. These methods have become popular in recent years by combining good scalability with predictive accuracy. In addition, they offer much flexibility for modeling various real-life situations.</p><p id="p-7">The two recommender systems mentioned above (IBCF and matrix factorization) work by building a matrix of preferences (called a rating matrix) where each row represents a user, each column represents an item, and the number at the intersection of a row and a column represents the user’s rating value. The absence of a rating score at this intersection indicates that that user has not yet rated the item. We will use simulated and real data to compute the prediction accuracy of both recommender systems (IBCF and matrix factorization) and we will compare their predictions with those produced by two conventional GS models that take into account the genotype × environment interaction term.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>IBCF</h3><p id="p-8">As mentioned above, the IBCF technique is a model-based algorithm for recommender items or products. In IBCF, similarities between items are calculated from a rating matrix (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>); based upon those similarities, a user’s preference for an item that has not been rated by a user is calculated. In general, <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> can be constructed for <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-1.gif"/></noscript></span></span> users and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-2.gif"/></noscript></span></span> items. Here, we present a step-by-step example of IBCF with four users and three items. For example, let us assume that we have the rating matrix shown in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>. With this information, we create an item-to-item similarity matrix (using the cosine similarity <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-3.gif"/></noscript></span></span> or Pearson correlation), which provides information on how similar an item is to another item. The complete cosine similarity matrix between items for this example is shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487940/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487940/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Rating matrix data set with four users (rows) and three items (columns), with the rating of each user given in an ordinal scale of three points</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/488060/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/488060/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Item-to-item similarity matrix constructed with information in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a></span><div class="sb-div caption-clear"></div></div></div><p id="p-11">We then predict each user’s ratings for items that have not previously been rated. In this example, we will calculate the rating for user <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-26.gif"/></noscript></span></span> in item <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-27.gif"/></noscript></span></span> along with user <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-28.gif"/></noscript></span></span> in item <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-29.gif"/></noscript></span></span> and for user <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-30.gif"/></noscript></span></span> in item <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-31.gif"/></noscript></span></span> Each of these predictions can be calculated using a simple weighted average to predict the rating, <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-32.gif"/></noscript></span></span> for user <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-33.gif"/></noscript></span></span> in item <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-34.gif"/></noscript></span></span> as follows (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Sarwar <em>et al.</em> 2001</a>):<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-35.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where the summation is over all other rated items (<span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-36.gif"/></noscript></span></span>) for user <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-37.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-38.gif"/></noscript></span></span> is the weight between items <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-39.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-40.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-41.gif"/></noscript></span></span> is the rating for user <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-42.gif"/></noscript></span></span> on item <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-43.gif"/></noscript></span></span> Therefore, the predicted rating for item <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-44.gif"/></noscript></span></span> for user <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-45.gif"/></noscript></span></span> will be <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-46.gif"/></noscript></span></span> while the predicted rating for item <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-47.gif"/></noscript></span></span> for user <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-48.gif"/></noscript></span></span> will be <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-49.gif"/></noscript></span></span> and the predicted rating for item <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-50.gif"/></noscript></span></span> for user <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-51.gif"/></noscript></span></span> will be <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-52.gif"/></noscript></span></span> We provide the R code for the IBCF in Appendix A1.</p><p id="p-12">It is important to point out that when calculating the similarities between users instead of between items, based upon these similarities, a user’s preference for an item that he/she has not previously rated is calculated. This method is called user-based collaborative filtering (UBCF) and is basically the same as IBCF but refers to users (lines in plant breeding). While item-based algorithms generate predictions based on similarities between items, user-based algorithms generate predictions based on similarities between users. In this article, the main focus is on the IBCF because it is more computationally efficient than UBCF when the number of items is lower than the number of users. Additionally, there is empirical evidence showing that IBCF is more accurate in predicting ratings than UBCF (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Sarwar <em>et al.</em> 2001</a>).</p></div><div id="sec-3" class="subsection"><h3>Matrix factorization (MF)</h3><p id="p-13">Matrix factorization is another algorithm for recommending items or products. Matrix factorization consists of factorizing a matrix, <em>i.e.</em>, to find two matrices, which, when multiplied, will produce the original matrix. Formally, the matrix factorization model looks for two matrices, <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-53.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-54.gif"/></noscript></span></span>, of order <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-55.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-56.gif"/></noscript></span></span> respectively, such that<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-57.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-58.gif"/></noscript></span></span> is the matrix that contains all the ratings that the users have assigned to the items and it is of order <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-59.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-60.gif"/></noscript></span></span> is the prespecified number of latent features (variables) that is lower than or equal to the <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-61.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-62.gif"/></noscript></span></span> is the estimated matrix of ratings. Each row of <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-63.gif"/></noscript></span></span> represents the strength of the associations between a user and the features, while each row of <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-64.gif"/></noscript></span></span> represents the strength of the associations between an item and the features. To obtain the prediction of a rating of item <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-65.gif"/></noscript></span></span>by user <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-66.gif"/></noscript></span></span> we calculate the dot product of the two vectors corresponding to <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-67.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-68.gif"/></noscript></span></span>:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-69.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>However, to obtain the predictions for users of the missing items, we need to estimate matrices <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-70.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-71.gif"/></noscript></span></span> with the no missing entries of the rating matrix. There are several methods for estimating matrices <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-72.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-73.gif"/></noscript></span></span> One method consists of minimizing the squared error function with the observed entries of <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-74.gif"/></noscript></span></span>:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-75.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-76.gif"/></noscript></span></span> is the regularization parameter that controls overfitting. To minimize the squared error function, we have to know in which direction we have to modify the values of <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-77.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-78.gif"/></noscript></span></span> In other words, we need to know the gradient at the current values, and therefore we differentiated the above equation with respect to these two variables separately:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-79.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-80.gif"/></noscript></span></span>Since we have the gradient, next we provide the updating rules for both <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-81.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-82.gif"/></noscript></span></span>:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-83.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-84.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-85.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-86.gif"/></noscript></span></span> is the learning rate; we usually choose a small value, for example, 0.0002. The learning rate affects the learning time, and too large a value may lead to divergence. In general, a smaller learning rate gives better performance, but the learning time is also longer. Using the updated rules, we can iteratively perform the operation until the error converges to its minimum. Before implementing this gradient descendent algorithm, we need to select a learning rate <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-87.gif"/></noscript></span></span> a regularization coefficient<span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-88.gif"/></noscript></span></span> and the required number of latent features <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-89.gif"/></noscript></span></span> Also, we need to set the starting values of matrices <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-90.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-91.gif"/></noscript></span></span> The R code for implementing this algorithm is provided in Appendix A2.</p></div><div id="sec-4" class="subsection"><h3>Multiple-environment (ME) mixed model</h3><p id="p-14">For each trait, the following univariate linear mixed model is proposed:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-92.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-93.gif"/></noscript></span></span> represents the normal response from the <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-94.gif"/></noscript></span></span>th line in the <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-95.gif"/></noscript></span></span>th environment (<span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-96.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-97.gif"/></noscript></span></span> For illustration purposes, we will use <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-98.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-99.gif"/></noscript></span></span>represents the fixed effect of the <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-100.gif"/></noscript></span></span>th environment and it is assumed as a fixed effect, <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-101.gif"/></noscript></span></span> represents the random effect of the genomic effect of the <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-102.gif"/></noscript></span></span>th line, with <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-103.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-104.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-105.gif"/></noscript></span></span> and represents the genomic relationship matrix. It was calculated using the <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">VanRaden (2008)</a> method as <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-106.gif"/></noscript></span></span> with <strong><span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-107.gif"/></noscript></span></span></strong> as the matrix of markers of order <strong><span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-108.gif"/></noscript></span></span></strong> <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-109.gif"/></noscript></span></span> is the random interaction term between the genomic effect of the <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-110.gif"/></noscript></span></span>th line and the <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-111.gif"/></noscript></span></span>th environment, where <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-112.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-113.gif"/></noscript></span></span> is a random error term associated with the <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-114.gif"/></noscript></span></span>th line in the <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-115.gif"/></noscript></span></span>th environment distributed as <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-116.gif"/></noscript></span></span> As previously mentioned, this model was used for each of the <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-117.gif"/></noscript></span></span> traits, where <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-118.gif"/></noscript></span></span> denotes the number of traits under study.</p></div><div id="sec-5" class="subsection"><h3>Multiple-trait and multiple-environment unstructured mixed model</h3><p id="p-15">To account for the correlation between traits, we stacked the information of all the <em>L</em> traits given in Equation (7). In matrix notation, the whole mixed model is equal to<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-119.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-120.gif"/></noscript></span></span> is of order <strong><span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-121.gif"/></noscript></span></span></strong> <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-122.gif"/></noscript></span></span> is of order <strong><span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-123.gif"/></noscript></span></span></strong> <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-124.gif"/></noscript></span></span> is of order <strong><span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-125.gif"/></noscript></span></span></strong> and contains the β coefficients of the environment–trait combinations, <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-126.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-127.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-128.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-129.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-130.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-131.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-132.gif"/></noscript></span></span> is of order <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-133.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-134.gif"/></noscript></span></span> is of order <strong><span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-135.gif"/></noscript></span></span></strong> Then <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-136.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-137.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-138.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-139.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-140.gif"/></noscript></span></span> is the genetic covariance matrix between traits and is assumed unstructured, and <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-141.gif"/></noscript></span></span> denotes a Kronecker product, <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-142.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-143.gif"/></noscript></span></span> is assumed as a general matrix of order <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-144.gif"/></noscript></span></span> It is important to point out that the trait × environment (T × E) interaction term is included in the fixed effect <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-145.gif"/></noscript></span></span> while the trait × genotype (T × G) interaction term is included in the random effect <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-146.gif"/></noscript></span></span>, and the three-way (T × G × E) interaction term is included in <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-147.gif"/></noscript></span></span> The errors are assumed to be correlated with the covariance defined as <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-148.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-149.gif"/></noscript></span></span> is the residual general covariance matrix between traits.</p></div><div id="sec-6" class="subsection"><h3>Proposed methods</h3><p id="p-16">In this section, we present the proposed methods for analyzing MTME data with continuous phenotypes.</p><div id="sec-7" class="subsection"><h4>Method IBCF:</h4><p id="p-17">In this method, the prediction analysis is performed with IBCF using Equation (1) directly. However, the training of the multiple-trait and multiple-envirionmnet (MTME) data first need to be scaled for each trait–environment combination given in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. When the genotypes under study are the same in all environments, we create a rectangular rating matrix where the number of rows is equal to the number of genotypes and the number of columns is equal to the resulting combinations of environments and traits; that is, if the number of traits is 4 and the number of environments is 5, then the resulting number of columns in this rating matrix should be 20 (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). However, when the data are only multiple trait (MT) or ME, the rows should be genotypes, while the number of columns should be the phenotypes measured in each environment or trait. <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a> gives an example of how the phenotypic information resulting from a MTME data set should be arranged to implement IBCF.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487856/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487856/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Phenotypic information for building the rating matrix for multiple-trait and multiple-environment data for <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/T3/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/T3/embed/mml-math-150.gif"/></noscript></span></span> genotypes, <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/T3/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/T3/embed/mml-math-151.gif"/></noscript></span></span> environments, and <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/T3/embed/mml-math-152.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/T3/embed/mml-math-152.gif"/></noscript></span></span> traits</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h4>Method MF:</h4><p id="p-19">This method implements the matrix factorization method described in Equations (2)–(6). To be correctly implemented, the data need to be placed as shown in <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>; implementation was performed using the R code given in Appendix A2.</p></div><div id="sec-9" class="subsection"><h4>Method ME mixed model:</h4><p id="p-20">Method ME consists of using Equation (7) for each trait; it is thus a typical genomic-based model with the main effects of environments, genotypes, and the genotype <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-202.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-202.gif"/></noscript></span></span> environment interaction term.</p></div><div id="sec-10" class="subsection"><h4>Method MTME mixed model:</h4><p id="p-21">Method MTME consists of using the following multitrait and multienvironment model:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-203.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-203.gif"/></noscript></span><span class="disp-formula-label">(9)</span></span>where <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-204.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-204.gif"/></noscript></span></span> represents the normal response of the <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-205.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-205.gif"/></noscript></span></span>th line in the <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-206.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-206.gif"/></noscript></span></span>th environment for trait <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-207.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-207.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-197"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-208.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-208.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-198"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-209.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-209.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-199"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-210.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-210.gif"/></noscript></span></span> represents the fixed effect of the <span class="inline-formula" id="inline-formula-200"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-211.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-211.gif"/></noscript></span></span>th trait, <span class="inline-formula" id="inline-formula-201"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-212.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-212.gif"/></noscript></span></span> is the fixed interaction term between the <span class="inline-formula" id="inline-formula-202"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-213.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-213.gif"/></noscript></span></span>th trait and the <span class="inline-formula" id="inline-formula-203"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-214.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-214.gif"/></noscript></span></span>th environment, <span class="inline-formula" id="inline-formula-204"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-215.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-215.gif"/></noscript></span></span> represents the random effect of the interaction of genotype <span class="inline-formula" id="inline-formula-205"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-216.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-216.gif"/></noscript></span></span> and the <span class="inline-formula" id="inline-formula-206"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-217.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-217.gif"/></noscript></span></span>th trait, with <span class="inline-formula" id="inline-formula-207"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-218.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-218.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-208"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-219.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-219.gif"/></noscript></span></span> is the three-way interaction of genotype <span class="inline-formula" id="inline-formula-209"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-220.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-220.gif"/></noscript></span></span> the <span class="inline-formula" id="inline-formula-210"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-221.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-221.gif"/></noscript></span></span>th environment and the <span class="inline-formula" id="inline-formula-211"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-222.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-222.gif"/></noscript></span></span>th trait, with <span class="inline-formula" id="inline-formula-212"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-223.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-223.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-213"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-224.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-224.gif"/></noscript></span></span> is a random error term associated with the <span class="inline-formula" id="inline-formula-214"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-225.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-225.gif"/></noscript></span></span>th line, the <em>i</em>th environment and the <em>l</em>th trait is distributed as <span class="inline-formula" id="inline-formula-215"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-226.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-226.gif"/></noscript></span></span></p></div></div><div id="sec-11" class="subsection"><h3>Simulated data sets</h3><p id="p-22">For testing the proposed models and methods, we simulated MTME data using Equation (8) with three environments, three traits, and 1000 genotypes, along with one replication for environment-trait-genotype combinations. We assumed that <span class="inline-formula" id="inline-formula-216"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-227.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-227.gif"/></noscript></span></span>15,8,7,12,6,7,14,9,8], where the first three beta coefficients belong to traits 1, 2, and 3 in environment 1, the next three values to the three traits in environment 2, and the last three to environment 3. Also, this first set of variance-covariance matrices were assumed to be <span class="inline-formula" id="inline-formula-217"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-228.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-228.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-218"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-229.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-229.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-219"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-230.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-230.gif"/></noscript></span></span>. These three variance-covariance matrices gave rise to a correlation of 0.85 between each pair of traits (genetic and residual) and between each pair of environments. We also assumed that the genomic relationship matrix is known, <span class="inline-formula" id="inline-formula-220"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-231.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-231.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-221"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-232.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-232.gif"/></noscript></span></span> is an identity matrix of order 1000 and <span class="inline-formula" id="inline-formula-222"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-233.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-233.gif"/></noscript></span></span> is a matrix of order <span class="inline-formula" id="inline-formula-223"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-234.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-234.gif"/></noscript></span></span> of ones. Therefore, the total number of observations was <span class="inline-formula" id="inline-formula-224"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-235.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-235.gif"/></noscript></span></span> that is, <span class="inline-formula" id="inline-formula-225"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-236.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-236.gif"/></noscript></span></span> for each trait. With these parameters, we simulated three data sets which were used for testing the prediction accuracy of the proposed models under each of the three studied scenarios. These scenarios included (S1), which generated the data as normal data using Equation (8); (S2), where the data were also generated with Equation (8) but with the error term replaced by exp[−1.25 <span class="inline-formula" id="inline-formula-226"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-237.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-237.gif"/></noscript></span></span> abs(<span class="inline-formula" id="inline-formula-227"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-238.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-238.gif"/></noscript></span></span>)], where <span class="inline-formula" id="inline-formula-228"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-239.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-239.gif"/></noscript></span></span> is exactly the same as what was generated under the first scenario, however, it now produced negative skewed data; and (S3), the last scenario, which also generated the data with Equation (8) but with the error term replaced by exp(1.25 <span class="inline-formula" id="inline-formula-229"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-240.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-240.gif"/></noscript></span></span> abs(<span class="inline-formula" id="inline-formula-230"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-241.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-241.gif"/></noscript></span></span>)) to induce positive skewed data.</p><p id="p-23">Additionally, under the same conditions as mentioned above, we studied two other sets of variance-covariance matrices: set 2, with <span class="inline-formula" id="inline-formula-231"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-242.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-242.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-232"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-243.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-243.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-233"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-244.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-244.gif"/></noscript></span></span>; and set 3, with <span class="inline-formula" id="inline-formula-234"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-245.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-245.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-235"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-246.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-246.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-236"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-247.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-247.gif"/></noscript></span></span>. The first set of variance-covariance matrices produced a pair of correlations between traits and between environments of 0.85. The second set produced a pair of correlations between traits and between environments of 0.5, while the third set of variance-covariance matrices produced a pair of correlations between traits and between environments of 0.25. These three sets of variance-covariance matrices were proposed for studying the performance of the methods proposed in the context of high correlation (first set of variance-covariances), medium correlation (second set of variance-covariances), and low correlation (third set of variance-covariances) between traits (genetic and residual) and between environments.</p></div><div id="sec-12" class="subsection"><h3>Experimental data sets</h3><p id="p-24">Here we present information on the data sets used for implementing the proposed methods. In total, three data sets were used (one of maize and two of wheat).</p><div id="sec-13" class="subsection"><h4>Data set 1:</h4><p id="p-25">A total of 250 wheat lines were extracted from a large set of 39 yield trials grown during the 2013–2014 crop season in Ciudad Obregon, Sonora, Mexico (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Rutkoski <em>et al.</em> 2016</a>). The trials were sown in mid-November and grown on beds with five and two irrigations, in addition to drip irrigation. Days to heading (HD) was recorded as the number of days from germination until 50% of spikes had emerged in each plot in the first replicate of each trial. Grain yield (GY) was the total plot grain yield measured after maturity, and plant height (PH) was recorded in centimeters.</p><p id="p-26">Image data of the yield trials were collected using a hyperspectral camera (A-series, Mirco-Hyperspec VNIR; Headwall Photonics, Fitchburg, MA) mounted on a manned aircraft, which allowed us to calculate vegetative indices for each plot. The green normalized difference vegetation index (GNDVI) was one of the traits used in this study, since it is considered a good predictor when used with pedigree and/or genomic prediction of GY in wheat due to its high heritability and genetic correlation. Trait GNDVI can also be measured remotely on large numbers of candidates for selection.</p><p id="p-27">Genotyping-by-sequencing (GBS) was used for genome-wide genotyping. Single nucleotide polymorphisms (SNPs) were called across all lines using the TASSEL GBS pipeline anchored to the genome assembly of Chinese Spring. SNP calls were extracted, and markers were filtered so that the percent of missing data did not exceed 80%. Individuals with &gt;80% missing marker data were removed, and markers were recorded as −1, 0, and 1, indicating homozygous for the minor allele, heterozygous, and homozygous for the major allele, respectively. Next, markers with &lt;0.01 minor allele frequency were removed, and missing data were imputed with the marker mean. A total of 12,083 markers remained after marker editing.</p></div><div id="sec-14" class="subsection"><h4>Data set 2:</h4><p id="p-28">A total of 309 doubled haploid maize lines were phenotyped and genotyped; they are part of the data set used by <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Crossa <em>et al.</em> (2013)</a> and <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Montesinos-López <em>et al.</em> (2016)</a>, which is comprised of a total of 504 doubled haploid lines derived by crossing and backcrossing eight inbred lines to form several full-sib families. Traits available in this data set include grain yield (GY), anthesis-silking interval (ASI), and plant height (PH); each of these traits was evaluated in three optimum rain-fed environments (Env1, Env2, and Env3). The experimental field design in each of the three environments was an α-lattice incomplete block design with two replicates. Data were preadjusted using block estimates, while environmental effects were derived from a linear model that accounted for the incomplete block design within environment and for environmental effects.</p><p id="p-29">The genomic data were obtained with GBS for each maize chromosome. The number of markers after initial filtering and the number of markers after imputation were summarized in <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Crossa <em>et al.</em> (2013)</a>. Filtering was first done by removing markers that had &gt;80% of the maize lines with missing values; then markers with a minor allele frequency ≤0.05 were deleted. The total number of GBS data were 681,257 SNPs and, after filtering for missing values and minor allele frequency, 158,281 SNPs were used for the analyses. About 20% of cells were missing in the filtered GBS information used for prediction; these missing values were replaced by their expected values before doing the prediction.</p></div><div id="sec-15" class="subsection"><h4>Data set 3:</h4><p id="p-30">This third data set was composed of 26,264 wheat lines that were planted over 3 yr (year_13_14; year_14_15, and year_15_16). In the first year (year_13_14), 7672 lines were planted, in the second year (year_14_15) 9091 lines were planted, and in the third year (year_15_16) the remaining 9501 lines were planted. The following five traits were measured on each line: days to heading (HD), days to maturity (DMT), plant height (PH), lodging, and grain yield (GY).</p></div></div><div id="sec-16" class="subsection"><h3>Assessing prediction accuracy</h3><p id="p-31">To assess prediction accuracy, 20 training–testing random partitions were implemented, as well as two types of cross-validations. The first one (CV1) mimicked a situation where lines were evaluated in some environments for the traits of interest; however, some lines were missing in all the other environments. In this cross-validation, we assigned 80% of the lines to the training set and the remaining 20% to the testing set. The second cross-validation (CV2) mimicked a situation where we wanted to predict all the information of one trait for a complete year; however, the available information was for a previous year for all the traits under study and, for the target year, we had information for several traits except the one of interest. We used the Pearson correlation to compare the predictive phenotipic values to the observed phenotype. Models with higher correlation values had better predictions.</p><p id="p-32">In the above-mentioned case, cross-validation CV1 was implemented for the simulated data sets and the first two real data sets (Data set 1 and Data set 2), while cross-validation CV2 was implemented for the real Data set 3. It is important to point out that in Data set 3, there were no common lines across years. Also, to illustrate how to predict one trait for the whole year, we assumed that one trait was missing. For example, we assumed that trait GY was missing in the year_14_15 (9091 lines missing for GY) and the information on the training set was the information from year_13_14 and year_14_15, but with GY missing in year_14_15. To predict GY for year_15_16, we used the information from year_13_14, year_14_15 and year_15_16 as the training set, but with GY missing in year_15_16 (9501 records missing for GY in year_15_16). The same was done when another trait was assumed to be missing.</p></div><div id="sec-17" class="subsection"><h3>Data availability</h3><p id="p-33">The maize and wheat phenotypic and genotypic data sets used in this study can be downloaded from the link: <a href="http://hdl.handle.net/11529/11099">http://hdl.handle.net/11529/11099</a>. The maize phenotypic and genotypic data sets are Maize_data.RData and Gg.RData, respectively. The wheat phenotypic and genotypic data sets are Data.Trigo.RData and G.Trigo.RData, respectively. The wheat large data set is Large_wheat_data.RData.</p></div></div><div class="section results" id="sec-18"><h2 class="">Results</h2><p id="p-34">The results are organized into two main sections. The first section presents the results in terms of prediction accuracy for the simulated data, while the second section gives the prediction accuracy of the four proposed methods but for the real data sets.</p><div id="sec-19" class="subsection"><h3>Prediction accuracies using simulated data</h3><p id="p-35">What follows are the prediction accuracies for each of the three studied scenarios obtained using the four proposed methods. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows the results of using a correlation between traits (genetic and residual) and between environments of 0.85 (high correlation), while <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> depicts the results when the correlation assumed between traits (genetic and residual) and between environments was 0.5 (intermediate correlation). Finally, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> displays the results when the correlation assumed between traits (genetic and residual) and between environments was 0.25 (low correlation).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/131/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulated data. Average Pearson correlation (APC) for each environment&#x2013;trait combination using the four methods under study [IBCF, matrix factorization (MF), ME, and MTME] for data simulated with a correlation of traits (genetic and residual) and correlation of environments of 0.85. S1 is the scenario under normality, S2 is the scenario under the error negative skew multiplied by 1.25, and S3 represents the scenario under the error positive skew multiplied by 1.25. The notation E1_T1 means environment 1, trait 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-901945736" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulated data. Average Pearson correlation (APC) for each environment&#x2013;trait combination using the four methods under study [IBCF, matrix factorization (MF), ME, and MTME] for data simulated with a correlation of traits (genetic and residual) and correlation of environments of 0.85. S1 is the scenario under normality, S2 is the scenario under the error negative skew multiplied by 1.25, and S3 represents the scenario under the error positive skew multiplied by 1.25. The notation E1_T1 means environment 1, trait 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/131/F1.medium.gif" width="440" height="285"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/1/131/F1.medium.gif" width="440" height="285"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/131/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/131/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487936" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-36" class="first-child">Simulated data. Average Pearson correlation (APC) for each environment–trait combination using the four methods under study [IBCF, matrix factorization (MF), ME, and MTME] for data simulated with a correlation of traits (genetic and residual) and correlation of environments of 0.85. S1 is the scenario under normality, S2 is the scenario under the error negative skew multiplied by 1.25, and S3 represents the scenario under the error positive skew multiplied by 1.25. The notation E1_T1 means environment 1, trait 1.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/131/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulated data. Average Pearson correlation (APC) for each environment-trait combination using the four methods being studied [IBCF, matrix factorization (MF), ME, and MTME] for data simulated with a correlation of traits (genetic and residual) and correlation of environments of 0.5. S1 is the scenario under normality, S2 is the scenario under the error negative skew multiplied by 1.25, and S3 represents the scenario under the error positive skew multiplied by 1.25. The notation E1_T1 means environment 1, trait 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-901945736" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulated data. Average Pearson correlation (APC) for each environment-trait combination using the four methods being studied [IBCF, matrix factorization (MF), ME, and MTME] for data simulated with a correlation of traits (genetic and residual) and correlation of environments of 0.5. S1 is the scenario under normality, S2 is the scenario under the error negative skew multiplied by 1.25, and S3 represents the scenario under the error positive skew multiplied by 1.25. The notation E1_T1 means environment 1, trait 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/131/F2.medium.gif" width="440" height="259"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/1/131/F2.medium.gif" width="440" height="259"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/131/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/131/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487922" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-37" class="first-child">Simulated data. Average Pearson correlation (APC) for each environment-trait combination using the four methods being studied [IBCF, matrix factorization (MF), ME, and MTME] for data simulated with a correlation of traits (genetic and residual) and correlation of environments of 0.5. S1 is the scenario under normality, S2 is the scenario under the error negative skew multiplied by 1.25, and S3 represents the scenario under the error positive skew multiplied by 1.25. The notation E1_T1 means environment 1, trait 1.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/131/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulated data. Average Pearson correlation (APC) for each environment-trait combination using the four methods under study [IBCF, matrix factorization (MF), ME, and MTME] for data simulated with a correlation of traits (genetic and residual) and correlation of environments of 0.25. S1 is the scenario under normality, S2 denotes the scenario under the error negative skew multiplied by 1.25, and S3 is the scenario under the error positive skew multiplied by 1.25. The notation E1_T1 means environment 1, trait 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-901945736" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Simulated data. Average Pearson correlation (APC) for each environment-trait combination using the four methods under study [IBCF, matrix factorization (MF), ME, and MTME] for data simulated with a correlation of traits (genetic and residual) and correlation of environments of 0.25. S1 is the scenario under normality, S2 denotes the scenario under the error negative skew multiplied by 1.25, and S3 is the scenario under the error positive skew multiplied by 1.25. The notation E1_T1 means environment 1, trait 1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/131/F3.medium.gif" width="440" height="274"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/1/131/F3.medium.gif" width="440" height="274"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/131/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/131/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487853" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">Simulated data. Average Pearson correlation (APC) for each environment-trait combination using the four methods under study [IBCF, matrix factorization (MF), ME, and MTME] for data simulated with a correlation of traits (genetic and residual) and correlation of environments of 0.25. S1 is the scenario under normality, S2 denotes the scenario under the error negative skew multiplied by 1.25, and S3 is the scenario under the error positive skew multiplied by 1.25. The notation E1_T1 means environment 1, trait 1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39"><a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> shows that under the first scenario (when the data were normal), method IBCF was the best in terms of prediction accuracy for all the trait-environment combinations. The second-best method was matrix factorization and the worst was method MTME. It is important to point out that, on average, method IBCF was 10.8% better than matrix factorization, 13.8% better than method ME, and 31.3% better than MTME. Under scenario S2 (considering the residual negative skew multiplied by 1.25), we observed that method MTME was the best, followed by method ME, then by IBCF, and finally matrix factorization, which was the worst. However, the gains in terms of average predictions of method MTME with regard to methods IBCF, matrix factorization, and ME were only 12.5, 15.8, and 6.2%, respectively (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Under scenario S3 (with the residual positive skew multiplied by 1.25), method IBCF was the best, followed by matrix factorization; MTME was the worst and, on average, IBCF was better than methods matrix factorization, ME, and MTME by 3.3, 39.3, and 67.1%, respectively (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). Appendix A7 gives the SE of the average Pearson correlations for each scenario and environment–trait combination of the predictions given in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, and <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>.</p><p id="p-40"><a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a> shows that in the first scenario (when the data were normal), method IBCF (collaborative filtering) was the best in terms of prediction accuracy, as shown in seven out of nine trait–environment combinations, as opposed to matrix factorization, which was the worst. Also, it must be noted that, on average, method IBCF was 14.5% better than matrix factorization, 11.3% better than ME, and 7.7% better than MTME. Under scenario S2 (with the residual negative skew multiplied by 1.25), method MTME was the best, while matrix factorization was the worst; MTME was, on average, 43.8% better than IBCF, ∼45.7% better than matrix factorization, and 29.9% better than ME (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). Under scenario S3 (with the residual positive skew multiplied by 1.25), method IBCF was the best, MTME was the worst and, on average, IBCF was better than methods matrix factorization, ME, and MTME by 3.62, 51.3, and 53.3%, respectively (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>).</p><p id="p-41"><a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> shows that under the first scenario (when the data were normal), method MTME was the best in terms of prediction accuracy, as shown by the results of all the environment–trait combinations; the second best was ME, and the worst was matrix factorization. On average, MTME was 34% better than IBCF, 44.9% better than matrix factorization, and 29.1% better than ME. Under scenario S2 (with the residual negative skew multiplied by 1.25), method MTME was the best, matrix factorization was the worst, and MTME was, on average, better than IBCF, matrix factorization, and ME by 70.9, 71.4, and 47.7%, respectively (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). Under scenario S3 (with the residual positive skew multiplied by 1.25), method IBCF was the best, followed by matrix factorization; ME was the worst and, on average, IBCF was better than methods matrix factorization, ME, and MTME by 3.0, 53.4, and 27.1%, respectively (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). In general, method IBCF was poorer when the correlation between traits (genetic and residual) and between environments was low; however, in <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a> under scenario (S3), we can see than sometimes it was better.</p></div><div id="sec-20" class="subsection"><h3>Prediction accuracy using real data sets</h3><p id="p-42">In this section, we present the results of predictions under the four proposed methods for the three real data sets. We begin by presenting the results obtained using the first data set (wheat data set), followed by the results for the second data set (maize data set), and finally, the results for the third real data set (large wheat data set).</p><p id="p-43">In <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> we present the prediction accuracies using the wheat data set under method MF with 10 different values of the regularization parameter lambda (<span class="inline-formula" id="inline-formula-237"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-248.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-248.gif"/></noscript></span></span> The table shows that the worst predictions were made without the regularization parameter, that is, when <span class="inline-formula" id="inline-formula-238"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-249.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-249.gif"/></noscript></span></span> Then it shows that when <em>λ</em> = 0.4, the prediction accuracy increased with regard to <span class="inline-formula" id="inline-formula-239"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-250.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-250.gif"/></noscript></span></span>by 15.5%, continuing to increase the value of <em>λ</em> until 1.6, the increase in the average prediction accuracy is up to 0.594 (21.88%). However, after a value of <em>λ</em> = 2.2, the prediction accuracy starts to decrease. This implies that to get the best performance of method matrix factorization, various values of the parameter <em>λ</em> need to be tested before doing the final implementation of this model.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487880/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487880/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Wheat data set 1</span><div class="sb-div caption-clear"></div></div></div><p id="p-45">In <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>, we compare the predictions of the four proposed methods using the first data set (wheat data set). <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a> indicates that IBCF without markers was the best method in terms of prediction accuracy, as shown by six out of the 12 environment-trait combinations; the second best was method matrix factorization, and the worst was method ME with and without markers. Method IBCF without markers was, on average, better than methods IBCF with markers, matrix factorization and ME without markers, ME with markers, MTME without markers, and MTME with markers by 44.6, 1.7, 99.2, 99.5, 29.9, and 36.3%, respectively. These findings of the good performance of method IBCF are related to the phenotypic correlations between traits, which are not low, as can be seen in Appendix A4. Here it is important to point out that to take into account the marker information, the genomic relationship matrix created with the marker information was used as a proxy for the phenotypic correlation between genotypes (users). Then we applied the UBCF instead of the IBCF. However, according to these results, there is evidence that using the genomic relationship matrix alone as a proxy for the phenotypic correlation between genotypes is not very reasonable.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487877/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487877/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Wheat data set 1</span><div class="sb-div caption-clear"></div></div></div><p id="p-47"><a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a> indicates that for the second data set (maize data set), the best prediction accuracies were observed under ME with markers (univariate model), followed by model MTME with markers (multivariate model), and the worst were observed under model matrix factorization without makers. In general, the predictions of ME were slightly better than those of MTME with and without markers; the second best were obtained using method IBCF with markers, while the worst were obtained using matrix factorization without markers. Methods ME and MTME were considerably better than the remaining methods. ME and MTME with markers were ∼30.9% better than method IBCF without markers, 27.8% better than IBCF with markers, 35.4% better than matrix factorization without markers, 37.5% better than ME without markers, and 37.7% better than MTME without markers. Here it is important to point out that when methods IBCF, ME, and MTME took into account the genomic relationship matrix, the predictions improved. However, the improvement of IBCF by taking into account the markers with regard to IBCF without markers was only 4.4%. In general, the predictions with this real data set were low, which is mainly explained by the low phenotypic correlations between the nine environment–trait combinations, as can be seen in Appendix A5.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487889/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487889/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Maize data set 2</span><div class="sb-div caption-clear"></div></div></div><p id="p-49"><a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a> shows the results for the third real data set (the large wheat data set), where only methods IBCF and matrix factorization were implemented. Method matrix factorization was implemented with three values of latent features<span class="inline-formula" id="inline-formula-251"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-262.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-262.gif"/></noscript></span></span> It is important to recall that in this data set there are five columns (items) in the rating matrix (GY, HD, DMT, PH, and lodging) and 36,181 rows (observations), with each row corresponding to a different wheat line. The lines were evaluated in 4 yr: 7672 were evaluated in year_13_14, 9091 were evaluated in year_14_15, 9501 were evaluated in year_15_16, and 9917 were evaluated in year_16_17. In this cross-validation scheme (CV2), we assumed trait GY was missing in the following year and predicted using as training data 1, 2, or 3 yr before. For example, when GY was predicted in year_14_15, only the information of year_13_14 and year_14_15 was used as training data, with 9091 lines missing trait GY in year_14_15 to be predicted. When 9501 lines (missing trait GY) were predicted for year_15_16, two scenarios were studied: (1) when only one previous year was used as training and denoted as GY_Year_15_16_1yb; and (2) when two previous years were used as training and denoted as GY_Year_15_16_2yb. Finally, when 9917 lines (missing trait GY) were predicted for year_16_17, three scenarios were studied: (1) when only one previous year was used as training, denoted as GY_Year_16_17_1yb; (2) when two previous years were used as training, denoted as GY_Year_16_17_2yb; and (3) when three previous years, denoted as GY_Year_16_17_3yb, were used as training sets. When the other traits were predicted, the same scenarios were studied, and the corresponding trait to be predicted was assumed to be missing.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487993/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487993/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Large wheat data set 3</span><div class="sb-div caption-clear"></div></div></div><p id="p-51">The results in <a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7</a> indicate that when we predicted GY with IBCF in all six scenarios, the average prediction in terms of the Pearson correlation was ∼0.305, with an average agreement of 30.93% among the first 2000 lines predicted and observed. Under matrix factorization, GY predictions were lower, with Pearson correlations between 0.16 (with <em>K</em> = 2 latent features) and 0.175 (with <em>K</em> = 3); however, it is interesting to point out that under the three latent features used, the average agreement among the first 2000 lines predicted and observed was also 30.93%. When the HD trait was predicted under all six scenarios, good performance was observed, as the average prediction for the Pearson correlation was 0.601 under IBCF, 0.454 under matrix factorization with <em>K</em> = 2, 0.59 under matrix factorization with <em>K</em> = 3, and 0.60 under matrix factorization with <em>K</em> = 4, with average agreement of 48.33% in the first 2000 lines predicted and observed using the two methods. Similar behavior was observed when predicting the DMT trait, as the average prediction accuracy was 0.569 (Pearson correlation) under IBCF, 0.454 under matrix factorization with <em>K</em> = 2, 0.60 with <em>K</em> = 3, and 0.61 under matrix factorization with <em>K</em> = 4. In this trait, an average agreement of 41.6% in the first 2000 lines was predicted and observed for the two methods. On the other hand, when we predicted the traits PH and lodging, the average predictions were really poor: lower than 0.1145 for PH using the two methods and lower than 0.11 for lodging using the two methods. Additionally, the average agreements in the first 2000 lines predicted and observed were 24.642 and 20.77% for traits PH and lodging, respectively. The results in <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a> show that methods IBCF and matrix factorization did a good job of predicting HD and DMT, and a reasonable job of predicting GY. However, they did a poor job of predicting PH and lodging. These contrasting results are due to the level of phenotypic correlation (see Appendix A6) since the traits that had high correlation with at least one other trait showed higher prediction accuracy and vice versa; that is, traits with low correlation with other traits were found to have low prediction accuracies. Also in <a id="xref-table-wrap-7-3" class="xref-table" href="#T7">Table 7</a> it is clear that for this data set, model IBCF was better than model matrix factorization.</p></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-52">In this article, we propose using two methods for recommender systems: IBCF and matrix factorization for predicting multiple traits of some genotypes that are missing in some environments. Our results using simulated and real data sets are interesting, since we found that when the phenotypic correlation between traits and environments is reasonably high, we can make predictions with reasonable accuracy (as shown in the simulation study presented) with both methods IBCF and matrix factorization, but method IBCF was the best. This is a very good sign, since the implementation of method IBCF is straightforward, as it is only necessary to place the information in a rectangular rating matrix where the rows are the genotypes and the columns represent the trait–environment combinations. Then, we need to scale each column by subtracting its mean and dividing by its SD. Obviously, this scaling process should be done only on the training data. Then, using the R code provided in Appendix A1, we can apply the IBCF technique. See the example in Appendix A3 to understand how to implement this algorithm.</p><p id="p-53">We also provided the R code (Appendix A2) for implementing the matrix factorization method. However, the implementation of matrix factorization, even though it produced competitive predictions, was lower than that produced by method IBCF (using the R code given in Appendix A1). For its successful application, the following should to be taken into account: (1) first obtain the tuning parameter <em>λ</em> using cross-validation, (2) choose the number of latent features (parameter <span class="inline-formula" id="inline-formula-252"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-263.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1/131/embed/mml-math-263.gif"/></noscript></span></span> also using cross-validation; and (3) obtain appropriate starting values to get convergence. Since the IBCF is extremely fast, we suggest using these values as starting values for implementing the matrix factorization method. See the example in Appendix A3 to learn how to implement the matrix factorization algorithm.</p><p id="p-54">From our results with simulated and real data sets, we found that the IBCF technique (method IBCF) is very competitive even compared to the MTME model for positive skew data. Under most scenarios where there was moderately high phenotypic correlation, the IBCF technique outperformed (in terms of prediction accuracy and implementation time) the MTME. The IBCF technique is very efficient in terms of the computational time required for its implementation and can be applied on large data sets. These techniques (IBCF and matrix factorization) can also be applied to MT or ME analysis only, as was done on the third real data set (large wheat data set). However, the placement of the data are slightly different, since in the rectangular rating matrix the rows are the genotypes but the columns are the phenotypes of traits or the phenotypes of environments. The expected predictions depend on the degree of the phenotypic correlation between traits or environments. We believe that these approaches could be very useful when the number of traits or environments, as well as the correlations between them, are large. The results obtained for the large wheat data set are very promising mostly under method IBCF, since they show that when the phenotypic correlation between traits is moderately high, we can predict the information of thousands of lines using cross-validation CV2 (in this case, 9091 lines were predicted for year_14_15, 9501 lines for year_15_16, and 9917 lines for year_17_17). The IBCF technique also has the advantage that the implementation time is fast and allows parallel computing in the event of larger data sets.</p><p id="p-55">A disadvantage of using IBCF is that it works with phenotypic correlations between items (trait–environment combinations) or users (genotypes), and when we use the genomic relationship matrix as a proxy for the phenotypic correlation between genotypes, this does not really represent the phenotypic correlation between genotypes and can produce poor predictions. This was observed in our results where only in the second real data set using the genomic relationship matrix produced a little improvement in prediction accuracy when the genomic relationship matrix was used. When applied to the first real data set (wheat data set), the predictions obtained were poor. Also, it is important to point out that due to the nature of both models (IBCF and matrix factorization), they not allow for inclusion of the genomic relationship matrix. For this reason, we used the genomic relationship matrix as a proxy for the phenotypic correlation between genotypes (users) to be able to implement the UBCF technique, but we did not find an efficient way to incorporate this information under the matrix factorization method. However, according to the results obtained, we have evidence that using the genomic relationship matrix alone as proxy for the phenotypic correlation between genotypes (users) using the UBCF is not reasonable. For this reason, we believe that to take into account the genomic relationship matrix in these models (IBCF and matrix factorization), some novel modifications need to be made to these recommender systems methods to be able to take advantage of all the genomic information.</p><p id="p-56">We must point out that the process for scaling each column is very important in order to have all the columns (items = trait–environment combinations) on the same scale, since methods IBCF and matrix factorization were originally proposed for ordinal items (example 1 = totally disagree, 2 = disagree, 3 = neutral, 4 = agree, 5 = totally agree). For this reason, the user must remember to scale each column of the rating matrix. We encourage the use of both IBCF and matrix factorization for ordinal or binary phenotypic data, given that IBCF was originally proposed in the context of ordinal data. However, according to our review of these two statistical techniques (methods IBCF and matrix factorization), they had only been implemented using the cosine and Pearson correlation. However, we believe that predictions can be improved using the polychoric correlation for ordinal data or the tetrachoric correlation for binary data. Also, since we usually use 0, 1, or 2 to denote the information of each marker in a line in its position in the genome with SNPs, we believe that when the correlation between the columns of SNPs is reasonably large, this method can be used successfully for imputation of missing markers.</p><p id="p-57">It is important to point out that both methods can be implemented using the R code given in Appendices A1 and A2. Although there are some R packages that can be used to implement IBCF and matrix factorization, we need to be careful when using these packages, since some of them were developed exclusively for binary or ordinal data; thus, when we want to use them for scaled continuous data, they do not work because the maximization process they use is only for positive values of the input provided. One package that has such restrictions is the package rrecsys (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Coba <em>et al.</em> 2017</a>).</p><p id="p-58">Finally, one of the main advantages of using IBCF is that its implementation is very fast and can be used on large data sets without problems. In the case of data sets with very large numbers of columns, parallel processing is possible because its computation is not iterative. However, if the correlation between the columns of the created rating matrix is poor (low correlation), this technique produces poor predictions; the larger the correlation between the columns of the rating matrix, the better the sample predictions will be.</p><div id="sec-22" class="subsection"><h3>Conclusions</h3><p id="p-59">It is necessary to improve the accuracy of the prediction models used in GS. For this reason, we explored two recommender systems techniques: IBCF and matrix factorization. Both are very popular in the context of online marketing to recommend products or items. The IBCF technique is attributed to Amazon.com (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Linden <em>et al.</em> 2003</a>), which implemented it as its recommender system. It works based on the similarity between items, calculated using people’s ratings of those items (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Sarwar <em>et al.</em> 2001</a>). The IBCF uses the items most similar to a user’s already rated items to generate a list of predictions (recommendations). Usually the predictions are a weighted sum or linear regression. From our results using real and simulated data, we obtained empirical evidence showing that both methods, IBCF and matrix factorization, work well for predicting phenotypes that are missing in some traits and environments, but the IBCF was the best. Both methods are very efficient if, and only if, the correlation between traits and between environments is moderately high. However, when the correlation between traits and between environments is low, the performance of both techniques in terms of prediction accuracy is poor. We believe that more empirical evidence is required to be able to consider these techniques as an attractive alternative for whole-genome prediction. Additionally, we observed that the implementation of the IBCF technique is fast and can be used for large data sets. It produced reasonable predictions of thousands of lines for a trait that needs to be predicted next year.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgements</h2><p id="p-60">The authors of this article are grateful to all field and lab technician of CIMMYT for generating the data used in this study.</p></div><div class="section app" id="app-1"><h2 class="">Appendix A1</h2><div class="section" id="sec-23"><h3>R Code for Implementing Item-Based Collaborative Filtering</h3><p id="p-62">rec_IBCF=function(geno_no)</p><p id="p-63">{ genoRatings=ratings[geno_no,]</p><p id="p-64">non_rated_items=list()</p><p id="p-65">rated_items=list()</p><p id="p-66">for(i in 2:ncol(genoRatings)){</p><p id="p-67">if(is.na(genoRatings[,i]))</p><p id="p-68">{non_rated_items=c(non_rated_items,colnames(genoRatings)[i])}</p><p id="p-69">else</p><p id="p-70">{rated_items=c(rated_items,colnames(genoRatings)[i])}}</p><p id="p-71">non_rated_items = unlist(non_rated_items)</p><p id="p-72">rated_items = unlist(rated_items)</p><p id="p-73">#create weighted similarity for all the rated items by geno</p><p id="p-74">non_rated_pred_score = list()</p><p id="p-75">for(j in 1:length(non_rated_items)){</p><p id="p-76">temp_sum=0</p><p id="p-77">df =item_sim[which(rownames(item_sim)==non_rated_items[j]),]</p><p id="p-78">for(i in 1:length(rated_items)){</p><p id="p-79">temp_sum = temp_sum+ abs(df[which(names(df)==rated_items[i])])}</p><p id="p-80">weight_mat = df*ratings[geno_no,2:(length(genoRatings))]</p><p id="p-81">non_rated_pred_score = c(non_rated_pred_score,rowSums(weight_mat,na.rm=T)/temp_sum)}</p><p id="p-82">pred_rat_mat = as.data.frame(non_rated_pred_score)</p><p id="p-83">names(pred_rat_mat) = non_rated_items</p><p id="p-84">for(k in 1:ncol(pred_rat_mat)){</p><p id="p-85">ratings[geno_no,][which(names(ratings[geno_no,]) == names(pred_rat_mat)[k])]=pred_rat_mat[1,k]}</p><p id="p-86">return(ratings[geno_no,])}</p></div></div><div class="section app" id="app-2"><h2 class="">Appendix A2</h2><div class="section" id="sec-24"><h3>R Code for Implementing the Matrix-Factorization Algorithm with Regularization</h3><p id="p-87">rec_Mat_fact&lt;-function(R,P,Q,K,maxiter,alpha,lambda,tol)</p><p id="p-88">{</p><p id="p-89">Q=t(Q)</p><p id="p-90">for(s in 1:maxiter)</p><p id="p-91">{for(i in 1:dim(R)[1]) {for(j in 1:dim(R)[2]){</p><p id="p-92">if(!is.na(R[i,j]))</p><p id="p-93">{e_ij = R[i,j] - crossprod(P[i,],Q[,j])</p><p id="p-94">for(k in 1:K)</p><p id="p-95">{</p><p id="p-96">P[i,k] = P[i,k] + alpha * (2 * e_ij * Q[k,j] - lambda * P[i,k])</p><p id="p-97">Q[k,j] = Q[k,j] + alpha * (2 * e_ij * P[i,k] - lambda * Q[k,j])</p><p id="p-98">}}}}</p><p id="p-99">sse=0</p><p id="p-100">for(i in 1:dim(R)[1])</p><p id="p-101">{for(j in 1:dim(R)[2]){</p><p id="p-102">if(!is.na(R[i,j])){</p><p id="p-103">sse = sse + (R[i,j] - crossprod(P[i,],Q[,j]))^2</p><p id="p-104">for(k in 1:K)</p><p id="p-105">{sse = sse + lambda/2*(P[i,k]^2+Q[k,j]^2)}}}}</p><p id="p-106">if(sse&lt;tol)</p><p id="p-107">break }</p><p id="p-108">list(P=P,Q=t(Q))}</p></div></div><div class="section app" id="app-3"><h2 class="">Appendix A3</h2><div class="section" id="sec-25"><h3>Example for Implementing the IBCF and Matrix-Factorization Algorithms</h3><p id="p-109">library(mvtnorm)</p><p id="p-110">library(lsa)</p><p id="p-111">########Generating data from 15 lines (users) and traits (items) ##############</p><p id="p-112">set.seed(2)</p><p id="p-113">A=matrix(0.85,ncol=5,nrow=5)</p><p id="p-114">diag(A)=1</p><p id="p-115">R=rmvnorm(15,mean=rep(0,5),sigma=A)</p><p id="p-116">########Selecting 20% as missing data######################################</p><p id="p-117">pos.missing=sample(1:75,15,replace=F)</p><p id="p-118">pos.missing</p><p id="p-119">R_mis=R</p><p id="p-120">R_mis[pos.missing]=NA</p><p id="p-121">############Loading the IBCF and the Matrix factorization methods###############</p><p id="p-122">source(“Colab_Filtering_IBCF.r”) ###Code given in Appendix A1</p><p id="p-123">source(“Colab_Filtering_Mat_Fact_Final.r”) ###Code given in Appendix A2</p><p id="p-124">############Predictions using the IBCF method#############################</p><p id="p-125">ratings=data.frame(cbind(c(1:nrow(R_mis)),R_mis))</p><p id="p-126">colnames(ratings)=c(“geno_no”,”I1”, “I2”, “I3”, “I4”, “I5”)</p><p id="p-127">ratings</p><p id="p-128">nrow_ratings=nrow(ratings)</p><p id="p-129">ncol_ratings=ncol(ratings)</p><p id="p-130">x=R_mis</p><p id="p-131">x[is.na(x)] = 0</p><p id="p-132">item_sim=cosine(as.matrix((x)))</p><p id="p-133">colnames(item_sim)=c(“I1”, “I2”, “I3”, “I4”, “I5”)</p><p id="p-134">rownames(item_sim)=c(“I1”, “I2”, “I3”, “I4”, “I5”)</p><p id="p-135">item_sim</p><p id="p-136">R.pred=data.matrix(ratings)</p><p id="p-137">##############Row positions with no missing values########################</p><p id="p-138">pos.wn.Na=c(as.numeric(noquote(rownames(na.omit(data.frame(ratings))))))</p><p id="p-139">R.pred=data.matrix(ratings)</p><p id="p-140">##############Row positions with no missing values########################</p><p id="p-141">pos.wn.Na=c(as.numeric(noquote(rownames(na.omit(data.frame(ratings))))))</p><p id="p-142">###############All rows to use###########################################</p><p id="p-143">pos.used=c(1:nrow_ratings)</p><p id="p-144">##############Row positions with missing values########################</p><p id="p-145">pos.final=pos.used[-pos.wn.Na]</p><p id="p-146">for (i in 1:length(pos.final))</p><p id="p-147">{</p><p id="p-148">pos=pos.final[i]</p><p id="p-149">R.pred[pos,c(2:ncol_ratings)]=c(data.matrix(rec_IBCF((pos))))[2:ncol_ratings]</p><p id="p-150">}</p><p id="p-151">R.pred1=R.pred[,-1]</p><p id="p-152">#R.pred1[pos.missing]</p><p id="p-153">#R.pred_IBCF</p><p id="p-154">setwd(“C:\\Multi-trait analysis Crossa”)</p><p id="p-155">library(mvtnorm)</p><p id="p-156">library(lsa)</p><p id="p-157">########Generating data from 15 lines (users) and traits (items) ##############</p><p id="p-158">A=matrix(0.85,ncol=5,nrow=5)</p><p id="p-159">diag(A)=1</p><p id="p-160">R=rmvnorm(15,mean=rep(0,5),sigma=A)</p><p id="p-161">R</p><p id="p-162">########Selecting 20% as missing data######################################</p><p id="p-163">pos.missing=sample(1:75,15,replace=F)</p><p id="p-164">pos.missing</p><p id="p-165">R_mis=R</p><p id="p-166">R_mis[pos.missing]=NA</p><p id="p-167">R_mis</p><p id="p-168">############Loading the IBCF and the Matrix factorization methods###############</p><p id="p-169">source(“Colab_Filtering_IBCF.r”) ###Code given in Appendix A1</p><p id="p-170">source(“Colab_Filtering_Mat_Fact_Final.r”) ###Code given in Appendix A2</p><p id="p-171">############Predictions using the IBCF method###############################</p><p id="p-172">ratings=data.frame(cbind(c(1:nrow(R_mis)),R_mis))</p><p id="p-173">colnames(ratings)=c(“geno_no”,”I1”, “I2”, “I3”, “I4”, “I5”)</p><p id="p-174">ratings</p><p id="p-175">nrow_ratings=nrow(ratings)</p><p id="p-176">ncol_ratings=ncol(ratings)</p><p id="p-177">x=R_mis</p><p id="p-178">x[is.na(x)] = 0</p><p id="p-179">item_sim=cosine(as.matrix((x)))</p><p id="p-180">colnames(item_sim)=c(“I1”, “I2”, “I3”, “I4”, “I5”)</p><p id="p-181">rownames(item_sim)=c(“I1”, “I2”, “I3”, “I4”, “I5”)</p><p id="p-182">item_sim</p><p id="p-183">R.pred=data.matrix(ratings)</p><p id="p-184">##############Row positions with no missing values########################</p><p id="p-185">pos.wn.Na=c(as.numeric(noquote(rownames(na.omit(data.frame(ratings))))))</p><p id="p-186">pos.wn.Na</p><p id="p-187">###############All rows to use############################################</p><p id="p-188">pos.used=c(1:nrow_ratings)</p><p id="p-189">##############Row positions with missing values########################</p><p id="p-190">pos.final=pos.used[-pos.wn.Na]</p><p id="p-191">for (i in 1:length(pos.final))</p><p id="p-192">{</p><p id="p-193">pos=pos.final[i]</p><p id="p-194">R.pred[pos,c(2:ncol_ratings)]=c(data.matrix(rec_IBCF((pos))))[2:ncol_ratings]</p><p id="p-195">}</p><p id="p-196">R.pred_IBCF=R.pred[,-1]</p><p id="p-197">R.pred_IBCF</p><p id="p-198">############Predictions using the matrix factorization method (MF)##############</p><p id="p-199">K = 3 #####Number of latent features</p><p id="p-200">n = dim(R_mis)[1]</p><p id="p-201">L = dim(R_mis)[2]</p><p id="p-202">P = matrix(rnorm(n*K,0,1),nc=K) ####Starting values of P</p><p id="p-203">Q = matrix(rnorm(L*K,0,1),nc=K) ####Starting values of Q</p><p id="p-204">Mat_fact=rec_Mat_fact(R=R_mis,P,Q,K,maxiter=1e3,alpha = 2e-3,lambda=21e-3,tol=1e-2)</p><p id="p-205">#Predictions</p><p id="p-206">output=Mat_fact$P%*%t(Mat_fact$Q)</p><p id="p-207">R_pred_Mat_Fact=R_mis</p><p id="p-208">R_pred_Mat_Fact[pos.missing]=output[pos.missing]</p><p id="p-209">R_pred_Mat_Fact</p></div></div><div class="section app" id="app-4"><h2 class="">Appendix A4</h2><div class="section" id="sec-26"><h3>Phenotypic cosine correlations for environment-trait combinations in data set 1 (wheat data set)</h3><div id="T8" class="table pos-anchor"><div class="table-inline"><table frame="border" id="table-8"><col width="11.45%" span="1" id="column-70"/><col width="6.44%" span="1" id="column-71"/><col width="6.44%" span="1" id="column-72"/><col width="7.21%" span="1" id="column-73"/><col width="7.24%" span="1" id="column-74"/><col width="7.24%" span="1" id="column-75"/><col width="9.88%" span="1" id="column-76"/><col width="7.24%" span="1" id="column-77"/><col width="7.24%" span="1" id="column-78"/><col width="7.24%" span="1" id="column-79"/><col width="7.24%" span="1" id="column-80"/><col width="7.24%" span="1" id="column-81"/><col width="7.9%" span="1" id="column-82"/><thead id="thead-8"><tr id="tr-108"><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-109" class="table-center table-vtop">Environments</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-110" class="table-center table-vtop">Bed5IR_ HD</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-111" class="table-center table-vtop">Bed5IR_ GNDVI</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-112" class="table-center table-vtop">Bed5IR_ GY</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-113" class="table-center table-vtop">Bed5IR_ PH</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-114" class="table-center table-vtop">Bed2IR_ HD</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-115" class="table-center table-vtop">Bed2IR_ GNDVI</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-116" class="table-center table-vtop">Bed2IR_ GY</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-117" class="table-center table-vtop">Bed2IR_ PH</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-118" class="table-center table-vtop">Drip_ HD</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-119" class="table-center table-vtop">Drip_G_ GNDVI</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-120" class="table-center table-vtop">Drip_ GY</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-121" class="table-center table-vtop">Drip_ PH</th></tr></thead><tbody id="tbody-8"><tr id="tr-109"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-998" class="table-left table-vtop">Bed5IR_HD</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-999" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1000" class="table-center table-vtop">0.698</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1001" class="table-center table-vtop">−0.271</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1002" class="table-center table-vtop">−0.120</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1003" class="table-center table-vtop">0.564</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1004" class="table-center table-vtop">0.461</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1005" class="table-center table-vtop">0.207</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1006" class="table-center table-vtop">0.341</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1007" class="table-center table-vtop">0.486</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1008" class="table-center table-vtop">0.192</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1009" class="table-center table-vtop">−0.036</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1010" class="table-center table-vtop">−0.223</td></tr><tr id="tr-110"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1011" class="table-left table-vtop">Bed5IR_GNDVI</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1012" class="table-center table-vtop">0.698</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1013" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1014" class="table-center table-vtop">−0.204</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1015" class="table-center table-vtop">−0.035</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1016" class="table-center table-vtop">0.470</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1017" class="table-center table-vtop">0.527</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1018" class="table-center table-vtop">0.131</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1019" class="table-center table-vtop">0.259</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1020" class="table-center table-vtop">0.398</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1021" class="table-center table-vtop">0.286</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1022" class="table-center table-vtop">−0.032</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1023" class="table-center table-vtop">−0.148</td></tr><tr id="tr-111"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1024" class="table-left table-vtop">Bed5IR_GY</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1025" class="table-center table-vtop">−0.271</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1026" class="table-center table-vtop">−0.204</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1027" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1028" class="table-center table-vtop">0.577</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1029" class="table-center table-vtop">−0.149</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1030" class="table-center table-vtop">−0.058</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1031" class="table-center table-vtop">0.207</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1032" class="table-center table-vtop">0.138</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1033" class="table-center table-vtop">−0.324</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1034" class="table-center table-vtop">−0.385</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1035" class="table-center table-vtop">0.481</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1036" class="table-center table-vtop">0.187</td></tr><tr id="tr-112"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1037" class="table-left table-vtop">Bed5IR_PH</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1038" class="table-center table-vtop">−0.120</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1039" class="table-center table-vtop">−0.035</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1040" class="table-center table-vtop">0.577</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1041" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1042" class="table-center table-vtop">−0.052</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1043" class="table-center table-vtop">0.083</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1044" class="table-center table-vtop">0.247</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1045" class="table-center table-vtop">0.318</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1046" class="table-center table-vtop">−0.157</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1047" class="table-center table-vtop">−0.262</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1048" class="table-center table-vtop">0.397</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1049" class="table-center table-vtop">0.236</td></tr><tr id="tr-113"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1050" class="table-left table-vtop">Bed2IR_HD</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1051" class="table-center table-vtop">0.564</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1052" class="table-center table-vtop">0.470</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1053" class="table-center table-vtop">−0.149</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1054" class="table-center table-vtop">−0.052</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1055" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1056" class="table-center table-vtop">0.787</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1057" class="table-center table-vtop">0.386</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1058" class="table-center table-vtop">0.628</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1059" class="table-center table-vtop">0.505</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1060" class="table-center table-vtop">0.201</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1061" class="table-center table-vtop">−0.020</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1062" class="table-center table-vtop">−0.268</td></tr><tr id="tr-114"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1063" class="table-left table-vtop">Bed2IR_GNDVI</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1064" class="table-center table-vtop">0.461</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1065" class="table-center table-vtop">0.527</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1066" class="table-center table-vtop">−0.058</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1067" class="table-center table-vtop">0.083</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1068" class="table-center table-vtop">0.787</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1069" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1070" class="table-center table-vtop">0.493</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1071" class="table-center table-vtop">0.640</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1072" class="table-center table-vtop">0.401</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1073" class="table-center table-vtop">0.166</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1074" class="table-center table-vtop">0.100</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1075" class="table-center table-vtop">−0.144</td></tr><tr id="tr-115"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1076" class="table-left table-vtop">Bed2IR_GY</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1077" class="table-center table-vtop">0.207</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1078" class="table-center table-vtop">0.131</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1079" class="table-center table-vtop">0.207</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1080" class="table-center table-vtop">0.247</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1081" class="table-center table-vtop">0.386</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1082" class="table-center table-vtop">0.493</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1083" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1084" class="table-center table-vtop">0.534</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1085" class="table-center table-vtop">0.147</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1086" class="table-center table-vtop">−0.034</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1087" class="table-center table-vtop">0.192</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1088" class="table-center table-vtop">−0.085</td></tr><tr id="tr-116"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1089" class="table-left table-vtop">Bed2IR_PH</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1090" class="table-center table-vtop">0.341</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1091" class="table-center table-vtop">0.259</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1092" class="table-center table-vtop">0.138</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1093" class="table-center table-vtop">0.318</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1094" class="table-center table-vtop">0.628</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1095" class="table-center table-vtop">0.640</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1096" class="table-center table-vtop">0.534</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1097" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1098" class="table-center table-vtop">0.213</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1099" class="table-center table-vtop">−0.089</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1100" class="table-center table-vtop">0.277</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1101" class="table-center table-vtop">0.013</td></tr><tr id="tr-117"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1102" class="table-left table-vtop">Drip_HD</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1103" class="table-center table-vtop">0.486</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1104" class="table-center table-vtop">0.398</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1105" class="table-center table-vtop">−0.324</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1106" class="table-center table-vtop">−0.157</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1107" class="table-center table-vtop">0.505</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1108" class="table-center table-vtop">0.401</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1109" class="table-center table-vtop">0.147</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1110" class="table-center table-vtop">0.213</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1111" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1112" class="table-center table-vtop">0.551</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1113" class="table-center table-vtop">−0.401</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1114" class="table-center table-vtop">−0.535</td></tr><tr id="tr-118"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1115" class="table-left table-vtop">Drip_G_NDVI</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1116" class="table-center table-vtop">0.192</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1117" class="table-center table-vtop">0.286</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1118" class="table-center table-vtop">−0.385</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1119" class="table-center table-vtop">−0.262</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1120" class="table-center table-vtop">0.201</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1121" class="table-center table-vtop">0.166</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1122" class="table-center table-vtop">−0.034</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1123" class="table-center table-vtop">−0.089</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1124" class="table-center table-vtop">0.551</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1125" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1126" class="table-center table-vtop">−0.575</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1127" class="table-center table-vtop">−0.298</td></tr><tr id="tr-119"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1128" class="table-left table-vtop">Drip_GY</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1129" class="table-center table-vtop">−0.036</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1130" class="table-center table-vtop">−0.032</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1131" class="table-center table-vtop">0.481</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1132" class="table-center table-vtop">0.397</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1133" class="table-center table-vtop">−0.020</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1134" class="table-center table-vtop">0.100</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1135" class="table-center table-vtop">0.192</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1136" class="table-center table-vtop">0.277</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1137" class="table-center table-vtop">−0.401</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1138" class="table-center table-vtop">−0.575</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1139" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1140" class="table-center table-vtop">0.460</td></tr><tr id="tr-120"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1141" class="table-left table-vtop">Drip_PH</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1142" class="table-center table-vtop">−0.223</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1143" class="table-center table-vtop">−0.148</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1144" class="table-center table-vtop">0.187</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1145" class="table-center table-vtop">0.236</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1146" class="table-center table-vtop">−0.268</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1147" class="table-center table-vtop">−0.144</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1148" class="table-center table-vtop">−0.085</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1149" class="table-center table-vtop">0.013</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1150" class="table-center table-vtop">−0.535</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1151" class="table-center table-vtop">−0.298</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1152" class="table-center table-vtop">0.460</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1153" class="table-center table-vtop">1.000</td></tr></tbody></table><div class="table-foot"></div></div></div></div></div><div class="section app" id="app-5"><h2 class="">Appendix A5</h2><div class="section" id="sec-27"><h3>Phenotypic cosine correlations for environment-trait combinations in data set 2 (maize data set)</h3><div id="T9" class="table pos-anchor"><div class="table-inline"><table frame="border" id="table-9"><col width="11.39%" span="1" id="column-83"/><col width="11%" span="1" id="column-84"/><col width="9.78%" span="1" id="column-85"/><col width="9.1%" span="1" id="column-86"/><col width="11.33%" span="1" id="column-87"/><col width="10.11%" span="1" id="column-88"/><col width="9.43%" span="1" id="column-89"/><col width="10.33%" span="1" id="column-90"/><col width="9.1%" span="1" id="column-91"/><col width="8.43%" span="1" id="column-92"/><thead id="thead-9"><tr id="tr-121"><th valign="top" align="left" scope="col" rowspan="1" colspan="1" id="th-122" class="table-left table-vtop"></th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-123" class="table-center table-vtop">EBU_GY</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-124" class="table-center table-vtop">EBU_ASI</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-125" class="table-center table-vtop">EBU_PH</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-126" class="table-center table-vtop">KAK_GY</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-127" class="table-center table-vtop">KAK_ASI</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-128" class="table-center table-vtop">KAK_PH</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-129" class="table-center table-vtop">KTI_GY</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-130" class="table-center table-vtop">KTI_ASI</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-131" class="table-center table-vtop">KTI_PH</th></tr></thead><tbody id="tbody-9"><tr id="tr-122"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1154" class="table-left table-vtop">EBU_GY</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1155" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1156" class="table-center table-vtop">−0.146</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1157" class="table-center table-vtop">0.197</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1158" class="table-center table-vtop">0.055</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1159" class="table-center table-vtop">−0.029</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1160" class="table-center table-vtop">0.236</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1161" class="table-center table-vtop">0.120</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1162" class="table-center table-vtop">−0.049</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1163" class="table-center table-vtop">0.107</td></tr><tr id="tr-123"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1164" class="table-left table-vtop">EBU_ASI</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1165" class="table-center table-vtop">−0.146</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1166" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1167" class="table-center table-vtop">−0.134</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1168" class="table-center table-vtop">0.097</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1169" class="table-center table-vtop">0.133</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1170" class="table-center table-vtop">0.049</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1171" class="table-center table-vtop">−0.013</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1172" class="table-center table-vtop">0.045</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1173" class="table-center table-vtop">−0.057</td></tr><tr id="tr-124"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1174" class="table-left table-vtop">EBU_PH</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1175" class="table-center table-vtop">0.197</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1176" class="table-center table-vtop">−0.134</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1177" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1178" class="table-center table-vtop">−0.094</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1179" class="table-center table-vtop">0.100</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1180" class="table-center table-vtop">0.068</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1181" class="table-center table-vtop">0.043</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1182" class="table-center table-vtop">0.034</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1183" class="table-center table-vtop">0.010</td></tr><tr id="tr-125"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1184" class="table-left table-vtop">KAK_GY</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1185" class="table-center table-vtop">0.055</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1186" class="table-center table-vtop">0.097</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1187" class="table-center table-vtop">−0.094</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1188" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1189" class="table-center table-vtop">−0.078</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1190" class="table-center table-vtop">0.351</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1191" class="table-center table-vtop">0.110</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1192" class="table-center table-vtop">0.047</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1193" class="table-center table-vtop">−0.013</td></tr><tr id="tr-126"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1194" class="table-left table-vtop">KAK_ASI</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1195" class="table-center table-vtop">−0.029</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1196" class="table-center table-vtop">0.133</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1197" class="table-center table-vtop">0.100</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1198" class="table-center table-vtop">−0.078</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1199" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1200" class="table-center table-vtop">−0.138</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1201" class="table-center table-vtop">0.020</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1202" class="table-center table-vtop">0.188</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1203" class="table-center table-vtop">−0.036</td></tr><tr id="tr-127"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1204" class="table-left table-vtop">KAK_PH</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1205" class="table-center table-vtop">0.236</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1206" class="table-center table-vtop">0.049</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1207" class="table-center table-vtop">0.068</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1208" class="table-center table-vtop">0.351</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1209" class="table-center table-vtop">−0.138</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1210" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1211" class="table-center table-vtop">0.016</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1212" class="table-center table-vtop">0.057</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1213" class="table-center table-vtop">0.080</td></tr><tr id="tr-128"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1214" class="table-left table-vtop">KTI_GY</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1215" class="table-center table-vtop">0.120</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1216" class="table-center table-vtop">−0.013</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1217" class="table-center table-vtop">0.043</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1218" class="table-center table-vtop">0.110</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1219" class="table-center table-vtop">0.020</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1220" class="table-center table-vtop">0.016</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1221" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1222" class="table-center table-vtop">−0.263</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1223" class="table-center table-vtop">0.429</td></tr><tr id="tr-129"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1224" class="table-left table-vtop">KTI_ASI</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1225" class="table-center table-vtop">−0.049</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1226" class="table-center table-vtop">0.045</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1227" class="table-center table-vtop">0.034</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1228" class="table-center table-vtop">0.047</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1229" class="table-center table-vtop">0.188</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1230" class="table-center table-vtop">0.057</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1231" class="table-center table-vtop">−0.263</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1232" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1233" class="table-center table-vtop">−0.318</td></tr><tr id="tr-130"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1234" class="table-left table-vtop">KTI_PH</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1235" class="table-center table-vtop">0.107</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1236" class="table-center table-vtop">−0.057</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1237" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1238" class="table-center table-vtop">−0.013</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1239" class="table-center table-vtop">−0.036</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1240" class="table-center table-vtop">0.080</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1241" class="table-center table-vtop">0.429</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1242" class="table-center table-vtop">−0.318</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1243" class="table-center table-vtop">1.000</td></tr></tbody></table><div class="table-foot"></div></div></div></div></div><div class="section app" id="app-6"><h2 class="">Appendix A6</h2><div class="section" id="sec-28"><h3>Phenotypic cosine correlations for environment-trait combinations in data set 3 (large wheat data set)</h3><div id="T10" class="table pos-anchor"><div class="table-inline"><table frame="border" id="table-10"><col width="16.7%" span="1" id="column-93"/><col width="16.66%" span="1" id="column-94"/><col width="16.66%" span="1" id="column-95"/><col width="16.66%" span="1" id="column-96"/><col width="16.66%" span="1" id="column-97"/><col width="16.66%" span="1" id="column-98"/><thead id="thead-10"><tr id="tr-131"><th valign="top" align="left" scope="col" rowspan="1" colspan="1" id="th-132" class="table-left table-vtop"></th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-133" class="table-center table-vtop">HD</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-134" class="table-center table-vtop">DMT</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-135" class="table-center table-vtop">PH</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-136" class="table-center table-vtop">Lodging</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-137" class="table-center table-vtop">GY</th></tr></thead><tbody id="tbody-10"><tr id="tr-132"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1244" class="table-left table-vtop">HD</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1245" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1246" class="table-center table-vtop">0.760</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1247" class="table-center table-vtop">0.228</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1248" class="table-center table-vtop">−0.069</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1249" class="table-center table-vtop">0.401</td></tr><tr id="tr-133"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1250" class="table-left table-vtop">DMT</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1251" class="table-center table-vtop">0.760</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1252" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1253" class="table-center table-vtop">0.251</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1254" class="table-center table-vtop">0.012</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1255" class="table-center table-vtop">0.454</td></tr><tr id="tr-134"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1256" class="table-left table-vtop">PH</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1257" class="table-center table-vtop">0.228</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1258" class="table-center table-vtop">0.251</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1259" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1260" class="table-center table-vtop">−0.165</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1261" class="table-center table-vtop">0.386</td></tr><tr id="tr-135"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1262" class="table-left table-vtop">Lodging</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1263" class="table-center table-vtop">−0.069</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1264" class="table-center table-vtop">0.012</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1265" class="table-center table-vtop">−0.165</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1266" class="table-center table-vtop">1.000</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1267" class="table-center table-vtop">−0.443</td></tr><tr id="tr-136"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1268" class="table-left table-vtop">GY</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1269" class="table-center table-vtop">0.401</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1270" class="table-center table-vtop">0.454</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1271" class="table-center table-vtop">0.386</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1272" class="table-center table-vtop">−0.443</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1273" class="table-center table-vtop">1.000</td></tr></tbody></table><div class="table-foot"></div></div></div></div></div><div class="section app" id="app-7"><h2 class="">Appendix A7</h2><div class="section" id="sec-29"><h3>Simulated data</h3><div id="T11" class="table pos-anchor"><div class="table-inline"><table frame="void" id="table-11"><col width="8.18%" span="1" id="column-99"/><col width="9.5%" span="1" id="column-100"/><col width="8.99%" span="1" id="column-101"/><col width="6.95%" span="1" id="column-102"/><col width="6.94%" span="1" id="column-103"/><col width="7.11%" span="1" id="column-104"/><col width="6.46%" span="1" id="column-105"/><col width="6.46%" span="1" id="column-106"/><col width="6.46%" span="1" id="column-107"/><col width="7.11%" span="1" id="column-108"/><col width="6.46%" span="1" id="column-109"/><col width="6.46%" span="1" id="column-110"/><col width="6.46%" span="1" id="column-111"/><col width="6.46%" span="1" id="column-112"/><thead id="thead-11"><tr id="tr-137"><th valign="top" align="left" scope="col" rowspan="1" colspan="1" id="th-138" class="table-left table-vtop"></th><th valign="top" align="left" scope="col" rowspan="1" colspan="1" id="th-139" class="table-left table-vtop"></th><th colspan="4" valign="top" align="center" scope="colgroup" rowspan="1" id="th-140" class="table-center table-vtop">Cor = 0.85</th><th colspan="4" valign="top" align="center" scope="colgroup" rowspan="1" id="th-141" class="table-center table-vtop">Cor = 0.5</th><th colspan="4" valign="top" align="center" scope="colgroup" rowspan="1" id="th-142" class="table-center table-vtop">Cor = 0.25</th></tr><tr id="tr-138"><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-143" class="table-center table-vtop">Scenario</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-144" class="table-center table-vtop">Env–Trait</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-145" class="table-center table-vtop">IBCF</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-146" class="table-center table-vtop">MF</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-147" class="table-center table-vtop">ME</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-148" class="table-center table-vtop">MTME</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-149" class="table-center table-vtop">IBCF</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-150" class="table-center table-vtop">MF</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-151" class="table-center table-vtop">ME</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-152" class="table-center table-vtop">MTME</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-153" class="table-center table-vtop">IBCF</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-154" class="table-center table-vtop">MF</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-155" class="table-center table-vtop">ME</th><th valign="top" align="center" scope="col" rowspan="1" colspan="1" id="th-156" class="table-center table-vtop">MTME</th></tr></thead><tbody id="tbody-11"><tr id="tr-139"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1274" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1275" class="table-center table-vtop">E1_T1</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1276" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1277" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1278" class="table-center table-vtop">0.034</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1279" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1280" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1281" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1282" class="table-center table-vtop">0.039</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1283" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1284" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1285" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1286" class="table-center table-vtop">0.042</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1287" class="table-center table-vtop">0.008</td></tr><tr id="tr-140"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1288" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1289" class="table-center table-vtop">E2_T1</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1290" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1291" class="table-center table-vtop">0.006</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1292" class="table-center table-vtop">0.024</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1293" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1294" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1295" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1296" class="table-center table-vtop">0.041</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1297" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1298" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1299" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1300" class="table-center table-vtop">0.045</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1301" class="table-center table-vtop">0.010</td></tr><tr id="tr-141"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1302" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1303" class="table-center table-vtop">E3_T1</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1304" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1305" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1306" class="table-center table-vtop">0.021</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1307" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1308" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1309" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1310" class="table-center table-vtop">0.041</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1311" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1312" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1313" class="table-center table-vtop">0.012</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1314" class="table-center table-vtop">0.049</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1315" class="table-center table-vtop">0.009</td></tr><tr id="tr-142"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1316" class="table-left table-vtop">S1</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1317" class="table-center table-vtop">E1_T2</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1318" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1319" class="table-center table-vtop">0.006</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1320" class="table-center table-vtop">0.014</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1321" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1322" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1323" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1324" class="table-center table-vtop">0.026</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1325" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1326" class="table-center table-vtop">0.012</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1327" class="table-center table-vtop">0.012</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1328" class="table-center table-vtop">0.031</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1329" class="table-center table-vtop">0.008</td></tr><tr id="tr-143"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1330" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1331" class="table-center table-vtop">E2_T2</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1332" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1333" class="table-center table-vtop">0.006</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1334" class="table-center table-vtop">0.036</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1335" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1336" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1337" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1338" class="table-center table-vtop">0.038</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1339" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1340" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1341" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1342" class="table-center table-vtop">0.038</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1343" class="table-center table-vtop">0.012</td></tr><tr id="tr-144"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1344" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1345" class="table-center table-vtop">E3_T2</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1346" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1347" class="table-center table-vtop">0.006</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1348" class="table-center table-vtop">0.021</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1349" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1350" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1351" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1352" class="table-center table-vtop">0.034</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1353" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1354" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1355" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1356" class="table-center table-vtop">0.038</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1357" class="table-center table-vtop">0.011</td></tr><tr id="tr-145"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1358" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1359" class="table-center table-vtop">E1_T3</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1360" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1361" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1362" class="table-center table-vtop">0.017</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1363" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1364" class="table-center table-vtop">0.006</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1365" class="table-center table-vtop">0.006</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1366" class="table-center table-vtop">0.036</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1367" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1368" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1369" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1370" class="table-center table-vtop">0.046</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1371" class="table-center table-vtop">0.010</td></tr><tr id="tr-146"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1372" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1373" class="table-center table-vtop">E2_T3</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1374" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1375" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1376" class="table-center table-vtop">0.023</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1377" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1378" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1379" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1380" class="table-center table-vtop">0.050</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1381" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1382" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1383" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1384" class="table-center table-vtop">0.060</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1385" class="table-center table-vtop">0.010</td></tr><tr id="tr-147"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1386" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1387" class="table-center table-vtop">E3_T3</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1388" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1389" class="table-center table-vtop">0.006</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1390" class="table-center table-vtop">0.029</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1391" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1392" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1393" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1394" class="table-center table-vtop">0.035</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1395" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1396" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1397" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1398" class="table-center table-vtop">0.038</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1399" class="table-center table-vtop">0.011</td></tr><tr id="tr-148"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1400" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1401" class="table-center table-vtop">Average</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1402" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1403" class="table-center table-vtop">0.006</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1404" class="table-center table-vtop">0.024</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1405" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1406" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1407" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1408" class="table-center table-vtop">0.038</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1409" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1410" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1411" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1412" class="table-center table-vtop">0.043</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1413" class="table-center table-vtop">0.010</td></tr><tr id="tr-149"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1414" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1415" class="table-center table-vtop">E1_T1</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1416" class="table-center table-vtop">0.006</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1417" class="table-center table-vtop">0.006</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1418" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1419" class="table-center table-vtop">0.002</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1420" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1421" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1422" class="table-center table-vtop">0.014</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1423" class="table-center table-vtop">0.003</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1424" class="table-center table-vtop">0.013</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1425" class="table-center table-vtop">0.013</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1426" class="table-center table-vtop">0.018</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1427" class="table-center table-vtop">0.004</td></tr><tr id="tr-150"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1428" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1429" class="table-center table-vtop">E2_T1</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1430" class="table-center table-vtop">0.003</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1431" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1432" class="table-center table-vtop">0.014</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1433" class="table-center table-vtop">0.003</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1434" class="table-center table-vtop">0.006</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1435" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1436" class="table-center table-vtop">0.031</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1437" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1438" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1439" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1440" class="table-center table-vtop">0.041</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1441" class="table-center table-vtop">0.006</td></tr><tr id="tr-151"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1442" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1443" class="table-center table-vtop">E3_T1</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1444" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1445" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1446" class="table-center table-vtop">0.018</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1447" class="table-center table-vtop">0.002</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1448" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1449" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1450" class="table-center table-vtop">0.034</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1451" class="table-center table-vtop">0.003</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1452" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1453" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1454" class="table-center table-vtop">0.038</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1455" class="table-center table-vtop">0.003</td></tr><tr id="tr-152"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1456" class="table-left table-vtop">S2</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1457" class="table-center table-vtop">E1_T2</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1458" class="table-center table-vtop">0.006</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1459" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1460" class="table-center table-vtop">0.021</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1461" class="table-center table-vtop">0.003</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1462" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1463" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1464" class="table-center table-vtop">0.040</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1465" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1466" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1467" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1468" class="table-center table-vtop">0.045</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1469" class="table-center table-vtop">0.005</td></tr><tr id="tr-153"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1470" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1471" class="table-center table-vtop">E2_T2</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1472" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1473" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1474" class="table-center table-vtop">0.014</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1475" class="table-center table-vtop">0.003</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1476" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1477" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1478" class="table-center table-vtop">0.025</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1479" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1480" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1481" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1482" class="table-center table-vtop">0.031</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1483" class="table-center table-vtop">0.007</td></tr><tr id="tr-154"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1484" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1485" class="table-center table-vtop">E3_T2</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1486" class="table-center table-vtop">0.006</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1487" class="table-center table-vtop">0.006</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1488" class="table-center table-vtop">0.023</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1489" class="table-center table-vtop">0.002</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1490" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1491" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1492" class="table-center table-vtop">0.047</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1493" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1494" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1495" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1496" class="table-center table-vtop">0.052</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1497" class="table-center table-vtop">0.006</td></tr><tr id="tr-155"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1498" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1499" class="table-center table-vtop">E1_T3</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1500" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1501" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1502" class="table-center table-vtop">0.019</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1503" class="table-center table-vtop">0.003</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1504" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1505" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1506" class="table-center table-vtop">0.040</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1507" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1508" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1509" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1510" class="table-center table-vtop">0.045</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1511" class="table-center table-vtop">0.005</td></tr><tr id="tr-156"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1512" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1513" class="table-center table-vtop">E2_T3</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1514" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1515" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1516" class="table-center table-vtop">0.014</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1517" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1518" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1519" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1520" class="table-center table-vtop">0.031</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1521" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1522" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1523" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1524" class="table-center table-vtop">0.034</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1525" class="table-center table-vtop">0.008</td></tr><tr id="tr-157"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1526" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1527" class="table-center table-vtop">E3_T3</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1528" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1529" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1530" class="table-center table-vtop">0.013</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1531" class="table-center table-vtop">0.003</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1532" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1533" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1534" class="table-center table-vtop">0.021</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1535" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1536" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1537" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1538" class="table-center table-vtop">0.025</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1539" class="table-center table-vtop">0.006</td></tr><tr id="tr-158"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1540" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1541" class="table-center table-vtop">Average</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1542" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1543" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1544" class="table-center table-vtop">0.016</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1545" class="table-center table-vtop">0.003</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1546" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1547" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1548" class="table-center table-vtop">0.031</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1549" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1550" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1551" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1552" class="table-center table-vtop">0.037</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1553" class="table-center table-vtop">0.006</td></tr><tr id="tr-159"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1554" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1555" class="table-center table-vtop">E1_T1</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1556" class="table-center table-vtop">0.003</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1557" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1558" class="table-center table-vtop">0.040</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1559" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1560" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1561" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1562" class="table-center table-vtop">0.046</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1563" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1564" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1565" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1566" class="table-center table-vtop">0.049</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1567" class="table-center table-vtop">0.010</td></tr><tr id="tr-160"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1568" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1569" class="table-center table-vtop">E2_T1</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1570" class="table-center table-vtop">0.003</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1571" class="table-center table-vtop">0.003</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1572" class="table-center table-vtop">0.051</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1573" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1574" class="table-center table-vtop">0.006</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1575" class="table-center table-vtop">0.006</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1576" class="table-center table-vtop">0.068</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1577" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1578" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1579" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1580" class="table-center table-vtop">0.067</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1581" class="table-center table-vtop">0.009</td></tr><tr id="tr-161"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1582" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1583" class="table-center table-vtop">E3_T1</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1584" class="table-center table-vtop">0.003</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1585" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1586" class="table-center table-vtop">0.040</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1587" class="table-center table-vtop">0.012</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1588" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1589" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1590" class="table-center table-vtop">0.050</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1591" class="table-center table-vtop">0.014</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1592" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1593" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1594" class="table-center table-vtop">0.051</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1595" class="table-center table-vtop">0.014</td></tr><tr id="tr-162"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1596" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1597" class="table-center table-vtop">E1_T2</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1598" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1599" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1600" class="table-center table-vtop">0.043</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1601" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1602" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1603" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1604" class="table-center table-vtop">0.060</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1605" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1606" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1607" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1608" class="table-center table-vtop">0.056</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1609" class="table-center table-vtop">0.010</td></tr><tr id="tr-163"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1610" class="table-left table-vtop">S3</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1611" class="table-center table-vtop">E2_T2</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1612" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1613" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1614" class="table-center table-vtop">0.059</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1615" class="table-center table-vtop">0.014</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1616" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1617" class="table-center table-vtop">0.007</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1618" class="table-center table-vtop">0.057</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1619" class="table-center table-vtop">0.014</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1620" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1621" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1622" class="table-center table-vtop">0.062</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1623" class="table-center table-vtop">0.012</td></tr><tr id="tr-164"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1624" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1625" class="table-center table-vtop">E3_T2</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1626" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1627" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1628" class="table-center table-vtop">0.040</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1629" class="table-center table-vtop">0.015</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1630" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1631" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1632" class="table-center table-vtop">0.048</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1633" class="table-center table-vtop">0.012</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1634" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1635" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1636" class="table-center table-vtop">0.050</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1637" class="table-center table-vtop">0.011</td></tr><tr id="tr-165"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1638" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1639" class="table-center table-vtop">E1_T3</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1640" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1641" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1642" class="table-center table-vtop">0.034</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1643" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1644" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1645" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1646" class="table-center table-vtop">0.039</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1647" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1648" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1649" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1650" class="table-center table-vtop">0.046</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1651" class="table-center table-vtop">0.008</td></tr><tr id="tr-166"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1652" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1653" class="table-center table-vtop">E2_T3</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1654" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1655" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1656" class="table-center table-vtop">0.049</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1657" class="table-center table-vtop">0.012</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1658" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1659" class="table-center table-vtop">0.009</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1660" class="table-center table-vtop">0.037</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1661" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1662" class="table-center table-vtop">0.012</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1663" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1664" class="table-center table-vtop">0.049</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1665" class="table-center table-vtop">0.011</td></tr><tr id="tr-167"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1666" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1667" class="table-center table-vtop">E3_T3</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1668" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1669" class="table-center table-vtop">0.005</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1670" class="table-center table-vtop">0.053</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1671" class="table-center table-vtop">0.012</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1672" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1673" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1674" class="table-center table-vtop">0.044</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1675" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1676" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1677" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1678" class="table-center table-vtop">0.039</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1679" class="table-center table-vtop">0.010</td></tr><tr id="tr-168"><td valign="top" align="left" scope="row" rowspan="1" colspan="1" id="td-1680" class="table-left table-vtop"></td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1681" class="table-center table-vtop">Average</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1682" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1683" class="table-center table-vtop">0.004</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1684" class="table-center table-vtop">0.045</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1685" class="table-center table-vtop">0.012</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1686" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1687" class="table-center table-vtop">0.008</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1688" class="table-center table-vtop">0.050</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1689" class="table-center table-vtop">0.011</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1690" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1691" class="table-center table-vtop">0.010</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1692" class="table-center table-vtop">0.052</td><td valign="top" align="center" rowspan="1" colspan="1" id="td-1693" class="table-center table-vtop">0.011</td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn" id="fn-9"><p id="p-210">SE of the prediction accuracies given in <span id="xref-fig-1-6" class="xref-fig">Figure 1</span>, <span id="xref-fig-2-6" class="xref-fig">Figure 2</span>, and <span id="xref-fig-3-7" class="xref-fig">Figure 3</span>, with Pearson correlation for each environment–trait (Env–Trait) combination using the four methods under study [IBCF, matrix factorization (MF), ME, and MTME] for data simulated with a correlation (Cor) of traits (genetic and residual) and correlation of environments of 0.85, 0.5, and 0.25. S1 is the scenario under normality, S2 is the scenario under the error negative skew multiplied by 1.25, and S3 represents the scenario under the error positive skew multiplied by 1.25. The notation E1_T1 means environment 1, trait 1.</p></li></ul></div></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-8"><p id="p-61"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-09-27"><span class="received-label">Received </span>September 27, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-10-31"><span class="accepted-label">Accepted </span>October 31, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Montesinos-Lopez <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-web" id="cit-8.1.131.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coba</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zanker</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Symeonidis</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> rrecsys: environment for evaluating recommender systems (R package). Available at: <a href="https://cran.r-project.org/package=rrecsys">https://cran.r-project.org/package=rrecsys</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.131.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in maize breeding populations with genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">G3</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1903</span>–<span class="cit-lpage">1926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%26rft.volume%253D3%26rft.spage%253D1903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.131.3" data-doi="10.1111/j.1439-0388.2007.00702.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic selection.</span> <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">323</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DGoddard%26rft.auinit1%253DM.%2BE.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D323%26rft.epage%253D330%26rft.atitle%253DGenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00702.x%26rft_id%253Dinfo%253Apmid%252F18076469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00702.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076469&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-other" id="cit-8.1.131.4" data-doi="10.1109/MIC.2003.1167344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Linden</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">York</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Amazon.com recommendations: item-to-item collaborative filtering.</span> <span class="cit-source">IEEE Internet Comput.</span> <span class="cit-vol">7</span>: <span class="cit-fpage">76</span>–<span class="cit-lpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BInternet%2BComput.%26rft.volume%253D7%26rft.spage%253D76%26rft_id%253Dinfo%253Adoi%252F10.1109%252FMIC.2003.1167344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/MIC.2003.1167344&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.131.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMTMxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.131.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toledo</span>  <span class="cit-name-given-names">F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Hernández</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A genomic Bayesian multi-trait and multi-environment model.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2725</span>–<span class="cit-lpage">2744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D2725%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.131.7" data-doi="10.1534/g3.117.039974"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toledo</span>  <span class="cit-name-given-names">F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Montesinos-López</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A Bayesian Poisson-lognormal model for count data for multiple-trait multiple-environment genomic-enabled prediction.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1595</span>–<span class="cit-lpage">1606</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.039974%26rft_id%253Dinfo%253Apmid%252F28364037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNS8xNTk1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzEzMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.131.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ricci</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rokach</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shapira</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Introduction to recommender systems handbook</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">35</span> in <abbr class="cit-jnl-abbrev">Recommender Systems Handbook</abbr>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.131.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mondal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Autrique</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">L. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2016</span> <span class="cit-article-title">Canopy temperature and vegetation indices from high-throughput phenotyping improve accuracy of pedigree and genomic selection for grain yield in wheat.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2799</span>–<span class="cit-lpage">2808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D6%26rft.spage%253D2799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-confproc" id="cit-8.1.131.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarwar</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karypis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konstan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riedl</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> Item-based collaborative filtering recommendation algorithms, pp. <span class="cit-fpage">285</span>–<span class="cit-lpage">295</span> in <em>Proceedings of the 10th International Conference on the World Wide Web</em>. <span class="cit-publ-name">ACM</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.131.11" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F131.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/1/131.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/123" title="Modeling the Evolution of Female Meiotic Drive in Maize" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/149" title="Severe Fertility Effects of sheepish Sperm Caused by Failure To Enter Female Sperm Storage Organs in Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/1"><span>Volume 8 Issue 1, January 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/1.cover-source.jpg?itok=vyoqxep2" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/131" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-K9Tbc7H8BTCriTnWfpQUmjoya9wDCQivWtNe2e7GL98" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/1/131" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/1/131" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/1/131" class="active" data-icon-position="" data-hide-link-title="0">Prediction of Multiple-Trait and Multiple-Environment Genomic Data Using Recommender Systems</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487825" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-7DE5ZBLD6Obx-0uCUgdYLOn18dvZ26Epkdo7cnJI_Zs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/131.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/1/131" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-RzKIIQGJfUUP1MfW9xvVtwPi7OnO9yqIAXGAL8DExzY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/1/131" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/131.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487825" id="node487825--2" data-pisa="ggg;8/1/131" data-pisa-master="ggg;g3.117.300309" data-apath="/ggg/8/1/131.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Prediction of Multiple-Trait and Multiple-Environment Genomic Data Using Recommender Systems</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">José C. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="4">David Mota-Sanchez</span>, <span class="highwire-citation-author" data-delta="5">Fermín Estrada-González</span>, <span class="highwire-citation-author" data-delta="6">Jussi Gillberg</span>, <span class="highwire-citation-author" data-delta="7">Ravi Singh</span>, <span class="highwire-citation-author" data-delta="8">Suchismita Mondal</span> and <span class="highwire-citation-author" data-delta="9">Philomin Juliana</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">131-147;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300309 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487825--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040 Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:oamontes1@ucol.mx" class="" data-icon-position="" data-hide-link-title="0">oamontes1@ucol.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430 Jalisco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600 México City, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:oamontes1@ucol.mx" class="" data-icon-position="" data-hide-link-title="0">oamontes1@ucol.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">José C. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), 36240 Guanajuato, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BC.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BC.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David Mota-Sanchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Entomology, Michigan State University, East Lancing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BMota-Sanchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mota-Sanchez%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BMota-Sanchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Fermín Estrada-González </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040 Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ferm%C3%ADn%2BEstrada-Gonz%C3%A1lez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Estrada-Gonz%C3%A1lez%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFerm%25C3%25ADn%2BEstrada-Gonz%25C3%25A1lez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jussi Gillberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Computer Science, Aalto University, FI-00076, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jussi%2BGillberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gillberg%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJussi%2BGillberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600 México City, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Suchismita Mondal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600 México City, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suchismita%2BMondal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mondal%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuchismita%2BMondal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Philomin Juliana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600 México City, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philomin%2BJuliana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juliana%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilomin%2BJuliana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/1/131" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/1/131&amp;title=Prediction%20of%20Multiple-Trait%20and%20Multiple-Environment%20Genomic%20Data%20Using%20Recommender%20Systems" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/1/131&amp;title=Prediction%20of%20Multiple-Trait%20and%20Multiple-Environment%20Genomic%20Data%20Using%20Recommender%20Systems" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/1/131&amp;title=Prediction%20of%20Multiple-Trait%20and%20Multiple-Environment%20Genomic%20Data%20Using%20Recommender%20Systems" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/1/131&amp;text=Prediction%20of%20Multiple-Trait%20and%20Multiple-Environment%20Genomic%20Data%20Using%20Recommender%20Systems" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/1/131&amp;title=Prediction%20of%20Multiple-Trait%20and%20Multiple-Environment%20Genomic%20Data%20Using%20Recommender%20Systems" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/1/131&amp;t=Prediction%20of%20Multiple-Trait%20and%20Multiple-Environment%20Genomic%20Data%20Using%20Recommender%20Systems" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/1/131&amp;title=Prediction%20of%20Multiple-Trait%20and%20Multiple-Environment%20Genomic%20Data%20Using%20Recommender%20Systems" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/1/131&amp;title=Prediction%20of%20Multiple-Trait%20and%20Multiple-Environment%20Genomic%20Data%20Using%20Recommender%20Systems" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487825/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487825" id="node487825--3" data-pisa="ggg;8/1/131" data-pisa-master="ggg;g3.117.300309" data-apath="/ggg/8/1/131.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Prediction of Multiple-Trait and Multiple-Environment Genomic Data Using Recommender Systems</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>José Crossa</span>, <span class="highwire-citation-author" data-delta="3">José C. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="4">David Mota-Sanchez</span>, <span class="highwire-citation-author" data-delta="5">Fermín Estrada-González</span>, <span class="highwire-citation-author" data-delta="6">Jussi Gillberg</span>, <span class="highwire-citation-author" data-delta="7">Ravi Singh</span>, <span class="highwire-citation-author" data-delta="8">Suchismita Mondal</span> and <span class="highwire-citation-author" data-delta="9">Philomin Juliana</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">131-147;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300309 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487825--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040 Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:oamontes1@ucol.mx" class="" data-icon-position="" data-hide-link-title="0">oamontes1@ucol.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Matemáticas, Centro Universitario de Ciencias Exactas e Ingenierías (CUCEI), Universidad de Guadalajara, 44430 Jalisco, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600 México City, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-9429-5855" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for José Crossa</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:oamontes1@ucol.mx" class="" data-icon-position="" data-hide-link-title="0">oamontes1@ucol.mx</a>
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">José C. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), 36240 Guanajuato, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BC.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20JC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BC.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">David Mota-Sanchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Entomology, Michigan State University, East Lancing, Michigan 48824</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BMota-Sanchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mota-Sanchez%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BMota-Sanchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Fermín Estrada-González </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, 28040 Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ferm%C3%ADn%2BEstrada-Gonz%C3%A1lez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Estrada-Gonz%C3%A1lez%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFerm%25C3%25ADn%2BEstrada-Gonz%25C3%25A1lez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jussi Gillberg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Computer Science, Aalto University, FI-00076, Finland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jussi%2BGillberg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gillberg%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJussi%2BGillberg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Ravi Singh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600 México City, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ravi%2BSingh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Singh%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARavi%2BSingh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Suchismita Mondal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600 México City, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Suchismita%2BMondal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mondal%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASuchismita%2BMondal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Philomin Juliana </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600 México City, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Philomin%2BJuliana%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Juliana%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APhilomin%2BJuliana%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=131&amp;rft.epage=147&amp;rft.atitle=Prediction%20of%20Multiple-Trait%20and%20Multiple-Environment%20Genomic%20Data%20Using%20Recommender%20Systems&amp;rft.volume=8&amp;rft.issue=1&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Montesinos-L%C3%B3pez%2C+Osval+A.&amp;rft.au=Montesinos-L%C3%B3pez%2C+Abelardo&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=Montesinos-L%C3%B3pez%2C+Jos%C3%A9+C.&amp;rft.au=Mota-Sanchez%2C+David&amp;rft.au=Estrada-Gonz%C3%A1lez%2C+Ferm%C3%ADn&amp;rft.au=Gillberg%2C+Jussi&amp;rft.au=Singh%2C+Ravi&amp;rft.au=Mondal%2C+Suchismita&amp;rft.au=Juliana%2C+Philomin"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487825/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487825/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487825/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487825/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487825/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487825/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487825/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487825/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487825/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487825/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487825/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487825/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/1/131" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgements</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A1</a></li><li class="odd"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A2</a></li><li class="even"><a href="#app-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A3</a></li><li class="odd"><a href="#app-4" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A4</a></li><li class="even"><a href="#app-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A5</a></li><li class="odd"><a href="#app-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A6</a></li><li class="even"><a href="#app-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A7</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/1/131.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/1/131.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/8/1/131.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
