<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/8/1/161" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/8/1/161.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/8/1/161.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/8/1/161.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/8/1/161.atom" />
<meta name="HW.pisa" content="ggg;8/1/161" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genotyping by Sequencing in Almond: SNP Discovery, Linkage Mapping, and Marker Design" />
<meta name="DC.Identifier" content="10.1534/g3.117.300376" />
<meta name="DC.Date" content="2018-01-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2018 Goonetilleke et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In crop plant genetics, linkage maps provide the basis for the mapping of loci that affect important traits and for the selection of markers to be applied in crop improvement. In outcrossing species such as almond ( Prunus dulcis Mill. D. A. Webb), application of a double pseudotestcross mapping approach to the F1 progeny of a biparental cross leads to the construction of a linkage map for each parent. Here, we report on the application of genotyping by sequencing to discover and map single nucleotide polymorphisms in the almond cultivars “Nonpareil” and “Lauranne.” Allele-specific marker assays were developed for 309 tag pairs. Application of these assays to 231 Nonpareil × Lauranne F1 progeny provided robust linkage maps for each parent. Analysis of phenotypic data for shell hardness demonstrated the utility of these maps for quantitative trait locus mapping. Comparison of these maps to the peach genome assembly confirmed high synteny and collinearity between the peach and almond genomes. The marker assays were applied to progeny from several other Nonpareil crosses, providing the basis for a composite linkage map of Nonpareil. Applications of the assays to a panel of almond clones and a panel of rootstocks used for almond production demonstrated the broad applicability of the markers and provide subsets of markers that could be used to discriminate among accessions. The sequence-based linkage maps and single nucleotide polymorphism assays presented here could be useful resources for the genetic analysis and genetic improvement of almond." />
<meta name="DC.Contributor" content="Shashi N. Goonetilleke" />
<meta name="DC.Contributor" content="Timothy J. March" />
<meta name="DC.Contributor" content="Michelle G. Wirthensohn" />
<meta name="DC.Contributor" content="Pere Arús" />
<meta name="DC.Contributor" content="Amanda R. Walker" />
<meta name="DC.Contributor" content="Diane E. Mather" />
<meta name="article:published_time" content="2018-01-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genotyping by Sequencing in Almond: SNP Discovery, Linkage Mapping, and Marker Design" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In crop plant genetics, linkage maps provide the basis for the mapping of loci that affect important traits and for the selection of markers to be applied in crop improvement. In outcrossing species such as almond (&lt;i&gt;Prunus dulcis&lt;/i&gt; Mill. D. A. Webb), application of a double pseudotestcross mapping approach to the F&lt;sub&gt;1&lt;/sub&gt; progeny of a biparental cross leads to the construction of a linkage map for each parent. Here, we report on the application of genotyping by sequencing to discover and map single nucleotide polymorphisms in the almond cultivars “Nonpareil” and “Lauranne.” Allele-specific marker assays were developed for 309 tag pairs. Application of these assays to 231 Nonpareil × Lauranne F&lt;sub&gt;1&lt;/sub&gt; progeny provided robust linkage maps for each parent. Analysis of phenotypic data for shell hardness demonstrated the utility of these maps for quantitative trait locus mapping. Comparison of these maps to the peach genome assembly confirmed high synteny and collinearity between the peach and almond genomes. The marker assays were applied to progeny from several other Nonpareil crosses, providing the basis for a composite linkage map of Nonpareil. Applications of the assays to a panel of almond clones and a panel of rootstocks used for almond production demonstrated the broad applicability of the markers and provide subsets of markers that could be used to discriminate among accessions. The sequence-based linkage maps and single nucleotide polymorphism assays presented here could be useful resources for the genetic analysis and genetic improvement of almond.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2018/01/01" />
<meta name="citation_mjid" content="ggg;8/1/161" />
<meta name="citation_id" content="8/1/161" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/8/1/161" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/8/1/161.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/8/1/161.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/8/1/161.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.117.300376" />
<meta name="citation_pmid" content="29141988" />
<meta name="citation_volume" content="8" />
<meta name="citation_issue" content="1" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="161" />
<meta name="citation_lastpage" content="172" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Shashi N. Goonetilleke" />
<meta name="citation_author_institution" content="School of Agriculture, Food and Wine, Waite Research Institute, The University of Adelaide, Glen Osmond, 5064, Australia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7539-011X" />
<meta name="citation_author" content="Timothy J. March" />
<meta name="citation_author_institution" content="School of Agriculture, Food and Wine, Waite Research Institute, The University of Adelaide, Glen Osmond, 5064, Australia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2066-9495" />
<meta name="citation_author" content="Michelle G. Wirthensohn" />
<meta name="citation_author_institution" content="School of Agriculture, Food and Wine, Waite Research Institute, The University of Adelaide, Glen Osmond, 5064, Australia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1174-6580" />
<meta name="citation_author" content="Pere Arús" />
<meta name="citation_author_institution" content="Institut de Recerca i Tecnologia Agroalimentàries, Centre de Recerca en Agrigenòmica, Consejo Superior de Investigaciones Científicas–Institut de Recerca i Tecnologia Agroalimentàries–Universitat Autònoma de Barcelona–University of Barcelona, Campus Universitat Autònoma de Barcelona, 08913, Spain" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0939-8038" />
<meta name="citation_author" content="Amanda R. Walker" />
<meta name="citation_author_institution" content="Agriculture and Food, Waite Campus, Commonwealth Scientific and Industrial Research Organisation, Glen Osmond, 5064, Australia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7596-8484" />
<meta name="citation_author" content="Diane E. Mather" />
<meta name="citation_author_institution" content="School of Agriculture, Food and Wine, Waite Research Institute, The University of Adelaide, Glen Osmond, 5064, Australia" />
<meta name="citation_author_email" content="diane.mather@adelaide.edu.au" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7506-2589" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. J. Aranzana;citation_author=A. Pineda;citation_author=P. Cosson;citation_author=E. Dirlewanger;citation_author=J. Ascasibar;citation_title=A set of simple-sequence repeat (SSR) markers covering the Prunus genome.;citation_pages=819-825;citation_volume=106;citation_year=2003;citation_issue=5;citation_pmid=12647055" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=P. Arús;citation_author=R. Messeguer;citation_author=M. Viruell;citation_author=K. Tobutt;citation_author=E. Dirlewanger;citation_title=The European Prunus mapping project progress in the almond linkage map.;citation_pages=97-100;citation_volume=77;citation_year=1994" />
<meta name="citation_reference" content="citation_author=P. Arús;citation_author=J. Ballester;citation_author=B. Jáuregui;citation_author=T. Joobeur;citation_author=M. J. Truco;citation_pages=331-338;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_author=E. M. Babiker;citation_author=T. C. Gordon;citation_author=S. Chao;citation_author=M. N. Rouse;citation_author=R. Wanyera;citation_title=Molecular mapping of stem rust resistance loci effective against the Ug99 race group of the stem rust pathogen and validation of a single nucleotide polymorphism linked to stem rust resistance gene Sr28.;citation_pages=208-215;citation_volume=107;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=D. G. Bielenberg;citation_author=B. Rauh;citation_author=S. Fan;citation_author=K. Gasic;citation_author=A. G. Abbott;citation_title=Genotyping by sequencing for SNP-based linkage map construction and QTL analysis of chilling requirement and bloom date in peach [Prunus persica (L.) Batsch].;citation_pages=e0139406;citation_volume=10;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=Ś. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Cock;citation_author=T. Antao;citation_author=J. T. Chang;citation_author=B. A. Chapman;citation_author=C. J. Cox;citation_title=Biopython: freely available Python tools for computational molecular biology and bioinformatics.;citation_volume=25;citation_year=2009;citation_pmid=19304878;citation_doi=10.1093/bioinformatics/btp163" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. Dirlewanger;citation_author=E. Graziano;citation_author=T. Joobeur;citation_author=F. Garriga-Caldere;citation_author=P. Cosson;citation_title=Comparative mapping and marker-assisted selection in Rosaceae fruit crops.;citation_volume=101;citation_year=2004;citation_pmid=15159547;citation_doi=10.1073/pnas.0307937101" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=J. M. Donoso;citation_author=R. Picañol;citation_author=O. Serra;citation_author=W. Howad;citation_author=S. Alegre;citation_title=Exploring almond genetic variability useful for peach improvement: mapping major genes in two interspecific peach × almond populations.;citation_pages=16;citation_volume=36;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=A. Fernández i Martí;citation_author=C. Font i Forcada;citation_author=R. Socias i Company;citation_title=Genetic analysis for physical nut traits in almond.;citation_pages=455-465;citation_volume=9;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=C. Font i Forcada;citation_author=A. Fernández i Martí;citation_author=R. Socias i Company;citation_title=Mapping quantitative trait loci for kernel composition in almond.;citation_pages=47-47;citation_volume=13;citation_year=2012;citation_pmid=22720975" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Font i Forcada;citation_author=N. Oraguzie;citation_author=S. Reyes-Chin-Wo;citation_author=M. T. Espiau;citation_author=R. Socias i Company;citation_title=Identification of genetic loci associated with quality traits in almond via association mapping.;citation_pages=e0127656;citation_volume=10;citation_year=2015" />
<meta name="citation_reference" content="citation_title=Breeding Plantation Tree Crops: Temperate Species;citation_title=Almond (Prunus dulcis) breeding;citation_author=T. M. Gradziel;citation_pages=1-31;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Z. Gu;citation_author=L. Gu;citation_author=R. Eils;citation_author=M. Schlesner;citation_author=B. Brors;citation_title=Circlize implements and enhances circular visualization in R.;citation_volume=30;citation_year=2014;citation_pmid=24930139;citation_doi=10.1093/bioinformatics/btu393" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=V. Guajardo;citation_author=S. Solís;citation_author=B. Sagredo;citation_author=F. Gainza;citation_author=C. Muñoz;citation_title=Construction of high density sweet cherry (Prunus avium L.) linkage maps using microsatellite markers and SNPs detected by genotyping-by-sequencing (GBS).;citation_pages=e0127750;citation_volume=10;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Biochem. Genet.;citation_author=K. Gürcan;citation_author=S. Teber;citation_author=S. Ercisli;citation_author=K. U. Yilmaz;citation_title=Genotyping by sequencing (GBS) in apricots and genetic diversity assessment with GBS-derived single-nucleotide polymorphisms (SNPs).;citation_pages=854-885;citation_volume=54;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=J. He;citation_author=X. Zhao;citation_author=A. Laroche;citation_author=Z.-X. Lu;citation_author=H. Liu;citation_title=Genotyping-by-sequencing (GBS), an ultimate marker-assisted selection (MAS) tool to accelerate plant breeding.;citation_pages=484;citation_volume=5;citation_year=2014;citation_pmid=25324846" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=K. E. Hyma;citation_author=P. Barba;citation_author=M. Wang;citation_author=J. P. Londo;citation_author=C. B. Acharya;citation_title=Heterozygous Mapping Strategy (HetMappS) for highresolution genotyping-by-sequencing markers: a case study in grapevine.;citation_pages=e0134880;citation_volume=10;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=T. Joobeur;citation_author=M. Viruel;citation_author=M. de Vicente;citation_author=B. Jauregui;citation_author=J. Ballester;citation_title=Construction of a saturated linkage map for Prunus using an almond x peach F2 progeny.;citation_pages=1034-1041;citation_volume=97;citation_year=1998;citation_doi=10.1007/s001220050988" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=T. Joobeur;citation_author=N. Periam;citation_author=M. Vicente;citation_author=G. King;citation_author=P. Arús;citation_title=Development of a second generation linkage map for almond using RAPD and SSR markers.;citation_pages=649-655;citation_volume=43;citation_year=2000;citation_issue=4;citation_pmid=10984177" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Kearse;citation_author=R. Moir;citation_author=A. Wilson;citation_author=S. Stones-Havas;citation_author=M. Cheung;citation_title=Geneious basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data.;citation_volume=28;citation_year=2012;citation_pmid=22543367;citation_doi=10.1093/bioinformatics/bts199" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=T. Koepke;citation_author=S. Schaeffer;citation_author=A. Harper;citation_author=F. Dicenta;citation_author=M. Edwards;citation_title=Comparative genomics analysis in Prunoideae to identify biologically relevant polymorphisms.;citation_pages=883-893;citation_volume=11;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=D. Kosambi;citation_title=The estimation of map distances from recombination values.;citation_pages=172-175;citation_volume=12;citation_year=1944;citation_doi=10.1111/j.1469-1809.1943.tb02321.x" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=A. Kujur;citation_author=H. D. Upadhyaya;citation_author=T. Shree;citation_author=D. Bajaj;citation_author=S. Das;citation_title=Ultra-high density intra-specific genetic linkage maps accelerate identification of functionally relevant molecular tags governing important agronomic traits in chickpea.;citation_pages=9468;citation_volume=5;citation_year=2015;citation_pmid=25942004;citation_doi=10.1038/srep09468" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Soc. Hortic. Sci.;citation_author=W. F. Lamboy;citation_title=Using simple sequence repeats (SSRs) for DNA fingerprinting germplasm accessions of grape (Vitis L.) species.;citation_pages=182-188;citation_volume=123;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=E. S. Lander;citation_author=M. S. Waterman;citation_title=Genomic mapping by fingerprinting random clones: a mathematical analysis.;citation_pages=231-239;citation_volume=2;citation_year=1988;citation_issue=3;citation_pmid=3294162;citation_doi=10.1016/0888-7543(88)90007-9" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Lu;citation_author=A. E. Lipka;citation_author=J. Glaubitz;citation_author=R. Elshire;citation_author=J. H. Cherney;citation_title=Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.;citation_pages=e1003215;citation_volume=9;citation_year=2013;citation_pmid=23349638;citation_doi=10.1371/journal.pgen.1003215" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Miller;citation_author=J. Dunham;citation_author=A. Amores;citation_author=W. Cresko;citation_author=E. Johnson;citation_title=Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers.;citation_volume=17;citation_year=2007;citation_pmid=17189378;citation_doi=10.1101/gr.5681207" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=I. Milne;citation_author=P. Shaw;citation_author=G. Stephen;citation_author=M. Bayer;citation_author=L. Cardle;citation_title=Flapjack—graphical genotype visualization.;citation_volume=26;citation_year=2010;citation_pmid=20956241;citation_doi=10.1093/bioinformatics/btq580" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. Peterson;citation_author=N. Weber;citation_author=E. Kay;citation_author=H. Fisher;citation_author=H. Hoekstra;citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.;citation_pages=e37135-e37135;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22675423;citation_doi=10.1371/journal.pone.0037135" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. Poland;citation_author=P. Brown;citation_author=M. Sorrells;citation_author=J.-L. Jannink;citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.;citation_pages=e32253-e32253;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22389690;citation_doi=10.1371/journal.pone.0032253" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=A. Rasheed;citation_author=W. Wen;citation_author=F. Gao;citation_author=S. Zhai;citation_author=H. Jin;citation_title=Development and validation of KASP assays for genes underpinning key economic traits in bread wheat.;citation_pages=1843-1860;citation_volume=129;citation_year=2016" />
<meta name="citation_reference" content="citation_title=Handbook of Plant Cell Culture;citation_title=Almond;citation_author=N. Rugini;citation_pages=507-611;citation_year=1986" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=J. A. Salazar;citation_author=I. Pacheco;citation_author=P. Shinya;citation_author=P. Zapata;citation_author=C. Silva;citation_title=Genotyping by sequencing for SNP-based linkage analysis and identification of QTLs linked to fruit quality traits in Japanese plum (Prunus salicina Lindl.).;citation_pages=476;citation_volume=8;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=R. Sánchez-Pérez;citation_author=W. Howad;citation_author=F. Dicenta;citation_author=P. Arús;citation_author=P. Martínez-Gómez;citation_title=Mapping major genes and quantitative trait loci controlling agronomic traits in almond.;citation_pages=310-318;citation_volume=126;citation_year=2007;citation_doi=10.1111/j.1439-0523.2007.01329.x" />
<meta name="citation_reference" content="citation_journal_title=HortScience;citation_journal_abbrev=hortsci;citation_author=R. Scorza;citation_title=Progress in Tree Fruit Improvement Through Molecular Genetics;citation_pages=855-858;citation_volume=36;citation_year=2001;citation_issue=5" />
<meta name="citation_reference" content="citation_journal_title=Extremophiles (Print);citation_journal_abbrev=Extremophiles (Print);citation_author=S. Söllner;citation_author=S. Berkner;citation_author=G. Lipps;citation_title=Characterisation of the novel restriction endonuclease SuiI from Sulfolobus islandicus.;citation_pages=629-634;citation_volume=10;citation_year=2006;citation_issue=6;citation_pmid=16896525;citation_doi=10.1007/s00792-006-0019-0" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=K. Sorkheh;citation_author=M. K. Dekhordi;citation_author=S. Ercisli;citation_author=A. Hegedus;citation_author=J. Halász;citation_title=Comparison of traditional and new generation DNA markers declares high diversity and differentiated population structure of wild almond species.;citation_pages=5966;citation_volume=7;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C.-T. Tan;citation_author=H. Yu;citation_author=Y. Yang;citation_author=X. Xu;citation_author=M. Chen;citation_title=Development and validation of KASP markers for the greenbug resistance gene Gb7 and the Hessian fly resistance gene H32 in wheat.;citation_pages=1867-1884;citation_volume=130;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=I. Tavassolian;citation_author=G. Rabiei;citation_author=D. Gregory;citation_author=M. Mnejja;citation_author=M. Wirthensohn;citation_title=Construction of an almond linkage map in an Australian population Nonpareil x Lauranne.;citation_pages=551-551;citation_volume=11;citation_year=2010;citation_pmid=20932335" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=J. Taylor;citation_author=D. Butler;citation_title=R package ASMap: efficient genetic linkage map construction and diagnosis.;citation_pages=1-29;citation_volume=79;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. R. Thomas;citation_author=N. S. Scott;citation_title=Microsatellite repeats in grapevine reveal DNA polymorphisms when analysed as sequence-tagged sites (STSs).;citation_pages=985-990;citation_volume=86;citation_year=1993;citation_pmid=24194007" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Viruel;citation_author=R. Messeguer;citation_author=M. C. de Vicente;citation_author=J. Garcia-Mas;citation_author=P. Puigdomènech;citation_title=A molecular marker map with RFLPs and isozymes for almond.;citation_pages=964-971;citation_volume=91;citation_year=1995" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. Voorrips;citation_title=MapChart: Software for the Graphical Presentation of Linkage Maps and QTLs;citation_pages=77-78;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011185;citation_doi=10.1093/jhered/93.1.77" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=S.-B. Wu;citation_author=I. Tavassolian;citation_author=G. Rabiei;citation_author=P. Hunt;citation_author=M. Wirthensohn;citation_title=Mapping SNP-anchored genes using high-resolution melting analysis in almond.;citation_pages=273-281;citation_volume=282;citation_year=2009;citation_issue=3;citation_pmid=19526371;citation_doi=10.1007/s00438-009-0464-4" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=S.-B. Wu;citation_author=T. Franks;citation_author=P. Hunt;citation_author=M. Wirthensohn;citation_author=J. Gibson;citation_title=Discrimination of SNP genotypes associated with complex haplotypes by high resolution melting analysis in almond: implications for improved marker efficiencies.;citation_pages=351-357;citation_volume=25;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Wu;citation_author=P. R. Bhat;citation_author=T. J. Close;citation_author=S. Lonardi;citation_title=Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.;citation_pages=e1000212-e1000212;citation_volume=4;citation_year=2008;citation_issue=10;citation_pmid=18846212;citation_doi=10.1371/journal.pgen.1000212" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genotyping by Sequencing in Almond: SNP Discovery, Linkage Mapping, and Marker Design" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
<meta name="twitter:description" content="In crop plant genetics, linkage maps provide the basis for the mapping of loci that affect important traits and for the selection of markers to be applied in crop improvement. In outcrossing species such as almond ( Prunus dulcis Mill. D. A. Webb), application of a double pseudotestcross mapping approach to the F1 progeny of a biparental cross leads to the construction of a linkage map for each parent. Here, we report on the application of genotyping by sequencing to discover and map single nucleotide polymorphisms in the almond cultivars “Nonpareil” and “Lauranne.” Allele-specific marker assays were developed for 309 tag pairs. Application of these assays to 231 Nonpareil × Lauranne F1 progeny provided robust linkage maps for each parent. Analysis of phenotypic data for shell hardness demonstrated the utility of these maps for quantitative trait locus mapping. Comparison of these maps to the peach genome assembly confirmed high synteny and collinearity between the peach and almond genomes. The marker assays were applied to progeny from several other Nonpareil crosses, providing the basis for a composite linkage map of Nonpareil. Applications of the assays to a panel of almond clones and a panel of rootstocks used for almond production demonstrated the broad applicability of the markers and provide subsets of markers that could be used to discriminate among accessions. The sequence-based linkage maps and single nucleotide polymorphism assays presented here could be useful resources for the genetic analysis and genetic improvement of almond." />
<meta name="og-title" property="og:title" content="Genotyping by Sequencing in Almond: SNP Discovery, Linkage Mapping, and Marker Design" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/8/1/161" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In crop plant genetics, linkage maps provide the basis for the mapping of loci that affect important traits and for the selection of markers to be applied in crop improvement. In outcrossing species such as almond ( Prunus dulcis Mill. D. A. Webb), application of a double pseudotestcross mapping approach to the F1 progeny of a biparental cross leads to the construction of a linkage map for each parent. Here, we report on the application of genotyping by sequencing to discover and map single nucleotide polymorphisms in the almond cultivars “Nonpareil” and “Lauranne.” Allele-specific marker assays were developed for 309 tag pairs. Application of these assays to 231 Nonpareil × Lauranne F1 progeny provided robust linkage maps for each parent. Analysis of phenotypic data for shell hardness demonstrated the utility of these maps for quantitative trait locus mapping. Comparison of these maps to the peach genome assembly confirmed high synteny and collinearity between the peach and almond genomes. The marker assays were applied to progeny from several other Nonpareil crosses, providing the basis for a composite linkage map of Nonpareil. Applications of the assays to a panel of almond clones and a panel of rootstocks used for almond production demonstrated the broad applicability of the markers and provide subsets of markers that could be used to discriminate among accessions. The sequence-based linkage maps and single nucleotide polymorphism assays presented here could be useful resources for the genetic analysis and genetic improvement of almond." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/8/1.cover-source.jpg" />
    <title>Genotyping by Sequencing in Almond: SNP Discovery, Linkage Mapping, and Marker Design | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"g4KnyBYDTJHarVALDKyRowAXG5vFR9xXS3lfY-Afeu0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"487826","apath":"\/ggg\/8\/1\/161.atom","pisa":"ggg;8\/1\/161","ac":{"\/ggg\/8\/1\/161.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/8\/1\/161.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;8\/1\/161"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;8\/1\/161"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/8\/1\/161":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022faYPD+oHnkPm5yakYYABiAfFbhVDAu60NQzzTYVnEWuEAfoCFZ7uyuO00mT2ZnQb2PAAK76cnv4nLeD4cAxIEuOGOstHwstWBOmUC00QiZECSihZOA4htCIicOwo0rDZHH9PdhMMEcwPZ5QV7kXsqZG1n5LdPlcv4gL4TmUGXVsQPCRDb0Ftl8sSLtg\\\\\\\/+QPA+F1JPH4Y17pabYklAtUz6lrkZim5uekVO7WIpZtxJBtqR+r5YpRgOt2uMEm5i3Qyujtcg2aKPgWCLXzt55uzj1rjtpetYkqRvj2W8q9LqAG4RnfNPH7IkaFnBAtvpusAARWFrxNLrlUDoQvKrtQSz09rmLwU5BvDXJXFE2+ysrGISiFbsHo1KVDF1WYH+y6Hoq0ehg\\\\\\\/CnLlW+u3O8Y6WnTuNt\\\\\\\/v5lMlbMAPPyqpnEXq21CN58OBG4vz1HWYZRsJBHZ8Dy26V\\\\\\\/EZ4DgLSpfgZ49zL6HrPY132MEhQ4RXgDHCLL2VmRpUUqQzmEygS6kUFmZRLlJCcYLbKZt5iiHtfeSYY7fB1P+KUvCrbthnEec4pDCadWAv7FJxzHcIQGe7OxoxTM4iBqnByVeBzUPnez1BOgfx+zNSVj9IP1gsDlQFKTFhJGi3iKiWMAcGNSz81mrUAWkI7UQFtAKv1X0Rs8mgt\\\\\\\/yYsFuc2OhRuA7IVzKqUUVva+S7lL2BV97lmRqFRhq7c55UzEFmJeYA7EaOaqCEVL8IiUroYTqWFAROQd6YPkVtJhIY4yaltk+loiAGiQhqbuIH\\\\\\\/p1xcc5qEemorHx\\\\\\\/blrvqTLSfbSHRbYW063bmEIXveGbG0etzAFQDefu8+8J5toVORcdkewIMoYOFQURLkUlh9l8hVGOIYkhyCDdHr29lmvBZA8I0FfX814bh6PuzYpBe2U9kUCJVlieyXMLZlCvSK9jePShzJDlu19c0LR7ZFjdlCZxfFmZ8wr9DUBZyTFgDw8CgNIM05sY059f8T9KnHvGoV1YvGg3DJKfITCP\\\\\\\/W6HGnzBumG8RzMERm6TE5+esFtrKt5q+l2UlNrejsluBMVwghQgeZxyZCjU6Lzyod5dstzjnQOdbPJyzfEDQM+kMcrj518xJ75xhJXcqYWo2z95lcHBWVRIiika3A3U7FKOP5Nf4xzlydUW7tD6QDEaxWffN7TZ6vkCcqAgznyCxJQuE9zhOO5bo1KPl+WovvtjU+LCjSvyouZe8tuJmlJgyIB7QHsKH94Rl8rOVkbM\\\\\\\/8+8+y+pSV\\\\\\\/6B84Z2zkd8cPSNomrcZ6\\\\\\\/POlBi\\\u0022,\\\u0022iv\\\u0022:\\\u0022qPPxeeOcrMcXqOQjgOvPkg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002259cfe680139c023368a9502e7b31fb16ccc188a46ca12b1f25a53c750431f71e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022zGMFSmWMRnj1ScKichbzs0\\\\\\\/K86F+TLRV34JM0ypDW7l1MXZucYGVSwihnuhf94vU6d\\\\\\\/3pVchuwfTYbCxToenXPKv69hfrxcjQVGwCiyMy5+HR6D+VLKRB1vHpk+4OUuqo8T23Fx147YtpZQ\\\\\\\/GWgmiUFzntXolCwPsZHBZoDm4HQ+bp2cBhXGTc\\\\\\\/hqWZMnRxBZVA4WAQx91EntpLMY9JlM2vMpeqvrzQ1RN9kXl\\\\\\\/h15vPOr+dqvxkHDj39TTQgX4zQ8mkI7CvUgM7RJu4IcjFriB5QbIF\\\\\\\/HELFpCwZOMhMqoO37\\\\\\\/7T1UhJ1FHwfO9n8tmxCY\\\\\\\/8Fq724TtDuEXctpIHFqTCphp\\\\\\\/t1\\\\\\\/\\\\\\\/EgfDdU93b5uhXMGESPimIfkI3X6LP1TpuSGEhYNiYSqRMWM6K8StJNZVQc9stOS8\\\\\\\/5qddwCC3nLJ\\\\\\\/2rCsAqlmgfRrV0pltx2wRonErjoRt3JxEoY3E6Phib+ABEhSY+rhq83jHIXqy43ddYhUTjWMw7Z53hT7O0\\\\\\\/6dPeKJPaqHzF+vd8l3qCh1uoHyAHtHE2lYP5NwWphYO4c0BhtvVAe8YezsuNGRSEWRC0QNGqtXN0g5CqGVL18JCnToGOSVbbW3gbKaBMwujl8sl3WvBx\\\\\\\/bfjDjohdg3UtZip\\\\\\\/PPXLIhyIyAaE9kKgf48E63P335S7ePoMLqYg2FqErBov5\\\\\\\/T35d7Vt9ifELy6KrR7XLnJqvA9tX9hbaKXAe5dorV13GUkD\\\\\\\/yVxbel6BpT4JyKqbiXWe7xDrMDoz9uzk82zsxW2pXcmotzCB1nD1ckwD5N6Ssyo6UOTfEoIZ9t3F5IxuQSXRiZj8BwCyliqc54V+yjrNX9UVQTOQCaWV4iXxV3vQ1+Cz6jM3ojpO+KQFN6qqnfM5QndCVPg\\\\\\\/euOZY7hGDaTKw5Fm5+7AS1VEOE9kVMLVGHtWj1U8Ylh4Qp2yXv73k0OlHV6cOfrB9\\\\\\\/nWQcNGSpL5ENAZbVaycnge45O1lyL9nSapZpU8NB\\\\\\\/tID4HkawpJddpZxSSU+Gf0pVsC0tZ\\\\\\\/nepi+VX2KWFn+N2oz7E8mXKkcX1FfEVTAIr5iOWUx5KH\\\\\\\/\\\\\\\/CNhG11139ZaoB+M+Xuxz\\\\\\\/5A8Ns41OT3dspIh3KBeIO4o0MOctgcedfKcsGmpGLhZVwxrQNaZVMO4duvr03CjWiSho9gY5hi9l6Ium9\\\\\\\/77HkMkfxlFPZte0TwCc2H8UzqjPqBiIdUWZvDucYfCN9laXQvDVIAdvpGaZDgHya2PaAbLKKGaoC+nhGh8vYzRYD5fOtK3CD5m6TSwA0SVgCxBpuayaUF\\\\\\\/3F1vMeXlG5bKlCjFu2c+YR1QPfzhj6kupwaMKEZZW8Jo2d5n7tfm0Lb3ijwy2GaEY\\\\\\\/3fTGU4qPQWfi3hE0wDbsjA32sPgvQWI3DbcNZKINs+39LJB0EWOFC7SENuviRAAf4zu8+rfmocNhdH0tJBRG0MC5cpDhAv\\\\\\\/NqD8Jkp9qUWT9et9pnzKfuiayu5aepui9Zarwo89tZql2We7HVWiOplia\\\\\\\/zDmTM+Ct+\\\\\\\/YxII66idmpd01JWbCJtlxjfYNv\\\\\\\/3AiKkgZNi8M2f\\\\\\\/dLE1HsOJOegX6wq6SxpfS2IjnKAh7JxK5D6MuwB\\\\\\\/ysYgp6i5u3IwXY3NzZX3mFS5XbTKOpmx3i8OpPFOuYwKtVltOZvZGvrWjMZwsM8zQXJ0YhKIk79VYBIypFnC901LuY2YuUr6L1Ej7fG7Y4NeWa9eyPXY+4BnwCVTr3\\\\\\\/ivj\\\\\\\/14mfhHxCu77AlIk1r+4l8aiNP2cucrItPmkDG0sIotbXNpd\\\\\\\/QaRLW6zUdqDmdRhEypxXNp+IX6J8MoOkw\\\\\\\/aq8jY1ACMUPAYGR176ZLVf0yoh7gnSvTrhZB10TF9ytoDwSYR1t4Pjm9ArnoK\\\\\\\/z3iF9XlHwLTglVw8MZNcNgY5TgNb0M+cqcx9Dd0uieG96PCfHKlfeooTvSK\\\\\\\/X8xA1dErzNIiHaYEZDrZzHqNXmB1F8mE8pB2rjPUO9I2hdDhfsIY1MVI+x9SS6ETy9MhwjfswjxVLuW952J4cMg63rspvbY8VAZPMyEQDOoL+lGYfuL0V43RPKjy0dT0FUqMPocCE\\\\\\\/9HV824rpl0IASTjNwvCH4xEcLzIyJ5fIdCvkC1WGAANy9ydPJ71SxgbUrgcozAC9ykmFtVuB\\\\\\\/soef9B62zQZjG13oTTT\\\\\\\/LozYZuof5iIJ4xHHrC1G4a0euHCNxFJKblVVpPHXc6ck73r15VlGLsLIFG5X2UJJ9Lzy8Mu0FRpTwyNjplmaYi0aemRTjC1o6oG\\\\\\\/8lLCnlZvmMjgZNiElaN8UfNXsGRdXBlH16hv6KAE0vXTx\\\\\\\/NPLzgIUsS44qBouwal5aYCMH5B8m4W\\\\\\\/Z4y2BVX5n62ekNW\\\\\\\/xU5rjB\\\\\\\/OSDXV7tEzXpZ2Z\\\\\\\/V4GU511NqMT4dRtkfhYf5irx5aScaTdM6vlH3Duoh36Rl5u3UBSwHIwaGpLvrlZebL\\\\\\\/xoN7YBp7xiCL3++4tdIPQ50da3K10w6MRQ0ghKuHrOfnajMwCDodJiWLk9N5fpXDSHe4wxwNTjp74JRkAdOIo2Pi3r9Nc\\\\\\\/s4vrqNuf8bFnPgzFwQ2NAWJqEAC2Nj9AqEmvgYlyMOTCFJC5mcNL1ibSXWS+YgbKV3FSzSgI5ZPHWVDhHQ4T5p2RPAqZGqhvwVb9pozPPG1RlM7\\\\\\\/SChXVHzZXSu9kFJHILC8cMykfiZX4hGVoBA5Ol+XEGGHkGZZTmtwztOrU07QxwI6YAHoZFXm5tyUjBk09JA0DPVGNm\\\\\\\/Xeg7\\\\\\\/GwyxBa7CebIfPToD\\\\\\\/1RXXh6Nnh\\\\\\\/NTcKfLk0fmuvEq6O26lW5bL6826NTYZyWU+WlOGSDnYAjwXqBMHEwODi+rFC79EG7KnaNv3yagApTEby7x\\\\\\\/vM0xiWasT3d8hqdBA+4Pafkwd7CQcD1ZUbA0cn\\\\\\\/BSHAZyhla3yR86Xy2W7i3jO5N9x44FjSiSlUwHL4AHFXfwe\\\\\\\/k+WqE4TYaJC8p5NHX4+8I7L+NkbiZkv+NZQvR\\\\\\\/H+Qda\\\\\\\/Q4mOUzj0HH6jwTRWqI5cuhALylbjoOWzFmLdYVyGdC\\\\\\\/kQ1iibfF9oly8APCdkwlu3R73WweDXcd8Hdm8ZgcddBTnfaAomFdPM96t73Ri\\\\\\\/PlPCpwFtgK2BxqZp1kZtQKUd6JXQvP87aAK0Pa8TT+IWSfMaJNpq+0Os+ijjNiVGzLhraH2i7FpCWhE+IYFvuVyMNPNJkTXhTal79H0EgYXnLX6oInZf4wnB\\\\\\\/gxCM4VuKe9n5cN59x49Sa4iWBOEFMB7ohQ06l4f+rBO5PtHSi28Twb1\\\\\\\/FLuM4aUygjfGm7ubnKl0Bkevib+aj3FqnjMjtxOZVVUFg6M98NJAwipSdAKguRDO9H+ThC5JngiLzGy1jHtlAxGdc+vNqwxMhIelubR\\\\\\\/oiacmj5SrCDJQvK47THgS5W+Wdl9dKYuX7Qs2mLS\\\\\\\/Q1\\\\\\\/kOZCgtCWbPmrB5vRAVZUVUnBWofyrMcDb\\\\\\\/CpC5O55pfMZW\\\\\\\/3t8p8DEQaOxdGqBWA2Af8WYft4DQvD+C1jgAObR+NljZIiCL9eM++zh1zg1t5JJKDch3AmfhlboLzvDVPGBovYoAvqpqiAvzcQj6zdwAdEKao\\\\\\\/WaaBQIeZyRVNeI+h5lAU90QNHhkcQIMY6ad7YUy1nBOjdU7\\\\\\\/pTfvz\\\\\\\/H7qFUT+ieqSWQV+aEhkm+Tbv3p3uowbpC5Ze7lNmY8UwtyUBqa\\\\\\\/d8\\\\\\\/x\\\\\\\/fUYjJTXufvuCRbdTdxRawc98XZhjCe6LAGbLrqct7nLiUeObAWmKDEQpATiIGObeCmOAZAOrX6Vg4d7+0bkY+O7f4UlR0Gh67\\\\\\\/H++xbiRvZT2+E6siSMj0yI+JfCEBDH3TBc5tiE1Hc29aCEJPIWVOO2xbQxaI3CTPKm2k0qIgeBxH2orAw\\\\\\\/ag36pQivx3nIrUhAl3DTIZz72j04g4Movq3q\\\\\\\/Pudul0CkdrbD6Wu7uL4LuhnA9iN+sO6atZ5yKNibXVoMvsJ\\\\\\\/DxEefKaSVccXWnk20KIOU1xjSWvcHoyBHVoB\\\\\\\/idaEYASPODXQOdebId7nnllwNj8pPhkJ4U34BTv1F2E\\\\\\\/3gA8CvbDAtYJSsXrn8og5\\\\\\\/yYgXyrmbR5Usr17I0c7oCpMNdad2lV3Ld05ajnn5hiVD50tC1y9o4AfC4UO6RTbqOrTtglx+on22bWA54bcwwzsU+BN79z2gsNLQDSKGlzozFD8pJN3XMyCbF6hxsTkbNnJfyQI85agGKZi\\\\\\\/\\\\\\\/Uuu4h+8OwwgwmcLEPtlPgznKvbaDdUEM4gjgRuQaRvcw1JhWK68rS1pjt0wWL6zjjO5EctAEsrszTVjSZpwPHZ3med8ASSvI7QTyC1ljKGABxsJjaplBe40blMzIuI2FnU75f4bWvCHGQ3azz58XbZpDg8tO5gPePDtP+M1NJAq95M1+0qJzNZxqGgxw2RiBLWZbsNrcEx3wqvXNsn\\\\\\\/SLl\\\\\\\/JfJzr9NCT8Gkv1LD\\\\\\\/nr0FGr3y9w4gNhJdhvxuwQUyWXFYdvJsoSEpEQgsepofHXzOf+\\\\\\\/6HGuj1J5yHks+Vt4Tpy9isSm42ovBipp7W5IXzvCF4t22yS1v9finQvx9CGVTFPuFDBADYpHrVQzSA8O5iyehdcpYaBXL3NQ7sSe9Z2+CD2scPn7TK87XFGDl83VW87ySLhfKqcXfZ1Ghmp6sbKtAxzTf8ihjY6Dwymkb6JVZMMOS+qR64FWZR0j40s1FjYWqIKCKKEk7SB9NOJLb3DFLrgPw3U0ACJNp5w\\\\\\\/hxuw3jjZZyqiKCyTZL4x3ZG7JYiiEUEC64r4u2mcdC58u2r7UgGPqteKTnoHDnftLUtaCPcPNZ6n34nNAUafCxr1TjAuXQqwWxjQCiv\\\\\\\/AOYxR6m9UARqhHBplrF\\\\\\\/AcaCLSWgvjBmU0Q4bqNfixqZQ9pZKhrwQGXXIQ8Nn7Mace7Cu1fyoO22Y3nu+efD0bfhHp0Ox3r+130WYqLekQui2tDjggG2JGr0MWPII6MYolvVVS+AfjsYtlwQS8JArox4bUlUdcEE9+SPL8G2cZuqSKYjNaGIOgK16Jz0Fhjrc2NfLqlxBMIXU6rCqlvBJNcMm\\\\\\\/FRAx+SFV9vCkW3\\\\\\\/g6RMO\\\\\\\/C\\\\\\\/kaWnvxD8J+ySIFpxLQtvLo8d3Ya\\\\\\\/QMkGKM+KNlI1abWgkN\\\\\\\/tN3Zm2jLs1Gz7RbNaW6+2k5upx0RP9bwBZ6UKB1ET5xVZF2OJLLaGtr3YNiDsfjjTgiZD+b9zLPnMKXbx5u5RZGuIofGX7K\\\\\\\/A93Jj2FaeZvJVQuks0tZqUWffeyMK1VQ+ugue5935mMDTQhALh4bizyLmkP3u6cGEoYxgLG4YA64H4hkGdNmE68kB\\\\\\\/XxfKrUaxAGsigBe820F1L4TZSzSjIc0wiyJpDqWcL0ue+LnDZxYtT64SgqSNYS4OlHgkTdUuzP4qVXtHitNvHqTf\\\\\\\/VN\\\\\\\/1sE56C84QR8\\\\\\\/EsheURCeIWrYnvM2FO5VFmdSHsTzez26kJych6LDY\\\\\\\/xuXDQpFdx53pidd7IafMRkn9AkZkVEBP9M14iniifAgWa\\\\\\\/f9F8qvL+EW6eN7pqIaOdekn60vle6zLZ8XHehoYIF7dNfpnE4fXeAbF9iCjf8OIO+tgf9jh8m0OHgpagWhQsa4WUt+jjx13AV\\\\\\\/M9WkDOjy3i11B4UgF9VwD+CvhqKKms8da50rvsrTKXJ6sf4VA2FRSnBhYaHFZyKpiafvHEqOlW4hCwsm2nopYNXQWkb6mWK3jQqQrQqxEMfGHj\\\\\\\/958KEpBmbGH+t\\\\\\\/Vi9g7YGeHl\\\\\\\/updgS\\\\\\\/dS35d9zKRjW2k4c1W59MYdNwwfkZgMUDhfliahVCChcoeff10m8pwXBH31gzgWAyPbCgNlh0VZEN1pbVOybJG0KnYAjwx2Oz4F0ZlaErFEzAV38YzeYa\\\\\\\/4oMzdDnXphHk7HyFdSrneXDNm5G4rfdLcegu832+z2zNdW+q3jkBhiPu4kTEn6hKONYglQdzmncOy25njq3BU\\\\\\\/8\\\\\\\/38IDKg5ZXKysWqDD0K+8HEv+xbdok3aRCgfUbtmkqOx6gfAZ0h867m3GrFyHCubw+kddIUDwTNXUubECQaJTKDT\\\\\\\/4gLI71AKok4TGM+zq\\\\\\\/Pjsoe1Sfv9X+ZH\\\\\\\/aUsrWuoNdRtPo7EWYUtZdcsd8JmUw7M\\\\\\\/x0n3SZpORYkivRoI5WTSmTEFPB\\\\\\\/qiILNCex8ayb6yUmo5LJ7o5TbxT3o2QI\\\\\\\/h8cfmpoqgJm3b32Lar02AsHhQ1cW9BtRPvIjgw2pbAPtJGGajoTz87y0OqMppgiE3zp6RhqxixRxNYOmdjWaYWFjq63ZDCYq9peSJVj+d34ZZzhTdzGvlcm92EewO\\\\\\\/bAjqwOJdqp3dz3Hq78K7XvtWSh406oPVOrteQIb02dqjw59HCKpuMyGAU9LSVw7eqPxkCRunVOTiNYEIH1IljYuWVuSGf9wyGrMq7rgSA4SrZ8b+CKpFc1bRBDehwU8doTi6RjfgclpHJnIbEUPhS2cgSg3uE\\\\\\\/p4AL4J00K4Dhi4a3IQlsVC+5A1O9HHbPQ12Pdgr7xcjnIk8Ix7it5vvkQSRGj44AqFadrGr5wnw2lpIuUzvn+oQdXEho9emCR2gkwIt3xcgmFLrnqSZ3gwg3Eh2cgkP6RwoUDGJt4I58ainfIUJ\\\\\\\/R84Ebc3MmDXsGRheGA7LJtLfPxiBv\\\\\\\/ajC+F1MOetEHgpJzBYYvbEsh8EI9ECc6SCnDsEaR2JjJG6qDeoV1kwAervU0Z\\\\\\\/ypFIf7cVwl4my1DHIyFLgYLaJgJnsDSb\\\\\\\/VWRfTEBblhTvZlDNwQlw\\\\\\\/FNlmRHl+r4soORcPHoGv59bjGldUYh5irwzwuqePnIAHjhYQiGZw9hUJ5C6kL+\\\\\\\/nBEuJDt0PeGGGdKfNyC6StKB\\\\\\\/DFam2aHYBUxsHG9FDeij0TpCdtlZm+hPaEE5CaKESHcWJgB+9rw\\\\\\\/zs7KyDXvnbAaK9n6ey7CxrlJif7O3kBQf5QN1NqQUtf3Qz8XeD0LicO1uCWIjt9nYLvY8BOi7FM7AZbm8znc8srej0ZX5k36hiH9wZuxJE76i1CVDmomOM+JfULgo+pgHCFHuLJaf5rP2MMZfSZw\\\\\\\/oEpn7MiACWzTh8q+ymRVQ0tug5mqO4qR3tZQ1mBARz1BCANpqrWnaJ3TPh8nrvpzE+qOuvElQUK4nbinvWkSzs2hUaiWO+aVM1g+cwIglPkx8PCCu3keN6y62QN+MpNHqRzC73v6myCmYNahkH7\\\\\\\/kZN5TJWa0FUPMvAw+ARo4oq1\\\\\\\/qIPruCsnKoDvffTdODb2Quih8u15ZKyxiP4khxs5MgD+TJ1TVjXlf9LyTybAo2fU7gI+IU9lcb1eEsoAoj9hztYMXJNi5YMUN0XfbKZdrjWaXHgyDtDkDXzpgSY0QndmWYfjsS3Hj4f+u32H8EkPKJO+wolpFeIp1Vb2sPHgi2Atn+UvPicHBs344ulWKSy+9UtKNfokj6MUio9AOjCAg2sSbBJJQRZP3RgO3gINF\\\\\\\/rhGRLbpaBxg53i\\\\\\\/3jKsbtTBhMBTcI3ZJFCZC2tdXQjjq+A9AQVduIHeJMpiwWyDam2t8KP4kzk4AqLAtwn6YwmbSOVw6ZQdBfN05nUo+p+xO8uP90qTs7qODJag\\\\\\\/0Umu6n3DKMcUc5y3PWJDcxKiN8dmx6NuW+GRyg+GhbU4WVkBmpw4DkYAUqDJpWr3sy6HzQExDxwbmhkD1+yrpZ1WAcE3j0xuEqnwj+bj7WItEnoJ4ROfobfBvkOMfhS6HTS+5\\\\\\\/XEftfLUyVtG1PFo1WUT7eJtox+pBj1N6Eox7hgw20voBUenM\\\\\\\/zAH\\\\\\\/Vj43HIPkzb0tbKGkMn6O55uzNYQEjn3xjt4G9ddFhoAGlZgQ7gjzlK2czQ4fIY1uYxWKcC+tly1oIDOQSTeG\\\\\\\/WPGWwca4B4TW3Dnfu8\\\\\\\/yitlB152lgEcPe1NHSD5oxcByR5b6JVU5t1o4Gdax8zWutje43iU+EvVp\\\\\\\/MdhNeVavWxRUVhOsyOJAEj3TNWnSBdiDN\\\\\\\/NybAsf3Cn4HJtVb2qv+io8T5LlzNL7y1hTI1gQQZ4TX\\\\\\\/DbVZkg3xZHyGrCNJ9bY6BcKTqQY8Ib4XqW2e6D77RrbuxnmHwKIgNunsuJk2TODIhKVswJo5oEcO2hwC5qSl6HRzMXQd48drbQHp7Ft6Oc0+do4qpUC0qu6JFbJUWrnfns587WGX8vl5uIGD08JdP+F3uBcH3TQQVD2D84rHX0UFIbrWXPdCDBREqleGu1388VP3djFkFGbXgeNWRlAg89Z6B1C4Z1HhANI72KxcThrQ5TrCtDfs+J7gKdc\\\\\\\/0mohmEiXCCtfyq\\\\\\\/rnuczMssZAr4hjAcuX0mFUXuVoBSRnA7QPwDDFN2qwRwivnJyGCLvQg\\\\\\\/rn24MWUswBvNEzpTLeFDYUw+at0xPWOnE9dmYk987idhg5eLS1MNMiMznyoMyAIXAzd4hZVKs5CdBIxyG7Vqf\\\\\\\/Nj\\\\\\\/O5ICR\\\\\\\/71GgTaYvr9EoJbCgZhW08W8p5Bnq0yU\\\\\\\/EixKzfG4AJ4E582xzQtI5yLo+IO\\\\\\\/+1EaScWE0U7cSCTPOra2FpY\\\\\\\/PqH7ATareAZh8xfG5TVTD6+Z8COrXQO8xLsoR9\\\\\\\/m8Caj1aXcJfp2g0j5NnFjbY4JTZp5COiaK8BL+vnSOu8tAzP26Ej8BMJTtueJ\\\\\\\/MoXOIbmmEt2U8MWBn4Vt3p8UqZEZ2leV3oEnO5cEveZYVveu4oztIl3DhZRf9UR5XB1Cp1bn7yHEEFHXDdk2v6oymUmyEU7\\\\\\\/NpwqbzKDdHfXwYv53hNpDBp93H9SAnogHd\\\\\\\/sIjsCtM3cqblU20re\\\\\\\/LPP2fheAxr6OJbrrxK36w8J8h8B7UgIOu0WMkdw8+A7ORy52hf+4zaAAuwAUGBFVXihGIcTk187TSzWFqEaXA6mKnVfUxOifMnOuZwCDL9RD2JfIDlo6D+8aeFPr7zpajtNPPfFOkaAE2ZRrbDbJtZt\\\\\\\/8v5Wg9rPdJs4eDYcx4ia0nbBz4Ox5FXz7UQq\\\\\\\/t7suP7vTvSTbUtFDFaDUjz7bdDuAdBhXOJXOycFJHdrnuzWEUwP5ayf9QcYJ\\\\\\\/9K+oo7CLlj2W8DYiNzY9pO+HeNe2nxGZ6OT7WiusMc39zLlTR9pvCB\\\\\\\/wpsHdo+pI+5zq\\\\\\\/97\\\\\\\/YhdqfcR6c3ULIOQklBE34BTIiGWbvxEjrvajMlRGT43+QLbgl+EKc+an6p7trvqh81aVTELHIwFdf8UYbrM8JrSV3u4sniloGicQld5woiWId9wkVxTaLYrsvRRbUqacrGbPwTO0q\\\\\\\/YwbUuPJVuEL7IAK75ExNfBR7tuNlaVP93Rt3wSPTK8h2KT7Ao2egUpUr\\\\\\\/bkj051CRN4X\\\\\\\/42QwLrcA\\\\\\\/qKCXwuhcMOJJcrx2NqhjzIdTX5NYrqXNtXLQm+9xFR2Q+MIW\\\\\\\/Fv64Q4q3EZ0LDe3M8TA1nZoPhGZil3IZvIOpqDiPR8XqtL64SctI5AkD+IfTw\\\\\\\/gG7AFWzPeAS8AV6vtv\\\\\\\/uuKaG5hk5cJA9hQfK2+s6Q0rYKd2JR+d5w9IRX8EZ8OZ55ku5fEfobsC\\\\\\\/WHeZJqMYTTLtujzCYtSjUpDFY4otfNunKu++pI+SZnRjLQHE4CGCkvOM8B1mt1CKzbKHSzwGYVzzQ4LOqiT0ildZcAioeVN12TXOnQ3yCssisAiq5UhHe45ngWywt+zlf22jzGg0t8uXf16s5iBx+bstUD6gVyI07ixYzUw\\\\\\\/xnS4dishhSnAPo+JsEh5\\\\\\\/cYDG7vYUI2ImwIkeiEKnneRgzIUj3BfOVoSiF74IJsf55dUKZ5l6a9x5UMDPgvJVZEI51gieBKqfiE63fRaOa6bBmkMnmtIi9uBVf9AFtEVvVf\\\\\\\/ZkotV9PTnM1vmb5TIpd8WsZj2+3klbuZtIdelz69WFPetEK7aiPuD6IX3ucOoybkRpO0YfkLHpF+Nwz9\\\\\\\/XaA97C6NET0juoPccz6LAbdAVs08OB5NTyQn5soClD31Gud9Utl+Fq4Lxr71UE7XvBCFWWqgOYeKHt+GT\\\\\\\/4s\\\\\\\/4vODcOCOg3Xh2aYhwXAh8JytyaIVrpaIvbnNSf9xsAgr2Aht2C6a2Z3o7IGBZRl7p58yzH7+f0l1KuFT1tS0mlrHgTw\\\\\\\/egNXpJvVvFPruEeTIpz9wDoHh2o7pqLJPbinmpNN3ZirEB1mkwatp2Vl9Ju8WaufgnA3Ce99zIL45R413lufcFKs2yqP\\\\\\\/ipBOrQ0s\\\\\\\/sRUGTeBjYBoSJjfOBLVG6OVPCOTtQ6pBV9dz3emLljV7zwzOAxlL5Y848KChHghK4\\\\\\\/3UWJ2idb\\\\\\\/Fq1GlPO6Xkabv8ZrEN1Tkij3BAeQp9irv+GjyOe6yRTjExuy\\\\\\\/FNNOsceu8D4ZLTeqSjIj782BDxRbFdTPfo8aUNuJFTHln0kYLMGy8T1+txuCsOC8gtN5VPCfEzl2YkPqzA2CyoSqxl+SvzY+5uGdXBb7XvS6\\\\\\\/2vbGpWVVIQBmYsphf+dw7AV8EYIRw4lBB5kOGmmkqiMygeslYpdfb5gVGarJkozvSURTnzPlrUMnvHHfasnv9i+NslTfph5P3zme7MsZ+rt7dvwOh8+hhfgMgOG\\\\\\\/8kwDEvC4nED56i6aCYsShiYFAsX\\\\\\\/UWo5l66u87DP68DmSlV9e9RX9RX7eOk4Ggh9wKQW5gui9NqI96TPZjsSgstt9lsq+70DMsv3IfAW647H+ogPE6dyG7oY2FeuNSEhOxn3YpA+poNp9YhriEDOzmg2QLH4Tv0yBGfHWteOe7eCpMAwjqWz4ZuByeVorLdFWJlYixG4xRLLttrqT9njEbblG4IJ\\\\\\\/JJzyLzZoE6tq3hUZZR\\\\\\\/SNNoqJvjeuAZdngRJmR5S1BqLgdEqFnuOcidSxjh\\\\\\\/2GI7K+l8T5ZxtUttof0A+zoCr6vwqviDNL9epbItOCg\\\\\\\/rjvcQ3CzLNFh0zd2KmWcetDvoLZwhWmyma4P9ZIpEW55t7ur7fxP5+cI1vItsunNiIuutdOLNF3aFAFocIFGZiKFIhnZcTQnzuy+yWdwXDD8A\\\\\\\/UPJax6I6ErqD1sp\\\\\\\/qb0hwOf4Nq+1x4DHkKqlQAzJ61TguN8pJRCwpYJqJFF7eJP5HSDUZbRzAq8+W\\\\\\\/pZitb0eaUFtH4AFsC+G\\\\\\\/Mr+mWKET9caQRnUzbDPnVblVMFowiGKLePWtZULh+Tpr0giHkJpV8i1FfvU\\\\\\\/vuqRsQvQnpO7\\\\\\\/697BASx6jFihSO9mOeqR1V8goHXIQbh2HKOklOQ6cwAaACO63lLjn0pTjNt7mDxRbgPiEjgRJQlIU6nph9gnQM7Dmz61VQM5nuverRgfKzxWmnuU6QFEM3hVNl71ARR26aZaWqlUHly5bQ0pOTjZZYNkVNvcbG3Z3gj4SfrDSsGP+AzT\\\\\\\/K0SgY7y0C2GAmBu3KoNjBwSyJAi7kQauFDa5LHp\\\\\\\/E1pkZ6BDFvlLKA0tis39mWAyNNpf8JPFqTlT8xbxjT17m3Z4\\\\\\\/Qh2bf3mqFI5iVlqKGht16swstfR1WEGDhI2NmHIQHhjdypepIq2fFtovQTRWvx9bX\\\\\\\/zjwLlInDEzT94HKSHihCYkAYgXwwVe24O7XgRIIcJbwJhxiCkoF9CEFIcRXyArU4qg1LrWYSUWgDxqh7O8W6IynOcM2q6zB9LMWEogrqW2joJtNmaKRhc4EQYXD3euHfqgz3ycjeaaUBJnUiZyevfU7x9p5hgV095+QFJjRg\\\\\\\/P2URD30XdvG9Npn9Tj9sy9UxMC6WsGK20JXlcf8UZzgGnTAW9uBWZacjKxJ0s1TYrCtc8NvBQrfqVffRN5XO2pjNanEhWBGsv7WzfZWfGxRKSMa\\\\\\\/QbikF+fwvca6hR2SxJOEdRBTaWiir1vWfDARBVUS6gs\\\\\\\/XHkldBUD3mRoLLDc72XAry8IKIXGLswI1NcRlvCpcSS6fDBIoWrbItkcIyjI41h8w\\\\\\\/FPaE8toYXfnfz4fPycHf9DRuLMxK91i43oDCcExgM9Lf9CMtcxplJie1KZeAxhpUSLudnBRcxbnMJcYnent9ErMoklX+6RTv3yJd2UVUChmc0\\\\\\\/rsR3b5j3x+Dwh6mdco10oHzP1rOWNbxHkat7mK1XoeH\\\\\\\/C8zzufSwDve+EzgYuTJ4OcpVeWwF\\\\\\\/9\\\\\\\/cRu2KrXI8rXIhNZdGlhsGAVUV32RVtHWkadSJ+GMrBKNn5nvFZWWmpkXAxliwXLIqoGD0N1jdT4244Vq2b+OWHGVp\\\\\\\/fZdNF6rPE1IbNt+cmhcqjnYXZDuZn7qGa0VK7pWQY5IgywveHyOCXJ+fed8UKDuwSGg0vTPm7bulTO2v5iL1kmYn6\\\\\\\/7yVZ8UsixDrcIHZKpuftVVA1Hoyii3lXXt6GUsBtQOfATOrDWoPqg2TAnJB1OH4EDOUsg5nknX2ZiV1Rd6WQ97e9hdHmSmddvPkPd2uUJNL+HHmWyaaSTeNF6hnpPTtAy5rdjZeiyMnPU6lXE\\\\\\\/Tz+CYGwzQrhBq3Z3WOi+WV6BGo0P1B6YNkr0V3E2956d0pfuvJlnE4IrTJDHho4SFyOs59t9zlX9Z9V9yMXWiC+TYq2CpDHZeWmqLo7TGMU3KbfIFLd\\\\\\\/eoxM4ggd1+TJlT8Z\\\\\\\/MX7nPgd2HAPH\\\\\\\/ZmwrzvGkM7T5U6kFFEsO0yOJkWQTtJpMY4FWAt3mU7\\\\\\\/Ln0uxtJAkaL2pbO8m4lX5RpCFpSB88eMsrN9iRaZBBL+x9vjLHY5QArQV\\\\\\\/NhJ1\\\\\\\/hXtTHLpUzpc+H9Sbn5T9IrpTePBQ5eIi2cJP60YNAyQl5yur5TNE+JR8scZvIMoZyqlmigZpeRi5lgad\\\\\\\/znD\\\\\\\/Z5t1Q9RRY0\\\\\\\/d+yrp3KAoTy1a6gPOQ5JDqv2BhdbD4Jh8uCF1BL1slciGnmMZ9dB+wDyFRjFCtDowpc+fYXqxTrpCCA\\\\\\\/Fca6t+93iV6rT3Xzi3YJA1Dy+UsiIsDeFL+EkyLNLFOgZB91108xsecJaDc7CbWxDWovptpVwBl\\\\\\\/v3CKJbd6WzWP+y2ZMcTUrdhcre91VAY3X9ZA\\\\\\\/XjQAMBi+lHYyEaLGDeoRfc4QPYZLgW4UOi1FzYXoDWJZHajyM8yrP4yn5EQfiPl5AmZq6xX8qAy8MDdJbHErJdDRkDWKkKQyvY02PsdejbfL4xRQ+yEe\\\\\\\/XiG\\\\\\\/Fx3Bl6dtuBgRo4iBYZpLUtu+90NlV96wIrgSM+qtgvbu0kQu+2q4yPiO+Imaw9TRa8r2QxyOVEOzAirsD6UFp8s+3CNmFQIv05MQrODFDHrhsmTjpKe\\\\\\\/4EAM52nu0s6amYCy5EH7WmTLedW4r\\\\\\\/7LAZY4rBXjf8eYFBxVPxCMuEIfNndmIcU7AlDOGM1mKMDyl4LbKtRiL6MOekxjV+z7LmvekNUCfZc7XUQhsljZNfeaVaLp77CO5fca3j0Co7L\\\\\\\/KDpHipRvqBEIbx090TA1uzX2PmsYLefK6\\\\\\\/pDKf+hl5WiaAWLB+zE3pp\\\\\\\/pkf0OQR+EXtVIPD+4Ua+VnCutvpdzTfspP8LLZAGulUWd6TOwET+kNycjk5c9nR31fxHzfJu5O8ZDtIL8ZdOjaRnqUGSYmR4X+eTWQT9vv8zIxANG6LbCd9nItbTvGp2Dmc4FOVo5Q+uIERiVvFA+RvMoW7cBayXJ1LtShPSWT6etX7Wy9+g4C5Du3bkG6bavhsdNcU3+xcxsrrgV\\\\\\\/ZPalnAmL6eWbLrfZihk1pe\\\\\\\/\\\\\\\/moSYYD0N6NgxxNSG3Fp6JKbz31LM7hc0+zjCJ0egLkwAXD\\\\\\\/5qK4U7FcIKdkYJVJLU5QpwjikoCu4eWOeqB6WneUXJ3MI6fhBdyP+4h2suoCFgnQ2PqJqt1JqewmLpeZwyou7WiGBNp3VLbIkHAtj4DX0iz4aMDWnPxFlDDsXwTVgRjPkOrT2efM561\\\\\\\/Ik0WrnT8Eam\\\\\\\/2B+IaDDveZcq99qdL6trFitjPsG20wPShmUQgphBZrWgkge+zjlTcWzKPkxBAH5SUEzZSH+r\\\\\\\/kyF7ZYdLwahTO8rFuTbVKcRw4kpUhAUwE7wNVLhYxPFa6dhcX0S6QvG4K3ibE7iurjxTHyd1U6bSrEn\\\\\\\/EDI2PjkweNtz92+6GqLdw0cYg3u0MkfmNLX5vRTsZwkSi\\\\\\\/beMPaHXJwjDsjCZgmolyQ+q6oZ4Srvz9\\\\\\\/1nW9lbpuoZarP5nDKbtBJrZtDwFMKZmGQL5HROOQzu8qe2RysRhVXerQZTeURA+Sw+pAv+uaTg5LlESmWzQIbO1C2QXDPPRmDbqQsdZQZ2yWcbA3KPvRZQI+kooLzox\\\\\\\/XUQdyzf0cnANN2\\\\\\\/Ty\\\\\\\/2\\\\\\\/TnrxY+e6fdeFTdlVX4oVX6Jjtk5nZaagMdmY\\\\\\\/prKa3q8dA\\\\\\\/dJahCCdlofllDrAVQC7uqdtWCM21UBN74gd7bqZVlTLJgqHVqx\\\\\\\/ymINcB2NVTFuGCG8rc3TPjGdC1422Xd5qzuOYL2w6jlbIERlW47bFfg5Q5VlzUluqUevc1Zv\\\\\\\/hFc6+O4gT+0T0mSpEiYI\\\\\\\/vceSGqrPjS95nTN6nDurt7iulPB0IDrzPoUTc7gAaFrKrfeFMzWbORiTf\\\\\\\/LEj1X0ulmnpqMpHSaUQ00VKw0jY+0SafMkwgTug6j4YR4b+yjyDEK0oreH0zm8kSPYwVg03JxvRAPoXQtjA0MwzpkQ5s65yIiOAB9NxBnCtW5MNWsoM5kMIhTGp2J1rYDwoqw2et5p1SzC+OGPKmKhZ3g4VU6cBJy1Ic5L7BdfBvvyRzPF5E65+Dp5MRMbqY2G3Sp+fb3p4VpFTTQAuIGCSTLiEKWF0\\\\\\\/ht1PgBFNgZZkulLFB2HxQ8teBmqNuinZaKzl\\\\\\\/LNsrnoVXj9BSpmAXIbJpBwHLxAUDjB1Fm\\\\\\\/q4c0TIE1NEydUJD2QOb\\\\\\\/\\\\\\\/oN0dc9bWgHNzfix1bAzDwc1in8mkOs+Y7EewHKaLokVrZljaBsq7cW58k8gZqsUnyIsdCfj8KHE8ZMzJlN97WSSp25NDyNFxfg7fCyLA2gYqaeeVHEboJjVS\\\\\\\/LOlQedIRiVEbbIZa9SDeodS2oaBFM+j66CN1AfrjOtt+4+sOPAIdG18vcIHaurwDAqzNjSy4xs3H\\\\\\\/S3W2mXPI+MEYq4+BzT7L5sFoAQFx9J0+RL3CAPgrHTJr8GM8Y8RJofuF7VjrjMUY04ylRQWLIbEAlVMJXhwv5NDT\\\\\\\/IOPO14jGLRnuOXN6rzBlTtffqp2Cy\\\\\\\/\\\\\\\/xTilthjHg424gYL1xaQpLOIkp71PurkS\\\\\\\/bL\\\\\\\/g2zjt2UPhVLIxlWJ7OwP6r5lGaFuMOB2Yka5pOrCz5b40zr28o4PS\\\\\\\/88yi++WHjQmbzYuc53pLTADKH8COqx85bP4DUSBgsZAQYviw76C9WgDeSxK8mj11qeC8sAiCgOw1nyMdkGxjnZg0aGTbXM8am8hS6Qjs+HWsmIgnHi8TqZ532k1xoa4ODTDdTbBc7A6BwwgFK0NBIhHI7cGpiYP+LH4ye3rwNLfSZUZSQHKXHhryox+aMi3KTgHDkxJNrv28l5YPNLhRDyRFbVaxfHiAf3W3p43SbaGLBSXjlU5Af\\\\\\\/iQufgp171FoQ6LEQLkYyAAYdnKqI8Vjj72pteNTlaY0pPJFVU9Day8DNjP1oWOfUnX5GYPXBaCD2qJ2tPwi9Orc6a2xZL4WLYeC4HSDzqsQrMXP1Nfwj37tXfJYWQ0uejTsuenrjm+nALhCtV\\\\\\\/j+qiSgBQ8GSJiWAMb3mc\\\\\\\/cB4F8cduu97buHZZKZQbe8mvPyAG8uNgFbA03\\\\\\\/BCRE9jZ009uu0R4DfZY77Viezqx3ew9jAbuYP4Z\\\\\\\/nca\\\\\\\/Ie0sO+PoWBBEdRh7NgGfSXnuKCSgKlbc1QZRE2kP9bdt6Z7omgsm0ap44rqKv5chTNi8TP09v6si5RUwzoal1KX1gPV\\\\\\\/SkoLxw2oXI5BBdx46lQtgHvmPDpZR9bE2gqBUv4n092KebVKRZgan0PAt6WOLnHBaE6v\\\\\\\/g0d59XAakJQJkOLYAJi0jlXfco9SiaIVBBA5lwA3DELmPvqJpu1TBkiCaAsd6+3i70KdJ7oxF2qaUSzsAT1c4bAQXYReEGVci6J24xRlVN3fzvjs7yBrqMDiwW3MAf5cRdujw14FRv78AIXogYWUOaIczYmaXGKv7ARp+TU+j7ybsipAtgKbtYABe7yshIK0jCBVgOBpNQctHKhAIF\\\\\\\/HVS2CuX7dWXTGR2M7qGyRHjfVkMRA+d70aeovwbpiSx6ccpPtpJp3FfYnxraBQ366TngiBhkebshM5BEHEE1ykJ6UN3cLSzMXelRsim+GfP21JEtlxvjDL4evqhCq9desoviHL7NVeUso19AO47QaRCkIyBN9Y5G0o3PnhQe19FXtCrI9tDX\\\\\\\/xvUFNlLVjnuuZ1PaTH34tMavI+KuRCn559H6wRCY9H\\\\\\\/a6PezMVvE+nwgfZDR71DR\\\\\\\/9Rg9zcCrIGWMVKjmQWmaJR0yXgpT0q\\\\\\\/+Rjjaxi821FrHuW21G9QfmtiLF07Tc7HNXt9gKLQxmnrJ2nCAYmA4Jb3+cUdNPl7qo4h0m0TmEKXL5n3n3SW\\\\\\\/Ja6uF4hfBgu4AcnmSIKIqxp\\\\\\\/Ok7P0KptvqXI6xhZQJ\\\\\\\/DDTr5jWhaNXukV8A0NOK79qdRM7OH9YaEpBIqW2R7pKABK7cy+\\\\\\\/U06A9eX0h3VHDJFKSEN85KCu5gV5cTw6z+2nULwvmDLcS85Uw\\\\\\\/lkg7vD0CD6+ymrJngcXs3D9+vWeNehAZJAvD5s0Oj8UmUtoniHdaH8YD5d+m0\\\\\\\/w6eT0CWH62j9HBBaoOJC+4gggdpFyUv1OY+cwgO0qXNK1leR5+6PDxlI5n2cRzdEq6yFt5KKomv3G+NUIA\\\\\\\/0t5WabLnJOUPBniz5XzAKCLIYNWS\\\\\\\/I6XIRb\\\\\\\/UtaZ1\\\\\\\/8u9npEgk7okUDZhNIsGNeChVlRNat4e1Nbhe7VQBGNOaiSdk9EmR7SZcX45uihKhX6JP9HzrYk9ANCquqq+zfXaHcqQ9Z6IOhIrueE7uz0DcUYGLRQRlzFtTl2a8gpmD9MVbdv9DBsyPhzv\\\\\\\/+C5zVv12jQVKG9BJ7PSsNnFcrESdYSZ4tpb9DtwPqYTW0\\\\\\\/xzaMRVBJ40jM46B84Z+0FlKqmFSd2Go9496497YCYQvHb4rlQeifx2p3ufVXRIQzddurM4gD8tskpTFF4pKYS3uZHCrOzhawq5\\\\\\\/zxGFrQpYWoN+k3KiWB3skn+NRrOla23ibFEvTKbciBbgUqCPHHtPabxrIfsBt3AoXhwnjP0K84+9Vh2mmEQj3n\\\\\\\/KBCT4Cc1M00d2v8OrCPxZ7FxNjiibkYG802yEj3c9yTz7Y2pQa2GdeMfRltDGZPS0h\\\\\\\/gteYaTmdAaiZqdNEnA6yj1xTDddK1IrtANwnj0Zsb6bBkIYJSNokQMGnPXCrIzSVMVysRXTR+st5NH+HgJpjXTtimGEP9FlUUt6U1zDa8L1JFFr10Kl+lSZfp\\\\\\\/Nv5PAjWdeGgDTNVllMVUhjOWpEdPhMHYzDqI3ja4AsKTAMFGg3ov3QJqkTzNC6\\\\\\\/uh\\\\\\\/QzDYVlRpms0sX46WMNNXgFHbuIP41zw96AstrUAxhmI4wLFsAjf8Td41lqMKNCJVsMgXI6mF9AvYnyDEgdJRTHNlnbyyvmUWkm5M9hOOIEjSO3HUm8bAJWsc8oNlBY08AuW0Bg3JbVhEAnPRwxI\\\\\\\/EyjDdLg4ALwpFDyZ5cM8\\\\\\\/HVWdneHm67KBL0MJi\\\\\\\/GJ4Io6XD8AZYJdEeOQfT7JAtt0vUlcJ8qf0kDUrPC7me5ELqcotKandElwlSEUzdLjLJxCiwmXf5EMkVx6B4wRfXgTSutUEDGJBf7g\\\\\\\/9HkwQjEIFfC\\\\\\\/fCx7cuQ31ZA48++8d74K7QWgCIqDZW1fYTh7PsjARFPsz\\\\\\\/rySNABOXixI\\\\\\\/JH6Ocb7\\\\\\\/sd4cqw1p5UszsrsLKU3ybiYqNIa3fpFkPwvhUN3UbftW3jW2dufp2I6cjpupU3dvXgYuyCpzU0ArZyJ49N2GrVTl5xHrBSWrfZQflw3XfbnuboncZYTZ25JhzOt0hWCdCkZhCbjGcHH+TYinAwD+40Pt3xK2X+3+JgsstLE1zwlNoyuO5Q904iBVQVyzd4jKPoQ6k4ezNSgrC9LcCQXo8ggmjSju8BNZSJWpCpqL24MUTbHvdsvT5DYgBLK0nM1Rd+SHlxiBauy\\\\\\\/zstbszlkBaf9nP9ahdma+dVePGFtaBsbwHPuVQynHJLpQH8iqfNTw++W94rihnYKzAX3zLtNEr8HYuhsvZyROsxiNjSAwMPfZ5sYOEJyoxWdXRLFel3LqmQ1kY02Ig960cjydmfKaysAtqHAs8u3zU+QHAtPKB99AklJNMjKOU\\\\\\\/3fM1NFs2U+a3v0VcxUK70jhJeYYbag\\\\\\\/p+N4PABSZVt88\\\\\\\/hWkyx9PAHuE3W\\\\\\\/g8eP1O393DT4lq6UO0nefrZUy1ls1YaLSalpT52JsNIOp1duhX0JXHdXPslruPw2VdWT\\\\\\\/lPWt1aH0dAycjB3+myVUeqGjyl+27cnqoJBDDtjtKWU82PcpYF2Y9\\\\\\\/5nPrGXOXm7OEb9oxDODd7X33v5Oe+FEs1gFUmVsAOcLe1j9mBPcX\\\\\\\/T\\\\\\\/fPWsjNc0Ou1X\\\\\\\/9HbwfYUs5sK1sJm0R1Axp+vd3DxyPfYMXAKRPTSES+TTxtJT4XzxTOIN3dbgcaJU2kNBl6vhZigdnxHwNvyRD\\\\\\\/loYfvciGFrl+k6u4uJPBC7FFS6uxnS96PK2a3uQhnGBmFdA8xykcMffM+G2ubey+ncsBCs9Spd3BjmTkcpBLiJMUWARkHDk62VVLPzaHjUQLarYl2sybN\\\\\\\/\\\\\\\/Sn8q\\\\\\\/XGosELLqrIhMbYokKQQBunWQXCKYyv0I6uaaNgaiXL2td0l5XSLCtqKSWkSzKsqSsTfmZ5dx7YRi7sq2\\\\\\\/d38YS3F3RKh\\\\\\\/dqzL1YJ\\\\\\\/uluPPF4e5V\\\\\\\/Boy31b9fYcxqGIVGIaG9O0u9tXZxTXW0s0Rpx7\\\\\\\/3W9l9J2emvc+pnpU\\\\\\\/3p5YDpX3JRtkFMlIvGJ8U+4CgwTunh3S88pbUt4eMtxqccoYr\\\\\\\/fPV+hSAFzLw3ICWeOK39MSwuRZf8i6Wy6\\\\\\\/OFyNt11H8n6PgNi7liAmPPTfByYKFy8EjE8GWo1M64Ryf8xy7k+ZDjXhN1HxIeqLXOIa5ak7NUBQP8X8KrjTuZw6QxzsjPH5V9sUvarfkh4H0Gb45Ir5Bbs1wH0COkUeYhTEWWPGWmqi6CDo3YxvAGGveSZemBusS5W4T5FxK0qBf03efBaFP8ZC+455QAV9e54RSeIfKhvu99splGspVuznZondcgit85pGl2KB+HIEIiQ5VfPs6+Vyi2sm6qiVA9rbBX9+I1x0gdmC69U3ykvkXdfWChO5+k1qn369BbCA+ryYGvKbc2b9IACLl4CvG9FcRoBrNpnfKkvEoUjAREtuhX\\\\\\\/SUuqDtP3wd+tw5rO\\\\\\\/mEwWBf8Q7Ci1UQk14YurRsi8ZuesrgYAlR1WWxOGwzNY8JaYwQjZZtXMstHQWb806i9KCirwjizLC7Dq6jIaGDY+l37FchHnvJewOFR0m6i4N+y1u0pQvSndKGEE92qtcAlywGV1Yj8OEmCC9QehdpdxZ7NwXniHw2Qkdl1YuyOehpbvUdEUmCLFBbhVnjbwJlzX8JJd\\\\\\\/DMqDNAE9uV4POlHos+6gDV+LxX3wXOJdJc\\\\\\\/wKF6NTKQL0WnHv\\\\\\\/5McoPQ68gH+zmFlorsRuI\\\\\\\/9w\\\\\\\/eDrY3COb8lu89Kch4x46KY\\\\\\\/HJNhRkjiE5JZ+pNg0bdnMsVD6OdcZLO2sBcuIk4fWhVflVUMyj9wfGh1EUWfsn7J1O5L0iB3QMB3OlzgThdOJGppZgfOme+wrshPx2XLYuo1YBuWdGir145\\\\\\\/+5XyMnVGKLiFuAeZihcAraEHq0laL4bcDhxf58d5uYg\\\\\\\/OkhhspAEcFNP+2CH25z9WS331veW+rpEGpqZprYsXB6FjKOl2EXf+JE2IqNnmsqVfa+moIo1vr55gT+REQaRawV\\\\\\\/2vLmwLm2MCwF0nFfUIkh6pZMBS7\\\\\\\/pk57u1WoIIPQOo0d+KINm4EZReDGb9HVh1MaBAGxqHDh5hdhRuJpWAoEXUMTKQGXVdWc4L5R0td665qzPsyaUB0SBPS7hvkon1BcBu2IO7kM7O8EYiLAvRE4aG5KFbSVQS2vZyATXYOYSJyW4WYU47OCOjhlzbQ2WCUDMMU3OIsNGT8yfw\\\\\\\/hqRcGG126NnXqVPExLsZk34sdqrVbBAfIep5GI1uQPpZQ9aSx3Q\\\\\\\/f3mZA4MxcW8wtpa21AmbSjFQsnJUU07CHV3mK0jRZ21JofzbJHjKmXAIU1ZW07bT+LDtBsngQCpKBADnrbq+Vri7ZiBjVMr+VL6\\\\\\\/llPM6wPP\\\\\\\/5VDPCx69vzxSL+JLwJc\\\\\\\/Djp3rCIqN4lLnOqXEvw32D2YbksOvFBNIF93g308wtiBOIptdgyIz9NsvFUCAftcY9h7Gzu+FdCopn84CAG+4GZCjs4KOI5y4DDE9KQkVjGe4ycPnEtxF3rc4EIrwNfhpVKA2oq6uUlRz6PjxfZfGIgUSxeccAq9oM9fPSRLAvUO87YtVMr1WZytS4\\\\\\\/BbBS5RN0NnmaM\\\\\\\/VJm971\\\\\\\/evMtWP1+BnbAZWkZwQKS0cErtmNwJV7QhtMp6G9ep0SEONdBDwcUgvvJB\\\\\\\/H2ODt9nYi38RMWSah6Y\\\\\\\/qn32+pL5BI02PQ3zRZsg5KsM0DcGW1vwojPniQ1iBIJ8yDYj6zKnAt+h5L0ceZu9sBPW2MgIIrIcbZb1nffIJvZmsYxK+ltxl8UDv5RAdbxY5PM76gdOyXuTed5c8i0\\\\\\\/zh+M9sQzMRsa9ymhvCWOwIpQ3yhyPQmx45j0YXWwvy\\\\\\\/vWTA+Jgljp+I\\\\\\\/c597cA8\\\\\\\/cTmmINVxb4UmJrBAeyXbFcvWitfZIzwBk78nQjfN8tne6dEPo271GcOIzmr0+VOEN9e9nVH4y+MGEAMwQb9gEsh2WeRmvvnVbTq\\\\\\\/7TtxZ4vlboRIKSHzGDlhod+mMEmcK54ELDDD13mejN4UCMj8WAqQQjXxdTPVCizACCpVNsgG6hjE\\\\\\\/QnjgPgytzEydq3vrNHukXDjZt8iJPRIybZISM5cTobwGcK4KGqtz\\\\\\\/zjrg2BSzLXk31EDvwuTxzm+E7YA34UtegpimYD6oiMBcZFFJd1oh3VtOaJPgfq+APDGhuVtSkrluukh\\\\\\\/ZSHGQmSrGqhYCPpnFp5EJRqx08Iokk\\\\\\\/NCLZ1u5Kg92PPkkJyT7seq2X8eOti7IGb6XLADKcsq0R9VRozkfz5wMRzWrFUJQEL5ElQiXypmgRpu6qdwZ2Z8a7cvXSv2NNXBAFA3sn+wXnmNFwzJDGdW+Jlpf80F8XNWg+pVY+RviF7ciCSUxsrIPxZHMJ2Z8ToX9z0c9TV0VUNY312Hd2lya4sEv5pd0xdogVY\\\\\\\/80yZGg4Bs+mvOdIXamHc0qEkkH8MVZ+\\\\\\\/4LRsXhxR0eeIlYQCLpSI\\\\\\\/CCTcC8Y4dynfHOOAdvhqBy3\\\\\\\/xdeZBMYB1tvmnP7qrlbi1+suX5IwLu7q7UykIAp6z\\\\\\\/rpvedyhZjQDHCTNKB52PrCrBH4LahP\\\\\\\/QlU4Ya0PQT3J34PTQpj8\\\\\\\/fpzNE\\\\\\\/G79dceWBBdcIruwwoM2pKZ2lbRhV+cDO9oHRxhKpS3D5PqYmVyipU0x7B+5srTLNlA59cwanIskLSTJsHpgoIxf66N++x5ehPJHLyvtzR3azIKX3PmBVJGmn1UMG8j0+XwWsdisSL5mtdj34SxullyEvDy7VkLw2H9tZY56+sh8+4vJ7UJ+MjP\\\\\\\/bBAsypS\\\\\\\/NnpbQ\\\\\\\/5s7wVZTaHwFFxiPh5t1lmxnCNp7Hl9HR7aQBlARxyUGDBaM\\\\\\\/XcXimMg1TtzaVB+wK0ZHxw5hBkxSG\\\\\\\/fStxxUHgxP+Uge2lzwQXt6S7NGiv29tkFYdvufSmbbFlclbA\\\\\\\/tYN6OheKyE6t79aRYbBlL1bGQQEaMwmFTwHNCV5o3xutr94eOPWrTlp68xj70UlySIqYaAq6T9WRmaD4kj8k3mIBzEjZ+IFSlv6uUDSezRlqnS2fWlKMegxJwpMyXTJqnq4DThD4TxnpUognBMWFEx6Y7wsoNYfFRH8pPRMl02rDtQleVDG0cpxsP0UBS6nMllUrdIK\\\\\\\/Cx\\\\\\\/TYztw8+vnkSXrDmq4fui0ORZJsmTDMBQxWF2\\\\\\\/szYqdsOq+uanBY3wiexm+qzAvu3FsDF2lVaQLSrjfA4Qzs0GDMwQx15GJsR38CndX5otiilzVCUBQYDMXsEkpg22V3nQgZylnFqlVCpWG7fprLiU79s5AERgkmCwXzQ4xbhSxKE11Jm5wDrZB2+oy37ck3ZI8j+lsAn6M9kTWAvJe2rdxkxCy+xqmK+ICs705ix+gILZJAKE4enqt4\\\\\\\/vnpJimXf8B+KpfUQjgK6otENgclQ6fI907HsLYQBiwD7ek3g9eGMG8kIGn9v7ltJ+sBC0ijg+AQDZmipp7giJiBzfoSauh7DvJ2pyuB66TejeSEfrffXuKvMtBKQwdtTX3kI6GsXN9UUsleUNeKsGH\\\\\\\/RxpregbMflZnwh68TWGaIflWy677PX+w3Za9O147Fw0Jsj\\\\\\\/5GwhlRz0yvxhXrwrVlzH\\\\\\\/a\\\\\\\/+QK\\\\\\\/Et\\\\\\\/WRZI0o9XgH+YC+8Of+F6UCrHyThDZkXR662vra\\\\\\\/dRQ+H8pyG+IhPQ0bVygvjnBmn4kqqLwBfs8vik\\\\\\\/q\\\\\\\/yAm9kheozcG4qksrmqGCaZVJBCm4rEITU1R21Cz5POadvPia1s2taVclxtHM4+m959aWpZf8rZ8DYCe9Ya8CJoXiSzjtYcX04WOrPNwqtK0iVRIy9LXnsltYkj7SUUvqtwTzSx8mrvKDS0PRaSyQveagzgsWh7umNDSva1cTkvPPjwEzPuxRnhpgDdX0sfSDeXetSE2rOmLIT17PHUTUlztocquKrFlwCXaCvcx7tG6R1wsB4dSJgt\\\\\\\/mAub\\\\\\\/IqNd\\\\\\\/nCCK3RKiWBadqkVnIPf6ZS62D5ixngxwRwSNDuw3ZOHN3d8fBcf98NnRrIYZeRvma4Cm6Z0zUXB0HwpU31c99EcMz08U18qISCavxw1g9Uc9hsW\\\\\\\/\\\\\\\/avJU\\\\\\\/\\\\\\\/K9zmN473sVgwUtoUJX+\\\\\\\/FG8OXyUYbjMezSQxp0Jmgyl2OvzIGZb1tF4rHKGuXY5SLQc8jgv1H29yDaEx0a7EsR6+MAONOZb8djtPVbWKKiA3wgFwEtwX+nu9aTb4Saiu4DdiHz0SgyRXOOWSB8dMPS4+nHBDQoBSkGPe0P2F2lWz7u4KOTP1TIGR1x7IKVl4exk2NW0wCUSOUlSoIBgUxYXoqEdSzxCuzDBI0TbiqYy4D9pqJ9Cm6vw1tuyLo5\\\\\\\/jK\\\\\\\/kpnmr1OXMc+L6kKdNrPUDiT\\\\\\\/UqIfU\\\\\\\/uk+WxvY5iL6T+67sQG53UxLzfIh5Gm\\\\\\\/NvWOZjYYB9dAQVZGtfDrudcPdWXAUhAC0w3IVKqFBwoay7CFLO\\\\\\\/bnDc3PLBvVGwlKh1\\\\\\\/UB+qYapGSuXUIKqgGz++aeKNJbSD4qmzuFdtmjuEW\\\\\\\/tQ\\\\\\\/unCZYelonLA2A7aGZJc+xJy\\\\\\\/FvO+f9XRuiStQhbUFfdnql57KWyDEBMIX0pHpKsar6WdGRce+qSCOuPysxp45LO0mdWH+hKi4y9IjzQ0wgarAjV7lS4+xa0aZFCGGZ94PakraxLoAyoxBFzmQWdv90fc\\\\\\\/mTK3fqnIDvJ6d+2pNxCUU\\\\\\\/9qYRfnnDSwh6uWraYP7Bslr0PGZFWTzIUqqDDmq25Gs6QfyNelELv4yLuoFVFuGY7+b7rolj2bNBKtnfAMO1If5RsARFCi0St15YfMfrziUwX9cs5gHlGfT0q6z7y+pO1l7KxoQzGUAHFAx6z8Q8oydfo\\\\\\\/fR98umxV9oUQRCtQrSJgStegAg4XyW54R8Q7PoBHbVd9AwYLyfFhdr9ZzcyF8ve8X1N2Ck8mEz89XxMlxMayILhoL5KDqkHJcin639MYRlNnszEW4lLdcknAEotxmYpv9VS1Ssx4cf6Hz8QYuFZ3\\\\\\\/vqBdt6Iv5H5iUFn2wSS0opqQnT803Q40p87s6bTwtXZkUfh8r1CqEbAL7fnaGFVZHxTnUg\\\\\\\/9wapfOnpTPnO+d25knXIEoxAgS6qt9\\\\\\\/TZ\\\\\\\/7oKQkmeTw9Bhm\\\\\\\/3oyQTb7RbclwVccH8aN0Ex56838NbhWbUujBJQTZOazfoOhm0hw\\\\\\\/jtDSDaZAXRGAUJzJhdhXNkjLT27JFDqLFp3gCaMcKbqaCHiKIrWo8UyB5FBcj6IcKM6owVkdBsCk05zhR740+oABldRTaWYx79+WneC1Advu4Kadklf16ypK4fTRwYVH0IXZAaOU7CgbC\\\\\\\/IwZNTYkVIjGHiuLv5K+wtRAiBS42Y00cAgKouQ5qp2eUhorb\\\\\\\/Wep2zQt06riCpIxSrdJGvrQtC\\\\\\\/TtrZ9G103R4aXu8tadGPeV\\\\\\\/ycnCk18L6taUPrtuvfVcOONylrWYqZG5brKZWH73kiCLxmaxWKRKOr7FtYQTuh2UrT1DnODPe4Ln\\\\\\\/VCDvgMww9O0luel5MBdUs68dixs\\\\\\\/QztldjTLvNtWlpEYJ2lFvKxJbXw8A7hfUMNmSBIEV9q8H8susujAmhOarbFxOXLBvX4sjYLXB040\\\\\\\/nK7+\\\\\\\/agTmLuMCf7x3yGcJTF7RlCbZsbcAn4Qs960jtaQtike\\\\\\\/wV6UjLjCAku6Z4kR1AQSM7XkQSYERjLr8YBKWcTmHGpvTrm8BFmPAvId0tk\\\\\\\/KI3PC5lmWmL6BNay0CcTG92VBaK29B4YpzVor66qXU\\\\\\\/RgGKZbVo3HoSgaLf80u62cNUrJ4I0B5raMmEHnIULt\\\\\\\/hifz2mFUPOl0IPNa+iFqno\\\\\\\/pylw04Jwu5KsuhvBBxfyy9ZhMOsT2\\\\\\\/zcdEpcXjMq0etIccq5\\\\\\\/DURl+kWx43pLPcz3w7LrS5xAC0zM4Oc2dyoape38Ra3TxQYear6JCAGPschgoix3XGB6WqvGbnQBNVGu5NV69bKhBium0fsXvJxCooE3AX3KH33lXjq60XPy8Z61gfmdtCPfYMO19RzsfTppGI6\\\\\\\/cxEpQP3v\\\\\\\/lLbqggJdHjexkveWqmPAVYb7EJnM98XzxzZImZUbQobkpxPwEO3h+DiPrtZp4QpgC9CZvz4JxsoWnebuWl2EqgdQdj5gbdADyYa1nalRlaco5wYX0NGTkG5P2jVMAz0NcZZu0kyJBOP26S5cpS\\\\\\\/Yo\\\\\\\/GuGq\\\\\\\/vHfrbQt8V0t9iXB31uIGgAWqxcwwfrY\\\\\\\/d1Z2NJtcRvedL0EFUp7XlXgODuiMpEYRPR1g\\\\\\\/ytUBxLMVQ8pZ7Ccflt+IwppWow1rS3X0Vj8x9f4zBC6qXxspXC\\\\\\\/pfbxAno\\\\\\\/OOrJRr+yZB50OP+EAlnfFCFd1XdmMOlgpY\\\\\\\/MXlJj5Y6JNpXAC\\\\\\\/pi3go6IqRrCmnWYPzrDvgewFXFVT2BLP5yIT8NaonFpfNrqfq+72IFqHLhX9h010hyWE2nkJfGuYb68aZV1rNhIFx4XNZ71ljabTY47yWwgm7Bo8XYGe2fIVhjwDUbSHIhlzCIR49JVA1VTYelbU9KOqdoD51LdMshT2mIAY3ISu5TIneHe2HepS9CVkwekn3MESFRCWUFQBaR\\\\\\\/s6PpP63CVypwoYLMV4zgKYoWu+9KCtefgpPSJ5VrZZ141LhACsdsdqcfm\\\\\\\/64oeBanR07O9InXr259eZ8dJIorxO5moIJb1x1MC1AOJqvPzalipAupZ5p16ZJvBecGI44vi2ogsRQ20\\\\\\\/BYowhGp2oRmdrNeO1aVSpNZ8D81OW7UxIQ4FGQVIXOOIZCVCD+zThssCNmwWDR8OSk\\\\\\\/j0qwuJ59W\\\\\\\/iUe20n2tPw1tTzQ5fvpQBUV9A5WbrKRjAD2KwLj0q\\\\\\\/wmJoJSZk9dQ2dkbeX3GmezSW2TfxpiTcHDGD0j2MNmYfgAmAQxLybQOb8TX8Hb81GmkYYj5dOFGULQh1JhbYvuqT\\\\\\\/xMi2+Dk4YtqIhfwO9qJJ1o\\\\\\\/hfLu0XGRBUWFaz2rbd7qunkZ15waXuiQmQJm39pZpoZHDvIeS0IC3np2+g5SrDUXbK5fM0Sx0iLVbyCH5242MRW15W48HPiw51BCPDQMrsLL+V9UaN7vNC2hEfkxKmvea7GGDgt1C+PTBhFidczqJ4OIGBzAYvMLauZTjen5V0CpXaY3nHebYKXkEUoqK\\\\\\\/PayJsOiokF\\\\\\\/e+FDPGpxoU1cK7j\\\\\\\/oUkJvYvXsKyrDfOni1LAxuGiv5o0mPz5cOF13fhcaKZvQTu1hfBNyMMutrDpIFHKV7R3P70Cd0r8d8r0+ILHiWOUYoTaVjOmz3DDJECwEVZkpUkL23dGw9VGC9H\\\\\\\/9SgehK6p9mbbPF4jLJiD0ylKXcAH5\\\\\\\/YqQAPdwPJm\\\\\\\/+S9gEG\\\\\\\/AnBRVurh+qOag4mkdG4P4sAg1U7PrvHblqj+lK8Afs87x308sZ9pFBYKwWCVphiCuju1L\\\\\\\/4XRLNJBKoAhk+cwBNPf0eL1ifl1yUgwZUqCaSMtmkFztWSbgNAqCYlUmNMMxWx9RFPmscv\\\\\\\/xxC5zlhbJ79OGSB+REdMEbfTTnWNtbNQ2ka2ZiPp3mlxseLd944V1JBO3jQ3v\\\\\\\/TBhyGsH0Yxtn31\\\\\\\/P82WpFKSwtTY4aqj1ScI1w6XDHcEcc\\\\\\\/v536eQTrNGQynuwh8DuhT62dzXbHfPH5qsPvC7VYpc0WnabVxvhGbavP1NCs2IuvEzQTr+8uOn+\\\\\\\/pQyg7ATXwHX7g2R7i4dl9hQA9PJ4GOTpt\\\\\\\/nde9RRuNUCOJlR9eCPtxbiOIMskAn+gTcYOeHSnw5Xi6MztMh1qyFPlvrZAmVRSr07x+CHxyCBt\\\\\\\/p\\\\\\\/dfNAzESFWv4UXhIz7JKNQTFz+KCJweQRTtbBQuFEaVN7MOzrHy78AkfFgXUYA6s4wEyPwXQ6snxG01NoE9LBkqHjMK70Bfv3lT7CwICF6pPWpFE+3ZWxLYwFcGr3R1XIuK5z1KGilqcrOFhdFEM9WnIv4pVE8cShkXCVYSFKZaHCI19p7fiPzOu1\\\\\\\/sqSgBMIuQ1SKFYmrqZNkjdvK57EyzaRflKuFOlhXirK\\\\\\\/YH2keyA7+rZnxBm0NpF0Utrzi0212QWuMVJqBLAGaXTPFH27Jec2b2Dma3osCVGm\\\\\\\/kTQEnLWt1TNVJxJrixtbiiLM9URm7a6a7n4ZtVXU4uVWa6dDQ4b6cyfRD+SwVcb5oHQazllybQXCMuCqq1IPB6OlfunMLXuc8WWLmQOc2ea62vsh68IpRtQdRJKjyzdlirLHCZFzfp9M0MrDpahVL+leFxGMoxuoYrAmiSogVSub1\\\\\\\/SPRR8fGEJoCEDetC3joyQj6Y4XWqRBphcHrNNQ\\\\\\\/dnsymGjIkl0JVrjJzhEBHvb5CLxi3rzzl0BKoUlkVLvV+Sr+e8losjhEB0octxVR\\\\\\\/2R1ra2XwQEALbLQxBHQ+Ypp16lpG5cLBni3FPXreFg3mQ3mOxRdq172NxBqZarOTr6t1jmVgqF0iAf25c\\\\\\\/V\\\\\\\/CQAlP59FRon7L3Q9ZSWPPIuo1Fed51cEfv1neeYDujT419Ss6I6oycwA9Al7zUZPOdQnOXZTUFHzbdNE\\\\\\\/xB5BXQMgeooSJghhXqu53PaGrfbte0YWixiiRHtKrlE3k2ZFJMy6eq3y3rUelyWqbpYLEE9gsSQeAN7zzY9anZrSPb95mFBupTn7su8PmBRWKUfxvlHM0ju2wAw+sD6AWZkmrljq36xlij3k4GOoAQ38ISRSMagSEhykAtvpp\\\\\\\/v+yOrBzYv7ViTg1zClpdU+wI6HWfUNtlGwHeqaoItiJwV4Qq2YGIBR9CunkLXnVzyoX+Q130F+4foH7jzdodcri40gCBo4hkyKGMASmC79sXWSVzxTbX0Qk9TF5n\\\\\\\/XgEMxPwOdPhEGFxJ20PfZStViAq3LoIHgD3PyyTBmk+QElUWDGxLp0m3s98IFUahfqWR33rUYeebLyU3m3S\\\\\\\/ZSBYc0fqanboK2aEZhBnUzItMqXWTcyy+0DGEZJRTPcEbhPditTkFTrg45svuw6acgiPrHMyB6wURw4CpyyjNP8zXz+u4P32+dLCsg11rUbUbyx\\\\\\\/q9QUhAFYlpmOX3ieKoZVS1Z0GWWgLZPpAB53rFOPs3qB9pm81A7kkn4S9Pk9ls2TA0yMRH5rZ8S0xGOXLKXeqStpIExuKMCkuVOkAg1RkHD8hfPPYaQWBY2+aMYb7rArgGpGj3pLR3xe0NQ6oDtRHjjE80olE84FvGfHKWkekFEkZqqAMJbAakVt7qSBp9gup0jAZedflXN3FcX0wEpF\\\\\\\/MvL7X232Gfydz8qYu+FsJA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022QUeVaTQDU7yWZ\\\\\\\/rPTmY6CA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002239260c0cf47bda90314397df155d150e120cc948aec276a798c942d3cbb86489\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022P8Akvpl9uAxXSAOPpXQqsqlDrraqNeFKrgPFRW3GFWCtg1NjAFM2vKO3POvrew9GzB+amfrwyR\\\\\\\/hrsI99lD2ZwVFlfvAv7dmkf6GYdtJN5isSYEq0Y8nFGUO262IsD6wMpXaENN6iaPrsRWwLP\\\\\\\/wAS\\\\\\\/sO4RTnKPlRLiuQmhD341+nYA27Ej+WgSI0mMh8aShQecUMYfL5pGe7UC2jFqP1BYOc03qexn9KqMC4Xb8Rll4j7S1XyId6B1ClpjMROYWOkKwYfJ1WjlnwpH49oXf8NHh6\\\\\\\/r0jvM5\\\\\\\/g\\\\\\\/MNgBW8fGQBPoP3il6uUFHv+wV1C+iW+G3BcqxZDbOLNtLg43zzgH5Ql8CPZafnj+My7Fa9EV1zuGF2e6PIiq1HAM4ArDb0mfuGaCZFq+E6zj3qGYpoM1MATuTBIWDwh8DD57fOi9EHf+UavDXdEtLqqHM\\\\\\\/Rx1\\\\\\\/0lYk7COgzPaXqQvQCsGk4SIUJmpS+slwxcy\\\\\\\/DJz8SLq0nzI7KrsjcLpZ4iZz1FIChoFL3uhAskCS5LTb4OytKbo79VOQzOIjs2ZjreG4hei2S58JhCBTafZFa3a+d\\\\\\\/16uBA1onKDRCAey0+xJeWsXAdXzGodDw+uQAOXciTr6G0in4KaQO3Ni+DApKLGfHBdKBTziQxFyH24hjPQQVcvCnudB94Rh3nbVKfGUfKsFABw4vTrF6y\\\\\\\/G5+yoz\\\\\\\/GAp81gkiDWmCDh\\\\\\\/m6hwNCMhXfkJXFFeQ\\\\\\\/GYSChowkaLXu8BPv8nyB3ggeWjBWOBEysFhFVR1Rj69QvCrdQBycGybGP78vW2bdxKE7tZ5FUQl0w2FgiVu8XzNiBxXrrbYOtRw2mmLDCLtoIpniYh\\\\\\\/XQwg5wyCfbOFipK7yHmiDz66b8bhPzHp0Xml4RM827O5y2bQNRpYsEXkTZSsOuhE5rfmAaHpUS2AWalapod4L+1szdMIpGaxUbilbuh2oi+RMAsfcibL33vfCUiRU9vDKfRIyyimB6tNBEdh2yCyGSrujCkUGJm6twwy7ywLr11J4FPLXB3ITve4iPz35AhXg7BAJOmTLrhMlgd3fdIeH3aRcNwVJONt+Sx4p4Uyfv1X1yAluYwOqCgnAy3KhfupGx3c3svPey9zwBoiZiwtKPNw2lHHuGSmcHJbkfYHPeDWPGm+ncaalmGzl+xTO6FrSuR7enAcPxceJWwKVUZczqLFKpstUaJN2BedW9kQfWSs9bz3D02iK9m3NehlZNgGt1\\\\\\\/FjSZVcgkLJ49ECsclVD+mYdXY0A2JGDavZZ5CZBEGFrM1vwefwawx2DTvGL3ydah5WnxHX6PrFFmkvMLD+9sC6RwW31tkYBzojNlEW3xJzbnbRNwzqWBgXqBhedaAg2a7HUjfJXP4PRMo0Jxf4UPzLiPibx35fI\\\\\\\/f+RzdNtMkyw2TUknpAk6TXxLM4l5dSyKnXAuJBqo08jgIf1GOAj5AlbrMxs\\\\\\\/yvdWxbCjVIdeVg44ANxz\\\\\\\/6mE7V7QZ+9lwZetY4pKQzxe5nKRBK6GWA4S0IybY2fQVUK1FulvAScCQkYjg2tO901cGFcxjAZTKtFz9LfhBTNX6EHyCwvg8PkAhvxkhBXmmSWcEX+rB0K1aWhF0ermwOgW9Af8Jwi\\\\\\\/lpaAOTA8J1UUJ3hx1Tw4WRhovnSvRbqaX\\\\\\\/1V+Z5ipVo+\\\\\\\/+yqNHascwtcApTc2tOC1O87HqKHKDiGRNjVvtB\\\\\\\/VKq299ZYeYH93f5dLq+Xgn+yxEn5VZy5tzsR2fCHRJDRz0CEcOkqU+cInXX2v6qJvXQNjafrp6kAZFTEOYsxmcCE1SdmV7rt3YVzRI0IBmzIjCFk318GzJZEBoFp9nwceM2ZqVNyJIw+D4qxhexlWSZiOWtG3xuEXe4VEg54LgXKTk2UF\\\\\\\/aRU04daV7d3HP4S+m4SYto3dbfzDPO3lkekSKPPtdKlUOe6CYVv5k67OCz4EXtTwL3gsj0pOchcbgQVcUJjo\\\\\\\/+MNvOaNn\\\\\\\/aKYNV8JbCNdIUP7PtEVBYquEgq1l5ZGojaQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022htSgo0H6JBFkZpZFT1FT5A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227e58f24c79a6fac48390d06e92200fb2b5cdb837fc8ad48f00fd913f7cd7b2d7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-487826 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/161" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-IPUFpPM5rTdac0w-jG1bYCU35vdoCrXIxwvZ7WOdJY4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/161" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--EA4950EL3FIresbDjAUwpn55EUnMteHs1a3kmGPPxQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/149" title="Severe Fertility Effects of sheepish Sperm Caused by Failure To Enter Female Sperm Storage Organs in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/173" title="Rapid and Efficient CRISPR/Cas9-Based Mating-Type Switching of Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487826" id="node487826" data-pisa="ggg;8/1/161" data-pisa-master="ggg;g3.117.300376" data-apath="/ggg/8/1/161.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotyping by Sequencing in Almond: SNP Discovery, Linkage Mapping, and Marker Design</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7539-011X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Shashi N. Goonetilleke</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2066-9495" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Timothy J. March</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-1174-6580" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michelle G. Wirthensohn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0939-8038" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pere Arús</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7596-8484" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amanda R. Walker</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-7506-2589" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diane E. Mather</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">161-172;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300376 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487826" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shashi N. Goonetilleke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Agriculture, Food and Wine, Waite Research Institute, The University of Adelaide, Glen Osmond, 5064, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shashi%2BN.%2BGoonetilleke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goonetilleke%20SN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShashi%2BN.%2BGoonetilleke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7539-011X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Shashi N. Goonetilleke</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Timothy J. March </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Agriculture, Food and Wine, Waite Research Institute, The University of Adelaide, Glen Osmond, 5064, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BJ.%2BMarch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=March%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimothy%2BJ.%2BMarch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2066-9495" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timothy J. March</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michelle G. Wirthensohn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Agriculture, Food and Wine, Waite Research Institute, The University of Adelaide, Glen Osmond, 5064, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michelle%2BG.%2BWirthensohn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wirthensohn%20MG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichelle%2BG.%2BWirthensohn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1174-6580" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michelle G. Wirthensohn</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Pere Arús </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institut de Recerca i Tecnologia Agroalimentàries, Centre de Recerca en Agrigenòmica, Consejo Superior de Investigaciones Científicas–Institut de Recerca i Tecnologia Agroalimentàries–Universitat Autònoma de Barcelona–University of Barcelona, Campus Universitat Autònoma de Barcelona, 08913, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pere%2BAr%C3%BAs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ar%C3%BAs%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APere%2BAr%25C3%25BAs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0939-8038" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pere Arús</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Amanda R. Walker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Agriculture and Food, Waite Campus, Commonwealth Scientific and Industrial Research Organisation, Glen Osmond, 5064, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BR.%2BWalker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walker%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmanda%2BR.%2BWalker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7596-8484" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amanda R. Walker</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Diane E. Mather </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Agriculture, Food and Wine, Waite Research Institute, The University of Adelaide, Glen Osmond, 5064, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diane%2BE.%2BMather%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mather%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiane%2BE.%2BMather%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7506-2589" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diane E. Mather</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:diane.mather@adelaide.edu.au" class="" data-icon-position="" data-hide-link-title="0">diane.mather@adelaide.edu.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/8/1/161" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:487826" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:487826/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/161.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:487826" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:487826/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/161.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:487826" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:487826/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/8/1/161.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:487826" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:487826/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/8/1/161.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:487826" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:487826/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">In crop plant genetics, linkage maps provide the basis for the mapping of loci that affect important traits and for the selection of markers to be applied in crop improvement. In outcrossing species such as almond (<em>Prunus dulcis</em> Mill. D. A. Webb), application of a double pseudotestcross mapping approach to the F<sub>1</sub> progeny of a biparental cross leads to the construction of a linkage map for each parent. Here, we report on the application of genotyping by sequencing to discover and map single nucleotide polymorphisms in the almond cultivars “Nonpareil” and “Lauranne.” Allele-specific marker assays were developed for 309 tag pairs. Application of these assays to 231 Nonpareil × Lauranne F<sub>1</sub> progeny provided robust linkage maps for each parent. Analysis of phenotypic data for shell hardness demonstrated the utility of these maps for quantitative trait locus mapping. Comparison of these maps to the peach genome assembly confirmed high synteny and collinearity between the peach and almond genomes. The marker assays were applied to progeny from several other Nonpareil crosses, providing the basis for a composite linkage map of Nonpareil. Applications of the assays to a panel of almond clones and a panel of rootstocks used for almond production demonstrated the broad applicability of the markers and provide subsets of markers that could be used to discriminate among accessions. The sequence-based linkage maps and single nucleotide polymorphism assays presented here could be useful resources for the genetic analysis and genetic improvement of almond.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/prunus-dulcis" class="hw-term hw-article-keyword hw-article-keyword-prunus-dulcis" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Prunus dulcis</em></a></li><li class="kwd"><a href="/keyword/single-nucleotide-polymorphisms" class="hw-term hw-article-keyword hw-article-keyword-single-nucleotide-polymorphisms" rel="nofollow">single nucleotide polymorphisms</a></li><li class="kwd"><a href="/keyword/allele-specific-molecular-markers" class="hw-term hw-article-keyword hw-article-keyword-allele-specific-molecular-markers" rel="nofollow">allele-specific molecular markers</a></li><li class="kwd"><a href="/keyword/composite-linkage-map" class="hw-term hw-article-keyword hw-article-keyword-composite-linkage-map" rel="nofollow">composite linkage map</a></li><li class="kwd"><a href="/keyword/shell-hardness" class="hw-term hw-article-keyword hw-article-keyword-shell-hardness" rel="nofollow">shell hardness</a></li></ul><p id="p-3">Almond (<em>Prunus dulcis</em> Mill. D. A. Webb) is an important nut crop with an annual global production of 1.2 million tons (United States Department of Agriculture 2015). Almond breeding relies mostly on phenotypic assessment of parents, crossing between selected parents, vegetative propagation of progeny, and phenotypic selection among progeny, with only limited use of molecular information (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Scorza 2001</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sánchez-Pérez <em>et al.</em> 2007</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gradziel 2009</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Koepke <em>et al.</em> 2013</a>). Development and implementation of modern molecular tools could support genetic mapping and the precision of the almond breeding process.</p><p id="p-4">Almond is an outcrossing species with a gametophytic self-incompatibility system. Genetic mapping in almond has therefore used the pseudotestcross strategy (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arús <em>et al.</em> 1994</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Viruel <em>et al.</em> 1995</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Tavassolian <em>et al.</em> 2010</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Font i Forcada <em>et al.</em> 2012</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">2015</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fernández i Martí <em>et al.</em> 2013</a>), which provides a linkage map for each parent. The first almond linkage maps to include all eight linkage groups were constructed based on application of isozyme and restriction fragment length polymorphism (RFLP) markers to progeny from a cross between the almond cultivars “Ferragnès” and “Tuono” (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Viruel <em>et al.</em> 1995</a>). Subsequently, a reference linkage map for almond was established with the application of isozyme and RFLP markers to progeny from a cross between the almond cultivar “Texas” and the peach (<em>P. persica</em> L. Batsch) cultivar “Earlygold” (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Joobeur <em>et al.</em> 1998</a>). Simple sequence repeat (SSR) markers were later added to this map (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aranzana <em>et al.</em> 2003</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Dirlewanger <em>et al.</em> 2004</a>). Other marker types that have been mapped in almond include random amplified polymorphic DNA (RAPD) markers, inter-SSR (ISSR) markers, sequence characterized amplified region markers, and single nucleotide polymorphism (SNP) markers (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Joobeur <em>et al.</em> 2000</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wu <em>et al.</em> 2009</a>, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">2010</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Tavassolian <em>et al.</em> 2010</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Donoso <em>et al.</em> 2016</a>). Among these, SNPs are particularly promising as they are the most abundant sequence differences in plants and they are usually biallelic and codominant. They have been used in many plant species, including almond (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Wu <em>et al.</em> 2009</a>, <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">2010</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Donoso <em>et al.</em> 2016</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Sorkheh <em>et al.</em> 2017</a>).</p><p id="p-5">With next-generation sequencing (NGS), it is possible to discover and directly assay large numbers of sequence polymorphisms without prior knowledge about the polymorphisms or their genomic positions. Given the size and complexity of plant genomes, NGS-based polymorphism discovery and genotyping benefit from the preparation of reduced representation libraries (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Miller <em>et al.</em> 2007</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baird <em>et al.</em> 2008</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Peterson <em>et al.</em> 2012</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Poland <em>et al.</em> 2012</a>). Among various available library preparation protocols, the method proposed for genotyping by sequencing (GBS) by <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Elshire <em>et al.</em> (2011)</a> is simple and makes it possible to discover thousands of SNPs. This method has been applied in many plant species, including peach (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bielenberg <em>et al.</em> 2015</a>), sweet cherry (<em>P. avium</em> L.) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Guajardo <em>et al.</em> 2015</a>), Japanese plum (<em>P. salicina</em> Lindl.) (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Salazar <em>et al.</em> 2017</a>), and apricot (<em>P. armeniaca</em> L.) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gürcan <em>et al.</em> 2016</a>).</p><p id="p-6">While GBS can be a cost-effective approach for the initial discovery and mapping of large numbers of SNPs, the quality of the linkage maps produced is somewhat limited by genotyping errors and missing data. One way to address this limitation is to develop allele-specific assays for SNPs discovered by GBS and to apply these to the mapping population to obtain more accurate and complete data. Such assays may also be useful for application to materials beyond those that were included in the GBS library. Assays for individual SNPs are particularly useful when only one or a few markers are to be tested, as in marker-assisted selection for one or a few loci. Among the many technologies that can be used to assay SNPs, allele-specific Kompetitive Allele Specific PCR (KASP) assays (LGC Genomics, Teddington, United Kingdom) are now widely used in plant genetics and breeding (<em>e.g.</em>, <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Babiker <em>et al.</em> 2016</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Rasheed <em>et al.</em> 2016</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Tan <em>et al.</em> 2017</a>).</p><p id="p-7">In this research, the GBS protocol was adapted for almond and applied to F<sub>1</sub> progeny from a cross between the almond cultivars “Nonpareil” and “Lauranne” to discover SNPs and construct linkage maps. KASP assays were then developed for a subset of the SNPs and were applied to the original mapping population and to additional materials. Quantitative trait loci (QTL) for shell hardness were mapped for Nonpareil and for Lauranne.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials and DNA samples</h3><p id="p-8">The almond clones used in this research were Nonpareil, Lauranne, “Chellaston,” “Constantí,” “Ferraduel,” “Glorieta,” “Johnston,” “Mandaline,” “Marta,” R1065, “Somerton,” “Tarraco,” “Vairo,” “White,” and 12-350 (Supplemental Material, Table S1 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>). In addition, 320 F<sub>1</sub> progeny were used from crosses involving Nonpareil: 320 from Nonpareil × Lauranne (N × L), 349 from Nonpareil × Constantí (N × C), 207 from Nonpareil × Tarraco (N × T) and 198 Nonpareil × Vairo (N × V). Nonpareil is of particular interest because it is a major cultivar in both California and Australia.</p><p id="p-9">Of the 320 N × L progeny, 89 had been previously used for genetic mapping by <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Tavassolian <em>et al.</em> (2010)</a>. For these clones and for Lauranne, the DNA samples used were residual samples from the earlier mapping work. These samples, which had been extracted from young leaves using the Lamboy and Alpha DNA extraction method (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lamboy 1998</a>), were checked for DNA quality by electrophoresis on 1% agarose gels, quantified using PicoGreen intercalating dye (Invitrogen, Carlsbad, CA), and normalized to a working concentration of 20 ng/µl. For the other 231 N × L progeny, and for all of the other almond clones mentioned above, DNA was extracted from young leaves using an Oktopure DNA extraction protocol that had been optimized for almond (LGC Limited, Teddington).</p><p id="p-10">In addition, some use was made of rootstock materials that are available in Australia: “Adafuel,” “Atlas,” “Bright’s Hybrid 1,” “Cornerstone,” “Felinem,” “Garnem,” “GF 557,” “Hansen 536,” “Krymsk 86,” “Monegro,” “Nemaguard,” “Nickels,” “Penta,” “Tetra,” and “Viking” (Table S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>). For these materials, DNA was extracted from young leaves using the method of <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Thomas and Scott (1993)</a> followed by sodium chloride/ethanol precipitation.</p></div><div id="sec-3" class="subsection"><h3>Library construction and sequencing</h3><p id="p-11">To select a restriction enzyme that might be suitable for digestion of the almond genome, <em>in silico</em> restriction of the peach whole genome sequence assembly v1.0 (<a href="http://www.rosaceae.org">www.rosaceae.org</a>) was conducted using Biopython (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cock <em>et al.</em> 2009</a>) for each of three methylation-sensitive restriction enzymes: <em>Ape</em>KI, <em>Pst</em>I, and <em>Hpa</em>II. The enzyme <em>Ape</em>KI, which has been used in GBS for other plants (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lu <em>et al.</em> 2013</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bielenberg <em>et al.</em> 2015</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Guajardo <em>et al.</em> 2015</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kujur <em>et al.</em> 2015</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Gürcan <em>et al.</em> 2016</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Salazar <em>et al.</em> 2017</a>), was selected. Of the three enzymes, it was predicted to yield the highest number of fragments within the size range that is considered suitable for GBS (between 150 and 500 bp) (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>). Further, it had been reported to generate uniform libraries with the degree of complexity reduction that is required for sequencing (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">He <em>et al.</em> 2014</a>).</p><p id="p-12">Barcode, primer, and adapter sequences for <em>Ape</em>KI (Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>) were obtained from <a href="http://www.maizegenetics.net/genotyping-by-sequencing-gbs">http://www.maizegenetics.net/genotyping-by-sequencing-gbs</a>. Barcodes, adapters, and primers were synthesized by Sigma Aldrich (Castle Hill, Australia). The complementary top and bottom strands of each barcode and adapter were diluted to 10 µM with 10× adapter buffer and annealed using the following PCR conditions: 95° for 1 min, followed by ramping down to 30° by 1° per cycle. The resulting double-stranded barcode and adapters were diluted separately in 1× TE to 0.6 ng/µl, quantified using PicoGreen intercalating dye, and normalized to 0.1 µM with 1× TE. Each barcode solution was mixed with the adapter solution in a 1:1 ratio in one well of a 96-well plate.</p><p id="p-13">To select an appropriate ratio between adapter and DNA concentrations, a titration experiment was carried out. For this, a pooled DNA sample was prepared by mixing equal amounts of DNA from 10 N × L F<sub>1</sub> progeny. Eight 200-ng samples of DNA were drawn from this pooled sample. After addition of 3.2 U of <em>Ape</em>KI in 2 µl 10× NEB buffer (New England Biolabs, Ipswich, MA) and water to bring the final volume to 20 µl, these samples were incubated for 2 hr at 75°. One of eight quantities of adapter (2, 5, 8, 10, 12, 15, 18, or 20 µl of a 0.1 M adapter solution), 10 µl of a solution containing 200 U of T4 DNA ligase (New England Biolabs), and 5 µl of 10× ligation buffer were added. Samples were incubated at 22° for 2 hr and then at 65° for 20 min. Ligation products were purified using a PureLink PCR Purification Kit (Invitrogen) as per the manufacturer’s instructions. Each purified ligation product was resuspended in a final volume of 50 µl. For the final library, 10 µl of each purified ligation product was used in a 25-µl PCR reaction with 2 µl of the 10-µM paired-end primers 5′-AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT-3′ and 5′-CAAGCAGAAGACGGCATACGAGATCGGTCTCGGCATTCCTGCTGAACCGCTCTTCCGATCT-3′ along with 12.5 µl of Taq 2X Master Mix (New England Biolabs). The PCR conditions used were as follows: 30 sec at 95°, 15 cycles of 30 sec at 95°, 20 sec at 65°, 30 sec at 68°, followed by a final extension at 72° for 5 min. Each amplified library was purified as described above and eluted in a final volume of 30 µl. Each library (2 µl) was run on 2% agarose at 90 V for 30 min to evaluate the library and the adapter dimer peaks. An adapter concentration of 4.5 ng in a volume of 15 µl was selected because it provided a satisfactory library with no adapter dimer peak.</p><p id="p-14">Library preparation was carried out using 200 ng (10 µl of 20 ng/µl) of DNA from each of the initial 89 N × L progeny and each of three aliquots of DNA from Nonpareil and Lauranne. The same procedure was carried out for a water sample as a negative control. Initial reactions were carried out in a 96-well plate using a separate well for each sample. After adapter ligation, samples were pooled for purification, PCR amplification, evaluation, and sequencing. The pooled library was sequenced using single-end sequencing (100-bp reads) on one flow-cell lane of an Illumina HiSeq 2000 instrument at the Australian Genome Research Facility (Melbourne, Australia).</p></div><div id="sec-4" class="subsection"><h3>SNP discovery</h3><p id="p-15">The GBS sequence data were analyzed using the Universal Network Enabled Analysis Kit (UNEAK) pipeline in TASSEL 3.0 software (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bradbury <em>et al.</em> 2007</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Lu <em>et al.</em> 2013</a>). This pipeline permits SNP calling based solely on GBS tag sequence data, without requiring a reference genome sequence. The output was filtered to select SNPs with at least 80% coverage across samples, a minimum read depth of 5, and a minimum relative heterozygosity value (ratio of heterozygotes to homozygotes) of 0.01. The level of almond genome coverage obtained was estimated using the Lander–Waterman equation (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lander and Waterman 1988</a>).</p></div><div id="sec-5" class="subsection"><h3>Construction of linkage maps</h3><p id="p-16">For the construction of initial framework linkage maps for Nonpareil and Lauranne, tag pairs with minor allele frequencies (MAF) between 0.2 and 0.3 in the mapping population were selected. This was based on the expectation of a MAF of 0.25 for the most informative SNPs (those that are heterozygous in one parent and homozygous in the other parent). The resulting data set was separated into two parental data sets based on whether the tags were homozygous in Nonpareil and heterozygous in Lauranne, or vice versa. Each of these data sets was further filtered to retain only the SNPs missing no more than 20 data points per marker and with segregation ratios not deviating significantly from 3:1 (α = 0.05). Separate parental linkage maps were constructed for Nonpareil and Lauranne using a double pseudotestcross strategy implemented using the backcross (BC) format in ASMap R package version 1.0-1 (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Taylor and Butler 2017</a>) with the following map construction strategy:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-17">An initial framework linkage map was constructed using data from progeny for which there were no missing data for the selected SNPs. Data for a few SSR, RAPD, and ISSR markers that <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Tavassolian <em>et al.</em> (2010)</a> had reported to be homozygous in one parent and heterozygous in the other parent were included in addition to data for the selected SNPs. Linkage mapping was carried out using the minimum spanning tree map algorithm (MSTmap) (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 2008</a>) as implemented in ASMap to assign markers to linkage groups and to order them within linkage groups. A <em>P</em>-value of 0.0001 was used to declare whether markers belong to the same linkage group. The Kosambi mapping function (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kosambi 1944</a>) was used to calculate genetic distances in cM.</p></li><li id="list-item-2"><p id="p-18">For each linkage group, ASMap was used to generate a heat map (rf/LOD plots) to evaluate pairwise associations between markers. For cases in which markers appeared to have had their alleles assigned to the incorrect parents, genotype designations were reassigned using the “switchAlleles” function of the R/qtl R package version 1.41-6 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Broman <em>et al.</em> 2003</a>). Maps were then reestimated using the mstmap.cross function.</p></li><li id="list-item-3"><p id="p-19">To further improve the quality of the Nonpareil and Lauranne linkage maps, markers were checked for segregation distortion and numbers of double crossover events involving adjacent marker intervals. Markers were removed if their segregation ratio deviated significantly from 1:1 (α = 0.05) and/or if they were associated with high numbers of apparent double crossover events. Maps were then reestimated using the mstmap.cross function.</p></li><li id="list-item-4"><p id="p-20">The orientation of each linkage group of the Nonpareil and Lauranne maps was established by comparing the maps constructed using the SNP data with the published maps of <a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Tavassolian <em>et al.</em> (2010)</a>. This was done using the AlignCross function of ASMap.</p></li></ol><p id="p-21">From the resulting framework maps for Nonpareil and Lauranne, a set of GBS markers from the eight linkage groups was selected for the design of allele-specific assays, with the objective of obtaining markers spaced at ∼10-cM intervals throughout the genome.</p></div><div id="sec-6" class="subsection"><h3>Primer design</h3><p id="p-22">Primer sets were designed for SNPs that had been discovered and mapped based on the GBS data. Some of these were heterozygous in Nonpareil and homozygous in Lauranne, and others were heterozygous in Lauranne and homozygous in Nonpareil. For some of these, it was possible to design primers based solely on the GBS data. For others, the SNPs were too close to one end of the GBS tags. For these, the tags were aligned to Nonpareil genomic contig sequences using the BLAST tool in Geneious software version 9.1.3 (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kearse <em>et al.</em> 2012</a>) to obtain sequences of ∼100 bp with the SNPs located near their midpoints. Each SNP-bearing sequence was used to design a set of three primers (two allele-specific primers and one common primer) using Kraken software (LGC Limited). The primer sets were named using the prefix WriPdK, with Wri referring to the Waite Research Institute, Pd referring to <em>P. dulcis</em>, and K referring to KASP technology.</p></div><div id="sec-7" class="subsection"><h3>Application of KASP assays</h3><p id="p-23">A total of 309 primer sets (146 designed for SNPs that were heterozygous in Nonpareil and 162 designed for SNPs that were heterozygous in Lauranne) were assayed on a panel consisting of DNA samples of Nonpareil, Lauranne, and seven N × L F<sub>1</sub> progeny, with a water sample included as a negative control. Samples of 10 ng of DNA (5 µl of 2 ng/µl) were dried at 55° for 1 hr. Aliquots of a primer mixture (0.028 µl, containing 12 µM of the allele-specific primers and 30 µM of the common primer) and of 1× KASP Master Mix (1.972 µl; LGC Limited) were added to each reaction sample. PCR amplification was conducted using the standard KASP PCR protocol in a Hydrocycler-<sup>16</sup> PCR system (LGC Limited). Fluorescence was detected in a Pherastar Plus plate reader (BMG LABTECH, Ortenberg, Germany). Data were analyzed using Kraken software (LGC Limited). Primer sets that detected polymorphism in the validation panel were selected and assayed on 311 N × L progeny: 80 of the 89 progeny that had been used to prepare the GBS library, plus 231 others. The same primer sets were also assayed on the panel of almond clones. Genotypic calls were compared among clones using Flapjack software version 1.16 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Milne <em>et al.</em> 2010</a>). Selected markers (those that were heterozygous in one parent and homozygous in the other) were assayed on the N × C, N × T, and/or N × V progeny.</p></div><div id="sec-8" class="subsection"><h3>Linkage mapping using KASP markers</h3><p id="p-24">Linkage maps were constructed for each parent using KASP marker data from 80 N × L progeny, 231 N × L progeny, 349 N × C progeny, 207 N × T progeny, and 198 N × V progeny, using the procedures described for the construction of the initial framework linkage map. Maps were drawn using MapChart version 2.3 software (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Voorrips 2002</a>).</p><p id="p-25">Data from 985 progeny from four crosses (N × C, N × L, N × T, and N × V) were used to construct a composite map for Nonpareil. This was achieved using a pseudotestcross mapping strategy with data coded in the BC format, considering only those markers for which Nonpareil was heterozygous. In populations for which the other parent was homozygous at a marker, the codes “ab” and “aa” were assigned to heterozygous and homozygous progeny, respectively. In populations for which the other parent was also heterozygous, all progeny were coded as having missing data.</p><p id="p-26">Recombination fractions between adjacent markers were estimated using the R/qtl package version 1.41-6 (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Broman <em>et al.</em> 2003</a>). Recombination fractions were converted to map distances using the Kosambi mapping function (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kosambi 1944</a>). The resulting composite map was compared with Nonpareil maps that had been constructed using data from individual populations. Markers for which there were substantial inconsistencies among maps were removed and the mapping analysis was repeated. Markers that had been mapped in only one population were assigned positions in the composite map based on their positions relative to flanking markers. The final composite map was drawn using MapChart version 2.3 software (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Voorrips 2002</a>).</p></div><div id="sec-9" class="subsection"><h3>Comparative mapping between almond and peach</h3><p id="p-27">Each unique GBS sequence read obtained for Nonpareil and Lauranne that was at least 64-bp long and had sequence coverage ≥10 was aligned against the peach (<em>P. persica</em>) whole genome sequence assembly v2.0.a1 (<a href="http://www.rosaceae.org">www.rosaceae.org</a>) using the BLAST+ tool version 2.2.27 (<a href="http://www.ncbi.nlm.nih.gov/blast">http://www.ncbi.nlm.nih.gov/blast</a>). Each sequence read was considered to have been anchored to the peach genome if it mapped to a unique site with &gt;90% sequence similarity and an <em>E</em>-value &lt;1<em>e</em><sup>−15</sup>. For sequences that met these criteria and for which marker assays had been developed, the Circlize R package version 0.4.1 (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gu <em>et al.</em> 2014</a>) was used to compare the genetic positions in almond with physical positions in the eight main scaffolds of the peach genome assembly.</p></div><div id="sec-10" class="subsection"><h3>QTL mapping for shell hardness</h3><p id="p-28">Shell hardness was evaluated in 2015 for 180 N × L progeny. For each tree, a random sample of 10 nuts was weighed to obtain in-shell weight. The nuts were then cracked open using a nutcracker, and kernels were weighed. The shell-hardness percentage was calculated as suggested by <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rugini (1986)</a>: (kernel weight/in-shell weight) × 100%. According to this measure, almond nuts may be classified as paper shell (≥55%), soft shell (45–54%), semihard shell (35–45%), hard shell (25–34%), or stone shell (≤24%). QTL for this trait were mapped using the R/qtl package version 1.41-6 (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Broman <em>et al.</em> 2003</a>), with the function Scanone used to test for putative QTL at 1-cM intervals throughout the genome. Significance was declared by comparing LOD values to a threshold determined using 10,000 permutations and a genome-wide significance level of 0.05.</p></div><div id="sec-11" class="subsection"><h3>Polymorphism detection in rootstocks</h3><p id="p-29">Using the KASP assay procedures described above, 253 SNPs (128 heterozygous in Nonpareil and 125 heterozygous in Lauranne) were assayed on duplicate samples of DNA samples extracted from the rootstock accessions.</p></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-30">Information on the parentage and origin of the almond clones and rootstocks used in this research is in Tables S1 and S2 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>. Sequence data for 89 N × L progeny have been deposited in the National Center for Biotechnology Information Short Read Archive: study SRR5722967. Information on fragment size distributions from <em>in silico</em> digestion of the peach genome sequence is in Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>. Barcode and primer sequences used for GBS are in Table S4 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>. Contig sequences for Nonpareil have been deposited in the European Nucleotide Archive under accession number PRJEB23106. Primer sequences for KASP assays are in Table S5 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>. Linkage maps for Nonpareil are in Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>. Linkage maps for Lauranne, Constantí, Tarraco, and Vairo are in Table S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>. Results obtained from the application of KASP markers to almond clones and rootstocks are in Tables S8 and S11 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>, respectively. The best BLAST hits in the peach genome for SNP-bearing tags from Nonpareil and Lauranne are in Tables S9 and S10 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>, respectively. The genotypic and phenotypic data used for QTL analysis are in Tables S12 and S13 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Sequence data</h3><p id="p-31">The GBS library generated 21.6 Gb of sequence data, with a total of 186 million sequence reads (a mean of 2.1 million per sample). Linear regression analysis indicated a strong positive relationship between the number of sequence reads and the number of tags for each individual (<em>R</em><sup>2</sup> = 0.92, <em>P</em> &lt; 0.0001, <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/161/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Number of unique sequence tags vs. number of sequence reads. Relationship between the number of sequence reads obtained and the number of unique 64-bp sequence tags obtained from GBS of 89 N &#xD7; L F1 progeny." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1391756120" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Number of unique sequence tags &lt;em&gt;vs.&lt;/em&gt; number of sequence reads. Relationship between the number of sequence reads obtained and the number of unique 64-bp sequence tags obtained from GBS of 89 N &#xD7; L F&lt;sub&gt;1&lt;/sub&gt; progeny.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/161/F1.medium.gif" width="405" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/8/1/161/F1.medium.gif" width="405" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/161/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/161/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487937" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-32" class="first-child">Number of unique sequence tags <em>vs.</em> number of sequence reads. Relationship between the number of sequence reads obtained and the number of unique 64-bp sequence tags obtained from GBS of 89 N × L F<sub>1</sub> progeny.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Across all samples, a total of 453,648 unique tags was obtained. Of these tags, 308,971 (68%) were anchored to the peach genome, with between 30,594 and 59,923 mapping to each of the eight main scaffolds (Pp1–Pp8) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) and 6088 mapping to other scaffolds. Tags were mapped throughout the entire length of each main scaffold, but with some variation in the marker density. There were a few regions (<em>e.g.</em>, on Pp5 and Pp7) with very high density.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/161/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Anchoring of almond sequence tags to peach genome. Numbers of unique tags anchored to each 500-kbp region of each of the eight main scaffolds (Pp1&#x2013;Pp8) of the peach whole genome sequence assembly v2.01a1. For each scaffold, the total number of unique tags is given in parentheses and the estimated position of the centromere is indicated with an arrow." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1391756120" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Anchoring of almond sequence tags to peach genome. Numbers of unique tags anchored to each 500-kbp region of each of the eight main scaffolds (Pp1&#x2013;Pp8) of the peach whole genome sequence assembly v2.01a1. For each scaffold, the total number of unique tags is given in parentheses and the estimated position of the centromere is indicated with an arrow.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/161/F2.medium.gif" width="434" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/8/1/161/F2.medium.gif" width="434" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/161/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/161/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488009" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-34" class="first-child">Anchoring of almond sequence tags to peach genome. Numbers of unique tags anchored to each 500-kbp region of each of the eight main scaffolds (Pp1–Pp8) of the peach whole genome sequence assembly v2.01a1. For each scaffold, the total number of unique tags is given in parentheses and the estimated position of the centromere is indicated with an arrow.</p><div class="sb-div caption-clear"></div></div></div><p id="p-35">From the unique tags, 11,936 SNP-containing tag pairs were identified. With the application of a series of filters, &gt;300 tag pairs that were considered suitable for mapping were selected for each of Nonpareil and Lauranne.</p></div><div id="sec-15" class="subsection"><h3>Linkage maps for Nonpareil and Lauranne</h3><p id="p-36">An initial framework map constructed for Nonpareil based on complete data for 52 progeny had 327 markers (310 GBS, 9 SSR, 5 ISSR, and 3 RAPD) on eight linkage groups with a total length of 1152 cM (Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>). The initial framework map constructed for Lauranne was based on complete data from 55 progeny. It has eight linkage groups, 295 markers (279 GBS, 5 SSR, 8 ISSR, and 3 RAPD), and is 1371 cM long (Table S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>).</p><p id="p-37">Of the 149 KASP primer sets designed based on sequence tags that exhibited heterozygosity in Nonpareil (<em>e.g.</em>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>), 138 detected polymorphism among the progeny (<em>e.g.</em>, <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Of the 162 primer sets designed based on sequence tags that exhibited heterozygosity in Lauranne (<em>e.g.</em>, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>), 155 detected polymorphism among the progeny (<em>e.g.</em>, <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3D</a>). None of the genotypic ratios observed for these polymorphisms deviated significantly from the expected 1:1 ratio.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/161/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Examples of KASP assay results. Intensities of FAM and HEX fluorescence detected when two KASP assays (WriPdK7 and WriPdK69) were applied to Nonpareil, Lauranne, and N &#xD7; L progeny. The WriPdK7 primers were designed for a SNP that is heterozygous (G:C) in Nonpareil and homozygous (C:C) in Lauranne. The WriPdK69 primers were designed for a SNP that is homozygous (C:C) in Nonpareil and heterozygous (T:C) in Lauranne. (A) WriPdK7 applied to Nonpareil (in duplicate), Lauranne (in duplicate), and seven N &#xD7; L F1 progeny. (B) WriPdK7 applied to 231 N &#xD7; L F1 progeny. (C) WriPdK69 applied to Nonpareil (in duplicate), Lauranne (in duplicate), and seven N &#xD7; L F1 progeny. (D) WriPdK69 applied to 231 N &#xD7; L F1 progeny." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1391756120" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Examples of KASP assay results. Intensities of FAM and HEX fluorescence detected when two KASP assays (WriPdK7 and WriPdK69) were applied to Nonpareil, Lauranne, and N &#xD7; L progeny. The WriPdK7 primers were designed for a SNP that is heterozygous (G:C) in Nonpareil and homozygous (C:C) in Lauranne. The WriPdK69 primers were designed for a SNP that is homozygous (C:C) in Nonpareil and heterozygous (T:C) in Lauranne. (A) WriPdK7 applied to Nonpareil (in duplicate), Lauranne (in duplicate), and seven N &#xD7; L F&lt;sub&gt;1&lt;/sub&gt; progeny. (B) WriPdK7 applied to 231 N &#xD7; L F&lt;sub&gt;1&lt;/sub&gt; progeny. (C) WriPdK69 applied to Nonpareil (in duplicate), Lauranne (in duplicate), and seven N &#xD7; L F&lt;sub&gt;1&lt;/sub&gt; progeny. (D) WriPdK69 applied to 231 N &#xD7; L F&lt;sub&gt;1&lt;/sub&gt; progeny.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/161/F3.medium.gif" width="429" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/8/1/161/F3.medium.gif" width="429" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/161/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/161/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488003" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">Examples of KASP assay results. Intensities of FAM and HEX fluorescence detected when two KASP assays (WriPdK7 and WriPdK69) were applied to Nonpareil, Lauranne, and N × L progeny. The WriPdK7 primers were designed for a SNP that is heterozygous (G:C) in Nonpareil and homozygous (C:C) in Lauranne. The WriPdK69 primers were designed for a SNP that is homozygous (C:C) in Nonpareil and heterozygous (T:C) in Lauranne. (A) WriPdK7 applied to Nonpareil (in duplicate), Lauranne (in duplicate), and seven N × L F<sub>1</sub> progeny. (B) WriPdK7 applied to 231 N × L F<sub>1</sub> progeny. (C) WriPdK69 applied to Nonpareil (in duplicate), Lauranne (in duplicate), and seven N × L F<sub>1</sub> progeny. (D) WriPdK69 applied to 231 N × L F<sub>1</sub> progeny.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">Relative to the initial framework maps that were derived from GBS data, the linkage maps constructed based on KASP marker data for 231 N × L progeny had very similar marker orders, but were much shorter (Tables S6 and S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>). The KASP map for Nonpareil had 138 markers and a total length of 609 cM (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). The KASP map for Lauranne had 155 markers and a total length of 659 cm (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/161/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Nonpareil linkage maps. Linkage maps constructed for Nonpareil using genotypic data from KASP assays applied to (A) 231 N &#xD7; L F1 progeny and (B) 985 F1 progeny from four crosses (N &#xD7; L, N &#xD7; C, N &#xD7; T, and N &#xD7; V)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1391756120" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Nonpareil linkage maps. Linkage maps constructed for Nonpareil using genotypic data from KASP assays applied to (A) 231 N &#xD7; L F&lt;sub&gt;1&lt;/sub&gt; progeny and (B) 985 F&lt;sub&gt;1&lt;/sub&gt; progeny from four crosses (N &#xD7; L, N &#xD7; C, N &#xD7; T, and N &#xD7; V).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/161/F4.medium.gif" width="440" height="404"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/8/1/161/F4.medium.gif" width="440" height="404"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/161/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/161/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488035" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-40" class="first-child">Nonpareil linkage maps. Linkage maps constructed for Nonpareil using genotypic data from KASP assays applied to (A) 231 N × L F<sub>1</sub> progeny and (B) 985 F<sub>1</sub> progeny from four crosses (N × L, N × C, N × T, and N × V).</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/161/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Lauranne linkage map. A linkage map constructed for Lauranne using genotypic data from KASP assays applied to 231 N &#xD7; L F1 progeny." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1391756120" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Lauranne linkage map. A linkage map constructed for Lauranne using genotypic data from KASP assays applied to 231 N &#xD7; L F&lt;sub&gt;1&lt;/sub&gt; progeny.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/161/F5.medium.gif" width="440" height="207"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/8/1/161/F5.medium.gif" width="440" height="207"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/161/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/161/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488017" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-41" class="first-child">Lauranne linkage map. A linkage map constructed for Lauranne using genotypic data from KASP assays applied to 231 N × L F<sub>1</sub> progeny.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Polymorphisms among almond clones</h3><p id="p-42">Of the 261 KASP assays tested, 239 exhibited polymorphism among 14 almond clones other than Nonpareil and Lauranne (Table S9 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>). Of these markers, 111 had been designed based on heterozygosity in Nonpareil, and 128 based on heterozygosity in Lauranne. Among the 14 other almond clones on which these markers were assayed, all except Marta and Somerton could be distinguished from all others by just one marker. A total of 11 KASP assays were selected (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>) that, in combination, could be useful for distinguishing among all of the clones that were examined here.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/161/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotypes of 15 almond clones for 11 KASP markers selected based on their ability to discriminate among these clones. For each marker, the least common genotype is shown in white text on a dark background." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1391756120" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genotypes of 15 almond clones for 11 KASP markers selected based on their ability to discriminate among these clones. For each marker, the least common genotype is shown in white text on a dark background.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/161/F6.medium.gif" width="440" height="185"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/8/1/161/F6.medium.gif" width="440" height="185"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/161/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/161/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487917" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-43" class="first-child">Genotypes of 15 almond clones for 11 KASP markers selected based on their ability to discriminate among these clones. For each marker, the least common genotype is shown in white text on a dark background.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Linkage maps based on N × C, N × T, and N × V</h3><p id="p-44">Of the 138 KASP markers that were developed based on Nonpareil heterozygosity and mapped using N × L progeny, 92, 85, and 103 markers detected polymorphism in N × C, N × T, and N × V, respectively. Of the 155 KASP markers that were derived based on Lauranne heterozygosity and mapped using N × L progeny, 68, 40, and 56 markers detected polymorphism in N × C, N × T, and N × V, respectively. In addition, several markers that had not exhibited polymorphism among N × L progeny were found to be polymorphic in other populations. Linkage maps for Nonpareil that were based on N × C, N × T, and N × V had 90, 82, and 94 markers, respectively, with total lengths of 439, 569, and 553 cM, respectively (Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>), and with marker orders very similar to those obtained with the N × L population. Linkage maps developed for Constantí, Tarraco, and Vairo had 65, 39, and 52 markers, respectively, with total lengths of 382, 295, and 148 cM (Table S7 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>).</p></div><div id="sec-18" class="subsection"><h3>QTL for shell hardness</h3><p id="p-45">For Nonpareil, QTL for shell hardness were detected in two regions, both on linkage group 5 (LG5). For Lauranne, QTL for shell hardness were detected in four regions: one on LG2, two on LG5, and one on LG8 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). In all of these regions, Nonpareil-like genotypes were associated with softer shells (higher mean shell hardness). Of the 180 progeny that were evaluated for shell hardness, just 17 had the Nonpareil-like genotype in all QTL regions. Like Nonpareil, these progeny exhibited the “paper-shell” trait (very high shell-hardness percentage). The genotypes at six markers (WriPdK251 and WPdK50 on LG2; WriPdK129, WriPdK18, and WriPdK264 on LG5; and WriPdK282 on LG8) were sufficient to separate progeny with the paper-shell trait from those with harder shells (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/487911/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/487911/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">QTL detected for shell-hardness percentage in Nonpareil and Lauranne based on evaluation of nuts harvested in 2015 from 180 N × L F<sub>1</sub> progeny</span><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/161/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marker-based discrimination of progeny with the paper-shell trait. Shell-hardness percentages and means for two sets of N &#xD7; L F1 progeny, one selected to have the Nonpareil genotypic combination across six markers (WriPdK251 and WPdK50 on LG2; WriPdK129, WriPdK18, and WriPdK264 on LG5; WriPdK282 on LG8) are associated with shell hardness and the other consisting of progeny with other genotypic combinations at those markers." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1391756120" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Marker-based discrimination of progeny with the paper-shell trait. Shell-hardness percentages and means for two sets of N &#xD7; L F&lt;sub&gt;1&lt;/sub&gt; progeny, one selected to have the Nonpareil genotypic combination across six markers (WriPdK251 and WPdK50 on LG2; WriPdK129, WriPdK18, and WriPdK264 on LG5; WriPdK282 on LG8) are associated with shell hardness and the other consisting of progeny with other genotypic combinations at those markers.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/161/F7.medium.gif" width="401" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/8/1/161/F7.medium.gif" width="401" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/161/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/161/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487943" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-47" class="first-child">Marker-based discrimination of progeny with the paper-shell trait. Shell-hardness percentages and means for two sets of N × L F<sub>1</sub> progeny, one selected to have the Nonpareil genotypic combination across six markers (WriPdK251 and WPdK50 on LG2; WriPdK129, WriPdK18, and WriPdK264 on LG5; WriPdK282 on LG8) are associated with shell hardness and the other consisting of progeny with other genotypic combinations at those markers.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Composite linkage map for Nonpareil</h3><p id="p-48">The composite linkage map constructed based on data from the progeny of four Nonpareil crosses had 129 KASP markers with a total length of 741 cM (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a> and Table S6 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>). Some markers that had collocated in Nonpareil genetic maps constructed for individual Nonpareil populations (N× L, N × C, N × T, and/or N × V) were separated in the composite map.</p></div><div id="sec-20" class="subsection"><h3>Comparison of almond genetic maps with the peach genome</h3><p id="p-49">Comparison of marker positions on Nonpareil and Lauranne parental maps with positions on peach genome scaffolds confirmed the expected high synteny and collinearity between the almond and peach genomes (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> and Tables S9 and S10 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>). Almost all markers anchored to the expected peach scaffolds. For the Nonpareil map, the exceptions are a few markers that genetically mapped on LG1, LG4, LG6, and LG8 but anchored to peach scaffolds Pp5, Pp1, Pp1, and Pp4, respectively. For the Lauranne map, there were markers that genetically mapped on LG2, LG3, and LG6 but anchored to peach scaffolds Pp6, Pp6, and Pp4, respectively. There are also a few discrepancies in marker order between the almond genetic maps and the peach scaffolds (<em>e.g.</em>, at each end of Nonpareil LG4 and peach scaffold Pp4). A few areas of the peach genome are not well represented on one or both almond linkage maps. For example, only two markers from the Nonpareil LG7 map and no markers from the Lauranne LG7 map anchored between 0 and 7 Mbp on peach Pp7 scaffold. There are also some regions in which markers that are closely linked in almond (<em>e.g.</em>, at 27 cM on Nonpareil LG2 and at 20 cM on Lauranne LG8) anchored to physically distant positions on peach scaffolds.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/161/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparisons of almond genetic maps with the peach genome sequence. (A) Nonpareil linkage groups 1&#x2013;8 compared with peach scaffolds Pp1&#x2013;Pp8. (B) Lauranne linkage groups 1&#x2013;8 compared with peach scaffolds Pp1&#x2013;Pp8. On the almond linkage groups, genetic distances are given in cM. On the peach scaffolds, physical distances are given in Mbp. Links between linkage groups and scaffolds indicate the positions at which markers genetically mapped in almond anchor to the genomic sequence of peach." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1391756120" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparisons of almond genetic maps with the peach genome sequence. (A) Nonpareil linkage groups 1&#x2013;8 compared with peach scaffolds Pp1&#x2013;Pp8. (B) Lauranne linkage groups 1&#x2013;8 compared with peach scaffolds Pp1&#x2013;Pp8. On the almond linkage groups, genetic distances are given in cM. On the peach scaffolds, physical distances are given in Mbp. Links between linkage groups and scaffolds indicate the positions at which markers genetically mapped in almond anchor to the genomic sequence of peach.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/161/F8.medium.gif" width="232" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/8/1/161/F8.medium.gif" width="232" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/161/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/161/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/488036" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-50" class="first-child">Comparisons of almond genetic maps with the peach genome sequence. (A) Nonpareil linkage groups 1–8 compared with peach scaffolds Pp1–Pp8. (B) Lauranne linkage groups 1–8 compared with peach scaffolds Pp1–Pp8. On the almond linkage groups, genetic distances are given in cM. On the peach scaffolds, physical distances are given in Mbp. Links between linkage groups and scaffolds indicate the positions at which markers genetically mapped in almond anchor to the genomic sequence of peach.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-21" class="subsection"><h3>Polymorphisms among rootstocks</h3><p id="p-51">Of the 220 KASP assays tested on rootstock materials, 169 were polymorphic and 66 were monomorphic (Table S11 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>). Of the 169 polymorphic assays, 93 had been designed based on Nonpareil heterozygosity, and 76 based on Lauranne heterozygosity. In most cases, just one marker was sufficient to distinguish a particular rootstock from all others. Penta and Tetra, both of which are derived from European plum, were very similar but there were four markers that distinguished between them. A total of 10 KASP assays were selected (<a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>) that, in combination, could be useful for distinguishing among all of the rootstock materials that were examined here.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/8/1/161/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotypes of 15 rootstocks for 10 KASP markers selected based on their ability to discriminate among these rootstocks. For each marker, the least common genotype is shown in white text on a dark background." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1391756120" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genotypes of 15 rootstocks for 10 KASP markers selected based on their ability to discriminate among these rootstocks. For each marker, the least common genotype is shown in white text on a dark background.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/8/1/161/F9.medium.gif" width="440" height="137"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/8/1/161/F9.medium.gif" width="440" height="137"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/8/1/161/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/8/1/161/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/487961" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-52" class="first-child">Genotypes of 15 rootstocks for 10 KASP markers selected based on their ability to discriminate among these rootstocks. For each marker, the least common genotype is shown in white text on a dark background.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-22"><h2 class="">Discussion</h2><p id="p-53">In this research, implementation of a GBS protocol enabled discovery of thousands of SNP-bearing GBS tags, providing an easy method to discover and assay SNPs without any prior sequence information. The restriction enzyme used here, <em>Ape</em>KI, is a type-II endonuclease that recognizes a degenerate 5-bp sequence (GCWGC, where W is A or T). It is useful for the reduction of sequence complexity, because it has relatively few recognition sites in the major classes of plant retrotransposons and will not cut if the 3′ base of the recognition sequence on the bottom strand is 5′ methyl cytosine (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Söllner <em>et al.</em> 2006</a>). It creates a 5′ overhang of 3 bp, providing sites for attachment of adapters to which primers can anneal to provide a uniform library for sequencing (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">He <em>et al.</em> 2014</a>).</p><p id="p-54">The mean number of sequence reads per sample that was obtained here (2.1 million) is similar to what has been reported for other <em>Prunus</em> species: 1.8 million for sweet cherry (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Guajardo <em>et al.</em> 2015</a>), 2.4 million for peach (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Bielenberg <em>et al.</em> 2015</a>), 2.3 million for Japanese plum (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Salazar <em>et al.</em> 2017</a>), and 3.5 million for apricot (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Gürcan <em>et al.</em> 2016</a>). Of the sequences generated, 68% were anchored to unique positions in the peach genome sequence assembly. This is higher than was reported for apricot (43%, <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Salazar <em>et al.</em> 2017</a>), which is not as closely related to peach. Within a total of 224 Mb of the peach genome to which almond tags were anchored, the density of almond–peach SNPs was ∼1 per 22 kb. Similar results have been reported for sweet cherry (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Guajardo <em>et al.</em> 2015</a>). Some variation was observed in the numbers of sequence tags and SNPs mapped to each scaffold and in the anchor positions of tags within scaffolds. Possible reasons for this variation could include: (1) variation in the distribution of <em>Ape</em>KI restriction sites across the almond genome, (2) variation in DNA methylation across the almond genome, and (3) structural differences between the almond and peach genomes. The unusually high numbers of sequence tags obtained in some regions (<em>e.g.</em>, one at 6.5 Mbp on Pp5 and one at 0.5 Mbp on Pp7) may indicate that these regions are more polymorphic or more repetitive in almond than in peach. Consistent with this, one of these regions (at 6.5 Mbp on Pp5) corresponds with a region of LG5 in which many polymorphisms were genetically mapped for Nonpareil (albeit not for Lauranne). However, another one (at 0.5 Mbp on Pp7) is on a chromosome arm for which only two polymorphisms were mapped in Nonpareil and none were mapped for Lauranne. Lack of polymorphism in this region may not be limited to these materials, nor even to almond, as similar observations have been reported for this region for the sweet cherry cultivars Riverdale and Rainer (<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Guajardo <em>et al.</em> 2015</a>). The consistent lack of polymorphism in certain genomic regions could reflect fixation of favorable alleles due to selection.</p><p id="p-55">While the total number of high-quality SNPs obtained for almond (11,936) was sufficient for genetic mapping and higher than was obtained for sweet cherry (8476; <a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Guajardo <em>et al.</em> 2015</a>) it is lower than what was obtained for apricot (18,322; <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Gürcan <em>et al.</em> 2016</a>) or plum (42,909; <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Salazar <em>et al.</em> 2017</a>). One approach to increase the number of SNPs discovered would be to use less-stringent filters in the sequence analysis. The TASSEL GBS 3.0 SNP-calling pipeline, which was developed mainly for highly homozygous materials, is considered to be sensitive to low sequence depth in highly heterozygous materials (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Hyma <em>et al.</em> 2015</a>). Therefore, a stringent read depth cutoff value of five was applied per marker call. With a value of three, a larger number of GBS tags (&gt;600) could have been selected for mapping, but maps constructed on this basis (data not shown) had very long linkage groups (&gt;300 cM); some of the additional SNPs may have been spurious.</p><p id="p-56">Another approach to increase the number of SNPs discovered could be to increase sequence depth. In this analysis, there was a strong positive relationship (<em>R</em><sup>2</sup> = 0.92) observed between total read number and the total number of unique tags, indicating that additional unique tags and SNPs might have been discovered by increasing sequencing depth.</p><p id="p-57">A third approach to increase the number of SNPs discovered could be to use an enzyme or combination of enzymes that would provide a larger number of digested fragments and increasing the depth of sequencing. To investigate this, we extended the <em>in silico</em> analysis of the peach genome sequence to include consideration of two-enzyme combinations. The results (Table S3 in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1/FileS1.xlsx">File S1</a>) indicated that the combination of <em>Ape</em>KI and <em>Hpa</em>II might provide a substantially higher number of fragments of suitable length than <em>Ape</em>KI alone.</p><p id="p-58">While GBS can generate large numbers of polymorphic markers, it can suffer from incorrect assignment of parental phase, underestimation of heterozygotes, and high proportions of missing data (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Lu <em>et al.</em> 2013</a>). Here, technical replicates of the parents were included in the genomic library and very stringent filters were applied to select subsets of markers and progeny for initial mapping. During map construction, diagnostic tests were conducted to detect and correct phasing errors. These approaches contributed to a very high success rate in KASP assay design. Only one incorrectly phased marker was detected. That marker (GBS tag pair TP37439), which had originally been assigned to the Nonpareil map, was reassigned to the Lauranne map based on results obtained with the WriPdK92 primer set. Three markers (GBS tag pairs TP15642, TP16449, and TP18643) that were originally assigned to the Lauranne map were determined to be heterozygous in both parents and were not used for map construction. Three other markers (GBS tag pairs TP11609, TP12109, and TP25403) that had originally been scored as heterozygous in one parent were determined to be homozygous in both parents and were not used for map construction. With the KASP markers, it was possible to obtain complete and accurate data for a larger number of progeny than had been used for the initial GBS map. Therefore it is not surprising that there are some differences in marker order between the initial and KASP maps; the KASP maps should be considered as more reliable.</p><p id="p-59">The numbers of GBS tag pairs used to construct the initial Nonpareil and Lauranne linkage maps (310 and 282, respectively) were similar to numbers that have been used for Japanese plum (232 for one parent and 324 for the other; <a id="xref-ref-37-6" class="xref-bibr" href="#ref-37">Salazar <em>et al.</em> 2017</a>) and for sweet cherry (443 for one parent and 474 for the other; <a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">Guajardo <em>et al.</em> 2015</a>). The initial genetic maps constructed using GBS data are about twice as long as the Nonpareil and Lauranne maps published by <a id="xref-ref-43-6" class="xref-bibr" href="#ref-43">Tavassolian <em>et al.</em> (2010)</a>, but the final maps constructed using data from KASP assays are similar in length to the previously published maps. This “shrinkage” was due to correction of genotypes that had been erroneously called in the GBS analysis. In most cases, corrections were from homozygous to heterozygous, indicating that although two alleles were present, only one of them had been sequenced in sufficient depth. Of a total of 12,720 heterozygous calls in the final KASP data set, 1526 (12%) had been miscalled as homozygous in the GBS analysis. This type of error was evenly distributed among markers. These observations are similar to what has been reported for GBS analysis in switchgrass (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Lu <em>et al.</em> 2013</a>).</p><p id="p-60">The success rate in converting SNP-bearing tag pairs to useful fluorescence-based marker assays was very high. Of the 309 SNP-bearing GBS sequences that were selected for assay design, 293 (95%) were successfully converted to KASP assays and mapped (138 for Nonpareil and 155 for Lauranne). However, not all of these could have been designed solely based on the GBS data. In many tags, the SNP position was too close to one end of the tag for primer design. These tag sequences had to be aligned against preexisting contig sequences to obtain flanking sequences.</p><p id="p-61">Until now, linkage mapping in almond has relied on data from the progeny of individual biparental crosses. Here, progeny from four Nonpareil crosses were used, providing four linkage maps for Nonpareil. There should be no differences in the true biological positions of the SNPs on these four maps, as each map is based on estimates of recombination frequencies for the same parent, Nonpareil. While there was very good agreement among the maps, there were also some differences, presumably due to sampling error. Given that the SNPs mapped in all four populations had been preselected because they were informative for N × L, the map derived from that population has most markers and the best genome coverage. Among the four individual Nonpareil maps, the one from N × L should be considered as the most reliable. Nevertheless, the quality of any genetic map is limited by sample size. Here, the availability of additional Nonpareil populations provided an opportunity to improve the map accuracy and resolution. The Nonpareil composite linkage map constructed here is the first almond linkage map constructed based on genotypic data from multiple cross combinations. The use of four crosses with a common parent (Nonpareil) provided an opportunity to exploit a larger total population size to resolve the order of some closely linked markers. Given that Nonpareil is the predominant almond cultivar in both the United States and Australia, this linkage map should be a particularly useful new resource for almond research, including QTL mapping and marker-assisted breeding.</p><p id="p-62">To demonstrate the utility of the linkage maps for QTL mapping and to provide examples of markers that could be used for selection, QTL results are presented here for shell hardness. For that trait, QTL were detected on three linkage groups: LG2, LG5, and LG8. QTL for this trait had previously been mapped on LG2 (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Sánchez-Pérez <em>et al.</em> 2007</a>) and LG8 (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Arús <em>et al.</em> 1998</a>), but not on LG5. At all of these QTL, Nonpareil-like marker genotypes were associated with softer shells. Depending on whether the paper-shell characteristic of Nonpareil is considered favorable or unfavorable (too soft), selection could be imposed for or against Nonpareil-like genotypes at six markers (two on LG2, three on LG5, and one on LG8). In the N × L population, this would have either fixed or eliminated the paper-shell trait.</p><p id="p-63">Given that this map was constructed using sequence-based markers, it was possible to anchor it to the peach genome sequence assembly. It will be possible to anchor it to almond genome sequence assemblies as they become available and to connect it with other sequence-based linkage maps as they are developed. Thus, this composite map could provide a platform for unification of genetic and genomic resources for the almond research community.</p><p id="p-64">This is the first report of genome-wide anchoring of almond genetic maps to the peach whole genome sequence assembly. The results confirmed the expected high similarity between the almond and peach genomes, with only a few of the mapped markers anchoring to unexpected positions. In most parts of the genome, marker positions on Nonpareil and Lauranne genetic maps were linearly related with the physical positions to which they were anchored in the peach genome. There were some regions of the peach genome for which few almond polymorphisms were discovered. Based the data generated here, it is not possible to distinguish whether these are simply regions in which Nonpareil and Lauranne are both highly homozygous, or whether these regions are structurally different between peach and almond.</p><p id="p-65">The polymorphisms detected here among almond clones and rootstocks provide information about the transferability of SNPs discovered in one population to other materials. Of the markers designed based on heterozygosity in Nonpareil or Lauranne, 70% were useful for polymorphism detection among almond rootstocks. Although SNPs are generally used for polymorphism detection within species, many of the SNP markers developed here based on almond polymorphism detected polymorphisms in material that originated from peach, apricot, plum, cherry, and complex backgrounds. The markers developed here could be broadly useful for detecting genetic differences among accessions of <em>P. dulcis</em>, related species, and interspecific crosses. Applications of these markers could include assessment of cultivar verification, genetic diversity assessment, genetic mapping, and marker-assisted selection. From among the markers that were assayed here, 11 were selected based on their ability to differentiate among 15 almond clones and 10 were selected based on their ability to differentiate among 15 rootstocks.</p><p id="p-66">This is the first report on the use of GBS in almond to discover SNPs and to generate linkage maps. The processes that were used here to select a restriction enzyme, conduct GBS data analysis, and design KASP markers could be applied more broadly for almond and other species.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-67">The authors gratefully acknowledge financial support from the following: Hort Innovation (the grower-owned, not-for-profit research and development corporation for Australian horticulture), which supported this research using the almond industry and development levy and contributions from the Australian Government; the Commonwealth Scientific and Industrial Research Organization; the South Australian River Murray Sustainability Program (IRSPR2-004) administered by Primary Industries and Regions South Australia; and the Spanish Ministry of Economy and Competitiveness [project AGL2015-68329-R, Severo Ochoa Program for Centres of Excellence in R&amp;D 201-2019 (SEV-2015-0533), CERCA Program of the Generalitat de Catalunya]. S.N.G. was the recipient of an Australian Postgraduate Award administered by the University of Adelaide.</p><p id="p-68">Author contributions: S.N.G. and T.J.M. prepared the GBS library and analyzed the GBS data. S.N.G. designed and assayed KASP markers, constructed linkage maps, and conducted comparative analysis between almond and peach. M.G.W. provided almond materials and previous mapping data. P.A. provided early access to contig sequences for Nonpareil. A.R.W. provided rootstock DNA. S.N.G and D.E.M. wrote the manuscript. All authors contributed to the design of the study, interpretation of the results, and revision of the manuscript. The authors declare no competing financial interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-69">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.117.300376/-/DC1</a>.</p></li><li class="fn" id="fn-3"><p id="p-70"><em>Communicating editor: I. Parkin</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2017-07-20"><span class="received-label">Received </span>July 20, 2017.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2017-11-04"><span class="accepted-label">Accepted </span>November 4, 2017.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2018 Goonetilleke <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aranzana</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pineda</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cosson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dirlewanger</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ascasibar</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A set of simple-sequence repeat (SSR) markers covering the <em>Prunus</em> genome.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">819</span>–<span class="cit-lpage">825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAranzana%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D106%26rft.issue%253D5%26rft.spage%253D819%26rft.epage%253D825%26rft.atitle%253DA%2Bset%2Bof%2Bsimple-sequence%2Brepeat%2B%2528SSR%2529%2Bmarkers%2Bcovering%2Bthe%2BPrunus%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F12647055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12647055&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182200900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arús</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messeguer</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viruell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tobutt</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dirlewanger</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The European <em>Prunus</em> mapping project progress in the almond linkage map.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">100</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D77%26rft.spage%253D97%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-confproc" id="cit-8.1.161.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arús</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballester</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jáuregui</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joobeur</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Truco</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> The European Prunus mapping project: update on marker development in almond, pp. <span class="cit-fpage">331</span>–<span class="cit-lpage">338</span> in Eucarpia Symposium on Fruit Breeding and Genetics, edited by <span class="cit-ed"><span class="cit-name-surname">Tobutt</span>  <span class="cit-name-given-names">K. R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Alston</span>  <span class="cit-name-given-names">F. H.</span></span>, Oxford.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Babiker</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rouse</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wanyera</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Molecular mapping of stem rust resistance loci effective against the Ug99 race group of the stem rust pathogen and validation of a single nucleotide polymorphism linked to stem rust resistance gene <em>Sr28</em>.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">208</span>–<span class="cit-lpage">215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.volume%253D107%26rft.spage%253D208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.5" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bielenberg</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rauh</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gasic</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abbott</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genotyping by sequencing for SNP-based linkage map construction and QTL analysis of chilling requirement and bloom date in peach [<em>Prunus persica</em> (L.) Batsch].</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0139406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0139406%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.7" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17586829&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250673800021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.8" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ś.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12724300&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182634800014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.9" data-doi="10.1093/bioinformatics/btp163"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cock</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapman</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Biopython: freely available Python tools for computational molecular biology and bioinformatics.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1422</span>–<span class="cit-lpage">1423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp163%26rft_id%253Dinfo%253Apmid%252F19304878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp163&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19304878&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266109500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.10" data-doi="10.1073/pnas.0307937101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dirlewanger</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graziano</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joobeur</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garriga-Caldere</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cosson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Comparative mapping and marker-assisted selection in Rosaceae fruit crops.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">9891</span>–<span class="cit-lpage">9896</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0307937101%26rft_id%253Dinfo%253Apmid%252F15159547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAxLzI2Lzk4OTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy84LzEvMTYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Donoso</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Picañol</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serra</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howad</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alegre</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Exploring almond genetic variability useful for peach improvement: mapping major genes in two interspecific peach × almond populations.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D36%26rft.spage%253D16%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.12" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernández i Martí</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Font i Forcada</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Socias i Company</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genetic analysis for physical nut traits in almond.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D9%26rft.spage%253D455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Font i Forcada</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernández i Martí</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Socias i Company</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Mapping quantitative trait loci for kernel composition in almond.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DFont%2BI%2BForcada%26rft.auinit1%253DC.%26rft.volume%253D13%26rft.spage%253D47%26rft.epage%253D47%26rft.atitle%253DMapping%2Bquantitative%2Btrait%2Bloci%2Bfor%2Bkernel%2Bcomposition%2Bin%2Balmond.%26rft_id%253Dinfo%253Apmid%252F22720975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22720975&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Font i Forcada</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oraguzie</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reyes-Chin-Wo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Espiau</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Socias i Company</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification of genetic loci associated with quality traits in almond via association mapping.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0127656</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0127656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.161.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gradziel</span>  <span class="cit-name-given-names">T. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Almond (<em>Prunus dulcis</em>) breeding</span>, pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">31</span> in <span class="cit-source">Breeding Plantation Tree Crops: Temperate Species</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Mohan Jain</span>  <span class="cit-name-given-names">S.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Priyadarshan</span>  <span class="cit-name-given-names">P. M.</span></span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.17" data-doi="10.1093/bioinformatics/btu393"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eils</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schlesner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brors</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Circlize implements and enhances circular visualization in R.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2811</span>–<span class="cit-lpage">2812</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu393%26rft_id%253Dinfo%253Apmid%252F24930139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24930139&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000343082900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guajardo</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solís</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sagredo</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gainza</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Construction of high density sweet cherry (<em>Prunus avium</em> L.) linkage maps using microsatellite markers and SNPs detected by genotyping-by-sequencing (GBS).</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0127750</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0127750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gürcan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Teber</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ercisli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yilmaz</span>  <span class="cit-name-given-names">K. U.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genotyping by sequencing (GBS) in apricots and genetic diversity assessment with GBS-derived single-nucleotide polymorphisms (SNPs).</span> <abbr class="cit-jnl-abbrev">Biochem. Genet.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">854</span>–<span class="cit-lpage">885</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BGenet.%26rft.volume%253D54%26rft.spage%253D854%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laroche</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Z.-X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genotyping-by-sequencing (GBS), an ultimate marker-assisted selection (MAS) tool to accelerate plant breeding.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D5%26rft.spage%253D484%26rft_id%253Dinfo%253Apmid%252F25324846%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=25324846&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hyma</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barba</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Londo</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Heterozygous Mapping Strategy (HetMappS) for highresolution genotyping-by-sequencing markers: a case study in grapevine.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e0134880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0134880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.22" data-doi="10.1007/s001220050988"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joobeur</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Viruel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Vicente</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jauregui</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballester</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Construction of a saturated linkage map for <em>Prunus</em> using an almond x peach F<sub>2</sub> progeny.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">1034</span>–<span class="cit-lpage">1041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D97%26rft.spage%253D1034%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050988&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000077465100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joobeur</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Periam</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vicente</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arús</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Development of a second generation linkage map for almond using RAPD and SSR markers.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">649</span>–<span class="cit-lpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DJoobeur%26rft.auinit1%253DT.%26rft.volume%253D43%26rft.issue%253D4%26rft.spage%253D649%26rft.epage%253D655%26rft.atitle%253DDevelopment%2Bof%2Ba%2Bsecond%2Bgeneration%2Blinkage%2Bmap%2Bfor%2Balmond%2Busing%2BRAPD%2Band%2BSSR%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F10984177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10984177&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.24" data-doi="10.1093/bioinformatics/bts199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kearse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moir</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stones-Havas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Geneious basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1647</span>–<span class="cit-lpage">1649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts199&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22543367&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305419800052&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koepke</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harper</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dicenta</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparative genomics analysis in Prunoideae to identify biologically relevant polymorphisms.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">883</span>–<span class="cit-lpage">893</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D11%26rft.spage%253D883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.26" data-doi="10.1111/j.1469-1809.1943.tb02321.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kosambi</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1944</span> <span class="cit-article-title">The estimation of map distances from recombination values.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D12%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1943.tb02321.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1943.tb02321.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.27" data-doi="10.1038/srep09468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kujur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Upadhyaya</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shree</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bajaj</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ultra-high density intra-specific genetic linkage maps accelerate identification of functionally relevant molecular tags governing important agronomic traits in chickpea.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">9468</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D5%26rft.spage%253D9468%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep09468%26rft_id%253Dinfo%253Apmid%252F25942004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep09468&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25942004&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamboy</span>  <span class="cit-name-given-names">W. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Using simple sequence repeats (SSRs) for DNA fingerprinting germplasm accessions of grape (Vitis L.) species.</span> <abbr class="cit-jnl-abbrev">J. Am. Soc. Hortic. Sci.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">182</span>–<span class="cit-lpage">188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BHortic.%2BSci.%26rft.volume%253D123%26rft.spage%253D182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.29" data-doi="10.1016/0888-7543(88)90007-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waterman</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Genomic mapping by fingerprinting random clones: a mathematical analysis.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DLander%26rft.auinit1%253DE.%2BS.%26rft.volume%253D2%26rft.issue%253D3%26rft.spage%253D231%26rft.epage%253D239%26rft.atitle%253DGenomic%2Bmapping%2Bby%2Bfingerprinting%2Brandom%2Bclones%253A%2Ba%2Bmathematical%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0888-7543%252888%252990007-9%26rft_id%253Dinfo%253Apmid%252F3294162%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0888-7543(88)90007-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3294162&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.30" data-doi="10.1371/journal.pgen.1003215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Switchgrass genomic diversity, ploidy, and evolution: novel insights from a network-based SNP discovery protocol.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003215%26rft_id%253Dinfo%253Apmid%252F23349638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23349638&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.31" data-doi="10.1101/gr.5681207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">240</span>–<span class="cit-lpage">248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5681207%26rft_id%253Dinfo%253Apmid%252F17189378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzIvMjQwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzE2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.32" data-doi="10.1093/bioinformatics/btq580"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milne</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shaw</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephen</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardle</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Flapjack—graphical genotype visualization.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">3133</span>–<span class="cit-lpage">3134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq580%26rft_id%253Dinfo%253Apmid%252F20956241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq580&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20956241&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284947700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.33" data-doi="10.1371/journal.pone.0037135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Double digest RADseq: an inexpensive method for <em>de novo</em> SNP discovery and genotyping in model and non-model species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeterson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37135%26rft.epage%253De37135%26rft.atitle%253DDouble%2Bdigest%2BRADseq%253A%2Ban%2Binexpensive%2Bmethod%2Bfor%2Bde%2Bnovo%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bmodel%2Band%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037135%26rft_id%253Dinfo%253Apmid%252F22675423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22675423&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.34" data-doi="10.1371/journal.pone.0032253"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e32253</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPoland%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De32253%26rft.epage%253De32253%26rft.atitle%253DDevelopment%2Bof%2Bhigh-density%2Bgenetic%2Bmaps%2Bfor%2Bbarley%2Band%2Bwheat%2Busing%2Ba%2Bnovel%2Btwo-enzyme%2Bgenotyping-by-sequencing%2Bapproach.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0032253%26rft_id%253Dinfo%253Apmid%252F22389690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0032253&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22389690&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rasheed</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Development and validation of KASP assays for genes underpinning key economic traits in bread wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">1843</span>–<span class="cit-lpage">1860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D129%26rft.spage%253D1843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-8.1.161.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rugini</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Almond</span>, pp. <span class="cit-fpage">507</span>–<span class="cit-lpage">611</span> in <span class="cit-source">Handbook of Plant Cell Culture</span>, Vol. 4, edited by <span class="cit-ed"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">A.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">D.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Phillip</span>  <span class="cit-name-given-names">A.</span></span>. <span class="cit-publ-name">Macmillan</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salazar</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pacheco</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinya</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zapata</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genotyping by sequencing for SNP-based linkage analysis and identification of QTLs linked to fruit quality traits in Japanese plum (<em>Prunus salicina</em> Lindl.).</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">476</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D8%26rft.spage%253D476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.38" data-doi="10.1111/j.1439-0523.2007.01329.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sánchez-Pérez</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howad</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dicenta</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arús</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martínez-Gómez</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mapping major genes and quantitative trait loci controlling agronomic traits in almond.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">310</span>–<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D126%26rft.spage%253D310%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0523.2007.01329.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0523.2007.01329.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000246367400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scorza</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Progress in tree fruit improvement through molecular genetics.</span> <abbr class="cit-jnl-abbrev">HortScience</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">855</span>–<span class="cit-lpage">858</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHortScience%26rft.stitle%253Dhortsci%26rft.aulast%253DScorza%26rft.auinit1%253DR.%26rft.volume%253D36%26rft.issue%253D5%26rft.spage%253D855%26rft.epage%253D858%26rft.atitle%253DProgress%2Bin%2BTree%2BFruit%2BImprovement%2BThrough%2BMolecular%2BGenetics%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo3OiJob3J0c2NpIjtzOjU6InJlc2lkIjtzOjg6IjM2LzUvODU1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvOC8xLzE2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.40" data-doi="10.1007/s00792-006-0019-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Söllner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berkner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipps</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Characterisation of the novel restriction endonuclease <em>SuiI</em> from <em>Sulfolobus islandicus</em>.</span> <abbr class="cit-jnl-abbrev">Extremophiles</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">629</span>–<span class="cit-lpage">634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExtremophiles%2B%2528Print%2529%26rft.stitle%253DExtremophiles%2B%2528Print%2529%26rft.aulast%253DSollner%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D629%26rft.epage%253D634%26rft.atitle%253DCharacterisation%2Bof%2Bthe%2Bnovel%2Brestriction%2Bendonuclease%2BSuiI%2Bfrom%2BSulfolobus%2Bislandicus.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00792-006-0019-0%26rft_id%253Dinfo%253Apmid%252F16896525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00792-006-0019-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16896525&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorkheh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekhordi</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ercisli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hegedus</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halász</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Comparison of traditional and new generation DNA markers declares high diversity and differentiated population structure of wild almond species.</span> <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">5966</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D7%26rft.spage%253D5966%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">C.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Development and validation of KASP markers for the greenbug resistance gene <em>Gb7</em> and the Hessian fly resistance gene <em>H32</em> in wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">1867</span>–<span class="cit-lpage">1884</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D1867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tavassolian</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabiei</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gregory</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mnejja</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wirthensohn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Construction of an almond linkage map in an Australian population Nonpareil × Lauranne.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DTavassolian%26rft.auinit1%253DI.%26rft.volume%253D11%26rft.spage%253D551%26rft.epage%253D551%26rft.atitle%253DConstruction%2Bof%2Ban%2Balmond%2Blinkage%2Bmap%2Bin%2Ban%2BAustralian%2Bpopulation%2BNonpareil%2Bx%2BLauranne.%26rft_id%253Dinfo%253Apmid%252F20932335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20932335&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">R package ASMap: efficient genetic linkage map construction and diagnosis.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D79%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">N. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Microsatellite repeats in grapevine reveal DNA polymorphisms when analysed as sequence-tagged sites (STSs).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">985</span>–<span class="cit-lpage">990</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D86%26rft.spage%253D985%26rft_id%253Dinfo%253Apmid%252F24194007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24194007&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993MA72300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-8.1.161.46"><div class="cit-metadata"><cite><span class="cit-comment">United States Department of Agriculture, 2017  Production, Supply and Distribution. Available at: <a href="https://apps.fas.usda.gov/psdonline/app/index.html#/app/home">https://apps.fas.usda.gov/psdonline/app/index.html#/app/home</a>. Accessed: December 2, 2017.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Viruel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messeguer</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Vicente</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Mas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puigdomènech</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">A molecular marker map with RFLPs and isozymes for almond.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">964</span>–<span class="cit-lpage">971</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D91%26rft.spage%253D964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1995TH31900021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.48" data-doi="10.1093/jhered/93.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MapChart: software for the graphical presentation of linkage maps and QTLs.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D78%26rft.atitle%253DMapChart%253A%2BSoftware%2Bfor%2Bthe%2BGraphical%2BPresentation%2Bof%2BLinkage%2BMaps%2Band%2BQTLs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.77%26rft_id%253Dinfo%253Apmid%252F12011185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/93.1.77&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12011185&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175627300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.49" data-doi="10.1007/s00438-009-0464-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavassolian</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabiei</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wirthensohn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mapping SNP-anchored genes using high-resolution melting analysis in almond.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DWu%26rft.auinit1%253DS.%2BB.%26rft.volume%253D282%26rft.issue%253D3%26rft.spage%253D273%26rft.epage%253D281%26rft.atitle%253DMapping%2BSNP-anchored%2Bgenes%2Busing%2Bhigh-resolution%2Bmelting%2Banalysis%2Bin%2Balmond.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-009-0464-4%26rft_id%253Dinfo%253Apmid%252F19526371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-009-0464-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19526371&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franks</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wirthensohn</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Discrimination of SNP genotypes associated with complex haplotypes by high resolution melting analysis in almond: implications for improved marker efficiencies.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">351</span>–<span class="cit-lpage">357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D25%26rft.spage%253D351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-8.1.161.51" data-doi="10.1371/journal.pgen.1000212"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient and accurate construction of genetic linkage maps from the minimum spanning tree of a graph.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWu%26rft.auinit1%253DY.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De1000212%26rft.epage%253De1000212%26rft.atitle%253DEfficient%2Band%2Baccurate%2Bconstruction%2Bof%2Bgenetic%2Blinkage%2Bmaps%2Bfrom%2Bthe%2Bminimum%2Bspanning%2Btree%2Bof%2Ba%2Bgraph.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000212%26rft_id%253Dinfo%253Apmid%252F18846212%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000212&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18846212&amp;link_type=MED&amp;atom=%2Fggg%2F8%2F1%2F161.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/8/1/161.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/8/1/149" title="Severe Fertility Effects of sheepish Sperm Caused by Failure To Enter Female Sperm Storage Organs in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/8/1/173" title="Rapid and Efficient CRISPR/Cas9-Based Mating-Type Switching of Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/8/1"><span>Volume 8 Issue 1, January 2018</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/8/1.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/8/1.cover-source.jpg?itok=vyoqxep2" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 8 (1)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/8/1/161" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-oxaSIJKIKc-INVDITSgPriBhNJznDeYfdeMBsZk9lMo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/8/1/161" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/8/1/161" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/8/1/161" class="active" data-icon-position="" data-hide-link-title="0">Genotyping by Sequencing in Almond: SNP Discovery, Linkage Mapping, and Marker Design</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/487826" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-of8Q2bdL8i6fxPgcFyAoSihf9qwXpUUAxWDbOx_fa1o" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/161.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/8/1/161" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-9cI4RijqWvy0ixScrkf0itqV5HjvctpMIUOmJgi4T4U" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/8/1/161" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/8/1/161.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487826" id="node487826--2" data-pisa="ggg;8/1/161" data-pisa-master="ggg;g3.117.300376" data-apath="/ggg/8/1/161.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotyping by Sequencing in Almond: SNP Discovery, Linkage Mapping, and Marker Design</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7539-011X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Shashi N. Goonetilleke</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2066-9495" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Timothy J. March</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-1174-6580" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michelle G. Wirthensohn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0939-8038" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pere Arús</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7596-8484" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amanda R. Walker</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-7506-2589" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diane E. Mather</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">161-172;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300376 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487826--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shashi N. Goonetilleke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Agriculture, Food and Wine, Waite Research Institute, The University of Adelaide, Glen Osmond, 5064, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shashi%2BN.%2BGoonetilleke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goonetilleke%20SN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShashi%2BN.%2BGoonetilleke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7539-011X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Shashi N. Goonetilleke</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Timothy J. March </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Agriculture, Food and Wine, Waite Research Institute, The University of Adelaide, Glen Osmond, 5064, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BJ.%2BMarch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=March%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimothy%2BJ.%2BMarch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2066-9495" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timothy J. March</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michelle G. Wirthensohn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Agriculture, Food and Wine, Waite Research Institute, The University of Adelaide, Glen Osmond, 5064, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michelle%2BG.%2BWirthensohn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wirthensohn%20MG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichelle%2BG.%2BWirthensohn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1174-6580" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michelle G. Wirthensohn</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Pere Arús </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institut de Recerca i Tecnologia Agroalimentàries, Centre de Recerca en Agrigenòmica, Consejo Superior de Investigaciones Científicas–Institut de Recerca i Tecnologia Agroalimentàries–Universitat Autònoma de Barcelona–University of Barcelona, Campus Universitat Autònoma de Barcelona, 08913, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pere%2BAr%C3%BAs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ar%C3%BAs%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APere%2BAr%25C3%25BAs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0939-8038" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pere Arús</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Amanda R. Walker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Agriculture and Food, Waite Campus, Commonwealth Scientific and Industrial Research Organisation, Glen Osmond, 5064, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BR.%2BWalker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walker%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmanda%2BR.%2BWalker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7596-8484" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amanda R. Walker</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Diane E. Mather </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Agriculture, Food and Wine, Waite Research Institute, The University of Adelaide, Glen Osmond, 5064, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diane%2BE.%2BMather%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mather%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiane%2BE.%2BMather%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7506-2589" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diane E. Mather</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:diane.mather@adelaide.edu.au" class="" data-icon-position="" data-hide-link-title="0">diane.mather@adelaide.edu.au</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/8/1/161" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/8/1/161&amp;title=Genotyping%20by%20Sequencing%20in%20Almond%3A%20SNP%20Discovery%2C%20Linkage%20Mapping%2C%20and%20Marker%20Design" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/8/1/161&amp;title=Genotyping%20by%20Sequencing%20in%20Almond%3A%20SNP%20Discovery%2C%20Linkage%20Mapping%2C%20and%20Marker%20Design" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/8/1/161&amp;title=Genotyping%20by%20Sequencing%20in%20Almond%3A%20SNP%20Discovery%2C%20Linkage%20Mapping%2C%20and%20Marker%20Design" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/8/1/161&amp;text=Genotyping%20by%20Sequencing%20in%20Almond%3A%20SNP%20Discovery%2C%20Linkage%20Mapping%2C%20and%20Marker%20Design" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/8/1/161&amp;title=Genotyping%20by%20Sequencing%20in%20Almond%3A%20SNP%20Discovery%2C%20Linkage%20Mapping%2C%20and%20Marker%20Design" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/8/1/161&amp;t=Genotyping%20by%20Sequencing%20in%20Almond%3A%20SNP%20Discovery%2C%20Linkage%20Mapping%2C%20and%20Marker%20Design" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/8/1/161&amp;title=Genotyping%20by%20Sequencing%20in%20Almond%3A%20SNP%20Discovery%2C%20Linkage%20Mapping%2C%20and%20Marker%20Design" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/8/1/161&amp;title=Genotyping%20by%20Sequencing%20in%20Almond%3A%20SNP%20Discovery%2C%20Linkage%20Mapping%2C%20and%20Marker%20Design" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/487826/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="487826" id="node487826--3" data-pisa="ggg;8/1/161" data-pisa-master="ggg;g3.117.300376" data-apath="/ggg/8/1/161.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genotyping by Sequencing in Almond: SNP Discovery, Linkage Mapping, and Marker Design</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7539-011X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Shashi N. Goonetilleke</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-2066-9495" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Timothy J. March</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0003-1174-6580" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michelle G. Wirthensohn</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-0939-8038" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Pere Arús</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-7596-8484" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Amanda R. Walker</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-7506-2589" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Diane E. Mather</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">January 1, 2018 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 8 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 1 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">161-172;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.117.300376 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node487826--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shashi N. Goonetilleke </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Agriculture, Food and Wine, Waite Research Institute, The University of Adelaide, Glen Osmond, 5064, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shashi%2BN.%2BGoonetilleke%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goonetilleke%20SN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AShashi%2BN.%2BGoonetilleke%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7539-011X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Shashi N. Goonetilleke</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Timothy J. March </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Agriculture, Food and Wine, Waite Research Institute, The University of Adelaide, Glen Osmond, 5064, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Timothy%2BJ.%2BMarch%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=March%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATimothy%2BJ.%2BMarch%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-2066-9495" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Timothy J. March</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michelle G. Wirthensohn </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Agriculture, Food and Wine, Waite Research Institute, The University of Adelaide, Glen Osmond, 5064, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michelle%2BG.%2BWirthensohn%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wirthensohn%20MG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichelle%2BG.%2BWirthensohn%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1174-6580" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michelle G. Wirthensohn</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Pere Arús </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institut de Recerca i Tecnologia Agroalimentàries, Centre de Recerca en Agrigenòmica, Consejo Superior de Investigaciones Científicas–Institut de Recerca i Tecnologia Agroalimentàries–Universitat Autònoma de Barcelona–University of Barcelona, Campus Universitat Autònoma de Barcelona, 08913, Spain</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Pere%2BAr%C3%BAs%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ar%C3%BAs%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APere%2BAr%25C3%25BAs%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0939-8038" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Pere Arús</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Amanda R. Walker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Agriculture and Food, Waite Campus, Commonwealth Scientific and Industrial Research Organisation, Glen Osmond, 5064, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Amanda%2BR.%2BWalker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Walker%20AR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAmanda%2BR.%2BWalker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7596-8484" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Amanda R. Walker</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Diane E. Mather </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>School of Agriculture, Food and Wine, Waite Research Institute, The University of Adelaide, Glen Osmond, 5064, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diane%2BE.%2BMather%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mather%20DE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADiane%2BE.%2BMather%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7506-2589" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Diane E. Mather</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:diane.mather@adelaide.edu.au" class="" data-icon-position="" data-hide-link-title="0">diane.mather@adelaide.edu.au</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=161&amp;rft.epage=172&amp;rft.atitle=Genotyping%20by%20Sequencing%20in%20Almond%3A%20SNP%20Discovery%2C%20Linkage%20Mapping%2C%20and%20Marker%20Design&amp;rft.volume=8&amp;rft.issue=1&amp;rft.date=2018-01-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Goonetilleke%2C+Shashi+N.&amp;rft.au=March%2C+Timothy+J.&amp;rft.au=Wirthensohn%2C+Michelle+G.&amp;rft.au=Ar%C3%BAs%2C+Pere&amp;rft.au=Walker%2C+Amanda+R.&amp;rft.au=Mather%2C+Diane+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/487826/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/487826/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/487826/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/487826/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/487826/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/487826/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/487826/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/487826/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/487826/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/487826/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/487826/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/487826/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/8/1/161" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/8/1/161.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/8/1/161.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/8/1/161.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/8/1/161.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
