<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/3/449" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/3/449.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/3/449.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/3/449.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/3/449.atom" />
<meta name="HW.pisa" content="ggg;5/3/449" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Genome-Wide Association Study of Resistance to Stripe Rust (Puccinia striiformis f. sp. tritici) in a Worldwide Collection of Hexaploid Spring Wheat (Triticum aestivum L.)" />
<meta name="DC.Identifier" content="10.1534/g3.114.014563" />
<meta name="DC.Date" content="2015-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Maccaferri et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="New races of Puccinia striiformis f. sp. tritici ( Pst ), the causal pathogen of wheat stripe rust, show high virulence to previously deployed resistance genes and are responsible for large yield losses worldwide. To identify new sources of resistance we performed a genome-wide association study (GWAS) using a worldwide collection of 1000 spring wheat accessions. Adult plants were evaluated under field conditions in six environments in the western United States, and seedlings were tested with four Pst races. A single-nucleotide polymorphism (SNP) Infinium 9K-assay provided 4585 SNPs suitable for GWAS. High correlations among environments and high heritabilities were observed for stripe rust infection type and severity. Greater levels of Pst resistance were observed in a subpopulation from Southern Asia than in other groups. GWAS identified 97 loci that were significant for at least three environments, including 10 with an experiment-wise adjusted Bonferroni probability &lt; 0.10. These 10 quantitative trait loci (QTL) explained 15% of the phenotypic variation in infection type, a percentage that increased to 45% when all QTL were considered. Three of these 10 QTL were mapped far from previously identified Pst resistance genes and QTL, and likely represent new resistance loci. The other seven QTL mapped close to known resistance genes and allelism tests will be required to test their relationships. In summary, this study provides an integrated view of stripe rust resistance resources in spring wheat and identifies new resistance loci that will be useful to diversify the current set of resistance genes deployed to control this devastating disease." />
<meta name="DC.Contributor" content="Marco Maccaferri" />
<meta name="DC.Contributor" content="Junli Zhang" />
<meta name="DC.Contributor" content="Peter Bulli" />
<meta name="DC.Contributor" content="Zewdie Abate" />
<meta name="DC.Contributor" content="Shiaoman Chao" />
<meta name="DC.Contributor" content="Dario Cantu" />
<meta name="DC.Contributor" content="Eligio Bossolini" />
<meta name="DC.Contributor" content="Xianming Chen" />
<meta name="DC.Contributor" content="Michael Pumphrey" />
<meta name="DC.Contributor" content="Jorge Dubcovsky" />
<meta name="article:published_time" content="2015-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Genome-Wide Association Study of Resistance to Stripe Rust (Puccinia striiformis f. sp. tritici) in a Worldwide Collection of Hexaploid Spring Wheat (Triticum aestivum L.)" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;New races of &lt;i&gt;Puccinia striiformis&lt;/i&gt; f. sp. &lt;i&gt;tritici&lt;/i&gt; (&lt;i&gt;Pst&lt;/i&gt;), the causal pathogen of wheat stripe rust, show high virulence to previously deployed resistance genes and are responsible for large yield losses worldwide. To identify new sources of resistance we performed a genome-wide association study (GWAS) using a worldwide collection of 1000 spring wheat accessions. Adult plants were evaluated under field conditions in six environments in the western United States, and seedlings were tested with four &lt;i&gt;Pst&lt;/i&gt; races. A single-nucleotide polymorphism (SNP) Infinium 9K-assay provided 4585 SNPs suitable for GWAS. High correlations among environments and high heritabilities were observed for stripe rust infection type and severity. Greater levels of &lt;i&gt;Pst&lt;/i&gt; resistance were observed in a subpopulation from Southern Asia than in other groups. GWAS identified 97 loci that were significant for at least three environments, including 10 with an experiment-wise adjusted Bonferroni probability &amp;lt; 0.10. These 10 quantitative trait loci (QTL) explained 15% of the phenotypic variation in infection type, a percentage that increased to 45% when all QTL were considered. Three of these 10 QTL were mapped far from previously identified &lt;i&gt;Pst&lt;/i&gt; resistance genes and QTL, and likely represent new resistance loci. The other seven QTL mapped close to known resistance genes and allelism tests will be required to test their relationships. In summary, this study provides an integrated view of stripe rust resistance resources in spring wheat and identifies new resistance loci that will be useful to diversify the current set of resistance genes deployed to control this devastating disease.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/03/01" />
<meta name="citation_mjid" content="ggg;5/3/449" />
<meta name="citation_id" content="5/3/449" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/3/449" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/3/449.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/3/449.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/3/449.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.014563" />
<meta name="citation_pmid" content="25609748" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="449" />
<meta name="citation_lastpage" content="465" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Marco Maccaferri" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author_institution" content="Department of Agricultural Sciences (DipSA), University of Bologna, Bologna 40127, Italy" />
<meta name="citation_author" content="Junli Zhang" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author" content="Peter Bulli" />
<meta name="citation_author_institution" content="Department of Crop and Soil Sciences, Washington State University, Pullman, Washington 99164-6420" />
<meta name="citation_author" content="Zewdie Abate" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author" content="Shiaoman Chao" />
<meta name="citation_author_institution" content="USDA-ARS, 1605 Albrecht Blvd, Fargo, North Dakota 58105" />
<meta name="citation_author" content="Dario Cantu" />
<meta name="citation_author_institution" content="Department of Viticulture and Enology, University of California, Davis, California 95616" />
<meta name="citation_author" content="Eligio Bossolini" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author" content="Xianming Chen" />
<meta name="citation_author_institution" content="USDA-ARS, Wheat Genetics, Quality Physiology, and Disease Research Unit, and Department of Plant Pathology, Washington State University, Pullman, Washington 99164" />
<meta name="citation_author" content="Michael Pumphrey" />
<meta name="citation_author_institution" content="Department of Crop and Soil Sciences, Washington State University, Pullman, Washington 99164-6420" />
<meta name="citation_author" content="Jorge Dubcovsky" />
<meta name="citation_author_institution" content="Department of Plant Sciences, University of California, Davis, California 95616" />
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Chevy Chase, Maryland 20815" />
<meta name="citation_author_email" content="Jdubcovsky@ucdavis.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7571-4345" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=T. B. Adhikari;citation_author=E. W. Jackson;citation_author=S. Gurung;citation_author=J. M. Hansen;citation_author=J. M. Bonman;citation_title=Association mapping of quantitative resistance to Phaeosphaeria nodorum in spring wheat landraces from the USDA National Small Grains Collection.;citation_pages=1301-1310;citation_volume=101;citation_year=2011;citation_issue=11;citation_pmid=21692647;citation_doi=10.1094/PHYTO-03-11-0076" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=G. M. Agenbag;citation_author=Z. A. Pretorius;citation_author=L. A. Boyd;citation_author=C. M. Bender;citation_author=R. Prins;citation_title=Identification of adult plant resistance to stripe rust in the wheat cultivar Cappelle-Desprez.;citation_pages=109-120;citation_volume=125;citation_year=2012;citation_issue=1;citation_pmid=22350093;citation_doi=10.1007/s00122-012-1819-5" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=S. Ali;citation_author=P. Gladieux;citation_author=M. Leconte;citation_author=A. Gautier;citation_author=A. F. Justesen;citation_title=Origin, migration routes and worldwide population genetic structure of the wheat yellow rust pathogen Puccinia striiformis f.sp. tritici.;citation_pages=e1003903;citation_volume=10;citation_year=2014;citation_pmid=24465211;citation_doi=10.1371/journal.ppat.1003903" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=K. G. Ardlie;citation_author=L. Kruglyak;citation_author=M. Seielstad;citation_title=Patterns of linkage disequilibrium in the human genome.;citation_pages=299-309;citation_volume=3;citation_year=2002;citation_issue=4;citation_pmid=11967554;citation_doi=10.1038/nrg777" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. C. Barrett;citation_author=B. Fry;citation_author=J. Maller;citation_author=M. J. Daly;citation_title=Haploview: analysis and visualization of LD and haplotype maps.;citation_volume=21;citation_year=2005;citation_pmid=15297300;citation_doi=10.1093/bioinformatics/bth457" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. Brachi;citation_author=N. Faure;citation_author=M. Horton;citation_author=E. Flahauw;citation_author=A. Vazquez;citation_title=Linkage and association mapping of Arabidopsis thaliana flowering time in nature.;citation_pages=e1000940-e1000940;citation_volume=6;citation_year=2010;citation_issue=5;citation_pmid=20463887;citation_doi=10.1371/journal.pgen.1000940" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Brachi;citation_author=G. P. Morris;citation_author=J. O. Borevitz;citation_title=Genome-wide association studies in plants: the missing heritability is in the field.;citation_pages=232-232;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=22035733;citation_doi=10.1186/gb-2011-12-10-232" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Breseghello;citation_author=M. E. Sorrells;citation_title=Association mapping of kernel size and milling quality in wheat (Triticum aestivum L.) cultivars.;citation_volume=172;citation_year=2006;citation_pmid=16079235;citation_doi=10.1534/genetics.105.044586" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. R. Cavanagh;citation_author=S. M. Chao;citation_author=S. C. Wang;citation_author=B. E. Huang;citation_author=S. Stephen;citation_title=Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.;citation_volume=110;citation_year=2013;citation_pmid=23630259;citation_doi=10.1073/pnas.1217133110" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. M. Chao;citation_author=J. Dubcovsky;citation_author=J. Dvorak;citation_author=M. C. Luo;citation_author=S. P. Baenziger;citation_title=Population- and genome-specific patterns of linkage disequilibrium and SNP variation in spring and winter wheat (Triticum aestivum L.).;citation_pages=727-727;citation_volume=11;citation_year=2010;citation_pmid=21190581;citation_doi=10.1186/1471-2164-11-727" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant Pathol.;citation_author=W. Chen;citation_author=C. Wellings;citation_author=X. M. Chen;citation_author=Z. Kang;citation_author=T. Liu;citation_title=Wheat stripe (yellow) rust caused by Puccinia striiformis f. sp. tritici.;citation_pages=433-446;citation_volume=15;citation_year=2014;citation_pmid=24373199;citation_doi=10.1111/mpp.12116" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Plant Pathol.;citation_author=X. M. Chen;citation_title=Epidemiology and control of stripe rust (Puccinia striiformis f. sp tritici) on wheat.;citation_pages=314-337;citation_volume=27;citation_year=2005;citation_doi=10.1080/07060660509507230" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Plant Sci.;citation_author=X. M. Chen;citation_title=High-temperature adult-plant resistance, key for sustainable control of stripe rust.;citation_pages=608-627;citation_volume=4;citation_year=2013;citation_doi=10.4236/ajps.2013.43080" />
<meta name="citation_reference" content="citation_journal_title=Can. J. Plant Pathol.;citation_author=X. M. Chen;citation_author=L. Penman;citation_author=A. M. Wan;citation_author=P. Cheng;citation_title=Virulence races of Puccinia striiformis f. sp tritici in 2006 and 2007 and development of wheat stripe rust and distributions, dynamics, and evolutionary relationships of races from 2000 to 2007 in the United States.;citation_pages=315-333;citation_volume=32;citation_year=2010;citation_doi=10.1080/07060661.2010.499271" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. T. Chisholm;citation_author=G. Coaker;citation_author=B. Day;citation_author=B. J. Staskawicz;citation_title=Host-microbe interactions: shaping the evolution of the plant immune response.;citation_pages=803-814;citation_volume=124;citation_year=2006;citation_issue=4;citation_pmid=16497589;citation_doi=10.1016/j.cell.2006.02.008" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=M. D. Christopher;citation_author=S. Y. Liu;citation_author=M. D. Hall;citation_author=D. S. Marshall;citation_author=M. O. Fountain;citation_title=Identification and mapping of adult-plant stripe rust resistance in soft red winter wheat cultivar ’USG 3555’.;citation_pages=53-60;citation_volume=132;citation_year=2013;citation_doi=10.1111/pbr.12015" />
<meta name="citation_reference" content="citation_journal_title=Technometrics;citation_author=R. R. Corbeil;citation_author=S. R. Searle;citation_title=Restricted maximum likelihood (REML) estimation of variance components in mixed model.;citation_pages=31-38;citation_volume=18;citation_year=1976;citation_doi=10.2307/1267913" />
<meta name="citation_reference" content="citation_journal_title=Behavior genetics;citation_journal_abbrev=Behav Genet;citation_author=A. Darvasi;citation_author=M. Soller;citation_title=A simple method to calculate resolving power and confidence interval of QTL map location.;citation_pages=125-132;citation_volume=27;citation_year=1997;citation_issue=2;citation_pmid=9145551;citation_doi=10.1023/A:1025685324830" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=K. M. Devos;citation_author=J. Dubcovsky;citation_author=J. Dvorak;citation_author=C. N. Chinoy;citation_author=M. D. Gale;citation_title=Structural evolution of wheat chromosomes 4A, 5A, and 7B and its impact on recombination.;citation_pages=282-288;citation_volume=91;citation_year=1995;citation_pmid=24169776;citation_doi=10.1007/BF00220890" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Sci.;citation_author=J. P. R. E. Dimmock;citation_author=M. J. Gooding;citation_title=The influence of foliar diseases, and their control by fungicides, on the protein concentration in wheat grain: a review.;citation_pages=349-366;citation_volume=138;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_abbrev=Int J Mol Sci;citation_author=D. Dodig;citation_author=M. Zoric;citation_author=B. Kobiljski;citation_author=J. Savic;citation_author=V. Kandic;citation_title=Genetic and association mapping study of wheat agronomic traits under contrasting water regimes.;citation_pages=6167-6188;citation_volume=13;citation_year=2012;citation_issue=5;citation_pmid=22754357;citation_doi=10.3390/ijms13056167" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Dubcovsky;citation_author=J. Dvorak;citation_title=Genome Plasticity a Key Factor in the Success of Polyploid Wheat Under Domestication;citation_pages=1862-1866;citation_volume=316;citation_year=2007;citation_issue=5833;citation_pmid=17600208;citation_doi=10.1126/science.1143986" />
<meta name="citation_reference" content="citation_journal_title=Conserv. Genet. Resour.;citation_author=D. A. Earl;citation_author=B. M. Vonholdt;citation_title=STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method.;citation_pages=359-361;citation_volume=4;citation_year=2012;citation_doi=10.1007/s12686-011-9548-7" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=G. Evanno;citation_author=S. Regnaut;citation_author=J. Goudet;citation_title=Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.;citation_pages=2611-2620;citation_volume=14;citation_year=2005;citation_issue=8;citation_pmid=15969739;citation_doi=10.1111/j.1365-294X.2005.02553.x" />
<meta name="citation_reference" content="citation_journal_title=Evol Bioinform;citation_author=L. Excoffier;citation_author=G. Laval;citation_author=S. Schneider;citation_title=Arlequin (version 3.0): an integrated software package for population genetics data analysis.;citation_pages=47-50;citation_volume=1;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Plant Biol.;citation_author=S. A. Flint-Garcia;citation_author=J. M. Thornsberry;citation_author=E. S. Buckler;citation_title=Structure of linkage disequilibrium in plants.;citation_volume=54;citation_year=2003;citation_pmid=14502995;citation_doi=10.1146/annurev.arplant.54.031902.134907" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=K. Forrest;citation_author=V. Pujol;citation_author=P. Bulli;citation_author=M. Pumphrey;citation_author=C. Wellings;citation_title=Development of a SNP marker assay for the Lr67 gene of wheat using a genotyping by sequencing approach.;citation_pages=2109-2118;citation_volume=34;citation_year=2014;citation_doi=10.1007/s11032-014-0166-4" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. Y. Hao;citation_author=Y. Q. Wang;citation_author=J. Hou;citation_author=C. Feuillet;citation_author=F. Balfourier;citation_title=Association Mapping and Haplotype Analysis of a 3.1-Mb Genomic Region Involved in Fusarium Head Blight Resistance on Wheat Chromosome 3BS.;citation_pages=e46444-e46444;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23071572;citation_doi=10.1371/journal.pone.0046444" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=D. P. Hodson;citation_title=Shifting boundaries: challenges for rust monitoring.;citation_pages=93-104;citation_volume=179;citation_year=2011;citation_doi=10.1007/s10681-010-0335-4" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=K. E. Holsinger;citation_author=B. S. Weir;citation_title=Genetics in geographically structured populations: defining, estimating and interpreting F(ST).;citation_pages=639-650;citation_volume=10;citation_year=2009;citation_issue=9;citation_pmid=19687804;citation_doi=10.1038/nrg2611" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=M. S. Hovmoller;citation_author=A. H. Yahyaoui;citation_author=E. A. Milus;citation_author=A. F. Justesen;citation_title=Rapid global spread of two aggressive strains of a wheat rust fungus.;citation_pages=3818-3826;citation_volume=17;citation_year=2008;citation_issue=17;citation_pmid=18673440;citation_doi=10.1111/j.1365-294X.2008.03886.x" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=G. Q. Jia;citation_author=X. H. Huang;citation_author=H. Zhi;citation_author=Y. Zhao;citation_author=Q. Zhao;citation_title=A haplotype map of genomic variations and genome-wide association studies of agronomic traits in foxtail millet (Setaria italica).;citation_pages=957-961;citation_volume=45;citation_year=2013;citation_pmid=23793027;citation_doi=10.1038/ng.2673" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=R. Johnson;citation_title=A Critical Analysis of Durable Resistance;citation_pages=309-330;citation_volume=22;citation_year=1984;citation_issue=1;citation_doi=10.1146/annurev.py.22.090184.001521" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. M. Kang;citation_author=N. A. Zaitlen;citation_author=C. M. Wade;citation_author=A. Kirby;citation_author=D. Heckerman;citation_title=Efficient Control of Population Structure in Model Organism Association Mapping;citation_pages=1709-1723;citation_volume=178;citation_year=2008;citation_issue=3;citation_pmid=18385116;citation_doi=10.1534/genetics.107.080101" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. S. Khatkar;citation_author=F. W. Nicholas;citation_author=A. R. Collins;citation_author=K. R. Zenger;citation_author=J. Al Cavanagh;citation_title=Extent of genome-wide linkage disequilibrium in Australian Holstein-Friesian cattle based on a high-density SNP panel.;citation_pages=187-187;citation_volume=9;citation_year=2008;citation_pmid=18435834;citation_doi=10.1186/1471-2164-9-187" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=S. Kollers;citation_author=B. Rodemann;citation_author=J. Ling;citation_author=V. Korzun;citation_author=E. Ebmeyer;citation_title=Whole genome association mapping of Fusarium head blight resistance in European winter wheat (Triticum aestivum L.).;citation_pages=e57500;citation_volume=8;citation_year=2013;citation_pmid=23451238;citation_doi=10.1371/journal.pone.0057500" />
<meta name="citation_reference" content="citation_journal_title=Annual review of phytopathology;citation_journal_abbrev=Annu Rev Phytopathol;citation_author=J. A. Kolmer;citation_title=GENETICS OF RESISTANCE TO WHEAT LEAF RUST;citation_pages=435-455;citation_volume=34;citation_year=1996;citation_issue=1;citation_pmid=15012551;citation_doi=10.1146/annurev.phyto.34.1.435" />
<meta name="citation_reference" content="citation_title=Wheat: Science and Trade;citation_title=Diseases which challenge global wheat production-the wheat rusts;citation_author=J. A. Kolmer;citation_author=X. Chen;citation_author=J. Yue;citation_pages=89-124;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=K. V. Krasileva;citation_author=V. Buffalo;citation_author=P. Bailey;citation_author=S. Pearce;citation_author=S. Ayling;citation_title=Separating homeologs by phasing in the tetraploid wheat transcriptome.;citation_pages=R66;citation_volume=14;citation_year=2013;citation_pmid=23800085;citation_doi=10.1186/gb-2013-14-6-r66" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. G. Krattinger;citation_author=E. S. Lagudah;citation_author=W. Spielmeyer;citation_author=R. P. Singh;citation_author=J. Huerta-Espino;citation_title=A putative ABC transporter confers durable resistance to multiple fungal pathogens in wheat.;citation_volume=323;citation_year=2009;citation_pmid=19229000;citation_doi=10.1126/science.1166453" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=P. Kulwal;citation_author=G. Ishikawa;citation_author=D. Benscher;citation_author=Z. Y. Feng;citation_author=L. X. Yu;citation_title=Association mapping for pre-harvest sprouting resistance in white winter wheat.;citation_pages=793-805;citation_volume=125;citation_year=2012;citation_issue=4;citation_pmid=22547141;citation_doi=10.1007/s00122-012-1872-0" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=E. S. Lagudah;citation_author=S. G. Krattinger;citation_author=S. Herrera-Foessel;citation_author=R. P. Singh;citation_author=J. Huerta-Espino;citation_title=Gene-specific markers for the wheat gene Lr34/Yr18/Pm38 which confers resistance to multiple fungal pathogens.;citation_pages=889-898;citation_volume=119;citation_year=2009;citation_issue=5;citation_pmid=19578829;citation_doi=10.1007/s00122-009-1097-z" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. J. Larsson;citation_author=A. E. Lipka;citation_author=E. S. Buckler;citation_title=Lessons from Dwarf8 on the strengths and weaknesses of structured association mapping.;citation_pages=e1003246;citation_volume=9;citation_year=2013;citation_pmid=23437002;citation_doi=10.1371/journal.pgen.1003246" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. Lawrence;citation_author=A. G. Day-Williams;citation_author=R. Mott;citation_author=J. Broxholme;citation_author=L. R. Cardon;citation_title=GLIDERS--a web-based search engine for genome-wide linkage disequilibrium between HapMap SNPs.;citation_pages=367-367;citation_volume=10;citation_year=2009;citation_pmid=19878600;citation_doi=10.1186/1471-2105-10-367" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=T. Letta;citation_author=M. Maccaferri;citation_author=A. Badebo;citation_author=K. Ammar;citation_author=A. Ricci;citation_title=Searching for novel sources of field resistance to Ug99 and Ethiopian stem rust races in durum wheat via association mapping.;citation_pages=1237-1256;citation_volume=126;citation_year=2013;citation_pmid=23429902;citation_doi=10.1007/s00122-013-2050-8" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Phytopathol.;citation_author=R. F. Line;citation_title=Stripe rust of wheat and barley in North America: a retrospective historical review.;citation_volume=40;citation_year=2002;citation_pmid=12147755;citation_doi=10.1146/annurev.phyto.40.020102.111645" />
<meta name="citation_reference" content="citation_title=Virulence, Aggressiveness, Evolution and Distribution of races of Puccinia striiformis (the Cause of Stripe Rust of Wheat) in North America, 1968–1987. Technical;citation_author=R. F. Line;citation_author=A. Qayoum;citation_volume=Vol. 1788;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. E. Lipka;citation_author=F. Tian;citation_author=Q. S. Wang;citation_author=J. Peiffer;citation_author=M. Li;citation_title=GAPIT: genome association and prediction integrated tool.;citation_volume=28;citation_year=2012;citation_pmid=22796960;citation_doi=10.1093/bioinformatics/bts444" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. E. Lipka;citation_author=M. A. Gore;citation_author=M. Magallanes-Lundback;citation_author=A. Mesberg;citation_author=H. N. Lin;citation_title=Genome-wide association study and pathway-level analysis of tocochromanol levels in maize grain.;citation_pages=1287-1299;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=I. Lowe;citation_author=D. Cantu;citation_author=J. Dubcovsky;citation_title=Durable resistance to the wheat rusts: integrating systems biology and traditional phenotype-based research methods to guide the deployment of resistance genes.;citation_pages=69-79;citation_volume=179;citation_year=2011;citation_doi=10.1007/s10681-010-0311-z" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=I. Lowe;citation_author=D. L. Jankuloski;citation_author=S. Chao;citation_author=X. Chen;citation_author=D. See;citation_title=Mapping and validation of QTL which confer partial resistance to broadly virulent post-2000 North American races of stripe rust in hexaploid wheat.;citation_pages=143-157;citation_volume=123;citation_year=2011;citation_issue=1;citation_pmid=21455722;citation_doi=10.1007/s00122-011-1573-0" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_title=A high-density, SNP-based consensus map of tetraploid wheat as a bridge to integrate durum and bread wheat genomics and breeding.;citation_author=M. Maccaferri;citation_author=A. Ricci;citation_author=S. Salvi;citation_author=S. G. Milner;citation_author=E. Noli;citation_year=2014" />
<meta name="citation_reference" content="citation_title=A Uniform System for Recording and Processing Cereal Research Data;citation_author=F. McNeal;citation_author=C. Konzak;citation_author=E. Smith;citation_author=W. Tate;citation_author=T. Russell;citation_year=1971" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=E. A. Milus;citation_author=K. Kristensen;citation_author=M. S. Hovmoller;citation_title=Evidence for increased aggressiveness in a recent widespread strain of Puccinia striiformis f. sp. tritici causing stripe rust of wheat.;citation_pages=89-94;citation_volume=99;citation_year=2009;citation_issue=1;citation_pmid=19055439;citation_doi=10.1094/PHYTO-99-1-0089" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=A. Mohan;citation_author=P. Kulwal;citation_author=R. Singh;citation_author=V. Kumar;citation_author=R. R. Mir;citation_title=Genome-wide QTL analysis for pre-harvest sprouting tolerance in bread wheat.;citation_pages=319-329;citation_volume=168;citation_year=2009;citation_doi=10.1007/s10681-009-9935-2" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. K. Monaco;citation_author=J. Stein;citation_author=S. Naithani;citation_author=S. Wei;citation_author=P. Dharmawardhana;citation_title=Gramene 2013: comparative plant genomics resources.;citation_volume=42;citation_year=2014;citation_pmid=24217918;citation_doi=10.1093/nar/gkt1110" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Nemri;citation_author=S. Atwell;citation_author=A. M. Tarone;citation_author=Y. S. Huang;citation_author=K. Zhao;citation_title=Genome-wide survey of Arabidopsis natural variation in downy mildew resistance using combined association and linkage mapping.;citation_volume=107;citation_year=2010;citation_pmid=20479233;citation_doi=10.1073/pnas.0913160107" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=J. A. Peiffer;citation_author=S. A. Flint-Garcia;citation_author=N. De Leon;citation_author=M. D. McMullen;citation_author=S. M. Kaeppler;citation_title=The genetic architecture of maize stalk strength.;citation_pages=e67066;citation_volume=8;citation_year=2013;citation_pmid=23840585;citation_doi=10.1371/journal.pone.0067066" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. A. Peiffer;citation_author=M. C. Romay;citation_author=M. A. Gore;citation_author=S. A. Flint-Garcia;citation_author=Z. W. Zhang;citation_title=The genetic architecture of maize height.;citation_volume=196;citation_year=2014;citation_pmid=24514905;citation_doi=10.1534/genetics.113.159152" />
<meta name="citation_reference" content="citation_title=The Wheat Plant: A Monograph;citation_author=J. Percival;citation_year=1921" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. C. Reif;citation_author=M. Gowda;citation_author=H. P. Maurer;citation_author=C. F. H. Longin;citation_author=V. Korzun;citation_title=Association mapping for quality traits in soft winter wheat.;citation_pages=961-970;citation_volume=122;citation_year=2011;citation_issue=5;citation_pmid=21153626;citation_doi=10.1007/s00122-010-1502-7" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=S. Reimer;citation_author=C. J. Pozniak;citation_author=F. R. Clarke;citation_author=J. M. Clarke;citation_author=D. J. Somers;citation_title=Association mapping of yellow pigment in an elite collection of durum wheat cultivars and breeding lines.;citation_pages=1016-1025;citation_volume=51;citation_year=2008;citation_issue=12;citation_pmid=19088814" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=C. E. Rexroad;citation_author=R. L. Vallejo;citation_title=Estimates of linkage disequilibrium and effective population size in rainbow trout.;citation_pages=83-83;citation_volume=10;citation_year=2009;citation_pmid=20003428" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda).;citation_author=C. Saintenac;citation_author=D. Jiang;citation_author=S. Wang;citation_author=E. Akhunov;citation_title=Sequence-based mapping of the polyploid wheat genome.;citation_volume=3;citation_year=2013;citation_pmid=23665877;citation_doi=10.1534/g3.113.005819" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. K. Santra;citation_author=X. M. Chen;citation_author=M. Santra;citation_author=K. G. Campbell;citation_author=K. K. Kidwell;citation_title=Identification and mapping QTL for high-temperature adult-plant resistance to stripe rust in winter wheat (Triticum aestivum L.) cultivar 'Stephens'.;citation_pages=793-802;citation_volume=117;citation_year=2008;citation_issue=5;citation_pmid=18584147;citation_doi=10.1007/s00122-008-0820-5" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Biometeorol.;citation_author=D. Sharma-Poudyal;citation_author=X. Chen;citation_author=R. A. Rupp;citation_title=Potential oversummering and overwintering regions for the wheat stripe rust pathogen in the contiguous United States.;citation_pages=987-997;citation_volume=58;citation_year=2013;citation_pmid=23722926" />
<meta name="citation_reference" content="citation_journal_title=Human Molecular Genetics;citation_journal_abbrev=Hum Mol Genet;citation_author=S. Shifman;citation_author=J. Kuypers;citation_author=M. Kokoris;citation_author=B. Yakir;citation_author=A. Darvasi;citation_title=Linkage disequilibrium patterns of the human genome across populations;citation_pages=771-776;citation_volume=12;citation_year=2003;citation_issue=7;citation_pmid=12651872;citation_doi=10.1093/hmg/ddg088" />
<meta name="citation_reference" content="citation_journal_title=Acta Phytopathol Hun;citation_author=R. P. Singh;citation_author=J. Huerta-Espino;citation_author=S. Rajaram;citation_title=Achieving near-immunity to leaf and stripe rusts in wheat by combining slow rusting resistance genes.;citation_pages=133-139;citation_volume=35;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Turk. J. Agric. For.;citation_author=R. P. Singh;citation_author=J. Huerta-Espino;citation_author=H. M. William;citation_title=Genetics and breeding for durable resistance to leaf and stripe rusts in wheat.;citation_pages=121-127;citation_volume=29;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. J. Somers;citation_author=P. Isaac;citation_author=K. Edwards;citation_title=A high-density microsatellite consensus map for bread wheat (Triticum aestivum L.).;citation_pages=1105-1114;citation_volume=109;citation_year=2004;citation_issue=6;citation_pmid=15490101;citation_doi=10.1007/s00122-004-1740-7" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=Q. J. Song;citation_author=J. R. Shi;citation_author=S. Singh;citation_author=E. W. Fickus;citation_author=J. M. Costa;citation_title=Development and mapping of microsatellite (SSR) markers in wheat.;citation_pages=550-560;citation_volume=110;citation_year=2005;citation_issue=3;citation_pmid=15655666;citation_doi=10.1007/s00122-004-1871-x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=O. Sosnowski;citation_author=A. Charcosset;citation_author=J. Joets;citation_title=BioMercator V3: an upgrade of genetic map compilation and quantitative trait loci meta-analysis algorithms.;citation_volume=28;citation_year=2012;citation_pmid=22661647;citation_doi=10.1093/bioinformatics/bts313" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=N. C. Stewart;citation_author=L. E. Via;citation_title=A rapid CTAB DNA isolation technique useful for RAPD fingerprinting and other PCR applications.;citation_pages=748-750;citation_volume=14;citation_year=1993;citation_issue=5;citation_pmid=8512694" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Stich;citation_author=J. Mohring;citation_author=H. P. Piepho;citation_author=M. Heckenberger;citation_author=E. S. Buckler;citation_title=Comparison of mixed-model approaches for association mapping.;citation_volume=178;citation_year=2008;citation_pmid=18245847;citation_doi=10.1534/genetics.107.079707" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. V. Sukhwinder-Singh;citation_author=J. Hernandez;citation_author=P. K. Crossa;citation_author=N. S. Singh;citation_author=. Bains;citation_title=Multi-trait and multi-environment QTL analyses for resistance to wheat diseases.;citation_pages=e38008-e38008;citation_volume=7;citation_year=2012;citation_issue=6;citation_pmid=22679489;citation_doi=10.1371/journal.pone.0038008" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=J. A. Sved;citation_title=Linkage disequilibrium and homozygosity of chromosome segments in finite populations.;citation_pages=125-141;citation_volume=2;citation_year=1971;citation_issue=2;citation_pmid=5170716;citation_doi=10.1016/0040-5809(71)90011-6" />
<meta name="citation_reference" content="citation_journal_title=Veg. Hist. Archeobot.;citation_author=M. Tengberg;citation_title=Crop husbandry at Miri Qalat, Makran, SW Pakistan (4000–2000 B.C.);citation_pages=3-12;citation_volume=8;citation_year=1999;citation_doi=10.1007/BF02042836" />
<meta name="citation_reference" content="citation_title=Curr. Protoc. Hum. Genet.;citation_title=Quality control procedures for genome wide association studies.;citation_author=S. Turner;citation_author=L. L. Armstrong;citation_author=Y. Bradford;citation_author=C. S. Carlson;citation_author=D. C. Crawford;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=D. Van Inghelandt;citation_author=A. E. Melchinger;citation_author=J. P. Martinant;citation_author=B. Stich;citation_title=Genome-wide association mapping of flowering time and northern corn leaf blight (Setosphaeria turcica) resistance in a vast commercial maize germplasm set.;citation_pages=56-56;citation_volume=12;citation_year=2012;citation_pmid=22545925;citation_doi=10.1186/1471-2229-12-56" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. D. Vazquez;citation_author=C. J. Peterson;citation_author=O. Riera-Lizarazu;citation_author=X. M. Chen;citation_author=A. Heesacker;citation_title=Genetic analysis of adult plant, quantitative resistance to stripe rust in wheat cultivar 'Stephens' in multi-environment trials.;citation_pages=1-11;citation_volume=124;citation_year=2012;citation_issue=1;citation_pmid=21912857;citation_doi=10.1007/s00122-011-1681-x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. G. Wallace;citation_author=S. J. Larsson;citation_author=E. S. Buckler;citation_title=Entering the second century of maize quantitative genetics.;citation_pages=30-38;citation_volume=112;citation_year=2014;citation_pmid=23462502;citation_doi=10.1038/hdy.2013.6" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=A. M. Wan;citation_author=X. M. Chen;citation_title=Virulence characterization of Puccinia striiformis f. sp. tritici using a new set of Yr single-gene line differentials in the United States in 2010.;citation_pages=1534-1542;citation_volume=98;citation_year=2014;citation_doi=10.1094/PDIS-01-14-0071-RE" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=J. R. Wang;citation_author=M. C. Luo;citation_author=Z. X. Chen;citation_author=F. M. You;citation_author=Y. M. Wei;citation_title=Aegilops tauschii single nucleotide polymorphisms shed light on the origins of wheat D-genome genetic diversity and pinpoint the geographic origin of hexaploid wheat.;citation_pages=925-937;citation_volume=198;citation_year=2013;citation_pmid=23374069;citation_doi=10.1111/nph.12164" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. H. Wang;citation_author=N. Jiang;citation_author=T. Y. Jia;citation_author=L. Leach;citation_author=J. Cockram;citation_title=Genome-wide association mapping of agronomic and morphologic traits in highly structured populations of barley cultivars.;citation_pages=233-246;citation_volume=124;citation_year=2012;citation_issue=2;citation_pmid=21915710;citation_doi=10.1007/s00122-011-1697-2" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=S. Wang;citation_author=D. Wong;citation_author=K. Forrest;citation_author=A. Allen;citation_author=S. Chao;citation_title=Characterization of polyploid wheat genomic diversity using a high-density 90,000 SNP array.;citation_pages=787-796;citation_volume=12;citation_year=2014;citation_pmid=24646323;citation_doi=10.1111/pbi.12183" />
<meta name="citation_reference" content="citation_title=Genetic Data Analysis II;citation_author=B. S. Weir;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=J. Yao;citation_author=L. X. Wang;citation_author=L. H. Liu;citation_author=C. P. Zhao;citation_author=Y. L. Zheng;citation_title=Association mapping of agronomic traits on chromosome 2A of wheat.;citation_pages=67-75;citation_volume=137;citation_year=2009;citation_pmid=19160058;citation_doi=10.1007/s10709-009-9351-5" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. M. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. V. Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=G. Zhang;citation_author=R. Karns;citation_author=G. Sun;citation_author=S. R. Indugula;citation_author=H. Cheng;citation_title=Finding missing heritability in less significant Loci and allelic heterogeneity: genetic variation in human height.;citation_pages=e51211;citation_volume=7;citation_year=2012;citation_pmid=23251454;citation_doi=10.1371/journal.pone.0051211" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Z. W. Zhang;citation_author=E. Ersoz;citation_author=C. Q. Lai;citation_author=R. J. Todhunter;citation_author=H. K. Tiwari;citation_title=Mixed linear model approach adapted for genome-wide association studies.;citation_pages=355-360;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208535;citation_doi=10.1038/ng.546" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=K. Zhao;citation_author=C. W. Tung;citation_author=G. C. Eizenga;citation_author=M. H. Wright;citation_author=M. L. Ali;citation_title=Genome-wide association mapping reveals a rich genetic architecture of complex traits in Oryza sativa.;citation_pages=467-467;citation_volume=2;citation_year=2011;citation_pmid=21915109;citation_doi=10.1038/ncomms1467" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=O. Zuk;citation_author=S. F. Schaffner;citation_author=K. Samocha;citation_author=R. Do;citation_author=E. Hechter;citation_title=Searching for missing heritability: designing rare variant association studies.;citation_volume=111;citation_year=2014;citation_pmid=24443550;citation_doi=10.1073/pnas.1322563111" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Genome-Wide Association Study of Resistance to Stripe Rust (Puccinia striiformis f. sp. tritici) in a Worldwide Collection of Hexaploid Spring Wheat (Triticum aestivum L.)" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3.cover-source.jpg" />
<meta name="twitter:description" content="New races of Puccinia striiformis f. sp. tritici ( Pst ), the causal pathogen of wheat stripe rust, show high virulence to previously deployed resistance genes and are responsible for large yield losses worldwide. To identify new sources of resistance we performed a genome-wide association study (GWAS) using a worldwide collection of 1000 spring wheat accessions. Adult plants were evaluated under field conditions in six environments in the western United States, and seedlings were tested with four Pst races. A single-nucleotide polymorphism (SNP) Infinium 9K-assay provided 4585 SNPs suitable for GWAS. High correlations among environments and high heritabilities were observed for stripe rust infection type and severity. Greater levels of Pst resistance were observed in a subpopulation from Southern Asia than in other groups. GWAS identified 97 loci that were significant for at least three environments, including 10 with an experiment-wise adjusted Bonferroni probability &lt; 0.10. These 10 quantitative trait loci (QTL) explained 15% of the phenotypic variation in infection type, a percentage that increased to 45% when all QTL were considered. Three of these 10 QTL were mapped far from previously identified Pst resistance genes and QTL, and likely represent new resistance loci. The other seven QTL mapped close to known resistance genes and allelism tests will be required to test their relationships. In summary, this study provides an integrated view of stripe rust resistance resources in spring wheat and identifies new resistance loci that will be useful to diversify the current set of resistance genes deployed to control this devastating disease." />
<meta name="og-title" property="og:title" content="A Genome-Wide Association Study of Resistance to Stripe Rust (Puccinia striiformis f. sp. tritici) in a Worldwide Collection of Hexaploid Spring Wheat (Triticum aestivum L.)" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/3/449" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="New races of Puccinia striiformis f. sp. tritici ( Pst ), the causal pathogen of wheat stripe rust, show high virulence to previously deployed resistance genes and are responsible for large yield losses worldwide. To identify new sources of resistance we performed a genome-wide association study (GWAS) using a worldwide collection of 1000 spring wheat accessions. Adult plants were evaluated under field conditions in six environments in the western United States, and seedlings were tested with four Pst races. A single-nucleotide polymorphism (SNP) Infinium 9K-assay provided 4585 SNPs suitable for GWAS. High correlations among environments and high heritabilities were observed for stripe rust infection type and severity. Greater levels of Pst resistance were observed in a subpopulation from Southern Asia than in other groups. GWAS identified 97 loci that were significant for at least three environments, including 10 with an experiment-wise adjusted Bonferroni probability &lt; 0.10. These 10 quantitative trait loci (QTL) explained 15% of the phenotypic variation in infection type, a percentage that increased to 45% when all QTL were considered. Three of these 10 QTL were mapped far from previously identified Pst resistance genes and QTL, and likely represent new resistance loci. The other seven QTL mapped close to known resistance genes and allelism tests will be required to test their relationships. In summary, this study provides an integrated view of stripe rust resistance resources in spring wheat and identifies new resistance loci that will be useful to diversify the current set of resistance genes deployed to control this devastating disease." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3.cover-source.jpg" />
    <title>A Genome-Wide Association Study of Resistance to Stripe Rust (Puccinia striiformis f. sp. tritici) in a Worldwide Collection of Hexaploid Spring Wheat (Triticum aestivum L.) | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"paWJySYFcmN_2vS9GMSfqI8Z0AmZm5Z_C_v_eR8fN-Q","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459781","apath":"\/ggg\/5\/3\/449.atom","pisa":"ggg;5\/3\/449","ac":{"\/ggg\/5\/3\/449.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/3\/449.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/3\/449"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/3\/449"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/3\/449":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022SAlfL+I1Qno0ymmJ+dh0VyKvRg+6Z6\\\\\\\/zvIezTUhcdHgSb7m2XX\\\\\\\/sJWjHp\\\\\\\/64iJAiM5sL5JGSoB9Wms76x9FW0MscEW8nDbotLgfzHwt53ifyS+T9RHL\\\\\\\/odP0zal64P\\\\\\\/GJKJ5ZDHfVE72ob8dynCIZq5QRJG1NakPOvJz+DlzcGnHJ0MvvAGVHLbBRWTnr1mSjZ9B49mZmJ+zh82BaJ7kmSr9OFp9u87qG3bTm3Z9BPN2IUmY\\\\\\\/QLZYyL\\\\\\\/1GPdmPUYx6tjz7laXg1Phh3wiYWREPTzGyS\\\\\\\/4OjfpTTABYrfNimvShkOqAJo4iPrEiWm4Bifuj8NwO4Vm038HsviAOuBYJad50AYxfj+dd6jk6hcMu+TeIEbpLBE6GuSmOK7bsQiT4FcO0vwEzO8F3xJlSaevp+OLf2+FEd5xLdcT5GqHPEFfWStcX+bSGs3fqbjSo3BxNaD8ff+ZKZUUAsVohaP7q3LDbnRs63RgY8gDuI1KvZWfLEdXfw6I7Fy2XBf4TSvI2nDsaiOKs6LsBJpCsIw2Bm3E2IVvq5N4aifQhpPnG5k2NM73CkqLCk\\\\\\\/c3D+mlBqJgXueO8pnwgg2HHeO1bz9J\\\\\\\/YObg9puw0yus61I\\\\\\\/J464a4URZR581rwIbZjUWH3jSjTVgmmufL5t\\\\\\\/kTahHKi\\\\\\\/jlCyUpReGsgGue8EzwqXj86+buw3uECY7Pz\\\\\\\/StHOMbzJMd2\\\\\\\/JgNPT13qZ3bNiEtlboht1SF0L7Pui176lAvQ0JMd1G+qHbIi3FTjZgaI13dQJLi68uFbyCEVi3ggNbQYaSC89vVlwtavvYRmG0qNSl40JU\\\\\\\/86+phl6tyRllQq9NmB8\\\\\\\/OmpctxxPsF0C5iB1qYIHkl9PHgM7LOVkSWXmf3tdrlwFxRKQaQBhvJrsQTLfg3Tc6knXpn5yk3++KLtvYhNWaaLWBJoANjoSPUFWBr7Nv\\\\\\\/Ugr01hqqmr\\\\\\\/51jfWw4ED4ihzHXuRQGPYRhLIo0Zre7a8RAVvxrQN1ANlKxSq1DdLpq4Qq16wfOJdn5R4pkpzTNPPIinAmjHGO3iiVpdgw3GaQdXUpH89ImoWd6JUT1l4nzrBjeNLeaT9vY4xRJsrd0jAKHcs1QpZ12VJr87VBrAr9i8SOgDJBR6O\\\\\\\/tqW5M3Z4DGPA51\\\\\\\/MdwZhVH6jIwXsUqR8BvMY58SuTGnZAmZkEhzK7E4eYRl9dn5nKOGColkWGixM\\\\\\\/HxpL3KhcUw\\\\\\\/NR\\\\\\\/BoR61ddWMIiQifuE8gT95sKLa1++HRFauTz3EAzx8Eg56cPtkq6RR0Q9htNmQ96\\\u0022,\\\u0022iv\\\u0022:\\\u0022iPKesxsnNXu36Lncro6LPg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022649a38c499d6e85b7656e4d57a1723029974947cf5bca23fd87830ae2a72e939\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022U50uQ9LR4EHCjQt\\\\\\\/+pbWHo9KBqU5PSEOZBSG9vGn+H2QpwOwEWy5Jux5v52aTm5vHpG8f5qD0G31x4JhwHCaWRcf\\\\\\\/JjUvCIWVkUlpE4GmdVB7xxor2+4mp2RazIH0rYYVV0nWbU9wkaHXgWvlyjl5QRwc1Yq50INRAxf3JLQmK4eF6Ty\\\\\\\/de5V\\\\\\\/PfaSNnqByGmvAKeO2G5GJIjPSclCf8y2anGHLZN8apZXGWh\\\\\\\/dFDiVBCx9tL2HkqxVGEGZ6Fnw19pJjAjfk3AHPsNsROm3Eq3Idb7PPYiHxKzsRVFHoqX6fGLSQO0J+CuYcOquLj\\\\\\\/cJJKMmAqiSqJG5omvpb5TziYCozhWmTaLJVFbTXw3+y03tK1z0FwPSeYnxa7DSXyDFBg9Uq2rUgjxjI1akI3yigUrGkSqLnsbV96+ljDW3cikk0426otLeRZPiyqdof8dI9ZIcfFpNHJImRvxglmncoM7CC44vvLiZ5\\\\\\\/BpvtDMIF9gGpGEQ8s5zH6SKW7sht+hm2HUYeqDwZppDpY98i4y\\\\\\\/ONnfM8sVEDDTZ53j8oMgW4OvZPe51ELOR58EE8TkJ3ug0PTr9MI6b741g1B\\\\\\\/qVBYc5BnIts+hQwxIRGUm\\\\\\\/TLVxDohYBG4aX+WgbTYw3lZFuAPghrGk7s9x2yVcroMV\\\\\\\/AdMtM8C6Y57jZhQmAKenXYSFOcMHj+DsF80qPgLKp9NLnGyATkbZir7fZGBNk8yNt2KsBN6S+E\\\\\\\/v6BuZvXxcHpyOCsgjN6+NkU3fbtZ0KONSSoprmB8pCRt7WkY47\\\\\\\/hBp69ns3gKlqH4Ch\\\\\\\/l7yElfZ3n1dtgb1qrnxJJNSAp3yxrB\\\\\\\/AwND+6EYHVO8C29ku6PcyGz\\\\\\\/+628NiA3XAM7LoSMMia\\\\\\\/7MJN0KgtITT5dFk1EFPNUtCBy17dvbbb2sEfJ\\\\\\\/75E0iGuC\\\\\\\/gBw5jYTpPWjCnPdGJVyARKnI5pBgf39aY6TFJB7IITM6MAWEGz6yvBxu+fHXOucquTDshu91EBdXsjzkJpBfcDGkzluubK6CRjeQklJncBSB\\\\\\\/HteIhGFbtx0bg8Zl7Km9HYvQpZNaqEhHgSOuPwEN3+3dUJxAxpSpde844gOMIraGclAukGAJalK\\\\\\\/wggNgbqVYY8IDwErhtx8yMPJLCvCOAQAChEDby\\\\\\\/H\\\\\\\/ex4vOOzFRp9KmxGyvd2YTBLI\\\\\\\/DEU290qUgG07gCuLDlyAjRetR6Q69qI+6cWVShg69UaGRGr+e7UJE0QNXG5ySoYhJC+9u8qyr8m+avZaiLsniFghdLCRjo7k3fJJoitNsF\\\\\\\/\\\\\\\/fcRY9U+3re5HRBcjmxzda7oU+rOvxKBeQci93mndpMSyECmZGDJi6YcR5mi48bWhvkNMvi1+ctnmpuz2N9hAlCVJmxwPTtJ8oHUQWtHzezGlTP1ubSbK2oCMEQx9zMatyc4BpKOx+pWqzBC2\\\\\\\/6pQc35BASdoHmz1mO99nRUqCsnvNj\\\\\\\/H4TaH6ADJRR+3GUQu6LMNsT7DhGe4ZM7X4Hry\\\\\\\/CEoN9wkBh9iYvADGK2M34Nqd+y\\\\\\\/9A+F4xVFlFbofnQ0PTK+9+cCuasMAuRor+3ZtLjkxvun0O\\\\\\\/UuHFvgrX+AdAMOk7jowqpwnSVJAoz3jIL99IImxpqLFmrl58PPB2TOrzT9qMTsUDqn9hWApIK7j7SefxII53y\\\\\\\/ilCCqFHiPp03zMCOlwzLk\\\\\\\/Qc4LRy3yF8ZmnFdJsQAmIgnCI2xg4kh0EFhF15\\\\\\\/wG+FyP5DI4aDy25fBwGOT4TAeRk8Z\\\\\\\/Ei+WT+Uz0XZqnFT3tk0JapNT86iUqdpapWz9OHzpNWhkwMu+570+IvMb5BE2MYhDX8Nr6DjqsZu9REf6Jx69tVt16I9S+FlpzkqsoauJTeOL9ELMg0zBxXYkgtLYUJJpnrqOfujH+JvXc8fLmucLiKQN2pgYKVn1C42Y51RcyCkgkUT3FZj0qEN+AjmbdC3FrCd6sDggx34Drfegy3zifRrgvvQ23eO0UbFQ6eLavk5hT1CNPr8HmB8zN+spU\\\\\\\/rei+QONM03CYTCGHR73jecNs2B7af2itzXzjtBREaHIwVicxxTSOV5Wizgso8SkhsGmM4eFwwDLOXHlPmDAhnDkyTqFuQS4UG5HUK5X7kBqYlz5WAIcD3TT0dmvx3oSVDzVlN5zzOkE1f4eNR22lnOXTaiA3SuAoYEMPjrocqxDNhwtJMOUpGknNd6ybt8DmwyfIZvAWLFKt3NcOmm597TbB590NhhzcpLlm680TWVX3nRbH6NXZ73o3JN8v+bsChRSB3UnLucUDQ2I+t\\\\\\\/mSJ3vPmObSRANYQas6XdtU8gVXpI9tLvD6iJDRlbLbAnRNJoRrNscThWkA0P7TNwpJt88o7ZYyIdmAcff2umTQki8FAtXCRnqMpQ05xMCiqKIBZNo4IydtLz\\\\\\\/mXSV4uj84Js0sWM1146hr7QhmkDMCwP7XW2p\\\\\\\/GcEnLkRJmSa6fMwHS0hl6NbSE73CcKJDj34EIa28Zm+fdScU1ip7ct4qneDMEibzaftyaL3aladmzutcNROxLuNpvifTw9GH1fMHXMqhD7LvdHQLeOrFkVJY0cYdwZ\\\\\\\/ISdVY8yr6zbSjKXD2UtkomJ8jJ66rmXpzOSWNUaJkotCFIKjdOIS7giD9umBuGHw2ObFkB79uYZvWJquQvaizS5T1M0bCUCZEAxEbANOOkb69jUWw18j9dWbYotkDMqDaj+t1JhgftTcZdQMC8lkgdnEMMk3A5MBfHsj2g9ynZx93fqUMExlS4UG371BpB9TmflMhrPOk34mdCoiEc1kVTs1B3LpBDO4s17mNezOOU7nVnmAfcRYYSB+pu5cwdnTyssrZIQHnpevuoVJ3mIavAHVDJdPsTK0yVJIxwxtPxZvi56TOLRmUTc2gJQiBJ5X7TNBt5YwiTTc3xHEjZDkP0BMleYmPuPcvRDH5zeu+xxRIYAnSkdPnbJMswnpA3QVmPsLt4Ab5zXPM7y43vgTVmRV2wq7pEMVhii6eEnwPBdQjR3LNGltSeZ\\\\\\\/p++7oZXGLLprwUHsF+z9BVlJmUMQeI2quF9NuD9lnUe+QTyzCYLdSg7R8WaNoLls4STXsFDsPNpogrW3f4vj3Rfm8AlAQhJtTZrkNh\\\\\\\/VYKVe6DiigCcHgUBAH2NEzpWOTasMQIxniX1sHa4jCAYCQQTLOiDBOBbFJ7CxeypdkxEnpv1aizgexsoD52nW+xbQA2UpkWQfbFAJWXwCKt8ma502KfMH7mMLmJEiTAZIhGr3gOY1h2NKeTEwQwBhmXq6kYoPESsBIKS550LcOLWd0twRAl0Ll7aMG5\\\\\\\/hc+T9uNIZAhA3VITdMj43SK+qLOewuTBy06z+HE0so3AJgOPVb\\\\\\\/hwWJeiQoRnhAFRut7EZZ\\\\\\\/lGgnd6P0559rKom51cWGK7tOCgMxb0ZL+IGCV73\\\\\\\/3Fk9CTcChGgLhLfr8d4OZzLxPSiA+fgnTtJ1pwGb0q91MtaI+ayGMQANVAq08vLcC1iG1zij+q4bsyrqzqp4Ebje6TV+2\\\\\\\/I6anagDlbpaRtbo6e6QbtYNiYODpk2E6Vegalc33qs\\\\\\\/jAShHsAaxhq34jO3l9nV5GH7RGfPcVDO3kyLMiqulYZVUq38+NJKJsM\\\\\\\/4hdHrRB8iix97Tct0GbGwHYOuXxd0++ctsc\\\\\\\/kTuq5nulmwBQT5AA7NVjrG7ZGbkPc05E3egFivOOAfxBqN\\\\\\\/D0\\\\\\\/E4NkaPMcEHUvzEjxhqJCZADtjFenbo3DfGRs2iQovDwSwuULMgbZBQs9cn80eHCbF4EKR5j+0oeF2YJMJ94vPg+iEbhunI7NPIQ+UGisvl1wQY8z2mcGeimYZacdUG78P9xQV6cU8sXcnICNaoY0cLEq+MEdsqMrriNS5RjzlA01y0whKJmBgTDS3A\\\\\\\/RBfEIQahrf5r45BmftEEcemksqWEe\\\\\\\/RZgcjiwoW3oSWEWF\\\\\\\/6WA+Wj2KK\\\\\\\/tvhacPTVVaajdrWdn5zp1ucOA7AtQ7mKHLSN0seqdPdl7mjzykm8UGCyHjVSc7g\\\\\\\/s7AhPoliOgBCbSx7uRqDyDzFlGgSRKlqISB2Q93lGfbbaJc2LhFXzdBdp8lJVg1o7dPl18gWTHKtiSHy9LyiszRrndfeekFKf3Evq7AqbsqbkBpt\\\\\\\/qkiCVBUVdUXBS58cIUOOUcckFuo3GwIXB2qcdhrpO\\\\\\\/Q6L42B2wxeIsDIXYTMxky0PmusMMz+MkGB2E0TMNeeHUzeAm+VD8hZ1Jrd0DAX0z4ijSYvjtKTruysr0UgdTUmMNhYLj1rB9q9\\\\\\\/OcQIqY1ic6XSxJKR6YOEMYlgyFr3TZMEbO\\\\\\\/zIrOJcSexoWesqBnW39AkCUlZS9\\\\\\\/pqsdhBhVsy7dDVyFM\\\\\\\/bBv19lFDHZukaT5+qkEBn6MT6KykACWsLYLe80i\\\\\\\/SNa7MFEdmonn+OTVRDNF1D\\\\\\\/bFGMIy\\\\\\\/PZBNLXT+g\\\\\\\/7sqO904qYAcUR1aMV0PK60SqTp6fAy5xZ2dQabYphUR9xtOmy4ma1vr6iKIOi5Cvmn1Eiyy\\\\\\\/y+cnjWyTNpLEL7nJ\\\\\\\/\\\\\\\/oNaQnjkvGi8KHKJJjGPYqfJlVwQzdfuhSmWLrFzKTC3JLshjgTxFldjc+hZ9gcpw\\\\\\\/HGhlibzRAtfr8KiWXzYHy\\\\\\\/wFstZfjwsI3gYzuVyqBwoWoXrui4nfVWkvZ76IPR3dUgETYrDfMcrAo8ugmduUXd+qnC66GicqoPHpXiCogIExVFg8\\\\\\\/PrGCpse+NGwLLQTU8or4o0U\\\\\\\/b1y5b8SpZD0bGY1RV6QoImrRnHOJcFM0ziJYFhiS1bUZvQj4rlVpHGMrNLGwvh6\\\\\\\/TIz7tUHofQX5oepOAxkkB6sqFGHPClL6XWO9cULsAPKwJCiHcPSFnV1MSJeL20VYg0GOL3rAievHOg0Et8AEEr78rm5UOOneVLEpaZ\\\\\\\/LgWIGJ3jf4UR42GCIjHrrEDpTWQlQ+v0RppA+9ejlJgNHAlVa9skMGAHBuoOaFyzaPNzQdUMGp4aoSAq1U9F\\\\\\\/41TJb7mAUZZ6y3bv0HM+zV+P6n2u0ADomtHwU+Dxd\\\\\\\/jenrM7lCv48X4nSDUMZtcGUt72p4YIc5o5+TMGwH8wC8Ekor6kpKSaA\\\\\\\/DW9a2H8ja6njZPTtMgnKX3mmajXQslAXvDBfiXO2JFqsQjG6tAqHxLW5HdvAoqZ3PcbSQ0xNDz74xky+CRREuMbn07LZ5a4uJmNMsAfRdDAOLVHTUC2Jb1BAiBkBgsiyrNuLc\\\\\\\/2AeSRsbky9rSXoBMG18z4U4lFhoYwU90gxuh6qfM49l+mHZT4\\\\\\\/m0yA6yrJYet7tsSA0pvwS3Y+2gZynTiZji6tDo5Ot5vDQimW0Jxqa81RGFXFgE9j1gbDEwl7S98NrcwXX8K1xWm57oljgZKLZX5ebajGDDPtzJYbylYn0UtpWYJ4u7YK8q7vVmzsVaC\\\\\\\/4XLqJKaywoHPKCv8Iw4JnTKlzfjxmDpBxC0EWZRjiUrsJ22pPsDwTE2DQ+rfaohH6HvC+cVhjjbMfQ+8+jQQe8Y7cfLgYATkOKbwB3aXcy3OPWk6mEWt1FaacIZH4cG4UGY2DHQSkp\\\\\\\/bVQmzX316RQLybM14CoYhdmwzz3RQG7NG7+tAVBGjY5DVehSerct3O7QGpWM0hupVZIzZdxFgGu0NzaTxadvX9p2uQQCWGn76b+vY\\\\\\\/O3uenadXYACgj+pHfQvbul87a3abSXQL5BgCsZt+gcE6SkW2BOxPQXUUQ88PBQytL72VRLMAh6Z8xMiMbpoaF63ZPwUWrdc2TGLd0YaaYo9pd6MNJe1ueScyK37ZqAv5uIoNAyabwVgqo2MrvKNR5wjboibtH03z8LQ7BVXdHG8cGroPnc3l\\\\\\\/wM2kzOmoQ2b02zHhKhUm9haXAgD8dnGuuQTJ7elyvs9lpAG9AN1c1WpxkZ6cTJhPD6UC+C1opaj7giX+20g4+YClbuHk4QQDrnSWJPGDCaPiyNveGR9ykIfkqsixsuwz+l1889l1RUo8mj1xDGtxvMW3pTIozD6IwkxeyPKZ101T04x3UjUZa6ZYr6FdT5f7kxpUonD5asrmx94yN3SjHb8P5cICgt4jweh6usBd2H\\\\\\\/ugATu9cMr4oMDg0qcXu\\\\\\\/J\\\\\\\/XMYDiszyEVgwkbEhye0hCkQ2HTGVbpCbaRbLWFo9cSiW9kcxkxQkafrNISbdFZFbM\\\\\\\/yynG2w1B8A10tnv503ndnGPIUjmbv\\\\\\\/UW1FSy1S9WeirLpd2ST5MMHc7mRffc0TMkFGcbCaflPxzc4XvfUqZTTZ2NWBihqvo1Y4\\\\\\\/+lq5FpA12tFl5+9m8YcEgcoG\\\\\\\/6166Mn0HrK1DwIXKDGtrF3XZ\\\\\\\/ldSezTcU71WjEOKP5LVDCwZAWc1BxU9kSAe7rMKspAS3CJ2epWNyLhCirivFb9kFy5mIRQdLxoA5I0Bu\\\\\\\/bO49Roc4C6k1dn+8CAmGUDaVZi9vBg5+cSqAanDrFk1Hs6X24u0+2WX0WpXp5BQ9\\\\\\\/K2isOxj2GKdOIHD1jzRAOcSS4m1+vrdnj5U5yy4HhDhT6N7d2lEAvS78vvvTor1hQW9CJkvX\\\\\\\/fzzW1QFwojbCzedbW1tWDSIArBcJloJit6yLgYRut3JOx4kIwUrNCXS9LaM1ajdX3oxmsasl1jHd0Yxuq9jzQWtakXCAWA6FkHPI87LTvc7fhCw8D\\\\\\\/PF0DeaOTi9s4i21eNVpuQdFkGHwWWVNrs322Ksz4opd6zkxTHcpIBRYCbo6OWJWlIWgauGmVOLcoV9+M9Nz1szkfYI0I9E+1nTiwD2iM8VgJR5HBr4OMqULQ59Z3ssoufW97TbtnuVhuJdvq10u3yAlaVxbLUoTSpKRXVtBmy22JtqGVVt+Sg9OE2d1wptggRQzeREkdhIAzWsUpMWECc+Jr\\\\\\\/PPFWWpHFdmP0hYN8aWO8OlxCELYP07CJy+KrWGyyYOc1njltNs8IanmvmjX2U9+B6DcR+5RhCHqdWNsl4iJOgpxHmzSLWOQrTbw4eHmpTlBJNDEDkAg\\\\\\\/BAqpLbP\\\\\\\/s5hCHp5\\\\\\\/w3yd3mDxcXYvSdDSpmoiu6wqCyxmFiWrD8HphJx\\\\\\\/N6SX0RxUim6sCcp1yDB\\\\\\\/tedqLTbfyy9FJ3VmG4CZzlbR46g3KDXCNxg1fIc\\\\\\\/DqYtIsVQaQYGb\\\\\\\/ECHitnhIahHW54DyZFRU3Ehxj\\\\\\\/pCtvR2tC0nZLuNes0HteTEEYHH2AJlUEJkHdovjjtWYUTexbJfOBHjgRpT6deMghKdakHC18dzvk5zljO3hOmnVNNeGaiOo7nW7UvB5dHRMwNr3Ul2AiXTA30Fj8ng7rcy1PDe6hEV+L0wt0Oyy2Holb9QWJQVacO86MlyhaZI+9tIrrN19Pg4Xo2s3M4HK5NhZ3VphBTA\\\\\\\/YWz\\\\\\\/Cld5OFmRth0DziJ16oJm75L7biFKXmFYtsdiTVkVG8LMjByLxgtMBelw9n3riX+UusOdlKzWIJbmlXNZzuy0zHiycfDkK7uMjqTbpz2UFi1R5XwDVeNeg74S1Y+THXFlF09Nyr6FdmCK+P3LSBguGBQne2poDATsNxRo20lWCAJUOwbhWeW8\\\\\\\/KQzotHwaEaqiogZkwlvsUb5yBwqQf7LqdWPJuXQrSm5fGpLlEqwQWiQg4wm2jO6nagDSOjbTTcaJuWMhxpdZqFoMktJIEynvjKBN5yKMcrA+ruxbLyNTkzRfrHMtq7FnfyP8Ig+H0y4SYFQhoVYMVMRHeLDc8emzKdmYZs2AXaMzzd6JPq7RWUIpyNmmBHZ1wVq00yjo0s6c+5FJ21iEYsJiWiWxAud+FGZaBUz28xM\\\\\\\/xbqflS0lp5jBeCqyBnSC+lUQid2Im7axFkx9txDowCO0VU1gvB8ZL8+yJBTc6MnWNopjLVcaaIxgUewmvqieGS1NpbkiCxAyuE4+ExKnYC54OWZcpXn+gtqZgAtLgmDDVFArTnrWiPc2Yp12Trj3lmHNq8IX2kXCHH1Jny6aUqtNbKwAvespjhsJLhzWLbfAik69bbQD8SZxhK2Oh7T\\\\\\\/DPa2xn+O4TN4\\\\\\\/J4Krv54+MfDmjgzaz3o6j\\\\\\\/ineFi9Asqd7Au3GN6OvBcuTDMZ\\\\\\\/3n4j2yr3qMigZ0wj9H0QUM5LyVovDnDTtvyXw6\\\\\\\/p8oAtg5gvjPbdnCZK8m81H3EUbRcIZyxpEat7sAm7V6XOgWWLLtsgCw+yi2AksrXMRMcoBg7caljwnyQiyX1EwNXe83mqPho7iUlN+WfLazntBe+W9GDYzmALcuKarCIAmDngNE\\\\\\\/KoPXv42b8j5yVhIAGDY7In5Siada7EzrIZal2xfXk\\\\\\\/LjLWCQAReNaMKe+UOaNRWOhfygNBG1jekytQNKJVcBa2qhGgJZV0dz1BGhoz7ITQUFw5k7pth7KCKamZxgY4jS\\\\\\\/kzHd56n6sSt1sD127KR\\\\\\\/AfFxksKogicxKK0rgqzlUz+SU3T8ICL0HPjClzusaCpLx6WQjomlcyUdSwTBxImogCe7AAKZmgg0TjAwz9q8J9rsH3x4yfBsHT7raI2lzhLdK4Xy4OeoCpeGAWyfWGvxdv9HRKHOHW0eyZBIOiEmDktsrDtI8EfDs+cgFMDsa4anruQc97suhDCvDSrf4iFv64rSQDmAMx16EQe1Sy2AJRpYV3BMtBNinJFjnVHaQPKgCbJaBlfUgzekZuWngViVMZDiY7fsYbGLSwpLtP9apCPgRHDwd93r6yaxdD4h8Z02UBo66lxgMcSDyYgG4Vrhgmsod6dtEckeRuKLe713kOlrFBNIgZsWh\\\\\\\/VgBf9h7MMHgxOEbKrz\\\\\\\/ESisFxhaDYS\\\\\\\/HRQFQHjtcd5tG5c3LZV5\\\\\\\/HKiZRdwkPkyP1BY+zNK8nyQmJqwwT+Wf+wyOJomsahe1U43YgT8522aqjf66+aonB89TmhcLk1sEtOGD11r+2DOkPNpwdMPZIHo2PXhF94rhbuSHvjNjL4T7DRkK+8nhV2mfBgGqJNvpXxyPdPqeVudXtLOj4U8tGnvAHZHDS4\\\\\\\/SEsrIJ2aKHkFqGO4WQ+rtZNOyXXDUtuatHHcp9d7Lkpdpt6TSnua846mYIruCXFQ68dJXZI7LAHD46c7\\\\\\\/1aMo+yNdpfRHp8vOMy88AqxtXiy3PmbnI4wrfTb+\\\\\\\/Ng9siXVL1E5sE64e4M3loXjt4pQ+51S2phBLZBfaMcFLbEdjtlB8szG3Xng3mq+kXkxJv03sN4dNuOmsjlRstmsHdvHjCtzZUjcd1Cse7\\\\\\\/d\\\\\\\/bpXwxh8v1RF0KYMw76aqdgnG850Z9KodOHQUa+7pmTXR2A4v\\\\\\\/zmrc0kqIe7AzA+RRb7YH6A1TOPsh+aM+wdAt9icwt4buupmUQjktGjxc99\\\\\\\/9u+NmruTE0FBQkrlERL5op7v3j50PajQp\\\\\\\/miwPH9cdNlSwqi3QYBLxEH1r+Bdb6M8\\\\\\\/S1NjC6PXVC6Hivy6HKr5ITGfqc1bdNReFTmhyaOlCK8epcRn8uh5Vvmu6zH2mQvim\\\\\\\/CEWd6FBm4zOQ9qJf8efcZeEi79XotI+YqqrGRBbRY6TzjUn9vq8xrKaNK1TAnQr6nY51SdMEJV4kOXo86koEGwFAe+F6cJaFVgPPiR3VETEGY7fpJy+GYYYYWIS6uHhT2bDkre9uiKdUBp9807FeH7djmu5SqI9Uk2JzmMKyM1Hevn3\\\\\\\/XmpN\\\\\\\/86S49lYPN38\\\\\\\/5rzP6wh2wcrS2tIoPtwMMTUoanr3EnA8\\\\\\\/7L7484xoka2Zm7XJh+c+6dHyYcIF17MV8s35w54cXuWiUHqDazblfVKvmXGGzo3OhQKOoSVC9tXzkPG\\\\\\\/iXkcgO7P\\\\\\\/roPTr9Cs5cKo03+4D05nSlrMjH8dDf8LrByfqEm\\\\\\\/9fnex6pYT3ySBEvR\\\\\\\/fBc7ItVKSKITPkGutPj0jMCMsrEXoaVWB863vLBfMFJ4+H2hFE6I\\\\\\\/HJVH+HqgvLqgeKUrtQcgBgUx3JJFpImBnK21QUl8GpkOeS\\\\\\\/4c7LkYsUCGF36weuAkqae0cdoN3cFsXWIHKuQSEnedJjbz1VkgjH+Inq1Gs11ix5ek6eae21HlfSnsmLZzGLl5qhqxi1OMKQIlHZvpMLdJu71OdGS3vZQJCIHqwnb9LKWh0xpWFlBOFe9tfCg5S5Jklx8jpeuMfA5CGxlMcgYANwslIjuF\\\\\\\/XhCEsBiepDI3EOJBw+qFo\\\\\\\/LzbNX78qJTTqC6IUMzR\\\\\\\/5BXl1iqQ+Odzf0HZpO5XwQ4iN9YCZ4CAGp1Rsb49ZxVndrRW0mwBRcYNIWRu6o+r5GbA0Gs+IlWHWrpYGj\\\\\\\/b2cKoMUreCqYBdMT\\\\\\\/jTuH9qhafzDOdcBSaSEv0TmFdSyuJcnHj9C4J3cztAvDmYIMnfwxaKl4KLyAUhGjqZWoPvDKnlkB0uhIR4r4xXZ0v3zepFkzgeW1mvZpdQIXyfYnSa22kmH85cq\\\\\\\/SaoogIv1pnDxqKvQQD6LvTkopqTjucG\\\\\\\/xw1fIMP2NKfGI2J0\\\\\\\/51KmOcc1bMAvZZW2wG+JeSrtpMl0WirO2HQzhL545hMzXXXZSZS1RXukWWQI90rpYc0PmP0Viec8C8Pz+R\\\\\\\/Cx9PvfD7e+AwEET5P+hsNy1D7OOQvqX0PkVH6E0ur0r6bGD7BazQU1od3zCKqdiR5evh74c75KIE7eIQ2sM\\\\\\\/zcvXBLUTpV2r5M2k9cwxSubkg5rsBf70dR0nqf8e0eMbVruujcamKWRjwNUg42IH\\\\\\\/RVvM\\\\\\\/3JjxbhbSb+96oFvVpuobKiaCKODazQgO3E6ytNSZuu1WD3Ps\\\\\\\/SMKF1ZqT\\\\\\\/Auknz6LA640ovfxVGflYCSHhgyRse7ibfbfXjGN\\\\\\\/71Ie2HKUq74joNv1JvT8OponBR5rbt02GE8+Iruk95qxknx7Ht\\\\\\\/naKhXXf8KHhtPdAbXhCUaPfJF0+FNQFDWyM+8RRVT8Q5nloDL7DUJCOJWpkawCwtLMB+12mKNCbJ4onMMThvCd7xpfsHCzBtYzW\\\\\\\/awvE3ickR+n8WnxbCOVy4dUH29ikUAPD4W0PdfXGo07sCfNe\\\\\\\/Mde7aq5qrxXMVNbNETCmGouktkhVsllgFyOAmQpVYalQWlPFTUGupr7Cu1hpEQn\\\\\\\/z5ACEcsUI87iOufRzqJ0bzMIgeGvL1pkEZRHi9++7FUMZIXghxVGE465+IU9H+rDw\\\\\\\/pDAhx4bJWW0oAZAKnH+z8GNP\\\\\\\/QD+Ay4brmBkc3GQRn\\\\\\\/PsdcTMv1wt9MPtr+V36UCZ9XugVaK\\\\\\\/Ms6OEkYfT9lhFDHIoZv+bofSyYLvrbbJzhQrioegVmk\\\\\\\/oTtcZvWAtj2evCtTqwL5eC+0+mlVTM+FM\\\\\\\/BqDq3JMkuYnTymgRrpPgPccyXioQUHXUBjWoWoyssMBD\\\\\\\/sJFt\\\\\\\/w9s+udXly94G\\\\\\\/kELrpafbOjPQiAOpZccOUrnEPiqTZFxtEpmmLwbOPVgWOmeYgI67\\\\\\\/FV+sYESBqIb+QMzc8fy044fjAtXHRIxieKj2493dGZERI9fikUHxbpDWSl3lDfLqKxb\\\\\\\/nAciigX6xgTMmdOfs\\\\\\\/5qDWnh\\\\\\\/cOnd6ncsXOzB9QMuZ2Oe6ciaFeFgPPo7MPBByr\\\\\\\/LZDBsvL7D6By08f2Iq1xPLNKm9oS2+DsfLIrhRqVb153bHP++GZtNUEPGGeah\\\\\\\/Hws2CvJ+kow3GeV0qn+lMUZlfI2\\\\\\\/\\\\\\\/1d8\\\\\\\/u2rz64WvWJUbUsbC5W6\\\\\\\/ureCF0YKBybxbmnDtsYx1UuQXrR43ZluruuY5I\\\\\\\/Zwywe0vvLoyIPaVJ3y8qq+vyYXbFv8up2mB7PetcRVV5FAzVgv\\\\\\\/gxyiDUfOIYYFF28QcYOux79BpGZjMA5tndbjyGiFSuRUyUZUK6bpqyKpdhNDChJAZ\\\\\\\/vokasaZH6jHByumwmT2jnefkB59iCKLrt0F+D12TIOar5leuvoiVp7fAmOr1oCAwaxaeqlG\\\\\\\/XS1\\\\\\\/6qaroGMujrizZeBE\\\\\\\/RlbfiK0KWy+nT5PHuZnKb+0LdHVJvXUkSKaRc951eevSWZNTnORzh5UZtY8ZW8hULBII02sppziid5+2+zvpTCZTuXv\\\\\\\/wTjX+iFag\\\\\\\/bj\\\\\\\/MtG15w3WXHMF9YEkln8v9CvlLvi5FTo+6aMNb46dA9d++IPy9VY0JUt0FxkdiFWKrP9D8I+FmxfOha5oeVRgUl3UPWD8QYTa85VGDohbDcJl8e25nhs5bkPjdoW\\\\\\\/KrCwjzqsdNxxHLaBN2Q7ckh1P8s2ojHjVbx2VpqrPTAfGIdsqRz93htv7emNYGudGCesI8oozaVcsi1YMX73AVQnhJVXszibRVkvQUoxjkRqOQzyOHB2Z5\\\\\\\/6TrEuix3Z3qxZpcPbNCAbxr72rTc2oxJvePnobCCw41T2ixg\\\\\\\/xwzBRQHudHIXEScIGJS1UYPBe7GwCyAMhD4yEpU\\\\\\\/Ksi88fbh19L2kzUazP0ItZYLQiadROfis03EONuZ8TjxBXmIOd2QdfURvx\\\\\\\/qZ50+z4mQ9o6jzxlBuPuh1ECoHKkT9gpzz\\\\\\\/vypWMJlbXV06CTrPDuX4pbhGDaGhsaeZoRZu2seOruQ98XZPzsKrGDYbK4CRzMM+Sl4FL3tppaNvKctfyG47dXydwA9qqtG1pk\\\\\\\/xKFAbZspXsGw8jkyT0J8wpI3A9wSyPLnWIcc5K1SUIHZDoOFLf8dxzHEHQq\\\\\\\/Frl539H9ITpWte2+4TY2NcQBQ2biNxmGoqEPwXHn+8blMDLWf8wUxkmo+BOejSmeUGxqderuE1mQ+dDIsAM7iPxtFCJLojCWs6INxBE\\\\\\\/Ri00e4BTaN7H9jYSf0bwkvdf5UCHWhSAGQNAhEFhaLdqgyGr8Fyuq71WdCRCdmMxu+kw9U4lqHw0vV2J6bmcSBuMfozHhwAp\\\\\\\/0EeoUAeSDwlooYSYwWAHR9rByfsippRsZk47cRp5YGGAONWIURwn4mbUZfm44TY4kb7lh0opBjjJmMg9s1VGRgGd6nMvmk0pxNbr5LUpljPWRdjcE0y8u8WRcyvM5+0ddsiB2I19rkLUc6arS3oRWG1hpq+CXjkq13UCPXqQGyr7W2vq1IicBtxCOmp8NvVH+QjN6VFHBd8sNiz9M3G1IHMyIMffe6EnLwazOQATLc8+UfpL\\\\\\\/ukpXbA\\\\\\\/Ol5vmxVMb1NU2yq6mg13oaXF70i1oc4b+4kiK492J8HgsjKBvx9PihwlRSDjZsveXBxkz9vA2SlRrZjdQqyxTZvj4v5S0lOEKErxGqxyRxmhn030FRnQhi0lRhuyulmeZq2pQ8PJ9oF\\\\\\\/3rEgQOsOqDqOYIxdx4h8eZMxO\\\\\\\/WSGf7UdGUj20SDFSOepnAVYzE6BMJBDyMaT\\\\\\\/NM5BjIqDaM6loRGdI+ArfXkgUF6GlslAs1n690dQ4xvjM+re6U8vSi7aQx1RbdItt9u3ZKJUlh9w950J3DU5olmRplcDd6mbSy2+HVWSdDYYFzXR0EwLFTLQXAI6MCiF6Swh\\\\\\\/NO7HUJxtMmyzhgZtluMwJiGHL7zsEy8m2ID+Fp1ibhcZwZwC1zctr+r492x4Q3GVCVG1fBabJmg0mx9JuNQ6a1xVbdMnV6Qaj2UmPhM4njETEiT1gGc8d\\\\\\\/7F8KNWkC7m5IZZ0TIYfCfjMw+KcqHKV43+LSUM7KVxkFcHtlX5yNnEPPyUt32gIrzO+ZbJBFIRAwuWMs+8C3031qZcBPdiB0OjCo1hToqt3Ya98M2cIXu5NSXF\\\\\\\/lf26rzpgW\\\\\\\/58nxoSWl3LTqXJqA80lLi94x9pX9d7zgNhVJw+59w3lgRHqMXwFqzxnbh\\\\\\\/Z9N27KKm2khQ4n58\\\\\\\/0e5G7rK5iz7woKJ9IXCHTKfo0Sl2f\\\\\\\/BHTzMe90uMWu5p80JNIRV+lVAV5wbofHwgurzIsHN7PcT4D1vyjnUSPAnNsaNcqIxriX14O2Cii2rfAhUmZ3P7E5HyMSzeagIzqogb9q3e6G7rNlvz1tyr+sknQwhMdbzRpMj4lRweV5Vlk76k9ICzGKW4ztsT499SdS+3TCQkgG32Iw0mnsLQ3ULbcDRtgl5WtGG19vnvHU6bM68zAengbQFi3laj3wdvzrm1PytDvcD1Rn+PuHKprmij6Sqcd6AQOXWnRVJNF\\\\\\\/QICslIt+jCtSz\\\\\\\/kjjhBrYpn\\\\\\\/E1F2qvBj+rSSpVJfqjpT0A8LL91uEPIXqy96HBV+JpatEptHSUNPJ6SMwmqvTCzrM53DCpHd8viEFRb0QCFXjFrt7UIKz2q277fgpePnFVMdmzJVJCdcX6q0vblkTYwBktYGMt\\\\\\\/lWEDF0s15B+GGPtWDZFXAR+6GNNOC02McuOOGPcTspZwt8cXkQDGHx2cmIDJz44az3ZvMg8y32eiJS+sP8660mRao5notqN49zZmy65tRqRkGAPXpvHPorA07Eg0PDEcPM2e\\\\\\\/7tMrVObSOZeviKI7YAQXVCdbtVYXD7fyU+sw2FWLmnRSET\\\\\\\/I+T8ID59GyN2Cbz2HCyaRTMzMtyVMWL5ptB0KV0q64CgROnCIIv0Rqxd0\\\\\\\/h60LaRKuqnDsUxjMZTMSXCHrWtudp63uAWFawIOlAaJvH2pbx4TuL62ZEsL0dKStnFjzVrdxsOUMX3MWoUAhHUvNMRSsO6cRTBI61yTDFJh5i8KTR1oCcEuPc\\\\\\\/a4z1S3ABQm3P86wvBGkrAfKFOEj0yC8TgEPmtAoyoUMiF0WZSWHbkbfms+rLniTREYmKpWbKNTQ8ah3haEwxQCJ4m8NtCBogmiE2bwUbyM0zhrnGnVavC+a4WldQWHIrjzI4lPRCj0ShgAWZX+D1E31gxAogjSbRIxENmJSdE\\\\\\\/YaYQEk5RuNrxwwlGYeN5i41aoy6+4p8z5tLbilq3fQ+Alurse0sqRH74sBQUxYRXcJcjuhPvkpGL7\\\\\\\/ymwTpwY+nh+NijEZsy+so9FBBMgmeyvYGIx\\\\\\\/IQk6UQemUrMUg2A\\\\\\\/MTTta\\\\\\\/2p9igmSJWvXv8VT+97h+2zqhqa4bGtdJ3Xu4AmESpLErLYBD+M5pEiAd\\\\\\\/ZL7OBkCfA+V2ilCk\\\\\\\/5wMh9UtX7tW4mpyN9xzpTqV5thi3Az77hYaurFpqy3ADsyNYM3oE6Q0y40NN9WccPoqLlGN2kEDZ\\\\\\\/59PZcPpTNhY7ABI7JBVdIIoYVZWEM6BhIrcBM\\\\\\\/JEATnYZn8AUfAtI3h0M1cJN3hI3pbedTUpparu+yzHWwtipOltKT9oNpa74ldu6SzQG3lueIM4s6hOzJ2p0Am6Szygd4Isj1S8apFIunUjaNQ6Ij1becs0hTShE64ARPqmknAkMCaiS4fFlC9md3vQXj9W1MsZKbsBufIp3sh3H3mP4PTTr9YkVDCzjPPEaFRVHIE5HTyPbMCWyb8HoQv+JJhrGu7M1A39rV5UCEMpSY1mdkl\\\\\\\/WA78KR9+3+UshctigeqH21WSt0OToPSpPRstw0\\\\\\\/abxR9nIB4Imt1vph3czWZYz8\\\\\\\/HWDO1mmK9YbK\\\\\\\/N4Z6vYXp30ZFqEmcsr4W\\\\\\\/bHcqY50mmpxiHCzVuZZSo3eHPhM4uvQQtpicUGL+By5sJNP+ibTVZ6m6umf6GqZw72swF65dYVMWXL6\\\\\\\/BEC81IF4Br9guuAHX49qLmjuYMviNzz8qkVHj5pdETNkUkjMndxBSE7PLE9IfbLSZLo+OAG37FbSa45nv2c1OQeoQPA+J+ValKiCP8YjKWVTakQBvay9cAJw78gFDc25KGdS4Dl\\\\\\\/VWRIk7lgANhS1YbMKnkURRAvKXvXPsg+\\\\\\\/qMCA8mIv\\\\\\\/tX1UTx\\\\\\\/ORks3Wn2KH0etq5uP54\\\\\\\/G20DhplTLKjF0RGrLEsClqPNvuX86C1D3cMUwcgn+MwDHTS6GDD6JAoDl\\\\\\\/AXJ8MJwbGriaYhTlsCM7fQf\\\\\\\/awxeO61kya+ZC5LfrVoxGK4LOwbxwVR1po2Ckt6Ovv9c4DqSgUMl2QFOYsTMDUfrCPJTDhlevlFdL5saXclTG6ElUrUNmBZeQJy\\\\\\\/P7MVn7kEv1ZrlQwA+4rplYcq2hjl9A8ySP38fwBc3iBpUWl+H+Irkb+Nz6P39NArihg7GNDQLSVTKybNMyRcDM1iuvr6762q03oy5uW\\\\\\\/9dM+HO953hHV6ax4KKkvG8EB95to57ocDQjgDghDYjC5yUl9TaSgppk6h8aUWh+cELNKPvAekdDkT\\\\\\\/yzjGZr\\\\\\\/xjHkY3v8n8L7Aom\\\\\\\/87jF6znMNXyl4zmqQT6RWIP7uLV5\\\\\\\/TtFEbj75R9j7zq\\\\\\\/5AyTR1GDCpOqbqLe4fUQNOp540hKep96KxuAMK6KYV+UQGWBpdZbS8wFOnpQIIjGErE8wbsVw2rru+zYh2xZhP5ACSCiTS6Nu5mdsGyWn6Tbf0dcxEb0jeWxSJMtvrkVhnOBJeHt3rTlWbph6OCY5dQ+u6u8s978A8WoivMZ+4zNyjo8ML1L1G86GsRAwy0NDWLw3Vs7jYGrYGSz7uwoo+SVLjbM6gQzsuis0sTSmG5vk45\\\\\\\/GavVuw+US0P2D5tK+OpHCwhqNyOE3QWutaRo7DARw3fhNXPAt5yhjlsz8fFwJE8I42\\\\\\\/Xq\\\\\\\/1JG45mpY+V6J\\\\\\\/yGX+yZbdUsFNqWYndoGdQrsNdXyqsyje+Bc3Cr4AvDPsBwZas9E\\\\\\\/FOrYgpZpMcWfGktyZu8jjR5EIL9GHRNRrtHTzEbWEc7pBzZz34HT3AQAea1Ek2JpcBbfMtW\\\\\\\/NesMyC+dU2vM+Ol+tD+AVIrHz+52GagSVSIQA6EQxMZcDvvA2loGyAFDfkCGHlgsvw6LBYGjTJoc7SfeOT54Ah\\\\\\\/4aF5x9gi6CzQKLOkOOEwGU8RnZyr+kqbbfed7x76y5RhZj26a5tS5iVkZrmrB5YGicGUefvTBEV8yp\\\\\\\/Im46H79kpf+PadF30MVUl+r7LU8\\\\\\\/oyxTMXXxh\\\\\\\/tRgMll7aU+JTWSh5mt5IVOs4XFFatczep7Ypp74\\\\\\\/0V00pvpx5QFDq5oNCPoUGbJYtwyAZIypxwu+VZejUJMqHCQVggoBef3Cjwg7n11V28LlbCPgA6D4+fSNSyv5JffNkqQqOkcaEyCC3v9SKt\\\\\\\/SIEcT6NDd4n1BWzQxIhSwxUlmM2sI3sTBvu4h1sjGV\\\\\\\/xqGGPLgwEysUePxaksv\\\\\\\/OBdRuhKu7xEIVegeI2Bn6qJjGRuOlHkqT+YfLxYN04\\\\\\\/VP+FWToAIAC2w8+lINVE3gGoTYVQCa4SC2JrTQx0qNN0jlCEkO0zUUUgVLNTEWjhG7EbrGi\\\\\\\/uho8zWaBxeET26BOBOJAaPnL3cviUAJ7XHIHx\\\\\\\/48b5s9QounRcd9uQVCjBXj2YdvPaufF4ctJOsiVmQwqz4Fn31H0zyIfbnqK6Wa77XgpFHHRaYLAUC\\\\\\\/wjoaZ1VzZHVmrncUHyNjFPGLZyXO4mKYBitt0RAFDgZdzCNPdis4h8S2ISuC431dUjxngYII2G4AjeesqzNQ\\\\\\\/nJZPS8rj8LVJp2joUnR5yclqmspuHxHB6KpBuLLy\\\\\\\/Atev6DOEHrHHguNEXH4YpcPlp\\\\\\\/qt1K25Qj+lNwIWyXqAWiHgxI2e2I4pObMDWAzD1t7WfganrjOmi07pXu3vOo5lwThlJRuXmJIrihW+kK1u7DcaeeUNWOcZT8QyQMShgnrz0mlRhXNlBAPZwsQadra3NqPW3lcEnP+tLi6fmFzN9yJoU5Tsb3Y4A57FivYieOVtUkA2djfJpmffzdtM+qSxAcxtKTE0pF50nL4ZpxXHK0O\\\\\\\/t6WCYRivDWTglPYob4E0GHK7fb29\\\\\\\/gWA7MVC09F3HvJ0Hnb0gLYs9GPbIBDlSBBCXYFYe8PAMyFN\\\\\\\/WQh4XmXnk89FreFSCCJOfcFYJjAhAi1WZTrQ2P4HtZZlqsQY+In\\\\\\\/sGF2VEF6eG3XEBjU0uVJjuEmmUnrFvfTWsV0aUBoSqBl9vurMcrngLlvmHItIqmFt2LLEiDVn1CimtJLs9j4zdub0p5R+fQXvIdsTsSnitCr6fmWMFYCiNnVMZzRm8ACNnwqRy+9IB1T0BvHxLdaSjxqjnRF\\\\\\\/MTmEueXbAIzBdnymHJIPX\\\\\\\/q2MTSZ8tB+a16JMGJ10dL8taRyqtpn27xhoJRyvk1PNRpDHw+I\\\\\\\/ZWsSjEIHNhuEwVypjwffFUksDO1kKM58wbcWbuKvU3pCBc4uyeRmmfU6PdxgnxF8InJuxsOlpLysXBG\\\\\\\/uG0QvSzoCbejIEslmlhC5mycigZw2VY44lpHI4L9r1HERuVk\\\\\\\/QvzVXm4VCjmXUsKNY4Jc8HKx2dgGxy0VJkUFH0YqRyqqTOQGTOc9tiX1eLSxjAnO9BebfzDYBwYKv7tZv6mvSzGJnAiN2ql4ZoMkv8K81KfpvQ29qJ3IkjSWMVoFYmdpBYYpNzA6Pk4bL8AHvCUalQgZCaGUpigT\\\\\\\/sHjd13r4Fk9pwhI8wfgDsP4vNITdGbjytBc\\\\\\\/yuihOX+VUbpf9zT63QHQH+Pd6D5aV7LbyQ7UQHvnuxJ3MDuwz\\\\\\\/Z0kMelcmpa9zJCYr4cS\\\\\\\/AUsvLGhyIhM9N1HvJKrMG\\\\\\\/Fqew4+6jqx\\\\\\\/8Q2ekaqWIl9nmoADcz8lzvrKEc2omQ9nWJiUG4UkiqMsOfZMas0WM993ee4k6cpmz3DLGIv4OBEbq0lcTTHk9qMrDf8H6TRgRNFTuzBJaf70pN4rt9+RhiAZ2uSYG61EZmsQhMtNqTlcpoTCK9vPldLsSE+060U4cwTggdChux4ZSc6o4aLn4Ij2+bFvWhfCrou872JtCSGtPRJJzaxIX\\\\\\\/wz8r5r+xQxiTjJOE2mm4MVtHFd+vO+E320X9jbq\\\\\\\/T6gXdS7\\\\\\\/ztCCLFwr84vTxWgeyQGZAzUeVJfYs2hwqLewUkddQGXLVffOKI9hFjrJnmVtEWzJBkDT8XeEHYMsmQZeSi0Aoxpkxj68EXh37UqJiSQCFYPXu+5LCRyGtkWoOfBzDOj6SFQfw0sUcijOlHkSPAeetYzv\\\\\\\/jU7wc6B\\\\\\\/E0AxowjpsOBBMEWJGYfyZcN7ocFL7JFwV0CMn2l01q8i4gTt8faOVR2SChAqvdHR9sxkj0gaOnSdXa99BiDW99GJRllBg279r72GsNzz4zI\\\\\\\/XqqIkfyRNWSdRxDefhgrNbY8i8wu4OjBN4DmjAhqN8ytT09yKlO05OweUxlfQffYoS\\\\\\\/0BxOgxcDG3xpd47T8Tmo1RjNMpVJhV997mDDy5sdl+JrLUlfPqBBWzZYfUIsllhTjSNO1kotyUreHquZr4t3a1KH99bMg3rk2bMx7E+Diw81nXqUbqGHgufUv4BTW7783IFNsFI\\\\\\\/hwH8JOT\\\\\\\/CMB2z3SNrPlZ1n9wAtapLp56aLQzNefP3z2bvvoL7dWqY8LLT1KsS+dVFSApr3c\\\\\\\/3youFoN+mUeJ2bVHyIpehOxKr0F1WnVfq52j7wLIsqKeSMgqmhWT6Z1HucPulI8ZshC7FHecBe0ajwVuIQ9WxmuX7OMMMAEH6tu7P2Ws56JFfhd7PYDg0YK76DXrf\\\\\\\/XpVij\\\\\\\/ES9V5dsLy2GdWPcR5AoTYUIriW1OkpoqpvvyIZs1eDinMomwv+\\\\\\\/hERxVFy1juhM6pyK679oDHuJ0dIsjWAWLwOqOf3SGOmCEdG9LlZnO9zvFEGg+1vf\\\\\\\/6hwDKJOvQttCaPN0ZdJVXqTm2+zdsLECYYmFdPr8f5RDWOPXgO\\\\\\\/uPREH0rSXYcPHPzpYT1PesfTJ9vKe7UuvvhYFQBiYrIc4yFV+TvVpldevO1tik7+QvxHltnbyrmJR0+bm9\\\\\\\/bmKZbywrFiZh3eaNHCNwD5nms9xF2av2xSIfm4g93EAYc2pxEN+7+\\\\\\\/eNfHIXHWjX5Om4wZsK\\\\\\\/PFHVWEq5gn3HK2V7Dn1pqbpCQLf52rm9Uvy5+tBkmTpfqUbITjy3IVoRuYx3VjHx\\\\\\\/GBaIFKnQjQpzyADvouLK7OlDGUPauK1tFMkreWIwGrwk75OHuHKWswBDpIShjPh9Xr0s0tXIIQ9AjE2lHEQ8Z10+3+m\\\\\\\/8zP50qm+dOfaxsKNfXVluDi+y97cYk177WMv7hM2q\\\\\\\/t1aUNHGQPsarlT7QFBFhWFA7qf0fSq5YaV54vN\\\\\\\/6UiVeU8uVvn8IzkN1pLVr\\\\\\\/ym1swRH4jy47BlD1VCIr2oH9c\\\\\\\/8basl\\\\\\\/H7WNTHH1oRC3siZcfT8P27qjyvYqAcivYAYvAYiQBS0vBHppZYOj09QSFtiW0c0vtoStbf79QzfSlSrpSfXqLb05BJIuhLWhDpkFGmOD\\\\\\\/fvy95zCd0oeZGJILmAYlc1GBO+dxCF0VWZHTef5bGVA8H8K2vBjonHuYCMzPS4Zv8HhSO02ZnQroYEMq1MGGN0QPS\\\\\\\/LAquKTyzJiRJSSHpl4\\\\\\\/BJI\\\\\\\/Bz6APxJzzS1KHFGjlHfnVYKqAbaYSzaY3XJEZQTkLatxMJX9oNYOhN7xmSKCWs\\\\\\\/AekHHA1LkRURCXwUcf3PX1MQ29uelgJcCgOFU0mN9SplbBKhM7fNoO6fEzqF9Wwa7rHOU2P1FPegNp5n9uyB1lRlirk8+70qP9SS39+N5kwM0yNihTldUmltPqlS4gtOSy8HEbpSPLiufQshTuJ8gUEP12GEwYzkFsObrRvC4Kt771noLmkqKpTHnX8mLS+u8ZUzqOPkUm9J9ii7tUS9PH1Syue7WFCg9+LJsvfjbZI6ftteLiWwqlzkzdIVsQQUPORfREoFRwZCFD3ZK\\\\\\\/PFRXNdE7jPK2AKZvn1HOkdc4eywMEndV\\\\\\\/xSjV4acFmu85VA7C6OwbN0ZMV+UHHaOl2iuC4zlQQCltK\\\\\\\/HpqDEr4M0ZE8RJIfsMD7+sPYK7S+WEOLlf57JimtxUIbPvBPutkozcOr63Pn8iPVXupO6nXo70sUsl25AX3D3qWiCdqDK7sYNfKQo1CnKtCxtLKdFm7xPFJgKWzo5QK+57Rky0g0aQeZgiX5oCh5qrlN6F6\\\\\\\/vruZ4TUBuQ58LefIeHILHRmZeMs\\\\\\\/sa2OInvuFBSP\\\\\\\/YQZnJTbcfPcsStwKEvupWrAHGIbWbfMKtQZqHkjfC6gyfe3rwln4y\\\\\\\/GeVl\\\\\\\/c3+dG0QTYv8UU3HjOT+JCLQahPHV59jZF9O+qTSfvqHcaUVtsOK1cPON0jKL9cssv+8\\\\\\\/RlZ1l5Ob8gOFUK8qrZS7ZdvyxBusEEljASYsF3RsSTYtOnjUr4ZvtNdbtMaF\\\\\\\/V+2tbTYt7D8FMLQ54Zt3BVAys2+c2DquYeoTKQjv1JWJ+FfXsFwG\\\\\\\/Nvw8l0nNO3u\\\\\\\/EYTD304ASV80Hrii973b8ojjia4gXKNqQ8ZPd6u+7oFEbJVbK71SKjKSriCapyg1Pa0XfkJ9SNjfi5R0N4sRas4nPbn0o7vDOQLGAG15kufs+sbmlKge4Fg\\\\\\\/V253D66Hh42hXZA9ZPML27FKrWAFYueT\\\\\\\/BwBZvN2NGla13W4xEmpmNchTnjku4S+1rSJJZUTqAJ4M1usGznNSB\\\\\\\/KCslWvHT5zZikSz876v8f5jsolaBdpxsnF3QuJjnKM0KgvAC7yGWL88RI+F\\\\\\\/6QgptPpMiOsojeVBJN48ZYPBj3ZaeoVkDvqbLihJomkDdSEGms2k8SazqQ8ezTX8vPOVY0CRohM8Hr2U3Tzev1smMLz76S9GXLIbPCa73cC7EQ\\\\\\\/muB2EhsiDmKUmEQ4ndd14A44Xy4Vw1NVknJhghaxyxv80K33cC29sAG5qhZx7o\\\\\\\/KbBJlj6MIucdFgPGWxqdfQ\\\\\\\/hFbZmi3P2NtjMlM47yQkQsxcG3v1YqA+wcgWb4piIRy2wrO08XcdYvDHG5Q6mxZN8OJGVtbvUQABnTqO2L6Zml6s\\\\\\\/0DPJWbhfv7ZOrhQPjFOsJ7t3KozZk9W2xGbHIEiyIyu4RWJD0mAbGnaH8EBqB7AYjNK7ZlQbbwg1qjZZgEuai7ZkrSfvMNeMn76l2J2O\\\\\\\/yYc+0nq+JIU8MAR2RCr+lvjpUDqdVJV6GrtBvZ+qxpyzBteLyTmZSR\\\\\\\/FuJkpiMqrcF8qzebreNPHj+05GwXg0IHfixOaqL7KpUoncF55k++sJJJnVy+bfiOsKZpgbgaO5H2kquQAeDnJefT5+iICNTMNdqcriaxN2HYn\\\\\\\/orE2KYF9Tfe+MwSqnh1oMJLsUNGUiBV5CSd\\\\\\\/TvR9OTrpfqaTSnlZnmt\\\\\\\/SryH0Zn4VD63OJx2x1JPNz31rfjyHCm9TY6ZP4TJRUr\\\\\\\/HvIf8bUaFXreSxI6hqjZ4UnOzLhzCKuaoYddtwK12vYHibSv7Oy0g7W7WEvufzqs9HLOIhPyd0dsEV2o9d3Rx1vO5MBMXCLRSKBb\\\\\\\/XNHkUK78o5v9WyqgYPUFudmEuwGRVaowbWTjEQpggLuD9rv7ADJfSCzo+pWxuKGK0\\\\\\\/M9l3WrQTNfbB5ucjADrf3vs30kXrgd9aegTiIV2MMTKuGHpSXMCDK+guqvuPO3G4UnYYcH0HMhATSoj7\\\\\\\/C1lbuJVpfCXOWeCJVzOKsDltYTKpXbUmdAPCssQoSnZy\\\\\\\/4QqNIR4Y5ewn6snu4mxwSNBIUUWT22skl6drxLX\\\\\\\/73DXcC31KrMip8JRdQ+pHDATkSJC4Idbh3I0Rg7OwmGIoFGkrDysNcc+VjYK4H49YlbbrKvBo7BKOf5Octs+taYSRgRRApQNsDS4FwgnV1PvV45dvWLqdjU6HlcNamELo+UAITVoAMdvRhfpViKXeQSPlD2Zw2i28d\\\\\\\/SeWcSfDyzCVFD4QWLVxKHs+iMhxGHR5M19gs5KgDP1OAGliUX4z\\\\\\\/hGVqxGIm1p74W2SHLxl9LAhtlFcgIQmvFH7fwsKOmTYCtvc883uViG3LUV3C3EiOBKwXigzfDv8qBNN+N72kCy\\\\\\\/0ZJ0LYq\\\\\\\/0XRLTurbaHoTQQfbA+Hcw4tRlJq+7esDz4ihVD6+cFPtRYjJeEYnVnbVpVZpuxSMfiJzSbfaClnC6s+7LXjXQst1iC8nYwhZAXXZK273OHh83OIkBCEfd6dUnuihvP6YmXD8LdyYJJ4SvhuOs6LHyxmQNXO7gqylM3KoP4\\\\\\\/+FaOSyXGrn1kYFkjaa9l1JJdILRTfK0GSf6BtmdPqTiJTmYchd1jCkl9iSgF\\\\\\\/SbBnFvRcIz\\\\\\\/nXCjdGcTl0uVweI1P8Jfnm64tE\\\\\\\/ne2rE81usX2hmtPq9Ezv+zIHHjjD6KtMDjYPFksnOUqNOzCCMiqP90wHxaSafgOMEDzzOnrw74iEkQGi1b+suVqRMVYOtBsE18FScCgmvajk6EznimVIO2IN\\\\\\\/m6Z6D3yc6Mj3CPifOXgd8cdAVHP5pJnZKPYn6kkiy6GJcu+KhYDBli8Qnz9SFP6dwoUTTX8LiPdXjBX+12wuunjrzKY0G\\\\\\\/x8Xx6kGnt\\\\\\\/wmQwhuPlsrK8IqUu\\\\\\\/4Ga\\\\\\\/YrPQNA3nhhc2bOctkbVKr8Zprt+NuCZkcIiJlnWnD+Z411gmCQ5CGnn6Pk6HT\\\\\\\/GUuy0MafbAUEed3aRv1xBVTA+QHLBA9HBxvtxZXnjCc3Dw7yUY0d8You+raUuf5MI84hk4OtmGCPQknz7tPX7f60vjzoRH\\\\\\\/FREYUMxqzsd5SbDUH49jM1I1Hb4Io5dHVC+Bsr97rRSyzj8+Ha7tMD9UUkoOw1+mD\\\\\\\/NLUPURNaLnOxye1sS0NK2IcAN1L+vyEgUK3CylMwx0yVUO0GERV+egXd\\\\\\\/VMPfSSkzYaRSifss90L\\\\\\\/za00TFzqQYTfIZAniuw0Q1y0V19UaXIoUagtYzeLcQKIeDZmsso+gvLWzZQVAi6zpNc2sDk9vvNhXKHsgopiB5QkgJyssgBLMHE2nnVd03WjcPj7hwNydvL99NLOGXgSV6B4lR\\\\\\\/OU8qj1uLg5J5ZHl+GdS9SrG9w0\\\\\\\/lCtzKgO3DJfR5ED48vxKpHC2H+WDZu+oOOBdX\\\\\\\/iucsaROoLIV1WUBiiHVD1I12cU0Q8MRQib8Mw6CA9uGPNOwyhHATIrzX0hTDXJ3WixVWDRDFYLytktBKFvG7KwzSTZhHT0qXBpOpDDobie\\\\\\\/Z5MYtgnbRbUfuIC0YvPiyyqkNxCSgUNxb1wf2rAm+TsJ4qIwzH+ffPuJciwm6rrZhWXlF6N98JN3Dbp+9brf8HJOE7+5BvfhRAEfe0NSf20BFZkNW8i39g4BLuOiTgGlGZBvVRP0v6NcQbbu2xo4zAgNYmrGPVYnidZw5IWzTdKHORCqlimsO11RvIoneKERbMbnCkrf3N77+HoMF7emobYfNHIUb\\\\\\\/6TUwguBSfipakUt7r1aGTqM2FItFzJz5AMj08vcF8JdfBmE3ndxqe1aHGMbeuIfmEgQ9tDZzcRxJKL8NJj406\\\\\\\/2WU73KzdRWx+58AwLU94FR1gI3XRiwy8PnAqnPhIyAhMDSDZ1LkPrz7tKOX\\\\\\\/fLStvlr31yz9hKwDDYZ+PEMlOyouSDZwKpQrAfyf8WrJ0AAfQRLu30HovHWTt8x\\\\\\\/zOvCN0tpIEfkTUukmC7VcMfURGylTPKv2bjJJ0XbFOMIQYSOS220B+x2Dx6N4TBWcSiyHkOlGB7xHQ9O32Ob++pWQMZZPqpMXlACqarUjRBTDU0SLFJ5K91N6+m6+H3T4tCP\\\\\\\/m3kaIffipMqKlB6ZDotFmumW1uR0nfshWaSy\\\\\\\/R3LRcoglU7bNek0lZbt1n1tKGeHVskb0Uot5CDrhULQJsmNKsYsCaY+ubRqQ8Lw3n4PqLiBAhnJIYLxB55SKuUMgKNdAtErfM39ZVeOE+X6YJpJrzQXtEmpw1q3k\\\\\\\/RsX1UsdMdJS7eFIYgbKQCUUeTvn95+quaUhBM66tiI1o2PnndMdjVQ+PxfJTyDsnFFL\\\\\\\/yAkuYju0W6fmfbopK2gRbm+HuB30dB2LQmEm5b4pZK\\\\\\\/clf8P1BHsIMxMIj2j5CZyyDbvqKcChk4dAgHjDm2c71OHozouQWIdkDfWAcbQFhiz9vgI+fNLeuBu2+RYz6FD5un8Q7aMmX2WNysXPP1xn8KRSBpOjLUNKxTK96powHXRHW1bnTbowGtEDkRK\\\\\\\/XB1nFNdQEq\\\\\\\/4Zwvvh\\\\\\\/oQxrwYXyk1oLg4i4sQWQe355o9mv1gjM3f9fjPRrFBfzTiHR1vWJgvMhqz3y\\\\\\\/7GHrOfJpRaJUbRstRAeB\\\\\\\/k3zLFGMy\\\\\\\/AokGG4SW+ez6f\\\\\\\/oPkqXKi8k5RHOGfJN3ZgeebzIUurwrscLOVAqyCgubtCB0wPxMufUjdHsK1hL2w+SE3ZxMCUBJ77Ripvb7+Fs7sYerjsrQZnSv2ID51jlpzg323Kwqt9cJ504w1Lb3LkeE4DqqnfNsyn5v7zNK6PZuNr43JEQ0c0gIGKArs3p53Q5sac0eCAHaAjEGujXKI5B+9NEFSwERlUdWkrMMEwGIf\\\\\\\/f8SCIZ7uYhZcRYRradPjBOnxYcPYPsqnCyR1gWPjolSQNBLqi8y4ICw9H7tgmHJQ8FcVAazDkfZZGir\\\\\\\/flNnD5b7hOSiYQ5hb4ESdK8WxMfQrBtYJoqPQ2lZ+MXcYHjRFwsNFzgUnq3qx0dfL0uGDOeZVUzbiro3A9hhO0l3lTPt8uubEckUdJ4tMzxNAc4t8auA8q98fmsJ4g2o78JE4akZVfzHW9el33Yy7SOcoIC0yAYvayB2BTc7vrRHPCwQ0+Q0\\\\\\\/ZgQ4NaDUdPh2Npn7tOrfIOLoD5CObbo5QLt4LTnH3RJynbwxRyGILTVVW22FT+nPTA1+zJAVZBYLySHUgaD6HmISFUeTHeqF9CN0J5bcKlRUq6x3Hln2XzbbTYjcklo5khUxbQwN2g3TvSW1E4NyknbyNsHfZJIS9KeqC43ZGucsdYRMU62bjVh3IM7SLTtLg0+rzpOSUudtKAscrgvLbe8X07Zi24f02i1HQdBNx\\\\\\\/EO1DTlKwkjRb2z+LyYUZWTsFaSWCEujWRLUS8e558e+Uerx3bCA6ZtWZ08NU9YuGaIKo\\\\\\\/qRfEYlKAAAFzQXNpa79xXU0w+ldcOV0lJpbyZ3ePFsZHa\\\\\\\/G6Ee7Fgvvd9HseBFyKB+03yyfr38G3VroeOOhl+VNHIMS6igb8j9KypZ0uRZnQ\\\\\\\/htzlLXgzS6f0cVwGjvxCdYRv\\\\\\\/+W8Fwfty6zJfKsSEDlgjCdeURJjEdKUwSgx\\\\\\\/fsGHc1++Ztqh8JUl8Msw76p0PckkvIOCvnkq5TouZ1uLFNroJh1xzZsezbk73kuYsEsafKWIMCtrW5yq8PiWfJLvbwcGUU17Vnh88g2c1i5NU821EjyjliuY8GyyW8ppJryB2Q6+M6q+7zaT4vWMUyRiip9j+tR72cxsZvm2SbesXr8zwnkp9q+thDHymrtzuZTji31aQvatxatwVwvkL8+3SavmgqBrFboNA0ZsDGtfsIGtrpkoeGXnPklSYQ7t+we\\\\\\\/8DwLpBQqbvbS\\\\\\\/JQoDOYGhAOuBwsRk\\\\\\\/sSpseZT6tqGbKFglUApbCXQmZRXozrHCS8hf1uLBtHBi7Xr0IT1HMWfflZddhrSWEIZDLJH5E7sCX0oi4XvF0KFLV+ar0uN4c82p1otkYMJ14IhCKm+lhq1sYZTKOUNvBWE9HjTvflAmf+y2NZ2+f\\\\\\\/1WK+myaUK3ilI5QB05Z2ecdnf8SZMe5X74j6Pw0kg82KK2hzeCxuG+xcYZMyiYWEXIiF+81uffPUHluTfYECXzlCTnOQFBZXc8H9aYotADMd92n0FdiOqpLnvPbIAVHRWdtuCJdNnykvkrDC\\\\\\\/2Vp7WWF1Vo5VPxsL6JYlASa7rQGW+\\\\\\\/enhqMWO+2a6R6UUc5HZv2SlfmvShTfXj1HN2YMCLUxdERhMZQPxNVABKarqYbaGIrrT\\\\\\\/br9Z5iwddY5sQWWco1PhcbmGTwCpf+zpy1p\\\\\\\/5bX+rFttsd0pIdY8ifNed8UI9uFudOjGpggLH0O9qVJikgJALjMllAvCDpa2JDKjKeZCLvlYH9Bi1Cpi5Sm0ZUBG9YR+TIuE1WMbvCzrqrXHzeLtYLhKacFZ+2N5IBnbPvrJYfxcY8EB5OBJtDoLr4EDT31HddHToOtmJGrVOS8r2Rq8hoYmlVsaFNf\\\\\\\/YTDHU1ZULTd9nCK3uwUG2735+fFnpXIXSW2kIru8N5cd0H\\\\\\\/\\\\\\\/1tQg8giF6hlVzZ2HyL2sZIpl5TdKKo1U\\\\\\\/tVuE4rvcHglTUnDOhJm3PVeg+FWkafXNi1uKtuiItgtiKnMnavYh9csjiqwpDI4g6xb4ziMgiswnzSdFzo07FYzFm0ULe5gIn64Z1TOe33\\\\\\\/ZEs78l9ygmSF4HT6ROvIflL7dzEtZm8k7aSdU1NVE8ajkievKWIBYUj9cgaCiyZsIWbTb7E8BuW\\\\\\\/xC+12b3qNK+uzS8WC4ZTh9\\\\\\\/7Cf8zaEXmoPiLOBtjR2jtMUGtQTpKdDIXhFVoHdPybY0FN7Fhu8bGFOr6hEsxmJUZBiYh47tMxcVi8fmmH0xUDojWUqJPNVMkz3xQN05rvbD+qZxbILdI5cpamjb0UgVDBG+OTOc8zs3NJssoQtuOZX+vBUDR\\\\\\\/U4wFDp\\\\\\\/NqaKdjziUyVdHzV2RVxgRlI45LEpLZwjVETWsWHVYwz\\\\\\\/rcduT4+UT4WYPGAFSa9cNwUjKMHlZEReQ5f4z3jdUBhZcBLQ0irtKEam5Yea4M7kUfSfCU67V8SmSio5+mdK17hvlTnDJhNtizBDMzk4WavQ+g+K\\\\\\\/htF2xj6L7Mn+wHzkfZ4pd+sc+auvzrZnfcu5XdQT0pDoILWLrg4dMQtKz2ShlklcNBIXSSmH9\\\\\\\/NoBd\\\\\\\/ZZyDsxm+PoHs1qGvitNOxf0JrWeil9VgM2nhbnkr70GqzeSwXG6iaJpM7LtjydLHBQJ0rCq7nahxuwLD7aeQlARTUOTPnyG3mEgvgdV7zVxA34+LLSddFTczPK4TLVezxWnFoASUo54PExJKQR0Yiyfu4IoxplnnTERgtiZnyoK\\\\\\\/gGbwzfwpO1M3+myzCZLFhg+x4hq7BZHQoQgDm5FaQNauP01dgvm4+IXrGPSITZ0qPNUPXZ0TUgIBk6ALsGlZjyFGLd\\\\\\\/0KGB2dvunJmtHY0tU2a\\\\\\\/M\\\\\\\/cJtTNcdOz8UT7KYJOtMrPkdlv\\\\\\\/QDIghyPI5Ye0WyZt3eJBREa3j6MJsRrSDF2thRQltn4uqT4NQ3tniAqWaWia7QShTErqbpFQo6Pn5QN+eSAZdkISxtIpsHXwFJpDDloTc2fdAHawNcDhTjJrnWsyWib\\\\\\\/fHr421xbQm2fOhjnqARixkYTWTTWP5j6Ao\\\\\\\/KTay9xb4gmOubjI0yhvnepTcjwJ8uzuhz2huWDXx8z04VX014NDSxChJcjRB2G1nMSVYUenQvvvi+vCFuH5VDhMD8CWswTEBwjjGjrmvC8qALoAddr72fchCfCC0aUZjMwmKQRWvDgpZLAESWYHp0K3EptV0r0o3miewUyrypPYFNRczxPxJQneglgYT+nB0QzewefB7KkeLpXzmK5QhY5doyjKCvVi0hktmXZQF1EqqTYigBcWvAGpxQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ilEqVMwpASc\\\\\\\/wn3h7dxKQA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dbd7f9b23da0a9c937af76069d6740423544582c99e2b98be71f9b98bb173dfc\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228mXTKuMpZn8DS26nG810jZCdRcE48yfJ4pyMiW+rUvAc70NlLObUL9ce8hvVgmR\\\\\\\/SKlRWZeW62T5rqhhTIQT8NLHFSjDK0aiCA40CnZpgu2Dqv3GaUUrXGW5efTauofpb9nsFYQMz4JoT0F1PKe9qS4JyE+EhxmutebfaB2P+yFNG0hJFHdeSR5jAdS4RyuJoKcjXHU+Vb4cgIMXti7bTGnRQ71TL8dFp7Qouxsp+FpuWA7JkzQ8Dc5Io2pXjWFjKNNnb1GyMEwaz7zmIB5zSejjdPKqEWprN3OuT+iX8aCbDcr9ubjnFpxRs6FAZLvFzwo4IbVwpHRpMV+PTqT+wyg0TXuitY6uwPopgv3WCQrX43icSMx0ICeh8wmuf7jbUI9+1qzpENGw44UYGN2ptqPIDZpPSr09owNusTdXCEpESx841IQ4Cxw+9z\\\\\\\/OALsuWYzFNGXJEu+PjMwuzDFpKchQZzveJ\\\\\\\/Sbq0AbjpO69jBVI7viZz3Rapkndcid3xtG1Qx4V8QxHQ5rXO\\\\\\\/7Xs4F9ftmX5MaX6bIDIL4duAphIiF3FzT8MxG2yY0586HrGL7dFfBNywCoPCSioE0EvX7Wm4J0CtQHERYeaHLN87Ls9Vyj4sIlEEQnbzeV8bqHHZnhwz8mu2hiUZYbspG1j68n1yiYiGvOvRvvY60yAjA0IGshyNd2tYtlXXG2UjPcwQIkvhaUMplKAO9FtQJqa2CS1gOc0lD0N95hsarGeA4NSpO+ALLgIKSSx5aQjXqQBc+Yg80mahEut\\\\\\\/3ECjiQNAsG5K0ujKfBajow3HDH825CzmlHUN4paHIx9vBI3paM4qDhTaLq5rUZZ3XJe4n2pV8ZjuG9Wi8p0faHYwipu1kkv4nLebbeXEBSd035V2Cb1u+mJLPXMArRty3eM3iVay0aYZVb6NCqvnY2vwt4JMPiRflRiwg46mg8HLVaspwN2TEv2Mtz2EHDqRF6LRXnsls+4tqcx85rb0SmmmkR1f2EaTb8ACHVg9hWusjLFcM3feoVP6Fb7LJ0m\\\\\\\/DzkMtmXArnUcD77LxDbu3p5XHwVnxCQ7m8ERgGgE1JPSS5auXzL79wAS3me3TAXwWI+q0QY3G521HOguEpE19+Syi7fRuLxIlL1hilRK\\\\\\\/tv9zEtyvQmPzztwOfYRTPAqe3TAyrr4PN75Po1Dd6RlGY5iY0Lr5dgpFzfDI\\\\\\\/vGM6HIO9pyCV7vY12Lhu+jZWAXO+CVSUpecpo+rifgffiHQ9b5E55hASifWcAgIGpV0C\\\\\\\/pvXyIPKoGfhiR8f0gzAv0RC0zwfzW21c\\\\\\\/+0lP9ZdEUxMmn1\\\\\\\/dMvbcHRCqvjuqieHsiBl\\\\\\\/uJjYKD0Qy4AjIqffR48jHPAP81Sf0SsomfMn7AmrfEe2WP5iz7YxfaidGLmwYD6LUW1\\\\\\\/gGcEYZmfHwYPprzLIFF5L0cCaGmx04ONvXJjj9Ay1QopSvrxMJt7KH2fit1hZ7j\\\\\\\/G0n0bh8zcVKML6cDVgWAd0ErKfCStlX1rpK6mcsebG+OFdE3Jh3Jp3Tph+N7GNuzhGY0FwCCoYwLsmI9vB3dz2NwxuXN56wOv8kRKIrdrUaZiP9r9D9SJjpd9XF13nvnstlW6xHHqOlocgXzGU58vrmYOlnBz03kX1yIs3JsLS2KUkaGchTfjiW2SF0I4\\\\\\\/UF4Yx43y0R0Kv+AIi\\\\\\\/GU7JsKawhmkJFM63i6oOGhzWzTu19XAtzb5OtnZ9lzZNQQz+TqodR8g4AtYWIzIkIVeY6P+3mhhJhGDQ+BI3eoaDRs+gzI3SV4M9TlHQjblA2iAm0jeesxGJ\\\\\\\/9\\\\\\\/xvsCLIliXVHSBYlYucyg6VQaOzxJMAfN+eyZAoVTgXi8aMZQ+sLLmZ+g1QK+P0YBMillGhgv6OMdeNePgepiLgBzpSoLb9WvhFKKSZBgX\\\\\\\/A4E0UPaFxQdoRs1SF6qT5wI62rN\\\\\\\/jTzmWVGPhHnIS6iQutf0new9wqgsHbW+2vd5ufqOaNCMBDwz9+umUteUipHnDsEVvJOGTQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022h4leXU9LW2lMbC9IahyNhQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022072b87781b98074e659d51abb37c1b597d7be0e4e66077c4d45216dd05ef6b1a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459781 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/3/449" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Em2BkyBzZnKlXp3HSp0yGumrKo9zwDGK3U0BkWZWVj4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/3/449" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-84z1e_Xbww0ENE6R_VGPVGgIEp7P8HIpJpKsujd79OA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/3/441" title="Evolutionary Consequences of DNA Methylation on the GC Content in Vertebrate Genomes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/3/467" title="Identification of Nonviable Genes Affecting Touch Sensitivity in Caenorhabditis elegans Using Neuronally Enhanced Feeding RNA Interference" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459781" id="node459781" data-pisa="ggg;5/3/449" data-pisa-master="ggg;g3.114.014563" data-apath="/ggg/5/3/449.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genome-Wide Association Study of Resistance to Stripe Rust (<em>Puccinia striiformis</em> f. sp. <em>tritici</em>) in a Worldwide Collection of Hexaploid Spring Wheat (<em>Triticum aestivum</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marco Maccaferri</span>, <span class="highwire-citation-author" data-delta="1">Junli Zhang</span>, <span class="highwire-citation-author" data-delta="2">Peter Bulli</span>, <span class="highwire-citation-author" data-delta="3">Zewdie Abate</span>, <span class="highwire-citation-author" data-delta="4">Shiaoman Chao</span>, <span class="highwire-citation-author" data-delta="5">Dario Cantu</span>, <span class="highwire-citation-author" data-delta="6">Eligio Bossolini</span>, <span class="highwire-citation-author" data-delta="7">Xianming Chen</span>, <span class="highwire-citation-author" data-delta="8">Michael Pumphrey</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-7571-4345" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jorge Dubcovsky</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">449-465;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014563 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459781" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marco Maccaferri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agricultural Sciences (DipSA), University of Bologna, Bologna 40127, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marco%2BMaccaferri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maccaferri%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarco%2BMaccaferri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Junli Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Junli%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJunli%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter Bulli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, Washington State University, Pullman, Washington 99164-6420</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BBulli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bulli%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BBulli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zewdie Abate </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zewdie%2BAbate%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abate%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZewdie%2BAbate%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Shiaoman Chao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>USDA-ARS, 1605 Albrecht Blvd, Fargo, North Dakota 58105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shiaoman%2BChao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chao%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShiaoman%2BChao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Dario Cantu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Viticulture and Enology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dario%2BCantu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cantu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADario%2BCantu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Eligio Bossolini </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eligio%2BBossolini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bossolini%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEligio%2BBossolini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Xianming Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA-ARS, Wheat Genetics, Quality Physiology, and Disease Research Unit, and Department of Plant Pathology, Washington State University, Pullman, Washington 99164</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xianming%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXianming%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Michael Pumphrey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, Washington State University, Pullman, Washington 99164-6420</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BPumphrey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pumphrey%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BPumphrey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Jorge Dubcovsky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Howard Hughes Medical Institute, Chevy Chase, Maryland 20815</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jorge%2BDubcovsky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dubcovsky%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJorge%2BDubcovsky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7571-4345" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jorge Dubcovsky</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Jdubcovsky@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">Jdubcovsky@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/3/449" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459781" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459781/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/3/449.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459781" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459781/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/3/449.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459781" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459781/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/3/449.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459781" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459781/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/3/449.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459781" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459781/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">New races of <em>Puccinia striiformis</em> f. sp. <em>tritici</em> (<em>Pst</em>), the causal pathogen of wheat stripe rust, show high virulence to previously deployed resistance genes and are responsible for large yield losses worldwide. To identify new sources of resistance we performed a genome-wide association study (GWAS) using a worldwide collection of 1000 spring wheat accessions. Adult plants were evaluated under field conditions in six environments in the western United States, and seedlings were tested with four <em>Pst</em> races. A single-nucleotide polymorphism (SNP) Infinium 9K-assay provided 4585 SNPs suitable for GWAS. High correlations among environments and high heritabilities were observed for stripe rust infection type and severity. Greater levels of <em>Pst</em> resistance were observed in a subpopulation from Southern Asia than in other groups. GWAS identified 97 loci that were significant for at least three environments, including 10 with an experiment-wise adjusted Bonferroni probability &lt; 0.10. These 10 quantitative trait loci (QTL) explained 15% of the phenotypic variation in infection type, a percentage that increased to 45% when all QTL were considered. Three of these 10 QTL were mapped far from previously identified <em>Pst</em> resistance genes and QTL, and likely represent new resistance loci. The other seven QTL mapped close to known resistance genes and allelism tests will be required to test their relationships. In summary, this study provides an integrated view of stripe rust resistance resources in spring wheat and identifies new resistance loci that will be useful to diversify the current set of resistance genes deployed to control this devastating disease.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/disease-resistance" class="hw-term hw-article-keyword hw-article-keyword-disease-resistance" rel="nofollow">disease resistance</a></li><li class="kwd"><a href="/keyword/genetic-map" class="hw-term hw-article-keyword hw-article-keyword-genetic-map" rel="nofollow">genetic map</a></li><li class="kwd"><a href="/keyword/yellow-rust" class="hw-term hw-article-keyword hw-article-keyword-yellow-rust" rel="nofollow">yellow rust</a></li><li class="kwd"><a href="/keyword/bread-wheat" class="hw-term hw-article-keyword hw-article-keyword-bread-wheat" rel="nofollow">bread wheat</a></li><li class="kwd"><a href="/keyword/association-mapping" class="hw-term hw-article-keyword hw-article-keyword-association-mapping" rel="nofollow">association mapping</a></li></ul><p id="p-4">Stripe rust disease of wheat, caused by the fungus <em>Puccinia striiformis</em> Westend. f. sp. <em>tritici</em> Erikss. (henceforth <em>Pst</em>), poses a major threat to global wheat production (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chen 2005</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Milus <em>et al.</em> 2009</a>) and negatively affects grain quality (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Dimmock and Gooding 2002</a>). Although multiple fungicide applications can control this pathogen, developing resistant varieties is the most efficient and environmentally sustainable means for reducing losses due to this disease (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Line 2002</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Chen 2005</a>).</p><p id="p-5"><em>Pst</em> accessions are classified into physiological races based on their virulence profile on a set of wheat differential lines (<a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Wan and Chen 2014</a>) (<em>e.g.</em>, <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/014563SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS1.pdf">Table S1</a>), whereas <em>Pst</em> resistance genes are classified as either race-specific or race nonspecific based on their effectiveness against different <em>Pst</em> races. Race-specific resistance genes, which are effective only against a subset of races, usually are expressed from the seedling to the adult plant stages and generally result in a strong hypersensitive response associated with high levels of resistance. Race nonspecific resistance genes are effective against all <em>Pst</em> races, generally are expressed at adult plant stages, and are characterized by various degrees of resistance (quantitative or partial resistance). Many of these partial resistance genes exhibit enhanced resistance at high temperatures and are designated as high-temperature adult plant resistance genes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chen 2013</a>). Pyramiding multiple partial resistance genes is required to confer adequate levels of resistance (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Singh <em>et al.</em> 2000</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Singh <em>et al.</em> 2005</a>). Both race-specific and race nonspecific resistance or partial resistance genes have been used in breeding for rust resistance (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Chen 2005</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Lowe <em>et al.</em> 2011a</a>). However, high genetic variation in the pathogen population and the rapid rate of selection for new virulent races have forced wheat breeders to focus on pyramiding strategies that combine multiple race-specific and/or race nonspecific resistance genes to increase the durability of the deployed resistance.</p><p id="p-6">The 2013 Catalogue of Gene Symbols for Wheat (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">McIntosh <em>et al.</em> 2013</a>) and the 2013−2014 Supplement (<a href="http://wheat.pw.usda.gov/GG2/Triticum/wgc/2013/2013-2014_Supplement.pdf">http://wheat.pw.usda.gov/GG2/Triticum/wgc/2013/2013-2014_Supplement.pdf</a>) include 67 officially named <em>Yr</em> genes (<em>Yr1</em>−<em>Yr67)</em> and 42 with temporary <em>Yr</em> designations. At least 16 of the named genes have been introduced into wheat varieties and breeding lines from wheat relatives and alien species (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chen <em>et al.</em> 2014</a>). Unfortunately, most of these resistance genes are no longer effective against a new group of <em>Pst</em> races that appeared around the year 2000 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chen <em>et al.</em> 2010</a>), which has generated the need for the identification of new sources of resistance. Although recent quantitative trait loci (QTL) studies have documented additional sources of resistance (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Lowe <em>et al.</em> 2011b</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Sukhwinder-Singh <em>et al.</em> 2012</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Vazquez <em>et al.</em> 2012</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Christopher <em>et al.</em> 2013</a>), the use of biparental populations limits the extent of the germplasm that can be explored for new sources of resistance.</p><p id="p-7">Genome-wide association studies (GWAS) provide comprehensive surveys of germplasm collections and are an excellent complement to biparental mapping studies. Because GWAS exploits historical recombination events accumulated over multiple generations, the short evolutionary history of polyploid wheat (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Dubcovsky and Dvorak 2007</a>) and its self-pollinating reproductive system result in lower mapping resolution compared with outcrossing species with a longer evolutionary history such as maize (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chao <em>et al.</em> 2010</a>). An advantage of the high levels of linkage disequilibrium (LD) reported in wheat is that the number of markers required for finding marker-trait associations is greatly reduced (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Chao <em>et al.</em> 2010</a>).</p><p id="p-8">To avoid the discovery of false marker-trait associations, GWAS studies require adequate assessment and correction for population structure. (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Flint-Garcia <em>et al.</em> 2003</a>; <a id="xref-ref-91-1" class="xref-bibr" href="#ref-91">Yu <em>et al.</em> 2006</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Kang <em>et al.</em> 2008</a>; <a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Stich <em>et al.</em> 2008</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Larsson <em>et al.</em> 2013</a>). Even with this correction, GWAS has limited power to detect allele variants present in rare frequencies (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Brachi <em>et al.</em> 2011</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Wallace <em>et al.</em> 2014</a>; <a id="xref-ref-95-1" class="xref-bibr" href="#ref-95">Zuk <em>et al.</em> 2014</a>) or loci with multiple allelic variants (<a id="xref-ref-92-1" class="xref-bibr" href="#ref-92">Zhang <em>et al.</em> 2012</a>).</p><p id="p-9">Despite these limitations, GWAS has been used successfully in mapping QTL for different traits in several plant species (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Brachi <em>et al.</em> 2010</a>; <a id="xref-ref-94-1" class="xref-bibr" href="#ref-94">Zhao <em>et al.</em> 2011</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Wang <em>et al.</em> 2012</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Jia <em>et al.</em> 2013</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Lipka <em>et al.</em> 2013</a>). In wheat, GWAS has been used successfully to study agronomic traits (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Breseghello and Sorrells 2006</a>; <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Yao <em>et al.</em> 2009</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Dodig <em>et al.</em> 2012</a>), quality traits (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Reimer <em>et al.</em> 2008</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Reif <em>et al.</em> 2011</a>), preharvest sprouting (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Mohan <em>et al.</em> 2009</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kulwal <em>et al.</em> 2012</a>), and disease resistance (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Adhikari <em>et al.</em> 2011</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hao <em>et al.</em> 2012</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kollers <em>et al.</em> 2013</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Letta <em>et al.</em> 2013</a>).</p><p id="p-10">In this study, we evaluated 1000 accessions from the US Department of Agriculture Agricultural Research Service (USDA-ARS) National Small Grains Collection (NSGC) spring wheat core collection for resistance against <em>Pst</em>. Resistance was evaluated both at the seedling stage in controlled environments (to four specific <em>Pst</em> races) and at the adult plant stage in multiple years and field locations in the western United States (to mixtures of naturally occurring <em>Pst</em> races). We found evidence for 10 high-confidence associations that were consistent across locations and compared their chromosome locations with previously mapped <em>Pst</em> resistance genes and QTL. We also identified the colinear regions in the rice and <em>Brachypodium</em> genomes to accelerate the identification of additional markers for these QTL regions.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials</h3><p id="p-11">One thousand accessions were selected randomly from the 2235 <em>Triticum aestivum</em> ssp. <em>aestivum</em> accessions available in the spring wheat core collection assembled by the USDA-ARS Small Grains and Potato Germplasm Research Unit. This core collection was assembled using passport and phenotypic data and includes accessions from the different wheat producing regions of the world. Accessions from 89 countries were represented in this study, including accessions from South America (20.9%), Africa (20.6%), Europe (19.6%), Asia (29.1%), North America (7.0%), and Australia (2.8%). Seeds and DNAs used in this study were obtained from single plant selections increased in a nursery grown at the USDA-ARS Small Grains and Potato Germplasm Research Unit, Aberdeen, Idaho. The genetic characterization of these 1000 accessions [see <em>SNP genotyping</em> of <em>Material and Methods</em> below] resulted in the selection of 875 nonredundant accessions for the GWAS analysis, which are listed in <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS1.xlsx">File S1</a>.</p></div><div id="sec-3" class="subsection"><h3>Stripe rust response evaluation: adult-plant field conditions</h3><p id="p-12">Accessions were evaluated under natural disease epidemics in six field trials performed at three locations: Mount Vernon (48° 25′ 12′′N 122° 19′ 34′′W) in the western side of Washington state, Pullman (46° 43’ 59′′ N 117° 10′ 00′′W) in the eastern side of Washington state, and Davis (38° 33′ 14′′N 121° 44′ 17′′W) in northern California. Trials in Davis and Pullman were performed in 2011 and 2012, and those in Mount Vernon in 2012 and 2013. The different year-location combinations are referred hereafter as “environments.” Planting was in mid-November in Davis and mid-April for both Mount Vernon and Pullman. Oversummering and overwintering of stripe rust can occur in both Pacific Northwest and California, facilitating local recurrent epidemics (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Sharma-Poudyal <em>et al.</em> 2013</a>). <em>Pst</em> inoculum often is shared between California, Arizona, New Mexico, and Northwestern Mexico. Additionally, <em>Pst</em> oversummering in northeastern California may constitute a source of inoculum for central California (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kolmer <em>et al.</em> 2009</a>) and for the Pacific Northwest (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Chen 2005</a>).</p><p id="p-13">In all field trials, accessions were evaluated as nonreplicated single rows. Rows were 2.0 m long with 0.40 m spacing between rows in experiments at Davis, and 1.0 m long with 0.25 m spacing at both Mount Vernon and Pullman. The susceptible checks used were “D6301,” planted every six rows in Davis, and “Lemhi,” planted every 20 rows in Washington. The same susceptible checks also were planted as spreader rows bordering the nurseries to ensure production of sufficient inoculum to provide uniform stripe rust infection.</p><p id="p-14">Stripe rust response was evaluated twice during the mid- to advanced-phases of disease development to limit the number of escapes. These evaluations were performed between plant heading (Zadoks 50) and grain filling stage (Zadoks 80), when most flag leaves of the susceptible checks displayed a disease severity of at least 50%. Only the evaluation showing the highest average disease pressure between the two stages (usually the second one) was used in the GWAS analysis. The infection type (IT) was scored using a 0−9 scale described previously (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Line and Qayoum 1992</a>). Disease severity (SEV) was scored as percentage of infected leaf area. Additionally, days to heading and plant height were evaluated to test their correlation with <em>Pst</em> resistance. Other morphophysiological traits such as occurrence of pseudo-black chaff, awns, wax, and glume hairiness were recorded as controls for line identification.</p></div><div id="sec-4" class="subsection"><h3>Stripe rust response evaluation: single-race seedling test</h3><p id="p-15">Seedlings of the 875 accessions were evaluated for IT response to four <em>Pst</em> races, PSTv-4, PSTv-14, PSTv-37, and PSTv-40 (<a id="xref-ref-85-2" class="xref-bibr" href="#ref-85">Wan and Chen 2014</a>), under controlled greenhouse conditions. The <em>Pst</em> races were maintained at the USDA-ARS, Washington State University. PSTv-37 is a predominant and widely distributed race in the United States, whereas PSTv-4, PSTv-14, and PSTv-40 are predominantly found in the Pacific Northwest and California. The virulence/avirulence formulas of the four races are described in <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS1.pdf">Table S1</a> (<a id="xref-ref-85-3" class="xref-bibr" href="#ref-85">Wan and Chen 2014</a>).</p><p id="p-16">Three seeds of each accession were planted per well in a 96-well tray containing Sunshine mix growing medium (SunGro Horticulture, Agawam, MA). Lines of the stripe rust differential set and the stripe rust-susceptible “Avocet S” were seeded in a separate 96-well tray. The seeded trays were placed in a rust-free greenhouse at 20° with 50% relative humidity. Seedlings were watered daily, and the gibberellin inhibitor (2-chloroethyl trimethylammonium chloride; Ohp, Inc, PA) was used at a concentration of 1500 ppm to slow down the seedling growth rate. Trays of 11-d-old seedlings were inoculated with urediniospores of each race mixed with talc. The inoculated seedling trays were placed in a dark dew chamber overnight at 10° with 100% relative humidity for 20 hr. After the incubation, seedlings were transferred to a greenhouse with a diurnal temperature cycle programmed to change gradually from 20° at 2:00 <span class="sc">pm</span> to 4° at 2:00 <span class="sc">am</span>. Day/night regimes of 16-hr light and 8-hr darkness were maintained throughout the experiment. ITs were scored 18−20 d after inoculation when the rust was developed fully on the susceptible checks. ITs were scored using a 0−9 scale (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">McNeal <em>et al.</em> 1971</a>) similar to that used for the adult plants under field conditions. Accessions with resistant to moderately resistant IT scores of 0−6 were retested with each respective <em>Pst</em> race.</p></div><div id="sec-5" class="subsection"><h3>SNP genotyping</h3><p id="p-17">For each accession, genomic DNA was extracted from the same plant used to increase the seeds evaluated in this study. DNA extractions were performed at the USDA-ARS Small Grains and Potato Germplasm Research Unit using the CTAB protocol (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Stewart and Via 1993</a>). The DNA was precipitated by adding isopropanol, followed by washing of the pellet with ice-cold 70% ethanol, and resuspension in 200 µL of Tris HCl ethylenediaminetetraacetic acid (pH 8.0).</p><p id="p-18">Genotyping was carried out at the USDA-ARS genotyping laboratory at Fargo, North Dakota, using the Infinium wheat SNP 9K iSelect assay from the Illumina platform (Illumina Inc., San Diego, CA) developed by the International Wheat SNP Consortium (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cavanagh <em>et al.</em> 2013</a>). The raw Illumina SNP data were processed with the GenomeStudio v2011.1 software (Illumina). The array yielded 5234 scorable SNP markers. The polymorphic SNPs were ordered according to the scaled map positions of the hexaploid wheat 9K SNP consensus map (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cavanagh <em>et al.</em> 2013</a>). The arm orientation of chromosomes 4A, 5A, and 5B presented here is in opposite orientation to the published consensus map (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Cavanagh <em>et al.</em> 2013</a>).</p><p id="p-19">The dataset was filtered using a 10% cutoff for missing data in either loci or accessions (23 accessions were eliminated). On the basis of the filtered SNP data, a triangular identity-by-state genetic similarity matrix (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Kang <em>et al.</em> 2008</a>) was then obtained for all possible pairs of accessions. For groups of accessions with ≥0.99 genetic similarity, only one representative accession (the one with the lowest number of missing data) was retained per group. After applying these filtering criteria, a total of 875 accessions were retained for the GWAS. Only SNPs with minor allele frequency (MAF) ≥0.10 (<em>i.e.</em>, minor allele present in at least 87 accessions) were considered for GWAS. Of the 4585 SNPs that satisfied this criterion, 4374 were positioned on the consensus map. Low-frequency SNPs were discarded to focus on SNPs with greater statistical power (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Turner <em>et al.</em> 2011</a>). The downside of this approach is the potential loss of true resistance loci present at low frequency (increase in false negatives). In this study we prioritized the reliability of the detected QTL over the sensitivity of the analyses.</p><p id="p-20">Molecular markers tightly associated to two well-characterized loci conferring resistance to multiple pathogens were included as internal controls. The diagnostic <em>KaspLr34</em> assay (<a href="http://www.cerealsdb.uk.net/cerealgenomics/CerealsDB/SNPs/Documents/MAS">http://www.cerealsdb.uk.net/cerealgenomics/CerealsDB/SNPs/Documents/MAS</a>, = wMAS000003) was designed around a 3-bp indel in exon 11 of the <em>Lr34/Yr18</em> gene (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lagudah <em>et al.</em> 2009</a>). Marker <em>csSNP856</em> (=<em>Kasp856</em>) is tightly linked to the <em>Lr67/Yr46</em> locus but is not diagnostic for the resistance gene (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Forrest <em>et al.</em> 2014</a>). Data for these two control markers are summarized in <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS3.xlsx">File S3</a>.</p></div><div id="sec-6" class="subsection"><h3>Population structure and genetic diversity</h3><p id="p-21">The mapped SNP data, together with the map information from <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Cavanagh <em>et al.</em> (2013)</a>, were analyzed with HAPLOVIEW v4.2 (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barrett <em>et al.</em> 2005</a>) using the tagger function <em>r</em><sup>2</sup> = 1.0 to define a set of 3114 nonredundant SNPs for calculation of the kinship matrix. A subset of 1036 highly informative, nonredundant representative SNPs (tagSNPs) also were selected with HAPLOVIEW using the tagger function <em>r</em><sup>2</sup> = 0.25. The tagSNPs were used for population structure analysis using a combination of distance- and model-based clustering analysis. Distance-based cluster analysis was carried out using the Ward clustering algorithm in R (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">R Core Team 2013</a>) as implemented in “stats” package (hclust). The model-based quantitative assessment of subpopulation memberships of the accessions was carried out in STRUCTURE v 2.3.4 (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Pritchard <em>et al.</em> 2000</a>) using inferences based on molecular data only and admixture model of population structure with correlated allele frequencies. Numbers of hypothetical subpopulations ranging from <em>k =</em>1 to 10 were assessed using 50,000 burn-in iterations followed by 100,000 recorded Markov-Chain iterations. To estimate the sampling variance (robustness) of population structure inference, five independent runs were carried out for each <em>k</em>. The output from STRUCTURE was analyzed in STRUCTURE HARVESTER (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Earl and Vonholdt 2012</a>).</p><p id="p-22">The <em>Δk</em> statistics based on the rate of change in the logarithm of the probability of likelihood [<em>LnP(D)</em>] value between successive <em>k</em> values (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Evanno <em>et al.</em> 2005</a>) was used to predict the optimum number of subpopulations. On the basis of the final <em>k</em> values (four groups and seven subgroups, respectively), two Q-matrices (875 × k) including the corresponding population membership coefficients were obtained.</p><p id="p-23">To determine the level of differentiation among subpopulations, we calculated the fixation index (<em>Fst</em>) among all possible pairwise combinations of the seven subpopulations using the software Arlequin v. 3.5 (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Excoffier <em>et al.</em> 2005</a>). For this calculation we used all 4585 SNPs and used 1000 permutations at <em>P =</em> 0.001. We also calculated the gene diversity (D) of each of the 7 subpopulations using the 4585 SNPs with published formulas (<a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Weir 1996</a>).</p></div><div id="sec-7" class="subsection"><h3>LD analysis</h3><p id="p-24">HAPLOVIEW was used to obtain the LD squared allele frequency correlation (<em>r</em><sup>2</sup>) estimates for all pairwise comparisons between intrachromosomal SNPs and to visualize the local LD patterns. To analyze the overall pattern of LD decay over genetic distances, syntenic pairwise LD <em>r</em><sup>2</sup> estimates from all chromosomes were plotted <em>vs.</em> the corresponding pairwise genetic distances and a nonlinear regression model was fitted in R based on the equation relating LD, recombination rate, and population size (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Sved 1971</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Rexroad and Vallejo 2009</a>). The curve fitting parameter <em>α</em> was set to 1 (no mutations). Nonlinear fitting of the model was carried out using the nonlinear least squares method in R. The map distance at which LD fell below the <em>r</em><sup>2</sup> thresholds of 0.3 was used to define the confidence intervals of QTL detected in this study. This is a frequently used LD threshold for QTL detection (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Ardlie <em>et al.</em> 2002</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Shifman <em>et al.</em> 2003</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Khatkar <em>et al.</em> 2008</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lawrence <em>et al.</em> 2009</a>).</p></div><div id="sec-8" class="subsection"><h3>Phenotypic data analysis</h3><p id="p-25">To improve normality of the original phenotypic data, logit, square root, arcsine, and log transformation methods were tested. For IT, the data were square root transformed [IT<sub>transf</sub> = √ (IT + 0.25)/10)] and for SEV the data were arcsine-square root transformed [SEV<sub>transf</sub> = arcsine (√ 1-(SEV+0.25)/100)]. The Shapiro-Wilk normality test confirmed the improved normality of the transformed data relative to the original data. Even though the correlations between normal scores and best linear unbiased estimates (BLUE) values from all six locations (BLUE-all) calculated from the transformed data were relatively high (IT W = 0.98 and SEV W = 0.99), departures from normality were still significant.</p><p id="p-26">The transformed data were subjected to combined analyses of variance (ANOVA) over environments using a mixed linear model procedure including genotype, environment and genotype by environment interactions as random factors. Broad sense heritability (<em>H<sup>2</sup></em>) estimates (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) were calculated using the Restricted Maximum Likelihood method (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Corbeil and Searle 1976</a>). BLUE values were obtained across locations and years considering genotypes as a fixed effect in the model. BLUE values were then used to perform GWAS.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465735/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465735/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Means and ranges for response to <em>Puccinia striiformis</em> f. sp. <em>tritici</em> of 875 spring wheat accessions from the NSGC in six environments (three locations × two years)</span><div class="sb-div caption-clear"></div></div></div><p id="p-29">Pearson correlation coefficients among environments were calculated for IT and SEV values to evaluate the consistency of the resistance responses. Correlations between field-based <em>Pst</em> resistance responses and both heading date and plant height were calculated to investigate the influence of these traits on resistance. The proportion of variation (<em>R<sup>2</sup></em>) in <em>Pst</em> response across accessions explained by population structure was calculated using multiple regression of single environment- and adjusted mean-phenotypes on the quantitative Q-7 STRUCTURE membership coefficient matrix.</p></div><div id="sec-9" class="subsection"><h3>Association analysis</h3><p id="p-30">GWAS for loci governing <em>Pst</em> response in the filtered set of 875 accessions was performed using 4585 informative SNPs and the compressed mixed linear model (<a id="xref-ref-91-2" class="xref-bibr" href="#ref-91">Yu <em>et al.</em> 2006</a>; <a id="xref-ref-93-1" class="xref-bibr" href="#ref-93">Zhang <em>et al.</em> 2010</a>) implemented in the R package GAPIT (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lipka <em>et al.</em> 2012</a>). Association tests were carried out for: (1) all single environment data sets, (2) BLUEs across experiments (years) for each location, and (3) BLUEs across all six environments.</p><p id="p-31">Different association test models were compared using the Bayesian information criterion calculated using GAPIT for IT and SEV data from all environments (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS2.pdf">Table S2</a>). The following models were tested: (1) fixed general linear model with no correction for population structure, (2) general linear model models corrected for population structure using the first 10 eigenvectors from principal component analysis (PC10), percent membership coefficients based on STRUCTURE (Q4 and Q7), and qualitative assignment to Ward clusters (W4 and W7, where 4 indicates the four main groups and seven the subpopulations), (3) mixed linear model with the 875 × 875 kinship matrix (K), and (4) all possible combinations of K and the other five methods. Models were also evaluated using plots of observed <em>vs.</em> expected cumulative <em>P</em> values based on the IT data from Davis 2011 (<a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Stich <em>et al.</em> 2008</a>) (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FigureS1.pdf">Figure S1</a>). All mixed linear models including K performed similar to each other and better than models without K (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS2.pdf">Table S2</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FigureS1.pdf">Figure S1</a>). In addition, the K mixed linear model without additional corrections showed the best Bayesian information criterion value for both IT and SEV across all environments (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS2.pdf">Table S2</a>) and was selected for all GWAS analyses. Association probability values were estimated and both marker-wise and experiment-wise thresholds were selected for different analyses. The selected experiment-wise threshold was based on the Bonferroni-corrected method with <em>α</em> = 0.10, which is roughly equivalent to a marker-wise threshold <em>P</em> value = 1 × 10<sup>−4</sup> (9.65 × 10<sup>−5</sup> for 1036 tag-SNPs).</p><p id="p-32">QTL that were significant (<em>P</em> ≤ 0.05, marker-wise) in at least three of the six environments for IT and/or SEV, with at least one environment with highly-significant differences (<em>P</em> ≤ 0.01, marker-wise) were first selected to identify <em>Pst</em> resistance loci with broad-spectrum resistance effective across multiple locations. The same QTL selection criteria were applied to a second GWAS that included only accessions with BLUE across all locations (BLUE-all) IT scores greater than 3. The objective of this second analysis, which included 593 accessions, was to find QTL for partial resistance that might have been masked by the presence of major resistance genes. The 97 QTL that passed the previous two selection criteria for IT or SEV are summarized in the Supplemental Information, <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS4.pdf">Table S4</a>, <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS5.pdf">Table S5</a>, <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FigureS2.pdf">Figure S2</a>, and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FigureS3.pdf">Figure S3</a>. Among these 97 QTL, only 10 were significant for the Bonferroni test (experiment-wise <em>P</em> ≤ 0.10) and are discussed in detail in this study. The cumulative effect of the 10 significant QTL and their interactions was estimated using an ANOVA model including population structure (Q7) as covariate.</p><p id="p-33">Multiple colocating and/or adjacent SNPs in the consensus map significantly associated with the disease response were assigned to a single QTL when the LD <em>r</em><sup>2</sup> values among markers were ≥ 0.3, inter-marker distances were within the QTL confidence interval, and the SNPs showed consistent direction of the effects. The SNPs with <em>P</em> values indicating the strongest association was considered as the QTL-representative marker (henceforth, QTL-tagging SNP).</p></div><div id="sec-10" class="subsection"><h3>Enrichment of markers in the QTL regions and preliminary annotation</h3><p id="p-34">To increase the number of markers in each QTL region, the consensus map from the Illumina 90K SNP assay (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Wang <em>et al.</em> 2014</a>) was projected onto the 9K SNP consensus map (<a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Cavanagh <em>et al.</em> 2013</a>), which was used as reference map in BIOMERCATOR v4.2 (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Sosnowski <em>et al.</em> 2012</a>). Flanking sequences of all the 9K and 90K Illumina SNPs that mapped within the QTL confidence intervals (± 1.6 cM) were then used to BLAST <em>T. turgidum</em> (Kronos) transcriptome, <em>T. urartu</em> transcriptome (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Krasileva <em>et al.</em> 2013</a>), and <em>A. tauschii</em> cDNA database (<a href="http://plants.ensembl.org/Aegilops_tauschii/Info/Index">http://plants.ensembl.org/Aegilops_tauschii/Info/Index</a>) to obtain longer query sequences for colinearity analysis. Sequence hits with alignment length more than 99% of the query and more than 98% identity were used as query in BLASTX searches of <em>Brachypodium</em> (<em>Brachypodium distachyon</em> (L.) P. Beauv) and rice (<em>Oryza sativa</em> L.) databases using Gramene (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Monaco <em>et al.</em> 2014</a>) (<a href="http://www.gramene.org">http://www.gramene.org</a>). Annotations for the orthologous <em>Brachypodium</em> and rice genes were retrieved from Phytozome v9.1 (<a href="http://www.phytozome.net/">http://www.phytozome.net/</a>). Results from protein domain databases (pFAM domain descriptions), Protein ANalysis THrough Evolutionary Relationships, EuKaryotic Orthologous Groups description, and Gene Ontology terms are summarized in <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS2.xlsx">File S2</a>. Based on these annotations, we identified <em>Brachypodium</em> and rice <em>R</em> genes, which encode proteins that recognize pathogen effectors or their modified host targets. These proteins are characterized by the presence of a variable leucine-rich repeat (LRR) and include CC-NB-LRR (coiled-coil domain, nucleotide binding site, LRR), RLP (receptor-like proteins coupled with extracellular LRR), and RLK (kinase domain coupled with LRR) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chisholm <em>et al.</em> 2006</a>).</p></div><div id="sec-11" class="subsection"><h3>Comparison of QTL locations with previously reported <em>Yr</em> genes and QTL</h3><p id="p-35">For comparison with previous studies, the 10 experiment-wise significant QTL identified in the GWAS, 56 named <em>Yr</em> genes (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">McIntosh <em>et al.</em> 2013</a>) (catalogue of gene symbols for wheat: 2013−2014 Supplement; <a href="http://wheat.pw.usda.gov/GG2/Triticum/wgc/2013/2013-2014_Supplement.pdf">http://wheat.pw.usda.gov/GG2/Triticum/wgc/2013/2013-2014_Supplement.pdf</a>) and 169 previously mapped QTL were projected onto a common integrated map including different marker types (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS4.xlsx">File S4</a>).</p><p id="p-36">The integrated map was produced extending the iterative map compilation process described in the previous section (“iterative maps compilation” tool in Biomercator v4.2 (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Sosnowski <em>et al.</em> 2012</a>)). The process started with the 9K SNP consensus map (<a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Cavanagh <em>et al.</em> 2013</a>), followed by the sequential projection of the 90K SNP consensus map (<a id="xref-ref-88-2" class="xref-bibr" href="#ref-88">Wang <em>et al.</em> 2014</a>), the tetraploid consensus map (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Maccaferri <em>et al.</em> 2014</a>), the Synthetic × Opata DH GBS map (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Saintenac <em>et al.</em> 2013</a>), the Diversity Array Technology integrated map (<a href="http://www.diversityarrays.com/search/node/Wheat%20DArT%20map">http://www.diversityarrays.com/search/node/Wheat%20DArT%20map</a>), the 2004 SSR consensus map (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Somers <em>et al.</em> 2004</a>), and the Synthetic × Opata ITMI BARC SSR map (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Song <em>et al.</em> 2005</a>). The software option for automatic resolution of common markers inversions was used.</p><p id="p-37">For the named <em>Pst</em> resistance genes the closest flanking markers were used to generate confidence intervals that are reported in <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS5.pdf">File S5</a>. Distances in cM were converted into relative % length distances by dividing them by total chromosome length, and were then projected onto schematic chromosomes. Confidence intervals for published QTL were calculated using Darvasi and Soller prediction formulas (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Darvasi and Soller 1997</a>). QTL identified in this GWAS were projected on the same compiled map, using confidence intervals of ± 1.6 cM (where LD was predicted to fall below the critical levels of <em>r</em><sup>2</sup> = 0.3).</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Spring wheat panel composition and population genetic structure</h3><p id="p-38">Based on genetic profiles of the Illumina iSelect 9K SNP array, 875 nonredundant genotypes with less than 10% missing data were identified among the 1000 spring <em>T. aestivum</em> ssp. <em>aestivum</em> accessions initially selected from the NSGC core wheat collection. These 875 accessions (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS1.xlsx">File S1</a>) included 172 landraces, 255 registered cultivars, 299 breeding lines, 6 genetic stocks, and 143 unclassified accessions. The accessions originated from 87 countries on six continents and are representative of the diverse wheat growing areas in the world (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The characterization of these 875 accessions using a set of nonredundant genome-wide tagSNPs and a combination of hierarchical Ward clustering and quantitative population structure model-based Bayesian clustering revealed the presence of four main groups. Groups 1 and 4 were further subdivided into two and three subgroups, respectively, giving a total of seven subgroups that reflect the population structure of these accessions (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/449/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Geographic distribution of seven subpopulations identified in the analysis of the population structure of 875 accessions from the National Small Grains Collection spring wheat core collection. The large pie chart indicates the relative number of accessions in the seven subpopulations. The smaller pie charts indicate their relative distribution in specific countries (the size of these pie charts is proportional to the number of accessions from that country)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1339148383" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Geographic distribution of seven subpopulations identified in the analysis of the population structure of 875 accessions from the National Small Grains Collection spring wheat core collection. The large pie chart indicates the relative number of accessions in the seven subpopulations. The smaller pie charts indicate their relative distribution in specific countries (the size of these pie charts is proportional to the number of accessions from that country).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/449/F1.medium.gif" width="440" height="213"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/3/449/F1.medium.gif" width="440" height="213"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/449/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/449/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465665" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-39" class="first-child">Geographic distribution of seven subpopulations identified in the analysis of the population structure of 875 accessions from the National Small Grains Collection spring wheat core collection. The large pie chart indicates the relative number of accessions in the seven subpopulations. The smaller pie charts indicate their relative distribution in specific countries (the size of these pie charts is proportional to the number of accessions from that country).</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/449/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population structure and its relationship to Puccinia striiformis f. sp. tritici (Pst) resistance. (A) Ward&#x2019;s clustering of 875 accessions from the NSGC spring wheat core collection. Vertical lines indicate genetic similarity thresholds used to classify accessions into 4 main groups (dashed lines) and 7 subgroups (dotted lines). (B) 875 &#xD7; 875 kinship matrix based on simple matching genetic similarities (IBS, identity by state). Separations among Ward&#x2019;s based groups are shown as horizontal dashed lines for main groups and as dotted-dashed lines for subgroups. (C) Matrices of membership coefficients of accessions corresponding to 4&#x2212;7 hypothetical subpopulations derived from the STRUCTURE analysis (D) Response of accessions to Pst (IT, infection types, SEV, disease severity). Blue lines indicate Pst resistance and red lines Pst susceptibility (based on the best linear unbiased estimates over six environments). (E) Percentage memberships of accessions from the seven subpopulations to the four main continents." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1339148383" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Population structure and its relationship to &lt;em&gt;Puccinia striiformis&lt;/em&gt; f. sp. &lt;em&gt;tritici&lt;/em&gt; (&lt;em&gt;Pst&lt;/em&gt;) resistance. (A) Ward&#x2019;s clustering of 875 accessions from the NSGC spring wheat core collection. Vertical lines indicate genetic similarity thresholds used to classify accessions into 4 main groups (dashed lines) and 7 subgroups (dotted lines). (B) 875 &#xD7; 875 kinship matrix based on simple matching genetic similarities (IBS, identity by state). Separations among Ward&#x2019;s based groups are shown as horizontal dashed lines for main groups and as dotted-dashed lines for subgroups. (C) Matrices of membership coefficients of accessions corresponding to 4&#x2212;7 hypothetical subpopulations derived from the STRUCTURE analysis (D) Response of accessions to &lt;em&gt;Pst&lt;/em&gt; (IT, infection types, SEV, disease severity). Blue lines indicate &lt;em&gt;Pst&lt;/em&gt; resistance and red lines &lt;em&gt;Pst&lt;/em&gt; susceptibility (based on the best linear unbiased estimates over six environments). (E) Percentage memberships of accessions from the seven subpopulations to the four main continents.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/449/F2.medium.gif" width="440" height="297"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/3/449/F2.medium.gif" width="440" height="297"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/449/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/449/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465669" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-40" class="first-child">Population structure and its relationship to <em>Puccinia striiformis</em> f. sp. <em>tritici</em> (<em>Pst</em>) resistance. (A) Ward’s clustering of 875 accessions from the NSGC spring wheat core collection. Vertical lines indicate genetic similarity thresholds used to classify accessions into 4 main groups (dashed lines) and 7 subgroups (dotted lines). (B) 875 × 875 kinship matrix based on simple matching genetic similarities (IBS, identity by state). Separations among Ward’s based groups are shown as horizontal dashed lines for main groups and as dotted-dashed lines for subgroups. (C) Matrices of membership coefficients of accessions corresponding to 4−7 hypothetical subpopulations derived from the STRUCTURE analysis (D) Response of accessions to <em>Pst</em> (IT, infection types, SEV, disease severity). Blue lines indicate <em>Pst</em> resistance and red lines <em>Pst</em> susceptibility (based on the best linear unbiased estimates over six environments). (E) Percentage memberships of accessions from the seven subpopulations to the four main continents.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">This population structure showed some association with the geographic distribution of the accessions present in this panel (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Group 1A was well represented in Mexico, in the Middle East (Egypt, Israel, and Syria), and in the eastern and southern regions of Africa (Kenya, Zimbabwe, and South Africa). Group 2 was predominant in accessions from Russia, Kazakhstan, Northern, and Eastern Europe as well as Canada and United States. Group 3 was frequent in Mediterranean countries in Europe and Northern Africa, Australia, and South America, where spring wheats are predominantly planted in the fall to better exploit winter precipitations. Group 4 was predominant in countries from Asia and is further subdivided into three distinct subgroups. Subgroup 4A was predominant among landraces and cultivated materials from Iran, Kazakhstan, Afghanistan, Tajikistan, and Pakistan on one side, and Taiwan and Japan on the other side. Subgroup 4B was found mainly in Western Asia (Iran, Armenia, and Turkey) and Saudi Arabia, whereas subgroup 4C was predominant in countries from Southern Asia (India, Nepal, and Pakistan).</p><p id="p-42">The results from Ward’s clustering, the genotypic similarities, and the subgroup membership coefficients based on STRUCTURE are shown in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>. The STRUCTURE membership coefficients revealed a high degree of admixture in a large number of accessions, particularly among cultivars and modern breeding materials (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). This result is consistent with the complex network of germplasm exchange that characterizes modern wheat breeding worldwide. It is important to point out that the proportions of subgroups presented in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> and in the clusters in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, represent the proportions of accessions in the NSGC spring wheat core collection, but may not necessarily represent the current composition of cultivars in a specific country or group.</p></div><div id="sec-14" class="subsection"><h3>Spring wheat panel response to <em>Pst</em></h3><p id="p-43">The response of the 875 selected accessions to <em>Pst</em> was evaluated in six environments (three locations × two years per location) characterized by very high infection levels of <em>Pst</em>. Phenotypic variation was observed in all environments, with infection types ranging from highly resistant (13% of accessions with IT 0−2) to highly susceptible (17% of accessions with IT 7−9). The frequency distributions of BLUE-all values for IT and SEV were approximately normal, with a slight shift in the IT response toward resistance (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, A and D</a>). The square root and arcsine transformations used in this study improved the normality of the data (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, B and E</a>). The analysis of variance (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>) showed that the variance components for genotype were highly significant (<em>P</em> ≤ 0.0001) for both the individual locations and the combined analysis across the six environments. By contrast, the variance components for environment were not significant across all analyses (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). The variance component for the genotype by environment interaction was significant only in Davis (<em>P</em> ≤ 0.01 for IT and <em>P</em> ≤ 0.05 for SEV). Consistent with the previous results, heritability values for IT and SEV were high for the individual locations (0.72 to 0.87, <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) and even greater for BLUE values across all locations (<em>H<sup>2</sup></em> for IT = 0.89 and for SEV = 0.91, <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/449/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distributions of infection type (IT) and disease severity (SEV). Distributions of best linear unbiased estimates (BLUEs) across all six environments for IT (A&#x2013;B) and SEV (D&#x2013;E). &#x201C;W&#x201D; indicates the correlation between observed values and normal scores for the original (A and D) and transformed (B and E) values. (C and F) Boxplot showing differences among subpopulations for IT (C) and SEV (F)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1339148383" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distributions of infection type (IT) and disease severity (SEV). Distributions of best linear unbiased estimates (BLUEs) across all six environments for IT (A&#x2013;B) and SEV (D&#x2013;E). &#x201C;W&#x201D; indicates the correlation between observed values and normal scores for the original (A and D) and transformed (B and E) values. (C and F) Boxplot showing differences among subpopulations for IT (C) and SEV (F).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/449/F3.medium.gif" width="287" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/3/449/F3.medium.gif" width="287" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/449/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/449/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465738" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-44" class="first-child">Distributions of infection type (IT) and disease severity (SEV). Distributions of best linear unbiased estimates (BLUEs) across all six environments for IT (A–B) and SEV (D–E). “W” indicates the correlation between observed values and normal scores for the original (A and D) and transformed (B and E) values. (C and F) Boxplot showing differences among subpopulations for IT (C) and SEV (F).</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">The high IT and SEV heritability values indicate limited environmental variation relative to genotypic variation, an observation supported also by high and significant correlations among environments. Pearson correlation coefficients for IT averaged 0.73 ± 0.03 within locations and 0.65 ± 0.01 across locations (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Very similar correlations among environments were observed for SEV (<em>R</em> = 0.74 ± 0.05 within locations and 0.66 ± 0.01 across locations, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). As expected, average correlations between SEV and IT values were the greatest within the same environment (<em>R</em> = 0.83 ± 0.02), intermediate within the same location in different years (<em>R</em> = 0.70 ± 0.03), and the lowest among locations (<em>R</em> = 0.63 ± 0.01, <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). The correlation between IT and SEV BLUE-all values was very high (<em>R</em> = 0.93, <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS3.pdf">Table S3</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465703/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465703/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Pearson’s correlation coefficients for IT and SEV response to <em>Puccinia striiformis</em> f. sp. <em>tritici</em> of 875 NSGC spring wheat accessions in six environments</span><div class="sb-div caption-clear"></div></div></div><p id="p-48">The extent to which the <em>Pst</em> response of the 875 different accessions was influenced by population structure is presented in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, C and F</a>. In <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2D</a> the IT and SEV values of each accession are presented as heat-maps with blue indicating resistance and red susceptibility. A greater proportion of resistant accessions was evident in group 4C, whereas a greater proportion of susceptible accessions was observed in group 4A. A more balanced distribution of resistant and susceptible accessions was observed for the other subpopulations (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2D</a>). A similar trend was observed in the box plots for IT (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>) and SEV (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3F</a>). An ANOVA among subpopulations showed significantly greater IT and SEV averages in subpopulation 4C and significantly lower averages in subpopulation 4A (Tukey’s test <em>P</em> &lt; 0.05, except for one nonsignificant SEV difference between subpopulations 4A and 4B). The association between population structure and <em>Pst</em> resistance also was reflected in significant multiple regressions (<em>P</em> ≤ 0.01) between BLUE-all values and population structure (Q7) for both IT (<em>R<sup>2</sup>=</em>0.141) and SEV (<em>R<sup>2</sup>=</em>0.166). These significant associations indicated that corrections for population structure were required for the GWAS. Comparison of different population structure correction methods (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FigureS1.pdf">Figure S1</a> and the section <em>Material and Methods</em>) indicated that the use of the kinship matrix (K) was the most informative option for this dataset.</p><p id="p-49">Corrections for plant height and heading time were evaluated to determine whether they influenced the GWAS results for <em>Pst</em> resistance. Both traits are critical for wheat performance in the field and are known to have multiple pleiotropic effects. However, in this study negligible correlations were observed between <em>Pst</em> resistance (IT and SEV BLUE values across all six environments) and both heading time (<em>R</em><sup>2</sup> &lt; 0.005) and plant height (<em>R</em><sup>2</sup> &lt; 0.05). Therefore, neither of these traits was included as covariate in the GWAS for <em>Pst</em> resistance. None of the high-confidence QTL selected in this study (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>) showed significant association with plant height or heading time. However, among the lower-confidence QTL, significant associations were detected for one heading time QTL (<em>IWA8513</em>) and five plant height QTL (<em>IWA692</em>, <em>IWA1923</em>, <em>IWA4347</em>, <em>IWA6630</em>, and <em>IWA3796</em>), which were excluded from <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS4.pdf">Table S4</a>.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465688/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465688/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Loci significantly associated with IT or SEV in at least three environments (one at <em>P</em> &lt; 0.01) and with experiment-wise Bonferroni <em>P</em> &lt; 0.10</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Spring wheat panel LD estimation</h3><p id="p-57">The extent of LD and the average trend of LD decay rate in this association panel were estimated based on pairwise LD squared correlation coefficients (<em>r</em><sup>2</sup>) for all intrachromosomal SNP loci. The trend of LD decay was described by a nonlinear regression of the pair-wise <em>r</em><sup>2</sup> values on the corresponding map distances based on the Illumina 9K SNP consensus map (<a id="xref-ref-9-9" class="xref-bibr" href="#ref-9">Cavanagh <em>et al.</em> 2013</a>) and by a box-plot diagram of LD <em>r</em><sup>2</sup> distribution (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Among the pairs of markers that were completely linked in the consensus map the median LD <em>r</em><sup>2</sup> was 0.69 (inter-quartiles ranging from 0.32 to 0.98). In the next class, including noncompletely linked markers separated by less than 1 cM, the median LD <em>r</em><sup>2</sup> decreased to 0.3. Thus, on average, a 50% LD decay rate was observed within 1 cM genetic distance. For the pairs of markers linked at 1−5 cM the median <em>r</em><sup>2</sup> decreased to 0.12.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/449/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide average linkage disequilibrium (LD) decay over genetic distances. (A) Plot of pair-wise single-nucleotide polymorphism LD r2 values as a function of intermarker map distance (cM) based on a reference consensus map (Cavanagh et al. 2013). The red curve represents the model fit to LD decay. The light-blue dashed line represents the &#xB1;1.6 cM confidence interval for the quantitative trait loci regions in which LD r2 = 0.3. (B) Boxplot showing LD r2 values for incremental classes of single-nucleotide polymorphism pairwise map distances." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1339148383" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome-wide average linkage disequilibrium (LD) decay over genetic distances. (A) Plot of pair-wise single-nucleotide polymorphism LD &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; values as a function of intermarker map distance (cM) based on a reference consensus map (Cavanagh &lt;em&gt;et al.&lt;/em&gt; 2013). The red curve represents the model fit to LD decay. The light-blue dashed line represents the &#xB1;1.6 cM confidence interval for the quantitative trait loci regions in which LD &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; = 0.3. (B) Boxplot showing LD &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; values for incremental classes of single-nucleotide polymorphism pairwise map distances.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/449/F4.medium.gif" width="440" height="176"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/3/449/F4.medium.gif" width="440" height="176"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/449/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/449/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465684" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-58" class="first-child">Genome-wide average linkage disequilibrium (LD) decay over genetic distances. (A) Plot of pair-wise single-nucleotide polymorphism LD <em>r</em><sup>2</sup> values as a function of intermarker map distance (cM) based on a reference consensus map (<a id="xref-ref-9-10" class="xref-bibr" href="#ref-9">Cavanagh <em>et al.</em> 2013</a>). The red curve represents the model fit to LD decay. The light-blue dashed line represents the ±1.6 cM confidence interval for the quantitative trait loci regions in which LD <em>r</em><sup>2</sup> = 0.3. (B) Boxplot showing LD <em>r</em><sup>2</sup> values for incremental classes of single-nucleotide polymorphism pairwise map distances.</p><div class="sb-div caption-clear"></div></div></div><p id="p-59">Similar results were obtained by fitting a nonlinear regression equation (<a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Sved 1971</a>). Based on the fitted model, LD was predicted to fall below the nominal critical levels of <em>r</em><sup>2</sup> = 0.3 at an inter-marker genetic distance of 1.6 cM (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). This 1.6 cM distance at each side of the peak of the significant associations was used to establish confidence intervals for the QTL-harboring regions.</p></div><div id="sec-16" class="subsection"><h3>Association analyses for <em>Pst</em> resistance across environments</h3><p id="p-60">Association analysis was performed separately for each of the six field environments to identify chromosome regions including <em>Pst</em> resistance genes effective against different combinations of <em>Pst</em> races. A total of 73 chromosome regions showed marker-wise significant associations with either IT or SEV in at least three environments (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FigureS2.pdf">Figure S2</a>). Among these QTL only seven were significant at the experiment-wise Bonferroni threshold of <em>P</em> &lt; 0.10 and are described in <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>.</p><p id="p-61">To test whether additional partial resistance QTL were masked by the major resistance genes segregating in this panel, we performed a second set of GWAS in which 282 accessions with highly resistant infection types (ITs of 0−3) were not included in the analyses. GWAS using the remaining 593 accessions with IT &gt; 3 revealed 35 QTL showing marker-wise significant associations with either IT or SEV in at least three environments (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS5.pdf">Table S5</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FigureS3.pdf">Figure S3</a>). Eleven of these 35 QTL overlapped with the previous set of 73 and 24 were new (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS5.pdf">Table S5</a>). Seven of the 35 QTL were significant at the experiment-wise Bonferroni threshold (<em>P</em> &lt; 0.10). Among these seven, four overlapped with the ones identified in the previous GWAS (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS4.pdf">Table S4</a>), and three (<em>IWA3892</em>, <em>IWA980</em>, and <em>IWA1034</em>) were new and were added to <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>.</p><p id="p-62">Detailed information for the 10 significant loci is presented in <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>, which includes the representative SNP, the suggested name for the QTL, and the probabilities of association with IT and SEV values from the six individual environments and BLUE values. <a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a> also includes the favorable allele and its frequency in the complete association panel and in each of the seven subpopulations. Five of the 10 significant loci had closely linked SNPs (&lt; 1.6 cM and in LD to each other) that also showed significant associations with <em>Pst</em> resistance (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS5.pdf">Table S5</a>), providing additional support for the association between that chromosome region and resistance to <em>Pst</em>. Two of these QTL are described in more detail in <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>. In this figure, the significance of the phenotype-SNP association (as –log <em>P</em>) is plotted along the consensus map (<a id="xref-ref-9-11" class="xref-bibr" href="#ref-9">Cavanagh <em>et al.</em> 2013</a>) and compared with local <em>r</em><sup>2</sup> LD patterns below. SNP-phenotype associations rapidly decayed within 1-2 cM, in accordance with the observed trend of LD decay rate estimated from SNP pairs (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/449/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="P value association plots and corresponding linkage disequilibrium (LD) r2 patterns for two significant Pst resistance quantitative trait loci (QTL). The upper part of the graph shows P value plots of marker-trait associations for best linear unbiased estimates (BLUEs) of infection types (IT) and disease severity (SEV) over six environments and for the three specific locations (MTV, Mount Vernon; PLM, Pullman; DVS, Davis). Map distances (X-axis) are from the 9K SNP consensus map (Cavanagh et al. 2013). &#x2013;Log (P) significance thresholds are reported using dashed lines. Single-nucleotide polymorphism (SNP) codes and corresponding local LD r2 value patterns are in the lower part of the graph. Numbers within the diamonds of the triangular LD matrix are the r2 values multiplied by 100." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1339148383" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;P&lt;/em&gt; value association plots and corresponding linkage disequilibrium (LD) &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; patterns for two significant &lt;em&gt;Pst&lt;/em&gt; resistance quantitative trait loci (QTL). The upper part of the graph shows &lt;em&gt;P&lt;/em&gt; value plots of marker-trait associations for best linear unbiased estimates (BLUEs) of infection types (IT) and disease severity (SEV) over six environments and for the three specific locations (MTV, Mount Vernon; PLM, Pullman; DVS, Davis). Map distances (X-axis) are from the 9K SNP consensus map (Cavanagh &lt;em&gt;et al.&lt;/em&gt; 2013). &#x2013;Log (&lt;em&gt;P&lt;/em&gt;) significance thresholds are reported using dashed lines. Single-nucleotide polymorphism (SNP) codes and corresponding local LD &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; value patterns are in the lower part of the graph. Numbers within the diamonds of the triangular LD matrix are the &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; values multiplied by 100.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/449/F5.medium.gif" width="440" height="283"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/3/449/F5.medium.gif" width="440" height="283"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/449/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/449/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465705" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5 </span> <p id="p-63" class="first-child"><em>P</em> value association plots and corresponding linkage disequilibrium (LD) <em>r</em><sup>2</sup> patterns for two significant <em>Pst</em> resistance quantitative trait loci (QTL). The upper part of the graph shows <em>P</em> value plots of marker-trait associations for best linear unbiased estimates (BLUEs) of infection types (IT) and disease severity (SEV) over six environments and for the three specific locations (MTV, Mount Vernon; PLM, Pullman; DVS, Davis). Map distances (X-axis) are from the 9K SNP consensus map (<a id="xref-ref-9-12" class="xref-bibr" href="#ref-9">Cavanagh <em>et al.</em> 2013</a>). –Log (<em>P</em>) significance thresholds are reported using dashed lines. Single-nucleotide polymorphism (SNP) codes and corresponding local LD <em>r</em><sup>2</sup> value patterns are in the lower part of the graph. Numbers within the diamonds of the triangular LD matrix are the <em>r<sup>2</sup></em> values multiplied by 100.</p><div class="sb-div caption-clear"></div></div></div><p id="p-64"><a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a> also indicates which of the 10 significant QTL identified in the field studies were also significantly associated with resistance to the four individual <em>Pst</em> races tested at the seedling stage in controlled environments. Five of the 10 loci were significant for both the adult plant (field studies) and seedling tests (controlled environment). A similar proportion was observed among the lower-confidence QTL (47%; <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS4.pdf">Table S4</a>). QTL identified in the second GWAS for partial resistance showed a lower proportion of loci that were significant in the seedling tests (30%; <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS5.pdf">Table S5</a>). The levels of significance of the adult plant and seedling QTL were generally similar, except for <em>IWA4280</em> that showed exceptionally strong associations in the seedling tests (<em>P</em> &lt; 10<sup>−24</sup>) but only moderate associations in the field experiments (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS4.pdf">Table S4</a>).</p><p id="p-65">The frequencies of favorable alleles in the different subpopulations for the 10 significant QTL-tagging SNPs identified from the multienvironmental trials are also summarized in <a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>. Similar frequency information is also available for lower-confidence QTL that were significant in at least three environments (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS6.pdf">Table S6</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS7.pdf">Table S7</a>). Favorable alleles for QTL associated with markers <em>IWA167</em>, <em>IWA5375</em>, and <em>IWA6988</em> were present at high frequency in subpopulation 4C (Pakistan-India-Nepal) and at relatively low frequencies in most of the other subpopulations. The favorable allele for <em>IWA422</em> was present at relatively high frequency among the three Asian subpopulations (4A, 4B, and 4C) and at lower frequencies in other subpopulations. The opposite profile was observed for <em>IWA7257</em> that was present at very low frequency in the three Asian subpopulations and higher frequencies in the other subpopulations. Subpopulation 4C showed a relatively higher proportion of favorable alleles with very high or very low frequencies, a trend associated with the lowest genetic diversity of subpopulation 4C among the seven subpopulations (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS6.pdf">Table S6</a>).</p><p id="p-66">The proportion of phenotypic variance explained by the individual QTL for IT and SEV BLUE values across all environments ranged from 0.4 to 2.2% (<a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>). When these 10 QTL were analyzed together in an ANOVA including population structure as covariates (Q7), they explained 15% and 12% of the observed variance for IT and SEV BLUEs across all environments, respectively (excluding variation explained by the population structure; <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS8.pdf">Table S8</a>). When the six significant interactions among loci were added to the model, the proportion of explained phenotypic variation (excluding population structure) increased to 19% for IT and to 16% for SEV BLUE values across all environments (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS8.pdf">Table S8</a>). Interestingly, when the interactions were added to the model, the highly significant effects of <em>IWA422</em> and <em>IWA5202</em> became non-significant. These two markers were involved in four of the six significant interactions that are presented in <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FigureS5.pdf">Figure S5</a>. In general, the presence of the resistance allele from one locus resulted in a reduced effect for the interacting locus (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FigureS5.pdf">Figure S5</a>, A−E). However, in the <em>IWA6988</em> × <em>IWA167</em> interaction, the presence of the susceptible alleles from one of these two loci was associated with a reduction or elimination of the effect of the interacting locus (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FigureS5.pdf">Figure S5</a>F).</p><p id="p-67">Two markers tightly linked to known <em>Pst</em> resistance genes on chromosome arms 7DS (<em>Lr34/Yr18/Pm38</em>) and 4DL (<em>Lr67/Yr46</em>) were used as controls. The scoring of these two markers, their linkage with other SNPs, and their association with <em>Pst</em> resistance traits are described in supplemental <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS3.xlsx">File S3</a>. Diagnostic marker <em>KaspLr34</em> showed a MAF of 17.8% (resistant allele), which is higher than the 10% threshold selected in this study. The <em>KaspLr34</em> marker was significantly associated to IT and SEV responses for three of the six environments (DVS_2012, MTV_2012, MTV_2013), and adjusted averages across all environments were significant experiment-wise (Bonferroni &lt; 0.10). <em>KaspLr34</em> did not show appreciable LD (maximum <em>r</em><sup>2</sup> = 0.04, <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS3.xlsx">File S3</a>) with any of the 22 polymorphic SNPs mapped on chromosome 7D.</p><p id="p-68">Marker <em>Kasp856</em>, which is tightly linked to <em>Lr67/Yr46</em> (chromosome 4DL), showed a MAF of 8.6%, and therefore, it would have been excluded from our GWAS. This marker was strongly associated to <em>Pst</em> severity values in DVS-2011, DVS-2012, and MTV-2013, and also showed experiment-wise significant associations to the adjusted averages across all environments. <em>Kasp856</em> was in LD to <em>IWA5375</em> (<em>r</em><sup>2</sup> value = 0.41, MAF = 10%; <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS3.xlsx">File S3</a>), which was found in our GWAS to be significantly associated to <em>Pst</em> resistance. Markers <em>IWA6277</em>, <em>IWA5381</em>, <em>IWA5766</em>, <em>IWA5375</em>, and <em>Kasp856</em> defined three main haplotypes within a 6.3 cM region. The haplotype G-A-G-T-A at these five markers was associated to <em>Pst</em> resistance, supporting the results obtained from the single SNP markers (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS3.xlsx">File S3</a>).</p></div><div id="sec-17" class="subsection"><h3>Relationship between number of favorable alleles and <em>Pst</em> resistance</h3><p id="p-69">Considering all QTL-tagging SNPs with marker-wise significant effects in at least three environments (97 in total, including the 24 for partial resistance), the number of favorable alleles present in a specific accession ranged from 23 to 65. The genetic profiles of the accessions for the 97 QTL-tagging SNPs are reported in <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS1.xlsx">File S1</a>, where accessions were ranked based on the number of favorable alleles. The 87 accessions (10%) with the highest number of favorable alleles (favorable alleles = 56.6 ± 0.3) showed significantly lower (<em>P</em> &lt; 0.0001) IT (2.3 ± 0.1) and SEV (22.5% ± 1.8%) values than the 87 accessions with the lowest number of favorable alleles (favorable alleles = 34.7 ± 0.3, IT = 6.8 ± 0.2, SEV = 79.0 ± 2.0). These significant associations were also reflected in highly significant correlations (<em>P</em> &lt; 0.0001) between the number of favorable alleles and both IT (<em>R =</em> 0.68) and SEV (<em>R =</em> 0.67) values. This result indicates that the variation in the number of favorable alleles in these 97 QTL explains 45–46% of the variation in <em>Pst</em> resistance in this germplasm collection, excluding variation explained by population structure (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FigureS4.pdf">Figure S4</a>). Thus, this dataset may be a powerful tool for genomic predictions of stripe rust resistance.</p></div><div id="sec-18" class="subsection"><h3>Comparison of significant QTL with rice and <em>Brachypodium</em> genomes</h3><p id="p-70">To facilitate the identification of additional markers and to accelerate the discovery of potential candidate genes, we established the colinearity between confidence intervals for the 10 QTL identified in this study and the sequenced <em>Brachypodium</em> and rice genomes (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS2.xlsx">File S2</a>). As a first step, we increased the number of markers in the QTL-confidence intervals by projecting the wheat SNP consensus map generated from the Illumina 90K assay (including 40,269 SNPs) onto the reference 9K Illumina consensus map using the program BIOMERCATOR v4.2 (<a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">Sosnowski <em>et al.</em> 2012</a>). On average, 12 SNP markers from the 9K assay and 80 projected from the 90K assay (total 92) were detected per QTL confidence interval (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS10.pdf">Table S10</a>). Sequences of the original and projected wheat SNP markers were then used to find the best hits in <em>Brachypodium</em> and rice.</p><p id="p-71">The <em>Brachypodium</em> and rice synteny relationships for the 10 QTL confidence intervals are described in detail in <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS2.xlsx">File S2</a>. On average, 60 <em>Brachypodium</em> and rice colinear annotated genes were identified per confidence interval (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS10.pdf">Table S10</a>). Based on the Phytozome v9.1 database (<a href="http://www.phytozome.net/">http://www.phytozome.net/</a>) 12.7% of the annotated colinear genes were classified as <em>R</em> genes (CC-NB-LRR, RLP and RLK), with higher proportions found in the confidence intervals for QTL <em>IWA3892</em>, <em>IWA422</em>, <em>IWA5202</em>, and <em>IWA1034</em> (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS10.pdf">Table S10</a>).</p></div><div id="sec-19" class="subsection"><h3>Comparison of significant QTL with known <em>Pst</em> resistance genes</h3><p id="p-72">To identify which of the 10 QTL-tagging SNPs described in <a id="xref-table-wrap-3-9" class="xref-table" href="#T3">Table 3</a> mapped on regions similar to 227 previously identified <em>Pst</em> resistance genes (<em>Yr</em>) and QTL, we projected both sets of markers on an integrated map including different types of markers (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS4.xlsx">File S4</a>). <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows the projection of these resistance loci onto standardized chromosomes of similar length. The 10 highly-significant QTL identified in this study are presented to the left of the chromosomes and previously mapped <em>Pst</em> resistance genes and QTL to the right. The numbers on top of the QTL (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, blue bars) indicate the reference used to determine the confidence interval for the QTL. These references, together with the confidence intervals (in % length) are summarized in <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS5.pdf">File S5</a>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/449/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosome positions of Pst-associated quantitative trait loci (QTL) identified in this study (experiment-wise Bonferroni P &lt; 0.1) relative to previously mapped Pst resistance genes and QTL. Chromosome lengths were all standardized to the same relative length. QTL-tagging single-nucleotide polymorphisms identified in this genome-wide association (GWAS) are on the left side of the chromosomes. Those with a red line were significant only in the overall GWAS, those with a blue line only in the GWAS for partial resistance, and those with a purple line were significant in both (Table 3). Previously mapped Pst resistance genes (green) and QTL (blue) are on the right side of the chromosomes. The ID numbers on top of the QTL indicate references and confidence intervals provided in File S5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1339148383" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Chromosome positions of &lt;em&gt;Pst&lt;/em&gt;-associated quantitative trait loci (QTL) identified in this study (experiment-wise Bonferroni &lt;em&gt;P&lt;/em&gt; &lt; 0.1) relative to previously mapped &lt;em&gt;Pst&lt;/em&gt; resistance genes and QTL. Chromosome lengths were all standardized to the same relative length. QTL-tagging single-nucleotide polymorphisms identified in this genome-wide association (GWAS) are on the left side of the chromosomes. Those with a red line were significant only in the overall GWAS, those with a blue line only in the GWAS for partial resistance, and those with a purple line were significant in both (Table 3). Previously mapped &lt;em&gt;Pst&lt;/em&gt; resistance genes (green) and QTL (blue) are on the right side of the chromosomes. The ID numbers on top of the QTL indicate references and confidence intervals provided in File S5.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/449/F6.medium.gif" width="440" height="292"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/3/449/F6.medium.gif" width="440" height="292"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/449/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/449/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465754" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-73" class="first-child">Chromosome positions of <em>Pst</em>-associated quantitative trait loci (QTL) identified in this study (experiment-wise Bonferroni <em>P</em> &lt; 0.1) relative to previously mapped <em>Pst</em> resistance genes and QTL. Chromosome lengths were all standardized to the same relative length. QTL-tagging single-nucleotide polymorphisms identified in this genome-wide association (GWAS) are on the left side of the chromosomes. Those with a red line were significant only in the overall GWAS, those with a blue line only in the GWAS for partial resistance, and those with a purple line were significant in both (<a id="xref-table-wrap-3-10" class="xref-table" href="#T3">Table 3</a>). Previously mapped <em>Pst</em> resistance genes (green) and QTL (blue) are on the right side of the chromosomes. The ID numbers on top of the QTL indicate references and confidence intervals provided in <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS5.pdf">File S5</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-74">Three of the 10 QTL-tagging markers reported in <a id="xref-table-wrap-3-11" class="xref-table" href="#T3">Table 3</a> (<em>IWA980</em>, <em>IWA424</em>, and <em>IWA7257</em>) were mapped far from any currently known <em>Pst</em> resistance gene or QTL, and likely represent novel resistance loci (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS5.pdf">File S5</a>). <em>IWA167</em> overlapped with <em>QYr.ufs-6D</em> (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6</a>), but the peaks of these two QTL were mapped on different chromosome arms and showed effects of different magnitude (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS5.pdf">File S5</a>), suggesting that they likely represent different genes.</p><p id="p-75">The other six QTL-tagging markers were found in the proximity (less than one tenth of the chromosome length) of named <em>Yr</em> resistance genes or previously mapped <em>Pst</em> resistance QTL (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a>). <em>IWA422</em>, <em>IWA5202</em>, and <em>IWA5375</em> were mapped less than 3 cM from previously mapped <em>Pst</em> resistance genes (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS5.pdf">File S5</a>), suggesting that they may represent alleles of the same genes. <em>IWA422</em> and <em>IWA5202</em> were mapped to the distal regions of chromosome arms 2AS and 3BS respectively, which include several previously mapped <em>Pst</em> resistance genes and QTL that can correspond to these 2 QTL (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS5.pdf">File S5</a>). <em>IWA5375</em> was mapped in the proximity of the <em>Lr67/Yr46</em> region, and likely represents the same locus since no other <em>Pst</em> resistance genes or QTL were mapped in their proximity.</p><p id="p-76">The other three QTL-tagging markers (<em>IWA3892</em>, <em>IWA1034</em>, and <em>IWA6988</em>) were mapped within the confidence intervals of previously mapped <em>Pst</em> resistance genes or QTL (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS5.pdf">File S5</a>). However, for each of these three QTL we found a second closely linked SNP that showed marker-wise significant associations with <em>Pst</em> resistance in at least three of the locations evaluated in this study (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS4.pdf">Table S4</a>, gray letters in parenthesis in <a id="xref-fig-6-9" class="xref-fig" href="#F6">Figure 6</a>). The relationship between each of these three pairs of linked SNP with previously mapped <em>Pst</em> resistance genes or QTL is discussed in detail in supplemental <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS5.pdf">File S5</a>.</p></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><div id="sec-21" class="subsection"><h3>Population genetic structure</h3><p id="p-77">The population structure of 1000 accessions from the spring wheat core collection was used as covariate in the GWAS mainly to reduce the number of false associations (<a id="xref-ref-91-3" class="xref-bibr" href="#ref-91">Yu <em>et al.</em> 2006</a>), but it also provided additional valuable information. First, the genetic characterization of the 1,000 spring lines revealed the existence of ~100 near-identical lines (&gt;99% identical) in this panel, which were eliminated from our analyses (together with another 25 with &gt;10% missing data). The elimination of the near-identical lines from the NSGC spring wheat core collection can reduce the size of the collection without a significant loss of genetic diversity.</p><p id="p-78">In addition, the genetic characterization of these accessions organized the spring wheat core collection into seven subpopulations of genetically related accessions (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figures 1</a> and <a id="xref-fig-2-8" class="xref-fig" href="#F2">2</a>). The Ward clustering analysis (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a>) revealed a major division between the subpopulations enriched in accessions from Asia (4A, 4B, and 4C, henceforth “Asian” subpopulations) and subpopulations that include a large proportion of accessions from other parts of the world (1A, 1B, 2, and 3, henceforth “Western” subpopulations). This division was also observed in a study of SNP diversity in the D-genome (<a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Wang <em>et al.</em> 2013</a>), and may reflect the ancestral eastward and westward expansion of agriculture (and of domesticated <em>T. aestivum</em>) after its initial domestication in the Fertile Crescent roughly 10,000 years ago (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Dubcovsky and Dvorak 2007</a>).</p><p id="p-79">This major division was further supported by pairwise comparisons among subpopulations using <em>Fst</em> (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS9.pdf">Table S9</a>), which provides a measure of population differentiation due to genetic structure (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Holsinger and Weir 2009</a>). Pairwise comparisons among the four “Western” subpopulations showed smaller <em>Fst</em> values than comparisons between the “Western” and “Asian” subpopulations, or among the “Asian” subpopulations (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS9.pdf">Table S9</a>). Among the “Asian” subpopulations, group 4C was the most divergent (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS9.pdf">Table S9</a>). This subpopulation is the smallest (44 accessions) and the less diverse one, with an average polymorphism information content that is roughly half of the averages observed in the other six subpopulations (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS6.pdf">Table S6</a>).</p><p id="p-80">The “Western” subpopulations include a large proportion of materials released after 1960 (55%), which were likely influenced by the Green Revolution. CIMMYT (Centro Internacional de Mejoramiento de Maíz y Trigo) played a central role in the development and distribution of these Green Revolution varieties, and accounts for 90% of the accessions from Mexico presented in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>. CIMMYT lines in this collection belong mainly to the four “Western” subpopulations, which may explain the abundance of these subpopulations in regions that frequently receive or exchange germplasm with CIMMYT. Many accessions included in the “Western” subpopulations show evidence of high levels of admixture (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2</a>), which likely reflects the frequent germplasm exchanges among wheat breeding programs from these regions. A greater level of admixture among spring wheat varieties than among winter wheat varieties has been reported also in a different germplasm panel (<a id="xref-ref-9-13" class="xref-bibr" href="#ref-9">Cavanagh <em>et al.</em> 2013</a>).</p></div><div id="sec-22" class="subsection"><h3>Association between population genetic structure and <em>Pst</em> resistance</h3><p id="p-81">Subpopulations1A, 1B, 2, and 3 show a uniform distribution of <em>Pst</em> IT and SEV values (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3, C and F</a>), possibly an additional reflection of the extensive admixture observed in these populations. By contrast, a significant association between population structure and <em>Pst</em> response was observed for the three subpopulations from Asia (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3</a>). Accessions from India, Nepal, and Pakistan in subpopulation 4C displayed a greater proportion of moderately resistant to resistant phenotypes than any of the other six subpopulations (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3</a>). Interestingly, a recent study of the ancestral relationships among worldwide populations of <em>Pst</em> has pointed to the same Himalayan and neighboring region as the putative center of origin for <em>Pst</em>. This hypothesis was supported by the existence of high levels of diversity, presence of private alleles, clear signatures of recombination, and ability to produce sex-related structures in the <em>Pst</em> races from this region (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Ali <em>et al.</em> 2014</a>). Archeological remains indicate that hexaploid wheat was already cultivated in India-Pakistan between 4000 and 2000 BC (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Tengberg 1999</a>), which suggests that wheat populations from this region may have the longest history of interactions with <em>Pst</em>.</p><p id="p-82">The ancient <em>T. aestivum</em> L. ssp. <em>sphaerococcum</em> (Percival) Mac Key (synonym: <em>T. sphaerococcum</em> Percival) endemic to southern Pakistan and northwestern India was described in the early 1920s as an early flowering semidwarf wheat with semispherical grains and with resistance to yellow rust (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Percival 1921</a>). This description indicates that sources of resistance to <em>Pst</em> had already evolved in this region before the introduction of modern wheat varieties. Several of the <em>Pst</em> resistance QTL identified in this study (<em>IWA167</em>, <em>IWA6988</em>, <em>IWA5375</em>, and <em>IWA422</em>) are present at higher frequencies in subpopulation 4C than in any of the other subpopulations (<a id="xref-table-wrap-3-12" class="xref-table" href="#T3">Table 3</a>) and may represent valuable alleles to enrich the “Western” subpopulation with novel or infrequent resistance alleles.</p><p id="p-83">By contrast, subpopulation 4A showed the lowest levels of resistance (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3</a>), suggesting that regions in which varieties from this subpopulation are grown may be at a greater risk of <em>Pst</em> epidemics. Regions where subpopulation 4A is at high frequency (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>) may benefit from the incorporation of resistance alleles identified in this study that are absent or at very low frequency in the 4A subpopulation (<a id="xref-table-wrap-3-13" class="xref-table" href="#T3">Table 3</a>, <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FigureS6.pdf">Figure S6</a>, and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FigureS7.pdf">Figure S7</a>). The observed heterogeneity of <em>Pst</em> resistance levels among subpopulations (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3, C and F</a>) poses additional challenges for the discovery of real associations by GWAS. Adjustment of the GWAS analysis for population structure can reduce this problem, as demonstrated in previous studies in rice (<a id="xref-ref-94-2" class="xref-bibr" href="#ref-94">Zhao <em>et al.</em> 2011</a>) and maize (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Van Inghelandt <em>et al.</em> 2012</a>), where similar levels of associations between phenotypes and population structure were reported.</p><p id="p-84">The benefits of correcting for population structure are partially offset by an increase in false negatives. Some real associations that are highly correlated with the population structure can be lost in GWAS analyses corrected for population structure. Similarly, the increased protection for the identification of false positives achieved in this study by the elimination of SNPs with MAFs lower than 0.1 is offset by the inability to detect real resistance genes with low allele frequencies. In this initial study, we favored the more stringent criteria. However, alternative analyses can be performed using this dataset, which is publicly available through the T3 database (<a href="http://triticeaetoolbox.org/">http://triticeaetoolbox.org/</a>) and the ARS-GRIN system (<a href="http://www.ars-grin.gov/">http://www.ars-grin.gov/</a>).</p></div><div id="sec-23" class="subsection"><h3>Significant associations in the GWAS</h3><p id="p-85">To identify new sources of resistance to <em>Pst</em> that were effective in different environments of the western United States, we performed field evaluations in three locations with very different ecological conditions. Despite these differences, we observed high correlations among IT and SEV values obtained from the different environments (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). These high correlations suggest that there might be similar <em>Pst</em> populations across the western United States. This hypothesis is supported by the known paths of spore dispersal by wind (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Chen 2005</a>) and by periodic spore surveys across this region. In the last published <em>Pst</em> race survey from 2010 (<a id="xref-ref-85-4" class="xref-bibr" href="#ref-85">Wan and Chen 2014</a>), 20% of the races detected in California and Washington were shared between the two states (PSTv-8, PSTv-14, PSTv-36, PSTv-37, PSTv-40, and PSTv-41), providing further support to the previous hypothesis. The high correlations among environments were also reflected in high heritability for IT and SEV values (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>), which were favorable for the identification of significant associations in the GWAS analyses.</p><p id="p-86">Even with the high heritability values observed in this study, only 10 QTL were significant at the experiment-wise Bonferroni threshold of <em>P</em> &lt; 0.10 selected for this study (<a id="xref-table-wrap-3-14" class="xref-table" href="#T3">Table 3</a>). Among these 10 significant QTL, seven were detected in the GWAS based on all 875 accessions and three in the second GWAS that excluded accessions with high levels of resistance. This suggests that the GWAS for partial resistance provided additional power to detect associations that were somehow masked by the effect of the major resistance genes. A similar proportion of additional loci were detected among the lower confidence QTL for partial resistance (24 of 97, <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS5.pdf">Table S5</a>). Among the SNPs associated with QTL for partial resistance a smaller proportion of loci were also effective at the seedling stage (30%) compared with the proportion observed in the complete GWAS analysis (47%). This is not surprising because many partial resistance genes are effective only at the adult plant stage. In summary, elimination of highly resistant accessions from a GWAS for disease resistance was a useful strategy to identify additional partial resistance genes. The incorporation and deployment of partial resistance genes is an important objective for wheat breeding programs because this type of resistance genes have historically provided more durable resistance than race specific resistance genes (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Johnson 1984</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kolmer 1996</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Santra <em>et al.</em> 2008</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Krattinger <em>et al.</em> 2009</a>; <a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Lowe <em>et al.</em> 2011a</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Chen 2013</a>).</p><p id="p-87">The two markers from known <em>Pst</em> resistance genes <em>Yr18</em> (<em>Lr34/Yr18/Pm38</em>) and <em>Yr46</em> (<em>Lr67/Yr46</em>) included as controls in a separate GWAS, both showed experiment-wise significant associations with <em>Pst</em> resistance (<em>P</em> &lt; 1 e<sup>-4</sup>, <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS4.xlsx">File S4</a>). No linked markers were identified in our study for <em>Yr18</em> suggesting that the 9K iSelect chip did not provide enough coverage in the D genome to detect all <em>Pst</em> resistance loci present in our panel. The <em>Yr46</em> marker was closely linked with markers identified in our study that were significantly associated to <em>Pst</em> resistance. However, the <em>Kasp856</em> marker for <em>Yr46</em> would have been excluded in our original GWAS because its MAF was below our selected 10% MAF threshold. Fortunately, some of the SNPs linked to <em>Kasp856</em> have MAF greater than 10% and the QTL was detected. This result exemplifies the potential loss in sensitivity associated with increased stringency.</p><p id="p-88">The proportion of variation in IT values explained by each of the 10 selected QTL was relatively small (0.4–2.2%, <a id="xref-table-wrap-3-15" class="xref-table" href="#T3">Table 3</a>) and similar to values previously reported in GWAS for other quantitative traits in maize and rice (<a id="xref-ref-94-3" class="xref-bibr" href="#ref-94">Zhao <em>et al.</em> 2011</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Peiffer <em>et al.</em> 2013</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Peiffer <em>et al.</em> 2014</a>). When the 10 QTL were combined in a single ANOVA, 15% of the variation (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS8.pdf">Table S8</a>) in <em>Pst</em> resistance was explained by the model (excluding the contribution of the population structure). This percentage increased to 19% when six significant two-way interactions were added to the ANOVA model (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS8.pdf">Table S8</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FigureS5.pdf">Figure S5</a>). We have initiated the introgression of these loci into the common susceptible background “Avocet S” to test these interactions experimentally.</p><p id="p-89">In this study we also identified 87 QTL that showed significant GWAS associations in at least three environments but that did not pass the stringent experiment-wise Bonferroni threshold. These loci may have a greater proportion of false positives and were, therefore, excluded from the results reported in <a id="xref-table-wrap-3-16" class="xref-table" href="#T3">Table 3</a>. However, the inclusion of all 97 QTL in a combined ANOVA model increased the percent of explained variation in IT BLUE-all values from 15 to 45% (excluding variation explained by population structure). These results suggest that some of the lower-confidence QTL likely represent real associations. As a compromise, we included the 87 lower-confidence QTL in <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS5.pdf">Table S5</a>, as candidates for future validation studies.</p><p id="p-90">The presence of real resistance genes among the lower-confidence QTL is also supported by a high and significant correlation (<em>R</em> = 0.68) between the number of favorable <em>Pst</em> alleles (among the 97 loci) and the level of <em>Pst</em> resistance (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS1.xlsx">File S1</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FigureS4.pdf">Figure S4</a>). This high correlation suggests that accessions carrying a high number of favorable alleles for the different QTL identified in this study may be useful parental lines for breeding programs interested in diversifying their sources of <em>Pst</em> resistance genes. This high correlation also suggests that a genomic selection approach aimed at increasing the number of favorable <em>Pst</em> alleles for the QTL identified in this study is likely to increase the levels of <em>Pst</em> resistance in the breeding populations.</p><p id="p-91">The results discussed above suggest that <em>Pst</em> resistance in the spring wheat core collection was governed by several resistance genes with relatively larger effects (<a id="xref-table-wrap-3-17" class="xref-table" href="#T3">Table 3</a>) modulated by a larger number of genes with smaller effects (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS5.pdf">Table S5</a>). This trait architecture is different from the architecture found for resistance to downy mildew in a previous GWAS in Arabidopsis. The Arabidopsis study, which tested resistance at the seedling level using specific races and in controlled environmental facilities, detected few dominant resistance genes that were concentrated in only four genomic regions (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Nemri <em>et al.</em> 2010</a>). The larger association panel used in our study and the increased statistical power may have contributed to the higher number of QTL identified in this study. This greater number of detected QTL also may be the result of the more complex environmental conditions found in the field relative to the controlled environment. In the field, the frequency of infection and the rate of development of the pathogen can be modulated by different morphological and physiological characteristics of the plant and also by the simultaneous presence of multiple <em>Pst</em> races and other pathogens. In addition, the use of adult plants in our field studies can result in the detection of additional adult plant resistance genes that are not effective at the seedling stage.</p></div><div id="sec-24" class="subsection"><h3>Comparison of significant QTL with colinear genes in rice and <em>Brachypodium</em></h3><p id="p-92">The LD decay rate between the 9K Illumina SNP (50% LD decay rate at 1 cM) in this association panel was faster than that observed in elite US spring wheat (50% LD decay rate at 6.3 cM; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Chao <em>et al.</em> 2010</a>). This is not surprising because this elite US spring wheat panel includes a less diverse germplasm than the NSGC spring wheat core collection analyzed in this study and therefore has a lower level of historical recombination. The intermarker distance at which LD fall below the critical levels of <em>r<sup>2</sup> =</em> 0.3 (1.6 cM) was selected to determine confidence intervals for the 10 significant QTL identified in <a id="xref-table-wrap-3-18" class="xref-table" href="#T3">Table 3</a>.</p><p id="p-93">The projection of the 90K Illumina data (<a id="xref-ref-88-3" class="xref-bibr" href="#ref-88">Wang <em>et al.</em> 2014</a>) onto these 10 QTL confidence intervals resulted in a 7.7-fold increase in the number of SNP per confidence interval relative to the initial results from the 9K chip map. The larger number of markers available per confidence interval (on average 92 markers) facilitated the identification of colinear regions in the sequenced genomes of the grass species <em>Brachypodium</em> and rice (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS2.xlsx">File S2</a>). The annotation of the proteins encoded by the <em>Brachypodium</em> and rice colinear genes was used to infer the putative function of the wheat genes from where the SNPs were derived. Among the wheat SNPs with a corresponding annotated rice or <em>Brachypodium</em> protein (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS10.pdf">Table S10</a>), 12.7% were classified as <em>R</em> genes with two large groups in the confidence intervals for QTL <em>IWA422</em> and <em>IWA5202</em> (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/TableS10.pdf">Table S10</a>). Interestingly, the colinear regions for <em>IWA422</em> (<em>Brachypodium</em> Chr. 5, 1.43−4.20 Mb and rice Chr. 4, 0.07−6.96 Mb) and for <em>IWA5202</em> (<em>Brachypodium</em> Chr. 2, 0.74−0.82 Mb and rice Chr. 4, 0.72−0.92 Mb) include several NB-LRR and LRR-receptor-like kinases. These results suggest that this region may include an ancestral <em>R</em> gene cluster that predates the divergence of the grass subfamilies. It would be interesting to investigate if the multiple <em>Pst</em> resistance genes and QTL mapped to the colinear distal regions of wheat chromosome arms 2AS and 3BS (<a id="xref-fig-6-10" class="xref-fig" href="#F6">Figure 6</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS5.pdf">File S5</a>) are associated with the presence of similar clusters of <em>R</em> genes in wheat.</p><p id="p-94">In summary, these comparative analyses provided a large number of molecular markers for each of the 10 targeted regions, which can be used both to identify haplotypes associated with resistance alleles, and to accelerate the construction of high-density maps for these QTL. In addition, pathogen-response related genes identified in the colinear regions in <em>Brachypodium</em> and rice may provide a starting point for the search of candidate genes in the colinear regions in wheat.</p></div><div id="sec-25" class="subsection"><h3>Comparison of significant QTL with previously mapped genes</h3><p id="p-95">The results presented in <a id="xref-fig-6-11" class="xref-fig" href="#F6">Figure 6</a>, provide an overview of the relationships between the loci identified in this GWAS and 227 previously mapped <em>Pst</em> resistance genes and QTL. However, these results should be considered with caution because of the inherent limitations of consensus maps. The limited number of common markers between previous SSR-based and new SNP-based maps can result in distorted distances in some region of the integrated map. The comparison also is complicated by the low resolution of some of the original maps of <em>Pst</em> resistance genes and by the extended LD in wheat. Therefore, the relationships described in the two paragraphs below and in <a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS5.pdf">File S5</a> should be considered as tentative. For closely linked loci, allelism tests will be required to determine which of the QTL identified here are new resistance loci and which are alleles of previously identified genes.</p><p id="p-96">Among the 10 significant QTL reported in <a id="xref-table-wrap-3-19" class="xref-table" href="#T3">Table 3</a>, <em>IWA980</em>, <em>IWA424</em>, and <em>IWA7257</em> were mapped on chromosome regions where, to our knowledge, no <em>Pst</em> resistance genes or QTL were reported before. These results suggest that they are likely novel <em>Pst</em> resistance loci (<a id="xref-fig-6-12" class="xref-fig" href="#F6">Figure 6</a>). <em>IWA167</em>, one of the most significant QTL detected in this study, was mapped within the flanking markers of a weak QTL from Cappelle Desprez (<em>QYr.ufs-6D</em>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Agenbag <em>et al.</em> 2012</a>). However, the peak of <em>QYr.ufs-6D</em> was mapped on the long arm of chromosome 6D, whereas <em>IWA167</em> is in the short arm. The different arm locations and the different strength of these two QTL suggest that they are likely the effect of different resistance genes (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS5.pdf">File S5</a>). The <em>IWA167</em> resistance allele was almost fixed in the subpopulation from South Asia (frequency = 0.95) and was found at very low frequency in some of the other subpopulations, suggesting that it might be a useful gene to diversify sources of resistance in wheat breeding programs outside South Asia.</p><p id="p-97">QTL associated with <em>IWA422</em>, <em>IWA5202</em>, and <em>IWA5375</em> were very close to previously mapped <em>Pst</em> resistance genes (&lt; 3 cM) and likely represent alleles of these genes (<a href="http://www.g3journal.org/content/suppl/2015/01/20/g3.114.014563.DC1/FileS5.pdf">File S5</a>). The results from the last three significant loci (<em>IWA3892</em>, <em>IWA1034</em>, and <em>IWA6988</em>) are more difficult to interpret because each of them has a linked SNP marker that was also significantly associated (marker-wise) with <em>Pst</em> resistance in most of the locations tested in this study and will require additional allelism tests. In spite of these uncertainties, <a id="xref-fig-6-13" class="xref-fig" href="#F6">Figure 6</a> provides a good overview of the distribution of <em>Pst</em> resistance genes on the wheat chromosomes.</p><p id="p-98">Since 2000, new aggressive strains of <em>Pst</em> were identified in three continents: North America, Australia, and Europe (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hovmoller <em>et al.</em> 2008</a>). These races have increased aggressiveness and the ability to produce more spores at greater temperatures than before (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Milus <em>et al.</em> 2009</a>). These new races with broader spectra of virulences have generated serious stripe rust epidemics in areas previously considered unsuitable for the disease, and have turned stripe rust into one of the most economically damaging wheat pathogens (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hodson 2011</a>). Therefore, the identification of novel <em>Pst</em> resistance genes in this GWAS represents a valuable addition to the set of tools available to wheat breeding programs to fight this devastating pathogen.</p><p id="p-99">This GWAS also provides a good overview of the distribution of resistance genes and the frequencies of the resistance alleles in different wheat subpopulations around the world. The frequency information is particularly useful for wheat breeders interested in diversifying the sources of <em>Pst</em> resistance in their regional programs. In particular, wheat germplasm from the Himalayan and neighboring regions emerged from this study as a valuable source of resistance genes that are absent or present at low frequencies in other regions of the world.</p><p id="p-100">As in other association studies, additional experimental validation will be required to identify which of the accessions carrying favorable SNP alleles actually carry the associated resistance gene. Allelism tests also will be required to determine which of the identified QTL represent novel resistance genes and which ones are alleles of previously mapped genes. To initiate this validation process we have crossed several accessions carrying favorable alleles for the selected QTL with the susceptible variety “Avocet S”, which is the common genetic background in the current single gene differential lines (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Chen <em>et al.</em> 2014</a>). These single QTL introgressions in a common genetic background will greatly simplify the planned allelism tests. In summary, this GWAS study has identified new sources of <em>Pst</em> resistance and provided closely linked markers to accelerate their validation and deployment in wheat breeding programs.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-101">This project was supported by the National Research Initiative Competitive Grant 2011-68002-30029 (Triticeae-CAP) from the USDA National Institute of Food and Agriculture, the Borlaug Global Rust Initiative, the Howard Hughes Medical Institute, and the Gordon and Betty Moore Foundation grant 3031.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-13"><p id="p-102">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.014563/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.014563/-/DC1</a></p></li><li class="fn" id="fn-14"><p id="p-103"><em>Communicating editor: M. Warburton</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-09-18"><span class="received-label">Received </span>September 18, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-01-17"><span class="accepted-label">Accepted </span>January 17, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Maccaferri <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.1" data-doi="10.1094/PHYTO-03-11-0076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adhikari</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gurung</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonman</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Association mapping of quantitative resistance to <em>Phaeosphaeria nodorum</em> in spring wheat landraces from the USDA National Small Grains Collection.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">1301</span>–<span class="cit-lpage">1310</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DAdhikari%26rft.auinit1%253DT.%26rft.volume%253D101%26rft.issue%253D11%26rft.spage%253D1301%26rft.epage%253D1310%26rft.atitle%253DAssociation%2Bmapping%2Bof%2Bquantitative%2Bresistance%2Bto%2BPhaeosphaeria%2Bnodorum%2Bin%2Bspring%2Bwheat%2Blandraces%2Bfrom%2Bthe%2BUSDA%2BNational%2BSmall%2BGrains%2BCollection.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO-03-11-0076%26rft_id%253Dinfo%253Apmid%252F21692647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO-03-11-0076&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21692647&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.2" data-doi="10.1007/s00122-012-1819-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Agenbag</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pretorius</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyd</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bender</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prins</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Identification of adult plant resistance to stripe rust in the wheat cultivar Cappelle-Desprez.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DAgenbag%26rft.auinit1%253DG.%2BM.%26rft.volume%253D125%26rft.issue%253D1%26rft.spage%253D109%26rft.epage%253D120%26rft.atitle%253DIdentification%2Bof%2Badult%2Bplant%2Bresistance%2Bto%2Bstripe%2Brust%2Bin%2Bthe%2Bwheat%2Bcultivar%2BCappelle-Desprez.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1819-5%26rft_id%253Dinfo%253Apmid%252F22350093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1819-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22350093&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.3" data-doi="10.1371/journal.ppat.1003903"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gladieux</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leconte</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gautier</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Justesen</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Origin, migration routes and worldwide population genetic structure of the wheat yellow rust pathogen <em>Puccinia striiformis</em> f.sp. <em>tritici</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">e1003903</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D10%26rft.spage%253De1003903%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003903%26rft_id%253Dinfo%253Apmid%252F24465211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003903&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24465211&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.4" data-doi="10.1038/nrg777"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ardlie</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kruglyak</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seielstad</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Patterns of linkage disequilibrium in the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DArdlie%26rft.auinit1%253DK.%2BG.%26rft.volume%253D3%26rft.issue%253D4%26rft.spage%253D299%26rft.epage%253D309%26rft.atitle%253DPatterns%2Bof%2Blinkage%2Bdisequilibrium%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg777%26rft_id%253Dinfo%253Apmid%252F11967554%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg777&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11967554&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174739800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.5" data-doi="10.1093/bioinformatics/bth457"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fry</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Haploview: analysis and visualization of LD and haplotype maps.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbth457%26rft_id%253Dinfo%253Apmid%252F15297300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyMS8yLzI2MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy80NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.6" data-doi="10.1371/journal.pgen.1000940"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brachi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faure</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flahauw</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Linkage and association mapping of <em>Arabidopsis thaliana</em> flowering time in nature.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000940</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBrachi%26rft.auinit1%253DB.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De1000940%26rft.epage%253De1000940%26rft.atitle%253DLinkage%2Band%2Bassociation%2Bmapping%2Bof%2BArabidopsis%2Bthaliana%2Bflowering%2Btime%2Bin%2Bnature.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000940%26rft_id%253Dinfo%253Apmid%252F20463887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000940&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20463887&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.7" data-doi="10.1186/gb-2011-12-10-232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brachi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association studies in plants: the missing heritability is in the field.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBrachi%26rft.auinit1%253DB.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D232%26rft.epage%253D232%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bin%2Bplants%253A%2Bthe%2Bmissing%2Bheritability%2Bis%2Bin%2Bthe%2Bfield.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-10-232%26rft_id%253Dinfo%253Apmid%252F22035733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-10-232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22035733&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.8" data-doi="10.1534/genetics.105.044586"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breseghello</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Association mapping of kernel size and milling quality in wheat (<em>Triticum aestivum</em> L.) cultivars.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1165</span>–<span class="cit-lpage">1177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.044586%26rft_id%253Dinfo%253Apmid%252F16079235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8yLzExNjUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvNDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.9" data-doi="10.1073/pnas.1217133110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">8057</span>–<span class="cit-lpage">8062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1217133110%26rft_id%253Dinfo%253Apmid%252F23630259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzIwLzgwNTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvNDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.10" data-doi="10.1186/1471-2164-11-727"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubcovsky</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baenziger</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population- and genome-specific patterns of linkage disequilibrium and SNP variation in spring and winter wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">727</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DChao%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.spage%253D727%26rft.epage%253D727%26rft.atitle%253DPopulation-%2Band%2Bgenome-specific%2Bpatterns%2Bof%2Blinkage%2Bdisequilibrium%2Band%2BSNP%2Bvariation%2Bin%2Bspring%2Band%2Bwinter%2Bwheat%2B%2528Triticum%2Baestivum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-727%26rft_id%253Dinfo%253Apmid%252F21190581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-727&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21190581&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.11" data-doi="10.1111/mpp.12116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wellings</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Wheat stripe (yellow) rust caused by <em>Puccinia striiformis</em> f. sp. <em>tritici</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">433</span>–<span class="cit-lpage">446</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%2BPathol.%26rft.volume%253D15%26rft.spage%253D433%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmpp.12116%26rft_id%253Dinfo%253Apmid%252F24373199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mpp.12116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24373199&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.12" data-doi="10.1080/07060660509507230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Epidemiology and control of stripe rust (<em>Puccinia striiformis</em> f. sp <em>tritici</em>) on wheat.</span> <abbr class="cit-jnl-abbrev">Can. J. Plant Pathol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">314</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BPlant%2BPathol.%26rft.volume%253D27%26rft.spage%253D314%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07060660509507230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07060660509507230&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.13" data-doi="10.4236/ajps.2013.43080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High-temperature adult-plant resistance, key for sustainable control of stripe rust.</span> <abbr class="cit-jnl-abbrev">Am. J. Plant Sci.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">608</span>–<span class="cit-lpage">627</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BPlant%2BSci.%26rft.volume%253D4%26rft.spage%253D608%26rft_id%253Dinfo%253Adoi%252F10.4236%252Fajps.2013.43080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4236/ajps.2013.43080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.14" data-doi="10.1080/07060661.2010.499271"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Penman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Virulence races of <em>Puccinia striiformis</em> f. sp <em>tritici</em> in 2006 and 2007 and development of wheat stripe rust and distributions, dynamics, and evolutionary relationships of races from 2000 to 2007 in the United States.</span> <abbr class="cit-jnl-abbrev">Can. J. Plant Pathol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">333</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BPlant%2BPathol.%26rft.volume%253D32%26rft.spage%253D315%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07060661.2010.499271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07060661.2010.499271&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.15" data-doi="10.1016/j.cell.2006.02.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chisholm</span>  <span class="cit-name-given-names">S. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coaker</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Day</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Staskawicz</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Host-microbe interactions: Shaping the evolution of the plant immune response.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">814</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DChisholm%26rft.auinit1%253DS.%2BT.%26rft.volume%253D124%26rft.issue%253D4%26rft.spage%253D803%26rft.epage%253D814%26rft.atitle%253DHost-microbe%2Binteractions%253A%2Bshaping%2Bthe%2Bevolution%2Bof%2Bthe%2Bplant%2Bimmune%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.02.008%26rft_id%253Dinfo%253Apmid%252F16497589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.02.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16497589&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237240900022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.16" data-doi="10.1111/pbr.12015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christopher</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fountain</span>  <span class="cit-name-given-names">M. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification and mapping of adult-plant stripe rust resistance in soft red winter wheat cultivar ’USG 3555’.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D132%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbr.12015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbr.12015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.17" data-doi="10.2307/1267913"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Corbeil</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Searle</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Restricted maximum likelihood (REML) estimation of variance components in mixed model.</span> <abbr class="cit-jnl-abbrev">Technometrics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTechnometrics%26rft.volume%253D18%26rft.spage%253D31%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1267913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1267913&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1976BG16700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.18" data-doi="10.1023/A:1025685324830"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darvasi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soller</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A simple method to calculate resolving power and confidence interval of QTL map location.</span> <abbr class="cit-jnl-abbrev">Behav. Genet.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehavior%2Bgenetics%26rft.stitle%253DBehav%2BGenet%26rft.aulast%253DDarvasi%26rft.auinit1%253DA.%26rft.volume%253D27%26rft.issue%253D2%26rft.spage%253D125%26rft.epage%253D132%26rft.atitle%253DA%2Bsimple%2Bmethod%2Bto%2Bcalculate%2Bresolving%2Bpower%2Band%2Bconfidence%2Binterval%2Bof%2BQTL%2Bmap%2Blocation.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1025685324830%26rft_id%253Dinfo%253Apmid%252F9145551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1025685324830&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9145551&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WX54100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.3.449.19" data-doi="10.1007/BF00220890"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devos</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubcovsky</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chinoy</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gale</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Structural evolution of wheat chromosomes 4A, 5A, and 7B and its impact on recombination.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">282</span>–<span class="cit-lpage">288</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D91%26rft.spage%253D282%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00220890%26rft_id%253Dinfo%253Apmid%252F24169776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00220890&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24169776&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RL40300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dimmock</span>  <span class="cit-name-given-names">J. P. R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gooding</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The influence of foliar diseases, and their control by fungicides, on the protein concentration in wheat grain: a review.</span> <abbr class="cit-jnl-abbrev">J. Agric. Sci.</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BSci.%26rft.volume%253D138%26rft.spage%253D349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.21" data-doi="10.3390/ijms13056167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodig</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zoric</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobiljski</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savic</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kandic</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic and association mapping study of wheat agronomic traits under contrasting water regimes.</span> <abbr class="cit-jnl-abbrev">Int. J. Mol. Sci.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">6167</span>–<span class="cit-lpage">6188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DInt%2BJ%2BMol%2BSci%26rft.aulast%253DDodig%26rft.auinit1%253DD.%26rft.volume%253D13%26rft.issue%253D5%26rft.spage%253D6167%26rft.epage%253D6188%26rft.atitle%253DGenetic%2Band%2Bassociation%2Bmapping%2Bstudy%2Bof%2Bwheat%2Bagronomic%2Btraits%2Bunder%2Bcontrasting%2Bwater%2Bregimes.%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fijms13056167%26rft_id%253Dinfo%253Apmid%252F22754357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/ijms13056167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22754357&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.22" data-doi="10.1126/science.1143986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dubcovsky</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dvorak</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome plasticity a key factor in the success of polyploid wheat under domestication.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">316</span>: <span class="cit-fpage">1862</span>–<span class="cit-lpage">1866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DDubcovsky%26rft.auinit1%253DJ.%26rft.volume%253D316%26rft.issue%253D5833%26rft.spage%253D1862%26rft.epage%253D1866%26rft.atitle%253DGenome%2BPlasticity%2Ba%2BKey%2BFactor%2Bin%2Bthe%2BSuccess%2Bof%2BPolyploid%2BWheat%2BUnder%2BDomestication%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1143986%26rft_id%253Dinfo%253Apmid%252F17600208%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTYvNTgzMy8xODYyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.23" data-doi="10.1007/s12686-011-9548-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Earl</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vonholdt</span>  <span class="cit-name-given-names">B. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">STRUCTURE HARVESTER: a website and program for visualizing STRUCTURE output and implementing the Evanno method.</span> <abbr class="cit-jnl-abbrev">Conserv. Genet. Resour.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DConserv.%2BGenet.%2BResour.%26rft.volume%253D4%26rft.spage%253D359%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12686-011-9548-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12686-011-9548-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.24" data-doi="10.1111/j.1365-294X.2005.02553.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Evanno</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regnaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Detecting the number of clusters of individuals using the software STRUCTURE: a simulation study.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2611</span>–<span class="cit-lpage">2620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DEvanno%26rft.auinit1%253DG.%26rft.volume%253D14%26rft.issue%253D8%26rft.spage%253D2611%26rft.epage%253D2620%26rft.atitle%253DDetecting%2Bthe%2Bnumber%2Bof%2Bclusters%2Bof%2Bindividuals%2Busing%2Bthe%2Bsoftware%2BSTRUCTURE%253A%2Ba%2Bsimulation%2Bstudy.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2005.02553.x%26rft_id%253Dinfo%253Apmid%252F15969739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2005.02553.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15969739&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229961500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Excoffier</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laval</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneider</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Arlequin (version 3.0): an integrated software package for population genetics data analysis.</span> <abbr class="cit-jnl-abbrev">Evol Bioinform</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">47</span>–<span class="cit-lpage">50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol%2BBioinform%26rft.volume%253D1%26rft.spage%253D47%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000207065900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.26" data-doi="10.1146/annurev.arplant.54.031902.134907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornsberry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Structure of linkage disequilibrium in plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">374</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPlant%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.arplant.54.031902.134907%26rft_id%253Dinfo%253Apmid%252F14502995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.arplant.54.031902.134907&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14502995&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185094100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.27" data-doi="10.1007/s11032-014-0166-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pujol</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bulli</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pumphrey</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wellings</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Development of a SNP marker assay for the <em>Lr67</em> gene of wheat using a genotyping by sequencing approach.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">2109</span>–<span class="cit-lpage">2118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D34%26rft.spage%253D2109%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-014-0166-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-014-0166-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.28" data-doi="10.1371/journal.pone.0046444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuillet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balfourier</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Association mapping and haplotype analysis of a 3.1-Mb genomic region involved in Fusarium head blight resistance on wheat chromosome 3BS.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e46444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHao%26rft.auinit1%253DC.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De46444%26rft.epage%253De46444%26rft.atitle%253DAssociation%2BMapping%2Band%2BHaplotype%2BAnalysis%2Bof%2Ba%2B3.1-Mb%2BGenomic%2BRegion%2BInvolved%2Bin%2BFusarium%2BHead%2BBlight%2BResistance%2Bon%2BWheat%2BChromosome%2B3BS.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046444%26rft_id%253Dinfo%253Apmid%252F23071572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046444&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23071572&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.29" data-doi="10.1007/s10681-010-0335-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodson</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Shifting boundaries: challenges for rust monitoring.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">93</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D179%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-010-0335-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-010-0335-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.30" data-doi="10.1038/nrg2611"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holsinger</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetics in geographically structured populations: defining, estimating and interpreting <em>F-<sub>ST</sub></em>.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">650</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHolsinger%26rft.auinit1%253DK.%2BE.%26rft.volume%253D10%26rft.issue%253D9%26rft.spage%253D639%26rft.epage%253D650%26rft.atitle%253DGenetics%2Bin%2Bgeographically%2Bstructured%2Bpopulations%253A%2Bdefining%252C%2Bestimating%2Band%2Binterpreting%2BF%2528ST%2529.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2611%26rft_id%253Dinfo%253Apmid%252F19687804%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2611&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19687804&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269063200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.31" data-doi="10.1111/j.1365-294X.2008.03886.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hovmoller</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yahyaoui</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milus</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Justesen</span>  <span class="cit-name-given-names">A. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid global spread of two aggressive strains of a wheat rust fungus.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">3818</span>–<span class="cit-lpage">3826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DHovmoller%26rft.auinit1%253DM.%2BS.%26rft.volume%253D17%26rft.issue%253D17%26rft.spage%253D3818%26rft.epage%253D3826%26rft.atitle%253DRapid%2Bglobal%2Bspread%2Bof%2Btwo%2Baggressive%2Bstrains%2Bof%2Ba%2Bwheat%2Brust%2Bfungus.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2008.03886.x%26rft_id%253Dinfo%253Apmid%252F18673440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2008.03886.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18673440&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.32" data-doi="10.1038/ng.2673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">G. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhi</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A haplotype map of genomic variations and genome-wide association studies of agronomic traits in foxtail millet (<em>Setaria italica</em>).</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">957</span>–<span class="cit-lpage">961</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D957%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2673%26rft_id%253Dinfo%253Apmid%252F23793027%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23793027&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.33" data-doi="10.1146/annurev.py.22.090184.001521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">A critical analysis of durable resistance.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">309</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DJohnson%26rft.auinit1%253DR%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D309%26rft.epage%253D330%26rft.atitle%253DA%2BCritical%2BAnalysis%2Bof%2BDurable%2BResistance%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.py.22.090184.001521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.py.22.090184.001521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1984TH18900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.34" data-doi="10.1534/genetics.107.080101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zaitlen</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wade</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckerman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient control of population structure in model organism association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1709</span>–<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKang%26rft.auinit1%253DH.%2BM.%26rft.volume%253D178%26rft.issue%253D3%26rft.spage%253D1709%26rft.epage%253D1723%26rft.atitle%253DEfficient%2BControl%2Bof%2BPopulation%2BStructure%2Bin%2BModel%2BOrganism%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080101%26rft_id%253Dinfo%253Apmid%252F18385116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3MDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvNDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.35" data-doi="10.1186/1471-2164-9-187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khatkar</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicholas</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zenger</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Al Cavanagh</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Extent of genome-wide linkage disequilibrium in Australian Holstein-Friesian cattle based on a high-density SNP panel.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">187</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DKhatkar%26rft.auinit1%253DM.%2BS.%26rft.volume%253D9%26rft.spage%253D187%26rft.epage%253D187%26rft.atitle%253DExtent%2Bof%2Bgenome-wide%2Blinkage%2Bdisequilibrium%2Bin%2BAustralian%2BHolstein-Friesian%2Bcattle%2Bbased%2Bon%2Ba%2Bhigh-density%2BSNP%2Bpanel.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-187%26rft_id%253Dinfo%253Apmid%252F18435834%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-187&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18435834&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.36" data-doi="10.1371/journal.pone.0057500"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kollers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodemann</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ling</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ebmeyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole genome association mapping of Fusarium head blight resistance in European winter wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e57500</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De57500%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0057500%26rft_id%253Dinfo%253Apmid%252F23451238%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0057500&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23451238&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.37" data-doi="10.1146/annurev.phyto.34.1.435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kolmer</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Genetics of resistance to wheat leaf rust.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bphytopathology%26rft.stitle%253DAnnu%2BRev%2BPhytopathol%26rft.aulast%253DKolmer%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D435%26rft.epage%253D455%26rft.atitle%253DGENETICS%2BOF%2BRESISTANCE%2BTO%2BWHEAT%2BLEAF%2BRUST%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.phyto.34.1.435%26rft_id%253Dinfo%253Apmid%252F15012551%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.phyto.34.1.435&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15012551&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VG88500023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-5.3.449.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kolmer</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yue</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Diseases which challenge global wheat production-the wheat rusts</span>, pp <span class="cit-fpage">89</span>−<span class="cit-lpage">124</span> in <span class="cit-source">Wheat: Science and Trade</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Carver</span>  <span class="cit-name-given-names">B. F.</span></span>. <span class="cit-publ-name">Wiley-Blackwell</span>, <span class="cit-publ-loc">Ames, IA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.39" data-doi="10.1186/gb-2013-14-6-r66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krasileva</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buffalo</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearce</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayling</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Separating homeologs by phasing in the tetraploid wheat transcriptome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR66%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-6-r66%26rft_id%253Dinfo%253Apmid%252F23800085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-6-r66&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23800085&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.40" data-doi="10.1126/science.1166453"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krattinger</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagudah</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spielmeyer</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Espino</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A putative ABC transporter confers durable resistance to multiple fungal pathogens in wheat.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">1360</span>–<span class="cit-lpage">1363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1166453%26rft_id%253Dinfo%253Apmid%252F19229000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjMvNTkxOS8xMzYwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.41" data-doi="10.1007/s00122-012-1872-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kulwal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ishikawa</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benscher</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Z. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">L. X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Association mapping for pre-harvest sprouting resistance in white winter wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">793</span>–<span class="cit-lpage">805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DKulwal%26rft.auinit1%253DP.%26rft.volume%253D125%26rft.issue%253D4%26rft.spage%253D793%26rft.epage%253D805%26rft.atitle%253DAssociation%2Bmapping%2Bfor%2Bpre-harvest%2Bsprouting%2Bresistance%2Bin%2Bwhite%2Bwinter%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1872-0%26rft_id%253Dinfo%253Apmid%252F22547141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1872-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22547141&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.42" data-doi="10.1007/s00122-009-1097-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lagudah</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krattinger</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herrera-Foessel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Espino</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gene-specific markers for the wheat gene <em>Lr34/Yr18/Pm38</em> which confers resistance to multiple fungal pathogens.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">898</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLagudah%26rft.auinit1%253DE.%2BS.%26rft.volume%253D119%26rft.issue%253D5%26rft.spage%253D889%26rft.epage%253D898%26rft.atitle%253DGene-specific%2Bmarkers%2Bfor%2Bthe%2Bwheat%2Bgene%2BLr34%252FYr18%252FPm38%2Bwhich%2Bconfers%2Bresistance%2Bto%2Bmultiple%2Bfungal%2Bpathogens.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-009-1097-z%26rft_id%253Dinfo%253Apmid%252F19578829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-009-1097-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19578829&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.43" data-doi="10.1371/journal.pgen.1003246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Lessons from <em>Dwarf8</em> on the strengths and weaknesses of structured association mapping.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003246%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003246%26rft_id%253Dinfo%253Apmid%252F23437002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003246&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23437002&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.44" data-doi="10.1186/1471-2105-10-367"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Day-Williams</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broxholme</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cardon</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">GLIDERS - A web-based search engine for genome-wide linkage disequilibrium between HapMap SNPs.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DLawrence%26rft.auinit1%253DR.%26rft.volume%253D10%26rft.spage%253D367%26rft.epage%253D367%26rft.atitle%253DGLIDERS--a%2Bweb-based%2Bsearch%2Bengine%2Bfor%2Bgenome-wide%2Blinkage%2Bdisequilibrium%2Bbetween%2BHapMap%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-10-367%26rft_id%253Dinfo%253Apmid%252F19878600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-10-367&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19878600&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.45" data-doi="10.1007/s00122-013-2050-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Letta</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maccaferri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Badebo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ammar</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ricci</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Searching for novel sources of field resistance to Ug99 and Ethiopian stem rust races in durum wheat via association mapping.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">1237</span>–<span class="cit-lpage">1256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D1237%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2050-8%26rft_id%253Dinfo%253Apmid%252F23429902%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2050-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23429902&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.46" data-doi="10.1146/annurev.phyto.40.020102.111645"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Line</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Stripe rust of wheat and barley in North America: a retrospective historical review.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Phytopathol.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPhytopathol.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.phyto.40.020102.111645%26rft_id%253Dinfo%253Apmid%252F12147755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.phyto.40.020102.111645&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12147755&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000178665700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-5.3.449.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Line</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qayoum</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source"><em>Virulence</em>, <em>Aggressiveness</em>, <em>Evolution and Distribution of races of Puccinia striiformis (the Cause of Stripe Rust of Wheat) in North America</em>, <em>1968–1987. Technical</em></span>, <span class="cit-vol">Vol. 1788</span>. <span class="cit-publ-name">United States Department of Agriculture</span>, <span class="cit-publ-loc">Washington, DC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.48" data-doi="10.1093/bioinformatics/bts444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GAPIT: genome association and prediction integrated tool.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2397</span>–<span class="cit-lpage">2399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts444%26rft_id%253Dinfo%253Apmid%252F22796960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTgvMjM5NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy80NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magallanes-Lundback</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mesberg</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">H. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide association study and pathway-level analysis of tocochromanol levels in maize grain.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1287</span>–<span class="cit-lpage">1299</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D3%26rft.spage%253D1287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.50" data-doi="10.1007/s10681-010-0311-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubcovsky</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">Durable resistance to the wheat rusts: integrating systems biology and traditional phenotype-based research methods to guide the deployment of resistance genes.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D179%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-010-0311-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-010-0311-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.51" data-doi="10.1007/s00122-011-1573-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jankuloski</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">See</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Mapping and validation of QTL which confer partial resistance to broadly virulent post-2000 North American races of stripe rust in hexaploid wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DLowe%26rft.auinit1%253DI.%26rft.volume%253D123%26rft.issue%253D1%26rft.spage%253D143%26rft.epage%253D157%26rft.atitle%253DMapping%2Band%2Bvalidation%2Bof%2BQTL%2Bwhich%2Bconfer%2Bpartial%2Bresistance%2Bto%2Bbroadly%2Bvirulent%2Bpost-2000%2BNorth%2BAmerican%2Braces%2Bof%2Bstripe%2Brust%2Bin%2Bhexaploid%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1573-0%26rft_id%253Dinfo%253Apmid%252F21455722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1573-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21455722&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maccaferri</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ricci</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salvi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milner</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noli</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A high-density, SNP-based consensus map of tetraploid wheat as a bridge to integrate durum and bread wheat genomics and breeding.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-comment">DOI: 10.1111/pbi.12288</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.3.449.53"><div class="cit-metadata unstructured">McIntosh, R. A., Y. Yamazaki, J. Dubcovsky, W. J. Rogers, C. F. Morris <em>et al.</em>, 2013 Catalogue of gene symbols for wheat in <em>12<sup>th</sup> International Wheat Genetics Symposium</em>, edited by R.A. McIntosh, Yokohama, Japan.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-other" id="cit-5.3.449.54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McNeal</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Konzak</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tate</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-source">A Uniform System for Recording and Processing Cereal Research Data</span>. <span class="cit-publ-name">USDA, Agricultural Research Service</span>, <span class="cit-publ-loc">Washington, DC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.55" data-doi="10.1094/PHYTO-99-1-0089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Milus</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kristensen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hovmoller</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evidence for increased aggressiveness in a recent widespread strain of <em>Puccinia striiformis</em> f. sp. <em>tritici</em> causing stripe rust of wheat.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DMilus%26rft.auinit1%253DE.%2BA.%26rft.volume%253D99%26rft.issue%253D1%26rft.spage%253D89%26rft.epage%253D94%26rft.atitle%253DEvidence%2Bfor%2Bincreased%2Baggressiveness%2Bin%2Ba%2Brecent%2Bwidespread%2Bstrain%2Bof%2BPuccinia%2Bstriiformis%2Bf.%2Bsp.%2Btritici%2Bcausing%2Bstripe%2Brust%2Bof%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO-99-1-0089%26rft_id%253Dinfo%253Apmid%252F19055439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO-99-1-0089&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19055439&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262090900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.56" data-doi="10.1007/s10681-009-9935-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kulwal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mir</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genome-wide QTL analysis for pre-harvest sprouting tolerance in bread wheat.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">319</span>–<span class="cit-lpage">329</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D168%26rft.spage%253D319%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10681-009-9935-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10681-009-9935-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000267592000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.57" data-doi="10.1093/nar/gkt1110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monaco</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naithani</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dharmawardhana</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Gramene 2013: comparative plant genomics resources.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D1193</span>–<span class="cit-lpage">D1199</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1110%26rft_id%253Dinfo%253Apmid%252F24217918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjExOiI0Mi9EMS9EMTE5MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy80NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.58" data-doi="10.1073/pnas.0913160107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nemri</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarone</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide survey of Arabidopsis natural variation in downy mildew resistance using combined association and linkage mapping.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">10302</span>–<span class="cit-lpage">10307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0913160107%26rft_id%253Dinfo%253Apmid%252F20479233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzIyLzEwMzAyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.59" data-doi="10.1371/journal.pone.0067066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Leon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaeppler</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genetic architecture of maize stalk strength.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e67066</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De67066%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0067066%26rft_id%253Dinfo%253Apmid%252F23840585%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0067066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23840585&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.60" data-doi="10.1534/genetics.113.159152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gore</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The genetic architecture of maize height.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">1337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.159152%26rft_id%253Dinfo%253Apmid%252F24514905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ni80LzEzMzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvNDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-book" id="cit-5.3.449.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Percival</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1921</span> <span class="cit-source">The Wheat Plant: A Monograph</span>. <span class="cit-publ-name">Duckworth and Co.</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.62" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-book" id="cit-5.3.449.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.64" data-doi="10.1007/s00122-010-1502-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gowda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurer</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Longin</span>  <span class="cit-name-given-names">C. F. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korzun</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Association mapping for quality traits in soft winter wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">122</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">970</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DReif%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D122%26rft.issue%253D5%26rft.spage%253D961%26rft.epage%253D970%26rft.atitle%253DAssociation%2Bmapping%2Bfor%2Bquality%2Btraits%2Bin%2Bsoft%2Bwinter%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1502-7%26rft_id%253Dinfo%253Apmid%252F21153626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1502-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21153626&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.65"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reimer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pozniak</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somers</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Association mapping of yellow pigment in an elite collection of durum wheat cultivars and breeding lines.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1016</span>–<span class="cit-lpage">1025</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DReimer%26rft.auinit1%253DS.%26rft.volume%253D51%26rft.issue%253D12%26rft.spage%253D1016%26rft.epage%253D1025%26rft.atitle%253DAssociation%2Bmapping%2Bof%2Byellow%2Bpigment%2Bin%2Ban%2Belite%2Bcollection%2Bof%2Bdurum%2Bwheat%2Bcultivars%2Band%2Bbreeding%2Blines.%26rft_id%253Dinfo%253Apmid%252F19088814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19088814&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rexroad</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vallejo</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Estimates of linkage disequilibrium and effective population size in rainbow trout.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DRexroad%26rft.auinit1%253DC.%2BE.%26rft.volume%253D10%26rft.spage%253D83%26rft.epage%253D83%26rft.atitle%253DEstimates%2Bof%2Blinkage%2Bdisequilibrium%2Band%2Beffective%2Bpopulation%2Bsize%2Bin%2Brainbow%2Btrout.%26rft_id%253Dinfo%253Apmid%252F20003428%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20003428&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.67" data-doi="10.1534/g3.113.005819"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saintenac</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akhunov</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sequence-based mapping of the polyploid wheat genome.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda).</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529.%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.005819%26rft_id%253Dinfo%253Apmid%252F23665877%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjMvNy8xMTA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.68" data-doi="10.1007/s00122-008-0820-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Santra</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santra</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidwell</span>  <span class="cit-name-given-names">K. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Identification and mapping QTL for high-temperature adult-plant resistance to stripe rust in winter wheat (<em>Triticum aestivum</em> L.) cultivar ’Stephens’.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">793</span>–<span class="cit-lpage">802</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSantra%26rft.auinit1%253DD.%2BK.%26rft.volume%253D117%26rft.issue%253D5%26rft.spage%253D793%26rft.epage%253D802%26rft.atitle%253DIdentification%2Band%2Bmapping%2BQTL%2Bfor%2Bhigh-temperature%2Badult-plant%2Bresistance%2Bto%2Bstripe%2Brust%2Bin%2Bwinter%2Bwheat%2B%2528Triticum%2Baestivum%2BL.%2529%2Bcultivar%2B%2527Stephens%2527.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-008-0820-5%26rft_id%253Dinfo%253Apmid%252F18584147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-008-0820-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18584147&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharma-Poudyal</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rupp</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Potential oversummering and overwintering regions for the wheat stripe rust pathogen in the contiguous United States.</span> <abbr class="cit-jnl-abbrev">Int. J. Biometeorol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">987</span>–<span class="cit-lpage">997</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BBiometeorol.%26rft.volume%253D58%26rft.spage%253D987%26rft_id%253Dinfo%253Apmid%252F23722926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23722926&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.70" data-doi="10.1093/hmg/ddg088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shifman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuypers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kokoris</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yakir</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darvasi</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Linkage disequilibrium patterns of the human genome across populations.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">771</span>–<span class="cit-lpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BMolecular%2BGenetics%26rft.stitle%253DHum%2BMol%2BGenet%26rft.aulast%253DShifman%26rft.auinit1%253DS.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D771%26rft.epage%253D776%26rft.atitle%253DLinkage%2Bdisequilibrium%2Bpatterns%2Bof%2Bthe%2Bhuman%2Bgenome%2Bacross%2Bpopulations%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddg088%26rft_id%253Dinfo%253Apmid%252F12651872%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjg6IjEyLzcvNzcxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQ0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Espino</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rajaram</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Achieving near-immunity to leaf and stripe rusts in wheat by combining slow rusting resistance genes.</span> <abbr class="cit-jnl-abbrev">Acta Phytopathol Hun</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BPhytopathol%2BHun%26rft.volume%253D35%26rft.spage%253D133%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Espino</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">William</span>  <span class="cit-name-given-names">H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetics and breeding for durable resistance to leaf and stripe rusts in wheat.</span> <abbr class="cit-jnl-abbrev">Turk. J. Agric. For.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTurk.%2BJ.%2BAgric.%2BFor.%26rft.volume%253D29%26rft.spage%253D121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.73" data-doi="10.1007/s00122-004-1740-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Somers</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isaac</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A high-density microsatellite consensus map for bread wheat (<em>Triticum aestivum</em> L.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">1105</span>–<span class="cit-lpage">1114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSomers%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D109%26rft.issue%253D6%26rft.spage%253D1105%26rft.epage%253D1114%26rft.atitle%253DA%2Bhigh-density%2Bmicrosatellite%2Bconsensus%2Bmap%2Bfor%2Bbread%2Bwheat%2B%2528Triticum%2Baestivum%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1740-7%26rft_id%253Dinfo%253Apmid%252F15490101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1740-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15490101&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224614100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.74" data-doi="10.1007/s00122-004-1871-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">Q. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fickus</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costa</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Development and mapping of microsatellite (SSR) markers in wheat.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">550</span>–<span class="cit-lpage">560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DSong%26rft.auinit1%253DQ.%2BJ.%26rft.volume%253D110%26rft.issue%253D3%26rft.spage%253D550%26rft.epage%253D560%26rft.atitle%253DDevelopment%2Band%2Bmapping%2Bof%2Bmicrosatellite%2B%2528SSR%2529%2Bmarkers%2Bin%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1871-x%26rft_id%253Dinfo%253Apmid%252F15655666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1871-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15655666&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000227049900015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.75" data-doi="10.1093/bioinformatics/bts313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sosnowski</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charcosset</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joets</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">BioMercator V3: an upgrade of genetic map compilation and quantitative trait loci meta-analysis algorithms.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2082</span>–<span class="cit-lpage">2083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts313%26rft_id%253Dinfo%253Apmid%252F22661647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTUvMjA4MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy80NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.76"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Via</span>  <span class="cit-name-given-names">L. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">A rapid CTAB DNA isolation technique useful for RAPD fingerprinting and other PCR applications</span>. <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">748</span>–<span class="cit-lpage">749</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DStewart%26rft.auinit1%253DC.%2BN.%26rft.volume%253D14%26rft.issue%253D5%26rft.spage%253D748%26rft.epage%253D750%26rft.atitle%253DA%2Brapid%2BCTAB%2BDNA%2Bisolation%2Btechnique%2Buseful%2Bfor%2BRAPD%2Bfingerprinting%2Band%2Bother%2BPCR%2Bapplications.%26rft_id%253Dinfo%253Apmid%252F8512694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8512694&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LA81200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.77" data-doi="10.1534/genetics.107.079707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohring</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piepho</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heckenberger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Comparison of mixed-model approaches for association mapping.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">1745</span>–<span class="cit-lpage">1754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.079707%26rft_id%253Dinfo%253Apmid%252F18245847%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC8zLzE3NDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvNDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.78" data-doi="10.1371/journal.pone.0038008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sukhwinder-Singh</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bains</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Multi-trait and multi-environment QTL analyses for resistance to wheat diseases.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e38008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSukhwinder-Singh%26rft.volume%253D7%26rft.issue%253D6%26rft.spage%253De38008%26rft.epage%253De38008%26rft.atitle%253DMulti-trait%2Band%2Bmulti-environment%2BQTL%2Banalyses%2Bfor%2Bresistance%2Bto%2Bwheat%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0038008%26rft_id%253Dinfo%253Apmid%252F22679489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0038008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22679489&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.79" data-doi="10.1016/0040-5809(71)90011-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sved</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span> <span class="cit-article-title">Linkage disequilibrium and homozigosity of chromosome segments in finite populations.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">125</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DSved%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D2%26rft.issue%253D2%26rft.spage%253D125%26rft.epage%253D141%26rft.atitle%253DLinkage%2Bdisequilibrium%2Band%2Bhomozygosity%2Bof%2Bchromosome%2Bsegments%2Bin%2Bfinite%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252871%252990011-6%26rft_id%253Dinfo%253Apmid%252F5170716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(71)90011-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5170716&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.80" data-doi="10.1007/BF02042836"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tengberg</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Crop husbandry at Miri Qalat, Makran, SW Pakistan (4000–2000 B.C.)</span>. <abbr class="cit-jnl-abbrev">Veg. Hist. Archeobot.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVeg.%2BHist.%2BArcheobot.%26rft.volume%253D8%26rft.spage%253D3%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02042836%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02042836&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-other" id="cit-5.3.449.81"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armstrong</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradford</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quality control procedures for genome wide association studies.</span> <span class="cit-source">Curr. Protoc. Hum. Genet.</span> <span class="cit-comment">Chapter 1, Unit 1.19</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.82" data-doi="10.1186/1471-2229-12-56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Inghelandt</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinant</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association mapping of flowering time and northern corn leaf blight (<em>Setosphaeria turcica</em>) resistance in a vast commercial maize germplasm set.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DVan%2BInghelandt%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D56%26rft.epage%253D56%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Bof%2Bflowering%2Btime%2Band%2Bnorthern%2Bcorn%2Bleaf%2Bblight%2B%2528Setosphaeria%2Bturcica%2529%2Bresistance%2Bin%2Ba%2Bvast%2Bcommercial%2Bmaize%2Bgermplasm%2Bset.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-12-56%26rft_id%253Dinfo%253Apmid%252F22545925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-12-56&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22545925&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.83" data-doi="10.1007/s00122-011-1681-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riera-Lizarazu</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heesacker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic analysis of adult plant, quantitative resistance to stripe rust in wheat cultivar ’Stephens’ in multi-environment trials.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DDolores%2BVazquez%26rft.auinit1%253DM.%26rft.volume%253D124%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D11%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Badult%2Bplant%252C%2Bquantitative%2Bresistance%2Bto%2Bstripe%2Brust%2Bin%2Bwheat%2Bcultivar%2B%2527Stephens%2527%2Bin%2Bmulti-environment%2Btrials.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1681-x%26rft_id%253Dinfo%253Apmid%252F21912857%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1681-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21912857&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.84" data-doi="10.1038/hdy.2013.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallace</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larsson</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Entering the second century of maize quantitative genetics.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">30</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D30%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.6%26rft_id%253Dinfo%253Apmid%252F23462502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23462502&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.85" data-doi="10.1094/PDIS-01-14-0071-RE"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wan</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">X. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Virulence characterization of <em>Puccinia striiformis</em> f. sp. <em>tritici</em> using a new set of <em>Yr</em> single-gene line differentials in the United States in 2010.</span> <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">1534</span>–<span class="cit-lpage">1542</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D98%26rft.spage%253D1534%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPDIS-01-14-0071-RE%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PDIS-01-14-0071-RE&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.86" data-doi="10.1111/nph.12164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Y. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title"><em>Aegilops tauschii</em> single nucleotide polymorphisms shed light on the origins of wheat D-genome genetic diversity and pinpoint the geographic origin of hexaploid wheat.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">925</span>–<span class="cit-lpage">937</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D198%26rft.spage%253D925%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12164%26rft_id%253Dinfo%253Apmid%252F23374069%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12164&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23374069&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=23374069&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.87" data-doi="10.1007/s00122-011-1697-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leach</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockram</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association mapping of agronomic and morphologic traits in highly structured populations of barley cultivars.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">233</span>–<span class="cit-lpage">246</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DM.%26rft.volume%253D124%26rft.issue%253D2%26rft.spage%253D233%26rft.epage%253D246%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Bof%2Bagronomic%2Band%2Bmorphologic%2Btraits%2Bin%2Bhighly%2Bstructured%2Bpopulations%2Bof%2Bbarley%2Bcultivars.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1697-2%26rft_id%253Dinfo%253Apmid%252F21915710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1697-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21915710&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.88" data-doi="10.1111/pbi.12183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Allen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Characterization of polyploid wheat genomic diversity using a high-density 90,000 SNP array.</span> <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">787</span>–<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D12%26rft.spage%253D787%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12183%26rft_id%253Dinfo%253Apmid%252F24646323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24646323&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339488200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-book" id="cit-5.3.449.89"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Genetic Data Analysis II</span>. <span class="cit-publ-name">Sinauer Publishers</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.90" data-doi="10.1007/s10709-009-9351-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">Y. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Association mapping of agronomic traits on chromosome 2A of wheat.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">75</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D137%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-009-9351-5%26rft_id%253Dinfo%253Apmid%252F19160058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-009-9351-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19160058&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-91-1" title="View reference  in text" id="ref-91">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.91" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-92-1" title="View reference  in text" id="ref-92">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.92" data-doi="10.1371/journal.pone.0051211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karns</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Indugula</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Finding missing heritability in less significant Loci and allelic heterogeneity: genetic variation in human height.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e51211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D7%26rft.spage%253De51211%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0051211%26rft_id%253Dinfo%253Apmid%252F23251454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0051211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23251454&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-93-1" title="View reference  in text" id="ref-93">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.93" data-doi="10.1038/ng.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">C. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todhunter</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiwari</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mixed linear model approach adapted for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D355%26rft.epage%253D360%26rft.atitle%253DMixed%2Blinear%2Bmodel%2Bapproach%2Badapted%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.546%26rft_id%253Dinfo%253Apmid%252F20208535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208535&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-94-1" title="View reference  in text" id="ref-94">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.94" data-doi="10.1038/ncomms1467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eizenga</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association mapping reveals a rich genetic architecture of complex traits in <em>Oryza sativa</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DZhao%26rft.auinit1%253DK.%26rft.volume%253D2%26rft.spage%253D467%26rft.epage%253D467%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Breveals%2Ba%2Brich%2Bgenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits%2Bin%2BOryza%2Bsativa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1467%26rft_id%253Dinfo%253Apmid%252F21915109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21915109&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F449.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-95-1" title="View reference  in text" id="ref-95">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.449.95" data-doi="10.1073/pnas.1322563111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zuk</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffner</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samocha</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Do</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hechter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Searching for missing heritability: designing rare variant association studies.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">E455</span>–<span class="cit-lpage">E464</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1322563111%26rft_id%253Dinfo%253Apmid%252F24443550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTExLzQvRTQ1NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy80NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/3/449.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/3/441" title="Evolutionary Consequences of DNA Methylation on the GC Content in Vertebrate Genomes" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/3/467" title="Identification of Nonviable Genes Affecting Touch Sensitivity in Caenorhabditis elegans Using Neuronally Enhanced Feeding RNA Interference" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/3"><span>Volume 5 Issue 3, March 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/3.cover-source.jpg?itok=g434JgGA" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/3/449" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PGLZtpnkfeLIe4jCU55KIXNCzHpWbj_2kCOSP9fXArM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/3/449" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/3/449" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/3/449" class="active" data-icon-position="" data-hide-link-title="0">A Genome-Wide Association Study of Resistance to Stripe Rust (Puccinia striiformis f. sp. tritici) in a Worldwide Collection of Hexaploid Spring Wheat (Triticum aestivum L.)</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459781" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-9fWI0qAqTB-O6p95h6XWvzN2zeFEF-UjfwkmrqS3eXQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/3/449.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/3/449" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-ma8pGb4Wt3PVECyYjCkwtFEzzseCctLXLixYeAZqeXk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/3/449" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/3/449.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459781" id="node459781--2" data-pisa="ggg;5/3/449" data-pisa-master="ggg;g3.114.014563" data-apath="/ggg/5/3/449.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genome-Wide Association Study of Resistance to Stripe Rust (<em>Puccinia striiformis</em> f. sp. <em>tritici</em>) in a Worldwide Collection of Hexaploid Spring Wheat (<em>Triticum aestivum</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marco Maccaferri</span>, <span class="highwire-citation-author" data-delta="1">Junli Zhang</span>, <span class="highwire-citation-author" data-delta="2">Peter Bulli</span>, <span class="highwire-citation-author" data-delta="3">Zewdie Abate</span>, <span class="highwire-citation-author" data-delta="4">Shiaoman Chao</span>, <span class="highwire-citation-author" data-delta="5">Dario Cantu</span>, <span class="highwire-citation-author" data-delta="6">Eligio Bossolini</span>, <span class="highwire-citation-author" data-delta="7">Xianming Chen</span>, <span class="highwire-citation-author" data-delta="8">Michael Pumphrey</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-7571-4345" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jorge Dubcovsky</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">449-465;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014563 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459781--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marco Maccaferri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agricultural Sciences (DipSA), University of Bologna, Bologna 40127, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marco%2BMaccaferri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maccaferri%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarco%2BMaccaferri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Junli Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Junli%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJunli%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter Bulli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, Washington State University, Pullman, Washington 99164-6420</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BBulli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bulli%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BBulli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zewdie Abate </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zewdie%2BAbate%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abate%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZewdie%2BAbate%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Shiaoman Chao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>USDA-ARS, 1605 Albrecht Blvd, Fargo, North Dakota 58105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shiaoman%2BChao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chao%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShiaoman%2BChao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Dario Cantu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Viticulture and Enology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dario%2BCantu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cantu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADario%2BCantu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Eligio Bossolini </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eligio%2BBossolini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bossolini%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEligio%2BBossolini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Xianming Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA-ARS, Wheat Genetics, Quality Physiology, and Disease Research Unit, and Department of Plant Pathology, Washington State University, Pullman, Washington 99164</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xianming%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXianming%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Michael Pumphrey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, Washington State University, Pullman, Washington 99164-6420</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BPumphrey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pumphrey%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BPumphrey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Jorge Dubcovsky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Howard Hughes Medical Institute, Chevy Chase, Maryland 20815</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jorge%2BDubcovsky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dubcovsky%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJorge%2BDubcovsky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7571-4345" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jorge Dubcovsky</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Jdubcovsky@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">Jdubcovsky@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/3/449" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/3/449&amp;title=A%20Genome-Wide%20Association%20Study%20of%20Resistance%20to%20Stripe%20Rust%20%28Puccinia%20striiformis%20f.%20sp.%20tritici%29%20in%20a%20Worldwide%20Collection%20of%20Hexaploid%20Spring%20Wheat%20%28Triticum%20aestivum%20L.%29" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/3/449&amp;title=A%20Genome-Wide%20Association%20Study%20of%20Resistance%20to%20Stripe%20Rust%20%28Puccinia%20striiformis%20f.%20sp.%20tritici%29%20in%20a%20Worldwide%20Collection%20of%20Hexaploid%20Spring%20Wheat%20%28Triticum%20aestivum%20L.%29" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/3/449&amp;title=A%20Genome-Wide%20Association%20Study%20of%20Resistance%20to%20Stripe%20Rust%20%28Puccinia%20striiformis%20f.%20sp.%20tritici%29%20in%20a%20Worldwide%20Collection%20of%20Hexaploid%20Spring%20Wheat%20%28Triticum%20aestivum%20L.%29" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/3/449&amp;text=A%20Genome-Wide%20Association%20Study%20of%20Resistance%20to%20Stripe%20Rust%20%28Puccinia%20striiformis%20f.%20sp.%20tritici%29%20in%20a%20Worldwide%20Collection%20of%20Hexaploid%20Spring%20Wheat%20%28Triticum%20aestivum%20L.%29" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/3/449&amp;title=A%20Genome-Wide%20Association%20Study%20of%20Resistance%20to%20Stripe%20Rust%20%28Puccinia%20striiformis%20f.%20sp.%20tritici%29%20in%20a%20Worldwide%20Collection%20of%20Hexaploid%20Spring%20Wheat%20%28Triticum%20aestivum%20L.%29" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/3/449&amp;t=A%20Genome-Wide%20Association%20Study%20of%20Resistance%20to%20Stripe%20Rust%20%28Puccinia%20striiformis%20f.%20sp.%20tritici%29%20in%20a%20Worldwide%20Collection%20of%20Hexaploid%20Spring%20Wheat%20%28Triticum%20aestivum%20L.%29" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/3/449&amp;title=A%20Genome-Wide%20Association%20Study%20of%20Resistance%20to%20Stripe%20Rust%20%28Puccinia%20striiformis%20f.%20sp.%20tritici%29%20in%20a%20Worldwide%20Collection%20of%20Hexaploid%20Spring%20Wheat%20%28Triticum%20aestivum%20L.%29" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/3/449&amp;title=A%20Genome-Wide%20Association%20Study%20of%20Resistance%20to%20Stripe%20Rust%20%28Puccinia%20striiformis%20f.%20sp.%20tritici%29%20in%20a%20Worldwide%20Collection%20of%20Hexaploid%20Spring%20Wheat%20%28Triticum%20aestivum%20L.%29" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459781/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459781" id="node459781--3" data-pisa="ggg;5/3/449" data-pisa-master="ggg;g3.114.014563" data-apath="/ggg/5/3/449.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genome-Wide Association Study of Resistance to Stripe Rust (<em>Puccinia striiformis</em> f. sp. <em>tritici</em>) in a Worldwide Collection of Hexaploid Spring Wheat (<em>Triticum aestivum</em> L.)</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Marco Maccaferri</span>, <span class="highwire-citation-author" data-delta="1">Junli Zhang</span>, <span class="highwire-citation-author" data-delta="2">Peter Bulli</span>, <span class="highwire-citation-author" data-delta="3">Zewdie Abate</span>, <span class="highwire-citation-author" data-delta="4">Shiaoman Chao</span>, <span class="highwire-citation-author" data-delta="5">Dario Cantu</span>, <span class="highwire-citation-author" data-delta="6">Eligio Bossolini</span>, <span class="highwire-citation-author" data-delta="7">Xianming Chen</span>, <span class="highwire-citation-author" data-delta="8">Michael Pumphrey</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-7571-4345" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jorge Dubcovsky</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">449-465;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.014563 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459781--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Marco Maccaferri </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Agricultural Sciences (DipSA), University of Bologna, Bologna 40127, Italy</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marco%2BMaccaferri%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maccaferri%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarco%2BMaccaferri%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Junli Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Junli%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJunli%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Peter Bulli </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, Washington State University, Pullman, Washington 99164-6420</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BBulli%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bulli%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APeter%2BBulli%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Zewdie Abate </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zewdie%2BAbate%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abate%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZewdie%2BAbate%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Shiaoman Chao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>USDA-ARS, 1605 Albrecht Blvd, Fargo, North Dakota 58105</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shiaoman%2BChao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chao%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShiaoman%2BChao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Dario Cantu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Viticulture and Enology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dario%2BCantu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cantu%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADario%2BCantu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Eligio Bossolini </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eligio%2BBossolini%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bossolini%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEligio%2BBossolini%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Xianming Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA-ARS, Wheat Genetics, Quality Physiology, and Disease Research Unit, and Department of Plant Pathology, Washington State University, Pullman, Washington 99164</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xianming%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXianming%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Michael Pumphrey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Crop and Soil Sciences, Washington State University, Pullman, Washington 99164-6420</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BPumphrey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pumphrey%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BPumphrey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Jorge Dubcovsky </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Sciences, University of California, Davis, California 95616</div><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>Howard Hughes Medical Institute, Chevy Chase, Maryland 20815</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jorge%2BDubcovsky%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Dubcovsky%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJorge%2BDubcovsky%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7571-4345" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jorge Dubcovsky</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Jdubcovsky@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">Jdubcovsky@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=449&amp;rft.epage=465&amp;rft.atitle=A%20Genome-Wide%20Association%20Study%20of%20Resistance%20to%20Stripe%20Rust%20%28Puccinia%20striiformis%20f.%20sp.%20tritici%29%20in%20a%20Worldwide%20Collection%20of%20Hexaploid%20Spring%20Wheat%20%28Triticum%20aestivum%20L.%29&amp;rft.volume=5&amp;rft.issue=3&amp;rft.date=2015-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Maccaferri%2C+Marco&amp;rft.au=Zhang%2C+Junli&amp;rft.au=Bulli%2C+Peter&amp;rft.au=Abate%2C+Zewdie&amp;rft.au=Chao%2C+Shiaoman&amp;rft.au=Cantu%2C+Dario&amp;rft.au=Bossolini%2C+Eligio&amp;rft.au=Chen%2C+Xianming&amp;rft.au=Pumphrey%2C+Michael&amp;rft.au=Dubcovsky%2C+Jorge"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459781/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459781/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459781/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459781/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459781/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459781/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459781/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459781/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459781/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459781/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459781/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459781/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/3/449" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/3/449.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/3/449.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/3/449.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/3/449.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
