<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/3/407" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/3/407.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/3/407.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/3/407.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/3/407.atom" />
<meta name="HW.pisa" content="ggg;5/3/407" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Comparison of T7E1 and Surveyor Mismatch Cleavage Assays to Detect Mutations Triggered by Engineered Nucleases" />
<meta name="DC.Identifier" content="10.1534/g3.114.015834" />
<meta name="DC.Date" content="2015-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Vouillot et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genome editing using engineered nucleases is used for targeted mutagenesis. But because genome editing does not target all loci with similar efficiencies, the mutation hit-rate at a given locus needs to be evaluated. The analysis of mutants obtained using engineered nucleases requires specific methods for mutation detection, and the enzyme mismatch cleavage method is used commonly for this purpose. This method uses enzymes that cleave heteroduplex DNA at mismatches and extrahelical loops formed by single or multiple nucleotides. Bacteriophage resolvases and single-stranded nucleases are used commonly in the assay but have not been compared side-by-side on mutations obtained by engineered nucleases. We present the first comparison of the sensitivity of T7E1 and Surveyor EMC assays on deletions and point mutations obtained by zinc finger nuclease targeting in frog embryos. We report the mutation detection limits and efficiencies of T7E1 and Surveyor. In addition, we find that T7E1 outperforms the Surveyor nuclease in terms of sensitivity with deletion substrates, whereas Surveyor is better for detecting single nucleotide changes. We conclude that T7E1 is the preferred enzyme to scan mutations triggered by engineered nucleases." />
<meta name="DC.Contributor" content="Léna Vouillot" />
<meta name="DC.Contributor" content="Aurore Thélie" />
<meta name="DC.Contributor" content="Nicolas Pollet" />
<meta name="article:published_time" content="2015-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Comparison of T7E1 and Surveyor Mismatch Cleavage Assays to Detect Mutations Triggered by Engineered Nucleases" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genome editing using engineered nucleases is used for targeted mutagenesis. But because genome editing does not target all loci with similar efficiencies, the mutation hit-rate at a given locus needs to be evaluated. The analysis of mutants obtained using engineered nucleases requires specific methods for mutation detection, and the enzyme mismatch cleavage method is used commonly for this purpose. This method uses enzymes that cleave heteroduplex DNA at mismatches and extrahelical loops formed by single or multiple nucleotides. Bacteriophage resolvases and single-stranded nucleases are used commonly in the assay but have not been compared side-by-side on mutations obtained by engineered nucleases. We present the first comparison of the sensitivity of T7E1 and Surveyor EMC assays on deletions and point mutations obtained by zinc finger nuclease targeting in frog embryos. We report the mutation detection limits and efficiencies of T7E1 and Surveyor. In addition, we find that T7E1 outperforms the Surveyor nuclease in terms of sensitivity with deletion substrates, whereas Surveyor is better for detecting single nucleotide changes. We conclude that T7E1 is the preferred enzyme to scan mutations triggered by engineered nucleases.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/03/01" />
<meta name="citation_mjid" content="ggg;5/3/407" />
<meta name="citation_id" content="5/3/407" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/3/407" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/3/407.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/3/407.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/3/407.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.015834" />
<meta name="citation_pmid" content="25566793" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="407" />
<meta name="citation_lastpage" content="415" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Léna Vouillot" />
<meta name="citation_author_institution" content="Institute of Systems and Synthetic Biology, CNRS, Université d’Evry Val d’Essonne, Evry, France" />
<meta name="citation_author" content="Aurore Thélie" />
<meta name="citation_author_institution" content="Institute of Systems and Synthetic Biology, CNRS, Université d’Evry Val d’Essonne, Evry, France" />
<meta name="citation_author" content="Nicolas Pollet" />
<meta name="citation_author_institution" content="Institute of Systems and Synthetic Biology, CNRS, Université d’Evry Val d’Essonne, Evry, France" />
<meta name="citation_author_email" content="Nicolas.Pollet@issb.genopole.fr" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=J. J. Babon;citation_author=R. Youil;citation_author=R. G. H. Cotton;citation_title=Improved strategy for mutation detection--a modification to the enzyme mismatch cleavage method;citation_pages=5082-5084;citation_volume=23;citation_year=1995;citation_issue=24;citation_pmid=8559669;citation_doi=10.1093/nar/23.24.5082" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=H. A. Behrensdorf;citation_author=M. Pignot;citation_author=N. Windhab;citation_author=A. Kappel;citation_title=Rapid parallel mutation scanning of gene fragments using a microelectronic protein-DNA chip format;citation_pages=e64-e64;citation_volume=30;citation_year=2002;citation_issue=14;citation_pmid=12136112;citation_doi=10.1093/nar/gnf063" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Bentley;citation_author=B. MacLennan;citation_author=J. Calvo;citation_author=C. R. Dearolf;citation_title=Targeted Recovery of Mutations in Drosophila;citation_pages=1169-1173;citation_volume=156;citation_year=2000;citation_issue=3;citation_pmid=11063692" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. J. Beumer;citation_author=J. K. Trautman;citation_author=A. Bozas;citation_author=J.-L. Liu;citation_author=J. Rutter;citation_title=Efficient gene targeting in Drosophila by direct embryo injection with zinc-finger nucleases.;citation_volume=105;citation_year=2008;citation_pmid=19064913;citation_doi=10.1073/pnas.0810475105" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=A. Bhattacharyya;citation_author=D. M. Lilley;citation_title=Single base mismatches in DNA. Long- and short-range structure probed by analysis of axis trajectory and local chemical reactivity.;citation_pages=583-597;citation_volume=209;citation_year=1989;citation_issue=4;citation_pmid=2585502;citation_doi=10.1016/0022-2836(89)90596-2" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=E. L. Coghill;citation_author=A. Hugill;citation_author=N. Parkinson;citation_author=C. Davison;citation_author=P. Glenister;citation_title=A gene-driven approach to the identification of ENU mutants in the mouse.;citation_pages=255-256;citation_volume=30;citation_year=2002;citation_issue=3;citation_pmid=11850622;citation_doi=10.1038/ng847" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=T. Colbert;citation_author=B. J. Till;citation_author=R. Tompa;citation_author=S. Reynolds;citation_author=M. N. Steine;citation_title=High-Throughput Screening for Induced Point Mutations;citation_pages=480-484;citation_volume=126;citation_year=2001;citation_issue=2;citation_pmid=11402178;citation_doi=10.1104/pp.126.2.480" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=L. Comai;citation_author=K. Young;citation_author=B. J. Till;citation_author=S. H. Reynolds;citation_author=E. A. Greene;citation_title=Efficient discovery of DNA polymorphisms in natural populations by Ecotilling.;citation_pages=778-786;citation_volume=37;citation_year=2004;citation_issue=5;citation_pmid=14871304;citation_doi=10.1111/j.0960-7412.2003.01999.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=T. J. Dahlem;citation_author=K. Hoshijima;citation_author=M. J. Jurynec;citation_author=D. Gunther;citation_author=C. G. Starker;citation_title=Simple methods for generating and detecting locus-specific mutations induced with TALENs in the zebrafish genome.;citation_pages=e1002861-e1002861;citation_volume=8;citation_year=2012;citation_issue=8;citation_pmid=22916025;citation_doi=10.1371/journal.pgen.1002861" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in structural biology;citation_journal_abbrev=Curr Opin Struct Biol;citation_author=A.-C. Déclais;citation_author=D. M. Lilley;citation_title=New insight into the recognition of branched DNA structure by junction-resolving enzymes.;citation_pages=86-95;citation_volume=18;citation_year=2008;citation_issue=1;citation_pmid=18160275;citation_doi=10.1016/j.sbi.2007.11.001" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=A.-C. Déclais;citation_author=J. Liu;citation_author=A. D. J. Freeman;citation_author=D. M. J. Lilley;citation_title=Structural recognition between a four-way DNA junction and a resolving enzyme.;citation_pages=1261-1276;citation_volume=359;citation_year=2006;citation_issue=5;citation_pmid=16690083;citation_doi=10.1016/j.jmb.2006.04.037" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry (Mosc.);citation_author=J. B. Dodgson;citation_author=R. D. Wells;citation_title=Action of single-strand specific nucleases on model DNA heteroduplexes of defined size and sequence.;citation_pages=2374-2379;citation_volume=16;citation_year=1977" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Biol.;citation_author=A. D. J. Freeman;citation_author=A.-C. Déclais;citation_author=D. M. J. Lilley;citation_title=The importance of the N-terminus of T7 endonuclease I in the interaction with DNA junctions.;citation_pages=395-410;citation_volume=425;citation_year=2013;citation_pmid=23207296;citation_doi=10.1016/j.jmb.2012.11.029" />
<meta name="citation_reference" content="citation_journal_title=Trends Biotechnol.;citation_author=T. Gaj;citation_author=C. A. Gersbach;citation_author=C. F. Barbas;citation_title=ZFN, TALEN, and CRISPR/Cas-based methods for genome engineering.;citation_pages=397-405;citation_volume=31;citation_year=2013;citation_pmid=23664777;citation_doi=10.1016/j.tibtech.2013.04.004" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. M. Geurts;citation_author=G. J. Cost;citation_author=Y. Freyvert;citation_author=B. Zeitler;citation_author=J. C. Miller;citation_title=Knockout Rats via Embryo Microinjection of Zinc-Finger Nucleases;citation_pages=433-433;citation_volume=325;citation_year=2009;citation_issue=5939;citation_pmid=19628861;citation_doi=10.1126/science.1172447" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=C. Gohlke;citation_author=A. I. Murchie;citation_author=D. M. Lilley;citation_author=R. M. Clegg;citation_title=Kinking of DNA and RNA helices by bulged nucleotides observed by fluorescence resonance energy transfer.;citation_pages=11660-11664;citation_volume=91;citation_year=1994;citation_issue=24;citation_pmid=7526401;citation_doi=10.1073/pnas.91.24.11660" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=X. Guo;citation_author=T. Zhang;citation_author=Z. Hu;citation_author=Y. Zhang;citation_author=Z. Shi;citation_title=Efficient RNA/Cas9-mediated genome editing in Xenopus tropicalis.;citation_volume=141;citation_year=2014;citation_pmid=24401372;citation_doi=10.1242/dev.099853" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=D. Y. Guschin;citation_author=A. J. Waite;citation_author=G. E. Katibah;citation_author=J. C. Miller;citation_author=M. C. Holmes;citation_title=A rapid and general assay for monitoring endogenous gene modification.;citation_pages=247-256;citation_volume=649;citation_year=2010;citation_pmid=20680839;citation_doi=10.1007/978-1-60761-753-2_15" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. M. Hadden;citation_author=A.-C. Déclais;citation_author=S. B. Carr;citation_author=D. M. J. Lilley;citation_author=S. E. V. Phillips;citation_title=The structural basis of Holliday junction resolution by T7 endonuclease I.;citation_pages=621-624;citation_volume=449;citation_year=2007;citation_issue=7162;citation_pmid=17873858;citation_doi=10.1038/nature06158" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=S. N. Ho;citation_author=H. D. Hunt;citation_author=R. M. Horton;citation_author=J. K. Pullen;citation_author=L. R. Pease;citation_title=Site-directed mutagenesis by overlap extension using the polymerase chain reaction.;citation_pages=51-59;citation_volume=77;citation_year=1989;citation_issue=1;citation_pmid=2744487;citation_doi=10.1016/0378-1119(89)90358-2" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Holkers;citation_author=I. Maggio;citation_author=J. Liu;citation_author=J. M. Janssen;citation_author=F. Miselli;citation_title=Differential integrity of TALE nuclease genes following adenoviral and lentiviral vector gene transfer into human cells.;citation_volume=41;citation_year=2013;citation_pmid=23275534;citation_doi=10.1093/nar/gks1446" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=J. T. Howard;citation_author=J. Ward;citation_author=J. N. Watson;citation_author=K. H. Roux;citation_title=Heteroduplex cleavage analysis using S1 nuclease.;citation_pages=18-19;citation_volume=27;citation_year=1999;citation_issue=1;citation_pmid=10407654" />
<meta name="citation_reference" content="citation_journal_title=Electrophoresis;citation_author=M. C. Huang;citation_author=W. C. Cheong;citation_author=L. S. Lim;citation_author=M.-H. Li;citation_title=A simple, high sensitivity mutation screening using Ampligase mediated T7 endonuclease I and Surveyor nuclease with microfluidic capillary electrophoresis.;citation_pages=788-796;citation_volume=33;citation_year=2012;citation_pmid=22437793;citation_doi=10.1002/elps.201100460" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=M. Isalan;citation_title=Zinc-finger nucleases: how to play two good hands.;citation_pages=32-34;citation_volume=9;citation_year=2012;citation_issue=1;citation_pmid=22205514" />
<meta name="citation_reference" content="citation_journal_title=Biol. Open;citation_author=S. Ishibashi;citation_author=R. Cliffe;citation_author=E. Amaya;citation_title=Highly efficient bi-allelic mutation rates using TALENs in Xenopus tropicalis.;citation_volume=1;citation_year=2012;citation_pmid=23408158;citation_doi=10.1242/bio.20123228" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. S. B. Judo;citation_author=A. B. Wedel;citation_author=C. Wilson;citation_title=Stimulation and suppression of PCR-mediated recombination.;citation_volume=26;citation_year=1998;citation_pmid=9512558;citation_doi=10.1093/nar/26.7.1819" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=Y. Kim;citation_author=J. Kweon;citation_author=J.-S. Kim;citation_title=TALENs and ZFNs are associated with different mutation signatures.;citation_pages=185;citation_volume=10;citation_year=2013;citation_pmid=23396284;citation_doi=10.1038/nmeth.2364" />
<meta name="citation_reference" content="citation_journal_title=Biotechniques;citation_author=D. J. G. Lahr;citation_author=L. A. Katz;citation_title=Reducing the impact of PCR-mediated recombination in molecular evolution and environmental studies using a new-generation high-fidelity DNA polymerase.;citation_pages=857-866;citation_volume=47;citation_year=2009;citation_pmid=19852769;citation_doi=10.2144/000113219" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Lei;citation_author=X. Guo;citation_author=Y. Liu;citation_author=Y. Cao;citation_author=Y. Deng;citation_title=Efficient targeted gene disruption in Xenopus embryos using engineered transcription activator-like effector nucleases (TALENs).;citation_volume=109;citation_year=2012;citation_pmid=23045671;citation_doi=10.1073/pnas.1215421109" />
<meta name="citation_reference" content="citation_journal_title=Hum. Gene Ther.;citation_author=D. A. Maier;citation_author=A. L. Brennan;citation_author=S. Jiang;citation_author=G. K. Binder-Scholl;citation_author=G. Lee;citation_title=Efficient clinical scale gene modification via zinc finger nuclease-targeted disruption of the HIV co-receptor CCR5.;citation_pages=245-258;citation_volume=24;citation_year=2013;citation_pmid=23360514;citation_doi=10.1089/hum.2012.172" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=R. D. Mashal;citation_author=J. Koontz;citation_author=J. Sklar;citation_title=Detection of mutations by cleavage of DNA heteroduplexes with bacteriophage resolvases.;citation_pages=177-183;citation_volume=9;citation_year=1995;citation_issue=2;citation_pmid=7719346;citation_doi=10.1038/ng0295-177" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. C. Miller;citation_author=S. Tan;citation_author=G. Qiao;citation_author=K. A. Barlow;citation_author=J. Wang;citation_title=A TALE nuclease architecture for efficient genome editing.;citation_pages=143-148;citation_volume=29;citation_year=2011;citation_issue=2;citation_pmid=21179091;citation_doi=10.1038/nbt.1755" />
<meta name="citation_reference" content="citation_journal_title=Biol. Open;citation_author=K. Nakajima;citation_author=Y. Yaoita;citation_title=Comparison of TALEN scaffolds in Xenopus tropicalis.;citation_volume=2;citation_year=2013;citation_pmid=24285709;citation_doi=10.1242/bio.20136676" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. A. Oleykowski;citation_author=C. R. Bronson Mullins;citation_author=A. K. Godwin;citation_author=A. T. Yeung;citation_title=Mutation detection using a novel plant endonuclease.;citation_volume=26;citation_year=1998;citation_pmid=9753726;citation_doi=10.1093/nar/26.20.4597" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=J. A. Perry;citation_author=T. L. Wang;citation_author=T. J. Welham;citation_author=S. Gardner;citation_author=J. M. Pike;citation_title=A TILLING Reverse Genetics Tool and a Web-Accessible Collection of Mutants of the Legume Lotus japonicus;citation_pages=866-871;citation_volume=131;citation_year=2003;citation_issue=3;citation_pmid=12644638;citation_doi=10.1104/pp.102.017384" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=P. Qiu;citation_author=H. Shandilya;citation_author=J. M. D. Alessio;citation_author=K. O. Connor;citation_author=J. Durocher;citation_title=Mutation detection using Surveyor nuclease.;citation_pages=702-707;citation_volume=36;citation_year=2004;citation_issue=4;citation_pmid=15088388" />
<meta name="citation_reference" content="citation_journal_title=Gene Ther.;citation_author=R. Van Rensburg;citation_author=I. Beyer;citation_author=X.-Y. Yao;citation_author=H. Wang;citation_author=O. Denisenko;citation_title=CHROMATIN STRUCTURE OF TWO GENOMIC SITES FOR TARGETED TRANSGENE INTEGRATION IN INDUCED PLURIPOTENT STEM CELLS AND HEMATOPOIETIC STEM CELLS;citation_pages=201-214;citation_volume=20;citation_year=2013;citation_pmid=22436965;citation_doi=10.1038/gt.2012.25" />
<meta name="citation_reference" content="citation_journal_title=BMC Biotechnol.;citation_author=T. Sakurai;citation_author=S. Watanabe;citation_author=A. Kamiyoshi;citation_author=M. Sato;citation_author=T. Shindo;citation_title=A single blastocyst assay optimized for detecting CRISPR/Cas9 system-induced indel mutations in mice.;citation_pages=69;citation_volume=14;citation_year=2014;citation_pmid=25042988;citation_doi=10.1186/1472-6750-14-69" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=N. E. Sanjana;citation_author=L. Cong;citation_author=Y. Zhou;citation_author=M. M. Cunniff;citation_author=G. Feng;citation_title=A transcription activator-like effector toolbox for genome engineering.;citation_pages=171-192;citation_volume=7;citation_year=2012;citation_issue=1;citation_pmid=22222791;citation_doi=10.1038/nprot.2011.431" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=A. J. Slade;citation_author=S. I. Fuerstenberg;citation_author=D. Loeffler;citation_author=M. N. Steine;citation_author=D. Facciotti;citation_title=A reverse genetic, nontransgenic approach to wheat crop improvement by TILLING.;citation_pages=75-81;citation_volume=23;citation_year=2005;citation_issue=1;citation_pmid=15580263;citation_doi=10.1038/nbt1043" />
<meta name="citation_reference" content="citation_journal_title=Biol. Open;citation_author=K.-I. T. Suzuki;citation_author=Y. Isoyama;citation_author=K. Kashiwagi;citation_author=T. Sakuma;citation_author=H. Ochiai;citation_title=High efficiency TALENs enable F0 functional analysis by targeted gene disruption in Xenopus laevis embryos.;citation_volume=2;citation_year=2013;citation_pmid=23789092;citation_doi=10.1242/bio.20133855" />
<meta name="citation_reference" content="citation_journal_title=Genet. Anal. Biomol. Eng.;citation_author=G. R. Taylor;citation_author=J. Deeble;citation_title=Enzymatic methods for mutation scanning.;citation_pages=181-186;citation_volume=14;citation_year=1999;citation_doi=10.1016/S1050-3862(98)00029-1" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet. TIG;citation_author=R. M. Terns;citation_author=M. P. Terns;citation_title=CRISPR-based technologies: prokaryotic defense weapons repurposed.;citation_pages=111-118;citation_volume=30;citation_year=2014;citation_doi=10.1016/j.tig.2014.01.003" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=L. Tesson;citation_author=C. Usal;citation_author=S. Ménoret;citation_author=E. Leung;citation_author=B. J. Niles;citation_title=Knockout rats generated by embryo microinjection of TALENs.;citation_pages=695-696;citation_volume=29;citation_year=2011;citation_issue=8;citation_pmid=21822240;citation_doi=10.1038/nbt.1940" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=B. J. Till;citation_author=C. Burtner;citation_author=L. Comai;citation_author=S. Henikoff;citation_title=Mismatch cleavage by single-strand specific nucleases;citation_pages=2632-2641;citation_volume=32;citation_year=2004;citation_issue=8;citation_pmid=15141034;citation_doi=10.1093/nar/gkh599" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=K. Triques;citation_author=B. Sturbois;citation_author=S. Gallais;citation_author=M. Dalmais;citation_author=S. Chauvin;citation_title=Characterization of Arabidopsis thaliana mismatch specific endonucleases: application to mutation discovery by TILLING in pea.;citation_pages=1116-1125;citation_volume=51;citation_year=2007;citation_issue=6;citation_pmid=17651368;citation_doi=10.1111/j.1365-313X.2007.03201.x" />
<meta name="citation_reference" content="citation_journal_title=BMC molecular biology [electronic resource];citation_journal_abbrev=BMC Mol Biol;citation_author=K. Triques;citation_author=E. Piednoir;citation_author=M. Dalmais;citation_author=J. Schmidt;citation_author=C. Le Signor;citation_title=Mutation detection using ENDO1: application to disease diagnostics in humans and TILLING and Eco-TILLING in plants.;citation_pages=42-42;citation_volume=9;citation_year=2008;citation_pmid=18433472;citation_doi=10.1186/1471-2199-9-42" />
<meta name="citation_reference" content="citation_journal_title=Electrophoresis;citation_journal_abbrev=Electrophoresis;citation_author=T. Tsuji;citation_author=Y. Niida;citation_title=Development of a simple and highly sensitive mutation screening system by enzyme mismatch cleavage with optimized conditions for standard laboratories.;citation_pages=1473-1483;citation_volume=29;citation_year=2008;citation_issue=7;citation_pmid=18300207;citation_doi=10.1002/elps.200700729" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Diagn.;citation_author=K. Voskarides;citation_author=C. Deltas;citation_title=Screening for mutations in kidney-related genes using SURVEYOR nuclease for cleavage at heteroduplex mismatches.;citation_volume=11;citation_year=2009;citation_pmid=19525337;citation_doi=10.2353/jmoldx.2009.080144" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. Wagner;citation_author=P. Debbie;citation_author=M. Radman;citation_title=Mutation detection using immobilized mismatch binding protein (MutS);citation_pages=3944-3948;citation_volume=23;citation_year=1995;citation_issue=19;citation_pmid=7479040;citation_doi=10.1093/nar/23.19.3944" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. Wienholds;citation_author=F. van Eeden;citation_author=M. Kosters;citation_author=J. Mudde;citation_author=R. H. A. Plasterk;citation_title=Efficient target-selected mutagenesis in zebrafish.;citation_volume=13;citation_year=2003;citation_pmid=14613981;citation_doi=10.1101/gr.1725103" />
<meta name="citation_reference" content="citation_journal_title=Biochemistry (Mosc.);citation_author=B. Yang;citation_author=X. Wen;citation_author=N. S. Kodali;citation_author=C. A. Oleykowski;citation_author=C. G. Miller;citation_title=Purification, cloning, and characterization of the CEL I nuclease.;citation_pages=3533-3541;citation_volume=39;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=BioTechniques;citation_journal_abbrev=Biotechniques;citation_author=A. T. Yeung;citation_author=D. Hattangadi;citation_author=L. Blakesley;citation_author=E. Nicolas;citation_title=Enzymatic mutation detection technologies.;citation_pages=749-758;citation_volume=38;citation_year=2005;citation_issue=5;citation_pmid=15948293;citation_doi=10.2144/05385RV01" />
<meta name="citation_reference" content="citation_journal_title=Neurogenetics;citation_journal_abbrev=Neurogenetics;citation_author=Q. Ymlahi-Ouazzani;citation_author=O. J. Bronchain;citation_author=E. Paillard;citation_author=C. Ballagny;citation_author=A. Chesneau;citation_title=Reduced levels of survival motor neuron protein leads to aberrant motoneuron growth in a Xenopus model of muscular atrophy.;citation_pages=27-40;citation_volume=11;citation_year=2010;citation_issue=1;citation_pmid=19517146;citation_doi=10.1007/s10048-009-0200-6" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. Youil;citation_author=B. W. Kemper;citation_author=R. G. Cotton;citation_title=Screening for mutations by enzyme mismatch cleavage with T4 endonuclease VII.;citation_pages=87-91;citation_volume=92;citation_year=1995;citation_issue=1;citation_pmid=7816853;citation_doi=10.1073/pnas.92.1.87" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. J. Young;citation_author=J. M. Cherone;citation_author=Y. Doyon;citation_author=I. Ankoudinova;citation_author=F. M. Faraji;citation_title=Efficient targeted gene disruption in the soma and germ line of the frog Xenopus tropicalis using engineered zinc-finger nucleases.;citation_volume=108;citation_year=2011;citation_pmid=21471457;citation_doi=10.1073/pnas.1102030108" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Comparison of T7E1 and Surveyor Mismatch Cleavage Assays to Detect Mutations Triggered by Engineered Nucleases" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3.cover-source.jpg" />
<meta name="twitter:description" content="Genome editing using engineered nucleases is used for targeted mutagenesis. But because genome editing does not target all loci with similar efficiencies, the mutation hit-rate at a given locus needs to be evaluated. The analysis of mutants obtained using engineered nucleases requires specific methods for mutation detection, and the enzyme mismatch cleavage method is used commonly for this purpose. This method uses enzymes that cleave heteroduplex DNA at mismatches and extrahelical loops formed by single or multiple nucleotides. Bacteriophage resolvases and single-stranded nucleases are used commonly in the assay but have not been compared side-by-side on mutations obtained by engineered nucleases. We present the first comparison of the sensitivity of T7E1 and Surveyor EMC assays on deletions and point mutations obtained by zinc finger nuclease targeting in frog embryos. We report the mutation detection limits and efficiencies of T7E1 and Surveyor. In addition, we find that T7E1 outperforms the Surveyor nuclease in terms of sensitivity with deletion substrates, whereas Surveyor is better for detecting single nucleotide changes. We conclude that T7E1 is the preferred enzyme to scan mutations triggered by engineered nucleases." />
<meta name="og-title" property="og:title" content="Comparison of T7E1 and Surveyor Mismatch Cleavage Assays to Detect Mutations Triggered by Engineered Nucleases" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/3/407" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genome editing using engineered nucleases is used for targeted mutagenesis. But because genome editing does not target all loci with similar efficiencies, the mutation hit-rate at a given locus needs to be evaluated. The analysis of mutants obtained using engineered nucleases requires specific methods for mutation detection, and the enzyme mismatch cleavage method is used commonly for this purpose. This method uses enzymes that cleave heteroduplex DNA at mismatches and extrahelical loops formed by single or multiple nucleotides. Bacteriophage resolvases and single-stranded nucleases are used commonly in the assay but have not been compared side-by-side on mutations obtained by engineered nucleases. We present the first comparison of the sensitivity of T7E1 and Surveyor EMC assays on deletions and point mutations obtained by zinc finger nuclease targeting in frog embryos. We report the mutation detection limits and efficiencies of T7E1 and Surveyor. In addition, we find that T7E1 outperforms the Surveyor nuclease in terms of sensitivity with deletion substrates, whereas Surveyor is better for detecting single nucleotide changes. We conclude that T7E1 is the preferred enzyme to scan mutations triggered by engineered nucleases." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3.cover-source.jpg" />
    <title>Comparison of T7E1 and Surveyor Mismatch Cleavage Assays to Detect Mutations Triggered by Engineered Nucleases | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"Lr6ajouAPj8kyFOEJi0B6XXS0ahM-BzT1CBQCOhBP-o","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459794","apath":"\/ggg\/5\/3\/407.atom","pisa":"ggg;5\/3\/407","ac":{"\/ggg\/5\/3\/407.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/3\/407.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/3\/407"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/3\/407"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/3\/407":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222neqUbt5Yv0eHG5Sn8grHJyPKUmxUx\\\\\\\/Ir9\\\\\\\/D+Fs\\\\\\\/yXz+pFNLqBdpAG4X2eWsrLDeqozhIlNmDquifMkt9DGHKf6tfh495mUg14Nym2RNvUoYs5Iu43RryffDR+j9mXWnXmPLHMoKq5WCxWjX+T\\\\\\\/m+1g4Myt0AgAkGzLpQ3aWZ4Z+IfuwvStneC79ZCg0YHhH6Gv\\\\\\\/PVKlx4MoMQNh0RllYwJHIN5RSgYxo\\\\\\\/L+KHJwaog+v8MkQyuYE5umDx567TYj7h0S1OGDiucHLKhF9SLATBB1XD0IzRqhT+P9As7xv3PcVm6KsRe7inj+Eg+nNC6Ca9tEU88w6RAVPMmaSWWu+aPwN1JHX26eYZMKXVofOD74+tZ51AXpVNOPoQiYtSum2GBtIy6HmZBZZhrvMuew7YFIdQT5gSdpgaqhGTNoZp6pNit1AHFKUoGL2scX81Sok1rn1x8W6sKden0HSTEPf+h75lNL31BVxT9CxUuDYbphFxP+vO8onZRXkSxLuhiM3ip50f3VzMBQjEkVAsgi8jloAuga++FDiY1jYo\\\\\\\/P\\\\\\\/YHmDywXEv6Mtgx8C9M3WM81Jwo+cca6DVAhUJGt9K2Li4sV7nhaY2TkjUpvXL7DGX1ZbGyYeCqNmKbuVV4N\\\\\\\/WA+v9ft7N5P9eswQm7A3DpUR09atDk9992jLwuz2FYxgSxIZZNdbNevCrFldG+I9Y2g43KBs9SsamdFyjygqg9YPMrmHKi0RUPUiZFLkKLr5c8Or2iQU01j\\\\\\\/LvfUaH86AXIkvKTTlfrwK6QHqLKaxrNNjEGmViSBmSf3N7AUFeBKyFqXAaSS\\\\\\\/OGsRzgzQk9LebTiO0FpUX9Rbfaxph0B8Du99oonJl9WPcPNPyGlhMaMPpx4ubPwPMHZ+XRS77C0YWgAef5VvegvHuMEGtwzWEqo8MY5k2vLmf1niC6ciqvwMkUIQ9xKFXHGf8f50mLol1H9LdUx7z2lqyTpBIeZAzyB1lkowgr4+KkHAzO10BDwP9knY5bxnSj9OH3kvH4ISHYYuVLPlC9p9vI+n7XE8\\\\\\\/LUsDAyH2K71b9K\\\\\\\/VBfRJoBo8reKyc2Q+czrATTJDOSaYPbcy1\\\\\\\/EOT+SOgKfpTtQifISZOb8YiHDG+0QYhNB6Dvn6ruNHTM8wtvg\\\\\\\/ygXXHgDlq\\\\\\\/BNGEVdA90eG7S0HvF0iHRGvQx+F98hM6GS4Wx5OyjZtoERRTdHUK86mF+ijl0Zq1PtutZCTPWQc77TzLGskFx16auqghMUozyWWl8\\\\\\\/7doIn1+nRVgr2WJC0hZPMlvpX\\\u0022,\\\u0022iv\\\u0022:\\\u0022C2hCf\\\\\\\/UhqiMV\\\\\\\/vOe3OECwQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226507c61e5f7be5bedda1f14a710b44a1ff16ba7693fa7e1c00fdff3c8a9a83c5\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qbq2qZsYPPvd7fHvnYVhMfwdnb9w+8Fanp+RWvHsGpY+dZNSHOT6eEkVORS2poF9JrTW72AarUQxXu1j+Fr3B4imovTS8Xrkn52h4XGnQ\\\\\\\/vGtPD8KtYkuERAQMuSftyhfGLwAOREKoTZU+9vS\\\\\\\/kMIjCozRUE3XqNeCW6BVtVUmHgCT87B4aihuKfMTaLVz30vS5HuXwcpCWNAivdUgJIRrQOyKB5a46dqSBULUrW3BF\\\\\\\/FpzZUJZ3pzByzb07h4w\\\\\\\/KiofjD6pHJVxtO4vkJDasZieHQUVPODJSFl+Tx3iq55fw1zYZLJAKEHESR9E9\\\\\\\/Yioyw8OYUbUZxJ426+5HdiY8GOc7xnUgrcIn4lSWAj3TtlXqrgGeOrSU4+UGaTs1E5KP1EoptJe3UC+DJoPRHj8vuJb9xju5IrrqVmY3cvqMMVn6alrIYxCb2\\\\\\\/o+Uo2zMLEiHfTttZ1CkQq9N1JUG+ifSKPYJD6dGxPKNOYRJaFgi6u\\\\\\\/4A2v0B\\\\\\\/etzncCySUTzGLOnXLx9Qa4\\\\\\\/2thnCDRg3DX4mb0TewoolOkTPeckb8lVgc3jSoenJWYz95SeP2YgBAA28pERuwx8K7HySYVtFKDJ+Tv06zinQAfFTrkTABTRK51pfLnKZTPq8IOUoMe\\\\\\\/VsEv9obBK2RBE9KWfXOumHK5+yH3h\\\\\\\/XO7OVyxhsfKIFgyZUMu1oWrmpSdB2wcLgxinCu2DwYYX\\\\\\\/IT6pOGU6jc1W6urpbl4CrjqsmdODjeWDOI9nQ2rVOnnBlUbLgGseWwW1w0Q2ayGtWcrwloTmvFsncLg34bsXJduKkyvURevzZAhluDpvYaaC1b8uSaTpS6zABcpcw+Q8vlATiPHRxNCLF4jqX\\\\\\\/67obBYlRoCh1rUzC0yG7RPiV\\\\\\\/357OFDrCcHonA8keZdk06Lv6uEGg786CdC\\\\\\\/3Sb+945KvoryXVs7AzLFZjvs5AndZ8dEZ5V0CaNb2Q0xSmHlWsiG22xq9kc\\\\\\\/wih6MCXwsWTEdajibUANo6cBbswGzgKoxzAWCrM1JlorI6KbqMxNJxuuavwjXezP7mVrJ0WpAwRozrHgBHlFC+fgSE4K2KXLDbfkT4kF+voNCbmfez98CePi6XIPorglakbpmUxW0N6JdqC5VN6u7hw8SmPlQlRDX7mFiZ3nOtXHAOqKNCkdomPtgYBJj6wS+YaZA0lsPPEl6C9GecmIOt1LXDJ5qvmpumCJNYVoNvA9MiTXVpby1elv\\\\\\\/eE2Jy4ggMDXHL\\\\\\\/v53XYWJYWwCDfvZ77pdKO4qcWhN5KRKD0fZN4ZTjOOgnuzbNAA4eD4nlq\\\\\\\/RcRWXqsK+3z7dz+k\\\\\\\/2FWcWp6zqxHEPz\\\\\\\/H4+\\\\\\\/L5SDBdFvc5CqQCY5kJhqCdEqUTV2erS3\\\\\\\/g0frfNrFoVGeGCt7dJDQIAMywEkszCTypWtd7FkooFz5Ct+9UWmi\\\\\\\/\\\\\\\/ZGnIyR4VPtr18wHl3HcehE3g4UFegNI6C3e4DkVmtM+nRvMX+eEVIUkfxkitVQf9BKqTxPeoDcPup5cAj\\\\\\\/kwUn3oA\\\\\\\/Os0IQDjwNrGHzdraHI1SG4jN3OGmpsCLYo260LJeQytRa2\\\\\\\/j1YBBiOT\\\\\\\/rPmP0PiFzeM8P2N6lo5TUPrL2adJELQhfRdPaW4cpu0VoiG3I9DHg9VUFUMke8i9PvXnN7KhUpQdnx0603IyFfHnDUnuWZs6W+7s10DLchDY7uJWR27PoibDxIcPAlGvwlCqNInE5pzmDlhzYGxy527C7O6naYoZoMriMBXwCV7xCsTCUOs0L1Ew\\\\\\\/zg5HJL54E8un0zKXpLqaklMv0ZRji3ECoGuyWYTGW\\\\\\\/t0sFRUOfzUi52g7J8HBv8iCX2vtobwZDLAD8Vsh1MMPmNPwary0TQZr5xf0Ol39myYjKmDi4PEawmW74OY0\\\\\\\/6YPgWOrpGd5edrpUu7uiFaYCxiZ5X0e5zDgOGhONu5imiBjhIyzdaXofboL2upvTDa1MDoD+Z\\\\\\\/iWxAhj6CdlVZ9IzwGxfvg822WymhOzWwEJtAFdGrdJVsPbiCKqL0AFHW63admsoVnQLSmByXVo+FuMUXZtxnRIk+1iG+HkD2PdhdQuF\\\\\\\/pXizT5INGj\\\\\\\/\\\\\\\/c3BuQRrnXvZ\\\\\\\/iE7va7a4kKlLgzqy8p57eXqBr2UgSiRzEB0Wrq9zLZzPl9e52UBkWV21tv4zZBcqZ\\\\\\\/xbj\\\\\\\/rwd3h40qMwlC+WWimDnNq+7SkdFqhjk2sn0eYWWTeA7\\\\\\\/wE4NwwvzuVcQ6vQX2aMKS+XpuHZGK9l6sybuG6+sOk7e+U4LeYmGDBqs3nzuNDcpZRvRP2KYHf9evBpwlwvTY\\\\\\\/7ImNcfQlb0aGpeaGmmFvZma4owjRZyjKWrqD6WVkrmVLo8ul56J5ZIie2CVK\\\\\\\/Sam25ovKdhofIrZMBlJwCqKwIOmsEGMMVRD61eW9gZ0KqrklivK5+xaqWTTB2b74kgIkoqWjZ8nHHPn4cGZqSTKR6yPtc13D8ADZbLkwUS3Wyot4hSqpakeo7gYZ\\\\\\\/rR\\\\\\\/mmecQR2fxuII603KX+SSVFKVn\\\\\\\/oEeWrgffKY7x54aEXntPDuAR2twNiy8Hr9RcFb4QwDebz6duw6YiMIIfHXagKGbv3ng+IkKrFY3UcGwGbsJ3EkhFBQg1Cwkeg7EvsUEBpdaffkFfiyXrccnbdBxvhYxxk7QO+CdDLXpEd\\\\\\\/n99zpg5GPDxvDZGmOGv414ODXobU+cYMIcDhHlOx1Turs8lBn1A9oYCFwccA4je8i3cZvS9nTp3h1ucddZSlFQk2bPw7pOwYVVT02+7cWApAUgPyM9kTbWS8DS29QW0l2z78+KOe9h+RU2FWQCTCMeQDwB\\\\\\\/YdbFktU7jTJeT7oAD+LUqcxg8ZcogsvMO0r4sZ+ZzfOuzuHBABdNR9qGRC7DrSSoyZfiWbJ8hpGXamuLY2TjKCOuhmmUyEN2k1DFq4po3HmsKr1qIEoVihL9Zqe0gb8cqZOUwQn0DqdayLGLgIj3sB+gk2tlyATQGvsbvcr5yT4wh8rF848C4JbpUM8Lcda3iRpvU3x\\\\\\\/SD5dnqQjfecbIPXRrMWxagAYzTY1vIfthsAh+f3j4wujesPsam5QdsT6q8PcKY2GLCh0ZeSfc7lV2Or8r0mFcp4J\\\\\\\/uETJZy7g\\\\\\\/XYf+xKD8WXvCn3pctabULF+5iApD8CFBW8gvBU7h55o5XXFNwmKKue749Bvpfbd3IaPCV4D20+0Gbo6\\\\\\\/uHofpXn6T0ehPq4aYJu\\\\\\\/GROTyMrwqaWNf1rpbWXrjDm609N9zLUM5iDddFffksaNHTUc7V0J57kTEqkp6cPkfjNak2ZoWMVgL\\\\\\\/11b8S\\\\\\\/G3VktN7QyOLaJnvkZ4NkhSV\\\\\\\/GE0SojDSQj4tlz22BY0iV5hiXHNjwrhBdhD\\\\\\\/XD2sg6BOuUQ0OSYmr1luloIq72eFO0EsPKThaHag7nYvYzwFanMqUtKbjCAXbdun7C4va5LLHGyyR1GR98RogyBCwlsLxcUzpRjGISaJ\\\\\\\/NZ9tOhw6tRBkbyQmdWJAvTUw2V6yiCJsVQ5MKmkBeBsSTTGYCLvhM1HGmzXGnEtrw0Dk3AijrB8mes1VQCTVjD0\\\\\\\/maEBAiMN4RrGjcLFkE2pHb5TVWQqIaVDr1z7AS3yGdJdXr4KEmzbJQdaASPeJ2ADibEUzaXSdKKqriVh7gUgshVm3mEzdLFUVgjOAzWIUvj6fGen5oUgmIAYOO9lTsrzk6HM0PPyi90SY\\\\\\\/D9RfmUfjhOuloJixhpcSO2MajpSbZLTMstD0tcL7BNHE4kTADbb0cHk0dnd\\\\\\\/Rio\\\\\\\/5wPJeCe2s4WkfR0oPNAeFW74yObLpzuk6r\\\\\\\/yO9A6oH0\\\\\\\/2+\\\\\\\/hFaTmGjwL9mQSRQxd4EB9TEdTu4PxgFq3wu3T0fSWujHgCLiqBYeiZdz\\\\\\\/96TJhywZImF62iYbYfBNrkMCJNY5i0IbiMu48JzzpmIJcDEh4QkOVSljqqwIOQikNyxikilPj6RiwuaR0BrlVCIGZp\\\\\\\/QrO24IR8YHFH6kocOJTMBYKexw4AHfwMT6fvX2D741Usedyqh0IH6jPlSNYCysukSNctlZwfUP1dvu6xhafrATm5giRZB25kGVswg5a9Vces0JGghyDjEIOyNqbNxvVliI+eX0LaFipPo6\\\\\\\/xvtt4wLjIbpLNH\\\\\\\/YlKIFDMkUCTpxvMPRH\\\\\\\/miKpVOaz1a\\\\\\\/xQEGqMtyGFC23eugyKo6d6WX0ZfVs+yxon2rgzJxyb4Nb\\\\\\\/Eow925zN24I+BeKh7Onffzbjo4WAvuZ6cYlJVo1pXwcUxU\\\\\\\/V0HM1MTBu5geANfqRUwwX0NnYYzlosSgT5Rpj+zjkoJjYb5dKJF3pcbSjdKn8q2mHEHDkVKGANFaY8ZcnQf2natwPXWhMXdc+7lLT06Ua79Ux7zdjQ1vXLOj9EtLZTNUGQWwnGzVSOc1lJeJmg+W1VyPg582bTuMXT\\\\\\\/KSp2RTfHeGGGMdlk7\\\\\\\/Bwgf+M2G\\\\\\\/mBqrQVxUUcRSZ5Af6CPYPVoWHZ0Pe9kKTSw+XroJxiZqTto0PGWjukMhEHw30DHxvYJZ4J8gSzOJTdF3EPOhCovrym\\\\\\\/73PGQdJTKr9AbOcKmLHmwt3w9gSYHe\\\\\\\/GLOA26ERo2WU0DCuTKwnRTQghZi9hO6WirjrHxggSeLi1fhMssXdjbeT35LKlKvkEx9R1y7TKuAroOl7X1bD8xz6+5G8hVIt5VzXhweapNXd7V0rGV6XXZTYY982m5yhzSdn+hnDCEgn\\\\\\\/JSirVvwQoOdkCG8l3glsWrmZsvJwsU9bj32Kim9gTLf0yPntyVyw6whuEqViuYsMv1o8Ptu1ZciO8pOXwu63GDTTXRewSGuTDKd9IfEvzxfNit8D4ppQl6jOnw9DmSQ60l\\\\\\\/qR1QxFTLAg4gIsiIbcwTT3\\\\\\\/4UXWJHHBOtPQQ007vrdytg18K8GpBAEkNr3ghZYoCWp6Q\\\\\\\/By81W3Fg1zjQu9Xf+7Pxnse0E3aHcMYfMSIl0QB5ZKZ33uV+47jiatkmDN+FUv8DO7x9fceULX0UImV7TlFvgNqh5h7Kdx3wnF4YFcdRbB7Oiof53TVMdCDalZu78Uv7w7gzH4Q9aeiZmL0pt95X2NhoL9Uzuhfu3oV2sBhwRqbP2qcH7dSlREjl0KrOW\\\\\\\/\\\\\\\/wV+GHSAG16UGd0Oj8C7hzyeLFSBYn8T0+cH7\\\\\\\/PCZLdXScKLT8zG2xjaRWG49l9U7YCuOd9lubE7WOhaMvgx5ZOl\\\\\\\/bmtneCOpZLiHS3xlp0JyvPM2AXMVX2pK516USk1kwDuG1WWNTxD0zTqciUqCpJLIZCWkNUaGgAk5x0VvFeOuPVfkLOpPhc1N3H6JFM1vWNHSKXpMFAHWv9WdNc934R2bSF7L3UlCzGOm1SVyxH2mY2nQT2NTFdVdWvYFgwrRHnokBuFQ4vgSaaUluIVbhPDUGPZ\\\\\\\/jhsqJzLRdRd1k+R4MoS47KtYjEArWfByACcKpN7sd8zf1w+yEm9xG+hk\\\\\\\/K0XQUajZ0J+889F+VNZT2Jx4zs+Chc3f2XJJtIjtAnDRT834E\\\\\\\/s3whlXcBAQtn7uzsnmZ1u1HV3nPOUBlL3e+czH9QW5n+4iswidjIjz\\\\\\\/mcLJNs5af3g0Tw4VeTomrZVfhAnCYGHPIFXaIyXJqYH4DnUM\\\\\\\/n7KNn6DLP8uClvqnx6CzhyCdZPvQRrLF6D6zMRxZXAculyjFyz81GbAAei1DNdEWm1dPwP9r+QxY+Op1YexOx2wBf\\\\\\\/YGlCQLzbptaLF2AOH0vp9kRrk5Jp8JY9IDvp3PMSwl76eCW5mdqQ0NtvfOxqAY\\\\\\\/skVTEAZGkJu6D3BZConswpKHPBgNAicQhSyrnWx5XCofTvT\\\\\\\/Tfgb\\\\\\\/5hskJIKn4IEGzcpg+KBLrDyGY+gACexfiHD48zNYhDx4cGRRqK+Lpjb121JGxghmb3feh+cUACgZtuaSq2ck+nQyBLQq8IIeDjg+sI35Q6uJHnXEhltkKx6RtLVCnINUwIPDDsRRCmzjqBPYCPO12efaBtF\\\\\\\/CTCOsGSf5BUQAS3zC5MZc+7qogBAvou\\\\\\\/xLqA20Jse8BMJLiV8mJvY42YC6rHwpU7ks0o+nz3Knt82T59Rm6KzaZ+UYFD\\\\\\\/5Rx7UcSRsj+hz7lyHaZL1opHStTB80bJ\\\\\\\/KGRDBNguSgFGpOoKzNy9+0GK5DRTk4mj4cmIZr1iNNZ2CGYGCFmt3NvSlhglgmcpEjR+EJvdYTIl4IOUTcQ3nMiB10VoabNWGaHbvIndeDcQE90Y+GXOj9FP5VmCVFj8GWXJ36zAbO8Y25+6252mHCqLvyPZ\\\\\\\/I4ecnc\\\\\\\/1EXoDWFEVCFLaj5K7G8RcG75u+qee9Uoc\\\\\\\/n5Q2m7TOmxnOfUv+CWsE95lOs6cdESLMwNt3VI0aLaqd\\\\\\\/fnCT60L2vLcFJs6\\\\\\\/iUXSMC4wjxqno+44HlV6g7iZn0f+HwKT0\\\\\\\/ltE7cDYt8eQ1Nz10aLWRvCXjOVq+OYRqedJRf6xCJ2q2oisriWKQ780odit5OTMwEA4gcSfF37+9TTLiXJt5\\\\\\\/XtVhFJO+EmJAVybuT\\\\\\\/WhvljBWYppOh82b+saGzpfYvPvJAUk9O4DBPorPDsNz53Ln1PpuQStaBO2qEqmPa+n1guUwkjI9ZTlPxT8dUGXoGBc5tJ4gr1DWdQnd3HmbMStWbveHMzgfj2MIzXcbmqbVwPmU7hNeMSEBTYiKaVEwhSN8TdrU8GVY+1KN6PGyNi5v1+3sMQlzUs0JPXbvV7wPGFDV24+ERybvUEAh5moxCacSBpsWEWQ\\\\\\\/rBbXIeBfyv8hTcwJK8TZiBFnP24It6GN9vUAQ6AjOlKJFnhxgrzyQFJe89QXImierlhUg2zX6OuQaADwCeVqJhjbbP1ylNMxtC6mx5AeIG5NhNgBwa3qKAB6untNK\\\\\\\/Q+5PRqWoJwPHE0kfu5EqTfcYtdjRg69Tj8GBEfX5TPJuMSD9Z37poqkewVgMh2GkLr2g97ESsBa4dQn8v+dYdpOPS5Djt8SBRZJsANR++SBxzrlelC1Xgj2jwzdZF9nPC8WqusIy5hSQaYCqItFb5J0T2ULPQ\\\\\\\/OQX5h4dqVOl1IR47JfgeBAGL2EiccuHf8f7drWgSw5C0d57G2Uzi1PbBA+H7OCDi2+Zbu\\\\\\\/ia4CqIk7Dsf4+U8SJNWaSfxdMqZaIb4lK5K0gg3DfiWJXH05UHj1OZyKVBM7vNCvyKlRZZGtQl1pGe83J67mwdZK6DTrMAs2eHyOb6zr87sUtShDcyBmrAo2jiMu+YmAG0wGI5AabKDmD4yqlGabJ\\\\\\\/iRfK5ZOZhIaAI22h9r9nVItgtl2TMCF4yq5\\\\\\\/9f6yiqLFZ1nQhNHnrDiwxJYs4tK+huo8ifPJt4wU3rGXlrDwzLifX07N+JI7PHWMxslkJViCNmnBPN\\\\\\\/jq0H8YmAImsIwJZ\\\\\\\/S8ImIR0UEacLO8OM0cvDEMMTLXsb88FTj64JoZUxLhVHp6+x18IR6gRLmg\\\\\\\/dmno\\\\\\\/3UdSa6PfwEQCWUY9tzmFcRRPpE2UdnMyBD4AGhquaZkQXK\\\\\\\/5G8tFU4oWwRJGckpotPG9syqXF2uVbGHIvk9Api8BofD9Dl0KP+cHMxqqbLHtjQt4rM01bxhv6A4yCxWUhLSdLMfu91O0NeebGJn76uOIgcaIhuLutAN+hMHOff+EQIzM0FeX1eWKQY0RtoBVwteYJa5DfrrQWqh+LvA12bEQAuVcUOEGq9WujCTFxkUWOfN1eQsStjWpS\\\\\\\/RSwwhyu2aU8p3t5ynR\\\\\\\/4pj7FpzlahSZJ0fr4ajfE7ei3\\\\\\\/llUN5CSZVsz9r5atZ5D3sKQUyBo6OlePpu69Uf04myOTS3oUySPOgl0mlD2u8lG\\\\\\\/ukfiXF4\\\\\\\/aE19NecWv7v4AMcFc7alFDQ79sDGr0C5Ql5KChXax7L0JL5OGMRzqNLQWrGxHBBsKZLdjIyw8IKmQ4NNKWCh0tSvJLoM8dKbuU5W8pw2ZS4YSVYoQpRqumS8tYLF\\\\\\\/YE7DnS2gPdfOvkqgHoFVheCGsdIDA4ogZR\\\\\\\/bmTy5V42jG2XPy66B9UfrCj3R8nKhPvXHq3RXEydlnotlvVtQjBV1dk36XeG14IbjYcdcIzk2e0R8hH3\\\\\\\/yPbl0MUqHYgVYkMOSvTSOP7xJjBNHqzwWcJDwECHyozALtlIETuLCGubEXoemDawXQ1ShXCDvOfe7VpLxVd7S170Ioi0GBf5hfu5HUT\\\\\\\/8\\\\\\\/v1Cxer2I6UbO2nx6qHOqMoG9npxi7V7YY3lnYmiLN5bs84VPmbYYuPDl8AJEGTIyqcCGHdU9ZS5BA\\\\\\\/TphoEbj0lkgx+j5VH7Jq8+X+VPUa9nCGscVOIKPCVOLMpqcShpt+eCwNkeyqHOb+f9sM4X0ItARZsoIHso9o11MTWbZr0o9w7TMxnFNyKDOWaWVKqvwAKVv1sZBeBSO86GS5\\\\\\\/br1mMguYjcMC9pQTd3heH8DvETIl4oKoGJgMavtquNZg1th10qVYdywNKyyMe+zAa0szRiEpB1HN1MP08cdK1zQPZr2WAqNfh3yY7G3X5zJw4fbizKRBOBU0RjdqHtin1nOVFmKFhAbix9f2fXZVS7EFxNo1990FVaL4wr0pW+Rh+zeALlRFp+BbEwu82IU+QhQGR3bciMLaXLEeYAY7FaW1fsB59J7hP+oxPQyQqsbiOzynDhCAfR3nskhpF9gWqR10U4McU+XBMzoQFDKXBZ67Yc7WfD6JGHyKEcZv1aqQg6N1s4BWou0Wm5HlDt46kG3yQG2w\\\\\\\/T4Jd3jE3uhISIn59KXgyWDov2Sd4BCU2Iimi4CiNamkAqxSKXjlevdK9nYbOnSz9coGVOHnPj\\\\\\\/Ky2\\\\\\\/fwr0txlNjXZB6keO5evW43ivJhg4zbUTr8zaONztwSgavU1P4jkeqfDjRzs1T5FDtnWDL86smlAne+5wXq52MN+kDhQ+GTVxRCzJDu3OVb58lHYR6przxuZugnnybuoERzc8LXU0zgTrKe87uT68k+XKwK43FAQiHVBzDb6u0mU2A5\\\\\\\/dZffzSmgM6E8CQXoXf7k6q2ASCHMzieBaBRNaF7zYXOXoqnoijioqsMzQQ7B9lTyHLj7uIJhNmxfgY2nU2AW6chHs59MfTORl6dgTz5+YKVpZ6FwdyOJE0+B\\\\\\\/uFa9Vfalbv0UH3V7TzJPEpIVsKDktW8KPbwhDH1THprCJ3bhnWTHDVJ8BVfoqRh1x0s5TMP6ehGPE8lULUg9xlL52P5JqXKDy8+q\\\\\\\/pF\\\\\\\/o1MlAsVrdGOuRr5PQUZZst4Dn43xiq1bHEbcp3lREEkLsK1QcnLdUNu0oTkq8dHcjqyETs43w0\\\\\\\/ipZZ2i+oj3Ag56n8hHaPq40lzHmcoi1w9PCyynIbIx\\\\\\\/MeIC0mJbDGGpn8rO3sAbUCXg2a3QdyViTVm1OVcmD7smTluWB9Eeeh6FHNuBw7P7rcBNvcOE\\\\\\\/ZmPENx2\\\\\\\/ABf3Sqes0dNGf7SJEHIdhh7RtyHuRgwBzjWwZVWeRVCrZRDxpwN1+4XjYhn3DUm9Q2WnNmN2Fj5nzieFc3QWQ\\\\\\\/qTPyFir+2+UFgy\\\\\\\/3yfiTW35GqzwmnfgBeoepCi+Io3LgawLJFbYh7ZdjMLd\\\\\\\/IlycF3hXgV9p+lSV7wTfxC+sQMFOl9VF8wkSTMkezS7OAYKuDKyoB1FV+cITfQANOLkSVo57h4pqmj7ueGmU3oEPXziyxbs3c1S3yFjebTXclIR2oUZ+pAva6cuK2RWk2Ldbc9uqiIIyAeMYBpE4\\\\\\\/QBLhPKFhmiMpj05ls6Y4l4UnSVQsX42sTfQhS2UiX178KHh4bFwPvFRKJ1J8M+jGQoyURqeUYooh4eoZZ0BcokblbhW9IyTXyKU0BXWjA\\\\\\\/c7EhHmtYhidxlgr5bNZ0DtrGZxSlN+LlURvZrjVAXutajhfXFSZuSL6wXGPgJJfXWsJv6k8Fb\\\\\\\/hBkNFcnxp\\\\\\\/Q8KaSvtylmeVfPcHhWQAn0k7eEkhX507mkxSc3RVI86e7QSRdXQW+NkIzwmX+aEjnUUiUreJ1BhonzsDngdxyx8QFSweZiFEXzU2ix7A7Ns5TjyfrAnZ2BTAa62jg\\\\\\\/LnEi1E5SVVmrcN5OlY6WMu9aMAe7DLbsyw4mQ9dTSGR5QHiNOtaJGDqyFRg5YCXJIWW\\\\\\\/e\\\\\\\/GN1Yo+qew7qf1DMl9lMXl2Hh7+n3zNuWkXDA\\\\\\\/P1lqRbhrLbAaCuoZYUXgXVzrKR+Px7pWWJXcgz0My7X91lCUTGBKPSUao1GXTlt829PYLUyLsQA3G8vUY4S8iPAytYrfAUv967y75RBxCPCwuYa+uqIhmSLYUiocfQXDA9ftqZBcfTZzccMLHzsazzx2Dyl9FpmckoCVkXFIddFo\\\\\\\/By2bF7oZ8InNQGU5SvpiaXHPjCKVcRHIqk32TKBIKikCsd6EpHRhWHmdge5yAXGJgcdknQPT74hM6jNi2jlVcpUZET2bHqKGTaS+H31noNs1ZBpLJlhearlF2t0KaH4EZCIBQEaPtP3z1sJgEnS+toTO4o0TW8+ieUhedxLMhl6CSm\\\\\\\/1KRnI8VlOq7t62s2xmxII6Y73uoKQ+XbPbVy1G8Xc3eUYF8H7CZ2r1sYHAzv5z0raq64zj9NEuDXec44ndU9tn38enpxKwmSoXDCjpSXoF2gT5Mq+n7iRX08IusxltUoMVeP6wKpsU9kPQgjpHOfSIBu4OsCwqv34pJVGx6Ws1mOhxivcObQGMRxJVLspHm3m65q\\\\\\\/gEwbDJkzx4t0cK\\\\\\\/yMQpY4Q7wjP1hlkNoOcG34XwwKJhYObjycRY+2rNFAvIVGTQBfgDcvAFzS0dhRp44f\\\\\\\/ATb5758W0dKx5\\\\\\\/ghzclXAclGLqO6Ehh3PSogDNYTZ2Ngr9Kr4ORzx0Ndx\\\\\\\/HjbZ+\\\\\\\/MxeQpCyVAjKRqr\\\\\\\/MpytUOdn84r+yDETvj88m6lXrgbZF6Z1pY2CLfnvipaBpzzI2dEBuGkKUfM+dECs6jb9IqEFSrTs+UHJBazZ7PnrKsT82jSBn8Rr2AxsfGx2RNop9Cptpi\\\\\\\/yvayNPEtd+nkcxZHTuuHVB8\\\\\\\/JiAlMsNgwJx3avYE2lKBMrvft4hOlSF7Za3ZK48D+0qGE38hot5zKRtxdAZRkqEMsEmF0NqtFL8n6xpSMEOdFiXCyqi1m18YzTvJGfbJZE\\\\\\\/jb8DAMi\\\\\\\/1cyb7xk4G+zeyn4BmqSM9iLJcF1l2bLev+tpJ4e2U40Ev4yjvC9dcEUzRlDxBsYCNl4nTehmSB86GsdrPLuzy5GrbZDn430Vw0ZpppZ14X0WYZ1yCzgdzKfmIHKFPMI\\\\\\\/m4NNKl3Zo5pkYzdIOLnUDVwbMt1tJa3B\\\\\\\/YYw3c9RbSoi39ErklPqR0K2oZ5\\\\\\\/WAWxi+Ziz5cRc4IkM1JLQeVpPeC2YRLFlzlsWJ98\\\\\\\/H5GC0WoI7du89HRa9ubROZaB2UuRkXmiG\\\\\\\/ip1kf3FwSh8Pw7kcFIePkkpqyeVpBNnNVoLu6xncAhaqtiUA66SnDMBEoctjfMVw398XT41djlqBy2WICedcsKh42aFw7LztyIG0Ii6wNVfWVJCjaaENKkjpjairIRMmVLHFpHk5KGIZnO5LbTF9\\\\\\\/cZG1Bk4099\\\\\\\/b\\\\\\\/+cSTeqwpjC77BBF3ZAiZGw1thuREghZsFm5w6VtbxFWCIYpuhR\\\\\\\/wXXcezvB39f6ofj7W51AHsbVZPNeRVEznKSr6UHQP2U+jJg6g0NnWELNMHZYNN9qblRCBz0HZFLZ2Ge0hdQWLJBI7P\\\\\\\/\\\\\\\/eHUIKBuCpk13aNXAuJi47gtRDmOPCMk9OAxym132Nnnfq61z8zJD44bna442XS9MbRVv\\\\\\\/IjUWf2Pxb6lX0X6qxRfaySMRjxosTKw\\\\\\\/6XF0uIt72qkoQCQpM53Js4YoHy0D5cfua2RpJA74qeoI5CoRkAPZRMs26HWUMXspo1HfUqp3n\\\\\\\/NsmPZvl63WdqeRe7hDtH8TwhSyPy277NwPkoi8wIMmSbiOmgby0t74YZY0bnr1\\\\\\\/MfrN5HR\\\\\\\/uSofIMv9plCqkxQp01JLLkDLECeCPSRfohu45DxL4qVGouHJArGZYBByXvNKqJlL3qqDv\\\\\\\/26Rppd\\\\\\\/ex9UrV7aqibIuOR5BWInwQ6rE+v3bdUUsnTrwgGLAJBJBBAPNVGoPpD4\\\\\\\/zJc8cEEwFMVAy1QGGevW3jvthaZDUR6xk+Btf5xPZMShnwXhGfGMd81bO1tgs7RziGVIeEooXoVGdYXFfEI9G3F1pirlOz7Gt04KLhl1LkM55xETf0IugiFjz6r4FkMujTkheCDMnDdXUUmCSXKh\\\\\\\/\\\\\\\/70hmRlgnpxNs8fXKHMTaqWvBBRiEkIIaMK+qXxyXWnsEgt43GIxLqgEt7lSZc+ejOPEUNyi2hkXzjhDQFEmMtpkR7+R7RDPBrEh+st5LzH6Or7aKxsKPPBJM2ya4dsVB4YGM77GgMkO9qqX9oL3wMsPvsFjyxywS69FUFvPmwYrjyVawuGcQoAAk1IiLP77whXiwd92ANhm6squ+TuU4apOX3NXnx+aa+q8T5uq50TTRJlk54nF2JQjDc71n3GuhntaVSbnOSHOlg0OPtS5NaFXWoZ8agrpOkmAF2A9yYLfmPubLLrK0Umep5t2P2tH8n87Z\\\\\\\/FeQ7njZ4M0hyluM6dcBU8L0kdZkAq+ZvQXpZPN1nqEnoq9xX3ADXZ1SRQGTAacUwcnb9PhxViiMjqAkE8rfD3Jh3h3sl4044kqWLCMw3uYlvSqGcuMLy+Eozvw8MmqFyMEh8UMijNVP81qghxVvyjB30To1xu\\\\\\\/uzKBUPiTTN4+oScR70C0zHf4UhkSN0tNmdOtTFqhijZyoQZADvsePYd53NRb5ay6bxxSpm28Jp62C5Xp9qYc7ymY2QRqIP7h7d9bsGDTW4mQE9pCxaQ+jWdHTq85Bb5yIUs0D9i9UUDgirbfFzHqCFbr1FwCVvGfwy1HyBX0qgRiS0f5lZa9v5Jk3p3doUK6zk8HLd6uCvHyZpekRaTVLDuMINsHxHrPbecrBEi24YM+bSdqOQNrQR2Z7QKm9LIgkBX9aJ7cqQni1E4NE254+ocT+tnQHf9dZpR+0vtA1AvJKLZ8bQE0a4mfsu1qHkzhtgJYxMHmbrxWBB1p1aqYt3LML6KJ1M6oPy0TpA\\\\\\\/hnvfZ4aMI8cAQecf2wGcCOI\\\\\\\/AI2pgWFywo\\\\\\\/8gYai2FhM7\\\\\\\/WHIOHSuOUTw4BqwyfT95HJWE2LeE2fayUyiQ0Vufg5C1Pon7B4J80zv2afQmdbVhZ1xNm2sP\\\\\\\/5MEEemgq6hfe6psc0HzTwpPn7fz8IT3Zk1w7tdk\\\\\\\/bBeff\\\\\\\/b0F7vhxYS5vIHHnmd05fPw8r130sc9xpvzOh0XKoLDWVV8uSZr7XjGj5Wvuh+FVEKdYxXzNNEA+UQNshEYORPn5UZCr7eSrx\\\\\\\/42KGKSR+y2J9POoSfaXHuTybrOj+pkYYO+DwS4NN1fjsOHK+GRjozBS1yjeIarnB7440WqIz136PYz7RMjOlHcdeEyqOeAZqgqfT8N3xOlxIea5vPWCf94EX6IgEImNfpdyupAKCICxkF15d7EKUa6lgvWqMtDmVvdD7zUBJI4j+bHx5jerXc+CUERNrxEfuZ4Iw4dxa98bQy51hRyBQheznYs6niblpCQuj7SEcT26dCLWLbBySBjFkQ0xNU\\\\\\\/C2t5L1fzPg3y3N6qZp3rKUsNoBrPCS2FkqJX4gTd2MbIfaJva6FfUb\\\\\\\/EsoFgvscTzsYiSTSzU9mWfR7qgc4otWbY94HsjevIyEj8FY5SFCcGWwKgaSAc77lUZ0XnegwW5yNiSe1KO0vWt+Lh2qAwn8nYw5ftkRBI7TwxItLegcnLSh2KWURD98YWJtOrEIEJwFFd7x225p79tIN6mpx0WE9OlIkUClaVyxV6FjuQuTp98N\\\\\\\/WO7pq7pftYa0fwEhoMx+cLoWx3DEwY9BWJ0dLg+7JOeGY0iLLKrUFJi8bGHEqTDFapAb6t7EsvQ8hIKB5vEDu9oN0DWhfuVr6XIP9iKCmdwuW\\\\\\\/iJH9wnA4Vh21S9bARDOEPy3gF25BXKjKM1Gb8fTZPIayreMnyyV1NDLf3zYoJUUUos1emj35sS0yW6lCC2UTYHnmQCQhTRD37twQah0iUlxowl6VtyV2Ao\\\\\\\/fuHnj\\\\\\\/qaRCv\\\\\\\/3gpxdjVsT74mQKOiF8LLVh56vxH5o7Hau7NAcYbKLE03KKDM2cdunPqHN4CeLQdyi9uz3YrIolt4FYtoRlAnzG7e17Ca9lWgSZDW7mipRF55Sk+lp6xhDCOj3vHIOqQ1P0KW8Ln\\\\\\\/p6MImXnm2m1gMbqezn5SrNCm\\\\\\\/b8DLDj21ssc6YYgNUcwXL91IasAm9XGg0tfv7jgy4nucKfjaR4sE90ZrdWWl+U2N9WBe4lg4UBUALlCvRK83eFTdbYiBh58eCcI9LcZlLO66uy3n2kXPK4za8Dm7ptpuv\\\\\\\/qoI1jOZJYpL7FXEm+oxUWltiGzKpW7RamrImvKQiUYGT+jkyBLYYyZWIwfo+cJfReqjgcgv1n6zzFHqvqiyPbVmA0V\\\\\\\/rMfSf1EJ2R2EaXPpYxOKd33Zh3kG558iNFyDOXIcD9lP43Ig5fRvJ4vvqDZpVvNG\\\\\\\/z3V7JJ3U5Qon5wL7GM7p\\\\\\\/7+Q6j1MsazEfXpQb8qob8vXzZO4JwIMiHpo5xSQgQfDDs4MEfxOFnZeyEnFUo7XlpZfKCUBlDDPY4C9bO+6Hu3xukDLP7CiayswTWoc+6tvMugp3IyT6nSxNSAwm9NN7esyjTTC7CSABmMZHUKWqAFKn9vuF797pg5yjEuZWGfRQG7Y+y6HgxwprIF9KlrBj3pNxy0ne7gOwEMeKf2pHUwKfz6gbgDBaTWW5tOY1U+\\\\\\\/Wz37PEQDUtlWUzZVcWv1FpFeriqfTo8i2IJTQ4+CHr6RTE01JM4UPlMOboBpV\\\\\\\/FykFOvUfqR7lElsY8dD3Q+P+MkVmKnQItp64yBQFOneecR6noaGcfdqotmxyCCzDnfGH6x9UlTn7sjYviWePc\\\\\\\/g9ADBDFU8xgdF1sPRzQJmfJcW1Xlg8+fmvgqKaSDAjHOQiM+EMHXgqPQnJW+j\\\\\\\/r9j38qtB6nlKM\\\\\\\/tCYAfVMalqJ6aXfWLhCsCqjWJXC+yI8xK654s8PoYAw8dQuvpU1U0GoD8L717XCa2IM5HxQxPjjCnLDmJYhFN\\\\\\\/QDhaq1LtylkIczFcoaUJX7mZXFopf7jYLaaEdyuxpLNYaehSysmNRnw3QRxYhNM2WJD1KWRGEm6RriqVA6cwtI6JMU1kneXsVgS9PYVI6RKpsKz7mdd6lCMpj2E8EXDfpOZbugAZavmAnKRCyX5HHiMHO+Qr70R3WtsQb07wcY1puKZgI2y+vd28rjifijgO6P5C4fzYUjnpLM6RNfR0+UPgGRZS9yDgcvtfn9NYhyeYHnimoPj9F0bPEQS64b98BYYzeAVf1HNOQAlsWBCOV\\\\\\\/Mumny3lHy\\\\\\\/p+H+1AB91QruSnc\\\\\\\/U3hUEbwYxwxKnyF6avRqGE4ZHN2xzB+RrmYsINsb31plAj4ZON7TyoEtJYJGLo4uzbg8sNtw6LNZNWzSRcwoiTsEjBsz4GTDjQyTdqwNN\\\\\\\/7loAYxute700y97Pcv7f1Lp\\\\\\\/dqXdfhH21lGxIMIz\\\\\\\/YJfIWkBzMwVW4oDmtDdmEqd1Rpmtl7zpqtFbaZ8YvsjBmhyuPAC3iQz+t7bFqxl+3VQ70Vay96l6\\\\\\\/nkp+v12z+k2vYz\\\\\\\/NX3RRU2tM4X1aFf3kKrR5eVCtRnSeqbOeBh4\\\\\\\/BFkJTdpQQuhPs879exdkvgcg60PGLqkIL+U2TCzCXydy5sXHzDMtP1BmYK13tDzNGyo1wwBJuHCtJ\\\\\\\/dF+7zQXy7AWuzk\\\\\\\/BpSVgHc2QLOphVssWFN96HhwNrAh7H7g+IaaXYqXhMEGTnhrI9sfzElRruNeTZVcAFN6Prv0enw08vOquc\\\\\\\/jAC+dXplSIT1UuiSSoHYxBIHFsv05a75G7RXYlpO6eKHz0EeqhjpUJpO2G3xTkABGSRrKZl4KSY6F8mFwsnnLZIaQYpsb4NpMxblfae41\\\\\\\/0NB9tpoLeHJTaR0kLP2sPrZqWX5+dv3OYk87YiO+Fnic8MN\\\\\\\/8olblcM\\\\\\\/hG\\\\\\\/Qk1m6TyDi\\\\\\\/kZckh95eZkql69sxlsCof3B5WkyljfbqqdVoHS2KMREDu4hgrA06soY+wNbmtbyM2WldUFpQ7ziQik8rkv32uDsmW8Gh6gq2ipMK2pubHdM5vSzSc5TuCNBaPSzc9EgnWezaMmUL++dmqwbivSQ1lQAPIVXPa6Tp5I71yTcj2fXSynlZdUHyw+lePmteK9NgR7VOJ7FCDIBHWD9FyUKoJbE1yWQ3KrHhNHhAzOQ2RA26ghNs9JzNOjN69n9y7KwYgwpARxK9xdQWpc347f5zxbPVtyKWuObLUtboxTzAde90IPbBPtSZecWjtfm3VwBwM7uoDLl1OAekcXsGlgh\\\\\\\/nmqNYlApOYuYatIkfNWWVp31+ep1g3OEcJx6wVlCbK11SMVg2kl3na5GrYH3vtc6zb7f0NGdGvJVDbWr1az6TGWKzzfICBl+kW2CMg41rHT\\\\\\\/3jLMUV+liZ7bW3FeMoOIBJ4Lo+AIbM3C810cKKLNrpECw6rZ5uCQ+j3HWTCKOYwWcy2OAjZJH\\\\\\\/rAkyeHDiwiiPlAbwQnQQGLm28Is7KYOcAC7SlvqWF3iyhJHtWbLGdygI4MxMyFW8Z80JNWzjkFA4VOv24VL8aEhlZ3tKEn6sxCntbXAvT\\\\\\\/HdqxgwRwOvXt3H3TiLMNl7edHIUm4dbRahjFKd2g\\\\\\\/tKXFD153XP7+qrVqrBVrHffV+rvu0cx+iJblBDrcs9wG7lzuW4tZswkVMmthRHWVyn+cNxl\\\\\\\/+8MhdY4ATgKaPmuhAuR39u1TzsIhnPPF7GOX\\\\\\\/zML1IBTB0LRiyCQraYwpINVzswEH9V5vIBJWQk\\\\\\\/aCG269frUD33vM8Ui\\\\\\\/q4ZO\\\\\\\/XJiTgW\\\\\\\/7ngr9f93mhSdhyI4mVoXd8sXTdXZ3FlY\\\\\\\/QWrqxcM69UG6yc5R\\\\\\\/oYlp4eiteqClvctY8udGeGXzcblmNcuo+VKD8LRp0rk6GU8HpsTSopn7jHES4Cw9n3Z+VIqcZTH2wDumNrmSNk83ZPGNUY2QnXfrnQl+AOyCEfp3\\\\\\\/pJO1PkM8n4Yfh\\\\\\\/nRmLnC9Y4WybD\\\\\\\/ze\\\\\\\/XlgRj8dNs2vgZ5USICTGCJRXWrBN5EdyUCG7LxbFLATeXUoDJFWswxgEbO\\\\\\\/eWhFm0XmSUqYrdJNJg8UPVmKHoBxtdLQZZHAE\\\\\\\/KoD1aNU\\\\\\\/iHyiIQX55IWSKoUe7XxajfqgxblMH8TmljWckCPwZHUhAHDpnT8Cfl27jnZMqKz3\\\\\\\/eUPZq8ihy3WDbr4BXwYXG9norgwffdJWrszx6R4d\\\\\\\/5y+47XC5usbq4aWTqUtwT\\\\\\\/8W6XO4+Cw44osoUEEiMmENkow1AV6L1Kshv4zgwAI0ltYvdkzofvDV2FqCsnRIVu6Tz+TLCseFjjrQ2Y1smWks64nBPkfmKEBKalVwHw6rENPx4l+\\\\\\\/B\\\\\\\/Igy4ZPoFfF999Gzxhqw6k9l1nL3oqTkvhASezwkaDAPXnV4hxH5y\\\\\\\/Y+hklVnVuN7Qycezz3B0d9redAqOFzulLglyQPrI7xxECFh4U6fHRgDwGVwg+o5Dcw9pHaIad9n7aP5Yg1aRGFME4f7QumaHlPqQ1mwuiEnLvKdPnovaBLr2W1ug2DJ+kcoik\\\\\\\/dR3oRbJilgYaLYtLLoehIfDQplM\\\\\\\/97edOf58+YOWDMnZjI+fdQL6bL9rTu\\\\\\\/XjtAnpQe6KqsU32mc8MILnypI8glEkzaxlnrzdDVIY4Guqq9QedwHlPhzuC+cs4WZ3OHBOWqWELOWp8s87nBu0RfV9TWfs5d5+YX\\\\\\\/FV8es66DUXHP7S3Rc1jMHYzQUDCcE5Rud2nVSFC3bmnA5O+tCQ28Uae\\\\\\\/B\\\\\\\/p3TH\\\\\\\/esA+R\\\\\\\/huMEB7M+wR0aTvtxDJmaBGzJs2vxDHgWjzYaVspugwQbpNakpVQWxcOYKeXzo5fhPeOag2JrS+mcrCLQy39+4F3xOC7CkGrj3cZL8ZJqmfyJl9Vh1YQGh\\\\\\\/KaNtTMFPhWL4Cmqs3GWbaKWlTAbv9ekaGiaMIcGowjkBlHyFeMgeN4dwdWen6+sQk0ILh1x9n5xuRKMto8eBuspcSXl7EYT2rJS6HcqRnnV23B7cuPYvoNdR9uN1Q\\\\\\\/6bWmCwv7rhGT\\\\\\\/b+K46kBGMIc44NbfbfoHU1LnY72Wq2NGHbKkQeyhGXM5cV+l6J7mf6+QASpObc3DAzHI8sbW6s2aTXOd0A4+XZrIhBmxSxvjDTsHUXkRZQWUUi0VcNZZcg\\\\\\\/3lc5hYk8yhF7PYGSXUzATX7yVIDgrn5OOAOuyC4UR4VdHWQ\\\\\\\/cb2xgcLki2EwtPGdmmQai2EsXD6IEEao3h2\\\\\\\/YXIDE9xTO3T3WRy3ixNYZ3a6iDRFB9QQtR0vliYDoEKOZnpnkLe6VkRWufmWMuuXrsdPQ8JysR25lxoophQCLudtHv1x1Zt\\\\\\\/Fc6xHB\\\\\\\/RoeG68gik5NbYybO2wN\\\\\\\/nlROtN9lAa\\\\\\\/8hqWwBIq7QuuLs\\\\\\\/CjExhAGBOZhCV1ii9y+Fhm5iMe0auSl+TmU4g4Ad2ESjXnHqCYAZxiYo+kfgH8uUE3y6nWCfOvWxS6yK1NAvVY8CE1peQ+SrmLPTTHnOh3n9AFsMTrofCeu0n9nXwMw0G1b\\\\\\\/7xBvWxA8mhgWvglxvIhgAoGxgUhC661Y2fwBUbgzY1BQLJz4s9f37p\\\\\\\/QmnKDePJMdsVoa9duck4ed\\\\\\\/Ie8yst6AbhWJZXK+xsMISPWyPfPr77nn2PZqR45HXOy5Chh+YebeGuTYg7btZt2LpO87bP7ccsf6YhYsNAtXSY4BX5woz\\\\\\\/dHOgCUp0F3Qy\\\\\\\/ByrQtjOuLlha3aum5eDgKb1ekSVGy\\\\\\\/+Y01OtUY\\\\\\\/ivoHCHnaCjP9QNzzf\\\\\\\/BQRMPBB25FpyFM\\\\\\\/rTGDNjtKShfuplliW+s968M2ychRP8JU4d2JMdkttDK16F9uCb\\\\\\\/jwDfCFfyUfvrZo1qimItgAq+TJhbtA1oQCv7hWR4+kh0chAagpQgPRYvcuqxiTSJJZiSRSlnGPKb61wh0jiO1+0X4rJDGNxoOicwtlSR1\\\\\\\/4B+Gl4DTBmsQN6R1heK70\\\\\\\/iQXp5RteYpL1OobZ03LYrtLk7pRcNgNenVyNNJjaqMgY0\\\\\\\/ZABKKcvW2Y3RX3TOP\\\\\\\/LF06w7rucYeXe5PjBVEAJAf2dF0ujwkkr8f0RcT78dMTpdXm1LMJXFhnX9TOTJmTkCL\\\\\\\/sJ8I3ZSCJPiLX5l6z1KRLNSl7gfUtEqvYqmZsD6Ssy+vjY9t+JcNKZS9e6PAqm0SaA8Yr8VML8QVuUPXjGmN3xQFkpufkQY0kVh0pkrq+aVRvaMeXfBEpQ3HzKJSLOo9E2vd3IEdft8WhoIopv7e\\\\\\\/hx3yT4TiBqmzwegmUdYcjwPBWqZ4XaJWStXks6tXgQZop8\\\\\\\/dv\\\\\\\/Zc1jEwGCoLHY\\\\\\\/4IfgR6G7oHNGIJ1bV+XLeBkHtjTsN4O8hsi3JISjihVvBPaDal+awoIxKR6LalMPOelc8vhNUEsIhm3plUR+kZ6vUEcx1sgUGLkoMlm4a59HWE33DeBwACmH\\\\\\\/is77aPgWAXGN8fhNkVg9BS+9y7ESKb5\\\\\\\/CV2thZqSi\\\\\\\/yCqeW+CIPvIo6hbNyj02fteu9CXhM+uwJHH+LoGUTv9QBRoQ2QrljC5ZzZ9Hj4QMQ1hnqOZqdqs9ft3lRFbMA7eQwbPuYlDEnZqnrm\\\\\\\/Cfllydd1X8fluT9Avir\\\\\\\/5UpDZl9J3qpWUFtD9xutBquzprRISQZpfgRoolz5SdyMr65A6smyNKzAFpHjhKFWZYyyDpMAY9fdGQ5pPB1+lOV6Aj\\\\\\\/ozhqEa+Dqt68rkz\\\\\\\/YhVr5RTjMHDdsTVXB7s85AX+DGxQ9Ip83PVZYVPQyE6qdDk64e1OqOaOEIBOiwzwEzmRNlBsixgx2CWWtl87Iusptme6gxDRxbS0xHniFSrDC6Nj2QvPhudXqX4wfIIyEhCDTWU33FFbhJ2mJdNPDK0vyPDsVBPD5bsWqpWEDAvA2QTGXa3pEYt+Zrq\\\\\\\/0RNNZcvuX1pN3Xpktjfz5mfYV4f3Q7PBHZoLvwId2A2ibOAm0ysZDT6atDRP5NR1I\\\\\\\/a0HeRmPOmrsokuiBodu69Y2zqRS\\\\\\\/XD285UWhqHL2ivxO\\\\\\\/8TCeDBDdIElpEdQfHm2VT5e907hJO1PpjMoutmwwX\\\\\\\/Hg4ICEM4XWiVphCX0UueeApJIn3Z6NGvLh7NiBVsfd1oxI\\\\\\\/6k8ssOYInrJJ52ltm8xYOIZJFcXoLW3V1UMs18HlrtSCINodU3Jnaol7ynbU\\\\\\\/W17VMiDKT+Jr75uO9tCN7Xn3yrccSJdR5\\\\\\\/k4Qo+DUxZLK6aROc\\\\\\\/RYYXETz\\\\\\\/BDAb8HjRXbDEImfba70NOY5ivjiMO7AlPEheoDlAbu0yeiv4\\\\\\\/ctggGu6hoL2RElC07buALh77oXn\\\\\\\/VIGm7X7FaiOC0CBoTDJu1z0C9DQiUc+w7DGu207fRQD7ICz0MUIPUBkmqnv5vOyJjEVLnWXSY6oNP4ebid\\\\\\\/puCGxUWiA+YlkliUnzvBGhqodR5L3lHmh6GpO0pH3DHYe95H8Xx4FJJLdlXIsGYuWai3hCcnUyMMKqBbVdafuTb5YurkJKRiTQwlKQioqBUBtQmtOnFKV2gaU5upgSlg6F2PKuXh\\\\\\\/Q7Ecs5eFQjPmFrS3Bw8QtDdSPkUXdOgTvAGrecNZpMzNTbKmpiiLfFIhoeSo\\\\\\\/3kxfTMvTpJKAl0rEqYfLT8txdeb9AHHBIEGL7bWxX3JdyKnR\\\\\\\/j3su22PXOPv93KS+7Q8p+ZhFPF\\\\\\\/XzOKaFrPSs9Ur0\\\\\\\/5mWQJMJCDiCfghsae+dYYKXdA1b7hzGvK6k++QVYoEJcZUsMDUs1AnNeX2X6WZngy+5Yr6OlPC3lIIcKrZOoN04uiOo8UYB3MjU4DMvsVdeRcpjpTh8DXPfR3Gen9lfZp1LgpZwQs1uxetrBnpkgOIMx5JkP8qt2e8EDAQ3vcsgcb8eknQ1DUcYNRWoIaviUtzL7\\\\\\\/g5SssEeeKj7q010uz83R4TdBbtN2WNEcPSiTntY80XgXjFrW7HysbWHz9+P5xZu6f0J8nn5tsFkWVmAYg7Hsw7V3NLNNYiEL\\\\\\\/HQMehNnA6cIyEnCURvMoa4Bihlv+I1UcXJcuRuVRN3h5gyeSe5YM8ZVR\\\\\\\/KwsV0fI2OLx6HHxd1Lhv1a4KnOmiLUHPGiVrXZkXK5OFYY074c6nglM\\\\\\\/HcXfo\\\\\\\/kGs4dad3jIsrtLklf\\\\\\\/pXgRox9Yh5Oil8m8Sw3Ue4Euf6kuY\\\\\\\/UCh4\\\\\\\/HpGR8Ve5Nkuo81oEOjWe1morD7UMVMBSbLr8aA1sRhVRc1K7Hm\\\\\\\/IIyIMf8a53X\\\\\\\/v7Eh1qJ3HogqciUs67Sjm87fH8CPhLu47pNnRebxaoF357wcWsGXLuN9ay6cwNznDyhGVCW++5C9Fecvd0oOf6At9ZuOQFX2Jzvlo5NyknHzcoyATLG2ggTuA01jh2e9govSCunk79g5\\\\\\\/7DU4mffB5QeBx4qyBIPXwl1pnVEDliZl4lvZDeVWIqrDjMrkGT3qjJxtPLCnud\\\\\\\/jXvfOT4aMy8gtqktg8\\\\\\\/bAMNiTq4ZxRfAwj3PwZP5VGlE\\\\\\\/UsdpjOEjvlXq75TeiubOF5WxKLrxYrom2HT1cgJBawh79Vbe8piQmw5vJBqNdp5lWQD64DKJleQoXf0u+Lr46KxRnEv3VFUymYKFcVGJCdXNbTVpyg68zbT4tbPBEcJvvQil19G7S\\\\\\\/jFqXwWg+EY30m+\\\\\\\/HX8BGZIWtJ9LeN0wRfDfWO6vlk8ACSyvd8LKp0+r98c6BuCg2KLHYG95RA6QSLW9\\\\\\\/9\\\\\\\/pQOMRpJBLLEIVhZxaYyqZuz+uyQOIOUJhPTLF350qzLpcUJDOw1PFNygXh5Z7cQzXo4jD8xfI0BfPikWEtDVUJRSfhE00eXh\\\\\\\/+dT47ph3dj6WB2BPmqiygXLqsXb\\\\\\\/a55nlMPUJlUgdS9y8MuXjw1t+tuvNdWxIkV91uz++Aa05jMBBrcK6QfeYndnPRirDX2M\\\\\\\/l+0ffHqWhkelpe\\\\\\\/JAThB9uNWblg\\\\\\\/G67SmESXE2oHVpEWHhuQzpMsQvIshU3E\\\\\\\/W2ENeRSLQdpCT3JgZ6BTh\\\\\\\/Ao+\\\\\\\/8ZiKTPgDs2GFFRTuZ4OwYBAr+PfRemNQCl\\\\\\\/PxEzwkId6tJX6ip3DYSpfB9RC2IzX\\\\\\\/1B7peR4wCSyMTZYMKtRhm3r39aZyVXDwf9XvJR6Vc6LQX+++hPbaHZMiqhjb6ldzoRySPTAf5zm4OzDs1byctWJS5ntCY+0EGEvhvXO7Tg9RuF0sYdQUrZkTgR882dsd7KdUMmY0Suur8hdIyZWCXwTzM\\\\\\\/hDyc+il4ewSBGuByTBKC62u03YiPmPYS1hxpb8Y79mQLCXakH8wULY\\\\\\\/TeSefh7CnZ\\\\\\\/76YoNQQ68bsyjkAW2xAkrV9FvWN8K8oWDI75Mwh5a9tRQ23UUK4ecXLiW9LN+u90eL9uPSQngEkGLPbYblm0K7HfyoFdimTPv4111v0J3smv7ihoViB7qmwq7T65bCGko2qnRbz2uacEMP55f+ehihdCVlvaiFsKJnTp8C9q\\\\\\\/BGpLXd86QtABuWBnn8gaBP+EUne3A5zF1ssiLXkgNdoLkO6CIS+xBLmqUwy3n8qojK4wBWpZLpU+woV5\\\\\\\/aB6KxdHYf5N+AnobElK5R+Hu7vvH4Ze0aJkF8IKYicBrQJaSjz\\\\\\\/nRi9PoO34yaNMNlWqyLBSdHY5EErjsWKshSklBrNB0YoApbroY2uCaqC4VIvhMRpzeYgUfjLpGBcAvp1od6LA4hekeHzk2Qwl0uZNxD0btyV2ilzZZ1IG\\\\\\\/YJjy\\\\\\\/erFs4tKjVeP04Kiidg8iPr6GnLtXBOpkt3jx5XSl2btxRfWvN6PK5IJFeWoslgFXQrjLbjma3C5y9su4LN6APUnUxqChY0zSZkEEXIqFHV2OXZQWfcOsSuCdvmNuaA3gIQWiywy9d69RKPSJ4L\\\\\\\/UCOXxUXz\\\\\\\/KGKs6MqGH+xqLSYvjE0A4UF2WqH9hy\\\\\\\/iWBSn+QXP1h7rZ8V1zoPUOoORpQN0AMyIkiuHy\\\\\\\/wOkqWlrkupSAX0N7Iur\\\\\\\/I\\\\\\\/NQLcBDrri0fG8Hz77rcItqMOxwFFHTEfshAwjN9KerVd6sO+i4+SJHkfYRLvyEo+sM775HZVDJTh1317dxRt0uf9WXk9UF2W0k20maq+WhoHhNX6HMvofSLpPXfN6HH1ZslitTGlV+Oz6ACxlDY\\\\\\\/sRuuU95W0lIf63aiZHBtwocnKNCnIAw+9x1FvkZ49JQNairNg32f4QRaj41guJnu2u9suvDuwHLd1mkh\\\\\\\/\\\\\\\/dJLrtjltrJa4J7t+ewWMjGaWuP9Y02Qj4nBvfs0YZJ6huuP7MBsBNAZUl1vvuzcqiLFipYizSn9lzkj1hoeIoJuH710dCuo\\\\\\\/h0exeNFzfWH4Lc81qtPYZTAXj\\\\\\\/RQx4kfxLefyUh2FW1dcy11wGd64jEPcyJCD\\\\\\\/dSiFzfdubK80LMFMBVOmUHKVfuG+zpZ7NZjl5JnAhkfpEz3qj3Nqy9GQDPb9ZrJYEwVJVxulk8am3KmcSnTmPZlzrlGWnCDof1hjCrmnwe3nHqb6\\\\\\\/zvMn41O6M9crS1ENTQVJ2ArAoAXQwDU7QSrRhPRVQG8y8ccwsmhEK75hmOlo7Y4fSi3gWx7HDRfpqQx1YkF4Dc8dXC5e2kl6OVZsmGLkA5no4ay1QR90tHkVyNCvqUYeaMarfuZGYk7OqQnMpATbE4BbeblECz6\\\\\\\/TlEbvEbrQEYkh+HTPGZ3mehwOaSKynXHJhD\\\\\\\/8axOxi1yi56Id3qSc42XLW+dM0Vun3Ih2yyIk+yZ\\\\\\\/TQSIckKVUm67AO+p5l+CJA4CIXKLADWqqvmFkVwist7M3YH\\\\\\\/QKaeEiOPN8h9KxYGdJLkSdeiLq3NSq5yUoyUlfGrJx3peXHbvqJHjwPMaavkXpNxUuzWvyXI5+txDzajnV21Vlz7ZxpXaupqhSVAx80oTC6ztxQ62vPvLQhuakTEluhkNPmKeFNJW8wB8GiXQeescMajiZmzbE3nmSroR86DIcL5mC8mrrEzibnWE5RhCMPvZnYmyxvcuWWQpsF1kCxEGUbl+kJjyPfr3coIUPb0rWu6jnPLUniW0WAFfwoJjm7hmywI6S81mhvKqe+Jn9OMrp\\\\\\\/3hIZJ3Zt2araievnaB+2wwkQQEVIbqRuvsqwjJSRsKKLxaB7EkVlUcBPJimTj2nlCKKR2TGZcsW+loD2ko8f9H0xweIGGXG3OOfEeeRnCH1EIj2Vl+4\\\\\\\/8vvDefeh0RUOFSubvCBCPZdUuexzelsu1xGpazyL859qUk9G1NC+etoFGSNlQ8w1EnkQ\\\\\\\/ZndhkReIhoGBQ0+TdvX5YQIMf+LZF4Oc4uSpT63jyw0y8F2PMqemYZbmy4hj1AmNYcmaRPOrlIBb2SH8jSm0HWKg\\\\\\\/Q8gj6SaSQE7gCUGxqCXRvXXQ1WkE3zq3qWUlZH8\\\\\\\/ZWknMsDH1zDOZHI\\\\\\\/heJk6gFMIVle9Kv6Wsa3UfH+4Xn+i4\\\\\\\/1aUAnKVNrhmZibkd7k7Br83NEiNiDPa0078rnHlQBnYz0tActcRNOrCkTa45ShsOE\\\\\\\/\\\\\\\/Zoq0sEeHcdMUfd1AFYcj80cqKKZKUL9IocEid6NUGJAUewBGI4PhbfbGGcHSffHXTCxcxk\\\\\\\/c7QrrErX6ho+FStVIq2HEVCyTWfWYlCybsBIV5ycuojq7FSLEKUk1+2a+ZnQt6gE3RKhPUEndEiXnwc60txUJsEXy92B+M51FAK1xuosCSqSYwSIGR7k7pJh2fHSb52I4+UQIG+e6fUCnuwGwf530++IWmy+mzPcGgpiu\\\\\\\/nbSkIA6qhBtoexgjTmOuaOvT\\\\\\\/QbYJ13zhj43N3e19mCmGmeDT5voA2k\\\\\\\/lFHB7vrKKUVxh65FUVIRNx88tMEWl+TJAQ7LWEFcspY0PFIW12MU72uVgyey9o3F8wD3a53MYS0hGCKf7lzLtB67PngZvTuAKs6R\\\\\\\/7DazwKPktNPTad\\\\\\\/jGHCQDAe+zOQp\\\\\\\/tqWh5X4kyQgjJ12fmyuRVwVRnUOnzCRhMtI0tPYBFXE3AV\\\\\\\/CwWITvr0aFRjBVE1KMLe796OsFpec5lhOhKRgcoUGQsPMwFUo0l5c9xtlRFxDmqCXBu1xeQVBrBWUBZT8jQ97FikRAbx2rmH47k+yf\\\\\\\/20jYry9kvglgoZUjHE5IOc+M\\\\\\\/Mrr7flNi1a5zt4Kabv9ashj1yDOsBseSxKBsPcWVfBvFPzxNlz0slWA18VtU7j0NEfC0+mkj\\\\\\\/Xl49h87KOk0k1V5OHVnOOdKLo17HJXhRlF8Bs0ua74rAM\\\\\\\/Rg53vOOkRzzs+p9Q9du294wtV0LTdmOQoYw7FUl+NvuTJnJGnS6E8d7Elcfvtbs6jMn2sLBpqkDql5c5Ip1B+n7o1rKOEf8yxXAXmXvbnTW6tdenlUtF4cdquo\\\\\\\/GIV+YxRM6cF7ZlAm8ORrhodnpt5\\\\\\\/mq84PDDsPS1oSSspWyVJ1S9EpATX0bO2WcJaI4CwDqSQXGX\\\\\\\/a5qvpw2McKY1gDCJl3EgSEeL\\\\\\\/MkP3MSJmyzVoEFABuvMgcAtFnBg7yN+t0j9w0Dra8lC7K8VnEKcJlWOQMs\\\\\\\/geGo2MX1wsFGvB7Kx9oD1jkYwJuN6zZ++ftr86GxIExeQRLmawKerR1KuKzbHoKTQcDxeczD9oautAXEIdKm\\\\\\\/OUbj2RSvcRcUCoXlX74tF5rKCarzYuf3\\\\\\\/PqAszicn4\\\\\\\/ncI2Cti746bieC5xhW7R9KZPNAaIf+xy8MpDpmggNuIWVf3WnBDLs4zp65O0i6mYXBlGjDJ\\\\\\\/UcCKmhVfxIdOPyHpLkM201m+2XG5l+OQJnWpE8suIwzN2PFy3vj7GN1fFeStjfv8rUlnEiANu6NOK+oj7QXbcYb5HJFq7EDIpBDa\\\\\\\/z79KMZfnbqNxvH2CtNfjsCZ17KZOHM9tf2CpwA2DzKHScPfcBF9E2bmNmC\\\\\\\/Wy3f1bSf8s69H6J9p9enzOIeIWTn31uEAhRkeMaPAPmw588Zvr8aTE441WicQO4\\\\\\\/lJO3N08Gx63FJ+AkojB4EO5p18LrAKAaup72XAmiPUoUWZxIV45kb4WwNyOqNy9LUgHqYxv2DYeQqSAnt10pIC3NygifvBCNkD8iq3EWB0Rca636KB8q7QVBmOBCH9kz7zsENGz9PY8um87WMImMyJRLjfZ4QmqFEoA8EeXUryfcjlceM0ZCo1hnnEiTkvGr6QBkcAMY8LypUeqiIawPHdqByuCG6dPl+jliAuXUoQq1y1nNqPP4cVc6r9sdSLnaEBY20K3uXuOghSddD902aa1wmoLVek\\\\\\\/YAhRN69mgYQTnhEzZbWbug4H+HgNLDdP+fBx44EogNbWaDVhD7OK4HfGwFVnYZDXmQorWhilqstUclDhfoMjElVHPtsXYRRgyRpbuf8ftTbz4ifC1og7fPyVeQSEsEwaeBxw+QGcHvJSOgXU+zdAKmd5FhMOYj\\\\\\\/1Thoyncge8oaaupATLOvSwzWOocXPJKygZ9AzRGzej61fsmg5siDidJEQHcNWb7p8NrOSVHgjCjtmmDg9xgJNGzXYZ4QYZradMBvCoYfuq46etzpu2ccpzOi+MVYQEJzzE7RzqlW\\\\\\\/MJZLSvSBLTfvCgZntn2HEgV39L4lH1r7PUh2Jee6fT6ywUwb4KxQ2lCcHg+U4t+5WXTA2PEKw4pqGm2yeWFErezpUtMixRMR\\\\\\\/N4MLkc8DcL2edjlajAeqzjtrB8W6EgxghJOH4jcjNqpBukRyBUzdQywgkZWtzIXLq3GfQ8iRWRZa7HZ2fhwJ6fzVUjOCJTt4u8tR1DIK1fvqAaDB0naflgni+ndZiSkcFR9Z2qM9D31DH5iOHq5pzZF+CJFE1NTTlOAUUhkU4XJJP2OyFkrmX+rhbX5h4RAIfeTY3ERfNTwkAK3ER1tIS6jIBlrvSz3nWWXaBlSUYLk9ZTBh0C4brrQh9yqwxnX7a7eGfd+Gxp21pGiGH6PJ3NaEQbkxVXZZLiGt3J2KK1vJN8\\\\\\\/+VnsRJvbuTw1il\\\\\\\/WJZicEmK++BCbQyKf02lf1eOSYEbmh9lNTdmfLfv1I0L3XMHVEvOGNWTrLWhdg\\\\\\\/afzUrh7bLFYR7ICxHEMPys06U54Zdf44GMoh6shMRnRL1+jLQ2lVlHb5TyaPrFbFjDppwOnthrswLOXLH8G3wNO+38owz6KVh2SaMcw8dEyI4HPFYnx09mq3OZFpEU+0inmp0oG87VNnXj+XM2lar7HRM\\\\\\\/v3rBg7NorHJ4tLJedtyDibNObcqjMNxAfxAs4XymGiUBa6kjUWQTOmMxey+b0dfk\\\\\\\/1q3Ua3Jo0CyE6EZvlUlc\\\\\\\/O9DMHoULhLygOoC8Dbtnb0atv9ml6STGIIFykW5SRt7ABiOyujmJUINnk7UGJvQBoOeFbEyhZ42XDgy50ZkicuKLG6ZHTWyP7bCDHiGiAL8t6piK44O9cxUQIiEfPWkrJQ0EDYq0Lgt3iYuHbWQjxAt\\\\\\\/L9HvRyIvvphWjseGeuKWf2111bmBixqMBFnC\\\\\\\/bpMrenn\\\\\\\/U+836aZTWz9pUdAjYqwVF02\\\\\\\/LLHzLbXeq1pDMdhmJd\\\\\\\/NE25uGn6yMhg7g4NtCx4yCdN19+n1grLwzf73YjLjom9WxVq53yU0TCyI2x+3LIb100b8Jpg1N0kGyUcIosiKoB88Zw8XMuOl+Yx78kFqAWGiDumdYwFXGLacta\\\\\\\/vjbyaDjpdcDxaHnMDG3mKpZXxnWbXMgARa3qf+vqmXTX77RKh65SDomPbOXmEzhmnIr4BjfqMDgDnTE4VTFBQOAh7mNUhVBb07Nt9VFL3zdINJ3LJ8\\\\\\\/ijeSemZsh0x\\\\\\\/M5zmhjqcNVuvTKlxvN4XwQ2UWS5qVJVtecDJdSNaro5kRjEtueQf3rd90Pecj2EARXcrcOEr4HMKevojgYZgUzfzTY+S875FrFn9Ai6CZnesMuSE7A34OXuxW5pnJQr0Gy107VYikjsLOZQuI5erk4XBf7LlRK4OPIfbSGI6qn+Zwv5YijQ4V\\\\\\\/+0wHQGbLT455H2zn7LctTDLKqnhH3od1ZZoN+bw18Sqw8Sh\\\\\\\/UzNgzdQEUt18VtjIKsoW7XN69T+mwvgy2AA5uAC2luQC8QxlcAWyGMEzCv9kN5CVWBBtBVGTYoC\\\\\\\/ymGhJY0iojRCBeVZUTcSoPaiE5G3GOU2wCSQBbKDJ2dtwwd0uWuHdJUvSo\\\\\\\/oyTWg0MSQ6iQ7xWUoarg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022+iCHVKOZWqEUoORassODwA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222f03ab0c0ed02aad725753389b23ed2ed5fa762d38d8364b9ac349a5b7c7a645\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022sk8nT\\\\\\\/fYZQ23rQM+wRAnxInwRB0osR9OsBpMUuySbBs+midF36pp0QoyaL\\\\\\\/tYnWwU2NMznfTFLd9mKMmnOfY9q4D+GpHZEjznuh9xu9Ab5eIZVXwHjsaAqFIgompVaPvRBOZ8+1oEk2\\\\\\\/Tts3dISXEAQSh7gqTWrigGX56RP4j29rk+SnwL54zP+H9ylr3IOpeyoNJAP4Wk4JfgmYg1SGDI9gMsw+\\\\\\\/pYoNrOfWFPldcb30NSoJTxkUDNHRGDVg7qKv14fNmgir5vnHiocXBWf7iB29Mn+ip\\\\\\\/vVrs9t8RPfXsZc2t3NWHhdT\\\\\\\/TPO4dvRMaoqYzY9t43lHkFGjjXJzhBAqH5itXQPycrsjdNLsjy83u3AdaII\\\\\\\/sAuwHY\\\\\\\/zyUQTy1ftX6IATU+Nmzgt5Fp\\\\\\\/t4NeBZ3Nl20yZCgcZSCRLICtPb+M6Gf+Hd9zAzHph19t4fi+RqIyTQgsUM\\\\\\\/MxcuGxSQitxtorUQZsV3QmVmALj4UYcnBr8wzvVPi+sh8uO7Vvse49fwJNKD5\\\\\\\/nkK8Uj7ASM023L155DhQrUOMZuZjw5u7f\\\\\\\/VTZfNM28nu2mhghsEavmnQ4N1KdhDW6vjl8G\\\\\\\/UaMctsMtqfh1Mql4PcUVBSv5PRtMlowBAo39NBn50WQpZlixP1SJ3LP9RR8cKYn8zErr8illHvzfHIwWBxgz9JNLysbV2+xmudw1A93SOFO6uDxEzrKnV9nj0OYY9S5aSsbou88OLo1bERJ0NblQcMHGWtkMeHIQxufBmbx++MeFArtMA5qNTIDa3AxLzwb6zGw4eUvqlhUfe4D2aIFYvT\\\\\\\/dxfy9h3+dj9NyIWxzC4loNi4dhVnqFjT+g8sBobGV6Sif+vB1jpuekWzkDM3VJyzj56ukNxEioNPTYzFVGH9ZfnNVNDaamn+hUwkX1X2Rr3gsAcOOUqV2wnArD+iOnWKpiH\\\\\\\/fmCKd8Fdz7y+h8kyXNGb\\\\\\\/b\\\\\\\/m6KfSuMMGE7yggOAFAediUu\\\\\\\/TTZDC\\\\\\\/\\\\\\\/Sr6G46OLkVrVzFM\\\\\\\/s7KZnmL2l0+YyZrcp8YYcS+TceIxc0XETGNfBFzVJen4eUjW51IMLArvJtNhgHwztrnJC65ycnRO8sub4a37HL0HUtTE6cv7vQ5p8QGhwy5SswcryInZTb1rAwB\\\\\\\/gUwpJ9aCv0U6JL+NsS1+vpNW9oXotGRcZvyPyY+0dpNomysa9nMyzz\\\\\\\/GSDaSD25I1Q2VJ16zKQwAqeFHgHSXzZDyhu+zVTHWugY+acmDfsEJZxniC0UoIo5Py0iioCqCzQ7pMXVejWBfdHHVA9LUB0WJXBmFOJL5AIB3fANWq8APs0Gkocw5XDGypXApZcpx87KVVeOHn25QSyOauJUXOk1aIxm7+MAuBcAgIt+2d2q4WN4VU6K4Ra9kn5yVb3HUgSvithVwMTmuDSkmimomq06KCO1jZa++h2o9jwHZZk63tooHaIjxE81kSQ5VHWb1dGTbzabKxjAMk6RQODisdBOJCRXKJc2Hjddlj5k8AROkwo+I1rGa7+jNI+WvhRAUZo\\\\\\\/CFCf57BPxnux0FTtU6vLdfb77cm136jnAtDeJh08MblKl7oXsnIWqoDOSzrZrvaIzgHwTJWn0grloySB1cYH0lbQ4DXYTo57gxrR1PUjuf0iqyL4K\\\\\\\/vnDLIsW9hsO2es4\\\\\\\/Y8CyxoKtJ11AtXFKEw8RuKh\\\\\\\/xHVtLRmcJ30b67o1uT1DGAU8AI4DubJ+lLdPdnGHPdRn2lWdYOj2Vvmnhf0xnt44vcKQGIIyZduFzq\\\\\\\/SycC2i\\\\\\\/vAgmi2Rww3gLsVb\\\\\\\/H\\\\\\\/ZvqImVsT+HRubmbQWoMSsZ2pCwz4DGvLXVvsdd465H25qWheTz5wrthXBRLuk8rhVGn2fwMsv5EG5t5B99I7V0kf2PBYlaYu5cFE0XsFFgT8WPDwEXa30p8bMTz+Ki165QquldpTpbOPXG8ddaz6d9GD3I5DGrU8oGFVTgbn1snZifdTV+HTfS\\\\\\\/QwNl808R6zVN1AypYmWaujZQ39fgTNnuXLAjHFETVQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ht9Yp2zuNrWDLoSC7BU4KQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b04bb63c88ce0b09cc2f85964a40b1c697c44c61bed6a47404780936a2e91d6a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459794 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/3/407" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_rWxMys4E1dr32qAjAPROEOGPzUWpR8wXal5VGhE0R4" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/3/407" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-B3d3nspHFVU9p0tUSJMvdvUHFMlGQx39o4CvfIx0XFY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/3/399" title="Genetic Markers for Western Corn Rootworm Resistance to Bt Toxin" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/3/417" title="A Genome-Wide Association Study for Nutritional Indices in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459794" id="node459794" data-pisa="ggg;5/3/407" data-pisa-master="ggg;g3.114.015834" data-apath="/ggg/5/3/407.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison of T7E1 and Surveyor Mismatch Cleavage Assays to Detect Mutations Triggered by Engineered Nucleases</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Léna Vouillot</span>, <span class="highwire-citation-author" data-delta="1">Aurore Thélie</span> and <span class="highwire-citation-author" data-delta="2">Nicolas Pollet</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">407-415;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015834 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459794" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Léna Vouillot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Systems and Synthetic Biology, CNRS, Université d’Evry Val d’Essonne, Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L%C3%A9na%2BVouillot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vouillot%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AL%25C3%25A9na%2BVouillot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aurore Thélie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Systems and Synthetic Biology, CNRS, Université d’Evry Val d’Essonne, Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aurore%2BTh%C3%A9lie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Th%C3%A9lie%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAurore%2BTh%25C3%25A9lie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Nicolas Pollet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Systems and Synthetic Biology, CNRS, Université d’Evry Val d’Essonne, Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BPollet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pollet%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicolas%2BPollet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Nicolas.Pollet@issb.genopole.fr" class="" data-icon-position="" data-hide-link-title="0">Nicolas.Pollet@issb.genopole.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/3/407" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459794" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459794/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/3/407.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459794" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459794/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/3/407.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459794" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459794/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/3/407.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459794" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459794/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/3/407.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459794" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459794/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genome editing using engineered nucleases is used for targeted mutagenesis. But because genome editing does not target all loci with similar efficiencies, the mutation hit-rate at a given locus needs to be evaluated. The analysis of mutants obtained using engineered nucleases requires specific methods for mutation detection, and the enzyme mismatch cleavage method is used commonly for this purpose. This method uses enzymes that cleave heteroduplex DNA at mismatches and extrahelical loops formed by single or multiple nucleotides. Bacteriophage resolvases and single-stranded nucleases are used commonly in the assay but have not been compared side-by-side on mutations obtained by engineered nucleases. We present the first comparison of the sensitivity of T7E1 and Surveyor EMC assays on deletions and point mutations obtained by zinc finger nuclease targeting in frog embryos. We report the mutation detection limits and efficiencies of T7E1 and Surveyor. In addition, we find that T7E1 outperforms the Surveyor nuclease in terms of sensitivity with deletion substrates, whereas Surveyor is better for detecting single nucleotide changes. We conclude that T7E1 is the preferred enzyme to scan mutations triggered by engineered nucleases.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/cel-nucleases" class="hw-term hw-article-keyword hw-article-keyword-cel-nucleases" rel="nofollow">CEL nucleases</a></li><li class="kwd"><a href="/keyword/enzyme-mismatch-cleavage" class="hw-term hw-article-keyword hw-article-keyword-enzyme-mismatch-cleavage" rel="nofollow">enzyme mismatch cleavage</a></li><li class="kwd"><a href="/keyword/mutation-screening" class="hw-term hw-article-keyword hw-article-keyword-mutation-screening" rel="nofollow">mutation screening</a></li><li class="kwd"><a href="/keyword/t7-endonuclease-i" class="hw-term hw-article-keyword hw-article-keyword-t7-endonuclease-i" rel="nofollow">T7 endonuclease I</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/xenopus" class="hw-term hw-article-keyword hw-article-keyword-xenopus" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Xenopus</em></a></li></ul><p id="p-3">Genome editing using engineered nucleases such as zinc finger nucleases (ZFNs), transcription activator-like effector nucleases, and clustered regularly interspersed palindromic repeats (CRISPR)/CRISPR-associated protein 9 riboendonucleases is of considerable importance to contemporary molecular genetics. One prominent application concerns targeted mutagenesis. Once delivered to the cells, engineered nucleases can induce the cleavage of their target DNA sequence. This cleavage triggers cellular mechanisms of DNA repair that eventually will lead to mutations (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gaj <em>et al.</em> 2013</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Terns and Terns 2014</a>). These mutations are mostly deletions, but other mutations can be found, including insertions and insertions combined with deletions (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kim <em>et al.</em> 2013</a>). Mutation hit-rate at a given locus needs to be evaluated because genome editing cannot target all loci with similar efficiencies. Moreover, in whole multicellular organisms, the delivery of engineered nucleases in one-cell stage embryos generally will lead to mosaic individuals. The genotype of such mosaic individuals can contain several mutant alleles with different frequencies and each mutation can be homo- or heterozygous at the cellular level. Thus, the subsequent analysis of mutant organisms obtained using engineered nucleases requires powerful techniques for large-scale mutation detection. Such techniques should allow the detection of one mutated allele in a background of wild-type (WT) alleles. Moreover, they should be reproducible, reliable, inexpensive, and should deliver clear results for all kinds of unknown mutations. These techniques are needed at the screening stage, upstream of the sequencing process that will ultimately identify the mutant DNA sequence. Yet, most mutation detection methods are not applicable to this field because they do not generally achieve high selectivity, they need specific sequence context, they are laborious, they need specialized instrumentation, or they are not amenable to routine analysis (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Yeung <em>et al.</em> 2005</a>).</p><p id="p-4">Today, we can quantify mutation hit-rate after the delivery of engineered nucleases by sequencing a representative panel of cloned polymerase chain reaction (PCR) products obtained from different individuals. The sequencing is done either by the traditional Sanger method or by next-generation sequencing, both of which being rather expensive approaches requiring several days to complete. High-resolution melting curve analysis recently has been proposed to identify mutations, but this method requires the development of a specialized quantitative PCR assay on small PCR products (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dahlem <em>et al.</em> 2012</a>).</p><p id="p-5">Another strategy to identify unknown mutations relies on the identification of heteroduplex DNA formed after melting and hybridizing mutant and WT alleles. The identification of heteroduplex DNA can be done with chemicals, enzymes, or proteins that bind mismatches (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dodgson and Wells 1977</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bhattacharyya and Lilley 1989</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wagner <em>et al.</em> 1995</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Youil <em>et al.</em> 1995</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Howard <em>et al.</em> 1999</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Taylor and Deeble 1999</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Yeung <em>et al.</em> 2005</a>). The enzyme mismatch cleavage (EMC) method takes advantages of enzymes able to cleave heteroduplex DNA at mismatches formed by single or multiple nucleotides. It is known that the cleavage activity of such enzymes is greater on mismatched than on Watson-Crick base pairs, but homoduplex DNA can be cleaved to a certain extent (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Babon <em>et al.</em> 1995</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Youil <em>et al.</em> 1995</a>).</p><p id="p-6">The first enzymes used for EMC were bacteriophage resolvases such as T4E7 and T7E1 (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Youil <em>et al.</em> 1995</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mashal <em>et al.</em> 1995</a>). The structure, function, and substrate specificity of T7E1 has been well studied (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hadden <em>et al.</em> 2007</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Freeman <em>et al.</em> 2013</a>). The substrates of T7E1 are distorted dsDNA undergoing conformational changes (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Déclais and Lilley 2008</a>). Thus, T7E1 can recognize and cleave different dsDNA molecules if their structure is kinked and able to bend further (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Mashal <em>et al.</em> 1995</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Déclais <em>et al.</em> 2006</a>). Typically, heteroduplex dsDNA containing bulges formed by extra-helical loops and even single base mismatches can adopt such polymorphic structures (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gohlke <em>et al.</em> 1994</a>). Conversely, perfectly paired homoduplex dsDNA will not constitute good substrates for T7E1. The ability of T7E1 to discriminate between homoduplex and heteroduplex dsDNA has formed the basis of a mutation detection assay that worked with moderate success (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Mashal <em>et al.</em> 1995</a>). Yet the sequence and the number of mismatched nucleotides and the flanking sequence between the two DNA strands affect the heteroduplex structure and will therefore affect the cleavage efficiency by T7E1. This is why deletions are cleaved more efficiently than single base mutations as noted by <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Mashal <em>et al.</em> (1995)</a>.</p><p id="p-7">Plant single-strand specific endonucleases of the S1 nuclease family such as CEL and ENDO have been used more recently for mutation detection (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Oleykowski <em>et al.</em> 1998</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Yang <em>et al.</em> 2000</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Qiu <em>et al.</em> 2004</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Triques <em>et al.</em> 2008</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">2007</a>). CELI, CELII (commercialized under the brand Surveyor), and ENDO1 are single-stranded nucleases active on DNA or RNA. These nucleases do not form dimers and they cleave DNA only one strand at a time on the 3′ side of a mismatch (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Oleykowski <em>et al.</em> 1998</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Yang <em>et al.</em> 2000</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Qiu <em>et al.</em> 2004</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Yeung <em>et al.</em> 2005</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Voskarides and Deltas 2009</a>). They have been described as useful for EMC assays but they contain also 5′ exonuclease activity (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Qiu <em>et al.</em> 2004</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Till <em>et al.</em> 2004</a>). The specificity of CEL nucleases to identify single-base mismatches has been extensively and empirically proven by its use in the reverse genetics strategy of TILLING (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bentley <em>et al.</em> 2000</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Colbert <em>et al.</em> 2001</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Coghill <em>et al.</em> 2002</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Perry <em>et al.</em> 2003</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wienholds <em>et al.</em> 2003</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Comai <em>et al.</em> 2004</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Till <em>et al.</em> 2004</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Slade <em>et al.</em> 2005</a>). Yet there is limited evidence of their activity on extrahelical loops. The initial reports have shown an ability to detect extrahelical loops of up to 11 or 12 nucleotides (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Oleykowski <em>et al.</em> 1998</a>; <a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Qiu <em>et al.</em> 2004</a>). Despite the preference of CEL and ENDO in cleaving single nucleotide mismatches, the Surveyor-based EMC assay is used commonly to scan mutations induced by engineered nucleases (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beumer <em>et al.</em> 2008</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Geurts <em>et al.</em> 2009</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Guschin <em>et al.</em> 2010</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Miller <em>et al.</em> 2011</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Tesson <em>et al.</em> 2011</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Isalan 2012</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sanjana <em>et al.</em> 2012</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Holkers <em>et al.</em> 2013</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Maier <em>et al.</em> 2013</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Van Rensburg <em>et al.</em> 2013</a>).</p><p id="p-8">EMC assays are cost-effective methods that can be performed with the use of simple laboratory setups. This is why T7E1 and Surveyor Mismatch nucleases enzymes are used widely in the context of engineered nucleases mutagenesis projects. Yet, the spectrum of mutations that can be induced by engineered nucleases is broad (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Kim <em>et al.</em> 2013</a>). The choice of one or the other EMC enzyme, however, is currently lacking a side-by-side comparison. Different teams compared these enzymes in the past but not in the framework of mutation induced by ZFNs, transcription activator-like effector nucleases, or CRISPR/CRISPR-associated protein 9 riboendonucleases (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tsuji and Niida 2008</a>).</p><p id="p-9">We present here the first comparison of the sensitivity of T7E1 and Surveyor EMC assays on deletions and point mutations obtained by ZFNs in frog embryos. We report here our findings on the mutation detection limits and efficiency of T7E1 and Surveyor.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Construction of WT and mutant plasmids</h3><p id="p-10">We used the <em>Xenopus tropicalis smn2</em> gene and its exon-intron structure as previously described as a starting point to design our plasmid constructs (GenBank NM_001100240) (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Ymlahi-Ouazzani <em>et al.</em> 2010</a>). We targeted three genomic segments containing exons 2a (488 bp, 35.0% GC), exon 3 (572 bp, 37.6% GC), and exon 6 (500 bp, 37.4% GC) of <em>smn2</em> (<a href="http://www.g3journal.org/content/suppl/2015/01/07/g3.114.015834.DC1/015834SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/01/07/g3.114.015834.DC1/FigureS1.pdf">Figure S1</a>). We amplified these three segments from 100 ng of genomic DNA (<em>X. tropicalis</em> TGA strain) using primers pairs described in <a href="http://www.g3journal.org/content/suppl/2015/01/07/g3.114.015834.DC1/TableS1.pdf">Table S1</a>. We cloned these PCR products, and all the others described herein, into pCRII Vector using TA Cloning Kit (Life Technologies) and checked them by sequencing. We then used these constructs as the WT PCR products. We used an overlap extension PCR approach to produce 20-bp deletions in these three genomic segments (<a href="http://www.g3journal.org/content/suppl/2015/01/07/g3.114.015834.DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/07/g3.114.015834.DC1/TableS1.pdf">Table S1</a>) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ho <em>et al.</em> 1989</a>). We designed the position of the deletion so that a cleavage would lead to two products with significant size differences (150−200 bp). We cloned the three deletion mutant DNA segments and checked them by sequencing. We obtained the D15 and D19 alleles from experiments of genome engineering using ZFNs. In short, we microinjected ZFN RNAs targeting <em>smn</em> exon2a sequences into one-cell-stage <em>X. tropicalis</em> embryos. We extracted genomic DNA from pool of ten tailbud-stage embryos (Nucleospin tissue kit; Macherey-Nagel). We amplified the segments containing the exon2a by PCR from 100 ng of genomic DNA with the aforementioned primers. We purified the PCR products and cloned them. We checked 10 clones by sequencing and selected the mutant clones D15 and D19.</p></div><div id="sec-3" class="subsection"><h3>PCR</h3><p id="p-11">We used 1 ng of plasmidic DNA template in our standard PCRs. Each 20-µL PCR contained 10 µL of One<em>Taq</em> Hot Start 2X Master Mix (New England Biolabs), 400 nM of each primer and nuclease-free water. PCR conditions were 1 cycle of 5 min at 95°, followed by 40 cycles of 30 sec at 95°, 30 sec at 50° or 58° (depending on the primer pairs, see <a href="http://www.g3journal.org/content/suppl/2015/01/07/g3.114.015834.DC1/TableS1.pdf">Table S1</a>), and 30 sec at 72°, and finishing with 5 min incubation at 72°. We checked PCR products amplification and concentration by spectrophotometry and standard gel electrophoresis.</p></div><div id="sec-4" class="subsection"><h3>Heteroduplex formation</h3><p id="p-12">We mixed 250 ng of PCR products obtained from WT and mutant plasmids and denatured them by heating at 99° for 5 min in a thermocycler (Biometra). We then formed the heteroduplexes by cooling down to 65° for 30 min and to 23° for 30 min using a thermocycler.</p></div><div id="sec-5" class="subsection"><h3>Digestion by T7 endonuclease I or Surveyor</h3><p id="p-13">We performed T7 endonuclease I digestion at 37° for 30 min using 5 U of T7 endonuclease I (T7E1; New England BioLabs) on 20 µL of unpurified PCR products (~250 ng) in a reaction volume of 50 µL. We stopped the reaction by adding 4 µL of 0.5 M ethylenediaminetetraacetic acid. We performed the Surveyor nuclease assay using 20 µL of unpurified PCR products incubated with 1 µL of Surveyor nuclease (Transgenomic SURVEYOR mutation detection kit for standard gel electrophoresis), 1 µL of Surveyor Enhancer, and 4 µL of 0.15 M MgCl<sub>2</sub> in a 50-µL reaction. We incubated the mix for 1 hr at 42° and stopped the reaction by adding 4 µL of the stop solution provided in the kit. The reactions were either kept at −20° or used immediately for electrophoresis.</p></div><div id="sec-6" class="subsection"><h3>Electrophoresis</h3><p id="p-14">To assess T7E1 and Surveyor nuclease digestion, we performed capillary electrophoresis on an Agilent 2100 Bioanalyzer. We used the DNA 1000 LabChip kit for electrophoresis of double-stranded DNA with detection by an intercalating dye and loaded 1 µL of the unpurified reaction products on the DNA chips. We used the 2100 Expert software (Agilent Technologies) for quantification and sizing of the digestion products. Data were exported in .csv files and electropherograms were drawn using R statistical software scripts.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><p id="p-15">We first compared the efficiency of two enzymatic assays based on mismatch cleavage: T7E1 and Surveyor. We set up an assay in which we mixed different ratios of WT and deletion mutant DNA to produce heteroduplex molecules. We analyzed the cleavage products by using capillary lab-on-a-chip nondenaturing electrophoresis on a Bioanalyzer. We reasoned that such assays could be dependent on the PCR product sequence and thus we analyzed three different PCR products. The sequences of the three PCR products are different, and their GC content ranged from 35.0 to 37.6%, close to the genome wide GC content (40.1%). Because we know that most mutations induced by engineered nucleases are deletions, we constructed 20-bp deleted versions of each PCR product (<a href="http://www.g3journal.org/content/suppl/2015/01/07/g3.114.015834.DC1/FigureS1.pdf">Figure S1</a>). We will refer to these three sets of different PCR products as <em>smn</em> exon 2a, <em>smn</em> exon 3, and <em>smn</em> exon 6.</p><p id="p-16">In a first set of experiments, we mixed WT and mutant PCR products in equal quantities before heteroduplex formation. We expected that treatment with T7E1 or Surveyor nucleases would lead to cleavage of heteroduplex molecules in two smaller molecules. Besides these cleavage products, we expected uncleaved WT or mutant homoduplexes, and eventually some uncleaved heteroduplexes. These uncleaved DNA would altogether compose the main peak (peaks *, <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) on an electropherogram provided by a BioAnalyzer fragment analysis. By design, we know that the WT and mutant homoduplexes differ in size by 20 bp but our electrophoresis assay cannot distinguish these products for exon 2a and exon 3, whereas it can for exon 6 (peaks *, <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). The smaller molecules generated by cleavage would compose the two smaller peaks on an electropherogram (peaks • and ⬥, <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). Thus, we would expect the nuclease digestion products would give rise to three peaks on an electropherogram.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/407/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of Surveyor and T7E1 efficiencies. Electropherograms obtained on three different DNA products (exon 2a, exon 3, and exon 6) treated using either the Surveyor (top two rows) or T7E1 enzyme (bottom two rows). The Surveyor and T7E1 assays were made on a 50% mix of wild-type and 20-bp deletion mutants. For the Surveyor and T7E1 negative controls (co-), we prepared individual reactions using only wild-type or mutant DNA and pooled the products before electrophoresis. Peaks colored in gray correspond to the internal molecular weight standards: the small molecular weight marker measures 15 bp and the large measures 1500 bp. Peaks labeled with an asterisk (*) correspond to uncleaved DNA, peaks labeled &#x2022; and &#x2B25; correspond to cleaved DNA heteroduplex." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1001983248" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of Surveyor and T7E1 efficiencies. Electropherograms obtained on three different DNA products (exon 2a, exon 3, and exon 6) treated using either the Surveyor (top two rows) or T7E1 enzyme (bottom two rows). The Surveyor and T7E1 assays were made on a 50% mix of wild-type and 20-bp deletion mutants. For the Surveyor and T7E1 negative controls (co-), we prepared individual reactions using only wild-type or mutant DNA and pooled the products before electrophoresis. Peaks colored in gray correspond to the internal molecular weight standards: the small molecular weight marker measures 15 bp and the large measures 1500 bp. Peaks labeled with an asterisk (*) correspond to uncleaved DNA, peaks labeled &#x2022; and &#x2B25; correspond to cleaved DNA heteroduplex.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/407/F1.medium.gif" width="440" height="299"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/3/407/F1.medium.gif" width="440" height="299"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/407/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/407/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465709" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-17" class="first-child">Comparison of Surveyor and T7E1 efficiencies. Electropherograms obtained on three different DNA products (exon 2a, exon 3, and exon 6) treated using either the Surveyor (top two rows) or T7E1 enzyme (bottom two rows). The Surveyor and T7E1 assays were made on a 50% mix of wild-type and 20-bp deletion mutants. For the Surveyor and T7E1 negative controls (co-), we prepared individual reactions using only wild-type or mutant DNA and pooled the products before electrophoresis. Peaks colored in gray correspond to the internal molecular weight standards: the small molecular weight marker measures 15 bp and the large measures 1500 bp. Peaks labeled with an asterisk (*) correspond to uncleaved DNA, peaks labeled • and ⬥ correspond to cleaved DNA heteroduplex.</p><div class="sb-div caption-clear"></div></div></div><p id="p-18">When equal quantities of mutant and WT DNA were used, we observed the same number of peaks with both nucleases: two peaks were distinguishable for exon 2a and three for exons 3 and exons 6 (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). The sizes of the generated fragments corresponded to what we expected from the deletion’s site. In all three cases the peak fluorescence values obtained with Surveyor were lower than those obtained with T7E1 (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>). This explained why the smallest fragment (peak •) was not observed for exon 2a using Surveyor. Furthermore, the baseline of the electropherogram was neither straight nor regular in all samples treated by Surveyor because of exonucleolytic activity, as previously reported (<a id="xref-ref-36-5" class="xref-bibr" href="#ref-36">Qiu <em>et al.</em> 2004</a>; <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Till <em>et al.</em> 2004</a>). This was particularly striking in the control samples for exon 3 and exon 6. In addition small peaks located near the 15-bp molecular weight marker in all samples treated by Surveyor suggested an aspecific digestion of the PCR products. Because we used the same quantity of DNA in all reactions, these lower fluorescence values and irregular base line indicated a loss of DNA due to the Surveyor treatment. Both T7E1 and Surveyor were able to cleave heteroduplexes independently of the PCR product. Yet, T7E1 treatment leads to a higher signal to noise ratio than the Surveyor treatment.</p><p id="p-19">We then evaluated the sensitivity of T7E1 and Surveyor for our three different DNA products. When we mixed 5% of mutant DNA with WT DNA, we observed only one peak for exon 2a after Surveyor treatment and two or three peaks for all other cases (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). We observed the largest peak obtained from the cleavage of heteroduplexes in all three DNAs samples treated by T7E1 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, peaks ⬥). We could guess this same peak by using the Surveyor assay on exon 3 DNA sample, but we could not observe this peak using the Surveyor assay on exon 2a DNA sample. In addition, we could never observe the smallest peak in DNA samples treated by Surveyor (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, peaks •). The T7E1 assay results were always easier to interpret than the Surveyor. In conclusion, these results showed that the T7E1 assay enables the detection of 5% of heteroduplex DNA in a background of WT DNA.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/407/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Limits of sensitivity of Surveyor and T7E1 on different templates. Electropherograms obtained on DNA products from exon 2a, 3, and 6 after Surveyor or T7E1 digestion. The Surveyor and T7E1 assays were made on a mix composed of 5% 20-bp deletion mutants and 95% of wild-type DNA. Peaks colored in gray correspond to the internal molecular weight standards: the small molecular weight marker measures 15 bp and the large measures 1500 bp. Peaks labeled with an asterisk (*) correspond to uncleaved DNA, peaks labeled &#x2022; and &#x2B25; correspond to cleaved DNA heteroduplex." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1001983248" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Limits of sensitivity of Surveyor and T7E1 on different templates. Electropherograms obtained on DNA products from exon 2a, 3, and 6 after Surveyor or T7E1 digestion. The Surveyor and T7E1 assays were made on a mix composed of 5% 20-bp deletion mutants and 95% of wild-type DNA. Peaks colored in gray correspond to the internal molecular weight standards: the small molecular weight marker measures 15 bp and the large measures 1500 bp. Peaks labeled with an asterisk (*) correspond to uncleaved DNA, peaks labeled &#x2022; and &#x2B25; correspond to cleaved DNA heteroduplex.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/407/F2.medium.gif" width="440" height="300"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/3/407/F2.medium.gif" width="440" height="300"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/407/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/407/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465766" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Limits of sensitivity of Surveyor and T7E1 on different templates. Electropherograms obtained on DNA products from exon 2a, 3, and 6 after Surveyor or T7E1 digestion. The Surveyor and T7E1 assays were made on a mix composed of 5% 20-bp deletion mutants and 95% of wild-type DNA. Peaks colored in gray correspond to the internal molecular weight standards: the small molecular weight marker measures 15 bp and the large measures 1500 bp. Peaks labeled with an asterisk (*) correspond to uncleaved DNA, peaks labeled • and ⬥ correspond to cleaved DNA heteroduplex.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">To detect the limits of sensitivity for T7E1 and Surveyor assay, we made mixes of mutant and WT exon 3 DNAs in different ratios from 5% up to 100% of mutant DNA. When digested by T7E1, the intensity of the main peak (*) was inversely proportional to the homoduplex:heteroduplex ratio (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, peak *). Conversely, the intensities of the smaller peaks were proportional to the homoduplex/heteroduplex ratio. We detected cleavage products even for 5% of heteroduplex DNA (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, peaks • and ⬥, samples 5% and 95%). We could see that the smallest peak was a doublet that differed by 20 bp. This 20-bp size difference corresponds to the size of the deletion. On the basis of these results, we conclude that the T7E1 assay could detect around 5% of mutant DNA.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/407/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of Surveyor and T7E1 sensitivity. Electropherograms obtained on exon 3 DNA products after Surveyor or T7E1 digestion. The percentage indicated on the right corresponds to the quantity of mutant DNA in a pool of mutant and wild-type DNA. The Surveyor and T7E1 negative controls (co-) correspond to reactions made using mutant DNA only (100%). Peaks colored in gray correspond to the internal molecular weight standards: the small molecular weight marker measures 15 bp and the large measures 1500 bp. Peaks labeled with an asterisk (*) correspond to uncleaved DNA, peaks labeled &#x2022; and &#x2B25; correspond to cleaved DNA heteroduplex." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1001983248" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of Surveyor and T7E1 sensitivity. Electropherograms obtained on exon 3 DNA products after Surveyor or T7E1 digestion. The percentage indicated on the right corresponds to the quantity of mutant DNA in a pool of mutant and wild-type DNA. The Surveyor and T7E1 negative controls (co-) correspond to reactions made using mutant DNA only (100%). Peaks colored in gray correspond to the internal molecular weight standards: the small molecular weight marker measures 15 bp and the large measures 1500 bp. Peaks labeled with an asterisk (*) correspond to uncleaved DNA, peaks labeled &#x2022; and &#x2B25; correspond to cleaved DNA heteroduplex.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/407/F3.medium.gif" width="440" height="284"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/3/407/F3.medium.gif" width="440" height="284"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/407/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/407/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465749" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">Comparison of Surveyor and T7E1 sensitivity. Electropherograms obtained on exon 3 DNA products after Surveyor or T7E1 digestion. The percentage indicated on the right corresponds to the quantity of mutant DNA in a pool of mutant and wild-type DNA. The Surveyor and T7E1 negative controls (co-) correspond to reactions made using mutant DNA only (100%). Peaks colored in gray correspond to the internal molecular weight standards: the small molecular weight marker measures 15 bp and the large measures 1500 bp. Peaks labeled with an asterisk (*) correspond to uncleaved DNA, peaks labeled • and ⬥ correspond to cleaved DNA heteroduplex.</p><div class="sb-div caption-clear"></div></div></div><p id="p-23">When we digested the DNAs with Surveyor, we did not observe the same sensitivity pattern. We detected the main peak, but its value of fluorescence was three times lower than with T7E1, even though we used the same quantity of starting DNA. In line with this observation, we observed small peaks around the 15-bp DNA marker. Besides, all Surveyor treated samples led to greater electropherogram base lines in comparison with T7E1-treated ones. These results suggested again an exonucleolytic activity as previously reported. Yet we could detect from 10 to 90% of mutant DNA using Surveyor (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>).</p><p id="p-24">We then quantified the efficiency of these two enzymes by comparing the quantity of cleaved PCR products (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/07/g3.114.015834.DC1/FigureS2.pdf">Figure S2</a>). In theory, when mutant and WT DNA are present in equal quantities, we can form up to 50% of heteroduplex DNA molecules. We quantified the quantity of cleaved heteroduplexes and we expressed our results as percentages of cleaved heteroduplexes relative to the total quantity of DNA.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/407/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quantification of Surveyor and T7E1 efficiency and sensitivity. This graph shows the fraction of cleaved products from all products (y-axis) in a mixture composed of various quantities of deletion mutants of exon 3 in a population of mutant and wild-type DNA molecules (x-axis). Open circles correspond to T7E1 digestion products, open squares correspond to Surveyor digestion products." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1001983248" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Quantification of Surveyor and T7E1 efficiency and sensitivity. This graph shows the fraction of cleaved products from all products (y-axis) in a mixture composed of various quantities of deletion mutants of exon 3 in a population of mutant and wild-type DNA molecules (x-axis). Open circles correspond to T7E1 digestion products, open squares correspond to Surveyor digestion products.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/407/F4.medium.gif" width="440" height="352"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/3/407/F4.medium.gif" width="440" height="352"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/407/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/407/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465762" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">Quantification of Surveyor and T7E1 efficiency and sensitivity. This graph shows the fraction of cleaved products from all products (y-axis) in a mixture composed of various quantities of deletion mutants of exon 3 in a population of mutant and wild-type DNA molecules (x-axis). Open circles correspond to T7E1 digestion products, open squares correspond to Surveyor digestion products.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">We observed a symmetry on either side of 50% of mutant DNA that correlates with the results presented in <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>. T7E1 cleaved nearly all heteroduplexes for mixes containing 30 and 70% of mutant DNA. Surveyor cleaved nearly all heteroduplexes for mixes containing 20 and 80% of mutant DNA. We did not observe any Surveyor cleavage product for 5, 90, and 95%. The quantity of cleaved heteroduplexes with T7E1 was always 8% superior to that obtained with Surveyor. In the range of 40–60%, the quantity of cleaved product reached a plateau: 30% for Surveyor (<em>i.e.</em>, 60% of all heteroduplexes) and 37% for T7E1 (<em>i.e.</em>, 74% of all heteroduplexes).</p><p id="p-27">We then tested these enzymes on two mutant DNA molecules obtained from ZFN-injected <em>Xenopus</em> embryos. The sequencing data obtained for clones D15 and D19 showed that both alleles contained a single-nucleotide deletion polymorphism at position 46 of the exon 2a amplicon. The D15 allele contained two single nucleotide substitutions (A &gt; G) at position 85 and 89 of the amplicon (position 29 and 33 of <em>smn</em> exon 2a) and one SNP at position 293 (T &gt; A). The second allele, D19, contained two single-nucleotide substitutions (T &gt; C) at position 78 and 127 of the amplicon (position 22 and 71 of <em>smn</em> exon 2a) and one SNP at position 347 (T &gt; C). To test the Surveyor and T7E1 assay on these complex substrates, we set up a digestion reaction composed of a mix containing 50% of the mutant alleles and 50% of the WT allele. We expected five DNA fragments from a whole digest for D15 (4, 39, 46, 195, and 204 bp) and D19 (32, 46, 49, 141, and 220 bp) heteroduplexes (<a href="http://www.g3journal.org/content/suppl/2015/01/07/g3.114.015834.DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/07/g3.114.015834.DC1/FigureS4.pdf">Figure S4</a>). The 5-bp digestion product for D15 could not be visualized. After T7E1 or Surveyor digestion, we observed a complex pattern of cleavage products (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). We reasoned that the digestions could be partial, so that more DNA fragments would be produced. Indeed, most peaks distinguishable on the electropherogram could fit a DNA fragment produced by an incomplete digestion (<a href="http://www.g3journal.org/content/suppl/2015/01/07/g3.114.015834.DC1/TableS2.pdf">Table S2</a>). The most visible example can be seen on <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> as peaks of about 400 bp resulting from a single cleavage at position 46 of the amplicon. Yet, we noticed that the single-nucleotide polymorphisms (<em>i.e.</em>, SNPs) SNP 293 from D15 and SNP 347 from D19 were cleaved with better efficiency when Surveyor nuclease was used (boxed peaks in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>). For D15, we observed a peak of 197 bp reaching about 10 fluorescence units (FUs) with Surveyor, whereas a peak at 200 bp only reached about 3 fluorescence units (FUs) and was not even marked for T7E1 (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). Similarly for D19, the 141-bp fragment resulting from cleavage of SNP 347 was digested more efficiently with Surveyor (peak at 143 bp), but we could not see a signal above the background at this size with T7E1. In conclusion, it appears that Surveyor outperforms T7E1 for single-nucleotide changes.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/407/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of Surveyor and T7E1 efficiencies on point mutations. The top of this figure presents schematics of the D15 and D19 sequence differences in comparison with the wild-type smn exon 2a. The electrophoregrams obtained on D15 or D19 DNA products are presented from top to bottom in the following order: after T7E1 digestion, after Surveyor digestion, and an overlay of the two conditions. The size of the peaks is given in base pairs. Only peaks called by the software were labeled. Peaks matching the size of a partial digestion product are written in bold italics. Boxed peaks are discussed in the text. The portions of the small and large markers have been omitted for clarity." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1001983248" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of Surveyor and T7E1 efficiencies on point mutations. The top of this figure presents schematics of the D15 and D19 sequence differences in comparison with the wild-type &lt;em&gt;smn&lt;/em&gt; exon 2a. The electrophoregrams obtained on D15 or D19 DNA products are presented from top to bottom in the following order: after T7E1 digestion, after Surveyor digestion, and an overlay of the two conditions. The size of the peaks is given in base pairs. Only peaks called by the software were labeled. Peaks matching the size of a partial digestion product are written in bold italics. Boxed peaks are discussed in the text. The portions of the small and large markers have been omitted for clarity.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/407/F5.medium.gif" width="440" height="341"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/3/407/F5.medium.gif" width="440" height="341"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/407/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/407/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465731" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-28" class="first-child">Comparison of Surveyor and T7E1 efficiencies on point mutations. The top of this figure presents schematics of the D15 and D19 sequence differences in comparison with the wild-type <em>smn</em> exon 2a. The electrophoregrams obtained on D15 or D19 DNA products are presented from top to bottom in the following order: after T7E1 digestion, after Surveyor digestion, and an overlay of the two conditions. The size of the peaks is given in base pairs. Only peaks called by the software were labeled. Peaks matching the size of a partial digestion product are written in bold italics. Boxed peaks are discussed in the text. The portions of the small and large markers have been omitted for clarity.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-8"><h2 class="">Discussion</h2><p id="p-29">We showed that both T7E1 and Surveyor enabled the detection of a mutant allele present in half of a DNA mixture (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1</a>). Thus, both assays can work on heterozygous individuals, and Surveyor was able to cleave heteroduplex substrates containing a 20 nucleotides bulge. As expected from substrate preferences, the signal-to-noise ratio is clearly greater for T7E1 than for Surveyor. Similar results have been reported using a regular agarose gel electrophoresis assay (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sakurai <em>et al.</em> 2014</a>). The known 5′-exonuclease activity of Surveyor is probably responsible of DNA degradation resulting in greater background. The exonuclease activity of Surveyor has previously been reported (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Till <em>et al.</em> 2004</a>; <a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">Yeung <em>et al.</em> 2005</a>) and shown to be highly dependent on the concentration of Mg<sup>2+</sup> and thus on the purification method and on the PCR buffer. We added MgCl<sub>2</sub> to the reaction to avoid this effect as much as possible. Since we obtained similar results on purified and unpurified PCR products (data not shown) we opted for the simpler and economical solution to avoid the PCR purification step.</p><p id="p-30">We could detect a 20-bp deletion mutation in a pool of DNA containing as little as 5% of a mutant allele when we used T7E1. When we used the Surveyor assay, the detection limit was closer to 20%. The exonuclease activity of Surveyor produced too many small DNA fragments that masked a potentially specific signal. For example we observed accumulation of DNA fragments below 25 bp (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a> and <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). Using T7E1, when the cleavage products are sufficiently small, we can detect pairs of cleavage products separated by 20 bp. This size difference arise from T7E1 cleaving on the 3′ end of the mismatches, thereby producing two fragments including one with a 20 bp overhang. Thus the size difference very likely corresponds to the size of the deletion in our mutant allele.</p><p id="p-31">The two enzymes exhibited different behaviors. T7E1 showed a maximum efficiency in a DNA pool containing between 5 and 30% of 20-bp deletion mutant DNA, and by symmetry from 70 to 95% of mutant DNA (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). A plateau was reached when the heteroduplex proportion was maximal, <em>i.e.</em>, at 50% of mutant DNA in the pool. We observed that at best T7E1 cleaved about 80% of heteroduplexes in the DNA pool.</p><p id="p-32">Because of the high background of nonspecific cleavage products by Surveyor assay, we could not reliably evaluate its efficiency on a DNA pool containing from 5 to 30% of mutant molecules. It is possible that the enzyme cut all the heteroduplex at the desired location, but it is possible that the 5′ exonuclease activity of Surveyor further process the cleaved products to generate size heterogeneity. Thus the observed Surveyor specificity was much lower than that of T7E1 in our protocol. We observed that at best Surveyor cleaved 60% of heteroduplexes DNA, a lower efficiency than T7E1.</p><p id="p-33">The different activities of these enzymes according to the precise nature of the mismatch were already known (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Tsuji and Niida 2008</a>). T7E1 identifies preferentially insertions and deletions, whereas Surveyor is better to recognize substitutions. Here we showed that the surrounding DNA sequence did not impact their activity (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). On the three DNA fragments tested, we could measure the limit of sensitivity for Surveyor (10% of mutant DNA) and for T7E1 (5%). Thus, T7E1 was always more sensitive than Surveyor in our assays using deletion heteroduplex substrates.</p><p id="p-34">We report here results obtained using a simple, reproducible, and economic protocol. Any molecular biology laboratory can use this protocol as a first step for mutation screening before sequencing. The whole procedure starting from genomic DNA can be performed in a single day of work, in contrast with the recently reported application of HRM analysis that requires specific instrumentation and the development of a specific PCR assay (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Dahlem <em>et al.</em> 2012</a>). We could detect 5% of deletion mutant DNA using T7E1. Yet, the reported mutation hit rate in F0 individuals issued from nuclease injections ranges from 10 to 47% (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Young <em>et al.</em> 2011</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ishibashi <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lei <em>et al.</em> 2012</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Nakajima and Yaoita 2013</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Suzuki <em>et al.</em> 2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Guo <em>et al.</em> 2014</a>). Thus, this T7E1 assay is sensitive enough to identify mutations in DNA samples obtained from F0 individuals. In addition a pooling strategy can be implemented to genotype F1 individuals. 5% sensitivity level corresponds roughly to one heterozygous mutant in a pool of eight individuals (one mutant allele of 16 alleles, <em>e.g.</em>, 6.25%). Using such a pooling strategy we could screen 96 individuals on a single Bioanalyzer DNA chip. Moreover, pooling can be beneficial for mutation detection, since the fraction of heteroduplex increases with the pool size.</p><p id="p-35">One possible bias in the EMC approach comes from the production of the template DNA using PCR. If the fidelity of the DNA polymerase is limited, mutations can be introduced in the template. Even if the proportions are expected to be low, this can impact the EMC assay in an unpredictable way. A solution to this problem is to use proof-reading polymerases. Yet such proof-reading polymerases could introduce biases when the quantity of mutant molecules in the template DNA is low. Upon denaturation and hybridation cycles, and especially in the late PCR cycles, heteroduplex dsDNA are formed and could lead to an increase of recombined PCR products (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Judo <em>et al.</em> 1998</a>). It has been shown that proof-reading polymerases lead to increased levels of recombined PCR products (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Judo <em>et al.</em> 1998</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lahr and Katz 2009</a>). Such recombination events during PCR could lead to dsDNA in which the mutation to be detected is completely or partially corrected. Another problem introduced by PCR is allele drop-out, the amplification bias of one allele <em>vs.</em> another, in one way or another. In particular deletions can affect the melting temperature of the PCR product. Usually, this effect tends to promote the amplification of smaller PCR products. Thus, the production of the template DNA using PCR can affect the sensitivity of the EMC assay and the estimation of the original quantity of the mutant allele. It has been shown that recombination during PCR can be limited by reducing the quantity of starting DNA, reducing the number of PCR cycles, and increasing extension time (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Judo <em>et al.</em> 1998</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Lahr and Katz 2009</a>). Similarly any genetic polymorphism will affect the usefulness of EMC to detect mutations, and it is crucial to examine the extent of polymorphisms on the DNA region amplified by PCR. The overall design of the PCR assay, including the amplicon size, is thus of paramount importance. Several studies took advantage of restriction enzymes sites to assess the mutation hit rate in given conditions. Yet it may be impossible to find a unique restriction site near the nuclease target sequence. Because the EMC assays can be used whatever the sequence of the PCR template, it is a more versatile strategy.</p><p id="p-36">The EMC assay can benefit from several optimizations. For example, the use of fluorescent oligonucleotides could improve the signal-to-noise ratio, especially if small products are expected from the cleavage (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Behrensdorf <em>et al.</em> 2002</a>). The exonuclease activity of Surveyor leads to the nibbling of dsDNA and a lower signal-to-noise ratio. Using 5′ fluorescently labeled oligonucleotides has been shown to increase dramatically the sensitivity of CEL assays (<a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Till <em>et al.</em> 2004</a>). This effect has been explained by the combination of the protection provided by the 5′ label against 5′ exonuclease activity, and an increased signal to noise ration since degraded products lose the label (<a id="xref-ref-45-6" class="xref-bibr" href="#ref-45">Till <em>et al.</em> 2004</a>). The use of a ligase has also been reported to improve the results (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Huang <em>et al.</em> 2012</a>). However, these ameliorations are associated with a loss of simplicity and an increase in costs.</p><p id="p-37">The two enzymes tested exhibit an exonuclease activity to some extent. Such a nonspecific exonuclease activity can be reduced by the combination of the EMC enzyme with ampligase (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Huang <em>et al.</em> 2012</a>). In their paper, <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Huang <em>et al.</em> (2012)</a> discuss two important factors affecting the exonuclease activity of the EMC enzymes. The first factor is the buffer used for the enzymatic incubation, and the second factor is the incubation time. It is crucial to avoid the incubation in the PCR buffer because it counteracts the effects of the ampligase. The incubation time should not exceed 30 min, and should ideally be 20 min long. Yet, these conditions are completely dependent of the enzyme used. The authors worked with samples containing 10 or 50% of single base mutant DNA. Here, we showed that T7E1 enabled the detection of 5% deletion mutant DNA. The exonuclease activity of Surveyor without ampligase led to the appearance of many small DNA fragments that masked the specific signal coming from heteroduplex digestions. In conclusion, we showed that T7E1 outperforms the Surveyor nuclease in terms of sensitivity with deletion heteroduplex substrates and should be preferred here.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-38">We thank Yann Audic and Sophie Dhorne-Pollet for critical reading of the manuscript. This work was funded by Genopole (Centre d’Exploration et de Recherche Fonctionnelle Amphibiens Poissons); the Association Française contre les Myopathies (Amphigenetools); Centre National de la Recherche Scientifique; and Université d’Evry-Val d’Essonne.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-39">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015834/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015834/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-40"><em>Communicating editor: J. R. Hesselberth</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-11-20"><span class="received-label">Received </span>November 20, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-01-06"><span class="accepted-label">Accepted </span>January 6, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Vouillot <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.1" data-doi="10.1093/nar/23.24.5082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Babon</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Youil</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotton</span>  <span class="cit-name-given-names">R. G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Improved strategy for mutation detection—a modification to the enzyme mismatch cleavage method.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">5082</span>–<span class="cit-lpage">5084</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBabon%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D23%26rft.issue%253D24%26rft.spage%253D5082%26rft.epage%253D5084%26rft.atitle%253DImproved%2Bstrategy%2Bfor%2Bmutation%2Bdetection--a%2Bmodification%2Bto%2Bthe%2Benzyme%2Bmismatch%2Bcleavage%2Bmethod%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F23.24.5082%26rft_id%253Dinfo%253Apmid%252F8559669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJuYXIiO3M6NToicmVzaWQiO3M6MTA6IjIzLzI0LzUwODIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvNDA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.2" data-doi="10.1093/nar/gnf063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Behrensdorf</span>  <span class="cit-name-given-names">H. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pignot</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Windhab</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kappel</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Rapid parallel mutation scanning of gene fragments using a microelectronic protein-DNA chip format.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">e64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBehrensdorf%26rft.auinit1%253DH.%2BA.%26rft.volume%253D30%26rft.issue%253D14%26rft.spage%253De64%26rft.epage%253De64%26rft.atitle%253DRapid%2Bparallel%2Bmutation%2Bscanning%2Bof%2Bgene%2Bfragments%2Busing%2Ba%2Bmicroelectronic%2Bprotein-DNA%2Bchip%2Bformat%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgnf063%26rft_id%253Dinfo%253Apmid%252F12136112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMwLzE0L2U2NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy80MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bentley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacLennan</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calvo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dearolf</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Targeted recovery of mutations in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">1169</span>–<span class="cit-lpage">1173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DBentley%26rft.auinit1%253DA.%26rft.volume%253D156%26rft.issue%253D3%26rft.spage%253D1169%26rft.epage%253D1173%26rft.atitle%253DTargeted%2BRecovery%2Bof%2BMutations%2Bin%2BDrosophila%26rft_id%253Dinfo%253Apmid%252F11063692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ni8zLzExNjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvNDA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.4" data-doi="10.1073/pnas.0810475105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beumer</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trautman</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bozas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient gene targeting in Drosophila by direct embryo injection with zinc-finger nucleases.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">19821</span>–<span class="cit-lpage">19826</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0810475105%26rft_id%253Dinfo%253Apmid%252F19064913%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzUwLzE5ODIxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.5" data-doi="10.1016/0022-2836(89)90596-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhattacharyya</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lilley</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Single base mismatches in DNA. Long- and short-range structure probed by analysis of axis trajectory and local chemical reactivity.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBhattacharyya%26rft.auinit1%253DA.%26rft.volume%253D209%26rft.issue%253D4%26rft.spage%253D583%26rft.epage%253D597%26rft.atitle%253DSingle%2Bbase%2Bmismatches%2Bin%2BDNA.%2BLong-%2Band%2Bshort-range%2Bstructure%2Bprobed%2Bby%2Banalysis%2Bof%2Baxis%2Btrajectory%2Band%2Blocal%2Bchemical%2Breactivity.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252889%252990596-2%26rft_id%253Dinfo%253Apmid%252F2585502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(89)90596-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2585502&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989AY10100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.6" data-doi="10.1038/ng847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coghill</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hugill</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parkinson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davison</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glenister</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A gene-driven approach to the identification of ENU mutants in the mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCoghill%26rft.auinit1%253DE.%2BL.%26rft.volume%253D30%26rft.issue%253D3%26rft.spage%253D255%26rft.epage%253D256%26rft.atitle%253DA%2Bgene-driven%2Bapproach%2Bto%2Bthe%2Bidentification%2Bof%2BENU%2Bmutants%2Bin%2Bthe%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng847%26rft_id%253Dinfo%253Apmid%252F11850622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11850622&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174157600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.7" data-doi="10.1104/pp.126.2.480"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colbert</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Till</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tompa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steine</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">High-throughput screening for induced point mutations.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">480</span>–<span class="cit-lpage">484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DColbert%26rft.auinit1%253DT.%26rft.volume%253D126%26rft.issue%253D2%26rft.spage%253D480%26rft.epage%253D484%26rft.atitle%253DHigh-Throughput%2BScreening%2Bfor%2BInduced%2BPoint%2BMutations%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.126.2.480%26rft_id%253Dinfo%253Apmid%252F11402178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMjYvMi80ODAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvNDA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.8" data-doi="10.1111/j.0960-7412.2003.01999.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Till</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Efficient discovery of DNA polymorphisms in natural populations by Ecotilling.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">778</span>–<span class="cit-lpage">786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DComai%26rft.auinit1%253DL.%26rft.volume%253D37%26rft.issue%253D5%26rft.spage%253D778%26rft.epage%253D786%26rft.atitle%253DEfficient%2Bdiscovery%2Bof%2BDNA%2Bpolymorphisms%2Bin%2Bnatural%2Bpopulations%2Bby%2BEcotilling.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0960-7412.2003.01999.x%26rft_id%253Dinfo%253Apmid%252F14871304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0960-7412.2003.01999.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14871304&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188865200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.9" data-doi="10.1371/journal.pgen.1002861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dahlem</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoshijima</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jurynec</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunther</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Starker</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Simple methods for generating and detecting locus-specific mutations induced with TALENs in the zebrafish genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDahlem%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D8%26rft.issue%253D8%26rft.spage%253De1002861%26rft.epage%253De1002861%26rft.atitle%253DSimple%2Bmethods%2Bfor%2Bgenerating%2Band%2Bdetecting%2Blocus-specific%2Bmutations%2Binduced%2Bwith%2BTALENs%2Bin%2Bthe%2Bzebrafish%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002861%26rft_id%253Dinfo%253Apmid%252F22916025%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002861&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22916025&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.10" data-doi="10.1016/j.sbi.2007.11.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Déclais</span>  <span class="cit-name-given-names">A.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lilley</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">New insight into the recognition of branched DNA structure by junction-resolving enzymes.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Struct. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">86</span>–<span class="cit-lpage">95</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bstructural%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BStruct%2BBiol%26rft.aulast%253DDeclais%26rft.auinit1%253DA.%2BC.%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D86%26rft.epage%253D95%26rft.atitle%253DNew%2Binsight%2Binto%2Bthe%2Brecognition%2Bof%2Bbranched%2BDNA%2Bstructure%2Bby%2Bjunction-resolving%2Benzymes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.sbi.2007.11.001%26rft_id%253Dinfo%253Apmid%252F18160275%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.sbi.2007.11.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18160275&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253937100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.11" data-doi="10.1016/j.jmb.2006.04.037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Déclais</span>  <span class="cit-name-given-names">A.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">A. D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lilley</span>  <span class="cit-name-given-names">D. M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Structural recognition between a four-way DNA junction and a resolving enzyme.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">359</span>: <span class="cit-fpage">1261</span>–<span class="cit-lpage">1276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DDeclais%26rft.auinit1%253DA.%2BC.%26rft.volume%253D359%26rft.issue%253D5%26rft.spage%253D1261%26rft.epage%253D1276%26rft.atitle%253DStructural%2Brecognition%2Bbetween%2Ba%2Bfour-way%2BDNA%2Bjunction%2Band%2Ba%2Bresolving%2Benzyme.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2006.04.037%26rft_id%253Dinfo%253Apmid%252F16690083%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2006.04.037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16690083&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dodgson</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wells</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Action of single-strand specific nucleases on model DNA heteroduplexes of defined size and sequence.</span> <abbr class="cit-jnl-abbrev">Biochemistry (Mosc.)</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">2374</span>–<span class="cit-lpage">2379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%2B%2528Mosc.%2529%26rft.volume%253D16%26rft.spage%253D2374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.13" data-doi="10.1016/j.jmb.2012.11.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">A. D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Déclais</span>  <span class="cit-name-given-names">A.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lilley</span>  <span class="cit-name-given-names">D. M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The importance of the N-terminus of T7 endonuclease I in the interaction with DNA junctions.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">425</span>: <span class="cit-fpage">395</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BBiol.%26rft.volume%253D425%26rft.spage%253D395%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2012.11.029%26rft_id%253Dinfo%253Apmid%252F23207296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2012.11.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23207296&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.14" data-doi="10.1016/j.tibtech.2013.04.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaj</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gersbach</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbas</span>  <span class="cit-name-given-names">C. F.</span>  <span class="cit-name-suffix">3rd</span>.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">ZFN, TALEN, and CRISPR/Cas-based methods for genome engineering.</span> <abbr class="cit-jnl-abbrev">Trends Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">397</span>–<span class="cit-lpage">405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D397%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tibtech.2013.04.004%26rft_id%253Dinfo%253Apmid%252F23664777%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tibtech.2013.04.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23664777&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321175200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.15" data-doi="10.1126/science.1172447"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geurts</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cost</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freyvert</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeitler</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Knockout rats via embryo microinjection of zinc-finger nucleases.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">325</span>: <span class="cit-fpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGeurts%26rft.auinit1%253DA.%2BM.%26rft.volume%253D325%26rft.issue%253D5939%26rft.spage%253D433%26rft.epage%253D433%26rft.atitle%253DKnockout%2BRats%2Bvia%2BEmbryo%2BMicroinjection%2Bof%2BZinc-Finger%2BNucleases%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1172447%26rft_id%253Dinfo%253Apmid%252F19628861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjUvNTkzOS80MzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvNDA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.16" data-doi="10.1073/pnas.91.24.11660"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gohlke</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murchie</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lilley</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">R. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Kinking of DNA and RNA helices by bulged nucleotides observed by fluorescence resonance energy transfer.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">11660</span>–<span class="cit-lpage">11664</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DGohlke%26rft.auinit1%253DC.%26rft.volume%253D91%26rft.issue%253D24%26rft.spage%253D11660%26rft.epage%253D11664%26rft.atitle%253DKinking%2Bof%2BDNA%2Band%2BRNA%2Bhelices%2Bby%2Bbulged%2Bnucleotides%2Bobserved%2Bby%2Bfluorescence%2Bresonance%2Benergy%2Btransfer.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.91.24.11660%26rft_id%253Dinfo%253Apmid%252F7526401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTEvMjQvMTE2NjAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvNDA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.17" data-doi="10.1242/dev.099853"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Efficient RNA/Cas9-mediated genome editing in <em>Xenopus tropicalis</em>.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.099853%26rft_id%253Dinfo%253Apmid%252F24401372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czo5OiIxNDEvMy83MDciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvNDA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.18" data-doi="10.1007/978-1-60761-753-2_15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guschin</span>  <span class="cit-name-given-names">D. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waite</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katibah</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A rapid and general assay for monitoring endogenous gene modification.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">649</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">256</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DGuschin%26rft.auinit1%253DD.%2BY.%26rft.volume%253D649%26rft.spage%253D247%26rft.epage%253D256%26rft.atitle%253DA%2Brapid%2Band%2Bgeneral%2Bassay%2Bfor%2Bmonitoring%2Bendogenous%2Bgene%2Bmodification.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60761-753-2_15%26rft_id%253Dinfo%253Apmid%252F20680839%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60761-753-2_15&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20680839&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281187700015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.19" data-doi="10.1038/nature06158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hadden</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Déclais</span>  <span class="cit-name-given-names">A.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carr</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lilley</span>  <span class="cit-name-given-names">D. M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">S. E. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The structural basis of Holliday junction resolution by T7 endonuclease I.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">621</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHadden%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D449%26rft.issue%253D7162%26rft.spage%253D621%26rft.epage%253D624%26rft.atitle%253DThe%2Bstructural%2Bbasis%2Bof%2BHolliday%2Bjunction%2Bresolution%2Bby%2BT7%2Bendonuclease%2BI.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06158%26rft_id%253Dinfo%253Apmid%252F17873858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17873858&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249893500050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.20" data-doi="10.1016/0378-1119(89)90358-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ho</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pullen</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pease</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Site-directed mutagenesis by overlap extension using the polymerase chain reaction.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">51</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DHo%26rft.auinit1%253DS.%2BN.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D51%26rft.epage%253D59%26rft.atitle%253DSite-directed%2Bmutagenesis%2Bby%2Boverlap%2Bextension%2Busing%2Bthe%2Bpolymerase%2Bchain%2Breaction.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252889%252990358-2%26rft_id%253Dinfo%253Apmid%252F2744487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(89)90358-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2744487&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1989U265400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.21" data-doi="10.1093/nar/gks1446"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holkers</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maggio</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Janssen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miselli</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Differential integrity of TALE nuclease genes following adenoviral and lentiviral vector gene transfer into human cells.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">e63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1446%26rft_id%253Dinfo%253Apmid%252F23275534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjQxLzUvZTYzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howard</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watson</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roux</span>  <span class="cit-name-given-names">K. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Heteroduplex cleavage analysis using S1 nuclease.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DHoward%26rft.auinit1%253DJ.%2BT.%26rft.volume%253D27%26rft.issue%253D1%26rft.spage%253D18%26rft.epage%253D19%26rft.atitle%253DHeteroduplex%2Bcleavage%2Banalysis%2Busing%2BS1%2Bnuclease.%26rft_id%253Dinfo%253Apmid%252F10407654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10407654&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081341100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.23" data-doi="10.1002/elps.201100460"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheong</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A simple, high sensitivity mutation screening using Ampligase mediated T7 endonuclease I and Surveyor nuclease with microfluidic capillary electrophoresis.</span> <abbr class="cit-jnl-abbrev">Electrophoresis</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">788</span>–<span class="cit-lpage">796</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectrophoresis%26rft.volume%253D33%26rft.spage%253D788%26rft_id%253Dinfo%253Adoi%252F10.1002%252Felps.201100460%26rft_id%253Dinfo%253Apmid%252F22437793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/elps.201100460&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22437793&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000303154000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isalan</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Zinc-finger nucleases: how to play two good hands.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DIsalan%26rft.auinit1%253DM.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D34%26rft.atitle%253DZinc-finger%2Bnucleases%253A%2Bhow%2Bto%2Bplay%2Btwo%2Bgood%2Bhands.%26rft_id%253Dinfo%253Apmid%252F22205514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22205514&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298667000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.25" data-doi="10.1242/bio.20123228"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ishibashi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cliffe</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amaya</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Highly efficient bi-allelic mutation rates using TALENs in <em>Xenopus tropicalis</em>.</span> <abbr class="cit-jnl-abbrev">Biol. Open</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">1273</span>–<span class="cit-lpage">1276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BOpen%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fbio.20123228%26rft_id%253Dinfo%253Apmid%252F23408158%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiYmlvbG9wZW4iO3M6NToicmVzaWQiO3M6OToiMS8xMi8xMjczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.26" data-doi="10.1093/nar/26.7.1819"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Judo</span>  <span class="cit-name-given-names">M. S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wedel</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Stimulation and suppression of PCR-mediated recombination.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1825</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F26.7.1819%26rft_id%253Dinfo%253Apmid%252F9512558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjI2LzcvMTgxOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy80MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.27" data-doi="10.1038/nmeth.2364"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kweon</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.-S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">TALENs and ZFNs are associated with different mutation signatures.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D185%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2364%26rft_id%253Dinfo%253Apmid%252F23396284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2364&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23396284&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316650800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.28" data-doi="10.2144/000113219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lahr</span>  <span class="cit-name-given-names">D. J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katz</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reducing the impact of PCR-mediated recombination in molecular evolution and environmental studies using a new-generation high-fidelity DNA polymerase.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">866</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechniques%26rft.volume%253D47%26rft.spage%253D857%26rft_id%253Dinfo%253Adoi%252F10.2144%252F000113219%26rft_id%253Dinfo%253Apmid%252F19852769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/000113219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19852769&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277559900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.29" data-doi="10.1073/pnas.1215421109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lei</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Efficient targeted gene disruption in <em>Xenopus</em> embryos using engineered transcription activator-like effector nucleases (TALENs).</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">17484</span>–<span class="cit-lpage">17489</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1215421109%26rft_id%253Dinfo%253Apmid%252F23045671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzQzLzE3NDg0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.30" data-doi="10.1089/hum.2012.172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maier</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brennan</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binder-Scholl</span>  <span class="cit-name-given-names">G. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Efficient clinical scale gene modification via zinc finger nuclease-targeted disruption of the HIV co-receptor CCR5.</span> <abbr class="cit-jnl-abbrev">Hum. Gene Ther.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BGene%2BTher.%26rft.volume%253D24%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1089%252Fhum.2012.172%26rft_id%253Dinfo%253Apmid%252F23360514%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1089/hum.2012.172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23360514&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316463400036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.31" data-doi="10.1038/ng0295-177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mashal</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koontz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sklar</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Detection of mutations by cleavage of DNA heteroduplexes with bacteriophage resolvases.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMashal%26rft.auinit1%253DR.%2BD.%26rft.volume%253D9%26rft.issue%253D2%26rft.spage%253D177%26rft.epage%253D183%26rft.atitle%253DDetection%2Bof%2Bmutations%2Bby%2Bcleavage%2Bof%2BDNA%2Bheteroduplexes%2Bwith%2Bbacteriophage%2Bresolvases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0295-177%26rft_id%253Dinfo%253Apmid%252F7719346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0295-177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7719346&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QE67200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.32" data-doi="10.1038/nbt.1755"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barlow</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A TALE nuclease architecture for efficient genome editing.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMiller%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D29%26rft.issue%253D2%26rft.spage%253D143%26rft.epage%253D148%26rft.atitle%253DA%2BTALE%2Bnuclease%2Barchitecture%2Bfor%2Befficient%2Bgenome%2Bediting.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1755%26rft_id%253Dinfo%253Apmid%252F21179091%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1755&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21179091&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287023000021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.33" data-doi="10.1242/bio.20136676"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nakajima</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yaoita</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparison of TALEN scaffolds in <em>Xenopus tropicalis</em>.</span> <abbr class="cit-jnl-abbrev">Biol. Open</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1364</span>–<span class="cit-lpage">1370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BOpen%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fbio.20136676%26rft_id%253Dinfo%253Apmid%252F24285709%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiYmlvbG9wZW4iO3M6NToicmVzaWQiO3M6OToiMi8xMi8xMzY0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.34" data-doi="10.1093/nar/26.20.4597"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oleykowski</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bronson Mullins</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godwin</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeung</span>  <span class="cit-name-given-names">A. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Mutation detection using a novel plant endonuclease.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">4597</span>–<span class="cit-lpage">4602</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F26.20.4597%26rft_id%253Dinfo%253Apmid%252F9753726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8yMC80NTk3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.35" data-doi="10.1104/pp.102.017384"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Welham</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pike</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A TILLING reverse genetics tool and a web-accessible collection of mutants of the legume <em>Lotus japonicus</em>.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">866</span>–<span class="cit-lpage">871</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DPerry%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D131%26rft.issue%253D3%26rft.spage%253D866%26rft.epage%253D871%26rft.atitle%253DA%2BTILLING%2BReverse%2BGenetics%2BTool%2Band%2Ba%2BWeb-Accessible%2BCollection%2Bof%2BMutants%2Bof%2Bthe%2BLegume%2BLotus%2Bjaponicus%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.102.017384%26rft_id%253Dinfo%253Apmid%252F12644638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMzEvMy84NjYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvNDA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shandilya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alessio</span>  <span class="cit-name-given-names">J. M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connor</span>  <span class="cit-name-given-names">K. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durocher</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mutation detection using Surveyor nuclease.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">702</span>–<span class="cit-lpage">707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DQiu%26rft.auinit1%253DP.%26rft.volume%253D36%26rft.issue%253D4%26rft.spage%253D702%26rft.epage%253D707%26rft.atitle%253DMutation%2Bdetection%2Busing%2BSurveyor%2Bnuclease.%26rft_id%253Dinfo%253Apmid%252F15088388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15088388&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220798400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.37" data-doi="10.1038/gt.2012.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Rensburg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyer</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">X.-Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Denisenko</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Chromatin structure of two genomic sites for targeted transgene integration in induced pluripotent stem cells and hematopoietic stem cells.</span> <abbr class="cit-jnl-abbrev">Gene Ther.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%2BTher.%26rft.volume%253D20%26rft.spage%253D201%26rft.atitle%253DCHROMATIN%2BSTRUCTURE%2BOF%2BTWO%2BGENOMIC%2BSITES%2BFOR%2BTARGETED%2BTRANSGENE%2BINTEGRATION%2BIN%2BINDUCED%2BPLURIPOTENT%2BSTEM%2BCELLS%2BAND%2BHEMATOPOIETIC%2BSTEM%2BCELLS%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fgt.2012.25%26rft_id%253Dinfo%253Apmid%252F22436965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/gt.2012.25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22436965&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000315074400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.38" data-doi="10.1186/1472-6750-14-69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sakurai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamiyoshi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shindo</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A single blastocyst assay optimized for detecting CRISPR/Cas9 system-induced indel mutations in mice.</span> <abbr class="cit-jnl-abbrev">BMC Biotechnol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBiotechnol.%26rft.volume%253D14%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1472-6750-14-69%26rft_id%253Dinfo%253Apmid%252F25042988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1472-6750-14-69&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25042988&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.39" data-doi="10.1038/nprot.2011.431"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanjana</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cunniff</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A transcription activator-like effector toolbox for genome engineering.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DSanjana%26rft.auinit1%253DN.%2BE.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D171%26rft.epage%253D192%26rft.atitle%253DA%2Btranscription%2Bactivator-like%2Beffector%2Btoolbox%2Bfor%2Bgenome%2Bengineering.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2011.431%26rft_id%253Dinfo%253Apmid%252F22222791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2011.431&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22222791&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.40" data-doi="10.1038/nbt1043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slade</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuerstenberg</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loeffler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steine</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Facciotti</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A reverse genetic, nontransgenic approach to wheat crop improvement by TILLING.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">75</span>–<span class="cit-lpage">81</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DSlade%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D23%26rft.issue%253D1%26rft.spage%253D75%26rft.epage%253D81%26rft.atitle%253DA%2Breverse%2Bgenetic%252C%2Bnontransgenic%2Bapproach%2Bto%2Bwheat%2Bcrop%2Bimprovement%2Bby%2BTILLING.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1043%26rft_id%253Dinfo%253Apmid%252F15580263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15580263&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226195700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.41" data-doi="10.1242/bio.20133855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">K.-I. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Isoyama</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kashiwagi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakuma</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ochiai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">High efficiency TALENs enable F0 functional analysis by targeted gene disruption in <em>Xenopus laevis</em> embryos.</span> <abbr class="cit-jnl-abbrev">Biol. Open</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">448</span>–<span class="cit-lpage">452</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BOpen%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fbio.20133855%26rft_id%253Dinfo%253Apmid%252F23789092%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiYmlvbG9wZW4iO3M6NToicmVzaWQiO3M6NzoiMi81LzQ0OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy80MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.42" data-doi="10.1016/S1050-3862(98)00029-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deeble</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Enzymatic methods for mutation scanning.</span> <abbr class="cit-jnl-abbrev">Genet. Anal. Biomol. Eng.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BAnal.%2BBiomol.%2BEng.%26rft.volume%253D14%26rft.spage%253D181%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1050-3862%252898%252900029-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1050-3862(98)00029-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.43" data-doi="10.1016/j.tig.2014.01.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Terns</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terns</span>  <span class="cit-name-given-names">M. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">CRISPR-based technologies: prokaryotic defense weapons repurposed.</span> <abbr class="cit-jnl-abbrev">Trends Genet. TIG</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%2BTIG%26rft.volume%253D30%26rft.spage%253D111%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2014.01.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2014.01.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000333225800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.44" data-doi="10.1038/nbt.1940"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tesson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usal</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ménoret</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leung</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niles</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Knockout rats generated by embryo microinjection of TALENs.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">696</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DTesson%26rft.auinit1%253DL.%26rft.volume%253D29%26rft.issue%253D8%26rft.spage%253D695%26rft.epage%253D696%26rft.atitle%253DKnockout%2Brats%2Bgenerated%2Bby%2Bembryo%2Bmicroinjection%2Bof%2BTALENs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1940%26rft_id%253Dinfo%253Apmid%252F21822240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1940&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21822240&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.45" data-doi="10.1093/nar/gkh599"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Till</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burtner</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comai</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Mismatch cleavage by single-strand specific nucleases.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">2632</span>–<span class="cit-lpage">2641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DTill%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D32%26rft.issue%253D8%26rft.spage%253D2632%26rft.epage%253D2641%26rft.atitle%253DMismatch%2Bcleavage%2Bby%2Bsingle-strand%2Bspecific%2Bnucleases%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh599%26rft_id%253Dinfo%253Apmid%252F15141034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjMyLzgvMjYzMiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy80MDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.46" data-doi="10.1111/j.1365-313X.2007.03201.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Triques</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sturbois</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gallais</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalmais</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chauvin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Characterization of Arabidopsis thaliana mismatch specific endonucleases: application to mutation discovery by TILLING in pea.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1116</span>–<span class="cit-lpage">1125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DTriques%26rft.auinit1%253DK.%26rft.volume%253D51%26rft.issue%253D6%26rft.spage%253D1116%26rft.epage%253D1125%26rft.atitle%253DCharacterization%2Bof%2BArabidopsis%2Bthaliana%2Bmismatch%2Bspecific%2Bendonucleases%253A%2Bapplication%2Bto%2Bmutation%2Bdiscovery%2Bby%2BTILLING%2Bin%2Bpea.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2007.03201.x%26rft_id%253Dinfo%253Apmid%252F17651368%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2007.03201.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17651368&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249424200014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.47" data-doi="10.1186/1471-2199-9-42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Triques</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piednoir</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalmais</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmidt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Le Signor</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mutation detection using ENDO1: application to disease diagnostics in humans and TILLING and Eco-TILLING in plants.</span> <abbr class="cit-jnl-abbrev">BMC Mol. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bmolecular%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BMol%2BBiol%26rft.aulast%253DTriques%26rft.auinit1%253DK.%26rft.volume%253D9%26rft.spage%253D42%26rft.epage%253D42%26rft.atitle%253DMutation%2Bdetection%2Busing%2BENDO1%253A%2Bapplication%2Bto%2Bdisease%2Bdiagnostics%2Bin%2Bhumans%2Band%2BTILLING%2Band%2BEco-TILLING%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2199-9-42%26rft_id%253Dinfo%253Apmid%252F18433472%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2199-9-42&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18433472&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.48" data-doi="10.1002/elps.200700729"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsuji</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niida</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Development of a simple and highly sensitive mutation screening system by enzyme mismatch cleavage with optimized conditions for standard laboratories.</span> <abbr class="cit-jnl-abbrev">Electrophoresis</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1473</span>–<span class="cit-lpage">1483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DElectrophoresis%26rft.stitle%253DElectrophoresis%26rft.aulast%253DTsuji%26rft.auinit1%253DT.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D1473%26rft.epage%253D1483%26rft.atitle%253DDevelopment%2Bof%2Ba%2Bsimple%2Band%2Bhighly%2Bsensitive%2Bmutation%2Bscreening%2Bsystem%2Bby%2Benzyme%2Bmismatch%2Bcleavage%2Bwith%2Boptimized%2Bconditions%2Bfor%2Bstandard%2Blaboratories.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Felps.200700729%26rft_id%253Dinfo%253Apmid%252F18300207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/elps.200700729&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18300207&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.49" data-doi="10.2353/jmoldx.2009.080144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voskarides</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deltas</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Screening for mutations in kidney-related genes using SURVEYOR nuclease for cleavage at heteroduplex mismatches.</span> <abbr class="cit-jnl-abbrev">J. Mol. Diagn.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">311</span>–<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BDiagn.%26rft_id%253Dinfo%253Adoi%252F10.2353%252Fjmoldx.2009.080144%26rft_id%253Dinfo%253Apmid%252F19525337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2353/jmoldx.2009.080144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19525337&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.50" data-doi="10.1093/nar/23.19.3944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wagner</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Debbie</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radman</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Mutation detection using immobilized mismatch binding protein (MutS).</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">3944</span>–<span class="cit-lpage">3948</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DWagner%26rft.auinit1%253DR.%26rft.volume%253D23%26rft.issue%253D19%26rft.spage%253D3944%26rft.epage%253D3948%26rft.atitle%253DMutation%2Bdetection%2Busing%2Bimmobilized%2Bmismatch%2Bbinding%2Bprotein%2B%2528MutS%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F23.19.3944%26rft_id%253Dinfo%253Apmid%252F7479040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyMy8xOS8zOTQ0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.51" data-doi="10.1101/gr.1725103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wienholds</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Eeden</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kosters</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mudde</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plasterk</span>  <span class="cit-name-given-names">R. H. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Efficient target-selected mutagenesis in zebrafish.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">2700</span>–<span class="cit-lpage">2707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.1725103%26rft_id%253Dinfo%253Apmid%252F14613981%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMy8xMi8yNzAwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wen</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kodali</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oleykowski</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Purification, cloning, and characterization of the CEL I nuclease.</span> <abbr class="cit-jnl-abbrev">Biochemistry (Mosc.)</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">3533</span>–<span class="cit-lpage">3541</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemistry%2B%2528Mosc.%2529%26rft.volume%253D39%26rft.spage%253D3533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.53" data-doi="10.2144/05385RV01"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yeung</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hattangadi</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blakesley</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Enzymatic mutation detection technologies.</span> <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">749</span>–<span class="cit-lpage">758</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioTechniques%26rft.stitle%253DBiotechniques%26rft.aulast%253DYeung%26rft.auinit1%253DA.%2BT.%26rft.volume%253D38%26rft.issue%253D5%26rft.spage%253D749%26rft.epage%253D758%26rft.atitle%253DEnzymatic%2Bmutation%2Bdetection%2Btechnologies.%26rft_id%253Dinfo%253Adoi%252F10.2144%252F05385RV01%26rft_id%253Dinfo%253Apmid%252F15948293%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/05385RV01&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15948293&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229057100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.54" data-doi="10.1007/s10048-009-0200-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ymlahi-Ouazzani</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bronchain</span>  <span class="cit-name-given-names">O. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paillard</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballagny</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chesneau</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Reduced levels of survival motor neuron protein leads to aberrant motoneuron growth in a <em>Xenopus</em> model of muscular atrophy.</span> <abbr class="cit-jnl-abbrev">Neurogenetics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeurogenetics%26rft.stitle%253DNeurogenetics%26rft.aulast%253DYmlahi-Ouazzani%26rft.auinit1%253DQ.%26rft.volume%253D11%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D40%26rft.atitle%253DReduced%2Blevels%2Bof%2Bsurvival%2Bmotor%2Bneuron%2Bprotein%2Bleads%2Bto%2Baberrant%2Bmotoneuron%2Bgrowth%2Bin%2Ba%2BXenopus%2Bmodel%2Bof%2Bmuscular%2Batrophy.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10048-009-0200-6%26rft_id%253Dinfo%253Apmid%252F19517146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10048-009-0200-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19517146&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F407.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273685200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.55" data-doi="10.1073/pnas.92.1.87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Youil</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kemper</span>  <span class="cit-name-given-names">B. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotton</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Screening for mutations by enzyme mismatch cleavage with T4 endonuclease VII.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DYouil%26rft.auinit1%253DR.%26rft.volume%253D92%26rft.issue%253D1%26rft.spage%253D87%26rft.epage%253D91%26rft.atitle%253DScreening%2Bfor%2Bmutations%2Bby%2Benzyme%2Bmismatch%2Bcleavage%2Bwith%2BT4%2Bendonuclease%2BVII.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.92.1.87%26rft_id%253Dinfo%253Apmid%252F7816853%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo3OiI5Mi8xLzg3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.407.56" data-doi="10.1073/pnas.1102030108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherone</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doyon</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ankoudinova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faraji</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Efficient targeted gene disruption in the soma and germ line of the frog <em>Xenopus tropicalis</em> using engineered zinc-finger nucleases.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">7052</span>–<span class="cit-lpage">7057</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1102030108%26rft_id%253Dinfo%253Apmid%252F21471457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE3LzcwNTIiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvNDA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/3/407.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/3/399" title="Genetic Markers for Western Corn Rootworm Resistance to Bt Toxin" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/3/417" title="A Genome-Wide Association Study for Nutritional Indices in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/3"><span>Volume 5 Issue 3, March 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/3.cover-source.jpg?itok=g434JgGA" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/3/407" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zn5T98tXWDD0zhAqD8iIj1hnnMlX1nlEsClLaOU3jO0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/3/407" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/3/407" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/3/407" class="active" data-icon-position="" data-hide-link-title="0">Comparison of T7E1 and Surveyor Mismatch Cleavage Assays to Detect Mutations Triggered by Engineered Nucleases</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459794" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-oKdm0aPDm-m6Co8JMAjaXg_37rlcg5ZYIA9vUjR9opk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/3/407.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/3/407" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-DEPLp461eKi2zZ4NnQM-HpBC-k9PBlPMCTLOop_eXLM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/3/407" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/3/407.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459794" id="node459794--2" data-pisa="ggg;5/3/407" data-pisa-master="ggg;g3.114.015834" data-apath="/ggg/5/3/407.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison of T7E1 and Surveyor Mismatch Cleavage Assays to Detect Mutations Triggered by Engineered Nucleases</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Léna Vouillot</span>, <span class="highwire-citation-author" data-delta="1">Aurore Thélie</span> and <span class="highwire-citation-author" data-delta="2">Nicolas Pollet</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">407-415;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015834 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459794--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Léna Vouillot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Systems and Synthetic Biology, CNRS, Université d’Evry Val d’Essonne, Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L%C3%A9na%2BVouillot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vouillot%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AL%25C3%25A9na%2BVouillot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aurore Thélie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Systems and Synthetic Biology, CNRS, Université d’Evry Val d’Essonne, Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aurore%2BTh%C3%A9lie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Th%C3%A9lie%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAurore%2BTh%25C3%25A9lie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Nicolas Pollet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Systems and Synthetic Biology, CNRS, Université d’Evry Val d’Essonne, Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BPollet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pollet%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicolas%2BPollet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Nicolas.Pollet@issb.genopole.fr" class="" data-icon-position="" data-hide-link-title="0">Nicolas.Pollet@issb.genopole.fr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/3/407" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/3/407&amp;title=Comparison%20of%20T7E1%20and%20Surveyor%20Mismatch%20Cleavage%20Assays%20to%20Detect%20Mutations%20Triggered%20by%20Engineered%20Nucleases" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/3/407&amp;title=Comparison%20of%20T7E1%20and%20Surveyor%20Mismatch%20Cleavage%20Assays%20to%20Detect%20Mutations%20Triggered%20by%20Engineered%20Nucleases" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/3/407&amp;title=Comparison%20of%20T7E1%20and%20Surveyor%20Mismatch%20Cleavage%20Assays%20to%20Detect%20Mutations%20Triggered%20by%20Engineered%20Nucleases" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/3/407&amp;text=Comparison%20of%20T7E1%20and%20Surveyor%20Mismatch%20Cleavage%20Assays%20to%20Detect%20Mutations%20Triggered%20by%20Engineered%20Nucleases" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/3/407&amp;title=Comparison%20of%20T7E1%20and%20Surveyor%20Mismatch%20Cleavage%20Assays%20to%20Detect%20Mutations%20Triggered%20by%20Engineered%20Nucleases" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/3/407&amp;t=Comparison%20of%20T7E1%20and%20Surveyor%20Mismatch%20Cleavage%20Assays%20to%20Detect%20Mutations%20Triggered%20by%20Engineered%20Nucleases" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/3/407&amp;title=Comparison%20of%20T7E1%20and%20Surveyor%20Mismatch%20Cleavage%20Assays%20to%20Detect%20Mutations%20Triggered%20by%20Engineered%20Nucleases" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/3/407&amp;title=Comparison%20of%20T7E1%20and%20Surveyor%20Mismatch%20Cleavage%20Assays%20to%20Detect%20Mutations%20Triggered%20by%20Engineered%20Nucleases" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459794/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459794" id="node459794--3" data-pisa="ggg;5/3/407" data-pisa-master="ggg;g3.114.015834" data-apath="/ggg/5/3/407.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Comparison of T7E1 and Surveyor Mismatch Cleavage Assays to Detect Mutations Triggered by Engineered Nucleases</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Léna Vouillot</span>, <span class="highwire-citation-author" data-delta="1">Aurore Thélie</span> and <span class="highwire-citation-author" data-delta="2">Nicolas Pollet</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">407-415;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015834 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459794--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Léna Vouillot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Systems and Synthetic Biology, CNRS, Université d’Evry Val d’Essonne, Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L%C3%A9na%2BVouillot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vouillot%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AL%25C3%25A9na%2BVouillot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aurore Thélie </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Systems and Synthetic Biology, CNRS, Université d’Evry Val d’Essonne, Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aurore%2BTh%C3%A9lie%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Th%C3%A9lie%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAurore%2BTh%25C3%25A9lie%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Nicolas Pollet </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Institute of Systems and Synthetic Biology, CNRS, Université d’Evry Val d’Essonne, Evry, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicolas%2BPollet%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pollet%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicolas%2BPollet%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Nicolas.Pollet@issb.genopole.fr" class="" data-icon-position="" data-hide-link-title="0">Nicolas.Pollet@issb.genopole.fr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=407&amp;rft.epage=415&amp;rft.atitle=Comparison%20of%20T7E1%20and%20Surveyor%20Mismatch%20Cleavage%20Assays%20to%20Detect%20Mutations%20Triggered%20by%20Engineered%20Nucleases&amp;rft.volume=5&amp;rft.issue=3&amp;rft.date=2015-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Vouillot%2C+L%C3%A9na&amp;rft.au=Th%C3%A9lie%2C+Aurore&amp;rft.au=Pollet%2C+Nicolas"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459794/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459794/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459794/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459794/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459794/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459794/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459794/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459794/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459794/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459794/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459794/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459794/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/3/407" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/3/407.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/3/407.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/3/407.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/3/407.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
