<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/3/385" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/3/385.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/3/385.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/3/385.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/3/385/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/3/385.atom" />
<meta name="HW.pisa" content="ggg;5/3/385" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Rapid and Inexpensive Whole-Genome Genotyping-by-Sequencing for Crossover Localization and Fine-Scale Genetic Mapping" />
<meta name="DC.Identifier" content="10.1534/g3.114.016501" />
<meta name="DC.Date" content="2015-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Rowan et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The reshuffling of existing genetic variation during meiosis is important both during evolution and in breeding. The reassortment of genetic variants relies on the formation of crossovers (COs) between homologous chromosomes. The pattern of genome-wide CO distributions can be rapidly and precisely established by the short-read sequencing of individuals from F2 populations, which in turn are useful for quantitative trait locus (QTL) mapping. Although sequencing costs have decreased precipitously in recent years, the costs of library preparation for hundreds of individuals have remained high. To enable rapid and inexpensive CO detection and QTL mapping using low-coverage whole-genome sequencing of large mapping populations, we have developed a new method for library preparation along with Trained Individual GenomE Reconstruction, a probabilistic method for genotype and CO predictions for recombinant individuals. In an example case with hundreds of F2 individuals from two Arabidopsis thaliana accessions, we resolved most CO breakpoints to within 2 kb and reduced a major flowering time QTL to a 9-kb interval. In addition, an extended region of unusually low recombination revealed a 1.8-Mb inversion polymorphism on the long arm of chromosome 4. We observed no significant differences in the frequency and distribution of COs between F2 individuals with and without a functional copy of the DNA helicase gene RECQ4A . In summary, we present a new, cost-efficient method for large-scale, high-precision genotyping-by-sequencing." />
<meta name="DC.Contributor" content="Beth A. Rowan" />
<meta name="DC.Contributor" content="Vipul Patel" />
<meta name="DC.Contributor" content="Detlef Weigel" />
<meta name="DC.Contributor" content="Korbinian Schneeberger" />
<meta name="article:published_time" content="2015-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Rapid and Inexpensive Whole-Genome Genotyping-by-Sequencing for Crossover Localization and Fine-Scale Genetic Mapping" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The reshuffling of existing genetic variation during meiosis is important both during evolution and in breeding. The reassortment of genetic variants relies on the formation of crossovers (COs) between homologous chromosomes. The pattern of genome-wide CO distributions can be rapidly and precisely established by the short-read sequencing of individuals from F&lt;sub&gt;2&lt;/sub&gt; populations, which in turn are useful for quantitative trait locus (QTL) mapping. Although sequencing costs have decreased precipitously in recent years, the costs of library preparation for hundreds of individuals have remained high. To enable rapid and inexpensive CO detection and QTL mapping using low-coverage whole-genome sequencing of large mapping populations, we have developed a new method for library preparation along with Trained Individual GenomE Reconstruction, a probabilistic method for genotype and CO predictions for recombinant individuals. In an example case with hundreds of F&lt;sub&gt;2&lt;/sub&gt; individuals from two &lt;i&gt;Arabidopsis thaliana&lt;/i&gt; accessions, we resolved most CO breakpoints to within 2 kb and reduced a major flowering time QTL to a 9-kb interval. In addition, an extended region of unusually low recombination revealed a 1.8-Mb inversion polymorphism on the long arm of chromosome 4. We observed no significant differences in the frequency and distribution of COs between F&lt;sub&gt;2&lt;/sub&gt; individuals with and without a functional copy of the DNA helicase gene &lt;i&gt;RECQ4A&lt;/i&gt;. In summary, we present a new, cost-efficient method for large-scale, high-precision genotyping-by-sequencing.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/03/01" />
<meta name="citation_mjid" content="ggg;5/3/385" />
<meta name="citation_id" content="5/3/385" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/3/385" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/3/385.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/3/385.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/3/385.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.016501" />
<meta name="citation_pmid" content="25585881" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="385" />
<meta name="citation_lastpage" content="398" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Beth A. Rowan" />
<meta name="citation_author_institution" content="Department of Molecular Biology, Max Planck Institute for Developmental Biology, 72076 Tübingen, Germany" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7240-4889" />
<meta name="citation_author" content="Vipul Patel" />
<meta name="citation_author_institution" content="Department of Developmental Biology, Max Planck Institute for Plant Breeding Research, 50829 Cologne, Germany" />
<meta name="citation_author" content="Detlef Weigel" />
<meta name="citation_author_institution" content="Department of Molecular Biology, Max Planck Institute for Developmental Biology, 72076 Tübingen, Germany" />
<meta name="citation_author_email" content="weigel@weigelworld.org" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2114-7963" />
<meta name="citation_author" content="Korbinian Schneeberger" />
<meta name="citation_author_institution" content="Department of Developmental Biology, Max Planck Institute for Plant Breeding Research, 50829 Cologne, Germany" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-5512-0443" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Alonso-Blanco;citation_author=S. E. El-Assal;citation_author=G. Coupland;citation_author=M. Koornneef;citation_title=Analysis of Natural Allelic Variation at Flowering Time Loci in the Landsberg erecta and Cape Verde Islands Ecotypes of Arabidopsis thaliana;citation_pages=749-764;citation_volume=149;citation_year=1998;citation_issue=2;citation_pmid=9611189" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=A. E. Anastasio;citation_author=A. Platt;citation_author=M. Horton;citation_author=E. Grotewold;citation_author=R. Scholl;citation_title=Source verification of mis-identified Arabidopsis thaliana accessions.;citation_pages=554-566;citation_volume=67;citation_year=2011;citation_issue=3;citation_pmid=21481029;citation_doi=10.1111/j.1365-313X.2011.04606.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. K. Anderson;citation_author=K. D. Hooker;citation_author=S. M. Stack;citation_title=The Distribution of Early Recombination Nodules on Zygotene Bivalents From Plants;citation_pages=1259-1269;citation_volume=159;citation_year=2001;citation_issue=3;citation_pmid=11729167" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. K. Anderson;citation_author=A. Reeves;citation_author=L. M. Webb;citation_author=T. Ashley;citation_title=Distribution of Crossing Over on Mouse Synaptonemal Complexes Using Immunofluorescent Localization of MLH1 Protein;citation_pages=1569-1579;citation_volume=151;citation_year=1999;citation_issue=4;citation_pmid=10101178" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=L. K. Anderson;citation_author=G. G. Doyle;citation_author=B. Brigham;citation_author=J. Carter;citation_author=K. D. Hooker;citation_title=High-Resolution Crossover Maps for Each Bivalent of Zea mays Using Recombination Nodules;citation_pages=849-865;citation_volume=165;citation_year=2003;citation_issue=2;citation_pmid=14573493" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. Andolfatto;citation_author=D. Davison;citation_author=D. Erezyilmaz;citation_author=T. T. Hu;citation_author=J. Mast;citation_title=Multiplexed shotgun genotyping for rapid and efficient genetic mapping.;citation_volume=21;citation_year=2011;citation_pmid=21233398;citation_doi=10.1101/gr.115402.110" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=M. J. Aukerman;citation_author=M. Hirschfeld;citation_author=L. Wester;citation_author=M. Weaver;citation_author=T. Clack;citation_title=A Deletion in the PHYD Gene of the Arabidopsis Wassilewskija Ecotype Defines a Role for Phytochrome D in Red/Far-Red Light Sensing;citation_pages=1317-1326;citation_volume=9;citation_year=1997;citation_issue=8;citation_pmid=9286109;citation_doi=10.1105/tpc.9.8.1317" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=M. B. Bagherieh-Najjar;citation_author=O. M. De Vries;citation_author=J. Hille;citation_author=P. P. Dijkwel;citation_title=Arabidopsis RecQI4A suppresses homologous recombination and modulates DNA damage responses.;citation_pages=789-798;citation_volume=43;citation_year=2005;citation_issue=6;citation_pmid=16146519;citation_doi=10.1111/j.1365-313X.2005.02501.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=A. Barakate;citation_author=J. D. Higgins;citation_author=S. Vivera;citation_author=J. Stephens;citation_author=R. M. Perry;citation_title=The synaptonemal complex protein ZYP1 is required for imposition of meiotic crossovers in barley;citation_volume=26;citation_year=2014;citation_pmid=24563202;citation_doi=10.1105/tpc.113.121269" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=N. H. Barton;citation_author=B. Charlesworth;citation_title=Why Sex and Recombination?;citation_pages=1986-1990;citation_volume=281;citation_year=1998;citation_issue=5385;citation_pmid=9748151;citation_doi=10.1126/science.281.5385.1986" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=F. Baudat;citation_author=Y. Imai;citation_author=B. De Massy;citation_title=Meiotic recombination in mammals: localization and regulation.;citation_pages=794-806;citation_volume=14;citation_year=2013;citation_pmid=24136506;citation_doi=10.1038/nrg3573" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=B. L. Browning;citation_author=S. R. Browning;citation_title=A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.;citation_pages=210-223;citation_volume=84;citation_year=2009;citation_issue=2;citation_pmid=19200528;citation_doi=10.1016/j.ajhg.2009.01.005" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=C. Buhler;citation_author=V. Borde;citation_author=M. Lichten;citation_title=Mapping meiotic single-strand DNA reveals a new landscape of DNA double-strand breaks in Saccharomyces cerevisiae.;citation_pages=e324-e324;citation_volume=5;citation_year=2007;citation_issue=12;citation_pmid=18076285;citation_doi=10.1371/journal.pbio.0050324" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Cao;citation_author=K. Schneeberger;citation_author=S. Ossowski;citation_author=T. Gunther;citation_author=S. Bender;citation_title=Whole-genome sequencing of multiple Arabidopsis thaliana populations.;citation_pages=956-963;citation_volume=43;citation_year=2011;citation_issue=10;citation_pmid=21874002;citation_doi=10.1038/ng.911" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=L. Chelysheva;citation_author=G. Gendrot;citation_author=D. Vezon;citation_author=M. P. Doutriaux;citation_author=R. Mercier;citation_title=Zip4/Spo22 is required for class I CO formation but not for synapsis completion in Arabidopsis thaliana.;citation_pages=e83-e83;citation_volume=3;citation_year=2007;citation_issue=5;citation_pmid=17530928;citation_doi=10.1371/journal.pgen.0030083" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=K. Choi;citation_author=X. Zhao;citation_author=K. A. Kelly;citation_author=O. Venn;citation_author=J. D. Higgins;citation_title=Arabidopsis meiotic crossover hot spots overlap with H2A.Z nucleosomes at gene promoters.;citation_pages=1327-1336;citation_volume=45;citation_year=2013;citation_pmid=24056716;citation_doi=10.1038/ng.2766" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. P. Copenhaver;citation_author=K. Nickel;citation_author=T. Kuromori;citation_author=M. I. Benito;citation_author=S. Kaul;citation_title=Genetic Definition and Sequence Analysis of Arabidopsis Centromeres;citation_pages=2468-2474;citation_volume=286;citation_year=1999;citation_issue=5449;citation_pmid=10617454;citation_doi=10.1126/science.286.5449.2468" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=A. De Muyt;citation_author=L. Jessop;citation_author=E. Kolar;citation_author=A. Sourirajan;citation_author=J. Chen;citation_title=BLM helicase ortholog Sgs1 is a central regulator of meiotic recombination intermediate metabolism.;citation_pages=43-53;citation_volume=46;citation_year=2012;citation_issue=1;citation_pmid=22500736;citation_doi=10.1016/j.molcel.2012.02.020" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Drouaud;citation_author=R. Mercier;citation_author=L. Chelysheva;citation_author=A. Berard;citation_author=M. Falque;citation_title=Sex-specific crossover distributions and variations in interference level along Arabidopsis thaliana chromosome 4.;citation_pages=e106-e106;citation_volume=3;citation_year=2007;citation_issue=6;citation_pmid=17604455;citation_doi=10.1371/journal.pgen.0030106" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=M. E. El-Lithy;citation_author=E. J. Clerkx;citation_author=G. J. Ruys;citation_author=M. Koornneef;citation_author=D. Vreugdenhil;citation_title=Quantitative trait locus analysis of growth-related traits in a new Arabidopsis recombinant inbred population.;citation_volume=135;citation_year=2004;citation_pmid=15122039;citation_doi=10.1104/pp.103.036822" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. E. El-Lithy;citation_author=L. Bentsink;citation_author=C. J. Hanhart;citation_author=G. J. Ruys;citation_author=D. Rovito;citation_title=New Arabidopsis recombinant inbred line populations genotyped using SNPWave and their use for mapping flowering-time quantitative trait loci.;citation_volume=172;citation_year=2006;citation_pmid=16361234;citation_doi=10.1534/genetics.105.050617" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. E. Francis;citation_author=S. Y. Lam;citation_author=B. D. Harrison;citation_author=A. L. Bey;citation_author=L. E. Berchowitz;citation_title=Pollen tetrad-based visual assay for meiotic recombination in Arabidopsis.;citation_volume=104;citation_year=2007;citation_pmid=17360452;citation_doi=10.1073/pnas.0608936104" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. F. Fransz;citation_author=S. Armstrong;citation_author=J. H. De Jong;citation_author=L. D. Parnell;citation_author=C. Van Drunen;citation_title=Integrated cytogenetic map of chromosome arm 4S of A. thaliana: structural organization of heterochromatic knob and centromere region.;citation_pages=367-376;citation_volume=100;citation_year=2000;citation_issue=3;citation_pmid=10676818;citation_doi=10.1016/S0092-8674(00)80672-8" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=L. Giraut;citation_author=M. Falque;citation_author=J. Drouaud;citation_author=L. Pereira;citation_author=O. C. Martin;citation_title=Genome-wide crossover distribution in Arabidopsis thaliana meiosis reveals sex-specific patterns along chromosomes.;citation_pages=e1002354-e1002354;citation_volume=7;citation_year=2011;citation_issue=11;citation_pmid=22072983;citation_doi=10.1371/journal.pgen.1002354" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. C. Hall;citation_title=Chromosome segregation influenced by two alleles of the meiotic mutant c(3)G in Drosophila melanogaster.;citation_pages=367-400;citation_volume=71;citation_year=1972;citation_issue=3;citation_pmid=4624918" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=F. Hartung;citation_author=S. Suer;citation_author=T. Bergmann;citation_author=H. Puchta;citation_title=The role of AtMUS81 in DNA repair and its genetic interaction with the helicase AtRecQ4A.;citation_volume=34;citation_year=2006;citation_pmid=16945961;citation_doi=10.1093/nar/gkl576" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=F. Hartung;citation_author=S. Suer;citation_author=H. Puchta;citation_title=Two closely related RecQ helicases have antagonistic roles in homologous recombination and DNA repair in Arabidopsis thaliana.;citation_volume=104;citation_year=2007;citation_pmid=18000056;citation_doi=10.1073/pnas.0705998104" />
<meta name="citation_reference" content="citation_title=Genetic Recombination;citation_title=Exchange and chromosomal segregation in eucaryotes;citation_author=R. S. Hawley;citation_pages=497-527;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=I. R. Henderson;citation_title=Control of meiotic recombination frequency in plant genomes.;citation_pages=556-561;citation_volume=15;citation_year=2012;citation_issue=5;citation_pmid=23017241;citation_doi=10.1016/j.pbi.2012.09.002" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. D. Higgins;citation_author=S. J. Armstrong;citation_author=F. C. Franklin;citation_author=G. H. Jones;citation_title=The Arabidopsis MutS homolog AtMSH4 functions at an early step in recombination: evidence for two classes of recombination in Arabidopsis;citation_pages=2557-2570;citation_volume=18;citation_year=2004;citation_issue=20;citation_pmid=15489296;citation_doi=10.1101/gad.317504" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=J. D. Higgins;citation_author=M. Ferdous;citation_author=K. Osman;citation_author=F. C. Franklin;citation_title=The RecQ helicase AtRECQ4A is required to remove inter-chromosomal telomeric connections that arise during meiotic recombination in Arabidopsis.;citation_pages=492-502;citation_volume=65;citation_year=2011;citation_issue=3;citation_pmid=21265901;citation_doi=10.1111/j.1365-313X.2010.04438.x" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=J. K. Holloway;citation_author=M. A. Morelli;citation_author=P. L. Borst;citation_author=P. E. Cohen;citation_title=Mammalian BLM helicase is critical for integrating multiple pathways of meiotic recombination.;citation_volume=188;citation_year=2010;citation_pmid=20308424;citation_doi=10.1083/jcb.200909048" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. N. Howie;citation_author=P. Donnelly;citation_author=J. Marchini;citation_title=A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.;citation_pages=e1000529-e1000529;citation_volume=5;citation_year=2009;citation_issue=6;citation_pmid=19543373;citation_doi=10.1371/journal.pgen.1000529" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=X. Huang;citation_author=Q. Feng;citation_author=Q. Qian;citation_author=Q. Zhao;citation_author=L. Wang;citation_title=High-throughput genotyping by whole-genome resequencing.;citation_volume=19;citation_year=2009;citation_pmid=19420380;citation_doi=10.1101/gr.089516.108" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=G. V. James;citation_author=V. Patel;citation_author=K. J. Nordstrom;citation_author=J. R. Klasen;citation_author=P. A. Salome;citation_title=User guide for mapping-by-sequencing in Arabidopsis.;citation_pages=R61;citation_volume=14;citation_year=2013;citation_pmid=23773572;citation_doi=10.1186/gb-2013-14-6-r61" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=L. Jessop;citation_author=B. Rockmill;citation_author=G. S. Roeder;citation_author=M. Lichten;citation_title=Meiotic chromosome synapsis-promoting proteins antagonize the anti-crossover activity of sgs1.;citation_pages=e155-e155;citation_volume=2;citation_year=2006;citation_issue=9;citation_pmid=17002499;citation_doi=10.1371/journal.pgen.0020155" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Y. Ji;citation_author=C. Wu;citation_author=P. Liu;citation_author=J. Wang;citation_author=K. R. Coombes;citation_title=Applications of beta-mixture models in bioinformatics.;citation_volume=21;citation_year=2005;citation_pmid=15713737;citation_doi=10.1093/bioinformatics/bti318" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=A. Knoll;citation_author=H. Puchta;citation_title=The role of DNA helicases and their interaction partners in genome stability and meiotic recombination in plants.;citation_volume=62;citation_year=2011;citation_pmid=21081662;citation_doi=10.1093/jxb/erq357" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=A. Knoll;citation_author=S. Schropfer;citation_author=H. Puchta;citation_title=The RTR complex as caretaker of genome stability and its unique meiotic function in plants.;citation_pages=33;citation_volume=5;citation_year=2014;citation_pmid=24575106" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=A. Konieczny;citation_author=F. M. Ausubel;citation_title=A procedure for mapping Arabidopsis mutations using co-dominant ecotype-specific PCR-based markers.;citation_pages=403-410;citation_volume=4;citation_year=1993;citation_issue=2;citation_pmid=8106085;citation_doi=10.1046/j.1365-313X.1993.04020403.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. Lu;citation_author=X. Han;citation_author=J. Qi;citation_author=J. Yang;citation_author=A. J. Wijeratne;citation_title=Analysis of Arabidopsis genome-wide variations before and after meiosis and meiotic recombination by resequencing Landsberg erecta and all four products of a single meiosis.;citation_pages=508-518;citation_volume=22;citation_year=2011;citation_pmid=22106370" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. Mancera;citation_author=R. Bourgon;citation_author=A. Brozzi;citation_author=W. Huber;citation_author=L. M. Steinmetz;citation_title=High-resolution mapping of meiotic crossovers and non-crossovers in yeast.;citation_pages=479-485;citation_volume=454;citation_year=2008;citation_issue=7203;citation_pmid=18615017;citation_doi=10.1038/nature07135" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=E. R. Mardis;citation_title=The impact of next-generation sequencing technology on genetics.;citation_pages=133-141;citation_volume=24;citation_year=2008;citation_issue=3;citation_pmid=18262675;citation_doi=10.1016/j.tig.2007.12.007" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=C. Melamed-Bessudo;citation_author=E. Yehuda;citation_author=A. R. Stuitje;citation_author=A. A. Levy;citation_title=A new seed-based assay for meiotic recombination in Arabidopsis thaliana.;citation_pages=458-466;citation_volume=43;citation_year=2005;citation_issue=3;citation_pmid=16045480;citation_doi=10.1111/j.1365-313X.2005.02466.x" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. G. Mets;citation_author=B. J. Meyer;citation_title=Condensins regulate meiotic DNA break distribution, thus crossover frequency, by controlling chromosome structure.;citation_pages=73-86;citation_volume=139;citation_year=2009;citation_issue=1;citation_pmid=19781752;citation_doi=10.1016/j.cell.2009.07.035" />
<meta name="citation_reference" content="citation_journal_title=Chromosoma;citation_journal_abbrev=Chromosoma;citation_author=P. B. Moens;citation_author=D. J. Chen;citation_author=Z. Shen;citation_author=N. Kolas;citation_author=M. Tarsounas;citation_title=Rad51 immunocytology in rat and mouse spermatocytes and oocytes.;citation_pages=207-215;citation_volume=106;citation_year=1997;citation_issue=4;citation_pmid=9254722;citation_doi=10.1007/s004120050241" />
<meta name="citation_reference" content="citation_journal_title=Journal of Cell Science;citation_journal_abbrev=J. Cell Sci.;citation_author=P. B. Moens;citation_author=N. K. Kolas;citation_author=M. Tarsounas;citation_author=E. Marcon;citation_author=P. E. Cohen;citation_title=The time course and chromosomal localization of recombination-related proteins at meiosis in the mouse are compatible with models that can resolve the early DNA-DNA interactions without reciprocal recombination;citation_pages=1611-1622;citation_volume=115;citation_year=2002;citation_issue=8;citation_pmid=11950880" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. C. Nottke;citation_author=S. E. Beese-Sims;citation_author=L. F. Pantalena;citation_author=V. Reinke;citation_author=Y. Shi;citation_title=SPR-5 is a histone H3K4 demethylase with a role in meiotic double-strand break repair.;citation_volume=108;citation_year=2011;citation_pmid=21768382;citation_doi=10.1073/pnas.1102298108" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=C. M. O’Neill;citation_author=C. Morgan;citation_author=J. Kirby;citation_author=H. Tschoep;citation_author=P. X. Deng;citation_title=Six new recombinant inbred populations for the study of quantitative traits in Arabidopsis thaliana.;citation_pages=623-634;citation_volume=116;citation_year=2008;citation_issue=5;citation_pmid=18193187;citation_doi=10.1007/s00122-007-0696-9" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Ossowski;citation_author=K. Schneeberger;citation_author=R. M. Clark;citation_author=C. Lanz;citation_author=N. Warthmann;citation_title=Sequencing of natural strains of Arabidopsis thaliana with short reads.;citation_volume=18;citation_year=2008;citation_pmid=18818371;citation_doi=10.1101/gr.080200.108" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=D. I. Pacurar;citation_author=M. L. Pacurar;citation_author=N. Street;citation_author=J. D. Bussell;citation_author=T. I. Pop;citation_title=A collection of INDEL markers for map-based cloning in seven Arabidopsis accessions.;citation_volume=63;citation_year=2012;citation_pmid=22282537;citation_doi=10.1093/jxb/err422" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=K. Paigen;citation_author=P. Petkov;citation_title=Mammalian recombination hot spots: properties, control and evolution.;citation_pages=221-233;citation_volume=11;citation_year=2010;citation_issue=3;citation_pmid=20168297;citation_doi=10.1038/nrg2712" />
<meta name="citation_reference" content="citation_journal_title=Mammalian genome :  official journal of the International Mammalian Genome Society;citation_journal_abbrev=Mamm Genome;citation_author=F. Pardo-Manuel De Villena;citation_author=C. Sapienza;citation_title=Recombination is proportional to the number of chromosome arms in mammals.;citation_pages=318-322;citation_volume=12;citation_year=2001;citation_issue=4;citation_pmid=11309665;citation_doi=10.1007/s003350020005" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=N. Phadnis;citation_author=R. W. Hyppa;citation_author=G. R. Smith;citation_title=New and old ways to control meiotic recombination.;citation_pages=411-421;citation_volume=27;citation_year=2011;citation_issue=10;citation_pmid=21782271;citation_doi=10.1016/j.tig.2011.06.007" />
<meta name="citation_reference" content="citation_journal_abbrev=Genome Dyn;citation_author=D. W. Pryce;citation_author=R. J. McFarlane;citation_title=The meiotic recombination hotspots of Schizosaccharomyces pombe.;citation_pages=1-13;citation_volume=5;citation_year=2009;citation_pmid=18948703" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Qi;citation_author=Y. Chen;citation_author=G. P. Copenhaver;citation_author=H. Ma;citation_title=Detection of genomic variations and DNA polymorphisms and impact on analysis of meiotic recombination and genetic mapping.;citation_volume=111;citation_year=2014;citation_pmid=24958856;citation_doi=10.1073/pnas.1321897111" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=B. Rockmill;citation_author=J. C. Fung;citation_author=S. S. Branda;citation_author=G. S. Roeder;citation_title=The Sgs1 helicase regulates chromosome synapsis and meiotic crossing over.;citation_pages=1954-1962;citation_volume=13;citation_year=2003;citation_issue=22;citation_pmid=14614820;citation_doi=10.1016/j.cub.2003.10.059" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Rosu;citation_author=D. E. Libuda;citation_author=A. M. Villeneuve;citation_title=Robust Crossover Assurance and Regulated Interhomolog Access Maintain Meiotic Crossover Number;citation_pages=1286-1289;citation_volume=334;citation_year=2011;citation_issue=6060;citation_pmid=22144627;citation_doi=10.1126/science.1212424" />
<meta name="citation_reference" content="citation_journal_title=Heredity (Edinb);citation_author=P. A. Salomé;citation_author=K. Bomblies;citation_author=J. Fitz;citation_author=R. A. Laitinen;citation_author=N. Warthmann;citation_title=The recombination landscape in Arabidopsis thaliana F2 populations.;citation_pages=447-455;citation_volume=108;citation_year=2011;citation_pmid=22072068" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. A. Salomé;citation_author=K. Bomblies;citation_author=R. A. Laitinen;citation_author=L. Yant;citation_author=R. Mott;citation_title=Genetic architecture of flowering-time variation in Arabidopsis thaliana.;citation_volume=188;citation_year=2011;citation_pmid=21406681;citation_doi=10.1534/genetics.111.126607" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. Sanchez-Moran;citation_author=S. J. Armstrong;citation_author=J. L. Santos;citation_author=F. C. Franklin;citation_author=G. H. Jones;citation_title=Variation in Chiasma Frequency Among Eight Accessions of Arabidopsis thaliana;citation_pages=1415-1422;citation_volume=162;citation_year=2002;citation_issue=3;citation_pmid=12454084" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=E. Sanchez-Moran;citation_author=J. L. Santos;citation_author=G. H. Jones;citation_author=F. C. Franklin;citation_title=ASY1 mediates AtDMC1-dependent interhomolog recombination during meiosis in Arabidopsis;citation_pages=2220-2233;citation_volume=21;citation_year=2007;citation_issue=17;citation_pmid=17785529;citation_doi=10.1101/gad.439007" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=A. Sboner;citation_author=X. J. Mu;citation_author=D. Greenbaum;citation_author=R. K. Auerbach;citation_author=M. B. Gerstein;citation_title=The real cost of sequencing: higher than you think!;citation_pages=125-125;citation_volume=12;citation_year=2011;citation_issue=8;citation_pmid=21867570;citation_doi=10.1186/gb-2011-12-8-125" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=K. Schneeberger;citation_author=J. Hagmann;citation_author=S. Ossowski;citation_author=N. Warthmann;citation_author=S. Gesing;citation_title=Simultaneous alignment of short reads against multiple genomes.;citation_pages=R98-R98;citation_volume=10;citation_year=2009;citation_issue=9;citation_pmid=19761611;citation_doi=10.1186/gb-2009-10-9-r98" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Schropfer;citation_author=D. Kobbe;citation_author=F. Hartung;citation_author=A. Knoll;citation_author=H. Puchta;citation_title=Defining the roles of the N-terminal region and the helicase activity of RECQ4A in DNA repair and homologous recombination in Arabidopsis.;citation_volume=42;citation_year=2014;citation_pmid=24174542;citation_doi=10.1093/nar/gkt1004" />
<meta name="citation_reference" content="citation_journal_title=Proc Biol Sci;citation_author=J. Segura;citation_author=L. Ferretti;citation_author=S. Ramos-Onsins;citation_author=L. Capilla;citation_author=M. Farre;citation_title=Evolution of recombination in eutherian mammals: insights into mechanisms that affect recombination rates and crossover interference;citation_volume=280;citation_year=2013;citation_pmid=24068360;citation_doi=10.1098/rspb.2013.1945" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. Simon;citation_author=O. Loudet;citation_author=S. Durand;citation_author=A. Berard;citation_author=D. Brunel;citation_title=Quantitative Trait Loci Mapping in Five New Large Recombinant Inbred Line Populations of Arabidopsis thaliana Genotyped With Consensus Single-Nucleotide Polymorphism Markers;citation_pages=2253-2264;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430947;citation_doi=10.1534/genetics.107.083899" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=A. H. Sturtevant;citation_title=A Case of Rearrangement of Genes in Drosophila;citation_pages=235-237;citation_volume=7;citation_year=1921;citation_issue=8;citation_pmid=16576597;citation_doi=10.1073/pnas.7.8.235" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=M. Terasawa;citation_author=A. Shinohara;citation_author=Y. Hotta;citation_author=H. Ogawa;citation_author=T. Ogawa;citation_title=Localization of RecA-like recombination proteins on chromosomes of the lily at various meiotic stages.;citation_pages=925-934;citation_volume=9;citation_year=1995;citation_issue=8;citation_pmid=7774810;citation_doi=10.1101/gad.9.8.925" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Vignard;citation_author=T. Siwiec;citation_author=L. Chelysheva;citation_author=N. Vrielynck;citation_author=F. Gonord;citation_title=The interplay of RecA-related proteins and the MND1-HOP2 complex during meiosis in Arabidopsis thaliana.;citation_pages=1894-1906;citation_volume=3;citation_year=2007;citation_issue=10;citation_pmid=17937504;citation_doi=10.1371/journal.pgen.0030176" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=B. M. Weiner;citation_author=N. Kleckner;citation_title=Chromosome pairing via multiple interstitial interactions before and during meiosis in yeast.;citation_pages=977-991;citation_volume=77;citation_year=1994;citation_issue=7;citation_pmid=8020104;citation_doi=10.1016/0092-8674(94)90438-3" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Weinig;citation_author=M. C. Ungerer;citation_author=L. A. Dorn;citation_author=N. C. Kane;citation_author=Y. Toyonaga;citation_title=Novel Loci Control Variation in Reproductive Timing in Arabidopsis thaliana in Natural Environments;citation_pages=1875-1884;citation_volume=162;citation_year=2002;citation_issue=4;citation_pmid=12524356" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=E. Wijnker;citation_author=G. Velikkakam James;citation_author=J. Ding;citation_author=F. Becker;citation_author=J. R. Klasen;citation_title=The genomic landscape of meiotic crossovers and gene conversions in Arabidopsis thaliana.;citation_volume=2;citation_year=2013;citation_pmid=24347547;citation_doi=10.7554/eLife.01426" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Xie;citation_author=Q. Feng;citation_author=H. Yu;citation_author=X. Huang;citation_author=Q. Zhao;citation_title=Parent-independent genotyping for constructing an ultrahigh-density linkage map based on population sequencing.;citation_volume=107;citation_year=2010;citation_pmid=20498060;citation_doi=10.1073/pnas.1005931107" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Xu;citation_author=F. Zou;citation_author=T. J. Vision;citation_title=Improving quantitative trait loci mapping resolution in experimental crosses by the use of genotypically selected samples.;citation_volume=170;citation_year=2005;citation_pmid=15781710;citation_doi=10.1534/genetics.104.033746" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. Ye;citation_author=M. H. Schulz;citation_author=Q. Long;citation_author=R. Apweiler;citation_author=Z. Ning;citation_title=Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.;citation_volume=25;citation_year=2009;citation_pmid=19561018;citation_doi=10.1093/bioinformatics/btp394" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Rapid and Inexpensive Whole-Genome Genotyping-by-Sequencing for Crossover Localization and Fine-Scale Genetic Mapping" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3.cover-source.jpg" />
<meta name="twitter:description" content="The reshuffling of existing genetic variation during meiosis is important both during evolution and in breeding. The reassortment of genetic variants relies on the formation of crossovers (COs) between homologous chromosomes. The pattern of genome-wide CO distributions can be rapidly and precisely established by the short-read sequencing of individuals from F2 populations, which in turn are useful for quantitative trait locus (QTL) mapping. Although sequencing costs have decreased precipitously in recent years, the costs of library preparation for hundreds of individuals have remained high. To enable rapid and inexpensive CO detection and QTL mapping using low-coverage whole-genome sequencing of large mapping populations, we have developed a new method for library preparation along with Trained Individual GenomE Reconstruction, a probabilistic method for genotype and CO predictions for recombinant individuals. In an example case with hundreds of F2 individuals from two Arabidopsis thaliana accessions, we resolved most CO breakpoints to within 2 kb and reduced a major flowering time QTL to a 9-kb interval. In addition, an extended region of unusually low recombination revealed a 1.8-Mb inversion polymorphism on the long arm of chromosome 4. We observed no significant differences in the frequency and distribution of COs between F2 individuals with and without a functional copy of the DNA helicase gene RECQ4A . In summary, we present a new, cost-efficient method for large-scale, high-precision genotyping-by-sequencing." />
<meta name="og-title" property="og:title" content="Rapid and Inexpensive Whole-Genome Genotyping-by-Sequencing for Crossover Localization and Fine-Scale Genetic Mapping" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/3/385" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The reshuffling of existing genetic variation during meiosis is important both during evolution and in breeding. The reassortment of genetic variants relies on the formation of crossovers (COs) between homologous chromosomes. The pattern of genome-wide CO distributions can be rapidly and precisely established by the short-read sequencing of individuals from F2 populations, which in turn are useful for quantitative trait locus (QTL) mapping. Although sequencing costs have decreased precipitously in recent years, the costs of library preparation for hundreds of individuals have remained high. To enable rapid and inexpensive CO detection and QTL mapping using low-coverage whole-genome sequencing of large mapping populations, we have developed a new method for library preparation along with Trained Individual GenomE Reconstruction, a probabilistic method for genotype and CO predictions for recombinant individuals. In an example case with hundreds of F2 individuals from two Arabidopsis thaliana accessions, we resolved most CO breakpoints to within 2 kb and reduced a major flowering time QTL to a 9-kb interval. In addition, an extended region of unusually low recombination revealed a 1.8-Mb inversion polymorphism on the long arm of chromosome 4. We observed no significant differences in the frequency and distribution of COs between F2 individuals with and without a functional copy of the DNA helicase gene RECQ4A . In summary, we present a new, cost-efficient method for large-scale, high-precision genotyping-by-sequencing." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3.cover-source.jpg" />
    <title>Rapid and Inexpensive Whole-Genome Genotyping-by-Sequencing for Crossover Localization and Fine-Scale Genetic Mapping | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"mDrwHNVNQEXCFXBkZIe2xIv560PCop6cVgJN4BPrnIY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459792","apath":"\/ggg\/5\/3\/385.atom","pisa":"ggg;5\/3\/385","ac":{"\/ggg\/5\/3\/385.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/3\/385.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/3\/385"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/3\/385"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/3\/385":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ISFvkUfUnM+Wxt2jdsrL1zm5C+wIkdQu31oG8C\\\\\\\/cc\\\\\\\/PkHc3xAehCiBg1Gn7vyuPx74GFxQLvjWs14U5POUTLTf+jGpy+p3d2JcNVmBnJZ8KcVbOOGCTVvffN9XGBTginid07n8rGQlR4xCY8Bj7aoVsNKzl+CW1+NhNQYPEyO8XiZlNUAclIRwyejLUELQ5JQ3rYJOR4TfEpo2sfFEPABG2W0AMeeE31t2Vm8xAuAF+bXNXgOCpNKe\\\\\\\/KjiUVX4zwA6kmemeUreiVb9OSiM3P6KJrW8BZZaOG05RVviZK+zWNmfbl6QpfeJ5UUyZSqq3dkUQ63SKmoGSkGj4C1kZLxrTfKloKUoFu8GpHsgEBUJtP1P5nU9IJb\\\\\\\/49ONBkYQsmBknLECP9X5uTnzL\\\\\\\/L11RNgipwGyTqRyOqd9LzF3PvXCON0PAWrWYoiFIFGcW6+d1vxermLloMC25NIp3z8\\\\\\\/RXE2gE6Jf0rLynh3IG0aCaMRTayy9T8ju1JYLtBi79JUfJZcU8szu3M3FmptQQe2mR7\\\\\\\/pGxmW6FZdG6213\\\\\\\/dmM19KOByKAn9oQHwmvU9cFOmINl5uBLMewXLFVcY7n\\\\\\\/SdFLpgCfHiW1nVxldI5viZSGtK9bUhqezELbyv1OdN05NGC0g0S2hf7jA\\\\\\\/N3zgZJa8XZK0OEPaU4UOOqf0lLd7fbUj2LE09eo58Fb9\\\\\\\/\\\\\\\/\\\\\\\/ntTEApyRl6QXp8N8PRJBM+w1kvOnqXDy78N4hcKPxjr5WlX+VuvFUI+Rlw5vW0q+7hFJIDnFRLfu\\\\\\\/zR7aI1bkj4UCqW6Ek25JlK8MF+dlkk9K+5\\\\\\\/6lrUhSCuI6ssqMCADpznfQ0sNK9DGOmIkOAOchPXJt2aY4BAGUYEWR8wDduMRNJaiAMI3flvMOYkGNj3IRcg6s6DiZL0PIg7\\\\\\\/h1SBvfwYZXDu\\\\\\\/amFaFj+A7Mo9nrdYT6Znd2kd5nRpEV3r+acWdveRk+itnqmpyTFtiuPxeBH\\\\\\\/OUminpU\\\\\\\/ftZNp5Yab2XT\\\\\\\/w5v\\\\\\\/NZr1nTAUS\\\\\\\/gHMDmBOjV7fo8MUlwq8lpXod4SmAMty2HxEWKaI4JVTAMHdQujCQk47U72Khij4XxD0Kg7rO5vR6b0CshWqAGK42YCEyfWUKQuxysFSufLE56utcbxddv8YR4Zcvgxfczi+ahFM6+2I6x6vLC4ztXFOaLFqrsYMTbPrWk1XRYifTGJXHDoWJwoWHpDHYs8WDEPZ3JwLk+r4FJ96P2TP6Ov9hxYPN9aku3p3tvNqZeUWy0yFIrcmT5A7UkEsLFkCfPK4IMfwb\\\u0022,\\\u0022iv\\\u0022:\\\u0022R7w3u+psMGoSY6lFsiRqPA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227b9b344e5a460a6e78cdf807d884452cde0523aacec7970faa94806d84439ffa\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00229ro1xaixtmrJH3po\\\\\\\/BHHw4Mn+2AnzxqtcxlIcGu5hPSGrGj4GFyp9d2wpiUUzTEq08chu7ELGRH5gYLziJCkvkg88TtUHar1ZqmKJAjMpKaA8gOyLT27Cq5OSo+IowNMl468TzQYlT2eyikWcX\\\\\\\/pIZUxfZIzajGYcpw+b\\\\\\\/B\\\\\\\/i\\\\\\\/dc+jWRiCICBqLKvjnDF9lZh7hRE8+oatGhTi0CywtafMitRl1DxswggRtKcZNVMWdczUb19bJaIT8vjoao\\\\\\\/RTy9w9Y+CD9cA8bSOe82YVfFcL4GgS2uzjHUEbgXwzExuF26w5frarlT9rZCfDsPqqcCPAyALex4lfGhEpoRVyE\\\\\\\/m\\\\\\\/PoQD5kbvXUEGQtdgFqSWuNuNRjm4Nr\\\\\\\/LKj2ZO0QkT1aZOv1YgiApa0hqVvNfwS4gp+qpt+S1grlPknqwEWpNF81HSolW200WIIYxMox08M0WJ39JPYgY1E\\\\\\\/qPJEsU9ueQBQk+BREWKOJowr\\\\\\\/xupOS7B2iRbOyKhN4UW+20mWltjbvqcLiEnNCNsdjdmMubJxQVDR5h1GFtcXi2H1yJK1GFo8L91ktEo9\\\\\\\/qxb7cQhMqHvEnLsQDU4ODHancxwCRzvITVWhTxT5zM7mDxF0ath7JuThmqq4WNCZ18gmEpNTzR54uB\\\\\\\/J9qf9crwQvjenKXFuFtVWO9GX4qWuzFEtcR1SuydDrtNO2sz0HANy\\\\\\\/OqufIjCrmWsMH1MdfKeH5zyYB+DBbspbbynN8rfSoHElOaJ1Qq7JPC7DTBto1YXXP8jtYJDj\\\\\\\/QIAgyVpDkIziAY7UvyeXGviefmyo3SRo2u+vrfg36Ryskm0QVg6vDPxx2J405gebcK9PD1cRExfLmW\\\\\\\/LTJjqh0+CVFVN+pGxPvDZa8EvaqEcf7Ce36\\\\\\\/bWUgmUZdWcWyOaevkEHFcu3FHQyvBwxhaF0SZeOSm+b3SvLfxIPxCDkFjiLtjOyas+ZmeRN8v+lKwX86YmQcYkUEt4QZsWXKRFc70P5c1QqmhxpbmHD\\\\\\\/SWvBtS51tLnDdYBJ8Yiw4Ddmb0NwF3spp0erdtEg1AAfoFPzYPBpwmtvPPuPWBe2r0MjmOZioJZEjl7Kw018\\\\\\\/Wt63VPTRcHUXddC6ApVD65F0kdqWLxY9Wb597GonwVsXe6Ri\\\\\\\/a\\\\\\\/DrCPypj\\\\\\\/J4lHm4IQsRXeeMlzmpQ5Bl6kXqOFlv2MPfWSEjVaTKuzn28F3P6xfoREopmlGG\\\\\\\/V1AR61RYsc4itD\\\\\\\/NKh6cFCx3pg78WZHSJOvWQ0Ihk5LUSZv8IBxh8A4QgSum32WojKus\\\\\\\/b75MVTuacq062zSj\\\\\\\/LOyk6c5R4bybTKFBqVbqDYlVtI\\\\\\\/IUMwzJ0Dh5OyvXb9LuCaJSviFFOnEUuo6Y2h5QZGPlBdNvJ3jI4ft4q2Fy5wO37WUmuZH9dXuG1HwgTzASFgeKqJccZuqWxLeDfSqyOs\\\\\\\/F3FKDANGtkdoQVsbZW\\\\\\\/o\\\\\\\/GWpMaZLjIZcFXpoNa+bIfD9TSd8Sds81NdLxfUZ3dIezfoeevC7so0Tbqhs0Kq4YNJtWMHAvCBt7W9gOh6dGoCTYg3VsRIJYXPMZE1Ba+469BvCfLKArSAJKqF7l2lUQwrnICc6daeYh3YKElfnOg4JhSQwa15g9mDdtM3jzLTBdWGMN1eAIgc2apRe8+Oie9\\\\\\\/WvyuLxIV+FZjZZCYJUIOCwVTWVoKeOyf7ERgZN9qngeMVB8vXstI5NQ4jAyqvKZpBsq8ZHFl9jprQqgJCxspFfgRDR0FtPoxao38KOAPSS25VYaYfXMHd9OC1B03s2zj12FOZMSV6PnM+uQ5mbVlzbSfVMR\\\\\\\/ctuX\\\\\\\/Kko3PJumfsG0T6fAiP1eYqP5Uyi2OD9VxNmXUnzn4EDqi3FKMvJERQxqQMYGCAbJf4B1cUblqCBAEvjoRonPcIdP04kJlsY2wsButU2SSs0EVaaeVX9uTJ9hjb2sRGKZ7kObd04zMG7EzsfRAdaCeHHOmaxGVCde1oahFGVmTnUyjy12vvu4l6LJ5HNuwfwnwzDUoYUAdG0JB42zklQdA53zwSzAMo5eExobO0SIzQi\\\\\\\/XtFs3U3A6CM2PTe8xS5ynG3aGGW2xQtA2t7zixAXHGxVWVJM2mG4Ib32caWtHXwA0cZhMXS24ZnR2fgFRVtb9lBGAGlI5fgvhad3li+AJ2evXCjQefKhD7n8wGvIXT4e3lRxYrKkpDDMxR+4sdzJBUGAbeCc2wS8YKqCa1nD+E5tANjB1vRs5PdJ9UL9dH3tLUVelB\\\\\\\/+dWpZWQEDT3oY647bHhT7mzydS8LTjWrkvqtORAwVlZfU\\\\\\\/+8p4Drfmfdq7Ioxq00wTEn+V9FXn2QymhUWmDPaF60fbsvwt6sDrItx+GIBMRYKqCR0Kdgc\\\\\\\/ZCuSGLZaKl9vc2jHrn3J9t8IouqPzaoUTJ0u53Bn0wBYDocLSY9EyZK2iVDRjGkfH4ozY3YEeeBnZNAXGHvMCOiIQ7wKflYZcFk\\\\\\\/xxgWOzX76HeExMQrWhUuUCw3NU363Jyg4j8+\\\\\\\/RiURrv94NH7UQ\\\\\\\/VE+UAVcpZesQ48vgeSiE2sZD1p9K6bDNESkfNS4hcKLOTZcZljPa7HvzfvNGqLS0K0zNrudtPoiFFjdmC+8i5P6Yg2c+lQLY3d5opM1NwETgzmVH2NO7L5q9LhqOQruMmXH2kFGGW40ni1ygPPIaPf8QO8+y2Nbzj8kHPPz2KQowbhpYyiE\\\\\\\/ZEjWtuZuNaXO3SmoCfxioBg0w2S2M\\\\\\\/IqsHRxGKixaL7\\\\\\\/rf4kVJHSaN+gyeyjb8uPdCcnMiURgvK7tNGnSmrRKybJAahXSkmweGcC4SvpnTEmqDAf5grWIamw31vcF1BdH2HR9Dic+LqkfVYDb\\\\\\\/5H2J3fvSJMexGYTVmcChLmqQZdc0gTSLD69ChiTtYUUN4R7KPyao4K0Mh2gphYyWAkVYFJedD+Dc7Oh6xZOsvTo\\\\\\\/9A87E08ldiVAC+jNwh0ThcBtrW6\\\\\\\/Y9D8IEEJ7LT9wreV7VQJTAtUvAa94YNhwiKogdeuBGjvP9i4RhFSH2N1tjGhJv1Tkf\\\\\\\/D8tMEJb+euWdaqWf+arQlWLPmLwhCbxvCnfRNPZcztdHdLyDc1dUKgl2WB9uSSWWV6swT4KYTEuMjGIfadQnnshRH\\\\\\\/1ne6EwG\\\\\\\/VaXVZ0780xXuRLUIbGA9Z5K3KcA13\\\\\\\/JHV6Jqix\\\\\\\/dw6kAiM+PW8t4QO1mQaI+R4HH+0mcY\\\\\\\/6wVESBO3WnJ1sL9COQRnZvImifNbErY7hvALkUN06oM5tgjppde2P6GyX5D4wT8\\\\\\\/agwF8weTWXpWlEJhVyDG7\\\\\\\/M6Cm\\\\\\\/jtBcjQx4gJqBiKapETKny70suKfW6N1+h78nsQR8wy\\\\\\\/+jGgo9c4FQvjl7gKDAHM0s53cC0YtzxVYOlYiCizXJPLBCDw7+\\\\\\\/LZhWq7uL4aRfe7R6jhVD+RB0aDNqc0TfnX2Bn4jaQqqpdsRi3U4cP6EBsHuVSr5V11CjklRqEb9nQQ3FwbXvKOxvjo9sebKBrkUA+2BVTsMnAnK98pfKW2CPq\\\\\\\/X1SK3tB+KkQHntEF\\\\\\\/7zOKeqSXlZUwYiKOT1D3U1cqxsjO8kORsqymTNG6EnUppOzO0wQsx1Ox\\\\\\\/TWDm3KqlhK8oxS2j60QfC4Kl+z0pmmeZy+VVLz6tJUxsEh0FRdkY8Sb01a0xfnsFblf9Ol6YK3BJS6wZ18GzaNQ3xrUBI6HmM56bbtU20uXqFXucnF\\\\\\\/eOJ\\\\\\\/E2ug9HRnkiklFh46dO9RkADuxioZhXjS5F5HiwQ3JYqWhLLEQ\\\\\\\/XnXik4scPrAzY4XC3FLmJU\\\\\\\/KlGKDabBACZ6djVec7sglnj2utYqox6ait2JAQD4EHoR12vTOMg5WhUBwxYaNvrHsVvHhTG1ginjmvJQ3Fm7XxDiVUOP+uvp7Tlcj2bSitbmt+s3e6Xzx2WQwikbdh7AL5JXA26AU9LGds9LB8iLj1qHLfZ3FJKjfxb2GsTgwyur0Lgg4ddKICeePacKMsSk3ThtpVxg5FnkH\\\\\\\/rEKA7eSJ1Ed+ClJ8pxjacOhVZaXxacLLmf252V\\\\\\\/6UZIXz6GxIbABt7HBnBoZeQkzASU7eIQex6LeYgWfdWyiXEeYmDtbtFH0jzlnQyUypWpyiJClA6SQ84My5sOC1SbE00awLyeBdlZhVq4jyuV12+uX4YHElesHq1tbTeMM8R19N5os5rdd8UYa85foUf6SVOat3ITkKC6SFU7afXVlL0Iu0aOppJmji2aedMBQg96L39KJm0sc1gmfXx2P31Na2d0hJk5wnMwPv6\\\\\\\/blCIKZwIwL6m2JPZZQHMDu4KG1i11rHK+7u2jDNxbjYMGNYud82YuIvvtJU0MQiiUC0J\\\\\\\/P4F4AI4Lo2F8185HsYNTzaOSNms5ZX+O+5MK7F6ggiDcT3wnN3Mp7G2jEFvW5D93gW3n+Xl+Yl+Fjt5753wvzHkSDCr4vGKH9fJjXsisOT0+DZGHtnVurozmyxC9oSVllW5RuVBzyTpxONzoDzKOsEvWz9gVQdPUydFRFeG246VLJGlU8\\\\\\\/\\\\\\\/eWvOe+L\\\\\\\/7lch7NGWH\\\\\\\/jbTaBzeKuxietD3FE9qbPNA7T5DmyzHgDvh69dVgVwPUCkNyPZvBCMM5LJ4nOIMq2YhFigZoeGNabcnF4VYtoTeqENTQgcRJTVTl2079zyvS0feQQ8q3x7gkdnTOxs\\\\\\\/NJVYNX4n2RlkMXzLnsPrPO+Opvuh6RJTV9+oTE3EGsTazEMMeBZDoAPvS56tJdOHsFksw70UUSCMrrpxSoCXEM80mq5tAx+qzQ7thxMnMA3Dw1eVgthZO5KPumEYFiAYY9H4aWf0S2ER049AuDRnUr9x6jHTUE4WTd7le00I3wxBKPqzYG4gJ4d\\\\\\\/OXi6dPnc1JJANI4Q8zZjhQkg3lq6iQNLg5WMB6a0wSbFGydJCKvmbrhFVn0bIu3TkLf1hOxzJ7YrwYU1NGSUTn8en9apbQeRjBI8V7W\\\\\\\/Tn0UZRZwe+4kKgEkNK9prx38TYqmYCleC9ngs7wSC5WGcu1qOPTZxAkIv5Rf3gEZaZFodjoW+m1kpa4lMsNc71hE3xdiIjSeRGtvpwEdiDHVd60KuUwdhdXLFyYmDe0EnlSPQShMF5TietMVjJPDb1VHVxwYhY9WTKemqSrSZgJnLx33vE20Yg1A6rfKZwFhINNORQaI0toTgcMJSf79BpIO0Ym+ffIKwEPCfxIM0hmfzTzZAnr9a1TVz5T5ecdEGiZf7YtOoU8KjlnHc9Px6oHV4cb5OMm7djbTZwTPJ2PybUhVpWS7eBC2N4YXn0pU2\\\\\\\/joM9onvxg+NHlh7XEQLo6aK1+\\\\\\\/ilon3+p\\\\\\\/rt0NlzOSJHXuG95z9QVhBdfytqCSrE4IOm+tfK9B56TZJq8IwP0l8lH3gQjBBZvTn8b+A\\\\\\\/Npaao3q5cGfPMvhAtfIFcm3FR9lK5Y1LANQzwzTJAv4dPSkM8oJCu1nYV3Oo53qETwPGRokm+kes5pCEVpbD00hMHY2TrhM8TjQFibuOs1rdEgE6WQ+FGmP8E7K+t8dDSx6TtSzp3P7ULn4zmbhfzLUKTPwW0HGSDFwl9ujTjSmwrhFYfkvtK9TDDYpj23mwz6Ph4xhiq7QDnQG6LIWEr2XltKLaFIA42BYF3CuowgXUoKalafvOTjCjATK7m2w6eEs8gOtIzGFwY2ZEGVm4JII1Q9JNb\\\\\\\/2XJmZyzUPy2fYy\\\\\\\/XXY03db6eAMlQNkMj33UOi8B+Hw41AhRVw2JSbesG4plgpLqDvkZu5DqsyjFjHT7vsb10MsbG1+HSFPGZzEgMPLp8SvjJ1sHKsEPTiBfskJ6ivh\\\\\\\/vBnRxDwUrCvdxhhUjtRRCxp4IAUOuAW2GeQWbrR2wrltgOTNFGwq8\\\\\\\/9RPoUyVPPNnIeAZ17SnlD9mHh8fya2P8hjAY5BSPYDeXAKaqlCBTmJpDgrOQo9R9sYxRahmsv\\\\\\\/xwkL80Zfr9zcYSPp7swyrg2QE4Sg1+CpgvuBN2QmcdWrQPP4+V7bUoseKUkx14y0mfS2lmv7BXDFNA+52JDYJoUAo8GIcyq9EwrIzxW5EOL7aQsDtoznHYPA\\\\\\\/VHtTZJAXpvKPaTbqgg5ZNYaFIbRWAJUKROdO2CtIXNNcJBZm+uz4jIQsoynKMUF3GBVk9JwYcjk6Zxer3d4IWxRUerunV6Yo5KtM5DmgnUDHvJXRsKr28JEKdrTenP4nWXoScxRwt4OaVD8TX3PkuxIDM2ad0KDsL52bmw0G2CwmX9nr0bglmdPE9ZuNvGOzz6bD5XFxa+6clHfAaIpSzxZRMn02GTJyef5f5mbH0hrpEBu2hR8M4OgAa9Zii+FhABGODdxUU8NFyyo4HqZUbLz+7oiDOnfEYirdcvSo\\\\\\\/G1Mcgb3Ef983btfr2eUxvuNdSYDjMNHNL7TYPhSUvyUzteZOfhqP9bSx2hUYmxO8FYaRHtgaYCA7lgdjuChBj3u0iFwtYg4hMz2kzmakA\\\\\\\/FfebZQysUOK5xezsxM1BFAwOj5YjYG4QnHlqE3imnRLv1TQ8wQZh1xUslnnESFamb4XbUoZN7bDoC7o2Iqm7n+V5Wqg3R6ngaZASY5sB4AEBpY0yQt9luGGZRVGZAcD2emiIpy5Nm\\\\\\\/T2\\\\\\\/1ercwMVYrb2WRMwwtc8N0ttCmbs0SFzZrpY\\\\\\\/k3SQgNP5TTYrLKArJlB+nJi5P1hCI7hMaxREC6ZFfep\\\\\\\/2nce+yOkR6LcoF2udLxJy+bfG3tcQ41Ulrjo80X9tdg9dfWg5jzNkhW5bq57QDsg4t0eDdvfDIRKC5pBSi8huKNx8N5t3rjMlRuA2Lvu8dF8FS0ZqkAH\\\\\\\/wLyw+2YM4KmuM+JfAjfP5G1vSAbzaYKq64rOdSorCTt2g82vxGb3UytZS8KfK+fKhiLicNd+PiTXOTyAmmFzZ1a\\\\\\\/L8aJk0BSxfSZU51Mkm3Kmt94c886Gv38n2VNJlQj+SWVzjHPisUPBr5qVMOGINSSNU4o5zNP775f7ys+UUpHVEJnhpOJAmKB\\\\\\\/K\\\\\\\/vs4urA60TxMBEqKA8yTJYEwvdQuf0USjW9pqBBOqOevMrjv\\\\\\\/yH9vk7f5wl\\\\\\\/WKbwSYFXEmhS+HmpwnzglHVsgoA+hJ0b6Gb2Xlkr8OrpsNmS2qvA95GoVA5I7e66HJkYmuunX9kBheJDlXxjp7hT62lNipYIK4ASC6Oyr8pk9vKkKZWZMX7JHhptaGm7CfoSgLuScPMasmOMeZmqrHD00LOliYO3waXWZrl5azA5BkdOd0GPIQRUl50ZvR++rAvsqvDI1Bv5hCfZkeQwgt6MlRQCMQtABiPc5nHZq8MTmpuZCXffF83JRo8luKofrynNW\\\\\\\/Q7HpBVc7wkHt6iVo5NT8NL4oCAke3iesqsdqMuVevaaLTy2IcyQ20vW13GpY6sPa36f9BUunXpjV7Sjl10C0OW551NvSQRLf35ydsrcIgr6iajJSpCmJT\\\\\\\/rgGT1lT6OoKt51rC23whOVg+Zwy6Zz2bZ0OMPcbbJK9yBxXWDkQX90h5rPwNextvlyBxZXlTgOLSC9TxAF06yoqAVD+V9NFJzWQIWqy+yrEQ0mgPZOvnGzHSb\\\\\\\/AJFnMyFAiz9HnAkCA0Qs0HfThMKZ1A+Hmn35WXb1zPXxTZagpKrwSrgmkImbPxvYMMM95Gnwj3ku\\\\\\\/sJb0hLC0BgZYZ1qbIzkU9d5\\\\\\\/d3Xby7tXhEcJjJrauaxh7V3dd2UXpn\\\\\\\/fzwPb9yjsqNilb3u1B8pyk\\\\\\\/2jqT9GNa\\\\\\\/Y1bQRsMNgtitikVKoJCKzZe5Co7BltRlupOdfgVN0UoALD3C+z7++f\\\\\\\/r+PF4JP8svuoPbqkI+UMGSEbbuCr4kqZvoDhvniocgd9iMsCclWQW5W8\\\\\\\/y6+olIo3eeivHvvHKTFpn00UJQUK+ZyW7S7X6oXaI+f4d\\\\\\\/4mNIp7Tl1fVAc9gA0tde4cUxWxY2vfg\\\\\\\/redr+QL\\\\\\\/2Aew0slKbzG8nOHG2rJ9gtxFUmVRegBFWZLqeHGCTkdP66phuGsZ7gzPTDYB4ZvWT5qoVHhdeodyP1N\\\\\\\/8TNejTmu3KZoDvqRyPCxCvAsAQ7v0g5to8eJpJWnWr3L7JoqDGn0SvpLGId2yFPP7PKgCtcqYGlB1leveSrb0houkmOdN1FLBXREBczPMCO13oFquGlQ5sXhlgnBedLOZdu6qgaUAipTDBBkRILhm0Ko1cn34XprqhXmQdduPcVOEcI5xXvDMQafeIfDss5JlCPYJKLP+ocdFlCb0VMUvgTSgz82wSrEryVkLi0yUvsi9XIFGI59Ep4SFYSKaugraU1\\\\\\\/xuvCsvy+n5zgtw2qw6tsLS5T4gFTTagWKNm2LpMqECGF86kqfsTA4v6sBApl4SmvuTZ6A4acXW61+zMHDUyY3WpiFco48Mi9DyteHMaoLy+sHWI7pALeAXa1Ct01Ikm5fls60iyiEwctoh4hz+FVsjL6eZLFDizyBECgSCPZC0bxC4KxHXmKOxi28k5Tx\\\\\\\/IrXaFnLWJZnKmy2d4X7ndRVY5xXzskle2dmzPp6J++CBIKsl9YLpxzlEwsAtqNqlJH4rFCPZxGCqI\\\\\\\/\\\\\\\/BqDSH85J9F\\\\\\\/eu0PcVqVxcN48FALwE1R0GyI1SiM0a5tiTl4AGEK8HqKyMv0igX5CSx+IKtmRos7bzCNf7qKcGsjFxzhpunrMfyzLkIh8Dh1oFE1s9NAif6nbRet5Fj3j\\\\\\\/YKPBSniiJ1MIxD64Vrp0ojrAYmHict3pY+lmojryGDjKnlANH3hy\\\\\\\/Wk6cRvB6N4OluPqh8lPkICAeUvmQbyNpidohJDdPWPR7RrxurGkv\\\\\\\/MTFRuUyYBQoj8EmCOBcp+VuQP23ice3IQoOAz9FoOfHBZ6BvU8zPbUsfwUA7kTK9Q7My9yc63s241GrJroUHcMcsCqBpfsExjCv0hGGJCU\\\\\\\/vqJ5UnJ2AOOQQiYtXd7QflkWmJXzNVB3skkJwNxdDU3AfPfXhCK08\\\\\\\/dERMYTNVOZQ9xRw\\\\\\\/G97RjZkEjEbCIaUWlohx\\\\\\\/jcTmQoym1oPoUTwKiuQdfvcF7xea9GSUSetNhRshKELV873RcIUUdurPBo5A4jRgJklgOAcT9pitnGMWbLcp6gC8r9MAqqaMhRwfp6CtxvA4lecbpgpdgjXNfuCLd75iaZXewJHgQy3uxkc7Z\\\\\\\/1Al\\\\\\\/zrmjAJlsRfYS4bW5Zjf0mA3U9EfNcuuVobiZ5RYvwnUEnSK9+ICNdu+Fadle8vytCNyHVO3EkYAxySN1Cs13VnzilUZcwkzdh8yUS0UUCozbvlxW1FSd6lCdrVOxvlr\\\\\\\/MwOKL1ZOxITOjtRcVzmWuObsABrVUP+HzfthI9Jmlm4EqAjvzLA\\\\\\\/y7Z47zy7\\\\\\\/+FeUWmtX2QdSZDTglnDSHPoWHQPqDY4AAW41aVhqdAyaBuJyDot0+CNsWoglyvWy+6H3U\\\\\\\/nGAgUmqLFH3sWMPn1kNDqxw5coWR4LeLbJqsV0dlwU0fmBcMxTxidQmDOuAagM75EK5oWRvcBQH8h50hP6hatmUwHNIlMVPQvJKSWLFnyswiOWOaG+GZuhYUDAoGAqXg7haDdixE8ENUhyCQAb86ANDDODGhg5U6\\\\\\\/tOGoESo4+DcVk8wl+lsLir+3GRTZAyIP\\\\\\\/fKuxybeQz6bDv6QDd9c5SM6OQs8DeR6mAaaywvLLRjjRYJVedpZSsyq\\\\\\\/7nTdYH61rfdjoYNgY7S\\\\\\\/rHHmPDx9rn3wy1Tcm7+S+jlM6XslR8iGkIg0L58VQ3gMXT55Mi2ipZapceyaKbvVIQSSiXlOXTvH7fB7L+0fbbhI+RN+V+u92Kx637E0iLPCwd+n4\\\\\\\/tDg\\\\\\\/z32LzSr9lFmmTX7EXPzQiCM8RjBzQ\\\\\\\/3xalqPlOiMjesLCbl2QoFHE0138K\\\\\\\/C5f\\\\\\\/cjZmhWRjQswjundjR+a0WkntxmrJM3QT9Ah6POSSdufbR8f2wWSLB28VgO\\\\\\\/Sar7zs\\\\\\\/n1155wUPIU1sfTSpX+eHEH4ivSI5G0X73p5FljYZOSJnKPRf5MrZXwQSIQr2cZ\\\\\\\/gjJ6qQNm6VMi03Xx\\\\\\\/EbhIBwOlWrhFq6nF41MWEZsYw2c+GcDXj0gdnd1ygsio0XeRVzrPB8OttQuXAgqQKD\\\\\\\/l4DbpXVGIEy4qYKVjUVTECZ5EEXWZggwzOBP7TG26VLPecQUzxQlSnZfeNwM\\\\\\\/3goyL+2iV0QJP98olBvzjS8pdU+4NspzhPxwBvZ1hkqreEjPROtKMPK4jB2y3+0MLxdmf\\\\\\\/6FtQhgma9pdgs1ncz56imG3RSqjlE\\\\\\\/tWi8B5PxpbGuG31zMR0LaKZx+fLyNEdN0Cc2qY6vCrYvkY37b3XWXqMkwxnkEQITL6y4+pJOvr3xAZmMgKTh3EcdbG21bkpOweLAk+U5rWqW43G6o38dlk7c4tmvITxGT\\\\\\\/6kA8IlEJTwczuO0ZLRQ3TFC02jCx6951\\\\\\\/Qpt\\\\\\\/kTRbRIwdIBfUZrUHGGP6NMeQe8oxh9bhJbZdXejyseX7kORTabiQuooztsDfpu9pS4ZgmrA\\\\\\\/NV6ld6vFqXe9LfmKtjYAaoOt6Kbn4VtCqOreVx9FOCqqWsmrQc\\\\\\\/RVudEKB7Zj45peTQEFGjP13gBYcbS86yQmeaYSFRG377ZiPr68bTvNFsP8QEP0cnAx7l7KXRAtwVsykNdCIzNYaFp3xmrq9AV\\\\\\\/btweNHyk\\\\\\\/1S38fHvOjgPGCnjusI0sx9gN9bxx9Z6+Sibj4GhMdGFtjPzx0qoGZRo\\\\\\\/FYCQLoGB7TgZeH381+9vZrMaQY1GOfQEgaPSo93qIcPij0kh5SNOGp4fkpJ1wFrrmlyNRqeha+scjLq1n5vUT3c7+pAlfIoQDX9n9Qegb4KSKVZi0Wnyk4KCUTrMOYhfmKkrUl1HKARK12X9j3t\\\\\\\/iGtnaQq5m9WVsuqWjMYEwE1UUMABTSmPHs6wV3hd\\\\\\\/50KZvlOZEW0PvreIbjYSKpWW7H6+D6+hDRKPaicpcSaEJMeFPLYmyRsbAf\\\\\\\/v2AMCzKNoHqYlzJoWA7U4LK9ql2UlmrUZQQfgVRd7SnuesDWtmH\\\\\\\/JfQxFP9LUuHfNWYjq+LfN0iWTCYOPht9BmWQPAPjdKwt0hbM6oJ9W2eWawysnbpidOSeTJMkrsXoc4+VYHtu\\\\\\\/MkyB\\\\\\\/VLIUztD+xDIzY2aOnjyjSKQxpFy4jv9N6cx0eyvnxarI3Prvo0ZfVDg8eO\\\\\\\/0YPbNu0NFT+qrWlUG6b0v0gjOkNMRbLYg\\\\\\\/RmyLAr1ViPmi\\\\\\\/ruJsNTTMbB0w36mm1oE8TlNKKKwDhqobUn\\\\\\\/7L8Hgve9Z4K\\\\\\\/T8m38FgJ7w3QiADBxXySLO2bPb2RKbMME0BFFqNQ0wgE\\\\\\\/t1Qr+x+NiPDfyFATzqhSrTZeuEqxWq\\\\\\\/v5n7mwWMo3vXk1JzQGQDXxrElS8HrzqC3dHcUeBvj5CRif5Uvr+oBnj52H5aLlkN3pa+TS8sRLGVu5sdv8xbxC1xK+OYGvkC3lMWaQ6CBZC5nrs030OXZnS3pNmNSaBcDddiljvmpJLwktOg4YQiysyk1aBdwsGDkJhzCcBrJUafIeUYFNhCmWBfpUgWRKbSHfIfDs8Z3f9lIbtPp7vWZnbJcmQzDZn3mwS8DWmWaHpWS\\\\\\\/TdpclfN9Qe8GrJGDCLXsdrl+CM6eJp+ImtIYlLoN5JEg7wDgi4mlEAktIf8c7QFdK1d5LE5Smp1sH7YZCeJFcuymlhYboCJGiujYwt0X1N5vxBI5betbeZKDtVemX8jvozI6gm2lrdnbfQY77Oe9il481V25qU7h\\\\\\\/6MTSpQK8oW3N3Fsqi5UYz\\\\\\\/yZxuKCqJAi2W7a28Uzgqbs2d0fnA3D7jhbDVRMaUblyLKj57GzNVO0mX1gj3TPfD3+GlQOZte4Gst45NAnjk10zosOeL9bcyDLGeDge\\\\\\\/suEaHtAJzEpaa5agvTAbX7eGuK5UGlqT9CLk3jZG\\\\\\\/CcB4aF69AnqKtL\\\\\\\/KDQ5Up9qpJwRxorNDALqIHYMNKDJa+9OfrSxXZAKFlZu\\\\\\\/iAR1mwrgQvps8fUBT4N+Pw8omFq\\\\\\\/NAVdvdSp\\\\\\\/E2mnIw6tzkB1FGjruy6JrxcdQOITSikBPyA3OxbZYKLZa7hJ\\\\\\\/jLGx2tScl3JfR+RI9g6WG0768wTz9TlChnJMfQsVXK2ZNAHA2aAOuRdTlJXub2xF6hGjjQ2HLw4+\\\\\\\/sbQjN45+quOdLstYxku0noTZPdkNxJDP2f3\\\\\\\/K36pRGubNES\\\\\\\/0Z5WYY4nlNDF3N4nwAUnQXO\\\\\\\/URRpIru4d+ClA7d\\\\\\\/eXFmu1TYsIQVoVBmouCIdXZLZyHRXUK3a9\\\\\\\/ngzrorGlp3wJ\\\\\\\/LQN+w29pK43IEhpZcCO9ehncfYkljdhu6z608JcWZtbkQBYbm2V1TvmBn431yovSF77lejBNHg1EE1V6SLW+\\\\\\\/by+pMHAVORUdJeLgCMc4cjmrRrmVtXugQyo8KDPWFervyTUzXqSNfr62CFAfAeKFDjHXBVHhdiOslEEvLLEeJeQNhqtOaZPooJllOLeYabakEs52cCOjgPHmX+004yVbO4dAgeUMAX+Zeyin+EiDN\\\\\\\/a5Q\\\\\\\/Zk6lSZv2un0B1c87HXxD3UXewj1AWwCc12WRzwsKOvGo73FSf5U3jUz46LahN5JrPPeUAc9GfJ1cq5r4ti2\\\\\\\/2NnRsik2DOsCLh29B71+aoP188LIQBLlP4tDHYlO1Umdwj0aJcX1Z4sbetuc1pzn25h66UKa0gtd3\\\\\\\/b3vZoOSg7JZRa6NiEAMzINPXfrmqsCyMG7puJ2BttalJQJEIH0GjovuX7Exa\\\\\\\/iUUdUy91e4ZEsCp9W4bwnuQjR881LKKePrSyadepJCgQmTcc\\\\\\\/Y9ij2HoskIujFIVh39qIqlCPQF41MzscXoVxrfZb+IDOMuWZP55\\\\\\\/R0G6gS0G\\\\\\\/Oadtm+CqtXdj4DTiZCIKNeWB0a4YjHMKAJSTCD6KfLDszUk4prgIahCN140FOTMvuZzHLSFE7dzIAXnvuvvbAeiW+3miMWPW1GwxYep0AN1y2witzKgFf5GMBISlPdJ9UFPa4ZplV+OZidHTJASpSXjbV+IaOoDDU26dbOR2PEgJ6zdmTYXqGM5nheSUV1X97hn8GjUEClXqzhjH1ibm4Vv8WlUnjsR1zoKs79fziXEJa97tOKr\\\\\\\/H09y+06dpD2CECOBAMy7FEmnWt7WT6Ad5Uuyw3UzgusXia78nEMuG\\\\\\\/P5Ra+dCWbEvFYndVebL2hV\\\\\\\/Ny7hHCU\\\\\\\/os6T8Y0+6TpUarnnSOPERz+z6\\\\\\\/owv00UqnfxOIvpEYXYCQ7+pclfFxc8pHR2iV6wGzeFc43MBIe2j22GPi9v6xJx5JVaT2lvbUjXHuFv3Z6Xm6GE19sHYcXJdmG1tJLOIogxtbkTyAZCpn3eQ\\\\\\\/S0KeNY96VGvxSnIQCWqXIDBZQrAbtHi+7ppXqdbdCvtBqGr3ATVF8ZDxuGPYu2e2112DnIJgi+BJ6HDI8FOuXIuLts704ducszqgBiakGDVtUei8qG10CJo7X8prkJjZYUWMEjn6y+vbR4piEfLLizi4XBp0Ek7tPwXfoMAJdOE5K+04ioSFLS26k1NozPVpwIL0+nfXzMnLOJ\\\\\\\/tKUORcilWZqbPgmS4To2tpS1JPKcgjjs7WnNc4RaU3KE0pak8rV8Mn5nCdXeROq79DZBy0mePaaUcnugqnA0GLzCaZH2hzSgAXiI3D5ho82LmyVeln4mLucxKQkM\\\\\\\/ODS3qb3D8ophtsnvNRqeipqiJ\\\\\\\/NGFOHBcYhXUOGZKEpAdt2Zr6U+s8nz12nfEDfZ0HUAoFiDRiTLk4Oa4BEUjs6JIGF5GrUhh8ZsX3ljb94P1nRiKI5sTYLwNm+G4aLKVeB7wOpFVgChfBxXVg\\\\\\\/c6mYD+UYtO8MB97QofB3UqRvU5LAZCpmvU6ok7eTQWeAD0+XeWvbwmrcGBdFORiB0Cq4FusGhe+bcL42JDdShhCWwsvZYBoXxhkgOKMrEkXAETaItHqlV1QdmW6u4tCDz3USrB28XuV0kCRJGLGnXKnOWAUOgizdc9Bh1K0uNTynRvUYW09GLlJo4t8Cnij0dVdK695C2h6V95nlhalgBGfEmFZKmn5gNAZdpEuwQ5hm6Cv2tvhWWcEQvLQjAPejPiMdSkSjISEaXihsa8Xi7Kn+LIsfyIms64BOoYy7vgcycTGfTsCqJYj2haBDKFsmp4oth7BsIKOQuFYPpUrJcJpjYwop9CjGulopkrseL6iHdBKpVlpuAMDpE1lhnvSS4T\\\\\\\/mmc8v4Ave\\\\\\\/slUTfYBf5Ft\\\\\\\/UFvwIgDyG\\\\\\\/VUeGZUOZMPJZnD4nud5HRHiI0R2cckfQ1BRMe4M\\\\\\\/nH4lTsPFeTMYNKkMnVVMrbj\\\\\\\/fV+qugR6Zn0wL8J8+IrENCndlo8W4IvL4VCIGkovMFHWDbYCh86fSekoOKKoBCWSnBh1BQXh\\\\\\\/1YUNb9QAFTzGp\\\\\\\/5OBy1lB9FOlP6xbqUestwe4kA1E7KPBq1IR1ZCAFTerOYqABqCLHNg\\\\\\\/SkQmsECCXEO0Sxwt+j+LvtkeFSWrpkX8DailYHHRmzUjQ6XeE3nGabioZj12lLzrltOnbcDypDv9uLmHgJvHi3VSM\\\\\\\/SOscToYcM9j\\\\\\\/K6CZQLikIMYuNu\\\\\\\/je2DLUh73JVmqcjrfTJ5998QoGzV5EtjbLh6JAUOJuZDiWZQH7BHlHLNFmXgJSuZqWnIe6+2MHjWMzGICljxfyEip22FEFEeHea+brUY11yw4MZJP2667bFGWbv+c8BGATsEe2UONdnAkii658GYzsnHLNLL4lnDpGPQ455hVBBp0ZX31hO5qpLjXEOT4lMDEf7TmNFJtqsUoByhZV40sh6SvkS\\\\\\\/alrlNo9krPAtH1n\\\\\\\/d74wuNWTx82BMc4wC0\\\\\\\/GBafUgd1iURhjDX5ZvWuTWoqLcBwbxw6Gup0r\\\\\\\/Beou8K3Da\\\\\\\/cSr2njcN\\\\\\\/3MMSuyiYeh6Hg2Ed9xRzEaN0AbJe3SEoTHxfviUSviJRv5IFRlUS\\\\\\\/zA0jQlHKbcFVlQv\\\\\\\/YNmtkL7nvBzzbSIsUHyokzrVnDBHSLrw7RkOOtZKV+\\\\\\\/uizYKhdeciQmheyehzulwOX6Z1gw4Y1oOb4tfcFwMk0TYPp\\\\\\\/NuITWnuyoXymT9+9nkZiIZss2ghQa4MKFOAlB+Ok4Q7LczcIVHSndsRmGFnsfBk9RGQmmcF8MdayyzDI+DXwEvwspq7aaKp\\\\\\\/G6z9Gqjs0mgHncX1hG1qdDPpnB7S\\\\\\\/AaAQTqnF+Yg2RKGL9fMXqHxF9\\\\\\\/EhD4qqRbiSVg6NdmCya9UTgGakblXWB+bYM9Dlj5gpQ7z8oeRBGlU9c7Rf4HEXBoaQvvG9D2mI3YmzyHoYdi+MFTm3B+o\\\\\\\/dDRpd4DQNegKieQIuMhDbj3yZfF5PgMWiXsoPvcYYCh8DtT\\\\\\\/Ig4TUv0KvwnKMR8GL8Lh2nd1s9GuZCBBPkBpN1huA2KgsqAdUTf3KUO8972AndLkSqJ4xpJNLvzV2Krbl+itP\\\\\\\/fiRcYbaSLH6ZxVyQ\\\\\\\/kGfUCJyPal0aeIsmLXzrwZSMnuKLwZTFal7uNjmfdcpiibHGRWRW2B0WkSSqo3ius\\\\\\\/MxlmHzSsk32d0OfEvPDT0xt4HShlMq2bXrF6J0U1NzdxmxTM6rh1N5\\\\\\\/iaKrI\\\\\\\/0HR0z5G0oGbmyBm5YoTxy3s7BN3mcSWhmdJ7HCzhki9xJPUGEoTKcMnSL58QjZMcirl12QoYnWHP0nFLty8w3EFe64kIk1vZVmGZc\\\\\\\/qnCnLyECJJj3p2I14CI9ebshOHJ23jla7xyGJcC2EFcwU3hOCI3M1sgST8IyqUDY5iRYZLJkXL9ZDxD0iy4K\\\\\\\/bCh9aKCLuvxU7UZ\\\\\\\/Sre9C\\\\\\\/TqxH0ixBu93QaQ3SXlItvufGQ3AuaiKLGQvH2AobHPUEDSHXPUJtzOZHgTGc1PgoWMEUl5Id+HISzTEWlUW3KrHGHugvVZfX1UZxYY0uXcUG5fZhTgbiHXiVSJTc+D2MGLVu8JRjKWlbtYVcqzEcLow9Kipke3GO+JqzJN7qilO8vR79INPWYNqw0nU6mGNHE2g+oeeK\\\\\\\/AYezDizHCfVbmv53rLP0BEzr+LvX\\\\\\\/yrZtKu2ERBpT2RzQlmWHMYbqaYL4rPHNKogANVUx4xo6KKOZ8Zmmq+PGPT41gi1CiLQGsVqQVLzFPsR20EjyyqftV22ldbY9BreInEssGCEH81SNhlaBKEWkwcwlvIqVkoonq3qvSQ5wT2PsSOnJcOp1CkcA9rnaWK8uFsQ3W5NJYqp4hy8IICZv5ct3N26qiT7KH3f7Fa6Luxfi3kseWI5pyU8bo\\\\\\\/cTQvVmd8c46gWsHDIGcyAvHFDwqJCxPRUfmJ1EQNYi09RO7ImR5O7TkDjdbiv5GlnlkvLg3zyusJ18jR86zEY2lkZ37pFVHoUv7BG2At04EJDFm2DY5YSSeEFF\\\\\\\/zDt3w\\\\\\\/0akBlPxk3CZOgQojFXsztlEk7Zc9X869oenxn8JqfZxRlnHJrTGMOMzyKb+lJCz+vBAMIpKeU2gxF9gDCzBL\\\\\\\/Ww1DP40lIiRMWvsI17YIz7cVhSu1ynELU92BBuRi397SPD5SdBAuUDMWNeQwyu4qEmewEz9MC5EiI4xuhtK35iEY+\\\\\\\/r6U6QjUGxJoKgQ0uC\\\\\\\/HDK6sEgKNwjIsgGMUlY4tw5WLIaRaDr5F356wzVa1y4kyJyURqHBGAUN6IKxTADaJTTt9PLlJRvXhmOiXwWHOgnTyJtzvGaSfJv2i6q6Kg\\\\\\\/ivl7JAWMr4xNYJZj1g92Wj8PZ2K\\\\\\\/L8blS3uAoxWvjW3TQArO+3F5cMD5\\\\\\\/UnlzROQm7JHLCVkc48CtgMaoShXBMX03UX9jSnuZBYRrsJ02QIxZ\\\\\\\/bbqB7p8sw0l6b4RBiECERrwuJ1C6pUpNwPMUpRq\\\\\\\/AwdeV7ZCAUcX2Obb1NezDP3TAErniVisOryUKkr2N6PSrl5uWe7mrNIehPMSFT5OnjllClmKCdGohUJSBhnu72QNNEfTMBLBCHFHO33Ez+kdbU9XQj6z2dVVCFZwEozdRhQ+VzS72rIYEmqzKtQBwxum03IHXdNgkKoIqguFoGE0GYVhqjeGhxJqFfYPbVlbnMs9YtzeCPkeuac59AdBTBBeyjXtX4idUUpZeK3aNXmF7SouJkFF9Jxa50ooVljUX8msAdOveTGniao2r9ML9D2tERflBRbAEBfSuiQtopxdM+E\\\\\\\/D9brpMIENbfxRE16O0fHTNmJ7oEL8mou4nK\\\\\\\/2v+B7QGx5hdhq\\\\\\\/ZFgKTlmj1+L0\\\\\\\/AfaxktFHSeSwKefqcdaQGwupx78D0hYm0WhXJlXFFQZe\\\\\\\/XgkrNX5s5KVtxrlUJGoKWE3Shvl4qsQIO3xoRHr2hBZxjbJQfDsRB8i77SKltEMeJRsKXcwD8+3dCYCo6ui\\\\\\\/L1JIQlaNNe48RRcd7TDOUNJdjiY\\\\\\\/YfwwBGITEFdkpADFz7uq10TXhIeU+784RxFT00moT3IiE0ibei4NpeNDSvbp6IxGi9vXszrycPnMaFfE\\\\\\\/SZWsq+6bWjbrngFDTu46t489Llg2i1wcTUDi32B5oSyrrKiGy25ZTfC2qloqLiaMZYw8AO3TbjF5\\\\\\\/B+ABeDvSFrhHStfRqJRHNnMecBKi\\\\\\\/Il1TnlCWd5fX90MgD7a0WzpF0g9XSMkp2llcKcVkZJZoO0hJS28VlO0+1U+7Gl51jiGaXApfiFutLj0jMNj9Dr8EWzBdDKC2VvoXigcUZkxoPO3zE1uAVfw+IazJ5esdoPWZALP580cHafQv8fK75Ee9oCvwRtVZbgBRqt\\\\\\\/jJxy4L44qR1DduXTR\\\\\\\/aR2I7ffDgh2pmCduTdL7WcH78PqkFBIvy3ZohY41epg5MvK8mB3n2r1e44rZkKbsKVvs\\\\\\\/WNxTMadW3v60LbP6a2n52eFaFlQSP+lPdaZM\\\\\\\/BCG5oJI69LSUE+wECXgk1p\\\\\\\/YgB6oCuDAWhCdg6d7a+rGI8R56162Dl\\\\\\\/SZzeZu0YO9DfdOM0YJ2so45fj6SpvqyrquhMTI+96tFMPdX\\\\\\\/bMdss1blL9Z0PVMXHRGl\\\\\\\/ADcObzNB+5+\\\\\\\/eIx3MilfrUASblqb+3a5zWt2+orjkHE99m8llUrqQdir40TjXTvJgzhaiFtJ67pqJ1SAjOGtGrPr+ycsawjYLLmFbPopFbkyRlaBcO4THC573DbtGfbPeg1aaYnyOzptuc9Rc4NpCa9n5qcauMoTe49\\\\\\\/kKkbO9dp9f77ar5SaiALTurfkFNLkVh9RK+esf6uP5dnHKbku6ap7HU5sSte8J+JuzNJFZavjDM8wyd79b6YCPbtReJDfEIWZD1g1\\\\\\\/\\\\\\\/RTtPU+YTH1cYcFjqP3SqkKOBNFIczKBjs14XHOE8Og6ioLtYRGM1TnLHtDdX89Z9UHamj85qAJY5KLmr96pp3OY24T2B5cvtesRJACxnqWAnkWi3axByJnmbKOAjACh7dxjFaFFDCoFoCd38nNtycpv8OLYZE8iuY4+Vk45jholjILIGWxzPpu1vuaV1RYWez3kjKxqNTLp1i4RoD9tPbRtB3lgEyybepbq3sewhVOEDLUwZ3Uo2R9uHXzUP8Hd4zJBVpVyIFoAsP8ILk4Qdt8tGFnrcls3FbvA\\\\\\\/F47PMZZzsHsR3RuAMUlFftlC\\\\\\\/jt19489U+Ljc6cY3t9rqNw9OZzd5aPxceZ\\\\\\\/8XoYgxzCOzbmJCrlrnHbRX6SUOZ5j1OhL37tE3DkjQ\\\\\\\/MyAYledMao+MsBFhQxbRk8rdMom8BBP9WywnybJD70ZiOqLQDYFGVM1diV2TRufgbdkwZSqH\\\\\\\/4CFcSaQltTlMJyRNIpk7spxcsHc0THBpWKJmpPT0WUSDps9qkUdJZNLM1fMYiqWazf81Y7QvNfuYOQe4tCR4cXr54TwpoiSpDk1D73CChINXaXvqfb4\\\\\\\/eFFfn+jXZiE17tvbh8zdFQLcq2jY+n2hzbkpMJ8tahOpqUt4S0oD0MTfTGgvAv4tIidpH0VwydAHF9zL9Y0gD0sxBqxM\\\\\\\/FFMZN\\\\\\\/6g31hLTQ0SwTWnKxFzneg4saDUgjcjgEcCMEVEzKKUsz1d39aeYG2kVbgBUyqFK\\\\\\\/tLTCUxHfDUBNUqHW6Z+8Fah7\\\\\\\/OPHfLEWAyd8NqX0q9s\\\\\\\/TNNM9uyAcJhR1qIYP8Q+fSFVuKs5CCUACMZFfosabft8PEnxF4g\\\\\\\/xEAXIiZKrCnqhwya8wkfJfNSNy0XtMu0b5VU0k+3iaU1pFEeJDhXjnxdhpjuHrTyGAaOQQ6VOvZBko7O\\\\\\\/4u7DeKpFvao23uYp+xtHj+HByNWWDl3J\\\\\\\/dM2i7XxXAa1R6aF7QHrB16pfScyiHaeD7Fy1F\\\\\\\/o2jIgDuR\\\\\\\/ZYMXVUAaGMH67rjEhluvHdAm3m6d7PqjlhWBiZhdotzGMG01sLmEchVUXoJXZO5jqot+W1ZRMW2z9kNBcKGUI8K77D0hhMiaCWM7RtHwyFeCsK3\\\\\\\/V+\\\\\\\/ZbCGFaCLdnGhz61ZAGe0bEx6KIzmkYLDrlC34w7xEJNnjb6mdkM3QKbRll22sb6n7QieAFP1ZzXz1mURuxu8Nc4istzHpt7ve\\\\\\\/lOFPg6wskXbI\\\\\\\/x\\\\\\\/lpBhaCXy9DJeJR9DlGi2XffAzTMvemI4OHuCSDetYNBQ8Q+kQLTyiYJoPYPmymXOVKLcD9yELefXNU2pQtSvOUkCKzYRhIFSm9mwFjI4\\\\\\\/UPHGssR\\\\\\\/1WlCxzJrcclttFNVwDLbAmIMLxd0cyijsKq7ONQZJ62+eatbfCa6vAW\\\\\\\/aek6fTFU1hW4f7kJ5yuRP2SjPkjL\\\\\\\/M+NbUPbwfJk7T8y6oCplGVyfJs0NxIZn8lzE23CjGsDZ5eZ7MmmbDupXdlOcJHP+5SLWFTArrSwvNjMgrFKv4k0PKrXydUjuQ4xGO+hVhvS8T2eePoSio0C8Vrgh+NkY1QhD1PYmvWqTNpll+\\\\\\\/04ogZrvdORbd0GUKwdwlTv4bChZjWeNPKvGHpExFDHW6oYhWZcCTeiSABjJG8E81X6qv+5i\\\\\\\/GFPMQFNSyIYSUcOPqd\\\\\\\/Q1uaoJ1vU+tsRTnC8d0HkfPp2xYPgJEgMy+mR6ISSCDH52eXk3Ru9r92kDnhuw6IEW7Nc8fHVGeDC+uckx0ZWa8f7WNil6n8jh773txgRBLw9RM6SqRlP0h+FVLimdhGdpJoaLMw98Ig1+XJJS1SPlAsyqMZ70kg9ySDdPBTa1P2nPuoFm6xMwiFFZtyL7h0Rc5admGlEcTlcXxWYKUTpJDVXr8AnK5ItEmBtIBBMK5K6yjpvgZIlg9Qae1cqokxcfMKcFe4M4Cjq5tzf7bVgh9v1HLu\\\\\\\/M0X79HoND8MO8HlYiem5FPmR9o0vWWG77JK5ooj7kahur1Z52x5M2585VVhQA0zAgT45mGV\\\\\\\/tXn5IIGJW6PXk\\\\\\\/KQP17QxH2KCj\\\\\\\/E8Xr0FWwp0miSPikR9r52Qqswb05vkjoq7Xx5qQAjC\\\\\\\/mDSykRPUMOrKcw6bQSMvB829DXtipYnQX6B8nn5FNvu26y57k+oAU1JnQtnKM+Ax21QC+a3AxwjxalRhDCzTconIVfLiZ1U8G75yfZbJlv\\\\\\\/NHQVfAQy9agBb5jrLRyCAzU735WuUJ\\\\\\\/EgL\\\\\\\/XWwVzm4Ap6V3hm3JfvZHl9PZRA+RYAkZvgGAFGITjavtENFyC+kLa0hMoP4J+fpEMe0xyCIR7HbgsueEC5k7q7gfHcCfl8ljPMXFMnVd8R0WISUOTFIgRnETZYBPd5XjQudXhdvQI58x6koUimCHhI80IrXs47gFAgTBwUIJftysN4Y2+mmHqK+Nx7Zcw5RrRQJkq7AQri0ylFzGkd\\\\\\\/9PviUDlzzkjcmkRg3mWl0NpmznM0qM0jSeQOFFuxWrwGRVwiYCMzXV1L7CarOxTAphrVKVA42gKPW6nGIyS7SXS3W7m2dQjGEPz1Vf+4FdWu0NdVDcfKcRsL+UyQpCzZ5HyORITMMlD8txwFnyGDUcm5SbKkqicCcbh7+XY8rVtJAvjRLyAtieLBfcmpRrC6iSZXhEcABvAZWnixclw+AZL9Cw5ZyznzPbUjPgLcQyqoXID8Puq2ypD3UQaPo5WjS6H9rexE+bfPZN\\\\\\\/L62zONvLcm1yXS2dz2aLY7moNyzS8cqpF6D6ywmMz5ifbF4McxlLOK9LoTMbEdWjI6pN4sKVw6DKhPJHdqjz1h\\\\\\\/fQP3jEVtKJ4Ygf+VXYHYFPXyHL9G115b\\\\\\\/vaWu3fR966KzBCvV4PL7Pqgt\\\\\\\/kgfQPwlCwrJaUrAf2FM1n0p1EddwwS+845loBtp+pjML74pCXRXT8p1z9eB2wDKIubDzPU9nfk3rbnkFxFkuDi2ft6yZ3FcDVXVYx+AzPGdAV+edwP34MVe2poJzJU+agiqJemULqtj\\\\\\\/hQtOU9trYweoVdokQ98J92+mpABPIPgxkAKWqmJWPjMQ9Jc7YX91Hwk\\\\\\\/AOcjWquYkZ5QBMWrpAoOofDugYg5pIPRwqwZruzHgUGmo+Zh38qeQ\\\\\\\/qtyMi9ZARW53aG+MYCMRM6godgj+PfI8gqF+aEOOBYiDoRt2K83sucdgobJxb02M4byGqoXUrO656WYxPGYYEtDi2NL6SD4wcIRHUc+6wHzgpkUmMlhNOL+2x3qu33jYoE1bmXBbsOSG8nsT8ap7U6m0lNmwVqF+pFC1sM3bE0LHLy5M3\\\\\\\/O7KZItKsHvykvnLJqdYq2Pj0qBw1UT1ZBPPbHeh8lOBtmwHDePrQr36ExoSoQaivBkXIEw1KmP6UpzESSz7fxnngp68SoIe1kdwsZkX0Oy+Pbl4TA+KoycmSG1uiEWVaAmjHcyfDr0fRRx0G87aWhQ2wS9TawAiHHgd+rCn3Atfs7ArAfB3WMveQzrY9c4j4DRWf+3JenmQ4Dzepl6FRT\\\\\\\/bmVDBFewdh5h0dIXxQklvx7fBnXQie0Me5vEJPROfG\\\\\\\/rYi\\\\\\\/WUZOWCauOygbxJsvEnD4\\\\\\\/dNzMjFQs6CdSlF911Md6\\\\\\\/t8XWq+y6ZswVwajQyjyuBjeJ0JyvvJkIFw75M9Z0b80XCbhx30TO9vokUyFdmQfBaC5+hzomsxLwS5R8+vHbouNZqIucV95oCqW3zBWsgNDbGqes5FUSLBRh0ZIVCHK06mqQFeUH3DBBH0UE4CUjql5SEDKyZJAU8Z\\\\\\\/FIMljIVAYLj53f8is2f80ncotuViAG4v63wQYR9Ys\\\\\\\/zRMSRUyV7+21Bw1as\\\\\\\/OT9sIv4rxz0rGBD6Ny6XOP1LElvetzZx4Ft+7cyC0tLW\\\\\\\/eVB4MIxfloW5k9Ud4yPN6\\\\\\\/\\\\\\\/dTjh8nzoP1WkBk4rhyeWMV14b59u5sbCK1+tAY6agN5P0H2Waf972fHkn\\\\\\\/kUBLxC0yElaWvPbtif0bT7GuaMb6EHo4W9A7mkJEVtReh74oGS91+4b5TTkdToUe9QrOE8HqCvRGDUCuNLezyE+LttfXCL7TYi8L3\\\\\\\/y04yRLiqUZ2BttQ7pn9gTROmma2rGeJuOOWN5FqTqEIVYU8hfZuknDMOCOc41UVNn5jgAq5KZ8b7JQobjELoaLWc8gjrpxreNuA2AQNbl0rZP8mQEfZa2\\\\\\\/NdmJHymNFhjzTU329Fcx+o\\\\\\\/gvjJVBiByOedHwea9DzdKgSc7d7q0jCvvS0qcou0v\\\\\\\/fcoZ+KOPWcKwQZVzKSFrr7PIgiGa9U8DMekUfL\\\\\\\/ImWuAyUZmPW4EfUwtxgJcyZDTzMayYJI6sD+l1WBdcoT2bQNYkscNnN6sYOZOEPmQ0Tn+gffgxmCSf42pJa8MNNmJL6X0KggrgUbBAdpwvs\\\\\\\/U4fba7EiClklIcUgyB6FgkwS8w3HK2tUuoA21w02a5fgXVk+aSZKAVv35DwOLSOSss2KcPnctBCMk8BI2EL+i3+\\\\\\\/yyVqM\\\\\\\/a1yi\\\\\\\/3meLhWKNkhZhVFBN87Ekb18GlDj+U+odxiA0lIPX2497nImNbGhs4Ilhe\\\\\\\/c+biq5fkYxQ+L\\\\\\\/AwKT2JpHUDflpluqVSG1LpLhWWYYm2DKSJ1\\\\\\\/6ooQbj6bz7Wou6EooAd2ZyivXo1fgIHy4VNbCG7McXMThMKcBOvh8Mu+A3PGJxWimeSxOgbT0WhGj3XbAFlbBe4Xr2OB79Cv5L78wZK4iE5k+jvq4thPH4u7uCsx+uzLoeirqa0FeIvhp2p3FRX5HdbBMMKJNq7A2nv9ZY7SWNN4leDv2OCe7M8QYo61BSCBH8L8GeXMRWb+aJnJI28f7vAz8HAyDhAF32n+iW++emfl9z8D\\\\\\\/O7NH3MmwxiynrOg++fOMaP+NkOli6B\\\\\\\/oM1w21nrUpZS9j7uNgK9aupd+bKsMM176NNNRAQNx\\\\\\\/kq73KuV1dTPJANxKbTBsQUJgCLaaVQpkAB6NzO7vVA0rt779SsK8tf448kQJE3N17L2wRZcGq\\\\\\\/iXAlnFBzFL2Q64eyqd9NtEIJVNqOZ3PgHRQDPSrcgOOJaZVebEjRYAmD0GDKymB28KmHZGqtnfRdwbAlgagGZl38MIpQy2\\\\\\\/J4sTBiNjyy5924qsNGtXSf+Ng3yLLIJwy9DBhWGyKQhFICmlwGhNyGPAvTZ6Ug5s2bAfAvB0vwj3QYpV3QzBaUZY5huwGOVZ+cTSVHYhmRenZQUzAFblMWwZPFDAmquY7zi2jDhr4HJHh54QFLmNskWvIdhpg6GU4xVNLpuaIiw5SdBXwYjEhaDqU4R0j2XR3sTotzl\\\\\\\/l9SN1Rns26z46De1PIisp+9k\\\\\\\/n+RuE\\\\\\\/84ypOrFOhK8H\\\\\\\/sA+HLeAK\\\\\\\/qxH152GDrQOCAtQqhsCzGuHPpOJc2zaMV4XN6cFYfwaNRsg1LLPqRjfa6xU0awHU\\\\\\\/TQju739uW4HEqkwp8bJARNlkpreDdpcgurjjII19uMCJ5pRmocSyzEtbxqrrPgIQ3yLSjrEyvDAhN1dNr0+gBOCEkeEdQ3wsEblYg9EoDoFO8uPDUyxu399\\\\\\\/uNAywGs4V2TKveGqAZDyp0OyDKeMH8HjuT7Aee0aoJKdkClGI9mcPfBqbFK7yJoWG3chzRgR0d6rqWib4Un44MFJylqpPiQewsB+4\\\\\\\/67Q3hp8zG0Z+oL7JT0Z5BZURAlPYjuVENC1iHnFQCQD6DgFgupsFHfXjfCk24tDyjELCByEqlBNnBLOztroLwNPj9niaOpYkvwdhbIN1PH0kk5PyFq1YLgzU\\\\\\\/2Y1bCb9BhWhFuSVWx3ns1XWjXYZKTGNc+fizpygMSmFVRO9AuIIHkgYK9GGfAQ8olb5du3gu3pH4B+oL31CrQWNR74sfy0eSOkuhXl3kpJGVneZxhRja1I6ggSnsjM2yRiLd+5GxpzkaZ+P5p42XsqMuENDiDQVD1\\\\\\\/82da+gEf5OM+6dqtn79HbqlePRvvxr4WB\\\\\\\/OmLB2gY0yRwcmldrxxMxMFENFj7nedXKjdofmtgMHndXPEgQLJF9AjSeWgF7Ro++4sQ23Vf5wSSIv1D\\\\\\\/Iq+JU08UAl\\\\\\\/2Xg86D3kUR99yzOGeox4uUpGJDGSziyTCI9o0mByMSkM03ynZpzkCQKFbywatJIzF2Pl+Pr4qgRdc0XokM2+LCPulHqs1q7XtmjlNwG2N1+ORpi+sVKyz8UPMCE1YBJa2D2ohPiOmyVhtH0VSIXTGRc6SQI9SMzfVxu0zw\\\\\\\/iEC6dnTO3FbqNgZ0zNFY9QBqwe7M+2w2dMYPcTFWMWMSjKttFLlc4xDkgEOOGlFifKBR\\\\\\\/zk+71Etpfflib\\\\\\\/FaEYAdIGmorPODwsmRrkdjKB493yuI0ZmXjdiEbpU2GNgpq8b9aYml7lxI17Necmj5s89WPu3s3PL6\\\\\\\/w24WUpRTCOZHnWu38hau+OGrfdLw1EhjOXpg5EyQvw3TYhNl41KA6mWeVfUJE0L\\\\\\\/CxQy6BTCUMVHy6ingqhJ4Ika11fLkzsf40GkUC4NOnyI3JYVRaZfPcSw0ZKNCsKfjBjgehQw2lmnFWRid1LwPmT\\\\\\\/fJYkvAcbHgs\\\\\\\/cG7au1d30D4NTCxRpb6qQem8Q46UWAOdi3ZOKlPV7QPN9h0VR\\\\\\\/3to8u0mhw\\\\\\\/Ahj2fwAVWs6RNtOToRgWdZZOjJnB+leEnnI2GlvtRN\\\\\\\/0KwcfmTXcL4AZk22OF+OJ9n2V3zIobUgwIOVUCJz6llmCPe68AdzDlupQ7ynjTeEDEuPe\\\\\\\/eVLCUYBhxhvlg6LWiPvWdceP+BZMQGgAIo7ANMIK+HKkkPpi4d5nUO3jgX\\\\\\\/hopSPTyIuobjc8+X9IWuOMmchnmUdZ9+hSMjqx27dEiZ6DL1wjl1Z6txsUyccdtgBTpKKU+bhh7\\\\\\\/wOJhXxYJvD4Ryibr690Gxet2wnAp1Dj56UAods5cevKLEcMotX067W\\\\\\\/q9UkLS3FDandu9Pxf3xCMwZReAXZb5dlNXbO9w5ZoZuPr4ONb4nxpVUxcJt6foNKl1DwWd0mGXZ5Yn6QLVZBPGFJmxNePWdmOGVnPaQaCdTr5sdWf64hm\\\\\\\/qcpWBNYSRw\\\\\\\/Ur1uHZxYuHyy5JpwK0\\\\\\\/sbTZ6YCfHiBv+aNlefDNDu04Oo3NHhKnrO2jOdtR5pZMVN2QX\\\\\\\/C++eTuCBnxN7G0KvoEb8wIvQaC6SxDlQcyNLNr5BsTfdczSgsFlIw78n3lJ5fgA9QIYvKkngw9xA6vhBYdxCAyJA9JdAPPmj6oZLwJ3hzLW9GzYkET14lczmzriPs56ALo3MwpLVV14BRkVtng6\\\\\\\/dIP\\\\\\\/ilxkRcTCUgCbk+IeEBGH+mCqcPqT4RUQ+vwg6J2fJulI2doVprNjVCVKAnjrEf1mRfbRQbR3F56H+8Fbb0zapwT7c+kLT9mb+N3uvzriPzTM5Rn9\\\\\\\/cQxIv2gbqTdnLK7Nm8\\\\\\\/1wmzMo+4yRTg+OHsgddSJYYwtTwQPPu4Av0pbVdjHp9L8emRSbvTmBG\\\\\\\/JrpYoDCWixFxbucVpc357htC5TeQUWWAvpdE1NZg1O3stirR+b9WCitdn4Hl\\\\\\\/17LWi\\\\\\\/xXq+6C979cdm7UDZWcnUcEoqA9vfMZsY0VfEQOhK\\\\\\\/mIIoE8EmAFtS1TWo1q5IIdbz17l0QshhiyKM8Jf+27zdKH\\\\\\\/C2EhTT+vbNKydj6vyn8HbunednC8JQ9E9iW6H99q420q5YIUpvjZNwomJlAwveOlqChQzdckTGKQr1IF+SzB0fgFNnVd5LDPGnuawQonGvTlz9pmrY0Ym+A9A0DEoCIYDi1iSb0+TjIuoXSjg6hIkXijSasygube2j45yZvufrEIBHxHIbCEh0kxJKPwiVn2n1J+6yVYrzTPNXplFrGX+Lc9zVvroqssRVASe7Qe3ZNZV3+u4L4SOrXb5uJe08A8iJSpcSPkO9MLMpzbIdW97mE6iiNYCd7i3BimPd0jnu4hyU+at2fbCVNUFLPZcpz+9sy4eQ\\\\\\\/QUYdfCHWtlJZOpYFrqjspjqFkKbUe\\\\\\\/Dm3Eqx6\\\\\\\/gfY3YQ\\\\\\\/s\\\\\\\/EIjYR0Z3+sv4aJay3mNmyK8etPewwkxh4dcNAWAS7vvWZdBwz1op0n7jjgfGgREcQ1IsUrwXUblF7cn0vv2IsSYipug1EsY\\\\\\\/mH8Ch349M98Cmyi01QNj8cgwFE89tRw6mHixy45yY3Xyk6C3o\\\\\\\/xyYVhBKzzNM0Ed915YtqJsJCA2mreZETUi3w1zYHsZAjUbgeltofQcxcRURto4xswrJ8TBhjxh5LxQn2ixVPgwncbjcKxhJ43fDC7+08+ZuUiEv2nSNudLpQiewSLIhT7qQ\\\\\\\/HewKVe+0ZxMqKD8DKuEbWiRE+FORuPBiKMYaABYNqRAQpDfkzQrX+YZUt9GGpW\\\\\\\/Xy9fY2\\\\\\\/tV5Ev4U3dq3F7LmznxLlNhMmnaHKrnrv1WHqYe8TC1mrtcxUbiryxgLnEA8i2CGPF+xza9YcZmHus9o7Wcu\\\\\\\/pojH5BkTks8J3H+Q+T8N6v\\\\\\\/vHUeiJrgNLvrDbRQOCp9PwYcHMJuYlVbP\\\\\\\/Z13VXOdmQMFI4WORPzyfQZL4tmsUKB6LpH5KSTp3T2eJte6KCwxCtxLllaVdIMiqCIzVV2Ne4ZjMOUaF2JRIE5xQNIulKwdu1Uhi\\\\\\\/Ciw41eo0EPUjIUE6PBBlnu6N3loCFAu8IRaW0w2RAtpExxwbcQUi9sz81uvFLhF+DO\\\\\\\/Ol0MN++at1fu+CDi8yQGLDbvhf62vA3iUu3EZkTuqC3\\\\\\\/U7YU6RHUb0CjK75ytI9CAGrfcbGFKQq2ec5nbvX1q9iohP69SYAYsVJffrc4C8gXKHy3ewSOF3YkhbTmB1yqEt08rbQ\\\\\\\/ttAUuMrenimeRzos3SNEhGlcgLNbp2sBnXPhf6TEO4gRhKz\\\\\\\/kjCVWAMAkn2+8fRapIIjIcbZ95AWkuZo93Sku3\\\\\\\/lSmag4RUt1pDNNDvNTj5OuCpz4tldXP2xivoJ6pUGtEq2viYN5QPUxp\\\\\\\/afI+bLRXVgJS1YJO0HcIxbObL6+Rwgi52R+t0j0185upHKJ66wMH4\\\\\\\/vIEQsW5dEBGjeHRFAhHmXtoYfnOV1U5sXz1n7VbgUCj3DjMQvgpWMg3dSQ2vajiv3RelXbRJ5htD6SyPfSEX5I3ADTak\\\\\\\/zjBouxOw8u2XwybcM4eDG8u6s+JaBZU3NJMzJvf3mKO3RMSfEp6\\\\\\\/Ga2i7QbRrIoHmz5G\\\\\\\/lL6By\\\\\\\/6n0fsoJLwUwTXVn0YziKZJk+DEBNdhEWRRpkRTtgwWQaK+0vq\\\\\\\/5Vh6jc8PdYkrBLk0qR0YhP77yr6wUwypfTXMOzyNClrlPelGdrsK0DL226LfFzn2njm3S3sWN4MYg8rofcpefF5FujROb1doTdaauMDKQafX5h9ebxrSDAbYsGNmKUGZDyg4jd1rY335ZE83ZXKTGz2sl3KQpbYX5AHagM4yvVrT1EfJIPSW+S10yZkPJwvMGAnzTaT5pb2hxu24unCEHuim3XY+c9keM42ZIzO81CuNKgPHI3VCe++yENw9G9n43UuMjEOSwJ0nh2Awy5qVzdTxajAaflqDz84Vr9zkajcuyrOqnzXyMQcUElglZsFz7fL4wz2EEfi4Lc5JdmoLO+LSOi2ASW4OuEgiq6RbpYQqjrn5J4itBCWPMF9942JxXWQfO3315hgNCN0PCiqhT6ar\\\\\\\/\\\\\\\/ZpQCAgM6uNB34JprqgMggujK59fZoS2CvBMiaCF1TurJZ1p7F2L\\\\\\\/d4DfQG1eBLOQmwxubglkGOLEot6JMDrMVsq4OaPoV\\\\\\\/8Zy9cWE47HM7Z\\\\\\\/LJ3CsjXKgBdEnGgpZ0I8Q1yF6Raq1ml0uM9+EksRtkFKyHEF6oaGKUKU4XjdOPx7b14zsUEaP7NwYmKrVIK77nxlayQAW9\\\\\\\/QW5cfIoJzi56jmDjYUoJjE4\\\\\\\/fnLdp9qcitz2RQAYWzDUIUYqr0B7Qa3b2WBPyDJ3FFYUzwrF6svW8MA==\\\u0022,\\\u0022iv\\\u0022:\\\u00221zAKO47VauUp5vREzkfCDw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002289c20a02d4926b04ea37ad4acac3e03b59785d96976c07d4e4e5954481d11ba9\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022sKMqkPLxDWv1GG+OeUC\\\\\\\/7nAQIm0OHjHcP1Eh4rTAvfK0GYCweEu69lduRn\\\\\\\/8G1PwO4BDSpgJT2hmDPut1uYfxbvmalyos6mut1lQeag8GVezdp5nDkr3dCO\\\\\\\/NSWFCbvuHTK7stWu6PnZ9zGSxw8dve+2zospn1Jx338sKyBC9hWoh13Uul1zMVSd\\\\\\\/83cKsgi7OFiAmY20DTsAIf1tefErqYjdWrpXJCDcEbPnHx2GpmJF+RIXVVPrdlCH7srlDBSYpvsVHVPH\\\\\\\/Cswe3AaAa789H\\\\\\\/0sG2WCWBHgq4AK1veYVbunLE\\\\\\\/jSky0nVNwmubUI8+Wwb3BdgGBRnorZ2IJT0ilXvlgJYZA6wItdhEVHE4NN+eEX2n2yG5ujqWkOFYrLYVGgBRKyl6dc1oY9ydcM3kpO1H3bxAsNVNIx7BlMJLC61OASEi1athDX0m0Oyt2T0gmYALuhZSxTFQYz4xnRohZI5ZdYCvH3l7QWu1lUMZAu4zA2CXjD1bHZ3Swntu8iB4ftVq6pdcQT1IVbr\\\\\\\/Ps79x7C48PxTkaIJChKTNJQ5J6clzfa0TKPspu859Ec44WX+42xHva5UXvH7clGXqzLvPIsRmqME1l\\\\\\\/165S5Qlp454wZVWcJvGmHtThAweDR7UcTSNbCHej\\\\\\\/SolPlukaLlfei\\\\\\\/k0TGWaT1qeLVIKS9\\\\\\\/OQkO6GA+yKO+2ouQ9KWBpM98Lb5lEaKtJoH858ywDVH\\\\\\\/CVL5Bpt2nyXlLp7HlKwxzrtu9vw83INq\\\\\\\/asMG\\\\\\\/CpTw9GhH2hrO836DRlFnlzZF0sq1Iliis398SwyWffjI64AMKw0CfBuT2Xyqbn1GM\\\\\\\/+d+SNB2weQMSxi4jJPK+\\\\\\\/tCQeA\\\\\\\/zSV+L9+m3YMFRR3kKbN0FJH7sNw2yKIt4GA+HN\\\\\\\/Qypce2AnVIome0I5Eg9klb+PJdSJuS9YcqRC5cnRz0naB4\\\\\\\/fwakNQH2h8uO\\\\\\\/RO4+hhxe7YEXfwrEoSkAQVgFOXEHI7x8VGVVyxZndJMJHRHzS5H7fBPiPW3CZzhmLyo8lhdoLbNlwPnA5+USy41O\\\\\\\/kHjDJBhravOIXsbiJGcf\\\\\\\/\\\\\\\/du2ZEwh0DPjba4+ZWIntf9Qd2NeKQm5VePx6sdBuWi8hfCEJ7elpi7jj7E9W22b3pwP4KO1jAyGwnXmEXajBtY9voCFEtCLZ7BhWbfaN1mPaTb521dXWu5Fh8A6yYZ\\\\\\\/9G0HT2WiarUpL0Y6wdcmLkgc65NYI5pDdJkbIZOqmjSkWaQt71HTRJQ67DzeVbO2pRAUwJfiVWdzC9lqbnWd0K5XanAJ1G5Cdp2RZegseRYpw9Uow37hJIrD8XoAFZVaFAzZYxXXlWwGiezgkJZKMQy70NlOcR5DtuZW0zYOcsGx\\\\\\\/MFepgq4QBrycfgc37QJvVSP+JezIuC1n+iawGsnpTAIFXygfTmaYyAjJxFhVNNkX3dMpacSE0lvWLH5pDzVjcx0DETZwJtyXciGVpZLvn6ECqv2SP8122DA6NVSyOlNAvfT+aHZEgdF\\\\\\\/an4g2Xp+11\\\\\\\/uN+pvs2Tq6ouLIT2FJ4BabtVx8xLFdZioeWn5+H5U5+6HtMpF1BowDSF0CuUNYxgEEVIRj03eQF2RCt+6Aym4P2XrCGjVoG872Z7rx790ns2dvo4P2QoN57xnToiZbwfd5MePMyu8hmasy8JLNUg0XmbUyzvT\\\\\\\/ww3C0XZz40NJycaj9qgrv+LM5tj+8CUBHOQjoCYO4aJuduFnU2MBZfTF4bBv7NJBROeo2iy\\\\\\\/XDN5B40gx3xf1Ork584WbZuNAsL1OzBEE09I2sByRavx2TRDxwmbmuyntUA3sy1TyFWBHc89N6XX2r8c8VSjTA5aOIotGvTIedxG0kz4ah40TqsLXmMcrKzDz+Gn3B4GfoqwTX6KaYQBFZVom1qIyrHmyOhRIEDWKsWGFv6zdfyCJbCpLCv10Z6xj5f15LKBkSGkXwPiSLFJifkx7o8s6YK9Kbg4K5eHHGc6L4NTQC0s3mPlvLCty6m56776SYQreaCQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022oYsiGo8O8rQRgqDXkZssgw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022722dd59a7377ebc5e3ee48d83a068f3712473923556d0648902af8746c65f531\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459792 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/3/385" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_Kb0VQca5a-SWTo2GDTXdAqhQGZ915_-XVF-6_tIy5c" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/3/385" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-nWtA1dgDNA5o5WonhjHxclHa9Cgw5qne6CZfw2JUOis" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/3/371" title="The Evolving Puzzle of Autosomal Versus Y-linked Male Determination in Musca domestica" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/3/399" title="Genetic Markers for Western Corn Rootworm Resistance to Bt Toxin" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459792" id="node459792" data-pisa="ggg;5/3/385" data-pisa-master="ggg;g3.114.016501" data-apath="/ggg/5/3/385.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid and Inexpensive Whole-Genome Genotyping-by-Sequencing for Crossover Localization and Fine-Scale Genetic Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7240-4889" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Beth A. Rowan</span>, <span class="highwire-citation-author" data-delta="1">Vipul Patel</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2114-7963" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Detlef Weigel</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5512-0443" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Korbinian Schneeberger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">385-398;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016501 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459792" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Beth A. Rowan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology, Max Planck Institute for Developmental Biology, 72076 Tübingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beth%2BA.%2BRowan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rowan%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABeth%2BA.%2BRowan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7240-4889" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Beth A. Rowan</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vipul Patel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Developmental Biology, Max Planck Institute for Plant Breeding Research, 50829 Cologne, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vipul%2BPatel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Patel%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVipul%2BPatel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Detlef Weigel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology, Max Planck Institute for Developmental Biology, 72076 Tübingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Detlef%2BWeigel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weigel%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADetlef%2BWeigel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2114-7963" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Detlef Weigel</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:weigel@weigelworld.org" class="" data-icon-position="" data-hide-link-title="0">weigel@weigelworld.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Korbinian Schneeberger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Developmental Biology, Max Planck Institute for Plant Breeding Research, 50829 Cologne, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Korbinian%2BSchneeberger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schneeberger%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKorbinian%2BSchneeberger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5512-0443" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Korbinian Schneeberger</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/3/385" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459792" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459792/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/3/385.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459792" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459792/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/3/385.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459792" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459792/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/3/385.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459792" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459792/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/3/385.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459792" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459792/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The reshuffling of existing genetic variation during meiosis is important both during evolution and in breeding. The reassortment of genetic variants relies on the formation of crossovers (COs) between homologous chromosomes. The pattern of genome-wide CO distributions can be rapidly and precisely established by the short-read sequencing of individuals from F<sub>2</sub> populations, which in turn are useful for quantitative trait locus (QTL) mapping. Although sequencing costs have decreased precipitously in recent years, the costs of library preparation for hundreds of individuals have remained high. To enable rapid and inexpensive CO detection and QTL mapping using low-coverage whole-genome sequencing of large mapping populations, we have developed a new method for library preparation along with Trained Individual GenomE Reconstruction, a probabilistic method for genotype and CO predictions for recombinant individuals. In an example case with hundreds of F<sub>2</sub> individuals from two <em>Arabidopsis thaliana</em> accessions, we resolved most CO breakpoints to within 2 kb and reduced a major flowering time QTL to a 9-kb interval. In addition, an extended region of unusually low recombination revealed a 1.8-Mb inversion polymorphism on the long arm of chromosome 4. We observed no significant differences in the frequency and distribution of COs between F<sub>2</sub> individuals with and without a functional copy of the DNA helicase gene <em>RECQ4A</em>. In summary, we present a new, cost-efficient method for large-scale, high-precision genotyping-by-sequencing.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/next-generation-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-next-generation-sequencing" rel="nofollow">next-generation sequencing</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li><li class="kwd"><a href="/keyword/hidden-markov-model" class="hw-term hw-article-keyword hw-article-keyword-hidden-markov-model" rel="nofollow">hidden Markov model</a></li><li class="kwd"><a href="/keyword/genetic-mapping" class="hw-term hw-article-keyword hw-article-keyword-genetic-mapping" rel="nofollow">genetic mapping</a></li><li class="kwd"><a href="/keyword/quantitative-trait" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait" rel="nofollow">quantitative trait</a></li></ul><p id="p-4">Meiotic recombination serves an essential function for sexually reproducing eukaryotes by promoting the formation of physical linkages (chiasmata) between pairs of homologous chromosomes that facilitate segregation and ensure that each gamete receives a proper complement of chromosomes (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hall 1972</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hawley 1988</a>). It also has a potential consequence for evolution because genetic material is exchanged reciprocally between the two parental chromosomes, allowing new combinations of alleles to be passed on to the next generation (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Barton and Charlesworth 1998</a>). These sequence exchanges require crossovers (COs) that occur as one of several possible outcomes of the repair of programmed double-strand breaks (DSBs) that are initiated at the beginning of meiosis [for recent reviews on this subject, see <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Phadnis <em>et al.</em> (2011)</a> and <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Baudat <em>et al.</em> (2013)</a>]. Although the number of DSBs vary over three orders of magnitude among taxa (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Weiner and Kleckner 1994</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Terasawa <em>et al.</em> 1995</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Moens <em>et al.</em> 1997</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Anderson <em>et al.</em> 2001</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Buhler <em>et al.</em> 2007</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Chelysheva <em>et al.</em> 2007</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Sanchez-Moran <em>et al.</em> 2007</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Vignard <em>et al.</em> 2007</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Mets and Meyer 2009</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Nottke <em>et al.</em> 2011</a>), the number of COs is typically limited to only one or two per chromosome pair per meiosis (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Anderson <em>et al.</em> 1999</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">2003</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Pardo-Manuel De Villena and Sapienza 2001</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Moens <em>et al.</em> 2002</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Sanchez-Moran <em>et al.</em> 2002</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Rosu <em>et al.</em> 2011</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Salomé <em>et al.</em> 2011a</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Segura <em>et al.</em> 2013</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Barakate <em>et al.</em> 2014</a>). It remains unclear how the proportion of DSBs that are resolved as COs is determined.</p><p id="p-5">The distribution of COs across chromosomes is nonrandom, and both “hotspots” and “coldspots” have been documented [reviewed in <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Henderson (2012)</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Paigen and Petkov (2010)</a>; and <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Pryce and McFarlane (2009)</a>]. In plants and fungi, COs tend to occur in regions of open chromatin (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Mancera <em>et al.</em> 2008</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Choi <em>et al.</em> 2013</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Wijnker <em>et al.</em> 2013</a>). Because it has traditionally been laborious and expensive to compare the exact genome-wide distributions of COs, transgenic recombination reporters have been used as an alternative, with the disadvantage that the number of genomic regions that can be examined is limited (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Melamed-Bessudo <em>et al.</em> 2005</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Francis <em>et al.</em> 2007</a>). With advances in high-throughput sequencing, one can easily obtain large amounts of whole-genome resequencing data (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Mardis 2008</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Sboner <em>et al.</em> 2011</a>), but the costs of library production and sequencing are often a barrier to studying hundreds of individuals (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). This has led to the development of several methods for genotyping-by-sequencing (GBS), where reduced representation of the genome is used to rapidly genotype a large number of individuals. Because these methods rely on markers near specific restriction endonuclease sites (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Baird <em>et al.</em> 2008</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Andolfatto <em>et al.</em> 2011</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Elshire <em>et al.</em> 2011</a>), they yield poorer resolution at CO positions than whole-genome sequencing.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465742/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465742/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Costs of library preparation for 96-sample multiplexing with small quantities of DNA</span><div class="sb-div caption-clear"></div></div></div><p id="p-7">Here, we present a GBS method in which the full genomes of hundreds of individuals can be resequenced inexpensively. Our efforts resulted in a wet-lab and analytical pipeline that can precisely detect CO positions even when coverage is sparse and be used for about a seventh of the cost of commercial whole-genome sequencing kits (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). We used this pipeline to investigate whether RECQ4A, whose homologs SGS1 and BLM help resolve recombination intermediates in yeast and humans [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hartung <em>et al.</em> 2007</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Holloway <em>et al.</em> 2010</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">De Muyt <em>et al.</em> 2012</a>; reviewed in <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Knoll and Puchta (2011)</a>], affects the distribution and frequency of COs in <em>Arabidopsis thaliana</em>. Yeast <em>sgs1</em> mutants exhibit elevated COs and abnormal meiotic progression (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Rockmill <em>et al.</em> 2003</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Jessop <em>et al.</em> 2006</a>), and introduction of the <em>A. thaliana RECQ4A</em> protein partially restores the meiotic defects of yeast <em>sgs1</em> mutants (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bagherieh-Najjar <em>et al.</em> 2005</a>). The frequency of somatic COs is two- to sevenfold greater in <em>recq4a</em> mutants of <em>A. thaliana</em> (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Bagherieh-Najjar <em>et al.</em> 2005</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Hartung <em>et al.</em> 2007</a>). Although RECQ4A localizes to future sites of COs during meiosis, it does not necessarily seem to be required for CO formation, but rather to resolve telomeric bridges that arise in meiotic cells (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Higgins <em>et al.</em> 2011</a>). Such evidence both for and against a role for <em>recq4a</em> in CO formation prompted us to further investigate its role during meiosis.</p><p id="p-8">Because the CO landscape is important for the genetic mapping of quantitative trait loci (QTL) (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Xu <em>et al.</em> 2005</a>), we also examined the fine-scale locations of COs and performed QTL mapping of flowering time in wild-type and <em>recq4a</em> mutant F<sub>2</sub> populations. This examination revealed an extended region almost devoid of apparent CO events, which led us to uncover a 1.8-Mb inversion distinguishing the genomes of the parental lines.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant growth</h3><p id="p-9">Seeds were stratified for 4–7 d in 0.1% (w/v) agarose in water at 4°. All plants were grown in soil in 14-pot trays at 23° on 16-hr light/8-hr dark cycles at 65% relative humidity. We limited the number of plants to 12 per pot by sowing seeds in 12 positions and thinning the plants after 1 wk of growth by selecting only the top-most plant in a single sowing position (regardless of size or appearance). Plants were covered with a clear plastic dome for 1 wk after sowing. Crosses were performed using wild-type Ws-2 or <em>recq4a-1</em> (Ws-2 background) as the female parent and wild-type Col-0 or <em>recq4a-4</em> (Col-0 background) as the male parent (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/016501SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS1.pdf">Figure S1</a>). The resulting F<sub>1</sub> plants were allowed to self-fertilize, and the resulting F<sub>2</sub> plants were grown in four staggered replicate experiments. A fifth replicate for <em>MAF4</em> genotyping was grown separately.</p></div><div id="sec-3" class="subsection"><h3>Flowering time measurements</h3><p id="p-10">Beginning at 14 d after sowing, all plants among the four replicates were observed once per day until all plants flowered. The plants for <em>MAF4</em> genotyping were only observed every other day. When the inflorescence shoot reached the height of 1 cm, the number of rosette leaves and the number of days after sowing were recorded. Statistical analyses of flowering time data were performed using R version 3.0.2 (<a href="http://www.r-project.org">http://www.r-project.org</a>).</p></div><div id="sec-4" class="subsection"><h3>DNA extraction and quantification</h3><p id="p-11">For the Ws-2 parent, DNA was isolated from nuclei following the method described in <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Cao <em>et al.</em> (2011)</a>. For the F<sub>2</sub> plants, 192 individuals from replicate number four were selected from each population and the tissue was frozen at −80° in 96-well block of 1.3-mL tubes with along with a 4-mm-diameter metal bead. The tissue was disrupted using a QIAGEN Mixer Mill (QIAGEN, Hilden, Germany) at 20 vibrations/sec for 2 min. DNA extraction was initiated by adding 500 µL of CTAB buffer [2% (w/v) hexadecyltrimethylammonium bromide, 1% (w/v) sodium bisulfite, 0.1 M Tris-HCl, 1.4 M NaCl, 20 mM ethylenediaminetetraacetic acid, pH 8] and incubating at 65° for 1 hr, followed by a flash spin before adding 500 µL of 24:1 chloroform:isoamyl alcohol. Samples were gently rocked for 10 min before centrifuging at 6000 × <em>g</em> in a Sigma 4K15C plate centrifuge (Sigma-Aldrich, St. Louis, MO) for 30 min. The upper phase was transferred to a 96-well plate containing 0.7 volumes isopropanol and incubated for 20 min at 4°, then centrifuged at 6000 × <em>g</em> in a Sigma 4K15C plate centrifuge for 20 min before a final wash with 70% ethanol and centrifugation at 6000 × <em>g</em> in a Sigma 4K15C plate centrifuge for 30 min. The ethanol supernatant was decanted and the DNA was allowed to dry completely before resuspension in MilliQ (Merck Millipore, Billerica, MA) water. Since we found that spectrophotometric measurements of DNA concentration were inaccurate for DNA extracted using the CTAB method, we used the Qubit 2.0 fluorometer with the high-sensitivity (HS) DNA quantification reagents (Invitrogen, Carlsbad, CA) to measure the DNA before normalizing all samples to approximately 100 ng before library preparation.</p></div><div id="sec-5" class="subsection"><h3>Library preparation</h3><p id="p-12">For each sample, 100 ng of DNA were digested with 0.5 units of dsDNA Shearase (Zymo Research, Orange, CA) for 30 min at 37° in a total reaction volume of 30 µL. The reactions were stopped by adding ethylenediaminetetraacetic acid to a final concentration of 50 mM and then the DNA fragments were cleaned up using AMPure XP (Beckman-Coulter, Brea, CA) solid-phase reversible immobilization (SPRI) magnetic beads according to the manufacturer’s recommendations, except that the wash steps with 70% EtOH were performed with 200 µL and the samples were eluted in 18 µL of EB buffer (QIAGEN) during the final step. Eleven samples from each 96-well-plate were chosen for analysis of fragment size distribution using the Bioanalyzer 2100 Desktop System (Agilent Technologies, Santa Clara, CA) with the HS Kit (an example is given in <a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS2.pdf">Figure S2</a>). The samples were then A-tailed at 37° for 30 min in a total volume of 25 µL using the Klenow exo-enzyme with 1X NEB Buffer 2, and 0.2 mM dATP (New England Biolabs, Ipswich, MA). The reactions were cleaned up using AMPure XP (Beckman-Coulter) magnetic SPRI beads according to the manufacturer’s protocol, except 70% EtOH washes were performed with 200 µL and the samples were eluted in 11 µL of 0.6X EB buffer (QIAGEN) without transferring the eluate to a fresh tube and leaving the SPRI beads in the reactions.</p><p id="p-13">The A-tailed fragments served as input into reactions with 0.4 µM custom adapter mix (barcoded version of Illumina P1 adapter and standard Illumina P2 adapter, see <a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/TableS1.pdf">Table S1</a>), 1X ligation buffer, and 0.5 µL of Quick Ligase (New England Biolabs) and incubated for 15 min at 20°, followed by heat inactivation for 5 min at 65°. The custom adapters used in this step were created by mixing complementary single-stranded oligonucleotides at a concentration of 10 µM in 1X NEB Buffer 2 (New England Biolabs) and incubating in a water bath at 100° that was left to cool to room temperature over several hours. After the ligation step was completed, eight samples were pooled into a tube and concentrated using the AMPure XP (Beckman-Coulter) magnetic SPRI beads remaining in the tube according to the manufacturer’s protocol, except that 1.8 volumes of buffer containing 2.5 M NaCl and 8 mM polyethylene glycol was used in place of more SPRI beads and the samples were eluted in 30 µL EB buffer (QIAGEN) during the final step.</p><p id="p-14">To ensure even representation of all samples, an optional quantitative polymerase chain reaction (PCR)-based quantification step can be performed using commercial reagent sets developed for your quantitative PCR system before pooling the samples together and performing size selection. For size selection, 15 µL of each eight-sample mix were combined into a single tube and 0.6 volumes of AMPure XP (Beckman-Coulter) magnetic SPRI beads were added to the tube. The samples were placed in a magnetic rack and allowed to clear. The supernatant was transferred to a fresh tube, and 0.2 volumes of AMPure XP (Beckman-Coulter) magnetic SPRI beads were added. The tubes were placed in a magnetic rack and allowed to clear before the supernatant was discarded, and the beads were washed twice with 70% ethanol and allowed to dry for 15 min at 37°. The libraries were then eluted according to the manufacturer’s procedure in a final volume of 30 µL of EB buffer (QIAGEN) and 3−15 µL was used in 50-µL PCRs with 0.2 mM standard Illumina primers and either the Phusion mastermix or the Phusion enzyme alone with 10X High Fidelity Buffer (all from New England Biolabs) and 0.2 mM dNTPs (Thermo Scientific Life, Waltham, MA). Reactions were incubated for 3 min at 72° and 30 sec at 98°, followed by 12 cycles of 10 sec at 98°, 30 sec at 65°, and 30 sec at 72° before a final incubation step at 72° for 5 min using an ABI GeneAmp PCR system 9700 (Applied Biosystems, Foster City, CA). Reactions were cleaned up using AMPure XP (Beckman-Coulter) SPRI magnetic beads according to the manufacturer’s recommendations, except that the samples were eluted in 30 µL of EB buffer (QIAGEN) in the final step. Final libraries were validated by quantification using the Qubit 2.0 fluorometer with the HS DNA quantification reagents (Invitrogen) and using the Bioanalyzer 2100 Desktop System with the DNA1000 Kit (Agilent Technologies). The library for the Ws-2 parent was prepared similarly, except 400 ng of DNA was used and all volumes and all library reagents were scaled up by a factor of two.</p></div><div id="sec-6" class="subsection"><h3>Sequencing and initial data processing</h3><p id="p-15">Each 96-plex library for the F<sub>2</sub> individuals was sequenced on an Illumina GAIIx analyzer (Illumina, San Diego, CA) in one flow cell lane using 2 × 150-bp length paired-end reads. Base calling was performed using the standard Illumina software to generate the raw reads, which were then filtered, demultiplexed, aligned to the TAIR10 reference genome for detection of sequence polymorphisms using the SHORE and GenomeMapper software (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Ossowski <em>et al.</em> 2008</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Schneeberger <em>et al.</em> 2009</a>). Final average coverage for each 96-plex lane was 99.3x. The Ws-2 parent was resequenced similarly to a coverage depth of 25x.</p><p id="p-16">To reduce genotyping errors that might arise from poor quality markers, we applied several stringent marker-filtering steps. Using the high-coverage resequencing data, we found 840,611 single-nucleotide polymorphisms (SNPs) between Ws-2 and Col-0 (TAIR10). After removing the SNPs located in the genomes of the mitochondrion and chloroplast and those close to insertion-deletion polymorphisms, we found that 745,273 SNPs remained. An additional 238,111 SNPs were removed after considering only those that were of high quality and supported exclusively by uniquely aligned reads. We further decreased the marker number to 302,082 after applying filtering for homozygous regions, transposons and coverage (accepting only those SNPs in which the local coverage was within two standard deviations of the mean genome-wide coverage). Finally, we used the segregation patterns of our markers in the F<sub>2</sub> populations to remove 40,287 SNPs that did not show a Mendelian pattern of inheritance to obtain a final set of 261,795 markers (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS4.pdf">Figure S4</a>).</p></div><div id="sec-7" class="subsection"><h3>Genotyping at individual marker positions</h3><p id="p-17">We obtained the read counts for each individual at each of the SNP markers that passed our strict filters and measured the ratio of counts supporting the Col-0 or Ws-2 allele. We transformed these continuous ratios into a discrete specified alphabet including six states, which are AA, AU, AB, BU, BB, and UU. AA stands for homozygous Col-0, BB for homozygous Ws-2, AB for heterozygous states, U for the uncertainty to be called homozygous, and UU for no information. To transform the read counts into the alphabet states, we used a threshold of five reads supporting only one parental allele for assigning the homozygous genotypes (AA or BB) because a heterozygous position covered by five reads has less than a 5% chance of presenting only one allele. For genotypes in which both of the parental alleles were observed or less than five reads were aligned, we calculated the probability of a certain read count ratio to result from homozygous or heterozygous genotypes by using a multinomial distribution. We assumed that observing reads in a homozygous background (x<sub>1</sub>, x<sub>2</sub>) would follow a binomial distribution, where the probability for observing the parental allele is 99% (considering 1% sequencing errors). For a heterozygous background (f(a,b)) the probabilities for drawing reads from both parental genotypes would be equal to <em>P</em> = 0.5. We assigned the genotypes according to the maximum value between the homozygous (max(x<sub>1</sub>,x<sub>2</sub>)) and heterozygous probabilities (f(a,b)). If the maximum value was obtained for homozygous, we assigned the genotype AU (if x<sub>1</sub> &gt; x<sub>2</sub>) or BU (if x<sub>1</sub> &lt; x<sub>2</sub>); otherwise, it was labeled AB. The probabilities for drawing one of the parental genotypes (x<sub>1</sub> or x<sub>2</sub>) in the homozygous background and the probabilities to draw them together coming from a heterozygous distribution (f(a,b)) have been estimated as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/385/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/385/embed/mml-math-1.gif"/></noscript></span></span>where <em>a</em> and <em>b</em> describe the read counts for the respective parental alleles.</p></div><div id="sec-8" class="subsection"><h3>State model of the hidden Markov modeling (HMM) and parameter estimations</h3><p id="p-18">Our HMM consisted of three hidden nodes (AA, AB, and BB), which reflected the three possible genotypes (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS5.pdf">Figure S5</a>). In our model, all hidden nodes were connected, each of them featuring six emission states representing the observed alphabet of genotypes (see <em>Genotyping at individual marker positions</em> section above.).</p><p id="p-19">After transformation of the read counts into our alphabet, transition and emission probability estimations were performed for each sample separately. We first estimated local allele frequencies along all chromosomes by applying a simple sliding window approach that estimates the local allele frequency for 1000 adjacent markers and reduces the noise in uncovering the distribution of the read counts. Although this could be used to assign genotypes to all markers (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Huang <em>et al.</em> 2009</a>), this does not allow for an accurate resolution of CO breakpoints. Ideally, the distribution of allele frequencies within the sliding windows should reflect a Mendelian distribution (<em>i.e.</em>, allele frequencies of 0, 0.5, and 1). However, due to the effects of random sampling, sequencing errors misalignments, and residual parental allele bias that may remain after marker filtering, the distribution is skewed. We fit three different beta curves to the observed allele frequency estimations, representing three different underlying allele frequency distributions introduced by the three different genotypes using a beta-mixture model with an expectation-maximization algorithm (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS6.pdf">Figure S6</a>). The beta-mixture model approach is adapted from <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ji <em>et al.</em> (2005)</a>, where the technique was introduced for identifying correlation of gene expression between different experiments. After fitting the three beta curves, we labeled each of the underlying allele frequency under each curve according to the expected genotype. The area under the curves is either limited by 0.1 or by the intersection of the neighboring beta curve. We then applied supervised learning strategy to obtain our probabilities by combining the labeled allele frequencies and our previously genotyped labels.</p></div><div id="sec-9" class="subsection"><h3>Simulations and reconstructions of the F<sub>2</sub> populations</h3><p id="p-20">To validate our pipeline, we simulated a mapping population with 5000 samples sequenced at three different coverage levels with a simulated sequencing error rate of 1%. Genotypes were generated at 261,795 high-quality SNP markers with the default recombination landscape from the Pop-seq software (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Salomé <em>et al.</em> 2011a</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">James <em>et al.</em> 2013</a>).</p></div><div id="sec-10" class="subsection"><h3>Prediction and validation of predicted CO and inversion breakpoints</h3><p id="p-21">Structural variants in Ws-2 relative to the Col-0 reference were predicted using Pindel (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Ye <em>et al.</em> 2009</a>). For validation, we PCR amplified 1−2 kb of sequence spanning the predicted CO breakpoints. For the Ws-2 chromosome 4 insertion, we amplified ~500 bp to 1 kb around the predicted breakpoints. Genomic DNA was PCR amplified using reactions containing 0.4 µM forward and reverse primers, 0.2 µM dNTPs, 1x Phusion High Fidelity Buffer, and 0.2 units Phusion Taq (New England Biolabs) along with template DNA and incubated in an MJ Research PTC-200 or PTC-225 thermal cycler (Hercules, CA) at 94° for 2 min, followed by 37 cycles of 20 sec at 94°, 20 sec at 50−60°, and 30 sec at 72° with a final extension step at 72° for 5 min. Primer sequences are provided in <a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/TableS2.pdf">Table S2</a>. We sequenced the amplified regions in the Col-0 and Ws-2 parents and the F<sub>2</sub> individuals with the predicted CO in sequencing by assembling reactions containing 1 µM primer, 0.5 µL of BigDye Terminator ready reaction mix, and 1X sequencing buffer and incubating for 20s at 96°, followed by 29 cycles of 10 sec at 50° and 4 min at 60° before analyzing on an ABI 3730xL sequencer.</p></div><div id="sec-11" class="subsection"><h3>QTL analyses of flowering time</h3><p id="p-22">QTL analyses using the genotype data from marker blocks designated from the F<sub>2</sub> genome reconstructions were performed using the R/qtl package for R version 3.0.2 (<a href="http://www.r-project.org/">http://www.r-project.org/</a>) designating the Kosambi map function and the expectation-maximization algorithm for all analyses. To determine significance thresholds for LOD scores, we performed 1000 permutations and set our threshold at an alpha of 0.05.</p></div><div id="sec-12" class="subsection"><h3>MAF4 genotyping</h3><p id="p-23">We designed a cleaved, amplified polymorphic sequence marker (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Konieczny and Ausubel 1993</a>) using the Sol Genomics web interface (<a href="http://solgenomics.net/tools/caps_designer/caps_input.pl">http://solgenomics.net/tools/caps_designer/caps_input.pl</a>) based on the position 25995356, where the Ws-2 accession has a G nucleotide compared with the Col-0 reference A nucleotide that introduces a recognition site for the restriction enzyme <em>Sty</em>I. We genotyped 90 F<sub>2</sub> individuals from each population (wild type and <em>recq4a</em>) using PCRs containing 0.4 µM forward and reverse primers, 0.2 µM dNTPs, 1x Phusion High Fidelity Buffer, and 0.2 units Phusion Taq (New England Biolabs). Reactions were incubated in an MJ Research PTC-200 or PTC-225 thermal cycler (Bio-Rad, Hercules, CA) at 94° for 2 min, followed by 37 cycles of 20 sec at 94°, 20 sec at 56°, and 30 sec at 72° and a final extension step at 72° for 5 min. Primer sequences are provided in <a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/TableS2.pdf">Table S2</a>.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>Rapid production of low-cost indexed libraries for paired-end-whole-genome sequencing</h3><p id="p-24">We developed a high-throughput method for preparing paired-end sequencing libraries for the Illumina platform that allows for multiplexing up to 96 individual DNA samples without the use of expensive sample preparation kits or specialized DNA fragmentation equipment, and with starting amounts of DNA as little as 100 ng per sample (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). Compared with the closest commercial method, Illumina TruSeq Nano, our protocol produced libraries at about one-seventh of the cost (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, <a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/TableS3.pdf">Table S3</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/385/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Library preparation workflow and sequencing coverage results. (A) Comparison of our protocol for the rapid production of paired-end libraries for whole-genome sequencing with Illumina TruSeq Nano protocol. &#x201C;A&#x201D; indicates size selection step, &#x201C;B&#x201D; indicates quantification/normalization step, and &#x201C;C&#x201D; indicates pooling step. * indicates an optional step. (B) Coverage distribution for reads generated from a single DNA library prepared for the A. thaliana accession Ws-2 using our high-throughput method and mapped against the Col-0 TAIR10 reference genome (including repetitive alignments). The average coverage in 5-kb bins is shown and the maximum coverage value has been capped to exclude the top 0.1% of average counts for each chromosome in order to compare all chromosomes at the same scale. Red circles indicate bins in which the average coverage was less than 1x. The genome-wide average depth of coverage was 25.8x. (C) The average representation of reads assigned to a specific index sequence over four separate multiplexed pools." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1587840381" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Library preparation workflow and sequencing coverage results. (A) Comparison of our protocol for the rapid production of paired-end libraries for whole-genome sequencing with Illumina TruSeq Nano protocol. &#x201C;A&#x201D; indicates size selection step, &#x201C;B&#x201D; indicates quantification/normalization step, and &#x201C;C&#x201D; indicates pooling step. * indicates an optional step. (B) Coverage distribution for reads generated from a single DNA library prepared for the &lt;em&gt;A. thaliana&lt;/em&gt; accession Ws-2 using our high-throughput method and mapped against the Col-0 TAIR10 reference genome (including repetitive alignments). The average coverage in 5-kb bins is shown and the maximum coverage value has been capped to exclude the top 0.1% of average counts for each chromosome in order to compare all chromosomes at the same scale. Red circles indicate bins in which the average coverage was less than 1x. The genome-wide average depth of coverage was 25.8x. (C) The average representation of reads assigned to a specific index sequence over four separate multiplexed pools.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/385/F1.medium.gif" width="202" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/3/385/F1.medium.gif" width="202" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/385/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/385/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465674" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-25" class="first-child">Library preparation workflow and sequencing coverage results. (A) Comparison of our protocol for the rapid production of paired-end libraries for whole-genome sequencing with Illumina TruSeq Nano protocol. “A” indicates size selection step, “B” indicates quantification/normalization step, and “C” indicates pooling step. * indicates an optional step. (B) Coverage distribution for reads generated from a single DNA library prepared for the <em>A. thaliana</em> accession Ws-2 using our high-throughput method and mapped against the Col-0 TAIR10 reference genome (including repetitive alignments). The average coverage in 5-kb bins is shown and the maximum coverage value has been capped to exclude the top 0.1% of average counts for each chromosome in order to compare all chromosomes at the same scale. Red circles indicate bins in which the average coverage was less than 1x. The genome-wide average depth of coverage was 25.8x. (C) The average representation of reads assigned to a specific index sequence over four separate multiplexed pools.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">We began library production with an enzymatic fragmentation step that generated a size range suitable for 300- to 500-bp insert libraries. Unlike restriction endonucleases, which reduce representation of the genome to tens of thousands of markers (or less), the recognition sites for the commercial enzyme formulation we used (dsDNA Shearase, Zymo Research) occur at 50,831,349 sites, or at almost every other base, across the <em>A. thaliana</em> genome (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS7.pdf">Figure S7</a>). This allowed for the detection of hundreds of thousands of markers. To test whether fragmentation by dsDNA Shearase introduces a biased representation of the genome and to generate markers for the Ws-2 parent of our mapping populations, we scaled up the same protocol to produce libraries for Ws-2 whole genome resequencing to an average depth of about 25x. We found that the average coverage of reads that were aligned to the Col-0 reference genome was highest along the chromosome arms and lowest at the five centromeres (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). There were only a few 5-kb bins in which the average coverage was less than one read, suggesting no substantial bias in the representation of the genome.</p><p id="p-27">The fragmentation and clean-up (using magnetic beads for SPRI) for 96 samples processed simultaneously required less than one and a half hours total time. Although the fragment size distribution among samples was not completely uniform (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS2.pdf">Figure S2</a>), the quantity of fragments in the targeted range of 300−500 bp was suitable for further processing. For A-tailing and ligation to custom-indexed adapters, we used standard enzymes (New England Biolabs). After the ligation step, we combined the 96 samples into a single pool without normalization, performed simultaneous size selection using SPRI beads, and enriched the single, multiplexed library by PCR. If a more even representation of all samples is desired, a normalization step can be included before pooling (note that this adds about $1−3 per sample, depending on the quantitative PCR platform and reagents needed for quantifying the libraries before pooling). The complete protocol (without the normalization step) requires only about a day and a half of laboratory work for 96 paired-end libraries starting from high molecular weight DNA.</p></div><div id="sec-15" class="subsection"><h3>Low-coverage sequencing of hundreds of <em>A. thaliana</em> F<sub>2</sub> individuals</h3><p id="p-28">To assess the meiotic role of <em>RECQ4A</em>, we generated two F<sub>2</sub> populations: one derived from a cross between wild-type Col-0 and Ws-2 accessions and the other from the same parents, but with T-DNA insertions into <em>RECQ4A</em> that reduced gene function (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS8.pdf">Figure S8</a>). We randomly selected 192 individuals from each population (1113 for wild type, and 1052 for <em>recq4a</em>, divided over four replicates), produced two 96-plex libraries (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS1.pdf">Figure S1</a>), and sequenced each pool in a single lane of an Illumina GAIIx sequencer (Illumina). Our custom adapters contained an 8-bp index sequence with a T overhang (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS3.pdf">Figure S3</a>). Without a normalization step, variation in the read counts for each index is most likely due to the molar representation of each library in the pool. Thus, we expect that any effect of the index sequence on ligation ability, PCR amplification, or cluster generation would be manifested as poor representation of that index sequence in all four pooled libraries. We compared the read counts for all 96 indexes across all four lanes (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>) and note that only three indices (GAACAGGC, GACAGTGC, ACACAGAA) were poorly represented in all lanes. The average number of reads per lane was almost 90 million, or an average of 1x genome-wide sequencing depth per sample. Almost half of the reads could be uniquely aligned to the reference genome (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS9.pdf">Figure S9</a>). The bias, if any, introduced by the indices was thus negligible.</p></div><div id="sec-16" class="subsection"><h3>GBS with sparse coverage using Trained Individual GenomE Reconstruction (TIGER)</h3><p id="p-29">GBS with sparse coverage is difficult because of the limited information for assigning genotypes to a region. Tools developed for the field of human genomics (IMPUTE2, BEAGLE; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Browning and Browning 2009</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Howie <em>et al.</em> 2009</a>) are designed for imputing haplotypes as found among individuals of natural populations and use internal parameters (recombination landscape or CO frequency) defined for human data. <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Andolfatto <em>et al.</em> (2011)</a> and <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Xie <em>et al.</em> (2010)</a> presented the first tools for GBS for artificial populations based on HMMs for sparse read coverage data. We improved their approaches by training the model using all reads at each marker position and avoiding the use of defined constants, implemented in TIGER, a GBS pipeline for reconstructing the mosaic genomes of recombinant individuals from sparse sequence coverage (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>, A−C). We used strict marker filtering, followed by an HMM where transition and emission probabilities were estimated sample-wise using a beta-mixture model on the estimated parental allele frequencies within sliding windows. Allele frequencies were estimated by counting short read alignments at marker positions that supported either of the parental alleles. Because there were only three possible genotypes (parent A, parent B, and heterozygous), we expected three distinguishable curves along a chromosome, each having an independent peak. Two major factors introduced skews in this distribution: the effects of random sampling, and a bias for reference alleles. With the beta-mixture model, we could fit the observed distributions to the expected pattern (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS6.pdf">Figure S6</a>); this information provided the input for the HMM (see the section <em>Materials and Methods</em>). This approach generates sample-specific probabilities that take the sample-specific-error-rate into account, allowing for more accurate genotype predictions from the HMM (see the section <em>Materials and Methods</em>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/385/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotyping by sequencing for sparse coverage sequencing from a biparental mapping population using Trained Individual GenomE Reconstruction (TIGER). The TIGER pipeline is summarized in (A&#x2212;C). (A) Single-nucleotide polymorphisms between the two parents (red and blue) are determined relative to the reference sequence are localized and filtered. (B) The sequencing reads for each sample are aligned against the reference sequence and the read counts for both alleles at the filtered marker position are estimated. The read counts are used to estimate the probabilities for the transition and emission for a Hidden-Markov-Model by using a beta-mixture-model fit. (C) The read count ratios determined in (B) are transformed into an alphabet coding system using the Basecaller module of the TIGER pipeline. This alphabet consists of six states, AA, homozygous parent A (red); BB, homozygous parent B (blue); AB, heterozygous; AU or BU, weakly homozygous; and UU, no information at all. The output from the Basecaller with the outcome from the beta-mixture-model fit is used as input for our HMM which predicts the genotypes using the Viterby algorithm. Afterward we increase the crossover (CO) resolution by incorporating markers near the predicted CO position that were previously filtered out." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1587840381" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genotyping by sequencing for sparse coverage sequencing from a biparental mapping population using Trained Individual GenomE Reconstruction (TIGER). The TIGER pipeline is summarized in (A&#x2212;C). (A) Single-nucleotide polymorphisms between the two parents (red and blue) are determined relative to the reference sequence are localized and filtered. (B) The sequencing reads for each sample are aligned against the reference sequence and the read counts for both alleles at the filtered marker position are estimated. The read counts are used to estimate the probabilities for the transition and emission for a Hidden-Markov-Model by using a beta-mixture-model fit. (C) The read count ratios determined in (B) are transformed into an alphabet coding system using the Basecaller module of the TIGER pipeline. This alphabet consists of six states, AA, homozygous parent A (red); BB, homozygous parent B (blue); AB, heterozygous; AU or BU, weakly homozygous; and UU, no information at all. The output from the Basecaller with the outcome from the beta-mixture-model fit is used as input for our HMM which predicts the genotypes using the Viterby algorithm. Afterward we increase the crossover (CO) resolution by incorporating markers near the predicted CO position that were previously filtered out.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/385/F2.medium.gif" width="368" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/3/385/F2.medium.gif" width="368" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/385/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/385/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465713" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-30" class="first-child">Genotyping by sequencing for sparse coverage sequencing from a biparental mapping population using Trained Individual GenomE Reconstruction (TIGER). The TIGER pipeline is summarized in (A−C). (A) Single-nucleotide polymorphisms between the two parents (red and blue) are determined relative to the reference sequence are localized and filtered. (B) The sequencing reads for each sample are aligned against the reference sequence and the read counts for both alleles at the filtered marker position are estimated. The read counts are used to estimate the probabilities for the transition and emission for a Hidden-Markov-Model by using a beta-mixture-model fit. (C) The read count ratios determined in (B) are transformed into an alphabet coding system using the Basecaller module of the TIGER pipeline. This alphabet consists of six states, AA, homozygous parent A (red); BB, homozygous parent B (blue); AB, heterozygous; AU or BU, weakly homozygous; and UU, no information at all. The output from the Basecaller with the outcome from the beta-mixture-model fit is used as input for our HMM which predicts the genotypes using the Viterby algorithm. Afterward we increase the crossover (CO) resolution by incorporating markers near the predicted CO position that were previously filtered out.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Evaluation of TIGER using simulation studies</h3><p id="p-31">To validate our approach, we simulated three <em>A. thaliana</em> F<sub>2</sub> mapping populations, each containing 1000 samples, with three different read coverage rates (0.1x, 1x, and 10x) using the published PopSeq tool (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">James <em>et al.</em> 2013</a>; <a href="http://sourceforge.net/projects/popseq">http://sourceforge.net/projects/popseq</a>) based on recombination and sequence data from <a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Salomé <em>et al.</em> (2011a)</a>. The 1000 samples were randomly distributed into 10 separate bins, and the number of predicted recombination events, the breakpoint resolution, and the types and genomic positions of errors were determined for each bin with TIGER (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/385/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Evaluation of Trained Individual GenomE Reconstruction (TIGER) on simulated data. The TIGER pipeline was applied to simulated read data from 1000 simulated recombinant individuals for three different coverage rates. For each coverage rate, the samples were subset into 10 bins of 100 individuals and genotypes and crossovers (COs) were predicted using TIGER pipeline within the bins. The first row of plots shows the difference between the expected (simulated) and predicted CO numbers for each coverage rate. The second row of plots shows the resolution of COs in marker space for each of the 10 bins. The x-axes indicate the distance between the predicted and expected CO point based on the number of markers with a false genotype prediction and the y-axes show the percentage of total COs. The same representation is presented in the last row of plots, except the x-axes are measured in physical distances (Mb)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1587840381" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Evaluation of Trained Individual GenomE Reconstruction (TIGER) on simulated data. The TIGER pipeline was applied to simulated read data from 1000 simulated recombinant individuals for three different coverage rates. For each coverage rate, the samples were subset into 10 bins of 100 individuals and genotypes and crossovers (COs) were predicted using TIGER pipeline within the bins. The first row of plots shows the difference between the expected (simulated) and predicted CO numbers for each coverage rate. The second row of plots shows the resolution of COs in marker space for each of the 10 bins. The x-axes indicate the distance between the predicted and expected CO point based on the number of markers with a false genotype prediction and the y-axes show the percentage of total COs. The same representation is presented in the last row of plots, except the x-axes are measured in physical distances (Mb).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/385/F3.medium.gif" width="440" height="339"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/3/385/F3.medium.gif" width="440" height="339"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/385/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/385/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465741" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Evaluation of Trained Individual GenomE Reconstruction (TIGER) on simulated data. The TIGER pipeline was applied to simulated read data from 1000 simulated recombinant individuals for three different coverage rates. For each coverage rate, the samples were subset into 10 bins of 100 individuals and genotypes and crossovers (COs) were predicted using TIGER pipeline within the bins. The first row of plots shows the difference between the expected (simulated) and predicted CO numbers for each coverage rate. The second row of plots shows the resolution of COs in marker space for each of the 10 bins. The x-axes indicate the distance between the predicted and expected CO point based on the number of markers with a false genotype prediction and the y-axes show the percentage of total COs. The same representation is presented in the last row of plots, except the x-axes are measured in physical distances (Mb).</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">We combined the results from 10 bins for each of the simulated coverage rates independently and compared the difference between the predicted and the expected number of COs. The difference between expectation and prediction was always positive, irrespective of coverage, indicating that our approach had the tendency to underestimate COs. As coverage increased, the percentage of expected COs that were predicted increased, from 97.5% at the lowest coverage level (0.1x) to 99.3% at the greatest (10x; <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p><p id="p-34">We estimated the resolution of COs by examining the physical distance and the number of markers between the predicted and the expected CO positions; as expected, resolution improved with increasing coverage (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). We could predict 90% of COs to within 2 kb of their actual position at all coverage levels. For the remaining COs, the distances depended on the distance between the CO and the closest marker that gave definitive information on a change in genotype along the chromosome. This uncertainty in assigning the marker closest to a CO was seven markers for 0.1x coverage, one for 1x, and none for 10x. The average resolution at 0.1x was 1986 bp (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465683/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465683/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">CO breakpoint resolution using simulated data</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">Examining the causes for errors in CO estimation, we found the most dominant error, about 90%, was the erroneous misclassification of heterozygous genotypes as homozygous. Most of these false homozygote prediction errors were in regions located within or next to the centromeres and telomeres. In these error-prone regions, the median false homozygote error rate was 2.4% for 0.1x coverage and approximately 1% for greater coverages (0.9%). The background error rate was 0.1% regardless of coverage (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS10.pdf">Figure S10</a>). Differences in marker density, and errors in marker assignment because of repetitiveness of sequences, may account for the greater error rates near centromeres and telomeres. The marker density in the pericentromeric regions was high and abruptly dropped off at the border of the centromere (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS11.pdf">Figure S11</a>). Importantly, Ws-2 and Col-0 homozygotes represented 42% and 47% of all errors, indicating that this type of error was not biased toward one of the two parental genotypes.</p></div><div id="sec-18" class="subsection"><h3>Reconstructions of wild-type and <em>recq4a</em> F<sub>2</sub> genomes</h3><p id="p-37">We reconstructed the genomes of our F<sub>2</sub> individuals using TIGER (an example is given in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). From the high-coverage resequencing data for Ws-2, we obtained a set of high-quality markers for GBS after applying a strict set of filters (see the section <em>Materials and Methods</em>). We applied an average genome-wide coverage threshold of 0.025x to select individuals for the reconstructions because the accuracy of correct CO breakpoint predictions was strongly reduced below this threshold. We reconstructed the genomes of 110 individuals from the wild-type population and of 106 from the <em>recq4a</em> population; these had average and median coverages of 0.6x and of 0.4x. The overall frequency of Col-0, Ws-2, and heterozygous genotypes in both populations was consistent with a Mendelian pattern of inheritance (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/TableS4.pdf">Table S4</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/385/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome reconstruction and crossover (CO) localization from experimental Ws-2 x Col-0 F2 populations. (A) A graphical example of reconstructions of chromosome 3 for 220 Ws-2 x Col-0 F2 individuals. Each vertical line represents a single individual. Red indicates homozygous for Col-0, blue homozygous for Ws-2 and heterozygous regions are in purple. (B) Histogram of the interval sizes for predicted COs. (C) Schematic representation for validation of CO intervals by PCR (Table 3)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1587840381" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome reconstruction and crossover (CO) localization from experimental Ws-2 x Col-0 F&lt;sub&gt;2&lt;/sub&gt; populations. (A) A graphical example of reconstructions of chromosome 3 for 220 Ws-2 x Col-0 F&lt;sub&gt;2&lt;/sub&gt; individuals. Each vertical line represents a single individual. Red indicates homozygous for Col-0, blue homozygous for Ws-2 and heterozygous regions are in purple. (B) Histogram of the interval sizes for predicted COs. (C) Schematic representation for validation of CO intervals by PCR (Table 3).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/385/F4.medium.gif" width="302" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/3/385/F4.medium.gif" width="302" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/385/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/385/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465745" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-38" class="first-child">Genome reconstruction and crossover (CO) localization from experimental Ws-2 x Col-0 F<sub>2</sub> populations. (A) A graphical example of reconstructions of chromosome 3 for 220 Ws-2 x Col-0 F<sub>2</sub> individuals. Each vertical line represents a single individual. Red indicates homozygous for Col-0, blue homozygous for Ws-2 and heterozygous regions are in purple. (B) Histogram of the interval sizes for predicted COs. (C) Schematic representation for validation of CO intervals by PCR (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">The simulation studies indicated several types of errors (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS10.pdf">Figure S10</a>) produced by our approach, but in addition to the errors anticipated from the simulations, we encountered an error in the experimental F<sub>2</sub> data in which small genotype blocks were embedded within larger blocks of a different genotype. These regions, termed “islands”, could have also represented real recombination events, as closely spaced double COs or gene conversions would have given rise to the same pattern. We used island length to distinguish errors from real recombination events. Of 67 islands, 7 were less than 400-kb long and were removed as errors (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS12.pdf">Figure S12</a>). The remaining 60 islands were categorized as double COs because most gene conversion tracts are typically no longer than a few kilobases (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lu <em>et al.</em> 2011</a>; <a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Wijnker <em>et al.</em> 2013</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Qi <em>et al.</em> 2014</a>). After this additional error correction step, we had the final genome reconstructions to be used for further analyses. As with the simulated data, the CO predictions for the experimental data were not substantially affected by coverage (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS13.pdf">Figure S13</a>). Using linear interpolation on masked markers near the breakpoint sites that had been previously filtered out, we resolved the majority of COs to an interval of less than 2 kb, as expected from the simulations (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). Using PCR and Sanger sequencing, we confirmed that eight of 11 predicted COs had the expected genotype transition (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4C</a> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Nearly all of these confirmed COs occurred within 1.5 kb of the predicted breakpoint, indicating that CO predictions using TIGER are within a few markers of the real CO position.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465755/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465755/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Confirmation of CO positions in wt and <em>recq4a</em> populations</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3><em>RECQ4A</em> does not affect the frequency or distribution of CO events in Col-0 x Ws-2 F<sub>2</sub> populations</h3><p id="p-45">To observe an influence of RECQ4A on meiotic recombination, we first localized the positions of all CO breakpoints in the wild-type and <em>recq4a</em> populations. As has been previously observed (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Copenhaver <em>et al.</em> 1999</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Drouaud <em>et al.</em> 2007</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Giraut <em>et al.</em> 2011</a>; <a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Salomé <em>et al.</em> 2011a</a>), CO frequency increased from telomeres toward the centromeres, with centromeres themselves having very few COs, with little difference between wild-type and <em>recq4a</em> mutants (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). <em>recq4a</em> mutants had on average slightly more COs per chromosome (1.52) than wild-type (1.46) (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>), but this difference was not statistically significant (Wilcoxon test <em>P</em>-value 0.32). The mean length of the genotype blocks generated by COs (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5C</a>) was slightly greater in wild-type (15.8 Mb) than in <em>recq4a</em> mutants (15.2 Mb), but this difference was also not significant (Wilcoxon test <em>P</em>-value 0.15). The mean distance between double COs that occurred on the same chromosome, or the inter-CO distance, was not significantly greater in wild type either (8.7 <em>vs.</em> 8.4 Mb) (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS12.pdf">Figure S12B</a>; Wilcoxon test <em>P</em>-value 0.33). Using 800-kb sliding windows, we observed a strong correlation between COs in the wild-type and <em>recq4a</em> populations (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS14.pdf">Figure S14</a>). Because the recombination landscape for Col-0 X Ws-2 F<sub>2</sub> plants (with or without functional <em>RECQ4A</em>) was similar to several previously described F<sub>2</sub> populations (<a id="xref-ref-61-5" class="xref-bibr" href="#ref-61">Salomé <em>et al.</em> 2011a</a>), we conclude that the loss of <em>RECQ4A</em> has either no or only a very minor effect on the frequency or distribution of COs in <em>A. thaliana</em>.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/385/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of crossover (CO) distribution and frequency in wt and recq4a F2 populations. (A) The CO rate over a sliding window of 800 kb for each of the five chromosomes. Regions shaded in gray correspond to centromeres. The CO numbers per chromosome (B) and lengths of genetic blocks created by the CO positions (C) are shown as box-and-whisker plots." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1587840381" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of crossover (CO) distribution and frequency in wt and &lt;em&gt;recq4a&lt;/em&gt; F&lt;sub&gt;2&lt;/sub&gt; populations. (A) The CO rate over a sliding window of 800 kb for each of the five chromosomes. Regions shaded in gray correspond to centromeres. The CO numbers per chromosome (B) and lengths of genetic blocks created by the CO positions (C) are shown as box-and-whisker plots.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/385/F5.medium.gif" width="340" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/5/3/385/F5.medium.gif" width="340" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/385/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/385/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465714" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-46" class="first-child">Comparison of crossover (CO) distribution and frequency in wt and <em>recq4a</em> F<sub>2</sub> populations. (A) The CO rate over a sliding window of 800 kb for each of the five chromosomes. Regions shaded in gray correspond to centromeres. The CO numbers per chromosome (B) and lengths of genetic blocks created by the CO positions (C) are shown as box-and-whisker plots.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Region with unusual suppression of COs reveals a 1.8-Mb inversion</h3><p id="p-47">In addition to the centromeres, we observed several large regions largely or entirely devoid of COs on the chromosome arms. The most notable spanned from around 7 to 9 Mb on the long arm of chromosome 4. Because inversions can suppress recombination (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Sturtevant 1921</a>), we predicted structural variants in the high-coverage Ws-2 short read data; Pindel detected potential inversion breakpoints at positions 7,139,542 and 8,914,936 bp. We confirmed these by PCR (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS15.pdf">Figure S15, A and B</a><strong>)</strong> and Sanger sequencing, which revealed that the downstream breakpoint was coupled with an additional insertion of 389 bp, of which 337 bp had 83% similarity to the CACTA-like transposable element Ptta/En/Spm. PCR-based screening revealed that the inversion was not present in Ws-0 (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS15.pdf">Figure S15C</a>). We found the closest COs to the inversion at 6,989,963 and 8,960,496, which were around 150 kb and 45 kb away from the actual inversion breakpoints. Thus, we were able to map the location of the inversion extremely accurately using only the CO information from a limited number of plants.</p></div><div id="sec-21" class="subsection"><h3>QTL mapping of flowering time using recombination blocks as markers</h3><p id="p-48">We used our methods to study the genetic architecture of flowering time phenotypes, days to flowering and rosette leaf number (see the section <em>Materials and Methods</em>), which are moderately correlated (R<sup>2</sup> = 0.43) (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6, A and B</a>, <a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS16.pdf">Figure S16</a>). On average, the wild-type Ws-2 parents flowered 7.5 d earlier and produced 6.6 fewer rosette leaves before flowering compared with wild-type Col-0 parents (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/TableS5.pdf">Table S5</a>). The <em>recq4a</em> mutation caused a small, but statistically significant acceleration of flowering, interacting with the Col-0 background (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). This was also apparent in the distribution of flowering times among the F<sub>2</sub> individuals.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/385/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Quantitative trait locus (QTL) mapping of flowering time. Box-and-whisker plots showing rosette leaf number (A) and the days-to-flowering (B) for wild-type (red) and recq4a (blue) parents and among F2 individuals. (C) QTL analysis of flowering time phenotypes in the wt and recq4a mutant populations. The horizontal lines indicate the significance threshold (P = 0.05 for 1000 permutations) for the wt (red) and recq4a (blue) populations. Vertical ticks along the x-axis indicate the positions of the single-nucleotide polymorphism markers genotyped. (D) Schematic diagram of QTL intervals for flowering time within a region of chromosome 5 from 25,980,146 to 26,017,972 bp, based on data from the combined mapping populations from (B). Vertical dashed lines indicate the boundaries of marker blocks that were not separated by a recombination event in any F2 individual which are used to delineate the QTL intervals." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1587840381" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Quantitative trait locus (QTL) mapping of flowering time. Box-and-whisker plots showing rosette leaf number (A) and the days-to-flowering (B) for wild-type (red) and &lt;em&gt;recq4a&lt;/em&gt; (blue) parents and among F&lt;sub&gt;2&lt;/sub&gt; individuals. (C) QTL analysis of flowering time phenotypes in the wt and &lt;em&gt;recq4a&lt;/em&gt; mutant populations. The horizontal lines indicate the significance threshold (&lt;em&gt;P&lt;/em&gt; = 0.05 for 1000 permutations) for the wt (red) and &lt;em&gt;recq4a&lt;/em&gt; (blue) populations. Vertical ticks along the x-axis indicate the positions of the single-nucleotide polymorphism markers genotyped. (D) Schematic diagram of QTL intervals for flowering time within a region of chromosome 5 from 25,980,146 to 26,017,972 bp, based on data from the combined mapping populations from (B). Vertical dashed lines indicate the boundaries of marker blocks that were not separated by a recombination event in any F&lt;sub&gt;2&lt;/sub&gt; individual which are used to delineate the QTL intervals.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/385/F6.medium.gif" width="287" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/5/3/385/F6.medium.gif" width="287" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/385/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/385/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465771" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-49" class="first-child">Quantitative trait locus (QTL) mapping of flowering time. Box-and-whisker plots showing rosette leaf number (A) and the days-to-flowering (B) for wild-type (red) and <em>recq4a</em> (blue) parents and among F<sub>2</sub> individuals. (C) QTL analysis of flowering time phenotypes in the wt and <em>recq4a</em> mutant populations. The horizontal lines indicate the significance threshold (<em>P</em> = 0.05 for 1000 permutations) for the wt (red) and <em>recq4a</em> (blue) populations. Vertical ticks along the x-axis indicate the positions of the single-nucleotide polymorphism markers genotyped. (D) Schematic diagram of QTL intervals for flowering time within a region of chromosome 5 from 25,980,146 to 26,017,972 bp, based on data from the combined mapping populations from (B). Vertical dashed lines indicate the boundaries of marker blocks that were not separated by a recombination event in any F<sub>2</sub> individual which are used to delineate the QTL intervals.</p><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465768/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465768/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Flowering time as a function of genetic background and <em>RECQ4A</em> genotype</span><div class="sb-div caption-clear"></div></div></div><p id="p-51">Because the <em>recq4a</em> mutation led to earlier flowering, we performed two separate QTL analyses of the wild-type and <em>rec4a</em> F<sub>2</sub> populations (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6C</a>). We identified sequence blocks without recombination events in any F<sub>2</sub> individual and chose markers on both ends of these blocks to represent the genotypes of the blocks (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS17.pdf">Figure S17</a>). After removing 10 individuals with spurious genotype data from the wild-type population (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS18.pdf">Figure S18</a>), both rosette leaf number and days-to-flowering mapped to a single major-effect QTL on the bottom of chromosome 5 in both populations (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a>). Discarding markers with severe segregation distortion did not change this result (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS18.pdf">Figure S18</a>). We also performed a QTL analysis with the combined wild-type and <em>recq4a</em> populations without removing any individuals and detected the same chromosome 5 QTL for both phenotypes (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS18.pdf">Figure S18</a>). The combined analysis revealed a second, minor-effect QTL on chromosome 3.</p><p id="p-52">To determine the potential causal genes underlying the major QTL on chromosome 5, we identified the boundaries of the recombination blocks containing the SNP marker with the highest association with flowering time for the combined population, since including more individuals resulted in more statistical power. For rosette leaf number, we identified a 26.9-kb interval (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6D</a>) where the associated SNP marker explained 16.7% of the phenotypic variation (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/TableS6.pdf">Table S6</a>). For days-to-flowering, the QTL interval mapped to an adjacent 9-kb block (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6D</a>), and the associated SNP marker explained 19.5% of the phenotypic variation (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/TableS6.pdf">Table S6</a>). Both intervals overlapped with the <em>MADS AFFECTING FLOWERING2-5 (MAF2-5)</em> gene cluster, which has been previously shown to be the causal factor underlying flowering time QTL in many different F<sub>2</sub> populations (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alonso-Blanco <em>et al.</em> 1998</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Weinig <em>et al.</em> 2002</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Salomé <em>et al.</em> 2011b</a>). We grew F<sub>2</sub> plants again (N = 266 for wt and N = 202 for <em>recq4a</em>), scored both flowering time phenotypes, and genotyped 90 individuals from each population with a cleaved amplified polymorphic sequence marker (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Konieczny and Ausubel 1993</a>) designed for <em>MAF4</em>. There was a significant association between <em>MAF4</em> genotype and flowering time in both populations (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS19.pdf">Figure S19</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/TableS7.pdf">Table S7</a>). We again observed that the <em>recq4a</em> mutation exerted a marginal effect on flowering time; however, there was no interactive effect between the <em>recq4a</em> mutation and the <em>MAF4</em> genotype (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/TableS7.pdf">Table S7</a>). In summary, we conclude that our whole-genome-GBS approach enables not only the fine-scale resolution of recombination breakpoints but also accurate and precise QTL mapping.</p></div></div><div class="section discussion" id="sec-22"><h2 class="">Discussion</h2><div id="sec-23" class="subsection"><h3>An inexpensive and precise pipeline for studying recombinant individuals</h3><p id="p-53">The construction of precise genetic maps, which requires whole-genome resequencing of many recombinant individuals, is of critical importance to any researcher who is interested in studying the connection between genotype and phenotype. The current published and commercial methods make the production of high-density genetic maps cost-prohibitive for many investigators, who often compromise by adjusting either the marker sampling density or the number of individuals to be analyzed. With our method, whole-genome, paired-end genomic DNA libraries can be produced at about one-seventh of the cost of commercial alternatives (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-54">GBS of low-coverage samples is analytically challenging. Unlike the previously published methods using an HMM approach (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Xie <em>et al.</em> 2010</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Andolfatto <em>et al.</em> 2011</a>), we used an HMM that was trained on each sample individually, estimating the error rate on the sample data instead of providing general parameters, and thus handling variation in error rates. Our method leverages the availability of a large number of high-quality markers distributed across the genome that segregate with a Mendelian pattern of inheritance and avoids bias of the HMM that occurs when only one parental allele per marker is considered (<a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Xie <em>et al.</em> 2010</a>; <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Andolfatto <em>et al.</em> 2011</a>). We validated our approach using simulated data and characterized the types and locations of errors. We found that most errors were due to heterozygous genotypes not being properly predicted, particularly near telomeres and centromeres. We suspect that the errors at the telomeres likely arose from missing information because the ends of chromosomes had fewer markers. The centromeric regions had fewer high quality markers, which lowered the power of detecting a breakpoint nearby. The overall error rates were low (less than 3%) even at coverage of 0.1x, indicating that genotype and CO predictions were fairly robust.</p><p id="p-55">Using TIGER, we could localize most CO positions to within 2 kb (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>), with most of the tested COs confirmed by PCR (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4C</a> and <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). This level of precision is more than 10-fold greater than what was previously achieved with reduced representation sequencing for <em>Drosophila melanogaster</em> (<a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Andolfatto <em>et al.</em> 2011</a>). The improved CO resolution is likely due to the combined effects of a more realistic representation of the genome, higher marker density and more accurate genotype predictions. In the future, even smaller resolution distances could be achieved by replacing the final linear filling process of incorporating filtered markers with a more flexible method (<em>e.g.</em>, an additional HMM model). We expect that similar resolutions would be achieved for species with a similar density of high-quality markers, and that precision would improve with higher coverage.</p><p id="p-56">Our study has provided a detailed characterization of the CO landscape between Col-0 and Ws-2. We find that the overall landscape is similar to what had been observed previously for <em>A. thaliana</em> recombinant populations at low resolution (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Copenhaver <em>et al.</em> 1999</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Drouaud <em>et al.</em> 2007</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Giraut <em>et al.</em> 2011</a>; <a id="xref-ref-61-6" class="xref-bibr" href="#ref-61">Salomé <em>et al.</em> 2011a</a>), with the greatest CO frequency close to pericentromeric regions and a general absence of recombination within the centromeres (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4A</a>). In addition, we detected a 1.8-Mb inversion suppressing recombination from 7,139,542 to 8,914,936 bp on chromosome 4 in Ws-2 that is distinct from the much smaller inversion on the short arm of this chromosome that was reported by <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Fransz <em>et al.</em> (2000)</a> and is not present in Ws-0 (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS14.pdf">Figure S14C</a>). This adds to several reports that Ws-0 is genetically distinct from other Ws accessions (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Aukerman <em>et al.</em> 1997</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anastasio <em>et al.</em> 2011</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Pacurar <em>et al.</em> 2012</a>).</p><p id="p-57">Our precise CO breakpoint predictions enabled us to define blocks of the genome that were uninterrupted by COs in any F<sub>2</sub> individual in both the wild-type and <em>recq4a</em> F<sub>2</sub> populations. Using SNP markers that represented the genotypes of the blocks, we were able to precisely map a flowering time QTL to the <em>MAF2-5</em> gene cluster on chromosome 5 (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6</a>). The <em>MAF2-5</em> cluster has been identified as a QTL for flowering time variation for more than 20 pairs of accessions (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Alonso-Blanco <em>et al.</em> 1998</a>; <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Weinig <em>et al.</em> 2002</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">El-Lithy <em>et al.</em> 2004</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">El-Lithy <em>et al.</em> 2006</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">O’Neill <em>et al.</em> 2008</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Simon <em>et al.</em> 2008</a>; <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Salomé <em>et al.</em> 2011b</a>). We note that 80% of recombination blocks were shorter than 106 kb, even though we had only a modest number of individuals (<a href="http://www.g3journal.org/content/suppl/2015/01/13/g3.114.016501.DC1/FigureS14.pdf">Figure S14</a>).</p></div><div id="sec-24" class="subsection"><h3>A role for REQ4A in Arabidopsis meiosis?</h3><p id="p-58">We expected that if <em>RECQ4A</em> normally acted to prevent meiotic COs, the frequency and possibly the distribution of CO events would be altered without <em>RECQ4A</em> function, similar to what has been reported for yeast (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Rockmill <em>et al.</em> 2003</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Jessop <em>et al.</em> 2006</a>); this was not the case (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>). The only significant effect of the <em>recq4a</em> mutation we observed was an acceleration of flowering time. Because this did not result from an interaction between the <em>recq4a</em> mutation and the flowering time QTL, <em>MAF2-5</em>, it is likely that the mild genotoxic stress in <em>recq4a</em> mutants promoted earlier flowering.</p><p id="p-59">Why was there no significant effect of <em>recq4a</em> on the CO landscape? <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Higgins <em>et al.</em> (2011)</a> also observed no increase in the frequency of chiasmata in <em>recq4a</em> mutants and suggested that this was either due to redundancy with another helicase or to <em>RECQ4A</em> only having anti-CO activity at interference-insensitive COs. We observed that the inter-CO distances between double COs on the same chromosome were generally shorter in the <em>recq4a</em> population, but this difference was not statistically significant. Because only a small fraction (15%) of COs is interference-insensitive (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Higgins <em>et al.</em> 2004</a>), a minor change in the number of interference-insensitive COs would likely not result in a statistically significant change in the total number or distribution of COs and would thus be undetectable. If <em>RECQ4A</em> acts only on this fraction of COs, its effect must be very small. If functional redundancy explains our observation that mutation of <em>RECQ4A</em> does not significantly impact the CO landscape, then it means that one of the other six RecQ helicases can substitute for it (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Knoll and Puchta 2011</a>). The RecQ helicase that is most similar is to RECQ4A is its paralog RECQ4B, which promotes COs and does not exhibit the same genetic interactions or DNA repair activities as RECQ4A in somatic cells (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Hartung <em>et al.</em> 2006</a>, <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">2007</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Schropfer <em>et al.</em> 2014</a>). Chiasma frequency in meiosis does not differ between wild type and <em>recq4b</em> mutants (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Higgins <em>et al.</em> 2011</a>), but the frequency of chiasma or COs in <em>recq4a recq4b</em> double mutants has not yet been assessed. Further work is needed to establish whether RECQ4B or one of the more distantly related RecQs can adopt anti-CO activity during meiosis when <em>RECQ4A</em> function is impaired.</p><p id="p-60">Another intriguing possibility is that the resolution or dissolution of recombination intermediates during meiosis may not be mechanistically similar to what has been observed in other eukaryotes, as recently proposed by <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Knoll <em>et al.</em> (2014)</a>. These authors suggested that TOP3α and RMI1 (homologs of partner proteins for yeast SGS1 and human BLM) in <em>A. thaliana</em> might act on traditional double Holliday junction intermediates without RECQ4A or act with RECQ4A on different meiotic recombination intermediates. The lack of any significant effect of <em>recq4a</em> on the CO landscape in <em>A. thaliana</em> adds to the growing body of evidence for alternative methods for resolving recombination intermediates in plants.</p><p id="p-61">Although there were no broad-scale changes in the distribution or frequency of COs due to the <em>recq4a</em> mutation, we believe that our approach can be used to assess the genome-wide effects of various mutants in the homologous recombination pathway. Our wet-lab and analytical pipeline will also enable cost-effective GBS of hundreds of individuals for QTL or association mapping, and thus removes a strong limitation to the scale of biological questions that can be asked and addressed.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-62">We thank Stephanie Günther, Laura Breimann, Lena Muckenfuß, and Christa Lanz for technical assistance, and Patrice Salomé for helpful discussions. Holger Puchta, Frank Hartung, and Mohammad B. Bagarieh-Najjar kindly provided the <em>recq4a-1</em> (Ws-2 background) and <em>recq4a-4</em> (Col-0 background) seed stocks. Claude Becker, Norman Warthmann, Eunyoung Chae, and Margarete Hoffmann provided helpful advice on library preparation; Danelle Seymour and Dan Koenig on QTL analysis; Jonas Müller and Jörg Hagmann on bioinformatics; and George Wang on R. Markus Schneller helped locate journal articles and book chapters that could not be obtained online. Beth Rowan was supported by a postdoctoral fellowship from the Alexander von Humboldt foundation. This work was supported by the Max Planck Society.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-10"><p id="p-63">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016501/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016501/-/DC1</a></p></li><li class="fn" id="fn-11"><p id="p-64">The short read sequence data from this article have been deposited in the National Center for Biotechnology Information Sequence Read Archive under accession no. PRJNA268060.</p></li><li class="fn" id="fn-12"><p id="p-65"><em>Communicating editor: D. J. de Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-10-30"><span class="received-label">Received </span>October 30, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-12-30"><span class="accepted-label">Accepted </span>December 30, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Rowan <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alonso-Blanco</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">El-Assal</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coupland</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koornneef</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Analysis of natural allelic variation at flowering time loci in the Landsberg erecta and Cape Verde Islands ecotypes of <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">149</span>: <span class="cit-fpage">749</span>–<span class="cit-lpage">764</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAlonso-Blanco%26rft.auinit1%253DC.%26rft.volume%253D149%26rft.issue%253D2%26rft.spage%253D749%26rft.epage%253D764%26rft.atitle%253DAnalysis%2Bof%2BNatural%2BAllelic%2BVariation%2Bat%2BFlowering%2BTime%2BLoci%2Bin%2Bthe%2BLandsberg%2Berecta%2Band%2BCape%2BVerde%2BIslands%2BEcotypes%2Bof%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Apmid%252F9611189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ5LzIvNzQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.2" data-doi="10.1111/j.1365-313X.2011.04606.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anastasio</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grotewold</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scholl</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Source verification of mis-identified <em>Arabidopsis thaliana</em> accessions.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">554</span>–<span class="cit-lpage">566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DAnastasio%26rft.auinit1%253DA.%2BE.%26rft.volume%253D67%26rft.issue%253D3%26rft.spage%253D554%26rft.epage%253D566%26rft.atitle%253DSource%2Bverification%2Bof%2Bmis-identified%2BArabidopsis%2Bthaliana%2Baccessions.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2011.04606.x%26rft_id%253Dinfo%253Apmid%252F21481029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2011.04606.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21481029&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293176600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hooker</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stack</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">The distribution of early recombination nodules on zygotene bivalents from plants.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">159</span>: <span class="cit-fpage">1259</span>–<span class="cit-lpage">1269</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAnderson%26rft.auinit1%253DL.%2BK.%26rft.volume%253D159%26rft.issue%253D3%26rft.spage%253D1259%26rft.epage%253D1269%26rft.atitle%253DThe%2BDistribution%2Bof%2BEarly%2BRecombination%2BNodules%2Bon%2BZygotene%2BBivalents%2BFrom%2BPlants%26rft_id%253Dinfo%253Apmid%252F11729167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OS8zLzEyNTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reeves</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Webb</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashley</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Distribution of crossing over on mouse synaptonemal complexes using immunofluorescent localization of MLH1 protein</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">151</span>: <span class="cit-fpage">1569</span>–<span class="cit-lpage">1579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAnderson%26rft.auinit1%253DL.%2BK.%26rft.volume%253D151%26rft.issue%253D4%26rft.spage%253D1569%26rft.epage%253D1579%26rft.atitle%253DDistribution%2Bof%2BCrossing%2BOver%2Bon%2BMouse%2BSynaptonemal%2BComplexes%2BUsing%2BImmunofluorescent%2BLocalization%2Bof%2BMLH1%2BProtein%26rft_id%253Dinfo%253Apmid%252F10101178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1MS80LzE1NjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doyle</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brigham</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hooker</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">High-resolution crossover maps for each bivalent of Zea mays using recombination nodules</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">849</span>–<span class="cit-lpage">865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAnderson%26rft.auinit1%253DL.%2BK.%26rft.volume%253D165%26rft.issue%253D2%26rft.spage%253D849%26rft.epage%253D865%26rft.atitle%253DHigh-Resolution%2BCrossover%2BMaps%2Bfor%2BEach%2BBivalent%2Bof%2BZea%2Bmays%2BUsing%2BRecombination%2BNodules%26rft_id%253Dinfo%253Apmid%252F14573493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY1LzIvODQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.6" data-doi="10.1101/gr.115402.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davison</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erezyilmaz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mast</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multiplexed shotgun genotyping for rapid and efficient genetic mapping.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">610</span>–<span class="cit-lpage">617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.115402.110%26rft_id%253Dinfo%253Apmid%252F21233398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzQvNjEwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.7" data-doi="10.1105/tpc.9.8.1317"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aukerman</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirschfeld</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wester</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weaver</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clack</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A deletion in the PHYD gene of the Arabidopsis Wassilewskija ecotype defines a role for phytochrome D in red/far-red light sensing.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1317</span>–<span class="cit-lpage">1326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DAukerman%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D1317%26rft.epage%253D1326%26rft.atitle%253DA%2BDeletion%2Bin%2Bthe%2BPHYD%2BGene%2Bof%2Bthe%2BArabidopsis%2BWassilewskija%2BEcotype%2BDefines%2Ba%2BRole%2Bfor%2BPhytochrome%2BD%2Bin%2BRed%252FFar-Red%2BLight%2BSensing%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.9.8.1317%26rft_id%253Dinfo%253Apmid%252F9286109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjkvOC8xMzE3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.8" data-doi="10.1111/j.1365-313X.2005.02501.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bagherieh-Najjar</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Vries</span>  <span class="cit-name-given-names">O. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hille</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dijkwel</span>  <span class="cit-name-given-names">P. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Arabidopsis RecQI4A suppresses homologous recombination and modulates DNA damage responses.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">798</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DBagherieh-Najjar%26rft.auinit1%253DM.%2BB.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D789%26rft.epage%253D798%26rft.atitle%253DArabidopsis%2BRecQI4A%2Bsuppresses%2Bhomologous%2Brecombination%2Band%2Bmodulates%2BDNA%2Bdamage%2Bresponses.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2005.02501.x%26rft_id%253Dinfo%253Apmid%252F16146519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2005.02501.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16146519&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231985500001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.9" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.10" data-doi="10.1105/tpc.113.121269"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barakate</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vivera</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The synaptonemal complex protein ZYP1 is required for imposition of meiotic crossovers in barley</span>. <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">729</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.113.121269%26rft_id%253Dinfo%253Apmid%252F24563202%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjI2LzIvNzI5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.11" data-doi="10.1126/science.281.5385.1986"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barton</span>  <span class="cit-name-given-names">N. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlesworth</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Why sex and recombination?</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">281</span>: <span class="cit-fpage">1986</span>–<span class="cit-lpage">1990</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBarton%26rft.auinit1%253DN.%2BH.%26rft.volume%253D281%26rft.issue%253D5385%26rft.spage%253D1986%26rft.epage%253D1990%26rft.atitle%253DWhy%2BSex%2Band%2BRecombination%253F%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.281.5385.1986%26rft_id%253Dinfo%253Apmid%252F9748151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODEvNTM4NS8xOTg2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.12" data-doi="10.1038/nrg3573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baudat</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Massy</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Meiotic recombination in mammals: localization and regulation.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">794</span>–<span class="cit-lpage">806</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D794%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3573%26rft_id%253Dinfo%253Apmid%252F24136506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3573&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24136506&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.13" data-doi="10.1016/j.ajhg.2009.01.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Browning</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">84</span>: <span class="cit-fpage">210</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DBrowning%26rft.auinit1%253DB.%2BL.%26rft.volume%253D84%26rft.issue%253D2%26rft.spage%253D210%26rft.epage%253D223%26rft.atitle%253DA%2Bunified%2Bapproach%2Bto%2Bgenotype%2Bimputation%2Band%2Bhaplotype-phase%2Binference%2Bfor%2Blarge%2Bdata%2Bsets%2Bof%2Btrios%2Band%2Bunrelated%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2009.01.005%26rft_id%253Dinfo%253Apmid%252F19200528%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2009.01.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19200528&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263799700011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.14" data-doi="10.1371/journal.pbio.0050324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buhler</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borde</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Mapping meiotic single-strand DNA reveals a new landscape of DNA double-strand breaks in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DBuhler%26rft.auinit1%253DC.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253De324%26rft.epage%253De324%26rft.atitle%253DMapping%2Bmeiotic%2Bsingle-strand%2BDNA%2Breveals%2Ba%2Bnew%2Blandscape%2Bof%2BDNA%2Bdouble-strand%2Bbreaks%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0050324%26rft_id%253Dinfo%253Apmid%252F18076285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0050324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076285&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.15" data-doi="10.1038/ng.911"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gunther</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bender</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Whole-genome sequencing of multiple <em>Arabidopsis thaliana</em> populations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">956</span>–<span class="cit-lpage">963</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCao%26rft.auinit1%253DJ.%26rft.volume%253D43%26rft.issue%253D10%26rft.spage%253D956%26rft.epage%253D963%26rft.atitle%253DWhole-genome%2Bsequencing%2Bof%2Bmultiple%2BArabidopsis%2Bthaliana%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.911%26rft_id%253Dinfo%253Apmid%252F21874002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.911&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21874002&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.16" data-doi="10.1371/journal.pgen.0030083"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chelysheva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gendrot</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vezon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doutriaux</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mercier</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Zip4/Spo22 is required for class I CO formation but not for synapsis completion in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DChelysheva%26rft.auinit1%253DL.%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253De83%26rft.epage%253De83%26rft.atitle%253DZip4%252FSpo22%2Bis%2Brequired%2Bfor%2Bclass%2BI%2BCO%2Bformation%2Bbut%2Bnot%2Bfor%2Bsynapsis%2Bcompletion%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030083%26rft_id%253Dinfo%253Apmid%252F17530928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030083&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17530928&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.17" data-doi="10.1038/ng.2766"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelly</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venn</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Arabidopsis meiotic crossover hot spots overlap with H2A.Z nucleosomes at gene promoters.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1327</span>–<span class="cit-lpage">1336</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D45%26rft.spage%253D1327%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2766%26rft_id%253Dinfo%253Apmid%252F24056716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2766&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24056716&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.18" data-doi="10.1126/science.286.5449.2468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Copenhaver</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nickel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuromori</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benito</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaul</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genetic definition and sequence analysis of <em>Arabidopsis</em> centromeres.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">286</span>: <span class="cit-fpage">2468</span>–<span class="cit-lpage">2474</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DCopenhaver%26rft.auinit1%253DG.%2BP.%26rft.volume%253D286%26rft.issue%253D5449%26rft.spage%253D2468%26rft.epage%253D2474%26rft.atitle%253DGenetic%2BDefinition%2Band%2BSequence%2BAnalysis%2Bof%2BArabidopsis%2BCentromeres%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.286.5449.2468%26rft_id%253Dinfo%253Apmid%252F10617454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODYvNTQ0OS8yNDY4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.19" data-doi="10.1016/j.molcel.2012.02.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De Muyt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jessop</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolar</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sourirajan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">BLM helicase ortholog Sgs1 is a central regulator of meiotic recombination intermediate metabolism.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DDe%2BMuyt%26rft.auinit1%253DA.%26rft.volume%253D46%26rft.issue%253D1%26rft.spage%253D43%26rft.epage%253D53%26rft.atitle%253DBLM%2Bhelicase%2Bortholog%2BSgs1%2Bis%2Ba%2Bcentral%2Bregulator%2Bof%2Bmeiotic%2Brecombination%2Bintermediate%2Bmetabolism.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2012.02.020%26rft_id%253Dinfo%253Apmid%252F22500736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2012.02.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22500736&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302838000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.20" data-doi="10.1371/journal.pgen.0030106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drouaud</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mercier</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chelysheva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Sex-specific crossover distributions and variations in interference level along <em>Arabidopsis thaliana</em> chromosome 4.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDrouaud%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.issue%253D6%26rft.spage%253De106%26rft.epage%253De106%26rft.atitle%253DSex-specific%2Bcrossover%2Bdistributions%2Band%2Bvariations%2Bin%2Binterference%2Blevel%2Balong%2BArabidopsis%2Bthaliana%2Bchromosome%2B4.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030106%26rft_id%253Dinfo%253Apmid%252F17604455%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17604455&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.21" data-doi="10.1104/pp.103.036822"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Lithy</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clerkx</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruys</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koornneef</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vreugdenhil</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Quantitative trait locus analysis of growth-related traits in a new <em>Arabidopsis</em> recombinant inbred population.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">444</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.103.036822%26rft_id%253Dinfo%253Apmid%252F15122039%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMzUvMS80NDQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.22" data-doi="10.1534/genetics.105.050617"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-Lithy</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentsink</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanhart</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruys</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rovito</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">New <em>Arabidopsis</em> recombinant inbred line populations genotyped using SNPWave and their use for mapping flowering-time quantitative trait loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">1867</span>–<span class="cit-lpage">1876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.050617%26rft_id%253Dinfo%253Apmid%252F16361234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi8zLzE4NjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.23" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.24" data-doi="10.1073/pnas.0608936104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Francis</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bey</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berchowitz</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Pollen tetrad-based visual assay for meiotic recombination in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">3913</span>–<span class="cit-lpage">3918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0608936104%26rft_id%253Dinfo%253Apmid%252F17360452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA0LzEwLzM5MTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.25" data-doi="10.1016/S0092-8674(00)80672-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fransz</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armstrong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">De Jong</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parnell</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Drunen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Integrated cytogenetic map of chromosome arm 4S of <em>A. thaliana</em>: structural organization of heterochromatic knob and centromere region.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DFransz%26rft.auinit1%253DP.%2BF.%26rft.volume%253D100%26rft.issue%253D3%26rft.spage%253D367%26rft.epage%253D376%26rft.atitle%253DIntegrated%2Bcytogenetic%2Bmap%2Bof%2Bchromosome%2Barm%2B4S%2Bof%2BA.%2Bthaliana%253A%2Bstructural%2Borganization%2Bof%2Bheterochromatic%2Bknob%2Band%2Bcentromere%2Bregion.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980672-8%26rft_id%253Dinfo%253Apmid%252F10676818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80672-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10676818&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000085204400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.26" data-doi="10.1371/journal.pgen.1002354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giraut</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Falque</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drouaud</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">O. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide crossover distribution in <em>Arabidopsis thaliana</em> meiosis reveals sex-specific patterns along chromosomes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1002354</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGiraut%26rft.auinit1%253DL.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De1002354%26rft.epage%253De1002354%26rft.atitle%253DGenome-wide%2Bcrossover%2Bdistribution%2Bin%2BArabidopsis%2Bthaliana%2Bmeiosis%2Breveals%2Bsex-specific%2Bpatterns%2Balong%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002354%26rft_id%253Dinfo%253Apmid%252F22072983%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002354&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22072983&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Chromosome segregation influenced by two alleles of the meiotic mutant <em>c(3)G</em> in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">367</span>–<span class="cit-lpage">400</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHall%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D71%26rft.issue%253D3%26rft.spage%253D367%26rft.epage%253D400%26rft.atitle%253DChromosome%2Bsegregation%2Binfluenced%2Bby%2Btwo%2Balleles%2Bof%2Bthe%2Bmeiotic%2Bmutant%2Bc%25283%2529G%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Apmid%252F4624918%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=4624918&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.28" data-doi="10.1093/nar/gkl576"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartung</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puchta</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The role of AtMUS81 in DNA repair and its genetic interaction with the helicase AtRecQ4A.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">4438</span>–<span class="cit-lpage">4448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl576%26rft_id%253Dinfo%253Apmid%252F16945961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzNC8xNi80NDM4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.29" data-doi="10.1073/pnas.0705998104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hartung</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Suer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puchta</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Two closely related RecQ helicases have antagonistic roles in homologous recombination and DNA repair in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">18836</span>–<span class="cit-lpage">18841</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0705998104%26rft_id%253Dinfo%253Apmid%252F18000056%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzQ3LzE4ODM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-5.3.385.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hawley</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Exchange and chromosomal segregation in eucaryotes</span>, pp. <span class="cit-fpage">497</span>–<span class="cit-lpage">527</span> in <span class="cit-source">Genetic Recombination</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Kucher-Lapati</span>  <span class="cit-name-given-names">R.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">G. R.</span></span>. <span class="cit-publ-name">ASM</span>, <span class="cit-publ-loc">Washington, DC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.31" data-doi="10.1016/j.pbi.2012.09.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">I. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Control of meiotic recombination frequency in plant genomes.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">556</span>–<span class="cit-lpage">561</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DHenderson%26rft.auinit1%253DI.%2BR.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D556%26rft.epage%253D561%26rft.atitle%253DControl%2Bof%2Bmeiotic%2Brecombination%2Bfrequency%2Bin%2Bplant%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2012.09.002%26rft_id%253Dinfo%253Apmid%252F23017241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2012.09.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23017241&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.32" data-doi="10.1101/gad.317504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armstrong</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franklin</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The <em>Arabidopsis</em> MutS homolog AtMSH4 functions at an early step in recombination: evidence for two classes of recombination in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2557</span>–<span class="cit-lpage">2570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DHiggins%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D18%26rft.issue%253D20%26rft.spage%253D2557%26rft.epage%253D2570%26rft.atitle%253DThe%2BArabidopsis%2BMutS%2Bhomolog%2BAtMSH4%2Bfunctions%2Bat%2Ban%2Bearly%2Bstep%2Bin%2Brecombination%253A%2Bevidence%2Bfor%2Btwo%2Bclasses%2Bof%2Brecombination%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.317504%26rft_id%253Dinfo%253Apmid%252F15489296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE4LzIwLzI1NTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.33" data-doi="10.1111/j.1365-313X.2010.04438.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Higgins</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferdous</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franklin</span>  <span class="cit-name-given-names">F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The RecQ helicase AtRECQ4A is required to remove inter-chromosomal telomeric connections that arise during meiotic recombination in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">492</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DHiggins%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D65%26rft.issue%253D3%26rft.spage%253D492%26rft.epage%253D502%26rft.atitle%253DThe%2BRecQ%2Bhelicase%2BAtRECQ4A%2Bis%2Brequired%2Bto%2Bremove%2Binter-chromosomal%2Btelomeric%2Bconnections%2Bthat%2Barise%2Bduring%2Bmeiotic%2Brecombination%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2010.04438.x%26rft_id%253Dinfo%253Apmid%252F21265901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2010.04438.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21265901&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286619000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.34" data-doi="10.1083/jcb.200909048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holloway</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morelli</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borst</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">P. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mammalian BLM helicase is critical for integrating multiple pathways of meiotic recombination.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">779</span>–<span class="cit-lpage">789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.200909048%26rft_id%253Dinfo%253Apmid%252F20308424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjE4OC82Lzc3OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy8zODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.35" data-doi="10.1371/journal.pgen.1000529"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Howie</span>  <span class="cit-name-given-names">B. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A flexible and accurate genotype imputation method for the next generation of genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000529</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DHowie%26rft.auinit1%253DB.%2BN.%26rft.volume%253D5%26rft.issue%253D6%26rft.spage%253De1000529%26rft.epage%253De1000529%26rft.atitle%253DA%2Bflexible%2Band%2Baccurate%2Bgenotype%2Bimputation%2Bmethod%2Bfor%2Bthe%2Bnext%2Bgeneration%2Bof%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000529%26rft_id%253Dinfo%253Apmid%252F19543373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000529&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19543373&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.36" data-doi="10.1101/gr.089516.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High-throughput genotyping by whole-genome resequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1068</span>–<span class="cit-lpage">1076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.089516.108%26rft_id%253Dinfo%253Apmid%252F19420380%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzYvMTA2OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy8zODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.37" data-doi="10.1186/gb-2013-14-6-r61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">James</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordstrom</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klasen</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salome</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">User guide for mapping-by-sequencing in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR61%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-6-r61%26rft_id%253Dinfo%253Apmid%252F23773572%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-6-r61&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23773572&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.38" data-doi="10.1371/journal.pgen.0020155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jessop</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Meiotic chromosome synapsis-promoting proteins antagonize the anti-crossover activity of sgs1.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DJessop%26rft.auinit1%253DL.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253De155%26rft.epage%253De155%26rft.atitle%253DMeiotic%2Bchromosome%2Bsynapsis-promoting%2Bproteins%2Bantagonize%2Bthe%2Banti-crossover%2Bactivity%2Bof%2Bsgs1.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020155%26rft_id%253Dinfo%253Apmid%252F17002499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17002499&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.39" data-doi="10.1093/bioinformatics/bti318"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coombes</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Applications of beta-mixture models in bioinformatics.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2118</span>–<span class="cit-lpage">2122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti318%26rft_id%253Dinfo%253Apmid%252F15713737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMS85LzIxMTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.40" data-doi="10.1093/jxb/erq357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knoll</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puchta</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The role of DNA helicases and their interaction partners in genome stability and meiotic recombination in plants.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">1565</span>–<span class="cit-lpage">1579</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferq357%26rft_id%253Dinfo%253Apmid%252F21081662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjk6IjYyLzUvMTU2NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy8zODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knoll</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schropfer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puchta</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The RTR complex as caretaker of genome stability and its unique meiotic function in plants.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D5%26rft.spage%253D33%26rft_id%253Dinfo%253Apmid%252F24575106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24575106&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.42" data-doi="10.1046/j.1365-313X.1993.04020403.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Konieczny</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ausubel</span>  <span class="cit-name-given-names">F. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">A procedure for mapping <em>Arabidopsis</em> mutations using co-dominant ecotype-specific PCR-based markers.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DKonieczny%26rft.auinit1%253DA.%26rft.volume%253D4%26rft.issue%253D2%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DA%2Bprocedure%2Bfor%2Bmapping%2BArabidopsis%2Bmutations%2Busing%2Bco-dominant%2Becotype-specific%2BPCR-based%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313X.1993.04020403.x%26rft_id%253Dinfo%253Apmid%252F8106085%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313X.1993.04020403.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8106085&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993LT72100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wijeratne</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Analysis of <em>Arabidopsi</em>s genome-wide variations before and after meiosis and meiotic recombination by resequencing <em>Landsberg erecta</em> and all four products of a single meiosis.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">508</span>–<span class="cit-lpage">518</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft.volume%253D22%26rft.spage%253D508%26rft_id%253Dinfo%253Apmid%252F22106370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22106370&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.44" data-doi="10.1038/nature07135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mancera</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bourgon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brozzi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinmetz</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-resolution mapping of meiotic crossovers and non-crossovers in yeast.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">454</span>: <span class="cit-fpage">479</span>–<span class="cit-lpage">485</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMancera%26rft.auinit1%253DE.%26rft.volume%253D454%26rft.issue%253D7203%26rft.spage%253D479%26rft.epage%253D485%26rft.atitle%253DHigh-resolution%2Bmapping%2Bof%2Bmeiotic%2Bcrossovers%2Band%2Bnon-crossovers%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07135%26rft_id%253Dinfo%253Apmid%252F18615017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18615017&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257860300043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.45" data-doi="10.1016/j.tig.2007.12.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mardis</span>  <span class="cit-name-given-names">E. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The impact of next-generation sequencing technology on genetics.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DMardis%26rft.auinit1%253DE.%2BR.%26rft.volume%253D24%26rft.issue%253D3%26rft.spage%253D133%26rft.epage%253D141%26rft.atitle%253DThe%2Bimpact%2Bof%2Bnext-generation%2Bsequencing%2Btechnology%2Bon%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2007.12.007%26rft_id%253Dinfo%253Apmid%252F18262675%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2007.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18262675&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000254058200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.46" data-doi="10.1111/j.1365-313X.2005.02466.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Melamed-Bessudo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yehuda</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stuitje</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A new seed-based assay for meiotic recombination in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">458</span>–<span class="cit-lpage">466</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DMelamed-Bessudo%26rft.auinit1%253DC.%26rft.volume%253D43%26rft.issue%253D3%26rft.spage%253D458%26rft.epage%253D466%26rft.atitle%253DA%2Bnew%2Bseed-based%2Bassay%2Bfor%2Bmeiotic%2Brecombination%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2005.02466.x%26rft_id%253Dinfo%253Apmid%252F16045480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2005.02466.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16045480&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230602500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.47" data-doi="10.1016/j.cell.2009.07.035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mets</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Condensins regulate meiotic DNA break distribution, thus crossover frequency, by controlling chromosome structure.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">139</span>: <span class="cit-fpage">73</span>–<span class="cit-lpage">86</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMets%26rft.auinit1%253DD.%2BG.%26rft.volume%253D139%26rft.issue%253D1%26rft.spage%253D73%26rft.epage%253D86%26rft.atitle%253DCondensins%2Bregulate%2Bmeiotic%2BDNA%2Bbreak%2Bdistribution%252C%2Bthus%2Bcrossover%2Bfrequency%252C%2Bby%2Bcontrolling%2Bchromosome%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.07.035%26rft_id%253Dinfo%253Apmid%252F19781752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.07.035&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19781752&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270388600017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.48" data-doi="10.1007/s004120050241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moens</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolas</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarsounas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Rad51 immunocytology in rat and mouse spermatocytes and oocytes.</span> <abbr class="cit-jnl-abbrev">Chromosoma</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChromosoma%26rft.stitle%253DChromosoma%26rft.aulast%253DMoens%26rft.auinit1%253DP.%2BB.%26rft.volume%253D106%26rft.issue%253D4%26rft.spage%253D207%26rft.epage%253D215%26rft.atitle%253DRad51%2Bimmunocytology%2Bin%2Brat%2Band%2Bmouse%2Bspermatocytes%2Band%2Boocytes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs004120050241%26rft_id%253Dinfo%253Apmid%252F9254722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s004120050241&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9254722&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XW37800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.49"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moens</span>  <span class="cit-name-given-names">P. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolas</span>  <span class="cit-name-given-names">N. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tarsounas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcon</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The time course and chromosomal localization of recombination-related proteins at meiosis in the mouse are compatible with models that can resolve the early DNA-DNA interactions without reciprocal recombination</span>. <abbr class="cit-jnl-abbrev">J Cell Sci</abbr> <span class="cit-vol">115</span>: <span class="cit-fpage">1611</span>–<span class="cit-lpage">1622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BCell%2BScience%26rft.stitle%253DJ.%2BCell%2BSci.%26rft.aulast%253DMoens%26rft.auinit1%253DP.%2BB.%26rft.volume%253D115%26rft.issue%253D8%26rft.spage%253D1611%26rft.epage%253D1622%26rft.atitle%253DThe%2Btime%2Bcourse%2Band%2Bchromosomal%2Blocalization%2Bof%2Brecombination-related%2Bproteins%2Bat%2Bmeiosis%2Bin%2Bthe%2Bmouse%2Bare%2Bcompatible%2Bwith%2Bmodels%2Bthat%2Bcan%2Bresolve%2Bthe%2Bearly%2BDNA-DNA%2Binteractions%2Bwithout%2Breciprocal%2Brecombination%26rft_id%253Dinfo%253Apmid%252F11950880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiam9jZXMiO3M6NToicmVzaWQiO3M6MTA6IjExNS84LzE2MTEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.50" data-doi="10.1073/pnas.1102298108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nottke</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beese-Sims</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pantalena</span>  <span class="cit-name-given-names">L. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinke</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SPR-5 is a histone H3K4 demethylase with a role in meiotic double-strand break repair.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">12805</span>–<span class="cit-lpage">12810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1102298108%26rft_id%253Dinfo%253Apmid%252F21768382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzMxLzEyODA1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.51" data-doi="10.1007/s00122-007-0696-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Neill</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morgan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tschoep</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">P. X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Six new recombinant inbred populations for the study of quantitative traits in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DO%2527Neill%26rft.auinit1%253DC.%2BM.%26rft.volume%253D116%26rft.issue%253D5%26rft.spage%253D623%26rft.epage%253D634%26rft.atitle%253DSix%2Bnew%2Brecombinant%2Binbred%2Bpopulations%2Bfor%2Bthe%2Bstudy%2Bof%2Bquantitative%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-007-0696-9%26rft_id%253Dinfo%253Apmid%252F18193187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-007-0696-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18193187&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253632700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.52" data-doi="10.1101/gr.080200.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lanz</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warthmann</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sequencing of natural strains of <em>Arabidopsis thaliana</em> with short reads.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2024</span>–<span class="cit-lpage">2033</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.080200.108%26rft_id%253Dinfo%253Apmid%252F18818371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMi8yMDI0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.53" data-doi="10.1093/jxb/err422"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pacurar</span>  <span class="cit-name-given-names">D. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pacurar</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Street</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussell</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A collection of INDEL markers for map-based cloning in seven <em>Arabidopsis</em> accessions.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">2491</span>–<span class="cit-lpage">2501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferr422%26rft_id%253Dinfo%253Apmid%252F22282537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjk6IjYzLzcvMjQ5MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy8zODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.54" data-doi="10.1038/nrg2712"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paigen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petkov</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mammalian recombination hot spots: properties, control and evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">221</span>–<span class="cit-lpage">233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DPaigen%26rft.auinit1%253DK.%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D221%26rft.epage%253D233%26rft.atitle%253DMammalian%2Brecombination%2Bhot%2Bspots%253A%2Bproperties%252C%2Bcontrol%2Band%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2712%26rft_id%253Dinfo%253Apmid%252F20168297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2712&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20168297&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274649900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.55" data-doi="10.1007/s003350020005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Manuel De Villena</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sapienza</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Recombination is proportional to the number of chromosome arms in mammals</span>. <abbr class="cit-jnl-abbrev">Mamm Genome</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">318</span>-<span class="cit-lpage">322</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMammalian%2Bgenome%2B%253A%2B%2Bofficial%2Bjournal%2Bof%2Bthe%2BInternational%2BMammalian%2BGenome%2BSociety%26rft.stitle%253DMamm%2BGenome%26rft.aulast%253DPardo-Manuel%2Bde%2BVillena%26rft.auinit1%253DF.%26rft.volume%253D12%26rft.issue%253D4%26rft.spage%253D318%26rft.epage%253D322%26rft.atitle%253DRecombination%2Bis%2Bproportional%2Bto%2Bthe%2Bnumber%2Bof%2Bchromosome%2Barms%2Bin%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs003350020005%26rft_id%253Dinfo%253Apmid%252F11309665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s003350020005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11309665&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167693600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.56" data-doi="10.1016/j.tig.2011.06.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phadnis</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyppa</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">G. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">New and old ways to control meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">411</span>–<span class="cit-lpage">421</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DPhadnis%26rft.auinit1%253DN.%26rft.volume%253D27%26rft.issue%253D10%26rft.spage%253D411%26rft.epage%253D421%26rft.atitle%253DNew%2Band%2Bold%2Bways%2Bto%2Bcontrol%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2011.06.007%26rft_id%253Dinfo%253Apmid%252F21782271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2011.06.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21782271&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295489100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.57"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pryce</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McFarlane</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The meiotic recombination hotspots of <em>Schizosaccharomyces pombe</em>.</span> <abbr class="cit-jnl-abbrev">Genome Dyn.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DGenome%2BDyn%26rft.aulast%253DPryce%26rft.auinit1%253DD.%2BW.%26rft.volume%253D5%26rft.spage%253D1%26rft.epage%253D13%26rft.atitle%253DThe%2Bmeiotic%2Brecombination%2Bhotspots%2Bof%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Apmid%252F18948703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18948703&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.58" data-doi="10.1073/pnas.1321897111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Copenhaver</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Detection of genomic variations and DNA polymorphisms and impact on analysis of meiotic recombination and genetic mapping.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">10007</span>–<span class="cit-lpage">10012</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1321897111%26rft_id%253Dinfo%253Apmid%252F24958856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzI3LzEwMDA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.59" data-doi="10.1016/j.cub.2003.10.059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fung</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Branda</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The Sgs1 helicase regulates chromosome synapsis and meiotic crossing over.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1954</span>–<span class="cit-lpage">1962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DRockmill%26rft.auinit1%253DB.%26rft.volume%253D13%26rft.issue%253D22%26rft.spage%253D1954%26rft.epage%253D1962%26rft.atitle%253DThe%2BSgs1%2Bhelicase%2Bregulates%2Bchromosome%2Bsynapsis%2Band%2Bmeiotic%2Bcrossing%2Bover.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2003.10.059%26rft_id%253Dinfo%253Apmid%252F14614820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2003.10.059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14614820&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186558100021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.60" data-doi="10.1126/science.1212424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Libuda</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villeneuve</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Robust crossover assurance and regulated interhomolog access maintain meiotic crossover number</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">334</span>: <span class="cit-fpage">1286</span>–<span class="cit-lpage">1289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRosu%26rft.auinit1%253DS.%26rft.volume%253D334%26rft.issue%253D6060%26rft.spage%253D1286%26rft.epage%253D1289%26rft.atitle%253DRobust%2BCrossover%2BAssurance%2Band%2BRegulated%2BInterhomolog%2BAccess%2BMaintain%2BMeiotic%2BCrossover%2BNumber%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1212424%26rft_id%253Dinfo%253Apmid%252F22144627%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMzQvNjA2MC8xMjg2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salomé</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bomblies</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laitinen</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warthmann</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">a</span> <span class="cit-article-title">The recombination landscape in <em>Arabidopsis thaliana</em> F2 populations.</span> <abbr class="cit-jnl-abbrev">Heredity (Edinb)</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%2B%2528Edinb%2529%26rft.volume%253D108%26rft.spage%253D447%26rft_id%253Dinfo%253Apmid%252F22072068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22072068&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.62" data-doi="10.1534/genetics.111.126607"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salomé</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bomblies</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laitinen</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yant</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mott</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span><span class="cit-comment">b</span> <span class="cit-article-title">Genetic architecture of flowering-time variation in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">433</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.126607%26rft_id%253Dinfo%253Apmid%252F21406681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzIvNDIxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Moran</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armstrong</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franklin</span>  <span class="cit-name-given-names">F. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Variation in chiasma frequency among eight accessions of <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">1415</span>–<span class="cit-lpage">1422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSanchez-Moran%26rft.auinit1%253DE.%26rft.volume%253D162%26rft.issue%253D3%26rft.spage%253D1415%26rft.epage%253D1422%26rft.atitle%253DVariation%2Bin%2BChiasma%2BFrequency%2BAmong%2BEight%2BAccessions%2Bof%2BArabidopsis%2Bthaliana%26rft_id%253Dinfo%253Apmid%252F12454084%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Mi8zLzE0MTUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.64" data-doi="10.1101/gad.439007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Moran</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santos</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franklin</span>  <span class="cit-name-given-names">F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">ASY1 mediates AtDMC1-dependent interhomolog recombination during meiosis in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2220</span>–<span class="cit-lpage">2233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DSanchez-Moran%26rft.auinit1%253DE.%26rft.volume%253D21%26rft.issue%253D17%26rft.spage%253D2220%26rft.epage%253D2233%26rft.atitle%253DASY1%2Bmediates%2BAtDMC1-dependent%2Binterhomolog%2Brecombination%2Bduring%2Bmeiosis%2Bin%2BArabidopsis%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.439007%26rft_id%253Dinfo%253Apmid%252F17785529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIxLzE3LzIyMjAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.65" data-doi="10.1186/gb-2011-12-8-125"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sboner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mu</span>  <span class="cit-name-given-names">X. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenbaum</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auerbach</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerstein</span>  <span class="cit-name-given-names">M. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The real cost of sequencing: higher than you think!</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DSboner%26rft.auinit1%253DA.%26rft.volume%253D12%26rft.issue%253D8%26rft.spage%253D125%26rft.epage%253D125%26rft.atitle%253DThe%2Breal%2Bcost%2Bof%2Bsequencing%253A%2Bhigher%2Bthan%2Byou%2Bthink%2521%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-8-125%26rft_id%253Dinfo%253Apmid%252F21867570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-8-125&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21867570&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.66" data-doi="10.1186/gb-2009-10-9-r98"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hagmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ossowski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warthmann</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gesing</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Simultaneous alignment of short reads against multiple genomes.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R98</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DSchneeberger%26rft.auinit1%253DK.%26rft.volume%253D10%26rft.issue%253D9%26rft.spage%253DR98%26rft.epage%253DR98%26rft.atitle%253DSimultaneous%2Balignment%2Bof%2Bshort%2Breads%2Bagainst%2Bmultiple%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-9-r98%26rft_id%253Dinfo%253Apmid%252F19761611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-9-r98&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19761611&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.67" data-doi="10.1093/nar/gkt1004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schropfer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kobbe</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartung</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knoll</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puchta</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Defining the roles of the N-terminal region and the helicase activity of RECQ4A in DNA repair and homologous recombination in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">1684</span>–<span class="cit-lpage">1697</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1004%26rft_id%253Dinfo%253Apmid%252F24174542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQyLzMvMTY4NCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy8zODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.68" data-doi="10.1098/rspb.2013.1945"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferretti</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramos-Onsins</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Capilla</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farre</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolution of recombination in eutherian mammals: insights into mechanisms that affect recombination rates and crossover interference</span>. <abbr class="cit-jnl-abbrev">Proc Biol Sci</abbr> <span class="cit-vol">280</span>: <span class="cit-fpage">20131945</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc%2BBiol%2BSci%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.2013.1945%26rft_id%253Dinfo%253Apmid%252F24068360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxNzoiMjgwLzE3NzEvMjAxMzE5NDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.69" data-doi="10.1534/genetics.107.083899"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loudet</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durand</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Quantitative trait loci mapping in five new large recombinant inbred line populations of <em>Arabidopsis thaliana</em> genotyped with consensus single-nucleotide polymorphism markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">2253</span>–<span class="cit-lpage">2264</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSimon%26rft.auinit1%253DM.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2253%26rft.epage%253D2264%26rft.atitle%253DQuantitative%2BTrait%2BLoci%2BMapping%2Bin%2BFive%2BNew%2BLarge%2BRecombinant%2BInbred%2BLine%2BPopulations%2Bof%2BArabidopsis%2Bthaliana%2BGenotyped%2BWith%2BConsensus%2BSingle-Nucleotide%2BPolymorphism%2BMarkers%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.083899%26rft_id%253Dinfo%253Apmid%252F18430947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzIyNTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.70" data-doi="10.1073/pnas.7.8.235"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sturtevant</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1921</span> <span class="cit-article-title">A case of rearrangement of genes in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">235</span>–<span class="cit-lpage">237</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSturtevant%26rft.auinit1%253DA.%2BH.%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253D235%26rft.epage%253D237%26rft.atitle%253DA%2BCase%2Bof%2BRearrangement%2Bof%2BGenes%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.7.8.235%26rft_id%253Dinfo%253Apmid%252F16576597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo3OiI3LzgvMjM1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.71" data-doi="10.1101/gad.9.8.925"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Terasawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shinohara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hotta</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ogawa</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Localization of RecA-like recombination proteins on chromosomes of the lily at various meiotic stages.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">925</span>–<span class="cit-lpage">934</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DTerasawa%26rft.auinit1%253DM%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D925%26rft.epage%253D934%26rft.atitle%253DLocalization%2Bof%2BRecA-like%2Brecombination%2Bproteins%2Bon%2Bchromosomes%2Bof%2Bthe%2Blily%2Bat%2Bvarious%2Bmeiotic%2Bstages.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.9.8.925%26rft_id%253Dinfo%253Apmid%252F7774810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6NzoiOS84LzkyNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy8zODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.72" data-doi="10.1371/journal.pgen.0030176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vignard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siwiec</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chelysheva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vrielynck</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonord</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The interplay of RecA-related proteins and the MND1–HOP2 complex during meiosis in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1894</span>–<span class="cit-lpage">1906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DVignard%26rft.auinit1%253DJ.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253D1894%26rft.epage%253D1906%26rft.atitle%253DThe%2Binterplay%2Bof%2BRecA-related%2Bproteins%2Band%2Bthe%2BMND1-HOP2%2Bcomplex%2Bduring%2Bmeiosis%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030176%26rft_id%253Dinfo%253Apmid%252F17937504%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030176&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17937504&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251072900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.73" data-doi="10.1016/0092-8674(94)90438-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weiner</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Chromosome pairing via multiple interstitial interactions before and during meiosis in yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">977</span>–<span class="cit-lpage">991</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DWeiner%26rft.auinit1%253DB.%2BM.%26rft.volume%253D77%26rft.issue%253D7%26rft.spage%253D977%26rft.epage%253D991%26rft.atitle%253DChromosome%2Bpairing%2Bvia%2Bmultiple%2Binterstitial%2Binteractions%2Bbefore%2Band%2Bduring%2Bmeiosis%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252894%252990438-3%26rft_id%253Dinfo%253Apmid%252F8020104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(94)90438-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8020104&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F385.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NV42500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.74"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weinig</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ungerer</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dorn</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kane</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toyonaga</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Novel loci control variation in reproductive timing in <em>Arabidopsis thaliana</em> in natural environments.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">1875</span>–<span class="cit-lpage">1884</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWeinig%26rft.auinit1%253DC.%26rft.volume%253D162%26rft.issue%253D4%26rft.spage%253D1875%26rft.epage%253D1884%26rft.atitle%253DNovel%2BLoci%2BControl%2BVariation%2Bin%2BReproductive%2BTiming%2Bin%2BArabidopsis%2Bthaliana%2Bin%2BNatural%2BEnvironments%26rft_id%253Dinfo%253Apmid%252F12524356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Mi80LzE4NzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.75" data-doi="10.7554/eLife.01426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wijnker</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Velikkakam James</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becker</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klasen</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genomic landscape of meiotic crossovers and gene conversions in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e01426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft_id%253Dinfo%253Adoi%252F10.7554%252FeLife.01426%26rft_id%253Dinfo%253Apmid%252F24347547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NToiZWxpZmUiO3M6NToicmVzaWQiO3M6MTA6IjIvMC9lMDE0MjYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvMzg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.76" data-doi="10.1073/pnas.1005931107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Parent-independent genotyping for constructing an ultrahigh-density linkage map based on population sequencing.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">10578</span>–<span class="cit-lpage">10583</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1005931107%26rft_id%253Dinfo%253Apmid%252F20498060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzIzLzEwNTc4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.77" data-doi="10.1534/genetics.104.033746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zou</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vision</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Improving quantitative trait loci mapping resolution in experimental crosses by the use of genotypically selected samples.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">170</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.033746%26rft_id%253Dinfo%253Apmid%252F15781710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTcwLzEvNDAxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzM4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.385.78" data-doi="10.1093/bioinformatics/btp394"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulz</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apweiler</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2865</span>–<span class="cit-lpage">2871</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp394%26rft_id%253Dinfo%253Apmid%252F19561018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMjEvMjg2NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy8zODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/3/385.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/3/371" title="The Evolving Puzzle of Autosomal Versus Y-linked Male Determination in Musca domestica" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/3/399" title="Genetic Markers for Western Corn Rootworm Resistance to Bt Toxin" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/3"><span>Volume 5 Issue 3, March 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/3.cover-source.jpg?itok=g434JgGA" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/3/385" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OWAxqZBC5kXrJfkh_Cl_0e1VYtqcqaEphrUc4SjLSsE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/3/385" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/3/385" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/3/385" class="active" data-icon-position="" data-hide-link-title="0">Rapid and Inexpensive Whole-Genome Genotyping-by-Sequencing for Crossover Localization and Fine-Scale Genetic Mapping</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459792" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-SyZGSaIeGeNFAojdKcr9ROGeZ_UmSVFz61NnmEW_2sQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/3/385.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/3/385" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-korgXkH-OfQrgDqECQazIjTWsXwdvCJ9RZaLF8IRHqo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/3/385" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/3/385.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459792" id="node459792--2" data-pisa="ggg;5/3/385" data-pisa-master="ggg;g3.114.016501" data-apath="/ggg/5/3/385.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid and Inexpensive Whole-Genome Genotyping-by-Sequencing for Crossover Localization and Fine-Scale Genetic Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7240-4889" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Beth A. Rowan</span>, <span class="highwire-citation-author" data-delta="1">Vipul Patel</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2114-7963" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Detlef Weigel</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5512-0443" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Korbinian Schneeberger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">385-398;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016501 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459792--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Beth A. Rowan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology, Max Planck Institute for Developmental Biology, 72076 Tübingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beth%2BA.%2BRowan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rowan%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABeth%2BA.%2BRowan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7240-4889" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Beth A. Rowan</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vipul Patel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Developmental Biology, Max Planck Institute for Plant Breeding Research, 50829 Cologne, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vipul%2BPatel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Patel%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVipul%2BPatel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Detlef Weigel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology, Max Planck Institute for Developmental Biology, 72076 Tübingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Detlef%2BWeigel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weigel%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADetlef%2BWeigel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2114-7963" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Detlef Weigel</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:weigel@weigelworld.org" class="" data-icon-position="" data-hide-link-title="0">weigel@weigelworld.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Korbinian Schneeberger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Developmental Biology, Max Planck Institute for Plant Breeding Research, 50829 Cologne, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Korbinian%2BSchneeberger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schneeberger%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKorbinian%2BSchneeberger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5512-0443" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Korbinian Schneeberger</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/3/385" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/3/385&amp;title=Rapid%20and%20Inexpensive%20Whole-Genome%20Genotyping-by-Sequencing%20for%20Crossover%20Localization%20and%20Fine-Scale%20Genetic%20Mapping" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/3/385&amp;title=Rapid%20and%20Inexpensive%20Whole-Genome%20Genotyping-by-Sequencing%20for%20Crossover%20Localization%20and%20Fine-Scale%20Genetic%20Mapping" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/3/385&amp;title=Rapid%20and%20Inexpensive%20Whole-Genome%20Genotyping-by-Sequencing%20for%20Crossover%20Localization%20and%20Fine-Scale%20Genetic%20Mapping" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/3/385&amp;text=Rapid%20and%20Inexpensive%20Whole-Genome%20Genotyping-by-Sequencing%20for%20Crossover%20Localization%20and%20Fine-Scale%20Genetic%20Mapping" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/3/385&amp;title=Rapid%20and%20Inexpensive%20Whole-Genome%20Genotyping-by-Sequencing%20for%20Crossover%20Localization%20and%20Fine-Scale%20Genetic%20Mapping" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/3/385&amp;t=Rapid%20and%20Inexpensive%20Whole-Genome%20Genotyping-by-Sequencing%20for%20Crossover%20Localization%20and%20Fine-Scale%20Genetic%20Mapping" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/3/385&amp;title=Rapid%20and%20Inexpensive%20Whole-Genome%20Genotyping-by-Sequencing%20for%20Crossover%20Localization%20and%20Fine-Scale%20Genetic%20Mapping" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/3/385&amp;title=Rapid%20and%20Inexpensive%20Whole-Genome%20Genotyping-by-Sequencing%20for%20Crossover%20Localization%20and%20Fine-Scale%20Genetic%20Mapping" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459792/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459792" id="node459792--3" data-pisa="ggg;5/3/385" data-pisa-master="ggg;g3.114.016501" data-apath="/ggg/5/3/385.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Rapid and Inexpensive Whole-Genome Genotyping-by-Sequencing for Crossover Localization and Fine-Scale Genetic Mapping</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-7240-4889" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Beth A. Rowan</span>, <span class="highwire-citation-author" data-delta="1">Vipul Patel</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-2114-7963" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Detlef Weigel</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-5512-0443" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Korbinian Schneeberger</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">385-398;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016501 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459792--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Beth A. Rowan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology, Max Planck Institute for Developmental Biology, 72076 Tübingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Beth%2BA.%2BRowan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rowan%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABeth%2BA.%2BRowan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7240-4889" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Beth A. Rowan</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Vipul Patel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Developmental Biology, Max Planck Institute for Plant Breeding Research, 50829 Cologne, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vipul%2BPatel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Patel%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVipul%2BPatel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Detlef Weigel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Molecular Biology, Max Planck Institute for Developmental Biology, 72076 Tübingen, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Detlef%2BWeigel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weigel%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADetlef%2BWeigel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2114-7963" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Detlef Weigel</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:weigel@weigelworld.org" class="" data-icon-position="" data-hide-link-title="0">weigel@weigelworld.org</a></span></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Korbinian Schneeberger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Developmental Biology, Max Planck Institute for Plant Breeding Research, 50829 Cologne, Germany</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Korbinian%2BSchneeberger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schneeberger%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKorbinian%2BSchneeberger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-5512-0443" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Korbinian Schneeberger</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=385&amp;rft.epage=398&amp;rft.atitle=Rapid%20and%20Inexpensive%20Whole-Genome%20Genotyping-by-Sequencing%20for%20Crossover%20Localization%20and%20Fine-Scale%20Genetic%20Mapping&amp;rft.volume=5&amp;rft.issue=3&amp;rft.date=2015-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Rowan%2C+Beth+A.&amp;rft.au=Patel%2C+Vipul&amp;rft.au=Weigel%2C+Detlef&amp;rft.au=Schneeberger%2C+Korbinian"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459792/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459792/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459792/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459792/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459792/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459792/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459792/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459792/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459792/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459792/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459792/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459792/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/3/385" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-22" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/3/385.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/3/385.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/3/385.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/3/385.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
