<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/3/427" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/3/427.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/3/427.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/3/427.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/3/427/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/3/427.atom" />
<meta name="HW.pisa" content="ggg;5/3/427" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Linkage Analysis and Map Construction in Genetic Populations of Clonal F1 and Double Cross" />
<meta name="DC.Identifier" content="10.1534/g3.114.016022" />
<meta name="DC.Date" content="2015-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Zhang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In this study, we considered four categories of molecular markers based on the number of distinguishable alleles at the marker locus and the number of distinguishable genotypes in clonal F1 progenies. For two marker loci, there are nine scenarios that allow the estimation of female, male, and/or combined recombination frequencies. In a double cross population derived from four inbred lines, five categories of markers are classified and another five scenarios are present for recombination frequency estimation. Theoretical frequencies of identifiable genotypes were given for each scenario, from which the maximum likelihood estimates of one or more of the three recombination frequencies could be estimated. If there was no analytic solution, then Newton-Raphson method was used to acquire a numerical solution. We then proposed to use an algorithm in Traveling Salesman Problem to determine the marker order. Finally, we proposed a procedure to build the two haploids of the female parent and the two haploids of the male parent in clonal F1. Once the four haploids were built, clonal F1 hybrids could be exactly regarded as a double cross population. Efficiency of the proposed methods was demonstrated in simulated clonal F1 populations and one actual maize double cross. Extensive comparisons with software JoinMap4.1, OneMap, and R/qtl show that the methodology proposed in this article can build more accurate linkage maps in less time." />
<meta name="DC.Contributor" content="Luyan Zhang" />
<meta name="DC.Contributor" content="Huihui Li" />
<meta name="DC.Contributor" content="Jiankang Wang" />
<meta name="article:published_time" content="2015-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Linkage Analysis and Map Construction in Genetic Populations of Clonal F1 and Double Cross" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In this study, we considered four categories of molecular markers based on the number of distinguishable alleles at the marker locus and the number of distinguishable genotypes in clonal F&lt;sub&gt;1&lt;/sub&gt; progenies. For two marker loci, there are nine scenarios that allow the estimation of female, male, and/or combined recombination frequencies. In a double cross population derived from four inbred lines, five categories of markers are classified and another five scenarios are present for recombination frequency estimation. Theoretical frequencies of identifiable genotypes were given for each scenario, from which the maximum likelihood estimates of one or more of the three recombination frequencies could be estimated. If there was no analytic solution, then Newton-Raphson method was used to acquire a numerical solution. We then proposed to use an algorithm in Traveling Salesman Problem to determine the marker order. Finally, we proposed a procedure to build the two haploids of the female parent and the two haploids of the male parent in clonal F&lt;sub&gt;1&lt;/sub&gt;. Once the four haploids were built, clonal F&lt;sub&gt;1&lt;/sub&gt; hybrids could be exactly regarded as a double cross population. Efficiency of the proposed methods was demonstrated in simulated clonal F&lt;sub&gt;1&lt;/sub&gt; populations and one actual maize double cross. Extensive comparisons with software JoinMap4.1, OneMap, and R/qtl show that the methodology proposed in this article can build more accurate linkage maps in less time.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/03/01" />
<meta name="citation_mjid" content="ggg;5/3/427" />
<meta name="citation_id" content="5/3/427" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/3/427" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/3/427.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/3/427.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/3/427.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.016022" />
<meta name="citation_pmid" content="25591919" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="427" />
<meta name="citation_lastpage" content="439" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Luyan Zhang" />
<meta name="citation_author_institution" content="The National Key Facility for Crop Gene Resources and Genetic Improvement, Institute of Crop Science, and CIMMYT China Office, Chinese Academy of Agricultural Sciences, Beijing 100081, China" />
<meta name="citation_author" content="Huihui Li" />
<meta name="citation_author_institution" content="The National Key Facility for Crop Gene Resources and Genetic Improvement, Institute of Crop Science, and CIMMYT China Office, Chinese Academy of Agricultural Sciences, Beijing 100081, China" />
<meta name="citation_author" content="Jiankang Wang" />
<meta name="citation_author_institution" content="The National Key Facility for Crop Gene Resources and Genetic Improvement, Institute of Crop Science, and CIMMYT China Office, Chinese Academy of Agricultural Sciences, Beijing 100081, China" />
<meta name="citation_author_email" content="wangjiankang@caas.cn" />
<meta name="citation_author_email" content="jkwang@cgiar.org" />
<meta name="citation_reference" content="citation_title=Principles of Plant Breeding;citation_author=R. W. Allard;citation_year=1999" />
<meta name="citation_reference" content="citation_title=Introduction to the Mathematical Theory of Genetic Linkage;citation_author=N. T. J. Bailey;citation_year=1961" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=Ś. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_title=Genetic maps of RAPD, AFLP and ISSR markers in Ananas bracteatus and A. comosus using the pseudo-testcross strategy.;citation_pages=186-192;citation_volume=123;citation_year=2004;citation_doi=10.1046/j.1439-0523.2003.00924.x" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=R. A. Fisher;citation_title=The detection of linkage with “dominant” abnormalities.;citation_pages=187-201;citation_volume=6;citation_year=1935;citation_doi=10.1111/j.1469-1809.1935.tb02227.x" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Fregene;citation_author=F. Angel;citation_author=R. Gomez;citation_author=F. Rodriguez;citation_author=P. Chavarriaga;citation_title=A molecular genetic map of cassava (Manihot esculenta Crantz).;citation_pages=431-441;citation_volume=95;citation_year=1997;citation_doi=10.1007/s001220050580" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=C. A. Hackett;citation_author=L. B. Broadfoot;citation_title=Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.;citation_pages=33-38;citation_volume=90;citation_year=2003;citation_issue=1;citation_pmid=12522423;citation_doi=10.1038/sj.hdy.6800173" />
<meta name="citation_reference" content="citation_journal_title=J. Genet.;citation_author=J. B. S. Haldane;citation_title=The combination of linkage values, and the calculation of distance between the loci of linked factors.;citation_pages=299-309;citation_volume=8;citation_year=1919" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=J. B. S. Haldane;citation_author=C. A. B. Smith;citation_title=A new estimate of the linkage between the genes for haemophilia and colour-blindness in man.;citation_pages=10-31;citation_volume=14;citation_year=1947;citation_pmid=18897933;citation_doi=10.1111/j.1469-1809.1947.tb02374.x" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=M. Hemmat;citation_author=N. F. Weeden;citation_author=A. G. Manganaris;citation_author=D. M. Lawson;citation_title=Molecular Marker Linkage Map for Apple;citation_pages=4-11;citation_volume=85;citation_year=1994;citation_issue=1;citation_pmid=7907101" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=D. D. Kosambi;citation_title=The estimation of map distances from recombination values.;citation_pages=276-278;citation_volume=12;citation_year=1944" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=S. Kunkeaw;citation_author=S. Tangphatsornruang;citation_author=D. R. Smith;citation_author=K. Triwitayakorn;citation_title=Genetic linkage map of cassava (Manihot esculenta Crantz) based on AFLP and SSR markers.;citation_pages=112-115;citation_volume=129;citation_year=2010;citation_doi=10.1111/j.1439-0523.2009.01623.x" />
<meta name="citation_reference" content="citation_journal_title=Acta Agron. Sin.;citation_author=A. Li;citation_author=Q. Liu;citation_author=Q. Wang;citation_author=L. Zhang;citation_author=H. Zhai;citation_title=Establishment of molecular linkage maps using SRAP markers in sweet potato.;citation_pages=1286-1295;citation_volume=36;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=J. Henan. Agri. Univ.;citation_author=Y. Li;citation_author=X. Li;citation_author=J. Chen;citation_author=B. Zhou;citation_author=Z. Zhou;citation_title=Identification of QTL for traits related to flowing time in maize based on four-way cross population.;citation_pages=231-240;citation_volume=47;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Operations Research;citation_journal_abbrev=Operations Research;citation_author=S. Lin;citation_author=B. W. Kernighan;citation_title=An Effective Heuristic Algorithm for the Traveling-Salesman Problem;citation_pages=498-516;citation_volume=21;citation_year=1973;citation_issue=2;citation_doi=10.1287/opre.21.2.498" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=D. Liu;citation_author=C. Ma;citation_author=W. Hong;citation_author=L. Huang;citation_author=M. Liu;citation_title=Construction and analysis of high-density linkage map using high-throughput sequencing data.;citation_pages=e98855;citation_volume=9;citation_year=2014;citation_issue=6;citation_pmid=24905985;citation_doi=10.1371/journal.pone.0098855" />
<meta name="citation_reference" content="citation_journal_title=Acta Agron. Sin.;citation_author=X. Liu;citation_author=J. Mao;citation_author=X. Lu;citation_author=L. Ma;citation_author=K. S. Aitken;citation_title=Construction of molecular genetic linkage map of sugarcane based on SSR and AFLP markers.;citation_pages=177-183;citation_volume=36;citation_year=2010;citation_doi=10.3724/SP.J.1006.2010.00177" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=C. Maliepaard;citation_author=J. Jansen;citation_author=J. W. van Ooijen;citation_title=Linkage analysis in a full-sib family of an outbreeding plant species: overview and consequences for applications.;citation_pages=237-250;citation_volume=70;citation_year=1997;citation_doi=10.1017/S0016672397003005" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=N. E. Morton;citation_title=Sequential tests for the detection of linkage.;citation_pages=277-318;citation_volume=7;citation_year=1955;citation_issue=3;citation_pmid=13258560" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=E. V. Nordheim;citation_author=D. M. O’Malley;citation_author=R. P. Guries;citation_title=Estimation of recombination frequency in genetic linkage studies.;citation_pages=313-321;citation_volume=66;citation_year=1983;citation_pmid=24263933" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=J. Ott;citation_title=Estimation of the recombination fraction in human pedigrees: efficient computation of the likelihood for human linkage studies.;citation_pages=588-597;citation_volume=26;citation_year=1974;citation_issue=5;citation_pmid=4422075" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. Ritter;citation_author=C. Gebhardt;citation_author=F. Salamini;citation_title=Estimation of recombination frequencies and construction of RFLP linkage maps in plants from crosses between heterozygous parents.;citation_pages=645-654;citation_volume=125;citation_year=1990;citation_issue=3;citation_pmid=1974227" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=E. Ritter;citation_author=F. Salamini;citation_title=The calculation of recombination frequencies in crosses of allogamous plant species with applications to linkage mapping.;citation_pages=55-65;citation_volume=67;citation_year=1996;citation_doi=10.1017/S0016672300033474" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Säll;citation_author=N. O. Nilsson;citation_title=The robustness of recombination frequency estimates in intercrosses with dominant markers.;citation_pages=589-596;citation_volume=137;citation_year=1994;citation_issue=2;citation_pmid=8070668" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=C. A. B. Smith;citation_title=Some comments on the statistical methods used in linkage investigations.;citation_pages=289-304;citation_volume=11;citation_year=1959;citation_pmid=13831904" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=P. Stam;citation_title=Construction of integrated genetic linage maps by means of a new computer package: JoinMap.;citation_pages=739-744;citation_volume=3;citation_year=1993;citation_doi=10.1111/j.1365-313X.1993.00739.x" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=Z. Sun;citation_author=H. Li;citation_author=L. Zhang;citation_author=J. Wang;citation_title=Estimation of recombination frequency in bi-parental genetic populations.;citation_pages=163-177;citation_volume=94;citation_year=2012;citation_issue=3;citation_pmid=22892204;citation_doi=10.1017/S0016672312000353" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. D. Tanksley;citation_author=M. W. Ganal;citation_author=J. P. Prince;citation_author=M. C. de Vicente;citation_author=M. W. Bonierbale;citation_title=High density molecular linkage maps of the tomato and potato genomes.;citation_pages=1141-1160;citation_volume=132;citation_year=1992;citation_issue=4;citation_pmid=1360934" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. van Os;citation_author=S. Andrzejewski;citation_author=E. Bakker;citation_author=I. Barrena;citation_author=G. J. Bryan;citation_title=Construction of a 10,000-marker ultradense genetic recombination map of potato: providing a framework for accelerated gene isolation and a genomewide physical map.;citation_volume=173;citation_year=2006;citation_pmid=16582432;citation_doi=10.1534/genetics.106.055871" />
<meta name="citation_reference" content="citation_title=JoinMap 4.0: Software for the Calculation of Genetic Linkage Maps in Experimental Populations;citation_author=J. W. van Ooijen;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. W. Van Ooijen;citation_title=Multipoint maximum likelihood mapping in a full-sib family of an outbreeding species.;citation_pages=343-349;citation_volume=93;citation_year=2011;citation_issue=5;citation_pmid=21878144;citation_doi=10.1017/S0016672311000279" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Wang;citation_author=M. van Ginkel;citation_author=D. Podlich;citation_author=G. Ye;citation_author=R. Trethowan;citation_title=Comparison of Two Breeding Strategies by Computer Simulation;citation_pages=1764-1773;citation_volume=43;citation_year=2003;citation_issue=5;citation_doi=10.2135/cropsci2003.1764" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. Wang;citation_author=M. van Ginkel;citation_author=R. Trethowan;citation_author=G. Ye;citation_author=I. Delacy;citation_title=Simulating the Effects of Dominance and Epistasis on Selection Response in the CIMMYT Wheat Breeding Program Using QuCim;citation_pages=2006-2018;citation_volume=44;citation_year=2004;citation_issue=6;citation_doi=10.2135/cropsci2004.2006" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=R. L. Wu;citation_author=C. X. Ma;citation_author=I. Painter;citation_author=Z. B. Zeng;citation_title=Simultaneous maximum likelihood estimation of linkage and linkage phases in outcrossing species.;citation_pages=349-363;citation_volume=61;citation_year=2002;citation_issue=3;citation_pmid=12027621;citation_doi=10.1006/tpbi.2002.1577" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=R. L. Wu;citation_author=C. X. Ma;citation_author=S. S. Wu;citation_author=Z. B. Zeng;citation_title=Linkage mapping of sex-specific differences.;citation_pages=85-96;citation_volume=79;citation_year=2002;citation_issue=1;citation_pmid=11974606;citation_doi=10.1017/S0016672301005389" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. Yamamoto;citation_author=T. Kimura;citation_author=M. Shoda;citation_author=T. Imai;citation_author=T. Saito;citation_title=Genetic linkage maps constructed by using an interspecific cross between Japanese and European pears.;citation_pages=9-18;citation_volume=106;citation_year=2002;citation_issue=1;citation_pmid=12582866" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=L. Zhang;citation_author=S. Wang;citation_author=H. Li;citation_author=Q. Deng;citation_author=A. Zheng;citation_title=Effects of missing marker and segregation distortion on QTL mapping in F2 populations.;citation_pages=1071-1082;citation_volume=121;citation_year=2010;citation_issue=6;citation_pmid=20535442;citation_doi=10.1007/s00122-010-1372-z" />
<meta name="citation_reference" content="citation_journal_title=J. Integr. Plant Biol.;citation_title=QTL Mapping with background control in genetic populations of clonal F1 and double cross.;citation_author=L. Zhang;citation_author=H. Li;citation_author=J. Wang;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Hereditas (Beijing);citation_author=X. Zhang;citation_author=T. Yin;citation_author=Q. Zhuge;citation_author=M. Huang;citation_author=L. Zhu;citation_title=RAPD linkage mapping in a Populus deltoides×Populus euramericana F1 family.;citation_pages=209-213;citation_volume=22;citation_year=2000" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Linkage Analysis and Map Construction in Genetic Populations of Clonal F1 and Double Cross" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3.cover-source.jpg" />
<meta name="twitter:description" content="In this study, we considered four categories of molecular markers based on the number of distinguishable alleles at the marker locus and the number of distinguishable genotypes in clonal F1 progenies. For two marker loci, there are nine scenarios that allow the estimation of female, male, and/or combined recombination frequencies. In a double cross population derived from four inbred lines, five categories of markers are classified and another five scenarios are present for recombination frequency estimation. Theoretical frequencies of identifiable genotypes were given for each scenario, from which the maximum likelihood estimates of one or more of the three recombination frequencies could be estimated. If there was no analytic solution, then Newton-Raphson method was used to acquire a numerical solution. We then proposed to use an algorithm in Traveling Salesman Problem to determine the marker order. Finally, we proposed a procedure to build the two haploids of the female parent and the two haploids of the male parent in clonal F1. Once the four haploids were built, clonal F1 hybrids could be exactly regarded as a double cross population. Efficiency of the proposed methods was demonstrated in simulated clonal F1 populations and one actual maize double cross. Extensive comparisons with software JoinMap4.1, OneMap, and R/qtl show that the methodology proposed in this article can build more accurate linkage maps in less time." />
<meta name="og-title" property="og:title" content="Linkage Analysis and Map Construction in Genetic Populations of Clonal F1 and Double Cross" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/3/427" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In this study, we considered four categories of molecular markers based on the number of distinguishable alleles at the marker locus and the number of distinguishable genotypes in clonal F1 progenies. For two marker loci, there are nine scenarios that allow the estimation of female, male, and/or combined recombination frequencies. In a double cross population derived from four inbred lines, five categories of markers are classified and another five scenarios are present for recombination frequency estimation. Theoretical frequencies of identifiable genotypes were given for each scenario, from which the maximum likelihood estimates of one or more of the three recombination frequencies could be estimated. If there was no analytic solution, then Newton-Raphson method was used to acquire a numerical solution. We then proposed to use an algorithm in Traveling Salesman Problem to determine the marker order. Finally, we proposed a procedure to build the two haploids of the female parent and the two haploids of the male parent in clonal F1. Once the four haploids were built, clonal F1 hybrids could be exactly regarded as a double cross population. Efficiency of the proposed methods was demonstrated in simulated clonal F1 populations and one actual maize double cross. Extensive comparisons with software JoinMap4.1, OneMap, and R/qtl show that the methodology proposed in this article can build more accurate linkage maps in less time." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3.cover-source.jpg" />
    <title>Linkage Analysis and Map Construction in Genetic Populations of Clonal F1 and Double Cross | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"-_ytNW7ivI69n7p3XpKn-Pz9xEoEzoLAhr5I2vZMEY8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459784","apath":"\/ggg\/5\/3\/427.atom","pisa":"ggg;5\/3\/427","ac":{"\/ggg\/5\/3\/427.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/3\/427.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/3\/427"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/3\/427"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/3\/427":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ybuPt+qCzD0MRwmpBW3JpsNmf+WI1cWvKxiOwMBTfsUqrHPIqbhNgASTnVHvBgtVQhdSVBRFJdr84BWGand7NdcbpJ82zsfENd2BettxAjY+B\\\\\\\/up15T8aN1gX4OfCsMSWOnVJp6v1dvy4P+JTNza4fIbHCrNwP4clKeEhRXiyYgEb0SzWIR0NKw9k+ta0L\\\\\\\/p6yX\\\\\\\/pDbYeDi8x1km1syAZ12jbExCzCwEqcdZEMXa9OIcb6MeVvGVXh0lXf8axkYRhA50t5V2y0EzjY35xLhYWpOSEipqYKuKCBwhJAe0bWF7\\\\\\\/t0SZXMofoE3dEWQQ0n9e4WeaujO4d8dYdzEJS7ohSmdnKXHqjaf9IMyqoSiPTDIzejVQQZy8tJsh83fiHXMSvQRA2i7hOEEGnxAZvsWDpE7hdyU4gDWzcDO2oI5X9J2qHleA+JVA4JOqBitezce\\\\\\\/laTGxb6Dst5uZFhS87Sbu771CUSG14lJIR4GYcjlqqGfytbTp5CmVoIfLsPAqzsagFVlf8gP7uTF7KfHDWpP\\\\\\\/4YD2D6TjN5rZ2oRYIHosYwkOGgVspmwYuJytYHd4rbdJowGTe92U6yjVDrtZioFl9\\\\\\\/2HwvpSw1TQRckEhezUnAIqN\\\\\\\/UfmFDNGJfURt9fHSWrZvmGWNz5MXdu+FOWgi+CfbINzaYtPKcS1NYBAHjYoJ80iUP6LDiY+QYnLqbm324T74S\\\\\\\/McQq4fhAdG6U4xS3FHKhWSZtyXOHggpUG2mSTWSIEDP77kJXcNe3JrZao6tWHyBPH\\\\\\\/kcAi3TeEQQlZlT3a\\\\\\\/GCBEkDEaSUOylSpkhviwy7HxGwghrPH1daxZSVRuTqBJ9\\\\\\\/rOeoPuzRAcAFYQgJImcOicLSiY322TF0sCk+TvY\\\\\\\/IA2i+4u33nYdSpIzZIlWyFBLGxC4JUQYjeQi4bKN6kRxCAulr0ifRSpl4H\\\\\\\/JzD8PJ\\\\\\\/rRdf\\\\\\\/6s\\\\\\\/gLIaHeV7hcuB3VzPkCz5Kh\\\\\\\/SxU37zdiC9pKhgAaB9MLpyVQn6\\\\\\\/JBHQ4wmcEVFQ8l3WfeWuwR6saxJ0hRmpKyCbIMakQWAfM+X3wt1ELtxNZUqjsLPpxx+konMBPTq2UkTs\\\\\\\/ufCCUfPp0L9Doev+DYGKXRqoCEx7dxhu2pxbaM4693oXpqqo9ximy3Wx++J1P0lYQ\\\\\\\/n1BAZXjN\\\\\\\/Mg427sd7XiFt1nqi40uBTjjtafx6JoMNNos+zgS4mQ\\\\\\\/2r\\\\\\\/bXhqMQ1vEJ\\\\\\\/NVuzE9\\\\\\\/LsccPyb5WLoJenWtub91RN0ifdIJQFyxEzwptEpmKyATN1z7CVpsIcUv8\\\u0022,\\\u0022iv\\\u0022:\\\u0022m7FLJmX10VxBFcqOzXw3Lg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002214a0fee6fd1dce4c7049282a17211c366464a82298b91a083d52c9d8acdf8c29\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YncLL2PbNt1yu7OWL91rRYKdUNPN2UGqCDgMMJFS3ePvVYeiWe4BHarzXUHulr0aF5AI7vSIId9ATVUFLApzcY82WGIyoGIiZw6f9LN3LQmc\\\\\\\/0IuPkoh0r5FCgyuEiRmjYa5RWVfLbZOW6ZKpMkYyJ4oW3rjRO\\\\\\\/d1bb0NtvRvESdw9XJ1SNpjd7SKKBwUXCHmMTutGMN7adPOHwCMzs0VDWHx3kZTZnn8QAl2VHbYaK8PI5+lKALJIyLheqb0KNCT2Xj+COfyum0Cttt78CCc9HJBvSQYx3KIoEvp5ep4zmaSE1tKS\\\\\\\/t3ntpfXDYmIKrTg4ER\\\\\\\/eo3yYAvJKFf4DOCWaDROBdtpuSwEeNBnSAEkZ0cBCOvrmivu\\\\\\\/ds3Ea44tol34FilhST3t7VXELDxKlluP+kCgxwhp+TdNxJA9Lu\\\\\\\/Low5FPBHoTep23kYTm\\\\\\\/spVuyawhpFPO8wW8xspbAvQ+KYb1ZZ7E0wqDdxw\\\\\\\/rp2eOXQ0PSfe9lOt259lhC2I1+90MaQoIv33+VcpZwoIrWG5Ti9VK7pCGFkjRrxAE\\\\\\\/H3840CTg2iUcf3w38QIjVNlFWJ\\\\\\\/sRta96paNvChdIi5PqOpUxmqGsie2eWHzmgVV0JMyIArY2laYb8P0NDyLgG9gfj4dfA6W6QaLVHz523oj2Nxow0XDWaoxHI15qyDH3BaoE3CXPsWSMTVk2md30b8SfS0u4h6cmrUI5Ci9MRh+CmB2EdL4MZXmzQFiDpEh3C99VQl089oUV\\\\\\\/TUg\\\\\\\/b9as9E5Gftd2lqATewxSuhFfPeYYG8asrlBg1D0tBkeVkzE8H2HL4c0z1nke\\\\\\\/\\\\\\\/8OeeVwDTbm\\\\\\\/sjRasJVWSGnVDdu7416VS32AB\\\\\\\/kLmxHDeGQJVykUsviQUAKgJSo16yVKkbaRXjIm9n8uGQMc4lkkTARYptZZ88MudOMhopYWfXqMv0Z4w3uAXVga8F44uUmb8yltNqBqgsbuoJvzmWvPYthOG3qRXr\\\\\\\/YDrFQ9pbfuwliDzFxzZHuW\\\\\\\/qZ5J4owLsNM1+uQbt8AWE04+9VvEVNthIzWDPcSfADVQAuo5YeltiVDviPCVRiQ52I+dSnOmNkghihaGtM5rOggcfp6eRvbd0X+GSZY8TAwlxiMMMfxoe6eQNdGoQUGGX1nVxR\\\\\\\/UxvEFrNVZ9CWex0P0xE83Cp5e9U8vUReDDuo7yczo+O0zODhICtxd+QJrDFiwjmmtax9hzE9XeCTihH71bSWUoxrWSCrGaihEXmPLIAs5Vs0DhlLlX25yDV\\\\\\\/CzruLHmV105+g0s9gG2g6DcPp91o4mF38WOK0JcxDS68HAiFiJshIhLxWnk6DcWAfjp3COvdqDzLzvxM5EsR+DqlRWq\\\\\\\/SBfgOVNeQHEOasPD6RVF7bkyHFGc8IvqvKo1guVdG89FyLnKCf2XAjkI6tNBpwBYBSvI0Xp54tharHEYjNKDGGfOfUqjsIRmhD8QmYxPTLYxFlJLPZXuDZ6G8N3P0dEkIaj0yY5x5k7dYLx6kBX988kGsAnWyOmQXarlMfdUyMDdShL9evDp3HnHf7o1Oou89kqyANh6dQyFlbsCA2rPLN10KgDdT7ivkVvlZGeEQW0CmxkfNtbVDgPGogkYZIz6wOHKWWeavj4GgtZBMWSqy1KfYBXjdVkAkHHYECQTq6DA9+PzIskVMlMDTsqGT3jKRdXL+Sz+6PNSG60wBJsdZ3c+CqNaBpD2imtDXEUM4lc39xxQdD8G5DsivQyKkoIqVh2oD0I3jEZ5Woes7JZWLLERoh0761lx1MPlsB6Jm1PyzO9cHL\\\\\\\/+\\\\\\\/oEYm02nb64PLhNTtb7kOjCHTI+47JzF2RgZnMDIz5ucY76cHjmKNlr6yyCXm1tkIpNBi8otguKSU7EIySz\\\\\\\/MaxDU4oM7wtU1RAsZdRR3R4lr9uj72v+nKGH6EKqMKoz96JdRmi+GsJQg1T\\\\\\\/OHAa3TxO0grYrFnSdySCqYDM7ICDyW5zqfGUTHGduIbZn\\\\\\\/5fLv9yd\\\\\\\/Ta+LlVlBsVwySZGfeRPmGSx+mlRqIEcwG87LohkE3ePCRbEBQuefAKMNpxGylUuRSnpNRqWHgSsAo2pcIfAinmwlvA4SPUThzeyQm3fFgP8VoBbiL1eozaYrYd2dELyWOLytbjTcLroho3GW7FkgDDy5pJQZjbfftbUTwZOo0N66s79OwsMJlXNz0AT+Kvz8titvA1iSIU6c9DD4785xOOgtW\\\\\\\/76DKAMQkS5629pblyeiiVyWjY4iMstb5Ow54mmzNzmlu51pHxcLYP2Wt0LtTcstWmBVz4kf5YU0Nwbx+an2oLX7zmvKa0AnOTR05a4lTGXrd4KzdSU3zF5dQ37RT65\\\\\\\/uU249ttXD\\\\\\\/r6hSjaSzTrGpNX7rj88aLA50+VI88y7OFqUmRU\\\\\\\/RNBvmX7hvxulBbz90XLEpKLFjV3r4WtdJKYcV6cIV4GltUSsk7YtjIPffVSSUqYe7iX1V2Tjdi0Gs\\\\\\\/3sA4+lHbNG29yslhDoX6pt7uac7+Vou3TMD70yfrnpRsOmWIx8BS3H9QsJ6SPraKQJHhbx8y0r9+5US6eZZxuwmFCrVMiiTQmjIqJaW4l3Zljk6cJXTQ80QMVZRcmF3duGFQgnCRQo7fK+knFH0bdAQ4RY7xd6zRL91JBBAa3fFzUDXi3HF8S1ar6xnUz290KO3dFlYnZtaokValJxxC57qUDkQnV8Y8F1I3UmzWJPz\\\\\\\/0+Os9EHAAB82TFoOQds8PrcUGnxX\\\\\\\/CwtkzE9IKfaG0mEXkJlIrbopgfc0XxtZnnyrGiZFQv4YxXWNGbSAgFu2fnEoPkdLKusgCREEBGQ1B3Ji9zV7s2IlTyGkmVKXARCSJm4DIcHTvnFVzc\\\\\\\/stZIznmfsXypR2dzOb7l1BFdqVxSYF69\\\\\\\/G55Ornc+3q4PmZjLoM\\\\\\\/02Qp9H+yOFwt2n+pXRBGd0FkE04xGVO9+Q8CO9tulaKyML6UC8lid+hntryM4Rdj0xoHBYqveKEw53j+QMZ97k+szFjGbJ0x11f1c\\\\\\\/cSPuynu\\\\\\\/TFw4E1barx6GTBkBjuK2jx7o\\\\\\\/uJP9KySkvZbdwnOs04K2sDdlk5Mx4hrTG4qqouH6+p+0I0cjyyp50nGW3vbMifxYTylDtydE4f+CY7i6cU\\\\\\\/MTeXnUCraLfxVuS3BTJ6Keek67amkO7cg\\\\\\\/cAC0Ob6bhaMIz5zRrwq9+3V25uOxHpWlXNf\\\\\\\/BR4\\\\\\\/tpkySmn5ZlassN+6\\\\\\\/j3SYjxOpSPo6iYqpDo1wpf+yUJuRdTMkH\\\\\\\/DRyDKV\\\\\\\/cXszF8tMYIX+QBUbP\\\\\\\/WPBY+nIi8CjeqPgHC1ZkUAiQ0MJc8EdT4Lt9HJ4f2J+5ai97bWhs60jaS9NyOuWuL0IBbThgjrteTV6A8WWjYzbL6mQltX7FDSbUuF6x9NxX73BjOX14wwgPH1hLOSF56E1y+Txz2QJnxEGoF7lNZbsKUf1FVXgNzKl4D28F7HYbBkOdL7sQ7GfQ\\\\\\\/zpcaNXzjzJTU5rWLtn6H7y9RLZMb482c9OogH+EhZkqpD641+4nXA0HyNOo\\\\\\\/hPThhGmiToFCSdQLVhgL4q9Kwwx\\\\\\\/KaZj7ZgwCU2lRMSIJoc8aXWAGmtak05X2eYfMwR9qcD8gmvWr07+Y0YHluCRmhfoDJMuS9xUji389SqqWSzyzx69iLxfYLah6Ci6qhPQI9enBgqQXFosTZdJxczQmRZbvgbNh6WoiQfyJZQN0gty9uHQ1mS3d9gW36Qj3uXT\\\\\\\/4sA8pItD2kWDEjWnvAUxZZDlJjmoTjf\\\\\\\/Rhu4\\\\\\\/QiXolk1R5uMLvytEbxe4MIoHo1MzUbgQM4s7XzOG0sZJ5D+gqCua+7Kp6ISd2JdzAJvzmDPbUXHmR2cV0VE3IQbj3qrAdgcpjwNjy3DPPVsz837tdyBMi1BvNiSCd6R8HXwrtCJPX+gtVP92svd5dcfzoSBSj4yv3pNLX0w7F7ud2EyJZJdO7nh3WIZVV2WgnmFfyYIcviDDnl5Hzm6QPLRWZEN3XOs\\\\\\\/z2GE5aSWzyPVhW0FfhTUvBA7lQO584DXsuE0SL4qFXbjqkVuu1KPYwGFOGhAowcvRhy6nImj9\\\\\\\/Z3PGKRvWZerznZB8PAnHa2f13krF+nxIWmlpcqFjV8kUsmAXkDG86J3yKYy1dqx9x79IdNUuY4QLEKW9OUHLMQqkR5zxVDLDLAVk6GJbU7ap7cJKO790TjPclnwIixNzfnnrNx2VU7K+TwiGUicQcLnWLIX0snZ6CuCUj4WyvI0dJz+l4IYb5RmN2fVRVtXQ68Dh0MHNVvlloBmSn5Z5YZ6Hq+OyiVmN\\\\\\\/gfcw7NpwSd69oOAC41\\\\\\\/aw8ukKRX3\\\\\\\/fuXwUGLmSgEPdosTGamTd62RWBCkEVcnxyMqiRrrf278eLnqrM4JE1Ue9xcI2eaKDpAXYheyQc0UJtSqyiZjaxHa9mM3J9ugVc4I0\\\\\\\/7xIaYJ4l4ORiJYccQVZI0OG4B5MGkrZHm5LXDhvx5AAW1ZzVR0lXDjGgyozOwiR9zsYh6plzJaa2btLrZjJeni\\\\\\\/OhT8a6VfMp3T+ctA199+0te6n95mUfYQBYSXYUaPYY1c8iGYqep0kqc9VPWlUjex+D+tGc2U8jU1oVKl+yTijxG5nJZjjZrq1QO1ItRjRElOnxmZvGWk5E16sHCAOqFV7\\\\\\\/F9tYcl0RuOBVJx2MmE++rfFSdOpIknm\\\\\\\/MMh9KS4ZCNRrThJp0k0zF\\\\\\\/iqrKRbUosRI\\\\\\\/CKAR9JLAejg4c7599I7DBT800K0yvUbEwTwABhS8qjptrY3t4DFUMQ4yog6FgMqxb550tj3oO0GtjfJbeVRPCFfqlsR\\\\\\\/xzbXw86+C13fsiGwvkFDqJkeiuuDrj3aP8fP42+laUyVtMaQUiWpPNSBpiiBGBWfBlBtHs1bxce8lmQxRRSN8ZWPcfcr2uBTURncGbK57Oaryhfv32CvZJNj3OnBRRIHNKYda4lprlKWb7F0cjMYnmCZJowRk\\\\\\\/WXrnI6foPG4l5MsOUNn\\\\\\\/I73uQ+sTM9Vx3RCdwosObokMFZKhcYesQg1pXjUcAconyTWNTSvjRhAOWzdYeq5KoJLCONEEMWOqUdJbVGaHScC6+1q9tpVpXlluEivlqCoN2jDjMu7KgyLxQ5ChK\\\\\\\/Z9Ai571cuM2oNZwsG4Z8T1GwjDS5gQNVYkRSJkdVKaYUuwy\\\\\\\/pXUPH87OdW2MxueQephXdQiNrn6SVaqcxwFJagtN4DbVFqKjs9X7+JGEWmWFOpZfRC8Nws47h0mIrVgMOIXim9XNPOaXKdFjhQELbMGHz47ozVDIWbPtl0w72bt9KpSFflGAxZHHrwZJEOrV\\\\\\\/lYaWbJn\\\\\\\/xqS5WAG2HgVIHhywYpvpbKn\\\\\\\/1GDPKcf4tJteQ5GACd6i1eI34rJLFLMPCG5d3Fww59g32QfiZeCuibLXf1YdSfsS0zsFEIvtXROTXlmEdpDkN90O88l0wxmnGZm693PgBZOY1c3vHt1IVy33H7qhioMAjXWPnH95N5GYhhNUi04Akbqzpq95qU8od9IJJJY3pYCO17+MRIviIN5B4mhGZONvIKoecM\\\\\\\/\\\\\\\/VAd8De0Irk5GS0HG0+gPmOQOO6s9IsR2+uX8GgwVi\\\\\\\/u0cFP0KJJ8D0WFXjZNAC4nDDJIB\\\\\\\/p4P4YLHR1AKexMdOdyW+22UT8Me9dPNYDXCQPG0lTnb7XdCt5mEpO64\\\\\\\/JAWgLo2HaEfhU570AVhIro2g\\\\\\\/aFvTn3CDYa\\\\\\\/bEN995awPjyxRydDkC0Fqi7QZc5SPeCrgpxstqZQlxxPeZO+ygPeB4suuMormXUz\\\\\\\/p2werdnifk7AyPXCE\\\\\\\/8HfzVEXoSaDZDOQyoYgBtyQhBSpq\\\\\\\/8puyt2UUjX6DSJ6npe1dnlCXmMEIXTDOn2OGIO2jZ9+qz3jAiCWcsO1bqxfBBqeOxPE5wGZ9pEN4FU7CGi+DyKeOY6rW01+FSra3V+1EhTygARMJOBojRCO5FCgf6iFb\\\\\\\/u9bxXBR5ws0GpKaSV4wo7eFoPFODxJmuUmPToaG2xpwM5MFKljtZg4\\\\\\\/Q8rodcXIvnLAF9eag4Q5dOKxmOWhGhHaQa9+5CTGunI8aYn8JOBu+d+LKINrLbyItzL7ZsjHDWYquuRdgtKq10Xa7IGHTvR5D5KuRE1vh42a5jPkYKhq\\\\\\\/0lDRz28IAY6GG7QV7Le67aL9dgNCCgiS94Z8pniQ5p+O2VJjk17kwq749AkuSSZmg0Pgcm8+w0NLBmp9hrQU9GkORyCYZuDRGp16Ab+dCVmKlAuYEP1ZlUBKcJbrZizHqj21ZlRLdSCLxmkQfKaEskZBAn84bvHr0eS5zehuNzJxsD8hsbcwduRzCTQm2n3gpyF4EcDjTpDw0ACKFgSEomqrQrmgIkdwHgrfS9cpSS9\\\\\\\/545AKgHEPcuY403ZSS+2ZNkLt+5wEU6+1K2605W\\\\\\\/W9shzv7cnpAgsuboLVCwEnuv6leCmDzzDUObT0ZxFuGBa5z\\\\\\\/vPpF3zTBmLy0l0Os0rlr8Uvq+YjBjJfC4Pr6U9QGMg7jxIN4uTjW9aa1H4cNULXsqcIsjRixTx5zfd\\\\\\\/NsWgux2NcYRhU1LeK8o+nog3XkGau\\\\\\\/L\\\\\\\/cYOxxmKSESdFY8iIfDxz8q4tBgA+LGlpj9pFJySU1YexM3U3EKJyZgJ\\\\\\\/a3HQMrNH2X+gxw07NPcWJ67\\\\\\\/7KuH+nBkFNbkYXhtJcy4jodOhUG5fSX1az6z9BdQoG5LmsK8qV1q4u8ylA6stomRo9hPSc6I40sKSJYeBSXBN0Cx0xOyQhTT5DBiiTKQy9Cd7q0elcQNzZkhh1pPJKAcAlUwwKQnoMa1D0pNW+4wmAIQQIk0\\\\\\\/ldMp20snxScolJYiUCfEkrYjDtn4iRRyoTTm3QqyvIr3amgCDtLV7KxINDSL8DEPpQUg+Q18V8KE5cPTvBwRBiCAfBa\\\\\\\/avpiWmMi6fKWlQc1Fze7zOl5TwzUYsPuJSZ\\\\\\\/ZeA\\\\\\\/Lsemr9sq84gogdjNhcnxp\\\\\\\/cgLSr0We7lusxwOAFgTtHoVNQ3cO4IH1QScETqlz+sbu5QuvrlkF6\\\\\\\/SD4BJ67\\\\\\\/lzx\\\\\\\/Fi2LT5HoG9Kb+1Jh1ZounAZBmiR1ACT7gepfRNSzbDwd+T6sGr2M0OeKcmSB0tXv\\\\\\\/myrwsUsGlbSp7y7gaoWvsTo3HPUaa0MfHOyKjCgl7Zr6DvRk4gku5IIAFOZUUpw+vx1\\\\\\\/dW4bySLEnen5zxf3Ob+G1n83MKmEXTFWZVWbfbKHJDzqq0ARSc94S918nGVLw0EUszWVA55Sb32uiIbWcG5BXulzGefetJWPf4dQzWJwep7vd30xvGr+Bw5wWuBQi2HA2z\\\\\\\/G0eEIf1MZG+LK3y8frEGhb6R\\\\\\\/XUP5OA\\\\\\\/MxiQN1cZs1dUptr4kO6zVk9gK0I+IoOAyId2hP7Ikkc8KW2EOb7m1MOzWj4ran41hoy8dv\\\\\\\/vtNRzYq3YlYPvUlIKsJkOl9UME8xn9JU+4Xjus9DuewUj98F34RpiVi9WZ\\\\\\\/koAqiNpkF0Ez\\\\\\\/L4khIaB\\\\\\\/kzIstGZdDctGcAEGKoc7ho8lLObbxgJiqoKVneoF8Rgpf3YGOSn78WHBAWGqFsX0cRtQit9403MRskhurIxVjCm0GuDMBL2qAT07okCMPvzwtJ19PkO7J97QZSXhRcLFASF1++bD9dnGnbxTKZoaY5R6UvyiyxyWblXj17ZrZueCTDGVuLo1IjNqeFfpiK+KmV\\\\\\\/GC4aQ+\\\\\\\/lLMHGs16Zgj7e8pPkTTFktozUNbrVCkBpfgB52WgV6zlaY3Twz002Wvk1Fgk0lU2ptnV3FN6OePAYmehE0CJkcqbCzT6L7QQ7ImU+CbE3RppsMLlauN09sqLxXefqG4eApuC4DV1vHZ1azYZaxhN\\\\\\\/caQo\\\\\\\/74KuIggpnmMhnKvf\\\\\\\/5UKqLG3y+c+DErdIIITn5MfUsCXZsTI5gdxK0tvL4FCh7JkEQWevTPURqHIQNOuLOPQbyQ71ix5NC2uGK9Z935dyHzc3kwTKG7NBF3utSrH3MBo8CtRgMheCLgKo2504rRqfCwu2pSq7IeRB04bch2xgndZhVB3HTpm4NvcV8w0t6EGIHNvhloTStW605WSvVmq+N4TUGGhSBLm2Poi+1+oA2j2YYQSj9aGl8XJ9QnlMRA4bhGVRb3OCoyVTF2zQicLpKw6DqTuln7zy+kzZPEFrZ7Ugi9lo5ZP4y6ITN44diTJ46PEqaZB8CtXrqM8fMhyQmZ50TK9JP5NEDXCtLQTSe3TkjeFDyq4pUcIRZYzr1lPqb6Tr1CaHhIBKZFis3uw6t7NkOjbTiV0Yac6UTK8xx6oS45DHttXHZ9wPBrRaipKgxc2pSTMmJjF6gelNClfNrK6Lya2praKOboIKUOBKIxWSA+InPK3o94ndtisjcRtFhSzYfr7Gcbi0npotZoZT8MMMGWIkQYkVDjv3\\\\\\\/tRbal5b7nWYK34pwCBNoGXPBwS8GAoPk0XdY3Miqr\\\\\\\/PY9BQgW3Ct35G+EXGzokogvucRxd77OCdOH9CsrRxB9a8RePAWzoBpu3d38\\\\\\\/azff\\\\\\\/lbndPcKC403XBN+gl0KZiWR9AFXLTZgTHVFop1\\\\\\\/hC+48N3rA2f7W\\\\\\\/pTivTP+RN3VLFg0kc1GVfybQdWYfmDXPOf5YuvXAvNCfNgMRPA+4iU9+qhiGDMM9QWd8dhk+GIli6mJUVF0GfKRwy8u31oBIQAeScRRtsVKqyehThZ7z1KRnwxM\\\\\\\/c7UELTJPScPTcX2bWL2R6k0AZWdJodMgeVLqXAIaEqP4DHQbuzGN\\\\\\\/a8qDUrXPmKQB9p5zZ4gGNklbnXoz3+KXWkidDiW2F+uY1KXG55eMqxri9HVkiYagsUIwMSoEG9zR6uL44psa9hCYLSolg8AbsGIN8Mo++fCxyRW3eI376Sdpp03Rhn2+z5k6CczeHq0sqY8Kn7qJvqKgyxqRwCvVInQ0BsbiA6szxcgrVeZhzfxzf3I9OBou+8dJ6ygm0gIn\\\\\\\/4uqfcB+Mev71AGftilUVdAfVxt8XzjFTKnDQnSS5Twuq925Q+TpPIAgWbVaupV8dB\\\\\\\/1pdLBHV3\\\\\\\/f71MpQvT91v8t8uFWtEgDQosD9rXZJRznJL1QYdLM+IZ3pCONT439QEXriGZN0WwSiW73JcE4kvGfVN\\\\\\\/9dF4S2FsYBOAguSE5+Qs4iNSei9Qr2FI0S9Ddtal+gfn2VPopedPCNQU+DWfFjtxn84ijnOOjQ7ETOvyAq4u2SPq15LZbGK5Yr+\\\\\\\/Q7cToGPeHYR1VB9E3gMCS5I2CdAG5O+R2uQY7kvFchBWELhYGZZxHRzKkfeWg6149HHuCzwbRxY6aMh7ZlaCanPWj\\\\\\\/895AcYuwgn6Tq54m0KNVlSQyWbe6R0RZvG\\\\\\\/hO3YhqBE\\\\\\\/pniZtz2cFuJ4VPzPBgvKsT8Mcx6NQ\\\\\\\/zIkJh1PbTSeOl\\\\\\\/EH1f1dx2nJv\\\\\\\/VUjrozzUkH+Yq\\\\\\\/5VOJajJdAED8uuaNQE+NVbp1wrYDZpLTbsRrpIMy5ovw9AYdjdFqm40ziGW2EP3UTcZBkGq2whJoa5V+3aGYmm+qpyfZVhtMLeGzvy\\\\\\\/JWxTAjIsSdMUZNQJJlMYHcpt3dd6YqGEhp\\\\\\\/SBt9PVdRMPonifPmbHsJIgkfb9Zr+CCAuJQLZZ5w66wUVeS9RAGwwB8SWfWuAVjp12sABpvkR1g67aik14aGrUBlj5TZ7iWuVvF+Vohbb\\\\\\\/BcS99exIzO8XvEmAXuiZPNMiUMRWoyCUlnOLycMkq\\\\\\\/vmzNQkXW97StT1gp1R7je8zlWyM3TIuKQVGbYbtsH+65BwmzjUGP1gl\\\\\\\/fhQ2Wi\\\\\\\/QAs3Oy22UMk72HoiH5mv25CbDVDneKu+\\\\\\\/borizSnnniS8f2yGBhysUc9Udv7XAgE0l2kL9AWSeQVcDvbIXah9awIobuUFjqlFXVCyP6SbjdzanEu616kVAbhGmnuLS\\\\\\\/q\\\\\\\/A2mf2O4IgastWH5AOUzhjAqV5KcnsHbEtaxMncyjMx39FyGtZNLdK9QkBAiOQZae4x\\\\\\\/A2DNFsaK79R6Hoayxn7PPyycCHOjK2xV1uNS1vijA7x1x96SOInUwSOCjDfUBwuAeArj+fGWF0UAv5R3ZGI\\\\\\\/IRdp\\\\\\\/c5G\\\\\\\/t2ZRb6OcED9m6peZr1lMhREDJoVQwaC2SGQflT2T2xZBo3CUcpms3KXbfl1QILW590B+Nx+5tysqcGpQvkWylIUL5FsEk1EieiwAXW68M2s5NvEIeMJUFDFrcUifzh8HM2BP4ZuakJgLoVVcCZ01oCMaceEZQWkXc4MIAKXRdYN+NU2d11ISBREWBzFg8uYwFPY8obLivfkdlqv3xXYCefdgirjck8mvhb9IAYhk8GPkupO3T+fAOWrraBNaPGfZfxJaMJfnNFG3+UXQtLzeIbaMJy5E3QUHLHchR3Zv4vkPkOjMEQ7DmgnEwRY6i49bDeZFlBQAjAqCyCPbqJn\\\\\\\/7RRJfXOi59\\\\\\\/Ugo2aADzH8vk4cY92xfRoUUhEFJg0K\\\\\\\/utnGfQWQQgnWrpWSxd40AtF02n1V\\\\\\\/pf3QmO\\\\\\\/9sdAug0VRf38nTHKEawVytOkThKmEG9wwkvVmH9TjH3emabDbNAhZojTK324BRrXFo0C+Qn6QXob89yeVZpfw3FJLp8PruCGm9nuxMIg7SsBq9HWE1g7DeAa81J\\\\\\\/OH09OYt4L8wa3A\\\\\\\/3zmFlxmiLn5Z1+kO8\\\\\\\/kybFLE2v6GhIuuI65EBAZy4uvjDt9vBqFMFVkkdEfa97VJMBf5I1deludacrg93AWtIrvGwkfkLXcc51gQbB2t2iEyFQlRbqcRtA\\\\\\\/sG3E4t6yKFIo8iSX83IaqEN9Q9P4Ufma9TNziDtH9xVfR1dlASlydfQc2t\\\\\\\/EDYzxS5nbzmy0O1CIyGR0qEOrfcK+YhMN36g+GDUf98xj39RCVKMwSxsR87vaFSnYIXvGvJFUYmhAkSL0ETx\\\\\\\/1LzLetE5oo+75sf2C3M4gPK+46iM6vxE2NAntnD3SvczcIOSAuOjBpj3oT1V4Nx\\\\\\\/TZwWn\\\\\\\/zGh0TfyhB7W2+dGAP\\\\\\\/KIAqJhWUHk5pZr3YtYwgn995bpmCAIb9+\\\\\\\/zJHp3K5oDR0+M2kezc4MZiih6437D2OjEzK59w9XHl1aB67\\\\\\\/R\\\\\\\/Wuz4Hy6asuhDhHO+VHSu1+oRyXR+ZxN6rowb9ncRnsTCcLHXaB6Qua25JyBuveJo8rvQpI3geahIDEYIlQCdPwMW0eriLyLkUn1JX6YXWyOvx400is9CzJKQbycfH1ZCU4n9nJbOEZHRLzAhPxNyE7ye4ZPkcpEsP8Tchqv1R+hJVsgruuuobpdfa14tL3thRpjjHtZYZ+syOJHZixyK6DmbMoZC4RfDtwYj+1CKDj7WhVzHucEbPmYhSWntxUWnMzK6Q\\\\\\\/ZYFb7acLW4iB6IW2Pt\\\\\\\/lek9CzdYbyFHrPpDWPjxtWngkEOyqGRgk9eaZ6sIxZlNqwh+lavgVer7\\\\\\\/9jXokq+EoFuKwLSUlMtpxyXXQF49\\\\\\\/58VordMVVhUc\\\\\\\/waBEQe2wnc3+QQYwl46JIc7FIeP+ruy8pGhultr1DnG\\\\\\\/rhnFtcWcx8a6uJBvGCO0wxsqcj6iUWjx8IbisZI+GY+vgAQ4qO6jtT1jw5lHIa3B+O66EqFXK9eTF\\\\\\\/bg99uMMEb+wfo4T3f3fFW3oA9trmd0dOxT1isHwEb45117s9W+eHW1uPg31Aghk\\\\\\\/qzIs8VoqSS0Pg5wCGdHfzxAtG\\\\\\\/m1N6ZV+lG+Ukz4tcrm4VmNlnuwN4nBECYUt8PL6ANIxG9SmZmsk3ba0sJdUlLnH8XfmaagO2mCgzeClHKLM3iQSKrayn6xYQVlB0a5rqGlCrGcC7qPthtkwmVqTHc4le9NTG+cLocB43dLT8N9ICLoWVUH8xTMzdDaq3xSvoCQjUUBtlbbz79+56XX8iq1rxlsboDRJgZ6ORqatEe\\\\\\\/qDYjq+DNvbX0IIckHKVHfcXL\\\\\\\/1C+YU4sW\\\\\\\/aSPnBjoVok39eh7QkjARCDgyPEP5l6tvk7rQEwIzklc0C8pzWe8paSgLJszdZNg\\\\\\\/VDihgG3VDPiVBk5bq0RFX+\\\\\\\/Iwz1VVsHC\\\\\\\/dbpEPcZi2RAwxA1B\\\\\\\/PvPrIbFw8889eJB4nEjBVDBYd0qUXenOXE+ltDmK7nVt6mMQQ3d6+khGZtOUm+xuYZtlQfylNXWQG86tEAptbawtYJVFodlno\\\\\\\/7f6iHnDDIsw42o\\\\\\\/Lnsd6Z8hScVUB6wFcpE5VKuJToPqlIUUHPI462DoxedhHRVjNIyv1l0RECEtNgOuGR+Ro1IJ55oRmLUJxhxxBPa\\\\\\\/ahR8SvTrfuQXiZhGAAA5fORJ1sOeJqL5eLs9Tzq\\\\\\\/04fC5NmnqBSurefXeS8HZmwC\\\\\\\/dZHvMNsQ5gDvsNCYifH61lOvhWSsw+utLETNzMJ761mW8ydxPYgJtAbyQ25woS0iQexaPBkVulBEmtXBg+ZYaVPqMxvliCuLo64Jq9EnSc+bWBZN7RtZ7liuVmKqD03QKLAENbm57dhW\\\\\\\/wFPg9YN+DQdLUq8LH5eFO6XJVK4lCFX17HHphHKKRHQWNrS9\\\\\\\/aYQ9NfEp+m8nOSiGsUetSJy4rp2rGECc0JQCRqDMza2nX+XMVi0t1+z0Gosf1Uhhbo06cboz9kEolAIsnacebML0co2FRQJfkMj2zAsEvtwd+BcnNuSWREHSxJHoTl\\\\\\\/jc0PqvEtAQIlOtU6u0uTL\\\\\\\/ccn7tHf7Wu9VfhElSsJxv+hnRRTzWxPc33sAGyElFUuF4WI3fpVQ5JhzCdQfNGVztSh+I1WlrEdOzk9uc\\\\\\\/WHmmmfrL2zxaLM9KePX8oqjsWH63M7othq4wM92Dk1KyJKgn2Wj6jrMLqokuTAQVzvS\\\\\\\/4z2idn\\\\\\\/fHyKkmIIaNHoYr7jMXXLld3c4eWV3AjZV9O4IfE\\\\\\\/ChdG7Y9x+nROoCg3nF1wQrdrbdlTEdq7UMXqMfcpoJoXxTk0\\\\\\\/NnYk2oyX2R\\\\\\\/FFk5pmloYgosl9zSp0Qbm23xqdHW+7PYNyqy7I+pFizA1h7KCf24hbZzMAmmF3Ow0r4h43Vjc8gp0kRjZIkGaVuQJq0CSdhu4PeuGgNm9QF5qxb6CAwDV7bOlyaZrOkeP7GiFtYeE4x6x9Aa+Vdlue9unbp2vz5omXVafT8OJplR9b67\\\\\\\/LTm0yeG69Y2fd4P+vGZWzE\\\\\\\/dud\\\\\\\/lcnqQYZGyEDzuypJsu31uZP9OwZBeuMvexwOV9hhEZiHwPtAeW3Mj\\\\\\\/GvN0Ai9voOecu10GGTnzsBpUAtW\\\\\\\/7E+LVKcw9ldNxqVrHLFP3Mx0HHuXXEZzKGqGT+fEdBdHV3fIGbZcCecNUimcTbzWyKJXVme9KlVGq1lDt75kd\\\\\\\/osh6eKO+rKRvrInGmotJu8uWXpNhKnBerSHZXfRDyUOp1SZIZ9bX5IK+iZDNyi473tGtPJg6F7INU5ukMtrK+a9i87TfHOlsS9wn9fqbXFXYBf2LXXLm\\\\\\\/XVfk6H7za8g4CoJecV96YSlV2ljD3CaX5Shmk+9Js475cJ8MB9xOiW5BrTMYrLbjlcWLLIsuhjDEEbfeWfCAX\\\\\\\/yuKi6HkGupHotdp8XxKzR1OtwILsdMQ2mklSNwj1kJ3LbI8exmV909YSVWIpGbiPfzLowN97+uvvgsFNMi\\\\\\\/0bRs8D4GXVzyVUqko77I+p\\\\\\\/+yQvdv3xCtHceAfY5zyT\\\\\\\/vg4ZaBgAw8wzTeNrJXZZGwkQEG1fvFZ1xge5LdFaeNaBkyRKZY0MYj\\\\\\\/1rpWGvq1Uwy2sUs5Ku6u4g46pWyMkkcvQCk7Cbk8wH7iGEr24CJFzZ759pAhkwByI0SM\\\\\\\/nWC8C8YA\\\\\\\/cGNQMBzdseIrP+99L3yvgUNQGcO44PM5U3SCtrokMJOlBZreWGGAxaSZGF+EtTmquLnpW9\\\\\\\/y2D6j\\\\\\\/CTtdODwB9sT8VvaQ336FmiuqXgtdaIThYMnQpPkTWPztAecEyBma\\\\\\\/ie+uxlaT740BrUI2RuQfHV45V+1dheRQ8Ym5xlP4C01W2oQwrBB1eql+WdQ5zNqMz\\\\\\\/1iv7haBJz3u1Swr4TzPYhtu5H98owJXC8P9cME6s1lyipLxDdMfJenml4yed8UfuBuYhp6Kzwh5bnRnECMqgENOVB+bKZtpWE20hVZt0F2jsv3fZMxMBDB3VTkmjUuxMmi9IIau6hThc1WaqGP8GuzcDs1PaeDZmWtWUwFq2J+i\\\\\\\/ZBRzb4r6aHa4vzg9K59B8C6BJwysOEaMgy99MXcMpDVm9L538tb3kuf8Um+R6Yd6\\\\\\\/6c8jifekH0+bdnouxEn0KocJlKQcZpL7eSomayCjZS73ocP\\\\\\\/7\\\\\\\/KXVrvE7MQSXkNqtoWPAB0QmZPkBi4kyL5\\\\\\\/YmBaPztpyk1D46h+gWPFkmQ2922YbOLvSrY+IyNCQVOA1IwROZfzKeNjj24Ia55VLB42m5T8wAc4BYa8WPVZ9qJ3x3sjkXloSDqvy1EQkTpnRNCF+BXDOqP6dLCYoIYBJ79u+4YugreaL\\\\\\\/keB3wO3AoLY7ntUTSWGTajS9LT2m5al98sHWJC6Fpv7BHp8GHcLptaE6lE6IzJCSDR0UzTEdZv8CvywMDv1vF6H0oknfa9FL74hgkbtnX3hyVtJwgIxTYIy4MFq9xndRL1VQY2fHhMHFIDAkQ0Bat4y9U+mAOTvxCH1dncXrGKF+54ATy4ymlfjZeJ2ACD65tCVbKwcjvK96J4x0hcsDkJBKAbYECNeM+Aodt6oIeAiYJmOhrV04g9wG\\\\\\\/fFcq\\\\\\\/QG503fdz+3akPnvXTim3pXzH1jrT9fljps2qaMFGb2kEQsA6uGCbZb1X4qTL+QVVFZ5M29El1Fr+v4daapnlsnkLzmSjqYm\\\\\\\/YEL7HdbyobsuNHWKF1fJQPfx4qyQzN2AsK0HpQxrZw2jfVc1uOZBO4+T4QcDELlw8XRDYXtzJhrZRM8bF7UMS4jvJtdY4QyXX74SRSwJvBJ3o\\\\\\\/VOjbTvufhoLtVdPc0KxXTvy3WYA7PqX\\\\\\\/zNDZt2JohdLhKko4XUsRgP2hekYRFD17nGX9l72K\\\\\\\/ZXoH2iabsrxRFiWTorEhkhJgcal7bqBlyy7s\\\\\\\/fMZ3Wo279CXgR8jrDs0rlDy6S3CMthHP5s4HJbF5Y9esil9Fw3FJ+bxTBItHUWfr31TU38Y7BBNsG6WXT0RK+1xctF46i1+MjzsAmrbcieBLmY8uI1FEroUMOpMbKymOWIpJWRdIQ2QV6CXqveL9adBVcJvxYEJM9jDS8I+TgNjhii8nQ0O\\\\\\\/AoV08uwZ9RtosIp1Pj77N8ZcGFZDneMkDgPnn\\\\\\\/\\\\\\\/J8pr9URZUJqPNV8O+O4a1wsIvK+7aQOodvCvv7u9i2wgEmdIrpLGd817fExh+ZqBHjMTTF9GVQMhUuegbZuRs7PmpgauXhUf1sp1cRsB6E7\\\\\\\/7PSqobtuehmPL2qHtxA6697lKR9JIbBvSef+pBeP8y96x57adZ1iVQejutl374MQR9VYmB7O3X8ULHHl4urCGQ\\\\\\\/XqDG9+fWh\\\\\\\/qrtf4gbIy4YLbECw\\\\\\\/Nx34ZFNNtaTtwT5VTWsBw+WOwJa8d9V1DhwMFnfOgfq0fAHao71UbXMUCYBoYU8Qp40+25\\\\\\\/x912\\\\\\\/k2shPPU9PQwLtHiBoNcQ8QTqS5zx8KGF4BsHO0UJmS5xFn6b0zMwHecV1wHZdIugk8Tsv8movZ3wV6KB03ttDslBH36g+jTR0wLkGVD6xPOJbRYdf8Cz5L9hbs3o7HqZ0a\\\\\\\/Yu5nzpQ5GX047s2HL6h9n4OgwT4ia15rbsZRV5XWVu1\\\\\\\/2LmAbziI\\\\\\\/GNE\\\\\\\/bfguiDGkr41OIis+6zRPm9d7ZndXePlB5fgg8TbMpwDw2d88nNFT+QfAJNtMs8+HZ4RaHejVwUHEU9zheLVNi8O1zoCWU0PklZeBdsXSQKnLdjpZ0VC3baNj+0+A96GJw2jx5ozfpA\\\\\\\/ouoTipIkFosboNW2E23AHF2BsI69KktqelcHvQgnSXlZbXtDNBdh2dJqaQ6Cv4qK2C6M+ZUp2D5GctpFW1mUmkG4NQF4pF0OPbL01dkEedVE0KDczEkf1l1GIfE+qNU8R3g0zpyWHUu7TLaTzDdfWRj5x4UdgQaPjizWw5aV\\\\\\\/3Yias\\\\\\\/VcixFdTFvMBPO6VMWnm2VmiDC6YDuttkX4V+Nu4A3qFOgYt94l1YTpp2Yabx\\\\\\\/JrE0c\\\\\\\/qrEaz1dZ\\\\\\\/CsHbD1ozQ184zyAiB+GPrEfl15J\\\\\\\/g4nNTkIPBZIuTzjdIy3UT5puaFTtuPUoshQLOAZb2dwxqQ4elYwiozbjK1u5qMWgGNvBn2uP2Q5Mr4jJ+LvpWfv2nBjK5AekJnM1cGH1WFGdOTdo6yRd1OoDu77YI1vTOcjeS7+1dUCJSpjqoUnVFuEdFLwmOfxgKSR31yJV5quAyN9HESC0j6rFN85hLT4Nu56L0B4DhBt3x6ZJV3Sw+ZsyI9QCVv6kYtOGWIRDzshU1UzlawTaZpXe4MaaLgLrwbOe8FXS2PElmE6QeXUAk54KNchMaNHAq3VlWLQAf\\\\\\\/P6CRV\\\\\\\/g36PnQcXq1eBPA6XQJCHzmbMSowYquzWDGEZUdA9l2CieAUBIVzvIq1+pFtgqTutsU+VCDxbnS5YwjhfCuIYyaOqyYm+tROMmsD1NyxU38g+fS5ks7jZ8iTUIPBhqteGWvN7vvGqiTmJ44EcDIG2\\\\\\\/73UYEt4JNAy8eGKXzGypu0HDuWkZhP61H9E0GLTw8JHJfl\\\\\\\/IcubTbQ1p5tBLWlOcTrs6UUz54vfo0ONXU7aDz5kyDj9WHnN5KDCyopAwTHjqhp9BAJucGEcZetZ+99SaJ3X2aGh1Dz5PhRzhgS\\\\\\\/fXjk9GixCABpS1hit83JzFykei5HsKr9UtIfzq5CJI41FGR9LASt5akh\\\\\\\/5z34ZBgISMfAWw53ccA2ZwSK9wuRTJ+hbHh6elii1ewYYXxpINvwtezyOovhpI9M31rgj6gpFW8XDnDVCcs9wbiza+v8ew3r2b9T\\\\\\\/h8nRwhFTl830+Mz83DQQmV0tLpOZ1BH5BFMvClIi2YK3LOR6tqTrku+6hIqcBPmbucPtI03x6oBNzlBWTswW8R6b75cXRcVvzVgHTpkCv1+WKVvwHLlnx8dY13R46YHd4JyYcuGJY8LdcVy8l\\\\\\\/MyFElgoIcXLrOOs5l1yuY4Z7xrCpKvt+UCSTxBGdThrDX8m8RyLFpHMWqgP08j68flfFl8IkxVrv4GZRE2tH9a8qay195B6x0\\\\\\\/ktCuHw2I\\\\\\\/iM46iv1hV5I8S71kq1NFozGGPYPBZqoeSOkZL0PC+sDU443hCii4xKoQwwHLihpz\\\\\\\/NoRrQByglcTTeYoK\\\\\\\/1+tEzbIw3bM3jaM3K\\\\\\\/iDUaBECLpulFvwHVBvLW++vs5VWbBjjRYi2gYdeL9HPWe6f0YMleCX+cmJHVFbrJ4haZoRKrImJmnxZkDHf9ZKK\\\\\\\/EyQjrCOyFDV1deY4WrHQ5g+xkRVSCKSb+3GSU7rQgQdhoFJHKLze7i3e8T4ImHnIZd07oWQpbArQeyRIr+rCmVSndsbEutIKchkpDyYgKwYe67vox5d\\\\\\\/boQQgnIJwu\\\\\\\/S\\\\\\\/ei\\\\\\\/1TBhIytEgpmhkJDc6FbhnnIyR17HTNn5p98H0EdPvxuwcH9tg27qhxQuCk6ex5uzdhEJdMKMVYPieXqZrM7jFMzFl+SFZ7OgsHc0J4MLFf5DrwLMiD5QAkGsAGL\\\\\\\/1KA1PtWlNQ2uiDDOvN8zxMNyG7QLgx3+xBK3zFL8Xg44I1cQWFRz4\\\\\\\/BWASUiM7SM0jYHRNDzfvs4Jkt\\\\\\\/bvgEMNLLxzbAqJyWPom+zM8RDTUTNGIY6k8xVOT+l4fVZyjv9Wqlis+TSOzIv4UdP8h7UDyRMplkQIEirdE6FYbVJpMcVgzgnv5cQvTjtyeTUp+knnlnLD5tvlHwyn4EUiAc0mNvsNe9l758N295DSM46iCrk3Yw7DeFIqRopGk7ehgyo7iVx8HaHpGXR7DplHDbKfWiXhUSCiOqmc+SuxNF4MaODt0MQoshUtp+a7vN1bFDziZiD4KD3Fgxiu2qubCKazqZzWsPuOBoNhYNqjRbsWlHp5HW3MqaMCwKZ9SKtvKcwX9kLieyOAv+OZr5ANl1FwZA1g+7ZjrbDuarPmwF+lDCMppkUgRdTQ2xgMA9M5N3aZ42jANBKK9B6yc\\\\\\\/NSM4H8wTO5WMoa3gAIcyv8cXaxpL1h8mGxWNfjoF+eDQIwvSGax+S26MGNCUg15jqIjICy1v+SN7tevSZxPs7WxXc\\\\\\\/OOFPVVqwqW6z8OFpwnH8tcJHjIL3JAblBaEMrpl7fAPf1txHb2+QHyOF8cB6kk14ZNwEEk4vNmSVCKcfc00pWkvCy+XQwSzsoaUYDi+M2n8kijAzY+snQS7U4fLM18kueF+IoFIbT8BAZina5FGLpfzm+FpZuubiXKxsRraFbVmyHBXzEooAA3fcphyxcjKAFHBrrl3u8BdQPc3MArOypEb\\\\\\\/4iplgNG5jHiiFPRQaOALykNzlXCxOn17gn2YVWf3Cm2K92GUax35ChRHbq2wVY8khaoEevTDY8ukBqPYnz0PTddS3FfhO236V6Y+FM+2hAnEkwtR7mPPa90aU91Gvlyjdt0uWBujjP95SC49hCN0AeCmKvDJCFwajqSU33PFOWkt+WAgAVjKCaJQXxz+ZTrFcvWv8l+Gihe6U46Vk\\\\\\\/IxlP92MHv+PAPDdCPvBxQ0Tn+VdAYeTvNKSIDMLWMtBxtIM1AdaXlySd9VGUai1ADf0UBsQxF0fupVWFDphygxIXjHbHSLH2\\\\\\\/Sdt5RYiPsIxFkqJPd5i\\\\\\\/NXuu9Zl7jMhaTP1R51SMmYxxDoQKvgH\\\\\\\/CUGP91A8hBA0cb79aKFKj7OzU7jlvi0Lbq13SpRV8w5N7tZ+F+h2WZ6QSVhnXysTsWOVur7QjfHFX1RdUPzNmWGweQqQgUdl\\\\\\\/MhywtNQ2ek8MNUQxg7LOS6+c1Cw4y\\\\\\\/ajg6xC75p+Kq7ZjmaqcNa8Cqp6zL0shHHwIeF9tFGhWP\\\\\\\/4PDKHS0h6Hqmvkc980mGyAcT7CQ93BtNPf7\\\\\\\/7KQUzg70uUQ1KqG2bj\\\\\\\/F2YFXHZG0KwksyfKrXlEya7HPhG0ked5Iwq4N74ktJZyMijbYDtRI8uZZNVyveOtjQ0Amg+wJvh9K5vNG8xTTSD7HUP+HP5FF5nBp03Mc1520T1uUiMLItjPVXA3N56NmmfDvMBo64qB4OK2GWrhCriv\\\\\\\/rBy\\\\\\\/WHL\\\\\\\/JjO6LmY+3SMU3LP1VtDvMpF+5m76OoYbS6UKSMGXtAmyEg88uck4Jx7BPDVmopfVD6TH1HP72wyrGcLY\\\\\\\/Bq\\\\\\\/aHa2c0UTO0ZH6FYmqRhv+YWswji5XXFuQDjOjfttQriMdhtydB8HJhAk\\\\\\\/ucm9OU70tbaQtO\\\\\\\/LD4ebkwiozHbzdwuXPtamfc4Q89LznWtaVWj3JeCZCMRTolEwD+b4kCbAt0ZSX2xLBPDH8k2oN0+rh+eVeHf43Qw8JR9U+j4zNbU3QN+P0+wbyOI3SZp9MAdP8Fv3AXX6f\\\\\\\/7u99ujVVL\\\\\\\/zxY08qjNtoKc5O6GSEqiwQ0TDoqqqKSttVAibo959kDQ+wzPAoY6iE\\\\\\\/IrybmC8sKZTZOpwWLaGaAv\\\\\\\/\\\\\\\/xW+o0woXCNy3FV10f2xZX6DY7WUtYvTgwAwsb56Bn14FSSKWJkM7VrGyQc0vapCs7e2OZRG+tAZHkuMlt9JKPxcuVIshID5XxeV8wtK6AbJcH9Y2yQ3EaJFLHrSF5RMXfwuNcTbdd5H77HmW3oe2KuU2IBxjP653Bi+mhCn5KsM8dtdDBeUDM6Baa\\\\\\\/nWlDboJp4PGbSA+mSlhmpA4faHr5z0M0jHTtAIoiu+8zBAOU5ibszH4hFRfW6FnBDoVDfOWACUx5fe94Yf6EVINPXiEp8Q7Qh9kNQpGB0UorNG7TNm1bZIGgZWqYBp\\\\\\\/f6GxTHZAYj6iWOFNSfkpWnjqoVusE+Ydlr9tAAM5wr+fgFUQTQzx9lrSmdFh1fwR8gwLrdent\\\\\\\/qNB3fXPK5vv72rhluJH7Q1da55A+xXUQzgQJ\\\\\\\/M88ux+Sd8DXQirwz9n4FtHrlY5SrG5+Ik7qRdaVjBh\\\\\\\/2zIu5BnFlTk7XZA34GKYEbPVqPVNH9ooQ9YL8F9jzp7iqRSFSc2JCZw2ZonuchFs6VDS7mt7A0jMqxz3whdedHyH90\\\\\\\/3\\\\\\\/PMkksl+03PD4U8XZHNWazlZHEMjG6ZFXwa0\\\\\\\/JYWcIBLh3ySVASd1M8LUSps6MnFSI5Qq09M3M6z87P9pL1rN8Otbpi4rpxnIVzci4KcMiJoDPEsvcux9+Ehn78jdhQLGN1zsiMadbHt6UeaQPT0WVLEF+0CD6t\\\\\\\/X8XFO\\\\\\\/DjpDbWtIW+wQrrYUhHKKbaqZfCTqC8hK3lcnAsJbywd\\\\\\\/4HITW0aNCUk3GiPC8DO9JLXU1V7IFRwILGBPNnXQbXrxcfndgiVTVTe+waxKzokJt1FtUXgBu+Al0jEHd6KOeUYrUf\\\\\\\/x7qSsn8mNQj18gdibs0VJQlusD8kuHbjFNvKIhVfnw+x\\\\\\\/ApL9oUQFUeIIPckuKm5I6MWvkaryMoWGPW\\\\\\\/FVt3p+4LtzZIHwibdn+POp9FUY5qkC5OaGEgCoipUbOxHHhuBE+OgWgCnDThkG6U07GfyXdAvmPyOYlTNG6HvxwLCwvCTZxquZjqU0mktt8sa6hFd3hX53ddwl+a0bP6Z2I9Kh4sGuT9WChU7rkxfuRMzP9lWWyN4SOzs0jIHE424K6r2AKk6RgguvYaLiYIymzlk+Rdg+ibPX4+FzaI2WHRbDnacFTzSWTEAE3hcbs0MTSU1AnmkjJhq0xJu9MczeEvpuawONovGsFhzjqwfCKU+qV30R7fIrfKcgPSeDtkXL5bGXtGW+fyPyBRKHxoQn9\\\\\\\/hCVVo8q6+Hc2cxPSvlLD7pzkLxOShwdNBYVmklitaO\\\\\\\/jQwFakG1bUm1bsH0TGOzevlP8tcSZhpOyGjGJm2f+iSTgtHfrNbJ59PugTTCRMaJEqUD5GeAL7gee7+kJyJJVHXdQ5PPHAa6T8wkOo5n3IoWnABH44nrXvUZI\\\\\\\/R2goCr5b++OrHoCuJb3dZQWQkmcdtvLwcEDhnfDVL2QZZ\\\\\\\/Cy243tGi\\\\\\\/GQEio3bBOXsIZpjRwDuv08ZB5YIW3rVYswogtT4CF3fTmhWTFeYbEqZiDQ03GbXh9PxQf+H68FfQiO\\\\\\\/HM522HHTY3pi6IkAK5RT7wJiLlvut8+260HPxRvwSnA1ILNkpSAjG0H3prFk2APScnO44txGp1HrrkpmBfJy1i9nFNaU9gZDDCqvHeEXo1\\\\\\\/QCFovBIqyRM3FlUy3Cvg4s27yN5Nd1ieTnjfYgSSbG0QS4TmEFmMnmSa0mhU26VppmqC\\\\\\\/FTR+8OUvp6bW32rEaPuYoeXjp1LlDnC0hYv1fzu8xaRSA7OrasorQ39icb6S9BphEX9VuXKOjM86\\\\\\\/baBDY1VGkxRiBgLjTBZTDtd1GGc57VU+D4dDw+yCQ+PQKPNqZmmS39uISBNezIDq6ukC7A04aiJK6Wm9G5Cndcsnq9Czc\\\\\\\/Cp6mSn15oBSnC0+hxeGrBonIxYrwY3B35FMJApNdmhT\\\\\\\/qVQ1vlcK5ElRoHemM4uk4v6oBazw4q+8XJsvFtuKsG0KamfsAmpItbm\\\\\\\/LQIxthq5k4aNANF4o8cxejAOHi7Oc33U0OsKXkiFXJUgMo\\\\\\\/Pu7HJfsdQ5W5GjhfNvPhPGfr+m57OPuxXCM6P\\\\\\\/PEwCbHYaYdewRnglY6r8JAUJ5PvA204tL9d3Lo\\\\\\\/CaL\\\\\\\/3nkSf8ISMv2pleNR6V6ivLtfhRnxAgtekfoErFLUMcH0ZNK4LPTdmJXkzrSpmGwKECsi+5dzaE+GzeQVJbe6iNGQnTbOsq6Ly3m1Ge\\\\\\\/baJHXTTQ5lUDlWbxRFlez8\\\\\\\/lTdaR1nyl1tTBDYFdw1PW2+CQglC5uxEzvFlUjVNYyR4MVQDTjgXtpsLMUPOiM\\\\\\\/ZvPL4voseU\\\\\\\/ZO5uAamfLb0ol1+tHdukzxeBBau9OYdWCfbTzTtKlFsjFTcYCvpI+MLVJBYxDHyU5Xcaw5Yo4MT0F3dxN+I8lQvCOXQWjZJbvDbSUgZIpi0fAJJEy+1kZLiKWUSy6qCv85T3KM3r2hSVkWM+IfjS\\\\\\\/v3ssgWbc5n44FdxeJn3X9X9zrNfA4xJSJ5lSACOQ6KMsqmAFoD4+3MRcaMGWssq7qDXaaladAL1QznoWP2U1AbDPKyTUvL5Et80J1yhfUSMvSwaq3pOBnjhChbqXNIiB6pdQ9Qy88QQ8jQHAD3CRWuroGeXZxMtn95aZ0EU0GwCyEvS6F6LvyhtpjYTgNp0XPOlw\\\\\\\/laekHw7QTG2o3zILPYtQ8gFXgreZC8G81Wp42EKSCwXwe6wxWGOZhXTQ6eE+9XHEvmASOyt2DBVuKDEtMqOPhXzrPeBwLtSexUww9TGLh+zAe\\\\\\\/Ej3lq+qfZA+xdk+eq5QxEzcXN92Nc47iRncbkQFBmAaNmGnxlokv0+RF0YeTSo4yyoJsW8UPVfqjT3KtYUbn9l+\\\\\\\/y0UZn5lIYsxBq+6v\\\\\\\/osuynoFIYkC8+sLwk40kbUa2\\\\\\\/N+8iaXkprN\\\\\\\/QBkiuQCiVVrX0JdHSGj4hkrJ\\\\\\\/7DmrlgRDbLE7vAjTFcjQnx1gjRxIlpuee2L6fV5C\\\\\\\/gR+lex5rS8vVXbD9U66pSmCMYAVn84CioCUO1vhhxl\\\\\\\/NXyA08DiX6L1lJcrljQqfSlb2Wa8Wx6vaRI7Wa0Y4REWY6aMaIjrlSFYEYxhIegF2kNKYZIBHqOm5gvwkUKbv9AbdTQ79u8IjQ6xIIgorfY70Ll5iI2jtqfms5QCFFmTXkOk9lbIztP2PD\\\\\\\/5T3rYj6JXw0Tk4e75L9SkWzu5D89HoWmWQfKQ\\\\\\\/baWmzn85n8Z5uooBzX0XgjdIS1\\\\\\\/6DOpvEexkKjOOKeN+WjAcoTPg8o\\\\\\\/CM45zq0aIkKkX1cbo10ONM8lVvG80tEEiytYBw\\\\\\\/MieEmyARzQXcSHAwVGdy+e7a2hGdZpMrd9ehZTcLb+K4409Vkn83jiPdRo8Lij\\\\\\\/R9OYAqGK7c60kq11lTAzy6rhtTnIJmT0Ve9T1nG9wdedlnML43cDn11o5YQSyR+9fHMPoH8hgxKkdFwmYNJWYgJqf7qVODOQuts4gYU1DMqHxEpwV++N7tbtX2\\\\\\\/5xxYYU5Su\\\\\\\/JLFTcVSldUgqhed0CxQZESE6pp5lQ2gdfsNylasAJZ\\\\\\\/kRWpxQlN8cbDprc75TISPqVAMDnSn5P5E4Ewaks8OqNGh3EPKNaq2Q59Kp0DfY3vvc38SYxd6uwvYdYfPvkZCASvnE0PZQ5+CYtQcmvR34H2YvmHHM4IGqbCY8IMloEs0LhhKCBLE3xpUmvUD\\\\\\\/f2Ne35Ug0\\\\\\\/gzHEkasScyIlX9KGRXil\\\\\\\/460obTLEoIKFPpsyvinnUwDaQrLdg2Voq2K2\\\\\\\/M6keWST\\\\\\\/2wasThG\\\\\\\/yZuo2V6qqdM6rMHLppWiqvZditnLgf14e6KHJDBbNvsZIgzP4qpTCiCn1eC8GNZQsJpvuGVBLI1yguHbDzHAvdHy+zDdFb8fTTXnAYUEkkQYfrJaj56tbWTsHuNUdR3P+fYItYkgCKaHYeueZkfH\\\\\\\/e1bBRqfwKeQPbvKQKL+Z4Ag0MGxqYpcBiP2PYqrnNCoAIe\\\\\\\/IT6DrRBU821wh3V6CUn7M2BGaZdV79ATC0ZC3aSWSIS\\\\\\\/6SsToA\\\\\\\/JgXbLsckOl2G1ZFnlfaiUIlQefvYT7QgBrY9rBXxoNK5kvTmiRoi7mBNDz8hW5A5R8vRlmt7eUTrmD9u2cuWXJbsMtrZwfEfM4BmLwoo87Lm\\\\\\\/bwTa5JxCupTpC5zrYUzndAGbZCIPHQ46Y5YKTRhO0ubn1LDeCbjrCQM9F5m4qFk45DCZFoP5H8CxZDPnTf7zuPofkbo1iRFPTPeQxoOeyuw3+M6VObaqcH3XJDgm7zO4rz+pJjPzVtzOcJ2uv9j6IgOz3IgALCuTTD3VConhau\\\\\\\/4OqaXY9238JXxnJr6WTfCDljEeP+YzXQLBfZEz\\\\\\\/K0b1rVdJTKqRDs4NyVlYtEkAFCDDrZDSJ48gcw70BYf7ENMLyGhSCk\\\\\\\/qmFcxWNpF6Ub6ZUgqcDrYM+UZRmdP+Jd9A5+thw3SCtvVsFLVRXBy6yHnEiZWOzDFLM7X0K2oGSlDTYJU9WjL95oStLtz2AzPSjY7O6f3Nt3vkSwCnPQCFKzy6v5b6I8y0EPIUIYBjcgpSIswp9QQFxXWXq7AwgDT8by1\\\\\\\/uq8LtKup78STSY6fSVBuijvIVFWitCHeGbKTP7NGjM\\\\\\\/6tsNPfGbtTzlqSI7iY2rtFEhbIakA+1aJ8x5tMcns4MHoMosGJejB3X70a7rJuzs\\\\\\\/pvGrXVauxTLP\\\\\\\/jO3QxFbOHkY0KeFz5PpxJqh6\\\\\\\/LX\\\\\\\/1o+LSF8IpQjVppyT7M4BnyyO1di2RTHUQ55E0AyLnaPlpp8RtHTqNeEKBXkts6BIEzzvwsSrCxhoKP7KJSyheFWBcfhIBadOtxSJvwSIaw8DelIky1A0iwIj4W8ZMs6esimrx7gy\\\\\\\/VNVRTy81ZdPpSM0HIu0rj7qGxQ43i8mFs3W0ZY9sv6A599ovi+jGKewhRxLMpuYLnIuANPivXa057fYBhPwbVtQEUaMbGZlK6LVHzrWBNBYevvekaJsqyvSUYraGWS8qmms+8ST0eY2qkk5HHgsaf+XX03NUJ+igpytkTPFiFQgq+zMsuKy96d4Ix1oHLzcEWolj0CxKBoiWn4Msg8p7LxIDg3v8dk8IJ1foaYrbS5utggYLSovh290jf8iHnEaZObp1myFVAiB2sR6Y4XP8NTlUOEBHzMbHyj1DEh8PnHdy21jP7kwxjuZiu266i0l0DHkzx5QSoIR4U6qov95ft3qIKsxVsOgq\\\\\\\/e+KYuTpIaVHzgKCIJl7hlymLAp35zQxuFPDoIM0TRGIEeSu4DUhCovVzUpZTUbkvCElswO1xXRBsB+lML\\\\\\\/MqQHRPhfuSoqOVrkjUcdtDFgrnFMbpCiCNFOIdizWanyz6BIk+UuS6ixKDgApDn5kjFbPliBBiMQy4ZKIb+OB98Zh41HITDcfksCBphzOCIruIxuXJy7KvufCtiWgk1qQTwH\\\\\\\/LQkRWz1nfBeFjLFW+rRPhuVE9OZ6OQlHoTSlaFzVy4JUoQesHmP52THfLcQpmy07eZjsSqtf+jwwavYRibvlKnOd9XJeSYcRA7p4ROILG8HZKqjRya8VE7xRvTb8pVqsxXo6dDjrRXMsx6MpHw4OJGmgNuVcMKG662Y7wwm\\\\\\\/cGxHMJRGM7FUo8nsaN9qWwGBEN7hX69fQIHIPfjO0RJ1zRHJvMY4P53ojTRjt4bCvZBGWQyPNkBsFgyhZS6S8RJjltAAMfybEAPKme+WvkdF3zQQyUjlYp2NabtOhG\\\\\\\/mqy49jHB7\\\\\\\/pnWOldP1Xi\\\\\\\/siebzdSE8J2BbLUtOEE0BbcxgD4tDyn5+KHo+zoklJ0vzmxDbNd3f338OSl2uH7y8XwsMgO0nsOQL\\\\\\\/9IuglSrl2LLcL6xSalSmy\\\\\\\/IX3yjC8qkAYdCjCKK6bhNRjfMbWpJbikXKEFgJx3KTEAdCNFKWNhSTsNCS+y7LRlyMXvDGv3WmjLCy4kNyv9Sg3VNyXpWrSKKrFoN6637YwVu+mC21AvrDysC5jlrfcFIwEN3NdwiALE58JNp4absCa8ogUmaw4zPH5n0JCMGhcshR+OQXD5pp51t2Pitmr5YoTqdgstcdGQGe2EgsI2JEEYqDCr94pxpOuhN88Lg++k8wCXugLA+LOfuIeYKgP2cHjfac2Q9XjwR8a1MWxmF+oLo4Tfgc\\\\\\\/NPVMbi76BXQtAtue1IfzaldsJtzl\\\\\\\/1eK9ks79bmW8UhGf\\\\\\\/MJjeleVtIlb6+4RCAMw8fylzflJZ+6ktNTeszwiW262AejwGUriSBy0crsmQb5B30bFBOot2qQwQYi2raYeMbOf7AEcwKcxSdyVLuZ5OkF7pW5l58zVah6rFIR4VKLiKeVqiLPj4p1j8h8\\\\\\\/fn7rsM4XuzW4OGgdvTah8lAzHiaVxaMGZVX2j4ynVvTkWTraxIYUsC2TY9PGzu9GlqsBBByaEkTZn3da+moW4a1Be\\\\\\\/RNmJVLyiuPwKn0YRrLBI3DLsce8uj30ncXG9mVTXI\\\\\\\/HhHKxcGfRMP2JIQeCuNa8uJhKevCh5sgfru07zlHxqUg9t1wMYiSoh+7\\\\\\\/U669KHJp3wiqjdwXHW0MT7MU66jCrR6D+f3IMhH2TWdZ2oPD1d7cb9W12jFRmCV0b\\\\\\\/eJ8mQ9\\\\\\\/N5+uHjiqVNutPuxhuZwnMMQjCXPoKQM6pqgPAoaTonB+umzlRrNteWraD0XvxJ8zeca8U04nW7Wux\\\\\\\/UohboHubnai0mhswZnQbyJFcJnYPs5mrSrF84AEKcmDRto3lzUPfDGSRszZ6Yypf1XWBN3URXEXwXy2TgdplqvsDQuEI8CihtnP7sF9RxS5lw0bChT5Pb71\\\\\\\/OwtSDo340UhA2ymzaFKhTTHKhhSmqwusMJrI6wpjV6xrvXz\\\\\\\/Qwe9TeqWlLYThXTLlG24pM1ceYgs5u7MkEpZQ86aZRxfIGKxQrlDCBwM5uZjsjNxCw09\\\\\\\/ELOxqP\\\\\\\/1aSvYpXRgQCu\\\\\\\/\\\\\\\/1F46moeu9VjRcp1euKTSAyR+TAy6MRSa6wlaaYjr7SasGYFGijTnYQSzQqUKFevjN\\\\\\\/zro2BZhGqRztBsybloFYWpWWWjnw5pWDnfqkarAbA5Uk857oRFiXcluuq4pYGgEe8al2aA4rAHy0i+TatIxuFyS6RWz1Zv6PV9GC8njn2TgLEEsGLzRlbnioXOB60H8f4gq2WxWV3UMgfqiFFZMw0Bo\\\\\\\/znLWC7jBafUUYpk4+z77trNqHsGPvcLBtfpzNg6rGTmMNstN\\\\\\\/OGEIZJtwG2no+pYP6r06tnSoCCFdadvaIyA4o5bExmHGTfXNjKnV7+Idc6xy89nSl7328XwC\\\\\\\/jQiKcxEa7OVZ7luafyDAH91Gr4tCFXpvgmdhsPqzRqTVDMmRDmL1GIUKNKY0hPcHxlHQyvXf+yEyEjC+AvjGKc7wJoHRpOanvsiQoe1BYtGo8WuF7Q0BcwD6otS9YNA4eB27J+Z4lW9pcZTwI5TWylrlWYRH1Yg71Z3b6yDXbVuvvw02jlgJfQFsTprH9gh7ZmDGAoaR+uQSM\\\\\\\/tynD8erLN4NCCujXWTjqBCZzSPKtzcPlg\\\\\\\/FiEiXyyvTrwKIFf3XI\\\\\\\/uvwuQcW6LmMz1c+izv9fHw4kBkqxBX9XcAMze0cHu\\\\\\\/mgBMQkgTxWBs9fjmP\\\\\\\/ot1GKn7yL2hK6D0WN3MxiG4EAAIAI3o1\\\\\\\/2RU8AM9iyETL40HNwy+mTaurmP72RgnXvFzUY0iJn0AcOFddkIJu33PO8ThDJDmQc3amd3yQoQa1RO1TfP760X8XJeR6KQDjXCzZ5E45OX\\\\\\\/zjYl0KleZMXxT1QV9S5rz6Q8BLg7dgd36bSHFTKkB1dKeqpa1gOsv7FAmCbIabjHMrVVpwh2HvqnLf9DP1JGv67m1x5b4Ew1VwV49Ba3k7nE0G+LwKQGOSlWZ7Wn\\\\\\\/pJGEA4ewRSwnLRXQqgCGdozH7GeQ2ClJFZaP+jYDobaIZyCSu08LUSU\\\\\\\/duq1cV8TnbyLy8ug9khIHTPOm1QR\\\\\\\/oYW0kgoYuslaFIDfFCupjbF3Z84SmP\\\\\\\/8TYCrmFdMr5f8IlgRH8U84ZJx3+CVHDuHusIuka4rwvrqSs4Aq4aa5Z7mEdYZGCOu1wRNsOLPJIkb1dKPaAgs85YPFPN6EIYW2SBzz+aySsEWJS++yQIY4Z4KoB\\\\\\\/0FI7pAFhQttuVU1aUzcUpDs8gHylZ6s1ZGT+dAZQg0Yos50yDbd\\\\\\\/WMBT6TSCGjQBmt+c3WFeaQx3CFIN5yd\\\\\\\/73ZXIoSfDdC9hqUWvqNknx9O\\\\\\\/Iq1ouGDGAln7\\\\\\\/C95sXq3HivxX61Fg9cbMr1DWHE1f8Id4Gm2+DY4oO\\\\\\\/q28SUN\\\\\\\/COhacjcZHtXqY8u1B+MmoNtW\\\\\\\/1\\\\\\\/PDOfzXBA3aWhF1WDkN\\\\\\\/tWPifbH2En1whWs69a1fz3KEfUzgNWChyG4h\\\\\\\/WBPDHXR4bVzb7IEKLqNRHGprfNBRuVkLZYyHxr4EH6oQE8cUfxt\\\\\\\/28PC0lQ7d3KQrTMLJYiL4K+bMNuGoj0c+g0tx9PS\\\\\\\/mP+dfEIavPZSKyNKExA1bK2XwuT35u2eP3JFmia\\\\\\\/NZiC+\\\\\\\/aCaXDW2mkSV2lZnfX33mbx\\\\\\\/ce5l5RhfKTHIwVC+zsUha+U1SYEEEpOaQf8F6KTv5BG+qGkrvwNf3Eje73dIHDJO7Qg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022U2PwR0YqlRFT9PlYOWCzSA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002231312caf6c300a894aa9f2bc50285155b907e87907d3ac118fbdf3b5c1b8430d\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022au\\\\\\\/K9Bqdb\\\\\\\/7MBKFHqIF11SotdXfzi8rBk1PcR+0VqIZ3yYyK3YHPLJkPEzjSXLVtVu+fCR\\\\\\\/IN\\\\\\\/7gfm0hVLhStdNRtGKutl+QMZutyi\\\\\\\/Xi3LYkR\\\\\\\/ItFLe0C\\\\\\\/hOQDPfZR1TsyVsBEkz+XvTppPy1\\\\\\\/P1kVyI6m\\\\\\\/sd02Mn\\\\\\\/bdhihz\\\\\\\/6vTjWs+7ChVw\\\\\\\/izinCRESYXOJA1lKRHIvbn8mBFoJheYQQVCUn7sA47yJ1buftaNkPGgcQu7K4bjUoE8dUoQa+TdVuOppiy335nl6\\\\\\\/RtE1cAH3xLcniXcFgC32dtJNH7YxmE1vvw0TBb4RJPpnNeNAmtaQAij6ttNxP4UybFYtXbWqWZSkFM7DKQyWortIPhlKpePA5H9XW5XF3EGGVnMrpBmaF9Re1Ab3s+DiR\\\\\\\/Xs88S0PdWQeLkDYQv38bjG+Kr\\\\\\\/WDRTbI\\\\\\\/Hc3vGermFDqVF+8EM59kL3znGPwS8Epx3eH96I+W8RxOtfWp3mEaDJtdf2dhpNHNNb7s3DfipRxJJYCFMR7GQwXowF2orXw9hNS271pjm\\\\\\\/i6Gk3TLjxb\\\\\\\/18gWEfAWfWA18Wi78ejhBrznmO9erpA6PXitdvpk6gh2UWp1oaulWKpvxQ+oyqfOQCsGYhPNF7zdQ6q14Qo7GEYonNr1DdUPlHp0ma6CWvlVpxh9ENhKFi0uigSUTSx7iaZtJA0KiUXzi\\\\\\\/MuL7ntP93g6RMEduF+Yuk8P4I20M7zX8e0Z1bX9YGSMUrBci9ehkHuFKZm9esNNZpWHkEikCon9TdjJ5hiUqgXk3IyeYRGAPY\\\\\\\/Te0oU9S1B5cGTs6r0k3RPH7jkk0dHZWZeuS0XDB1+OClVHb7CieazlHRw8Hip8AbJDH86snuCXk4pnb2eXjn7ac3R\\\\\\\/eTdbFa3Kce72HTayup1cbNddVhBRwl\\\\\\\/blmMtxzchUBYNeC24D20Plo5saXn46Vsdvr9OEMRukZ4XrPplvAlw\\\\\\\/60wK6fLIChugWQ+CD\\\\\\\/y+SWNw98hAnnmCjN+TMb5oIalQ\\\\\\\/X6JoF1HXcmE7juc6SaMePRg914BAztnAvdNOp4ENASi+Od2TRuJ1JS1Pzq7e866OwoSG1QeX08+\\\\\\\/mHLkH3GXhEVd8B6u4utLUBNZm5WtJ3O58LAVe8SexPiaaTYqaBMaBfVCJHduAsgxBC7YI\\\\\\\/GCIJtxwyrbH1i76ZwPUlC7DBeXdwfcS0zfa6vTHjYMmGDHfRE2RRqABUq+qdqvwSJYtqZathAbBZndJB5Xs0SY5Imt1yOPJoXwOMT5KEOIsfyrMps5d4MibFuuFcKoMDeSkNG76NAJbSsz2+NQJ5KGZNaAq9AmA\\\\\\\/zqP4aI0z\\\\\\\/m\\\\\\\/U2YLpW3qPrdWhSOHbWx8Y+w\\\\\\\/DZcARKc+W8stbrn5\\\\\\\/BhHDyNsbsUEMeDC2hOkf2gvit9HV8z0BMflA9pZ7IFswRnYrsjuZCWpDRqSbz+sxrYDXg8VrGZ9Ijxm4qb5Bf9Ivn9NJBHynRIjKhzB\\\\\\\/smWQ34QxX1a\\\\\\\/elmVkZIa6Q3IvpB\\\\\\\/DITHNXxdY5v98qp82UmbLoKgH0UqlLxVcvgsIjkezt77vDbiQIzt4DC4AvQAhYtZ43Pwb45o+2xm5oiKT2OF\\\\\\\/rPOfpsceY8zmwZDZgSqGJ1PgIGV1u5QA1f46sbX0uBv\\\\\\\/IwmbBTFRHV0yD2LnePzr8bH2TvpnrW72Pslme3uJV1MoEAv5SEnzE+5NujZaojI4ZKA55liXx2V9+KHlZnByfB20El4ScCcuCL2D+JO57iMJMpnqpCbwuczRA8x9BbfdAZ6zgDavu3ZYipZi\\\\\\\/Kpk3TE5hlYKs4EWdTFqls6\\\\\\\/0iqHHKNFZZXireSaridC7pVVqqX4Nf6ruBA1S8cU5uCH2EaPc3rAdt1cYoW\\\\\\\/HD1yI+i8eFlvzb2FUcg+7IPL9R27UsxQdpq+9BadAMEqcjQqYa1CApMVYiGDS3ml37kyfXSPlGyKdNhbTfVquyr6Axf8kJD++GtCxqz+y152sWIMH9LUPHUoESMYATbzk4HF3fq7QKp8ffmhSAA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Lk92vMN0dhYjtGDBOSIqDw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022982d74c1ff80be09067d58426c3097330d4568efbf5ce4bda6aa4d213cd26fbe\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459784 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/3/427" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-H6OZ4Od4l86JwmljLQgv3IEVdO7hP_e0k33YabQUP1o" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/3/427" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-g9EawawMJOaqGAKGjtc6loqrz3pRVgewrzY6RtjLsLM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/3/417" title="A Genome-Wide Association Study for Nutritional Indices in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/3/441" title="Evolutionary Consequences of DNA Methylation on the GC Content in Vertebrate Genomes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459784" id="node459784" data-pisa="ggg;5/3/427" data-pisa-master="ggg;g3.114.016022" data-apath="/ggg/5/3/427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Analysis and Map Construction in Genetic Populations of Clonal F<sub>1</sub> and Double Cross</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Luyan Zhang</span>, <span class="highwire-citation-author" data-delta="1">Huihui Li</span> and <span class="highwire-citation-author" data-delta="2">Jiankang Wang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">427-439;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016022 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459784" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Luyan Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The National Key Facility for Crop Gene Resources and Genetic Improvement, Institute of Crop Science, and CIMMYT China Office, Chinese Academy of Agricultural Sciences, Beijing 100081, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luyan%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuyan%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Huihui Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The National Key Facility for Crop Gene Resources and Genetic Improvement, Institute of Crop Science, and CIMMYT China Office, Chinese Academy of Agricultural Sciences, Beijing 100081, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huihui%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHuihui%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jiankang Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The National Key Facility for Crop Gene Resources and Genetic Improvement, Institute of Crop Science, and CIMMYT China Office, Chinese Academy of Agricultural Sciences, Beijing 100081, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiankang%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJiankang%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wangjiankang@caas.cn" class="" data-icon-position="" data-hide-link-title="0">wangjiankang@caas.cn</a>
<a href="mailto:jkwang@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">jkwang@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/3/427" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459784" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459784/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/3/427.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459784" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459784/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/3/427.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459784" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459784/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/3/427.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459784" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459784/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/3/427.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459784" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459784/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">In this study, we considered four categories of molecular markers based on the number of distinguishable alleles at the marker locus and the number of distinguishable genotypes in clonal F<sub>1</sub> progenies. For two marker loci, there are nine scenarios that allow the estimation of female, male, and/or combined recombination frequencies. In a double cross population derived from four inbred lines, five categories of markers are classified and another five scenarios are present for recombination frequency estimation. Theoretical frequencies of identifiable genotypes were given for each scenario, from which the maximum likelihood estimates of one or more of the three recombination frequencies could be estimated. If there was no analytic solution, then Newton-Raphson method was used to acquire a numerical solution. We then proposed to use an algorithm in Traveling Salesman Problem to determine the marker order. Finally, we proposed a procedure to build the two haploids of the female parent and the two haploids of the male parent in clonal F<sub>1</sub>. Once the four haploids were built, clonal F<sub>1</sub> hybrids could be exactly regarded as a double cross population. Efficiency of the proposed methods was demonstrated in simulated clonal F<sub>1</sub> populations and one actual maize double cross. Extensive comparisons with software JoinMap4.1, OneMap, and R/qtl show that the methodology proposed in this article can build more accurate linkage maps in less time.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/clonal-f1" class="hw-term hw-article-keyword hw-article-keyword-clonal-f1" rel="nofollow">clonal F<sub xmlns:default="http://www.w3.org/1999/xhtml">1</sub></a></li><li class="kwd"><a href="/keyword/double-cross" class="hw-term hw-article-keyword hw-article-keyword-double-cross" rel="nofollow">double cross</a></li><li class="kwd"><a href="/keyword/recombination-frequency" class="hw-term hw-article-keyword hw-article-keyword-recombination-frequency" rel="nofollow">recombination frequency</a></li><li class="kwd"><a href="/keyword/linkage-analysis" class="hw-term hw-article-keyword hw-article-keyword-linkage-analysis" rel="nofollow">linkage analysis</a></li><li class="kwd"><a href="/keyword/map-construction" class="hw-term hw-article-keyword hw-article-keyword-map-construction" rel="nofollow">map construction</a></li></ul><p id="p-3">Plant species can be divided into three groups with respect to their sexual mating and asexual reproductive systems, <em>i.e.</em>, self-pollination, cross-pollination, and vegetative (or clonal or asexual) propagation (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Allard 1999</a>). An asexually propagated population consists of clones that are genetically identical to that of their parents. Reproduction by asexual propagation is common in higher plants, including nearly all fruit and nut trees such as strawberries, grapes, and pineapples; some field crops such as potatoes, sugarcane, yams, cassavas, and sweet potatoes; and many ornamental species (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Allard 1999</a>). Individual clonal plants usually show high heterozygosity. Once the superiority of any heterozygous clone is identified, this superiority can be protected and utilized by continued vegetative reproduction for a long period of time (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Allard 1999</a>).</p><p id="p-4">Most clonal species have the problem of inbreeding depression, but hybridization between different clones, or even self-pollination of one clonal line, can produce seeds and therefore generate segregating clonal F<sub>1</sub> progenies. Many genetic linkage studies have been conducted in clonal species, such as potatoes (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Tanksley <em>et al.</em> 1992</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">van Os <em>et al.</em> 2006</a>), cassavas (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Fregene <em>et al.</em> 1997</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Kunkeaw <em>et al.</em> 2010</a>), sweet potatoes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Li <em>et al.</em> 2010</a>), sugarcanes (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Liu <em>et al.</em> 2010</a>), populus (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zhang <em>et al.</em> 2000</a>), pears (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Yamamoto <em>et al.</em> 2002</a>), apples (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Hemmat <em>et al.</em> 1994</a>), and pineapples (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Carlier <em>et al.</em> 2004</a>). Most studies focused on linkage map construction by adapting the clonal F<sub>1</sub> progenies into inbred line–derived populations, such as pseudo-backcrosses or pseudo-testcrosses. This is a tedious procedure, and many less informative markers may not be used. For example, <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Hemmat <em>et al.</em> (1994)</a> only considered three groups of markers in linkage map construction: those segregating as a result of heterozygosity in the female or male parent or in both parents. Many markers were discarded in estimation of recombination frequency before linkage map construction. Some studies on clonal species used the CP model (cross pollinators) in the software JoinMap (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Stam 1993</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">van Ooijen 2006</a>), which translates the clonal F<sub>1</sub> progenies into a pseudo-backcross or pseudo-testcross population to estimate the recombination frequency in female and male parents.</p><p id="p-5"><a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Ritter <em>et al.</em> (1990)</a> proposed a method of recombination frequency estimation between heterozygous parents based on RFLP markers, using part of the informative markers in the clonal F<sub>1</sub> progenies. <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ritter and Salamini (1996)</a> considered more allelic configurations as an improvement of the previous work. <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Maliepaard <em>et al.</em> (1997)</a> presented an overview of marker pair segregation configurations and then acquired the maximum likelihood estimators for the recombination frequency. Based on 18 cross types and the assumption that both parents had the same meiotic recombination, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wu <em>et al.</em> (2002a)</a> proposed a methodology for linkage analysis in outcrossing species. Pairwise recombination frequency and linkage phase were estimated simultaneously by the posterior probabilities of the four different assignments conditional on the observed phenotype of the markers. <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Wu <em>et al.</em> (2002b)</a> used the same algorithm in another study (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Wu <em>et al.</em> 2002a</a>), but considering the sex-specific recombination frequencies. Algorithms proposed by <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Wu <em>et al.</em> (2002a</a>, <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">b)</a> were implemented in the R software (<a href="http://www.r-project.org">www.r-project.org</a>) as a package called OneMap (Margarido <em>et al.</em> 2007). However, EM algorithm and Markov chains used in recombination frequency estimation and linkage phase determination were time-consuming. In addition, some configurations in the previous studies (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Ritter and Salamini 1996</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Maliepaard <em>et al.</em> 1997</a>; <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Wu <em>et al.</em> 2002a</a>, <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">b</a>) were identical in recombination frequency estimation. For example, Wu <em>et al.</em> (2002) gave 18 cross combinations based on the genotypes of the two parents. The first four each generates four genotypes, which can be properly identified in the progenies. They are identical when used in linkage analysis. Redundant configurations complicate the application of those methods in practical populations.</p><p id="p-6">The R/qtl package could be used for linkage analysis in phase-known double cross (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Broman <em>et al.</em> 2003</a>), but it was not suitable for clonal F<sub>1</sub> and phase-unknown double cross. It has been noted that software packages in R software were computationally slow and always failed to construct dense maps (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">van Ooijen 2011</a>). Based on five segregation types of markers, <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">van Ooijen (2011)</a> proposed a Monte Carlo multipoint maximum likelihood algorithm to simultaneously estimate recombination frequency and determine marker order. An integrated map was generated by averaging lengths over anchored segments from two separate parental maps and by interpolating or extrapolating for markers segregating in only one parent. The methodology in <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">van Ooijen (2011)</a> was implemented in JoinMap4.1. The ordering algorithm used in JoinMap4.1 was called simulated annealing, which determines the best marker order by minimizing the sum of recombination frequencies in adjacent segments.</p><p id="p-7">Genetic analysis methodology of clonal species is less investigated compared with self-pollinated and cross-pollinated species. In self-pollinated and cross-pollinated species, double crosses (or four-way crosses) can be made from four inbred lines to extend the genetic diversity in genetic studies and plant breeding. In clonal F<sub>1</sub> and double cross, the number of alleles at each locus may be up to four. For each marker pair, there are four potential linkage phases in clonal F<sub>1</sub>. Once the linkage phase is determined, one clonal F<sub>1</sub> can be viewed as a double cross population.</p><p id="p-8">The unknown linkage phase and multiple alleles complicate recombination frequency estimation in clonal F<sub>1</sub> and double cross populations. Our objectives in this study were: (1) to identify and classify informative markers based on the number of distinguishable alleles and the number of distinguishable genotypes; (2) to derive the theoretical frequencies of identifiable genotypes for each scenario of marker pairs and maximum likelihood estimates of recombination frequencies; (3) to build the female, male, and combined linkage maps; (4) to build the four haploids of the female and male parents based on the estimated recombination frequencies and the combined linkage map; and (5) to demonstrate the advantage of the proposed methods in comparison with other software.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Marker categories and coding criteria in clonal F<sub>1</sub> progenies</h3><p id="p-9">Genetic studies in clonal species are normally conducted in F<sub>1</sub> hybrids of two clonal parents, one used as female and the other used as male (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). The two parents are normally heterozygous and unrelated or less related in genetics, and therefore may have up to four identifiable alleles at each polymorphism locus. In this study, <em>A</em> and <em>B</em> were used to represent the two potential alleles in the female parent; <em>C</em> and <em>D</em> represented the two potential alleles in the male parent, as indicated at two loci in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>. Based on the actual number of identifiable alleles in the two parents and the actual number of identifiable genotypes in the F<sub>1</sub> progenies, each marker locus can be classified into four categories (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/427/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diagram of the development of a clonal F1 population derived from two clonal parents, which are highly heterozygous at a large number of loci, assuming locus 1 and locus 2 were two linked polymorphism markers. A1, B1, C1, and D1 were the four alleles at marker locus 1. A2, B2, C2, and D2 were the four alleles at marker locus 2. It should be noted that the female parent could have genotype A1B1/A2B2 or A1B2/A2B1, and the male parent could have genotype C1D1/C2D2 or C1D2/C2D1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2115005789" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Diagram of the development of a clonal F&lt;sub&gt;1&lt;/sub&gt; population derived from two clonal parents, which are highly heterozygous at a large number of loci, assuming locus 1 and locus 2 were two linked polymorphism markers. &lt;em&gt;A&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;, &lt;em&gt;B&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;, &lt;em&gt;C&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;, and &lt;em&gt;D&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; were the four alleles at marker locus 1. &lt;em&gt;A&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;, &lt;em&gt;B&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;, &lt;em&gt;C&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;, and &lt;em&gt;D&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; were the four alleles at marker locus 2. It should be noted that the female parent could have genotype &lt;em&gt;A&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;/&lt;em&gt;A&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; or &lt;em&gt;A&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;/&lt;em&gt;A&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;, and the male parent could have genotype &lt;em&gt;C&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;D&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;/&lt;em&gt;C&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;&lt;em&gt;D&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; or &lt;em&gt;C&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;D&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;/&lt;em&gt;C&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;&lt;em&gt;D&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/427/F1.medium.gif" width="440" height="130"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/3/427/F1.medium.gif" width="440" height="130"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/427/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/427/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465751" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">Diagram of the development of a clonal F<sub>1</sub> population derived from two clonal parents, which are highly heterozygous at a large number of loci, assuming locus 1 and locus 2 were two linked polymorphism markers. <em>A</em><sub>1</sub>, <em>B</em><sub>1</sub>, <em>C</em><sub>1</sub>, and <em>D</em><sub>1</sub> were the four alleles at marker locus 1. <em>A</em><sub>2</sub>, <em>B</em><sub>2</sub>, <em>C</em><sub>2</sub>, and <em>D</em><sub>2</sub> were the four alleles at marker locus 2. It should be noted that the female parent could have genotype <em>A</em><sub>1</sub><em>B</em><sub>1</sub>/<em>A</em><sub>2</sub><em>B</em><sub>2</sub> or <em>A</em><sub>1</sub><em>B</em><sub>2</sub>/<em>A</em><sub>2</sub><em>B</em><sub>1</sub>, and the male parent could have genotype <em>C</em><sub>1</sub><em>D</em><sub>1</sub>/<em>C</em><sub>2</sub><em>D</em><sub>2</sub> or <em>C</em><sub>1</sub><em>D</em><sub>2</sub>/<em>C</em><sub>2</sub><em>D</em><sub>1</sub>.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/427/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Four categories of polymorphism markers that can be used in genetic study in clonal F1 populations. In category I or ABCD, each marker shows four identifiable alleles between the two clonal parents, represented by A, B, C, and D (see the four different colors in Figure 1). In the clonal population, four genotypes can be identified, represented by AC, AD, BC, and BD. In category II or A = B, one allele can be seen in the female parent and two alleles can be seen in male parent. In the clonal population, only two genotypes can be identified, represented by XC and XD, where X can be either A or B. In category III or C = D, two alleles can be seen in the female parent and one allele can be seen in male parent. The two identifiable genotypes in the clonal population are represented by AX and BX, where X can be either C or D. In category IV or AB = CD, both clonal parents show the same two heterozygous genotype. The two alleles in parents are represented by A and B, and three genotypes in their progenies are represented by AA, AB, and BB." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2115005789" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Four categories of polymorphism markers that can be used in genetic study in clonal F&lt;sub&gt;1&lt;/sub&gt; populations. In category I or ABCD, each marker shows four identifiable alleles between the two clonal parents, represented by &lt;em&gt;A&lt;/em&gt;, &lt;em&gt;B&lt;/em&gt;, &lt;em&gt;C&lt;/em&gt;, and &lt;em&gt;D&lt;/em&gt; (see the four different colors in Figure 1). In the clonal population, four genotypes can be identified, represented by &lt;em&gt;AC&lt;/em&gt;, &lt;em&gt;AD&lt;/em&gt;, &lt;em&gt;BC&lt;/em&gt;, and &lt;em&gt;BD&lt;/em&gt;. In category II or A = B, one allele can be seen in the female parent and two alleles can be seen in male parent. In the clonal population, only two genotypes can be identified, represented by &lt;em&gt;XC&lt;/em&gt; and &lt;em&gt;XD&lt;/em&gt;, where &lt;em&gt;X&lt;/em&gt; can be either &lt;em&gt;A&lt;/em&gt; or &lt;em&gt;B&lt;/em&gt;. In category III or C = D, two alleles can be seen in the female parent and one allele can be seen in male parent. The two identifiable genotypes in the clonal population are represented by &lt;em&gt;AX&lt;/em&gt; and &lt;em&gt;BX&lt;/em&gt;, where &lt;em&gt;X&lt;/em&gt; can be either &lt;em&gt;C&lt;/em&gt; or &lt;em&gt;D&lt;/em&gt;. In category IV or AB = CD, both clonal parents show the same two heterozygous genotype. The two alleles in parents are represented by &lt;em&gt;A&lt;/em&gt; and &lt;em&gt;B&lt;/em&gt;, and three genotypes in their progenies are represented by &lt;em&gt;AA&lt;/em&gt;, &lt;em&gt;AB&lt;/em&gt;, and &lt;em&gt;BB&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/427/F2.medium.gif" width="440" height="324"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/3/427/F2.medium.gif" width="440" height="324"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/427/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/427/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465757" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-11" class="first-child">Four categories of polymorphism markers that can be used in genetic study in clonal F<sub>1</sub> populations. In category I or ABCD, each marker shows four identifiable alleles between the two clonal parents, represented by <em>A</em>, <em>B</em>, <em>C</em>, and <em>D</em> (see the four different colors in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). In the clonal population, four genotypes can be identified, represented by <em>AC</em>, <em>AD</em>, <em>BC</em>, and <em>BD</em>. In category II or A = B, one allele can be seen in the female parent and two alleles can be seen in male parent. In the clonal population, only two genotypes can be identified, represented by <em>XC</em> and <em>XD</em>, where <em>X</em> can be either <em>A</em> or <em>B</em>. In category III or C = D, two alleles can be seen in the female parent and one allele can be seen in male parent. The two identifiable genotypes in the clonal population are represented by <em>AX</em> and <em>BX</em>, where <em>X</em> can be either <em>C</em> or <em>D</em>. In category IV or AB = CD, both clonal parents show the same two heterozygous genotype. The two alleles in parents are represented by <em>A</em> and <em>B</em>, and three genotypes in their progenies are represented by <em>AA</em>, <em>AB</em>, and <em>BB</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-12">Category I (or ABCD) represents the case of fully informative markers. By fully informative, we mean the four genotypes at one locus in progenies can be clearly identified. In other words, the two alleles in any clonal progeny can be traced back to its female and male parents (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). For category I markers, two alleles can be identified in either parent. The four genotypes in progenies are coded as <em>AC</em>, <em>AD</em>, <em>BC</em>, and <em>BD</em> (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). When no distortion occurs, the four genotypes will follow the Mendelian ratio of 1:1:1:1. However, it is possible that one female allele is the same as one male allele. For example, when allele <em>A</em> is equal to allele <em>C</em> at a marker locus, there is no problem assigning the two alleles in a progeny to the two parents. This marker is still classified as category I.</p><p id="p-13">Category II (or A = B) represents the case of male polymorphism markers. By male polymorphism markers, we mean they show no polymorphism in the female parent, but they show polymorphism in the male parent. For category II markers, only two genotypes can be observed in the clonal F<sub>1</sub> progenies (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). Genotypes <em>AC</em> and <em>BC</em> cannot be separated; neither can genotypes <em>AD</em> and <em>BD</em>. In this category, <em>XC</em> is used to code genotypes <em>AC</em> and <em>BC</em>; <em>XD</em> is used to code genotypes <em>AD</em> and <em>BD</em>, where <em>X</em> stands for either allele <em>A</em> or allele <em>B</em> (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). When no distortion occurs, the two genotypes will follow the Mendelian ratio of 1:1.</p><p id="p-14">Category III (or C = D) represents the case of female polymorphism markers. By female polymorphism markers, we mean they show polymorphism in the female parent, but they show no polymorphism in the male parent. For category III markers, only two genotypes can be observed in the clonal F<sub>1</sub> progenies (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>). Genotypes <em>AC</em> and <em>AD</em> cannot be separated; neither can genotypes <em>BC</em> and <em>BD</em>. In this category, <em>AX</em> is used to code genotypes <em>AC</em> and <em>AD</em>; <em>BX</em> is used to code genotypes <em>BC</em> and <em>BD</em>, where <em>X</em> stands for either allele <em>C</em> or <em>D</em> (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>). When no distortion occurs, the two genotypes will follow the Mendelian ratio of 1:1.</p><p id="p-15">Category IV (or AB = CD) represents the case of co-dominant markers. By co-dominant markers, we mean they show the same polymorphism pattern in both female and male parents, similar to an F<sub>2</sub> population derived from two inbred parents in self-pollinated and cross-pollinated species. For category IV markers, three genotypes can be observed in the clonal F<sub>1</sub> progenies, which are coded by <em>AA</em>, <em>AB</em>, and <em>BB</em>, respectively (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>). When no distortion occurs, the three genotypes will follow the Mendelian ratio of 1:2:1.</p><p id="p-16">Missing marker types are common in most genetic populations (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Zhang <em>et al.</em> 2010</a>). In these four categories, any missing values of marker type are coded as <em>XX</em>.</p></div><div id="sec-3" class="subsection"><h3>Nine scenarios between two loci in recombination frequency estimation in clonal F<sub>1</sub> progenies</h3><p id="p-17">Assuming that locus 1 and locus 2 are two linked polymorphism markers, falling into one of the four categories in <a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a>, let <em>A</em><sub>1</sub>, <em>B</em><sub>1</sub>, <em>C</em><sub>1</sub>, and <em>D</em><sub>1</sub> be the four alleles at locus 1 and let <em>A</em><sub>2</sub>, <em>B</em><sub>2</sub>, <em>C</em><sub>2</sub>, and <em>D</em><sub>2</sub> be the four alleles at locus 2. Recombination frequencies in the female and male parents were denoted as <em>r<sub>F</sub></em> and <em>r<sub>M</sub></em>, which can be used to construct the female and male linkage maps, respectively. The combined recombination frequency is denoted as <em>r</em>, which can be used to construct the combined map. Due to the symmetry of marker pairs, we consider nine scenarios between two loci in clonal F<sub>1</sub> populations where at least one of the above three recombination frequencies can be estimated (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Scenario 1 represents the most ideal situation where all recombination frequencies can be properly estimated. If one locus is category II and the other one is category III (not included in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), then the four genotypes at the two loci have an equal theoretical frequency of 0.25. In this scenario, none of <em>r<sub>F</sub></em>, <em>r<sub>M</sub></em>, and <em>r</em> can be estimated.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465671/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465671/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">The nine scenarios between two linked loci in the clonal F<sub>1</sub> population for estimating the recombination frequency</span><div class="sb-div caption-clear"></div></div></div><p id="p-19">When one locus is category II, there is no polymorphism in the female parent; therefore, <em>r<sub>F</sub></em> cannot be estimated (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Similarly, when one locus is category III, there is no polymorphism in the male parent; therefore, <em>r<sub>M</sub></em> cannot be estimated (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). In scenario 4, only half of samples can be used to estimate <em>r<sub>F</sub></em> and <em>r<sub>M</sub></em> (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). In scenario 9, the linkage information in the two parents is confounded. It is impossible to estimate <em>r<sub>F</sub></em> and <em>r<sub>M</sub></em>. However, the combined <em>r</em> can still be estimated (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>).</p></div><div id="sec-4" class="subsection"><h3>Linkage phases between two loci to be determined in clonal F<sub>1</sub> progenies</h3><p id="p-20">In clonal F<sub>1</sub> progenies, linkage phases of the two loci in both parents are unknown before linkage analysis. When marker loci 1 and 2 show polymorphism in the female parent, <em>A</em><sub>1</sub> and <em>B</em><sub>1</sub> are randomly assigned for the two alleles at locus 1, and <em>A</em><sub>2</sub> and <em>B</em><sub>2</sub> are randomly assigned for the two alleles at locus 2. Genotype of the female parent can be either <em>A</em><sub>1</sub><em>A</em><sub>2</sub>/<em>B</em><sub>1</sub><em>B</em><sub>2</sub> or <em>A</em><sub>1</sub><em>B</em><sub>2</sub>/<em>B</em><sub>1</sub><em>A</em><sub>2</sub>, where “/” was used to separate the two homologous chromosomes. For both phases, genotype of the female parent is <em>A</em><sub>1</sub><em>B</em><sub>1</sub> at locus 1 and is <em>A</em><sub>2</sub><em>B</em><sub>2</sub> at locus 2. The same situation applies in the male parent. Genotype of the male parent can be either <em>C</em><sub>1</sub><em>C</em><sub>2</sub>/<em>D</em><sub>1</sub><em>D</em><sub>2</sub> or <em>C</em><sub>1</sub><em>D</em><sub>2</sub>/<em>D</em><sub>1</sub><em>C</em><sub>2</sub>. For both phases, genotype of the female parent is <em>C</em><sub>1</sub><em>D</em><sub>1</sub> at locus 1 and <em>C</em><sub>2</sub><em>D</em><sub>2</sub> at locus 2. Linkage phases in both parents are to be determined by linkage analysis.</p><p id="p-21">Taking the female parent as an example, the four female gametes are <em>A</em><sub>1</sub><em>A</em><sub>2</sub>, <em>A</em><sub>1</sub><em>B</em><sub>2</sub>, <em>B</em><sub>1</sub><em>A</em><sub>2</sub>, and <em>B</em><sub>1</sub><em>B</em><sub>2</sub>, and their frequencies are represented by <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-1.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-2.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-3.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-4.gif"/></noscript></span></span> (see <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/016022SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS1.pdf">Table S1</a>). In the case of genotype <em>A</em><sub>1</sub><em>A</em><sub>2</sub>/<em>B</em><sub>1</sub><em>B</em><sub>2</sub>, gametes <em>A</em><sub>1</sub><em>A</em><sub>2</sub> and <em>B</em><sub>1</sub><em>B</em><sub>2</sub> are the two noncrossover (or parental) types with a frequency of <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-5.gif"/></noscript></span></span>, and <em>A</em><sub>1</sub><em>B</em><sub>2</sub> and <em>B</em><sub>1</sub><em>A</em><sub>2</sub> are the two crossover (or recombination, or nonparental) types with a frequency of <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-6.gif"/></noscript></span></span>. The estimated <em>r<sub>F</sub></em> will be lower than 0.5 if the two loci are linked. In the case of genotype <em>A</em><sub>1</sub><em>B</em><sub>2</sub>/<em>B</em><sub>1</sub><em>A</em><sub>2</sub>, gametes <em>A</em><sub>1</sub><em>A</em><sub>2</sub> and <em>B</em><sub>1</sub><em>B</em><sub>2</sub> are the two crossover types with a frequency of <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-7.gif"/></noscript></span></span>, and <em>A</em><sub>1</sub><em>B</em><sub>2</sub> and <em>B</em><sub>1</sub><em>A</em><sub>2</sub> are the two noncrossover types with a frequency of <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-8.gif"/></noscript></span></span>. The estimated <em>r<sub>F</sub></em> will be more than 0.5 when the two loci are linked. Obviously, whether the estimated <em>r<sub>F</sub></em> is less or more than 0.5 could help to determine the linkage phase and genotype of the female parent. Similarly, whether the estimated <em>r<sub>M</sub></em> is less or more than 0.5 could help to determine the linkage phase and genotype of the male parent.</p><p id="p-22">Therefore, linkage phases and genotypes of both parents can be determined by their estimated recombination frequencies, respectively. If estimated <em>r<sub>F</sub></em> is less than 0.5, then the female parent will be in linkage phase <em>A</em><sub>1</sub><em>A</em><sub>2</sub>/<em>B</em><sub>1</sub><em>B</em><sub>2</sub>; otherwise, it will be in linkage phase <em>A</em><sub>1</sub><em>B</em><sub>2</sub>/<em>B</em><sub>1</sub><em>A</em><sub>2</sub>. If estimated <em>r<sub>M</sub></em> is less than 0.5, then the male parent will be in linkage phase <em>C</em><sub>1</sub><em>C</em><sub>2</sub>/<em>D</em><sub>1</sub><em>D</em><sub>2</sub>; otherwise, it will be in linkage phase <em>C</em><sub>1</sub><em>D</em><sub>2</sub>/<em>D</em><sub>1</sub><em>C</em><sub>2</sub>.</p><p id="p-23">Considering the two phases to be determined in both parents together, four potential linkage phases of the two parents can be defined. In phase I, the female parent has genotype <em>A</em><sub>1</sub><em>A</em><sub>2</sub>/<em>B</em><sub>1</sub><em>B</em><sub>2</sub> and the male parent has genotype <em>C</em><sub>1</sub><em>C</em><sub>2</sub>/<em>D</em><sub>1</sub><em>D</em><sub>2</sub>. In phase II, the female parent has genotype <em>A</em><sub>1</sub><em>A</em><sub>2</sub>/<em>B</em><sub>1</sub><em>B</em><sub>2</sub> and the male parent has genotype <em>C</em><sub>1</sub><em>D</em><sub>2</sub>/<em>D</em><sub>1</sub><em>C</em><sub>2</sub>. In phase III, the female parent has genotype <em>A</em><sub>1</sub><em>B</em><sub>2</sub>/<em>B</em><sub>1</sub><em>A</em><sub>2</sub> and the male parent has genotype <em>C</em><sub>1</sub><em>C</em><sub>2</sub>/<em>D</em><sub>1</sub><em>D</em><sub>2</sub>. In phase IV, the female parent has genotype <em>A</em><sub>1</sub><em>B</em><sub>2</sub>/<em>B</em><sub>1</sub><em>A</em><sub>2</sub> and the male parent has genotype <em>C</em><sub>1</sub><em>D</em><sub>2</sub>/<em>D</em><sub>1</sub><em>C</em><sub>2</sub>. The four phases will be used later for some scenarios in estimating the combined recombination frequency <em>r</em>, to make sure the estimated <em>r</em> is less than 0.5, and the estimation will not be affected by the linkage information confounding in one or both parents.</p></div><div id="sec-5" class="subsection"><h3>Recombination frequency estimation in scenario 1 in clonal F<sub>1</sub> progenies</h3><p id="p-24">We begin with the most ideal situation where locus 1 has four identifiable genotypes <em>A</em><sub>1</sub><em>C</em><sub>1</sub>, <em>A</em><sub>1</sub><em>D</em><sub>1</sub>, <em>B</em><sub>1</sub><em>C</em><sub>1</sub>, and <em>B</em><sub>1</sub><em>D</em><sub>1</sub>, and locus 2 has four identifiable genotypes <em>A</em><sub>2</sub><em>C</em><sub>2</sub>, <em>A</em><sub>2</sub><em>D</em><sub>2</sub>, <em>B</em><sub>2</sub><em>C</em><sub>2</sub>, and <em>B</em><sub>2</sub><em>D</em><sub>2</sub>. The first row and first column of <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS1.pdf">Table S1</a> show the four female and male gametes and their frequencies, from which we can easily derive theoretical frequencies of the 16 identifiable genotypes at the two linked loci. For convenience, the 16 genotypes were rearranged in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>, and sample sizes of the 16 genotypes were represented by <em>n</em><sub>1</sub>, <em>n</em><sub>2</sub>, …, and <em>n</em><sub>16</sub>. Based on theoretical frequencies and sample sizes in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, the likelihood function (<em>L</em>) and logarithm likelihood (log<em>L</em>) can be constructed in <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a>.<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-25.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>where <em>C</em> is a constant independent of the unknown recombination frequencies. The maximum likelihood estimates (MLE) of recombination frequencies can be calculated either by solving the likelihood equation (<em>i.e.</em>, <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-26.gif"/></noscript></span></span>) or by some approximate algorithms when there is no analytic solution to the likelihood equation. From <a id="xref-disp-formula-1-2" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a>, MLE of <em>r<sub>F</sub></em> and <em>r<sub>M</sub></em> can be directly calculated from <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation (2)</a>.<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-27.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where <em>n<sub>i</sub></em> is the observed sample size for the <em>i</em>th genotype (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>), <em>n<sub>i</sub></em><sub>:</sub><em><sub>j</sub></em> is the sum of <em>n<sub>i</sub></em> to <em>n<sub>j</sub></em>, and <em>n</em> is the total sample size (<em>i.e.</em>, <em>n</em>=<em>n</em><sub>1:16</sub>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465759/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465759/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Theoretical frequencies of the 16 identifiable genotypes in the clonal F<sub>1</sub> population at two linked loci</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">Define the estimate of the combined recombination frequency <em>r</em> in <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Equation (3)</a>.<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-28.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>It can be easily seen that the estimate thus defined in <a id="xref-disp-formula-3-2" class="xref-disp-formula" href="#disp-formula-3">Equation (3)</a> is always less than 0.5. In addition, it can be proved that the estimate in <a id="xref-disp-formula-3-3" class="xref-disp-formula" href="#disp-formula-3">Equation (3)</a> is also MLE of <em>r</em>, when directly calculated from its likelihood function.</p></div><div id="sec-6" class="subsection"><h3>Recombination frequency estimation in scenarios 2 and 3 in clonal F<sub>1</sub> progenies</h3><p id="p-27">In scenario 2, locus 1 has four genotypes <em>A</em><sub>1</sub><em>C</em><sub>1</sub>, <em>A</em><sub>1</sub><em>D</em><sub>1</sub>, <em>B</em><sub>1</sub><em>C</em><sub>1</sub>, and <em>B</em><sub>1</sub><em>D</em><sub>1</sub>, and locus 2 has two genotypes <em>X</em><sub>2</sub><em>C</em><sub>2</sub> and <em>X</em><sub>2</sub><em>D</em><sub>2</sub>. In scenario 3, locus 1 has four genotypes <em>A</em><sub>1</sub><em>C</em><sub>1</sub>, <em>A</em><sub>1</sub><em>D</em><sub>1</sub>, <em>B</em><sub>1</sub><em>C</em><sub>1</sub>, and <em>B</em><sub>1</sub><em>D</em><sub>1</sub>, and locus 2 has two genotypes <em>A</em><sub>2</sub><em>X</em><sub>2</sub> and <em>B</em><sub>2</sub><em>X</em><sub>2</sub>. <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> shows theoretical frequencies of the eight identifiable genotypes at the two loci. The theoretical frequencies do not contain the female recombination frequency in scenario 2, and they do not contain the male recombination frequency in scenario 3. Therefore, <em>r<sub>F</sub></em> cannot be estimated in scenario 2; <em>r<sub>M</sub></em> cannot be estimated in scenario 3. MLE of <em>r<sub>M</sub></em> in scenario 2 can be calculated from its likelihood functions, given in <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">Equation (4)</a>.<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-45.gif"/></noscript></span>, <span class="disp-formula-label">(4)</span></span>where <em>n<sub>i</sub></em> is the observed sample size for the <em>i</em>th genotype (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>), <em>n<sub>i</sub></em><sub>:</sub><em><sub>j</sub></em> is the sum of <em>n<sub>i</sub></em> to <em>n<sub>j</sub></em>, and <em>n</em> is the total sample size (<em>i.e.</em>, <em>n</em>=<em>n</em><sub>1:8</sub>). Define the estimate of <em>r</em> in <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">Equation (5)</a>.<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-46.gif"/></noscript></span>. <span class="disp-formula-label">(5)</span></span>It can be easily seen that the estimate thus defined is less than 0.5. In addition, the estimate in <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">Equation (5)</a> is MLE of <em>r</em>, when directly calculated from its likelihood function.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465750/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465750/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Theoretical frequencies of the eight identifiable genotypes in the clonal F<sub>1</sub> population</span><div class="sb-div caption-clear"></div></div></div><p id="p-29">MLE of <em>r</em><sub><em>F</em></sub> in scenario 3 can be calculated from its likelihood function, given in <a id="xref-disp-formula-6-1" class="xref-disp-formula" href="#disp-formula-6">Equation (6)</a>.<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-47.gif"/></noscript></span>, <span class="disp-formula-label">(6)</span></span>where <em>n<sub>i</sub></em> is the observed sample size of the <em>i</em>th genotype (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>), <em>n<sub>i</sub></em><sub>:</sub><em><sub>j</sub></em> is the sum of <em>n<sub>i</sub></em> to <em>n<sub>j</sub></em>, and <em>n</em> is the total sample size (<em>i.e.</em>, <em>n</em>=<em>n</em><sub>1:8</sub>). Define the estimate of <em>r</em> in <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">Equation (7)</a>.<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-48.gif"/></noscript></span> <span class="disp-formula-label">(7)</span></span>Similar to <a id="xref-disp-formula-5-3" class="xref-disp-formula" href="#disp-formula-5">Equation (5)</a>, the estimate thus defined is less than 0.5, and is MLE of <em>r</em>.</p></div><div id="sec-7" class="subsection"><h3>Recombination frequency estimation in scenario 4 in clonal F<sub>1</sub> progenies</h3><p id="p-30">In this scenario, locus 1 has four genotypes <em>A</em><sub>1</sub><em>C</em><sub>1</sub>, <em>A</em><sub>1</sub><em>D</em><sub>1</sub>, <em>B</em><sub>1</sub><em>C</em><sub>1</sub>, and <em>B</em><sub>1</sub><em>D</em><sub>1</sub>, and locus 2 has three genotypes <em>A</em><sub>2</sub><em>A</em><sub>2</sub>, <em>A</em><sub>2</sub><em>B</em><sub>2</sub>, and <em>B</em><sub>2</sub><em>B</em><sub>2</sub>. <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> shows theoretical frequencies of the 12 identifiable genotypes at the two loci. Information on <em>r<sub>F</sub></em> and <em>r<sub>M</sub></em> is confounded in half of the genotypes. MLE of <em>r<sub>F</sub></em> and <em>r<sub>M</sub></em> using the other half of the genotypes are given in <a id="xref-disp-formula-8-1" class="xref-disp-formula" href="#disp-formula-8">Equation (8)</a>.<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-109.gif"/></noscript></span> <span class="disp-formula-label">(8)</span></span>where <em>n<sub>i</sub></em> is the observed sample sizes of the <em>i</em>th genotype and <em>n<sub>i</sub></em><sub>:</sub><em><sub>j</sub></em> is the sum of <em>n<sub>i</sub></em> to <em>n<sub>j</sub></em>.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465690/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465690/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Theoretical frequencies of the 12 identifiable genotypes in the clonal F<sub>1</sub> population</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">As stated, estimated <em>r<sub>F</sub></em> and <em>r<sub>M</sub></em> in <a id="xref-disp-formula-8-2" class="xref-disp-formula" href="#disp-formula-8">Equation (8)</a> can be used in determining the linkage phases in both parents. Then, the theoretical frequencies of the 12 genotypes can be calculated based on the combined recombination frequency <em>r</em> (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>), from which the likelihood function can be constructed to estimate MLE of <em>r</em>. However, there is no analytic solution for MLE of <em>r</em>, and therefore some iterative algorithms have to be used (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Sun <em>et al.</em> 2012</a>). As an example, Newton-Raphson method for estimating MLE of <em>r</em> was given in Supplementary Materials (see <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/FileS1.pdf">File S1</a>). Because the theoretical frequencies (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>) are calculated from the identified linkage phase, the estimated <em>r</em> is less than 0.5 when the two loci are genetically linked.</p></div><div id="sec-8" class="subsection"><h3>Recombination frequency estimation in scenarios 5 and 7 in clonal F<sub>1</sub> progenies</h3><p id="p-33">In scenario 5, locus 1 has two genotypes <em>X</em><sub>1</sub><em>C</em><sub>1</sub> and <em>X</em><sub>1</sub><em>D</em><sub>1</sub>, and locus 2 has two genotypes <em>X</em><sub>2</sub><em>C</em><sub>2</sub> and <em>X</em><sub>2</sub><em>D</em><sub>2</sub>. In scenario 6, locus 1 has two genotypes <em>A</em><sub>1</sub><em>X</em><sub>1</sub> and <em>B</em><sub>1</sub><em>X</em><sub>1</sub>, and locus 2 has two genotypes <em>A</em><sub>2</sub><em>X</em><sub>2</sub> and <em>B</em><sub>2</sub><em>X</em><sub>2</sub>. <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> shows theoretical frequencies of the four identifiable genotypes at the two loci. Obviously, theoretical frequencies do not contain the female recombination frequency in scenario 5 and do not contain the male recombination frequency in scenario 7. Thus, <em>r<sub>F</sub></em> cannot be estimated in scenario 5; <em>r</em><sub><em>M</em></sub> cannot be estimated in scenario 7. MLE of <em>r<sub>M</sub></em> in scenario 5 can be calculated from its likelihood functions, given in <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">Equation (9)</a>.<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-118.gif"/></noscript></span>, <span class="disp-formula-label">(9)</span></span>where <em>n<sub>i</sub></em> is the observed sample size of the <em>i</em>th genotype (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>), <em>n<sub>i</sub></em><sub>:</sub><em><sub>j</sub></em> is the sum of <em>n<sub>i</sub></em> to <em>n<sub>j</sub></em>, and <em>n</em> is the total sample size (<em>i.e.</em>, <em>n</em>=<em>n</em><sub>1:4</sub>). Define the estimate of <em>r</em> in <a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">Equation (10)</a>.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465720/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465720/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Theoretical frequencies of the four identifiable genotypes in the clonal F<sub>1</sub> population</span><div class="sb-div caption-clear"></div></div></div><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-119.gif"/></noscript></span> <span class="disp-formula-label">(10)</span></span><p id="p-35">MLE of <em>r<sub>F</sub></em> in scenario 7 can be calculated from its likelihood functions, given in <a id="xref-disp-formula-11-1" class="xref-disp-formula" href="#disp-formula-11">Equation (11)</a>. Define the estimate of <em>r</em> in <a id="xref-disp-formula-12-1" class="xref-disp-formula" href="#disp-formula-12">Equation (12)</a>.<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-120.gif"/></noscript></span> <span class="disp-formula-label">(11)</span></span><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-121.gif"/></noscript></span> <span class="disp-formula-label">(12)</span></span>Similar to <a id="xref-disp-formula-5-4" class="xref-disp-formula" href="#disp-formula-5">Equation (5)</a> and <a id="xref-disp-formula-7-2" class="xref-disp-formula" href="#disp-formula-7">Equation (7)</a>, the estimates defined in <a id="xref-disp-formula-10-2" class="xref-disp-formula" href="#disp-formula-10">Equation (10)</a> and <a id="xref-disp-formula-12-2" class="xref-disp-formula" href="#disp-formula-12">Equation (12)</a> are less than 0.5, and are MLE of <em>r</em> for scenarios 5 and 7, respectively.</p></div><div id="sec-9" class="subsection"><h3>Recombination frequency estimation in scenarios 6 and 8 in clonal F<sub>1</sub> progenies</h3><p id="p-36">In scenario 6, locus 1 has two genotypes <em>X</em><sub>1</sub><em>C</em><sub>1</sub> and <em>X</em><sub>1</sub><em>D</em><sub>1</sub>, and locus 2 has three genotypes <em>A</em><sub>2</sub><em>A</em><sub>2</sub>, <em>A</em><sub>2</sub><em>B</em><sub>2</sub>, and <em>B</em><sub>2</sub><em>B</em><sub>2</sub>. In scenario 8, locus 1 has two genotypes <em>A</em><sub>1</sub><em>X</em><sub>1</sub> and <em>B</em><sub>1</sub><em>X</em><sub>1</sub>, and locus 2 has three genotypes <em>A</em><sub>2</sub><em>A</em><sub>2</sub>, <em>A</em><sub>2</sub><em>B</em><sub>2</sub>, and <em>B</em><sub>2</sub><em>B</em><sub>2</sub>. <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a> shows theoretical frequencies of the six identifiable genotypes at the two linked loci. The theoretical frequencies do not contain the female recombination frequency in scenario 6 and do not contain the male recombination frequency in scenario 8. Thus, <em>r<sub>F</sub></em> cannot be estimated in scenario 6, and <em>r<sub>M</sub></em> cannot be estimated in scenario 8. MLE of <em>r<sub>M</sub></em> in scenario 6 can be calculated from its likelihood function, given in <a id="xref-disp-formula-13-1" class="xref-disp-formula" href="#disp-formula-13">Equation (13)</a>.<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-134.gif"/></noscript></span> <span class="disp-formula-label">(13)</span></span>where <em>n<sub>i</sub></em> is the observed sample size of the <em>i</em>th genotype (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>) and <em>n<sub>i</sub></em><sub>:</sub><em><sub>j</sub></em> is the sum of <em>n<sub>i</sub></em> to <em>n<sub>j</sub></em>. Define the estimate of <em>r</em> in <a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">Equation (14)</a>.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465748/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465748/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Theoretical frequencies of the six identifiable genotypes in the clonal F<sub>1</sub> population</span><div class="sb-div caption-clear"></div></div></div><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-135.gif"/></noscript></span>. <span class="disp-formula-label">(14)</span></span><p id="p-38">Maximum likelihood estimates of <em>r<sub>F</sub></em> in scenario 8 can be calculated from its likelihood function, given in <a id="xref-disp-formula-15-1" class="xref-disp-formula" href="#disp-formula-15">Equation (15)</a>. Define the estimate of <em>r</em> in <a id="xref-disp-formula-16-1" class="xref-disp-formula" href="#disp-formula-16">Equation (16)</a>.<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-136.gif"/></noscript></span> <span class="disp-formula-label">(15)</span></span><span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-137.gif"/></noscript></span> <span class="disp-formula-label">(16)</span></span>Similar to <a id="xref-disp-formula-5-5" class="xref-disp-formula" href="#disp-formula-5">Equation (5)</a>, <a id="xref-disp-formula-7-3" class="xref-disp-formula" href="#disp-formula-7">Equation (7)</a>, <a id="xref-disp-formula-10-3" class="xref-disp-formula" href="#disp-formula-10">Equation (10)</a>, and <a id="xref-disp-formula-12-3" class="xref-disp-formula" href="#disp-formula-12">Equation (12)</a>, the estimates defined in <a id="xref-disp-formula-14-2" class="xref-disp-formula" href="#disp-formula-14">Equation (14)</a>, <a id="xref-disp-formula-15-2" class="xref-disp-formula" href="#disp-formula-15">Equation (15)</a>, and <a id="xref-disp-formula-16-2" class="xref-disp-formula" href="#disp-formula-16">Equation (16)</a> are less than 0.5 and are MLE of <em>r</em> for scenarios 6 and 8, respectively.</p></div><div id="sec-10" class="subsection"><h3>Recombination frequency estimation in scenario 9 in clonal F<sub>1</sub> progenies</h3><p id="p-39">In this scenario, locus 1 has three genotypes <em>A</em><sub>1</sub><em>A</em><sub>1</sub>, <em>A</em><sub>1</sub><em>B</em><sub>1</sub> and <em>B</em><sub>1</sub><em>B</em><sub>1</sub>, and locus 2 has three genotypes <em>A</em><sub>2</sub><em>A</em><sub>2</sub>, <em>A</em><sub>2</sub><em>B</em><sub>2</sub> and <em>B</em><sub>2</sub><em>B</em><sub>2</sub>. Linkage information in both parents cannot be separated; therefore, <em>r<sub>F</sub></em> and <em>r<sub>M</sub></em> cannot be estimated. Linkage phases in parents are unknown before estimating the combined recombination frequency <em>r</em>. <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a> shows theoretical frequencies of the nine identifiable genotypes at the two loci in the four potential linkage phases I to IV. For linkage phase I, female and male parents have the same genotype <em>A</em><sub>1</sub><em>A</em><sub>2</sub>/<em>B</em><sub>1</sub><em>B</em><sub>2</sub>. For linkage phase II, female and male parents have genotypes <em>A</em><sub>1</sub><em>A</em><sub>2</sub>/<em>B</em><sub>1</sub><em>B</em><sub>2</sub> and <em>A</em><sub>1</sub><em>B</em><sub>2</sub>/<em>B</em><sub>1</sub><em>A</em><sub>2</sub>, respectively. For linkage phase III, female and male parents have genotypes <em>A</em><sub>1</sub><em>B</em><sub>2</sub>/<em>B</em><sub>1</sub><em>A</em><sub>2</sub> and <em>A</em><sub>1</sub><em>A</em><sub>2</sub>/<em>B</em><sub>1</sub><em>B</em><sub>2</sub>, respectively. For linkage phase IV, female and male parents have the same genotype <em>A</em><sub>1</sub><em>B</em><sub>2</sub>/<em>B</em><sub>1</sub><em>A</em><sub>2</sub>. Phases II and III are equivalent in genetics and have the same genotypic frequencies.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465734/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465734/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">Theoretical frequencies of the nine identifiable genotypes in the clonal F<sub>1</sub> population</span><div class="sb-div caption-clear"></div></div></div><p id="p-41">For linkage phases I and IV, Newton-Raphson algorithms to estimate <em>r</em> can be found in Supplementary Materials (see <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/FileS2.pdf">File S2</a>). For linkage phases II and III, MLE of <em>r</em> can be found from <a id="xref-disp-formula-17-1" class="xref-disp-formula" href="#disp-formula-17">Equation (17)</a>.<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-165.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-165.gif"/></noscript></span> <span class="disp-formula-label">(17)</span></span>where <em>n<sub>i</sub></em> is the observed sample size of the <em>i</em>th genotype and <em>n</em> is the total sample size (<em>i.e.</em>, <em>n</em>=<em>n</em><sub>1:9</sub>).</p><p id="p-42">To explain how the linkage phase can be determined by the estimated <em>r</em> from the four potential linkage phases, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows likelihood function profiles on experimental recombination frequency when both marker loci are category IV. When true recombination frequency was 0.2 (<em>i.e.</em>, two loci were linked) and true linkage phase was I (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>), <em>r</em> was estimated at 0.2 in linkage phase I, at 0.5 in linkage phases II and III, and at 0.8 in linkage phase IV. If the true linkage phase was II or III (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>), then <em>r</em> was estimated at 0.5 in linkage phases I and IV and at 0.2 or 0.8 in linkage phases II and III. If the true linkage phase was IV (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>), then <em>r</em> was estimated at 0.8 in linkage phase I, at 0.5 in linkage phases II and III, and at 0.2 in linkage phase IV. Obviously, if the experimental phase coincides with the true linkage phase, then the estimated <em>r</em> would be the lowest among all estimates of the four potential phases, which is actually equal to its true value. In other words, the experimental phase that has the lowest estimate of <em>r</em> can be viewed as the true linkage phase, and the lowest estimate can be viewed as the true value of <em>r</em>. When estimated <em>r</em> is lowest in linkage phases II and III, the two loci are randomly assigned to phase II or phase III. If the two loci were not linked (<em>i.e.</em>, true recombination frequency is 0.5), then <em>r</em> should be estimated at approximately 0.5 in all linkage phases (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3D</a>). In this case, linkage phase does not make any sense and is randomly assigned to one of the four phases.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/427/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Likelihood function of experimental recombination frequency when both loci are marker category IV, i.e., A1B1= C1D1 and A2B2= C2D2 (scenario 9 in Table 1), and the true recombination frequency is 0.2 (A, B, and C for close linkage) or 0.5 (D for no linkage). In this scenario, the female parent may have either genotype A1A2/B1B2 or genotype A1B2/B1A2, and so does the male parent. Therefore, there are four possible linkage phases, but only one could be true. The four possible phases are (1) the female and male parents have the same genotype A1A2/B1B2; (2) the female parent has genotype A1A2/B1B2 and the male parent has genotype A1B2/B1A2; (3) the female parent has genotype A1B2/B1A2 and the male parent has genotype A1A2/B1B2; and (4) the female and male parents have the same genotype A1B2/B1A2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2115005789" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Likelihood function of experimental recombination frequency when both loci are marker category IV, &lt;em&gt;i.e.&lt;/em&gt;, &lt;em&gt;A&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;= &lt;em&gt;C&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;D&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt; and &lt;em&gt;A&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;= &lt;em&gt;C&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;&lt;em&gt;D&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; (scenario 9 in Table 1), and the true recombination frequency is 0.2 (A, B, and C for close linkage) or 0.5 (D for no linkage). In this scenario, the female parent may have either genotype &lt;em&gt;A&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;A&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;/&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; or genotype &lt;em&gt;A&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;/&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;A&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;, and so does the male parent. Therefore, there are four possible linkage phases, but only one could be true. The four possible phases are (1) the female and male parents have the same genotype &lt;em&gt;A&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;A&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;/&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;; (2) the female parent has genotype &lt;em&gt;A&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;A&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;/&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; and the male parent has genotype &lt;em&gt;A&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;/&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;A&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;; (3) the female parent has genotype &lt;em&gt;A&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;/&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;A&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt; and the male parent has genotype &lt;em&gt;A&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;A&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;/&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;; and (4) the female and male parents have the same genotype &lt;em&gt;A&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;/&lt;em&gt;B&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;&lt;em&gt;A&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/427/F3.medium.gif" width="440" height="261"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/3/427/F3.medium.gif" width="440" height="261"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/427/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/427/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465726" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-43" class="first-child">Likelihood function of experimental recombination frequency when both loci are marker category IV, <em>i.e.</em>, <em>A</em><sub>1</sub><em>B</em><sub>1</sub>= <em>C</em><sub>1</sub><em>D</em><sub>1</sub> and <em>A</em><sub>2</sub><em>B</em><sub>2</sub>= <em>C</em><sub>2</sub><em>D</em><sub>2</sub> (scenario 9 in <a id="xref-table-wrap-1-13" class="xref-table" href="#T1">Table 1</a>), and the true recombination frequency is 0.2 (A, B, and C for close linkage) or 0.5 (D for no linkage). In this scenario, the female parent may have either genotype <em>A</em><sub>1</sub><em>A</em><sub>2</sub>/<em>B</em><sub>1</sub><em>B</em><sub>2</sub> or genotype <em>A</em><sub>1</sub><em>B</em><sub>2</sub>/<em>B</em><sub>1</sub><em>A</em><sub>2</sub>, and so does the male parent. Therefore, there are four possible linkage phases, but only one could be true. The four possible phases are (1) the female and male parents have the same genotype <em>A</em><sub>1</sub><em>A</em><sub>2</sub>/<em>B</em><sub>1</sub><em>B</em><sub>2</sub>; (2) the female parent has genotype <em>A</em><sub>1</sub><em>A</em><sub>2</sub>/<em>B</em><sub>1</sub><em>B</em><sub>2</sub> and the male parent has genotype <em>A</em><sub>1</sub><em>B</em><sub>2</sub>/<em>B</em><sub>1</sub><em>A</em><sub>2</sub>; (3) the female parent has genotype <em>A</em><sub>1</sub><em>B</em><sub>2</sub>/<em>B</em><sub>1</sub><em>A</em><sub>2</sub> and the male parent has genotype <em>A</em><sub>1</sub><em>A</em><sub>2</sub>/<em>B</em><sub>1</sub><em>B</em><sub>2</sub>; and (4) the female and male parents have the same genotype <em>A</em><sub>1</sub><em>B</em><sub>2</sub>/<em>B</em><sub>1</sub><em>A</em><sub>2</sub>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">Consistent with previous scenarios, <em>r<sub>F</sub></em> and <em>r<sub>M</sub></em> need to be defined to reflect the identified linkage phase after <em>r</em> and linkage phase are determined. For this purpose, <em>r<sub>F</sub></em> and <em>r<sub>M</sub></em> are both assigned to <em>r</em> in linkage phase I, assigned to <em>r</em> and 1−<em>r</em>, respectively, in linkage phase II, assigned to 1−<em>r</em> and <em>r</em>, respectively, in linkage phase III, and assigned to 1−<em>r</em> in linkage phase IV. For convenience, estimates of <em>r<sub>F</sub></em> and <em>r<sub>M</sub></em> are given in <a id="xref-disp-formula-18-1" class="xref-disp-formula" href="#disp-formula-18">Equation 18</a>.</p><span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-166.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-166.gif"/></noscript></span> <span class="disp-formula-label">(18)</span></span></div><div id="sec-11" class="subsection"><h3>Haploid building in clonal parents from their segregating progenies</h3><p id="p-45">For the clonal F<sub>1</sub> progenies, genotype of the female parent can be either <em>A</em><sub>1</sub><em>B</em><sub>1</sub>/<em>A</em><sub>2</sub><em>B</em><sub>2</sub> or <em>A</em><sub>1</sub><em>B</em><sub>2</sub>/<em>B</em><sub>1</sub><em>A</em><sub>2</sub>; genotype of the male parent can be either <em>C</em><sub>1</sub><em>D</em><sub>1</sub>/<em>C</em><sub>2</sub><em>D</em><sub>2</sub> or <em>C</em><sub>1</sub><em>D</em><sub>2</sub>/<em>D</em><sub>1</sub><em>C</em><sub>2</sub>. The linkage phase can be identified from the estimated recombination frequencies and constructed linkage maps and, finally, the four haploids in the two clonal parents can be built. Two haploids of the female parent are called HapA and HapB; those of the male parent are called HapC and HapD. Female haploid building tries to assign the two alleles <em>A</em> and <em>B</em> at each locus on the female map to haploids HapA and HapB. Male haploid building tries to assign the two alleles <em>C</em> and <em>D</em> at each locus on the male map to haploids HapC and HapD. Haploid building of ordered markers on one chromosome is similar for both parents. We use the female parent as an example to explain the building procedure.</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-46">Step 1: At the first ordered locus, allele <em>A</em> is assigned to HapA; allele <em>B</em> is assigned to HapB.</p></li><li id="list-item-2"><p id="p-47">Step 2: For the second ordered locus, if estimated <em>r<sub>F</sub></em> with the first locus is lower than 0.5, then allele <em>A</em> is assigned to HapA; allele <em>B</em> is assigned to HapB. Otherwise, allele <em>B</em> is assigned to HapA and allele <em>A</em> is assigned to HapB.</p></li><li id="list-item-3"><p id="p-48">Step 3: For the next ordered locus, if estimated <em>r<sub>F</sub></em> with its previous locus is lower than 0.5, then allele <em>A</em> is assigned to HapA, and allele <em>B</em> is assigned to HapB if allele <em>A</em> at the previous locus is on HapA; allele <em>B</em> is assigned to HapA and allele <em>A</em> is assigned to HapB if allele <em>B</em> at the previous locus is on HapA. If estimated <em>r<sub>F</sub></em> with its previous locus is more than 0.5, then allele <em>B</em> is assigned to HapA, and allele <em>A</em> is assigned to HapB if allele <em>A</em> at the previous locus is on HapA; allele <em>A</em> is assigned to HapA and allele <em>B</em> is assigned to HapB if allele <em>B</em> at the previous locus is on HapA.</p></li><li id="list-item-4"><p id="p-49">Step 4: Repeat the process from step 3 until the last ordered locus on the chromosome.</p></li></ul></div><div id="sec-12" class="subsection"><h3>Marker categories and linkage analysis in double cross populations</h3><p id="p-50">Double cross populations in plants have four inbred lines, A, B, C, and D, as parents that are homozygous at most chromosomal locations (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/FigureS1.pdf">Figure S1</a>). First, one F<sub>1</sub> hybrid is made between inbred lines A and B; the other F<sub>1</sub> hybrid is made between inbred lines C and D. Then, a double cross is made between the two F<sub>1</sub> hybrids; one is used as female and the other one is used as male. When polymorphism markers are screened in the four inbred lines, the four alleles in double cross populations can be clearly assigned. In this case, five marker categories can be differentiated on the number of identifiable alleles in the four original lines and the number of identifiable genotypes in their double cross progenies (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/FigureS2.pdf">Figure S2</a>). Categories I to III are similar to those in clonal F<sub>1</sub>. Category IV in clonal F<sub>1</sub> can be further divided into two categories in double cross, which are denoted as categories IV and V. For category IV (or A = CB = D), allele <em>A</em> is the same as allele <em>C</em>, and allele <em>B</em> is the same as allele <em>D</em>. For category V (or A = DB = C), allele <em>A</em> is the same as allele <em>D</em>, and allele <em>B</em> is the same as allele <em>C</em>.</p><p id="p-51">For two loci, genotypes of the four inbred lines are <em>A</em><sub>1</sub><em>A</em><sub>1</sub>, <em>B</em><sub>1</sub><em>B</em><sub>1</sub>, <em>C</em><sub>1</sub><em>C</em><sub>1</sub>, and <em>D</em><sub>1</sub><em>D</em><sub>1</sub> at locus 1, and <em>A</em><sub>2</sub><em>A</em><sub>2</sub>, <em>B</em><sub>2</sub><em>B</em><sub>2</sub>, <em>C</em><sub>2</sub><em>C</em><sub>2</sub>, and <em>D</em><sub>2</sub><em>D</em><sub>2</sub> at locus 2. Linkage phases in the female and male F<sub>1</sub> hybrids are known as <em>A</em><sub>1</sub><em>A</em><sub>2</sub>/<em>B</em><sub>1</sub><em>B</em><sub>2</sub> and <em>C</em><sub>1</sub><em>C</em><sub>2</sub>/<em>D</em><sub>1</sub><em>D</em><sub>2</sub>, which are equivalent to linkage phase I in clonal F<sub>1</sub>. When category V is absent, scenarios 1 to 9 in clonal F<sub>1</sub> are still applicable in double cross populations. For these scenarios, theoretical genotypic frequencies and formulas in estimating <em>r<sub>F</sub></em>, <em>r<sub>M</sub></em>, and <em>r</em> are the same as those for clonal F<sub>1</sub> in the case of linkage phase I, <em>i.e.</em>, <em>r<sub>F</sub></em> and <em>r<sub>M</sub></em> are both smaller than 0.5 if they can be estimated.</p><p id="p-52">There are five new scenarios for recombination frequency estimation when category V is present. In scenario 10, locus 1 is category I and locus 2 is category V. In scenario 11, locus 1 is category II and locus 2 is category V. In scenario 12, locus 1 is category III and locus 2 is category V. In scenario 13, locus 1 is category IV and locus 2 is category V. In scenario 14, the two loci are category V.</p><p id="p-53">In scenario 10, the 12 identifiable genotypes are the same as scenario 4 in <a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>. Theoretical frequency of each genotype is equal to the corresponding value in <a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a> by substituting <em>r<sub>M</sub></em> with 1−<em>r<sub>M</sub></em> (see <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS2.pdf">Table S2</a>). In scenario 11, the six identifiable genotypes are the same as scenario 6 in <a id="xref-table-wrap-6-3" class="xref-table" href="#T6">Table 6</a>. Theoretical frequency of each genotype is equal to the corresponding value of scenario 6 in <a id="xref-table-wrap-6-4" class="xref-table" href="#T6">Table 6</a> by substituting <em>r<sub>M</sub></em> with 1−<em>r<sub>M</sub></em> (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS3.pdf">Table S3</a>). In scenario 12, the six identifiable genotypes and their theoretical frequencies are the same as scenario 8 in <a id="xref-table-wrap-6-5" class="xref-table" href="#T6">Table 6</a> (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS3.pdf">Table S3</a>). In scenario 13, genotypes and their theoretical frequencies are the same as linkage phases II and III of scenario 9 in <a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7</a> (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS4.pdf">Table S4</a>). In scenario 14, genotypes and their theoretical frequencies are the same as linkage phase I of scenario 9 in <a id="xref-table-wrap-7-3" class="xref-table" href="#T7">Table 7</a> (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS4.pdf">Table S4</a>). Methods for estimating <em>r</em> are similar to the corresponding scenarios in clonal F<sub>1</sub>. For convenience, theoretical genotypic frequencies at two loci for scenarios 10 to 14 are given in <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS2.pdf">Table S2</a>, <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS3.pdf">Table S3</a>, and <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS4.pdf">Table S4</a>.</p></div><div id="sec-13" class="subsection"><h3>LOD score in testing the linkage relationship between two loci</h3><p id="p-54">The existence of the linkage can be tested by the following two hypotheses.<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-167.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-167.gif"/></noscript></span></span>where <em>H</em><sub>0</sub> is the null hypothesis corresponding to no genetic linkage, <em>H<sub>A</sub></em> is the alternative hypothesis corresponding to the linkage relationship between two loci, and <em>r</em> is the combined recombination frequency. The log-likelihood function under the null hypothesis is <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-168.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-168.gif"/></noscript></span></span>, whereas the log-likelihood function under the alternative hypothesis is <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-169.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-169.gif"/></noscript></span></span>. The <em>LOD</em> score can be calculated from the log-likelihoods under the two hypotheses, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-170.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-170.gif"/></noscript></span></span>, where log is the logarithm function of base 10.</p></div><div id="sec-14" class="subsection"><h3>One simulated population and one actual population</h3><p id="p-55">We considered one chromosome with 20 evenly distributed markers in simulation. Recombination frequencies between any two neighboring markers were set at 0.05, equivalent to a genetic distance of 5.27 cM using Haldane mapping function (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Haldane 1919</a>).</p><p id="p-56">One population with 200 clonal F<sub>1</sub> progenies was simulated by the genetics and breeding simulation tool of QuLine (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wang <em>et al.</em> 2003</a>, <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">2004</a>). Five markers were randomly chosen and assigned to each of the four categories (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2</a>). Markers 8, 11, 14, 17, and 19 were category I; markers 1, 2, 13, 15, and 20 were assigned to category II; markers 4, 5, 7, 9, and 18 were assigned to category III. Alleles <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-171.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-171.gif"/></noscript></span></span> for markers 10 and 12, and alleles <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-172.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/3/427/embed/mml-math-172.gif"/></noscript></span></span> for markers 3, 6 and 16, with both representing markers of category IV. To simulate the unknown linkage phases, alleles <em>A</em> and <em>B</em> were purposely swapped for markers 5 and 18. Alleles <em>C</em> and <em>D</em> were swapped for markers 14, 15, and 20. For markers 8, 12, and 16, alleles <em>A</em> and <em>B</em> were swapped and alleles <em>C</em> and <em>D</em> were swapped.</p><p id="p-57">The actual double cross population used in this study was derived from four maize inbred lines, developed by the College of Agronomy, Henan Agricultural University (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Li <em>et al.</em> 2013</a>). The population consists of 277 double cross individuals. Two single crosses were first made in Zhengzhou, Henan, China, in summer 2008. One was between maize inbred lines 276 and 72, and the other was between maize inbred lines A188 and Jiao51. The two single crosses were then planted in Ledong, Hainan, China, in winter 2008, and the double cross was made at the flowering stage. The double cross population was planted in Zhengzhou in spring 2009 for phenotyping. Polymorphism of SSR molecular markers was first screened in the two single crosses. Then, the double cross population was genotyped by 220 polymorphism SSR markers. The original four parental lines were not genotyped. Therefore, linkage phases in this population are unknown, and the linkage analysis method of clonal F<sub>1</sub> is applicable.</p><p id="p-58">A threshold of recombination frequency 0.3 was used for marker grouping in the actual population. A combined algorithm of nearest neighbor and Two-opt algorithm of Traveling Salesman Problem (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Lin and Kernighan 1973</a>) was used for marker ordering in both populations. The nearest neighbor algorithm was used to determine an initial solution that quickly yielded a short tour, but usually not the shortest one. Then Two-opt algorithm was used for improving the solution (Supplementary Materials, see <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/FileS3.pdf">File S3</a>). Algorithms for estimating recombination frequencies and building linkage map were implemented in the software called GACD (available from <a href="http://www.isbreeding.net">www.isbreeding.net</a>). For comparison, JoinMap4.1, OneMap, and R/qtl were used for linkage map construction in the simulated population. The mapping algorithm in JoinMap4.1 was maximum likelihood mapping with the following parameters: chain length = 1000; initial acceptance probability = 0.25; cooling control parameter = 0.001; and stop after 10000 chins without improvement. Function “order.seq” in OneMap was used for ordering, where the best order was determined in a window size of five markers. The best order in R/qtl was determined by function “orderMarker,” where the initial order was established by a greedy algorithm and was refined by rippling. In the simulated population, Haldane mapping function was used to convert recombination frequency (<em>r</em>) to map distance (<em>d</em>) in cM. In the maize population, Kosambi mapping function (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Kosambi 1944</a>) was used to convert <em>r</em> to <em>d</em> in cM.</p></div></div><div class="section results" id="sec-15"><h2 class="">Results</h2><div id="sec-16" class="subsection"><h3>Estimated recombination frequencies in simulated population</h3><p id="p-59">Theoretical recombination frequencies between the 20 simulated markers were shown in the upper triangular matrix (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS5.pdf">Table S5</a>). The closer to the diagonal, the lower the recombination frequencies would be. For example, recombination frequencies between marker 1 and markers 2, 8, and 19 were 0.05, 0.26, and 0.42, respectively. Recombination frequencies of marker pairs 8 and 9, 8 and 15, and 8 and 20 were 0.05, 0.26, and 0.36 (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS5.pdf">Table S5</a>), respectively.</p><p id="p-60">The lower triangular matrix of <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS5.pdf">Table S5</a> showed the estimated recombination frequencies between the 20 markers. Combined recombination frequencies cannot be estimated if one marker is category II and the other one is category III. For example, recombination frequencies between marker pair 1 and 4 and marker pair 5 and 13 cannot be estimated, which were left as blank in <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS5.pdf">Table S5</a>. When the combined recombination frequencies could be estimated, the estimates were close to their true values. For example, marker 1 was category II, its true recombination frequencies with markers 2, 8, and 19 were 0.05, 0.26, and 0.42, and the estimates were 0.05, 0.22, and 0.48, respectively. Marker 8 was category I, its true recombination frequencies with markers 9, 15, and 20 were 0.05, 0.26, and 0.36, and the estimates were 0.03, 0.27, and 0.42, respectively.</p><p id="p-61">If combined recombination frequency cannot be estimated, then the corresponding marker distance and LOD score cannot be calculated either. The upper triangular matrix showed the estimated map distance between the 20 markers (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS6.pdf">Table S6</a>). The closer between two markers, the smaller the estimated distance is. For example, the true recombination frequencies of marker pairs 1 and 2, 1 and 8, and 1 and 19 were 0.05, 0.26, and 0.42 (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS5.pdf">Table S5</a>). Their estimated distances were 5.3 cM, 29.0 cM, and 160.9 cM (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS6.pdf">Table S6</a>), respectively. The true recombination frequencies of marker pairs 8 and 9, 8 and 15, and 8 and 20 were 0.05, 0.26, and 0.36 (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS5.pdf">Table S5</a>). Their estimated distances were 3.1 cM, 37.8 cM, and 88.6 cM (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS6.pdf">Table S6</a>), respectively. It should be noted that the map length of a chromosome is calculated from lengths of individual ordered intervals, rather than the recombination frequency between the first and the last markers.</p><p id="p-62">The lower triangular matrix of <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS6.pdf">Table S6</a> showed LOD score between the 20 markers. The closer between two markers, the greater the LOD score is. For example, the true recombination frequencies between marker pairs 1 and 2, 1 and 8, and 1 and 19 were 0.05, 0.26, and 0.42 (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS5.pdf">Table S5</a>). Their LOD scores were 43.0, 14.4, and 0.1 (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS6.pdf">Table S6</a>), respectively. The true recombination frequencies between marker pairs 8 and 9, 8 and 15, and 8 and 20 were 0.05, 0.26, and 0.36 (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS5.pdf">Table S5</a>). Their LOD scores were 48.5, 10.0, and 1.3 (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS6.pdf">Table S6</a>), respectively.</p></div><div id="sec-17" class="subsection"><h3>Marker ordering in simulated population</h3><p id="p-63">Estimates of the combined recombination frequencies were used to order the 20 markers, and the best order with the shortest map length was shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>, which was the same as the predefined order. The estimated length of the chromosome was 101.79 cM, close to the true length 100.13 cM. Average marker distance was 5.36 cM, close to the true value 5.27 cM.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/3/427/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The combined, female, and male linkage maps of 20 markers in a simulated clonal F1 population with 200 progenies. Haldane mapping function was used to convert recombination frequency to genetic distance." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2115005789" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The combined, female, and male linkage maps of 20 markers in a simulated clonal F&lt;sub&gt;1&lt;/sub&gt; population with 200 progenies. Haldane mapping function was used to convert recombination frequency to genetic distance.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/3/427/F4.medium.gif" width="440" height="222"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/3/427/F4.medium.gif" width="440" height="222"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/3/427/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/3/427/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465772" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-64" class="first-child">The combined, female, and male linkage maps of 20 markers in a simulated clonal F<sub>1</sub> population with 200 progenies. Haldane mapping function was used to convert recombination frequency to genetic distance.</p><div class="sb-div caption-clear"></div></div></div><p id="p-65">The female map does not contain markers of category II, and the male map does not contain markers of category III. The order of markers in the female and male maps were the same as that in the combined map, but map distances between markers were estimated by the female and male recombination frequencies, respectively. In the simulated population, lengths of the female and male maps were 81.90 cM and 103.02 cM, respectively (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4, B and C</a>). For the 20 markers, 1, 2, 13, 15, and 20 are category II (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS5.pdf">Table S5</a>) and therefore do not appear on the female map. Marker 3 was located at the beginning and marker 19 located at the end on the female map, which explained the reduced female map length. Markers, 4, 5, 7, 9, and 17 are category III (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS4.pdf">Table S4</a> and <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS5.pdf">Table S5</a>); therefore, they do not appear on the male map. However, marker 1 was still located at the beginning and 20 was still located at the end on the male map, which explained the map length similar to the combined one.</p></div><div id="sec-18" class="subsection"><h3>Four haploids of two parents in the simulated population</h3><p id="p-66">Using estimated <em>r<sub>F</sub></em> and <em>r<sub>M</sub></em> between neighboring markers, four haploids of parents at 20 marker loci were determined (<a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a>). The first marker is category II, which had no polymorphism in the female parent. It was not included on the female map, but it was included on the male map (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4, B and C</a>). Alleles on HapA and HapB were represented by <em>X</em>, which can be either allele <em>A</em> or allele <em>B</em>. Alleles on HapC and HapD were <em>C</em> and <em>D</em>, respectively. The second marker is category II as well. The estimated <em>r<sub>M</sub></em> with previous marker was 0.05 (less than 0.5). Alleles on HapA and HapB were represented by <em>X</em>, which could be either allele <em>A</em> or allele <em>B</em>. Alleles on HapC and HapD were <em>C</em> and <em>D</em>, respectively, which were the same haploids as those of the previous locus. Marker 3 was the first on the female map (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>). Alleles <em>A</em> and <em>B</em> were on HapA and HapB (<a id="xref-table-wrap-8-2" class="xref-table" href="#T8">Table 8</a>). It was the third marker on the male map (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4C</a>). Estimated <em>r<sub>M</sub></em> with previous marker was 0.975, which was more than 0.5. Alleles <em>D</em> and <em>C</em> were assigned to HapC and HapD, respectively, which were opposite to the previous locus. The four haploids in <a id="xref-table-wrap-8-3" class="xref-table" href="#T8">Table 8</a> were consistent with the predefined haploid types.</p><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465723/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465723/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8</span> <span class="caption-title">Haploid building of female and male parents and the updated marker category</span><div class="sb-div caption-clear"></div></div></div><p id="p-68">Marker category IV in clonal F<sub>1</sub> can be further divided into two categories, <em>i.e.</em>, categories IV and V in double cross (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/FigureS2.pdf">Figure S2</a>). In a simulated population, markers 3, 6, 10, 12, and 16 were category IV. Taking marker 3 as an example, alleles on HapA, HapB, HapC, and HapD were <em>A</em>, <em>B</em>, <em>D</em>, and <em>C</em>, respectively. Its category was redefined as category V of double cross (<a id="xref-table-wrap-8-4" class="xref-table" href="#T8">Table 8</a>).</p><p id="p-69">For HapA and HapB of the female parents (<a id="xref-table-wrap-8-5" class="xref-table" href="#T8">Table 8</a>), if we exchange alleles <em>A</em> and <em>B</em> at loci 5, 8, 12, 16, and 18, then HapA will have <em>A</em> alleles at all loci and HapB will have <em>B</em> alleles at all loci. For HapC and HapD of the male parents (<a id="xref-table-wrap-8-6" class="xref-table" href="#T8">Table 8</a>), if we exchange alleles <em>C</em> and <em>D</em> at loci 3, 6, 8, 12, 14, 15, and 20, then HapC will have <em>C</em> alleles at all loci and HapD will have <em>D</em> alleles at all loci. If the four haploids built earlier could be viewed as haploids of the four inbred lines in a double cross, then clonal F<sub>1</sub> is equivalent to double cross!</p></div><div id="sec-19" class="subsection"><h3>Comparison with JoinMap, OneMap, and R/qtl for linkage map construction</h3><p id="p-70">General information of combined linkage maps of the simulated population built by GACD, JoinMap4.1, OneMap, and R/qtl were shown (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS7.pdf">Table S7</a>). R/qtl can only conduct linkage mapping in phase-known double cross, so marker categories and genotypes after haploid building were imported into R/qtl. Marker orders given by GACD, OneMap, and R/qtl were the same as the predefined order in the simulated model. However, marker order given by JoinMap4.1 was far from the predefined (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS7.pdf">Table S7</a>). The first and last markers were Marker 12 and Marker 18, respectively. The true map length was 100.13 cM. Length was estimated at 101.79 cM from GACD, 15211.04 cM from JoinMap, 103.83 cM from OneMap, or 104.22 cM from R/qtl. The reason for the extremely large map length from JoinMap was the estimated value of 0.5 of recombination frequency between some neighboring markers in the female or male maps, which was converted to a distance of 10,000.0 cM in JoinMap. For example, recombination frequency between markers 3 and 5 belonging to category V and III was estimated at 0.5 on the female map, corresponding to a distance of 10,000.0 cM on the female map and 5007.99 cM on the combined map. Time spent for building the maps was 8 sec by GACD, 30 sec by JoinMap, 455 sec by OneMap, and 63 sec by R/qtl on a computer with 1.60 GHz CPU and 3.00 GB RAM.</p><p id="p-71">Comparison of different software packages was also conducted in a simulated clonal F<sub>1</sub> population with distorted markers (Supplementary Materials, see <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/FileS4.pdf">File S4</a>) and a simulated clonal F<sub>1</sub> population with 200 individuals and 200 markers belonging to category IV (Supplementary Materials, see <a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/FileS5.pdf">File S5</a>). A greater advantage was observed for the marker number 200 in one single chromosome (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS8.pdf">Table S8</a>). GACD took 0.5 min, JoinMAP took 5 min, OneMAP took 537 min, and R did not output any results. GACD results in the shortest linkage map closest to the true length in the shortest time (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/TableS8.pdf">Table S8</a>). The reason may be as follows. Previous studies tried to estimate recombination frequency, determine linkage phase, and build linkage map simultaneously. In our study, we first estimate all pair-wise recombination frequencies (<em>i.e.</em>, step 1). Linkage phases were determined from the estimated recombination frequencies (<em>i.e.</em>, step 2). Linkage map was built based on the matrix of all pair-wise recombination frequencies (<em>i.e.</em>, step 3). Finally, the four haploids were built from the completed linkage maps (<em>i.e.</em>, step 4). Separating a complicated genetic question into four clearly defined steps results in more accurate genetic linkage maps in shorter time. In addition, we believe the adoption of the optimization algorithm in solving the Traveling Salesman Problems also contributes to the ordering efficiency.</p></div><div id="sec-20" class="subsection"><h3>Linkage maps in actual double cross population</h3><p id="p-72">In the actual population, the missing marker rate was at 6.49%. Among the 220 markers, 60 markers showed segregation distortion under significance level 0.05. Recombination frequencies of all marker pairs were estimated and then used for linkage map construction. The combined genetic linkage map was constructed by 219 SSR molecular markers using the software GACD. One marker cannot be linked with any other markers and was deleted. The 10 chromosomes had 25, 28, 25, 24, 21, 19, 18, 16, 25, and 18 relatively evenly distributed markers, respectively (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/FigureS3.pdf">Figure S3</a>). The whole genome was 1778.09 cM in length, and the average marker distance was 8.51 cM.</p><p id="p-73">The 10 female chromosomes (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/FigureS3.pdf">Figure S3</a>) had 19, 19, 20, 13, 16, 13, 12, 14, 17, and 15 markers, respectively, with a total of 158 markers. The total female map length was 1796.92 cM. The 10 male chromosomes (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/FigureS3.pdf">Figure S3</a>) had 18, 19, 22, 21, 17, 14, 14, 9, 19, and 15 markers, respectively, with a total of 168 markers. The total male map length was 1599.13 cM.</p><p id="p-74"><a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Li <em>et al.</em> (2013)</a> used JoinMap4.0 to build the linkage maps for this actual population. Kosambi mapping function was used to convert recombination frequency to genetic distance. As indicated in their study, 213 makers were included in the 11 linkage groups of the combined map. The other seven markers were not linked. The whole genome was 1626.3 cM, and the average marker distance was 1626.3/(213−11) = 8.05 cM. Compared with the map by JoinMap, our method provided a methodology that has the following advantages. First, the number of linkage groups from GACD was the same as the number of chromosomes in maize genome. Second, GACD links more markers than JoinMap. One marker was identified by GACD to be unlinked, but seven markers were unlinked by JoinMap. The length of genome from GACD was slightly longer than that from JoinMap. This may be caused by two possible reasons: more markers were included on the linkage maps by GACD and chromosome 2 was split into two by JoinMap.</p></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><div id="sec-22" class="subsection"><h3>Linkage analysis in clonal F<sub>1</sub> progenies using all informative markers</h3><p id="p-75">Linkage analysis and map construction are crucial steps in genetic studies of quantitative traits and provide the basis for map-based gene cloning and marker-assisted breeding. A key to linkage map construction is the accurate estimation of recombination frequency, which has been widely studied for various populations in plants over a long period of time (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Fisher 1935</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Haldane and Smith 1947</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Morton 1955</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Smith 1959</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bailey 1961</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ott 1974</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Nordheim <em>et al.</em> 1983</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Ritter <em>et al.</em> 1990</a>, <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">1996</a>; <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Wu <em>et al.</em> 2002a</a>, <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">b</a>; <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">van Ooijen 2011</a>). <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Säll and Nilsson (1994)</a> showed that the accuracy of recombination frequency estimation was affected by limited sample size, heterogeneity in recombination frequency between sexes or among meiosis, and factors that distort the segregation misclassification or differential viability. <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Hackett and Broadfoot (2003)</a> investigated that accuracy of linkage maps was reduced by missing values and/or typing errors in genotyping, but segregation distortion had little effect on marker order. <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Sun <em>et al.</em> (2012)</a> investigated the estimation efficiency of recombination frequency in 12 bi-parental populations. They concluded that larger population size and smaller recombination frequency resulted in higher LOD score and smaller deviation. Advanced backcrossing and selfing populations had lower precision in estimating the recombination frequency due to the enlarged recombination frequency.</p><p id="p-76">The four marker categories (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2</a>) considered in this study represented all polymorphism markers that could provide the required information for genetic studies. Linkage analysis was conducted for markers not only in the same category but also in different categories. Three sets of recombination frequencies could be estimated accordingly to build the female, male, and combined linkage maps simultaneously. Results from simulated populations and one actual maize population demonstrated the accuracy of the proposed method and its advantages over other software packages. Methodology developed in this study, together with the freely available GACD software, provides an integrated and convenient approach that will greatly facilitate the genetic research of clonal species and double crosses.</p><p id="p-77">Single-nucleotide polymorphism (SNP) markers are more and more often being used in genetic analysis. <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Liu <em>et al.</em> (2014)</a> presented a HighMap method for constructing high-density linkage maps from next-generation sequencing (NGS). HighMap used an iterative ordering and error correction strategy based on a k-nearest neighbor algorithm and a Monte Carlo multipoint maximum likelihood algorithm, which also provided an idea for dealing with NGS data. Due to the bi-allelic characteristic, individual SNP markers cannot be in category I. However, any SNP marker can be category II, III, or IV in clonal F<sub>1</sub>, or category II, III, IV, or V in double crosses. In addition, by using the concept of haplotypes, it is possible to covert SNP markers to fully informative category I markers. For example, one haplotype is consisted of two closely linked SNP loci. Four genotypes can be identified by considering the two loci together, <em>i.e.</em>, 11, 10, 01, and 00. Then, the haplotype can be treated as category I marker in genetic analysis.</p></div><div id="sec-23" class="subsection"><h3>Difference and similarity between clonal F<sub>1</sub> and double cross</h3><p id="p-78">In clonal F<sub>1</sub>, genotype of the female parent can be either <em>A</em><sub>1</sub><em>B</em><sub>1</sub>/<em>A</em><sub>2</sub><em>B</em><sub>2</sub> or <em>A</em><sub>1</sub><em>B</em><sub>2</sub>/<em>B</em><sub>1</sub><em>A</em><sub>2</sub>; genotype of the male parent can be either <em>C</em><sub>1</sub><em>D</em><sub>1</sub>/<em>C</em><sub>2</sub><em>D</em><sub>2</sub> or <em>C</em><sub>1</sub><em>D</em><sub>2</sub>/<em>D</em><sub>1</sub><em>C</em><sub>2</sub>. In double cross, there are four homozygous inbred lines whose genotypes may be known. Alleles <em>A</em>, <em>B</em>, <em>C</em>, and <em>D</em> at each polymorphism locus can be traced back to the four inbred lines, when the four lines have been genotyped. In this case, genotype of the single cross between lines A and B is <em>A</em><sub>1</sub><em>B</em><sub>1</sub>/<em>A</em><sub>2</sub><em>B</em><sub>2</sub>; genotype of the single cross between lines C and D is <em>C</em><sub>1</sub><em>D</em><sub>1</sub>/<em>C</em><sub>2</sub><em>D</em><sub>2</sub>. Therefore, double cross is actually a special case of clonal F<sub>1</sub> in which only linkage phase I is applicable (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/FigureS4.pdf">Figure S4</a>).</p><p id="p-79">In a double cross where polymorphism loci are only screened in the two single crosses, linkage phases become unknown before estimating recombination frequencies. Genotype of one single cross can be either <em>A</em><sub>1</sub><em>B</em><sub>1</sub>/<em>A</em><sub>2</sub><em>B</em><sub>2</sub> or <em>A</em><sub>1</sub><em>B</em><sub>2</sub>/<em>B</em><sub>1</sub><em>A</em><sub>2</sub>; genotype of the other single cross can be either <em>C</em><sub>1</sub><em>D</em><sub>1</sub>/<em>C</em><sub>2</sub><em>D</em><sub>2</sub> or <em>C</em><sub>1</sub><em>D</em><sub>2</sub>/<em>D</em><sub>1</sub><em>C</em><sub>2</sub>. In this case, the double cross must be treated as one clonal F<sub>1</sub> population for genetic analysis (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/FigureS4.pdf">Figure S4</a>), as is the case for the actual maize population used in this study.</p><p id="p-80">Linkage phases in both parents of the clonal F<sub>1</sub> can be determined by linkage analysis, from which four haploids can be built. If the four haploids could be viewed as haploids of the four inbred lines in a double cross, then clonal F<sub>1</sub> is equivalent to double cross. In short, there are many similarities between clonal F<sub>1</sub> and double cross, although difference does occur (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/FigureS4.pdf">Figure S4</a>). It is important in genetics to know when clonal F<sub>1</sub> and double cross are equivalent and when they are not. Previous genetic studies focused on only one of clonal F<sub>1</sub> or double cross population. To our understanding, this study is the first that tried to combine the two kinds of populations. Based on the linkage analysis, two haploids of the female parent and two haploids of the male parent can be built, and then the clonal F<sub>1</sub> progenies can be viewed as a double cross population derived from four inbred lines. The unified QTL mapping method for the two kinds of populations will be fully investigated in another article (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Zhang <em>et al.</em> 2015</a>).</p></div><div id="sec-24" class="subsection"><h3>Classification of marker categories in clonal F<sub>1</sub> and double cross</h3><p id="p-81">In clonal F<sub>1</sub> and double crosses, both the number of identifiable alleles in parents and the number of identifiable genotypes in F<sub>1</sub> progenies need to be considered in the classification of each marker locus. <a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">Wu <em>et al.</em> (2002a)</a> only considered parents in marker classification, resulting in 18 possible cross types. However, many of them are identical in linkage analysis, and most cross types can be classified into the four marker categories in this study. For example, types A<sub>1</sub> to A<sub>4</sub> in <a id="xref-ref-34-7" class="xref-bibr" href="#ref-34">Wu <em>et al.</em> (2002a)</a> are identical to category I as defined in this study, because they all generate four genotypes that can be identified in the progenies.</p><p id="p-82">Null alleles were also considered in <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Maliepaard <em>et al.</em> (1997)</a> and <a id="xref-ref-34-8" class="xref-bibr" href="#ref-34">Wu <em>et al.</em> (2002a</a>, <a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">b)</a>. To our understanding, it is difficult to determine whether one parent carries two identical alleles or carries one allele and one null allele in practice. In the case of no missing data and no segregation distortion, type D<sub>1</sub> in <a id="xref-ref-34-9" class="xref-bibr" href="#ref-34">Wu <em>et al.</em> (2002a)</a> can be decided by the 1:1 ratio test of two marker types in the progenies, and type A<sub>3</sub> can be decided by the 1:1:1:1 ratio test of four marker types in the progenies. Unfortunately, missing data and segregation distortion are common in practical populations. In the case of type D<sub>1</sub> and a large amount of missing marker points, we may wrongly say there are three or four marker classes. Even though we do know the number of marker type classes, the segregation ratio could be seriously affected by distortion. Therefore, we do not make the difference between cross types D<sub>1</sub> and A<sub>3</sub>. Instead, both types were treated as nonpolymorphism in the male parent, <em>i.e.</em>, category III in this study.</p></div><div id="sec-25" class="subsection"><h3>Wider applications of the clonal genetic analysis methods</h3><p id="p-83">In practice, clonal F<sub>1</sub> progenies may come from the selfing pollination of one clonal parent, <em>i.e.</em>, female and male parents are from one clone population (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/FigureS4.pdf">Figure S4</a>). In this case there are two alleles at each locus, and only marker category IV and linkage phases I and IV are applicable. Methods proposed in this study can be readily used to estimate recombination frequency, identify linkage phase, and build the two haploids of the clonal parent. In self-pollinated and cross-pollinated species, an F<sub>2</sub> population is the selfing generation of one F<sub>1</sub> hybrid between two inbred parents. Linkage phases are known when both inbred parents are genotyped. In this case, the clonal F<sub>1</sub> derived from the selfing of one clonal parent can be viewed as an F<sub>2</sub> population, after the two parental haploids are built.</p><p id="p-84">If selfing can be viewed as a cross between the F<sub>1</sub> hybrid and itself, the F<sub>2</sub> population becomes a special case of clonal F<sub>1</sub> when linkage phases are unknown, or a special case of double cross when linkage phases are known (<a href="http://www.g3journal.org/content/suppl/2015/01/15/g3.114.016022.DC1/FigureS4.pdf">Figure S4</a>). In the F<sub>2</sub> population, there are two alleles at each locus; therefore, only marker category IV is applicable. Haploids built in clonal F<sub>1</sub> and double cross may help to identify and correct markers that are misclassified for the two inbred parents. Moreover, genetic analysis in an F<sub>2</sub> population can still be performed by the clonal genetic analysis methods, even when there is no genotypic data on its two parental lines or on its F<sub>1</sub> ancestry.</p><p id="p-85">More broadly, methodology proposed in this study can be applied in genetic populations derived from any two heterozygotes in animals and plants. For example, in animals, linkage analysis is normally conducted in progenies between one female parent and one male parent, both are highly heterozygous, and they are drawn from a large random-mating population. By using the methodology of clonal F<sub>1</sub>, it is possible to build the female and male linkage maps to reflect the sex-specific recombination frequencies.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-86">This work was supported by the Natural Science Foundation of China (project no. 31200917), the National 973 Research Program of China (project no. 2014CB138105), and the Generation Challenge Program (GCP) and HarvestPlus Challenge Program of CGIAR.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-9"><p id="p-87">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016022/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.016022/-/DC1</a></p></li><li class="fn" id="fn-10"><p id="p-88"><em>Communicating editor: J. D. Faris</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-12-02"><span class="received-label">Received </span>December 2, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-01-02"><span class="accepted-label">Accepted </span>January 2, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Zhang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-book" id="cit-5.3.427.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allard</span>  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-source">Principles of Plant Breeding</span>, <span class="cit-edition">Ed. 2nd</span>. <span class="cit-publ-name">John Wiley &amp; Sons, Inc.</span>, <span class="cit-publ-loc">New York, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-5.3.427.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">N. T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1961</span> <span class="cit-source">Introduction to the Mathematical Theory of Genetic Linkage</span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.3" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">Ś.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzg4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.4" data-doi="10.1046/j.1439-0523.2003.00924.x"><div class="cit-metadata"><cite>Carlier, J. D., A. Reis, M. F. Duval, G. C. D’Eeckenbrugge, and J. M. Leitão, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic maps of RAPD, AFLP and ISSR markers in <em>Ananas bracteatus</em> and <em>A. comosus</em> using the pseudo-testcross strategy.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">186</span>–<span class="cit-lpage">192</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D123%26rft.spage%253D186%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1439-0523.2003.00924.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1439-0523.2003.00924.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.5" data-doi="10.1111/j.1469-1809.1935.tb02227.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1935</span> <span class="cit-article-title">The detection of linkage with “dominant” abnormalities.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">187</span>–<span class="cit-lpage">201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D6%26rft.spage%253D187%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1935.tb02227.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1935.tb02227.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.6" data-doi="10.1007/s001220050580"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fregene</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angel</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rodriguez</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chavarriaga</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">A molecular genetic map of cassava (<em>Manihot esculenta</em> Crantz).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">95</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">441</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D95%26rft.spage%253D431%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050580%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050580&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1997XV26800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.7" data-doi="10.1038/sj.hdy.6800173"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broadfoot</span>  <span class="cit-name-given-names">L. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Effects of genotyping errors, missing values and segregation distortion in molecular marker data on the construction of linkage maps.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.stitle%253DHeredity%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D38%26rft.atitle%253DEffects%2Bof%2Bgenotyping%2Berrors%252C%2Bmissing%2Bvalues%2Band%2Bsegregation%2Bdistortion%2Bin%2Bmolecular%2Bmarker%2Bdata%2Bon%2Bthe%2Bconstruction%2Bof%2Blinkage%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6800173%26rft_id%253Dinfo%253Apmid%252F12522423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.hdy.6800173&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12522423&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181165800007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">J. B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1919</span> <span class="cit-article-title">The combination of linkage values, and the calculation of distance between the loci of linked factors.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BGenet.%26rft.volume%253D8%26rft.spage%253D299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=000200614300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.9" data-doi="10.1111/j.1469-1809.1947.tb02374.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haldane</span>  <span class="cit-name-given-names">J. B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1947</span> <span class="cit-article-title">A new estimate of the linkage between the genes for haemophilia and colour-blindness in man.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">10</span>–<span class="cit-lpage">31</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D14%26rft.spage%253D10%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1947.tb02374.x%26rft_id%253Dinfo%253Apmid%252F18897933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1947.tb02374.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18897933&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hemmat</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weeden</span>  <span class="cit-name-given-names">N. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manganaris</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawson</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Molecular marker linkage map for apple.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">4</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DHemmat%26rft.auinit1%253DM.%26rft.volume%253D85%26rft.issue%253D1%26rft.spage%253D4%26rft.epage%253D11%26rft.atitle%253DMolecular%2BMarker%2BLinkage%2BMap%2Bfor%2BApple%26rft_id%253Dinfo%253Apmid%252F7907101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjY6Ijg1LzEvNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMy80MjcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kosambi</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1944</span> <span class="cit-article-title">The estimation of map distances from recombination values.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">276</span>–<span class="cit-lpage">278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D12%26rft.spage%253D276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.12" data-doi="10.1111/j.1439-0523.2009.01623.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kunkeaw</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tangphatsornruang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Triwitayakorn</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic linkage map of cassava (<em>Manihot esculenta</em> Crantz) based on AFLP and SSR markers.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">112</span>–<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D129%26rft.spage%253D112%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0523.2009.01623.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0523.2009.01623.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Establishment of molecular linkage maps using SRAP markers in sweet potato.</span> <abbr class="cit-jnl-abbrev">Acta Agron. Sin.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">1286</span>–<span class="cit-lpage">1295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BAgron.%2BSin.%26rft.volume%253D36%26rft.spage%253D1286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification of QTL for traits related to flowing time in maize based on four-way cross population.</span> <abbr class="cit-jnl-abbrev">J. Henan. Agri. Univ.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">231</span>–<span class="cit-lpage">240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHenan.%2BAgri.%2BUniv.%26rft.volume%253D47%26rft.spage%253D231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.15" data-doi="10.1287/opre.21.2.498"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kernighan</span>  <span class="cit-name-given-names">B. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">An effective heuristic algorithm for the traveling-salesman problem.</span> <abbr class="cit-jnl-abbrev">Oper. Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">498</span>–<span class="cit-lpage">516</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOperations%2BResearch%26rft.stitle%253DOperations%2BResearch%26rft.aulast%253DLin%26rft.auinit1%253DS.%26rft.volume%253D21%26rft.issue%253D2%26rft.spage%253D498%26rft.epage%253D516%26rft.atitle%253DAn%2BEffective%2BHeuristic%2BAlgorithm%2Bfor%2Bthe%2BTraveling-Salesman%2BProblem%26rft_id%253Dinfo%253Adoi%252F10.1287%252Fopre.21.2.498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1287/opre.21.2.498&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1973P739300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.16" data-doi="10.1371/journal.pone.0098855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Construction and analysis of high-density linkage map using high-throughput sequencing data.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">e98855</span>.<span class="cit-pub-id-scheme">doi:</span><span class="cit-pub-id cit-pub-id-doi">10.1371/journal.pone.0098855</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D9%26rft.spage%253De98855%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0098855%26rft_id%253Dinfo%253Apmid%252F24905985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0098855&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24905985&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.17" data-doi="10.3724/SP.J.1006.2010.00177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aitken</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Construction of molecular genetic linkage map of sugarcane based on SSR and AFLP markers.</span> <abbr class="cit-jnl-abbrev">Acta Agron. Sin.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">177</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DActa%2BAgron.%2BSin.%26rft.volume%253D36%26rft.spage%253D177%26rft_id%253Dinfo%253Adoi%252F10.3724%252FSP.J.1006.2010.00177%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3724/SP.J.1006.2010.00177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.18" data-doi="10.1017/S0016672397003005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maliepaard</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jansen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Linkage analysis in a full-sib family of an outbreeding plant species: overview and consequences for applications.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">70</span>: <span class="cit-fpage">237</span>–<span class="cit-lpage">250</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D70%26rft.spage%253D237%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672397003005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672397003005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morton</span>  <span class="cit-name-given-names">N. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1955</span> <span class="cit-article-title">Sequential tests for the detection of linkage.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">318</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DMorton%26rft.auinit1%253DN.%2BE.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D277%26rft.epage%253D318%26rft.atitle%253DSequential%2Btests%2Bfor%2Bthe%2Bdetection%2Bof%2Blinkage.%26rft_id%253Dinfo%253Apmid%252F13258560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=13258560&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1955WA21200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nordheim</span>  <span class="cit-name-given-names">E. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Malley</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guries</span>  <span class="cit-name-given-names">R. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Estimation of recombination frequency in genetic linkage studies.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D66%26rft.spage%253D313%26rft_id%253Dinfo%253Apmid%252F24263933%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24263933&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ott</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">Estimation of the recombination fraction in human pedigrees: efficient computation of the likelihood for human linkage studies.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">588</span>–<span class="cit-lpage">597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DOtt%26rft.auinit1%253DJ.%26rft.volume%253D26%26rft.issue%253D5%26rft.spage%253D588%26rft.epage%253D597%26rft.atitle%253DEstimation%2Bof%2Bthe%2Brecombination%2Bfraction%2Bin%2Bhuman%2Bpedigrees%253A%2Befficient%2Bcomputation%2Bof%2Bthe%2Blikelihood%2Bfor%2Bhuman%2Blinkage%2Bstudies.%26rft_id%253Dinfo%253Apmid%252F4422075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=4422075&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1974U024100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gebhardt</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salamini</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Estimation of recombination frequencies and construction of RFLP linkage maps in plants from crosses between heterozygous parents.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">654</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRitter%26rft.auinit1%253DE.%26rft.volume%253D125%26rft.issue%253D3%26rft.spage%253D645%26rft.epage%253D654%26rft.atitle%253DEstimation%2Bof%2Brecombination%2Bfrequencies%2Band%2Bconstruction%2Bof%2BRFLP%2Blinkage%2Bmaps%2Bin%2Bplants%2Bfrom%2Bcrosses%2Bbetween%2Bheterozygous%2Bparents.%26rft_id%253Dinfo%253Apmid%252F1974227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTI1LzMvNjQ1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.23" data-doi="10.1017/S0016672300033474"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salamini</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The calculation of recombination frequencies in crosses of allogamous plant species with applications to linkage mapping.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D67%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300033474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300033474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Säll</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nilsson</span>  <span class="cit-name-given-names">N. O.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">The robustness of recombination frequency estimates in intercrosses with dominant markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">589</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSall%26rft.auinit1%253DT.%26rft.volume%253D137%26rft.issue%253D2%26rft.spage%253D589%26rft.epage%253D596%26rft.atitle%253DThe%2Brobustness%2Bof%2Brecombination%2Bfrequency%2Bestimates%2Bin%2Bintercrosses%2Bwith%2Bdominant%2Bmarkers.%26rft_id%253Dinfo%253Apmid%252F8070668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM3LzIvNTg5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8zLzQyNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span> <span class="cit-article-title">Some comments on the statistical methods used in linkage investigations.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DSmith%26rft.auinit1%253DC.%2BA.%26rft.volume%253D11%26rft.spage%253D289%26rft.epage%253D304%26rft.atitle%253DSome%2Bcomments%2Bon%2Bthe%2Bstatistical%2Bmethods%2Bused%2Bin%2Blinkage%2Binvestigations.%26rft_id%253Dinfo%253Apmid%252F13831904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=13831904&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1959WF07600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.26" data-doi="10.1111/j.1365-313X.1993.00739.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stam</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Construction of integrated genetic linage maps by means of a new computer package: JoinMap.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">739</span>–<span class="cit-lpage">744</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D3%26rft.spage%253D739%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.1993.00739.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.1993.00739.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LC75800012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.27" data-doi="10.1017/S0016672312000353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Estimation of recombination frequency in biparental genetic populations.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">163</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DSun%26rft.auinit1%253DZ.%26rft.volume%253D94%26rft.issue%253D3%26rft.spage%253D163%26rft.epage%253D177%26rft.atitle%253DEstimation%2Bof%2Brecombination%2Bfrequency%2Bin%2Bbi-parental%2Bgenetic%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672312000353%26rft_id%253Dinfo%253Apmid%252F22892204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672312000353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22892204&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prince</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Vicente</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonierbale</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">High density molecular linkage maps of the tomato and potato genomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">1141</span>–<span class="cit-lpage">1160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTanksley%26rft.auinit1%253DS.%2BD.%26rft.volume%253D132%26rft.issue%253D4%26rft.spage%253D1141%26rft.epage%253D1160%26rft.atitle%253DHigh%2Bdensity%2Bmolecular%2Blinkage%2Bmaps%2Bof%2Bthe%2Btomato%2Band%2Bpotato%2Bgenomes.%26rft_id%253Dinfo%253Apmid%252F1360934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzMi80LzExNDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-5.3.427.29" data-doi="10.1534/genetics.106.055871"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Os</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andrzejewski</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bakker</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrena</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bryan</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Construction of a 10,000-marker ultradense genetic recombination map of potato: providing a framework for accelerated gene isolation and a genomewide physical map.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1075</span>–<span class="cit-lpage">1087</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.055871%26rft_id%253Dinfo%253Apmid%252F16582432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8yLzEwNzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzMvNDI3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-5.3.427.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-source">JoinMap 4.0: Software for the Calculation of Genetic Linkage Maps in Experimental Populations</span>, <span class="cit-publ-name">Kyazma B.V.</span>, <span class="cit-publ-loc">Wageningen, Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.31" data-doi="10.1017/S0016672311000279"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Ooijen</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multipoint maximum likelihood mapping in a full-sib family of an outbreeding species.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">349</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DVAN%2BOoijen%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D93%26rft.issue%253D5%26rft.spage%253D343%26rft.epage%253D349%26rft.atitle%253DMultipoint%2Bmaximum%2Blikelihood%2Bmapping%2Bin%2Ba%2Bfull-sib%2Bfamily%2Bof%2Ban%2Boutbreeding%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672311000279%26rft_id%253Dinfo%253Apmid%252F21878144%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672311000279&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21878144&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.32" data-doi="10.2135/cropsci2003.1764"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Ginkel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Podlich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trethowan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Comparison of two breeding strategies by computer simulation.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1764</span>–<span class="cit-lpage">1773</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D1764%26rft.epage%253D1773%26rft.atitle%253DComparison%2Bof%2BTwo%2BBreeding%2BStrategies%2Bby%2BComputer%2BSimulation%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2003.1764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2003.1764&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000185174300026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.33" data-doi="10.2135/cropsci2004.2006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Ginkel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trethowan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delacy</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Simulating the effects of dominance and epistasis on selection response in the CIMMYT Wheat Breeding Program using QuCim.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">2006</span>–<span class="cit-lpage">2018</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DWang%26rft.auinit1%253DJ.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D2006%26rft.epage%253D2018%26rft.atitle%253DSimulating%2Bthe%2BEffects%2Bof%2BDominance%2Band%2BEpistasis%2Bon%2BSelection%2BResponse%2Bin%2Bthe%2BCIMMYT%2BWheat%2BBreeding%2BProgram%2BUsing%2BQuCim%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2004.2006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2004.2006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000225153000022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.34" data-doi="10.1006/tpbi.2002.1577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Painter</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span><span class="cit-comment">a</span> <span class="cit-article-title">Simultaneous maximum likelihood estimation of linkage and linkage phases in outcrossing species.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWu%26rft.auinit1%253DR.%26rft.volume%253D61%26rft.issue%253D3%26rft.spage%253D349%26rft.epage%253D363%26rft.atitle%253DSimultaneous%2Bmaximum%2Blikelihood%2Bestimation%2Bof%2Blinkage%2Band%2Blinkage%2Bphases%2Bin%2Boutcrossing%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.2002.1577%26rft_id%253Dinfo%253Apmid%252F12027621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.2002.1577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12027621&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176540100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.35" data-doi="10.1017/S0016672301005389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span><span class="cit-comment">b</span> <span class="cit-article-title">Linkage mapping of sex-specific differences.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWu%26rft.auinit1%253DR.%26rft.volume%253D79%26rft.issue%253D1%26rft.spage%253D85%26rft.epage%253D96%26rft.atitle%253DLinkage%2Bmapping%2Bof%2Bsex-specific%2Bdifferences.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672301005389%26rft_id%253Dinfo%253Apmid%252F11974606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672301005389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11974606&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174721100008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoda</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imai</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saito</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic linkage maps constructed by using interspecific Cross between Japanese and European pears.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DYamamoto%26rft.auinit1%253DT.%26rft.volume%253D106%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D18%26rft.atitle%253DGenetic%2Blinkage%2Bmaps%2Bconstructed%2Bby%2Busing%2Ban%2Binterspecific%2Bcross%2Bbetween%2BJapanese%2Band%2BEuropean%2Bpears.%26rft_id%253Dinfo%253Apmid%252F12582866%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12582866&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000180630000002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.37" data-doi="10.1007/s00122-010-1372-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Effects of missing marker and segregation distortion on QTL mapping in F<sub>2</sub> populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">1071</span>–<span class="cit-lpage">1082</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DL.%26rft.volume%253D121%26rft.issue%253D6%26rft.spage%253D1071%26rft.epage%253D1082%26rft.atitle%253DEffects%2Bof%2Bmissing%2Bmarker%2Band%2Bsegregation%2Bdistortion%2Bon%2BQTL%2Bmapping%2Bin%2BF2%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1372-z%26rft_id%253Dinfo%253Apmid%252F20535442%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1372-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20535442&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F3%2F427.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">QTL Mapping with background control in genetic populations of clonal F<sub>1</sub> and double cross.</span> <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> (<span class="cit-comment">under review</span>).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.3.427.39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhuge</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">RAPD linkage mapping in a <em>Populus deltoides</em>×<em>Populus euramericana</em> F<sub>1</sub> family.</span> <abbr class="cit-jnl-abbrev">Hereditas (Beijing)</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">213</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%2B%2528Beijing%2529%26rft.volume%253D22%26rft.spage%253D209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/3/427.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/3/417" title="A Genome-Wide Association Study for Nutritional Indices in Drosophila" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/3/441" title="Evolutionary Consequences of DNA Methylation on the GC Content in Vertebrate Genomes" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/3"><span>Volume 5 Issue 3, March 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/3.cover-source.jpg?itok=g434JgGA" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/3/427" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-43TaoDwVcHDUn3IPa27t1gYnWKz5r51kekALLfd2O20" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/3/427" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/3/427" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/3/427" class="active" data-icon-position="" data-hide-link-title="0">Linkage Analysis and Map Construction in Genetic Populations of Clonal F1 and Double Cross</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459784" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-dTv2v1vWF37y3veSKmcSVOHdptBf_A21sVY3PfWz9dY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/3/427.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/3/427" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-I3EE3iyajqLcnpNnRvLsdaj9iEgWDGUXdcVIfs_VV28" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/3/427" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/3/427.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459784" id="node459784--2" data-pisa="ggg;5/3/427" data-pisa-master="ggg;g3.114.016022" data-apath="/ggg/5/3/427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Analysis and Map Construction in Genetic Populations of Clonal F<sub>1</sub> and Double Cross</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Luyan Zhang</span>, <span class="highwire-citation-author" data-delta="1">Huihui Li</span> and <span class="highwire-citation-author" data-delta="2">Jiankang Wang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">427-439;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016022 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459784--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Luyan Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The National Key Facility for Crop Gene Resources and Genetic Improvement, Institute of Crop Science, and CIMMYT China Office, Chinese Academy of Agricultural Sciences, Beijing 100081, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luyan%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuyan%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Huihui Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The National Key Facility for Crop Gene Resources and Genetic Improvement, Institute of Crop Science, and CIMMYT China Office, Chinese Academy of Agricultural Sciences, Beijing 100081, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huihui%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHuihui%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jiankang Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The National Key Facility for Crop Gene Resources and Genetic Improvement, Institute of Crop Science, and CIMMYT China Office, Chinese Academy of Agricultural Sciences, Beijing 100081, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiankang%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJiankang%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wangjiankang@caas.cn" class="" data-icon-position="" data-hide-link-title="0">wangjiankang@caas.cn</a>
<a href="mailto:jkwang@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">jkwang@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/3/427" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/3/427&amp;title=Linkage%20Analysis%20and%20Map%20Construction%20in%20Genetic%20Populations%20of%20Clonal%20F1%20and%20Double%20Cross" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/3/427&amp;title=Linkage%20Analysis%20and%20Map%20Construction%20in%20Genetic%20Populations%20of%20Clonal%20F1%20and%20Double%20Cross" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/3/427&amp;title=Linkage%20Analysis%20and%20Map%20Construction%20in%20Genetic%20Populations%20of%20Clonal%20F1%20and%20Double%20Cross" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/3/427&amp;text=Linkage%20Analysis%20and%20Map%20Construction%20in%20Genetic%20Populations%20of%20Clonal%20F1%20and%20Double%20Cross" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/3/427&amp;title=Linkage%20Analysis%20and%20Map%20Construction%20in%20Genetic%20Populations%20of%20Clonal%20F1%20and%20Double%20Cross" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/3/427&amp;t=Linkage%20Analysis%20and%20Map%20Construction%20in%20Genetic%20Populations%20of%20Clonal%20F1%20and%20Double%20Cross" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/3/427&amp;title=Linkage%20Analysis%20and%20Map%20Construction%20in%20Genetic%20Populations%20of%20Clonal%20F1%20and%20Double%20Cross" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/3/427&amp;title=Linkage%20Analysis%20and%20Map%20Construction%20in%20Genetic%20Populations%20of%20Clonal%20F1%20and%20Double%20Cross" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459784/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459784" id="node459784--3" data-pisa="ggg;5/3/427" data-pisa-master="ggg;g3.114.016022" data-apath="/ggg/5/3/427.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Linkage Analysis and Map Construction in Genetic Populations of Clonal F<sub>1</sub> and Double Cross</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Luyan Zhang</span>, <span class="highwire-citation-author" data-delta="1">Huihui Li</span> and <span class="highwire-citation-author" data-delta="2">Jiankang Wang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">427-439;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016022 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459784--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Luyan Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The National Key Facility for Crop Gene Resources and Genetic Improvement, Institute of Crop Science, and CIMMYT China Office, Chinese Academy of Agricultural Sciences, Beijing 100081, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luyan%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuyan%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Huihui Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The National Key Facility for Crop Gene Resources and Genetic Improvement, Institute of Crop Science, and CIMMYT China Office, Chinese Academy of Agricultural Sciences, Beijing 100081, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Huihui%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHuihui%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jiankang Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>The National Key Facility for Crop Gene Resources and Genetic Improvement, Institute of Crop Science, and CIMMYT China Office, Chinese Academy of Agricultural Sciences, Beijing 100081, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiankang%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJiankang%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:wangjiankang@caas.cn" class="" data-icon-position="" data-hide-link-title="0">wangjiankang@caas.cn</a>
<a href="mailto:jkwang@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">jkwang@cgiar.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=427&amp;rft.epage=439&amp;rft.atitle=Linkage%20Analysis%20and%20Map%20Construction%20in%20Genetic%20Populations%20of%20Clonal%20F1%20and%20Double%20Cross&amp;rft.volume=5&amp;rft.issue=3&amp;rft.date=2015-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zhang%2C+Luyan&amp;rft.au=Li%2C+Huihui&amp;rft.au=Wang%2C+Jiankang"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459784/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459784/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459784/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459784/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459784/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459784/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459784/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459784/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459784/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459784/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459784/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459784/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/3/427" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/3/427.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/3/427.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/3/427.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/3/427.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
