<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/2/241" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/2/241.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/2/241.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/2/241.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/2/241.atom" />
<meta name="HW.pisa" content="ggg;5/2/241" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A High-Density Linkage Map for Astyanax mexicanus Using Genotyping-by-Sequencing Technology" />
<meta name="DC.Identifier" content="10.1534/g3.114.015438" />
<meta name="DC.Date" content="2015-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Carlson et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The Mexican tetra, Astyanax mexicanus , is a unique model system consisting of cave-adapted and surface-dwelling morphotypes that diverged &gt;1 million years (My) ago. This remarkable natural experiment has enabled powerful genetic analyses of cave adaptation. Here, we describe the application of next-generation sequencing technology to the creation of a high-density linkage map. Our map comprises more than 2200 markers populating 25 linkage groups constructed from genotypic data generated from a single genotyping-by-sequencing project. We leveraged emergent genomic and transcriptomic resources to anchor hundreds of anonymous Astyanax markers to the genome of the zebrafish ( Danio rerio ), the most closely related model organism to our study species. This facilitated the identification of 784 distinct connections between our linkage map and the Danio rerio genome, highlighting several regions of conserved genomic architecture between the two species despite ∼150 My of divergence. Using a Mendelian cave-associated trait as a proof-of-principle, we successfully recovered the genomic position of the albinism locus near the gene Oca2 . Further, our map successfully informed the positions of unplaced Astyanax genomic scaffolds within particular linkage groups. This ability to identify the relative location, orientation, and linear order of unaligned genomic scaffolds will facilitate ongoing efforts to improve on the current early draft and assemble future versions of the Astyanax physical genome. Moreover, this improved linkage map will enable higher-resolution genetic analyses and catalyze the discovery of the genetic basis for cave-associated phenotypes." />
<meta name="DC.Contributor" content="Brian M. Carlson" />
<meta name="DC.Contributor" content="Samuel W. Onusko" />
<meta name="DC.Contributor" content="Joshua B. Gross" />
<meta name="article:published_time" content="2015-02-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A High-Density Linkage Map for Astyanax mexicanus Using Genotyping-by-Sequencing Technology" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The Mexican tetra, &lt;i&gt;Astyanax mexicanus&lt;/i&gt;, is a unique model system consisting of cave-adapted and surface-dwelling morphotypes that diverged &amp;gt;1 million years (My) ago. This remarkable natural experiment has enabled powerful genetic analyses of cave adaptation. Here, we describe the application of next-generation sequencing technology to the creation of a high-density linkage map. Our map comprises more than 2200 markers populating 25 linkage groups constructed from genotypic data generated from a single genotyping-by-sequencing project. We leveraged emergent genomic and transcriptomic resources to anchor hundreds of anonymous &lt;i&gt;Astyanax&lt;/i&gt; markers to the genome of the zebrafish (&lt;i&gt;Danio rerio&lt;/i&gt;), the most closely related model organism to our study species. This facilitated the identification of 784 distinct connections between our linkage map and the &lt;i&gt;Danio rerio&lt;/i&gt; genome, highlighting several regions of conserved genomic architecture between the two species despite ∼150 My of divergence. Using a Mendelian cave-associated trait as a proof-of-principle, we successfully recovered the genomic position of the albinism locus near the gene &lt;i&gt;Oca2&lt;/i&gt;. Further, our map successfully informed the positions of unplaced &lt;i&gt;Astyanax&lt;/i&gt; genomic scaffolds within particular linkage groups. This ability to identify the relative location, orientation, and linear order of unaligned genomic scaffolds will facilitate ongoing efforts to improve on the current early draft and assemble future versions of the &lt;i&gt;Astyanax&lt;/i&gt; physical genome. Moreover, this improved linkage map will enable higher-resolution genetic analyses and catalyze the discovery of the genetic basis for cave-associated phenotypes.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/02/01" />
<meta name="citation_mjid" content="ggg;5/2/241" />
<meta name="citation_id" content="5/2/241" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/2/241" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/2/241.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/2/241.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/2/241.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.015438" />
<meta name="citation_pmid" content="25520037" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="241" />
<meta name="citation_lastpage" content="251" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Brian M. Carlson" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Cincinnati, Cincinnati, Ohio 45221" />
<meta name="citation_author" content="Samuel W. Onusko" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Cincinnati, Cincinnati, Ohio 45221" />
<meta name="citation_author" content="Joshua B. Gross" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Cincinnati, Cincinnati, Ohio 45221" />
<meta name="citation_author_email" content="grossja@ucmail.uc.edu" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. Borowsky;citation_author=H. Wilkens;citation_title=Mapping a Cave Fish Genome: Polygenic Systems and Regressive Evolution;citation_pages=19-21;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011170;citation_doi=10.1093/jhered/93.1.19" />
<meta name="citation_reference" content="citation_journal_title=Copeia;citation_author=C. M. Breder;citation_title=Apparent changes in phenotypic ratios of the Characins at the type locality of Anoptichthys jordani Hubbs and Innes.;citation_pages=26-30;citation_volume=1943;citation_year=1943;citation_doi=10.2307/1437876" />
<meta name="citation_reference" content="citation_journal_title=Trans. N. Y. Acad. Sci.;citation_author=C. M. Breder;citation_title=Problems in the behavior and evolution of a species of blind cave fish.;citation_pages=168-176;citation_volume=5;citation_year=1943;citation_doi=10.1111/j.2164-0947.1943.tb00880.x" />
<meta name="citation_reference" content="citation_journal_title=J. Biogeogr.;citation_author=J. Briggs;citation_title=The biogeography of otophysan fishes (Ostariophysi: Otophysi): A new appraisal.;citation_pages=287-294;citation_volume=32;citation_year=2005;citation_doi=10.1111/j.1365-2699.2004.01170.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. W. Broman;citation_author=H. Wu;citation_author=S. Sen;citation_author=G. A. Churchill;citation_title=R/qtl: QTL mapping in experimental crosses;citation_pages=889-890;citation_volume=19;citation_year=2003;citation_issue=7;citation_pmid=12724300;citation_doi=10.1093/bioinformatics/btg112" />
<meta name="citation_reference" content="citation_journal_title=Behav. Processes;citation_author=H. Burchards;citation_author=A. Dölle;citation_author=J. Parzefall;citation_title=Aggressive behaviour of an epigean population of Astyanax mexicanus (Characidae, Pisces) and some observations of three subterranean populations.;citation_pages=225-235;citation_volume=11;citation_year=1985;citation_pmid=24896447;citation_doi=10.1016/0376-6357(85)90017-8" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. W. Davey;citation_author=P. A. Hohenlohe;citation_author=P. D. Etter;citation_author=J. Q. Boone;citation_author=J. M. Catchen;citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing.;citation_pages=499-510;citation_volume=12;citation_year=2011;citation_issue=7;citation_pmid=21681211;citation_doi=10.1038/nrg3012" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=E. R. Duboué;citation_author=A. C. Keene;citation_author=R. L. Borowsky;citation_title=Evolutionary convergence on sleep loss in cavefish populations.;citation_pages=671-676;citation_volume=21;citation_year=2011;citation_issue=8;citation_pmid=21474315;citation_doi=10.1016/j.cub.2011.03.020" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=Y. Elipot;citation_author=H. Hinaux;citation_author=J. Callebert;citation_author=S. Retaux;citation_title=Evolutionary shift from fighting to foraging in blind cavefish through changes in the serotonin network.;citation_pages=1-10;citation_volume=23;citation_year=2013;citation_pmid=23159600;citation_doi=10.1016/j.cub.2012.10.044" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. V. Everett;citation_author=M. R. Miller;citation_author=J. E. Seeb;citation_title=Meiotic maps of sockeye salmon derived from massively parallel DNA sequencing.;citation_pages=521-521;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23031582;citation_doi=10.1186/1471-2164-13-521" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. B. Gross;citation_author=M. Protas;citation_author=M. Conrad;citation_author=P. E. Scheid;citation_author=O. Vidal;citation_title=Synteny and candidate gene prediction using an anchored linkage map of Astyanax mexicanus.;citation_volume=105;citation_year=2008;citation_pmid=19104060;citation_doi=10.1073/pnas.0806238105" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. B. Gross;citation_author=R. Borowsky;citation_author=C. J. Tabin;citation_title=A novel role for Mc1r in the parallel evolution of depigmentation in independent populations of the cavefish Astyanax mexicanus.;citation_pages=e1000326-e1000326;citation_volume=5;citation_year=2009;citation_issue=1;citation_pmid=19119422;citation_doi=10.1371/journal.pgen.1000326" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=J. B. Gross;citation_author=A. Furterer;citation_author=B. M. Carlson;citation_author=B. A. Stahl;citation_title=An integrated transcriptome-wide analysis of cave and surface dwelling Astyanax mexicanus;citation_pages=e55659;citation_volume=8;citation_year=2013;citation_issue=2;citation_pmid=23405189;citation_doi=10.1371/journal.pone.0055659" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. B. Gross;citation_author=A. J. Krutzler;citation_author=B. M. Carlson;citation_title=Complex craniofacial changes in blind cave-dwelling fish are mediated by genetically symmetric and asymmetric loci.;citation_volume=196;citation_year=2014;citation_pmid=24496009;citation_doi=10.1534/genetics.114.161661" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=B. C. Hecht;citation_author=N. R. Campbell;citation_author=D. E. Holecek;citation_author=S. R. Narum;citation_title=Genome-wide association reveals genetic basis for the propensity to migrate in wild populations of rainbow and steelhead trout.;citation_pages=3061-3076;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12082" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. D. Houston;citation_author=J. W. Davey;citation_author=S. C. Bishop;citation_author=N. R. Lowe;citation_author=J. C. Mota-Velasco;citation_title=Characterisation of QTL-linked and genome-wide restriction site-associated DNA (RAD) markers in farmed Atlantic salmon.;citation_pages=244-244;citation_volume=13;citation_year=2012;citation_pmid=22702806;citation_doi=10.1186/1471-2164-13-244" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=W. R. Jeffery;citation_title=Cavefish as a model system in evolutionary developmental biology.;citation_pages=1-12;citation_volume=231;citation_year=2001;citation_issue=1;citation_pmid=11180948;citation_doi=10.1006/dbio.2000.0121" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=W. R. Jeffery;citation_title=Regressive evolution in Astyanax cavefish.;citation_pages=25-47;citation_volume=43;citation_year=2009;citation_pmid=19640230;citation_doi=10.1146/annurev-genet-102108-134216" />
<meta name="citation_reference" content="citation_journal_title=Copeia;citation_author=R. F. Kirby;citation_author=K. W. Thompson;citation_author=C. Hubbs;citation_title=Karyotypic similarities between the Mexican and blind tetras.;citation_pages=578-580;citation_volume=1977;citation_year=1977;citation_doi=10.2307/1443283" />
<meta name="citation_reference" content="citation_journal_title=Ann. Eugen.;citation_author=D. D. Kosambi;citation_title=The estimation of map distances from recombination values.;citation_pages=172-175;citation_volume=12;citation_year=1943;citation_doi=10.1111/j.1469-1809.1943.tb02321.x" />
<meta name="citation_reference" content="citation_journal_title=Curr. Biol.;citation_author=J. E. Kowalko;citation_author=N. Rohner;citation_author=S. B. Rompani;citation_author=B. K. Peterson;citation_author=T. A. Linden;citation_title=Loss of schooling behavior in cavefish through sight-dependent and sight-independent mechanisms.;citation_pages=1874-1883;citation_volume=23;citation_year=2013;citation_pmid=24035545;citation_doi=10.1016/j.cub.2013.07.056" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Krzywinski;citation_author=J. Schein;citation_author=I. Birol;citation_author=J. Connors;citation_author=R. Gascoyne;citation_title=Circos: An information aesthetic for comparative genomics.;citation_volume=19;citation_year=2009;citation_pmid=19541911;citation_doi=10.1101/gr.092759.109" />
<meta name="citation_reference" content="citation_journal_title=Evol. Appl.;citation_author=W. A. Larson;citation_author=L. W. Seeb;citation_author=M. V. Everett;citation_author=R. K. Waples;citation_author=W. D. Templin;citation_title=Genotyping by sequencing resolves shallow population structure to inform conservation of Chinook salmon (Oncorhynchus tshawytscha).;citation_pages=355-369;citation_volume=7;citation_year=2014;citation_pmid=24665338;citation_doi=10.1111/eva.12128" />
<meta name="citation_reference" content="citation_journal_title=Comp. Biochem. Physiol. D;citation_author=Z. Liu;citation_title=Development of genomic resources in support of sequencing, assembly, and annotation of the catfish genome.;citation_pages=11-17;citation_volume=6;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=S. Lorenz;citation_author=S. Brenna-Hansen;citation_author=T. Moen;citation_author=A. Roseth;citation_author=W. S. Davidson;citation_title=BAC-based upgrading and physical integration of a genetic SNP map in Atlantic salmon.;citation_pages=48-54;citation_volume=41;citation_year=2010;citation_issue=1;citation_pmid=19917045;citation_doi=10.1111/j.1365-2052.2009.01963.x" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=F. Lu;citation_author=A. E. Lipka;citation_author=J. Glaubitz;citation_author=R. Elshire;citation_author=J. H. Cherney;citation_title=Switchgrass genomic diversity, ploidy, and evolution: Novel insights from a network-based SNP discovery protocol.;citation_pages=e1003215;citation_volume=9;citation_year=2013;citation_pmid=23349638;citation_doi=10.1371/journal.pgen.1003215" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=S. E. McGaugh;citation_author=J. B. Gross;citation_author=B. Aken;citation_author=M. Blin;citation_author=R. Borowsky;citation_title=The cavefish genome reveals candidate genes for eye loss.;citation_pages=5307;citation_volume=5;citation_year=2014;citation_pmid=25329095;citation_doi=10.1038/ncomms6307" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=P. Ninwichian;citation_author=E. Peatman;citation_author=H. Liu;citation_author=H. Kucuktas;citation_author=B. Somridhivej;citation_title=Second-Generation Genetic Linkage Map of Catfish and Its Integration with the BAC-Based Physical Map;citation_pages=1233-1241;citation_volume=2;citation_year=2012;citation_issue=10;citation_pmid=23050234;citation_doi=10.1534/g3.112.003962" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=K. E. O’Quin;citation_author=M. Yoshizawa;citation_author=P. Doshi;citation_author=W. R. Jeffery;citation_title=Quantitative genetic analysis of retinal degeneration in the blind cavefish Astyanax mexicanus.;citation_pages=e57281;citation_volume=8;citation_year=2013;citation_pmid=23437360;citation_doi=10.1371/journal.pone.0057281" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol.;citation_author=R. Ogden;citation_author=K. Gharbi;citation_author=N. Mugue;citation_author=J. Martinsohn;citation_author=H. Senn;citation_title=Sturgeon conservation genomics: SNP discovery and validation using RAD sequencing.;citation_pages=3112-3123;citation_volume=22;citation_year=2013;citation_doi=10.1111/mec.12234" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=Y. Palti;citation_author=C. Genet;citation_author=M. C. Luo;citation_author=A. Charlet;citation_author=G. Gao;citation_title=A first generation integrated map of the rainbow trout genome.;citation_pages=180-180;citation_volume=12;citation_year=2011;citation_pmid=21473775;citation_doi=10.1186/1471-2164-12-180" />
<meta name="citation_reference" content="citation_journal_title=Marine biotechnology (New York, N.Y.);citation_journal_abbrev=Mar Biotechnol (NY);citation_author=Y. Palti;citation_author=C. Genet;citation_author=G. Gao;citation_author=Y. Hu;citation_author=F. M. You;citation_title=A second generation integrated map of the rainbow trout (Oncorhynchus mykiss) genome: analysis of conserved synteny with model fish genomes.;citation_pages=343-357;citation_volume=14;citation_year=2012;citation_issue=3;citation_pmid=22101344;citation_doi=10.1007/s10126-011-9418-z" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=K. Pottin;citation_author=H. Hinaux;citation_author=S. Rétaux;citation_title=Restoring eye size in Astyanax mexicanus blind cavefish embryos through modulation of the Shh and Fgf8 forebrain organising centres;citation_pages=2467-2476;citation_volume=138;citation_year=2011;citation_issue=12;citation_pmid=21610028;citation_doi=10.1242/dev.054106" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. E. Protas;citation_author=C. Hersey;citation_author=D. Kochanek;citation_author=Y. Zhou;citation_author=H. Wilkens;citation_title=Genetic analysis of cavefish reveals molecular convergence in the evolution of albinism.;citation_pages=107-111;citation_volume=38;citation_year=2006;citation_issue=1;citation_pmid=16341223;citation_doi=10.1038/nm1334" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=M. Protas;citation_author=M. Conrad;citation_author=J. B. Gross;citation_author=C. Tabin;citation_author=R. Borowsky;citation_title=Regressive evolution in the Mexican cave tetra, Astyanax mexicanus.;citation_pages=452-454;citation_volume=17;citation_year=2007;citation_issue=5;citation_pmid=17306543;citation_doi=10.1016/j.cub.2007.01.051" />
<meta name="citation_reference" content="citation_journal_title=Evolution &amp; development;citation_journal_abbrev=Evol Dev;citation_author=M. Protas;citation_author=I. Tabansky;citation_author=M. Conrad;citation_author=J. B. Gross;citation_author=O. Vidal;citation_title=Multi-trait evolution in a cave fish, Astyanax mexicanus.;citation_pages=196-209;citation_volume=10;citation_year=2008;citation_issue=2;citation_pmid=18315813;citation_doi=10.1111/j.1525-142X.2008.00227.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=H. C. Rowe;citation_author=S. Renaut;citation_author=A. Guggisberg;citation_title=RAD in the realm of next-generation sequencing technologies.;citation_pages=3499-3502;citation_volume=20;citation_year=2011;citation_issue=17;citation_pmid=21991593" />
<meta name="citation_reference" content="citation_journal_title=Copeia;citation_author=P. Şadoǧlu;citation_title=A preliminary report on the genetics of the Mexican cave characins.;citation_pages=113-114;citation_volume=1956;citation_year=1956;citation_doi=10.2307/1440425" />
<meta name="citation_reference" content="citation_journal_title=Comparative Biochemistry and Physiology. Part A, Molecular &amp; Integrative Physiology;citation_journal_abbrev=Comparative Biochemistry and Physiology. Part A, Molecular &amp; Integrative Physiology;citation_author=K. Salin;citation_author=Y. Voituron;citation_author=J. Mourin;citation_author=F. Hervant;citation_title=Cave colonization without fasting capacities: an example with the fish Astyanax fasciatus mexicanus.;citation_pages=451-457;citation_volume=156;citation_year=2010;citation_issue=4;citation_pmid=20382251;citation_doi=10.1016/j.cbpa.2010.03.030" />
<meta name="citation_reference" content="citation_journal_title=Zeitschrift fÃ¼r Tierpsychologie;citation_journal_abbrev=Z Tierpsychol;citation_author=C. Schemmel;citation_title=Studies on the genetics of feeding behaviour in the cave fish Astyanax mexicanus f. anoptichthys. An example of apparent monofactorial inheritance by polygenes.;citation_pages=9-22;citation_volume=53;citation_year=1980;citation_issue=1;citation_pmid=7434991" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Y. Yamamoto;citation_author=D. W. Stock;citation_author=W. R. Jeffery;citation_title=Hedgehog signalling controls eye degeneration in blind cavefish.;citation_pages=844-847;citation_volume=431;citation_year=2004;citation_issue=7010;citation_pmid=15483612;citation_doi=10.1038/nature02864" />
<meta name="citation_reference" content="citation_journal_title=BMC Biol.;citation_author=M. Yoshizawa;citation_author=Y. Yamamoto;citation_author=K. E. O’Quin;citation_author=W. R. Jeffery;citation_title=Evolution of an adaptive behavior and its sensory receptors promotes eye regression in blind cavefish.;citation_pages=108;citation_volume=10;citation_year=2012;citation_pmid=23270452;citation_doi=10.1186/1741-7007-10-108" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A High-Density Linkage Map for Astyanax mexicanus Using Genotyping-by-Sequencing Technology" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2.cover-source.jpg" />
<meta name="twitter:description" content="The Mexican tetra, Astyanax mexicanus , is a unique model system consisting of cave-adapted and surface-dwelling morphotypes that diverged &gt;1 million years (My) ago. This remarkable natural experiment has enabled powerful genetic analyses of cave adaptation. Here, we describe the application of next-generation sequencing technology to the creation of a high-density linkage map. Our map comprises more than 2200 markers populating 25 linkage groups constructed from genotypic data generated from a single genotyping-by-sequencing project. We leveraged emergent genomic and transcriptomic resources to anchor hundreds of anonymous Astyanax markers to the genome of the zebrafish ( Danio rerio ), the most closely related model organism to our study species. This facilitated the identification of 784 distinct connections between our linkage map and the Danio rerio genome, highlighting several regions of conserved genomic architecture between the two species despite ∼150 My of divergence. Using a Mendelian cave-associated trait as a proof-of-principle, we successfully recovered the genomic position of the albinism locus near the gene Oca2 . Further, our map successfully informed the positions of unplaced Astyanax genomic scaffolds within particular linkage groups. This ability to identify the relative location, orientation, and linear order of unaligned genomic scaffolds will facilitate ongoing efforts to improve on the current early draft and assemble future versions of the Astyanax physical genome. Moreover, this improved linkage map will enable higher-resolution genetic analyses and catalyze the discovery of the genetic basis for cave-associated phenotypes." />
<meta name="og-title" property="og:title" content="A High-Density Linkage Map for Astyanax mexicanus Using Genotyping-by-Sequencing Technology" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/2/241" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The Mexican tetra, Astyanax mexicanus , is a unique model system consisting of cave-adapted and surface-dwelling morphotypes that diverged &gt;1 million years (My) ago. This remarkable natural experiment has enabled powerful genetic analyses of cave adaptation. Here, we describe the application of next-generation sequencing technology to the creation of a high-density linkage map. Our map comprises more than 2200 markers populating 25 linkage groups constructed from genotypic data generated from a single genotyping-by-sequencing project. We leveraged emergent genomic and transcriptomic resources to anchor hundreds of anonymous Astyanax markers to the genome of the zebrafish ( Danio rerio ), the most closely related model organism to our study species. This facilitated the identification of 784 distinct connections between our linkage map and the Danio rerio genome, highlighting several regions of conserved genomic architecture between the two species despite ∼150 My of divergence. Using a Mendelian cave-associated trait as a proof-of-principle, we successfully recovered the genomic position of the albinism locus near the gene Oca2 . Further, our map successfully informed the positions of unplaced Astyanax genomic scaffolds within particular linkage groups. This ability to identify the relative location, orientation, and linear order of unaligned genomic scaffolds will facilitate ongoing efforts to improve on the current early draft and assemble future versions of the Astyanax physical genome. Moreover, this improved linkage map will enable higher-resolution genetic analyses and catalyze the discovery of the genetic basis for cave-associated phenotypes." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2.cover-source.jpg" />
    <title>A High-Density Linkage Map for Astyanax mexicanus Using Genotyping-by-Sequencing Technology | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"DDbLLI750tF8OEcwsJWZ7rpi5Y4kI9-ms91--jU83Bc","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459771","apath":"\/ggg\/5\/2\/241.atom","pisa":"ggg;5\/2\/241","ac":{"\/ggg\/5\/2\/241.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/2\/241.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/2\/241"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/2\/241"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/2\/241":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ZxEQLYaxKQqhm0frfPDmBbSWTXTjc5IOO7VjsAf0rguji5Su0XwOrD26eM0L4ZMBGgXKtfbAJ47\\\\\\\/IxYx2w0GIRHQ\\\\\\\/0+G15FJJn3N7ZVl2JOBYr0IGFgnAf\\\\\\\/K38WESyoq+TWMbgk3BV4NwtI204VrGFM1p62SxlTh59N9+h4OhtvcBmYiAxosnDqnILBwvlTAJAx9Wwz986OM1xptSa+YlRYb5wugUGrJZimHZ9LmIEWff+3xMdv8F3xoAXX2p+b59\\\\\\\/lefkGq39hjzVhnepLAamPPpD9\\\\\\\/G6rjmW6HXl8g\\\\\\\/kXic\\\\\\\/vyX+RnSiwFn5d918IAn6005bxupQVfED8lKirx1huRsz3vwmeQ70Ub8coBt\\\\\\\/QfjW1nQfEWkkbvDuQovKXKynZsYMxftpj\\\\\\\/56dLZY+YIKYF1P\\\\\\\/xoMk85Dpn063EkIIiEo+KWyGxhKSBh2zCG4WiwS\\\\\\\/TFKIUxStXHKN3EvbNPAKkI\\\\\\\/80qlJ\\\\\\\/HmTUOxBwLNm8hYscS7Qyfvqcq1sdH5w5fMT8AYAfTvf4mcSVeiVUhH1\\\\\\\/JxW1dgvnfVb8r3zj+aSGYtBWsJbriYskrBw5ag3cPt2xYzje2eHJiTajNnaAgdJE6SQ4t9n79eaEY9hyeson\\\\\\\/wE5evYkOQRISzcuhs6dv12o3oouzo9hqxVcWhEnvl6POJfFSQiQe3ydUEbCFydnaBaFtejmYKM7T\\\\\\\/j\\\\\\\/hv4+O\\\\\\\/SeON6tgjmxls\\\\\\\/BW7NhUsktXV+QP3ByK6qfwwXbFVlK2i4hWbzrN5GIxri+Uc7XwXrXxA6UB3JqJEtrRYv0j5NssHL2KEjRW9qJ2cgirfrd3ve\\\\\\\/4eB5gbc4OqGtA8LFkL5VtYuPknJMCFwoGYs1yrVwalC6dxXlxsyIHXnRWrnPFOqIINkq+DD+HETjnF8cuLbTXcjwEZ730a9oOasUPEBVsN3s3hXwqQ\\\\\\\/z8\\\\\\\/VGM2j\\\\\\\/57av4oHqTsqYmffskeyrYQjS4Kwya1Lzi6ZrB6n9uk6apGQ3Fm0FpKi4sLqn4Kve9JPfHi606Cmqp6OemQr0Wj\\\\\\\/ymGJQBzbBiUXcviqih5Cm2CX5zhIaWJdapfJcEo2pyBBQpLbBDGyuJK12pfDe0bbWghektav2Sf5OEvVrbhRnHIpFhD0CrNLMnSBJahVhEopkIBRakeVfED3W1GMtg9TaYOcYPkNNVi5CuSi7u72zonG8XSki1UD3qNDLcRKE2FV389E45UGdGDFCxyd92XymlW6WLderooCDIcIs4Z0QJ\\\\\\\/nitdT3ZmmlFrmks1Z8rGlY1hHb5tkGAZNAgFZA\\\u0022,\\\u0022iv\\\u0022:\\\u0022qoOHg7Ept6rt9QuEVNXygg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e2929837a23a625b6811721a8b7b6310dc26abb77914f8ab30c6d5769ea3577f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MRWXcj8+13O2NPN7z\\\\\\\/CU1ynp3kxAc2K4J3KfnmTFh3DCU+DaF4j3zfckEVm1LzL8sAK00PvOUzf5CxG7mTyRaqaF0G9hoE1tHYoBBERxtoQbViTjU2zvZ3tHni919jZXwHygcdTtAYLcnwReLWWcEmykM579lpBsM42orHn95Svc1Sjj8FL0B05VtbfCR9VeKw8aap5bA40p4YwYmvniK+qzl1hx5AFUdJl5oUdtyDOzB5Tx7azY+qrbVIYvxKG4Jhz\\\\\\\/+j54omzsCO5\\\\\\\/kMe\\\\\\\/UaEOkWu7Vl7nDbMpKwL\\\\\\\/5iGi3bxjlgpHivc99OZfkvYfauecyMm6z8dI3Vf5egBxWbfqrBvAtHTNQuB\\\\\\\/Nifzbunh3X1Xx9NErBddy9PP3kfJ0qowbc+q58Ph6dU5P0G\\\\\\\/N49lc2cDdhON6jK64RWVU0bJUn6n1nS6aU+lfQ5uhtDnSdH+wsMkKG84F+RpxLVvTsRDInul3N0i0s2mT0cABoMC5N6W4HaYy4H0dNvNXnWM3V8O9olabqbrwsa6XC\\\\\\\/0MaBfx7xWIIOg5PV2cTFJNvx+xfb9SpHSd8Mxg81Roso3d6K97bt25PuS2Iyn5wFSl9L6wZhKl2gzrD+luvQUpuA29Ub0Hu1f5+ApUDhKu3azG0eXkKyNI+vty3eLzp0sQJP\\\\\\\/CMfgwoBRV5\\\\\\\/TP2YNTlk\\\\\\\/RzYoZn36Sp0WMeaf0fmTDfJYpM5X7JSsXtolF1zXgvljVa6vmLJncuoorNAigY7MfQeOo8Jhp\\\\\\\/Fs4PlrUliaALf0Ctp3jPymGceUae32TNDZn\\\\\\\/3Pl7b9MoEALj7xpBMlq7q8uO27mpwZT05BdQWCtiK8s5Ga+HqBzgUStooPLcgxwdYr\\\\\\\/Zt02bzq+mjO6mkAf1+vE4gquWMwsrZ13KoBsoJXQ8CR44QUkhp6CkZm5FA3Y8X5t077YM12n4+yQSnhflb69kMS9+BalvfA4TUy5DE5YMTAh1Om6gR2SoCu8+RGtG\\\\\\\/+84SnUrnxS3xIMoSlXv\\\\\\\/m8qELPfl4S\\\\\\\/OZAWEmRP0EkW4Y9YW2t6P73AtizJn+JClb3cy3Fh1sq7ZP7rAK6lquM0uWJmv3ciy\\\\\\\/RmblGBQeusjbM+MQJI8ELncbaAzrfl28HQuZFEOdGje3O\\\\\\\/8kpaUmSTgPxJqzPaxCBSf8jiILn9aEezLwHnkBsV1g1W\\\\\\\/Icg2VSpo2JXr5+UK2d5dDpjNzCbpsRaG2\\\\\\\/YUEEzM7nHZM6nBV7KVtLPH5HO3+5MiWZAzQm3NyWL7bA9+MA55g9GQkbMQq1zKAitPuGTkDMF6mqNgq\\\\\\\/XSRL4VyDS0mFQ9y\\\\\\\/AbXQDyoAf39BY0HK+ZyyMdDCnnrvVSx\\\\\\\/7kFDk4JB\\\\\\\/mW3AAEpGuKvmFSWY9E4JpQwsJ1KzhXzE6wOt6TdOexGKhLwPhQKg3URmhQix9EWtO45k97+c8x+53FSHFLaYcRJACaOFJ2\\\\\\\/AhtAkepWxWgMOiWkwjWD2NjYVKwv4Pmyj2x2XakIspetVx5JN4s1JHROSIiuzD1TF44EzMk2UYlY0LhAF8B4dp8axGsUUAU095R3+YKZg1in3213ryf\\\\\\\/FbmNqS14u8MEGzYkbf1j12+f8rHzsRpdqhcqN758Mpsh7syAYzDgA1PdP+wU4JRxIFQPdxexuXswFBcDSWCw7kzy4hm9kQVHBVj+LV87iAiTBUk7tqfrQeXdB7fbb\\\\\\\/0wnib7hu5nEwM6RBf1T\\\\\\\/lVeb2GXEZ5OPhIFm1N48cK\\\\\\\/HvW1zkKdPF1R3S2so0HpLJq+FNPGjq0Qki7d32RD0yG8N0OJsHz8pufSNq5RkklTajTX+2HwWzUAtks5flTT645i8T\\\\\\\/d\\\\\\\/ouLW+w69nCvXqOL7UPvMtZphU31AuTPBiHybRpVy3Sr6VWo2UmqJ+08FXnEyJne9ebsG5KblBKWKfVibb\\\\\\\/CcZtTJZbv1tyrKlwhFOkPeLyvyxk\\\\\\\/uZhdD3XR0NPuY1jijYl\\\\\\\/R3Qy3UPwv\\\\\\\/kS1B7mWFreachcwI+yN5Cm7RqvV7SDVDUZ7vN9JkqviG4hshevxhnjt1wnYNEt6mjyL4rQl5NcSS6Sw+i\\\\\\\/Hc0X4fLoOJfa\\\\\\\/YoqKk0jqworOhqGZsy6Gt2pmRBkhq8I6AS5nPhZWnBTFPo4wOEkzxu79NGlkQYfGEmq3rn2GqRntU5WGOPikPkvpB1u4TqNBmZkoABWojBqomrRvIR9VII00829ycFemSGpHiZim+\\\\\\\/4XFdbXRGxBXkucBksxX7QGIJWIbYLhb0u294ueV2QDKM5U9TZZ4xaYZunTxyDMjLjBFkyDEwPtgZcv3ae4FlJE1dG67fKOh8nXsWL8Q1DZRKT7qMnMdDmuUDS\\\\\\\/p0thAUtYDjDQDsqRU8f\\\\\\\/7kr9IlT8XGC6roBCHwhO7e\\\\\\\/xF2Ycxbms9Q6TneU61EoQEXDVDzU2q\\\\\\\/N1lE7mcmYX\\\\\\\/1sppresuOzjZrNPVx\\\\\\\/aXrKbAqH9W7aTvxq+odRpa3BE+UuOwN39TtqohpwUznFRsZxWvXwkPKGMQrJuTP2heOODk1IchFyGIXJcPC\\\\\\\/OSQ4NTUk+z8MRg\\\\\\\/kBUx7IOZj2xtcF0t1EKxcmq2URzQpPFvH58csPsd531fcTajQrUQDDeyiL16NtuDzRcW9Lr2tiucbVdKe\\\\\\\/nzNybz6\\\\\\\/NGMzvnTKCEWI58HjiRg\\\\\\\/0T1Xa\\\\\\\/wtWRuB7ZHWXfqEiVmfI4tCdye4GSOYvWGQc6vpt3PaD7OqOi+pBfqET0CtbcCOLIhFugE6OGgFeMskF7HIRKT13ztDMtUrVo51ktyL2vhItCVJaHqzULLsPbbxij37yeD7YoSK9MEM6fdlDWXUECSG3\\\\\\\/6KBCY2k7e3fscZSCadwp3QN\\\\\\\/6oXutZEk\\\\\\\/Bnk7vzhfYMXd3Pl0q5xD45AY6RjpRxY+cXGjonVSO7MfiltoiEceaD1yAW0\\\\\\\/iLbFBNHRZtiRJIZo06D5U1l\\\\\\\/BvdNYDSv++RGEsccSna1PPZdA2DsFyHW0CME80J02o+VHNOrLca+qQBwtragI+adRCVviFQLvZl4Dh7Jya68rO1GK1U2PPm4N4Jq\\\\\\\/ZNMNqNm2QN8cx4nv6e822tFVv77C8AxyxhdH5xrsKmiJwoH+0J2eqXpRnCD96sG1qIzdiqyrxJ+DSzDRED8laZ9g6ZB2ouidNEjKE5ijgvOfXtEuXBnFu3OKcke6ZPHEE6biDCkuKkb6vK5UzNbK81EKazErZeCQBSSMkD6SekCL51bmNx9Z+4XI2vdNyGUDE1tWf3p5pBG5QUxP65fLYHk2r1pWnfYBZJA9Blrn3W9shLI53vBMWakS1Fwi09vVgav2t1uPNtASTLSejIVWR7jhhCUDduCDGcVMvBegQjouGtzxHRKrEnRnvFFus8sq19qr7bJHi8UPYO8CuVZ0TPoplU0wCM6+veFWaZpHqwDh0zUw88RJxSvUNoeJy2ilmbpwYxM9BsgmpTa6t6tW78YBvD2ehNtTgoIWOitfZfxXSc2YNHnLYOmMrrY6+n6TUIbznnlHhbhTfPe2wnZhicGeQhx+zvH3SbnZqn0X1Qu6mSc5MHMR2dGwhGcB1D+e\\\\\\\/ugaYNT9rX7z8jqWZ6zinGuwqqoG6vmqeGQY4fDO2\\\\\\\/JsjJsn2FVAYhW5r0zkr2QVBYUUvoPi\\\\\\\/AV+nx9yuH8EcoxiRzM37qDhw3I3beBv23EsUdxgF7VrCG9msS7tgFVIk07IBGMoVzLvb\\\\\\\/zNi5x7dHTLjwxc38rJXZ0kVGona76wPPpt\\\\\\\/fvmRRGd8Fej9zVVl8TnqExtPv7iLknW9K+uFZvvfzG3fBtNEu4dQN6eiOcpWo\\\\\\\/zRALzMWhRjNhaF\\\\\\\/x7RXLfKFAF0M3SwH8BP2ORbqI75orfTunNN+aNj60xrTSGxYK4m1rMlluAwxk1vv2EbcLr6cC44\\\\\\\/5rp2ua9qKvyMo1wTH39XNSdFn\\\\\\\/oX0xr8SdwvZLXzYKd0Xq93MIo6CTrBi6vahs0iV3uTZOpt9qCgUYCNAQfNEW\\\\\\\/K6qBkmli8q2tdxrFaDpaunx01gfuKBw+pnNj40CZVjfQ+T8uFxjOXwhYkRwAgrSonAREIkhn5iowezOV6dRsk364dVyldoTRv8nVeIKRqSbV1Gf+zmmyMmNO2ORUKMCYsB0cbyc6x0Kigp54\\\\\\\/hXxwoCB8cwa8FTmpE27vI5svHEEJjtIurDJsGXGVz3fR3ujgvLBJGnAYzMRwqyfKMFEBEOJD26QEuMWEulFDuFdxi4+2F+lC0djkzl2uItZvaoBMDVzjf7tXkSWLkZxttf40uSd31gidYwTTTy7ODukhBGe7AJjXLaGNaDr\\\\\\\/6XF3EsETdoB1hBnagpcOgAOlOZoE3+1j4J\\\\\\\/gHVxBlSCY4oU4jaP3UfjoFzNwvdCHmwOsROe4+oNjrzF4356FfbAdknhya0zoK0go9cL4mURz\\\\\\\/PNdgG0dvquDKOZ8nmB0S9ailxInR6zy3rf64JOWLDPEsmujHztfdNw4V0hcXYmtrmdft6iTH65P3HIFvSVUE8nGAe8WmKJxpKuZMKqGgfJmAamtk3API6Xn32eLFKLXPrDS5x87wmxNCiTMWIiMZVyDDIsbrPpSL+ZG3UkFkywLyE1RmGWW+aMInVylMwWsGmCAEP5eopfA67BKq3V1ajzdJ2ZQxIXXPHPKIi\\\\\\\/qWPgmDgQtoPaecnkaYoQqtunIlzH53QsADZw8ALeDXqCaUhR4sGp7ZGQY542YuA8nu2BxY1lRjPT+79ty9yR1jh\\\\\\\/+JOyuy\\\\\\\/9FnGNDf2hbfTzhhXTQd\\\\\\\/bujZH7xXHtFAc9yAuRjv8dirtSgvEB4yIMNTAPczdYvPTMO\\\\\\\/S6Lq\\\\\\\/EusoRcY1zwMbQE1M1gLHFixS4KFbF4uUETHKcLhG+gqOI+lYC8+ilkqy2ls6POrrOWctV0o0Wph98gZ0iKtmE9llqUSz\\\\\\\/90h3C2GZhlgD46jVB6vvu0zebr0XW9\\\\\\\/9d2PN8r8RRvFlVVXkXE8+gz70f\\\\\\\/GPkAMoVL433CkUquBIoFw0DXr1UKG0506W5mt+N60+90jGor3lSS8p5PhvIWuba3Dg\\\\\\\/wZtRmTY8tvig3p5QImrCuA9FfCdTevxnzoLrWi8OfXqiu4jnlZ9sje0+vLSt5XIw16w0AiP83rdZEz\\\\\\\/zYI5h7kmz6gc+wME2q0rzyNZQla3NwS0YQS0T6Ucf17fadFFt2u8iM+kjCWShn26JhZeggvU1eb\\\\\\\/dOgKBQp\\\\\\\/2cXweTXY7zMuwKJWZ6T05YmzAnkufGbOa4FZIYQjBGXKP0vzFVLOurWZCLZuJKng0fUoUBvEz1BjE7xEZUVaIHhZNvsNhwFxiZYwbmPP6Wp4o3nmldKuk84eX5P39GrelJ46lWuCv77Nj8E0YSO561KR7eda73Ca7N0g0cr7JXIaAqNqpmbYcDtYQE9ZLCNyyhBKGsci9FLBM+2Pz+1I2cvwoLU+X2AM2x6VPidBKi\\\\\\\/mAQcDqcMLeGNSHYtGd3SNRR0Z6xLUpGgdfdyutVYxDNOvWR\\\\\\\/hhgV6Gi1MJmpVXdd4cFvtG\\\\\\\/M\\\\\\\/8n02I0gIuq+78YirLsvyq\\\\\\\/HLtSHuCVhlHbKgvt645tV0K46B8iUUJ1n6AzY4RjwMFXgLn\\\\\\\/7iydl5ekAGXRGs7UyDojob27IASGdPxi26BS4xpSw7wMoRTtY1ZUnXlqsS25Nq7wSxcmjp\\\\\\\/aFxPhAvuir4ePO3bXv7QB66cOXW+f86MFvoYcN6bxd1JJuIMyCBdAeeyYz\\\\\\\/i5rS\\\\\\\/Hf25tKUw9Smt5NRxZVhTX3chfA9gB\\\\\\\/O6tziyFjSMHZoc5wFws4gFLy6LEE82iQ6rgEVQImDSo8vhX5BWykeHag8Czko1HCmvppqaeocLGpH6mPeJoevQA7qshcBOR1Xs+1XHqRFhwFARfMGYenkg4LikzxYUn+gwwpjXjzuMqkOdSdr8vu3mATB6HyMVe7wzgMauL8u\\\\\\\/KT8rN\\\\\\\/mgX6DSI5TChy+Nrk+Tp97TnEQ2UvJJ6hoDdie8T9prlrUFVygN7fVPLyLFXFwyYrb\\\\\\\/meOMaw4cDDkZQ2nBz53vUG1ANt0erVNA\\\\\\\/DV8c4CHbcZumPEc5Novd\\\\\\\/Rv40Mik2pPWQRLDrLYTLyf8SYe0CQG0ecKIK6mhuKux5mLt6SC1pgONDl5L7MdM5iBOEiDcDg8B1yutBHv+5m4y6p4E\\\\\\\/r3qnFlGcrqeq9MVPpThj71nBHYvcMrojfeou7oHcx+YywYaniM2Ttub0Ui4VzeN1DA6mGGHgpRQDKaQgEbfYFTsEQajlSLD6So4B+8DyPhqu47mAsfKea7WLXO\\\\\\\/mEN6Z9TSR1LLr8rS66dZpN+LoIDEoD+0pHHcLprSe01qhUvkN8jCW+aoqppUk1b5J25iuwaQ4+eHeW7f7CAFE3T8pw8rwt+DHRvB9u+GQiq7R\\\\\\\/DoPdIgu\\\\\\\/xA0mBwOqwY30teBANMx9qN2Rs8vLlea\\\\\\\/2OyJzAusKZit1t3XZ6bzWw\\\\\\\/adZL36ledhozkx+K9MMs3vlVi+5+k2xh4aFAmNOijBpa8BBnnlrv9VEcOJsQ8TeyEwTQYo2JRQy1sFV3QMqoaH5kJuS2zhPm2sV5OrwI78nabSN+CGEx1+5Rhweyi5T\\\\\\\/K+LzGjAyFzo6hiQfLOYM2ARmh1mJuMhx0Ew973CsyP21c49S32+\\\\\\\/A4731HISIKUJZInWD30nkH111mRQxgx3XCfv1HzNo9mRRQgWqXJvU+s0y9M4gxHqqc9CubKhCymJ8gqRL2ROfDnCCAFob18EmS25qJ09tb9PVWjfirfDzBZV6IrDbncEtraG5pJ4uEI04pkwkMgmVggNH3WnEHz3iDgX4AbQ+ywoLMd7dLQuAAdrvoTdgrWpe5W1d9sZ36oQ9VjLXj6o1ksUgCQXrw5ICzYZ+NKR9q4FJmin7LaPF9TQtEZRwN+9uTYz3OMrYW3QLU8\\\\\\\/XXU708IXMPH4LfdzNO4x8XKjK9Vw1\\\\\\\/d60oyqUBYKO+V6JLrvmh0SA7rlkPCsGqdObADpWPGVD5h7JtOZ61Sf8u5ELRpJD5ApVukPzGLOJbxCkv26RxulwCHOUDbcIxP+AaTTKR76sTp+WXDnWf57vCMVMZLBvH1nmWh4VCePhONssUsFIFLtY3fVdNd3tXsWakwmGvgvBzuKJYoo8Y01aNL+E4m1a0fMoEaMRLODdpoHozB1Y\\\\\\\/khyC6J0MK9ygiweVuET40sqDQS949uFGMk8TzQMWYqw\\\\\\\/Y6UxJJfPdmkUNF0rcQRSGnTOTbAPlsjCS5187z6X6qB2t4tpNpUKSHkPuMpPJihQSNt3Ihng\\\\\\\/iNZsr1e1ymKKmmWx76o73hj9U0VPxuXMvGmU\\\\\\\/37+RuJkr8eZ\\\\\\\/UDyHCPkiJUF1BA+LMBj1pyfr5DexYZ\\\\\\\/Ep62\\\\\\\/epRRA+ysuA7mziv6j0VqKGGNFnPP7oWDNYrTUs9N7iYxzwyvnKsFnxH7+Xj2oWpNw47FQ7zaMWR+nKEys+36C1oPZE7Y\\\\\\\/0el\\\\\\\/V15sm9sq3IBPoggxQHSBvixUcH7e\\\\\\\/OBTCCkQtUP6vxaKjMEW7WeGSUKAdHdF12rXUcM+JpgV5flu5D1oty1Ko\\\\\\\/v3l6kmwQDJNxYAlx6UfdOjgIqyBuov6q74kj3Y7\\\\\\\/aqLPdz\\\\\\\/eSmbLESAAhG+XRYO3zhm8r8hnKlm8OJ8b2oBZ4xfrRGzHJ7RChBM5oDU\\\\\\\/YxPwJgllzmlFxMqQwPs\\\\\\\/uDJuBbRJoBbV54PJIXmOUSpNBJxz3TVRs2c9BLvIi2MUfRO8wYKPtR1ajzq3IhapZTrdmCNQBg4NjcWeD6xYyDrZpy1uVSel946dZ4qLtyqM1hCUb09qEGVGugRXp2rsQG9+my4VlNK68J4vVnZEDFnukHdORKwmVzyRQfREb2eg3VNbYiDQ5r5IRqh5vo23MZZSOnZDZTz7qrheIFSocZDaklcVbbApK6Z4S1F7\\\\\\\/1ildHapcRG8T7Zi1qvvyVvObisHJBDki0h9LCz1lIz5QC0c5tEy+96gHZFYoEtSkj20YdtKCc4KGf4MfGiVZLNknDEb0EOe74TTeRJ8so1N43yL8Ki\\\\\\\/gsvyREj0D4QvtQyKGzfG17nGsKd9pU1+ajCCGKT9qPGXcrs1UhTOZBXp60sXenSO6bWgpBgDbV3zPmnSUN9Gg95z6MCHvm0NVEtITh0kDCRlOU0E4yx3fHGRWqY8ysYl7srEo89eNjXIXFLnZ3tHR3n0LqmiKsRIaymZTDZuchBpsWo2h94xzEJ5+t0kdgTEt5R1TCgoxYSw7wOhCitUqp74LQTXsGZjNbdTqCHLaE89eoeTMcEXX+9c\\\\\\\/5+7ZrODCk93qKqDjXt7VV7NQr4qKLMsdLcRwedYkVV6gdEvzyGG8mIij2kME03Lkp0fHPBn2flAkRb9jQPDptg8BEfaiW0oiAFjTA\\\\\\\/2Um72P+3RpYsxYGo7yeetAgfqvPMDb5TcqHoSOYVomdZg4cjpfqZ3iX1U0nRWazeLkhZkOWDXP5B2gL0wxKVJ6gy+ZMM9zNTpws7jqdeKbmirrchdmxNNnqLop\\\\\\\/kYVEVzLPpB7R9XHYto2rqpNfbvzbvLYNx7RvXiFP++yEF3gFIMwoxIJK2PcIp1NGtiIqvU15WmJCkFXUn+F3qxilqXlPFyzdHUKjvDxWW0hdmSyW4jqrwSxZ1v6GiRGIXvjVxFmIIlTeVivF0edhkhw4IQIuUjIgXoQZv1tvX1cMuNkZRfeAp2fo4FwELngBL\\\\\\\/66SJdIYVlyUu7wuVAP0khqUktsIK32adJuzQI2ke3P92UMDd1idzmDnCakyuTLtc4ZfxBTFIAHfjYDKouISPuNR4YpSkXTK5O8MwgUtaO\\\\\\\/cLoZ1U8KHIiWUWtAvw1fPsYR\\\\\\\/iMmT0zJZvaIBZiNex3zWu5O1Pg0X5QtCjXLjoBEuSFWajwA0RQXL5fBcrPjnWfeiupPN0585V4N9+UG2nCsNa\\\\\\\/PG2eTzVxA+yYNDDux7vGozIHnF36k+feEGZkLxMHqRMUW3XUrwPTHeU4LAjHY12vUpM7t41uZXbpoO\\\\\\\/29HHOs1vVAkXbvPkDh5gJCq+Eyvux19NtLW\\\\\\\/AOy4xmzq8NjneNVvOtbQYW\\\\\\\/RiPDG5cqLSuz5WyYInGVq5mFgSKOxJskjj5A77uQenuE+p7yXzjmJPe\\\\\\\/AJs0wpUxRgMC1VnsqZ6OVD44to5j\\\\\\\/Oqz7Bj+WPWaUwJ9Kbby8unMG\\\\\\\/JpEkSKDRXlHWfXkD\\\\\\\/33GT6FlerZG\\\\\\\/7vVB2U+6m4tXN1v9yvL260dMWa\\\\\\\/UrU8mPB2BvYFYIHGN9p2lBoVHH2mDCX9A00X\\\\\\\/mA9t8kP9KORi+755iovd7d5kFN1fQOkUdndw9KnHd8MKPqKKbMV7jVFSuBj8u\\\\\\\/GzhP+uDJde0AkAs2UgdJtgwC4LqroWLNNrZhIocvCzusEslM0uMZlQVYFGM6H6kRLTQBNwaIJ4f45nfvcNqzPKkzTEePhOZojTvU\\\\\\\/3RzrogY4ZXRLmH2gEDIi7GZ\\\\\\\/LamuY6L8kw\\\\\\\/U2TthQ3USodATsjRowAYqJlPhMK+u3NR9bhpR\\\\\\\/4Ev9+lDsXhTEjrFFshkJ424u\\\\\\\/o3sWBXAijcszgf2nJlVMmxS\\\\\\\/4AvIYW5MYgS5qOjxOlFF6EmrOJVbWUI+r7mjGBnaxy3bCxky4N8WIvlQjpmsyFcydv1IbBs1sAGnsJF+8FqlBvXQbuKDRhMXBTr3bSC+3WP4gnKKcPxty2J2UOnvknEf9hibJW9wES\\\\\\\/6MBvsLjpJZ7ghmy05ZS7fbswxxkv\\\\\\\/OLMiF9X9XcwFy3tjynBRuEblqmvi4oJP+L2kw2bGNuduHdoPreiTLUj+e8XKjv8hCRILIiP42EOwvVhvv7Tc2aOVgRFPfabPgQp0GJV71QGofaUtBCaMnqDzwVvC\\\\\\\/tBz\\\\\\\/eWPodwsjJd0HMgKcya3Bjro6J1qJB1i7AHJ5OtKcNZltI\\\\\\\/1muI9wYh8fpnFKfKnqcmywPM+PL9FJ6M2emIvaNcvZYnD6pLQyfLA4eERTyGYL78GXtQEjPbH7ASYep40qJq9cti7VOHLk09uRLaUmgAYR7gdWXnBbWkljFlEpwqW\\\\\\\/Yo9Dab4AYRQm+VuU8KuI0AJdYNZrZidtLVmnF10ANrgbJARiz5C4+Y6mimZ2j3vIAelRp6O2jXykpxYFQ7uZZdtHAV6CqVR5JBzsG5mF4WfKxXUFYFIQjEIwwf0Rkys+hwLYUotyKkmNEC78QqDdx7iGkdh7YQDkIIocfqV\\\\\\\/wEQyc50EYS9z7iBKt2+XmdDOg2rQTNG7qMln\\\\\\\/VbposDYv8LN0UkuxydtcZo9tjYpoL5\\\\\\\/zvzTMZgbxYDWwM6qptCWm77hM1dCMJ+CH50gtAfnB0dkK28MfkmzIqqSPR5eikQSuzUpDxoRipDf8jR0XojR7EYNODKp7d5jrZ4RQNwgjMni2eZK8fGCKqtb7IpReNUa4gi8OvgcVm\\\\\\\/EZsR9PRZNY+2UGbL6I+KzdVgoJKpvdrGLMyXztTwEFnqOn+SAVROi3LRn2Rr0NZHMUAzHplgSKKyvsa\\\\\\\/Tpi4UM6J50HSqtnoNXRHgZx6A7IBOzs+hVLd1xk36022Vzq1THjqPgYP13uaj6NJQez2uJ1UL1QhPaztT1SLfsXwNNd\\\\\\\/V6j2xjA8eSjkeo2asOrG9\\\\\\\/\\\\\\\/9ZjSKyXK5wErtBV1JfeDyQSo7blSgbtZDE1zp2SPI++0L136pkv4wbhrNDG64Zw26oYU2r5xAXvCVcx1O1Z8hbnonsADOadoI2gMNtjqXB4xV+uqkynSgNOyZABVKWAsDKlsWA339t5N7O1Q+BjlPGzox89qFetCiQnqKA38BhR9cOic83sj52zmNa2t0lUzjnGxq25xU9k5mcEa6uodW3P\\\\\\\/aM3fI1rlAG97r7xiuqvpGGaUIJ4A678yYVVNMzeFSi7dRSw0ounXFXf25GELUGCsfUKAuY1ft8L21b8n9KBF3fU+9acKo4sXFKNncc5H83YjeFVTi6hkJ2hLBPIjndUf2sl3HiDTHhj9h3p\\\\\\\/dC5l7kxh5WIYEC+xICDuJ3BdaYyewKqwW+JRh+QjpoeY8S53+rU5AHIexOPL79jEUN9qdUhoeJ7En0KQYEk69Ka0B4ff\\\\\\\/bCACMd+U4Aoh9wj0TLw142efXhN+u3DW8blst0Hyi0PXmHhqDGSJqJy0+LKxCDeqDzkVieg0+M1KGFmFb87odnL4XJW2Fm1BewsfZywBn0YVUM0yNCEFCyZ+VY9iW999QvRoS0Iun6agdP+l7IwUeEPsiGrtt9AFalUBHvb8QLoH0gI+jJAmEmV9QyeyHJuYzjM6VWyG2ZqWylLUdwXOpSY+fZD8HyC04JPQ9c\\\\\\\/eIK0Noq76VTBvvdkV2BHMG8p8zRU4cDNOc5Edo6IQjlDC2I3FvWiZ3U3SsKIHFopOrmi+TkxzBq7TJ0gIZRBuPt+90yef+n7\\\\\\\/UuuWbGbk1yuRU4Z73wKkyo1DCpcgV\\\\\\\/ZlrNUtUOXFZkzVNqSRDrXcRyIEDUz1DjRnNM8+lL29MJrV62gSBEvd69zr5nYz8y8ty5sbRnucpepQEPsZEsUjUi9gT+RtClUF0IODUneQxOdSdDQtOCGUOOCemhWTdU11FWejznVKJ5geY5wD2d0NKZM6kiq\\\\\\\/fR7ZVw89y2d3jcIfbGqWam0OjjbjztmHuNba08HX2XVJMxBiVc6WWT\\\\\\\/6VYqXAG3RqW0usec320ouSTHT2LhjCNTpTmsiuHXeNRQWibpCJQWF3Qeg7I6N0LuNir8VpLFSh6WkfvsfC\\\\\\\/0Vi\\\\\\\/K6nJBelXiB3JchTVGNEKXPdtAOjMxcOeLD6D52msbpGYYjaW\\\\\\\/\\\\\\\/JUaVTsq+NI64LfzROlg\\\\\\\/yYqWaO3sCCQib2Lul\\\\\\\/DEIARz3FLjvqSc24mTWK7hbkHOE\\\\\\\/jc0b0Eu9iRIIikzvkYa0zRky+3KxWTLC\\\\\\\/+xBGnWlnnl9xwxz7e2e7kysofy6P49zsTwi\\\\\\\/f2aLETMC41J8xm5o1xWOisIKmFYCvuy0k+baWWIsA4F9E30APIuhg9heOV8fYnMgb57t\\\\\\\/akVPDt7WJSIkXVbFdLDA42n3\\\\\\\/TQoqZ9Gg8ZHkd6nibblcIoKgIOho7G4elyPP0oBzutnPROxtLKH4yrNy1SByzPdpBj3YPzkUkIMUKZvgv66shkATArUHu6v5odTvNh4mxpOtpheOOqVvOa7cGgs8J8RRga7R46imBxqPbwn1+20Uc2U8G+YWLQyxbMlOzfQOZCW\\\\\\\/AQLnR32PgxJPOCvuydJISZevbcMqEtKpW\\\\\\\/OvrfSMJyFQtprTk3UxqgZpD5C\\\\\\\/AXCWXKpUUBVtT47hTHbrh8Ckae5M5AYXrAvEu8mcirIbHpTHg+IKnPXW0C0\\\\\\\/wUWj863KQsQeKvGJCYX9q4iP8FAdHrXaDG8ptqJ+PmYE\\\\\\\/i2mgz7I7dEUuVUPtsn2MgbrtWdeZRwlT37xyzFMi0ih5mDkOOd0sWsqTCwjyZfIocTVz6Cl5u3N7bW2sjrhfwNon2YlJMvV9LSvR4rgv+nuDuGs44eeym22y2\\\\\\\/\\\\\\\/Es59r4Wyfjfr+6+Ai3k3pzOqiNyslCldwTFrX6xcC1E2gUZO0dYm07rRyr9x9geCxQh+IPbRnktNWzmw5b6s1jBlbpKKhjBACSYkQYQpl8CzE70ZCWMqUuq17De4SfraR+ng3ZTg0PiAuh6kTJoLN21\\\\\\\/OXLAKAIhq0Rj5DQYKbynlPHyYp0JysI6iM4LLpaAjEAIEnskWMGc7FnmfxER\\\\\\\/F9ZNoRMZkmJAK+\\\\\\\/Sq2e5mVQy2XrkiXH34gP5sphf4mmm+pl81oHGfs1GZcbGDrtDixqDfGxkzpZKQHjv0wWO8Q6YliIYdEkxTJsHe94F92CUpHtsR\\\\\\\/AnXot0C5joMCkpZ8P23jZ1fU+onNQcInpH3n4Ykwx35bIU+A4lxHv2pwT\\\\\\\/hFbX1fIwpn9CWpH4VOmCsy8I5VVPLfvxRV\\\\\\\/tKdoXRAMJtm6b6nVA\\\\\\\/\\\\\\\/pnYADk63\\\\\\\/pIaxx6+GLxCLOvn31fZvBSpa1EG2oesj4ZTky2s4ybu1xlhibFjDSWu42bfuj+H+XWEoK6sWRtWB5plbFBGLjFUmy1BCUxV46eslOGTwA+If0AVS22MaqVoO4uDWDZeX0iQnu\\\\\\\/+Abaa4e0AzeY1mlsxspKsMPIA\\\\\\\/SfES1\\\\\\\/JSYAFERgYYuk9YmBZwVZMuBYQfWDgFItpqKvXJ4PotrXmst4AiRtKLUWfnGVbAJjoX3E4eoFm3iLADJlgsu3INalQ2O8sjfzVUaI6\\\\\\\/DlmmIwCvtfKgjXVM+DUWwjgOLdlW9SZbAnLD\\\\\\\/S5D06rvXbePFwAnOtuFYPv4b8a8QdNojrmJ2SroPV5fwfYgiqq7pJibMtoeISOw+8AIMG65XqCkKZ3bpS070AC+Owa3HmvPGbo+ZfSHmkVvWdHnAGVfx2FZXwPsJubJMi3T4LvJIFeKnfKbVYrqM3FJ46Qr95\\\\\\\/+JQWvFyYklKF7ibuwzlfTMPEboq54qNgsK3VLHK8MyZzWKKtsA1\\\\\\\/U4I\\\\\\\/f+0QRbqiYzz4bJO5T3Uq6xpY5fToUaREMy\\\\\\\/qN6B+QTKOnPxshiaIKaCFCtJ5mV25xZEvksTOdMHnZAhrD7eYQdSFQaR2BlQiNJErpJOmYq7Zd0J8mSMqKSzUPyMOKdeiBqf0DecDB34UE8kyfese\\\\\\\/31ePEYQ0PcDjhswjEWG+B5E79CtUWF\\\\\\\/6XzL6\\\\\\\/29DWAGQi3OK3DgzYdjoH5g1\\\\\\\/cbB79Dye2shwqNNyjaldWdkd0RPvcTjJwZ1gEdyM2xI5pKpDvnoFJGklaP0UKjRt7V41PzZfRS2a2Iy7WTNI\\\\\\\/c8AhrA68W1cnvE5YCZEjKC1Y8BPHbiWw53OGFn5vPzrzvHL6nNpi9p04eovRcApZA03q1oEH\\\\\\\/bgxAC6wIcYQrAY0GfFgt004b+HYLgMZj3lvz2MY2SKqTPnw7+ju8zylM1Oa\\\\\\\/NdesEgpywQQUbvPzUWIuTeKjbCgyp3kiKWL+3EVw+cQc\\\\\\\/cNDw0iWCT0kqqo7209slB\\\\\\\/3rE5dCs3\\\\\\\/FbqaPvFCPXfsipSjW6VPTfiIWqxqj6rY7smknu7V\\\\\\\/quMvz9xw2eitX0BeBN1pXoUhpr86DXGuWQoRTRtZmNk\\\\\\\/hdGJZ3NtvSBOLXjj8hosZuKpFRfjvCkdR3gxlStAi8wjB7o3Mv1qTeNQE2muJnW0iYFSE0B61tsZgHE4nxSkb4jmYI54sU8bmSB+tSfRS59b+ABezf9nd51mDGHexlYjSFFqMKDp1pEgqVFOGgVfZ56rl0sK\\\\\\\/i0Ezggpv1VS8uiq5oyY+Ub3f+mwl17+naZR7BD9m5Ld3cW\\\\\\\/3SP2NzWHMmVZRN\\\\\\\/wxvIU8ZPqHhAMltKzAhcIWpTIZZB\\\\\\\/TZTuyfUJLGYeQDQ2QqwmKf5d4piUjT63VpGimj0KNIdbSlI365GY+3Hr4t5regXA1rc0WFxGSSrGrLvAGeDn9uFA4KIOxHfLZTbi3ylXwyjPa06sAri\\\\\\\/dCXoDab10BYsSUOTZeSV77T1eCtOc8HBRNCEsf+VqR6Z\\\\\\\/J\\\\\\\/wPhZl1VnHuQcH188FkP0avbTQbcDUla2OQUWtlDr2dEuxRD+PPAmkFOAQP4Ebatfb3sVQocW+zhdACt+eJQBJjzuY5OYvKiOj620EQ8Xp6Fvn\\\\\\\/rwoO+npNj9dqMcGA8JUN3G0VBiTTim2hDI5CvJth2ZBZgJoIj\\\\\\\/dC82YM+7DPXxhbntjOri\\\\\\\/klPqQyqeA33xlPAt1Q3DahMKl9P5WdYJWRGGqUVTGnh94IfsHf4AYic\\\\\\\/Ah+eXJsZx6n7potWpW1EoqM6lTTffB0IQkMGqMUYjCfaILC\\\\\\\/CsX5aIqlFR7kkAZW8LyzI8tJCtlo777L0QScEgMIzbo88wgO1YLaPpWMteCzdv7pvNRDmk9fYER7lJ95rBqJTUGv6z\\\\\\\/Z2Lh2yQmWmZNVuvK9txjRE4GxPdQ7pEQT6n6L4CePsYPCRUSddBCYSobeAG2DrsJoqTPuY5TY8Rp4d8c\\\\\\\/ISrQsNKIzUFt\\\\\\\/7UKSu2a4fOWWnX6BCmRdPsD92BjnFUWBW0GYA7S\\\\\\\/2hG8hMuyuax3ONj1Hc\\\\\\\/u7lBpgaQyizOPC3Zk5n8Lj+cyExSXRD4wXnyYjg5iIxHCIUuAi7aOM+ccZBuMNmo3zglIakmSd9k7YAdLvQQmme8Q62KE2zj1wh32zF9pt\\\\\\\/cS0WMvL5kYo1FCBoCNXxvjpBUqPvDK35g7Urhgqyw2hpTP9+V+vC6EqRsxY3ybEZdl81M8XMGKHP7I7F7uhsffwU6ouiyXlRINs7ToGcTgmg2CIw84iVguvU6wFo+i7K2KOCNoV3cU2P7sOgYNk8fQ1t5FJLrMnnVLYZ29zVS\\\\\\\/b\\\\\\\/kHuDTp+XxnUtAc5Kym\\\\\\\/gSh805Tp8lr\\\\\\\/wUbhiP0np1gKFxaLOV58\\\\\\\/PNthWw\\\\\\\/bBxc\\\\\\\/96dnPvpjw5C\\\\\\\/C+9JBGfPyJw6POk1FZbOuO1+WHynpvG2QKpNjQCtb8\\\\\\\/Ntbnl+j9Y6RN4\\\\\\\/SxANGjcgTcoK60NCUcbZBoSs4CunOllXPGqeJ2vBPGWT0bTDTQbGCtVH4o6YLabyBoNxliyKOQVl\\\\\\\/OEtOOEslsWXZhgL3Cu\\\\\\\/6QzRZWB03AzgM6AVvbuWJMk5hKmu0hF0Oangs1cYdPUYb2JZK0fja68wamuGrhLjBTwG6pd5VyshvhycPUZNGBy2H6BbJ48dI8FI+MVpfKhReoVaAHAtVFg+h0DdLIOw7TjFXthrT6KMw5GpTqw3AykQcvxQcqqgEoLLPSnvZ1XNQP5B9KnKmax3MGlkEX78lI\\\\\\\/RVEvdMnqezxbn7pAIXhnd6r4ETt+EGy0DOwDMmwGhCh6MIzshs4\\\\\\\/MYD1Iz1ZEEBWss32i3FhOH6NKCnXEBicuh5Ic5tdzrNGKut9pyI7lEDiMFilxH5aq1foWPTAxhA19s\\\\\\\/L\\\\\\\/LTfve7I0AsRVHfF2dYdPxT5fBC2a6Kv+bh95Pgo17Go644I7R0T+cm1gmlmw9wZqe3RyXf0qq8pcG\\\\\\\/BKaGoWdisCX+8v+krLFZF++2zCpt16b+lvoUJglVGtlyq5K5DgyoWurYhqza42709\\\\\\\/1jkVsk7FFKbvRRE9o2qormLkEgQSnAXzVVxTeWiu52awXWYS\\\\\\\/bTsiDh4pIpAOLemsvUk2vftH5i4R9PdXfULuGGiGPBGfCp9FUcSLM\\\\\\\/Xk3aKLR6+KSJehdCpxOkCusfE\\\\\\\/i4xzu4gnGX3Ahus5YWiGm9GIy+0mDWKpTCJphu7icL3Gcyr3Zk88bYyVrdmlW5zfii0PmX5o98W9sv36Af\\\\\\\/o6\\\\\\\/IZBjngD7ikeFIQvNHOh4VxhYqVkPzTDeGt1EnN4LzpMaB9qAcL3zYDdeRgvnPMUDn9XWx3VTHRV7BJvKfc3VcyjqKckQ0pU46w7qWPWYxOR0OG1piZNPwkXonqUJNrslTS69mcKRy30fppQnejDPIHsigIdGf3OLBzv7fC2EmhVoZTEfequkbhehv0bJaD32SDlwOaWXLPwYY3vRyVlX0vF+z\\\\\\\/aCcHdpXX8BvRWaUokr+jPjoE50DbgXpVidt2P7eZuJ0rEiGqvdv24H7kUU8i\\\\\\\/3cIsVfQqaY+pJlYNN9iUpZScEDLbcyfSN0gSmUeUsphYPLILvf5l\\\\\\\/zetQ8oUdFiAwgrSYnBnVRhEZgdYQcoR6wtPkXZkRHlgyorM2\\\\\\\/yttg6QPEtjf0ktqh3muj52E8IA2bU1mFxDdoG+H2BsSKDlaHUl0h1BWnYLCbvzIdV45MTgF01niHGojXD+s\\\\\\\/XgSg6nNJn\\\\\\\/jvMbc7y42xIyaa9b7vk7AenGD9nV0PwQ09vKB0Nc+Uircgf1yAFg1l2kvabHNabE8umuRkyKVToy\\\\\\\/7ALfsozmbpdIHB76UITHQHQ36WWa99k5mo+eWeoMSFi6vAs6ag25gHf27\\\\\\\/pOAwYdunPjnrTS7I0z6\\\\\\\/5quXwK3ARskjXicGHYnKG41txX6aL1WPL42tnCn00QIoJL28zjyCOled8tJsha518TaInrlJ2dpsCt5zfL\\\\\\\/0TDC06DNYZZOukYsRMeKXwSALL\\\\\\\/+peQ49UP85C0DbBpfxa312VglcKtuYoY2WX6mbTUEIEw9a8LvXycWmbDpnVOIjheqGmKRaetpmgOV7V2Vw91y5zJv7Y030IUINOCLoD4GHz0oqbfKnEYlyXaeHfZQYdx7eSJcICyitLVGZQ1JCIGtGoXXq9qcTX814gbg8tWou12X61k+uv4OaG7yKh6uXb\\\\\\\/p52CD5jQXcJpgqLe5A8F76xDB+TAU2dGg\\\\\\\/dXqG754iT\\\\\\\/YfPQ7D9M0xIeFhlCH\\\\\\\/hMQUUeLRy2t7TgwTAyA3PtYUNWLWseGxxh\\\\\\\/EO9Vhie9c527brL2OHkIS1qspTbG6AcIXIhB7sk9CJ\\\\\\\/Y\\\\\\\/yp6ILqvexNOGaLGNr\\\\\\\/d9KbKhkifr+tCYYCsMZWlXmQmAcjCMTbfefN1i5zUYepcva6NPmjktSFu8wIQ5Pj9F4qjFUGbEAimHu6QBuh8cEqFuC283MBMlpO\\\\\\\/MXHTfjjh6t8BIHawig01IjDHoY1GRRu+SpiIrKaXIi8Bj6hF8aRYt5dRM+JHhnYS64j4c5Zk2QKG0rUHBHMzFgF5aVcQ\\\\\\\/JP+GlA57WVpew6dnA97LESXBNG1pFIIphq2uKvojYrC+MS8Deqe4HsNbL0SlGjQdzzrzA\\\\\\\/cuc3Gur98hieOht2iXdryZ3+Tw3C1xOMReKUo\\\\\\\/f3vZqx+CVcE23PeKvOZQ3OUGQBpPlhYmF2slEapbL0WT0LbzW+CSHJ1evpAaMEFEsw9nHl394kRXTzNQUSsanx38bZXfITGi8wDmQmdsTmomUmY9DEBQTWYbJatLVvPn7n9YWDn2NmM8YRCM\\\\\\\/SBtZCI3vVO+P549Sg+QF1uvhWk4vAf67d5cfZ8etpr8jnORraeIm94nLEl\\\\\\\/SfGfzq\\\\\\\/7leC28fy3ErftVgBVWHr1EnGwB31R53wzssOCx4FuZ6whiRaw00pFv7aS+ApLKUn+6d3q0esvDenKcbr3jQpMqVAbey80j0UYTUbXSlsscvkM+Eo\\\\\\\/mfFleycHSRN3ETiNsAXGQ4omt6HirWL44FlEgiC0pxxsJqEpw\\\\\\\/K0c2grR89w3tadPZGogpXwnstVX\\\\\\\/1Rbzpnt7yGfoU3iZ\\\\\\\/1i0maqJJIfrlNbHLWyaZS1LHggPFd51YL9+S8YEzlA1vBAUy1F7Y\\\\\\\/LwMiyORZQ5prIqinDOShnN55UoLQWGFsjXbZZNno2vcoNTbcBAZrSRSehFJw2YYms+FQWp60fKGogGcG2wh8c4FoKLsyBye20+U\\\\\\\/OWIClb4g7jAicHkeAeAZrjcyROCUWjh+\\\\\\\/1xEoXe01LyQycB+1cGuGAanqNevUj4r+Tso+ZcNtTIqXS23S8bzyznT3azX9IFAlRrGTD9Qs3GeZbNkMFpbsxla8eMLgoj6NuWw1Q8FudoPxMlbKjIhdYvTL7doK0VGAvsh+SDOVb0fcXwaP+4NwmDwlMdKPhEzNt3ANTWsSuJk64Tzsdkyc\\\\\\\/BX3EOVjjLVWLeWQ7NN4k86WBMVgG5rk6GZE7cO+8CnYMT6YF30Xp7ayLuuByC7PQ2mGqIvg2fz0oXRDeiL5CU\\\\\\\/4WSD4QN+YQr\\\\\\\/o9uV0lRrJ2CFXk7DROu+8cBXBpMZK\\\\\\\/8UF270czw7IRIhELB+xpi8SXBONQh0g9QGcstjtDdeUruyB0KlrqW1RcQ70fv4KBV+UVhT\\\\\\\/mBkDVh7iDwy\\\\\\\/skOWBjIib\\\\\\\/vd467zkNqBzRB8jwJJeKExVnNIVQPeYqm5\\\\\\\/tBtMGAvYvXvIYkAaEf3ij8b5WU1zOU42t9uXnjgbuR3BgDaMazRulli8tLe5rQYBVm2QRMcnGMlY5v9Gs0lDuAHFpnFcEuVck6YMn\\\\\\\/ptEh2\\\\\\\/lql82lls\\\\\\\/1+aUVsDoduAzEWFNrUeY5vUOOvBFZT41AjNpg1enjKcUPrTUfrEkg36+ewM\\\\\\\/cjKt\\\\\\\/I3vGvd1miGWBHEBKP3dUghZtvfGg+kBmpodqROqP6A1nCiyZ9xowtW7Fr\\\\\\\/ZRqxiVyZw6LaJ3MSz\\\\\\\/AFzHZqMfH\\\\\\\/7vw0pvfs+WLWaZa7o5BKNeKD9UjVsCmkpYlida2+thQ1Mj0rLX3s0VEjc+l\\\\\\\/O3A0c0phTkUt72r4rbEQnAAFpcVEvp2BUCa1r3GnfLZmERhunGPAmZXc5KCE9Ry3u2or7mJ1LZJW6sFd31mBYUcA5geSYwFXIQwR3RRcn4K1h2N3HzUxhKsvi2\\\\\\\/YOuw3alIstXP8ZOSKIv73khUfJp4QoX6lLZ0RrzwM7aFdYvoRPRXU\\\\\\\/6bOwVUVTKE8KeOWYF4IjR8msQs1ER2kLBQlkz\\\\\\\/m4eCjnleVLc6ZgjsdYtvHRXEBqxLiD6XwadMzSbETDDWsOKWwMCjy5CAE7HENJK4WMtmk4tm30B9R7m5UnR2zTAnBRfrtW83jwB\\\\\\\/HvlpE61hXQ0aDSmKrTx5j9JZWsbvoSny3DAsklanj1hmLX+lh\\\\\\\/RxQmvGZ9ucrBCFIfBQnQK+W\\\\\\\/Fh+S312BdRUlOtUkxZgxt0z4D8CMX25gO0gQAbADcqPfzP+t+Qyga\\\\\\\/7pWQFe1wKe\\\\\\\/n9obF+w\\\\\\\/\\\\\\\/UCOczlH40HO7TdcUwPXuiFzv6WOhkgYFSSJuGc4ymxwQKGyeToXyA7G0JOROKmvx585MGbUz5qcKXsJc5GZwa05vC3dhS0rx54cSYhjzaKhhN8ONB3krqSqE3gmAGLhRzmD7vrXTgqpoZy1X5hEUivyXTLi7ZAKJz4JrcbNMs8M+60GtBm87PKRKw0zatQovLzOgu78RUZFwQuIKY8fizUiLOBIDXeXFf7ywq3NkfU3kuAuP+8KOOzn7J5ZWmuz+uTX+JMsdvS29CKSD\\\\\\\/n5i0Q+ZNacT3KSP6QeLDXge08Q2OAHakSQD8uJKZ61bSYvS6m3C3I5lgcMaNPbxNV\\\\\\\/tjicJRdVsVXVYENr\\\\\\\/HR4xkH65KV332op3\\\\\\\/XTMYM16FjKCyZeuzFVk7A078CkH8Btu3hqCpEbCdZgRgTDc9rsAx8j0Djq7bIdRwaYYj5IiO3j++y8mryyKcz5lSj\\\\\\\/e+kiXLY0BoEM5vicA5TbxnJxgIcdSKUQBuS7vaN89TMSSZgmWECt78E+D4XvqA04NKxVLRhZ0esL71EAWkJGQK2lYhGBVsV6z7xyKVbCsqNOsbBk+K1EYtyoU5oDySXwXLrsjKX1Ab3b3yXZ\\\\\\\/p7YaGE0nIC6Ga3aRgC\\\\\\\/jeazUkppP8jxm3fFC0frd1JR7EkAtsItcAVCybSL\\\\\\\/qhzIo0fI+X1knLyoPudi0Elo\\\\\\\/kSB2eJ1w6cdppKSvh9pMyY8EgrjmJV+t4MgwIn+W5aOGkby22zFNU0gROLBWSwW6TgTaLWpwLUjk+i2PJrpDG8UnBPhx1MqymqDAT3jwdB6T4HhmgU+ljnciOr9w3+kaX25dTV71AooaC\\\\\\\/VZS4gUcQLU0w3TKmeWPNY6V\\\\\\\/gFNg9RncsAv7zPawh0sYaRV3Exz8sZyPVSJohTRdryC6kcnUx2\\\\\\\/okt0r0Y+dJHb92CD\\\\\\\/N+4fcdlIdqOclCwS3caRQEyGEVPTVpmMnKxoHIAwtDw7PVWwl\\\\\\\/bEOvtR8xIkY21vI6PHqA\\\\\\\/Dq2Z3yGl+B8yU9ztlvVpqnZj9dIMnKhr\\\\\\\/eNudNGKNM6wxPwPUs4Z4bUSai9gRQlzW5R0tOaKj1HNv7paJ6ZQjW7TDMhfR522vhvwGSvVgRkoroCs\\\\\\\/IiCVAuDqGCWFxtRSXruxZB00E9Ik7dxLfJd96MKW4TNBS+WM3q\\\\\\\/wqB2ObJLGnD+ZD+CqD61Zi5XYktlL16EsCbjgv8g\\\\\\\/FiyNtRw5uUMm87t\\\\\\\/U254bMWo7X2GXjyKqlQjgdji+pwZIFJctmNfq4XQF1\\\\\\\/7SjfbQ+jODgiPzaE20DvRf80bGOXedmcq\\\\\\\/9fmjuIDvHYA\\\\\\\/7jMV0AdvHknsT1Xm4hMxvw8wHBFayGnVk2cmqWZI57p+CTXLzkmr3+ndIGkUXhfrcMjyji+tf24f6FAj1q0DZH4gTrNHw07B7ZDDRZK33QmJNYIL2Ikdj7b5uDLbraYhoCsVpWeezcPPKivJ76EVlhcQpZM6mnvrUie3MRvT1PVtgA0ibnHrwba27Pjsgq0iEc\\\\\\\/qjeDxJ+gWWQnqA8+2e51HSjMEroWIModfycL5KaeJJTybHgejLPp+3fxXG7VBODo83RD8RfWvWFbH4gklPoNN8HqjI\\\\\\\/WnxE0bx9PUKyO+6FJWBUT484GZQOvJprK0lQqG51i8TTswKFGMAbtloMXRwFxy9bfBEu2K6RKWuzCKrndb9LyLWMoU0ua7EU+5axS5TTXciK8PdykBwQ3hY8aDuWCnJgpxzF3QFIS4wN1GT7ufCqB3SU9ncYKkvzJQzCxPvUfh6JEn7tzSul6M9S2PeYyNUdAPida+0wK8KKYZoSRG99PBBXSfDHG4CUFtu7kXUVZx6klVRxhZeC2Mtu84lkYoUNpnG5hyH7tuMprqoacHclC1ZTRoP3DA3V6zRT5P\\\\\\\/1l9YgWPmiX60P0g1H8KUxj26Fkj+eogY4pj\\\\\\\/M\\\\\\\/MlqeKZz7qod3w+7P+5ODZaFQ1TiyDJjUtzA1tpWuJkWnJSVU8QfDJ1ZZe7+y7jiGGRqyPMn1SzAuBpnqLcAASDsjNhQSkRTlb8phDn9ZhEuA+OriGjm2kIGR4eoHLPhCh91SzWxxSpyi8Dr9q3WqGIAElRW4CcdxoVg7Na\\\\\\\/IPlnF\\\\\\\/j5+l+t5Hp4cyHJpo8xqAsPwlyi51VDkJMEtHTJdm2Qvjry4RKMhsBEy2M61Cu+ORVTxvmfIoYgNn6wP650pE0C3SmSfBKI9DRcPANTWlYyJ57QozVs09nEEkRGZLnfWwCgeiuzQibO241zxxzUZfD0YtcKAmi+CYhEoBCpQVwVDLqJw\\\\\\\/hLu2964F9HU67xBso9\\\\\\\/MOKUrtVnswlI0gr2crRmEMaUyLrsMmWV7LQTfyfU8MzWjKVeNDrryZ2ZlKOlyUpk5Q\\\\\\\/SRZtlqyXnPGfrDshr+Kvr5kfHnaTKbamKcKevlp7L37gwug4kemWWr1Uvu4BVhuITQcu6TNR\\\\\\\/cLjyV2dT8Y\\\\\\\/w7TNJyfoHLoR1W83YkRtWBz7yff1\\\\\\\/74CUjaeyDBCHIqxW7pP4TxXVKLl5pJG22lXJE2xMbuPfDshNUCSwRsh8FEHNJMjNX9M3uzjs8iW+kNosM2cg9hPE0AnKAnyZxujGLNGwp0Qo240Ljt\\\\\\\/3nZMjoiFe3e3IJgr\\\\\\\/gMO6Ehj8B11EP2wsOzhIDnPZdhZyXUxV+I3WiZJ4L\\\\\\\/PS4sV8yRdT2bL6Rq3JCBWpt1jSxRMU4L1ixMzztgLzwNW6J65JdxC0ZJh2WF3DBZyC3Mr+c8s46AemlX3apjvHjXuQdqkjl5CB5fp4lXtz7qvyVuynWtKjitBhNQDOpdFFO8O4MRp8TYCNyJ1mOc6zDU7cvSbFQtanVQM+iB\\\\\\\/p2PXyRP\\\\\\\/1sIk2cJfq5znQHz6xu6wPMZKAqMKMePsTY2VOgVlRLq1VaY\\\\\\\/MztlA+l7dogMRGZ7FvSqvIlKkm0h73hjkRJkT9U+l0HdUw0dDBgeV+z9aso3Uew2MDIXqhWTEhbPje8\\\\\\\/nqpj2n+WF1V\\\\\\\/zSPdkNZ1CslTYAiyDXiREXO0m4sDJQRXi3t984TRf\\\\\\\/ZxgtomHoO+hkJ\\\\\\\/E\\\\\\\/q0WbBCdBpg\\\\\\\/qaBr5TQt01K22suGsQN2hgH\\\\\\\/CcUdBEj+c9h4tO55MTmX7JfMMpKM1QpE\\\\\\\/9nNkRq8HExJMjd+7FcoxBS1RdPTq8S5FmUS9V4pK0b2ZqtSXPnqQQzQGfkgjIxe\\\\\\\/2JmxXbZX9hPQt+pEVXD6zUpU86YrMlNEGaKSscmQWdVXpBx7uHDy4TIXzDO97lywxkpnJRNTZCZ8rbjawD7aqnF57FLtD0pnyA8RekA7zlgUXAZb7m+XJtL7d99IKeqRxWNTZ20QvRZJaJwohcHVCGAL2Fqm47RsavWrjtmTHT8jnKlCMnGoRCUPXO3O5UfA2w\\\\\\\/HQWXzhf5hi6TUK0g8iQmdmOoHa5VO9wmM5rJ6MsCl6DIe+kDVGknriJ\\\\\\\/Ncaqjvx+Z8sC7e+1iXWILjBuLy6tJckFneIKeN3Ny6CgNnw17eWLAu9OkRa7LJg2qINBpjpzMvSB7MCd9DsIWLNOVOY6Ju2ZmzWF8CWHjuN2gQsu9UD3CQnKRBnlOA0vElCjLBO1u0dOZXon09PyxM3KtPCXhpC8CPT7J4vQ5D3WhuDmKACr0tZKoGv70ZAZn8f8V9qXt\\\\\\\/dXMl5\\\\\\\/yQthdugPdpDqIHwegYXCNdRYQYI3FWo8KD7jlr+rl+6sIDQDcWemdrYx7Y+Nx5TKR3zR2zOVKYCoFiPoDW87+6B57dt\\\\\\\/sHzAzV+drnl\\\\\\\/z+k1gkewGZeqn1tBs1ze44L9oDAvC9dSx44BrtF6IIIMmNR3TkFd4a5lEsACkry+jGMcrgn6s2JSMpTS3hSfs4wGGNZbknx7ObIcSbhx0aqMgdWHtma7\\\\\\\/alZ6PgotmwX52wQ2O3Bbf9axoHR1hUt8D+8THOrW\\\\\\\/EZaiFCVmqBwE8s6WgrnqIWfbacj\\\\\\\/IM+4MAyiiN25wEuYnIu86pEw\\\\\\\/ZJjsGVdtWpQg3RGtIykVDckYokbpf83TGrEDaaj2oTI4KQl1wbCBNAJGGTkM5PVt5hqabrJW5tw3+lSPJln5Q7eNLuGUfJIvqpPQhoDCVSJ8\\\\\\\/SQBVOf0l6SJ6OKHNlp9adTqTBzIqP+ZGa1ERVoT1rv60AUYLXVvzKsgtRwai4VFNqh2GI5QaCQCNzHcmHyCfvBXs3OgBcXxUXw\\\\\\\/iWGGjAkZgi6E2gQyf+k56k8NpdOGgwGUcJl+ObiHfJ3zfl2DgNpnI+tbKJ4rTwRYlsmw1944erqVywcC8S06ogJkgAXytd71Ig5mPS3dFhnpZyGN4B3+nSJy0Cgp9k23zl2nYPC+mNCeWAyXGQchyhvttcQTgDXxJJnWZ8L5oayNzNLwyXNdSlbm9LI8YL+Qgu167A6RyWzvV+tpqBNzDspXYpEs41EgfKi7C\\\\\\\/lj0OKecPUiF9b0\\\\\\\/Omi0tQhmd3+bZ\\\\\\\/ac1VQNjYVd47e798cSAexReQ1HPSHwgIiNtoQrfzL+N3y0IQPcL8kZXa4nTitHkTkrXCfSlcuCBk\\\\\\\/KN8hGpKKpCfV8b1GF\\\\\\\/bX06jnaPMKDRILeejrllOorX6eOyUbA+Q6dZBGqCQyL4TPObJx7Bl7+SI52+cQc3M+cyWuEcmVlhgeoMUrNVTjWt7gOMf7amsMBPta3Z4zEWiJqTBsXqq3DfKhgUDllligihqyURO6EdNAUpZkBtVljmiyGdX4B19QewP0SPG4FolIQWr7wo79du8cdqn67G3xalGSEMxOjbS+YH2vpLMaHktCFhehhTBn\\\\\\\/T+x9SmCROIO2egT1UyX7TmCmUBHHju8zjmfzMHjkwsUCfBNOxJtTnPyq5364GHQZxV+CbGAY9xt6FkWbN1pPRLjp+EgKtyKcH\\\\\\\/hdMvKDhz0NlyziOoScxJ4U+tOCXzvuy0+B4usaDHXjBUInSmDY4360SzgAFxYsB5pT\\\\\\\/3r9nbvdBr2NRZin2Ibk6G+k9g2H+NErrNlbJbqmIQqdeCnGPsCyee4P3NRvilXR0MoeP+9P3iJE7x\\\\\\\/gTdBaQPIFH0fipV13Q1kJlRMplqlCw8a4btVsSINuRNZNMlREdFrNJX5dxxqfzyC5vBOmOvcP+Htcm6o3dqkxw6IEVJEFQgQzys5qSCPRefFH4xoZgy9YW\\\\\\\/IADBlD\\\\\\\/yZlLNC55OZxjzHoJWX\\\\\\\/2uWFcwIdbETCnEHfpIVW\\\\\\\/g3VyVxdwtCcKazEtm+Q0mqQC9RHr4+pV8MoAnMp21KCIsgKAbBLLnbKIs399amvizYelbD1kasvZzAR2p+pkt4fN5ZJgQFQbuM+P+udF8dUWMzWIxpNRTJyN9aIy087fTgto0NhbU6GfUHybrwQIWDFMoRFVRvG3COAlXZAuL7bagyi6fg2ShPg76mOYIhdUXzM9A8ObVMFB\\\\\\\/dBLQX4ecb7L7RS\\\\\\\/7GrOOsbB0qlUm+vPvLsap3wZDn9YDK3iuqQ3+i+BuGZUHoG28mMlgZwCUvcVNMCz0ZiJBdEFNoa6CviCb7Xcd7xI+EQ0JmWFhnAIERLhMt2lT1VUjCYWjyHSRTixUU6BxKl91dVtcVcIXyj3HXu7gqGqm0T5P3ViSYTRTsMVOnPmszdhc9IKLg0fz\\\\\\\/7epcCW00MwQnU5AcAUtwNaA8bFoNVo4m0W4gvRYh1I8zvKN7nWY3Bjq7bTOAU4OAfbm7Xv2+bAJRHnmN\\\\\\\/EAD0TMxRe2E3q2LaxpPRkeb60wQH1DlWaY6x5\\\\\\\/ij36pVhhkGvNN5eZ7emHnKkhR9ASfhXHmQbCq8NsKJbs+o40N\\\\\\\/O1sjXcqVwD5iiSCvgIgJuUmzEf7BaLSNaSCEEmI2hdwsAUAu7RjYcdUnHR2FxIbrxVJ1w78c3v4FEILpkZFa21MkxPrUH406aFM2fkkOuOqz9rmECLv37uy2pv4iqwQuqHh4a0lJ+HhONqBPqKp\\\\\\\/idMpnjwnnWom\\\\\\\/1UUxiiredheR6hM8MFm1RrRP4AFBhEoHSjqO0slLX1zpzZZusPX0+LMQVeCjqlveMd4c26l16nxdsm7m6Iz1l8sfjHe8vggGg6sJ0KBWm24tERMxChMfkkHmF+glC+5geGr95ckCcyvV8ymXRiHN5DN7o7en26N9cAHvomx7vw0YlmoTHxpBeCJq1l22od6ko0z0ETbUb\\\\\\\/7PFxAnDGejS4HXyd\\\\\\\/DMDve920cF9q6+oqKn5L+J5Sv3iH2tI2J1ju+X9DN1\\\\\\\/NUb9JvnMNJo4Y\\\\\\\/zR8qggLOR2+Icn+FldVmLR\\\\\\\/LVVvTHOzduXCpEcaIfHgJP+1WJF5cA5HtFxOIqF1vtHZdPvcfoNCVSn+l5odKR9TJX40LeetVgpxD\\\\\\\/mbtwKcoyfDJvJlfCcDE\\\\\\\/89xMNbegyqHED++TpLtPjIHFtJhddI6ET4VzT81XsdgZgBEMpA3U+VcBGRm+wS8iDZe4laROaeX7Jqo7kwuQzSR8XKAWyP0gkJ7N0p\\\\\\\/OkrdkmRLVnN5++F8rF5QJZH4MbYfQwdrU3LM6UG\\\\\\\/rityjzXOyCTFmHs5ze9hhvmD4Hr4\\\\\\\/AZh8e9hpqHZz\\\\\\\/7+pEuWw3+MLwCniWXnpe196u59yFon\\\\\\\/HnQ0Ir+ZQ\\\\\\\/ecdlixFT5IZVd+M0S+nq57CfpEnAV9YTIai2dyn30jH0saUIZAr4azoSR7AzUxJq3aBVBtDF2nEV3gBLfYW6STY+4g5p\\\\\\\/hpH+B\\\\\\\/vHJQSQpr9ZMXRqBVgzoVu3fYLAj9l66e3ARuSACnUK+G9Z1Wn0mlCf8azrVGRKJbOBg+rvXUXow5hZvc9eMa4krj5SjEiNzgapQaYAK0HtVHKGzYn4kawmiIRLnZMxq9VkfoRI33Vf7ryBwq4JBaumAimtS+LGOuNLQkZo0G\\\\\\\/PbCKNNT\\\\\\\/szNVjnAoBnduhEsxgUj1ESakz2tXKnbXhMXduvrzYpynv2uDgMvzm2+4gMwoX3GhhVkr3+n1qVcmv7\\\\\\\/SM9LEsdoJiduOSV2h8RGcWsCbU8he5UQDtwT5idkhMiks\\\\\\\/BERO6EP57go9D\\\\\\\/fYYcJC58kh3frzzSh\\\\\\\/ZVsAIPSmyq30IPZiYnYB5TpLcj093hVEve182Ulk23AlAwVgZL1yHPcbJejL\\\\\\\/qCNAhKAhYBi\\\\\\\/CnDZzrESM5jaZmEQrf\\\\\\\/W+CnkBIN6QgsI+\\\\\\\/LnlArcFtKsaPZag9KQMMUEeTSxA4FKVoVVGDWbIZp4NJhg3dgjOlfvkIqylUAhRxDDTdQwP8eCF7LpnHQDoXHHWqc83yDsMFegUnyCCGzipeUVt4RBIb71z41aO3J3C8z411NyUvHvPV9cS6UwgtQEjPKdsSmxa8V0KZsNQVEqQ1S7kxcHSTFcxbmqkj5NIzp56wJqo+iXrcqufJObO\\\\\\\/fPxBuxh7hrqT64gLnaiakCVm0OhdSD4ni43cx0Tc8jrM5g8ib3dNhwvoJ2UVkfAUX0pnbWLv2eiT80ns9NMFhGLIbrjlxb5v9pacNYFzoPYCXYwInM2GnUgZtLVD9\\\\\\\/YV7sOozsKVhrDSmCVrDysz4oY7BEVaEcWwVd1\\\\\\\/DcWlpiNYwgBmBzonCjPJ2q9\\\\\\\/dPfh7kiJqvR2zKhxHg271WS3rwGuCIHfrpggAwZegrTULRdGRgguig+xpskGbh5LjHC0HDoV2Kg4hu9rtN8hRu5dTSplEN\\\\\\\/SpJJnO4pYbu3AacgIjI+OGCutaN2ax0CsNU3JSZX4tZYUKxz3mVZKBPu8hLCWGLepKLfpZzUW90I36fBdgtq+PslrPkBhNg\\\\\\\/u0PoOq\\\\\\\/96QDei77mi4ySoGNOXgjuwo6t6AbJxJP44THIm5\\\\\\\/69OGLVbl9qedligjQa9s2KWEFrD4XmY8\\\\\\\/zXGAhk7246ceoa9d09oQS3GqHrtbWRRZS71\\\\\\\/VEWjE47xCb54rzGHjkTecjNsQUx0Z\\\\\\\/7O2oQ3D3NQ1d96ITLlU+MN7Gdz4rPX6xezA+MZZzZmPolE9Cwlz+pARgc5bJTiMgspmztHCXiJSKbHJbLTRYL\\\\\\\/WJ1u5p4phuoo8D5GWO+qg\\\\\\\/\\\\\\\/LghmcHbx9\\\\\\\/Vm1QSQl\\\\\\\/45V4p5ndFpMoqxGIF68ehbvxB99ACBzACcnWFwW3FWcsjKfuEkNXOOYNFPj5c37sHWcHVzvBj6q+D2zpfZqFvGqFcSPkzFPJpxdCiddXA+HcHW7VY+OIbCOYiEngbp7lOTNrY\\\\\\\/LMpL8ROJvaxPYrABXflmmziFB8rMgMM9v9QwpZQzXxZRW9xbZ2jcKnzmS55VqEIZFG0oxK7gIcjznJbeZSyg3lM5RKQC7lt5qzP1spSFj7AqHaY6BL3Q8XI5A9cmEdQwAHS5ClqtR8A1jrjlFWkkb+x23NgEQJ6j1oBd7ieZPRWGQVUgY2HVYXh9cPdNCgMP2VM40ITpNE6EkQarfJoxf260s\\\\\\\/\\\\\\\/KxCtQ+cjW2z\\\\\\\/o88XhQxctBeqYoazV8dCoQkENM4EiV3sSKGz2WMfIHKqHnprxBY1oULHXRFHAWiOnsgZLrdgjgYmPzfPx4KjRCj85qn03OKikcgToRRPSBWZeVlWDA0HZlT90lNrtQPl27H8j6qE+Lz4zcRuRcHpBp80Vb1+C4bDwk\\\\\\\/pA03WKpg32Oup5yf1vXOL4McSUbhjLXGkaLq2dUoFts2aK9bh3jfp4W0Mw3erYHYYjclz7UYTzWoEFh8TEY08F9iMH7pBW8ea8h2UZyJvaW51D+Ko+l5VFiXcf2jrO+\\\\\\\/mQMgO7mSJdcaCaVAAK7QNIzp3jkfKEfxpiiP62iukg\\\\\\\/wdorg5hmsGiePbrSps0jdK6abTe9HTkI+UeZZAZOBCg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022xMdDLWDE5qzw04GkDLBK2A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ff4f46283b449f398d748b0d8dbe3a2ca85c7de769c2cb91367240fdf0eb2a26\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022fijOlcD\\\\\\\/4o5xjwCmSEULQyndtZsv46ivYK\\\\\\\/xb8L01XyZ3UFVe9eYELLe12cDQH2eb9IzL7LLR2lCJXg\\\\\\\/uMbnAQeaFfsR70iqImbBHj0lsN7WkmHgAhJVwz\\\\\\\/U0f\\\\\\\/K4\\\\\\\/QWkym4uu1lfWhoRqeIaySB09egKGaqYacg4P9QolwW1wxBk\\\\\\\/bdcbr0gJb6oEpbLzdLpM2\\\\\\\/8oweipLBCpZa1oaOvu7o37hJXFS4YJH1XWBK7SmHS3QbvgB5SGlXDb1HUK2S76nqRHofJ0muOWONtGB\\\\\\\/ObuGMxtfIwzv2xEVlOkJaPhMfgu8HoOJEIvAlvnzA+pLTNNnqJmrFT75r8AopaJbhfXPLHrlgoL\\\\\\\/GpRGgrQRjjqTCv+Xxsu7+kPOnWMVf0IWWGfFVQbNIrPcJ88qo5gVm9EYJ54H6mOa90+DDegRPfoLOjsijOme5D7pl0UC6JExeShVZ89xV4ZbGLXP35skjovRC\\\\\\\/9w9o1HDKOi7Ic60Q7mwiVk59gjrpXKZ\\\\\\\/T\\\\\\\/9spE2dl+3QgmdcK4pKB8UWR7sEg4sv4LaKxyWazfVq4FktUTiNyKDL8hE+qjiVwSiwJ\\\\\\\/afPcDgS6Uj6qizyZGCXqyA1xp2lFYoq+Kxfj8\\\\\\\/YMjkC\\\\\\\/SnFpT3gQ4bF3PI5i83kRGKyupiaVgZXBYUUFE5AgC8K4En2Pl\\\\\\\/bUlO5vXXJk9WPJvy+aakiYljm93p6g2UHHlL6RNmSW1eyNj1A3uUZz7dEbBbAv5oRlUHtBePWXUiG75\\\\\\\/ASEA1zPKiY+y1bmW7NRzuPABqB8T+Yup5j7ry1jEg3ShhJ2zMFxIj\\\\\\\/+HvyOUdXKeu0Dr7T4j2tRaojjIH\\\\\\\/62n+HNHVQPBF4lt2tTKrG\\\\\\\/qvKTzVPO4JsVYfNThn64VNnBV2u5NFCynnHARqd2UW3Yktx8D3X8upCTnJTOWpFEsWfVm96XYxuZx1BhXdt9uxE6l+CUWOOPce2faJFZ26dlspqbw2vaApFwMYfz16qbGLDNdRg0Ej6WqVh9mNHFN02eJcgh1VIj7NlA4W7PCR1Byoca0xoViBUWQ6p9kG\\\\\\\/AwbXugxYAZBysoWFNafibDvV6dfK+ZkwVaLII65Xj8\\\\\\\/TRpr67Q3uc3jttBEPPAvefhKejy\\\\\\\/btuqVgnzeNwddRvYYqptuyiY1kapXYeauPCMemW5O0mdchZA2k\\\\\\\/HP0MEbPOpxKSe53m5xuaBPotvkZjtJdVhIejjvB3DI4Xb0g2TwW1trZ\\\\\\\/W+mofCXc3i6tRdCQaTK45W+D\\\\\\\/Wc\\\\\\\/y35eOBa2ZAmlDN78wgmyYCP2FGKKb8N2eqB3oZBc9gyj87ijYjxBIQgnZlAvGcym7y\\\\\\\/GQ3r5LuDwpjZdxoOjBQQBbyB5FtiM2xhC6zHPn5Y+4QzrBm43HjOgyE7BHDdBluUDkenPko4aRNhDiDgSX7h9yNzu3FaYteTmeBiyrOXiIrjrQ+qMSWGcqOSRbWdtiOq3b2bvGKKWJNlH6xmKDMxeQPUgFeNleS+IdAgugFJKD8kJaBjI4tczNGN49nK2pLFGBdBQ\\\\\\\/NM5orMFws27pec9O\\\\\\\/1dq5k0xnskFXJ0EnCQC3pVOzr4Fy0VNgAyTJSsxT+OSqsSbRNd6w11UwYdAJff52t6LPmAYFXba0oGKVxfBuza41etLm2D544fGvskgqP9GWVwW6IdHzVQ\\\\\\\/jTiBBmZy55TppobuT7PWu0kIVBerDyITGDvNlMxpe5qWxrldld0uKKKqodpfKKooTt\\\\\\\/2DhRjOoR4SHUc89jwuRuNM\\\\\\\/zJRuKHQiswFJVmcXN0jqvUa2qxxd1\\\\\\\/ONbtl\\\\\\\/mhp8wkLNhQljrwlEEgscm2a8ZPzpg++l6ymFOj+wMOwFvKOFZQmtQwYjuP9IzaNesrySkQKDVxm49+k+j6RKACA5XsElGci7o5Qi8jdXS\\\\\\\/ihKCfhrSA1DGE4T89h6\\\\\\\/Qzg98tgMIhAHSF2Y3x2hBDEnDI5+6QQ8Woa5ygfBdhJJ4zAvwvx48D5mF+9T1Zyi9t\\\\\\\/8P\\\\\\\/jRi9dhkZt1w8bOSkxAwVduzguE2MDyFA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022iDnVzA3DkTYNhU6iT0l5Ow==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223e1a263a786a13709f8231575c4c885b878293c58c65ffc9c49da30d6e90a114\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459771 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/2/241" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BJ6s6nw2lXByKc44xfgh7h7y4vneVjWB46fR1CKbofo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/2/241" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CIFBPtFLAlIRQ5YLzaN6xGLyK3BhbEkH70ReWBfEMWs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/2/235" title="Segregation of a Spontaneous Klrd1 (CD94) Mutation in DBA/2 Mouse Substrains" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/2/253" title="Plant Genetic Archaeology: Whole-Genome Sequencing Reveals the Pedigree of a Classical Trisomic Line" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459771" id="node459771" data-pisa="ggg;5/2/241" data-pisa-master="ggg;g3.114.015438" data-apath="/ggg/5/2/241.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density Linkage Map for <em>Astyanax mexicanus</em> Using Genotyping-by-Sequencing Technology</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Brian M. Carlson</span>, <span class="highwire-citation-author" data-delta="1">Samuel W. Onusko</span> and <span class="highwire-citation-author" data-delta="2">Joshua B. Gross</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">241-251;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015438 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459771" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Brian M. Carlson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, University of Cincinnati, Cincinnati, Ohio 45221</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BM.%2BCarlson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlson%20BM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BM.%2BCarlson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Samuel W. Onusko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, University of Cincinnati, Cincinnati, Ohio 45221</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samuel%2BW.%2BOnusko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Onusko%20SW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASamuel%2BW.%2BOnusko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joshua B. Gross </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, University of Cincinnati, Cincinnati, Ohio 45221</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BB.%2BGross%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gross%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoshua%2BB.%2BGross%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:grossja@ucmail.uc.edu" class="" data-icon-position="" data-hide-link-title="0">grossja@ucmail.uc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/2/241" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459771" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459771/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/2/241.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459771" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459771/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/2/241.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459771" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459771/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/2/241.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459771" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459771/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/2/241.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459771" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459771/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The Mexican tetra, <em>Astyanax mexicanus</em>, is a unique model system consisting of cave-adapted and surface-dwelling morphotypes that diverged &gt;1 million years (My) ago. This remarkable natural experiment has enabled powerful genetic analyses of cave adaptation. Here, we describe the application of next-generation sequencing technology to the creation of a high-density linkage map. Our map comprises more than 2200 markers populating 25 linkage groups constructed from genotypic data generated from a single genotyping-by-sequencing project. We leveraged emergent genomic and transcriptomic resources to anchor hundreds of anonymous <em>Astyanax</em> markers to the genome of the zebrafish (<em>Danio rerio</em>), the most closely related model organism to our study species. This facilitated the identification of 784 distinct connections between our linkage map and the <em>Danio rerio</em> genome, highlighting several regions of conserved genomic architecture between the two species despite ∼150 My of divergence. Using a Mendelian cave-associated trait as a proof-of-principle, we successfully recovered the genomic position of the albinism locus near the gene <em>Oca2</em>. Further, our map successfully informed the positions of unplaced <em>Astyanax</em> genomic scaffolds within particular linkage groups. This ability to identify the relative location, orientation, and linear order of unaligned genomic scaffolds will facilitate ongoing efforts to improve on the current early draft and assemble future versions of the <em>Astyanax</em> physical genome. Moreover, this improved linkage map will enable higher-resolution genetic analyses and catalyze the discovery of the genetic basis for cave-associated phenotypes.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/next-generation-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-next-generation-sequencing" rel="nofollow">next-generation sequencing</a></li><li class="kwd"><a href="/keyword/qtl-analysis" class="hw-term hw-article-keyword hw-article-keyword-qtl-analysis" rel="nofollow">QTL analysis</a></li><li class="kwd"><a href="/keyword/blind-mexican-cave-tetra" class="hw-term hw-article-keyword hw-article-keyword-blind-mexican-cave-tetra" rel="nofollow">blind Mexican cave tetra</a></li><li class="kwd"><a href="/keyword/regressive-phenotypic-evolution" class="hw-term hw-article-keyword hw-article-keyword-regressive-phenotypic-evolution" rel="nofollow">regressive phenotypic evolution</a></li></ul><p id="p-3">The blind Mexican cave tetra is a powerful system for understanding the evolutionary mechanisms governing regressive phenotypes. These animals were discovered in 1936 and initially were assigned to a new genus—<em>Anoptichthys</em> (“bony fish without eyes”) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hubbs and Innes 1936</a>). Breeding studies in the 1940s led to the discovery of viable hybrid offspring resulting from crosses between the (derived) blind cave-dwelling forms and (ancestral) surface-dwelling forms from the same geographical region of northeast Mexico (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Breder 1943a</a>, <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">b</a>). Both morphotypes are now regarded as members of the same (or a closely related) species, <em>Astyanax mexicanus</em>. This system has spurred more than half a century of comparative research (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Şadoǧlu 1956</a>) focusing on unresolved problems in evolution (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jeffery 2001</a>), development (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Pottin <em>et al.</em> 2011</a>), genetics (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Schemmel 1980</a>), physiology (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Salin <em>et al.</em> 2010</a>), and behavior (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Burchards <em>et al.</em> 1985</a>).</p><p id="p-4">Classical and quantitative genetic approaches have provided clear evidence that many troglomorphic (cave-associated) phenotypes evolved through heritable genetic changes. These studies centered on both Mendelian and complex phenotypes, including eye regression (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Yamamoto <em>et al.</em> 2004</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Protas <em>et al.</em> 2007</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Yoshizawa <em>et al.</em> 2012</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> 2013</a>), feeding-related behaviors (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Schemmel 1980</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Yoshizawa <em>et al.</em> 2012</a>), sleep loss (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Duboué <em>et al.</em> 2011</a>), schooling behavior (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kowalko <em>et al.</em> 2013</a>), pigmentation loss (reviewed in <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jeffery 2009</a>), and intraspecific aggression (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Elipot <em>et al.</em> 2013</a>). QTL studies have identified candidate genes mediating a variety of these traits, such as retinal degeneration (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> 2013</a>), rib number, eye size (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> 2008</a>), albinism (<em>Oca2</em>) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Protas <em>et al.</em> 2006</a>), and the <em>brown</em> phenotype (<em>Mc1r</em>) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gross <em>et al.</em> 2009</a>).</p><p id="p-5">Genomic resources for this model system, however, have historically been limited. The first linkage map was calculated based on recombination frequencies of an experimental F<sub>1</sub> × Pachón cave backcross pedigree using markers generated from random amplified polymorphic DNA (RAPD) fingerprinting (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Borowsky and Wilkens 2002</a>). This map was supplanted by a higher-resolution map using more individuals and markers composed of polymorphic microsatellites identified using CA<sub>N</sub> dinucleotide repeats (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Protas <em>et al.</em> 2006</a>). Using this second-generation linkage map, <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Protas <em>et al.</em> (2008)</a> discovered a genetic basis for several cave-associated phenotypic changes including pigmentation regression, reduced rib numbers, slower weight loss, and increased chemical sensitivity. Early comparative genomic analyses utilizing this map first demonstrated extensive synteny conserved between <em>Astyanax</em> and <em>Danio rerio</em>, despite ∼150 My of divergence (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> 2008</a>). The first next-generation sequencing (NGS)-based linkage map using restriction-associated DNA sequencing (RAD-seq) technology was published by <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> (2013)</a>. This map, comprising 698 markers on 25 linkage groups, strengthened the evidence for vast regions of synteny between the genomes of <em>Astyanax</em> and zebrafish and identified several critical loci associated with retinal degeneration (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> 2013</a>).</p><p id="p-6">Here, we present the most dense, comprehensive linkage map to date using genotyping-by-sequencing (GBS) technology. This technology enables accurate and high-throughput collection of massive amounts of sequence data (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Davey <em>et al.</em> 2011</a>), including thousands of single-nucleotide polymorphisms (SNPs) segregating between cave-dwelling and surface-dwelling morphs. GBS utilizes deep Illumina sequencing of restriction enzyme-nicked genomic DNA libraries that are uniquely barcoded for each member of an experimental pedigree. This technique is optimized to avoid inclusion of repetitive portions of the genome and is extremely specific and highly reproducible (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> 2011</a>). Fish are well-represented among studies using GBS and other RAD-seq–based methodologies (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rowe <em>et al.</em> 2011</a>). However, a majority of GBS studies in fish have focused on species of commercial (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Everett <em>et al.</em> 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Houston <em>et al.</em> 2012</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Li <em>et al.</em> 2014</a>) or conservational concern (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hecht <em>et al.</em> 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Ogden <em>et al.</em> 2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hess <em>et al.</em> 2014</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Larson <em>et al.</em> 2014</a>). Here, we adapted this technology to construct a high-density linkage map for evolutionary and developmental studies in our emerging model system. The resulting linkage map will enable higher-resolution genomic studies and inform the assignment of chromosomal builds for the ongoing <em>Astyanax</em> genome sequencing project (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">McGaugh <em>et al.</em> 2014</a>).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Pedigree, husbandry, and genomic DNA isolation</h3><p id="p-7">Linkage mapping and QTL studies were performed using genotypic and phenotypic data obtained from two separate F<sub>2</sub> hybrid mapping populations (n = 129; n = 41) bred from a male surface fish and female cavefish from the Pachón cave. In addition, surface (n = 4), Pachón cave (n = 4), and surface × Pachón F<sub>1</sub> hybrid (n = 4) specimens were used to evaluate and code GBS markers for use with JoinMap software (v. 4.1; Kyazma; see below), but were not included in linkage mapping calculations. Parental specimens belonged to laboratory populations originally sourced from the El Abra region of northeastern Mexico and all fish used were generously provided to our laboratory by Dr. Richard Borowsky (New York University). All live fish used in this study were maintained as previously described (see <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gross <em>et al.</em> 2013</a>). Every individual from the “Asty66” F<sub>2</sub> population (n = 129) was individually reared in a 1-liter tank. All phenotypic data from the “Asty12” F<sub>2</sub> population (n = 41) were obtained from paraformaldehyde-preserved specimens.</p></div><div id="sec-3" class="subsection"><h3>Genotyping-by-sequencing</h3><p id="p-8">Genomic DNA was extracted from caudal tail fin tissue of live surface, cave, and F<sub>1</sub> and F<sub>2</sub> hybrid <em>Astyanax mexicanus</em> specimens using the DNeasy Blood and Tissue Kit (Qiagen) as previously described (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gross <em>et al.</em> 2013</a>). Twenty genomic samples were digested with <em>Eco</em>RI, subjected to gel electrophoresis and imaged to verify that sample quality, concentration, and restriction fragment size distributions were suitable for use in downstream analyses. DNA samples were then pipetted into individual wells of 96-well plates and diluted to a final volume of 30 µl (100 ng/µl). Samples were processed by the Institute for Genomic Diversity (Cornell University), where genomic libraries were constructed and GBS was performed as described elsewhere (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lu <em>et al.</em> 2013</a>).</p></div><div id="sec-4" class="subsection"><h3>GBS marker selection</h3><p id="p-9">Genotypes for each of 7956 GBS markers (each consisting of a single SNP in a 64-bp-long sequence fragment) were screened in cave and surface (parental) forms to assign the morphotypic origin of each allele. F<sub>1</sub> individuals were then evaluated to confirm heterozygosity at each locus. The morphotypic origin of each allele was assigned by consensus—if three or more (out of four) surface or cave individuals had the identical nucleotide at a particular locus, then the genotype was assigned to the consensus parental population. Likewise, a true “hybrid” genotype was assigned if three or more F<sub>1</sub> individuals harbored the same heterozygous condition (<em>e.g.</em>, M, R, S, W, Y, K SNP code) at a given locus. Those genotypes with an ambiguous morphotypic origin were denoted “NA.”</p><p id="p-10">Markers were then screened for suitability in linkage calculations. Markers were deemed unsuitable and discarded from further analysis if neither parental genotype could be assigned (<em>i.e.</em>, both the surface and cave genotypes were scored “NA”) or if the assigned surface and cave genotypes were identical; 6006 genomic markers were deemed suitable and prepared for linkage map calculation using the “cross-pollination” (CP) segregation coding used in JoinMap. At this stage, 107 markers were found to be uninformative (<em>i.e.</em>, a single genotype was shared by all F<sub>2</sub> individuals) and discarded from further analysis. We screened the remaining set (n = 5899) to identify markers failing to conform to predicted genotypic ratios (<em>e.g.</em>, 1:2:1 ratios across the entire pedigree); 2896 markers demonstrated a χ<sup>2</sup> value more than 50, implying significant departure from the predicted genotype ratio and were discarded from further analysis. Our final GBS marker set included 3003 markers evaluated in 170 F<sub>2</sub> individuals.</p></div><div id="sec-5" class="subsection"><h3>Linkage map construction and QTL analysis</h3><p id="p-11">Linkage map calculations were performed using JoinMap (v.4.1, Kyazma). Our workflow used program default settings, with the following exceptions: the maximum grouping independence LOD value was set to 50.0; linkage groups were calculated using regression mapping; and linkage mapping was performed using the Kosambi method (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kosambi 1943</a>). Linkage groups were assigned based on independence LOD scores. We increased the maximum grouping independence LOD value to 50.0, because the default value of 10.0 did not allow sufficient subdivision of our data into an appropriate number of groups. Initial groupings identified 29 groups populated with between 10 and 225 markers, with independence LOD scores ranging from 7.0 to 21.0. These groups were then processed for formal mapping calculations.</p><p id="p-12">The first round of mapping produced 28 linkage groups comprising a total map length of 2956 cM. At this stage, one linkage group (comprising 10 markers, independence LOD = 19.0) failed to assemble into a consolidated group and was therefore eliminated from further analysis. The remaining individual linkage groups ranged in length from 27.25 to 187.46 cM, containing between 10 and 225 markers with an average intermarker distance between 0.51 and 6.40 cM. After this initial round of mapping, we further screened existing linkages to target the most optimal 25 groups (<em>Astyanax mexicanus</em> has karyotypic number of 25; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kirby <em>et al.</em> 1977</a>) and reduce the average intermarker distance to a target of ∼1 cM. Accordingly, nine groups (10 ≤ n ≤ 45 markers) were removed because of low marker number and/or unusually high average intermarker distance. The five largest groups (154 ≤ n ≤ 225 markers) were then subdivided at the lowest independence LOD value resulting in two linkage groups comprising 20 or more markers. Throughout mapping, we limited the inflation of the overall map length by eliminating certain markers sparsely populating distal ends of otherwise densely populated linkage groups. This resulted in size reduction of the five longest remaining linkage groups (142.041 ≤ n ≤ 187.458 cM) by splitting them at the lowest independence LOD score at which a group (comprising 10 or more markers) was separated. In these cases, the larger of the two resulting groups was retained. The resulting 25 linkage groups (independence LOD scores 10.0 ≤ n ≤ 24.0) were subjected to additional mapping. Groupings of markers eliminated during this or a subsequent round of mapping were excluded from further analysis.</p><p id="p-13">The second round of mapping produced a 2556.6-cM linkage map composed of 25 linkage groups, each consisting of 25 to 171 markers, ranging in length from 31.18 to 142.78 cM, with mean intermarker distances ranging from 0.47 to 3.66 cM. Using the same criteria described above, an additional group (comprising 25 markers and an average intermarker distance of 3.658) was eliminated. A densely populated group with a high independence LOD (153 markers; 135.73 cM; independence LOD of 24.0) was split and 12 linkage groups (103.982 ≤ n ≤ 142.783 cM) were trimmed.</p><p id="p-14">The result of this third and final round of mapping was then analyzed for genomic synteny shared between <em>Astyanax mexicanus</em> and the zebrafish genome and used to map albinism as a proof of concept. Albinism was scored as a binary phenotype wherein presence of melanin (0) or absence of melanin (1) was assigned to each of the members of our experimental F<sub>2</sub> pedigrees. All QTL analyses of albinism were conducted using R/qtl (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Broman <em>et al.</em> 2003</a>) run for each of three scan-one mapping methods: marker regression (MR), expect maximum (EM), and Haley-Knott (HK), according to the methodology in <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gross <em>et al.</em> (2014)</a>.</p></div><div id="sec-6" class="subsection"><h3>Assignment of genomic synteny between the <em>Astyanax mexicanus</em> and <em>Danio rerio</em> genomes</h3><p id="p-15">At present, physical genome resources for <em>Astyanax mexicanus</em> are in their early draft phases (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">McGaugh <em>et al.</em> 2014</a>). Therefore, we anchored our GBS-based linkage map to the physical genome of the most closely related fish model system with comprehensive resources, <em>Danio rerio</em>. <em>Astyanax</em> and <em>Danio</em> are members of the superorder Ostariophysii, which diverged ∼150 My ago (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Briggs 2005</a>). Despite this distance, significant genome-level synteny remains between these species (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> 2008</a>; <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> 2013</a>). Our GBS marker set was derived from endonuclease restriction site-based libraries and was therefore anonymous. We first identified all GBS markers that could be directly localized to a conserved region in the <em>D. rerio</em> genome. Accordingly, we performed BLAST searches of the 64-bp sequences comprising our marker sequences directly against the <em>Danio</em> genome (downloaded from the Ensembl genome browser; <a href="http://www.ensembl.org">www.ensembl.org</a>).</p><p id="p-16">These and all subsequent searches were performed using a BLASTN script run on the Ohio Supercomputing Cluster (OSC). All quality control defaults, including an expect value (e-value) cutoff of 10, were maintained. The script permitted the return of alignments between a given 64-bp marker sequence and regions of up to three distinct targets (<em>e.g.</em>, three different <em>Danio rerio</em> chromosomes). In cases where a single marker sequence aligned multiple times with the same target, raw results were filtered by e-value, retaining the lowest e-value alignment for each marker-target pairing. There are two 64-bp sequences for each GBS marker, differing only in that each contains one of the two alleles for the imbedded SNP. Because both of these sequences were included when BLAST searches using the 64-bp marker sequences were conducted, this filtering step also served to collapse these results into a single set of results, retaining the better of the two alignments for each marker-target pairing.</p><p id="p-17">In some instances, a single queried sequence returned alignments with multiple targets. These instances were resolved by sorting results to determine the “top hit,” which was defined as having the lowest e-value and highest percent identity (in case of an e-value tie) to a particular target sequence. If the target of the top hit (<em>i.e.</em>, the alignment with the lowest e-value) for a given marker sequence agreed with the target reported for one or more other markers on the same linkage group that returned only a single robust hit, then the top hit for the marker in question was considered “supported” and retained. If the top hit was not supported in this fashion but a different BLAST result was, then the latter “not top hit, supported” result was retained instead. If none of the results returned for a marker sequence were supported, then the top hit was retained, despite the lack of support. In rare cases, there was no way to resolve which result should be retained. Results for these “unresolved” markers were discarded.</p><p id="p-18">When using BLAST searches to align our 64-bp markers directly to the <em>Danio rerio</em> genome returned relatively few high-quality hits, we developed a strategy whereby we first aligned our GBS marker sequences to the <em>Astyanax mexicanus</em> genome and transcriptome data. This information was then used to identify homologous <em>Danio</em> genomic and transcriptomic sequences. Current genomic resources in <em>Astyanax</em> consist of &gt;10,000 unplaced genomic scaffolds (Bioproject PRJNA89115). The collective sequence data for the <em>Astyanax</em> genome (GenBank Assembly ID GCA_000372685) were downloaded from Ensembl, along with the transcript sequences for 23,042 predicted genes. BLAST searches were used to determine putative locations for the 64-bp sequences of the 2235 GBS markers comprising our final linkage map in the <em>Astyanax</em> genomic and transcriptomic data sets. After initial searches were performed as described, ∼2000-bp stretches of genomic sequence harboring our 64-bp GBS marker sequences were aligned with the <em>Danio</em> genome. Similarly, full sequences for predicted <em>Astyanax</em> transcripts to which our GBS markers aligned were queried against a <em>Danio</em> cDNA database downloaded from Ensembl. Both data sets were then filtered (as described), yielding a single “best” <em>Danio</em> alignment for each informative query. This process enabled us to leverage draft genomic and transcriptomic data to augment the amount of sequence information associated with our 64-bp GBS markers and to identify homologous genomic positions in a well-characterized model system.</p><p id="p-19">After BLAST searches using the direct, genomic, and transcriptomic alignment methods were completed, the filtered results for all three were combined. When multiple methods returned results for the same marker, a single result was chosen and retained using the same filtering process applied to single data sets (above). The Circos program (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Krzywinski <em>et al.</em> 2009</a>) was used to visualize comparative genomic positions between our linkage map and the <em>Astyanax</em> and <em>Danio rerio</em> genomes.</p></div><div id="sec-7" class="subsection"><h3>Position identification for previously published markers in the Astyanax genome</h3><p id="p-20">Previous maps published by <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> (2008)</a> and <a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> (2013)</a> were used to examine synteny between <em>Astyanax</em> and <em>Danio</em> and to provide a comparison between this study and prior studies. These authors provided predicted <em>Danio</em> positions for the markers used in their analyses, but positions in the draft <em>Astyanax</em> genome were not determined because these studies predated available genomic resources. Our GBS-based map does not share any markers with the two previous maps, so it was necessary to identify positions of previously generated markers in <em>Astyanax</em> to enable comparison between previous mapping efforts and those described here. Accordingly, microsatellite and RAD-seq marker sequences (where available) for each data set were aligned with <em>Astyanax</em> genome scaffolds using the same BLAST and filtering protocols used for our own data (above). Both previous studies included markers located in candidate genes. The locations of <em>Astyanax</em> orthologs of these candidate genes were identified using Ensembl.</p><p id="p-21">GBS marker sequences and genotyping data are available from the Dryad Digital Repository (<a href="http://dx.doi.org/10.5061/dryad.6s718">http://dx.doi.org/10.5061/dryad.6s718</a>).</p></div></div><div class="section results-discussion" id="sec-8"><h2 class="">Results and Discussion</h2><div id="sec-9" class="subsection"><h3>A high-density linkage map in <em>Astyanax mexicanus</em></h3><p id="p-22">Here, we present a dense linkage map for <em>Astyanax mexicanus</em> generated using genotyping-by-sequencing technology. This map was created using 170 experimental F<sub>2</sub> individuals based on genotypic information for 3003 loci. The construction of this map ultimately yielded 25 linkage groups (the karyotypic number for <em>Astyanax</em>) comprising 2235 markers spanning 2110.7 cM, with an average intermarker distance of 1.052 cM (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>, Supporting Information, <a href="http://www.g3journal.org/content/suppl/2014/12/17/g3.114.015438.DC1/015438SI.pdf">Table S1</a>). The strategy we used enables application of powerful, cost-effective, next-generation sequencing technology to facilitate genetic studies in emerging or nonmodel systems.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/2/241/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A GBS-based linkage map in the Mexican cave tetra, Astyanax mexicanus. We analyzed 3003 SNP markers in 170 individuals using genotyping-by-sequencing technology. This linkage map consists of 2235 markers in 25 linkage groups (A. mexicanus karyotype number = 25), spanning a total distance of 2110.7 cM (mean intermarker distance = 1.052 cM). Astyanax linkage group 8 (red box) illustrates typical marker density observed in most groups. This group consists of 52 GBS markers spanning 67.061 cM with a mean intermarker distance of 1.315 cM." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-291836724" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A GBS-based linkage map in the Mexican cave tetra, &lt;em&gt;Astyanax mexicanus&lt;/em&gt;. We analyzed 3003 SNP markers in 170 individuals using genotyping-by-sequencing technology. This linkage map consists of 2235 markers in 25 linkage groups (&lt;em&gt;A. mexicanus&lt;/em&gt; karyotype number = 25), spanning a total distance of 2110.7 cM (mean intermarker distance = 1.052 cM). &lt;em&gt;Astyanax&lt;/em&gt; linkage group 8 (red box) illustrates typical marker density observed in most groups. This group consists of 52 GBS markers spanning 67.061 cM with a mean intermarker distance of 1.315 cM.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/2/241/F1.medium.gif" width="282" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/2/241/F1.medium.gif" width="282" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/2/241/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/2/241/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465700" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-23" class="first-child">A GBS-based linkage map in the Mexican cave tetra, <em>Astyanax mexicanus</em>. We analyzed 3003 SNP markers in 170 individuals using genotyping-by-sequencing technology. This linkage map consists of 2235 markers in 25 linkage groups (<em>A. mexicanus</em> karyotype number = 25), spanning a total distance of 2110.7 cM (mean intermarker distance = 1.052 cM). <em>Astyanax</em> linkage group 8 (red box) illustrates typical marker density observed in most groups. This group consists of 52 GBS markers spanning 67.061 cM with a mean intermarker distance of 1.315 cM.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">Cross-genera marker identification was greatly facilitated by alignment first to draft <em>Astyanax</em> genomic and transcriptomic resources, followed by searches of the homologous sequences in <em>Danio</em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2, A–C</a>). Although direct BLAST searches of our 64-bp GBS marker sequences returned results for few of the markers in our map (1.2%), success rates were much higher when using <em>Astyanax</em> genomic (26.5%) or transcriptomic (13.3%) sequences as an intermediary (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Each <em>Danio rerio</em> chromosome was represented in our comparative genomic analysis, with <em>Astyanax</em> linkage groups containing 14–52 markers (average = 30.84) comprising ancient syntenic blocks shared with each of 25 zebrafish chromosomes (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2D</a>). Of the 2235 GBS markers that constitute our linkage map, 784 marker sequences (35.1%) were successfully identified in the <em>Danio rerio</em> genome (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/2/241/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Short GBS sequences identify syntenic stretches between two Ostariophysian freshwater fish species. To reveal syntenic regions between Astyanax mexicanus and Danio rerio, we first identified stretches of the Danio genome harboring homologous sequences to our anonymous GBS marker sequences (A). Individual 64-bp sequences for the 2235 GBS markers in our linkage map were compared with the Danio genome both directly and by first aligning to larger Astyanax genomic scaffolds and predicted gene transcripts (B), followed by alignment of some or all of the larger sequence to the Danio genome based on BLAST sequence analysis (C). This resulted in identification of homologous sequences for 784 Astyanax GBS markers within the Danio genome. The markers shared between Danio chromosomes and Astyanax linkage groups are represented using an Oxford plot (D)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-291836724" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Short GBS sequences identify syntenic stretches between two Ostariophysian freshwater fish species. To reveal syntenic regions between &lt;em&gt;Astyanax mexicanus&lt;/em&gt; and &lt;em&gt;Danio rerio&lt;/em&gt;, we first identified stretches of the &lt;em&gt;Danio&lt;/em&gt; genome harboring homologous sequences to our anonymous GBS marker sequences (A). Individual 64-bp sequences for the 2235 GBS markers in our linkage map were compared with the &lt;em&gt;Danio&lt;/em&gt; genome both directly and by first aligning to larger &lt;em&gt;Astyanax&lt;/em&gt; genomic scaffolds and predicted gene transcripts (B), followed by alignment of some or all of the larger sequence to the &lt;em&gt;Danio&lt;/em&gt; genome based on BLAST sequence analysis (C). This resulted in identification of homologous sequences for 784 &lt;em&gt;Astyanax&lt;/em&gt; GBS markers within the &lt;em&gt;Danio&lt;/em&gt; genome. The markers shared between &lt;em&gt;Danio&lt;/em&gt; chromosomes and &lt;em&gt;Astyanax&lt;/em&gt; linkage groups are represented using an Oxford plot (D).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/2/241/F2.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/2/241/F2.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/2/241/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/2/241/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465631" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-25" class="first-child">Short GBS sequences identify syntenic stretches between two Ostariophysian freshwater fish species. To reveal syntenic regions between <em>Astyanax mexicanus</em> and <em>Danio rerio</em>, we first identified stretches of the <em>Danio</em> genome harboring homologous sequences to our anonymous GBS marker sequences (A). Individual 64-bp sequences for the 2235 GBS markers in our linkage map were compared with the <em>Danio</em> genome both directly and by first aligning to larger <em>Astyanax</em> genomic scaffolds and predicted gene transcripts (B), followed by alignment of some or all of the larger sequence to the <em>Danio</em> genome based on BLAST sequence analysis (C). This resulted in identification of homologous sequences for 784 <em>Astyanax</em> GBS markers within the <em>Danio</em> genome. The markers shared between <em>Danio</em> chromosomes and <em>Astyanax</em> linkage groups are represented using an Oxford plot (D).</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465658/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465658/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of BLAST results and identification of markers used in <em>Astyanax</em>-to-<em>Danio</em> syntenic analysis</span><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/2/241/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Whole-genome synteny between Astyanax and Danio and a proof-of-concept analysis of albinism. Syntenic links between our GBS map and the Danio genome were visualized using Circos (A). Each line represents a connection between the position of a particular marker in our linkage map (black; scale in cM) and a homologous sequence in Danio (various colors; scale in Mb). We scored albinism, a Mendelian trait associated with the Oca2 gene in cave-dwelling Astyanax (C), and performed QTL analysis using R/qtl. Each of three mapping methods (MR in red; EM in blue; HK in black) revealed peak LOD scores of &#x223C;20 (LOD at 0.001&#x3B1; threshold = 6.75) at, or adjacent to, GBS marker TP71406 on Astyanax linkage group 13 (B). Homologous sequences to TP71406 and several of its neighbors on Astyanax linkage group 13 are clustered together on Danio chromosome 6 near the Oca2 gene. A phenotypic effect plot for marker TP71406 revealed the predicted association between the homozygous &#x201C;cave&#x201D; condition (genotype CC) and albinism in F2 individuals (D)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-291836724" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Whole-genome synteny between &lt;em&gt;Astyanax&lt;/em&gt; and &lt;em&gt;Danio&lt;/em&gt; and a proof-of-concept analysis of albinism. Syntenic links between our GBS map and the &lt;em&gt;Danio&lt;/em&gt; genome were visualized using Circos (A). Each line represents a connection between the position of a particular marker in our linkage map (black; scale in cM) and a homologous sequence in &lt;em&gt;Danio&lt;/em&gt; (various colors; scale in Mb). We scored albinism, a Mendelian trait associated with the &lt;em&gt;Oca2&lt;/em&gt; gene in cave-dwelling &lt;em&gt;Astyanax&lt;/em&gt; (C), and performed QTL analysis using R/qtl. Each of three mapping methods (MR in red; EM in blue; HK in black) revealed peak LOD scores of &#x223C;20 (LOD at 0.001&#x3B1; threshold = 6.75) at, or adjacent to, GBS marker TP71406 on &lt;em&gt;Astyanax&lt;/em&gt; linkage group 13 (B). Homologous sequences to TP71406 and several of its neighbors on &lt;em&gt;Astyanax&lt;/em&gt; linkage group 13 are clustered together on &lt;em&gt;Danio&lt;/em&gt; chromosome 6 near the &lt;em&gt;Oca2&lt;/em&gt; gene. A phenotypic effect plot for marker TP71406 revealed the predicted association between the homozygous &#x201C;cave&#x201D; condition (genotype &lt;em&gt;CC&lt;/em&gt;) and albinism in F&lt;sub&gt;2&lt;/sub&gt; individuals (D).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/2/241/F3.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/5/2/241/F3.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/2/241/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/2/241/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465622" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-31" class="first-child">Whole-genome synteny between <em>Astyanax</em> and <em>Danio</em> and a proof-of-concept analysis of albinism. Syntenic links between our GBS map and the <em>Danio</em> genome were visualized using Circos (A). Each line represents a connection between the position of a particular marker in our linkage map (black; scale in cM) and a homologous sequence in <em>Danio</em> (various colors; scale in Mb). We scored albinism, a Mendelian trait associated with the <em>Oca2</em> gene in cave-dwelling <em>Astyanax</em> (C), and performed QTL analysis using R/qtl. Each of three mapping methods (MR in red; EM in blue; HK in black) revealed peak LOD scores of ∼20 (LOD at 0.001α threshold = 6.75) at, or adjacent to, GBS marker TP71406 on <em>Astyanax</em> linkage group 13 (B). Homologous sequences to TP71406 and several of its neighbors on <em>Astyanax</em> linkage group 13 are clustered together on <em>Danio</em> chromosome 6 near the <em>Oca2</em> gene. A phenotypic effect plot for marker TP71406 revealed the predicted association between the homozygous “cave” condition (genotype <em>CC</em>) and albinism in F<sub>2</sub> individuals (D).</p><div class="sb-div caption-clear"></div></div></div><p id="p-32">We performed a proof-of-concept analysis using the albinism phenotype to validate the utility of our GBS-based linkage map (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, B–D</a>). Accordingly, we mapped the monogenic trait of albinism using the R/qtl package to evaluate phenotypic and genotypic data for the 170 F<sub>2</sub> hybrid individuals used to construct our map. We identified a peak LOD score of 20.68 on linkage group 13, associated with marker TP71406. This marker and the surrounding region form a syntenic block within a region of <em>Danio rerio</em> chromosome 6. This genomic interval contains the gene <em>Oca2</em>, previously demonstrated to be the causative locus for albinism in <em>Astyanax</em> cavefish. This supports previous findings of conserved synteny inclusive of significant portions of chromosome 6 in <em>Danio</em> (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> 2008</a>; <a id="xref-ref-33-7" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> 2013</a>) and implies our densely populated map will enable future QTL studies of trait evolution in <em>Astyanax</em>.</p></div><div id="sec-10" class="subsection"><h3>Conserved genomic architecture between <em>Astyanax</em> and <em>Danio</em> based on GBS markers</h3><p id="p-33">Our analysis of synteny between <em>Astyanax</em> and <em>Danio</em> illustrates variable levels of genomic conservation across linkage groups (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2D</a>, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>). Certain chromosomes, for instance, appear to have changed little since the divergence of these teleost species (<em>e.g.</em>, <em>Danio</em> chromosomes 6 and 23 in <em>Astyanax</em> linkage groups 13 and 15, respectively). However, other <em>Danio</em> chromosomes appear scattered across several linkage groups, without a consensus representation for any particular group (<em>e.g.</em>, <em>Danio</em> chromosomes 2 and 5).</p><p id="p-34">We believe these findings most likely reflect genomic rearrangements that have occurred since the divergence of these two species. However, this finding could also be attributed to low representation of particular <em>Danio</em> chromosomes within our GBS marker set. We examined this possibility by assessing the number of syntenic links between our GBS-based linkage map and each <em>Danio</em> chromosome. We would anticipate that longer chromosomes would naturally harbor more syntenic links. Values were therefore expressed as a ratio of syntenic links per megabase (mean = 0.59 GBS markers/Mb). Although the mean value for chromosomes that were not strongly represented on any particular linkage group in our map (<em>i.e.</em>, had fewer than 10 syntenic links with each linkage group, mean = 0.52 GBS markers/Mb, n = 8) was lower than that for chromosomes demonstrating strong synteny with a particular linkage group (mean = 0.61 GBS markers/Mb, n = 17), there was not a significant difference between the two groups (t<sub>23</sub> = 0.5809, <em>P</em> = 0.5670). This leads us to conclude that, although representation of particular chromosomes in our data set may be a contributing factor, it is unlikely that this is the primary cause of the differences in chromosomal representation patterns observed.</p><p id="p-35">Alternatively, BLAST results for <em>Astyanax</em> GBS markers (or the larger <em>Astyanax</em> sequences to which they were aligned) may include paralogous genes or otherwise ambiguous results that could lead to erroneous links between a linkage group and a <em>Danio</em> chromosome. Although we cannot rule out this possibility, we feel our strategy prioritized the “optimal” BLAST result among multiple hits for a single marker leading to alignments that agree with nearby unambiguous results (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). As a result, of the 784 markers in our map for which a putative <em>Danio</em> position was determined, only 15.9% (n = 125) of final calls were unsupported by the results for other markers belonging to the same linkage group (<a href="http://www.g3journal.org/content/suppl/2014/12/17/g3.114.015438.DC1/015438SI.pdf">Table S1</a>). Given that chromosomal arrangements have occurred over the ∼150 My since divergence, we feel our systematic approach best identifies paralogous genes and other potential sources of ambiguity.</p><p id="p-36">Erroneous or ambiguous genotyping data may have led to incorrect assignment of “cave” and “surface” alleles for particular markers. These erroneous assignments could have adversely affected downstream efforts, causing markers to be incorrectly placed during the grouping and/or mapping stages of linkage map construction. All efforts were made to ensure allelic identification was accurate using a stringent screening process (see <em>Materials and Methods</em>); however, we relied on a relatively small number of cave, surface, and F<sub>1</sub> hybrid individuals (n = 4 each) to identify parental allelic origin. Similarly, the relatively small number of meiotic events represented by the 170 F<sub>2</sub> individuals may have resulted in linkage map inaccuracies (<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> 2008</a>). Future comparisons between the map we present here and a finished-grade <em>Astyanax</em> genome will clarify if regions lacking synteny between <em>Astyanax</em> and <em>Danio</em> are attributable to errors in our linkage map or genomic rearrangements that have occurred since the divergence of these taxa.</p></div><div id="sec-11" class="subsection"><h3>Unplaced <em>Astyanax</em> genome scaffolds can be anchored to our new linkage map</h3><p id="p-37">Positional locations in the current draft of the <em>Astyanax</em> genome were established for 93.6% (n = 2091) of the 2235 GBS markers present in our map. These markers were localized to positions spread across 598 different <em>Astyanax</em> genome scaffolds. Our 25 <em>Astyanax</em> linkage groups contain markers representing between 12 (linkage groups 8 and 22) and 55 (linkage group 3) genome scaffolds each, with a map-wide average of 27.64 scaffolds/linkage group. Individual genome scaffolds contained between 1 and 31 GBS markers appearing in our final map, with an average of 3.50 markers per scaffold. GBS markers located on the same genomic scaffold colocalized to a single linkage group 87.3% of the time. This suggests that our recombination mapping successfully recapitulated the true genomic positions of the markers used to construct our map.</p></div><div id="sec-12" class="subsection"><h3>Improved linkage mapping resources in <em>Astyanax</em></h3><p id="p-38">We sought to compare our linkage map with maps previously published by <a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> (2008)</a> and <a id="xref-ref-33-8" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> (2013)</a> that also examined synteny between <em>Astyanax</em> and <em>Danio</em>. Metrics such as the number of linkage groups, total map length, number of markers, and marker density are commonly used to compare linkage maps within species. Both our GBS-based map and the RAD-seq and microsatellite-based map published by <a id="xref-ref-33-9" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> (2013)</a> consist of 25 linkage groups, matching the <em>Astyanax mexicanus</em> karyotype number of 25. The microsatellite-based map presented by <a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> (2008)</a> contains 28 groups (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Although our map is of comparable length, it represents a dramatic increase in marker number (+559% compared with that of <a id="xref-ref-12-9" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> 2008</a>; +320% compared with that of <a id="xref-ref-33-10" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> 2013</a>) and marker density (+473% compared with that of <a id="xref-ref-12-10" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> 2008</a>; +279% compared with that of <a id="xref-ref-33-11" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> 2013</a>) relative to previously published linkage maps for this system. As a result, we saw a substantial increase in the number of syntenic links between our map and <em>Danio</em> (+506% compared with that of <a id="xref-ref-12-11" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> 2008</a>; +453% compared with that of <a id="xref-ref-33-12" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> 2013</a>) and an increase in the number of unplaced <em>Astyanax</em> scaffolds that can be anchored to our map (+263% compared with that of <a id="xref-ref-12-12" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> 2008</a>; +171% compared with that of <a id="xref-ref-33-13" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> 2013</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465614/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465614/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Comparison of <em>Astyanax</em> linkage maps and syntenic studies with <em>Danio rerio</em></span><div class="sb-div caption-clear"></div></div></div><p id="p-39">Our map contains a total of 784 links between our linkage groups and the <em>Danio rerio</em> genome and an average of 30.84 links (minimum = 14, maximum = 52) per <em>Danio rerio</em> chromosome (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). This represents a considerable improvement over the results presented by <a id="xref-ref-12-13" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> (2008)</a> (155 total links, average links per <em>Danio</em> chromosome = 6.20, minimum = 0, maximum = 15) and <a id="xref-ref-33-14" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> (2013)</a> (173 total links, average links per <em>Danio</em> chromosome = 6.92, minimum = 1, maximum = 20). Additionally, although instances of synteny strongly represented in previous maps were also identified in this analysis, our map demonstrated increased representation of certain <em>Danio</em> chromosomes poorly represented in previous maps. For example, <a id="xref-ref-12-14" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> (2008)</a> did not identify links between their map and <em>Danio rerio</em> chromosome 11; however, we identified 36 links between our map and chromosome 11. Similarly, <em>Danio</em> chromosomes 17 and 19 are each represented once in the map of <a id="xref-ref-33-15" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> (2013)</a>. We identified substantial links between these chromosomes and our linkage groups 9 (n = 21) and 23 (n = 15), respectively.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465615/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465615/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Comparison of syntenic analyses between <em>Astyanax</em> linkage maps and their association with the <em>Danio rerio</em> genome across multiple studies</span><div class="sb-div caption-clear"></div></div></div><p id="p-44">Our linkage map uses an entirely different marker set than those used in previous maps. Therefore, it was not possible to make direct comparisons with the linkage groups across prior studies. However, we could indirectly compare maps by examining connections between <em>Astyanax</em> genomic scaffolds and each linkage map. We examined the five strongest syntenic links between single linkage groups in our GBS-based map and single <em>Danio</em> chromosomes and then identified analogous connections between those chromosomes and specific linkage groups in the maps presented by <a id="xref-ref-12-15" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> (2008)</a> and <a id="xref-ref-33-16" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> (2013)</a>.</p><p id="p-45"><em>Astyanax</em> genomic scaffolds harboring markers associating each linkage group with a particular <em>Danio</em> chromosome were then compared (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). We found that many of the identified <em>Astyanax</em> genomic scaffolds colocalize to putatively analogous linkage groups in both our GBS-based map and those of <a id="xref-ref-12-16" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> (2008)</a> and/or <a id="xref-ref-33-17" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> (2013)</a>. However, in every case examined, our linkage groups were inclusive of a much higher number of <em>Astyanax</em> genomic scaffolds compared with prior studies. Thus, while the linkage groups in our map represent genomic intervals similar to those represented in prior maps, our map achieves a higher level of detail and resolution. These results also suggest that future mapping efforts in <em>Astyanax</em> may benefit by combining GBS marker discovery with those markers used by <a id="xref-ref-12-17" class="xref-bibr" href="#ref-12">Gross <em>et al.</em> (2008)</a> and <a id="xref-ref-33-18" class="xref-bibr" href="#ref-33">O’Quin <em>et al.</em> (2013)</a> to generate the most comprehensive linkage mapping resource.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465626/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465626/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Representative analysis of linkage group equivalence and quality based on highly syntenic chromosomes in <em>Danio rerio</em> and linkage groups in <em>Astyanax mexicanus</em></span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>High-density GBS-based linkage mapping will inform the Astyanax genome sequencing project</h3><p id="p-50">Preliminary <em>Astyanax</em> genomic resources enabled us to locate 64-bp, anonymous GBS markers, and to assess the quality and reliability of our <em>Astyanax</em> linkage map. This emerging resource did not allow us to determine how well the 25 <em>Astyanax</em> chromosomes are represented in our map. However, these resources allowed us to determine if markers predicted to occur in the same genome scaffolds also co-occur in our GBS-based linkage map. Overall, we observed a high level of agreement between our linkage groups and one or more unplaced <em>Astyanax</em> genomic scaffolds.</p><p id="p-51">In many cases, markers present on the same scaffold clustered together over a portion of a linkage group with little or no interruption from unplaced markers or markers from other scaffolds (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). We expect these results will help inform chromosomal positions of scaffolds, given that linkage maps have been successfully used to augment genomic resources in other fish species, including several species of catfish (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Liu 2011</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Ninwichian <em>et al.</em> 2012</a>), rainbow trout (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Palti <em>et al.</em> 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Palti <em>et al.</em> 2012</a>), and Atlantic salmon (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Lorenz <em>et al.</em> 2010</a>). We believe our high-density GBS-based map resources will both provide a resource for more refined QTL analyses and inform the genomic architecture of the <em>Astyanax</em> genome sequencing project.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/2/241/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Colinearity between Astyanax linkage groups and genome scaffolds. We visualized the &#x201C;anchoring&#x201D; of seven unplaced Astyanax genome scaffolds (various colors) to linkage group 23 (black) in our Astyanax linkage map. For clarity, only scaffolds harboring four or more GBS markers were included. Scaffolds correspond to discrete, colinear sections of the linkage group with minimal overlap. The linear arrangement of markers is largely preserved between the scaffold and the linkage group. The scale for Astyanax scaffolds is in Mb; the scale for linkage group 23 is shown in cM." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-291836724" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Colinearity between &lt;em&gt;Astyanax&lt;/em&gt; linkage groups and genome scaffolds. We visualized the &#x201C;anchoring&#x201D; of seven unplaced &lt;em&gt;Astyanax&lt;/em&gt; genome scaffolds (various colors) to linkage group 23 (black) in our &lt;em&gt;Astyanax&lt;/em&gt; linkage map. For clarity, only scaffolds harboring four or more GBS markers were included. Scaffolds correspond to discrete, colinear sections of the linkage group with minimal overlap. The linear arrangement of markers is largely preserved between the scaffold and the linkage group. The scale for &lt;em&gt;Astyanax&lt;/em&gt; scaffolds is in Mb; the scale for linkage group 23 is shown in cM.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/2/241/F4.medium.gif" width="440" height="398"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/5/2/241/F4.medium.gif" width="440" height="398"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/2/241/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/2/241/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465666" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-52" class="first-child">Colinearity between <em>Astyanax</em> linkage groups and genome scaffolds. We visualized the “anchoring” of seven unplaced <em>Astyanax</em> genome scaffolds (various colors) to linkage group 23 (black) in our <em>Astyanax</em> linkage map. For clarity, only scaffolds harboring four or more GBS markers were included. Scaffolds correspond to discrete, colinear sections of the linkage group with minimal overlap. The linear arrangement of markers is largely preserved between the scaffold and the linkage group. The scale for <em>Astyanax</em> scaffolds is in Mb; the scale for linkage group 23 is shown in cM.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section conclusions" id="sec-14"><h2 class="">Conclusions</h2><p id="p-53">We constructed a high-density linkage map for <em>Astyanax mexicanus</em> based on high-throughput genotyping-by-sequencing data. We leveraged emerging <em>Astyanax</em> genomic and transcriptomic resources and <em>Danio rerio</em> genomic and transcriptomic data to locate syntenic regions shared between our map and the <em>Danio</em> genome. These findings were based on the physical position of homologous (64-bp) GBS marker sequences. As expected, based on the significant divergence between these species, we recovered varying levels of synteny between portions of our <em>Astyanax</em> linkage groups and regions of the <em>Danio</em> genome. As a proof of concept, we successfully mapped a strong QTL associated with albinism and demonstrated significant conserved genomic architecture in the regions surrounding the gene <em>Oca2</em>, between <em>Astyanax</em> and <em>Danio</em>. We successfully anchored emerging <em>Astyanax</em> genomic information to our GBS-based linkage map, identifying the putative location of thousands of anonymous GBS marker sequences within unplaced <em>Astyanax</em> genome scaffolds. This strategy revealed significant colinearity between genomic scaffolds and our linkage map, and it demonstrated the utility of high-density, GBS-based linkage maps to inform and improve nascent genomic resources. Multiple comparisons with previously published maps suggest that our GBS-based map offers a higher level of resolution and a greater number of connections between <em>Astyanax</em> and <em>Danio</em> genomes. We hope that this resource and technology will accelerate the search and identification of genes mediating cave-associated traits in <em>Astyanax</em>, facilitate the genomic assembly for this system, and prove useful to other natural model systems of evolutionary and biomedical relevance.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-54">The authors thank Amanda Krutzler, Bethany Stahl, and members of the Gross laboratory for valuable effort and input. The authors are also grateful to Wesley Warren, Suzanne McGaugh, and the Genome Institute at Washington University for providing access to the draft genome assembly (Bioproject PRJNA89115 NCBI accession number APWO00000000; supported by NIH grant R24 <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=RR032658&amp;atom=%2Fggg%2F5%2F2%2F241.atom">RR032658</a>-01 to W.W.). Additionally, the authors thank Suzanne McGaugh for providing B.M.C. and other members of the Gross laboratory for instruction in script-based BLAST search methods. This project was supported by National Institutes of Health (National Institute of Dental and Craniofacial Research) grant DE022403 (to J.B.G.) and a Cave Research Foundation Graduate Student Research Grant (to B.M.C.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-14"><p id="p-55">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015438/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.015438/-/DC1</a></p></li><li class="fn" id="fn-15"><p id="p-56"><em>Communicating editor: A. S. McCallion</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-10-30"><span class="received-label">Received </span>October 30, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-12-11"><span class="accepted-label">Accepted </span>December 11, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Carlson <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.1" data-doi="10.1093/jhered/93.1.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borowsky</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkens</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Mapping a cave fish genome: polygenic systems and regressive evolution.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">21</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DBorowsky%26rft.auinit1%253DR.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D19%26rft.epage%253D21%26rft.atitle%253DMapping%2Ba%2BCave%2BFish%2BGenome%253A%2BPolygenic%2BSystems%2Band%2BRegressive%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.19%26rft_id%253Dinfo%253Apmid%252F12011170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjc6IjkzLzEvMTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzIvMjQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.2" data-doi="10.2307/1437876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breder</span>  <span class="cit-name-given-names">C. M.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>, <span class="cit-pub-date">1943</span><span class="cit-comment">a</span> <span class="cit-article-title">Apparent changes in phenotypic ratios of the Characins at the type locality of <em>Anoptichthys jordani</em> Hubbs and Innes.</span> <abbr class="cit-jnl-abbrev">Copeia</abbr> <span class="cit-vol">1943</span>: <span class="cit-fpage">26</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCopeia%26rft.volume%253D1943%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1437876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1437876&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.3" data-doi="10.1111/j.2164-0947.1943.tb00880.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breder</span>  <span class="cit-name-given-names">C. M.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>, <span class="cit-pub-date">1943</span><span class="cit-comment">b</span> <span class="cit-article-title">Problems in the behavior and evolution of a species of blind cave fish.</span> <abbr class="cit-jnl-abbrev">Trans. N. Y. Acad. Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">168</span>–<span class="cit-lpage">176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrans.%2BN.%2BY.%2BAcad.%2BSci.%26rft.volume%253D5%26rft.spage%253D168%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2164-0947.1943.tb00880.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2164-0947.1943.tb00880.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.4" data-doi="10.1111/j.1365-2699.2004.01170.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The biogeography of otophysan fishes (Ostariophysi: Otophysi): A new appraisal.</span> <abbr class="cit-jnl-abbrev">J. Biogeogr.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiogeogr.%26rft.volume%253D32%26rft.spage%253D287%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2699.2004.01170.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2699.2004.01170.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000226472300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.5" data-doi="10.1093/bioinformatics/btg112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Broman</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">R/qtl: QTL mapping in experimental crosses.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">889</span>–<span class="cit-lpage">890</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBroman%26rft.auinit1%253DK.%2BW.%26rft.volume%253D19%26rft.issue%253D7%26rft.spage%253D889%26rft.epage%253D890%26rft.atitle%253DR%252Fqtl%253A%2BQTL%2Bmapping%2Bin%2Bexperimental%2Bcrosses%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg112%26rft_id%253Dinfo%253Apmid%252F12724300%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxOS83Lzg4OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMi8yNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.6" data-doi="10.1016/0376-6357(85)90017-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burchards</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dölle</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parzefall</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span> <span class="cit-article-title">Aggressive behaviour of an epigean population of <em>Astyanax mexicanus</em> (Characidae, Pisces) and some observations of three subterranean populations.</span> <abbr class="cit-jnl-abbrev">Behav. Processes</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBehav.%2BProcesses%26rft.volume%253D11%26rft.spage%253D225%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0376-6357%252885%252990017-8%26rft_id%253Dinfo%253Apmid%252F24896447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0376-6357(85)90017-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24896447&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985AQU4300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.7" data-doi="10.1038/nrg3012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boone</span>  <span class="cit-name-given-names">J. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">510</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DDavey%26rft.auinit1%253DJ.%2BW.%26rft.volume%253D12%26rft.issue%253D7%26rft.spage%253D499%26rft.epage%253D510%26rft.atitle%253DGenome-wide%2Bgenetic%2Bmarker%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3012%26rft_id%253Dinfo%253Apmid%252F21681211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21681211&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.8" data-doi="10.1016/j.cub.2011.03.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duboué</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keene</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borowsky</span>  <span class="cit-name-given-names">R. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Evolutionary convergence on sleep loss in cavefish populations.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">671</span>–<span class="cit-lpage">676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DDuboue%26rft.auinit1%253DE.%2BR.%26rft.volume%253D21%26rft.issue%253D8%26rft.spage%253D671%26rft.epage%253D676%26rft.atitle%253DEvolutionary%2Bconvergence%2Bon%2Bsleep%2Bloss%2Bin%2Bcavefish%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2011.03.020%26rft_id%253Dinfo%253Apmid%252F21474315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2011.03.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21474315&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.9" data-doi="10.1016/j.cub.2012.10.044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elipot</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinaux</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Callebert</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Retaux</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolutionary shift from fighting to foraging in blind cavefish through changes in the serotonin network.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D23%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2012.10.044%26rft_id%253Dinfo%253Apmid%252F23159600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2012.10.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23159600&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.10" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.11" data-doi="10.1186/1471-2164-13-521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Everett</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Meiotic maps of sockeye salmon derived from massively parallel DNA sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">521</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DEverett%26rft.auinit1%253DM.%2BV.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D521%26rft.epage%253D521%26rft.atitle%253DMeiotic%2Bmaps%2Bof%2Bsockeye%2Bsalmon%2Bderived%2Bfrom%2Bmassively%2Bparallel%2BDNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-521%26rft_id%253Dinfo%253Apmid%252F23031582%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23031582&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.12" data-doi="10.1073/pnas.0806238105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Protas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheid</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidal</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Synteny and candidate gene prediction using an anchored linkage map of <em>Astyanax mexicanus</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">105</span>: <span class="cit-fpage">20106</span>–<span class="cit-lpage">20111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0806238105%26rft_id%253Dinfo%253Apmid%252F19104060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA1LzUxLzIwMTA2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8yLzI0MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.13" data-doi="10.1371/journal.pgen.1000326"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borowsky</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabin</span>  <span class="cit-name-given-names">C. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A novel role for <em>Mc1r</em> in the parallel evolution of depigmentation in independent populations of the cavefish <em>Astyanax mexicanus</em></span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr>. <span class="cit-vol">5</span>(1): <span class="cit-fpage">e1000326</span>. <span class="cit-comment">DOI: 10.1371/journal.pgen.1000326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGross%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253De1000326%26rft.epage%253De1000326%26rft.atitle%253DA%2Bnovel%2Brole%2Bfor%2BMc1r%2Bin%2Bthe%2Bparallel%2Bevolution%2Bof%2Bdepigmentation%2Bin%2Bindependent%2Bpopulations%2Bof%2Bthe%2Bcavefish%2BAstyanax%2Bmexicanus.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000326%26rft_id%253Dinfo%253Apmid%252F19119422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000326&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19119422&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.14" data-doi="10.1371/journal.pone.0055659"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furterer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stahl</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">An integrated transcriptome-wide analysis of cave and surface dwelling <em>Astyanax mexicanus</em></span>. <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e55659</span>. <span class="cit-comment">DOI: 10.1371/journal.pone.0055659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De55659%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0055659%26rft_id%253Dinfo%253Apmid%252F23405189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0055659&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23405189&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.15" data-doi="10.1534/genetics.114.161661"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krutzler</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">B. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Complex craniofacial changes in blind cave-dwelling fish are mediated by genetically symmetric and asymmetric loci.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">1303</span>–<span class="cit-lpage">1319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.161661%26rft_id%253Dinfo%253Apmid%252F24496009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5Ni80LzEzMDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzIvMjQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.16" data-doi="10.1111/mec.12082"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hecht</span>  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holecek</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narum</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide association reveals genetic basis for the propensity to migrate in wild populations of rainbow and steelhead trout.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3061</span>–<span class="cit-lpage">3076</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3061%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12082%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12082&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000319675300017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-other" id="cit-5.2.241.17"><div class="cit-metadata unstructured">Hess, J. E., N. R. Campbell, M. F. Docker, C. Baker, A. Jackson <em>et al.</em>, 2014 Use of genotyping-by-sequencing data to develop a high-throughput and multi-functional SNP panel for conservation applications in Pacific lamprey. Mol. Ecol. Resour. Advance online publication. Epub 2014 Jun 5. doi: 10.1111/1755–0998.12283.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.18" data-doi="10.1186/1471-2164-13-244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davey</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bishop</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mota-Velasco</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Characterisation of QTL-linked and genome-wide restriction site-associated DNA (RAD) markers in farmed Atlantic salmon.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHouston%26rft.auinit1%253DR.%2BD.%26rft.volume%253D13%26rft.spage%253D244%26rft.epage%253D244%26rft.atitle%253DCharacterisation%2Bof%2BQTL-linked%2Band%2Bgenome-wide%2Brestriction%2Bsite-associated%2BDNA%2B%2528RAD%2529%2Bmarkers%2Bin%2Bfarmed%2BAtlantic%2Bsalmon.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-244%26rft_id%253Dinfo%253Apmid%252F22702806%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-244&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22702806&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-5.2.241.19"><div class="cit-metadata unstructured">Hubbs, C. L., and W. T. Innes, 1936 The first known blind fish of the family Characidae: A new genus from Mexico. Occas. Pap. Mus. Zool. Univ. Mich. 342: 1–7.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.20" data-doi="10.1006/dbio.2000.0121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeffery</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Cavefish as a model system in evolutionary developmental biology.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">231</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DJeffery%26rft.auinit1%253DW.%2BR.%26rft.volume%253D231%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DCavefish%2Bas%2Ba%2Bmodel%2Bsystem%2Bin%2Bevolutionary%2Bdevelopmental%2Bbiology.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fdbio.2000.0121%26rft_id%253Dinfo%253Apmid%252F11180948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/dbio.2000.0121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11180948&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167271000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.21" data-doi="10.1146/annurev-genet-102108-134216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeffery</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Regressive evolution in <em>Astyanax</em> cavefish.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DJeffery%26rft.auinit1%253DW.%2BR.%26rft.volume%253D43%26rft.spage%253D25%26rft.epage%253D47%26rft.atitle%253DRegressive%2Bevolution%2Bin%2BAstyanax%2Bcavefish.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-102108-134216%26rft_id%253Dinfo%253Apmid%252F19640230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-102108-134216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19640230&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273580300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.22" data-doi="10.2307/1443283"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirby</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubbs</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Karyotypic similarities between the Mexican and blind tetras.</span> <abbr class="cit-jnl-abbrev">Copeia</abbr> <span class="cit-vol">1977</span>: <span class="cit-fpage">578</span>–<span class="cit-lpage">580</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCopeia%26rft.volume%253D1977%26rft.spage%253D578%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1443283%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1443283&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.23" data-doi="10.1111/j.1469-1809.1943.tb02321.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kosambi</span>  <span class="cit-name-given-names">D. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span> <span class="cit-article-title">The estimation of map distances from recombination values.</span> <abbr class="cit-jnl-abbrev">Ann. Eugen.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">175</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D12%26rft.spage%253D172%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1943.tb02321.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-1809.1943.tb02321.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.24" data-doi="10.1016/j.cub.2013.07.056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kowalko</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rohner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rompani</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Linden</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Loss of schooling behavior in cavefish through sight-dependent and sight-independent mechanisms.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1874</span>–<span class="cit-lpage">1883</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BBiol.%26rft.volume%253D23%26rft.spage%253D1874%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2013.07.056%26rft_id%253Dinfo%253Apmid%252F24035545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2013.07.056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24035545&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.25" data-doi="10.1101/gr.092759.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krzywinski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birol</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connors</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascoyne</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Circos: An information aesthetic for comparative genomics.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1639</span>–<span class="cit-lpage">1645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.092759.109%26rft_id%253Dinfo%253Apmid%252F19541911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTYzOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMi8yNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.26" data-doi="10.1111/eva.12128"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Larson</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeb</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Everett</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waples</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Templin</span>  <span class="cit-name-given-names">W. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genotyping by sequencing resolves shallow population structure to inform conservation of Chinook salmon (<em>Oncorhynchus tshawytscha</em>).</span> <abbr class="cit-jnl-abbrev">Evol. Appl.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">369</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol.%2BAppl.%26rft.volume%253D7%26rft.spage%253D355%26rft_id%253Dinfo%253Adoi%252F10.1111%252Feva.12128%26rft_id%253Dinfo%253Apmid%252F24665338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/eva.12128&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24665338&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-other" id="cit-5.2.241.27"><div class="cit-metadata unstructured">Li, C., G. Waldbieser, B. Bosworth, B. H. Beck, W. Thongda <em>et al.</em>, 2014 SNP discovery in wild and domesticated populations of blue catfish, <em>Ictalurus furcatus</em>, using genotyping-by-sequencing and subsequent SNP validation. Mol. Ecol. Resour. 14: 1261–1270.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Development of genomic resources in support of sequencing, assembly, and annotation of the catfish genome.</span> <abbr class="cit-jnl-abbrev">Comp. Biochem. Physiol. D</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComp.%2BBiochem.%2BPhysiol.%2BD%26rft.volume%253D6%26rft.spage%253D11%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.29" data-doi="10.1111/j.1365-2052.2009.01963.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brenna-Hansen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roseth</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davidson</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BAC-based upgrading and physical integration of a genetic SNP map in Atlantic salmon.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">54</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DLorenz%26rft.auinit1%253DS.%26rft.volume%253D41%26rft.issue%253D1%26rft.spage%253D48%26rft.epage%253D54%26rft.atitle%253DBAC-based%2Bupgrading%2Band%2Bphysical%2Bintegration%2Bof%2Ba%2Bgenetic%2BSNP%2Bmap%2Bin%2BAtlantic%2Bsalmon.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2009.01963.x%26rft_id%253Dinfo%253Apmid%252F19917045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2009.01963.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19917045&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.30" data-doi="10.1371/journal.pgen.1003215"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherney</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Switchgrass genomic diversity, ploidy, and evolution: Novel insights from a network-based SNP discovery protocol.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003215%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003215%26rft_id%253Dinfo%253Apmid%252F23349638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23349638&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.31" data-doi="10.1038/ncomms6307"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGaugh</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aken</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borowsky</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The cavefish genome reveals candidate genes for eye loss.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">5307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D5307%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6307%26rft_id%253Dinfo%253Apmid%252F25329095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6307&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25329095&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.32" data-doi="10.1534/g3.112.003962"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ninwichian</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peatman</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kucuktas</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somridhivej</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Second-generation genetic linkage map of catfish and its integration with the BAC-based physical map. </span><abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1233</span>–<span class="cit-lpage">1241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DNinwichian%26rft.auinit1%253DP.%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253D1233%26rft.epage%253D1241%26rft.atitle%253DSecond-Generation%2BGenetic%2BLinkage%2BMap%2Bof%2BCatfish%2Band%2BIts%2BIntegration%2Bwith%2Bthe%2BBAC-Based%2BPhysical%2BMap%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.003962%26rft_id%253Dinfo%253Apmid%252F23050234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjIvMTAvMTIzMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMi8yNDEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.33" data-doi="10.1371/journal.pone.0057281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Quin</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshizawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doshi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffery</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Quantitative genetic analysis of retinal degeneration in the blind cavefish <em>Astyanax mexicanus</em>.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e57281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De57281%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0057281%26rft_id%253Dinfo%253Apmid%252F23437360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0057281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23437360&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.34" data-doi="10.1111/mec.12234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ogden</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mugue</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinsohn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Senn</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sturgeon conservation genomics: SNP discovery and validation using RAD sequencing.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">3112</span>–<span class="cit-lpage">3123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%26rft.volume%253D22%26rft.spage%253D3112%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fmec.12234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/mec.12234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.35" data-doi="10.1186/1471-2164-12-180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palti</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Genet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charlet</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A first generation integrated map of the rainbow trout genome.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DPalti%26rft.auinit1%253DY.%26rft.volume%253D12%26rft.spage%253D180%26rft.epage%253D180%26rft.atitle%253DA%2Bfirst%2Bgeneration%2Bintegrated%2Bmap%2Bof%2Bthe%2Brainbow%2Btrout%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-12-180%26rft_id%253Dinfo%253Apmid%252F21473775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-12-180&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21473775&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.36" data-doi="10.1007/s10126-011-9418-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palti</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Genet</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A second generation integrated map of the rainbow trout (<em>Oncorhynchus mykiss</em>) genome: Analysis of conserved synteny with model fish genomes.</span> <abbr class="cit-jnl-abbrev">Mar. Biotechnol. (NY)</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMarine%2Bbiotechnology%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft.stitle%253DMar%2BBiotechnol%2B%2528NY%2529%26rft.aulast%253DPalti%26rft.auinit1%253DY.%26rft.volume%253D14%26rft.issue%253D3%26rft.spage%253D343%26rft.epage%253D357%26rft.atitle%253DA%2Bsecond%2Bgeneration%2Bintegrated%2Bmap%2Bof%2Bthe%2Brainbow%2Btrout%2B%2528Oncorhynchus%2Bmykiss%2529%2Bgenome%253A%2Banalysis%2Bof%2Bconserved%2Bsynteny%2Bwith%2Bmodel%2Bfish%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10126-011-9418-z%26rft_id%253Dinfo%253Apmid%252F22101344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10126-011-9418-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22101344&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.37" data-doi="10.1242/dev.054106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pottin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinaux</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rétaux</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Restoring eye size in <em>Astyanax mexicanus</em> blind cavefish embryos through modulation of the Shh and Fgf8 forebrain organising centres.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">2467</span>–<span class="cit-lpage">2476</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DPottin%26rft.auinit1%253DK.%26rft.volume%253D138%26rft.issue%253D12%26rft.spage%253D2467%26rft.epage%253D2476%26rft.atitle%253DRestoring%2Beye%2Bsize%2Bin%2BAstyanax%2Bmexicanus%2Bblind%2Bcavefish%2Bembryos%2Bthrough%2Bmodulation%2Bof%2Bthe%2BShh%2Band%2BFgf8%2Bforebrain%2Borganising%2Bcentres%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.054106%26rft_id%253Dinfo%253Apmid%252F21610028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTM4LzEyLzI0NjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzIvMjQxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.38" data-doi="10.1038/nm1334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Protas</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hersey</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kochanek</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkens</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic analysis of cavefish reveals molecular convergence in the evolution of albinism.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DProtas%26rft.auinit1%253DM.%2BE.%26rft.volume%253D38%26rft.issue%253D1%26rft.spage%253D107%26rft.epage%253D111%26rft.atitle%253DGenetic%2Banalysis%2Bof%2Bcavefish%2Breveals%2Bmolecular%2Bconvergence%2Bin%2Bthe%2Bevolution%2Bof%2Balbinism.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnm1334%26rft_id%253Dinfo%253Apmid%252F16341223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nm1334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16341223&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234227200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.39" data-doi="10.1016/j.cub.2007.01.051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Protas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borowsky</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Regressive evolution in the Mexican cave tetra, <em>Astyanax mexicanus</em>.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">452</span>–<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DProtas%26rft.auinit1%253DM.%26rft.volume%253D17%26rft.issue%253D5%26rft.spage%253D452%26rft.epage%253D454%26rft.atitle%253DRegressive%2Bevolution%2Bin%2Bthe%2BMexican%2Bcave%2Btetra%252C%2BAstyanax%2Bmexicanus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2007.01.051%26rft_id%253Dinfo%253Apmid%252F17306543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2007.01.051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17306543&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244817000029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.40" data-doi="10.1111/j.1525-142X.2008.00227.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Protas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabansky</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidal</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Multi-trait evolution in a cave fish, <em>Astyanax mexicanus</em>.</span> <abbr class="cit-jnl-abbrev">Evol. Dev.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">196</span>–<span class="cit-lpage">209</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%2B%2526%2Bdevelopment%26rft.stitle%253DEvol%2BDev%26rft.aulast%253DProtas%26rft.auinit1%253DM.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D196%26rft.epage%253D209%26rft.atitle%253DMulti-trait%2Bevolution%2Bin%2Ba%2Bcave%2Bfish%252C%2BAstyanax%2Bmexicanus.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1525-142X.2008.00227.x%26rft_id%253Dinfo%253Apmid%252F18315813%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1525-142X.2008.00227.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18315813&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253707300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rowe</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Renaut</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guggisberg</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">RAD in the realm of next-generation sequencing technologies.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">3499</span>–<span class="cit-lpage">3502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DRowe%26rft.auinit1%253DH.%2BC.%26rft.volume%253D20%26rft.issue%253D17%26rft.spage%253D3499%26rft.epage%253D3502%26rft.atitle%253DRAD%2Bin%2Bthe%2Brealm%2Bof%2Bnext-generation%2Bsequencing%2Btechnologies.%26rft_id%253Dinfo%253Apmid%252F21991593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21991593&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294224400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.42" data-doi="10.2307/1440425"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Şadoǧlu</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1956</span> <span class="cit-article-title">A preliminary report on the genetics of the Mexican cave characins.</span> <abbr class="cit-jnl-abbrev">Copeia</abbr> <span class="cit-vol">1956</span>: <span class="cit-fpage">113</span>–<span class="cit-lpage">114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCopeia%26rft.volume%253D1956%26rft.spage%253D113%26rft_id%253Dinfo%253Adoi%252F10.2307%252F1440425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/1440425&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.43" data-doi="10.1016/j.cbpa.2010.03.030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Salin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voituron</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mourin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hervant</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Cave colonization without fasting capacities: An example with the fish <em>Astyanax fasciatus mexicanus</em>.</span> <abbr class="cit-jnl-abbrev">Comp. Biochem. Physiol. A</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComparative%2BBiochemistry%2Band%2BPhysiology.%2BPart%2BA%252C%2BMolecular%2B%2526%2BIntegrative%2BPhysiology%26rft.stitle%253DComparative%2BBiochemistry%2Band%2BPhysiology.%2BPart%2BA%252C%2BMolecular%2B%2526%2BIntegrative%2BPhysiology%26rft.aulast%253DSalin%26rft.auinit1%253DK.%26rft.volume%253D156%26rft.issue%253D4%26rft.spage%253D451%26rft.epage%253D457%26rft.atitle%253DCave%2Bcolonization%2Bwithout%2Bfasting%2Bcapacities%253A%2Ban%2Bexample%2Bwith%2Bthe%2Bfish%2BAstyanax%2Bfasciatus%2Bmexicanus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cbpa.2010.03.030%26rft_id%253Dinfo%253Apmid%252F20382251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cbpa.2010.03.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20382251&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schemmel</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Studies on the genetics of feeding behaviour in the cave fish <em>Astyanax mexicanus</em> f. <em>anoptichthys</em>. An example of apparent monofactorial inheritance by polygenes.</span> <abbr class="cit-jnl-abbrev">Z. Tierpsychol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZeitschrift%2Bf%25C3%25BCr%2BTierpsychologie%26rft.stitle%253DZ%2BTierpsychol%26rft.aulast%253DSchemmel%26rft.auinit1%253DC.%26rft.volume%253D53%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D22%26rft.atitle%253DStudies%2Bon%2Bthe%2Bgenetics%2Bof%2Bfeeding%2Bbehaviour%2Bin%2Bthe%2Bcave%2Bfish%2BAstyanax%2Bmexicanus%2Bf.%2Banoptichthys.%2BAn%2Bexample%2Bof%2Bapparent%2Bmonofactorial%2Binheritance%2Bby%2Bpolygenes.%26rft_id%253Dinfo%253Apmid%252F7434991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7434991&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.45" data-doi="10.1038/nature02864"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stock</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffery</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Hedgehog signalling controls eye degeneration in blind cavefish.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">431</span>: <span class="cit-fpage">844</span>–<span class="cit-lpage">847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DYamamoto%26rft.auinit1%253DY.%26rft.volume%253D431%26rft.issue%253D7010%26rft.spage%253D844%26rft.epage%253D847%26rft.atitle%253DHedgehog%2Bsignalling%2Bcontrols%2Beye%2Bdegeneration%2Bin%2Bblind%2Bcavefish.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02864%26rft_id%253Dinfo%253Apmid%252F15483612%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02864&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15483612&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000224435500047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.241.46" data-doi="10.1186/1741-7007-10-108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yoshizawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Quin</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffery</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution of an adaptive behavior and its sensory receptors promotes eye regression in blind cavefish.</span> <abbr class="cit-jnl-abbrev">BMC Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBiol.%26rft.volume%253D10%26rft.spage%253D108%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1741-7007-10-108%26rft_id%253Dinfo%253Apmid%252F23270452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1741-7007-10-108&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23270452&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F241.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/2/241.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/2/235" title="Segregation of a Spontaneous Klrd1 (CD94) Mutation in DBA/2 Mouse Substrains" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/2/253" title="Plant Genetic Archaeology: Whole-Genome Sequencing Reveals the Pedigree of a Classical Trisomic Line" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/2"><span>Volume 5 Issue 2, February 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/2.cover-source.jpg?itok=xW4cmeO6" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/2/241" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AKo49MNBwrV0-Xhzz_k4wZzuyi4jpCACAecT2055n34" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/2/241" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/2/241" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/2/241" class="active" data-icon-position="" data-hide-link-title="0">A High-Density Linkage Map for Astyanax mexicanus Using Genotyping-by-Sequencing Technology</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459771" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-AwD5-GN1VD06lGpha_HMcVwOGpiCX_s0vH11U_BUDtk" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/2/241.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/2/241" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-m_ND_n6P8NK3DLRIDnB9pHGv8cMSbvxcPbkjduxit5c" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/2/241" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/2/241.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459771" id="node459771--2" data-pisa="ggg;5/2/241" data-pisa-master="ggg;g3.114.015438" data-apath="/ggg/5/2/241.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density Linkage Map for <em>Astyanax mexicanus</em> Using Genotyping-by-Sequencing Technology</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Brian M. Carlson</span>, <span class="highwire-citation-author" data-delta="1">Samuel W. Onusko</span> and <span class="highwire-citation-author" data-delta="2">Joshua B. Gross</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">241-251;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015438 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459771--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Brian M. Carlson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, University of Cincinnati, Cincinnati, Ohio 45221</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BM.%2BCarlson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlson%20BM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BM.%2BCarlson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Samuel W. Onusko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, University of Cincinnati, Cincinnati, Ohio 45221</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samuel%2BW.%2BOnusko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Onusko%20SW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASamuel%2BW.%2BOnusko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joshua B. Gross </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, University of Cincinnati, Cincinnati, Ohio 45221</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BB.%2BGross%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gross%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoshua%2BB.%2BGross%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:grossja@ucmail.uc.edu" class="" data-icon-position="" data-hide-link-title="0">grossja@ucmail.uc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/2/241" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/2/241&amp;title=A%20High-Density%20Linkage%20Map%20for%20Astyanax%20mexicanus%20Using%20Genotyping-by-Sequencing%20Technology" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/2/241&amp;title=A%20High-Density%20Linkage%20Map%20for%20Astyanax%20mexicanus%20Using%20Genotyping-by-Sequencing%20Technology" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/2/241&amp;title=A%20High-Density%20Linkage%20Map%20for%20Astyanax%20mexicanus%20Using%20Genotyping-by-Sequencing%20Technology" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/2/241&amp;text=A%20High-Density%20Linkage%20Map%20for%20Astyanax%20mexicanus%20Using%20Genotyping-by-Sequencing%20Technology" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/2/241&amp;title=A%20High-Density%20Linkage%20Map%20for%20Astyanax%20mexicanus%20Using%20Genotyping-by-Sequencing%20Technology" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/2/241&amp;t=A%20High-Density%20Linkage%20Map%20for%20Astyanax%20mexicanus%20Using%20Genotyping-by-Sequencing%20Technology" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/2/241&amp;title=A%20High-Density%20Linkage%20Map%20for%20Astyanax%20mexicanus%20Using%20Genotyping-by-Sequencing%20Technology" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/2/241&amp;title=A%20High-Density%20Linkage%20Map%20for%20Astyanax%20mexicanus%20Using%20Genotyping-by-Sequencing%20Technology" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459771/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459771" id="node459771--3" data-pisa="ggg;5/2/241" data-pisa-master="ggg;g3.114.015438" data-apath="/ggg/5/2/241.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A High-Density Linkage Map for <em>Astyanax mexicanus</em> Using Genotyping-by-Sequencing Technology</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Brian M. Carlson</span>, <span class="highwire-citation-author" data-delta="1">Samuel W. Onusko</span> and <span class="highwire-citation-author" data-delta="2">Joshua B. Gross</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">241-251;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.015438 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459771--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Brian M. Carlson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, University of Cincinnati, Cincinnati, Ohio 45221</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brian%2BM.%2BCarlson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Carlson%20BM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrian%2BM.%2BCarlson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Samuel W. Onusko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, University of Cincinnati, Cincinnati, Ohio 45221</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Samuel%2BW.%2BOnusko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Onusko%20SW&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASamuel%2BW.%2BOnusko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joshua B. Gross </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biological Sciences, University of Cincinnati, Cincinnati, Ohio 45221</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joshua%2BB.%2BGross%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gross%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoshua%2BB.%2BGross%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:grossja@ucmail.uc.edu" class="" data-icon-position="" data-hide-link-title="0">grossja@ucmail.uc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=241&amp;rft.epage=251&amp;rft.atitle=A%20High-Density%20Linkage%20Map%20for%20Astyanax%20mexicanus%20Using%20Genotyping-by-Sequencing%20Technology&amp;rft.volume=5&amp;rft.issue=2&amp;rft.date=2015-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Carlson%2C+Brian+M.&amp;rft.au=Onusko%2C+Samuel+W.&amp;rft.au=Gross%2C+Joshua+B."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459771/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459771/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459771/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459771/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459771/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459771/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459771/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459771/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459771/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459771/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459771/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459771/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/2/241" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#sec-14" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/2/241.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/2/241.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/5/2/241.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/5/2/241.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
