<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/5/2/291" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/5/2/291.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/5/2/291.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/5/2/291.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/5/2/291/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/5/2/291.atom" />
<meta name="HW.pisa" content="ggg;5/2/291" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Threshold Models for Genome-Enabled Prediction of Ordinal Categorical Traits in Plant Breeding" />
<meta name="DC.Identifier" content="10.1534/g3.114.016188" />
<meta name="DC.Date" content="2015-02-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2015 Montesinos-López et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Categorical scores for disease susceptibility or resistance often are recorded in plant breeding. The aim of this study was to introduce genomic models for analyzing ordinal characters and to assess the predictive ability of genomic predictions for ordered categorical phenotypes using a threshold model counterpart of the Genomic Best Linear Unbiased Predictor ( i.e. , TGBLUP). The threshold model was used to relate a hypothetical underlying scale to the outward categorical response. We present an empirical application where a total of nine models, five without interaction and four with genomic × environment interaction (G×E) and genomic additive × additive × environment interaction (G×G×E), were used. We assessed the proposed models using data consisting of 278 maize lines genotyped with 46,347 single-nucleotide polymorphisms and evaluated for disease resistance [with ordinal scores from 1 (no disease) to 5 (complete infection)] in three environments (Colombia, Zimbabwe, and Mexico). Models with G×E captured a sizeable proportion of the total variability, which indicates the importance of introducing interaction to improve prediction accuracy. Relative to models based on main effects only, the models that included G×E achieved 9–14% gains in prediction accuracy; adding additive × additive interactions did not increase prediction accuracy consistently across locations." />
<meta name="DC.Contributor" content="Osval A. Montesinos-López" />
<meta name="DC.Contributor" content="Abelardo Montesinos-López" />
<meta name="DC.Contributor" content="Paulino Pérez-Rodríguez" />
<meta name="DC.Contributor" content="Gustavo de los Campos" />
<meta name="DC.Contributor" content="Kent Eskridge" />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="article:published_time" content="2015-02-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Threshold Models for Genome-Enabled Prediction of Ordinal Categorical Traits in Plant Breeding" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Categorical scores for disease susceptibility or resistance often are recorded in plant breeding. The aim of this study was to introduce genomic models for analyzing ordinal characters and to assess the predictive ability of genomic predictions for ordered categorical phenotypes using a threshold model counterpart of the Genomic Best Linear Unbiased Predictor (&lt;i&gt;i.e.&lt;/i&gt;, TGBLUP). The threshold model was used to relate a hypothetical underlying scale to the outward categorical response. We present an empirical application where a total of nine models, five without interaction and four with genomic × environment interaction (G×E) and genomic additive × additive × environment interaction (G×G×E), were used. We assessed the proposed models using data consisting of 278 maize lines genotyped with 46,347 single-nucleotide polymorphisms and evaluated for disease resistance [with ordinal scores from 1 (no disease) to 5 (complete infection)] in three environments (Colombia, Zimbabwe, and Mexico). Models with G×E captured a sizeable proportion of the total variability, which indicates the importance of introducing interaction to improve prediction accuracy. Relative to models based on main effects only, the models that included G×E achieved 9–14% gains in prediction accuracy; adding additive × additive interactions did not increase prediction accuracy consistently across locations.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2015/02/01" />
<meta name="citation_mjid" content="ggg;5/2/291" />
<meta name="citation_id" content="5/2/291" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/5/2/291" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/5/2/291.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/5/2/291.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/5/2/291.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.016188" />
<meta name="citation_pmid" content="25538102" />
<meta name="citation_volume" content="5" />
<meta name="citation_issue" content="2" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="291" />
<meta name="citation_lastpage" content="300" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Osval A. Montesinos-López" />
<meta name="citation_author_institution" content="Facultad de Telemática, Universidad de Colima, Avenida Universidad 333, Colima, México" />
<meta name="citation_author" content="Abelardo Montesinos-López" />
<meta name="citation_author_institution" content="Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), Guanajuato, Guanajuato, México" />
<meta name="citation_author" content="Paulino Pérez-Rodríguez" />
<meta name="citation_author_institution" content="Colegio de Postgraduados, Edo. de México, México" />
<meta name="citation_author" content="Gustavo de los Campos" />
<meta name="citation_author_institution" content="Department of Biostatistics, University of Alabama at Birmingham, 443, Birmingham, Alabama 35294" />
<meta name="citation_author" content="Kent Eskridge" />
<meta name="citation_author_institution" content="Department of Statistics, University of Nebraska, Lincoln, Nebraska" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), México, D.F., México" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_reference" content="citation_journal_title=Bull. Psychon. Soc.;citation_author=L. Atkinson;citation_title=The measurement-statistics controversy: factor analysis and subinterval data.;citation_pages=361-364;citation_volume=26;citation_year=1988;citation_doi=10.3758/BF03337683" />
<meta name="citation_reference" content="citation_journal_title=Mon. Weather Rev.;citation_author=G. W. Brier;citation_title=Verification of forecasts expressed in terms of probability.;citation_pages=1-3;citation_volume=78;citation_year=1950;citation_doi=10.1175/1520-0493(1950)078&lt;0001:VOFEIT&gt;2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype×environment interaction using pedigree and dense molecular markers.;citation_pages=707;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=J. Crop Improv.;citation_author=J. Crossa;citation_author=P. Pérez-Rodríguez;citation_author=G. de los Campos;citation_author=G. Mahuku;citation_author=S. Dreisigacker;citation_title=Genomic selection and prediction in plant breeding.;citation_pages=239-261;citation_volume=25;citation_year=2011;citation_doi=10.1080/15427528.2011.558767" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=J. Crossa;citation_author=Y. Beyene;citation_author=S. Kassa;citation_author=P. Pérez-Rodríguez;citation_author=J. M. Hickey;citation_title=Genomic prediction in maize breeding populations with genotyping-by-sequencing.;citation_volume=3;citation_year=2013;citation_pmid=24022750;citation_doi=10.1534/g3.113.008227" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Crossa;citation_author=P. Pérez-Rodríguez;citation_author=J. Hickey;citation_author=J. Burgueño;citation_author=L. Ornella;citation_title=Genomic prediction in CIMMYT maize and wheat breeding programs.;citation_pages=48-60;citation_volume=112;citation_year=2013;citation_pmid=23572121" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree.;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=4;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=P. H. Garthwaite;citation_author=J. B. Kadane;citation_author=A. O’Hagan;citation_title=Statistical methods for eliciting probability distributions.;citation_pages=680-701;citation_volume=100;citation_year=2005;citation_doi=10.1198/016214505000000105" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=D. Gianola;citation_title=A Method of Sire Evaluation for Dichotomies;citation_pages=1266-1271;citation_volume=51;citation_year=1980;citation_issue=6;citation_pmid=7204270" />
<meta name="citation_reference" content="citation_journal_title=Journal of Animal Science;citation_journal_abbrev=J ANIM SCI;citation_author=D. Gianola;citation_title=Theory and Analysis of Threshold Characters;citation_pages=1079-1096;citation_volume=54;citation_year=1982;citation_issue=5;citation_doi=10.2527/jas1982.5451079x" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. Gianola;citation_author=J. L. Foulley;citation_title=Sire evaluation for ordered categorical data with a threshold model.;citation_pages=1-23;citation_volume=15;citation_year=1983;citation_pmid=22879102;citation_doi=10.1186/1297-9686-15-1-1" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. M. González-Camacho;citation_author=G. de los Campos;citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=J. E. Cairns;citation_title=Genome-enabled prediction of genetic values using radial basis function neural networks.;citation_pages=759-771;citation_volume=125;citation_year=2012;citation_issue=4;citation_pmid=22566067;citation_doi=10.1007/s00122-012-1868-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=O. González-Recio;citation_author=S. Forni;citation_title=Genome-wide prediction of discrete traits using Bayesian regressions and machine learning.;citation_pages=7-7;citation_volume=43;citation_year=2011;citation_pmid=21329522;citation_doi=10.1186/1297-9686-43-7" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=A. J. Lorenz;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Plant breeding with genomic selection: gain per unit time and cost.;citation_pages=1681-1690;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.11.0662" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H.-P. Yang;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic selection in plant breeding: a comparison of models.;citation_pages=146;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=D. Jarquín;citation_author=J. Crossa;citation_author=X. Lacaze;citation_author=P. Cheyron;citation_author=J. Daucourt;citation_title=A reaction norm model for genomic selection using high-dimensional genomic and environmental data.;citation_pages=595-607;citation_volume=127;citation_year=2014;citation_pmid=24337101;citation_doi=10.1007/s00122-013-2243-1" />
<meta name="citation_reference" content="citation_title=BMC Proc.;citation_author=K. Kizilkaya;citation_author=R. G. Tait;citation_author=D. J. Garrick;citation_author=R. L. Fernando;citation_author=J. M. Reecy;citation_title=Whole genome analysis of infectious bovine kerato conjunctivitis in Angus cattle using Bayesian threshold models.;citation_pages=S22;citation_volume=5;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=AStA Adv. Stat. Anal.;citation_author=T. Kneib;citation_author=B. Baumgartner;citation_author=W. J. Steiner;citation_title=Semiparametric multinomial logit models for analysing consumer choice behaviour.;citation_pages=225-244;citation_volume=91;citation_year=2007;citation_doi=10.1007/s10182-007-0033-2" />
<meta name="citation_reference" content="citation_title=SAS for Linear Models;citation_author=R. C. Littell;citation_author=W. Stroup;citation_author=R. J. Freund;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez-Rodríguez;citation_author=G. de los Campos;citation_title=Genome-wide regression &amp; prediction with the BGLR statistical package.;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=P. Pérez-Rodríguez;citation_author=G. de los Campos;citation_author=J. Crossa;citation_author=D. Gianola;citation_title=Genomic-enabled prediction based on molecular markers and pedigree using the Bayesian Linear Regression Package in R.;citation_pages=106-116;citation_volume=3;citation_year=2010;citation_pmid=21566722;citation_doi=10.3835/plantgenome2010.04.0005" />
<meta name="citation_reference" content="citation_title=G3 (Bethesda);citation_author=P. Pérez-Rodríguez;citation_author=D. Gianola;citation_author=J. M. González-Camacho;citation_author=J. Crossa;citation_author=Y. Manes;citation_author=S. Dreisigacker,;citation_title=Comparison between linear and non-parametric models for genome-enabled prediction in wheat.;citation_pages=1595-1605;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=D. A. Sorensen;citation_author=S. Andersen;citation_author=D. Gianola;citation_author=I. Korsgaard;citation_title=Bayesian inference in threshold models using Gibbs sampling.;citation_pages=229-249;citation_volume=27;citation_year=1995;citation_doi=10.1051/gse:19950303" />
<meta name="citation_reference" content="citation_title=Generalized Linear Mixed Models: Modern Concepts, Methods and Applications;citation_author=W. Stroup;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=B. Villanueva;citation_author=J. Fernandez;citation_author=L. A. Garcia-Cortes;citation_author=L. Varona;citation_author=H. D. Daetwyler;citation_title=Accuracy of genome-wide evaluation for disease resistance in aquaculture breeding programs.;citation_volume=89;citation_year=2011;citation_pmid=21742941;citation_doi=10.2527/jas.2010-3814" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=C. L. Wang;citation_author=X. D. Ding;citation_author=J. Y. Wang;citation_author=J. F. Liu;citation_author=W. X. Fu;citation_title=Bayesian methods for estimating GEBVs of threshold traits.;citation_pages=213-219;citation_volume=110;citation_year=2012;citation_pmid=23149458" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Threshold Models for Genome-Enabled Prediction of Ordinal Categorical Traits in Plant Breeding" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2.cover-source.jpg" />
<meta name="twitter:description" content="Categorical scores for disease susceptibility or resistance often are recorded in plant breeding. The aim of this study was to introduce genomic models for analyzing ordinal characters and to assess the predictive ability of genomic predictions for ordered categorical phenotypes using a threshold model counterpart of the Genomic Best Linear Unbiased Predictor ( i.e. , TGBLUP). The threshold model was used to relate a hypothetical underlying scale to the outward categorical response. We present an empirical application where a total of nine models, five without interaction and four with genomic × environment interaction (G×E) and genomic additive × additive × environment interaction (G×G×E), were used. We assessed the proposed models using data consisting of 278 maize lines genotyped with 46,347 single-nucleotide polymorphisms and evaluated for disease resistance [with ordinal scores from 1 (no disease) to 5 (complete infection)] in three environments (Colombia, Zimbabwe, and Mexico). Models with G×E captured a sizeable proportion of the total variability, which indicates the importance of introducing interaction to improve prediction accuracy. Relative to models based on main effects only, the models that included G×E achieved 9–14% gains in prediction accuracy; adding additive × additive interactions did not increase prediction accuracy consistently across locations." />
<meta name="og-title" property="og:title" content="Threshold Models for Genome-Enabled Prediction of Ordinal Categorical Traits in Plant Breeding" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/5/2/291" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Categorical scores for disease susceptibility or resistance often are recorded in plant breeding. The aim of this study was to introduce genomic models for analyzing ordinal characters and to assess the predictive ability of genomic predictions for ordered categorical phenotypes using a threshold model counterpart of the Genomic Best Linear Unbiased Predictor ( i.e. , TGBLUP). The threshold model was used to relate a hypothetical underlying scale to the outward categorical response. We present an empirical application where a total of nine models, five without interaction and four with genomic × environment interaction (G×E) and genomic additive × additive × environment interaction (G×G×E), were used. We assessed the proposed models using data consisting of 278 maize lines genotyped with 46,347 single-nucleotide polymorphisms and evaluated for disease resistance [with ordinal scores from 1 (no disease) to 5 (complete infection)] in three environments (Colombia, Zimbabwe, and Mexico). Models with G×E captured a sizeable proportion of the total variability, which indicates the importance of introducing interaction to improve prediction accuracy. Relative to models based on main effects only, the models that included G×E achieved 9–14% gains in prediction accuracy; adding additive × additive interactions did not increase prediction accuracy consistently across locations." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2.cover-source.jpg" />
    <title>Threshold Models for Genome-Enabled Prediction of Ordinal Categorical Traits in Plant Breeding | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"mfvsvYmQ_VKHD9zm-0Jiy3G5LYlA52n8X5CstcUKatw","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459779","apath":"\/ggg\/5\/2\/291.atom","pisa":"ggg;5\/2\/291","ac":{"\/ggg\/5\/2\/291.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/5\/2\/291.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;5\/2\/291"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;5\/2\/291"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/5\/2\/291.figures-only":true,"\/content\/5\/2\/291":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022H47u1yuKOR90kfiS3mCf+6bFhRhwfqPBcdDOzI5lM0JLqrVZd5Gx2EFl3qNzhkUWLSkr3jzX9zUc6s3orIOcYneiXsxEfTTRJGZBFzwmxLlpEOfFKULlknEaA8ZKVet70QXLBP9ES6n02809kQhLZwt2p2em75BKIcpmIqatPJX9JKZhxvLnKAHvzMFAMtr4\\\\\\\/a4rxXXYvLOLT0o490MkX+cRBkU33XL9hBOtMe+F2+L0+YJZGpgfNlgGv5QVU8OcRAinx2Ir+7\\\\\\\/muMO\\\\\\\/DCKBGzu\\\\\\\/b2rhP2fFP0ksnccpDV9k\\\\\\\/PDQeH2J9gMYZW2laHy+15KKkoVRQ661J4CU7aJUv+jd7vGDVwt\\\\\\\/Na5FBARSdnuOF9qe8DN4iTUVyOduV9DYnuQq5Mhl3CSwFxwf3b6EBx1TPoybub5Vrvbzsds0YA4fOh\\\\\\\/5l631iyBRahWiZsUygq41AgEbBsBi9nDL0CoOUvNdjQrESFiu2RNwXneXceZCT39BCUWp6G5hPELN86oxn+2Y1Zr\\\\\\\/BPb8qkDWNL116aS53\\\\\\\/gybeo7bNQ4YxEPONqCbC\\\\\\\/ckxzz67CX\\\\\\\/kpI3D99hwqXJc31u\\\\\\\/xXyTi2HPyiLNQsL717wC7SFS7zN4IVZf\\\\\\\/vs8CQe9KKoNTLKh5B2R2HnhCCbFgGbdmO1AsLkLRstru1Sr2WVY\\\\\\\/NyxSJDhaj2P7BPt\\\\\\\/uE3eyIGQdcIC\\\\\\\/B9+VqnIFaef8GJvwvmVkBV5uAIp48WEn1dokD1zkSC5w2XB72my1ODds2\\\\\\\/ykUZmvDj5IKFoRvJbe4SUFNXrQxELKUJySrkS0A6VwLw394wpxU0hd9Uh+U7hR7Sj5aFnlSjfcv\\\\\\\/96KdrXU\\\\\\\/D3\\\\\\\/82HGkfNlwGzpXI1ZFCqZgV0t4Da+n3amYSi\\\\\\\/Cyk1pvzMiyH2QhH3eQqvO\\\\\\\/MEiRQ0UpsaTGK\\\\\\\/QeUSvn0w\\\\\\\/6YMqYBsKCX1H6F53wiTvd0iCR+1KaPqHgdfemgd7V\\\\\\\/C79zYsImbYOPb8jKqQpOU9sC2Y+fuq4MHdqlHtPm775Wt1dv3rEQnuVFvahIFEtqYgDmx\\\\\\\/+\\\\\\\/MA1H0nw8u9uum3uarNBrvdMKUwcf4kwk1z8huIpdeomDJQT++XgGXt6iuP3A68xnkHj39VYqhSV9iTGW9sPMsspmqKDySMg2bqnBUN\\\\\\\/FU5k539rM9jaCYF1FO2QzKI29rdPEe2oueiBE2BOXxKROW4t2V8S1KPqS7k6r3kJm3zNDhO+\\\\\\\/G2ez6I\\\\\\\/v2zW+xYRI\\\\\\\/DPOPZCgKcHbZtRGuAhUCe9rR4XW782jqyk3scm8TgPIFsMK\\\u0022,\\\u0022iv\\\u0022:\\\u0022xgV2EKUicM6x9NNL7hiyyQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226b91a61cc7e0fe56dbe5b01e97231e185f911e1f27b76af8bd922df95ff4f183\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022X1SIAUY2R2+fJ98d\\\\\\\/+Qvjw\\\\\\\/x7A2MjYNpu3jpIor7cEzanO5cc3WKtolpqbZX5eNK\\\\\\\/ZAdcAaEuIlTEY\\\\\\\/hVPSUm6aPTELSBFdgIoIpv0JOumIAZHHowFNzJ5GdavH0S7EuBZzlMFrEyOmng6lDn6hmRhcsHThYdDUVJCzJK6V4BWx6WP19rKtu1DR916EfObamH\\\\\\\/z05SZenXkeJHii35HXyT4LAFSSfy7eOIW2UNxqOm9wOJATEEuGqFSMb3yeD4s+OyxZw4GpypGNfBJZNz3RJK4y1mGU5euYqDZXOQ89zd3M9N+Ef9wvht1IqoGVAGPBc8EAIKkO59k2HsGXwJacFG6IiTfs7us3xTv7iAzq4QWx+XLZxovKiyXcNgYHNykNws6hnMGxb0l3nI+iQ3F1M4a0hj5quCzV9Bz2SMWlbxTxL9eQ4j4zjS2ErsiDUE+iHbU795kF7YgcXDRmH75L8mXg4DazMrgEjbafMyFaxI+b84Cxi14qr0d\\\\\\\/nngPaCWg6iXfcbroZTuoF\\\\\\\/GfE8+A3ly8adSfDayysyQSf7GxToK+9nMp\\\\\\\/r3o9Rx+++gixjJPzzauoC\\\\\\\/sMcyCp\\\\\\\/qPN\\\\\\\/8onpgS2ZvWxr+BOYnRn342DGXvREnyCmYu\\\\\\\/tbSfpUB+Z6ke6Y1Mt2JlQUgfMnniyFKaReOSd\\\\\\\/lJjar\\\\\\\/YQdZpVSwNSIhgAVZkgyMauEdiJe0nvX70cir4mNC21GZhJFdFJtWFykxiWy7T+R0I4SPx8q7wBVlqPgj76PzQ9NbbU0yjhwOISqquI3WKVM74gYClq7P5CXFHb+zbW5YFnrUpXA2JC3W6Us+QcaFYBIQ0lMtgVCWbRSVomgKfiLmyPy4PKGd3\\\\\\\/vVgWUPiALXFe17nuAa\\\\\\\/uBnWQMvSkpsTXp0yGWeSvfPKj5al8BYfan0HzxSE\\\\\\\/NEB795wjmrheRAcx+B0fiIXWeavM0kj7lVQDlEvz+wpxizshPU9iOn0KnQf7vq2Mb1hBNP4c0QW5FaR0QgeDuEW3IsVbpy4Ej2grRcSM1IJi3M0HAWJpauz25khrIKAYPBqyIaord0KNXhCijyvzIula6DerkULU1P5\\\\\\\/XPAAiLW3+9GQO03W8t4PsHw2gHIe4UouH6kgcYqVrWgS5TZi7fGRX68ylab4o\\\\\\\/UGooICAFap8gnXkRdbbP9UPOf2fJ7cxCyXW3UsSi6XD39mXXMFUKt3PmM1kP65EQnFfPi4FlCTXO14tnj9zOxb9eKOcCohGqAlJNmuNLPaUKEl9gYByMMs\\\\\\\/4IH1vZbOOlYKgcy0NYpNkNdJeKuBVo14z4gRe6IAOJjd2sJdSJRpxM8Qt6zsyex5\\\\\\\/1fFwORkpXgQSTWnlsv9EsWoDw+XSsr0+jGe8SW2GKV72qD9j9Ehl0nmXxkMFyx1FEMaheV1K0Cbits7aH9d+QyH5dQs+mEL6dQiIzK0VW1ET5OFioBaIkCu3T94Vy1Myu1CtuPBIjNFemRgymzwMgilymwO72axL2O+jylG5oanwRJ9PHM4kfEP8ocxVYWfGqhknfaKtQK4Ko3ExBTe8mtf\\\\\\\/whCnEeO\\\\\\\/I2aPa4cI6I4h+7X6bTy1OCSRd1CPXGMGi36jUb86NJrhb5ihKLyO3GcspUxtMdDlsvwXI+FIZxbFcwRnj0dJvjqLTbikd90Zc6w5RScAoloRcDP60hdg2kpm26NvPKPBn4VfA8rqAXSJ+hP7i1rsMphbwuneS+Ls92Cu7wMrGlEcyWZ\\\\\\\/KRj8w1IUyD+mqNeVyR9g6+SaSN23lgfWXHp9\\\\\\\/rV6tj9cYXd2mhwz1Vsb7QPUBMYfF2vP0Tcq7\\\\\\\/i1fMQ7fklYCxk4wPS4\\\\\\\/E0NCzm99kIVAI4STii0vEqKi6UBMHUQh8tUJXfpynkVzV4xXMwyNZIzEZhW2a9JUuJi89s6C1YydpitoipPAopcNP326C5y8DWlL8gchIs1paMwPqsAP9VY+RW3PI85rSSOKrvmUIX6rMnjpCko2eKHAFe5J5Udn5TF0qth2Kn5XEEByc3snPgOaD2XB83JEdNWhtmFNPuPPi1R2fONj8Nm+1hrJ0Hn0niPDrjxL4AYxyqJupUH6YptlYUT\\\\\\\/XHXaXfvfMSce6XlvtQ0e2syM9c09n0B2ar3\\\\\\\/C44tRh6EP34QK0sAULPKw5zqsEfkyTiEGDZ7Tp1abHYSoT3khYTIyKsklf8A9bmRCWf0CcCECw2fv2W3kOUyZ3LSivtQb8S370CN+Z7npxT1xL8jJvUsTpqm27r\\\\\\\/Rjro7rni4BXYaFzAlXw\\\\\\\/pXlxmpR2680IYCb9Mu3qc2zTm4ZajhUa8yjRRRKaXuQbcvU3hyyqYY0gIXkG266Al8+niPbqbtnttHfPFB070502+TO6q5WLIzwXiXUdUAxGAK9Vyex0mjY4m6KJ85xCCOZwS3rOzCACNV89Y14vmN7ip2Ijga4fut6LWKS0ZZPrciJIrcCSTRFZbw5ZZc6ZKb6POjoYJadEaHogZZAMYgR3DxzVqREom0IVZx1D7LcH60Zu0\\\\\\\/VGzntO4U+\\\\\\\/8yvl6nrW1poR9e+xRUVO947fi+HuAQAFVxay87D6XHCAIecd2c0ksMLjHjPbmAz919H5weQa0hGimGH+A3OIhAYp7Xok8Goj1+uB7eAc6sLr1sjwaLAVOrFgnMhVE98RPJq6fQuhBRc3ltfoardRH7EGMiWal0dm2UWQseTEwUU1VtRVXqgrT97NVDbYQSk0vzUaisMeciWPFmnjXIYF4rOmi0U\\\\\\\/++hYa925dqVM+Xw0tM0PSfZw13ILq56DzWWa3xtx1doMTrVMVx9fEokGZRrXlTDvVZFEOerylN53N74a21hW2LpwDRuUtzos3m+9MdSn+YltbhVG4lI7u+g4ZzyTfwphrtxMoNX6s8DsYhM6\\\\\\\/rl5zW438mb+D3P2v+MTZ0JApHi0mbYte4oXydJ73Ol95VQXQt3GRK82mmIlEIYCQvvteQTlNW14sqtrsz7YKijXskuqFecLis8M95buCqChGaE6IgFGhmiSCl\\\\\\\/4QvOIPPuMkKYFes7h6nMipenfki9HvSS8gpzV196S1jVKOyjLLwKQgg+DXJMvDdUmvXHbmOAmcmk1upSweuyajqCbpj4DACIAiaytn4DJHzHY47JUVHkp5pKEe095X3xCuu121hAph0p0uZ4YSBinw7T34j0UyDzVggltB6lImZwhGqnxyhdfecKCVqx889TOms9YT\\\\\\\/40kDtbhalZRoNh86f5qkZ0a0VLBhyqgYcm5aBQVu+4RLT9ee0tsdQmHfYn+ZfYJdQbFTLi7qtRyBVXMmo3SF\\\\\\\/ClZNPtvXjVJl2dPaLT0Wz5yi\\\\\\\/T3POOS3mHkZX8SuuX953U9PH8zzxQsn\\\\\\\/1DvQHFyDRpDxyz4C7SKZ0zZ6IcNBvwqreVTSrNFQa\\\\\\\/LdEb0KWstNLkRHdHJNR7oCH3lvuESFTwLS\\\\\\\/OICEqp\\\\\\\/QnNvs23LPDGjSYQKRK\\\\\\\/D7kbm\\\\\\\/BMsgFkhe03PJUe\\\\\\\/lAFqrm4WqGJl27ZUWEh6fZ2MHpMq69P\\\\\\\/hTJF4a8gwjVzGAnT82y\\\\\\\/WKmQEtzQPJYx11dLO\\\\\\\/wGHWOh1pczSyjPsCuLL29bEXC8fMQDQBjt0O4udyyWP\\\\\\\/d5ftLrCauESCo5X30OjYktKyWGbW6egnKq7qgCFxBwZq6he6bgsJbSpekENf4bgbZBRYWtvZbYToFkZ1iW3JKR3EiLKK80PoxxiO7fkJZiOXv6i6Tp9Bgkxo6At+BktLE2DJQmVTTVj9XNXdr4JUNhHjA2C+EEOYBKlWaMWCHEk37NLdnPR78iUEhbE70WM3eC+nvBiWFCx1MAxx1vk5ugBKt8L0SZt8KU0ep3THcFy0W8lQnxLmmevbA515Xjis+rJcvm\\\\\\\/qkm7QhDiEzALwIgwSVvkRs0sL2v7+B1qnX14x08f8EWWZAavxlM\\\\\\\/0TqAWRIYsiaSfAMwn7HAvRnCM2o2KNLITPE144aTq+fuKbMSrminWypaFosoG0J15Pcy5HGvlgxj5JcVC9ojG5KelUhfTpSMSYbC58wBxTtE5snHStBp4IMTPDOo+aHUchHU79edj6TW3yUkOLTqwJUlhixvACcBkT6TH8P+Oc1mAY\\\\\\\/SoAzLQYrJthWQx8v2LGmF95Xml4i8FdsrVl91WzLdj9bKi6YEW1HZ3Uq426hAgzWWx9uHAymj1NAg\\\\\\\/vRj8xVRz78W9x3cY7GAn5FXfmsmgRgQtm12dOfbWerrYAmE7smvLdcO4yTtt4VO\\\\\\\/hwe1zR\\\\\\\/gcNgTvQMmwhH5Hs6lvS1VHA\\\\\\\/odo+1vdQM\\\\\\\/XRM\\\\\\\/s5QdK9lvk2C504sLatJL0xS5pMFcSex\\\\\\\/9+WCAoiaGaLGRvbODDWhLUo5dvMoG+8SQRFXkpqrNqaMsBAhFOTpoX039NQ+DymVC7S45NRm1LwN8mbnOK4vtmZzSpSbGyqjhd+\\\\\\\/2NPabUdg56niqstPN81+7BTKjEH2oyV\\\\\\\/VhC+7HxejlyAZMHOS2ZvE4JGbd+dR8oXxdVfe\\\\\\\/QcdIOi4Yg+S+jhlG6DQdSjRUBJV3nqaRrsAmbZ5GcAOmSN8el9qLRGy8AIr9OQtyTeuTgxOBAkOQTJnA8pzvbBkA5ECja6NtDumzeyvMIGxiFEokNH5MTahVLu9CuhGWZuH240jX5DOR6pcqJFcHab32r0ClKznZou5vu02dSAc8zXesyiADQfyraIRoa89qXufWn3W2SmP3zMH+AEcbnD1jdJ1YxG+\\\\\\\/pIB47Zkhn6hgeJDQYQYKM7WSjn+kElpao5JmN5IgD\\\\\\\/6KylSDN\\\\\\\/0W6KuEzJuzbs8pW9uy6lL5EI2IwXHhTClZFKGncwfpr31IARsauIXp91Qu413EeClMDz\\\\\\\/lzek6vcN0a+yXcYU0DeV7DUPmUzjMLgJJ5991cTK2vqSzWE5f+QshknVgd6H+j4vDu5osfl\\\\\\\/5nJ0nr2Os\\\\\\\/TMR1ggI+lxjxBhCKJL+kuZtOQfukW2kbq9hrZ4Pu+93mHnesBCJYPZaWEUfFCjf9bWPwGd\\\\\\\/q1\\\\\\\/Zdz+lsz86a0d7wxe4HqiBS0+UiHXSoyCho8\\\\\\\/0s5Dx5O90OvnN7Z2ODDkd9Ehzxa73h5+ZNpnn+dgDj2X3cMZ4\\\\\\\/i87h3j0AlFbSiRLTV3i6OISZCxALt4bRhZ66Y6XCq7eCpC7uzsKMS3OJBFlERk3KrYKf+a7fRCd9uSE+5RMu7B4tn+G6GbU5pTpnq96m9TW1nIqoP+U6sBFPqR3j7XQE2BypOu33dSk4KjHXG\\\\\\\/Tfu2lZGK\\\\\\\/VjnywZX\\\\\\\/6PAaxhIw18iPBDx2jzOolnbU2ctmetj9iiubpgXNQjEL6iZAULmz+eapjJ5NlGwGD\\\\\\\/+yoOBUjjT6gt6en4o76kW3yCkTRGi2n0ff+1Kwqfmo2pX\\\\\\\/xq2POmhaxle02o7xevySmaNHnCxgxwGnzR7mNq7a4jOga5MLfaxDp4Dxo7fqilnGLuv6tk\\\\\\\/7i2WXrE9rRhfpeGD9g2X7mepSOt2fOe4D2ATA38MSGlod7yU+BQxcZTJNkkjGgAx3G7jZl6afXx92D5W3YZZHEXqSrzMHr05Dgx+gqfgpfZbcdKzhOH8Vw6AaVc06Ir1dddMiFTmsoPE\\\\\\\/9jY3pJY1l2oqZD8ssMYBrygKvcrslxUJELCn5wuUzuAfiPTCsNfV41QQ4iqW\\\\\\\/IeIm5KIb3yCCmlhA1J6a1ivqZzjxZAgLh+5TP+2JH2nlBhc7OWBXwvjg+r33Dwh7J+7lOqxY57OF\\\\\\\/WGqDZbadsD6zYpn8o93cMBJjqRCRz5c8tbWwrq6MlJ4aewhbX2U2Ax462pnsPsgptKz6bNTAc3OjL5QBgJoGmUooFpmFHNCGUA5p1bE5itSJ0Q8PK+slkS6rO2MXvKDa+MNpXe3dG69wnsZNSG81tvAcXu5YQXonuMMw5ecYT6g1nmSIOCL+Hk+oQcqHUJQr0fKbrZM4shbLJk+aIxLeP0M4yZ6aLWvsJnUmtW4n0zHl59iTPW3OMHYiO\\\\\\\/FosXIBbBnz9C1R7BtkJIR15SjSszrrM8RdIrv1aZMgWKsltj++Rqi4fEeRohqU4svuM6neiezOS7WW6ferPLbXe+AQTcmKhsit452WZn07FO1JzK3HjXbpUSGhj5PEhFzMdwSro9izGz2TLKHjXHTSI2YqJae6621ZqFtBvGklIegtMJ2UN2Z8bj5nIKFrSdo0B+RKYjHICyKItN4jr5NmiWZOgK+mR3egh6luSDCEOxZUQ45O9vFjsUK0+r+qfqkcB5E7P\\\\\\\/kbEcIaxreFyOUmv+lp2UF8tTkrmpjLoBQsjFJ3v33kBHHrALkxmt3jP9iD77DxnjApKFWwgqDCArv6Mg1hlKhZauWs3JqNaLmkZyCinRK0Q\\\\\\\/xVtAf2HLWwc8YydSypxdeS5jJU5ybIrMkdletMYBA71L8h71xMqq0oXdam7PYBpeahXV8sW6vSe4y0WNJ3TQMV5IcM1eowXRqT4rSfKYnXlHQ410yfIELS9\\\\\\\/71IneWr\\\\\\\/7hT4TSvybaSXwU9QJtv3Fh+APqsVmUnHXks2xdIs5jZwqOxg9bzn6MD5uCwmhW7+yAezcv3FS8nX1CjeFspZf19duoSSt312RpGmeMOMkiD13zw41vy9KGkldpBrmziCdvxqUwaPMuPc6krniz+axwa7SexPooZ8aC0CNmQyk8PFhTCmX+1fnj7WK0oDomJzox7aE7OAmeq1b5YU\\\\\\\/qQ2MCyzpjuB8nsDdj19fI154kVMfwZlbNQsrssYljtXdWgAoZLuUb7Y+F9uvAgwXBPveSwFBKWtLwLNpYOOBETlQxL1EteW2u2MMoDlZuIFlBps9aXYYVSEZn47brlJ\\\\\\\/tx7HsLeXCz4JqDpeuVpu9UjSY5igsYm5GXM0LrTUiX6sNaOaWVoq5Ky6vwnzzRUOJhEe8VTccSezy0vUzrAw8uDgDJycu+hpH2WJVgYALfha6RL45xUMUxAe7+lJkv+CT456hsDjIimpjK0sc6WfWDTOFB0XDCok42p0RpLEc9ylV3mgCM9wpWqw8dMUuhTP\\\\\\\/VjZwRR8K\\\\\\\/HwkvqxYHjEG2NATt+H0oBGWh4gq2T3MlWcFuKKDPb3PWncBj5jlSWaRYPfek5nHzHyogXF\\\\\\\/qfOSZUHOPhGuaynjCWE8u5r3guNeqC2yuoJlA6yjf61qiTRS\\\\\\\/pn7379FcL7cKosfUo2DrZ1PN3okGjzMSaffJ1qaBs52zEK19M2J27K3xiKTP2hlGC044y7RO8cv8dAA5tDw4rWfrztk4py+tFc+JDqFQBvszbrhpi++qHkOaMqM+Kv+n4gwDPyb\\\\\\\/wHxj\\\\\\\/ryMTIEQEowldu3dxa0XyTON1qtdrBXnrW3Binnhdv33+lyWG+ebv4Yh8eqjC\\\\\\\/n6zcwGLuCOxSSYHTUSX8Ur4bz4Z42yYQrB6UzbPW\\\\\\\/8W6wF8YnJNi+2ETD0N5ugETUnlnz6fqdjGgju\\\\\\\/P4oT\\\\\\\/uON+nNbSYjS6ymlC0WfJKdYc6NHmf0jTLCK\\\\\\\/Ux1E1MLpJVXmoVIgLTSj26OMSRu8pc7l6ESTgHeCbe5ZCNd356yJLhvXa3FRGRQmUEi0MXBDDM1LRKYnBKu+goX+petzidE\\\\\\\/GvoWVGc+raMvG5HBMJfmXU+hKRg0p7vyo+1pGgX6kgSpD3hZ2xhmNQk8qtfnt4i41K+IOTbTC2AraQ2ZWF8lMQ+awnUL8nox7Mj7vyvFgNVnaHnQMmeGZHbwLkYiHTZzFHxwLIUxcI8qP9Yo387UbaD\\\\\\\/okFcFjrFSNcdS50mzshBl8QBP9FDi+2zULXv9smCOwWrHoRokIjFVnRgZNSu3nhqyt7p7jYSAcSZ7xcu8UqgeQlzsVkZ7YJ4gMiyjuBuNYEBKbW\\\\\\\/wQYj3s9VZcaulelhPKqXHPqjkds4k0RZEsisUhUvJrpVhCbtye0Ymgeo7Jwtwv6AcghzugWoebVDQHj6a2cTJtwF19lg0e3Wt9uElLlfrE73odhjz92eFqEKYt7cHu+9JqGxtLmwrHDH7s6Xg0zlUlNM2QKimfvOACbFqP70y3CHGU5SM0aDG1a\\\\\\\/yD1iLSKoPGCDo+tIxLNCkW4xjlqVhE76IOb7DcpdqNVe2xNmB3Ea08UwNBOgMcugmXA3hRh4UX8+xSCNLH0yKQ8Ljz1vHz0kDTub1RBCDSRAI1fkJwas2Bvfuwf+GZkvWZx3GC3bfG3gFfLLp8g12YApmZzvndC9F9De5Mew7SxQI6Mb2Be7mPvm\\\\\\\/JvP3FEr\\\\\\\/61TCEUapM0HqEQYT0YZz9+m8r2adob23xxvjAn6u2VCRIqxLSezbCatiApgcIZ9vqt5RHbBWUs3Zk\\\\\\\/uhFtnAPEFbBur1Xj6jDG1ocLJN4HD4jb05BM6gdz2rlVKkx8noMzV1eXL6ydsWoxhPSqiTp0Or1vhNI\\\\\\\/I0vVAtRZbivPjOnTeTuSwU1yuuiwuFD17YuwxBT8gTe9H6HyBTSUme1zshSvNmRknkyml4Sce\\\\\\\/qdE8owAHpj+Gc1ko0ff66KY7F3lp2dMq3MXo0tuC\\\\\\\/PGWdtJy\\\\\\\/FvpkVGgzC1jrus1n\\\\\\\/Dfej\\\\\\\/vBxWjtv9skAy4t0pMzOY8YNDwQpl\\\\\\\/vkTXyMAHbN1HN35hYXuULFnhKpHteT8WCb4WdnBcPFGQATOZj7MRL7fkPC9eTgDRwVgPZsWeTmOGbWiVJksIpK5erKALECydFEqhmCRnyYWT06fqu1+k8v9eVcicnX\\\\\\\/4zkKmdegXtCKnPKR+Sjeg+9u7JiizOWQD4Qj+aTQSVTJkw4QKyFDLchdAMbYvtyOuKaqEbTpFqaD80Xkn8TIBZJk7aAhKsyzEc8HME7WlWEqBHFcsRJpyM6vd3Pxvpn+lEUDvuqOjc9p0Rh\\\\\\\/fCd0WeW+qRL2anHQCDJc4FLqRCOpYorqB5QCdi7428GCI+SVQHCerUfPRwYZ75NNaxjhyFj\\\\\\\/PzIwoiwZMM8rVrl+aZOzAoWmJjeuA6W7F3fvg+69pmeqzj1+4hhGNdw6HLnmyQXU\\\\\\\/5mWsO5MfWRIk\\\\\\\/kcSlSFepLPb1MGK4zpU2+VkOFeCVylneI0ePypJ+pWZXG2n7xNbKysVmqUGw4lMkb3cbu2GmQsEpLk8+Z1kGa\\\\\\\/qT3lExJNZsNBmqxHSDPtYN2QjYslOWaS+z2mtUZQcHflaQF+OkKt9yE5GP49m09E9BNcbE3XKZzfe9rpBHS9EkZNcBGUspeREAoMTEq1tm8NjGjEXPqGThH825DybnDn5WiAbdMe5sKAtD\\\\\\\/MHerFrHHrljN9L1IE6bbbuQUlfg8Rol7g2DXFcXpnTUSt7b8Krbmm6xXSR+vrOqZqcqxoZZtwTUh+Yhuoaho03i3t6Tjb+r3QEcMy5YLnVew6oXcZ+vqhF4Pea0Vm2tYGNX9aU5jVB0et\\\\\\\/DVhA6FrVKF8j1SZOoWL5nW8DPvC7feO8EkH7CEHsb+iOxVj4mbI2fWsiEuebC811R4y3F7gA9KlogQ7lFj3VWH5xm+2lObeJXaTB+Cm2uCWE7I630dG8Wyttx+3Tv8zxtky6au2dEm1HqgiIdwUe1DP7662oWOmza4ZN74Zh2mDhphadj74V6o\\\\\\\/aIhoo\\\\\\\/kz90cpIerss3L4TUa6iHF0uVLspa5+C8DnNcljOdqfs52xCFBOgjitTqOLozXm\\\\\\\/++EXsbXKorOnOL07az\\\\\\\/5y0y4Dkb4E0dWHhjcVF7YuAOe0NC\\\\\\\/ZEOXg\\\\\\\/Vd5AIDrMyzJqyY\\\\\\\/BnyylT52cDt06V4Dq\\\\\\\/FU\\\\\\\/1vtQ37aoMNPbfradM7Z2rMOzMyGNf0iS0blU72VSMTrAHtQYB+NaytUCEf1tAj0Jhekdr48I2sidPdmzQ2VYb6glDlUFO+Cv9r+WSXellhmlSWeKcvAjjh6dQsxrvlo\\\\\\\/UJ\\\\\\\/eE6Qa7436C8P3KlfcYjqQ9oh3x98ed0Gt7NmuaKQXUl2K9JG7isB716260gT5ButUw\\\\\\\/Tg2TP5FTG8anV4TLpw\\\\\\\/fqSrMu3AOmXITVCa1Fr+Awnb4s7+CWprLe5z04LZPQ32Sn7oowG9J4wqSV0roAE6Kzz+DqogmFwLPCh8RRjv0OT0BsgnfLRzWlMOamjScjFWB6z0T+wKFgg20eh\\\\\\\/LKWp\\\\\\\/tE9MC05+9qrkowABtkgqX40t7MTd1QY6X\\\\\\\/q++TNg0D+ri2PMKOU6TNbk7eCE\\\\\\\/K0C0dKw3huWdjrKUMsN0c9QXoRxddg2Kuyyi8VUeb7gk36dETdVKlFgMgSsY7CP9yx\\\\\\\/r\\\\\\\/QazfKOZ92Hst47jPE9ZsBs3RMVPwbFeek\\\\\\\/CBWn58qAaXuBJS84Lerp\\\\\\\/YBJgtrIFzmpeQTshAZ\\\\\\\/kU+RRsvLJv5d12z39LNq7sYqMMyDomGepyu9fVlVubPT2tUj8T8acfOuT1nJPSixBd8u4tKjUQ0SRxMQIQN5bBwQuVxE7AOunL1R61fjDraED3FrYjKSIRYXXrLU5z1LdzLCCtouDe2yTAgOU3Vr7DF2\\\\\\\/3itNkiXv14+1CTZYkdiqrqtKJYOxSMpXalSgrnHESuFDJYdk72R7tLyFzqHtP80J0Cv8LiFFxpq1wkSRJMsI7N0t9Go7sR6JzvVAnR6COf6FQe8o8E2BVNXzzTgYQzZ7ls0IcpUp0ce3+c21YIrQV8V1eoQmKDVgMUIL+FxujiqUsHkUBtPVOWvQ4gGgoBpujNjk9rPvDtMvGfRpHa8XFaei\\\\\\\/ObP23EphRSThyBJpzmwCsT5cGd\\\\\\\/fOuhMpyNPeGv6I1fmYDdUP0zxNTgQ4UWYHgI4PJwWX7wy5LVK6jyhkneKgvvSoN3VQ+\\\\\\\/YDE6JjxqlCOWD0nngAvgXqfoKga2xdlQSYqFFUj5sMfI4qgnWtvHZBvOOoMjRC7FKxGGZieVgj0aDNgMTCXKxXYsLWoIhJxBLOjHU\\\\\\\/V3YPuRPW2qYn5lMK6pNGtGeF7QXLROwvButNWdgYKyimPARiQqzRwJGsMoMHFgP\\\\\\\/BGDjGfB1uDL6gh75sSI+32bzDn891GsN9PqB1ZS5RTjBb2idF+fWE23qmC3oWYo0uR8W9KnWaHuzlQJL4OwOSSlr5Jb7rJBTQVmJgw1XoLb+IvmPhvtTBd6EmoSTMEl2eegAawiXX8r6430AXdhKTWEEoRDi\\\\\\\/uNlAICygNeB2WFA82XJUsyj55xK6y2lHnuaHfjAo8w\\\\\\\/7pC1p02PumY+0cMDusf0pt5b7z4xY8JZIklKXifU4zQhAuPM5Ok5Ub\\\\\\\/pNQRIvoY7jC1j6cT96hDmpJpVhKYv+rznG2I9MSrz3KaZA3SzOD4TAneDcAsVqqhkCbkkQjKXfEnx94LDL2hiYA6+jLYMuJUqvZ3bcqnfO8zryXknk5QZYqHYsM7+laKL\\\\\\\/KyE4LCq5Me0fUhOYUstW13kDBCa5mI4jAyMSLdvc5x9I6xBml+ykQHpjLWb1o8CMw45MBOFIO1pKSDeRqZcjkGDvXYjjjT2DUHhM968Bf3ANxR2+U6szqC7u2KxIUabrfxXGIO15FNC+YmSFHl1qDuaOF+WxO7oGtWfrWJ+jIVg4mnJXQKCI+I2BTUwAS3MAB7ACn8DaRu282E0XEFLbzXcl1FDdK4xyTVURVGSmM\\\\\\\/My7o9wEdQb2ggDz7thV9TbUXpCa0eqkZu4SSeCf6zEaNg8rit8zyd2cQLklfarYUkVx1S6AOzb42f0CoylBSWzw6AKyHFVfoc+72dY1y+VWrP9DpVn15C1lj2TLePtMnBKzoAdhLkJhIDaWuLfWQ+cN0yV0SWMgN5tSH3p\\\\\\\/QnIKHQhEIq7vMVsHGMq0QRCuLSL87KRpYbhcbk71QJ99R9zXRKztrNq8\\\\\\\/9VJa4Qf61BR17DyRSnN4AfFQG0oFXhzIDNPdhNPSxTWsq0kTCJm5CzI7aZlJa37irK8+8QlX8\\\\\\\/5gPR6SullAkbwxcM\\\\\\\/yJoDvlpyiq6EHgcECMAJyhl4PhoAAHi8a9GKHogp9sNBxJPVAdECEecNuQuSBfhTDl5mEl3VBG8wSVn4GhJlVfeUSdBnJVOv\\\\\\\/VjSgcKG3e3KxlFVSwImH091is7+058Z56PKEcoGhDzzS46bNoTt510f9ZcHWddjkT9dMquS3eskOfNBv\\\\\\\/7DajPV9l5A5TMMG3c\\\\\\\/GSkOLLAifyaQDIF8chiIosMoWd+eudcGy2n7TH8NS7AVx72avMd+k2Z3NNnEspZvn7w\\\\\\\/3LwP8uh41zSo1SJrqE113S93O2+mFt4gjmdhKZoWghioXL0YBY5vQgfA29YnW5SIGXSbok7tfBGrRvzsfRR+\\\\\\\/IZa6fWZBmdzqMsYDTlgpYEM7mByVVePrx2YPgWV9mo8\\\\\\\/h1UGjK4uZlz8a\\\\\\\/XfrjXyaylhd2gLFciIwWc5oTiqjwolWXDg\\\\\\\/ua4UhCtI+3Neup+3xBDt37SIDRz\\\\\\\/eHJT855tC\\\\\\\/QoD3\\\\\\\/aHe2YIa+vtlpcJBTDxrVjaMPKFAKVtvuADEwqSL4qsAzo7fJJaiaNSqcftWrB0PL9720Lv0VQ5HdMndhv53WhyHDYDN5ntjMcVvGWFwGMEfjBRDh20uEvNrIMMsH0MYvgwUCbZ6WqdH9T7zYVNxRSXOd\\\\\\\/qNmfkt0j1+fyoEEvIBFPL+qh+ahYWLzdh0lhbdMGw1GA\\\\\\\/Ze8LHqKRQhx34ynMDUwhnUcS59Z19hb7jXtSJoPkXFY94PBVDV3867aBX7J\\\\\\\/T\\\\\\\/8j9WzmDnqpmAp4P\\\\\\\/L7RqjjDDMrzjiG1k5C93fyqct8v3v\\\\\\\/hG5vVrSOqqdvjcYFEbNMy\\\\\\\/dwDk2eRHtStgkK0a4eFJxzEar+0FQ6ISHvdQT7z525j9yc+RRbV788OG4f5kd9HNZBVi363teaYaUNSzJ5ZVCu2mQMSNZTq211LEZLcj\\\\\\\/oIn9JpRRfujTcyzK\\\\\\\/tOk9M14kJ8U9cZoU\\\\\\\/Ct30mrYd+4GBBevvIQrbkF9KbY8teMI4YhF5FluCGvT+Ozl59NYx3UOnGnVSWi\\\\\\\/OlYlDG3MpQ8iIeT2YhRscSN\\\\\\\/E3eklVwojHf\\\\\\\/q+bD94RzvGZoyDExk6h1AJEfCb+uR7dMGDiIoXB7ow3IxEC86CZnnN8QRUKQKfWVrdxSB6p5l0BfdylzVc4eI5471rbABWQC0uOvGHs9Cs2Y6qVppEWvm0V16d4mfjhrx5zM9GrAGzVWB77nFAfqij9RPhPUVRYphjXPqwxQeXStvOrLKHPIo7a8CBJqC0RxgUJP3rHwYo1j1O4uJOmqREQlx2R2BYkwDgnEW8IHbuHAPeGQeK2V58HLrCo9LAhpvenZxnQFN0piiPVzvd1JiE3zvk75U4TH8efEyg30AJkcxabk4qMqzjA1VXRXPeSWFzFOCvKVA67k2SBLIemCPdDzPjtX2gzc\\\\\\\/BD6kCBpThaZYWhxJ\\\\\\\/Wl\\\\\\\/Y3y5VX0+JU50tTpmmhFhv+hTz16NZqTLl5FiX+4O5lxprD9tiIbdnNleCe8ZDQTt8YlzS+O+WvobwXDRl06A1jTOppttHYim0UU\\\\\\\/p8qX2tujxGYQ0LUQYEr5Kwfyknqhx8vcDyJHiWK93cFcwLcMsCifTIjI1\\\\\\\/fkoJOhaxJKvzQw2vmzch72pIamWumnViSpT3HEzymNc1XlfO34YtUIoX1tlAVao2vKJS4qmYwY9cFzN2Y62DE76Jhrkq6EA5bZgiq2YZEAPdmbfsxySSJ3\\\\\\\/6\\\\\\\/LUzEYWj0qolrHETIIFGONwGN1uIotGc4VSzbIpRsnaBXycotmjlSJxf\\\\\\\/9lDwZu5fWzkJYktfDSUriyura52rn+2DEf2cVhgmxmfNCSrLQGjtSyQ\\\\\\\/ty7figjYk7FbJ2QXXm5ITjD0VdHqmAf5c3M0tcxz9FNEy2fsaej\\\\\\\/WEuLW1x8qmh88yXYrQ0jd+fwXIVBcKIYW9BJ18ptRRmpwyv+vlh\\\\\\\/tmrDVz7KLsTPORryX+ZOQoAb997qVyAJn2NNNlBnZw2lrsjTB98fZQyT8o4z1DdUks\\\\\\\/PRIaomAwtM+wKTqVi3hfZ1TESbnvivEeUcR6ozNkNEiNBx2e\\\\\\\/dbMYFWwLmXSFScE4w4sQ7X\\\\\\\/uvlcGyqW9yflWDoHOnKAWRLOennvfCpOYqGQInjvzY7XTYju5ERihv1qen7I+JZxvoU4700PrOAyn8sElBY9P+oIRlUNdRKzEXgItQ3JTtA3V+\\\\\\\/kFPGprZY5Xg0VSyL4o7loj2HYfukIns63B5j0GUQfZ4VtZXiXsGgkvTP2jL49zpOKBco\\\\\\\/S5vU3OZXJlO\\\\\\\/nTLrpuazbTJjMiTI1nh7bH+HxF2lmyVyYoHLQs0ZfhT99S7lQuCezbPHO99\\\\\\\/TctWGZbVWw3d92fgZfyQr2YBt7JPor6tuaQ2LI0J0uD4zuNh4+rdxaiYq1332dJ\\\\\\\/+hDryinVzRrBR7ZxToGQwUhmLKfOAdOCFDUiyzatnyEy9cFJmPf9IbU1BCe\\\\\\\/3OJbHCjifSzUG2kYnI9fDVN3K9MxQ+FmwS5byUv+0okO6w27T1EeN0MzMUMqq42t0pm8E6L2IEhKW\\\\\\\/kwwYGUIVuejj9BK4aP2pWhBri85dSMNPbl47DqH96lziYRa7C8e4mdkTH0ZI0EoTqv1kxe+FWLA6c2aua+ff\\\\\\\/nZMiwrDd7e8EoHnPxOcpRicRNehk7kcxy\\\\\\\/DAMSkkjh+MorqEtutXCfhAH\\\\\\\/ncHOYGROAzOVnHsMkyYObIy5+SCXtA6\\\\\\\/eDcsq\\\\\\\/Z5RJT8nl78mPbFNifKsbZCFb+BrDoEXXiWGSSI0Revte7ilXT+OeHdQtTSzE1TZS5vHgTCNPjDfk3XCTPH65xy6IN2N9UOnqDiGy0yLqcZzLj5TCNcg0D0Q4JMM21C8C+HBU1uhwVCSoFhmuuHpwxuXPc7i0TCkTcO8MXs7H8qw63n7zKXsToZmQB1GQu5djro1BwSme+baSWdcXJfOZPE+dbt\\\\\\\/PCKYosyYhErgN53dOZ4LSLzJU88OejVUrbv4t4\\\\\\\/EUx0U3wIeTWG+jDZQA\\\\\\\/Xo9\\\\\\\/1v83On\\\\\\\/AI2KIvPTdXdyRC\\\\\\\/7e9yCWrVlELTEBw6sDx0WN9QDYsaKYCP1aC9KCLlZuFJlY0XNYTlxuFcICh+Ck3Dkl1f9sCqSOQShkD6gZ249fC+Gq2jwUbxXBdFU9zv9frSaVWfYut4MKwCw5vjbW3twPPOc9cJ4nPXFlHM+gEPGizhjXMQkDOUSZtcU2+iYnXydg2Xvd+QkIYLx+3iF7LZEjBRXKsDngWH2\\\\\\\/oGvIqIu2Yj5oU+9acIk+Q\\\\\\\/qJPuq9uf8IGsC7OGqeeisi7F6o7ekAWNnYLDXlP0+wBfkWx8+Hpriqof7qeHqqUL92LLsiz9F7H7fQsSFZEEPTsoVYmCG7dg0iiJ36VcLD9I4pdpU6rvPuI8sTSEUNF1KlklCNf8Wnvvhd6XS6Djy2qL0fwEiRluEhi3fXTjn9Oa07Y4LQsFWuRvHxU9kUM4eu96u2fKQoqQKcZvNTj2O1Wqe9lyubOmS1\\\\\\\/TG+y3LcXl3irWC+8nCc56nkjwijL7bVqNQNgmdcxJNc\\\\\\\/o6holLf\\\\\\\/TNDtVqbMNqHlCNe5+8h4fPL7fCbDabI102WhmhcUse0vuzfCRb\\\\\\\/xgB+9CGvga35\\\\\\\/wnLmIR7hEBPgf5P4uH14C0JlgXQk48DYPR1oXde40llEX+SLlDkHcAw67jeZzh5CgqP\\\\\\\/LIYD2CoWNey3t\\\\\\\/M+4fatdHS8PMaM7Y0SzYrhTFbn94QpuvLDmC\\\\\\\/YMW\\\\\\\/cEmP85aRT8op3Pd5a\\\\\\\/Xf5BsF0v25gSs\\\\\\\/zeWBKvXrSYsdXLnNVRlX\\\\\\\/XbJEFVfVmRJQ0sIGiGgKSApQc6GZodo3Bd+O2cLnPIEeAOs9i\\\\\\\/Epd8gwyjiHmHXkfesT\\\\\\\/T3x2+jz0GWfFuUeLnxq03BUdbp+1dSnrpfJgzn+hVec+0YFRf3cmDaQ9n\\\\\\\/yG4PLC9m9A2ERTRJgovnvJF2\\\\\\\/245rFpN4jlYbQEptedRLcjCuRu+jKoJ+7leYgS93RC5vRdiW0HbKZNyF5Qx1iGu6AX2JbQ1RaDJ7qhZD274d\\\\\\\/ZQ6Oo0bFy\\\\\\\/\\\\\\\/n2KCFKtrFvUgxoe\\\\\\\/2CjQxY6p0wLQUKsUDXCxmDmn0xHK8nZk\\\\\\\/l1mkYEKxbhC6DOjq8XyEyahZ\\\\\\\/ezxDK1ETy5jhQu6m8synEJ1CxxmWQ4OzFaG5WBXe9QPNXoiuAd6gVoV1jBPAgKm7h5tvK68mHvm\\\\\\\/T1mDxGKRSAq4UVRwCdj2sgtAE1v420jp1F1FcPlKlOBGjDEiwu7x1oQ+EJxSc1+tR2Qk+FPyvm6CcYRxVw\\\\\\\/uj2VVbd+m8cl2VSCqVv\\\\\\\/wsaM9SLH2BXblintvNCd\\\\\\\/RB4YC\\\\\\\/ZT0iUl37nbji7rAyKEgAmaxCHBmT9\\\\\\\/9NLeL+NOJXBYwTfWs7pw74twRDfpHTJzpCflUh5CWLM6guX7mbhd\\\\\\\/E2ywABwYsBQphYLZ0tYCu\\\\\\\/C2RMcqDCf8pMcrR1ax963d8ZyDyuEjV169UNAF1uOzAw+Yi0BvhBPwjfZJXtKuh\\\\\\\/+I+yQdOq+xQBqbga4M4FPqaQDerrqnnXdQ4GOI2BU6Jjq7vOUNzC2yOoOifezkMa4y7NNqESRW+c8+HC0Fbqoa\\\\\\\/KKPFmjnqUw0nUOKO2RofDgkpG1259+O31VYvGMZst1r3N00o2y8s0gQVzDE4Zgh2lM1zh2PUuMKki3zSzpqswKn\\\\\\\/egEnLJQdR1guWqgE5XXC6LyWBD2dkxeT9ESeH3ygeF8JNzsT+h7GxJyFSbCWLi7cmpaf1fkROCFPBo4YvjVnv4PC7XuuyerBkZWEvu4++YLHdilHvh0vvlxvxlWdAtx1AvsURz0agQe2IN1uIN67yDcdo8OGIUiXJ+WIW+v4Jl4yUYE7p3vR8p2WenuD7NRYnBy2uNo\\\\\\\/C6J10TTesoPlvWpQvpObFPRjQDmk6+yI2ZxmBpONjaCRxOolmq6mlivjBJeZ3Z32e21il6zmwrfWL0TAP00ZLF9\\\\\\\/gWisOeu+8GPaGpVJ73VabL5FV5uNh0AwBBx3Qsssa4hCZvL3eG1jprkHacej+NwGDps0yft8lNOokfL7LIxhBrMfBRD+qU+biW9mBS3lyzyLdukHR+fBlyOaZA2mxY44O09HTaiCuyndGXEyyybGIx74NWNKxbB0pK74wAVskBM1D3BYRuqGNsn0OO+UsBA\\\\\\\/7BiW00vyK9QtYlivUOLQU3L7hDGFZcMZ7ck44EHxGyikEvWt+Ks2L+P6Jp4HM6WRIj1iDLxif1rHxVIqJ3lOe5bFDIh98riTfIBxYob+2l8QzU7Cd6HB0C2MQ1hxjfUilUKVuH0+FVKPY5Akiq5nz71AhlYSONgfMH8epifYA75GYIOsp\\\\\\\/DkLGe3VJpAL3AUzmjx3NUtIE9bg0+e7LBV4Al21OQ0XgyQ0+6ARke0lEFs9boyGsKJRi6Jd5ig2a28NK68LrI7111P7yWi18\\\\\\\/v6zeubR5KAj6I259q21mUeTngtAJ+u3g3xW3CB+STlnyCNo3YFp6bpb9NqWXF14ow\\\\\\\/0BUnEaATAq0XN5AKZeZR+BgQX97W+lKGewXvOdG8n8PbdR\\\\\\\/mCNZXsT8jaVIYteuHmhgyc1RBxLUAK1UZgK79ZMIh1cZ9x9\\\\\\\/woEldnyx8KdbVLm0OYylz213Tz6EmrDZFUrfAphF1GgKIqVK7Wn5HrajqyGjdBES+1No+NOhZ5K9RExBQKb62RAKVxIWTiLew62Ce0HBHt2BoO5R9zrzjcX+GvAgTzJ5\\\\\\\/+TKu0RxfOhkj+ouc\\\\\\\/TeFbAScDZv+tewlJUwQriujRqmMBSQZJXA+697bERX9VDCZK8UXsxpzl0zg7HRA9\\\\\\\/pt7TF7u4vG4SYNl9aCkIVEQfKh9czhuO5wo81bBBz3uwY8tw3QRVWQYaGL7sPGFr1Tp4Xo2NaN7Bkpk8hFEt0nH2mWluCnFSui1ZL+TJOBCtSQvJJTkmOQ8bYZqurrT\\\\\\\/8Ub3oIM1MacKssGs+4pY48Vphf\\\\\\\/aNJ2Z\\\\\\\/MVSOsS1xB5TvaKAl6wBtSSD7KI7Zmw4m5nRvpGfeRCvpxNeULzPsGw3pc1Dp3+Yh6RLlnWyT6GNBbJklNvA+t28HX42UWrse7RwkYcPavNEc9OK2FU2Y2JfqpJgJAX6NIX70sJHfYmRjarGomOlkUNdwhdWqPKBmrNPvn7gPlxoEOuzFVJDkwBr\\\\\\\/g8ssl44L+jyN+WrWTLTg8tPCbft6KZt3wydP6sPRvSohYc7PJLDpB6u0x+cSeDCl1rce9Drwa0ZXBngIHjRpRj09+9\\\\\\\/mNh0Gd0TAye7jJ9Tz+Kg6G9sz\\\\\\\/GEK9i9Rf9vi9zJVegXzWf+9cxofdkwSOZp\\\\\\\/ly\\\\\\\/MnbQ8YtH+bdrp0\\\\\\\/f5vPKdfR92NZ475r5ox9mTEGDo6bKNIkrxxUoZ0lT1P3OPDpPPuGkg2LeuM\\\\\\\/4RgoQVbCVRCCcjmu+QoJqIg7dw9Wz8TCCj0CzUudzg8EEamEFnIQUIHVnzcmTPah0oKudtfbQ5DxGJUvYBu3ChP9XARbCIeJOkzmJt5BDVAG0VqkhrAt0GfrrOEIdk\\\\\\\/GOeJGNZeewlA7MRaApW0ntNi\\\\\\\/KX1UnskEeCp+iaszVX2U9NY2n33mKNRn7HAE3ktWql1YNBgbgPW2OPbyCeAEdisyEaXWR\\\\\\\/LfHkJ54YSpBzaJXrTu3m0vkLzN0sf23h6ifv1KKJbgvwexNpYMYHcuZ3YqYkJZqbIkVEm9tm5CLWX7P0SmPSPFhj6bC4\\\\\\\/a66A+MhOOO4EjPirxu0yfsw3e2wHSVoPeb6iGGYISbJAbk\\\\\\\/31DEZpjXf7bZ0BTdZgsnjU4+ZGZ5btVt+4MxqLnVAz1GVC9+e9ZrRpMDhzenvHdu2b\\\\\\\/gJgsRHBDC28kMlKjwkTdDQxFnDPW6qgxLsm33hwK13ajYhdqHZki72PJwxMX0CtF\\\\\\\/p8VJb94VClW8X3+y7hePJb52kkjYP9gTtGLCJ5VUPLCbPx5CcF2rzK1xCpEuWh3AsQX8u8NgGuVWFX61sNcJ43Q9el6NoAf+hWKwV\\\\\\\/Ot5KW9lyowXbg+t+qfog6aV+wm06QC20M0Ha1pQQiHZpPIChOqHCWHJRmH8a6r0PFBnjJZtGca8pdW61lcaqcmS7LuaFyUo0jA5kNjSclYAYyrgg6mLOu2TVnQNk0lbx4tEsQAZu0udXGKwUFyAlkJOrPN\\\\\\\/zg58nxbZkRyhHaU0PZY4L19MeXV4w+mrpUDjH9j1EC0LeVfJN10j9GEbR1X5AqdeIkFUGdzweZP+To\\\\\\\/ARm1x5uNb5S1p+\\\\\\\/azyzTTbEOknf+Eeemf\\\\\\\/CRAFyuydO3N8SQEVNlgMUkg7TAfVpbmpYcOUNCuTAn56LRoMsRygNdyf4rsc4xqsFYBqBeDFIH81r8QI4TRzGL23pH6V21B1l4knqCmriJoopWdqJarMeY4dJEcbgqXHWLFpkpbMqNNEEQYtHpN7YVzSbHHSpcX7Y+BB6hJGGheHAmmVAsVB2OJnGGRkLZMNL0WLsl7LJx3m9roeq09AlIXhhfhm0OEmnT0mcZXaqvFY\\\\\\\/F3o6x52Ucx6u1s+RWEd7oHnIyIjFfgfqcHlB+RjZMGFat8h7gPLSIbnEMxo1OWT72N5er4sU7mKvNWZEUv8tGd\\\\\\\/WCBPjXaok6CUNR1F5rmV9oxPvTIznC0cFYLCL5mVuCPSgnVm78ssHsly2bD+mT63SAb4wE\\\\\\\/cU8nNCPOnfZchSEubjvM4IdImqU7nEapi1ivBa\\\\\\\/L8LZgjPavs5hlUGiT\\\\\\\/hFd4jBYVE3l\\\\\\\/3KAvAkDzlNZzo6BnPRYIQGtH5d4xqs9M4l37koUdegKhrxFoQPzYbWz7sjO1M0S4XJGP3mDr+2MHDMRunOobIsfhv3\\\\\\\/8JAmldFCQ8oEjoiLJ7xxYZIZ\\\\\\\/nqOApyXi90bSC5OmHE2+VmPChsyty8ARgN4VlXePM2EYzCBYWgnIruI1kIFxafcJvQTCgdkMctDc9\\\\\\\/FLHYVOEi6D6E1jFQ1NTPcXQchrxng2sw2cb0az4IaMpwbMkU476+CiaAW2pKMsNM5WN8YvRvYU89iJ1Sy\\\\\\\/8c2OpKPKNk4IVPcZsUKHo4shEL1jwt1FbJcfzlX\\\\\\\/3JMfSi7MkrnnI++Tq7pZIFV\\\\\\\/uhvRwm0ITEKLRuDwRD92HxYAxNiNpmbEpBjHwo2qLCaEE4wl36Co\\\\\\\/NUhguGmPx21v1sPgXH4SppbASQrKtGSCR5qXFRG\\\\\\\/qdRUtPSLe39tz8BWlCmSzs\\\\\\\/2fSiPrtUngfyfFlivnmpfNTgKb6cb0g1rlwkpRXj4EN\\\\\\\/BVXR66jo6AhL\\\\\\\/JCAOT6AzKuamsekwxjW80+PMz\\\\\\\/Y6\\\\\\\/+3q\\\\\\\/OD++jufO1snbC4hT5zw5AfFwZm7Fyo1s0dK85YqMsas1HkgNWlhy6IhfJQSGQjvKqB3JWqngf9\\\\\\\/6vhuczFQ0CQPU+g3lIsgnjvl\\\\\\\/3QgogCNFMZVzxkMGHpubuILgL1UjE3Z1WE\\\\\\\/n608CuJhdUnrQ4psFf1rvEGupB6ay1oyB8gDteVQMFz+6O1SLM8Fp2sayycuN5QwgtGTNhTwBbkrHkV2xTYAkPx4CPY8p5kRn5bPAD6phScKCTovR87IDjlEy6gin41wV7P1W4hLR04Y3lPQGvOBRrmCY2fcLQynnBREfmLt6wPbUraghQsM3O54hUY+gq10XF8QIDaQKqgiIL3VtQd+yeMLJdRdxnaYG21\\\\\\\/\\\\\\\/\\\\\\\/7xMyzj12+NG+gMt+HMMW3\\\\\\\/vp5BcKYwXlhkfB\\\\\\\/npkzz1JVetaCAQn\\\\\\\/+Fo8mfdGJKrlq9B2Ka+Au7slZ4OFqb443s1OwJgHB6NvR0kACRFDZM2WsaWUqz6Pkq9acuojYiILUtAadVYnHkyCDsudrGhg7VkrFGIhYCaM7jB8Fime71BnyBZTcEd+OyzrNxWO2QnaXrcBtflGex+hhklcsVgzVNMHzqQLMu0mK1JCaoCVGAf\\\\\\\/\\\\\\\/jqX6A4fyYa6YijqN0IIlKATWZk9zduBRAXhYddpqx3uxFYIhoyA\\\\\\\/iTS1bXFojh6TIs8kjjEHmTCI090ns8Jizif05rbcHAxxia2G\\\\\\\/GwLVE+dAH1Ge\\\\\\\/DCdspCcWRjW9Nw8OwZm8MWMWcr5CRej6EGId\\\\\\\/Qb28Zare0ZmbLUsgneZelsCEAYXs1dvhnEFuuVefidDF6MwukalJNq\\\\\\\/z2DwYeZd4wGDAp8cmOMKFKRU6I+7t5EsY9Np8uaMB69eGEES4tGY7YanrsxOUVJsqg0dNv4qGJmOCKBNHnIcQvRX2JTd8R1t2B+r2gexl7+RdZzm3PkPtNaDnGSXh3sP5px6fl+6YgXoCuGdCWRTVa+lJBlK+kqXZeOwZdZ6YrX\\\\\\\/7mBH10DdYN0H6FNfO9UVNGXhobhDESp4Z+pFc1Hk1zVwyiM1NDMgJRz7sySePLVzX6Cuhkvis9WSf6nUNW8wbHy6+6R8r5aKNfbZuHUBKsnFeMUu4C9SqGHZFM9r0SlQ3tw8J+j90FY70WYOiMOVen2uU8BQOAgimtWR05fjQtTM\\\\\\\/Zj2OmH8sbKbevQgzBVIDoraLNcpsH7zPrsaEkCTt\\\\\\\/LK4ACS39tA8ejrt76idFNIdRTweMLJrM688V2iGIIDyHu8PobNgPZfxkug2ZeHiTb39ylzSUht0UBuxhQR+dniiLu73Uwuyxuqhh3hfNcyi+AYNaMUa6Oo7IO3IKpFypmwZr20hJPn5syiBVhfnubnT4CV2tOpPhi4t\\\\\\\/UBaDIFAe4embAToayeFEwDc9PIvyGKTrXiRGFkB1CyLEjqaNXphHqYzXtxJ6f82UmPwmbCnE4rYnxFqVw+s7Imox0Q7nkOdnMwufcbTxdk3J+PNqdxn8dhqsbhnnZ+7U4\\\\\\\/PFlCZuSFCXCuFjoogDufPBGCdAyTZ1Q5bZuYuerBOW0pNPIFd9QsqweXT0A7IB1d7hMKP+s9UMg6bF8XzI6VxflaNkK7tYOT2ha2sfldwt8t8H7cQYo8cfxx60ONeTuGOZ7wpk5aIELEU+69b5OmIfNukjDKhFek80sIewJYqrG8u\\\\\\\/9Zldub4TAO12rZxfeCNDc7Ft4LssGPSWV9J9vKsGWWXSTU5+CZ749rYakxNwx5ROCE\\\\\\\/+WbgcR9myDFsw3Nt0rjWhq32YQh7zucgB6oBQ2Peu8HPnIxl\\\\\\\/ZZh9r957g3J5k9x\\\\\\\/tz2sXhW+AxvhIbI85rPsJwPFhMizb7K9\\\\\\\/syo4N4qB0MfVKzpO1H8tJhF1MrLE9puJfsbNFdnwfvJhroHQ60txs0fRiqD9MfJpHjSM6ChBE8FIYj+fJQb1v1JoUCL897WjzlKeeKL+Rd6UEDGdWk9WWGkAh6tXAAK5Wq9IY5D22QKj\\\\\\\/7yZ7XC7GRqqxM7e\\\\\\\/bAYZVsr52a1EDMaMDoASrZ6DMpdoqHQSua6xDNrutyln8juGG4tgGoTAE+HaLMsAYSyih5iXQKu+SYpOQktfSeMoxT5zYkICXdTdUL4zAsudYRP7w6wuq9nW7TaeLRKbkv27cvvxZIt4uijyldtSG8J2g8\\\\\\\/WItlMFyrCr0FmgBvi0ufGsEUEN5Tcd27Ej7o1utlE6pM7cUTjFZsXn1JZVp2B7ob6uHR7+k98rfRRAtQ3Bbv2IpM15fY+rc1rUvTRSAu2iItqpBaRk3zwkSN7JJ0h1feyyXlYwP0S5o+E5K5rODrMrsTGpIpWo84t1QCMhEP5KmBZ+a4\\\\\\\/XSiR++39phoL1NJ6VfjcTtLAtyRTqDXwI2CzTGu7l76WHL5IGcjS2wXRWnJD6eaTVFQ+2syn1HmDfqd9mI8Ohaaxv394+A7ROroomQ1MreApq6KBiAV97PloI2rC7\\\\\\\/OGk43W0rR2hM+vK4uDP010UEVqisH19r0lhgPfI0pNPw2hoYD0lb7o5X4amBlt2\\\\\\\/q6ijBdV8LjHxj\\\\\\\/l56VGWATCdxpnmUrx+CE3lLX\\\\\\\/MBvpKFh+P\\\\\\\/YLFdQnuTkSvNxhIuuBetjEkQ86dFfv4H0d7UAthxkfUFIPLtC9zo\\\\\\\/cCnva0NAbdhCz7An\\\\\\\/Qb6QQjNHzWtjEEGfI+93OVTQgyJEll2CKrgAbEVjlCQoWcf1PZlGF0o\\\\\\\/XorrwN\\\\\\\/WJwlLh2qkixhBmSToRW\\\\\\\/x4iFRWy2OaA01bL+f2QW00mR\\\\\\\/N4dxSrvef+u5KiCGLFGW8zuSKMk1xA1OAC\\\\\\\/FRmW3YJAU964\\\\\\\/PlFMN4VNZhx0PcH7WvgPXPqJFqyc6mXIMkMSYqYspUPW1S89iM8Hvg7NszNRciftHNkTo9l2tpCyDDq1aUgJhZSgCHnmwIMlH8Xc3NYCg2L0AzBRaHCxKE8SBl2nQk9KAQmTCpvw71qmufzChYjteVpFHG1A4Q+g+CSr3ipoLHwjfO4FwPTrKpUlfWbkhq46K31q6nTgeaG7zRncOlN0vanjyfQ6SonhtI822C3tOU7bmBniLGhi65sjWuMpXHwd5XDkfu\\\\\\\/JdgLJ\\\\\\\/k+zv+h6+dPrixHTSmDDKpsH+jOmm3Jfhr9ZOdlBfy3vauiQvXyJHLjCBCziicqs0AtAyN8l0jdydkoZOixbISqxWRMyw3SWRbIbXobc8cwtRjRvsTHVN3RRAEMWmqAIXK+pIkCAWu\\\\\\\/WUEqrqEPvfoOccy7sRGfR7qv+tefScB14uMlDdctrEmEfAWVKoZOnRWV7XZBhVgNSBMs+BsSIjmXb2xtcuIilzYQA9j2tq7PA2gMcb9+qAHMNpyhULWXEmbAjKOtLttebYrXGBloZRddeZ+3CxtL6B97ll1NsJFJUXUJshbi9Qn\\\\\\\/glkHJnOsWPX3FtDxedjoOujQMq2gr7DSsew6ywRMksP8uFfkdWnOwbGjCGMwv\\\\\\\/dF4ZZ3b3fs\\\\\\\/KVkf264LORnJTlC3SrZ4t7O+d3a+hnlPTo7mjb98+jBo5GeTCpqCV1oLR\\\\\\\/RHYK396O1XIR9M9rcVCD9ZM8PaWk\\\\\\\/hfXlaDYEV6Rn+m2AfXk+emUr+PjcSkIM2DD1QUORziWstPGZiAmwt3SIMEYL2b1DWCXbTw\\\\\\\/oJcq7j6Hndh6KCnLHTXEyE2PshH02DuM8s6ELaCTS6uCaGJtS2\\\\\\\/MIVpvF7FBsg7HPjcGljg9RcwPAx21dWGmsjTb8COliupManUwrHw8JF1Mh1diTxw+DhNRXxiPeuqElpibEzOlBfetARmq6yXLUaQpI0M5u1SgU+Jdf2ttqtGr4wfzXdZQ+bSgJfIb8pmpFUJiFRkZiIdBiBT+39idTb8hC3YZAeFaOP5bO1K80fphLwT8VOvo1xHzJzB9fK4RGQn7gyIMK2GFZB7mHIw\\\\\\\/6CrtRs+xHxcPF2\\\\\\\/a5+tMKmJFwpPCf5JB8LYkfdAa17fz30u+Mm5r7rSbuKFN3cnWAj\\\\\\\/1COImAxAPDQTErzJ8huQEzQVPxHszPk4zgFomZuO0xiYKEyHBL91X5InVTDR2yIx2WY\\\\\\\/lNCEkdmDEtuNKzuhrDPx4kmYl3t9\\\\\\\/pPf6kbGIYgCoUooaepVw3BB96vedb86yRPIMJLgB6fTJ9eiJ4Ut204TBSk4VFL+qcNSTxvq6ttOSrnT4izk4iAtc6WM1ctQDGnAtYKVmY9T2WosrOfYx2zrueCKAKUsval0E+5XF+xih1fJdWCRnbhVCLu7NCcj1Le40UQRQD9yETcdHxxt3D4gnGvInHJgUv2Q0tCPGJFp1TjbBtWT5MPhgG7heFkdxzQbANEAyKtcJImMXioNnM8FO6rqqRe\\\\\\\/CdArRmhB6GLAMsp2NB6byFQ8R107ChsdrcbZNuzLnhXV9oBlM6QIlfmR\\\\\\\/i114GT0pC4u+\\\\\\\/KxzRSwaKlvXYDufFQrIlChezJwNKz41vannf1PckHgT9mllvK5z0y\\\\\\\/zEAMX3F5q82iES9Odwq5mAL+fMWa5p7Ph\\\\\\\/FpJumpB8dAUQd+iQxmYbAmEvRcJ\\\\\\\/3n9ozflydKcHAZCpVd0vBLrH9opgwA4KuMaHPt\\\\\\\/b5yz4yk1RkIs3YVk25dIy3HostsulRHDd4SxmtPsKpuU94peDZVdozCqJnMM6GCy4zOCm6ZBpMnxmNigyG9ZFElbdrTxMrQIJDXwyVq9zndbRcnIyJs\\\\\\\/eWbRZxtpecq2y6d\\\\\\\/Xm1viOmiO4ZO8HAI+z\\\\\\\/\\\\\\\/SNIiPFAnIxSMLzG56Ui2GoDoNVSHFKnS+OLVTcaaffg9BcRPPPAWm\\\\\\\/pAG5f7F19msUGcl8wKq1NWeMN4RqivwjqVpvshra8VMaoxrPcUadvkSOYll8YPmvTfAevKavd9be2ukwCogMntv6Wal\\\\\\\/XiGDvZJiXQn4yy4b1WVliUSM65H+ekF2PMmVVqPHTIjpCq5hx86tNrR09d\\\\\\\/Pv1cdBPjFYjYlBiJ\\\\\\\/2hIkvw+ysHX1XX6qa3SvvyJDXIyB13wEh8aROFFYsmtO5wKB1GvY50Aerl8j4Vq2nQhC+riblrCSdvzq82GqK6Yq2J0wGG9992aMYQOodzoK+F3KxFbf5aHJDm4J2MNjc5jYnDzMwNhtsD3RxQJGkN9w4l19Ce8akjXgUWk8RGO6JJA0TaYyjdrcqwoWucEJNURwPEsGkHAm23ID6MWTbEqyF\\\\\\\/FFX27\\\\\\\/jYp+ligpy7UcdEa8tp1HokNpGAnnMU\\\\\\\/QLxXWxAfBeH2LuIkMCB\\\\\\\/V9aCyyXoMzDKcTQih\\\\\\\/Ck81QQSJq2OVOBNOruJKNdxruYYj3d79uuyDFsiAVrMNqIls\\\\\\\/pvEIbDqSZ9iWXG69fbyPLRH4BzI+Fz9LE3Y9cbcTBo9epNgaX\\\\\\\/yUQ5mkqlnhd4Ah0gA5gPROm5PRFst81fP3M9R+8kRsVD7\\\\\\\/Xjr\\\\\\\/uQWqFhrX6wSuAjsHzkWW0vS72rXSMKhF+VE4isgkPgYc7o99ymU2rv0DyR4wdIBBg0naW7I2fcHmKZ++mFCPssvVsncLK5LkfVnUbLdxzQmjWyuH0tdB3InXSg+2BinLlPBgr7\\\\\\\/qNTWmlsVoH4HFJrj4VILVD81pjevx6NWUF\\\\\\\/kSQfVK1bbhp3EUzydJB2O3j8MMlLx7dlHs3jTBtprwhdo9T7vm5KO+U9d2obfaMFUkBZ0STtRFuatZ2MTDgVXygEbLZrQFvSR4S\\\\\\\/77au0gmUXwe9sSFLbWVX4qNbZVhT7wY3CfSjf+R7kFpPftf8CJACmzuXsTwePUasyWFroQyY\\\\\\\/AGhtraJ6Wh9Ju3aJX4+\\\\\\\/NYw1Y1CVwD09Go9myQjFxnD0Z+FPKqKPQMmGPHkumAUOMjUlCiFyqT4VL9lm7PmSaXUd59FkWt0OMbmvnebtRV2aEp5\\\\\\\/hk8iniU99kKiFEla1R33a0sX1aKk7kpsY59NptwqVIgWVdh1V9Mtm3Yq+C8T4eFLEKaQjDKegHGqq28eMThRIeYuR0tav34laYVZc7E50uYbI+GcUL9HZKMmEnJs6Ymrf\\\\\\\/75tuMGnl76myUVkSWyueuxj1vkdDXIIDWjfX4Wk5L8ofAueMnAGjmbBREO9jEmegsQTIK5f1k02W8dehThaWeDx2xvsLZTq0r7vfkKEggQ4kjh1B15aIyZL0FmOMRLmeq2GSysJHu+\\\\\\\/LCZGp5fg9OaJsFygsWwa3Jf5Kpm2hEPr9uoojYjENCzThNoY+aGhgl\\\\\\\/pM6M9\\\\\\\/5FNqqZyETA4cL5QfvU6idiKiRFgKPFSYtbAVnoHTkIIo5szVsLFoj0KoAN+t8ohaw6oXZuW5HxrA0fmf6PVDgb2QKOyEdRE1QpvoBzCpgXxGID+f1AYSc8E9zCNARKnmUfsYV1ioQt2yFgeixcNtCdlhAIweCHo8Ys73ZGCQC9i1TRCZfqvtsBNjjmII8IFJX90rm+nwmjS3pbRuMGvgsiyhoXsRjfNKI22ahbT8n6M0k6r6G7VmSZrmRcWH3CnVQvoFLpFB\\\\\\\/wyzf41rXEH5AhGlgSSskcbsmqtBG8yyw0Vm4tYLZLxWBGzBT6KV\\\\\\\/0IQHsmXlhHF4TEM8KWf1hciYffqvy7uGRlDyN7v9p\\\\\\\/SIpXskp+4+Q91Z+iQctq0jZGmwYSbqywU6zCCrpt1wDJk+GefLJGBVN8qaoNUN3pgpZibEOZV40\\\\\\\/pOW6HcEFfn2Gll7M3R23BiHCKb4pL+FxvOPIjOLU7V88JFRVeHSbAVIhyZdQEiZhLSfWs3PuxBgfg\\\\\\\/mcemDrDbV31ulOUjWrSqmudOm136n31Wy4ncY9yo1Sbpco1cmuE2j1fk86x8816oQvnvtlXFZBRgxdEUWnbt6\\\\\\\/wPx\\\\\\\/mk94eWZwC1aJ\\\\\\\/sAIcc\\\\\\\/tugA6vDpt02qzvuuHPbpb1fFR5mOZDWAwPyH+9wZUyoxcFiaVXPL70veCHTPM0qt0k3rbKJfqud4BSbZK4hdhnkzJoEuAxOI\\\\\\\/7d12PuHKtrK1fitz0HwLqYVZVjcqdr97YAyQhohPW5P1Zkz22o1u2arnVwSJ7jVL5JmewVGXq+cB0TvrAtzvhRZ\\\\\\\/xTz0RHjP70yRcPLI2msMHWzG4A1QZl8OLbhw27j0Z5+\\\\\\\/BeCpyJboHnOG88iAQsf9xCQKcvB99RiqkQV9hJL8kkh8AhookGxJ8II3J\\\\\\\/FBpdLiAKIK8Th5EnO6+GT\\\\\\\/ywFvO5eF24IL0AGcYqGgXl7267kbVBqgjj4bhJ8rbV7BCy7KBxGzRMpTSXM\\\\\\\/ga3uoN5GLfscH0ce1kRqypWNU+g0BOkvmoOEfoz4i6Mgu5yZ7XdX8v\\\\\\\/qUUdqTTa\\\\\\\/U8YoFPN3VdsNg795HLW4CN2O\\\\\\\/UWeeIseCX8kJ+Zz5LhTOAFSS6aapYA0Z4miaNildJZC85u4dZHVeOXlnrrywzxDrnhMBlffQbCWf+SHLotB3IcUyJNFpTyIUP4eTjsId1V\\\\\\\/iQQKwpKI9s9zzZpwCGb9hV\\\\\\\/ueOt4q26rxsDfhOe4pxWD5f9bQfW\\\\\\\/e6hFmBF\\\\\\\/DnoFzjgmm\\\\\\\/VnIIORB31a7gCjC0u8AIp8Hi2Z9E\\\\\\\/XvK5lVBPx6KKve0btj0dKq1PIQ0+7nx+glLHS4Npp4FmN2DBOFh06jVIUvqLohPPjxJ\\\\\\\/lBunJ7MFimgwpqNhEGqNLPKxqEfgqxM64wpQwl0tyI3Ur7huM4+i\\\\\\\/6Yy4hQf72DTLyOfmBy7WbzMj3ZCKHLQ1PR16GbtjWcDvA01tR4RjbrYcAZYO4Lo+FIm4d8\\\\\\\/tJ4j88iah9GMtitN8qzOk5dRx5jBCgc3STYKVeRjzAILbWBySe9EAnMExWyFoYuKfUuFri0ArZufzwGneiS0cjKC\\\\\\\/9RRpRg6AwEkqtcQpgm678bqbpSQu6jskpuZBcVd9289wOIVzk1Dbim2tDoiJ9iPprCFqF8fkkn12GgDR9m7o0yD2rljUThDOzERo1FS+YH09nMi7gopD7j3OnAG5ChgoGu0vehkCUzlJlAHfvY8h8V8PfP4YNvhBhi48mrNxrXCxkcORsH6+IhPx8Fpnea96qe1yxf46MCAr57VPlXbm1+Ucvo2pyRgDtLdpO+b9XjSCqtmUioJtbH7BbdmMoEjIcQEFeslEuN78a4ULcEjUr17NCvSbZaiBkiRxgEp80bcrJobuoz+vZ71ctYjyO2Ru46WaOLa\\\u0022,\\\u0022iv\\\u0022:\\\u0022tcFpC8den+no5DqqYMrtpQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223214d4f8b4f95bd655d1728c5a959e17ee171c1b780a3dc10e33cde8460553b1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022g5xVflyEv1f+8iVDsvvr3B\\\\\\\/d3H+mDjzvOzJLMNe9oOMVr\\\\\\\/1n3VbXCaZUMfRqGXlcxnt5uBM6xoYzM6Vdd4en3cwc3t1rIvFT9aCEH1PFUkdiMPg5Q111US4vL0S5hi6F5xheGghQ49jhZ61AzCLFqkTqUbd0rWnPZigg0jjxMgR1TPq63xq6Y2\\\\\\\/S6xYLwXi9f10WaRqkX7cop9O+JtEkU8Yf1fr\\\\\\\/YqhYDOfYFUFmdKwp6n3xMdMcTEAhY6ToAYCc19UnSb1mwqr3m6yoEyeg5drAeADUe6xZwsjCskuf2ZxNUX806ROvgqrMsSoR204GEXIPHqUq89ED9wRygPLbKZJaY9dzy0JMy3fel80OleY0EKztQSb31OYxOTqKZ7EeSWDhuisLUpptSZcpHAn+UlpCG70T68diteqdLMsJIyYCaXfJ4d4os1tPGReNwIQzBH\\\\\\\/M\\\\\\\/XCqpLiNVudZXF\\\\\\\/xWvsOPrl2F5Nmfm3jTSf8DZYaA99hBytL+ndNDvqssvmfalr2zanlwROq2bnCi5XOiVuJEFps5yD86q8KBODZsqR40WpPlrUvbZTgfCx4kW\\\\\\\/48sCYIuFZafEfiiSOVSWXTKSbnNSjIfBwV0xJMJ\\\\\\\/d27rirgfYD8PlrexwKIkKgmbsmCxnBUPkyjY8SH+FmeXi6seFpAADKYGV7CkeX9x7bV\\\\\\\/Jn7mX1hmPYI2+Esf\\\\\\\/P9EOvgDZgZcm7G6lhL4cLk55XsFn2L4iIfDtY7f89Gf8L+4v8Q2XGI7dZ2acFUV2L4tvy66k8vVD\\\\\\\/YEwg+N5lqqgAu3VapSs11nWz11KyVe2z1thk5nz4UMfVUq24dZVXKfE0uFbYlfXKIYVt6nnbwbWZ6GpbiLrngygfRQgXWyo5CFeUT0danZqizTy77QxNbNgypdQG0MBepouXh+fA3rnHCchuVPIiV2ilPU38FgIchavJBwDB4qVLVNOa0hQRrAFtTr0VRDddhNOSVApiA1+tkLmTn1iXYJsqr37eDTgEGwWPSHMpKyYALfcP1KtfUYlw2nff4pGhhmXilD+cOywOa5gHygX3c+52TXqZ+J842VNqdsA4irIyB\\\\\\\/G7oyUu49zZTQqkY+aEfndyh\\\\\\\/aS3byJBVnddVQv15UWOE38KpibK5xd4ArER69hvbSRlEMPvfiaLDc\\\\\\\/sv\\\\\\\/OW0upizuyZk1xWycm6vDe5mfU8hffxXExHqXEd\\\\\\\/G91zF0G\\\\\\\/5JUDy8KVdm7GXk2t4LXjeT+QMFeahnNFXpeZxh8M4hpLfK4iW2k3Xb85v\\\\\\\/4aECy2WMzF1MWPUhHJzMu5YAnrzkTRl\\\\\\\/236KrwuRFb1Cl+RQLRMepzTcWid9nUhPfxEN5WW0fiuVykiiZENhLWA+nAL\\\\\\\/unFuF0pQSL2FHGyIu98+oTUQq7zRxaybeAkW\\\\\\\/AZ2tMz0oKOgM4t06ywDob4Nq7JJzvL6xX\\\\\\\/KvUD1PfXvexyIVm+Y\\\\\\\/iMpImzRRJg49UsBnXp+VDCbBB\\\\\\\/LYmvkuE0Is0Jsev8eBBj+CuJuGQwo4+9\\\\\\\/0GF3V60DRWUNWnNEgqFOr8ftQ4uV0KcJIRyLfsyoLaXseSq80jDquttjyO8YJgwTtbdX3hmfhuRrZbG80V6CCzvUZKdZ8WOmCZB9R0gUzjCl7v4J3\\\\\\\/NRLG9IhFr67zBZBvP4q1+D9arqAO\\\\\\\/8DlmqvKR8bzdOrV\\\\\\\/HAuEHS3RSX5y2FOtMxjlRcZyQFjMHctkBCB297hmTkz79ibu0vcIg5vVy2hnIe+pOenNcD9mnyVrQhoj541UpZ7OiSmzKPG281EXHH40aY8dhQRRtUSl0w47im+gRPkSsQRXcdKcR+XrnzxCELU4SjPxOYQk3DY54OKUkvdCz6lDXPRrkB+K\\\\\\\/CMf\\\\\\\/INKd1ieWIa31msb1kt3rdhL4kG7GDG8xwjHGQVn+R6rHzJJKLKNi6XXAU8TlPrled+3rVShB2qVKPmFG\\\\\\\/io0\\\\\\\/pcZnomazAeY6IasXZdgmLW3LELyJLFzr7Eewgc3bVuBD6ghSSAPTZNhwZfvWJ\\\\\\\/wlylv1iVhBEeojUJz0CYb8mUw+eCpMq8xN9qplzvbAaakmnj1M2\\\\\\\/\\\u0022,\\\u0022iv\\\u0022:\\\u0022+EtsMGYFY6Oti2YjKRUZBA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226327a5a8c3cb6312e22469db20eadf437ed2ab609f40f06ac910c841f8d07619\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459779 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/2/291" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-l8bP7AgreI7FNtD5BKObKTwd9f5RoX9XOuTcB7qV2Co" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/2/291" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-fg8qyAeq3aLNIhdLFNc5K10IEU8jMNo1b9q5CPzAr9I" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/2/281" title="Genetic Dissection of Internode Length Above the Uppermost Ear in Four RIL Populations of Maize (Zea mays L.)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/2/301" title="Ectopic Expression Screen Identifies Genes Affecting Drosophila Mesoderm Development Including the HSPG Trol" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459779" id="node459779" data-pisa="ggg;5/2/291" data-pisa-master="ggg;g3.114.016188" data-apath="/ggg/5/2/291.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Threshold Models for Genome-Enabled Prediction of Ordinal Categorical Traits in Plant Breeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="2">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="3">Gustavo de los Campos</span>, <span class="highwire-citation-author" data-delta="4">Kent Eskridge</span> and <span class="highwire-citation-author" data-delta="5">José Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">291-300;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016188 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459779" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, Avenida Universidad 333, Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), Guanajuato, Guanajuato, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Colegio de Postgraduados, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biostatistics, University of Alabama at Birmingham, 443, Birmingham, Alabama 35294</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kent Eskridge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Statistics, University of Nebraska, Lincoln, Nebraska</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kent%2BEskridge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskridge%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKent%2BEskridge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), México, D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/5/2/291" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459779" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459779/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/2/291.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459779" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459779/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/5/2/291.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459779" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459779/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/5/2/291.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459779" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459779/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Categorical scores for disease susceptibility or resistance often are recorded in plant breeding. The aim of this study was to introduce genomic models for analyzing ordinal characters and to assess the predictive ability of genomic predictions for ordered categorical phenotypes using a threshold model counterpart of the Genomic Best Linear Unbiased Predictor (<em>i.e.</em>, TGBLUP). The threshold model was used to relate a hypothetical underlying scale to the outward categorical response. We present an empirical application where a total of nine models, five without interaction and four with genomic × environment interaction (G×E) and genomic additive × additive × environment interaction (G×G×E), were used. We assessed the proposed models using data consisting of 278 maize lines genotyped with 46,347 single-nucleotide polymorphisms and evaluated for disease resistance [with ordinal scores from 1 (no disease) to 5 (complete infection)] in three environments (Colombia, Zimbabwe, and Mexico). Models with G×E captured a sizeable proportion of the total variability, which indicates the importance of introducing interaction to improve prediction accuracy. Relative to models based on main effects only, the models that included G×E achieved 9–14% gains in prediction accuracy; adding additive × additive interactions did not increase prediction accuracy consistently across locations.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/prediction-accuracy" class="hw-term hw-article-keyword hw-article-keyword-prediction-accuracy" rel="nofollow">prediction accuracy</a></li><li class="kwd"><a href="/keyword/threshold-model" class="hw-term hw-article-keyword hw-article-keyword-threshold-model" rel="nofollow">threshold model</a></li><li class="kwd"><a href="/keyword/disease-resistance" class="hw-term hw-article-keyword hw-article-keyword-disease-resistance" rel="nofollow">disease resistance</a></li><li class="kwd"><a href="/keyword/gblup" class="hw-term hw-article-keyword hw-article-keyword-gblup" rel="nofollow">GBLUP</a></li><li class="kwd"><a href="/keyword/genotype-%C3%97-environment-interaction" class="hw-term hw-article-keyword hw-article-keyword-genotype-&#xD7;-environment-interaction" rel="nofollow">genotype × environment interaction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resource" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resource" rel="nofollow">shared data resource</a></li></ul><p id="p-3">Since genomic selection (GS) initially was proposed by <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Meuwissen <em>et al.</em> (2001)</a>, a large number of plant breeding studies have assessed the prediction accuracy of GS in different economically important crops. These studies have used different marker platforms and marker densities, as well as different parametric and nonparametric statistical models (<em>e.g.</em>, <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de los Campos <em>et al.</em> 2009</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">2010</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Heffner <em>et al.</em> 2009</a>, <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Crossa <em>et al.</em> 2010</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">2011</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">2013a</a>,<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">b</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Pérez-Rodríguez <em>et al.</em> 2010</a>, <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">2012</a>). Both simulation and empirical studies have shown that GS has greater prediction accuracy than standard pedigree-based prediction, and most of the benefits of GS arise from obtaining accurate predictions early in the breeding cycle. The success of genomic prediction is affected by the choice of model, the size of the training data, the heritability of the trait, the span of linkage disequilibrium, the marker density, and the strength of the genetic relationships between the training and validation populations, among other factors. A number of studies have assessed how these factors affect prediction accuracy for quantitative traits (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kizilkaya <em>et al.</em> 2014</a>); however, little has been done with categorical traits. As first pointed out by <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Gianola (1980</a>, <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">1982)</a> and <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gianola and Foulley (1983)</a>, these traits cannot be dealt with by the use of Gaussian models.</p><p id="p-4">Several studies support the notion that when the number of categories is large and the data seem to follow an approximately normal distribution, failure to address the ordinality of the data is likely negligible (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Atkinson 1988</a>). However, significant bias is observed when the number of categories is less than five and sample size is not large enough. In addition, analyzing the data as normally distributed has less power of detection of effects and often produces estimates of frequency categories outside the 0−1 range, which does not make sense. These problems often occur when the frequencies are small or very large. Also, even when treatment means obtained with a normal approximation can be interpreted as estimates of the proportions of each category, this is not the case for the variance of their estimates as the data are binary and polychotomous (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Stroup 2012</a>). If the data are transformed, many of the aforementioned problems remain in a linear model analysis, given that the most commonly used transformations are intended to stabilize variance but fail to address the problem of skewness. Consequently, transformations often are ineffective and, in addition, express the data on scales that are unfamiliar to those who use the results of the analysis (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Littell <em>et al.</em> 2002</a>).</p><p id="p-5">Categorical traits are scored by assigning a data point to one of several mutually exclusive and exhaustive ordered categories. If these scores are treated as continuous variables, as in standard GS linear models, the following assumptions do not hold: (1) the relationship between genomic data and phenotypes is linear; (2) phenotypes follow a normal distribution; and (3) the variance is constant and not a function of the expected value. Therefore, standard GS linear models currently used in plant and animal breeding do not meet the assumptions required for categorical data.</p><p id="p-6">Results of empirical and simulation studies indicate that generalized linear mixed models (GLMMs) provide more sensible results and have greater power to identify model effects as statistically significant (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Stroup 2012</a>). In a linear model, the response variable (observation) equals the sum of explanatory variables plus a residual, and a probability assumption about the residual is made. In GLMM, the statistical model for a multinomial variable is expressed in a probability distribution form. It is important to point out that the cumulative probit model (also known as the threshold model) assumes that the process that gives rise to the observed categories is an underlying continuous variable with a standard normal distribution that in many applications uses the linear predictor with reversed signs for the fixed and random effects: <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-1.gif"/></noscript></span></span>.</p><p id="p-7">Threshold models have been used in animal GS to relate a hypothetical underlying scale to the outward categorical response. For example, <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">González-Recio and Forni (2011)</a> developed versions of BayesA, Bayesian Lasso, and two machine-learning methods for analyzing dichotomous traits and showed that the differences between methods are small, particularly with a large number of quantitative trait loci. On the other hand, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Villanueva <em>et al.</em> (2011)</a> developed a version of the BayesB method for dichotomous traits and concluded that the threshold BayesB method improves GS accuracy when disease-resistant dichotomous phenotypes are encountered, compared with accuracies obtained with the linear model. The threshold model showed an increase in accuracy of up to 16%, as well as significant advantages when heritability and disease prevalence were low and individuals were genotyped but not measured (testing set). Both <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">González-Recio and Forni (2011)</a> and <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Villanueva <em>et al.</em> (2011)</a> pointed out that the models they developed for dichotomous phenotypes can easily be extended to traits with more than two discrete categories.</p><p id="p-8">Recently, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wang <em>et al.</em> (2012)</a> proposed threshold models using the GS framework and extended three Bayesian methods (BayesA, BayesB, and BayesC<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-2.gif"/></noscript></span></span>) to estimate genomic breeding values of animal threshold traits with more than two categories; they named these methods extended BayesTA, extended BayesTB, and extended BayesTC<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-3.gif"/></noscript></span></span>. They derived computing procedures for the three BayesT methods by using the Gibbs sampler algorithm. Through a simulation study, they found that the three BayesT methods generally performed better than the corresponding standard Bayesian methods, especially when only two phenotypic categories are present. They also found that BayesTB and BayesTC<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-4.gif"/></noscript></span></span> produced similar accuracies and that both performed better than BayesTA. Also, they addressed how heritability, number of quantitative trait loci, incidence, and number of phenotypic categories affect the performance of the three BayesT methods. Recently, <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Kizilkaya <em>et al.</em> (2014)</a> proposed performing genomic prediction of ordinal categorical phenotypes using BayesTC<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-5.gif"/></noscript></span></span> and compared it with BayesC<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-6.gif"/></noscript></span></span>; they found that the accuracies of BayesTC<span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-7.gif"/></noscript></span></span> for ordinal categorical phenotypes were greater than those of BayesC<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-8.gif"/></noscript></span></span> and that there was a greater advantage in using BayesTC<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-9.gif"/></noscript></span></span> when the training population was small. These authors pointed out that a 2.25-fold increase in the training population size for ordinal categorical phenotypes analyzed as a linear model using BayesC<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-10.gif"/></noscript></span></span> was sufficient to achieve an accuracy equal to or greater than that for continuous phenotypes with a training population size of 1000 individuals.</p><p id="p-9">In plant breeding, the most economically important trait is grain yield, which is greatly affected by environmental factors, as well as by genotype × environment interaction (G×E). The GS models first introduced by <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Meuwissen <em>et al.</em> (2001)</a> were further extended into multienvironment models that can handle large numbers of individuals genotyped with large numbers of markers and evaluated in multiple environments. <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> (2012)</a> analyzed multienvironment data using a multivariate version of the genomic best linear unbiased predictor (GBLUP). <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jarquín <em>et al.</em> (2014)</a> extended GBLUP to incorporate highly dimensional markers, environmental covariates, and their interactions into a class of random-effects models where main and interaction effects are modeled using Gaussian processes with covariance functions based on genetic and environmental similarity among entries. In their study, <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Jarquín <em>et al.</em> (2014)</a> used as a covariance function the structure induced by a reaction norm model, and applied the proposed approach to the grain yield trait in wheat lines evaluated over multiple years and locations.</p><p id="p-10">However, in plant breeding, many economically important traits, such as the degree of resistance/susceptibility, are categorical. Despite this, to our knowledge, threshold models have not been considered in GS for plant breeding. Therefore, in this study, we introduced a threshold GS model that is an extension of the GBLUP of <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Jarquín <em>et al.</em> (2014)</a> which incorporates G×E and additive × additive × environment (G×G×E) interactions. The proposed model was evaluated using a data set consisting of 278 maize lines scored for resistance to gray leaf spot (GLS) measured using an ordered categorical scale [1 (no disease) to 5 (complete infection)] in three environments and genetic information consisting of 46,347 single-nucleotide polymorphisms (SNPs).</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Experimental data</h3><div id="sec-3" class="subsection"><h4>Phenotypic data:</h4><p id="p-11">The trait analyzed in 278 maize lines was gray leaf spot (GLS), caused by the fungus <em>Cercospora zeae-maydis</em>, which was evaluated in three environments, Mexico, Zimbabwe, and Colombia. GLS is one of the most important foliar diseases of maize worldwide. The GLS trait was analyzed using an ordinal scale from 1 (no disease), 2 (low infection), 3 (moderate infection), 4 (high infection), to 5 (complete infection). These data are part of the data set previously analyzed by <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Crossa <em>et al.</em> (2011)</a> and <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">González-Camacho <em>et al.</em> (2012)</a>, among others.</p></div><div id="sec-4" class="subsection"><h4>Genotypic data:</h4><p id="p-12">Genotypes of all 278 lines were obtained using 53,401 SNPs. Those SNPs with &gt;10% missing values or a minor allele frequency ≤0.05 were excluded from the data. Call rate per line was considered sufficient (&gt;90%) for all lines. Most lines (250 of 278) had a call rate greater than 99%. After line-specific quality control (applying the same quality control to each line separately), the maize data still contained 46,347 SNPs, which were used in the analysis.</p></div></div><div id="sec-5" class="subsection"><h3>Statistical models</h3><div id="sec-6" class="subsection"><h4>Threshold (cumulative probit) model:</h4><p id="p-13">Let <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-11.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-12.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-13.gif"/></noscript></span></span> represents the environment, <em>j</em> denotes the genotype and <em>k</em> is the number of replicates of each genotype in each environment. The response variable (disease resistance), <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-14.gif"/></noscript></span></span>, represents an assignment into one of <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-15.gif"/></noscript></span></span> mutually exclusive and exhaustive categories (here <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-16.gif"/></noscript></span></span>=5, I = 3 and J = 278) that follow an order, since 1 indicates no infection, 2 low infection, 3 moderate infection, 4 high infection, and 5 complete infection. Therefore, in a GLMM framework, this model can be described by defining the distribution, the linear predictor and the link function.</p><div id="sec-7" class="subsection"><h5>Distribution:</h5><p id="p-14">There are two distributions, one for observations in the response variable: <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-17.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-18.gif"/></noscript></span></span> is the I×1 vector of fixed environmental effects and another distribution for the random effects, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-19.gif"/></noscript></span></span>, where <strong>G</strong> is the variance-covariance matrix of <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-20.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-21.gif"/></noscript></span></span> is the effect of line <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-22.gif"/></noscript></span></span>.</p></div><div id="sec-8" class="subsection"><h5>Linear predictor:</h5><p id="p-15"><span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-23.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-24.gif"/></noscript></span></span> denotes the <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-25.gif"/></noscript></span></span><sup>th</sup> link (<span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-26.gif"/></noscript></span></span> for the fixed and random effects combination, <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-27.gif"/></noscript></span></span> is the intercept (threshold) for the <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-28.gif"/></noscript></span></span><sup>th</sup> link, and <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-29.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-30.gif"/></noscript></span></span> are known row incidence vectors corresponding to fixed and random effects in <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-31.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-32.gif"/></noscript></span></span>, respectively. Since there are <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-33.gif"/></noscript></span></span> categories, a total of <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-34.gif"/></noscript></span></span> link functions are required to fully specify the model.</p></div><div id="sec-9" class="subsection"><h5>Link function:</h5><p id="p-16">Cumulative probit <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-35.gif"/></noscript></span></span>.<span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-36.gif"/></noscript></span></span> is the cumulative distribution function of a standard normal distribution (probit link) and <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-37.gif"/></noscript></span></span> its corresponding inverse.</p><p id="p-17">The inverse link for this model is given as follows: <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-38.gif"/></noscript></span></span>. Once we have estimates of <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-39.gif"/></noscript></span></span> we estimate <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-40.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-41.gif"/></noscript></span></span>,…,<span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-42.gif"/></noscript></span></span>. This threshold model assumes that the process that gives rise to the observed categories is an underlying continuous variable with a normal distribution <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-43.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-44.gif"/></noscript></span></span> are called “liabilities,” <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-45.gif"/></noscript></span></span> (<em>e.g.</em>, <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Gianola 1982</a>, and <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Sorensen <em>et al.</em> 1995</a>). Furthermore, to generate ordinal categorical phenotypes with <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-46.gif"/></noscript></span></span> categories, the underlying phenotypic values are mapped to ordinal categorical phenotypes based on threshold parameters <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-47.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-48.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-49.gif"/></noscript></span></span> which are <em>cutpoints</em> of the continuous scale such that the observed ordinal categorical response (<span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-50.gif"/></noscript></span></span> is given by:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-51.gif"/></noscript></span></span>That is, <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-52.gif"/></noscript></span></span> falls into category <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-53.gif"/></noscript></span></span> when the latent variable falls into the <em>c</em><sup>th</sup> interval of values. Given <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-54.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-55.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-56.gif"/></noscript></span></span> are conditionally independent and distributed as</p><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-57.gif"/></noscript></span></span><p id="p-18">where <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-58.gif"/></noscript></span></span> is fixed to 1 to achieve identifiability in the likelihood because the liabilities are unobservable. Also, one of the thresholds was fixed (<span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-59.gif"/></noscript></span></span> to center the distribution. Therefore, <span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-60.gif"/></noscript></span></span>Note that <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-61.gif"/></noscript></span></span> is the predictor of the GLMM for the multinomial data given previously. Then, the conditional probability that <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-62.gif"/></noscript></span></span> falls into category <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-63.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-64.gif"/></noscript></span></span> given <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-65.gif"/></noscript></span></span>,<span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-66.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-67.gif"/></noscript></span></span> is given by<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-68.gif"/></noscript></span></span>The data are assumed conditionally independent, given <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-69.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-70.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-71.gif"/></noscript></span></span>. Therefore, the sampling model is<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-72.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-73.gif"/></noscript></span></span> is an indicator function taking a value of 1 if the response falls into category <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-74.gif"/></noscript></span></span> and 0 otherwise.</p></div></div><div id="sec-10" class="subsection"><h4>Models fitted:</h4><p id="p-19">Based on <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Jarquín <em>et al.</em> (2014)</a>, nine models are proposed for analyzing these data sets (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The sequence of models described below is similar to those presented by <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Jarquín <em>et al.</em> (2014)</a> for a continuous variable. However, for simplicity, only the underlying latent variables (<span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-75.gif"/></noscript></span></span> are presented together with the distribution of the corresponding random effects that give rise to the observed categorical phenotypes, given that a probit link function is assumed by all models described below.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465689/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465689/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Nine models used to fit the data set</span><div class="sb-div caption-clear"></div></div></div><div id="sec-11" class="subsection"><h5>Model 1:</h5><p id="p-21">The first model used to explain the liability value of the <em>k</em><sup>th</sup> individual in the <em>j</em><sup>th</sup> line at the <em>i</em><sup>th</sup> environment is<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-76.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-77.gif"/></noscript></span></span> is the fixed effect of the <em>i</em><sup>th</sup> environment and coefficient regressions were assigned a flat Gaussian prior with mean zero and variance equal to <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-78.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-79.gif"/></noscript></span></span> is the random effect of the <em>j</em><sup>th</sup> line which is assumed to be identically and independently distributed (<em>IID</em>) as normal, <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-80.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-81.gif"/></noscript></span></span> is an error term distributed as <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-82.gif"/></noscript></span></span>. The unknown variance parameters (<span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-83.gif"/></noscript></span></span> were assigned scaled-inverted χ<sup>2</sup> distributions as prior. This density is indexed by two hyper-parameters: degrees of freedom (<em>df</em>) and the scale (<em>S</em>) parameter. In this case, we defined these values using the internal rules provided by BGLR. By default, that is, if the user does not specify these parameters, BGLR assigns mildly informative priors with <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-84.gif"/></noscript></span></span>=5 and a scale parameter that gives a prior mode, <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-85.gif"/></noscript></span></span>, that obeys a prior variance partition where 50% of the variance of the liability score corresponds to error terms, and the remaining 50% to the random effects included in the model. The internal rules implemented in BGLR are fully explained in <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Pérez-Rodríguez and de los Campos (2014)</a>.</p></div><div id="sec-12" class="subsection"><h5>Model 2:</h5><p id="p-22">The second model is an extension of the GBLUP for ordered categorical phenotypes. This model was obtained from model 1 by replacing the line effect, <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-86.gif"/></noscript></span></span>, with a random effect that incorporates marker information using the genomic relationship matrix <strong><em>G</em></strong>. Model 2 expresses the liability value of the <em>k</em><sup>th</sup> individual in the <em>j</em><sup>th</sup> line in the <em>i</em><sup>th</sup> environment as<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-87.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>with <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-88.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-89.gif"/></noscript></span></span> represents an approximation to the true genetic value of the <em>j</em><sup>th</sup> line, <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-90.gif"/></noscript></span></span> is the genotype of the <em>j</em><sup>th</sup> line at the <em>m</em><sup>th</sup> marker (scored as 0 or 2 for genotypes that are homozygous at minor and major allele frequency, respectively, and 1 for heterozygous genotypes), and <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-91.gif"/></noscript></span></span> is the effect of the <em>m</em><sup>th</sup> marker with <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-92.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-93.gif"/></noscript></span></span>. Therefore, <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-94.gif"/></noscript></span></span> contains the genomic values of all lines, and <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-95.gif"/></noscript></span></span> is assumed to follow the normal distribution <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-96.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-97.gif"/></noscript></span></span> is a marker-derived genomic relationship matrix that has an expected value equal to (under ideal conditions) twice the coefficient of parentage matrix of lines, and <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-98.gif"/></noscript></span></span> is a genomic variance.</p><p id="p-23">The entries of <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-99.gif"/></noscript></span></span> were computed as <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-100.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-101.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-102.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-102.gif"/></noscript></span></span> is the estimated frequency of the <em>m</em><sup>th</sup> marker. Subtracting <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-103.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-103.gif"/></noscript></span></span> from the genotype codes (centering) and dividing each marker covariate by <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-104.gif"/></noscript></span></span> (standardizing) is not strictly needed; however, standardization allows interpreting <span class="inline-formula" id="inline-formula-98"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-105.gif"/></noscript></span></span> as a genomic variance. <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-106.gif"/></noscript></span></span> was constructed using the genotypes of all 278 lines.</p></div><div id="sec-13" class="subsection"><h5>Model 3:</h5><p id="p-24">The third model is an extension of model 2 that accounts for “marked” epistatic additive × additive relationships, <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-107.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-108.gif"/></noscript></span></span> represents a regression on genomic epistatic additive × additive relationships with <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-109.gif"/></noscript></span></span> (# is the element-wise multiplication operator, that is, a Hadamard product). The prior used for <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-110.gif"/></noscript></span></span> was a scaled-inverted χ<sup>2</sup> distribution. In this model, the liability score of the <em>k</em><sup>th</sup> individual in the <em>j</em><sup>th</sup> line in the <em>i</em><sup>th</sup> environment is equal to</p><span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-111.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-111.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span></div><div id="sec-14" class="subsection"><h5>Model 4:</h5><p id="p-25">The third model combines model 1 and model 2 as <span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-112.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-112.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span>Model 4 partitions the line effects into two components, one that is explained by a regression on markers (<span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-113.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-113.gif"/></noscript></span></span>) and another representing variation among lines that is not explained by regression on markers.</p></div><div id="sec-15" class="subsection"><h5>Model 5:</h5><p id="p-26">This model is an extension of model 4 that accounts for epistatic additive × additive relationships, with liability equal to<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-114.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-114.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>Note that none of the aforementioned models accounts for G×E (<span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-115.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-115.gif"/></noscript></span></span> and G×G×E (<span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-116.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-116.gif"/></noscript></span></span> interactions. Next, we consider models that incorporate G×E and G×G×E.</p></div><div id="sec-16" class="subsection"><h5>Model 6:</h5><p id="p-27">Model 6 considers model 2 but, in addition to the main effects of environments (E) and markers (G), it takes into account the interaction between markers and environments (G×E). Because the data include multiple phenotypic records per line, the genetic covariance structure of genetic effects in the full data set is equal to <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-117.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-117.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-118.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-118.gif"/></noscript></span></span> is the incidence matrix for the vector of additive genetic effects of markers (<a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Jarquín <em>et al.</em> 2014</a>). Therefore, the covariance structure for the vector of interaction terms in the full data set <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-119.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-119.gif"/></noscript></span></span> is the Hadamard product of <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-120.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-120.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-121.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-121.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-122.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-122.gif"/></noscript></span></span> represents the incidence matrix of the effects of environments (<em>i.e.</em>, the matrix that connects phenotypes with environments) (<a id="xref-ref-20-7" class="xref-bibr" href="#ref-20">Jarquín <em>et al.</em> 2014</a>). Therefore, the liability value of the <em>k</em><sup>th</sup> individual in the <em>j</em><sup>th</sup> line in the <em>i</em><sup>th</sup> environment is explained by <span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-123.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-123.gif"/></noscript></span> <span class="disp-formula-label">(7)</span></span>where <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-124.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-124.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-125.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-125.gif"/></noscript></span></span>.</p></div><div id="sec-17" class="subsection"><h5>Model 7:</h5><p id="p-28">Model 7 extends model 6 to account for marked epistatic additive × additive relationships and the interaction between the epistatic additive × additive term and the environments. The liability value of the <em>k</em><sup>th</sup> individual in the <em>j</em><sup>th</sup> line in the <em>i</em><sup>th</sup> environment is now <span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-126.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-126.gif"/></noscript></span><span class="disp-formula-label">(8)</span></span>where <span class="inline-formula" id="inline-formula-115"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-127.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-127.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-128.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-128.gif"/></noscript></span></span>.</p></div><div id="sec-18" class="subsection"><h5>Model 8:</h5><p id="p-29">Model 8 extends model 6 by adding the random effect of the lines (<span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-129.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-129.gif"/></noscript></span></span>). Thus, the liability value of the <em>k</em><sup>th</sup> individual in the <em>j</em><sup>th</sup> line in the <em>i</em><sup>th</sup> environment is explained by</p><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-130.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-130.gif"/></noscript></span> <span class="disp-formula-label">(9)</span></span></div><div id="sec-19" class="subsection"><h5>Model 9:</h5><p id="p-30">Finally, model 9 extends model 8 by adding marked epistatic additive × additive relationships and the interaction between the epistatic additive × additive term and the environments. The liability value of the <em>k</em><sup>th</sup> individual in the <em>j</em><sup>th</sup> line in the <em>i</em><sup>th</sup> environment is explained by </p><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-131.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-131.gif"/></noscript></span> <span class="disp-formula-label">(10)</span></span></div></div><div id="sec-20" class="subsection"><h4>Implementation with BGLR:</h4><p id="p-31">The nine models were fitted using the R-package BGLR (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">de los Campos and Pérez-Rodríguez 2013</a>) in the R-software (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">R Core Team 2014</a>). Parametric and semi-parametric models with both molecular marker and pedigree data can be fitted in this R package. It also allows including various random effects with user-defined covariance matrices. In <a id="xref-app-1-1" class="xref-app" href="#app-1">Appendix 1</a>, we provide the R code used for fitting the most parameterized model (model 9). The implementation of the other models is similar to that of model 9 but with fewer random effects. All these models are implemented via a Bayesian approach using the Gibbs sampler algorithm, and sampling from the fully conditional distributions, as shown in <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Sorensen <em>et al.</em> (1995)</a>. For more details about Gibbs sampler implementation used in the R-package BGLR (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">de los Campos and Pérez-Rodríguez 2013</a>) for binary and ordered categorical phenotypes, refer to <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Sorensen <em>et al.</em> (1995)</a>. The prior distributions of the variance components of all the models are described in the appendix of the BGLR package (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">de los Campos and Pérez-Rodríguez 2013</a>).</p></div><div id="sec-21" class="subsection"><h4>Assessing predictive ability for ordered categorical phenotypic data:</h4><p id="p-32">The performance of the models was evaluated employing a cross-validation approach to estimate their prediction accuracies. To that end, we split each of the data sets into two parts (training and validation sets), where the training set was used to fit the model and the validation set was used to evaluate the training model’s prediction ability. A total of 20 random partitions were performed, and the validation results were averaged over the 20 partitions with observations assigned to training and testing completely at random. This represents a prediction problem similar to that labeled as CV2 in <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> (2012)</a>, where the performance of some lines was observed in some environments (training set) but not in others (testing set).</p><p id="p-33">Measuring prediction accuracy for categorical traits is more challenging than for quantitative traits, where the Euclidean metric appears as a natural choice. A variety of scoring rules have been proposed to assess accuracy for categorical traits. A scoring rule provides a summary measure for evaluating probabilistic prediction by assigning a numerical score based on the predictive distribution and on the event or value that materializes (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Garthwaite <em>et al.</em> 2005</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kneib <em>et al.</em> 2007</a>). This goal is achieved in a reasonable way by taking into account not only point prediction but the whole predictive distribution.</p><p id="p-34">In the case of a multinomial model, this predictive distribution is simply obtained by computing the probabilities for all <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-132.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-132.gif"/></noscript></span></span> categories of the response according to the estimated model, <em>i.e.</em>, we obtain the predictive distribution <span class="inline-formula" id="inline-formula-119"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-133.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-133.gif"/></noscript></span></span>), derived from the estimated model for observation <em>i</em>, and <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-134.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-134.gif"/></noscript></span></span> is the realized value for this observation in the data set. A scoring rule is any real-value function <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-135.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-135.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-136.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-136.gif"/></noscript></span></span>, that assigns a value to the event that category <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-137.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-137.gif"/></noscript></span></span> is observed when <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-138.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-138.gif"/></noscript></span></span> is the predictive probability for individual <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-139.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-139.gif"/></noscript></span></span> in category <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-140.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-140.gif"/></noscript></span></span>. A suitable score is the sum<span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-141.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-141.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-142.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-142.gif"/></noscript></span></span> is the sum over all observations in the test data set. The hit rate (<em>i.e.</em>, the percentage of true positive predictions) and the log-likelihood are two popular scoring rules. However, both have the drawback that they involve only one of the probabilities of the predictive distribution (<span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-143.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-143.gif"/></noscript></span></span>). In addition, it has been well documented that the log-likelihood score is sensitive to extreme observations. In our applications, we utilize the Brier score (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Brier 1950</a>), which is equal to<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-144.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-144.gif"/></noscript></span> <span class="disp-formula-label">(11)</span></span>where <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-145.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-145.gif"/></noscript></span></span> takes the value of 1 if the ordinal categorical response observed for individual <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-146.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-146.gif"/></noscript></span></span> falls into category <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-147.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-147.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-148.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-148.gif"/></noscript></span></span>. This scoring rule uses all the information contained in the predictive distribution, not just a small part such as the hit rate or the log-likelihood score. Therefore, it is a reasonable choice for comparing categorical regression models, even though there are other scoring rules that also have good properties. The range of <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-149.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-149.gif"/></noscript></span></span> in <a id="xref-disp-formula-16-1" class="xref-disp-formula" href="#disp-formula-16">equation (11)</a> is between 0 and 2. For this reason, we took <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-150.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-150.gif"/></noscript></span></span> to get the Brier score bound between 0 and 1, and lower scores imply better predictions.</p></div><div id="sec-22" class="subsection"><h4>Data and software:</h4><p id="p-35">The phenotypic data for GLS in three environments (Mexico, Zimbabwe, and Colombia) for the 278 maize lines, the 46,347 SNPs data and the R scripts developed to fit the predictive models used in this study are given in the GLScode.rar file deposited at <a href="http://repository.cimmyt.org/xmlui/handle/10883/4128">http://repository.cimmyt.org/xmlui/handle/10883/4128</a>. The BGLR package (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">de los Campos and Pérez-Rodríguez 2013</a>) can be downloaded from CRAN.</p></div></div></div><div class="section results" id="sec-23"><h2 class="">Results</h2><p id="p-36"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows the relative frequencies of each category for the whole data set and for each country. The whole data set contains 2798 observations; category 3 has the most observations (923), and category 1 is the one with the fewest (234). This pattern was also observed in Zimbabwe (1485 total observations; 37 in category 1 and 581 in category 3) but somewhat more pronounced. Colombia had 832 observations, with 215 in category 3, 208 in category 2, and only 63 in category 5. Mexico had 481 observations, most of them in category 2 (212), and category 1 had the fewest data (26 observations).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/2/291/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative frequency of each category in the whole data set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1516877692" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relative frequency of each category in the whole data set.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/2/291/F1.medium.gif" width="440" height="326"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/5/2/291/F1.medium.gif" width="440" height="326"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/2/291/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/2/291/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465617" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-37" class="first-child">Relative frequency of each category in the whole data set.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">When applying eigenvalue decomposition to the genomic relationship matrix, 82 eigenvectors (components) of a total of 278 were needed to explain 80% of the total variance of genotypes. The first eigenvector captured 13.68% of the total variation of marker genotypes, whereas the second eigenvector explained 6.28% of the total variation. For these reasons, we can say that there is some, but not strong, evidence of population (and family) stratification, indicating a relatively diverse set of lines. This was expected because these lines came from different breeding programs.</p><p id="p-39"><a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> gives the estimates of the fixed (environment) effects and threshold parameters for each of the nine models; the first five models (1−5) had similar parameter estimates (fixed and threshold) that were different from those of models 6−9. Models 6−9 produced similar fixed and threshold parameter estimates. In plant and animal breeding, the focus is on estimating response probabilities associated with specific linear parameter combinations (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gianola and Foulley 1983</a>). <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> gives the probabilities for each ordinal categorical phenotype for model 9, for the whole data set, and for each country. In <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>, the average probabilities for category 5 (complete infection) were slightly greater than 0.20 (20%) in the whole data set and in each country. Colombia shows the greatest probability of complete infection (category 5), but it was only slightly greater than the probabilities in Mexico and Zimbabwe.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465594/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465594/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Mean and SD of posterior distributions of fixed (environment) effects <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/T2/embed/mml-math-151.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/T2/embed/mml-math-151.gif"/></noscript></span></span> and threshold parameters of the nine proposed models</span><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/5/2/291/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Estimated probability of each category in the whole data set and of each location in model 9." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1516877692" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Estimated probability of each category in the whole data set and of each location in model 9.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/5/2/291/F2.medium.gif" width="440" height="284"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/5/2/291/F2.medium.gif" width="440" height="284"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/5/2/291/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/5/2/291/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/465634" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-41" class="first-child">Estimated probability of each category in the whole data set and of each location in model 9.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">It is important to point out that the average probabilities of high infection (category 4) were very similar to the average probabilities of complete infection (category 5) for the whole data set and for each country, and that no infection (category 1) was the category with the lowest average probabilities (around 15%). These probability estimates were very different from estimates obtained based on raw frequencies (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>) because they take into account the distribution of records across countries, the structure of lines, and the interaction between markers and environments (countries). The linear predictor used for doing this calculation was taken from model 9, which is given in <a id="xref-disp-formula-14-1" class="xref-disp-formula" href="#disp-formula-14">Equation (10)</a>.</p><p id="p-43">Estimates of variance components derived from the full data analysis are given in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. The interaction (G×E) explained the largest proportion of the variance of liabilities, with estimated posterior means greater than 1.03 (models 6−9). The total variance explained by model 1 was 1.20, which is 1.7659 times smaller than the total variance captured by the most parameterized model (model 9), with a total variance of 2.1191. Also, model 2 (with E and G as main effects) only captured a total variance equal to 1.1911, which was 1.7791 times smaller than the total variance captured by model 9. The variance explained by models 3, 4, and 5 were, respectively, 1.7755, 1.7767, and 1.782 times smaller than the variance captured by model 9. However, models 6, 7, and 8 captured a total variance equal to 2.1321, 2.0967, and 2.116, respectively, which were almost identical to the total variance captured by model 9; model 6 captured the highest variability. It should be noted that in models 6, 7, 8, and 9, the percentages of variance explained by the interaction term (G×E) were 51.68, 49.42, 51.23, and 50.19%, respectively, clearly indicating that the interaction term was the most important source of variability in these models. It should be noted that if interactions were omitted, a large proportion of the variability in models 6 and 8 was not captured by main effects and the variance of the error term <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-159.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-159.gif"/></noscript></span></span> was the greatest source of variability. Another important finding was that the additive × additive epistatic terms explained 1.72, 1.06, 0.58, and 0.46% of the total variability in models 3, 5, 7, and 9, respectively, which means that adding this type of epistasis is not very important for this trait in these populations.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465663/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465663/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Estimated variance components of the nine proposed models</span><div class="sb-div caption-clear"></div></div></div><p id="p-45"><a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> presents the Brier scores of the validation samples for each country and for the nine models. Because phenotype was ordinal categorical, we used Brier scores instead of Pearson’s correlation coefficient for assessing prediction accuracy, because the Brier scores are bounded between 0 and 1, and values closer to zero imply better prediction accuracy. In <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>, models 6, 7, 8, and 9 presented the lowest Brier scores, so these four models had better prediction ability. Models 1−5 showed the worst prediction ability. In Colombia, the prediction ability of model 9 was 19.85% greater than that of model 1. In Mexico, the increase in the prediction ability of model 9 over model 1 (the simplest model) was 11.26%, while in Zimbabwe this increase was 8.71%. Although <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> showed that inclusion of the interaction term (G×E) explained the largest proportion of variability in models 6 and 8, this did not produce a large increase in the prediction ability of these two models over that of the models without interactions (models 1, 2, and 4). This may be due to three reasons: (1) predictive power differs from goodness of fit, that is, a model may fit a particular data set well even though the predictive power the model provides is small; (2) even though the added interaction term explained the largest proportion of variability in models 6 and 8, the total variability explained in all models was not large; and (3) the set of lines was relatively diverse. However, including the interaction terms G×E plus the additive × additive epistatic terms [main (<span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-161.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-161.gif"/></noscript></span></span> and interaction term (<span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-162.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-162.gif"/></noscript></span></span> did not improve predictions by much, and the inclusion of these terms did not explain a large percentage of the total variability.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/465692/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/465692/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Brier scores (mean, minimum and maximum; smaller indicates better prediction) evaluated for the validation samples</span><div class="sb-div caption-clear"></div></div></div></div><div class="section discussion" id="sec-24"><h2 class="">Discussion</h2><p id="p-46">GS offers important opportunities to improve genetic gains in plant and animal breeding programs; however, more powerful estimation and prediction methods are required to fully exploit the advantages of GS. Most GS methods assume a Gaussian response; however, many important traits in plant breeding, such as disease resistance, percent protein content, and proportion of seed or plant damage, are not normally distributed and need special treatment. In this study, we extended the GBLUP method to ordinal categorical responses. To this end, we used the implementation available in BGLR with a probit link that can accommodate both binary and ordinal traits (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Pérez-Rodríguez and de los Campos 2014</a>). To our knowledge, this is the first study in GS that uses an ordinal threshold model to analyze a categorical response in plant breeding.</p><p id="p-47">Most non-normal response variables are not linear with respect to model parameters and, unlike normal responses, the variance of the phenotypes is dependent on the mean. The GLMM uses: (1) a link function (typically nonlinear on the parameters) to specify the relationship between the mean of the response variable and a linear model; and (2) a variance function to describe the relationship between the mean and the variance of the distribution of the response variable, which enables it to relate traditional linear regression to non-normal data. This gives the GLMM framework a stronger basis for hypothesis testing and for more precise estimates of fixed and random effects. This statement applies to response variables coming from many different distributions, and not only to ordinal categorical responses.</p><p id="p-48">GS methods have been implemented for threshold (ordinal categorical) traits in animal breeding. For example, <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">González-Recio and Forni (2011)</a> developed versions of BayesA, Bayesian Lasso, and two machine learning methods for dichotomous traits; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Villanueva <em>et al.</em> (2011)</a> also developed a version of BayesB for dichotomous traits; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Wang <em>et al.</em> (2012)</a> implemented methods BayesA, BayesB, and BayesC<span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-163.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-163.gif"/></noscript></span></span> for ordinal categorical traits; and <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Kizilkaya <em>et al.</em> (2014)</a> implemented BayesC<span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-164.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/5/2/291/embed/mml-math-164.gif"/></noscript></span></span> for ordinal categorical traits. All these studies concluded that the traditional GS linear models are not suitable for threshold traits, since the basic assumptions of the linear model are violated, which produces a considerable reduction in prediction accuracy. However, to our knowledge, no plant breeding study so far has addressed the analysis of categorical traits. Our study fills this gap by providing a full description of a multithreshold GBLUP model (TGBLUP) and an empirical evaluation based on real data.</p><p id="p-49">Prediction accuracy in GS usually is assessed using the sample correlation between predictions and phenotypes. However, this metric is not appropriate for categorical outcomes. For this reason, we used the Brier score for assessing the prediction ability of the proposed threshold model, which assigns a numerical score based on the predictive distribution. This scoring rule is “strictly proper” in the sense that it maximizes the expected score of an observation (and is unique) and allows making fair comparisons among models. To our knowledge, our study is the first one in GS to use this score for assessing prediction accuracy.</p><p id="p-50">We presented nine specifications of the TGBLUP model which differ on the type of effects included. We considered main effects models and models for interactions between genetic factors (<em>e.g.</em>, additive ×additive epistatic) and between genetic and environmental factors <em>e.g.</em>, additive ×additive× environment interaction. For specifying these interactions, we used the framework proposed by <a id="xref-ref-20-8" class="xref-bibr" href="#ref-20">Jarquín <em>et al.</em> (2014)</a> within a threshold model framework. We found that models that take into account interactions (models 6−9) explained almost two times more variability than those without interaction (models 1−5). However, the interaction that was clearly most important was the G×E term; this effect captured the largest proportion of the total variability explained by the models (51.68% in model 6 and 50.19% in model 9), and was the one that led to the largest gain in prediction accuracy. This result is in agreement with previous studies that have highlighted the importance of modeling G×E in GS in plant breeding (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-20-9" class="xref-bibr" href="#ref-20">Jarquin <em>et al.</em> 2014</a>).</p><p id="p-51">Inclusion of additive × additive epistatic terms did not explain much of the total variability, and did not help much to improve prediction ability. Also, we found that estimated threshold parameters could be clustered into two groups, one for those without interactions (models 1−5), and another for those with interaction terms (models 6−9). This was because including interactions increases the variance of the liability score and, therefore, changes in threshold values are needed to accommodate the observed probabilities of each of the categories.</p><p id="p-52">Finally, although the analyses presented here used Gaussian priors for marker effects, the multi-threshold model used in this study can be implemented with any of the priors commonly used in GS, including those that induce differential shrinkage of estimates of effects or a combination of variable selection and shrinkage. The current implementation of BGLR allows users to do this.</p><p id="p-53">We extended the GBLUP, a model commonly used in GS for analyses of normal traits, to situations where the response is ordinal (TGBLUP). We provided a detailed description of the model and introduced a metric, the Brier score, for assessing prediction accuracy of ordinal categorical outcomes. We presented an empirical evaluation using a (real) data set of 278 maize lines genotyped with 46,347 SNPs and evaluated for disease resistance using an ordinal categorical scoring system in three environments (Colombia, Zimbabwe, and Mexico). A total of nine models were used. In addition, we provide details of the R code used to implement these models using the BGLR package.</p><p id="p-54">Our results highlight the importance of including G×E (capturing at least 49.42% of the total variability); when this interaction was taken into account, this increased the total variability explained by these models and increased prediction accuracy between 8 and 19% relative to models based on main effects only. Considering additive × additive epistasis did not produce a sizable increase in prediction accuracy.</p></div><div class="section app" id="app-1"><h2 class="">Appendix 1</h2><div class="section" id="sec-25"><h3>FITTING MODEL 9 TO RANDOMLY SAMPLED TRAINING AND TESTING SETS</h3><p id="p-55">library(BGLR)</p><p id="p-56">library(matrixcalc)</p><p id="p-57"># X, data frame containing the elements described below;</p><p id="p-58"># - X$loc: (n×1), a factor giving the IDs for the environments (location);</p><p id="p-59"># - X$line: (n×1), a factor giving the IDs for the varieties;</p><p id="p-60"># - X$y: (n×1), a numeric vector for rating the 5 levels of the disease;</p><p id="p-61"># - M: a matrix containing the genetic information (dimensions equal to the number of lines by the</p><p id="p-62"># - number of SNPs [46,347]).</p><p id="p-63">### Note: To replicate the results presented in this article, the sampling of training-testing sets must be done #according to the methods described in the article (70% training and 30% testing); this example simply #illustrates how to fit model 9 for a randomly chosen testing data set. The code for the other model is similar.</p><p id="p-64">#Calculating the marker-derived genomic relationship matrix (GRM)</p><p id="p-65">M&lt;-scale(M,center=TRUE,scale=TRUE)</p><p id="p-66">G&lt;-tcrossprod(M)/ncol(M)</p><p id="p-67"># Incidence matrix and covariance for main effects of environments.</p><p id="p-68">ZE&lt;-model.matrix(∼factor(X$loc)-1) </p><p id="p-69">KE=tcrossprod(ZE);</p><p id="p-70"># Incidence matrix and covariance for main effects of lines.</p><p id="p-71">X$lines&lt;-factor(x=X$lines, levels=rownames(M), ordered=TRUE)</p><p id="p-72">ZL&lt;-model.matrix(∼X$lines-1)</p><p id="p-73">KL=tcrossprod(ZL);</p><p id="p-74"># Genetic covariance structure of genetic effects in the full data</p><p id="p-75">KG= ZL%*%G%*%t(ZL);</p><p id="p-76"># Epistasis additive × additive covariance structure in the full data</p><p id="p-77">GA=hadamard.prod(G,G)</p><p id="p-78">KGG= ZL%*%GA%*%t(ZL);</p><p id="p-79"># G×E covariance structure for the full data set</p><p id="p-80">KGE=hadamard.prod(KG, KE)</p><p id="p-81">diag(KGE)=diag(KGE)</p><p id="p-82">KGE=KGE/mean(diag(KGE))</p><p id="p-83"># GG×E Epistasis additive × additive structure for the full data set</p><p id="p-84">KGGE=hadamard.prod(KGG, KE)</p><p id="p-85">diag(KGGE)=diag(KGGE)</p><p id="p-86">KGGE=KGGE/mean(diag(KGGE))</p><p id="p-87">yNA&lt;- X$y; n=length(X$y); p=round(0.30*n);</p><p id="p-88">tst&lt;-sample(1:nrow(X), size=p, replace=FALSE)</p><p id="p-89">yNA[tst]&lt;-NA</p><p id="p-90">ETA&lt;-list(ENV=list(K=ZE, model=’FIXED’), LINE=list(K=KL, model=’RKHS’),</p><p id="p-91">G=list(K=KG, model=’RKHS’), GG=list(K=KGG, model=’RKHS’), </p><p id="p-92">GE=list(K=KGE, model=’RKHS’), GGE=list(K=KGGE, model=’RKHS’))</p><p id="p-93">fm9&lt;-BGLR(y=yNA,response_type=’ordinal’, saveAt=’M9_’, nIter=52000, burnIn=6000)</p></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-94"><em>Communicating editor: D. J. de Koning</em> </p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-10-31"><span class="received-label">Received </span>October 31, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-12-19"><span class="accepted-label">Accepted </span>December 19, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2015 Montesinos-López <em>et al</em>.</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.1" data-doi="10.3758/BF03337683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atkinson</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">The measurement-statistics controversy: factor analysis and subinterval data.</span> <abbr class="cit-jnl-abbrev">Bull. Psychon. Soc.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBull.%2BPsychon.%2BSoc.%26rft.volume%253D26%26rft.spage%253D361%26rft_id%253Dinfo%253Adoi%252F10.3758%252FBF03337683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3758/BF03337683&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.2" data-doi="10.1175/1520-0493(1950)078&lt;0001:VOFEIT&gt;2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brier</span>  <span class="cit-name-given-names">G. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1950</span> <span class="cit-article-title">Verification of forecasts expressed in terms of probability.</span> <abbr class="cit-jnl-abbrev">Mon. Weather Rev.</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMon.%2BWeather%2BRev.%26rft.volume%253D78%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1175%252F1520-0493%25281950%2529078%253C0001%253AVOFEIT%253E2.0.CO%253B2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1175/1520-0493(1950)078&lt;0001:VOFEIT&gt;2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.3" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype×environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.4" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8yLzI5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.5" data-doi="10.1080/15427528.2011.558767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahuku</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection and prediction in plant breeding.</span> <abbr class="cit-jnl-abbrev">J. Crop Improv.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCrop%2BImprov.%26rft.volume%253D25%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15427528.2011.558767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15427528.2011.558767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-other" id="cit-5.2.291.6" data-doi="10.1534/g3.113.008227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beyene</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kassa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic prediction in maize breeding populations with genotyping-by-sequencing.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">3</span>: <span class="cit-fpage">1903</span>–<span class="cit-lpage">1926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.008227%26rft_id%253Dinfo%253Apmid%252F24022750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTEvMTkwMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzUvMi8yOTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomic prediction in CIMMYT maize and wheat breeding programs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">60</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D48%26rft_id%253Dinfo%253Apmid%252F23572121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23572121&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F291.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-web" id="cit-5.2.291.8"><div class="cit-metadata unstructured">de los Campos, G., and P. Pérez-Rodríguez, 2013 BGLR: Bayesian Generalized Linear Regression. R package version 1.0.3. Availabe at: <a href="http://CRAN.R-project.org/package=BGLR">http://CRAN.R-project.org/package=BGLR</a>. Accessed December 30, 2014.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.9" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8yLzI5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.10" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F291.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.11" data-doi="10.1198/016214505000000105"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garthwaite</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kadane</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Hagan</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Statistical methods for eliciting probability distributions.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">680</span>–<span class="cit-lpage">701</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D100%26rft.spage%253D680%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214505000000105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214505000000105&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000233311300028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">A method of sire evaluation for dichotomies.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1266</span>–<span class="cit-lpage">1271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D51%26rft.issue%253D6%26rft.spage%253D1266%26rft.epage%253D1271%26rft.atitle%253DA%2BMethod%2Bof%2BSire%2BEvaluation%2Bfor%2BDichotomies%26rft_id%253Dinfo%253Apmid%252F7204270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=7204270&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F291.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.13" data-doi="10.2527/jas1982.5451079x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Theory and analysis of threshold characters.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">1079</span>–<span class="cit-lpage">1096</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAnimal%2BScience%26rft.stitle%253DJ%2BANIM%2BSCI%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D54%26rft.issue%253D5%26rft.spage%253D1079%26rft.epage%253D1096%26rft.atitle%253DTheory%2Band%2BAnalysis%2Bof%2BThreshold%2BCharacters%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas1982.5451079x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas1982.5451079x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1982NP93300025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.14" data-doi="10.1186/1297-9686-15-1-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulley</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Sire evaluation for ordered categorical data with a threshold model.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D15%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-15-1-1%26rft_id%253Dinfo%253Apmid%252F22879102%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-15-1-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22879102&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F291.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.15" data-doi="10.1007/s00122-012-1868-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-enabled prediction of genetic values using radial basis function neural networks.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">771</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGonzalez-Camacho%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D125%26rft.issue%253D4%26rft.spage%253D759%26rft.epage%253D771%26rft.atitle%253DGenome-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Bradial%2Bbasis%2Bfunction%2Bneural%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1868-9%26rft_id%253Dinfo%253Apmid%252F22566067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1868-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22566067&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F291.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.16" data-doi="10.1186/1297-9686-43-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Recio</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forni</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide prediction of discrete traits using Bayesian regressions and machine learning.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">7</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DGonzalez-Recio%26rft.auinit1%253DO.%26rft.volume%253D43%26rft.spage%253D7%26rft.epage%253D7%26rft.atitle%253DGenome-wide%2Bprediction%2Bof%2Bdiscrete%2Btraits%2Busing%2BBayesian%2Bregressions%2Band%2Bmachine%2Blearning.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-43-7%26rft_id%253Dinfo%253Apmid%252F21329522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-43-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21329522&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F291.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.17" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.18" data-doi="10.2135/cropsci2009.11.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant breeding with genomic selection: gain per unit time and cost.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1681</span>–<span class="cit-lpage">1690</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D1681%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.11.0662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.11.0662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000281060300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.19" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.-P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorrells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: a comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.20" data-doi="10.1007/s00122-013-2243-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jarquín</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lacaze</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheyron</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daucourt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2014</span> <span class="cit-article-title">A reaction norm model for genomic selection using high-dimensional genomic and environmental data.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">595</span>–<span class="cit-lpage">607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D595%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2243-1%26rft_id%253Dinfo%253Apmid%252F24337101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2243-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24337101&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F291.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-confproc" id="cit-5.2.291.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kizilkaya</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tait</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reecy</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Whole genome analysis of infectious bovine kerato conjunctivitis in Angus cattle using Bayesian threshold models.</span> <span class="cit-source">BMC Proc.</span> <span class="cit-vol">5</span>: <span class="cit-fpage">S22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BProc.%26rft.volume%253D5%26rft.spage%253DS22%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.22" data-doi="10.1007/s10182-007-0033-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kneib</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baumgartner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steiner</span>  <span class="cit-name-given-names">W. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Semiparametric multinomial logit models for analysing consumer choice behaviour.</span> <abbr class="cit-jnl-abbrev">AStA Adv. Stat. Anal.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">225</span>–<span class="cit-lpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAStA%2BAdv.%2BStat.%2BAnal.%26rft.volume%253D91%26rft.spage%253D225%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10182-007-0033-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10182-007-0033-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-5.2.291.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Littell</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stroup</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freund</span>  <span class="cit-name-given-names">R. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-source">SAS for Linear Models</span>. <span class="cit-edition">ed. 4</span>. <span class="cit-publ-name">SAS Institute</span>, <span class="cit-publ-loc">Cary, NC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic values using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy81LzIvMjkxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.25" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression &amp; prediction with the BGLR statistical package.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNS8yLzI5MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.26" data-doi="10.3835/plantgenome2010.04.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic-enabled prediction based on molecular markers and pedigree using the Bayesian Linear Regression Package in R.</span> <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D3%26rft.spage%253D106%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2010.04.0005%26rft_id%253Dinfo%253Apmid%252F21566722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2010.04.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21566722&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F291.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-other" id="cit-5.2.291.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Rodríguez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manes</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker,</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite> <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparison between linear and non-parametric models for genome-enabled prediction in wheat.</span> <span class="cit-source">G3 (Bethesda)</span> <span class="cit-vol">2</span>: <span class="cit-fpage">1595</span>–<span class="cit-lpage">1605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D2%26rft.spage%253D1595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-web" id="cit-5.2.291.28"><div class="cit-metadata unstructured">R Core Team, 2014 R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. Available at: <a href="http://www.R-project.org/">http://www.R-project.org/</a>. Accessed December 30, 2014.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.29" data-doi="10.1051/gse:19950303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korsgaard</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Bayesian inference in threshold models using Gibbs sampling.</span> <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">229</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D27%26rft.spage%253D229%26rft_id%253Dinfo%253Adoi%252F10.1051%252Fgse%253A19950303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1051/gse:19950303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1995RE69800003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-book" id="cit-5.2.291.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stroup</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source"><em>Generalized Linear Mixed Models: Modern Concepts</em>, <em>Methods and Applications</em></span>. <span class="cit-publ-name">CRC Press</span>, <span class="cit-publ-loc">Boca Raton, FL</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.31" data-doi="10.2527/jas.2010-3814"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Villanueva</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Cortes</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varona</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of genome-wide evaluation for disease resistance in aquaculture breeding programs.</span> <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">3433</span>–<span class="cit-lpage">3442</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2010-3814%26rft_id%253Dinfo%253Apmid%252F21742941%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2010-3814&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21742941&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F291.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-5.2.291.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">X. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">W. X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2012</span> <span class="cit-article-title">Bayesian methods for estimating GEBVs of threshold traits.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D110%26rft.spage%253D213%26rft_id%253Dinfo%253Apmid%252F23149458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23149458&amp;link_type=MED&amp;atom=%2Fggg%2F5%2F2%2F291.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/5/2/291.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/5/2/281" title="Genetic Dissection of Internode Length Above the Uppermost Ear in Four RIL Populations of Maize (Zea mays L.)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/5/2/301" title="Ectopic Expression Screen Identifies Genes Affecting Drosophila Mesoderm Development Including the HSPG Trol" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/5/2"><span>Volume 5 Issue 2, February 2015</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/5/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/5/2.cover-source.jpg?itok=xW4cmeO6" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 5 (2)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/5/2/291.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-E2NaBykOkmsCq8mDHPsruBola5aLWv2mUzDv-Dp9rIE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/5/2/291" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/5/2/291" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/5/2/291" class="active" data-icon-position="" data-hide-link-title="0">Threshold Models for Genome-Enabled Prediction of Ordinal Categorical Traits in Plant Breeding</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459779" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-lg-QGp4fT9Tthmb4eGbkIq6EbZyqJniAHidHWIYy198" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/2/291.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/5/2/291" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-N1e0AfjKIpEOLgQ7CnNbtgzBNNqDYh45CeYveRhASN4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/5/2/291" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/5/2/291.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459779" id="node459779--2" data-pisa="ggg;5/2/291" data-pisa-master="ggg;g3.114.016188" data-apath="/ggg/5/2/291.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Threshold Models for Genome-Enabled Prediction of Ordinal Categorical Traits in Plant Breeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="2">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="3">Gustavo de los Campos</span>, <span class="highwire-citation-author" data-delta="4">Kent Eskridge</span> and <span class="highwire-citation-author" data-delta="5">José Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">291-300;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016188 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459779--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, Avenida Universidad 333, Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), Guanajuato, Guanajuato, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Colegio de Postgraduados, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biostatistics, University of Alabama at Birmingham, 443, Birmingham, Alabama 35294</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kent Eskridge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Statistics, University of Nebraska, Lincoln, Nebraska</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kent%2BEskridge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskridge%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKent%2BEskridge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), México, D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/5/2/291" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/5/2/291&amp;title=Threshold%20Models%20for%20Genome-Enabled%20Prediction%20of%20Ordinal%20Categorical%20Traits%20in%20Plant%20Breeding" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/5/2/291&amp;title=Threshold%20Models%20for%20Genome-Enabled%20Prediction%20of%20Ordinal%20Categorical%20Traits%20in%20Plant%20Breeding" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/5/2/291&amp;title=Threshold%20Models%20for%20Genome-Enabled%20Prediction%20of%20Ordinal%20Categorical%20Traits%20in%20Plant%20Breeding" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/5/2/291&amp;text=Threshold%20Models%20for%20Genome-Enabled%20Prediction%20of%20Ordinal%20Categorical%20Traits%20in%20Plant%20Breeding" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/5/2/291&amp;title=Threshold%20Models%20for%20Genome-Enabled%20Prediction%20of%20Ordinal%20Categorical%20Traits%20in%20Plant%20Breeding" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/5/2/291&amp;t=Threshold%20Models%20for%20Genome-Enabled%20Prediction%20of%20Ordinal%20Categorical%20Traits%20in%20Plant%20Breeding" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/5/2/291&amp;title=Threshold%20Models%20for%20Genome-Enabled%20Prediction%20of%20Ordinal%20Categorical%20Traits%20in%20Plant%20Breeding" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/5/2/291&amp;title=Threshold%20Models%20for%20Genome-Enabled%20Prediction%20of%20Ordinal%20Categorical%20Traits%20in%20Plant%20Breeding" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459779/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459779" id="node459779--3" data-pisa="ggg;5/2/291" data-pisa-master="ggg;g3.114.016188" data-apath="/ggg/5/2/291.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Threshold Models for Genome-Enabled Prediction of Ordinal Categorical Traits in Plant Breeding</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Osval A. Montesinos-López</span>, <span class="highwire-citation-author" data-delta="1">Abelardo Montesinos-López</span>, <span class="highwire-citation-author" data-delta="2">Paulino Pérez-Rodríguez</span>, <span class="highwire-citation-author" data-delta="3">Gustavo de los Campos</span>, <span class="highwire-citation-author" data-delta="4">Kent Eskridge</span> and <span class="highwire-citation-author" data-delta="5">José Crossa</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">February 1, 2015 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 5 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">291-300;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.016188 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459779--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Osval A. Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Facultad de Telemática, Universidad de Colima, Avenida Universidad 333, Colima, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Osval%2BA.%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20OA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOsval%2BA.%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Abelardo Montesinos-López </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Departamento de Estadística, Centro de Investigación en Matemáticas (CIMAT), Guanajuato, Guanajuato, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Abelardo%2BMontesinos-L%C3%B3pez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Montesinos-L%C3%B3pez%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAbelardo%2BMontesinos-L%25C3%25B3pez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Paulino Pérez-Rodríguez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Colegio de Postgraduados, Edo. de México, México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez-Rodr%C3%ADguez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez-Rodr%C3%ADguez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez-Rodr%25C3%25ADguez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biostatistics, University of Alabama at Birmingham, 443, Birmingham, Alabama 35294</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Kent Eskridge </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Statistics, University of Nebraska, Lincoln, Nebraska</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kent%2BEskridge%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Eskridge%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKent%2BEskridge%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Biometrics and Statistics Unit, International Maize and Wheat Improvement Center (CIMMYT), México, D.F., México</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=291&amp;rft.epage=300&amp;rft.atitle=Threshold%20Models%20for%20Genome-Enabled%20Prediction%20of%20Ordinal%20Categorical%20Traits%20in%20Plant%20Breeding&amp;rft.volume=5&amp;rft.issue=2&amp;rft.date=2015-02-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Montesinos-L%C3%B3pez%2C+Osval+A.&amp;rft.au=Montesinos-L%C3%B3pez%2C+Abelardo&amp;rft.au=P%C3%A9rez-Rodr%C3%ADguez%2C+Paulino&amp;rft.au=de+los+Campos%2C+Gustavo&amp;rft.au=Eskridge%2C+Kent&amp;rft.au=Crossa%2C+Jos%C3%A9"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459779/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459779/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459779/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459779/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459779/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459779/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459779/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459779/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459779/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459779/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459779/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459779/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/5/2/291" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-24" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix 1</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/5/2/291.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/5/2/291.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/5/2/291.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
