<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/11/1903" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/11/1903.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/11/1903.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/11/1903.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/11/1903/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/11/1903.atom" />
<meta name="HW.pisa" content="ggg;3/11/1903" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Prediction in Maize Breeding Populations with Genotyping-by-Sequencing" />
<meta name="DC.Identifier" content="10.1534/g3.113.008227" />
<meta name="DC.Date" content="2013-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Crossa et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genotyping-by-sequencing (GBS) technologies have proven capacity for delivering large numbers of marker genotypes with potentially less ascertainment bias than standard single nucleotide polymorphism (SNP) arrays. Therefore, GBS has become an attractive alternative technology for genomic selection. However, the use of GBS data poses important challenges, and the accuracy of genomic prediction using GBS is currently undergoing investigation in several crops, including maize, wheat, and cassava. The main objective of this study was to evaluate various methods for incorporating GBS information and compare them with pedigree models for predicting genetic values of lines from two maize populations evaluated for different traits measured in different environments (experiments 1 and 2). Given that GBS data come with a large percentage of uncalled genotypes, we evaluated methods using nonimputed, imputed, and GBS-inferred haplotypes of different lengths (short or long). GBS and pedigree data were incorporated into statistical models using either the genomic best linear unbiased predictors (GBLUP) or the reproducing kernel Hilbert spaces (RKHS) regressions, and prediction accuracy was quantified using cross-validation methods. The following results were found: relative to pedigree or marker-only models, there were consistent gains in prediction accuracy by combining pedigree and GBS data; there was increased predictive ability when using imputed or nonimputed GBS data over inferred haplotype in experiment 1, or nonimputed GBS and information-based imputed short and long haplotypes, as compared to the other methods in experiment 2; the level of prediction accuracy achieved using GBS data in experiment 2 is comparable to those reported by previous authors who analyzed this data set using SNP arrays; and GBLUP and RKHS models with pedigree with nonimputed and imputed GBS data provided the best prediction correlations for the three traits in experiment 1, whereas for experiment 2 RKHS provided slightly better prediction than GBLUP for drought-stressed environments, and both models provided similar predictions in well-watered environments." />
<meta name="DC.Contributor" content="José Crossa" />
<meta name="DC.Contributor" content="Yoseph Beyene" />
<meta name="DC.Contributor" content="Semagn Kassa" />
<meta name="DC.Contributor" content="Paulino Pérez" />
<meta name="DC.Contributor" content="John M. Hickey" />
<meta name="DC.Contributor" content="Charles Chen" />
<meta name="DC.Contributor" content="Gustavo de los Campos" />
<meta name="DC.Contributor" content="Juan Burgueño" />
<meta name="DC.Contributor" content="Vanessa S. Windhausen" />
<meta name="DC.Contributor" content="Ed Buckler" />
<meta name="DC.Contributor" content="Jean-Luc Jannink" />
<meta name="DC.Contributor" content="Marco A. Lopez Cruz" />
<meta name="DC.Contributor" content="Raman Babu" />
<meta name="article:published_time" content="2013-11-01" />
<meta name="article:section" content="Genomic Selection" />
<meta name="citation_title" content="Genomic Prediction in Maize Breeding Populations with Genotyping-by-Sequencing" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genotyping-by-sequencing (GBS) technologies have proven capacity for delivering large numbers of marker genotypes with potentially less ascertainment bias than standard single nucleotide polymorphism (SNP) arrays. Therefore, GBS has become an attractive alternative technology for genomic selection. However, the use of GBS data poses important challenges, and the accuracy of genomic prediction using GBS is currently undergoing investigation in several crops, including maize, wheat, and cassava. The main objective of this study was to evaluate various methods for incorporating GBS information and compare them with pedigree models for predicting genetic values of lines from two maize populations evaluated for different traits measured in different environments (experiments 1 and 2). Given that GBS data come with a large percentage of uncalled genotypes, we evaluated methods using nonimputed, imputed, and GBS-inferred haplotypes of different lengths (short or long). GBS and pedigree data were incorporated into statistical models using either the genomic best linear unbiased predictors (GBLUP) or the reproducing kernel Hilbert spaces (RKHS) regressions, and prediction accuracy was quantified using cross-validation methods. The following results were found: relative to pedigree or marker-only models, there were consistent gains in prediction accuracy by combining pedigree and GBS data; there was increased predictive ability when using imputed or nonimputed GBS data over inferred haplotype in experiment 1, or nonimputed GBS and information-based imputed short and long haplotypes, as compared to the other methods in experiment 2; the level of prediction accuracy achieved using GBS data in experiment 2 is comparable to those reported by previous authors who analyzed this data set using SNP arrays; and GBLUP and RKHS models with pedigree with nonimputed and imputed GBS data provided the best prediction correlations for the three traits in experiment 1, whereas for experiment 2 RKHS provided slightly better prediction than GBLUP for drought-stressed environments, and both models provided similar predictions in well-watered environments.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/11/01" />
<meta name="citation_mjid" content="ggg;3/11/1903" />
<meta name="citation_id" content="3/11/1903" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/11/1903" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/11/1903.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/11/1903.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/11/1903.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.008227" />
<meta name="citation_pmid" content="24022750" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="24" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Selection" />
<meta name="citation_firstpage" content="1903" />
<meta name="citation_lastpage" content="1926" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="José Crossa" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico" />
<meta name="citation_author_email" content="j.crossa@cgiar.org" />
<meta name="citation_author" content="Yoseph Beyene" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico" />
<meta name="citation_author" content="Semagn Kassa" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico" />
<meta name="citation_author" content="Paulino Pérez" />
<meta name="citation_author_institution" content="Colegio de Postgraduados, Montecillos, Edo. de Mexico, 56230, Mexico" />
<meta name="citation_author" content="John M. Hickey" />
<meta name="citation_author_institution" content="The Roslin Institute, University of Edinburgh, Easter Bush, Midlothian, EH25 9RG, United Kingdom" />
<meta name="citation_author" content="Charles Chen" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico" />
<meta name="citation_author" content="Gustavo de los Campos" />
<meta name="citation_author_institution" content="Department of Biostatistics, School of Public Health, University of Alabama at Birmingham, Alabama 35294" />
<meta name="citation_author" content="Juan Burgueño" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico" />
<meta name="citation_author" content="Vanessa S. Windhausen" />
<meta name="citation_author_institution" content="Saaten Union Recherche, 163 Avenue de Flandre, 60190 Estrées Saint Denis, France" />
<meta name="citation_author" content="Ed Buckler" />
<meta name="citation_author_institution" content="USDA—ARS, Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York, New York 14850" />
<meta name="citation_author" content="Jean-Luc Jannink" />
<meta name="citation_author_institution" content="USDA—ARS, Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York, New York 14850" />
<meta name="citation_author" content="Marco A. Lopez Cruz" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico" />
<meta name="citation_author" content="Raman Babu" />
<meta name="citation_author_institution" content="International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. M. Beissinger;citation_author=C. N. Hirsch;citation_author=R. S. Sekhon;citation_author=J. M. Foerster;citation_author=J. M. Johnson;citation_title=Marker density and read depth for genotyping populations using genotyping-by-sequencing.;citation_volume=193;citation_year=2013;citation_pmid=23410831;citation_doi=10.1534/genetics.112.147710" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=F. V. Brito;citation_author=J. B. Neto;citation_author=M. Sargikzaei;citation_author=J. A. Cobuci;citation_author=F. S. Schenkel;citation_title=Accuracy of genomic selection in simulated populations mimicking the extent of linkage disequilibrium in beef cattle.;citation_pages=80-80;citation_volume=12;citation_year=2011;citation_pmid=21933416" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. Burgueño;citation_author=G. de los Campos;citation_author=K. A. Weigel;citation_author=J. Crossa;citation_title=Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.;citation_pages=707-719;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0299" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J.-M. Chia;citation_author=C. Song;citation_author=P. J. Bradbury;citation_author=D. Costich;citation_author=N. de Leon;citation_title=Maize HapMap2 identifies extant variation from a genome in flux.;citation_pages=803-807;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22660545;citation_doi=10.1038/ng.2313" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=G. de los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=J. Crop Improv.;citation_author=J. Crossa;citation_author=P. Pérez;citation_author=G. de los Campos;citation_author=G. Mahuku;citation_author=S. Dreisigacker;citation_title=Genomic selection and prediction in plant breeding.;citation_pages=239-261;citation_volume=25;citation_year=2011;citation_doi=10.1080/15427528.2011.558767" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_title=Genomic prediction in CIMMYT maize and wheat breeding programs.;citation_author=J. Crossa;citation_author=P. Pérez;citation_author=J. Hickey;citation_author=J. Burgueño;citation_author=L. Ornella;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigrees;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=4;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. Calus;citation_title=Whole genome regression and prediction methods applied to plant and animal breeding.;citation_pages=327-345;citation_volume=193;citation_year=2012;citation_pmid=22745228" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Fu;citation_author=H. K. Dooner;citation_title=Intraspecific violation of genetic colinearity and its implications in maize.;citation_volume=99;citation_year=2002;citation_pmid=12060715;citation_doi=10.1073/pnas.132259199" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=R. L. Fernando;citation_author=A. Stella;citation_title=Genomic assisted prediction of genetic value with semi-parametric procedures.;citation_volume=173;citation_year=2006;citation_pmid=16648593;citation_doi=10.1534/genetics.105.049510" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=D. Gianola;citation_author=H. Okut;citation_author=K. A. Weigel;citation_author=G. J. M. Rosa;citation_title=Predicting complex quantitative traits with Bayesian neural networks: a case study with Jersey cows and wheat.;citation_pages=87-87;citation_volume=12;citation_year=2011;citation_pmid=21981731;citation_doi=10.1186/1471-2156-12-87" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. M. González-Camacho;citation_author=G. de los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Cairns;citation_title=Genome-enabled prediction of genetic values using radial basis function neural networks.;citation_pages=759-771;citation_volume=125;citation_year=2012;citation_issue=4;citation_pmid=22566067;citation_doi=10.1007/s00122-012-1868-9" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=N. Heslot;citation_author=H.-S. Yang;citation_author=M. E. Sorells;citation_author=J.-L. Jannick;citation_title=Genomic selection in plant breeding: A comparison of models.;citation_pages=146-160;citation_volume=52;citation_year=2012;citation_doi=10.2135/cropsci2011.06.0297" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_title=Factors affecting the accuracy of genotype imputation in populations from several maize breeding programs.;citation_author=J. M. Hickey;citation_author=J. Crossa;citation_author=R. Babu;citation_author=G. de los Campos;citation_year=2012" />
<meta name="citation_reference" content="citation_title=BioJava: An open-source framework for bioinformatics;citation_author=T. Holland;citation_author=M. Down;citation_author=A. Pocock;citation_author=A. Prlić;citation_author=D. Huen;citation_year=2008" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=L. Janss;citation_author=G. de los Campos;citation_author=N. Sheehan;citation_author=D. Sorensen;citation_title=Inferences from genomic models in stratified populations.;citation_volume=192;citation_year=2012;citation_pmid=22813891;citation_doi=10.1534/genetics.112.141143" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform.;citation_volume=25;citation_year=2009;citation_issue=14;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and SAM tools.;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. Lu;citation_author=T. Shah;citation_author=Z. Hao;citation_author=S. Taba;citation_author=S. Zhang;citation_title=Comparative SNP and haplotype analysis reveals a higher genetic diversity and rapider LD decay in tropical than temperate germplasm in maize.;citation_pages=e24861-e24861;citation_volume=6;citation_year=2011;citation_issue=9;citation_pmid=21949770;citation_doi=10.1371/journal.pone.0024861" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome;citation_journal_abbrev=The Plant Genome;citation_author=P. Pérez;citation_author=G. de los Campos;citation_author=J. Crossa;citation_author=D. Gianola;citation_title=Genomic-Enabled Prediction Based on Molecular Markers and Pedigree Using the Bayesian Linear Regression Package in R.;citation_pages=106-116;citation_volume=3;citation_year=2010;citation_issue=2;citation_pmid=21566722;citation_doi=10.3835/plantgenome2010.04.0005" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. M. González-Camacho;citation_author=J. Crossa;citation_author=Y. Manès;citation_title=Comparison between linear and non-parametric regression models for genome-enabled prediction in wheat. G3: Genes, Genomes;citation_pages=1595-1605;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome;citation_author=J. Poland;citation_author=J. Endelman;citation_author=J. Dawson;citation_author=J. Rutkoski;citation_author=S. Wu;citation_title=Genomic selection in wheat breeding using genotyping-by-sequencing.;citation_pages=103-113;citation_volume=5;citation_year=2012;citation_doi=10.3835/plantgenome2012.06.0006" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. C. Romay;citation_author=M. Millard;citation_author=J. C. Glaubitz;citation_author=J. A. Peiffer;citation_author=K. L. Swarts;citation_title=Comprehensive genotyping of the USA national maize inbred seed bank.;citation_pages=R55;citation_volume=14;citation_year=2013;citation_pmid=23759205;citation_doi=10.1186/gb-2013-14-6-r55" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. M. Springer;citation_author=K. Ying;citation_author=Y. Fu;citation_author=T. Ji;citation_author=C.-T. Yeh;citation_title=Maize inbreds exhibit high levels of copy number variation (CNV) and presence/absence variation (PAV) in genome content.;citation_pages=e1000734-e1000734;citation_volume=5;citation_year=2009;citation_issue=11;citation_pmid=19956538;citation_doi=10.1371/journal.pgen.1000734" />
<meta name="citation_reference" content="citation_journal_title=Interbull Bull.;citation_author=P. VanRaden;citation_title=Genomic measures of relationship and inbreeding.;citation_pages=33-36;citation_volume=37;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. I. Vazquez;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_author=G. de los Campos;citation_author=D. Gianola;citation_title=Predictive ability of subsets of single nucleotide polymorphisms with and without parent average in US Holsteins.;citation_pages=5942-5949;citation_volume=93;citation_year=2010;citation_issue=12;citation_pmid=21094768;citation_doi=10.3168/jds.2010-3335" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Z. Wei;citation_author=W. Wang;citation_author=P. Hu;citation_author=G. J. Lyon;citation_author=H. Hakonarson;citation_title=SNVer: a statistical tool for variant calling in analysis of pooled or individual next-generation sequencing data.;citation_volume=39;citation_year=2011;citation_pmid=21813454;citation_doi=10.1093/nar/gkr599" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=V. S. Windhausen;citation_author=G. N. Atlin;citation_author=J. Crossa;citation_author=J. M. Hickey;citation_author=P. Grudloyma;citation_title=Effectiveness of Genomic Prediction of Maize Hybrid Performance in Different Breeding Populations and Environments;citation_pages=1427-1436;citation_volume=2;citation_year=2012;citation_issue=11;citation_pmid=23173094;citation_doi=10.1534/g3.112.003699" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yan;citation_author=C. B. Kandianis;citation_author=C. E. Harjes;citation_author=L. Bai;citation_author=E.-H. Kim;citation_title=Rare genetic variation at Zea mays crtRB1 increases beta-carotene in maize grain.;citation_pages=322-327;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20305664;citation_doi=10.1038/ng.551" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Zhong;citation_author=J.C.M. Dekkers;citation_author=R.L. Fernando;citation_author=J-L. Jannink;citation_title=Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: a barley case study.;citation_volume=182;citation_year=2009;citation_pmid=19299342;citation_doi=10.1534/genetics.108.098277" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Prediction in Maize Breeding Populations with Genotyping-by-Sequencing" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="twitter:description" content="Genotyping-by-sequencing (GBS) technologies have proven capacity for delivering large numbers of marker genotypes with potentially less ascertainment bias than standard single nucleotide polymorphism (SNP) arrays. Therefore, GBS has become an attractive alternative technology for genomic selection. However, the use of GBS data poses important challenges, and the accuracy of genomic prediction using GBS is currently undergoing investigation in several crops, including maize, wheat, and cassava. The main objective of this study was to evaluate various methods for incorporating GBS information and compare them with pedigree models for predicting genetic values of lines from two maize populations evaluated for different traits measured in different environments (experiments 1 and 2). Given that GBS data come with a large percentage of uncalled genotypes, we evaluated methods using nonimputed, imputed, and GBS-inferred haplotypes of different lengths (short or long). GBS and pedigree data were incorporated into statistical models using either the genomic best linear unbiased predictors (GBLUP) or the reproducing kernel Hilbert spaces (RKHS) regressions, and prediction accuracy was quantified using cross-validation methods. The following results were found: relative to pedigree or marker-only models, there were consistent gains in prediction accuracy by combining pedigree and GBS data; there was increased predictive ability when using imputed or nonimputed GBS data over inferred haplotype in experiment 1, or nonimputed GBS and information-based imputed short and long haplotypes, as compared to the other methods in experiment 2; the level of prediction accuracy achieved using GBS data in experiment 2 is comparable to those reported by previous authors who analyzed this data set using SNP arrays; and GBLUP and RKHS models with pedigree with nonimputed and imputed GBS data provided the best prediction correlations for the three traits in experiment 1, whereas for experiment 2 RKHS provided slightly better prediction than GBLUP for drought-stressed environments, and both models provided similar predictions in well-watered environments." />
<meta name="og-title" property="og:title" content="Genomic Prediction in Maize Breeding Populations with Genotyping-by-Sequencing" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/11/1903" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genotyping-by-sequencing (GBS) technologies have proven capacity for delivering large numbers of marker genotypes with potentially less ascertainment bias than standard single nucleotide polymorphism (SNP) arrays. Therefore, GBS has become an attractive alternative technology for genomic selection. However, the use of GBS data poses important challenges, and the accuracy of genomic prediction using GBS is currently undergoing investigation in several crops, including maize, wheat, and cassava. The main objective of this study was to evaluate various methods for incorporating GBS information and compare them with pedigree models for predicting genetic values of lines from two maize populations evaluated for different traits measured in different environments (experiments 1 and 2). Given that GBS data come with a large percentage of uncalled genotypes, we evaluated methods using nonimputed, imputed, and GBS-inferred haplotypes of different lengths (short or long). GBS and pedigree data were incorporated into statistical models using either the genomic best linear unbiased predictors (GBLUP) or the reproducing kernel Hilbert spaces (RKHS) regressions, and prediction accuracy was quantified using cross-validation methods. The following results were found: relative to pedigree or marker-only models, there were consistent gains in prediction accuracy by combining pedigree and GBS data; there was increased predictive ability when using imputed or nonimputed GBS data over inferred haplotype in experiment 1, or nonimputed GBS and information-based imputed short and long haplotypes, as compared to the other methods in experiment 2; the level of prediction accuracy achieved using GBS data in experiment 2 is comparable to those reported by previous authors who analyzed this data set using SNP arrays; and GBLUP and RKHS models with pedigree with nonimputed and imputed GBS data provided the best prediction correlations for the three traits in experiment 1, whereas for experiment 2 RKHS provided slightly better prediction than GBLUP for drought-stressed environments, and both models provided similar predictions in well-watered environments." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
    <title>Genomic Prediction in Maize Breeding Populations with Genotyping-by-Sequencing | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"2Jt4l-etEXwa69fq1FnkAZrgYWnGLqdtjzzNa5bxYM0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459578","apath":"\/ggg\/3\/11\/1903.atom","pisa":"ggg;3\/11\/1903","ac":{"\/ggg\/3\/11\/1903.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/11\/1903.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/11\/1903"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/11\/1903"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/11\/1903":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00223vsKUia5o8AccuxUWOl5Hd0Nn0Q5gLGvANDfNNUvx4T7IDqx5n6sHpU0yRCfYg3kp9vmuFpHT68cc896kuRUqIoADpdP49Dsc835+6fmR\\\\\\\/lJWq4ORT4g\\\\\\\/oe+0+Wkut8sxpUDd5bFym3ywZ\\\\\\\/1I3pPUmEiGPi3jHFVU\\\\\\\/LX8ZtivP\\\\\\\/8VF1eHr36n9GnWp4O+jXyBJELmoA7d2zldO1nTwD7xO47tZS07v3Mqj4Y+X2rdVfGvUIVww6xiEUUrZ+kBa+iCFTYhEMUqcK7DCrX2oYfA5J4ywgG3bFJJRmMH333Rl7624w7zhRmY\\\\\\\/S1Mu2enkRFMd4k7Y\\\\\\\/c3vNm\\\\\\\/rFteYHTlmQDEwkfc5LXAovJUBvrIk8sMMBvL+A7Kd3+QNnKoEjCZ8gqYF7q6g8iOSF3l5shVxSGDyqjFKE6sT12I2CGhUTxeog0ombbVAHdvf1Df6At37PAKyeFFaYvZB4qr1PKhMiZOINZUY0hOvT0vbxs2S0xBqOZudWbHGHClf\\\\\\\/eCnCpdZoqRd38QvEkp\\\\\\\/qQliHuPy2gkjhRBhXdImGL\\\\\\\/\\\\\\\/N1ArMNNzp8cj1tlNv4DD6\\\\\\\/ibDPio1FVfZfFLbKrcL3dgdGYVQ29DRXZDiWNugVhhCtikJd5htC+KzVyF8XKbOaHoLXZtHmFFm9lHCiFZHTCh2d1IjDp9tpxJ5nN1+mbgacFvr7PVZrjY6hGIltqWlOWidlZq06EvwSXXXujY\\\\\\\/5jgOewJu2rhgfPpd7Qc54cYV9sICq8q8fyy7jTyC2ekdOq\\\\\\\/jqtofhDkZGYQcNUKKSLpzqZPcAKsDol1Fcxn6+1tRsBOnm66owYzPo47s+CeegwaohBfTbfwW33MUnGjghh+vSuQfwQ33K3uAsKTgCsjU64vjgN2AnB4nEVGF8yQmtu+QojSA5ldWYhhA7MYHmOFCbvpD5A7UGMN0U4gRofotylBfYgQXfCuITdR3sgmGAJ3pfP2ARAfBdvL3MQpPC+IehAVE+LjSuc21O0rLg3w2CmUZvzzr5knEo3oYcRwJ8f+wn4M3mwK8MJKKw5n1WzRyrvNOiZX6HPXsp\\\\\\\/a7M04HolOuiwt\\\\\\\/fhCylWngIRxNMovLdPzbSqO3OZHEAoU\\\\\\\/5re6fDMT\\\\\\\/BzAqJdW0kvA\\\\\\\/yU2bhNXyhSj1eNjMrnpacNFsFBEzF2jt\\\\\\\/KEUw31WpxqTSo09RUW4O0lRz97BqUx0VZOD4QStBozlyb1juBcZyS5X9BUSI7TnSDfxoXXjs4eRtaoAeOvXsPjkkzu64GK8kAH0ikkIgtZbeU++bes5xzNC7gt7\\\u0022,\\\u0022iv\\\u0022:\\\u0022r62IkBtDammSOmb64b9O1g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229ac745e7a893c69f827e8185453c4c7ceb268c22eb49885dbb3c41b6498da280\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ciXI3ZjNsl8ONzZAUR+wahwslgovfp9oiIR70bJB2smZw3C4ANiRIVyJaGCx5IEcq1jadUAcMtJIkjuAdnnQWraBWwykgjPXvA4UAyWFLWPNifzAM42B6CMMolUba76yCoTuEni2cLXimgsSTtwludFVbO3YM9B1zZFx47zB77Y6kr7TO3rl4FNLz5G5BQdZtnVEUE3NDq8wRQC6M9rS+nzuKatOMtL8fJyk5L6tRvgoLNBLQrw3TYKXF7kLKiGL9yfLdPT++aK00bjtq0iNbMaVCiuGy3QIy3ms8JKDbnIzV0inMLgLSVEwxLWfDD\\\\\\\/B1cJl3jtqL6pvonBjh78Qe6eB6Lv86yEv\\\\\\\/qbElLFf\\\\\\\/wjsjEI3CyHir3Kl96ruseqZJgtyvL5\\\\\\\/nvplF4cZD+kDoMnhyemk0kBdEuk8RR1P1LGPUOZtza45CjW5rOLLmQQUbNQYTT+uZRP5vfB4q7KNkFUJamFVM91ccOr1DxDc2Zw7uEc37qqHxJaQjkxyUOcpf4ynVuFZ7ldmJQrLVkVRrgAySqCRIASLUEdWcGFeKx3fCCelxYCThCSA1XyaZZzMjUpoKq9IeRHK57oaVQBslh\\\\\\\/CPo+\\\\\\\/MfJx6skOHV8t0nO+FDYa7aPqcrlzBC+59alZHysuM4WyVCOAvHbVyykqaoTuqjaKSmT8lf\\\\\\\/dg6yxu6vB8ZU72aIavDynb399UHSWunrjSm6HdVY1mT1lFKXKhME3ZTTXVedFMhqBJXbyuwFvGMymgQTFBgfpUO6iCPJuiCSS8CEXqNutfiJ8c5KSvZK4INwKcPQlo6IPVHLg6l9kjboJPfo3l7W8BdgjBaX1mYKvT2Bank6owJaraxB6W3ovDliew1jN1p02eJSW8Qde1WZnyKLLvQUj0lMH4v52aqwMUhNAbSSlyVuAmH5uj6WhXz4\\\\\\\/pMeB4UMZK65MbmaJO0BG+UPNojcDMI6BARSCO81wjMe4zGAMpQm+lBSEFpfXj1hK44H9RwVw86dTf7ImC5kFOhPI\\\\\\\/a3WaOWsr\\\\\\\/QheEuipqGvao\\\\\\\/owI9EBtSTjAPaELNnTJHH0nnjQGyTKhS6cgNQrSvdC5UM2YMk9aJhUwikN3JkdDbG5+pk1FFsEpSGEu48R5XHkSWvKbLeKcnEhWFqHxqlZsx0iTrPBXAWhrZL5YZSDGfdspSE4gay5sAKpy9D8Uqsppm+mF3bqpNBJJupYpVqwdDxAfPpdUVFUKANE129Cubge\\\\\\\/9OOba\\\\\\\/o+cR0wg9l7z+qN6t8hw0Kv1xrzeun6PnaJhwdTEMErSoihz\\\\\\\/A7t5gQTI8D8lyyTMJ\\\\\\\/\\\\\\\/+Jpx4S08NtWKTXZWBEhcdcvbhk9dmA5yQBOFDDENgrv+vFl9OSdpoKaHUe1FvUEBwCnrAsj2j6UpInXQSUhQqIPEvoiJL\\\\\\\/t\\\\\\\/vyIVN567S1wmmn8VeHqPdAIStOiYj\\\\\\\/wH\\\\\\\/TNHT8wtllpkvPCdOgupJg2+\\\\\\\/6cgqskaXdXYB1LSNAdWERy4BzEEXjgCjYAVDI\\\\\\\/RyLqu70cNn3esfJXbSyX3p8PSD4v8+7XOm7m0kP3YARm1nHDEIbLvinQKqozbAw2jNYK4nEaXaa4luhtXqOBS1\\\\\\\/kaXF9gwJr4j+oNO+9GEowFIxNaCqxaJAc4gFgT88LI2ygaXn5OyzYpoO\\\\\\\/2U9Q9XwcDalX1AlDcVRa\\\\\\\/jRKb3F7oZU07ccMBwfgmV+EwT38AAlmD\\\\\\\/1rHrZ5TQb55XpjguVBDZltrvDEeufs\\\\\\\/1f+o0a6W90SwdjR0Kbk2gs1XNxN9hb4n34GZNfPAwCq3ql0YXA8pPuz+mnGxiu3TclmXqpcg1FIwJSinMVLOVgYE+2MfQV+kU\\\\\\\/c4vBqluK0\\\\\\\/0oUyJcjRpj9gB4Ya6tkxZfjWpsZo2VDsYOzKSDOp5Z62jJEcfd0Fc95FUC50OWM1Nc6Alc0PRUbGUxAush2qdIIa6WrpVxYHJ4Z9nDsTi0rVoO0BDn+\\\\\\\/VOOML6S5XKM1my1va8LFOLhNapHtZx78DuoxVbWp0aHL0Le\\\\\\\/CbVxw1iObYnlSRYSZ4cFfqsf8PVeiuy9PRWw4bGq8NQ0KCvuEVMCSd3LlizYy4nSiaLEmc7KhWcFTXRRcNICICsAvdulb\\\\\\\/xKt1uXqlZP6qDNBjpr8Wkwrg6nPskuZwfYx1EyL0XdHTY4sr6t8KGNow5TlFVq3OU4akjWJrc\\\\\\\/cfU0klPCw9UoAlNL3HwPidaWcmS4Qlg8uWLi6ANPHgsf10vgkXYFhxf4DhX2wRmR93khSKmezid0Pkk1icB5hWgRdMzwd6NjmU\\\\\\\/aPeJ0fmDKlXLIbuCmyjLy7MAQ0pvO2nWg6bH2VIOVCPLpOucLo5DnYERhJyrQSTFdWgKlmUjxzM3TzbpqAyjaOe8OkOdwZgtgat2FHtn84NFOW5nL9KL5B2uScYeLrkb1roOzfCtMLLD+Y0n7jQbepm+x8HpdtyKqXV\\\\\\\/5nsJDT1Zgc3wB9YAjccsb1bTvXivj5qUvbdqrGj+i\\\\\\\/zge6T0uPWjAYFH6o7AYfLwZHI5ntEZ2GanOxmLxqTNkmG8f3Pes5aAhuzExPsY6Mk2uhtCi5U0tAAEduNK2GBabGgzaECNByUGoHW99rgCG+6oWJZuxafpSX3JaTxNvyVstyp57GhQzfmz3mB1EieIK1Eo8XhjaF6lPH+uM9C33+XlpDwskuQ9tmFw3fl2jCCH988+V6GrrL8HfXlkRncnXYYeuyRV77FL9YXjlSYKRgwoaF1Gyq+Y\\\\\\\/8fCvNRW2Hct7iHErvln1yBx72ar61Lr0DsbqwgsTeZbIrO5og+k7BSdooc06j8KvgH54HJYbSnGcVGBwdr8WYJX5e0luuEmbNYNnbemvjtI6SjWc\\\\\\\/lLBmVRFg3ZbBU2AAnLfvVg6p1b8N72dfAXZSqjU7BUcP28IjhV8gkz2JP3rkL07wq7CG9XuqDx\\\\\\\/lYXuz4MnF4DHMpAOxQ2Eff9WJ3T2ltxHeGbdQdf7Ofrw6ERsk4R8603nqgWYcU60fM2CIIi9+Xcn7CxKful2i4fEs0HhIlaaqCITXSkJRVZPq8recpAfl52d\\\\\\\/APxwR63\\\\\\\/zjkQPDjlBziNXFswNa4i1XI\\\\\\\/Nc6fgUzQu5pygW4BjizXLlJ2WHLSpumGNXtN\\\\\\\/ZarryumwX46A7MRG\\\\\\\/6txsmzTysqLhsMCnpnOnyST51STCJKexwbS1djsfVFXVF0P0kT9Y28hVNJLtiCJo2\\\\\\\/+6T1o1DtymaU9ZSu9IgwYfAIi1X+FBb8wqJ\\\\\\\/zDU6M+31U2h8gjKcwfRqtXKdeGGzagJruSqfyXm7nisGq5UH8PLSfjrOmGlksngmGvKQXibEik75CTi+t5icNNt4xCV3HJa5N83hyK\\\\\\\/qjH8UGM2s4PO\\\\\\\/7\\\\\\\/Wgacff4PP9VZp5BT22uCkstq7bNTzmYHBHVpJqfLn3wxJQq+zsV5ZiuMDKXz4rarYWOhF6BDxCzVgTlm6KxyhYtHeEalkShN6eBa4jbc0qejHsRdv7b46mLTlYRGDlkzrS+ixR6lBOjh+4pFY+Add2nTlbtUj1IHTeFLz2usB2LytfTpaDCMd2KSOwCGbDwYk+JRH3SvAFkk+QD1e6xUa1HIHWWa9zH69S9OYpcd8OIvE7pb15EjjIx02RlgVs+mOjKhGHjCi1NqK1ZydkuMLAXd3TaOFheJXI\\\\\\\/IiwasAHQbcmQYJHeaDWbSsHH5\\\\\\\/XSX445uyetabGhZClCZkQnN8xXgYQP8sw\\\\\\\/ThW4g2HstKzyVEQRdREeIRiLjp+xrI0eBFmNBGwEozUWQMWtbDW7+CekBtL94e33xyrvpjTBk87NzUMG8KCN\\\\\\\/VIxJScsslzV9Jvp4jLXIWZEBSZr20tt6Xn2JhtBoJySVfn\\\\\\\/YXQpDqp4ShmT2ib57sGqUYEwKtJFpkK2xh+CDbAc3kWjDbaRPT7qxq7fkT50KKdacQ\\\\\\\/SsWFS+aIWCRGov8w\\\\\\\/f5altQJtbhKs38queEzW8t4I9C6VJXqu8qpElnVICruPXL3GCJXa2DRrxHCN5ukhrMO0Eh+eP56FWu1n3ICQhwCLl\\\\\\\/tgtP9X91DYUbtu50gqlusIhwp\\\\\\\/SD41rQ3bw1D7YB8OE410pzQygv9o4TeMNbMTx8\\\\\\\/WPhaaUNeElkEy9W281XxAIgZZzs6YUcsbJqM+aH53uOAm4ciNC8+rlry4ulW44dKL6stciNr3Nc\\\\\\\/1eHLnxMJWpdgj6tVu+FoCyEzNeIP9haI2RwqQHfCdBcmECg\\\\\\\/YFXuE84hSBF2Pvn\\\\\\\/lu30ySTAgEWeRhoOyNhod1o47eX6Q+HkwF0mKLWzUElyWtArChrb9bg6gf11koxJvw\\\\\\\/OvFvy3ThND14HaEwOHWCIhhp0XhpoJDowO5VBrOH2Ckd5lMSqopDR1h5dMSMkAJE70mERmIA0h1hFDY4nsclJk6wZSBcRH7ulSb4B\\\\\\\/QWb3t+p+gZXYx\\\\\\\/yrhzzftEHLPwEogc2H91spofWK398Gt4EVugTFL0Y6PhuVpY2+29q86bv6bL73s83SDGkBFwDcSARyv9Y3AQPvpU4bOSK47sxN7PjMVy0iPCjoC89xQS4n6cNU6jMi1G\\\\\\\/ZvVB1xkhnxsOWGy8FFxa9rxqWD9YviG6lyyevoyMHj3N+vkZEcdDn5EgV5fCrhxwdQUwZ\\\\\\\/44QNutZSOSj82UVkT9u4K35WJuaQkB0KENYdGPh7UDWEh9QqG8YqGsjk1boOqUhsv62KwTXqPklimSniu5uVjNiVvHRgH1TN2uOFMyT\\\\\\\/G4l1rDlOssZfO0h7eqLzNkfHgwzxP5SvD4KIbEmWoX8RCX\\\\\\\/1a+uFb8phDEAwBCSajbnfwb4dy\\\\\\\/HpMw8qU9sxbEGslYZjRDV\\\\\\\/UEsdqI6Ulp71DhdS3fVnoeypWp3C2tPgs5j+RKrRKsMIkNjKISvv7p2VrTUMS7JmJOXQymnf+pY8y3GEUBQOOU7TBJM3LPIZPMililYmM85zKWzUS4OQQBcv9PvfHeoedWUlJYSzEzVjxBj\\\\\\\/\\\\\\\/YiZcsimSQESi5fE8RHQXdgTcJJwJgsXW\\\\\\\/Ay4KMIWduRPx2nk3gQe\\\\\\\/S4Pls5LnTpjiVlLkmu\\\\\\\/3EtUz6tENHHTB2NK17z\\\\\\\/zKX5DApxJ37YhU8W4W7Fr1l8WeDrXP5dbQ9KigLAQCrk0D2gbISAqa\\\\\\\/mmUN7688VZGSe7AwV+lfrTw5ogapRXrWCUPC11y9\\\\\\\/YuNLBi4MW9cdgnolOqxGrSBsAsnm4d8Yz+TnyH5vxx7lk+XvldnfLnFVaY5lzH69RSfVh\\\\\\\/HYWvzrdhkPsuBe6C\\\\\\\/UfrOZHgtsHx0Wg1y6YduBiF2oVWgr\\\\\\\/kgWX2qZYF5M9ViwnnsdqtxiiS\\\\\\\/5Yv9xg5voEjlT0heAI2RNgtobzQ89Dolemv2PeyFjarLeqWshYsTS9iTFMIk5Wp4zi7b5oXg8J7yZfFhWTV45038ZttP9uqt3xyV\\\\\\\/WJDOn7xzFslqovV9d5g0uSgD77L3NfImAkL9Z1O4dCXuU3JNQ8qV9icR7sNMEWh02FOUlmp7zwN2fjBz8qQqzLNfzAPY40dzAzPBdAB8n+WN5yt\\\\\\\/BnZjuL\\\\\\\/PI3D1GjrZwwfsBhwcbifsCv8lPZgedXWSfnJ8iVXG0cIdSPLkMBVPf8Ws\\\\\\\/RQSAyxa4JpC23Q8uycQ8CTPVdDMKPKKVLtp\\\\\\\/i\\\\\\\/90YcZz5hxS8NsFCuxLmM2jWQ9ViVwrSBkVI\\\\\\\/5sEdrlUdJLTE0CE4MxwjhMa8z4RgKIypgSCLmChl7H9MztvO2KKjT\\\\\\\/7S2I6lLtG6Tne3Y7TFY85b1NwlL1ra2W0+uJjn3jZw4n7NsTY0fFTJkuMm\\\\\\\/wnijEFJH+CDrBnbJfJtMvVRR\\\\\\\/1Vu+cZiCogyG0q3ku3u9RFs9caFA8kKBhhXwqTxrg\\\\\\\/3nd6s+lf4ssbSbNAbr9aOfkhBBgG+0wLTD29bpP56scSdQcPDWs9K6Ex0Mt7eaXrUuEIvvadFiCmLd6CeitFh\\\\\\\/xpEl8e3J\\\\\\\/tWScLRjFpAyV6vuLcopGtqcBgSJiRruXbf0ENh85FuguNgR5ZOa+\\\\\\\/a\\\\\\\/\\\\\\\/XFlulMuvEnN3ibZm1bqeRXf2Lu+xdfaHc7Jfb0jzUL4M4d6InmDPARjWj4cMAsQRMmK8RhOC6dvClpEA0i5SdcfToAPCJlMX6sIcy\\\\\\\/SITg0Ln8mDO5O+dpY3Czz4xG8yGPdChVQnL97PHDLaRrlC8qnIj7TiDOhUFY2O0x65vChXXM\\\\\\\/xHgrplTk7\\\\\\\/1ZvQSUaNI4nIgRF4ZZEzlfO21D99IJH7LiGdQy7p+CRJxcVhZf2jPfDJR1YgKdOO4u\\\\\\\/9t5drb9kepq1bYXzosELMtVnntOgs54AH0XZ8\\\\\\\/dXBhmUVAiZ6AjOAK50J+dm01kjT5iueVCMbWCopihl87ck1rkocVs9paDWQBqFXULvjoD95pv79RiKVfwW+7LMOAAjaA7kQkGoEurt1yaNPVXDTKSEUBHabNSaXeECWrbKRd1zFgU827uK+aiHsGxko3Dejo5y\\\\\\\/jSU5rEmGM16Cb3lIFkDfzKI3GDbT3YoSZT4WiUfwXbOETSEdHojulcdwjYbng1079yZnO0dRwYpSvAQXZAFZGa7EQcR6BB9YCV+naF6NqiNT4oB4B874dpQvzR2p5S2OtZv8gRzRxieBe\\\\\\\/ePv41IZP3BnMNh4kw7jc+VyNGbxJcQ0Bo8pPyh2tJrbMoKdv1vi99JyDT6PZjzHJrH7ipBJFRXkym8BVXR7IvwympWSyWFpIrJY10WWHuEHfN36Hhbbz6dUye78eRkstOW6kWjp\\\\\\\/0N8P1WiwdpvCbZ3iL2if4jBTfbl2wHWehKeTleo3LGqhe6DlnYAnT0Y68ykUDCyrz\\\\\\\/XedvZE3ktoxhcFobZ1bZOLLMKaItmZ+7COIShrLZLLrhOcb1bh5ysEs5M5MllVQr8kld8\\\\\\\/43VEnnnSf5PusJKJcrORlZCY\\\\\\\/621h\\\\\\\/6FzWvnLtYGwAETcvNvBiJyGMV9p74A7khc3fN5IHZ2WslCX\\\\\\\/OAU975keB+rI5+BIxh9CJOzkqjjhKXbo6NgSWKrMoz4dpN7mgld5aEm+Y5VTkT7KJ43lexRWKZ5Havw13oZZHGENU9hquTJPSZmJmjDaGeEfQZpB4Uy+VpHH7hWKwp+szan\\\\\\\/yAinZSOZU5g+gkfdk4WDCKPySMG0oTDF9UQ9rU6tU+lV6HepStaFVRdF3odBaVVX2l9eTFS+xnWCqvhpL8OyIZTgjos5Bc50p9em3iWVbbbYGyRNsucePcLxZWWPQ+YkDkhLWkr3m0WjzbJOcCiQEJOKZSEENHKViGVXFfei0AgK6aTNug+i8gJyTp5\\\\\\\/FLBAEde7tjVVi9DxuE1CPDuzHa0fosNfA76TRXYiJPzibynge4cKfQqJ6Gm6+edOYeg9G8HrUSjzfFnohhJ0trjYpCftnSpQbUA7Io4wABJDfG8jgb9Tye6YpOi+\\\\\\\/IwE8V74ABnEKv1KdFN\\\\\\\/XhdYtYA1LzO1rFACavLPHBvTmgy+5xM5e7AJIlqPaTOXysSMVpbDVBc0G6sIkHpgjWiV8IiNglhKogamu9BKJwMSUbwOf0JbnrJVuB4SQuyNX+82JVWQHk3AxPu8\\\\\\\/jvtoYbUDsiTb0wsxHWT0JW0ZS7sj1u4tVDink2zhas9Svc9y7Q6n2qanbeZnc+KBzSzTXaPkhz5dXVFF8fziIjnkNRdBy5MaJCRv3JR264SEV6R8tK8BeLz+V\\\\\\\/CjxJQ1F+OhmCpzzjtFWleRYrPQe0evijcve\\\\\\\/iy5pyLmVXkjY0vLyXON\\\\\\\/n1Bv3JkFQe3nvoBKLz\\\\\\\/SAvwoCs1O\\\\\\\/45+6lfVnP\\\\\\\/w4h5mcc7ZkDGH9QbQj4RtNrfrSbliXTRum1wJQidLjwUDH3dRXvGqoyMOBJBDqomWRzedNnEbYB\\\\\\\/TME7oi3QEZw7A4wDyVuE2vGHJCXeEJPqiDiMD79aNRxgj9vGaZ80pPL5PdatS7nzAI1krGfe9oxwUvQ5GFNleRTND9NZ\\\\\\\/Jc6uMnAqN3mXYe+JIgu7aOa6K\\\\\\\/vs2K+Zd1s\\\\\\\/fpHHkGMnTZotvx6wAH2iBlxfanQ9Vtfpz2EOFRgmmXVSoMjrMmgtjhB4J1JCw7V2J8LhvygV1E4gAMdVqoD11ojIj7K6M5TEvWtWKpSsV827E4NLUKSgV2xsdNK1XqzLKOcWup5GCtXvYuYHrl\\\\\\\/i4J4L31NZvD65HbMQ6EcYV\\\\\\\/qDAljgBCq\\\\\\\/tNpJhQTGJvRz6gv1o4tdlTfuKtVX8ojABDVlmsPwOCfS37OTSukhxJ1t79TgbqeiHGdWOR70rqRgAa7dgGg6DQWzsXwuvRmNASoC9HIUVSxED52PBtVvQvYnpcxaTmmNEPybzDcuG1i9WpympTKRwrDM4RVXiSm2b3Lbea0ibS9tBzhH+ZZB8m55l3I\\\\\\\/H7YqT+2OOe2uOzLC78O4kZX+jVNtSC04agIoI6Z+ne5gFvmVBwOimoIKb5L91arA5iCt0x6639k9Y6eK5R+Sxt9rll\\\\\\\/5oYbVTImXpGxGda8NeqLHgIRw1b3WqipJdg9oZzudXNdRFGKukJIKsQMpS5mBCi0K849aO66cuFzdi47UEYeEyYVgOdW4bjCR48Li5r1uUjHW4sdJUHx1qUJ5kM2R09oVP6m3Eyjz+XjGR1VBO\\\\\\\/mzgO++5UrK2cZFYZgY9EULZB1gG8oks6BHenPEVDiy8+rt56KLG94p\\\\\\\/fhtNzZd737e6LLDyFEkh09dLh4ldXqaaFm\\\\\\\/sGv7b6Wwvo3OpNuR7BkBZw0J3\\\\\\\/mBCgKxOyC9ymR8nH0HVr\\\\\\\/QB8JlwcDVUiDwkg5\\\\\\\/+GTfzRtSk33eV6UYGHQqqVCNOrsAM7Pjy4IuPjtaf0WBANcCSLq2adscrAONpMyLC6geAfA7bUfHx4gqWVJHFrnJ081CBuJ\\\\\\\/730YbF6Ok5s0IfDPSc7\\\\\\\/dNdUag2uu7NXzTaTOj07iNqj6cJM2kQqsRtgAfUEOD\\\\\\\/\\\\\\\/cAEkIB4TgsrpsRlpPAsiZO6QjkvdZNTTtwt+qmFWYmGv0\\\\\\\/zmheu43Bdw0gyXHAsSoQ\\\\\\\/Pd6N0ftBt1ZzXxftaIQ1U9mOmoioYzl4mYd6b9YCp6VEcx5hd6F9ADKAZcg0O\\\\\\\/SK4UNy43PIuNehMbmPuuN8G24G6A3xcJiKoO75FwzIwxX9xwty7K6e5KS9wE2AKtxDNLKP6u2o8eeTPCMY6VR4xsPgFfzpME3c6YssqEo5mtyvasf+XxPCvyY5AGw\\\\\\\/cWYCdHWyaAvShmVdljmS0ZNdQGamc8UVvx1nhbL3ykw9FFfhI1bBB0djJ8n+d9zfKS1\\\\\\\/Rf2Yz\\\\\\\/svqFUvJxJY9leE8RuZnAf5b8r+VkBX5Oeg35bqlUyaKqrvUsCndgDP\\\\\\\/hY0W3Bm+FY9OodGMeNheYarZDbPyrJltASMmNR9Jx0bmc5mCFsNZqZ7N7HYM6Cblaw0s5nmlmrin9Ldkg74eEVm6OJf7YXK6vim8BlCdI6rGIrt7DDx6z+dwJpyFfDKejFzI5kaeb\\\\\\\/YyAvdZ5pasu9OhpTg7wijQTIRluIpi0WOZJ67ZaHaFXqGCHxkzSeZTxUuBqilR1O0CxGBjWPay2AHtLRGAJpwd8sUjy2sM4hTSMV\\\\\\\/udKbmwag0cc7oLkjPa83hvpaDliiPcF8J7FqIFyTU\\\\\\\/sQOmNaF2ba+9ta6bV61yfI6KirOBzJ2Ja9hLPgmJ6zOOUVcMYG5afdNvKaHN6whqngkdCju77zWlVDAJtmvBDnFJ9PPPxIyDytZS8ibTbdJY6XElnendp+921FYj1WXwx\\\\\\\/mJtSs3x4VoE2UWF43hD5FfwKyfeEyTvVZeJYgdwIXDR3t6jKcKaTOtSmD3YqQfNwlRMD0QfGkYNA7+rucZeV2AMwri8A61fvo+2dWZFQgfxc0aUQzCxmTow8dv5oOgE64FPWaPISAa2HdNnDF4sJS1HqPO8YE\\\\\\\/+9bpHFegqzYFL97RozBqPbn0O5Cll5fV7mRjJ95KHhFpOkeGHNsZLRsDsvUqzId\\\\\\\/BJI7QFwqz+gaFJdmc5+PN5G5RMvu8F9x7tVIbNxO4k7rCJyR1odq4Luroyfp6n88\\\\\\\/vIjfMxSYkbpjpd6PbaO6mbh9FaoQpOywZ8SgQjAQNYMcsMekCh\\\\\\\/V\\\\\\\/PsXtzO7DOSgFGMSXicEssqWYJIN0KIiEsAqn5ito4VX6aOiOlhXyrJViOcV5Hyj6oAKE4ciouXwQd5KJqz8HE9KechPWGerzR2bDYNrwb1JHDufNSH8IqOo5qec+v5joPFVuuOJylOdYrp8SPs+08t3HDDGoKEj7Vg6X13VMZdk9BjrzZEECyjaoTi0GIroIp2YHelmpgKDs8BtPe0tEgxSKzkjeUN+76xuqapbkvaE8IjJ6+d3JwLwumIkcbV53d\\\\\\\/B5HI0+lDP2WYaZnK411BDzcGFly2l92Hl8b0HsG+5yWxVYNU37FGHcMX4IHa2+Z07VJgJSS7P5S+EGxZv2rrysPgs1Exfm\\\\\\\/Z2+a6jz5UaGd1I4iIopDXAUEoV7wJTLgwisCKzTqgQmntrjc6QU4g7boPa16OeeVVLifyDFAU59Ov90zuSTEB91pYNvQE6B\\\\\\\/dJYYNduPV8GVyM70kjXsYo9Lrp0S5yEl2t+ex+6iOPFUmJy+E0kOYlkhfTrXqL2BKxynsvjNWosIn5cxm3OAcVUSifVx6sKiKvga1eqAj5jZ1Ang2Q7kKyUNXeQk84+qazYDxyT1wlsEymp2m6zTdrxBVAazzIkBoRaKpH4IpOGwn0tmansqp\\\\\\\/rFdie04kvXET91R\\\\\\\/T50bQSRmHsZxf63bxNyrtg0H2sBaV8nGQioWi\\\\\\\/eDX+RGfDbsKju7n6GN\\\\\\\/YFpBFTC4g9K+0qHjR\\\\\\\/rpOF\\\\\\\/iIG7k85FPOSyc32w+0fW91whTnromh99emAPj51tiTr8FwJEP6t1oYxQ8+9ocIdLaiPq7yLtwZBvOz9ViVvOBgKoH29ceoy8ls16hc6hZSNeYVu+\\\\\\\/lwUlF6CIZaXK\\\\\\\/CP1rj41Xn0SzPVd+SLhIPqxGiRCNr5U+NTE7tFjcnVWjDJbyCE8J9qrvzPwn5X3HwG2abjctvqVSQo3h04n1nhLubvwVJR0oxui+IDL6uEEqbmZsdh3RmUmaiizatc0\\\\\\\/iYgHQ67CN+5DhguSyALURwbRVk54bqYDn+Ru4E0CB0Or+71NTX+0zhD05xNWyfdUYieCWLvjg9J3qIiunDXeovDM0eKsRwzfDsJ4qPV8YBpKpxZJyK0hjwgddv5idbq7cV2jA3oOs9JaB\\\\\\\/5t03C3WlUQn3eGBDOVdZXZCfJclRdGB1vILicd8CVJjdEUlF9dryVV2H6o2sc1aBp4QUzIM9ngxbyu+B6m88aPDFr\\\\\\\/gJy3UdjXjNQMWxV2YRV7usjWj54QXF5VM4SVLOGf02BUAA+QLufcW14iuzr4p\\\\\\\/Jt3JP4Aaz+pWoBNeJr+tgLSsKi7XOCD8D3ORNwOWgEV0fsujJxGK3h1t4soPSae8NB8wJx+uHGuNQDYxpKgBW77QqnlDrUrkp6b3s4w6KhUamNiWMYXvUwOU\\\\\\\/i\\\\\\\/drFvhOuwN3QuoUURbM6pEU2rexLeb50YpFREEY1CnqrrrDKaAt1wuCt2SXwos1bUhisGg3ID5fmmj0s5fnko8K0hVnvAXMgVTbtkhbaLwM63KoSZBnJL7cs4gRndZi8IhZeh0BI8C5cWbHDTJ2tP0GL7OcwhrZcB1WrBMrSOjQL5QxjCxxJmUJsncKGd2ADpCCfHAyfsLN49pALv5gvZZqoRFTQpNBmPeA6+CCl+vMMe7L0jKqm5INlnWYqK+uuQXkJjCWiTXxIyTdvQtPUbm4YMU8qoQvfFXoNUmlCuVoZt5cWqgqKTOrSZs0s6SO6ONcwk67N7byscHz5Reqm2mzcxGlyr7K5JnQTWk0Lz5TL8btSr4B4NQW7TeYdwJJXTCcpvBOcnI7iiD+Yg\\\\\\\/NjNTlxADr9nPFz8qVTLARdDCs1So\\\\\\\/vuTs1g8trtOuzR\\\\\\\/IvnAPQ5NfCxPjVVjVBNiXJB1VI2TNm0zfpjZ+plYnUBzhzmFJxXL7ysMI4aa3bM\\\\\\\/\\\\\\\/\\\\\\\/RvQT8HFZDegxMEX38oQD19BwyOibaM54\\\\\\\/DCgCjNpNn6Pq+13T3aW6E+xS4PpIbipQdEg6dSgvRe\\\\\\\/VjUTVSoz+u4MmTMB+Zi2sluenomFgMj0fUi2+IV1cOfWt5LXGM8Fff4+IC1ojyS7MKaI8qpQgCY\\\\\\\/XcxcxavexPYPwCloO0RgKd0ZHFz6h4qtHbRs7Qbf7rgssIjgLFxohSkZ0hi07OzpLGAQiXe7\\\\\\\/sqRkWqyy1paTzcUPA7rZVisPlnSfNLtFZFwKkOgeZle3qQZIJq2uTqPe+SaZXXJ2r4YQRMkE5S13fKC\\\\\\\/vVYj6tGsU5N8YhNU1QSwwf\\\\\\\/nHNRJz3CcDax09iKLDMbfDX38WykDfGRDeQJO4nil9bJElF4LPEEEGrjlufBQU5nxeVGB2+uBbT5GKXY0xHYRHeo9qgj+\\\\\\\/Z0Rxcm+flboHIsB1DN\\\\\\\/vLYOZeJGdqPVa7Eb35nK3tPdL5fg1avY+1X+PuYOE9x0UN4A6Ha2Ha5pSI+X\\\\\\\/t7baacCT121RR68mk\\\\\\\/xrHirSuVhST+F6phZisUbjtiWsiQSpPdtXdMY1Gb1VtIguRTTHyMpcHP72sdDCyATKTWjJtmLh0E0I94EivgNRn5l5k3Q1DHzD\\\\\\\/AyxN63mWCWfTw58LGtOcYownLhpvOPD2h85t95vQLN78vD8L1BTwf67s3wlqvUlFqJ5JMsaO3XG\\\\\\\/Yi4lE+qEwVe8ioFSdMC5RU9P9sdoWLxtXQwGq6erUbbL1txNuRJ2xq2GnY4IqClG6RuHbFS46toUe4Q5zxKDf1pauiOCRSzHMHqBF7t9TVyWqxuUwDGcpS784F0LVJoz4V+50j7oDqTWOhqWhUh10lDDEa7y6BQ48JMBhZQN0nM2RIH49ZYJl4HSmzaTcQdP7AxxaILtey+V1BLSGkXkxF7ygQWfVvqPZXhe4t+B3Gfx0mHWIlfSNo4xtnb0Boc2x+Z2Xytcg6rfsvvsDI8U85W4O0Ie29MqVpP0zqat6HXVZ+01gJbOnq0MvHOeM7JF3r2IphDUNj+db7wFNLW257wbsbDI2ct7agCHfRO0YPpzSQESwIM7GKVLRwS7899sf6ewTWdAQDFsRkLdfYjW49uecyWutwNXjB8Fno5nx7q8snTj3UXkcw93Cd7VzB3dtKRDM7FNNwddV+HrjVR9L6roEbCMb5cPx9XVnlmY3VWvkiuFpaLaJELx5PMjVpAWy+bF5R1gi8h+178508VOztGjzE3X6c38zLZ+bTWzk8zPKXMCGrMFiIlJPczf7b3GHDgdCJ2Lt9K75u3ZBXZJau4H4OwESF4D62aDwqST6Ir\\\\\\\/GeKhANXxS2JM1SgoFeCfrFE6X6+Bp0sok3czCyk8Qvhaac5ZKxyT\\\\\\\/ZRfe\\\\\\\/acpRXMM4NfGfD\\\\\\\/oYKgW9jR86sbRmSCqZLo11UW5IIFLYyElJQUJVaq8E6q6tZU\\\\\\\/jdZ6qb9LZVZPiumpw8JMmL9VatfJlE\\\\\\\/Yt9OymJ+66U9oNaZ8CecGWr4IdBqo0sly+BGlvYG7BYKO8C4dGr5VQNAyl6waAjzyD\\\\\\\/r1l0um\\\\\\\/22cQxWu+QCEB6qvLwMybHJ8ckl+yn7OO7wPWjn03o0u\\\\\\\/+M2uXCN5blko9U+R4WCXs9AIDd2A6oxYHx8qo8lQazXBEsb0IsZcs+EU0WZ8vXXns\\\\\\\/MuJYYOfHSAyj9U+EGBj3tpKwvWqpAOcyecmB1M18mZAFBjsdY6MJRk6sITw7YBKTnoPslQQQ4teRr2+NfhkWDwKl+xBXK+1CvYnh4Lw4aMcPsjsbZLkepvaUHy4sw3pVbZBtN1YlU2ohQSygBl5vYMpioCnKc6MmOSt4XnN2CDzvD36BYRPiyvnZE8DzoCGPuleyDzHNB24bld9t60L8Olf3ecFW9kfyU\\\\\\\/F7UhG95NGRd5LU1hu8Z+OjXhZ3Bc5wiTmA76B2Rb9VyJh4votg5P0yWrtAaoM+UHXZmfR0VCDrh9ZgiVBaG9U4VGi++G7VXd7ltkNTjSq3KcvkOWwsBIAefZf3MVs7fK80BT2tUm0A1X0YhUeMMGtmgz9FymKEVZ6D8s9qg8xEG7mOG9HqghBsWpahTnfcSZRu7EwEJnHy82fPn0SOn38HXaJKiky0y3mryAt7aqmpRWneczLh\\\\\\\/PQ6+Hz6hHyBwdBPIPjZGOH3SaiNhL7LzcjRL\\\\\\\/yuu2adu43TAFixN1bmy0Ihi+CXOwTG3KH0yHQCM7WA8MezfNCi\\\\\\\/S25EGOsuZXeZ2K2DsOGWb+dQQjK+J7OA2ofJY55LBl+ET11Ug2I1A2Dh5sv9fld1OxsBpO5aIic7stu2l4q\\\\\\\/R6g8zNuBWSfoplcVQZ1mnegUe1tXHe0BF+1m+Ufs1OPHfMCD38czY3JvyWaLV8diZJB71ZAwSr6r8AKo\\\\\\\/C\\\\\\\/cdRtPukBktckUPH+ifN0moNxlTV5nO7JfCersrUW2pkVJMkRWA+AQp1J6Uva4xqQXfKm\\\\\\\/ka09WRcDZQwxpc0EJq97B2TBge1KQZUW0OOy0BSZNOzsN4\\\\\\\/sl9wKsIvgwhTozBUf8vl3S5jQ7wuTD5VybHDNGFFihjIyUDo6BDWksKR1qhsTIPrEE7OeEohbmK8+Yeqf6UZZLgu7p+GqkhohIBoEID1uzO0q1d0yt9l+i7Ognxj8Ix8EyfsUu17CLfw2m65mD7jkuc7ZqsSp8YKIZaBUTE5jJM+ElxED+3lxh+\\\\\\\/3H94HXGhC1qZKg1gpTbXSyo7y9g6l6eZHhWqNCKALNkpEx6ZmuBAcV8dRjUdOXfdCA6XYEAW\\\\\\\/GaPlOfOy5Mwm18Qu9Yc2Vx5JVC8GcuGixFpaq5aRRwE+vPzKXX4Suwm7stYgoj4ZOTVPVnyRtKOnSSN0jf1huCQswSHwQ1ZTv6cxRB0hNNMID1QFeogNg+ZzYuMJteqmryK3JFkIInObnNmB+5iT1BJ8qnp6wZB32iPd4GnrQlYwtX4jiiaiSlDDeQLUqZmVUFExmIDpwNY8XLhTcTpwmjDXaZ++Tg6aZ5tFzudfcipV9\\\\\\\/fHm1R\\\\\\\/3ZfDlLjyMpsYdYhPJn1Anz2z7tth6SPJhX4vfyMGMjUzAa+3CxnhUQFc9yoWMNvL2dS8JMTo\\\\\\\/XZJFoW7OG0VSXzF2zIDb5\\\\\\\/dLobqV9bbwSbtHq0uUPSkPFXuoNHmVohAy11SxnDkT5xGKDZPy3676o0I5Mj\\\\\\\/PN8CLZFjGo7\\\\\\\/oCpjhU+4yEmbi5hfF31Uss+GUAjxBCuPH\\\\\\\/Li2DIs7Dc3T07Q6bwHB7VaJ3mAaKYWIHMDT1iUykhHpvEwjP8QW+eBEou20aicjWLJ+0qVOhd01BUujb9Num\\\\\\\/Djy3thTTh+mVmjDcucDmpNUmBOZ72HROIqThLpsFF81Z18EaBh2gXXjGqrYnNYibYHtbafDfCkSLVo9xCTc6vBw5Ga9B5RNowMlP0sAb\\\\\\\/dQhZLJrzGc7GB62FKraXLDdKr130s9FPXIWUdynrn+DpvJh+88Z+kUwEInQDPtpJGL1kYyS\\\\\\\/eCPhZ1tiE3NweB2Q\\\\\\\/aQQsxOvMp4kbhjb2PfTG0zgoBZcLuc9wa\\\\\\\/NPz2XfVe3cuh6\\\\\\\/yVlIULnQ+3Jf93UnIoJlDjWNDhndHVnexmX95kMLOYfzwbNVDIn9lMvB4xM6cVha7mZVXJxz1uHup3B7IG0Vg9Ro1Ylv\\\\\\\/3JxebI6J1q7s30N73Zb1hSUWVMT3TFToOV4NGVD3MDnafV2OEj6wdoBaz215UI5jnA6tUr9512lz4HS2Uok9ohpMW1EtzSaBNm4CWMdX67qgqtrPYXn\\\\\\\/WLfenQkJPQJWFTWOPelTEEUL0Ms3R5rCRsGEvbnCgMG7Jj7UqlpoLgv5xFXq9WqLTUV4xXZk89lQfNneHD50IhDOGL3pv6wKwrUIKz80YzyElipIhODm03IfEZf6ZkEa0OlI3RLjbN\\\\\\\/a\\\\\\\/bn6BNurOKYYfBE\\\\\\\/QUWiKBQED7tMufq3ozCoSf0RRoi2jFroWf5daCvFV1UfwMpsIhILoQYkoOwiq+UzCKCLJp3e+vTyjKxRoXx416vmkZJhy2EaAHQZhJ2JAeFF7\\\\\\\/BTwzzo\\\\\\\/CNA11YYxx3OQn5wc3eWCkMm9idd3hMcFYm\\\\\\\/Cp3j\\\\\\\/FdO\\\\\\\/wEB5wMGcX0hRDy5snKXs\\\\\\\/BZ9WYWffatn5lfkBIfS\\\\\\\/mAe+PiQjYm+rqFFwiOdNV5y4XTg0NDJGLjHVN9ax6d\\\\\\\/gLyNSK1QyAnRr\\\\\\\/r5Wl3vcFb77L6xQq\\\\\\\/SOSK42LX1nFGndXwDHc4zxi9IB5o4mmppoOBFAkywToM1rdVB1XQZZuHb7m7OMwk5Z+gNgEGE7Y5PlGXfbealj+MYuTp\\\\\\\/+uyjSThXY4iLNYKSrzqioHupFQc7Fclc2tSPdz3yl8yZ8lDjwlhBScoqKoQogadplHqVaySsOOF40OgDgJLo0qoVWaYGD8bd\\\\\\\/T7FfLEj6W5H8xAbJ+rHZNl1eOlPL+EKMm3eTSm1TDLa84ju4+biqpb9NneKJDMxDGUkDNKnH+1CQ9Iw6tRO7unPE7SNfHT2w4g+En9tf5Bc8zoYpMLPYPH2HZe3s6bID94FuxClufUXoPO2GNgX7YR5UrBaF16GrsSfByMbzelwxLph3QCeTAOoYy3pcruG+4bg7vrgo0Te02Bk3XvoIM1hsah+57HbG0XzFZl8Y6TU4\\\\\\\/6bCu7OFDAtNFYGiVWIEo2xQzNtt0W8yB5nBd\\\\\\\/4ZPWyw76NsD1JWcZVPPYqUnI0KRk8TvQTQV14lKOVCQKiY+ak6fpxEZhaaox88yIov0TOcBa3HSPIDRRPs4ZOgZmGTZL\\\\\\\/pmHiy9g7ySHI\\\\\\\/jNOE1DNrAtgL5rdTyMZoPeMzmM+4Z7xSSE2AYuMk3csMQF6JzG1Z9Ai8\\\\\\\/pbgqGuah0UVcsNGbrIXnmkIUMHaiTJK+3CWzAN63yIwQTT3FmwWX5TmNpns+z3c6hoiJjWyKZDOjP35S04B0j1HI73UXYx41BfoUpd5E\\\\\\\/oMFlWH+kmSamtoQn0yHrCW\\\\\\\/5P36oxLguT3XCPhuxq55lZXfTFOu275F++UUpmwaoz7SRlwI17YuyPQlMO\\\\\\\/sLiWs+PR+\\\\\\\/YZMHyjZaJMuVaXyORCV1QYCY6ZGxjFZnt0KvHOmxfAIui38gTAzcBnDHTQcD0gQ0kSsqCt6nVd3A5bHCChnnibJaGv+FNMVj3bB0QwTLU90dEiGcO\\\\\\\/TwbY0fAythWGYgRVYWKvXOuOZcvOf4co4zC3NPzWiVheSBJufYLRE99wnhKhUzkhcYM09geKhincIkUGaeg\\\\\\\/Wx7LD\\\\\\\/4ja\\\\\\\/IEVV\\\\\\\/3dNEB6LMP+\\\\\\\/hEeOeHap4LKFm5P9iyva5m9ex\\\\\\\/5UayczYywBPrqFyebj+IA8kF8hlnp6WvEPZokLTOxsO8YYQnJwNPUUm7Ku4BMOOMhZHABNYqS60KL1YIVmRRPIXPWruwwWl1Rt7UOSqt+N7\\\\\\\/ptdTfIGFXUsgoEboAn02NB3DHOWv1Uir\\\\\\\/DJYpaT2chuGBdif8gw90+pLY0cGbmYKQW7gN5B4fm3xwzWaXZvH3V8kHv5Cs1wY4l\\\\\\\/AbYlqGf+9AQK+5Hwg+OFs2ctVOV8vpqGCfuq6Z9+sUEUUq\\\\\\\/sd5UHHODafe1cGtrJMyPqmDjifJ2JQZAsCQpk4SrvkLZj2jut7I3MX1V7tMVSdRL37fJDm3050ahurn80Cgsajg\\\\\\\/CBrPMRIHHOMfj7o0jcr7e10r03FIuHM+KcGDsoI0b+K0I0EvhE6eF6g\\\\\\\/HZV8kqJ5fKz7BNNCZKpFIIqhfvwDTAU1psWj7GkjNbVZCmk70kvlksuKGwWRV8cKXbMW2jfMH+1q+hoXOYddmlP6btrISSTLbrVS+R3\\\\\\\/sKmXi3XUloxYYSk6YbYN4FqosdIUKw649LpPLhEAS2L6zUO5Er13JmGYYA\\\\\\\/58l8EZnIWeCp83oOlNVgjgEuhXtA5fFBGi6cxavYAv0EK7KtDXcEFFcvDUrpP2\\\\\\\/zXSBc2EvL6bDKsbO+R+yncQtGRsi8MQLm80NEdlSrRGirH4E7LYo03W8N1YnbXbhT0Ieb68TU4\\\\\\\/l+TLC4T06rj1cZXp3UaxBVHsamV+K+Rd3h3NS5uS0\\\\\\\/3PndXUojzr6urfuT6zudg166OJug7PuEOgvDmoHzRNCCCl5t6ym7xIoP2obtz1yDs36JV5br14ItF1Z3qWTgt\\\\\\\/6BeZS6KVehzSEj2HahS4gyMi25pMrVNcft5BloZDejf4CMtm7\\\\\\\/tPL09g11hLVwt58B3pODu8fxY6cb+MUEJmpYC4UB5jdVFsRhl\\\\\\\/svh4NeQbR+4WqMwsIIhlt4kC4fOmKjm0OD96yf8hEJBh68UqzQMbQRjn8o8wnvzG7ytKewyZXOz04Rv68714adhkMEVXNY19MLq00swafFndP0QLkpO5A3796fhO8MTy6BUnQ\\\\\\\/+9harmlUJYXxxw4qF8wV8OLmKB69efqhAvmd8VDEqSQLxxrR2YSlzyczdvmy+ULMliiTePdmx6Wc4s0brmIoX9X6MLD4k9F+iK1cY5vjS6tIgtlFdgSH5pwPqE40caaPFL1SvmnkrId61j38b96PjgAZMqV96DrIvxjQdIEcRFqpR6W11meryU+8GMm33cGOr09SDAzGwbjbuwL9LNFWbEkxSrIzOmxQG7EbJJjdqclTSFn7E6W1hNOKqNVSA644SiLXUmHuLa888KP3vK9iC9fS+2pCCl5s0Mj0nat27Q1B6ezQlw+Swh6HKBsF\\\\\\\/m0ASyfRq6BFkljUSey4uuZg+HnNv+\\\\\\\/RegzMD1\\\\\\\/j0i3ELHkGYtFZQBfKcpwJpLrOmcdb9dRE24IyTYYr9cumPmvIkNzsSRK3yHND5\\\\\\\/nYdy\\\\\\\/mUxMj63zwsF\\\\\\\/BcZBNCHcMFn3R7MoSGI95wZYB8GaZ3+l42Ymr9ZLywIcOrtCGp3i74ZspBldVfcHWknNDQjKpJIE\\\\\\\/z0+SrYqNPFylJNLer8wefiz44QB4VZzEyMjGz7ExLG6F4nUg1B0xs4SBmOaJQHcgEFhqzqKRqkpoRGh64BcW1DhTmPilyEuWXpH1ug6vQ+\\\\\\\/BEAwAro9o2iUtWfMBJIBaUMi19dovE4ylz5YGkde6w8WPPppPUInnyWlkWyDdR3Ar4FZbohlppOTxPqg1ZsIvTOdzkSHNA\\\\\\\/ruLu6r1TvfA3K+ShEpQpG5JrIPluiWMVuZPiOea4uDC5ztyetR2nFvKR7iH\\\\\\\/gtZC4MILHybUlsYhHhDgMx5bI2DFc\\\\\\\/qf8m0Q4MnefaePQt5Op6saMD8eWkMnkh9effm3ZgvyM+kzzmhZTeL7SK5deKuBHzu8ujGaSTl+wBVEtm5hVOaOEyTs5hbb3i1AsQG6BXkSnOx8xVCgID5BfEXJHtgkrOzhqher+XsRNUiYdvWXtu67pVJJO+W0YDIVQqgrKlAbxs3ThDuBqPugbgk4c72p87CC4kRTNI53jpKR8MdKGHsiMuj+09NgXfTMZA7X9EtZvJ7bbafL6+I8+Bxg09S8K++PxAXf1Omuyg8ELL5j6PqqlFgpEv++2uOpr3KmuH3dDRervAtS3s2UXeoFf7a971F38a7DNuzGerGmRdnj0US9ZfWHpahiTZcuQcItGUrPFAj+ib9qyUaTRpaI0nWHVwv7Sv\\\\\\\/ROnhXsWe\\\\\\\/rhQw8dtNTETSseup3Z\\\\\\\/7qWEhY8JshONnpiLJydrvoaYtKcq2KCg9Crsoq0uocfNTqQYYsQDMxgnFYk98ouFqICzBpbdMGHq72DfP9t1brdtw3Pj3kHMMekkQF+lHonvebggB6altxRMAY61wDNoX2OzcFDiFvCFyFAdFQYK2VFOo69KGhDcN5nMrhxJJCy3EsLZH7WQ8A7AP978PfG8+uCRc5KaSy\\\\\\\/3xzvM05wnsY1tl6H63oYcyZD60+OFmuqxwl1JmDw4aUUTg+rZmWmbJfCuCsYyt7WafPS10OZIZSlVENV1m3A6ia6NOHFMTrB1SAYk7ERKG6E8lZv85LVO6TLUHed2jZiXwHAWGDBl133OYq2RM6gIvDnDkZjygI1zBQmJV\\\\\\\/fikO5vRGKaG+9l\\\\\\\/WMjci7nFSYiQ5U1ODI1GopWLMpHt5aA+PyCbVbG\\\\\\\/rMTG0MVMAQyoiEboQ8tLE56xf+4rFFWuKys1K0LD3xp2k685ocnUnlWag+DEstI6iKGpjOaOxT9KMiPcFzYNp8LwTr4q8LhwvwVtCbRztenfEjMDg8IACIpJRm54Ue9Cl+vIwoEJ9lxdvOWZMZjKjCszZkEH+tyziY29M9GR+8GwuwwiIYVDB7O7M4UFqQYszl6Aq4Q4PrGz2dJ+Je+bTLMy0hb5FfNTeFcfFz3P2AAOzixnUx5s9QdkdG6TebwV8MBVXNjeyBIKG8YpCl1wVWSvoo7K4tRnnAIeR6kMES4fK+hQwrqMXB0eB2qJmLV1css6Rd\\\\\\\/+axBrdq1x0qYr55lbc4ovPbzIvlU9vPiYPKRZHcyWZbbduo5q7irh\\\\\\\/bvD2o5dWXh9pL2dSys\\\\\\\/AIledfr9iIpau6WyNXzQ9tnpHciv0VFw\\\\\\\/WdymaNHlmCt8xgaiKaw9Sp7rFHTnpbFlvMe0+qDlnzX25yZi5TiWbkY1IDcKodT81DI2qRsiM1K7y1LrnnDsvHYDivyMhykgsrMQKAj2EVeT5vQdOo59GCvpOMjIK4spjjE7HQlgqFww0no5Q2CuLu8uoHLn0gIn5Xug1asY4hLPUTTO\\\\\\\/Lj32Fh+hw0d19sVokcQjgGHadP0Xy+WHI74vB6fCkXxqJdEZSMmRVDJra9CdAckeR7X+DWKYP0pzwTHH5KplfZ6nYWH0S0Rgw0ff4R27tTusN9rLo41cD25vNj5+gIQpqyFws89H+lNz3LDrj1lxINIJJ\\\\\\\/70nu9M+73yodvcJQ4uLggOYNUcSC8qu5s0ft2TADvIDJcfFBYuB0cbAW+rw\\\\\\\/G6JU5cCUBLjuX9LktH4PqlouPcTsp1EjPn1XyPDB6NwWnIC3W1EtAUSwz9E6X1ALkl10TEgfr0BaT5Ezbm5rgosJ+VcnJ2ihOCGK5cf5Qx3yf\\\\\\\/eIrFi6MiqHtHqqUtoqrgZY5xotuMSUPKqnPuqwaeCZVjB4QY0h7REfIpOGEYz581n7Yno2HvULxcjir+H\\\\\\\/ztxGgYFK61DRpN5bPJrTGH8Of4jjuqdzsrOfMu8ZCcVyXLZXvwTQrBRiyfvzG4uYXqpE5\\\\\\\/5OwCMZmtuNKLz1m775B3EKF2jgTaTq68hyjTSbWSIkTVkX1uK9s6GCj1SG+7EYDoQ4tWS3lKiR9c+K6f05R5mRTh0Gbt93Ov2HPFZt2t6ZbGGseuomR8Lt4MzGV0FKiirrHWp8er8jEoOKsSv43IPduXQo4eWhMS0kjOX7J3MEAfDxesvQA68XbAjZhky3mFuCV9EzPyi+DN\\\\\\\/eiBGybwBPMUeVbpcEPTg\\\\\\\/iQMFbTBlbLmvxJmJfZnUXzoduiCsKh1jfiiEmK02LLhj5I5oTH\\\\\\\/wym5xedQYDoMKvS7W3gnoGVfiFxvkKDmx4daDlzCHYYlf5m1q3H4OEu64EKoql7sD7TQpIC3koLOckE1IquowaSE\\\\\\\/RRSR+ANLgXIcE2GyLDkoDY3y9fccqGzjg81hiPpNhdcOV8RHlfhAYCFgf9o1JMZr0xkwMzSW1U2kHqPzr2YEm7qGw3fxGbF3Dsui4ElH75i\\\\\\\/83n90XK155en\\\\\\\/WBkFsZV6cOTeiJDBdcsnd9EReIf22Tttch98GNqQlZyXlrIhyEhp89xLtMWp5Yq+y39aMRJHLma3CJOJI+S+sJuT\\\\\\\/xRLU5Zz2M42JvJuzemJ6I\\\\\\\/RY9NT\\\\\\\/xMqn2bd3BkPblOepPBNPQyDmfukQ4lPfueKvlV4XOdqFbux3r38TRyC6Q5KVIHWer7JgYsvmSf5vS3seMkn2UNRxKaQ8RXPSYmNjIDyh9bzu02ey0ThZJi0g10XEcr4h4eo8XX0XLUcUtIIFh1XMvm0DDJ4eaxhFIfiuDety3Ttwe6M5luK0KSaOwX\\\\\\\/BZz4qOW8FvSIhOUrSRZBouF8NSDvJiXfwgzDbn88UNtyVbyG+1+QZs4wk9cVltTbv4MvV+rkYVHeOc4\\\\\\\/KH1zjdeMi\\\\\\\/Cwm9RKYYnWlRtn3cM42gQCtI4rSsO17ql1LkBt6bQQqaeJunJEuGwAdVJA5LW3poiUv5e5FzzSNvsXxNQWLwniLsDbtpRpLTo3CG54jXATrTN6kHMWrpBQURJpkBNPleT60zctF45LN9rD5L3ttYUMwfd244+qDiLLcerZ12+o9fKHABAiMPUkxr5K0sEc\\\\\\\/lS6SzJscSiYKQov5GnEmF+HyRsJ4yixDEYcUlqYSZuxffQIfFgrweqS3qj6177iDkAZCiIpeb+tioJ\\\\\\\/pFF9dy1bCC2ytnReyeeQd2xpfVqPG+R\\\\\\\/Ez7wKnfsg0ov+ZtrnxD8QOGDec9\\\\\\\/RgG2Jbb7yoZKyPPNIGNntkPGucdpsxA2C64YUn\\\\\\\/RgQakVj+LnbOvIvqqtB3fUw5kTNTB8WmYMsTQc3Uzsvv3m1xcg6dER8+03NdZMnj\\\\\\\/wAv6pE5tLqJjqdDVI1QNaKEhRf\\\\\\\/qVAbVaCAdAxIHlwA4bFXkz1PUJirC3tHqs6XDnWVWpOTPVcjwY\\\\\\\/LzJL\\\\\\\/DVlod\\\\\\\/+MxNIDzco2lwtIiCDCXrMeUYzxs8UpyJKiIrPJgoRLwg+vQLJWO5Rjb6PesdEocxCk3FIPZ1VFGtRwRPO2bNgQCZak9EYl4GY5g0HUE46oYlYR8WNjn6EPJ5vWWSFr9kFrdP0uKMF2gfesVOt+wUUHIVF9zQNDlS6t\\\\\\\/Tna8vPDx\\\\\\\/voIivq4\\\\\\\/Ay8OASb7OF36KpZT3UYl4uVBNlaLwmBV7XlVwHWxEsWTNXnAMw7m3DQtsGSKFV0qPhvbTGdYHyzAk0XIrtF597ettyAReQHBz0GW1oJr8MNQxn1rIcBQn1htZOKM125Cf4HLCYAoQ66qoT3KzAVzrahvLpx7\\\\\\\/tDEjfNJag2vivy64g18X7NNP9+eOnM3xQkvi22gvw9ToFQ6U+\\\\\\\/Vhhap4rF1YFegkzJEvWhqpsPrxWCysSNDZN1vtOmGjfomFPgmajc6yOdDcOeKBK\\\\\\\/yFDYp5ZNZdGteAzoh29D22kOXelBf01oNhGxdiWOEp4+lVH2nQdsZJ1StZwGY4uDKOUuvHLr0qQbn0Rzga6IgUoNqlcoVKePP67KaYY8WfeS1+ZK\\\\\\\/SnG+eFC3Z268neKfEtXRlt5sIFBVlVHgvshpYHvk6N93xwL+V65DL8yLGVM7NfFuPl0BM9SUJVRnr37Xz6Ha66hn+nLBEhOORegtv6Gzths5dX6Ldbll3i45MQoGzfjlAE2mEQysPWokOrA31NdwsDDcv3mPA4rpUPqV91t6J4SaTzgiPUUerFAjD\\\\\\\/r6rxzkBoltSbtzdMmiOYGf8A9bDhM33Znxlk5Twqzuh7bIS00FzGGaRb19EtGcqIS6JmE75\\\\\\\/R3Y2Ec1pwCgLEH0XahlSpuU4Bgz9j15x2ewR2ID8+Cg7izzHBcSLxN+1r8huAcYZeoj8YsDZDh5HFJvPqIOuTNBo9txZFtbQ7MOYlqnco+6m07BaJmn+\\\\\\\/fqns7ZhTG3VVCmMRRHTYPaRO9xoMqfodfnwc55sR4sy7qMPb7iZ0mT8pp0X6FsXJBVu7r\\\\\\\/KNeNipdOR0KcCJlJfesZZdWli+o12WkYyLwNNI5dTfI3fTYPB66553DwnawEClt7mos5CW3+gJh5puyR\\\\\\\/NSrSGeM6+GqxxTIQIDqzg1hkQd\\\\\\\/7WHPikQr4BllIQWeLPlQuG0rgaA7ko2I7nkYh0w7wvvF5nB4foRiQt2ogOmmcWsDBvPk7Z2uX9w95tsxZGHPnBD42lpMyn3+vkeWah\\\\\\\/dyfs3wQMpxiWIuXYRZEVXTOLzv2kp5Ehjn1\\\\\\\/u6GsvyGefpFAL5H9syNMJsIYhQgmR8GIGJ1qdgSAEG\\\\\\\/U0tO3sv2bWY5R05hNHzHphHiEOckfWke1IrGmnPhDDdCQkIvbMj7VsNIZ6+vqA5vY7o0CxPXx1tA2kPOWaIqBR+0Mfb4FPBoMCybihGsWnjfbE7UE\\\\\\\/jfcmKnl\\\\\\\/pO\\\\\\\/bhveMiT9DHgzC1VkoXamudgTSbtQ67pKgtl1zyFs8+xgrkDnaPxjTSo5dAp2dzUEFAqhNqeLlmCmklGOXOZ3dn4HnWMeydrcwbokOnTdYr5OsQGOX90RhSaaneQoklnhncKfomA0JDCmhRv34prwERaDn4AzbG9o+3pzoMKaZCluRL0iNkSyTyTbIsmNFMLoqbs9JhipHdVsl37DTa1C9ug40Jn01rdRIb05+5rCw\\\\\\\/aThB8ij3ChVTfp+AR7IkxbFaDm2S\\\\\\\/LadjVYfvSxbjgHxGZAZ7LAXb8AlX6G3CRQXaN2OpCat3wy9VmEatdUoB8ZoK+vDj5riXJHgdla8obeyqVsvPsWL6Lty85gRFs+unxcKYRpU+f9uzkT5l\\\\\\\/6DjqgHLJwCUwNMr4nEbYxD5S9n2PRM\\\\\\\/ASV1Lt8vONtYg\\\\\\\/UaIp\\\\\\\/m\\\\\\\/jbT2azb8rPifzMjrfNEBDY1m9g7QvmN9RUlyppzE1wluaUAleFoFdpM1fh9Xf6eaew0urKI5y\\\\\\\/e0U04eWMnci2bIKbJ3sP6OR0bZE8RxLPU4EoIkx29B5t3TIF7n3gDvBahgbunB7g+UW7ndrWLb42lspPk9U9cTdRqtEnsT0mVil2Xcly\\\\\\\/ZGY1P5tX08VUfgFCEnCStW7KiH5WEyi7cMQvVSF9n4vANCRudZQd+LJ6GaKEjbVuoPn7Tm8Frl73FXUuHMEcVG3RE7gXJ6MtB19TNzxbXum2QO2BopAVsgRgvkfCb31fhmGPAiFS\\\\\\\/pCU8HTa7qR3ftxOqHh1oRLSYQ4ISK11p8dQ78La8R91cFDrgVLVviskoi3yx9ofIlNA005ymMJSqL469W2SM576WFp0ku70\\\\\\\/wG\\\\\\\/YEc0dQh9\\\\\\\/86n3twaiWi+nqFQSicXc8XuPLsWU5TPsybGDimravBIE9hL4FBcA84hSlez3OSDY0mZAqhhqbMUL0eo3PogVU+Rb54dprRusEqrWgoOovyyYJ5prcIZdYpYavZbcdG9q6GFChr5OzZZMZydCcBQtLMJ10RXgkPaZEPmMGOaz9KmtaF0VSMaHkctHb3qK+ULhTbA9HGn9KlZcIdc+OLdMZEvgDLLOIQSyBlXMhXWeREjIaR\\\\\\\/H9Yie+2EBxnzX4uYiu42DBcdZWPbApRsMDaNbtkM7C6EKpvBf57brrBanmZ0wnJCiyed0NFcZ0o6qYKe\\\\\\\/Sxyfhfw\\\\\\\/yYglim\\\\\\\/ZesCgosxmFNBpcqnQUbD9EybN3u9ZMo1dcdYhhIgYlt+IbYJMuxrUM+itI7NCcQyz9AJiOWBJ8RhQ3Qm+UK2ACdIMANarL1Gvf1K822wt2gusmBytxv+aXpPZAieRCtOd\\\\\\\/9WFeRZGZX+ujpNcz70aaKgSgGcvGNuIicBDRqnHZAOgGok+aOifzFkkrK1cjh\\\\\\\/P24bDGxOP0grlyMvVErlZMYFJrRNdNmqAqwp0dJpEnPSTsb+AYruwBV7pKlaMC8um2+eZ+gAdBhm60KOd\\\\\\\/GQB9pQv\\\\\\\/kp0+SAcYfId6b2mmlqoxiXhO2wVRG\\\\\\\/XEMRPwGBl5x\\\\\\\/PucSG0JjXSSv0KJDHTD4VKzCui31TpSf+ob00Xt84l645SRNQws1JAobzt90Y35uIDhmlSw48nUs127V\\\\\\\/H0OACRYUwpSy6YWqLa\\\\\\\/\\\\\\\/mw33VzbQQKmc7TBxntTYaiGQPZgHhddW4z5++v7JBtGWtR5Vws7pyvawmSTcJ1n4gkBW1z59spoH2RY7ZH\\\\\\\/b72772VK6PK9AtStgpKOdmBu\\\\\\\/7P6iznB4owXD3ISqBHZFWFC2nzunvTGgztXt5a45LwVBRVAgaO\\\\\\\/miAlOC7p+ZMSLQg\\\\\\\/PlaGCsYr5VRyncw8mHYZvO8tXA4n5MZgp9E7FHl5tj+iCBipBBupgdY0G\\\\\\\/px254wp7JBM72mjAhe7RF481GRXP7R7iv5mbFBVbHKfmEtHqH3CWhQaQqiABxybtkSBpqgDhPLihvW61XqG9p9gw9EaWyhg86j0WIOBMh7j3JJfuKsR3rBpbJETcn2Co5YRrQUFQ9BVRGKodFRuUAN86ccZZdRNoNNHh59s1\\\\\\\/WsgajHqZb7rztfBihy7+DQf7KddrQGi5N1HWaS+dw8seDK1H+fwCSVCiP8IQ7lmreijgYXv20Q7nQpTAR8oQAVKICb4L+aUJ1lxiLVmbGNgJYYfqZEW\\\\\\\/U0iYb\\\\\\\/g+HcS3B2r5nMZiVEApKnch\\\\\\\/Jest+TDgYobpMCMJO9jP4oYHEZ0oFdY2g2Nj\\\\\\\/vVeItfA4bhzAilWuNNPa0Z5pNJj+4M67uOHZGKljiRkjTQtTZlwL\\\\\\\/\\\\\\\/1okka5kfTO8\\\\\\\/TRULHLRTY0Xe+KFO40nhKZWiGTWPuWuGfaWTmAdoR7uery8La3WptsNt\\\\\\\/A11cEN8m+btJAN4pVljnSoMZNue+HNBynva8jAcZSbh4bt3JNdExtm5cd7hKDs1uoYqzN3buQXc3yxzwK9P7u0+EZqKCM8ZA\\\\\\\/\\\\\\\/OD5aoTp9PFsWUXZj6CxVljIPePZ96m+Mima\\\\\\\/z1z4p5I9h\\\\\\\/s8yvjRkc42fMY3YJq9Zi4UKTXNgZPVgOTDVl\\\\\\\/0TmDEfgt4lDFO5PwSoSmGm5zvVYAIo\\\\\\\/u0kX8WTm\\\\\\\/523WG6G3Hck+aGkd0gopQJ7gjlD2pEtDPNubFjfXdFCpYWDl24eA3cPTL32doAvOfjoBmq+N7A8mKej3FO0POWUWeg264lb\\\\\\\/Xijkjuqk68ogJKFEajqzduBL8zaB+e0ARedU5ECvrSv92\\\\\\\/H7NN1qOQ6bEzZWnBafh7j90SylTH\\\\\\\/N8sVSonWru6pzDXyDqdxtrUcsjQCa+bYr9dHF2X\\\\\\\/c2gyRd1eso++mlf9fxlQ+IF0NeuqaGB2TjTakqg7h\\\\\\\/tuE6x9DNgbiwl9H8F0sVsyBLXIAYQ9ZiLZs7w7uwwX6wFBtii3Wiyn6OJrz5Whgcu0UZXg6xpvdshsG7Haj9pcZ0eee4kx7+oTFfCfZwpByfpfLx0ACoPityQZVx2mVRpsYF8Dfg6\\\\\\\/WHKQJ8oPIEXwo0SBylTzxoXmoMqnNmcrhcZP8APa8VLf2bm9lrmWZdhF1jWmUd9Y+XoJZZyzfQVAHUKc15YwudiLsGhaV8laHRHEz5XwVvwLl2P7imqWZeMnEAVTmVwLvuJe8iKsLNqEmGeOo1qdlogM01cr0TSwi4lxN7oO77NfB8vPWeOM\\\\\\\/XhGdRY2yJBTt4YDfsxgTwFsGUB3oxZMQzcBxpmepeLW\\\\\\\/sBiCW6bmABbsqHsCXhFLRLHLyHDNCZx8FFKUgVGdmlXDhK85p6TsXqx4GG5sKM96SHWeQohQSFU+3Poq3HghMkwjXEVRlMhnmwzJCSXVq51RO06Yh3BxJdhmQH1hHq5BlaiLM6JjFDLrwfbYOJ+nBhn3Wy67mvKHvOluv7k3SrFvINr1+rwt5nVvPpIFnp9WCI0aagIb7jM842T6QPbJKX9ZK+7LhCKVwkIVYwZw9UQ2RF4q2RbAhQndYMvWqOtyvaGHazFgfUnpcppiAacqtKYOEKtNh\\\\\\\/Vf0yvJyWW3pHhWLntF5ubZZlvA8otg3wPtsg97GgSKY994w2dWp9aHt\\\\\\\/6eSBgIJg1HVU41KY5bgcD+euwN2XDMpdC7KGVPsJgA3KFk27PiSfYXwndIrWFJxyyXnRiX0rAGjLmMKEAYCDPtLTNAtXdmAe7tuz3Pq5l4Qe9XdQi1tWaprZ\\\\\\\/jzHIcaDy627fj+9w+v1Ts+sFRrkmw4E1gpUe3vI0W2LAC5OjSxQkufqIHTrjHG4JhiQy\\\\\\\/fcRJGqrMZ881++khMw5Yk7GiF8dqdM6InapM1DD1lDt56YGLrHIir6xxELfp+DqGHB5sB4IUg7F7pSzIkgd3uX1GvGugX1sFWPk\\\\\\\/\\\\\\\/TF962nGBCvA6IrfaJxJFixQydEzVdmU8KymzIvZOWgV0bQUOZp5iLr0T7fwtyZ67hi7zvfgPUQXiln\\\\\\\/u4YCtbJCKwRGGLAqNDmn+PzRbEEtNUGMx2wENfMLaOmoOEhqzA9ZLleA1+nVIL94AnLOpZAEWfhNggUUzyvtEm2PZh8b\\\\\\\/C97f0iDSXmtdRgGsDMfIuarEPnwrFZcOpTaeK+YGoLmYEWYb0qWOnm7q+p12hGwKQrR0Khe8z77WuL5OPLuKAL30rE07HhJYR7aPcixSVmPoprOs+D+XznZKOnQlaOO\\\\\\\/Ps9kx1V9f9Nu6+tw2S\\\\\\\/tWvzRF\\\\\\\/Q2ZNfgz3dgcaZeWlmIgmwbelwvRej66QwSkJw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022lWdRxw5bECPOAWm+PtSOQA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f58d3ef55e78bf383b31ade3c0c1855df844ee0899f0d8a3366116e3f4ffb422\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RSVSCFXBKmG+MUxC2fDYHGYav\\\\\\\/6plEhihXfamagGBZy1+0rXO1GbC8K4T6Mw\\\\\\\/9L4YtXKABgXvVgWP1v54UE\\\\\\\/M\\\\\\\/edxdXRRr1\\\\\\\/3n405WhbQvXFgyMSQCRNCXvVzAFeSPkboTrPKPjj9s6BbbUV0OrHHhG7YhFc1cd4Ib7eDPhzl7jf7QolypvJE5sO3zXK8iDVbxAG3t5KF+42HaOfeyR4UbaHohFJGROFLeq6Pa\\\\\\\/qqNIz2es4j5Ix+k2Fu+R2TzYpfxpi\\\\\\\/vqvex8gXIhy4\\\\\\\/nJkPjeuKWNzF2toPTZyu\\\\\\\/lm3qRbTIf8otM4Wpb1hm4UGFiwPCkZriAuC98403BBMURXGZlJ1Os0kVuUDKlmVKaM\\\\\\\/fx5jMIAxbbm7Wkqp5xQzL2ekzxR9dHEVsvc5GSXnrXrfSbELyzpYSRX\\\\\\\/4ZDtGJKAQad48BzJjLiZEl5mkrLjLApZyl97lcQCuM4YqFfrnc+pzxuPZzexGIz6x3JqRZ26b8VrOiZs5o3XIRcNMvc1MfLvd5FBmbFbKslf9dDyCP2RafaRbWCSgfGECDAG4c3LIDZ1Une23MMBIsFBVCz5AO7WD0f8rKb6EQsDoWBCW5NDZeb4MlTvfGbczfrg6AT4VN8L9MAW5bwYmOXsCAG6E650ibcOnWWzH\\\\\\\/u6DCgkt7SynG7HpVdq2LzFwH0OH0FvZCM4W482uwICKRDpMX9PAHsuMK4voyRu1GGNzMAvb2tuChbOPWUKh2A9onmXUFNRXaxeMRzaSKBV14NvtB3YuCGYVtUbuMuD6MGBmFJ63mFDU\\\\\\\/Qt4pZrMjfxM7nTtrf7aMFWPL8Lc4fMADxoxwKRzPksFqkVLd86GaOHP3U8KYtVssOnXPG4nJhTozgUf5jlo\\\\\\\/HdrNKzZSKAKdOKvbjdOjSfWNNMXNLD0rysmHo\\\\\\\/vq\\\\\\\/OHkKK5iR4lglm9O7gW455RIRXBKKkLgUkIBYVlULYbeI+GmzEjuOE2d6dnxpkt03iCzdOg2ZqkIVjUTkhG2WRCiSWu2xLnu5PsWnTv\\\\\\\/f5A\\\\\\\/WoWfy76aqcVnBNhfZU5orqBNAkXjhRYmYnODw1j3y6ns2CEZm8P7pXgWDfA9SJS1DFbei9QOimt2V9FcvK5ExBb+6mLIjKDruUuyoQ04A1YkDEHCrER8DjUT7IcLCBvTByglPqmCpdKyaJ\\\\\\\/9HNr+cHkdDZaFWdMhNRB6x\\\\\\\/T+w4wy40MgFo7UZMVPV\\\\\\\/6+x6\\\\\\\/9sFYUbl5xzjg8Sm0MS16k+x8W3NN0UCphv4jrFXMnHTFBP8MjVP+wk4YBTOMnprdl\\\\\\\/eYMCKDsywNFwhVYpr3dRzsmkrXMMqxL8C4ZuOgv7tDgFUisH0GteJ5psuQQX1gz5suAYsmsK2F0VBv0Jes2lBSrx2eDt8wTGrcf2I52pc3X6xqh0e2BoUW\\\\\\\/GL4MYnJ6xArxp9F0FU1YIlCHYa7P2Ajl+dXyV\\\\\\\/2tc6s4WX+R\\\\\\\/yHbOY6xmow6y9yEHCZtxD66MwcoKZxkG97izBdLINW\\\\\\\/Zm\\\\\\\/lQN6y\\\\\\\/tK\\\\\\\/jOInuSeKTpCO6PZjHBDI4iC3TKVX9cLhmSRi3qu+RRcVctXk5XostawAqvHyZe4aT4drPPDMY2MNX882f3klAFXIQxD\\\\\\\/WXRsYRwf8vv2n+3LXegc9XinOitOlTsZNPYj1xrVlcSgUUTafIX16xAZy29rYUmqX4udJwa1nLH2aSBXBi7kNGotBxba7Yt1NA9O\\\\\\\/VJ2iCDiXoclMIQUNVi6JjXffJgn5sG54whBk51uDYKW131EWzDW\\\\\\\/E9oRQfSAyRq4fSDQAkU5K7s+C57pSMusugUUP64jv6NA1nF0Fn+7qNMAaSzDZmGla7+Kc8UfmR7gocLgkBHeXFsusXvh+PT183xsGW82Za0MF95QGfyXmbLxvr\\\\\\\/H6g1RXBpngCRnRwoD6UTPPzZmlrNYjydiAa0mqw96Q1iA5R6cZA4Y+LwCir7xV\\\\\\\/KEGNP6TNwJNYz7uWvhNq9W7hk1Mi9ZNlSwx0V9FtB6A17BZQaazwFb6z\\\\\\\/Lnu5dGLkK7rIhQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022+WsCievnctq3UJCw\\\\\\\/lXfag==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a91c0a9459945cb1b246de7e8db598ec50475b4fdc50e5c55e5d81f7495e22d0\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459578 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-selection">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/1903" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-r1Ku6t7eRt5X-LwbFGIq7KexJfrOzJndwsD1n9l-8qo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/1903" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-hsiRx5_JsHr2lOBAjtGDm8g9Oj7EsVePnl8KqZ44mu4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/1893" title="Genetic Background and GxE Interactions Modulate the Penetrance of a Naturally Occurring Wing Mutation in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/11/1927" title="A Mutation in the FHA Domain of Coprinus cinereus Nbs1 Leads to Spo11-Independent Meiotic Recombination and Chromosome Segregation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459578" id="node459578" data-pisa="ggg;3/11/1903" data-pisa-master="ggg;g3.113.008227" data-apath="/ggg/3/11/1903.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction in Maize Breeding Populations with Genotyping-by-Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">José Crossa</span>, <span class="highwire-citation-author" data-delta="1">Yoseph Beyene</span>, <span class="highwire-citation-author" data-delta="2">Semagn Kassa</span>, <span class="highwire-citation-author" data-delta="3">Paulino Pérez</span>, <span class="highwire-citation-author" data-delta="4">John M. Hickey</span>, <span class="highwire-citation-author" data-delta="5">Charles Chen</span>, <span class="highwire-citation-author" data-delta="6">Gustavo de los Campos</span>, <span class="highwire-citation-author" data-delta="7">Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="8">Vanessa S. Windhausen</span>, <span class="highwire-citation-author" data-delta="9">Ed Buckler</span>, <span class="highwire-citation-author" data-delta="10">Jean-Luc Jannink</span>, <span class="highwire-citation-author" data-delta="11">Marco A. Lopez Cruz</span> and <span class="highwire-citation-author" data-delta="12">Raman Babu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1903-1926;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008227 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459578" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yoseph Beyene </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoseph%2BBeyene%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beyene%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoseph%2BBeyene%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Semagn Kassa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Semagn%2BKassa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kassa%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASemagn%2BKassa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulino Pérez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Colegio de Postgraduados, Montecillos, Edo. de Mexico, 56230, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John M. Hickey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>The Roslin Institute, University of Edinburgh, Easter Bush, Midlothian, EH25 9RG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BHickey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hickey%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BHickey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Charles Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biostatistics, School of Public Health, University of Alabama at Birmingham, Alabama 35294</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Vanessa S. Windhausen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Saaten Union Recherche, 163 Avenue de Flandre, 60190 Estrées Saint Denis, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vanessa%2BS.%2BWindhausen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Windhausen%20VS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVanessa%2BS.%2BWindhausen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ed Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA—ARS, Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ed%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEd%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA—ARS, Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Marco A. Lopez Cruz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marco%2BA.%2BLopez%2BCruz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez%20Cruz%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarco%2BA.%2BLopez%2BCruz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Raman Babu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raman%2BBabu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Babu%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARaman%2BBabu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/11/1903" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459578" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459578/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/1903.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459578" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459578/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/1903.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459578" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459578/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/11/1903.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459578" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459578/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genotyping-by-sequencing (GBS) technologies have proven capacity for delivering large numbers of marker genotypes with potentially less ascertainment bias than standard single nucleotide polymorphism (SNP) arrays. Therefore, GBS has become an attractive alternative technology for genomic selection. However, the use of GBS data poses important challenges, and the accuracy of genomic prediction using GBS is currently undergoing investigation in several crops, including maize, wheat, and cassava. The main objective of this study was to evaluate various methods for incorporating GBS information and compare them with pedigree models for predicting genetic values of lines from two maize populations evaluated for different traits measured in different environments (experiments 1 and 2). Given that GBS data come with a large percentage of uncalled genotypes, we evaluated methods using nonimputed, imputed, and GBS-inferred haplotypes of different lengths (short or long). GBS and pedigree data were incorporated into statistical models using either the genomic best linear unbiased predictors (GBLUP) or the reproducing kernel Hilbert spaces (RKHS) regressions, and prediction accuracy was quantified using cross-validation methods. The following results were found: relative to pedigree or marker-only models, there were consistent gains in prediction accuracy by combining pedigree and GBS data; there was increased predictive ability when using imputed or nonimputed GBS data over inferred haplotype in experiment 1, or nonimputed GBS and information-based imputed short and long haplotypes, as compared to the other methods in experiment 2; the level of prediction accuracy achieved using GBS data in experiment 2 is comparable to those reported by previous authors who analyzed this data set using SNP arrays; and GBLUP and RKHS models with pedigree with nonimputed and imputed GBS data provided the best prediction correlations for the three traits in experiment 1, whereas for experiment 2 RKHS provided slightly better prediction than GBLUP for drought-stressed environments, and both models provided similar predictions in well-watered environments.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">shared data resources</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing-0" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing" rel="nofollow">genotyping by sequencing</a></li><li class="kwd"><a href="/keyword/genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-genomic-selection" rel="nofollow">genomic selection</a></li><li class="kwd"><a href="/keyword/imputation" class="hw-term hw-article-keyword hw-article-keyword-imputation" rel="nofollow">imputation</a></li><li class="kwd"><a href="/keyword/gblup" class="hw-term hw-article-keyword hw-article-keyword-gblup" rel="nofollow">GBLUP</a></li><li class="kwd"><a href="/keyword/rkhs" class="hw-term hw-article-keyword hw-article-keyword-rkhs" rel="nofollow">RKHS</a></li></ul><p id="p-3">Implementation of genomic selection (GS) in plant and animal breeding programs is usually based on genotypes derived from marker panels discovered using reference samples. The rapid progress of next-generation DNA sequencing (NGS) technologies, especially those based on short read sequencing of a fractional genome representation, have allowed direct and inexpensive single nucleotide polymorphism (SNP) detection from large and diverse germplasm collections. Genotyping-by-sequencing (GBS) is an NGS approach that reduces genome complexity via restriction enzymes (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Elshire <em>et al.</em> 2011</a>). GBS has been applied to study trait association in a diverse seed bank collection of maize germplasm (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Romay <em>et al.</em> 2013</a>) to identify agriculturally important but uncommon alleles (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Yan <em>et al.</em> 2010</a>). GBS has also been used to construct GS models for large and complex polyploidy wheat breeding materials (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Poland <em>et al.</em> 2012</a>).</p><p id="p-4">One biological complication of applying GBS in a maize breeding program is that maize possesses a dynamic genome with extensive presence–absence variation (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fu and Dooner 2002</a>), which results in many regions of the genome not being imputable for a given SNP. Recent studies suggest 80–90% of the genome shows some presence–absence variation (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chia <em>et al.</em> 2012</a>). High-depth coverage studies using GBS suggest that only 75–82% of the sites are present in these samples, except for the reference genome B73, which exhibits near-complete coverage.</p><p id="p-5">GBS data are highly dimensional and usually come with a large percentage of uncalled genotypes; therefore, incorporating this information into models poses important statistical and computational challenges that need to be addressed but have not yet been studied in detail. First, with low sequencing coverage, the proportion of “missing” marker genotypes can be high (∼70–80% or more for 1× sequencing coverage; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Poland <em>et al.</em> 2012</a>), and it is not clear how massive amounts of GBS information can be incorporated into statistical models for prediction of performance of unobserved genotypes (as required in GS). Second, it is unknown how GBS data information can be combined with pedigree information in model prediction. Third, until now, it is not known how imputation of missing genotype data or haplotype inferences may affect prediction accuracies when GBS data are used.</p><p id="p-6">Haplotypes can be used to impute missing data; to perform this imputation, only a few individuals need to carry a given haplotype in the data set. However, many more copies of a given haplotype are required for an accurate statistical estimation of its effect. Longer haplotypes may be more accurately determined, but by nature they are carried by fewer individuals, especially when the sample size is small. Longer haplotypes can directly parameterize presence–absence variants or other types of QTL alleles nested within haplotype alleles, but SNP markers need to parameterize them indirectly through linkage disequilibrium (LD). It is not known how the imputation and prediction with different haplotypes lengths affect genomic prediction.</p><p id="p-7">Most GS research so far has focused on evaluation based on low-to-intermediate marker panels with a low-to-intermediate proportion of missing markers (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">de los Campos <em>et al.</em> 2009</a>, <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">2010</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> 2010</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">2011</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Pérez <em>et al.</em> 2010</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">2012</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">González-Camacho <em>et al.</em> 2012</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hickey <em>et al.</em> 2012</a>). GBS has attracted great interest for the application of GS in plant breeding. <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Poland <em>et al.</em> (2012)</a> recently reported that they successfully applied GBS to a large set of advanced wheat lines from the semi-arid wheat breeding program of CIMMYT and predicted breeding values with 0.28 to 0.45 prediction accuracy for grain yield (GY), an improvement of 0.1 to 0.2 over another marker platform. The authors concluded that its low cost makes GBS a good method for GS. The extensive LD of most wheat populations probably aids the accuracy of imputation of missing genotype data. However, the accuracy of genomic predictions based on imputed data may be quite different in other species, such as maize, with much less extensive LD. Although GBS appears promising as a genotyping method to use in GS, currently, no studies have been performed that evaluate the use of GBS data for prediction accuracy in diverse maize breeding populations.</p><p id="p-8">Therefore, the main objective of this research was to quantify the accuracy of genomic prediction using GBS data in two maize data sets with different complex (or quantitative) traits evaluated under differing environmental conditions. Experiment 1 included three traits evaluated in four environments, as well as pedigree information regarding the maize lines. Experiment 2 included one trait and two environments, but no information regarding the pedigree of the maize lines. The accuracy of genomic predictions using GBS was assessed by examining the prediction accuracy of three different sources of GBS information: nonimputed GBS; imputed GBS; and GBS-inferred haplotypes. In experiment 1, pedigree data were available; therefore, in this data set, we were able to assess how much prediction accuracy can be gained by combining pedigree and GBS data, relative to family-based predictions. Experiment 2 has been analyzed by other authors using SNP arrays; therefore, in this experiment, we were able to compare the prediction accuracy that can be attained with GBS data relative to previous reports using SNP arrays.</p><p id="p-9">In one dimension of this study, the predictions of the different sources of GBS information are compared. In another dimension, we examine the prediction accuracy of two statistical models: genomic best linear unbiased predictor (GBLUP) and reproducing kernel Hilbert spaces (RKHS) for all three types of GBS information. The last dimension evaluated in this study, which applies only to experiment 1 (with a family structure), is concerned with the prediction accuracy of GBLUP and RKHS including or ignoring pedigree information in combination with the three types of GBS information. A default GBS-inferred haplotype length consisting of 1000 flanking markers on each side of the marker being imputed was used for both experiments. For experiment 2 (with a smaller sample size than experiment 1), we used two other haplotype lengths, short and long, with 50 and 100 flanking markers on each side of the marker being imputed, respectively.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-10">Two compressed files containing phenotypic and genotypic information regarding the two data sets used in this study [experiment 1 with 504 maize doubled haploid (DH) lines and experiment 2 with 296 maize lines] can be downloaded from <a href="http://repository.cimmyt.org/xmlui/handle/10883/1380">http://repository.cimmyt.org/xmlui/handle/10883/1380</a>.</p><div id="sec-2" class="subsection"><h3>Phenotypic data</h3><p id="p-11">The phenotypic data set consisted of two different maize data sets, experiment 1 and experiment 2. The data set from experiment 1 was a collection of 504 DH lines derived from crossing and backcrossing eight inbred parents to form 10 full-sib families. The 504 DH were crossed to a single-cross tester. Basic information regarding the parents in experiment 1 is provided in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table A1</a> (see Appendix A). Data from experiment 2 consisted of a diverse panel of 296 CIMMYT maize inbred lines, including several breeding populations.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463555/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463555/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table A1</span> <span class="caption-title">Number of doubled-haploid lines in each cross (P1×P2) and backcross, and pedigree of the eight parents (A, B, C, D, E, F, G, and H) included in experiment 1</span><div class="sb-div caption-clear"></div></div></div><div id="sec-3" class="subsection"><h4>Phenotypic data for 504 DH maize lines (experiment 1):</h4><p id="p-12">A total of 504 DH maize lines were obtained by crossing and backcrossing eight parents to form 10 full-sib families of different sizes. Each DH line was crossed to an elite single-cross hybrid from the opposite heterotic group to produce 504 test-crosses that were genotyped with GBS. Traits available in this data set included GY (kg/ha), anthesis date (AD; days after sowing), and anthesis-silking interval (ASI; days); each of these traits were evaluated in four environments, three optimum rain-fed trials and one managed drought trial. The experimental design in each of the four environments was an alpha-lattice incomplete block design with two replicates.</p><p id="p-13">Although individual trial prediction analyses were performed for the three optimum and one drought environment, here we present results for the four trials combined. Data were balanced across the four trials and pre-adjusted using estimates of block and environment effects derived from a linear model that accounted for the incomplete block design within environment and for environment effects. Given that these data had a family structure, pedigree information was incorporated into the prediction in two forms, as a pedigree model <em>per se</em> and by including it in the GBLUP and RKHS models.</p><p id="p-14">Broad-sense heritability of the combined analyses across environments was calculated as<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-1.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-2.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-3.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-4.gif"/></noscript></span></span> are the genotype, genotype × environment interaction, and error variance components, respectively, and <em>e</em> and <em>r</em> are the number of environments and replicates within each environment included in the corresponding analyses, respectively.</p></div><div id="sec-4" class="subsection"><h4>Phenotypic data for the 296 inbred lines (experiment 2):</h4><p id="p-15">These lines represent a wide range of germplasm from the global maize breeding program of CIMMYT and were derived from different breeding subpopulations. The two trait combinations considered in this study were GY (kg/ha) evaluated in four severe drought stress (SS) trials and five well-watered (WW; ample rainfall) trials. All 296 lines were included in all the trials and pre-adjusted based on estimates of block and plot effects derived from the linear model, as described for data in experiment 1; broad-sense heritability of the combined analyses across environments was calculated as explained in experiment 1. This data set was also used by <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> (2010)</a>, <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">González-Camacho <em>et al.</em> (2012)</a>, and <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Windhausen <em>et al.</em> (2012)</a>.</p></div></div><div id="sec-5" class="subsection"><h3>Genotypic data</h3><div id="sec-6" class="subsection"><h4>Genotyping-by-sequencing:</h4><p id="p-16">GBS is a reduced representation approach that uses restriction enzymes to sample the genome. In this study, we adopted a GBS protocol commonly used by the maize research community (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Elshire <em>et al.</em> 2011</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Romay <em>et al.</em> 2013</a>). DNA was digested with <em>Ape</em>KI restriction enzyme and 96 samples were multiplexed per Illumina flow cell for sequencing.</p><p id="p-17">To increase the genome coverage and read depth for SNP discovery, raw read data from sequencing the samples were analyzed together with 19,000 additional maize samples from the CIMMYT global maize breeding program, USA and Chinese NAM populations, and the USDA Ames maize collection and teosinte mapping populations. SNP identification was performed using TASSEL 3.0 GBS Discovery Pipeline with B73 as the reference genome (<a href="http://www.maizegenetics.net/tassel/docs/TasselPipelineGBS.pdf">http://www.maizegenetics.net/tassel/docs/TasselPipelineGBS.pdf</a>). Briefly, raw Illumina DNA sequence data (∼100 base pair qseq file) were first trimmed to remove bar codes, and further trimmed or padded with “A's on the 3′ end to 64 base pair lengths. First, the SNP calling process was initiated by identifying all distinct 64-base pair tags occurring in more than 0.1% of the overall maize collection (∼20,000 lines). Second, the distribution of these tags across all <em>Zea</em> was then generated, and tags were aligned against the reference genome sequence of B73 using the Burrows-Wheeler aligner (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Li and Durbin 2009</a>). Third, all tags aligning in the same region of the genome are aligned with the full multiple sequence alignment of Biojava3 (<a href="http://biojava.org/">http://biojava.org/</a>) (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Holland <em>et al.</em> 2008</a>). Finally, a SNP was defined for every maize sample based on a binomial distribution from the aligned unique tags.</p><p id="p-18">Because of extensive paralogy in maize, SNPs were then further filtered based on homozygosity in inbred lines, the biparental error correction plug-in in Tassel GBS Discovery Pipeline, which was applied to eliminate SNP with high errors rates (−m × E = 0.01), and a minimum median <em>r<sup>2</sup></em> (parameter: −mnPLD = 0.5) for LD with markers in the local genome region across biparental families. And, finally, only those that segregate in the mapping populations were used in this study. Source code and the Tassel GBS discovery pipeline are available at <a href="http://www.maizegenetics.net">www.maizegenetics.net</a> and SourceForge Tassel project (<a href="http://sourceforge.net/projects/tassel/">http://sourceforge.net/projects/tassel/</a>). Tassel was also used for computing pairwise measures of LD (<em>r</em><sup>2</sup>) between loci for each data set (experiments 1 and 2) and for each chromosome within the data sets (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bradbury <em>et al.</em> 2007</a>).</p></div></div><div id="sec-7" class="subsection"><h3>Imputation and haplotype methods</h3><p id="p-19">A simple imputation algorithm was developed for the purpose of imputing the markers that were missing because of technical limitations with GBS data (not biologically missing). The imputation algorithm borrowed ideas related to haplotype library imputation (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Hickey <em>et al.</em> 2012</a>). In the variant of the algorithm used in this study, imputation works by first identifying individuals who are likely to share a common haplotype. Then, the genotype information from these individuals is combined to form a consensus haplotype. Finally, any individuals who share this haplotype have their missing markers imputed on the basis of this consensus haplotype.</p><p id="p-20">The individuals used in this study were inbred lines and therefore were homozygous for the majority of markers. The imputation procedure began by first setting to missing any residual heterozygote markers. Next, diploid genotype data were formally phased based on the fact that phase could be directly observed, because markers not originally missing or set to missing were phased <em>de facto</em> by virtue of the genotypes being homozygous.</p><p id="p-21">Imputation was performed at each marker using information from flanking markers, both called and missing, to define the haplotype allele that each individual carried. At each marker, the genotyped individuals in the data set were partitioned into clusters, and individuals in the same cluster were assumed to carry the same haplotype because they had no opposing homozygote loci. This was performed by searching a certain number of flanking markers on either side of the marker of interest for called alleles that disagreed between pairs of individuals. Pairs of individuals with more than a threshold with four disagreeing alleles were assumed not to belong to the same cluster. Individuals belonged to only one cluster, and the minimum number of individuals in a cluster was one. Individuals in the same cluster were assumed to carry the same haplotype and, therefore, the same allele at the marker of interest. If some individuals in the cluster were missing this allele, but others were not, then it was assumed that the allele was technically missing (rather than biologically missing) in individuals in this cluster who were missing the allele. The missing allele was therefore imputed in those individuals in whom it was missing. If the allele was missing for all individuals in a cluster, then it was not imputed because it was assumed to be biologically missing for these individuals. The algorithm moved along each chromosome, moving the locus to be imputed and its flanking markers, one marker at a time.</p><div id="sec-8" class="subsection"><h4>Haplotype length:</h4><p id="p-22">The imputation algorithm that was performed at each marker uses information from flanking markers (that are called or missing) and defines the haplotype allele that each individual carries. Therefore, the algorithm imputes not only technically missing markers but also inferred haplotypes that can be used for genomic prediction. Explicitly, the inferred haplotypes are, in fact, the clusters to which each individual belongs.</p><p id="p-23">In the analysis of both data sets, the default imputation method used 1000 flanking markers on each side of the marker being imputed to infer the haplotype allele carried. However, because of the small size of the data set in experiment 2, the long default haplotypes would not be informative because too few individuals were present for enough individuals to share long haplotypes; therefore, two other parameter values for the imputation method were used to examine the sensitivity of haplotypes length, one using a short haplotype inference from 50 flanking markers on each side of the marker being imputed and another using a long haplotype inference from 100 flanking markers. The haplotype lengths were chosen as a compromise between computation time and the overall yield of imputed alleles in some cases and imputation accuracy in other cases. Shorter haplotypes take more time and generate a greater amount of imputed alleles. Longer haplotypes have a greater accuracy of imputation, but if they are too long they are shared by very few individuals and consequently contribute fewer imputed alleles.</p><p id="p-24">Clusters that were created and used to perform the imputation also contained information that could be used for genomic prediction directly. Individuals belonging to the same cluster at each marker position were assumed to carry the same haplotype at that marker position. Because there were several clusters at each marker position, there were also several haplotypes at each marker position. These clusters/haplotypes may be able to parameterize biologically missing data more optimally than partially imputed SNP. For example, an allele that is missing for all individuals in a cluster is not directly parameterized by SNP because all individuals in this cluster will be missing for this SNP. However, the cluster/haplotype alleles may be able to more directly parameterize this because all individuals carry one of the haplotype alleles, and the biologically missing variant may be nested within only one or a few haplotype alleles. Longer haplotypes may be more precise in their definition of the haplotype allele carried by an individual; however, by virtue of their length, long haplotypes are not shared by as many individuals as short haplotypes. Therefore, because more copies of each haplotype allele are present in the data for short haplotypes, short haplotypes may be more powerful for genomic prediction.</p><p id="p-25">In essence, there are multiple tradeoffs involved. Longer haplotypes are more precisely defined, but alleles may not be shared by many individuals and therefore have reduced power for estimating their effects. Furthermore, long haplotypes may have greater ability to capture nonadditive genetic effects or the effects of recent mutations that are more likely to be shared by very close relatives. Short haplotypes and SNPs may have greater ability to parameterize additive effects caused by older mutations.</p></div></div><div id="sec-9" class="subsection"><h3>Statistical models</h3><p id="p-26">In an applied breeding program, GS could be applied for two purposes: predicting breeding values of individuals for rapid selection cycling or predicting genotypic values of advanced lines that are in the last stages of testing. Although predicting breeding values necessitates models for estimating additive effects, predicting the genetic values of advanced lines requires models that account for additive as well as nonadditive (<em>i.e.</em>, epistatic) genetic effects (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em> 2013</a>). In general, linear models such as GBLUP capture additive relationships, whereas nonlinear models such as RKHS can account for nonadditive genetic components (<em>e.g.</em>, gene × gene effects) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gianola <em>et al.</em> 2006</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2010</a>; <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">González-Camacho <em>et al.</em> 2012</a>).</p><p id="p-27">Genomic prediction (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Meuwissen <em>et al.</em> 2001</a>) can be implemented using parametric or semi-parametric procedures. Among the parametric methods, GBLUP (a ridge-regression–type estimator) is the most commonly used; among the semi-parametric procedures, RKHS is the one used most frequently (<em>e.g.</em>, <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">de los Campos <em>et al.</em> 2012</a>) and has consistently shown high prediction accuracy (<em>e.g.</em>, <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2010</a>; <a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> 2010</a>, <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">2011</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Heslot <em>et al.</em> 2012</a>; <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">González-Camacho <em>et al.</em> 2012</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Pérez <em>et al.</em> 2012</a>). In this study, we considered using either GBLUP or RKHS regressions.</p><p id="p-28">The GBLUP method uses a genomic relationship matrix computed using imputed or nonimputed marker information. Furthermore, the genomic relationship matrix can also be computed on a SNP-by-SNP basis or on the basis of haplotypes spanning multiple adjacent SNP (see Appendix A for further details). Haplotypes can be used instead of imputed GBS markers when constructing genomic relationship matrices for GBLUP or the reproducing kernel for RKHS regressions (further details are provided in Appendix A).</p><p id="p-29">The RKHS implementation applied here uses a Gaussian kernel evaluated on Euclidean distances computed from either imputed or nonimputed GBS data or from SNP <em>vs.</em> haplotype information. We implemented the RKHS regression using a multi-kernel approach, termed kernel averaging (KA), described by <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> (2010)</a>. Further details regarding our implementation are given in Appendix A. For the data set with available pedigree information (504 DH lines), we evaluated the GBLUP and RKHS models using genomic information and a combination of genomic and pedigree information.</p><p id="p-30">We implemented GBLUP and RKHS using the Bayesian framework by assigning appropriate prior probability distributions to marker effects (see Appendix A). The posterior distributions of all unknowns in both models were obtained using MCMC techniques. We used the Bayesian implementation of RKHS described by <a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> (2010)</a> with the kernels and prior probability distributions described in <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">González-Camacho <em>et al.</em> (2012)</a>.</p><div id="sec-10" class="subsection"><h4>Notation of the different models:</h4><p id="p-31">Concerning the notation of the models, we appended to either GBLUP or RKHS subscripts that denote the imputation and haplotype methods used. No subscripts were used for nonimputed GBS data. For experiments 1 and 2 data sets, the default marker imputation method (1000 flanking markers) is denoted by the subscript IM (GBLUP<sub>IM</sub>, RKHS<sub>IM</sub>); for the haplotype-based method, we used H as the subscript (GBLUP<sub>H</sub>, RKHS<sub>H</sub>). For experiment 1, the pedigree model denoted as P is fitted as described in Appendix A and when combined pedigree and marker information is used with nonimputed GBS, the notation is PGBLUP and PRKHS. When imputed (IM) GBS and haplotype-based (H) GBS methods are used, the notations are PGBLUP<sub>IM</sub>, PRKHS<sub>IM</sub>, PRKHS<sub>H</sub>, and PRKHS<sub>H</sub>, respectively.</p><p id="p-32">For experiment 2, when imputed marker data are from short haplotypes (50 flanking markers; IMS) and long haplotypes (100 flanking markers; IML), the models are denoted as GBLUP<sub>IMS</sub>, GBLUP<sub>IML</sub>, RKHS<sub>IMS</sub>, and RKHS<sub>IML</sub>, respectively, and the models that use the inferred short haplotypes (HS) and long haplotypes (HL) are named GBLUP<sub>HS</sub>, GBLUP<sub>HL</sub>, RKHS<sub>HS</sub>, and RKHS<sub>HL</sub>, respectively. Further details regarding the computation of GBLUP and RKHS are provided in Appendix A.</p></div><div id="sec-11" class="subsection"><h4>Cross validation scheme:</h4><p id="p-33">We quantified prediction accuracy using a replicated training–testing evaluation. In each replicate, data were randomly partitioned into training (TRN; with 70% of the observations) and testing (TST; with 30% of data points) data sets. Models were fitted to the TRN data set and prediction accuracy was evaluated in the TST data set. This was replicated 50 times with independent random assignments into TRN and TST; in this manner, we obtained 50 estimates of the prediction accuracy (one per replicate) for each of the methods evaluated. Prediction ability was quantified using the correlation between predicted and observed values in each of the TST data sets. A scheme similar to this was used by <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gianola <em>et al.</em> (2011)</a> and <a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">González-Camacho <em>et al.</em> (2012)</a>; one advantage of this approach is that in addition to yielding a point estimate of prediction accuracy, it provides measures of uncertainty of such estimates (<em>e.g.</em>, variance of predictive correlation across partitions of the data into TRN and TST data sets).</p></div></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><p id="p-34">For each data set, results are presented in relation to general information on the GBS data as well as information on GBS data in each experiment data set (number of missing SNPs by chromosome and distribution of minor allele frequency), and in relation to the prediction accuracy of the two statistical models (GBLUP and RKHS) on the three different types of GBS information (imputed, nonimputed, and haplotype method). For experiment 1, results of the prediction accuracy of GBS for models with and without pedigree are examined.</p><div id="sec-13" class="subsection"><h3>GBS data</h3><p id="p-35">Obtained from a large collection of maize samples (∼20,000 lines), approximately 66% of the unique reads mapped to unique positions in the reference genome; if mapped uniquely, then 80% of the 64 base pair reads carried at least one polymorphic site among all varieties. This is the result of sampling a large collection of diverse material, in which 45% of the SNP sites are polymorphic, plus more from small indels; the rest of the SNPs (∼55%) are classified as rare alleles (with very low allele frequency). On average, 2.09 billion single base pair reads can be collected in a single flow cell. At the plex level of 96, the coverage of GBS SNPs in this study is approximately at 1×, given the averaged raw read counts for our maize samples being 2.3 million reads. However, sequence read depth varies greatly across the genome, especially when samples are run from 1× to 2× coverage (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Beissinger <em>et al.</em> 2013</a>); genome coverage also varies by taxa and SNP sites, and thus it has been complicated to systematically describe the read depth variation along the genome and across taxa. At 1×, the Poisson distribution expects 36.8% of missing information. Our data in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 1</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 2</a> suggest GBS has a higher than expected missing data ratio.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463584/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463584/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Experiment 1 (504 doubled-haploid maize lines), initial number of markers per chromosome, number of filtered markers per chromosome, and percentage of missing cells per chromosome after imputation</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463610/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463610/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Experiment 2 (296 maize lines), initial number of markers, number of filtered markers per chromosome, percentage of missing cells in the default imputation method, imputation method for short haplotypes, and imputation method for long haplotypes</span><div class="sb-div caption-clear"></div></div></div><p id="p-38">Based on the repeated samples in this large maize collection (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Romay <em>et al.</em> 2013</a>), the error rate per base was estimated at 0.0018 based on repeated samples, and the average SNP call rate (per sample) was 42%, with values ranging from 2% to 75%. However, when analyzing taxa that have deeper coverage, an average of 16–23% of the sites are missing from any given sample, suggesting a substantial amount of missing data are attributable to the biology of presence/absence variation in the maize genome (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Springer <em>et al.</em> 2009</a>).</p><p id="p-39">Information on genotypic data for each maize chromosome, including the number of original markers, the percentage of missing cells (where cell refers to the value of a locus in an individual line) in the original marker data, the number of markers after initial filtering, and the number of markers after imputation, is summarized in <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 1</a> and <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 2</a> for experiments 1 and 2, respectively. Filtering was performed by first removing markers that had more than 80% of lines with missing values and then deleting markers with MAF ≤0.05. Overall, before filtering, we obtained a total of 681,257 SNP directly from the GBS calling pipeline; after filtering for missing values and minor allele frequency, 158,281 SNPs were used in experiment 1 (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 1</a>) and 235,265 SNPs were used in experiment 2 (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 2</a>).</p><p id="p-40">For experiment 1 (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 1</a>), the initial GBS data before filtering had a percentage of missing cells per chromosome ranging from 51.3 to 52.8%; after filtering, this percentage decreased to approximately 43–44% of the total number of cells. Approximately 20% of cells were missing in the edited GBS information used for prediction after imputation (last column in <a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 1</a>). These 20% missing genotypes were replaced by their expected values before performing the prediction. The distribution of minor allele frequency is depicted in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>. Regarding GBS experiment 2 (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 2</a>), the initial GBS markers for this data set had approximately 57–58% of missing cells. The percentage of missing cells in the edited GBS data was 51–52% (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 2</a>), and minor allele frequency distribution is shown in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Distribution of minor allele frequency (MAF) for experiment 1. (B) Distribution of minor allele frequency for experiment 2. Data used is non-imputed genotyping-by-sequencing (GBS)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-416123368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;(A) Distribution of minor allele frequency (MAF) for experiment 1. (B) Distribution of minor allele frequency for experiment 2. Data used is non-imputed genotyping-by-sequencing (GBS).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1903/F1.medium.gif" width="440" height="201"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/11/1903/F1.medium.gif" width="440" height="201"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463541" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-41" class="first-child">(A) Distribution of minor allele frequency (MAF) for experiment 1. (B) Distribution of minor allele frequency for experiment 2. Data used is non-imputed genotyping-by-sequencing (GBS).</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">Measures of LD at various genetic distances by data set and chromosome were computed. Plots of <em>r</em><sup>2</sup> <em>vs.</em> distance by chromosome and data set are depicted in Appendix B (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure B1</a>, a and b; <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure B2</a>, a and b; <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure B3</a>, a and b; <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure B4</a>, a and b; <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure B5</a>, a and b; <a id="xref-fig-10-1" class="xref-fig" href="#F10">Figure B6</a>, a and b; <a id="xref-fig-11-1" class="xref-fig" href="#F11">Figure B7</a>, a and b; <a id="xref-fig-12-1" class="xref-fig" href="#F12">Figure B8</a>, a and b; <a id="xref-fig-13-1" class="xref-fig" href="#F13">Figure B9</a>, a and b; and <a id="xref-fig-14-1" class="xref-fig" href="#F14">Figure B10</a>, a and b). The <em>r</em><sup>2</sup> between adjacent markers decreased very quickly in experiment 2, as expected for maize, and the median <em>r</em><sup>2</sup> achieved very low values at distances of 0.5 Mb or longer. The patterns of LD in experiment 1 were very different; here, the average <em>r</em><sup>2</sup> remained relatively high (values of ∼0.2) even at very long distances, and there was great deal of variability in <em>r</em><sup>2</sup> even at long distances. This occurs because the association of alleles in this data set is largely driven by family linkage, whereas in experiment 2 the association patterns of alleles are dominated by population LD.</p></div><div id="sec-14" class="subsection"><h3>Imputation and haplotype</h3><div id="sec-15" class="subsection"><h4>Imputation: GBS experiment 1:</h4><p id="p-43">For this data set, the default imputation method had 1000 flanking markers for deriving the haplotypes, creating several hundred thousand alleles per chromosome (<em>i.e.</em>, one SNP can be in different haplotypes and this is considered an allele). The information in <a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 1</a> shows different information on markers and missing cells (where cell refers to the value of a locus in an individual line).</p></div><div id="sec-16" class="subsection"><h4>Imputation: GBS experiment 2:</h4><p id="p-44">As previously mentioned for experiment 2, in data sets with a smaller sample size than in experiment 1, the LD is likely to be lower than that in the experiment 1 data set, thus making the long default haplotypes (with 1000 flanking markers) much less informative. Therefore, for this data set, we tested the sensitivity of shortening the flanking markers by performing imputation with 100 flanking markers [long haplotype imputation method (IML) that produces long haplotypes (LH)] and with 50 flanking markers [short haplotype imputation method (IMS) that produces short haplotypes (SH)] and studied the prediction accuracy of these cases as well. The percentage of missing cells in the default imputation method was approximately 48–49%, whereas the imputation method using 50 flanking markers (SH) produced approximately 24–25% of missing cells, and the imputation method using 100 flanking markers (LH) had approximately 32–33% of the total number of missing cells replaced by their expected values before performing the prediction.</p></div></div><div id="sec-17" class="subsection"><h3>Prediction accuracy</h3><div id="sec-18" class="subsection"><h4>Prediction: experiment 1:</h4><p id="p-45">Estimates of broad-sense heritability and estimates of prediction accuracy for this experiment are provided in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 3</a>. Correlations for the pedigree (P) model were 0.518, 0.589, and 0.588 for GY, AD, and ASI, respectively. In general, GBLUP models showed lower predictive correlations than the RKHS regression models for the three traits. However, for all traits, PGBLUP and PRKHS had very similar prediction accuracies and showed consistently better predictive correlations than their corresponding counterparts without pedigree, GBLUP and RKHS. In several instances, GBLUP and RKHS showed lower prediction correlations than the pedigree model (<em>i.e.</em>, RKHS<sub>H</sub> for GY and ASI, and all the GBLUPs for GY and ASI). For trait AD, the increase in prediction accuracy of all models with GBS information is remarkable as compared with the pedigree model.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463543/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463543/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Experiment 1 (504 doubled-haploid maize lines), estimated broad-sense heritability and average predictive ability between observed and predicted values for grain yield, days to anthesis, and anthesis-silking interval for 13 models and their percent change relative to the pedigree model</span><div class="sb-div caption-clear"></div></div></div><p id="p-47">In general, SEs of prediction for GY and ASI were approximately 0.04–0.05, whereas for AD they were approximately 0.03–0.04. Exceptions are the SEs of predictions for models fitted using inferred haplotypes, which are consistently high for all traits. Across imputation methods, the correlations of the models that used inferred haplotypes (H) gave the lowest prediction accuracy across methods, with GBS information, traits, and models (GBLUP<sub>H</sub>, PGBLUP<sub>H</sub>, RKHS<sub>H</sub>, PRKHS<sub>H</sub>) showing, as already mentioned, the largest SEs. Several correlations values are within the SE interval; therefore, it is adequate for describing patterns and trends rather than significances.</p><p id="p-48">These results indicated, at least for this data set, the limited accuracy of using inferred haplotypes with GBS information. However, prediction correlation on imputed GBS based on 1000 flanking markers was similar to nonimputed GBS in several, but not all, cases. For example, PRKHS, PRKHS<sub>IM</sub>, PGBLUP, and PGBLUP<sub>IM</sub> models yielded the best predictive correlations, with gains in predictive correlation relative to the pedigree (P) model of approximately 12% for GY and approximately 23% for AD. Models PGBLUP and PGBLUP<sub>IM</sub> had the best predictive correlations for ASI, with gains relative to the P model of 5.9% and 5.4%, respectively (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 3</a>).</p><p id="p-49"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> provides a scatter plot of the 50 correlations obtained with PRKHS<sub>IM</sub> <em>vs.</em> those of the P model. A point above the 45-degree line implies that for that particular partition, the predictive correlation of the model on the vertical axis (PRKHS<sub>IM</sub>) is higher than that of the model on the horizontal axis (P). Although there is great variability in the correlations, the superior prediction accuracy of PRKHS<sub>IM</sub> over the pedigree model is consistent across most partitions (<em>i.e.</em>, partitions that gave correlations of &lt;0.5, those with correlations between 0.50 and 0.60, and those that produced correlations &gt;0.6).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plots of the predictive correlation for each of 50 cross-validation partitions for grain yield (GY) in experiment 1. When the best model is pedigree reproducing kernel Hilbert spaces regression with imputed genotyping-by-sequencing (PRKHSIM), this is represented by &#x25CB;; when the best model is pedigree (P), this is represented by &#x25CF;. The histograms depict the distribution of the correlations in the testing set obtained from 50 partitions for two models. The horizontal and vertical dashed lines represent the averages of the correlations for the testing sets in the 50 cross-validation partitions for the models shown in the Y axis and in the X axis, respectively. The solid line represents Y = X (when both models have the same prediction ability)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-416123368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plots of the predictive correlation for each of 50 cross-validation partitions for grain yield (GY) in experiment 1. When the best model is pedigree reproducing kernel Hilbert spaces regression with imputed genotyping-by-sequencing (PRKHS&lt;sub&gt;IM&lt;/sub&gt;), this is represented by &#x25CB;; when the best model is pedigree (P), this is represented by &#x25CF;. The histograms depict the distribution of the correlations in the testing set obtained from 50 partitions for two models. The horizontal and vertical dashed lines represent the averages of the correlations for the testing sets in the 50 cross-validation partitions for the models shown in the Y axis and in the X axis, respectively. The solid line represents Y = X (when both models have the same prediction ability).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1903/F2.medium.gif" width="440" height="408"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/11/1903/F2.medium.gif" width="440" height="408"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463572" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-50" class="first-child">Plots of the predictive correlation for each of 50 cross-validation partitions for grain yield (GY) in experiment 1. When the best model is pedigree reproducing kernel Hilbert spaces regression with imputed genotyping-by-sequencing (PRKHS<sub>IM</sub>), this is represented by ○; when the best model is pedigree (P), this is represented by ●. The histograms depict the distribution of the correlations in the testing set obtained from 50 partitions for two models. The horizontal and vertical dashed lines represent the averages of the correlations for the testing sets in the 50 cross-validation partitions for the models shown in the Y axis and in the X axis, respectively. The solid line represents Y = X (when both models have the same prediction ability).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h4>Prediction in experiment 2:</h4><p id="p-51">Estimates of broad-sense heritability and of prediction accuracy for this experiment are given in <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 4</a>. As expected, estimates of broad-sense heritability and prediction accuracy were lower under drought stress conditions with a larger SE of the predictive correlation (0.088–0.107) than those obtained under WW (ample rainfall) conditions (0.057–0.071). In general, the correlation values are within the SE boundaries; therefore, we discuss prediction trends within the context of nonsignificant differences.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463623/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463623/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Experiment 2 (296 maize lines), estimated broad-sense heritability and estimated average predictive ability between observed and predicted values for grain yield under drought stress and grain yield under well-watered conditions for 14 models</span><div class="sb-div caption-clear"></div></div></div><p id="p-53">For GY-SS, the prediction accuracy of GBLUP was consistently lower than that of RKHS models across all imputation methods, and it displayed larger SEs. Prediction differences within GBLUP and RKHS for the different imputation methods were very small. The best predictive models for GY-SS were for the imputed GBS models RKHS<sub>IML</sub> and RKHS<sub>IMS</sub>. However, for GY under optimum conditions (GY-WW), prediction differences within GBLUPs and RKHSs for the different imputation methods were larger than for GY-SS. Models RKHS (with nonimputed GBS) and imputed GBS model RKHS<sub>IM</sub> were the best predictors, closely followed by GBLUP<sub>IMS</sub> and GBLUP<sub>IML</sub>.</p><p id="p-54">The variability of estimates of predictive correlation across data partitioned into training and testing sets was particularly high under severe stress (GY-SS), reflecting the lower precision obtained in field experiments under severe drought (also reflected in the lower broad-sense heritability and lower prediction accuracy). This dispersion of the actual differences in correlations between RKHS<sub>IMS</sub> <em>vs.</em> GBLUP<sub>IMS</sub> is shown in the scatter plots of the 50 random cross-validations in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. Most variability is attributable to the different partitions, and the prediction for RKHS<sub>IMS</sub> surpassed those from GBLUP<sub>IMS</sub> for most of the partitions. Results indicated that some random partitions picked subsets of training data that had lower correlations with observed values in the validation set (≤0.3), whereas other random partitions had closer relationships between the training and validation sets (≥0.5).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plots of the predictive correlation for each of 50 cross-validation partitions for grain yield (GY) in the severe drought environment (SS; GY-SS) of experiment 2. When the best model is reproducing kernel Hilbert spaces regression with genotyping-by-sequencing (GBS) imputation from short haplotypes (RKHSIMS), this is represented by &#x25CB;; when the best model is genomic best linear unbiased predictors with imputed GBS from short haplotypes (GBLUPIMS), this is represented by &#x25CF;. The histograms depict the distribution of the correlations in the testing set obtained from the 50 cross-validation partitions for two models. The horizontal and vertical dashed lines represent the averages of the correlations for the testing sets in the 50 cross-validation partitions for the models shown in the Y axis and in the X axis, respectively. The solid line represents Y = X (when both models have the same prediction ability)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-416123368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plots of the predictive correlation for each of 50 cross-validation partitions for grain yield (GY) in the severe drought environment (SS; GY-SS) of experiment 2. When the best model is reproducing kernel Hilbert spaces regression with genotyping-by-sequencing (GBS) imputation from short haplotypes (RKHS&lt;sub&gt;IMS&lt;/sub&gt;), this is represented by &#x25CB;; when the best model is genomic best linear unbiased predictors with imputed GBS from short haplotypes (GBLUP&lt;sub&gt;IMS&lt;/sub&gt;), this is represented by &#x25CF;. The histograms depict the distribution of the correlations in the testing set obtained from the 50 cross-validation partitions for two models. The horizontal and vertical dashed lines represent the averages of the correlations for the testing sets in the 50 cross-validation partitions for the models shown in the Y axis and in the X axis, respectively. The solid line represents Y = X (when both models have the same prediction ability).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1903/F3.medium.gif" width="440" height="432"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/11/1903/F3.medium.gif" width="440" height="432"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463599" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-55" class="first-child">Plots of the predictive correlation for each of 50 cross-validation partitions for grain yield (GY) in the severe drought environment (SS; GY-SS) of experiment 2. When the best model is reproducing kernel Hilbert spaces regression with genotyping-by-sequencing (GBS) imputation from short haplotypes (RKHS<sub>IMS</sub>), this is represented by ○; when the best model is genomic best linear unbiased predictors with imputed GBS from short haplotypes (GBLUP<sub>IMS</sub>), this is represented by ●. The histograms depict the distribution of the correlations in the testing set obtained from the 50 cross-validation partitions for two models. The horizontal and vertical dashed lines represent the averages of the correlations for the testing sets in the 50 cross-validation partitions for the models shown in the Y axis and in the X axis, respectively. The solid line represents Y = X (when both models have the same prediction ability).</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">It is interesting to examine the dispersion among predictive correlations between short and long haplotype imputation methods for GY-SS (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> depicts RKHS<sub>IMS</sub> <em>vs.</em> RKHS<sub>IML</sub>). As in <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 4</a>, results from <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> indicate that predictions of kernel models derived from 50 (short) and 100 (long) SNPs as flanking markers are very similar across all range of correlations (from &lt;0.3 to almost 0.7). Furthermore, similar results indicate not much difference in prediction ability between short and long haplotype inference when using the GBLUP<sub>IMS</sub> model for both GY-SS and GY-WW.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Plots of the predictive correlation for each of 50 cross-validation partitions for grain yield (GY) in the severe stress environment (SS; GY-SS) of experiment 2. When the best model is reproducing kernel Hilbert spaces regression with genotyping-by-sequencing (GBS) imputation from short haplotypes (RKHSIMS), this is represented by &#x25CF;; when the best model is reproducing kernel Hilbert spaces regression with GBS imputation from long haplotypes (RKHSIML), this is represented by &#x25CB;. The histograms depict the distribution of the correlations in the testing set obtained from the 50 cross-validation partitions for two models. The horizontal and vertical dashed lines represent the averages of the correlations for the testing sets in the 50 cross-validation partitions for the models shown in the Y axis and in the X axis, respectively. The solid line represents Y = X (when both models have the same prediction ability)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-416123368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Plots of the predictive correlation for each of 50 cross-validation partitions for grain yield (GY) in the severe stress environment (SS; GY-SS) of experiment 2. When the best model is reproducing kernel Hilbert spaces regression with genotyping-by-sequencing (GBS) imputation from short haplotypes (RKHS&lt;sub&gt;IMS&lt;/sub&gt;), this is represented by &#x25CF;; when the best model is reproducing kernel Hilbert spaces regression with GBS imputation from long haplotypes (RKHS&lt;sub&gt;IML&lt;/sub&gt;), this is represented by &#x25CB;. The histograms depict the distribution of the correlations in the testing set obtained from the 50 cross-validation partitions for two models. The horizontal and vertical dashed lines represent the averages of the correlations for the testing sets in the 50 cross-validation partitions for the models shown in the Y axis and in the X axis, respectively. The solid line represents Y = X (when both models have the same prediction ability).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1903/F4.medium.gif" width="440" height="368"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/11/1903/F4.medium.gif" width="440" height="368"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463615" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-57" class="first-child">Plots of the predictive correlation for each of 50 cross-validation partitions for grain yield (GY) in the severe stress environment (SS; GY-SS) of experiment 2. When the best model is reproducing kernel Hilbert spaces regression with genotyping-by-sequencing (GBS) imputation from short haplotypes (RKHS<sub>IMS</sub>), this is represented by ●; when the best model is reproducing kernel Hilbert spaces regression with GBS imputation from long haplotypes (RKHS<sub>IML</sub>), this is represented by ○. The histograms depict the distribution of the correlations in the testing set obtained from the 50 cross-validation partitions for two models. The horizontal and vertical dashed lines represent the averages of the correlations for the testing sets in the 50 cross-validation partitions for the models shown in the Y axis and in the X axis, respectively. The solid line represents Y = X (when both models have the same prediction ability).</p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-58">The overall aim of the study was to assess the prediction accuracy of GBS using two maize populations evaluated for different traits measured in different environments. Four specific questions were considered. How does the prediction accuracy of genomic models using GBS data compare with that of pedigree methods? Does the inclusion of pedigree information in the genomic model increase prediction accuracy? What is the impact of using imputation methods on the prediction accuracy attained by genomic models using GBS data? What is the prediction difference between the parametric GBLUP and the semi-parametric RKHS models?</p><p id="p-59">Using GBS, one can obtain large numbers of markers at a relatively low cost and with potentially less ascertainment bias than that present in commercially available SNP arrays. There are concerns about using GBS markers in general regarding the cost of informatics, the error rate, and missing data issues. However, as modern SNP discovery pipelines such as Tassel, GBS Discovery Pipeline, SAMtools (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Li <em>et al.</em> 2009</a>), and SNVer (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wei <em>et al.</em> 2011</a>) become available, many of the informatics requirements have been addressed for the users. <a id="xref-ref-29-4" class="xref-bibr" href="#ref-29">Romay <em>et al.</em> (2013)</a> compared GBS-SNP calls with Illumina array-based genotype values and estimated a mean discrepancy rate of 0.0118 between these two genotyping platforms. When analyzing maize inbred materials that are closely related to the B73 reference genome, such as our DH lines, excluding heterozygote calls further reduces the discrepancy rate to 0.0058, suggesting the reliability of GBS when reference genome information is available. However, interpreting missing data from GBS markers attributable to the low coverage might still be challenging for some applications, especially genome-wide association analysis and GS. Further, incorporating GBS data in genomic prediction is not free of hurdles and, to our knowledge, no empirical evaluation has yet been performed regarding the use of GBS genotypes for genomic-enabled prediction.</p><div id="sec-21" class="subsection"><h3>Dealing with missing genotypes in GBS data</h3><p id="p-60">Typically, GBS data contain high proportions of missing genotypes, which poses a real challenge. In this study, we proposed and evaluated two approaches to face that challenge. First, we proposed models based on called genotypes only circumventing imputation. Second, we developed and implemented several imputation methods for GBS data and evaluated the performance of prediction approaches using imputed data. Phasing was greatly facilitated by the fact that the data were from pure lines, given that the imputation of genotypes with large percentages of heterozygous loci is considerably more challenging.</p><p id="p-61">Results from experiment 1 indicated, for the three traits, a very slight increase in prediction (not significant) of models using imputed GBS as compared to those with uncalled GBS, but not for models with inferred haplotypes. Results from experiment 2 indicated no prediction improvement on the imputed GBS. The average gains in prediction accuracy varied greatly across traits and imputation methods, and in some cases methods that use imputed genotypes or inferred haplotypes were outperformed by methods based on called genotypes only. Clearly, if GBS is to become a common genotyping approach in breeding programs, then further research of developing and calibrating imputation methods for GBS data are needed.</p></div><div id="sec-22" class="subsection"><h3>Prediction accuracy using GBS data</h3><p id="p-62">Successful prediction accuracies in models using GBS data in wheat were recently reported (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Poland <em>et al.</em> 2012</a>). In this study, we provide the first extensive evaluation of genomic-enabled prediction using GBS on real maize phenotypic data. We considered two maize data sets: three traits were evaluated in experiment 1 and one trait was evaluated under three environmental conditions in experiment 2. Importantly, these two data sets have very different genetic structures and, therefore, represent very different prediction problems. Experiment 1 comprises full-sib and other related families, whereas experiment 2 includes a diverse panel of lines from the global maize breeding program of CIMMYT, some of which are distantly related.</p></div><div id="sec-23" class="subsection"><h3>Prediction accuracy with family data</h3><p id="p-63">In experiment 1, results show that combining GBS with pedigree data yields consistent gains in prediction accuracy, relative to pedigree-based predictions. These gains ranged from modest (2–5%) to substantial (10–24%), depending on the trait (regardless of whether GBS genotypes were imputed) and on the statistical method used to incorporate GBS data (either GBLUP or RKHS). Our results are in agreement with previous reports of consistent gains in prediction accuracy when pedigree was combined with genomic information derived from hybridization-based genotyping methods in maize and wheat (<em>e.g.</em>, <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">de los Campos <em>et al.</em> 2009</a>; <a id="xref-ref-6-4" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Burgueño <em>et al.</em> 2012</a>; <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Heslot <em>et al.</em> 2012</a>). The fact that combining GBS and pedigree data leads to higher prediction accuracy than using pedigree data alone indicates that GBS data capture information beyond what can be captured by the pedigree only, which in the case of experiment 1 reduces to the mean of the family. Unlike pedigree information, the use of marker data allows capturing within-family differences and, to some extent, bridging the signal of genetic similarity detected by genetic markers, which might have been overlooked by pedigree information.</p><p id="p-64">It should be mentioned that prediction from the pedigree GBLUP model operates not only within family means but also between families by capturing family structure. Furthermore, GBLUP and RKHS captured population structure and substructures (because of different parental relationships), as well as within and between family means, because regressing the phenotypic values on all marker values is equivalent to regressing phenotypes on all marker-derived principal components (<a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2010</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Janss <em>et al.</em> 2012</a>).</p><p id="p-65">Models using GBS data with no pedigree information were systematically outperformed by those combining pedigree and GBS data. Similar results were reported before using low-density markers in wheat data (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">de los Campos <em>et al.</em> 2009</a>; <a id="xref-ref-6-5" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> 2010</a>). However, other studies have shown that as marker density increases, the benefits of combining pedigree and genotype information diminish, relative to models using marker information only, until a point is reached at which, above a given marker density, combining pedigree and markers yields a prediction accuracy similar to the one obtained with markers only (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Vazquez <em>et al.</em> 2010</a>).</p><p id="p-66">Populations with extensive LD are also expected to gain high prediction accuracy because of the linkage between tagged SNPs with the unobserved casual genetic polymorphism (or QTL) (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Zhong <em>et al.</em> 2009</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Brito <em>et al.</em> 2011</a>). The maize genome reportedly suggests rapid LD decay in both tropical and temperate germplasm (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lu <em>et al.</em> 2011</a>; <a id="xref-ref-29-5" class="xref-bibr" href="#ref-29">Romay <em>et al.</em> 2013</a>), indicating a higher number of genetic markers are needed to arrive at the same prediction power. When applying genomic prediction models to closely related breeding materials (such as the families in experiment 1), in theory, considerable prediction accuracy can be expected because family linkages, low levels of diversity, small number of meiosis, and relatively large LD occurred in the genome.</p></div><div id="sec-24" class="subsection"><h3>Prediction accuracy in a diverse panel</h3><p id="p-67">Experiment 2 comprises a collection of maize lines with no close family relationships. However, these lines have different degrees of genetic similarity that could be described as a combination of population structure, substructure, and cryptic relationships, all of which are captured by markers and consequently accounted for in genomic regressions such as GBLUP (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Janss <em>et al.</em>, 2012</a>) or RKHS (<a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> 2010</a>).</p><p id="p-68">In this experiment, pedigree information is not available; therefore, comparing marker <em>vs.</em> pedigree prediction is not possible. However, our results can be compared with those of others who used this data set for genomic-enabled prediction using SNP arrays (<em>e.g.</em>, <a id="xref-ref-6-6" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">González-Camacho <em>et al.</em> 2012</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Windhausen <em>et al.</em> 2012</a>). In our study, prediction accuracy using GBS data for GY under WW conditions ranged from 0.544 to 0.624, depending on the model and depending on whether GBS data were imputed. <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Windhausen <em>et al.</em> (2012)</a> used 55,000 SNPs and experiment 2 data with the GBLUP model, and achieved a prediction accuracy of 0.50 for GY in optimum environments. <a id="xref-ref-6-7" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> (2010)</a>, using experiment 2 with only 1148 SNPs, reported predictions of approximately 0.51 for GY-WW for RKHS, Bayesian-LASSO, and GBLUP; however, <a id="xref-ref-16-8" class="xref-bibr" href="#ref-16">González-Camacho <em>et al.</em> (2012)</a>, using experiment 2 data with 55,000 SNPs, obtained predictions of approximately 0.55 for GY-WW using linear and nonlinear models. Regardless of the prediction model, GBS information seemingly performs better than other genotyping platforms with or without imputation for traits under WW conditions.</p><p id="p-69">In experiment 2, the prediction accuracy of GY under stressed conditions ranged from 0.373 to 0.4541, depending on whether GBS data were imputed and depending on the statistical model used. These correlations are considerably lower than those obtained under WW conditions and are consistent with previously reported results for this trait under these conditions. For instance, using this same data set but only 1,148 SNPs, <a id="xref-ref-6-8" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> (2010)</a> found prediction accuracies ranging from 0.42 to 0.45 for GY-SS.</p><p id="p-70">Part of the differences across studies may be attributable to the specifics of the validation designs used; however, the similarities of results across traits, environmental conditions, and studies allow us to conclude that, at a minimum, prediction accuracy with GBS-imputed data are on par with that of 55,000 SNPs, with the advantage that GBS data are less expensive.</p></div><div id="sec-25" class="subsection"><h3>Parametric <em>vs.</em> semi-parametric methods</h3><p id="p-71">In our study, we compared two ways of incorporating GBS data into models: parametric method (GBLUP) and semi-parametric method (RKHS). In general, no single method appeared to be clearly superior. Overall, in models using GBS data only (<em>i.e.</em>, without pedigree information), RKHS performed slightly better than GBLUP, with the only exception of GY under WW conditions in experiment 2. However, when GBS data were combined with pedigree information, prediction accuracy increased and the difference between the methods (GBLUP <em>vs.</em> RKHS) diminished. Overall, our results are in agreement with those of other authors who reported only small differences in the predictive performance of different statistical methods (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Heslot <em>et al.</em> 2012</a>). However, still other authors (<a id="xref-ref-6-9" class="xref-bibr" href="#ref-6">Crossa <em>et al.</em> 2010</a>; <a id="xref-ref-16-9" class="xref-bibr" href="#ref-16">González-Camacho <em>et al.</em> 2012</a>) have found differences across methods depending on the genetic architecture of the trait and depending on marker density. In many previously published studies, low marker density was pointed out as a possible reason for the lack of differences across methods; essentially, poor LD between markers and QTL at low marker density acts as a factor limiting overall prediction accuracy and the ability of each method to express its potential. Here, we have much higher density; however, because of the large proportion of imputed genotypes, rapid LD decay and weak LD between GBS-derived markers and causal loci may limit overall prediction accuracy and the ability for each of the models to show differences.</p><p id="p-72">Including all pairwise (or higher-order) interactions among markers in linear models continues to be a difficult problem because there are thousands of cryptic gene × gene interactions with very small effects. Although parametric linear regression imposes a linear relationship between markers and phenotype, nonparametric models do not impose any assumptions on the phenotype–genotype relationship, therefore making it possible to capture interactions among loci. The results of this study using GBS in two maize data sets show that linear regression and nonparametric models provided similar prediction correlations for most trait–environment combinations, except for GY under drought conditions (GY-SS), in which GBLUP predictions were lower than those from RKHS. However, this trend changed under WW conditions (GY-WW), in which both models performed very similarly in terms of prediction accuracy.</p></div></div><div class="section conclusions" id="sec-26"><h2 class="">Conclusions</h2><p id="p-73">Our results indicate that prediction of the performance of inbred maize lines using GBS data combined with pedigree information outperforms the prediction accuracy of pedigree-based methods and, at a minimum, can yield prediction accuracies comparable to those obtained with low-density to mid-density SNP arrays. The relatively low cost of GBS data further makes GBS a competitive alternative for improving the efficiency of breeding programs. Regarding the models, we found no clear differences in the prediction accuracy of the two models; however, even though RKHS was slightly better than GBLUP when using GBS alone, both models performed similarly with pedigree information. Dealing with missing genotypes poses an important challenge when using GBS data, and in this study we propose several ways of confronting that challenge. Results of this study indicated that prediction based on imputed SNPs and/or inferred haplotypes does not significantly improve predictions based on nonimputed SNPs. Further research is needed to develop and calibrate effective imputation methods for GBS data and to compare their predictive ability with those of models that use only uncalled GBS.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-92">We thank all national programs in Africa and Mexico that performed the experiments and collected the data analyzed in this study. We acknowledge the financial support provided to the Drought Tolerance Maize for Africa project, to the Water Effective Maize for Africa project, and to Cornell-CIMMYT by the Bill and Melinda Gates Foundation, and by the Howard Buffet Foundation. National Institutes of Health also provided financial support (grants R01GM099992-01A1 and R01GM101219-01 to G.d.l.C. and P.P.).</p></div><div class="section app" id="app-1"><h2 class="">Appendix A</h2><div class="section" id="sec-27"><div id="sec-28" class="subsection"><h4>Statistical models</h4><p id="p-74">The methods used in this study (including pedigree methods), GBLUP, and RKHS can all be cast within a general class of Gaussian random effects models. The general form of these models can be described as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">[1]</span></span>where <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-6.gif"/></noscript></span></span> is the phenotype of the <em>i<sup>th</sup></em> individual (<em>i=</em>1,…<em>n)</em>, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-7.gif"/></noscript></span></span> is a genetic value, and <em>ε<sub>i</sub></em> represents model residuals, all following the same normal distribution. For ease of notation, we have ignored the intercept and other nongenetic effects. Extensions used to accommodate nongenetic effects are straightforward.</p><p id="p-75">In the model described in equation [1], the separation of signal <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-8.gif"/></noscript></span></span> from noise <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-9.gif"/></noscript></span></span> is achieved by assigning different distributions to the vector of model residuals and the vector of genetic values. All the models used in this study share the sampling model described by equation [1]. In vector notation, we have<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-10.gif"/></noscript></span><span class="disp-formula-label">[2]</span></span>where <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-11.gif"/></noscript></span></span>and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-12.gif"/></noscript></span></span> are vectors of phenotypes and genetic values, respectively. However, the different models differ regarding the prior probability distributions assigned to genetic values. In its most general formulation, and following the study by <a id="xref-ref-10-8" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> (2010)</a>, the vector of genetic values can be represented as the sum of <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-13.gif"/></noscript></span></span> random effects as follows<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-14.gif"/></noscript></span><span class="disp-formula-label">[3]</span></span>where each of these random effects follow independent normal distributions of the form<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-15.gif"/></noscript></span><span class="disp-formula-label">[4]</span></span>where <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-16.gif"/></noscript></span></span> is the <em>n</em>×<em>n</em> covariance matrix computed either from markers or from pedigree. Combining expressions [2] through [4], we find that the joint distribution of the phenotypes and of the random effects is given by<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-17.gif"/></noscript></span><span class="disp-formula-label">[5]</span></span>The fully Bayesian model is completed by assigning a prior density for variance parameters. Following the standard assumption, we assigned scaled-inverse chi-squared densities to each of the variance parameters<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-18.gif"/></noscript></span><span class="disp-formula-label">[6]</span></span>where <em>H</em> denotes all hyperparameters indexing the prior probability distributions. The models used in this study are all special cases of the model described in equation [6]. <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table A2</a> lists all models used, the number of random effects included in each model, and the covariance matrices used.</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463561/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463561/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table A2</span> <span class="caption-title">Models, number of random effects for each model, and covariance matrices associated with the random effects</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section" id="sec-29"><h3>Computation of covariance matrices</h3></div><div class="section" id="sec-30"><h3>Pedigree model</h3><p id="p-77">In the pedigree model, the matrix <strong>A</strong> is a numerator relationship matrix (twice the matrix of kinship coefficients) computed from a pedigree.</p></div><div class="section" id="sec-31"><h3>GBLUP</h3><p id="p-78">In standard GBLUP (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">VanRaden 2007</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">2008</a>), the entries of the genomic relationship matrix, <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-20.gif"/></noscript></span></span>, represent proportions of allele sharing between pairs of individuals (<em>ii′</em>), realized at markers. Commonly, these are computed using cross-products of marker genotypes of the form<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-21.gif"/></noscript></span><span class="disp-formula-label">[7]</span></span>where <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-22.gif"/></noscript></span></span> represents the marker covariates (<em>i=</em>1,…,<em>n</em> individuals and <em>j=</em>1,…,<em>p</em> markers) and <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-23.gif"/></noscript></span></span> is the estimated allele frequency computed from nonmissing genotypes. Let <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-24.gif"/></noscript></span></span>; then, equation [7] in matrix notation can be expressed as <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-25.gif"/></noscript></span></span>. This formulation assumes that all genotypes are available. With GBS data, there is a large proportion of marker genotypes that are missing. To circumvent this problem we considered imputation, but even after imputation uncalled genotypes were still present, so the missing genotypes were replaced by their expected values and the proportion of missing genotypes for each individual was taken into account using a modified version of equation [7] given by <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">VanRaden (2008)</a>:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-26.gif"/></noscript></span><span class="disp-formula-label">[8]</span></span>where <strong>W</strong> is a diagonal matrix whose entries are given by <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-27.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-28.gif"/></noscript></span></span>denotes the set of markers for which genotypes were observed at individual <em>i</em>.</p></div><div class="section" id="sec-32"><h3>Reproducing kernel Hilbert spaces models with kernel averaging</h3><p id="p-79">In this implementation, we used three random effects. The covariance structures used were computed by means of a Gaussian kernel; therefore, the (<em>ii′)</em> of the kernels were given by <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-29.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-30.gif"/></noscript></span></span> is a bandwidth parameter peculiar to the <em>k<sup>th</sup></em> reproducing kernel and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-31.gif"/></noscript></span></span> is the genetic distance between lines <em>i</em> and <em>i′</em> computed as follows: <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-32.gif"/></noscript></span></span>. The bandwidth parameters were chosen to have an extremely local kernel (<em>i.e.</em>, one in which the covariance function declines very fast as genetic distance increases), one intermediate kernel, and one kernel in which the covariance function gives a high degree of resemblance even between distantly related lines. Specifically, we chose <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-33.gif"/></noscript></span></span>, where is the fifth percentile of <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-34.gif"/></noscript></span></span>.</p><p id="p-80">As with GBLUP, the computation of the Gaussian kernel requires observed marker genotypes at pairs of lines. With GBS, several marker genotypes are missing and, even after imputation, uncalled genotypes are still present. To cope with missing genotypes, the computation of distances was based only on observed genotypes, that is: <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-35.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-36.gif"/></noscript></span></span> represents the number of observed genotypes for lines <em>i</em> and <em>i′</em>.</p></div><div class="section" id="sec-33"><h3>Implementation</h3><p id="p-81">Samples from the posterior distribution, equation [6], were obtained using the Gibbs sampler described by <a id="xref-ref-10-9" class="xref-bibr" href="#ref-10">de los Campos <em>et al.</em> (2010)</a>. The software was implemented in R (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">R Core Team 2013</a>) and is available on request. The degree-of-freedom parameters of the scaled inverted chi-squared distributions were 5, and the scale parameters were chosen to match a heritability of 0.5, that is: <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-37.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-38.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-39.gif"/></noscript></span></span> is the sample variance of <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1903/embed/mml-math-40.gif"/></noscript></span></span>. Posterior means were estimated using 30,000 samples obtained after discarding 5000 that were taken as burn-in.</p></div></div><div class="section app" id="app-2"><h2>Appendix</h2><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A and B) Graphs of the box plot of r2 and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The r2 and linkage disequilibrium decay are shown at different marker distances for chromosome 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-416123368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A and B) Graphs of the box plot of &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay are shown at different marker distances for chromosome 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1903/F5.medium.gif" width="440" height="279"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B1" src="https://www.g3journal.org/content/ggg/3/11/1903/F5.medium.gif" width="440" height="279"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463500" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B1</span> <p id="p-82" class="first-child">(A and B) Graphs of the box plot of <em>r<sup>2</sup></em> and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The <em>r<sup>2</sup></em> and linkage disequilibrium decay are shown at different marker distances for chromosome 1.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A and B) Graphs of the box plot of r2 and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The r2 and linkage disequilibrium decay are shown at different marker distances for chromosome 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-416123368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A and B) Graphs of the box plot of &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay are shown at different marker distances for chromosome 2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1903/F6.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B2" src="https://www.g3journal.org/content/ggg/3/11/1903/F6.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463576" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B2</span> <p id="p-83" class="first-child">(A and B) Graphs of the box plot of <em>r<sup>2</sup></em> and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The <em>r<sup>2</sup></em> and linkage disequilibrium decay are shown at different marker distances for chromosome 2.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A and B) Graphs of the box plot of r2 and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The r2 and linkage disequilibrium decay are shown at different marker distances for chromosome 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-416123368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A and B) Graphs of the box plot of &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay are shown at different marker distances for chromosome 3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1903/F7.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B3" src="https://www.g3journal.org/content/ggg/3/11/1903/F7.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463557" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B3</span> <p id="p-84" class="first-child">(A and B) Graphs of the box plot of <em>r<sup>2</sup></em> and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The <em>r<sup>2</sup></em> and linkage disequilibrium decay are shown at different marker distances for chromosome 3.</p><div class="sb-div caption-clear"></div></div></div><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A and B) Graphs of the box plot of r2 and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The r2 and linkage disequilibrium decay are shown at different marker distances for chromosome 4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-416123368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A and B) Graphs of the box plot of &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay are shown at different marker distances for chromosome 4.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1903/F8.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B4" src="https://www.g3journal.org/content/ggg/3/11/1903/F8.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463523" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B4</span> <p id="p-85" class="first-child">(A and B) Graphs of the box plot of <em>r<sup>2</sup></em> and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The <em>r<sup>2</sup></em> and linkage disequilibrium decay are shown at different marker distances for chromosome 4.</p><div class="sb-div caption-clear"></div></div></div><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A and B) Graphs of the box plot of r2 and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The r2 and linkage disequilibrium decay are shown at different marker distances for chromosome 5." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-416123368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A and B) Graphs of the box plot of &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay are shown at different marker distances for chromosome 5.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1903/F9.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B5" src="https://www.g3journal.org/content/ggg/3/11/1903/F9.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463585" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B5</span> <p id="p-86" class="first-child">(A and B) Graphs of the box plot of <em>r<sup>2</sup></em> and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The <em>r<sup>2</sup></em> and linkage disequilibrium decay are shown at different marker distances for chromosome 5.</p><div class="sb-div caption-clear"></div></div></div><div id="F10" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F10.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A and B) Graphs of the box plot of r2 and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The r2 and linkage disequilibrium decay are shown at different marker distances for chromosome 6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-416123368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A and B) Graphs of the box plot of &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay are shown at different marker distances for chromosome 6.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1903/F10.medium.gif" width="440" height="281"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B6" src="https://www.g3journal.org/content/ggg/3/11/1903/F10.medium.gif" width="440" height="281"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F10.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F10.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463511" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B6</span> <p id="p-87" class="first-child">(A and B) Graphs of the box plot of <em>r<sup>2</sup></em> and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The <em>r<sup>2</sup></em> and linkage disequilibrium decay are shown at different marker distances for chromosome 6.</p><div class="sb-div caption-clear"></div></div></div><div id="F11" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F11.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A and B) Graphs of the box plot of r2 and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The r2 and linkage disequilibrium decay are shown at different marker distances for chromosome 7." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-416123368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A and B) Graphs of the box plot of &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay are shown at different marker distances for chromosome 7.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1903/F11.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B7" src="https://www.g3journal.org/content/ggg/3/11/1903/F11.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F11.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F11.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463515" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B7</span> <p id="p-88" class="first-child">(A and B) Graphs of the box plot of <em>r<sup>2</sup></em> and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The <em>r<sup>2</sup></em> and linkage disequilibrium decay are shown at different marker distances for chromosome 7.</p><div class="sb-div caption-clear"></div></div></div><div id="F12" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F12.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A and B) Graphs of the box plot of r2 and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The r2 and linkage disequilibrium decay are shown at different marker distances for chromosome 8." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-416123368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A and B) Graphs of the box plot of &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay are shown at different marker distances for chromosome 8.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1903/F12.medium.gif" width="440" height="279"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B8" src="https://www.g3journal.org/content/ggg/3/11/1903/F12.medium.gif" width="440" height="279"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F12.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F12.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463601" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B8</span> <p id="p-89" class="first-child">(A and B) Graphs of the box plot of <em>r<sup>2</sup></em> and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The <em>r<sup>2</sup></em> and linkage disequilibrium decay are shown at different marker distances for chromosome 8.</p><div class="sb-div caption-clear"></div></div></div><div id="F13" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F13.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A and B) Graphs of the box plot of r2 and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The r2 and linkage disequilibrium decay are shown at different marker distances for chromosome 9." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-416123368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A and B) Graphs of the box plot of &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay are shown at different marker distances for chromosome 9.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1903/F13.medium.gif" width="440" height="280"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B9" src="https://www.g3journal.org/content/ggg/3/11/1903/F13.medium.gif" width="440" height="280"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F13.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F13.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463510" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B9</span> <p id="p-90" class="first-child">(A and B) Graphs of the box plot of <em>r<sup>2</sup></em> and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The <em>r<sup>2</sup></em> and linkage disequilibrium decay are shown at different marker distances for chromosome 9.</p><div class="sb-div caption-clear"></div></div></div><div id="F14" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F14.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A and B) Graphs of the box plot of r2 and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The r2 and linkage disequilibrium decay are shown at different marker distances for chromosome 10." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-416123368" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A and B) Graphs of the box plot of &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The &lt;em&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt; and linkage disequilibrium decay are shown at different marker distances for chromosome 10.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure B10" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1903/F14.medium.gif" width="440" height="279"/><noscript><img class="highwire-fragment fragment-image" alt="Figure B10" src="https://www.g3journal.org/content/ggg/3/11/1903/F14.medium.gif" width="440" height="279"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F14.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure B10" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1903/F14.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463593" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure B10</span> <p id="p-91" class="first-child">(A and B) Graphs of the box plot of <em>r<sup>2</sup></em> and linkage disequilibrium decay at different marker distances in the (A) 504 DH maize lines (experiment 1) and (B) 296 maize lines (experiment 2). The <em>r<sup>2</sup></em> and linkage disequilibrium decay are shown at different marker distances for chromosome 10.</p><div class="sb-div caption-clear"></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-93"><em>Communicating editor: D.-J. De Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-05-30"><span class="received-label">Received </span>May 30, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-08-31"><span class="accepted-label">Accepted </span>August 31, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Crossa <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.1" data-doi="10.1534/genetics.112.147710"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beissinger</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sekhon</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foerster</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Marker density and read depth for genotyping populations using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147710%26rft_id%253Dinfo%253Apmid%252F23410831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5My80LzEwNzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.2" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTkvMjYzMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMTkwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brito</span>  <span class="cit-name-given-names">F. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neto</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sargikzaei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cobuci</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schenkel</span>  <span class="cit-name-given-names">F. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy of genomic selection in simulated populations mimicking the extent of linkage disequilibrium in beef cattle.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">80</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DBrito%26rft.auinit1%253DF.%2BV.%26rft.volume%253D12%26rft.spage%253D80%26rft.epage%253D80%26rft.atitle%253DAccuracy%2Bof%2Bgenomic%2Bselection%2Bin%2Bsimulated%2Bpopulations%2Bmimicking%2Bthe%2Bextent%2Bof%2Blinkage%2Bdisequilibrium%2Bin%2Bbeef%2Bcattle.%26rft_id%253Dinfo%253Apmid%252F21933416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21933416&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.4" data-doi="10.2135/cropsci2011.06.0299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">707</span>–<span class="cit-lpage">719</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D707%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.5" data-doi="10.1038/ng.2313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">J.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Costich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Leon</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Maize HapMap2 identifies extant variation from a genome in flux.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">807</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DChia%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D803%26rft.epage%253D807%26rft.atitle%253DMaize%2BHapMap2%2Bidentifies%2Bextant%2Bvariation%2Bfrom%2Ba%2Bgenome%2Bin%2Bflux.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2313%26rft_id%253Dinfo%253Apmid%252F22660545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660545&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.6" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.7" data-doi="10.1080/15427528.2011.558767"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mahuku</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dreisigacker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genomic selection and prediction in plant breeding.</span> <abbr class="cit-jnl-abbrev">J. Crop Improv.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">239</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCrop%2BImprov.%26rft.volume%253D25%26rft.spage%253D239%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15427528.2011.558767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15427528.2011.558767&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burgueño</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ornella</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in CIMMYT maize and wheat breeding programs.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-comment">doi: 10.1038/hdy.2013.16</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.9" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naya</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Legarra</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigrees</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.10" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pong-Wong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daetwyler</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Whole genome regression and prediction methods applied to plant and animal breeding.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D193%26rft.spage%253D327%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22745228&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.12" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.13" data-doi="10.1073/pnas.132259199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dooner</span>  <span class="cit-name-given-names">H. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Intraspecific violation of genetic colinearity and its implications in maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">9573</span>–<span class="cit-lpage">9578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.132259199%26rft_id%253Dinfo%253Apmid%252F12060715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTkvMTQvOTU3MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMTkwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.14" data-doi="10.1534/genetics.105.049510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stella</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic assisted prediction of genetic value with semi-parametric procedures.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1761</span>–<span class="cit-lpage">1776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049510%26rft_id%253Dinfo%253Apmid%252F16648593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzE3NjEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.15" data-doi="10.1186/1471-2156-12-87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okut</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Predicting complex quantitative traits with Bayesian neural networks: a case study with Jersey cows and wheat.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">87</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D87%26rft.epage%253D87%26rft.atitle%253DPredicting%2Bcomplex%2Bquantitative%2Btraits%2Bwith%2BBayesian%2Bneural%2Bnetworks%253A%2Ba%2Bcase%2Bstudy%2Bwith%2BJersey%2Bcows%2Band%2Bwheat.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-12-87%26rft_id%253Dinfo%253Apmid%252F21981731%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-12-87&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21981731&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.16" data-doi="10.1007/s00122-012-1868-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cairns</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-enabled prediction of genetic values using radial basis function neural networks.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">771</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGonzalez-Camacho%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D125%26rft.issue%253D4%26rft.spage%253D759%26rft.epage%253D771%26rft.atitle%253DGenome-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Bradial%2Bbasis%2Bfunction%2Bneural%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1868-9%26rft_id%253Dinfo%253Apmid%252F22566067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1868-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22566067&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.17" data-doi="10.2135/cropsci2011.06.0297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heslot</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorells</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannick</span>  <span class="cit-name-given-names">J.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in plant breeding: A comparison of models.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D52%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2011.06.0297%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2011.06.0297&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Factors affecting the accuracy of genotype imputation in populations from several maize breeding programs.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.1903.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Down</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pocock</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prlić</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-source">BioJava: An open-source framework for bioinformatics</span>, <span class="cit-publ-name">R.C.G. Bioinformatics</span>. <span class="cit-comment">doi:<a href="http://dx.doi.org/10.1093/bioinformatics/btn397">http://dx.doi.org/10.1093/bioinformatics/btn397</a></span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.20" data-doi="10.1534/genetics.112.141143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Janss</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheehan</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Inferences from genomic models in stratified populations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">192</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.141143%26rft_id%253Dinfo%253Apmid%252F22813891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkyLzIvNjkzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.21" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows–Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">14</span>): <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMTkwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.22" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wysoker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fennell</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and SAM tools.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTYvMjA3OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMTkwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.23" data-doi="10.1371/journal.pone.0024861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taba</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparative SNP and haplotype analysis reveals a higher genetic diversity and rapider LD decay in tropical than temperate germplasm in maize.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e24861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLu%26rft.auinit1%253DY.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253De24861%26rft.epage%253De24861%26rft.atitle%253DComparative%2BSNP%2Band%2Bhaplotype%2Banalysis%2Breveals%2Ba%2Bhigher%2Bgenetic%2Bdiversity%2Band%2Brapider%2BLD%2Bdecay%2Bin%2Btropical%2Bthan%2Btemperate%2Bgermplasm%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0024861%26rft_id%253Dinfo%253Apmid%252F21949770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0024861&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21949770&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goddard</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic values using genome-wide dense marker maps.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.25" data-doi="10.3835/plantgenome2010.04.0005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genomic-enabled prediction based on molecular markers and pedigree using the BLR package in R.</span> <abbr class="cit-jnl-abbrev">The Plant Genome</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">106</span>–<span class="cit-lpage">116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%26rft.stitle%253DThe%2BPlant%2BGenome%26rft.aulast%253DPerez%26rft.auinit1%253DP.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253D106%26rft.epage%253D116%26rft.atitle%253DGenomic-Enabled%2BPrediction%2BBased%2Bon%2BMolecular%2BMarkers%2Band%2BPedigree%2BUsing%2Bthe%2BBayesian%2BLinear%2BRegression%2BPackage%2Bin%2BR.%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2010.04.0005%26rft_id%253Dinfo%253Apmid%252F21566722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2010.04.0005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21566722&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manès</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparison between linear and non-parametric regression models for genome-enabled prediction in wheat. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1595</span>–<span class="cit-lpage">1605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D2%26rft.spage%253D1595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.27" data-doi="10.3835/plantgenome2012.06.0006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rutkoski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection in wheat breeding using genotyping-by-sequencing.</span> <abbr class="cit-jnl-abbrev">The Plant Genome</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">103</span>–<span class="cit-lpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%26rft.volume%253D5%26rft.spage%253D103%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2012.06.0006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2012.06.0006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.28"><div class="cit-metadata"><cite><span class="cit-comment">R Core Team 2013. R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. Available at: <a href="http://www.R-project.org/">http://www.R-project.org/</a></span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.29" data-doi="10.1186/gb-2013-14-6-r55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comprehensive genotyping of the USA national maize inbred seed bank.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR55%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-6-r55%26rft_id%253Dinfo%253Apmid%252F23759205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-6-r55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23759205&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.30" data-doi="10.1371/journal.pgen.1000734"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Springer</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ying</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C.-T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Maize inbreds exhibit high levels of copy number variation (CNV) and presence/absence variation (PAV) in genome content.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000734</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DSpringer%26rft.auinit1%253DN.%2BM.%26rft.volume%253D5%26rft.issue%253D11%26rft.spage%253De1000734%26rft.epage%253De1000734%26rft.atitle%253DMaize%2Binbreds%2Bexhibit%2Bhigh%2Blevels%2Bof%2Bcopy%2Bnumber%2Bvariation%2B%2528CNV%2529%2Band%2Bpresence%252Fabsence%2Bvariation%2B%2528PAV%2529%2Bin%2Bgenome%2Bcontent.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000734%26rft_id%253Dinfo%253Apmid%252F19956538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000734&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19956538&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genomic measures of relationship and inbreeding.</span> <abbr class="cit-jnl-abbrev">Interbull Bull.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInterbull%2BBull.%26rft.volume%253D37%26rft.spage%253D33%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.32" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.33" data-doi="10.3168/jds.2010-3335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosa</span>  <span class="cit-name-given-names">G. J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weigel</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Predictive ability of subsets of single nucleotide polymorphisms with and without parent average in US Holsteins.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">5942</span>–<span class="cit-lpage">5949</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVazquez%26rft.auinit1%253DA.%2BI.%26rft.volume%253D93%26rft.issue%253D12%26rft.spage%253D5942%26rft.epage%253D5949%26rft.atitle%253DPredictive%2Bability%2Bof%2Bsubsets%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bwith%2Band%2Bwithout%2Bparent%2Baverage%2Bin%2BUS%2BHolsteins.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3335%26rft_id%253Dinfo%253Apmid%252F21094768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21094768&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.34" data-doi="10.1093/nar/gkr599"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lyon</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hakonarson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SNVer: a statistical tool for variant calling in analysis of pooled or individual next-generation sequencing data.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">e132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr599%26rft_id%253Dinfo%253Apmid%252F21813454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOS8xOS9lMTMyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.35" data-doi="10.1534/g3.112.003699"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Windhausen</span>  <span class="cit-name-given-names">V. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atlin</span>  <span class="cit-name-given-names">G. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grudloyma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">1427</span>–<span class="cit-lpage">1436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DWindhausen%26rft.auinit1%253DV.%2BS.%26rft.volume%253D2%26rft.issue%253D11%26rft.spage%253D1427%26rft.epage%253D1436%26rft.atitle%253DEffectiveness%2Bof%2BGenomic%2BPrediction%2Bof%2BMaize%2BHybrid%2BPerformance%2Bin%2BDifferent%2BBreeding%2BPopulations%2Band%2BEnvironments%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.003699%26rft_id%253Dinfo%253Apmid%252F23173094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjIvMTEvMTQyNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMTkwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.36" data-doi="10.1038/ng.551"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kandianis</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harjes</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">E.-H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rare genetic variation at <em>Zea mays</em> crtRB1 increases beta-carotene in maize grain</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">322</span>–<span class="cit-lpage">327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYan%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D322%26rft.epage%253D327%26rft.atitle%253DRare%2Bgenetic%2Bvariation%2Bat%2BZea%2Bmays%2BcrtRB1%2Bincreases%2Bbeta-carotene%2Bin%2Bmaize%2Bgrain.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.551%26rft_id%253Dinfo%253Apmid%252F20305664%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.551&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20305664&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1903.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1903.37" data-doi="10.1534/genetics.108.098277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dekkers</span>  <span class="cit-name-given-names">J.C.M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R.L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jannink</span>  <span class="cit-name-given-names">J-L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: a barley case study.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.098277%26rft_id%253Dinfo%253Apmid%252F19299342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/11/1903.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/1893" title="Genetic Background and GxE Interactions Modulate the Penetrance of a Naturally Occurring Wing Mutation in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/11/1927" title="A Mutation in the FHA Domain of Coprinus cinereus Nbs1 Leads to Spo11-Independent Meiotic Recombination and Chromosome Segregation" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/11"><span>Volume 3 Issue 11, November 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/11.cover-source.jpg?itok=A7iZ1ztq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/1903" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-PCEu2M3ZUolqlps5BhD5iCYi3UiQdXvOBo9euEvZM8s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-selection" class="" data-icon-position="" data-hide-link-title="0">Genomic Selection</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/11/1903" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/11/1903" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/11/1903" class="active" data-icon-position="" data-hide-link-title="0">Genomic Prediction in Maize Breeding Populations with Genotyping-by-Sequencing</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459578" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-OztaAveiDxdCZtE7irYb1sdKbr9WY65xGmpNeHwrlGY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/1903.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/11/1903" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-InxFGi1yZ1wu-Myos3ImmQzwMe-UeerIZzG9JCGuqOM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/11/1903" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/1903.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459578" id="node459578--2" data-pisa="ggg;3/11/1903" data-pisa-master="ggg;g3.113.008227" data-apath="/ggg/3/11/1903.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction in Maize Breeding Populations with Genotyping-by-Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">José Crossa</span>, <span class="highwire-citation-author" data-delta="1">Yoseph Beyene</span>, <span class="highwire-citation-author" data-delta="2">Semagn Kassa</span>, <span class="highwire-citation-author" data-delta="3">Paulino Pérez</span>, <span class="highwire-citation-author" data-delta="4">John M. Hickey</span>, <span class="highwire-citation-author" data-delta="5">Charles Chen</span>, <span class="highwire-citation-author" data-delta="6">Gustavo de los Campos</span>, <span class="highwire-citation-author" data-delta="7">Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="8">Vanessa S. Windhausen</span>, <span class="highwire-citation-author" data-delta="9">Ed Buckler</span>, <span class="highwire-citation-author" data-delta="10">Jean-Luc Jannink</span>, <span class="highwire-citation-author" data-delta="11">Marco A. Lopez Cruz</span> and <span class="highwire-citation-author" data-delta="12">Raman Babu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1903-1926;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008227 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459578--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yoseph Beyene </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoseph%2BBeyene%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beyene%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoseph%2BBeyene%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Semagn Kassa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Semagn%2BKassa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kassa%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASemagn%2BKassa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulino Pérez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Colegio de Postgraduados, Montecillos, Edo. de Mexico, 56230, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John M. Hickey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>The Roslin Institute, University of Edinburgh, Easter Bush, Midlothian, EH25 9RG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BHickey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hickey%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BHickey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Charles Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biostatistics, School of Public Health, University of Alabama at Birmingham, Alabama 35294</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Vanessa S. Windhausen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Saaten Union Recherche, 163 Avenue de Flandre, 60190 Estrées Saint Denis, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vanessa%2BS.%2BWindhausen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Windhausen%20VS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVanessa%2BS.%2BWindhausen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ed Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA—ARS, Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ed%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEd%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA—ARS, Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Marco A. Lopez Cruz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marco%2BA.%2BLopez%2BCruz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez%20Cruz%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarco%2BA.%2BLopez%2BCruz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Raman Babu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raman%2BBabu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Babu%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARaman%2BBabu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/11/1903" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/11/1903&amp;title=Genomic%20Prediction%20in%20Maize%20Breeding%20Populations%20with%20Genotyping-by-Sequencing" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/11/1903&amp;title=Genomic%20Prediction%20in%20Maize%20Breeding%20Populations%20with%20Genotyping-by-Sequencing" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/11/1903&amp;title=Genomic%20Prediction%20in%20Maize%20Breeding%20Populations%20with%20Genotyping-by-Sequencing" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/11/1903&amp;text=Genomic%20Prediction%20in%20Maize%20Breeding%20Populations%20with%20Genotyping-by-Sequencing" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/11/1903&amp;title=Genomic%20Prediction%20in%20Maize%20Breeding%20Populations%20with%20Genotyping-by-Sequencing" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/11/1903&amp;t=Genomic%20Prediction%20in%20Maize%20Breeding%20Populations%20with%20Genotyping-by-Sequencing" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/11/1903&amp;title=Genomic%20Prediction%20in%20Maize%20Breeding%20Populations%20with%20Genotyping-by-Sequencing" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/11/1903&amp;title=Genomic%20Prediction%20in%20Maize%20Breeding%20Populations%20with%20Genotyping-by-Sequencing" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459578/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459578" id="node459578--3" data-pisa="ggg;3/11/1903" data-pisa-master="ggg;g3.113.008227" data-apath="/ggg/3/11/1903.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction in Maize Breeding Populations with Genotyping-by-Sequencing</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">José Crossa</span>, <span class="highwire-citation-author" data-delta="1">Yoseph Beyene</span>, <span class="highwire-citation-author" data-delta="2">Semagn Kassa</span>, <span class="highwire-citation-author" data-delta="3">Paulino Pérez</span>, <span class="highwire-citation-author" data-delta="4">John M. Hickey</span>, <span class="highwire-citation-author" data-delta="5">Charles Chen</span>, <span class="highwire-citation-author" data-delta="6">Gustavo de los Campos</span>, <span class="highwire-citation-author" data-delta="7">Juan Burgueño</span>, <span class="highwire-citation-author" data-delta="8">Vanessa S. Windhausen</span>, <span class="highwire-citation-author" data-delta="9">Ed Buckler</span>, <span class="highwire-citation-author" data-delta="10">Jean-Luc Jannink</span>, <span class="highwire-citation-author" data-delta="11">Marco A. Lopez Cruz</span> and <span class="highwire-citation-author" data-delta="12">Raman Babu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1903-1926;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008227 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459578--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">José Crossa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BCrossa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Crossa%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJos%25C3%25A9%2BCrossa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:j.crossa@cgiar.org" class="" data-icon-position="" data-hide-link-title="0">j.crossa@cgiar.org</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Yoseph Beyene </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yoseph%2BBeyene%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Beyene%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYoseph%2BBeyene%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Semagn Kassa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Semagn%2BKassa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kassa%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASemagn%2BKassa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Paulino Pérez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Colegio de Postgraduados, Montecillos, Edo. de Mexico, 56230, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Paulino%2BP%C3%A9rez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=P%C3%A9rez%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APaulino%2BP%25C3%25A9rez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">John M. Hickey </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>The Roslin Institute, University of Edinburgh, Easter Bush, Midlothian, EH25 9RG, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BM.%2BHickey%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hickey%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BM.%2BHickey%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Charles Chen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BChen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chen%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BChen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Gustavo de los Campos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Biostatistics, School of Public Health, University of Alabama at Birmingham, Alabama 35294</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gustavo%2Bde%2Blos%2BCampos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20los%20Campos%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGustavo%2Bde%2Blos%2BCampos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Juan Burgueño </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juan%2BBurgue%C3%B1o%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgue%C3%B1o%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuan%2BBurgue%25C3%25B1o%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Vanessa S. Windhausen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Saaten Union Recherche, 163 Avenue de Flandre, 60190 Estrées Saint Denis, France</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vanessa%2BS.%2BWindhausen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Windhausen%20VS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVanessa%2BS.%2BWindhausen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ed Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA—ARS, Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ed%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEd%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jean-Luc Jannink </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>USDA—ARS, Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York, New York 14850</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jean-Luc%2BJannink%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jannink%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJean-Luc%2BJannink%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Marco A. Lopez Cruz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marco%2BA.%2BLopez%2BCruz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lopez%20Cruz%20MA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarco%2BA.%2BLopez%2BCruz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Raman Babu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641, 06600, Mexico DF, Mexico</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Raman%2BBabu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Babu%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARaman%2BBabu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1903&amp;rft.epage=1926&amp;rft.atitle=Genomic%20Prediction%20in%20Maize%20Breeding%20Populations%20with%20Genotyping-by-Sequencing&amp;rft.volume=3&amp;rft.issue=11&amp;rft.date=2013-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Crossa%2C+Jos%C3%A9&amp;rft.au=Beyene%2C+Yoseph&amp;rft.au=Kassa%2C+Semagn&amp;rft.au=P%C3%A9rez%2C+Paulino&amp;rft.au=Hickey%2C+John+M.&amp;rft.au=Chen%2C+Charles&amp;rft.au=de+los+Campos%2C+Gustavo&amp;rft.au=Burgue%C3%B1o%2C+Juan&amp;rft.au=Windhausen%2C+Vanessa+S.&amp;rft.au=Buckler%2C+Ed&amp;rft.au=Jannink%2C+Jean-Luc&amp;rft.au=Lopez+Cruz%2C+Marco+A.&amp;rft.au=Babu%2C+Raman"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459578/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459578/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459578/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459578/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459578/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459578/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459578/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459578/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459578/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459578/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459578/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459578/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490720" id="node490720" data-pisa="ggg;8/9/3019" data-pisa-master="ggg;g3.118.200430" data-apath="/ggg/8/9/3019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">A Bayesian Decision Theory Approach for Genomic Selection</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490724" id="node490724" data-pisa="ggg;8/9/2889" data-pisa-master="ggg;g3.118.200311" data-apath="/ggg/8/9/2889.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/2889" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimising Genomic Selection in Wheat: Effect of Marker Density, Population Size and Population Structure on Prediction Accuracy</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="490727" id="node490727" data-pisa="ggg;8/9/3039" data-pisa-master="ggg;g3.118.200435" data-apath="/ggg/8/9/3039.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/8/9/3039" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">BGGE: A New Package for Genomic-Enabled Prediction Incorporating Genotype × Environment Interaction Models</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Selection" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Selection</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/11/1903" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-26" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix A</a></li><li class="even"><a href="#app-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/11/1903.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/11/1903.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/3/11/1903.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
