<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/11/2069" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/11/2069.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/11/2069.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/11/2069.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/11/2069.atom" />
<meta name="HW.pisa" content="ggg;3/11/2069" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Allelic Variation in a Cellulose Synthase Gene (PtoCesA4) Associated with Growth and Wood Properties in Populus tomentosa" />
<meta name="DC.Identifier" content="10.1534/g3.113.007724" />
<meta name="DC.Date" content="2013-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Du et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Lignocellulosic biomass from trees provides a renewable feedstock for biofuels, lumber, pulp, paper, and other uses. Dissecting the mechanism underlying natural variation of the complex traits controlling growth and lignocellulose biosynthesis in trees can enable marker-assisted breeding to improve wood quality and yield. Here, we combined linkage disequilibrium (LD)-based association analysis with traditional linkage analysis to detect the genetic effect of a Populus tomentosa cellulose synthase gene, PtoCesA4 . PtoCesA4 is strongly expressed in developing xylem and leaves. Nucleotide diversity and LD in PtoCesA4 , sampled from the P. tomentosa natural distribution, revealed that PtoCesA4 harbors high single nucleotide polymorphism (SNP) diversity (πT = 0.0080 and θw = 0.0098) and low LD ( r2 ≥ 0.1, within 1400 bp), demonstrating that the potential of a candidate-gene-based LD approach in understanding the molecular basis underlying quantitative variation in this species. By combining single SNP, multi-SNP, and haplotype-based associations in an association population of 460 individuals with single SNP linkage analysis in a family-based linkage populations (1200 individuals), we identified three strong associations (false discovery rate Q &lt; 0.05) in both populations. These include two nonsynonymous markers (SNP49 associated with α-cellulose content and SNP59 associated with fiber width) and a noncoding marker (SNP18 associated with α-cellulose content). Variation in RNA transcript abundance among genotypic classes of SNP49 was confirmed in these two populations. Therefore, combining different methods allowed us to examine functional PtoCesA4 allelic variation underlying natural variation in complex quantitative traits related to growth and lignocellulosic biosynthesis." />
<meta name="DC.Contributor" content="Qingzhang Du" />
<meta name="DC.Contributor" content="Baohua Xu" />
<meta name="DC.Contributor" content="Wei Pan" />
<meta name="DC.Contributor" content="Chenrui Gong" />
<meta name="DC.Contributor" content="Qingshi Wang" />
<meta name="DC.Contributor" content="Jiaxing Tian" />
<meta name="DC.Contributor" content="Bailian Li" />
<meta name="DC.Contributor" content="Deqiang Zhang" />
<meta name="article:published_time" content="2013-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Allelic Variation in a Cellulose Synthase Gene (PtoCesA4) Associated with Growth and Wood Properties in Populus tomentosa" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Lignocellulosic biomass from trees provides a renewable feedstock for biofuels, lumber, pulp, paper, and other uses. Dissecting the mechanism underlying natural variation of the complex traits controlling growth and lignocellulose biosynthesis in trees can enable marker-assisted breeding to improve wood quality and yield. Here, we combined linkage disequilibrium (LD)-based association analysis with traditional linkage analysis to detect the genetic effect of a &lt;i&gt;Populus tomentosa&lt;/i&gt; cellulose synthase gene, &lt;i&gt;PtoCesA4&lt;/i&gt;. &lt;i&gt;PtoCesA4&lt;/i&gt; is strongly expressed in developing xylem and leaves. Nucleotide diversity and LD in &lt;i&gt;PtoCesA4&lt;/i&gt;, sampled from the &lt;i&gt;P. tomentosa&lt;/i&gt; natural distribution, revealed that &lt;i&gt;PtoCesA4&lt;/i&gt; harbors high single nucleotide polymorphism (SNP) diversity (π&lt;sub&gt;T&lt;/sub&gt; = 0.0080 and θ&lt;sub&gt;w&lt;/sub&gt; = 0.0098) and low LD (&lt;i&gt;r&lt;sup&gt;2&lt;/sup&gt;&lt;/i&gt; ≥ 0.1, within 1400 bp), demonstrating that the potential of a candidate-gene-based LD approach in understanding the molecular basis underlying quantitative variation in this species. By combining single SNP, multi-SNP, and haplotype-based associations in an association population of 460 individuals with single SNP linkage analysis in a family-based linkage populations (1200 individuals), we identified three strong associations (false discovery rate &lt;i&gt;Q&lt;/i&gt; &amp;lt; 0.05) in both populations. These include two nonsynonymous markers (SNP49 associated with α-cellulose content and SNP59 associated with fiber width) and a noncoding marker (SNP18 associated with α-cellulose content). Variation in RNA transcript abundance among genotypic classes of SNP49 was confirmed in these two populations. Therefore, combining different methods allowed us to examine functional &lt;i&gt;PtoCesA4&lt;/i&gt; allelic variation underlying natural variation in complex quantitative traits related to growth and lignocellulosic biosynthesis.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/11/01" />
<meta name="citation_mjid" content="ggg;3/11/2069" />
<meta name="citation_id" content="3/11/2069" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/11/2069" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/11/2069.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/11/2069.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/11/2069.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.007724" />
<meta name="citation_pmid" content="24048648" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2069" />
<meta name="citation_lastpage" content="2084" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Qingzhang Du" />
<meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China" />
<meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China" />
<meta name="citation_author" content="Baohua Xu" />
<meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China" />
<meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China" />
<meta name="citation_author" content="Wei Pan" />
<meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China" />
<meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China" />
<meta name="citation_author" content="Chenrui Gong" />
<meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China" />
<meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China" />
<meta name="citation_author" content="Qingshi Wang" />
<meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China" />
<meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China" />
<meta name="citation_author" content="Jiaxing Tian" />
<meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China" />
<meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China" />
<meta name="citation_author" content="Bailian Li" />
<meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China" />
<meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China" />
<meta name="citation_author_institution" content="Department of Forestry, North Carolina State University, Raleigh, North Carolina 27695-8203" />
<meta name="citation_author" content="Deqiang Zhang" />
<meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China" />
<meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China" />
<meta name="citation_author_email" content="DeqiangZhang@bjfu.edu.cn" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. F. Altschul;citation_author=T. L. Madden;citation_author=A. A. Schaffer;citation_author=J. H. Zhang;citation_author=Z. Zhang;citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.;citation_volume=25;citation_year=1997;citation_pmid=9254694;citation_doi=10.1093/nar/25.17.3389" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=I. I. Atanassov;citation_author=J. K. Pittman;citation_author=S. R. Turner;citation_title=Elucidating the mechanisms of assembly and subunit interaction of the cellulose synthase complex of Arabidopsis secondary cell walls.;citation_volume=284;citation_year=2009;citation_pmid=19056734;citation_doi=10.1074/jbc.M807456200" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Atwell;citation_author=Y. S. Huang;citation_author=B. J. Vilhjalmsson;citation_author=G. Willems;citation_author=M. Horton;citation_title=Genome-wide association study of 107 phenotypes in Arabidopsis thaliana inbred lines.;citation_pages=627-631;citation_volume=465;citation_year=2010;citation_issue=7298;citation_pmid=20336072;citation_doi=10.1038/nature08800" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Beaulieu;citation_author=T. Doerksen;citation_author=B. Boyle;citation_author=S. Clement;citation_author=M. Deslauriers;citation_title=Association genetics of wood physical traits in the conifer white spruce and relationships with gene expression.;citation_volume=188;citation_year=2011;citation_pmid=21385726;citation_doi=10.1534/genetics.110.125781" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=B. Brachi;citation_author=N. Faure;citation_author=M. Horton;citation_author=E. Flahauw;citation_author=A. Vazquez;citation_title=Linkage and association mapping of Arabidopsis thaliana flowering time in nature.;citation_pages=e1000940-e1000940;citation_volume=6;citation_year=2010;citation_issue=5;citation_pmid=20463887;citation_doi=10.1371/journal.pgen.1000940" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Branca;citation_author=T. D. Paape;citation_author=P. Zhou;citation_author=R. Briskine;citation_author=A. D. Farmer;citation_title=Whole-genome nucleotide diversity, recombination, and linkage disequilibrium in the model legume Medicago truncatula.;citation_volume=108;citation_year=2011;citation_pmid=21949378;citation_doi=10.1073/pnas.1104032108" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=D. M. Brown;citation_author=L. A. H. Zeef;citation_author=J. Ellis;citation_author=R. Goodacreb;citation_author=S. R. Turner;citation_title=Identification of novel genes in Arabidopsis involved in secondary cell wall formation using expression profiling and reverse genetics.;citation_volume=17;citation_year=2005;citation_pmid=15980264;citation_doi=10.1105/tpc.105.031542" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=R. A. Burton;citation_author=N. J. Shirley;citation_author=B. J. King;citation_author=A. J. Harvey;citation_author=G. B. Fincher;citation_title=The CesA gene family of barley (Hordeum vulgare): quantitative analysis of transcripts reveals two groups of co-expressed genes.;citation_volume=134;citation_year=2004;citation_pmid=14701917;citation_doi=10.1104/pp.103.032904" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Chen;citation_author=D. W. Ehrhardt;citation_author=C. R. Somerville;citation_title=Mutations of cellulose synthase (CESA1) phosphorylation sites modulate anisotropic cell expansion and bidirectional mobility of cellulose synthase.;citation_volume=107;citation_year=2010;citation_pmid=20855602;citation_doi=10.1073/pnas.1012348107" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=Z. Chen;citation_author=X. Hong;citation_author=H. Zhang;citation_author=Y. Wang;citation_author=X. Li;citation_title=Disruption of the cellulose synthase gene, AtCesA8/IRX1, enhances drought and osmotic stress tolerance in Arabidopsis.;citation_pages=273-283;citation_volume=43;citation_year=2005;citation_issue=2;citation_pmid=15998313;citation_doi=10.1111/j.1365-313X.2005.02452.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. D. Coleman;citation_author=J. Yan;citation_author=S. D. Mansfield;citation_title=Sucrose synthase affects carbon partitioning to increase cellulose production and altered cell wall ultrastructure.;citation_volume=106;citation_year=2009;citation_pmid=19625620;citation_doi=10.1073/pnas.0900188106" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=A. d’Alesio;citation_author=M. Garabedian;citation_author=J. P. Sabatier;citation_author=G. Guaydier-Souquieres;citation_author=C. Marcelli;citation_title=Two single-nucleotide polymorphisms in the human vitamin D receptor promoter change protein-DNA complex formation and are associated with height and vitamin D status in adolescent girls.;citation_volume=14;citation_year=2005;citation_pmid=16210379;citation_doi=10.1093/hmg/ddi382" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. K. Dillon;citation_author=M. Nolan;citation_author=W. Li;citation_author=C. Bell;citation_author=H. X. Wu;citation_title=Allelic variation in cell wall candidate genes affecting solid wood properties in association populations and land races of Pinus radiata.;citation_volume=185;citation_year=2010;citation_pmid=20498299;citation_doi=10.1534/genetics.110.116582" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=S. K. Dillon;citation_author=J. T. Brawner;citation_author=R. Meder;citation_author=D. J. Lee;citation_author=S. G. Southerton;citation_title=Association genetics in Corymbia citriodora subsp. variegata identifies single nucleotide polymorphisms affecting wood growth and cellulosic pulp yield.;citation_pages=596-608;citation_volume=195;citation_year=2012;citation_issue=3;citation_pmid=22680066;citation_doi=10.1111/j.1469-8137.2012.04200.x" />
<meta name="citation_reference" content="citation_journal_title=Cellulose;citation_author=S. Djerbi;citation_author=H. Aspeborg;citation_author=P. Nilsson;citation_author=B. Sundberg;citation_author=E. Mellerowicz;citation_title=Identification and expression analysis of genes encoding putative cellulose synthases (CesA) in the hybrid aspen, Populus tremula (L.) × P. tremuloides (Michx.).;citation_pages=301-312;citation_volume=11;citation_year=2004;citation_doi=10.1023/B:CELL.0000046408.84510.06" />
<meta name="citation_reference" content="citation_journal_title=J. Hered.;citation_author=Q. Du;citation_author=B. Wang;citation_author=Z. Wei;citation_author=D. Zhang;citation_author=B. Li;citation_title=Genetic diversity and population structure of Chinese white poplar (Populus tomentosa) revealed by SSR markers.;citation_volume=103;citation_year=2012;citation_pmid=23008443;citation_doi=10.1093/jhered/ess061" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=Q. Du;citation_author=W. Pan;citation_author=B. Xu;citation_author=B. Li;citation_author=D. Zhang;citation_title=Polymorphic simple sequence repeat (SSR) loci within cellulose synthase (PtoCesA) genes are associated with growth and wood properties in Populus tomentosa.;citation_pages=763-776;citation_volume=197;citation_year=2013;citation_pmid=23278184;citation_doi=10.1111/nph.12072" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. J. Eckert;citation_author=A. D. Bower;citation_author=J. L. Wegrzyn;citation_author=B. Pande;citation_author=K. D. Jermstad;citation_title=Association genetics of coastal Douglas fir (Pseudotsuga menziesii var. menziesii, Pinaceae). I. Cold hardiness related traits.;citation_volume=182;citation_year=2009;citation_pmid=19487566;citation_doi=10.1534/genetics.109.102350" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=A. J. Eckert;citation_author=J. L. Wegrzyn;citation_author=W. P. Cumbie;citation_author=B. Goldfarb;citation_author=D. A. Huber;citation_title=Association genetics of the loblolly pine (Pinus taeda, Pinaceae) metabolome.;citation_pages=890-902;citation_volume=193;citation_year=2012;citation_issue=4;citation_pmid=22129444;citation_doi=10.1111/j.1469-8137.2011.03976.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. X. Fu;citation_author=W. H. Li;citation_title=Statistical tests of neutrality of mutations.;citation_pages=693-709;citation_volume=133;citation_year=1993;citation_issue=3;citation_pmid=8454210" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. Geisler-Lee;citation_author=M. Geisler;citation_author=P. M. Coutinho;citation_author=B. Segerman;citation_author=N. Nishikubo;citation_title=Poplar carbohydrate-active enzymes. Gene identification and expression analyses.;citation_volume=140;citation_year=2006;citation_pmid=16415215;citation_doi=10.1104/pp.105.072652" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. C. González-Martínez;citation_author=N. C. Wheeler;citation_author=E. Ersoz;citation_author=C. D. Nelson;citation_author=D. B. Neale;citation_title=Association genetics in Pinus taeda L. I. Wood property traits.;citation_volume=175;citation_year=2007;citation_pmid=17110498;citation_doi=10.1534/genetics.106.061127" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=F. P. Guerra;citation_author=J. L. Wegrzyn;citation_author=R. Sykes;citation_author=M. F. Davis;citation_author=B. J. Stanton;citation_title=Association genetics of chemical wood properties in black poplar (Populus nigra).;citation_pages=162-176;citation_volume=197;citation_year=2013;citation_issue=1;citation_pmid=23157484;citation_doi=10.1111/nph.12003" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=Linkage disequilibrium in finite populations.;citation_pages=226-231;citation_volume=38;citation_year=1968;citation_pmid=24442307;citation_doi=10.1007/BF01245622" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=N. Holland;citation_author=D. Holland;citation_author=T. Helentjaris;citation_author=K. S. Dhugga;citation_author=B. Xoconostle-Cazares;citation_title=A Comparative Analysis of the Plant Cellulose Synthase (CesA) Gene Family;citation_pages=1313-1324;citation_volume=123;citation_year=2000;citation_issue=4;citation_pmid=10938350;citation_doi=10.1104/pp.123.4.1313" />
<meta name="citation_reference" content="citation_journal_title=J Beijing Forestry Univ;citation_author=Z. H. Huang;citation_title=The study on the climatic regionalization of the distributional region of Populus tomentosa.;citation_pages=26-32;citation_volume=14;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. K. Ingvarsson;citation_title=Multilocus patterns of nucleotide polymorphism and the demographic history of Populus tremula.;citation_volume=180;citation_year=2008;citation_pmid=18716330;citation_doi=10.1534/genetics.108.090431" />
<meta name="citation_reference" content="citation_journal_title=BMC evolutionary biology [electronic resource];citation_journal_abbrev=BMC Evol Biol;citation_author=P. K. Ingvarsson;citation_title=Ancestral feeding state of ruminants reconsidered: earliest grazing adaptation claims a mixed condition for Cervidae.;citation_pages=13-13;citation_volume=8;citation_year=2008;citation_pmid=18205907;citation_doi=10.1186/1471-2148-8-13" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=P. K. Ingvarsson;citation_author=N. R. Street;citation_title=Association genetics of complex traits in plants.;citation_pages=909-922;citation_volume=189;citation_year=2011;citation_issue=4;citation_pmid=21182529;citation_doi=10.1111/j.1469-8137.2010.03593.x" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=C. P. Joshi;citation_author=S. Bhandari;citation_author=P. Ranjan;citation_author=U. C. Kalluri;citation_author=X. Liang;citation_title=Genomics of cellulose biosynthesis in poplars.;citation_pages=53-61;citation_volume=164;citation_year=2004;citation_doi=10.1111/j.1469-8137.2004.01155.x" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=I. Keller;citation_author=D. Bensasson;citation_author=R. A. Nichols;citation_title=Transition-transversion bias is not universal: a counter example from grasshopper pseudogenes.;citation_pages=e22-e22;citation_volume=3;citation_year=2007;citation_issue=2;citation_pmid=17274688;citation_doi=10.1371/journal.pgen.0030022" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. Kim;citation_author=V. Plagnol;citation_author=T. T. Hu;citation_author=C. Toomajian;citation_author=R. M. Clark;citation_title=Recombination and linkage disequilibrium in Arabidopsis thaliana.;citation_pages=1151-1155;citation_volume=39;citation_year=2007;citation_issue=9;citation_pmid=17676040;citation_doi=10.1038/ng2115" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. V. Krutovsky;citation_author=D. B. Neale;citation_title=Nucleotide diversity and linkage disequilibrium in cold-hardiness-and wood quality-related candidate genes in Douglas Fir.;citation_volume=171;citation_year=2005;citation_pmid=16157674;citation_doi=10.1534/genetics.105.044420" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=I. J. Kullo;citation_author=K. Ding;citation_title=Patterns of population differentiation of candidate genes for cardiovascular disease.;citation_pages=48-48;citation_volume=8;citation_year=2007;citation_pmid=17626638;citation_doi=10.1186/1471-2156-8-48" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=M. Kumar;citation_author=S. Thammannagowda;citation_author=V. Bulone;citation_author=V. Chiang;citation_author=K. H. Han;citation_title=An update on the nomenclature for the cellulose synthase genes in Populus.;citation_pages=248-254;citation_volume=14;citation_year=2009;citation_issue=5;citation_pmid=19375973;citation_doi=10.1016/j.tplants.2009.02.004" />
<meta name="citation_reference" content="citation_journal_title=Crit. Rev. Plant Sci.;citation_author=L. Li;citation_author=S. Lu;citation_author=V. L. Chiang;citation_title=A genomic and molecular view of wood formation.;citation_pages=213-233;citation_volume=25;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=X. Li;citation_author=H. Wu;citation_author=S. Dillon;citation_author=S. G. Southerton;citation_title=A first-generation microsatellite-based genetic linkage map of the Siberian jay (Perisoreus infaustus): insights into avian genome evolution.;citation_pages=1-1;citation_volume=10;citation_year=2009;citation_pmid=19121221;citation_doi=10.1186/1471-2164-10-1" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Lu;citation_author=S. Zhang;citation_author=T. Shah;citation_author=C. Xie;citation_author=Z. Hao;citation_title=Joint linkage-linkage disequilibrium mapping is a powerful approach to detect quantitative trait loci underlying drought tolerance in maize.;citation_volume=107;citation_year=2010;citation_pmid=20974948;citation_doi=10.1073/pnas.1006105107" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. Lin;citation_author=W. H. Li;citation_title=Evolution of 5′untranslated region length and gene expression reprogramming in yeasts.;citation_volume=29;citation_year=2012;citation_pmid=21965341;citation_doi=10.1093/molbev/msr143" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. Manenti;citation_author=A. Galvan;citation_author=A. Pettinicchio;citation_author=G. Trincucci;citation_author=E. Spada;citation_title=Mouse genome-wide association mapping needs linkage analysis to avoid false-positive Loci.;citation_pages=e1000331-e1000331;citation_volume=5;citation_year=2009;citation_issue=1;citation_pmid=19132132;citation_doi=10.1371/journal.pgen.1000331" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Y. Miyamoto;citation_author=A. Mabuchi;citation_author=D. Shi;citation_author=T. Kubo;citation_author=Y. Takatori;citation_title=A functional polymorphism in the 5' UTR of GDF5 is associated with susceptibility to osteoarthritis.;citation_pages=529-533;citation_volume=39;citation_year=2007;citation_issue=4;citation_pmid=17384641;citation_doi=10.1038/2005" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. L. Morrell;citation_author=D. M. Toleno;citation_author=K. E. Lundy;citation_author=M. T. Clegg;citation_title=Low levels of linkage disequilibrium in wild barley (Hordeum vulgare ssp. spontaneum) despite high rates of self-fertilization.;citation_volume=102;citation_year=2005;citation_pmid=15699350;citation_doi=10.1073/pnas.0409804102" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=S. Myles;citation_author=J. Peiffer;citation_author=P. J. Brown;citation_author=E. S. Ersoz;citation_author=Z. Zhang;citation_title=Association mapping: Critical considerations shift from genotyping to experimental design.;citation_volume=21;citation_year=2009;citation_pmid=19654263;citation_doi=10.1105/tpc.109.068437" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=D. B. Neale;citation_author=O. Savolainen;citation_title=Association genetics of complex traits in conifers.;citation_pages=325-330;citation_volume=9;citation_year=2004;citation_issue=7;citation_pmid=15231277;citation_doi=10.1016/j.tplants.2004.05.006" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=D. B. Neale;citation_author=A. Kremer;citation_title=Forest tree genomics: growing resources and applications.;citation_pages=111-122;citation_volume=12;citation_year=2011;citation_issue=2;citation_pmid=21245829;citation_doi=10.1038/nrg2931" />
<meta name="citation_reference" content="citation_title=Molecular evolutionary genetics;citation_author=M. Nei;citation_year=1987" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. S. Olson;citation_author=A. L. Robertson;citation_author=N. Takebayashi;citation_author=S. Silim;citation_author=W. R. Schroeder;citation_title=Nucleotide diversity and linkage disequilibrium in balsam poplar (Populus balsamifera).;citation_pages=526-536;citation_volume=186;citation_year=2010;citation_issue=2;citation_pmid=20122131;citation_doi=10.1111/j.1469-8137.2009.03174.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Persson;citation_author=H. Wei;citation_author=J. Milne;citation_author=G. P. Page;citation_author=C. R. Somervill;citation_title=Identification of genes required for cellulose synthesis by regression analysis of public microarray data sets.;citation_volume=102;citation_year=2005;citation_pmid=15932943;citation_doi=10.1073/pnas.0503392102" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Persson;citation_author=A. Paredez;citation_author=A. Carroll;citation_author=H. Palsdottir;citation_author=M. Doblin;citation_title=Genetic evidence for three unique components in primary cell-wall cellulose synthase complexes in Arabidopsis.;citation_volume=104;citation_year=2007;citation_pmid=17878302;citation_doi=10.1073/pnas.0706592104" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=T. Quesada;citation_author=V. Gopal;citation_author=W. P. Cumbie;citation_author=A. J. Eckert;citation_author=J. L. Wegrzyn;citation_title=Association mapping of quantitative disease resistance in a natural population of loblolly pine (Pinus taeda L.).;citation_volume=186;citation_year=2010;citation_pmid=20628037;citation_doi=10.1534/genetics.110.117549" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. L. Remington;citation_author=J. M. Thornsberry;citation_author=Y. Matsuoka;citation_author=L. M. Wilson;citation_author=S. R. Whitt;citation_title=Structure of linkage disequilibrium and phenotypic associations in the maize genome.;citation_volume=98;citation_year=2001;citation_pmid=11562485;citation_doi=10.1073/pnas.201394398" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. F. Resende;citation_author=P. Muñoz;citation_author=M. D. Resende;citation_author=D. J. Garrick;citation_author=R. L. Fernando;citation_title=Accuracy of genomic selection methods in a standard data set of Loblolly Pine (Pinus taeda L.).;citation_pages=695-708;citation_volume=188;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Genet. Res.;citation_author=K. Ritland;citation_title=Estimators for pairwise relatedness and individual inbreeding coefficients.;citation_pages=175-185;citation_volume=67;citation_year=1996;citation_doi=10.1017/S0016672300033620" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=J. Rozas;citation_author=J. C. Sa’nchez-Delbarrio;citation_author=X. Messeguer;citation_author=R. Rozas;citation_title=DnaSP, DNA polymorphism analyses by the coalescent and other methods;citation_pages=2496-2497;citation_volume=19;citation_year=2003;citation_issue=18;citation_pmid=14668244;citation_doi=10.1093/bioinformatics/btg359" />
<meta name="citation_reference" content="citation_journal_title=Can. J. For. Res.;citation_author=L. R. Schimleck;citation_author=P. S. Kube;citation_author=C. A. Raymond;citation_title=Genetic improvement of kraft pulp yield in Eucalyptus nitens using cellulose content determined by near infrared spectroscopy.;citation_pages=2363-2370;citation_volume=34;citation_year=2004;citation_doi=10.1139/x04-119" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=T. R. Sexton;citation_author=R. J. Henry;citation_author=C. E. Harwood;citation_author=D. S. Thomas;citation_author=L. J. McManus;citation_title=Pectin methyltransferase genes influence solid wood properties of Eucalyptus pilularis.;citation_volume=158;citation_year=2012;citation_pmid=22052017;citation_doi=10.1104/pp.111.181602" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=D. Shriner;citation_author=L. K. Vaughan;citation_author=M. A. Padilla;citation_author=H. K. Tiwari;citation_author=S. M. Williams;citation_title=Problems with genome-wide association studies.;citation_pages=1840-1842;citation_volume=316;citation_year=2007;citation_issue=5833;citation_pmid=17605173" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=G. T. Slavov;citation_title=Genome resequencing reveals multiscale geographic structure and extensive linkage disequilibrium in the forest tree Populus trichocarpa.;citation_pages=713-725;citation_volume=196;citation_year=2012;citation_issue=3;citation_pmid=22861491;citation_doi=10.1111/j.1469-8137.2012.04258.x" />
<meta name="citation_reference" content="citation_journal_title=Annual review of cell and developmental biology;citation_journal_abbrev=Annu Rev Cell Dev Biol;citation_author=C. Somerville;citation_title=Cellulose synthesis in higher plants.;citation_pages=53-78;citation_volume=22;citation_year=2006;citation_pmid=16824006;citation_doi=10.1146/annurev.cellbio.22.022206.160206" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. Somerville;citation_author=S. Bauer;citation_author=G. Brininstool;citation_author=M. Facette;citation_author=T. Hamann;citation_title=Toward a Systems Approach to Understanding Plant Cell Walls;citation_pages=2206-2211;citation_volume=306;citation_year=2004;citation_issue=5705;citation_pmid=15618507;citation_doi=10.1126/science.1102765" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=D. Song;citation_author=J. Shen;citation_author=L. Li;citation_title=Characterization of cellulose synthase complexes in Populus xylem differentiation.;citation_pages=777-790;citation_volume=187;citation_year=2010;citation_issue=3;citation_pmid=20546138;citation_doi=10.1111/j.1469-8137.2010.03315.x" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=A. V. Spokevicius;citation_author=S. G. Southerton;citation_author=C. P. MacMillan;citation_author=D. Qiu;citation_author=S. Gan;citation_title=beta-tubulin affects cellulose microfibril orientation in plant secondary fibre cell walls.;citation_pages=717-726;citation_volume=51;citation_year=2007;citation_issue=4;citation_pmid=17605757;citation_doi=10.1111/j.1365-313X.2007.03176.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. Stephens;citation_author=P. Scheet;citation_title=Accounting for decay of linkage disequilibrium in haplotype inference and missing-data imputation.;citation_pages=449-462;citation_volume=76;citation_year=2005;citation_issue=3;citation_pmid=15700229;citation_doi=10.1086/428594" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. Sterken;citation_author=R. Kiekens;citation_author=J. Boruc;citation_author=F. Zhang;citation_author=A. Vercauteren;citation_title=Combined linkage and association mapping reveals CYCD5;1 as a quantitative trait gene for endoreduplication in Arabidopsis.;citation_volume=109;citation_year=2012;citation_pmid=22392991;citation_doi=10.1073/pnas.1120811109" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=B. Stich;citation_author=A. E. Melchinger;citation_title=Comparison of mixed-model approaches for association mapping in rapeseed, potato, sugar beet, maize, and Arabidopsis.;citation_pages=94-94;citation_volume=10;citation_year=2009;citation_pmid=19250529;citation_doi=10.1186/1471-2164-10-94" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. D. Storey;citation_author=R. Tibshirani;citation_title=Statistical significance for genomewide studies.;citation_volume=100;citation_year=2003;citation_pmid=12883005;citation_doi=10.1073/pnas.1530509100" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=S. Suzuki;citation_author=L. Li;citation_author=Y. H. Sun;citation_author=V. L. Chiang;citation_title=The cellulose synthase gene superfamily and biochemical functions of xylem-specific cellulose synthase-like genes in Populus trichocarpa.;citation_volume=142;citation_year=2006;citation_pmid=16950861;citation_doi=10.1104/pp.106.086678" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=P. M. Szyjanowicz;citation_author=I. McKinnon;citation_author=N. G. Taylor;citation_author=J. Gardiner;citation_author=M. C. Jarvis;citation_title=The irregular xylem 2 mutant is an allele of korrigan that affects the secondary cell wall of Arabidopsis thaliana.;citation_pages=730-740;citation_volume=37;citation_year=2004;citation_issue=5;citation_pmid=14871312;citation_doi=10.1111/j.1365-313X.2003.02000.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=F. Tajima;citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.;citation_pages=585-595;citation_volume=123;citation_year=1989;citation_issue=3;citation_pmid=2513255" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=J. Dudley;citation_author=M. Nei;citation_author=S. Kumar;citation_title=MEGA4: molecular evolutionary genetics analysis (MEGA) software version 4.0.;citation_volume=24;citation_year=2007;citation_pmid=17488738;citation_doi=10.1093/molbev/msm092" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. G. Taylor;citation_author=R. M. Howells;citation_author=A. K. Huttly;citation_author=K. Vickers;citation_author=S. R. Turner;citation_title=Interactions among three distinct CesA proteins essential for cellulose synthesis.;citation_volume=100;citation_year=2003;citation_pmid=12538856;citation_doi=10.1073/pnas.0337628100" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. R. Thumma;citation_author=B. A. Matheson;citation_author=D. Zhang;citation_author=C. Meeske;citation_author=R. Meder;citation_title=Identification of a cis-acting regulatory polymorphism in a Eucalypt cobra-like gene affecting cellulose content.;citation_volume=183;citation_year=2009;citation_pmid=19737751;citation_doi=10.1534/genetics.109.106591" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Thumma;citation_author=M. F. Nolan;citation_author=R. Evans;citation_author=G. F. Moran;citation_title=Polymorphisms in cinnamoyl coa reductase (CCR) are associated with variation in microfibril angle in Eucalyptus spp.;citation_volume=171;citation_year=2005;citation_pmid=16085705;citation_doi=10.1534/genetics.105.042028" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=B. Thumma;citation_author=S. Southerton;citation_author=J. Bell;citation_author=J. Owen;citation_author=M. Henery;citation_title=Quantitative trait locus (QTL) analysis of wood quality traits in Eucalyptus nitens.;citation_pages=305-317;citation_volume=6;citation_year=2010;citation_doi=10.1007/s11295-009-0250-9" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=J. X. Tian;citation_author=Q. Z. Du;citation_author=M. Q. Chang;citation_author=D. Q. Zhang;citation_title=Allelic variation in PtGA20Ox associates with growth and wood properties in Populus spp.;citation_pages=e53116;citation_volume=7;citation_year=2012;citation_pmid=23300875;citation_doi=10.1371/journal.pone.0053116" />
<meta name="citation_reference" content="citation_journal_title=Theor. Popul. Biol.;citation_author=G. A. Watterson;citation_title=On the number of segregating sites in genetical models without recombination.;citation_pages=188-193;citation_volume=7;citation_year=1975" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=J. L. Wegrzyn;citation_author=A. J. Eckert;citation_author=M. Choi;citation_author=J. M. Lee;citation_author=B. J. Stanton;citation_title=Association genetics of traits controlling lignin and cellulose biosynthesis in black cottonwood (Populus trichocarpa, Salicaceae) secondary xylem.;citation_pages=515-532;citation_volume=188;citation_year=2010;citation_issue=2;citation_pmid=20831625;citation_doi=10.1111/j.1469-8137.2010.03415.x" />
<meta name="citation_reference" content="citation_journal_title=Transcult. Psychiatry;citation_author=C. S. Weickert;citation_author=Y. Tiwari;citation_author=P. R. Schofield;citation_author=B. J. Mowry;citation_author=J. M. Fullerton;citation_title=Schizophrenia-associated HapICE haplotype is associated with increased NRG1 type III expression and high nucleotide diversity.;citation_pages=e104;citation_volume=2;citation_year=2012;citation_doi=10.1038/tp.2012.25" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=G. A. Wray;citation_author=M. W. Hahn;citation_author=E. Abouheif;citation_author=J. P. Balhoff;citation_author=M. Pizer;citation_title=The evolution of transcriptional regulation in eukaryotes.;citation_volume=20;citation_year=2003;citation_pmid=12777501;citation_doi=10.1093/molbev/msg140" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=L. Wu;citation_author=C. P. Joshi;citation_author=V. L. Chiang;citation_title=A xylem-specific cellulose synthase gene from aspen (Populus tremuloides) is responsive to mechanical stress.;citation_pages=495-502;citation_volume=22;citation_year=2000;citation_issue=6;citation_pmid=10886769;citation_doi=10.1046/j.1365-313x.2000.00758.x" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=L. Xie;citation_author=C. Yang;citation_author=X. Wang;citation_title=Brassinosteroids can regulate cellulose biosynthesis by controlling the expression of CESA genes in Arabidopsis.;citation_volume=62;citation_year=2011;citation_pmid=21617247;citation_doi=10.1093/jxb/err164" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. V. Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=B. Zhang;citation_author=L. Deng;citation_author=Q. Qian;citation_author=G. Xiong;citation_author=D. Zeng;citation_title=A missense mutation in the transmembrane domain of CESA4 affects protein abundance in the plasma membrane and results in abnormal cell wall biosynthesis in rice.;citation_pages=509-524;citation_volume=71;citation_year=2009;citation_issue=4-5;citation_pmid=19697141;citation_doi=10.1007/s11103-009-9536-4" />
<meta name="citation_reference" content="citation_journal_title=Can. J. For. Res;citation_author=D. Q. Zhang;citation_author=Z. Y. Zhang;citation_author=K. Yang;citation_title=QTL analysis of growth and wood chemical content traits in an interspecific backcross family of white poplar (Populus tomentosa × P. bolleana) × P. tomentosa.;citation_pages=2015-2023;citation_volume=36;citation_year=2006;citation_doi=10.1139/x06-103" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=D. Q. Zhang;citation_author=Q. Z. Du;citation_author=B. H. Xu;citation_author=Z. Y. Zhang;citation_author=B. Li;citation_title=The actin multigene family in Populus: organization, expression and phylogenetic analysis.;citation_pages=105-119;citation_volume=284;citation_year=2010;citation_issue=2;citation_pmid=20577761;citation_doi=10.1007/s00438-010-0552-5" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=D. Q. Zhang;citation_author=X. H. Yang;citation_author=Z. Y. Zhang;citation_author=B. Li;citation_title=Expression and nucleotide diversity of the poplar COBL gene.;citation_pages=331-344;citation_volume=6;citation_year=2010;citation_doi=10.1007/s11295-009-0252-7" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=R. Zhong;citation_author=W. H. Morrison;citation_author=G. D. Freshour;citation_author=M. G. Hahn;citation_author=Z. H. Ye;citation_title=Expression of a Mutant Form of Cellulose Synthase AtCesA7 Causes Dominant Negative Effect on Cellulose Biosynthesis;citation_pages=786-795;citation_volume=132;citation_year=2003;citation_issue=2;citation_pmid=12805608;citation_doi=10.1104/pp.102.019331" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=R. Zhong;citation_author=T. Demura;citation_author=Z. H. Ye;citation_title=SND1, a NAC domain transcription factor, is a key regulator of secondary wall synthesis in fibers of Arabidopsis.;citation_volume=18;citation_year=2006;citation_pmid=17114348;citation_doi=10.1105/tpc.106.047399" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Allelic Variation in a Cellulose Synthase Gene (PtoCesA4) Associated with Growth and Wood Properties in Populus tomentosa" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="twitter:description" content="Lignocellulosic biomass from trees provides a renewable feedstock for biofuels, lumber, pulp, paper, and other uses. Dissecting the mechanism underlying natural variation of the complex traits controlling growth and lignocellulose biosynthesis in trees can enable marker-assisted breeding to improve wood quality and yield. Here, we combined linkage disequilibrium (LD)-based association analysis with traditional linkage analysis to detect the genetic effect of a Populus tomentosa cellulose synthase gene, PtoCesA4 . PtoCesA4 is strongly expressed in developing xylem and leaves. Nucleotide diversity and LD in PtoCesA4 , sampled from the P. tomentosa natural distribution, revealed that PtoCesA4 harbors high single nucleotide polymorphism (SNP) diversity (πT = 0.0080 and θw = 0.0098) and low LD ( r2 ≥ 0.1, within 1400 bp), demonstrating that the potential of a candidate-gene-based LD approach in understanding the molecular basis underlying quantitative variation in this species. By combining single SNP, multi-SNP, and haplotype-based associations in an association population of 460 individuals with single SNP linkage analysis in a family-based linkage populations (1200 individuals), we identified three strong associations (false discovery rate Q &lt; 0.05) in both populations. These include two nonsynonymous markers (SNP49 associated with α-cellulose content and SNP59 associated with fiber width) and a noncoding marker (SNP18 associated with α-cellulose content). Variation in RNA transcript abundance among genotypic classes of SNP49 was confirmed in these two populations. Therefore, combining different methods allowed us to examine functional PtoCesA4 allelic variation underlying natural variation in complex quantitative traits related to growth and lignocellulosic biosynthesis." />
<meta name="og-title" property="og:title" content="Allelic Variation in a Cellulose Synthase Gene (PtoCesA4) Associated with Growth and Wood Properties in Populus tomentosa" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/11/2069" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Lignocellulosic biomass from trees provides a renewable feedstock for biofuels, lumber, pulp, paper, and other uses. Dissecting the mechanism underlying natural variation of the complex traits controlling growth and lignocellulose biosynthesis in trees can enable marker-assisted breeding to improve wood quality and yield. Here, we combined linkage disequilibrium (LD)-based association analysis with traditional linkage analysis to detect the genetic effect of a Populus tomentosa cellulose synthase gene, PtoCesA4 . PtoCesA4 is strongly expressed in developing xylem and leaves. Nucleotide diversity and LD in PtoCesA4 , sampled from the P. tomentosa natural distribution, revealed that PtoCesA4 harbors high single nucleotide polymorphism (SNP) diversity (πT = 0.0080 and θw = 0.0098) and low LD ( r2 ≥ 0.1, within 1400 bp), demonstrating that the potential of a candidate-gene-based LD approach in understanding the molecular basis underlying quantitative variation in this species. By combining single SNP, multi-SNP, and haplotype-based associations in an association population of 460 individuals with single SNP linkage analysis in a family-based linkage populations (1200 individuals), we identified three strong associations (false discovery rate Q &lt; 0.05) in both populations. These include two nonsynonymous markers (SNP49 associated with α-cellulose content and SNP59 associated with fiber width) and a noncoding marker (SNP18 associated with α-cellulose content). Variation in RNA transcript abundance among genotypic classes of SNP49 was confirmed in these two populations. Therefore, combining different methods allowed us to examine functional PtoCesA4 allelic variation underlying natural variation in complex quantitative traits related to growth and lignocellulosic biosynthesis." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
    <title>Allelic Variation in a Cellulose Synthase Gene (PtoCesA4) Associated with Growth and Wood Properties in Populus tomentosa | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"UmKfDdxfwie6gcUonJ_OZ530JL3uWYn8oHjJOFXP7Go","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459573","apath":"\/ggg\/3\/11\/2069.atom","pisa":"ggg;3\/11\/2069","ac":{"\/ggg\/3\/11\/2069.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/11\/2069.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/11\/2069"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/11\/2069"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/11\/2069":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022q5fyWvY6kJSPLQxaX2oH22x1206kUphxhRSUiYlym1gYikC8CcSHMk1IMHAET7OgJAd2ABemyfOH3BzzKLcVF17NfUnXr62Wb+P\\\\\\\/TqZCOhswiUnnAdMvLJ+yvvOH4j3oRdb4cO5BYHFXus16zY16KHUc2661V0qxuVhqKdFRnNfEkKiixqvpsTOLEH6kLRCeEvmWiEW2pz2xq2CqjqvsozNcwKQ1olSDf1gqvKZZS\\\\\\\/QUY6GOx5Sewm7yY5jwMIW3IQlgq0mxTA2FM60uYujIzmViy2Uzw238YpqT1ZhMgt5qU9ZJrC6YMdKGiUwm0iKjJBAOvNxC751XOPuMWJ614rGrI7afUAEqVML9ZtXE3TTFR+tY2K9a4Tar+V+5\\\\\\\/ueATLQEQm5T\\\\\\\/Ggs8Z0EO\\\\\\\/JI0XU9FTKthMfRMZTadT0jlLINPBZ6kyxuKDfm7tf4pbkIi4sfXKz1ZHqGdhU35my3cIuCYCXd6S7C75bcMm9xCTaQ1SxoU9H3kdT2qI6dOR3zh\\\\\\\/ZbDkgtNJzpnp7FXsv0g2p4JQDpiCWIZJJpaVl5WCxhpgFVqciMMfaSKfgPPyvhF9si25bepIngHeXlQGd5xr\\\\\\\/25aP6Wq4iNy+c9E2A6JGoKwY4qWEtkbLxgUgGX8j0VlXN9yNHWy\\\\\\\/oPT+7hDPWJ0E0VEn+BU8LmYWyt8Xp8jiJm9ik\\\\\\\/Mcc0TSTe9sPcb3FWBMuvEWMoL\\\\\\\/4gTmuwbZ+MWQh7iyLpPhtzcOO4AcOzK9iYs9l6WzsHKMssDoG9KBluLIZqxMjkS8oZGeG2mjtHz4Gu7vx3od\\\\\\\/CuRNSuhfTEyzX\\\\\\\/hdwXglqbNulbJNOH8IHsJrweQ6ExrDh15Z+Oq1PEYP6iA7PB93v+T6nQPD+ANko0oiO7EUWhCraTKC\\\\\\\/kciXLcyQPxvRzi9Z0cLoUdYhYfDdwjFHkBCG1VMlpDv7nJVGIIlfuUzGUz5UCWeb4EO4Iz1icqZohhEJ\\\\\\\/4eKDhoG4SBxwSPdp4o\\\\\\\/0jHu6ml6NcTj\\\\\\\/NyYeN1oFSn2OvdPoijtKUItJIPY367jNl1HbEi6EHS4GqcxXC9VTpKZMT+2JSKZH2dN2Sy+dDucUmDYzXI5ZLB657r+N\\\\\\\/R8cB6HT5GJH0zM8r7PEfU9\\\\\\\/Eu3vPZd4MDnWk6UAkUaYngwM\\\\\\\/QkJvo09gRfG6e6Etucgsu\\\\\\\/2LM6ZN9OfuZ9fsmYrpApHBkG\\\\\\\/FTzH3xVrjodh86Tq9wdqVavZkTn66YaBeuNDktFYSL1wu3\\\\\\\/So\\\\\\\/EbSWOy8RR6e7GMXyuvgTwp4X3ZYtYk22V+28\\\u0022,\\\u0022iv\\\u0022:\\\u0022yzL6pyA4U6Ji37D\\\\\\\/rmA8qg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002243b1f75a29443b1c2375135f66f9d7accf43e139573d29690563a4143611b393\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022i0OcSpTaxTXZVGNGdaUSPj+zJQ7DfKVMx\\\\\\\/d74gOo7JYNQ0bZENQcIkgsaal01tWbzv0cRmWEtOl8exU\\\\\\\/Aq\\\\\\\/dyRfubbpDJ\\\\\\\/TvERgPdTT9yJxyjrywrvlAygvLM2K5Gakv2LdYk3K1EtBsXdqqItl\\\\\\\/ERXG1cNJAgrxB9pZJ49MZNhs0gZ9PKc0stL9IRXQtKSO7tfuapCrVcbzYff\\\\\\\/t7QWEtZH7PGCf7Q5huAVa1uH8aa\\\\\\\/hEIJpWY7cvU4M1p+gIam\\\\\\\/i8fwzjga7UQrObOihRJ+7Dj\\\\\\\/U94GToQADDifH9XpHKnbfXlSh3RtjJks0cveLSSB\\\\\\\/002QpYhi5aMdul44clKkhQQ63dG7tN8w0q8+e7u2K9ZQqXkGxvCis43h8AmUIyUG1+48ndwy5xoChh\\\\\\\/F7LfbkvCXBKhjsbL3DbRc\\\\\\\/DaqZ+SMTyLJ4wnROlqK6h6dPE1q2Yhxxkf3g0f6rYQPUUMxyVJqQkFyA3lPilpOr7BnL983\\\\\\\/be5vS8D+XXcr5YGK+O9km+G4sc8yStE+00uHqeLCY59hcpwl21aZIBOv5vFiXVSdcozXktaCLfxGJvvVfnfS5tVyCn2iGWaSZtISk4S8OdFlklugIHpuojG5WzTlniFDIBHfnlcEcw1BLjz5rr+XuKeZVDhokqmJY1KbDGJeN9xrZnPD6SoRELt4CU9g9wbiQUTvErU82ZMeq74fLMgYG7c8i8U0zaBcZP89KFwzGygDZKwLfkAk3SfmWO4My4mwk5Fpla5T4RHnOHlkb835EuNC0PYlXr3H19o\\\\\\\/9VCnJ7R+Kn6uKqXi1L4yv0Bwv9U6D2ujBGXMReC1LD0TyzsKsD6WxWQMKkdk3uLEoGuV2IrxvGuSW9pVhM33ymCQNnmWT6IeqgoqJJ8sGA5zCjAzTKo6rGDlTOy5QzqUHVswCS3HfHLD\\\\\\\/sxelw\\\\\\\/OCfaXnduV6GvmIQ8hBfnc8im1HUxU8m+tAtwWpRuLkTrJXOnuWfHHOClICcrAr9522kJXUJhdG6G1YT0mJX8fzaKqReFbGHWl1jG4RpDfsPSfcGeLxMza5PJCVyps4+mVVNOCIpj0eXDaQZx3aEXcnvYk53I1ODgS7ZPBax5NGQLwQio3KWcOz23T0N1CaEv7\\\\\\\/Dbyw6jxlo1HzO3xCFl0iS6viLfEUSs\\\\\\\/vKEANoF9TkWdwzp3RQGVMFtS\\\\\\\/bqNkST3HabxWPofH7wyurgH1Vo84JBCB2TTE9tabdA8PKhnCMNUq17ndJ+uyCnmTfcCH6Q6sBHYH0Z1YX8ymv\\\\\\\/AoseMKGS2qLi0UrX8WRrfNC88XIdIoOh8L98jAXoU+oM9CRzX5jlaDAaHssfJLp7xQpMi8\\\\\\\/QEWCCXzSSoDmcy2De+GI3qRWObsPGExWSYwXqHqyqvqr+unHpKUIHjO0o4+bbnADQAv4YVWMF7vg8E+NL5DK9vFrp5B3CWIf1HUMnRd5hPc1fYMHvtFXTTS0fbYAaIL6M4zIyA3l4E7WVsQ+nuX6Qkjur5gF6lSQ97U6V2GhP\\\\\\\/oTe+72TncXFSNmDMP3aGDgG3bHfWDGMvEiECcJthctAh38SrN4Z73IxmFjzBVRbZbQy9exAihsCzaDqlvLKK\\\\\\\/J5jejYpI6MXd1r4eyKkY76mc8KfZkS8HGNXeL705vPTVGQkYKdW2BtzX\\\\\\\/4gqP2ii3w+U1nTlMSQXKoLXDJ4xOiI\\\\\\\/SxdvULkJ4coRN5SPzPj3OZFsuXiNTD4tFz5x2W4xUY38M3osI1DYyTaDFXGry3evw1IdbQVhlAZJWRJjFL1UhAokhG5VmyWlgdyano0ygII8P9LT1c1PiNG\\\\\\\/UHZB6FoIKwBa4ghfD+bAiVodiiXy6iLkOZiTG8AKfFOU5y9i7HVqshTExKKp2gB19V0iKQN8MRRXj8gyWkl8oOGlw\\\\\\\/9EWCgqpdcVp72YQg6+\\\\\\\/s1H21Ofs1He+XUOlYvB0XWTw8Qm5l\\\\\\\/hEsFZNbPlzT5Kg2DW+ugj2raAuBXOpnD8qb9qJruku5JQG66D8trbnODak+\\\\\\\/8IoTa+Baks7TGF21YtTFr\\\\\\\/1g3itng\\\\\\\/DVTynk15Uv+jx12724ta4yLQE7N+6AQbPu\\\\\\\/+xewcEUJjb5BJmlZXqR01pzlPMqugYVQXotq+kTYz0H6R\\\\\\\/TtkTiGADSVGRbqjrHmCnjHNpWz5\\\\\\\/O9y4ywRRkFpjFG+UGeaVsat6dxozoDpVdfgqOHKxd12XwO0EQjSWYUsMT0uJAg2py3f+MOvjyfhIFttjJVCMtm4Pe537OnLsCyunu\\\\\\\/BZKXPnSUsok\\\\\\\/ScFTmmu7HW5FKt+YqNvWXwEeUQSb8ynxu7yOxTHXBV+E0FieJhSn2fUee8ZdQs5CR1L8C\\\\\\\/VHLPhONhNTdK7GAdmcQNXCf6rTcOWJ2NdBGlBb966oPyVHNoxPh+v+VCykUKO9d\\\\\\\/1S+sa6mdb7jqvnt9I\\\\\\\/sfAUXRIjrwT2D97De\\\\\\\/XIFls+\\\\\\\/JaxpmKni9aIcXS5zZFaOExWzErPBwApWA7QkHCMsXHFUJfZLGdSrdPU43difj93ID5rlrMXQQ4PeIprOh3t89rmCad1A6EpHL5GZCgJ+paSQ3CJH8StsvqRqmfoHnoU1R3oxAVuDAEINi+K6pq0vE4xHoAMdZT8blRZJrYLpEzfUvjSUT2irlpsFdTXsFCd7NxfZNPY5oleOfS0QU70uxfv4AtpFPSZRKlC\\\\\\\/RZWUoO9KuT5k9Wddt60zMI3u8aB4NkTWOqvbYdi74QJBbjXo6aDy99bpzjYCZeY6JtycA2yVFW0jLFnFlIXQ5iHgKscS1tZK0cwbs0hL8YtYyktPiY9Wd1amHp2\\\\\\\/ZU313rGDB+3OuBuLBcbIhQfXAm5BvP2aF2hAe8WPqH+pM82mKeVK1zD9IWElJRaJtU1fuBZ1mMlv3LuSPINNUhqivUKy8GIV2iuad\\\\\\\/1q8XqZRLKCkiSdgu61Rtw\\\\\\\/Uta6m2FcQRiB2HErhe2pO97nZW5oNUSs9RG7wm1Rph+EW4A\\\\\\\/e7STIsDFtSq07ObHrjWywpfoLgq5KAgDsQ6onXFE7CAxLEgR8RxMwk1CUytZxMF+8iNlo9ojQ0wJPIdw8yoAa5rnAatMh5ZJuUSSVgXsF45SLpOLvQsuf44cj1Z6gCaZLbYKXrBWxyqhPX3EhFYQTh+dn8B5zYS8IiGdROl1wZW9BdsZjTkAkKSaUeSZH6WsjvDNTD6B21ry1MlVHNGI82xmoSPU3VVM6MvTH5ba0DesNYh0C96hihzY96t8UDeuBVk0eIpeL1Czg4WPUdi9GIs\\\\\\\/Jd9dIFPk+UG2RaR+CD0H1k0V5574t0rQlLzHxKzvG7RXInkY4wM4PWNYv1lFjnAegXXZEVRCVmG\\\\\\\/YieGeoblIRb3WDvN8iUYtb34jGh13gJCVxhDK\\\\\\\/7QAK0iH1meOwALN\\\\\\\/2MiBt8dpB7bUNv4UyhQ+cthZc4qmq8zQoOmXvqEYjzYlAUKt+ruBkKDknRLJJrGAk6SA1Kb77Zo8PwVLa3XMQLoHHpb1Z\\\\\\\/OE8PK\\\\\\\/QRbA073p605C5ZUSMZEGyda+BIeBouu53+eFXMoHFl60sr3s1W6XCHUMoFnfkm8s\\\\\\\/KzrSZ7TrF6KsQCeM57oDhRbUeu46YgPk13cGuD2t8envgoLKF1DWg\\\\\\\/amEC28F4yISO7RDz3P+HsE5Y8g0\\\\\\\/PTKNz7JN7khEFKGSP5sXXALIpDm50IREABGa0KL8WZSk8JDYLzqTYp71eLB8bov0PfBjX3XWY66jl79qigpdz3k\\\\\\\/LjTuD+zUfRt0MAF6IFXEeBREpDjwIpJvuT4VbSwxcYSSE41gjbEZCxxn6qLJlrElqH0MbN8dwux+pFHkUGohp0\\\\\\\/bWvGkbkq7lYEGWdHORCZU+PqP288Jynu9bF1TphqL7qGh5zkysBPtSpfkUQrcGFAif\\\\\\\/vmMOI2jymTy+HUouvAhATinl+gDFEFEizCWpE\\\\\\\/dfcwP4Mm\\\\\\\/+nS00oJ6MAIZdepTzACEe7iU6m1dwzhhAscLu+ZmWkc2efTDlXW11yMWz4x7zD\\\\\\\/tczoqlMAdO+XqaBincwbK\\\\\\\/mLd1OZdD25vEBvx\\\\\\\/7alzTH6+uxXCYgXnwWaul6HJotsBMYG2RoFKK6ljiIXbRpXTiiGRRejgpJj64HijVRaFzeUbJs\\\\\\\/G9tWtFasI5IYfS+iT6tlyPx9DcPPIgQ93Fa+eN6k6LAWlMsMrAaWqH\\\\\\\/lOsyvOMRqk8l1KLqm4PamdoP\\\\\\\/g2mbzl06ZvbiFfCr4r77Ye\\\\\\\/HwtQ0zi\\\\\\\/DXxdgAHA7SVijuzRPyM5mUtEMjP+2ku0n\\\\\\\/IPugny8Ui89Baj5MWjcAqDBfb13W4LkO5aqgafHXP2q9d+TnuLh4k9x22f+q+dnWICx9ahlj9enoYryo4Uvl3XM0FEp93KPvr4tgsGn0FkVE6hJtOYbJUAf1+LLV23Y9DwTLnVqM\\\\\\\/1dkJfLslSJjTbs8DQQhIsYW6uN0r+MWj0rNfFg+\\\\\\\/drmQFdBGnabu9TTh+qevPBXaGlgdrKweXGHm9EhjjawPKZlHZKwx8M51Og8DEzShUlFSLShjGHbd5CuUTlJgsytxHOGUse6G9m6nbW5teesEbE4DXpwRAk2VndqlyCGXhN3cAOKr4JnFppUqTpNMAkwQZga567Lre1+L2l1Vn7tjB0fMW7VGv7dmEFVbk3W3l3yeGbAgPUkABsa2WlAlmt34QBsu6HkoZgZGtlq+JZWSEHuiKyZuU3PqihBxyhbGusBOQA8UCXdcWkpZlzhHqNmskyxJRk+\\\\\\\/k0oEnomhMXBf+9LBRiXtVpo5ZIQFjaFKYslDNRyFguQPBvEYlaZvo5s15Km8X5K3CA7n1OKgUIYpusH7TMJtK5itcYfk61abNtjLZPCQeMZWcStBVGuSrtBRMARyfL0OPSwVTntJCX2\\\\\\\/jcXyrADUDP0ruyT93XMgez9UDBuK6ezY0UVbKNxY4WI9B2dmsEGW24WbpY7s1iHi6BPeOuFq6ggtgpVm19XMPsmngf1mnZsyuASvh8zudbnKXWviIBOqy4oMP+dfpPCReN5j46NbCPcE6Un8XDBc5NB5U1+w4D+MA12NmU+Rsfr77IyuYyVChkZRkWIK22PMKSRZn2CAaTvzh9qD3y4ZO+oTbgePziHmZ\\\\\\\/R4S4dNq8p0WjK\\\\\\\/Evj2U\\\\\\\/fHpoWQ7jYXUiu89BvlxXtu9uS03GQtHF9KCSj8CPVG7jzG7Z1QW6low5y2pmLNWM0NsZwnS324\\\\\\\/N6pka9XqfwB5HfB\\\\\\\/4Xo4hsBAdxbE2dHdO2ozgwrtT83YIO5McbD1azP6WRFQR0amSKlU8sKy2KdevMbQ+ymA01IsLu4XT8HhwtPQF7q12UdhNuDV2pZxG5lmqYtHZaRjY7vCVX2sYFH4PZUDqFvT3ZyUrN1SLwZNccK6bQ+sAATrSMafVYYHc8uPSqwv1vGE7ZfZjD7NztPSrxpHSSn34\\\\\\\/hez6tiONbeovVFi3DLfZpgd\\\\\\\/f60T6FqjlvXt91CzkRQVbHvWu84MgVhb0KRYCBGIL+QyX4tz99oY\\\\\\\/JCqk9ZWp4spc1zFUlrKUBAA8cnVUT8quAUc4pfoqnYqWHZuv+sxabl2VFTzaSJMz1Qd1RO4QAxFqbqZL38cfjDt3xmqye9vQRQNpyrzOgEkEPAiK8Rd\\\\\\\/7yULYyD+lNeNd+GlhNq4vopuDAOZweCZl5QumH6JiprILW7gBwFSXXqRqj8OXly5micIm4VQE\\\\\\\/pePlGHq5GYALX42CApM0rA\\\\\\\/qdfFWaCawU9dtjF53TsgiGrJ6VKQwt2ZCAxC152k+aHLf9dccjhGugqiIPT9qGZgl8ImhJs8pVKy0N6FFdbyZ9Kw47ZOgFz40N\\\\\\\/wnjudfkI6DOmNdkq+jbCPpTH\\\\\\\/FRpWPHKXX3jkXPg\\\\\\\/MKb5wMK61NCFv+9R1yD3KbtLdWHRWggLtvEIT4\\\\\\\/yfzwMjw3TN\\\\\\\/dAPGmoM76QiAlvgVKy25IF2gvew1ztZc4C5jUHt9\\\\\\\/KBkK8jUhgvEXjMVjxBAWarAZ1EY4WvVl+JTN3VnC0cQxmZ2r36Op06hWgz9HGQf+lDZMnaaixCEMARtTnynpBpCuUZIFypHuEL4uoL\\\\\\\/W9Hed7uH9+xtGWysJwbpYREphF7txy50lYhaIeT+RtJMFYgRTN6fn9vjxEoLvqD5Sw0viBTMW1pAl9F00odRqzNOAxYemWWP1CW2+jLVqgydPSz6zRBwMfQgCBh5pR5WVeZM7ktFRhpMwsdNcnpot3dnbVhRBmV3wVks9NbHejkUW5ZpXEIFz9\\\\\\\/fmpxfItVA5zU1TGSYOURjPuQo1LIBmX1Pd6IESeYrSh9c9K3NIua2JuH42hFr2ntoH+FnVw8wH23fvkUI976jSununkkfsL9JHioX6TjrsIFO71s6vesbutPxjmfPp0wcQJeKppoQyR+8vK+9WdWUqfzAYi9r0i570CS1P2iUMQ55aa\\\\\\\/5GIz6jLFkiNO0GYIu8iuQIDyaeEj5fq6fEEBGJ1dwUIZDbBxk6bUrURDk3KzC+LzUynkyN7lDICDGYt6QfmXo3MgL18SYqpNtf6Smh6QLynQ93F8VjlZMJ3KAAg6sVk3tJzqYksCVILyyIF5qhMwznwKP1tDDtagJOfcERh6Omeralg3bpkyMGStKgn65lMlrr0FuldCrwte9ZDpF0As7eZPkL7aP+ziLs3wwv1Y4VaxAjfhCro6iMsKVFwNI\\\\\\\/H8y4XsiqUOxzL1BnIhIH7X5lpPIaDgcYjoX89gm3yGbkCAiBIPt4G1r9Yc7htdSeD8U+0IhWN0epHY6WlGyY+p0YeEmAeS3i8w\\\\\\\/jyVsb6lGj+Z4YrpvZfsqcDUHyx4bblCes2iumGg3odndulpITHsHdMWochc6nfhELt5qyenBxk2L480PnidrVLdenHK7pprWVhifXmmwlV4zEqx1G8uzlpSL+3EzM3sSPZMOtEYbkAVRO09e70dT8Y1KLlqPRtucep8JkNXZx7nHYB7K\\\\\\\/diYHeEPEDweRCd6P6ldS8\\\\\\\/zQrX9kt73ZunKwpVEqpvHYtan3Z\\\\\\\/3i\\\\\\\/2TP83oUHFKJ0Ii6AKUoPfV6C+cwSHA4hrw04RgsKsNk0jAVhv6TC8fvmcVfbHX868jVGLrRoYsOgBVrt8g1sXeU69e4RaYqphtATJsG1hGScNkyO+jGaRzogkMA7T8BB6IEOc\\\\\\\/R5xb+SIQoSmwIXGYZciRXd1the00GWsy+qVOZGQcHbpUZwFSl1J2FFdsCy1SWGpAwCz0yC1n2nPezivskHG3lvN5205iUOXXHp\\\\\\\/2Sk\\\\\\\/Rgvk7vOPTMjkHL+UtIKEHoxG78Pb0bizhhwct1qvD7XKxIiCGWF812QN1jNjnLUURSPQmMrMfqp4M9hFFtRDoVlAPp6ZJwfS\\\\\\\/SDnkvEXcDSQUB55d\\\\\\\/YcoBrv2aHrutibIUqHrNNy\\\\\\\/X7tm23AR7xSRT7TEyhfbYvOxkutduS3dFtdO2uhNfL4vaKiDtw7645mZA6iKiX3l0HntrX9U7Gu+7dBvK7im483uhVDwwHJChJxSmVuKsj2WVmNxW8vU+rchlVJYpj9CUFUGd8W97cwNH8a9YpPtXJBOPj1ta5WjgLXXrIT+h9PjtmJpUutbpgJ9UTfBdzTOzg0tcz8b0MKCmS0QRhqgISlm156wiHAGubHy0dVz\\\\\\\/dQGa94Cpj\\\\\\\/YXMG0kWAL3Lw+9qrVTiMiVVsKkzL0wmDEF8F1xN0ubZEc6eeLJGtU67l9hcOqwCHHcpXHSaQAV85m5zv3uhrZKKFgftqoZuHPwQx1OoFewbVz0GPQn\\\\\\\/NRw2PZ9Ymwx08Q2\\\\\\\/5xDcsRI8YSEoe4x8Oj4RMvF7ZpJ9VuDn0L+kORxkDIRSyluLbg6cQf+ou9ojpMsTxcfZJAM9Wd1m6jzb\\\\\\\/17Tn1a7E6ABbMEKUvteZD3PanZJq7d\\\\\\\/1NyZKw8GFcQTkWRedtKocot\\\\\\\/7K6l36cVi561AG8q5yty5\\\\\\\/nG0Pf9O7B6GDN4KZsiQbZ3Iye5HxAJmEIClTBwrT1B6WiYXpRRH\\\\\\\/u5KSrhPOWEARO0ExxzqkEuJJmyWzlxctlZFbqaY1Lu+j6lxDvtYnkTFxEmlN2bH60X6pxmautL6LiVU3HTQiLKWsHfH\\\\\\\/ObAzs969WGXQwEPZtfGyMYow2lJwcGSchT0nW5VqV\\\\\\\/mdxr+CMbSwZuQYdUbWam5Rscam6wksMDR91WQVx4bAs8N+Y6aFJ8L8xLv293S7SfHu\\\\\\\/WH5Og4BylZiEMXW7ZcWI2i+HuQgX4mN8YTRLiZtafGXbiH3nARoPPo0Ao9NPqGoQKYIZ6A7CtdN0W6aBj+dNDuA1VCPWkdI689yop\\\\\\\/HIpKnLBwE7G3hTrOSjTOshjP+ouBK9J0n2KJZHuMQGnY1il3rLRfY\\\\\\\/SPXR5WVZ+HznV8l1ERSlgedX2tfaR6AEmm5\\\\\\\/V3fDdaMPpfdsD7kbUSkrLRQbQKTnrDkRZSDBkVx3+1F+WSfAFkw5fTj5RS7+wXTfF3QLp1OJI1\\\\\\\/93rmVw1fIDVkhAEsqne5l3RmDZsy\\\\\\\/h8rV2sqxGLwXJoFjKMtVHka2WtDb+\\\\\\\/wQP2FKbj4y64PIM1Q\\\\\\\/iBjTzAgy5wkMqUwXm\\\\\\\/ZvKRkiHzXr8w5++tgXxB0tSHPuE+RRKdJox5j8bF7yhDV35OxVN2MICDLY51EBvlthFp1n\\\\\\\/LoQFO2OJsOxSfQYwXIH8ccVcG5smoJwB6dKNfNPI2jwj\\\\\\\/x1qNuGkAP2gbJc\\\\\\\/JL2Wp4z1E148+dSb3n5Rmn0K146zaEymnTYV5EwINdGmw8jh\\\\\\\/PP20t+ixTz9n1Poe3+AUIZwF6ljL2gzbT75F7EhNVp56ueQHkrgTRULN8q8+BU1kbaBTf+K2U78DWMQ1pyic9DyGpMIsuOsdqyft6B5j4gpCCF\\\\\\\/JtrIUhYJjZC9bgE8wJ64pAiuw7tMSeFoP5msVjyAFeTYspwIoe5D38lrWGVzvQxH0MlvdFHZuLuah\\\\\\\/QSFWImbl1AlnZKLUjuJqIlXHroe\\\\\\\/clioa1fmydc+aMXEGVYc1fr+ZeW\\\\\\\/l9uQ+nIAHxgTtc6tlweY25ohIRljEX2TBS1JgsR5jqSXrLebpdFqZQi32UG24ogMqw3ruHQEx8+fbiNHUpLMekGuVPgGjzixr2\\\\\\\/GYE47awU0G53X2MMabJz7Dc2wT1jCOslCNnVGF+xQuLMKoZD4BerftcCVMFVpTAc1ZFFGNMxKZiT7O\\\\\\\/SX6wshRDs4gAvALCTxFhbWgej\\\\\\\/ctFP54zskfj7yzAY1jNAFhdydu0m+fJC6t8nSpHT+cwW2RMRup9YPpwYlSBJjjw4cLfd9LYpNNco8WsV3gQqGY7knAm8jn3klVx\\\\\\\/+gECpmlSu0HkljYp8l5SnmhXRSkBosh0GHvHslqHjMaeCL37NKUy4lFfMqlDOe1IXWHNY\\\\\\\/RE4TjOj6pO7Pl9zVVx2QHqfQW\\\\\\\/PgZTRYa8EjBbGLPsRjuyLwl3RwYVj6AjwOYGUiooYtt7kIG6GZ4rgSJ0EWJYl238qVBnKTt5NyGYmEDEA\\\\\\\/QPUV4gu0vP81VXallf1m4WsZ8pDue4vAo2n7nzFdkB5snhhqxjhQacaF9m+FbfF8s1p1hEnoqMaBCrXSNzK5D4VVJ9im95D21d+EoL27bvOrbbSFKjrTpDB\\\\\\\/ucYZVedEftz9jfVTwUtj2nJd5fP\\\\\\\/1zshn3OuD\\\\\\\/crT1hcNT0ruB0xmh0LHrmsrhkyTgFPTZ6WWczTAOMdwk34jYCsWk7AU+SBOVFE5yTajjb+P5+QiSl\\\\\\\/f4J1FoL6+GKEm5njT7ccu58MD3Fe+vcVUrsJdaWH6FVA2jkdrebf2rRbxjAqrReoNP\\\\\\\/xZpX4D1w9FFMaszGgpZ1dVrVrS+DELjnXFnvTIPzqs9st63lAe9A\\\\\\\/UYukFT649DUFfD7UKZqsAh1+7IUQbTCFOuoJE4MiQx43x+hlrFGYPuCszyqJodA+s2aP0LULoPy3yFYra4ewfDSLNqr4Bs100MUM4TLjwff1wXM\\\\\\\/pxl4ZY6YVJEDUtvIqpy9n4Ol0oA9bCZrKsLCUfcBv7BXghSTMCRtMzHD8MF4lcrVq26r3OTBKmqLua0Y\\\\\\\/nVlDR4UNwrqYZgJT04UGh2gXSzsyitssp\\\\\\\/xDRV\\\\\\\/KSA\\\\\\\/7WB1lWuIr7qAQeTMIs6h58+vNUhNnI901ivPdvsHy1oed5OCwqpDTaDxC24yaHIsQ23ZImsTZXQsszwJb51vxYbv\\\\\\\/mbtwUPRJ2rbJxPWdXRYEPWV5QEvHPYo5IDAzkq1S3f0pMaG4XBmjbeIBNK534lopJudGudtqlB4zbVvh2S4QXoBTv7gK21W6hbyrI6nbYdqaBGSb4kxZwBrJuuymcgSZVVhXg4bKeqLgUAWVxrO7IVn7IOc8rCwt3unAWVeWpxZl6jpqCVG2JvOnX4r2Wq+K3iED+h0ltR5DdoMxXkff27EC91xRw5iCmIFChlFGm4P5l3auHTxV9pCCZSYzR42wwhEL+prHetNO+2WPKMo4o2IGvLIEyyG79UbWpoVla5B5MiFJihuU74VxXvnHdrUZNUZANQuK+llSHFXZtIH84c1Jux0kuaV8OfYBXASmY\\\\\\\/cDABEYx0AQ30jYOJ9jYM0ShJh5ewFooyGP+p6idVSrWLxlpL2YWVbTJ9qsdE6FZ+iNlkWrCedGLlyLVN+nTx5EKvw+54GqoAvnFZi\\\\\\\/YIoET1Swo35vAA2A69YiC5pBijqQV7KKwhchAVW8R38GRBKbbopZpq1apl8WPIplj0rJVfj\\\\\\\/6+\\\\\\\/67PC8Iv90jkfRmzb4+BUSKEu4RQGvDJ5WXb8z+\\\\\\\/XcH1CkrXtPYIQ5tuuXKiI06zbUc3h0pnBtK4tvII8uXFX5QXC6ueJn9doJQZ4VMPO8FhFCYBvzwIUb+7kk1MJYq2F9QjddN94oESXKQSU8pSJp0Ws7b3A63UAdSYKYAa\\\\\\\/MwTfXnuhza0yRXtb5NOHqDNV7X3ckDN55m8sonzpIG8NSeMwN1MQKA8RhWOITSIhIKYqF5z2\\\\\\\/B8TkeaAv8+6+aDUaItUrp0ZKylK5uidKhm47psGhNT6lk3TfTGtyH9VapOh89FT3Ua3OPx7SG+D3adPwQEekanpDJ3uJn8x4Nss+nEetqihYBSeGBMsanlr2alUtSX33v18tlnVxezKpxYHjfy1i1WzBndvNTps90nrpJNQH9wN9JZOmeihZpn3PAs4he2jvIrl+CXiQ2u3GDJAQD2wCkqGqzLGUgHkeJkOtbmT2z+cPvdiaF8iJifhe9vZVy0x8SIdxUptIJSxZyHGrVSJYEm83KA\\\\\\\/9Dvr1mZWDtDpwrSTbpBDv+h9s2sP0sKJ8in\\\\\\\/TOwsL6xbd7Xed9YxFTLTc3j715IpE6iMIoxdGehUcKUsoZYIu8NTXg+I4KtBB2CKDM6OgCjujhExCOpOGdGKRUylSu\\\\\\\/E9LLwGKv1RUn0U0tCFjNnTMyBzwKBv7ku04UrlF+627ykDmRmkA5oY3EicA79auntalTsbg3f74QacO+H2LtISnn6FKMe6SOyFy0vU0N8DQMHdg5OhFfW\\\\\\\/+D5+fAoRI\\\\\\\/0WGakhAhGHoeKEO7NGXtDi9LvsB5XMFN9QCOeY+FlIpVxL9xEiTZ57VgBmsI9A9x9uNejL5VgnSlZgYaAEu7XKtSdxrIPTgn4BHI\\\\\\\/zD2npuF7OoydtKe+x2npLs1WnyMoOMm\\\\\\\/j2BYQTrXMfDO8FoFEL3MyAUkvTxj4K5ZtIwZjlODOoP2ck6SJTaSo+7xTYWfNabkHHR8vjAia9YB7+wkQd++WLvY9K6ME\\\\\\\/9wfIPRlV5HzYVtn5p8AY74yImkh+ZgYpVIB62HRcePOyaPF5FaJ\\\\\\\/pbKSYUrjoW2hAeJGLGRWpH\\\\\\\/c1Ni9X7eMHy5rQ1smjwq4uxwox0H\\\\\\\/YV5Q5SOREP5V\\\\\\\/3DRZKNjMEYgrU4C6nmpJL6QnfAXEAXX\\\\\\\/rWlhgovzTwe958MnxyWy+K8dX5Ok5ltMNM4Q4bdQDdeFpZxnCXlkcG6fkOYEG0r6lTlsqCa31TDNWw06fDHmw7wTYSHpNpONq+SZ8qTX+1hF8OwOTvlIHtsU93ATdpSBkwrBlJt8edgixflH\\\\\\\/XvT+sqKWQx+cns0FOYs0bOCo72z+RT0CrQ\\\\\\\/IQx1PDtcTCrEp49ngZm5pBfIT0hCN+6Mb8\\\\\\\/a+ea7GTiTMZsYzPhGiC+BZpXnt1H4yi+mcxMNht5pvxNCIxEJ5vZWJHEGQV5MH8jks5j9028vPRarUPENgAHe4UhNkpJlA4wbBAGCe61x4RP1D8EItx9756Ocb1qcHZuJvTBdb7oPaJdigS5AOh8CKwauD+MUDck14N+Dem9wFY8umoW+CgBUnHyG53UCNNoEgRZRrUZY66m4T1xhq6Jpz18qkoz3eF0njxagNNGH2tebY8JsovEd74Hd\\\\\\\/52t96l+TA4fynRsrN21Hv8MIW+9xHFs5y4ery3o8ienhUWTVNyFZVKLb7vWR3HZjVZvoXYMiCEmkVexkGs47IF7Av5dGN6t7o4JbkuononTY0jH1IteLwTv2FPhagNbCPRPaTzJRH8o8h45qjHOV8xxukSYumpRvmclk5XtyMY6i\\\\\\\/mUlRoP7HA+tv1u3sF1LqENT7aYoChp1smBXITu\\\\\\\/WKsyJeA+\\\\\\\/TabXPBOc0DN5eEZmSWJGU5VRgEGwYOpXBzaOME75VrIsWtomAmPD\\\\\\\/+XMYRxv0QFPOviuEn5JwXcl3a4SpsYVRrOfBqmsNsb47om3XK\\\\\\\/94T42nkbHB80I2j22OR3jpCq\\\\\\\/hlrRe26xRG2y\\\\\\\/fzTs\\\\\\\/hDUA2JffEI+F8wBAyiuRpfBuNAXutfa+JDRQsz4G3513hDIWiehJlnuYWEtru5h31a\\\\\\\/dYCdI4XsgCnGbsAXIW8NqPi27dfY2ol1Oc52ygKnTUnFImITEv7JD95W0Ap7W2KS4vRZ8ndSSE2RWmHYrvqULPBuuw5N8Eldf\\\\\\\/GMVTQAn6CKeq+VlYOwxB5S4ZJvtlcGcDNPL1q2qMTPZa5BU\\\\\\\/3\\\\\\\/s69Y7LyyVUMKUZLi6wi6tKqGubKSrjmZ+A0T7JHrzqKYwAAiMoXL2Hpg2jL5KV8I1tZvWldLbfZZSuI\\\\\\\/1QCB0Zdys80kxRA7BDFjjAmhAJzpBmbtLqb5bSL\\\\\\\/cPmWFVY0QRUUAKdimCEXNs1Tsk5\\\\\\\/\\\\\\\/aTZht+4U4q36eCkWOBro4+dTbPRQFAMOxNAXg4VMPNZVToVpl7ri\\\\\\\/P2hXQjtkBJsTOq8BYrXYCP\\\\\\\/bpEtcbEZbtT3LopUxrt2UUEHT4n3md\\\\\\\/XnpJyj8RUlEY9s7QQHbwQuY\\\\\\\/icVCF\\\\\\\/Cno6ZNIoZcJpFw1VN+hLvUCKghkhPI9TBCZOziEjrJjT8q3DC6Gao4zmdqD5jtRnajLlu8mFtY7vsKhpe3KU0aCMdZWlwJociS7M\\\\\\\/ZrO5rD1mt0IgtB90GpSCJXJq4YHqHh4eXMXz7qYUDSAu+ojIOHeAoTv87ZrMPumv\\\\\\\/b65K1GYOy+ZUrgAbHXL\\\\\\\/iM18AG4nP0C2li4yXfT6eIrJcGLUtkL0t6GSNLmPHz8gzirNzaY8mCmTGRCEw+3QtzbmYDthtjR7afhucmiFEDvGFnaH8FDKt9HaXWxcU0qoO4gteScBGsNLAfyAXY54DJF38Hr5NDpuTCTWwT5CqXKzGFyC7sCRyrCTOFr5SGXlD2e71p37yp3N85MR77axNp0DhUpVexnBcNBqWfc4w3cjWZAqd5YIJU5vYU8B2CptFn9vjT7ZD\\\\\\\/aWPoy++GIYLr0JB37wJS4L88I2MdioxsyaAD2Z1PCyonwJQWUUM1j0C92yuzrVtazuHckdSZCUgU7tmf+BcG5jSPO1BxVO1eagdOE0f2Jrs\\\\\\\/JrABmRgzhFESEBv7L3trITJdPKy30QJdJEFuu7YAF3zD29tM8Fx\\\\\\\/NslfhVx90jJ9krrZ0KWayHVX4Ag3a3pXCIqttyet8ZJzGmScscld2CVJKJ5GqrWp42wmlR5QsQCoC2R0Z4FIKIUUEDwq4ilM1BZwBfcxiJ3dtCrRTXqJ8HW72hgDyKxJrVsmYh36CvgjhmwLpWHbFkpr1v5QVAonYOclxqg6zCds5Cg8DaC2eEXRTYzrddTww0XPrV2V4v5JuDWBGisM1ln0O0DEiiEvQBTY1AqZFV5vX5aKJSQI7ThUfGIXDAl8iHr8InNrqreiWVK5dBkkKKAXbahaNVA6TM0du+FMcl0JfV5vL0rT16yr+E2qYFpHnob1I5akeU1h5t8sQG7XCopNQ6hLAHkGQrc58K1bOVFbRNqAJjnpZfeNpHIC9Jbe67JyNfiacImpmoNYxfQHBuS6Tpbnt\\\\\\\/g+Qo4FdnRF80JeZlreI2JYSJkQ1NrZlCx8A6RMGD9cgPtNov7LDj1euyJIDkUxA32ln00oH+fyAVVk\\\\\\\/eFfZ\\\\\\\/Gt9CIy3K3fxSBRzMMmuok15HC+XIVSzYuClUfmacbR3uf9vhL1olODfVoiegLntoBwOEOVvo8YCcCjeyBmF8tJg5\\\\\\\/Z6\\\\\\\/XZYbeg2k9xiD\\\\\\\/\\\\\\\/u4aCbJNkrbFrlV5tfbkWPQP9iMiUQXF8aNrPr9R9TibgtpDM5SKlf5lOAB3iooNf01FWYTUlnRteBNYW\\\\\\\/gcfDFVlmU9rzj7NaSxVdbedu4UtxAqT+pED4oJiG1\\\\\\\/5AbOnEyFdxF1q0TBER18nUZvUyfKwZY3Q2h2VZ2tNa3b88I\\\\\\\/pK9iiwAYZa7v6edS3mejQE9gBqV90i6nc712aZgTFdEPafYG+oanp\\\\\\\/ccSViL6JAbfQl0RVC4XQxVd5cv64T6gdam\\\\\\\/32mks+tYI5Mz8AMS3bz+aZ2OfWjVyyW4++qEwdlnX5\\\\\\\/\\\\\\\/4pGYXMT9HtZss9VOMZLMN1Y8uxNAnH7mU+kTjeLL3Vw3Znu3grQuCpFecy9FPhuZ2YSfv9MxiCzshwZT5xL5WKhloI2Vu7jK7KjVTXfcdFk0t9N1P\\\\\\\/h\\\\\\\/9ETSLLCcrsdBcBldr9yxflREAYkXKMiClzezf0ts87enNx+BtOR\\\\\\\/Fw8n57bSi8TnZH4DBed9FSjtgHGbJuNNBJ3YfiaUvC6xqOiNP21GDXrCGGLebK4KxsEtbjZVFKruep\\\\\\\/P4POLtIZwksviVw9xC1z93B9PoJZZYEnv1gA1xscU2WbCDq5zWh3DJx8ToY6Vu2kK3wkvKePXNNwA9pmauYFobzMio64sPP0ggUH3XKbwdPI4VBtjezGZ+OaG+mNaDncQjM0VStavIguVIzFR63\\\\\\\/lJlQ0QbzY+IicZkcSdQCwytKa3muZ6HawXkXTUiyDJL6eQMuXTlaRhRINKk4DrHaMj1awuy8GHEcPG2RC6f+hQJYb9iDNe2GrNHf\\\\\\\/7vv\\\\\\\/KQOMnbIjVifGijyXbeyocrcRR0hR0BPnYQHhk9M9AHDU0RVJhbm+wWosrUPMC+tUcwAIBfEyhKz\\\\\\\/Jk44gMXBcXFg4iultY07DUzTGkzJTq2iWu3zT1LuubZ+bYutceE65CGnaPExqzdb1dyVbgb14mV8z5uC1x\\\\\\\/mDQ7O9JjhwdolUjpKPf2\\\\\\\/p9X5uCaC4j8+JL0lDRdxh4s\\\\\\\/RbhJ+Z5eF3i6kfaMnAL25R1l0tVCid\\\\\\\/mt\\\\\\\/b5azGR4yvib9XkRCHCmCT8LJkEiXWw92M1S67AmhzcYp3oRAr++7AiYgZDkuQ9q6Nq7dvgwpsT\\\\\\\/oye\\\\\\\/MY9AQBJ1W5I3h\\\\\\\/XFklCPZeFzYuQ4ENgZugHyrs6ZELqHqoi+NpCSQrUJGHrxMCIwn8BeT0MBfycq6YA+ZO85RdgW+1BxPsgCFb7rlkfO647Hg9z19gcbV7T9pUFvek69yRpYJMEY2gOa\\\\\\\/7KmFEenpV2zwDWu9T2wUlO4cT5dbJk0ZHfuucT\\\\\\\/H2g8bDZWrkCNs+rfmz8PTYz84faFI8a1gGKOOedzXHAFARR28SJb9M4r+\\\\\\\/Gom6\\\\\\\/CicYhKtEk+p6U4iP\\\\\\\/cc20trjzhBcgvtnUwdMRppC4eu85UBNphSjkm2waXu4VHAyqZgTT1ykXraAckkwjMX66PH6MHNoBaEbCh71MFuVCFSbUUyLcqO2sO+SYrEtVhWx8YblisxjFn4BzMTUOBY4Uq4otx2yxAa4xhqr6u0Xg9FOPgQgBo2Q7MMVKvAIo0ib4xauR+S+JxuobotYe8u25Knu3NC6BtPNzwv8BOFxlajnQA8FK3Wxah3fHkavUwoE8QKhI4s0MBhJTE6qqO2KaqTqmwb0zZOAmWUULAC1S44bTf20i7MqkUV\\\\\\\/AyXoz2jK08mksKd4zRwIoRS3IsNhppmZtCgzrtLNbsppDjkxl9yhwPqN7IohLnnFqDlqZ47rBo4byc6i57PeUprGX+pT7MSNSc4oWmilJIumJD3Hsp9NlNwLkHeak\\\\\\\/uAZdyOvbucRZKagBh\\\\\\\/jmdwZXaO13hj2xQyFGHkidoWz56M9lqNy+W2xRkXXsKmxpNOxDCP6CpLipA4TrvHeJz6jqU11U1h9qJ3dhlyuZ0r0Sf8nwj1JRLHim4yxPZfP+DTy1aU1DN66OHJM6Zyb5ShF7d93KnhNJrVX5x2PkEBzJK3\\\\\\\/SU1wK95zXmCI2WFJ1ptlvgbRPDUNGr2xpby+cupMXlaQKu0834mq9OIaoWc1TLDDP7\\\\\\\/bfJPxsuAylp+TYsQuPwCn0KpzzS8qEhNKWqsLXb46oCl5\\\\\\\/wMsQBgLiLJFnF9MdfdO+ovOxk4iGN2q6nhoi3REagH3sCQepfwj0RtWMCNgx6MntAFAEQDlUtUkDrT7lXMvRNiEpV7BYOPcNA09SLeGcDih+tfVpKFP0BBBR0svOiByJldQrQKd5lIheOm\\\\\\\/YA+kDy\\\\\\\/josisVO4WQoT2G9K3ZvbLXRIbawsMam1sEN9n\\\\\\\/xT4BnzjTg6BBiEQtMCCv7DjHhe5uNzlgXHEP54rhf0ReTIyWq00nSNd4ujGFBAbWKvf1wIGVUdpyWt31LlgviBxs6c2k9oUDY1MLWfsp81\\\\\\\/r2jhmrb\\\\\\\/3JQdrtJK\\\\\\\/GSkcvOjAENd8kUu0WV5w9LtGdcl8UvVF\\\\\\\/HDh9Z91f7nI6sCOCTT9bOxGtO+kj8pYQ2wSRL9U++nSVKdRVQKFgDYEMfwT59XmF4SKaOW+jvIBCwwS376jeeehpk9\\\\\\\/IEGaPzIjq9w71t1ro4ULpmwQuFnTyM9naPzWVAqxCcVPdvLI+JcRAMRK2UVjn0Z67L7kGEYmf+NRUBIK30BwSvYuKkNBl7m3aMOUp50ahp6C1CPi8S5sIgsQUUrtNjJW8RCx7pgGGX1fETE8KV5krn9ssiivO9tRiUtmpS3U9ht7dG9SAVA6U44I0AcvKKpHmnMskpE8ji5F8qvTU9nPIvS65VqfNHcNHIOYm251uul4lclpoLsgcqL6fTfH3YxJEY5zp48Q1fYQnsRKH8YchAvyZFv24lRJFI0OLjjP4O6IUIAQZcFGgNQQA8UFSOTry3lvzthxgGaKKFtwYk7ThpITtUhEgQpWVkKJOcDJw4Tk7cGIPZty0fuhj+\\\\\\\/oW3otC0Fojv62nZNzGTBEFxXDaBL+RnHNE8GEakbcsM6+uEqQS3lmZ2anVokFTCMddrRbPVkX4SiRH+EG3FOp140p+ChWZNWKTIvbXtvrIEBPBgMIdXP1nRFTe18nXLFkGCENpIq4s2Spi9I5gF7DTRUMDgP4fsv+Jb2bWDdwgzm02FBT8v1PTArS5N3d64TjgvynMkM\\\\\\\/Npxv6UpsNzRmp29+PnlPcAaZUp\\\\\\\/dwuB7cC22MFlMnARV2YSYrRsWiDwJu8VEAeIk6oPJqLoyCtz1LUputprVWOugmOxWGE51C6sKKqcvqqKGSPI8zkZGQX0eIK019d5s2kD3SL8wydUAZ4piGv6mj4aMUi2Lja8h6w5\\\\\\\/08HpVWXUIEhI7WBftqvn\\\\\\\/ekwmXzSp8fiSy7OzOtp9MDBeLzREmQyho0X+mMsFss8BoMlejUA5i0ZtWClB01XgiTH4q3yka1eVSyZunnXOcfZpreP1rcKrE0ZyCE89514gnI3i1VMSQhWgJKGQ+pEhYI+kdO0E32QxiaV5\\\\\\\/Lr\\\\\\\/RHTCMYMTtEKLQmf17BPKb3tpsTiJpCJn41+T5WwlcHOlwbZoj1XXelnXSdf9+T3Vu9NFpcSLtnBZWUmKbcuYucZNU6Kb9ff2SW9lkw1j+lJuBkHZc8ZmZh3cRvNvAQWhvTU22rYjCQbwyzJXk9r3kuPSkbg92hdD\\\\\\\/7usme8hHhHEsm3hEL8DMPxw33VjDTCFH\\\\\\\/OQHCF5hr7IkUNtYJif9C0wKzqri+x2peCgZRFVt3vPN2oRQoygi8FtnEC0jhe8PLMT2HafNE8p\\\\\\\/Z7Km\\\\\\\/DITcXUDB0P6ks1wwZw1i8Bp4z7lGRjTmeioIvNpweo89EvgtQC3eFO4NcgUlz0kxZVGzZp9CsSq0uz8QFAUBOHriTZE9WduoA7cgII1Byklin8L7NRgYB2zrtvEp\\\\\\\/jIlLx6W8dgBwRFjKMIKZqXPu8oigi5IULHzNSBoqtY1KWh6JAz3S+Nz2GCLwSpFDC9g7b5BoCF0v11YYJYpSQEGd8S2GY7urZDlnsAEPciEHPesAdmIkvPWMuALHTUH9CZ8CZHFjnS1R6hgochLp3OjJ9XLRysQAMXiI64XRLG350VIxSwUb4TPpanAjnCByjLE0A4QB04jFkcXILL7jLqY1+YSO1kV2mTks0i76fAIxF3I1KPUI6\\\\\\\/IoTu47WYPQ2VGyp7+DOeDZYlUY6GfVgQ4VyrD7tYTOZ7HSRxf2TfJXS7EuQTu1yxHKHDRuu6dDIAdDWgWbOzWnpKOkcLSGBpfKFdTbw0OT\\\\\\\/Po8gmiR83b1L1JXMMxwEWfCrGDrCc1Av\\\\\\\/NI\\\\\\\/eezV1Vunn\\\\\\\/FtZ7ODhJElfAz1iBY2i+6Y63qSJC8DOm+ugQkYIu1ZsYqqoZQRA2WH4fa9KTXvlq6UP\\\\\\\/WgAEtPq6+Ph89NSp6zG8Gq+D88p5iuwZ4Z0tCYIAaAU5FCmP7sEnRk29LExCpabTVTlNH6875559REdsopo7a6nrpf6jfo4G16qdJ8w8ybFEmHr3X+gjgQu7e4B0OW7\\\\\\\/zu1RyUElk82NWdqPldh4anhmKJPRggXJEe5rYhqX95N8a8xYCR+k5305FhmWdts3JgoaDFHioKBfJ1RNwlxg\\\\\\\/Kfh3saskDGnzI2s5fuk\\\\\\\/AyNgqHf1SOFyjgglUDbHNvjkGrSDASJ9mvsf+tBGVCs+Z1mgbzgqinrtHrPTtmNLKp3IS3D0FIrGq9ymDezNS3t1P0wF9IPMoHVAVfRCDrG5G8ESJdZGNmDf2v+RyrBKWoNR3NZw+FgTG17A8u0Sg9rgRLtVm23oippxUHBQc2RPBvDC3uCBB+x2XwMOHxhFENZqB3d7Ofd6devdyF1vsFGb9KRg+NuHD2\\\\\\\/ZSvFIY70ChIcIkIZCNgBHcYcPKlCYu2HABwPwbKOfZ2m\\\\\\\/oC+52\\\\\\\/8Kqr4MvBPZ0mshSG8uLM4ShGh1Irl9EqerUCDLFrXnvaurLIte5S3IqYtFR2Aj+KsmdfIjGc+e\\\\\\\/UbvURxYGDFTX54KU+JKlArBQgFs1OzSho1XOJrH1KexCLlxDxgb7Sf1q8c77UxIbF65mXTrzj7wUx5h+tlWHojuTYcxFfeMxeAe77qbZrClFMKS962hVB7yFc0lj42ozOvieHmhWgaOmpDKunfVe95fuwsevKwyoC8Hr93R\\\\\\\/PqtyZ0nYnaK1bc3w53HarKiv5fqmJtIhU+HVGe0GhRuCKr+C3OJ2Ro4DtcsAyajWegitONOfr1iJZp\\\\\\\/C3d+ERgHp4p7IJURDe67akdyeK66uZwLs\\\\\\\/ZmtCG\\\\\\\/J4w+pAUNc7isWbi1xHIvvoM+YQkE\\\\\\\/ZZ8v4xN6WTt9VBr1PuTaRU5vIMEI\\\\\\\/9+ynZdNaQoMI5pSB6cn0cUW4syZPEJVyGraeCWquzdkU1pC+neWzx3jn+7oHDXisGLRlrlFheurwB73kiAiTZZeaXWuYzmRBHiL5zFiETUvFReaxGvpjhn9zPe\\\\\\\/APuiJjlTQLiJMBBPhU4eIPe+szpvIe+JEo3eCRHUSE342VF44uP5vXZ9bFDyEe+WNdN\\\\\\\/OOXqRwKDA8IGg0MKziRRmxFkjltRtyNpNnQ5S4sb3TjaEApYCszEK4E6AXZxMqe6B+BnUvzwW6MRlVrBvyCwMNwXdYgpYHK5fJmmFzWqYeLbEwORzOTHZkjWMp3j9v4FjUbtczRTYUyte2tN62RzHmc\\\\\\\/s9ZRSxVdm807FleE\\\\\\\/9Od55aAU2f4jC4gbVJrSqw1\\\\\\\/Z+jWmdEDM7m1QeGwrGnRmDDgIhlvbZ9wbduFgg0n721Rce0s6rkX5KyX3tGAEjCKdzfRxEOep87IyFUXBfOyYTZP1WyhPPXcFNLYXL2ltoWPpVWaqwLRWp30clFd+ppwC7d2MJieFz5ORYLvQFGakKjMkqxcXT7lLMO6TrxfI4UyIJT5j6NhU99Zl4KVvRWgSjC8IVEqX2hIO+juJKUc2xAP3TiSMDQCb5\\\\\\\/OB62Xczv5wrc4oJFCqqxUHVrUxjmF2wbk5xWOMTIo5NuLPFfEJEZei8hTmL\\\\\\\/38BuTpAk\\\\\\\/iKYdA8xaaLz2VAwokYmuaa3LSgY0pQQ309q1NAi9xM2RZrGGizfWGP\\\\\\\/FwpH+\\\\\\\/o\\\\\\\/94FqKP4VF1LI5+GG0lopjg9SkknFvFSaeNfTcf1tp0ZttRqS\\\\\\\/wby\\\\\\\/UhOV0s+pg7EfnhsB3HKRNHB3szEIkMG4BlbbcinPIgGkCAfRzbXdu9dxiYPMBUtJfuVRush6CXiRmKncIRAB98peUufBo+Cctst27Pr3FVZNy6wt+pmf8isxinNGz82R\\\\\\\/IsYpN5se98d5MG8ldw2fC7Hr3eTNTJM1Bf2h2aCIDBwW1GsTWs2MZdoLrCqovchM41T7myB0nbwq+aEJGc5zHO+wIoiQr1xVJBzCiIoctOSc6p5f0mXKcSSgi+peyGzyKND\\\\\\\/Z8ucowf++Km6CxiMGIDXiZPIKlrLcl6mlqvx6nFip2\\\\\\\/CFrRnBU60JqdQ+OTHHxgeh3nm87k68lAitEz1+ua8dXtOaKYeyH57KM2\\\\\\\/tV0jTDXmhAXoshA5bv8yZSIzBav3cmvSuMwicVUfiYcBWNxBUNVTEmmb55HcUNw2CA0yGJobNBjStaN3xL4E2u+9ST7Alazzv7P\\\\\\\/o0+Txv2fS\\\\\\\/ngpf+YY3fDeDTDpeTWOG\\\\\\\/yS9uEJKV3bC7UWHMCsq5PXL3PkYWcE+PT1ciAQTOR6VEk3gsgyclLuq+h+F078Pb3K+Pqd0YH28iYg6gI1Zjnoi6Y4UkfqwayVs\\\\\\\/G1qQVmn09wXJUKX06DcgymBSB\\\\\\\/a1dLWeeYsDRWf27RWGyTXyiKyqkoOVS29IlnvTFhv0ztJm5mTJiDQ7UYz1NGIKx6W8vpMUiG0wSZ1VcTQ387roBVD02lfY1FGLeUSt3QDudBW5hDLdtR++88J5i8iqadqZQPzjsHnrEf3RowHPaQ93KbcA+pRnacTyHGNyiVuGo3GbqtFbYNFOgXie7pshQrtbQHYcfYfik54RM8LqWqlppORelKuB81qmxFIdJ85BYLZlYxIAr1wctS0J0FrolBNUyET0hLhk+g3pWI5hAdqa3e6Ka36FX5ez8BSsbkKDyArV9RjWjwlXOdTs6YrZq+TNmmLt1ZZiq+RrFl4WF8UrJy1EV0P7m6CSVBKT8uGVU3oRxmjqttkvh043BCjRzGCbjYWN0NGwZnt5rgq+Vws5j95cwxWgqe1k7Q93kKykUOa90+82hgAjnW5lCBqv39gGpMVn7Tn\\\\\\\/qgsUCaqp1BebJ+n\\\\\\\/tGh3Focdp6vTlDkRzACLVrsw\\\\\\\/YCWmlAVJIkYMKivR7rVdcNywx6pa34PPYBp5mohebTwgCyF4cQq0FjN5nkii3\\\\\\\/56CLVgToWZQfnPrQToWOtHlSh42XM++tPwTwhWztHHmSpAb6VCCmWvH57tZos96XJ2FkgeiproSSWCGYmn\\\\\\\/qNxD0tjdUZ1eN54QZI0jpjKyVAaIPJPjiJ2B+FBUUXq3HIJe9exVXNnlZxDa7UHVrs9Qo8y1xKtHFCOTfZdrM1cfCcJQW8ZQXAhIQgePzgYMjZoy8yYdj3+TgOhavdDzniKaz6OXzUZOfPCDKpuw6jOS\\\\\\\/Lr1yDFWGweQcDcLAq5g8jjSsMhevfCvCtXDOwrQfyBjAZGDDAJ2afsb2sTI8GjFf2fXs\\\\\\\/haItUNJygg9A1Y0Fft6eYeAUkD9UepExD28B2npTRf+DUymdGCKDc+MgpGpBTuMvlXVi1YvFLGaCE3pr47lSSS+hOrNFhptswJVr0ZKJjr2co9VMLvMYg+PIUEjHtYm+56LpVY60BU0VX+evXRGWonmOo0sMfj9EgQ9TGcK89YoF+XC61Ceo\\\\\\\/OPO4WpOvgtx8xxfq\\\\\\\/UAyFTzbrG6UCv5NiON5ffILkmxMwolEqygAguSMD77L0GUyaKgQpBT68SFsJMmDE3fO3mFavk3zmkz27MmzZOrxZy0cv\\\\\\\/64vYXeTkg8edjcymQR2AGreHS8ig3jUIAamcZhLxFsIoK5S2btEGnrsDzJYAAodg05fSU8BJgkrmIE5ClZu+m0Qi94lzSBy9Fd+nfIG64VGnpjUuymZmNof6uJ3\\\\\\\/W8OqXAM1Iu3wPTNpcWnvPyZy10kAsVUj\\\\\\\/LG\\\\\\\/XVKbGlPfq5SMv2TgZvjTd0LhXhZcC1oLtgJ1TCm89JXRPi9Jg3WvPJuSCnXJalbhRG4gyRIXOWaZ5lHD2O42f1zfcr2+MlvCjRZ9q705ZJJy\\\\\\\/OmJGZ7kvOTwZHwJpk3PGco6xCvA0IltYUPUSHbXuVE1YRqSWdAt+6D6r\\\\\\\/CCRMnHiDpjFxzK3X9RAnWn04peq5ME1PbyRxMufY1rkmuQXiDzvwnVJyYjPwhitBuJCBB\\\\\\\/PeG3FOk9EicmPTCkuWDdqwwl113Idv0IWkwMlsdzfvt8VqJDq3h3m7wwzdLItENZxIXeS79y1+kvQc\\\\\\\/arTA8ghvTYnvP5sbgsDHEhR8\\\\\\\/x596Z57IFwEzsRFTJGeVD9Ik\\\\\\\/iXYdQmHuHOFTcx0lT+3cn5+wME8JI2miax1BwvPSrbLtPABWxO1k1x4r2mTC03iiBcO1hEQKDoISv9DmcyD\\\\\\\/k0LzjbcygUc9lFm1mhA2QpayOba5g+KUjoqSE8B+JkovrQok+E1ihL0+4pN4XcS03nSyuUtMwCdW3IN4wNr3AOThPRSPxx3PPO\\\\\\\/SCo37KVlyauXpIGSJxtyEK5QKN5NPh0mPJkkMGR232MDCDIQ5dddJn6kmHr64AJsE52EOQ5\\\\\\\/PLsokAnkd8qjy8r+Hjv\\\\\\\/YwZwqqa+Zr9jJZWMDGs1zW1QQniVakKbr9rmokeeIvAW2FodHr9yzN9zOE3sl1aU6B7I9lRPSY\\\\\\\/GyGYKshM5ZLvSG7Qn\\\\\\\/JhyLGUZQd5lIJ7jcCjbLJQu\\\\\\\/zIj7QF1SbheL9upH7t\\\\\\\/6NZhep7VZp5ahhQvZkJIkQEnCQpA86P4YxLECJmi72A8FkbZiAKNz\\\\\\\/qOzaXzz5p1APvw+tPHG\\\\\\\/wwyfyv2RRoRORjKUPo\\\\\\\/7qbXBOpG0EQ3\\\\\\\/T\\\\\\\/AWF3mWCgjzMa314MHCkyuqphT0WnFBeVVlCuXDqc0y16HFbmGMvjXbPDXsAa677ImlEEMVPgsts73hY\\\\\\\/7sIyh8RuT\\\\\\\/h60buWqpW3Kay1ozZsyTJrfSPCOK4jPUjhR\\\\\\\/XRGNM8AK3cArgJ9avT0cW+Vo\\\\\\\/i1VzJIgQgEou3V8oGqNPCBKaP9UiJeoFqeZSLPKB3hhDnUgeHLNrheNS9+B9YIvE+gnM3QBf8rTBWaf9nxGRv6hTpxWxNtTOfdsnV1w8gjOS6dTbf96WcBd7YIfPXfNlJ9XUBEhwM3YVK06jErrWyXa1q2y\\\\\\\/FsYxmEzkv6jYuCR1ESXprf0ghCkLgBRoesd05h29LDP2z\\\\\\\/UwkNuYK6RC3BDFz9+XqUiyDLYjxssTVZUt69TTL5LKNB5aFLdgYJ1dvurF7oDEzVHOyd4nF1wuauveXQGRTfgdmWKmdGCiNBght043GTBTOcrrgtMkznEdWEN3BlaXbgL6DNlFoyi+PhQzq3yG4acZ6K7uiZc6KCYpy50lR\\\\\\\/5wxnc2a1+ugTFf0lOh\\\\\\\/X9v4pTo8AL53WY98qVP8leiFM8kTNFZ3BPoQhJ2+Of9p9S06YaawZBvTtm+t8QiZxdloqfvdv7ku\\\\\\\/lg0BjnT7bMcSiNoYbT6LVRLxqOj7HM9rRYIikK\\\\\\\/oqporX1aQbzEtKEPhGdbUNjHRpMk3P9C\\\\\\\/JvCZMtpPS8wDiUvGdDqwCbRk0CfQd0ajoxbGqPfwPERHczJbviZYNFBv1T0iFHar72fFVt6yiMLAkRD\\\\\\\/9ijJ3PM1j41dbNMo8SsoQyIL5STPLFc4JKUkFEayCLOH0SqBl6ot48jCTy2+IHdckZEKBFQlu9ZgWYgKQGvs8EaFFHsKbi+EKSxjThM4mRgzy0hKZMeHJm6VIGrF7M5JdeJi+LwmCXnHa6T36o+Gi4QJDmqtX2Jo0XFdNZOSPY\\\\\\\/ST7WsvciJoef61TjqpudQ2YvpLK8Zk8wObLhICMKkWFC6VBqcoIzZtwqMpMOQc5kEEJ7Mnt4gUkKuBgTfodaCALnwoPlxF1KGZcoqqAh63YvtN+KzRerLVSJeKW4xBHu+ecnINcR1DrDkdwNKUiuoT\\\\\\\/Fcy3kRXH\\\\\\\/2HPAorxVcQZvoqA1njHAoszswq3JBVB0StoLdgpKqZJreRkoFS3eW8btDkHQbgjJRXbIoSClfo1o1yN9muLE+oOleILj0+QqvWyEpOImofXW0rRyK0qoYUoC\\\\\\\/x5H4Tx\\\\\\\/BlwwQ4DOPZcvN95YtlDnD5eGGrLukYBnOZ5CxEnGWmBHqoe5LWO3pEou1zLHhorK8BpO91UgzZ+\\\\\\\/CaRK9Asv5w5h3vWYPatJ\\\\\\\/eUPiABuwZ\\\\\\\/5o09bAA9zog58xhPnrLZZcsL6mwp6rnh6PtADG+3HJiYOtH4ly1N7jOORyvES+KfxamEoDRsA9OFzjQbzb9IdPuXcid68bkJJTvCGqkb3y9t7HCV2B4zAl38oyGT4XVmV9ZQanRO12DoT0\\\\\\\/we\\\\\\\/ZglBJnhjnINygIUcQCDEm14KCpgY784pc5z4VqS7g2sxfcCcQLSu8MZ1lEEjP7j45l32CfEmPG+32CHGih9okEBcT3aL31duVu0zzoPnWE7N\\\\\\\/6K+OQPthhlTi0mLzHkTHk7guW0tUGlwHGHJ0uMqmy7QkZrGRmkY5Yb\\\\\\\/XsdrsVzYJwkE\\\\\\\/PTDHcbh\\\\\\\/cfOw5mY8JnthudL17j1y5lsiRaNtPAT2IdakmkeO1mwOfFuIV3BOCyiM4WK6fo\\\\\\\/xR2\\\\\\\/pCnnwzymIsssgLBSEXsXAcWYZyRJe29YMvJz1\\\\\\\/93uLdGGRI8frtbg7aqgTOvVKVC1SbQE8wdX1pai79yuSMMMQKwnJINN8XbdFm46LW0lNnFUxcGxBx6\\\\\\\/v9TDuVzhYX9f8d3w19Z+NWxfUaBHSHVAD0nLi86MLgSs3WjdwtFPABPsal73IEoB4uTEUxJ0dWzXpDn3JzDWhngSSafphTgzn8SVlPb+iE+cxZzmDnO7DWUFAM7Gjf\\\\\\\/lYF7PZrEB1FM3tOOczMFuguo\\\\\\\/vqzQCK5ohcQJ7sBSD\\\\\\\/a0FVtk2eJANemDIuR9fWJAk152tDnXIKEW3H7R+pXRp39nD3IpimMrU9ZdyhLNeEAjUar2GsmW3meAfWcFbixW4IvRCaDP3pH8z1zaIaj+btEP6hDnQslYZnBHGBhqU4dIMOZfGaBa2Rc5a3QmqMESm\\\\\\\/MQhY45654LX3iK8jDGc8SEfIFemOI9j5EbBXo10zmCRbSMBmT4Ib9nr7mIGxVuk+3LbMUxFmelsWvD8aK10oeS+oXjkbRRZnCIyr3oOdHj8R9Ya9D9+SyYJ8o8wZTymMuJK3+d7Qm28wBSlXMiL\\\\\\\/NxGCOxd4x7nw+iSChF7Nw6mZ7UKJJ\\\\\\\/ToMROh3VwxdQnwKRE5nZbY0bxuFNww2i2VKYAkPu9OLtTMkbP8m5hUdldY9L8b\\\\\\\/Uf+nESZckMB3z11H4dUFZdAV0wWz3kfIf+Cz0X9pxR6r6MpKViToVrKO3sQg6zkF1AzWDtPREA\\\\\\\/a4x8rAV27MZJuY9zio8e2CP0rV0jF4kE9MmQS0MVwD6wz6IsGzjCkp6e1f4UrMlt4B5cU8CaQp4vMUCBMUXhFNsHrUihk1RpfzmQlKoVcdZzxqILMC4oqX17IWKCwVcMAel58wcMZw1bD9iZfNLU8PPIbFCtG\\\\\\\/5+iWcQP3QuALmzhDDUomtZRH398gNPro8HiyB37EA6t39kFfCNu9J5JbkDs9uYy+shUs6+C0K\\\\\\\/CVxkL10KSV5JFAh6csagJQcVryIXfTiezUxaA\\\\\\\/2BNj8ZhbzOZNlkv2z+NOpiXBX7cji3AcQz4\\\\\\\/remeEb\\\\\\\/yZxZlent35ZkHdGRI8sB8vbSA+FZzZZxfRXD7QXQURZ55RTTuVDADEspQAwmVvdmZpeevAuZDck9slQK6XqcAULrpmUlddk34lClvnQ7VGIEElZkBOuEGQv81\\\\\\\/j26e\\\\\\\/5ssJAA2ZRCTkHTZunBZ6EUuTNwTQy12FS9yEhP9FUkEPI1Oq4VaYP2SEqCuVs3qnd48ZDzExG3c1Dx6znxFUdss2+0LDTHv9BlvkWDfD0TX4kopslIAM52NdZjLm9+tHCqimY4yAA364MZgiPF5r4KWAMJ0bBM9XmfZcsihJFMmyu\\\\\\\/ileDtX6jCNnXOl5dNCLfRP3uOrupb99vi0+Hw9elnm+xOo09K7kVC8+NPHju3weRQYGxw+u0gXH7+VtalOOA8xnb2fXBpMr1om52XVGptnFSKz2cAXQjnqYYtGZq7DQiUpy0e2pEpTzf61FtQt6wKCTAZAFoGXlZvzjFMitt85vXnwA\\\\\\\/fk840XwwnFXp7xheDLXuiXYrPhTmS\\\\\\\/5QyNkOIWGsX5tZfCgQ\\\\\\\/kvSXgoUTc+IcKQL9N\\\\\\\/hAgfVrfCwAZprWF2mRBkeQIl\\\\\\\/ojnx3a\\\\\\\/U2idY8ulpclJq31Kq\\\\\\\/1g1ayrdOX4y+wQJGuaZ+QB+8vMW1QGKFskZtnNWeYhCe6z7TzPg\\\\\\\/oJm8di0toh0xiWeljTUe\\\\\\\/ctTF0egrTJDDcLWf5T\\\\\\\/KA3Ml6ZO9a8hHvxEYffruEsOGJQUjjyfbLxgiSQggEBconOcQFz4hJgD\\\\\\\/KKVx2yTmuLPi5IzSUw5dHgAH1rE3afjgC+WzuEJGq2kHT9Nu9SKoxTz9QVWHH5DlMQniEKmcfdQllWOKjnjVV7Ma0hhQYC4R42z2hvaKxMM603OoVsSJNnr7fa7o\\\\\\\/mdaK6izUoDK0Zy\\\\\\\/HUaq8sXOpnlpP0DI1uqQCYU23h4O7reamrMkRObkdGUDglxWs221rkWQIjcgNKtnZRsIr6nCSYpRDOIprueiKbNTOpwooUIOQxBxNBDYjePQHbMsqSVV5VzCbHSbDrIY22B5IJTwb072cPFiyq9BWuL28VCBxEWv3h3QDt\\\\\\\/J0tcZuqq68vi\\\\\\\/w8X8K0Tz6BP\\\\\\\/mtvqDD9tlFDpG07mI8Wv55loTsli3CW8BLRi0U0VUJrWaCBUWYWQ6WrzuRADY3Ao4AIjhVwhnLLdOnHsu0T+Ebx+vzEhquDsxrH7sLCB7ZZw9Hn3aViJWdpDxEUlIMBJEh30SB7OMNB7OIrZEMkLrgLGFvJyuZbhoP7gHPPbq0ep6OammpiJvOhk7flveMgtebVlyhjpH6xo9X79kNB4W327OEfLrTd0mntxyRKMGkNh99x\\\\\\\/8tx\\\\\\\/G+sOFtW8ARbHdH35zfQN+icQfcY5rpPTV\\\\\\\/bLdejjLblSFN32qAqyAbQSPOvxCM1jKfnLLdZhXurxKEU5e\\\\\\\/CvpGi6IYxwWlkEyY1gVVO2zTIa4Ud3s+zR0I0M17VTOyd9Z7iFMqVw9tibIvIQDL+5fDApmsiQjmjYy+P\\\\\\\/sJs3\\\\\\\/H+CVo1Jd8\\\\\\\/2GN9SdvnZus+W3JQLslTls7F\\\\\\\/UFJPcpPBTj+LY\\\\\\\/dvpI7NCVbdsukOdeedfA4NLDhN9WjSsz0C15KbPELLdu2Hj9Tw+zg7t2ejLXLk4fRHePEf1Ebnh3LUlvgTeWqKFaypoTR8ImeRijx3fPLvodvwW4DWT28uk5ppKLybTtEP08vmkks6enkwu5QM6S3tD53fwVhl0zvdinVjVhvvsHzW+F6Y8pHMI4kDnINHtalGPXVJOst1JSaffJWxwi06a80oypYA305O+mZeLm\\\\\\\/\\\\\\\/lodHo8mmhRaa2j\\\\\\\/5QBuHPwN\\\\\\\/7BLjUikFwyyEQZh0P4rx\\\\\\\/C73O8hfB48PryStbu+52Zy5ZGSwusEpxMfsHydG+i6tIyvnXtiABKuhorqUigDqyR03aNGzkD0d8\\\\\\\/KgLxhL\\\\\\\/gIPNhwbGqWaMld6bBO0eCpm7xdibm8+S4\\\\\\\/Ggi9F67jZoZJhg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022PG+nFn0Mad2ZYH8jkRzLPw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e0b34bc7ecdfcd21a0ac2a8ea9b9d9804ff4f7e44aaa569b49b4374b74916356\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022RJNgB4nzUP73a3x4NL5NvwS\\\\\\\/cEoaP6tzV1nVKkihQPV\\\\\\\/kVQBFByG\\\\\\\/ZYCQ1\\\\\\\/XRU0\\\\\\\/Xrfna3tnJMN9NKknYqnZS\\\\\\\/zawKzg4orGLq8\\\\\\\/3MS8amxOVlKb1qOY\\\\\\\/Dw8R2GbpNxzSXeMXOcRkdw+zhF9Px5ZphhBCuWAQDosDZT8ljzE5VNDuw63t6xpy76o+i7AM9PihKQy+FLPmCyFP8fWF92laBTmRSP0y4051MPUfJFxQoUL0EEzfC6ftJSyRJTwrELlUW\\\\\\\/Xgkf2cd4txXJzd0B410UNMqwXSBzG4S7183Ol0\\\\\\\/stlLh1xFGR0OHq2r6FkzFsxuJPHZcjQ25fJbwxGATQ7iLeko7CmHV3kUS0gMZQ8b4yGWMDkL20FgTSs\\\\\\\/qcYlKxrVZ9GoY\\\\\\\/twqjzq7nGN63FFR1XOsIBMMbiT+a5C1J8Io4yJbtkae3Gp1J3DsgWt1N5i6eYCo4WD5yG8Luz\\\\\\\/Z46kEh+p9xKUbWGsi6ndxsAt65ZUAeFc1kmKj4W37gvP7Jpb4woftblPIycgcJu3ypMbItxhDH8kil4rQOZSAD5br4vicSWwPIfi0ZCDvbg4HCBNUym+hd094GttwuC00QKzoTfz2tCAB6cEtbflyVbnQnKGqzN\\\\\\\/Tn0aHUz2ifEcA8rAg+UrEVRETbcw8HMzTzpp8sE5hcfmTGB4\\\\\\\/7TpT1bRIybIZgOXh8LKpHZGYuw\\\\\\\/MA6xk\\\\\\\/NLF+l6cs\\\\\\\/q7EYuFytaP8cpqP1fGc9yrMXcBvep\\\\\\\/gbe9+o39tknt8R2Rn9kWNX5+NnicXCj6aubtWVDtEF+cIB3i3smGUErVeXkE2OcbEVNur18+klv2ow3y5WrwmObeXI1C0SCOJhovFq8DgzVU40Ys3FTjT07UwZsmdqL2Ya\\\\\\\/FlYIcS5GtDm4AOlO8kPHMloYz76awUP69YmVkLYJ1ZpmbVgL9S90p+do1PkKU4BciIOvEGoERd\\\\\\\/AdfMSOHhbeIG42Yf2rphokWTte3JU9G4aPGA4FDnpTxr\\\\\\\/IHEvBCm2D5AEk\\\\\\\/2MO2CCua\\\\\\\/+c\\\\\\\/IJuZ2UB1K+3FUao1Lp3PmSgKLseBcl3zVUIHEK3Bg0Mp6mkjQnqfNin740v5g0ykHKGVIuTcHqrSOmdmRHB5bkVODIckL33S2qvyUvFMg\\\\\\\/HTqtDnPaZdC1x9hODC3WbnuEjRawn8R0bHpzOpnxfqCgG+3UaYKRlG1a+o36Wd3wtuX4aTgj5nZtfugOiFOuRsJfPgvrCzSpJt655UUj5wMCjF7CNmyFDZC8e4ZDvwUZZqkzvVv9XETMmUdW8bkM4w9NIsbHgyc3UyRaFFwRL3RHAGxozw3BuroP7Z2heZFmK9FyspxCMjC3atFoVdbtRuaeQdI2\\\\\\\/2IhA11II7NpqAWkBd+k3TrgFTroNjByYEuqyWWBafx079pDFBktfRrla4fT0zswb7zFCImvmZobpOwnxQUN+7kmpRdKkzdaO62L6XIMRyfvXKpJQ7UG+0tIdvBu+umPXuYaXXGdSh8PtJJqu8xMb14jyPSR0vCmSzat1A\\\\\\\/hgAAAk5jfDUt1E8Nx\\\\\\\/7TVRBdY4QpnBrY1ym7Oc4KDTEUmxRD7RRar9x3ec+2hZquwbrfnaxbt+JRHMB3x9XFrrjNy4k08sDt1tWg2oev\\\\\\\/Vd2ZuRdU\\\\\\\/H9GaxNCZV73HT\\\\\\\/sXKhQNO8a\\\\\\\/BJ\\\\\\\/qBM4UheUjlrMbl5KSjbfZe\\\\\\\/s+5r0yAQE6KOiIvQacHrkOYM\\\\\\\/bSWBEv4qbxj6eXuioXVtQXxJzOaNYEA4QMsF42HSVRRkAfDXSY65FPaY5mj947JTIHlt+Oyp0VpWpv566JeXX3tNDzL1sXUqDfFUSnYVUidLw75vLvyVVeP87oO5KHZ4Rey9A9L32G6fxUubM2OrV1Zyl0DNWy92fGuyWi5QxA1JqS++YeB6UXvMZC39Nr7OOY0qNtiZgLPbaHu3HokVFM3fG+wnxbaW6xAwyunKOEMCrrbRn98OyxO\\\\\\\/rwbA8UnIHcrt2\\\\\\\/NVj\\\\\\\/XgzEX+3z6XbL96ZN4dbFrv88DB0Z0i\\\\\\\/Kqg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022QWKDJ6ZWaD42jMFl9N2VlA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022235669b18d6014fd95fe639e9cefd45f36fa938ef4ba63eeb117c82ca0df44b5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459573 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/2069" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-flPBjjwCP5xK9oMhJ885-kv1-1oDXjL1Ow7Tr2g98cM" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/2069" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-KTge7m6B_MjyvwZP9BVHj2lgaVCDLGZZPWseDXxITRk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/2059" title="Molecular Phylogeography of a Human Autosomal Skin Color Locus Under Natural Selection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/11/2085" title="Dissecting Genome-Wide Association Signals for Loss-of-Function Phenotypes in Sorghum Flavonoid Pigmentation Traits" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459573" id="node459573" data-pisa="ggg;3/11/2069" data-pisa-master="ggg;g3.113.007724" data-apath="/ggg/3/11/2069.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Allelic Variation in a Cellulose Synthase Gene (<em>PtoCesA4</em>) Associated with Growth and Wood Properties in <em>Populus tomentosa</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Qingzhang Du</span>, <span class="highwire-citation-author" data-delta="1">Baohua Xu</span>, <span class="highwire-citation-author" data-delta="2">Wei Pan</span>, <span class="highwire-citation-author" data-delta="3">Chenrui Gong</span>, <span class="highwire-citation-author" data-delta="4">Qingshi Wang</span>, <span class="highwire-citation-author" data-delta="5">Jiaxing Tian</span>, <span class="highwire-citation-author" data-delta="6">Bailian Li</span> and <span class="highwire-citation-author" data-delta="7">Deqiang Zhang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2069-2084;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007724 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459573" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Qingzhang Du </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingzhang%2BDu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Du%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQingzhang%2BDu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Baohua Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Baohua%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABaohua%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Wei Pan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BPan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pan%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWei%2BPan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chenrui Gong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chenrui%2BGong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gong%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChenrui%2BGong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Qingshi Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingshi%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQingshi%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jiaxing Tian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiaxing%2BTian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tian%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiaxing%2BTian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Bailian Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Forestry, North Carolina State University, Raleigh, North Carolina 27695-8203</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bailian%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABailian%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Deqiang Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Deqiang%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADeqiang%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:DeqiangZhang@bjfu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">DeqiangZhang@bjfu.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/11/2069" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459573" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459573/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/2069.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459573" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459573/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/2069.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459573" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459573/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/2069.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459573" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459573/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/11/2069.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459573" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459573/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">Lignocellulosic biomass from trees provides a renewable feedstock for biofuels, lumber, pulp, paper, and other uses. Dissecting the mechanism underlying natural variation of the complex traits controlling growth and lignocellulose biosynthesis in trees can enable marker-assisted breeding to improve wood quality and yield. Here, we combined linkage disequilibrium (LD)-based association analysis with traditional linkage analysis to detect the genetic effect of a <em>Populus tomentosa</em> cellulose synthase gene, <em>PtoCesA4</em>. <em>PtoCesA4</em> is strongly expressed in developing xylem and leaves. Nucleotide diversity and LD in <em>PtoCesA4</em>, sampled from the <em>P. tomentosa</em> natural distribution, revealed that <em>PtoCesA4</em> harbors high single nucleotide polymorphism (SNP) diversity (π<sub>T</sub> = 0.0080 and θ<sub>w</sub> = 0.0098) and low LD (<em>r<sup>2</sup></em> ≥ 0.1, within 1400 bp), demonstrating that the potential of a candidate-gene-based LD approach in understanding the molecular basis underlying quantitative variation in this species. By combining single SNP, multi-SNP, and haplotype-based associations in an association population of 460 individuals with single SNP linkage analysis in a family-based linkage populations (1200 individuals), we identified three strong associations (false discovery rate <em>Q</em> &lt; 0.05) in both populations. These include two nonsynonymous markers (SNP49 associated with α-cellulose content and SNP59 associated with fiber width) and a noncoding marker (SNP18 associated with α-cellulose content). Variation in RNA transcript abundance among genotypic classes of SNP49 was confirmed in these two populations. Therefore, combining different methods allowed us to examine functional <em>PtoCesA4</em> allelic variation underlying natural variation in complex quantitative traits related to growth and lignocellulosic biosynthesis.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/linkage-analysis" class="hw-term hw-article-keyword hw-article-keyword-linkage-analysis" rel="nofollow">linkage analysis</a></li><li class="kwd"><a href="/keyword/multi-locus-association-models" class="hw-term hw-article-keyword hw-article-keyword-multi-locus-association-models" rel="nofollow">multi-locus association models</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/populus-tomentosa" class="hw-term hw-article-keyword hw-article-keyword-populus-tomentosa" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Populus tomentosa</em></a></li><li class="kwd"><a href="/keyword/rna-transcript-analysis" class="hw-term hw-article-keyword hw-article-keyword-rna-transcript-analysis" rel="nofollow">RNA transcript analysis</a></li><li class="kwd"><a href="/keyword/single-nucleotide-polymorphism" class="hw-term hw-article-keyword hw-article-keyword-single-nucleotide-polymorphism" rel="nofollow">single nucleotide polymorphism</a></li></ul><p id="p-6">Wood formation represents a major carbon sink for the biosphere and provides an important renewable resource for lumber, pulp, and paper, and possibly for biofuels feedstocks (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li <em>et al.</em> 2006</a>). Wood formation mainly includes deposition of strong secondary cell walls that contain cellulose microfibrils, lignin, and other components. Many studies have examined the molecular biology of secondary cell wall biosynthesis and have shown that the complex, dynamic process of secondary wall formation requires the coordinate regulation of the diverse metabolic pathways involving polysaccharides and lignin (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Persson <em>et al.</em> 2005</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Somerville 2006</a>). As a biomaterial, wood varies in its properties. Specific compositions and structural features make wood more suited for different applications. For example, high-lignin wood can release more thermal energy and thus would be well-suited for thermochemical biofuels applications. Moreover, wood yield is another important trait for trees used as crop species for lumber or biofuels feedstocks. Variation in wood properties likely depends on variation in genes involved in xylogenesis (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Neale and Savolainen 2004</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Li <em>et al.</em> 2009</a>), making these traits amenable to a candidate gene approach. Few functional studies of forest trees have identified genes directly affecting wood quality (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Spokevicius <em>et al.</em> 2007</a>), largely because of their long generation intervals, large size, and lack of mutant libraries for reverse genetics (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">González-Martínez <em>et al.</em> 2007</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Zhang <em>et al.</em> 2010a</a>).</p><p id="p-7">Association studies are an effective means to bridge the gap in our understanding between complex quantitative traits and the underlying genetic variation at specific candidate genes or multiple loci dispersed genome-wide (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Sexton <em>et al.</em> 2012</a>). A diverse group of growth and wood properties has been studied in forest tree species by a candidate gene approach (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Thumma <em>et al.</em> 2005</a>, <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">2009</a>; <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">González-Martīnez <em>et al.</em> 2007</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Wegrzyn <em>et al.</em> 2010</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Sexton <em>et al.</em> 2012</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beaulieu <em>et al.</em> 2011</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Dillon <em>et al.</em> 2010</a>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Guerra <em>et al.</em> 2013</a>). However, linkage disequilibrium (LD) mapping may generate false-positive results because of population structure (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Atwell <em>et al.</em> 2010</a>), although statistical methods to control for population structure have been developed (<a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Yu <em>et al.</em> 2006</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Shriner <em>et al.</em> 2007</a>). Results from association studies in these species must be cautiously evaluated and, ideally, verified or supported by other approaches, such as quantitative trait locus (QTL) linkage analysis, transgenesis, or transcriptome profiling (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Manenti <em>et al.</em> 2009</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Ingvarsson and Street 2011</a>).</p><p id="p-8">Conventional QTL linkage analysis in controlled crosses and alternative LD-based association mapping using diverse germplasms are two broadly used approaches for the dissection of the genetic architecture of complex traits (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Brachi <em>et al.</em> 2010</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Sterken <em>et al.</em> 2012</a>). A linkage approach is powerful for detecting genetic effects at loci involved in the expression of target traits, often identifying large chromosome regions of interest with relatively low marker coverage because QTL mapping uses only the recombination information found in the progeny of two parents. By contrast, LD mapping offers the ability to exploit all recombination events that have occurred in the evolutionary history of a sample set of germplasm, allowing for increased mapping resolution with either previous information on candidate genes or genome-wide scans with very high marker coverage (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Manenti <em>et al.</em> 2009</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lu <em>et al.</em> 2010</a>). The complementary use of traditional linkage mapping and LD-based association mapping would further improve mapping resolution without requiring dense marker maps by combining the advantages and overcoming some of the inherent limitations of both approaches (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Myles <em>et al.</em> 2009</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Lu <em>et al.</em> 2010</a>). This integrated strategy enables a closer examination of the number and effect sizes of genes responsible for traits of interest through complex trait dissection in several plant species (<a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Thumma <em>et al.</em> 2005</a>, <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">2009</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Stich and Melchinger 2009</a>; <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Lu <em>et al.</em> 2010</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Brachi <em>et al.</em> 2010</a>). Here, we examined the number and effect magnitudes of allelic polymorphisms in a candidate gene underlying natural variation of growth and wood properties using integrated linkage-LD mapping.</p><p id="p-9">Cellulose is the major component of secondary cell walls; its biosynthesis is catalyzed by cellulose synthases (CesA) located in the plasma membrane (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Suzuki <em>et al.</em> 2006</a>). The catalytic subunits of the cellulose synthesizing complexes are encoded by the <em>CesA</em> gene family, and different sets of <em>CesAs</em> dominate cellulose synthesis in primary and secondary cell walls (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Taylor <em>et al.</em> 2003</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Burton <em>et al.</em> 2004</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Persson <em>et al.</em> 2007</a>). For example, <em>Arabidopsis CesAs</em> (<em>AtCesA4</em>, <em>AtCesA7</em>, and <em>AtCesA8</em>) are involved in cellulose biosynthesis of the secondary walls (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Atanassov <em>et al.</em> 2009</a>). Poplars (<em>Populus</em> spp.) have a long tradition as a model system for studies of angiosperm tree physiology and genetics. <em>CesA</em> homologs have been identified in <em>Populus</em>, and they also have been used to investigate the mechanisms of cellulose biosynthesis (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Suzuki <em>et al.</em> 2006</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kumar <em>et al.</em> 2009</a>). The first tree <em>CesA</em> gene was isolated from aspen (<em>P. tremuloides</em>) by <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Wu <em>et al.</em> (2000)</a>. Since then, 17 <em>CesA</em> family members have been identified in aspen and its hybrids (<em>P. tremula</em> × <em>tremuloides</em>) (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Djerbi <em>et al.</em> 2004</a>). Eighteen <em>CesAs</em> (encoding 17 proteins) have been identified in <em>P. trichocarpa</em> (<a id="xref-ref-68-3" class="xref-bibr" href="#ref-68">Suzuki <em>et al.</em> 2006</a>), and five <em>P</em>. <em>trichocarpa CesAs</em> (<em>PtiCesA4</em>, <em>PtiCesA7-A</em> and <em>PtiCesA7-B</em>, and <em>PtiCesA8-A</em> and <em>PtiCesA8-B</em>) are expressed in developing xylem tissue undergoing secondary wall thickening (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Kumar <em>et al.</em> 2009</a>).</p><p id="p-10">Here, candidate gene approaches were used to examine genetic variation in only one of the <em>Populus tomentosa</em> cellulose synthase gene homologs, <em>PtoCesA4</em>, underlying complex quantitative traits related to growth and lignocellulosic biosynthesis. We used a combination of single SNP models, multi-SNP models, and haplotype-based association methods in an association population (460 individuals) with single SNP linkage analysis in a family-based linkage population (1200 individuals) to identify several associations underlying natural variation of important wood properties. However, to probe the mechanism of this variation, we also examined <em>PtoCesA4</em> expression and found that it is expressed in developing xylem and that its expression varies in lines with different genotypes for one of the associated markers.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Population materials and phenotypic data</h3><div id="sec-3" class="subsection"><h4>Association population:</h4><p id="p-11">An association population of 460 unrelated <em>P</em>. <em>tomentosa</em> individuals representing all of the original provenances in the entire natural distribution region of <em>P</em>. <em>tomentosa</em> (30–40°N, 105–125°E) were used for initial SNP association (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Du <em>et al.</em> 2013</a>). The distribution zone from which these individuals were collected can be divided into three large climatic regions, southern (S), northwestern (NW), and northeastern (NE), on the basis of a principal components analysis and isodata fuzzy clustering using 16 meteorological factors (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Huang 1992</a>). Forty individuals were randomly selected from this association population and used to identify SNPs within <em>PtoCesA4</em> using a direct sequencing method.</p></div><div id="sec-4" class="subsection"><h4>Linkage population:</h4><p id="p-12">The hybrid population used for linkage analysis consists of 1200 individuals randomly selected from 5000 F<sub>1</sub> progeny of controlled crosses between two elite poplar parents (members of the section <em>Populus</em>), clone “YX01” (<em>P</em>. <em>alba</em> × <em>P. glandulosa</em>) as the female and clone “LM50” (<em>P. tomentosa</em>) as the male. These two related species are members of the section <em>Populus</em> in the genus <em>Populus</em>. The progeny were grown in 2008 in the Xiao Tangshan horticultural fields of Beijing Forestry University, Beijing, China (40°2′N, 115°50′E) using a randomized complete block design with three clonal replications (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Du <em>et al.</em> 2012</a>).</p></div><div id="sec-5" class="subsection"><h4>Phenotypes:</h4><p id="p-13">All individuals of these two populations were scored for nine growth and wood property traits, with at least three ramets per genotype. These nine traits included tree height (H), diameter at breast height (D), stem volume (V), fiber length (FL), fiber width (FW), microfiber angle (MFA), holocellulose, α-cellulose, and lignin content. The growth traits, including tree height (H), diameter at breast height (D), and stem volume (V), were measured during field surveys in 2009 using the methods described by <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Zhang <em>et al.</em> (2006)</a>. Wood chemical compositions (holocellulose, α-cellulose, and lignin contents) were determined using near-infrared reflectance spectroscopy (NIRS) according to <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Schimleck <em>et al.</em> (2004)</a> based on training sets (models) derived from wet chemistry analyses techniques described in <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Tian <em>et al.</em> (2012)</a>. Fiber length and width were measured using the Color CCTV Camera (Panasonic SDII), MFA was measured by X-ray powder diffractometer (Philips, Eindhoven, the Netherlands), and the X-ray diffraction profile was integrated at Chi between −180° and +180°. ANOVA and phenotypic correlations for these nine traits in these two populations were reported by <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Tian <em>et al.</em> (2012)</a> and <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Du <em>et al.</em> (2013)</a>, respectively.</p></div></div><div id="sec-6" class="subsection"><h3>Isolation of <em>PtoCesA4</em> cDNA</h3><p id="p-14">Developing xylem tissues were collected by scraping the thin (approximately 1.0 mm) and partially lignified layer on the exposed xylem surface at the bottom stems of 1-year-old <em>P. tomentosa</em> clone “LM50.” These tissues were immediately frozen in liquid nitrogen and then stored in the laboratory at −80° for later RNA extraction. The <em>P. tomentosa</em> stem developing xylem cDNA library was constructed using the Superscript k System (Life Technologies, Rockville, MD) as part of our effort to identify genes expressed predominantly in the <em>P. tomentosa</em> stems. The details of constructing the cDNA library were previously described by <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Li <em>et al.</em> (2009)</a>. The constructed cDNA library consisted of 5.0×10<sup>6</sup> pfu, with an insert size range of 1.0–4.0 kb. Random end-sequencing of 1000 cDNA clones and comparison with <em>Arabidopsis</em> or <em>P</em>. <em>trichocarpa CesA</em> sequences identified a full-length cDNA with high similarity to <em>AtCesA4</em> (73.6%) or <em>PtiCesA4</em> (97.4%). Therefore, we named this cDNA <em>PtoCesA4</em>.</p></div><div id="sec-7" class="subsection"><h3>DNA extraction, <em>PtoCesA4</em> identification, and phylogenetic analysis</h3><p id="p-15">Total genomic DNA was extracted from young leaves with the DNeasy Plant Mini kit (Qiagen China, Shanghai). Specific primers were designed for sequencing <em>PtoCesA4</em> based on cDNA sequence; 6421 bp of genomic DNA sequences for <em>PtoCesA4</em>, including the promoter (1111 bp), were obtained by direct sequencing in the <em>P. tomentosa</em> LM50 clone, using conserved primers, and the BigDye Terminator Cycle Sequencing kit version 3.1 (Applied Biosystems, Beijing, China) run on a Li-Cor 4300 genetic analyzer (Li-Cor Biosciences, Lincoln, NE). The <em>PtoCesA4</em> sequence was deposited in GenBank under the accession number <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KC762249&amp;atom=%2Fggg%2F3%2F11%2F2069.atom">KC762249</a>.</p><p id="p-16">To analyze the phylogenetic relationship of <em>PtoCesA4</em> to the <em>CesAs</em> from other species, the amino acid sequences of CesA from <em>Arabidopsis thaliana</em>, rice (<em>Oryza sativa</em>), maize (<em>Zea mays</em>), <em>Eucalyptus grandis</em>, and <em>Populus trichocarpa</em> were identified from NCBI (<a href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>) using BLAST (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1997</a>). Phylogenetic and molecular evolutionary analyses were conducted using MEGA version 4, and the neighbor-joining (NJ) method was used to build phylogenetic trees (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Tamura <em>et al.</em> 2007</a>). Statistical confidences of the nodes of the tree are based on 1000 bootstrap replicates.</p></div><div id="sec-8" class="subsection"><h3>Tissue-specific expression analysis of <em>PtoCesA4</em></h3><div id="sec-9" class="subsection"><h4>RNA extraction and cDNA synthesis:</h4><p id="p-17">Total RNA was extracted from various tissues, including root, stem (bark, phloem, cambium, developing xylem, mature xylem), mature leaf, and apical shoot meristem of 2-year-old <em>P. tomentosa</em> clone “LM50” with the Qiagen RNAeasy kit (Qiagen China, Shanghai). Additional on-column DNase digestions were performed three times during RNA purification using RNase-Free DNase (Qiagen). RNA was quantified based on absorption at 260 nm. Quantified RNA was reverse-transcribed into cDNA with the SuperScript First-Strand Synthesis system and the supplied polythymidylate primers (Invitrogen). All cDNA samples were used for tissue-specific expression analysis of <em>PtoCesA4</em> using real-time quantitative PCR (RT–qPCR).</p></div><div id="sec-10" class="subsection"><h4>Real-time qPCR:</h4><p id="p-18">The qPCR was performed on a DNA Engine Opticon 2 machine (MJ Research) using the LightCycler-FastStar DNA master SYBR Green I kit (Roche). The real-time qPCR and the generated real-time data analysis were performed following the study of <a id="xref-ref-86-2" class="xref-bibr" href="#ref-86">Zhang <em>et al.</em> (2010a)</a>. The <em>PtoCesA4</em>-specific (F: 5′-GCCAGTCTGCAACGTCGAA-3′; R: 5′-GGAAAGCCACACACA TGAC-3′) and internal control (<em>Actin</em>) primer pairs (F: 5′-CTCCATCATGAAATGCGATG-3′; R: 5′-TTGGGGCTAGTGCTGAGATT-3′) were designed using Primer Express 3.0 (Applied Biosystems). All reactions were performed in triplicate technical and triplicate biological repetitions, respectively. The results obtained for the different tissues were standardized to the levels of <em>Actin</em>.</p></div></div><div id="sec-11" class="subsection"><h3>Transcript analysis of SNP genotypes</h3><p id="p-19">Transcript levels were determined for SNP genotypes significantly associated with phenotypic traits to test whether transcript abundance varied in the different SNP genotypic classes. Only significant SNPs [false discovery rate (FDR) <em>Q</em> ≤ 0.10] were targeted in the association and linkage populations, respectively (see Results). Transcript levels were determined by RT-qPCR with gene-specific primers. For each SNP genotypic class, 10 trees were individually sampled by obtaining secondary xylem at 1.3 m from the ground; tissue handling and RNA extractions were performed as described. The differential expression across three or two genotypic classes was tested by ANOVA.</p></div><div id="sec-12" class="subsection"><h3>SNP discovery and genotyping</h3><p id="p-20">The <em>PtoCesA4</em> gene, including 1.111 kb of the promoter, was sequenced and analyzed in 40 unrelated individuals from the <em>P. tomentosa</em> association population to identify SNPs without considering insertions/deletions (INDELs). Sequencher v.4.0 and BioEdit were used for sequence alignment, and manual editing was used to confirm sequence quality and to remove primer sequences. To identify putative SNP variants, eight clones for each individual were randomly picked for initial allele sequencing on a Li-Cor 4300 genetic analyzer (Li-Cor Biosciences, Lincoln, NE). Alignments and SNP discovery for this gene among 40 unrelated individuals described here are based on the <em>P</em>. <em>trichocarpa</em> genome (<a href="http://genome.jgi-psf.org/Poptr1/Poptr1.home.html">http://genome.jgi-psf.org/Poptr1/Poptr1.home.html</a>). All 40 sequences have been deposited in GenBank (accession no. <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KC762252&amp;atom=%2Fggg%2F3%2F11%2F2069.atom">KC762252</a>–KC762291). Subsequently, 92 common SNPs (minor allele frequencies &gt; 0.10; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/007724SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/TableS1.pdf">Table S1</a>) were genotyped by single nucleotide primer extension using a Beckman Coulter sequencing system for all DNA samples.</p></div><div id="sec-13" class="subsection"><h3>Nucleotide diversity and linkage disequilibrium</h3><div id="sec-14" class="subsection"><h4>Nucleotide diversity:</h4><p id="p-21">Diploid sequences were disambiguated into haplotypes using Phase v.2.1 using 10,000 iterations of the Bayesian Markov chain Monte Carlo chain (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Stephens and Scheet 2005</a>). We then used the phased haplotypes to estimate the number of segregating sites, nucleotide diversity, and neutrality test. The DnaSP program version 4.90.1 (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Rozas <em>et al.</em> 2003</a>) was used to calculate summary statistics for SNP polymorphisms. Nucleotide diversity was estimated using both the average number of pairwise differences per site between sequences, π (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Nei 1987</a>), and the average number of segregating sites, θw (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Watterson 1975</a>). Diversity statistics were also calculated separately for noncoding, synonymous, and nonsynonymous sites. Neutrality test statistics, Tajima’ D and Fu and Li’s D (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Tajima 1989</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Fu and Li 1993</a>), were calculated separately for three climatic regions and the complete data set and were tested using 10,000 simulations to test whether a gene or genomic region is evolving randomly (neutral evolution) or whether the region is under selection (non-neutral evolution), and the statistical significance of Tajima’s D was determined using the software DnaSP version 4.90.1.</p></div><div id="sec-15" class="subsection"><h4>Linkage disequilibrium:</h4><p id="p-22">LD was measured as the squared correlation of allele frequencies <em>r</em><sup>2</sup>, which is affected by both recombination and differences in allele frequencies between sites (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hill and Robertson 1968</a>). The <em>r</em><sup>2</sup> value between each pair of common SNPs (minor allele frequencies &gt; 0.10) in the candidate gene was calculated with 10<sup>5</sup> permutations using TASSEL v.2.0.1 (<a href="http://www.maizegenetics.net/">http://www.maizegenetics.net/</a>). To assess the extent of LD within the sequenced region of <em>PtoCesA4</em>, the decay of LD within a specific physical distance (base pairs) between common SNP sites within this gene was estimated by nonlinear regression (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Remington <em>et al.</em> 2001</a>). This analysis was performed both within three climatic regions and for the complete data set. Singletons were excluded in the LD analyses.</p></div></div><div id="sec-16" class="subsection"><h3>Marker–trait association analysis</h3><div id="sec-17" class="subsection"><h4>Single-SNP models:</h4><p id="p-23">The unified mixed linear model (MLM) was used for single SNP trait analysis, with 10<sup>4</sup> permutations in TASSEL v.2.0.1 (<a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Yu <em>et al.</em> 2006</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bradbury <em>et al.</em> 2007</a>). These phenotypes were centered and standardized before analysis. In this MLM (Q + K model) described previously, the population structure matrix (<em>Q</em>) was identified based on the significant subpopulation structure in this association population (<em>K</em> = 11) (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Du <em>et al.</em> 2012</a>). The relative kinship matrix (K) has been obtained using the method proposed by <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Ritland (1996)</a> in <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Du <em>et al.</em> (2013)</a>. Corrections for multiple testing were performed using the positive FDR with 10<sup>4</sup> permutations in QVALUE (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Storey and Tibshirani 2003</a>). The modes of gene action were quantified using the ratio of dominance (<em>d</em>) to additive (<em>a</em>) effects estimated from least-square means for each single SNP association. Details of the algorithm and formulas for calculating gene action were previously described (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Eckert <em>et al.</em> 2009</a>; <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Wegrzyn <em>et al.</em> 2010</a>).</p></div><div id="sec-18" class="subsection"><h4>Multi-SNP models:</h4><p id="p-24">Bayesian linear mixed models incorporating effects of population structure were used to construct multi-locus models for each trait (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Quesada <em>et al.</em> 2010</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Eckert <em>et al.</em> 2012</a>). These phenotypes were centered and standardized before analysis. For each trait, multi-locus models were subsequently constructed from the list of SNPs with significant effects (<em>P</em> &lt; 0.05). Model parameters, including 95% credible intervals for SNP effects, were estimated using Markov chain Monte Carlo with 50,000 steps after an initial burn-in of 10,000 steps. All linear mixed-model analyses were conducted using the Bayesian association with the missing data (BAMD) program in R (<a href="http://cran.r-project.org/package=BAMD">http://cran.r-project.org/package=BAMD</a>).</p></div><div id="sec-19" class="subsection"><h4>Haplotype analysis:</h4><p id="p-25">On the basis of the information from the LD blocks surrounding the significant SNPs (<em>P</em> &lt; 0.05) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/TableS2.pdf">Table S2</a>), the haplotype (a block of linked ordered markers) frequencies of loci genotypes were estimated based on genotypic data of 460 individuals, and haplotype-based association tests with growth and wood quality traits were performed using FAMHAP v.19 (<a href="http://famhap.meb.uni-bonn.de/index.html">http://famhap.meb.uni-bonn.de/index.html</a>). FAMHAP estimates haplotype frequencies using maximum likelihood. Singleton alleles were ignored when constructing the haplotypes, and haplotypes with a frequency &lt;5% were also discarded. The input consisted of genotype matrices with structure analysis matrices (<em>Q</em>) and phenotypic value matrices, and significances of the haplotype associations were identified based on 10<sup>4</sup> permutation tests. A correction for multiple testing was performed using the positive FDR.</p></div><div id="sec-20" class="subsection"><h4>Single-SNP linkage analysis:</h4><p id="p-26">Comparing the <em>PtoCesA4</em> sequences in parents of this linkage population (accession no. <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KC762249&amp;atom=%2Fggg%2F3%2F11%2F2069.atom">KC762249</a>–KC762252), we identified a panel of SNP markers that was based on the common SNPs detected in the association population (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/TableS1.pdf">Table S1</a>). Inheritance tests of all SNPs were first examined in the linkage population with 1200 individuals by performing a chi-square (<em>χ</em><sup>2</sup>) test at 0.01 probability, and then SNPs following Mendelian expectations (<em>P</em> ≥ 0.01) were used in single-marker analysis in the linkage mapping population (excluding the genotype data involving null alleles at each locus). Significant SNPs were calculated by fitting the data to the model <em>y</em> =<em>µ</em> + <em>m<sub>i</sub></em> + <em>e<sub>ij</sub></em>, where <em>y</em> is the trait value, <em>µ</em> is the mean, <em>m<sub>i</sub></em> is the genotype of the <em>i</em>th marker, and <em>e<sub>ij</sub></em> is the residual associated with the <em>j</em>th individual in the <em>i</em>th genotypic class. Percent phenotypic variance explained by the most significant marker was calculated, and the FDR method was used to perform a correction for multiple testing.</p></div></div></div><div class="section results" id="sec-21"><h2 class="">Results</h2><div id="sec-22" class="subsection"><h3>Identification and phylogenetic analysis of <em>PtoCesA4</em></h3><p id="p-27">We used reverse-transcription PCR to isolate a full-length cDNA of <em>PtoCesA4</em> from a cDNA library prepared from the developing xylem zone of <em>P</em>. <em>tomentosa</em>. The cDNA clone <em>PtoCesA4</em> (GenBank Accession no. <a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KC762292&amp;atom=%2Fggg%2F3%2F11%2F2069.atom">KC762292</a>) is 3757 bp in length, and the open reading frame (3129 bp) encodes a polypeptide of 1042 amino acids with an estimated molecular mass of 118.4 kD and a pI of 7.60, flanked by 297 bp of 5′untranslated leader region (5′UTR) and 331 bp of 3′UTR (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Nucleotide sequence comparison of <em>PtoCesA4</em> cDNAs with known full-length <em>Arabidopsis CesA</em> cDNA sequences revealed that <em>PtoCesA4</em> is a member of the CesA gene family because it contains all of the conserved features (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Holland <em>et al.</em> 2000</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2010</a>), such as a putative zinc-binding domain (at amino acid residues 31–76), two transmembrane helices (at residues 217–238 and 250–267) in the N-terminal region, and six transmembrane helices in the C-terminal region (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PtoCesA4 gene structure and the positions of common SNPs (minor allele frequencies &gt; 0.10). All common SNPs are represented by dark spots; putative transcription factor binding sites around SNPs in the PtoCesA4 promoter were predicted and numbers above the promoter region indicate the positions of putative transcription factor binding sites in base pairs relative to the predicted transcription start site. (A) Zinc-binding domain. (B&#x2013;I) Two transmembrane helices in the N-terminal region and six in the C-terminal region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-570814342" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;PtoCesA4&lt;/em&gt; gene structure and the positions of common SNPs (minor allele frequencies &gt; 0.10). All common SNPs are represented by dark spots; putative transcription factor binding sites around SNPs in the &lt;em&gt;PtoCesA4&lt;/em&gt; promoter were predicted and numbers above the promoter region indicate the positions of putative transcription factor binding sites in base pairs relative to the predicted transcription start site. (A) Zinc-binding domain. (B&#x2013;I) Two transmembrane helices in the N-terminal region and six in the C-terminal region.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2069/F1.medium.gif" width="440" height="119"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/11/2069/F1.medium.gif" width="440" height="119"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463488" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-28" class="first-child"><em>PtoCesA4</em> gene structure and the positions of common SNPs (minor allele frequencies &gt; 0.10). All common SNPs are represented by dark spots; putative transcription factor binding sites around SNPs in the <em>PtoCesA4</em> promoter were predicted and numbers above the promoter region indicate the positions of putative transcription factor binding sites in base pairs relative to the predicted transcription start site. (A) Zinc-binding domain. (B–I) Two transmembrane helices in the N-terminal region and six in the C-terminal region.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">To investigate the evolutionary relationship between the <em>PtoCesA4</em> and other eukaryotic <em>CesAs</em>, including genes from monocots (rice and maize), Arabidopsis, <em>Eucalyptus grandis</em>, and black cottonwood, an unrooted tree was generated from 37 full CesA protein sequences using 1000 replication bootstrap values (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/FigureS1.pdf">Figure S1</a>). Phylogenetic analysis revealed that the cloned <em>PtoCesA4</em> is an ortholog of the <em>AtCesA4</em> and <em>PtiCesA4</em> (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/FigureS1.pdf">Figure S1</a>).</p></div><div id="sec-23" class="subsection"><h3>Tissue-specific expression patterns of <em>PtoCesA4</em></h3><p id="p-30">We first determined to what extent <em>PtoCesA4</em> exhibited xylem-specific expression. Levels of <em>PtoCesA4</em> mRNA in various poplar tissues, including root, bark, phloem, cambium, developing xylem, mature xylem, mature leaf, and apical shoot meristem, wer examined by RT-qPCR with gene-specific primers and <em>Actin</em> as an internal control. <em>PtoCesA4</em> transcripts were present in all plant organs, including root, stem, and leaf, with varying patterns of expression (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/FileS1.xlsx">File S1</a>). In leaf and root, <em>PtoCesA4</em> is most abundant in mature leaf (0.8513). Low abundance is observed in apical shoot meristem and root (0.1092 and 0.1247). In the stem, <em>PtoCesA4</em> shows the highest abundance in the developing xylem tissue (0.3490), followed by the mature xylem (0.3023); it has moderate abundance in the primary tissues of the bark (0.0838) and phloem (0.0387), and the lowest abundance is found in the cambium (0.0048). Collectively, <em>PtoCesA4</em> expression in secondary tissues (xylem) was at least 60-fold higher than in primary tissues (cambium) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>), suggesting that <em>PtoCesA4</em> may be a highly expressed gene associated with secondary wall formation. The highest transcript level of this gene was found in the mature leaf (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>), suggesting that it may participate in shared pathways for assimilating the products of photosynthesis into sugars and starch, synthesize cell wall biopolymers, and the create various glycosylated compounds (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Persson <em>et al.</em> 2005</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Geisler-Lee <em>et al.</em> 2006</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relative transcript levels of PtoCesA4 in Populus tomentosa tissues and organs. The error bars represent &#xB1;SD." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-570814342" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Relative transcript levels of &lt;em&gt;PtoCesA4&lt;/em&gt; in &lt;em&gt;Populus tomentosa&lt;/em&gt; tissues and organs. The error bars represent &#xB1;SD.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2069/F2.medium.gif" width="440" height="244"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/11/2069/F2.medium.gif" width="440" height="244"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463499" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-31" class="first-child">Relative transcript levels of <em>PtoCesA4</em> in <em>Populus tomentosa</em> tissues and organs. The error bars represent ±SD.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-24" class="subsection"><h3>Nucleotide diversity in <em>PtoCesA4</em></h3><p id="p-32">To characterize the intraspecific molecular evolution of the <em>PtoCesA4</em>, an approximately 6421-bp genomic region of <em>PtoCesA4</em>, including 1111 bp of promoter region, 297 bp of 5′UTR, 3129 bp of exons, 1553 bp of intron, and 331 bp of 3′UTR, was amplified and sequenced from 40 unrelated individuals in a discovery population that encompassed most of the natural range of <em>P</em>. <em>tomentosa</em>. After definition of phased haplotypes among these 40 unrelated individuals using Phase v.2.1, a more detailed analysis of SNP variation was conducted over different regions of <em>PtoCesA4</em>, and the profile of nucleotide diversity at these loci was calculated (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). On the basis of the aligned sequences for 40 samples, 218 SNPs were detected in <em>PtoCesA4</em>, with a high frequency of 1/29 bp (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). The highest level of nucleotide polymorphism was in the promoter region, whereas the lowest was found in the exons, as expected if the coding region is conserved under selective pressure. Fifty-five SNPs were found in exons; of these, eight led to nonsynonymous changes and the other 47 SNPs were synonymous mutations (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). In total, 210 SNPs were categorized as silent sites; 92 of the 218 SNPs (42.2%) were considered common (frequency &gt; 0.10) (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463566/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463566/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Nucleotide polymorphism at the <em>PtoCesA4</em> locus</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">The <em>PtoCesA4</em> locus has high nucleotide diversity, where π<sub>T</sub> = 0.0080 and θ<sub>w</sub> = 0.0098 (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). The average levels of nucleotide diversity (π) were 0.0124 (silent sites), 0.0040 (exons), and 0.0065 (introns). Both π and θ<sub>w</sub> were higher in noncoding than in coding regions (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). In coding regions, the average levels of nucleotide diversity for nonsynonymous polymorphisms (<em>d<sub>N</sub></em>, π = 0.0009 and θ<sub>w</sub> = 0.0010) were approximately six-fold lower than for synonymous polymorphisms (<em>d<sub>S</sub></em>, π = 0.0149 and θ<sub>w</sub> = 0.0194). The <em>d<sub>N</sub></em>/<em>d<sub>S</sub></em> for the exon regions was significantly less than one, reflecting the action of purifying selection at the nonsynonymous sites in exons. Of the 218 single-base changes, 150 (68.8%) were transitions and 68 (31.2%) were transversions, and the ratio of transitions to transversions for these SNPs was approximately 2.17. Furthermore, for synonymous polymorphisms (47) in exons, 39 (80.1%) were transitions, indicating that translational selection has shaped synonymous codon usage.</p><p id="p-37">Genetic differentiation within and among three geographically independent climatic regions were studied using the nucleotide diversity data from <em>PtoCesA4</em> (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Levels of nucleotide variation (measured using π) in the three climatic regions varied but showed similar patterns of π<sub>tot</sub>, π<sub>sil</sub>, π<sub>s</sub>, and π<sub>n</sub> (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>), suggesting that the level of selective constraint was similar between the climatic regions. Tajima’s D was positive in the southern and northwestern climatic regions but was negative in the northeastern region and in the <em>P. tomentosa</em> population as a whole, but no significant departures from the neutral expectation were observed (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). The Fu and Li’s D statistical tests were positive for the northwestern region but were negative for the other regions and the <em>P. tomentosa</em> population as a whole (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>), revealing an excess of low-frequency mutations for this gene region in the <em>P. tomentosa</em> species-wide samples.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463538/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463538/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of nucleotide variations for <em>PtoCesA4</em> in <em>Populus tomentosa</em> natural populations from three climatic regions</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-25" class="subsection"><h3>Linkage disequilibrium</h3><p id="p-39">Using genotypic data for 92 common SNPs located in <em>PtoCesA4</em>, the <em>r<sup>2</sup></em> values were pooled to assess the overall behavior of LD within the <em>PtoCesA4</em> gene. The average value of <em>r<sup>2</sup></em> is 0.45 for all SNPs within the <em>PtoCesA4</em> region, with a range from 0.0 (equilibrium) to 1.0 (disequilibrium). Several high-LD distinct haplotype blocks (<em>r<sup>2</sup></em> &gt; 0.75; <em>P</em> &lt; 0.001) across the sequenced regions were shown. A higher LD level among physically linked loci (the longer haplotype block) is present in the promoter region compared with the other regions (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/FigureS2.pdf">Figure S2</a>). There are larger numbers of markers that are in linkage equilibrium among these distinct haplotype blocks (<em>r<sup>2</sup></em> &lt; 0.3) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/FigureS2.pdf">Figure S2</a>). The nonlinear regression shows a clear and rapid decline of LD with distance in base pairs within <em>PtoCesA4</em> (<em>r<sup>2</sup></em> ≥ 0.1 within 1400 bp) (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>), indicating that LD did not extend over the entire gene region. Nevertheless, within-group analyses of LD show a slightly higher level of LD within each geographical climatic region, with the <em>r</em><sup>2</sup> values declining to 0.1 within approximately 2800 bp (Southern and Northwestern regions) and approximately 3200 bp (Northeastern region) (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The decay of short-range linkage disequilibrium within PtoCesA4 for all samples and each climatic region. Pairwise correlations between SNPs are plotted against the physical distance between the SNPs in base pairs. The curves describe the nonlinear regressions of r2 (Er2) onto the physical distance in base pairs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-570814342" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The decay of short-range linkage disequilibrium within &lt;em&gt;PtoCesA4&lt;/em&gt; for all samples and each climatic region. Pairwise correlations between SNPs are plotted against the physical distance between the SNPs in base pairs. The curves describe the nonlinear regressions of &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; (Er2) onto the physical distance in base pairs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2069/F3.medium.gif" width="440" height="218"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/11/2069/F3.medium.gif" width="440" height="218"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463533" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-40" class="first-child">The decay of short-range linkage disequilibrium within <em>PtoCesA4</em> for all samples and each climatic region. Pairwise correlations between SNPs are plotted against the physical distance between the SNPs in base pairs. The curves describe the nonlinear regressions of <em>r</em><sup>2</sup> (Er2) onto the physical distance in base pairs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-26" class="subsection"><h3>Detection of phenotype-genotype associations</h3><div id="sec-27" class="subsection"><h4>Single-SNP–based associations:</h4><p id="p-41">A total of 828 tests (92 SNPs × 9 traits, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/FileS2.xlsx">File S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/FileS3.xlsx">File S3</a>) were conducted with 10<sup>4</sup> permutations using MLM. In all, 41 significant associations with all nine phenotypic traits were identified at the threshold of <em>P</em> &lt; 0.05, representing 24 SNPs from different regions within <em>PtoCesA4</em> (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/TableS2.pdf">Table S2</a>). Corrections for multiple testing using the FDR method reduced these 41 associations to 14 (<em>Q</em> &lt; 0.10) (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). These 14 associations representing 10 unique SNPs from the promoter, exon, intron, and 5′UTR regions of <em>PtoCesA4</em> were significantly associated with seven phenotypic traits, excluding fiber length and microfibril angle traits (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). These loci explained a small proportion of the phenotypic variance, ranging from 1.6% to 5.3% (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>), in accordance with polygenic quantitative models of wood traits (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Neale and Savolainen 2004</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Beaulieu <em>et al.</em> 2011</a>).</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463612/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463612/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of significant SNP marker–trait pairs identified in the <em>Populus tomentosa</em> association population using the mixed linear model after a correction for multiple testing</span><div class="sb-div caption-clear"></div></div></div><p id="p-48">Of these 10 unique SNPs, there were two nonsynonymous, two synonymous, and six noncoding SNPs (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). The nonsynonymous marker SNP49 in exon 3 results in an amino acid change from His to Asn, associated significantly with multiple traits, <em>i.e.</em>, α-cellulose (5.3%), lignin (3.0%), and H (2.3%). In this case, the mode of gene action seems additive, with the minor allele (A) conferring a lower lignin content and higher values in α-cellulose and H (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). For the other nonsynonymous marker, SNP59 in exon 6, which has the minor allele (A), results in an amino acid change from Ser to Tyr, associated significantly with fiber width (<em>R<sup>2</sup></em> = 2.6%). The genotypic effects on fiber width were significant (22.38 µm in AA, 23.21 µm for AC, and 23.99 µm for CC), consistent with the additive effect of gene action on fiber width (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). Also, a synonymous marker SNP45 in exon 2, associated with holocellulose content, showed a difference among three genotypic classes (two significant) (74.62% in AA, 74.55% in AC, and 73.30% in CC), indicating patterns of gene action consistent with dominant effects (<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>). SNP75 in exon 10, the other synonymous mutation, associated with D and V, explaining 3.2% and 2.6% of the phenotypic variance, respectively (<a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genotypic effects of the significant single nucleotide polymorphisms (SNPs) in PtoCesA4 on the same phenotypic trait in association and linkage populations. The marker SNP49 in exon 3 of PtoCesA4, a nonsynonymous mutation, which results in an encoded amino acid change from His to Asn, was significantly associated with &#x3B1;-cellulose content in association and linkage populations. The AA homozygotes were associated with higher &#x3B1;-cellulose values and CC homozygotes were associated with lower &#x3B1;-cellulose values, and mean values in AC heterozygotes were medium in both populations, which are supported by the observation that SNP49 has an additive effect on gene action in cellulose content. The nonsynonymous marker SNP59 in exon 6 of PtoCesA4 significantly associated with fiber width in both populations and shows patterns of gene action consistent with additive effects on fiber width. The A allele at SNP59 causes a Ser-to-Tyr amino acid substitution (d) SNP18 from the promoter of PtoCesA4 and showed significant association with &#x3B1;-cellulose content in both populations. The differences in &#x3B1;-cellulose content among the three genotypes of this marker indicate that patterns of gene action are consistent with overdominance effects. P1 represents the female clone YX01 (Populus alba &#xD7; Populus glandulosa), P2 represents the male clone LM 50 (Populus tomentosa), and F1 represents the hybrid progeny." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-570814342" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genotypic effects of the significant single nucleotide polymorphisms (SNPs) in &lt;em&gt;PtoCesA4&lt;/em&gt; on the same phenotypic trait in association and linkage populations. The marker SNP49 in exon 3 of &lt;em&gt;PtoCesA4&lt;/em&gt;, a nonsynonymous mutation, which results in an encoded amino acid change from His to Asn, was significantly associated with &#x3B1;-cellulose content in association and linkage populations. The AA homozygotes were associated with higher &#x3B1;-cellulose values and CC homozygotes were associated with lower &#x3B1;-cellulose values, and mean values in AC heterozygotes were medium in both populations, which are supported by the observation that SNP49 has an additive effect on gene action in cellulose content. The nonsynonymous marker SNP59 in exon 6 of &lt;em&gt;PtoCesA4&lt;/em&gt; significantly associated with fiber width in both populations and shows patterns of gene action consistent with additive effects on fiber width. The A allele at SNP59 causes a Ser-to-Tyr amino acid substitution (d) SNP18 from the promoter of &lt;em&gt;PtoCesA4&lt;/em&gt; and showed significant association with &#x3B1;-cellulose content in both populations. The differences in &#x3B1;-cellulose content among the three genotypes of this marker indicate that patterns of gene action are consistent with overdominance effects. P1 represents the female clone YX01 (&lt;em&gt;Populus alba&lt;/em&gt; &#xD7; &lt;em&gt;Populus glandulosa&lt;/em&gt;), P2 represents the male clone LM 50 (&lt;em&gt;Populus tomentosa&lt;/em&gt;), and F1 represents the hybrid progeny.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2069/F4.medium.gif" width="440" height="172"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/11/2069/F4.medium.gif" width="440" height="172"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463497" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-49" class="first-child">Genotypic effects of the significant single nucleotide polymorphisms (SNPs) in <em>PtoCesA4</em> on the same phenotypic trait in association and linkage populations. The marker SNP49 in exon 3 of <em>PtoCesA4</em>, a nonsynonymous mutation, which results in an encoded amino acid change from His to Asn, was significantly associated with α-cellulose content in association and linkage populations. The AA homozygotes were associated with higher α-cellulose values and CC homozygotes were associated with lower α-cellulose values, and mean values in AC heterozygotes were medium in both populations, which are supported by the observation that SNP49 has an additive effect on gene action in cellulose content. The nonsynonymous marker SNP59 in exon 6 of <em>PtoCesA4</em> significantly associated with fiber width in both populations and shows patterns of gene action consistent with additive effects on fiber width. The A allele at SNP59 causes a Ser-to-Tyr amino acid substitution (d) SNP18 from the promoter of <em>PtoCesA4</em> and showed significant association with α-cellulose content in both populations. The differences in α-cellulose content among the three genotypes of this marker indicate that patterns of gene action are consistent with overdominance effects. P1 represents the female clone YX01 (<em>Populus alba</em> × <em>Populus glandulosa</em>), P2 represents the male clone LM 50 (<em>Populus tomentosa</em>), and F1 represents the hybrid progeny.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">Of the remaining noncoding markers, SNP3 and SNP18 from the promoter region and SNP41 from the 5′UTR were significantly associated with α-cellulose content, with the small single SNP effects ranging from 1.6% to 2.5% (<a id="xref-table-wrap-3-9" class="xref-table" href="#T3">Table 3</a>). SNP81 in intron 10 was significantly associated with holocellulose and D traits, and this marker has the same allelic effects in these two traits: heterozygous trees (CT) for this marker showed intermediate average holocellulose content (73.78% in CT <em>vs.</em> 74.03% and 73.42% in CC and TT, respectively). An additive effect of gene action appeared in the D trait (20.26 cm in CC, 21.43 cm in CT and 22.31 cm in TT). SNP44 in intron 1 and SNP48 in intron 2 were associated with lignin content and D, respectively. Four of the 10 SNP markers exhibited significant associations with at least one trait, suggesting a pleiotropic effect of these loci (<a id="xref-table-wrap-2-5" class="xref-table" href="#T2">Table 2</a>).</p></div><div id="sec-28" class="subsection"><h4>Multi-SNP associations:</h4><p id="p-51">Application of Bayesian linear mixed models, in which each trait was evaluated against multi-SNP models, identified a multitude of new genetic associations (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/TableS3.pdf">Table S3</a>). In total, 38 associations were obtained across all traits representing nine unique growth and wood properties and 22 unique SNPs. Of the 22 unique SNPs, three were nonsynonymous, four were synonymous, and 15 were noncoding (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/TableS3.pdf">Table S3</a>). Ten of these SNPs were associated with more than one trait (range, 1–4), which is likely to be attributable to the strong correlation between some wood and growth property traits. Effect sizes for SNPs identified with the multi-locus models when analyzed using a single locus test were nearly two-fold lower than those detected only in the single-marker models (average <em>R<sup>2</sup></em> = 0.015). All of the SNPs identified as significant in the single locus tests (FDR <em>Q</em> &lt; 0.10) were revealed with significant effects in multi-SNP models (<a id="xref-table-wrap-3-10" class="xref-table" href="#T3">Table 3</a>). The number of SNPs retained in these models ranged from two (H) to seven (α-cellulose and D), with a mean of four SNPs per trait, and explained larger portions of genetic effects for many traits ranging from 3.9% to 12.4% (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Multi-locus single nucleotide polymorphism (SNP) models explain a large percentage of the phenotypic variance for growth and wood properties in the Populus tomentosa association population. The gray line and points denote the numbers of SNPs identified for each trait and the marker effects (R2) explained by the list of SNPs identified using the Bayesian mixed linear model in the Bayesian association with missing data (BAMD) program in R (http://cran.r-project.org/package=BAMD)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-570814342" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Multi-locus single nucleotide polymorphism (SNP) models explain a large percentage of the phenotypic variance for growth and wood properties in the &lt;em&gt;Populus tomentosa&lt;/em&gt; association population. The gray line and points denote the numbers of SNPs identified for each trait and the marker effects (&lt;em&gt;R&lt;sup&gt;2&lt;/sup&gt;&lt;/em&gt;) explained by the list of SNPs identified using the Bayesian mixed linear model in the Bayesian association with missing data (BAMD) program in R (http://cran.r-project.org/package=BAMD).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2069/F5.medium.gif" width="440" height="305"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/11/2069/F5.medium.gif" width="440" height="305"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463564" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-52" class="first-child">Multi-locus single nucleotide polymorphism (SNP) models explain a large percentage of the phenotypic variance for growth and wood properties in the <em>Populus tomentosa</em> association population. The gray line and points denote the numbers of SNPs identified for each trait and the marker effects (<em>R<sup>2</sup></em>) explained by the list of SNPs identified using the Bayesian mixed linear model in the Bayesian association with missing data (BAMD) program in R (<a href="http://cran.r-project.org/package=BAMD">http://cran.r-project.org/package=BAMD</a>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-29" class="subsection"><h4>Haplotype-based associations:</h4><p id="p-53">In this low-LD tree species, haplotype-based association tests were performed to identify significant haplotypes with growth and wood quality traits (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). Sixty-eight sets (based on the significant single SNPs at the threshold of <em>P</em> &lt; 0.05) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/TableS2.pdf">Table S2</a>) were analyzed with each of the nine traits, and the number of common haplotypes (frequency &gt; 1%) per set varied from 2 to 10, with an average of 5.0. Seventeen significant regions including 80 common haplotypes were identified at the significance threshold of <em>P</em> &lt; 0.05 (details not shown). Multiple test corrections reduced this number to 11, at a significance threshold of <em>Q</em> &lt; 0.10, and 21 significant haplotypes were associated with the five phenotypic traits, excluding MFA and H phenotypes (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463547/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463547/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">List of haplotypes with significant associations with wood quality and growth traits in the <em>P. tomentosa</em> association population (n = 460) after a correction for multiple testing</span><div class="sb-div caption-clear"></div></div></div><p id="p-56">Most of significantly haplotype-based associations were trait-specific, but some were shared among the traits. For instance, several haplotypes from SNP44 to SNP46 were simultaneously associated with lignin and holocellulose, which are supported by significant single SNP associations (<em>Q</em> &lt; 0.10) (<a id="xref-table-wrap-3-11" class="xref-table" href="#T3">Table 3</a> and <a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). SNP89 to SNP91, in the 3′UTR region, were associated with fiber length without a supporting single marker association (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>), and significant differences among haplotypes were observed for this trait (1.180 mm in G-G-A and 1.501 mm in C-A-A). Each haplotype explained a small proportion of phenotypic variation, from 2.6% to 5.6%, and many were strongly supported by single-SNP associations (<a id="xref-table-wrap-3-12" class="xref-table" href="#T3">Table 3</a> and <a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a>).</p></div><div id="sec-30" class="subsection"><h4>Single-SNP linkage analysis:</h4><p id="p-57">Based on all 92 common SNPs detected by association mapping we observed 56 SNPs from the <em>PtoCesA4</em> in the linkage population, including six novel alternative SNPs identified in the parents of this hybrid population and 50 corresponding to the positions of common SNPs detected in the association population (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/TableS1.pdf">Table S1</a>). Of these SNPs, 46 markers segregated in the 1200 progeny, with a segregation ratio close to 1:2:1 for 26 SNP loci and 1:1 for 20 loci (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/TableS1.pdf">Table S1</a>); the 10 significant SNP markers (<em>Q</em> &lt; 0.10) (<a id="xref-table-wrap-3-13" class="xref-table" href="#T3">Table 3</a>) identified in the association population were involved in this single-SNP linkage analysis. Therefore, 414 single-marker analyses (46 SNPs × 9 traits) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/FileS4.xlsx">File S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/FileS5.xlsx">File S5</a>) were conducted in this linkage mapping population. In all, 18 associations were first observed at the threshold of <em>P</em> &lt; 0.05 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/TableS4.pdf">Table S4</a>). However, a multiple test correction reduced this number to seven (<em>Q</em> &lt; 0.10) (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). Of these, three significant SNP markers were associated with α-cellulose content (SNP18, SNP49, and SNP75); one SNP marker each was associated with holocellulose, fiber length, and fiber width, and H traits were observed in the linkage population, with marker effects that varied from 1.5 to 3.6% (<em>Q</em> &lt; 0.10) (<a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463544/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463544/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Summary of significant SNP marker–trait pairs identified in <em>PtoCesA</em>4, using a linkage population, after correction for multiple testing errors</span><div class="sb-div caption-clear"></div></div></div><p id="p-59">SNP18 and SNP49 with α-cellulose and SNP59 with fiber width were identified in both association and linkage populations (<a id="xref-table-wrap-3-14" class="xref-table" href="#T3">Table 3</a> and <a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Table 5</a>). Because SNP49 was heterozygous in both parents (AC: AC) of the family-based linkage population, the effects of different genotype classes (AA, AC, and CC) at SNP49 for α-cellulose content were similar in both populations (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), which are supported by the observation that SNP49 has an additive effect on cellulose content (<a id="xref-table-wrap-3-15" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). Similarly, for the noncoding marker SNP18, the heterozygous trees (AT) showed higher average α-cellulose content than the homozygous trees TT (40.18% in AT and 39.20% in TT; significant), indicating that the minor allele T was dominant, and genotypic effects on α-cellulose content were consistent in both populations (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). In the linkage population, the SNP59 genotype was different in the parents (AA: AC), and the significant effects among the corresponding genotypic classes in SNP59 (20.29 µm in AA, 21.40 µm in AC) were consistent in the association population (<a id="xref-table-wrap-3-16" class="xref-table" href="#T3">Table 3</a> and <a id="xref-table-wrap-5-4" class="xref-table" href="#T5">Table 5</a>). Further analysis of the data suggested that these single markers also supported the haplotype-based associations with corresponding traits (<a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a> and <a id="xref-table-wrap-5-5" class="xref-table" href="#T5">Table 5</a>).</p></div></div><div id="sec-31" class="subsection"><h3>Transcript analysis of SNP genotypes</h3><p id="p-60">To determine whether these significant allelic SNPs affect the <em>PtoCesA4</em> RNA transcript abundance, transcript levels were compared among the different genotypic classes for the 10 significant SNPs (<em>Q</em> &lt; 0.10, <a id="xref-table-wrap-3-17" class="xref-table" href="#T3">Table 3</a>) in the association population and seven (<em>Q</em> &lt; 0.10) (<a id="xref-table-wrap-5-6" class="xref-table" href="#T5">Table 5</a>) in the linkage population using RT–qPCR with gene-specific primers. Measurement of differential transcript abundance across three or two genotypic classes (10 trees for each genotype) for each of the 17 SNPs indicated that two markers (SNP41 and 49) exhibited significant differences in the RNA transcript levels among the three genotypes in the association population, but only SNP49 was detected in the linkage population (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). The genotypic abundance ratio estimates for these SNPs had very low SEs, suggesting that these estimates are robust (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="PtoCesA4 transcript abundance varies among genotypic classes for significant SNP associations. (A) Transcript abundance variation of three genotypic classes for SNP49 in both association and linkage populations. The black and gray lines represent the transcript levels among three genotypic classes in association and linkage populations, respectively. (B) The relative mRNA transcript levels of PtoCesA4 among three genotypic classes for SNP41, a significant noncoding marker in the 5&#x2032;UTR region of PtoCesA4. The error bars represent &#xB1;SD." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-570814342" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;PtoCesA4&lt;/em&gt; transcript abundance varies among genotypic classes for significant SNP associations. (A) Transcript abundance variation of three genotypic classes for SNP49 in both association and linkage populations. The black and gray lines represent the transcript levels among three genotypic classes in association and linkage populations, respectively. (B) The relative mRNA transcript levels of &lt;em&gt;PtoCesA4&lt;/em&gt; among three genotypic classes for SNP41, a significant noncoding marker in the 5&#x2032;UTR region of &lt;em&gt;PtoCesA4&lt;/em&gt;. The error bars represent &#xB1;SD.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2069/F6.medium.gif" width="440" height="343"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/3/11/2069/F6.medium.gif" width="440" height="343"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2069/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463583" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-61" class="first-child"><em>PtoCesA4</em> transcript abundance varies among genotypic classes for significant SNP associations. (A) Transcript abundance variation of three genotypic classes for SNP49 in both association and linkage populations. The black and gray lines represent the transcript levels among three genotypic classes in association and linkage populations, respectively. (B) The relative mRNA transcript levels of <em>PtoCesA4</em> among three genotypic classes for SNP41, a significant noncoding marker in the 5′UTR region of <em>PtoCesA4</em>. The error bars represent ±SD.</p><div class="sb-div caption-clear"></div></div></div><p id="p-62">In the association population, for the marker SNP49 (exonic) associated with three traits, the highest transcript abundance was found in the AA group (0.6955), followed by the AC group (0.4307), and the transcript levels of the CC group were lowest (0.3308) (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6A</a>). In examining genotype-specific transcript levels for SNP41 (5′UTR), the heterozygous trees (0.7511 in CT) for this marker showed higher relative abundance than the homozygous trees, and the transcript level differences between SNP homozygous trees were not significant (0.5533 in CC and 0.5606 in TT) (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a>). In the linkage population, SNP49 also showed a significant difference in transcript level among three distinct genotype classes, and the transcript levels for the groups AA, AC, and CC were 0.7819, 0.6322, and 0.4010, respectively (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6A</a>). These differences in transcript abundance among genotypic classes are discussed regarding their putative function and relationship with variation in phenotypic trait.</p></div></div><div class="section discussion" id="sec-32"><h2 class="">Discussion</h2><p id="p-63"><em>PtoCesA4</em>, a highly targeted candidate gene belonging to key pathways of secondary cell wall biosynthesis, was investigated using a more comprehensive approach than previous studies of trees (<a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Wegrzyn <em>et al.</em> 2010</a>; <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Sexton <em>et al.</em> 2012</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Beaulieu <em>et al.</em> 2011</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Dillon <em>et al.</em> 2010</a>, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">2012</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Guerra <em>et al.</em> 2013</a>). Phylogenetic analysis of <em>PtoCesA4</em> and other eukaryotic <em>CesA</em>s in land plant lineages revealed that <em>PtoCesA4</em> is an ortholog of <em>AtCesA4</em> and <em>PtiCesA4</em>. Functional studies of these putative orthologs informed the structure of <em>PtoCesA4</em> and its allelic diversity affecting complex traits controlling growth and lignocellulosic biosynthesis in <em>P. tomentosa</em>. Given the rapid decay of within-gene linkage disequilibrium in <em>Populus</em>, it is better to extend SNP discovery across the full-length gene sequence, including promoter regions, to avoid missing important allelic polymorphisms. Several small-effect single-SNP or haplotype-based associations were detected in this species, indicating wood properties are quantitative traits controlled by multiple alleles, and limited statistical power for single-marker or haplotype-based association methods (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Beaulieu <em>et al.</em> 2011</a>; <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Du <em>et al.</em> 2013</a>). Therefore, using linkage-LD mapping approaches combined with transcriptomic comparison of genotypic classes of each significant SNP allowed us to examine functional <em>PtoCesA4</em> allelic variation responsible for complex quantitative traits related to lignocellulosic biosynthesis in trees.</p><div id="sec-33" class="subsection"><h3>Nucleotide diversity and LD in <em>PtoCesA4</em></h3><p id="p-64">Detailed knowledge of levels of nucleotide diversity and the extent of LD in natural populations is important for understanding the forces responsible for evolutionary change and for evaluating the precision and power of association mapping (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Ingvarsson 2008a</a>; <a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Zhang <em>et al.</em> 2010b</a>). As a prerequisite for SNP-based association mapping, a comprehensive investigation of the patterns of SNP distribution and frequency within the full-length <em>PtoCesA4</em> locus and among natural populations of <em>P. tomentosa</em> was required. Levels of average nucleotide diversity in coding regions were substantially lower than in the noncoding regions (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>), reflecting that the coding regions are conserved relative to the other regions under natural pressure. Within coding regions, the π<sub>nonsyn</sub>:π<sub>syn</sub> ratio (0.0604) was significantly less than 1 for <em>PtoCesA4</em>, which was commonly observed in natural populations of forest trees (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Krutovsky and Neale 2005</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">González-Martínez <em>et al.</em> 2007</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Ingvarsson 2008a</a>). Synonymous mutations occurring during evolution may be fixed with a higher probability than neutral ones because of purifying selection (<a id="xref-ref-87-2" class="xref-bibr" href="#ref-87">Zhang <em>et al.</em> 2010b</a>). Furthermore, an excess of transitional over transversional substitutions was found in this gene. There is a universal bias in favor of transitions over transversions, possibly as a result of the underlying chemistry of mutation, such as the relatively high rate of mutation of methylated cytosines to thymine or, particularly, the selection for codon usage bias in coding regions (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Keller <em>et al.</em> 2007</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ingvarsson 2008b</a>). For synonymous polymorphism (47) in <em>PtoCesA4</em>, 80.1% (39) were transition substitutions, indicating that translational selection has shaped synonymous codon usage. This finding was supported by the most common model of synonymous codon usage indicating that one or a few synonymous codons are preferentially used in genes with high codon bias, and such codons usually end in C or G (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Ingvarsson 2008b</a>). Moreover, the intron region harbored significantly less nucleotide diversity than synonymous sites (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>), similar to patterns recently reported for <em>P. balsamifera</em> (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Olson <em>et al.</em> 2010</a>) and <em>Medicago truncatula</em> (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Branca <em>et al.</em> 2011</a>). Introns experience higher selective constraints than synonymous coding sites, possibly because introns harbor key regions for regulation of expression (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Wray <em>et al.</em> 2003</a>; <a id="xref-ref-74-3" class="xref-bibr" href="#ref-74">Thumma <em>et al.</em> 2005</a>). Surveys of SNPs diversity in genes have been mainly focused on exons, introns, and UTRs, with less attention to promoter regions. It will be of interest to compare the level of nucleotide diversity within the promoters with a much larger diverse survey of the other gene regions. In this study, a significantly higher frequency of polymorphisms was found in the promoter region than in the other regions in <em>PtoCesA4</em> (π<sub>T</sub> = 0.0230) (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). This finding suggests that this regulatory region may be relatively unstable and a “hotspot” for genetic change (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Weickert <em>et al.</em> 2012</a>).</p><p id="p-65">In this study, the level of LD decay in <em>PtoCesA4</em> was analyzed separately within each of the three climatic regions and for the complete natural population (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>), and results showed that northwestern and southern regions seem to have experienced similar histories. The northeastern region had higher LD than the northwestern and southern regions, consistent with the higher frequency of exclusive SNPs observed in this region (<a id="xref-table-wrap-2-6" class="xref-table" href="#T2">Table 2</a>). The fine-scale LD pattern among polymorphic SNP markers in candidate genes may be influenced by gene conversion in different sampling populations. Generally, low LD might result from a species-wide scale of sampling, which incorporates the entire history of polymorphism and recombination over thousands of generations (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Morrell <em>et al.</em> 2005</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Kim <em>et al.</em> 2007</a>). Our results support this sample-scale explanation, which shows that the LD in <em>PtoCesA4</em> within three climatic regions may be more extensive than the LD found in our range-wide <em>P. tomentosa</em> samples (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>), consistent with previous studies (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Olson <em>et al.</em> 2010</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Branca <em>et al.</em> 2011</a>). However, a recent genome-wide study of the extensive LD in <em>P. trichocarpa</em> (<em>r<sup>2</sup></em> &gt; 0.2, within 3–6 kb) suggests that genome-wide association studies and genomic selection in natural populations may be more feasible in <em>Populus</em> than previously assumed (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Slavov <em>et al.</em> 2012</a>). Therefore, our future work will focus on estimation of LD decay with greater genomic coverage and exploration of the variability of haplotype structure across the entire genome. Such studies also will help to elucidate how <em>Populus</em> managed to adapt to a wide variety of environmental conditions (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Ingvarsson and Street 2011</a>).</p></div><div id="sec-34" class="subsection"><h3>Dissecting allelic polymorphisms underlying growth and wood properties</h3><p id="p-66">Estimating population structure is an important prerequisite in LD-based association analysis, and it is important to avoid false-positive results or spurious associations and to constrain association studies in association populations (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Du <em>et al.</em> 2012</a>). MLM methods have proven useful in controlling for population structure and individual relatedness within association mapping studies, with the population structure matrix (<em>Q</em>) and the relative kinship matrix (K) as the covariance. In this study, the observed effect of the population structure (when K = 11 or K = 3) (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Du <em>et al.</em> 2012</a>) on phenotypic variation is not significant, and the number and power of significant associations identified in two settings are stable (data not shown), suggesting that tree species are ideal for the fine-mapping of candidate genes and functional analysis of gene variants, because they are predominantly outcrossing, and have large, effective, relatively unstructured population sizes.</p><p id="p-67">The variation in quantity and quality of primary and secondary wall cellulose in plants is suggested to be the result of enzymatic activities of different types of cellulose synthase (CesA) (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Somerville 2006</a>). Secondary cell walls have a higher percentage of cellulose, a higher degree of polymerization, and a higher crystallinity than xylem primary walls (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Joshi <em>et al.</em> 2004</a>). Because of the importance of secondary walls in determining wood quality traits, many researchers have focused on secondary cell wall <em>CesAs</em>. In this study, <em>PtoCesA4</em> was originally isolated from a developing xylem cDNA library of <em>P. tomentosa</em> and was found to have xylem-specific expression patterns (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). Similarly, its putative ortholog in Arabidopsis (<em>AtCesA4</em>) was specifically associated with secondary cell wall development (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Atanassov <em>et al.</em> 2009</a>), and the ortholog in <em>P. trichocarpa</em> (<em>PtiCesA4</em>) also is expressed in developing xylem tissue undergoing secondary wall thickening (<a id="xref-ref-68-4" class="xref-bibr" href="#ref-68">Suzuki <em>et al.</em> 2006</a>). In addition to the direct coding of CesA subunit proteins, genetic evidence has confirmed an effect of <em>CesA</em> on wood chemical properties, influencing cellulose/hemicellulose content as well as lignin content and composition (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Song <em>et al.</em> 2010</a>; <a id="xref-ref-78-4" class="xref-bibr" href="#ref-78">Wegrzyn <em>et al.</em> 2010</a>). Several candidate genes in other pathways are also involved in synthesizing cellulose (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Szyjanowicz <em>et al.</em> 2004</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Coleman <em>et al.</em> 2009</a>). On the basis of these studies, we dissected allelic polymorphisms within <em>PtoCesA4</em>, underlying growth and wood properties, by using LD-based association in <em>P. tomentosa</em>, combined with single-SNP linkage analysis. Because of the low LD in <em>P. tomentosa</em> (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>), once a marker–trait association has been discovered and validated, it is likely that such a marker is located in close proximity to the causal polymorphisms or even the functional variant itself (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Neale and Kremer 2011</a>). Two nonsynonymous markers (SNP49 and SNP59) and a noncoding marker (SNP18) were associated with the same traits in both the association and linkage populations (<a id="xref-table-wrap-3-18" class="xref-table" href="#T3">Table 3</a> and <a id="xref-table-wrap-5-7" class="xref-table" href="#T5">Table 5</a>), confirming the value of an integrated approach for characterizing the genetic basis of wood traits. We also found that the population differentiation (<em>Fst</em> = 0.075) (<a id="xref-table-wrap-3-19" class="xref-table" href="#T3">Table 3</a>) for these significant SNPs was greater than that (<em>Fst</em> = 0.028) of all common SNPs identified in the association population, which is consistent with the report that putative functional SNPs in genes in etiologic pathways for CVD show greater population differentiation than nonfunctional SNPs (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kullo and Ding 2007</a>).</p><p id="p-68">Using single marker association, a nonsynonymous substitution in exon3 of <em>PtoCesA4</em> (SNP49) was in strong association with multiple traits (α-cellulose, lignin, and H), and the modes of gene action appeared to be additive, with the minor allele (A) conferring a lower lignin content and higher values in α-cellulose and H. This marker also was identified with these three traits in the multi-SNP analysis (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/TableS3.pdf">Table S3</a>). This is consistent with the significant phenotypic correlation between these three traits (<a id="xref-ref-18-5" class="xref-bibr" href="#ref-18">Du <em>et al.</em> 2013</a>) and also represents a pleiotropic effect of <em>PtoCesA4</em> on certain traits. Wood is composed of cellulose microfibrils embedded in a lignin–hemicellulose matrix. The observed associations of <em>PtoCesA4</em> with diverse traits suggest that <em>PtoCesA4</em> influences two distinct pathways (lignin and cellulose biosynthesis) in secondary cell wall synthesis (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Song <em>et al.</em> 2010</a>; <a id="xref-ref-18-6" class="xref-bibr" href="#ref-18">Du <em>et al.</em> 2013</a>). A similar phenomenon has been identified in previous studies (<a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Thumma <em>et al.</em> 2009</a>; <a id="xref-ref-78-5" class="xref-bibr" href="#ref-78">Wegrzyn <em>et al.</em> 2010</a>). Cellulose biosynthesis is coexpressed with other biological processes in plant vascular development, and the genes involved in these shared pathways often are functional homologs (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Somerville <em>et al.</em> 2004</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Eckert <em>et al.</em> 2012</a>). For instance, genes encoding lignin monomer-polymerizing laccases and lignin monomer synthesis enzymes are among the most closely coexpressed genes with secondary cell wall <em>AtCesA4</em>, <em>AtCes7</em>, and <em>AtCes8</em> (<a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Persson <em>et al.</em> 2005</a>). The inverse genotype effects in SNP49 between α-cellulose and lignin content might be indirectly related to carbon distribution toward the synthesis of C5 or C6 sugars (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Guerra <em>et al.</em> 2013</a>), which is in accordance with the significant negative phenotypic correlation between α-cellulose and lignin content in both <em>P. tomentosa</em> association and linkage populations (<a id="xref-ref-18-7" class="xref-bibr" href="#ref-18">Du <em>et al.</em> 2013</a>). Moreover, common haplotypes (SNPs48–SNP50) associated with α-cellulose traits surround SNP49, and this locus was identified in the multi-SNP associations (<a id="xref-table-wrap-4-7" class="xref-table" href="#T4">Table 4</a>). We also observed a significant association between SNP49 and α-cellulose in the linkage population with the same genotypic effect for this locus in association populations (<a id="xref-table-wrap-5-8" class="xref-table" href="#T5">Table 5</a> and <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>), suggesting that SNP49 may be a functional polymorphism that is in or near a locus involved in the control of α-cellulose content. This conjecture was also supported by the significant differences in expression among three genotype classes of SNP49 in either association or linkage populations (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6A</a>).</p><p id="p-69">Fibers are the most abundant secondary wall–containing cells in wood of dicot species. During secondary wall formation, highly coordinated expression of multiple genes controls cell elongation or secondary wall thickening of fibers (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Burton <em>et al.</em> 2004</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Zhong <em>et al.</em> 2006</a>). For example, <em>AtCesA7</em>/<em>IRX3</em> and <em>AtCOBL4</em>/<em>IRX6</em> are coexpressed in tissues during secondary cell wall development, and loss-of-function mutation of either of these genes causes diminished cellulose content and loss of mechanical strength of the plant body (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brown <em>et al.</em> 2005</a>). A mutant allele of <em>AtCesA7</em> in fra5 (fragile fiber 5) causes a severe decrease in cellulose content and the thickness of fibers (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Zhong <em>et al.</em> 2003</a>). Cellulose is a biopolymer that provides a major contribution to secondary cell wall formation during cell expansion and elongation (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Xie <em>et al.</em> 2011</a>). These early studies laid the research foundation for elucidation of a significant nonsynonymous association (SNP59) in exon 6 of <em>PtoCesA4</em> with fiber width in <em>P. tomentosa</em> by using linkage-LD mapping, which demonstrated modes of gene action consistent with additive effects (<a id="xref-table-wrap-3-20" class="xref-table" href="#T3">Table 3</a>, <a id="xref-table-wrap-4-8" class="xref-table" href="#T4">Table 4</a>, and <a id="xref-table-wrap-5-9" class="xref-table" href="#T5">Table 5</a>). Furthermore, a haplotype-based association with fiber width (SNP57–SNP59) suggests that this locus may be located close to causative polymorphisms. This is consistent with the finding that nonsynonymous mutations play special roles in assigning functions to specific domains or motifs of the CESA (<a id="xref-ref-88-2" class="xref-bibr" href="#ref-88">Zhong <em>et al.</em> 2003</a>). Further analyzing the protein structure encoded by <em>PtoCesA4</em>, we found that the nonsynonymous mutation of amino acid 245 (Ser to Tyr) is close to the two putative transmembrane domains (TMDs) at the N-terminus (217–238 and 250–267), which are involved in CESA protein–protein interactions (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Joshi <em>et al.</em> 2004</a>), suggesting that this nonsynonymous locus may affect the TMDs and also affect regulation of gene expression related to fiber width. CESAs are membrane-spanning proteins and small side chain residues often occur at the TMDs as a requirement of helix folding and structural stability. <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Zhang <em>et al.</em> (2009)</a> identified a missense mutation (G858R) in the fifth TMD of the rice ortholog of <em>PtoCesA4</em> (<em>OsCesA4</em>); this mutation affects protein abundance in the plasma membrane and results in abnormal cell wall biosynthesis. Additionally, novel point mutations in the TMD have also been reported to affect cellulose synthesis in <em>Arabidopsis</em> (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chen <em>et al.</em> 2005</a>).</p><p id="p-70">Many functional analyses of SNPs have examined coding regions and splicing sites in candidate genes related to wood traits that can alter proteins and mRNA splicing. However, SNPs in noncoding regulatory regions can also influence important biological regulation (<a id="xref-ref-73-4" class="xref-bibr" href="#ref-73">Thumma <em>et al.</em> 2009</a>; <a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Beaulieu <em>et al.</em> 2011</a>). <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">d’Alesio <em>et al.</em> (2005)</a> detected that several SNPs are predicted to be related to genes by influencing the binding affinity of transcription factors in the promoter region. In this work, we detected a significant marker (SNP 18) at 273 bp (T/A) upstream of the transcriptional start site of the <em>PtoCesA4</em> promoter (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a>). Genotypic effect analysis of α-cellulose content in either association or linkage population showed that the trees heterozygous (AT) for this marker showed higher average α-cellulose content than the homozygous trees (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>), indicating overdominance. Moreover, a common haplotype (SNP16–SNP18) associated with α-cellulose traits and SNP18 were also determined by using the multi-SNPs association model (<a id="xref-table-wrap-4-9" class="xref-table" href="#T4">Table 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1/TableS3.pdf">Table S3</a>). These results support that SNP18 might have a regulatory effect on <em>PtoCesA4</em> expression, or it could to be in very strong LD with a nearby regulatory polymorphism; the detailed regulatory mechanisms of this locus will require further investigation. Although the mutation in the 5′ flanking region did not result in an amino acid changes, phenotypic traits can be affected because 5′UTRs play crucial roles in the regulation of gene expression, especially for transcriptional mRNA stability, translational efficiency, or subcellular localization (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Miyamoto <em>et al.</em> 2007</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Lin and Li 2012</a>). In our study, SNP41, located in the 5′UTR of <em>PtoCesA4</em>, had a significant association (<em>Q</em> &lt; 0.10) with α-cellulose in the association populations (<a id="xref-table-wrap-3-21" class="xref-table" href="#T3">Table 3</a>). The estimated allelic effects of SNP41 on α-cellulose corresponded well with estimates of transcript levels (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6B</a>), <em>i.e.</em>, heterozygotes (CT) had both higher average α-cellulose content and higher transcript levels than the homozygotes (CC and TT), suggesting that SNP41 may be a functional polymorphism affecting the regulation of gene expression. However, our follow-up study of this SNP in the linkage population did not support this observation. Similarly, no replication case has been reported in several previous association studies of wood traits (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Dillon <em>et al.</em> 2010</a>, <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">2012</a>; <a id="xref-ref-18-8" class="xref-bibr" href="#ref-18">Du <em>et al.</em> 2013</a>). The differences between association and linkage populations may explain the “lack of validation” for this association, including their genetic background, complex gene–environment interactions, mapping resolution, population structure, and age-dependent effects (<a id="xref-ref-18-9" class="xref-bibr" href="#ref-18">Du <em>et al.</em> 2013</a>). In this study, the linkage population has a limited genomic data set from both the parents and the interspecific genetic background of the female parent. Therefore, improved power to detect and validate associations in future experiments could be achieved by establishing validation populations with families, or clonal material, from the discovery population of the same species (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Dillon <em>et al.</em> 2010</a>).</p></div><div id="sec-35" class="subsection"><h3>Conclusions</h3><p id="p-71">Tissue-specific expression profiles revealed that <em>PtoCesA4</em> is highly expressed during secondary cell wall formation. Therefore, selection of optimal candidate genes through different approaches, such as EST database searches, transcript abundance profiles, QTL mapping, and comparison of orthologs in a model or related species, is very important for identifying useful alleles located within functional genes controlling traits of interest (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Neale and Savolainen 2004</a>; <a id="xref-ref-74-4" class="xref-bibr" href="#ref-74">Thumma <em>et al.</em> 2005</a>). Our work revealed that the greater length may lead to slightly higher LD than candidate genes analyzed previously in <em>Populus</em> and combined with a recent genome-wide study of LD in <em>P. trichocarpa</em> (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Slavov <em>et al.</em> 2012</a>), suggesting that LD studies in <em>P. tomentosa</em> should focus on a better understanding of the variability of haplotype structure across the entire genome. Comparatively, the preliminary application of multi-SNP analysis in <em>PtoCesA4</em> suggests that it will be promising to conduct association studies with virtually all the related genes that share biological pathways and to have a more complete understanding of the genetic architecture of quantitative variation (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Eckert <em>et al.</em> 2012</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Resende <em>et al.</em> 2012</a>). Wood quality traits are quantitative traits controlled by multiple genes, with a moderate to high degree of heritability; however, growth traits have relatively low heritability compared with wood property traits (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Thumma <em>et al.</em>, 2010</a>). Therefore, several significant SNP associations with wood traits, obtained using linkage-LD mapping approaches combined with RNA transcript abundance among the genotypes of each significant SNP, represent important progress toward the identification of allelic variation responsible for wood traits and the development of successful marker-aided selection in trees. In the coming years, the rapid development of high-throughput sequencing is very likely to drive association studies toward genome-wide studies in trees.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-72">We thank Professors Ronald R. Sederoff, Zhao-Bang Zeng, and Ross W. Whetten (North Carolina State University, Raleigh, NC), Harry Wu (Umeå Plant Science Centre, Swedish University of Agricultural Sciences, Umeå, Sweden), and Rongling Wu (Center for Statistical Genetics, Pennsylvania State University, Hershey, Pennsylvania) for their detailed comments and specific suggestions for improving the manuscript. This work was supported by the grants from the Introduction of Foreign Advanced Agricultural Science and Technology into China (no. 2009-4-22), the State Key Basic Research Program of China (no. 2012CB114506), and the Project of the National Natural Science Foundation of China (no. 31170622, 30872042).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007724/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Sequence data from this article has been deposited in the GenBank Data Library under the accession no..<a class="external-ref external-ref-type-gen" href="/lookup/external-ref?link_type=GEN&amp;access_num=KC762249&amp;atom=%2Fggg%2F3%2F11%2F2069.atom">KC762249</a>–KC762292</p></li><li class="fn" id="fn-17"><p id="p-73"><em>Communicating editor: D.-J. De Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-07-15"><span class="received-label">Received </span>July 15, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-09-11"><span class="accepted-label">Accepted </span>September 11, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Du <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.1" data-doi="10.1093/nar/25.17.3389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madden</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffer</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">3389</span>–<span class="cit-lpage">3402</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F25.17.3389%26rft_id%253Dinfo%253Apmid%252F9254694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xNy8zMzg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.2" data-doi="10.1074/jbc.M807456200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atanassov</span>  <span class="cit-name-given-names">I. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pittman</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Elucidating the mechanisms of assembly and subunit interaction of the cellulose synthase complex of <em>Arabidopsis</em> secondary cell walls.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">3833</span>–<span class="cit-lpage">3841</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M807456200%26rft_id%253Dinfo%253Apmid%252F19056734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEwOiIyODQvNi8zODMzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.3" data-doi="10.1038/nature08800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atwell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjalmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willems</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association study of 107 phenotypes in a common set of <em>Arabidopsis thaliana</em> inbred lines.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">465</span>: <span class="cit-fpage">627</span>–<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAtwell%26rft.auinit1%253DS.%26rft.volume%253D465%26rft.issue%253D7298%26rft.spage%253D627%26rft.epage%253D631%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2B107%2Bphenotypes%2Bin%2BArabidopsis%2Bthaliana%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08800%26rft_id%253Dinfo%253Apmid%252F20336072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20336072&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278249000044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.4" data-doi="10.1534/genetics.110.125781"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaulieu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerksen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clement</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deslauriers</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Association genetics of wood physical traits in the conifer white spruce and relationships with gene expression.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">197</span>–<span class="cit-lpage">214</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.125781%26rft_id%253Dinfo%253Apmid%252F21385726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzEvMTk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.5" data-doi="10.1371/journal.pgen.1000940"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brachi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faure</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flahauw</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Linkage and association mapping of <em>Arabidopsis thaliana</em> flowering time in nature.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000940</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DBrachi%26rft.auinit1%253DB.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De1000940%26rft.epage%253De1000940%26rft.atitle%253DLinkage%2Band%2Bassociation%2Bmapping%2Bof%2BArabidopsis%2Bthaliana%2Bflowering%2Btime%2Bin%2Bnature.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000940%26rft_id%253Dinfo%253Apmid%252F20463887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000940&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20463887&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.6" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTkvMjYzMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.7" data-doi="10.1073/pnas.1104032108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Branca</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paape</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briskine</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farmer</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Whole-genome nucleotide diversity, recombination, and linkage disequilibrium in the model legume <em>Medicago truncatula</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">E864</span>–<span class="cit-lpage">E870</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1104032108%26rft_id%253Dinfo%253Apmid%252F21949378%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzQyL0U4NjQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.8" data-doi="10.1105/tpc.105.031542"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeef</span>  <span class="cit-name-given-names">L. A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodacreb</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Identification of novel genes in Arabidopsis involved in secondary cell wall formation using expression profiling and reverse genetics.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">2281</span>–<span class="cit-lpage">2295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.105.031542%26rft_id%253Dinfo%253Apmid%252F15980264%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE3LzgvMjI4MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.9" data-doi="10.1104/pp.103.032904"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burton</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shirley</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harvey</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fincher</span>  <span class="cit-name-given-names">G. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The <em>CesA</em> gene family of barley (<em>Hordeum vulgare</em>): quantitative analysis of transcripts reveals two groups of co-expressed genes.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">224</span>–<span class="cit-lpage">236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.103.032904%26rft_id%253Dinfo%253Apmid%252F14701917%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMzQvMS8yMjQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.10" data-doi="10.1073/pnas.1012348107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ehrhardt</span>  <span class="cit-name-given-names">D. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somerville</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mutations of cellulose synthase (CESA1) phosphorylation sites modulate anisotropic cell expansion and bidirectional mobility of cellulose synthase.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">17188</span>–<span class="cit-lpage">17193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1012348107%26rft_id%253Dinfo%253Apmid%252F20855602%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzQwLzE3MTg4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.11" data-doi="10.1111/j.1365-313X.2005.02452.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Disruption of the cellulose synthase gene, <em>AtCesA8/IRX1</em>, enhances drought and osmotic stress tolerance in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DChen%26rft.auinit1%253DZ.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D273%26rft.epage%253D283%26rft.atitle%253DDisruption%2Bof%2Bthe%2Bcellulose%2Bsynthase%2Bgene%252C%2BAtCesA8%252FIRX1%252C%2Benhances%2Bdrought%2Band%2Bosmotic%2Bstress%2Btolerance%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2005.02452.x%26rft_id%253Dinfo%253Apmid%252F15998313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2005.02452.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15998313&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230729700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.12" data-doi="10.1073/pnas.0900188106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Coleman</span>  <span class="cit-name-given-names">H. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mansfield</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Sucrose synthase affects carbon partitioning to increase cellulose production and altered cell wall ultrastructure.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">13118</span>–<span class="cit-lpage">13123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0900188106%26rft_id%253Dinfo%253Apmid%252F19625620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzMxLzEzMTE4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.13" data-doi="10.1093/hmg/ddi382"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">d’Alesio</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garabedian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sabatier</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guaydier-Souquieres</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcelli</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Two single-nucleotide polymorphisms in the human vitamin D receptor promoter change protein-DNA complex formation and are associated with height and vitamin D status in adolescent girls.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">3539</span>–<span class="cit-lpage">3548</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddi382%26rft_id%253Dinfo%253Apmid%252F16210379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjEwOiIxNC8yMi8zNTM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.14" data-doi="10.1534/genetics.110.116582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dillon</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H. X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Allelic variation in cell wall candidate genes affecting solid wood properties in association populations and land races of <em>Pinus radiata</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">185</span>: <span class="cit-fpage">1477</span>–<span class="cit-lpage">1487</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.116582%26rft_id%253Dinfo%253Apmid%252F20498299%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4NS80LzE0NzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.15" data-doi="10.1111/j.1469-8137.2012.04200.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dillon</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brawner</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meder</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Southerton</span>  <span class="cit-name-given-names">S. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Association genetics in <em>Corymbia citriodora</em> subsp. Variegate identifies single nucleotide polymorphisms affecting wood growth and cellulosic pulp yield.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">596</span>–<span class="cit-lpage">608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DDillon%26rft.auinit1%253DS.%2BK.%26rft.volume%253D195%26rft.issue%253D3%26rft.spage%253D596%26rft.epage%253D608%26rft.atitle%253DAssociation%2Bgenetics%2Bin%2BCorymbia%2Bcitriodora%2Bsubsp.%2Bvariegata%2Bidentifies%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Baffecting%2Bwood%2Bgrowth%2Band%2Bcellulosic%2Bpulp%2Byield.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2012.04200.x%26rft_id%253Dinfo%253Apmid%252F22680066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2012.04200.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22680066&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.16" data-doi="10.1023/B:CELL.0000046408.84510.06"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Djerbi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aspeborg</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nilsson</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sundberg</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mellerowicz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Identification and expression analysis of genes encoding putative cellulose synthases (CesA) in the hybrid aspen, <em>Populus tremula</em> (L.) × <em>P. tremuloides</em> (Michx.).</span> <abbr class="cit-jnl-abbrev">Cellulose</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellulose%26rft.volume%253D11%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1023%252FB%253ACELL.0000046408.84510.06%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/B:CELL.0000046408.84510.06&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000225573000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.17" data-doi="10.1093/jhered/ess061"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic diversity and population structure of Chinese white poplar (<em>Populus tomentosa</em>) revealed by SSR markers.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">862</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BHered.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fess061%26rft_id%253Dinfo%253Apmid%252F23008443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjk6IjEwMy82Lzg1MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.18" data-doi="10.1111/nph.12072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Polymorphic simple sequence repeat (SSR) loci within cellulose synthase (<em>PtoCesA</em>) genes are associated with growth and wood properties in <em>Populus tomentosa</em>.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">763</span>–<span class="cit-lpage">776</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D197%26rft.spage%253D763%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12072%26rft_id%253Dinfo%253Apmid%252F23278184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23278184&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313266000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.19" data-doi="10.1534/genetics.109.102350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pande</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jermstad</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Association genetics of coastal Douglas fir (<em>Pseudotsuga menziesii</em> var. <em>menziesii</em>, Pinaceae). I. Cold hardiness related traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">1289</span>–<span class="cit-lpage">1302</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.102350%26rft_id%253Dinfo%253Apmid%252F19487566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Mi80LzEyODkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.20" data-doi="10.1111/j.1469-8137.2011.03976.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cumbie</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldfarb</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Association genetics of the loblolly pine (<em>Pinus taeda</em>, Pinaceae) metabolome.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">890</span>–<span class="cit-lpage">902</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DEckert%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D193%26rft.issue%253D4%26rft.spage%253D890%26rft.epage%253D902%26rft.atitle%253DAssociation%2Bgenetics%2Bof%2Bthe%2Bloblolly%2Bpine%2B%2528Pinus%2Btaeda%252C%2BPinaceae%2529%2Bmetabolome.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.03976.x%26rft_id%253Dinfo%253Apmid%252F22129444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.03976.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22129444&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299778300011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Statistical tests of neutrality of mutations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">133</span>: <span class="cit-fpage">693</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFu%26rft.auinit1%253DY.%2BX.%26rft.volume%253D133%26rft.issue%253D3%26rft.spage%253D693%26rft.epage%253D709%26rft.atitle%253DStatistical%2Btests%2Bof%2Bneutrality%2Bof%2Bmutations.%26rft_id%253Dinfo%253Apmid%252F8454210%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMzLzMvNjkzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.22" data-doi="10.1104/pp.105.072652"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Geisler-Lee</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geisler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coutinho</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Segerman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishikubo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Poplar carbohydrate-active enzymes. Gene identification and expression analyses.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">946</span>–<span class="cit-lpage">962</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.105.072652%26rft_id%253Dinfo%253Apmid%252F16415215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNDAvMy85NDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.23" data-doi="10.1534/genetics.106.061127"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Martínez</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wheeler</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Association genetics in <em>Pinus taeda</em> L. I. Wood property traits.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">399</span>–<span class="cit-lpage">409</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.061127%26rft_id%253Dinfo%253Apmid%252F17110498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc1LzEvMzk5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.24" data-doi="10.1111/nph.12003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guerra</span>  <span class="cit-name-given-names">F. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sykes</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanton</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Association genetics of chemical wood properties in black poplar (<em>Populus nigra</em>).</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">162</span>–<span class="cit-lpage">176</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DGuerra%26rft.auinit1%253DF.%2BP.%26rft.volume%253D197%26rft.issue%253D1%26rft.spage%253D162%26rft.epage%253D176%26rft.atitle%253DAssociation%2Bgenetics%2Bof%2Bchemical%2Bwood%2Bproperties%2Bin%2Bblack%2Bpoplar%2B%2528Populus%2Bnigra%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.12003%26rft_id%253Dinfo%253Apmid%252F23157484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.12003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23157484&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311606800019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.25" data-doi="10.1007/BF01245622"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1968</span> <span class="cit-article-title">Linkage disequilibrium in finite populations.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">226</span>–<span class="cit-lpage">231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D38%26rft.spage%253D226%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01245622%26rft_id%253Dinfo%253Apmid%252F24442307%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01245622&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24442307&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.26" data-doi="10.1104/pp.123.4.1313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Helentjaris</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhugga</span>  <span class="cit-name-given-names">K. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xoconostle-Cazares</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A comparative analysis of the plant cellulose synthase (CesA) gene family.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">1313</span>–<span class="cit-lpage">1324</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DHolland%26rft.auinit1%253DN.%26rft.volume%253D123%26rft.issue%253D4%26rft.spage%253D1313%26rft.epage%253D1324%26rft.atitle%253DA%2BComparative%2BAnalysis%2Bof%2Bthe%2BPlant%2BCellulose%2BSynthase%2B%2528CesA%2529%2BGene%2BFamily%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.123.4.1313%26rft_id%253Dinfo%253Apmid%252F10938350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTIzLzQvMTMxMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Z. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The study on the climatic regionalization of the distributional region of <em>Populus tomentosa</em>.</span> <abbr class="cit-jnl-abbrev">J Beijing Forestry Univ</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">26</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ%2BBeijing%2BForestry%2BUniv%26rft.volume%253D14%26rft.spage%253D26%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.28" data-doi="10.1534/genetics.108.090431"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">a</span> <span class="cit-article-title">Multilocus patterns of nucleotide polymorphism and the demographic history of <em>Populus tremula</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090431%26rft_id%253Dinfo%253Apmid%252F18716330%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvMzI5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.29" data-doi="10.1186/1471-2148-8-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">P. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">b</span> <span class="cit-article-title">Molecular evolution of synonymous codon usage in <em>Populus</em>.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">13</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bevolutionary%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BEvol%2BBiol%26rft.aulast%253DDemiguel%26rft.auinit1%253DD.%26rft.volume%253D8%26rft.spage%253D13%26rft.epage%253D13%26rft.atitle%253DAncestral%2Bfeeding%2Bstate%2Bof%2Bruminants%2Breconsidered%253A%2Bearliest%2Bgrazing%2Badaptation%2Bclaims%2Ba%2Bmixed%2Bcondition%2Bfor%2BCervidae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-8-13%26rft_id%253Dinfo%253Apmid%252F18205907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-8-13&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18205907&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.30" data-doi="10.1111/j.1469-8137.2010.03593.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ingvarsson</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Street</span>  <span class="cit-name-given-names">N. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Association genetics of complex traits in plants.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">909</span>–<span class="cit-lpage">922</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DIngvarsson%26rft.auinit1%253DP.%2BK.%26rft.volume%253D189%26rft.issue%253D4%26rft.spage%253D909%26rft.epage%253D922%26rft.atitle%253DAssociation%2Bgenetics%2Bof%2Bcomplex%2Btraits%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2010.03593.x%26rft_id%253Dinfo%253Apmid%252F21182529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2010.03593.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21182529&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286940500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.31" data-doi="10.1111/j.1469-8137.2004.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhandari</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranjan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kalluri</span>  <span class="cit-name-given-names">U. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genomics of cellulose biosynthesis in poplars.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">61</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D164%26rft.spage%253D53%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2004.01155.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2004.01155.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000223662000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.32" data-doi="10.1371/journal.pgen.0030022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bensasson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nichols</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Transition-transversion bias is not universal: A counter example from grasshopper pseudogenes.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DKeller%26rft.auinit1%253DI.%26rft.volume%253D3%26rft.issue%253D2%26rft.spage%253De22%26rft.epage%253De22%26rft.atitle%253DTransition-transversion%2Bbias%2Bis%2Bnot%2Buniversal%253A%2Ba%2Bcounter%2Bexample%2Bfrom%2Bgrasshopper%2Bpseudogenes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0030022%26rft_id%253Dinfo%253Apmid%252F17274688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0030022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17274688&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.33" data-doi="10.1038/ng2115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Plagnol</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toomajian</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recombination and linkage disequilibrium in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKim%26rft.auinit1%253DS.%26rft.volume%253D39%26rft.issue%253D9%26rft.spage%253D1151%26rft.epage%253D1155%26rft.atitle%253DRecombination%2Band%2Blinkage%2Bdisequilibrium%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2115%26rft_id%253Dinfo%253Apmid%252F17676040%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17676040&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.34" data-doi="10.1534/genetics.105.044420"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krutovsky</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Nucleotide diversity and linkage disequilibrium in cold-hardiness-and wood quality-related candidate genes in Douglas Fir.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">2029</span>–<span class="cit-lpage">2041</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.044420%26rft_id%253Dinfo%253Apmid%252F16157674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MS80LzIwMjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.35" data-doi="10.1186/1471-2156-8-48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kullo</span>  <span class="cit-name-given-names">I. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Patterns of population differentiation of candidate genes for cardiovascular disease</span>. <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DKullo%26rft.auinit1%253DI.%2BJ.%26rft.volume%253D8%26rft.spage%253D48%26rft.epage%253D48%26rft.atitle%253DPatterns%2Bof%2Bpopulation%2Bdifferentiation%2Bof%2Bcandidate%2Bgenes%2Bfor%2Bcardiovascular%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-8-48%26rft_id%253Dinfo%253Apmid%252F17626638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-8-48&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17626638&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.36" data-doi="10.1016/j.tplants.2009.02.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thammannagowda</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bulone</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">An update on the nomenclature for the cellulose synthase genes in <em>Populus</em>.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">248</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DKumar%26rft.auinit1%253DM.%26rft.volume%253D14%26rft.issue%253D5%26rft.spage%253D248%26rft.epage%253D254%26rft.atitle%253DAn%2Bupdate%2Bon%2Bthe%2Bnomenclature%2Bfor%2Bthe%2Bcellulose%2Bsynthase%2Bgenes%2Bin%2BPopulus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2009.02.004%26rft_id%253Dinfo%253Apmid%252F19375973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2009.02.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19375973&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266367600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">V. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A genomic and molecular view of wood formation.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Plant Sci.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit.%2BRev.%2BPlant%2BSci.%26rft.volume%253D25%26rft.spage%253D213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.38" data-doi="10.1186/1471-2164-10-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dillon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Southerton</span>  <span class="cit-name-given-names">S. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Generation and analysis of expressed sequence tags from six developing xylem libraries in <em>Pinus radiata</em> D.Don.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">18</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DJaari%26rft.auinit1%253DS.%26rft.volume%253D10%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DA%2Bfirst-generation%2Bmicrosatellite-based%2Bgenetic%2Blinkage%2Bmap%2Bof%2Bthe%2BSiberian%2Bjay%2B%2528Perisoreus%2Binfaustus%2529%253A%2Binsights%2Binto%2Bavian%2Bgenome%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-1%26rft_id%253Dinfo%253Apmid%252F19121221%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19121221&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.39" data-doi="10.1073/pnas.1006105107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Joint linkage-linkage disequilibrium mapping is a powerful approach to detect quantitative trait loci underlying drought tolerance in maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">19585</span>–<span class="cit-lpage">19590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1006105107%26rft_id%253Dinfo%253Apmid%252F20974948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzQ1LzE5NTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.40" data-doi="10.1093/molbev/msr143"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Evolution of 5′untranslated region length and gene expression reprogramming in yeasts.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">89</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr143%26rft_id%253Dinfo%253Apmid%252F21965341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjI5LzEvODEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.41" data-doi="10.1371/journal.pgen.1000331"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manenti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galvan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pettinicchio</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trincucci</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spada</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mouse genome-wide association mapping needs linkage analysis to avoid false-positive loci.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DManenti%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253De1000331%26rft.epage%253De1000331%26rft.atitle%253DMouse%2Bgenome-wide%2Bassociation%2Bmapping%2Bneeds%2Blinkage%2Banalysis%2Bto%2Bavoid%2Bfalse-positive%2BLoci.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000331%26rft_id%253Dinfo%253Apmid%252F19132132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000331&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19132132&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.42" data-doi="10.1038/2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miyamoto</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mabuchi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kubo</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takatori</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A functional polymorphism in the 5′UTR of GDF5 is associated with susceptibility to osteoarthritis.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">533</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHelgason%26rft.auinit1%253DA.%26rft.volume%253D39%26rft.issue%253D4%26rft.spage%253D529%26rft.epage%253D533%26rft.atitle%253DA%2Bfunctional%2Bpolymorphism%2Bin%2Bthe%2B5%2527%2BUTR%2Bof%2BGDF5%2Bis%2Bassociated%2Bwith%2Bsusceptibility%2Bto%2Bosteoarthritis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F2005%26rft_id%253Dinfo%253Apmid%252F17384641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/2005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17384641&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245271200024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.43" data-doi="10.1073/pnas.0409804102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toleno</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lundy</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">M. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Low levels of linkage disequilibrium in wild barley (<em>Hordeum vulgare ssp. spontaneum</em>) despite high rates of self-fertilization.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">2442</span>–<span class="cit-lpage">2447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0409804102%26rft_id%253Dinfo%253Apmid%252F15699350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAyLzcvMjQ0MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.44" data-doi="10.1105/tpc.109.068437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myles</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Association mapping: Critical considerations shift from genotyping to experimental design.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">2194</span>–<span class="cit-lpage">2202</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.109.068437%26rft_id%253Dinfo%253Apmid%252F19654263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMjE5NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.45" data-doi="10.1016/j.tplants.2004.05.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Savolainen</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Association genetics of complex traits in conifers.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">325</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DNeale%26rft.auinit1%253DD.%2BB.%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D325%26rft.epage%253D330%26rft.atitle%253DAssociation%2Bgenetics%2Bof%2Bcomplex%2Btraits%2Bin%2Bconifers.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2004.05.006%26rft_id%253Dinfo%253Apmid%252F15231277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2004.05.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15231277&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223010600004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.46" data-doi="10.1038/nrg2931"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neale</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kremer</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Forest tree genomics: growing resources and applications.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNeale%26rft.auinit1%253DD.%2BB.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D111%26rft.epage%253D122%26rft.atitle%253DForest%2Btree%2Bgenomics%253A%2Bgrowing%2Bresources%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2931%26rft_id%253Dinfo%253Apmid%252F21245829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2931&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21245829&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2069.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-source">Molecular evolutionary genetics</span>, <span class="cit-publ-name">Columbia University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.48" data-doi="10.1111/j.1469-8137.2009.03174.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olson</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takebayashi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Silim</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schroeder</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Nucleotide diversity and linkage disequilibrium in balsam poplar (<em>Populus balsamifera</em>).</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">526</span>–<span class="cit-lpage">536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DOlson%26rft.auinit1%253DM.%2BS.%26rft.volume%253D186%26rft.issue%253D2%26rft.spage%253D526%26rft.epage%253D536%26rft.atitle%253DNucleotide%2Bdiversity%2Band%2Blinkage%2Bdisequilibrium%2Bin%2Bbalsam%2Bpoplar%2B%2528Populus%2Bbalsamifera%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2009.03174.x%26rft_id%253Dinfo%253Apmid%252F20122131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2009.03174.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20122131&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276024900031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.49" data-doi="10.1073/pnas.0503392102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Persson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Milne</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Page</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somervill</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Identification of genes required for cellulose synthesis by regression analysis of public microarray data sets.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">8633</span>–<span class="cit-lpage">8638</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0503392102%26rft_id%253Dinfo%253Apmid%252F15932943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAyLzI0Lzg2MzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.50" data-doi="10.1073/pnas.0706592104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Persson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paredez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palsdottir</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doblin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic evidence for three unique components in primary cell-wall cellulose synthase complexes in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">15566</span>–<span class="cit-lpage">15571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0706592104%26rft_id%253Dinfo%253Apmid%252F17878302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzM5LzE1NTY2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.51" data-doi="10.1534/genetics.110.117549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quesada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gopal</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cumbie</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Association mapping of quantitative disease resistance in a natural population of loblolly pine (<em>Pinus taeda</em> L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">677</span>–<span class="cit-lpage">686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.117549%26rft_id%253Dinfo%253Apmid%252F20628037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNjc3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.52" data-doi="10.1073/pnas.201394398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Remington</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thornsberry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matsuoka</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitt</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Structure of linkage disequilibrium and phenotypic associations in the maize genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">11479</span>–<span class="cit-lpage">11484</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.201394398%26rft_id%253Dinfo%253Apmid%252F11562485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTgvMjAvMTE0NzkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrick</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernando</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Accuracy of genomic selection methods in a standard data set of Loblolly Pine (<em>Pinus taeda</em> L.).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">695</span>–<span class="cit-lpage">708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D188%26rft.spage%253D695%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.54" data-doi="10.1017/S0016672300033620"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ritland</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Estimators for pairwise relatedness and individual inbreeding coefficients.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">185</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BRes.%26rft.volume%253D67%26rft.spage%253D175%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300033620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300033620&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996UK49900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.55" data-doi="10.1093/bioinformatics/btg359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sa’nchez-Delbarrio</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messeguer</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">DnaSP, DNA polymorphism analyses by the coalescent and other methods.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">2496</span>–<span class="cit-lpage">2497</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DRozas%26rft.auinit1%253DJ.%26rft.volume%253D19%26rft.issue%253D18%26rft.spage%253D2496%26rft.epage%253D2497%26rft.atitle%253DDnaSP%252C%2BDNA%2Bpolymorphism%2Banalyses%2Bby%2Bthe%2Bcoalescent%2Band%2Bother%2Bmethods%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg359%26rft_id%253Dinfo%253Apmid%252F14668244%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMTkvMTgvMjQ5NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.56" data-doi="10.1139/x04-119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schimleck</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kube</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raymond</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Genetic improvement of kraft pulp yield in <em>Eucalyptus nitens</em> using cellulose content determined by near infrared spectroscopy.</span> <abbr class="cit-jnl-abbrev">Can. J. For. Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">2363</span>–<span class="cit-lpage">2370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BFor.%2BRes.%26rft.volume%253D34%26rft.spage%253D2363%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fx04-119%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/x04-119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.57" data-doi="10.1104/pp.111.181602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sexton</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harwood</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McManus</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title"><em>Pectin methyltransferase</em> genes influence solid wood properties of <em>Eucalyptus pilularis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">531</span>–<span class="cit-lpage">541</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.181602%26rft_id%253Dinfo%253Apmid%252F22052017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTgvMS81MzEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shriner</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaughan</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padilla</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiwari</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Problems with genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">316</span>: <span class="cit-fpage">1840</span>–<span class="cit-lpage">1842</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWilliams%26rft.auinit1%253DS.%2BM.%26rft.volume%253D316%26rft.issue%253D5833%26rft.spage%253D1840%26rft.epage%253D1842%26rft.atitle%253DProblems%2Bwith%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Apmid%252F17605173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17605173&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.59" data-doi="10.1111/j.1469-8137.2012.04258.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slavov</span>  <span class="cit-name-given-names">G. T.</span></span></li></ol><cite>, S. P. DiFazio, J. Martin, W. Schackwitz, W. Muchero <em>et al</em>., <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome resequencing reveals multiscale geographic structure and extensive linkage disequilibrium in the forest tree <em>Populus trichocarpa</em>.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">196</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">725</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSlavov%26rft.auinit1%253DG.%2BT.%26rft.volume%253D196%26rft.issue%253D3%26rft.spage%253D713%26rft.epage%253D725%26rft.atitle%253DGenome%2Bresequencing%2Breveals%2Bmultiscale%2Bgeographic%2Bstructure%2Band%2Bextensive%2Blinkage%2Bdisequilibrium%2Bin%2Bthe%2Bforest%2Btree%2BPopulus%2Btrichocarpa.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2012.04258.x%26rft_id%253Dinfo%253Apmid%252F22861491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2012.04258.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22861491&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309757200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.60" data-doi="10.1146/annurev.cellbio.22.022206.160206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Somerville</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Cellulose synthesis in higher plants.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Cell Dev. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">53</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bcell%2Band%2Bdevelopmental%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BCell%2BDev%2BBiol%26rft.aulast%253DSomerville%26rft.auinit1%253DC.%26rft.volume%253D22%26rft.spage%253D53%26rft.epage%253D78%26rft.atitle%253DCellulose%2Bsynthesis%2Bin%2Bhigher%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.cellbio.22.022206.160206%26rft_id%253Dinfo%253Apmid%252F16824006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.cellbio.22.022206.160206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16824006&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242325100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.61" data-doi="10.1126/science.1102765"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Somerville</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brininstool</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Facette</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Toward a systems approach to understanding plant cell walls.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">306</span>: <span class="cit-fpage">2206</span>–<span class="cit-lpage">2211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSomerville%26rft.auinit1%253DC.%26rft.volume%253D306%26rft.issue%253D5705%26rft.spage%253D2206%26rft.epage%253D2211%26rft.atitle%253DToward%2Ba%2BSystems%2BApproach%2Bto%2BUnderstanding%2BPlant%2BCell%2BWalls%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1102765%26rft_id%253Dinfo%253Apmid%252F15618507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDYvNTcwNS8yMjA2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.62" data-doi="10.1111/j.1469-8137.2010.03315.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Song</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Characterization of cellulose synthase complexes in <em>Populus</em> xylem differentiation.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">187</span>: <span class="cit-fpage">777</span>–<span class="cit-lpage">790</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSong%26rft.auinit1%253DD.%26rft.volume%253D187%26rft.issue%253D3%26rft.spage%253D777%26rft.epage%253D790%26rft.atitle%253DCharacterization%2Bof%2Bcellulose%2Bsynthase%2Bcomplexes%2Bin%2BPopulus%2Bxylem%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2010.03315.x%26rft_id%253Dinfo%253Apmid%252F20546138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2010.03315.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20546138&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280122500020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.63" data-doi="10.1111/j.1365-313X.2007.03176.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spokevicius</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Southerton</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacMillan</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Beta-tubulin affects cellulose microfibril orientation in plant secondary fibre cell walls.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">726</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DSpokevicius%26rft.auinit1%253DA.%2BV.%26rft.volume%253D51%26rft.issue%253D4%26rft.spage%253D717%26rft.epage%253D726%26rft.atitle%253Dbeta-tubulin%2Baffects%2Bcellulose%2Bmicrofibril%2Borientation%2Bin%2Bplant%2Bsecondary%2Bfibre%2Bcell%2Bwalls.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2007.03176.x%26rft_id%253Dinfo%253Apmid%252F17605757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2007.03176.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17605757&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249203100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.64" data-doi="10.1086/428594"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Accounting for decay of linkage disequilibrium in haplotype inference and missing-data imputation.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">449</span>–<span class="cit-lpage">462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DStephens%26rft.auinit1%253DM.%26rft.volume%253D76%26rft.issue%253D3%26rft.spage%253D449%26rft.epage%253D462%26rft.atitle%253DAccounting%2Bfor%2Bdecay%2Bof%2Blinkage%2Bdisequilibrium%2Bin%2Bhaplotype%2Binference%2Band%2Bmissing-data%2Bimputation.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F428594%26rft_id%253Dinfo%253Apmid%252F15700229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/428594&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15700229&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226851900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.65" data-doi="10.1073/pnas.1120811109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sterken</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kiekens</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boruc</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vercauteren</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Combined linkage and association mapping reveals <em>CYCD5;1</em> as a quantitative trait gene for endoreduplication in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">4678</span>–<span class="cit-lpage">4683</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1120811109%26rft_id%253Dinfo%253Apmid%252F22392991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzEyLzQ2NzgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.66" data-doi="10.1186/1471-2164-10-94"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Comparison of mixed-model approaches for association mapping in rapeseed, potato, sugar beet, maize, and Arabidopsis.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DStich%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.spage%253D94%26rft.epage%253D94%26rft.atitle%253DComparison%2Bof%2Bmixed-model%2Bapproaches%2Bfor%2Bassociation%2Bmapping%2Bin%2Brapeseed%252C%2Bpotato%252C%2Bsugar%2Bbeet%252C%2Bmaize%252C%2Band%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-94%26rft_id%253Dinfo%253Apmid%252F19250529%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-94&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19250529&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.67" data-doi="10.1073/pnas.1530509100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Statistical significance for genomewide studies.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">9440</span>–<span class="cit-lpage">9445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1530509100%26rft_id%253Dinfo%253Apmid%252F12883005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzE2Lzk0NDAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.68" data-doi="10.1104/pp.106.086678"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">V. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The cellulose synthase gene superfamily and biochemical functions of xylem-specific cellulose synthase-like genes in <em>Populus trichocarpa</em>.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">142</span>: <span class="cit-fpage">1233</span>–<span class="cit-lpage">1245</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.106.086678%26rft_id%253Dinfo%253Apmid%252F16950861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTQyLzMvMTIzMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.69" data-doi="10.1111/j.1365-313X.2003.02000.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szyjanowicz</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKinnon</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gardiner</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jarvis</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The irregular xylem 2 mutant is an allele of korrigan that affects the secondary cell wall of <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">730</span>–<span class="cit-lpage">740</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DSzyjanowicz%26rft.auinit1%253DP.%2BM.%26rft.volume%253D37%26rft.issue%253D5%26rft.spage%253D730%26rft.epage%253D740%26rft.atitle%253DThe%2Birregular%2Bxylem%2B2%2Bmutant%2Bis%2Ban%2Ballele%2Bof%2Bkorrigan%2Bthat%2Baffects%2Bthe%2Bsecondary%2Bcell%2Bwall%2Bof%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2003.02000.x%26rft_id%253Dinfo%253Apmid%252F14871312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2003.02000.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14871312&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188865200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tajima</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">595</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTajima%26rft.auinit1%253DF.%26rft.volume%253D123%26rft.issue%253D3%26rft.spage%253D585%26rft.epage%253D595%26rft.atitle%253DStatistical%2Bmethod%2Bfor%2Btesting%2Bthe%2Bneutral%2Bmutation%2Bhypothesis%2Bby%2BDNA%2Bpolymorphism.%26rft_id%253Dinfo%253Apmid%252F2513255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTIzLzMvNTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.71" data-doi="10.1093/molbev/msm092"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">MEGA4: molecular evolutionary genetics analysis (MEGA) software version 4.0.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1596</span>–<span class="cit-lpage">1599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsm092%26rft_id%253Dinfo%253Apmid%252F17488738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTU5NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.72" data-doi="10.1073/pnas.0337628100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howells</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huttly</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vickers</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Interactions among three distinct CesA proteins essential for cellulose synthesis.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">1450</span>–<span class="cit-lpage">1455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0337628100%26rft_id%253Dinfo%253Apmid%252F12538856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzMvMTQ1MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.73" data-doi="10.1534/genetics.109.106591"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thumma</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matheson</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meeske</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meder</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification of a <em>cis</em>-acting regulatory polymorphism in a Eucalypt <em>cobra</em>-like gene affecting cellulose content.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1153</span>–<span class="cit-lpage">1164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.106591%26rft_id%253Dinfo%253Apmid%252F19737751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4My8zLzExNTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.74" data-doi="10.1534/genetics.105.042028"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thumma</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nolan</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">G. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Polymorphisms in <em>cinnamoyl coa reductase</em> (<em>CCR</em>) are associated with variation in microfibril angle in <em>Eucalyptus</em> spp.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">1257</span>–<span class="cit-lpage">1265</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.042028%26rft_id%253Dinfo%253Apmid%252F16085705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3MS8zLzEyNTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.75" data-doi="10.1007/s11295-009-0250-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thumma</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Southerton</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Owen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henery</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Quantitative trait locus (QTL) analysis of wood quality traits in <em>Eucalyptus nitens</em>.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">305</span>–<span class="cit-lpage">317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D6%26rft.spage%253D305%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-009-0250-9%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-009-0250-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.76" data-doi="10.1371/journal.pone.0053116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">J. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">Q. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">M. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Allelic variation in <em>PtGA20Ox</em> associates with growth and wood properties in <em>Populus</em> spp.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e53116</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D7%26rft.spage%253De53116%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0053116%26rft_id%253Dinfo%253Apmid%252F23300875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0053116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23300875&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watterson</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">On the number of segregating sites in genetical models without recombination.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">188</span>–<span class="cit-lpage">193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BPopul.%2BBiol.%26rft.volume%253D7%26rft.spage%253D188%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.78" data-doi="10.1111/j.1469-8137.2010.03415.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wegrzyn</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eckert</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stanton</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Association genetics of traits controlling lignin and cellulose biosynthesis in black cottonwood (<em>Populus trichocarpa</em>, Salicaceae) secondary xylem.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">532</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DWegrzyn%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D188%26rft.issue%253D2%26rft.spage%253D515%26rft.epage%253D532%26rft.atitle%253DAssociation%2Bgenetics%2Bof%2Btraits%2Bcontrolling%2Blignin%2Band%2Bcellulose%2Bbiosynthesis%2Bin%2Bblack%2Bcottonwood%2B%2528Populus%2Btrichocarpa%252C%2BSalicaceae%2529%2Bsecondary%2Bxylem.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2010.03415.x%26rft_id%253Dinfo%253Apmid%252F20831625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2010.03415.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20831625&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282225000020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.79" data-doi="10.1038/tp.2012.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weickert</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiwari</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schofield</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mowry</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fullerton</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Schizophrenia-associated HapICE haplotype is associated with increased NRG1 type III expression and high nucleotide diversity.</span> <abbr class="cit-jnl-abbrev">Transcult. Psychiatry</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTranscult.%2BPsychiatry%26rft.volume%253D2%26rft.spage%253De104%26rft_id%253Dinfo%253Adoi%252F10.1038%252Ftp.2012.25%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/tp.2012.25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.80" data-doi="10.1093/molbev/msg140"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abouheif</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balhoff</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pizer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The evolution of transcriptional regulation in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1377</span>–<span class="cit-lpage">1419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg140%26rft_id%253Dinfo%253Apmid%252F12777501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTM3NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.81" data-doi="10.1046/j.1365-313x.2000.00758.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Joshi</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">V. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A xylem-specific cellulose synthase gene from aspen (<em>Populus tremuloides</em>) is responsive to mechanical stress.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">495</span>–<span class="cit-lpage">502</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DWu%26rft.auinit1%253DL.%26rft.volume%253D22%26rft.issue%253D6%26rft.spage%253D495%26rft.epage%253D502%26rft.atitle%253DA%2Bxylem-specific%2Bcellulose%2Bsynthase%2Bgene%2Bfrom%2Baspen%2B%2528Populus%2Btremuloides%2529%2Bis%2Bresponsive%2Bto%2Bmechanical%2Bstress.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313x.2000.00758.x%26rft_id%253Dinfo%253Apmid%252F10886769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313x.2000.00758.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10886769&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087868300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.82" data-doi="10.1093/jxb/err164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Brassinosteroids can regulate cellulose biosynthesis by controlling the expression of CESA genes in <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">4495</span>–<span class="cit-lpage">4506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferr164%26rft_id%253Dinfo%253Apmid%252F21617247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjEwOiI2Mi8xMy80NDk1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.83" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.84" data-doi="10.1007/s11103-009-9536-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiong</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A missense mutation in the transmembrane domain of CESA4 affects protein abundance in the plasma membrane and results in abnormal cell wall biosynthesis in rice.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">509</span>–<span class="cit-lpage">524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DZhang%26rft.auinit1%253DB.%26rft.volume%253D71%26rft.issue%253D4-5%26rft.spage%253D509%26rft.epage%253D524%26rft.atitle%253DA%2Bmissense%2Bmutation%2Bin%2Bthe%2Btransmembrane%2Bdomain%2Bof%2BCESA4%2Baffects%2Bprotein%2Babundance%2Bin%2Bthe%2Bplasma%2Bmembrane%2Band%2Bresults%2Bin%2Babnormal%2Bcell%2Bwall%2Bbiosynthesis%2Bin%2Brice.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-009-9536-4%26rft_id%253Dinfo%253Apmid%252F19697141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-009-9536-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19697141&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270984900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.85" data-doi="10.1139/x06-103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">QTL analysis of growth and wood chemical content traits in an interspecific backcross family of white poplar (<em>Populus tomentosa</em> × <em>P. bolleana</em>) × <em>P. tomentosa</em>.</span> <abbr class="cit-jnl-abbrev">Can. J. For. Res</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">2015</span>–<span class="cit-lpage">2023</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BFor.%2BRes%26rft.volume%253D36%26rft.spage%253D2015%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fx06-103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/x06-103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.86" data-doi="10.1007/s00438-010-0552-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">Q. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">a</span> <span class="cit-article-title">The actin multigene family in <em>Populus</em>: organization, expression and phylogenetic analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DZhang%26rft.auinit1%253DD.%26rft.volume%253D284%26rft.issue%253D2%26rft.spage%253D105%26rft.epage%253D119%26rft.atitle%253DThe%2Bactin%2Bmultigene%2Bfamily%2Bin%2BPopulus%253A%2Borganization%252C%2Bexpression%2Band%2Bphylogenetic%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-010-0552-5%26rft_id%253Dinfo%253Apmid%252F20577761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-010-0552-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20577761&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2069.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.87" data-doi="10.1007/s11295-009-0252-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">D. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">X. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">b</span> <span class="cit-article-title">Expression and nucleotide diversity of the poplar <em>COBL</em> gene.</span> <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">331</span>–<span class="cit-lpage">344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D6%26rft.spage%253D331%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-009-0252-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-009-0252-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000274112600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.88" data-doi="10.1104/pp.102.019331"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrison</span>  <span class="cit-name-given-names">W. H.</span>  <span class="cit-name-suffix">III</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freshour</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">Z. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Expression of a mutant form of cellulose synthase AtCesA7 causes dominant negative effect on cellulose biosynthesis.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">786</span>–<span class="cit-lpage">795</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DZhong%26rft.auinit1%253DR.%26rft.volume%253D132%26rft.issue%253D2%26rft.spage%253D786%26rft.epage%253D795%26rft.atitle%253DExpression%2Bof%2Ba%2BMutant%2BForm%2Bof%2BCellulose%2BSynthase%2BAtCesA7%2BCauses%2BDominant%2BNegative%2BEffect%2Bon%2BCellulose%2BBiosynthesis%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.102.019331%26rft_id%253Dinfo%253Apmid%252F12805608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMzIvMi83ODYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2069.89" data-doi="10.1105/tpc.106.047399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ye</span>  <span class="cit-name-given-names">Z. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">SND1, a NAC domain transcription factor, is a key regulator of secondary wall synthesis in fibers of <em>Arabidopsis</em>.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">3158</span>–<span class="cit-lpage">3170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.106.047399%26rft_id%253Dinfo%253Apmid%252F17114348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMS8zMTU4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/11/2069.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/2059" title="Molecular Phylogeography of a Human Autosomal Skin Color Locus Under Natural Selection" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/11/2085" title="Dissecting Genome-Wide Association Signals for Loss-of-Function Phenotypes in Sorghum Flavonoid Pigmentation Traits" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/11"><span>Volume 3 Issue 11, November 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/11.cover-source.jpg?itok=A7iZ1ztq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/2069" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-eCmoKfvsUWHL6csUo2F0M8NlnDUbuc2aWmopluw6YDI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/11/2069" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/11/2069" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/11/2069" class="active" data-icon-position="" data-hide-link-title="0">Allelic Variation in a Cellulose Synthase Gene (PtoCesA4) Associated with Growth and Wood Properties in Populus tomentosa</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459573" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Zm5iPTZodZpoCu47BsOwOpZEg-ufGEV6HtYSt2I1VJs" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/2069.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/11/2069" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-owH8GPH5uo7oWnC53KEnVtteJ33vy-rSo7SBvEecYR8" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/11/2069" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/2069.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459573" id="node459573--2" data-pisa="ggg;3/11/2069" data-pisa-master="ggg;g3.113.007724" data-apath="/ggg/3/11/2069.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Allelic Variation in a Cellulose Synthase Gene (<em>PtoCesA4</em>) Associated with Growth and Wood Properties in <em>Populus tomentosa</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Qingzhang Du</span>, <span class="highwire-citation-author" data-delta="1">Baohua Xu</span>, <span class="highwire-citation-author" data-delta="2">Wei Pan</span>, <span class="highwire-citation-author" data-delta="3">Chenrui Gong</span>, <span class="highwire-citation-author" data-delta="4">Qingshi Wang</span>, <span class="highwire-citation-author" data-delta="5">Jiaxing Tian</span>, <span class="highwire-citation-author" data-delta="6">Bailian Li</span> and <span class="highwire-citation-author" data-delta="7">Deqiang Zhang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2069-2084;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007724 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459573--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Qingzhang Du </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingzhang%2BDu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Du%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQingzhang%2BDu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Baohua Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Baohua%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABaohua%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Wei Pan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BPan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pan%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWei%2BPan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chenrui Gong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chenrui%2BGong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gong%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChenrui%2BGong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Qingshi Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingshi%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQingshi%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jiaxing Tian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiaxing%2BTian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tian%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiaxing%2BTian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Bailian Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Forestry, North Carolina State University, Raleigh, North Carolina 27695-8203</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bailian%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABailian%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Deqiang Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Deqiang%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADeqiang%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:DeqiangZhang@bjfu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">DeqiangZhang@bjfu.edu.cn</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/11/2069" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/11/2069&amp;title=Allelic%20Variation%20in%20a%20Cellulose%20Synthase%20Gene%20%28PtoCesA4%29%20Associated%20with%20Growth%20and%20Wood%20Properties%20in%20Populus%20tomentosa" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/11/2069&amp;title=Allelic%20Variation%20in%20a%20Cellulose%20Synthase%20Gene%20%28PtoCesA4%29%20Associated%20with%20Growth%20and%20Wood%20Properties%20in%20Populus%20tomentosa" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/11/2069&amp;title=Allelic%20Variation%20in%20a%20Cellulose%20Synthase%20Gene%20%28PtoCesA4%29%20Associated%20with%20Growth%20and%20Wood%20Properties%20in%20Populus%20tomentosa" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/11/2069&amp;text=Allelic%20Variation%20in%20a%20Cellulose%20Synthase%20Gene%20%28PtoCesA4%29%20Associated%20with%20Growth%20and%20Wood%20Properties%20in%20Populus%20tomentosa" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/11/2069&amp;title=Allelic%20Variation%20in%20a%20Cellulose%20Synthase%20Gene%20%28PtoCesA4%29%20Associated%20with%20Growth%20and%20Wood%20Properties%20in%20Populus%20tomentosa" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/11/2069&amp;t=Allelic%20Variation%20in%20a%20Cellulose%20Synthase%20Gene%20%28PtoCesA4%29%20Associated%20with%20Growth%20and%20Wood%20Properties%20in%20Populus%20tomentosa" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/11/2069&amp;title=Allelic%20Variation%20in%20a%20Cellulose%20Synthase%20Gene%20%28PtoCesA4%29%20Associated%20with%20Growth%20and%20Wood%20Properties%20in%20Populus%20tomentosa" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/11/2069&amp;title=Allelic%20Variation%20in%20a%20Cellulose%20Synthase%20Gene%20%28PtoCesA4%29%20Associated%20with%20Growth%20and%20Wood%20Properties%20in%20Populus%20tomentosa" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459573/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459573" id="node459573--3" data-pisa="ggg;3/11/2069" data-pisa-master="ggg;g3.113.007724" data-apath="/ggg/3/11/2069.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Allelic Variation in a Cellulose Synthase Gene (<em>PtoCesA4</em>) Associated with Growth and Wood Properties in <em>Populus tomentosa</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Qingzhang Du</span>, <span class="highwire-citation-author" data-delta="1">Baohua Xu</span>, <span class="highwire-citation-author" data-delta="2">Wei Pan</span>, <span class="highwire-citation-author" data-delta="3">Chenrui Gong</span>, <span class="highwire-citation-author" data-delta="4">Qingshi Wang</span>, <span class="highwire-citation-author" data-delta="5">Jiaxing Tian</span>, <span class="highwire-citation-author" data-delta="6">Bailian Li</span> and <span class="highwire-citation-author" data-delta="7">Deqiang Zhang</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2069-2084;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007724 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459573--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Qingzhang Du </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingzhang%2BDu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Du%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQingzhang%2BDu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Baohua Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Baohua%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABaohua%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Wei Pan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Wei%2BPan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pan%20W&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWei%2BPan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Chenrui Gong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chenrui%2BGong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gong%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChenrui%2BGong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Qingshi Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qingshi%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQingshi%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jiaxing Tian </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiaxing%2BTian%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tian%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiaxing%2BTian%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Bailian Li </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Forestry, North Carolina State University, Raleigh, North Carolina 27695-8203</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bailian%2BLi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Li%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABailian%2BLi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Deqiang Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, Ministry of Education, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, People’s Republic of China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Deqiang%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADeqiang%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:DeqiangZhang@bjfu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">DeqiangZhang@bjfu.edu.cn</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2069&amp;rft.epage=2084&amp;rft.atitle=Allelic%20Variation%20in%20a%20Cellulose%20Synthase%20Gene%20%28PtoCesA4%29%20Associated%20with%20Growth%20and%20Wood%20Properties%20in%20Populus%20tomentosa&amp;rft.volume=3&amp;rft.issue=11&amp;rft.date=2013-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Du%2C+Qingzhang&amp;rft.au=Xu%2C+Baohua&amp;rft.au=Pan%2C+Wei&amp;rft.au=Gong%2C+Chenrui&amp;rft.au=Wang%2C+Qingshi&amp;rft.au=Tian%2C+Jiaxing&amp;rft.au=Li%2C+Bailian&amp;rft.au=Zhang%2C+Deqiang"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459573/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459573/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459573/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459573/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459573/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459573/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459573/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459573/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459573/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459573/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459573/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459573/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/11/2069" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-32" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/11/2069.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/11/2069.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/11/2069.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/11/2069.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
