<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/11/2085" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/11/2085.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/11/2085.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/11/2085.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/11/2085/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/11/2085.atom" />
<meta name="HW.pisa" content="ggg;3/11/2085" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Dissecting Genome-Wide Association Signals for Loss-of-Function Phenotypes in Sorghum Flavonoid Pigmentation Traits" />
<meta name="DC.Identifier" content="10.1534/g3.113.008417" />
<meta name="DC.Date" content="2013-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Morris et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genome-wide association studies are a powerful method to dissect the genetic basis of traits, although in practice the effects of complex genetic architecture and population structure remain poorly understood. To compare mapping strategies we dissected the genetic control of flavonoid pigmentation traits in the cereal grass sorghum by using high-resolution genotyping-by-sequencing single-nucleotide polymorphism markers. Studying the grain tannin trait, we find that general linear models (GLMs) are not able to precisely map tan1-a , a known loss-of-function allele of the Tannin1 gene, with either a small panel ( n = 142) or large association panel ( n = 336), and that indirect associations limit the mapping of the Tannin1 locus to Mb-resolution. A GLM that accounts for population structure (Q) or standard mixed linear model that accounts for kinship (K) can identify tan1-a , whereas a compressed mixed linear model performs worse than the naive GLM. Interestingly, a simple loss-of-function genome scan, for genotype-phenotype covariation only in the putative loss-of-function allele, is able to precisely identify the Tannin1 gene without considering relatedness. We also find that the tan1-a allele can be mapped with gene resolution in a biparental recombinant inbred line family ( n = 263) using genotyping-by-sequencing markers but lower precision in the mapping of vegetative pigmentation traits suggest that consistent gene-level resolution will likely require larger families or multiple recombinant inbred lines. These findings highlight that complex association signals can emerge from even the simplest traits given epistasis and structured alleles, but that gene-resolution mapping of these traits is possible with high marker density and appropriate models." />
<meta name="DC.Contributor" content="Geoffrey P. Morris" />
<meta name="DC.Contributor" content="Davina H. Rhodes" />
<meta name="DC.Contributor" content="Zachary Brenton" />
<meta name="DC.Contributor" content="Punna Ramu" />
<meta name="DC.Contributor" content="Vinayan Madhumal Thayil" />
<meta name="DC.Contributor" content="Santosh Deshpande" />
<meta name="DC.Contributor" content="C. Thomas Hash" />
<meta name="DC.Contributor" content="Charlotte Acharya" />
<meta name="DC.Contributor" content="Sharon E. Mitchell" />
<meta name="DC.Contributor" content="Edward S. Buckler" />
<meta name="DC.Contributor" content="Jianming Yu" />
<meta name="DC.Contributor" content="Stephen Kresovich" />
<meta name="article:published_time" content="2013-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Dissecting Genome-Wide Association Signals for Loss-of-Function Phenotypes in Sorghum Flavonoid Pigmentation Traits" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genome-wide association studies are a powerful method to dissect the genetic basis of traits, although in practice the effects of complex genetic architecture and population structure remain poorly understood. To compare mapping strategies we dissected the genetic control of flavonoid pigmentation traits in the cereal grass sorghum by using high-resolution genotyping-by-sequencing single-nucleotide polymorphism markers. Studying the grain tannin trait, we find that general linear models (GLMs) are not able to precisely map &lt;i&gt;tan1-a&lt;/i&gt;, a known loss-of-function allele of the &lt;i&gt;Tannin1&lt;/i&gt; gene, with either a small panel (&lt;i&gt;n&lt;/i&gt; = 142) or large association panel (&lt;i&gt;n&lt;/i&gt; = 336), and that indirect associations limit the mapping of the &lt;i&gt;Tannin1&lt;/i&gt; locus to Mb-resolution. A GLM that accounts for population structure (Q) or standard mixed linear model that accounts for kinship (K) can identify &lt;i&gt;tan1-a&lt;/i&gt;, whereas a compressed mixed linear model performs worse than the naive GLM. Interestingly, a simple loss-of-function genome scan, for genotype-phenotype covariation only in the putative loss-of-function allele, is able to precisely identify the &lt;i&gt;Tannin1&lt;/i&gt; gene without considering relatedness. We also find that the &lt;i&gt;tan1-a&lt;/i&gt; allele can be mapped with gene resolution in a biparental recombinant inbred line family (&lt;i&gt;n&lt;/i&gt; = 263) using genotyping-by-sequencing markers but lower precision in the mapping of vegetative pigmentation traits suggest that consistent gene-level resolution will likely require larger families or multiple recombinant inbred lines. These findings highlight that complex association signals can emerge from even the simplest traits given epistasis and structured alleles, but that gene-resolution mapping of these traits is possible with high marker density and appropriate models.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/11/01" />
<meta name="citation_mjid" content="ggg;3/11/2085" />
<meta name="citation_id" content="3/11/2085" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/11/2085" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/11/2085.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/11/2085.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/11/2085.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.008417" />
<meta name="citation_pmid" content="24048646" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2085" />
<meta name="citation_lastpage" content="2094" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Geoffrey P. Morris" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208" />
<meta name="citation_author_email" content="gpmorris@ksu.edu" />
<meta name="citation_author" content="Davina H. Rhodes" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208" />
<meta name="citation_author" content="Zachary Brenton" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208" />
<meta name="citation_author" content="Punna Ramu" />
<meta name="citation_author_institution" content="ICRISAT, Patancheru PO, Hyderabad 502 324, Andhra Pradesh, India" />
<meta name="citation_author" content="Vinayan Madhumal Thayil" />
<meta name="citation_author_institution" content="ICRISAT, Patancheru PO, Hyderabad 502 324, Andhra Pradesh, India" />
<meta name="citation_author" content="Santosh Deshpande" />
<meta name="citation_author_institution" content="ICRISAT, Patancheru PO, Hyderabad 502 324, Andhra Pradesh, India" />
<meta name="citation_author" content="C. Thomas Hash" />
<meta name="citation_author_institution" content="ICRISAT-Sadoré, BP 12404, Niamey, Niger" />
<meta name="citation_author" content="Charlotte Acharya" />
<meta name="citation_author_institution" content="Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="Sharon E. Mitchell" />
<meta name="citation_author_institution" content="Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="Edward S. Buckler" />
<meta name="citation_author_institution" content="USDA-ARS, Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="Jianming Yu" />
<meta name="citation_author_institution" content="Department of Agronomy, Iowa State University, Ames, Iowa 50011" />
<meta name="citation_author" content="Stephen Kresovich" />
<meta name="citation_author_institution" content="Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Atwell;citation_author=Y. S. Huang;citation_author=B. J. Vilhjalmsson;citation_author=G. Willems;citation_author=M. Horton;citation_title=Genome-wide association study of 107 phenotypes in Arabidopsis thaliana inbred lines.;citation_pages=627-631;citation_volume=465;citation_year=2010;citation_issue=7298;citation_pmid=20336072;citation_doi=10.1038/nature08800" />
<meta name="citation_reference" content="citation_journal_title=Molecular genetics and genomics :  MGG;citation_journal_abbrev=Mol Genet Genomics;citation_author=A. Beló;citation_author=P. Zheng;citation_author=S. Luck;citation_author=B. Shen;citation_author=D. Meyer;citation_title=Whole genome scan detects an allelic variant of fad2 associated with increased oleic acid levels in maize.;citation_pages=1-10;citation_volume=279;citation_year=2008;citation_issue=1;citation_pmid=17934760;citation_doi=10.1007/s00438-007-0289-y" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Bergelson;citation_author=F. Roux;citation_title=Towards identifying genes underlying ecologically relevant traits in Arabidopsis thaliana.;citation_pages=867-879;citation_volume=11;citation_year=2010;citation_issue=12;citation_pmid=21085205;citation_doi=10.1038/nrg2896" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Brachi;citation_author=G. P. Morris;citation_author=J. O. Borevitz;citation_title=Genome-wide association studies in plants: the missing heritability is in the field.;citation_pages=232-232;citation_volume=12;citation_year=2011;citation_issue=10;citation_pmid=22035733;citation_doi=10.1186/gb-2011-12-10-232" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=A. M. Casa;citation_author=G. Pressoir;citation_author=P. J. Brown;citation_author=S. E. Mitchell;citation_author=W. L. Rooney;citation_title=Community Resources and Strategies for Association Mapping in Sorghum;citation_pages=30-40;citation_volume=48;citation_year=2008;citation_issue=1;citation_doi=10.2135/cropsci2007.02.0080" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Cockram;citation_author=J. White;citation_author=D. L. Zuluaga;citation_author=D. Smith;citation_author=J. Comadran;citation_title=Genome-wide association mapping to candidate polymorphism resolution in the unsequenced barley genome.;citation_volume=107;citation_year=2010;citation_pmid=21115826;citation_doi=10.1073/pnas.1010179107" />
<meta name="citation_reference" content="citation_title=On the Origin of the Species by Means of Natural Selection: Or, The Preservation of Favoured Races in the Struggle for Life;citation_author=C. Darwin;citation_year=1859" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. P. Dickson;citation_author=K. Wang;citation_author=I. Krantz;citation_author=H. Hakonarson;citation_author=D. B. Goldstein;citation_title=Rare variants create synthetic genome-wide associations.;citation_pages=e1000294-e1000294;citation_volume=8;citation_year=2010;citation_issue=1;citation_pmid=20126254;citation_doi=10.1371/journal.pbio.1000294" />
<meta name="citation_reference" content="citation_title=Sorghum;citation_author=H. Doggett;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_author=J. P. Esele;citation_author=R. A. Frederiksen;citation_author=F. R. Miller;citation_title=The association of genes controlling caryopsis traits with grain mold resistance in sorghum.;citation_pages=490-495;citation_volume=83;citation_year=1993;citation_doi=10.1094/Phyto-83-490" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Flint;citation_author=E. Eskin;citation_title=Genome-wide association studies in mice.;citation_pages=807-817;citation_volume=13;citation_year=2012;citation_issue=11;citation_pmid=23044826;citation_doi=10.1038/nrg3335" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=D. B. Goldstein;citation_title=The importance of synthetic associations will only be resolved empirically.;citation_pages=e1001008-e1001008;citation_volume=9;citation_year=2011;citation_issue=1;citation_pmid=21267066;citation_doi=10.1371/journal.pbio.1001008" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. M. Goodstein;citation_author=S. Shu;citation_author=R. Howson;citation_author=R. Neupane;citation_author=R. D. Hayes;citation_title=Phytozome: a comparative platform for green plant genomics.;citation_pages=D1178-D1186;citation_volume=40;citation_year=2011;citation_pmid=22110026" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=B. L. Gross;citation_author=K. M. Olsen;citation_title=Genetic perspectives on crop domestication.;citation_pages=529-537;citation_volume=15;citation_year=2010;citation_issue=9;citation_pmid=20541451;citation_doi=10.1016/j.tplants.2010.05.008" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Nutrition;citation_journal_abbrev=J. Nutr.;citation_author=L. Gu;citation_author=M. A. Kelm;citation_author=J. F. Hammerstone;citation_author=G. Beecher;citation_author=J. Holden;citation_title=Concentrations of Proanthocyanidins in Common Foods and Estimations of Normal Consumption;citation_pages=613-617;citation_volume=134;citation_year=2004;citation_issue=3;citation_pmid=14988456" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Huang;citation_author=X. Wei;citation_author=T. Sang;citation_author=Q. Zhao;citation_author=Q. Feng;citation_title=Genome-wide association studies of 14 agronomic traits in rice landraces.;citation_pages=961-967;citation_volume=42;citation_year=2010;citation_issue=11;citation_pmid=20972439;citation_doi=10.1038/ng.695" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=X. Huang;citation_author=Y. Zhao;citation_author=X. Wei;citation_author=C. Li;citation_author=A. Wang;citation_title=Genome-wide association study of flowering time and grain yield traits in a worldwide collection of rice germplasm.;citation_pages=32-39;citation_volume=44;citation_year=2012;citation_issue=1;citation_pmid=22138690;citation_doi=10.1038/ng.1018" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=F. Ibraheem;citation_author=I. Gaffoor;citation_author=S. Chopra;citation_title=Flavonoid phytoalexin-dependent resistance to anthracnose leaf blight requires a functional yellow seed1 in Sorghum bicolor.;citation_volume=184;citation_year=2010;citation_pmid=20083611;citation_doi=10.1534/genetics.109.111831" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=D. R. Jordan;citation_author=E. S. Mace;citation_author=A. W. Cruickshank;citation_author=C. H. Hunt;citation_author=R. G. Henzell;citation_title=Exploring and exploiting genetic variation from unadapted sorghum germplasm in a breeding program.;citation_pages=1444-1457;citation_volume=51;citation_year=2011;citation_doi=10.2135/cropsci2010.06.0326" />
<meta name="citation_reference" content="citation_journal_title=J. Sci. Food Agric.;citation_author=R. C. Kaufman;citation_author=T. J. Herald;citation_author=S. R. Bean;citation_author=J. D. Wilson;citation_author=M. R. Tuinstra;citation_title=Variability in tannin content, chemistry and activity in a diverse group of tannin containing sorghum cultivars;citation_pages=1233-1241;citation_volume=93;citation_year=2013;citation_pmid=23011944;citation_doi=10.1002/jsfa.5890" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. Lamesch;citation_author=T. Z. Berardini;citation_author=D. Li;citation_author=D. Swarbreck;citation_author=C. Wilks;citation_title=The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools.;citation_pages=D1202-D1210;citation_volume=40;citation_year=2011;citation_pmid=22140109" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Z. Lin;citation_author=X. Li;citation_author=L. M. Shannon;citation_author=C.-T. Yeh;citation_author=M. L. Wang;citation_title=Parallel domestication of the Shattering1 genes in cereals.;citation_pages=720-724;citation_volume=44;citation_year=2012;citation_issue=6;citation_pmid=22581231;citation_doi=10.1038/ng.2281" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. E. Lipka;citation_author=F. Tian;citation_author=Q. Wang;citation_author=J. Peiffer;citation_author=M. Li;citation_title=GAPIT: genome association and prediction integrated tool.;citation_volume=28;citation_year=2012;citation_pmid=22796960;citation_doi=10.1093/bioinformatics/bts444" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=F. Liu;citation_author=M. V. Struchalin;citation_author=K. van Duijn;citation_author=A. Hofman;citation_author=A. G. Uitterlinden;citation_title=Detecting low frequent loss-of-function alleles in genome wide association studies with red hair color as example.;citation_pages=e28145-e28145;citation_volume=6;citation_year=2011;citation_issue=11;citation_pmid=22140526;citation_doi=10.1371/journal.pone.0028145" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=E. S. Mace;citation_author=D. R. Jordan;citation_title=Location of major effect genes in sorghum (Sorghum bicolor (L.) Moench).;citation_pages=1339-1356;citation_volume=121;citation_year=2010;citation_issue=7;citation_pmid=20585750;citation_doi=10.1007/s00122-010-1392-8" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=B. McClintock;citation_title=The origin and behavior of mutable loci in maize;citation_pages=344-355;citation_volume=36;citation_year=1950;citation_issue=6;citation_pmid=15430309;citation_doi=10.1073/pnas.36.6.344" />
<meta name="citation_reference" content="citation_journal_title=Food Res. Int.;citation_author=É. A. Moraes;citation_author=D. I. G. Natal;citation_author=V. A. V. Queiroz;citation_author=R. E. Schaffert;citation_author=P. R. Cecon;citation_title=Sorghum genotype may reduce low-grade inflammatory response and oxidative stress and maintains jejunum morphology of rats fed a hyperlipidic diet.;citation_pages=553-559;citation_volume=49;citation_year=2012;citation_doi=10.1016/j.foodres.2012.07.029" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=G. P. Morris;citation_author=P. Ramu;citation_author=S. P. Deshpande;citation_author=C. T. Hash;citation_author=T. Shah;citation_title=Population genomic and genome-wide association studies of agroclimatic traits in sorghum.;citation_volume=110;citation_year=2013;citation_pmid=23267105;citation_doi=10.1073/pnas.1215985110" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. L. Murphy;citation_author=R. R. Klein;citation_author=D. T. Morishige;citation_author=J. A. Brady;citation_author=W. L. Rooney;citation_title=Coincident light and clock regulation of pseudoresponse regulator protein 37 (PRR37) controls photoperiodic flowering in sorghum.;citation_volume=108;citation_year=2011;citation_pmid=21930910;citation_doi=10.1073/pnas.1106212108" />
<meta name="citation_reference" content="citation_journal_title=Lunds Universitets Arsskrift;citation_author=H. Nilsson-Ehle;citation_title=Kreuzunguntersuchungen an Häfer und Weizen.;citation_pages=1-122;citation_volume=5;citation_year=1909" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=G. Orozco;citation_author=J. C. Barrett;citation_author=E. Zeggini;citation_title=Synthetic associations in the context of genome-wide association scan signals.;citation_volume=19;citation_year=2010;citation_pmid=20805105;citation_doi=10.1093/hmg/ddq368" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=A. H. Paterson;citation_author=J. E. Bowers;citation_author=R. Bruggmann;citation_author=I. Dubchak;citation_author=J. Grimwood;citation_title=The Sorghum bicolor genome and the diversification of grasses.;citation_pages=551-556;citation_volume=457;citation_year=2009;citation_issue=7229;citation_pmid=19189423;citation_doi=10.1038/nature07723" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Sci;citation_author=K. Petroni;citation_author=C. Tonelli;citation_title=Recent advances on the regulation of anthocyanin synthesis in reproductive organs.;citation_pages=219-229;citation_volume=181;citation_year=2011;citation_issue=3;citation_pmid=21763532;citation_doi=10.1016/j.plantsci.2011.05.009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Platt;citation_author=B. J. Vilhjálmsson;citation_author=M. Nordborg;citation_title=Conditions under which genome-wide association studies will be positively misleading.;citation_volume=186;citation_year=2010;citation_pmid=20813880;citation_doi=10.1534/genetics.110.121665" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=TAG Theoret. Appl. Genet;citation_author=J.-F. Rami;citation_author=P. Dufour;citation_author=G. Trouche;citation_author=G. Fliedel;citation_author=C. Mestres;citation_title=Quantitative trait loci for grain quality, productivity, morphological and agronomical traits in sorghum (Sorghum bicolor L. Moench).;citation_pages=605-616;citation_volume=97;citation_year=1998;citation_doi=10.1007/s001220050936" />
<meta name="citation_reference" content="citation_title=Sorghum: Origin, History, Technology, and Production;citation_title=Genetics and cytogenetics;citation_author=W. L. Rooney;citation_pages=261-307;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=N. A. Rosenberg;citation_author=L. Huang;citation_author=E. M. Jewett;citation_author=Z. A. Szpiech;citation_author=I. Jankovic;citation_title=Genome-wide association studies in diverse populations.;citation_pages=356-366;citation_volume=11;citation_year=2010;citation_issue=5;citation_pmid=20395969;citation_doi=10.1038/nrg2760" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. Sax;citation_title=THE ASSOCIATION OF SIZE DIFFERENCES WITH SEED-COAT PATTERN AND PIGMENTATION IN PHASEOLUS VULGARIS;citation_pages=552-560;citation_volume=8;citation_year=1923;citation_issue=6;citation_pmid=17246026" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=J. C. Schnable;citation_author=M. Freeling;citation_title=Genes identified by visible mutant phenotypes show increased bias toward one of two subgenomes of maize.;citation_pages=e17855-e17855;citation_volume=6;citation_year=2011;citation_issue=3;citation_pmid=21423772;citation_doi=10.1371/journal.pone.0017855" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=V. Segura;citation_author=B. J. Vilhjálmsson;citation_author=A. Platt;citation_author=A. Korte;citation_author=Ü. Seren;citation_title=An efficient multi-locus mixed-model approach for genome-wide association studies in structured populations.;citation_pages=825-830;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22706313;citation_doi=10.1038/ng.2314" />
<meta name="citation_reference" content="citation_journal_title=Agronomy Journal;citation_journal_abbrev=Agron. J.;citation_author=J. C. Stephens;citation_title=A Second Factor for Subcoat in Sorghum Seed;citation_pages=340-342;citation_volume=38;citation_year=1946;citation_issue=4;citation_doi=10.2134/agronj1946.00021962003800040008x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Tian;citation_author=Q. Qian;citation_author=Q. Liu;citation_author=M. Yan;citation_author=X. Liu;citation_title=Allelic diversities in rice starch biosynthesis lead to a diverse array of rice eating and cooking qualities.;citation_volume=106;citation_year=2009;citation_pmid=20018713;citation_doi=10.1073/pnas.0912396106" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=H. N. Vinall;citation_author=A. B. Cron;citation_title=IMPROVEMENT OF SORGHUMS BY HYBRIDIZATION;citation_pages=435-443;citation_volume=12;citation_year=1921;citation_issue=10" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. M. Visscher;citation_author=M. A. Brown;citation_author=M. I. McCarthy;citation_author=J. Yang;citation_title=Five years of GWAS discovery.;citation_pages=7-24;citation_volume=90;citation_year=2012;citation_issue=1;citation_pmid=22243964;citation_doi=10.1016/j.ajhg.2011.11.029" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=B. Winkel-Shirley;citation_title=Flavonoid Biosynthesis. A Colorful Model for Genetics, Biochemistry, Cell Biology, and Biotechnology;citation_pages=485-493;citation_volume=126;citation_year=2001;citation_issue=2;citation_pmid=11402179;citation_doi=10.1104/pp.126.2.485" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA.;citation_author=Y. Wu;citation_author=X. Li;citation_author=W. Xiang;citation_author=C. Zhu;citation_author=Z. Lin;citation_title=Presence of tannins in sorghum grains is conditioned by different natural alleles of Tannin1.;citation_volume=109;citation_year=2012;citation_pmid=22699509;citation_doi=10.1073/pnas.1201700109" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. V. Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Z. Zhang;citation_author=E. Ersoz;citation_author=C.-Q. Lai;citation_author=R. J. Todhunter;citation_author=H. K. Tiwari;citation_title=Mixed linear model approach adapted for genome-wide association studies.;citation_pages=355-360;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208535;citation_doi=10.1038/ng.546" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. Zhao;citation_author=Y. Pang;citation_author=R. A. Dixon;citation_title=The mysteries of proanthocyanidin transport and polymerization.;citation_volume=153;citation_year=2010;citation_pmid=20388668;citation_doi=10.1104/pp.110.155432" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=K. Zhao;citation_author=C.-W. Tung;citation_author=G. C. Eizenga;citation_author=M. H. Wright;citation_author=M. L. Ali;citation_title=Genome-wide association mapping reveals a rich genetic architecture of complex traits in Oryza sativa.;citation_pages=467-467;citation_volume=2;citation_year=2011;citation_pmid=21915109;citation_doi=10.1038/ncomms1467" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Zhu;citation_author=J. Yu;citation_title=Nonmetric multidimensional scaling corrects for population structure in association mapping with different sample types.;citation_volume=182;citation_year=2009;citation_pmid=19414565;citation_doi=10.1534/genetics.108.098863" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Dissecting Genome-Wide Association Signals for Loss-of-Function Phenotypes in Sorghum Flavonoid Pigmentation Traits" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="twitter:description" content="Genome-wide association studies are a powerful method to dissect the genetic basis of traits, although in practice the effects of complex genetic architecture and population structure remain poorly understood. To compare mapping strategies we dissected the genetic control of flavonoid pigmentation traits in the cereal grass sorghum by using high-resolution genotyping-by-sequencing single-nucleotide polymorphism markers. Studying the grain tannin trait, we find that general linear models (GLMs) are not able to precisely map tan1-a , a known loss-of-function allele of the Tannin1 gene, with either a small panel ( n = 142) or large association panel ( n = 336), and that indirect associations limit the mapping of the Tannin1 locus to Mb-resolution. A GLM that accounts for population structure (Q) or standard mixed linear model that accounts for kinship (K) can identify tan1-a , whereas a compressed mixed linear model performs worse than the naive GLM. Interestingly, a simple loss-of-function genome scan, for genotype-phenotype covariation only in the putative loss-of-function allele, is able to precisely identify the Tannin1 gene without considering relatedness. We also find that the tan1-a allele can be mapped with gene resolution in a biparental recombinant inbred line family ( n = 263) using genotyping-by-sequencing markers but lower precision in the mapping of vegetative pigmentation traits suggest that consistent gene-level resolution will likely require larger families or multiple recombinant inbred lines. These findings highlight that complex association signals can emerge from even the simplest traits given epistasis and structured alleles, but that gene-resolution mapping of these traits is possible with high marker density and appropriate models." />
<meta name="og-title" property="og:title" content="Dissecting Genome-Wide Association Signals for Loss-of-Function Phenotypes in Sorghum Flavonoid Pigmentation Traits" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/11/2085" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genome-wide association studies are a powerful method to dissect the genetic basis of traits, although in practice the effects of complex genetic architecture and population structure remain poorly understood. To compare mapping strategies we dissected the genetic control of flavonoid pigmentation traits in the cereal grass sorghum by using high-resolution genotyping-by-sequencing single-nucleotide polymorphism markers. Studying the grain tannin trait, we find that general linear models (GLMs) are not able to precisely map tan1-a , a known loss-of-function allele of the Tannin1 gene, with either a small panel ( n = 142) or large association panel ( n = 336), and that indirect associations limit the mapping of the Tannin1 locus to Mb-resolution. A GLM that accounts for population structure (Q) or standard mixed linear model that accounts for kinship (K) can identify tan1-a , whereas a compressed mixed linear model performs worse than the naive GLM. Interestingly, a simple loss-of-function genome scan, for genotype-phenotype covariation only in the putative loss-of-function allele, is able to precisely identify the Tannin1 gene without considering relatedness. We also find that the tan1-a allele can be mapped with gene resolution in a biparental recombinant inbred line family ( n = 263) using genotyping-by-sequencing markers but lower precision in the mapping of vegetative pigmentation traits suggest that consistent gene-level resolution will likely require larger families or multiple recombinant inbred lines. These findings highlight that complex association signals can emerge from even the simplest traits given epistasis and structured alleles, but that gene-resolution mapping of these traits is possible with high marker density and appropriate models." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
    <title>Dissecting Genome-Wide Association Signals for Loss-of-Function Phenotypes in Sorghum Flavonoid Pigmentation Traits | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"VXYYsl6HRqvQ0x1a5LL9gM6c293Tu_VoE0ievUqoNws","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459579","apath":"\/ggg\/3\/11\/2085.atom","pisa":"ggg;3\/11\/2085","ac":{"\/ggg\/3\/11\/2085.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/11\/2085.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/11\/2085"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/11\/2085"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/11\/2085":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022cyHd2V3YQzDQMs\\\\\\\/9z2z17kgl\\\\\\\/r5bPT06SGDA1IgrCUBPNLYN9LcA\\\\\\\/Gohjyb9rBI7ufRedfFEuvTRd4QDCmQBPCWy51nWAHF2FOfnP21KDsRJmIM1dJwR9ol0PonTSolsUI0sqnXmdwss2jiBBtWOZivB0mTNEgJqTbr+joMe9wAzMuxLrZcHB92GRSgvS79qjxSr9Doa1RmWFLahKUr81WFxZdeXsEIhnEoc0dt1dHfC+q7pVi1q4vwGD7hEE5PNzl5TGAQtCGJR5+Y6STYV1TaBBmrPtimHEmus\\\\\\\/zcFwvqyWdQaSGGQos2N08g3NbaiOnTy90qYjOh4hakoIY77ZF2kDNzwt1jaCP3KwKf1tWr5WOC6lvRw\\\\\\\/1P7Sud1gk4OCkIlnJLLVntL41r3rF\\\\\\\/rXR\\\\\\\/ZtdE9V2XCPeT8X8iw8rt5jWVCEsNVZzW9oWKv5W5xoC443KZgrFqxD1Zqn8J6CYZ7YDVFkOE7ysK2pfNXXTdbUdJwAh3OxWdGxuM+uohV87a8kF8Iz+jJ59Py+NJqxqHzlYezNgFxrRKDpBTTKS\\\\\\\/wGRcxiZMh3uuR0bYBhmSYrjjwyyO+fBgvwiss+x7BJEPXrAwCrDHhAqOTa\\\\\\\/H8EGDroFYcZF760+EPrUR3NR2db9Fb+ZZ3CsqKJH4Ix7EdNXzO+1faskuu5Gs5W4vUQkuOx0X\\\\\\\/wSDyBVDWO8p7FyhyGu5f\\\\\\\/sKYZp+GGqrmB7XDanu7dH8KhDBcBDDMdxXWLrPq4RdtTDZz86jTtJyF6Hi5Jet6tVdQ4SVREyVzpDMMMxm1+PPJwlAZDQF6FwXcrbsSA\\\\\\\/y1MgobkwzbeDRsVoV\\\\\\\/dhVIzqbDI5K7m9Tj9EZv4XqFednTMl0PH\\\\\\\/vr041AirMBAq4tD\\\\\\\/XsbB5rhX7RYdC0mLUdGaxCnunBxmY7d0I+As+w115TIHlT6F8iSmLhO1Rn7fjhjyfsAmdvt8rbiXr1kOncBKw4HYcdw0BNpIMpYZajZzFkENmoo+0+\\\\\\\/691hDHqir3dGCaYH7KsRAAJt5rd5gfBoo0fGIMa1ryQvrNbyeEIcsEbXaGU59olSj2KYfp9YS5mxp2+GFnbvYCvDfuMiWDxyA6kk5MEG6tLHs2SJuvS6zp+cE1\\\\\\\/1R1alUEiIQye\\\\\\\/g5+2OLDpBzYAnCVHsv0lfgx12JvQ06L0gpMgeZfy8K58VbhkkdZXV6FIWJbKDxBHuhXW2aaR5b5WLOYkiFMEI\\\\\\\/okZi1jzBhwGX5Ja3bgZU1oasMxbQXFq4N3KyyNrj2vk3Mg1npXZXB1YII959K\\\u0022,\\\u0022iv\\\u0022:\\\u0022B2Q+uxEtW88ChsXYWIxZMg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d003e99be259cf5cb5e595da6e9b2c08b3cd1b783326b4d32a86078d61b51666\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022BHZnqR4ZDkwGFr2GQlCTP\\\\\\\/WhsLnzOITEkN38JxQNHcpqbufig7HgKsseDqUS\\\\\\\/hvCwHCi3RE8Lm2EqiYPQJiRcEBCXBql1TTG+TpQkyux3S5tksvQW8dc5ivLmksE2ggYj\\\\\\\/q\\\\\\\/z8vsyzAnA31HJUviOCioiQRRZXoCQzniEKuI6+WuiikO+jB8nb8IfPgRpD4uVu1W2gfUUvviCIc13XgVjKpMCOCL1oVd1t8Tb03J27mAsbLvtoop5JJr0gXwxRw+5tKd9t478TwJCoHGGygDauhSYr3PifUsH+\\\\\\\/zNqktSAqfvqhXu41RG5tveeyCLFr+dmZ72R2imUAZi4gW+evJDOlnyKMutVGvmss5r875Vem6wIriX7O5tEZA0mgS9UbMWYFgfKUC6vH9+8JNIB5FqoojKbeMpg+QvwQL2y9GMI8K0n4D+i5oBbo7nhZyRACCONxq8FF8JoLukpwMK6TDY9o5bJ8\\\\\\\/OTGi0HWYQgV9LlWV11gsSlCbGSAq\\\\\\\/N4QwsXZ749s0066pDN9cgeAABTFdk4USR4oui1v5GMSUt+hHVn2+cnVy7AfsJWxye75omk+BZn1Jk\\\\\\\/EtJYLgGy7DxCOBatH8Bd4Hat8MsV3qHLeIprmOn+DabDKTvLX\\\\\\\/\\\\\\\/q\\\\\\\/KqU9A5d8RhL2x4NhmUbtbPyKM0GVpS6TFJu83FIg2W7k8I0mX6CoBRX806z+gF7IdLT7wrdJanhVtn0FzMmkEbVdthDVZg0f4I7yPeb5UmtcQBbZ5TMAG\\\\\\\/NXIXkSeS+nPKFsNB4gu3h0uKJO932mwfm6IaShfOicEvcOkTzQzQTgaQDbAtlEw2vjTOGBBrTOCFCCEGNkX8ZBn2oVWJlibGaIF40ImWyyHkZH1eExrJV5Af2L1ojodI7ji8UtfanhfFVNregAJ5gB7LNbvR+ep\\\\\\\/oPjIDGdDfRZci6gA18DNadDUhHbA2pizg4AifUejeSrC+Xk31xqLhiW2gfcairEXhG9eSu3irzS5oFeFOKxyMPBKZ2kjuMqRqdncPxHg06GNIIaZPlXVf7UclLaY\\\\\\\/7GYzmWbwNn0sne7ageW4hx4fgzQRife5BMs+QayPfrNvp2WxEAPlO3AW3M1ruxECQqkTu0hC+1lwca34XzvF5xUjeKvRkRK\\\\\\\/+mD4kqiXhl9FWYz6KhmYiaQIEgbKGeiCY8DEFc2+w2S9BOtM+1+LHDQuQ8JawJBztfwhrWK8J96kHo4qXXgopR7Hnh1DW2xFQGcvPDwBs9DVMIde5nX0xxgKTGIPTkYDPx64VoVxU7\\\\\\\/lA9EPdeuCA9bjS6iK5uTgLpW0ZsFElBnU90M526C2rlRP25Ka6qX7Vq0Vzy8\\\\\\\/FH7kcmAfW6LStkm33cwRyk9wIL14H61Z\\\\\\\/etPR3+TGhElMXzO+WFfn89ewNoM\\\\\\\/JLddd\\\\\\\/iAiMMQuVtv3oZ9mA9DN3x6QdzrtKZMstyOhu+JrE\\\\\\\/OaFfBfO0hx\\\\\\\/4lN1MYbDyFCsrb6Xb3igCecm+EcZaX2\\\\\\\/AG7K1vy3dBe4ut2FjZRoon9viumz9aJmsakxuToKI+IBZc1LY6RxOylgZl0m4844nHdBpkZ7puUb5g8EVTtvB4fXKva9+aaCJQ1gNB5oHwgCk\\\\\\\/9LOUkH4zaFHlbvEg7tQqFeiNHKYlR1IGgy7MlF8iLRmmG\\\\\\\/SD8WJaf8UmjFH6WdRPwZ0wL7hLbxQmRGwvhUxS9Mm2Wj3noBROX0kDXfppe8qpwNKax6MakWJVilFFjcTL3yKmq+dILl9mZ73gU58v2eBw7tWtqmvMVuRhgqce4ef72X+L6tgDRE2j91\\\\\\\/cA2\\\\\\\/UP50IJW7l9\\\\\\\/QJZo+dONdLQWyKchIKUalhcjErIaLTbtBGvFFIfJzBBCdwx9igEYKGQlObVmnJdalXcZmAjpBukUlyJqI30LOBV\\\\\\\/nA+tBWu4dA9OE5WReOGtKbN2spMl6dOvJhneKbiRRjvOFrmF5TkfFwzzOjiq6nOzmAj6IB3W\\\\\\\/4MoA3NCmi+uMzJGUKIhDsKwaWfgd4Apd37gHyTfAKs9oFVKVOxOziZuO6kvrmkxXPjEuNR2bG7GHvakytRD7Y2QgVFp793Llm9eb67sYy9OrB6tmE\\\\\\\/RnPcs5QriBIs7oJV7UI7P9mR34eT9Wiy04kZ0KJCi7NsrvhRyn94pfMjWKkQeQxkBSxa0OBS5KiTgJWtgZMExOga4WCpe\\\\\\\/GLgAXqlR9r3FWGXpyrZvfQlNBtqyNfqS6TL0YitdnzhUFZM2DjIZOZvHQchhTkoWkQZfxUDnwNMC62GcZBPpsDShS+s5AM0JY9tR+oNA4Pz0hyHjeI1+FFRRL6nuJqgCbrD57MGjttXBCDcCV9R7KXZK5oThRaW+p2b6XSPybhqDQcXkDdSO4ltd9OTigRmXnV4DrP6pTWLpwy949sDjjbrsJ7eXuSIfzC0rQLUocQ\\\\\\\/nfMR0sDeVwOx1\\\\\\\/2a6GuVXqAb1Iur3h4u+1I0\\\\\\\/9dknJSsJUux60CgStIOrXjOeVAGk4JiCtNGZobWeeNRsjzKuM1mrrOAa2iAck9FJgmEOCGsVatNM6GKk6xuWMIoJMG7RQlsIhH5nChuLDZtDdQKhQo8tcC0J8Cfy6eOxCjgxrTcuYeYQBImn+X+VaMz9+k9fXt1kePyvAVi3mG6PmB+2KGIJQ515XgCindsf43ANlfczhytLWg6D0wBECdXtTLSk0YxeMbhLMmFC4pYPVse+EmYSL81U8aynabcW\\\\\\\/B\\\\\\\/zYP2c3rFh72ijM1HOSbt20ISma2Pqd7PJjza0jPLyysOB8HHaB8u5\\\\\\\/33IaWRSL+tUX+Cb+kG9nNRIaA1aUQDc29\\\\\\\/CL8LZoLejWiQtduTimgKK7igOyuEQSpHZCoqV6ZTnzVFzJfh2LlHFfxucurJYfO1+G6nKuFlyiWWVXim3tF2JTFLworpRDsP+zF+IwXygSdUieBBvi6VwIE7Vupzg2qr\\\\\\\/3v\\\\\\\/365n6wJICqU1369vlTcfZWJnuMDZAoojmAHgkvkv9vQBSfrhtkvA5XQuGNvNoPy97SkXupHgjpwzHnw8GcLbDxe426lB5IQqN6pCG7TSYK5qXW1oCCFXuE+HeRSLfaoe0PW2JpuroE2g0FKmRyoLN6kADKvGYUGiiZJSPW28A91ER5MeP2FXLJ\\\\\\\/GL64XoP2S8ZjigjcD1lxMW3shdnd\\\\\\\/s1QTXHsiECzo47fRJewDNWMNMJpdofHzCGUXYWjyCqVivJluP32ajuht+gbydWCWevgOVjqfagc8oSTf95i0eM9ve9xpqrgOk4Hljtv174mnyAtwlhoAGfKUtxADIbcYt6\\\\\\\/wC9zKu3dKxV3PS6nd40Pb9RxrwsuWI3EwUd261K\\\\\\\/QwyOvkvZWtyHqSlvEEsqcyK7hb1oewFB\\\\\\\/RpsN+hKwrUGNZyJqVKsZT6mBU+TINyS33AAzQ748fpbS9D\\\\\\\/s4iqsXEHbNHPpYCUEnsVWW2q\\\\\\\/TLze53dZrye1ZkQtXs3N7ZNoC2xgwssRkPtmSnByxbYCPQ0LuEF5rkrOaTUv4sOxxjEwDCU5tD9RMCO7d2Xwe6lbWPdvC4F8wvq74CtNWHmJrBHTinQ4VT0HXMFM02V05ZwT5bvECle+yANuIWbSyQzUyh6uCkv+oxNEd31kctZsSslMcF2OkJOrIthrrhSsgBA4kUQnTDqtKtr7tz8p3U9lWlN3uNEY85rMbdZ3gKGT929SKxu\\\\\\\/lo5bp9PMiIUE6uzPDIBAyjLebNhcFEXp3cB\\\\\\\/SQsC8MbiCWFfQd0PQj3LMq6yyBpxqCavazb4e0vbFjGzzVeXZpmsrEY9ip2tFMDb3oioBHcTeG4qhTcRyC8RD3MHnV4ruu+zm7kz3V1qZVqiMP3gcqU4WCIUxUrMymFRcIxSiYNSRsFbvQ+GfUMMJsyJ0ek6KUg9dEr0vI58t8JWq8Bp\\\\\\\/3jy3xTteuu\\\\\\\/6xaeSQM7gg7KEHuG8OczrdJvUonpRxnl+zRVLklJuSXvuPAvQsUI7GMtImzgl4B5bj\\\\\\\/SNAOF47RyLVA+4R8oHDXhVeTI8nUhZhfEMlY6LE8zg\\\\\\\/CsvklSBmRSZmalKRnMYCtuRCmu1wKJSbM2M85R9LtxGu18yIDbIwxEDV6DkIbeOWO2KfWYKUc1nIX11ux+rTOHn7CSHYSbuuhLZYwMyf5IXQ2x8fTc3gixEJoVyut91l+1krXNIcjEE+jmPDDxW5g+LPR28QS3fPwjBGmpg2LAd48JZ\\\\\\\/qMr6IqQQ4vnd4fI9uhKEYSjIe1cw9Y7LqDHgB4z6JS+EyVgbGK08eWrA\\\\\\\/H54yB7pnU8c6RlL0U1XdnLSmu3DqZV5PAAg43OHaF0Xdb6lUoQWzA2IUSxH8Tt84MExXFmB87q5sA0Y3d9tKv4jzDoa2HebvwB1ftzDcfUFo4MUz\\\\\\\/glZgXkB91Rg8Q8gVpTak8Z\\\\\\\/YrI4NMUFlgct2RpQll1ecOWeKJzMbjtbd8g5fu8OBeQg30\\\\\\\/MdvK+vFJAwfY1leaWqgs9NDBTFgks0wlXrKqgifWKPpjoWX3Ntiqn1dz\\\\\\\/Mlzo8F6bYIUnyDy39t4hcaGdRzOooVLDcmUImVyGRUANA9R1ewHjp8dweWni9hHO0MVwerDgj6J7LC9C4+oFD5IVAryvCbYtaePNPHFNWDSuvDTkZRC5lRDrABy2N35Gvo6Y5I55hGQGLspUELrY0acBYTM2nany3tJbTkqv\\\\\\\/Ju+H3YXrjvTuwtGRsz94jIvHpE1qADhXLcw0CuCz3ZTNPC8k\\\\\\\/Z3e8B7xQ+db8e1U0sT+ePvaxgLHo6cHWTjre1DqT35FKFJ5dZGanvSJlKtStzNAFpmOY0ix8bLtnKq4ihtd0PytUDnENAfOxVvtlMdfUUP3OwYzOVRcH1btjekcmfaiKQjV1j1WggEZZhZI3HCwelNdJiOiwTzqnUYOBCDu2sn0vsH2inpp10ytbbvoyMPW+v9QQG\\\\\\\/aSZGW3BrNr7lZBdZHaCRVnTn4j7g5sz0LHoOhrgXvtb0X+XzNiVyRszDDJBRATUeI4hd+Os+GbDDwGDVHBVCn0d40LiwAkJxTCrNe5Ju0nYGoZJYvHAWLlBYqXojpGbINQnCcW9KvYWvOOwhi3FxBuzcAjyBIYiJATFMvtBfO9PIh\\\\\\\/\\\\\\\/K8aAdUcz5o1pJ0SqWdqcLwvj13HW+zztkpUZYp\\\\\\\/4F2\\\\\\\/csgzu7VQpyJQET80M6BWEMFQ1+pEFezZ6eii5dvlo3sEdz52TIdla7C+wLaqtyGfI1sw6Hic0LzS+t111ZUss88IEC2rT\\\\\\\/u4PRWqIhrY6jjxFHEEKO6fFPHRLKwiBx41K8e+hePHcUiv7gvHlCGl6TeCSgY4N6kJZYs9p1eiF+FRur5d23cJ5GyBkH48BC990KXeMfF80uzFIyFSldo7f8erzZEboSVGi5lJmp\\\\\\\/+jyKyh66Ln4dkIyIoexEDI4cicJrsHNR+26DGdFn\\\\\\\/lMjoKGo95eicGX7x8OOVZL5FQRSJ2tyO0vUmS8y\\\\\\\/S\\\\\\\/VxOPgXwqpDpWEIy0o\\\\\\\/ycdHO85sixdhSyRpD4Tv6G6yP5rwzwccd3zZUyUhfrcWiah+y5vLKJHegYnhB6QPMRXsHYmseWy8FXJ8hV\\\\\\\/fnjrVbTgdvhK4r+kSvtz7KqZ4uRz1XwQTMd97OYQQsPNrJ6eGxcP\\\\\\\/MYThMcjCRET05biGaO4WHUOCKWgk19VD2wImRYgqzxRwl7eHn0FzLqnDzna4bEYuPVfW+VCsYOxE2q3lRYhiDHq22WBgqJkMddCvLdGeQol\\\\\\\/tlOiV74NmjyunzHaUzX02xbVt9HjRPnCqZeee1FcsTNDDE6+jGq7Uq8KarjzUA4FixaqWIHPfi6kKIFOU1BlreIz8Egwe3fTKs9UdUoiahY5Tyepn6RWxB10ruYqiddqnwPhBWdP0pSUGu3RstT3SzDVOQ3MJOJIt9O0uLQioAVS80qCjGORzd0+I\\\\\\\/b\\\\\\\/ejGy1Zfb6kDZC8GyAq9vubIccqzoymFraVtdlMvJPooKKc8YoeDeEFj59G85dTrSiMmxEyY5J+UfVg1jfoLnidLj26u9zGc6dOVtQxUov9yhuk0bGsLvvOfNifJF+i\\\\\\\/YsLnXQmhlEy3bWwbK6MxpZDlZTZ3AwGzsxKtV6k8Pqtc8lWscf6TWmVDLiBfDGquHvQt9dUbX1CY5lY2j6O7ziMcnjEWvfq51lo5oHIB0WH7F6vCs0NIOTdK6OJ7HOXwvsOJqdqoy8JA8VTCAfx03+fc4+FtwpNZSRNWdHCdtgPpOAG8nZ\\\\\\\/YPuUKG0hONHrXHgeYXX9bYbYdhWE+4\\\\\\\/TvnMgXGsAaT2BbWxbmRpSkzN4xoRjuhczoCEcTFpTFIBScvPzW+wxdENS\\\\\\\/0OjH38RSRg8brDgesw9R5ha1oUaQ70rc+khhRPjxFZ3LhLkyfZJ68oP6LfXQlvQ+wIEmxJswcJmcjB2fkRf2Nasf2Q4uH1sIpNKW1CmenJRj+HWVDELzDybwRM9zySF2h7Kl8k9J5PbVbYvxuyI1V7abdBXaID1U38SemJGccvwSQlpRnxju+W5pRYp6xyCU2kpp+\\\\\\\/8yJWfgQ+F3CQ3QM1aySF4T0svpAOyle\\\\\\\/bq7VG8yOblxrMKQR0RptwZuN2Sq6Eg39qdjRs0HO6x4W\\\\\\\/b1chylEZhsfHz91e8Ms+7GI\\\\\\\/ImeqwO0jLG6HEAiqt0VCVMTzs2rgSD8WeHzoYjyhGOF1G6pqkhNg4tAsMp7yGkJ9kLq98GnLKB3f\\\\\\\/iKoYdsex+blm4y5A9KrrgwybAHSK+XbM\\\\\\\/vG79jeecneN\\\\\\\/a2BYUGIyyjwher63jTVyAb1WS92Y26Mf1mKGAEC2rO15S7ve4BKHS+R9NBLlotk8HSoOtLwbg00YOErz8Zwc9t4\\\\\\\/FDimQjFvi79huYLvZZi0T80jSTkQ5r9nxqWGOdMUPHX9x0IDarOEbU8GXJxuGNZ3t0GSmA2nOafORH3Xahn2ADmwulQ0pVzJ4hCgdvx+5hQy1quBKOYj1Exat6HIwjs3Bi+Q2Ejf+soG\\\\\\\/s1O90o5LFLD+ErXC9EB3GbRjhvRv9P9PAyl5fo+h9jotrzWzteKaFjE563zgPjZUlJB7GWI44mzgI9s471vT8LLe+9qx01l7tbiI4+ycVk\\\\\\\/WIgK95cZAB9TsHPPhpwsJ1wL0b6eKhyDk\\\\\\\/IJnHvP8vsiaObf96iogBcC0nyIVF6mIieXGKIdtH4uuBmv4Sc5awVpS8WEGgxVP+3LAsWptpEkRaCWioxs29QeWspPjNTM2KDK9eFCJR4YKDhjbHw8qXAIqNfoZiwtAOXvh8HMDwviTMIUZuTeyvYz83RfSdT15kBmiar7orDiop+Pl+58pjdpY++Unsx5HUwlRRsVgeIrDDvQUu1OncD1KphjzeryDLHVRHEGnYfu1rr11Ym2tW6YfYI\\\\\\\/xdg7riu4FycVipDQ3fdXEZxxfeeXwMVQGnXip+szbemm5olmZ14Qi70L3rQatKKIdWVR0+tYoanxksdxX9S7tPRGM1m34oq1xW8C\\\\\\\/Npgq9h7SDa6mowySl1lfngZi3OK5bjBPgXJ8IQvWFWj2Q2MnSJuqbaqVwYuHAdtCxhE3KX2\\\\\\\/4z1CqkcsuI2sWKjKM5Rsk1d+zGREeQe5O6zU8yAHwcwbNKjNgxe0SBKEJtAxpnNf2bppjdSxTtd3PbI7L22Zct+NHMwlj+yZ7\\\\\\\/1C30foIr8x6+ty4W2shnpmWI6Q3+YUHjI5N\\\\\\\/m708Hp+JWxwnwIJNCnCF08LXzXl1F08ZMu\\\\\\\/xAackjd+byYkOIr8bCRjfiAWxg31NHUx\\\\\\\/7of4d1Ce8snwoQBWUQAS9BQxi5+fVm\\\\\\\/6P7NVx7hb1U08125s7ZJmvrQWQukYNr8sVXxqYS9MoIWAZGJv8m2hAauyct3SiRTbAC1V+R5IsTlIrFKD5ri6t5NYmIsN765ebmWA4YwiQlW9nei8uS2A6se8RNLV8UiOR4SoK2MFWqH\\\\\\\/MRz3dhWD2pn9\\\\\\\/X8Q1lSu0qh797A4SKcqHHzBDxDgTgoDV+TVeEwafhcCWJCyJv7kjm6JWXoBH3O9vnORdy4fdMdWMUAsecTqDn99NCWvcTv\\\\\\\/op1OEId\\\\\\\/4qNW89hRzMn3SHufRvu1aIRuwwwl0Ns+faXTY+E+PEfqazoTG8Ny1lQQdnXPDdafZ2hlViTsDbcE1vPTO6Vr5Rs3LPKuivvUJ8jfrSgOYxmAx2Bsq4g2WRO6FBdWjSTg5O+WPV0CEMniwtJmAUdnzbgIkJvq2C0LYftE3DCEySQk5MN9NeWR9w+705IxFgvqe1uAHzfeFnKE4plRXB0bNSgqCv7KwNkyusawMuRVdaiRKqWcznu4\\\\\\\/vfNL7+MvIdkClBviy52Ib6gwPZ2tTROkNh8IEw0Q4dBU6ZD4JSYhIauGeC76x42NIhSplKci\\\\\\\/4PIi8cUWyDYaNTKrMMbfUuoeN1PqkqoyOkNlIzIDshvyCEVHZMhwk7c+bpsYqGSdqJ140io4VwQ\\\\\\\/oMpehmWCWiCXpieuAx351L68glb0qq5PMfoq1xEvhlMZwtZExQh1krHA1jJNXsxPAPJoV4f39OgjAN+A+ULlRtcpnRm7aoGQDCinwIQeKTbyNGLos1Fm6N6003nEOaxcZF5zkdjH4lbjzkOE7W8IKsUHN+jSp5cVz2N5lNBxI9p1MAwpcgoaxfiS\\\\\\\/bJJ\\\\\\\/6ZiHGeRRWuEAtJ5txSK2WDCCafTJ\\\\\\\/XnkzFonmab5ylWGT2dAf9TYnJpoky5\\\\\\\/HVKdumqQBNgNMLkuKDLgLfLmK7Luu+KvpHS2p4BMP8UVvoT5mI8cyW+L3rK0N1RRgLUEb3aqNGl6otfsPBRF24D1yeFwQMDal12z\\\\\\\/gPZha29H\\\\\\\/jqVJeAYALXd5q1eCinImj2EtH+sRhbLPmlye+EHI3ovYidCQ8mTxOjm6hBKyj7L4u0AMkMylm9XbF9CjxafEsEMOx7JtMl3fF6xRvvqI+ol\\\\\\\/rYZm90Wu3EDyb++HdYWrOJxdVxXNCr4ZK8rHJyHDfj8ofZYsxwpARerG0t2gelZOfiefX4f\\\\\\\/Ff81Hno+llJz5DhVer6BYsyEj5D42nrJc0CwMkwoDDmJTd+PGqKIS3g7GNQElECMK6HxiulELpPz115\\\\\\\/+iP7stBB+0+5ucasmwzyXpeEJzBBOYQMXnwqrqfASeI4rSGtjAmRpWpFOk\\\\\\\/IC9uI\\\\\\\/QcaYk9aKgb4UJLH5o2Vpk90+Fnl\\\\\\\/NlLuiBH67bfomOIkcFIwDlOaeZ81o0o+F5AGPMluGRbAjgO\\\\\\\/AwdfS3DcL+MhTXxIf1vamMDOuUY1nBZxPFQpTG6pTo3HB7W4kRLyhe7Ipt8+XvpT0wTWIK5OJ4ph48anYZxHCuYEVGPJ1D6T9ctS9atShnpzno9Nzrnh3ALf+OvVHuzxU54mU3i7v3XVzGXZAsvZ85K3zUKVQ8SWsgdId6d0B1IntDPrakvY13k00asgKVDGTpaC6U+xJS5UTLsWNXq6gWiBTM17HMrWG9KRvM8uZwPlU9mmYKYJDzO7daNEs73DtD7rMqyCfRelEDflVyt5AUwppBQKAchjwWNyUY3ro38d1BlTeFnTGnkqF5VwQq7KPjfhPra+nFYS53QiTpfXwP9Ug4nLsDUifhVpH1CkO6vL9hkDfccCX\\\\\\\/6wg5AW96hRGeEbgmFvHQ2F\\\\\\\/o+3IvgqbEmfir\\\\\\\/3UduyxXTH+07AFtmxhOXGoSrKwCOJ55ZwU0pZHbmk8114JJOtE11c6KXn+iHmR6x+eJsQ6If7oyQ7OahQlFJyalH+MjTpo6AnPMeKrOMO3k5WL3GCvxkWU8Q3RI+R+YMf7puRqhd2dAFBLmi12pKFEkJQNH4CxeNPqpaCavkfCXJ0Prc3NBqyC+B0y\\\\\\\/pSCA8kC6K+zMSIVlKYvKivn+UzvtNqkz031tPg\\\\\\\/ubwMK5N3bJIjV+pwssz\\\\\\\/Jr8GSeIw1vCrB5lM0SzCOTZFtg+V9i12LjTIIca2KL3s9ZadPF2m\\\\\\\/r8tQrhRcm7\\\\\\\/sSSJRLsoXIB9DnHkTQKspTEGxygm0M6\\\\\\\/RcvLheDkTqUlIc43H5ZrCPg4dnkAkJ3oEJsGRRy4jDdGr4JxYG9yatJraUUqPkboHviWIv5ZOWkBvHXwK2DrFC7HFF\\\\\\\/P7X9sYupr2oWtQ0APO2xhvLEoSELKeis\\\\\\\/bEl7JRYWwP33HB+ASDBlNoK5D63VZWLHZ\\\\\\\/tuwki0rQ3Nbu2F7bx6YAB2FOQjJWFhC4Zpe7CY8awB1tz6Uw6vmqfGr7ZaFT0gYzXe+fXwpIgGqsBJmakl9gAWJi6p5qKR7y2xn33g5UefSRWKH5cyiAgv\\\\\\\/qSJrwYv6YDIaA8k9LAEkrnRlXThWyWdFBGOqlexBP7xpMrOHe7NpZhWCPSVbHZvLMOS4em4C4mXuhjSPhTq1bfH8c\\\\\\\/VkQlWS1s+YdqCpYpn3b5LBnwkBU2zhStpzQwEqyvsXxqNE7gmz6tn5vAEx1pf8GnN92piWOnWWTujV2XZqJtpqoC3IQu6JfI+DMBr\\\\\\\/i5tyIbKQnalYf14Br769oAVGCVp6yp8vwkIlpFHj2b\\\\\\\/xUEZqgZ2VFeur24WvYIhEh4zoFsWCAnMz+vnVoNd7aBG2vHlNSUi3wXgLKuVeUCcKogAK5djbHVW4yXFzH\\\\\\\/N0Ytd6qA9Sq+MBTHoSUI\\\\\\\/CeGnbo8lnASTDz8NZI50A0x8tHEqMQNdYCn5fG7lTC8dtKFv3V+T0zPMFMtQSqACa+hgV0wFgbvsNgI3JxajwbMtEOOrVCAOhROKn1Wdfz\\\\\\\/gPp56E\\\\\\\/0d4\\\\\\\/PIb+sBAxLQS8vcQr0DtE4OBrVurj8FCfmVQrPEaByJjQ9AJnMZoJioxD5+d\\\\\\\/6Aih7RiT53BC2nOhpUZjJxIzfB11haIq\\\\\\\/Cwvbp+YaYcIqrkuyzu7YPSXakESxM7ZvuKCYLXc8fYt9SLrfBACN59OIhGjS9shLDVt4gLWJWJPN3z1lsDiNK3HysHhUN+dJlwICjhBPFpf4dTde2MUloWvz05jXzfzDsV6LnIcUrZRxIrDC6SHa9h8lIq3jUiq1OPpy4aApfmewo0l\\\\\\\/w2BHVuRLHJbv\\\\\\\/DC37bTPG8fl9N7bhe5MrFEyUf92uOEgb1clBtMNT5JEOBVYcSTQd7jz6FsDLOFEO7Kc1E\\\\\\\/HOu6n5byC2T7aKJSbCkLDWEv7+8BcusCrpbGNHhlO7S4WIXD78NXVl1bKccSJJ7O9bhkHkBX\\\\\\\/lRakx5F3piOEharXIVTmRKPusxLjSMOaHbqa\\\\\\\/Is8vDagxYMNoOeaKN1\\\\\\\/vA6opnikebEzkSC78j91qTTTpZv\\\\\\\/U2wlev5lMYZlNaI2NGN+QdqkXToUf5iGU0EmBmAVS+rnga46QMuq67FDyawvC\\\\\\\/tuHVJ5x8dNP5GB5XYK4KNPv+856bB1bYFhJm0N2n2NiU4PET0R+YEA3tu8nRJNqwRrYrMSh1mfMo1P5QGVFZOksbdUyuQKdhSQka5ucs3U+WC4d627pAfpO6DJeaQryYIvXQuOOE2SmErto1o+ZTjrjY60zkHCsa1Df6uSZiqSILe0vao7XfFYc0WeayNl6nMseK3IKYAHDAotiXgxkbP6DGEsmUNkR7nTVPebbn3pnwwpKTvSYkMfyLD5K1TGBcD3SxnwHzDCvakA+EQPGNh669faIkLmsDTKA2qNe+zzYMFYiGDz\\\\\\\/H9LpmeAxJECaq\\\\\\\/DykvK9dQeDUhvsOBPsW21IWErOzQg8sWE+PJc0JcFHZXbBm2bafnzqRuPrvJ+thEnB72GO9PLFS2givq9p4HH5lxOyWfoIQUeleFq0rFfeUfcNmYRB3P+2Ih48DiODq4mmw8y+ImCkxb76R4nH88I6X3hEeZAI4\\\\\\\/KorrxKixbSFAZccVn16+gHbPW\\\\\\\/JF5EdVQTwuPWEsjSWRTDBEskko3ufCK6TXOjWm8WiA5H7ZVk2UAzGuUWQgpsjl8I\\\\\\\/yahD3\\\\\\\/S4JT4lwbO\\\\\\\/zC+NRvBPwDS9Whf4jDU448afENARV9KfYsUiW8sa4D09GOE5qLUO3J\\\\\\\/6WUp2l2kCzN6QjciRb\\\\\\\/dG0zFzwimQX\\\\\\\/b9k1luA8kPYrHSBxQK0mtBxearfvDjvEnkav0V5ALWh660Yqi6JYyErHFwfozPIgYd03fgJ3NRONR3cfqZOAbxfab33kvoys+1SfF9rc73jbfsKI7M\\\\\\\/MoN6kjcbo\\\\\\\/bVabpVYrpzltD7d6NSNlmkXk1Q2RjPNhcwd+ZvVwUHNnTANWqmYFciNjjMFMYinIkfY4aqHjrW1J3HUjKMeOz0R0j\\\\\\\/gajOdogJBrC43hNf0uaV6+RyTmTxzGmOfFQ2f8gIYHiq1iAyi3S2Yc0ZQMsVuwQK4tw2Ts+OT+kSzm\\\\\\\/mw5w3n41MamDVF9S\\\\\\\/O7Fot\\\\\\\/OYbTQn+TsToqb2YmPYxf8dzc5AvetSTXW6ChWxv4w+0VweUhe3TYY5TroRxxLmvDJ3JCn5jurnNpYWPY9lgb6tLoeiNXjfkJlhl8bDyZU\\\\\\\/5tbHfV+zFpFjUmmfxetqH6v6Rf6eRHsynCL8fneTd+P+KM3DQPix4PrI+L7CVza2KW8o9NzwFNFADYyJyLaHVKUWndE5rU4zQXfzLEuTodWk9K6k35JqIMjtJe1I7tYeAlRZxwulcFM5ZDQJwV0hOYYIiM00hqZCBEokjT73hpGxwJQ+LgffHNEaP\\\\\\\/bh60ECUIaGOdeGCI3JAezlrY2SaowY\\\\\\\/TyO\\\\\\\/lhP1yRo3UZ5Uzn62gBcLZmsI1Q1ixUuxqSZvxz10UXVwiLih2VYQCq+SPALAI7ky6fL8GwEVdb9cxy0zIsCF8Ac5LfrWMpCG\\\\\\\/NyUZnYqBHSLIrTRD\\\\\\\/W9ux535yf8+aMfOzJd1L98H3SbFtXn0JON5l35tdYen\\\\\\\/TrrRtrj57J3YHcc0XKNLC+s1bY3Go\\\\\\\/stl83DkcXjbq3hCBLkx51V5BXcGZfF4HKNVIlXmzLGBTjPGvpl3vb05GaTbcUz2PW5Dm4IsFC6TKK13J4aE0QlvojE\\\\\\\/hCv92YZaJPTOCXDKLUEGY7jLyhvYEL8DAwIuL1w+SSVQtq98Bu56QlGThNdSkvuGpZJ6zueInV+7Wf4ZkNge9G5suGzGS07cieY1mnQbOzFbBHWArBMmUxvcN3KEdDKXB0lmBdn5Sp\\\\\\\/b+K1vD2U0kpsNHK8l2co6AK5GDcyksIJQPCsMsktJaBvT4Q\\\\\\\/zuHcbT94WMxG5x+6Cp4h+RebKtGKRSsiprQNOfxdepg2ooG73YkR1gT\\\\\\\/bCRnsZl\\\\\\\/JpjX7yuPf9icy8UHm2sXFG2lk1ymwXiLf6Nz5plXQrMlm+8PL\\\\\\\/Yuo9y67Pe9VvJDvEUGAbFY4RYbtMaBYw722RfNjO0CBPlZWO9FD434CeHG\\\\\\\/G7V0WsLswehRP7ni1xoNKUwSGhTDcwMrVGAGpoNixOkJI2w11kuF2+K0NqNcN3p0GIBn71wcn1j3Ep68kGV0jQen7VIhvXkQrnWYpXg555T8vQ3pJl09y1TeV73pqAMI+PDy2ovNZ3R6EZpGdjUN023kUUks0I+FdE54TAfrswDD9ikDTAimP42SGwcCTEEjhJ4ekMa+VK9xYDRXEq+J\\\\\\\/8zdh3B0ej5OQemaJhCY8wKww1M\\\\\\\/cHghvrWgy3mG3cvDxIRnS\\\\\\\/x0Jj91akuIgBVF2nVzIqyPmmm1YX0TifIuyHoQ1NbcSdqcQ5lynAMp2M5bEMARGtcr89S5AUVDnGzC4ngiRkfPJIEssXK5MzNZ3Sc0VDJ1uAlXRk0wKOO13Wz\\\\\\\/\\\\\\\/91t0uAOKqZws4bjQNqRAGCOKPinoGTsznrwuX3lea1LIfxxZLlqK+DYJEy6OHFgLsS0+9Ic2r+7jqBZi5fLu2C9hRtb0AO9TMmK2fbTaTvv8bVuVCS0RJfWFomj21ROKmiftnZSJ1gJtvsvDtwLud5\\\\\\\/NqyzQsE\\\\\\\/R91vy\\\\\\\/7QARfjBNVep88QnNcBZ\\\\\\\/xOVBlJyPfeBakoT1iPOvPhhlV6x9FtAReMb03gse0xd29lMU7+wPfOeDogGnXFV2eeXlIZlCEXH+OpErmwYU1zbrJl7sHcFMaps8k1XYRjYOfUN9A9DwyEoo6LWnY\\\\\\\/QL+wEYRr8eZXJfbLFf63lq7Fjfi\\\\\\\/kCaORbin9YAI6Bl1ovy3jZq9k+UEGWT40KHZXqoyZkRUh3JwaukoQzHIMgF\\\\\\\/igdbdtxmQtnjagI4DLJW8g47UgTYLFY2PZmS+lboj5w88xasyheIWuPQJzF3X5fTwsxmiPk\\\\\\\/7aNFZpQDE3MY9NzHELW0OqgrzSWLWE4Bf0Em7LWcykeSSszc5fQSMpt2I98fJuR0k25BAAkErrGRYmGwlT82VYc80klrXHZ1twolKwFqHocFoHnZfX0xydisKltz7YlYoPSsfx+ZUTUhphDqZ95e+8capOHL8abmXW04\\\\\\\/M1ev3ZI+\\\\\\\/xVBmN1Rn4jxWkWv1pqfVIiPSPayDze9ej9Hun8IV2+V8b8omOMt\\\\\\\/rkRoxokkqM4w\\\\\\\/MBG47AN1EsIIAFXbTnsGpthjqAT7uWu9FGBz4HMv7960H6KD0146l+QpCPA\\\\\\\/ph7NO\\\\\\\/eLPxcu8rViAUtYEHCbbdNs1oiKVM5TOLcGOkPoaOdhUND00YC3T3SSlPZRQ\\\\\\\/3bVAfNKM5FH7+ACJYz22BVE3Y5MCt\\\\\\\/JrlP\\\\\\\/U+5X66EAXH0qkic0sEQNYH3hpiYopQORRO44sVAeUWIJEcs+WE+4IbFh+NSZ49tV7g1xL23eg3GXxJ23W7DL7pByBwYm2Fvqav35j5LSBBKpm9qcnbeiPmaJWUOl+uSgvUvb0YtCCyx736bA3fU1jGS8LMF+t\\\\\\\/JBEhTZmeBTL5pmGO5cO0O8CIJZUUFJkkTXB3xvRUgtoPm5buMKELB53m6re+LVcG0sdcBiHT2uHRIh6w2DAnsPihV90W+m+we18504WVf997SxD90LbnE+ThsusUHkNZWu3Pqjs1E5tYGcFq0rKfzqw\\\\\\\/A+zKEvc4KDrZ+hRO8D0vSTa3fGizb2ASDBfNqRDnVIe9MACJ+qt1tktYOX0\\\\\\\/at5VFaPBMTgPqsFCRrYpPI\\\\\\\/vzufaxmyy26yD7ZyrYc+6w7wkWqDZ1jMNbO3\\\\\\\/gPykfWykLPJA59XaDcf\\\\\\\/2x4Mnq+PSP7Ix5by8f4l7FBfACObnVwAwYqK2\\\\\\\/ZixRUsfAFex+NzgbMb\\\\\\\/dWfbndNAzD5OIUp0i\\\\\\\/z3fUTtQGZ0PtXJEi3b40xByUVCBoG4BJpP07l3zzE5mMP+31p2gDCWbx9DL0GHSiCSuxBUcUiSpKuT7x0WYY43Phe9IUOX2yAJGy8UfgVGUCMIQV24oKILEN1n\\\\\\\/CR9aWV9RfR24igk2MKEYMnVb+BikH59GEo6A1ve5sGSIQvlHPoxJ2DE\\\\\\\/Tg569PRdufJUOR\\\\\\\/D8iabIok\\\\\\\/4XAJ1mNpNt0m13UNJgHQFsGHIeJsZSmEoYGJn7mdZWHrLqZKPe2fye1rQMmfPh9fy1zGZmtd\\\\\\\/HUvvjWRr2viVt5Whe\\\\\\\/1C8o9XfhY5oPGHzl0H\\\\\\\/tu34Sgk4RK95F\\\\\\\/BZ5wrtdX5q8LL353J9BvBPBcG8IcJ87dBmk+UeqOOhPuJAHOZOAW1v5QxzGQzQdg92iyoFzlo95Z+zRoR0lBMo160G75oXLyKUskIrpyPhTNXOImsozvnVX8sc8ikICZ6Fg\\\\\\\/wddPPrnDSqw01YzI1D4iX0\\\\\\\/ysyc7vVvmODKMkTRillN4wDqqRbvB76b8GSGpc6D27CZzd2d60Kpjd4uyn3VoHumIE3Fh+65h16yvHMfJLrXsNrd9gVwdHLF6g3U9bMRcNOitR4GjoRfbH9LTe6gcvRtcNB8Nc3mmmM9M+ZovCY3Hrt+IrpRu4r13fLTkcynY57Zu2IbOBLtmrnXFZ7o7b2kVLpZ7wNb8vbMZRQYX7m6NUDhLFcC3sjJipcUB1SZ4Gjehcd0Z0Cp+JxARUeCtQfeeROvkRH+I02k70WFRWErPXUVQm82fVVpc7OV2zWZise0x98VVAIO5J\\\\\\\/FsdI1lUSS1mBhM692cGoNuMf67b+lo3volbeQaFv4KZR0KRIYS314y7N2eTxAL+S+7nnGBfiFWqwNvxfMfoUGXbfbPJaf7QU1qwpM3DNyhkJOWwj3+7N22+Z3dJ2cGaaFtewEnCH7AEPh9BM5w5I7hyc8BNm1x2Uh3kVc41J9X3eTl2t4htR2YUpHQ9IvG86vG+qhaNiQvuSQqOkpnUvtzwdwkoKXGNvO4Dm0ONCjWuspUwNXDqACSqvb9VVc8BzXKgn9OMkZXiutyI+mFDM\\\\\\\/kK13x7tjYopT+U6rMZsQmV+7ShEtuQ7YoihoGeSuQFpnymtsJGmSC66B3sKflQFJMQ7benOTGbWxN+J2Ovrs9fPJXtQi\\\\\\\/Cm0hKJ9KCsNbP7djLmBcyNMcnJw5FRp1N6JQjMLGFfUIV8byBdqFekRCmCQ1bKDQ9a2O2DkPj+hwxWqUgTQ1eiGbs522taUC\\\\\\\/mX+nLcjC7mzi3U1tRDPLcY2pRRyBf8V9mC9pafP14lBZF24IzLA+ZEOd98ZHyEPHwwqMK0R4xr1xtoi3NGeWxj+j4W5YAiFN2P5owDE\\\\\\\/cEE65i0ZMPhAUuUlGrc\\\\\\\/v6rmPcRtrxszQ33bg+qAW3km7LzvdWH0hrbY9qPoWtsVgI+Ue913hSx0vtTlLmI8vefnwkONJKCunPL9UH5JU38FYTFAGrHsYXxmP4ZVpADFgARjURfSr\\\\\\\/9UKBLp810HvhNoyr2jCaGf0bgFXpQcsa6kashlyb3\\\\\\\/ekHNFphMnt4x4GGsG1WWMf7C5sSOd00RYI8T8JdzubGUQkM5nuGN8hD5KPID4vODZMVl+9tY5k7VqNszXKNxgVZfyxds3xf\\\\\\\/GIe153nPB6AZAWIG9HSkSSkvVzRvGjO8oXBeTzCcOvHfytFJz\\\\\\\/ZpnEonzalXlg5SY4FUB5IRBqtEzshjz0rE2aqiiHedHxRfefqtWxd6KsHfRHet9\\\\\\\/ZmwOlz9sgEoaY59lpb\\\\\\\/wtUdE6bUQZZqz5+phNayVfYxcfkHkLKeAMZlDr+QDEOEjQhruJ6jS5Kb2zJu7CpcJbNOkq8P9D+39+OK5HvUA7p\\\\\\\/5YhZmFMfO6JcfkiV9cEx+CUssjFL1WAYw5hAI74MEwBOPBLqWnSFCYhO+SeI0FerobnrytR5S4L8EJdY3K5CkAVUbU+n5GOFkptHRbgABWa1qgv9hmRg9zJLPmUTf2JvcAwneKf\\\\\\\/\\\\\\\/GJLXVB08yCWumLHLuXTckfdc0WdyMM+1U+SXkWCDnlBUmMXiFBy1n\\\\\\\/xzj8menOSbS+NCiT+kZzmeyADfz0M+D7RFN80\\\\\\\/X87wOyfUGojvV5DpaBrop5remAnKWflyj5wTO\\\\\\\/Cdl+hKt3WoabdUZ+ugU5H1yB3z8+eQFL2N+VlC+VlWTYdnDbNQOhn3++76RUzGsu4fMdpOmuf1TlMKzISN+PkmJ2l9iKVNssgblhFBJ1lakUj8rhdPaX4fsUnFjpHC3uxxoImBnJXx4DoQF3\\\\\\\/9Xwqk1pLDgi3cLB5OjqXlXyIccShqPGDcQVePUSB5atWqA4zhSgafArknedZ9vtXEeNHOvIm0bsLt1INL3\\\\\\\/qhVm5qpdG23G7Ztzb9f+NBj++\\\\\\\/SIXr3ZQpuWbMnGqGVaPiHQKOwt+IPdwOBhC0XZy4udy\\\\\\\/Ls8EHi3Qad9taFD9lgSOfva3fVZCZs8\\\\\\\/sX0+aqfLPWQfJxR0VeZ2dR53g5AsCwdSuPfeM6HLFP5IEO7FloKI3HKd7GS5KeT+8R03uM4xt\\\\\\\/AVyJA4XT3gZeClvOf5PALjWbE2Tf6FrMAiH1dUw8onCilbdo7vAcEDeHGsKVFXHmkhoOz3eZcRlQN9Q\\\\\\\/8Xv3blL2BZC4utbtyKJQQn2ihJY+LSSU\\\\\\\/cUZ2KqXBLUlZz\\\\\\\/nl2KD+MduqcjfsWFwRCyodS2uAuxggG3HYCQgmRdQz95Fc17wMWExckzHeaPx\\\\\\\/9OiCBQaHBDqlEJtFk8rWGxeUcjg21Su\\\\\\\/OSFC1Qv4wxMc9JLt0Wr+I5RkIWRzSYyh1yK4\\\\\\\/QkoLFZ8R+fhBvGBsVo3mMxpWCt7P8S40dxBcsqt\\\\\\\/p01zhXd\\\\\\\/loCfZuylsAWxPMvBk5DBmTSUHlhKNLjkfUtZdkG7xbKnDzHaPHP99NsPFX0x1f0uBkQHN6A5h9J2DZp03majMc+Fp0Vf6Gn3NrD3cvQY7azc49pRdZ99D1uOab0DE1sEPVYvgoW96ev5UPAgD\\\\\\\/99FBDodB\\\\\\\/eFtcPEEwK9Ow5iAVySiqut1MRb1+ZreudCgDvV3xT9BCCFOYwAigHAozr2VRqV1H6TsmWom4B5HY37Ofw6mPRQkuEyTQt5K965xZ9U9OC63qm82\\\\\\\/V2xVfuWz2jTZOdW2aJlS6Sd1Hs20RqSGEvnnqYfvBF3NL1lMl65o\\\\\\\/ucB2mWBH05FxZESxeUTCPXoRwDox72IMCsqlRNeCqtH6qYo\\\\\\\/WlJ4Zfo4ycdy9LvVrbgy1mRYU5cH6IhJhzWiTeH+SMElpENn9F5v5MYaQi5Qfpxc5EjDbN0lGMFgRr1Wi3mSj9bgnjcW\\\\\\\/ab8SlgJmeUxgSIBHB7RMxAaK6CS\\\\\\\/tIfoiLtSvvqPbH\\\\\\\/UuxuquFzf5qNmLwQjk45XcW\\\\\\\/UC+W50b7TduyLHHbcji1lUMp9XYFgjbrN3\\\\\\\/jjE3YdSR+8dZYDScOrgqF53\\\\\\\/bfBAvzsDXmdzjfw6\\\\\\\/TmKN3WRn4MMjGjXbW5qH2Y0Tklbs9uZMeADVf4NDDyGTHsQB1lKeaCqZCchxiNdEDYB86Few395p66b1YmRda7WHkVdFrqIJx7b\\\\\\\/HwfL6Le1V+JpQ0bKCGfRqWRNSFZw6YKEqpZw3VBR6Z3J0RA+SCTf7R2xWWv6o18Wexkwb0zFDyvrPbvmqnzEuutttelIFQvNLoZ4Qit2AcspmVtP\\\\\\\/1s4dYJlE2gtA8aKux0p45u91iYYlDC\\\\\\\/koCI8XZtPh8PCrEIsgyDNGi9LP2C8qLtQ4PVFyu8qjo3S9dcv9CVfnpzYL+Jvduv62PHpz+8kbIGDEqT6vOchIqeV4nlP\\\\\\\/swxQqf9HXwbgTqZV6XgdDUBNIzJ9Su31tydIxDnccp5U7HMPvKaKS9qz8b7AT+OcN71VHAxDSAAz0GY2XzzyMxg0cHb8pQJuZJ7Kp3srPTTOoyyBqfxKnKTQPm6ZEJInqX6mJi1ZqpqxkfYgVHumGQ2gWxC78lgdhpnk4o0FNIxARjHRqviBEE5kpA4ykyD1KQHcWLbulE8s6LJyhyKZhGn6P2RqlS+1buB\\\\\\\/9wqQGRcIFawXlf012PoRID2ITqIlLKaIg1m4XxopcNfGM8bH8KHTHkdr0waYDLBGuQD0zpBCZ0n03BgsJd4rmW5ATsXOiYPaovoOUvgGw59iUHJF6npDRQ3Z2sTHJKYz5mHtKjUy61ccpQKShyzNyMEL20NYI1CcixZ3SInuvRSLZSI8PHdAcl9uu\\\\\\\/K0tsH+kkKwWSNIAKLXlJXQJ5YzQw6WMYlqsYDkJ0Py67rAUFWf2fu5VqrU2hHomuguZTvCzcTf40N2NUl4K\\\\\\\/SeZHafmdW2D4JpXHHElbmDqmmaGt0K6HK+5FN0sZ4uUyxziSfBbxf13GCoiGTDkRysYilZ3L7WmUuhKjMWwo2kpi095m4Ya3N7cWgDu47s4ly\\\\\\\/htrBW1ZrmKnWH6mMpFxYS2Lrkf\\\\\\\/YeesHzmZWOen4Hk0AobZT7aGma2zOeZXCjeDcQxXblxkZg\\\\\\\/ylS01Z6yCZI1rPh1KIM\\\\\\\/DstL59OguEbLARngfpQtJ9HM9IA0KZV3UdYpZfuu7t0bOuLvcRZhMJ8WKq0ydJmdjw0PvozSgSAZk4wRjQ2Cd4dpOPy8yF2KL2TgVjSTEF2JP0K\\\\\\\/JjNyk9gKpbVKGGkNb8pFq43EGr7WuuKagRA28oxt4hX8di5eLw9aZvapg4rwytjKS0rpyTvBv1a0wT\\\\\\\/4xMCnGlYp1fXLsQVCy+S6Kp85ucK9hBexjjJwdZlKiEag8DR3j7B+Bz5KFXP2f3gOc1KH4t\\\\\\\/7\\\\\\\/MGDqlP9Y61GP5nVx32nUU8o2GVl5S0RRI8WSWDsq94PYInzqUQz4a22YVCxa4MF0nHQFmFnRKbrwLMLTxPTf6d9MHxnwJIIYM0bPxE1l8kwd5TzQKOr71QGQMhnF\\\\\\\/AMNfqWcwuCQqRAyvbNvspr3JcF6A0BH4Z14rz\\\\\\\/q6TMTdYCj3Y5BGiubwftu+NBN+bcTMMO+E1yPUzSsO5ggQ++hj21T91Szj3IJSJx\\\\\\\/g4N85BG3bsDVLWDVOnNjfXZM+el\\\\\\\/wWYZL5NAizvAw49yNyUUGUgg2kG+6w19C\\\\\\\/PzJh7aCQbOYLWbyrDvpTB2QW\\\\\\\/NuaH5GPexby+YI7SlzfJ9gnY44j2EZDZg+LZHxbGNleBYkwIQLwdAGfwAlBkXYEgZUC6bwQfmLpnzVD5Zz4F9d95eLLbgQJkg+Ygh+2yMFces3LWgzLVcUJ48pSfqCvzR7pwerD19keYGFyD98jGeUYfJd3hjN0vRg\\\\\\\/d3vjJLMgg\\\\\\\/rUc7yZgzQntSUEWLFklAUVA3xSAcHbmdgOVqR7GdbROsLrNFXJvipfB6fSdIOJ3EC4FxtrvdzhSwN5vGptBqmqtzYSaWDqKsh6klpgxY4Yvt9pfy6uVlzOWO3l\\\\\\\/bCA\\\\\\\/ZgJPr8yASC9RP6nsaf3AKyvkgHS99URHCHyojPZ6cn6FsNcyFQsB+mMJEnBXaxxvtgazRUEO4IxFFsp0UznGIJcf6r6y7v0LmYZC8d8o2Dg7lRq7NBBEseYPIsemvwV+FObFEZzXMUgsB\\\\\\\/8hjMFmXg4jOh1hdwkW2AH1S7aZbce3LX4Y1JaCLzEcdQtv1xwKRG+XM70nafOjafh0EJsSQF\\\\\\\/4j2VSpcQB8SZbHlqN8RNvJ4dZgizCQly0GGqOrnnhZ6P47w\\\\\\\/pPpeMpm0pqzIeyNnxGIXP9i0Nbv+F6wVpOFrJmNz6gChPixf9Qk3uyEz2PJNGWsg8qtjG7OrFlPo+omSEUu0ZssB0JirIFjfCJTQZJGcfkovJjDK0cDHb7YwnGN11SKvTECGqxgsKODSpm2l4FDkrk05ssZbt5+W+o+4i\\\\\\\/+L0q3hf2x\\\\\\\/3ydq23AYqec2ZQSoP6UjKqbyIDgqyQ4CuQiL8a7fw8vG4WjPmvCma2u2Leg+i6ysAEhJYdjwb4URzMq258LZFPmchFU\\\\\\\/CW3BypFNlf28sAUV02KyHxe0wc1C+JZCgXsVR1jK86MzZmi+BvPCaSNrQHh1jT46hhx1jexvdJUjhOHlnR+PJIKyjPgf6qM1D+IVb1K2S82V2NfBTMnFompt0PP\\\\\\\/PFmNIl4GaIeM3\\\\\\\/Z8rFaeiTmHFmulacMnCHPnr0HVNvKZPQuvGw6y6nU8ZQhqnvoTdUv94ahB9s\\\\\\\/u3bP9z6dcpjtjMv0nuYCbRcHkMW4D5D9+7AxZvCrZcp3IxkFNbupReO02ARQE5P4tBR+nJYYsGt61GeKqWr8C1DglAl5YnSraONBceM\\\\\\\/DtbKpuVIlweWZ9Ez9SAvN8FSgdG5xZiEvxURAwEbTCXdzJpkMEEyoEkaXVvxvWMVFkxrxl6SG6aQJJd0JhmRhyWjMk0q3D\\\\\\\/kOJJsZML+R2XNekP5vRbIwn5oZZFfoDFvqQzraZ90+JdvG4vHUJhAfh3PFDY03e4me3Y4atHBtg9Fah5vdGBIUYEWXJTqDTk6n1GpAlaydqT2iFRCT9oDigXPO61MJoh9JeEeVfKoi689vUeUAnLz8+Bu\\\\\\\/\\\\\\\/opBeaS\\\\\\\/+5KaHVqsqRm\\\\\\\/3mDqj\\\\\\\/\\\\\\\/jR8GWRDlcyCSIBRyWBTGkcRPYlu30EdmXSH0h5t928gQm9\\\\\\\/kHidYgH9iL33FMcL\\\\\\\/luvunXSlzV2UKCmrJ2DcvCDDUF3K0NXeh21vNBRqjkVFWjyXX9DAmHI3TN8IlblTbkPKvSA52muB9lXgfkx0cWQxyS14ZyoGYz8PfLjBEdTxTjNEm9RV2nbYuO9SyzKF07JQfhuHPj76lmaV73zHG8PsfxIHSV8VB3I+umXBSCfhT8OZHiqNwK1o79f\\\\\\\/ZZAKNDvz5b+aRGVdp2XMuS+OCD0GO1ec71jm\\\\\\\/Ha7mUCIKe6GTiG96wCoQTDbjpW+ovOvdoJdTFxi3NrUBCEQGvNDUBBmnpk4w2RAMpT31HjAGa853E+B95Gj7csdEk8L3+unZONj+OQSNwHKx6FBoKbGhmgSsEr2GXcjcSHPy6Tdn9o0lLxhB9xEBYtIrLStaCXWvrweWeHPq83mfEvHjpyZjBzGtWmk+YEznkX30gPsasz2SuHhc3VIJ7vxV3Fk6zh5PjjwznR1Uyv7PfnyGNh++IZuPoBIFtejwdETQN0E2RqHNiLVl19rlw96QZPYXjmtJYVpoTfb5\\\\\\\/yzMwuKdvV5VPQ7oYzWw\\\\\\\/T7qHYvu4w2NXrR5\\\\\\\/EYSGUoYBzKTDTfVCTpQ9Qcwmtl9wmjeBnEEU1ea0blVsVsPfAm4tIa3MmrklDj9y72lZUGcKhw+HlZWCGQbO3LtfaH+AdEyY5XizWXG5rjQdKf7kRWP5xx1+sPutTr+vhJyX6cBOBe\\\\\\\/\\\\\\\/MJknRbBtLqgF5wZZFi7O+5fwPeJ7Uz9E\\\\\\\/Y6JHwL5oPsu05EYzGh6TlgaESjkYan3s52fRkRFo\\\\\\\/cn6qxDuuA+rh8Tm4pwuFbS2M4Lf0dYR8ObH\\\\\\\/EwlkfvdQnXmm0+BZ2aNaPkeFcZ5tY3LmnaEjMwtEdGFxEpabCjGJOI2A0gyRMZdamZgHQDZxcqSXctHdIi8zpWFzKeZHYkiJQdGxrnjckIofYfNI2D1IXC+ycsaEn56avSbfS0I0f37\\\\\\\/Uwa6rIjPKtdTbBsHIIIa7J7aFx9aQ1lJeDxdYtWtwL\\\\\\\/0rbhgSaZy0aEKwA6Jw+4Xw220bveBbjUjbjS\\\\\\\/h98\\\\\\\/O9hjmFTDh9p8e+dLmCef4\\\\\\\/fPWnPRNW0j6Hp64McHkF2euE84fLVCSbGBZt6zMt7VHZGkhnGUlgLpCv65qtIM5kvualL6rulWRP3NEdhRZMP8XfjEX88XeWo8\\\\\\\/6mF4QAr2QadJrQqr1vYMqEFfzSEFBfOn6B5rEDTVgWcBjX\\\\\\\/kT+p8Q0T2TBujo+9IPQcyKemeumqeE32B36GQq\\\\\\\/KDvDNXNtWFTA7PRei7fI+VK9wSDy2PtaR5UTOWOI0bwdVqwEio\\\\\\\/DAHKvdIn8ehwNm29rt0DqrSnn\\\\\\\/HzeQbc9\\\\\\\/mB4vhqGVuAlR8VoY7twSfoReToFqYUFzrbXIxRUPMPhWvuZZliwTgfWbS7IpfgsWQoDGH7bejsjgbUF7tsgy\\\\\\\/fMs9\\\\\\\/4LFHms8fcmha3p2bwBMlwjU\\\\\\\/EEahkyCLdx6jJ1pR0CR0bssyUIMeg1buKYZCyf4zYXESNe\\\\\\\/h9CLqeGtziUTBwSp\\\\\\\/sF\\\\\\\/QXLj4H1iu2LJLMfN8Kn5\\\\\\\/mQ+KON3YTKgbZifXHbCCUNIUKYU11FcqfmLkVh2bNiL9zsIQtdl542AWYY0Ii9eeqdbgL4ei6JpKM9rzONit+lBAxZ5MGvUwXgc5AUc\\\\\\\/y1b7DQkdXo+u\\\\\\\/6uD5tZHHih3IN6aTn\\\\\\\/Lf\\\\\\\/XTDrkg2ltQCuz9RKZj9FGfNxr7cneRQEGbXbPF31rLWluFcYrWbkLyM9LdogEmLYNITGNULxKAoPkzrbUKrA1g3bprEinMAUd8qbXJwEG8ynaVSOB\\\\\\\/VoXMkP0WWjxKwmsJzBcgZgmS9MaINtiWVeQI3xf4qWsHm7\\\\\\\/TzTabZiMMUmibdZQtq\\\\\\\/6PycRWsQAy9SECDo2x8kq5wXFWoj349OAkeApciq0Tlhb4mVwXwWoro0+Ec1W5gUGneHh4JFmAOXgxfSqwN2HzeLC9g+hZtZALT4xeyqFBp3ILQwgFDKQdm2rwdP9Sk+owtibkLgvnG8GpUPM3Y+wZP9TxzTx8TaH5WBfoOVP7kevSp+4gVp164xqZjM3z1GL0UuJ+XPGG\\\\\\\/J2SctuGeCwspE2EQLFBCiOwZzvDOFvAXzzQ9SjZe12uyriOeryoXLilOin32vVY+qh\\\\\\\/P2MXV9TqH\\\\\\\/Jq7nB8kCoLg52xQf0gly6hRmAaJR+8YdlGxio9A2OHiTK+C10eACwVchttEyTr65pgjRkDrwPwtvowBp+0iX4JQtWnbm2q\\\\\\\/LZxlOvgDGI\\\\\\\/I+VGWELIdZkndyFxRXV8zTSOLciccmwpxjdRy1siMz55ZGbkeetVaeuwQJAtJd3FLS5B7iPMbdeaibBYYgtvortB0FI8oWp9hAtKgD1U+xgBovbV4iXdgF57ZjEybTINfXjDe+PW18xXi60ib414gpnmPBenziZFm4uVmmE9KnvTzkig9yf+xN\\\\\\\/9VzecjfmFrX\\\\\\\/6qRQe9AsUiwFFJabZq97W45bdJY6MPqdFFnptCgShMXnNJruqQsPc5SZgFNh7mjbg2YJQj02wes968bLyeH\\\\\\\/PdzkazCxUHylrbfO5Ou8MBCQowNvmI0bRU5IIDkW9tpMXnWAjqdpd41TtE4R10yVeM3ruPBzoC+5y\\\\\\\/B2Iy0nh6RDjuXryX92y5p4VXFXUVoo2DIXe3Sst1jI4il8Mae3F7pSloMQCSSptQXq6jpKAzqUOuPhnWtnrzPKsQuDPILi2MLDoOyTd321jrEl6Ihu\\\\\\\/8SD8VeLuqaTpw0lI1\\\\\\\/mRxdyndyihJnrQvfbAZ0IQ8\\\\\\\/TDvPb2nORE9BbmL\\\\\\\/G9R6MXJh+Trm08O189Rk\\\\\\\/LFgTAi0wKRkEOVpFVFmxZoQw91CGYRV3oB5lvblmjuobfdFvNxSAMzorF5qPMkX0McZpQ1lk6eu+o8JVCjHzIBID7a9bnNz2mKtngFuqi3NzopJB5mkJmhLmnTFfbrJgSbbpaGnH8NNwcI00w9S\\\\\\\/PQH9rujXtlsQ+07ic344U4R46zFvLFz5O6T4HXQ5jWDYJp8I+gzFIT4N9\\\\\\\/inrILWpqVMdmsKbCS\\\\\\\/o4r18V\\\\\\\/wvAnXsemUa1VpAc1QdE++96gnixdAY7JStjhGE85SBXPSGJPQYIJziT5NFC1PrdBLVlxTjGJQilJCN2eUXGffRTHcxKAoBvR55CYFz1BRrQ00Di1pn\\\\\\\/wPnsgA7DBnAUjs\\\\\\\/E5mcT31bjKdCU+SfGfDYG+WN\\\\\\\/dJg0PgzF1DK4sOJMfIrSZAHOT8\\\\\\\/hL5woqsDfSxTpXXZHIaL9EoHCPOZiUNGEOYMcV3QzmTZr05k1mlC3bCEyd\\\\\\\/eRvPZxUww9U9MHLoFehD16D8MvoaN0eBL2Nq+ssnTVNMFzTXx69H8CLiwQvOgYwtE\\\\\\\/V31Sxc8E3nTgNIfiSr8Szq0iZLOyTRwYW65qZl6dfBtE1ffGNZ2zPOhs3Qk6Fkot76x34bPJ+TD4kfDzVWQ5RgVtNc+zGMnqM+3K2rmRfesYVTp8ZpJk4ImQAqjkKYTR5S\\\\\\\/IEmSwAy0SLQY5nb6Y+3YKNamiUzemkf1BaK7aYdPjyZgWSV43z575ddgi7bSslA0Lzj1tK0+ayT6W2BCF5TzEAzVaC8O3UUi3Yd+B9oMrPatSEY9uP9MpQmeFMjCzPXL58Nm6FYY25bQ8KfB+HWHifeJ7Efoqlb9PbJ+HjYN7f6p38svCf9fCk6PSLxcenRMTYwG4qhGX5yRv206\\\\\\\/EZ07B\\\\\\\/UlJNCGdl1GxzVTiiUAfOd2pBfog6Dra7oEyG7n6l1I4jCXscx7HYSfm9Z+kv6AGBScbA5o6NlCv1AJBkQoe0TEMkCN2PfhAFVOZV\\\\\\\/uNLYkWUQwcPsHSa6x5vlMyxWJdVixdRW20R8MPhhQrWlZYk0Ec\\\\\\\/ESbR\\\\\\\/ZuyFjizYJMtQy9htb5n8TXnl6T\\\\\\\/d+mFiHtlOlBAnJQ+7ZAbJHjO1qnyBadeTi++zsgwST764SHKp2s64VakuOFvHbttvwcTU6TJemg2JiYWW9RvgKYqBTHFcqdW\\\\\\\/IrDSUmYdPx6MOjbm9TpSRZdI\\\\\\\/i5vlaon8Tp5SX1ND46RnV9D4WW\\\\\\\/xmmUW8yBs6lzu0WPB4sk5eII7Vhbn3xyInaTaPS481P3Na8yrNSR6yp2MxWQrKXzxBoaN+UywBrX8k1p1spb8f9vJi160TDfE8BsbHA\\\\\\\/4Rng2asf8QU0TeGV2Qh5kULYnUsNRava7WxAiFVHJlKQix\\\\\\\/SUo0JnAw2aIljtvdo\\\\\\\/th8qxAQC2bWR0uJDOOpBAystVUUSZOLlsFNpPd4YkJvhKAnczVpmMrJbL90jqGrONfWKlQ5k08QZqtWnAdonhwYu6MfB3EDk+ys1FHKdhANvM6TraAPwvwkBN97OWcalsCrQ4Nr4PhEGjt5Wt+OaXeP8lIm1R6tCJ0oUyksCZqHhRZyeAwJ5zh0+f+nzbc8AmpbdwSZIDSd8KwY0UhqV+pO0bkGy3aK1MYQkBUTPQvHM2pkV7OCbsxixCcpyyGg7Aqf7jptDkrUionbHfnkIP3Qr6AnOf05JJGA1dnYB7+yGnF0RQgRR78pqyH7wE32y6Hu2nuVUPpN6FxSAMdwyB8Tiz7pU89KT5LbL\\\\\\\/0T5Unspa\\\\\\\/k9ejGGBuGCGiev\\\\\\\/7iF6fqq3v0nhs80qs9nakG\\\\\\\/pAncIVGa40TuOmTizMfyOv4C0bypyNxH3PeIl0Omf7D+hUOx6\\\\\\\/jdxj+KA6bqQ\\\\\\\/9WtZXdfWnwQBFTJTdfV5+ezmSU6C8bzVoh4SDnCS0ruBjGtC91P2Dt9Q5FXo\\\\\\\/wqAalhcpVcBL449KCRS4AJfbeqFpt2KSIrMV9Sr+sD0LHQNEaQ8xHfu087lDnMaXj4HkcPNNJlfyweu15pbS1Wx3svqO0\\\\\\\/Wxv9AggWIXRBy7y3KImdcTYCoJ+remMU9KgTgCRAuXjoMIlrPc8REtyv5HbMSzPJIrB0Z5RQQZviKi5+fP8HIg7Xh7sHA1OAicVCF8SMWn7p0LcxBZCZotQhvQaDYfuYWerpoR1kEZGUCead25Nr0RM7QQcLuKl8wbIew77oEga8T\\\\\\\/P5+1+vLi83FrYtcKJCJoACGb+X4nPrnKtAo+QCzyeIms+qNWbbYnIBjuQxDjm34hyT\\\\\\\/MqNv+ticaizlicdYOpLSHC17oGIGI3NVCAbNiMo6UcNkCacG+Ibu2P7rhHk9nwzXh\\\\\\\/3V3eJu\\\\\\\/K5W3lJ\\\\\\\/zb5ufB7RnqeSBUko35YhOMzQFavZa15C9U5xSXtCWSZUAO3Pb9Egl5mmX47+F7dgkg3v4s3vM2NmvAPOZLqhoAwThszxbNCRIciXC+RN0YJZTnZVdsQgbh5iivjD3UOQGHDFuVuHkHJ2r86EbUnJZ\\\\\\\/PHplH2vE894\\\\\\\/Zuq9d39Z4QCbr3BTo4TMRTknjJtVXkwl+xir\\\\\\\/Nh6\\\\\\\/WBwUt3hRlID1jvV\\\\\\\/2u5bMhfAQI065y0R9zqp49s4vJLD3jEbY3dojBQW4SUZIr+kl5oLvkV4Ttxbl\\\\\\\/xNZyMncOwur6I6+1CrPcIxOTtSMCYdAr6vUfzBeBBCHFM\\\\\\\/caVapcLs5IAI\\\\\\\/OKsSvgG6vfMHseIt9q\\\\\\\/0s0Zt6YlgDO5mSFqBYico9WM\\\\\\\/59TgSXgV9mmNGoxAo8OB2uo8JXAETe5EEGnJfduZezpMZW6+iXjtu5skyEb306Qf1p+yxtkVxQrgUczQBHB07g5Pmh9\\\\\\\/hOIbt1YgERHk0q4bBnyn4WerPee+kC4Gx4dNv9g\\\\\\\/GU1Mz53d7utbQvBfrzpicUGcU59IQUjlIuEZ\\\\\\\/7NafJBW6CuukNNMKjcRFOHze+aBfVySp2OEd2boR8d55c2iLCVRitRQghSNvIwCy6ELPegPP1Ld+ng5LCuTd2Ms0FVoFURA7SWNcYloDFEUQuODw+dkQQTb7+tu5y5nnup0mbEusa3VJwfoPqEUNxnHcB73gZQx\\\\\\\/xZY6VxnkCEMtAfA5gB9go+eaE43MITy+z67JR3iL6HRDHqo05+dW0MfLTfvFIQMlfPaHjeUqDpajP+xBLwZAudr\\\\\\\/P9n5q1gqyFdxarf4+O1X0+GsyGhTOQGI\\\\\\\/1kY20dYbSAMujnH\\\\\\\/szoqrjqWyXFViw5XCkAnQDIwPg3ZjxIOdDDu4J9tAow4NLT7UeQjhtoBEQ24yujxM6KL2JXS9XrwDIz1iwFT8jd9lTHnMptgMahJlYdZObr20zYkr0qPKd2GvAzVI7JcpfT9St\\\\\\\/zsctPTvz\\\\\\\/7mfOtw4+nvq1jpTJziOk69f2k5H3cS8SPSGBTxsoOsuC+gh+DipKpfXCLkGqwi8\\\\\\\/lWHG6\\\\\\\/nvY7j+yfMV2L6F3j\\\\\\\/D2fdpVUuFJj\\\\\\\/F2av3SK7n2qfwKG6z34b7PWLLrbrMK3FeBusvtrTKuLCSRGWGcDqaBJq9oWCWgkX8JFKc7U0fSnSS3l9hjHp92BwC6QMONUkgmDRboQvEYR4h+ZSkjaKCw+WhrWiBCbDhna1in86YTaVzBoIAqHGTfGiGllW+L1D+hYLzPyaFwEWWc0JTV+uCdcyz9opdi5sRbxTQeSMW4\\\\\\\/8KirM49\\\\\\\/MfKhhUsRg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022aVR9jM9zmKg9Xiqa4v8mEw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022170ce7a2668640e4247f4ab1b305e3c7aab1787fad1584029db81cecbccfc9fb\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00225ksmLo5Y4mK6WcwYWH55Dkt4bw9ftT4toL+tYlrxJAl057b\\\\\\\/g+xzCyxyft5WfFrAQWtWO78AVnLW4bQ8hLQcYuvAt07oyjTSvjPhpTf1qfUMGSgX5dePFRbwy58ICWycdm8TZpiHL6iNfSeOSPHRf8t7sVHPGzNlsS2j+qcqKibH+fIuvr\\\\\\\/2B+4MlgK8\\\\\\\/vXhCmHLbMKjhtmApXSlzGC9wmkJ8wlZLURmFQm3wFgdsXSXayT5pS7RZ+Hyya0ltpxaKZ4W3haXKxTwkfdlVz\\\\\\\/xhk8ckeFF04C9tWgWfWzVVg3JDLwfsbt4Do0a+wicne3zUeymNFfkYgV1yXz1rf96eGPMW3cCVnpDGp5OuknZ6cRF7wXaoXec9C3o6NSlXWNY+\\\\\\\/FwgVzlGuG4qn970D8rlVtAtES9f12lfwvKRurMt3pFcBIy0o5qSpQ\\\\\\\/xL1wZTOm6HKFmxUFOwEXGL55N46Gfd3imc7mzjsBN76NHqpsiyad4wKg74sBShFqwZFZniRxauE1F8EYgFTLY24wjJJuUr3quDnlinynTUyywvNeV9EFi8ZkMx1cXyqI9s8bvYkdtf7W+oKHC41bspkBgU\\\\\\\/Ng4zoerJifXSL7dRwBnwAQ1\\\\\\\/FlaVhM+xBEEQt4q9sM2ucFRYPlEBs0MLlvbrLdAwE82+itlzNVRehxCAZxk\\\\\\\/hB9be1++h2804tXulk2ttfW1vJwWLvU95UngwNoEva2c1BXAgYfUFzDawN4oAfisw7zd4dKNIhdjFdhJu9rtaBUnGKHmnRPJ1cqvjoAM8acIN+Q7p\\\\\\\/M\\\\\\\/7edH0q+463xJXPJSNUvuWCLEvcEh\\\\\\\/ALlsVNKMNTx7m5KDVTQnRg0UPRbgjxRfpuoERfv9ofrgUMdpCITrB3M2N5fFbnO54Zd9lN8klfKWBzQOYKGZ66sWw+FPS6f9n6ujWMr8Kqd58ekJ1JpnZqAB6T2WzqhueWm\\\\\\\/Yn2xPGrmDWmocjUNdUeAs4g1YgpCCmUWZ\\\\\\\/DlHs9tZXaUCQrfAkoupKhjfhGW5xQMvQXeBLcN6A4z9jKC2EKq83OpUXfh9R4Eq21OYdaOHmw9gYvzCXm85fp3o8rozG2JiCfSVmWeoGVw\\\\\\\/y8JyhdeM49v6zbbC4ZxnN\\\\\\\/rKWOxoSq1L08\\\\\\\/1TguJSsUfo9UdY2BPxRg9k8vJ+0yxzemb7TGdmaZDkPsmMvdJDDwjxDsEAmpOsYmYDsjrAHf+5pTWN6iy57nL3a4wGOIGX\\\\\\\/bNY9Ixbjbt0wg1wJlG4E8ezdgU1x5Fi0pXQySLIS576qgFcvEFbCFZYKfNMu6OXfgo78I5WE2YH+H2TG\\\\\\\/KR6zgH0Ix0vKFBLIjdYGWXOHdaDzUC3dymmBmzKaQ+bE2l0vwGfwgkNdBqSfmXAGTBxwzIQ+TNJnSHmA\\\\\\\/dA7ILeIDz5OofX\\\\\\\/FujWVE\\\\\\\/oiv\\\\\\\/4RT7RDVARC7SkiW7mAh8Xf\\\\\\\/0LVu7kikU87d74sZAgk6FSpHIF0NNe0p5\\\\\\\/XDROi7\\\\\\\/N9+89ZOq+Tcojmtg01Wk6QauFmWaLVm+LLiPSnwA4vRrM+ZNFw4hmMYVwJFp5hXIcXYtMsOMwJiu3XdIgbZFMHla+OlwqqQE8NQ07dQIQlH2TZX+m7BTlm0dGklr6h+x2XxO5by1ky0jzOkkNwSIiIR70deIWLk0GcMmXtkNF\\\\\\\/RzZww4lTDgJ2ufeSpfnw1igsiwRwZFVsekih7oXVh\\\\\\\/Z40UrW9U+hkxyL4sk7ucLTzw4L3SASCsOlh\\\\\\\/PQU2CbYs\\\\\\\/f8ZehTV3VVglTTRDgbORZ3SQ49336SJa9JPdm9SnpzsHuefTZGSFfxcwlF7mp+meKsCVGQDg+IRz6NIs4KKrbPiZP5ZFyaAegmig6XS\\\\\\\/I9xniYz6j\\\\\\\/l9Al7j12kB\\\\\\\/QYehFL0GY2+xqaVbKLZ2Noop5yzogn2pc5ireUAxX1ugnELaQgZ5il6Gikl4HnVbAV8ci9tuQUzjIIAJCCEK\\\\\\\/uokGifPecZUgDmAcEiLNKf19hN4GnvI4hT3aQqI+yxiVxtEeATgFU1\\\\\\\/S8Vig==\\\u0022,\\\u0022iv\\\u0022:\\\u0022cVbrcR59o0zOouNnxiOBkg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002213d77cb58dc88d89b68589be569d53b89774562c6c7cba7de2002a9885f0afbf\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459579 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/2085" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-OkLJILq0eAeteBYtpg7G70Myy90f08zrlN5vetvkakA" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/2085" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qrZ4Si61UmZftbbEsz9fADXmxH4HjeREUJWdFb_lur0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/2069" title="Allelic Variation in a Cellulose Synthase Gene (PtoCesA4) Associated with Growth and Wood Properties in Populus tomentosa" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/11/2095" title="A Genome-Wide Association Study Reveals Genes Associated with Fusarium Ear Rot Resistance in a Maize Core Diversity Panel" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459579" id="node459579" data-pisa="ggg;3/11/2085" data-pisa-master="ggg;g3.113.008417" data-apath="/ggg/3/11/2085.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Dissecting Genome-Wide Association Signals for Loss-of-Function Phenotypes in Sorghum Flavonoid Pigmentation Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Geoffrey P. Morris</span>, <span class="highwire-citation-author" data-delta="1">Davina H. Rhodes</span>, <span class="highwire-citation-author" data-delta="2">Zachary Brenton</span>, <span class="highwire-citation-author" data-delta="3">Punna Ramu</span>, <span class="highwire-citation-author" data-delta="4">Vinayan Madhumal Thayil</span>, <span class="highwire-citation-author" data-delta="5">Santosh Deshpande</span>, <span class="highwire-citation-author" data-delta="6">C. Thomas Hash</span>, <span class="highwire-citation-author" data-delta="7">Charlotte Acharya</span>, <span class="highwire-citation-author" data-delta="8">Sharon E. Mitchell</span>, <span class="highwire-citation-author" data-delta="9">Edward S. Buckler</span>, <span class="highwire-citation-author" data-delta="10">Jianming Yu</span> and <span class="highwire-citation-author" data-delta="11">Stephen Kresovich</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2085-2094;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008417 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459579" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Geoffrey P. Morris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Geoffrey%2BP.%2BMorris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morris%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGeoffrey%2BP.%2BMorris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gpmorris@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">gpmorris@ksu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Davina H. Rhodes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Davina%2BH.%2BRhodes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rhodes%20DH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavina%2BH.%2BRhodes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Zachary Brenton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zachary%2BBrenton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brenton%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZachary%2BBrenton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Punna Ramu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>ICRISAT, Patancheru PO, Hyderabad 502 324, Andhra Pradesh, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Punna%2BRamu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramu%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APunna%2BRamu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Vinayan Madhumal Thayil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>ICRISAT, Patancheru PO, Hyderabad 502 324, Andhra Pradesh, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vinayan%2BMadhumal%2BThayil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thayil%20VM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVinayan%2BMadhumal%2BThayil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Santosh Deshpande </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>ICRISAT, Patancheru PO, Hyderabad 502 324, Andhra Pradesh, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Santosh%2BDeshpande%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deshpande%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASantosh%2BDeshpande%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">C. Thomas Hash </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>ICRISAT-Sadoré, BP 12404, Niamey, Niger</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BThomas%2BHash%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hash%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC.%2BThomas%2BHash%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Charlotte Acharya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charlotte%2BAcharya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Acharya%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharlotte%2BAcharya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sharon E. Mitchell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sharon%2BE.%2BMitchell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitchell%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASharon%2BE.%2BMitchell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Edward S. Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>USDA-ARS, Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BS.%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BS.%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jianming Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Agronomy, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianming%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianming%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Stephen Kresovich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BKresovich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kresovich%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BKresovich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/11/2085" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459579" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459579/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/2085.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459579" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459579/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/2085.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459579" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459579/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/2085.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459579" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459579/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/11/2085.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459579" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459579/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-5">Genome-wide association studies are a powerful method to dissect the genetic basis of traits, although in practice the effects of complex genetic architecture and population structure remain poorly understood. To compare mapping strategies we dissected the genetic control of flavonoid pigmentation traits in the cereal grass sorghum by using high-resolution genotyping-by-sequencing single-nucleotide polymorphism markers. Studying the grain tannin trait, we find that general linear models (GLMs) are not able to precisely map <em>tan1-a</em>, a known loss-of-function allele of the <em>Tannin1</em> gene, with either a small panel (<em>n</em> = 142) or large association panel (<em>n</em> = 336), and that indirect associations limit the mapping of the <em>Tannin1</em> locus to Mb-resolution. A GLM that accounts for population structure (Q) or standard mixed linear model that accounts for kinship (K) can identify <em>tan1-a</em>, whereas a compressed mixed linear model performs worse than the naive GLM. Interestingly, a simple loss-of-function genome scan, for genotype-phenotype covariation only in the putative loss-of-function allele, is able to precisely identify the <em>Tannin1</em> gene without considering relatedness. We also find that the <em>tan1-a</em> allele can be mapped with gene resolution in a biparental recombinant inbred line family (<em>n</em> = 263) using genotyping-by-sequencing markers but lower precision in the mapping of vegetative pigmentation traits suggest that consistent gene-level resolution will likely require larger families or multiple recombinant inbred lines. These findings highlight that complex association signals can emerge from even the simplest traits given epistasis and structured alleles, but that gene-resolution mapping of these traits is possible with high marker density and appropriate models.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/quantitative-trait-loci" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait-loci" rel="nofollow">quantitative trait loci</a></li><li class="kwd"><a href="/keyword/null-alleles" class="hw-term hw-article-keyword hw-article-keyword-null-alleles" rel="nofollow">null alleles</a></li><li class="kwd"><a href="/keyword/structured-populations" class="hw-term hw-article-keyword hw-article-keyword-structured-populations" rel="nofollow">structured populations</a></li><li class="kwd"><a href="/keyword/genome-scan" class="hw-term hw-article-keyword hw-article-keyword-genome-scan" rel="nofollow">genome scan</a></li><li class="kwd"><a href="/keyword/grain-pigmentation" class="hw-term hw-article-keyword hw-article-keyword-grain-pigmentation" rel="nofollow">grain pigmentation</a></li></ul><p id="p-6">Genome-wide association studies (GWAS) have been used to dissect the genetic basis of complex traits in humans (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Rosenberg <em>et al.</em> 2010</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Visscher <em>et al.</em> 2012</a>), model systems (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Atwell <em>et al.</em> 2010</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Flint and Eskin 2012</a>), and crop species (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Beló <em>et al.</em> 2008</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Cockram <em>et al.</em> 2010</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Huang <em>et al.</em> 2010</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Morris <em>et al.</em> 2013</a>). By the use of natural populations instead of biparental families traditionally used in linkage mapping, GWAS offers a number of potential advantages: avoiding the need for crosses; capturing greater genetic diversity for the trait of interest; and providing greater mapping resolution through the use of historical recombination events. However, complex genetic architectures and structured traits can create spurious signals and indirect associations in GWAS, and the interactions between genetic architecture and population structure remain poorly understood. Complex genetic architecture can arise from allelic heterogeneity (multiple independent alleles at the same gene) or genetic heterogeneity (multiple genes controlling the trait), and epistasis (nonadditive interactions among multiple genes; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Platt <em>et al.</em> 2010</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Segura <em>et al.</em> 2012</a>). A simple, common form of epistasis is complementary dominance, in which functional alleles of two or more genes are required for the expression of the dominant phenotype, so loss-of-function alleles interact epistatically. When multiple functional alleles are present on the same haplotype block, synthetic associations with shared ancestral alleles may result in positively misleading GWAS signals (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Dickson <em>et al.</em> 2010</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Platt <em>et al.</em> 2010</a>). Complex genetic interactions are thought to underlie cases in which known causative alleles do not emerge as the most significant associations in GWAS, such as the case of <em>FRIGIDA</em> expression in Arabidopsis (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Atwell <em>et al.</em> 2010</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Segura <em>et al.</em> 2012</a>).</p><p id="p-7">Given the complexity of mapping traits in diverse populations, much effort has gone to developing and characterizing various statistical approaches to GWAS. It is well-known that the simplest GWAS methods that ignore population structure, such as Wilcoxon rank sum tests or general linear models (GLMs), will yield inflated association signals when used for structured traits (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Atwell <em>et al.</em> 2010</a>; <a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Huang <em>et al.</em> 2010</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Zhao <em>et al.</em> 2011</a>). Conversely, approaches that control for inflated signals by accounting for population structure [including the structured association (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Pritchard <em>et al.</em> 2000</a>) and mixed linear models (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Yu <em>et al.</em> 2006</a>)] can yield false-negative results when causal variants are structured (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bergelson and Roux 2010</a>). Given the trade-offs in the use of simple <em>vs.</em> structured models, further empirical studies of validated functional variants are needed to inform investigations of novel trait loci. Moreover, it has been argued that the impact and prevalence of synthetic associations will only be determined empirically (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Goldstein 2011</a>).</p><p id="p-8">To better characterize the interaction of genetic architecture and population structure, and compare methods for GWAS in structured populations, we investigated the genome-wide association signals of flavonoid pigmentation traits. Flavonoid pigmentation shows abundant natural variation in many plant species and therefore has been a classic empirical model in genetics (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Darwin 1859</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Nilsson-Ehle 1909</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sax 1923</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">McClintock 1950</a>; <a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Huang <em>et al.</em> 2010</a>). As such, the flavonoid pathway has been almost completely elucidated in Arabidopsis and maize, and many of the enzymes, regulators, and transporters underlying flavonoid traits are widely conserved across plant families (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Petroni and Tonelli 2011</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1/008417SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1/TableS1.pdf">Table S1</a>). Although the core components of the flavonoid network are relatively well-understood, there are a number of areas that remain to be elucidated, including the polymerization and transport of tannins (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Zhao <em>et al.</em> 2010</a>) and pathways responsible for lineage-specific end products involved in defense and environmental adaptation (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ibraheem <em>et al.</em> 2010</a>). Moreover, even when the genes underlying a trait are known, the characterization of functional allelic variation remains an important and challenging goal (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Tian <em>et al.</em> 2009</a>).</p><p id="p-9">In sorghum [<em>Sorghum bicolor</em> (L.) Moench] there is abundant natural variation for flavonoid pigmentation, which underlies a number of agronomic traits, such as grain mold (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Esele <em>et al.</em> 1993</a>) and anthracnose resistance (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Ibraheem <em>et al.</em> 2010</a>), and nutritional traits, such as digestibility (Kaufman <em>et al.</em> 2012) and anti-inflammatory properties (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Moraes <em>et al.</em> 2012</a>). The role of pigmentation in crop diversification and improvement is complex (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gross and Olsen 2010</a>), as exemplified by grain tannins, which provide defense against molding and bird predation but also impart bitterness and astringency (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Doggett 1988</a>). Classical inheritance and linkage studies have mapped loci controlling pigmentation of several sorghum tissues including the testa (inner seed coat; <em>B1</em> and <em>B2</em>), pericarp (outer seed coat; <em>R</em> and <em>Y</em>), coleoptile (seedling leaf sheath; <em>Rs1</em> and <em>Rs2</em>), and adult vegetative leaf and stem (<em>P</em> and <em>Q</em>) (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Vinall and Cron 1921</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Stephens 1946</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Doggett 1988</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Mace and Jordan 2010</a>). In addition, two flavonoid genes have been cloned in sorghum, which can be used to validate mapping approaches: a MYB transcription factor (<em>Y1</em>; <em>Yellow seed1</em>), which controls pericarp pigmentation and phytoalexin production (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Ibraheem <em>et al.</em> 2010</a>), and a WD40 regulator (<em>Tannin1</em>), which controls the presence of tannins in the testa (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 2012</a>). Although testa pigmentation segregates as a simple Mendelian trait, from the standpoint of GWAS it may present a complex genetic architecture because of multiple <em>tan1</em> loss-of-function alleles (<em>tan1-a</em> and <em>tan1-b</em>) and complementary dominance between at least two loss-of-function loci (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 2012</a>). Moreover, sorghum has complex population structure because of extensive ancient crop diffusion and a propensity for inbreeding, which presents a challenge for GWAS of agroclimatic traits (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Morris <em>et al.</em> 2013</a>). Here, we take advantage of high-resolution genotyping-by-sequencing single-nucleotide polymorphism (SNP) maps to compare the ability of several genome-wide mapping approaches to identify known flavonoid pigmentation loci, contrast linear model GWAS to a simple loss-of-function genome scan, and use GWAS to identify additional loci that may underlie flavonoid pigmentation in sorghum.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Flavonoid-related candidate genes</h3><p id="p-10">To identify potential components of the flavonoid network in sorghum and to define an <em>a priori</em> candidate gene set for comparison with mapping results, we conducted a systematic survey of flavonoid-related gene families in the reference sorghum genome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1/TableS1.pdf">Table S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1/FileS1.xlsx">File S1</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Paterson <em>et al.</em> 2009</a>). Because Arabidopsis is by far the best-understood model for the flavonoid network (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Winkel-Shirley 2001</a>; <a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Zhao <em>et al.</em> 2010</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Petroni and Tonelli 2011</a>), we defined the candidate gene set primarily based on the Arabidopsis flavonoid-related genes in TAIR (<a href="http://www.arabidopsis.org">www.arabidopsis.org</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lamesch <em>et al.</em> 2011</a>). Sorghum homologs of the reference genes were obtained from Phytozome (<a href="http://www.phytozome.org">www.phytozome.org</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Goodstein <em>et al.</em> 2011</a>). Note that because most flavonoid-related genes are conserved across diverse plant species, an Arabidopsis-based homology search captures the sorghum orthologs of many flavonoid-related genes in maize, rice, and other plant species (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Schnable and Freeling 2011</a>; <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Petroni and Tonelli 2011</a>), as well as the two cloned flavonoid genes in sorghum, <em>Yellow seed1</em> and <em>Tannin1</em> (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Ibraheem <em>et al.</em> 2010</a>; <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 2012</a>).</p></div><div id="sec-3" class="subsection"><h3>Genotyping-by-sequencing (GBS)</h3><p id="p-11">Genotypes for this study were generated with genotyping-by-sequencing (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Elshire <em>et al.</em> 2011</a>) using the GBS pipeline 3.0 in the TASSEL software package (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bradbury <em>et al.</em> 2007</a>) and the BTx623 genome as a reference (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Paterson <em>et al.</em> 2009</a>). Genotypes for the association panels at 265,487 SNPs were previously obtained (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Morris <em>et al.</em> 2013</a>). For this study we also genotyped the same 265,487 SNPs in 263 F<sub>6-7</sub> recombinant inbred lines (RILs; ICSV 745 × PB 15520) that were developed as a stem borer resistance mapping population but also segregate flavonoid pigmentation phenotypes (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Vinayan 2010</a>). Note, SNP positions in the GBS data may differ slightly (several base pairs) from the reference genome because of small indel polymorphisms. Missing genotype calls were imputed using the FastImputationBitFixedWindow plugin in TASSEL 4.0 (<a href="http://sourceforge.net/projects/tassel/">http://sourceforge.net/projects/tassel/</a>).</p></div><div id="sec-4" class="subsection"><h3>Pigmentation phenotypes</h3><p id="p-12">Tannin phenotypes for the small association panel (<em>n</em> = 142) were previously published (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 2012</a>). This panel represents a subset of early-maturing, semi-dwarf accessions from the U.S. Sorghum Association Panel (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Casa <em>et al.</em> 2008</a>). For the large association panel, seeds for the full Sorghum Association Panel were obtained from the U.S. National Plant Germplasm System via the Germplasm Resource Information Network (<a href="http://www.ars-grin.gov">http://www.ars-grin.gov</a>). The presence or absence of a pigmented testa on the grains was visually assessed (three seeds per accession) after removal of the pericarp on the dorsal side and scored as 1 and 0, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1/FileS2.xlsx">File S2</a>). Eleven accessions that showed a segregating phenotype were dropped from the analysis. Pericarp pigmentation was visually assessed for three seeds per accession, scored as 0 for white and 1 for red or yellow (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1/FileS2.xlsx">File S2</a>). Brown pericarp accessions were dropped from the analysis because this phenotype is known to be caused by the spread of tannin from the testa and masks the expression of the <em>R</em> and <em>Y</em> genes (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Doggett 1988</a>). For analysis of structuring of tannin phenotypes, we obtained testa pigmentation data (<em>n</em> = 14,785) in world sorghum collections from the Germplasm Resource Information Network. For the RIL family, phenotyping was performed in the 2007 and 2008 <em>kharif</em> (rainy) season in Patancheru, India, with presence/absence of pigmentation scored for 10-wk-old seedlings (coleoptile color) or physiologically mature plants (testa and adult plant color; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Vinayan 2010</a>).</p></div><div id="sec-5" class="subsection"><h3>Genomic analysis</h3><p id="p-13">Genome-wide mapping in RILs and the association panels was carried out using a GLM, mixed linear model (MLM), or compressed mixed linear model (CMLM) with population parameters previously determined (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zhang <em>et al.</em> 2010</a>) as implemented in the Genomic Association and Prediction Integrated Tool (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lipka <em>et al.</em> 2012</a>). When a population structure (Q) term was included, the model selection feature of the Genomic Association and Prediction Integrated Tool was used to determine the optimal number of principal components (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Zhu and Yu 2009</a>). Genetic structure of the world sorghum populations was estimated by the use of principal components analysis implemented by cmdscale in R (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">R Core Team 2012</a>).</p><p id="p-14">Loss-of-function genome scan <em>P</em>-value for a given SNP (loss of function, <em>i.e.</em>, LOF) is calculated with binomial tests in R (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">R Core Team 2012</a>) as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2085/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2085/embed/mml-math-1.gif"/></noscript></span></span>where L<sub>1</sub> and L<sub>2</sub> are the counts of the loss-of-function phenotype for alleles 1 and 2 (“successes”), respectively, W<sub>1</sub> and W<sub>2</sub> are the counts of the wild-type phenotype for alleles 1 and 2 (“failures”), respectively, and P is the overall proportion of loss-of-function phenotypes given by <em>P</em> = (L<sub>1</sub>+ L<sub>2</sub>)/(L<sub>1</sub>+ L<sub>2</sub>+ W<sub>1</sub>+ W<sub>2</sub>).</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>Grain tannin GWAS in a small association panel</h3><p id="p-15">To identify loci underlying natural variation in grain tannin pigmentation, we first characterized associations between published tannin (presence/absence) phenotypes from a small (<em>n</em> = 142) global diversity panel (<a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 2012</a>) and genotypes from a 265,487 SNP genotyping-by-sequencing data set (<a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Morris <em>et al.</em> 2013</a>). Included in the GBS SNP map is a G-to-T transition in the <em>Tannin1</em> coding region (S4_61667908) that is 218 bp upstream of, and in perfect linkage disequilibrium with, the G-deletion that is causative for the <em>tan1-a</em> null allele (<em>n</em> = 161) (<a id="xref-ref-51-6" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 2012</a>). Therefore, we can use this SNP (hereafter referred to as the <em>tan1-a</em> SNP) as a positive control to compare mapping approaches. Using a simple model without statistical control for population structure (General Linear Model; GLM) we find that, as expected, the most significant association signals lies at ~61 Mb on chromosome 4 colocalized with the <em>Tannin1</em> locus (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1/FigureS1.pdf">Figure S1</a>A-B and <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). However, the <em>tan1-a</em> SNP is not the most significant association in this region (rank = 230; <em>P</em> &lt; 10<sup>−8</sup>). Instead, the most significant association peaks are found in the ~1-Mb region surrounding <em>Tannin1</em>, at 61.1 Mb (S4_61121403; <em>P</em> &lt; 10<sup>−12</sup>), 61.2 Mb (S4_61233495; <em>P</em> &lt; 10<sup>−10</sup>), and 61.8 Mb (S4_61862778; <em>P</em> &lt; 10<sup>−12</sup>). Including the <em>Tannin1</em> locus, we observe at least 13 peaks of association that are significant at a nominal Bonferroni-corrected <em>P</em>-value of 0.01 (<em>P</em> &lt; 2 × 10<sup>−7</sup>), among which are additional candidate loci for genetic control of tannin pigmentation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1/FileS3.pdf">File S3</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463512/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463512/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of testa pigmentation GWAS</span><div class="sb-div caption-clear"></div></div></div><p id="p-17">Given the extensive population structure in sorghum, as well as kinship among elite sorghum lines, we also carried out GWAS using MLMs that take into account population structure (Q) and/or kinship (K) (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). We fit both standard MLMs (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Yu <em>et al.</em> 2006</a>), which treat each individual as an independent group when fitting the random effect of kinship, and CMLMs, which have been shown to outperform GLMs and standard MLMs in some cases (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Zhang <em>et al.</em> 2010</a>). In addition, we estimated the optimal number of principal components to include as fixed effects (Q) on the basis of model selection and Bayesian information criteria (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Zhu and Yu 2009</a>). The genomic locations of GWAS peaks from the GLM are largely unchanged with the CMLM (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1/FigureS1.pdf">Figure S1</a>C-D and <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>), although the nominal significance of the peaks is reduced. All the nominally significant association peaks identified with the GLM remain in the CMLM, although most fall below the threshold for genome-wide significance. The top associations remain near <em>Tannin1</em> on chromosome 4 but controlling for population structure and kinship with a MLM (MLM and CMLM) does not improve the mapping resolution at the <em>Tannin1</em> locus, as the <em>tan1-a</em> SNP remains less significant (<em>P</em> &lt; 10<sup>−4</sup>) than several other SNPs in the ~1 Mb region surrounding <em>Tannin1</em> (<em>P</em> &lt; 10<sup>−6</sup>).</p></div><div id="sec-8" class="subsection"><h3>Grain tannin GWAS in a large association panel</h3><p id="p-18">If the inability to precisely map <em>Tannin1</em> in the small association panel is caused by a small sample size, then increasing the size of the association panel should improve the mapping resolution. Therefore, we phenotyped testa presence/absence in the sorghum association panel (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Casa <em>et al.</em> 2008</a>) and performed GWAS by using a variety of linear models (<em>n</em> = 336; <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). Fitting a GLM with this larger panel, the association for the <em>tan1-a</em> SNP is increased in rank and significance (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A-B</a>; rank = 9; <em>P</em> &lt; 10<sup>−14</sup>), but is still not the top association. Adding control for population structure and kinship using CMLM (K or Q+K), we find the rank and significance of the <em>tan1-a</em> SNP is actually reduced compared with the results using a GLM (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>; rank = 14355; <em>P</em> = 0.09). Finally, a GLM with population term (Q) or a standard MLM (K only; <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1C-D</a>) that treats each individual as a separate group for estimating kinship does identify the <em>tan1-a</em> SNP precisely (rank = 1; <em>P</em> &lt; 10<sup>−15</sup> and <em>P</em> &lt; 10<sup>−12</sup>, respectively).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2085/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide association study of testa presence using linear models. Manhattan plots using a GLM (A, B) and standard MLM kinship (K) (C, D) with 265,487 SNPs and 336 accessions, with (B, D) detailed view at Tannin1 locus on chromosome 4 for GLM and MLM (K), respectively. Although the Tannin1 locus is identified as the major effect locus for tannin presence with GLM, the Tannin1 gene (red bar) is precisely identified with a MLM. Other flavonoid-related genes are indicated by the blue bars, whereas all other annotated genes in the detailed view are indicated in green." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-615362286" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome-wide association study of testa presence using linear models. Manhattan plots using a GLM (A, B) and standard MLM kinship (K) (C, D) with 265,487 SNPs and 336 accessions, with (B, D) detailed view at &lt;em&gt;Tannin1&lt;/em&gt; locus on chromosome 4 for GLM and MLM (K), respectively. Although the &lt;em&gt;Tannin1&lt;/em&gt; locus is identified as the major effect locus for tannin presence with GLM, the &lt;em&gt;Tannin1&lt;/em&gt; gene (red bar) is precisely identified with a MLM. Other flavonoid-related genes are indicated by the blue bars, whereas all other annotated genes in the detailed view are indicated in green.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2085/F1.medium.gif" width="340" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/11/2085/F1.medium.gif" width="340" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2085/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2085/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463496" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Genome-wide association study of testa presence using linear models. Manhattan plots using a GLM (A, B) and standard MLM kinship (K) (C, D) with 265,487 SNPs and 336 accessions, with (B, D) detailed view at <em>Tannin1</em> locus on chromosome 4 for GLM and MLM (K), respectively. Although the <em>Tannin1</em> locus is identified as the major effect locus for tannin presence with GLM, the <em>Tannin1</em> gene (red bar) is precisely identified with a MLM. Other flavonoid-related genes are indicated by the blue bars, whereas all other annotated genes in the detailed view are indicated in green.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Why do some models that account for population structure (CMLM) perform worse than a naive model (GLM), generating a false-negative result for the <em>tan1-a</em> SNP? To better understand the population structure of natural variation in tannins, we characterized the distribution of pigmented testa phenotype in worldwide sorghum collections. The tannin trait segregates in all the botanical races of sorghum but shows modest population structure, with durra and guinea types having the lowest proportion of tannin accessions (15%) and caudatum and guinea-caudatum accessions having the greatest proportion (76% and 83%, respectively). However, with respect to model selection in GWAS, the structuring of the trait itself may be less important than structuring of the alleles underlying the trait. The <em>tan1-a</em> allele is found at high frequency in African and Indian durra accessions and at low frequency in Chinese and southern African accessions (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The <em>tan1-a</em> allele explains the tannin phenotypes in all durra-derived accessions studied, but only partially accounts for the phenotype in caudatum accessions and not at all in guinea accessions. Among caudatum, guinea, and kafir types there are numerous accessions that have wild-type <em>Tannin1</em> coding regions yet have nontannin phenotypes, which suggests that the population structuring of heterogeneous alleles may account for the overcorrection by the CMLM and the effective correction by the MLM.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2085/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of tannin phenotype and loss-of-function alleles in a worldwide sorghum diversity panel. (A) Accessions plotted according to the first two principal components of sorghum population structure (small panel; n = 142), with phenotyped accessions color-coded by Tannin1 allele and tannin phenotype (nonphenotyped accessions in gray). Although some nontannin accessions are explained by tan1-a or tan1-b (green), others must be due to additional loss-of-function alleles (blue). (ND: Not Determined) (B) Distribution of wild-type allele (G; orange) and loss-of-function allele (T; red) for the tan1-a SNP (S4_61667908) in source-identified sorghum accessions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-615362286" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of tannin phenotype and loss-of-function alleles in a worldwide sorghum diversity panel. (A) Accessions plotted according to the first two principal components of sorghum population structure (small panel; &lt;em&gt;n&lt;/em&gt; = 142), with phenotyped accessions color-coded by &lt;em&gt;Tannin1&lt;/em&gt; allele and tannin phenotype (nonphenotyped accessions in gray). Although some nontannin accessions are explained by &lt;em&gt;tan1-a&lt;/em&gt; or &lt;em&gt;tan1-b&lt;/em&gt; (green), others must be due to additional loss-of-function alleles (blue). (ND: Not Determined) (B) Distribution of wild-type allele (G; orange) and loss-of-function allele (T; red) for the &lt;em&gt;tan1-a&lt;/em&gt; SNP (S4_61667908) in source-identified sorghum accessions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2085/F2.medium.gif" width="294" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/11/2085/F2.medium.gif" width="294" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2085/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2085/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463613" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Distribution of tannin phenotype and loss-of-function alleles in a worldwide sorghum diversity panel. (A) Accessions plotted according to the first two principal components of sorghum population structure (small panel; <em>n</em> = 142), with phenotyped accessions color-coded by <em>Tannin1</em> allele and tannin phenotype (nonphenotyped accessions in gray). Although some nontannin accessions are explained by <em>tan1-a</em> or <em>tan1-b</em> (green), others must be due to additional loss-of-function alleles (blue). (ND: Not Determined) (B) Distribution of wild-type allele (G; orange) and loss-of-function allele (T; red) for the <em>tan1-a</em> SNP (S4_61667908) in source-identified sorghum accessions.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-9" class="subsection"><h3>Mapping of flavonoid traits in a RIL family</h3><p id="p-22">Given that complex association signals made the precise mapping of <em>tan1-a</em> in an association population difficult, we considered whether the reduced genetic and allelic heterogeneity of a biparental family would allow precise mapping of <em>Tannin1</em> using linear models. We phenotyped testa pigmentation (tannin presence) in a population of 263 RILs that we genotyped at 265,487 SNPs. There is a single locus associated with the pigmented testa in this family that is precisely colocalized with <em>Tannin1</em> on chromosome 4, with the most significant SNP being the <em>tan1-a</em> SNP (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). To determine whether this gene resolution mapping in RILs is likely to be a typical result, we also mapped two other flavonoid pigmentation traits (coleoptile color and adult plant color) that are segregating in this RIL family. We mapped coleoptile color to a region around 54 Mb on chromosome 6 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1/FigureS2.pdf">Figure S2</a>). This peak colocalizes with the classical <em>Rs1</em> locus and <em>a priori</em> candidate gene <em>Sb06g025060</em>, a putative basic helix-loop-helix (bHLH) transcription factor, and a sorghum co-ortholog of Arabidopsis <em>TRANSPARENT TESTA8</em> and maize <em>B1</em>/<em>R1</em> anthocyanin regulators (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1/FileS3.pdf">File S3</a>). However, the most significant SNP (S6_53849573) is 220 kb upstream of <em>Sb06g025060</em>, and no promising <em>a posteriori</em> candidate genes are found at S6_53849573, suggesting gene resolution was not achieved in this case. Adult plant color maps to 58 Mb on chromosome 6, colocalized with the classical <em>P</em> locus (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Doggett 1988</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Mace and Jordan 2010</a>) and a large cluster of putative reductase genes that are homologous to Arabidopsis <em>TRANSPARENT TESTA3</em> and <em>BANYULS</em> and maize <em>ANTHOCYANINLESS1</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1/FileS1.xlsx">File S1</a>). Here again, the most significant SNP (S6_57865283) is not colocalized precisely with the <em>a priori</em> candidates (<em>TT3</em>/<em>BANYULS</em> cluster) but 260 kb upstream (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1/FigureS3.pdf">Figure S3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2085/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide mapping of testa presence in a RIL biparental family. Manhattan plots for a linear model using 263 recombinant inbred lines genotyped at 265,487 SNPs, (A) scanning genome-wide and (B) with a detailed view at Tannin1 locus on chromosome 4, with Tannin1 indicated by the red bar. Other flavonoid-related genes are indicated by the blue bars, whereas all other annotated genes in the detailed view are indicated in green." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-615362286" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome-wide mapping of testa presence in a RIL biparental family. Manhattan plots for a linear model using 263 recombinant inbred lines genotyped at 265,487 SNPs, (A) scanning genome-wide and (B) with a detailed view at &lt;em&gt;Tannin1&lt;/em&gt; locus on chromosome 4, with &lt;em&gt;Tannin1&lt;/em&gt; indicated by the red bar. Other flavonoid-related genes are indicated by the blue bars, whereas all other annotated genes in the detailed view are indicated in green.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2085/F3.medium.gif" width="440" height="278"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/11/2085/F3.medium.gif" width="440" height="278"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2085/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2085/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463525" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">Genome-wide mapping of testa presence in a RIL biparental family. Manhattan plots for a linear model using 263 recombinant inbred lines genotyped at 265,487 SNPs, (A) scanning genome-wide and (B) with a detailed view at <em>Tannin1</em> locus on chromosome 4, with <em>Tannin1</em> indicated by the red bar. Other flavonoid-related genes are indicated by the blue bars, whereas all other annotated genes in the detailed view are indicated in green.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Loss-of-function genome scan on grain tannin</h3><p id="p-24">Why did several of the linear models we tested fail to precisely identify the <em>tan1-a</em> allele in association panels, even though it is common and highly penetrant? A comparison of the 2 × 2 contingency tables for the <em>tan1-a</em> SNP <em>vs.</em> the more significant SNPs provides some insight here (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). The loss-of-function <em>tan1-a</em> allele shows striking covariation with the testa phenotype (T allele: nontannin = 78 <em>vs.</em> tannin = 0) but little signal of covariation for the wild-type allele (G allele: nontannin = 112 <em>vs.</em> tannin = 139). Although this lack of covariation for the wild-type allele is to be expected (because there is no reason that accessions carrying the wild-type allele at <em>Tannin1</em> cannot carry loss-of-function alleles at other loci), it reduces the significance of a linear model fitting the genotype-phenotype association. In contrast, the other more significant SNPs near <em>Tannin1</em> show covariation for both alleles (in opposite directions), with the wild-type allele more often found with the wild-type phenotype (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). This pattern of covariation increases the significance of the fit of a linear model or contingency test, even though the genotype-phenotype covariation for the wild-type allele is irrelevant when considering a loss-of-function polymorphism.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2085/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome-wide mapping of testa presence using a loss-of-function genome scan. (A) Allele counts for the tan1-a SNP and other SNPs near Tannin1 that are associated with a nontannin phenotype. The nominal P-value is from a binomial test on the putative loss-of-function allele at each SNP, (B) scanning genome-wide, and (C) a detailed view of the Tannin1 locus on chromosome 4, with Tannin1 indicated by the red bar. Other flavonoid-related genes are indicated by the blue bars, whereas all other annotated genes in the detailed view are indicated in green." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-615362286" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome-wide mapping of testa presence using a loss-of-function genome scan. (A) Allele counts for the &lt;em&gt;tan1-a&lt;/em&gt; SNP and other SNPs near &lt;em&gt;Tannin1&lt;/em&gt; that are associated with a nontannin phenotype. The nominal &lt;em&gt;P&lt;/em&gt;-value is from a binomial test on the putative loss-of-function allele at each SNP, (B) scanning genome-wide, and (C) a detailed view of the &lt;em&gt;Tannin1&lt;/em&gt; locus on chromosome 4, with &lt;em&gt;Tannin1&lt;/em&gt; indicated by the red bar. Other flavonoid-related genes are indicated by the blue bars, whereas all other annotated genes in the detailed view are indicated in green.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2085/F4.medium.gif" width="440" height="348"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/11/2085/F4.medium.gif" width="440" height="348"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2085/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2085/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463507" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-25" class="first-child">Genome-wide mapping of testa presence using a loss-of-function genome scan. (A) Allele counts for the <em>tan1-a</em> SNP and other SNPs near <em>Tannin1</em> that are associated with a nontannin phenotype. The nominal <em>P</em>-value is from a binomial test on the putative loss-of-function allele at each SNP, (B) scanning genome-wide, and (C) a detailed view of the <em>Tannin1</em> locus on chromosome 4, with <em>Tannin1</em> indicated by the red bar. Other flavonoid-related genes are indicated by the blue bars, whereas all other annotated genes in the detailed view are indicated in green.</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">To investigate an approach that may be appropriate for loss-of-function alleles we used a simple heuristic genome scan based on a binomial test (see the section <em>Materials and Methods</em>). We identified SNPs with alleles that are often found in individuals with the loss-of-function phenotype (testa absent) and rarely or never found in individuals with the wild-type phenotype (testa present). In effect, the phenotypes at the putative wild-type allele at each SNP is ignored. With this loss-of-function genome scan approach, the <em>tan1-a</em> SNP is precisely identified whereas other SNPs near the <em>Tannin1</em> locus that had strong indirect associations with linear modeling have reduced association signals (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4, B and C</a>, <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1/FigureS4.pdf">Figure S4</a>). Thus, in this case we find that a simple heuristic scan that considers the underlying genetics of the trait outperforms more sophisticated models.</p></div><div id="sec-11" class="subsection"><h3>Pericarp pigmentation GWAS</h3><p id="p-27">Although we found that gene-resolution mapping was possible with <em>Tannin1</em>, we wondered whether this would be true for other flavonoid pigmentation traits. To assess the mapping resolution in this panel with another trait, we used the white pericarp phenotype, which reflects a lack of flavonoid pigmentation in the outer seed coat (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Doggett 1988</a>; <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Ibraheem <em>et al.</em> 2010</a>). It is known that white pericarp phenotype can be caused by loss-of-function mutations in the <em>Yellow seed1</em> gene (<em>Y1</em>, Chr1: 61,237,360–61,241,520; <a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Ibraheem <em>et al.</em> 2010</a>). Note, the <em>Y1</em> gene was cloned based on the excision of a transposable element, not genetic analysis of natural variation, so we do not have a validated genetic variant of <em>Y1</em> in the SNP data as we did for the <em>Tannin1</em> case. However, it is known from the classical inheritance literature that natural variation at the <em>Y</em> locus exists (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rooney 2000</a>). As expected, the most significant associations were found around 61 Mb on chromosome 1 (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). As was the case for <em>Tannin1</em>, though, the <em>Y1</em> gene was broadly but not precisely identified: among the top associations are SNPs that are flanking <em>Y1</em> (5 kb from <em>Y1</em>, S1_61246791; <em>P</em> &lt; 10<sup>−8</sup>) but the most significant association are ~0.5 Mb from <em>Y1</em> (S1_61717652). Because white pericarp represents loss-of-function phenotype, we also used the loss-of-function genome scan approach for the pericarp pigmentation trait. In this case, the loss-of-function genome scan does not improve the mapping resolution as compared to the linear models.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2085/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="GWAS of white pericarp phenotype. Manhattan plots for GWAS using CMLM kinship (K) (A, B) and a loss-of-function genome scan (C, D) and with 265,000 SNPs and 347 accessions, showing a (A, C) genome-wide and (B, D) a detailed view around Y1 on chromosome 1, with the Y1 gene indicated by the red bar. Other flavonoid-related genes are indicated by the blue bars, whereas all other annotated genes in the detailed view are indicated in green. &#xD7;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-615362286" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;GWAS of white pericarp phenotype. Manhattan plots for GWAS using CMLM kinship (K) (A, B) and a loss-of-function genome scan (C, D) and with 265,000 SNPs and 347 accessions, showing a (A, C) genome-wide and (B, D) a detailed view around &lt;em&gt;Y1&lt;/em&gt; on chromosome 1, with the &lt;em&gt;Y1&lt;/em&gt; gene indicated by the red bar. Other flavonoid-related genes are indicated by the blue bars, whereas all other annotated genes in the detailed view are indicated in green. &#xD7;&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2085/F5.medium.gif" width="342" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/11/2085/F5.medium.gif" width="342" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2085/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2085/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463616" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-28" class="first-child">GWAS of white pericarp phenotype. Manhattan plots for GWAS using CMLM kinship (K) (A, B) and a loss-of-function genome scan (C, D) and with 265,000 SNPs and 347 accessions, showing a (A, C) genome-wide and (B, D) a detailed view around <em>Y1</em> on chromosome 1, with the <em>Y1</em> gene indicated by the red bar. Other flavonoid-related genes are indicated by the blue bars, whereas all other annotated genes in the detailed view are indicated in green. ×</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><div id="sec-13" class="subsection"><h3>Genetics of flavonoid pigmentation</h3><p id="p-29">GWAS have been useful to characterize the contribution of known and novel flavonoid pigmentation genes in several plant species (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Atwell <em>et al.</em> 2010</a>; <a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Huang <em>et al.</em> 2010</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Cockram <em>et al.</em> 2010</a>). Given that control of flavonoids by a WD40-bHLH-MYB regulatory system is broadly conserved (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Petroni and Tonelli 2011</a>), it seems likely that these regulators underlie some natural variation in sorghum pigmentation. Is there evidence of natural variation in each of the three types of transcription factors that control tannins in plants (WD40, bHLH, MYB)? From classical inheritance studies, it is known that at least two loci control the presence of brown coloration in grain subcoat (<em>B1</em> and <em>B2</em>) that have been mapped to chromosome 2 and chromosome 4, and one of these was cloned as WD40 gene <em>Tannin1</em> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Rami <em>et al.</em> 1998</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Mace and Jordan 2010</a>; <a id="xref-ref-51-7" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 2012</a>). On the basis of a comparison of <em>B1</em>/<em>B2</em> genotypes from classical inheritance studies (<a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Doggett 1988</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Rooney 2000</a>) and <em>Tannin1</em> genotypes (<a id="xref-ref-51-8" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 2012</a>), it can be inferred that <em>Tannin1</em> corresponds to <em>B2</em>. Multiple studies have identified a second major effect locus controlling tannin presence at around 8 Mb on chromosome 2, and from the <em>Tannin1</em>-controlled GWAS (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1/FileS3.pdf">File S3</a>), there is evidence that the gene underlying <em>B1</em> is the putative bHLH transcription factor <em>Sb02g006390</em>. This study is based on qualitative tannin phenotypes (presence/absence) so quantitative phenotyping may reveal additional loci underlying the observed variation among intermediate and high tannin varieties (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gu <em>et al.</em> 2004</a>).</p><p id="p-30">Given that flavonoid-related gene families are well-characterized and well-conserved across distant plant lineages, we would expect that most association peaks would colocalize to <em>a priori</em> candidate genes. Although a number of peaks do colocalize precisely with candidate genes, many more of them do not, despite the large candidate gene set and the liberal inclusion criteria. It is possible that some unexpected signals represent true associations at novel genes, though it is unlikely for most peaks. In some cases, these unexpected signals may represent stochastic noise, spurious associations that do not reflect any underlying genetic heritability. However, most of these peaks increased in significance with a larger sample size so they are unlikely to represent stochastic noise and more likely to represent indirect associations (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Platt <em>et al.</em> 2010</a>). In particular, the significant SNPs near <em>Tannin1</em> are likely to represent synthetic associations (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Goldstein 2011</a>), instances in which a phenotype caused by multiple rare alleles is spuriously assigned to a common allele that is linked.</p></div><div id="sec-14" class="subsection"><h3>Mapping loss-of-function alleles</h3><p id="p-31">The development of experimental design strategies and statistical methods that account for complex genetic architecture and population structure in genome-wide mapping studies is an active area of research (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Zhang <em>et al.</em> 2010</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Zhao <em>et al.</em> 2011</a>; <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Huang <em>et al.</em> 2012</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Lipka <em>et al.</em> 2012</a>; <a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Segura <em>et al.</em> 2012</a>). Here we were able to compare several mapping approaches empirically by using the testa presence/absence phenotype in sorghum and a GBS SNP map that includes a validated SNP in the <em>Tannin1</em> major effect gene. All the approaches we tested here identified the <em>Tannin1</em> locus, broadly defined (+/− 1 Mb of the <em>Tannin1</em> gene), as the major locus underlying testa presence/absence, but the precise identification of the <em>Tannin1</em> gene was not consistently obtained. Increasing the panel size increased the significance of the <em>tan1-a</em> SNP relative to the indirect associations. Given that minor allele frequencies (MAFs) of indirect associations (<em>i.e.</em>, the SNPs near <em>Tannin1</em> that have greater association signals than the <em>tan1-a</em> SNP) are greater (MAF = 0.3 – 0.45) than the <em>tan1-a</em> SNP (MAF = 0.2), these indirect associations fit the expectation for synthetic associations: older, more widespread SNPs that tag a haplotype block on which multiple loss-of-function alleles of <em>Tannin1</em> have arisen (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Dickson <em>et al.</em> 2010</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Orozco <em>et al.</em> 2010</a>). Although methods have been developed to account for epistasis by stepwise model fitting (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Segura <em>et al.</em> 2012</a>), it will be difficult to derive benefits from these models if the most significant association in the first model step is itself an indirect association. Given that allelic heterogeneity is abundant among the handful of well-characterized sorghum genes [<em>Tannin1</em> (<a id="xref-ref-51-9" class="xref-bibr" href="#ref-51">Wu <em>et al.</em> 2012</a>), <em>Maturity1</em> (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Murphy <em>et al.</em> 2011</a>), and <em>Shattering1</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lin <em>et al.</em> 2012</a>)] synthetic associations could be a common feature of major effect loci in sorghum and other species of similar population structure.</p><p id="p-32">Loss-of-function variants are a common source of natural variation, and recently methods have been developed to identify multiple, low-frequency loss-of-function alleles at the same gene in GWAS (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Liu <em>et al.</em> 2011</a>). The loss-of-function genome scan approach we used here allowed us to precisely map a high frequency loss-of-function allele to the exclusion of nearby indirect associations. Note, that although we applied this approach to binary phenotypes the principle does not depend on binary phenotypes. Rather, it depends on the epistasis that exists because the loss-of-function allele cannot be rescued (<em>i.e.</em>, wild-type phenotype is never found with loss-of-function allele) while the wild-type allele can easily be found in an individual that harbors an independent loss-of-function allele at a different locus (<em>i.e.</em>, loss-of-function phenotype is often found with wild-type allele).</p></div><div id="sec-15" class="subsection"><h3>Improving resolution with genomic data</h3><p id="p-33">We found that mapping in a biparental family using high-density SNP markers can achieve gene resolution. Traditionally, high-resolution SNP maps have not been used for biparental families due to technical limitations and the expectation that the small number of recombination events limits the utility of greater marker density. The gene-resolution mapping of <em>Tannin1</em> in a modest-sized (<em>n</em> = 263) RIL family with simple phenotyping (field-based scoring of testa presence/absence) demonstrates that the combination of advanced mapping populations with high-resolution genotyping is an effective strategy for trait dissection (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bergelson and Roux 2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brachi <em>et al.</em> 2011</a>). Still, the lower-resolution mapping results for the adult plant color and coleoptile color traits in the RILs suggest that consistent gene-level mapping will require larger families and/or advanced multi-parent mapping approaches (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jordan <em>et al.</em> 2011</a>). Given the substantial investment that has already been made in RIL development, the cost-effectiveness of GBS, and the suitability of the system for “genotype once, phenotype many times” approach, a broader effort to make high-density genotypes available for existing advanced mapping populations seems warranted.</p><p id="p-34">It is worth noting that if the <em>tan1-a</em> SNP had not, by chance, been represented in the GBS SNP map, the results of the loss-of-function genome scan would have been qualitatively equivalent to the linear models. This highlights that to derive a benefit from the loss-of-function genome scan approach over linear models the density of genotyping must be high enough that causative SNPs, or SNPs in perfect LD with the causative polymorphism (<em>e.g.</em>, <em>tan1-a</em>), are represented in the genotyping data. A lack of a causative or perfectly linked SNP for pericarp pigmentation would explain why both the linear model and loss-of-function genome scan approaches achieved just Mb resolution for this trait. Similarly in the RIL mapping, had the <em>tan1-a</em> SNP not been represented the other SNPs in the data set would not have precisely tagged the <em>Tannin1</em> gene. Although the identification of <em>a priori</em> candidates through comparative genomics provides some guidance, there are typically many reasonable candidates within Mb-scale mapping intervals; for example, around 60−63 Mb on chromosome 4, there are four other homologs of tannin genes that would have been equally promising candidates. As genomic coverage increases, imputation methods improve, and mapping populations are refined and expanded, genome-wide mapping approaches will increasingly need to be optimized to identify causative variants as opposed to tagging SNPs (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Huang <em>et al.</em> 2012</a>). Given the ever-lowering costs of sequencing <em>vs.</em> the high cost of candidate gene validation efforts, the use of whole-genome resequencing to increase the resolution of mapping studies is likely to be cost effective. Although this study of relatively simple pigmentation traits highlights the trade-offs between population- and family-based mapping approaches because of the complexity of association signals, in either approach high density genotyping can facilitate gene resolution mapping of traits.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-35">We thank National Science Foundation for providing funds (ID: IOS-0965342) to carry out this research under Basic Research to Enable Agricultural Development project. This work was also supported by a grant from United States Department of Agriculture-National Institute of Food and Agriculture Plant Feedstock Genomics for Bioenergy Program (#2011-03502) to S. K. This work has been undertaken as part of the CGIAR Research Program on Dryland Cereals. We thank the editors and two anonymous reviewers for helpful suggestions.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008417/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Sequence data from this article have been deposited in the NCBI Short Read Archive database under accession no. SRA062716). Genotypes are available from the Dryad Repository at <a href="http://doi.org/10.5061/dryad.620cq">http://doi.org/10.5061/dryad.620cq</a>.</p></li><li class="fn" id="fn-5"><p id="p-36"><em>Communicating editor: D.-J. De Koning</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-06-17"><span class="received-label">Received </span>June 17, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-09-11"><span class="accepted-label">Accepted </span>September 11, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Morris <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-4">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.1" data-doi="10.1038/nature08800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atwell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjalmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Willems</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association study of 107 phenotypes in <em>Arabidopsis thaliana</em> inbred lines.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">465</span>: <span class="cit-fpage">627</span>–<span class="cit-lpage">631</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAtwell%26rft.auinit1%253DS.%26rft.volume%253D465%26rft.issue%253D7298%26rft.spage%253D627%26rft.epage%253D631%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2B107%2Bphenotypes%2Bin%2BArabidopsis%2Bthaliana%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08800%26rft_id%253Dinfo%253Apmid%252F20336072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20336072&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278249000044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.2" data-doi="10.1007/s00438-007-0289-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beló</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luck</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Whole genome scan detects an allelic variant of <em>fad2</em> associated with increased oleic acid levels in maize.</span> <abbr class="cit-jnl-abbrev">Mol. Genet. Genomics</abbr> <span class="cit-vol">279</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bgenetics%2Band%2Bgenomics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGenet%2BGenomics%26rft.aulast%253DBelo%26rft.auinit1%253DA.%26rft.volume%253D279%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D10%26rft.atitle%253DWhole%2Bgenome%2Bscan%2Bdetects%2Ban%2Ballelic%2Bvariant%2Bof%2Bfad2%2Bassociated%2Bwith%2Bincreased%2Boleic%2Bacid%2Blevels%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00438-007-0289-y%26rft_id%253Dinfo%253Apmid%252F17934760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00438-007-0289-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17934760&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251600600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.3" data-doi="10.1038/nrg2896"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bergelson</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roux</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Towards identifying genes underlying ecologically relevant traits in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">867</span>–<span class="cit-lpage">879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBergelson%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D867%26rft.epage%253D879%26rft.atitle%253DTowards%2Bidentifying%2Bgenes%2Bunderlying%2Becologically%2Brelevant%2Btraits%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2896%26rft_id%253Dinfo%253Apmid%252F21085205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2896&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21085205&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.4" data-doi="10.1186/gb-2011-12-10-232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brachi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borevitz</span>  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association studies in plants: the missing heritability is in the field.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DBrachi%26rft.auinit1%253DB.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D232%26rft.epage%253D232%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bin%2Bplants%253A%2Bthe%2Bmissing%2Bheritability%2Bis%2Bin%2Bthe%2Bfield.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-10-232%26rft_id%253Dinfo%253Apmid%252F22035733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-10-232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22035733&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.5" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTkvMjYzMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.6" data-doi="10.2135/cropsci2007.02.0080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casa</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooney</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Community resources and strategies for association mapping in sorghum.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">30</span>–<span class="cit-lpage">40</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCasa%26rft.auinit1%253DA.%2BM.%26rft.volume%253D48%26rft.issue%253D1%26rft.spage%253D30%26rft.epage%253D40%26rft.atitle%253DCommunity%2BResources%2Band%2BStrategies%2Bfor%2BAssociation%2BMapping%2Bin%2BSorghum%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2007.02.0080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2007.02.0080&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000252971400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.7" data-doi="10.1073/pnas.1010179107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cockram</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zuluaga</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comadran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association mapping to candidate polymorphism resolution in the unsequenced barley genome.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">21611</span>–<span class="cit-lpage">21616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1010179107%26rft_id%253Dinfo%253Apmid%252F21115826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA3LzUwLzIxNjExIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2085.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darwin</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1859</span> <span class="cit-source">On the Origin of the Species by Means of Natural Selection: Or, The Preservation of Favoured Races in the Struggle for Life</span>. <span class="cit-publ-name">John Murray</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.9" data-doi="10.1371/journal.pbio.1000294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dickson</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krantz</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hakonarson</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Rare variants create synthetic genome-wide associations.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1000294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DDickson%26rft.auinit1%253DS.%2BP.%26rft.volume%253D8%26rft.issue%253D1%26rft.spage%253De1000294%26rft.epage%253De1000294%26rft.atitle%253DRare%2Bvariants%2Bcreate%2Bsynthetic%2Bgenome-wide%2Bassociations.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000294%26rft_id%253Dinfo%253Apmid%252F20126254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000294&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20126254&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2085.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doggett</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-source">Sorghum</span>. <span class="cit-publ-name">Longman Scientific &amp; Technical</span>, <span class="cit-publ-loc">Essex, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.11" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.12" data-doi="10.1094/Phyto-83-490"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Esele</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frederiksen</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">F. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">The association of genes controlling caryopsis traits with grain mold resistance in sorghum.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">490</span>–<span class="cit-lpage">495</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.volume%253D83%26rft.spage%253D490%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPhyto-83-490%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/Phyto-83-490&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1993LE13200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.13" data-doi="10.1038/nrg3335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association studies in mice.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">807</span>–<span class="cit-lpage">817</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFlint%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D807%26rft.epage%253D817%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3335%26rft_id%253Dinfo%253Apmid%252F23044826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23044826&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.14" data-doi="10.1371/journal.pbio.1001008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The importance of synthetic associations will only be resolved empirically.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1001008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DGoldstein%26rft.auinit1%253DD.%2BB.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253De1001008%26rft.epage%253De1001008%26rft.atitle%253DThe%2Bimportance%2Bof%2Bsynthetic%2Bassociations%2Bwill%2Bonly%2Bbe%2Bresolved%2Bempirically.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1001008%26rft_id%253Dinfo%253Apmid%252F21267066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1001008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21267066&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodstein</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Howson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neupane</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Phytozome: a comparative platform for green plant genomics.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D1178</span>–<span class="cit-lpage">D1186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D40%26rft.spage%253DD1178%26rft_id%253Dinfo%253Apmid%252F22110026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22110026&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300176&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.16" data-doi="10.1016/j.tplants.2010.05.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gross</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsen</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genetic perspectives on crop domestication.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">537</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DGross%26rft.auinit1%253DB.%2BL.%26rft.volume%253D15%26rft.issue%253D9%26rft.spage%253D529%26rft.epage%253D537%26rft.atitle%253DGenetic%2Bperspectives%2Bon%2Bcrop%2Bdomestication.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2010.05.008%26rft_id%253Dinfo%253Apmid%252F20541451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2010.05.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20541451&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282403000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kelm</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hammerstone</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beecher</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Concentrations of proanthocyanidins in common foods and estimations of normal consumption.</span> <abbr class="cit-jnl-abbrev">J. Nutr.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">613</span>–<span class="cit-lpage">617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BNutrition%26rft.stitle%253DJ.%2BNutr.%26rft.aulast%253DGu%26rft.auinit1%253DL.%26rft.volume%253D134%26rft.issue%253D3%26rft.spage%253D613%26rft.epage%253D617%26rft.atitle%253DConcentrations%2Bof%2BProanthocyanidins%2Bin%2BCommon%2BFoods%2Band%2BEstimations%2Bof%2BNormal%2BConsumption%26rft_id%253Dinfo%253Apmid%252F14988456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToibnV0cml0aW9uIjtzOjU6InJlc2lkIjtzOjk6IjEzNC8zLzYxMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.18" data-doi="10.1038/ng.695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sang</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association studies of 14 agronomic traits in rice landraces.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">961</span>–<span class="cit-lpage">967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D42%26rft.issue%253D11%26rft.spage%253D961%26rft.epage%253D967%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bof%2B14%2Bagronomic%2Btraits%2Bin%2Brice%2Blandraces.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.695%26rft_id%253Dinfo%253Apmid%252F20972439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20972439&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000283540500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.19" data-doi="10.1038/ng.1018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association study of flowering time and grain yield traits in a worldwide collection of rice germplasm.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHuang%26rft.auinit1%253DX.%26rft.volume%253D44%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D39%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2Bflowering%2Btime%2Band%2Bgrain%2Byield%2Btraits%2Bin%2Ba%2Bworldwide%2Bcollection%2Bof%2Brice%2Bgermplasm.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1018%26rft_id%253Dinfo%253Apmid%252F22138690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22138690&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.20" data-doi="10.1534/genetics.109.111831"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ibraheem</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaffoor</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chopra</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Flavonoid phytoalexin-dependent resistance to anthracnose leaf blight requires a functional <em>yellow seed1</em> in <em>Sorghum bicolor</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">915</span>–<span class="cit-lpage">926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.111831%26rft_id%253Dinfo%253Apmid%252F20083611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg0LzQvOTE1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.21" data-doi="10.2135/cropsci2010.06.0326"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mace</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cruickshank</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henzell</span>  <span class="cit-name-given-names">R. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Exploring and exploiting genetic variation from unadapted sorghum germplasm in a breeding program.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1444</span>–<span class="cit-lpage">1457</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D51%26rft.spage%253D1444%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2010.06.0326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2010.06.0326&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000291972900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-3.11.2085.22" data-doi="10.1002/jsfa.5890"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaufman</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herald</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bean</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuinstra</span>  <span class="cit-name-given-names">M. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Variability in tannin content, chemistry and activity in a diverse group of tannin containing sorghum cultivars</span>. <abbr class="cit-jnl-abbrev">J. Sci. Food Agric.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">1233</span>–<span class="cit-lpage">1241</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BSci.%2BFood%2BAgric.%26rft.volume%253D93%26rft.spage%253D1233%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fjsfa.5890%26rft_id%253Dinfo%253Apmid%252F23011944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/jsfa.5890&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23011944&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamesch</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berardini</span>  <span class="cit-name-given-names">T. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarbreck</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilks</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D1202</span>–<span class="cit-lpage">D1210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D40%26rft.spage%253DD1202%26rft_id%253Dinfo%253Apmid%252F22140109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22140109&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300179&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.24" data-doi="10.1038/ng.2281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shannon</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yeh</span>  <span class="cit-name-given-names">C.-T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Parallel domestication of the <em>Shattering1</em> genes in cereals.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">720</span>–<span class="cit-lpage">724</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLin%26rft.auinit1%253DZ.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D720%26rft.epage%253D724%26rft.atitle%253DParallel%2Bdomestication%2Bof%2Bthe%2BShattering1%2Bgenes%2Bin%2Bcereals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2281%26rft_id%253Dinfo%253Apmid%252F22581231%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22581231&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.25" data-doi="10.1093/bioinformatics/bts444"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GAPIT: genome association and prediction integrated tool.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2397</span>–<span class="cit-lpage">2399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts444%26rft_id%253Dinfo%253Apmid%252F22796960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTgvMjM5NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.26" data-doi="10.1371/journal.pone.0028145"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Struchalin</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Duijn</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hofman</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Uitterlinden</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Detecting low frequent loss-of-function alleles in genome wide association studies with red hair color as example.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e28145</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLiu%26rft.auinit1%253DF.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253De28145%26rft.epage%253De28145%26rft.atitle%253DDetecting%2Blow%2Bfrequent%2Bloss-of-function%2Balleles%2Bin%2Bgenome%2Bwide%2Bassociation%2Bstudies%2Bwith%2Bred%2Bhair%2Bcolor%2Bas%2Bexample.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0028145%26rft_id%253Dinfo%253Apmid%252F22140526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0028145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22140526&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.27" data-doi="10.1007/s00122-010-1392-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mace</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Location of major effect genes in sorghum (<em>Sorghum bicolor</em> (L.) Moench).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">1339</span>–<span class="cit-lpage">1356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DMace%26rft.auinit1%253DE.%2BS.%26rft.volume%253D121%26rft.issue%253D7%26rft.spage%253D1339%26rft.epage%253D1356%26rft.atitle%253DLocation%2Bof%2Bmajor%2Beffect%2Bgenes%2Bin%2Bsorghum%2B%2528Sorghum%2Bbicolor%2B%2528L.%2529%2BMoench%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1392-8%26rft_id%253Dinfo%253Apmid%252F20585750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1392-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20585750&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.28" data-doi="10.1073/pnas.36.6.344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McClintock</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1950</span> <span class="cit-article-title">The origin and behavior of mutable loci in maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">344</span>–<span class="cit-lpage">355</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DMcClintock%26rft.auinit1%253DB.%26rft.volume%253D36%26rft.issue%253D6%26rft.spage%253D344%26rft.epage%253D355%26rft.atitle%253DThe%2Borigin%2Band%2Bbehavior%2Bof%2Bmutable%2Bloci%2Bin%2Bmaize%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.36.6.344%26rft_id%253Dinfo%253Apmid%252F15430309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiIzNi82LzM0NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA4NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.29" data-doi="10.1016/j.foodres.2012.07.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moraes</span>  <span class="cit-name-given-names">É. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Natal</span>  <span class="cit-name-given-names">D. I. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Queiroz</span>  <span class="cit-name-given-names">V. A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffert</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cecon</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Sorghum genotype may reduce low-grade inflammatory response and oxidative stress and maintains jejunum morphology of rats fed a hyperlipidic diet.</span> <abbr class="cit-jnl-abbrev">Food Res. Int.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">559</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFood%2BRes.%2BInt.%26rft.volume%253D49%26rft.spage%253D553%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.foodres.2012.07.029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.foodres.2012.07.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.30" data-doi="10.1073/pnas.1215985110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morris</span>  <span class="cit-name-given-names">G. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deshpande</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hash</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population genomic and genome-wide association studies of agroclimatic traits in sorghum.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">453</span>–<span class="cit-lpage">458</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1215985110%26rft_id%253Dinfo%253Apmid%252F23267105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMTAvMi80NTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.31" data-doi="10.1073/pnas.1106212108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klein</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morishige</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brady</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rooney</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Coincident light and clock regulation of <em>pseudoresponse regulator protein 37</em> (<em>PRR37</em>) controls photoperiodic flowering in sorghum.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">16469</span>–<span class="cit-lpage">16474</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1106212108%26rft_id%253Dinfo%253Apmid%252F21930910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzM5LzE2NDY5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nilsson-Ehle</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1909</span> <span class="cit-article-title">Kreuzunguntersuchungen an Häfer und Weizen.</span> <abbr class="cit-jnl-abbrev">Lunds Universitets Arsskrift</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">122</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLunds%2BUniversitets%2BArsskrift%26rft.volume%253D5%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.33" data-doi="10.1093/hmg/ddq368"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orozco</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeggini</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Synthetic associations in the context of genome-wide association scan signals.</span> <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">R137</span>–<span class="cit-lpage">R144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddq368%26rft_id%253Dinfo%253Apmid%252F20805105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiaG1nIjtzOjU6InJlc2lkIjtzOjEwOiIxOS9SMi9SMTM3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.34" data-doi="10.1038/nature07723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowers</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bruggmann</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubchak</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grimwood</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The <em>Sorghum bicolor</em> genome and the diversification of grasses.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">556</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPaterson%26rft.auinit1%253DA.%2BH.%26rft.volume%253D457%26rft.issue%253D7229%26rft.spage%253D551%26rft.epage%253D556%26rft.atitle%253DThe%2BSorghum%2Bbicolor%2Bgenome%2Band%2Bthe%2Bdiversification%2Bof%2Bgrasses.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07723%26rft_id%253Dinfo%253Apmid%252F19189423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07723&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19189423&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262852200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.35" data-doi="10.1016/j.plantsci.2011.05.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petroni</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tonelli</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Recent advances on the regulation of anthocyanin synthesis in reproductive organs.</span> <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">219</span>–<span class="cit-lpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BSci%26rft.aulast%253DPetroni%26rft.auinit1%253DK.%26rft.volume%253D181%26rft.issue%253D3%26rft.spage%253D219%26rft.epage%253D229%26rft.atitle%253DRecent%2Badvances%2Bon%2Bthe%2Bregulation%2Bof%2Banthocyanin%2Bsynthesis%2Bin%2Breproductive%2Borgans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plantsci.2011.05.009%26rft_id%253Dinfo%253Apmid%252F21763532%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plantsci.2011.05.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21763532&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293552100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.36" data-doi="10.1534/genetics.110.121665"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjálmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Conditions under which genome-wide association studies will be positively misleading.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">1045</span>–<span class="cit-lpage">1052</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.121665%26rft_id%253Dinfo%253Apmid%252F20813880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ni8zLzEwNDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.37" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Inference of population structure using multilocus genotype data.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>–<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2085.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Core Team</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.39" data-doi="10.1007/s001220050936"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rami</span>  <span class="cit-name-given-names">J.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dufour</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trouche</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fliedel</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mestres</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Quantitative trait loci for grain quality, productivity, morphological and agronomical traits in sorghum (<em>Sorghum bicolor</em> L. Moench).</span> <abbr class="cit-jnl-abbrev">TAG Theoret. Appl. Genet</abbr> .<span class="cit-vol">97</span>: <span class="cit-fpage">605</span>–<span class="cit-lpage">616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG%2BTheoret.%2BAppl.%2BGenet%26rft.volume%253D97%26rft.spage%253D605%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050936&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2085.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rooney</span>  <span class="cit-name-given-names">W. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genetics and cytogenetics</span>, pp. <span class="cit-fpage">261</span>–<span class="cit-lpage">307</span> in <span class="cit-source"><em>Sorghum: Origin</em>, <em>History</em>, <em>Technology</em>, <em>and Production</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">C. W.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Frederiksen</span>  <span class="cit-name-given-names">R. A.</span></span>. <span class="cit-publ-name">John Wiley and Sons</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.41" data-doi="10.1038/nrg2760"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jewett</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szpiech</span>  <span class="cit-name-given-names">Z. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jankovic</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide association studies in diverse populations.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">356</span>–<span class="cit-lpage">366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DRosenberg%26rft.auinit1%253DN.%2BA.%26rft.volume%253D11%26rft.issue%253D5%26rft.spage%253D356%26rft.epage%253D366%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bin%2Bdiverse%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2760%26rft_id%253Dinfo%253Apmid%252F20395969%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2760&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20395969&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276771400013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sax</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1923</span> <span class="cit-article-title">The association of size differences with seed-coat pattern and pigmentation in <em>Phaseolus vulgaris</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">552</span>–<span class="cit-lpage">560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSax%26rft.auinit1%253DK.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253D552%26rft.epage%253D560%26rft.atitle%253DTHE%2BASSOCIATION%2BOF%2BSIZE%2BDIFFERENCES%2BWITH%2BSEED-COAT%2BPATTERN%2BAND%2BPIGMENTATION%2BIN%2BPHASEOLUS%2BVULGARIS%26rft_id%253Dinfo%253Apmid%252F17246026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiI4LzYvNTUyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.43" data-doi="10.1371/journal.pone.0017855"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeling</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genes identified by visible mutant phenotypes show increased bias toward one of two subgenomes of maize.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e17855</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSchnable%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253De17855%26rft.epage%253De17855%26rft.atitle%253DGenes%2Bidentified%2Bby%2Bvisible%2Bmutant%2Bphenotypes%2Bshow%2Bincreased%2Bbias%2Btoward%2Bone%2Bof%2Btwo%2Bsubgenomes%2Bof%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0017855%26rft_id%253Dinfo%253Apmid%252F21423772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0017855&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21423772&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.44" data-doi="10.1038/ng.2314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Segura</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilhjálmsson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Platt</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korte</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seren</span>  <span class="cit-name-given-names">Ü.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An efficient multi-locus mixed-model approach for genome-wide association studies in structured populations.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">825</span>–<span class="cit-lpage">830</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DSegura%26rft.auinit1%253DV.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D825%26rft.epage%253D830%26rft.atitle%253DAn%2Befficient%2Bmulti-locus%2Bmixed-model%2Bapproach%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies%2Bin%2Bstructured%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2314%26rft_id%253Dinfo%253Apmid%252F22706313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22706313&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.45" data-doi="10.2134/agronj1946.00021962003800040008x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1946</span> <span class="cit-article-title">A second factor for subcoat in sorghum seed.</span> <abbr class="cit-jnl-abbrev">Agron. J.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">340</span>–<span class="cit-lpage">342</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgronomy%2BJournal%26rft.stitle%253DAgron.%2BJ.%26rft.aulast%253DStephens%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D38%26rft.issue%253D4%26rft.spage%253D340%26rft.epage%253D342%26rft.atitle%253DA%2BSecond%2BFactor%2Bfor%2BSubcoat%2Bin%2BSorghum%2BSeed%26rft_id%253Dinfo%253Adoi%252F10.2134%252Fagronj1946.00021962003800040008x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2134/agronj1946.00021962003800040008x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.46" data-doi="10.1073/pnas.0912396106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Allelic diversities in rice starch biosynthesis lead to a diverse array of rice eating and cooking qualities.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">21760</span>–<span class="cit-lpage">21765</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0912396106%26rft_id%253Dinfo%253Apmid%252F20018713%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzUxLzIxNzYwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vinall</span>  <span class="cit-name-given-names">H. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cron</span>  <span class="cit-name-given-names">A. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1921</span> <span class="cit-article-title">Improvement of sorghums by hybridization.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">435</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DVINALL%26rft.auinit1%253DH.%2BN.%26rft.volume%253D12%26rft.issue%253D10%26rft.spage%253D435%26rft.epage%253D443%26rft.atitle%253DIMPROVEMENT%2BOF%2BSORGHUMS%2BBY%2BHYBRIDIZATION%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo2OiJqaGVyZWQiO3M6NToicmVzaWQiO3M6OToiMTIvMTAvNDM1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2085.48"><div class="cit-metadata unstructured">Vinayan, M. T., 2010 <em>Genetic Architecture of Spotted Stem Borer Resistance in Sorghum as Inferred From QTL Mapping and Synteny with the Maize Genome</em>, Ph.D. Thesis, Tamil Nadu Agricultural University, Tamil Nadu, India.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.49" data-doi="10.1016/j.ajhg.2011.11.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Visscher</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Five years of GWAS discovery.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">7</span>–<span class="cit-lpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DVisscher%26rft.auinit1%253DP.%2BM.%26rft.volume%253D90%26rft.issue%253D1%26rft.spage%253D7%26rft.epage%253D24%26rft.atitle%253DFive%2Byears%2Bof%2BGWAS%2Bdiscovery.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2011.11.029%26rft_id%253Dinfo%253Apmid%252F22243964%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2011.11.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22243964&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.50" data-doi="10.1104/pp.126.2.485"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winkel-Shirley</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Flavonoid biosynthesis. a colorful model for genetics, biochemistry, cell biology, and biotechnology.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">485</span>–<span class="cit-lpage">493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DWinkel-Shirley%26rft.auinit1%253DB.%26rft.volume%253D126%26rft.issue%253D2%26rft.spage%253D485%26rft.epage%253D493%26rft.atitle%253DFlavonoid%2BBiosynthesis.%2BA%2BColorful%2BModel%2Bfor%2BGenetics%252C%2BBiochemistry%252C%2BCell%2BBiology%252C%2Band%2BBiotechnology%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.126.2.485%26rft_id%253Dinfo%253Apmid%252F11402179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMjYvMi80ODUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.51" data-doi="10.1073/pnas.1201700109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Presence of tannins in sorghum grains is conditioned by different natural alleles of <em>Tannin1</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA.</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">10281</span>–<span class="cit-lpage">10286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1201700109%26rft_id%253Dinfo%253Apmid%252F22699509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzI2LzEwMjgxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.52" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">I. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.53" data-doi="10.1038/ng.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">C.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todhunter</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiwari</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mixed linear model approach adapted for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D355%26rft.epage%253D360%26rft.atitle%253DMixed%2Blinear%2Bmodel%2Bapproach%2Badapted%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.546%26rft_id%253Dinfo%253Apmid%252F20208535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208535&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.54" data-doi="10.1104/pp.110.155432"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dixon</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The mysteries of proanthocyanidin transport and polymerization.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.110.155432%26rft_id%253Dinfo%253Apmid%252F20388668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTMvMi80MzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwODUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.55" data-doi="10.1038/ncomms1467"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tung</span>  <span class="cit-name-given-names">C.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eizenga</span>  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ali</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association mapping reveals a rich genetic architecture of complex traits in <em>Oryza sativa</em>.</span> <abbr class="cit-jnl-abbrev">Nat Commun</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DZhao%26rft.auinit1%253DK.%26rft.volume%253D2%26rft.spage%253D467%26rft.epage%253D467%26rft.atitle%253DGenome-wide%2Bassociation%2Bmapping%2Breveals%2Ba%2Brich%2Bgenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits%2Bin%2BOryza%2Bsativa.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1467%26rft_id%253Dinfo%253Apmid%252F21915109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1467&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21915109&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2085.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2085.56" data-doi="10.1534/genetics.108.098863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Nonmetric multidimensional scaling corrects for population structure in association mapping with different sample types.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">875</span>–<span class="cit-lpage">888</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.098863%26rft_id%253Dinfo%253Apmid%252F19414565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzMvODc1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDg1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/11/2085.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/2069" title="Allelic Variation in a Cellulose Synthase Gene (PtoCesA4) Associated with Growth and Wood Properties in Populus tomentosa" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/11/2095" title="A Genome-Wide Association Study Reveals Genes Associated with Fusarium Ear Rot Resistance in a Maize Core Diversity Panel" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/11"><span>Volume 3 Issue 11, November 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/11.cover-source.jpg?itok=A7iZ1ztq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/2085" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Vg-9syKiKxkILd_-badWUrFsUOhxiYksm__aPJRbjBA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/11/2085" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/11/2085" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/11/2085" class="active" data-icon-position="" data-hide-link-title="0">Dissecting Genome-Wide Association Signals for Loss-of-Function Phenotypes in Sorghum Flavonoid Pigmentation Traits</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459579" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-EfGhRUwvrQBPik6lN13rxwF3e_rGggqubaY51fnbdUM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/2085.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/11/2085" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-VquihwbSftgObsZhiwl_iZP8OX8sblszYNR_edPLJZk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/11/2085" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/2085.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459579" id="node459579--2" data-pisa="ggg;3/11/2085" data-pisa-master="ggg;g3.113.008417" data-apath="/ggg/3/11/2085.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Dissecting Genome-Wide Association Signals for Loss-of-Function Phenotypes in Sorghum Flavonoid Pigmentation Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Geoffrey P. Morris</span>, <span class="highwire-citation-author" data-delta="1">Davina H. Rhodes</span>, <span class="highwire-citation-author" data-delta="2">Zachary Brenton</span>, <span class="highwire-citation-author" data-delta="3">Punna Ramu</span>, <span class="highwire-citation-author" data-delta="4">Vinayan Madhumal Thayil</span>, <span class="highwire-citation-author" data-delta="5">Santosh Deshpande</span>, <span class="highwire-citation-author" data-delta="6">C. Thomas Hash</span>, <span class="highwire-citation-author" data-delta="7">Charlotte Acharya</span>, <span class="highwire-citation-author" data-delta="8">Sharon E. Mitchell</span>, <span class="highwire-citation-author" data-delta="9">Edward S. Buckler</span>, <span class="highwire-citation-author" data-delta="10">Jianming Yu</span> and <span class="highwire-citation-author" data-delta="11">Stephen Kresovich</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2085-2094;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008417 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459579--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Geoffrey P. Morris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Geoffrey%2BP.%2BMorris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morris%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGeoffrey%2BP.%2BMorris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gpmorris@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">gpmorris@ksu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Davina H. Rhodes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Davina%2BH.%2BRhodes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rhodes%20DH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavina%2BH.%2BRhodes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Zachary Brenton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zachary%2BBrenton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brenton%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZachary%2BBrenton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Punna Ramu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>ICRISAT, Patancheru PO, Hyderabad 502 324, Andhra Pradesh, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Punna%2BRamu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramu%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APunna%2BRamu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Vinayan Madhumal Thayil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>ICRISAT, Patancheru PO, Hyderabad 502 324, Andhra Pradesh, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vinayan%2BMadhumal%2BThayil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thayil%20VM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVinayan%2BMadhumal%2BThayil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Santosh Deshpande </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>ICRISAT, Patancheru PO, Hyderabad 502 324, Andhra Pradesh, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Santosh%2BDeshpande%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deshpande%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASantosh%2BDeshpande%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">C. Thomas Hash </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>ICRISAT-Sadoré, BP 12404, Niamey, Niger</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BThomas%2BHash%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hash%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC.%2BThomas%2BHash%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Charlotte Acharya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charlotte%2BAcharya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Acharya%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharlotte%2BAcharya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sharon E. Mitchell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sharon%2BE.%2BMitchell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitchell%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASharon%2BE.%2BMitchell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Edward S. Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>USDA-ARS, Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BS.%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BS.%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jianming Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Agronomy, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianming%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianming%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Stephen Kresovich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BKresovich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kresovich%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BKresovich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/11/2085" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/11/2085&amp;title=Dissecting%20Genome-Wide%20Association%20Signals%20for%20Loss-of-Function%20Phenotypes%20in%20Sorghum%20Flavonoid%20Pigmentation%20Traits" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/11/2085&amp;title=Dissecting%20Genome-Wide%20Association%20Signals%20for%20Loss-of-Function%20Phenotypes%20in%20Sorghum%20Flavonoid%20Pigmentation%20Traits" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/11/2085&amp;title=Dissecting%20Genome-Wide%20Association%20Signals%20for%20Loss-of-Function%20Phenotypes%20in%20Sorghum%20Flavonoid%20Pigmentation%20Traits" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/11/2085&amp;text=Dissecting%20Genome-Wide%20Association%20Signals%20for%20Loss-of-Function%20Phenotypes%20in%20Sorghum%20Flavonoid%20Pigmentation%20Traits" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/11/2085&amp;title=Dissecting%20Genome-Wide%20Association%20Signals%20for%20Loss-of-Function%20Phenotypes%20in%20Sorghum%20Flavonoid%20Pigmentation%20Traits" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/11/2085&amp;t=Dissecting%20Genome-Wide%20Association%20Signals%20for%20Loss-of-Function%20Phenotypes%20in%20Sorghum%20Flavonoid%20Pigmentation%20Traits" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/11/2085&amp;title=Dissecting%20Genome-Wide%20Association%20Signals%20for%20Loss-of-Function%20Phenotypes%20in%20Sorghum%20Flavonoid%20Pigmentation%20Traits" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/11/2085&amp;title=Dissecting%20Genome-Wide%20Association%20Signals%20for%20Loss-of-Function%20Phenotypes%20in%20Sorghum%20Flavonoid%20Pigmentation%20Traits" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459579/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459579" id="node459579--3" data-pisa="ggg;3/11/2085" data-pisa-master="ggg;g3.113.008417" data-apath="/ggg/3/11/2085.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Dissecting Genome-Wide Association Signals for Loss-of-Function Phenotypes in Sorghum Flavonoid Pigmentation Traits</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Geoffrey P. Morris</span>, <span class="highwire-citation-author" data-delta="1">Davina H. Rhodes</span>, <span class="highwire-citation-author" data-delta="2">Zachary Brenton</span>, <span class="highwire-citation-author" data-delta="3">Punna Ramu</span>, <span class="highwire-citation-author" data-delta="4">Vinayan Madhumal Thayil</span>, <span class="highwire-citation-author" data-delta="5">Santosh Deshpande</span>, <span class="highwire-citation-author" data-delta="6">C. Thomas Hash</span>, <span class="highwire-citation-author" data-delta="7">Charlotte Acharya</span>, <span class="highwire-citation-author" data-delta="8">Sharon E. Mitchell</span>, <span class="highwire-citation-author" data-delta="9">Edward S. Buckler</span>, <span class="highwire-citation-author" data-delta="10">Jianming Yu</span> and <span class="highwire-citation-author" data-delta="11">Stephen Kresovich</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2085-2094;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008417 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459579--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Geoffrey P. Morris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Geoffrey%2BP.%2BMorris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morris%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGeoffrey%2BP.%2BMorris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:gpmorris@ksu.edu" class="" data-icon-position="" data-hide-link-title="0">gpmorris@ksu.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Davina H. Rhodes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Davina%2BH.%2BRhodes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rhodes%20DH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavina%2BH.%2BRhodes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Zachary Brenton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zachary%2BBrenton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brenton%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZachary%2BBrenton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Punna Ramu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>ICRISAT, Patancheru PO, Hyderabad 502 324, Andhra Pradesh, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Punna%2BRamu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ramu%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APunna%2BRamu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Vinayan Madhumal Thayil </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>ICRISAT, Patancheru PO, Hyderabad 502 324, Andhra Pradesh, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vinayan%2BMadhumal%2BThayil%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thayil%20VM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVinayan%2BMadhumal%2BThayil%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Santosh Deshpande </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>ICRISAT, Patancheru PO, Hyderabad 502 324, Andhra Pradesh, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Santosh%2BDeshpande%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Deshpande%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASantosh%2BDeshpande%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">C. Thomas Hash </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>ICRISAT-Sadoré, BP 12404, Niamey, Niger</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=C.%2BThomas%2BHash%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hash%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AC.%2BThomas%2BHash%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Charlotte Acharya </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charlotte%2BAcharya%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Acharya%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharlotte%2BAcharya%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sharon E. Mitchell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Institute for Genomic Diversity, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sharon%2BE.%2BMitchell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mitchell%20SE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASharon%2BE.%2BMitchell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Edward S. Buckler </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>USDA-ARS, Department of Plant Breeding and Genetics, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Edward%2BS.%2BBuckler%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Buckler%20ES&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEdward%2BS.%2BBuckler%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Jianming Yu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Agronomy, Iowa State University, Ames, Iowa 50011</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianming%2BYu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianming%2BYu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Stephen Kresovich </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BKresovich%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kresovich%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BKresovich%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2085&amp;rft.epage=2094&amp;rft.atitle=Dissecting%20Genome-Wide%20Association%20Signals%20for%20Loss-of-Function%20Phenotypes%20in%20Sorghum%20Flavonoid%20Pigmentation%20Traits&amp;rft.volume=3&amp;rft.issue=11&amp;rft.date=2013-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Morris%2C+Geoffrey+P.&amp;rft.au=Rhodes%2C+Davina+H.&amp;rft.au=Brenton%2C+Zachary&amp;rft.au=Ramu%2C+Punna&amp;rft.au=Thayil%2C+Vinayan+Madhumal&amp;rft.au=Deshpande%2C+Santosh&amp;rft.au=Hash%2C+C.+Thomas&amp;rft.au=Acharya%2C+Charlotte&amp;rft.au=Mitchell%2C+Sharon+E.&amp;rft.au=Buckler%2C+Edward+S.&amp;rft.au=Yu%2C+Jianming&amp;rft.au=Kresovich%2C+Stephen"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459579/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459579/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459579/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459579/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459579/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459579/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459579/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459579/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459579/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459579/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459579/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459579/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/11/2085" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/11/2085.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/11/2085.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/11/2085.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/11/2085.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
