<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/11/2095" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/11/2095.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/11/2095.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/11/2095.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/11/2095/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/11/2095.atom" />
<meta name="HW.pisa" content="ggg;3/11/2095" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Genome-Wide Association Study Reveals Genes Associated with Fusarium Ear Rot Resistance in a Maize Core Diversity Panel" />
<meta name="DC.Identifier" content="10.1534/g3.113.007328" />
<meta name="DC.Date" content="2013-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Zila et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Fusarium ear rot is a common disease of maize that affects food and feed quality globally. Resistance to the disease is highly quantitative, and maize breeders have difficulty incorporating polygenic resistance alleles from unadapted donor sources into elite breeding populations without having a negative impact on agronomic performance. Identification of specific allele variants contributing to improved resistance may be useful to breeders by allowing selection of resistance alleles in coupling phase linkage with favorable agronomic characteristics. We report the results of a genome-wide association study to detect allele variants associated with increased resistance to Fusarium ear rot in a maize core diversity panel of 267 inbred lines evaluated in two sets of environments. We performed association tests with 47,445 single-nucleotide polymorphisms (SNPs) while controlling for background genomic relationships with a mixed model and identified three marker loci significantly associated with disease resistance in at least one subset of environments. Each associated SNP locus had relatively small additive effects on disease resistance (±1.1% on a 0–100% scale), but nevertheless were associated with 3 to 12% of the genotypic variation within or across environment subsets. Two of three identified SNPs colocalized with genes that have been implicated with programmed cell death. An analysis of associated allele frequencies within the major maize subpopulations revealed enrichment for resistance alleles in the tropical/subtropical and popcorn subpopulations compared with other temperate breeding pools." />
<meta name="DC.Contributor" content="Charles T. Zila" />
<meta name="DC.Contributor" content="L. Fernando Samayoa" />
<meta name="DC.Contributor" content="Rogelio Santiago" />
<meta name="DC.Contributor" content="Ana Butrón" />
<meta name="DC.Contributor" content="James B. Holland" />
<meta name="article:published_time" content="2013-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Genome-Wide Association Study Reveals Genes Associated with Fusarium Ear Rot Resistance in a Maize Core Diversity Panel" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;&lt;i&gt;Fusarium&lt;/i&gt; ear rot is a common disease of maize that affects food and feed quality globally. Resistance to the disease is highly quantitative, and maize breeders have difficulty incorporating polygenic resistance alleles from unadapted donor sources into elite breeding populations without having a negative impact on agronomic performance. Identification of specific allele variants contributing to improved resistance may be useful to breeders by allowing selection of resistance alleles in coupling phase linkage with favorable agronomic characteristics. We report the results of a genome-wide association study to detect allele variants associated with increased resistance to &lt;i&gt;Fusarium&lt;/i&gt; ear rot in a maize core diversity panel of 267 inbred lines evaluated in two sets of environments. We performed association tests with 47,445 single-nucleotide polymorphisms (SNPs) while controlling for background genomic relationships with a mixed model and identified three marker loci significantly associated with disease resistance in at least one subset of environments. Each associated SNP locus had relatively small additive effects on disease resistance (±1.1% on a 0–100% scale), but nevertheless were associated with 3 to 12% of the genotypic variation within or across environment subsets. Two of three identified SNPs colocalized with genes that have been implicated with programmed cell death. An analysis of associated allele frequencies within the major maize subpopulations revealed enrichment for resistance alleles in the tropical/subtropical and popcorn subpopulations compared with other temperate breeding pools.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/11/01" />
<meta name="citation_mjid" content="ggg;3/11/2095" />
<meta name="citation_id" content="3/11/2095" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/11/2095" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/11/2095.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/11/2095.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/11/2095.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.007328" />
<meta name="citation_pmid" content="24048647" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2095" />
<meta name="citation_lastpage" content="2104" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Charles T. Zila" />
<meta name="citation_author_institution" content="Department of Crop Science, North Carolina State University, Raleigh, North Carolina 27695" />
<meta name="citation_author" content="L. Fernando Samayoa" />
<meta name="citation_author_institution" content="Misión Biológica de Galicia, CSIC, Pontevedra, Spain, 36080" />
<meta name="citation_author" content="Rogelio Santiago" />
<meta name="citation_author_institution" content="Misión Biológica de Galicia, CSIC, Pontevedra, Spain, 36080" />
<meta name="citation_author" content="Ana Butrón" />
<meta name="citation_author_institution" content="Misión Biológica de Galicia, CSIC, Pontevedra, Spain, 36080" />
<meta name="citation_author" content="James B. Holland" />
<meta name="citation_author_institution" content="Department of Crop Science, North Carolina State University, Raleigh, North Carolina 27695" />
<meta name="citation_author_institution" content="U.S. Department of Agriculture—Agricultural Research Service, Plant Science Research Unit, Raleigh, North Carolina 27695" />
<meta name="citation_author_email" content="james_holland@ncsu.edu" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. M. Andorf;citation_author=C. J. Lawrence;citation_author=L. C. Harper;citation_author=M. L. Schaeffer;citation_author=D. A. Campbell;citation_title=The Locus Lookup tool at MaizeGDB: identification of genomic regions in maize by integrating sequence information with physical and genetic maps.;citation_volume=26;citation_year=2010;citation_pmid=20124413;citation_doi=10.1093/bioinformatics/btp556" />
<meta name="citation_reference" content="citation_title=Breeding for Quantitative Traits in Plants;citation_title=Phenotypic and genetic variances;citation_author=R. Bernardo;citation_pages=89-116;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=C. Bolduan;citation_author=T. Miedaner;citation_author=W. Schipprack;citation_author=B. S. Dhillon;citation_author=A. E. Melchinger;citation_title=Genetic Variation for Resistance to Ear Rots and Mycotoxins Contamination in Early European Maize Inbred Lines;citation_pages=2019-2028;citation_volume=49;citation_year=2009;citation_issue=6;citation_doi=10.2135/cropsci2008.12.0701" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. J. Bradbury;citation_author=Z. Zhang;citation_author=D. E. Kroon;citation_author=T. M. Casstevens;citation_author=Y. Ramdoss;citation_title=TASSEL: software for association mapping of complex traits in diverse samples.;citation_volume=23;citation_year=2007;citation_pmid=17586829;citation_doi=10.1093/bioinformatics/btm308" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=B. J. Bush;citation_author=M. L. Carson;citation_author=M. A. Cubeta;citation_author=W. M. Hagler;citation_author=G. A. Payne;citation_title=Infection and Fumonisin Production by Fusarium verticillioides in Developing Maize Kernels.;citation_pages=88-93;citation_volume=94;citation_year=2004;citation_issue=1;citation_pmid=18943824;citation_doi=10.1094/PHYTO.2004.94.1.88" />
<meta name="citation_reference" content="citation_journal_title=Proteomics;citation_journal_abbrev=Proteomics;citation_author=S. Chivasa;citation_author=W. J. Simon;citation_author=X.-L. Yu;citation_author=N. Yalpani;citation_author=A. R. Slabas;citation_title=Pathogen elicitor-induced changes in the maize extracellular matrix proteome.;citation_pages=4894-4904;citation_volume=5;citation_year=2005;citation_issue=18;citation_pmid=16281185;citation_doi=10.1002/pmic.200500047" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=M. J. Clements;citation_author=C. E. Kleinschmidt;citation_author=C. M. Maragos;citation_author=J. K. Pataky;citation_author=D. G. White;citation_title=Evaluation of inoculation techniques for Fusarium ear rot and fumonisin contamination of corn.;citation_pages=147-153;citation_volume=87;citation_year=2003;citation_doi=10.1094/PDIS.2003.87.2.147" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=M. J. Clements;citation_author=C. M. Maragos;citation_author=J. K. Pataky;citation_author=D. G. White;citation_title=Sources of resistance to fumonisin accumulation in grain and fusarium ear and kernel rot of corn.;citation_pages=251-260;citation_volume=94;citation_year=2004;citation_issue=3;citation_pmid=18943973;citation_doi=10.1094/PHYTO.2004.94.3.251" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. P. Cook;citation_author=M. D. McMullen;citation_author=J. B. Holland;citation_author=F. Tian;citation_author=P. Bradbury;citation_title=Genetic architecture of maize kernel composition in the nested association mapping and inbred association panels.;citation_volume=158;citation_year=2012;citation_pmid=22135431;citation_doi=10.1104/pp.111.185033" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=B. Cullis;citation_author=A. Smith;citation_author=N. Coombes;citation_title=On the design of early generation variety trials with correlated data.;citation_pages=381-393;citation_volume=11;citation_year=2006;citation_doi=10.1198/108571106X154443" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. S. Dhugga;citation_title=Plant Golgi cell wall synthesis: from genes to enzyme activities.;citation_volume=102;citation_year=2005;citation_pmid=15684049;citation_doi=10.1073/pnas.0409649102" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=J.-Q. Ding;citation_author=X.-M. Wang;citation_author=S. Chander;citation_author=J.-B. Yan;citation_author=J.-S. Li;citation_title=QTL mapping of resistance to Fusarium ear rot using a RIL population in maize.;citation_pages=395-403;citation_volume=22;citation_year=2008;citation_doi=10.1007/s11032-008-9184-4" />
<meta name="citation_reference" content="citation_journal_title=Toxin Rev.;citation_author=M. S. Eller;citation_author=J. B. Holland;citation_author=G. A. Payne;citation_title=Breeding for improved resistance to fumonisin contamination in maize.;citation_pages=371-389;citation_volume=27;citation_year=2008;citation_doi=10.1080/15569540802450326" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=M. S. Eller;citation_author=G. A. Payne;citation_author=J. B. Holland;citation_title=Selection for reduced Fusarium ear rot and fumonisin content in advanced backcross maize lines and their topcross hybrids.;citation_pages=2249-2260;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.11.0683" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics;citation_title=Correlated characters;citation_author=D. S. Falconer;citation_author=T. F. C. Mackay;citation_pages=312-334;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=S. A. Flint-Garcia;citation_author=A. Thuillet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=S. M. Romero;citation_title=Maize association population: a high-resolution platform for quantitative trait locus dissection.;citation_pages=1054-1064;citation_volume=44;citation_year=2005;citation_issue=6;citation_pmid=16359397;citation_doi=10.1111/j.1365-313X.2005.02591.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Int Rev Cell Mol Biol;citation_author=I. Gadjev;citation_author=J. M. Stone;citation_author=T. S. Gechev;citation_title=Programmed cell death in plants: new insights into redox regulation and the role of hydrogen peroxide.;citation_pages=87-144;citation_volume=270;citation_year=2008;citation_pmid=19081535;citation_doi=10.1016/S1937-6448(08)01403-2" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. W. Ganal;citation_author=G. Durstewitz;citation_author=A. Polley;citation_author=A. Bérard;citation_author=E. S. Buckler;citation_title=A large maize (Zea mays L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.;citation_pages=e28334-e28334;citation_volume=6;citation_year=2011;citation_issue=12;citation_pmid=22174790;citation_doi=10.1371/journal.pone.0028334" />
<meta name="citation_reference" content="citation_title=ASReml User Guide Release 3.0;citation_author=A. R. Gilmour;citation_author=B. J. Gogel;citation_author=B. R. Cullis;citation_author=R. Thompson;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=F. Guillon;citation_author=B. Bouchet;citation_author=F. Jamme;citation_author=P. Robert;citation_author=B. Quéméner;citation_title=Brachypodium distachyon grain: characterization of endosperm cell walls.;citation_volume=62;citation_year=2011;citation_pmid=21062963;citation_doi=10.1093/jxb/erq332" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=K. Hématy;citation_author=C. Cherk;citation_author=S. Somerville;citation_title=Host-pathogen warfare at the plant cell wall.;citation_pages=406-413;citation_volume=12;citation_year=2009;citation_issue=4;citation_pmid=19616468;citation_doi=10.1016/j.pbi.2009.06.007" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=J. B. Holland;citation_title=Genetic architecture of complex traits in plants.;citation_pages=156-161;citation_volume=10;citation_year=2007;citation_issue=2;citation_pmid=17291822;citation_doi=10.1016/j.pbi.2007.01.003" />
<meta name="citation_reference" content="citation_journal_title=Plant and Cell Physiology;citation_journal_abbrev=Plant Cell Physiol;citation_author=A. Ishikawa;citation_author=H. Tanaka;citation_author=M. Nakai;citation_author=T. Asahi;citation_title=Deletion of a Chaperonin 60{beta} Gene Leads to Cell Death in the Arabidopsis lesion initiation 1 Mutant;citation_pages=255-261;citation_volume=44;citation_year=2003;citation_issue=3;citation_pmid=12668771;citation_doi=10.1093/pcp/pcg031" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_author=S. B. King;citation_author=G. E. Scott;citation_title=Genotypic differences in maize to kernel infection by Fusarium moniliforme.;citation_pages=1245-1247;citation_volume=71;citation_year=1981" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. H. Liepman;citation_author=C. G. Wilkerson;citation_author=K. Keegstra;citation_title=Expression of cellulose synthase-like (Csl) genes in insect cells reveals that CslA family members encode mannan synthases.;citation_volume=102;citation_year=2005;citation_pmid=15647349;citation_doi=10.1073/pnas.0409179102" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. Lu;citation_author=T. Shah;citation_author=Z. Hao;citation_author=S. Taba;citation_author=S. Zhang;citation_title=Comparative SNP and haplotype analysis reveals a higher genetic diversity and rapider LD decay in tropical than temperate germplasm in maize.;citation_pages=e24861-e24861;citation_volume=6;citation_year=2011;citation_issue=9;citation_pmid=21949770;citation_doi=10.1371/journal.pone.0024861" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. A. Manolio;citation_author=F. S. Collins;citation_author=N. J. Cox;citation_author=D. B. Goldstein;citation_author=L. A. Hindorff;citation_title=Finding the missing heritability of complex diseases.;citation_pages=747-753;citation_volume=461;citation_year=2009;citation_issue=7265;citation_pmid=19812666;citation_doi=10.1038/nature08494" />
<meta name="citation_reference" content="citation_journal_title=Advances in experimental medicine and biology;citation_journal_abbrev=Adv Exp Med Biol;citation_author=W. F. Marasas;citation_title=Fumonisins: history, world-wide occurrence and impact.;citation_pages=1-17;citation_volume=392;citation_year=1996;citation_pmid=8850601;citation_doi=10.1007/978-1-4899-1379-1_1" />
<meta name="citation_reference" content="citation_journal_title=Plant Breed.;citation_author=Á. Mesterházy;citation_author=M. Lemmens;citation_author=L. M. Reid;citation_title=Breeding for resistance to ear rots caused by Fusarium spp. in maize—a review.;citation_pages=1-19;citation_volume=131;citation_year=2012;citation_doi=10.1111/j.1439-0523.2011.01936.x" />
<meta name="citation_reference" content="citation_title=Mycotoxins in Grain: Compounds Other Than Aflatoxin;citation_title=The biosynthetic diversity of secondary metabolites; Epidemiology of Fusarium ear diseases of cereals;citation_author=J. D. Miller;citation_author=H. L. Trenholm;citation_pages=3-36;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Plant Dis.;citation_author=C. Nankam;citation_author=J. K. Pataky;citation_title=Resistance to kernel infection by Fusarium moniliforme in the sweet corn inbred IL 125b.;citation_pages=593-598;citation_volume=80;citation_year=1996;citation_doi=10.1094/PD-80-0593" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. A. Olukolu;citation_author=A. Negeri;citation_author=R. Dhawan;citation_author=B. P. Venkata;citation_author=P. Sharma;citation_title=A connected set of genes associated with programmed cell death implicated in controlling the hypersensitive response in maize.;citation_volume=193;citation_year=2013;citation_pmid=23222653;citation_doi=10.1534/genetics.112.147595" />
<meta name="citation_reference" content="citation_journal_title=Agrociencia;citation_author=D. Pérez-Brito;citation_author=D. Jeffers;citation_author=D. González-de-León;citation_author=M. Khairallah;citation_author=M. Cortés-Cruz;citation_title=QTL mapping of Fusarium moniliforme ear rot resistance in highland maize, Mexico.;citation_pages=181-196;citation_volume=35;citation_year=2001" />
<meta name="citation_reference" content="citation_journal_title=Crop Prot.;citation_author=D. A. Presello;citation_author=G. Botta;citation_author=J. Iglesias;citation_author=G. H. Eyhérabide;citation_title=Effect of disease severity on yield and grain fumonisin concentration of maize hybrids inoculated with Fusarium verticillioides.;citation_pages=572-576;citation_volume=27;citation_year=2008;citation_doi=10.1016/j.cropro.2007.08.015" />
<meta name="citation_reference" content="citation_title=R: A Language and Environment for Statistical Computing;citation_year=2013" />
<meta name="citation_reference" content="citation_title=Screening Maize for Resistance to Gibberella Ear Rot;citation_author=L. M. Reid;citation_author=R. I. Hamilton;citation_author=D. E. Mather;citation_year=1996" />
<meta name="citation_reference" content="citation_title=Screening Corn for Resistance to Common Diseases in Canada;citation_author=L. M. Reid;citation_author=X. Zhu;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Int. J. Plant Sci.;citation_author=D. L. Remington;citation_author=M. D. Purugganan;citation_title=Candidate genes, quantitative trait loci, and functional trait evolution in plants.;citation_pages=S7-S20;citation_volume=164;citation_year=2003;citation_doi=10.1086/367812" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=L. A. Robertson;citation_author=C. E. Kleinschmidt;citation_author=D. G. White;citation_author=G. A. Payne;citation_author=C. M. Maragos;citation_title=Heritabilities and Correlations of Fusarium Ear Rot Resistance and Fumonisin Contamination Resistance in Two Maize Populations;citation_pages=353-361;citation_volume=46;citation_year=2006;citation_issue=1;citation_doi=10.2135/cropsci2005.0139" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=L. A. Robertson-Hoyt;citation_author=M. P. Jines;citation_author=P. J. Balint-Kurti;citation_author=C. E. Kleinschmidt;citation_author=D. G. White;citation_title=QTL Mapping for Fusarium Ear Rot and Fumonisin Contamination Resistance in Two Maize Populations;citation_pages=1734-1743;citation_volume=46;citation_year=2006;citation_issue=4;citation_doi=10.2135/cropsci2005.12-0450" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=Mdel C. Rodríguez-Gacio;citation_author=R. Iglesias-Fernández;citation_author=P. Carbonero;citation_author=Á. J. Matilla;citation_title=Softening-up mannan-rich cell walls.;citation_volume=63;citation_year=2012;citation_pmid=22553284;citation_doi=10.1093/jxb/ers096" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. C. Romay;citation_author=M. J. Millard;citation_author=J. C. Glaubitz;citation_author=J. A. Peiffer;citation_author=K. L. Swarts;citation_title=Comprehensive genotyping of the USA national maize inbred seed bank.;citation_pages=R55;citation_volume=14;citation_year=2013;citation_pmid=23759205;citation_doi=10.1186/gb-2013-14-6-r55" />
<meta name="citation_reference" content="citation_title=SAS 9.2 Intelligence Platform: System Administration Guide;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. S. Schnable;citation_author=D. Ware;citation_author=R. S. Fulton;citation_author=J. C. Stein;citation_author=F. Wei;citation_title=The B73 Maize Genome: Complexity, Diversity, and Dynamics;citation_pages=1112-1115;citation_volume=326;citation_year=2009;citation_issue=5956;citation_pmid=19965430;citation_doi=10.1126/science.1178534" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=R. S. Sekhon;citation_author=H. Lin;citation_author=K. L. Childs;citation_author=C. N. Hansey;citation_author=C. R. Buell;citation_title=Genome-wide atlas of transcription during maize development.;citation_pages=553-563;citation_volume=66;citation_year=2011;citation_issue=4;citation_pmid=21299659;citation_doi=10.1111/j.1365-313X.2011.04527.x" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. D. Storey;citation_author=R. Tibshirani;citation_title=Statistical significance for genomewide studies.;citation_volume=100;citation_year=2003;citation_pmid=12883005;citation_doi=10.1073/pnas.1530509100" />
<meta name="citation_reference" content="citation_journal_title=Molecular plant-microbe interactions :  MPMI;citation_journal_abbrev=Mol Plant Microbe Interact;citation_author=H. Tsunezuka;citation_author=M. Fujiwara;citation_author=T. Kawasaki;citation_author=K. Shimamoto;citation_title=Proteome analysis of programmed cell death and defense signaling using the rice lesion mimic mutant cdr2.;citation_pages=52-59;citation_volume=18;citation_year=2005;citation_issue=1;citation_pmid=15672818;citation_doi=10.1094/MPMI-18-0052" />
<meta name="citation_reference" content="citation_journal_title=Analytical and bioanalytical chemistry;citation_journal_abbrev=Anal Bioanal Chem;citation_author=H. P. van Egmond;citation_author=R. C. Schothorst;citation_author=M. A. Jonker;citation_title=Regulations relating to mycotoxins in food: perspectives in a global and European context.;citation_pages=147-157;citation_volume=389;citation_year=2007;citation_issue=1;citation_pmid=17508207;citation_doi=10.1007/s00216-007-1317-9" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=D. Van Inghelandt;citation_author=J. C. Reif;citation_author=B. S. Dhillon;citation_author=P. Flament;citation_author=A. E. Melchinger;citation_title=Extent and genome-wide distribution of linkage disequilibrium in commercial maize germplasm.;citation_pages=11-20;citation_volume=123;citation_year=2011;citation_issue=1;citation_pmid=21404061;citation_doi=10.1007/s00122-011-1562-3" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=W. Wang;citation_author=B. Vinocur;citation_author=O. Shoseyov;citation_author=A. Altman;citation_title=Role of plant heat-shock proteins and molecular chaperones in the abiotic stress response.;citation_pages=244-252;citation_volume=9;citation_year=2004;citation_issue=5;citation_pmid=15130550;citation_doi=10.1016/j.tplants.2004.03.006" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. J. Wisser;citation_author=J. M. Kolkman;citation_author=M. E. Patzoldt;citation_author=J. B. Holland;citation_author=J. Yu;citation_title=Multivariate analysis of maize disease resistances suggests a pleiotropic genetic basis and implicates a GST gene.;citation_volume=108;citation_year=2011;citation_pmid=21490302;citation_doi=10.1073/pnas.1011739108" />
<meta name="citation_reference" content="citation_journal_title=Current Opinion in Biotechnology;citation_journal_abbrev=Current Opinion in Biotechnology;citation_author=J. Yu;citation_author=E. S. Buckler;citation_title=Genetic association mapping and genome organization of maize.;citation_pages=155-160;citation_volume=17;citation_year=2006;citation_issue=2;citation_pmid=16504497;citation_doi=10.1016/j.copbio.2006.02.003" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yu;citation_author=G. Pressoir;citation_author=W. H. Briggs;citation_author=I. Vroh Bi;citation_author=M. Yamasaki;citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.;citation_pages=203-208;citation_volume=38;citation_year=2006;citation_issue=2;citation_pmid=16380716;citation_doi=10.1038/ng1702" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Z. Zhang;citation_author=E. Ersoz;citation_author=C.-Q. Lai;citation_author=R. J. Todhunter;citation_author=H. K. Tiwari;citation_title=Mixed linear model approach adapted for genome-wide association studies.;citation_pages=355-360;citation_volume=42;citation_year=2010;citation_issue=4;citation_pmid=20208535;citation_doi=10.1038/ng.546" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Genome-Wide Association Study Reveals Genes Associated with Fusarium Ear Rot Resistance in a Maize Core Diversity Panel" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="twitter:description" content="Fusarium ear rot is a common disease of maize that affects food and feed quality globally. Resistance to the disease is highly quantitative, and maize breeders have difficulty incorporating polygenic resistance alleles from unadapted donor sources into elite breeding populations without having a negative impact on agronomic performance. Identification of specific allele variants contributing to improved resistance may be useful to breeders by allowing selection of resistance alleles in coupling phase linkage with favorable agronomic characteristics. We report the results of a genome-wide association study to detect allele variants associated with increased resistance to Fusarium ear rot in a maize core diversity panel of 267 inbred lines evaluated in two sets of environments. We performed association tests with 47,445 single-nucleotide polymorphisms (SNPs) while controlling for background genomic relationships with a mixed model and identified three marker loci significantly associated with disease resistance in at least one subset of environments. Each associated SNP locus had relatively small additive effects on disease resistance (±1.1% on a 0–100% scale), but nevertheless were associated with 3 to 12% of the genotypic variation within or across environment subsets. Two of three identified SNPs colocalized with genes that have been implicated with programmed cell death. An analysis of associated allele frequencies within the major maize subpopulations revealed enrichment for resistance alleles in the tropical/subtropical and popcorn subpopulations compared with other temperate breeding pools." />
<meta name="og-title" property="og:title" content="A Genome-Wide Association Study Reveals Genes Associated with Fusarium Ear Rot Resistance in a Maize Core Diversity Panel" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/11/2095" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Fusarium ear rot is a common disease of maize that affects food and feed quality globally. Resistance to the disease is highly quantitative, and maize breeders have difficulty incorporating polygenic resistance alleles from unadapted donor sources into elite breeding populations without having a negative impact on agronomic performance. Identification of specific allele variants contributing to improved resistance may be useful to breeders by allowing selection of resistance alleles in coupling phase linkage with favorable agronomic characteristics. We report the results of a genome-wide association study to detect allele variants associated with increased resistance to Fusarium ear rot in a maize core diversity panel of 267 inbred lines evaluated in two sets of environments. We performed association tests with 47,445 single-nucleotide polymorphisms (SNPs) while controlling for background genomic relationships with a mixed model and identified three marker loci significantly associated with disease resistance in at least one subset of environments. Each associated SNP locus had relatively small additive effects on disease resistance (±1.1% on a 0–100% scale), but nevertheless were associated with 3 to 12% of the genotypic variation within or across environment subsets. Two of three identified SNPs colocalized with genes that have been implicated with programmed cell death. An analysis of associated allele frequencies within the major maize subpopulations revealed enrichment for resistance alleles in the tropical/subtropical and popcorn subpopulations compared with other temperate breeding pools." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
    <title>A Genome-Wide Association Study Reveals Genes Associated with Fusarium Ear Rot Resistance in a Maize Core Diversity Panel | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"-LK7dEqBrOSeuheq03sMDqD4yXyLEzSgNx6wvWQNfaQ","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459574","apath":"\/ggg\/3\/11\/2095.atom","pisa":"ggg;3\/11\/2095","ac":{"\/ggg\/3\/11\/2095.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/11\/2095.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/11\/2095"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/11\/2095"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/11\/2095":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022lvVlvWddB4APxAycXYvrJLLlPMdY71YUSMkgYYny8\\\\\\\/AAvJdfq8osIXgV1QTzZ4uIUgBygiSSR8Hw60857CYm3bxd3eQ09bl3eC\\\\\\\/K6vctEAmpL9v\\\\\\\/LNbke1doTTzSnEWphUwUx4CwYW0g99NZvGvJbbGVZD9UWD9lJr+GhIMJfc4JBt8T\\\\\\\/0FDkS8h\\\\\\\/idWzT56EwBNcbwO7G+x\\\\\\\/YpSzlRdeVkvA7JijujIw2Rv23uTYKI5KrxXNX1nUlSNrdjvAETSLEFY3nkXD6lTMF80LJKyslTyE7oEW5lkPFRWL4avQDaoAgV8FEn8AURkFXAbJjIVhz1Rmxe\\\\\\\/0xndR3DQaFiHq5poqv1EWC+U7HyuVkKCyNZ0t0buZLRESpiDLxZCyUqoXu10JxDXXH\\\\\\\/7rRulAwhgZUi+rWw5kXXSMlkLGunjPdomYF9kRS2IboOtohqYGbn1npCbIcGg7JEgWHW7dgEI\\\\\\\/GMRr9ARoyGDz9utbZ1oB0mQ6rvxQxCojdSEewfTZCUqKZVHjOrfcanRBV+LmZgkN4yN\\\\\\\/OhdDsZXSihR6oEYtwzR6rqkt7UAFioIZmSWHzJhqzQnVDEDcCKr81DoPhThVFz+iDQhHXDSnavXWlE3EarjgeoprTII82AC++ZdDUQOS3EoE0lt9fiUYJDlZloSixQ0bderq2VXS1n3SpOgVW0qwviHZUh4wF+bCtGzrdMUB4YCi8pNb8hlCcmo8DABKR64OEaWEIda9rMjrSOXm6gbLqOrcnL7fxwVlKxq+pb9t\\\\\\\/4En3ofkZWJ0IbH6tobi11TAhxOOjdYGcR+8D5\\\\\\\/FBiMhRXEJKYa6sGXBIfrxRcCaFGDAJRKjoT2lW3vcZr3990voi2o7AaxIUDdAZsKY24gtTdgyqsFHWL1DrDKvv7EZA+o8iHS+WSbU\\\\\\\/bS2LlsxhvvoDQ7D2VxXgjhtIC\\\\\\\/kCl0avhkyP7xQzvI93fr4hizWS1SrPdCYPb9Hxn6TWxJZWLXGz3t9NFArpJEhVWWPFzyWMH4yD+Wr9866YJgsJBzwevoBQKKVShXTFonllfsAhoy\\\\\\\/84jvQMwNB393E16ycx4w+PQpwY3+nUrEN4Q0G1NJn7uTIYVxuXOr2+lB6mdJfIVu4Xir2hqP5DI8bfY8T504\\\\\\\/NVhFaeEX0BD7VQu1FR3dpKPYCsho+6nr8xHPaZgwrf3SRK\\\\\\\/ZA+s8ARvY+vpOp4rN9hK5ZZRJz6jDwHFPObwm4RG636PA0P8CjOGYief6c1dnWi92wIis9\\\\\\\/A1u0lC5SpSD4ROI\\\\\\\/YJUj2OmLqL7N\\\u0022,\\\u0022iv\\\u0022:\\\u0022zTBlptOpHUe3TU+M99Q9+A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002294e750cd0e1e9c62bd3d3f98e65fa5088aec2f6b19d18e7c4ae4c8f98ddf396c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00222RoKH8PPD41H356Cz\\\\\\\/ECNEZFsOH1BqhS1C9y8y2CH4IwrACIVve1iZ8Va6PVb61RXEFUlw8qfGT+FA5LNM\\\\\\\/Aas4ePIan5me\\\\\\\/\\\\\\\/N77ENpklcaTSGkfOaZFfY1ZwSj+5PWqLEtX3Wfz8+lCYvEwSMFLVtCU2f7VWsKgcLzTqFXfscKbjl4a4bcAuoIc2h8LOhj5Hu7a2fuUcBIjFYUsU8VSSrXIcbM2QQSm+N5RVDRSvLfLJr0401YGXwgM99IqVbXs8aOu9PiipmH5qy3cT7xMo857q+rcYs8g4XhCwYM92ufEV8Y3rFbywBOKfhOx2Fir2HVDBF92kb1IxRuUqnz2EhQrlPhAaju2UkqzSvn6xtpJdswj78udBaDg0tRaC83OkZLgIgwUGN2UcVBNSq51sZQJLL\\\\\\\/rK8NtifGyGGPHuHR\\\\\\\/F4Z0+miX7obFNbY4TT+Wg7j4YyQ8alpPTjflEJ6Q6k6zX3v1jVL7KTgK8XV3MlCCUyZqhtGILNbYWUF\\\\\\\/oSgaV2IurKAYH52bSj\\\\\\\/kpt47ivfQ9y9TzaiuerOb+txzelTKzkQN\\\\\\\/Ac8bLvLq1msE9mULHo7vbEAGsT5eDZpb3Wp5h8oYuNWmPD93+6qCxYKEM0G\\\\\\\/kwM4Xydc6X9LvvcC+58kXlo6+GSFaNOu+ib6Z8jkERpzCqIM\\\\\\\/ZtXVaDwmGdGpEDAmBUZwPH2McKK0Cs8OuFEWaDZq0249Y5sw7O2DaF8tYvD6vz6DWRsPaBsvIId2jUDIFHBN\\\\\\\/YKACjvVJ5KMMt23iQ\\\\\\\/XshqK3TpxjYtkKHNj5i8dvCEXfo0uEENyqyNSyDrzMHnOTCykee0Ie6fi84Nu\\\\\\\/8M1WEOIRdfCaZAkpYPafp144olAOtcqX0kwnSjVwRabo+6xa1vJSFFqtRUgfKS8xyut2ONW79dOOtdj5WpPMM2CGd4iVxSk8800OCYxHYx2K5uhrFiMpCrsfta7NEwaJlBbSjdX3pAW8GYZOUjtyKlkmq58scum5jbfvuXpKY\\\\\\\/4JJ7opprHBAviYk4dNZk+2GDcgyAG+9sYoj5xizXTrkS0r7BZWWBa4Tn55cTrMamP41wSGROJOQraXMeJLxo1CfQXx+blRvVR\\\\\\\/++WEhO58CYJ7B7GvFUj9tsaNdIvfdz01We3RY85CQjOhio2bE7hGbpDx7h9cYHNX7KPLiL7x+rXjtP8z7uNxeXpgjKwAqt4n3zgjgfefMsAW3Bxhu4nj8anWnxd0s5LyMWZXJaP9oDyQH8OZ8FAfBK7FlRwslx6+HHYzhUDESwbfitdaQbDFt80bCGHrQJJPIawBwzGsW4\\\\\\\/xetsDa6Zq3uAMIyo7V6eY2TprDHQ9TAg8gufwUfcU0JhtGyYP7xQC77KWEM2FnPrRD5tFSemoEFzbw2BmGzg2kK\\\\\\\/9HkrRNVZv7ffIiV1IPOLwjIge8DXsZ57Av3BEGjct2\\\\\\\/rsi8P1gDoVIXNJ1XATAa4QDw+tUsL9uBMwIrc44L+OOS8\\\\\\\/tNka\\\\\\\/eVlezdFY4eF2zxwgtiCL2xaMZv52n\\\\\\\/IkvHRBQR4bzP\\\\\\\/9vC\\\\\\\/NXKo6cDz02+6tF4RwSf7e4S9roUSYv3lcbQuMxcwNzf5HnCdNJtVUoPd9lT2F0vp0vFWfoIh5ZZo3PvuQMIiTKVII+so1s\\\\\\\/UEj3Q1Ik4GNQGH\\\\\\\/B+9UmGMmARLFR0XIUJTauJYR6fJzWVvE2RU7lKd6dYSswRzX2sHd57uXgnVgR\\\\\\\/hhkrRh8aQWFXoBuHTWLEUMKwqyf9vhkCWWnAXqUcg5OCeESiZfX2DgdWMKOQJO27Em11irNxM+ehNLorZuIYSmhIfPFgN1Yiuzl87aY7Hiw5IKABBP72p9x2kKd1oy+uXeavDtcLaJRbrF3ZbHbj4udB0dYrBOOzOlbrPlOD\\\\\\\/CvsO51\\\\\\\/NXuzcOkj8yQuwIV7HtJos0eGLhez\\\\\\\/U1HIV1tZpZ0iT6usxT\\\\\\\/v8ivDj2ulWFn3QfU2ueVky15NFG7tg9+NYBQL82aVO\\\\\\\/dvHj4TOKJksDWxqcgRaN5K4qi7jtrfxsBvjI14gButbT5Jm9MBaRkvjZ8r2WwfyWcxNPBYnDv1IRoKLXLmWD3MCOmafI8iyzyb4xNC07b35N5SmKW1Aa3UiaX0iwzP8XprJCkIgLpbyxS03aq4HaqTKZ\\\\\\\/BIXkoK9G4TpmsaWq5gyhSz05L21zuUq2wgjbaEn9sylz0WB3o4Kykl\\\\\\\/KxljtvqXfSE\\\\\\\/g8En14Xa18\\\\\\\/yWp5e\\\\\\\/Du6YaNhWdyGvKkkGCmDmrAyXocm39AKIpCufRvkPkffEAI5o6DJYoPTLOA3unRqweC4UQY\\\\\\\/fImSogEFJQpusrfeTxb7Ba9hhFmUsRMaLJRFrgMc9hdQRZXNJiEhfjw1kO8Xn9DytcR88gtvojjfqaaAZvgXvYkKmUorM3bFpPdQW+\\\\\\\/wVzzgMzbsigWgwAW8VO65H7zcsfKtXUh6k8MYuzmoMM72cJjLT+eY\\\\\\\/rCVC\\\\\\\/aWl9DYLKgIEyERoGOnZ7VDHFV\\\\\\\/FiYKqZ1a\\\\\\\/cJgNFyVu7CC2Z52W495fd3Ci\\\\\\\/3WWHdSwo7clQVsnFk\\\\\\\/haDVTXpRBjPqc+KZ6u3uX9BSKLUS2WBfszNvxanMAy\\\\\\\/GZpNQ\\\\\\\/1AgRo0zoM3q1oSwCXnbySp1FrxE7u5H+1svEEp1gvPesUqOS+u1mRyhw3X8l60HoLAg7WY3n1JXc2aW7kCmwlanequ8+e4zwZJLAfmcAZXhIBIEigRc+2lWh1quE1DOlO4\\\\\\\/fyaIf8lxJu+GdKZsACm\\\\\\\/3CegzLK6NlyR+ingXJdZL02nDsnw73vBTESsOAoXrh40Pe8dOao1RGu2ywm7oqTKZDSpZ7SJa3llFJeWM3h5SAEvrGX8ec6Kb2\\\\\\\/TPqYyhGOyykolHCvu6knz+Wtt2AeR7gJ9zi4NkKe\\\\\\\/XBc+v30RKAhYRhNPWO707DlWUPJIATCjVxWz+tqbYyYDnlzta7NV33deLQiX+zjDBB8d0B5SSj7bbuDlxX5fnz2gAodORig\\\\\\\/wbzs04KxprOQSWg5GoQ8iTnR9IwKqsYBACZ\\\\\\\/3Ih26OW5Oi4z72qGk5MygxxHKBgGeKZWrIcnFl3imcw\\\\\\\/BmDurWhuqz2qn70sFjqGavSgs+9YQBEhZnbNjax8WJnqTMOFmVnHqdEsuxGL1CdxEuiizha+4mlhPpXdPO9WKzDTPzEzs+hsiFlF44QkJ3Wj8Tnju+Ws3jaqghsSbpkCtLluYR\\\\\\\/Eboc4s69caoz20ayI1S8joz1r67Zpmdw+GzdUVSZeh+QUDcgliC6+Sa5XLuXHCNhiBTIANTOo2H0rKOL2QKMMMAOpER+km+ekwSx7OrZDEOvdi8T4nQEl1CqRj\\\\\\\/pjL7qhhwTFuqCCh5UDICzQdWUMuhZ6gbFb7J\\\\\\\/+C1FXWt52Ahl9aT+7DWKNfg3piK8yyY4b4xSuPmbwn\\\\\\\/KDU1xrpz2Jo8hBG4ji5cVoWtkRRyhTFJskRDIBVwc6BydOKEmE4XT2yFmpHa5RLWrDVxL+FgkAsMt2euedpyF7XxT75aZhqLxZyrTASDEXg3CYyB4P3HkOyztXUNWT0qqXkbNfksMM0FYuC9RAuFn0v+eQhBTR+i7J2Ww3Zrm0K1876rkdymbGVmoJpLtooC+KcufOG8ePZeWeomQjOPCupfKGAnAkyhLdB+DIt+DyHCz1hTpAAzhxGL+cfp9Tc9ln+L0zgDMJFh468QfZ0HETAURBPVYt+OxNYJtpRrSDdhRmyCJ0fT20sohS5B6qrVzfQM0C803j7DHUtgsPkFgw5qM+D2QsDir1jDF26jI8WqLwYAT0YsJjEBRvQ6zuALDJYA1YYVtERc7pwnK2B+4PVmA9Hi1I1o7967istA5149WDw3t9EC2F6BxH3T7JS8+F8XjzeT1k6Szk+ipD9Qljxk+J9cCxvjxH0JZsN151gjvDGPFBdgm7FKbCk9w3sEOEJ3AVhYmb4q7V3l2hO+E5BS0nbGqRCk3VNOQR7GeQw7qGAWwGhYGOOVfbQ7Kh8C+7lSFQ1B0QHzDN2chVaa5OCxqhIDas4gtaHbYDxgoYwjtJUpcG01KBByLy5FTVYz5K6LsuZgvY07G9yj0rxthezIhoQNyPzshXXYle6\\\\\\\/NDmfW5hoPEpj47S+L\\\\\\\/HIwSqfQIx3Skc6uMkI3rZUiU8PwAU+vOfR+QhHI60t+YqrqHhi6KJOgG2blD5mDb8RGpn3jMTJikgH2COXr9wAmOrEZ2nqnkkDrNTKkta8UFKgqfSOVQX62C5W6gR59Zp66L\\\\\\\/6MBCEsYHmqojq3MdiY+DBIyQ3ic5\\\\\\\/G5Y+gjUQ8AA3VQlPod\\\\\\\/VOiao\\\\\\\/fmVl1qYkM\\\\\\\/Aec17nIRzpMY0dXEd2qC6MLc9+LWI4LyitUlySrkuXPfvt9xM7JevlBuVR6pIC9Jq+LxP1MQSFCtLMPa3abxWOYZ3YAXxOp+FebHhD6zwFocBT67Qg7VXmUCj2AKdxR6VSnk+pmCxh\\\\\\\/1rzvlNFXGgPhLj3M+mLjWMgGBh1gLmMaV2\\\\\\\/P11NnG2lPP27eXBEhnzAZXqS7HUW5\\\\\\\/Tc+A0m5HR\\\\\\\/KyfmUPd4woIm7ytHaiV9D5s+jS7iFlQL7EVwexvLcy5rIlZ3GDpJCMsM8zkx+e4Z2QUzsO3affg9krQSTJ3Wm9+y79JGBwwJBgvDOM+Idg8ocWGh3qUBhr3PBv2bguzheXVmFuMlmsxfX7phWvb1xy0dIspzGaW8rkUXjrmwuLtgKfqcvKUQ9Gqasg69GTvmHeMLkQyoWJFdEKNsKJORQFvaaCrhKd8DtX8UYbHjBFTlr4dy33zBd\\\\\\\/YQ1mqtOK6juXKXgqcqorZ9WPNfrMSO\\\\\\\/gNhGDHH6Ryb3625v7mF6mW\\\\\\\/GVim2Ei5v5wIwZdWQmRQZf7Z0SGv0\\\\\\\/CSGVzotzQ0jhWx2q4fcDG3QwpUxU9X1Dhg4Fin2DMxkldM60zNXKbF+4B06flf\\\\\\\/j0mnjlBrScMAQKQNsZjoPo6DOPgzzjZfOCthj1F8q0T8dMf1O+Qd2gRmstBmcoqZdUQ6Hf\\\\\\\/5RZVIFGC3\\\\\\\/w3K4U\\\\\\\/PaTgiPsIic9SZ72c7YlN5Aorwy0wYHNDZOSemLlRKrQBamB6++zeSg+vl0vk\\\\\\\/0xCYcyayjAwFgccqzRIfRMJea4I1BuWY1v+U+9wbQwBOqQExeR29UgVcDc+nypnwFXP37ua+ZmSGkeF65gpYTUpMSB1QTX1V7effrvv\\\\\\\/uycyHwW3gFwvAzaB0RO9p+7kVLM+ba6eN9AfBc1H4zvoFXUgTSV142vrdKV7GMHvYRD5MnrOjD4EyESXo6mTw+7MwHHfKawyVWs\\\\\\\/SyQBrBMpzfamITS+GRvIN39BI02kEr37HIKrFARswq0gHe7he9UZe6UDp9\\\\\\\/Hgj4CQqHkwuFts4Zk6Fs\\\\\\\/U9cZLACunpTYONxjIDw0mc4mg8JCetvmU49I4DXN+AzOLImwzWuN5aD08KfN6ON0zHCy02l0hdz0IRZz+xMuggvRjXxpnezg9GnYdBCo1BKd6Tbrd40SnwVMeup27zFIGDZiR\\\\\\\/ubDK2J14G3waJy4A\\\\\\\/Y9KS+VBaItwySW3sZXqtRr1bOSL23htTnFH0PA9DFi6oJoIRoHxijbUseRRsjWvxxEZHZo+Pd6aTIpDyWBRJyMxS9zbGj6Pl8Llbg5S80\\\\\\\/UIyRqQeVfhot+bmvCpLV4CqR2799JHe2LMRVWcQbdPu4FcGCvzdSJ4nwzxwKgkxxbYv06AYWv3nn0IerV6O5rNu2keV\\\\\\\/uMy8vAUEZ\\\\\\\/4nOrsJqgSg8ndHQ6igoNAljb8\\\\\\\/CRSMC8ct+ail4aiWkVSmOaPKVzCstZa7BbvocwXqr+1ooCUOn1xHZakFKdG2Gk\\\\\\\/EZ82pqOd0fXwGQlSN\\\\\\\/1sBSgrEYooMyGKwuCq4g0jqfXcGP61HsERdLcLY4V+pkaJ\\\\\\\/UMJyNkUd7jutUg0FrdGglleDV\\\\\\\/P8yrKX26OxnKHxQkzg8QEv9OQWDvzJHxp5cSU99oTqP+YGHNQaUpy105WSeKw7b3Wxf8HC8CGTJ\\\\\\\/ExRpfBc1YjK5g5f+k5nzvmH62boXjq16oUdkzAy6r2lmwChqxfOk0awimNCRuHMaCLUEknnarn3ljLtQWUYhUjRw91dnqmG3\\\\\\\/6vCNJwC+63xqULPXPxN1QFffnznfzvt21HbPWWM8CcvpiAF3y63VvzwjfqE0XCL\\\\\\\/G2elQHQoDIS+SqxL+5mFCzbS2v9Xxl4KXjuV8UcYr6TOmBIXUBhbsgtK6IxUYKN3FCVUhZC4ZiQBmBOzDh3gj+CKKBNg6YUrJYj9dgP3GXYrfqKB5k0CiLirg0t8lJh3n\\\\\\\/4nITT3uYryBOh7F30wkfakpqv350wxSWYk8rXH4FGXV1nx7J\\\\\\\/n2huAwl4VlzzG1JDJJje+apS2l4HnWgnx4Y7RhbLDxf\\\\\\\/xt7M1o5K\\\\\\\/YkyUbr7CHeMLU3lTnD+qJZoCUBFCeEHGnhZsex3Jj2q1cgKx72U19dVYGF8ONRWK4LoSwRGOUxKnvSsKqxtw4E80rSjJFo2Jft2i3z7Y+a6q1xqFv\\\\\\\/SOOpLAi9R6EEs\\\\\\\/EQpprdGJwPEqEkwTHg48kECGc1nDc9Q11UNIvkZa7SdaA37TagxomzswTs+hr7SJtbQ0mcBVtUChtlfJ5VVk3TF1lql6CVlX6WKVTuIQvYywfbrFXv0xpJe\\\\\\\/BNfc8+cjHETb6UtS41HrSD+Aqm2ZuI+vHdavbjhGzt+HXXDYXbiBE4MWyKICI8U58+25PzdpB8x+5DcR49D0tOw+jPAeRUv\\\\\\\/2c\\\\\\\/Jm0VNkm8H5cPIUSGc7WnJErY7MYb693\\\\\\\/rHi39GSGUXwK9dR1iC1Rg8A79MsJImgMvkwjJkWnyj4R3GfrcGIPCGXzBJSVIDhFOZG2u2Rm\\\\\\\/JWHqYy51cgogi0QwOcclwtQibexLUb0YXmR49VwukD\\\\\\\/N+XBTf9P3Ojq\\\\\\\/o\\\\\\\/vBqvjJGYa1EFjIpheRp004FJlkUZdFA9CrIcDAwIzoWD2UZML\\\\\\\/QVUbwKz\\\\\\\/TZrgBbGpRXVfrTzZVP1CvVbzPiJ2dC2VN23zF+KU5Z\\\\\\\/og7LS67QbFq4pZysrzOE8mVvGcsvSd7GQ32JOC+j3BzFp1irGb1ePdL+Om3iNk96E\\\\\\\/kwpEIZW6EOe4PB51o6w6TYnSG1ajWOFl5qAe+nsuF6AelSctxlKo8cZpdy3KDzq8rl1glr7vw0TrHyqa5bsUDx8batwFk4eI8P9m\\\\\\\/j5zOD3bMILmcHsEYgu\\\\\\\/YZdROmvl9l7DUgnnh1RbQOwhex2ehASKZYbtykDIDb2UqjIoNIMjDcW0ygV\\\\\\\/YCXAdq\\\\\\\/KJJbq6wH1EmNMnC1blWbD5Z78EA7mVoKrrGEHBZZIbd7aUMRdj4tefYKe4M+NtiuyWi59NMm7YBHhnKuBKu4KIi\\\\\\\/6a2zQIRLh1E\\\\\\\/5Xg8j+Mn+q+h+pC0b+H+J5PzlLQ6WEJX8F8dnN5gOaALdLNrDbz+0Fu1tAdQBhGhXR2EZXwGZsHbZwZMjiVvhjJiETk3+ZAjbhQwPGBw+cmf3mLCwBuJl5u+MuUcevG0B+a1q1Iu47x7MEJGonlNBriyHwTzfZbLX\\\\\\\/KnYkEvCwVDhLY8qvEVSW4hAYCYkIsmGOATKzYu37PAZRYT4paN\\\\\\\/O3Clvnok7J0sCnA2qbvBN1U3DN\\\\\\\/BTsTQvMMtIgv2x8sPG9rlDy7y6qacg8WbWVo99fGdzmNzcj2rPLir\\\\\\\/EkRgJuN3u+RxmLyyJlY\\\\\\\/AhLpKA0nzsq0lEelmHN2KN+nva\\\\\\\/Q5qKx7YtUD1Mp1xAl\\\\\\\/LzF2XIVH1AM53vA2gGGXIIle+Pj\\\\\\\/TawZQ7o7yc56foNQ5GCBi\\\\\\\/2cSc\\\\\\\/qrtQsAWP88LbF9hv4If8dnv5fuW4vDj2\\\\\\\/lmSXzQti3eYCvre+BbvsZtFXUKsCBV+jeIAW\\\\\\\/1MwgoDdzyaJI7dtn6F0K+mPjc0xMJPk9GaKB\\\\\\\/AcqBMWR5MesG9yVD0S9SupRsaZA5a\\\\\\\/CT37UepXy\\\\\\\/KkqbojlEzDbfJUveecowpDOOdZMRoQf7KA4LiL8yBB41N4ZgQ4Naz6HYBFcUHndCRuhKLTRw+0uNOkfHcIT\\\\\\\/9Iq1WfpUWsCKdKFbarxDzDy1QrsvcqB3mnW7x+XyhADIgfKn3hH+uaGIT73KjFuIVnb6f15\\\\\\\/Gx1JMqzmaPc53my3NAHj6GmKEzoVG2f6\\\\\\\/LbZAWLjw5Wlut2cjHHoTqKyl97zwgDb7HY5sVVqBi01quK\\\\\\\/AwZlu5IscmCmK38NluM4MoW6Zt7fbJq9RVetabzqeSfP0VtsRfKpb3HuCKt2Rfk383wVcTMuWZhcI2FWX7Smy\\\\\\\/2MiQRp1RKAFF8BQaEBwj49vB875WtrdjAkkhNdXTv7cJaPZOxe84xpLwZThI1akbUJ9EAzmyp6f06KWFOJs0SUn2sdqt74kAGYtcf4GVpkA9kWleei4Cqrx8t8Ngo9ZAHzqzWkvvkERofWve\\\\\\\/duiadouLt0WLiFvrbDoECiv3POOQFi0dnV6AZD2wMwkW9rwPrOz6y5mg8vQRFUOh7vM4nuJ+neaLfohBwu32bsthdT++qDFyLHCrh5kEZL1cywyG7rxZC\\\\\\\/lzA1Udjxo2YtTsHBJZXLI2N\\\\\\\/6Y3Trk5r9TS5adLavAPg6yWAGr+MKuyuCb8NZPGVctDKa3dE6WgxV4wYLFUYsUANGmTFY2BZqE86uJFMQg9p4XsZIeaRIinCjGcf96McRl1Z+tyTqU4cY3h\\\\\\\/NS6Qxm6Uwc2Y4zXK649XwpUIk46\\\\\\\/jyeqUgsrDEFW0yBWt\\\\\\\/EmEvPub3QifCSkOZQdNR9ToWCS\\\\\\\/WtiRBW0zJsEJDCZMHvEAnXsRckuInSx2g22BKqcn1OGesAZDTmfMv5ytKBIWOaQ6Hl7q1IZP3bMawEKCHssGtb2hkv+MCWYxtINAoej5T03zWScFpa0VuA4EndK19JP8FKeUMBkZzRQnqY0RZWuaskoCtjjm4saxKLFSDTdL+D9VYuhAd21tU3xXNpbmlPjhxtkqHm0iMj3qggT9B1yTln8MEDg7udj7BW\\\\\\\/lG47jDeq\\\\\\\/hOtUaNAonV6Jjqi71cZRi2nao0quf\\\\\\\/EVrjxk0yy9lseay+MG03m4qHC94\\\\\\\/tfiIClnYJAerqbr2PO3IiDm5aEydqDPpvgdxl5vvByPsWOJ5xUtDsigwtGl+MQhZbfnh8S2UnbwR1deiUmHgJZObICEXd5n31Csz99KoEJARthmapDnm8YXrBi7XDagIuHyw79uylTnWvRjrFFJp4Sg\\\\\\\/8H4QrGyDm2kGBSKULAUHGkvQWJx1Oxd1jEQoTnxdIzGPP6QZrlcAtKnYnB0uk6e09a5T+\\\\\\\/BzbBoSkh\\\\\\\/v8q4zNvvizVmE4TKTTl6Y63Zu5fM1bEetD1pz5bEmhlstRhfLfWY+yiTYcQLwxfOTktOd0LX0laHCSx1pnQ0HPCPqHCNTfUdzxwe2iDJ4dmslPPcAzz1UI1m39F8S0hNyTlB01FPpRaFBnYTTuLk1M18c3PGo9xSpe6W7rMKWE6WX54gTnybDFdqub1GxOCTwzxmnjlp7fTBjT7hJudej8bIRJcN4Hqe57AbvZvHQLVTdBSXMXtiBfOnNkG8Tvvq3vsBXnwrWACeKDfIaL\\\\\\\/htIJFweBLJkewXwoc3S8n0OR2W65aAO8fIoSgUoQt8JqpK24NI5JZASAjyMGrKEdpTiaFXVtDAbZzJGD3cY3yIFK4zCRqiMbQM96Ednlt2Czi9atyOnxeHZ8IIWzsHzDRIwQ08xmEDu\\\\\\\/6f0OnaLq60KszIgFV9zuJqr0C+CT1vlxSeUmgbPP6F6YhMB0e0xoCEBuoSTNDsJcd8Cybc4z9MlurOALmSLm1tcqdGsrSpvuWD4+WTNbGIZHDXQ+\\\\\\\/6LCi\\\\\\\/cT+JzAp1PFpVq5ffVVaHNNcfKxvdfi8ETHCnZJfG17pFHOhbmMk8ZjEgLSwPpuMP6TAWZ7Bj7xErOR2EAx3ULBWelJSafY3xzhvkSMvQ2M8akFZBr0wfApV9IzaY7SGNgzQk0O4KP9e5T7nGRDEPcU5SDf6c\\\\\\\/t8soPjjxL2Bm46FExoPquVfcPFmsqTU5ctEREdqTxSSwlylWhGHZzazha4Y\\\\\\\/\\\\\\\/edoCzq3I3yhzrTzPJ3zn6ETMeJ7P3IsdiCg+3Xz687hXvSzI9w4bpvuWa3SJkv4cprkFJNsjQJ3EJ8GfzSyCp0YcG5dWR9LOvXGV5uD8UMV8pt2\\\\\\\/Vi\\\\\\\/SrJAlS4PtJPd3ZcTPKkoI5gNuLN1sLNpajQDeRdYhzt2tVTWXUmG8vpTClmeNyz7EcQhHZuMEurWBd3RCaF+Ts263VQh1HSsILPrSK1n30T\\\\\\\/vzLpRPrBTsMAzONh4onBOGQvkgwMR20ca+BAiOcA1vKOJO+BBBrdJI0c8XVFIEed2ncZAOCs2LyyWsDAbg6BiJVoytLb5DsQDno0LEliycoAkufUSTWdEeZmjj0aCSZopPX6XV3X0fBW1XuAnUu2Pm4oVcbZOeeMd7HUUsNB1xvNQDPW2s5p4WanT4XQMpIxmZASkU5RyiFuhr1aowRUqariIp16xCZyU15Fcf7oy9dUdLx1vW0ZzGUs+iSTTDIbM6wAjrrm62WYzAja8CVQY6XUa1HCXzY5WwW5\\\\\\\/OBmrEen2jcKEah8VW97PqjVTFPDO7H\\\\\\\/UXGsSp2ek4kjC1WbaJ9FAiEgGhNqb1UdcbnHuuJKWILkAEbgbrzjETQ7+nihIMvS3Vkjd5hMV0rzIhmBnz4D\\\\\\\/GT9SrmkAmBDSGNsLWC4h8AbjQDLXoIFEVy5QVh8h9WpjL2MIzdvVdCJ3vPJn20vbnG5dSOgI1R6WqaKacYbJGoH3qU9TzgTEm3tqUjFkGvHT8PIwo2d178hEGtRnHHBS7FfE9JxwwtQBXkPV4V7fSsUovIjJx0grFZyABjFef0L\\\\\\\/SNmEJRvc1N61O0U59cdpd2m5OfJ48GD5XVZcQhTDQnIarxfB23cuKaEjVc3poWzbW1Ivlyogdu0sbeHMy5fbW1o2zGQV9d0qiAUa5JxsOafMZmAm0uauVGyWWvNP\\\\\\\/hWg79vbbEQQsu3ZPIAMjF\\\\\\\/TwSaxlUKkdPPvSVrd2itbpkOFWg\\\\\\\/cgnIkT+Da9Mm1pT6TOb\\\\\\\/V\\\\\\\/rk73Mkb1MRQAPmxEFaNWozSGkAcgi7f+2o4qD5HbTX3n96ZJhqEijuwQJL+BbfyfrIEJtY0c5S5jpnKalcJ0RpEyCHmc6pwTsT7zEzVOrT5B5peM40qEGVMn0qG\\\\\\\/kxoc14YWpj\\\\\\\/aPS3E7wg25ErMbYsmfX9xIfiuYiN5GoBg8eacF2ae+uEgVO3iPej4QyAo7fV0donS\\\\\\\/13nQPl\\\\\\\/PEPBbSGJRWxQJL4UF1rdBZuZKp4sZYBkY6lTBNX6d0ttqT2mLnBONm1nZDayyyRNcptLbXySlSQniBbC8V1irKfTxhy6AShu+w4L7OHjnxD+DGgonhfS1VfPLXCIdit\\\\\\\/0+ggIc7tLjacGrQR+Pc6rpdvkjTcJwHBZ9dJCnmsi1eDa67jrPyui+H1WQw8oB0rX2PJLdwtrZ2URfM7uWksVnuWNAHhDH8JTyKpvg0tMSPmZ9M+PK9Y0vJnK\\\\\\\/OhyKQMKyTp25YnWV9lIvROcI\\\\\\\/P0JauWb\\\\\\\/7iBRzU2mqCX009YOtXudoyThg8y98hmd87NPx6EnpdOkklOpgQRfHm0Zx6TZn+JvszeZhYdRDO1L\\\\\\\/\\\\\\\/B2grrIrRudCWJgP0k+399q861Wy2eLgaygikihpq1mmdFO6T\\\\\\\/npc+JHJekbBOZYW9dP6qGJAoXhy0Bj+bZb8\\\\\\\/M5IKAqLa6AUo\\\\\\\/RkHD4GMBVUAZbDBBrV0cnvmKNH6DzMPRTtKRp9KrqKxnPZs\\\\\\\/LVoK4k3\\\\\\\/2rLDZlkSp17gZS9Y1RnFqTa\\\\\\\/JAE0q+4AOkaxO\\\\\\\/tGWqZIlt\\\\\\\/Pwey2ZTKqCndkip3GJDDfvE\\\\\\\/nD\\\\\\\/5ijDk5YketQoEj6\\\\\\\/qEMLA5hhUeBfJtIhbxgK1ecGuaF4twBOAWXFgvyMIPtmaIiE\\\\\\\/38QDjUfNeY0BqIU4nMGkha53cm2epPtKU1EKG4mr7XrnMId9caCW47ca3s\\\\\\\/0f0MWMBM2LDZ09MYrk8WtllnqrkLfAGzvQw63e25dLqmyc1JXGj2uP9foP2iLS7e6WgXuhOVGHLCgm4v0waGHxqUDFbx23gW1bAhSGJ5ax71ilopYIto8aYo0RaCb0ROqxPPrIQ01JOw9BQXz1ERFX\\\\\\\/VojupW2Hxl7BtKwMtNO7KsU8h83iHTrdNW17SDZ4bO1UGRMDCPskUm1IipVhLKdb8UO6zSTC2kE9o5D+atzDNpKpdoo08SefffA3fJb7Xhmq6w1PO5G1xk2cT4O2Ns9YQ8kQJttfl3wNhoDGy+tKuVOKJkhQMvfnBLKmc4F5bTo5hLWtb7yHdF06V1pjvUsJqyd2IW2U20BafZ6PgF5mPdVya6Pgi31A0tBuWO40ozKGCw3hApUN7+TnIEyv2u+RvhmUsAWBEl72rLSjR3j9vgdYtJ1mXoAYhKunnr184dN\\\\\\\/mKy946diKgIvPvEsbKLE6ovxnEL5hjqW4ViU2f708+Y6jvOLhNEgcGxvxRNZHGumAKcyGO0INLc\\\\\\\/4UmMtsdITKvESxoVjkUOGQIUNsBUn1YS\\\\\\\/cW1K5BCbBng2bEuZJk93lyMeLk1liCV0AqfC7M82ukAYiYfs2YApLTXvJGU6aElC9CuwuNUTe3ZI+wmwFZZOR8JbaFBFJSoqBU7+mHlzjxIXcAJDJyzt8n1sc2cDIo7Sruulu3VWyLYWLtKncCVPKTZLlgEZEmAxbsLlPBJPuHd0WjL5dS66W3Wt5RWtowbRLgvfWreNNiLRrYCfdXFIITScieClspcZnT5dA+HYae0PJOxZ4AEI3wNPQlTAFuB15+aRFssyCJ6x7W7MONwbXkaIAyJNhCRO289rkP12ufBtieBOHRvdvjXDS9vQWPHyRkayKomT7WzouqYkKPRKa7RcbANdmvAVB2Ye7IwAZowemdf1Z8uav8jC5MQdNEdaKhiIvSb41RbZ7M3vO\\\\\\\/Sf7+J7tKcdwCVn57GZCf7gSBIaEsKH52mupp4qJ6sRRTsOVcoqMKX3RH2KK1PvJvmJXTlzf\\\\\\\/VRap+XWDzbMsOWxwcuc6jiAxCpctPtcquEAZ1GCzMndlayctQkW9nAdttBqyDZFlOujCSqLgN96lGWoQ2VodJP09wLmdxlmICGcVBE0TLRVsaxkPcAQxrHdO44bauHu0yaiwP+ZgtzyxC9Ir2OkVOzd5iHr1KOfFtCdfUNyuh9R8BAbOLzlDYYeEKUqfPuI\\\\\\\/u+VoTsCj2olB6LGGikppo8XRH\\\\\\\/rNWFWeMiwluTuKpMoZfA7SbFYrBcv\\\\\\\/4vn005\\\\\\\/Gv6Rb72Xi96fMarfFnr5WKhxkZvEZhTyW1zNX65mrnErEC+EXRgLGDgDja2ZOdveWZDYf\\\\\\\/wxD4E\\\\\\\/yj8\\\\\\\/zfsiXwY518UgpfV\\\\\\\/Z8v9LRBT6jcyu+P5\\\\\\\/8V\\\\\\\/DrCiMSJCUJpyA2HhGYnJuH4QP7Esm4oC6EILxUbqB+xGttaxiQ\\\\\\\/5tR26LKgzBNkwl\\\\\\\/nN4UmEym4oXuk9aP43AJs0QgRjaWRYwa4pex0J1VeMAVBqoOiNHm2RRA38qbwmmbc73VQV3glfNC1BKFNrWhrrQk7jb\\\\\\\/ix6hDpVkgwkJYnBCctBJT1YUoWmexPc\\\\\\\/\\\\\\\/WcfZU8FE1Yc4M0xBhq98eVabsF4dBeSZArcTaMewTQmK4ndjJ2ym\\\\\\\/FrocvVIsnWoVjuAleTjaSIxixsyNxMOi7ipYvjyzoB3AV2jErlXflX+tC8+dF\\\\\\\/9+NNmYTG9gBh7NShztwFP0EUYlAR20HGFlYbjlTntrBxsAc1MnbURAOF5UA8sbfMJODabSWR2qEg1HBTC04TzgCo0NtlmWo0B3haKMD01fpYTkpnBWei1S4Hp0C8nyxgkKD0UpHJwGprdOgu3tlDHcgAEEjDXvDHpUjxOqCb4YlTgnN5CK6\\\\\\\/u5csGh+O2zWJRgH704sEMOV3So7MdlAXqQVi10z3\\\\\\\/D1ELETo7jniNiozjTNOrcLvB3hXlWQSB3CQnqNHyNuNeJXDeC6IGgF7nJro8TC+0NSVdXgHhNoJAt8UmdiDb7851\\\\\\\/l0IV4SFfqBa2E2o8USk9DPKyTpkU9k8Pb+\\\\\\\/g0J4Rs+CeMEoiyRTfqWfvGBNBWn1yXt7OEFHc7kvnEGXYbGPq4VHZ1yUFHLi+CjmfSewMqnOjTza6lIWmeblIg7XC16zLDpb4z34Tqc+f+t6G2H6JZIuJF\\\\\\\/ooZfq30dHer0S2AIfF0ITYoXBBFDqwQUpPWzgmmbKxOSacb2HA7aACF7Uir36\\\\\\\/nFXeh1cIDAgbycLsi0XZ\\\\\\\/n94kzlQ8jQH6sPELW\\\\\\\/ZNHpH1LY9sSTls7YhEVj06bE5aQ9zxWjFpRTwKqxnhMB0G7xYb0ir2mNT2OEo+vLdOF41GOOvXxmB7D16WlmpM7FQzN4fu++X29vnQ3FOJMA\\\\\\\/TYZAV16BWyBAXY3vAeNtaLrAPV8uXM1exzF0V\\\\\\\/EeQUT+E6gaUz+cU4hGBkqqHrTIkq9TjiJ\\\\\\\/mWuSaqbbagdUJwb8R\\\\\\\/Rhh7+nxOCmToUD0iERjk5gjy249gUkIszF3nhngppif7nO7nT18WPWTG7DnvtX2GGbuFi2pJI5T051Ft4QwFHSJoFrjNUBoZNQt+hhiFiTfYRwdCeeJq+KWzSyK\\\\\\\/6g7\\\\\\\/QmXjYKRYVd22a8g6aPg0K6ym2RDHk1ARIkmuAHqKGRWyLrd4hjd82gtP2CCQajn4MCteG\\\\\\\/FAqhzq6qwFl2\\\\\\\/ZxPVrgMyyANtUYEFPvwWAyyAKmfKjxw3ESkNCqtgh110EJgfh0xTgw3sncTUEAVkTzUs1k81bxfX+aikd9msuqyP13XYq6huZ7vHC9X6taUe0z7deGoGFRCEiCTBLlvS6KtePcOYqyd\\\\\\\/JCcPtL48vweWew5wdQTO8MHvYeDSBVr1nlM3jauAvqT8ansyhwTpbvaBrQoAOvZ5qhWq8t5TIgFQRH+FClQSbV6T1uuG1WUt6ooFHfWt3bRdNfACeDKVT0qmWuggH36VTlZjmjZUSB2kSAcjZ644l4y8P5sZPQ1NRWeksq\\\\\\\/v8hiNn\\\\\\\/jD9UmibPAUCTpuEZec+yB55MikCtX2V5g+mj7bgJMkUiJeaDkjJfh5Z0XNZ29sVyXkbNsmJzPBjOpA7vGQWUvW\\\\\\\/cq8\\\\\\\/DiSI4joTzxUQOu\\\\\\\/5gRfzHbrnfeWTC0flwAq53hPrJHzrqqw3kCBD9Ldz438wEeH8KF4HoJRKnVRuaojb7alhkZYmPcqhmZXw+\\\\\\\/QmJiFZH7ku9oWQVYlrSdURFTLAkrRHyD\\\\\\\/0yFUy1\\\\\\\/UIQiZ0+kKZFg4wYzPlXwo\\\\\\\/pKkWhloBcepNN3xOTP9WJwPkaz\\\\\\\/em80fpTQ3VofwkfTLQKEM95Xd3CndzBJBJi\\\\\\\/z5CQPP7Me5iM8WZQvxfznPoKO4lOZyVNIlt7YKYjyDuZoo8wFV+oZlPFt+r9NDq7S9e6+oecoild28GZ4ePcEwyOnpu8rCGZPxFjLGPnduY7tkFGI14WiLgYEjcCDMhoTw0iL1Gg5UfUBE02OWXvMIb97x3axUZnI7j7dLGzs7hlSz4tASI9GFda+sp+v5ij6rfGZvFwwgbm6wlXNx9h6ChJ1x+Jdikmr6EoOfIiyKXhZpjZKUzGskGMuSBioFvSO0OMo5M5gblZkmS2cU\\\\\\\/3uxtascQ6ozcr5g0s2I0s6i4\\\\\\\/0vOWCsiW2oH5nDU5wEtwz9Knoqk8lHUyvFAXZIu9pXD\\\\\\\/Wt+LzRXUXB3A1WA43QFeasapScdV+1Zc3YlQ48MAEzBi1rBmKevSYQFguL3Dkdjd3fOJgJQiVjI8yQ46ASdyJHiNRBFzlinEkYHhatEg2ZUIWaLQKRw9cqLllOBNwriHTyDqz5CJPoN6xTDUSBU4njeHFZokzJoPu1sZbBAHK+hWr7XDv2NjPgj4tCYyVR9y\\\\\\\/QeTPzPUtE6nZNRmmxG1EQm81MCDVjo3LueIQ8FyR9QyBpcLKIum4v0lOdVf4Ip43VauKxHgOAXyWLhhajPa1lmrXlsMSV+NMK38qaX1Hhi\\\\\\\/EXYpMHticlohPR7n3CvFTlycSaiRHaC8OGF\\\\\\\/mX\\\\\\\/FeqiSVMf8vu7j69lXP0MPFZzYpEhpRW+74xfPNimTZKdIw1rXXjLu6ZWRmKiGexFNDZbJzSMvwqYHH7wVABydeNtUHwxBDGBFuxCG40mACJC8+17XkeHXxPMf0q1gi1N\\\\\\\/v3bQBhWACkVQShQEA\\\\\\\/8ISWpDF61Yx+tYcxTG+8mQvYN+kHJE6+tp0qzoSxFmQ1aVcDPahkt6XOEkU0RhkWUH33nqljsoEVBjsBPtRiq407lAfY6vfErJkF6BcHt9tybJiNFhamA9DdA9j6Nt5n\\\\\\\/o4FZPv3dO6f0sakMONxiGt7P7LRI7TbpGi0l\\\\\\\/kU3KRuTXpvHONCkbzRtytXzC1djAXlWy4iCi17oRSnzZb+\\\\\\\/p427EdbHr87wB0sgtSUYYz7oHUOvDPOGy1ijKvSffvNcFOQBh9Ng3XQ28XpuA0FTrSKpTQPeQ1Hc5N5EPcN+6qbbCe+HCIo6DtwqtgPpBYlKNB+4JvGKsl8ehWYNrET\\\\\\\/gM0SZmdrY\\\\\\\/OjmRYzCdhXb6lDncT4Os\\\\\\\/Hxzbgp9JzezS0+CuyvYikbXKnW1q7cakm5COsKN8J7O0CrWy5S9Hxl43OraFu4cnx0Gfq6fOsEwzu6C2ndb7ik\\\\\\\/DLh6PXtROS+rnur6hUzo\\\\\\\/GEZUeuZ7d8xWIf6+PZtm5FnW1sUnd\\\\\\\/h72Nj5U\\\\\\\/DnQhKaqELWfBkrQcImnu0e2Ce9PmHBL86vkpgZVSmuB1WI0Gh2CvYZZ2RvWI\\\\\\\/uQ1r3fr4YcrxH8+eq9XMEjlekyCzOmNiMwPD1V+EPVxHWzOng9cD1D3GWJmPSEju5L84zY5U6XZ0dfBGvySj0fwnKIUqJp+KW4Mfg5r5w37o6BKWlm91UmMo1MBoSada3g5NzfNqsYog9cnYGJ7LCSx5G8vjRORMC78ehg5E9VXDc2nqjYkE6Tgq1JAa+kwRs4W69\\\\\\\/MbQj8Q7eQvGwPHp4sNPZIIaDZoZkSxpdSU\\\\\\\/fGtyJMLcGQ+HU1NubBZssdXRb2PKtzieO5RE9YnZLdFRtIiIHYqyGtMmlGipw+YXQ5RuLuR6yBdOj4s\\\\\\\/4KvtXciOrp6xIiVtMHe\\\\\\\/WGS51squDgvPxyP6uryIsiBs+pPaSw973me5XLep+PSjDahKAWfeRhAcM+ylMZYixvP\\\\\\\/SGisevjce5Xy4aUk+QDG5X\\\\\\\/YRdAslSNNVkl3LD112C\\\\\\\/1diRAg9eB39Ya4o\\\\\\\/j2PS97NqA4nc3nNdg2CzJaLpfY0r3ctZ4LVlF472MIHjm8lIhvA4APy0S4NmBBBJAvlmtdk\\\\\\\/8KHSHE\\\\\\\/XU4N5MxfsgMwUkDm4u6gG2CVLoi4cXyaJDYW4osaqfYocpg3oGVleCEPdmbDBuHNpNmEXXqxZ+hzmKHKT17dkisagA27\\\\\\\/nx0H8WNtm8XkyuJ6CZ0j1YgkUfiaOtlebL9cEC0zjZOyLFtGNufL9JFExcQoIEB06tKhRnDyVscQHQ4qd3mNM+FS1LCus0GuFh+BYKJLmeFyCjhIqcQR1r4nmDdHLKuYSzk9+CHC6MNgV55mwvbFjt6akUVGduyhmzY2eUs6a81IN47QaWXVucTNELJF9s4Y5Gy8OUbzFEdJd6fY9I+HPe+uHmK3NF\\\\\\\/sP4I+gdv1\\\\\\\/26ZbZBNm\\\\\\\/jhaQ1ckNckZVC\\\\\\\/N9pb9wqDj19M2J4mixaP+tRRwDPk1dnH7N9AxbNoiD1lQTQ6B0aHGQqAioFBxNDO7TM2O3WcUBwW0ewJgGvn3gwhB4ymCnBt\\\\\\\/z2mfnbequSzZ5pZd2L3ekdQUTd6NwQ+nmLfQli2Ocd8aiTGe\\\\\\\/hgItvKZkZ1QErv6j3n7l+YyxvH\\\\\\\/TvOOsa1FHv+C432T1kQyIDDQxkHbO+StIwO+EQL3ELeV8R2bDWqVcqjGR8tIFuM8zq7TtXHiXmehJuH+NixDqoNgpuF49Vn1hQwRZjaS2Xkc\\\\\\\/poB051+fvQhQvpD0dOn\\\\\\\/2O8sJZtOYBugnMaHLNi0pkE50ynJ3nkSOVaUM4vTyj+4gIK5z5P81bQB7WkmXLt94xJMAyeYgxMkfnErTa00BUFsuYReSZgzGLZpsvQTpPbCmnCq9cNAm91otF6S4hB7b\\\\\\\/c8Ovwi9y4+b3BhNicsDqKC2BLZsiZyH1GKfaHvcBrd44v1hMTeUFjHV1cgWjgjShJs458rjPFHmkyQdBk\\\\\\\/r2U6m4GJUu5ttvj6cKTKGAq3\\\\\\\/5p9GcPuqol7GvW32XvJ0wThpqwNnneFC8BcCbh2+YZTSK+SFEdDxk3uo\\\\\\\/ro0dlYzVjphPYVhMBlB1wLPqK1PJG6KWL8RStrC8F6gwYzjSjGuEfvY7KOY0UNh+cgt1p7BiHQOlLPYgfVSZB0jBQC6pqQ3WIa3zcRhMh\\\\\\\/m6B1Y8m7eKRMjGrhazuh3eGzzLrQREPNiZZDhfgBkDHq64UGQR\\\\\\\/Vol8KxyDTVazHcaUZVHrI7U1IhCfuxlTsBsAWwd2VnQt5qDrTxLg\\\\\\\/M2IoPibeivBNMWbAGYbhfrqijK22TPRpjTEGsHogb9L9Lt\\\\\\\/U3d3R5BMUfzFTc0OOUxszfMrF0LeRNndZfzjzCpLroJyoeqFieYsCTRXuX9ZdCjHEQh2bfdWQcGou28i4yznEhTGlQhDd9+KszsrpLod8MyTvoE84RDME0pfXpkVh4wcnjhx+FuWej4HR8hJEBIrnPpoSRsVXVJhU0Gk\\\\\\\/fB09VXPPBVhtSyag1eW1TwXJTjaKq2KMcp6GQ+HO4CfHuNmYHv7sKj\\\\\\\/7pVh1gKOAsBbYlNqqJQKxwOkqUTKnosOCu6xlRStS4km6euwp32+RYOwfT\\\\\\\/fiJL7cy+JxGwMovuJNEHY657FNsNsiS4ZHl675fPZKWmBHnVGgx36PeJWLOzA\\\\\\\/YTF+s2C\\\\\\\/u66jt+YlzHKH\\\\\\\/2LdmCZIZqZRCHXEMIYK9cV36ib4VQs33B4KQLFbK2rdRr7TuqFZe0fjw5Agmvd2x3Lxa7wIyq2hNk7bJYVBFwW0ERiyt3hvECLc2cEubLwczdYdHqMTk6rfzYvvODBzQNN8qd9w\\\\\\\/ZYKgc5r2H0Yfim6\\\\\\\/95pSCaT2m4KtgEZiCtVoNXJeCZC4JcVevG69uHNfspMYXdVbX7RPVa5Lz1Vu4r1gX3Kw2sYphba4GzeEo7zDv4ttUQZrkbynw9sTzgxS\\\\\\\/Bu6R1tNJm47OmVVcAB04AYtO9u2PZNoK0Br1L70zHpKT34N9eWFwr6wGj1e61KWpAWck3zxFgKuu0J4D+CYZ5Jv9QKfekXy\\\\\\\/05rIKA4pwOURJ4DhTDgaNFQlle8S7guOMPDdOPbNh6nVGVq7wgEcB5h+EZ5t8reCM6aYws8c4oiR1RHxk3C7Mb7B6TNxtsaRIKTAheTTZLqzIDy5GX6drSwsOrGg1K\\\\\\\/SuYuaU5skvEw0Wf2m2fHX2IPrRPuF2\\\\\\\/mGd0swPPbGLvyQ+mUQy7bPavt\\\\\\\/S7Nk0cegwmdDc92XXQfvH7rwMvJmcZ1qJPyKP4gMUM8HzFCL8Rv5fStSRbPNvfNBHBopXJ76J6QR3dRIFsfPTpvJB38ZZOIbeQWlUz4T0IPMVNQuf+xtDsuXV7Mqy+qV0YNXU0i+1GsZoSaaNoLLkSqyGXxpUgYMSMFEnaBBK1nQer8if1KAFpI4NeSt3jXVqHLfnrqmkkVldjweSn168l8s\\\\\\\/pagntQVf+Jmk1YHT8zVEp7WjFG6QkEasp1Fo0GNtl1zHAbG2F++uDF+ZKUD2nUapwDohpgEdu6yS9dKDKxjDZm23P6gS6yMgW3qHCSTYrt1gLJdovDCyvOSdPcfPfa6m3z1U3J3oZef4fTWgbbMz7wdrzLBPxKl9jldR9PC8Hzdasbw8cEJ5pGkdvMwTUdMvbflSBU8PYcW45TBlZiXQmWH8b9JKYMF8FZhLeT1VNcosPfPoOP2OPsOkvI5ullbtoTNop3Ag+\\\\\\\/3T64QmCmO95+Nfc2AnQxDccjrqEn0DwxKb26RkQ0EOhAebR17L+yUAY7n0UDcuvbLcQ0UaEFe\\\\\\\/zJIXy3jfBzgXyKfocK7GxTS3olZefCE8fmPqMRTvHBk4nW575SWgfUK1dKJ2YLnDHWuQQpu6+I0HaA15dfLKUTGX5CpknRNfrFs5BP\\\\\\\/\\\\\\\/apn7cGalntLWZqCJ1FdAKqDzsDzjIo0ET7uclT3EQhTDsoSS\\\\\\\/ALZh1OXgTUZFXC7QV4wHfXbmB8RP\\\\\\\/7Y0ruImX1JzHTMdQ8MYYTbHRp0x2HeI07fA+kV\\\\\\\/m08imo5DzNjpmG5LX6fIb6I1sI1ZFR0sDO0P1CLHJio4u2dVSaw+vd\\\\\\\/rVQBRhbSvm73RsVotPWc5buQCmQY5ZpqVq6SgTLCYBgRjRpE8Mv+8Oe+QMXNfPz+6mFLOkTaNxrBddPZiEkSC3\\\\\\\/RzoPEyhwW31yJHcpeFNsQVaxJL7\\\\\\\/dNzJyGLItX7zsoc1c9It6CnQRCKrOCBqUgZWNX4pL9\\\\\\\/qHRlMr4QOFsd2Pu42dNilmNwlla1esNdED+wAwlNI2Q4V7ae5f+8i3rJ4CwXz+tMnm8hMeq25K8JtZzhZAJa1ejbSiD+Y89+vHyGy0qu46Fr8jliWRs17dG\\\\\\\/Nct0apXadhZqVmejrHnnA+Cdg07YTZrN+YdMqXI0EpQ+rcr54CsFpJqsQl0\\\\\\\/uRvYceoxwcWst31zKDRwE7d2vnSNmVLo0RVWDM6ms8qtIcUMyLk79vDGZTDhkjmU6S3LinADrntcX2kIcwyL3QhOMYgStXVQwx1S96XZI\\\\\\\/6iCjFJRjcKKb\\\\\\\/BXYwzA75jrgW3yK2F3lNhkc80cq6CFFmsPG3v6qoJfERrKZV1hjG4l\\\\\\\/wLj9zvLzc3SQnD\\\\\\\/gckRdYrko6wA878UuKh9FzxrmuJnLi3yVWhat+nLSk9NsraR697O2Im84usqHH59GW9UHBb1j1woh82plBWyY5Vp1TE3HkGYCZkGS0Js615jHJFJPsK28q29qDDJ35NoBY7OPLbqleqytB60gsqPF\\\\\\\/3AbUZXPMHU0EYojNcpkaENP+O5ph14a9WK23OJF167dr3Y4RsflzhO69POlbI80M\\\\\\\/1O5mk91J6UNpUc8u0nVj7AmAD+mPj3PrD5xdZNga9EaIqp9D46kmyoSMHpgOEM2WRapqt0alocEkDg0CVenGxhmn1CgErSsjkOnCdVxaFk1XCbfGen16yam9Kh8EUPuQNQFkCV0roNyVb\\\\\\\/oi1Rj5yGXIl7JuYddZo\\\\\\\/pyO91msaTsIB6jjOWmrIE0YofF\\\\\\\/IpTC8SmgJQLxaqLNih\\\\\\\/QYMXRvNae2mLiyU4wCkZbCLhYV3XozYY9qqJkSdrwJnT85QfDgOkRVSsDE8yC5423ZV3f6jjglEPji2FLwgKWibLqTyFKM4cvJ4Sc6Zv\\\\\\\/pgLk2Q9ebZlqWDiL+tt8D7eZB\\\\\\\/+o8lJ\\\\\\\/35\\\\\\\/aAZjJ210xaXA5\\\\\\\/B5GLzXtxhkwaKIky2udSXWpbWjkUaX9GI1e\\\\\\\/Gwk3ZouejDMlU\\\\\\\/TLtWEyHb8P44Gr+Dyv56eXY456Qc61CD3Heqg\\\\\\\/MOaxdarUZWZqNxpLJAja4w\\\\\\\/fF5c\\\\\\\/1OES4sGzdb1oV5GM9gNPNOqVrNyM19yeZFHoggxmnrTdSRlvL8IQ4GaHPMboGtM0JtIKfDz39emLldsofUguEoTDF5WdBrTvlulT7wMWMqbZuez\\\\\\\/kpxiNUj6GkOznyv5fnd94yf8mPNNDh\\\\\\\/k4l4PyHW6A4rYUjMJoUZLhpVSFuq58P0isOPBoi7UMEg7X2I7PDlxHYdyhfz8y65SMqLdNk8TV0a6tqI3Ox8FwA\\\\\\\/rNchqi7Th9Cb2rePv4TKJ\\\\\\\/iv1qsT43QbA\\\\\\\/aQt7SZkRlFDZWOkGzmhAT+MbE1A0lXJTd+5eTdo5DyrVZpQhi+IMz0+2TJWLhnrKRpvYOeSjHGbaJS5S\\\\\\\/IODfGT7YBnupVS5caKro7f9ccSv8OTBzKieCIkBZbLBuDLwzHdBLmhGG88yhZHsBtn5c\\\\\\\/4F8VEvbfd3YCucMdaQXkpD09vZxNfLjqeceGzBGGnOB9UiV3wpVeZiLNwnVMyhPNID\\\\\\\/roTWX7PndXJhPt2HMUaYhTSddR\\\\\\\/1Fr7yl3GcWc0YfpqVloSv7pfAKyfKJE3qagh2NUKEAAk\\\\\\\/BrpJ\\\\\\\/+mnLTBeJHYlzAvBPmELdjk0vivAp7Fam3VeTk75diY1Ve9xraEHFlViuWHrSJyKk5WwRs7J\\\\\\\/ajSEVpUjB+tIPjdDF3sJFjUUQEKQY8+eOI0pVZBcsyrLXk1dS34Orx0yNIZGpZlGYsEMpHmuYTi4sTT128pGmcn09v1EgEhST0mELKiGJWGmPdkcd7A7+BEAyGcRIBuvR\\\\\\\/IOVbIzlQiOpOwfcEZWbekLpcSYWFKC70LDa714HuaFPrxeYfUi9nk8AamByv0G61NlUMg3KdXYU+39cAn5eCY\\\\\\\/yMKvSDIpsbk\\\\\\\/84dn0EtcUzsa1TfKmMu59oSeLJ3NOCrSD3nMdlfuOXhT9ug2tlZ3+48S66IgQ3cJBoUrvmDgh8llmrBy1b90FB0hC+ErD8bFxuSHq1TZZTFYDqmet1lAlrHrIWHpr4x5QpteSf5NFhlxs2+M8yMS8dp7A9eka2n68d95ykt4p811qLen6FHMAC\\\\\\\/aEgVCBrqgFP\\\\\\\/sD3py9dlXcncj+8xNDCqN3smuRB12gnT9ZWfdKkbKMaUeFt2auR+AAT9Z3dleV4W\\\\\\\/79PldCRMxzOvF3M\\\\\\\/ShqDCIc7ts3Ry8iGktGXTrctxBh7YKb4IKFr4uDQibLvT5nE9o35k5+OtFUi5ojdO701vZ39kqd2TGzrP0CcGyOFJO8ZQu1+7agoEFEKA9tk3Zb\\\\\\\/2s\\\\\\\/ckwvuqTMdBt+pLO0IduG6kXVETMwbDUnZrDrwia5K0YbrejUb24yxu6vqtU5iC4F97olhZMqqWCKHvL5mRDxWNHYDYuxPD4ZSI2olzuwBl8eWi5kKYI6vV0A8SOn8rSSaGtBP6p\\\\\\\/cNJtyA9Cq\\\\\\\/Lcnr7KKahviPX5XsY2fSNdgM2Z6KRplI54V3roHNE74TwjT0qpFlS\\\\\\\/ZLOLLz9iSjZunIvAzINdV4ic3EgaFNZFvbISw0TyVKWrRM18+v8vqstZT8mkTkAKFJ3Fs6Nbx5YyB0Vtpcu6xO8vqp0tHi5lWy02HPfAXkQ3B+dgISnzqdyMAYz7P1Xijz03VfHjEYwzb98PBzRxrNtkU6xpNPoBMhhNv0Kvwk8eT8W8L9fpAm\\\\\\\/6TtfUb\\\\\\\/0sfYuc8uTTDf4\\\\\\\/GDQioUcar4EkrD4S3SMMHz1BXHio0AefR4yGuLPmu4iMB0Ira4adkiSy4oQm+hhSrPJmX5agdKWrce2Bny3QbedLkI+5P9rQPkee111ky7klc6nkh9PxmsNhj5mPA4Lrh8Wf7Tg2Xct0zLqRqcdIge2u+fnvx52bii+sIDlsFW2nFvqBRZXAMTYEUw\\\\\\\/pD2ucqStaJ4x4\\\\\\\/rIj2iswNuWa+Xm5P7MNC1+7grnHpLhk6otLdDPMfmBgv+aLACrgG+12PP6VTFO4qqIRQFz8Np84MNrR9YS5BT0WzW2e7rKdYFkWqYQHoe8Q6KE4nfRBYImJwle\\\\\\\/7UJ7o6z45OJl6LQ51O62q80TWgUeT0c1cD6YMf0LnGRLtJTT6VjGI0XU6h7LSAB+\\\\\\\/M6OaSsX1+WHPC50Egmzf8yVrYJ4xeUiB3oXSP6IYOkTVDAvds5dLOZwP7oxETFQADz+eCtn72nvv6kG0Ucf3fwQewrRop0CwkgWe3STOcSB4pvVPN18stk5KyYDHDMyD4NRrxKXoqZGhHuxOcOIScF2uXMyjoywCsG+C0nbPxqNgA4sBiaZos8HoekpgzbBhJEZDLVsAdJVKGAVAk1zdD6DZevPvpo\\\\\\\/P0Ihy5feiwBLJ2bAF\\\\\\\/pt2l15j\\\\\\\/Kyp2lQEStegY4rKoQNK00yi\\\\\\\/MDSHXDyeQ7EccawcOlVFHFVCr6seXUBGpHY+kQ2\\\\\\\/uhs\\\\\\\/UCGQg1yNV\\\\\\\/ZsJ5H+yR27cna0Q322dZ4WnIz8EDgBSXqSUIDv\\\\\\\/tYkIHZ3rkcgH2UFFOEFLfB1ZfwX4ecIw89w7f+OWWy2Plx\\\\\\\/E19ZLdt\\\\\\\/1o39H51vbemJYVci8Wyq9wTyRQRt+0pRoxKnzG+MJRAY4987peYeQwHSiPjambjwxKLJfjjPjBMStAd2h76TJBv+6uM6rRZkXtnI9XTwNGQZV88H5Z1WIR1FCFA4WJRyu2xE3z14AfG21Or1dCkMJkXJLRlI0oYHDZxG8G863xudNmNRTSGoWByVleJ5EuyO4Lg4az526tKecubuvzfoT\\\\\\\/yRe1+umuxClhUHeHbd9m0WxIFjWSLvYuwqw0\\\\\\\/nDv\\\\\\\/ZMxUrdHf0+glQwUXqYE2TYgHvyBd4Z2NYVUmz2jXi98KExrzBxObeO9ZbAWoRqbtUzKVxyBqNFCZZJbDwWJqPBz\\\\\\\/ISoJJ8ptFeyrOypxhLR3s2s113etRUHb7HO+ApOxEWWimzhTzfo3jinrU\\\\\\\/MbjZUqf25ZiLQtzFbuvamIl36AeOgSSB7VnMlPILKgp+gZXwwoOJDxTuxpeoBOr4LqQI8vyLLGtNtBazSyjMhPUBXItHcIXUhIH2iz6BwkT\\\\\\\/9Z7DDKz0Uh1bK6ZGqnYoTfjxAeATuCU9NII36CPyCDoKPr6tkCdD32UVcySe4jAXQ\\\\\\\/ha8HUugBnf8cG3wiA3SF4DF3jXE7gmpJBCB5AEv91Ou159V+E27AldLLTYIb2vPr7WuPuKu1qxvjBGsFr0en8kBFlKdKtJsdsyMAX9jF9HufgysZjlMGlz1ppzcaqQxPoZFNAtB7EHmfsXvAzyQgY7011IMZugtjgHoBqi9pLDPfdlRUxWGx9IaHf+lOs7LEgUeah637T2c015kgkCH5w9UOOBdc0epJmFTRy+t8RMlG7Cd5vNa4FGttJhkZ9b+O4Q5ViMCgDlzkvodd1X23Cc5Gmu71shFxRraFzrFpBNbsNSIbspHSxV8WSBRWpsKqlx8AYmMVOgx2G42oqRa21dM20QCzrkw0kN0anoRKnl74tm17MqBQkBiNYirp5S8d9kvkTA1pA0iSTbCoi92V58JmsM5t1YtW4n5ny3goqYQhGj\\\\\\\/BMyl7QgTjHQQZchaU6+iB+\\\\\\\/nmBe1T9egNd5ivfK040GZslHW04C2Gptz75sj1VH5MxsRGba46+3E8Z7f31mEm249bZgr9s3h4n8CyUJ\\\\\\\/p3Skp41VowuuOadUQAR8+TVNUhNJdxb9iQfSB8UJInGuZxO3Qe92TNTjmrICxs9dytxbycu1wiE\\\\\\\/\\\\\\\/iuMck1kYUfyJ83r3TL0ZEfgd63xCU+6lkQzLl1y3u7QDxwQnBaIBj\\\\\\\/4jETqTNRvEbvi8lypn6tJchtdGYIykr8Z7PGCvxJOuAR\\\\\\\/78aNi2Icb2uB3m0Wnu4RF42KTa3J4VnVeaC4sJIS4MT1IblZ+53pVyzg4rqm14ukcw40H4rnLJRKUH++xROmAwV9EY3G1WiesSsHxQ80sIVhzyt5UKiGRtx\\\\\\\/9u2HF8ZhCcffRHxPTy5MUkunkWNwYTstKtxRToTwVuQRGH8s2ea8s+e8FghYMzuiAMfBHbbGkyKsetFVkbx\\\\\\\/AV3b8o7OZ50P3V3aRlJQXRhvjJBh5pd3lEyfYtYhWYt0Ii93qs01X0S0N4UUL5kdOLwHEK6NivDotmr4wh1j3AliJ\\\\\\\/Y5DmI4o6mhp2wL751jD1lGs5UkMTLniMgkwFQC+guYtiDg\\\\\\\/QXsRgmBYOi5gmW1QAUmLxq4\\\\\\\/zPSdrYyDUdYUdKbhb5\\\\\\\/McHjRpEYJo7Zp\\\\\\\/VUH9N2fkAT8kooMG11n7l7zCt3sF4cAa0rZzl1mRiaaBCrW7eOt3tZo107KBzch+DjH1JNs0kQVdGqSBGdorHRJZUU\\\\\\\/StnWe16DelJGNYhxEcHku\\\\\\\/LkSn34OgCE+ZJ5eHJTHyLQ0xllt+w67n5L8EGK+w6RA3lp3zQaUHxQvJbAuU8oqsL8ANjUz6fHorcuB5AFzaHzN27Ds2lpy6fn8D2bJiyaMV2\\\\\\\/\\\\\\\/M95AbEpO+P\\\\\\\/suurPj3PfVeABZisTOruy3g85LF9Mg4W++K316FZ7WmSdq3SB1SVplMtNeYKSGCuRNseC\\\\\\\/TWtMA6au4sIa1+1OyrfHr8ivKwIYwrsobZcH5VkjLU9oMB725JG5TsZrUmi4mq76rPQoCMs94x0LLCdZV2O8twuPShbNxieP+38B0WYULxIpDAgwA0\\\\\\\/czG2xtohugiuO38KoeXCi\\\\\\\/HED0HKkVB7mRITYtt6eqpk2+ik2u3M8czmFP94VBrae5Yv+bfD7N8uKNPyer5rfS++0edtIldv+Zw70l9W7aLh1MM1KWlzqbxtzsmsNQ9IUB5hVXwLqOX8LuZ09BRbnouod7Ffm3bJE4DIUzeOmQmcg1r4q6bjEnMRIOiYVouVDTi7V9XfzWMKasY6vA3A0iYpd9HiPXmDBy9+E2viKfbqDz9+MW3rpHljw6Qf3y6AO1cylllEitWxkNm6j0mXec2MmMomqOROjvCtW167QFGDcY+eq4GU2Eu6pCYa7Y9KFEw8FgDfWM9vfjsFyMQVUvrWYdcjj\\\\\\\/Nol5QvVGE2U3CFFnB7ty76+YBhLZfjKsKzFeQDZ8iy91hpSmPLL3KzF\\\\\\\/isvRUo6YpPzybq4weD0yZRUze\\\\\\\/rQVWxpRc4rFHDoF+c77FEL5xftiyeMPbnwqaqY9LONPg9vGmKyUWYROzaUK3\\\\\\\/gjuTKRfH2OZ3t3Um7UnNFAFrxrDqw\\\\\\\/yElhXSmrruq5a8tA2bU+xhNqQmVMMD6bYkAyB69NW7ZlDXPTqhcdE9vHfWCF3srWRa56c096yiwM9a80tKhgKBgLoRXdxcZbvRr8Ug1lhacrBL3w3ffVXWO+3bkHJ4r5ymVGWKqCFNryGU3DPS7wYW+Fm90utjegM9\\\\\\\/2+D0isPUI\\\\\\\/0Uedwsi\\\\\\\/bhZoKeo52wBIPVgn6X40g4Yup3lWSBY7voLVXdDkwsL69O6FJLhZYKchiH3Fi9hievgycge9e+APt6zJ400qIWn2lS1LAzeaH4qxfRxVuKuUEnG86TRM25t+aBMY9v1QOrPdw7PWSA5DGlLWLSM6JyXSR0OMNIlg+dbGjTErayJgmQjAm59c5AG06+2WgXce+VznmJlk1rSRBYKpb5FvuDQScCTnVG52wxCDZA3PfxYpZziAD5CQggmVrm2H89lmxoySwbUzHBy0hBcWdtmEbR\\\\\\\/8F3IXO8gpInCBwlITe\\\\\\\/wvYZdJ5ssBYGDJFpzozsc491+tr4o28o+g6T\\\\\\\/RAXvrB+2R3lLGB76EBURKteXyuv67hmNuz0P4DDcJjaFGLdeCcxvDk+CW8Qq\\\\\\\/sQe8RoDEyfpViUwt1bQItbgmFrlriJ9ie4m\\\\\\\/ELDIi9cexkIw+ePhppa9xh6oecAUvjJO8iezY3+D1KIWD4lna+HDWpQMd+68bLC3hPogu97pKXmvXvdbppCU2iaN9b9DMVE+zXnlRZHDrbyaNo8YefRyvylePRkKayabhwujfxV3vWFUjj38bzxnED\\\\\\\/BtvMfmgDb1ZBhUp5\\\\\\\/1oePC6Q1v1kcXtp6g8GtBz8TjcAaBtG7mS4Vz+YQ8meXTFLHp3q3umL8ZYRl+SKxuJH6e9q75nVZTktBySEITRAD1EmabYNN1MxTvP2Z937+FJncrpEt\\\\\\\/QgGhLJzZi3iQ\\\\\\\/um\\\\\\\/Mslj8EOLFtBv4FZ4ngypM6P1y43TJwId5J+W3UaDrbaDT9TI5rOnMELDuRWP997o8eHuHALr6KM76DDKYjLOc8NIcGC+o658eArEr\\\\\\\/G\\\\\\\/ShWrfZih6EExUzydx9CVDCPrhZTXM0vqqa8abfY21TjqinMTjlXmfLQC5o5+HZZa1jQBa3UU6+y59+89KsvsF8Jmxe+7CaM2+omFpqSWQ\\\\\\\/Hzx\\\\\\\/iE115BMLC8X5cQRo7vEPhZmAaX04LMARFNWhkGcXNVWcOhlOzTHD2ZJMT16GMyjtNJEbXKUnLqjrulQVFeSJZr7Riq8TqGyiOASQtIC1KHqAHvqcecad9a4xXfVj+Ang5H5IdWptNf617CqPwJ68M8WZjE3PXgYIhpQPBcTKYeu4Htnn6\\\\\\\/kIoIVT7\\\\\\\/K\\\\\\\/RIYQT5H99WNNurLbkvb\\\\\\\/sFG9pcDc\\\\\\\/qVxH\\\\\\\/CGs6Y4auPbkxBxdhVowDQ82S5axs2qRAlUomOk1IDIjCkCSIfFDkbClK\\\\\\\/TKQzfMZq7FWN5yJRzEiLIe6cTEyFjpZ7AiWlD0HIzUe9Mq4NVp+0C3KDkEaFFgsKqurw5XfRyXVuSZDLYRZeB4x2\\\\\\\/VfEXqGxgacO7jcDg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022vsvDu\\\\\\\/PHXgesTnFJ2tHK\\\\\\\/g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e0d0ab7649d6b21527517b385031325c88ff87e3278ab6e933d6a683475dd721\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Nsq0WKFnE9t+i0egvCww2H0jGQOzF6qVaZQKGZHMS0zNI7AB792nYHg36UcZaH1+s6snjRK4y+wqsspUrF1ut3Mt0PyhG9\\\\\\\/SazSsJ+qWaCP6pS95qqBUtVc5c+OvEV3HdGCGLOBrFlMn6+cSe6DbTLTR3siOvzmM4vxkOff4VX1WFBFQvBv9Hqj55Pp3bJJUOQGykDbnEK5QU+tYm\\\\\\\/DmJPUWNxNpJ\\\\\\\/vlbKfesQ9zVhwJSyxv+TOfGtpx41\\\\\\\/CX9Fl\\\\\\\/w+kSAFrqyK4cgaavQjj4GUmS1HQcribhWllNuP\\\\\\\/sLp+qgoDLA+3LDeX5XS7CLOhQGm6lN0Nf\\\\\\\/rM6lz4vl1PmIXK9Tcr9sJ0G88kRUPlLbUk9AZqrX7DlT5a3o4AGN1odqsnMx9qjPR1QJIJDN8dYRT8PoIO9JW\\\\\\\/2gQhSrGJmHTlQKe8Q\\\\\\\/zAaRj7pz5UbqlWDDgnEeBnr3HGqneYYZUod+7ZtabIogcYNtuSfmNDpB2tSQuURJdEzhMS8\\\\\\\/\\\\\\\/f0xr5IC\\\\\\\/yQAiEXXtTDYrTDvuN1O3U2ok9UinONxs\\\\\\\/d8wcnuTOlPDbH\\\\\\\/E95rWC28fr7OLYA0sZqksz5iOJYrCuMaGiUFL6s28EBfI\\\\\\\/OoOCZfa7C0vbZXcUJNbm2Kghe2loxKvsAgyx1Y5eTDzA+43lSvHbiQUl+FBNPERbAfowlrYgr6JWN+wVJsX\\\\\\\/JjJLtzSCK8mFpRCTpcl9SGDKiy+iRxfz+cwW14SVETKQQ15TAesaIaXMO1SepAsp7FdeMm\\\\\\\/dlL7Ve+sn\\\\\\\/4mGqwqxudmLFGqOsqhG5Zhe8ROu8+Lp20htIyKugJlyjUYPXwsZQDVykchqP1X2P6CkOonysueCfUphTDBd7LIX2p37IAmqDaonfqhYdqoUKIaiqpJHClxgQktec0Y8Z9EP8T7AFUBGOQZnBkNX1lSB65F9xRs9IPu\\\\\\\/d9fQp+dr6DJc7cpr7h2pXnI+rttJigf3eMZHiuglth3UxOhLlxHIz2ByoO32CbpvHBPG9sUvJCL3RNVCw12iqPYTW\\\\\\\/raSqMDzuxnk3Gg63xHdIQAnBy62UJcjVxsv+qIHrLEURYR7jJL9yKe1q6\\\\\\\/OkpmJ3Gpa9+RNg+SNUkkPvR8lC7Ot1NDnN\\\\\\\/F6pCapUKiAXNU1jLeJXU4Yww69VXdW\\\\\\\/vasV1ePDTSd94P5VDdgu+Y3idKOQHD8bAGhgVDVgiwPfZ8lhQiYI\\\\\\\/ZOKWjbB6QyVmCkN3w+l1wHwGvm5hR\\\\\\\/IdaaPxejlbiFyaeYLG\\\\\\\/M5sfwdvT0\\\\\\\/ne48Ps\\\\\\\/6dS8h0MkKNWyyj\\\\\\\/0b6cMUsjYFUkXnn\\\\\\\/3QQIVBUPj3JVg8268UuycuzA8SZiQ0AREIDZioGqGtKhWrpiQzgQQUpi1pmgWq4v72BJ88TGTECO3TRQVuUhgcGSvH4IeAZvPwxb9VMKYmLrYd\\\\\\\/Rj2B9rxGeAKJT+qUT0Edl6naiGLUixgwVTe3bKDeczts+yR+q2jbPaWlWosmAudedSrLeAmAuWcaV5dRE+wmh8RNfwT0kr6G7didjf7tyCK1ePO9uJTccOphyg5qi4noWklwyVfPsSG5DZUSnfLyWy4p+QPIlmjkTQ1r2xQ6jS\\\\\\\/JFaqUGnaIkZT\\\\\\\/0QCJsLahDGFwJj2zMvPdDH76v44oJrJNMn9+Pu3WNlq9c4sfWIKxBSoeho9HJQn1tzYvNYMRpiMOxeFZjeSVXIvHWZsvKDYF4Y2ITJjn12C36F+ASS\\\\\\\/epmwhc+yfXfeUU4xh42W9+YEpHbXjxMZUdxQd1CtS0nA+orKUcfJVcVA3ificdHnj6Sg\\\\\\\/gpB+xuTUFhqSMY\\\\\\\/suljtZNn1W353wBRlQxt13SCSsh5ll7Y8BtGAbMvZIFeq6xo1sqE5rm1hVHOWGZp7PFFTe+qxZ0DFv6J7jfx3qU\\\\\\\/I5KjEELQXAlPuYiu9meuIGKICWQH2PYTkESuVVFGt9QAdt5ZrU4prig4PJzfJ7u1W4BIE7I9xS3AFG+NIIHKP5oVvtKNQVj\\\\\\\/UvKqYOMtiQKgybwQ+dpPR4JUx+sw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022O\\\\\\\/kMkyNvARFser5D9psOYA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022360c728c4c741a33792aa5a6f95c1a0d06fd9f71eb4ed51071a10b678aa4e8c7\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459574 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/2095" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qs6WchkmYoKl76Bw-7EraixJZmeB6PGPxPh7OWnOO6Q" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/2095" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-xK16uYV0n3VLwy2oDvnAU6CkZwEVpaTu2nXSRTVYVUA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/2085" title="Dissecting Genome-Wide Association Signals for Loss-of-Function Phenotypes in Sorghum Flavonoid Pigmentation Traits" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/12/2105" title="Increased Genomic Prediction Accuracy in Wheat Breeding Through Spatial Adjustment of Field Trial Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459574" id="node459574" data-pisa="ggg;3/11/2095" data-pisa-master="ggg;g3.113.007328" data-apath="/ggg/3/11/2095.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genome-Wide Association Study Reveals Genes Associated with Fusarium Ear Rot Resistance in a Maize Core Diversity Panel</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Charles T. Zila</span>, <span class="highwire-citation-author" data-delta="1">L. Fernando Samayoa</span>, <span class="highwire-citation-author" data-delta="2">Rogelio Santiago</span>, <span class="highwire-citation-author" data-delta="3">Ana Butrón</span> and <span class="highwire-citation-author" data-delta="4">James B. Holland</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2095-2104;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007328 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459574" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charles T. Zila </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Science, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BT.%2BZila%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zila%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BT.%2BZila%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">L. Fernando Samayoa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Misión Biológica de Galicia, CSIC, Pontevedra, Spain, 36080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L.%2BFernando%2BSamayoa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Samayoa%20LF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AL.%2BFernando%2BSamayoa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rogelio Santiago </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Misión Biológica de Galicia, CSIC, Pontevedra, Spain, 36080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rogelio%2BSantiago%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santiago%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARogelio%2BSantiago%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ana Butrón </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Misión Biológica de Galicia, CSIC, Pontevedra, Spain, 36080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BButr%C3%B3n%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Butr%C3%B3n%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAna%2BButr%25C3%25B3n%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">James B. Holland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Science, North Carolina State University, Raleigh, North Carolina 27695</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>U.S. Department of Agriculture—Agricultural Research Service, Plant Science Research Unit, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BB.%2BHolland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holland%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BB.%2BHolland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:james_holland@ncsu.edu" class="" data-icon-position="" data-hide-link-title="0">james_holland@ncsu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/11/2095" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459574" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459574/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/2095.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459574" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459574/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/2095.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459574" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459574/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/2095.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459574" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459574/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/11/2095.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459574" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459574/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3"><em>Fusarium</em> ear rot is a common disease of maize that affects food and feed quality globally. Resistance to the disease is highly quantitative, and maize breeders have difficulty incorporating polygenic resistance alleles from unadapted donor sources into elite breeding populations without having a negative impact on agronomic performance. Identification of specific allele variants contributing to improved resistance may be useful to breeders by allowing selection of resistance alleles in coupling phase linkage with favorable agronomic characteristics. We report the results of a genome-wide association study to detect allele variants associated with increased resistance to <em>Fusarium</em> ear rot in a maize core diversity panel of 267 inbred lines evaluated in two sets of environments. We performed association tests with 47,445 single-nucleotide polymorphisms (SNPs) while controlling for background genomic relationships with a mixed model and identified three marker loci significantly associated with disease resistance in at least one subset of environments. Each associated SNP locus had relatively small additive effects on disease resistance (±1.1% on a 0–100% scale), but nevertheless were associated with 3 to 12% of the genotypic variation within or across environment subsets. Two of three identified SNPs colocalized with genes that have been implicated with programmed cell death. An analysis of associated allele frequencies within the major maize subpopulations revealed enrichment for resistance alleles in the tropical/subtropical and popcorn subpopulations compared with other temperate breeding pools.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/association-analysis" class="hw-term hw-article-keyword hw-article-keyword-association-analysis" rel="nofollow">association analysis</a></li><li class="kwd"><a href="/keyword/disease-resistance" class="hw-term hw-article-keyword hw-article-keyword-disease-resistance" rel="nofollow">disease resistance</a></li><li class="kwd"><a href="/keyword/genotype-environment-interaction" class="hw-term hw-article-keyword hw-article-keyword-genotype-by-environment-interaction" rel="nofollow">genotype-by-environment interaction</a></li><li class="kwd"><a href="/keyword/maize" class="hw-term hw-article-keyword hw-article-keyword-maize" rel="nofollow">maize</a></li><li class="kwd"><a href="/keyword/quantitative-trait" class="hw-term hw-article-keyword hw-article-keyword-quantitative-trait" rel="nofollow">quantitative trait</a></li></ul><p id="p-4">The hemibiotrophic fungus <em>Fusarium verticillioides</em> (Sacc) Nirenberg is endemic in most maize fields in the United States and is present in many arable regions of the world (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Van Egmond <em>et al.</em> 2007</a>). This fungus causes <em>Fusarium</em> ear rot disease of maize, especially in low-rainfall, high-humidity environments, such as the southern United States and some lowland tropics (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Miller and Trenholm 1994</a>). Infection by <em>F. verticillioides</em> can result in decreased grain yields, poor grain quality, and contamination by the mycotoxin fumonisin, a suspected carcinogen associated with various diseases in livestock and humans (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Miller and Trenholm 1994</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Marasas 1996</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Presello <em>et al.</em> 2008</a>).</p><p id="p-5">The best strategy for controlling <em>Fusarium</em> ear rot and reducing the incidence of fumonisin contamination of grain is the development and deployment of maize hybrids with genetic resistance. <em>Fusarium</em> ear rot resistance is under polygenic control and strongly influenced by environmental factors; no fully immune genotypes have been discovered (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">King and Scott 1981</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Nankam and Pataky 1996</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Clements <em>et al.</em> 2004</a>). The complexity of this resistance trait has impeded breeding, such that most commercial maize hybrids have lower levels of resistance than are desirable (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bush <em>et al.</em> 2004</a>). Linkage-based mapping studies in biparental populations have shown that <em>Fusarium</em> ear rot resistance quantitative trait loci (QTL) have relatively small effects and are not consistent between populations (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pérez-Brito <em>et al.</em> 2001</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Robertson-Hoyt <em>et al.</em> 2006</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ding <em>et al.</em> 2008</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Mesterházy <em>et al.</em> 2012</a>).</p><p id="p-6">Despite the genetic complexity of resistance to <em>Fusarium</em> ear rot and fumonisin accumulation and despite the very low heritability of resistance measured on individual plants, resistance on the basis of family means from well-replicated studies is moderately to highly heritable (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Robertson <em>et al.</em> 2006</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Eller <em>et al.</em> 2008</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bolduan <em>et al.</em> 2009</a>). <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Robertson <em>et al.</em> (2006)</a> and <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Bolduan <em>et al.</em> (2009)</a> reported genotypic correlations between ear rot resistance and fumonisin accumulation of 0.87 in North Carolina and 0.92 in Germany, respectively, indicating that visual selection on <em>Fusarium</em> ear rot resistance should be effective in simultaneously reducing fumonisin contamination. The heritability estimates predict, and empirical selection studies demonstrate, that selection for improved ear rot resistance can be effective (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Robertson <em>et al.</em> 2006</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Bolduan <em>et al.</em> 2009</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Eller <em>et al.</em> 2010</a>). Unfortunately, most sources having high levels of ear rot resistance are older or exotic unadapted inbreds that lack the agronomic performance of modern elite maize lines (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Clements <em>et al.</em> 2004</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Eller <em>et al.</em> 2008</a>, <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">2010</a>). Thus, breeders are faced with the difficulty of introducing polygenic resistance alleles of generally small effect linked to inferior polygenic alleles for agronomic performance if they attempt to incorporate improved genetic resistance from unadapted lines into elite breeding gene pools. Identification of specific allelic variants that confer improved resistance would permit maize breeders to select for rare recombinant chromosomes in backcross progeny that have desired target resistance allele sequences in coupling phase with the favorable elite polygenic background, facilitating the improvement of disease resistance without decreasing agronomic performance.</p><p id="p-7">Resolving small-effect QTL to causal genes for traits that are difficult to accurately measure phenotypically is exceedingly difficult in biparental mapping populations (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Holland 2007</a>). Compared with traditional linkage-based analyses, association mapping offers greater mapping resolution while eliminating the time and cost associated with developing synthetic mapping populations (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Flint-Garcia <em>et al.</em> 2005</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Yu and Buckler 2006</a>). Historically, a major limitation to association mapping in low linkage disequilibrium (LD) species such as maize has been the large number of genetic markers required to detect marker-trait associations. Limiting the search space to predetermined candidate genes allows for association mapping with a smaller number of markers but requires extensive knowledge of the biochemical pathway contributing to the trait of interest (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Remington and Purugganan 2003</a>). To date, nothing is known about the pathways contributing to <em>Fusarium</em> ear rot resistance in maize. However, the recent availability of the maize 50k SNP genotyping array (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ganal <em>et al.</em> 2011</a>) has provided almost 50,000 SNP markers scored on 279 of the 302 inbred lines of a commonly used maize core diversity panel (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Flint-Garcia <em>et al.</em> 2005</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Cook <em>et al.</em> 2012</a>). The maize diversity panel captures much of the diversity present in public breeding programs worldwide. The large number of markers available on the diversity panel has enabled genome-wide association studies (GWAS) for several complex traits in maize, including kernel composition traits (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Cook <em>et al.</em> 2012</a>) and the hypersensitive response (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Olukolu <em>et al.</em> 2013</a>). <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Olukolu <em>et al.</em> (2013)</a> identified SNPs associated with the hypersensitive defense response in or adjacent to five genes not previously known <em>a priori</em> to affect disease resistance but whose predicted gene functions all involved the programmed cell death pathway. In this study, we used GWAS to identify SNPs associated with <em>Fusarium</em> ear rot resistance in the maize core diversity panel both within and across two contrasting environments: North Carolina, United States, and Galicia, Spain.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genotypes and experimental design</h3><p id="p-8">The maize core diversity panel [sometimes referred to as the “Goodman” association panel, because the seed stocks were originally assembled by Major Goodman at North Carolina State University (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Flint-Garcia <em>et al.</em> 2005</a>)] was evaluated in several years in both North Carolina, United States, and Galicia, Spain. Only the 279 inbred lines with available genotypic data were considered in this study. In the Galicia experiment, a subset of 270 inbred lines from the maize diversity panel was evaluated for <em>Fusarium</em> ear rot resistance in a randomized 15 × 18 <em>α</em>-lattice block design with two replicates in 2010 and 2011. Nine lines with insufficient seed were dropped from the Galicia experiment before randomization. In the North Carolina experiment, the maize diversity panel was part of an evaluation of the entire U.S. Department of Agriculture maize seed bank collection of inbred lines in 2010 (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Romay <em>et al.</em> 2013</a>) and subsets of that collection evaluated in 2011 and 2012. The genotypic data on the maize seed bank collection reported by <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Romay <em>et al.</em> (2013)</a> were not available at the time of analysis. The 2010 seed bank collection evaluation included 2572 inbred line entries and was arranged in an augmented, single-replicate design. Experimental entries were divided into 18 sets of differing sizes on the basis of maturity and field assignment. Each block within each set was augmented with a B73 check plot in a random assignment, and five other checks (IL14H, Ki11, P39, SA24, and Tx303) were included once per set in a random position. <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Romay <em>et al.</em> (2013)</a> reported flowering time evaluations of the entire collection evaluated at three locations in 2010, including North Carolina. Here we include data only from North Carolina because it was the only environment used for <em>Fusarium</em> ear rot evaluation.</p><p id="p-9">In 2011 and 2012, the maize core diversity panel was part of a larger sample of inbreds evaluated. The larger population consisted of 771 diverse entries divided into eight sets on the basis of maturity and replicated across years. Although disease measurements were collected on all experimental entries in both years, genotypic data were not available on inbreds outside of the core diversity panel at the time of analysis. Sets were randomized within the field, and each set was blocked using an <em>α</em>-lattice design. As with the seed bank collection evaluation, each block was augmented by a randomly assigned B73 check plot, and five other checks (GE440, NC358, NK794, PHB47, and Tx303) were included once per set.</p><p id="p-10">The three North Carolina environments were artificially inoculated with local toxigenic <em>Fusarium verticillioides</em> isolates via the toothpick method (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Clements <em>et al.</em> 2003</a>). Approximately 1 wk after flowering, a toothpick containing <em>F. verticillioides</em> spores was inserted directly into the primary ear of five plants in each plot. At maturity, inoculated ears were harvested and visually scored for <em>Fusarium</em> ear rot symptoms. Scores were assigned to each ear in increments of 5% from 0% to 100% diseased based on the percentage of the ear presenting disease symptoms (<a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Robertson <em>et al.</em> 2006</a>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/007328SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FigureS1.pdf">Figure S1</a>). In Galicia, between 7 and 14 d after flowering, five primary ears per plot were inoculated with 2 mL of a spore suspension of the local toxigenic isolate of <em>F. verticillioides</em>. The spore suspension contained 10<sup>6</sup> spores mL<sup>−1</sup> and was prepared following the protocol established by <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Reid <em>et al.</em> (1996)</a> with some modifications. Inoculum was injected into the center of the ear with a four-needle vaccinator that perforated the husks and injured three to four kernels. Ears from each plot were collected 2 mo after inoculation and were individually rated for <em>Fusarium</em> ear rot symptoms by the use of a seven-point scale (1 = no visible disease symptoms, 2 = 1–3%, 3 = 4–10%, 4 = 11–25%, 5 = 26–50%, 6 = 51–75%, and 7 = 76–100% of kernels exhibiting visual symptoms of infection, respectively) devised by <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Reid and Zhu (2005)</a>. Phenotypic data on the seven-point scale from the Galicia environments were transformed to the 0–100% scale used in North Carolina in the analyses. Reliable data could not be obtained for some line-environment combinations because seed set for some plots was limited as the result of poor adaptation. Raw data are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FileS1.csv">File S1</a>. Climate data from on-farm weather stations were obtained from <a href="http://www.nc-climate.ncsu.edu/">http://www.nc-climate.ncsu.edu/</a> and <a href="http://www2.meteogalicia.es/galego/observacion/estacions/listaEstacions.asp">http://www2.meteogalicia.es/galego/observacion/estacions/listaEstacions.asp</a>.</p></div><div id="sec-3" class="subsection"><h3>Genotypic data</h3><p id="p-11">The genotypic data were 47,445 SNPs from the Illumina maize 50k genotyping array filtered by <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Olukolu <em>et al.</em> (2013)</a>. The original array consists of 49,585 SNPs designed by <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Ganal <em>et al.</em> (2011)</a>. <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Olukolu <em>et al.</em> (2013)</a> filtered the data set to include only those SNP markers that mapped to defined single locations in the maize genome and had &lt;20% missing data (<a href="http://www.genetics.org/content/suppl/2012/12/05/genetics.112.147595.DC1/genetics.112.147595-3.txt">http://www.genetics.org/content/suppl/2012/12/05/genetics.112.147595.DC1/genetics.112.147595-3.txt</a>).</p></div><div id="sec-4" class="subsection"><h3>Statistical analyses</h3><div id="sec-5" class="subsection"><h4>Estimation of least-square means and heritabilities:</h4><p id="p-12">The Galicia and North Carolina experiments were analyzed separately and then combined in a single multienvironment analysis. Each year of data within each experiment was first analyzed separately by fitting a mixed linear model that included line as a fixed effect, silking date as a fixed linear covariate, and replication (Galicia only), block within replication (Galicia only), set (North Carolina only), and block within set (North Carolina only) as random effects. The mixed linear model for the Galicia experiment across years included line as a fixed effect, silking date as a fixed linear covariate, and year, line × year interaction, replication within year, and block within replication as random effects. The North Carolina experiment was analyzed across years with a model including line as a fixed effect, silking date as a fixed linear covariate, and year, line × year interaction, set within year, and block within set as random effects. In the combined experiment analysis, each combination of location and year was considered an environment. The combined analysis model included a fixed line effect, silking date as a fixed covariate nested within environment, a random line × environment interaction effect, and nested random experimental design effects (replication within environment and block within replication at Galicia and set within environment and block within set at North Carolina). All analyses were weighted by the number of ears scored within each plot and used a heterogeneous error variance structure. In both experiments, larger predicted ear rot values were associated with larger residuals, so a natural logarithmic transformation of raw ear rot scores (which largely eliminated the relationship between residual variance and predicted values) was used for all analyses. Least square means were estimated for 267 inbred lines within each experiment and across experiments (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FileS2.txt">File S2</a>) with the use of ASReml version 3 software (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gilmour <em>et al.</em> 2009</a>). Means for 12 lines were not estimable due to missing phenotypic observations in all environments (generally because of poor seed production).</p><p id="p-13">We conducted a second analysis treating inbred lines as random effects for the purposes of estimating heritability for <em>Fusarium</em> ear rot resistance in the diversity panel. The same models as above were used except lines were treated as random effects to obtain estimates of genetic variance. Line mean-basis heritability was estimated as<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2095/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2095/embed/mml-math-1.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2095/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2095/embed/mml-math-2.gif"/></noscript></span></span> is the average prediction error variance for all pairwise comparisons of lines and <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2095/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2095/embed/mml-math-3.gif"/></noscript></span></span> is the estimated genetic variance (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Cullis <em>et al.</em> 2006</a>). We estimated line mean-basis heritabilities for each environment individually, across the North Carolina environments, across the Galicia environments, and we also estimated line mean-basis heritability for the combined data set across all environments. The model used to estimate line mean-basis heritability in the combined data set was further modified by nesting the random line effect within environment and modeling the genotype-by-environment effect (<strong>G</strong>) matrix as unstructured, thereby allowing estimation of unique genetic variance within each environment and a unique genetic correlation between each pair of environments. For the purpose of estimating heritability, the average of the 10 pair-wise covariance estimates between environments (which are expected to equal the genotypic variance) was used in the denominator of the above equation.</p><p id="p-14">Silking date heritabilities also were calculated for each environment and across environments. The same models used to compute ear rot heritabilities were used to estimate silking date heritabilites, but silking date was treated as the dependent variable instead of as a fixed linear covariate.</p></div></div><div id="sec-6" class="subsection"><h3>Association analyses</h3><p id="p-15">A genetic kinship matrix (<strong>K</strong>; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FileS3.txt">File S3</a>) based on observed allele frequencies (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">VanRaden 2008</a>; method 1) was created using R software version 3.0.0 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">R Core Team 2013</a>). A subset of 4000 SNP markers were used to estimate <strong>K</strong>. Markers were uniformly distributed across the genome (at least 60 kbp between adjacent markers) and had no missing data after we excluded heterozygous genotypes. <a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Olukolu <em>et al.</em> (2013)</a> used a kinship matrix produced by Tassel software (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bradbury <em>et al.</em> 2007</a>), which is appropriate for population structure correction for GWAS. In addition to population structure correction, we also wanted to estimate the polygenic background genetic variance component, so we estimated a new <strong>K</strong> matrix that is scaled appropriately to represent realized genomic average identity by descent relationships among the lines (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">VanRaden 2008</a>).</p><p id="p-16">Tassel version 4.1.24 was used for the GWAS based on a mixed linear model (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Bradbury <em>et al.</em> 2007</a>). The least-square means for inbred lines were used as the input phenotypes, and each set of means (North Carolina, Galicia, and combined) was analyzed separately (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FileS2.txt">File S2</a>). The mixed linear model implemented by Tassel was<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2095/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2095/embed/mml-math-4.gif"/></noscript></span></span>where <em><strong>y</strong></em> is the vector of ear rot least square means (on the natural-log scale), <em><strong>β</strong></em> is a vector of fixed effects, including SNP marker effects, <em><strong>u</strong></em> is a vector of random additive genetic effects from background QTL for lines, <em><strong>X</strong></em> and <em><strong>Z</strong></em> are design matrices, and <em><strong>e</strong></em> is a vector of random residuals. The variance of the <em><strong>u</strong></em> vector was modeled as<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2095/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2095/embed/mml-math-5.gif"/></noscript></span></span>where <strong>K</strong> is the <em>n</em> × <em>n</em> matrix of pairwise kinship coefficients ranging 0−2 and <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2095/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2095/embed/mml-math-6.gif"/></noscript></span></span> is the estimated additive genetic variance (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Yu <em>et al.</em> 2006</a>).</p><p id="p-17">Restricted maximum likelihood estimates of variance components were obtained by use of the optimum compression level and population parameters previously determined options in Tassel (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Zhang <em>et al.</em> 2010</a>). The optimum compression level option reduces the dimensionality of <strong>K</strong> by clustering <em>n</em> lines into <em>s</em> groups on the basis of their genomic similarity, thereby reducing computational time and potentially improving model fit. The <em>P</em>-values for each of the 47,445 tests of associations between one SNP and ear rot resistance within each analysis were used to estimate the false-positive discovery rate using the QVALUE version 1.0 package in R (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Storey and Tibshirani 2003</a>). SNPs significant at <em>q</em> &lt; 0.10 in the initial GWAS scan for a particular environment set were then included together in a joint SNP association model together using the GLM procedure in SAS software version 9.2 (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">SAS Institute Inc 2010</a>) to estimate the total amount of variation explained by the SNPs together and to re-estimate their effects jointly. Candidate genes either containing or located adjacent to associated SNPs were identified using the MaizeGDB genome browser (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andorf <em>et al.</em> 2010</a>).</p></div><div id="sec-7" class="subsection"><h3>Allele frequency analysis</h3><p id="p-18">Lines were grouped into one of five major maize subpopulations (stiff stalk [SS], non-stiff stalk [NSS], tropical/subtropical [TS], popcorn [PC], and sweet corn [SC]) on the basis of the population structure analysis of the maize core diversity panel reported by <a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Flint-Garcia <em>et al.</em> (2005)</a> (<a href="http://panzea.org/db/gateway?file_id=pop_structure_xls">http://panzea.org/db/gateway?file_id=pop_structure_xls</a>). Lines of mixed ancestry (the result of admixture among the subpopulations) were dropped from the analysis. On the basis of the results of the association analyses, the frequencies of alleles that reduced disease severity at significant SNPs were estimated within each subpopulation using the FREQ procedure using SAS software version 9.2 (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">SAS Institute Inc 2010</a>). At each SNP locus, a Fisher’s exact test in R software version 3.0.0 (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">R Core Team 2013</a>) was used to test the null hypothesis that frequency of the allele conferring increased disease resistance was the same across all five subpopulations.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Line means and heritability</h3><p id="p-19">Significant (<em>P</em> &lt; 0.001) genotypic variation for ear rot resistance was observed in both the North Carolina and Galicia experiments. The mean ear rot observed among 267 inbred lines of the association panel ranged from 4.4 to 100% with an overall mean of 41.1% in North Carolina and from 0 to 89.3% with an overall mean of 7.4% in Galicia (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FileS2.txt">File S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/TableS1.pdf">Table S1</a>). In the combined analysis, mean ear rot ranged from 1.6 to 79.6% with an overall mean of 22.1%. The silking date covariate was highly heritable (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2095/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2095/embed/mml-math-7.gif"/></noscript></span></span> = 0.98 in the combined analysis) and was significantly associated with ear rot resistance in the North Carolina and combined analyses (<em>P</em> &lt; 0.001), but not in the Galicia analysis (<em>P</em> = 0.099, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/TableS1.pdf">Table S1</a>).</p><p id="p-20">A significant (<em>P</em> &lt; 0.001) line × environment interaction was detected in the combined analysis. Results of the mixed model analysis that estimate unique genotypic covariances for each pair of environments indicated that the two Galicia environments had a much stronger genotypic correlation (<em>r</em> = 0.93; <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FigureS2.pdf">Figure S2</a>) than did any other pair of environments (range, <em>r</em> = 0.28 to 0.51; <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FigureS2.pdf">Figure S2</a>). Thus, there was little genotype-by-environment (G × E) interaction between the two Galicia environments, and the heritability of line means across the 2 yr in Galicia was 0.71. In contrast, pair-wise genotypic correlations were much lower among the North Carolina environments and between North Carolina and Galicia environments (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FigureS2.pdf">Figure S2</a>), generating much of the observed G × E interaction in the combined analysis. Despite the strong G × E interaction among North Carolina environments, heritability of genotype means across the 3 yr in North Carolina (0.73) was greater than within any single North Carolina environment (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/TableS1.pdf">Table S1</a>). In addition, heritability of line means across all five environments was 0.75, greater than within any single environment or group of environments (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/TableS1.pdf">Table S1</a>). Therefore, we conducted separate association analyses on three different sets of genotypic mean values for ear rot: (1) means from three North Carolina environments, (2) means from two Galicia environments, and (3) means from the combined analysis of all five environments.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463556/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463556/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Genotypic covariance/variance/correlation matrix for <em>Fusarium</em> ear rot from the combined analysis of a maize diversity panel evaluated in five environments</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Association mapping of <em>Fusarium</em> ear rot resistance</h3><p id="p-22">The optimum compression option in Tassel clustered the 267 lines into 229 groups in the Galicia analysis and 197 groups in the North Carolina and combined analyses (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Background genetic effects modeled by <strong>K</strong> accounted for 31% of the total variation among line means in the North Carolina analysis, 57% of the total phenotypic variation in the Galicia analysis, and 48% of the total phenotypic variation in the combined analysis (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). In the analysis of means from North Carolina environments, two SNPs were identified as significantly associated with ear rot resistance at <em>q</em> ≤ 0.05 (raw <em>P</em>-value = 2.4 × 10<sup>−7</sup>), and one additional SNP was identified at <em>q</em> ≤ 0.10 (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). In the combined analysis, one SNP was identified as significantly associated with ear rot resistance at <em>q</em> ≤ 0.05 and coincided with one of the SNPs identified in the North Carolina analysis. No SNPs significant at <em>q</em> ≤ 0.10 were identified in the Galicia analysis, where the minimum raw <em>P</em>-value among SNP association tests was 2.1 × 10<sup>−4</sup>.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463580/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463580/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Number of lines, number of groups, compression level, polygenic additive background genetic variance component, residual genotypic variance component, and proportion of total line mean variance explained by additive relationship matrix from the three MLM analyses</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463605/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463605/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Chromosome locations (AGP v2 coordinates), allele effect estimates, genes containing or adjacent to SNP, and other summary statistics for the three SNPs significantly associated with <em>Fusarium</em> ear rot resistance in the North Carolina analysis and the single SNP associated with resistance in the combined analysis</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2095/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results of the three GWAS showing significant associations (points above red false-positive discovery rate = 0.10 threshold lines) in the North Carolina (A), Galicia (B), and combined (C) analyses. The vertical axis indicates &#x2013;log10 of P-value scores, and the horizontal axis indicates chromosomes and physical positions of SNPs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1372503085" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Results of the three GWAS showing significant associations (points above red false-positive discovery rate = 0.10 threshold lines) in the North Carolina (A), Galicia (B), and combined (C) analyses. The vertical axis indicates &#x2013;log&lt;sub&gt;10&lt;/sub&gt; of &lt;em&gt;P&lt;/em&gt;-value scores, and the horizontal axis indicates chromosomes and physical positions of SNPs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2095/F1.medium.gif" width="300" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/11/2095/F1.medium.gif" width="300" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2095/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2095/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463614" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-34" class="first-child">Results of the three GWAS showing significant associations (points above red false-positive discovery rate = 0.10 threshold lines) in the North Carolina (A), Galicia (B), and combined (C) analyses. The vertical axis indicates –log<sub>10</sub> of <em>P</em>-value scores, and the horizontal axis indicates chromosomes and physical positions of SNPs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h3>Candidate genes colocalized with associated SNPs</h3><p id="p-35">Genes containing or nearby SNPs significantly associated with ear rot resistance were characterized by the use of the filtered predicted gene set from the annotated B73 reference maize genome (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Schnable <em>et al.</em> 2009</a>). Two of the three genes identified in the North Carolina analysis have predicted functions that have been implicated in disease response pathways in other plant species (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tsunezuka <em>et al.</em> 2005</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hématy <em>et al.</em> 2009</a>). The SNP at physical position 151,295,233 bp on chromosome 9, which was identified in both the North Carolina and combined analyses, is located in an intronic region of a cellulose synthase-like family A/mannan synthase gene (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). Mean LD <em>r</em><sup>2</sup> between this SNP and other chromosome 9 SNPs dropped below 0.1 within approximately 100 kbp (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The other two SNPs identified in the North Carolina analysis on chromosomes 1 and 5 were located inside of a gene of unknown function and nearby a heat-shock 60-kD protein (HSP60), respectively. Mean LD <em>r</em><sup>2</sup> between the chromosome 1 and chromosome 5 SNPs and other SNPs dropped below 0.1 within approximately 10 kbp and 100 kbp, respectively (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Although the chromosome 1 and 9 SNPs were not significantly associated with ear rot resistance in Galicia, the allele effects at these loci were consistent between North Carolina and Galicia (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). However, the allele effect at the chromosome 5 SNP locus showed a change in direction between North Carolina (+1.149%, <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>) and Galicia (−0.017%).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2095/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LD heatmaps showing LD measure (r2) calculated for each pairwise combination of SNPs in an approximately &#xB1;1 Mbp region surrounding each SNP significantly associated with ear rot resistance in the North Carolina analysis. (A) LD around chromosome 1 SNP. (B) LD around chromosome 5 SNP. (C) LD around chromosome 9 SNP. The significant SNP on each chromosome is highlighted by the perpendicular black lines within each heatmap. Colors indicate the magnitude of each pairwise r2 measure (r2 = 1 is red to r2 = 0 is white)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1372503085" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;LD heatmaps showing LD measure (&lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt;) calculated for each pairwise combination of SNPs in an approximately &#xB1;1 Mbp region surrounding each SNP significantly associated with ear rot resistance in the North Carolina analysis. (A) LD around chromosome 1 SNP. (B) LD around chromosome 5 SNP. (C) LD around chromosome 9 SNP. The significant SNP on each chromosome is highlighted by the perpendicular black lines within each heatmap. Colors indicate the magnitude of each pairwise &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; measure (&lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; = 1 is red to &lt;em&gt;r&lt;/em&gt;&lt;sup&gt;2&lt;/sup&gt; = 0 is white).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2095/F2.medium.gif" width="394" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/11/2095/F2.medium.gif" width="394" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2095/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2095/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463559" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-36" class="first-child">LD heatmaps showing LD measure (<em>r</em><sup>2</sup>) calculated for each pairwise combination of SNPs in an approximately ±1 Mbp region surrounding each SNP significantly associated with ear rot resistance in the North Carolina analysis. (A) LD around chromosome 1 SNP. (B) LD around chromosome 5 SNP. (C) LD around chromosome 9 SNP. The significant SNP on each chromosome is highlighted by the perpendicular black lines within each heatmap. Colors indicate the magnitude of each pairwise <em>r</em><sup>2</sup> measure (<em>r</em><sup>2</sup> = 1 is red to <em>r</em><sup>2</sup> = 0 is white).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Allele frequencies at candidate genes</h3><p id="p-37">We estimated the allele frequency at the three SNPs significantly associated with ear rot resistance in five of the major maize subpopulations: SS, NSS, TS, PC, and SC (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Flint-Garcia <em>et al.</em> 2005</a>). European flint types are poorly represented in this maize core diversity panel and thus were not considered. Popcorn and sweet corn types were considered in the analysis, but comparisons with either of these two subpopulations may be less reliable than comparisons with other subpopulations because of smaller sample size (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The allele that reduced disease severity at the chromosome 1 SNP locus is only present in the NSS and TS subpopulations but not at high enough frequencies to be considered significantly different from the other three subpopulations (<em>P</em> = 0.15, <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). The allele with reduced disease severity at the chromosome 5 SNP locus is significantly (<em>P</em> = 6.2 × 10<sup>−6</sup>) over-represented in TS and PC lines relative to other temperate (SS, NSS, and SC) lines. At the chromosome 9 SNP locus, the allele associated with reduced disease severity is significantly (<em>P</em> = 3.846 × 10<sup>−4</sup>) overrepresented in PC lines compared with the other four subpopulations (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>). Averaging least square means from the combined analysis across members of each subpopulation, the SS, NSS, TS, PC, and SC subpopulations had average ear rot scores of 24.0%, 24.3%, 14.6%, 17.9%, and 46.5%, respectively (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>).</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463628/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463628/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Allele frequencies of significantly associated SNPs in the five major maize subpopulations</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><div id="sec-14" class="subsection"><h3>Heritability and false discovery rate estimation</h3><p id="p-40">The mean ear rot severity observed across experimental entries was 41.1% in North Carolina and 7.4% in Galicia (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/TableS1.pdf">Table S1</a>). Mean ear rot in North Carolina 2012 was particularly high (55%; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/TableS1.pdf">Table S1</a>). The very strong genotypic correlation between Galician environments (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FigureS2.pdf">Figure S2</a>) justified their grouping as one environmental set in the analysis. Genotypic effects were significantly correlated between each pair of North Carolina environments, but at much lower magnitude (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FigureS2.pdf">Figure S2</a>). Genotypic values in North Carolina 2010 had slightly higher correlations with the genotypic values in Galicia than in other years of North Carolina (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>), so grouping the three North Carolina environments has little justification on the basis of G × E patterns. Nevertheless, this environment grouping has a natural interpretation in terms of geography and adaptation, and the heritability of line means across these environments was greater than any individual environment, such that analysis of the three North Carolina environments as a group simplified interpretation of results.</p><p id="p-41">The relationship between the <em>F. verticillioides</em> isolates used in each location is unknown; as such, it is possible that differences in pathogen aggressiveness could have contributed to the disparity in mean ear rot values across environments. In addition, differences in inoculation methods, as well as variation in temperature and precipitation levels, may have allowed for more favorable disease development in North Carolina compared with Galicia. Although precipitation levels varied across all five environments, average daily temperatures (both pre- and postflowering) were greater in all three North Carolina environments compared with the two Galicia environments (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/TableS2.pdf">Table S2</a>).</p><p id="p-42">Heritabilities observed across environments in this study (<span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2095/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2095/embed/mml-math-13.gif"/></noscript></span></span>) are consistent with estimates from biparental populations (<a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Robertson <em>et al.</em> 2006</a>) and a small sample of North American and European public inbred lines (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Bolduan <em>et al.</em> 2009</a>). These heritability estimates were obtained with a model that assumed each line is a random sample from the reference population of global maize inbreds, modeled by a genotypic variance-covariance structure equal to the genotypic variance component multiplied by an identity matrix. For the purpose of controlling population structure in association analysis, adjusted line means from the original model were then used as observations in a mixed model analysis that modeled the genotypic variance-covariance structure as proportional to the realized genomic relationship matrix, thus incorporating the different pairwise relationships among the lines. This mixed model was simplified by the compression method of <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Zhang <em>et al.</em> (2010)</a>, which clusters lines according to genetic similarity and replaces the full pair-wise realized genomic relationship matrix with a reduced matrix of average relationships among the groups. The optimal level of clustering or compression is determined empirically based on model fit to the observed phenotypic data. A compressed relationship matrix can have better model fit than the original matrix when the empirically observed covariance relationships among lines follow the group relationship averages better than the individual pairwise relationships. Typically, this can happen when closely related lines are grouped and estimates of the group phenotypes and their relationships with other group phenotypes are improved. The optimal compression level can vary among phenotypes for the same set of lines, as observed in this study.</p><p id="p-43">Among environment groups, the proportion of phenotypic variance explained by background genetic effects (<strong>K</strong>) was much smaller in North Carolina (31%, <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) compared with Galicia (57%). Besides the small polygenic additive effects captured by the kinship matrix, rare allele variants (minor allele frequency &lt; 0.05) with larger effects, as well as epistatic interactions, may explain some of the genotypic variation not captured by either <strong>K</strong> or the significantly associated SNPs (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Manolio <em>et al.</em> 2009</a>).</p><p id="p-44">Analyzing the Galicia environments separately from the North Carolina environments revealed no significant SNPs, whereas the North Carolina analysis identified three SNPs significantly associated with <em>Fusarium</em> ear rot resistance (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). Examination of the empirical distribution of <em>P</em>-values for the Galicia analysis revealed a slight skew toward greater <em>P</em>-values, whereas the North Carolina and combined analyses exhibited excesses of small <em>P</em>-values (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FigureS4.pdf">Figure S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FigureS5.pdf">Figure S5</a>). The <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Storey and Tibshirani (2003)</a> method used to compute the false-discovery rate assumes that the distribution of <em>P</em>-values for truly null tests follows a flat distribution, such that if the observed proportion of very low <em>P</em>-values is lower than expected based on the flat distribution, the false discovery rate will be high even for the lowest <em>P</em>-values, as we observed in the Galicia analysis. Whereas a few significant SNPs were identified in the North Carolina and combined analyses at <em>q</em> &lt; 0.10, no SNP had a <em>q</em>-value of &lt;0.9 in the Galicia analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FigureS4.pdf">Figure S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/FigureS5.pdf">Figure S5</a>). The disparity between the two individual experiment analyses highlights the importance of conducting individual environment association analyses in the presence of significant G × E interaction. It should be noted, however, that the appropriate threshold proportion of variation caused by G × E interaction to warrant individual location analyses instead of an overall combined analysis is not clear.</p><p id="p-45">One possible mode of G×E interaction is the relative increase or decrease of additive allelic effects among different loci between environments (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Falconer and Mackay 1996</a>). Comparison of the absolute value of the allele effect at each of the identified SNP loci between North Carolina and Galicia revealed that allele effects were larger in North Carolina across all three loci (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>), congruent with the greater mean ear rot values in North Carolina (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1/TableS1.pdf">Table S1</a>). The largest proportion of phenotypic variance explained by <strong>K</strong> was in Galicia (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>), and when combined with comparatively smaller allele effects, suggested that more loci may have contributed to ear rot resistance in Galicia than North Carolina, and on average each locus had a smaller additive effect on disease phenotype in Galicia. Collectively, these two points may explain the deficiency of SNPs significantly associated with ear rot resistance in the Galicia analysis.</p></div><div id="sec-15" class="subsection"><h3>Association analyses</h3><p id="p-46">Three SNPs significantly associated with ear rot resistance were identified in the North Carolina analysis (<a id="xref-table-wrap-3-7" class="xref-table" href="#T3">Table 3</a>), and all localized to separate chromosomes. One of these three SNPs, located on chromosome 9, was also identified in the combined analysis. None of the three SNPs localized to any of the linkage map bins containing resistance QTL reported by <a id="xref-ref-40-6" class="xref-bibr" href="#ref-40">Robertson <em>et al.</em> (2006)</a> and <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Ding <em>et al.</em> (2008)</a>. However, the proportion of phenotypic variance explained by each SNP is consistent with individual QTL <em>r</em><sup>2</sup> values reported by each of the two aforementioned mapping studies. The chromosome 9 SNP explained the largest proportion of the variation in line mean values for ear rot resistance (<em>R</em><sup>2</sup> = 11.5% in NC and <em>R</em><sup>2</sup> = 9.6% in the combined analysis, <a id="xref-table-wrap-3-8" class="xref-table" href="#T3">Table 3</a>), whereas the chromosome 1 and chromosome 5 SNPs explained 8.8% and 9.6% of the variation in line mean values for ear rot resistance in North Carolina, respectively. Modeling all three SNPs together collectively explained 26% of the line mean variation in ear rot resistance in North Carolina.</p><p id="p-47">Although all three SNPs explained a relatively large portion of the total variation in line means, each SNP had a relatively small additive effect on ear rot resistance (±1.1 percentage points ear rot severity on the back-transformed scale, <a id="xref-table-wrap-3-9" class="xref-table" href="#T3">Table 3</a>). Additive genetic variance estimates for each SNP were computed based on allele effects and frequencies (<a id="xref-table-wrap-3-10" class="xref-table" href="#T3">Table 3</a>), and when scaled to the total line mean variance coincided with the SNP <em>r</em><sup>2</sup> values computed by Tassel (<a id="xref-table-wrap-3-11" class="xref-table" href="#T3">Table 3</a>). In every case, an increase in disease resistance (decrease in ear rot severity) was associated with the rare allele at each locus. Resistance alleles at the chromosome 1 and 5 SNP loci were overrepresented in the tropical subpopulation relative to the other temperate subpopulations (<a id="xref-table-wrap-3-12" class="xref-table" href="#T3">Table 3</a>), consistent with enriched disease resistance observed in tropical maize for some foliar diseases of maize (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wisser <em>et al.</em> 2011</a>; <a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Olukolu <em>et al.</em> 2013</a>) and the lower level of ear rot disease observed in tropical lines in this study.</p><p id="p-48">Using the same association panel and marker set as this study, <a id="xref-ref-33-7" class="xref-bibr" href="#ref-33">Olukolu <em>et al.</em> (2013)</a> reported that LD in the maize core diversity panel is variable across chromosomes and subpopulations. The authors also reported that marker pairs separated by more than 10 kbp had <em>r</em><sup>2</sup> &lt; 0.1 on average, which is consistent with estimates of <em>r</em><sup>2</sup> &lt; 0.1 between marker pairs separated by 5−10 kbp on average in tropical subpopulations and 10−100 kbp on average in temperature subpopulations (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lu <em>et al.</em> 2011</a>). Increased marker coverage, such as the genotype-by-sequencing data (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Elshire <em>et al.</em> 2011</a>) used in <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Romay <em>et al.</em> (2013)</a>, in conjunction with a larger association panel, may be able to uncover more SNPs in greater LD with ear rot resistance loci. Assuming an association panel of between 350 and 400 inbred lines, <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Van Inghelandt <em>et al.</em> (2011)</a> indicated that as few as 4000 markers would be necessary in a GWAS to detect individual QTL explaining &gt;10% of the total phenotypic variation for a complex trait within the stiff stalk subpopulation, whereas 65,000 markers would be required to detect QTL at the same threshold within European flint types. In a sample of 2815 inbred lines from the USA National Plant Germplasm System representing the same heterotic groups described in this study, <a id="xref-ref-43-5" class="xref-bibr" href="#ref-43">Romay <em>et al.</em> (2013)</a> reported that the use of more than 680,000 genotype-by-sequencing markers was sufficient to detect most known candidate genes associated with flowering time in maize. Even so, polymorphisms that strongly associated with the lower LD tropical/subtropical subpopulation (such as <em>ZmCTT</em>) were more difficult to detect compared with polymorphisms that more frequently associated with greater LD temperate subpopulations (such as <em>Vgt1</em>). The results of <a id="xref-ref-43-6" class="xref-bibr" href="#ref-43">Romay <em>et al.</em> (2013)</a> indicate that although increased marker coverage and association panel size can improve the power of a GWAS, care needs to be taken to ensure that lower LD subpopulations, such as the tropical/subtropical subpopulation, are adequately represented in an association panel in order to capture rare allele variants associated with those subpopulations.</p></div><div id="sec-16" class="subsection"><h3>Candidate genes</h3><p id="p-49">We used the B73 maize genome reference sequence to identify genes that either included or were nearby SNPs significantly associated with ear rot resistance. The chromosome 9 gene (GRMZM2G178880) that was identified in both the North Carolina and combined analyses belongs to the cellulose synthase-like family A (<em>CslA</em>) protein family. Given that the associated SNP localized to an intron segment within this gene, it is likely that this SNP is in LD with the causal variant and not the causal variant itself. The expression of this gene is highest in the endosperm of the developing seed kernel between 20 and 24 d after flowering during the growing season (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Sekhon <em>et al.</em> 2011</a>; <a href="http://www.plexdb.org">http://www.plexdb.org</a>). Peak expression of this gene coincides with the initial onset of <em>Fusarium</em> ear rot symptoms, which occurs approximately 28 d after flowering (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bush <em>et al.</em> 2004</a>). Genes in the <em>CslA</em> family encode for noncellulose polysaccharides (such as mannan polymers) that form part of the wall matrix in plant cells (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dhugga 2005</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Liepman <em>et al.</em> 2005</a>). In the model grass species <em>Brachypodium distachyon</em>, mannan polymers make up a significant portion of the seed endosperm (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Guillon <em>et al.</em> 2011</a>). Dismantling of mannan-rich cell walls may play an important role in programmed cell death in host-pathogen interactions (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gadjev <em>et al.</em> 2008</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Rodríguez-Gacio <em>et al.</em> 2012</a>). Although the interaction between <em>F. verticillioides</em> and maize is complex, cell wall structure and programmed cell death may play a role in quantitative resistant to the disease (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chivasa <em>et al.</em> 2005</a>).</p><p id="p-50">The SNP on chromosome 5 is located downstream of an HSP60 gene (GRMZM2G111477). Expression levels of this gene are highest in the developing endosperm 12 d after flowering (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Sekhon <em>et al.</em> 2011</a>; <a href="http://www.plexdb.org">http://www.plexdb.org</a>). HSP60s are chaperonins that are involved with protein folding under plant stress primarily in the mitochondria and cholorplasts (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2004</a>). The role of HSP60s in programmed cell death has been demonstrated in mutants of both rice and Arabidopsis (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Ishikawa <em>et al.</em> 2003</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Tsunezuka <em>et al.</em> 2005</a>). The SNP on chromosome 1 is contained within the coding region of GRMZM2G703598. Unfortunately, this gene has no predicted function and has no sequence orthology with related grass species.</p><p id="p-51">In conclusion, we used GWAS to identify three novel loci associated with improved resistance to <em>Fusarium</em> ear rot in maize. The identified loci each explain a relatively small proportion of the overall phenotypic variance for ear rot, and each locus has a very small additive genetic effect on resistance, consistent with the highly quantitative nature of the <em>F. verticillioides</em>-maize pathosystem. The large amount of variation captured by the kinship matrix, in combination with high false discovery rates for the vast majority of SNPs, suggests that additive polygenic variation across many loci underlies resistance to <em>Fusarium</em> ear rot. Given the rapid decay of LD along the chromosomes in the maize core diversity panel (<a id="xref-ref-33-8" class="xref-bibr" href="#ref-33">Olukolu <em>et al.</em> 2013</a>), future studies employing increased marker density and larger association panels may be able to identify other novel loci associated with ear rot resistance. Maize breeders can use targeted allele selection for these three resistance alleles, but may need to also select for recombinations near them as they are introgressed into elite maize from unadapted or undesirable genotypes (such as the tropical maize or popcorn germplasm pools that appear to be enriched for resistance alleles). In addition, given the substantial additive polygenic variation for ear rot resistance, phenotypic and genomic selection approaches should be effective as long as high quality phenotypic evaluations of resistance can be performed to permit direct selection or provide training data for genomic selection models.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-52">We thank Jason Brewer, Steve Pigozzo, David Horne, Brittany Scott, Sarah Davidson-Dyer, and Colt Jackson for technical assistance and Bode Olukolu, Funda Ogut, and Shang Xue for assistance with the analyses. We also thank the anonymous reviewers for helpful suggestions. This research was supported by the National Science Foundation (projects DBI-0321467, IOS-0820619, and IOS-0604923), USDA-ARS, the National Plan for Research and Development of Spain (AGL2009-12770), and the Excma. Diputación Provincial de Pontevedra. USDA-ARS supported the graduate fellowship of C.T.Z. R.S. acknowledges postdoctoral contract “Isidro Parga Pondal,” supported by the Autonomous Government of Galicia and the European Social Fund.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007328/-/DC1</a></p></li><li class="fn" id="fn-16"><p id="p-53"><em>Communicating editor: R. W. Doerge</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-06-21"><span class="received-label">Received </span>June 21, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-09-11"><span class="accepted-label">Accepted </span>September 11, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Zila <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.1" data-doi="10.1093/bioinformatics/btp556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andorf</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harper</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaeffer</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Locus Lookup tool at MaizeGDB: identification of genomic regions in maize by integrating sequence information with physical and genetic maps.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">434</span>–<span class="cit-lpage">436</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp556%26rft_id%253Dinfo%253Apmid%252F20124413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNi8zLzQzNCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-book no-rev-xref" id="cit-3.11.2095.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Phenotypic and genetic variances</span>, pp. <span class="cit-fpage">89</span>–<span class="cit-lpage">116</span> in <span class="cit-source">Breeding for Quantitative Traits in Plants</span>. <span class="cit-publ-name">Stemma Press</span>, <span class="cit-publ-loc">Woodbury, MN</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.3" data-doi="10.2135/cropsci2008.12.0701"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolduan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miedaner</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schipprack</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic variation for resistance to ear rots and mycotoxins contamination in early European maize inbred lines.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">2019</span>–<span class="cit-lpage">2028</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DBolduan%26rft.auinit1%253DC.%26rft.volume%253D49%26rft.issue%253D6%26rft.spage%253D2019%26rft.epage%253D2028%26rft.atitle%253DGenetic%2BVariation%2Bfor%2BResistance%2Bto%2BEar%2BRots%2Band%2BMycotoxins%2BContamination%2Bin%2BEarly%2BEuropean%2BMaize%2BInbred%2BLines%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.12.0701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.12.0701&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.4" data-doi="10.1093/bioinformatics/btm308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroon</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casstevens</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramdoss</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">TASSEL: software for association mapping of complex traits in diverse samples.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">2633</span>–<span class="cit-lpage">2635</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm308%26rft_id%253Dinfo%253Apmid%252F17586829%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjMvMTkvMjYzMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.5" data-doi="10.1094/PHYTO.2004.94.1.88"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bush</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carson</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cubeta</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hagler</span>  <span class="cit-name-given-names">W. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payne</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Infection and fumonisin production by <em>Fusarium verticillioides</em> in developing maize kernels.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">88</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DBush%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D94%26rft.issue%253D1%26rft.spage%253D88%26rft.epage%253D93%26rft.atitle%253DInfection%2Band%2BFumonisin%2BProduction%2Bby%2BFusarium%2Bverticillioides%2Bin%2BDeveloping%2BMaize%2BKernels.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO.2004.94.1.88%26rft_id%253Dinfo%253Apmid%252F18943824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO.2004.94.1.88&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18943824&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.6" data-doi="10.1002/pmic.200500047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chivasa</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">X.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yalpani</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slabas</span>  <span class="cit-name-given-names">A. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Pathogen elicitor-induced changes in the maize extracellular matrix proteome.</span> <abbr class="cit-jnl-abbrev">Proteomics</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">4894</span>–<span class="cit-lpage">4904</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProteomics%26rft.stitle%253DProteomics%26rft.aulast%253DChivasa%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D18%26rft.spage%253D4894%26rft.epage%253D4904%26rft.atitle%253DPathogen%2Belicitor-induced%2Bchanges%2Bin%2Bthe%2Bmaize%2Bextracellular%2Bmatrix%2Bproteome.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpmic.200500047%26rft_id%253Dinfo%253Apmid%252F16281185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pmic.200500047&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16281185&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234413400024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.7" data-doi="10.1094/PDIS.2003.87.2.147"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clements</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleinschmidt</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maragos</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pataky</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Evaluation of inoculation techniques for Fusarium ear rot and fumonisin contamination of corn.</span> <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D87%26rft.spage%253D147%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPDIS.2003.87.2.147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PDIS.2003.87.2.147&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.8" data-doi="10.1094/PHYTO.2004.94.3.251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clements</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maragos</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pataky</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">D. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Sources of resistance to fumonisin accumulation in grain and Fusarium ear and kernel rot of corn.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DClements%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D94%26rft.issue%253D3%26rft.spage%253D251%26rft.epage%253D260%26rft.atitle%253DSources%2Bof%2Bresistance%2Bto%2Bfumonisin%2Baccumulation%2Bin%2Bgrain%2Band%2Bfusarium%2Bear%2Band%2Bkernel%2Brot%2Bof%2Bcorn.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO.2004.94.3.251%26rft_id%253Dinfo%253Apmid%252F18943973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO.2004.94.3.251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18943973&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220532500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.9" data-doi="10.1104/pp.111.185033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMullen</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genetic architecture of maize kernel composition in the nested association mapping and inbred association panels.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">824</span>–<span class="cit-lpage">834</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.185033%26rft_id%253Dinfo%253Apmid%252F22135431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTgvMi84MjQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.10" data-doi="10.1198/108571106X154443"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coombes</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">On the design of early generation variety trials with correlated data.</span> <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D11%26rft.spage%253D381%26rft_id%253Dinfo%253Adoi%252F10.1198%252F108571106X154443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/108571106X154443&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000242477500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.11" data-doi="10.1073/pnas.0409649102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dhugga</span>  <span class="cit-name-given-names">K. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Plant Golgi cell wall synthesis: from genes to enzyme activities.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">1815</span>–<span class="cit-lpage">1816</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0409649102%26rft_id%253Dinfo%253Apmid%252F15684049%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAyLzYvMTgxNSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.12" data-doi="10.1007/s11032-008-9184-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">J.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">X.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chander</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">J.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.-S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">QTL mapping of resistance to Fusarium ear rot using a RIL population in maize.</span> <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">395</span>–<span class="cit-lpage">403</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D22%26rft.spage%253D395%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-008-9184-4%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-008-9184-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.13" data-doi="10.1080/15569540802450326"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eller</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payne</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Breeding for improved resistance to fumonisin contamination in maize.</span> <abbr class="cit-jnl-abbrev">Toxin Rev.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">371</span>–<span class="cit-lpage">389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DToxin%2BRev.%26rft.volume%253D27%26rft.spage%253D371%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15569540802450326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15569540802450326&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.14" data-doi="10.2135/cropsci2009.11.0683"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eller</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payne</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Selection for reduced Fusarium ear rot and fumonisin content in advanced backcross maize lines and their topcross hybrids.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">2249</span>–<span class="cit-lpage">2260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D2249%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.11.0683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.11.0683&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.15" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2095.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>  <span class="cit-name-given-names">D. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Correlated characters</span>, pp. <span class="cit-fpage">312</span>–<span class="cit-lpage">334</span> in <span class="cit-source">Introduction to Quantitative Genetics</span>. <span class="cit-publ-name">Longman</span>, <span class="cit-publ-loc">Essex, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.17" data-doi="10.1111/j.1365-313X.2005.02591.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint-Garcia</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thuillet</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romero</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Maize association population: a high-resolution platform for quantitative trait locus dissection.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">1054</span>–<span class="cit-lpage">1064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DFlint-Garcia%26rft.auinit1%253DS.%2BA.%26rft.volume%253D44%26rft.issue%253D6%26rft.spage%253D1054%26rft.epage%253D1064%26rft.atitle%253DMaize%2Bassociation%2Bpopulation%253A%2Ba%2Bhigh-resolution%2Bplatform%2Bfor%2Bquantitative%2Btrait%2Blocus%2Bdissection.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2005.02591.x%26rft_id%253Dinfo%253Apmid%252F16359397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2005.02591.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16359397&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233776900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.18" data-doi="10.1016/S1937-6448(08)01403-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gadjev</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gechev</span>  <span class="cit-name-given-names">T. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Programmed cell death in plants: new insights into redox regulation and the role of hydrogen peroxide.</span> <abbr class="cit-jnl-abbrev">Int. Rev. Cell Mol. Biol.</abbr> <span class="cit-vol">270</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">144</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DInt%2BRev%2BCell%2BMol%2BBiol%26rft.aulast%253DGadjev%26rft.auinit1%253DI.%26rft.volume%253D270%26rft.spage%253D87%26rft.epage%253D144%26rft.atitle%253DProgrammed%2Bcell%2Bdeath%2Bin%2Bplants%253A%2Bnew%2Binsights%2Binto%2Bredox%2Bregulation%2Band%2Bthe%2Brole%2Bof%2Bhydrogen%2Bperoxide.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1937-6448%252808%252901403-2%26rft_id%253Dinfo%253Apmid%252F19081535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1937-6448(08)01403-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19081535&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260776100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.19" data-doi="10.1371/journal.pone.0028334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durstewitz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polley</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bérard</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A large maize (<em>Zea mays</em> L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e28334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGanal%26rft.auinit1%253DM.%2BW.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De28334%26rft.epage%253De28334%26rft.atitle%253DA%2Blarge%2Bmaize%2B%2528Zea%2Bmays%2BL.%2529%2BSNP%2Bgenotyping%2Barray%253A%2Bdevelopment%2Band%2Bgermplasm%2Bgenotyping%252C%2Band%2Bgenetic%2Bmapping%2Bto%2Bcompare%2Bwith%2Bthe%2BB73%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0028334%26rft_id%253Dinfo%253Apmid%252F22174790%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0028334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22174790&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2095.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilmour</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gogel</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-source">ASReml User Guide Release 3.0</span>. <span class="cit-publ-name">VSN International Ltd</span>, <span class="cit-publ-loc">Hemel Hempstead, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.21" data-doi="10.1093/jxb/erq332"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guillon</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bouchet</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jamme</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quéméner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>Brachypodium distachyon</em> grain: characterization of endosperm cell walls.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">1001</span>–<span class="cit-lpage">1015</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferq332%26rft_id%253Dinfo%253Apmid%252F21062963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjk6IjYyLzMvMTAwMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.22" data-doi="10.1016/j.pbi.2009.06.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hématy</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cherk</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Somerville</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Host–pathogen warfare at the plant cell wall.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">406</span>–<span class="cit-lpage">413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DHematy%26rft.auinit1%253DK.%26rft.volume%253D12%26rft.issue%253D4%26rft.spage%253D406%26rft.epage%253D413%26rft.atitle%253DHost-pathogen%2Bwarfare%2Bat%2Bthe%2Bplant%2Bcell%2Bwall.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2009.06.007%26rft_id%253Dinfo%253Apmid%252F19616468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2009.06.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19616468&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269384900004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.23" data-doi="10.1016/j.pbi.2007.01.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic architecture of complex traits in plants.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">156</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bplant%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BPlant%2BBiol%26rft.aulast%253DHolland%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D10%26rft.issue%253D2%26rft.spage%253D156%26rft.epage%253D161%26rft.atitle%253DGenetic%2Barchitecture%2Bof%2Bcomplex%2Btraits%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.pbi.2007.01.003%26rft_id%253Dinfo%253Apmid%252F17291822%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.pbi.2007.01.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17291822&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245573100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.24" data-doi="10.1093/pcp/pcg031"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ishikawa</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakai</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asahi</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Deletion of a chaperonin 60β gene leads to cell death in the Arabidopsis <em>lesion initiation 1</em> mutant.</span> <abbr class="cit-jnl-abbrev">Plant Cell Physiol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Band%2BCell%2BPhysiology%26rft.stitle%253DPlant%2BCell%2BPhysiol%26rft.aulast%253DIshikawa%26rft.auinit1%253DA.%26rft.volume%253D44%26rft.issue%253D3%26rft.spage%253D255%26rft.epage%253D261%26rft.atitle%253DDeletion%2Bof%2Ba%2BChaperonin%2B60%257Bbeta%257D%2BGene%2BLeads%2Bto%2BCell%2BDeath%2Bin%2Bthe%2BArabidopsis%2Blesion%2Binitiation%2B1%2BMutant%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpcg031%26rft_id%253Dinfo%253Apmid%252F12668771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGNlbGxwaHlzIjtzOjU6InJlc2lkIjtzOjg6IjQ0LzMvMjU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scott</span>  <span class="cit-name-given-names">G. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Genotypic differences in maize to kernel infection by <em>Fusarium moniliforme</em>.</span> <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">1245</span>–<span class="cit-lpage">1247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.volume%253D71%26rft.spage%253D1245%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1981MV02800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.26" data-doi="10.1073/pnas.0409179102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liepman</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilkerson</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keegstra</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Expression of cellulose synthase-like (<em>Csl</em>) genes in insect cells reveals that <em>CslA</em> family members encode mannan synthases.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">2221</span>–<span class="cit-lpage">2226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0409179102%26rft_id%253Dinfo%253Apmid%252F15647349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAyLzYvMjIyMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.27" data-doi="10.1371/journal.pone.0024861"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hao</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taba</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Comparative SNP and haplotype analysis reveals a higher genetic diversity and rapider LD decay in tropical than temperate germplasm in maize.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e24861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLu%26rft.auinit1%253DY.%26rft.volume%253D6%26rft.issue%253D9%26rft.spage%253De24861%26rft.epage%253De24861%26rft.atitle%253DComparative%2BSNP%2Band%2Bhaplotype%2Banalysis%2Breveals%2Ba%2Bhigher%2Bgenetic%2Bdiversity%2Band%2Brapider%2BLD%2Bdecay%2Bin%2Btropical%2Bthan%2Btemperate%2Bgermplasm%2Bin%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0024861%26rft_id%253Dinfo%253Apmid%252F21949770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0024861&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21949770&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.28" data-doi="10.1038/nature08494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manolio</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">F. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldstein</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hindorff</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Finding the missing heritability of complex diseases.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>: <span class="cit-fpage">747</span>–<span class="cit-lpage">753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DManolio%26rft.auinit1%253DT.%2BA.%26rft.volume%253D461%26rft.issue%253D7265%26rft.spage%253D747%26rft.epage%253D753%26rft.atitle%253DFinding%2Bthe%2Bmissing%2Bheritability%2Bof%2Bcomplex%2Bdiseases.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08494%26rft_id%253Dinfo%253Apmid%252F19812666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08494&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19812666&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270547500027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.29" data-doi="10.1007/978-1-4899-1379-1_1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marasas</span>  <span class="cit-name-given-names">W. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Fumonisins: history, world-wide occurrence and impact.</span> <abbr class="cit-jnl-abbrev">Adv. Exp. Med. Biol.</abbr> <span class="cit-vol">392</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bexperimental%2Bmedicine%2Band%2Bbiology%26rft.stitle%253DAdv%2BExp%2BMed%2BBiol%26rft.aulast%253DMarasas%26rft.auinit1%253DW.%2BF.%26rft.volume%253D392%26rft.spage%253D1%26rft.epage%253D17%26rft.atitle%253DFumonisins%253A%2Bhistory%252C%2Bworld-wide%2Boccurrence%2Band%2Bimpact.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4899-1379-1_1%26rft_id%253Dinfo%253Apmid%252F8850601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4899-1379-1_1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8850601&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.30" data-doi="10.1111/j.1439-0523.2011.01936.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mesterházy</span>  <span class="cit-name-given-names">Á.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lemmens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Breeding for resistance to ear rots caused by <em>Fusarium</em> spp. in maize—a review.</span> <abbr class="cit-jnl-abbrev">Plant Breed.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">19</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBreed.%26rft.volume%253D131%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0523.2011.01936.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0523.2011.01936.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2095.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trenholm</span>  <span class="cit-name-given-names">H. L.</span></span></li></ol><cite> (Editors), <span class="cit-pub-date">1994</span> <span class="cit-article-title">The biosynthetic diversity of secondary metabolites; Epidemiology of Fusarium ear diseases of cereals</span>, pp. <span class="cit-fpage">3</span>–<span class="cit-lpage">36</span> in <span class="cit-source">Mycotoxins in Grain: Compounds Other Than Aflatoxin</span>. <span class="cit-publ-name">Eagan Press</span>, <span class="cit-publ-loc">St. Paul, MN</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.32" data-doi="10.1094/PD-80-0593"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nankam</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pataky</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Resistance to kernel infection by <em>Fusarium moniliforme</em> in the sweet corn inbred IL 125b.</span> <abbr class="cit-jnl-abbrev">Plant Dis.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">598</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDis.%26rft.volume%253D80%26rft.spage%253D593%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPD-80-0593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PD-80-0593&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.33" data-doi="10.1534/genetics.112.147595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Olukolu</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Negeri</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhawan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Venkata</span>  <span class="cit-name-given-names">B. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A connected set of genes associated with programmed cell death implicated in controlling the hypersensitive response in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">609</span>–<span class="cit-lpage">620</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.147595%26rft_id%253Dinfo%253Apmid%252F23222653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvNjA5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Brito</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffers</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">González-de-León</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khairallah</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cortés-Cruz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">QTL mapping of <em>Fusarium moniliforme</em> ear rot resistance in highland maize, Mexico.</span> <abbr class="cit-jnl-abbrev">Agrociencia</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">181</span>–<span class="cit-lpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAgrociencia%26rft.volume%253D35%26rft.spage%253D181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.35" data-doi="10.1016/j.cropro.2007.08.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Presello</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botta</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iglesias</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eyhérabide</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Effect of disease severity on yield and grain fumonisin concentration of maize hybrids inoculated with <em>Fusarium verticillioides</em>.</span> <abbr class="cit-jnl-abbrev">Crop Prot.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">572</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BProt.%26rft.volume%253D27%26rft.spage%253D572%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cropro.2007.08.015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cropro.2007.08.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2095.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source">R: A Language and Environment for Statistical Computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2095.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">R. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Screening Maize for Resistance to Gibberella Ear Rot</span>. <span class="cit-publ-name">Agriculture and Agri-Food Canada</span>, <span class="cit-publ-loc">Ottawa, ON</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2095.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reid</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-source">Screening Corn for Resistance to Common Diseases in Canada</span>. <span class="cit-publ-name">Agriculture and Agri-Food Canada</span>, <span class="cit-publ-loc">Ottawa, ON</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.39" data-doi="10.1086/367812"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Remington</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purugganan</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Candidate genes, quantitative trait loci, and functional trait evolution in plants.</span> <abbr class="cit-jnl-abbrev">Int. J. Plant Sci.</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">S7</span>–<span class="cit-lpage">S20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInt.%2BJ.%2BPlant%2BSci.%26rft.volume%253D164%26rft.spage%253DS7%26rft_id%253Dinfo%253Adoi%252F10.1086%252F367812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/367812&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000184018500002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.40" data-doi="10.2135/cropsci2005.0139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleinschmidt</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payne</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maragos</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Heritabilities and correlations of Fusarium ear rot resistance and fumonisin contamination resistance in two maize populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">353</span>–<span class="cit-lpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DRobertson%26rft.auinit1%253DL.%2BA.%26rft.volume%253D46%26rft.issue%253D1%26rft.spage%253D353%26rft.epage%253D361%26rft.atitle%253DHeritabilities%2Band%2BCorrelations%2Bof%2BFusarium%2BEar%2BRot%2BResistance%2Band%2BFumonisin%2BContamination%2BResistance%2Bin%2BTwo%2BMaize%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.0139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.0139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000235103700047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.41" data-doi="10.2135/cropsci2005.12-0450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson-Hoyt</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jines</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balint-Kurti</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleinschmidt</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">QTL mapping for Fusarium ear rot and fumonisin contamination resistance in two maize populations.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1734</span>–<span class="cit-lpage">1743</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DRobertson-Hoyt%26rft.auinit1%253DL.%2BA.%26rft.volume%253D46%26rft.issue%253D4%26rft.spage%253D1734%26rft.epage%253D1743%26rft.atitle%253DQTL%2BMapping%2Bfor%2BFusarium%2BEar%2BRot%2Band%2BFumonisin%2BContamination%2BResistance%2Bin%2BTwo%2BMaize%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2005.12-0450%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2005.12-0450&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000239054100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.42" data-doi="10.1093/jxb/ers096"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodríguez-Gacio</span>  <span class="cit-name-given-names">Mdel C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iglesias-Fernández</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carbonero</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matilla</span>  <span class="cit-name-given-names">Á. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Softening-up mannan-rich cell walls.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">3976</span>–<span class="cit-lpage">3988</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Fers096%26rft_id%253Dinfo%253Apmid%252F22553284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjEwOiI2My8xMS8zOTc2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.43" data-doi="10.1186/gb-2013-14-6-r55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Romay</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Millard</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peiffer</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarts</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comprehensive genotyping of the USA national maize inbred seed bank.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R55</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR55%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-6-r55%26rft_id%253Dinfo%253Apmid%252F23759205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-6-r55&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23759205&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2095.44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">SAS Institute Inc</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">SAS 9.2 Intelligence Platform: System Administration Guide</span>. <span class="cit-publ-name">SAS Institute Inc</span>, <span class="cit-publ-loc">Cary, NC</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.45" data-doi="10.1126/science.1178534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ware</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The B73 maize genome: complexity, diversity, and dynamics.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1112</span>–<span class="cit-lpage">1115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSchnable%26rft.auinit1%253DP.%2BS.%26rft.volume%253D326%26rft.issue%253D5956%26rft.spage%253D1112%26rft.epage%253D1115%26rft.atitle%253DThe%2BB73%2BMaize%2BGenome%253A%2BComplexity%252C%2BDiversity%252C%2Band%2BDynamics%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1178534%26rft_id%253Dinfo%253Apmid%252F19965430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ni8xMTEyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.46" data-doi="10.1111/j.1365-313X.2011.04527.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sekhon</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Childs</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansey</span>  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buell</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide atlas of transcription during maize development.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">563</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DSekhon%26rft.auinit1%253DR.%2BS.%26rft.volume%253D66%26rft.issue%253D4%26rft.spage%253D553%26rft.epage%253D563%26rft.atitle%253DGenome-wide%2Batlas%2Bof%2Btranscription%2Bduring%2Bmaize%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2011.04527.x%26rft_id%253Dinfo%253Apmid%252F21299659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2011.04527.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21299659&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000290456400001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.47" data-doi="10.1073/pnas.1530509100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Storey</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tibshirani</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Statistical significance for genomewide studies.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">9440</span>–<span class="cit-lpage">9445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1530509100%26rft_id%253Dinfo%253Apmid%252F12883005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAwLzE2Lzk0NDAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.48" data-doi="10.1094/MPMI-18-0052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsunezuka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujiwara</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawasaki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimamoto</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Proteome analysis of programmed cell death and defense signaling using the rice lesion mimic mutant <em>cdr2</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Plant Microbe Interact.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">59</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bplant-microbe%2Binteractions%2B%253A%2B%2BMPMI%26rft.stitle%253DMol%2BPlant%2BMicrobe%2BInteract%26rft.aulast%253DTsunezuka%26rft.auinit1%253DH.%26rft.volume%253D18%26rft.issue%253D1%26rft.spage%253D52%26rft.epage%253D59%26rft.atitle%253DProteome%2Banalysis%2Bof%2Bprogrammed%2Bcell%2Bdeath%2Band%2Bdefense%2Bsignaling%2Busing%2Bthe%2Brice%2Blesion%2Bmimic%2Bmutant%2Bcdr2.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FMPMI-18-0052%26rft_id%253Dinfo%253Apmid%252F15672818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/MPMI-18-0052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15672818&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225925700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.49" data-doi="10.1007/s00216-007-1317-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Egmond</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schothorst</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jonker</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Regulations relating to mycotoxins in food; Perspectives in a global and European context.</span> <abbr class="cit-jnl-abbrev">Anal. Bioanal. Chem.</abbr> <span class="cit-vol">389</span>: <span class="cit-fpage">147</span>–<span class="cit-lpage">157</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2Band%2Bbioanalytical%2Bchemistry%26rft.stitle%253DAnal%2BBioanal%2BChem%26rft.aulast%253DMusshoff%26rft.auinit1%253DF.%26rft.volume%253D389%26rft.issue%253D1%26rft.spage%253D147%26rft.epage%253D157%26rft.atitle%253DRegulations%2Brelating%2Bto%2Bmycotoxins%2Bin%2Bfood%253A%2Bperspectives%2Bin%2Ba%2Bglobal%2Band%2BEuropean%2Bcontext.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00216-007-1317-9%26rft_id%253Dinfo%253Apmid%252F17508207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00216-007-1317-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17508207&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248974300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.50" data-doi="10.1007/s00122-011-1562-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van Inghelandt</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reif</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dhillon</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flament</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Melchinger</span>  <span class="cit-name-given-names">A. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extent and genome-wide distribution of linkage disequilibrium in commercial maize germplasm.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">123</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DVan%2BInghelandt%26rft.auinit1%253DD.%26rft.volume%253D123%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D20%26rft.atitle%253DExtent%2Band%2Bgenome-wide%2Bdistribution%2Bof%2Blinkage%2Bdisequilibrium%2Bin%2Bcommercial%2Bmaize%2Bgermplasm.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1562-3%26rft_id%253Dinfo%253Apmid%252F21404061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1562-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21404061&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.51" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions.</span> <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.52" data-doi="10.1016/j.tplants.2004.03.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vinocur</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shoseyov</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altman</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Role of plant heat-shock proteins and molecular chaperones in the abiotic stress response.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">244</span>–<span class="cit-lpage">252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DWang%26rft.auinit1%253DW.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D244%26rft.epage%253D252%26rft.atitle%253DRole%2Bof%2Bplant%2Bheat-shock%2Bproteins%2Band%2Bmolecular%2Bchaperones%2Bin%2Bthe%2Babiotic%2Bstress%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tplants.2004.03.006%26rft_id%253Dinfo%253Apmid%252F15130550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tplants.2004.03.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15130550&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221661000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.53" data-doi="10.1073/pnas.1011739108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wisser</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolkman</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patzoldt</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holland</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multivariate analysis of maize disease resistances suggests a pleiotropic genetic basis and implicates a <em>GST</em> gene.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">7339</span>–<span class="cit-lpage">7344</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1011739108%26rft_id%253Dinfo%253Apmid%252F21490302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzE4LzczMzkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.54" data-doi="10.1016/j.copbio.2006.02.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genetic association mapping and genome organization of maize.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Biotechnol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.stitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D17%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D160%26rft.atitle%253DGenetic%2Bassociation%2Bmapping%2Band%2Bgenome%2Borganization%2Bof%2Bmaize.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.copbio.2006.02.003%26rft_id%253Dinfo%253Apmid%252F16504497%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.copbio.2006.02.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16504497&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237135800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.55" data-doi="10.1038/ng1702"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pressoir</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Briggs</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vroh Bi</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamasaki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">203</span>–<span class="cit-lpage">208</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYu%26rft.auinit1%253DJ.%26rft.volume%253D38%26rft.issue%253D2%26rft.spage%253D203%26rft.epage%253D208%26rft.atitle%253DA%2Bunified%2Bmixed-model%2Bmethod%2Bfor%2Bassociation%2Bmapping%2Bthat%2Baccounts%2Bfor%2Bmultiple%2Blevels%2Bof%2Brelatedness.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1702%26rft_id%253Dinfo%253Apmid%252F16380716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1702&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16380716&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234953200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2095.56" data-doi="10.1038/ng.546"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ersoz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lai</span>  <span class="cit-name-given-names">C.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todhunter</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiwari</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mixed linear model approach adapted for genome-wide association studies.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D42%26rft.issue%253D4%26rft.spage%253D355%26rft.epage%253D360%26rft.atitle%253DMixed%2Blinear%2Bmodel%2Bapproach%2Badapted%2Bfor%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.546%26rft_id%253Dinfo%253Apmid%252F20208535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.546&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20208535&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2095.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276150500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/11/2095.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/2085" title="Dissecting Genome-Wide Association Signals for Loss-of-Function Phenotypes in Sorghum Flavonoid Pigmentation Traits" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/12/2105" title="Increased Genomic Prediction Accuracy in Wheat Breeding Through Spatial Adjustment of Field Trial Data" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/11"><span>Volume 3 Issue 11, November 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/11.cover-source.jpg?itok=A7iZ1ztq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/2095" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ZPdr4xvdh1gs7fnhRJmmZgTnV_tuaolhV645xng6NdU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/11/2095" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/11/2095" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/11/2095" class="active" data-icon-position="" data-hide-link-title="0">A Genome-Wide Association Study Reveals Genes Associated with Fusarium Ear Rot Resistance in a Maize Core Diversity Panel</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459574" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-tcgR0fGhP8GqieO3ontXZgBJC-32HCwSZA3vJadoKvM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/2095.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/11/2095" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-aN3VtXSt4TXwjnq4ZSrX4-59BzK1ZTNain9DaDUrB8k" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/11/2095" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/2095.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459574" id="node459574--2" data-pisa="ggg;3/11/2095" data-pisa-master="ggg;g3.113.007328" data-apath="/ggg/3/11/2095.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genome-Wide Association Study Reveals Genes Associated with Fusarium Ear Rot Resistance in a Maize Core Diversity Panel</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Charles T. Zila</span>, <span class="highwire-citation-author" data-delta="1">L. Fernando Samayoa</span>, <span class="highwire-citation-author" data-delta="2">Rogelio Santiago</span>, <span class="highwire-citation-author" data-delta="3">Ana Butrón</span> and <span class="highwire-citation-author" data-delta="4">James B. Holland</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2095-2104;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007328 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459574--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charles T. Zila </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Science, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BT.%2BZila%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zila%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BT.%2BZila%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">L. Fernando Samayoa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Misión Biológica de Galicia, CSIC, Pontevedra, Spain, 36080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L.%2BFernando%2BSamayoa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Samayoa%20LF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AL.%2BFernando%2BSamayoa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rogelio Santiago </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Misión Biológica de Galicia, CSIC, Pontevedra, Spain, 36080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rogelio%2BSantiago%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santiago%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARogelio%2BSantiago%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ana Butrón </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Misión Biológica de Galicia, CSIC, Pontevedra, Spain, 36080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BButr%C3%B3n%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Butr%C3%B3n%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAna%2BButr%25C3%25B3n%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">James B. Holland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Science, North Carolina State University, Raleigh, North Carolina 27695</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>U.S. Department of Agriculture—Agricultural Research Service, Plant Science Research Unit, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BB.%2BHolland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holland%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BB.%2BHolland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:james_holland@ncsu.edu" class="" data-icon-position="" data-hide-link-title="0">james_holland@ncsu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/11/2095" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/11/2095&amp;title=A%20Genome-Wide%20Association%20Study%20Reveals%20Genes%20Associated%20with%20Fusarium%20Ear%20Rot%20Resistance%20in%20a%20Maize%20Core%20Diversity%20Panel" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/11/2095&amp;title=A%20Genome-Wide%20Association%20Study%20Reveals%20Genes%20Associated%20with%20Fusarium%20Ear%20Rot%20Resistance%20in%20a%20Maize%20Core%20Diversity%20Panel" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/11/2095&amp;title=A%20Genome-Wide%20Association%20Study%20Reveals%20Genes%20Associated%20with%20Fusarium%20Ear%20Rot%20Resistance%20in%20a%20Maize%20Core%20Diversity%20Panel" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/11/2095&amp;text=A%20Genome-Wide%20Association%20Study%20Reveals%20Genes%20Associated%20with%20Fusarium%20Ear%20Rot%20Resistance%20in%20a%20Maize%20Core%20Diversity%20Panel" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/11/2095&amp;title=A%20Genome-Wide%20Association%20Study%20Reveals%20Genes%20Associated%20with%20Fusarium%20Ear%20Rot%20Resistance%20in%20a%20Maize%20Core%20Diversity%20Panel" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/11/2095&amp;t=A%20Genome-Wide%20Association%20Study%20Reveals%20Genes%20Associated%20with%20Fusarium%20Ear%20Rot%20Resistance%20in%20a%20Maize%20Core%20Diversity%20Panel" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/11/2095&amp;title=A%20Genome-Wide%20Association%20Study%20Reveals%20Genes%20Associated%20with%20Fusarium%20Ear%20Rot%20Resistance%20in%20a%20Maize%20Core%20Diversity%20Panel" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/11/2095&amp;title=A%20Genome-Wide%20Association%20Study%20Reveals%20Genes%20Associated%20with%20Fusarium%20Ear%20Rot%20Resistance%20in%20a%20Maize%20Core%20Diversity%20Panel" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459574/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459574" id="node459574--3" data-pisa="ggg;3/11/2095" data-pisa-master="ggg;g3.113.007328" data-apath="/ggg/3/11/2095.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Genome-Wide Association Study Reveals Genes Associated with Fusarium Ear Rot Resistance in a Maize Core Diversity Panel</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Charles T. Zila</span>, <span class="highwire-citation-author" data-delta="1">L. Fernando Samayoa</span>, <span class="highwire-citation-author" data-delta="2">Rogelio Santiago</span>, <span class="highwire-citation-author" data-delta="3">Ana Butrón</span> and <span class="highwire-citation-author" data-delta="4">James B. Holland</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2095-2104;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007328 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459574--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Charles T. Zila </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Science, North Carolina State University, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Charles%2BT.%2BZila%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zila%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ACharles%2BT.%2BZila%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">L. Fernando Samayoa </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Misión Biológica de Galicia, CSIC, Pontevedra, Spain, 36080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=L.%2BFernando%2BSamayoa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Samayoa%20LF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AL.%2BFernando%2BSamayoa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Rogelio Santiago </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Misión Biológica de Galicia, CSIC, Pontevedra, Spain, 36080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rogelio%2BSantiago%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santiago%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARogelio%2BSantiago%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Ana Butrón </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Misión Biológica de Galicia, CSIC, Pontevedra, Spain, 36080</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BButr%C3%B3n%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Butr%C3%B3n%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAna%2BButr%25C3%25B3n%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">James B. Holland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Crop Science, North Carolina State University, Raleigh, North Carolina 27695</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>U.S. Department of Agriculture—Agricultural Research Service, Plant Science Research Unit, Raleigh, North Carolina 27695</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BB.%2BHolland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holland%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJames%2BB.%2BHolland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:james_holland@ncsu.edu" class="" data-icon-position="" data-hide-link-title="0">james_holland@ncsu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2095&amp;rft.epage=2104&amp;rft.atitle=A%20Genome-Wide%20Association%20Study%20Reveals%20Genes%20Associated%20with%20Fusarium%20Ear%20Rot%20Resistance%20in%20a%20Maize%20Core%20Diversity%20Panel&amp;rft.volume=3&amp;rft.issue=11&amp;rft.date=2013-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zila%2C+Charles+T.&amp;rft.au=Samayoa%2C+L.+Fernando&amp;rft.au=Santiago%2C+Rogelio&amp;rft.au=Butr%C3%B3n%2C+Ana&amp;rft.au=Holland%2C+James+B."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459574/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459574/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459574/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459574/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459574/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459574/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459574/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459574/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459574/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459574/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459574/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459574/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/11/2095" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/11/2095.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/11/2095.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/11/2095.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/11/2095.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
