<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/11/2059" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/11/2059.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/11/2059.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/11/2059.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/11/2059/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/11/2059.atom" />
<meta name="HW.pisa" content="ggg;3/11/2059" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Molecular Phylogeography of a Human Autosomal Skin Color Locus Under Natural Selection" />
<meta name="DC.Identifier" content="10.1534/g3.113.007484" />
<meta name="DC.Date" content="2013-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Canfield et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Divergent natural selection caused by differences in solar exposure has resulted in distinctive variations in skin color between human populations. The derived light skin color allele of the SLC24A5 gene, A111T , predominates in populations of Western Eurasian ancestry. To gain insight into when and where this mutation arose, we defined common haplotypes in the genomic region around SLC24A5 across diverse human populations and deduced phylogenetic relationships between them. Virtually all chromosomes carrying the A111T allele share a single 78-kb haplotype that we call C11, indicating that all instances of this mutation in human populations share a common origin. The C11 haplotype was most likely created by a crossover between two haplotypes, followed by the A111T mutation. The two parental precursor haplotypes are found from East Asia to the Americas but are nearly absent in Africa. The distributions of C11 and its parental haplotypes make it most likely that these two last steps occurred between the Middle East and the Indian subcontinent, with the A111T mutation occurring after the split between the ancestors of Europeans and East Asians." />
<meta name="DC.Contributor" content="Victor A. Canfield" />
<meta name="DC.Contributor" content="Arthur Berg" />
<meta name="DC.Contributor" content="Steven Peckins" />
<meta name="DC.Contributor" content="Steven M. Wentzel" />
<meta name="DC.Contributor" content="Khai Chung Ang" />
<meta name="DC.Contributor" content="Stephen Oppenheimer" />
<meta name="DC.Contributor" content="Keith C. Cheng" />
<meta name="article:published_time" content="2013-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Molecular Phylogeography of a Human Autosomal Skin Color Locus Under Natural Selection" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Divergent natural selection caused by differences in solar exposure has resulted in distinctive variations in skin color between human populations. The derived light skin color allele of the &lt;i&gt;SLC24A5&lt;/i&gt; gene, &lt;i&gt;A111T&lt;/i&gt;, predominates in populations of Western Eurasian ancestry. To gain insight into when and where this mutation arose, we defined common haplotypes in the genomic region around &lt;i&gt;SLC24A5&lt;/i&gt; across diverse human populations and deduced phylogenetic relationships between them. Virtually all chromosomes carrying the &lt;i&gt;A111T&lt;/i&gt; allele share a single 78-kb haplotype that we call C11, indicating that all instances of this mutation in human populations share a common origin. The C11 haplotype was most likely created by a crossover between two haplotypes, followed by the &lt;i&gt;A111T&lt;/i&gt; mutation. The two parental precursor haplotypes are found from East Asia to the Americas but are nearly absent in Africa. The distributions of C11 and its parental haplotypes make it most likely that these two last steps occurred between the Middle East and the Indian subcontinent, with the &lt;i&gt;A111T&lt;/i&gt; mutation occurring after the split between the ancestors of Europeans and East Asians.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/11/01" />
<meta name="citation_mjid" content="ggg;3/11/2059" />
<meta name="citation_id" content="3/11/2059" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/11/2059" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/11/2059.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/11/2059.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/11/2059.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.007484" />
<meta name="citation_pmid" content="24048645" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2059" />
<meta name="citation_lastpage" content="2067" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Victor A. Canfield" />
<meta name="citation_author_institution" content="Department of Pharmacology, Penn State College of Medicine, Hershey, Pennsylvania 17033" />
<meta name="citation_author_email" content="vcanfield@gmail.com" />
<meta name="citation_author_email" content="kcheng76@gmail.com" />
<meta name="citation_author" content="Arthur Berg" />
<meta name="citation_author_institution" content="Penn State Cancer Institute, Penn State College of Medicine, Hershey, Pennsylvania 17033" />
<meta name="citation_author_institution" content="Department of Public Health Sciences, Penn State College of Medicine, Hershey, Pennsylvania 17033" />
<meta name="citation_author" content="Steven Peckins" />
<meta name="citation_author_institution" content="Jake Gittlen Cancer Research Foundation, Penn State College of Medicine, Hershey, Pennsylvania 17033" />
<meta name="citation_author_institution" content="Division of Experimental Pathology, Penn State College of Medicine, Hershey, Pennsylvania 17033" />
<meta name="citation_author" content="Steven M. Wentzel" />
<meta name="citation_author_institution" content="Jake Gittlen Cancer Research Foundation, Penn State College of Medicine, Hershey, Pennsylvania 17033" />
<meta name="citation_author_institution" content="Division of Experimental Pathology, Penn State College of Medicine, Hershey, Pennsylvania 17033" />
<meta name="citation_author" content="Khai Chung Ang" />
<meta name="citation_author_institution" content="Jake Gittlen Cancer Research Foundation, Penn State College of Medicine, Hershey, Pennsylvania 17033" />
<meta name="citation_author_institution" content="Division of Experimental Pathology, Penn State College of Medicine, Hershey, Pennsylvania 17033" />
<meta name="citation_author" content="Stephen Oppenheimer" />
<meta name="citation_author_institution" content="School of Anthropology and Museum Ethnography, Oxford University, Oxford, OX2 6PE, UK" />
<meta name="citation_author" content="Keith C. Cheng" />
<meta name="citation_author_institution" content="Department of Pharmacology, Penn State College of Medicine, Hershey, Pennsylvania 17033" />
<meta name="citation_author_institution" content="Penn State Cancer Institute, Penn State College of Medicine, Hershey, Pennsylvania 17033" />
<meta name="citation_author_institution" content="Division of Experimental Pathology, Penn State College of Medicine, Hershey, Pennsylvania 17033" />
<meta name="citation_author_email" content="vcanfield@gmail.com" />
<meta name="citation_author_email" content="kcheng76@gmail.com" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=An integrated map of genetic variation from 1,092 human genomes.;citation_pages=56-65;citation_volume=491;citation_year=2012;citation_issue=7422;citation_pmid=23128226;citation_doi=10.1038/nature11632" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. M. Altshuler;citation_author=R. A. Gibbs;citation_author=L. Peltonen;citation_author=E. Dermitzakis;citation_author=S. F. Schaffner;citation_title=Integrating common and rare genetic variation in diverse human populations.;citation_pages=52-58;citation_volume=467;citation_year=2010;citation_issue=7311;citation_pmid=20811451;citation_doi=10.1038/nature09298" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. M. Behar;citation_author=B. Yunusbayev;citation_author=M. Metspalu;citation_author=E. Metspalu;citation_author=S. Rosset;citation_title=The genome-wide structure of the Jewish people.;citation_pages=238-242;citation_volume=466;citation_year=2010;citation_issue=7303;citation_pmid=20531471;citation_doi=10.1038/nature09103" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Beleza;citation_author=A. M. dos Santos;citation_author=B. McEvoy;citation_author=I. Alves;citation_author=C. Martinho;citation_title=The timing of pigmentation lightening in Europeans.;citation_volume=30;citation_year=2013;citation_pmid=22923467;citation_doi=10.1093/molbev/mss207" />
<meta name="citation_reference" content="citation_journal_title=Pac. Symp. Biocomput.;citation_title=ALFRED: a Web-accessible allele frequency database.;citation_author=K. H. Cheung;citation_author=P. L. Miller;citation_author=J. R. Kidd;citation_author=K. K. Kidd;citation_author=M. V. Osier;citation_pages=639-650;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. R. Grossman;citation_author=I. Shylakhter;citation_author=E. K. Karlsson;citation_author=E. H. Byrne;citation_author=S. Morales;citation_title=A composite of multiple signals distinguishes causal variants in regions of positive selection.;citation_volume=327;citation_year=2010;citation_pmid=20056855;citation_doi=10.1126/science.1183863" />
<meta name="citation_reference" content="citation_journal_title=Journal of genetics;citation_journal_abbrev=J Genet;citation_title=Genetic landscape of the people of India: a canvas for disease gene exploration.;citation_pages=3-20;citation_volume=87;citation_year=2008;citation_issue=1;citation_pmid=18560169;citation_doi=10.1007/s12041-008-0002-x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. A. Frazer;citation_author=D. G. Ballinger;citation_author=D. R. Cox;citation_author=D. Hinds;citation_title=A second generation human haplotype map of over 3.1 million SNPs.;citation_pages=851-861;citation_volume=449;citation_year=2007;citation_issue=7164;citation_pmid=17943122;citation_doi=10.1038/nature06258" />
<meta name="citation_reference" content="citation_journal_title=Journal of Human Evolution;citation_journal_abbrev=Journal of Human Evolution;citation_author=N. G. Jablonski;citation_author=G. Chaplin;citation_title=The evolution of human skin coloration.;citation_pages=57-106;citation_volume=39;citation_year=2000;citation_issue=1;citation_pmid=10896812;citation_doi=10.1006/jhev.2000.0403" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=A. Keller;citation_author=A. Graefen;citation_author=M. Ball;citation_author=M. Matzas;citation_author=V. Boisguerin;citation_title=New insights into the Tyrolean Iceman's origin and phenotype as inferred by whole-genome sequencing.;citation_pages=698-698;citation_volume=3;citation_year=2012;citation_pmid=22426219;citation_doi=10.1038/ncomms1701" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Kumar;citation_author=A. Filipski;citation_author=V. Swarna;citation_author=A. Walker;citation_author=S. B. Hedges;citation_title=Placing confidence limits on the molecular age of the human–chimpanzee divergence.;citation_volume=102;citation_year=2005;citation_pmid=16365310;citation_doi=10.1073/pnas.0509585102" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. L. Lamason;citation_author=M. A. Mohideen;citation_author=J. R. Mest;citation_author=A. C. Wong;citation_author=H. L. Norton;citation_title=SLC24A5, a Putative Cation Exchanger, Affects Pigmentation in Zebrafish and Humans;citation_pages=1782-1786;citation_volume=310;citation_year=2005;citation_issue=5755;citation_pmid=16357253;citation_doi=10.1126/science.1116238" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Z. Li;citation_author=D. M. Absher;citation_author=H. Tang;citation_author=A. M. Southwick;citation_author=A. M. Casto;citation_title=Worldwide Human Relationships Inferred from Genome-Wide Patterns of Variation;citation_pages=1100-1104;citation_volume=319;citation_year=2008;citation_issue=5866;citation_pmid=18292342;citation_doi=10.1126/science.1153717" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. F. Loomis;citation_title=Skin-Pigment Regulation of Vitamin-D Biosynthesis in Man: Variation in solar ultraviolet at different latitudes may have caused racial differentiation in man;citation_pages=501-506;citation_volume=157;citation_year=1967;citation_issue=3788;citation_pmid=6028915;citation_doi=10.1126/science.157.3788.501" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=V. Macaulay;citation_author=C. Hill;citation_author=A. Achilli;citation_author=C. Rengo;citation_author=D. Clarke;citation_title=Single, Rapid Coastal Settlement of Asia Revealed by Analysis of Complete Mitochondrial Genomes;citation_pages=1034-1036;citation_volume=308;citation_year=2005;citation_issue=5724;citation_pmid=15890885;citation_doi=10.1126/science.1109792;citation_georef=2005041733" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. D. Meiklejohn;citation_author=Y. Kim;citation_author=D. L. Hartl;citation_author=J. Parsch;citation_title=Identification of a Locus Under Complex Positive Selection in Drosophila simulans by Haplotype Mapping and Composite-Likelihood Estimation;citation_pages=265-279;citation_volume=168;citation_year=2004;citation_issue=1;citation_pmid=15454542;citation_doi=10.1534/genetics.103.025494" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. Mellars;citation_title=A new radiocarbon revolution and the dispersal of modern humans in Eurasia.;citation_pages=931-935;citation_volume=439;citation_year=2006;citation_issue=7079;citation_pmid=16495989;citation_doi=10.1038/nature04521" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=H. L. Norton;citation_author=R. A. Kittles;citation_author=E. Parra;citation_author=P. McKeigue;citation_author=X. Mao;citation_title=Genetic evidence for the convergent evolution of light skin in Europeans and East Asians.;citation_volume=24;citation_year=2007;citation_pmid=17182896;citation_doi=10.1093/molbev/msl203" />
<meta name="citation_reference" content="citation_title=Out of Eden: The Peopling of the World;citation_author=S. Oppenheimer;citation_year=2003" />
<meta name="citation_reference" content="citation_title=Human Mitochondrial DNA and the Evolution of Homo sapiens;citation_title=A model for the dispersal of modern humans out of Africa;citation_author=M. Richards;citation_author=H.-J. Bandelt;citation_author=T. Kivisild;citation_author=S. Oppenheimer;citation_pages=223-263;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Rozas;citation_author=M. Gullaud;citation_author=G. Blandin;citation_author=M. Aguadé;citation_title=DNA Variation at the rp49 Gene Region of Drosophila simulans: Evolutionary Inferences From an Unusual Haplotype Structure;citation_pages=1147-1155;citation_volume=158;citation_year=2001;citation_issue=3;citation_pmid=11454763" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=P. C. Sabeti;citation_author=P. Varilly;citation_author=B. Fry;citation_author=J. Lohmueller;citation_author=E. Hostetter;citation_title=Genome-wide detection and characterization of positive selection in human populations.;citation_pages=913-918;citation_volume=449;citation_year=2007;citation_issue=7164;citation_pmid=17943131;citation_doi=10.1038/nature06250" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=A. Scally;citation_author=R. Durbin;citation_title=Revising the human mutation rate: implications for understanding human evolution.;citation_pages=745-753;citation_volume=13;citation_year=2012;citation_issue=10;citation_pmid=22965354;citation_doi=10.1038/nrg3295" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=P. Soares;citation_author=A. Achilli;citation_author=O. Semino;citation_author=W. Davies;citation_author=V. Macaulay;citation_title=The archaeogenetics of Europe.;citation_pages=R174-R183;citation_volume=20;citation_year=2010;citation_issue=4;citation_pmid=20178764;citation_doi=10.1016/j.cub.2009.11.054" />
<meta name="citation_reference" content="citation_journal_title=International journal of legal medicine;citation_journal_abbrev=Int J Legal Med;citation_author=M. Soejima;citation_author=Y. Koda;citation_title=Population differences of two coding SNPs in pigmentation-related genes SLC24A5 and SLC45A2.;citation_pages=36-39;citation_volume=121;citation_year=2007;citation_issue=1;citation_pmid=16847698;citation_doi=10.1007/s00414-006-0112-z" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=A comparison of bayesian methods for haplotype reconstruction from population genotype data.;citation_pages=1162-1169;citation_volume=73;citation_year=2003;citation_issue=5;citation_pmid=14574645;citation_doi=10.1086/379378" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=M. Stephens;citation_author=N. J. Smith;citation_author=P. Donnelly;citation_title=A new statistical method for haplotype reconstruction from population data.;citation_pages=978-989;citation_volume=68;citation_year=2001;citation_issue=4;citation_pmid=11254454;citation_doi=10.1086/319501" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=R. P. Stokowski;citation_author=P. V. K. Pant;citation_author=T. Dadd;citation_author=A. Fereday;citation_author=D. A. Hinds;citation_title=A genomewide association study of skin pigmentation in a South Asian population.;citation_pages=1119-1132;citation_volume=81;citation_year=2007;citation_issue=6;citation_pmid=17999355;citation_doi=10.1086/522235" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Molecular Phylogeography of a Human Autosomal Skin Color Locus Under Natural Selection" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="twitter:description" content="Divergent natural selection caused by differences in solar exposure has resulted in distinctive variations in skin color between human populations. The derived light skin color allele of the SLC24A5 gene, A111T , predominates in populations of Western Eurasian ancestry. To gain insight into when and where this mutation arose, we defined common haplotypes in the genomic region around SLC24A5 across diverse human populations and deduced phylogenetic relationships between them. Virtually all chromosomes carrying the A111T allele share a single 78-kb haplotype that we call C11, indicating that all instances of this mutation in human populations share a common origin. The C11 haplotype was most likely created by a crossover between two haplotypes, followed by the A111T mutation. The two parental precursor haplotypes are found from East Asia to the Americas but are nearly absent in Africa. The distributions of C11 and its parental haplotypes make it most likely that these two last steps occurred between the Middle East and the Indian subcontinent, with the A111T mutation occurring after the split between the ancestors of Europeans and East Asians." />
<meta name="og-title" property="og:title" content="Molecular Phylogeography of a Human Autosomal Skin Color Locus Under Natural Selection" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/11/2059" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Divergent natural selection caused by differences in solar exposure has resulted in distinctive variations in skin color between human populations. The derived light skin color allele of the SLC24A5 gene, A111T , predominates in populations of Western Eurasian ancestry. To gain insight into when and where this mutation arose, we defined common haplotypes in the genomic region around SLC24A5 across diverse human populations and deduced phylogenetic relationships between them. Virtually all chromosomes carrying the A111T allele share a single 78-kb haplotype that we call C11, indicating that all instances of this mutation in human populations share a common origin. The C11 haplotype was most likely created by a crossover between two haplotypes, followed by the A111T mutation. The two parental precursor haplotypes are found from East Asia to the Americas but are nearly absent in Africa. The distributions of C11 and its parental haplotypes make it most likely that these two last steps occurred between the Middle East and the Indian subcontinent, with the A111T mutation occurring after the split between the ancestors of Europeans and East Asians." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
    <title>Molecular Phylogeography of a Human Autosomal Skin Color Locus Under Natural Selection | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__34Arr8QetTzezDWW4Etyv7gbgH-x-ZTNo3yO15Q-gHU__Mk-lyyG5WoK4SRy56eUq5eomxN7OGXjpDv-Ob1a1Q1I__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"GIYC4MDQwiGFSfzlLWgvjpzNu9CA9t3mI8W3v212qbA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459577","apath":"\/ggg\/3\/11\/2059.atom","pisa":"ggg;3\/11\/2059","ac":{"\/ggg\/3\/11\/2059.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/11\/2059.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/11\/2059"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/11\/2059"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/11\/2059.short":true,"\/content\/3\/11\/2059":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ysbIxB8WNxxjuC5ZLSAPi0kpLzEoCHEa8bUpaMWUSFeeDi8alD3wZg+eVmwbrvi\\\\\\\/MsRtbpLBR0cAkUJE41pggb256czTQbJJ6yDXaKNQaLKFp6mQ1\\\\\\\/+6Mw+hBpnV2dSAMOMrZmofJAL5L7Y\\\\\\\/oq3AnQYAHu\\\\\\\/xfD40QaRL5duu3lry3mCc4TQU3yzrzLAl1PCt8i7fD2jVBV0ELNyGwz5SHnGc03wKJ3VCJ1tq5gIQJG1UJCnIuD3C3xdkRlKNGMT4ivGz0FOy8d0Lq7tLTeGtJFWuiRw4O14sgfSmNnxAolv1XquKY0p4MMsiIitnKKaeh7JuxCwJcvBFtx+bbwQIHaj3+NxNh2lgA0S4a6eXb1rS1uxv8Y3JZABMn50HhzmNpjuOujV05sPCcbeodpdJA3MQjEixocf8GxNwZVAv17nrDKFkDNw\\\\\\\/S6kyY+V7PjXUCBfd7n3ZMuJ4vqyy3L0aJ3bn0w8Ntliu8HCsabnVD5kU0Ysfm5OQkoRFs3z6caKH4NZ1Sn2rtdBT+trlf+7ORTwCCmN43DM1RUL67kmB0Mle2EhT8H3WWXh1ZyQ4NYZCu3\\\\\\\/pzCAE2ooNGsYju7GQLAhYjYwEvM4q2Wxe7BuWknRfb+3qC7VjHS8WPil1jXES2iT51rmu5l2DxddtwrW\\\\\\\/cKDgCiG0NTN6D+Ler3bQ6z\\\\\\\/sTIQzYdHzL1IRXsW7SIAT7rVeWd+9qTxqz25q6qOvI+lWC9J3cJ+wwMJXOZME6GPcFnAeoW\\\\\\\/A+ZctxcvAWrD0wgqfMucSzwb7Is2XcaznbB3M\\\\\\\/CazCL53ddKl24lbHP3UdWKpg5GpylSaflxQ0peiDkCsUePVGOQYRSCVT0R2t1DuSZNQc6iOOUt378VfeMphsfQz3xIIzIhiLIjibf1DdAD9sF+IHG+i81BmsQp4wu4kGQYlQAnfr87aNA+UcOiVBIpXHLr6lYRpuxqaJRLpUG3CK9xMp9Rd9BaJOWrS8rO95WAz+USvDmfumOyhn6nOFA9hkpAZ1OlRijKPMsTTWjYWa+2Jz\\\\\\\/1aTkSVw8MUvkfoqq+Gt0cFA3y7yRoGzVkIH9z\\\\\\\/uP8koXjv68GFjvlDhFQMf1kpbEoNcPHA7F4uyYWE0YJR73rukgQazAMGlyMbfsMI53nK3jtboJ\\\\\\\/maDb0G5ZhmeHhp3+P5874v4tjhyiwa3mAeoW0JWVvStdTVi0aa8n0+HINnX25YVHEXlw39TgjszJVKrnjgsNkQxRXX4eIlaoVhzNCd9xbPyVTBZiYA8N97PwBsiXEYiTLzh84\\\u0022,\\\u0022iv\\\u0022:\\\u0022nLwKRzNh2K6c8CROxR0E3w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b0ff932f77deb4fdd1ceb06a3a24c77fc33831d3475d1c0945689d963fb56d1e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+lYc+gZ0\\\\\\\/AMl2kGJaRGYrK95iu6fyvtaTMWjKDJfjhQcoXICkdMyDIrceo4psHKUVs3y4PZbL2vrnyfkyBfg5wH0JPteCqoNHDrvmhMLmxE4TPXEYp7uNBECaF6+NdA5NQYTIZ63ZzVpwVeAuNHC4vfuIdbod6Lcj8LnZPD1EBhF5gV+IMYJ1H2jXuQUhQYwatQbRvPBNqGuQop9tPFVlVKxk2bdH7PiTwjSrzXqnOGb5NwwWIqkvSDiF8kdbVPd6aj80ILQ1cr+u8Ve3cAGdfasDqB2AO5FM9bGFR4uMDtHj0SF2AQMW0C5DRSYWuvbhYXLeaALr+W\\\\\\\/svxlsXKDnffxef\\\\\\\/IzYMWSpZxaN+NjRxNx+FDDid4Q2EixkrMXsVqeUnehDf1Pc3OTjGGUR6WdG\\\\\\\/zMzD4zNXlFnrPX0ECvmBKuW42s3uK34aD2ifnIMTTU90BxIxN+VRzwdhemt1KdtEY7VB+wWVVxq0pmTyk4XzWCo2tfglTZOGHbey+Q8OUxYVnK6D497TnAgA4gXUYNPSSJj1npp4hRZKK5rv\\\\\\\/ccQWj\\\\\\\/WHD\\\\\\\/477SLtB9BBDkM3b\\\\\\\/qKkCJVQ96JuN6t05317\\\\\\\/FCvtacoOGhb56auWqC0FFztANBf1riXtonadKABuUN+Wut7VH6bKPcjLFNRCp7kXBOt0SLAFbJWRjsu9OJ5i2vLZsFqOUV\\\\\\\/vDEZtsGGTySVPlIkzBU2RW8udzW9ec05qH300aDVEGblGCKOefaMDXjXfgHf8d9SWpj9595YPOcJy7+L3D0L\\\\\\\/DQZN8B6z+CXRzGCYflad23Qti21Q8oMwLzITRmwtj01j+ZqAjLkqZXwdyYUBj\\\\\\\/LwSFVeO976Fq\\\\\\\/kKs91DtKtKSKKqbIKNowwEW9cvtdtRVEptdQF\\\\\\\/wK75zHlZJl7lkCjMxl5TOzR+G48VGWAPVPmn9CQfpomUMTEvDWaJg525YveOJuv9gmFihnvbb8l4FcQAnAGeqCYoGEeOgco3jq34r6r3D6RuuY0jZ8zWPdI44XyRTFc69T1RKWY8RwIBhsfE182ZA9QYtw91cOPNW7zXRlweycg9pzQWHR42v4+BHuYW6FRkHsg4mJGdIywLl36WfUe4h6e0FWjf6\\\\\\\/CEqlSkuJo+gc472wL0PxEOUu9YuuA4DeVfJnqGdpuntWPAHxvPuqXGm1ifLNajqymYx+0kVg45o3bd0be1xy+0KucaPNPXbYD2UqaK65ipXA95fn1NqyKtyvuSnEbELQLn0PERH3fOTRoOKWL8VnJeg5gpmXefLndtOrJBfaP\\\\\\\/CcOj6xbbXMnaqilzae4WmOFjm8opcWWqVh6kLxT8t5Wv7p5MaizxbY4QVgM0ySzTHgy7DeVPunjp0IZIFok9wembqZtHedX\\\\\\\/eDrJLidFVcR475RLnkogKQYLnATGr03Vkdj+sor33t1Z\\\\\\\/Sz\\\\\\\/QOuKB\\\\\\\/2lNlTYSX5\\\\\\\/GUkjoIq9\\\\\\\/XcEQ50srLj+h2+BX573aydKPK4Li6TM9Uh+cFf56YTRlLkD2K7wJp8Fe5HcK6lOipqJt\\\\\\\/Tm0tZbZw3PSv\\\\\\\/lyaI+Tq\\\\\\\/suJ1\\\\\\\/1DUgguY8pBHfXfnp3N8APdEGiRewcm66oOUDEa7yHXq\\\\\\\/SJg9nQxfLBuv41\\\\\\\/KmxYjX5JmAtku2XYfiUpNYE7b01YD+5l7F5ipm2nK\\\\\\\/1ewDqxvsaJdeyjZQa0Ciw5O7ttl7tVh3zGjMyLe1JvkhUBg76d01TYk35xZoE0ubQamG1DFjfoEsETLSbBEem3CreZXFyO4Zuwds0Lr1eYqfzhzkIXmF8RegZjVxnPvznDfbGgGlGR\\\\\\\/P2n6EEAP\\\\\\\/Mo7v6fwqACRa3LuqfZlvcg9b5RxUJ1DtvGWAVdGyGs8C+N+5SI6h92u5rXBpuipszdC7K\\\\\\\/PZCO9pNw4REm42Vxz6iAL3wWsrEQ8D0EX91J3UCldQsAuB3E\\\\\\\/Bg6Y5EtKXOYoirXwdd5B4lcAkXXaNOeKLhc7M1+TfgyrNsigug3vqotv5qGZbafTPcDH4K1aLWPGW9YV0mU7mgNBB\\\\\\\/NrbxXqN\\\\\\\/52cfwigDpPbi9WlZKi7mOsen7Hl2GoIb81IkYdPwyjV3Ed8ey8y7gj6rT3wgI4q13LlLdc3yhLim6\\\\\\\/Xds+4BiLEVFD7W47jj360hn5yuBuENjZ9wzaqrjf0VqSfutaqddijxLwGpgDSreSf2Qi21M6gkCNM9maY0XjWNf06I2bRJelzNKpfjR3Tiaw0X1jAiju\\\\\\\/xzBmkmrsqfDDcY\\\\\\\/fbEnd8y7GRe9LIgNsCuaW9745Mrv6nJTyotVHFUxcdq8JI4dSRu8816YM7JSLuHfrwY69zkyKFCx7TYqhiKQa\\\\\\\/lDKi4pODnchK0ZfFAQBdaI2ZoN9aLa13Bz8kti1z0BCXHicOPuChxIy6WM3FLEn0ogIOfxcgzS1V12c1TMuW6h4vzb9AtLmoFRB0T\\\\\\\/dkUOCchUED6L2\\\\\\\/YWD6TLHULBNQ8ZjbvKDM3S8ER3RwtPYym\\\\\\\/9tfhIZ2bgqvaEtDnbCqVzExjGCL9dML162oOei73L4REsIsZ0S+qabVleTdSlVj6u6XTKBxFTSa7x5wrU0kA6UFSx\\\\\\\/QsE90flwwnQVbnn0juEZPLdT0OhW8P7cWG5+bGHOg7m+8sdB+uy2ktmpTz+mzyowwQPMwLNDXi46ETRuKB3cJCclE743e74SN8A07VarRyspZ41KBRLE4mq7nosboYJVfAZL8kmghxWPmZLcqaO1eqMUMM9YVQrkfhu3QWgHAanH\\\\\\\/Pwpof8sL9AF51gxe2d99T6FfKA9bE4PDCTeqtRVGMq9xIAXNDs\\\\\\\/5FrLWMwq005Sq9\\\\\\\/Aeo1fX\\\\\\\/hkxFXO40u+yoPTipOq0UVaJGqb\\\\\\\/mXaggmts5R1ZfOB6K3SDyo1mDCrIIsgo9ZwIjwTZBvyXOTi6cXQQQAIp2XfZNLHR2\\\\\\\/mwLnM8oAV\\\\\\\/A0LQoVlMDRJ+nKHbdvbli98JJp3OqgkcQiv3NwV7zWico\\\\\\\/0WHpD\\\\\\\/SaQ1EixC+8E8hkVibQpVEDg1r\\\\\\\/JlL1zqUsEWnFl9Iwuday1+zCbx8COJrLklHilE+pfYiz8B0Wb\\\\\\\/jZqfqigxFgXjXDjnpGqDraMM95V+CPgfiETYTFk1NLTZ6lVrPdfiMrdv6BiUsd8r562LFOIwHhMqXFonzVXvTmOQlbqXy1TYOR0h8ZXtp4EfOwJYTyLMJZ7KD2vJDJg8k\\\\\\\/HPyOwUUhjzMGN2xWCi17WTqTXzLj0apot106uioope\\\\\\\/5jGGx7lpyiQLvcCQC+VMsQefKcpD0WeV6m2Kq7SPU8YboiBIrb5zsg6c5RdXKjLHVCGlc3Muhp5HET+aWtgYbJqiJwIwGeSiM+eaBbIpSRErNw+z7r3GoBDOuTrrg3Yi9JW\\\\\\\/R\\\\\\\/JxpqOZw3Ot1Qu5VeB6m\\\\\\\/acOgoyTJSQUQ\\\\\\\/+9sByY9ftuH1SYKE53LPeEyCpcws\\\\\\\/EiZSgLZ17EXy6+ZlTOC7fihpFewiPYzKvW9mwP8vbg97w4uSJkRP4IPZiYICvZT5xD3sZ+GckLOBD+TiWfg6Ejb1MDDo1QkLPYrBw+3MFz11oh8ho1waoAUUJ5\\\\\\\/I5O+0oMrJ0gIJhfsl+WlpQETo5X\\\\\\\/Piicgkh3M5hsihgaKDzxCk3wnbwdWHCqwIHCnBPPb0OadLYvkgEuHVfGFCxiLZVJB+s20QYf4\\\\\\\/Z00qm9poA9kGHw3ebgg574ez38l9Jf8Pg8WvPTg3AL15s1+p7oR30c+OS1ke1KSMS\\\\\\\/\\\\\\\/l1lxwhcTRDRz20T7CMxVpvr5iZYpR7UpH0TLc3rwPD7RtO40mZfOpFyFtj4Mymx7+mAkcDtJZM8B\\\\\\\/Ef8Egag7f5ZXeth6e9qynSnmuHP+iUsKi3QsPqmAi2znnzg4iElrakh+JltNz0XPMaoQPxpEzIVD88naYVOlXPZHHei7Yp1DbU\\\\\\\/BNuGbr7uXf5DCMn1yWHhwm+zyhqRgAIxlelP8LcjfPz\\\\\\\/42n3I8hEIUKfJRWgrEbeiNNLYej1Mwm3IoBehRTBrcZdo0rLyqxPBiXGJMm0LGByQTP7HlsrjNiR6Em13QrlSJpu9t3TtGn8dXTqg48nRkgUC3TvCumI2zJThD2\\\\\\\/fuSkXAXK4AeIyFL2VFRixWqFtBpuoN7Kr3gJ9t5epqbSjwVmRMkfcmKj2E5i3Cu2YgWGvE8zNBO0wqyBMykC2rPObxpr0XATckdP3NsCIJier1ur05xehJb2VI6A9Z+cKPAxP8VQi1TKqvR7+t\\\\\\\/PUr35+U2j1snjiBYoJNOKkD7PEo6whf5tENg\\\\\\\/LNuvQWDQ3a3lURYps+6D3wObqL4TU\\\\\\\/XR7zdrYVnxpo6vyilE0rqv7VoBs2K3geqfq6y8mVTZ+SDCe\\\\\\\/CRiX\\\\\\\/nEM6zveAONNoikY7BfTlPrh8JFoojFJAVDLwqesSDHx0MQmryDO9FwDvGyqKzmrQR8bdVUQyoQggYOnyNp6NA3vGt+ufw4U3xsGm\\\\\\\/ZmMxpCfX7j80yEFZsiGiIqyVskG6Ol\\\\\\\/1d1xTch\\\\\\\/WscJalFW6p7L7MIYycm3\\\\\\\/dr5xhHTPuuA76rV6bgZt1ihDirSTFyEI9LExy9\\\\\\\/XrV5DNDjCl08B2ply+k\\\\\\\/eKO8j9jjE9h8uix7zu52oGcZbiG4hvNcwGXPguJVY5q5gWjwXPIwGrKBkth1wGOhTXsrHShSlyERSo0PwjO2h8N3jEfAlxMqzN6WCxe2K1KXXmv+yTanQhUQ97DkSKGZpwajAPsCHqdX4wI62n+M+nVS29xH3jtDFbHlMCLPg8IrdNDEy37qDa8O1hEQcxUsem5Tb6FWw8pSyUmBo2uTCOU+p3cKARJv9BGItIXOcu\\\\\\\/CiMoE+LHaWEU7Kw1wvG8Hf0OHS+c79gzi459LBE6v4tO5Vrm0MW\\\\\\\/JrAy9Fdd4C8rZ+JYn5p7ab+PVQ8OWeHGQbL6AvZK2Vwbg6JKvedfc2dTUHLvuV+9GZ\\\\\\\/7soQ88w5uLE1WaqG1S8CMC94rKxtSvfB8ewCbjaLrEIX7esWa59o9ocX6cVdEpK28LY+PfAtBOSMwZFXwRMU\\\\\\\/IV2quw+HiC4Njgkp8MbDHGzhU9m9oxsqcpUCIVxarc\\\\\\\/cKomN7ooPOC+GcDFh69KozFqvJzrxryQLDGW45R\\\\\\\/tLKlkPwvppFRkpwl1UO+ZWdyyiAyGsCdIrFkXAN84ULq4PisIhAdG4fDwWQK5Ig\\\\\\\/tCv2s+j5iNj5mCTUwahouVt4z5IQO+SDeGMET9EonrjK7icmBZgUQFEAtBIqMcLqqE0frua+bvCJT5NpCCEllVs\\\\\\\/0K8JdSYm6EXzVNtDJZEJD6bmNxXPBIJLZNp0yozwTTPuBYa4EJdPPQqg7ObJmEDj94YxFm3GQzVX1l3V8Y9WhbtpkZeD\\\\\\\/4qmmOxV40smRskLDIMxUYU1psSNB+Q3rioY0IlTh81HEXT+UEUyaM9MNGvZNjE+a\\\\\\\/NFTwDY8UQcgXo24yc5P1uxiOkDyQXaZ5QYqz0m\\\\\\\/lFytaod4\\\\\\\/iBG1mYacwoH3Bw7N+jDbc3ieh4K4VGnZTb9\\\\\\\/vkBSK\\\\\\\/AuFVxT8\\\\\\\/fmYzlskdwi+kUP0iLkmdA7oIqGkHBSQnMRcBaXDJrxuaSAo+X+sWBI9iMUcxURLNwpnmDAgvUsG5vWrBm4c+zmmxQ1a8wDcFaEm3M7HvC6p27T1fxP17rxak+QrD5DDggfZYdAV\\\\\\\/Kxx5L8UId4km40oQEeRpm9ln+Jcdqjm1\\\\\\\/A8w1FgmIut1p9RYiwiFEauXXxFDgOGW+8jSpufxYuVg0FYHJ9SIm5QGdXCiU2keUZuDuUJrP7+71GunmxN2wNZ0+S66wlx0eJqj1ZBu\\\\\\\/WoFtH9zD\\\\\\\/kKWekPdmBK7HV54NM91APirVEmwoaqTCZCKnfuVtHslKDwVg61SYmDevi\\\\\\\/\\\\\\\/1KDtfRCyay8s2zpPKahOKt8VS\\\\\\\/sNdw\\\\\\\/gT\\\\\\\/WqW\\\\\\\/JI\\\\\\\/fo1W7ee6dlitRhSjLJj5\\\\\\\/CWcuAsHVUCwik1cGBxCCUoy0zwnibaUZd1MAIrQskT6mxOnKbKWRrU65i93jPfkcYUrgCO56iyPpziz0dmNEUDQRz5So7IBURofE22TtSnf3Q3o9zQ1Xhva66VTWBrMTsuCChx8bXy5PecSEmp9YS1QpAKsE0KuTDRsHUsmB\\\\\\\/ec0tgEr3gwfv9UQVWJljQgGCBNyx8nK71U8wPDbwlg2cMJS7UJWeMPKyeQ\\\\\\\/WahO9v3962fvjnNpgqxKV5JYv4sL\\\\\\\/16XgIJvk0B3jGTP2\\\\\\\/fOSDhlBuw9GxzDfbu9\\\\\\\/ZUVO24tAdjLEsJiuglmxIdzzl3YSC3fkibH5tEDCwNx4cRuOnYE7AgJSHIoG33keRDtw\\\\\\\/Clsluja1T19iiqV3sCxHf5qZWYK2OfvK3pwmxosY0KnbU0tmB2tkLZ7kYkl8ybN7b7V7twmcyzoONO1KHqKlEq9hukmxnuQb0QHg3dKd4Y\\\\\\\/GwDeSJfwLtvj8TNBennxL0S2EsRYIhe5z\\\\\\\/QO4cflKiN6cxlZvWOx03EBQztdShXHqtBms4XHAj4dAmWlqHeJ07eOeRaJuF9eFJWauWF6S0UDVKINbWLCrHYAuD8MX9tloUCa0mOer0qD3KCc3bHVXxswxgm\\\\\\\/iKf13D3S31ZpDnpj60SAjAIS89ZdlDctatkkgAYdd1PDliN4p0k8QQRLeBeLjeuE67dE1EMiF\\\\\\\/HaY5AWqvKRsK6H92HoDQ+GkB1r0MwBwv\\\\\\\/bn8ySIfcJCtbYbABeytzRhN70ctX\\\\\\\/BRKuj8wLkWfNRDQZ9988R5Y1lje0bq+U7IONT+gNkpSmivsChf33UyFFu3v0ULUL8YEltMk9uqKki7EFu5oyXiBCYvdW64fGgfqZMn8DNjLVu3h2SxDPYsAGgQwIpOVj8GRhdNXSQfIaBn6zNHeTAOqJsYAbGC8MyaPgxMF5KxR4Q+oFXqkDUxC7cd4+silzyXbU5IBPio\\\\\\\/rIiyuAFj95PnW8Qy\\\\\\\/JrgZ3\\\\\\\/6llShiywVBuiGTVqTvCZza9sye4EBSfD\\\\\\\/QvdyvT\\\\\\\/ojmCD7W\\\\\\\/f5lA\\\\\\\/G9shZQT5+RMRc0T40D\\\\\\\/pAsiIAbNQ6pOHvfzTz8DZt9+a3yc6uh8spHLYSt6kTleelQazD5wH7zu\\\\\\\/o7zJUKrei5dfHWO4FU1nctBzACayXP79oFrEMaDo6Mb9+2Qi\\\\\\\/apPUyO01Khb4Ngt8Mqq\\\\\\\/ch0rhyYd482WMgPxqlQo2ilQO7XaEkDgA3cC+tNNTKo5cZPnGy5eOWfuOfq0n3VK+XnzwmrP5dsj5s\\\\\\\/l+xcnJly0kMb24zqJhOHegc\\\\\\\/vCAVsAlY14IKdcZRs0Pcgk4R\\\\\\\/0\\\\\\\/\\\\\\\/YngF3Ik53yTWmhnwzOtqX3ePQWXEk3xgf9AlNorMplz+iXAK4LvM4oecz9jB4f9ZPnslE3OLz3rU\\\\\\\/d7d9nmKBLaTtjS+NF8Lr8+jt7nRkuwaU\\\\\\\/vFSy+77SdU5mkw2S6w\\\\\\\/+CnPLrF+QuTh1wYCg53xE2Xk76u8tRR5E3lpuyCcHtKSw0RJShujEdb8rnm2L32CY+Pfu29lxchcrcyoiwTVstCN3IVBAgc5jgqEj0JKVdfXFHFvzS\\\\\\\/fdAfZDsxq1rHGxMYk+e64SbeRRe+DFitdAAoDEGAagRzSlXZgbDRm+I7vR3WFNCOVAgzxx0xFfiFLsmB0ONvZ0MFbaUYuMXfLxqN1FEHktPbrAqekT80D8qAhPwvFdGGOlR0cWoUut\\\\\\\/fXbtdoYvKa8jX0VfqXvOi8csIZ3lCXiSxugqxRHVspGPYafFN+jjReaPY683DVy0xeU09YbN0Sej6+DNH4xs\\\\\\\/Z+Z9joww4iqNvwiJgTp0UWai0a4vxeqwhtnCAGjOgk6XPwi0NcF63U5AwDsmXligrRt8HD9xV8Hdc55wbioWz6eFs6bR16+y8UjfXk8+4UqJiasyNgjsEjdATkzlYc76Gv2eTIkYeHwonM4Nt4i3bGND7LlUzFfMV1mD1bDHKFC6odEQt6jwxroE2fLHXRQEpABT7pbNnBdYy\\\\\\\/TXYsoXf7lSWqv9aoHirrXC0VNsJpAbsbTLVywORZ\\\\\\\/0iMJmbpTMLHchW4pjMgFnh1mX5QhyfsWKHvIXzdMcd6TAzaIm1zk39ppkJAFmUvs0w6xWIw4MjTIw1gOmI16Eec49YjtsiQQEtkc9nsYy7uEtHL\\\\\\\/jYs4aRNAHSjGd1bEVcB9XCqffYAi9tL8ctiMl1OoswP64z2ykVue7JEt\\\\\\\/ZR0IhXA0U29gCm6kOZt6ZhCD3tA3mYxRISco8q2O3dFEkNlP6tAxkp7A8XOuOfO9maUOKOyZwmzAPsoe0ZDq9IfaVLv9deuUwBf2KsGcU\\\\\\\/IPKOuE5o1DiOXcNsLE2M3Ce8x9vYtP7VJybnTkbCpYl04BZule01Zc8ZiF7SbkNyOp2zTKc7hmlx\\\\\\\/sr+dky\\\\\\\/dXnEtnMQ1oSsQuSWiYEPrfLzj0ipD3hjDXqyOmaJzn1NGMmHdQmVDCZacN2F0QfZUaFuVaj9HbjoZ9SiWUcm+zxUpJVQkHjL+SXcK7MJvcqvUVJTI3HF5C7Tl4KFVa3mJeyInEHiWb2ARj7VONsbd6OAbO+bI17Kb751zEVfr8JYGcMhbX50tpcfdAHh0kHrX5WjcoV689td8lajDE+nqscEuK9uwlp7xr31twj6vUFb\\\\\\\/qoMXMQvp6sD+p5tzP0WGdZwvhYHlHSvpHchzxSU4\\\\\\\/28AL20hggbSusBCCFCBaeJ2byhZM+xBEkgfnnT39dLy4typAqGZBSEqxtABIzMvuNqV07AJkRn6c9JJc\\\\\\\/hhXRlY9RIxangcAFcpygxwUnIKGztaZy\\\\\\\/T79UqA\\\\\\\/oLarSeLUyYEsUEQFFyUrb0hskOFiMUDtzs1OJ\\\\\\\/TqBgmMjso+79Sro2pJYrrAH+XYW2vbLuiNUpBgXb9t12K5TQrci8ikjzig46eAjLNBq87TNr2MFa+wOvFCcOU\\\\\\\/yP0Gg7Pvsx8BjLooDgpdiyal75x6B6x7\\\\\\\/yZl4zOIIJ9sdT+DPhCMhW6UFvfBjpnf4d5By1mripdLKjyBVtuoJl1IMfKbME4Dr20yIg3asOaKYhiAY7Mv8yYu1bKWrRtTwKqvezvnKtIvyPLxhoO5ohHFugyrsJ9wcBQzBkZGEsHUyk\\\\\\\/QyjO6pw17dozrM0x121UGQQdBDzQB3hbf2BwJD\\\\\\\/NraDaHX87j9x7n8Iob9\\\\\\\/wmDjOmYtrEVIsEYQuoTHyy6aTysFg6SeUNCotYOmanrd+fUHHTZc9dfPSaQWQSWy2IOsHdIxCSnMsDQ0UE8\\\\\\\/FAuE2zyvs6fMuR8g4NMFr1\\\\\\\/hkGLqCqO89Xq2jyh\\\\\\\/\\\\\\\/4RUmlPJy6+dPVQPHGb1bM+YdTSrutfclykYWfeeTFi4qzlp1kJ7CR1140hE2CQGY1dzCS2JBX5NWQ33uWXp21fgjdGkOD4UP8HGE2DCPV8mml2t\\\\\\\/XzfFk4WY6eLvthMy7pYQU3BMAXEjWmTaEOOFIEpI31abpodVV8McDx2gB\\\\\\\/ACNLB+D6hTESJQNLsMJvCRN70v1\\\\\\\/50CGmXl5o0VcqYaOaOlH+kjPY+KFnGMFIMyZ1+Ii18T7nFqMzJsImO5t7DovjqMCqt\\\\\\\/mn04yoo6ZAkyJF4iZ2hQ4Z1FvwpDks3X6dN8Ab+oGsnwcb5ai9HBgw791HtKMh7g7vyQIoNYRn5PhUhvQogQ9YxGHAWrDBWdcq8BwZV\\\\\\\/x4nFtreBm4p65GWHcXfDP1KFzdF4cDJuFbNYXU+6eutufcVOwdiPeCC2d9pgS+A2j36YMG+AefQ5jYd4SAasWdpeJwywNW1waF2GIdxcXPPxIbQixx\\\\\\\/fY1s27WNFdDigBbHH2KH0\\\\\\\/ES2QpthxeEpxIkd6d5HpyHvQCmD4KmDq\\\\\\\/7OFKMlsir66n0tR43A1fKTsI+0vm1R3ZrVKAdhTkqyo1omkBSWv8jpl+mXt+tvS2bDcSEopgimx+v4UouOtSPJ8JKpbb38Mtrffg9ZDyXybDvBlAQwi1FhQfVM8qAVei1RekBFaNh0wu5XpAeVVuesoAA0vTmM+AuCAqoapLl2RXrQsT2vHM8SyHNfQgEs4QZFfzeHD8KSQLspGZWM9BSB19s81MGXLc1Ma7gls4PA5qNupr+F0AoA1r0ErQ3CbRd47LbAvu2Ejks0qtc\\\\\\\/W3Lc9FEn8v7c2zWLVW9EfsgPK8HDw+aPbsAgIivV39AlaqtIvsx6sXdznEXlVyVz3TGbyfaUi2Evkmiz2eOhaQXeL\\\\\\\/PxSWV5vu5X0tvB209KnKu9cMbhs3hx5tZddbuOqyObuiw4mhN+i7mV7gLVpMpYO3WGhMbGzXgnhxKfHV\\\\\\\/J1WUDjhWZcLajSqfLCoufCphEp6ZHFozxs1iw+OEbG905SEoVdwQUnZEq3K3aYFpwwVB3Mi+mWypPdJUL3L5nHMqogQKkI\\\\\\\/NnszyxMmzjwH92w0w948aOnxG1gmzcY9U250\\\\\\\/qx3YXQa81Bjo874YPO2GeTNT2XGKWB3DW6zdOsfZlPf7N6qfkYUPXG2eKMdLjzbwzRVg8MQVgsERh4gmXE13HWwutyy7mvhPWJW0Ul93W+7wuuIGNugtr1F4VinLwAJQ4phTiUIcarE7Ce8Ue7yHU1Dmc5\\\\\\\/G\\\\\\\/a+eahagJZvrqoHvagpLmXBGboc2fyg7Jo2ySS6VACgCI2PC0Isz20AYze34IgO+OkeTb1Nez07Gzqqs3u5M7oXGry+Og2l2G4clg5wFMIcN6mWaBJ6lqwbqC7pfoow1Q5qN2CSrB8hXoIXJqYNzswqk0svdkJ4\\\\\\\/cUGQDRKDHEsPgA\\\\\\\/YiAIws3hDjy7x0Q4ngftec6gm0gRIlfzN6uJxt9AGh7a4YBuNAYdT2x2ZTdV07VV\\\\\\\/0gC1RicR7nJh\\\\\\\/2nD2NOtZBiwS3Q8iQF7nqHfiQ0HlrPi0bvUJTHxY2aEj7deBepiinjScm3i56JXrSKFZW7IjB4s5U0xsKN2HdRprT0Ea5L\\\\\\\/JQzQctDC3\\\\\\\/vn4vWutrwxAdP75A7cH6xABHLjV+bz1a5I+lv4Yh8JWX3KMt9WY7cIiJ+y5gxvFkxDUbUtsC33QonTccjwKXDiGrTZTgUXr1n+M9zgblmpIFyLMqEwuZRRkZTn+GlOktma+AwHU+tM0Q095waFx3giBcx34i2xAVTLzN3YbLVArz7YWSQLjVkwRvltXwcIIuFvDt\\\\\\\/WEao2GxE6KBiaimyiTgzd3Y\\\\\\\/Upg4LlHJ9kmgYpJuhUSae5jFnZAQnPlyg0K6cVChEdWaYeincTiTLwDfTn6TmGCVXd2d\\\\\\\/ew\\\\\\\/vYYv+Y4mHtruKKE2\\\\\\\/Yxi4BtD62bkut2t6J2jhF1+p5FWlFHq8CSZ8r+cQFA7pvZtxtFsKBKE0LvQdvHo2J7bILNn8pR92dS1ohylEzden2ErU458POjk6ALsM117PsEd4FxWaI4fbtCuZUQ6QTE0BFescNyeL+AZrFx8HyuhT+d47wVueJcD017Saxubcg7lZyKfe75UjprUFdZ+u83ptXsoo3yZta4IrhGk0OMUGliw\\\\\\\/kEM\\\\\\\/0\\\\\\\/I1PKjl\\\\\\\/X9dhB7vPzvbe5\\\\\\\/fZ1VmcYbsyI4hsfhoaVQJynem702TKywPLfogt+rnGf7Fp+\\\\\\\/a+++oJdkT3prBHc8ilS6dqVXkU1XvXErtMzPv2R4U1qEOlAldbAbiTmp\\\\\\\/ArDtL93Zl4BzudVjc3Bzb6PtIkxxllPlgIJ3cv+tm7PRpxNhD99+3IXIux1YmlqQhkASPnyu2oNifyCL4aU6JH0tca\\\\\\\/btDjZ3yVorg5zjKE7Kni1INHt0mqaQXPjp69oaUCsZseEAVA5s40q7qAPV1Nm4o7rkoiA6ebvK12oEpA3RwxHlA9h3gOHBbsooJeEgehF3k9w46gckcPrSM3+i9q3z7Lo9KUmoGB\\\\\\\/k+BZ7UfGYlcVDYwSaW+61PBRaz+U53t25N9Y68qejseOQEfw3g2niTEhloK1PhmzjHnYpzy5zTyV9XyBCI0rvjhCxDgDdOC7v9iCP6w5bq\\\\\\\/LieQ4Xe4NnyPyCTMntVUQPTLLRUlrLNdkId5cclkqlPGiVuHStT7LvzPV+ANQEiMzJnUKvPmvbxYErPoShKbQvpuvsuuNJhC9z67zO0qp9D8\\\\\\\/j6veHnDfyKoLjX4iMRha6iamf4GojZf1KAQ4W3SF8mSEgfrhMmOqlzp\\\\\\\/rHmUC\\\\\\\/gsrwwc0esYQLdHtDA0LBaxmRLtM395CA1X1AAalAXFgzOPGdw6RwDMqAmOQJ6GBsApgQ2QX\\\\\\\/CaS04w4Z4Q5o\\\\\\\/IMaALBmfmJV3lIo9VaqQnhcDazK9VDVopoRQTxqXiV+ZniyPT0972OJpmWNUOM85LTDTgj39NKWsEFTgLosClQc7O5C7el4i\\\\\\\/aC0F4mue3kPr4puWJnRbJK4vuYzuQ2CCspiefBobhhVgOdFoHPfxVBA+Q40VsEaSZpLeScPO6ILNtNIvHvmWkk6Tkl+cnMsk3Bu+L3Bpo3d2Qt2YslYTKhsAahO9UxJeu6u60DGE6MGXWRLNoS8QdpyQWXS8IH\\\\\\\/xevRtPhvsfuUJsfoJpI7CEyH+iOqtW\\\\\\\/5QqDYt2lJnI+3y2X6C0pIJYxCoUpcOgR\\\\\\\/WSIpNWsIWYTYQ8obVacnPiyV2XY41IRv5RmlcKh7Y6rZYV4OMn06KSPXX4+Pf6G+rxMZGvrXZwWwtoC4A\\\\\\\/bhljlwEvrMXzn3Ly3OYVRdhVpXYfMW3+BOCaGD1lU+VdMoJwYg1xquSwYdLkAR3cS6lJVwwcymAwn1UeCCnBxaCWCQlwNg8D94I\\\\\\\/9d4NyOc5yfWXPF6UhCwgnW6jOhnClizCCTO+6luqu8VRVDQR6FlYmBiWHJhJt06GgP29oy30hxUaBCR4F92Z4ZElyGeAJYK\\\\\\\/FpaDjP2iiWRxPe8JXg0llX4EZqEwiLyWJidIdeR7Nk+OJnn5rBNtgEVB9kLHPr7gou4Fgfrql1JeqnQGtzZa28CdVNIWCV+rzPvqT6Pkuk0nyjhochDQ+rN0UQbb9vdQ5\\\\\\\/pafMYEkd1ZPLTwGRi9r9KgJmL\\\\\\\/lO7MGtY0ynegEJ7ObzhpUIsdSNrqbUocEhARg3zBod0H3cGmvjBdUZwqS2sdxJpBRqR5jzyNlzXuoMfk2YfV+FwLUekHMwZcdV+PwUK4Nn3zstGmqMzijZDa0+7yIVBy4nVq6OpmvNhGofrASQYdYQjH+ARm3BeWA8opZZeuKqe1lgf\\\\\\\/lImieTM\\\\\\\/wLRRZecqHTi6fW7IRot1gzcaVIQp3kwHggCbaOckY6N9fTlwGpUIwfKHKOGYIryplDxM0PmfOb1jxWp7Ud7hELG83yeo2xx8ivSor8vSidn7QmlyfNNEkq8vKxxWXn1NseG0jdkfs1toTtrkkzQVz+3F2yJN+ZWntpDyXFdDKkILv6bSl3B6iWLoXHAdhIzkM9MPcueAA406PuuyOKYbVSzCSavRkbuMpcg6RKi+Ot83qaChZlA9kamvFcvWxu0dtnpWg1JJsFAyXzqiqDFYdigpZXtWXmWN8yzYHJCC74r6e+sgbue3FGmhSSXCtCeL8a4YryEUVfcMkXZwvQxk39hOh1EYJmHeUgTYymP5yzVcJ7isf209+1m\\\\\\\/cJ6PQZvsisL6B3yFxz\\\\\\\/UedwCc72eSt7EOmHTYubpk1+EDyDk1rw\\\\\\\/6hi77ioSgmmV+kWFhR7iD0tTkg0+um6EQkr3llh4XkL1STdwmxxWJLW1tgl7OmozVmsrUOczVdbPSzJPfG0mQcnYb5Db\\\\\\\/pboOa5\\\\\\\/T4Wko75RS5i1myO18oeHNrwMYabg5Fc3QlyTduVcUP54\\\\\\\/zpgO01s3bLdR244C9OBsGJjY\\\\\\\/YtqrvC8qOkeLLPd\\\\\\\/DNXQamZwzoHAUnCuAKH9a3Y8r55NEGivxvGrv2kiEqN4KTE\\\\\\\/1HJSh9vZAxRk1OakUfO6U57qQ2L2F61tlyUWcU4b6SHWg3rOB8N\\\\\\\/LSlgu+eKoqa0vBvIYd2XWxikc+KBiTYOHa7b6bll3PVNBlPKEhbcynSZiiIkVV7pD3b6cQ8WwXlsEbs8XfwZRjv1rY3Vt01hieZpny1e61uvme92WW69wb1SGs1siq3EP0dmVyCRAHJG9UOEZfHhl523nMetd2jZ\\\\\\\/7ilX89LZZgb9hlL7+sjC0bHjxO83zoiqvUNN9E2x6ckNoeAnqcDqreOMPcHR1Gfc8+NQL65dUuK5GIbFO05fqWRzO5SXvAny7uXE3Rc9jxjIHQZAmGLx7N2+4rY2iwqr1wdn0Bbtc7ZJ4jVigqVw7UjJQoDTk2VHBqbtBsXltZRIettoI4rMWx9sq\\\\\\\/0ugd0B+d0HczgsdtMKHfMrS8mMzrRJ\\\\\\\/fUWVeGpgBy+YknBbj+e28886KN4OXFB9zWB\\\\\\\/2iwRqEUq7sCS8l8ccnekDnFEzvtXzv9QuO9WZe89+xU8XO\\\\\\\/G71yTuK3LJUd4cgRWOI1vDNOIxxsPvyfQHE6jZGk3GxxLNh0aIPQX7\\\\\\\/vINrKpG9BdnFdoeHkb3HY6Obg0FaQkXyBxx4y3MPK7BUryZS6b746nvTXCESQ7ry6ZGMmwQQdxRnXL6ecn414lqO9RSVV0eemNhIQ5AWAxPTXki9ETU6ExRmLVeU+rm5\\\\\\\/PavT3GOXh+4EBayVirWwU5XejXIhmigDELD11wBYGDLX99LVjxiFzdWKWy2FEV8C1WlXrh5W3d8zcOPVMxcckniSAttGgNiZevpsZ551ZNtXHA77r0gehvnd\\\\\\\/LTEAII+U4IiMnM0BNL30+tPCx9RkhN2wTYygdol8OJl163t9UB1kdwopM3Ze0o9U+Z3ly\\\\\\\/fY5Z0\\\\\\\/t7hmiO0na3VU1WVu8n8sDyABiq4nOYEXNBvNA0PlPD5DWXXasR8epcwNDzpG2RRcIGCdlaHofiMJbJ5sgjPWvufSw4tERuhdJDFltS\\\\\\\/FpqVECFA+redA6wAZ75xhNuDE4nqrtMxqS7jvd8owMX7Jbwr9A193WJmSRxHFZw5+SLZVRQY7dmt03+eIIZq1wQ9VtBDcdcjW9QOxRsv5KLOWd85rlvSL5zcpVH3psx7Fh89EiyMrG+yCedWd1LlVr3jgIDI0JMCSE\\\\\\\/Qw5LVpIUJ\\\\\\\/y6oOuan9iWBjRnym8lhdNhUH+Nm1qIrv51SLQhUlDTUEmNj8bUWhC+ecVJ4cMOi0Bnkv4URoz0cVx2BeX2KKgAxuYkMAKXsj4uMPbJBux+zL70YipJDEuq0LjQ7BhxnhZAG7YyYuZj0dZ\\\\\\\/Qabi+1ufQpWcTT0RFanKWdVZe\\\\\\\/LYK+ADaGUnXyf3gfepSDwxNgHmTFXF+FFieSY3Tl5BC2goInhTZ\\\\\\\/qyyrcA\\\\\\\/fKa50ZzyV6SuRbiH4yyLpXKu+mHy+kFgyXF\\\\\\\/aIMYB1DFv+r9o0aa6ZQyW707ly1dhJxTxDOiZzG64i4IjW9ECN8MUvrooHTXIqVDietaiGRQ9zn68M7fm8EDLMzK3P+GwndIpqc\\\\\\\/kuV3ZjkxF6fqkLJVdS20WrJ7tGRGTOdkBvjYlhzEt9EGQtxJHuNLUXOTSMKsGW\\\\\\\/7kaYVxnTR2QH+NkkaOjYzTY27vu18f8T8Ekyl8LzMDXov5f4+ryz2LGs7gON29Ixb4avYjCmpPVhF1+1+iQFL0FRoTXBk3oOXoGNVZh48hYMZJaGLK2rsqTS1Jxng9Yv8YxVlVy2b2q6WyACMwllZtBEjgUrA05ytCInRGxOug7TPZZ720hywjoMQfgaoUcgiIaz3b7lxxFIdisVZoV+bqV6rC6zNj1p5AmtC1StQM+zLMnxGJuqypkH36KTRK6vysLdZ5h+1HYElgUandQ1zfusm90QyqY7TdZMbjb+\\\\\\\/uPlBro+6LSEIMnFRs1Mk6DrGBMchiK94va8x6jFAqKK3tsFQd7UkowYXKo++mp1OE9SxXwUowUYcb92C3NW5E+ubAoaQ0h6F4j3Ae+AAU7E5EdcVnXdcZ1JVukpIkbzB81vbEifajEJO8kg0Dh5uP19f8L8W0b1qfjfTBuKThvFPSETVnpcZFHfLJKORfyucQjeXEaPVUZgUhsj7Er3WqBD2wCtjg4YszfFf7XAg1aMmlFUM06RlUx5YEyFlF9I9PT7HvemC2gBxLaY\\\\\\\/zUrvhCk\\\\\\\/ShmuLYib3n+aDwB9AtXoj4rFEqqqEFMZVHVXgvRO+gq3lOv6ttS7Kr5HzzRWRaMIeSmaJzWjh\\\\\\\/yo552up+ZV7ar6UDhufovTxoPOCj8NGendvqN5TiW6ozIS0TB0U\\\\\\\/ys\\\\\\\/LC9Yt6r5y5p7XSRaJsoLZjpC6eODgcdLafnG2KhOqEyUvzhDEDfwe7ujDje9IOSD1LkzbAHRUVouhsgL9Xs5gopYHVUDfGZHUvWZdcNAX3WIH4\\\\\\\/Pxzb7EWSaBTXubOL1jvt7dp5QHZeuz27jF4nF9AG56QVjJkJBwhJPQe+W\\\\\\\/Q\\\\\\\/HkAjCxAlHRGvppvnZFBZhJBWe0i1K9VS0h1sTaeDxdNVmoM6k5LAowE2Hv0\\\\\\\/Hy1dbcXPtb\\\\\\\/0\\\\\\\/rLqOs5AzR5h6EVy02dusyqpdDHtfjx+dazKiJPHI7KHwnqRq6DWGA\\\\\\\/eMmNGoDQPWJxdhtwA3f6tF\\\\\\\/WxU3mm8pxAig8YkXFHYldqRee5zw+2MTSItIDu+tu\\\\\\\/mo3ufIHhxuGWUYy2XL+X5Ad3NQIWL2e8tiHxfFWe9WMRTPuPNucH+vjEVdsqnjR3gBs2bx\\\\\\\/BRzc2CRxZCjFGpHgE6j56kg+sFt1Z+f1EgLSs4Xg3xsMqJbMhexG8PDzl2fo8sphY6N64Xlxl3MKnROCTrWL1ohp5BVuhcNUORgjDAW5CuPkqP3aT7PwguyOCb9AkHnCTHmu4Rw1KCYZ203I5oF9zNhOj4ktTx4bWCbPIVDcG1fjQDhWXGb8YFoiQH1XIAdYo+9nQZ8+tvxWJKbBstOwazn+Ww9yw6TgxAAY2kAMzM8epS0Jcsvtpt0E0RrSQEIbi9EFMSlurrh3Lv6J6x5cmn3pOvrOvUhWgOd1OVCC+bjWAOosNIXb0+6as19RMa2CU\\\\\\\/GvF+0Q3crv0EqyRxEgSHhDDg7D9Y6sb0CdzBbCf0wB\\\\\\\/O6HJtO4H7Wub7yMUf75G5pUaKL6fk060v1PYuYS41XZdpa3pRlVouTwF26Q1ajS+j2n+e2gr\\\\\\\/CJuqAxARYogqOq5FZKWbNnVfvbEgbNkW\\\\\\\/OiadcjY4EIn0fa9srY4zMfpDZNg8hpUkGgZeNIdMR9zpCNfROlpWFh2Piw3g5bpF\\\\\\\/Bg3mospFUltAkPQ2hryjUvSX3Yntl57n3RzAEs1g2IkZI4FOzHJs+kWqIUMp3VaPYaAZx7Xq83qJH46Su0HF+5iOb\\\\\\\/e7WB59BZDfwZY1ulsn1oyjZ\\\\\\\/KX0Sn6BRdT7v78g0Ik2cS0H6wud1Nx1UP8YAI8AYSEzCAAC2IJrwlI719+hJCQ0pcvgzj7wlNBEiiYxnsPsL8MnN1vYF0fLRYSUB9knc2Ut9xljuYFXLloe\\\\\\\/w6OsbrgQtXd0yLY97+uwUH9gs+f06FiIVKlr90SKtCN1wALdONljjseZMGxS+CYQzT\\\\\\\/SAvADCGU44nXrXk0i2iUc3JImv\\\\\\\/qJ7VPY5\\\\\\\/eM2e84sSgzOxXIaYvRieTmmnUchU6gh9+QwKzyWOgzKEItYycckJDG1EHKo2UcZ+Vz+T9QAG4k7sgEqzzms8wmNZvLsN\\\\\\\/VBWpWSb3nU8copw2Y8Fv\\\\\\\/o7tNYBVfgEacEjoheVgMTsSaRGyDNAlUwOSvXY00fzFm4Tg+n2Mt4xkNH2J5bLxk+m0kCEuctpCAAYN\\\\\\\/\\\\\\\/LujRxoIY8uq0gAE\\\\\\\/dmAlyXFaAnm5eS3JPzv8FlaUrqjivzaCT+GO5T+BJyOPsSOAUBtGKGW7y8Rzu6+ASFVCFe42hLLG1voYpeVItgKDb6LiyBq57tTKTkOvzNn\\\\\\\/VtQG451te73pIZYSnnEqHt+gJe3jVDiF\\\\\\\/5TU5W0MvrMmd6OAA3yfuwOFDMH5FUYUEttv2STabXcBm+53bsfhse3hxFFg73vHgwRaYwEg5V0w0ihJNFc4Wb7ZC2niJhj6uCIl1LlMtQ6RyFkIoaeWQuiftR1pTgjWpVjXl\\\\\\\/g5kehag8GF3kTDXo\\\\\\\/gcd8o+rVVb2BpRcBHjmCvU7h2Qn7Bel4OaYwOZMBDyMfMd7hYFfwu5jxxCJ5vsD3qA0nufQPvhH5MU6iWymZuh2mbrZlabShmpWxXdzP\\\\\\\/HdbjF3fZG8L0SZ3zzX404X2oyjVrKKwo0A4aA7FsjmqNrLlGDuCcFePIg5w1IeXaChbofY5BvH851Fcn1Y9miAIVibV1PYD7Gfc6N61Q2VT5Z2jWBqQB3qYIAip45oo9kdTMfLgPGxEgQA85rKP5T+nFdGpgcMTykhvH7qpq78byyo4cLsCI+6p5TxmXyyHsgjXt3j\\\\\\\/fDZP7pGH8OF0OTPOQmZReTQNPVJmXY3w6lCAlkY6t6ITsVbjsN\\\\\\\/uZeFfAq6FElQxC1jBRgIq4cutZeA3ubJ5vbuCsR8fccH1CiYGufOlfKsR39FnYYBgWtYU64SWH8f9OxNcHMLqD0vlBiuXS1bzXi\\\\\\\/GQ7nrBVauOAv\\\\\\\/+l16bCRccqbuiIBlSAFKTaTNT17a794Anm6OBrGQfcC4PvLVKo3YHr93dhTaUPDUKppMBnfzIGshgU97pFX6uXmx+sAb\\\\\\\/36L6X8LTNR2iB7JE2bdCAY97UEiLHDqVA7K\\\\\\\/SD6BaTgCCqF58ccFcpOM1UBmTLsnx6LGVq2KxLoZB6vEl3NOVCwqYmq9nFMSXD8Hh7XA8DQ6NYYjt88LwblR7KZNVdN+i60YnuLKJWZrWmqCLm61sIW3Pl4ga79dJ8eFoLYtckUpkrugPSO4xZhfdTjIFWL4a6sVUewCzRTVFHtdammjMzgFFuWt3FolO9NUETZjYH3OzfW6ilUR4Lu7ughEBYd\\\\\\\/91B6vZ9v9716GYceRapX0ZUEM9Fu2vo7CIAqjJJtj6EsynjOwbhrhFgKTAmC2Tx7KKjDWyRN9WgeZ9IQV\\\\\\\/tAC8kU\\\\\\\/wQmDkiioY66AZ7mlBwmPN4cDkr7qJkr0MXe5I5bcC3r5vwxbdSqv+ZkXIGaKXRLur8cmpj5jy43CqDGuhDU8toG5bXaIHA889roOJ4EmepEBgLOZfJkA9SQsDI6U+2D4NmJ9mN00UNt0VTTy3MDLebTt5CLoiFUamHyt7ojoCM1CejstAXRtI1QLk0J0FYK+Dqf0q2j9cX4dPZjQw38ow4GGFjybBjTAB9OexV412U\\\\\\\/XXfQnMx0YYKixhYFMUGGIpTrMXODhwIKYZvle6VftDaXD77iTKUVSbhF6LmbZGPg\\\\\\\/lWtojmh9R\\\\\\\/Osar2RmK68V3OwWi+8+Hfi9P3tgEJIGfsJeRvg4wqGFiSa4LBNo2qTPjaHrzrA4kvi\\\\\\\/qrKkeeBVr699vNee823AOG0bxY2jb+DKpTsQ+8icH+DMPrz2V+hvFPKhGQX22dqFT3uz2DsGlOQKzb1+05uRbj14FyHx9vzX2tEFCh0zF9EBtlcBpfddI432aLKqT2f0\\\\\\\/BBmG2fuhjECv+\\\\\\\/V8oJ8UXPb6+Jzfh2NeIfj9kGhUes3vdzhakRrqWOUlDUPv+piM8BpiQXyxbNXN1RYp0V5+uAHsSl78CZCQSK9JLnwyjHFgPBu5odh06ZtsIvbgof9EuokQUXHaTy8yqmYvFqGxV9Z+oQoLuZ39WrFQKBo2BMnryiLSDAtnehApc\\\\\\\/Lg3ed8doJ0TR++Pr73l4h0ZLV+fr9B6I9Mq2k+k\\\\\\\/CXif8X+IKjcD8EjT+RyS4dSbRqVLJyGqOqwdn8CqcbbiRkTk1lfXOWh1XMX26uo9io2TSMHB+TEctK+xjgYPvDX6gEs+Yf0QZ2iEGpGuGg3AN9rk8smmOF5umSMCpKwjKGOQPccKQcJYZs+ObMHR9kdq0KjhwhZ1bI4uYGJZrTGW8qOQXh+M2TpszpH1oGxOh0VmEMJmg7DO+eFl5Ky9XFzw88Zt9hPLI4hEATO43V+UwFyZjNaGotJuatM+a5iSxl0YTSMLxDlZPYGIQ2repnKEgOE3aMgyL9jtvs9H5Ly3hO2PgLZBt3oM4ZvaP9uVN4fWkHZ\\\\\\\/c6cV3VHei7nOe5HpD1xMBOs6IfJW5FBj0gz1SxyNlPFNoxOqVWCuNsdf\\\\\\\/3zLhkZPQRmvNh1buF\\\\\\\/P9fSL6BV01Vq8S9Vauc7COvDdqvncIGNPUtvGutN4e\\\\\\\/ynJCwpkcvhMimlgELYif4LotB9E3WQLqi842ECJXfPpt06\\\\\\\/atzzOYQj3wcKtV2g0Bz6AqHmLX\\\\\\\/PVVwQHcEHIry6PEhHL18Xtu3NksH5P0Jkqnzwm6\\\\\\\/ccIV3Cd0L06BM+0ZWQ+2LJm8uR5IYDKH3\\\\\\\/N8BiSbjxVcsnhk1ww9c\\\\\\\/2wNhzZXuBT5lAgV6KGvwAwsFZM9BMqXH6kl9oJk8CMYaBARZ0rvRMKeK\\\\\\\/FLb+l4I2AilHuKUpYkd5ghOYpDfmzFY0yFbzDsqAAHzazxue\\\\\\\/6pb4chA0VnwDdKNXFgP3HuErEUQOLjuWY0TflvqiMkoJVOE7KdvaDZ7P3V+IdcB\\\\\\\/H4wDGH4JJt2In2O+LGT\\\\\\\/jpaFVw3SKtYHT151tvWvH0IQLzaWLQs1MIIA4N\\\\\\\/sEPcvf76PeIVEUhQblVlrxVjDQ5klFP1fw7QFGtmBJ4ICggvXKcP0dlogDu6A8o47TYtuMGj9xG+UM2rmJ+6S0ihAocf6hB\\\\\\\/KRp1xKfg8f1CL364CZWQy\\\\\\\/\\\\\\\/H2SnrZJ0SCXU7kd1WimMhFw+SKi1AaDMtg\\\\\\\/teu\\\\\\\/azBVdP\\\\\\\/t8IeJwQUp0RgFitYBTJMUJHxnaLZ9e+Q7XWwzH1UUiOlcPUKKLDDPMSktcps9\\\\\\\/b1wzUzW+u5XOl\\\\\\\/wr1Mm+wZHvwkwi1rVmSvABZBGYi7irTWuJ4lvymJVpanVhsat\\\\\\\/fNvNc6owpTzAY5h+diKOOltf5fK4Bi0FlvTDtsWIRM3rzih3kzjhmQFZUAm+EpJfBWkG12TEoF78GTvht2BiuyygUf6CA6skjXp+09eqkF2NVQYrAXNITGZNLKTUo8hEeebUa7qzLz9sWsJ8az9W04GSGPppU6YOU6RgLz0kR3rVvtLPt+eY3zIJJLv4jQql7UfiYtVzUfzi2LZHStAuTigHzFiOccOmEWEEwageqnuPEUI7BB9AcRBbLkabT1SvMoTMT6qb2jBkJ98bDLU4X60ClL4FUo7tzbSoPLa9m70hn\\\\\\\/q+2z7IQxCGJcoEyQgWucpgtU\\\\\\\/Z1T5QcN1K0sN2FLUnfNAGw5fUC\\\\\\\/eHiyY6WQABItHBerVdvb70agHYkCHXq\\\\\\\/weWuwOX4+RGZwAd+SjpRvKG9UiFGMilrUeabMLgjrJRKlmWswU7PzZLM3n2TWsEGzhcXyg5NVPm53w8f3w3gt68RYUNORE7rWIkjmANRkPjfDYqp73qwGvyXi78D2sUeAZj97qifTJDC1CQEB6ySdrmdoq1X8sD2r1yvw2es+KLleCAF1ri2KYtoK2WGixE7G6YKUN7diwMLQQOXiVpD5kg69H04SaSHRjWgNx5fu0mu27vdlrEncFWTfDQ+BGZbCICcurtJr6y6ItWGDtg7K5xFVKokAd8gteU3oErPs1qTTq5pIa5S7+iBJqUAjfB8wBa1qp9Q9F46bLRNrKoUKiCrG71KG3OQeZl2a6jJawEJCnLeIV8DuY2ifWwdyAgpUfIredvwPeQI8jmbRsl\\\\\\\/QfCQ6i+NKVnaPOVUHH1ORT5VBSeHDgfg\\\\\\\/jPKXZi+zyZ5sqy3JLNT4QBYoPjUncSWvQI17+1GZDOJ6hQjBS74xRCw4z+THQUhiG3ZSmOhU8mvYqNa539ugq7JRfhc09jPHQeivkMDO1V2n8mi37MtD\\\\\\\/SO4UyiJw\\\\\\\/uiMuIonZbL61TWz2WjBjRw7WurCrR97GF\\\\\\\/zvRIH7H6bi8aJifTah7mexh5qDN9VeY+IefJNr7Iq4OcQRxktgcI\\\\\\\/YmYADvvVkbDMav+idwL165tLAPXwyaxN+wjsNzc8slfdzRgxFFzUHa2+AJ04cyCOJiQLtHGCH1aVl7IvQjER4IR\\\\\\\/uiHfgqOxntMPDK9BHf5Fg8zpUSNISmSR2ofTvlCeRqvEi7ja+znnpMMULgbNXaDyrZM1CHKUljOKAOg+49geM9aIY+mn2ZEOKKfxSPJWhPmjS71rMru6YlnPbnVLzf4LrpDLPVkV864usYP9cou+5kBS16EncPbzusFzgHRnOYd1fZKTnCAow02OACDM57sW\\\\\\\/rnxu7eyPoz65Bm7BziziDsXDHpV4yUirs4kiFbeTP+qJzPPp4koGx6jt6hmav+AqWBw3eLNy6hRAPbPUhMAiPGAZ0h4HEKoeJGLUAech7231txSiZu6ybfo1OZSjA3j7fjQn1eY\\\\\\\/DvutmgaIZkMkxwKR9ZbNOcmSW\\\\\\\/QBpRsZMEQQXUHxjgTWkmG6OlrqZbpQxOlacLZd7JegMq5hy3FGWLf2kAASWkTAOkPPxOScX8ATY1mxDscKCwq6b6HZ4aMX5YY1wjozPKu1+wCREBFS3e4AIUDTYuqDnUB36j+CUcFXsioeUg8DwIHvHu8j4psAG6bZpkarks7hx4GhAiKLYDu0SQGrh\\\\\\\/JqfxhVOf0+jfRW4q6Ptws8H1SqPzBhkT0eT6+e3PoNa0PL6u5KE4JEWNPbeLG9W+glpVqLQf4JDhFXRxwuyvpkI6mTJpEQFPrMqJzNknfQkmUDu5UgCoijYLyUN3GyjmvLyxpluIP9BAXoBoIMUeipdjKlX7CNGZDwzmmapJQy3DAr0TQ1mkThHQZHK3iNNcHZEfC08GMYAxKmz6VyhutNi8c6dQX\\\\\\\/nRC9uF8bYhLUrbupNAzAwwefkzXBeo1svi+kwW3Kh+eDqMNPKrQZGVCf\\\\\\\/7J8jqQ1pmxJBTuQEX5udBk8gR8GBzzYfrDO1EGOWob41FMFF0vad4PULRpURKrGH2ZgWFLPD4uesUMSgIrZv29+8r6IoCoXB1\\\\\\\/OttLwpsB1zrR7sAEfyfUNn4xwUos7zda2rul7WknrXdV2x\\\\\\\/uI62OV8UF+sFzqSCKdCEaLkOtCzkI7S2KVYyTuhHPBvEC3jV2QFS\\\\\\\/MkAcuVyLdjr8E1\\\\\\\/Pja\\\\\\\/lVzTj7QNhUSkVjhMTDCAMSzlsAefcja2TufrP+JWZdqYRHfR9ZHrRzHCyDlxPU1OnjT5FxD45vWUHs0sRbih4lrr0cL4eXPDAlgNwgZBFihnHRUOxDGSv09W1prRsOuDow61kYW5fXyz9949O+aKlhCR5pmQ53gD+lNCdSIE\\\\\\\/SjmfqIIBu6nWZo0VfLvLwJt6ZvOM+TV2+pjAc8J7u7jE0hb5K0pI+5aT60egZonQCTm9lOh353YB\\\\\\\/2wDklQziM80C9\\\\\\\/b03WWbcb9IwAC3oLaCP\\\\\\\/2Y\\\\\\\/gs2ivAu6C\\\\\\\/3MzPv1C3Zuup1MoSbfne3QIVHy7ftJx+eBKXksZiEh6WbpNWEwznwuouKTNfz1wQuf47Z2l5UkRzbf\\\\\\\/aSCFSROngCJjUohKUeg9\\\\\\\/Qhr88wVoFM\\\\\\\/dC6VZMDgK6xpYoyoE\\\\\\\/s8iU6XBN0NnYQcPIBr1w0O9nZgT6TWPpDJjTmvjZIBYxcFD2qFvi2FVX4RpU4z+5AShuCGgyH2brYKcXGFCFsRtZv0or+eqX5Sy0Xm680d9+iENfauYtLhp9UEAwqYshIuc+UBvBQGceV7vnhPMypCPuJHYX+bTLXpJjeEjFiMuj3uHPaTXs2nGWhQt5FduhFIbOhDQGOwitFct8y1OTIPQjZduegXjcFpfHaTmS8I09q8+EUqwK42Edwf8Fr+45ZNNO5XR0uul28R9mSOfIvdGckyQ7qqcZTVsyY4OdBiXc0eIe5SqRf7rKE6maFsnAfmZEpQmZ2KQ1mbjZvSuTu2l01TnVD3Yp\\\\\\\/8PgKxA2R7e3tasTJR7X4Gv98fHnte1oVHeWdtrs3Rjq5fBy8oa3XwIRC5gHcyVsKmgmZDKftb7OJ1xY4r4pQnsIqlEyLi0miLlyh2YOSefzCz0dbMbJyLcQGYzwwZQmP4wqjGqdfEK51hux7t1vLhQh2M7kuBGB00D80yNrjO7l0Ve7Re5F2MOeW80RiJjjq1DCFcI\\\\\\\/C2MNtOxJ+Gs4dZJYmTOwBS93QW5d\\\\\\\/Fw6dKP1NXbyIm7dOkvTMIlL23qkg\\\\\\\/JzzVpKStD5BexdScOhPX2uH3Im1y\\\\\\\/fEfNUWcER1teNCFqsQtQqosd0sRIDGgPtNj\\\\\\\/pnXgZvvR03kc0JpLTCBIRZKqs7GAyFzc03HE8ORuwLLckZgJv04JLyfezHCq4KLO68Su6zJZd9FE8402YjJ\\\\\\\/c85SmnJFp1L0BPondO\\\\\\\/UdiNwhMulzfqDyBGu99jsBOC49mZxIup34vqPinOf91aSemVNPuhiHerLiYpTgmUT2915z0TFjltwFlTw9OHN5XgEPEbJWAgqFD8PvaCI+AGR6gR9Cov7eqRKoDAidQiifHFRKn\\\\\\\/0cks5gVkfPY1XeAruYvh5MjbqrMK7U+eBdjeEq9owjmJR56up4KwnTOWblV15q\\\\\\\/udVV8VKA9UmMaM+tHot\\\\\\\/imQFFywrlnWhXcGilzZpG4yvMLg497twA6bfp7S+hfR64HxA+S\\\\\\\/fgO0ZASQmvRDk30nPXJSARAdWO6dSsM1POrrIV+xivOgGc16ABpOHN\\\\\\\/e6SHCat0OawWLw\\\\\\\/oSvODZ5YmGGJErOkgevpw54znrYt8kLirvzl43wrHQu7iVx6kYmP2pevSG\\\\\\\/7rJcnapj1qGRo2DloJrrq2eD2MKREq5OAHCDVC3MN9\\\\\\\/X\\\\\\\/PtkqMKlFa8K02Dy+CuXY4jOwS9Wm\\\\\\\/E8YSQlAxVUYCiOZI8ahINcfjnQgsvNMAprHO0mL1rDKSJ2SfSIRqJfgnuPsOWYqW4J5V3LJqZca3bHZZmn9Z5Qfk5iFbF\\\\\\\/za0uuB5iYWL6GoWlkQs2S6Q5yKxvnPjW7gzZvilkh1nUqdrpia8vo68egN0sIqXD7ANt0T0jhwKnkZ3wV6VC86cRSs0l7uy8rZSsOrbViUGQUEIfyWF5HRBIGzi1aXVY4RCQVjqlwKUFKucD9cgeer1lXeaoqgKXmgaxQ1zd+D8wt0lKWyuIJyHgfblwIayFl\\\\\\\/xXJUfKzYwajVbUGRYDVbSuYmtvTK6KSihQ1Qev9jDdV025RjEdInbVrZUJKyMi3VJUQ6nh9Z5+5wj42mnMQiJ0FdpwhtsuMV19pLJJOayGz12vaxNOt0A4N5r4MHY9nLeylpbBh8qg91tUDrXQMkiHrsMc1gKJ6eHSOFy0rn0C2jodO8IkJ7FRrsjASmTLLD1a5dbddHEjYGUFY1YvYO4XLuby9vwCYmqzpB8WjR70+GMc0pdovvgBtZvqj6l58SMuEsdh8aJz\\\\\\\/AjsrDLTTCv9nbRnS3Fun6Mx\\\\\\\/iP7WwGQhEP+tHYVqjU8JgdfO7QiCm5s\\\\\\\/5iVFeteDUhTNoMKDdaw8jSbNRsPbG+Vl\\\\\\\/FwqP2ojl5+9jrmjtgV+9aSqeYLsLiNrBqc0wWGGFz\\\\\\\/hQ4FBa0AYXvl+lG0STmxK+8qoHJ5CXioGsdfN2wqju2F3mb74YL6ZbBgT6+13\\\\\\\/WtY6XlbLhWv3sxtsxNyVtCT06qIPx\\\\\\\/nWdWdMa+KvCAyZfADcaO2mxOtxoqcThkszYoEHtudLUvHV5WyjVKfzdDeXLpmvGJuavQprcM6gAK01xrvfoQuo4GzLL86Ucvg5e3CIpUXXpOKFjxxZW1YiWsEjlNPHFpgHfJO1fqs8nW4hxMjACU5RuscYgAGOLAuFmv2stp9pCiFZ6nrHfvboX62QD+AV1AJG5JIXkur9izFtgWpckDUyede7tdYlj8z69o+LvtOWZVqVyiKWaNwb2K6vxI0xqjRXff28eMrZ07+Ic4pb3M39KNoz98EEzDZHS7YcePR3qigTqNpoBQ+E2B1l8Bom9QOsSEJ7h5InjPBaVavi8Zk3z0z8hS\\\\\\\/9fvtcF5f8XY\\\\\\\/VYpyF6aHVw3BZpFOVR2rdd7XveCwkF2RLtM4m75RlgXoW7pbRVH1aYbICv7tCf7P52mJrcHw9YRnu7uWnRe2dok5acek5dUiUgjG7PMMhspuk7WRtg2GsCHpkJ0jMQqIipdRGvOrJfy0nYnJ1avO5eOnqKgKbFiV7dE\\\\\\\/0J3IR5Ptlw7r8oXHZLRDcbk9GbdEb2UnCQyj3EkBxqhSvEu2DVlUeMgYC6iU+CIvGv\\\\\\\/e9cHjP+N29\\\\\\\/ygHTXIKRwgtP7qhVpXN+7OiLMnnr8czrP7SVoOmlWUe22\\\\\\\/\\\\\\\/0jZEHZlNqtQoEcwcLfbfyRpMZLO6AHGrYQcLReR+67ddMiy2ngmeXanvnCUrsNBcjzNtiMkQnoy3PteeIvFbLh183sTrZlpMdOuX4tNrnryLmneWT2IbmyRSCudimqVdatwDAe+M4plbSHqJu7ZVz\\\\\\\/3ZK+pnMHSzGKqAEAtoaNDRTl1ro0y0l1lfdpIhkGmdVpAV5b7LOvOMk4BNJF5QPra9YGex22O1AJWflROWc\\\\\\\/qmT\\\\\\\/5pI7cOjatlrnlvz4UfqUIzGwPaG1f9Cw7FumohuQfMNBUuIEgAyUvd\\\\\\\/ET4Adtt90ux38bAe\\\\\\\/NPdSviZO5JjTsg58xKpnVNCt\\\\\\\/vaaGNsIcoHFW88TYpaDU0eC3ZPjM4lv5o5SG7FPORUySj8awJuWFElIOnP3\\\\\\\/nXxCtxYuIRtBTOVNkpN0qkUpxicehaawWQzcA\\\\\\\/xmlZTbxbOUGVkcHkMEfEw1xefg62y0ID4fPCk1ZnEBftkxGMDAp0l1GvQID8ZG4KfmhzzH8xuOmywS3Q7k2Zd9H9j4+2ycqcs8gvw3QDY8QBIq9oSy84MtKQqOZRA1DgaV45F1TIynfb\\\\\\\/BeAbr9yURg8aqMPcZIyRF3SkYZD+R805\\\\\\\/qq4S9RMCGPVpnaou4ASUhMw\\\\\\\/VzNqV2pQZZGkYQDhCOdh1jUFSHB0oIi1aGX4zCDsBngmPgP+pFkaSU3LqBHT7lm+5XPFRoNxyMEi7eDR2QBALzwdGf3i9C+juvBqfU7uVFQXXZIpOWaJqGGl3s9sszhYg1BwaOSZMef7BCyWKNNaIIk3zQp6yp+agaODrRQphR7UiwKzm1fYZfAKJIDoKuknuvcL77AY0v\\\\\\\/KFNPku8bqHOomragP2pr2RYxFD1JG04y7zRFaMOxfSsX7Ymo9uiJF0bvSGMPD7giLhTI0A4aQ1Feiv4MSD6zUWg3j84LQDRI\\\\\\\/qqCCmU6PJHnyN27L4aiiO0bSKRUdRqnVETXjY98TD1vqRGidVGLMa39Gse\\\\\\\/yzQlo3zwOPRx4TH0Jueq4B4PTDtLzaMsIaQN4LK4d5CX1xpbRbeBJYm3i7YeE+UorwQlDdAe1AwOJwlEB69\\\\\\\/vIGHy\\\\\\\/8pUlyqzaKCw9XZhap05p823HYiDBIqOkTgPhR2uDSdhIJbAz4S4itb5Bxp96QvTNfGviIMBYVyYMYL4Qs1SNsNJerkGMeAhAJaSBb9Nu6ST8AKChXuqqcxdTqpEeO33uah15Wg2wZLOp\\\\\\\/6rr79d6g4CerV0eEsS0coHqTWAIeWnb6lRtw1HCqZmOpj3oRIpG+46WlOWxNgp2ZpfLZGGIYG1WN31\\\\\\\/Ya7fPQClUaUVChB1N2pzYevHjUeiRzPvZBKJWHxp46g7ykJsP51jUMYyy3WMGLZXUcWypzDmEIng+dk8cXDWB9y1GVzu0SPjIIxiRaXFuEwpzGZm4S6nOHXSrFdH+c6dhgUYRAzlAoE6b05osl32moXt4NoyOZdB9zCeEZ1vjFwmUcWcwHvK\\\\\\\/9Cdfb1Sruon6sDA3kn1p8dcLUGljj5p0MBLy\\\\\\\/qKgW6QxpDo3ZRuwG+P3kB\\\\\\\/nzgTs2iDW50K54quDiKYavX+OeqTEpU\\\\\\\/UKB+k01dXc9nbaZ670053Vkb+3Se\\\\\\\/3fnr2tX936yndAsGZtTcFX+RGh20XxZed8gcu764zReDBEtk1uy6Db\\\\\\\/blEGb1UZNX\\\\\\\/iKyfnosRtbqsscB3U\\\\\\\/m5xra0+y1TcNl8WLFa1ICKN5Skaf9b2XGAjIm9kI8IC7gYCZ0beGYAG\\\\\\\/\\\\\\\/OVjXUBlmNoCQ5sCykXgXC+tIEdsdyH4oFkcAjcei0gJ7KMDzvA3XGlG+id2UFY4W8tNB4pewMHM6hFe7O0+WN8bQ+NpYfubhWfGDAETzbdDX0cVGNsQ\\\\\\\/MbCW7EeC4\\\\\\\/E0unPKnsUOy0dgBxsa4uOwwuQr9DKPQ0vLWhJpiOWQqV1T1KF5ZO7Yok6F65nmJvvG9E6bZiHw9rsO3e9pB6+xzidiRLR3gIsU8DO2al577ND9RmFtV+E\\\\\\\/x+3ah0+VzUOQcq9KM5KKnBbeISABRCJ\\\\\\\/3Y4omOebToekmf6PZJwSLhFXLg37mHCeLyjldnA+36uh069Vt15XFEhQFJZSEU1lNSBEyk7Hy69MtyS4+HBDnZmHepUw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022F2sGkAdi1dEv8ewBPYlOxg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f472d1ed4bab3c7dfd878e89513bc0e87c7d8eac824b220fe4f18cefebe7cd58\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XVbZEBhOSTeBSFHSncoJc9gwHP1Cr03pXd2O8GtCYGs+zne0OHWGiJ87VzI\\\\\\\/G9NEnwFDG\\\\\\\/zmWsvCwMly5axJG8NkYwO3Tr294Ocny\\\\\\\/+SJ\\\\\\\/rKcZDwRgg20dg1hRmTTqVNMV5MTfD8YVZo+9tVSotcQ5spkMTs3+KqvFJA0wQBaoj8c5kMVOAQ0MCMB9zE6UU5XkuWP9k\\\\\\\/J20MEJhYHofTxAXHP9wteqhv7A1jSl2EZ1NFo1Xk2G1XDzxiXqoGSJiPDdRCN\\\\\\\/cXZPzA+4dPwMoVGlAV1pEIJXZlGd1nL4YQ7E9m4G8QEKWQpy318tfP77bQWZ9BWI\\\\\\\/AeLm4VcCFq4QWVpesXWKHAGKZjDrDsjbVE3gmbAmpjwHB5+TteAE5c09ze6N7wwEVMpp7j+z2prIFBrooLdhd8uraL2vJa\\\\\\\/8fjaOARRtl+U0ch5ya070syuUzIDscYGWOLnbTwUuvZ7ES3rvjyAPz0vd4LBvOzy4CV73XihqOF+ULE7LilrY7wLlfSTbxx4F82pyCBv5abXC5fwkLGNJZIh2Y9LStKe2beI3FLtVqeLva61BotlWDCl2aUnA11L6yg0\\\\\\\/Wr69IAbKV1I2tuKB+J+7XDlC1pD+4GnYsoRivSs6qjWxHICfeg80JAjSktcFQ3H\\\\\\\/s2iTmjhPhyhrO6zdNmYIwfqP6UkjV7syucrXYfWHyp3yx4jpMhbJ6wY8pAnvdshzRI+mLS+jrnydvlt3d76GilAxT\\\\\\\/cReQroH9zWLnrEg9LoAoCBixcH186Tdwh3q1D3ClBXNnQ2VQ1V6mcVtDoRDl54myPuv6vCRt2D2WemHOsAk3Rx4+VnPQi2LdbmtWJ4l5EIB0LKrWXg8H7rId7YyQI7jKpH79XDN8dZrvrTuNHyD\\\\\\\/yc1ifwE\\\\\\\/s8mtBS\\\\\\\/\\\\\\\/E3uNbWXxDOI4krFLduLemgoFsMs0jZTvw3J5KhRYts93xERzNOaTCLpl8xPJ4W3Tdp0a2g0vSHEcZJQT6Z1VMdZOl3tK5nVO0aovzZWIE+qgPMkt7UxxzhyJIENWhlf+clRyhuW1iLZZIyhhmWr58RxOBJEvvb5JObbDq1oJ4rAt8OhDy1isbIp2HgrKqjQosbF88e000yCgYbnVbtAUWRL+8h2PSoWctGb\\\\\\\/MAmS8tU+tVrUH3kyTVeEKvsLUar2RwIhW6iQK0lcyyGgrHAQViSnglbDE2ef6SBOfZz3qpL88Vv2N\\\\\\\/YTmxbckjy1uQFU0WIN6aJYexA8PSDT7Ef7uULOZ8ficMdAUPFBXHDe0Iu3xF0i4+YC8iuxnbaAkL1sgkRqbwdZsqW35iQVToaig4YaFzGDu0yQkcaGAjRyuGHmvKmJWdz\\\\\\\/Ws6te9B3Imz5ZNgQGIIEgtF1FZxIR77ozIsFu5AOcYDrkAoDk1d4xVaa1CpoMqVXTcpiOSgTXNCCaJkYHke6dYtWCRnTW+AGycguIk0MBc0YP24BzM6mnAwb5dkDaw2xZnw+5uZBs335KPNeQnN3J6JCIEh+pPWKa86uauL1HzLFiPj7lNOJiNR6j5oJn2lLHWE2jDkYBg3AFvh3mkK1sCPmusRqYXZhIv\\\\\\\/HharncOW3NpFfcCd0Sy2RBg+M5MEQbuARV51nv53dERSrtKwVUbUwzHb+c0qbIO39KvKeufUbjGezFFuJMR2WM91vlxSn4rt+niSZ6hyhnhzQfZwzXv1YzuEe\\\\\\\/L4SUBC4pgMXr3FYVMoZfku9NSW5xLBEtbaXBNEFa1nyF32vH\\\\\\\/fXxfCIiptZ2B3SfNgDul\\\\\\\/SICc9NxOd2xrSY8V5RSumswm+5f8tHL7LHq2bcKuvw8vsjBJ16ngo8rEnKoQGbcms42YTULAWLqZxtL5uZPXtQuyqvHRINueliVb5BOfmq5maQn1dMnQGrHFMMLCMIZm7HArG4ONiB4DHCEqCe0ZqrbUAQY1TlFF\\\\\\\/1SPDnjxHaEuMpwgLPouqzGF++yn6OLn04kAk8CUcEnuW8WjMkPm3M+QZ3RtsfjRVrSEIptYwsiyeY5hyB\\\\\\\/lpA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022m17blQwQUZqvV2skMZ2gVw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228962430a3f9a4a4fa521401e4918f0ecacf75b8ed858d4469062fa433f7a22ff\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459577 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/2059" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-FPJWOIdh5RY8Oae_brxEf-4lZ6anfgOXy_KmfN8S0wI" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/2059" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DYvwtc4AMikGxN0mYTu50zr6rqYXh2BvzrXm2fCT51E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/2049" title="Evolution After Whole-Genome Duplication: A Network Perspective" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/11/2069" title="Allelic Variation in a Cellulose Synthase Gene (PtoCesA4) Associated with Growth and Wood Properties in Populus tomentosa" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459577" id="node459577" data-pisa="ggg;3/11/2059" data-pisa-master="ggg;g3.113.007484" data-apath="/ggg/3/11/2059.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Molecular Phylogeography of a Human Autosomal Skin Color Locus Under Natural Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Victor A. Canfield</span>, <span class="highwire-citation-author" data-delta="1">Arthur Berg</span>, <span class="highwire-citation-author" data-delta="2">Steven Peckins</span>, <span class="highwire-citation-author" data-delta="3">Steven M. Wentzel</span>, <span class="highwire-citation-author" data-delta="4">Khai Chung Ang</span>, <span class="highwire-citation-author" data-delta="5">Stephen Oppenheimer</span> and <span class="highwire-citation-author" data-delta="6">Keith C. Cheng</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2059-2067;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007484 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459577" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Victor A. Canfield </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pharmacology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victor%2BA.%2BCanfield%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Canfield%20VA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVictor%2BA.%2BCanfield%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vcanfield@gmail.com" class="" data-icon-position="" data-hide-link-title="0">vcanfield@gmail.com</a>
<a href="mailto:kcheng76@gmail.com" class="" data-icon-position="" data-hide-link-title="0">kcheng76@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Arthur Berg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Penn State Cancer Institute, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Public Health Sciences, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arthur%2BBerg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Berg%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArthur%2BBerg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Steven Peckins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Jake Gittlen Cancer Research Foundation, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Experimental Pathology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BPeckins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peckins%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BPeckins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Steven M. Wentzel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Jake Gittlen Cancer Research Foundation, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Experimental Pathology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BM.%2BWentzel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wentzel%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BM.%2BWentzel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Khai Chung Ang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Jake Gittlen Cancer Research Foundation, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Experimental Pathology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Khai%2BChung%2BAng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ang%20KC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKhai%2BChung%2BAng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Stephen Oppenheimer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>School of Anthropology and Museum Ethnography, Oxford University, Oxford, OX2 6PE, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BOppenheimer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oppenheimer%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BOppenheimer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Keith C. Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pharmacology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Penn State Cancer Institute, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Experimental Pathology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Keith%2BC.%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20KC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKeith%2BC.%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vcanfield@gmail.com" class="" data-icon-position="" data-hide-link-title="0">vcanfield@gmail.com</a>
<a href="mailto:kcheng76@gmail.com" class="" data-icon-position="" data-hide-link-title="0">kcheng76@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/11/2059" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459577" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459577/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/2059.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459577" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459577/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/2059.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459577" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459577/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/2059.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459577" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459577/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/11/2059.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459577" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459577/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Divergent natural selection caused by differences in solar exposure has resulted in distinctive variations in skin color between human populations. The derived light skin color allele of the <em>SLC24A5</em> gene, <em>A111T</em>, predominates in populations of Western Eurasian ancestry. To gain insight into when and where this mutation arose, we defined common haplotypes in the genomic region around <em>SLC24A5</em> across diverse human populations and deduced phylogenetic relationships between them. Virtually all chromosomes carrying the <em>A111T</em> allele share a single 78-kb haplotype that we call C11, indicating that all instances of this mutation in human populations share a common origin. The C11 haplotype was most likely created by a crossover between two haplotypes, followed by the <em>A111T</em> mutation. The two parental precursor haplotypes are found from East Asia to the Americas but are nearly absent in Africa. The distributions of C11 and its parental haplotypes make it most likely that these two last steps occurred between the Middle East and the Indian subcontinent, with the <em>A111T</em> mutation occurring after the split between the ancestors of Europeans and East Asians.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/natural-selection" class="hw-term hw-article-keyword hw-article-keyword-natural-selection" rel="nofollow">natural selection</a></li><li class="kwd"><a href="/keyword/skin-color" class="hw-term hw-article-keyword hw-article-keyword-skin-color" rel="nofollow">skin color</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/slc24a5" class="hw-term hw-article-keyword hw-article-keyword-slc24a5" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">SLC24A5</em></a></li><li class="kwd"><a href="/keyword/haplotype" class="hw-term hw-article-keyword hw-article-keyword-haplotype" rel="nofollow">haplotype</a></li><li class="kwd"><a href="/keyword/recombination" class="hw-term hw-article-keyword hw-article-keyword-recombination" rel="nofollow">recombination</a></li></ul><p id="p-4">Human skin pigmentation varies widely between populations, generally decreasing with distance from the equator. According to a hypothesis proposed by <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Loomis (1967)</a> and elaborated by <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Jablonski and Chaplin (2000)</a>, decreased exposure to solar ultraviolet at high latitudes produces a strong selective advantage for decreased skin pigmentation because it permits increased dermal vitamin D synthesis. Consistent with this hypothesis, in people of European descent, the pigmentation locus <em>SLC24A5</em> shows strong evidence of selection (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Lamason <em>et al.</em> 2005</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Sabeti <em>et al.</em> 2007</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Grossman <em>et al.</em> 2010</a>) A specific coding polymorphism in this gene (rs1426654) is a major contributor to the pigmentation difference between Africans and Europeans (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Lamason <em>et al.</em> 2005</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Stokowski <em>et al.</em> 2007</a>). Frequencies display strong population differentiation, with the derived light skin pigmentation allele (<em>A111T</em>) fixed or nearly so in all European populations and the ancestral allele predominant in sub-Saharan Africa and East Asia (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Lamason <em>et al.</em> 2005</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Norton <em>et al.</em> 2007</a>). The genomic region of diminished sequence variation in Europeans spans ~150 kb (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Lamason <em>et al.</em> 2005</a>). To learn about the time and location of origin of the <em>A111T</em> mutation, we studied haplotypes in the region around <em>SLC24A5</em> across world populations.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-5">Population-specific frequency data used for <em>A111T</em>, shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS1.pdf">Table S1</a>, were obtained from the following sources: HapMap populations (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Altshuler <em>et al.</em> 2010</a>) (<a href="http://hapmap.ncbi.nlm.nih.gov/">http://hapmap.ncbi.nlm.nih.gov/</a>); Human Genome Diversity Project (HGDP) populations (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Norton <em>et al.</em> 2007</a>) or the CEPH database (<a href="http://www.cephb.fr/en/cephdb/">http://www.cephb.fr/en/cephdb/</a>); populations from Sri Lanka (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Soejima and Koda 2007</a>); ALFRED (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cheung <em>et al.</em> 2000</a>) (alfred.med.yale.edu/alfred/); Indian samples (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Indian Genome Variation Consortium 2008</a>) (<a href="http://igvdb.res.in">http://igvdb.res.in</a>); and additional Mediterranean and South Indian samples (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Behar <em>et al.</em> 2010</a>). HapMap populations consist of CEU (CEPH individuals of northern and western European descent, sampled in Utah), TSI (Tuscan, Italy), GIH (Gujarati, sampled in Houston), CHB (Chinese, Beijing), CHD (Chinese descent, sampled in Denver), JPT (Japanese, Tokyo), YRI (Yoruba, Ibadan, Nigeria), LWK (Luhya, Webuye, Kenya), MKK (Maasai, Kinyawa, Kenya), ASW (African-Americans from southwest United States), and MEX (Mexican, sampled in Los Angeles).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="World distribution of A111T polymorphism in SLC24A5. The origins of sampled populations are indicated (+). Contours of global frequencies of SLC24A5A111T are shaded according to the frequency/color scale to the right. Frequency data by population that were used for this figure are tabulated in Table S1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2035880792" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;World distribution of &lt;em&gt;A111T&lt;/em&gt; polymorphism in &lt;em&gt;SLC24A5&lt;/em&gt;. The origins of sampled populations are indicated (+). Contours of global frequencies of &lt;em&gt;SLC24A5&lt;sup&gt;A111T&lt;/sup&gt;&lt;/em&gt; are shaded according to the frequency/color scale to the right. Frequency data by population that were used for this figure are tabulated in Table S1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2059/F1.medium.gif" width="440" height="161"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/11/2059/F1.medium.gif" width="440" height="161"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463570" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-6" class="first-child">World distribution of <em>A111T</em> polymorphism in <em>SLC24A5</em>. The origins of sampled populations are indicated (+). Contours of global frequencies of <em>SLC24A5<sup>A111T</sup></em> are shaded according to the frequency/color scale to the right. Frequency data by population that were used for this figure are tabulated in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS1.pdf">Table S1</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">Haplotypes were derived from several datasets. Haplotypes initially were classified by use of the polymorphism data of HapMap phase 3 (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Altshuler <em>et al.</em> 2010</a>). Denser data such as those of the 1000 Genomes Project phase 1 data (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium 2012</a>) were used to define phylogenetic relationships between haplotypes. Clarity also was gained by considering haplotypes within blocks of linkage disequilibrium, designated A, B, C, and D, and naming longer haplotypes as combinations of these local haplotypes. Blocks A, B, and C+D are delimited by locations of recombination hotspots (estimated rates &gt;5 cM/Mb) deduced from HapMap phase 2 data (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">International HapMap Consortium <em>et al.</em> 2007</a>), whereas the boundary between blocks C and D was defined by an interval with modestly elevated recombination (~0.8 cM/Mb). Precise boundaries chosen were nt 48321236−48370103 for A, 48370104−48390052 for B, 48390053−48468019 for C, and 48468020−48517471 for D, using b37/hg19 coordinates.</p><p id="p-8">Initial analysis of regions A and B used only single-nucleotide polymorphisms (SNPs) from HapMap Phase 3, Release 27 (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Altshuler <em>et al.</em> 2010</a>) that had been genotyped and phased in each included population (11 and 6 SNPs, respectively), whereas for regions C and D, SNPs lacking data for the TSI and/or MEX samples also were included (16 and 8 SNPs, respectively); these SNPs are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS2.pdf">Table S2</a>. Phased haplotypes were retrieved from HapMap and compared with raw genotype data for consistency. For the core region C, the absence of genotypes for SNPs c1 and c13 prevented distinguishing between members of the C2/C3 haplotype pair (in TSI) and the C6/C7 pair (in TSI and MEX), respectively. For TSI and GIH, which were phased on the basis of CEU, a small number of individuals heterozygous for the ancestral allele of <em>A111T</em> (1/1 and 3/8, respectively) were misphased. Corrected haplotype assignments are shown in all figures and tables except <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> and those in which haplotype combinations are displayed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FileS2.txt">File S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FileS3.txt">File S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FileS4.txt">File S4</a>). For subregion D, only 1 of 8 SNPs was genotyped in TSI, precluding haplotype assignment for this sample.</p><p id="p-9">A more complete picture of variation was obtained by examining phased haplotype information from the 1000 Genomes Project (<a href="http://mathgen.stats.ox.ac.uk/impute/ALL_1000G_phase1integrated_v3_impute.tgz">http://mathgen.stats.ox.ac.uk/impute/ALL_1000G_phase1integrated_v3_impute.tgz</a>; phase 1 version 3 of March 2012). The core region encompassed 767 polymorphisms in the core region, consisting of 23 biallelic indels and 744 SNPs. Because phasing becomes less reliable as allele frequency decreases and is undetermined for variants observed only once (287), analysis focused on the 156 polymorphisms with minor allele frequencies of 1% or greater, corresponding to 291 distinguishable haplotypes. A second dense set of SNPs also was compared with HapMap phase 2 data. After removal of monomorphic positions, 84 SNPs remained, 18 of which were found only in rare haplotypes. Phased haplotypes were retrieved from HapMap, Release 21. For phylogenetic analysis, graphs were drawn by the use of a simple nearest-neighbor approach and rooted by the use of ancestral alleles determined by comparison with other primate sequences. The few recombination events were identified by inspection, appearing as polymorphisms with transitions on more than one edge.</p><p id="p-10">For HGDP sample haplotypes, the data of <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Li <em>et al.</em> (2008)</a> were used. The genotype data contain 13 SNPs in region C that include 8 SNPs shared with HapMap Phase 3 plus 5 additional SNPs listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS2.pdf">Table S2</a>. Phased haplotypes were estimated using Phase 2.1.1 (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Stephens <em>et al.</em> 2001</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Stephens and Donnelly 2003</a>). In this dataset, all haplotypes were distinguishable except C6/C7 and C9/C10, due to the absence of data for SNP c13 for the former and c2 or c12 for the latter. Although the dataset lacks genotypes for SNP c11 (rs1426654) (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Li <em>et al.</em> 2008</a>), two independently obtained genotypes (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Norton <em>et al.</em> 2007</a>) and <a href="http://www.cephb.fr/en/hgdp/main.php">http://www.cephb.fr/en/hgdp/main.php</a> (laboratory HDCEPH68) were available. These were generally consistent with the deduced haplotypes (~1% mismatch) and were used to distinguish two haplotype pairs that were not otherwise distinguishable (C2 and C11 <em>vs.</em> C3 and C9/C10). Similar procedures were used to analyze haplotypes for seven Middle Eastern, North African, East African, and South Indian samples by the use of the data from <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Behar <em>et al.</em> (2010)</a>. The genotype data contain the same set of SNPs available for the HGDP, plus SNP c11 and rs4775737.</p><p id="p-11">Because haplotypes were derived from a variety of datasets, we tested both SNPs and deduced haplotypes for consistency. In the genomic region analyzed, we found no evidence of errors in allele designation. No source reported common haplotypes that did not correspond to those found in other datasets. All common core haplotypes in HapMap Phase 3 samples were represented by unambiguously phased examples. For a small number of individuals, HapMap and 1000 Genomes analyses yielded divergent phasing across the B-C region boundary. We have avoided drawing conclusions on the basis of rare haplotypes that could be artifactual products of genotyping or phasing errors.</p><p id="p-12">To estimate the age of the <em>A111T</em> mutation, we used a molecular clock approach. We first determined the rate of mutation in the combined C and D regions from number of differences between human and chimpanzee reference sequences. In this alignment, nt changes within 4 nt of gaps were excluded to remove potential biases caused by misalignment. For calibration, we used 6 million years, the midpoint of the range of estimates (5−7 million years) for the divergence time between human and chimpanzee as identified by <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Kumar <em>et al.</em> (2005)</a> and assumed equal mutation rates (per year) in the human and chimpanzee lineages. We then counted the number of single-nucleotide differences from the modal haplotype for each C11-D4−containing chromosome in the 1000 Genomes dataset by using all reported variants. Each chromosome provides an imprecise estimate of the time since the origin of the haplotype; values were averaged over individual populations, or the entire sample. Because accumulation of mutations in a single lineage is independent of population size, this procedure does not require demographic assumptions or data. Although <em>A111T</em> is subject to selection, we assume only that subsequent mutations are neutral. Our approach to dating selective sweeps differs from that used by <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Rozas <em>et al.</em> (2001)</a> and <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Meiklejohn <em>et al.</em> (2004)</a>, which counts the number of affected sites and underestimates the coalescence time unless each sampled lineage is independent. In contrast, our estimate is unbiased in the presence of nonindependent samples. Because the most frequent C11 + D4 variant carrying an additional mutation occurs 36 times in 1013 chromosomes, the independence condition is clearly not met.</p><p id="p-13">To estimate confidence limits, we considered the effective sample size to be the number of chromosomes that could be sampled without violating the assumption of independence. This was determined by resampling, with replacement, until a variant haplotype was duplicated. The median counts without repeat in 10,000 replicates ranged from 9 to 14 for individual population subsamples, to 19 for the combined sample, substantially lower than the total number of chromosomes sequenced. These values, multiplied by the observed mutation frequencies, were then used to calculate confidence limits under the assumption that mutations follow a Poisson distribution.</p><p id="p-14">The aforementioned approach is most applicable when all variants in the chromosomal region under study have been determined. At the current sequencing depth in 1000 Genomes data (5.1-fold across autosomes), we expect some rare variants to be undetected, resulting in an underestimate of the age of <em>A111T</em>. We made an approximate correction for the underestimate by inverting the estimated power relationship (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium 2012</a>). In particular, the corrected count is given by<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2059/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2059/embed/mml-math-1.gif"/></noscript></span></span>where <em>A<sub>i</sub></em> is the observed number of sites at which variants are reported to occur <em>i</em> times in the sample and <em>P<sub>i</sub></em> is the power to detect variant sites with <em>i</em> occurrences. This calculation ignores the effects of miscounts (<em>e.g.</em>, a variant occurring three times is detected but inaccurately reported as having two or four occurrences), and assumes that the genome-wide power applies to the genomic region under investigation. Because the largest contribution is undetected singletons (87% of the estimated excess), the assumption that miscounts contribute little to the outcome is reasonable. Applying this procedure using the SNP frequency spectrum for C11-D4 across all samples suggests that the reported age range should be increased by a factor of approximately 1.58.</p><p id="p-15">Comparing different population subsamples, we noted a trend toward greater counts in the admixed New World populations. Misassignment of rare or unique variants in individuals heterozygous for <em>A111T</em> (common in the admixed, but rare in the European samples) was a potential explanation of this trend, but the absence of any difference in variant frequency between heterozygotes and homozygotes in pooled Puerto Rican/ Colombian/Mexican samples suggests that this was not a significant source of bias.</p></div><div class="section results-discussion" id="sec-2"><h2 class="">Results and Discussion</h2><div id="sec-3" class="subsection"><h3>Global distribution of the <em>A111T</em> mutation in <em>SLC24A5</em></h3><p id="p-16">The geographical distribution of the <em>A111T</em> allele of <em>SLC24A5</em> (<a id="xref-ref-18-4" class="xref-bibr" href="#ref-18">Norton <em>et al.</em> 2007</a>), updated with the use of additional population samples (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>), shows that <em>A111T</em> is nearly fixed in all of Europe and most of the Middle East, extending east to some populations in present-day Pakistan and north India. <em>A111T</em> shows a latitudinal decline toward the Equator, with high frequencies in Northern Africa (&gt;0.80), intermediate (0.4−0.6) in Ethiopia and Somalia, and lower (&lt;0.35) in sub-Saharan Africa. This pattern is broadly consistent with strong positive selection for decreased skin pigmentation throughout Europe. There is a cline of decreasing frequency of <em>A111T</em> in indigenous populations east of approximately longitude 75° in Central Asia, with near-absence in East Asia, Oceania, and the Americas. The extent to which the spread of <em>A111T</em> to the east has been inhibited by the absence of substantial eastward population migrations postdating its origin or by the presence of other loci responsible for decreased skin pigmentation in East Asia is presently unclear.</p></div><div id="sec-4" class="subsection"><h3>Characterization of haplotypes in the genomic region encompassing <em>SLC24A5</em></h3><p id="p-17">Diminished variation in the genomic region around <em>SLC24A5</em> in the HapMap CEU (European ancestry) sample led us to ask what the haplotypes associated with the <em>A111T</em> allele looked like, and how, when, and where they might have arisen. We therefore investigated haplotypes spanning this genomic region (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The haplotypes are described in the context of four contiguous subregions defined by blocks of linkage disequilibrium, here designated A (49 kb), B (20 kb), C (78 kb), and D (49 kb) (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Blocks B, C, and D together encompass the region of diminished variation in CEU. Analysis of the core subregion C, which includes <em>SLC24A5</em>, yielded 46 haplotypes in HapMap Phase 3 populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS3.pdf">Table S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS4.pdf">Table S4</a>). The 11 haplotypes with individual abundances &gt;0.5%, which we designate C1 through C11, collectively comprise 93–98% of the total in each population (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). A single haplotype, C11, accounts for 97% of all instances of the <em>A111T</em> variant of <em>SLC24A5</em>. Most of the haplotypes with frequencies &lt;0.5% appear to be products of recombination between more frequent haplotypes. Analysis of common haplotypes found in HGDP and other samples (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Li <em>et al.</em> 2008</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Behar <em>et al.</em> 2010</a>) yielded results matching those derived from HapMap samples (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS5.pdf">Table S5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS6.pdf">Table S6</a>), including the equivalence between haplotype C11 and the derived allele of rs1426654. This finding is consistent with a common origin for <em>A111T</em> worldwide. Analysis of data from the 1000 Genomes Project indicated that haplotypes defined on the basis of 16 SNPs corresponded to sets of closely related haplotypes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FileS1.txt">File S1</a>). Common haplotypes defined using 1000 Genomes data differed from the ancestral state at 27−48 positions. The number of haplotypes detected depends on the number of polymorphisms used to define them; with inclusion of lower frequency variants, an increasing fraction of chromosomes corresponds to rare haplotypes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS7.pdf">Table S7</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genomic region surrounding SLC24A5. The first three panels show SNPs (red symbols, SNPs used for haplotype analysis in HapMap Phase 3, Release 27, also listed in Table S2; black symbols, additional SNPs genotyped in fewer HapMap populations, or flanking SNPs not analyzed here). SNPs monomorphic in the original four HapMap population samples (CEU, CHB, JPT, YRI) are omitted. Top, SNP heterozygosity in CEU, calculated from allele frequencies, illustrating the region of diminished variation in this European sample. Middle, SNP allele frequency difference between CEU and YRI, showing several SNPs with extremely high frequency differentiation (&#x394;f &#x2265; 0.8); rs1426654 is circled. Beneath is a recombination map showing boundaries of blocks used for haplotype analysis (A through D). Region C, as described in text, is the core region containing SLC24A5. Bottom, positions of SLC24A5 and other genes in region, with common coordinates (NCBI build B36) used for all panels. The centromere is to left." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2035880792" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genomic region surrounding &lt;em&gt;SLC24A5&lt;/em&gt;. The first three panels show SNPs (red symbols, SNPs used for haplotype analysis in HapMap Phase 3, Release 27, also listed in Table S2; black symbols, additional SNPs genotyped in fewer HapMap populations, or flanking SNPs not analyzed here). SNPs monomorphic in the original four HapMap population samples (CEU, CHB, JPT, YRI) are omitted. Top, SNP heterozygosity in CEU, calculated from allele frequencies, illustrating the region of diminished variation in this European sample. Middle, SNP allele frequency difference between CEU and YRI, showing several SNPs with extremely high frequency differentiation (&#x394;f &#x2265; 0.8); rs1426654 is circled. Beneath is a recombination map showing boundaries of blocks used for haplotype analysis (A through D). Region C, as described in text, is the core region containing &lt;em&gt;SLC24A5&lt;/em&gt;. Bottom, positions of &lt;em&gt;SLC24A5&lt;/em&gt; and other genes in region, with common coordinates (NCBI build B36) used for all panels. The centromere is to left.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2059/F2.medium.gif" width="440" height="258"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/11/2059/F2.medium.gif" width="440" height="258"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463632" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-18" class="first-child">Genomic region surrounding <em>SLC24A5</em>. The first three panels show SNPs (red symbols, SNPs used for haplotype analysis in HapMap Phase 3, Release 27, also listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS2.pdf">Table S2</a>; black symbols, additional SNPs genotyped in fewer HapMap populations, or flanking SNPs not analyzed here). SNPs monomorphic in the original four HapMap population samples (CEU, CHB, JPT, YRI) are omitted. Top, SNP heterozygosity in CEU, calculated from allele frequencies, illustrating the region of diminished variation in this European sample. Middle, SNP allele frequency difference between CEU and YRI, showing several SNPs with extremely high frequency differentiation (Δf ≥ 0.8); rs1426654 is circled. Beneath is a recombination map showing boundaries of blocks used for haplotype analysis (A through D). Region C, as described in text, is the core region containing <em>SLC24A5</em>. Bottom, positions of <em>SLC24A5</em> and other genes in region, with common coordinates (NCBI build B36) used for all panels. The centromere is to left.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Core-region haplotypes defined using 16 SNPs. (A) Diagram showing relationships between common haplotypes C1 through C11. SNPs that differ between adjacent haplotypes are labeled by nickname (SNP c1 through c16, keyed to Table S2). Haplotype C1 contains ancestral alleles for each SNP in this series. Haplotype C11 carries the derived Thr-111 allele of SLC24A5 (rs1426654, SNP c11). The only duplicated transition in this diagram is for SNP c1 (rs1834640). Note that analysis of additional SNPs reveals cases in which adjacent haplotypes in the diagram do not have ancestor-descendent relationships (compare Figure 4). (B) Histograms showing the distribution of haplotypes in several HapMap populations. The three East Asian populations are combined, as are CEU and TSI. Indeterminate C2/C3 in TSI indicated. MEX and ASW are not depicted. The frequency scale is split at 0.2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2035880792" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Core-region haplotypes defined using 16 SNPs. (A) Diagram showing relationships between common haplotypes C1 through C11. SNPs that differ between adjacent haplotypes are labeled by nickname (SNP c1 through c16, keyed to Table S2). Haplotype C1 contains ancestral alleles for each SNP in this series. Haplotype C11 carries the derived Thr-111 allele of &lt;em&gt;SLC24A5&lt;/em&gt; (rs1426654, SNP c11). The only duplicated transition in this diagram is for SNP c1 (rs1834640). Note that analysis of additional SNPs reveals cases in which adjacent haplotypes in the diagram do not have ancestor-descendent relationships (compare Figure 4). (B) Histograms showing the distribution of haplotypes in several HapMap populations. The three East Asian populations are combined, as are CEU and TSI. Indeterminate C2/C3 in TSI indicated. MEX and ASW are not depicted. The frequency scale is split at 0.2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2059/F3.medium.gif" width="163" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/11/2059/F3.medium.gif" width="163" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463483" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-19" class="first-child">Core-region haplotypes defined using 16 SNPs. (A) Diagram showing relationships between common haplotypes C1 through C11. SNPs that differ between adjacent haplotypes are labeled by nickname (SNP c1 through c16, keyed to <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS2.pdf">Table S2</a>). Haplotype C1 contains ancestral alleles for each SNP in this series. Haplotype C11 carries the derived Thr-111 allele of <em>SLC24A5</em> (rs1426654, SNP c11). The only duplicated transition in this diagram is for SNP c1 (rs1834640). Note that analysis of additional SNPs reveals cases in which adjacent haplotypes in the diagram do not have ancestor-descendent relationships (compare <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). (B) Histograms showing the distribution of haplotypes in several HapMap populations. The three East Asian populations are combined, as are CEU and TSI. Indeterminate C2/C3 in TSI indicated. MEX and ASW are not depicted. The frequency scale is split at 0.2.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463554/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463554/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Description of common core-region haplotypes defined using 16 SNPs</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463582/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463582/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Distribution of common core-region haplotypes in HapMap populations</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic relationships among core haplotype clusters. Haplotypes were deduced from 1000 Genomes data by the use of 156 polymorphisms with frequency &#x2265;1%. (A) Haplotypes having frequencies &#x2265;0.5% (representing 78% of total) are depicted. Complete dataset is found in File S1. Sets of haplotypes corresponding to those deduced using 16 SNPs are enclosed within red boxes and labeled C1 through C11. C8, which constituted " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2035880792" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Phylogenetic relationships among core haplotype clusters. Haplotypes were deduced from 1000 Genomes data by the use of 156 polymorphisms with frequency &#x2265;1%. (A) Haplotypes having frequencies &#x2265;0.5% (representing 78% of total) are depicted. Complete dataset is found in File S1. Sets of haplotypes corresponding to those deduced using 16 SNPs are enclosed within red boxes and labeled C1 through C11. C8, which constituted &lt;0.5%, is not shown. Two variants of C3 differ at an indel that shows evidence of recurrent mutation and for which the ancestral state is unknown. Values within each circle indicate number of occurrences. Gray and yellow shading indicates haplotypes that are predominantly African or East Asian (in this dataset), respectively. Numbers of polymorphic differences on each branch are indicated if greater than one. Inclusion of lower frequency polymorphisms would raise the number of C1-specific variants to 26, whereas C4-specific variants would increase to 11. Early haplotype lineage divergence, depicted within the dashed circle, provides evidence for multiple recombination events that are not resolved in this phylogeny. The numbers within the dashed circle represent number of polymorphic differences from the ancestral state; this involves 31 polymorphisms that have derived alleles shared by more than one lineage. (B) Relationships among early diverging branches supported by 21 polymorphisms. (C) Alternative relationships among early diverging branches supported by 5 polymorphisms. The ancestral state is represented by the open circle in B and C.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2059/F4.medium.gif" width="440" height="335"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/11/2059/F4.medium.gif" width="440" height="335"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463563" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-24" class="first-child">Phylogenetic relationships among core haplotype clusters. Haplotypes were deduced from 1000 Genomes data by the use of 156 polymorphisms with frequency ≥1%. (A) Haplotypes having frequencies ≥0.5% (representing 78% of total) are depicted. Complete dataset is found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FileS1.txt">File S1</a>. Sets of haplotypes corresponding to those deduced using 16 SNPs are enclosed within red boxes and labeled C1 through C11. C8, which constituted &lt;0.5%, is not shown. Two variants of C3 differ at an indel that shows evidence of recurrent mutation and for which the ancestral state is unknown. Values within each circle indicate number of occurrences. Gray and yellow shading indicates haplotypes that are predominantly African or East Asian (in this dataset), respectively. Numbers of polymorphic differences on each branch are indicated if greater than one. Inclusion of lower frequency polymorphisms would raise the number of C1-specific variants to 26, whereas C4-specific variants would increase to 11. Early haplotype lineage divergence, depicted within the dashed circle, provides evidence for multiple recombination events that are not resolved in this phylogeny. The numbers within the dashed circle represent number of polymorphic differences from the ancestral state; this involves 31 polymorphisms that have derived alleles shared by more than one lineage. (B) Relationships among early diverging branches supported by 21 polymorphisms. (C) Alternative relationships among early diverging branches supported by 5 polymorphisms. The ancestral state is represented by the open circle in B and C.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Phylogenetic relationships between core haplotypes</h3><p id="p-25">Phylogenetic relationships between haplotypes determined using 1000 Genomes data (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>) were equivalent to those deduced from HapMap phase 2 data. The branches comprising C1, C2, C3, C4, and C5-C11 are early diverging clades. The most abundant and extensive lineage includes three branches: C5, C6-C7, and C9-C11. Within the C9-C11 branch, the C9 cluster is ancestral to C10, but nearly all instances of C9 include additional polymorphisms that distinguish them from common ancestors with C10. In contrast, the most commonly observed C10 variant is ancestral to C11.</p></div><div id="sec-6" class="subsection"><h3>Identification of a 147-kb founder haplotype containing <em>A111T</em></h3><p id="p-26">The region of diminished variation in Europeans includes subregions B and D, flanking the core region (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). B- and D-region haplotypes are summarized in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FigureS1.pdf">Figure S1</a>, A and B and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS8.pdf">Table S8</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS9.pdf">Table S9</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS10.pdf">Table S10</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS11.pdf">Table S11</a>. It is apparent that a variety of combinations of B- and C-region haplotypes arose by recombination (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FileS2.txt">File S2</a>). The smaller number of combinations of C- and D-region haplotypes observed (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FigureS2.pdf">Figure S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FileS3.txt">File S3</a>) is consistent with the occurrence of fewer crossovers near this location, including one ancestral to C10, C11, and a subset of C9. The B- and D-subregion haplotypes associated most strongly with haplotype C11 are B6 (94%) and D4 (&gt;99%), respectively (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FigureS2.pdf">Figure S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FileS2.txt">File S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FileS3.txt">File S3</a>). This pattern also holds in HapMap populations in which C11 is far from fixation and in which a diversity of other B- and D-region haplotypes not associated with C11 are found, including MKK. Interestingly, the greatest diversity of B-region haplotypes associated with C11 is found in GIH (89% B6). Taken together, these results establish that the 147-kb founder haplotype containing <em>A111T</em> was B6 + C11 + D4.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Relationships between local haplotypes in B and C regions. For each HapMap population, the distribution of haplotype combinations is shown as a heat map (scale on right). Recurrent recombination between core- and B-region is apparent. Predominant association of C11 with B6 contrasts with associations of C10 (B2) and C9 (B2, B7). Counts are shown in File S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2035880792" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Relationships between local haplotypes in B and C regions. For each HapMap population, the distribution of haplotype combinations is shown as a heat map (scale on right). Recurrent recombination between core- and B-region is apparent. Predominant association of C11 with B6 contrasts with associations of C10 (B2) and C9 (B2, B7). Counts are shown in File S2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2059/F5.medium.gif" width="440" height="314"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/11/2059/F5.medium.gif" width="440" height="314"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463519" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-27" class="first-child">Relationships between local haplotypes in B and C regions. For each HapMap population, the distribution of haplotype combinations is shown as a heat map (scale on right). Recurrent recombination between core- and B-region is apparent. Predominant association of C11 with B6 contrasts with associations of C10 (B2) and C9 (B2, B7). Counts are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FileS2.txt">File S2</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">Subregion A, to the left of subregion B, lies outside the region of diminished variation around <em>SLC24A5</em> observed in Europeans (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Lamason <em>et al.</em> 2005</a>), indicating that recombination occurred between the A and B regions after the origin of <em>A111T</em> but before its fixation. An analysis of the A subregion is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FigureS1.pdf">Figure S1</a>C, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS12.pdf">Table S12</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS13.pdf">Table S13</a>. Two haplotypes, A1 and A5, predominate (together totaling 89–99%) in association with C11 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FileS4.txt">File S4</a>). The relative proportions of A1 and A5 associated with B6 + C11 vary considerably among populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS14.pdf">Table S14</a>), presumably a result of genetic drift. The data do not allow a determination of whether A1 or A5 was a part of the founding <em>A111T</em> haplotype.</p></div><div id="sec-7" class="subsection"><h3>Recombination was involved in the creation of C11</h3><p id="p-29">Haplotype C3 and C11 share the derived allele of SNP c1 (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>), suggesting the possibility of recombination. To test this possibility, we examined SNPs not genotyped in HapMap Phase 3. Haplotype C11 carries ancestral alleles of SNPs rs12441154 and rs57108441, whereas C10 carries the derived alleles, a pattern readily explained by a single crossover between C3 and C10 (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). In support of this notion, the B-region haplotype found associated with C11, B6, is also the one most commonly associated with C3; conversely 96% of C10 haplotypes are associated with B region haplotypes other than B6 (68% with B2; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FileS2.txt">File S2</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Generation of haplotype C11. Recombination between chromosomes containing C10 (white) and C3 (shaded) is indicated by X. The crossover product represented by the solid line is the precursor to C11; the reciprocal recombinant represented by the dotted line is not recovered. Ancestral alleles are represented by open symbols, and derived alleles are solid black, where rs12441154, rs1834640 (c1), rs2675345 (c2), rs57108441, and rs1426654 (c11) are represented by the triangle, square, triangle, circle, and star, respectively. The B6 haplotype (here defined by 6 SNPs), just to the left of the C region, is that most commonly found in phase with the C3 allele, as indicated; B2 and B3 are most commonly in phase with C10. Subsequent mutation at rs1426654, represented by the black star, produced the predominant SLC24A5A111T-containing haplotype, C11, which is globally associated with B6. 1000 Genomes Project data localize the crossover to the 0.9-kb interval between rs57108441 and rs78729596. Flanking regions are not shown. Note that the final product is identical under an alternate model in which mutation precedes recombination." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2035880792" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Generation of haplotype C11. Recombination between chromosomes containing C10 (white) and C3 (shaded) is indicated by X. The crossover product represented by the solid line is the precursor to C11; the reciprocal recombinant represented by the dotted line is not recovered. Ancestral alleles are represented by open symbols, and derived alleles are solid black, where rs12441154, rs1834640 (c1), rs2675345 (c2), rs57108441, and rs1426654 (c11) are represented by the triangle, square, triangle, circle, and star, respectively. The B6 haplotype (here defined by 6 SNPs), just to the left of the C region, is that most commonly found in phase with the C3 allele, as indicated; B2 and B3 are most commonly in phase with C10. Subsequent mutation at rs1426654, represented by the black star, produced the predominant &lt;em&gt;SLC24A5&lt;sup&gt;A111T&lt;/sup&gt;&lt;/em&gt;-containing haplotype, C11, which is globally associated with B6. 1000 Genomes Project data localize the crossover to the 0.9-kb interval between rs57108441 and rs78729596. Flanking regions are not shown. Note that the final product is identical under an alternate model in which mutation precedes recombination.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2059/F6.medium.gif" width="440" height="394"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/3/11/2059/F6.medium.gif" width="440" height="394"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2059/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463526" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-30" class="first-child">Generation of haplotype C11. Recombination between chromosomes containing C10 (white) and C3 (shaded) is indicated by X. The crossover product represented by the solid line is the precursor to C11; the reciprocal recombinant represented by the dotted line is not recovered. Ancestral alleles are represented by open symbols, and derived alleles are solid black, where rs12441154, rs1834640 (c1), rs2675345 (c2), rs57108441, and rs1426654 (c11) are represented by the triangle, square, triangle, circle, and star, respectively. The B6 haplotype (here defined by 6 SNPs), just to the left of the C region, is that most commonly found in phase with the C3 allele, as indicated; B2 and B3 are most commonly in phase with C10. Subsequent mutation at rs1426654, represented by the black star, produced the predominant <em>SLC24A5<sup>A111T</sup></em>-containing haplotype, C11, which is globally associated with B6. 1000 Genomes Project data localize the crossover to the 0.9-kb interval between rs57108441 and rs78729596. Flanking regions are not shown. Note that the final product is identical under an alternate model in which mutation precedes recombination.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">Can we determine whether recombination involving C3 preceded or followed the mutation that created <em>A111T</em>? Models in which the recombination or mutation occurred first produce the same end product but proceed through different intermediates, corresponding to C26 or C22, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS3.pdf">Table S3</a>). Rare haplotypes matching both potential C11 precursors were found. Because either could have been produced by recombination subsequent to the origin of C11, their occasional occurrence is not informative. However, an evolutionary argument strongly suggests an order of events (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FigureS4.pdf">Figure S4</a>). If the crossover predated the mutation, the predicted intermediate (C26) would not have experienced positive selection on the basis of lighter pigmentation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FigureS4.pdf">Figure S4</a>A). Selection for decreased skin pigmentation would cause the predominant haplotype containing <em>A111T</em> to be C11, as is observed. On the other hand, if the <em>A111T</em> mutation preceded the crossover, the intermediate haplotype (C22) would be predicted to experience the <em>same</em> selective pressure as C11 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FigureS4.pdf">Figure S4</a>B). Because C11 is derived by recombination between C3 and C22 in this model, C22 would be expected to predominate over C11, unless C11 had a selective advantage over C22. This outcome is not what is observed. Rather, the frequency of C22 is only approximately 1% that of C11. Furthermore, association with diverse B-region haplotypes rather than one makes it most likely that the existing instances of C22 are the result of recombination after the formation of C11 rather than relicts of a precursor to C11. We conclude that the crossover most likely preceded the <em>A111T</em> mutation.</p></div><div id="sec-8" class="subsection"><h3>Phylogeographic analysis of <em>SLC24A5</em> haplotype distributions</h3><p id="p-32">The world distribution of core region haplotypes, together with their phylogenetic relationships, suggests which haplotypes likely originated in Africa and which most likely arose outside of Africa. As expected from the near fixation of <em>A111T</em> in Europe, the C11 clade predominates there, and all other haplotypes are rare. Of the remaining 10 common core haplotype groups, all ancestral at <em>rs1426654</em>, eight clearly have their origins in Africa (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>, <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS4.pdf">Table S4</a>). Three early diverging haplotypes, C1, C2, and C4, are rare outside of Africa and clearly originated there. In the lineage containing the majority of haplotypes, each of the three branches, containing C5, C6-C7, and C8-C11, give strong evidence of having originated in Africa. C5 reaches its greatest abundance in West Africa and is rare outside of Africa. Within the other two branches, C6 and C9, which are the most common haplotypes in Africa, are also common worldwide, whereas C7 is abundant in East Asia and much less common but widespread in Africa. Consideration of the relationships among haplotype variants (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>) indicates that C6, C7, and C9 (but not C8) dispersed out of Africa and have diverse descendants present and originating in East Asia. Among these descendants is C10, which is abundant in East Asia (and the New World) but extremely rare in Africa (0.5% in LWK). Haplotype C3 represents the final early diverging lineage (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>). Although the lineage containing this haplotype must have originated in Africa, C3 is rare in Africa (1.0% in MKK) but widely distributed in East Asia, the New World, and Oceania. The distributions of C3 and C10 are most consistent with origin outside of Africa and subsequent introduction into Africa by migrations such as those documented by uniparental markers (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Richards <em>et al.</em> 2006</a>).</p></div><div id="sec-9" class="subsection"><h3>Can we date the <em>A111T</em> mutation?</h3><p id="p-33">The preceding analysis is consistent with a wide range of possible dates for the origin of <em>A111T</em>, including the period before the initial colonization of Europe by anatomically modern humans &gt;40 thousand years ago (kya) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Mellars 2006</a>). An estimate for the date of origin of <em>A111T</em> based on microsatellites (Beleza <em>et al.</em> 2012) places the origin at 19 kya (95% confidence interval 6−38 kya), for a dominant model, or 11 kya (95% confidence interval 1−56 kya), for a more plausible additive model. To create an independent estimate, we applied a molecular clock approach to 1000 Genomes data by using the combined C and D subregions. Because proportions of different classes of nucleotide substitutions in the C11 + D4 variants and in the human-chimpanzee alignment are not significantly different (χ<sup>2</sup> = 4.42, df = 5, <em>P</em> = 0.49; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS15.pdf">Table S15</a>), we combined these classes for analysis. For the combined population samples, before making corrections for undercounts in the source data, we obtained an estimate of 7.8 kya for the most recent common ancestor of the C11 + D4 haplotype combination (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Corresponding 95% confidence limits are 4.8−12.2 kya, whereas uncorrected estimates derived from individual European samples or the combined New World samples (also of European origin) ranged from 5.2 to 10.4 kya (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). These values are clearly underestimates as a result of low sequence depth (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">1000 Genomes Project Consortium 2012</a>). Adjustment for undercounting is substantial, increasing the estimated age for the combined samples to 12.4 (95% confidence interval 7.6−19.2) kya. If mutation rates in recent humans are lower than predicted from the human-chimpanzee divergence (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Scally and Durbin 2012</a>), true ages will be even older. Our adjusted dates overlap those previously reported (Beleza <em>et al.</em> 2012) and are also consistent with the lower limit for the origin of <em>A111T</em> set by the finding that the Alpine “iceman” dated to 5.3 kya was homozygous for this variant (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Keller <em>et al.</em> 2012</a>). This date range implies an origin clearly preceding the Neolithic transition in Europe. These dates are later than the initial colonization of Europe but are consistent with an <em>A111T</em> origin before or after post-glacial population expansions.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463553/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463553/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title"><em>A111T</em> date estimation using 1000 Genomes Project data</span><div class="sb-div caption-clear"></div></div></div><p id="p-36">Obtaining a better date for the origin of the <em>A111T</em> mutation is challenged by a number of issues. Our approach provides a date for the common ancestor of the sampled C11-D4−containing chromosomes. This common ancestor may be significantly younger than the origin of <em>A111T</em> if positive selection was initially weak or nonexistent, or if there was a subsequent bottleneck. In addition, our date estimation relies on samples of predominantly European origin. Inclusion of Middle Eastern or South Asian examples would be expected to yield a more representative result. Incomplete detection of rare variants is a limitation that can be improved with higher-coverage sequencing. More direct limits on the age of <em>A111T</em> could result from genotyping of ancient human DNA.</p></div><div id="sec-10" class="subsection"><h3>Where did C11 originate?</h3><p id="p-37">The precursors to C11, haplotypes C3 and C10, are common in East Asia and the New World (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FigureS5.pdf">Figure S5</a>), but the distribution of C11 indicates that these locations are not likely sites for the origin of C11 or its immediate precursor. Similarly, B6 not associated with C11 is distributed widely in East Asia and the New World (data not shown). The paucity of C3 and C10 among existing African haplotypes suggests that both events leading to the origin of C11 took place outside this continent. Our dating for this haplotype is consistent with a non-African origin. The most likely location for the origin of C11 is, therefore, within the region in which it is fixed or nearly so. As both models for the origin of C11 imply that C3 and C10 were present in ancestors of Europeans, the observed and inferred distributions of these autosomal haplotypes are consistent with the single-out-of-Africa hypothesis derived using uniparental markers (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Oppenheimer 2003</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Macaulay <em>et al.</em> 2005</a>).</p><p id="p-38">The presence in Africa of <em>A111T</em> only in association with C11 indicates that the observed examples, like those of C3 and C10, resulted from introduction into the continent subsequent to origin. The low diversity of B-region haplotypes associated with C11 in MKK, equivalent to that seen in European samples (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/FileS2.txt">File S2</a>) supports this view because those individuals live among a majority population with high B-region diversity. Although too few African C11 sequences have been determined to draw strong conclusions, those available from the 1000 Genomes Project show no evidence of greater age in the form of greater SNP diversity than the European examples. It should be noted that the relatively high abundance of <em>A111T</em> in several equatorial East African samples indicates the absence of sustained strong negative selection against this allele at low latitudes.</p><p id="p-39">Although a non-African origin for C11 is clear, near fixation of this haplotype over a wide geographical region prevents strong inferences regarding a precise location of origin. Existing data are consistent with a model in which the C11 precursor did not extend outside the geographical region in which C11 is now nearly fixed, a conclusion subject to limited haplotype sampling in some neighboring regions, such as India. With sufficiently strong positive selection for C11, it is possible that this haplotype could have originated anywhere within its current range and spread via local migration. However, selection acting in concert with major population migrations would have facilitated a much more rapid dispersal. Archeological, mitochondrial, and Y-chromosomal data suggest involvement of multiple dispersals in shaping the current populations of Europe and the Middle East (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Soares <em>et al.</em> 2010</a>). Because <em>A111T</em> is far from fixation in most Indian samples (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1/TableS1.pdf">Table S1</a>), the high diversity of B-region haplotypes associated with C11 in the GIH sample may be the result of prolonged recombination rather than early arrival of <em>A111T</em>. In fact, the decrease in frequency of <em>A111T</em> to the east of Pakistan suggests that C11 originated farther to the west and after the initial genetic split between western and eastern Eurasians. On this basis, we hold the view that an origin of C11 in the Middle East, broadly defined, is most likely.</p><p id="p-40">Traditionally, uniparental markers have been used for the construction of phylogenetic trees due to the absence of recombination. Here, autosomal haplotypes and the characterization of recombination events have helped us to define the genotypic phylogeny of a genomic region known to have been subject to strong natural selection. Such an approach to studying the phylogeny of autosomal regions may also be useful for the study of other loci under selection in humans and in other organisms.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">We thank Kat Reid for helpful comments on the manuscript. This work was supported by National Institutes of Health grants R01 AR052535 and R24 RR017441 to K.C.C., UL1 TR000127 for the CTSA Bioinformatics Core of Pennsylvania State University, and the Jake Gittlen Cancer Research Foundation.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.007484/-/DC1</a></p></li><li class="fn" id="fn-8"><p id="p-42"><em>Communicating editor: S. A. Tishkoff</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-06-27"><span class="received-label">Received </span>June 27, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-09-11"><span class="accepted-label">Accepted </span>September 11, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Canfield <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.1" data-doi="10.1038/nature11632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">1000 Genomes Project Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An integrated map of genetic variation from 1,092 human genomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAbecasis%26rft.auinit1%253DG.%2BR.%26rft.volume%253D491%26rft.issue%253D7422%26rft.spage%253D56%26rft.epage%253D65%26rft.atitle%253DAn%2Bintegrated%2Bmap%2Bof%2Bgenetic%2Bvariation%2Bfrom%2B1%252C092%2Bhuman%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11632%26rft_id%253Dinfo%253Apmid%252F23128226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23128226&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310434500030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.2" data-doi="10.1038/nature09298"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibbs</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peltonen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dermitzakis</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffner</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Integrating common and rare genetic variation in diverse human populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">467</span>: <span class="cit-fpage">52</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAltshuler%26rft.auinit1%253DD.%2BM.%26rft.volume%253D467%26rft.issue%253D7311%26rft.spage%253D52%26rft.epage%253D58%26rft.atitle%253DIntegrating%2Bcommon%2Band%2Brare%2Bgenetic%2Bvariation%2Bin%2Bdiverse%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09298%26rft_id%253Dinfo%253Apmid%252F20811451%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09298&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20811451&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281461200033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.3" data-doi="10.1038/nature09103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Behar</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yunusbayev</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metspalu</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Metspalu</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosset</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome-wide structure of the Jewish people.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">466</span>: <span class="cit-fpage">238</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBehar%26rft.auinit1%253DD.%2BM.%26rft.volume%253D466%26rft.issue%253D7303%26rft.spage%253D238%26rft.epage%253D242%26rft.atitle%253DThe%2Bgenome-wide%2Bstructure%2Bof%2Bthe%2BJewish%2Bpeople.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09103%26rft_id%253Dinfo%253Apmid%252F20531471%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20531471&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279580800037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-3.11.2059.4" data-doi="10.1093/molbev/mss207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beleza</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">dos Santos</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McEvoy</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alves</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinho</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The timing of pigmentation lightening in Europeans.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">35</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmss207%26rft_id%253Dinfo%253Apmid%252F22923467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjMwLzEvMjQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osier</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">ALFRED: a Web-accessible allele frequency database.</span> <abbr class="cit-jnl-abbrev">Pac. Symp. Biocomput.</abbr> <span class="cit-fpage">639</span>–<span class="cit-lpage">650</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.6" data-doi="10.1126/science.1183863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grossman</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shylakhter</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karlsson</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrne</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morales</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A composite of multiple signals distinguishes causal variants in regions of positive selection.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">327</span>: <span class="cit-fpage">883</span>–<span class="cit-lpage">886</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1183863%26rft_id%253Dinfo%253Apmid%252F20056855%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjcvNTk2Ny84ODMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.7" data-doi="10.1007/s12041-008-0002-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Indian Genome Variation Consortium</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic landscape of the people of India: a canvas for disease gene exploration.</span> <abbr class="cit-jnl-abbrev">J. Genet.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bgenetics%26rft.stitle%253DJ%2BGenet%26rft.volume%253D87%26rft.issue%253D1%26rft.spage%253D3%26rft.epage%253D20%26rft.atitle%253DGenetic%2Blandscape%2Bof%2Bthe%2Bpeople%2Bof%2BIndia%253A%2Ba%2Bcanvas%2Bfor%2Bdisease%2Bgene%2Bexploration.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs12041-008-0002-x%26rft_id%253Dinfo%253Apmid%252F18560169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s12041-008-0002-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18560169&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255869300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.8" data-doi="10.1038/nature06258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International HapMap Consortium</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frazer</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ballinger</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cox</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinds</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A second generation human haplotype map of over 3.1 million SNPs.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFrazer%26rft.auinit1%253DK.%2BA.%26rft.volume%253D449%26rft.issue%253D7164%26rft.spage%253D851%26rft.epage%253D861%26rft.atitle%253DA%2Bsecond%2Bgeneration%2Bhuman%2Bhaplotype%2Bmap%2Bof%2Bover%2B3.1%2Bmillion%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06258%26rft_id%253Dinfo%253Apmid%252F17943122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17943122&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250230600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.9" data-doi="10.1006/jhev.2000.0403"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jablonski</span>  <span class="cit-name-given-names">N. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chaplin</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The evolution of human skin coloration.</span> <abbr class="cit-jnl-abbrev">J. Hum. Evol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHuman%2BEvolution%26rft.stitle%253DJournal%2Bof%2BHuman%2BEvolution%26rft.aulast%253DJablonski%26rft.auinit1%253DN.%2BG.%26rft.volume%253D39%26rft.issue%253D1%26rft.spage%253D57%26rft.epage%253D106%26rft.atitle%253DThe%2Bevolution%2Bof%2Bhuman%2Bskin%2Bcoloration.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjhev.2000.0403%26rft_id%253Dinfo%253Apmid%252F10896812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jhev.2000.0403&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10896812&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088185100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.10" data-doi="10.1038/ncomms1701"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graefen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matzas</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boisguerin</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">New insights into the Tyrolean Iceman’s origin and phenotype as inferred by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">698</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DKeller%26rft.auinit1%253DA.%26rft.volume%253D3%26rft.spage%253D698%26rft.epage%253D698%26rft.atitle%253DNew%2Binsights%2Binto%2Bthe%2BTyrolean%2BIceman%2527s%2Borigin%2Band%2Bphenotype%2Bas%2Binferred%2Bby%2Bwhole-genome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1701%26rft_id%253Dinfo%253Apmid%252F22426219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1701&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22426219&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.11" data-doi="10.1073/pnas.0509585102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filipski</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Swarna</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hedges</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Placing confidence limits on the molecular age of the human–chimpanzee divergence.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">18842</span>–<span class="cit-lpage">18847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0509585102%26rft_id%253Dinfo%253Apmid%252F16365310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzUyLzE4ODQyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.12" data-doi="10.1126/science.1116238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lamason</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mohideen</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mest</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Norton</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">SLC24A5, a putative cation exchanger, affects pigmentation in zebrafish and humans.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">310</span>: <span class="cit-fpage">1782</span>–<span class="cit-lpage">1786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLamason%26rft.auinit1%253DR.%2BL.%26rft.volume%253D310%26rft.issue%253D5755%26rft.spage%253D1782%26rft.epage%253D1786%26rft.atitle%253DSLC24A5%252C%2Ba%2BPutative%2BCation%2BExchanger%252C%2BAffects%2BPigmentation%2Bin%2BZebrafish%2Band%2BHumans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1116238%26rft_id%253Dinfo%253Apmid%252F16357253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTAvNTc1NS8xNzgyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.13" data-doi="10.1126/science.1153717"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Absher</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Southwick</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casto</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Worldwide human relationships inferred from genome-wide patterns of variation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">1100</span>–<span class="cit-lpage">1104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLi%26rft.auinit1%253DJ.%2BZ.%26rft.volume%253D319%26rft.issue%253D5866%26rft.spage%253D1100%26rft.epage%253D1104%26rft.atitle%253DWorldwide%2BHuman%2BRelationships%2BInferred%2Bfrom%2BGenome-Wide%2BPatterns%2Bof%2BVariation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1153717%26rft_id%253Dinfo%253Apmid%252F18292342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTkvNTg2Ni8xMTAwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.14" data-doi="10.1126/science.157.3788.501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loomis</span>  <span class="cit-name-given-names">W. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1967</span> <span class="cit-article-title">Skin-pigment regulation of vitamin-D biosynthesis in man.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLoomis%26rft.auinit1%253DW.%2BF.%26rft.volume%253D157%26rft.issue%253D3788%26rft.spage%253D501%26rft.epage%253D506%26rft.atitle%253DSkin-Pigment%2BRegulation%2Bof%2BVitamin-D%2BBiosynthesis%2Bin%2BMan%253A%2BVariation%2Bin%2Bsolar%2Bultraviolet%2Bat%2Bdifferent%2Blatitudes%2Bmay%2Bhave%2Bcaused%2Bracial%2Bdifferentiation%2Bin%2Bman%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.157.3788.501%26rft_id%253Dinfo%253Apmid%252F6028915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIxNTcvMzc4OC81MDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.15" data-doi="10.1126/science.1109792"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Macaulay</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Achilli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rengo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Single, rapid coastal settlement of Asia revealed by analysis of complete mitochondrial genomes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">308</span>: <span class="cit-fpage">1034</span>–<span class="cit-lpage">1036</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMacaulay%26rft.auinit1%253DV.%26rft.volume%253D308%26rft.issue%253D5724%26rft.spage%253D1034%26rft.epage%253D1036%26rft.atitle%253DSingle%252C%2BRapid%2BCoastal%2BSettlement%2Bof%2BAsia%2BRevealed%2Bby%2BAnalysis%2Bof%2BComplete%2BMitochondrial%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1109792%26rft_id%253Dinfo%253Apmid%252F15890885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDgvNTcyNC8xMDM0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.16" data-doi="10.1534/genetics.103.025494"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meiklejohn</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hartl</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parsch</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Identification of a locus under complex positive selection in <em>Drosophila simulans</em> by haplotype mapping and composite-likelihood estimation.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">168</span>: <span class="cit-fpage">265</span>–<span class="cit-lpage">279</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeiklejohn%26rft.auinit1%253DC.%2BD.%26rft.volume%253D168%26rft.issue%253D1%26rft.spage%253D265%26rft.epage%253D279%26rft.atitle%253DIdentification%2Bof%2Ba%2BLocus%2BUnder%2BComplex%2BPositive%2BSelection%2Bin%2BDrosophila%2Bsimulans%2Bby%2BHaplotype%2BMapping%2Band%2BComposite-Likelihood%2BEstimation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.103.025494%26rft_id%253Dinfo%253Apmid%252F15454542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY4LzEvMjY1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.17" data-doi="10.1038/nature04521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mellars</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A new radiocarbon revolution and the dispersal of modern humans in Eurasia.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">439</span>: <span class="cit-fpage">931</span>–<span class="cit-lpage">935</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMellars%26rft.auinit1%253DP.%26rft.volume%253D439%26rft.issue%253D7079%26rft.spage%253D931%26rft.epage%253D935%26rft.atitle%253DA%2Bnew%2Bradiocarbon%2Brevolution%2Band%2Bthe%2Bdispersal%2Bof%2Bmodern%2Bhumans%2Bin%2BEurasia.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04521%26rft_id%253Dinfo%253Apmid%252F16495989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16495989&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000235486100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.18" data-doi="10.1093/molbev/msl203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Norton</span>  <span class="cit-name-given-names">H. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kittles</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKeigue</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic evidence for the convergent evolution of light skin in Europeans and East Asians.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">710</span>–<span class="cit-lpage">722</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl203%26rft_id%253Dinfo%253Apmid%252F17182896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI0LzMvNzEwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2059.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oppenheimer</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-source">Out of Eden: The Peopling of the World</span>. <span class="cit-publ-name">Constable</span>, <span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2059.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bandelt</span>  <span class="cit-name-given-names">H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kivisild</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oppenheimer</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A model for the dispersal of modern humans out of Africa</span>, pp. <span class="cit-fpage">223</span>–<span class="cit-lpage">263</span> in <span class="cit-source">Human Mitochondrial DNA and the Evolution of Homo sapiens</span>: <span class="cit-source">Nucleic Acids and Molecular Biology</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Bandelt</span>  <span class="cit-name-given-names">H.-J.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Macaulay</span>  <span class="cit-name-given-names">V.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">M.</span></span>. <span class="cit-publ-name">Springer Verlag</span>, <span class="cit-publ-loc">Hamburg, Germany</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rozas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gullaud</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blandin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguadé</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">DNA variation at the rp49 gene region of <em>Drosophila simulans</em>: evolutionary inferences from an unusual haplotype structure.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">1147</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRozas%26rft.auinit1%253DJ.%26rft.volume%253D158%26rft.issue%253D3%26rft.spage%253D1147%26rft.epage%253D1155%26rft.atitle%253DDNA%2BVariation%2Bat%2Bthe%2Brp49%2BGene%2BRegion%2Bof%2BDrosophila%2Bsimulans%253A%2BEvolutionary%2BInferences%2BFrom%2Ban%2BUnusual%2BHaplotype%2BStructure%26rft_id%253Dinfo%253Apmid%252F11454763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1OC8zLzExNDciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.22" data-doi="10.1038/nature06250"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varilly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fry</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohmueller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hostetter</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide detection and characterization of positive selection in human populations.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">913</span>–<span class="cit-lpage">918</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSabeti%26rft.auinit1%253DP.%2BC.%26rft.volume%253D449%26rft.issue%253D7164%26rft.spage%253D913%26rft.epage%253D918%26rft.atitle%253DGenome-wide%2Bdetection%2Band%2Bcharacterization%2Bof%2Bpositive%2Bselection%2Bin%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06250%26rft_id%253Dinfo%253Apmid%252F17943131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06250&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17943131&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250230600049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.23" data-doi="10.1038/nrg3295"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scally</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Revising the human mutation rate: implications for understanding human evolution.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">745</span>–<span class="cit-lpage">753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DScally%26rft.auinit1%253DA.%26rft.volume%253D13%26rft.issue%253D10%26rft.spage%253D745%26rft.epage%253D753%26rft.atitle%253DRevising%2Bthe%2Bhuman%2Bmutation%2Brate%253A%2Bimplications%2Bfor%2Bunderstanding%2Bhuman%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3295%26rft_id%253Dinfo%253Apmid%252F22965354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3295&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22965354&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.24" data-doi="10.1016/j.cub.2009.11.054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soares</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Achilli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semino</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davies</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macaulay</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The archaeogenetics of Europe.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">R174</span>–<span class="cit-lpage">R183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DSoares%26rft.auinit1%253DP.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253DR174%26rft.epage%253DR183%26rft.atitle%253DThe%2Barchaeogenetics%2Bof%2BEurope.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2009.11.054%26rft_id%253Dinfo%253Apmid%252F20178764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2009.11.054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20178764&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275149200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.25" data-doi="10.1007/s00414-006-0112-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Soejima</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koda</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Population differences of two coding SNPs in pigmentation-related genes SLC24A5 and SLC45A2</span>. <abbr class="cit-jnl-abbrev">Int. J. Leg. Med.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">39</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2Bjournal%2Bof%2Blegal%2Bmedicine%26rft.stitle%253DInt%2BJ%2BLegal%2BMed%26rft.aulast%253DSoejima%26rft.auinit1%253DM.%26rft.volume%253D121%26rft.issue%253D1%26rft.spage%253D36%26rft.epage%253D39%26rft.atitle%253DPopulation%2Bdifferences%2Bof%2Btwo%2Bcoding%2BSNPs%2Bin%2Bpigmentation-related%2Bgenes%2BSLC24A5%2Band%2BSLC45A2.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00414-006-0112-z%26rft_id%253Dinfo%253Apmid%252F16847698%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00414-006-0112-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16847698&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242969600006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.26" data-doi="10.1086/379378"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A comparison of bayesian methods for haplotype reconstruction from population genotype data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">1162</span>–<span class="cit-lpage">1169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DStephens%26rft.auinit1%253DM.%26rft.volume%253D73%26rft.issue%253D5%26rft.spage%253D1162%26rft.epage%253D1169%26rft.atitle%253DA%2Bcomparison%2Bof%2Bbayesian%2Bmethods%2Bfor%2Bhaplotype%2Breconstruction%2Bfrom%2Bpopulation%2Bgenotype%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F379378%26rft_id%253Dinfo%253Apmid%252F14574645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/379378&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14574645&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186493400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.27" data-doi="10.1086/319501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">A new statistical method for haplotype reconstruction from population data.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">978</span>–<span class="cit-lpage">989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DStephens%26rft.auinit1%253DM.%26rft.volume%253D68%26rft.issue%253D4%26rft.spage%253D978%26rft.epage%253D989%26rft.atitle%253DA%2Bnew%2Bstatistical%2Bmethod%2Bfor%2Bhaplotype%2Breconstruction%2Bfrom%2Bpopulation%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F319501%26rft_id%253Dinfo%253Apmid%252F11254454%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/319501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11254454&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167666000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2059.28" data-doi="10.1086/522235"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stokowski</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pant</span>  <span class="cit-name-given-names">P. V. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dadd</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fereday</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinds</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A genomewide association study of skin pigmentation in a South Asian population.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1119</span>–<span class="cit-lpage">1132</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DStokowski%26rft.auinit1%253DR.%2BP.%26rft.volume%253D81%26rft.issue%253D6%26rft.spage%253D1119%26rft.epage%253D1132%26rft.atitle%253DA%2Bgenomewide%2Bassociation%2Bstudy%2Bof%2Bskin%2Bpigmentation%2Bin%2Ba%2BSouth%2BAsian%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F522235%26rft_id%253Dinfo%253Apmid%252F17999355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/522235&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17999355&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2059.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251098800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/11/2059.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/2049" title="Evolution After Whole-Genome Duplication: A Network Perspective" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/11/2069" title="Allelic Variation in a Cellulose Synthase Gene (PtoCesA4) Associated with Growth and Wood Properties in Populus tomentosa" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/11"><span>Volume 3 Issue 11, November 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/11.cover-source.jpg?itok=A7iZ1ztq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/2059.short" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DStj4r73q-kZ3FrhKDt5n9gFIxiidu9mB7cDiViJVYo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/11/2059" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/11/2059" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/11/2059" class="active" data-icon-position="" data-hide-link-title="0">Molecular Phylogeography of a Human Autosomal Skin Color Locus Under Natural Selection</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459577" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-VWJu4_glwNZtxf44MRL31TwBv15rOntu8sKe7QHVam4" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/2059.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/11/2059" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-sUF-xVoDS8Iu2cYIqNfmP1YqCMu7xW9ArMfTtUmiiNc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/11/2059" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/2059.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459577" id="node459577--2" data-pisa="ggg;3/11/2059" data-pisa-master="ggg;g3.113.007484" data-apath="/ggg/3/11/2059.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Molecular Phylogeography of a Human Autosomal Skin Color Locus Under Natural Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Victor A. Canfield</span>, <span class="highwire-citation-author" data-delta="1">Arthur Berg</span>, <span class="highwire-citation-author" data-delta="2">Steven Peckins</span>, <span class="highwire-citation-author" data-delta="3">Steven M. Wentzel</span>, <span class="highwire-citation-author" data-delta="4">Khai Chung Ang</span>, <span class="highwire-citation-author" data-delta="5">Stephen Oppenheimer</span> and <span class="highwire-citation-author" data-delta="6">Keith C. Cheng</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2059-2067;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007484 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459577--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Victor A. Canfield </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pharmacology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victor%2BA.%2BCanfield%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Canfield%20VA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVictor%2BA.%2BCanfield%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vcanfield@gmail.com" class="" data-icon-position="" data-hide-link-title="0">vcanfield@gmail.com</a>
<a href="mailto:kcheng76@gmail.com" class="" data-icon-position="" data-hide-link-title="0">kcheng76@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Arthur Berg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Penn State Cancer Institute, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Public Health Sciences, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arthur%2BBerg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Berg%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArthur%2BBerg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Steven Peckins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Jake Gittlen Cancer Research Foundation, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Experimental Pathology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BPeckins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peckins%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BPeckins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Steven M. Wentzel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Jake Gittlen Cancer Research Foundation, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Experimental Pathology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BM.%2BWentzel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wentzel%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BM.%2BWentzel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Khai Chung Ang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Jake Gittlen Cancer Research Foundation, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Experimental Pathology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Khai%2BChung%2BAng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ang%20KC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKhai%2BChung%2BAng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Stephen Oppenheimer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>School of Anthropology and Museum Ethnography, Oxford University, Oxford, OX2 6PE, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BOppenheimer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oppenheimer%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BOppenheimer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Keith C. Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pharmacology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Penn State Cancer Institute, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Experimental Pathology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Keith%2BC.%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20KC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKeith%2BC.%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vcanfield@gmail.com" class="" data-icon-position="" data-hide-link-title="0">vcanfield@gmail.com</a>
<a href="mailto:kcheng76@gmail.com" class="" data-icon-position="" data-hide-link-title="0">kcheng76@gmail.com</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/11/2059" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/11/2059&amp;title=Molecular%20Phylogeography%20of%20a%20Human%20Autosomal%20Skin%20Color%20Locus%20Under%20Natural%20Selection" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/11/2059&amp;title=Molecular%20Phylogeography%20of%20a%20Human%20Autosomal%20Skin%20Color%20Locus%20Under%20Natural%20Selection" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/11/2059&amp;title=Molecular%20Phylogeography%20of%20a%20Human%20Autosomal%20Skin%20Color%20Locus%20Under%20Natural%20Selection" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/11/2059&amp;text=Molecular%20Phylogeography%20of%20a%20Human%20Autosomal%20Skin%20Color%20Locus%20Under%20Natural%20Selection" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/11/2059&amp;title=Molecular%20Phylogeography%20of%20a%20Human%20Autosomal%20Skin%20Color%20Locus%20Under%20Natural%20Selection" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/11/2059&amp;t=Molecular%20Phylogeography%20of%20a%20Human%20Autosomal%20Skin%20Color%20Locus%20Under%20Natural%20Selection" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/11/2059&amp;title=Molecular%20Phylogeography%20of%20a%20Human%20Autosomal%20Skin%20Color%20Locus%20Under%20Natural%20Selection" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/11/2059&amp;title=Molecular%20Phylogeography%20of%20a%20Human%20Autosomal%20Skin%20Color%20Locus%20Under%20Natural%20Selection" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459577/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459577" id="node459577--3" data-pisa="ggg;3/11/2059" data-pisa-master="ggg;g3.113.007484" data-apath="/ggg/3/11/2059.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Molecular Phylogeography of a Human Autosomal Skin Color Locus Under Natural Selection</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Victor A. Canfield</span>, <span class="highwire-citation-author" data-delta="1">Arthur Berg</span>, <span class="highwire-citation-author" data-delta="2">Steven Peckins</span>, <span class="highwire-citation-author" data-delta="3">Steven M. Wentzel</span>, <span class="highwire-citation-author" data-delta="4">Khai Chung Ang</span>, <span class="highwire-citation-author" data-delta="5">Stephen Oppenheimer</span> and <span class="highwire-citation-author" data-delta="6">Keith C. Cheng</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2059-2067;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.007484 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459577--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Victor A. Canfield </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pharmacology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Victor%2BA.%2BCanfield%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Canfield%20VA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVictor%2BA.%2BCanfield%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vcanfield@gmail.com" class="" data-icon-position="" data-hide-link-title="0">vcanfield@gmail.com</a>
<a href="mailto:kcheng76@gmail.com" class="" data-icon-position="" data-hide-link-title="0">kcheng76@gmail.com</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Arthur Berg </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Penn State Cancer Institute, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Public Health Sciences, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arthur%2BBerg%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Berg%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AArthur%2BBerg%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Steven Peckins </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Jake Gittlen Cancer Research Foundation, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Experimental Pathology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BPeckins%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peckins%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BPeckins%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Steven M. Wentzel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Jake Gittlen Cancer Research Foundation, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Experimental Pathology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Steven%2BM.%2BWentzel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wentzel%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASteven%2BM.%2BWentzel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Khai Chung Ang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Jake Gittlen Cancer Research Foundation, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Experimental Pathology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Khai%2BChung%2BAng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ang%20KC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKhai%2BChung%2BAng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Stephen Oppenheimer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>School of Anthropology and Museum Ethnography, Oxford University, Oxford, OX2 6PE, UK</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BOppenheimer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Oppenheimer%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AStephen%2BOppenheimer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Keith C. Cheng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Pharmacology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Penn State Cancer Institute, Penn State College of Medicine, Hershey, Pennsylvania 17033</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Division of Experimental Pathology, Penn State College of Medicine, Hershey, Pennsylvania 17033</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Keith%2BC.%2BCheng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cheng%20KC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKeith%2BC.%2BCheng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:vcanfield@gmail.com" class="" data-icon-position="" data-hide-link-title="0">vcanfield@gmail.com</a>
<a href="mailto:kcheng76@gmail.com" class="" data-icon-position="" data-hide-link-title="0">kcheng76@gmail.com</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2059&amp;rft.epage=2067&amp;rft.atitle=Molecular%20Phylogeography%20of%20a%20Human%20Autosomal%20Skin%20Color%20Locus%20Under%20Natural%20Selection&amp;rft.volume=3&amp;rft.issue=11&amp;rft.date=2013-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Canfield%2C+Victor+A.&amp;rft.au=Berg%2C+Arthur&amp;rft.au=Peckins%2C+Steven&amp;rft.au=Wentzel%2C+Steven+M.&amp;rft.au=Ang%2C+Khai+Chung&amp;rft.au=Oppenheimer%2C+Stephen&amp;rft.au=Cheng%2C+Keith+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459577/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459577/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459577/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459577/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459577/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459577/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459577/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459577/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459577/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459577/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459577/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459577/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/11/2059" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-2" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/11/2059.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/11/2059.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/11/2059.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/11/2059.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
