<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/11/1955" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/11/1955.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/11/1955.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/11/1955.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/11/1955.atom" />
<meta name="HW.pisa" content="ggg;3/11/1955" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A DNA Sequence Element That Advances Replication Origin Activation Time in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.113.008250" />
<meta name="DC.Date" content="2013-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2013 Pohl et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Eukaryotic origins of DNA replication undergo activation at various times in S-phase, allowing the genome to be duplicated in a temporally staggered fashion. In the budding yeast Saccharomyces cerevisiae , the activation times of individual origins are not intrinsic to those origins but are instead governed by surrounding sequences. Currently, there are two examples of DNA sequences that are known to advance origin activation time, centromeres and forkhead transcription factor binding sites. By combining deletion and linker scanning mutational analysis with two-dimensional gel electrophoresis to measure fork direction in the context of a two-origin plasmid, we have identified and characterized a 19- to 23-bp and a larger 584-bp DNA sequence that are capable of advancing origin activation time." />
<meta name="DC.Contributor" content="Thomas J. Pohl" />
<meta name="DC.Contributor" content="Katherine Kolor" />
<meta name="DC.Contributor" content="Walton L. Fangman" />
<meta name="DC.Contributor" content="Bonita J. Brewer" />
<meta name="DC.Contributor" content="M. K. Raghuraman" />
<meta name="article:published_time" content="2013-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A DNA Sequence Element That Advances Replication Origin Activation Time in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Eukaryotic origins of DNA replication undergo activation at various times in S-phase, allowing the genome to be duplicated in a temporally staggered fashion. In the budding yeast &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt;, the activation times of individual origins are not intrinsic to those origins but are instead governed by surrounding sequences. Currently, there are two examples of DNA sequences that are known to advance origin activation time, centromeres and forkhead transcription factor binding sites. By combining deletion and linker scanning mutational analysis with two-dimensional gel electrophoresis to measure fork direction in the context of a two-origin plasmid, we have identified and characterized a 19- to 23-bp and a larger 584-bp DNA sequence that are capable of advancing origin activation time.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/11/01" />
<meta name="citation_mjid" content="ggg;3/11/1955" />
<meta name="citation_id" content="3/11/1955" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/11/1955" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/11/1955.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/11/1955.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/11/1955.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.008250" />
<meta name="citation_pmid" content="24022751" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1955" />
<meta name="citation_lastpage" content="1963" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Thomas J. Pohl" />
<meta name="citation_author_institution" content="Molecular and Cellular Biology Program, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Katherine Kolor" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author_institution" content="Office of Public Health Genomics, Centers for Disease Control and Prevention, Atlanta, Georgia 30329" />
<meta name="citation_author" content="Walton L. Fangman" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="Bonita J. Brewer" />
<meta name="citation_author_institution" content="Molecular and Cellular Biology Program, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author" content="M. K. Raghuraman" />
<meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington, Seattle, Washington 98195" />
<meta name="citation_author_email" content="raghu@u.washington.edu" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=B. J. Brewer;citation_author=W. L. Fangman;citation_title=Initiation preference at a yeast origin of replication.;citation_pages=3418-3422;citation_volume=91;citation_year=1994;citation_issue=8;citation_pmid=8159762;citation_doi=10.1073/pnas.91.8.3418" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=B. J. Brewer;citation_author=D. Lockshon;citation_author=W. L. Fangman;citation_title=The arrest of replication forks in the rDNA of yeast occurs independently of transcription.;citation_pages=267-276;citation_volume=71;citation_year=1992;citation_issue=2;citation_pmid=1423594;citation_doi=10.1016/0092-8674(92)90355-G" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=A. J. Cosgrove;citation_author=C. A. Nieduszynski;citation_author=A. D. Donaldson;citation_title=Ku complex controls the replication time of DNA in telomere regions;citation_pages=2485-2490;citation_volume=16;citation_year=2002;citation_issue=19;citation_pmid=12368259;citation_doi=10.1101/gad.231602" />
<meta name="citation_reference" content="citation_journal_title=Anal. Biochem.;citation_author=W. P. Deng;citation_author=J. A. Nickoloff;citation_title=Site-directed mutagenesis of virtually any plasmid by eliminating a unique site.;citation_pages=8;citation_volume=200;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=B. M. Ferguson;citation_author=W. L. Fangman;citation_title=A position effect on the time of replication origin activation in yeast.;citation_pages=333-339;citation_volume=68;citation_year=1992;citation_issue=2;citation_pmid=1733502;citation_doi=10.1016/0092-8674(92)90474-Q" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=B. M. Ferguson;citation_author=B. J. Brewer;citation_author=A. E. Reynolds;citation_author=W. L. Fangman;citation_title=A yeast origin of replication is activated late in S phase.;citation_pages=507-515;citation_volume=65;citation_year=1991;citation_issue=3;citation_pmid=2018976;citation_doi=10.1016/0092-8674(91)90468-E" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=K. L. Friedman;citation_author=B. J. Brewer;citation_title=Analysis of replication intermediates by two-dimensional agarose gel electrophoresis.;citation_pages=613-627;citation_volume=262;citation_year=1995;citation_pmid=8594382;citation_doi=10.1016/0076-6879(95)62048-6" />
<meta name="citation_reference" content="citation_journal_title=Journal of cell science.  Supplement;citation_journal_abbrev=J Cell Sci Suppl;citation_author=K. L. Friedman;citation_author=M. K. Raghuraman;citation_author=W. L. Fangman;citation_author=B. J. Brewer;citation_title=Analysis of the temporal program of replication initiation in yeast chromosomes.;citation_pages=51-58;citation_volume=19;citation_year=1995;citation_pmid=8655647" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=K. L. Friedman;citation_author=J. D. Diller;citation_author=B. M. Ferguson;citation_author=S. V. Nyland;citation_author=B. J. Brewer;citation_title=Multiple determinants controlling activation of yeast replication origins late in S phase.;citation_pages=1595-1607;citation_volume=10;citation_year=1996;citation_issue=13;citation_pmid=8682291;citation_doi=10.1101/gad.10.13.1595" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Gupta;citation_author=J. A. Stamatoyannopoulos;citation_author=T. L. Bailey;citation_author=W. S. Noble;citation_title=Quantifying similarity between motifs.;citation_pages=R24-R24;citation_volume=8;citation_year=2007;citation_issue=2;citation_pmid=17324271;citation_doi=10.1186/gb-2007-8-2-r24" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=C. S. Hoffman;citation_author=F. Winston;citation_title=A ten-minute DNA preparation from yeast efficiently releases autonomous plasmids for transformation of Escherichia coli.;citation_pages=267-272;citation_volume=57;citation_year=1987;citation_issue=2-3;citation_pmid=3319781;citation_doi=10.1016/0378-1119(87)90131-4" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. A. Huberman;citation_author=L. D. Spotila;citation_author=K. A. Nawotka;citation_author=S. M. el-Assouli;citation_author=L. R. Davis;citation_title=The in vivo replication origin of the yeast 2 microns plasmid.;citation_pages=473-481;citation_volume=51;citation_year=1987;citation_issue=3;citation_pmid=3311385;citation_doi=10.1016/0092-8674(87)90643-X" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. R. Knott;citation_author=J. M. Peace;citation_author=A. Z. Ostrow;citation_author=Y. Gan;citation_author=A. E. Rex;citation_title=Forkhead transcription factors establish origin timing and long-range clustering in S. cerevisiae.;citation_pages=99-111;citation_volume=148;citation_year=2012;citation_issue=1-2;citation_pmid=22265405;citation_doi=10.1016/j.cell.2011.12.012" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Cell;citation_author=H. Y. Lian;citation_author=E. D. Robertson;citation_author=S. Hiraga;citation_author=G. M. Alvino;citation_author=D. Collingwood;citation_title=The effect of Ku on telomere replication time is mediated by telomere length but is independent of histone tail acetylation.;citation_volume=22;citation_year=2011;citation_pmid=21441303;citation_doi=10.1091/mbc.E10-06-0549" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=Y. Marahrens;citation_author=B. Stillman;citation_title=A yeast chromosomal origin of DNA replication defined by multiple functional elements;citation_pages=817-823;citation_volume=255;citation_year=1992;citation_issue=5046;citation_pmid=1536007;citation_doi=10.1126/science.1536007" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. M. McCarroll;citation_author=W. L. Fangman;citation_title=Time of replication of yeast centromeres and telomeres.;citation_pages=505-513;citation_volume=54;citation_year=1988;citation_issue=4;citation_pmid=3042152;citation_doi=10.1016/0092-8674(88)90072-4" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. J. McCune;citation_author=L. S. Danielson;citation_author=G. M. Alvino;citation_author=D. Collingwood;citation_author=J. J. Delrow;citation_title=The temporal program of chromosome replication: genomewide replication in clb5â Saccharomyces cerevisiae.;citation_volume=180;citation_year=2008;citation_pmid=18832352;citation_doi=10.1534/genetics.108.094359" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=D. A. Natale;citation_author=A. E. Schubert;citation_author=D. Kowalski;citation_title=DNA helical stability accounts for mutational defects in a yeast replication origin.;citation_pages=2654-2658;citation_volume=89;citation_year=1992;citation_issue=7;citation_pmid=1557369;citation_doi=10.1073/pnas.89.7.2654" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=D. A. Natale;citation_author=R. M. Umek;citation_author=D. Kowalski;citation_title=Ease of DNA unwinding is a conserved property of yeast replication origins;citation_pages=555-560;citation_volume=21;citation_year=1993;citation_issue=3;citation_pmid=8441667;citation_doi=10.1093/nar/21.3.555" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=T. Natsume;citation_author=C. A. Muller;citation_author=Y. Katou;citation_author=R. Retkute;citation_author=M. Gierlinski;citation_title=Kinetochores coordinate pericentromeric cohesion and early DNA replication by cdc7-dbf4 kinase recruitment.;citation_pages=661-674;citation_volume=50;citation_year=2013;citation_pmid=23746350;citation_doi=10.1016/j.molcel.2013.05.011" />
<meta name="citation_reference" content="citation_title=DNA Replication in Yeast;citation_author=C. S. Newlon;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. A. Nieduszynski;citation_author=S. Hiraga;citation_author=P. Ak;citation_author=C. J. Benham;citation_author=A. D. Donaldson;citation_title=OriDB: a DNA replication origin database.;citation_volume=35;citation_year=2007;citation_pmid=17065467;citation_doi=10.1093/nar/gkl758" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=T. J. Pohl;citation_author=B. J. Brewer;citation_author=M. K. Raghuraman;citation_title=Functional centromeres determine the activation time of pericentric origins of DNA replication in Saccharomyces cerevisiae.;citation_pages=e1002677-e1002677;citation_volume=8;citation_year=2012;citation_issue=5;citation_pmid=22589733;citation_doi=10.1371/journal.pgen.1002677" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. K. Raghuraman;citation_author=E. A. Winzeler;citation_author=D. Collingwood;citation_author=S. Hunt;citation_author=L. Wodicka;citation_title=Replication Dynamics of the Yeast Genome;citation_pages=115-121;citation_volume=294;citation_year=2001;citation_issue=5540;citation_pmid=11588253;citation_doi=10.1126/science.294.5540.115" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Cell Biology;citation_journal_abbrev=JCB;citation_author=C. J. Rivin;citation_author=W. L. Fangman;citation_title=Replication fork rate and origin activation during the S phase of Saccharomyces cerevisiae;citation_pages=108-115;citation_volume=85;citation_year=1980;citation_issue=1;citation_pmid=6767729;citation_doi=10.1083/jcb.85.1.108" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=J. B. Stevenson;citation_author=D. E. Gottschling;citation_title=Telomeric chromatin modulates replication timing near chromosome ends;citation_pages=146-151;citation_volume=13;citation_year=1999;citation_issue=2;citation_pmid=9925638;citation_doi=10.1101/gad.13.2.146" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=S. S. Walker;citation_author=S. C. Francesconi;citation_author=B. K. Tye;citation_author=S. Eisenberg;citation_title=The OBF1 protein and its DNA-binding site are important for the function of an autonomously replicating sequence in Saccharomyces cerevisiae.;citation_pages=2914-2921;citation_volume=9;citation_year=1989;citation_issue=7;citation_pmid=2674674" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=R. J. Wellinger;citation_author=A. J. Wolf;citation_author=V. A. Zakian;citation_title=Origin activation and formation of single-strand TG1-3 tails occur sequentially in late S phase on a yeast linear plasmid.;citation_pages=4057-4065;citation_volume=13;citation_year=1993;citation_issue=7;citation_pmid=8321213;citation_doi=10.1128/MCB.13.7.4057" />
<meta name="citation_reference" content="citation_journal_title=Genes to Cells;citation_journal_abbrev=GENES CELLS;citation_author=N. Yabuki;citation_author=H. Terashima;citation_author=K. Kitada;citation_title=Mapping of early firing origins on a replication profile of budding yeast;citation_pages=781-789;citation_volume=7;citation_year=2002;citation_issue=8;citation_pmid=12167157;citation_doi=10.1046/j.1365-2443.2002.00559.x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A DNA Sequence Element That Advances Replication Origin Activation Time in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="twitter:description" content="Eukaryotic origins of DNA replication undergo activation at various times in S-phase, allowing the genome to be duplicated in a temporally staggered fashion. In the budding yeast Saccharomyces cerevisiae , the activation times of individual origins are not intrinsic to those origins but are instead governed by surrounding sequences. Currently, there are two examples of DNA sequences that are known to advance origin activation time, centromeres and forkhead transcription factor binding sites. By combining deletion and linker scanning mutational analysis with two-dimensional gel electrophoresis to measure fork direction in the context of a two-origin plasmid, we have identified and characterized a 19- to 23-bp and a larger 584-bp DNA sequence that are capable of advancing origin activation time." />
<meta name="og-title" property="og:title" content="A DNA Sequence Element That Advances Replication Origin Activation Time in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/11/1955" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Eukaryotic origins of DNA replication undergo activation at various times in S-phase, allowing the genome to be duplicated in a temporally staggered fashion. In the budding yeast Saccharomyces cerevisiae , the activation times of individual origins are not intrinsic to those origins but are instead governed by surrounding sequences. Currently, there are two examples of DNA sequences that are known to advance origin activation time, centromeres and forkhead transcription factor binding sites. By combining deletion and linker scanning mutational analysis with two-dimensional gel electrophoresis to measure fork direction in the context of a two-origin plasmid, we have identified and characterized a 19- to 23-bp and a larger 584-bp DNA sequence that are capable of advancing origin activation time." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
    <title>A DNA Sequence Element That Advances Replication Origin Activation Time in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"i1MoMXa_WCZ2hdHTvZi-kA8TouMMkJ_EnPZLTMgj5LM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459580","apath":"\/ggg\/3\/11\/1955.atom","pisa":"ggg;3\/11\/1955","ac":{"\/ggg\/3\/11\/1955.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/11\/1955.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/11\/1955"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/11\/1955"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/11\/1955.long":true,"\/content\/3\/11\/1955":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/0LAJ1ZSwMGWRlUgXMibXN6WfkFh7fR\\\\\\\/n9J55iwIFsd+AkRviRDGm9KCkQxnWz5pJ5b6x4p\\\\\\\/gtfGfCE5MujvHXYt86TeTDHfTO6AcwukwA8zsdE3QofUeUCozEBtar5cnsXEc48zNbHwMV\\\\\\\/bkqk\\\\\\\/2by8nSqsApiPjj+RvqCsWoza++6aog1sQMDRRRjMVV+srvBqoy+swKa3XHuI6GZJXZsx3INrWB5kGUXMM7Yy4t5MZjS6G2iIdkWY+C9lkTUI+Trp9TzfPPPs6VvagYZikuZvLD92fJ5btM+V2BrpOmB7wadFma5t4kiwLOhGABoi09U9UfHOfLW5R5SlVSi9XN10q1XDgHi83qhtosn\\\\\\\/xlKhm1JwdZznyDMrVGUyVAHpRd6Zf3JDSrtLoT8XjX\\\\\\\/UIQTmY2AMyxnFu0gtFTJ3X3rjjAYi0a2b\\\\\\\/tYy5VnpiWIm+lpyMSHt4BcpahnKyb+R8AYN\\\\\\\/bbeF8ofoy2oaIjo6UZ5Wz5ap7WE\\\\\\\/JaFZDoK+3UUEZfKowa\\\\\\\/jrRucjM0aacxBD7zMwtjYKJRC0KiB8RU7DOhIliP9Gi4f92GDTgRexHfOvda5MKL7ZiafnkOCoGfX74LSMnEFKQpofknrmtf0eadX3LPaRG9yVwg8g+J35ZV3aA5Oulx2piG2asEZm5BSqTLLZ\\\\\\\/fJpATQklShsrMO8wKKbbgTnUkp72YXtEIC6mz9pk3K5tQGiQL7VcVrIBeqzJSBa1XYrbg50nqJYeXwWwf0jM55Si3Fe2qwf3HKaRkyna93jAbclV49d\\\\\\\/8l02MnbpkafxRxKYmUFikxotdwJu0uRQvRnAmIkYs1Fq3jiPBaJk5bun5AFF3IVsiCz37CWBo2jHuEF9mYfY8NMKvtAT6IbRX1D0q4tebDZlnpP7HctoHUP7nWBcadebk4VoiDhtZbiaTifvsw4hWN0q3YV\\\\\\\/WKCVdylBpuHU1DtgezteTTcLa8YyDfxP+CoyO6g4h+H1O9PGkNZigeWHosq5N0\\\\\\\/rWCK9T1mD+L5op0\\\\\\\/vcYvnKpHfXIxyiHsGXgWItLgkVww1KOTp0NfTtIKDOIfy5VFeY1NedPxmf52f097Yth6GS0RXELBBs9buFzTAwXZzjmGpx+9WWeCG2IqQtjOAkz62i+h4oBuFCyShFlSSRg8bWtrlMS9iicbN3CnaEfBq\\\\\\\/wsLAbWQko2zyLLpdXPxt7TPbVGklcqtbOJkWQeXGwc2DEtddvY3zQxJlzoa46IkjuRgFIggKpMNFSvPv44Tv2a2JX5vfe\\\\\\\/k2+Pjb7DWpWmD+\\\u0022,\\\u0022iv\\\u0022:\\\u0022MVfoedFbSGh2nknTUyoKsA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220f239a802e9051a8808778eeaeafe3b93bb7bcf0a4765956d553f188695138e7\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UTE90UCxkZ\\\\\\\/qmHaHLmKXVYQugYYYhre3heWAgGryhRkdiTC2huf5EkT2oCnLuBnt3iDNtZtosUtDVkGWC\\\\\\\/FqsJ1HJSItieXtIvGxnkWRi\\\\\\\/AwcWwKyWUj70QPDixuAlQ1435O0F54yhx2O6nsMnBEoLebPnSdlo\\\\\\\/HaS2wS+6oltAV7LCOKNp3H\\\\\\\/hUDS7BTTNhDSrA3SJZX6zRK0R8FTKq9kCHsO426rhOTaiYQp7pkDiGI3oJoDLQ4p92vXnoILty5sez5vRFHf1gfUvDEb6hKyA5d8cS7Vph6VRMaDQbhlna2l3nmrPP6zkSMcYWzbkw0LuyPxWb9S\\\\\\\/uCHELkX+lQb870HC0845GSeY6J+btZJ9ywMFbYfDSgOdFL+eksKE112\\\\\\\/uIqIqCZopsQZ0Hy8JuDmhrj8E8JaooHPEMI+KSigx2eibZH1NeAm\\\\\\\/aMQ0MIK41WT2a+ZBPn1uaYrYn4x84Ntkm88MuxIWq+RHUPXOV8vaMsDah0xjWyaIs2JyENKRyQf\\\\\\\/MfFTr\\\\\\\/YqTWUmuTgZst4TQ39emF9VwUB+OYoM92CbChvZKhnOqV\\\\\\\/UusPWvczspHvJbjK4zlEugElrZZRsuwR7DHVJDdsMvQdWkzSTpqCGz4kp51174xGiXcY8\\\\\\\/R2I4juAUv3n8cvDUzML\\\\\\\/t5VaYeps33YLsevmdaYcRMDynL8mghhMRyzdbh7EYn+4KZGlqcehmtxkZ5D3omAAXCeDfLOp6ER24Pc\\\\\\\/sQf4PXszpX0hrtGmbzMVd5IaAxVezA\\\\\\\/J9GniZBDtZAyfCyzij9gejJWD1tNhe7vSSp7HSxhF6g+L5KA0s5dVBF3\\\\\\\/yJcx0oDbeDMIBA\\\\\\\/bqXV7zz1OhZU6ZTKuUC0AYXNZ8K2lCQXti3g+eaJKKSsXzJ5ArAuDQT9hBSjLx03UiSUOHkV0hfjkF0FGWZOa1JVvmSThyrEAt4rZV\\\\\\\/nKvOnYzUO8mruOuMkhs+SkFv+sbs\\\\\\\/zSLmrwRKnqLzOCklNieQex16yGyYo0lquu1ND7uivCYTKVSTD9h0ZgeM0lEHerxJfEo2v+cyz6TGjOcAuw55quVSq6P4SdhdSeUZjfIe\\\\\\\/HOLUq205+f+3D\\\\\\\/t9ZEjFpjW3jKAkOIh7Gcvy7S2IrprPJEgfP9akLIrFSwTetY2uJxS3gtCkb2HNsxSjpJm6SeQC55emo7B\\\\\\\/N6CkIAbGM845mZLBlPONSYp9EjLjaoVKXZMKZAt4+SVMR10GyfOLQDao9boGERCuQNiJLMBgR4e8veWGLISJf7cASntaPkIQJYzlvameuUIORuT9kZfhW8DvWOlAisi\\\\\\\/a1TZc8zE5g+eg+ssj\\\\\\\/\\\\\\\/wlhH56lTIv8ZGeoT2wdhUIN22qwjl0l9vXAc1z9X2KMUxF8aGG2seYnpF7tI+W9XgGNmTDSCB4RcbchgWZ4K1FEWT4apnurakEAN9\\\\\\\/GhfOKNBouTMIw5mA0uKzlwKFjw6IZCDI+m3QDK2dbo9hMq5V0oP7JQ1cozsA0D581nc6BM1Ek\\\\\\\/pVr4qcfR9u2r6ZcsAnSFrVoU3x1ISKSZ1FnCIP\\\\\\\/M3Wzlr3TKK8XQgmwd6mpRyvUtZAPR+lsf0YCNXP7SPd4GE06+0KIlIkZRShASKFJpizmYBYEidkALe51c5tvgRJ3F2PhAoqV9vrDz8phoXR5I+a0mXByzlqSSDY3F1MAmkOUhvTP9IWc2AgOq+Orqd7KCU5KDyTCUr9WXrT8k3MTluunFOJct24Zr9yiSXEOtSDQ5LCJk+okgck4npAKSedOCvbzpOgxTz1nfr0P5zpGaatXS+hMKuf3BwqVtDrS3f4aCxwZOfHTQ3NCRreNm4xmGKbxEbN2mRn1wEHn17yrb3an0TJQjyE\\\\\\\/HNYTzyAZUqL6UicAEo36tY1Foo3jtgv5ewp+lHri5DcCBmYXUa\\\\\\\/lNp+O6YzQ7fVLZr8Z8xDMF2rAp7ZkyBcVw2bBsF2+g0edqJYiAs3FqvbRPPUuJEnAL8KNR+wk7lQ8gRiNxMh2xtrea4IFwQcT+0cKtgSLEi8p7uXxR+BkUUyF5Mq4fye\\\\\\\/9e14TcAGJ5tEMG+qZPZNrw+O6FZ+La3WiSpgOdEgvXZeYGab1Uhy3IDJ2KrxJSe+VsduJdjTnDe7for73vgGhwJBdxVf\\\\\\\/ClpCduUXDSdB9koiT1A1gZKT8ztm4odl8sUMaQza6bzLj7U7RJSbucHxQmK3yCO2jKqs1iZfAu9gCT0P9N0XRL2zJY5PHdzxIskaXwyzGEHdbKteyD4w\\\\\\\/5ET3rCmFm1yr5iCl02sYv2ubyDlriyfTTjaiPYr82GTRDPI4LRgDr3l2YUA3wFyfU5URCEeaq6zXe6tyQf9rVPszAzCMFHPT5xvIHMsIkw9AM1l5FyxMR5M4d50UX+Bln8lvIWSM9eXhX9Zs+64TU2WPgSamzAhL0PEzS2IUJ5sj5y7WId7uGnTP3q\\\\\\\/yFAk7TlLC0mrJoTYMIzhGO\\\\\\\/k+Sl0w1WKmTWV57K26n2Kx6jLGImFrJVgBZ\\\\\\\/76JTei1ZLkxFZ9O1LXZB7RfcT\\\\\\\/bZEeHXHgmXkZvptJKFrCuSOhVbrpczoa56f0A+Vh+PxtAhvmQLduAr0NK3NZCq63LY0ltvgnTCHYM65iw5aY6Vi0JBnzuqqY7FS3riqzYart5g+9f9Z20CdoV3OziampdyfniSk\\\\\\\/l+sVehXBCzpDhN9053Ela7cxSHwg6lxthoQD340RxcwO9TN9lha0ZH9U14NYbAHTkasogtwuFxicpzxmHf7H82E1\\\\\\\/5TJ8VWDmW9G0hNx8XTpK7y8CTz\\\\\\\/sGUv9VK3Lhiqd6UTcY0P6h8tdFD5aW4PAMejvvWPVpXA6Qrb3FMF07pehQufBa3UU8y4tebWduPR0tTpJGaIP1LVKmLg4sSc9GpBD8OHn83wSbGv5gTbwuZfFYJwvYnTn3Or1NzevizwBVhFWhWmIx7629bqR+6XGb0\\\\\\\/14EPblk2gbeo5U2fnA8CWP0GPmDCu0JjnVmRD2zaTL\\\\\\\/Z4ClE7kZjdVddkEHPobVvEJNrxFmDJAOZdIAw05D3rMMAS0TnqFy+p1902CmHEkDQRnCbBfGOW9pDj7no\\\\\\\/8Ynhvm\\\\\\\/wAQHDCnM7EB\\\\\\\/JerZmfrhv9f4gt5dqErkEB7MJ086WNdewpqgzhJHg3cf71lUscBEb83NGtrqVC3JzPS9wMmRbysYo6126PUKO9PuS49q\\\\\\\/VZ50dA24F7AZfWv23hha7RCwttpwH0H3yUTqzrFsoeVyI2gph6jqRXqWvqKVO0nLYC\\\\\\\/b9V2QzG6OaLMKOx7zk7L6ZIm7jiXsGPCxW\\\\\\\/kIZcRCs1Rekjqdmr7+\\\\\\\/KAqC8pOdCFeiF\\\\\\\/+imyx3Kxf09PJGSm4qOZmCzYwhKIiOXYntGkvVLZB1q0YwkWRG5iZOlIE6tY1jq+MGu1GK\\\\\\\/C1fK4HYP8sbxOML\\\\\\\/7banH5iWVJ6Dzr2Lv+onVCggk7WsuN7YFyWJaubpDw1kOkEqnbSjUkGvii1WuegMrcXCU1g60PUuY4x+YhpAOchp+Qh3Ge\\\\\\\/bBkPB4y7w540Rh\\\\\\\/IKvctPZlBYv1qR5Rq8tyGE7+dJhGbd\\\\\\\/FlY0fvvX7Rvkvak4qQwJaGTWodSZ7+Rl8+EKmuySpOda+goDurBh764DmMD\\\\\\\/jMylsItKlKZyjpeP65\\\\\\\/qv7Fm0KfQqxprJGZPh21g+N7dW99mAJ7Brjr73UNgoJZUpnYZIaJEVIgWEeuWFmRk2mE\\\\\\\/d0QVaGLKxG1HN1DnHZ1gf5bJVZLTN12OEIq\\\\\\\/Oyl\\\\\\\/2KrdiQaApCzP8xluFOkOBy3TzzoLidHxtW8aMV\\\\\\\/0p2NuTAzW7v3+4sOuqFs9ZB7Q51LZa2D61XLjHBKoC6DIB1HuZzq6nzEvt1k+OViNebC0TpHi8c\\\\\\\/OsoOb1T3MJsNBIlsdRoMwN1UGVBGtcbknx2NFxi+C9pfaHY8Zgq8cJErrshpTtJP47kbiMhBkn3mRzEqQ2laqSuZKyIKTmHL+GX9W1SwYDW5kfXD9NKOTaJGjEZUQ+eGDljnnImDE6zyIqwol0NRreaAkN1ApIm7597cUM2lR\\\\\\\/jeP7UDpBlTurhaSDuvDgWgW4Y\\\\\\\/xW55uKv+5MPZV1aXkwdxcYshC4nAEK8CmgbCLzZmUpIVm\\\\\\\/CN2A7+mkcVz1zlRYN+b8aabDr4\\\\\\\/FW5V7A1PmXyIHag87RI6NjBLUlmFVk4Nq5WH7X2DB9SmWUpctt\\\\\\\/dq84cmGcY0nf9LDlsUwYZGM\\\\\\\/BYOIVsJADt9vdsoaFKaQ3Yc6Lq4p\\\\\\\/xr\\\\\\\/7O4uQAuWRci\\\\\\\/p1yBiUkTakg314bKFeNK\\\\\\\/YYp+9t40\\\\\\\/5W6aLUl4AFqZkKC2FJJaLbmJKzL0xFF0J4oMd82Ihv1qFQUS4MLLA7RbpuNg1FNNZ+DkLjwXtVZSB1LaFjXhbN2LDqkA1SEQbzcKmwq3nXtlDzsuHBEeNDyiam4KqZ6eKX2V+mGc42q5Ty5zvKO9+DPQRni9nf3lNEnQ2rVwvdxrZvNJjuubAb50cy9sKACjDJNQ9YRHYGF+ScvxDLsPAa\\\\\\\/cxPmRhC19oMDGqNTrJeOcCvot5b7Ied+kqUxCFQKHRAkVAGaL+qfyhMVnIgA5smvO0TZ8nLvKChDXL8OTDGpKApvYglplwpEHRpV7o+ncW3UpZudRb9pEQGkPtFNhnvrXcX4bSSWpFvkxHln08fsEDDPkHWUr2IPMev9mzz62A4plOfruULouo6sWJZ7OqahKTzJxbwTCWs+v43AXUxdJ0elSP6HmQV27DaRcff6wtzrTLQ6j9EH7Xl6Gb4QyGM5gw41eTqfiwJaUsIdkcOYcVDSCvwHNsbqImV+205bRYyBcCr5rMe3LyuLuD2aDcGhBhu4sMsh87DJCUFHPCLIM8NVy\\\\\\\/VgH3cBTkSY5khnbFbtcIm1NzuScXxoTQ5lP7Gu4ZW8fyuD6DJ1Wtpz1uujDBdhY5Neh9OpeteG2StT0NkCa8Zq3a2wz2w2ruGZ485vcSycb5A9ZbxMuNbw\\\\\\\/8K\\\\\\\/5qcS4vv0JUi2NnCVulhKcht1yUW3OCOVNrnsokldKUUZcjrmMClxQprTiNB8rhIEj++QFaetIhIteUxeS8BjzX0dQUWg8Bw+Y\\\\\\\/3EBxfOdUbXmxhEZIL+asdo8KmiCIU0jAn62KunPFL5BBYRcwAEI+zw0XunB7xWCmkk2vl\\\\\\\/EFXC6OatuIFCvUd\\\\\\\/f05gSa3PJcp7MSZcu8Iw6q\\\\\\\/+uA3Pl4JY+Ry0Qny\\\\\\\/CJtYaP+x01pTcMvEJp2zesWWi+YtCWR4AYqe9tWuFxqoM7ImM5kSqoogtjjaokNx0PUXzQv1Iv9da9tm+QFW7vkNJIZG+bvGdPwM6bKanLk6Kv4AHs6Vqr3cxt0Fx+9KS5QGrBQZ7C2lS+UZaJuQoUwcSBde560KRqSIgq0aV9S05DnzO2QRYjU5dUgPn6flxpyZLmmX\\\\\\\/C65\\\\\\\/RsnUhNymgHWBHKujXQhHbesVyXK52FlIng1sMzZWz3yfjM6\\\\\\\/4fY5XQYclhhexyccMGaL2SamZTEch27SmTJRBfOXt52oNKEYFTjnUWPxXund+iuEmCNLn8LZBOoAmvgug8aU5H1xNC7C9JGSbPEMiXos5xonHeS3d3kQFfSNjqaiwfrUU7LSHksx4QIFSipAEDXFxs6ItLt\\\\\\\/v3i3zLL11ZVnb85sItgEodj8yiMS3RH48ERo\\\\\\\/qf+tAX4PhkdWGhbtGDvg0x2iELdCG5kEeESQufoLZR5GtuUdiNPHenKMWrDTXyosNjdBwp9N+INKGej+2hMJeadbuoDCli2CnIbt6+9VKRaT\\\\\\\/gB\\\\\\\/0F0jPSkH3zY4kKV62eGhqh0glUhoN6\\\\\\\/Aeb9siNwQgk7GTzBTfa6\\\\\\\/ZuCPX7uhfq9W4+UGxEywll1DZLGlAATIooHy\\\\\\\/BwoPTAls\\\\\\\/i7VRPUuKJK99d+EZ7532lBOqqUSsH2XOsiiRZzMgtVUDySnE2OeG\\\\\\\/Pkqi6rLxFXntxFh4cvMvEHqvZ1Sfg+qMHHusXixZeT3qaE7EtrfWiJc7lTbiG9b+Bu9s6Zz5qr84xEqf6P5yMxtRIyM2ivT6piAq6X32tZXMcfddUuF+XvKNiOLea2pLTmEtiYfR08\\\\\\\/DapulzRoDIlJ8h9v+L2+cObmNNra3H1oQG1bO2SfQjl7lrdOkb0JvT\\\\\\\/NfkbUzuq5sPctO7z8AN\\\\\\\/OUbcIETiLouLzrmWD8F7MpHor\\\\\\\/KnqTaOrfaM1EdNuTE6xLNnKVnNxMEE\\\\\\\/nq3KmlBSzTSSSsFHKxBt8K6CWzaOwpUT3Zf0PRp2KvUPEh3IQ3pzWv3SGL33F9nj9X5+lhBdxKko5HDYKCTaw5WGimfczDURVB1yIQj2QMAkhvBKKfqBVgm3f3ZSGQZMFxVEQnvjr\\\\\\\/pFB62bWafOHLGv7QoX0YMW1gGesojAhuzQVoRU4+2NTsIc3rJU2yzY8Syoek8XEaocMdErHgkJwICMy8\\\\\\\/DeO8W3RlP1znY+7QFMK5vkDSCwzedllGmhJ0Oe4Vx+StElOLpF6RClWRms8EiBtKQHMqCIR9\\\\\\\/xgQ1Y33j3rkOjJYVzfA70DYYYtVRvP99IzyKnfO7gndW16HVvLXMRKOtppTE\\\\\\\/kNbuOx\\\\\\\/BjIXam+KNSWUaUqmaNjUUuEFD8fpU6Lm75maPWKPRGViG+lRYTT5THWxh+ndC9qzOQ7FhjKlb\\\\\\\/MjXamBidSpyIcxFyYmMvGlq4fTDuSihO1JA665HGYJwBqW4lY75BB0D68cA5zwJAhlngoe9DFY+0uusKcRo6QYsesnKsTLkcITvUZ2Ze9u+c45p9yKRBTKttA\\\\\\\/qpBDlNU0C8spWXijS3+FbMfoe2BhT8KFWTRgQFHjjInKHsRGeZGJrPRWdJOUOAPFALdRkLfLq74u6GmwNx8sC9B4tc9mPiFGJwOHq+ak2RwHSo5Gw1BMHCTBcyzXHLQaPA9QJ4xOnCsAwY8EbuUwuID22AS3ZMpWn\\\\\\\/dqhuo2OeiFuZdpegaVDr2S5iuBtxlRy5SmJCkyj6KbJDtDZjP+HpPMSl+6Wfv76OjZBZwOWqc0A5EWVxgI7aiYi6wKcps2hR7uVF58nWleRLNUnZ0Ul2BmT+IbvN9kA\\\\\\\/7tclDQjtDXmAfxQ8PBYcUd+mYdrLo1DG7+bVgcTCdShtoDJbn0XgZhV8ins3RIjQyrQbFe0961CR1rZNv9PZY1oNAd+19ss0ZiW\\\\\\\/f5ZDIJN+OJ43BiXXbs3iCAsMStRXQN8AH4qBHVD5MG9LrQU05PEH5NvTJHyRn4NraIY+gT1AHbyEJzDuNQaqIepT\\\\\\\/huHkITLUqnQMqEYBoI6nRGB0Vmhg5wAj\\\\\\\/tYskL0p6aqhinwBdw6aMGrSSYifkjHI0L+gQ5NFxKN\\\\\\\/W6HEkgEkMWsxYvAAtNjqtu1X6+5SwH9iEbKJAkxSon\\\\\\\/4tknBwfkMiEcak9vaROB11mhIgfh32NCgxN54DG9GXgF8trFAADrS2M010AGUQRpFIdeMFkN+9XTc0FaPX4Nf6BXiYEt\\\\\\\/8aAMoJvJRQgzAgXpB2VS0HC\\\\\\\/B1I6cHw0TTNUouKOiqHDATGr\\\\\\\/2BA1V8SZxXhGkCZ2riV6fXsI+17T0hKpTklawKp34o7bsrbdirBXfhULKxxOPYAe3Wnm+4wW3\\\\\\\/pS7\\\\\\\/190GkOIlNJzc6Toj1Cj8IamC3GVxDiQmg\\\\\\\/u2a90E+UbuVaiYsUGtm3Y\\\\\\\/ePKPRbqdDAjEUlKygIqIVMEyuCfJ+gCcsiYvOSHeeIVFCdayZS9wNnNoVww\\\\\\\/nFu6IBl0LtswTMF4DeyU9+ahBCw1z7YSOZUlHgdZEC4drHTEpLjIrGaErcIDbO955iF2N9AbNyMOLrWr5UEMZwOGPQAhoeZv3pe3EloVdoPWtpAFwpTEc1M+ONLEtXbAQ8qDEqBB3GxaFZAxu2Boa5HZyDEUS+XQ09+Gthn0iOHpTYsGXGvTltdYwDbqmxqddqdE+VExyRpGJCNwMZzZ\\\\\\\/rjyCvRLAwshFUU0LBRif8i8JhhgNDsP5X+vuP2EaYmeTNeUqxQho3+M1OUiYdsSb8HDlyk23\\\\\\\/9uaVrhxrkpiyLmge0fMDgh0xOqUNZlyPPPNQmbK1Mhjg6CgTbOaRXYzDwNKUIVAMYF2notqufXBFLkDpGhUizw41iU7xWfU0OtR3\\\\\\\/URwWfVLqoWPhR5maWlvBPYmh7\\\\\\\/p+LnHKPHmOOS+m4m\\\\\\\/zlkxTWQC1ZBwEvKoiT2Yj10OwJpkGaeEBisW\\\\\\\/6dW33TqlaQ5uiVe+RoZLF0IpJu5Eqv5Y0az5eMDuG6UqIrjT5lmrPo+Q6qNSfmJirfqOaSe7AI9t\\\\\\\/x6KZm0KYzCqwUXc03OBhO5xk3Hd+czKZdJ44sbU+SR+8lXp8fjZm+LaH1XdPb+n7gEJ6Kx0Ynkv2JguZeecW9d3DC8OC2KXSauaIy2NIH2jvr3PtH6UEF6xoecmun+n9peau7yfBSlzUsoy3ZYWw8IYm\\\\\\\/kHEBLpqkDXf44yXqhyQyf3YZYYRfOGunnmaC1PnKmrV+zWczgxUWn56sHsDNnihiH14goFdraZoN5nk4JZkeyaXYElhIOXkZ7pIftfKPtcsIK7JsSFOVX3RVxEp4mLe891ah8xzWhp\\\\\\\/fCMSLIVaEnzAKb51C3pKa2QQa6vCxtcTl9vPDvC3CCh2SwKxX5KBUEH5b3ZdJz6wpfs7bOzVK2AwsRPoBFykP1+YQxoms5pQD2ZmnkX8kNpWMqI6dKlrwhKXHSgl3fc3nruYEsCDFbIjrSZbMfqLXJ2oLWqvduNdyGqLStu1E0FDaXrEyB+53sHsu36oLrqW\\\\\\\/yXsH0Ne+7ci0\\\\\\\/ygMOeYQwdZAImTxh0p6HaaqlpPa9cSAfuyiUk5y3ZCXVZ3ONL5PGtf3rfOnKkoQf4t1T\\\\\\\/O3ydf4Z4AAedTrXYe9k363kHJWSdBS9e1qeBUljz2Qw2cfN6\\\\\\\/\\\\\\\/yB6rqvT\\\\\\\/gpfZznvBO1tmGA\\\\\\\/4qNoxkOYEUOchjBbtiGqDo2GeUdKPlwVAPR2DWXgAlivv5tGEISqrHzVRco+CPwG4EG7GBHr51o+XVoLriG2pgoPQT4xLs2AvGAIbpm64vpIzLqNL9JpITqntNWHHDjZ1XOlWbWsHgxxo3ZMT9GlUmxC7w0YbCMl8BX54ES3\\\\\\\/1tGu3OdF5czR+5aeKgpNCl3LQIKb8NygcxQSIHmaeX+4OSkBKFlG9lsY8GD6AiaPk7XBwntVkbxpUOHN93Se9vECXvKziI7xwUiuQwwUlwwXEuCKH+z9XelIx0PN5RpGsbHm4lqsAiwvBhOIfkQBqvscvdR5OvWMSKKBNY9wUDPTcrcW2UQCc77AUE3gDeBn+vxWRJ9NJsSQb6mWx9\\\\\\\/ZNcHyn7R+w6lNgYVB321blQRXwmn11I11+bBa5PaMWPWYFU+DQG4\\\\\\\/UUvh03NhUKCGcHZIvqroIhsC0YO524nTyx5G+atsjvt2ywJ7Sytl\\\\\\\/2oGPtgc9IOY9lLqnyQNnXvBB0AJhUg4\\\\\\\/Fm9bgMxLJIEcJabaDIdGf28oxS7f2D0ETKXK4pGdGrIsz1JJ4OiOWsZEQVaFhliAbuzGK6YBS5vqzGNycdNgJzn9cTlMkDwE4m8qd0pEg5lVIyTk\\\\\\\/xEgnEIlmzMqziq+o+SAGQ\\\\\\\/dGv+bbnNXcTMwEB\\\\\\\/V9qvXcxVN7J+otTV5vZHvwrJj+x6fT9ylBoRFDM\\\\\\\/MxY+F7qrddwIut28GxAbTg2h7Yi7w3Lr2LELwO29g9d+9OtucY5EOkq3c8vczV5xiNcmXhpKqh1M8RD50E1b67hhMauY8DC6NeeggGaT5ro5KI7yuTYH2LujR2\\\\\\\/k9RTyWWuSpJc2iZS+iQMMhSOQ9FFrVymq2pCAt7c1mNGnpyV97bpnOt91g8641\\\\\\\/Bb12oMNaK\\\\\\\/T8zEAuOsv5qeeD6+MZWj1g7lIXAJmhkiVyRODwYdT4ce971yz0548pWJy79Y1OwpI5p8X4StXHn\\\\\\\/kEJihXd6jTWhtr+Pk38Srz4JlnYnJSYatESTwAJaWwZNymeLb5KznqjKiqbwnBqvIhhZH2vVxulwayLlam7VvP\\\\\\\/I4dNc6m1wSriMNegSgCkvNrk3M8BpJfl2YfZjY5SAIZAX26SnWAwGJTewbHlCu5QmCVWOF61PspeYRV64TVhE0n95BKoTIJkrDobZcYMjTW5oH+4MJzIJ\\\\\\\/DgaK+sYo38a4hN0tfv5J7VIfEjMmFw8Qq5dWlk+sHWSKfqB1OkY+\\\\\\\/ppUBxFK\\\\\\\/JiDhwm+AOchEINxR9EBb408dtkW8ImZtpIhd0VqytuqyNVpYtR\\\\\\\/sVAq4AaSZfrhgMLuD5lun+fCgDlHWBEm6Ph4PCAqtShoTFp\\\\\\\/Xu7x5jweJ4gvGTT5bTWwgPnH4I+n7w0twJWRYJ6NEfAfCC6+mHYafVa6ZmL5HBYhSztNigsbgTj9Yp2OzUMS64v9TGdf3QcpdukK6QtaUD16a8n5H4KMofi2RnWd1eLfkzW6c9WnG4Tl+5Ibw9JTnYpz1sT2jPEO1ov+jIkoEbOJat9rkdM6+9l3PMn3IqD0DK+niBqs5cXdiAVkv3mDNCIKpm4ItbklTGZ0elkVsayxg9wAfit9Qmg04GeVBcilScj6w1hilmxqmNcvET9qnhgrfgv3KW6mhMlf+EKgeVg9jj82zTRT7elOdPI7vlugbh\\\\\\\/pSL1WGwzwK7GPJkT9Ap5kR+fdf5sEJXZJELpucknKwNNyvJrnLvNzRgqwJW+EuAsTId3tC7MemJo4QSbeSYqsGPIrGOD6tMh7KwsZeULsnWzC8iSzcOeqYN6UFUHtp0bCs\\\\\\\/X965Y8R\\\\\\\/uNzREMlAQLhEarOVQ\\\\\\\/IAZoHa9tSJ3GJ6BAA0Kwu7ZQAGZoOlw9rqvEJHcewgKIINO3KmjV3yP7O7KMP4SSq+kDh7Gu7i725rI5wSjQdGwrLBT7kPYJdJzejo5P8H0KiJHaR18FBmhzdAMm4zRpQO6yblhs4lBU3bOeSs9F7GJGSNsMiqf2D2jSTDqKvAtaEIL1zHNxeGEqpUovMxUXvoYGya2pSS9pg6BDE+jM6nn\\\\\\\/HHaGYDU+2BH1LXYiWx+VFCT35hKrwW7Vb1mZrDDvqhUibzBQP3Vv30UTjd1qhvwDRQGac5zzJggBovZ\\\\\\\/Q4dC+jLIIw5xROnnte9wC6SJIro5iOjMj2xCJRAGMGS71ULU71UpXrLxmB9qK5XoHLDtE0KDspPZJ1a8YadWpmA8XV2mmBSFz6soQbDh1eLEqdwlA3Dlw2PVyJjkZDX4AEjZ34neCVaZY1TyDoaCaR2H5tkLnnCPFtcqm4FUtmid04v5KdlWooh\\\\\\\/hS6xhivUrCi2tn+\\\\\\\/gvIORDGtuH3W6JmAqGodJuzojLc5TATu5g05beKJU4GSpVLG4iwiNX1ekprQBVQepT\\\\\\\/8fj9k6xKKCYpkNOM7d0xnJhGjWwP3NH8Pfn0zQQVG\\\\\\\/SiSQNO5gApGViscQ3mxazLWFTfw4G1+XmlQ7isVnoSigOnVxnvNz3dbCNX7rhA4haOwQMbURUCdNzNzztOsNszh1+RB3galNXs0dfPeMrnScCSZEcSzwbst3g2QEJrElvaKZdg2OplP+hbQiCvMWqGQoEuIOuB5WOAaK2CSYKq\\\\\\\/tUDWS93+11RK\\\\\\\/8oFy607yQOIJs2BBH3NnQcxyBiRUhCds9l2luzIUL2lQ6pwx9YJQ6gsTmXdm5EajZwNO5YQS0IxKLj8CjTVJrgwV\\\\\\\/uL4uAx079IeO0OfAtJEmjfBoWkvmX1YHH8t9y6RttzOMcXBNgRqBi0D8fGZsNX1QpNXcRueYiPOEXq2nkKj9jC8frHosGKZSbDKnhzsjENbvREScFrcMM6UcOVbh0Ltwr1lEmXci++6cdlp\\\\\\\/KL\\\\\\\/HVNNWlqBYRU+vm+F4\\\\\\\/eBittkHWRG7rQMbs21LR2MQr0JY6s5w30uiSIDeMG3WKG4E0LxmiuOrxoQlPmav2\\\\\\\/+mA3aACHL7zHRUjOtsh9QHm3eMi\\\\\\\/69QD8JfnJ9a\\\\\\\/acI3njA55nA4vvscxn+uuI3e1hJXVTdhs6RNVY+DMgtRMSeD\\\\\\\/QiFjooqTknTuFxyyOFbzYyxZVzl6pIVp1maGqqoy1ilD+nLNzmP9av0qrDbDy0uVsHM2k81Pv55fh4efocqUomDN+\\\\\\\/YKTYwiYcV0+Uyz5BPeXmGwNlOsIznx6f4wsLLOhgLYbflokwSZ4X8eqiXLKBpD4m2\\\\\\\/SoJH6\\\\\\\/1skzNcwCq5UQFApnIoK+8cBACA6W8QDVcmqPHF1i2AUVqHDvAoXWEIvTgvxXSJjjDCIukDte1lx92vYJnmjU1ATHcegrhhYmSvroBnNPRZdZYpJn7CCF7GiwojxsFRhZSBJn0whAV5lzmJHBzX\\\\\\\/8mafhMmOZplHNHwsRmYcvObcejrxHvfG4aK+5O8mNp+JDTpXekQGlXhJfcWFz3LSuJfF7zeXeMfgPawUcz6nU\\\\\\\/+9KGuEyK04RQoQfDicRsAMAStIqhG6Ey9p3gayL7uiE7V8Z3Sz9\\\\\\\/D9BO\\\\\\\/IpLqcRpqa7TCmCSToFq3ITHn0UzELKC\\\\\\\/LV+Lp0AvIWaSaOCBaFlY8HhjqRX6UmMb2EtB74KnTdXoJxIQqHHXsgFiwH72eBcwh4J+ziGpctZOIzpvznxX5Bl6rD\\\\\\\/HY\\\\\\\/QzyXxFkJ5AgbklCTSK6wdFVt0zlpo+Kep7+lK4Sqw4vLvth6J+sTNIiJoDP8qsv02NvTSUbjl9FZWeWZIiKCl07Te8+b4hxqFA3Y66gwf9D3IZU2i7K30em+mhJoEphFrFp9JsGA6Sq1EhC+2u3eBn2UBvyJhZEc3uFpxck1LPbK8jxlKdbPoBgfv2Wxij8zvici2UeaQWbkHIxNri6eRuvQ8IFKJHgLX\\\\\\\/g87KGnKTO+CZ0Dx82wXxZAm1k97TMWdm9vWrFVc0gCYcGuympJ3diQi0icNgmGb+CWWTbPiymHIF23\\\\\\\/11xJMiJvcb6x2txz0olpad\\\\\\\/XHv1NvPjARAr2cJVASQfovrM6pUHS+dVPl13\\\\\\\/HUKq4SfNB1zeDggyXtcq+a7csI0doqCjmb5NtLvZW+hBAMdKbgrznqyLNY01+XJFMdS8u0ae1yrqPFKLu5uIo2Pol1TDmpPI66is6\\\\\\\/q5yN2BM9p2jP3DRdwkcH1WX\\\\\\\/UmEOATGdI9nTPGRse6z5kNLaOK+O\\\\\\\/4RExaKuYTII2LyciMPJ6SeBWp\\\\\\\/jGTdBjpPvcH2tths\\\\\\\/xDJG\\\\\\\/fBzC2MEv6bSa+UBkvRIZ+tzx61wo1nr4vwQAKRNzmJw3O1e11oI20Wa6XKgh95CB722\\\\\\\/2gIzCqwe9GfXBBG36AObRNi6\\\\\\\/D67\\\\\\\/Y8pfwQTzonkKs5XeMztf7E2fX2i5g\\\\\\\/85zYwiWcVB57zbDl0dl31tU\\\\\\\/5dZf14t15AqmhaFNEAnOhepw2\\\\\\\/0It08Qnl8lTE4aTgiTpPjl5lVirgQap98vVFZzB1Fi7jY6Rb\\\\\\\/Ix8lVzmLHrv27a7aDS59TwQi\\\\\\\/5D9REKQIrmtiFD\\\\\\\/vKViXOiD+CsePutvFeWTGjQu5Ph\\\\\\\/ckIzhDhFv65zW1wHkfo4DHqwQVKUKRSDXKJiyLA9qtRJJ+6qFJvBiaPO34P680VQHmdMOtuJsoRwACOq961k290KnS1y\\\\\\\/mIKUWxi9YprS8SnEE94PWIZkSREr7N1AqkrmGEY+rL\\\\\\\/2juT6yMxU+x8KVMZHta6GU8pV55VL0hNALhuA2ldcpNgODT1eaePTVWQHvGDL6aAKkyeLAjtfWcMAn7Hb54AnSZLJJYTzj8Gji+r\\\\\\\/EU2qfm65FBsTDMndvYuQG9IFrAyFhpRS1zfSWid3S\\\\\\\/L8nyDQQ+0wS4EPb1YxB5PaDIcSAFloPcDwuxtAZaD\\\\\\\/q6JhqdblRHJF6L+qg9NH79En\\\\\\\/4A6LxKnA1ep2kml+X8Jrqz8WJyY62tcl+Bh37cMcF8cTh57QaA6SGtQGblB52Jmb\\\\\\\/4HbQluBYlzYMnbBz6vV+\\\\\\\/TWaRcSvaI5gFWfw+lDD5uovtbJ6CasuBcCTWNGHD2+CC2wVZbso4Mo4a9KYAl3uxg9HhPF0n6Un6zOCGFF\\\\\\\/15Ynj2tYCVZ7CBfM3Lhd\\\\\\\/T82Q18X4XTq6K0k7tqHujpeZNbXu9bhXwQ9ikorxJPakJaS4TuhyTxsIHgsufOHZ2IcTeq6JZGEupt++sHF1DD2Op419yhXoF9RzepWD6d3c1Itzm2Fr3NA\\\\\\\/AFG3Y85oUB7+kxJ56g53aDZTFF2SNOgSXZJ94VixpIuRHbMNABqWuXCmznX9m0NI68y81Fv5MCPS6K6X5UYY5oyYmd+fC7pUsUj28ZTN6kTRYtgZgHBCxYsI7ClzCyCppX2nJfRZcSY0foxZw2rLuKPVIlcXkNXuviLkvx6q8URlJ5Ti4wWpNoDMHE5EZI8KtcwvyzYvClY8DOZkICK5J\\\\\\\/U6AoDaoaKButCfYevfwL+SNvQW49VehcRYfkPhkh2xz+m8bE+b2HngFAe1XuT3dHZVBUtYys1FKOmUj3ps75NpRxkR22YSYkysi7sB50ree26WDJF4H3Y0xkTN7UR3XTA4lcXud+N8spHqLSsdrwqtZ1IbZnj8x+1\\\\\\\/67Uz94PbtYrtbqndsFeJdXmX5Ii7rqqFwZFuqI\\\\\\\/wvtd474TO0VPjxZFIvmXpJps5kgti1h2yCf8xOASZ4rN9aw5mJsXxgf3KcIqC8PS97JLaiXeCM4mA2tAtHulArKvvALhkS\\\\\\\/tYirLoRfMiQtiT3FC8zZAbXA1cNdouJd+G7wZlxV6VRg8fdnNB1QsQZlbIdIeW12xUQxr01wL6VsyeSYhTJGQNAhT1XuDhZ6EoPLXKDKEYcA12XKuYx8LGJwQMZsb3bt\\\\\\\/j5ppoFPFNR2A9AmLuO2xtQBpbDUfvjY\\\\\\\/1jEfm4yyun+MWe5xV2+\\\\\\\/9IgWU6ZUi0IoxLfZrJt66iDJpzpFcAqCcb+DFDSVUOB8sf\\\\\\\/z\\\\\\\/hHAGDkIY+TgYv9L+nIeVmTHhWWkJtahe4nEiF7HygLK6tl6j+8xWA0ef021CkurwShBa3SPmN3z5v5Jf0GwV+TLXEjGP6z4TbAV1Zid44jkFBveziX5Ps1H355rcO540BV5wnprOMCKnGTvzXCIMoDhYPzGac\\\\\\\/MQVAUZ\\\\\\\/VaEhvqgaiZNQMP49HCe8X2K15O7oWiv7YlW65pvMGkwoySvYUvREi5zK+BMBzl8rHkJirSsysBC8\\\\\\\/gpgHf8ns+qm2rHSrUgYz5vnKjbhwmxT4BMmyHkEM5H5afG\\\\\\\/D2ms21b55C5a+zNGSlBLxZf7nLZHJC7jLh18+iwiQnXePQTgxtvND9V83LvF+CRHQ5LxnPs7QQHTa4hMr1rgX8kRegwq+6o9qih8GZ4YfMOXnk5aeUEZ2idd7hdc1mNwsm79XSwVwoM1uy8ROJNXSnaTNWDoPwgTcaYSgQuFZW0\\\\\\\/qLhbFmFy0s+COjFO7Tz1yoXW\\\\\\\/dJxi\\\\\\\/dBcInVv\\\\\\\/wXamWEuak9CSmy\\\\\\\/U24vGvxSovWMecmaUOCWLUCk9soOIBdizxZB56sWu9W1PivO9+9Gm86XDP8hfQFDF+uz5cI\\\\\\\/osLwcU1G4yLAZDheT01LiA6P5eMmbSlyLAriZk9wTO\\\\\\\/MnzkVyri01lBPGxSVZ\\\\\\\/OAZHAB76qxuXsIQbnLT0ZGFrSjNNcWNdae4w5s7CRJRSoiTsdaadHv6kZ4QWKx5H\\\\\\\/aGkgy9SN0zJhnDiCIggOvPFXLL2QkyNx1j5DRSZ3pAR3I1VNpY0D1vRXPqzEgG+O\\\\\\\/+r70J7LYXFdV8VUY8zbL8AgoHMCeA9O9W7z7mIK6XhVH4iWiOb0pF2oDpl+I7Tb0kZnoDKub1ffEX4Ohd5DYcJtTB0N20NnlnhH2A1mOIi9w5hwWahgPDSkS6j9SJL4foW6cuw9XAH88P4xHYuvE+fxvTevnVmgJADghzvq8Uf+YURoKwep6XyR8jqoUGDmubIGcStzyIwBuZI4OouNnKed4M4mkgMVDanYMBUQ+HI\\\\\\\/FZOOVQmfhMNTz0SQUb+NZOYehTEcuyMUz3uckhWPTEBbTUkcRmU3Gp3eCkwlqErDupXK6FEXZ27o4JyUi4DiCCappb7OpopFIY6MIXe3\\\\\\\/1snnO\\\\\\\/Wu2I+\\\\\\\/SJFbch3h1K3wEv7nGBOTaD0zT\\\\\\\/poaF1Iq1Crj1QhoTgoY6lbBbHNFLmg9499KMkWp2szivQ6B4LKqXkopC3cY76RWe\\\\\\\/bK8EQzVhTjvJYnryYgLNoTRLFst6TUstnrRGA8bFoory5ngE6+5ZjCDrXBUirF001JZjGNtVM0FRNxFgJOfFgJ3JRBoLm9Ei8vMTZOHV6mICmar4bt6y1QL7m1h+wboCvBmnZBscBkmUa\\\\\\\/rIatPqCRFtKt9JtnNXbrNyrRHB222VRSIBtjQIYVZes5AvwlzzFQCPckWwkU4OGq9GHxWYByEmoVaAAnhR3jdUgsnqMbxCXMZHk5Engw8m4vgQ7JUGds0KenANeb8t00QWgkl1wY89mI0l8ApB6VhKvGwl6Uip3A7qPFzQl4K9hdhKNQ16oUrUXCmlu4IXyA2bNOsDlLAYeYFcaSBcrN0M4VKHjAdf\\\\\\\/a\\\\\\\/Np5FnfoPYp2vRAVSBejJKFEVKyIySdebysUo7ziN3+kaDjhKJXo3gIUocdbTjrhu9g2QGZ98vCPbzKv+3wXdp26WAkA8Pu2OfUvjdx7ExUcudSnVSQPa9DThJKuzDXMmiPcRRzmr9NJPThHFSAcZ1WYv3QD5cV3uKAEoCmaaFHL7eWOZXz4s3DMB+MNbjjsbGVrSjh18Fi0wx5HH+KLgtEA2kCca0wFf+63wys+PwBAvrXy7iX8vG6uLh1688ZdG5nqLd+0i5vbdOyqaaM+4TS0qwHpIziw3yFMUoEpquz6MCbMz1vC75QPQRuC7IdM1KBOLwxBzpfNa03Imo4FsdHk0xZGJ464lyxTGRwCNjchaSMeqw5WXHvZpsjL7kH3VhBJLX9J8YBAl99LhoAFqpCtZ2CXsC\\\\\\\/najfNDDUuBoL8ikeqSTxKFc9ph7Uqh+1Y0z4XWGMbflQuePQaNr+fq9twD1aqR33a2fOhNvCjqobEw4VZCRKkPPHOcRKMZ98ChR+WhuXvLG0cw93jgrxL+k1p2bx1Cn5oIY85AKxBNk2Hu0pQN56fa\\\\\\\/8MV3RilekKxf2dOaIdipbGNbZqvAyEofuWVVKsLhKdPXn84MuQREojw50AXZwsDPlOil63ayGA6JXT\\\\\\\/F6ZddSfbU7i5UkXo2QohCys\\\\\\\/LBEoem+9e7raZyGkwHzU+KweNCqo+EKShkBgK\\\\\\\/gSS4nQPisDP58Xfm3wo26U9M6YN1ajYX+UXg2\\\\\\\/SFF2nwTXrQnEcVBInioC8wDm5YDjZrTGudubDLEU2RYwJIW6meZcQu\\\\\\\/NU+4wzR3C\\\\\\\/R9wpQ\\\\\\\/OAi8+dNjqPg0Mh8OQBayFBppvfewOvIK9oaQu0e0+2wwy2IHKj\\\\\\\/crFpn+lMEdl8S62jhRoijLs1FmEAPWSbjLI6Pn4dDnFkaPJcmHaQujra6exJwDmcaiP1VvKqZUI8gA8aNs2FBnFeVCvubyUsDKv1vU1l4UBD5+rAdDWsTx3eZ1y4hX6Kxq9CUqWUBoxokxt+Z66+23LB2YZEXGhbRMUcpNfTB\\\\\\\/0qM2cMwI7\\\\\\\/Xzb756odCvFIWI88Tv87l5Qau1NdphN6skHmGd6tcCC7IgYijCW\\\\\\\/7ci3sZco4l\\\\\\\/2+V\\\\\\\/f2SP476wg58kjBGCCFcrdcsZ\\\\\\\/5ZW6+KIBj6ratuOG3Wo594eabFLju2DS+\\\\\\\/p+nbKkYhZs\\\\\\\/tZNXhby0PyCv7YH7BKMIicL82C6QvcEA3hVWZS7ye8IYSdiouMJM3i6QyJOPXlSqK5rcf5qrGz3VpD5qseYZ5gwohrv2H\\\\\\\/7KZtzIbt9nXtenZlEniy\\\\\\\/a9fQXTgJ5Fso7i1IJe5PoeGinnUfgGDl69Jm8zkjOTihrZ0rChVq1W28YpbYDssytirc+SMSKXCgTuCkIZH9dyrKusKSdZ1rnnIExaJ6TSAc6EQoIXzC0J5RaSKbL7F3cH4VrgxZg219qKVE45GEt7ZwHkrZ3Aa1BoAJppmK0bPliMK9bXvSVbSZO6zdwOJbQjNj41Jg4SqgDicfI+eKwK9WIdV6V8kEBZPStrna\\\\\\\/CUCQxyGR72eRL76JabLGlJWJQNl324eeIa0jKhqkYis8lkuQS3vsWOUEfyJ2vmPE1mTLDYvew9Pjs2qzYqFFcDhfQvKyk0Irc+08a2rC\\\\\\\/bwID01A3+uQ+WGIO0jyP9paIGSMVEEEj0knkapp6FSoYop4OI3c1m4+j5lZB\\\\\\\/4mAsz0UDrfXsdtCKoTEItJKj+zB5Jlmsr7gZkmEO+YCYNc36a2L0\\\\\\\/xslTUVzVIhJqRHyGFiLbPYhpxnPntDdxIeAfyEPYLSpijV800pCaK8IEl6tUewxFxInPVYIzz8kHzWclrcscV9ZhoRTfmMiMl93WT2e+s\\\\\\\/RUVgDpuqumv3vkTaI1ZMSobHo+Bh9WFcTlFwsc\\\\\\\/0H+KR\\\\\\\/hFsqpbufvWFpsaA4FCPkU3sh9MgBeL1AdllflqtLa3vRZauvQQPwcqiHVkHKW3WNnaixPs54QeluLyHotZMGDLsWHq8go\\\\\\\/YcrQgo8IWDLMSit1Wf+oljvl6sXTawiFUUT5GRTfFfCLjZF\\\\\\\/r4aCznwM1s4rbfbOVm1p7bcD39rTq9+WkmOVcuksvE4q70OckbzUt\\\\\\\/jcLaZQWJShur\\\\\\\/hw1B5TqMhkZua6TdoDGi\\\\\\\/1EpJOhIJOtIEO3nn9nqs5me31XT1V3yDJ4thIDB4eLmTZORMf6y\\\\\\\/UGqD7dZVrJ7zGDp446Uvq9SWMUC8iRqDXRuc2luSVhOlo+nghHEyM7yJekYy3Cg68vhfh5A7T3feNOzr0l+an6WaP9+iOjEuvcS8a1vmn15J241XQjQPSav3OeR1NEvnTVg6KRN1wo7r2P\\\\\\\/03SwlBH3LWfqQ7uwEn+eoabm564FBob10Uy\\\\\\\/gfWvX21xwYrNCFfUXt8ckfyfLKPIIb1zJbbQcN5JOOAB91uIOyC7Ie945M3qZAfzBxtipY1UzhkBsMR7oIOxmn4mKmdgKTzi16BI4BtD0E9FBi+aMW1a8JKkopZ+zB7t5SQmktkEUCgvUZQHMWX7LtOs4by5c4M\\\\\\\/MjEv1z\\\\\\\/WMaxrz4XL0JeO5ZkItwq6mknXGSoM1e2RqM1ksox2BHjuNuqZN6x8w1aURk8xiWL4adyohrbtMkjY2PKxSRCuoXaDIvjXYc1g6P1VihqwI6wAkAMnt+pvLtA1EmQwr5P4Wpf1hmw4\\\\\\\/1lGgXvS78bTkXuo\\\\\\\/KjKTsSa3SiVZJYdIyDF4vAy77kiZCyy8wvbmKxAcopQBnAF9GUBTgp3zTklSAAcM2qU\\\\\\\/SCQckGfvJGWWQRVQiQhLbjdHFY0orXmweRbEklioUAjS7yVYgdzsgbRHTXqpxpDyD8\\\\\\\/29FbhA\\\\\\\/eXx8WD35cZflxnAWgBVkMwqOnxEgjWQDvRFmVPZrTnWIdEmminx+pnmwGJYxTKMrUqneBFD\\\\\\\/Cz2aU2Ue0WDdH9YnE5CZGPTyD81PYBSKnPfH\\\\\\\/Rk+NxI2eMDANtzXtfsNZgHFbZQBIAKyxQ\\\\\\\/zZKzjCMrcDvCS2TwE7e3QNXb\\\\\\\/+PgbPDnJrtWe\\\\\\\/+1u4kd7k8A6cGcRKVoAjMOalihTPJ0u0fW411HhXhVdnLqTbFeTwlAeWFz10oFjkuCZPA3L35F\\\\\\\/yJbeEWPCGKQzNDlov31ZKemlnUY\\\\\\\/2QGeq0XXy1IAIz9ODgSv05S+b+Ykg2LpLqYeSHdT1kA3HjkZHXUIsBRDsOj+rQMk3M2+\\\\\\\/4lo8hfsqmbM2pe4DYH\\\\\\\/Ad0a95I1LkAUcptMzY25JdoUxUa+62aY1MidtvIJvsRdw9aPNSg9wyEmvgp3tggXzDELOugIWr2XckxftJl1h6TIY8om0c9Pv4Q8dlglzDv1lKrk2YeXSfA3ArxxVPf79zGUz17RFCDox8rE5KF5dfUBUgcHJUktOXX0Dy9nbMQ3PQY1fJ6+4FVKurxfycMNDnhqugCYF3DrYI9eoT1UCddJXDKVyOhg3xPWRxZg+osBG1lf8VGUgVefzXuf7Bib0SVNCINJLZvSP+3EUSD5k1lCt9jGBQxSUwPx6M84DmuZb+OMHPJVx7xz6uIukS7+TC49GHWL38A6liOXZjJ\\\\\\\/\\\\\\\/BD20s6J\\\\\\\/PVpP9HW5KdkSbl10TsxAH9FHATKkMwEjEb8yWaBbiMQPa0EXCE8RePy5ELJrDW9rXk7wCM1UkYVwTNhWX70\\\\\\\/1r0BzUGpLb1RNmAn6hIKU9mjs5sybTdBegs8KbYUU6wX+qbixXTtgS7CnShq8JjfLnHb6pi0235Wgc5Lev0ZoSgzoE\\\\\\\/99xt\\\\\\\/5vEJHFHVCArBatxd6ceESSD2zkZpDAe28TkAixnWpRyIL0m7TC34EHYTtqX5i+mIP1jUKlU7CZRtmdx84R7tU8rju\\\\\\\/xVVZmlkGLBqWq+ni4+U3\\\\\\\/mNet4YRhSSDXWi1OAFytMhMyNL4tBkl8pZEiXZjW2LwwNkxwKEBkzAKlERfBk2yVR3VOglr8MPBu5BGjXn69ff3RgGbCL36tgr1mOF5hMn3ZIkUHTVuGvF0orLn5Ox9OTaaTia4VEYs9g2hHqulohcBU6MWYevSX2SC\\\\\\\/aVg0LnJz\\\\\\\/sdq5u8Hzt4YxIbuONX13eOdH4f3qmgv44RSkVskW0UCUQZ7aI809aLGbKUqY43h7RkqdyWbf6NB+gtXNSMUPpDg6mfxqgHx25ndWn7J6FX1ccURh8ymZpXuoiOOlZHv2XmNBPvA9ZBGNUfymNDSuRZtEm20aibmVHdSl5nqZ0JVkfnR92JoDdBV5KoqK9znj\\\\\\\/R31VEYxD1on2HiYPZdH\\\\\\\/CFj4N1d7WXD2nyWOgwDAwfuedY3ttOjQyPnBS9R526JqD6mhfkHga5GtQbf57EhuxYqHQcrRXDDkl6i9ImBhUv0ndLtXKsEf9Jv4fIPkdM\\\\\\\/R\\\\\\\/MlaLMRDxvTWZQxB0ISZjcdAuXuKY1jbEph36bJCekj8YeczGfFUf7pr80pp2\\\\\\\/TKrdz4gzHXREJIYg2d3YWXlSXdBIZ+Ak5pgJQxrvZOkLGmAvHxVE1Ff0LPhjePutGHm2jtTC48F4vei526NOmRFR0n+AcDOYrL\\\\\\\/ilq5fbM8GJ+KvIF2iydP2ZvI1fpIPrIrpqNLZphLr4BzY8WUoHzycP4T83JOAk+G3zxfM5Cxt4LcV9SkGqFu0KGevUL+QOX\\\\\\\/Ey0XpAmFpjEemqStHjTxmf2ABC8a+IHMbgRgItPVwpkGmrgNlDBKkFy4bTj\\\\\\\/6lA34beHCfr0XInPcEivpCllV4cAxYvTgSvmGVOB3ya4aNnY11GA0WG3HWkQIzfbqwzdcGIB8e5XGfuxBljMMW\\\\\\\/XGwgtuSicd+KO2qwQSDn84XIn3PiZADBTiMxMYQoi8JLI5IcClS0dEsQ9r2f8Gj5WWx+QsBj48Kmx04qCVJCOLI05W+QfguLr94A3EJpmdimMJ+G7KXrZFn9aCb6Iq7kVsXQrFk8n4L4qFPA9AxWGIhrPJ1A7hcRcw3sqZfDWr5iZurkUBrKgtGQkJBpR+blCv8quV\\\\\\\/d5k2rfqHAEJTNErIjr4yHvpRMY2\\\\\\\/539dKSnFvqHwnHlGVh+QQceDbCxEDQtI6g9Jfi4raIp6ScDs8tNupTeK1zAx7IbFFeZq9LBfGRKOJZ+2r9enRW+SE7Su4kXJmRg+Txg0lIidZMCUvmj4krv2eCQju2DzyD6dU5c1juj6Q6X88smcrBCzh+QNUZpLmvkAsE48b4uYPuUlY2jeFLNSst6vvVWDTk3P0HIPNWfsgcPeaFFEZhcu3Ujow\\\\\\\/IClD9QBj80ssq\\\\\\\/0ilx4adJr19csAK7ix0Rok0g4R86g467ooeSHb9jpLHopZWgQ54o3tQUtJ6rYtklQNFqx9B04eETCforP8URrqlJHPLFcicGTxsl5JqKFj\\\\\\\/JtddgOcOcO87cRp2sIDMaWKvV60ENgM44k+8PaT2e+xQhd5VOUtguP1wO8uLunooWwBRsOMFu+bGDOLPx\\\\\\\/WXGhRVl6oT1I2WZEIN7MQ0lFYlgTyHdpMThJNgtxFvS+3ljIUdUQB0v6q6QeAMbSSpazbKjuldrHM6YpaMzVtlBpPAM6LAJLBS\\\\\\\/VjphEZWqPMrLSixnK49npWJUtrCTwwHfhbKmUEycX0KaVUdQbsaFVg4GtKLG3obNZtfj\\\\\\\/M3QaJaoxczAY31YbGBsz9P1UpujWahjaQimMxj\\\\\\\/5z+KzhyMoWn8fFZc3G67kFwytCLRaRoP5aQixCfCZdCqIHoFFT8TnFantYgDFR75Qyk55Ck3fV7KJ57B3K1NShgFxzjsRpXgCjK7O\\\\\\\/v3wQAo4pMNMG3AUkA0HSHmAWb0hBQxVy+O5NG\\\\\\\/lDaFmdWqSxEvWEVsfuw\\\\\\\/V+zBRQ85uJR43hntUDtnz3khSXJypp8AK6gdQg8x7wZFLNtYTBsyVZgW+1T+7JCJiUYz5tROW6O\\\\\\\/grQCTPy5th\\\\\\\/v6E+ve0bVaGIvNPUnVtYfKJ7VpY+k3bgQLUK9l+gi3AJVFaKj8WSdXNJvach6ggSELiPcDt2RaFYastzugu7JUlqt7gLJcBcNOEuv+Z4NLMKxusF1AgShj\\\\\\\/iqyqnrbF7Bu3fsXpb1M9tDulNlOmembbrUB8IfULgUW58zb1syAye3DTLVPAa0QgPKim\\\\\\\/bM3757FU\\\\\\\/1ABwRVk16DpxKVNQS0XLUNmYBEIlzQO0iSGuffDfe2aGRjEJVwxq8pUqq62S5Uje\\\\\\\/Ya0Hdd8vyq0YtADA8D8kKRNGUV0g7bd+0DOh0pTAHkHBycyaz5rlvCDL5cKRZhOOTzJ7kyNv8L4KNOqa\\\\\\\/5bF0cSUgxyyNTrqzMgnQlD8OFexBPykiQpUQypfe2Vlxn734ns+55FZBH5mQ4MVSopGUddZV3AeadM8oxzyYos7oaX5yN+ce8o3H+yiY\\\\\\\/zRiU2nM\\\\\\\/qbwykU+F6V62BB5xD4wFzoPW0+0oGHWmV96H0aqRAFToB11Q9Bh6Ky6X+lJKumWXcSY2ZFOMg1\\\\\\\/aH8f+sTVntwaNXoexKuTCmFSWx6LJCsYndbI8MPLlMt2w5USgTw\\\\\\\/meMAECb18u6BsI6iexDDvW7e2tW2Lq6OiWzXyWWp465Kws4o9Zh9qIZ8936M9TMb6etHD1cz08gB9O8+gcm1FvSaxH3rPEJCHp+aIUVkhJsZn8uIkUmd1Cb5ruCSEDVZshXsPUPlP3fGFaP0SFv\\\\\\\/rnOp6zKpSm4ZTRUir9aaEBooV2liduJhWgocpzSAzdlDUjEjqgeWKYjNxXLXRcE\\\\\\\/Y6NyXXmZwAMAqQBeMwh6Iut9CnqiuWvLaN+KQzMiFqPGLp7451YG0rwMzVVCgjUtLqbD2OZsLkBEqE6Pzb4xRNNiqDUbTMzUs7Fp3skcQMjFNAq7vlN2IAdul2MTU+pszRrEDcmoX7q\\\\\\\/kz46KQjZXbilm0dbzdGl\\\\\\\/qaoR5wq4jrNjvDqYP60W38pcopqZHTlkn\\\\\\\/4sUGGKdoKYaQteAlU8yvfiP6YHvgbpzO7tx7pCIGVQQ4KGlFSfWRcB\\\\\\\/ARznE6hh8mN\\\\\\\/XrXhSfK+UvRbB8P0WLBnZtlUVoqzEthQAJL1XYWAQs90pxmU8IXTr6d9zPkbjBQMGWy7gyTArMZM8OnsaR0dKNMMm+Z8rq425wQs\\\\\\\/QSIzR2uKVUCvZhtnut2YxyFsq4ecqZfiUwPiDqAVvawydC69RkaggTEkrX2\\\\\\\/iODs7qVR\\\\\\\/HSapT+bMDqnyzPh0CQZBIMJVjjwFtoc183+0B8S+P5H9tn8NPH0fwq+6uhPI0GBIIiv5mjXSiwuIWojGmfkROx2rC8JA1fu3DZA6LWPY1I8RFfF7xxfjpOuiQLmO3HWOW0SBftoGZvvkIL4ahHJJ48LnEtp+VZyqUZmilWEi3v9raxdhijUTdt+qKDlzdB5YJ0AnkreWHjjYCGUop+WVfih8+ZtCCljP\\\\\\\/2qPuHsuO3Wt3JF9uSiQzJL0JVQuuBIodk80udZ6w38RRGH26CwU9UEuVT9s88xnFY\\\\\\\/bcQyMt6+Wtw+R+UvQr1sD7pZRYnFkIhzwuZR4IwfEjkaSryOr2IXyB6RhWWIk16sjOivs4W6xGfUtbnVBad3ROeKDYrx2RuSeXFHOuOQIx43gKQ4cxESEy6+yufK5TXWuzfj1kGWMHtsMX5h+TJHIrVZzveEKcoTI5u\\\\\\\/2gzmAv+MJxV9uIMM3tROD6039cZY4T126E4pD3awLeYYM33SpXAcxgPMX3CG4Y\\\\\\\/kQAw\\\\\\\/GI9Hhsi0OEGXW7VBYMjvC77b0qXcqKetcSYUmbp\\\\\\\/iO7Xr+JlBxF4Y3BHgD0USAbWYcOoN\\\\\\\/zidEZ8TmmzFuITbQmwA6XCuk+W2noujhn41O8mFx4O5a2xQDDIv84Pk4yICkXp82DQrN96jK3BclCbg9Gr5Rqs5LD2QXqQEKMvz1mNZc\\\\\\\/RJPFnY+EWIE3bkDs2G1RaJ93QO\\\\\\\/GBfhKhnPmChsPvfeupgieo0DDmXSYWlFrrrIcU50giAsiIB4\\\\\\\/JgGXu1V2VC1Smicx7FSO\\\\\\\/OnzngmlgGZcI8mUruVUKahoUJn3N8IwKIypJkU+DSVKPZa\\\\\\\/vdXtLJDZLn2GhI1T+AqDTLx14XCHy0HL2YjILQo5ligF3Jqh2hhdTB0yGl0Cian5YpI+lpTErPrsGWC1Q7YQmcad31jUXG0r7nbMRH2SCPNE23EZZVxmniRpbNAm1Itve6jpFO53Qk160ZG1IwDcZadn7o276165Nf9Br+ww\\\\\\\/kuzwnxmO8k2SHNtLbi3MrVbawQge1AP6o+i3V8g74Am\\\\\\\/Qeq0\\\\\\\/bOX+TXgYiVy19koqQXf6BE2ckmqfZeD6AcQgvrE82Ag8EoCEwYMpcKNufnjl5BrGG3uPpIxMiqoGtzulBCbR8xNYv7\\\\\\\/ogIdBXEKhzc3LrfGI4pW1VBtrqsRvUKgQHTXacsaFdV5sf\\\\\\\/hdQKftzG3L9ALLqL7yZmBrcUKgz288Bq5sVjOrBnrlNN6niD8ndWZm+SNBijt2SpX2SIwtu8KJYeu0bqt2sECjZch4veY+akWThC1BgvZ93pBKTpps1OGxdL43Nn597JrVZK\\\\\\\/FePdapRao3TN2ZCSCgHUSjW4HVKpoz12inaenzkKE+abWG27ZViLRLUL8Nap\\\\\\\/XWq4F5UHGt78imjjJ7GZ62WFti\\\\\\\/wDrIfU7g7Pn1yjJiEp6HKPTgjqXMLraBQ1j2BRPqryOpzWivcvc5iCH6QHcD6rL157TqCdBUJWtyWwawXVeYas+936iXJS4GW0Cq2x9\\\\\\\/i+VNfkR5rH6h2HPQrL\\\\\\\/lsGed1bjN\\\\\\\/9NoOnzoSEL7v66LCdFIiBR3KZ4K3X8Y\\\\\\\/Tba3bmfNmnIWfKadEbJFsTFwL3qrtibh7EiXL1w1yglPByE6l1l2BUija5jtGLFeovJzKWH3uwzTYIA12bdtMIDurekaJNHDui5uBEQdjYqcqC6EFJXf9fDo9jqD8C4gpy1v1tMwyAs8RfK2d5y+qJzD\\\\\\\/6hAhKyDCPLgbhFbiHaMVxulDrp5PNe2xVjhpjaYxAdhXeD9JWD0lnA4xsw+\\\\\\\/O3wFcd4rqU6DDwXtQgdR9jneiGq6MpfoQ4VeGU2tlCO3uyL7rxOUR9fQY8xkSykNMWkJVtyYAAvOyFan+BSrltICzm0qY2nUi4xqQf7Z5PPGDvz\\\\\\\/byUrAP3G5HQwhBH4wIEYRlCPIcEaNo9cVQLl7ekljVhWiy9p9WgLu+1n69dt4Kh1Tb0LDCrCefTPoqf\\\\\\\/GBF3jiiqcHiKM4ATW72HFMiXW+bm8b4J+aMZUtGukcNvm\\\\\\\/HjGhyS8QsTsl8mOGPk2XTC\\\\\\\/O+LHYrTDG8hf7Js2JjcqJ2frbNfMU1fB1OFoh2C2619+8b2g285RPaP+YxB+ryuGHjUKs4FrdQ9f7HgzmSwDLH4gwjhmgERrDPiJQIjflUP35H26VKWJsLXiTOKufuc1r92341Xn++HGlmT29icq0oL4J5LSl7KEwCqJ6hqyDq1dBWc1+kasoSQbzBtOQQS\\\\\\\/10Kb01SK\\\\\\\/0NKkmhZeWNaeZFfZIWieyBTSoGIJzjP+YkLH2n0q7tsf0QRA9XemyN6QunW3YV5zTOof0I2kRC921XHBREC3C7Va45PvIVHEUSl8kiF9b0DmpoVyiimoHVirL8bCFyhvDampXvOhqLtNGeW9L5TO1k11njrvnfOt2ysr6c0Yq3IBs+cigJio+dgteIqGUsGUgarRsX8FHKUFyWU1eEfoH1BEPYasyNYrJmaVHct1DLe\\\\\\\/cynyjeQsMrRpLOEupRJx3VAQ1HT\\\\\\\/sYh\\\\\\\/qhLjDlsQMVdr2o7aGK7CNsn4UC3PchMEk1zzgsV2uehAElP4YiSAmvMKkkWteEqNoLDLWJ\\\\\\\/fsT99c9CbDW30nlIsS+Vw5E1pCV6583fjwnhkdX8dAVJhCMnUbCzRy3pGrtJEfMBy8v2xN3\\\\\\\/9LXroZOmLuXKgyk6mKaPMJ0g7JU48vPHiFUlTK2BvMJWam78XIsNya\\\\\\\/iQ2ACuu9sOUS8pnW7+hmIZsmfgi1tyi4\\\\\\\/iTU83Lo5wxN04AchJGAqMTFOvV2wnxTYYBegBFWwd+vhU3kDPd8MDc+++40a+a+gA7ADky5Kickf4Zh6CpM02drKSlvkiQoU3IPs1okIE3fTH+iTeRwU0WJMcPH88KPBWvilX+JSlEXUl6+Ggc1Y+uq0Ozi3\\\\\\\/CXnNpB7etiP3TdfN4mK2WljaL9zqaYp1ge6+Lirvl49UYbO1BOskuN5HTwbDmtqVzJ4Jgrq3gyW8vM33VPog2jRL\\\\\\\/RoLpSVqLZLgp2nr2+mtGIvaNQMHD+wK6dFnhXkFQFs5GC7Kau4buKWs6ENvm5tfDNEw8VhEHHTqRVNkCnmd5ivlnmpFYCTDip7j\\\\\\\/n6QSroPG4lhjPYyTccUbJy9OMqExxzQWK03qssm25xXxMiN9RMRUZxENyTEEqHMmFeH4qrXXWvZcU2PJqthQcu1eopwFZrr2nvTJNSbUT9NF4Wj+8S31aR00xIR+W\\\\\\\/1PuxzLKWWeiHHqecyTEJrvU\\\\\\\/gEbwiB4HbvUXQP90WP67W7jJ29NgfvH+2+maU3ygdRVO2vxrOV8UEiuxdVwaA+4Ew8skOetv5pFeS8UQMWowakJAYfEuBr+SKAa0vqgOX5RJdxRkxjeL7yHruqbfNrEh8y5uK42sDoc\\\\\\\/5KFCiPU1VN7rDTi3iuy4FFmYvwSumXVtGhWe91Husj1PxEfA8X1hey9GWUeLgIo7zuXeIVsPSTPci4T3qZ7Ts4vXabbcPhhxw6m0xJowc0tth9uQ60iMlzTf28c7gF9xTsuSKkuDsRHRUm33U4MBpcPddc4pppY7HPWmIpMVV8k1TZXwU77h57zfiYKYQhw4oYnMpJCedbvxm0xlV9LamGDnIAIznTbSD4n9gyehA8lR2nyV4B1559WmdrK8ViRwuqJxihENqwlQwPDvIxBEELMsaCxiQafi62JtkofJhFqXMq75ieWmfutLGk2n2TzV7pwy9ahFUoUK\\\\\\\/F8iuvxV23qXWGwDLVfjv76GBq6pR1OXUmcvGhSCJ2Hpg+5mRkBI4htMCbqcH6\\\\\\\/SG4MgVjSUniqpp2OoTrIaxalTv2tZvLUm0TCWQZM+HQVR1OdsvrRnhBd0gsr8XCKkOXuFCX3rxxl6cj+5n8rDs7b35IoEd0r\\\\\\\/bGlB77uQT8yHBE\\\\\\\/L8Trhu5nO+r5oo2WJ0WzIqLst1y4VkCgxn53wIoa8oFhLBVueFuXxTU25NOIub8DeB9XDUKqbB+SpQn2+eVdCXXs2mPJ0YU8KZnsLOcnz\\\\\\\/6bjTYeP5n2gV7WQKK4+10BJz4MhK91rkiQx\\\\\\\/Z69tDvm++WbFJgFI1e+okq4c8c7Ou6rlmfveDP+IxHhxhHnCefY+Nmyb+VWGgljb\\\\\\\/+qAFVmLka30qrz8Kskn4aT\\\\\\\/1Pz6joZsVHooAn+8qMEtoINNpu+0Y9b9zR2IYXOmeQVi8dCxRaVhfGto2PJDp4fLkEmRgxVaYKDo3YxeKT2KGnO+aJTbcvv9JfsPLsr0JBwLEcX+tL5\\\\\\\/j+C98OqSLwZS9oGgMcPmwpNyGoQPTQvS0CF2hKzagDvvUu1Ho3kRZKkE1yPOYQHH3ht32ZUig+sXSFfnrpNiMfqsMTi\\\\\\\/+KItNqWZ+1kGa8vJUTCji6tI63Dmc+VIN0sTx51l7t0e1cQ8XGR0spyPdeumCrlXioE\\\\\\\/gqyHTWNReZjosg24QpuAju0bZejbq26jrHkwEzMDtqov8iuzI8FO3zU+BQrreYbqDupipnpeaTG8CTw7mGcGPSEFrVl2r+\\\\\\\/pDMW77rxKBubAIGXDiZMwENO0JUjoqYl8UcnwQ3GoppXW2yv+jau\\\\\\\/hcE9\\\\\\\/zD6\\\\\\\/\\\\\\\/GfuS0YGJr2+guY5yw\\\u0022,\\\u0022iv\\\u0022:\\\u00225IQxWxFZ5lxQBPDB1j5IcQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002235917c506cdaa5fc6677eee1717a67d74505c1c4ac1054c8fbae77a1dc7086f0\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LANHsvcCP5AyHHFWM+ae8lxa6pTJwvcRUqATfmQDu3nDP6aCSfypxscDuNFjsQ1yygcVeYFQCbMPzY7740jvmyYE8iCiuMssTn4OuRtJUfMScRMuR8BKuqYgEzHmTaamnShEbKoGh0BAKhJLvaQTobDaRPQRK\\\\\\\/GDFmYSz2UeDoWxIL+wjbb+3Z349TwlpAEpMf+mVCzC3mb4\\\\\\\/+RyBY3iBGIb8bQG8jHPdgZVyOWGfeMSBVCkB8MgPya5qR1YwzumgpsO4CfQ+eiHyBOlTii\\\\\\\/UPwsk1Z3+DZUeCEmHCwX\\\\\\\/\\\\\\\/GWnuVHPviymxFc21NYnFfOHbpQgHQdKLrQopaOZv9riY4eEm8D89lL8QOC5\\\\\\\/KRQIDeOvZ6fw\\\\\\\/qs6B0Pd1xfR9uMh\\\\\\\/FLdKSuyzSfRlZ+Eq1hq5Z2jMCvhf09ev7zEpy0KYxwPag9JBsSQAAz71F+OFgtPwDr1FY9v+\\\\\\\/faaK\\\\\\\/Z1G5PjJFOxwq3DYYnsun+LYWQbxoJDBoR+e6Tbu4e8ZXuZfUC5vWeqs+andGTfaEZWCcD6FiYLdERRNwy926xCajJzSVW9wAvAprqTOQ4g3hRA+WeW4Hx1n2JjZ2Jx\\\\\\\/p6TEQdARjSE4dzuQxyDwVSwDAh6\\\\\\\/9YiTi+ryi62BcJemWRuNaLLIzf+GAGUlZLp4VlrMir0cy+DIUpiExhMlyTs4LS\\\\\\\/wi5rqVZyrsm8dafUg\\\\\\\/kUrkemwEXit0CsetDxJIq1gKbcvf9x2oml4wq22loXtrX1KzUwg\\\\\\\/+nwY83lQaViFYGQyuMpp1QW2e4K7RzMm3AS1TRIXz6CaFeHyFquIAigrVfNsoiE2eS9TvlQeOaVAIjI5Uv3RizM0Tiy5M8voKkNpLTZmKLeDbe\\\\\\\/\\\\\\\/2C34\\\\\\\/aeFpLzrYG9dY5OWiOdQH5nH1yYcPv70ojs7b1Yu8q4DQ93zzein2AiaYAvMsBOvx+fhkbuTrvIfKYUfqooL4Gk4fRuZiNJBHo6MXNZPUmy8zrm02+zjxoYr1zy9VGK6YEAUvrklGdjunwbMvexlG0ISJBrJ\\\\\\\/hZVQ\\\\\\\/PhlWGHqEacHFBrCI8ndNviNKkcGv1DO2noriD5bPtgrRCXM2PGVhNSffAnZyRep+oj7HgwpqmvFAFlYnai+9wdGzIdxa\\\\\\\/GtLUrYrWgcsfwXxUjMOFDXhOtfY4gKO1\\\\\\\/w5qpc5NTZGPcXRoBOCq2qv1Z9RBznAEH\\\\\\\/L03kGQ6qJEopOEFmH0DvA9eitowPGquZGd7dQOPhJppn0+5D73njS2mxztJhzywlAsTdjncztEAsLz\\\\\\\/zIMroObQqnx3LuQfoq6lDaw+8CNJJQiIidOiOtJuCxAsvvL\\\\\\\/c8kIwaobj89q7L1CpSDtdIj3sMC16RYmBLQPl5xR5\\\\\\\/ROgPTpS9+hTOfh4LzVf3tTaQ7mH\\\\\\\/6JiTIizp4fG+G\\\\\\\/M3IzvqHrGniGD2xSCjO9VJmQTAMuoZpyWzty1q4Vz8kQII94B+mtMCQnmvP7DKqEi2IJiixsQ8H6\\\\\\\/KKUaRIuwC+jdghqvLf5X59vAO9N8xdcWX6CEDSGPOqt\\\\\\\/m1mYYoTCQ0nTi0c6G7WYKJZYdmx358xQvlWeGUb0q+B0fvQBuoh9nOdILZUvowi303YdDUKGKcedU0UnUDWovPFJAI8JEIbrO91OWpijgVfxQ1CpZljGBPtICJx5bgX3nuw+aY1QKjuWv2W4z5rf7hpXnoVuQBpw7klJxjI1sN8EBkESdbwSf\\\\\\\/YN+Z7ma0HxT4hZjDMDfJR76fJGIX04JF+ANhdUVJ3b1cT1FDCNHLk+Rqp6viTlWp8MG3s4j6nl+BydNhflJkMJnnNy0rC0c\\\\\\\/JKmbTSZaZBGgoLE0LgQl8hUnxO\\\\\\\/TRgk\\\\\\\/L9DYaOJnDtQ032h0+wCQ6sWl0B\\\\\\\/LVg6bsYMvTqYGBSLeGCkzJ\\\\\\\/DwD9r2dN6QZ7DwKSSDDilrIWE9do0LWgYWffGybTuVmLq9kvkfdyubMJcVylFAiXCB1ltBzzxJyoxdSTdei8+0iOMZq9sJ1bEBvYBcOc2\\\\\\\/KXH6JWrz4u8sf91wHJX3r8jzB9vhOMZ08pXjSBN38knflELY4bDY6DXs30qU\\\u0022,\\\u0022iv\\\u0022:\\\u0022f7s2emzeegJUmMLwXEFSDA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225d26651a844cf603fbcc9ddca36a3f16b00249e1b118a45a39116370473149ee\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459580 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/1955" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-90Aj8jErdcBhwvRdsV1ZB1oRbrUUNKMDemBXtrv4CKE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/1955" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-eH9Z8rF9QImHNSzOc8rvs-XfzVYgxIWqgIrfMzp0uO0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/1945" title="Comparative Analyses Identify the Contributions of Exotic Donors to Disease Resistance in a Barley Experimental Population" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/11/1965" title="Characterization and Prediction of Haploinsufficiency Using Systems-Level Gene Properties in Yeast" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459580" id="node459580" data-pisa="ggg;3/11/1955" data-pisa-master="ggg;g3.113.008250" data-apath="/ggg/3/11/1955.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A DNA Sequence Element That Advances Replication Origin Activation Time in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Thomas J. Pohl</span>, <span class="highwire-citation-author" data-delta="1">Katherine Kolor</span>, <span class="highwire-citation-author" data-delta="2">Walton L. Fangman</span>, <span class="highwire-citation-author" data-delta="3">Bonita J. Brewer</span> and <span class="highwire-citation-author" data-delta="4">M. K. Raghuraman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1955-1963;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008250 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459580" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thomas J. Pohl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular and Cellular Biology Program, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BJ.%2BPohl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pohl%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BJ.%2BPohl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katherine Kolor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Office of Public Health Genomics, Centers for Disease Control and Prevention, Atlanta, Georgia 30329</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katherine%2BKolor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kolor%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatherine%2BKolor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Walton L. Fangman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Walton%2BL.%2BFangman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fangman%20WL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWalton%2BL.%2BFangman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bonita J. Brewer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular and Cellular Biology Program, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bonita%2BJ.%2BBrewer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brewer%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABonita%2BJ.%2BBrewer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">M. K. Raghuraman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BK.%2BRaghuraman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raghuraman%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BK.%2BRaghuraman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:raghu@u.washington.edu" class="" data-icon-position="" data-hide-link-title="0">raghu@u.washington.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/11/1955" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459580" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459580/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/1955.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459580" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459580/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/1955.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459580" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459580/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/1955.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459580" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459580/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/11/1955.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459580" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459580/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Eukaryotic origins of DNA replication undergo activation at various times in S-phase, allowing the genome to be duplicated in a temporally staggered fashion. In the budding yeast <em>Saccharomyces cerevisiae</em>, the activation times of individual origins are not intrinsic to those origins but are instead governed by surrounding sequences. Currently, there are two examples of DNA sequences that are known to advance origin activation time, centromeres and forkhead transcription factor binding sites. By combining deletion and linker scanning mutational analysis with two-dimensional gel electrophoresis to measure fork direction in the context of a two-origin plasmid, we have identified and characterized a 19- to 23-bp and a larger 584-bp DNA sequence that are capable of advancing origin activation time.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/dna-replication" class="hw-term hw-article-keyword hw-article-keyword-dna-replication" rel="nofollow">DNA replication</a></li><li class="kwd"><a href="/keyword/origins-replication" class="hw-term hw-article-keyword hw-article-keyword-origins-of-replication" rel="nofollow">origins of replication</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/saccharomyces-cerevisiae" class="hw-term hw-article-keyword hw-article-keyword-saccharomyces-cerevisiae" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Saccharomyces cerevisiae</em></a></li><li class="kwd"><a href="/keyword/ars" class="hw-term hw-article-keyword hw-article-keyword-ars" rel="nofollow">ARS</a></li><li class="kwd"><a href="/keyword/temporal-regulation" class="hw-term hw-article-keyword hw-article-keyword-temporal-regulation" rel="nofollow">temporal regulation</a></li></ul><p id="p-5">The replication of eukaryotic chromosomes is temporally regulated, with different sections along a chromosome replicating, on average, at different times during S-phase. In the bakerâs yeast, <em>Saccharomyces cerevisiae</em>, the times at which origins undergo activation, or fire, have been shown to contribute to this temporal regulation (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Raghuraman <em>et al.</em> 2001</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Yabuki <em>et al.</em> 2002</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">McCune <em>et al.</em> 2008</a>). Two classes of origins have been described whose activation is delayed until late in S-phase; in both cases, the origins fire late because of their location in the genome. One class of late origins includes those located near telomeres, with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000077394">ARS501</a></em> near the right end of chromosome V being the prototype of this class (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Ferguson <em>et al.</em> 1991</a>, 1992). Late firing is not an intrinsic property of the origin: transfer to a circular plasmid results in activation in early S-phase. Moreover, when normally early-activated origins are inserted near a telomere, they become late activated (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Ferguson and Fangman 1992</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Wellinger <em>et al.</em> 1993</a>). The replication timing of telomere proximal origins has been shown to be governed by the Ku complex (a heterodimer consisting of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004897">Yku70</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004712">Yku80</a> subunits), the silencing chromatin protein <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004434">Sir3</a> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Stevenson and Gottschling 1999</a>), and the TG<sub>1-3</sub> repeat binding protein <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000479">Rif1</a> (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Cosgrove <em>et al.</em> 2002</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Lian <em>et al.</em> 2011</a>). A second class of late origins is exemplified by the chromosome XIV origin, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000077374">ARS1412</a></em> (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Friedman <em>et al.</em> 1996</a>). Here, delayed activation of the origin is caused by unspecified nontelomere sequence elements in flanking DNA. These delay elements can influence the activation times of origins that are located up to 6 kb away.</p><p id="p-6">These observations suggested that in the absence of external influences, origins are inherently early activated, and that later firing origins are delayed in their activation because of their particular surrounding sequences. However, it has recently been shown that centromeres function in <em>cis</em> to confer early activation on origins residing up to 20 kb away (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Pohl <em>et al.</em> 2012</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Natsume <em>et al.</em> 2013</a>). In addition, some origins that are not close to centromeres owe their early activation time, at least in part, to the binding of transcription factors <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001393">Fkh1</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005012">Fkh2</a> up to 500 bp from the origins (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Knott <em>et al.</em> 2012</a>). These data indicate that early origin activation is not merely a default state but that DNA elements such as centromeres and forkhead protein binding sites found in flanking sequences can promote early origin activation through the binding of kinetochores and forkhead proteins, respectively. In this manuscript, we provide evidence for a third DNA sequence element capable of advancing the time of activation of nearby origins.</p><p id="p-7">The DNA sequence capable of imparting early origin activation was found through analysis of a plasmid, pN&amp;S, that contains two copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em>, located 180Â° from each other and designated <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> [<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a> north</em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a> south</em>, respectively (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Brewer and Fangman 1994</a>); <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>]. In most cell cycles, only one of the two ARSs on the molecule is activated, and although the two copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> on pN&amp;S are identical in sequence, they are activated differentially such that initiation at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> is favored approximately 4:1 over initiation at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em>. The element causing this origin bias on plasmid pN&amp;S has been localized by deletion analysis to the 3â² end of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> gene, which is near <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Brewer and Fangman 1994</a>). We have concentrated on two hypotheses to explain the bias in origin use, origin competence and origin timing.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fork direction analysis of the two-ARS1 plasmid by 2-D gel electrophoresis. (A) Map of pN&amp;S, a ~7.5-kb plasmid described previously (Brewer and Fangman 1994). Thick lines indicate sequences derived from yeast, whereas thin lines indicate prokaryotic vector sequences. The &#x201C;compass&#x201D; bearings (see text) are indicated by N, S, E, and W. A BglII site occurs immediately 3&#x2032; to the T-rich strand of the 11-bp ARS consensus sequence of each ARS1. The orientation of each copy of ARS1 is indicated by the boxes containing letters representing the A and B regions (B1-3) defined for ARS1 (Marahrens and Stillman 1992). The URA3 gene is drawn as a gray arrow. The black arrows represent TRP1 sequences. The TRP1 gene fragment nearest to ARS1s lacks its promoter and 5&#x2032; sequences. The small arrows (labeled &#x201C;r&#x201D;) located in the plasmid&#x2019;s interior represent the positions of the recombinase sites (cloned at the SalI and PvuII sites indicated) to make pN&amp;Sdir (Figure S1). Not all sites for each enzyme are shown. The dotted line represents the region and the probe used to analyze fork direction 2-D gels. (B) Illustration depicting 2-D gel separation of forks moving in opposite directions through the southwestern BglII-PstI fragment of pN&amp;S. BglII digestion was carried out before electrophoresis in the first dimension and an in-gel digestion with PstI before the second dimension. Forks emanating from ARS1N (left arc) are distinguished from the forks emanating from ARS1S (right arc). The efficiency of each ARS is expressed as a percentage of the total hybridization signal from an arc (two lower rectangles) after subtracting background hybridization (two upper rectangles). (C-E) Fork direction gels for (C) pN&amp;S, (D) pARS1Ninv (pN&amp;S with the 1.45-kb EcoRI fragment containing ARS1N reversed), and (E) pARS1Sinv (pN&amp;S with the 1.05-kb EcoRI-EcoRV fragment containing ARS1S reversed). The numbers in the lower left of each panel refer to the percent initiation at ARS1S (see text for details)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2044600196" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Fork direction analysis of the two-&lt;em&gt;ARS1&lt;/em&gt; plasmid by 2-D gel electrophoresis. (A) Map of pN&amp;S, a ~7.5-kb plasmid described previously (Brewer and Fangman 1994). Thick lines indicate sequences derived from yeast, whereas thin lines indicate prokaryotic vector sequences. The &#x201C;compass&#x201D; bearings (see text) are indicated by N, S, E, and W. A &lt;em&gt;Bgl&lt;/em&gt;II site occurs immediately 3&#x2032; to the T-rich strand of the 11-bp ARS consensus sequence of each &lt;em&gt;ARS1&lt;/em&gt;. The orientation of each copy of &lt;em&gt;ARS1&lt;/em&gt; is indicated by the boxes containing letters representing the A and B regions (B&lt;sub&gt;1-3&lt;/sub&gt;) defined for &lt;em&gt;ARS1&lt;/em&gt; (Marahrens and Stillman 1992). The &lt;em&gt;URA3&lt;/em&gt; gene is drawn as a gray arrow. The black arrows represent &lt;em&gt;TRP1&lt;/em&gt; sequences. The &lt;em&gt;TRP1&lt;/em&gt; gene fragment nearest to &lt;em&gt;ARS1&lt;sup&gt;s&lt;/sup&gt;&lt;/em&gt; lacks its promoter and 5&#x2032; sequences. The small arrows (labeled &#x201C;r&#x201D;) located in the plasmid&#x2019;s interior represent the positions of the recombinase sites (cloned at the &lt;em&gt;Sal&lt;/em&gt;I and &lt;em&gt;Pvu&lt;/em&gt;II sites indicated) to make pN&amp;Sdir (Figure S1). Not all sites for each enzyme are shown. The dotted line represents the region and the probe used to analyze fork direction 2-D gels. (B) Illustration depicting 2-D gel separation of forks moving in opposite directions through the southwestern &lt;em&gt;Bgl&lt;/em&gt;II-&lt;em&gt;Pst&lt;/em&gt;I fragment of pN&amp;S. &lt;em&gt;Bgl&lt;/em&gt;II digestion was carried out before electrophoresis in the first dimension and an in-gel digestion with &lt;em&gt;Pst&lt;/em&gt;I before the second dimension. Forks emanating from &lt;em&gt;ARS1&lt;sup&gt;N&lt;/sup&gt;&lt;/em&gt; (left arc) are distinguished from the forks emanating from &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt; (right arc). The efficiency of each ARS is expressed as a percentage of the total hybridization signal from an arc (two lower rectangles) after subtracting background hybridization (two upper rectangles). (C-E) Fork direction gels for (C) pN&amp;S, (D) pARS1&lt;sup&gt;N&lt;/sup&gt;inv (pN&amp;S with the 1.45-kb &lt;em&gt;Eco&lt;/em&gt;RI fragment containing &lt;em&gt;ARS1&lt;sup&gt;N&lt;/sup&gt;&lt;/em&gt; reversed), and (E) pARS1&lt;sup&gt;S&lt;/sup&gt;inv (pN&amp;S with the 1.05-kb &lt;em&gt;Eco&lt;/em&gt;RI-&lt;em&gt;Eco&lt;/em&gt;RV fragment containing &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt; reversed). The numbers in the lower left of each panel refer to the percent initiation at &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt; (see text for details).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1955/F1.medium.gif" width="440" height="426"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/11/1955/F1.medium.gif" width="440" height="426"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463524" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-8" class="first-child">Fork direction analysis of the two-<em>ARS1</em> plasmid by 2-D gel electrophoresis. (A) Map of pN&amp;S, a ~7.5-kb plasmid described previously (<a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Brewer and Fangman 1994</a>). Thick lines indicate sequences derived from yeast, whereas thin lines indicate prokaryotic vector sequences. The âcompassâ bearings (see text) are indicated by N, S, E, and W. A <em>Bgl</em>II site occurs immediately 3â² to the T-rich strand of the 11-bp ARS consensus sequence of each <em>ARS1</em>. The orientation of each copy of <em>ARS1</em> is indicated by the boxes containing letters representing the A and B regions (B<sub>1-3</sub>) defined for <em>ARS1</em> (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Marahrens and Stillman 1992</a>). The <em>URA3</em> gene is drawn as a gray arrow. The black arrows represent <em>TRP1</em> sequences. The <em>TRP1</em> gene fragment nearest to <em>ARS1<sup>s</sup></em> lacks its promoter and 5â² sequences. The small arrows (labeled ârâ) located in the plasmidâs interior represent the positions of the recombinase sites (cloned at the <em>Sal</em>I and <em>Pvu</em>II sites indicated) to make pN&amp;Sdir (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008250/-/DC1/FigureS1.pdf">Figure S1</a>). Not all sites for each enzyme are shown. The dotted line represents the region and the probe used to analyze fork direction 2-D gels. (B) Illustration depicting 2-D gel separation of forks moving in opposite directions through the southwestern <em>Bgl</em>II-<em>Pst</em>I fragment of pN&amp;S. <em>Bgl</em>II digestion was carried out before electrophoresis in the first dimension and an in-gel digestion with <em>Pst</em>I before the second dimension. Forks emanating from <em>ARS1<sup>N</sup></em> (left arc) are distinguished from the forks emanating from <em>ARS1<sup>S</sup></em> (right arc). The efficiency of each ARS is expressed as a percentage of the total hybridization signal from an arc (two lower rectangles) after subtracting background hybridization (two upper rectangles). (C-E) Fork direction gels for (C) pN&amp;S, (D) pARS1<sup>N</sup>inv (pN&amp;S with the 1.45-kb <em>Eco</em>RI fragment containing <em>ARS1<sup>N</sup></em> reversed), and (E) pARS1<sup>S</sup>inv (pN&amp;S with the 1.05-kb <em>Eco</em>RI-<em>Eco</em>RV fragment containing <em>ARS1<sup>S</sup></em> reversed). The numbers in the lower left of each panel refer to the percent initiation at <em>ARS1<sup>S</sup></em> (see text for details).</p><div class="sb-div caption-clear"></div></div></div><p id="p-9">The origin competence hypothesis is that the inherent biochemical ability of one of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> copies to fire is affected because either its licensing or some subsequent initiation step is influenced by the 3â² <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> sequence. Consequently, either the ability of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> to fire is reduced or that of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> is increased. If this hypothesis were true, then the proposed difference in competence might be manifested as a difference in the ability of cells to maintain plasmids containing only the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> region or the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> region in the same context relative to the 3â² <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> sequences as plasmid pN&amp;S. However, the observed relative rates of plasmid loss for such constructs do not reflect the initiation bias. A plasmid bearing <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em>, which is predominantly used on pN&amp;S, exhibits a slightly greater rate of loss than does a plasmid with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> (<a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Brewer and Fangman 1994</a>).</p><p id="p-10">An alternative hypothesis is that the 3â² <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> sequence causes <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> to fire earlier on average than <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em>, resulting in passive replication of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> before it has the opportunity to fire. A difference in the average time of activation of as little as one minute would be adequate to account for the passive replication of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> in a majority of cells given that the distance between the two origins on plasmid pN&amp;S is only ~3.75 kb and the average replication fork rate at 23Â° is about 3.5 kb/min (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Rivin and Fangman 1980</a>). This hypothesis more readily explains the absence of molecules in which both origins are active and is also consistent with the similar maintenance observed for plasmids containing either just the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> region or just the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> region.</p><p id="p-11">In this study, we have dissected the bias element by deletion analysis and by scanning mutagenesis, and show that this element does indeed cause earlier activation in S-phase of the nearby copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> relative to the second, more distant copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em>. Our results provide the identification of a potential third mechanism by which DNA sequences can advance origin activation time.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strains and plasmids</h3><p id="p-12">Strain 3xRZeoDIR, which is RM14-3a (background A364a, <em>MATa</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002175">cdc7</a>-1</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001277">bar1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001282">his6</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">leu2</a>-3,112</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">trp1</a>-289</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">ura3</a>-52</em>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">McCarroll and Fangman 1988</a>) with three copies of the 6891 bp <em>Cla</em>I fragment of pRINT (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Friedman <em>et al.</em> 1995</a>) tandemly integrated at the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000523">LEU2</a></em> locus and a zeocin ARS-less cassette on the right arm of chromosome V, was used for the comparative hybridization experiments (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Friedman <em>et al.</em> 1995</a>). The ARS-less cassette was created by integrating the 3083 bp <em>Pvu</em>II fragment of pZeodir (described in the next paragraph) by homologous recombination. pN&amp;S deletion derivatives were analyzed in RM14-3a or MR14-3a. Because the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">trp1</a>-1</em> allele of RM14-3a is somewhat revertible, MR14-3a, which is RM14-3a with a <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> knockout allele, was used for all deletion constructs except pÎ43 and pÎ5. MR14-3a was used for all of the oligonucleotide mutagenesis constructs.</p><p id="p-13">Plasmids were constructed in <em>Escherichia coli</em> strain DH5Î±. All bias-determinant plasmid constructs are based on plasmid pN&amp;S (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Brewer and Fangman 1994</a>). Because of the duplication of the <em>TRP1ARS1</em> sequences in the two-<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> plasmid, it was often difficult to find appropriate unique restriction sites for plasmid modifications. To circumvent this problem, several plasmids were constructed by a multistep process beginning with plasmids that contain only the south half of the plasmid. These plasmids are pBB3, pBB6, pBB6mut, and pBB6TA. pBB3 has the 967-bp <em>Nde</em>I-<em>Sma</em>I <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> fragment from YIp5 cloned into the 245-bp <em>Nde</em>I-<em>Sma</em>I fragment of pUC18. pBB6 has the 1068-bp <em>Eco</em>RI-<em>Eco</em>RV fragment of <em>TRP1ARS1</em> fragment cloned into the 3386-bp <em>Eco</em>RI-<em>Sma</em>I fragment of pBB3. pBB6TA has a 1453-bp <em>Eco</em>RI <em>TRP1ARS1</em> fragment cloned into the <em>Eco</em>RI site of pBB3 in the same orientation as pBB6. pBB6mut is pBB6TA with a 293-bp <em>Nde</em>I-<em>Nco</em>I deletion. After the desired alteration had been made, the two-ARS plasmid was reconstructed. pZeodir was constructed by ligating the 1181-bp blunt <em>Kpn</em>I-<em>Hin</em>dIII fragment of pUT322 (CAYLA) into the 4258-bp blunt <em>Hin</em>dIII fragment of pKAN-DIR (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Friedman <em>et al.</em> 1995</a>).</p></div><div id="sec-3" class="subsection"><h3>Mutagenesis and consensus sequence alignments</h3><p id="p-14">The revertible selectable marker method was used for mutagenesis (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Deng and Nickoloff 1992</a>). Oligonucleotides were purchased from Gibco BRL. The sequences of the mutations m1-10 are given in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>. The sequences of the mutations m11, m12, and âNsiI are given in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3D</a>. The sequences of the DNA unwinding elements (DUE) mutations corresponding to the m5-m7 positions were m5DUE (ATCAAGTACT; data not shown), m6DUE (ATAGACGTCA; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008250/-/DC1/008250SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008250/-/DC1/FigureS2.pdf">Figure S2</a>), and m7DUE (ATTAATAGT; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008250/-/DC1/FigureS2.pdf">Figure S2</a>). Correct transformants were identified by restriction digestion screening for incorporation of a restriction site (<em>Xho</em>I for m1-10; <em>Sca</em>I for m5DUE; <em>Aat</em>II for M6DUE; <em>Ase</em>I for m7DUE), and the mutations were confirmed by sequencing. The success rate of the mutagenesis was generally approximately 60%.</p><p id="p-15">Consensus matches for the binding sites of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001595">Abf1p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005160">Rap1p</a> were identified by the use of the Intelligenetics program Seq. The parameters used were as follows: GapPenalty = 1, GapSizePenalty = 0.3333, MinAlign = 4, MisMatchPenalty = 0, OverLap = 33.3%, SegmentSize = 30, Expect = 10, LoopOut = 3, and MinMatch = 5. Sequences were entered using the Intelligenetics program GENED. Computer analysis of DUE was performed using VooDUE (M. K. Raghuraman, unpublished results), a Macintosh version of the Thermodyne program (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Natale <em>et al.</em> 1992</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">1993</a>). It calculates the energy of unwinding of the DNA duplex in the same manner as the Thermodyne program.</p></div><div id="sec-4" class="subsection"><h3>Two-dimensional (2-D) gel electrophoresis</h3><p id="p-16">Samples for 2-D gels were collected at 1.5-min intervals for 33 min after release of the cells into S phase, then pooled. Cell synchrony and release were performed as described (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Brewer <em>et al.</em> 1992</a>). DNA was prepared as described (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Huberman <em>et al.</em> 1987</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Brewer <em>et al.</em> 1992</a>). Determining the direction of replication fork movement by 2-D gels involved an in-gel digestion as described (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Friedman and Brewer 1995</a>). For every construct, the west half of the plasmid was analyzed. The blots were hybridized to a radioactively labeled 1185-bp <em>Bgl</em>I-<em>Nde</em>I fragment of pBR322 and analyzed on an InstantImager (Packard) for quantitation.</p></div><div id="sec-5" class="subsection"><h3>Determination of replication time</h3><p id="p-17">Density shift experiments on the strains analyzed in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4B</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5E</a> were respectively performed in strains RM14-3a and KK14-3a (a URA+ version of RM14-3a) as previously described (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">McCarroll and Fangman 1988</a>). The data were analyzed using an InstantImager or phosphorimager (Packard or BioRad, respectively). The comparative hybridization experiments were performed essentially as described (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Friedman <em>et al.</em> 1995</a>). Strain 3xRZeoDIR containing pN&amp;Sdir or pN&amp;SDdir (described in the section <em>Results</em>) was incubated for 1.25 generations in Î±-factor. Upon arrest in the G1 phase, an uninduced sample was taken and galactose was added to 2%. Excision of the ARS-less cassette was allowed to proceed for four hours. Glucose was then added to 2% and the cells were released from Î±-factor arrest at 37Â°, the restrictive temperature for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002175">cdc7</a><sup>ts</sup></em>. Upon <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002175">cdc7</a></em> arrest, a sample was taken, and the cells were returned to 23Â° to allow entry into S phase. Samples were taken at 4-min intervals throughout S (60 min). The DNA was prepared by the âSmash and Grabâ procedure (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hoffman and Winston 1987</a>) and digested with <em>Bam</em>HI and <em>Eag</em>I. The samples were run on a 0.7% agarose gel, blotted onto Hybond (Amersham) nylon membrane, and hybridized to a <em>Bam</em>HI-<em>Eag</em>I fragment of pBR322 labeled with <sup>32</sup>P. Quantification of the blots was performed on a PhosphorImager (Molecular Dynamics) using ImageQuant software.</p></div></div><div class="section results-discussion" id="sec-6"><h2 class="">Results and Discussion</h2><div id="sec-7" class="subsection"><h3>Identifying the target of the bias determinant</h3><p id="p-18">We have previously shown that in the plasmid pN&amp;S (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>), <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> is used four times more frequently than is <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> and that the preferred use of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> is dependent on sequences residing in the 3â² end of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> gene (<a id="xref-ref-1-6" class="xref-bibr" href="#ref-1">Brewer and Fangman 1994</a>). Furthermore, this biased use of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> is not dependent on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> transcription.</p><p id="p-19">Initiation bias in this two-<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> plasmid was determined by examining the direction in which replication forks move through a restriction fragment that lies between the two copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>). If <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> is active, forks move northward; if <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> is active, forks move southward. Replication intermediates of the <em>Bgl</em>II fragment from the west half of the plasmid generate an arc of Y molecules on standard 2-D gels with replication forks from <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> producing the same arc of Ys as forks from <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> (<a id="xref-ref-1-7" class="xref-bibr" href="#ref-1">Brewer and Fangman 1994</a>). To distinguish the two sets of replication intermediates, we included an in-gel restriction digestion with <em>Pst</em>I before the second dimension of electrophoresis and probed for the larger <em>Pst</em>I-<em>Bgl</em>II fragment. Ys created by forks emanating from <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> create an arc that rises from the 1N spot, whereas Ys from <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> create an arc that rises from a position displaced from the 1N spot (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Friedman and Brewer 1995</a>). The amount of signal in each arc of Ys is quantified (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1B</a>) to yield the percentage of molecules replicated by forks from <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em>âvalues greater than 50% indicating biased initiation from <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em>. The original two-ARS plasmid, pN&amp;S (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1C</a>), shows <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> initiation 75.2% of the time based on the values obtained from five different cultures (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> initiation values of 71%, 74%, 74%, 77%, and 80%). These values provide a baseline for comparison with other constructs. Because the labor involved in each gel experiment limits the sample sizes that can reasonably be obtained, in testing other constructs described hereafter, we have chosen to be conservative in deciding whether biased initiation is significantly different from that of pN&amp;S. Thus, we conservatively conclude that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> initiation percentages falling outside the range of 68.1â82.3% (75.2 Â± 7.1%; Students <em>t</em>-test 99% confidence interval) are significantly different from pN&amp;S at the 1% level.</p><p id="p-20">We first determined the target of the bias determinant residing at the 3â² end of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> gene. This initiation bias could be due to an effect on the activation of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> or on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em>. To determine which copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> was being affected by the 3â² <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> sequences, we made two derivatives of pN&amp;S, each having one of the two copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> inverted compared to its original orientation. ARS sequences are inherently asymmetrical because they are composed of discrete A and B elements: the three B elements in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> lie on the 3â²-side of the T-rich strand of the 17-bp AT-rich ARS consensus sequence (A element) (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1A</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Newlon 1996</a>). Because of this asymmetry, we reasoned that the orientation of one of the copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> with respect to the bias determinant might be important for the initiation bias.</p><p id="p-21">We found that inversion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> in pN&amp;S fails to reduce the bias (74% <em>vs.</em> 83%; compare <a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1, C and D</a>), whereas inversion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> reduces it substantially (74% <em>vs.</em> 56%; compare <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1, C and E</a>). Because the endpoints of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> inversion do not include the sequences required for the initiation bias (described below), the reduction of the initiation bias can be attributed to the failure of the bias determinant to act on the ARS when the ARS is in inverted orientation and not to the disruption of the sequence causing the bias. These data indicate that the primary target of the bias determinant is <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> rather than <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em>.</p></div><div id="sec-8" class="subsection"><h3>Location and structure of the bias determinant</h3><p id="p-22">Determination of the initiation bias in the two-ARS plasmid provides a relatively simple assay for a mutational analysis of this element. Previous observations showed that deletion of the <em>Nco</em>I-<em>Nru</em>I fragment, which included the 5â² half of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> (plasmid pÎ34), does not eliminate the preferential use of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em>, but that deletion of the <em>Nsi</em>I-<em>Nco</em>I fragment from the 3â² end of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> (plasmid pÎ43) results in equivalent use of both copies of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>; <a id="xref-ref-1-8" class="xref-bibr" href="#ref-1">Brewer and Fangman 1994</a>). To further localize the early determinant, four deletions (plasmids pÎ5, pÎ6, pÎ7, and pÎ8) that remove sections of the <em>Nsi</em>I-<em>Nco</em>I sequence were constructed using convenient restriction sites, and tested for origin bias. Deletion of the 169-bp <em>Apa</em>I-<em>Nco</em>I fragment (pÎ5), the 302-bp <em>Alw</em>NI-<em>Apa</em>I fragment (pÎ6), or the 113-bp <em>Hae</em>III-<em>Alw</em>NI fragment (pÎ7) had no significant reduction of the bias. However, the 34-bp <em>Nsi</em>I-<em>Hae</em>III fragment contributes substantially to the bias because its deletion (pÎ8) significantly reduced the bias (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Deletion analysis of 3&#x2032; URA3 sequences important for the initiation bias. Top, A map of the southern region of pN&amp;S including ARS1S and the URA3 gene is shown. A bubble indicates the position of ARS1S. Values for percent initiation at ARS1S are given for pN&amp;S and nine deletion derivatives. Values that are significantly reduced compared with the value obtained for pN&amp;S on five separate experiments are indicated by (*). Deletion constructs are as follows: (p&#x394;34) &#x2206;NcoI-NruI (Brewer and Fangman 1994); (p&#x394;43) &#x2206;NsiI-NcoI (Brewer and Fangman 1994); (p&#x394;5) &#x2206;ApaI-NcoI; (p&#x394;6) &#x2206;AlwNI-ApaI; (p&#x394;7) &#x2206;HaeIII-AlwNI; (p&#x394;8) &#x2206;NsiI-HaeIII; (p&#x394;9) &#x2206;SmaI-NsiI; (p&#x394;10) &#x2206;HindIII-EcoRV; (p&#x394;11) &#x2206;HaeIII-NcoI. Restriction sites are as follows: (Al) AlwNI; (Ap) ApaI; (RI) EcoRI; (RV) EcoRV; (Ha) HaeIII; (Hi) HindIII; (Nc) NcoI; (Nr) NruI; (Ns) NsiI; (S) SmaI." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2044600196" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Deletion analysis of 3&#x2032; &lt;em&gt;URA3&lt;/em&gt; sequences important for the initiation bias. Top, A map of the southern region of pN&amp;S including &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt; and the &lt;em&gt;URA3&lt;/em&gt; gene is shown. A bubble indicates the position of &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt;. Values for percent initiation at &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt; are given for pN&amp;S and nine deletion derivatives. Values that are significantly reduced compared with the value obtained for pN&amp;S on five separate experiments are indicated by (*). Deletion constructs are as follows: (p&#x394;34) &#x2206;&lt;em&gt;Nco&lt;/em&gt;I-&lt;em&gt;Nru&lt;/em&gt;I (Brewer and Fangman 1994); (p&#x394;43) &#x2206;&lt;em&gt;Nsi&lt;/em&gt;I-&lt;em&gt;Nco&lt;/em&gt;I (Brewer and Fangman 1994); (p&#x394;5) &#x2206;&lt;em&gt;Apa&lt;/em&gt;I-&lt;em&gt;Nco&lt;/em&gt;I; (p&#x394;6) &#x2206;&lt;em&gt;Alw&lt;/em&gt;NI-&lt;em&gt;Apa&lt;/em&gt;I; (p&#x394;7) &#x2206;&lt;em&gt;Hae&lt;/em&gt;III-&lt;em&gt;Alw&lt;/em&gt;NI; (p&#x394;8) &#x2206;&lt;em&gt;Nsi&lt;/em&gt;I-&lt;em&gt;Hae&lt;/em&gt;III; (p&#x394;9) &#x2206;&lt;em&gt;Sma&lt;/em&gt;I-&lt;em&gt;Nsi&lt;/em&gt;I; (p&#x394;10) &#x2206;&lt;em&gt;Hin&lt;/em&gt;dIII-&lt;em&gt;Eco&lt;/em&gt;RV; (p&#x394;11) &#x2206;&lt;em&gt;Hae&lt;/em&gt;III-&lt;em&gt;Nco&lt;/em&gt;I. Restriction sites are as follows: (Al) &lt;em&gt;Alw&lt;/em&gt;NI; (Ap) &lt;em&gt;Apa&lt;/em&gt;I; (RI) &lt;em&gt;Eco&lt;/em&gt;RI; (RV) &lt;em&gt;Eco&lt;/em&gt;RV; (Ha) &lt;em&gt;Hae&lt;/em&gt;III; (Hi) &lt;em&gt;Hin&lt;/em&gt;dIII; (Nc) &lt;em&gt;Nco&lt;/em&gt;I; (Nr) &lt;em&gt;Nru&lt;/em&gt;I; (Ns) &lt;em&gt;Nsi&lt;/em&gt;I; (S) &lt;em&gt;Sma&lt;/em&gt;I.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1955/F2.medium.gif" width="440" height="291"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/11/1955/F2.medium.gif" width="440" height="291"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463609" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Deletion analysis of 3â² <em>URA3</em> sequences important for the initiation bias. Top, A map of the southern region of pN&amp;S including <em>ARS1<sup>S</sup></em> and the <em>URA3</em> gene is shown. A bubble indicates the position of <em>ARS1<sup>S</sup></em>. Values for percent initiation at <em>ARS1<sup>S</sup></em> are given for pN&amp;S and nine deletion derivatives. Values that are significantly reduced compared with the value obtained for pN&amp;S on five separate experiments are indicated by (*). Deletion constructs are as follows: (pÎ34) â<em>Nco</em>I-<em>Nru</em>I (<a id="xref-ref-1-9" class="xref-bibr" href="#ref-1">Brewer and Fangman 1994</a>); (pÎ43) â<em>Nsi</em>I-<em>Nco</em>I (<a id="xref-ref-1-10" class="xref-bibr" href="#ref-1">Brewer and Fangman 1994</a>); (pÎ5) â<em>Apa</em>I-<em>Nco</em>I; (pÎ6) â<em>Alw</em>NI-<em>Apa</em>I; (pÎ7) â<em>Hae</em>III-<em>Alw</em>NI; (pÎ8) â<em>Nsi</em>I-<em>Hae</em>III; (pÎ9) â<em>Sma</em>I-<em>Nsi</em>I; (pÎ10) â<em>Hin</em>dIII-<em>Eco</em>RV; (pÎ11) â<em>Hae</em>III-<em>Nco</em>I. Restriction sites are as follows: (Al) <em>Alw</em>NI; (Ap) <em>Apa</em>I; (RI) <em>Eco</em>RI; (RV) <em>Eco</em>RV; (Ha) <em>Hae</em>III; (Hi) <em>Hin</em>dIII; (Nc) <em>Nco</em>I; (Nr) <em>Nru</em>I; (Ns) <em>Nsi</em>I; (S) <em>Sma</em>I.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">To examine a possible role of sequences immediately to the left of the <em>Nsi</em>I-<em>Hae</em>III fragment, we made a 56-bp <em>Sma</em>I-<em>Nsi</em>I deletion (plasmid pÎ9). This deletion greatly reduced the bias (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>), showing that these deleted sequences also play a major role. Further deletion to the left via the removal of the 228-bp <em>Hind</em>III-<em>Sma</em>I fragment, including a portion of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002414">TRP1</a></em> sequences (plasmid pÎ10), had little effect on the initiation bias (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). On the basis of these results, plasmid sequences that contribute substantially to initiation bias appear to lie largely within the 90-bp <em>Sma</em>I to <em>Hae</em>III sequence in the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> gene (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>, bracket 1). However, despite finding little effect of deletions â5, â6, and â7, which together span the 584-bp <em>Hae</em>III-<em>Nco</em>I fragment to the right of the <em>Sma</em>I-<em>Hae</em>III section, deletion of the entire <em>Hae</em>III-<em>Nco</em>I fragment (plasmid pÎ11) eliminated the origin bias (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>). Because no single internal deletion in this region (pÎ5, pÎ6, and pÎ7) dramatically reduced the initiation bias, there appear to be functionally redundant contributions to origin bias by the <em>Hae</em>III-<em>Nco</em>I region. In summary, sequences within the <em>Hae</em>III-<em>Nco</em>I fragment (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>, bracket 2), along with those in the <em>Sma</em>I-<em>Hae</em>III fragment, contribute to biased origin use. Due to the apparent complexity of sequences within the <em>Hae</em>III-<em>Nco</em>I region, we focused our further studies on the unique 90 bp fragment (<a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2</a>, bracket 1).</p><p id="p-25">To further determine which sequences within the 90-bp <em>Sma</em>I-<em>Hae</em>III fragment (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a>, bracket 1) contribute to bias determinant activity, this fragment was examined by scanning site-directed mutagenesis. Typically, 10-bp changes were made (one mutation, m7, was a 9-bp change) (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>). Because an <em>Xho</em>I restriction site was also included for screening purposes two base pairs of the 90 bp examined were not changed by the mutagenesis (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3A</a>, overlines). Analysis of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> initiation percentage for the 10 mutants (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3B</a>) revealed that two of them, m6 and m7 (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3C</a>), exhibited dramatic reductions in the initiation bias whereas a third mutant, m5, showed a slightly reduced initiation bias. The other seven mutations gave initiation biases that were not significantly different from that of the original two-ARS plasmid (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3B</a>, vertical bracket). Therefore, through scanning mutagenesis we identified a 29-bp sequence (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3A</a>, red) within the <em>Sma</em>I-<em>Hae</em>III fragment that contains bases important for origin bias.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scanning mutagenesis analysis for sequences involved in initiation bias. (A) The wild-type sequence of the SmaI-HaeIII fragment of URA3 is shown in capital letters. Sequences of the mutations are given in lowercase letters beneath the wild-type sequences they replace. The two overlined nucleotides were not changed by the mutagenesis. The boxes identify 11 of 13 bp matches to Rap1 or Abf1 consensus binding sites as well as matches to Fkh1 and Fkh2. The 3&#x2032; end of the URA3 coding sequence is indicated by the single letter amino acid code. The locations of the SmaI, NsiI, and HaeIII restriction enzyme sites are indicated. Sequences that when mutated significantly reduce the ARS1S bias are indicated in red. (B) Values for percent initiation at ARS1S are plotted for the 10 mutations that span the SmaI-HaeIII fragment. The bar on the right of the graph indicates a 99% confidence interval extending on both sides of the mean percent initiation at ARS1S for pN&amp;S based on five separate experiments. Values that fall outside of this range are considered significantly different from pN&amp;S at the 1% level. The approximate locations of the mutations relative to the 3&#x2032; half of the URA3 gene (arrow) are shown. (C) Fork-direction gel analysis of mutants m6 and m7 as examples of the mutations that eliminate the origin bias. (D) The wild-type sequence corresponding to the sequences of the URA3 fragment that were mutated in m5, m6, and m7 are shown in capital letters. Sequences of three different mutations (m12, &#x2206;NsiI, m11) are given in lowercase letters beneath the wild-type sequences they replace. Deleted bases are depicted by a dash. The overlined nucleotides were not changed by the mutagenesis while the underlined nucleotides mark the NsiI site. Sequences important for the bias are indicated in red letters. Percent initiation at ARS1S is indicated as bar graphs for the three mutations, m11, m12, and &#x2206;NsiI. (E) Fork direction 2-D gels of mutants m11 and m12 (see panel D for sequences). Percent initiation at ARS1S is indicated in the upper right corners." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2044600196" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Scanning mutagenesis analysis for sequences involved in initiation bias. (A) The wild-type sequence of the &lt;em&gt;Sma&lt;/em&gt;I-&lt;em&gt;Hae&lt;/em&gt;III fragment of &lt;em&gt;URA3&lt;/em&gt; is shown in capital letters. Sequences of the mutations are given in lowercase letters beneath the wild-type sequences they replace. The two overlined nucleotides were not changed by the mutagenesis. The boxes identify 11 of 13 bp matches to Rap1 or Abf1 consensus binding sites as well as matches to Fkh1 and Fkh2. The 3&#x2032; end of the &lt;em&gt;URA3&lt;/em&gt; coding sequence is indicated by the single letter amino acid code. The locations of the &lt;em&gt;Sma&lt;/em&gt;I, &lt;em&gt;Nsi&lt;/em&gt;I, and &lt;em&gt;Hae&lt;/em&gt;III restriction enzyme sites are indicated. Sequences that when mutated significantly reduce the &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt; bias are indicated in red. (B) Values for percent initiation at &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt; are plotted for the 10 mutations that span the &lt;em&gt;Sma&lt;/em&gt;I-&lt;em&gt;Hae&lt;/em&gt;III fragment. The bar on the right of the graph indicates a 99% confidence interval extending on both sides of the mean percent initiation at &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt; for pN&amp;S based on five separate experiments. Values that fall outside of this range are considered significantly different from pN&amp;S at the 1% level. The approximate locations of the mutations relative to the 3&#x2032; half of the &lt;em&gt;URA3&lt;/em&gt; gene (arrow) are shown. (C) Fork-direction gel analysis of mutants m6 and m7 as examples of the mutations that eliminate the origin bias. (D) The wild-type sequence corresponding to the sequences of the &lt;em&gt;URA3&lt;/em&gt; fragment that were mutated in m5, m6, and m7 are shown in capital letters. Sequences of three different mutations (m12, &#x2206;&lt;em&gt;Nsi&lt;/em&gt;I, m11) are given in lowercase letters beneath the wild-type sequences they replace. Deleted bases are depicted by a dash. The overlined nucleotides were not changed by the mutagenesis while the underlined nucleotides mark the &lt;em&gt;Nsi&lt;/em&gt;I site. Sequences important for the bias are indicated in red letters. Percent initiation at &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt; is indicated as bar graphs for the three mutations, m11, m12, and &#x2206;&lt;em&gt;Nsi&lt;/em&gt;I. (E) Fork direction 2-D gels of mutants m11 and m12 (see panel D for sequences). Percent initiation at &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt; is indicated in the upper right corners.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1955/F3.medium.gif" width="440" height="300"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/11/1955/F3.medium.gif" width="440" height="300"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463606" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-26" class="first-child">Scanning mutagenesis analysis for sequences involved in initiation bias. (A) The wild-type sequence of the <em>Sma</em>I-<em>Hae</em>III fragment of <em>URA3</em> is shown in capital letters. Sequences of the mutations are given in lowercase letters beneath the wild-type sequences they replace. The two overlined nucleotides were not changed by the mutagenesis. The boxes identify 11 of 13 bp matches to Rap1 or Abf1 consensus binding sites as well as matches to Fkh1 and Fkh2. The 3â² end of the <em>URA3</em> coding sequence is indicated by the single letter amino acid code. The locations of the <em>Sma</em>I, <em>Nsi</em>I, and <em>Hae</em>III restriction enzyme sites are indicated. Sequences that when mutated significantly reduce the <em>ARS1<sup>S</sup></em> bias are indicated in red. (B) Values for percent initiation at <em>ARS1<sup>S</sup></em> are plotted for the 10 mutations that span the <em>Sma</em>I-<em>Hae</em>III fragment. The bar on the right of the graph indicates a 99% confidence interval extending on both sides of the mean percent initiation at <em>ARS1<sup>S</sup></em> for pN&amp;S based on five separate experiments. Values that fall outside of this range are considered significantly different from pN&amp;S at the 1% level. The approximate locations of the mutations relative to the 3â² half of the <em>URA3</em> gene (arrow) are shown. (C) Fork-direction gel analysis of mutants m6 and m7 as examples of the mutations that eliminate the origin bias. (D) The wild-type sequence corresponding to the sequences of the <em>URA3</em> fragment that were mutated in m5, m6, and m7 are shown in capital letters. Sequences of three different mutations (m12, â<em>Nsi</em>I, m11) are given in lowercase letters beneath the wild-type sequences they replace. Deleted bases are depicted by a dash. The overlined nucleotides were not changed by the mutagenesis while the underlined nucleotides mark the <em>Nsi</em>I site. Sequences important for the bias are indicated in red letters. Percent initiation at <em>ARS1<sup>S</sup></em> is indicated as bar graphs for the three mutations, m11, m12, and â<em>Nsi</em>I. (E) Fork direction 2-D gels of mutants m11 and m12 (see panel D for sequences). Percent initiation at <em>ARS1<sup>S</sup></em> is indicated in the upper right corners.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">To further characterize bases that are essential for the origin bias, three plasmids were constructed to have a 6-bp mutation on either the 5â² or the 3â² ends of the 29-bp sequence or containing a central 4-bp deletion. These plasmids were also analyzed via fork direction 2D gel analysis (<a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3, D and E</a>). Although the six bases on the 3â² end of the fragment are essential for origin bias, the six bases on the 5â² end and the four bases that were centrally deleted are not. These data result in the identification of a 19-bp bipartite sequence that is essential for origin bias (<a id="xref-fig-3-10" class="xref-fig" href="#F3">Figure 3D</a>, red). The essential 19 bases are consistent with the results obtained for plasmids pÎ8 and pÎ9 in our deletion analysis (see <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2</a>). Interestingly, our data suggest that although the identified 19 bases are essential for the bias they are not sufficient to cause the bias, requiring the presence of sequences within the broader 584-bp <em>Hae</em>III-<em>Nco</em>I fragment.</p></div><div id="sec-9" class="subsection"><h3>The bias determinant advances <em>ARS1<sup>S</sup></em> origin firing time relative to <em>ARS1<sup>N</sup></em></h3><p id="p-28">The observation that the bias determinant at the 3â² end of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> is influencing <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1</a>) without preferentially affecting its efficiency relative to that of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> (<a id="xref-ref-1-11" class="xref-bibr" href="#ref-1">Brewer and Fangman 1994</a>) led us to the hypothesis that the determinant causes the bias by advancing <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> activation time. Because <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> is used approximately 20% of the time it suggests that, even if the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> is usually activated earlier than is <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em>, there is some overlap in the distribution of their activation times. It seems likely, therefore, that the difference between their mean times of activation could differ by a small increment, perhaps by as little as one minute. We tested this origin timing hypothesis using three different experimental approaches.</p><div id="sec-10" class="subsection"><h4>Comparison of 3â²URA3 sequence and a centromere:</h4><p id="p-29">If the bias determinant were to impart early activation to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> we would expect that other DNA elements, such as centromeres, that are known to advance origin timing would produce a similar bias when placed in the context of the two-ARS plasmid. We would also predict that a centromere placed near the opposite copy of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> would diminish or neutralize any bias that would be imposed by the 3â² <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> bias element.</p><p id="p-30">We first asked whether a centromeric sequence could produce a similar bias as the 3â² <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> sequence in the context of pN&amp;S. To do so, we used a derivative of pN&amp;S that lacks the sequence necessary for 3â² <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> dependent bias and therefore shows no ARS bias (pÎ9, see <a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2</a>). Integration of a centromere close to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> in this plasmid (pÎ9C) resulted in a marked bias toward <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> use, indicating that sequences that are known to advance origin activation time in chromosomal DNA can in fact impose an origin use bias in pN&amp;S (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>, left). Although a centromereâs ability to advance origin activation time is bidirectional, the centromere also has a greater timing effect on closer origins than on more distant ones (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Pohl <em>et al.</em> 2012</a>). Therefore, it is conceivable that in the context of the two-ARS plasmid the centromere is advancing the activation time of both <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> copies, with <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> being advanced to a greater extent. Insertion of the centromere near <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> can also neutralize the bias imparted by the 3â² <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> sequence: <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> were equally used on a plasmid harboring both a centromere nearest to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> and the intact 3â²<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> sequences (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>, right). The equal usage of the two <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> copies can be interpreted as <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> undergoing activation at a time similar to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em>, which is replicated early in this situation due to the nearby centromere. These data suggest that the 3â² <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> sequences are capable of advancing origin activation time.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Replication kinetics of ARS1 in the presence and absence of URA3 sequences that cause biased origin activation. (A) Fork direction 2-D gel analysis was performed on plasmids in which the bias determinant is present (pN&amp;SC, illustration: *) or absent (p&#x394;9C, illustration: &#x2206;). Both plasmids contained a centromere near ARS1N (illustration: star). Percent initiation at ARS1S is indicated in the bottom left corners. (B) Replication kinetics of stable plasmids containing only ARS1S from dense isotope transfer experiments. The Trep value for each plasmid is shown as a dotted line intersecting the X-axis. Upper left: replication kinetics for a plasmid (p5-6C) containing the wild-type 3&#x2032; URA3 sequence that results in preferential use of ARS1S in the original pN&amp;S plasmid (solid blue squares). Upper right: replication kinetics for a plasmid (p5-6m7C) with a 9-bp substitution mutation in the 3&#x2032; URA3 sequence that results in equivalent origin use in the original pN&amp;S plasmid (open blue squares). The kinetics of plasmid replication are compared with a sequence that replicates early in S phase, ARS305 (dashed); in mid-S phase, chromosomal ARS1 (filled or open green diamonds); and late in S phase, R11 (dotted). The presence (*) or absence (&#x2206;) of the wild-type 3&#x2032; URA3 sequence is shown on the illustrations of the plasmids. Replication indices for the plasmids and ARS1 in each strain are shown in the lower panel." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2044600196" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Replication kinetics of &lt;em&gt;ARS1&lt;/em&gt; in the presence and absence of &lt;em&gt;URA3&lt;/em&gt; sequences that cause biased origin activation. (A) Fork direction 2-D gel analysis was performed on plasmids in which the bias determinant is present (pN&amp;SC, illustration: *) or absent (p&#x394;9C, illustration: &#x2206;). Both plasmids contained a centromere near &lt;em&gt;ARS1&lt;sup&gt;N&lt;/sup&gt;&lt;/em&gt; (illustration: star). Percent initiation at &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt; is indicated in the bottom left corners. (B) Replication kinetics of stable plasmids containing only &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt; from dense isotope transfer experiments. The &lt;em&gt;T&lt;/em&gt;&lt;sub&gt;rep&lt;/sub&gt; value for each plasmid is shown as a dotted line intersecting the X-axis. Upper left: replication kinetics for a plasmid (p5-6C) containing the wild-type 3&#x2032; &lt;em&gt;URA3&lt;/em&gt; sequence that results in preferential use of &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt; in the original pN&amp;S plasmid (solid blue squares). Upper right: replication kinetics for a plasmid (p5-6m7C) with a 9-bp substitution mutation in the 3&#x2032; &lt;em&gt;URA3&lt;/em&gt; sequence that results in equivalent origin use in the original pN&amp;S plasmid (open blue squares). The kinetics of plasmid replication are compared with a sequence that replicates early in S phase, &lt;em&gt;ARS305&lt;/em&gt; (dashed); in mid-S phase, chromosomal &lt;em&gt;ARS1&lt;/em&gt; (filled or open green diamonds); and late in S phase, R11 (dotted). The presence (*) or absence (&#x2206;) of the wild-type 3&#x2032; &lt;em&gt;URA3&lt;/em&gt; sequence is shown on the illustrations of the plasmids. Replication indices for the plasmids and &lt;em&gt;ARS1&lt;/em&gt; in each strain are shown in the lower panel.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1955/F4.medium.gif" width="316" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/11/1955/F4.medium.gif" width="316" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463550" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Replication kinetics of <em>ARS1</em> in the presence and absence of <em>URA3</em> sequences that cause biased origin activation. (A) Fork direction 2-D gel analysis was performed on plasmids in which the bias determinant is present (pN&amp;SC, illustration: *) or absent (pÎ9C, illustration: â). Both plasmids contained a centromere near <em>ARS1<sup>N</sup></em> (illustration: star). Percent initiation at <em>ARS1<sup>S</sup></em> is indicated in the bottom left corners. (B) Replication kinetics of stable plasmids containing only <em>ARS1<sup>S</sup></em> from dense isotope transfer experiments. The <em>T</em><sub>rep</sub> value for each plasmid is shown as a dotted line intersecting the X-axis. Upper left: replication kinetics for a plasmid (p5-6C) containing the wild-type 3â² <em>URA3</em> sequence that results in preferential use of <em>ARS1<sup>S</sup></em> in the original pN&amp;S plasmid (solid blue squares). Upper right: replication kinetics for a plasmid (p5-6m7C) with a 9-bp substitution mutation in the 3â² <em>URA3</em> sequence that results in equivalent origin use in the original pN&amp;S plasmid (open blue squares). The kinetics of plasmid replication are compared with a sequence that replicates early in S phase, <em>ARS305</em> (dashed); in mid-S phase, chromosomal <em>ARS1</em> (filled or open green diamonds); and late in S phase, R11 (dotted). The presence (*) or absence (â) of the wild-type 3â² <em>URA3</em> sequence is shown on the illustrations of the plasmids. Replication indices for the plasmids and <em>ARS1</em> in each strain are shown in the lower panel.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-11" class="subsection"><h4>The bias determinant promotes early ARS1<sup>S</sup> activation:</h4><p id="p-32">To further test whether the bias determinant promotes the earlier activation of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em>, replication times were determined for plasmids that contain only <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em>, either with or without a functional bias determinant. If the bias determinant were causing a delay in the activation of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em>, then deletion of the bias determinant should have no effect on the time of activation of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> and the two plasmids should replicate at the same time. If, instead, the bias determinant were advancing the activation time of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> then the plasmid with the bias determinant should replicate earlier than the one without it. Plasmid p5-6C is identical to the two-<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> plasmid, pN&amp;S, with two exceptions: (1) the 1.45-kb <em>EcoRI</em> fragment containing <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> has been removed, and (2) a 1.1-kb <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006467">CEN5</a></em> fragment has been cloned into the <em>BamHI</em> site. A second plasmid, p5-6m7C, is identical to p5-6C except that it contains a 9-bp substitution mutation, m7, which eliminates the initiation bias in the two-ARS plasmid (<a id="xref-fig-3-11" class="xref-fig" href="#F3">Figure 3C</a>). Because <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006467">CEN5</a></em> is present at the same location in both p5-6C and p5-6m7C, any timing effect that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006467">CEN5</a></em> would impart on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> would be equal in both plasmids. Integration of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006467">CEN5</a></em> does not interfere with detecting the influence of the 3â² <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> sequence on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>) and was necessary to achieve the efficient plasmid maintenance required for density transfer experiments.</p><p id="p-33">Plasmids p5-6C and p5-6m7C were transformed separately into <em>S. cerevisiae</em> and their times of replication were determined in parallel cultures using a modified version of the Meselson-Stahl density shift experiment with synchronized cultures (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">McCarroll and Fangman 1988</a>). Cultures of cells with these two different plasmids have the same population doubling times. The percent replication of p5-6C and p5-6m7C was calculated for each sample and plotted with respect to time (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>). The time of replication (<em>T</em><sub>rep</sub>) for each locus was calculated as the time at which that locus reached half maximal replication. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000028482">ARS305</a></em>, one of the earliest known origins, and R11, a late replicating fragment on chromosome V, were used as timing standards. To facilitate comparison between cultures, these <em>T</em><sub>rep</sub> values were converted to replication indices (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Friedman <em>et al.</em> 1996</a>) by assigning <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000028482">ARS305</a></em> a replication index (RI) of 0 and R11 a RI of 1.0 (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4B</a>, lower panel). Most other genomic loci have a RI between 0 and 1.0. The <em>T</em><sub>rep</sub> values for the two plasmids and a sequence immediately flanking chromosomal <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> were then converted to RIs corresponding to the fraction of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000028482">ARS305</a></em>-R11 interval elapsed when the <em>T</em><sub>rep</sub> for each locus was obtained. p5-6C (RI = â0.14) replicated 3â4 min earlier than did a chromosomal fragment containing the early replicating origin, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000028482">ARS305</a></em>, whereas p5-6m7C (RI = 0.1) replicated at the same time as <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000028482">ARS305</a></em> (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4B</a>). As expected, the replication indices for chromosomal <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> fragments were similar between the two strains (RI = 0.35 and 0.36). These results indicate that the bias determinant acts positively on the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> origin to advance its time of activation.</p><p id="p-34">Consistent with these results, the 3-min timing advantage that the bias determinant imparts on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> was recapitulated using a site-directed recombinase to split plasmid pN&amp;S into a south half and a north half <em>in vivo</em>, allowing the firing time of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>N</sup></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> to be directly compared within the same cell (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008250/-/DC1/FigureS1.pdf">Figure S1</a>). Together, the results of these three assays indicate that <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> sequences are influencing replication initiation time on the two-ARS plasmid by advancing <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> activation time, and that this difference in initiation time can explain the initiation bias. The 3- to 4-min advancement in activation time caused by the 3â²<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> sequence explains how two equally efficient ARSs can exhibit large differences in levels of activation when placed in competition on a single plasmid molecule.</p></div></div><div id="sec-12" class="subsection"><h3>The bias determinant can regulate origin activation time over a distance of at least 1 kb</h3><p id="p-35">Other <em>cis</em>-acting timing determinants that have been identified can affect replication timing over great distances. Centromeres advance origin activation time over a distance of 20 kb (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Pohl <em>et al.</em> 2012</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Natsume <em>et al.</em> 2013</a>), whereas telomeres delay the activation time of origins over about 30 kb (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Ferguson <em>et al.</em> 1991</a>). The activation times of two origins on chromosome XIV, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000077374">ARS1412</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000077375">ARS1413</a></em>, are delayed by cryptic sequences that reside up to 6 kb away (<a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Friedman <em>et al.</em> 1996</a>). The forkhead transcription factors, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001393">Fkh1</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005012">Fkh2</a>, are capable of advancing origin activation time over a distance of at least 500 bp (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Knott <em>et al.</em> 2012</a>). We therefore sought to determine the distance over which the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> timing determinant can act. An 8-bp <em>Xho</em>I linker was inserted at the <em>Sma</em>I/<em>Eco</em>RV junction between <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> and the bias determinant to provide a cloning site for several subsequent insertions. This small insertion had no effect on the bias (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>). Insertion of either one or two copies of a 75-bp <em>Xho</em>I PCR lambda fragment at the XhoI site (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5, B and C</a>) or of a 604-bp lambda <em>Stu</em>I fragment at the <em>Sma</em>I/<em>Eco</em>RV junction (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5D</a>) also had no effect on the bias. Because the distance from the <em>Sma</em>I/<em>Eco</em>RV junction to the ACS of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> is 465 bp, these data indicate that the bias determinant can function over a distance of at least 1069 bp.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Determining distances over which the bias determinant can function. (A-D) Fork direction 2-D gel analysis on pN&amp;S plasmids that have various lengths of sequence inserted within the NcoI-SmaI fragment, between the bias determinant and ARS1S. The amount of sequence that was inserted for each construct is indicated above each image. Percent initiation at ARS1S is indicated in the lower left corners. (E) Replication indices for chromosomal URA3 (pink) and ARS510 (blue) in wild-type (WT) cells harboring the bias determinant at its native location (closed diamonds) and in cells lacking the bias determinant (open diamonds) are plotted. ARS607 (black dashed arrow) and R11 (black dotted arrow) were used as early and late timing standards, respectively. In the WT strain, URA3 and ARS510 had replication indices of 0.73 and 0.4, respectively. In the strain lacking the bias determinant (YTP38), URA3 had a replication index of 0.66, whereas the replication index for ARS510 was 0.34." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2044600196" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Determining distances over which the bias determinant can function. (A-D) Fork direction 2-D gel analysis on pN&amp;S plasmids that have various lengths of sequence inserted within the &lt;em&gt;Nco&lt;/em&gt;I-&lt;em&gt;Sma&lt;/em&gt;I fragment, between the bias determinant and &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt;. The amount of sequence that was inserted for each construct is indicated above each image. Percent initiation at &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt; is indicated in the lower left corners. (E) Replication indices for chromosomal &lt;em&gt;URA3&lt;/em&gt; (pink) and &lt;em&gt;ARS510&lt;/em&gt; (blue) in wild-type (WT) cells harboring the bias determinant at its native location (closed diamonds) and in cells lacking the bias determinant (open diamonds) are plotted. &lt;em&gt;ARS607&lt;/em&gt; (black dashed arrow) and R11 (black dotted arrow) were used as early and late timing standards, respectively. In the WT strain, &lt;em&gt;URA3&lt;/em&gt; and &lt;em&gt;ARS510&lt;/em&gt; had replication indices of 0.73 and 0.4, respectively. In the strain lacking the bias determinant (YTP38), &lt;em&gt;URA3&lt;/em&gt; had a replication index of 0.66, whereas the replication index for &lt;em&gt;ARS510&lt;/em&gt; was 0.34.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1955/F5.medium.gif" width="440" height="250"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/11/1955/F5.medium.gif" width="440" height="250"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463532" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-36" class="first-child">Determining distances over which the bias determinant can function. (A-D) Fork direction 2-D gel analysis on pN&amp;S plasmids that have various lengths of sequence inserted within the <em>Nco</em>I-<em>Sma</em>I fragment, between the bias determinant and <em>ARS1<sup>S</sup></em>. The amount of sequence that was inserted for each construct is indicated above each image. Percent initiation at <em>ARS1<sup>S</sup></em> is indicated in the lower left corners. (E) Replication indices for chromosomal <em>URA3</em> (pink) and <em>ARS510</em> (blue) in wild-type (WT) cells harboring the bias determinant at its native location (closed diamonds) and in cells lacking the bias determinant (open diamonds) are plotted. <em>ARS607</em> (black dashed arrow) and R11 (black dotted arrow) were used as early and late timing standards, respectively. In the WT strain, <em>URA3</em> and <em>ARS510</em> had replication indices of 0.73 and 0.4, respectively. In the strain lacking the bias determinant (YTP38), <em>URA3</em> had a replication index of 0.66, whereas the replication index for <em>ARS510</em> was 0.34.</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">An alternative, though unlikely, interpretation of this experiment is that the bias determinant is not able to function over this distance and that instead, the lambda sequences are promoting an initiation bias of their own. To eliminate this possibility, the bias determinant was compromised in the pN&amp;S+604 bp construct by an <em>Nco</em>I-<em>Nsi</em>I deletion (see pÎ43 in <a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2</a>). This construct exhibits no bias (data not shown), indicating that the lambda sequences are not capable of conferring a bias on their own.</p><p id="p-38">At its native location <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> resides at a distance from its neighboring origins that is ideal to test the outer limits over which the bias element can function. In the genome, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> is flanked by two efficient early activating origins, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000077381">ARS508</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000077382">ARS510</a></em>, that are located at respective distances of 21.9 and 28.8 kb from <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Nieduszynski <em>et al.</em> 2007</a>). However, only <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000077382">ARS510</a></em> is in the correct orientation to have its activation time regulated by the bias determinant. Therefore, we expect that if the bias determinant regulates origin activation at its native location its effect would be on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000077382">ARS510</a></em>. To determine whether the bias determinant influences replication at its native genomic location, we performed a density shift experiment on cells in which the chromosomal copy of the 3â² <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> bias element had been removed (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5E</a>). Cells lacking the bias determinant replicated both the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> gene and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000077382">ARS510</a></em> similar to control cells, suggesting that the bias determinant does not detectably regulate the activation time of flanking origins at its native location. These results could also be obtained if the bias determinant was effective only on plasmids and incapable of advancing origin activation time in the context of a linear chromosome. However, we can rule out this possibility because the bias effect persists when pN&amp;S is integrated into the chromosomal <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> locus (<a id="xref-ref-1-12" class="xref-bibr" href="#ref-1">Brewer and Fangman 1994</a>).</p><p id="p-39">Taken together, these experiments indicate that the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> timing determinant can act over a distance of at least 1 kb but not over a distance of tens of kilobases. As more sequences that are capable of regulating origin activation time are identified, it will be interesting to determine more precise limits or boundaries for each given regulator.</p></div><div id="sec-13" class="subsection"><h3>DNA binding motif search of the bias determinant</h3><p id="p-40">The effects of deletions in the 3â² <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> region and of the inversion of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> on bias activity indicate that the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> element and the bias determinant are separate entities. Does the bias determinant communicate with the ARS through the binding of specific proteins to the bias determinant? On the basis of our observation that the 3â² end of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> consists of a complex, multipartite timing determinant, we were aware that identifying any <em>trans</em>-acting factor(s) mediating the effect of this determinant might be challenging, for example, that redundancy introduced by multiple <em>trans</em>-acting factors interacting with the determinant might complicate the analysis. Nevertheless, finding that at least one component of the determinant could be narrowed to a small (19â23 bp) element encouraged us to embark on a search for proteins such as transcription factors that might mediate the timing effect by binding to that sequence. To this end, we first considered proteins known to influence origin activity. <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> contains a binding site for the transcription factor <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001595">Abf1</a> in element B3 that contributes to ARS activity (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Marahrens and Stillman 1992</a>), and it has been demonstrated for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007553">ARS121</a></em> that a second <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001595">Abf1</a> binding site, outside of the B region, contributes to maintenance of an <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000007553">ARS121</a></em> plasmid under some culture conditions (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Walker <em>et al.</em> 1989</a>). In addition, a binding site for a different transcription factor (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005160">Rap1</a>) was shown to substitute for the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001595">Abf1</a> site at <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Marahrens and Stillman 1992</a>). Although the mechanism by which transcription factors enhance ARS function is not known, these observations suggest that transcription factor binding might contribute to the bias observed in pN&amp;S. There is a potential binding site for <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001595">Abf1</a> and one for <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005160">Rap1</a> in the 90 bp <em>Sma</em>I-<em>Hae</em>III fragment of the bias element region. However, neither of these potential sites overlaps the 23 bp essential core element identified by scanning mutagenesis (<a id="xref-fig-3-12" class="xref-fig" href="#F3">Figure 3</a>), indicating that these transcription factors are not primarily responsible for the origin initiation bias. Given the complexity of the bias determinant, however, a possible redundant role for <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001595">Abf1</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005160">Rap1</a> cannot be excluded.</p><p id="p-41">The forkhead transcription factors, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001393">Fkh1</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005012">Fkh2</a>, recently have been shown to influence the initiation time of some origins (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Knott <em>et al.</em> 2012</a>), raising the possibility that these transcription factors might have some role in the observed bias determinant. Furthermore, there are potential binding sites for <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001393">Fkh1</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005012">Fkh2</a> close to or partially overlapping the core bias element (<a id="xref-fig-3-13" class="xref-fig" href="#F3">Figure 3A</a>). However, these binding sites do not overlap with all sequences that were shown to be crucial for the bias (see m11, <a id="xref-fig-3-14" class="xref-fig" href="#F3">Figure 3D</a>), suggesting that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001393">Fkh1</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005012">Fkh2</a> are likely also not involved in the observed origin initiation bias. Consistent with this logic, preliminary chromatin immunoprecipitation studies performed by the Fox lab indicate that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001393">Fkh1</a> does not bind this sequence at its native chromosomal location (T. Hoggard and C. Fox, personal communication).</p><p id="p-42">Because the obvious candidates are likely not responsible for the bias, we embarked on a more systematic search for <em>trans</em>-acting factors that might bind to the bias determinant. Both the essential 23-bp sequence and the 19-bp sequence (with the four central bases deleted) were run through the motif comparison tool TOMTOM (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gupta <em>et al.</em> 2007</a>). In both instances, binding motifs for multiple proteins were identified, many of which are specific to transcription factors (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Motif search of sequences important for the bias determinant. (A) Both the 23-bp (+TGCA) and the bipartite 19-bp (&#x2212;TGCA) sequences were run through the motif identification tool TOMTOM (Gupta et al. 2007). Candidate proteins that were identified for each sequence are listed in order of significance (top to bottom) with proteins common to both searches in the last column. Transcription factors are indicated by red text. (B) The WT sequence corresponding to the 23 bps (+TGCA) that are required for the bias is indicated by small black capital letters. The NsiI site is underlined. The logos for Cst6 and Yap3 binding motifs are indicated above or below the corresponding sequence. Percent initiation at ARS1S on pN&amp;S in cells lacking either CST6 or YAP3 are indicated as bar graphs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2044600196" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Motif search of sequences important for the bias determinant. (A) Both the 23-bp (+TGCA) and the bipartite 19-bp (&#x2212;TGCA) sequences were run through the motif identification tool TOMTOM (Gupta &lt;em&gt;et al.&lt;/em&gt; 2007). Candidate proteins that were identified for each sequence are listed in order of significance (top to bottom) with proteins common to both searches in the last column. Transcription factors are indicated by red text. (B) The WT sequence corresponding to the 23 bps (+TGCA) that are required for the bias is indicated by small black capital letters. The &lt;em&gt;Nsi&lt;/em&gt;I site is underlined. The logos for Cst6 and Yap3 binding motifs are indicated above or below the corresponding sequence. Percent initiation at &lt;em&gt;ARS1&lt;sup&gt;S&lt;/sup&gt;&lt;/em&gt; on pN&amp;S in cells lacking either &lt;em&gt;CST6&lt;/em&gt; or &lt;em&gt;YAP3&lt;/em&gt; are indicated as bar graphs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1955/F6.medium.gif" width="440" height="318"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/3/11/1955/F6.medium.gif" width="440" height="318"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1955/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463560" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-43" class="first-child">Motif search of sequences important for the bias determinant. (A) Both the 23-bp (+TGCA) and the bipartite 19-bp (âTGCA) sequences were run through the motif identification tool TOMTOM (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Gupta <em>et al.</em> 2007</a>). Candidate proteins that were identified for each sequence are listed in order of significance (top to bottom) with proteins common to both searches in the last column. Transcription factors are indicated by red text. (B) The WT sequence corresponding to the 23 bps (+TGCA) that are required for the bias is indicated by small black capital letters. The <em>Nsi</em>I site is underlined. The logos for Cst6 and Yap3 binding motifs are indicated above or below the corresponding sequence. Percent initiation at <em>ARS1<sup>S</sup></em> on pN&amp;S in cells lacking either <em>CST6</em> or <em>YAP3</em> are indicated as bar graphs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-44">The top hit in both searches was the binding motif for the transcription factor <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001298">Cst6</a> (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). Of particular interest was the finding that while the Cts6 binding motif overlaps the <em>Nsi</em>I site, the motif would be restored in the four-base (TGCA) deletion (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>). The next candidate on the list was the transcription factor, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001001">Yap3</a>. Like all other proteins on the list, the <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001001">Yap3</a> consensus binding motif only overlapped with part of the sequence that was found to be essential for the bias and like many of the other proteins <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001001">Yap3</a> has an imperfect target-site match (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a>). To determine whether either of the top two hits (<a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001298">Cst6</a> or <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001001">Yap3</a>) was responsible for the bias, plasmid pN&amp;S was transformed into cells lacking either <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001298"><em>CST</em>6</a> or <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001001">YAP3</a></em> and analyzed for the presence or absence of the bias (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6B</a>). As in WT cells, an initiation bias for <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> was observed for pN&amp;S indicating that neither <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001298">Cst6</a> nor <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001001">Yap3</a> binding is responsible for the bias.</p><p id="p-45">Because we have excluded the âtopâ two candidates from our motif search, a systematic approach will need to be taken to analyze the remaining proteins that are potentially involved. However, from these studies, we have concluded that given the large number of potential binding candidates, the degeneracy of transcription factor binding sites, the possibility of redundancy at the bipartite site, and the labor involved in testing each candidate, an approach other than gene knockouts will be required to identify the putative <em>trans</em>-acting factors that regulate this initiation timing determinant. Furthermore, it is possible that none of the candidate proteins identified in our motif search are uniquely involved in creation of the bias. This possibility is supported by the observation that none of the motifs for the remaining candidates span the entire 23-bp motif that is essential to the bias. It is worth noting that the percentages of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> usage in both <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001298">CST6</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001001">YAP3</a></em> knockout strains are either very close to or slightly below the bottom of the confidence interval. Though these data were not deemed significantly different from pN&amp;S, it would be interesting to determine how a double knockout of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001298">CST6</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001001">YAP3</a></em> would affect <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> bias. The observation that double knockouts of transcription factors can present adverse phenotypes such as pseudohyphal growth (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Knott <em>et al.</em> 2012</a>) adds to the complexity of such studies.</p><p id="p-46">Finally, we considered the possibility that bias determinant acts directly via some intrinsic property of its own, not necessarily mediated by the binding of <em>trans</em>-acting factors. The B2 element of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> is part of a DNA unwinding element (DUE), which is believed to enhance ARS function through its inherent helical instability (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Natale <em>et al.</em> 1992</a>, <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">1993</a>). Computer analysis of the <em>Sma</em>I-<em>Hae</em>III fragment of pN&amp;S shows that it is part of a DUE (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Natale <em>et al.</em> 1992</a>, <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">1993</a>). Does this DUE contribute, autonomously or in conjunction with the binding of a protein, to earlier activation of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em>? We made several site-directed mutations of the 19-bp core sequence, scrambling the sequence without altering the predicted helical stability (see the section <em>Materials and Methods</em>). These mutations did substantially reduce the origin bias (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008250/-/DC1/FigureS2.pdf">Figure S2</a>). Therefore, the 19-bp sequence appears to contribute to origin bias in a way that depends on its specific sequence and not simply on its propensity to unwind. Although a role for this particular DUE at the 3â² end of the <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> gene in origin bias cannot be excluded, a DUE alone is not sufficient to advance the time of origin firing.</p><p id="p-47">Other than our studies examining the bias determinant at its native location, the ability of this early timing determinant to influence other ARS elements has not yet been tested. The possible response of other ARS elements can be approached first, and most simply, by using the origin bias assay. However, it may be that response to the early determinant requires specific sequences within <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em>. If so, the sequence elements in <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> that respond to the determinant could be identified by the use of mutagenesis of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a><sup>S</sup></em> in the context of the two-ARS plasmid assay. For any mutation that reduces the origin bias, it would be important to alter both ARSs in the plasmid identically to distinguish between mutations that reduce origin function from those that are specific to the originâs ability to respond to the bias determinant. In addition, the bias assay with the two-ARS plasmid has the potential for providing a sensitive tool for characterizing other <em>cis</em>-acting factors involved in replication initiation timing control. For example, sequences on chromosome XIV have been shown to delay the time of activation of origins that they flank, and they have been shown to act on <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000029652">ARS1</a></em> transplanted from chromosome IV (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Friedman <em>et al.</em> 1996</a>). These late determinants can delay origin initiation by 10 min or more, and might cause an even greater bias in the two-ARS plasmid than that conferred by the 3â² <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000747">URA3</a></em> early determinant.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">We thank past and present members of the Brewer/Raghuraman lab along with members of the Dunham lab for thoughtful discussions and comments on this manuscript. This research was supported by a National Institute of General Medical Sciences grant (18926) to W.L.F., B.J.B., and M.K.R. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. The work conducted by Katherine Kolor was performed in her private capacity. No official support or endorsement by the Centers for Disease Control and Prevention, Department of Health and Human Services is intended, nor should it be inferred.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008250/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.008250/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-49"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-07-16"><span class="received-label">Received </span>July 16, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-09-03"><span class="accepted-label">Accepted </span>September 3, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2013 Pohl <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.1" data-doi="10.1073/pnas.91.8.3418"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fangman</span>  <span class="cit-name-given-names">W. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span>â<span class="cit-article-title">Initiation preference at a yeast origin of replication.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">3418</span>â<span class="cit-lpage">3422</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBrewer%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D91%26rft.issue%253D8%26rft.spage%253D3418%26rft.epage%253D3422%26rft.atitle%253DInitiation%2Bpreference%2Bat%2Ba%2Byeast%2Borigin%2Bof%2Breplication.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.91.8.3418%26rft_id%253Dinfo%253Apmid%252F8159762%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5MS84LzM0MTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.2" data-doi="10.1016/0092-8674(92)90355-G"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lockshon</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fangman</span>  <span class="cit-name-given-names">W. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-article-title">The arrest of replication forks in the rDNA of yeast occurs independently of transcription.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">267</span>â<span class="cit-lpage">276</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBrewer%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D71%26rft.issue%253D2%26rft.spage%253D267%26rft.epage%253D276%26rft.atitle%253DThe%2Barrest%2Bof%2Breplication%2Bforks%2Bin%2Bthe%2BrDNA%2Bof%2Byeast%2Boccurs%2Bindependently%2Bof%2Btranscription.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252892%252990355-G%26rft_id%253Dinfo%253Apmid%252F1423594%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(92)90355-G&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1423594&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JU39500012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.3" data-doi="10.1101/gad.231602"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cosgrove</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nieduszynski</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donaldson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Ku complex controls the replication time of DNA in telomere regions.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">2485</span>â<span class="cit-lpage">2490</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DCosgrove%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D16%26rft.issue%253D19%26rft.spage%253D2485%26rft.epage%253D2490%26rft.atitle%253DKu%2Bcomplex%2Bcontrols%2Bthe%2Breplication%2Btime%2Bof%2BDNA%2Bin%2Btelomere%2Bregions%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.231602%26rft_id%253Dinfo%253Apmid%252F12368259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE2LzE5LzI0ODUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nickoloff</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-article-title">Site-directed mutagenesis of virtually any plasmid by eliminating a unique site.</span> <abbr class="cit-jnl-abbrev">Anal. Biochem.</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnal.%2BBiochem.%26rft.volume%253D200%26rft.spage%253D8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.5" data-doi="10.1016/0092-8674(92)90474-Q"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fangman</span>  <span class="cit-name-given-names">W. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-article-title">A position effect on the time of replication origin activation in yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">333</span>â<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DFerguson%26rft.auinit1%253DB.%2BM.%26rft.volume%253D68%26rft.issue%253D2%26rft.spage%253D333%26rft.epage%253D339%26rft.atitle%253DA%2Bposition%2Beffect%2Bon%2Bthe%2Btime%2Bof%2Breplication%2Borigin%2Bactivation%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252892%252990474-Q%26rft_id%253Dinfo%253Apmid%252F1733502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(92)90474-Q&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1733502&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HB35300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.6" data-doi="10.1016/0092-8674(91)90468-E"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fangman</span>  <span class="cit-name-given-names">W. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span>â<span class="cit-article-title">A yeast origin of replication is activated late in S phase.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">507</span>â<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DFerguson%26rft.auinit1%253DB.%2BM.%26rft.volume%253D65%26rft.issue%253D3%26rft.spage%253D507%26rft.epage%253D515%26rft.atitle%253DA%2Byeast%2Borigin%2Bof%2Breplication%2Bis%2Bactivated%2Blate%2Bin%2BS%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252891%252990468-E%26rft_id%253Dinfo%253Apmid%252F2018976%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(91)90468-E&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2018976&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FK18200017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.7" data-doi="10.1016/0076-6879(95)62048-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">Analysis of replication intermediates by two-dimensional agarose gel electrophoresis.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">262</span>: <span class="cit-fpage">613</span>â<span class="cit-lpage">627</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DFriedman%26rft.auinit1%253DK.%2BL.%26rft.volume%253D262%26rft.spage%253D613%26rft.epage%253D627%26rft.atitle%253DAnalysis%2Bof%2Breplication%2Bintermediates%2Bby%2Btwo-dimensional%2Bagarose%2Bgel%2Belectrophoresis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0076-6879%252895%252962048-6%26rft_id%253Dinfo%253Apmid%252F8594382%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0076-6879(95)62048-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8594382&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995BE40P00046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghuraman</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fangman</span>  <span class="cit-name-given-names">W. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">Analysis of the temporal program of replication initiation in yeast chromosomes.</span> <abbr class="cit-jnl-abbrev">J. Cell Sci. Suppl.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">51</span>â<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bcell%2Bscience.%2B%2BSupplement%26rft.stitle%253DJ%2BCell%2BSci%2BSuppl%26rft.aulast%253DFriedman%26rft.auinit1%253DK.%2BL.%26rft.volume%253D19%26rft.spage%253D51%26rft.epage%253D58%26rft.atitle%253DAnalysis%2Bof%2Bthe%2Btemporal%2Bprogram%2Bof%2Breplication%2Binitiation%2Bin%2Byeast%2Bchromosomes.%26rft_id%253Dinfo%253Apmid%252F8655647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=8655647&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.9" data-doi="10.1101/gad.10.13.1595"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diller</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nyland</span>  <span class="cit-name-given-names">S. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Multiple determinants controlling activation of yeast replication origins late in S phase.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">1595</span>â<span class="cit-lpage">1607</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DFriedman%26rft.auinit1%253DK%2BL%26rft.volume%253D10%26rft.issue%253D13%26rft.spage%253D1595%26rft.epage%253D1607%26rft.atitle%253DMultiple%2Bdeterminants%2Bcontrolling%2Bactivation%2Bof%2Byeast%2Breplication%2Borigins%2Blate%2Bin%2BS%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.10.13.1595%26rft_id%253Dinfo%253Apmid%252F8682291%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjEwLzEzLzE1OTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.10" data-doi="10.1186/gb-2007-8-2-r24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamatoyannopoulos</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">W. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Quantifying similarity between motifs.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DGupta%26rft.auinit1%253DS.%26rft.volume%253D8%26rft.issue%253D2%26rft.spage%253DR24%26rft.epage%253DR24%26rft.atitle%253DQuantifying%2Bsimilarity%2Bbetween%2Bmotifs.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-2-r24%26rft_id%253Dinfo%253Apmid%252F17324271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-2-r24&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17324271&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.11" data-doi="10.1016/0378-1119(87)90131-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffman</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winston</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span>â<span class="cit-article-title">A ten-minute DNA preparation from yeast efficiently releases autonomous plasmids for transformation of <em>Escherichia coli</em>.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">267</span>â<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DHoffman%26rft.auinit1%253DC.%2BS.%26rft.volume%253D57%26rft.issue%253D2-3%26rft.spage%253D267%26rft.epage%253D272%26rft.atitle%253DA%2Bten-minute%2BDNA%2Bpreparation%2Bfrom%2Byeast%2Befficiently%2Breleases%2Bautonomous%2Bplasmids%2Bfor%2Btransformation%2Bof%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252887%252990131-4%26rft_id%253Dinfo%253Apmid%252F3319781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(87)90131-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3319781&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987K812400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.12" data-doi="10.1016/0092-8674(87)90643-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huberman</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spotila</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nawotka</span>  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">el-Assouli</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">L. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span>â<span class="cit-article-title">The in vivo replication origin of the yeast 2 microns plasmid.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">473</span>â<span class="cit-lpage">481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHuberman%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D51%26rft.issue%253D3%26rft.spage%253D473%26rft.epage%253D481%26rft.atitle%253DThe%2Bin%2Bvivo%2Breplication%2Borigin%2Bof%2Bthe%2Byeast%2B2%2Bmicrons%2Bplasmid.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252887%252990643-X%26rft_id%253Dinfo%253Apmid%252F3311385%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(87)90643-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3311385&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987K785400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.13" data-doi="10.1016/j.cell.2011.12.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Knott</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peace</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ostrow</span>  <span class="cit-name-given-names">A. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rex</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Forkhead transcription factors establish origin timing and long-range clustering in <em>S. cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">99</span>â<span class="cit-lpage">111</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DKnott%26rft.auinit1%253DS.%2BR.%26rft.volume%253D148%26rft.issue%253D1-2%26rft.spage%253D99%26rft.epage%253D111%26rft.atitle%253DForkhead%2Btranscription%2Bfactors%2Bestablish%2Borigin%2Btiming%2Band%2Blong-range%2Bclustering%2Bin%2BS.%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.12.012%26rft_id%253Dinfo%253Apmid%252F22265405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.12.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22265405&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299540700017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.14" data-doi="10.1091/mbc.E10-06-0549"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lian</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">E. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiraga</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvino</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collingwood</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">The effect of Ku on telomere replication time is mediated by telomere length but is independent of histone tail acetylation.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1753</span>â<span class="cit-lpage">1765</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.E10-06-0549%26rft_id%253Dinfo%253Apmid%252F21441303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMC8xNzUzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.15" data-doi="10.1126/science.1536007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marahrens</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stillman</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-article-title">A yeast chromosomal origin of DNA replication defined by multiple functional elements.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">255</span>: <span class="cit-fpage">817</span>â<span class="cit-lpage">823</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMarahrens%26rft.auinit1%253DY%26rft.volume%253D255%26rft.issue%253D5046%26rft.spage%253D817%26rft.epage%253D823%26rft.atitle%253DA%2Byeast%2Bchromosomal%2Borigin%2Bof%2BDNA%2Breplication%2Bdefined%2Bby%2Bmultiple%2Bfunctional%2Belements%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1536007%26rft_id%253Dinfo%253Apmid%252F1536007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyNTUvNTA0Ni84MTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.16" data-doi="10.1016/0092-8674(88)90072-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCarroll</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fangman</span>  <span class="cit-name-given-names">W. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span>â<span class="cit-article-title">Time of replication of yeast centromeres and telomeres.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">505</span>â<span class="cit-lpage">513</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMcCarroll%26rft.auinit1%253DR.%2BM.%26rft.volume%253D54%26rft.issue%253D4%26rft.spage%253D505%26rft.epage%253D513%26rft.atitle%253DTime%2Bof%2Breplication%2Bof%2Byeast%2Bcentromeres%2Band%2Btelomeres.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252888%252990072-4%26rft_id%253Dinfo%253Apmid%252F3042152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(88)90072-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3042152&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988P798400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.17" data-doi="10.1534/genetics.108.094359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCune</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danielson</span>  <span class="cit-name-given-names">L. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvino</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collingwood</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Delrow</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">The temporal program of chromosome replication: genomewide replication in clb5â <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">1833</span>â<span class="cit-lpage">1847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.094359%26rft_id%253Dinfo%253Apmid%252F18832352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC80LzE4MzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.18" data-doi="10.1073/pnas.89.7.2654"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Natale</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schubert</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalski</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-article-title">DNA helical stability accounts for mutational defects in a yeast replication origin.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">2654</span>â<span class="cit-lpage">2658</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DNatale%26rft.auinit1%253DD.%2BA.%26rft.volume%253D89%26rft.issue%253D7%26rft.spage%253D2654%26rft.epage%253D2658%26rft.atitle%253DDNA%2Bhelical%2Bstability%2Baccounts%2Bfor%2Bmutational%2Bdefects%2Bin%2Ba%2Byeast%2Breplication%2Borigin.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.89.7.2654%26rft_id%253Dinfo%253Apmid%252F1557369%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4OS83LzI2NTQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.19" data-doi="10.1093/nar/21.3.555"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Natale</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Umek</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kowalski</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span>â<span class="cit-article-title">Ease of DNA unwinding is a conserved property of yeast replication origins.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">555</span>â<span class="cit-lpage">560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DNatale%26rft.auinit1%253DD.%2BA.%26rft.volume%253D21%26rft.issue%253D3%26rft.spage%253D555%26rft.epage%253D560%26rft.atitle%253DEase%2Bof%2BDNA%2Bunwinding%2Bis%2Ba%2Bconserved%2Bproperty%2Bof%2Byeast%2Breplication%2Borigins%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F21.3.555%26rft_id%253Dinfo%253Apmid%252F8441667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjIxLzMvNTU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.20" data-doi="10.1016/j.molcel.2013.05.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Natsume</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katou</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Retkute</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gierlinski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Kinetochores coordinate pericentromeric cohesion and early DNA replication by cdc7-dbf4 kinase recruitment.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">661</span>â<span class="cit-lpage">674</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D50%26rft.spage%253D661%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2013.05.011%26rft_id%253Dinfo%253Apmid%252F23746350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2013.05.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23746350&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000320678500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-book" id="cit-3.11.1955.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newlon</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-source">DNA Replication in Yeast</span>. <span class="cit-publ-name">Cold Spring Harbor Press</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.22" data-doi="10.1093/nar/gkl758"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nieduszynski</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiraga</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ak</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Benham</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donaldson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">OriDB: a DNA replication origin database.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">D40</span>â<span class="cit-lpage">D46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl758%26rft_id%253Dinfo%253Apmid%252F17065467%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE0OiIzNS9zdXBwbF8xL0Q0MCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMTk1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.23" data-doi="10.1371/journal.pgen.1002677"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pohl</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brewer</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghuraman</span>  <span class="cit-name-given-names">M. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Functional centromeres determine the activation time of pericentric origins of DNA replication in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002677</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPohl%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253De1002677%26rft.epage%253De1002677%26rft.atitle%253DFunctional%2Bcentromeres%2Bdetermine%2Bthe%2Bactivation%2Btime%2Bof%2Bpericentric%2Borigins%2Bof%2BDNA%2Breplication%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002677%26rft_id%253Dinfo%253Apmid%252F22589733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002677&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22589733&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.24" data-doi="10.1126/science.294.5540.115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raghuraman</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winzeler</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collingwood</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wodicka</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Replication dynamics of the yeast genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">294</span>: <span class="cit-fpage">115</span>â<span class="cit-lpage">121</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DRaghuraman%26rft.auinit1%253DM.%2BK.%26rft.volume%253D294%26rft.issue%253D5540%26rft.spage%253D115%26rft.epage%253D121%26rft.atitle%253DReplication%2BDynamics%2Bof%2Bthe%2BYeast%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.294.5540.115%26rft_id%253Dinfo%253Apmid%252F11588253%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTQvNTU0MC8xMTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.25" data-doi="10.1083/jcb.85.1.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rivin</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fangman</span>  <span class="cit-name-given-names">W. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span>â<span class="cit-article-title">Replication fork rate and origin activation during the S phase of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">108</span>â<span class="cit-lpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BCell%2BBiology%26rft.stitle%253DJCB%26rft.aulast%253DRivin%26rft.auinit1%253DC.%2BJ.%26rft.volume%253D85%26rft.issue%253D1%26rft.spage%253D108%26rft.epage%253D115%26rft.atitle%253DReplication%2Bfork%2Brate%2Band%2Borigin%2Bactivation%2Bduring%2Bthe%2BS%2Bphase%2Bof%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.85.1.108%26rft_id%253Dinfo%253Apmid%252F6767729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjg6Ijg1LzEvMTA4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.26" data-doi="10.1101/gad.13.2.146"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stevenson</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gottschling</span>  <span class="cit-name-given-names">D. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Telomeric chromatin modulates replication timing near chromosome ends.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">146</span>â<span class="cit-lpage">151</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DStevenson%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D13%26rft.issue%253D2%26rft.spage%253D146%26rft.epage%253D151%26rft.atitle%253DTelomeric%2Bchromatin%2Bmodulates%2Breplication%2Btiming%2Bnear%2Bchromosome%2Bends%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.13.2.146%26rft_id%253Dinfo%253Apmid%252F9925638%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTMvMi8xNDYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francesconi</span>  <span class="cit-name-given-names">S. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tye</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisenberg</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span>â<span class="cit-article-title">The OBF1 protein and its DNA-binding site are important for the function of an autonomously replicating sequence in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">2914</span>â<span class="cit-lpage">2921</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DWalker%26rft.auinit1%253DS%2BS%26rft.volume%253D9%26rft.issue%253D7%26rft.spage%253D2914%26rft.epage%253D2921%26rft.atitle%253DThe%2BOBF1%2Bprotein%2Band%2Bits%2BDNA-binding%2Bsite%2Bare%2Bimportant%2Bfor%2Bthe%2Bfunction%2Bof%2Ban%2Bautonomously%2Breplicating%2Bsequence%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Apmid%252F2674674%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjg6IjkvNy8yOTE0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTU1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.28" data-doi="10.1128/MCB.13.7.4057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wellinger</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zakian</span>  <span class="cit-name-given-names">V. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span>â<span class="cit-article-title">Origin activation and formation of single-strand TG1â3 tails occur sequentially in late S phase on a yeast linear plasmid.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">4057</span>â<span class="cit-lpage">4065</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DWellinger%26rft.auinit1%253DR%2BJ%26rft.volume%253D13%26rft.issue%253D7%26rft.spage%253D4057%26rft.epage%253D4065%26rft.atitle%253DOrigin%2Bactivation%2Band%2Bformation%2Bof%2Bsingle-strand%2BTG1-3%2Btails%2Boccur%2Bsequentially%2Bin%2Blate%2BS%2Bphase%2Bon%2Ba%2Byeast%2Blinear%2Bplasmid.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.13.7.4057%26rft_id%253Dinfo%253Apmid%252F8321213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjEzLzcvNDA1NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMTk1NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-3.11.1955.29" data-doi="10.1046/j.1365-2443.2002.00559.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yabuki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Terashima</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kitada</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Mapping of early firing origins on a replication profile of budding yeast.</span> <abbr class="cit-jnl-abbrev">Genes Cells</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">781</span>â<span class="cit-lpage">789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2Bto%2BCells%26rft.stitle%253DGENES%2BCELLS%26rft.aulast%253DYabuki%26rft.auinit1%253DN%26rft.volume%253D7%26rft.issue%253D8%26rft.spage%253D781%26rft.epage%253D789%26rft.atitle%253DMapping%2Bof%2Bearly%2Bfiring%2Borigins%2Bon%2Ba%2Breplication%2Bprofile%2Bof%2Bbudding%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-2443.2002.00559.x%26rft_id%253Dinfo%253Apmid%252F12167157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-2443.2002.00559.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12167157&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1955.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177365700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/11/1955.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/1945" title="Comparative Analyses Identify the Contributions of Exotic Donors to Disease Resistance in a Barley Experimental Population" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/11/1965" title="Characterization and Prediction of Haploinsufficiency Using Systems-Level Gene Properties in Yeast" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/11"><span>Volume 3 Issue 11, November 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/11.cover-source.jpg?itok=A7iZ1ztq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/1955.long" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jxoXG7ub_8guSQ9s_xxgvrVK-9Ceyc2LIRpIHF0y3RU" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/11/1955" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/11/1955" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/11/1955" class="active" data-icon-position="" data-hide-link-title="0">A DNA Sequence Element That Advances Replication Origin Activation Time in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459580" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-s4U4GFpd3sOetrHG5gYmjsbr-PnzOu0-RtN1OpYx2FQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/1955.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/11/1955" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-MgZjABZ9DQ-OJQcvElRdm0r8V_fhaa_uEAjoS3J2Av4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/11/1955" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/1955.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459580" id="node459580--2" data-pisa="ggg;3/11/1955" data-pisa-master="ggg;g3.113.008250" data-apath="/ggg/3/11/1955.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A DNA Sequence Element That Advances Replication Origin Activation Time in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Thomas J. Pohl</span>, <span class="highwire-citation-author" data-delta="1">Katherine Kolor</span>, <span class="highwire-citation-author" data-delta="2">Walton L. Fangman</span>, <span class="highwire-citation-author" data-delta="3">Bonita J. Brewer</span> and <span class="highwire-citation-author" data-delta="4">M. K. Raghuraman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1955-1963;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008250 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459580--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thomas J. Pohl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular and Cellular Biology Program, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BJ.%2BPohl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pohl%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BJ.%2BPohl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katherine Kolor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Office of Public Health Genomics, Centers for Disease Control and Prevention, Atlanta, Georgia 30329</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katherine%2BKolor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kolor%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatherine%2BKolor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Walton L. Fangman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Walton%2BL.%2BFangman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fangman%20WL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWalton%2BL.%2BFangman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bonita J. Brewer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular and Cellular Biology Program, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bonita%2BJ.%2BBrewer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brewer%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABonita%2BJ.%2BBrewer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">M. K. Raghuraman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BK.%2BRaghuraman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raghuraman%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BK.%2BRaghuraman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:raghu@u.washington.edu" class="" data-icon-position="" data-hide-link-title="0">raghu@u.washington.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/11/1955" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/11/1955&amp;title=A%20DNA%20Sequence%20Element%20That%20Advances%20Replication%20Origin%20Activation%20Time%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/11/1955&amp;title=A%20DNA%20Sequence%20Element%20That%20Advances%20Replication%20Origin%20Activation%20Time%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/11/1955&amp;title=A%20DNA%20Sequence%20Element%20That%20Advances%20Replication%20Origin%20Activation%20Time%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/11/1955&amp;text=A%20DNA%20Sequence%20Element%20That%20Advances%20Replication%20Origin%20Activation%20Time%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/11/1955&amp;title=A%20DNA%20Sequence%20Element%20That%20Advances%20Replication%20Origin%20Activation%20Time%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/11/1955&amp;t=A%20DNA%20Sequence%20Element%20That%20Advances%20Replication%20Origin%20Activation%20Time%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/11/1955&amp;title=A%20DNA%20Sequence%20Element%20That%20Advances%20Replication%20Origin%20Activation%20Time%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/11/1955&amp;title=A%20DNA%20Sequence%20Element%20That%20Advances%20Replication%20Origin%20Activation%20Time%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459580/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459580" id="node459580--3" data-pisa="ggg;3/11/1955" data-pisa-master="ggg;g3.113.008250" data-apath="/ggg/3/11/1955.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A DNA Sequence Element That Advances Replication Origin Activation Time in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Thomas J. Pohl</span>, <span class="highwire-citation-author" data-delta="1">Katherine Kolor</span>, <span class="highwire-citation-author" data-delta="2">Walton L. Fangman</span>, <span class="highwire-citation-author" data-delta="3">Bonita J. Brewer</span> and <span class="highwire-citation-author" data-delta="4">M. K. Raghuraman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1955-1963;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008250 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459580--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Thomas J. Pohl </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular and Cellular Biology Program, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BJ.%2BPohl%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pohl%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BJ.%2BPohl%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Katherine Kolor </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Office of Public Health Genomics, Centers for Disease Control and Prevention, Atlanta, Georgia 30329</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katherine%2BKolor%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kolor%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatherine%2BKolor%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Walton L. Fangman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Walton%2BL.%2BFangman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fangman%20WL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AWalton%2BL.%2BFangman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bonita J. Brewer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Molecular and Cellular Biology Program, University of Washington, Seattle, Washington 98195</div><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bonita%2BJ.%2BBrewer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brewer%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABonita%2BJ.%2BBrewer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">M. K. Raghuraman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Department of Genome Sciences, University of Washington, Seattle, Washington 98195</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M.%2BK.%2BRaghuraman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Raghuraman%20MK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AM.%2BK.%2BRaghuraman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:raghu@u.washington.edu" class="" data-icon-position="" data-hide-link-title="0">raghu@u.washington.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1955&amp;rft.epage=1963&amp;rft.atitle=A%20DNA%20Sequence%20Element%20That%20Advances%20Replication%20Origin%20Activation%20Time%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=3&amp;rft.issue=11&amp;rft.date=2013-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Pohl%2C+Thomas+J.&amp;rft.au=Kolor%2C+Katherine&amp;rft.au=Fangman%2C+Walton+L.&amp;rft.au=Brewer%2C+Bonita+J.&amp;rft.au=Raghuraman%2C+M.+K."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459580/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459580/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459580/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459580/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459580/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459580/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459580/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459580/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459580/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459580/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459580/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459580/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/11/1955" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/11/1955.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/11/1955.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/11/1955.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/11/1955.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
