<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/11/1979" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/11/1979.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/11/1979.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/11/1979.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/11/1979/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/11/1979.atom" />
<meta name="HW.pisa" content="ggg;3/11/1979" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Mapping Condition-Dependent Regulation of Lipid Metabolism in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.113.006601" />
<meta name="DC.Date" content="2013-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Jewett et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Lipids play a central role in cellular function as constituents of membranes, as signaling molecules, and as storage materials. Although much is known about the role of lipids in regulating specific steps of metabolism, comprehensive studies integrating genome-wide expression data, metabolite levels, and lipid levels are currently lacking. Here, we map condition-dependent regulation controlling lipid metabolism in Saccharomyces cerevisiae by measuring 5636 mRNAs, 50 metabolites, 97 lipids, and 57 13C-reaction fluxes in yeast using a three-factor full-factorial design. Correlation analysis across eight environmental conditions revealed 2279 gene expression level-metabolite/lipid relationships that characterize the extent of transcriptional regulation in lipid metabolism relative to major metabolic hubs within the cell. To query this network, we developed integrative methods for correlation of multi-omics datasets that elucidate global regulatory signatures. Our data highlight many characterized regulators of lipid metabolism and reveal that sterols are regulated more at the transcriptional level than are amino acids. Beyond providing insights into the systems-level organization of lipid metabolism, we anticipate that our dataset and approach can join an emerging number of studies to be widely used for interrogating cellular systems through the combination of mathematical modeling and experimental biology." />
<meta name="DC.Contributor" content="Michael C. Jewett" />
<meta name="DC.Contributor" content="Christopher T. Workman" />
<meta name="DC.Contributor" content="Intawat Nookaew" />
<meta name="DC.Contributor" content="Francisco A. Pizarro" />
<meta name="DC.Contributor" content="Eduardo Agosin" />
<meta name="DC.Contributor" content="Lars I. Hellgren" />
<meta name="DC.Contributor" content="Jens Nielsen" />
<meta name="article:published_time" content="2013-11-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Mapping Condition-Dependent Regulation of Lipid Metabolism in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Lipids play a central role in cellular function as constituents of membranes, as signaling molecules, and as storage materials. Although much is known about the role of lipids in regulating specific steps of metabolism, comprehensive studies integrating genome-wide expression data, metabolite levels, and lipid levels are currently lacking. Here, we map condition-dependent regulation controlling lipid metabolism in &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; by measuring 5636 mRNAs, 50 metabolites, 97 lipids, and 57 &lt;sup&gt;13&lt;/sup&gt;C-reaction fluxes in yeast using a three-factor full-factorial design. Correlation analysis across eight environmental conditions revealed 2279 gene expression level-metabolite/lipid relationships that characterize the extent of transcriptional regulation in lipid metabolism relative to major metabolic hubs within the cell. To query this network, we developed integrative methods for correlation of multi-omics datasets that elucidate global regulatory signatures. Our data highlight many characterized regulators of lipid metabolism and reveal that sterols are regulated more at the transcriptional level than are amino acids. Beyond providing insights into the systems-level organization of lipid metabolism, we anticipate that our dataset and approach can join an emerging number of studies to be widely used for interrogating cellular systems through the combination of mathematical modeling and experimental biology.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/11/01" />
<meta name="citation_mjid" content="ggg;3/11/1979" />
<meta name="citation_id" content="3/11/1979" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/11/1979" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/11/1979.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/11/1979.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/11/1979.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.006601" />
<meta name="citation_pmid" content="24062529" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="17" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1979" />
<meta name="citation_lastpage" content="1995" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Michael C. Jewett" />
<meta name="citation_author_institution" content="Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark" />
<meta name="citation_author_institution" content="Department of Chemical and Biological Engineering and Chemistry of Life Processes Institute Northwestern University, Evanston, Illinois 60208" />
<meta name="citation_author" content="Christopher T. Workman" />
<meta name="citation_author_institution" content="Center for Biological Sequence Analysis, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark" />
<meta name="citation_author" content="Intawat Nookaew" />
<meta name="citation_author_institution" content="Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark" />
<meta name="citation_author_institution" content="Department of Chemical and Biological Engineering, Chalmers University of Technology, SE-412 96 Göteborg, Sweden" />
<meta name="citation_author_institution" content="Faculty of Engineering, King Mongkut’s University of Technology Thonburi, Bangkok 10140, Thailand" />
<meta name="citation_author" content="Francisco A. Pizarro" />
<meta name="citation_author_institution" content="Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark" />
<meta name="citation_author_institution" content="Department of Chemical and Bioprocess Engineering, School of Engineering, Pontificia Universidad Católica de Chile, Macul, Santiago 8320000, Chile" />
<meta name="citation_author" content="Eduardo Agosin" />
<meta name="citation_author_institution" content="Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark" />
<meta name="citation_author_institution" content="Department of Chemical and Bioprocess Engineering, School of Engineering, Pontificia Universidad Católica de Chile, Macul, Santiago 8320000, Chile" />
<meta name="citation_author" content="Lars I. Hellgren" />
<meta name="citation_author_institution" content="Center for Biological Sequence Analysis, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark" />
<meta name="citation_author" content="Jens Nielsen" />
<meta name="citation_author_institution" content="Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark" />
<meta name="citation_author_institution" content="Department of Chemical and Biological Engineering, Chalmers University of Technology, SE-412 96 Göteborg, Sweden" />
<meta name="citation_author_email" content="nielsenj@chalmers.se" />
<meta name="citation_reference" content="citation_journal_title=Molecular microbiology;citation_journal_abbrev=Mol Microbiol;citation_author=P. S. Aguilar;citation_author=D. de Mendoza;citation_title=Control of fatty acid desaturation: a mechanism conserved from bacteria to humans.;citation_pages=1507-1514;citation_volume=62;citation_year=2006;citation_issue=6;citation_pmid=17087771;citation_doi=10.1111/j.1365-2958.2006.05484.x" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Lipid Research;citation_journal_abbrev=J. Lipid Res.;citation_author=M. I. Aveldano;citation_author=L. A. Horrocks;citation_title=Quantitative release of fatty acids from lipids by a simple hydrolysis procedure;citation_pages=1101-1105;citation_volume=24;citation_year=1983;citation_issue=8;citation_pmid=6631237" />
<meta name="citation_reference" content="citation_journal_title=Pacific Symposium On Biocomputing;citation_author=G. Bebek;citation_author=M. Koyuturk;citation_author=M. R. Chance;citation_author=N. D. Price;citation_title=Integrative-omic for translational science—Session introduction.;citation_pages=1-3;citation_volume=16;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=A. Beyer;citation_author=C. Workman;citation_author=J. Hollunder;citation_author=D. Radke;citation_author=U. Moller;citation_title=Integrated assessment and prediction of transcription factor binding.;citation_pages=e70-e70;citation_volume=2;citation_year=2006;citation_issue=6;citation_pmid=16789814;citation_doi=10.1371/journal.pcbi.0020070" />
<meta name="citation_reference" content="citation_journal_title=Pubblicazioni del R Istituto Superiore di Scienze Economiche e Commerciali di Firenze;citation_author=C. E. Bonferroni;citation_title=Teoria statistica delle classi e calcolo delle probabilità;citation_pages=3-62;citation_volume=8;citation_year=1936" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=P. H. Bradley;citation_author=M. J. Brauer;citation_author=J. D. Rabinowitz;citation_author=O. G. Troyanskaya;citation_title=Coordinated concentration changes of transcripts and metabolites in Saccharomyces cerevisiae.;citation_pages=e1000270-e1000270;citation_volume=5;citation_year=2009;citation_issue=1;citation_pmid=19180179;citation_doi=10.1371/journal.pcbi.1000270" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=M. Brehme;citation_author=M. Vidal;citation_title=A global protein-lipid interactome map.;citation_pages=443-443;citation_volume=6;citation_year=2010;citation_pmid=21119635" />
<meta name="citation_reference" content="citation_journal_title=Science Immunology;citation_journal_abbrev=Sci. Immunol.;citation_author=S. Brix;citation_author=P. Lund;citation_author=T. M. Kjaer;citation_author=E. M. Straarup;citation_author=L. I. Hellgren;citation_title=CD4(+) T-cell activation is differentially modulated by bacteria-primed dendritic cells, but is generally down-regulated by n-3 polyunsaturated fatty acids.;citation_pages=338-350;citation_volume=129;citation_year=2010;citation_issue=3;citation_pmid=19909377;citation_doi=10.1111/j.1365-2567.2009.03163.x" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=G. M. Carman;citation_author=G. S. Han;citation_title=Regulation of phospholipid synthesis in Saccharomyces cerevisiae by zinc depletion.;citation_pages=322-330;citation_volume=1771;citation_year=2007;citation_issue=3;citation_pmid=16807089" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=G. M. Carman;citation_author=S. A. Henry;citation_title=Phosphatidic acid plays a central role in the transcriptional regulation of glycerophospholipid synthesis in Saccharomyces cerevisiae.;citation_volume=282;citation_year=2007;citation_pmid=17981800;citation_doi=10.1074/jbc.R700038200" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=G. M. Carman;citation_author=G. S. Han;citation_title=Regulation of phospholipid synthesis in the yeast Saccharomyces cerevisiae.;citation_pages=859-883;citation_volume=80;citation_year=2011;citation_pmid=21275641;citation_doi=10.1146/annurev-biochem-060409-092229" />
<meta name="citation_reference" content="citation_journal_title=Journal of biology (Online);citation_journal_abbrev=J Biol;citation_author=J. I. Castrillo;citation_author=L. A. Zeef;citation_author=D. C. Hoyle;citation_author=N. Zhang;citation_author=A. Hayes;citation_title=Growth control of the eukaryote cell: a systems biology study in yeast.;citation_pages=4-4;citation_volume=6;citation_year=2007;citation_issue=2;citation_pmid=17439666;citation_doi=10.1186/jbiol54" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=M. Chen;citation_author=L. C. Hancock;citation_author=J. M. Lopes;citation_title=Transcriptional regulation of yeast phospholipid biosynthetic genes.;citation_pages=310-321;citation_volume=1771;citation_year=2007;citation_issue=3;citation_pmid=16854618;citation_doi=10.1016/j.bbalip.2006.05.017" />
<meta name="citation_reference" content="citation_journal_title=FEBS Journal;citation_journal_abbrev=FEBS J;citation_author=B. Christensen;citation_author=A. K. Gombert;citation_author=J. Nielsen;citation_title=Analysis of flux estimates based on (13)C-labelling experiments.;citation_pages=2795-2800;citation_volume=269;citation_year=2002;citation_issue=11;citation_pmid=12047390" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=M. S. Cline;citation_author=M. Smoot;citation_author=E. Cerami;citation_author=A. Kuchinsky;citation_author=N. Landys;citation_title=Integration of biological networks and gene expression data using Cytoscape.;citation_pages=2366-2382;citation_volume=2;citation_year=2007;citation_issue=10;citation_pmid=17947979;citation_doi=10.1038/nprot.2007.324" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=L. A. Cowart;citation_author=L. M. Obeid;citation_title=Yeast sphingolipids: recent developments in understanding biosynthesis, regulation, and function.;citation_pages=421-431;citation_volume=1771;citation_year=2007;citation_issue=3;citation_pmid=16997623" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Cvijovic;citation_author=R. Olivares-Hernandez;citation_author=R. Agren;citation_author=N. Dahr;citation_author=W. Vongsangnak;citation_title=BioMet Toolbox: genome-wide analysis of metabolism.;citation_pages=W144W149-W149;citation_volume=38;citation_year=2010;citation_issue=Suppl. 2" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=T. Czabany;citation_author=K. Athenstaedt;citation_author=G. Daum;citation_title=Synthesis, storage and degradation of neutral lipids in yeast.;citation_pages=299-309;citation_volume=1771;citation_year=2007;citation_issue=3;citation_pmid=16916618;citation_doi=10.1016/j.bbalip.2006.07.001" />
<meta name="citation_reference" content="citation_journal_title=Analytical biochemistry;citation_journal_abbrev=Anal Biochem;citation_author=W. de Koning;citation_author=K. van Dam;citation_title=A method for the determination of changes of glycolytic metabolites in yeast on a subsecond time scale using extraction at neutral pH.;citation_pages=118-123;citation_volume=204;citation_year=1992;citation_issue=1;citation_pmid=1514678;citation_doi=10.1016/0003-2697(92)90149-2" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genetics;citation_journal_abbrev=Annu Rev Genet;citation_author=P. J. Espenshade;citation_author=A. L. Hughes;citation_title=Regulation of sterol synthesis in eukaryotes.;citation_pages=401-427;citation_volume=41;citation_year=2007;citation_pmid=17666007;citation_doi=10.1146/annurev.genet.41.110306.130315" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. Fazio;citation_author=M. C. Jewett;citation_author=P. Daran-Lapujade;citation_author=R. Mustacchi;citation_author=R. Usaite;citation_title=Transcription factor control of growth rate dependent genes in Saccharomyces cerevisiae: a three factor design.;citation_pages=341-341;citation_volume=9;citation_year=2008;citation_pmid=18638364;citation_doi=10.1186/1471-2164-9-341" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=A. M. Feist;citation_author=B. O. Palsson;citation_title=The growing scope of applications of genome-scale metabolic reconstructions using Escherichia coli.;citation_pages=659-667;citation_volume=26;citation_year=2008;citation_issue=6;citation_pmid=18536691;citation_doi=10.1038/nbt1401" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=S. M. Fendt;citation_author=J. M. Buescher;citation_author=F. Rudroff;citation_author=P. Picotti;citation_author=N. Zamboni;citation_title=Tradeoff between enzyme and metabolite efficiency maintains metabolic homeostasis upon perturbations in enzyme capacity.;citation_pages=356-356;citation_volume=6;citation_year=2010;citation_pmid=20393576;citation_doi=10.1038/msb.2010.11" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=J. Folch;citation_author=M. Lees;citation_author=G. H. Sloane Stanley;citation_title=A SIMPLE METHOD FOR THE ISOLATION AND PURIFICATION OF TOTAL LIPIDES FROM ANIMAL TISSUES;citation_pages=497-509;citation_volume=226;citation_year=1957;citation_issue=1;citation_pmid=13428781" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=O. Gallego;citation_author=M. J. Betts;citation_author=J. Gvozdenovic-Jeremic;citation_author=K. Maeda;citation_author=C. Matetzki;citation_title=A systematic screen for protein-lipid interactions in Saccharomyces cerevisiae.;citation_pages=430-430;citation_volume=6;citation_year=2010;citation_pmid=21119626;citation_doi=10.1038/msb.2010.87" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=M. L. Gaspar;citation_author=M. A. Aregullin;citation_author=S. A. Jesch;citation_author=L. R. Nunez;citation_author=M. Villa-Garcia;citation_title=The emergence of yeast lipidomics.;citation_pages=241-254;citation_volume=1771;citation_year=2007;citation_issue=3;citation_pmid=16920401;citation_doi=10.1016/j.bbalip.2006.06.011" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. A. Gianoulis;citation_author=J. Raes;citation_author=P. V. Patel;citation_author=R. Bjornson;citation_author=J. O. Korbel;citation_title=Quantifying environmental adaptation of metabolic pathways in metagenomics.;citation_volume=106;citation_year=2009;citation_pmid=19164758;citation_doi=10.1073/pnas.0808022106" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=A. K. Gombert;citation_author=M. Moreira dos Santos;citation_author=B. Christensen;citation_author=J. Nielsen;citation_title=Network Identification and Flux Quantification in the Central Metabolism of Saccharomyces cerevisiae under Different Conditions of Glucose Repression;citation_pages=1441-1451;citation_volume=183;citation_year=2001;citation_issue=4;citation_pmid=11157958;citation_doi=10.1128/JB.183.4.1441-1451.2001" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Guell;citation_author=V. van Noort;citation_author=E. Yus;citation_author=W. H. Chen;citation_author=J. Leigh-Bell;citation_title=Transcriptome Complexity in a Genome-Reduced Bacterium;citation_pages=1268-1271;citation_volume=326;citation_year=2009;citation_issue=5957;citation_pmid=19965477;citation_doi=10.1126/science.1176951" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Syst Biol;citation_author=S. Gupta;citation_author=M. R. Maurya;citation_author=A. H. Merrill;citation_author=C. K. Glass;citation_author=S. Subramaniam;citation_title=Integration of lipidomics and transcriptomics data towards a systems biology model of sphingolipid metabolism.;citation_pages=26-26;citation_volume=5;citation_year=2011;citation_pmid=21303545;citation_doi=10.1186/1752-0509-5-26" />
<meta name="citation_reference" content="citation_title=Extraction of lipids and derivative formation;citation_author=S. Hamilton;citation_author=R. J. Hamilton;citation_author=P. A. Sewell;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=Y. A. Hannun;citation_author=L. M. Obeid;citation_title=Principles of bioactive lipid signalling: lessons from sphingolipids.;citation_pages=139-150;citation_volume=9;citation_year=2008;citation_issue=2;citation_pmid=18216770;citation_doi=10.1038/nrm2329" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. T. Harbison;citation_author=D. B. Gordon;citation_author=T. I. Lee;citation_author=N. J. Rinaldi;citation_author=K. D. Macisaac;citation_title=Transcriptional regulatory code of a eukaryotic genome.;citation_pages=99-104;citation_volume=431;citation_year=2004;citation_issue=7004;citation_pmid=15343339;citation_doi=10.1038/nature02800" />
<meta name="citation_reference" content="citation_journal_title=Current Opinion in Biotechnology;citation_journal_abbrev=Current Opinion in Biotechnology;citation_author=M. Heinemann;citation_author=U. Sauer;citation_title=From good old biochemical analyses to high-throughput omics measurements and back.;citation_pages=1-2;citation_volume=22;citation_year=2011;citation_issue=1;citation_pmid=21190838;citation_doi=10.1016/j.copbio.2010.12.002" />
<meta name="citation_reference" content="citation_journal_title=Seminars in cell &amp; developmental biology;citation_journal_abbrev=Semin Cell Dev Biol;citation_author=A. L. Henneberry;citation_author=S. L. Sturley;citation_title=Sterol homeostasis in the budding yeast, Saccharomyces cerevisiae.;citation_pages=155-161;citation_volume=16;citation_year=2005;citation_issue=2;citation_pmid=15797826;citation_doi=10.1016/j.semcdb.2005.01.006" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. E. Hodges;citation_author=W. E. Payne;citation_author=J. I. Garrels;citation_title=The Yeast Protein Database (YPD): a curated proteome database for Saccharomyces cerevisiae.;citation_volume=26;citation_year=1998;citation_pmid=9399803;citation_doi=10.1093/nar/26.1.68" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=K. Hunter;citation_author=A. H. Rose;citation_title=Lipid composition of Saccharomyces cerevisiae as influenced by growth temperature.;citation_pages=639-653;citation_volume=260;citation_year=1972;citation_issue=4;citation_pmid=4554953" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=N. Ishii;citation_author=K. Nakahigashi;citation_author=T. Baba;citation_author=M. Robert;citation_author=T. Soga;citation_title=Multiple high-throughput analyses monitor the response of E. coli to perturbations.;citation_volume=316;citation_year=2007;citation_pmid=17379776;citation_doi=10.1126/science.1132067" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Kanehisa;citation_author=S. Goto;citation_author=Y. Sato;citation_author=M. Furumichi;citation_author=M. Tanabe;citation_title=KEGG for integration and interpretation of large-scale molecular data sets.;citation_volume=40;citation_year=2012;citation_pmid=22080510;citation_doi=10.1093/nar/gkr988" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Kuhner;citation_author=V. van Noort;citation_author=M. J. Betts;citation_author=A. Leo-Macias;citation_author=C. Batisse;citation_title=Proteome Organization in a Genome-Reduced Bacterium;citation_pages=1235-1240;citation_volume=326;citation_year=2009;citation_issue=5957;citation_pmid=19965468;citation_doi=10.1126/science.1176343" />
<meta name="citation_reference" content="citation_journal_title=Critical Reviews in Biochemistry and Molecular Biology;citation_journal_abbrev=CRIT REV BIOCHEM MOL BIOL;citation_author=N. D. Lees;citation_author=M. Bard;citation_author=D. R. Kirsch;citation_title=Biochemistry and molecular biology of sterol synthesis in Saccharomyces cerevisiae;citation_pages=33-47;citation_volume=34;citation_year=1999;citation_issue=1;citation_pmid=10090470" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=D. B. Martin;citation_author=P. R. Vagelos;citation_title=The Mechanism of Tricarboxylic Acid Cycle Regulation of Fatty Acid Synthesis;citation_pages=1787-1792;citation_volume=237;citation_year=1962;citation_issue=6;citation_pmid=14470343" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=F. R. Maxfield;citation_author=I. Tabas;citation_title=Role of cholesterol and lipid organization in disease.;citation_pages=612-621;citation_volume=438;citation_year=2005;citation_issue=7068;citation_pmid=16319881;citation_doi=10.1038/nature04399" />
<meta name="citation_reference" content="citation_journal_title=Analytical biochemistry;citation_journal_abbrev=Anal Biochem;citation_author=A. H. Merrill;citation_author=E. Wang;citation_author=R. E. Mullins;citation_author=W. C. Jamison;citation_author=S. Nimkar;citation_title=Quantitation of free sphingosine in liver by high-performance liquid chromatography.;citation_pages=373-381;citation_volume=171;citation_year=1988;citation_issue=2;citation_pmid=3407935;citation_doi=10.1016/0003-2697(88)90500-3" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. F. Moxley;citation_author=M. C. Jewett;citation_author=M. R. Antoniewicz;citation_author=S. G. Villas-Boas;citation_author=H. Alper;citation_title=Linking high-resolution metabolic flux phenotypes and transcriptional regulation in yeast modulated by the global regulator Gcn4p.;citation_volume=106;citation_year=2009;citation_pmid=19346491;citation_doi=10.1073/pnas.0811091106" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=J. Nielsen;citation_title=Systems biology of lipid metabolism: from yeast to human.;citation_pages=3905-3913;citation_volume=583;citation_year=2009;citation_issue=24;citation_pmid=19854183;citation_doi=10.1016/j.febslet.2009.10.054" />
<meta name="citation_reference" content="citation_journal_title=J. Chromatogr. A;citation_author=K. F. Nielsen;citation_author=J. O. Madsen;citation_title=Determination of ergosterol on mouldy building materials using isotope dilution and gas chromatography-tandem mass spectrometry.;citation_pages=227-234;citation_volume=898;citation_year=2000;citation_pmid=11117420;citation_doi=10.1016/S0021-9673(00)00873-6" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=I. Nookaew;citation_author=B. G. Gabrielsson;citation_author=A. Holmang;citation_author=A. S. Sandberg;citation_author=J. Nielsen;citation_title=Identifying molecular effects of diet through systems biology: influence of herring diet on sterol metabolism and protein turnover in mice.;citation_pages=e12361-e12361;citation_volume=5;citation_year=2010;citation_issue=8;citation_pmid=20808764;citation_doi=10.1371/journal.pone.0012361" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Syst Biol;citation_author=I. Nookaew;citation_author=M. C. Jewett;citation_author=A. Meechai;citation_author=C. Thammarongtham;citation_author=K. Laoteng;citation_title=The genome-scale metabolic model iIN800 of Saccharomyces cerevisiae and its validation: a scaffold to query lipid metabolism.;citation_pages=71-71;citation_volume=2;citation_year=2008;citation_pmid=18687109;citation_doi=10.1186/1752-0509-2-71" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=H. Ochman;citation_author=R. Raghavan;citation_title=Excavating the Functional Landscape of Bacterial Cells;citation_pages=1200-1201;citation_volume=326;citation_year=2009;citation_issue=5957;citation_pmid=19965458;citation_doi=10.1126/science.1183757" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Chem Biol;citation_author=B. Palsson;citation_author=K. Zengler;citation_title=The challenges of integrating multi-omic data sets.;citation_pages=787-789;citation_volume=6;citation_year=2010;citation_issue=11;citation_pmid=20976870;citation_doi=10.1038/nchembio.441" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=M. D. Piper;citation_author=P. Daran-Lapujade;citation_author=C. Bro;citation_author=B. Regenberg;citation_author=S. Knudsen;citation_title=Reproducibility of oligonucleotide microarray transcriptome analyses. An interlaboratory comparison using chemostat cultures of Saccharomyces cerevisiae.;citation_volume=277;citation_year=2002;citation_doi=10.1074/jbc.M204490200" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=F. J. Pizarro;citation_author=M. C. Jewett;citation_author=J. Nielsen;citation_author=E. Agosin;citation_title=Growth temperature exerts differential physiological and transcriptional responses in laboratory and wine strains of Saccharomyces cerevisiae.;citation_volume=74;citation_year=2008;citation_pmid=18723660;citation_doi=10.1128/AEM.00602-08" />
<meta name="citation_reference" content="citation_journal_title=Progress in lipid research;citation_journal_abbrev=Prog Lipid Res;citation_author=S. Rajakumari;citation_author=K. Grillitsch;citation_author=G. Daum;citation_title=Synthesis and turnover of non-polar lipids in yeast.;citation_pages=157-171;citation_volume=47;citation_year=2008;citation_issue=3;citation_pmid=18258205;citation_doi=10.1016/j.plipres.2008.01.001" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Regenberg;citation_author=T. Grotkjaer;citation_author=O. Winther;citation_author=A. Fausboll;citation_author=M. Akesson;citation_title=Growth-rate regulated genes have profound impact on interpretation of transcriptome profiling in Saccharomyces cerevisiae.;citation_pages=R107-R107;citation_volume=7;citation_year=2006;citation_issue=11;citation_pmid=17105650;citation_doi=10.1186/gb-2006-7-11-r107" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=J. Rohrmann;citation_author=T. Tohge;citation_author=R. Alba;citation_author=S. Osorio;citation_author=C. Caldana;citation_title=Combined transcription factor profiling, microarray analysis and metabolite profiling reveals the transcriptional control of metabolic shifts occurring during tomato fruit development.;citation_pages=999-1013;citation_volume=68;citation_year=2011;citation_issue=6;citation_pmid=21851430;citation_doi=10.1111/j.1365-313X.2011.04750.x" />
<meta name="citation_reference" content="citation_journal_title=Applied Microbiology and Biotechnology;citation_journal_abbrev=Applied Microbiology and Biotechnology;citation_author=R. Ruenwai;citation_author=S. Cheevadhanarak;citation_author=S. Rachdawong;citation_author=M. Tanticharoen;citation_author=K. Laoteng;citation_title=Oxygen-induced expression of Delta(6)-, Delta(9)- and Delta(12)-desaturase genes modulates fatty acid composition in Mucor rouxii.;citation_pages=327-334;citation_volume=86;citation_year=2010;citation_issue=1;citation_pmid=20187298;citation_doi=10.1007/s00253-009-2338-4" />
<meta name="citation_reference" content="citation_journal_abbrev=Biotechnol J;citation_author=R. Ruenwai;citation_author=A. Neiss;citation_author=K. Laoteng;citation_author=W. Vongsangnak;citation_author=A. B. Dalfard;citation_title=Heterologous production of polyunsaturated fatty acids in Saccharomyces cerevisiae causes a global transcriptional response resulting in reduced proteasomal activity and increased oxidative stress.;citation_pages=343-356;citation_volume=6;citation_year=2011;citation_issue=3;citation_pmid=21184438;citation_doi=10.1002/biot.201000316" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=U. Sauer;citation_author=M. Heinemann;citation_author=N. Zamboni;citation_title=GENETICS: Getting Closer to the Whole Picture;citation_pages=550-551;citation_volume=316;citation_year=2007;citation_issue=5824;citation_pmid=17463274;citation_doi=10.1126/science.1142502" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=T. A. Schulz;citation_author=W. A. Prinz;citation_title=Sterol transport in yeast and the oxysterol binding protein homologue (OSH) family.;citation_pages=769-780;citation_volume=1771;citation_year=2007;citation_issue=6;citation_pmid=17434796;citation_doi=10.1016/j.bbalip.2007.03.003" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=J. J. Smith;citation_author=S. A. Ramsey;citation_author=M. Marelli;citation_author=B. Marzolf;citation_author=D. Hwang;citation_title=Transcriptional responses to fatty acid are coordinated by combinatorial control;citation_pages=115-115;citation_volume=3;citation_year=2007;citation_issue=1;citation_pmid=17551510;citation_doi=10.1038/msb4100157" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. J. Steen;citation_author=Y. Kang;citation_author=G. Bokinsky;citation_author=Z. Hu;citation_author=A. Schirmer;citation_title=Microbial production of fatty-acid-derived fuels and chemicals from plant biomass.;citation_pages=559-562;citation_volume=463;citation_year=2010;citation_issue=7280;citation_pmid=20111002;citation_doi=10.1038/nature08721" />
<meta name="citation_reference" content="citation_journal_title=Analytical Chemistry (Washington, DC);citation_journal_abbrev=Analytical Chemistry (Washington, DC);citation_author=M. P. Styczynski;citation_author=J. F. Moxley;citation_author=L. V. Tong;citation_author=J. L. Walther;citation_author=K. L. Jensen;citation_title=Systematic identification of conserved metabolites in GC/MS data for metabolomics and biomarker discovery.;citation_pages=966-973;citation_volume=79;citation_year=2007;citation_issue=3;citation_pmid=17263323;citation_doi=10.1021/ac0614846" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=S. L. Tai;citation_author=V. M. Boer;citation_author=P. Daran-Lapujade;citation_author=M. C. Walsh;citation_author=J. H. de Winde;citation_title=Two-dimensional transcriptome analysis in chemostat cultures. Combinatorial effects of oxygen availability and macronutrient limitation in Saccharomyces cerevisiae.;citation_volume=280;citation_year=2005;citation_pmid=15496405;citation_doi=10.1074/jbc.M410573200" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=S. Tavares;citation_author=T. Grotkjaer;citation_author=T. Obsen;citation_author=R. P. Haslam;citation_author=J. A. Napier;citation_title=Metabolic engineering of Saccharomyces cerevisiae for production of Eicosapentaenoic Acid, using a novel {Delta}5-Desaturase from Paramecium tetraurelia.;citation_volume=77;citation_year=2011;citation_pmid=21193673;citation_doi=10.1128/AEM.01935-10" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=O. Tehlivets;citation_author=K. Scheuringer;citation_author=S. D. Kohlwein;citation_title=Fatty acid synthesis and elongation in yeast.;citation_pages=255-270;citation_volume=1771;citation_year=2007;citation_issue=3;citation_pmid=16950653;citation_doi=10.1016/j.bbalip.2006.07.004" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=B. Teske;citation_author=S. Taramino;citation_author=M. S. Bhuiyan;citation_author=N. S. Kumaraswami;citation_author=S. K. Randall;citation_title=Genetic analyses involving interactions between the ergosterol biosynthetic enzymes, lanosterol synthase (Erg7p) and 3-ketoreductase (Erg27p), in the yeast Saccharomyces cerevisiae.;citation_pages=359-366;citation_volume=1781;citation_year=2008;citation_issue=8;citation_pmid=18555807;citation_doi=10.1016/j.bbalip.2008.04.017" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=R. Usaite;citation_author=M. C. Jewett;citation_author=A. P. Oliveira;citation_author=J. R. Yates;citation_author=L. Olsson;citation_title=Reconstruction of the yeast Snf1 kinase regulatory network reveals its role as a global energy regulator;citation_pages=319-319;citation_volume=5;citation_year=2009;citation_issue=1;citation_pmid=19888214;citation_doi=10.1038/msb.2009.67" />
<meta name="citation_reference" content="citation_journal_title=Enzyme and Microbial Technology;citation_journal_abbrev=Enzyme and Microbial Technology;citation_author=J. P. van Dijken;citation_author=J. Bauer;citation_author=L. Brambilla;citation_author=P. Duboc;citation_author=J. M. Francois;citation_title=An interlaboratory comparison of physiological and genetic properties of four Saccharomyces cerevisiae strains.;citation_pages=706-714;citation_volume=26;citation_year=2000;citation_issue=9-10;citation_pmid=10862876;citation_doi=10.1016/S0141-0229(00)00162-9" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=C. Verduyn;citation_author=E. Postma;citation_author=W. A. Scheffers;citation_author=J. P. Van Dijken;citation_title=Effect of benzoic acid on metabolic fluxes in yeasts: a continuous-culture study on the regulation of respiration and alcoholic fermentation.;citation_pages=501-517;citation_volume=8;citation_year=1992;citation_issue=7;citation_pmid=1523884;citation_doi=10.1002/yea.320080703" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=A. Vik;citation_author=J. Rine;citation_title=Upc2p and Ecm22p, Dual Regulators of Sterol Biosynthesis in Saccharomyces cerevisiae;citation_pages=6395-6405;citation_volume=21;citation_year=2001;citation_issue=19;citation_pmid=11533229;citation_doi=10.1128/MCB.21.19.6395-6405.2001" />
<meta name="citation_reference" content="citation_journal_title=Yeast (Chichester, England);citation_journal_abbrev=Yeast;citation_author=S. G. Villas-Boas;citation_author=J. Hojer-Pedersen;citation_author=M. Akesson;citation_author=J. Smedsgaard;citation_author=J. Nielsen;citation_title=Global metabolite analysis of yeast: evaluation of sample preparation methods.;citation_pages=1155-1169;citation_volume=22;citation_year=2005;citation_issue=14;citation_pmid=16240456;citation_doi=10.1002/yea.1308" />
<meta name="citation_reference" content="citation_journal_title=Biochemical Journal;citation_journal_abbrev=Biochem. J.;citation_author=S. G. Villas-Boas;citation_author=J. F. Moxley;citation_author=M. Akesson;citation_author=G. Stephanopoulos;citation_author=J. Nielsen;citation_title=High-throughput metabolic state analysis: the missing link in integrated functional genomics of yeasts.;citation_pages=669-677;citation_volume=388;citation_year=2005;citation_issue=Pt 2;citation_pmid=15667247;citation_doi=10.1042/BJ20041162" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=L. J. Wilcox;citation_author=D. A. Balderes;citation_author=B. Wharton;citation_author=A. H. Tinkelenberg;citation_author=G. Rao;citation_title=Transcriptional profiling identifies two members of the ATP-binding cassette transporter superfamily required for sterol uptake in yeast.;citation_volume=277;citation_year=2002;citation_pmid=12077145;citation_doi=10.1074/jbc.M204707200" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. T. Workman;citation_author=H. C. Mak;citation_author=S. McCuine;citation_author=J. B. Tagne;citation_author=M. Agarwal;citation_title=A systems approach to mapping DNA damage response pathways.;citation_pages=1054-1059;citation_volume=312;citation_year=2006;citation_issue=5776;citation_pmid=16709784;citation_doi=10.1126/science.1122088" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Molecular cell biology;citation_journal_abbrev=Nat Rev Mol Cell Biol;citation_author=T. Yamada;citation_author=P. Bork;citation_title=Evolution of biomolecular networks: lessons from metabolic and protein interactions.;citation_pages=791-803;citation_volume=10;citation_year=2009;citation_issue=11;citation_pmid=19851337;citation_doi=10.1038/nrm2787" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. Yus;citation_author=T. Maier;citation_author=K. Michalodimitrakis;citation_author=V. van Noort;citation_author=T. Yamada;citation_title=Impact of Genome Reduction on Bacterial Metabolism and Its Regulation;citation_pages=1263-1268;citation_volume=326;citation_year=2009;citation_issue=5957;citation_pmid=19965476;citation_doi=10.1126/science.1177263" />
<meta name="citation_reference" content="citation_journal_title=Journal of Agricultural and Food Chemistry;citation_journal_abbrev=Journal of Agricultural and Food Chemistry;citation_author=M. Zangenberg;citation_author=H. B. Hansen;citation_author=J. R. Jorgensen;citation_author=L. I. Hellgren;citation_title=Cultivar and year-to-year variation of phytosterol content in rye (Secale cereale L.).;citation_pages=2593-2597;citation_volume=52;citation_year=2004;citation_issue=9;citation_pmid=15113163;citation_doi=10.1021/jf0351873" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=J. Zhang;citation_author=S. Vaga;citation_author=P. Chumnanpuen;citation_author=R. Kumar;citation_author=G. N. Vemuri;citation_title=Mapping the interaction of Snf1 with TORC1 in Saccharomyces cerevisiae;citation_pages=545-545;citation_volume=7;citation_year=2011;citation_issue=1;citation_pmid=22068328;citation_doi=10.1038/msb.2011.80" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Mapping Condition-Dependent Regulation of Lipid Metabolism in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="twitter:description" content="Lipids play a central role in cellular function as constituents of membranes, as signaling molecules, and as storage materials. Although much is known about the role of lipids in regulating specific steps of metabolism, comprehensive studies integrating genome-wide expression data, metabolite levels, and lipid levels are currently lacking. Here, we map condition-dependent regulation controlling lipid metabolism in Saccharomyces cerevisiae by measuring 5636 mRNAs, 50 metabolites, 97 lipids, and 57 13C-reaction fluxes in yeast using a three-factor full-factorial design. Correlation analysis across eight environmental conditions revealed 2279 gene expression level-metabolite/lipid relationships that characterize the extent of transcriptional regulation in lipid metabolism relative to major metabolic hubs within the cell. To query this network, we developed integrative methods for correlation of multi-omics datasets that elucidate global regulatory signatures. Our data highlight many characterized regulators of lipid metabolism and reveal that sterols are regulated more at the transcriptional level than are amino acids. Beyond providing insights into the systems-level organization of lipid metabolism, we anticipate that our dataset and approach can join an emerging number of studies to be widely used for interrogating cellular systems through the combination of mathematical modeling and experimental biology." />
<meta name="og-title" property="og:title" content="Mapping Condition-Dependent Regulation of Lipid Metabolism in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/11/1979" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Lipids play a central role in cellular function as constituents of membranes, as signaling molecules, and as storage materials. Although much is known about the role of lipids in regulating specific steps of metabolism, comprehensive studies integrating genome-wide expression data, metabolite levels, and lipid levels are currently lacking. Here, we map condition-dependent regulation controlling lipid metabolism in Saccharomyces cerevisiae by measuring 5636 mRNAs, 50 metabolites, 97 lipids, and 57 13C-reaction fluxes in yeast using a three-factor full-factorial design. Correlation analysis across eight environmental conditions revealed 2279 gene expression level-metabolite/lipid relationships that characterize the extent of transcriptional regulation in lipid metabolism relative to major metabolic hubs within the cell. To query this network, we developed integrative methods for correlation of multi-omics datasets that elucidate global regulatory signatures. Our data highlight many characterized regulators of lipid metabolism and reveal that sterols are regulated more at the transcriptional level than are amino acids. Beyond providing insights into the systems-level organization of lipid metabolism, we anticipate that our dataset and approach can join an emerging number of studies to be widely used for interrogating cellular systems through the combination of mathematical modeling and experimental biology." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
    <title>Mapping Condition-Dependent Regulation of Lipid Metabolism in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"FW4lIyygrHF_rCJIY2D3EGIP1f41BleNVPWBbWTdAYA","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459570","apath":"\/ggg\/3\/11\/1979.atom","pisa":"ggg;3\/11\/1979","ac":{"\/ggg\/3\/11\/1979.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/11\/1979.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/11\/1979"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/11\/1979"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/11\/1979.figures-only":true,"\/content\/3\/11\/1979":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WEUdHFYUTlPQl1kERO3bb+UfQDMmNbWfFU\\\\\\\/mHobtXqMs7shqxZcXqrPAkBjWk3oHAMKYPKseNCSYC1x\\\\\\\/tpZC4KNiLGnYjC6o8rBg5p8S9AObIHt0r1j8G+XwA+x+Dkzr8R7O42CBudkJJFgKyn9fciWMS0yBKid\\\\\\\/fopS7aiD2ClF\\\\\\\/fOHmNuLrRsBEtLJZsYSFJXEWOWdZKusYfczB5xsPTn9aYOdI3GSbCLA+B7\\\\\\\/0xKMEvtHxTVfvnxrmDhB1ho1oJE7WqGMXdxAyJDfQK4iAKv9ig\\\\\\\/svbcUnwtDjdvmlSAddh8zs8pZdzpOi5bX5bIeVm4xtuBdqgBSwiC9R5TdagcQJ9TW38XhR5bmgUHEnNvFrqWiyCb283uk1P6L7k+plUK52BxqTGZEHvZdB+66En4bO94aw9FZcDQfT8Td3nvd9ct6TJ3VYibMYXfvTxpYEaHI0ej7+QggTNboqqK\\\\\\\/T3ZZ1AJc1exQD54ZqcKrdLzW2qJ+bC1qDtkJ+IzmSmMwq24XEUmnuRNqjBV9fKPC7yQ\\\\\\\/PGO0ow2V2jS1yBStvLdmIYn0z+inSmFacCNnJbmwSEmuNdRgxJCgQR+9pYR5URS\\\\\\\/6uLuRUemALgb\\\\\\\/sP2rskRFZPNU8lwy9k93TZLJRpJ5YxGBLp9aP5NVZnkgSCZuQMtaAaYV5GWF415SRA91CtV\\\\\\\/2n744lqYfqJql34w2KRSlhWQAeiiamPwYwWbLi+yMaHJzaD6ogA06Du\\\\\\\/dJxjZkhEtY+vxEaGjkuPSF+jMoceef1VbihN4W6PpJwD20UTbgdHgFX2pEoq\\\\\\\/0lEfVEYU32evbD+YNxyEz4+1tzjJsQRrKkkv6nkBKezN\\\\\\\/sej9D43rWrWVL9k9ftXcnxNJW3U2oOrrgtjtExRzzEIMI4MqUjauLyDBqr25Ui55oAkBpH7vRQEdOdhun00eWx8fe+7TYo9DiCHUv6umKo+acPLI\\\\\\\/54f49LOpkoA2YgSUoIVcBI+yX9vRBl2XLs3GCMdwb4akYVFB\\\\\\\/pBVszMvA+lYAsT4UoByJEj8DJv6olbL9oFeIzZ5ODIOmejsgrJk\\\\\\\/I8bZC+1vOvwD72hdKyYHmQ4ovBKlABR0NeKU7rU9YYUTd+yji+CNe0B2qutS2ENNZVMar9ZhldSg3+Pv00eX8eR2lGMTmGQmcsyYVeZQFZeMA+oKZ\\\\\\\/2+3dCE4GlgezunH91qKJJNNRRGiZBmXrZjNKjyeSws1FbYcqdN\\\\\\\/DVEIM2lTdTumhtKlJrlVjEOpL0Nr8Q2YzOm1Q5TAE+NIXGtocIoFXZ\\\u0022,\\\u0022iv\\\u0022:\\\u0022ZsusSKr4yQiLINJBLC7xKw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ef5fe247327956ae6f0b69add93e1fb516799eded7fe606de0078bab49b139ba\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022K5nNu32gsaLC\\\\\\\/rNgWalDrtAuD9dhU0KUuiwk0w6yfnBY9ol4q6DkooncOTClHbUdE9ybkp0XlS1ZgiriDxt5+hkMq5FCedcq1NEWhF5K1bY3rnzUgGvvosLaVRFjSxgQqy3y+jtbj+HKj\\\\\\\/mdddZi2TiISZDf8Ks3hNEPC5aK0mBUmgkwDApSWn\\\\\\\/GUVDzPsETGZCuDuHgCpPG19oyeXgwhygFtgStitaGySARcQ6MiHwcsLma1FQgQOzVFqp1frRDpyR70OtMDDRLiSnOd4yTrtVKnCsI2wmVHRhbsr0qPIOXZSCR5VJ1a8LQapUCIKDmMrHnQp1D21u1zMYvDJUyLifGkXZ7Zvt488lsza4z8iGXJ6lF7eC+htFPH5tbtyYhNAA+68q4aYRNpt\\\\\\\/LJ9RLt50AtABICHJ7SDqgtVpqwzO80lPA\\\\\\\/BAKiCztfMz\\\\\\\/P0zCK4D5nkwhQ\\\\\\\/waAYQX2R0itM2AILVGY9rv1+joWzp7Vvt9+Adxfdc3JwehyuAHtHqLUhhJ0TnGJjid6vhl+U2\\\\\\\/5ha4kEDGn\\\\\\\/SlX8KruT88yEqrq5IyhYUtSoQkXkBqq6aIgjEF36J11Tt3ongLcMXDf3P0JDjlDIlbeyjq0ZZH8w3yujQ1xlfqev4RX8Ls7G0feAFliblsREG2QHaD02osp7KXf8VqdpBxiuDaU18VzzNQridiRYt8MO6620LIoUnTHWXrViBlwNEapIxYVXXsWypiQ3Kt5SEPkdvOtgXMDVxfUsZYuFG9Zb7o0m55ruCfbtnb\\\\\\\/9KRJxybZiLmDCzNmpM1Z6I4+3vN9\\\\\\\/DInqL+mCQcRKqRYJ3vSNmnfZK4iqel8Js4wtuNMkPPQn1PzLVl0gKYnAv2Nwn9IEzIM882BTVeAg87dn1mSeU0abZuYVBH7N9KYLngoeQaldUWks8Iw+x406KcibX5kGGviN4L\\\\\\\/5CVn+xeGfRTidFr1UOieCxP58DbwqL88E3oeZdT+Y6AXGrNrBqqRZ0HGHE7Ojjg44t3cwXwEeW5G68rSEj9lOsBbQMKNy8uN0g32JP0ARly\\\\\\\/XKSDQrxDpMFGC6\\\\\\\/Q8BMX291jdgWuGLhN7JxnwYRW+Iv7Wx7MqlwkTrjUM5yylGsizpP4IA4+ZwBRuYS9WNXmURLAanxC6JmZ6b6rXEZlpjQSrE\\\\\\\/wFScHRxrK2LH69k2rIIy8SERizfvV31e0Veinx\\\\\\\/OrR8\\\\\\\/Yd4X2c7C7LX\\\\\\\/ZiG+xaogQOFhr+0k2ZrFrbv+x4+cEKBcTsjWqG51WlEiZABS6jcL4YX4GPw5+Zpl+mW5wFZybyT7GTW8xlY3sRSuwR6hikZ8Y0zJUMkwtRanljcX4RJH4g8U\\\\\\\/LD4ZeO7gNE7kQdRR+jLAp5NSzN1BCpP+Gt9SzP4ythA7DYBLYgHrLTnGdX\\\\\\\/H6lq9NfVQ9KV9E489P832do262XZGcdXiPpw7pbHedM90eoaGAhADnYIuRc97FdUhypa\\\\\\\/mqhi3KH2+vVlKU4vaCaqmG3CRQlwIgY4ZP02Bl0Lf5bKZrTBtrHSgO87Ui6LKEZhNzipGnQ\\\\\\\/+JpOs1x0TDIn53dKyoT7dq2m2iXQ5J1YLbsD5Ixgex2aP31SgaDsGnBl7dzUxPpM\\\\\\\/u1UszEkf1\\\\\\\/1WReSkhmo297DvZ8ZmqXQMZDmDXr1tdhvb8MM8j6eb3Rqw1EHlxp+3QLBn7orVshFwo9afOGjkN2KSH+IGT5MAaiWjrOBc7\\\\\\\/AG70qcK4GSgvsMck\\\\\\\/zO\\\\\\\/P4DfaYPAQIp\\\\\\\/\\\\\\\/EfELDupmAczu\\\\\\\/5QEdqnqQ7t3ooVQ5KFEqMeTl2uw1jjkZ29IneYXF87s\\\\\\\/1iZLDjIu0UcI57oF97SdhHbgu1o8X8w+2xX4NGFOhI1X\\\\\\\/loJAzH4nopakgBbTuJ0LxUkq0Q8NQQSds\\\\\\\/kVczr\\\\\\\/e6LE\\\\\\\/NXuly3Pu5VxtjtJr+X4HOoPSECI\\\\\\\/tFGuemb2kI240gfHR0YydtL3Uxxk57qNcCjWC68UyKs0eAS3jTx7Rye8+3aY4RbPp9NTmdYTgo+VdFCrqm72oPy2iEWPbWIWEFozDiCFdyTbOuc\\\\\\\/SmiOVGMR02H7InHqT6sDt+7HUzgBeDOESAdFaefeKJNSMxQBwRDRXZGAUsOnmO8+AOY6W3jrX36wkFs+qEbBUJLYoJRu\\\\\\\/X2HL9KZ6S1orItwpK0++4ODMzL+Q5DOZg3mQum5P9siv2prWbjjVLOlAI0rHyyiYp9g0GjXSnOQdbK8ULL2+2aY6XRtncSMHUQIQHq6XPGppkIoIlc4wVzrIJocrr279bjg9nvyC8VJ5lggN4qsyvK\\\\\\\/n+ZPnYphjkP96pPHWGkoBpG+Prn+C89ekOhMGlpiBCbiFnB0iOhk87KCPHfz3K7YwG6uu\\\\\\\/7btqWM7Jd7cyVsleZJ6\\\\\\\/8VtGMeOLqTgMP7RRExOhsOXqiG+v8KjZmUlQ\\\\\\\/MEiA5iXpS4l9JbXAXs+NyYOxGlYShReryzyMC2p0raZ\\\\\\\/13gg7d4WrOiQe8hLDykH62MdbltybQN8JWCEZfsSyVrZcea9G0xWHLvpMBJXz06vaJuFI6tXYnYSeai\\\\\\\/r5NztZCJEIYUcF\\\\\\\/AuBSsPjCzL7KlmQRJtug43PmICyRI+MjNXtOrBuWomxln92MlzZEti20Rv8wnamzkp0H+2jGssZiI4FRvUetle7++7eW+I6Pd3he0CILEj4EJgLyKbzMZs4PO6Lb20w6bVuedIAzTncODAmLgsHidG2N11J1QUAKejkhLB2kBHY7zbSvhoN9IttwpSkyP53ngKuCLzigNSAsx4tXoMfdU5UKIr7GGOUN60PPEMMpLqUkhHwq2XL+HwSQQCoj9pjQFplk7j3ZMgZ0E65nh3XfaqHKt87+3FhPCbvUMZCHvPiojvN3E0An\\\\\\\/q68Lxizw+03of9LJNYgGlTEQibFHjQ+ap7IUYE8j3gw8Tua7q3agyYgQ\\\\\\\/dpk67A+9hf7AepC8Xmh5vHdH8eOAAiGOAg7ZVgmA8aI0snqKXYbQtCUylQJ4SMJfmFjaxFobwExAZrG+c7xaQtMLOEVnWq69Vs2MOaNtzYiHZFGkkEhqeW4hDstVInKecPauxskHQxYpM8JPFaKref23qVeV7UeDXDxUL0o2G6i+9pIOwl3is8HZHYXB+0Egd+xQCWauK1IBX70KhDh9NaEiNGIJBOhJoc+WY8bGCnY7bbnH3YSWCaOryur+gV+OuEgZj+8iDLa11lcj\\\\\\\/Hi\\\\\\\/13FtM7zOqBtqva8DDD9DXdAF5ICGCZkxi6xxH0R+J94A9bDInnfXbHN0CdQaRTVGfnBHO6jM1wYmQeUrGFiZYFDk+S7aUzwymr9IxRzSdk61i4Hsji8SzAw79WBcV3EWyaTduFX2c\\\\\\\/R17GdO6eGRxFOwr2X2Td6A86AowInawPTkCDeksMv5a4yB7\\\\\\\/28nOb7rPYCVCFngVbeBRS9rnW0PCbg322mDGr1VvI\\\\\\\/W44IUvsW75zkuXjNLWcPK9kNEnvHz6CT4SV4NlDW6QWISslfbdSnNMdCoK\\\\\\\/nkYJCTuV6pchZRR\\\\\\\/CB6HipRPf9LJwP5me3Vxg3lFYeQhYUSG+KDVVac7J1TMK2V2HNgJGuw0AbTv8f\\\\\\\/tvlJd53BGbbhTX8PB3Tmz80i9aRBj3yjZvlt9PrPbwaXwdlSj7hkKbSZumHDeffCJ8F618wHnyqG3gdMeaqy3e1apmGg0LwYxr6uGlF84c5MQIJq59ttlZBAMAp68t5V5imR2znPfDOJphTcbgafiLJNa9Eq7NwyfvuVaHMTKG7Hi3+5evcDHlqeYbQE6mi9ed15Nz4QXy441kB3gLl0+DKlVLCv1ZTFBOk6szkr2tvJIWTSY0xI+vYG6OS4EOkFkfPaUrqht5ymhpy7ITwLFbcNBhoZ9zlsyet126IQn2oSZTScpO1BU01YnpUovC8rGf00EV2HkizrrFXa3V0dvqUACB7Xxm+vVP1jL9KsH2NUHuNeOuEiVZaJijGXD8yMC57X3NtNiS0XwN+Myl1bb62DWiQ9YHkADTsKbjNblGvCVmimhyTQj\\\\\\\/wVAwxeMJZiBVL+OuQ70b0MaAX2WXP\\\\\\\/vrhkG2M6c7\\\\\\\/wrRQbF+XPRYF2W2+96BKqTCqf3FbxXmmTKljPqwuBfWWSVy5cABJPv6byvJ9G60G+luTRdS5gZx3aAw+NWunDKfZlVIOFh6Hxoh0R9tOVuEHm55Guoemv6HC4SRDH3WL93vqFP7gwCsYne+ZpFrQvk\\\\\\\/NKvGoTB6kq6md51NI5\\\\\\\/VhMdBLElMcylTNIqCLh0GKOMd6Xu9KXX8nXGS1qLbjgqOADpTEZVEHBTslkS1Y6zr70AUsy+CpoeWsihYPlrO\\\\\\\/HHv5h2ljA0RGjM4iREqT6TpAyE8mKacOz\\\\\\\/8lphq0h96ah5puDfhwVVRnFHa3D73BAeEE9VYbWm\\\\\\\/rgPtTJX+MiFJZW8d4+R44g+vPNudGDrMmWt6spd4E1b7N4++Pl6\\\\\\\/BSE6qV3r+2ntVLu7jllGShNGNvkIz3o2smUFUw4t+yOjz+iHNe2FQOgqvlnKFl3q3jqZWRf5JJExJzyZttwChIcWxJQ6HR2W4YdmMWUWGJ03c9XrPcyNmuH8MyhZ6gX8m3rtSBl9kkkfr0+7b78UGGGa\\\\\\\/NGpcRUkLXi\\\\\\\/aWIvplYG5uqa3f3523B15oIpiRethKzZC0ZFcEJ7iaZBaU+Q6x7gGte1bFI+WVQ7\\\\\\\/URJR4alv9GhyfuVC7x3XVz8HztK+w6h9K2AjGAbTLqnbR\\\\\\\/b0RNmKgdsEzCvEIdQ3MD6+dlCO0ESlBGFVth0LkTEIjV0gZqMRrjMyo5TIhswmNQvN99GApY0\\\\\\\/qw+WevG4SvhxIPEHq\\\\\\\/AAIDexd2+DWZltY0RnLLeU45gfYTFRRZ9ic0FykaYF7URVAi3LmKkERjeUkuaHJkRtRfPVuhJLk7bielEKiS9JF4QBeds\\\\\\\/2yWhkbo7x0Me1JL6qVn1IHLHFN9JToOIk1jdQ1xHrl4gaEUiE68NNkYtIdpqm7LxKEy+Y5lqSXg7Qw9ISzgzT5IhNnLQI1kt8\\\\\\\/e0qccLRlTU2OTekc+K30H4WIIhf4yfxQ2nfaGGz2tyPfUaOd6CR56I1emdiMt0EQEQdlswzNwBEjrd09pgLoVORgfFF4teL4KmzqPiBjPE8xnz0MHG0NBHMd4WaXkrveB5cD81YTDIpz99bEjfjpY8BiF3Bt9aEscCOxq06GIt2nqYnJvuWDIXJcaXgUQ55c0UZoGo4kMz0sahFVyUYP9x6caSsXPrNMJtWcIAN1rux+QjJC77PWaYYvPG2Dz\\\\\\\/DkW782FHx8pQWD+3p5YvNGO3\\\\\\\/Hsdv3SNOoU8C3SxwgvKxKyo4ikS8s\\\\\\\/\\\\\\\/qe7PkcwSXmAKHFLYCzKWsb336R+idapFDKDSPHYhVGlIvUx7a71+5YftxahLMdjCAmxCqaXWKiJpi7sCGHZMn\\\\\\\/s2CZYR\\\\\\\/tsi1QdOFylzrGByB01eGwwg\\\\\\\/4epXeO9Mq7XVEiCmJQrtCOGoji33vxGs5yaVSWlk4ZxDTsRpVXcQ5QUcXv7PITx8XDgcL+K42jSsUXhLqMECOOKje1lxpHgiaBEfmkPxfI6jBogVPUdloSf4vHkRwin0JtPnl\\\\\\\/BbSYOQ\\\\\\\/MVFaMYOPHLATyTnF1PE1w2qOIaLxYj7upU7ze43KH4OJqBS1XFqjuCgNy2ebkfXBr3fEjKRoCdMjA7Gt4gJqwdPX1iqDLb6rKdLa9mqJbItV+Ylptoe9qopnT1yMrJUKU9EGVV7whQbT4IxzQHSc7qlGgV5mEK\\\\\\\/A8cd0MnqbvoReB5e4fqWudFKdJkarSvR8CBbeKjg6obq7HHgET7CgsRgsUnrOYQ3h+uDRlu7UttR6a76s53qiYY60iS0vGzJ94WIWBYvKAUEOYGmVHMvcQarjk6VGit5HTwRmAX+LBe59A7v3S16F4dY1v4piIoK2NS0wBEXQnhyWBVP\\\\\\\/joC2tu4Ykk3iGh+OGQa2kgII68OS3DMqOhLr+RR\\\\\\\/ccoxX3u4Ru5\\\\\\\/elvgx6hbEekNYpaTkR3CoVWHYXNI8zyhjf31luIPv7u46WF0+uypSGBSNno8\\\\\\\/iNGLT32EWpuGMob0KFYpqT8nNNS3UijLpDwq4KSthZ3v\\\\\\\/h9cyPr2BMTtovUrWxEPl64g8jHEG4MVE8wSJ3T9oJlinI9CVWT2WvSjnSMSCJi+bHIp8wb7qLIwLwLGL5Z6v+xbthhnTfLCRjkhmZRnF1pMA5+ZbksAVrNLdF95FeS5IRJOchiu9p8suQsvHgTC5+Rn0scW63kaZUzBS9HjClAF\\\\\\\/7L9j1lP2+p+5HPd5sTZW6abJUdRt+PFnEoiIN4mHaYDed2OxArYsZotdFhhIDVG622uoBxXutaUFW6LYQ1Ol8VgzUq+9PV0Vqfe5Z+WPc1kYjS1QqZz8GpuCsBGvqWap7VPnRzrXVrXkcgpHqB+Y1VlOhOaWqUAmNfU0f6+ms4ztOQmIihLasiwV5bYZYUv\\\\\\\/RoF3eR4K8DVNg88FkRgIqjFjqacFhDmGRZQjxNVO6JRSKPgzNTxUnc9m55S6+u6x6U0Io86mnnXRhwMvyZ5C6QbHVtT0jZgHQ\\\\\\\/Wp1LgaWkqybt2GxF4LPZ7v2rC8HtaOXc1TZL3BZmshzBYjRCJNy2\\\\\\\/PIPfsKLQHQsjEq45rmTN6vhy+GFlOUA3U43QtT2Qa8sdvwmhaLymkEzmM++8w+hbAIMCtxnE5v+Eke1l6AItRg06mST0Wq\\\\\\\/lO13efh6fn3V4HwKk8brkGR3PLZM21nkx6z\\\\\\\/5amcbI54BsdQ34UrwW9jXTJmcT01Br53pxHg1E08Y1NN1oFzj6SvkdlVNoMFqhseO4D5j9MuV9c6KMg\\\\\\\/tb8\\\\\\\/JdZ0mu7mzi35Dy8b9q+zwu5mzN6\\\\\\\/V7QMpi5kTo2wrQm78n2zxA9ZbjUXUhDKh2aprQEp+X4TPTrWeC2jqsO+aFwOIcxBavD1pKOfjNt0gq4ObglO4leyXW4XEEc8VK8f1TcAqJbIrj6\\\\\\\/JRQND80S0GMk68A7RgxWK9WCjtgBIdCRnX4max2vpuljFeGySNbaHxpAQ\\\\\\\/dPcwqKM7OVELqmM8Byn6carlvex2aALRRPt5luPoJvaA7UMPIsVaMFCPH1p2IC3bqb9wz4C2rJaCM0QLTDNGMxK+kiqG+YqeZ1V9OW+VbLludvjS3NsVw2sqWZOOEcExw+B+B71ia34lSt5jAk64V49adZRukQOh9q+g+I0Coj\\\\\\\/u3NBLHYoqHI92HL6ZtJR3RYZpSlz4ClA8sQKMNDPtC888rD8b6pXooJO5sUFSymu7iOP+SzMJMYibxx5vWJWtHUQ5DgIhCBmJ+bHO637xZzAVThNh120xIosLCmpJ6CtFXXWKVvNIlNVz4WhMjyIXz7pqWWqOU4MuSElgFC1MAK39IlBXYbwkrBSCHtY5Mktbx0jxnS8lCDiU6oeBCHPAwPZuoTrKGZdG6zgEHJqiTYgAQ2NmvN4fIaiiTJLdfyMPOhhhr+p2+37kT8WDNZrXU5Kr2EA\\\\\\\/TtnYiSwDXbwC0KYo0dHI9WPJ\\\\\\\/lsAoY8V+OZfqptDQ8Kok6qCqjpF0ED3h8nGt1g2S0wLdxC2gF1OwXIvN6CeOHvDqMl\\\\\\\/GdOoX3YzE3YexTwIadE+zAcIQ+GANrzWvYJfzA7xB1TsAjD4znkGQxZpb3gBrI1HZw+B+YrLBWsYbWa\\\\\\\/xBfcSMh7\\\\\\\/sealJkx00d+M+2C+zc+hw0taXt+wUsJ3A0rTeJupaM3GNTUPDvmE\\\\\\\/tsWM9lnNMM4EEXDx2JO1wS11S85e+gnx7oZxiKOh1Az8aZLxxZjZwuuD0K3Jz0QtDVrBbksPv3O4P0RRX3xhWz4jhIy2rcjZS+jKgwZ+sLBS3XK4GUFvtafb4z7njCx+6VQNQwPg3cD\\\\\\\/I8gwqcu61OUgt0XVB5ZqHyb8G9l1aDUPc0IprjCLdsL9QWQG39ythBMfhiD0VG9lpoTZ7TqQ6fYQ\\\\\\\/mR19VTooc3LO3NDsuzT0VjW1VGvGioNcaGu\\\\\\\/b47CS7xNEvW7qqYrrKREj9KLk+Ki5OCwTARzJmbLIPaZoHBy3tPp0qKIc\\\\\\\/ylzaxWemgypAAoybT73Ye\\\\\\\/JADAyU6v5XkkXsh0fQLTR1riI8ig7s1EUveMdso302IqflFviw4et2sHl0xWD8Sgh0mmQQhTT3TZbTyqKe6ctLGN+jcEFiNlDQswCwbAExMA\\\\\\\/0veS8WOUc\\\\\\\/MbHh0Nh+\\\\\\\/PspgTbqP8jyKNDpxPQAM42ZXO4vs7nrw70hCtFZ5Sy0LMZtOkfyASQAccNpaZDdwB7CaBmxFkBCacqO7k3oftabjBuciZ0WVMOAnscMytgUJx\\\\\\\/9JIob73V1\\\\\\\/giJO7xE351zpesGtrQ1lrQ2RjBHXDsdBzCo9PT+3RjwBt9qqzjFYnD1MjnMZMstWGDZpscOK6SfOr8Nxbma16YPDUVz9YWmPBy8qrKtVmSKo7S7KaUuLYTpEzH0UUnPlytMiLav1K1iRRXt3ydJwlg0QHuszUiiBekEkkjRVssAhiAFELEjE00S4GZlnJ6aV8dNaMaNZBiak3R\\\\\\\/F3+HdeSj5InCIeLE5FflzjJturKyzWsmsCDUqUef1as8vdFg6PYE42JIhvpCJ+G0wx7BQ0cLUUZjzFwf3TSE+5Pug5DD5u7oaOkwgbY6qTXULR03UmHK07D\\\\\\\/1gCX6d+8hb4JyzRNKwN5yruijUuar5m5cs+2krLBbiLPXb0YfGGPQmazqRU1u3oP3YFbdStkzMEvwl8h5hBak9lpGTehky6qocC7a5DvlIrhi\\\\\\\/T7aGMG13YEJBLQ0uYtvAhaSfRxYjQIzwSzlkKn9\\\\\\\/t\\\\\\\/zWf2+gLXw0nsXo+QwwlWbQ0fGiLr6XgPJRcNQNtINGX9rZuXFbjx750xXSNdkFmqE8igr64LWXlA5LDCFxir1i7rXpT9BPP5ouPJnYhY8YBVI1cbXzrF8nC4OmO4vXpnDDXJYiF9\\\\\\\/4H078MOSYw9QTjaDzzKLI0UnUrKFJCjp+ioXpLhWxjbjQ28buf7EH2FCXo+tSgX+YrUXUPdPAwCE\\\\\\\/WxnebiAdqsomp52FM0Mj4ERaiqVoHDtSGRfV339XAyf6YU\\\\\\\/nGEf28QvwqpIQQDFRFhCV9ZNgl6dngl7hFeFkxlTzrkYcEu2AdrIACxHkuCTil8BuPWm\\\\\\\/MkvKJVvJkHAGAEAITNa4+sSk0ikmFnC52WZDQI+71D\\\\\\\/\\\\\\\/X1DY3B1Ak89D3+5TCgmO\\\\\\\/fdy7Gxl7e5Ly66wRWDeoXKKDeHeieihfZoGaFAjYinXi8RGRgFwtnmTnRbUbMDlTcQB5dcY6L4g7\\\\\\\/kV8TUW99nkTfk0nOvSoEIPgfJjTicZK2f52Ve4abck+Znj687ty8EL6VfYeRsms7CooKfr3VHK9\\\\\\\/FHPLdWS3xbnQ6bKVCcath3hXdB+JOLSZZIMgLT47kvA0IoY63k1czjx46jevUxr4wVeVCB635ODTEBaYBZe8+o1+HACpc\\\\\\\/j08wzqV+R94CPWsdiAwwM2Kkoh49xmmyFhtol0NE14MZ4aOBnZpmfu8r3V7+Ud0k2fQefyfNhQXfCi06HeQkyDr3hZDlgtybaDmg4e5IZEno+7Gz6ojXYRGf19\\\\\\\/ttnDIC2XdHefD2dpMNrzP7OvS3fBs\\\\\\\/diWSQ2ciXOp1bckksWltBrpMRA2FbFjG2z+zRoQD24H9\\\\\\\/Lc4Y4cfo9s8p66NqRTQgmClyNqIX38IlYOBcegW2I+TFWVGVzN6rvgkYVhvKYH4u9stPndIVa6QWHSK+YuhS7uNS9YhqRFxKTBHg\\\\\\\/+LqoJIqUHpf0JcPaeqFM8Jk1GbrYpF9OlRbtlNNPijhT2IsHq3H8Pr9cq0p5Jj73iHsxx7JH+Gy0M1XJj5xo2FZJbkB3KTVnMe6BWM7yO9c9Uf24V33dwhbg+hy4QJP\\\\\\\/I3GrImyyOOIkqwxbF\\\\\\\/v9w1cFOQNzvBLVmhWi14gpSvvpHYD56tYfteRaYsMg4caz3IF2zXov7mTx0zHThx\\\\\\\/U9jxafXQ0bl8TOGSTZewPwnh1xtAwsYWdNmRbKMfC0k0Nph1IF4GOLUQ\\\\\\\/E\\\\\\\/lggJWbxiyeFu9UuSuB7HHGujdhkQFRzcIadZEil2EDlTkFpk8o3drVnqQ4DinCVfjAAoM5s4ujF6SQ1DSKwK8dJBnT3WWcN9Gf2GhyVcFdFncFnEXMULbMalVZzDvBG6BxQExR9viAy4Myw8o6G5HQnGMI7b7qiKn7H4Hs1lIxUO7GCsEG3nckaZvlXnNAN6GXs6zMOZeR\\\\\\\/VTohsQtcffFEU58\\\\\\\/Yov8C\\\\\\\/OhNl3aV32JXFqDX+G1ibuZIWqbjaPuBm6HCm8EP9E0QqIH3XFmfSs\\\\\\\/k4Z36Lyst7bC\\\\\\\/acEXaOLOAc+NrvTwbk7GWohq8cTLRyTJNwJulpG5MfgsN3i356mEFxFoN3CJ7TfyiFKgOsAvQB1R81x2LJ3VRYXRKq8D6rWWym+hTRLUH6r1Dp61Dyp1ZXP4Yh0DthJGcBMLVNXXFovsHXGsMwEoEdGq5qjqz1\\\\\\\/qom08hIXi0tFIpSLfLvESdT+azf\\\\\\\/vuZ28mTKHATd70nnr8jFXE65TXs8O4CkV5v\\\\\\\/FlfPnmcvTXG810IY9hEqw1nJdYCEJhg+42NHBKEY14sFI89bhAqWC6Rpl1143+bp2YZktRTFvoGvUycS+gO+mFY6Gmm\\\\\\\/8a81rGXtdMMIEOpo5aLTf+b0n4ehqFbKs5CEokKda7B6yv2xYqwGIXPX55EId+NhaJP5Bi0L19lj1P2yt9JkBjideWqze6J34UPqnssa2sy2wRXi8+gBvbflEDokVXil8RSdzJQ\\\\\\\/f7SSkN94mzt2yul4NSm9QPutemVKTYaAIaPeRaP+6ZiLL\\\\\\\/IAgOk+g8nPW7\\\\\\\/6rxn6gRLVuT2t4dmjMANb\\\\\\\/izV18d16AU28sT1Btds8y2EFBdkRUedsTZANPvDCYD6y7LAIsGc9wzGtN6MQyRLpmG6YZ8OVf2gz1feSs9os+Mi084gxBWP+XP2VlhoILXUEaAwScg9KawXPKDNJqkEMSNARml0nYHcuKSwjQc0RTGOXFUfKMJ5s+IJAGpRi+4urqAjwBmbqBNamSnRRZAFW5nMGUgmRWKGlHcYN\\\\\\\/9npa71iDUsqZKQrK7o8zZgxJyKBC5VLIDqvQUqunRaQt6Ht0uujx0VIR2oH795gNpy3XimT\\\\\\\/Oa\\\\\\\/NZhQzP773oEkwLs2a0mb\\\\\\\/ETdUSu6ZeFQ8cAZZtRWVK+mIe3Q1iHI9SgH45W2Lgq2QK5u3b\\\\\\\/9QQzRFVB4zSZ90g3rkHpeU7mqbAfI9w+FGJHj1fiIVoF7vdaTBNGojiearPXdlSzofZ7nO+c\\\\\\\/Xi\\\\\\\/c6YlRi6j52wVuZAcPNZV1nUfos5KKDR3O5nTQGGHduHfFyABlZhaETPAHUN7e5sLPD8nMkaNZ1tYOn643vVSYEGZtT\\\\\\\/kyo36tCng4s\\\\\\\/SMvdVbvRRlO3CotmCSXIwsMQ2wkRPq+MgeIk3pOsDx5+fJk0B8GpV5SrA1tcm5v5fEvTj4Olb5WqfekkBGKQQxt\\\\\\\/g4rfmZ+l\\\\\\\/jR+ypRDSK5WSaZSX5HNuGdR2jTYBbFx8vl2zWbasWdoJijGlv+6rb8hKBsZB+EcXsjSl6QVUPRHewFM9Hna2GLDq4PiR+84SdJrqY5\\\\\\\/U4Eu7ZwGQsb2lvOBWyYGUAnnfi2ilns+H954sDvJ2lw9DFVgKn9LfJEjI0192mK+m5yCxxuVS9ByehskISNEvh4y6J1mG5KoIOP++qUdEdlhCbPYbMvheD5wtqYVVQ3Sq0aUGpdiGhiTZAIGkBxSXxIqIpwQaQW\\\\\\\/QDvMvWizs+EAzjHeuXw5YoizN5F2vR8U\\\\\\\/ybFv3Jx1xedJO52GToDhFeiYPzYO7ZTXmja4\\\\\\\/iWCi8GWCmgL09WVHi5o2gTTmn9+5GfDZ1F\\\\\\\/5ojKCPzsVrANL+HJd19VyJylW0WANWfR2w0GSHVbG6qBmPRjHijhGlDRnhKNOoRHplfVcmVqU42r9yeNPDHpjSQUD6rvuHuDL3PBVsUMCQkCVTFmcHCyQA3\\\\\\\/yGZ5ABiKsIaX8XpNmwx1rcurWYxDQgzzru3\\\\\\\/5KwgLTrowf3lyCAroi+buEKDwh3RiHoos3jIZD49fKy9CSF79dlYN5to0zHgH2ihxjQmfwdJCssP+wyh6Ppqg45c\\\\\\\/poV+0zneAjxe8yTFg3rnC9TnKTmUzUaZUPUnGfRKEIFpnVLNaAQjfy\\\\\\\/LnQTfP6bADz7GKlRyqKnGXLhnMFRPNEImJynk0sUVfop2QcBbbhmKseSA940UvYcyAB8z6zi7BqhwS3c6YCGCzhkA5Dg2L5ObbGk\\\\\\\/fvrnWOl540M+Z5qxRQ0x6bYkHpVmAdUh6u1\\\\\\\/stCs7KS4YN0baHPQowrYICJmp5+U\\\\\\\/JuOWcUmWIQud50Pl\\\\\\\/wwfbTvKU4\\\\\\\/IbaywRXsezdy0niTKWOBX94QM6Osm26Y0hcezLEwO3q7m6ypzZy+RtvyQ0OiKUatgZaMWXrbBSfNYqByx5ENTxkpoBLbATw1nrY\\\\\\\/fqq4eEVcuD0sOr7\\\\\\\/CqJ9CBgy5JE+jL8pedvhhY7ium8Zwvq+1zSWaAKGtBS7DmWg4BFPriUn9klUOvoKJjoa3oOCC81q8V3v4VTC9Oa1+QlGoI+Y1DuwaUcZPnKzd9Qgj6gKAnoIn\\\\\\\/Bt1Z8I78cIEFeBsopcRYnUX3Y5o0+O8fOKSHQhYAH\\\\\\\/ajQhrVk3BFWVGTUhN+yGS5X1d3W05sy1ZxNfhM2IvFGY4EF24FN9eyzA8+gwk61a6Yc589yhksfM5qBkDOH7qQ+O3ZzGdxYFQlCMjnmGzNAoNGP3tlOLAtOzZekk8i9WX6Y\\\\\\\/dEfPmpBGo3X540+otVcABXymX3eetLsWNFTVCIpnNKCEmnj7H795F5vdcAqiLRbhSHt0MpWHGe7wbUbQk8L0wPovu7ZzLOgCkQnsKxYbzgT+V1ssKMJdLeLfaGRumkPN3qiJmd9tCC1Fpz5OwvTzuWEcWSVMxnc7ilg\\\\\\\/QyyQhNzqO16oSr2m7SEAjTZjBhlma+fYfqlbDyeYx1yxJ7eXshvVV5FjkzJ+OSMFvKpNS9KgZiUW59Z2E1wRkqkbe6dX+JbKrPo1J3yFQodTf7q7PdzvjqAOtpN4sIBi2EQgEedgmzLa+C9ZwzDBHR+3CYj1ckXxhUy\\\\\\\/8TXfGywf3jikjlUh6UqbpgK0Tn8uYBpBz4rVSvZBLvCtWjwQ8MikQyiJKvbIHL2\\\\\\\/QkawuwvsgtazjRUokI2GHX179mG2SV9CMsTG8iz7MUMG2Poc74nDax3s6\\\\\\\/24qF5g7L4BlP3rces1vRLIlan7LNXWi+kze2RDcR5R5xkg5XijSOcRY6LZ2a2aiiBL7zqEtbf6v2G4kFyBOSedy4aGkBq5Eo8rABmaF8XGxd5O6OHXWB+6ZXZedJS\\\\\\\/PkykCMK+kTYT8zrvcV93qIJp\\\\\\\/uxlu3TIXNQZQqtwcOmIzjPVJwm9ppCaPHip\\\\\\\/M\\\\\\\/0kbAN8UUJr566Rk7Z4NEWdbeVooWZmqGxAHAxOvGzlloMgpPhw1WfeU5PbNTozsSJI\\\\\\\/OuSvMrf1en\\\\\\\/FpxdWMZAcYoVGaNGBydOtuLxxT9YdLNb0GWP\\\\\\\/tavICfvGaCrZtj7GcZmGWt3982D6RUpHaO0EOIM+9VjmBYFSEIMSEsH0q4nqJa5BKLKa38wuCaT90y0l7XSigLkgbN0M3p1OBx4oinYm7Xk+atorG13oADu4hnFkI7aBMYOJNb8NvuVZygpxP+tT8puiYFv7hAV4snjwi8ITjS4KFgszQk41B7HtBtcLe5QA24TaVMgDtX0Z1iJteQbys6ooOIw18aMkR77kVkSEtUqpV+OLAogAFhcqmke9dUiTR1TvzFxJPQU45VsyWa2HOESCBfjx6QDEf9K\\\\\\\/ge2LzwBx9vLxFq8qZvngoxjNQHwKboNtLi30H1kyE4AbRl9XJ27KyLZK1FcpK3HwgYaNSIP7LAu5XiJveWpUdQX\\\\\\\/Ec\\\\\\\/4tIssNiUJ6\\\\\\\/I0nOIRHfU33O04ZwUxhqTCP4puvvz4UVOMS8AVg+dwgi+BDxZEcefu3oV9YHa3Ecfum7nwGQVlgA9OuXKVtz+Uj6dqYC97CA4dM6SLuopwxPjSSjr8HaywCmXrl+ltDj2J4Uc5hqbbbT0uH9GBFjWQeT8dzLhZdtFOt6CmLsabCQWvcuhSZOxElWuE8OAYj6UPSfLxrBV8X5hckR\\\\\\\/4RjZg9SsHYc9CxGJk9bSn\\\\\\\/6+XZk2EZn0bxGelP6KRHHhA5rJcQSwo7CzFO2PapUlyzwtX1TlTiIt0uefRaL1+MSEuA6RRe5Ks5nIp3yiR6AzZlyDk\\\\\\\/r0N+9gwwrAfgEgJuDWM51KPp2Rr7ombUl6znEb+Sv+xY+ltRKYTM\\\\\\\/\\\\\\\/pthVB9Bod7u6+7fGUj2o\\\\\\\/SpOqwQrvtztUZPcw9vf\\\\\\\/LWfpo9GIghRGeQRfR\\\\\\\/b76yU8vSiJywYJW1\\\\\\\/TAHQSa1888E0d5AVETik4m7BnR3i5bBIl6g1pj2ViW9vnI6GVm401NxeZaHRyjTUsQ\\\\\\\/Qx9hpUN0WNh1kSxUDAklEHQTcFcp76vn0duGzezv6fx2dg3RmEpBQHCm92s7r2Qz3vcXbPXhN3eqvF\\\\\\\/scgZUPZRY+i8L2fczojQzyUtCLk5oBuRX33UaIklXO3UVrdZUg2HIxYTnjPVKG4QW7FT1Gt5gYG\\\\\\\/9F2fQn66XCzt0i44ICg+du3SXm3tucsJ0uNHEYOYQR5a1Wqx4nwFhW+8\\\\\\\/UbJI9r6OOwRjvpBE3MvgLEie4J+ddydWG9xhmHyjUzYK2IJndU92exqhcGIdcGvDXesStIYcBWzxGv29tBkTnX52UdfCg1attpAf5xNm2XV5ZoSVBlO2tWwJNUU0SwpmsbgLhdM5brRhYsOSidIV9cq6ayk29axsDh91uRza2W0HL\\\\\\\/mucf6YMaoOErh6xcSzCH+Atu007X9\\\\\\\/PwzgffKTVrlcbIRbujy4cnF4NecmVaoW01LXO5gAAslm62BlQy2kZjY524nRowger8F3o5\\\\\\\/K24BjLegwsoujBD+bAHLrZ09LNGNyiKOimdrYi5G+gYvO8vDrNMqRk3zWUxPGDFErjk0z0+1yzyXSfLG2WgAOYIrEujEhbEB\\\\\\\/AeN3jI2Wo+eWDTs01htjZU5nLwgQD1BCwT0k7eljso0vtUW8bMQq0PDcMqW2vRjekbpRE8vBo2X9UXlokIjCnW9K7tDVX400x\\\\\\\/TUuNyDiV4eQKdxjCiz1TLdkScrVk5kCb5X6Bl2KPkH5c8VWikxK9KPyIHZHg1dOZBXtOIAyJ76srxvxpHhPQEXI15M8u57Eb90UNo\\\\\\\/5cwvbVJkJrdcYyUGBPg78HKZCaBgxGOpnNR7Ws8eTPrILT00DRITcovWL3YewO6EUl4+DyOoCW+nJtXaqkR+ACZPqaQN68zZ3VqUbS0r07Xl8llDVhIMLRgQE+genAxzMKyiRf\\\\\\\/BSmBGNZMDe56pecIkeEpMbdpicjQ6eC7cP9N2hGEXYts62xKiKDQE9+ZK9VLEXIrkpbyL6pKQEdXMu0c0RPUUt6AYuaPkvCA1LM+gQCW\\\\\\\/5yEArIyf\\\\\\\/zXTn5G\\\\\\\/2kOz8IcX2B6O\\\\\\\/VO95mxCEVX5wlZE2a1CBqYv9bleWw6D46HzXK7isAkLodcpsfWECGJonFGLO790XOzCHfOz9ga8Dk9LPhKiIL5WxW1x5puQf3KDnnrMGrgvUCFTpOgvo2tltmG9AgSamq1lTndOiL2IuRYbXWqCk1y6zelViufmjT4mpx6dV8qOo4fe5qYa5qi3A6lVt0BYdBidUSsATINBSLuU8WgSIMNkru0nPQO7tN1oA63X6RitXGyY2Pu3NdQ4mMCm3xLo\\\\\\\/Ak\\\\\\\/07Yag4tLwYbVkz4rRr0AQrGAEODpwGDJ2BbgSyKHor1qcKdcpdX4jTl32tN5ti2kQvanlJhgwODUtvyxtExtPdMM84PUQOnZ+YPd22Kp36VxNoiH9\\\\\\\/VqOXdV0VeGro44\\\\\\\/VVwBjM3JMxKiESrCtA728mFIlcGhwUS4yYZ4RdU\\\\\\\/i0HSpMVSFR3Q9ZzxX7D73qBaafV\\\\\\\/0k4qIZL8IFZcqos7Vovbw4xmgQ\\\\\\\/54eJmngpmE+xulvrLxi2qlpYpBgjarBmARRKuUo24M0JNrbBedxezQ7qKAmteTngPS+4myxadLMSfBzzbmIFNwymVQHZskW8JfjPggsnZ\\\\\\\/KGvHmCtnbgp519OS9+9u+6Th+gkY9enWg+cmZkBIajqodAKChk5MZWr+t0EJ0p4OVwZzII3twXoS5J2sxL2jS1HqGhR0w\\\\\\\/TNNlwB5fZ8T4VQtgtBY0CPUMlaxdPUd5kf1cX2KnbSIiw0U7maDelLSXxkiNhaHZGwJdJU4\\\\\\\/rC+f0pPZ9EPvqpkmXH+H9qketbyOYJQnuz\\\\\\\/sA199PF15BU0WKUOMwfJNSHbrA4zxLVq\\\\\\\/+EP8\\\\\\\/+ivebAEtt9GZlIv6NjcjV\\\\\\\/kgqJ\\\\\\\/K2pw4HBe25uM69+P3XWJKnq32j5myNQrWehssGUb5choZ6mI+TNPjRXU53Cyak94q2jGxUTPdUXrAOr4v8fIm0KphGJVUVXczaeyDUCV81VECYaVnNt8KQ9bDYvdZELv3jr7VolaDRorm+zqv2QqmfrbGGnNAh0e5R9uhaiEYu1284R6RRLrnfToVFxn7ILXGoh65ToD66y3QNTQWzkPgRLN8VSfvfViZkFyCbh41ZW61kEC4iwMDCpMzb5IrishF20y1kKqQt3MGU3QmhwAivOA782KFChaKlYKjtKjhZjARwZh8zfnn4\\\\\\\/iHbKTfM\\\\\\\/eUifhfFdJR5esYcu06uBAHXaQxd2MO\\\\\\\/jndv2qCUbT+fNKb55ExxYjnkvDjZbQGBhzFwW1WQy\\\\\\\/BL\\\\\\\/MOvGVT96INAS9IEFXAeR+e7ZKjuHtPr+MMdFjV0VLkGgMqQ7taN\\\\\\\/wM8ZBJLfqmHXnfbhZ1+hKFNrfayJv43hgtN1uj\\\\\\\/VLRry6oXq3viTb9J9nCVmRcqIOAIctFcz4TgDBNqESWOlNcMYo2OoVleMju0yPBBUEF7Rkex7qV\\\\\\\/54IgrpTfbgnFAM3YmPDK8bnZ0OiDYUU5GP4v1KiPwAbN7f7Tg6FDJ6t9iqQGgnljpraMGXRrhR4HaDSwUd6yvlpDLbv3k3Z7h3at7n758BHjA5J\\\\\\\/8wWu4j\\\\\\\/QolKFr9q4t1CRKotu44uBvPz6CpiymkFqeaRpX1uIviWNZYdnUS6l5DO93j6O9iuHDNkOy14a0VvFiJVHvvgmWl1rWr8WhFK4s6MMhBlyMbitVnIcSKMRNg8ThG1vl\\\\\\\/iBSEA77HSXF4otaz0XBWSpqkfd1FMpq4P+ROgjKYElP1BxW1PWGKJkpd999hYEy1LTry8T\\\\\\\/XPZKKH6S5tst6h1sPj55pLeO0bzFi5GHP5OK1T2CYricesybMQdfMA0k6YJIdAW7Pc25HTsXWbP05SafKImjE5jtNwNxuOpGckGvG1\\\\\\\/9iAXjfKjBBaO0LBdxMMmFcI\\\\\\\/zG7KsMK9F3F2r1oNDeUHOtAdi8e2dyHHSs8hvsInMBEt6Gsm5SzmDfo57A\\\\\\\/LC21QUPZpvMMxB7jo1FaI6acf5xa5dG8oCPArQFNBxNe5\\\\\\\/kqRJeBV5rfIoEuG0ukxsHiqeuoV2Dz4CNEScp7rCfxC3rufMMDG8wmJHsw7Eg9Tc+P91jZ1vMhrtf4uFRFvd0qr3w89zGIww6EeWiUa0KHZzFogorrFa5TfYvE+nxWBOPVNcVN1QZlbz7LM34QOGwk2n9UpYHv8FgEgbyzv\\\\\\\/CKtJrR1AsVyR2x7n0PP\\\\\\\/TJyrzuTlaUtzojme3khYLDJfAsN9sZ1eotUql5g7fxaHaDIWn9Fvvu0TIBYzyS2R6WiwSttNRZ3kGyGr5qYV3D9VRDlHczA9BEnvBiwuH8myHdkJLi18remWwrTcoefX4tYMMmacwVGCUk0jrg8j1grYKQv3sAOEKvY6IyHRyqlOmFqgD7YbmNt8lgCjb8Kww2yeTXRxKbZ4LZsmywzsvBOUnIzEtdDwGSEkziJcg+jZiGWfInJN8splwrenRQa9ypWrpj2wQ7fjM8hOHpVcqv0+Lx3uA1plGbP\\\\\\\/cFU4Hxedb7BWR5EiVpyqx6khTWI3IaGOncvXa0Hi7YNagwNqZzdaEo7fKFZCMMKQgElMkMhlY14+OXOnqHy5+O6wtMqG5HDW3Bqd077MlyPngoiIuIQB+O0FUAOyPl8iP6OeixNkKHaq+D\\\\\\\/tCnco4ihK+EsgNcJYtdRTJyzRkaTQ5zg9m+VWPbbJq1sdMOQpCPbQ131Wa\\\\\\\/R8ClBlHT5IdebdBYqP5Pn0RGMJUxyr8akRlO0U3V\\\\\\\/LXyFg6qqVGZoI2641M6whE0OXRCcqGzw1awaiWzdoC2S6ldGGI++O0kQRl+scpAjp3sdhsit+WYbVHayaCUhashYu8X1ZJIa+p6JPDbJdJuSr6tV8fMIu54hEUm9CKvqErj+KBem7CXXQJ5qoAg9T7gtX2X0UYFCCCCp8tD8HShg3BVwP8xUKjVcnZ\\\\\\\/iTSA8g94CCoCJrwd76Y4wpJKj4iw6z3pcOboCtHGNk+EVDYd0HZKcGp7UbAJnGdiscjzOpXnsXuLvV1ERsxAlUnCW7V1F1GX2RvMUYfPty2ybmLoTEiNYzskqK2CB6oS+RrdweCgVbGTRUItBpAkHecEIpXvZrhAUBbv6n5LS63Jn+gkLKtulES9UbGdSzz\\\\\\\/dFgJHVoR23cdtpv7Efn4fNiSoE7Q2jgB4At+YDVoQpVgvFwmQALMnErn\\\\\\\/KhRhW8FApUVT0q1k2FFNX5BCkby5prctsBO\\\\\\\/hxOSuSSIehHStEgGzHCX94Hao0k6jTcc7OoKOWv+xPCwOopPmdmrNa5nboFg6LgeWG9YA3dAlS\\\\\\\/1fPlfPk15jJ0MQc\\\\\\\/oBvi1D5ZDwKiRg25XlIX099TxxkSjloQgBhxYj7LX+3Zs7AyZcFRWHk8QVptWiWHzv9qRN4oWBBOB3DCny2YBsVOzLdDZAaOZL9ATZ9Le8vGjAP8ftIrlZOFRJH4ELNA\\\\\\\/Eu\\\\\\\/sLRuWRuZZg443KfAz7plxT46jnxta9onS5imRa7B++dpaXl+sZtBvfB4zkMWBi7cXZ7nSj+91d4vY\\\\\\\/6qvGYETH44LrGoCuNo5jgZb5EvEyobpVwimFot5etA3AKZRrk6R+qVLctSkOnI4kyZQMrvZM7tq7pXgrq3zqdQo3dhfjDFfI\\\\\\\/3dLPKx0yhghmYoYUhAXrJ06igz4q8dVVl5fJV2n3ncImGYDDPk13KwDq5FvgH6RFzPr3RmHYcEUPu2HZhiebP5O9+2gUZ6CMACr5+uuRzCkDJ6\\\\\\\/65wpvvVvLV\\\\\\\/EZ6rDSs\\\\\\\/H7MbVrJ1VQNzzGk6WDR1umQyPt2J2c3Ui5WDdcbsTxVRIxz4YUSOUaydcRNwIwBGqQczAr\\\\\\\/r7AfNezHRW2KATCPA0bktO+alxh5SPxf6ItOo687or0cNl+ao6oMWQXx+tdsqC6y\\\\\\\/mQI7Wad4LSlKFEpfSXgdBWO+QCSVQXxSmlkwPkMG5LUZmCNm8dOiMQUeqZfxBdWGaG2DzjrsjIswqRY\\\\\\\/MmBnVztb5OmERu2DHfH57sx9SEEkc5JcQjdUZUYxG79Nj0u9BjDsnrf3Tt7OsdToU300iVTGLFRt+MEiCAEsjjtoZjuep7jyguTE8K0zaNJVjmbvJ0zCDDveNPantafQ3V07NwDSgbw28bsNKFdJM14+H\\\\\\\/42CDfMLPMWBZjEy4sEmzPvO0gWaoHd18SopMleStOGJFANCvCFNGl8GmHX7ngQVpA88aC6Oo2ABjNM8ht+MeHUNatxs9vAgYkQ2QhQIc2iB7SmUzNYwGHAit0VzdYrae9wwWlNy4bKofC3\\\\\\\/UznGcV57Z\\\\\\\/HOTJ3rIqZDV7C1THCMvblpTL4kemQq\\\\\\\/a2aLYzNOdHq8TQIWtObMIDWISvud+TzQ3Dm7qts41So9KRExFzx84\\\\\\\/GIIab2TiFXFpzufxwHqLHUE6Z9x8SsjlNhq8EHOfoNDwSCIpjzjhpv\\\\\\\/v0faG71XS9ut6UDdvh7Y7KFxYA\\\\\\\/DebsYsV7tRjcnk8i8YjnVUfVZOLrtKwUAmOfWA50tLVYBrAj\\\\\\\/WQPKK1cUEWps5WXk5tm8PfI\\\\\\\/FliHlLjg\\\\\\\/OfPPtCPKacxUjJc4NB3Fbu6\\\\\\\/JUkzJ07k7IvRH+xZ7mpKnFmTvO80xvhhAw5b3JEzAkO7P7LTYtubXSLNEtVwCJrSNxgyBipwMyUZVhXXK+JtSr+UeJmLiG6Du3dCr3ffrG1ocnaW8dgrp+2ZbyZaRDGS6WVYG2Eg8YTIZTBvkMzNENOnsNmn3gJOO0eARhw23PW1Lyt1UNazpVjqoJkRSXfBwCzB0OgdIzRq1R94dNe5uctNqcZLs++2PkN6BSBzZpTfIu4NMC+IJbhFG2iaws9NkcM+0fsUttLumNYUrB4wGfh80hlr0e9aoFYoXvtzjC\\\\\\\/AlX+qFvP8KJzHxM\\\\\\\/Z5FzqFEbTfZQryCYTOqEK0fvlZZsnpeesJT6KYseqhRVeaYnF\\\\\\\/P9oPww3IxeXM\\\\\\\/OPaH3u0K+pQh7CFs5vmSL6ZKVCWr1mbDAdLdRZ\\\\\\\/GnjWH2piEQnOD8NJ28\\\\\\\/TFEI0XVhgwZwC5JxqF7mGBib13xEEKia7p+uZGdmOqcRISahsoFp3DyrYd1KBkBs8ZvB60vWclIJJRkCj34P5xuG6oIUaoriLN1RMgwKT8iPcs4uIkp9zslhnXaKbkWwQpoCaEn+wiRP6NTVKHZ1Rz1zVOP3eD0S5grBcCXLYKtbnXi\\\\\\\/xaFi4eMZwuPRNm7z3eBT8w8WffkGDbK\\\\\\\/03zmi6Il\\\\\\\/QEr6zod8QxayJYpWWVB6q7+xetDVq57DBfJ1kLpKb9YFcSPFKHemS3f14+9T\\\\\\\/AL8m8YOvhid+CYlDarDG4HZrfBYrtRpXcroiFNrd+oCDTEJPwj0nBc0O2xISD+1bm4aIKUQvPFPoVSeX+VQyMdfqYdKKoklMqnkygpap7mllgLU2vpY6DeaSDdCybqpXKdsFiLZnTIkyj9CJznpYZXzXvnQq4r3iTEw\\\\\\\/eMuCWAhP+Qy6+pTY1x\\\\\\\/Q3i\\\\\\\/nasw7xj4dSRX7X5F\\\\\\\/8moSqF3++88seFRZOtZF47L5u0XD2ORoFQm3Ldi7aMPdUfd7PkGRNPGlfpwWpxMNHJUq3pdUOrPJFfCZvsDSeyUVme7HQeRinnibTrKZS9bYOpwVheIW2WUIpHwzijAcTh0713eDAlHZwExEUeKfvj+xm1YAaBjgdlbYZKgg\\\\\\\/1pWjthBG42vPOGWfiF1U1Em7aPTrvmlYb4GagymRWVCgiGHVUvMdjmQmvzJr5L9SmhT76lNfKMg06lv9IgD8dMKT573HVYR1\\\\\\\/B3+NOAcB9OEvqsJQJu9TT++M6UzeiolXc0G9wCPmVLOIdlLtBxVLxzuBtZ0ZeeTkDXDXDNgsvWT8RMUJNhy\\\\\\\/SBLduEDRFGYa8TeZPhzlOr8KYOgwToZLu1JdFSJnGemtRN6r3YVNgWd2Hb1TbrujNQnxNItnqd9Aaz12qT2j7RDD5WKsdp7pWPzBLdNUbrIKkcp\\\\\\\/hnT+FAU7jWTpWuF0YSPG9pYWwaxiTk9ZOIeWOgHxH4ADVA4abJKE8sNk2SOVYzpEXooBfBOZkHxGWL1\\\\\\\/jYa6Z7cT+JmfBffMCZIgWmuTwS3Jed7xCgoDboiKlHzEG4Zza\\\\\\\/Xmu6qruVuQMKQoqUNOPUKJktS6XS+1k2dfiy64uB04J\\\\\\\/up8vrOX2fWcEZkjmqdYovfVIY8DSjfYEHTEFfDLEkCOzJnPNNXYc7iMmb5Tu1+3Zyt3XlkFEo7PcOBq7S\\\\\\\/5+GRsvEiG42pbx5jtCr3dAx\\\\\\\/oO8AR28hAGwy0jCH9Yw61a9\\\\\\\/5Sdcfj0ZEC4SgAfsUQjQBxPe\\\\\\\/Mn52Lqd97jhmtBUixD8uuOqTs4RPcSqRlKW5Ju9DKNAdL+rysXzCXAnQ9lQ4zMlra1oL6j+Yn7Kof4RzwQR6FaswcbchJM7VXT9Yx4z6EDZAm194AGnI0rjcBpayouSLB3WD86\\\\\\\/tHYHum6cUXhGOVWShU1o0ZhwHcsgb+WoFjFUfpbmyGMdyX505yVjzxLOaF3s6OZWLw9TZEQKmduFPAu7CB4wQOrPAgLVADWMo\\\\\\\/6dGEovI0xtOBnmyWBvqiKGVaPB\\\\\\\/5KcMA0dusvg9CiHyOWwYb+ZmsNmC9+Odsz68vXRAfU7aTJogwqo3SVaodKSJpshKJqI1oyhI192kaEds1XYz5KiTeeIRvRx0WWgf9Z3FT64L9vxW7qcKCNgIvYQ7XfF6X3uz7ThW\\\\\\\/d+hVsE951WqAI4gs+11fJKyzqtvMe7ddkm4rC14V7bDQIUOxeD5vb3ov8FgYk62V8cPJ6hy65MH2f++xwf1wDan6glm\\\\\\\/I1wRImjGsh6R0U0OvuacStFcqOxVYEWX2GNJiTYnnJ\\\\\\\/Fr7bEHQqJPGnbUG\\\\\\\/KjmRJEvpaxfYBfsqjRhx\\\\\\\/ybEq+GUy8IkGZbRsrJillNGFsrXS7xOqTcABnEoxBN1ZRvrVxfaN9p5wHAEhnlxLj3SmaJxmlkzde1gk3DsdpfDYy5U65DEBONhts6pllQ6PxD\\\\\\\/Odyz3K6tlSSPCLhJGZ0rYmMt4uFTk3yJLJBFYfoTz4EBKhh35zsaCDr3ZONqEeh2t7vrCREuGxtZx5RXmrb6F5aDO3URJs4mJ+uH+Df77xGMwwVu20SJvkwPuRCWI4I6v2BADnjYhAfHgCAXGMuCENybMuhrPYAyjKwlp2cOybeQAbfq\\\\\\\/NelBoZX5DMsI2U7Lu+CJ+TlVqc9GzE0bNtzmPrn6jhKToCrs++L2hfg\\\\\\\/\\\\\\\/G8cl\\\\\\\/EyNNOZ0RRGP1D8SWY5VbsWuFN2HESoQOoKYALQW0fFKiO0Q1dQJB0W2V8oTKM\\\\\\\/xl1YCqTBei\\\\\\\/Hbs6KuDoWBaBRkrCqWn7ZdNWEm2QAnAtpR9Yxqo9YVFPzz1WKZzIZi2+J7M\\\\\\\/egfjW4ZvbXXt9JULcuCzXfMitzY3qAe\\\\\\\/5jOZhgY1+dAik+bUZP9WfdlyJEyTJ4H7cleOOLc65s\\\\\\\/ip4O451pJ+BhlhxlmSE4t5Hp1Or3\\\\\\\/GNDdoOzytJgQdwBpA5vARmFJ9beY4g17EF2l4MK5VuvxdEDQj\\\\\\\/ibsUGtBm1uvFJjPyiHIDRLzueAkpOe+qg+rf8wSakF9mNi9MIa5Xw\\\\\\\/cWJuQ8yvPkNeJh1fAAwwXgfj9Dg7aLWc7FR6HPy1xYoMdLRhtO6PZyRUKiCslRLbsYi96rbvjL8q+PQk9zs+HX6hv5OPVkY7US+aiw3oW49F+NngAYi69nILtgwXNaXSYWaUm7JzXhjhePbf5Bexbmkum8SWMarCUdghy2hXiVsFFkeSjK0zD\\\\\\\/U7b10F79vXVGWHx1ZdMmmt4\\\\\\\/F73AXsVpKmWd0SN+HtURHANcWI9fh7s+Zutg\\\\\\\/X1DHlAOlT3VxHC9WeGbfU7OSv2cBAboKrEcP3KuaE1CVnOtwLonx789\\\\\\\/6Rod1nwaae06Oy\\\\\\\/mJTTfZ9kUSEen99qaJWUvOom+Z2HckjbPA8m4b9CbQHdmeQmhrKuwKEXsUIEDztkTH1uQnMCvmYi2muO7lp0V93gSGiPc1eOHGgw5kL0H\\\\\\\/UF3JWEKhtxsP1BVSBzOLXPajo4AhWDZOnzzUWn2XrOlhMcbxXlbyVNWLPneylH8eXhZ3Sbn0pvXHCso4819JLBvn\\\\\\\/hcTQTvczVg3VVLdAqPnaaYuzp49ghxg4zc0nn8s7rQiV1yaWSXeZBQG2197qgIt8n241WIGW56nzFyafuTQv6T7KWKFTDFVk+GFh7T0PzLFH6sbiuJpEPH4ad\\\\\\\/i4Ry8tVwHf3Om57Qv\\\\\\\/RXiLEN\\\\\\\/iWNCvVxFJryu79u8i3BjeWdX+9MelNEPlZCQ\\\\\\\/BCa1PWbnhZ7kqQdXQe5Xnd\\\\\\\/obilXEUQMDUtoAu3TxUKbrbVVRZZQ6OFecYszGHeSqfhk1B1tGwlaAnEncRzs7J7HpEPzJisxj48VMGn8QR96sHxZFyBqdZjnBA4Nt9V6EF5HTfXDPTjY2Z2vfAdpBSyjbILhdGak+5+Gz44B1k17JM3UmwatP9daFo0051fz8abi8x7Z4U6QHH6CYDflZc6hQLRd7wnr5bO6HEcIUuew5dJYVXm6x25e2TdSWySuTdZPjRdcL4DqAlvLbLB6oAgxNXVt6V84\\\\\\\/4rxaYZ+be51bLuEtHHE1Ul7NC\\\\\\\/b47oufffS7aPgInhGkUVc6zwDhYiX4P91h6cthay\\\\\\\/\\\\\\\/xv5klIiiZgDPKZziiDrKgd7AxrNE+h7DXCX17BV+1luhh0hXfG1HF1nVBHRBsTBHi4OHgEWl7gO+GdgeN8C5iPtGY67mviYUHnTQYndMHbZvvMxu4nbQKCL\\\\\\\/k0m6H9TSJk2yc4QwE+tUwzSCrrma3Z6M5XJYqFSe11fziPvcOE1DM0eIt7rs52HORGdtOKKE7MpBXEqMO73h3nVJ+n7gkf5NGIZ2sRZPl5I2xTaE+Vi2sXCogfZOdTkjeq\\\\\\\/xvh2PbmdKJI2ZpADwka\\\\\\\/MxY1kMcBz2WplMmkoos8r5S65dR4DPh2evKt5TU5HFW1zLKJaKoN8flHFS\\\\\\\/3aIYMjMCEW1x7mrKYfraifeEtwrk+Btahc3chL9ysaAEYlyXrHCqNAE8fJb9UN\\\\\\\/569y8+GZ3WJ8QUXRt+JpGLTuls8XCSR61sAAsVJwz02idHSG2tDMg1f0lTkjesJsI78NljTo0gTI5z8zvAvmNet0X9i0Y2tPc3Dj96RcNDiXawkHLyOlacUPonSY9qQPxPzE59pDjApriD+3MSmPa2\\\\\\\/g0a6lJMp5T3Cr\\\\\\\/35JOLfkE5EkLZPVDNIsLDMzxa0aaUpyxb7tNNJGqT4NADxCQ8EM+oKHqHVFKqlGC7SF61p3hwik1RT8VZgYolf6+xwNUSW6BuKWyoUcFdvGpHctgLHwcyXUhiMVl9U71ZGM5+3z8rH1k2pdLBWGxijyUPusL4pPt2jJr4olDSLE9nKSEsLNDdsPIlFJjLTzlK8MqTc+sytWYyzkTRtttME3Xh\\\\\\\/roLzB6g4ViPt8VRmVhMiGYOlRW0Pg43zY9fIqYYG8uPj+\\\\\\\/uEMSTEUjhnsSGeejJkMCz+XZXbq8xUa7B3B50GETHiWdj7xf+BgGUfFnFi4y0NHq71bnVnEmqm4tGImMm0pexdTBqxHP1O+fR5kZO+WpCEExlLphGJf\\\\\\\/feWksDOp0rlcHD1HSmtNCUck0OzNA5MShtflJEMOvL61pClcig7sAVMSYn8cl5EmQc+lBJJ4S0VArrCpKH2v0egDtH2HbidN\\\\\\\/sTjnmg5jnKp4+o9+TOu5Cq7txSITvmnb\\\\\\\/z\\\\\\\/pa3ZAxmE85rkz\\\\\\\/91OgOcYYAapzlpxmAYzGHvRn4D+z47lxTG6OvHFKQ8B4J2m1exxAf+dMtqxmYqLvQHZZwInVzmThebexRfhG4z9\\\\\\\/VjwoNgmUEy8Z0q19F7ITeQYqWxZH7SqVEFbYM8DzasUFitAlnp5hF+7Z3x2bWDTx346uvEI\\\\\\\/g1Yp+eJLKISe2ufmtajDyvGSJJN1ej4LyPwcr\\\\\\\/Ow2OEthlUii2ccoUXHGQHD6vpyjGCSd65FXGAmlN2PJqxGfGQO6PLqFsETlny5185nM89qVQgdaEKQmNYQDOiNeM1SfRDFGzHuaatSzG0VvRoi00Kg78gsnDaCpUVHZ+gActDws4Zz5O\\\\\\\/QETPQGK4fqNnD5INs7YLHnsZ1W6qDlegAjelS63Nmsd1pnB6z6iAYOhHnOGjqTXB08ihmlrn3JV+rv7Ps\\\\\\\/0fG3KaBXR+2AcZKCmrF6WS\\\\\\\/8X9Vhh\\\\\\\/D4OiFY6\\\\\\\/izr9I9Q8awMhwV7g19ROudyLO\\\\\\\/P4V08O0gdb3HsIGOaHYyqmL2yjTbo4olT0ntGg7uDzSS+IE7z6qr4+01ct0kN79KnhgC6qZo7Ds96fp9wPQkfLcIWW5roT\\\\\\\/uegoq9fxq1DVjNAtakAGNr2VgLaGreoZkhfuD\\\\\\\/tirlcACLiLRJ3dnUf+cXYzP3Omhv3uqPO6\\\\\\\/NEpwS8jJc07NuQbtlvv\\\\\\\/EGdlszklj\\\\\\\/VBZB9nw24Ncuu1+rfH+hM3B7fMEmBMzwC5GxMZEyrb3RQrexq7zz2y76ZN2m9o8a8Fh3RCWavIt6pZXYFv4k7Mph+BhzJJyWZA3Q\\\\\\\/SOQ+seJNeGzEsFM8S9OoUc0YWbLjtbJgVDBJN41Yrxl32WuZl4QBmhSPIzKS2hR8CyxWCz1rJnCf0vjOlIRdoDjZlMx7KDrwn1fJRnO2XlBF0zKyKF+Jg+W1pJyJ+Rue6YAAmidGObl8F8xFz9ahm4OvcX2EuJ+sOGimD8H9Xm6lbrEEiUGhEjQemeRhppyd7gsw+Ck\\\\\\\/8dqsVqCAc4HAtkwgdqsNFiYbEIhhy+rvWcmsoTvOBDShYeG+wVTd5O\\\\\\\/KcMwk4k8VRQaLIDiEbJc9bMe1oe8gEaLyM73TY0qniB7bHr3oF64xQj0cuoSYOj84jektD1yt1m5JCJuMEdfGNeszI138Jsvn18PFST4gDMqaTmWUMOQOuzntrZhKbnyN+ZYf8OFvErpcg3TVBZDxJtxa069lCTF0ioX8JRd+8PL5bAUDbUw81TXFJz56MTWZShq0VbLJeE2\\\\\\\/OgtyXm56bPcqyvYobJB\\\\\\\/WjijXMH8b\\\\\\\/M7RP5gGvb2GmDpaA\\\\\\\/4X3UX5lfdVVt7dvoctxICtgHk9uQzK4zF+z\\\\\\\/FoLsMj9ILy8LwjwZNmiZsxk4uk2\\\\\\\/HwE5zpWdwig3y9MFP7SXh00IQXIniEwJqVWhmAxE+eMFurQ5kzVl4UE3Wx8o6QnWYeepnlTHJyl2+Qsb+mElbBQSdNZCVI7tDPRjl31gNAAlzDnRcaMg7VbqV8jb0BibPVPOvYFJE7C+SQs+qbh\\\\\\\/F0DeEQpPaQD08\\\\\\\/xGnGmZGzS5l62hp6Vmc3FG6fqiUfD6je6WNVSWtYnAFdgb7lHXYFN6JYwbP6FgWskdXKPFcweJ\\\\\\\/Y1vu5FZ1m0U56KK9hVmON\\\\\\\/F3Tk5hOYg2qihR39Va9s0MLB5wB7AWSkc1459\\\\\\\/UZwkJLr5SBJxWo12LhnNsAS3Je49qff+sIUuNe7ML9a4rr7XfQOJFBnGo2AH8XW8SK+Tmg30WoRyLeGwlk7rHpQIpSlEow0ff37ghfAVCU0eL5uKLJjAxh9LEcgEuhxPHEmSFeNPZTQn8GHL1cheEkYmLKsJChHAIanTD+B7mlTJ6FBgJ9aDYDPsicR4j2p\\\\\\\/255fr7TKHriZYSYMZI03kM07dWvePpHuPUBN+X1YwF9pqkVtpOGMpFLWPWH\\\\\\\/t6j4NDw4cKB6+rSuc2N7+YGmelaQrQ5\\\\\\\/+BLD82kd5mOoGZRos5r+LJ3LDsgSFgmQl\\\\\\\/GKjVXczSVjz662yLeIY5Q0QjsX+FhtK6LPhJU61N2zU375xcRSDx7\\\\\\\/svbYeLpzaisbyZ4i0rruj+bSQG04dIf8BbcDRFkHh\\\\\\\/i\\\\\\\/31sMODqhcjXvLrmpOex78z0Jp\\\\\\\/hVVvf1FB5Yj35p3TCDDkcFppGy+T9qmec\\\\\\\/ZHoZhxXahPWW\\\\\\\/el7Ne2p5XdkdismMCWN4GdbRCGO7nEcLwy90TrMK7F\\\\\\\/a2Q0wFGohOj\\\\\\\/2n5W5bsOatTVtFnVuauto2D9aI9+34j+Oue3PtQtPT6eUczkfEhjRNddoSs88W+x+T2N2XuvOaX\\\\\\\/LYJ5vhxlYXIK0NQFt6g1nZhi\\\\\\\/MgyR6bDq5gVD6pxEpMXd1U5cMa9tf7hhXBUh70UHd30EecDj1eZ4LgDPs+FDsxFPj53sOqIhV9lK6WK+ZvDLN7MJd00DbquWb2Q+BXy7iTbPQ3SVHV2CBZJ9GnAMrMGhMyPQ80dJkb5fmRpF+gY9NWAUfKDA+5t+QEHd1rsqUfI+DDyENc7lmYje8BbEzgV8nibeMMF6yRuovpdKnSrecUX9fz+3E8Z21ntzuWPbuD+EzsE\\\\\\\/O7pchiizCJs\\\\\\\/rkCBblrac5m0U0ptw9UQBu2hSsnPQ54YiUU2XkNi0iWhBwhouybooAoN+Z8Z9GLedr0ghuG0Wqd5lqMAWP9FgLTr4tHWcWluppmeSVsB7CMJCVpnSA9YzCX+0MYC8IuiRhmAntnmeyv+4Phw0JqSIUvWJgWgEjMLf9AGGCGW1SfkBjsHk1wW6t3fQO\\\\\\\/bSf6CnO5dbw2KLRaxNNbOJKXrNu1YOtCsYgvDMoGqKkDvfRY8zYss6P+beuWWOyvyzphYiNJ4mw5NbpLbowoSpt0SJ6\\\\\\\/NqDeRfOjTC7jOrh8Kk+itOMYFZdJQeMFlpCOcKRrPMCjOlnJD6Q\\\\\\\/P\\\\\\\/CkkM2j6Vk2D\\\\\\\/njrMrzYHzgPDCDnjg==\\\u0022,\\\u0022iv\\\u0022:\\\u002262lmzAnUi3VaBZeZVfkJ0Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022c24c3302fc46f2e391ad1460bd0d5bd31902a52af5a8a0b52fd54001ff20604c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022fqBMXxLtarxTe+gpwzoLxSYtsofdySVuJaGzybcQrIaBQzdCCirx0GAfYy5hWKovRw0LB4OWB9u8jyZphgfpqtsHAHDjHcSkMBdkD57dAdy\\\\\\\/i0QOSmsB1gofFA8cb1NX4LBeQ9820ohaj6Rr4bGfPt1V3TGuu7qQMYtDxjPZ8uQJlQRqifMWZJqjQiDWmRY1\\\\\\\/PXDbHknHUD5ShZj7RWmPsfsTwERfDn7IMcMYhshf2HSh965ajrAAHxicrXWtSXw2qV2BfhCLoipZKppMs+LAQwwCorXA1kcLOSQu3SNr2pxWtqyOW42+\\\\\\\/8QTeNbp7fAt31Ko0ctvqbz1IdDadQCVim7YGp6JAGy1HYmqMSK1TTi1jarVpuBAXKe7PtZ5kQuFHNxnjYhUDjWLrIrFfp96tHH3DhY\\\\\\\/Z3PrX6GErPgX2IQp\\\\\\\/dRwqgQI3t\\\\\\\/Y1gReYDUAD2YDiTAAUX5xks5ZxosWqKpxyN9ClRlCuh4E\\\\\\\/nbU9qYTYNVdazGjz66BcGkxEci+35jf\\\\\\\/r1Wum\\\\\\\/Q8GwunGcNeGdQHkBeS9n0PiM6\\\\\\\/tU+zgW\\\\\\\/qupum3ZEax+AqSbpvgtuqMvGcqsRx22pW0itinXSgFgQuqYl31xPCLWSQ6NWcv2pd3bin9p+MdUF2MYyqU+xsh\\\\\\\/b4vCqu9yjpL3sUufVNiXiuKEdi3FBhLRkkx96ZdFmVzfLIFkJon5KZPdBsK4GFvtEXYKyZdSHWYHK9NC2\\\\\\\/bleY82+9450VUTUF+gOmyOAIuLWuVoCG6OpsN783buCC8brTJy4gEhXsyFrwrPQVXhSVfoiTD33mtOeGTq8l0zyIxPlS8pwduThCILT9YtPwcvtOWPQLIqmEfQldwVKkGUGuWA+htJQxC1Qys3bGGKEAUUVRB7Ceod7adsgzlJngx9Pdq3XP0SjpF2ivJ3IO24r+c2jtfmw2KcoArjyMuJ5TdW0qtD8WiKhKy2Ub8piEWqcf6iUcxmb8Cll8F8pP+tYty0mj3jV7X9iwk7JDdvTC08Bry1mlherpEBOJ4goZcu19I0uShgujhCcfj7OpEeeiNe5p0WrbuMqPqerqDWrHaLdux800X5y+B8mPsA+CgBwei4aFCH61Dq1hSZ0rMuyKAP9KjyM47mGsQ7VvaUukL1Tld5UEqq\\\\\\\/2q0\\\\\\\/QMFjknG7a8xRSwaptdQvArLkYU8gQbPFy6X9a+HifMZ+Zs8+qxZ0Io1GCHr89wC6IxogS88C0Ef3bBSfyoy4IuC0I9CECKpLcw3\\\\\\\/l1tP5SYUwrpq9kVj7NPT+G1ztPtwDs9rlYIX8VAjB+NyfePgu+qIMwgBOH\\\\\\\/7fh127b9lkgyA9XAQFvqssW0OaFgFgsvYkI3ezVXtXVMp72m53WldGM8pHtJFVLEyTczrBBwA4etqCR2jgtk5fmxLlNO6pLuUmciG22bfNc9vI7byiSOMYELiWU4pzTl05BK+v6PO+Ot9qcicxzFUBQI+8AKc0ef279Gg8fKPyXvcOlzO+Or6ZTM7aAeC+dJ1ugMCTP90og6n20hIYbm897Yw7MRPT3kAIaD1RNJb4GTPd37fmvAm8RBasCMVfkhPcmwiOT\\\\\\\/8RiKawwu46cXJGz6dQvEiiGA3SwEHthnpPjP8FAI1JCqRfMzzROCgTe8gL1sGbHpaAF8qUI3usdKKsX1dDzQ2+Zne7xKEvW5GEPYFXPe9W3z6vf41jYfpx4iJS0lJBYTytd+gjo6WERtUxn\\\\\\\/g0P883wb3FDcihpYgmD0eJqrVONbSl8DIJHtT\\\\\\\/miB1G5vYtlitml50gIcYxmugTwtN9JOyeYW\\\\\\\/+rUVS6FgbN7vyThFpLgmpWwsktHCAuDy\\\\\\\/Xewuy939LT\\\\\\\/FttNzgXxKN+EjJ9NbIwUyeOyqSUv+AZoxP5t7WnJVz8P6Shqvl3hjDbIrkngxak0VpkJ6I+idJJ5oAJFu1OmWMO04v5XJnNZtAdutArZlgEhGpOa5s1\\\\\\\/MJQl4acoj5HEQr5OapXjA\\\\\\\/2PYfW780loouFMI9T+B68s\\\\\\\/fi7DJ24TqEw==\\\u0022,\\\u0022iv\\\u0022:\\\u00227enR6JJopXB3FSl1eCkB4A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002285d8bf0c28b2fa4b6efb62c61d4c85c09238ba10ab7eab04ff978d81f17f27ad\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459570 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/1979" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-uby3rdyfJsVmmMvCqXWswSXP00x9W2fgCSOO5RZf9bE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/1979" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-le09R6Yv-eyNCt9q0HYDIdRwuLkKlSeUARzuN8XeoJI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/1965" title="Characterization and Prediction of Haploinsufficiency Using Systems-Level Gene Properties in Yeast" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/11/1997" title="Cell-Type–Specific Transcriptional Profiles of the Dimorphic Pathogen Penicillium marneffei Reflect Distinct Reproductive, Morphological, and Environmental Demands" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459570" id="node459570" data-pisa="ggg;3/11/1979" data-pisa-master="ggg;g3.113.006601" data-apath="/ggg/3/11/1979.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Condition-Dependent Regulation of Lipid Metabolism in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Michael C. Jewett</span>, <span class="highwire-citation-author" data-delta="1">Christopher T. Workman</span>, <span class="highwire-citation-author" data-delta="2">Intawat Nookaew</span>, <span class="highwire-citation-author" data-delta="3">Francisco A. Pizarro</span>, <span class="highwire-citation-author" data-delta="4">Eduardo Agosin</span>, <span class="highwire-citation-author" data-delta="5">Lars I. Hellgren</span> and <span class="highwire-citation-author" data-delta="6">Jens Nielsen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1979-1995;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006601 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459570" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michael C. Jewett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Chemical and Biological Engineering and Chemistry of Life Processes Institute Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BC.%2BJewett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jewett%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BC.%2BJewett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christopher T. Workman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Biological Sequence Analysis, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BT.%2BWorkman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Workman%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BT.%2BWorkman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Intawat Nookaew </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Chemical and Biological Engineering, Chalmers University of Technology, SE-412 96 Göteborg, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Faculty of Engineering, King Mongkut’s University of Technology Thonburi, Bangkok 10140, Thailand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Intawat%2BNookaew%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nookaew%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIntawat%2BNookaew%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Francisco A. Pizarro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Chemical and Bioprocess Engineering, School of Engineering, Pontificia Universidad Católica de Chile, Macul, Santiago 8320000, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BA.%2BPizarro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pizarro%20FA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BA.%2BPizarro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Eduardo Agosin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Chemical and Bioprocess Engineering, School of Engineering, Pontificia Universidad Católica de Chile, Macul, Santiago 8320000, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eduardo%2BAgosin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Agosin%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEduardo%2BAgosin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Lars I. Hellgren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Biological Sequence Analysis, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BI.%2BHellgren%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hellgren%20LI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALars%2BI.%2BHellgren%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jens Nielsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Chemical and Biological Engineering, Chalmers University of Technology, SE-412 96 Göteborg, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jens%2BNielsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nielsen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJens%2BNielsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nielsenj@chalmers.se" class="" data-icon-position="" data-hide-link-title="0">nielsenj@chalmers.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/11/1979" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459570" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459570/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/1979.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459570" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459570/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/1979.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459570" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459570/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/1979.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459570" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459570/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/11/1979.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459570" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459570/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Lipids play a central role in cellular function as constituents of membranes, as signaling molecules, and as storage materials. Although much is known about the role of lipids in regulating specific steps of metabolism, comprehensive studies integrating genome-wide expression data, metabolite levels, and lipid levels are currently lacking. Here, we map condition-dependent regulation controlling lipid metabolism in <em>Saccharomyces cerevisiae</em> by measuring 5636 mRNAs, 50 metabolites, 97 lipids, and 57 <sup>13</sup>C-reaction fluxes in yeast using a three-factor full-factorial design. Correlation analysis across eight environmental conditions revealed 2279 gene expression level-metabolite/lipid relationships that characterize the extent of transcriptional regulation in lipid metabolism relative to major metabolic hubs within the cell. To query this network, we developed integrative methods for correlation of multi-omics datasets that elucidate global regulatory signatures. Our data highlight many characterized regulators of lipid metabolism and reveal that sterols are regulated more at the transcriptional level than are amino acids. Beyond providing insights into the systems-level organization of lipid metabolism, we anticipate that our dataset and approach can join an emerging number of studies to be widely used for interrogating cellular systems through the combination of mathematical modeling and experimental biology.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/integrated-systems-biology" class="hw-term hw-article-keyword hw-article-keyword-integrated-systems-biology" rel="nofollow">integrated systems biology</a></li><li class="kwd"><a href="/keyword/lipid-metabolism" class="hw-term hw-article-keyword hw-article-keyword-lipid-metabolism" rel="nofollow">lipid metabolism</a></li><li class="kwd"><a href="/keyword/regulation" class="hw-term hw-article-keyword hw-article-keyword-regulation" rel="nofollow">regulation</a></li><li class="kwd"><a href="/keyword/metabolome" class="hw-term hw-article-keyword hw-article-keyword-metabolome" rel="nofollow">metabolome</a></li><li class="kwd"><a href="/keyword/omics" class="hw-term hw-article-keyword hw-article-keyword-omics" rel="nofollow">omics</a></li></ul><p id="p-5">Lipid metabolism in yeast comprises more than 300 chemical reactions and 100 chemical species (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Nookaew <em>et al.</em> 2008</a>). These chemical species include fatty acids, sterols, steryl esters, acylglycerols, phospholipids, and sphingolipids. Despite a wide range of chemical and structural diversity, all lipid species share a common carbon precursor, namely acetyl-CoA, and only a few additional molecules, which include, for example, dihydroxyacetone phosphate, glucose-6-phosphate, serine, and S-adenosyl methionine, are needed to synthesize almost all lipids. With so few precursors needed to synthesize such a large number of diverse lipid species, a high level of genetic and cellular regulation is required to ensure homeostasis in lipid composition. It is well-known that lipid metabolism is tightly regulated, particularly at the level of transcription (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Nielsen 2009</a>). For example, the transcriptional regulation of sterol homeostasis by <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002621">Upc2p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004218">Ecm22p</a> (sterol regulatory element-binding protein in humans) uses a feedback pathway to concomitantly control synthesis and uptake (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Vik and Rine 2001</a>). As another example, regulation of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003689">INO1</a></em> is well-established and serves as a model for studying transcriptional control in general (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chen <em>et al.</em> 2007</a>).</p><p id="p-6">Elucidating global regulatory modules in lipid metabolism and how they interact in response to environmental conditions has direct implications for human health and biotechnology (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Smith <em>et al.</em> 2007</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Brehme and Vidal 2010</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gallego <em>et al.</em> 2010</a>). In humans, disorders in lipid metabolism, transport, and trafficking are involved in the etiology of many diseases, such as atherosclerosis, diabetes, and cancer (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Henneberry and Sturley 2005</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Maxfield and Tabas 2005</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Hannun and Obeid 2008</a>). Thus, understanding how lipid metabolism is regulated could lead to identification of new therapeutic strategies for treatment of some of these diseases. In biotechnology, efforts to engineer lipid metabolism in microbes offer a promising route toward the production of biodiesel (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Steen <em>et al.</em> 2010</a>) and dietary supplements or food ingredients (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Ruenwai <em>et al.</em> 2010</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Ruenwai <em>et al.</em> 2011</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Tavares <em>et al.</em> 2011</a>). Hence, mechanistic insights into cellular regulation could help to identify novel metabolic engineering targets, and thus the development of efficient cell factories for the production of these lipids. Despite these interests in understanding regulation of lipid metabolism, comprehensive studies aimed at bridging the gap between transcriptional state and metabolic phenotype are lacking.</p><p id="p-7">Although lacking in lipid metabolism, integrated systems biology studies have been important in gaining a quantitative understanding of complex biological systems (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Sauer <em>et al.</em> 2007</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Yamada and Bork 2009</a>). For example, such studies have revealed DNA damage response pathways (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Workman <em>et al.</em> 2006</a>), the systemic impact of growth rate (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Regenberg <em>et al.</em> 2006</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Castrillo <em>et al.</em> 2007</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fazio <em>et al.</em> 2008</a>), the role of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002885">Snf1p</a> as a global energy regulator (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Usaite <em>et al.</em> 2009</a>), the interactions between <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002885">Snf1p</a> and TORC1 (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Zhang <em>et al.</em> 2011</a>), novel biosynthetic control mechanisms in amino acid metabolism (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Moxley <em>et al.</em> 2009</a>), physiological differences between laboratory and wine yeasts (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Pizarro <em>et al.</em> 2008</a>), the functional landscape of a genome-reduced bacterium (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Guell <em>et al.</em> 2009</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kuhner <em>et al.</em> 2009</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Ochman and Raghavan 2009</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Yus <em>et al.</em> 2009</a>), strategies metabolic networks use to achieve robust operation (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ishii <em>et al.</em> 2007</a>), the key factors involved in programmed fruit ripening and tomato development (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Rohrmann <em>et al.</em> 2011</a>), a model of sphingolipid metabolism (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Gupta <em>et al.</em> 2011</a>), metabolic pathways involved in environmental adaptation (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gianoulis <em>et al.</em> 2009</a>), potential gene–metabolite interactions (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bradley <em>et al.</em> 2009</a>), and general design principles controlling gene expression levels/enzyme abundance levels (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Fendt <em>et al.</em> 2010</a>), among others. The wealth of information generated from these studies emphasizes the importance of integrating data across multiple levels of the cell (mRNAs, proteins, metabolites, fluxes, and others) and protein interaction information (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Feist and Palsson 2008</a>; <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Yamada and Bork 2009</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Heinemann and Sauer 2011</a>).</p><p id="p-8">In this study, we set out to elucidate global regulatory structure controlling lipid metabolism under different environmental conditions (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Toward this goal, we created a model of lipid metabolism that allows for the integration of mRNA, metabolite, lipid, and flux data with known networks of protein-DNA interactions and metabolic reaction stoichiometry. We present the results of our measurements together with integrative methods for analyzing high-throughput experimental datasets. The condition specificity of the measured data provides the possibility to infer some insights into the global regulatory architecture of lipid metabolism (<em>i.e.</em>, regulation structure that is general and can be applied over different environmental perturbations) and into how fluxes toward different lipid species are controlled.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="A systems approach to mapping condition-dependent lipid metabolism in the yeast Saccharomyces cerevisiae. (A) Cartoon representation of the 2 &#xD7; 3 factorial design showing each experimental condition as a point on a cube (C-limited, C; N-limited, N; aerobic, O; anaerobic, A; 30&#xB0;, T; and 15&#xB0;, t). (B) For each condition (e.g., COT), mRNAs, metabolites, lipids, and reaction fluxes were measured and mapped onto a metabolic model visualized in Cytoscape. (C&#x2013;F) Integrative analyses used to query the measurement data. These included the identification of correlation networks (C), co-regulated gene neighborhoods (D), co-regulated pathway neighborhoods (E), and transcription factor (TF) correlated modules (F). (D and E) M1 represents a target metabolite in the reaction network. Node colors are meant to indicate a log2 color bar for measurement ratios, such as aerobic vs. anaerobic conditions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1964801122" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;A systems approach to mapping condition-dependent lipid metabolism in the yeast &lt;em&gt;Saccharomyces cerevisiae&lt;/em&gt;. (A) Cartoon representation of the 2 &#xD7; 3 factorial design showing each experimental condition as a point on a cube (C-limited, C; N-limited, N; aerobic, O; anaerobic, A; 30&#xB0;, T; and 15&#xB0;, t). (B) For each condition (&lt;em&gt;e.g.&lt;/em&gt;, COT), mRNAs, metabolites, lipids, and reaction fluxes were measured and mapped onto a metabolic model visualized in Cytoscape. (C&#x2013;F) Integrative analyses used to query the measurement data. These included the identification of correlation networks (C), co-regulated gene neighborhoods (D), co-regulated pathway neighborhoods (E), and transcription factor (TF) correlated modules (F). (D and E) M1 represents a target metabolite in the reaction network. Node colors are meant to indicate a log&lt;sub&gt;2&lt;/sub&gt; color bar for measurement ratios, such as aerobic &lt;em&gt;vs.&lt;/em&gt; anaerobic conditions.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1979/F1.medium.gif" width="440" height="308"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/11/1979/F1.medium.gif" width="440" height="308"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463531" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">A systems approach to mapping condition-dependent lipid metabolism in the yeast <em>Saccharomyces cerevisiae</em>. (A) Cartoon representation of the 2 × 3 factorial design showing each experimental condition as a point on a cube (C-limited, C; N-limited, N; aerobic, O; anaerobic, A; 30°, T; and 15°, t). (B) For each condition (<em>e.g.</em>, COT), mRNAs, metabolites, lipids, and reaction fluxes were measured and mapped onto a metabolic model visualized in Cytoscape. (C–F) Integrative analyses used to query the measurement data. These included the identification of correlation networks (C), co-regulated gene neighborhoods (D), co-regulated pathway neighborhoods (E), and transcription factor (TF) correlated modules (F). (D and E) M1 represents a target metabolite in the reaction network. Node colors are meant to indicate a log<sub>2</sub> color bar for measurement ratios, such as aerobic <em>vs.</em> anaerobic conditions.</p><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Strain and chemostat cultivations</h3><p id="p-10">The reference laboratory strain <em>S. cerevisiae</em> CEN.PK113-7D (<em>MAT</em><strong>a</strong>) (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">van Dijken <em>et al.</em> 2000</a>) was grown in well-controlled 2-liter jacketed chemostats (Braun Biotech) with a constant working volume of 1.0 liter. A factorial design was pursued with three factors comprising two levels each: carbon source [C-limited (C); N-limited (N)]; oxygen availability [aerobic (O); anaerobic (A)]; and temperature [30° (T); 15° (t)]. Cultivations were performed (in triplicates) at each of the eight conditions (COT, CAT, COt, CAt, NOT, NAT, NOt, NAt) for a total of 24 experiments. Fermentation conditions used a stirrer speed of 800 rpm, with pH of 5.0 (maintained by automatic addition of 2 N potassium hydroxide) and a dilution rate of 0.05 hr<sup>−1</sup>. Aerobic conditions were maintained by sparging the cultures with air (1.0 liter⋅min<sup>−1</sup>) and the concentration of dissolved oxygen was measured with Mettler Toledo polarographic electrode. Anaerobic conditions were maintained by sparging the medium reservoir and the fermentor with pure nitrogen gas (0.5 liter⋅min<sup>−1</sup>). Moreover, oxygen diffusion was minimized by using norprene tubing and butyl septa. The bioreactors were fitted with cooled condensers (2°–4°) and the off-gas was led to a gas analyzer (INNOVA and NGA 2000 Rosemount) to measure the content of CO<sub>2</sub> and O<sub>2</sub>. Steady-state was reached when at least five but less than seven residence times had passed since starting the continuous cultivation and carbon dioxide evolution, dry weight measurements, and high-performance liquid chromatography (HPLC) measurements of extracellular metabolites were constant.</p></div><div id="sec-3" class="subsection"><h3>Media</h3><p id="p-11">The medium composition was as previously described (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tai <em>et al.</em> 2005</a>). For N-limited cultivations, residual glucose concentration in the chemostat was targeted to 17 ± 2 g⋅liter<sup>−1</sup>. This was to sustain glucose repression at the same level in all cultivations. To achieve the targeted glucose concentrations in the N-limited cultivations, the glucose feed concentration was 46 g⋅liter<sup>−1</sup> for cells grown at 15° and 74 g⋅liter<sup>−1</sup> for cells grown at 30°. The mineral medium composition for the N-limited cultivations was (amounts per liter) as follows: 1 g (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>; 3 g KH<sub>2</sub>PO<sub>4</sub>; 5.3 g K<sub>2</sub>SO<sub>4</sub>; 0.5 g MgSO<sub>4</sub>⋅7H<sub>2</sub>O; 1 ml Trace Metal Solution; 0.05 ml antifoaming agent; and 1 ml vitamin solution. The mineral medium composition for the C-limited cultivations was (amounts per liter) as follows: 5 g (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>; 3 g KH<sub>2</sub>PO<sub>4</sub>; 0.5 g MgSO<sub>4</sub>⋅7H<sub>2</sub>O; 1 ml Trace Metal Solution; 0.05 ml antifoaming agent; and 1 ml vitamin solution. The trace metal solution and vitamin solution are reported in <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Verduyn <em>et al.</em> (1992)</a>. The anaerobic cultivation medium was supplemented with 420 mg⋅L<sup>−1</sup> Tween 80 and 10 mg⋅L<sup>−1</sup> ergosterol (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Pizarro <em>et al.</em> 2008</a>). Antifoaming agent was used at a concentration of 0.15 ml/liter.</p></div><div id="sec-4" class="subsection"><h3>Dry weight measurements</h3><p id="p-12">The concentration of biomass at steady-state was determined on a dry weight basis by filtering 5 ml of culture through a preweighed 0.45-µm nitrocellulose filter (Gelman Sciences, Ann Arbor, MI). The filter was washed with distilled water, dried in a microwave oven at 150 W for 15 min, and finally weighed to determine its increase in dry weight.</p></div><div id="sec-5" class="subsection"><h3>RNA sampling and isolation</h3><p id="p-13">Samples for RNA isolation from aerobic cultivations were taken by rapidly sampling 20 ml of culture into a tube with 35–40 ml crushed ice to decrease the sample temperature to less than 2° in less than 10 sec. Cells were then centrifuged (4500 rpm at 0° for 3 min), instantly frozen in liquid nitrogen, and stored at −80° until further use. Sampling for RNA isolations from anaerobic cultivations was performed as described (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Piper <em>et al.</em> 2002</a>). Total RNA was extracted using FastRNA Pro RED kit (QBiogene, Inc, USA) according to manufacturer’s instructions after partially thawing the samples on ice. RNA sample integrity and quality was assessed before hybridization with an Agilent 2100 Bioanalyzer and RNA 6000 Nano LabChip kit.</p></div><div id="sec-6" class="subsection"><h3>Probe preparation and hybridization to arrays</h3><p id="p-14">mRNA extraction, cDNA synthesis, cRNA synthesis and labeling, and hybridization to Affymetrix Yeast Genome 2.0 arrays were performed with a GeneChip one-cycle target labeling kit as described in the Affymetrix user’s manual (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Affymetrix 2004</a>). Washing and staining of arrays were performed using a GeneChip fluidics station 400 and scanning with an Affymetrix GeneChip Scanner 3000.</p></div><div id="sec-7" class="subsection"><h3>Microarray data analysis</h3><p id="p-15">Raw microarray data (CEL files) were analyzed using R and BioConductor (<a href="http://www.bioconductor.org">http://www.bioconductor.org</a>). Briefly, CEL files were normalized and the probe sets were summarized using the GeneChip RMA (GCRMA) package (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Wu <em>et al.</em> 2003</a>) and the “affinities” method. The CEL files and the normalized GCRMA table can be found in the ArrayExpress database (<a href="http://www.ebi.ac.uk/arrayexpress/">http://www.ebi.ac.uk/arrayexpress/</a>) with accession E-MEXP-3704.</p></div><div id="sec-8" class="subsection"><h3>Multi-way ANOVA</h3><p id="p-16">A linear model was applied to each gene to estimate the influence of each experimental factor (using C, O, and T here as abbreviations for C:N, O:A, and T:t respectively) and the potential interaction between pairs of factors (C:O, C:T, O:T) using an ANOVA approach (multi-way ANOVA). The model for mRNA expression each gene, <em>i</em>, was defined as:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1979/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1979/embed/mml-math-1.gif"/></noscript></span></span>where parameters <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1979/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1979/embed/mml-math-2.gif"/></noscript></span></span> were fit for each experimental factor in set F = {C,O,T} and those for <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1979/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/1979/embed/mml-math-3.gif"/></noscript></span></span> were fit for interaction factors.</p></div><div id="sec-9" class="subsection"><h3>Flux balance analysis</h3><p id="p-17">Estimation of <em>in silico</em> fluxes for all growth conditions was performed following a flux balance analysis (FBA) approach using the biopt software (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Cvijovic <em>et al.</em> 2010</a>). The specific medium and all measurements of highly secreted metabolites (<em>e.g.</em>, ethanol, glycerol, acetate) from our experiments were directly used as additional constraints in addition to the stoichiometry of the <em>iIN800</em> model (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Nookaew <em>et al.</em> 2008</a>), with the maximization of growth rate used as the objective function. The biomass equations were formulated based on specific lipid composition and carbon-limited or nitrogen-limited growth from each of the eight growth conditions.</p></div><div id="sec-10" class="subsection"><h3>Biomass amino acid <sup>13</sup>C enrichment analysis</h3><p id="p-18"><sup>13</sup>C-enriched biomass was achieved by chemostat experiments performed in in-house-built reactors with a working volume of 200 ml. Cultivations were performed at 30° or 15° with an agitation speed of 600 rpm, a dilution rate of 0.05 hr<sup>−1</sup> (to match our other experiments), and pH of 5.0, and gas flow rate was kept at 1 vvm by air and N<sub>2</sub> for aerobic growth and anaerobic growth, respectively. Steady-state samples were taken after five volume changes; 100% of the glucose used was labeled in position 1 (1-<sup>13</sup>C glucose was from Omicron Biochemicals Inc, USA). <sup>13</sup>C-labeled biomass was harvested by centrifugation at 4000 rpm and 0° for 5 min. After centrifugation, the supernatant was poured off, and the cell pellet was frozen instantaneously in liquid nitrogen and stored at –80°. The samples were derivatized with both ECF derivatization and DMFDMA derivatization (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Christensen <em>et al.</em> 2002</a>). Derivatized samples were injected on a GC-MS as described previously to determine metabolite isotopes (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Christensen <em>et al.</em> 2002</a>). The flux distribution of any growth condition was calculated by the adopted model of <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gombert <em>et al.</em> (2001)</a> under the concept of summed fractional labeling (SFL) (<a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Christensen <em>et al.</em> 2002</a>).</p></div><div id="sec-11" class="subsection"><h3>Endometabolome analysis</h3><p id="p-19">Cells were rapidly quenched according to <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">de Koning and van Dam (1992)</a> in 60% (v/v) buffered (12.5 mM Tricine, pH 7.4) cold methanol at –40 to –45°. After quenching, the cells were immediately centrifuged at 10,000 <em>g</em> for 4 min in a rotor precooled to −20° to separate the cells from the quenching solution. Chloroform:methanol:buffer (CMB) extraction was performed (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Villas-Boas <em>et al.</em> 2005a</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Villas-Boas <em>et al.</em> 2005b</a>). After extraction, samples were freeze-dried at a low temperature (−56°) using a Christ-Alpha 1-4 freeze dryer (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">de Koning and van Dam 1992</a>). Amino and nonamino organic acid levels were determined by HPLC on an Aminex HPX-87H column (Biorad) according to <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Fazio <em>et al.</em> (2008)</a> and by GC-MS analysis according to <a id="xref-ref-74-2" class="xref-bibr" href="#ref-74">Villas-Boas <em>et al.</em> (2005b)</a>, except that a Finnegan FOCUS gas chromatograph coupled to single quadrupole mass selective detector (EI; Thermo Electron Corporation, Waltham, MA) was used. Peak enumeration was conducted with AMDIS (NIST, Gaithersburg, MD) with default parameters, and identification of conserved metabolites was conducted with SpectConnect (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Styczynski <em>et al.</em> 2007</a>) using default parameters and a support threshold of 3. Samples were normalized by an internal standard chlorophenylalanine (30 μl of a 4 mM solution was added before extraction) and by the biomass weight per sample.</p></div><div id="sec-12" class="subsection"><h3>Exometabolome analysis</h3><p id="p-20">Culture samples for determination of exometabolites were immediately filtered through a cellulose acetate filter (CAMEO 25GAS 0.22; Osmonics, Minnetonka, MN). GC-MS and HPLC analysis were performed as indicated. Samples were normalized by the biomass weight per sample.</p></div><div id="sec-13" class="subsection"><h3>Lipid analysis</h3><p id="p-21">Samples for lipid isolation were taken by rapidly centrifuging at 4° and 10,000 rpm. The cell pellets were rinsed twice with DI water, instantly frozen in liquid nitrogen, and stored at −80° until lipid extraction. For quantification of lipid levels, internal standards [heptadecanoic acid, TG (17:0/17:0/17:0); phosphatidylcholine (PC) (17:0/17:0); PS (17:0/17:0); PI (17:0/17:0); phosphatidylethanolamine (PE) (17:0/17:0); ceramide (d17:1/18:1); and sitostanol] were added to each sample before extraction. The lipid extraction was performed using a modified Folch method (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Folch <em>et al.</em> 1957</a>) using a shaker at 4° with an agitation speed of 200 rpm overnight. The lipid extract was divided into three parts for further analysis of polar lipids, neutral lipids, and sphingolipid components. Using TLC, the polar lipid fraction was separated into PC, PS, PI, and PE using a solvent system consisting of chloroform:methanol:HAc:boric acid [40:20:30:10:1.8g (v/v/v/v/w)] and neutral lipid fraction was separated into sterols, free fatty acids (FFAs), TG, and steryl esters using a solvent mixture of hexane:diethylether:formic acid [80:20:1 (v/v/v)]. After development, lipid spots were visualized using 2,7-dichlorofluorescein and silica gel in the spots was carefully scraped off of TLC plates for analysis of fatty acid content and distribution of each lipid class. Specifically, glycerolipids were <em>trans</em>-methylated using a method combining alkaline methanolysis and BFl<sub>3</sub>-catalyzed <em>trans</em>-methylation described by <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Hamilton <em>et al.</em> (1992)</a>, which has been validated in-house and analyzed on a GC equipped with a flame-ionization detector (GC-FID), as described (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brix <em>et al.</em> 2010</a>). Masses of each lipid class were calculated based on the peak area of the heptadecanoic acid released from the internal standards. The use of preparative TLC for separation of the different lipid classes before quantification of fatty acid content in each class using GC-FID analysis has been clearly established as a standard method, and when appropriate internal standards are used relative losses during methylation and extraction of the FAME from the silica are fully compensated by equal losses of the internal standards. Hence, quantification of each lipid class is highly reliable. Sterols were extracted from TLC plates following the validated method described previously (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Zangenberg <em>et al.</em> 2004</a>). The sterol extract was then derivatized with the TMS method as described previously (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Nielsen and Madsen 2000</a>) and injected into GC-MS equipped with 5% phenol column (Zebron column) and quantified in relation to the peak area for sitostanol. For analysis of the sphingolipid fraction, the lipid fraction was hydrolyzed using the method of <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Aveldano and Horrocks (1983)</a>. The long-chained base from the sphingolipids was extracted, derivatized with o-phtalaldehyde (OPA) reagent, and then analyzed by reverse-phase HPLC as described (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Merrill <em>et al.</em> 1988</a>). Sphingosine, phytosphingosine, C-20 sphingosine, and sphinganine were identified using authentic standards, and the content of sphingolipids was quantified based on the peak area of these compounds compared to the C17 long-chained base derived from the internal standard. Samples were normalized by the biomass weight per sample. All lipid data are presented as μmol/g dry cell weight (DCW), having been converted from mass units obtained from the GC-MS using the molecular weight for each species.</p></div><div id="sec-14" class="subsection"><h3>Visual metabolic map</h3><p id="p-22">To gain an integrated perspective of environmental growth condition perturbations on gene, metabolite, and lipid expression, these data were visualized in Cytoscape (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cline <em>et al.</em> 2007</a>) in a topological map based on the genome-scale metabolic model <em>iIN800</em>. Genes, metabolites, lipids, and <em>in silico</em> computed reaction fluxes were defined as nodes. These nodes were connected by one of three types of edges: gene (enzyme)-reaction; metabolite/lipid-reaction; and lipid distribution edges. Gene (enzyme)-reaction and metabolite/lipid-reaction edges were defined from the stoichiometry of the genome-scale model. Lipid distribution edges mapped acyl-chain composition for different species. For reasonable visualization, linear pathway reactions were merged, for example, the following set of reactions, PE + SAM → PMME + SAH, PMME + SAM → PDME + SAH, and PDME + SAM → PC + SAH was simplified to PE + SAM → PC + SAH for metabolites S-adenosyl-L-methionine (SAM), S-adenosyl-homocysteine (SAH), phosphatidylethanolamine (PE), phosphatidylmonomethylethanolamine (PMME), phosphatidyldimethylethanolamine (PDME), and phosphatidylcholine (PC). The simplified reaction PE + SAM → PC + SAH had two gene reaction associations: <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003389">CHO2</a></em> (encoding the enzyme catalyzing reaction PE + SAM → PMME + SAH ) and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003834">OPI3</a></em> (encoding the enzymes catalyzing reactions PMME + SAM → PDME + SAH and PDME + SAM → PC + SAH). To help parse the complexity, we found it useful to visualize smaller networks, such as central metabolism, the TCA cycle and amino acid biosynthesis pathways, sterol biosynthesis, phospholipid biosynthesis, fatty acid biosynthesis, sphingolipid biosynthesis, and lipase degradation.</p></div><div id="sec-15" class="subsection"><h3>Correlation analysis</h3><p id="p-23">The influence of gene expression on lipid and metabolite levels, or vice versa, was investigated using Pearson product-moment correlation coefficient, <em>r</em>. Correlation coefficients were calculated between transcript levels for all gene expression level–lipid and gene expression level–metabolite combinations across all 24 samples tested. The significance of each correlation coefficient was estimated (test for no correlation using <em>cor.test</em> method in R with standard “two-sided” alternative hypothesis) and the resulting <em>P</em> values were adjusted using the Bonferroni method to account for the number of gene expression level–lipid or gene expression level–metabolite combinations tested.</p></div><div id="sec-16" class="subsection"><h3>Metabolic network gene neighborhood analysis</h3><p id="p-24">An extended yeast metabolic network based on <em>iIN800</em> that included acyl chain information and that contained 1754 reactions, 781 genes, and 897 lipids and metabolites was used to investigate regions of surprising correlation between lipids/metabolites and the mRNA of enzymes that may generate or utilize them (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">Nookaew <em>et al.</em> 2008</a>). The modified reconstruction is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/006601SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/TableS1.xlsx">Table S1</a>. To achieve this, a neighborhood analysis for each lipid or metabolite was performed as follows. For each lipid or metabolite, the Pearson correlation coefficients to all neighboring enzyme transcripts (lipid–gene expression level or metabolite–gene expression level correlations) were compared to their correlation coefficients to all non-neighbor gene expression levels using a Mann-Whitney <em>U</em>-test (Wilcoxon rank-sum test). The nonparametric, two-sided hypothesis test provided a <em>P</em> value for each lipid and metabolite regarding how positively or negatively correlated it was to genes directly upstream or downstream in the metabolic network as compared to all other non-neighboring genes. In addition to this “first-neighbor” metabolic network analysis, a similar “second-neighbor” analysis was performed that defined neighbor correlations for all genes within two reaction steps of the lipid/metabolite in question.</p></div><div id="sec-17" class="subsection"><h3>KEGG pathway analysis</h3><p id="p-25">Using the pathway definitions in KEGG to group sets of reactions and sets of lipids/metabolites, we performed a Mann-Whitney analysis similar to that applied to individual lipids/metabolites to assess the significance of correlation within a defined pathway represented by the mRNAs encoding the enzymes and the lipids/metabolites they produce. This test was performed to compare the sample of correlation values between lipids/metabolites and gene expression levels within a pathway to all other correlations between lipids/metabolites and gene expression levels not in the tested pathway.</p></div></div><div class="section results-discussion" id="sec-18"><h2 class="">Results and Discussion</h2><div id="sec-19" class="subsection"><h3>Profiling of mRNAs, metabolites, and lipids in <em>S. cerevisiae</em></h3><p id="p-26">To globally map networks that control lipid metabolism in yeast, we applied a systems approach integrating measurements across multiple cellular component layers (mRNAs, lipids, metabolites) and data from protein interaction databases with metabolic network topology (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). We pursued a full factorial design in which we varied different environmental conditions known to impact lipid metabolism (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>). Specifically, we varied the following three factors: carbon availability, known to impact fatty acid and phospholipid biosynthesis (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Tehlivets <em>et al.</em> 2007</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Carman and Han 2011</a>); oxygen availability, known to impact sterol biosynthesis (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Espenshade and Hughes 2007</a>); and temperature, known to affect membrane composition (<em>e.g.</em>, fatty acid desaturation) and sphingolipid metabolism (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Hunter and Rose 1972</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Aguilar and De Mendoza 2006</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Cowart and Obeid 2007</a>). Each factor comprised two levels as follows: carbon source [carbon-limited (C); nitrogen-limited N)], oxygen availability [aerobic (O); anaerobic (A)]; and temperature [30° (T); 15° (t)]. Wild-type <em>S. cerevisiae</em> CEN.PK113-7D was grown in chemostat cultivations at a fixed dilution rate <em>D</em> = 0.050 hr<sup>−1</sup> to avoid confounding affects associated with the dynamic nature of batch cultures or attributable to different specific growth rates under these conditions (<a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Regenberg <em>et al.</em> 2006</a>).</p><p id="p-27">From each chemostat cultivation, we measured mRNA, metabolite, and lipid levels (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>). We were able to detect and quantify a total of 5636 mRNAs, 50 metabolites, and 97 lipid species. The large number of measured lipids arises from having data regarding acyl-chain composition for different lipid species (<em>i.e.</em>, the fatty acid composition of phospholipid species; C12:0, C16:1, and others). Detailed physiological data, carbon balances, and complete datasets are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/TableS2.xlsx">Table S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/TableS3.xls">Table S3</a>.</p><p id="p-28">Despite similar specific growth rates, marked changes were observed at the molecular level. For example, our lipid measurements confirmed that lipid class levels vary based on the three types of environmental conditions evaluated (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS5.pdf">Figure S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS6.pdf">Figure S6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS7.pdf">Figure S7</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS8.pdf">Figure S8</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS9.pdf">Figure S9</a>). We used a multi-factor ANOVA to identify significant changes at the mRNA, metabolite, and lipid levels (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/TableS4.xlsx">Table S4</a>). In this way, the main experimental factors tested (<em>i.e.</em>, C:N, O:A, T:t) and, in addition, all the possible interaction factors were tested (<em>i.e.</em>, CN:OA, CN:Tt, OA:Tt, and CN:OA:Tt) to identify mRNAs, metabolites, and lipids significantly influenced by combinations of growth conditions, highlighting the power of the full factorial design. At a threshold of <em>P</em> ≤ 0.001 after Bonferroni correction (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bonferroni 1936</a>), we observed 1855 differentially expressed genes by one or more of the tested design factors (including interaction factors), thus showing that one-third of the 5636 measured genes were significantly affected by one or more of the investigated factors. Interestingly, temperature and carbon availability induced a comparably large number of gene expression changes (951 and 910, respectively, at <em>P</em> ≤ 0.001). By comparison, aerobicity appeared to influence expression of approximately half as many genes (516 at <em>P</em> ≤ 0.001). By contrast, the lipidome data were most influenced by the presence or absence of oxygen. A majority of the 97 lipids measured were significantly changing due to at least one of the tested factors (78 at P \x{2264} 0.001). Aerobicity accounted for changes in two-thirds (65) of the lipids tested. C:N and T:t accounted for 47 and 30 lipid changes, respectively. Two-thirds (33) of the 50 metabolites measured were changing because of at least one factor. C:N and O:A each affected approximately half (24 and 21, respectively), whereas T:t only influenced approximately one-third (17) of the 50 metabolites measured. The overlaps among significant genes, lipids, and metabolites are highlighted in the <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/006601SI.pdf">Supporting Information</a> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS10.pdf">Figure S10</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/TableS4.xlsx">Table S4</a>).</p><p id="p-29">Principle components analysis (PCA) was used to provide a nonbiased means of assessing the influence of growth factors (C:N, O:A, and T:t) on mRNA, metabolite, and lipid levels (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Because this analysis considered all measured quantities of each type, it provided a more general overview than the multi-factor ANOVA analysis. Strikingly, when the first three components of our transcriptome data were plotted in three-dimensions, the six faces of a cube representing our three-factor experimental design were observed (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a> and <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). Together, the first three components accounted for &gt;75% of the variation in the mRNA expression data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/TableS5.pdf">Table S5</a>). The cube of factor combinations was tilted and rotated in the first three PCA dimensions, indicating that each of these dimensions was influenced by all three factors but to differing extents. The first principle component (37% of mRNA variance) was primarily attributable to C:N factor; the second dimension (24% of mRNA variance) was an approximately equal balance of all experimental factors, whereas the third dimension (15% of mRNA variance) was primarily attributable to aerobicity (O:A). This indicates that mRNA levels were most influenced by C-limitation and N-limitation, which makes sense when we consider that few cellular process are independent of carbon and nitrogen utilization. The PCA analysis of the metabolite data attributed &gt;80% variance to the first two PCA dimensions (63% to PC 1), with C:N as the primary source of variability. Metabolite PC 1 showed similarity to mRNA PC 2 and, vice versa, PC 2 showed similarity to mRNA PC 1. By contrast, lipidome data displayed the greatest variance along the aerobic–anaerobic split (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2, A–C</a>). Because lipid levels are controlled by a subset of metabolic processes and enzymes, they do not necessarily vary on aggregate as metabolites and mRNAs do. Lipids were separated in the PC 1 between aerobic and N-limited <em>vs.</em> the rest and in PC 2 by aerobic and C-limited <em>vs.</em> the rest. Because the first two PCA dimensions accounted for ∼80% of the variance in the lipid data, we can conclude that aerobicity had the largest influence on the lipid levels, whereas C:N was secondary. A hierarchical clustering of these datasets showed the same trends arising from the environmental conditions (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2, D–F</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Molecular signatures of mRNAs, lipids, and metabolites for each experimental condition: C-limited, C; N-limited, N; aerobic, O; anaerobic, A; 30&#xB0;, T; and 15&#xB0;, t (COT, gray; COt, cyan; CAT, purple; CAt, blue; NOT, yellow; NOt, green; NAT, red; NAt, black). (A&#x2013;C) Principal components analysis (PCA) with projections for PC 1, PC 2, and PC 3 for all mRNA, lipid, and metabolite levels, respectively. The size of the point represents location in the PC 3 axis, which is perpendicular to the page. (D&#x2013;F) Hierarchical clustering and resulting heat maps for mRNAs, lipids, and metabolites found to be significant by ANOVA. Measurement z-scores for all data types were mapped as defined by the color bar (note that mRNA z-scores were based on GCRMA values which are log2). For gene, lipid, and metabolite name abbreviations, see Table S6." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1964801122" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Molecular signatures of mRNAs, lipids, and metabolites for each experimental condition: C-limited, C; N-limited, N; aerobic, O; anaerobic, A; 30&#xB0;, T; and 15&#xB0;, t (COT, gray; COt, cyan; CAT, purple; CAt, blue; NOT, yellow; NOt, green; NAT, red; NAt, black). (A&#x2013;C) Principal components analysis (PCA) with projections for PC 1, PC 2, and PC 3 for all mRNA, lipid, and metabolite levels, respectively. The size of the point represents location in the PC 3 axis, which is perpendicular to the page. (D&#x2013;F) Hierarchical clustering and resulting heat maps for mRNAs, lipids, and metabolites found to be significant by ANOVA. Measurement &lt;em&gt;z&lt;/em&gt;-scores for all data types were mapped as defined by the color bar (note that mRNA &lt;em&gt;z&lt;/em&gt;-scores were based on GCRMA values which are log&lt;sub&gt;2&lt;/sub&gt;). For gene, lipid, and metabolite name abbreviations, see Table S6.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1979/F2.medium.gif" width="353" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/11/1979/F2.medium.gif" width="353" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463604" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-30" class="first-child">Molecular signatures of mRNAs, lipids, and metabolites for each experimental condition: C-limited, C; N-limited, N; aerobic, O; anaerobic, A; 30°, T; and 15°, t (COT, gray; COt, cyan; CAT, purple; CAt, blue; NOT, yellow; NOt, green; NAT, red; NAt, black). (A–C) Principal components analysis (PCA) with projections for PC 1, PC 2, and PC 3 for all mRNA, lipid, and metabolite levels, respectively. The size of the point represents location in the PC 3 axis, which is perpendicular to the page. (D–F) Hierarchical clustering and resulting heat maps for mRNAs, lipids, and metabolites found to be significant by ANOVA. Measurement <em>z</em>-scores for all data types were mapped as defined by the color bar (note that mRNA <em>z</em>-scores were based on GCRMA values which are log<sub>2</sub>). For gene, lipid, and metabolite name abbreviations, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/TableS6.xlsx">Table S6</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h3>Conditional dependence of molecular players based on environmental perturbation</h3><p id="p-31">To determine the conditional dependence of mRNA, metabolite, and lipid data, we first constructed a visual map based on the genome-scale metabolic model, <em>iIN800</em> (<a id="xref-ref-50-4" class="xref-bibr" href="#ref-50">Nookaew <em>et al.</em> 2008</a>). Because we lacked measurements for many of the metabolites in <em>iIN800</em>, we centered our biosynthetic network on central metabolism, the tricarboxylic acid (TCA) cycle, amino acid biosynthesis, sterol biosynthesis, steryl ester biosynthesis, triacylglyceride (TAG), biosynthesis, FFA biosynthesis, phospholipid biosynthesis, and sphingolipid biosynthesis. In total, our visual network map consisted of 368 gene, 174 metabolite/lipid, and 131 reaction nodes connected by 849 gene (enzyme)-reaction and metabolite/lipid-reaction edges (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/TableS6.xlsx">Table S6</a>). Nine distribution nodes were also included for mapping information on lipid acyl-chain composition. A distribution node is connected to all of the acyl-chain species involving that particular lipid or fatty acid type. For the reaction nodes, we performed flux balance analysis to estimate values of reaction fluxes <em>in silico</em> under the constraints of maximized biomass production, a steady-state metabolic network, and fixed protein composition (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/TableS7.xlsx">Table S7</a>) (<a id="xref-ref-50-5" class="xref-bibr" href="#ref-50">Nookaew <em>et al.</em> 2008</a>). Our network model linking all measurement types was visualized in Cytoscape (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Cline <em>et al.</em> 2007</a>) as shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> [with color mapping for measurement log-fold-changes, such as aerobic (O) <em>vs.</em> anaerobic (A) conditions]. Although the interplay between different metabolic pathways could be missed in our visualization approach, we have previously shown that such a model provides a useful framework for exploring the connectivity of mRNAs, metabolites, lipids, and fluxes (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Moxley <em>et al.</em> 2009</a>). Additionally, subsequent integrative analyses we performed for this work were not solely based on this visualization and contain the entire metabolic network (see <em>Data integration through hypothesis testing across multiple cellular levels</em> section).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Metabolic map for interrogating the connectivity of mRNAs, metabolites, lipids, and reaction fluxes based on the genome-scale model iIN800. This network contains gene (enzyme)-reaction, metabolite/lipid-reaction, and lipid distribution edges (see node type and edge key). Measurement ratios can be visualized with log2-fold changes (see node color key; here, shown for aerobic vs. anaerobic conditions) and gray indicates the lack of a measurement for that node. Individual pathway networks (e.g., phospholipid biosynthesis, sterol biosynthesis, and amino acid biosynthesis) are highlighted. Findings for individual pathways are discussed in the text." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1964801122" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Metabolic map for interrogating the connectivity of mRNAs, metabolites, lipids, and reaction fluxes based on the genome-scale model &lt;em&gt;iIN800&lt;/em&gt;. This network contains gene (enzyme)-reaction, metabolite/lipid-reaction, and lipid distribution edges (see node type and edge key). Measurement ratios can be visualized with log&lt;sub&gt;2&lt;/sub&gt;-fold changes (see node color key; here, shown for aerobic &lt;em&gt;vs.&lt;/em&gt; anaerobic conditions) and gray indicates the lack of a measurement for that node. Individual pathway networks (&lt;em&gt;e.g.&lt;/em&gt;, phospholipid biosynthesis, sterol biosynthesis, and amino acid biosynthesis) are highlighted. Findings for individual pathways are discussed in the text.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1979/F3.medium.gif" width="440" height="323"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/11/1979/F3.medium.gif" width="440" height="323"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463587" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Metabolic map for interrogating the connectivity of mRNAs, metabolites, lipids, and reaction fluxes based on the genome-scale model <em>iIN800</em>. This network contains gene (enzyme)-reaction, metabolite/lipid-reaction, and lipid distribution edges (see node type and edge key). Measurement ratios can be visualized with log<sub>2</sub>-fold changes (see node color key; here, shown for aerobic <em>vs.</em> anaerobic conditions) and gray indicates the lack of a measurement for that node. Individual pathway networks (<em>e.g.</em>, phospholipid biosynthesis, sterol biosynthesis, and amino acid biosynthesis) are highlighted. Findings for individual pathways are discussed in the text.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Visualization in Cytoscape enabled the targeted and rapid identification of changes across multiple levels of the cellular hierarchy (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS11.pdf">Figure S11</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS12.pdf">Figure S12</a>). These networks provided a direct route for querying cellular changes one at a time and validated the high quality of our data (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS13.pdf">Figure S13</a>). For example, we observed the downregulation of acetyl-CoA carboxylase (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005299">ACC1</a></em>), which catalyzes the carboxylation of acetyl-CoA to form malonyl-CoA as the first committed step of <em>de novo</em> fatty acid biosynthesis when comparing growth at 30° to 15° (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS13.pdf">Figure S13</a>E). This was positively correlated to a reduction in levels of pyruvate, alpha-ketoglutarate, malate, and citrate (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS13.pdf">Figure S13</a>G). Citrate is known to allosterically activate <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005299">Acc1p</a> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Martin and Vagelos 1962</a>).</p><p id="p-34">Some of the most dramatic changes were observed for the storage depots of the cell, TAG and steryl esters (SE) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS2.pdf">Figure S2</a>). Under nitrogen-limited (<em>i.e.</em>, carbon excess) and aerobic conditions, levels of TAG increased from 0.7 ± 0.2 μmol/g DCW to 3.8 ± 0.2 μmol/g DCW, and levels of SE increased from 0.8 ± 0.6 μmol/g DCW to 4.9 ± 0.4 μmol/g DCW. When compared to all other conditions (N-limited aerobic <em>vs.</em> others), only 26 genes, 11 lipids, and 3 metabolites were significantly changed (<em>P</em> ≤ 0.01 following Bonferroni correction) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/TableS8.pdf">Table S8</a>). Within this small subset, it is striking that none of the biosynthetic genes encoding enzymes that directly produce diacylglycerol (DAG), TAG, ergosterol, or SEs is significantly upregulated. Rather, we observed a confluence of changes that contribute to the observed phenotype, including high levels of pyruvate and alpha-ketoglutarate (key signaling molecules indicating high carbon levels), more reducing power (flux through the pentose phosphate pathway increases), downregulation of regulatory proteins (<em>e.g.</em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005461">IZH4</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000505">DAN3</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005763">HES1</a></em>) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Carman and Han 2007</a>), and a lower flux toward phosphatidic acid (the precursor to <em>de novo</em> phospholipids synthesis) (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Carman and Han 2011</a>). Consistent with the apparent shift from production of phospholipids to the production of storage compounds, the ratio of nonpolar lipids (sterols, TAG, SE, and FFAs) to phospholipids was higher under nitrogen-limited aerobic conditions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS14.pdf">Figure S14</a>). We also observed an inverse regulatory relationship between the expression of <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004467">ERG6</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001114">ERG7</a></em> and TAG and SE levels (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004467">Erg6p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001114">Erg7p</a> are involved in sterol biosynthesis and are found almost exclusively in lipid particles (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Schulz and Prinz 2007</a>) interacting with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000644">Are1p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005302">Are2p</a>, respectively. Because <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000644">Are1p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005302">Are2p</a> catalyze SE biosynthesis, our data are consistent with the hypothesis that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004467">Erg6p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001114">Erg7p</a> may play a regulatory role in SE synthesis (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Czabany <em>et al.</em> 2007</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Teske <em>et al.</em> 2008</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Condition-dependent response of lipid metabolism. (A) The impact of nitrogen-limited aerobic conditions (NOx = NOT and NOt) vs. all others on the storage depots triacylglycerol (TAG) and sterylesters (SE). TAG and SE significantly accumulate under NOx conditions and are inversely correlated to ERG7 and ERG6. (B) The impact of aerobicity on phospholipid metabolism. We observed significant increases in phosphotidylethanolamine (PE) and phosphotidylcholine (PC) and in phosphatidylinositol (PINS) species under anaerobic growth. Uniquely, the presence of di-substituted medium acyl-chain fatty acids, which we denote with an &#x201C;S&#x201D; (e.g., PCS or PINS), under anaerobic growth suggested redox constraints. Measurement ratios were visualized with a log2 color bar (see node color key). The relative node size, thickness of each node border, and color of each node border represents the log10(P value). See node key. Gray indicates the lack of a measurement for that node." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1964801122" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Condition-dependent response of lipid metabolism. (A) The impact of nitrogen-limited aerobic conditions (NOx = NOT and NOt) &lt;em&gt;vs.&lt;/em&gt; all others on the storage depots triacylglycerol (TAG) and sterylesters (SE). TAG and SE significantly accumulate under NOx conditions and are inversely correlated to &lt;em&gt;ERG7&lt;/em&gt; and &lt;em&gt;ERG6&lt;/em&gt;. (B) The impact of aerobicity on phospholipid metabolism. We observed significant increases in phosphotidylethanolamine (PE) and phosphotidylcholine (PC) and in phosphatidylinositol (PINS) species under anaerobic growth. Uniquely, the presence of di-substituted medium acyl-chain fatty acids, which we denote with an &#x201C;S&#x201D; (&lt;em&gt;e.g.&lt;/em&gt;, PCS or PINS), under anaerobic growth suggested redox constraints. Measurement ratios were visualized with a log&lt;sub&gt;2&lt;/sub&gt; color bar (see node color key). The relative node size, thickness of each node border, and color of each node border represents the log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;P&lt;/em&gt; value). See node key. Gray indicates the lack of a measurement for that node.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1979/F4.medium.gif" width="381" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/11/1979/F4.medium.gif" width="381" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463542" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">Condition-dependent response of lipid metabolism. (A) The impact of nitrogen-limited aerobic conditions (NOx = NOT and NOt) <em>vs.</em> all others on the storage depots triacylglycerol (TAG) and sterylesters (SE). TAG and SE significantly accumulate under NOx conditions and are inversely correlated to <em>ERG7</em> and <em>ERG6</em>. (B) The impact of aerobicity on phospholipid metabolism. We observed significant increases in phosphotidylethanolamine (PE) and phosphotidylcholine (PC) and in phosphatidylinositol (PINS) species under anaerobic growth. Uniquely, the presence of di-substituted medium acyl-chain fatty acids, which we denote with an “S” (<em>e.g.</em>, PCS or PINS), under anaerobic growth suggested redox constraints. Measurement ratios were visualized with a log<sub>2</sub> color bar (see node color key). The relative node size, thickness of each node border, and color of each node border represents the log<sub>10</sub>(<em>P</em> value). See node key. Gray indicates the lack of a measurement for that node.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">Our visual metabolic model also highlighted changes in phospholipid metabolism. Phospholipid levels varied significantly with temperature and oxygen availability. Globally, this trend is a result of increased phosphotidylinositol (PINS) levels, as well as increased PC and PE levels at lower temperatures under anaerobic conditions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS3.pdf">Figure S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS4.pdf">Figure S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS5.pdf">Figure S5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS7.pdf">Figure S7</a>). Uniquely, the distribution nodes of our model enabled mapping of quantitative acyl-chain composition information (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). After a shift from aerobic to anaerobic growth, we observed the production of a population of phospholipids with di-substituted medium acyl-chain (C10:0, C12:0) fatty acids (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS3.pdf">Figure S3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS4.pdf">Figure S4</a>). To our knowledge, this fatty acid species has not previously been reported as a major component inside the cell. The presence of di-substituted medium acyl-chain fatty acids, which we denote with an “S” (<em>e.g.</em>, PCS), under anaerobic growth suggested redox constraints. Because the fatty acid synthase uses two reduced NADPH for each 2C-atom added to a growing fatty acid, we hypothesized that a decrease in reducing power would hamper fatty acid elongation and cause a shift toward production of medium-chain fatty acids. The flux through the pentose phosphate pathway (PPP) is relatively low under anaerobic conditions, causing cells to be redox-constrained. Previously, for example, it was shown that NADPH is balanced under fermentative growth conditions, whereas excess NADPH is produced under aerobic conditions (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Gombert <em>et al.</em> 2001</a>). In addition, our <em>in silico</em> flux balance analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/TableS7.xlsx">Table S7</a>) revealed a 10-fold average decrease in the PPP flux when comparing anaerobic to aerobic conditions. To experimentally validate this hypothesis, we performed follow-up <sup>13</sup>C-flux analysis experiments. To experimentally determine fluxes, we cultivated cells on 1-<sup>13</sup>C-glucose and experimentally measured isotopic labeling patterns with GC-MS. We observed that measured flux through the PPP is reduced 3.5-fold in anaerobic conditions (<em>P</em> ≤ 0.01) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS15.pdf">Figure S15</a>), which is consistent with our <em>in silico</em> fluxes and the previous work of <a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Gombert <em>et al.</em> (2001)</a>.</p><p id="p-37">To reveal how the transcriptional network was reprogrammed across different environmental conditions, we scored the significance of overlap between condition-dependent genes, <em>i.e.</em>, genes found significant for each tested experimental factor by multi-factor ANOVA, and known transcription factor (TF) target sets (hypergeometric test at <em>P</em> ≤ 0.01) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Harbison <em>et al.</em> 2004</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Beyer <em>et al.</em> 2006</a>; <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Fazio <em>et al.</em> 2008</a>). In total, this analysis revealed 13 TFs having significantly enriched target sets for four of the six factors (C, O, T, C:O) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS16.pdf">Figure S16</a>). For example, 14 of 17 known <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002621">Upc2p</a> regulatory targets (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hodges <em>et al.</em> 1998</a>) were found to have significantly lower expression in aerobic conditions relative to anaerobic conditions. Although these TFs appear to be responsible for regulating some of the differentially expressed genes, the relative number of differentially expressed genes targeted was less than 10% of the affected genes in all cases.</p></div><div id="sec-21" class="subsection"><h3>Data integration through hypothesis testing across multiple cellular levels</h3><p id="p-38">Beyond interrogating data types one component at a time, we sought to address the pluralism of causes and effects in biological networks by developing integration strategies for observing regulatory signatures. This is one of the grand challenges of 21<sup>st</sup> century systems biology (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Palsson and Zengler 2010</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bebek <em>et al.</em> 2011</a>). To identify the most highly correlated and anti-correlated relationships between gene expression level and metabolites, gene expression level and lipids, and gene expression level and lipids as characterized by acyl-chain (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>), we tested whether the Pearson correlation coefficient, as estimated over all triplicates in the eight conditions (24 measurements), was significantly not zero (<em>P</em> ≤ 0.001 after Bonferroni correction). Although experimentally determined or computed fluxes were not directly integrated into this analysis, fluxes are indirectly considered because the lipid levels in the biomass were directly correlated to the fluxes when we used chemostat cultures and lipid levels, which are given per unit biomass. Our analysis revealed 831 significant correlations among gene expression levels and metabolites, 245 significant correlations among gene expression levels and lipids, and 1203 significant interactions among gene expression levels and lipid species characterized by length. Relationships were then separated into positive and negative correlations and filtered based on ANOVA significance main effects (<em>e.g.</em>, C-limited <em>vs.</em> N-limited) for visualization in Cytoscape. Gene expression level-metabolite and gene expression level–lipid interactions were combined into a single network, leaving gene expression level–lipid by acyl-chain composition correlations separate. The organization of the network contained mainly single-connection, chain, and bi-fan motifs (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS17.pdf">Figure S17</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS18.pdf">Figure S18</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS19.pdf">Figure S19</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS20.pdf">Figure S20</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS21.pdf">Figure S21</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS22.pdf">Figure S22</a>). There were 2279 significant correlations.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Correlation analysis enabled data integration across multiple levels of the cellular hierarchy. Highly correlated and anti-correlated relationships between gene expression levels&#x2013;metabolites, gene expression levels&#x2013;lipids, and gene expression levels&#x2013;lipids as characterized by acyl-chain length are shown. (A) Correlations with ergosterol (ERGOST) under aerobic (&#x201C;O&#x201D;) and anaerobic (&#x201C;A&#x201D;) conditions. (B) Correlations with SE and TAG under C-limited (&#x201C;C&#x201D;) vs. N-limited (&#x201C;N&#x201D;) conditions and aerobic vs. anaerobic conditions reveal condition-independent conditions (e.g., IZH1, ERG7, and HXK1). (C) Positive correlations with phosphatidylinositol (PINS) under aerobic vs. anaerobic conditions. (D) Correlations between gene expression levels and lipids as characterized by length under aerobic vs. anaerobic conditions. Measurement ratios were visualized with a log2 color bar, and the color of each node border represents the log10(P value). See node and edge color key. Gray indicates the lack of a measurement for that node. Complete networks are found in Figure S17, Figure S18, Figure S19, Figure S20, Figure S21, and Figure S22." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1964801122" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Correlation analysis enabled data integration across multiple levels of the cellular hierarchy. Highly correlated and anti-correlated relationships between gene expression levels&#x2013;metabolites, gene expression levels&#x2013;lipids, and gene expression levels&#x2013;lipids as characterized by acyl-chain length are shown. (A) Correlations with ergosterol (ERGOST) under aerobic (&#x201C;O&#x201D;) and anaerobic (&#x201C;A&#x201D;) conditions. (B) Correlations with SE and TAG under C-limited (&#x201C;C&#x201D;) &lt;em&gt;vs.&lt;/em&gt; N-limited (&#x201C;N&#x201D;) conditions and aerobic &lt;em&gt;vs.&lt;/em&gt; anaerobic conditions reveal condition-independent conditions (&lt;em&gt;e.g.&lt;/em&gt;, &lt;em&gt;IZH1&lt;/em&gt;, &lt;em&gt;ERG7&lt;/em&gt;, and &lt;em&gt;HXK1&lt;/em&gt;). (C) Positive correlations with phosphatidylinositol (PINS) under aerobic &lt;em&gt;vs.&lt;/em&gt; anaerobic conditions. (D) Correlations between gene expression levels and lipids as characterized by length under aerobic &lt;em&gt;vs.&lt;/em&gt; anaerobic conditions. Measurement ratios were visualized with a log&lt;sub&gt;2&lt;/sub&gt; color bar, and the color of each node border represents the log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;P&lt;/em&gt; value). See node and edge color key. Gray indicates the lack of a measurement for that node. Complete networks are found in Figure S17, Figure S18, Figure S19, Figure S20, Figure S21, and Figure S22.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1979/F5.medium.gif" width="359" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/11/1979/F5.medium.gif" width="359" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463597" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-39" class="first-child">Correlation analysis enabled data integration across multiple levels of the cellular hierarchy. Highly correlated and anti-correlated relationships between gene expression levels–metabolites, gene expression levels–lipids, and gene expression levels–lipids as characterized by acyl-chain length are shown. (A) Correlations with ergosterol (ERGOST) under aerobic (“O”) and anaerobic (“A”) conditions. (B) Correlations with SE and TAG under C-limited (“C”) <em>vs.</em> N-limited (“N”) conditions and aerobic <em>vs.</em> anaerobic conditions reveal condition-independent conditions (<em>e.g.</em>, <em>IZH1</em>, <em>ERG7</em>, and <em>HXK1</em>). (C) Positive correlations with phosphatidylinositol (PINS) under aerobic <em>vs.</em> anaerobic conditions. (D) Correlations between gene expression levels and lipids as characterized by length under aerobic <em>vs.</em> anaerobic conditions. Measurement ratios were visualized with a log<sub>2</sub> color bar, and the color of each node border represents the log<sub>10</sub>(<em>P</em> value). See node and edge color key. Gray indicates the lack of a measurement for that node. Complete networks are found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS17.pdf">Figure S17</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS18.pdf">Figure S18</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS19.pdf">Figure S19</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS20.pdf">Figure S20</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS21.pdf">Figure S21</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS22.pdf">Figure S22</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">Because of our full factorial design, these data offer a rich resource for exploring correlations across mRNAs, lipids, and metabolites influenced by broad perturbations that were not possible before (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Although significance of correlation was assessed across all eight conditions, in some examples, the correlation was evident from changes attributable to a single factor. For example, when comparing aerobic to anaerobic conditions, ergosterol levels are correlated to nine mRNA levels and anti-correlated to 23 mRNA levels (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5A</a>). Identification of gene expression levels that have been previously implicated in controlling sterol levels validated our approach. <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003407">Erg1p</a>, for example, encodes for squalene synthase, an enzyme in the ergosterol biosynthesis pathway that requires oxygen (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lees <em>et al.</em> 1999</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Espenshade and Hughes 2007</a>). <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005537">Aus1p</a> is a sterol transporter required for growth under anaerobic conditions (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Lees <em>et al.</em> 1999</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Wilcox <em>et al.</em> 2002</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Espenshade and Hughes 2007</a>). As expected, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005537">Aus1p</a> is anti-correlated to ergosterol levels when comparing aerobic and anaerobic conditions. In other cases, we observed more than one factor strongly influencing the correlation. For example, nine gene expression level–lipid correlations for TAG and SE were observed in both O:A and C:N conditions (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5B</a>), suggesting strong regulatory control. Genes linked to TAG and SE included <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001949">HXK1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005461">IZH4</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002900">IZH1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001799">SRL3</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001114">ERG7</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000271">TIP1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000796">DSF1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003281">SCM4</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002924">EMI2</a></em>. As discussed, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001114">Erg7p</a> is known to interact with <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000644">Are1p</a>, which contributes the major sterol esterification activity under anaerobic conditions (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Czabany <em>et al.</em> 2007</a>). The family of Izh proteins is involved in zinc ion homeostasis, which regulates phospholipid synthesis (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Carman and Han 2007</a>).</p><p id="p-41">Correlation between mRNA and metabolite levels may depend on more than one factor, and potentially in differing ways. For example, phosphoenolpyruvate (PEP) levels were observed to increase when <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004714"><em>ILV2</em></a> or <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002442"><em>ARO3</em></a> levels increased in N-limiting conditions, but PEP was negatively correlated to these genes in C-limiting conditions (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Bradley <em>et al.</em> 2009</a>). To check for such condition-dependent and opposing correlations, we performed analyses to test for gene expression level–metabolite or gene expression level–lipid pairs that were, for example, significantly positively correlated with C-limited trials and significantly negatively correlated with N-limited trials, or vice versa. This was achieved by splitting the data in half by each of the three factors, testing both halves of each split separately and comparing the results. No gene expression level–metabolite or gene expression level–lipid pairs were observed to have significant and opposite correlations in the data set halves, although some pairs with agreeing correlations were observed.</p><p id="p-42">Our correlation analysis enabled exploration of two key features that would not have otherwise been observed by mapping data onto our visual metabolic model alone (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>). First, metabolites and lipids were linked in ways not found in traditional metabolic maps. In other words, metabolites and lipids not sharing the same metabolic enzymes were correlated. For example, extracellular ethanol levels were linked to phosphatidylinositol through by <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005362">IZH2</a></em> and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005537">AUS1</a></em> (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C</a>) and phospholipid species with di-substituted medium acyl-chain fatty acids were connected to genes known to impact redox metabolism (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5D</a>), an observation consistent with our <sup>13</sup>C-flux analysis. Second, the significance of metabolic genes that are not present in the metabolic model (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> and <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>) could be identified. Although biosynthetic metabolic genes in the correlation networks were slightly enriched relative to their percentage in yeast (19% <em>vs.</em> 13%), only 5 of 2279 correlations were between first neighbors and direct substrates or products (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/TableS9.pdf">Table S9</a>). At a quantitative systems level, this suggests that biosynthetic enzymes that either use metabolites and/or lipids as reactants or catalyze their formation were not directly connected. In other words, transcriptional regulation is not tightly controlled at the one-step level.</p></div><div id="sec-22" class="subsection"><h3>Integrative methods for correlation of omics data</h3><p id="p-43">Because lipids/metabolites and gene expression levels were not co-regulated at the one-step level, we next sought to refine the correlation analysis using the following three parallel and independent approaches: co-regulated gene neighborhoods (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1D</a>); co-regulated pathway neighborhoods (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1E</a>); and TF-regulated modules (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1F</a>). Identification of such neighborhoods and modules would lead to a better understanding of the underlying organization of the cellular response. For co-regulated gene neighborhoods, we created a topology bi-partite map, whereby metabolites were connected to genes that encode for proteins that catalyze reactions that include a metabolite as a reactant or product. The metabolic network consisted of 945 metabolites and 715 protein-coding genes in which a median of two genes were connected to each metabolite and a median of four metabolites were connected to each gene. We then determined if gene neighbors for each metabolite/lipid had a bias to be significantly correlated (<em>r</em> &gt; 0) or anti-correlated (<em>r</em> &lt; 0).</p><p id="p-44">We initially looked at first gene neighbors within the metabolic network (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1D</a>). Using this approach, a significant bias would suggest that metabolite production/consumption is driven by differential expression of upstream or downstream enzymes in the metabolic network. First gene neighbors did not show any bias relative to that expected of a random sample (Mann-Whitney <em>U</em>-test). However, 22 metabolites and lipid species had a significant bias when considering both first and second neighbors (<em>P</em> ≤ 0.01, Benjamini Hochberg <em>P</em> value adjustment) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), with ergosterol being the most highly correlated. Our observations provide new systems-level insight, suggesting that sterols, and perhaps phosphatidylinositol, are more regulated across the range of different growth conditions tested here than amino acids at the transcriptional level (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/TableS10.pdf">Table S10</a>). Given that amino acids are chiefly directed into proteins, this suggests that the cell has developed more complex methods to regulate proteins post-transcriptionally than sterols and the entry point into phospholipid biosynthesis. Furthermore, biosynthesis of amino acids involves far more branched pathways than the biosynthesis of lipids; to ensure a balanced supply of the 20 different amino acids required for protein biosynthesis, cells have evolved complex regulation of flux at the enzyme level. This finding is consistent with our recent work (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Moxley <em>et al.</em> 2009</a>) demonstrating that amino acid biosynthesis is primarily metabolically controlled.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/463548/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/463548/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Metabolites and lipids whose levels are most significantly driven by expression of neighboring genes in a bi-partite genome-scale metabolic map</span><div class="sb-div caption-clear"></div></div></div><p id="p-46">Another way to refine the correlation analysis is to look for transcriptional regulation across genes in pathways (<a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1E</a>), rather than first-order or second-order gene neighbors (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1D</a>). We therefore tested to see if gene neighbors for sets of metabolites in a KEGG pathway, regardless of reaction direction (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kanehisa <em>et al.</em> 2012</a>), have a bias to be significantly correlated or anti-correlated. This approach, therefore, is testing which pathways contain metabolites that tend to correlate with all transcripts in the pathway, as a group, rather than our gene neighbor analysis, which reveals specific metabolites that tend to correlate with their neighboring genes. Although KEGG pathways could be isolated from global metabolic networks that were captured in our gene neighborhood analysis because of choices of database curators, the results were consistent with our co-regulated gene neighborhood analysis (which is unbiased). For example, we observed that the sterol biosynthesis KEGG pathway was significantly co-regulated (<em>P</em> ≤ 0.01, Benjamini Hochberg <em>P</em> value adjustment) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/TableS11.pdf">Table S11</a>), validating the utility of using a systems perspective to integrate omics information.</p><p id="p-47">In an effort to identify factors controlling co-regulated correlation networks, smaller sets of significantly changing genes identified as significantly correlated to specific metabolites or lipids were analyzed for enrichment of TF targets (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1F</a>). This analysis identified 16 TFs significantly associated with four metabolites and 15 different lipid species through the gene sets levels to which they were significantly correlated. Positively and negatively correlated gene expression level–lipid and gene expression level–metabolite sets were analyzed separately. <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows the significant (hypergeometric <em>P</em> ≤ 0.01) TF gene expression level–lipid and TF gene expression level–metabolite enrichments capture a global regulatory model of lipid metabolism. The Hap family of TFs and, particularly, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001592">Hap4p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004246">Hap1p</a> were found associated with genes negatively correlated to a number of lipid levels. Interestingly, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002621">Upc2p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000006269">Rox1p</a> targets were enriched for genes positively correlated to PINS, PINSS, PC, and PCS, and were negatively correlated to ergosterol. These regulators are known players in sterol regulation (<a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Espenshade and Hughes 2007</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Nielsen 2009</a>).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Transcription factor associations with genes correlated to lipid or metabolite levels. The Fisher exact test was used to identify significant overlaps between transcription factor target sets and genes significantly correlated (blue arrows) or anti-correlated (red arrows) to the implicated lipid or metabolite. The relative thickness of each arrow represents the &#x2212;log10(P value) from the Fisher exact test." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1964801122" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Transcription factor associations with genes correlated to lipid or metabolite levels. The Fisher exact test was used to identify significant overlaps between transcription factor target sets and genes significantly correlated (blue arrows) or anti-correlated (red arrows) to the implicated lipid or metabolite. The relative thickness of each arrow represents the &#x2212;log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;P&lt;/em&gt; value) from the Fisher exact test.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1979/F6.medium.gif" width="440" height="241"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/3/11/1979/F6.medium.gif" width="440" height="241"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463598" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-48" class="first-child">Transcription factor associations with genes correlated to lipid or metabolite levels. The Fisher exact test was used to identify significant overlaps between transcription factor target sets and genes significantly correlated (blue arrows) or anti-correlated (red arrows) to the implicated lipid or metabolite. The relative thickness of each arrow represents the −log<sub>10</sub>(<em>P</em> value) from the Fisher exact test.</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">Finally, we built correlation networks for ergosterol and phosphatidylinositol based on the overlap between first and second gene neighbors from the co-regulated gene neighborhoods and the genome-scale metabolic map (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). The key idea was to identify areas of metabolism that are closely connected with significant and coordinated response to genetic or environmental perturbations. For ergosterol, we identified 76 significant gene expression level–ergosterol interactions in the correlation analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS23.pdf">Figure S23</a>) and mapped 102 genes linked to ergosterol in the genome scale metabolic model. By taking the intersection of these two groups, we identified nine metabolites that were both linked as a first and second gene neighbors to ergosterol in the metabolic map and were significantly correlated (<em>e.g.</em>, ergosterol → <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000644">ARE1</a></em> → acylCoA → <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005844">FAA1</a></em>). This topological map was then integrated with the TF-regulated modules implicated in the enrichment analysis to provide a global regulatory picture (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7A</a>). One TF, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002621">Upc2p</a>, is known to activate the expression of sterol biosynthetic genes in sterol-depleted cells (<a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Espenshade and Hughes 2007</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gaspar <em>et al.</em> 2007</a>) and could be directly linked to <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004815">Erg2p</a> and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005537">Aus1p</a>. The identification of <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002621">Upc2p</a> as significant when comparing aerobic <em>vs.</em> anaerobic conditions validates our undirected approach. Notably, our results suggest that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002621">Upc2p</a> functions by controlling the enzyme levels at the transcriptional level and that this results in altered fluxes toward ergosterol. This points to <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002621">Upc2p</a> playing a similar function as the SREBP-1 transcription factor in mammals (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Nielsen 2009</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Nookaew <em>et al.</em> 2010</a>). Our analysis also identified a significant link between ergosterol and 1-acyl-sn-glycerol-3-phosphate acyltransferase (<em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002210">SLC1</a></em>), which is responsible for the synthesis of phosphatidic acid (PA). PA is the central precursor for glycerophospholipids, DAG, and TAG, and it is also a signaling lipid and key transcriptional regulator of lipid biosynthesis (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Carman and Henry 2007</a>). Our correlation network from the co-regulated gene neighborhoods was expanded to provide context within the scope of the genome-scale metabolic map (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7B</a>). Follow-up studies are expected to use the systems interactions here to bring new understanding to lipid metabolism.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F7/graphic-7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Integrative method for correlation of multi-omics datasets reveals systems-level regulatory signatures. Correlation networks for ergosterol (A and B) and phosphatidylinositol (C and D) show first (green highlight) and second (blue highlight) significantly linked gene neighbors. (A and C) Genes in small white boxes were not identified as significantly correlated to ergosterol or phosphatidylinositol, but they are represented as &#x201C;connector&#x201D; nodes between metabolites. TFs implicated by the enrichment analysis are shown. Co-regulated gene neighborhood networks from (A and C) were expanded to include genes and metabolites necessary to perform the metabolic transformations indicated (B and D). This provides a more integrated perspective of cellular regulation. Measurement ratios for aerobic vs. anaerobic conditions were visualized with a log2 color bar, and the color of each node border represents the log10(P value). See node and edge color key. Gray indicates the lack of a measurement for that node." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1964801122" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Integrative method for correlation of multi-omics datasets reveals systems-level regulatory signatures. Correlation networks for ergosterol (A and B) and phosphatidylinositol (C and D) show first (green highlight) and second (blue highlight) significantly linked gene neighbors. (A and C) Genes in small white boxes were not identified as significantly correlated to ergosterol or phosphatidylinositol, but they are represented as &#x201C;connector&#x201D; nodes between metabolites. TFs implicated by the enrichment analysis are shown. Co-regulated gene neighborhood networks from (A and C) were expanded to include genes and metabolites necessary to perform the metabolic transformations indicated (B and D). This provides a more integrated perspective of cellular regulation. Measurement ratios for aerobic &lt;em&gt;vs.&lt;/em&gt; anaerobic conditions were visualized with a log&lt;sub&gt;2&lt;/sub&gt; color bar, and the color of each node border represents the log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;P&lt;/em&gt; value). See node and edge color key. Gray indicates the lack of a measurement for that node.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1979/F7/graphic-7.medium.gif" width="367" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/3/11/1979/F7/graphic-7.medium.gif" width="367" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F7/graphic-7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F7/graphic-7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463506" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F7/graphic-8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Integrative method for correlation of multi-omics datasets reveals systems-level regulatory signatures. Correlation networks for ergosterol (A and B) and phosphatidylinositol (C and D) show first (green highlight) and second (blue highlight) significantly linked gene neighbors. (A and C) Genes in small white boxes were not identified as significantly correlated to ergosterol or phosphatidylinositol, but they are represented as &#x201C;connector&#x201D; nodes between metabolites. TFs implicated by the enrichment analysis are shown. Co-regulated gene neighborhood networks from (A and C) were expanded to include genes and metabolites necessary to perform the metabolic transformations indicated (B and D). This provides a more integrated perspective of cellular regulation. Measurement ratios for aerobic vs. anaerobic conditions were visualized with a log2 color bar, and the color of each node border represents the log10(P value). See node and edge color key. Gray indicates the lack of a measurement for that node." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1964801122" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Integrative method for correlation of multi-omics datasets reveals systems-level regulatory signatures. Correlation networks for ergosterol (A and B) and phosphatidylinositol (C and D) show first (green highlight) and second (blue highlight) significantly linked gene neighbors. (A and C) Genes in small white boxes were not identified as significantly correlated to ergosterol or phosphatidylinositol, but they are represented as &#x201C;connector&#x201D; nodes between metabolites. TFs implicated by the enrichment analysis are shown. Co-regulated gene neighborhood networks from (A and C) were expanded to include genes and metabolites necessary to perform the metabolic transformations indicated (B and D). This provides a more integrated perspective of cellular regulation. Measurement ratios for aerobic &lt;em&gt;vs.&lt;/em&gt; anaerobic conditions were visualized with a log&lt;sub&gt;2&lt;/sub&gt; color bar, and the color of each node border represents the log&lt;sub&gt;10&lt;/sub&gt;(&lt;em&gt;P&lt;/em&gt; value). See node and edge color key. Gray indicates the lack of a measurement for that node.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/1979/F7/graphic-8.medium.gif" width="332" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.g3journal.org/content/ggg/3/11/1979/F7/graphic-8.medium.gif" width="332" height="440"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F7/graphic-8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/1979/F7/graphic-8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463506" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-50" class="first-child">Integrative method for correlation of multi-omics datasets reveals systems-level regulatory signatures. Correlation networks for ergosterol (A and B) and phosphatidylinositol (C and D) show first (green highlight) and second (blue highlight) significantly linked gene neighbors. (A and C) Genes in small white boxes were not identified as significantly correlated to ergosterol or phosphatidylinositol, but they are represented as “connector” nodes between metabolites. TFs implicated by the enrichment analysis are shown. Co-regulated gene neighborhood networks from (A and C) were expanded to include genes and metabolites necessary to perform the metabolic transformations indicated (B and D). This provides a more integrated perspective of cellular regulation. Measurement ratios for aerobic <em>vs.</em> anaerobic conditions were visualized with a log<sub>2</sub> color bar, and the color of each node border represents the log<sub>10</sub>(<em>P</em> value). See node and edge color key. Gray indicates the lack of a measurement for that node.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">For phosphatidylinositol species identified in the co-regulated gene neighborhood analysis (PINS, PINS160, PINS181, PINSS181), we observed that <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003636">Lsb6p</a>, <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005844">Faa1p</a>, and <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001407">Pan6p</a> were significantly correlated and directly linked as first or second neighbors to PINS (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7, C and D</a>). These data suggest that global regulatory points centered at this hub include precursor synthesis (<em>e.g.</em>, coenzyme A synthesis, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001407">PAN6</a></em>; long chain fatty acyl-CoA synthesis, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005844">FAA1</a></em>) and PINS utilization (phosphatidylinsoitol phosphate biosynthesis, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000003636">LSB6</a></em>). We also analyzed the storage depots of the cell, noting that steryl esters (se161) and triacylglycerol (tag100 and tag140) were significantly correlated and linked in the metabolic network to <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004608">PLB2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000817">FAA2</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000000245">FAT1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000005291">LRO1</a></em>, <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000001422">POT1</a></em>, and <em><a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000004506">CAT2</a></em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1/FigureS24.pdf">Figure S24</a>).</p><p id="p-52">In summary, we provide a new integrative method for mapping condition-dependent regulation in cells through the integrated analysis of multi-omics datasets and genome-scale metabolic models given previous knowledge (<em>e.g.</em>, known network topology or results from the analysis of other omics datasets such as transcription factor interactions). Specifically, we leveraged a full factorial design to build a visual metabolic model of the regulatory architecture controlling lipid metabolism in <em>S. cerevisiae</em>. Simple analysis led to the identification of genetic and metabolic changes involved in TAG and SE accumulation under carbon excess. From an adaptive perspective, yeast and, more broadly, eukaryotes make use of TAG as the major storage unit for energy and fatty acids used in membrane biosynthesis. Our results from growth conditions with high levels of residual glucose are entirely consistent with this phenomena and much is known about the synthesis, turnover, and regulation of nonpolar lipids in yeast (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Rajakumari <em>et al.</em> 2008</a>). We further revealed a major shift toward the synthesis and utilization of di-substituted medium acyl chain fatty acids under anaerobic conditions and then used <sup>13</sup>C-reaction flux data to demonstrate this was attributable to redox constraints.</p><p id="p-53">We also developed integrative methods for data analysis that use correlation analysis, metabolic topology, and transcription factor enrichment to interrogate and characterize the complex relationships that arise between gene expression levels, metabolite levels, lipid levels, and reaction fluxes across multiple conditions. Our approach quantitatively revealed that transcriptional regulation is not tightly controlled at the one-step level. Strikingly, only 22 metabolites and lipids were tightly correlated to their first and second gene neighbors. In the case of ergosterol, our analysis identified <a href="http://www.yeastgenome.org/cgi-bin/locus.fpl?dbid=S000002621">Upc2p</a> as the key regulator (<a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Espenshade and Hughes 2007</a>; <a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Nielsen 2009</a>). Our systems approach also quantitatively shows that sterol biosynthesis is more regulated at the transcriptional level than amino acid biosynthesis. Looking forward, our work will serve as a rich data resource for studying lipid metabolism, opening the door to new hypothesis-driven targeted experiments. In addition, we expect that it will contribute meaningfully to further efforts to use genome-scale metabolic models for contextualizing information obtained in systems-level data for bridging the gap between transcriptional state and metabolic phenotype.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-54">This work was supported by Danish Research Agency for Technology and Production, Nature and Universe, the NSF International Research Fellowship Program grant (0504168 to M.C.J.), the National Institutes of Health Pathway to Independence Award grant (GM081450 to M.C.J.), and the Thai Graduate Student Institute Science and Technology (TGIST). M.C.J. and J.N. conceived the project. M.C.J., I.N., and F.A.P. performed the experiments. C.T.W., M.C.J., and I.N. designed and performed the analysis. E.A., L.H., and J.N. participated in advice and discussion. M.C.J., C.T.W., and J.N. wrote the manuscript. All authors commented and agreed on the manuscript. The authors declare that they have no conflict of interest.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.113.006601/-/DC1</a>.</p></li><li class="fn" id="fn-4"><p id="p-55"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-05-03"><span class="received-label">Received </span>May 3, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-09-10"><span class="accepted-label">Accepted </span>September 10, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Jewett <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-web" id="cit-3.11.1979.1"><div class="cit-metadata unstructured">Affymetrix, 2004 Affymetrix GeneChip expression analysis technical manual. Affymetrix, Santa Clara, CA. Available at: <a href="http://www.affymetrix.com/support/technical/manual/expression_manual.affx">http://www.affymetrix.com/support/technical/manual/expression_manual.affx</a>. Accessed: June 2007.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.2" data-doi="10.1111/j.1365-2958.2006.05484.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Mendoza</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Control of fatty acid desaturation: a mechanism conserved from bacteria to humans.</span> <abbr class="cit-jnl-abbrev">Mol. Microbiol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">1507</span>–<span class="cit-lpage">1514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bmicrobiology%26rft.stitle%253DMol%2BMicrobiol%26rft.aulast%253DAguilar%26rft.auinit1%253DP.%2BS.%26rft.volume%253D62%26rft.issue%253D6%26rft.spage%253D1507%26rft.epage%253D1514%26rft.atitle%253DControl%2Bof%2Bfatty%2Bacid%2Bdesaturation%253A%2Ba%2Bmechanism%2Bconserved%2Bfrom%2Bbacteria%2Bto%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2958.2006.05484.x%26rft_id%253Dinfo%253Apmid%252F17087771%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2958.2006.05484.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17087771&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242236900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aveldano</span>  <span class="cit-name-given-names">M. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horrocks</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Quantitative release of fatty acids from lipids by a simple hydrolysis procedure.</span> <abbr class="cit-jnl-abbrev">J. Lipid Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1101</span>–<span class="cit-lpage">1105</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BLipid%2BResearch%26rft.stitle%253DJ.%2BLipid%2BRes.%26rft.aulast%253DAveldano%26rft.auinit1%253DM.%26rft.volume%253D24%26rft.issue%253D8%26rft.spage%253D1101%26rft.epage%253D1105%26rft.atitle%253DQuantitative%2Brelease%2Bof%2Bfatty%2Bacids%2Bfrom%2Blipids%2Bby%2Ba%2Bsimple%2Bhydrolysis%2Bprocedure%26rft_id%253Dinfo%253Apmid%252F6631237%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamxyIjtzOjU6InJlc2lkIjtzOjk6IjI0LzgvMTEwMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMTk3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bebek</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koyuturk</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chance</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">N. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integrative-omic for translational science—Session introduction.</span> <abbr class="cit-jnl-abbrev">Pacific Symposium On Biocomputing</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPacific%2BSymposium%2BOn%2BBiocomputing%26rft.volume%253D16%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.5" data-doi="10.1371/journal.pcbi.0020070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beyer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Workman</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hollunder</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Radke</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moller</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Integrated assessment and prediction of transcription factor binding.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DBeyer%26rft.auinit1%253DA.%26rft.volume%253D2%26rft.issue%253D6%26rft.spage%253De70%26rft.epage%253De70%26rft.atitle%253DIntegrated%2Bassessment%2Band%2Bprediction%2Bof%2Btranscription%2Bfactor%2Bbinding.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0020070%26rft_id%253Dinfo%253Apmid%252F16789814%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0020070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16789814&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bonferroni</span>  <span class="cit-name-given-names">C. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1936</span> <span class="cit-article-title">Teoria statistica delle classi e calcolo delle probabilità</span>. <abbr class="cit-jnl-abbrev">Pubblicazioni del R Istituto Superiore di Scienze Economiche e Commerciali di Firenze</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">3</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPubblicazioni%2Bdel%2BR%2BIstituto%2BSuperiore%2Bdi%2BScienze%2BEconomiche%2Be%2BCommerciali%2Bdi%2BFirenze%26rft.volume%253D8%26rft.spage%253D3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.7" data-doi="10.1371/journal.pcbi.1000270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bradley</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brauer</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rabinowitz</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troyanskaya</span>  <span class="cit-name-given-names">O. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Coordinated concentration changes of transcripts and metabolites in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DBradley%26rft.auinit1%253DP.%2BH.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253De1000270%26rft.epage%253De1000270%26rft.atitle%253DCoordinated%2Bconcentration%2Bchanges%2Bof%2Btranscripts%2Band%2Bmetabolites%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000270%26rft_id%253Dinfo%253Apmid%252F19180179%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19180179&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brehme</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vidal</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A global protein-lipid interactome map.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DBrehme%26rft.auinit1%253DM.%26rft.volume%253D6%26rft.spage%253D443%26rft.epage%253D443%26rft.atitle%253DA%2Bglobal%2Bprotein-lipid%2Binteractome%2Bmap.%26rft_id%253Dinfo%253Apmid%252F21119635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21119635&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.9" data-doi="10.1111/j.1365-2567.2009.03163.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brix</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lund</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kjaer</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Straarup</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellgren</span>  <span class="cit-name-given-names">L. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">CD4(+) T-cell activation is differentially modulated by bacteria-primed dendritic cells, but is generally down-regulated by n-3 polyunsaturated fatty acids.</span> <abbr class="cit-jnl-abbrev">Immunology</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">338</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BImmunology%26rft.stitle%253DSci.%2BImmunol.%26rft.aulast%253DBrix%26rft.auinit1%253DS.%26rft.volume%253D129%26rft.issue%253D3%26rft.spage%253D338%26rft.epage%253D350%26rft.atitle%253DCD4%2528%252B%2529%2BT-cell%2Bactivation%2Bis%2Bdifferentially%2Bmodulated%2Bby%2Bbacteria-primed%2Bdendritic%2Bcells%252C%2Bbut%2Bis%2Bgenerally%2Bdown-regulated%2Bby%2Bn-3%2Bpolyunsaturated%2Bfatty%2Bacids.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2567.2009.03163.x%26rft_id%253Dinfo%253Apmid%252F19909377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2567.2009.03163.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19909377&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carman</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Regulation of phospholipid synthesis in Saccharomyces cerevisiae by zinc depletion.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1771</span>: <span class="cit-fpage">322</span>–<span class="cit-lpage">330</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DCarman%26rft.auinit1%253DG.%2BM.%26rft.volume%253D1771%26rft.issue%253D3%26rft.spage%253D322%26rft.epage%253D330%26rft.atitle%253DRegulation%2Bof%2Bphospholipid%2Bsynthesis%2Bin%2BSaccharomyces%2Bcerevisiae%2Bby%2Bzinc%2Bdepletion.%26rft_id%253Dinfo%253Apmid%252F16807089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16807089&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.11" data-doi="10.1074/jbc.R700038200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carman</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">S. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Phosphatidic acid plays a central role in the transcriptional regulation of glycerophospholipid synthesis in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">37293</span>–<span class="cit-lpage">37297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.R700038200%26rft_id%253Dinfo%253Apmid%252F17981800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODIvNTIvMzcyOTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.12" data-doi="10.1146/annurev-biochem-060409-092229"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carman</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Regulation of phospholipid synthesis in the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">80</span>: <span class="cit-fpage">859</span>–<span class="cit-lpage">883</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DCarman%26rft.auinit1%253DG.%2BM.%26rft.volume%253D80%26rft.spage%253D859%26rft.epage%253D883%26rft.atitle%253DRegulation%2Bof%2Bphospholipid%2Bsynthesis%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-biochem-060409-092229%26rft_id%253Dinfo%253Apmid%252F21275641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-biochem-060409-092229&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21275641&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292895600034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.13" data-doi="10.1186/jbiol54"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castrillo</span>  <span class="cit-name-given-names">J. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeef</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoyle</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Growth control of the eukaryote cell: a systems biology study in yeast.</span> <abbr class="cit-jnl-abbrev">J. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">4</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bbiology%2B%2528Online%2529%26rft.stitle%253DJ%2BBiol%26rft.aulast%253DCastrillo%26rft.auinit1%253DJ.%2BI.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253D4%26rft.epage%253D4%26rft.atitle%253DGrowth%2Bcontrol%2Bof%2Bthe%2Beukaryote%2Bcell%253A%2Ba%2Bsystems%2Bbiology%2Bstudy%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fjbiol54%26rft_id%253Dinfo%253Apmid%252F17439666%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/jbiol54&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17439666&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.14" data-doi="10.1016/j.bbalip.2006.05.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopes</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Transcriptional regulation of yeast phospholipid biosynthetic genes.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1771</span>: <span class="cit-fpage">310</span>–<span class="cit-lpage">321</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DChen%26rft.auinit1%253DM.%26rft.volume%253D1771%26rft.issue%253D3%26rft.spage%253D310%26rft.epage%253D321%26rft.atitle%253DTranscriptional%2Bregulation%2Bof%2Byeast%2Bphospholipid%2Bbiosynthetic%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbalip.2006.05.017%26rft_id%253Dinfo%253Apmid%252F16854618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbalip.2006.05.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16854618&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gombert</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Analysis of flux estimates based on (13)C-labelling experiments.</span> <abbr class="cit-jnl-abbrev">Eur. J. Biochem.</abbr> <span class="cit-vol">269</span>: <span class="cit-fpage">2795</span>–<span class="cit-lpage">2800</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.stitle%253DFEBS%2BJ%26rft.aulast%253DChristensen%26rft.auinit1%253DB.%26rft.volume%253D269%26rft.issue%253D11%26rft.spage%253D2795%26rft.epage%253D2800%26rft.atitle%253DAnalysis%2Bof%2Bflux%2Bestimates%2Bbased%2Bon%2B%252813%2529C-labelling%2Bexperiments.%26rft_id%253Dinfo%253Apmid%252F12047390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12047390&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175906900020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.16" data-doi="10.1038/nprot.2007.324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cline</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smoot</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cerami</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuchinsky</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Landys</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Integration of biological networks and gene expression data using Cytoscape.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">2366</span>–<span class="cit-lpage">2382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DCline%26rft.auinit1%253DM.%2BS.%26rft.volume%253D2%26rft.issue%253D10%26rft.spage%253D2366%26rft.epage%253D2382%26rft.atitle%253DIntegration%2Bof%2Bbiological%2Bnetworks%2Band%2Bgene%2Bexpression%2Bdata%2Busing%2BCytoscape.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2007.324%26rft_id%253Dinfo%253Apmid%252F17947979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2007.324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17947979&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253139900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cowart</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Obeid</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Yeast sphingolipids: recent developments in understanding biosynthesis, regulation, and function.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1771</span>: <span class="cit-fpage">421</span>–<span class="cit-lpage">431</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DCowart%26rft.auinit1%253DL.%2BA.%26rft.volume%253D1771%26rft.issue%253D3%26rft.spage%253D421%26rft.epage%253D431%26rft.atitle%253DYeast%2Bsphingolipids%253A%2Brecent%2Bdevelopments%2Bin%2Bunderstanding%2Bbiosynthesis%252C%2Bregulation%252C%2Band%2Bfunction.%26rft_id%253Dinfo%253Apmid%252F16997623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=16997623&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245832900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cvijovic</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olivares-Hernandez</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agren</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dahr</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vongsangnak</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">BioMet Toolbox: genome-wide analysis of metabolism.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>(<span class="cit-issue">Suppl. 2</span>): <span class="cit-fpage">W144</span>–<span class="cit-lpage">W149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D38%26rft.spage%253DW144W149%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.19" data-doi="10.1016/j.bbalip.2006.07.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Czabany</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Athenstaedt</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daum</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Synthesis, storage and degradation of neutral lipids in yeast.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1771</span>: <span class="cit-fpage">299</span>–<span class="cit-lpage">309</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DCzabany%26rft.auinit1%253DT.%26rft.volume%253D1771%26rft.issue%253D3%26rft.spage%253D299%26rft.epage%253D309%26rft.atitle%253DSynthesis%252C%2Bstorage%2Band%2Bdegradation%2Bof%2Bneutral%2Blipids%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbalip.2006.07.001%26rft_id%253Dinfo%253Apmid%252F16916618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbalip.2006.07.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16916618&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245832900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.20" data-doi="10.1016/0003-2697(92)90149-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Dam</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">A method for the determination of changes of glycolytic metabolites in yeast on a subsecond time scale using extraction at neutral pH.</span> <abbr class="cit-jnl-abbrev">Anal. Biochem.</abbr> <span class="cit-vol">204</span>: <span class="cit-fpage">118</span>–<span class="cit-lpage">123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2Bbiochemistry%26rft.stitle%253DAnal%2BBiochem%26rft.aulast%253Dde%2BKoning%26rft.auinit1%253DW.%26rft.volume%253D204%26rft.issue%253D1%26rft.spage%253D118%26rft.epage%253D123%26rft.atitle%253DA%2Bmethod%2Bfor%2Bthe%2Bdetermination%2Bof%2Bchanges%2Bof%2Bglycolytic%2Bmetabolites%2Bin%2Byeast%2Bon%2Ba%2Bsubsecond%2Btime%2Bscale%2Busing%2Bextraction%2Bat%2Bneutral%2BpH.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0003-2697%252892%252990149-2%26rft_id%253Dinfo%253Apmid%252F1514678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0003-2697(92)90149-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1514678&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JB52400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.21" data-doi="10.1146/annurev.genet.41.110306.130315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Espenshade</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Regulation of sterol synthesis in eukaryotes.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">401</span>–<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenet%26rft.aulast%253DEspenshade%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D41%26rft.spage%253D401%26rft.epage%253D427%26rft.atitle%253DRegulation%2Bof%2Bsterol%2Bsynthesis%2Bin%2Beukaryotes.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.41.110306.130315%26rft_id%253Dinfo%253Apmid%252F17666007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.genet.41.110306.130315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17666007&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252359500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.22" data-doi="10.1186/1471-2164-9-341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fazio</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jewett</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daran-Lapujade</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mustacchi</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Usaite</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Transcription factor control of growth rate dependent genes in Saccharomyces cerevisiae: a three factor design.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DFazio%26rft.auinit1%253DA.%26rft.volume%253D9%26rft.spage%253D341%26rft.epage%253D341%26rft.atitle%253DTranscription%2Bfactor%2Bcontrol%2Bof%2Bgrowth%2Brate%2Bdependent%2Bgenes%2Bin%2BSaccharomyces%2Bcerevisiae%253A%2Ba%2Bthree%2Bfactor%2Bdesign.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-341%26rft_id%253Dinfo%253Apmid%252F18638364%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18638364&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.23" data-doi="10.1038/nbt1401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Feist</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palsson</span>  <span class="cit-name-given-names">B. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The growing scope of applications of genome-scale metabolic reconstructions using Escherichia coli.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DFeist%26rft.auinit1%253DA.%2BM.%26rft.volume%253D26%26rft.issue%253D6%26rft.spage%253D659%26rft.epage%253D667%26rft.atitle%253DThe%2Bgrowing%2Bscope%2Bof%2Bapplications%2Bof%2Bgenome-scale%2Bmetabolic%2Breconstructions%2Busing%2BEscherichia%2Bcoli.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1401%26rft_id%253Dinfo%253Apmid%252F18536691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18536691&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256611900025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.24" data-doi="10.1038/msb.2010.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fendt</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buescher</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rudroff</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Picotti</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamboni</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Tradeoff between enzyme and metabolite efficiency maintains metabolic homeostasis upon perturbations in enzyme capacity.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DFendt%26rft.auinit1%253DS.%2BM.%26rft.volume%253D6%26rft.spage%253D356%26rft.epage%253D356%26rft.atitle%253DTradeoff%2Bbetween%2Benzyme%2Band%2Bmetabolite%2Befficiency%2Bmaintains%2Bmetabolic%2Bhomeostasis%2Bupon%2Bperturbations%2Bin%2Benzyme%2Bcapacity.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2010.11%26rft_id%253Dinfo%253Apmid%252F20393576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/msb.2010.11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20393576&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Folch</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lees</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sloane Stanley</span>  <span class="cit-name-given-names">G. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1957</span> <span class="cit-article-title">A simple method for the isolation and purification of total lipides from animal tissues.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">226</span>: <span class="cit-fpage">497</span>–<span class="cit-lpage">509</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DFolch%26rft.auinit1%253DJ.%26rft.volume%253D226%26rft.issue%253D1%26rft.spage%253D497%26rft.epage%253D509%26rft.atitle%253DA%2BSIMPLE%2BMETHOD%2BFOR%2BTHE%2BISOLATION%2BAND%2BPURIFICATION%2BOF%2BTOTAL%2BLIPIDES%2BFROM%2BANIMAL%2BTISSUES%26rft_id%253Dinfo%253Apmid%252F13428781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJqYmMiO3M6NToicmVzaWQiO3M6OToiMjI2LzEvNDk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.26" data-doi="10.1038/msb.2010.87"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallego</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betts</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gvozdenovic-Jeremic</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maeda</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matetzki</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A systematic screen for protein-lipid interactions in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DGallego%26rft.auinit1%253DO.%26rft.volume%253D6%26rft.spage%253D430%26rft.epage%253D430%26rft.atitle%253DA%2Bsystematic%2Bscreen%2Bfor%2Bprotein-lipid%2Binteractions%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2010.87%26rft_id%253Dinfo%253Apmid%252F21119626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/msb.2010.87&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21119626&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.27" data-doi="10.1016/j.bbalip.2006.06.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gaspar</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aregullin</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jesch</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nunez</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villa-Garcia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The emergence of yeast lipidomics.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1771</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DGaspar%26rft.auinit1%253DM.%2BL.%26rft.volume%253D1771%26rft.issue%253D3%26rft.spage%253D241%26rft.epage%253D254%26rft.atitle%253DThe%2Bemergence%2Bof%2Byeast%2Blipidomics.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbalip.2006.06.011%26rft_id%253Dinfo%253Apmid%252F16920401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbalip.2006.06.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16920401&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.28" data-doi="10.1073/pnas.0808022106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianoulis</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raes</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patel</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bjornson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korbel</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Quantifying environmental adaptation of metabolic pathways in metagenomics.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">1374</span>–<span class="cit-lpage">1379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0808022106%26rft_id%253Dinfo%253Apmid%252F19164758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA2LzUvMTM3NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMTk3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.29" data-doi="10.1128/JB.183.4.1441-1451.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gombert</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreira dos Santos</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christensen</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Network identification and flux quantification in the central metabolism of Saccharomyces cerevisiae under different conditions of glucose repression.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1441</span>–<span class="cit-lpage">1451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DGombert%26rft.auinit1%253DA.%2BK.%26rft.volume%253D183%26rft.issue%253D4%26rft.spage%253D1441%26rft.epage%253D1451%26rft.atitle%253DNetwork%2BIdentification%2Band%2BFlux%2BQuantification%2Bin%2Bthe%2BCentral%2BMetabolism%2Bof%2BSaccharomyces%2Bcerevisiae%2Bunder%2BDifferent%2BConditions%2Bof%2BGlucose%2BRepression%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.183.4.1441-1451.2001%26rft_id%253Dinfo%253Apmid%252F11157958%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTA6IjE4My80LzE0NDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.30" data-doi="10.1126/science.1176951"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guell</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Noort</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yus</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leigh-Bell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Transcriptome complexity in a genome-reduced bacterium.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1268</span>–<span class="cit-lpage">1271</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGuell%26rft.auinit1%253DM.%26rft.volume%253D326%26rft.issue%253D5957%26rft.spage%253D1268%26rft.epage%253D1271%26rft.atitle%253DTranscriptome%2BComplexity%2Bin%2Ba%2BGenome-Reduced%2BBacterium%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1176951%26rft_id%253Dinfo%253Apmid%252F19965477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ny8xMjY4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.31" data-doi="10.1186/1752-0509-5-26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurya</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merrill</span>  <span class="cit-name-given-names">A. H.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glass</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Subramaniam</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Integration of lipidomics and transcriptomics data towards a systems biology model of sphingolipid metabolism.</span> <abbr class="cit-jnl-abbrev">BMC Syst. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BSyst%2BBiol%26rft.aulast%253DGupta%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.spage%253D26%26rft.epage%253D26%26rft.atitle%253DIntegration%2Bof%2Blipidomics%2Band%2Btranscriptomics%2Bdata%2Btowards%2Ba%2Bsystems%2Bbiology%2Bmodel%2Bof%2Bsphingolipid%2Bmetabolism.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-5-26%26rft_id%253Dinfo%253Apmid%252F21303545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-5-26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21303545&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.1979.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sewell</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-source">Extraction of lipids and derivative formation</span>, <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.33" data-doi="10.1038/nrm2329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hannun</span>  <span class="cit-name-given-names">Y. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Obeid</span>  <span class="cit-name-given-names">L. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Principles of bioactive lipid signalling: lessons from sphingolipids.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">150</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DHannun%26rft.auinit1%253DY.%2BA.%26rft.volume%253D9%26rft.issue%253D2%26rft.spage%253D139%26rft.epage%253D150%26rft.atitle%253DPrinciples%2Bof%2Bbioactive%2Blipid%2Bsignalling%253A%2Blessons%2Bfrom%2Bsphingolipids.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2329%26rft_id%253Dinfo%253Apmid%252F18216770%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm2329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18216770&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000252676600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.34" data-doi="10.1038/nature02800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harbison</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinaldi</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macisaac</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Transcriptional regulatory code of a eukaryotic genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">431</span>: <span class="cit-fpage">99</span>–<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHarbison%26rft.auinit1%253DC.%2BT.%26rft.volume%253D431%26rft.issue%253D7004%26rft.spage%253D99%26rft.epage%253D104%26rft.atitle%253DTranscriptional%2Bregulatory%2Bcode%2Bof%2Ba%2Beukaryotic%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02800%26rft_id%253Dinfo%253Apmid%252F15343339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15343339&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223641500050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.35" data-doi="10.1016/j.copbio.2010.12.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heinemann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sauer</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">From good old biochemical analyses to high-throughput omics measurements and back.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Biotechnol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">2</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.stitle%253DCurrent%2BOpinion%2Bin%2BBiotechnology%26rft.aulast%253DHeinemann%26rft.auinit1%253DM.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D2%26rft.atitle%253DFrom%2Bgood%2Bold%2Bbiochemical%2Banalyses%2Bto%2Bhigh-throughput%2Bomics%2Bmeasurements%2Band%2Bback.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.copbio.2010.12.002%26rft_id%253Dinfo%253Apmid%252F21190838%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.copbio.2010.12.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21190838&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287840700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.36" data-doi="10.1016/j.semcdb.2005.01.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henneberry</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sturley</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Sterol homeostasis in the budding yeast, Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Semin. Cell Dev. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">161</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSeminars%2Bin%2Bcell%2B%2526%2Bdevelopmental%2Bbiology%26rft.stitle%253DSemin%2BCell%2BDev%2BBiol%26rft.aulast%253DHenneberry%26rft.auinit1%253DA.%2BL.%26rft.volume%253D16%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D161%26rft.atitle%253DSterol%2Bhomeostasis%2Bin%2Bthe%2Bbudding%2Byeast%252C%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.semcdb.2005.01.006%26rft_id%253Dinfo%253Apmid%252F15797826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.semcdb.2005.01.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15797826&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.37" data-doi="10.1093/nar/26.1.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hodges</span>  <span class="cit-name-given-names">P. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Payne</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrels</span>  <span class="cit-name-given-names">J. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The Yeast Protein Database (YPD): a curated proteome database for Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">68</span>–<span class="cit-lpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F26.1.68%26rft_id%253Dinfo%253Apmid%252F9399803%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjc6IjI2LzEvNjgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rose</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Lipid composition of Saccharomyces cerevisiae as influenced by growth temperature.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">260</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">653</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DHunter%26rft.auinit1%253DK.%26rft.volume%253D260%26rft.issue%253D4%26rft.spage%253D639%26rft.epage%253D653%26rft.atitle%253DLipid%2Bcomposition%2Bof%2BSaccharomyces%2Bcerevisiae%2Bas%2Binfluenced%2Bby%2Bgrowth%2Btemperature.%26rft_id%253Dinfo%253Apmid%252F4554953%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=4554953&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.39" data-doi="10.1126/science.1132067"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ishii</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakahigashi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baba</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Soga</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Multiple high-throughput analyses monitor the response of E. coli to perturbations.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">316</span>: <span class="cit-fpage">593</span>–<span class="cit-lpage">597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1132067%26rft_id%253Dinfo%253Apmid%252F17379776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTYvNTgyNC81OTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.40" data-doi="10.1093/nar/gkr988"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanehisa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furumichi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanabe</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">KEGG for integration and interpretation of large-scale molecular data sets.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D109</span>–<span class="cit-lpage">D114</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr988%26rft_id%253Dinfo%253Apmid%252F22080510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9EMS9EMTA5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.41" data-doi="10.1126/science.1176343"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuhner</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Noort</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Betts</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leo-Macias</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batisse</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Proteome organization in a genome-reduced bacterium.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1235</span>–<span class="cit-lpage">1240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKuhner%26rft.auinit1%253DS.%26rft.volume%253D326%26rft.issue%253D5957%26rft.spage%253D1235%26rft.epage%253D1240%26rft.atitle%253DProteome%2BOrganization%2Bin%2Ba%2BGenome-Reduced%2BBacterium%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1176343%26rft_id%253Dinfo%253Apmid%252F19965468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ny8xMjM1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lees</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bard</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirsch</span>  <span class="cit-name-given-names">D. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Biochemistry and molecular biology of sterol synthesis in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Crit. Rev. Biochem. Mol. Biol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCritical%2BReviews%2Bin%2BBiochemistry%2Band%2BMolecular%2BBiology%26rft.stitle%253DCRIT%2BREV%2BBIOCHEM%2BMOL%2BBIOL%26rft.aulast%253DLees%26rft.auinit1%253DN.%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D33%26rft.epage%253D47%26rft.atitle%253DBiochemistry%2Band%2Bmolecular%2Bbiology%2Bof%2Bsterol%2Bsynthesis%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F10090470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10090470&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000078891400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vagelos</span>  <span class="cit-name-given-names">P. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1962</span> <span class="cit-article-title">The mechanism of tricarboxylic acid cycle regulation of fatty acid synthesis.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">237</span>: <span class="cit-fpage">1787</span>–<span class="cit-lpage">1792</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DMartin%26rft.auinit1%253DD.%2BB.%26rft.volume%253D237%26rft.issue%253D6%26rft.spage%253D1787%26rft.epage%253D1792%26rft.atitle%253DThe%2BMechanism%2Bof%2BTricarboxylic%2BAcid%2BCycle%2BRegulation%2Bof%2BFatty%2BAcid%2BSynthesis%26rft_id%253Dinfo%253Apmid%252F14470343%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJqYmMiO3M6NToicmVzaWQiO3M6MTA6IjIzNy82LzE3ODciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.44" data-doi="10.1038/nature04399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Maxfield</span>  <span class="cit-name-given-names">F. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabas</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Role of cholesterol and lipid organization in disease.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">438</span>: <span class="cit-fpage">612</span>–<span class="cit-lpage">621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMaxfield%26rft.auinit1%253DF.%2BR.%26rft.volume%253D438%26rft.issue%253D7068%26rft.spage%253D612%26rft.epage%253D621%26rft.atitle%253DRole%2Bof%2Bcholesterol%2Band%2Blipid%2Borganization%2Bin%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04399%26rft_id%253Dinfo%253Apmid%252F16319881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04399&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16319881&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233593100039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.45" data-doi="10.1016/0003-2697(88)90500-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Merrill</span>  <span class="cit-name-given-names">A. H.</span>  <span class="cit-name-suffix">Jr.</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullins</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jamison</span>  <span class="cit-name-given-names">W. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nimkar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Quantitation of free sphingosine in liver by high-performance liquid chromatography.</span> <abbr class="cit-jnl-abbrev">Anal. Biochem.</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">373</span>–<span class="cit-lpage">381</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2Bbiochemistry%26rft.stitle%253DAnal%2BBiochem%26rft.aulast%253DMerrill%26rft.auinit1%253DA.%2BH.%26rft.volume%253D171%26rft.issue%253D2%26rft.spage%253D373%26rft.epage%253D381%26rft.atitle%253DQuantitation%2Bof%2Bfree%2Bsphingosine%2Bin%2Bliver%2Bby%2Bhigh-performance%2Bliquid%2Bchromatography.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0003-2697%252888%252990500-3%26rft_id%253Dinfo%253Apmid%252F3407935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0003-2697(88)90500-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3407935&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1988N915300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.46" data-doi="10.1073/pnas.0811091106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moxley</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jewett</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Antoniewicz</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Villas-Boas</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alper</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Linking high-resolution metabolic flux phenotypes and transcriptional regulation in yeast modulated by the global regulator Gcn4p.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">6477</span>–<span class="cit-lpage">6482</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0811091106%26rft_id%253Dinfo%253Apmid%252F19346491%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzE2LzY0NzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.47" data-doi="10.1016/j.febslet.2009.10.054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Systems biology of lipid metabolism: from yeast to human.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">583</span>: <span class="cit-fpage">3905</span>–<span class="cit-lpage">3913</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DNielsen%26rft.auinit1%253DJ.%26rft.volume%253D583%26rft.issue%253D24%26rft.spage%253D3905%26rft.epage%253D3913%26rft.atitle%253DSystems%2Bbiology%2Bof%2Blipid%2Bmetabolism%253A%2Bfrom%2Byeast%2Bto%2Bhuman.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.febslet.2009.10.054%26rft_id%253Dinfo%253Apmid%252F19854183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.febslet.2009.10.054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19854183&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273208000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.48" data-doi="10.1016/S0021-9673(00)00873-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Madsen</span>  <span class="cit-name-given-names">J. O.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Determination of ergosterol on mouldy building materials using isotope dilution and gas chromatography-tandem mass spectrometry.</span> <abbr class="cit-jnl-abbrev">J. Chromatogr. A</abbr> <span class="cit-vol">898</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BChromatogr.%2BA%26rft.volume%253D898%26rft.spage%253D227%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0021-9673%252800%252900873-6%26rft_id%253Dinfo%253Apmid%252F11117420%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0021-9673(00)00873-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11117420&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.49" data-doi="10.1371/journal.pone.0012361"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nookaew</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabrielsson</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmang</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandberg</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Identifying molecular effects of diet through systems biology: influence of herring diet on sterol metabolism and protein turnover in mice.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e12361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DNookaew%26rft.auinit1%253DI.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253De12361%26rft.epage%253De12361%26rft.atitle%253DIdentifying%2Bmolecular%2Beffects%2Bof%2Bdiet%2Bthrough%2Bsystems%2Bbiology%253A%2Binfluence%2Bof%2Bherring%2Bdiet%2Bon%2Bsterol%2Bmetabolism%2Band%2Bprotein%2Bturnover%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0012361%26rft_id%253Dinfo%253Apmid%252F20808764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0012361&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20808764&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.50" data-doi="10.1186/1752-0509-2-71"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nookaew</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jewett</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meechai</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thammarongtham</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laoteng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The genome-scale metabolic model iIN800 of Saccharomyces cerevisiae and its validation: a scaffold to query lipid metabolism.</span> <abbr class="cit-jnl-abbrev">BMC Syst. Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">71</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BSyst%2BBiol%26rft.aulast%253DNookaew%26rft.auinit1%253DI.%26rft.volume%253D2%26rft.spage%253D71%26rft.epage%253D71%26rft.atitle%253DThe%2Bgenome-scale%2Bmetabolic%2Bmodel%2BiIN800%2Bof%2BSaccharomyces%2Bcerevisiae%2Band%2Bits%2Bvalidation%253A%2Ba%2Bscaffold%2Bto%2Bquery%2Blipid%2Bmetabolism.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1752-0509-2-71%26rft_id%253Dinfo%253Apmid%252F18687109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1752-0509-2-71&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18687109&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.51" data-doi="10.1126/science.1183757"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ochman</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raghavan</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Systems biology. Excavating the functional landscape of bacterial cells.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1200</span>–<span class="cit-lpage">1201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DOchman%26rft.auinit1%253DH.%26rft.volume%253D326%26rft.issue%253D5957%26rft.spage%253D1200%26rft.epage%253D1201%26rft.atitle%253DExcavating%2Bthe%2BFunctional%2BLandscape%2Bof%2BBacterial%2BCells%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1183757%26rft_id%253Dinfo%253Apmid%252F19965458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ny8xMjAwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.52" data-doi="10.1038/nchembio.441"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palsson</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zengler</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The challenges of integrating multi-omic data sets.</span> <abbr class="cit-jnl-abbrev">Nat. Chem. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">787</span>–<span class="cit-lpage">789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BChem%2BBiol%26rft.aulast%253DPalsson%26rft.auinit1%253DB.%26rft.volume%253D6%26rft.issue%253D11%26rft.spage%253D787%26rft.epage%253D789%26rft.atitle%253DThe%2Bchallenges%2Bof%2Bintegrating%2Bmulti-omic%2Bdata%2Bsets.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnchembio.441%26rft_id%253Dinfo%253Apmid%252F20976870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nchembio.441&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20976870&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.53" data-doi="10.1074/jbc.M204490200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Piper</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daran-Lapujade</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bro</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regenberg</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knudsen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Reproducibility of oligonucleotide microarray transcriptome analyses. An interlaboratory comparison using chemostat cultures of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">37001</span>–<span class="cit-lpage">37008</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M204490200%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzcvNDAvMzcwMDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.54" data-doi="10.1128/AEM.00602-08"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pizarro</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jewett</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agosin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Growth temperature exerts differential physiological and transcriptional responses in laboratory and wine strains of Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">6358</span>–<span class="cit-lpage">6368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.00602-08%26rft_id%253Dinfo%253Apmid%252F18723660%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI3NC8yMC82MzU4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.55" data-doi="10.1016/j.plipres.2008.01.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rajakumari</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grillitsch</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daum</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Synthesis and turnover of non-polar lipids in yeast.</span> <abbr class="cit-jnl-abbrev">Prog. Lipid Res.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">157</span>–<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProgress%2Bin%2Blipid%2Bresearch%26rft.stitle%253DProg%2BLipid%2BRes%26rft.aulast%253DRajakumari%26rft.auinit1%253DS.%26rft.volume%253D47%26rft.issue%253D3%26rft.spage%253D157%26rft.epage%253D171%26rft.atitle%253DSynthesis%2Band%2Bturnover%2Bof%2Bnon-polar%2Blipids%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plipres.2008.01.001%26rft_id%253Dinfo%253Apmid%252F18258205%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plipres.2008.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18258205&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255378900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.56" data-doi="10.1186/gb-2006-7-11-r107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Regenberg</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grotkjaer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Winther</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fausboll</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akesson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Growth-rate regulated genes have profound impact on interpretation of transcriptome profiling in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">R107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DRegenberg%26rft.auinit1%253DB.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253DR107%26rft.epage%253DR107%26rft.atitle%253DGrowth-rate%2Bregulated%2Bgenes%2Bhave%2Bprofound%2Bimpact%2Bon%2Binterpretation%2Bof%2Btranscriptome%2Bprofiling%2Bin%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2006-7-11-r107%26rft_id%253Dinfo%253Apmid%252F17105650%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2006-7-11-r107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17105650&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.57" data-doi="10.1111/j.1365-313X.2011.04750.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rohrmann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tohge</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alba</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Osorio</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caldana</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Combined transcription factor profiling, microarray analysis and metabolite profiling reveals the transcriptional control of metabolic shifts occurring during tomato fruit development.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">999</span>–<span class="cit-lpage">1013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DRohrmann%26rft.auinit1%253DJ.%26rft.volume%253D68%26rft.issue%253D6%26rft.spage%253D999%26rft.epage%253D1013%26rft.atitle%253DCombined%2Btranscription%2Bfactor%2Bprofiling%252C%2Bmicroarray%2Banalysis%2Band%2Bmetabolite%2Bprofiling%2Breveals%2Bthe%2Btranscriptional%2Bcontrol%2Bof%2Bmetabolic%2Bshifts%2Boccurring%2Bduring%2Btomato%2Bfruit%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2011.04750.x%26rft_id%253Dinfo%253Apmid%252F21851430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2011.04750.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21851430&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298094000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.58" data-doi="10.1007/s00253-009-2338-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruenwai</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheevadhanarak</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rachdawong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanticharoen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laoteng</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Oxygen-induced expression of Delta(6)-, Delta(9)- and Delta(12)-desaturase genes modulates fatty acid composition in Mucor rouxii.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">334</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.stitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.aulast%253DRuenwai%26rft.auinit1%253DR.%26rft.volume%253D86%26rft.issue%253D1%26rft.spage%253D327%26rft.epage%253D334%26rft.atitle%253DOxygen-induced%2Bexpression%2Bof%2BDelta%25286%2529-%252C%2BDelta%25289%2529-%2Band%2BDelta%252812%2529-desaturase%2Bgenes%2Bmodulates%2Bfatty%2Bacid%2Bcomposition%2Bin%2BMucor%2Brouxii.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00253-009-2338-4%26rft_id%253Dinfo%253Apmid%252F20187298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00253-009-2338-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20187298&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.59" data-doi="10.1002/biot.201000316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruenwai</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neiss</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laoteng</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vongsangnak</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dalfard</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Heterologous production of polyunsaturated fatty acids in Saccharomyces cerevisiae causes a global transcriptional response resulting in reduced proteasomal activity and increased oxidative stress.</span> <abbr class="cit-jnl-abbrev">Biotechnol. J.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBiotechnol%2BJ%26rft.aulast%253DRuenwai%26rft.auinit1%253DR.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253D343%26rft.epage%253D356%26rft.atitle%253DHeterologous%2Bproduction%2Bof%2Bpolyunsaturated%2Bfatty%2Bacids%2Bin%2BSaccharomyces%2Bcerevisiae%2Bcauses%2Ba%2Bglobal%2Btranscriptional%2Bresponse%2Bresulting%2Bin%2Breduced%2Bproteasomal%2Bactivity%2Band%2Bincreased%2Boxidative%2Bstress.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbiot.201000316%26rft_id%253Dinfo%253Apmid%252F21184438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/biot.201000316&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21184438&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000288035100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.60" data-doi="10.1126/science.1142502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sauer</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heinemann</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamboni</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetics. Getting closer to the whole picture.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">316</span>: <span class="cit-fpage">550</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSauer%26rft.auinit1%253DU.%26rft.volume%253D316%26rft.issue%253D5824%26rft.spage%253D550%26rft.epage%253D551%26rft.atitle%253DGENETICS%253A%2BGetting%2BCloser%2Bto%2Bthe%2BWhole%2BPicture%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1142502%26rft_id%253Dinfo%253Apmid%252F17463274%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTYvNTgyNC81NTAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.61" data-doi="10.1016/j.bbalip.2007.03.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulz</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prinz</span>  <span class="cit-name-given-names">W. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Sterol transport in yeast and the oxysterol binding protein homologue (OSH) family.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1771</span>: <span class="cit-fpage">769</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DSchulz%26rft.auinit1%253DT.%2BA.%26rft.volume%253D1771%26rft.issue%253D6%26rft.spage%253D769%26rft.epage%253D780%26rft.atitle%253DSterol%2Btransport%2Bin%2Byeast%2Band%2Bthe%2Boxysterol%2Bbinding%2Bprotein%2Bhomologue%2B%2528OSH%2529%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbalip.2007.03.003%26rft_id%253Dinfo%253Apmid%252F17434796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbalip.2007.03.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17434796&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247554900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.62" data-doi="10.1038/msb4100157"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramsey</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marelli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marzolf</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hwang</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Transcriptional responses to fatty acid are coordinated by combinatorial control.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">115</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DSmith%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D115%26rft.epage%253D115%26rft.atitle%253DTranscriptional%2Bresponses%2Bto%2Bfatty%2Bacid%2Bare%2Bcoordinated%2Bby%2Bcombinatorial%2Bcontrol%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb4100157%26rft_id%253Dinfo%253Apmid%252F17551510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjMvMS8xMTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.63" data-doi="10.1038/nature08721"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steen</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bokinsky</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schirmer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Microbial production of fatty-acid-derived fuels and chemicals from plant biomass.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">463</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSteen%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D463%26rft.issue%253D7280%26rft.spage%253D559%26rft.epage%253D562%26rft.atitle%253DMicrobial%2Bproduction%2Bof%2Bfatty-acid-derived%2Bfuels%2Band%2Bchemicals%2Bfrom%2Bplant%2Bbiomass.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08721%26rft_id%253Dinfo%253Apmid%252F20111002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08721&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20111002&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273981100055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.64" data-doi="10.1021/ac0614846"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Styczynski</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moxley</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tong</span>  <span class="cit-name-given-names">L. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walther</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Systematic identification of conserved metabolites in GC/MS data for metabolomics and biomarker discovery.</span> <abbr class="cit-jnl-abbrev">Anal. Chem.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">966</span>–<span class="cit-lpage">973</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.stitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.aulast%253DStyczynski%26rft.auinit1%253DM.%2BP.%26rft.volume%253D79%26rft.issue%253D3%26rft.spage%253D966%26rft.epage%253D973%26rft.atitle%253DSystematic%2Bidentification%2Bof%2Bconserved%2Bmetabolites%2Bin%2BGC%252FMS%2Bdata%2Bfor%2Bmetabolomics%2Band%2Bbiomarker%2Bdiscovery.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fac0614846%26rft_id%253Dinfo%253Apmid%252F17263323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ac0614846&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17263323&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.65" data-doi="10.1074/jbc.M410573200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tai</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boer</span>  <span class="cit-name-given-names">V. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daran-Lapujade</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Winde</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Two-dimensional transcriptome analysis in chemostat cultures. Combinatorial effects of oxygen availability and macronutrient limitation in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">280</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M410573200%26rft_id%253Dinfo%253Apmid%252F15496405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjk6IjI4MC8xLzQzNyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMTk3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.66" data-doi="10.1128/AEM.01935-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tavares</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grotkjaer</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Obsen</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haslam</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Napier</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Metabolic engineering of Saccharomyces cerevisiae for production of Eicosapentaenoic Acid, using a novel {Delta}5-Desaturase from Paramecium tetraurelia.</span> <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">1854</span>–<span class="cit-lpage">1861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.01935-10%26rft_id%253Dinfo%253Apmid%252F21193673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6Ijc3LzUvMTg1NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMTk3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.67" data-doi="10.1016/j.bbalip.2006.07.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tehlivets</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheuringer</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kohlwein</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Fatty acid synthesis and elongation in yeast.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1771</span>: <span class="cit-fpage">255</span>–<span class="cit-lpage">270</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DTehlivets%26rft.auinit1%253DO.%26rft.volume%253D1771%26rft.issue%253D3%26rft.spage%253D255%26rft.epage%253D270%26rft.atitle%253DFatty%2Bacid%2Bsynthesis%2Band%2Belongation%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbalip.2006.07.004%26rft_id%253Dinfo%253Apmid%252F16950653%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbalip.2006.07.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16950653&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245832900003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.68" data-doi="10.1016/j.bbalip.2008.04.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teske</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taramino</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhuiyan</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumaraswami</span>  <span class="cit-name-given-names">N. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Randall</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genetic analyses involving interactions between the ergosterol biosynthetic enzymes, lanosterol synthase (Erg7p) and 3-ketoreductase (Erg27p), in the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1781</span>: <span class="cit-fpage">359</span>–<span class="cit-lpage">366</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DTeske%26rft.auinit1%253DB.%26rft.volume%253D1781%26rft.issue%253D8%26rft.spage%253D359%26rft.epage%253D366%26rft.atitle%253DGenetic%2Banalyses%2Binvolving%2Binteractions%2Bbetween%2Bthe%2Bergosterol%2Bbiosynthetic%2Benzymes%252C%2Blanosterol%2Bsynthase%2B%2528Erg7p%2529%2Band%2B3-ketoreductase%2B%2528Erg27p%2529%252C%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbalip.2008.04.017%26rft_id%253Dinfo%253Apmid%252F18555807%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbalip.2008.04.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18555807&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.69" data-doi="10.1038/msb.2009.67"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Usaite</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jewett</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliveira</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yates</span>  <span class="cit-name-given-names">J. R.</span>  <span class="cit-name-suffix">3rd</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Olsson</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reconstruction of the yeast Snf1 kinase regulatory network reveals its role as a global energy regulator.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DUsaite%26rft.auinit1%253DR.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D319%26rft.epage%253D319%26rft.atitle%253DReconstruction%2Bof%2Bthe%2Byeast%2BSnf1%2Bkinase%2Bregulatory%2Bnetwork%2Breveals%2Bits%2Brole%2Bas%2Ba%2Bglobal%2Benergy%2Bregulator%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2009.67%26rft_id%253Dinfo%253Apmid%252F19888214%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjUvMS8zMTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.70" data-doi="10.1016/S0141-0229(00)00162-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Dijken</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bauer</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brambilla</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duboc</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francois</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">An interlaboratory comparison of physiological and genetic properties of four Saccharomyces cerevisiae strains.</span> <abbr class="cit-jnl-abbrev">Enzyme Microb. Technol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">706</span>–<span class="cit-lpage">714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnzyme%2Band%2BMicrobial%2BTechnology%26rft.stitle%253DEnzyme%2Band%2BMicrobial%2BTechnology%26rft.aulast%253Dvan%2BDijken%2BJP%26rft.volume%253D26%26rft.issue%253D9-10%26rft.spage%253D706%26rft.epage%253D714%26rft.atitle%253DAn%2Binterlaboratory%2Bcomparison%2Bof%2Bphysiological%2Band%2Bgenetic%2Bproperties%2Bof%2Bfour%2BSaccharomyces%2Bcerevisiae%2Bstrains.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0141-0229%252800%252900162-9%26rft_id%253Dinfo%253Apmid%252F10862876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0141-0229(00)00162-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10862876&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087854600011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.71" data-doi="10.1002/yea.320080703"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verduyn</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postma</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Scheffers</span>  <span class="cit-name-given-names">W. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Dijken</span>  <span class="cit-name-given-names">J. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Effect of benzoic acid on metabolic fluxes in yeasts: a continuous-culture study on the regulation of respiration and alcoholic fermentation.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">501</span>–<span class="cit-lpage">517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DVerduyn%26rft.auinit1%253DC.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253D501%26rft.epage%253D517%26rft.atitle%253DEffect%2Bof%2Bbenzoic%2Bacid%2Bon%2Bmetabolic%2Bfluxes%2Bin%2Byeasts%253A%2Ba%2Bcontinuous-culture%2Bstudy%2Bon%2Bthe%2Bregulation%2Bof%2Brespiration%2Band%2Balcoholic%2Bfermentation.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.320080703%26rft_id%253Dinfo%253Apmid%252F1523884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.320080703&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1523884&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992JF01900002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.72" data-doi="10.1128/MCB.21.19.6395-6405.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rine</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Upc2p and Ecm22p, dual regulators of sterol biosynthesis in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">6395</span>–<span class="cit-lpage">6405</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DVik%26rft.auinit1%253DA.%26rft.volume%253D21%26rft.issue%253D19%26rft.spage%253D6395%26rft.epage%253D6405%26rft.atitle%253DUpc2p%2Band%2BEcm22p%252C%2BDual%2BRegulators%2Bof%2BSterol%2BBiosynthesis%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.21.19.6395-6405.2001%26rft_id%253Dinfo%253Apmid%252F11533229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xOS82Mzk1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.73" data-doi="10.1002/yea.1308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Villas-Boas</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hojer-Pedersen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akesson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smedsgaard</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">a</span> <span class="cit-article-title">Global metabolite analysis of yeast: evaluation of sample preparation methods.</span> <abbr class="cit-jnl-abbrev">Yeast</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1155</span>–<span class="cit-lpage">1169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DYeast%2B%2528Chichester%252C%2BEngland%2529%26rft.stitle%253DYeast%26rft.aulast%253DVillas-B%25C3%25B4as%26rft.auinit1%253DS.%2BG.%26rft.volume%253D22%26rft.issue%253D14%26rft.spage%253D1155%26rft.epage%253D1169%26rft.atitle%253DGlobal%2Bmetabolite%2Banalysis%2Bof%2Byeast%253A%2Bevaluation%2Bof%2Bsample%2Bpreparation%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fyea.1308%26rft_id%253Dinfo%253Apmid%252F16240456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/yea.1308&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16240456&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233223200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.74" data-doi="10.1042/BJ20041162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Villas-Boas</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moxley</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akesson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephanopoulos</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span><span class="cit-comment">b</span> <span class="cit-article-title">High-throughput metabolic state analysis: the missing link in integrated functional genomics of yeasts.</span> <abbr class="cit-jnl-abbrev">Biochem. J.</abbr> <span class="cit-vol">388</span>: <span class="cit-fpage">669</span>–<span class="cit-lpage">677</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2BJournal%26rft.stitle%253DBiochem.%2BJ.%26rft.aulast%253DVillas-B%25C3%25B4as%26rft.auinit1%253DS.%2BG.%26rft.volume%253D388%26rft.issue%253DPt%2B2%26rft.spage%253D669%26rft.epage%253D677%26rft.atitle%253DHigh-throughput%2Bmetabolic%2Bstate%2Banalysis%253A%2Bthe%2Bmissing%2Blink%2Bin%2Bintegrated%2Bfunctional%2Bgenomics%2Bof%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1042%252FBJ20041162%26rft_id%253Dinfo%253Apmid%252F15667247%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1042/BJ20041162&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15667247&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229919500030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.75" data-doi="10.1074/jbc.M204707200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilcox</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balderes</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wharton</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tinkelenberg</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Transcriptional profiling identifies two members of the ATP-binding cassette transporter superfamily required for sterol uptake in yeast.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">277</span>: <span class="cit-fpage">32466</span>–<span class="cit-lpage">32472</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M204707200%26rft_id%253Dinfo%253Apmid%252F12077145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyNzcvMzYvMzI0NjYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.76" data-doi="10.1126/science.1122088"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Workman</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mak</span>  <span class="cit-name-given-names">H. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCuine</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tagne</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agarwal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A systems approach to mapping DNA damage response pathways.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">312</span>: <span class="cit-fpage">1054</span>–<span class="cit-lpage">1059</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DWorkman%26rft.auinit1%253DC.%2BT.%26rft.volume%253D312%26rft.issue%253D5776%26rft.spage%253D1054%26rft.epage%253D1059%26rft.atitle%253DA%2Bsystems%2Bapproach%2Bto%2Bmapping%2BDNA%2Bdamage%2Bresponse%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1122088%26rft_id%253Dinfo%253Apmid%252F16709784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTIvNTc3Ni8xMDU0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-other" id="cit-3.11.1979.77"><div class="cit-metadata unstructured">Wu, Z., R. A. Irizarry, R. Gentleman, F. M. Murillo, and F. Spencer, 2003 <em>A model based background adjustment for oligonucleotide expression arrays</em>, John Hopkins University, Baltimore, MD.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.78" data-doi="10.1038/nrm2787"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bork</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolution of biomolecular networks: lessons from metabolic and protein interactions.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">791</span>–<span class="cit-lpage">803</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BMolecular%2Bcell%2Bbiology%26rft.stitle%253DNat%2BRev%2BMol%2BCell%2BBiol%26rft.aulast%253DYamada%26rft.auinit1%253DT.%26rft.volume%253D10%26rft.issue%253D11%26rft.spage%253D791%26rft.epage%253D803%26rft.atitle%253DEvolution%2Bof%2Bbiomolecular%2Bnetworks%253A%2Blessons%2Bfrom%2Bmetabolic%2Band%2Bprotein%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm2787%26rft_id%253Dinfo%253Apmid%252F19851337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm2787&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19851337&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000271071200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.79" data-doi="10.1126/science.1177263"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yus</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maier</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Michalodimitrakis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Noort</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamada</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Impact of genome reduction on bacterial metabolism and its regulation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1263</span>–<span class="cit-lpage">1268</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DYus%26rft.auinit1%253DE.%26rft.volume%253D326%26rft.issue%253D5957%26rft.spage%253D1263%26rft.epage%253D1268%26rft.atitle%253DImpact%2Bof%2BGenome%2BReduction%2Bon%2BBacterial%2BMetabolism%2Band%2BIts%2BRegulation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1177263%26rft_id%253Dinfo%253Apmid%252F19965476%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ny8xMjYzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8xOTc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.80" data-doi="10.1021/jf0351873"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zangenberg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>  <span class="cit-name-given-names">H. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jorgensen</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellgren</span>  <span class="cit-name-given-names">L. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Cultivar and year-to-year variation of phytosterol content in rye (Secale cereale L.).</span> <abbr class="cit-jnl-abbrev">J. Agric. Food Chem.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">2593</span>–<span class="cit-lpage">2597</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BAgricultural%2Band%2BFood%2BChemistry%26rft.stitle%253DJournal%2Bof%2BAgricultural%2Band%2BFood%2BChemistry%26rft.aulast%253DZangenberg%26rft.auinit1%253DM.%26rft.volume%253D52%26rft.issue%253D9%26rft.spage%253D2593%26rft.epage%253D2597%26rft.atitle%253DCultivar%2Band%2Byear-to-year%2Bvariation%2Bof%2Bphytosterol%2Bcontent%2Bin%2Brye%2B%2528Secale%2Bcereale%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fjf0351873%26rft_id%253Dinfo%253Apmid%252F15113163%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/jf0351873&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15113163&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F1979.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.1979.81" data-doi="10.1038/msb.2011.80"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaga</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chumnanpuen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vemuri</span>  <span class="cit-name-given-names">G. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mapping the interaction of Snf1 with TORC1 in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">545</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DZhang%26rft.auinit1%253DJ.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D545%26rft.epage%253D545%26rft.atitle%253DMapping%2Bthe%2Binteraction%2Bof%2BSnf1%2Bwith%2BTORC1%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.80%26rft_id%253Dinfo%253Apmid%252F22068328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjcvMS81NDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzE5NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/11/1979.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/1965" title="Characterization and Prediction of Haploinsufficiency Using Systems-Level Gene Properties in Yeast" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/11/1997" title="Cell-Type–Specific Transcriptional Profiles of the Dimorphic Pathogen Penicillium marneffei Reflect Distinct Reproductive, Morphological, and Environmental Demands" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/11"><span>Volume 3 Issue 11, November 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/11.cover-source.jpg?itok=A7iZ1ztq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/1979.figures-only" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-kLVZQFkVMhA6vAxUNRMeBMeY6mx-UllHp09rEURbyog" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/11/1979" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/11/1979" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/11/1979" class="active" data-icon-position="" data-hide-link-title="0">Mapping Condition-Dependent Regulation of Lipid Metabolism in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459570" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-232fJp4bpQ8NbXO0_O_HLWRw6JE3KvreXw3yYimAyik" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/1979.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/11/1979" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-OJ8He8e0MCylzxmUAex4QveB0dXvyO7ABj8DwCQuvDQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/11/1979" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/1979.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459570" id="node459570--2" data-pisa="ggg;3/11/1979" data-pisa-master="ggg;g3.113.006601" data-apath="/ggg/3/11/1979.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Condition-Dependent Regulation of Lipid Metabolism in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Michael C. Jewett</span>, <span class="highwire-citation-author" data-delta="1">Christopher T. Workman</span>, <span class="highwire-citation-author" data-delta="2">Intawat Nookaew</span>, <span class="highwire-citation-author" data-delta="3">Francisco A. Pizarro</span>, <span class="highwire-citation-author" data-delta="4">Eduardo Agosin</span>, <span class="highwire-citation-author" data-delta="5">Lars I. Hellgren</span> and <span class="highwire-citation-author" data-delta="6">Jens Nielsen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1979-1995;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006601 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459570--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michael C. Jewett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Chemical and Biological Engineering and Chemistry of Life Processes Institute Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BC.%2BJewett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jewett%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BC.%2BJewett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christopher T. Workman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Biological Sequence Analysis, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BT.%2BWorkman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Workman%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BT.%2BWorkman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Intawat Nookaew </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Chemical and Biological Engineering, Chalmers University of Technology, SE-412 96 Göteborg, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Faculty of Engineering, King Mongkut’s University of Technology Thonburi, Bangkok 10140, Thailand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Intawat%2BNookaew%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nookaew%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIntawat%2BNookaew%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Francisco A. Pizarro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Chemical and Bioprocess Engineering, School of Engineering, Pontificia Universidad Católica de Chile, Macul, Santiago 8320000, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BA.%2BPizarro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pizarro%20FA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BA.%2BPizarro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Eduardo Agosin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Chemical and Bioprocess Engineering, School of Engineering, Pontificia Universidad Católica de Chile, Macul, Santiago 8320000, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eduardo%2BAgosin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Agosin%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEduardo%2BAgosin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Lars I. Hellgren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Biological Sequence Analysis, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BI.%2BHellgren%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hellgren%20LI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALars%2BI.%2BHellgren%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jens Nielsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Chemical and Biological Engineering, Chalmers University of Technology, SE-412 96 Göteborg, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jens%2BNielsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nielsen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJens%2BNielsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nielsenj@chalmers.se" class="" data-icon-position="" data-hide-link-title="0">nielsenj@chalmers.se</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/11/1979" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/11/1979&amp;title=Mapping%20Condition-Dependent%20Regulation%20of%20Lipid%20Metabolism%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/11/1979&amp;title=Mapping%20Condition-Dependent%20Regulation%20of%20Lipid%20Metabolism%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/11/1979&amp;title=Mapping%20Condition-Dependent%20Regulation%20of%20Lipid%20Metabolism%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/11/1979&amp;text=Mapping%20Condition-Dependent%20Regulation%20of%20Lipid%20Metabolism%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/11/1979&amp;title=Mapping%20Condition-Dependent%20Regulation%20of%20Lipid%20Metabolism%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/11/1979&amp;t=Mapping%20Condition-Dependent%20Regulation%20of%20Lipid%20Metabolism%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/11/1979&amp;title=Mapping%20Condition-Dependent%20Regulation%20of%20Lipid%20Metabolism%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/11/1979&amp;title=Mapping%20Condition-Dependent%20Regulation%20of%20Lipid%20Metabolism%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459570/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459570" id="node459570--3" data-pisa="ggg;3/11/1979" data-pisa-master="ggg;g3.113.006601" data-apath="/ggg/3/11/1979.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Mapping Condition-Dependent Regulation of Lipid Metabolism in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Michael C. Jewett</span>, <span class="highwire-citation-author" data-delta="1">Christopher T. Workman</span>, <span class="highwire-citation-author" data-delta="2">Intawat Nookaew</span>, <span class="highwire-citation-author" data-delta="3">Francisco A. Pizarro</span>, <span class="highwire-citation-author" data-delta="4">Eduardo Agosin</span>, <span class="highwire-citation-author" data-delta="5">Lars I. Hellgren</span> and <span class="highwire-citation-author" data-delta="6">Jens Nielsen</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1979-1995;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.006601 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459570--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Michael C. Jewett </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Chemical and Biological Engineering and Chemistry of Life Processes Institute Northwestern University, Evanston, Illinois 60208</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BC.%2BJewett%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jewett%20MC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BC.%2BJewett%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Christopher T. Workman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Biological Sequence Analysis, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christopher%2BT.%2BWorkman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Workman%20CT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristopher%2BT.%2BWorkman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Intawat Nookaew </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Chemical and Biological Engineering, Chalmers University of Technology, SE-412 96 Göteborg, Sweden</div><div class='author-affiliation'><span class='nlm-sup'>**</span>Faculty of Engineering, King Mongkut’s University of Technology Thonburi, Bangkok 10140, Thailand</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Intawat%2BNookaew%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nookaew%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIntawat%2BNookaew%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Francisco A. Pizarro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Chemical and Bioprocess Engineering, School of Engineering, Pontificia Universidad Católica de Chile, Macul, Santiago 8320000, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Francisco%2BA.%2BPizarro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pizarro%20FA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFrancisco%2BA.%2BPizarro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Eduardo Agosin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Chemical and Bioprocess Engineering, School of Engineering, Pontificia Universidad Católica de Chile, Macul, Santiago 8320000, Chile</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eduardo%2BAgosin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Agosin%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEduardo%2BAgosin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Lars I. Hellgren </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Center for Biological Sequence Analysis, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lars%2BI.%2BHellgren%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hellgren%20LI&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALars%2BI.%2BHellgren%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jens Nielsen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Center for Microbial Biotechnology, DTU Systems Biology, Technical University of Denmark, DK-2800 Kongens Lyngby, Denmark</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Chemical and Biological Engineering, Chalmers University of Technology, SE-412 96 Göteborg, Sweden</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jens%2BNielsen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nielsen%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJens%2BNielsen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nielsenj@chalmers.se" class="" data-icon-position="" data-hide-link-title="0">nielsenj@chalmers.se</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1979&amp;rft.epage=1995&amp;rft.atitle=Mapping%20Condition-Dependent%20Regulation%20of%20Lipid%20Metabolism%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=3&amp;rft.issue=11&amp;rft.date=2013-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Jewett%2C+Michael+C.&amp;rft.au=Workman%2C+Christopher+T.&amp;rft.au=Nookaew%2C+Intawat&amp;rft.au=Pizarro%2C+Francisco+A.&amp;rft.au=Agosin%2C+Eduardo&amp;rft.au=Hellgren%2C+Lars+I.&amp;rft.au=Nielsen%2C+Jens"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459570/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459570/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459570/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459570/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459570/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459570/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459570/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459570/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459570/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459570/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459570/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459570/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/11/1979" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/11/1979.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/11/1979.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/3/11/1979.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/3/11/1979.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
