<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/3/11/2049" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/3/11/2049.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/3/11/2049.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/3/11/2049.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/3/11/2049/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/3/11/2049.atom" />
<meta name="HW.pisa" content="ggg;3/11/2049" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Evolution After Whole-Genome Duplication: A Network Perspective" />
<meta name="DC.Identifier" content="10.1534/g3.113.008458" />
<meta name="DC.Date" content="2013-11-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2013 Zhu et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Gene duplication plays an important role in the evolution of genomes and interactomes. Elucidating how evolution after gene duplication interplays at the sequence and network level is of great interest. In this work, we analyze a data set of gene pairs that arose through whole-genome duplication (WGD) in yeast. All these pairs have the same duplication time, making them ideal for evolutionary investigation. We investigated the interplay between evolution after WGD at the sequence and network levels and correlated these two levels of divergence with gene expression and fitness data. We find that molecular interactions involving WGD genes evolve at rates that are three orders of magnitude slower than the rates of evolution of the corresponding sequences. Furthermore, we find that divergence of WGD pairs correlates strongly with gene expression and fitness data. Because of the role of gene duplication in determining redundancy in biological systems and particularly at the network level, we investigated the role of interaction networks in elucidating the evolutionary fate of duplicated genes. We find that gene neighborhoods in interaction networks provide a mechanism for inferring these fates, and we developed an algorithm for achieving this task. Further epistasis analysis of WGD pairs categorized by their inferred evolutionary fates demonstrated the utility of these techniques. Finally, we find that WGD pairs and other pairs of paralogous genes of small-scale duplication origin share similar properties, giving good support for generalizing our results from WGD pairs to evolution after gene duplication in general." />
<meta name="DC.Contributor" content="Yun Zhu" />
<meta name="DC.Contributor" content="Zhenguo Lin" />
<meta name="DC.Contributor" content="Luay Nakhleh" />
<meta name="article:published_time" content="2013-11-01" />
<meta name="article:section" content="6th Annual RECOMB/ISCB Conference on Regulatory &amp;amp; Systems Genomics" />
<meta name="citation_title" content="Evolution After Whole-Genome Duplication: A Network Perspective" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Gene duplication plays an important role in the evolution of genomes and interactomes. Elucidating how evolution after gene duplication interplays at the sequence and network level is of great interest. In this work, we analyze a data set of gene pairs that arose through whole-genome duplication (WGD) in yeast. All these pairs have the same duplication time, making them ideal for evolutionary investigation. We investigated the interplay between evolution after WGD at the sequence and network levels and correlated these two levels of divergence with gene expression and fitness data. We find that molecular interactions involving WGD genes evolve at rates that are three orders of magnitude slower than the rates of evolution of the corresponding sequences. Furthermore, we find that divergence of WGD pairs correlates strongly with gene expression and fitness data. Because of the role of gene duplication in determining redundancy in biological systems and particularly at the network level, we investigated the role of interaction networks in elucidating the evolutionary fate of duplicated genes. We find that gene neighborhoods in interaction networks provide a mechanism for inferring these fates, and we developed an algorithm for achieving this task. Further epistasis analysis of WGD pairs categorized by their inferred evolutionary fates demonstrated the utility of these techniques. Finally, we find that WGD pairs and other pairs of paralogous genes of small-scale duplication origin share similar properties, giving good support for generalizing our results from WGD pairs to evolution after gene duplication in general.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2013/11/01" />
<meta name="citation_mjid" content="ggg;3/11/2049" />
<meta name="citation_id" content="3/11/2049" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/3/11/2049" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/3/11/2049.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/3/11/2049.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/3/11/2049.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.113.008458" />
<meta name="citation_pmid" content="24048644" />
<meta name="citation_volume" content="3" />
<meta name="citation_issue" content="11" />
<meta name="citation_num_pages" content="9" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="6th Annual RECOMB/ISCB Conference on Regulatory &amp;amp; Systems Genomics" />
<meta name="citation_firstpage" content="2049" />
<meta name="citation_lastpage" content="2057" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yun Zhu" />
<meta name="citation_author_institution" content="Department of Computer Science, Rice University, Houston, Texas 77005" />
<meta name="citation_author" content="Zhenguo Lin" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Rice University, Houston, Texas 77005" />
<meta name="citation_author" content="Luay Nakhleh" />
<meta name="citation_author_institution" content="Department of Computer Science, Rice University, Houston, Texas 77005" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Rice University, Houston, Texas 77005" />
<meta name="citation_author_email" content="nakhleh@rice.edu" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=A. Bhan;citation_author=D. Galas;citation_author=T. Dewey;citation_title=A duplication growth model of gene expression networks;citation_pages=1486-1493;citation_volume=18;citation_year=2002;citation_issue=11;citation_pmid=12424120;citation_doi=10.1093/bioinformatics/18.11.1486" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=Evolution of pathogenicity and sexual reproduction in eight Candida genomes.;citation_pages=657-662;citation_volume=459;citation_year=2009;citation_issue=7247;citation_pmid=19465905;citation_doi=10.1038/nature08064" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=E. Dean;citation_author=J. Davis;citation_author=R. Davis;citation_author=D. Petrov;citation_title=Pervasive and persistent redundancy among duplicated genes in yeast.;citation_pages=e1000113-e1000113;citation_volume=4;citation_year=2008;citation_issue=7;citation_pmid=18604285;citation_doi=10.1371/journal.pgen.1000113" />
<meta name="citation_reference" content="citation_title=Evolution after Gene Duplication;citation_author=K. Dittmar;citation_author=D. Liberles;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Giaever;citation_author=A. Chu;citation_author=L. Ni;citation_author=C. Connelly;citation_author=L. Riles;citation_title=Functional profiling of the Saccharomyces cerevisiae genome.;citation_pages=387-391;citation_volume=418;citation_year=2002;citation_issue=6896;citation_pmid=12140549;citation_doi=10.1038/nature00935" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=X. He;citation_author=J. Zhang;citation_title=Rapid subfunctionalization accompanied by prolonged and substantial neofunctionalization in duplicate gene evolution.;citation_volume=169;citation_year=2005;citation_pmid=15654095;citation_doi=10.1534/genetics.104.037051" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=H. Innan;citation_author=F. Kondrashov;citation_title=The evolution of gene duplications: classifying and distinguishing between models.;citation_pages=97-108;citation_volume=11;citation_year=2010;citation_issue=2;citation_pmid=20051986;citation_doi=10.1038/nrg2689" />
<meta name="citation_reference" content="citation_title=Evolution of Protein Molecules;citation_author=T. Jukes;citation_author=C. Cantor;citation_pages=21-132;citation_year=1969" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kellis;citation_author=B. Birren;citation_author=E. Lander;citation_title=Proof and evolutionary analysis of ancient genome duplication in the yeast Saccharomyces cerevisiae.;citation_pages=617-624;citation_volume=428;citation_year=2004;citation_issue=6983;citation_pmid=15004568;citation_doi=10.1038/nature02424" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=M. Middendorf;citation_author=E. Ziv;citation_author=C. Wiggins;citation_title=Inferring network mechanisms: the Drosophila melanogaster protein interaction network.;citation_volume=102;citation_year=2005;citation_pmid=15728374;citation_doi=10.1073/pnas.0409515102" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=S. Navlakha;citation_author=C. Kingsford;citation_title=Network archaeology: uncovering ancient networks from present-day interactions.;citation_pages=e1001119-e1001119;citation_volume=7;citation_year=2011;citation_issue=4;citation_pmid=21533211;citation_doi=10.1371/journal.pcbi.1001119" />
<meta name="citation_reference" content="citation_title=Molecular Evolution and Phylogenetics;citation_author=M. Nei;citation_author=S. Kumar;citation_year=2000" />
<meta name="citation_reference" content="citation_title=Evolution by Gene Duplication;citation_author=S. Ohno;citation_year=1970" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=C. Pál;citation_author=B. Papp;citation_author=M. Lercher;citation_title=Adaptive evolution of bacterial metabolic networks by horizontal gene transfer.;citation_pages=1372-1375;citation_volume=37;citation_year=2005;citation_issue=12;citation_pmid=16311593;citation_doi=10.1038/ng1686" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=J. Paps;citation_author=J. Baguna;citation_author=M. Riutort;citation_title=Bilaterian phylogeny: a broad sampling of 13 nuclear genes provides a new Lophotrochozoa phylogeny and supports a paraphyletic basal A coelomorpha.;citation_volume=26;citation_year=2009;citation_pmid=19602542;citation_doi=10.1093/molbev/msp150" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. W. Pinney;citation_author=G. D. Amoutzias;citation_author=M. Rattray;citation_author=D. L. Robertson;citation_title=Reconstruction of ancestral protein interaction networks for the bZIP transcription factors.;citation_volume=104;citation_year=2007;citation_pmid=18077348;citation_doi=10.1073/pnas.0706339104" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=W. Qian;citation_author=X. He;citation_author=E. Chan;citation_author=H. Xu;citation_author=J. Zhang;citation_title=Measuring the evolutionary rate of protein-protein interaction.;citation_volume=108;citation_year=2011;citation_pmid=21555556;citation_doi=10.1073/pnas.1104695108" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=O. Ratmann;citation_author=O. Jørgensen;citation_author=T. Hinkley;citation_author=M. Stumpf;citation_author=S. Richardson;citation_title=Using likelihood-free inference to compare evolutionary dynamics of the protein networks of H. pylori and P. falciparum.;citation_pages=e230-e230;citation_volume=3;citation_year=2007;citation_issue=11;citation_pmid=18052538;citation_doi=10.1371/journal.pcbi.0030230" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Ruths;citation_author=L. Nakhleh;citation_title=Neutral forces acting on intragenomic variability shape the Escherichia coli regulatory network topology.;citation_volume=110;citation_year=2013;citation_pmid=23610404;citation_doi=10.1073/pnas.1217630110" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=N. Saitou;citation_title=Evolutionary rates of insertion and deletion in noncoding nucleotide sequences of primates;citation_pages=504-512;citation_volume=11;citation_year=1994;citation_issue=3;citation_pmid=8015443" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=D. Segre;citation_author=A. DeLuna;citation_author=G. Church;citation_author=R. Kishony;citation_title=Modular epistasis in yeast metabolism.;citation_pages=77-83;citation_volume=37;citation_year=2005" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=P. Spellman;citation_author=G. Sherlock;citation_author=M. Zhang;citation_author=V. Iyer;citation_author=K. Anders;citation_title=Comprehensive Identification of Cell Cycle-regulated Genes of the Yeast Saccharomyces cerevisiae by Microarray Hybridization;citation_pages=3273-3297;citation_volume=9;citation_year=1998;citation_issue=12;citation_pmid=9843569;citation_doi=10.1091/mbc.9.12.3273" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=C. Stark;citation_author=B. Breitkreutz;citation_author=T. Reguly;citation_author=L. Boucher;citation_author=A. Breitkreutz;citation_title=BioGRID: a general repository for interaction datasets;citation_pages=D535-D539;citation_volume=34;citation_year=2006;citation_issue=suppl_1;citation_pmid=16381927;citation_doi=10.1093/nar/gkj109" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. A. Teichmann;citation_author=M. M. Babu;citation_title=Gene regulatory network growth by duplication.;citation_pages=492-496;citation_volume=36;citation_year=2004;citation_issue=5;citation_pmid=15107850;citation_doi=10.1038/ng1340" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=A. Tsankov;citation_author=D. Thompson;citation_author=A. Socha;citation_author=A. Regev;citation_author=O. Rando;citation_title=The role of nucleosome positioning in the evolution of gene regulation.;citation_pages=e1000414-e1000414;citation_volume=8;citation_year=2010;citation_issue=7;citation_pmid=20625544;citation_doi=10.1371/journal.pbio.1000414" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. Wiuf;citation_author=M. Brameier;citation_author=O. Hagberg;citation_author=M. Stumpf;citation_title=A likelihood approach to analysis of network data.;citation_volume=103;citation_year=2006;citation_pmid=16682633;citation_doi=10.1073/pnas.0600061103" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. Wolfe;citation_author=D. Shields;citation_title=Molecular evidence for an ancient duplication of the entire yeast genome.;citation_pages=708-713;citation_volume=387;citation_year=1997;citation_issue=6634;citation_pmid=9192896;citation_doi=10.1038/42711" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=I. Xenarios;citation_author=D. Rice;citation_author=L. Salwinski;citation_author=M. Baron;citation_author=E. Marcotte;citation_title=DIP: the Database of Interacting Proteins;citation_pages=289-291;citation_volume=28;citation_year=2000;citation_issue=1;citation_pmid=10592249;citation_doi=10.1093/nar/28.1.289" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Zeng;citation_author=S. Hannenhalli;citation_title=Inferring evolution of gene duplicates using probabilistic models and nonparametric belief propagation.;citation_pages=S15;citation_volume=14;citation_year=2013;citation_pmid=23368094" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. Zhu;citation_author=P. Du;citation_author=L. Nakhleh;citation_title=Gene duplicability-connectivity-complexity across organisms and a neutral evolutionary explanation.;citation_pages=e44491-e44491;citation_volume=7;citation_year=2012;citation_issue=9;citation_pmid=22984517;citation_doi=10.1371/journal.pone.0044491" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Evolution After Whole-Genome Duplication: A Network Perspective" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
<meta name="twitter:description" content="Gene duplication plays an important role in the evolution of genomes and interactomes. Elucidating how evolution after gene duplication interplays at the sequence and network level is of great interest. In this work, we analyze a data set of gene pairs that arose through whole-genome duplication (WGD) in yeast. All these pairs have the same duplication time, making them ideal for evolutionary investigation. We investigated the interplay between evolution after WGD at the sequence and network levels and correlated these two levels of divergence with gene expression and fitness data. We find that molecular interactions involving WGD genes evolve at rates that are three orders of magnitude slower than the rates of evolution of the corresponding sequences. Furthermore, we find that divergence of WGD pairs correlates strongly with gene expression and fitness data. Because of the role of gene duplication in determining redundancy in biological systems and particularly at the network level, we investigated the role of interaction networks in elucidating the evolutionary fate of duplicated genes. We find that gene neighborhoods in interaction networks provide a mechanism for inferring these fates, and we developed an algorithm for achieving this task. Further epistasis analysis of WGD pairs categorized by their inferred evolutionary fates demonstrated the utility of these techniques. Finally, we find that WGD pairs and other pairs of paralogous genes of small-scale duplication origin share similar properties, giving good support for generalizing our results from WGD pairs to evolution after gene duplication in general." />
<meta name="og-title" property="og:title" content="Evolution After Whole-Genome Duplication: A Network Perspective" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/3/11/2049" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Gene duplication plays an important role in the evolution of genomes and interactomes. Elucidating how evolution after gene duplication interplays at the sequence and network level is of great interest. In this work, we analyze a data set of gene pairs that arose through whole-genome duplication (WGD) in yeast. All these pairs have the same duplication time, making them ideal for evolutionary investigation. We investigated the interplay between evolution after WGD at the sequence and network levels and correlated these two levels of divergence with gene expression and fitness data. We find that molecular interactions involving WGD genes evolve at rates that are three orders of magnitude slower than the rates of evolution of the corresponding sequences. Furthermore, we find that divergence of WGD pairs correlates strongly with gene expression and fitness data. Because of the role of gene duplication in determining redundancy in biological systems and particularly at the network level, we investigated the role of interaction networks in elucidating the evolutionary fate of duplicated genes. We find that gene neighborhoods in interaction networks provide a mechanism for inferring these fates, and we developed an algorithm for achieving this task. Further epistasis analysis of WGD pairs categorized by their inferred evolutionary fates demonstrated the utility of these techniques. Finally, we find that WGD pairs and other pairs of paralogous genes of small-scale duplication origin share similar properties, giving good support for generalizing our results from WGD pairs to evolution after gene duplication in general." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11.cover-source.jpg" />
    <title>Evolution After Whole-Genome Duplication: A Network Perspective | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__82tjGXVwuamgwcEobtpUBsLMqu40GvBmI4CZB4kYsxo__5x0JKv9MzhFwci-dmA_VhhDOSHkXTMqsZCNqRXzJ1Nw__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"OxcmDiKuShl6UwSt_eFEPgEmjLb7xUOvvH2UDGo4w2o","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459567","apath":"\/ggg\/3\/11\/2049.atom","pisa":"ggg;3\/11\/2049","ac":{"\/ggg\/3\/11\/2049.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/3\/11\/2049.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;3\/11\/2049"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;3\/11\/2049"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/3\/11\/2049":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022II\\\\\\\/9QCJh9wJBbjmzphRZJlrvhCfa93c\\\\\\\/HCmFSsbtXZUwt92KbMCZnF2G6Abl76XYD3B0FMfzRLTIlVEiurGuTOYwZIw9KzdlDlUdfupXOVNQdxPCyqr6ygaOz1BcG9m\\\\\\\/gDuddKL2S4iPd3NZrQxDmN3iMA\\\\\\\/Wz7yePQPj\\\\\\\/xITfrlaBsyhQkRFkcMdp9MhZUXUx\\\\\\\/C9u93yoL1sLRYrxp3yVzF1gi7gcGEcDqJtQSTXMT7G3Fb4ZQswts6kCgXOYrQMEFsQ+fHIAHkWCqQckyTPT6DOqEck199U2xvz\\\\\\\/6lPkSHh5vzM\\\\\\\/UQVzCtyM8ByY0N6mnaCgpL99\\\\\\\/7H5IMw7wvWiRTFYVUMcd7cc2iLH4Mvq5wRF1PWvp73a98\\\\\\\/SPIcNL9u+ggobgzLK2XDJIvVRxphKF\\\\\\\/iRbuW6jWC3nLEmiEVmfzet4H95EjfMEQ+Eu0C0fiwEztPStfas7Z8ROeEBFi91dwcidqm2AS50ye+0Vmp8xYOyskGbY6x6XOBwzJDahJLHw7fX\\\\\\\/2GCZ5yIP3MQnTEPIwaHZHLyTusaFeMmtL8RfU1IIhilYO1MaZYG\\\\\\\/sNYORO8I9GwNlHLJQk6R045whoXi5NF81ND3f3dUQpaDM98heQ0AQUzriUsXKN2rvyMgSQiMvw2I1NKMpD\\\\\\\/4E+yizBpQ26BYN5qkw2fQtFLCCh\\\\\\\/DQ9uBVExWBfGKVwTRDeG+lhBBHkyYGs5S5NvPvwku0+V0Pw0isCkfUD7zRombz2tLPneznzCYDf+JoPyOmrtzYnwvhbzXyPeh8wcHMgUT1AxUoVcWDA+O\\\\\\\/OULYgQsye\\\\\\\/tgLDNai8BXGqFLhzMB5NcZ8WvdRcDp\\\\\\\/N7PFONHN0B0HFuyJB\\\\\\\/wcIuPa+Z7olhY9ypgOaF8KHYyg3L8L2hmyVHXCB23OS0iFVo80pWb0kuwNUchOhwVqf1Frb+lv\\\\\\\/VsPwb8hLiBqWcIeVvE08IZwHOn70eYSaXZe+KcrrOaggubB4il5DS0zs5N+7aAln0ywtulnkryRMOJDGUpO0u8YwJbVh0H7l2Ri61mc1Ai6LPQ+MQBuGhbAjm1RoB9l5x+MZHk6BxR4aYoy+GXHhEzxNMUNb3hcGER0dqgywlc0h4nbsscXG\\\\\\\/2tqRs5V3ZZl6lINruEu1wd6ZjRChrLvWxfHke4V3+2vZh8iY0DKxUsrsFZEcddEu4SxP8oPYMaGRsnk9yLyc6MJNNF78ArPK6G6+4gc6wB0CVjsEAaJ\\\\\\\/h2HKvY+t0jl7V\\\\\\\/3Ohj0PKTb\\\\\\\/wyj2VAU4aDIO5RbweM4fSGATcg\\\u0022,\\\u0022iv\\\u0022:\\\u0022xkrAxaeKe2N+ciuwSTSZLA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002228d29323aca5e6e1ac11bd4b5286b5e5531a4ab7461206c8faeb32022185998c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00227nGkfE+I8OIbVbAtig6qVJNJCzTnVka+HKRLh0m7FEZBfdceeQd28uxlYFE084YeArEVoWIdiI2d3hDYgq82fVQzqfdV06z3Lj7Jor3DLwFEq\\\\\\\/rWWlpkiB9m1xwgZGZsaLGgPub0OWlaFkXd0hTwilYkHqDzjArFyoyoX0JmYd36gdRU906MwkQZSHPsffUbo+LH00Gnx85UF5rIBBWFXGkXgcw2LjaZdmX1u\\\\\\\/yolqNmSX3mxPdhNKPjfCUT7SItNVsuii85as8MI+JqLQhizzDTJCbqd6+VGaFIgomceNj0zeic8k5VRSxgXPmm+mwdAOE7qvhfPgolzyyYHihk01Wp1wkJ2OePX3\\\\\\\/vRT2dpYaj7AX\\\\\\\/eEIHDqpMZFwUPXWfODfWPmhL9Xu\\\\\\\/mqin2h\\\\\\\/QWV0JUT9gdAxVIKnTFfrK\\\\\\\/lkUd7hF3QQNSg\\\\\\\/WHKzvoYyzZQJ5gK98eYN+lWStF7+QpO1E9ltGZuDLwSEDKLLN\\\\\\\/3X4UOYz6Pm8Jfobug5x5AIFN7S6fzyfHGrmoY+DVjj9qcghQq561xXKJBfj2M7TPfapUtYVRUFmcgaKbVVqXlOKXz8YnTDgUuv\\\\\\\/IDVKG7sv5do2G8PChSfyScbUAI1hZjQtwm+lmZSn1PsqrxN9XxG4VRp9lPlNbZiSDQiPCE\\\\\\\/eDmXOAgbGAzaBjsD1veT57mzZOObtIH4\\\\\\\/wVEZowhV2OIMZF4CbdI1nF4mf1lX0+BtmKmWAmR9DXAkWRD5KgImhGQ4Taalti0BZUd2Z19l3BPtnYMZ3I\\\\\\\/JjIZLKYY5G0fxNBAAVWXAkpr9yGwhK8TqjrkhPlgSKENLLcl8rvVhBX\\\\\\\/z\\\\\\\/aF9GhTSYbnz85lI6tlBJlbf2h4fxlBWo110ylwTH8+wHz9RNKlHk1GcnO3QrTjkBBFFEVs6\\\\\\\/8pY\\\\\\\/LE3lqHGaI8rTf5U0pT14Fd6Fz\\\\\\\/ZmpUDFVynzWsFfBbYE35fzUX9+7Hct7jXTiGrT+tUDapkz7QwJ\\\\\\\/ZONBzEwzwicMqgssLpV47hM3hh0tioENUItbT8X0lYuRfgToN7pOIsJuRioiRmzjAhUiq9LbT87C46gJkcZRqdRTXCzBMP72HP65aDgQ\\\\\\\/6IbR24nhGk3EIUXHVd7ujtJnkAItTyLTRQu8++JYeq5\\\\\\\/FIpfyacNlLOKNVuqXaJ3D0P9ROge9R6yMMKrCJlV9deAlnzb7kMqRyxP+81c49XmXfR06e+0HScn\\\\\\\/UkXOUnuW0rawXcV4XiT+hUggxEwrD+Wca8esTa2mYj+BmmJ9r3aJcgXPh46UMceZ3S2XlAXO8JMj0mFGo2wQ2V8QI\\\\\\\/Qdxb8TTk+w20NyrD2OYLAZtK6Liu23gnlNUv4mE0MC8046h7DQd9+VXl3l0AcHxfpad2opJMrMC6h\\\\\\\/vHBnQJWMQOLlMOEU52r8wCIvySKWDyrlOiWS+TuBOUU0Fy1LO\\\\\\\/MCMA2kLFrdygH8HUod582\\\\\\\/Pu9lZQA70mkks+\\\\\\\/TfeKKukxnJ7vkBhmE2SCRueuHd1kMnAO4sH\\\\\\\/CnFZYFNBJoPi8hCodSrGfuea+Au0bsNZ+OyzSQZLHm6dV7BwrE0d2wYC3C50ODjrf1PxtC4x8jSPEtXqK2ZhCllv2CPcLM5Pmjom1YftBt5nui9t0ME85Dd99YKCqNwpg\\\\\\\/Dt\\\\\\\/Pp3BqM2Te5P1fP7bj717u3JFY4TTZT9Uw7drKALXtOy+LlePdoOJ0K6Zgi1EkEhDBVKJgaM5wyrrblLgH+3lrFhS21HkShLXGGDm2FWTiEEXrvFFv6MS3XnO9\\\\\\\/tNljsvrHiH6XpuWpkBohHexUysPlKxnNmBjytWaVe5gPebpZspxdoUlnIbTrl7VBhl9g0NFkPlbXN8GRwKI+2f0r3sopYGiN9qBaG4BaITJJN9f3vYY52ah0Kiul+c3xxh3KvDELSigD\\\\\\\/01e7Vtvq41A+SrwDMTVm03vaaQm8wgppDkhdUt9Es0zG2tQ2NIgdvsYecqrsJfpmFNFG1d52CI5Uc0UwrSqdR75+vxbnwYQDZ6ygMpD\\\\\\\/wIFkFpdHnSSL4Gm5QdnmYpdp8\\\\\\\/QBP0wjXkgSqVNBOl7W52nnY27dNnt576h1BMR2F0ao4bQ9310UOLHVCSyClCyeWnzMa+8VE67amGJDHKr95GLvkoFrWdzf4zUy57I0WMhPfXYPD0tYLNs+JCQweGkuLzUWtrNWNH7DVIarXhmGLJjnDm4\\\\\\\/jCBdl0s0zQuiP37H7P0grCxytKYQE7Hk7t7\\\\\\\/cu6PlPBkWJodMC5VrNH6a8OCwuft1kOjLNnvOICeL73rQRFCnRckOJ51B\\\\\\\/0A3Zjiup2Dl9GAp+xosT8MeD2emIwOX+dfTCp2vVGDw1hvL0jKabyy6k1poibNtRFrLtItbuyhwbe0pt4S4p9Tbvco0fXmBeNjavLEVqxhu8Bkn+7tPU\\\\\\\/yprZ+dj5MG3AhuX9NFlr6fp2TcnFfb2WyeDL9Z+U+ASmV6wzEMFqWlS37b8sv+ma5P02q9qjmCo+3uQXVLIGkFCyNmMUG+KXtDH9nnt7rF52u11An6H1Lw44joHfNrxB0LFtV0\\\\\\\/SAy45lNnhtk1AcEk3rHZHb\\\\\\\/nE3sQoPFFF5+3vnDQk5mH4GzJOuU6bIqoocoHDUZRqZ+tGwpzrhJOwIlBWqI8qiLa9ywQpTjW9P5duHAyeTizUfIV7pwfxjQ4wH5xVhIWKhR7ntCXOKB4OXLN7jHJjtiZFYOb2G3W43zzhTCnnjjdYhrYg6wY\\\\\\\/9Q5rG58QYjp8N8INY1X\\\\\\\/0W89aZ2KoxwnAf6BuNtt2dS5pOONyqgg0hi4dq9wukYkiMahvkf\\\\\\\/cagY94H2hg3Kk1TgzbNrwYC1PnUx8ZVNYYqhQNCIcyfrT6Pth2br98zpnYc8sr3L4N6rpZkn7ZjqeutnbZFk6S36G3SFJM\\\\\\\/NnfZOrHCFxIgGuZQh8EcbrLN3uOJOfHRd1tZiCOhuJns\\\\\\\/NOY0FUYN2CdiUfUcwRCArh2911QInlHmp5ncEuv21mHelyJIV3EVtQVXst3RNpn2ch0ZJ3SQsPlCqj2R18qzexGt6mrktdTRQAHlUyYF7QBBzfVrpHgqGzuxAys7f\\\\\\\/HSRoKyTf62S0kR05c\\\\\\\/O2r+5jtLrOQA+lpsScx+bPWnhVLsZA+esobfM2hFjCYmaVRwRanZXQcVQ4Mgqm5Zc1sk5JdYxvWDu4FjL77u+hUrvuqyNHCuT5jmbRzQgZrZ0feFSDwUG4\\\\\\\/0xMwdl7z2ltkdp2ZZalx0aGeCLP0Nhg3Pkvq\\\\\\\/XUQZI9uugD\\\\\\\/A\\\\\\\/KJDThxmOop\\\\\\\/rqIzbNBhh1kaRFQ6MsUqpUC1m5T6YMI3uLESPFMnXe9y8bX3mt5SFQYzCd2Ca1Lzxo+NuMKOJr7VimHlET8MxqYzGVYA1PZ6ZSZBinhGguHgTU2+j41HxZgEZCqscksKuIQTSG8DIIegxuAApo+e6i9\\\\\\\/H1+tOtj+AyyaJhYGQqsU7E5+5bjQIjO+UA9GLGUyi0umI2SxUupBzahSSGiYdLjycawQu3sbVy2OsKFnNMmf+vrcoNBcoaRRfboHbL97wxkSMgNC1xEgYsvhrdfVYhCFtApa1q8\\\\\\\/PAfkD4Famxd+Gi54eOATiZ+w3EiVnStDFGuhqZUw5XjNWkQ2naAMoAkteNuZVd0DXZiMTzEC0RvM26dU10x67oI\\\\\\\/EC7o8e1y5JkkaoceBXH38IVQSgfXcIAKinHwVae1XrQnzuUqkYsi0Dz5DwXWbqy3S02teKIC3xtr+aj4jrkHkvnT2J8iLPPeeCR9yaRFoUgrht+3Wa88joWDxIqSFF1lHTWQT10V4Hlw2xOSg38vxCyHNL4RnOIM02bGhNAMPWtM\\\\\\\/oi6aeGbx3KrLAYJ7Q1\\\\\\\/w0DosRiWfg81jdlTHvOXsIqdP\\\\\\\/1N3NcxWW6Z+xUOLl+ZJ6sQ6IgDt3AZ9raDremmcd+fpwvh4U3e6kjS1k+XznkCxoGphhPrins9PSahrGCPjyC7kZLmgWBMlZ1coPWyelmhvE8EWhERossbvPrZHF9\\\\\\\/k4fP67g2sAH+\\\\\\\/EMdy+HnnbFeSwLPGwjmB79H94ens\\\\\\\/sIdDVRTzLvW8VEeg3kL0Brv8jcplwFScrA678G1QlG3W+vrVP9oCIcTTYOBS4ogVTWZQb\\\\\\\/DsN4CAY\\\\\\\/PEyyLrw9VirbbTTVI5Iy0dJ+Ge5kdMW1AzAAeXUjBmgap1yigOXTa1Yei8ModpDVYcQMQ017LeTgxBEtEtRICrG+16OJQe6Mwtp6ISmNNkwKo06GAy1vzFo9dZYSf5G1cLkeBrwL8uSodo8ymOO6yb3LnQTXiPUG+suIKbqcSxjY4xF5IO7nbHjwJyrmLcqcr08QRuh36EvVrU0ZIC+fifBDDhdekm2eyNnSJyqnlx\\\\\\\/7uh7AGIw7akCBkskBLbXZ5dAIoSJw3WGw8AZO7ooVBno8lU3GwlElj6cygm9AVyAlTy\\\\\\\/nKflqTa8qt\\\\\\\/YP68dZXlT+CJz8XQN0sRGf650JuA3xQBUVRKDkW125k0nBBcX6sBzWBg\\\\\\\/4NtNLmR7DGUiYgV3WCabh4W61f8YYI+iijxFhgI3gbWbuY7BdK9Hc0PK0VllSSTLg2fR9td6tRzkWC\\\\\\\/08j8KXzV8E8dEgQuA3wybvu+dxu9cYf\\\\\\\/iZT7qAi1QLv4vegp\\\\\\\/bTGwKSgZO0mHVp7YFUOhfrP98ec+8MB1QOWwz08l7xsquv4ow4DZsPShz75KvPN2j+w9TrdxQyjvA9aboUcuqSmYbVqfBM5b5morzCbnzLgbVjHMDAQ+dZtzFBja+ukwahAqK6NvDAbwQSV5BTOAs4Yxuw8icrXoww71LAoBSkp+bYpumFlTllosKIEj0lZWQqAyiEPbpt6JUUaXeNQx9IpoUXi4lYeqs3DCUAtgZIYRicibU6cz+PUMOVdQZ\\\\\\\/BEehbJ97L3vWt7lJPuti5uy2BLiN4TLWt6lgkRISihkwL7dRnU7lOM7KHUnp7Gbbcy7XkrJ\\\\\\\/tdm7eeVwrGeDXvoixfQGnFZLRFbpiW5DGJLhw07cIc+3JQO\\\\\\\/VJBH\\\\\\\/Dowfsj2eO+S6LGa\\\\\\\/+5y\\\\\\\/Sih4pfoBLKMrhPE4+QjW8\\\\\\\/7GVkPxZtHR3x0KBookwvcP75c3ryLoub2K8TEvSMd+WY2W34NahKhwGwxMpULwFY+OUc1M3RLEiRhCxCze8RBt+vaXRmux\\\\\\\/USa8W6P6cuW++uNmpx+ZzZg707cLlHBzaYHJFymFTMMVUh\\\\\\\/hJN5QF\\\\\\\/kQjjZ71wKwmqqQXWeISHxWP2+xZOwsJiotTX0+IWR2xmosxIcgqSnrjPKFS6XZIOaxwspzp7RPmMro8\\\\\\\/NWvbrEsEqTz2FVJPE2O1V+jaqnLQfO+MpLuPPjyur\\\\\\\/+RnasxoSyeFdqdiTQxnXWMMrfNiZcCjMTdiPyR4j5S06rSJs0Wrrm6BNIZDJOOs4EdpcyuN9GhMKiOOLFlMOEBwqJyMCtbbqvLWs+RUh\\\\\\\/bIZZ3CKB68\\\\\\\/kEyW1OdQJikfzQHSTD5hUOPT4+Zz9emTe2fzIWYFCSEEFHAmaoHHU9WEiLs6zXd7x7+Y7b5+ujo6Q7MOx1yaPh4VTdjGc6Gu2RXBqEp6hk3qDh7wA2x0tTj1S22Ij3drvAlu0uMkq8bp2TCA5g628+sM4Ve\\\\\\\/4mSvYmAI9dld2U8t0PcSyj08l1ghb78\\\\\\\/CPssZWKZATbT7Xfh1nSvTzg1nH\\\\\\\/DvuH2LWfJhQm2fIoUmN+x7Ls2QWoV8ICXXYfrj2DFHZai89sihJGLQ28BsNaTA4rfB0JaFaPaO0mj5nzcHmDcikwio1iR5ydO2EmcYIxPdCZ0\\\\\\\/mfwRvy+KPCKEQhjVpGJahsGVN5r9j0P\\\\\\\/K+qdugnspJXO2IGyqf4AJDGxIDm89zO8URk47bykIxbpBHP4BIaRb4+31nuzvBvyxviT0sm+8xX9MGnSsiC0nU2d\\\\\\\/K93l72Rhq7bl7qWpJc9ng6zhiI4Z0giSkrz7YB1fesGCP29bK0MaCUxlySKKaXTWCss+f+PlMa8QWNzKJw5dO9EbtWnWQRAdIqHxaX42LYRbyWqtb4q+bcfG1x3WvanMf+3TXV1ZV0A1ChUPf0l0L15xJtlULveKf0k5WkBeTjDa68eNDGJzX3cDEa1l\\\\\\\/dwU1uAUBxvg+I+vsO4eVa+3rgARlWbkMrb+h2FtU8APq86\\\\\\\/B2pvj5psRjYkk0W2Ij8Hq36ZklXs\\\\\\\/6zwCApO3nM9+KCzbwABpxteK9wBBmQ6Q1QxeiFJ+DRmnuBt\\\\\\\/cGLZzjnkdWGQRVsrdClk4zdW7oJ24\\\\\\\/G49Hiq6U\\\\\\\/i7X7knJjfBVUV3xftpUSmJ3r5oG3oLGfvUTtJXdR9G8ZhJjG4wJU1qZvwZWJTD6Q29yUCEUh5rXe+WafbCIYnPuoWF4Xo5jQgzdfEP2\\\\\\\/YvTag8yoa63jejFQA4biuUPaEP24hL4D640lRMtCAdgt3MtJ4cEhC0VHxucg4cy3thA4MWLkwT272ZVokekCD9iXhGUpm8E\\\\\\\/LbO3sFiQJZ4quwGw\\\\\\\/IjfCr7Q0eKkfoi\\\\\\\/06+1GyxztAnhVmT3TKLmG42mZunLxQomjiPiTqqitbNnMMy+1WHzObr+QlTeTvWLH5MF8akZFdSTW3\\\\\\\/nnwmiUR2HAK\\\\\\\/BUbKu5pakAFKnydOOBRfs8c+tb91HLlTUxEBdN8IYG8eds0026AMaj7s4eTddXl6xbFW1+\\\\\\\/tG+oHzZsCk3RtqUiutNaoxfguj4NatIohnWMPiRlpUcAhsDbTM0WZQy4hfUj38fRvFj4D\\\\\\\/3UZ6eCqohqaOJC\\\\\\\/eI2Ouf7ifltuSvKOd7IOQEbR9CiEMa1QVzlklP7\\\\\\\/NpqfSaadxVm6HQSVnmis79Mg65SkrxJh5gfNL\\\\\\\/D\\\\\\\/A55wMkaTqK\\\\\\\/lFQuAGtzEjcDD+BIWSvuiZSls7ZMDbrb92qfayE2V+PATNbGB0M6SF249sYsePxY64Iw+LyfwwcDgLUx9O5nuBn\\\\\\\/vnOsAlLLDgpIOIur0LDvCHII7WpKW7lbfxqe3ESV3iYrOBos2Zwm5wSfTfVfln743i5JC4fhxZDcQHDDEVrwR+APcZypbKxU0Wr6DBGliocONtJsehfmopPoSVPsXJXFta1nq3eNbc8o1QzDGORNCJzMxh0bihM2c8ny+M+HSkXpzkCmVAdvjEHRvOwqUZrBT\\\\\\\/s6By19Bx7JFLh37C3c4bdGKCal+oxu3Pmk8qgVEBz9PaaKxePhRrlr\\\\\\\/LFWaSLDDuH3OsSvyg4e7TYsRpW\\\\\\\/lZrFEgZMWTzhe6HlKEzuMRKTTBwZHR+nHqmexrze5w7msc+GPiVt4Wf49hwXSYB4AxwxSE8gZgfeBMh3K\\\\\\\/8dbSGDK5AobH\\\\\\\/hGN+h7EiGSdnu\\\\\\\/1bGVzVlePlgIEpYGfDclMMRYxQaJM0gAXxlyoy37vhcdJidguWhcpOvAy2EuqO4WbKQdLzfnN9g0LkUTR+GDvt2iyZxGNpyf4IYr5zv86SbaHzvR6rvrvLeKJArt7irTSv0tLGFt8i\\\\\\\/CS0gPpw8mzbVOehaMjPVKZjUv+ZqiMPHE61mQLN3dwt6RJJuiWj38lSL+Kvn6VwF5c7WdDvFITnCtAvvRyfx\\\\\\\/7ygRrfEILWGhGHbSGgLOAowt2q+FOLQPmlJD9Jwt5IuLtHyBzwCv+b22N73m16fVT6iHrLZFl0Wprpcihs9UC10sr2OYy8xT5KVon3RW4Qwfs\\\\\\\/gqekqx5hlWRUoCOYwXBpEf\\\\\\\/ScfXAvIEA7UevoQMfEap2Nvq6uoybyGQ8ghGCkgR3YOi9bTaVnU2TSm96FWcDmzsJzBSQcddBU9S5Mo2rHRnXQC4ik6Z2hEAKYmXdwNB1SaEhC85FFP8SgEdND9Jswb7cNeLsLx\\\\\\\/DOYjYHM0XuyJ1LE4mtO1IFyQjf\\\\\\\/bpxf9RzY1eD0gFuX9BZ9txFvBtHrB31SCMWSabTqXK9OeA3T3IjsUX927poBbiTfsNbgRw9pDHUB03huN2L5\\\\\\\/HqBTElNflSXEUC27qsYEduecW9cxVlWV3UaY5NbmCJ1wB4L2bwUxBBwKI009csTi6ZoTRnZ7NH42eXiuE5FeVLwoqiz7e+PL766libFeZ0A3iW\\\\\\\/qWKlFMwkeUd3eMJaTXUsV+yRSXeOMZVyD5\\\\\\\/4+cvcgsaCbRLsp8DxIK55yr88uvkJhuV5nDndaRmTcuD1uRs8K56Y1kXjrw9YZpqupI5NGRblg4XvAiUX5AoJBjSAZY68nHACg5OrJBkCTlk8IFl1qYgj3PDELrpAyHaWwG\\\\\\\/+539AlLegTxg3aBuOBbsdPaSXLM1IeRLvOeGeDlnfXTeBlRDu\\\\\\\/+uRbhXLCckJcd+\\\\\\\/56YNAT17tNDo9d\\\\\\\/RkQw+rPqgmDu2cwOehNacqrbeUiB93m2l+CSVAPt5lZhfvyGB\\\\\\\/vOk3md3gjfBmeMqUadEChXbekSPKpr6IbVZVmfH6uOluDf\\\\\\\/xQmxnw63GgSKt6j59PKmy4HN9L\\\\\\\/Oce65cDDlKZIjtpDPQj8eEEGQcEPq1HeETJ39aU3lfpGOdPV600Ua1im9xN+a8nv1Ty7zXS5yUnEnsQaehbDiUBYx\\\\\\\/eLQ2F3fh05I5Q1b9KI3PdDMbPmDD9ZREQKihL9e9RsA3nzok8ySsgdqy+YwYCu+GJrXjxGk41wjBVcnM2y2782dfiNo+UTuXZXBbrKoJJEFqG2zRcQxCDz5eodrCI+OEoI\\\\\\\/yRZE5QiX6v8nNHFJz2SrAbHewJMdcLwA9zPbkfSCB4gi0HSSOche6xwq35vgw1UVXdug5pBUERgWbaNMXEEsFQAyfNLlackbej4EQe3BOWaPYoPvh+GxC8eFqZi\\\\\\\/ouHf3d+pV8aOQTGbbWtc44aHbc3PvQnNN2d5XC9SwpobM2eemjI0EfefOyxNFt2ewPo83aJN9Deoy\\\\\\\/i+rIG9NiBnEbNVrzQpLI6ehPSvoR420NBUBR7lXb5v8S1PU4NXQKBq+V5fsSyMwlAWXdaVyQ7wcth39kZbXpAMyiZDKzSYOIQcecvWRTtphUPZqSSTSiCNk0cvDPcKcTkcNKPCYE23XpUpS7K7aYiYc6pFpYLF7RP\\\\\\\/0ZZgT+HoT3YMLXMqgwfZblVMINLBdpFvE9kN3gXi6SKcCy6yfrgJPjz\\\\\\\/HEAfqN64N5X8XEXQtbrneC+lEc3563ySuJLh9VFvJC250J3AjvgnBNMED4aBNSXv94bfRvIlWbmOemW5TQxzTG73SZPStTj2c4ikb4J+5L\\\\\\\/X3hyiysqFk3KlKQlfevo\\\\\\\/rtVCu35LYecn4IaQfLBbGa4RJXNObj4Wbp4zeK7A7Z92CgD89cjtXNZBSs6oAPx\\\\\\\/+G8RzdEQahmhJduvZsUbysFnVTJxjNOo66QDe\\\\\\\/LxTqrUBO5\\\\\\\/CToZnQd\\\\\\\/TNZSnMVQbNn6bFBFNowS9JFG\\\\\\\/XTX+7+NwgVI7Cbn\\\\\\\/g2wm1+s8soq6M3gFtreZ0S1h1gxO1UEjcPBMVQKkVIhWFenOHNbpVrOu\\\\\\\/Mz0\\\\\\\/+DpUzPoM2mvsby2XK1jFCPmZfguXgtnahaBLGw46qwXEkxOwpnTFxPSmL7EDWEQTiQA1DMxXN3hGxiJFlD+qVV5\\\\\\\/wUf\\\\\\\/712FO4h9zSoyow\\\\\\\/gyJ5c4wB38VxMUioIqZButEOLdDKzWMycs6sm7++mj6PtTSNUVVp+WRBcdRNbGwctn2Y9Jnda9to\\\\\\\/4oVdMOfwgaptlTGIKejssXSNZ9GePDsMxqGG+i0mImKFB7Sd0jDfQVBt4UbQ0vxTLFgYMGKnpYE44MnJyV0v3xg9H4wmmdiUNVTRwiclEhBV0CLJh+Z7E2h9oD+M68OqSsCulT6w2gKh8LCOmsOPtQZMc0\\\\\\\/T2PbaCid3IQn6H4JGIq9wy35xwST1338LXzyx48g3bCJsTLGnXBpRWCYG8VWeAouWXaURuxrY7sosx2HZeHdoxCTeZHq5bMsHgHiQPApeVVhfcOM25BsB2ZaDpApSpaD3F59vDPTVytsxb8LmLv\\\\\\\/J87QHVABmkI3mZldULk51CpZgntPMCiP4kijNKGpsGwCTF3qkn0SsvUaWhsHy4gvo6E5mDpwElbngP+o3q8JcaBcKKVwOZ4lAgLogptWxj9piEMbVzYmNHjVke04aNu+Ng46ozgw3xUfSD++4mmvFLSeBFCDEN1mdT0cyd1E0Q61lv7LofIoENOs4rDMQMoD9n7XHphVrwoTYSbNEsNT2Se6Qhbz49rzSWjprj4rled81SdJU7HS5KPKnm3WXL367RVrsDMy+SWdtCUDcafYeg646B97WfJFQI5qOVewEa7UEPp25VKiFNUmZoPBVRlCw33Oih7HBc9TvKCZbcYuBeVXBP5MXtV5OGS0rOBMPC4ZrPzFlcuthFUvyys+8VwzNbDkLA4jrccA3pXoSDaHAjr\\\\\\\/4\\\\\\\/jBDBnUfsBTtNNmeC3KFMM6i1zElU3F78WRvvPqJjxqAaAX1\\\\\\\/k2Ti+HIlCUOsfzX\\\\\\\/uA\\\\\\\/OG7zFVqRHltY5ueg4eLOEfHXicINZEGSN+bZj0Sx8c3G45ecg9yWFWV6X1MC8+ZsfxbTzJd7uw8CJqFFjoCLK7Y2h2p\\\\\\\/FTKDqdQzoUAVxq+IRohJ3CoqblQWAae0\\\\\\\/lzGDd5H\\\\\\\/OBAaJwTpLNu5K2Rpc7OwfFYd1WIiQumkY0OvbT5Q4REiZY0j60kqUiSIk9oo3mKhcnzYdr21sPLPdLdcfKK57dLhYOGZvmdNV+Iu18KXwAZspsmTqfdA24KNew\\\\\\\/P50gcY\\\\\\\/7JagX\\\\\\\/Jh3VL4teSAxo4kWZxOnk8rLVhHvl9B9aMN+RBirG54Vab9ZUxjNuklnrhjuXEGMV45cmn13TL\\\\\\\/9azy4uXXpHEbbkeRfsfOkFc3ABZIWjAZQ7rHF5ucW7mYPgwJFn20xEKA3SDDXQ0\\\\\\\/QqHoPcVI2XVYK+Kdj5GPQ1qMaofinjl7ilSUN+IIZ2ABGCwqDtNBaALc9rFn0uoqs3RkkCEJvEZFfCY\\\\\\\/Y7cYIZWl7eMrCK\\\\\\\/4up97VjXQ2dT7Ws9fe1qh7yINh4eQtKuVlGaSsAFBpXliD1Xj0cFLKMzLDlIzGW9PZgyHJz4RJHnY\\\\\\\/ZJvIq1Fh7klI5soShk5nHCp3C\\\\\\\/l5Tgo7BbWJ2rwjDhI4Jju7a\\\\\\\/4c\\\\\\\/i+nj1ekJpjosHnjkm\\\\\\\/i6yYEoJl8W8ksG6qOZUNtV0eg\\\\\\\/5yY7IUgzU5GFRCLZlRNjv\\\\\\\/82KpCp60dPdqjb4IpRy9ZFfWdUlxThZZd0\\\\\\\/IPXxIf7V7KCGkTvcjcTI7y9Lqs4jQuK35ddH2v+irWTWcdUkIYxPN3lC4UnJEmmHsKYF\\\\\\\/F1BiItD+jY1MB0MQfrofMHFz+aLtkHmi5z5rViw2sF\\\\\\\/5TntGsJEUj7sUKWZt4ohrJ6KCoDJfOMplFMDn4AtvFTTUPvOtCb\\\\\\\/cWjI5E+sTccHJSIwQ83fHtMisP8VoeBWmw5SsI1V5NRCvLLqTZJDM+FWVOS0gA4cIQ2n9k9f5nDyfTEuue9qIlyc5dW8FkkpMNocBvJCON3wFIgD3jvXRD3PgUZ6EIUpHDXd88n8P4Fpc+oNgFjHL28Ee2K57HROGH4HWesFQjCuqGzUHROSAARDtFo6UnhkXWiADzwtnsUACvovtuBBLRRz+sbtAIKaPWr2X08ZFAJ3fIqT1JQ04ZH2crsa7mhR0EbmEN841X2opdVtfG6wthnmAWERWgNOdOnF8ODIYCFHLYDbNvqVQQmqMMbHuPRK4zKVJN+Ep5N1BUuiyaMJGCGj3Mg4YIakI8VFRPIvs4xRblxqYchwBTZqB6ZCZ0JHtxMjav1juYddBO8gHlTJBFQvHxq4dLBQ5Nqz0kGL1sVT9wRnVkUhHGd+DLG2AWCY2dR0G40cASg1wpMcK+qsC5JCfSDDBRwT9d\\\\\\\/KHBwAtamLU3Ri2qnP3WzBHdKj62m3VmnFRs0omdQTdPm5G9huj0CcNPETbpDRBaHF5vjJQODJfi0PJZphxlTCI6rQb5sEzPYu57xh3KFrXOIEPhmPad7Av3eHHUFbr856AHzhk6W\\\\\\\/rmDFtABUfA\\\\\\\/5oHzJ7zlbGr9gzZ841HTd9V0tQwL6s8VCpwyK4BO0DRT3TXiWFkw9IRT5Jd2dUTTiyrapkm6OwTv7KANnu2cqXEUqx46LVGUVeW8SRRc\\\\\\\/ItpdblBeIwwqbpRGwcA3kQUndBgE6tC0Hmx44Edl\\\\\\\/BiHQQAswVIvx5ALQ\\\\\\\/tQ1ntyOSQqK1XjYjbxlwkDTMuM73P\\\\\\\/QDNV8bEflneBrfP9ymGDVfpvU0sUybcItMgVtyYfH9GUJeUuPWtLn\\\\\\\/B52f1Y5zhRrZI8fdDHMKwnWvZnFyoUvfo46qcn1A0MM4L2NeaKi37xVNlLtlyidytn1tIYpfVocgCapZOPgVfFgviyGv6L1snmzVv85H1Y1vnnxDqb2kcgoT7zMREZvxsDF9132PzNDvDAR9PLVf3PkMVbWsVAU24q6FZEwJ8p89NX76xvIkIGtJPqbmTRIeU7SzRWKBQLhXoWdZYByFwZ5jjBGr\\\\\\\/HOI9EOgTS2IrueC+\\\\\\\/QNw0bXk9fdZGRi05jDgfIIwH+iWea59jUfDfDijlc55Yr7OekHWIU3lc2AqikDdNzEgAxo+\\\\\\\/kdgNlbzcOMpECpNP5QXZSB1NGthsses7mYgCsMsF1ld5z0HbUP+BJN6Y4\\\\\\\/3kTeJHZeAkPwjFf7ZeXNE6kOZ7OUNdGagH0R88L3\\\\\\\/vFWTTkzi9j3oLfXrknwQREmJ1E67fdedLiNJaE75LExLdBWuIj+DIjVpmBaQKNW1NyQPknd4SVC+NAL1IrPgk\\\\\\\/NoPC\\\\\\\/wm+cuG82gur0U4DzAInndYojcNgXTeBcupT2QPQLXaa\\\\\\\/+1fuWT6aptDP5oyy0ybF3fZ6yqlW7O28TB4KnEnzTiVwP1\\\\\\\/G\\\\\\\/h4PB6i+GC0L34ah7+AJyEuacBez+C\\\\\\\/kw\\\\\\\/oZO+rDS95v63B6Fmh7psaB\\\\\\\/BvbKB8Nh7JaVefR45P\\\\\\\/EQ0gz6g558GB7eZksGonX6NtNsEBK0Y3S2dV\\\\\\\/HL4W5U\\\\\\\/Fy8GP6jozSbvFHkf\\\\\\\/XJKH2JfI9ZhENKznmQIRsTHP\\\\\\\/+CVuwTvGEcnc+901R7mA7TSFd7eTlPRAyYM7HEpn66sTPiLGcOoFgMbhJeBVulzFfl\\\\\\\/4cdJpFBYbTJgEfaUirlvIF3GMFTB\\\\\\\/ZfekPLPeQfJMhZ04Gm8TALRA8uXaLYxkvoqbtuc6ncGLyhugLSwRb6gso30d4816Bd05+X5AHUpPRHSsnHbkhQ8f+9Ks7ev5TpV4kmdiC6ez604sEXe5GaI6HusKf+Y8BcYYRY2HpAHsdXU+hu0OPLtCjNDx2Ve\\\\\\\/s+hf\\\\\\\/m+nPUCT5NbhpT5l3lDZhpPJ\\\\\\\/N3z08If8KcEhB\\\\\\\/EpUHTKcUuhC1NdKCWAeifd8PVfe9nsBGN9c1QiA8RHeT81fz22f3kpf+HuxCawmdaXDaCFuc\\\\\\\/N8FYbImFqyGcmWiNWBW176cTZgGhqsQ+AakV1eoKburWDWlegC\\\\\\\/FUiOOLPnGgd2YVK1+1Q6qgx50\\\\\\\/XLAJk9oli7QuIA6oR+j\\\\\\\/VAzP4g7qL2japMsSns6JTrroCMYngtgl0sGr3TlqfainpBYJVd69wkE48Nt03TzHW8\\\\\\\/Rj1kDlsJmH03jMO4dWQ+64SOJDBUzeidtB4fGgcIlw+D+5lQe30kNUkSLCsAKZyywUnnPQxdSnR2Aq4pGKuEOiopLYqtvB6e\\\\\\\/FePdrWFh6S1JX2Pqs8KyNz0dB3XPOOh0yj2ii4nQ0g9xB0G\\\\\\\/oxSrmqAbldKvreQ2WaBH4E6MLPczwsZWK03Z+rFBJUUM3+BGqzTuate0qF3\\\\\\\/BzgDHV\\\\\\\/+98z+X04klCWdXTou8J1jl2HnMSUFQdACK\\\\\\\/PwWF0oF4HJWqgjsNPHah+qoEnKSFsUVnXM4BZypHvApNK5jHv100x30HJdmKO5QxJ9OuT7ChiY4k0wdYB7g\\\\\\\/+9+IQ4shw8yqW5Wx+J2k5URCWgN27HfnZM\\\\\\\/sIud+F\\\\\\\/GSHUOv3P2C9DAfYXKvmUFLySeuf4sUf7ck1RWXfZyMuE0aVH1\\\\\\\/FmUFAHsVP+pXf8HgHnNqZqZUIdctT\\\\\\\/ywXSteIFM5HGEmS\\\\\\\/kz2zqkA\\\\\\\/J5QSzS5ngdCURRegsfhXM7qoaZRWUtEcYmSecpxi4LJkl1WjQM8stEX9fRQ7Bep+Tq9MW+t0sdzzKJ9VdtBvhiAWhfIU0SAEfRSiVRMHUdR73iGPkri4mmvCQTUnQtUVYpYSZJJxak\\\\\\\/SwUUlu\\\\\\\/pZ3v3NzzedIYdbYI2mcstqST885Gc7XIHAQsaUfDxTEVmD1s5OeNMlDiehtIeEaUDHHSNA15VgQ3sjqlVFb7YbmTqBMkk1xHalhC+DArSw4SmIPtzYVgUH\\\\\\\/6K\\\\\\\/+iZnkwGczDJOYVtCFsnoJauLypt7D\\\\\\\/vZdgb3IwWLIuYHVFoZ72SMVwuMEJvSc7RKoRs8xsb4kIzbKjMkLzfnRp2Q8e0A7SEGV3AopgJ85qLxgxwBdffanKYIyqUOAIPeuJfxEWkQlWJohZf+vIPhKYpbLVTezV9OBwpqXGfiaIJXul193aRShcKhg+qEF2gH6VWy49+gRs3vepGkywY1ZJlUWceojkYaa\\\\\\\/3XJl79lDVnunDlVusvI\\\\\\\/EXbIMuTIXI\\\\\\\/TuCU2aV3k6AzGnFTCG7c06X2fBje9riZrGlYfC+dDljlPEkp0CS9CyLiCwHW5Ue8HH5QODjE\\\\\\\/XlzmXOCDE6qBb4g2dgkvEhUJLqdFfXj5fmNnsU9mk\\\\\\\/hjj1vXqUa11icWOvCZKwZZ7hAfsNWJVvsQqK+vRhfOd\\\\\\\/t77ZDORPyI0XxLTvFtCljsjGzl+eLtdTxNgfA6ZFcQgKbWQDGTItCkgXyjipWPO+2v3fK6M+JNtIcSpc+tSK6sx5LQWAUN6XZDn1cYhHK6+L8iU7BJx9IbRIyfvR3WnSsCFTINWK+zcxx\\\\\\\/UbGoAqlKan9e\\\\\\\/bJFxKWWr++ybmzoXdxMEJtamnwxAAP\\\\\\\/2ElYJAvaRd3bIRUVaX9Drn9kz1ok9rnjz6VwVdyq6DpHomuoY43ehynIoYjCx36y4rPmfuwn0Ki1nuE9JxkkVTp60LN9muLJfswX7g3gAOE44VCxYCvNBwG23xMp8Fr4U3+MWx11qIozU8V6zyp1lNQqbEp+rrU0jZi939nSVtq68rO0mGF2b0lx7HxMfY\\\\\\\/iee9lpmY6DKk6nio2KFxTo7pBDY0\\\\\\\/vLbjUhjiFJkseaGN+vCWFXTq2JEZdEKiLu+F9bbcE7FupVMCFCnBVPSYZ\\\\\\\/iK6QYSKE9UsJ+t\\\\\\\/BRw0CVz4ZiVXkm+lV4tNiMSWaoixSYFUiJlR4beMNuPFVsWNoI17NOegli9Wmxxkp21iTfdAqe4Zna0ZFmO9ruujfiVn7ElWC40KizYQPLYq4\\\\\\\/SD365I2ND+6qgkd4w9XfquxdGD602qX8fUiYXG75s1B4zxNHvNprfG9GLtL9HNPLrfQpDYXwZeWXB9YDAQIHs1mmndUOk5m0mV937qYciiGViP+OA7hyD01pFnV3vv4ssXrcUd2mXA3QVuGTPzy8vJLYIoQQpEcqqMXxhMMdvR1TfBHd017xCv1FeiJQF37Xe88GgN0PY9qkcxH+ZAToWDpOD9dXp8I1f1DkwGVLOgr9qaLgYD4mG6CdAyGHIxyy5\\\\\\\/psIFd29xmveFphM+Xqw7hXEyTbCMDqBvnLCTvdtFo4QXOp8QMn95I4HNI5gTrBUzZjqmu1aPrdw4CSTzxfkYH05ccrEjwl2EwkNmQBTcsqnx8Q6cvv0+XuDAc0RZqjDPGEyPFTwpEvd0nS3hvLkrCDpSPXGHGKh\\\\\\\/altK0vrQ1b\\\\\\\/7Vtk\\\\\\\/DsxS+iH89f0BpwsEHkKpg95b4ItMrQxEjkoDImcvCwmX3IKk9XqE5IP+ETdz91lKxAwk4y+gjaYztn3eCTHcsh4\\\\\\\/Dl2hc8R+hB5+7zB46OM5zRbvg\\\\\\\/qHF4QUKI6Sb+zxNcVXhS9XtwwRMVLwI13kXFhijHB7eIVdmmUAKb5pJ0BrEczY4wYPySnM6ZJJCpGkNC9wPAjfnujD6KB6fB7NGOFYrJFXn0cn+bRcxhsC9rmXAAbBFQTgCoIOb776sfOuv44Ttq3gpFjXd9BN3h4shOYktnmJn+4Esfeo2fyKanhnlinLF\\\\\\\/tjbJwHQVnOQHSQwz0OxRVugtSNL0W6s2TribscxvYjEtV0VcZsP1JJoOfzRKZCSPWMBCM9Kl+27gtfVDDZ59PXVzm8KZKTvNdrx8zkwkOK9+qd2ylNoLtILGMwXglOSYDtL1\\\\\\\/Qy4E2AyJvapMCN\\\\\\\/G2L1y5Ce3Hgh\\\\\\\/6qmu1ziAMm0DFtmI4IKxnyEjuJYjRs9PVBJjvin5A8z2T6A\\\\\\\/C80FPHMNnZT19uQvqxlRzdx96nyP5hL8+dcO57lP0\\\\\\\/ef3XSdI3F\\\\\\\/Cc+WFJdzCoGw+AsyfxXnkQoLYqRovdos5bIUkqdOHt4xZIreSeNVo7AH086xdsjRBSymrB2OkJryUlXyfjE\\\\\\\/8gV5Tq3HBhsnHEZhvbAW4fWEUWS1h0\\\\\\\/HjxlDBklH2qT01ceDBC7kdZkWDCYYjnWVyQFOkoiLU0OHjSUPHIZTNtqPifpwLd8y4f14g3vRmVsnSlyBOKqzoxB8efbEIfuORp3JRFUACALBLSSnmY47rkCAcSe\\\\\\\/Nfn43561JucO1I99rEQO2WYwl3HWEZGQRMNgPZWKFnWgmZ4zmAk6Go0RbXtoMrvK48UU35Wk3Agx314SUByf+q85jnredloznLl2LiUrqFL2w2ZLiKnjbh33vDj4p8HJtp0r2XdKkeb2HGN\\\\\\\/hyCUCNej4cFqpgiP3Yl2GtsCNwV8UP3S9KsA\\\\\\\/7nJVuue2Ezpl2Npb4mNLMGbw56K4edNXzpYH8JAHcnSZYrtgdiz2mFMuShwl1To0NiXtfNrdU1eGynTsd6IOjIuZ4OpG2PLSfF3Ms+ICe\\\\\\\/qQgt1aZ9jLd1ULkj3ZMgCYlj0eyzPl9if0vRzSu2PtFESm57yGUcqyOm\\\\\\\/QLmq4AF1zohjHVtd2SGo+cTWHvWOMoCGWSpmVLXUSthffad3DtLxanYlaf8PZg9\\\\\\\/XdPW3bXhJTJuw+UHWFnaYkzHBPz5ecGkDi8\\\\\\\/aKF6XgcXNG7iQUWPfUm60wNPtXDCIUBBNj3lL1TOXN2JKueRCgXdvYf2LPNL61MXneuGSk45nP76PX2fBHuD\\\\\\\/aT6lbXLvlWEMPP2fkGYiQOIDq\\\\\\\/1u4fK4dZjRYIKmlKBjNkZ823DHKmPE2Jqf32ZKuV96je9htBFUW7BLxFcslUEGmYzHnwngUS0oJdijiM9qbDzb9rO60QEvCNgSZSIbl3yY8zefzpPLYAG16ZQE5xJY1TU7ExFomNehL8OTrxI00SfaZp0d55yMffpjwTBxi0bKAeNSScZMypmuP4lc08Oz\\\\\\\/ZYuUkZvXkNe4MixS+PENUw2UlUlsKnWXYjvhFgcheX2TITUnPdMdWmfIhlOm0TwwmA9NQzFSpKePGK6kCbhE4sLmaDOvHZJ00XbsR+aws\\\\\\\/Q7ABb1qK+0qrVddpH01ZSxyMljCleadDIobUUsHPIQ3uDVwK1ce8OeqA\\\\\\\/\\\\\\\/zuVOzu8eGpPvDiYsqAA1zM7AvhtDjkX00Iv8\\\\\\\/GJ1PE41VxtgXjHlKbZLVwDBCKXDLmcyGM38bKvWxlOfq0czEfMegEfZFJzFttGuydtkVRDUcSphQe3aQUWn4wJVsCXfL5kRItCWc\\\\\\\/7+zhKVScKHSCKqhhVpJ42Uh1hDxQplRyjZPLcidaioueUD6BXxDE5ZDpreg8moLLbzHdSlGvKFpnkLvww7lC9073sStP2NF3e5\\\\\\\/hKvRb7EMHXINZfs6y82nJ9Cxxv\\\\\\\/ysWV9cuip1HWpaXRlkpJYOxfIbwo75W8yqZK1DBPamV8hZeWPYfT7WZi0\\\\\\\/3X8ise80uC749G7jl8zVj6aEV6NsG3JK5+PPsvUzLx\\\\\\\/BQkig+s61596fhutEVPuxGBSEvjFt5xxDzxYHh4wuYUA2F6lfSi3w+Wbcm+3LiHVH0aEmk++20pGrySa9CUw6uNfE1W5oelvDOhOtgHbzKYvkPpWT4SzhtKan8CrD5fhSV5f\\\\\\\/X+u6KTUqQ0M+kZ26pIFS3hANi1HLkPXXal79NCDiX9Bd7lntUF27+gckZL\\\\\\\/Hs3x0CnIdhE0dxxnoeFpFfzfRAzaACLSj2EhYOMLdvueAijeehrtvSwuYs8tYmJHV9odO10Ly2t9aDG8PS7IarFEA0yK9U7UAfnu4Sm14CMtOh3QS4My5sRopCNsBZEMR1zozeEanSXUquBV\\\\\\\/EJYCMQGfeztEYzD2XlV2TKAgCM3VD4Q+hz1ncAWH0eFogC1PFDaCrBft3ciGTS4n2csPCwECkPtDjSX2SvISYKf\\\\\\\/DxnWzIYgUdqFcTpRBu99KEXTZ0t185WEcE0rrAqOUpnLZ93EX\\\\\\\/GwM9mxecLbS9SIOm0ezGcl4DfsQdNUPcs\\\\\\\/An3uhkYHeO6OndbvAXpCE36QlL1JsTnvDYTGyi\\\\\\\/sJ0MBB1IHxVWscQvByLtk8K5BfrPB4s6yH7QwArhTqaOafY9IxAxujI90qaoQhSyPckWgoyqds55MBgQ\\\\\\\/G0BJ8JFvKTyAB0C\\\\\\\/xBWFoKq4A5gibJBAxE3VTifbi4FYurGYytMG6x9DwmkgPBAfYgr4fwLtN0z99gvpZCh4cGEuixIelM6o8arp\\\\\\\/fGI\\\\\\\/k4c\\\\\\\/s9r2HWEt53W221gfqhQtdR0H3nYiFQ9RrOl9irqboxhpEe3UQUg2NUFp8nttfdWSSIoQzu6mkW2Lv1ulVuRIr9uCIvGLX4tHpNZQTPxH0meSBjOafOv\\\\\\\/oooeXQM52sejSH7Hyvn4qX0oR5EW\\\\\\\/NRiplEVFsZkmOBoe71TSzCcogJBxoLg19gkTnRNw8sL4SnbQzXaHMnCqNc3jtNrpYu7NisjXFqR4AVcRcGlXRwjHAJfmdPDzVJCND\\\\\\\/mYtj91RvSSTmr4orv2qugCs3QJfxvIJD6mJv6x4jpdmYpdUgUgq\\\\\\\/SJIIsMDd\\\\\\\/DpfKsGRrD2ukU2cgsVS5qy3SQ9pu\\\\\\\/vKNp\\\\\\\/UxSbyxvxWaB0eHDBuPcONpDroBr2ciMwlFnagdR\\\\\\\/oOhAiASYeoKf9PUcUjd+wM1b1n7nOywLRtsSENAJoj0e1x7c6Ep+GIACdKAHRoTJppzZU6uiJAnfHnzIJGlO2McjNQmRKRCQg1qqNHR4z5uIQRJtgSDZKyQKHXjLVAE+\\\\\\\/M5zqKvjnK1bt05NEeB4FO2JYUCni26edfabZk+++M3GLPw0yGfiK+7\\\\\\\/N3MmdPYJvZyutcTgK7qBdZ48IxaHFZhnFFUqkOPYs6A+U7Kao8CHufeYEgOwh+Io8DF8xBoBeUrAt+Op2VMISE9bDB8hw2HssFPQqoDm4sMKa2ki1uA\\\\\\\/14QLHAlHyZAti5bP2XneuUY+cW4YYLvHV8iHB3dyx90wWq6gZ0UESVBviUe238HvDujCDiZ+zka2lKu78YZxu4DEpbwgUWVDFq3MHG7Me5+IFpCFpJnBhGXGukFeKvcZY4zN4L59WQyZhO1U5ds4AYFgsl5JOt\\\\\\\/CzBe9WTXPkQwzSO4KelPKpCF7+DJuHiX8qwgzyDPhy0RBiHOu2c+Qc560vvXbHrrTwepPCvpDUTrMagUbVZzCYwtBbLNnIH9kqN6JLbBro7+1RluvwxiHHdEojKzTshMXVOLS5u2NDYmC9f6G4LLwGTD3RbIxgbZ+DSKMJLBqSCIdNhNQO90Ktb8mOZzrzOItuDQtxSWocfWtCQeLvQVr+Fkl44gWklFOJx5vLra7cOYRJJ2VsYJ9h8iM5ybbtWsrYTc+2uSsiHYEIqgd0swg5DduAy5t1rzzvRAJnNPa1X7Ox6Tdr7EHrrC0no4cS83AZttUrn0aQI5Wjj2BfeIJoooV\\\\\\\/YMjCf3BM1kdR1TmRVvJ6syWwt9PKijKRt\\\\\\\/AQSXpdOlvRwI+bol9eFWBclFTU6O9ueBnuYx0\\\\\\\/OI\\\\\\\/kXA8tuF4bOrN22SOIyBnH8R5YY9YFwb0aTO3Zx1JvdoKj5Gbo+ezo1N6UviSJ\\\\\\\/Zf5YTw8c55IW5M9hQXaTWgthDSY4HaClfw4IG7UxLY0GfCMT7ET\\\\\\\/laInlgSV5hV1GsZdtCOWQizWj6q0MQeVdrIalrHXL4STmZAkm6YnyR\\\\\\\/Ji7xfxfvwLGtZSH6GDjJpI+uapYUsogQWbqkcGDxHGmK3RIVDs4iF5TBJ9IJVQMJ18BpDVx2enrCEJHSK+Ou0r6sqwNSAuWvv31H3DtQMOVSJ4XeJhFYbsVFKFu10nsclYGooVRUAoS7Kh8wPbQfVYnKbjBWvQT4qyzT1phK\\\\\\\/YGKOzLj6Hi3pg39yD3niSzpP8pohcr0N+xPPzq7LprFQVMJoZ3ho7\\\\\\\/INGUWMPCS01tuBRSV32wnsBa5\\\\\\\/NJV\\\\\\\/RrW7m\\\\\\\/np0JgZuKUhgJiSrnMf6fie6G14Ikp9SFfKCiQ9pmLx0x7PohdANK1fFmD7xbKgkMz4hzV199gdALs6oKwWAp9EunGZhmy+DcuvTDy6ku2PVLgJDmgZllG2blx1FOM4UJ6sY1qXOOQdQBB\\\\\\\/s1ITlSYN98sZMZ2GCUwA1NCuXkhSAjFDFGFGfwG7J2WF1hoJF+jtpMB0UlyHVRUQaJeIiJCB8TiN7LDMqnXMFdN8AMTV38NFT4fjJvEUVsQWHGnuNPyd\\\\\\\/bVaEHY2InRanXHY2iJVjtN9aYcmLYFANSmJZP2a77V+vLqF7ptg0nTp9tdpxYDIsW0RwGw7bl7+2Fl3L2jK1HaauygzTWKEorn9E8vxVqaqANeCaj2\\\\\\\/9e8Oj\\\\\\\/kEi+np5mFWQKpiaS0hNaASAtnF1n7IeM6nkKxnhxaCYr3tvbbqGVuAVzhuCkE\\\\\\\/WXQCzGCKL72BsZScF\\\\\\\/HIBFuWsls2k8uuEKVdBwXBJefxTu+j\\\\\\\/ud3WtFMT\\\\\\\/htEKC3XlrUaYKUP0YW963L3OWq1Juzuedw7R21JXHIYFlUAloDywsJlgQI64NX9PnxbzPeElvDWZglNJNJPHIyZrOHyJwUxl5xVmofFAghoObDDYOoTxXuzMhAEQUEJfKGTQUj+0hXGo4eORL0M7lcaLApgfl9\\\\\\\/pszpJ\\\\\\\/1zf+eNw60l9jR7toBg7tpItslAEZ6M9pDDozguTiUbvGEDmbBHrrBTEZJW7ScRHvNu6x4uRF3O1a5XcxuEthjFBj8IsJzJfvjjoL2mj3dOXfQbez44TkxC\\\\\\\/mX+v2TdbN4lvoqVlNLzIkzNIsU99t6esAuFHaZQQkDR3UeN1t4qNenIlYgYIS7IRLiM0Qz3xzu5nD7xQJdb5SxcjVGNoTp\\\\\\\/sMtEWp0R3cZti8XfZiwgwQAWMNc8gwO2v\\\\\\\/I\\\\\\\/tQb+0fu1jV6aquVEm2ex4zBTotc6iaeGPbuBAmwNWTkRocX1m1vW\\\\\\\/VtSysPQ9Av1n86bOIKJYgCV2\\\\\\\/l7hcnhjTi50wfHXvv8LY7I9C9mqJnPBBRyRqllF\\\\\\\/Vc332jvmuPRzpGuet\\\\\\\/DAhIUBZFm3fmzReCDqj34WkFYPgcAIXiQHYrjOnd2cGNxIeJb+vQlROzXdjpMAozD5xJKaiHE5jNGv3HB4nIHSSyT9hJM8fcZH1dvlX5LdF7sOEVooRzhA3kUVZG\\\\\\\/akTpzTi0VUyLHKo0QYD5cAtUPdQHqon13nae+jME57iCn6+M+xxH2pxkoHq1k+FVPN9Zgo61Cj40ERx1UbCulDyOjqpYOV96BMuWBMR6JjkXqtMPlH\\\\\\\/XfBB3qtYUnR+NU78zYvhFpwxfY+t33z6C8Jw1wUICcZ60x8oiC2cNd6E328Oee2PDoQd1Iyh5xD8C8rVadIrVJj8\\\\\\\/6FDEHyvJTu22zm9e2P4HqkbrV7YKmzqKBHPRESdkVVYIlJpR2M5sYbS6Nd5PP7b6Q\\\\\\\/GRvxvunEUW5kihSC4I3HCAVDQ9hKd0eLYjy8VA9EfGrvYp0TVR7OJbOVV3hBnCcJKRzD5CaNl\\\\\\\/olPE2Wwr4zqXyYuHjqGBPs3vSdMUIIZj8aBdo4dyfCjJA\\\\\\\/x6Cz0\\\\\\\/OyfEzBlr1gz0pZ9eyHoZcTxGrJUZx6HPj3i4Jj48zdNk6C48b6TWBLTw\\\\\\\/KRfE9SwLc8MYyeRvOxA29kmJztpCZwbkGOS4KNri1GsIlVnOEGtAbJGUvNhyVFiAxSjsNIiSz\\\\\\\/xLAtk6n0BxqvS8eUdVSa8z60AKF7L+BPib4q6CjTd7zn7joFzfE8Il4X6KL+HpIAMooMwH7ogSJGGsB9j7h2kmrpZEn0Hgswxw\\\\\\\/wnFyPzAOKgm34T5g7DnZQoZ+Bq60VHQgWkrvCcIZ1Q0i+LKABzsnYvAWBQUZmkZtJ545G4K5pppV6LFUf5Uflpv\\\\\\\/pKgVJWUuQUbTFGUWBuJ1tsIsvm64cNJq+aLE55pHbS8R1Rj4soYtrb6Asv+LdQlVWBEedBmyA3W2XvNMT6LdlEOBVWFBc7vUALhTKrFSsm27s3nUDMMh\\\\\\\/aUBAw59sfSgd15Cxz35vHVx1s94E2VtsuHJHM9BjlUFneVggJ5j8Mn2WCqxvVmgK\\\\\\\/OcEQKuI4cx0W1FI9BXaUIF\\\\\\\/lyz+25CYHvkygQK9KTJFduRpBJ9sdgcvxa5Ece0Voeo8j6slc49qF5ytBe6aqhdFUqwQZf5tYnuhb1e718ZEM6ELe6vzlGfETG0i+aD9RuNWueQzOeQI6DIU2JprKKFxguo7XBN0C8mBvcFsyJzIQ90U9FfC0Og29yCOHM8vtdMs1kxVsjRrNQYASv7+f9n6bJHMgn+qsaVQn6U8NcRa2lIXtIgLRe4r\\\\\\\/Jbn2yGSdKYg9Jaselv5Jw4LzjDsvISXfDT12JVBjkvQou2SJxALLdLaXZC\\\\\\\/lxZvJ\\\\\\\/W1inukayPNFjCEnQqrfGtsqlGi0yMBx\\\\\\\/AKmAjjMIu1Br\\\\\\\/B0+DB54DyWX1VLQsI3uE\\\\\\\/sqMdo\\\\\\\/17EhV6cJUqOqKedKWQCBZPxjiFqQE\\\\\\\/eI6ln4dlLh8ebP1QlKJm7kUJ1QCJD0l5rEfLb189egectud+3Jt9ET9QVRShthqh9xfYnbhiKdSDPFrlFV3VuWbSft0hDLGolyKkc9w4Pr4DthpoTEz+ot3DE9RrXcKyVc5J8t9z22an8J2PW2iuA8ptlYAEGb9wwBrbmHiOUwPMgxIjiETnX2PeaO0dw3JHCRsej04xVg7cnZ3oqvxM59pvkIRPZYutN2CQoruNFfBJ8UqucWtG\\\\\\\/nzwaTP1uER3pO1qdEmlH70mIJttgU56bV0b2tYlGO8xOPkqJsLAsRDQdBmC7mlBpxHC8acTc\\\\\\\/H6LrtXGjuLZkfw8ouZgjedWmeBB7ntnDk2RT0eW+CcSZ8axfJRdjvPLYOS8QHpDZy+sdx5aOeKV8WHmIIve+AAhP+PSyLlGk+qtsf4TQp7FShZDz6t4EG4bc9U3aXgJGOLIuwgG09WvW8XdSpDmBfIgsqIlEStqqFotLkEq65qYANU8lWHP1zD8viA\\\\\\\/DQRGGGjk6eTQBn9xRpzWZcZ6SNk7Ac25\\\\\\\/jP3yZztXlN0Z0PIb0UStvFgJPQLtpZmTGKYAbw6DLcmtmer60KOo8rheWy0p6pshCl\\\\\\\/\\\\\\\/NZl3yMlRhEqR0WgaP6rT24yrAAZ0KNbb6RdqiiPnM1lnn0p8zEnmebfjN7Nhnkb\\\\\\\/d2rmZPVzEXBlW3t9vdOQ8NY2bPSgUwiuF8yky\\\\\\\/Im6+9lpcgf3DpVRhgNnFra21+HAHcSVBuNWiZzkUTMkXFl\\\\\\\/xeeZ14BhE7FmpZG2z1m165rPCXX1gKjkAiEOY1MAEXc4TJjfv5SkHr82qakVUPLpAd\\\\\\\/E+rk5hz5xl2l1V4VfkzyGvUWHTBefBBFXh6J2\\\\\\\/4MtycvtOLfAA6ThYaZLRmGk0A7DMOjs0tLXncSzXQYpYFL33qo6kZOA0+y2HRWawwEDEovyau+MfPefdwf1FxR+GurZq0ENz+XYwU8sW9d+9KCUAQd9nL7VwW9rm+i3n9rreym5hukWlmnW6lUkrSkoJDiB\\\\\\\/PxQBCPxD7VjyhfGqM6VvPLOjZxYOvVW3sBfeRqTKfim0iqUTSB555SwhjTckPjY6p3\\\\\\\/YKwV0AUFL2\\\\\\\/MpIAt61LAuMwTGkuHNCQgvfuap85JM4hJyrjc1swEVJKalIttY8KI3q21wG+h4dpu6LDliZNoaYFPikkSacgu1nDfhmWTqR1hGGsbgYtNRBlx6R1rJD3gvB\\\\\\\/xmvXU01pE67i7nAKpQN0N+n2AtCc0jjCLgUtr1TIIXfKyW+r9Sswhcoc\\\\\\\/odAvyFfTYQFXjby8KwQRe7GEhtt3ArHVBjy6x0fUMjjvOXkUuywVd+ry0f7WL0GA7+QdrUGl\\\\\\\/XrL4ibTQwjevwpmlRVJ1PjW58Cmrxh8+Yq2Wz+U8u0HMrsyMjAoYGUuT03hPRnoMxKYpxu2in7DVSOcf9nyGvG7VrWkcwuRBylzh+EF83Sa63UxOBDY\\\\\\\/DMRpG1zu6LISD2tBNC7SzrbI1PLmCwr+8edZyYcaKyUfJUPRUw41bAuBDqkGe5QQkh3hGpeye3mrhT92rEp8RH7F01YjIR46jns8zCVKFVpuIwXMH9pIS+CaUJftLoyg3SZbKveJItILb0oR15vjnrDoHRlmi+arJfP\\\\\\\/FGCgoxtnji3I8h2mPiEG1tF5zPqUTtv\\\\\\\/R8FsG8wDYx1UBazbA1j90HIqyDGgw9lrXo+Os\\\\\\\/uWx9fOAljMyqStC9ckv47P90Pd+tDwXo0VrC9wdI4e32HCgoB5kAar8W44ZRLfaNPGRFRFMK6cSDPjfXN+6bNHmgSEMZUHhCSJ6zG66mMds+gMBp1ysIsYZ\\\\\\\/++P9sfaBctN3twBFuef2THMLprCYP6ySjp6Jbe9rJ\\\\\\\/GSkvE5kHJu8QdeGh0JvKjF6Wt62vBKRRMU8UhGiHL65LYWxeoiLNdGjB8k1Sf9oxClkkugVa73tjNxdnFF5LkGmruF3Ysycu9GWkGd3d\\\\\\\/3MkZ4QaJ6sQM7gYGyLJw8W89CfEUAzVD1i+an6yOHwuVyJUXSeCUOIf4SGMSJBfevPiBx5kSsfsLqAA\\\\\\\/L9QiDnio\\\\\\\/p7uWGifotET\\\\\\\/fBKyhSSi7f9yHtGCAlgJ3CD5xwi57o7on\\\\\\\/UXPwY26qz\\\\\\\/+PLky2BAXGhZaV5vQFe9yNxfxKx+BdMh0pvn9iA9JVte4ZIE00XXpXEm+VqLS9Qr\\\\\\\/+7aBWReRlbQ6WQgifjIwuBTZxI246jxqqCLY+gYUt3JoBuJUT4qv86EuGCljkxtnnkFJ8yBF+nR5GvfxHz1e4tUwrlrJ3twRMvDIb0xIEiMG\\\\\\\/6wU\\\\\\\/YwLX5ZqdS6G50njP0iO1LC415UowjOmZk+Aa4+TNQO9T9vKGXxbBvyoLFsyL\\\\\\\/LBfhMeMqMKWR+A2LgFjhFl1E4pf\\\\\\\/VPE40ZXeEh\\\\\\\/4eYimOJF++1tX4dJa5U1wOoF2KmmxVRlAMDHJ9NTwfcK\\\\\\\/qWilBfm1jtzmiO2+uO3ra\\\\\\\/9sAgrJlf8Vi6Ss42IfLEJka0qZ5t9Hjxu6G+lOpSYRwdvAHmnHP8+O93YjL8SKRNddX+znFtAb4mXFKWv0ottQOJpYhSyxrnBkkJmjiAQD3RvsBzXXsRcSpAbPCNvB5XVvLGEXt67W+bbZmTcVKVyi6g\\\\\\\/c\\\\\\\/hpNIsr3hjrrcRx5vIJbTE5AVdAE\\\\\\\/fYkhjGb5O3O\\\\\\\/SYxynBr6eg4EaH6OsRESrqs7qFaYpH8UYJXruIrQTbF8vflvdmkaA8OKyUIPbbD08ZjePwqcSl0E1IRNRtD830DkDTb+bsrCfbL6TLKPOMT\\\\\\\/iRE2mLHC0FHJxKHDALm0SErEFM2v0rKDvLGNyXlCxhqI+6zgSW1KcRux0pZsMuixonXFRED4ktGsMnFd1OaE6AMpZIxjcyJObKd5twbcx1sadNvABLNYujFc58xvbmG+w0PWdCo92ytu0w1J1kbGet05LhGRFpp2XwS\\\\\\\/Ihr0jb80\\\\\\\/bZZ36iwMK3Nn0K4Q5Aeo4aVOvMKRBIyjYJXjrFLQwXpXzn4CgYs7EYN0a4ERiAuuIYmHSnv9niUEdYt0avEZuOb9gKFqWbMp\\\\\\\/xPHB\\\\\\\/Qvq5P+CxCWyTfyICmOox644GyLMCvCNkBSny9t7IoHoxZCl12XQN87IyApejbkQb3ljzHG6uWAw06KHz9gUL\\\\\\\/+nHB7YKSo2m+1f6B2SKlXldsfKWqvH4anPXSP3ZAGZkSdw\\\\\\\/ATsxfAuEz3BvfztYtHvcIFsZWyA+Jb9n9I6qlO9aA6O6Qy2FzwziXGKe1bKSA27\\\\\\\/QQgxX08LRJz6aMoHNXkBzE0aEOYvCOM4ZdJGemC6ZHVfp88jToihHxsoIdcQRrQRDbRRwYVAuUU8Yn6v5wC3QnzNutiuuZTP+79Hav7m7AHBDckZxxU4f5U0Eut1ws24qod6\\\\\\\/gnp+7A9YhnXsSik92CQi46lrw4fxXfJd411F6ZYzaVVQjmErpNZM+8kACFHo+udYYoB8Nltni5e+PRMxlIqXCpWbOlGcsEkucbOybmVBLquXBwCSc\\\\\\\/GwDKsUzVV0yR9xeft44JL49rkO7DDHT\\\\\\\/q5Ha7g1p3kxHm81ezjxf+qOQrWRJAgp2l\\\\\\\/bKbwK34V16eUvpnxjlaROHumhcBhTg1TmuAvVmzW45vEceI8IKsTAhaR6TnAyYJRFn2sVln+jKy5wqCZRoPx4vdz1YWwmCQdxhAax4+f\\\\\\\/1H7HFCIQkd+bbzef21RHN9WL+VLhOytMW9ubYY\\\\\\\/6SMtKAitMoAKdgCnRFyAs0YgcTjvOHSS21fVW1+0e5C9wdVIegjSkBsiynTX6uD4RIGMqSLnkQu\\\\\\\/WHCOV1mBM+6YM49aytI9EX9JJkmkWvLrGsIrQBPt8NDCsDs\\\\\\\/wSLEVxPmCWd1pjFhp59WuiDBBkKhX6dLKRXeOq+VlOKDN2uzo6ARYcMnBXfdfAgfgxDjcn4lJeGUdyIdei59e+T4NioDLTGY5gNe+TkU5INqVyB0NtxZD6oFWdFJXBW80BS8upGx\\\\\\\/m0uSOYJ9h2wbcOGScOkCEj0UTwXFu7sr+3p8+15Oc6biWD3K7XTaBDGHXGMnxUuUdiKYa+Mkbz7\\\\\\\/LG+FSWnkN67PATfapSzvxCRISUNn6pfwNzpiEOBm7MHCGrqCisLx3molsGViA9GjQyiJCD7FL7rYtIHYRWSte8PGIPqY3lNnbZdXAuFekaqyen2QVKVnlf7Igikx8RurG\\\\\\\/PVcAIEQMUFDneJ4cZtnZmzw0olJ3\\\\\\\/VgN3bjejxm53FMbvm6OiVee77nDpe7YlcuE50lK1p88u2ppaVpdw8oITRCUbgzsBsV6O8Ren9ERb018UE4VlVwLvhGSEtE5Gg722\\\\\\\/TuFttK5J7fh4JoV2AYeHl19nymGD0Y+r2VJ2Y1eHNG+IGM8DEI8D4bIIxyP8\\\\\\\/TUHzHVpQLMFJSQzHSMfatOb1pgYLIsFHhDj0vF9aB5pKd1uMbWVNjzgNf4u6AnG0+znQS+nbDmCMp1dDkySDQOMsDVPNUN3UkFoTl\\\\\\\/i\\\\\\\/r7CUtpzDzCXiV\\\\\\\/Ot+Eqb9tT2YPHcRdomSBSt3luaTc5CHEHwWvEQNCgdNRl6WuSa6orzoERkq7RvV5t8US\\\\\\\/dQHoGqsVvEi\\\\\\\/RMKpejsUKIF2ns88bIgA4BiGcgnB33MvA8oGE4T+hIIVmLFHhZ+uk8x+KNgxkYhho8G2Gf4xHNEmlFRIs+tEcta0X5xKnE8oWt9BX5NI+IwbwFqOQncEyw7lAh6NMgnq7HLbUYywTVUVakOjVEWEvPytX7eqJl\\\\\\\/oHGJrcV2dZ2VwncRKWF8M\\\\\\\/deCoTvXoIkTSgO0OknDMkul+GwCV2raEdga3NlujsHqYXOqOzdMzIBDkG0umpvO\\\\\\\/GAbPTm11HSv5En6KgHO7YkXPaUBOMXJjPPnUisUOmHSKld+DsY1Zr1tUjphJKzxrG8\\\\\\\/50gS7L3EchN+Zbw2j2zziBXfpDusdq4NNYkL\\\\\\\/4owyPzdVE9SA+tKt9ILEhYbgonbNE6YXLjdKTl3WLlD4tjFKVf5Z633i6FBcVtakNxEBY2JHOrzwkRe7hr\\\\\\\/zFnOPBbr81BcMnCxyBkXEtIY5LEWdbwX+oNnU+9qEUxCu7Lc443U7IgXMMWLqF7J2IhOApF76+GHC7apfwd8FPq4cBrKWrQUNNGBY7cJixRHkFtR78WBIQLsSmwUa3XQgxX6M4gMn8DxsqCxYs+J5vheSEoOtlJr35fUOk\\\\\\\/KK2RGpBYZXPMdGoBNBqbN9KSD7ON\\\\\\\/cU63LAU776pBONNVT74E2RKy6RB5GepFQjNr2E29RGQOGlSbxdgAqGA+9EjoiZ96c3mMlmXmEeijKRCyoHGleo7pt8ISFCnqJaBbXaRvymVCL2BA+wFyEuaKDAKgLU9fWrZ+kIKikaJRYQ2xiKej6\\\\\\\/Wp1GM1U5FAaU6QXVomSfXQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022SDbxDdd9TJLBfTGfljf9FA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022253f8e04776c55563a7f4feed2dc6a9b0d168681cb587d66a1180416d907eba8\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228U\\\\\\\/BC6zTkyb6cGro3MpeonUaE+GxEqjodWCy96izD+Jtm8hTnD3gEwgylEiWhsTNsaVYCRfsSdz7\\\\\\\/Enti9vUXR7R6M6V\\\\\\\/srqZ4GS62mpdJUF+kyS71IJeWxeQtWSe8OIGQTXyqodgWBM2OxQY5fanoaFiDhQxkVpzLtnfsHeQNhv8fYxB01R1TPjUr9NaKzCIikRNfHebw5O\\\\\\\/sCmiyorVUMieZ9GDJbhpaWtYX8Ir+NYG6UjHcHyH8Yhk14Y\\\\\\\/sjbkaefU4obBrcb23qQAgGe5XrfwgR\\\\\\\/8Q++XwW6+xuB8Tfh5eimrw5BNs2mAKFhS+\\\\\\\/0ftBAXNq8MeF+qAHpkARAr1ZJsGoERHol9TkNjqJISvwcoF0qDOS2ELHMuEvjEOldtgu7DFy\\\\\\\/hNn92FHo3scM71aQlmb+3X1MQ1C+oDnTKJhoVU\\\\\\\/XU1i9BffViArhCRq2T0svyTPvaKs23odrTBZH0xlpTwd3lDxogTX1Q+T8Q8qHxS1NdoFWczivQEef\\\\\\\/kdj+PEbR79amK+D5Vob\\\\\\\/wdJUPS+jujISpispUVq0AxSOb0yhvAM55HA5aw+YcrmLf34pC+aJ514vcJvoAHeMX410RubuomkYDu5Yl2mihg6Es7WDYo8iA9xRTfqo\\\\\\\/33tKCoz2ugCuBw3rIa9RYKr94Ck1J7qgCA+3TjJkm7saMt2Zwl8PyauHg59aARSkF8PEYoflSoIwEzlEtu\\\\\\\/6Rxqq89longA2oSr8BcZc9twqQ3ojJfoJjLgq6IJCM8YwY6c\\\\\\\/TdMboXYTdsbl+wCNpJ62PVzJAIMSLTTPCy0vK9b9aKvp1kcYt9EbBitON6nCax9QKsXhzKl5U7be1mKppInQWacG77HUop5vM0ml40B8Bt1c21OK5\\\\\\\/YbrdMeMK1Jhx8Ep9uI0ql+iEJJM+GfdmoIkUoKlgcmhfbm2GJEYdYZVpBqtcfFX+ExXodAHwybshS5PqLF7tf+HQo1fy30ywz4qeTxBo1GtC17uCSKHiy1nqPkoXJavfyc1q9d4T78+SW\\\\\\\/Z04zyGMYBXVnonDuMFA\\\\\\\/wKCfAcfIWO+NqLD9S+B2kORBuwA9BAK8LHx1dfd4roa22A6xee76MMTLluvhTlY2ps4T+s9wJwwcV1jraFOjWomlsOY0WhS0ZDsvQnFK3FwFi89ODimGYjiM3dpk0lNW1ltTQgiKMNNwqd3GRSnUaQFdTRBN+QXdI7NJ+efnrQGXQioyerffOcJFDPnZ+jB9zwdWoPMOUwowIeL0qgyYp+RTBwMeZGQLPTeolDi7YEUvdZQm+ihSXOj8EWhZTTiXW1UHL83fz+g7HBEZNG3xpmtZX2Mj2LJ2a6RS8XdWyx\\\\\\\/0LEnnYZanydYJhTMt0+wcBCA+EwLDLJ7\\\\\\\/aOmtUPoYNmG9hfF+HzFg2gMmY2xyLrGQQJT+3MuGRVfY7eIoy8Xd5BsQPHbmerzgsfBIU7taM3sm2D30Wsxg8cAbnczW0ewG1TvqP2O80S0AV5xQYv+hMBHJGVRXuoiAVszlfo1SjKjFYzv3c3HwVVkkcnObmHukT9CP9I2HZ7ARmokKW0Nbor69AHN3TWKPVtXXcPgCHBb8e9cX20HxLa7qqFp9Mj5coM7irKXngIWSRgDlGu2cw70TluU7\\\\\\\/cjEI6gHto+Px8vuF0Bdm0evI+7ys5roD2wJVtahOli3srlsuRcp8CAKk0Z67EBaVtbqApCp6XBfNx4zI912xvOzeqbASVLuOdOiHGFwPwgnzsx6mvW2+sICcuDofBj06cma\\\\\\\/FrEQrPMe+8oeFu7w9MfL+a2g7AWCvKqW2JhdQEiyFJzE22rK5Cuj3TO280Yn4\\\\\\\/MwJxgsiZQ+LNeQwBF6jHIk3ggXvcqqldc2y6hENswsQl2AC\\\\\\\/2Rp0IP\\\\\\\/4szaQOVLY1+RMKSjkGlX3AVawlWd16\\\\\\\/la0nw1OWXbhlztUM5VSxmQym\\\\\\\/2c5I4xq599NtcBqAOzggJ2cIUej61WPrZlxLnmaRrlwpKSTP3Law\\\\\\\/TSmKACDbXoBj9YVUuxlOA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022mRQ9jJlH4PKemkurphaTMA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022112614cb425d95e51cebc349de91a35098b84cafba84fa47b5d343c4defd828a\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459567 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-6th-annual-recomb-iscb-conference-on-regulatory-amp-systems-genomics">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/2049" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UsjlzD-L1kjNT_TxKnJ7nA0AsL0EbYINM5rr0UDPRnE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/2049" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-iAuX6S9gVFPpmv6DBa2fZvFlZSNV1ok00vB54dEhtq8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/2031" title="Construction of Reference Chromosome-Scale Pseudomolecules for Potato: Integrating the Potato Genome with Genetic and Physical Maps" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/11/2059" title="Molecular Phylogeography of a Human Autosomal Skin Color Locus Under Natural Selection" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459567" id="node459567" data-pisa="ggg;3/11/2049" data-pisa-master="ggg;g3.113.008458" data-apath="/ggg/3/11/2049.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evolution After Whole-Genome Duplication: A Network Perspective</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yun Zhu</span>, <span class="highwire-citation-author" data-delta="1">Zhenguo Lin</span> and <span class="highwire-citation-author" data-delta="2">Luay Nakhleh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2049-2057;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008458 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459567" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yun Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Computer Science, Rice University, Houston, Texas 77005</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yun%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYun%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zhenguo Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, Rice University, Houston, Texas 77005</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhenguo%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhenguo%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Luay Nakhleh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Computer Science, Rice University, Houston, Texas 77005</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, Rice University, Houston, Texas 77005</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luay%2BNakhleh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nakhleh%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuay%2BNakhleh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nakhleh@rice.edu" class="" data-icon-position="" data-hide-link-title="0">nakhleh@rice.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/11/2049" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459567" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459567/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/2049.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459567" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459567/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/3/11/2049.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459567" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459567/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/3/11/2049.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459567" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459567/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Gene duplication plays an important role in the evolution of genomes and interactomes. Elucidating how evolution after gene duplication interplays at the sequence and network level is of great interest. In this work, we analyze a data set of gene pairs that arose through whole-genome duplication (WGD) in yeast. All these pairs have the same duplication time, making them ideal for evolutionary investigation. We investigated the interplay between evolution after WGD at the sequence and network levels and correlated these two levels of divergence with gene expression and fitness data. We find that molecular interactions involving WGD genes evolve at rates that are three orders of magnitude slower than the rates of evolution of the corresponding sequences. Furthermore, we find that divergence of WGD pairs correlates strongly with gene expression and fitness data. Because of the role of gene duplication in determining redundancy in biological systems and particularly at the network level, we investigated the role of interaction networks in elucidating the evolutionary fate of duplicated genes. We find that gene neighborhoods in interaction networks provide a mechanism for inferring these fates, and we developed an algorithm for achieving this task. Further epistasis analysis of WGD pairs categorized by their inferred evolutionary fates demonstrated the utility of these techniques. Finally, we find that WGD pairs and other pairs of paralogous genes of small-scale duplication origin share similar properties, giving good support for generalizing our results from WGD pairs to evolution after gene duplication in general.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/whole-genome-duplication" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-duplication" rel="nofollow">whole-genome duplication</a></li><li class="kwd"><a href="/keyword/protein-networks" class="hw-term hw-article-keyword hw-article-keyword-protein-networks" rel="nofollow">protein networks</a></li><li class="kwd"><a href="/keyword/yeast" class="hw-term hw-article-keyword hw-article-keyword-yeast" rel="nofollow">yeast</a></li><li class="kwd"><a href="/keyword/duplication-rate" class="hw-term hw-article-keyword hw-article-keyword-duplication-rate" rel="nofollow">duplication rate</a></li></ul><p id="p-3">Gene duplication is a major evolutionary event both for the genome sequence and for the protein−protein interaction (PPI) network growth. It is considered to be a major contributor to shaping and refactoring the functionalities of the organism and thus has been widely studied especially in terms of its role in evolution. After the seminal work of (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Ohno 1970</a>), more and more analyses have been conducted and more models have been developed for gene duplication on the basis of ever-increasing data sources (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Dittmar and Liberles 2010</a>). Among all the studies, some focused on gene duplication from sequence level, and to estimate, for example, probabilities, timings, and rates of duplication events (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Pál <em>et al.</em> 2005</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Pinney <em>et al.</em> 2007</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Paps <em>et al.</em> 2009</a>). Some focused at the role of duplication in network evolution and proposed graph-theoretic models of network growth, such as the duplication-attachment model (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Wiuf <em>et al.</em> 2006</a>) and duplication-divergence model (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bhan <em>et al.</em> 2002</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Zhang <em>et al.</em> 2006</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Ratmann <em>et al.</em> 2007</a>). Several other studies also have explored how duplicated genes maintain, lose, or modify their functions (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Jukes and Cantor1969</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Gibson and Goldberg 2008</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Innan and Kondrashov 2010</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Li <em>et al.</em> 2012</a>).</p><p id="p-4">From a network (<em>e.g.</em>, protein−protein interaction, or PPI, network) perspective, gene duplication results in the birth of new gene copy whose connections in the network are identical to those of the ancestral copy immediately before duplication. After gene duplication, because of the accumulation of different mutations on each of the duplicated pair, gain and loss of PPI connections in the network would be expected. However, little is known about how mutations at the sequence level of a duplicate gene pair would affect the evolution of an interaction network. <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Qian <em>et al.</em> (2011)</a> experimentally examined 87 potential interactions between <em>Kluyveromyces waltii</em> proteins, whose one-to-one orthologs in the related budding yeast <em>Saccharomyces cerevisiae</em> have been reported to interact. In their study, duplicated genes are avoided to obtain the one-to-one correspondence in two different species. In other words, while this study considered network evolution and its rate, it focused on orthologs and deliberately excluded paralogs.</p><p id="p-5">Given the central role that duplication plays in the evolution of interaction networks, it would be interesting to understand how networks shed light on the evolution of gene duplicates, and how to estimate evolutionary rates of network evolution by using duplicated genes. To investigate these issues, we focus on the whole-genome duplication (WGD) in yeast. An ancestor of <em>S. cerevisiae</em> underwent a WGD event (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Wolfe and Shields 1997</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Kellis <em>et al.</em> 2004</a>). Only approximately 10% of WGD gene pairs (550 pairs) are still present in the extant <em>S. cerevisiae</em> genome (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Kellis <em>et al.</em> 2004</a>). Because the duplication of these survived WGD gene pairs occurred at the same time and their sequence evolved at potentially different rates, these WGD gene pairs can be used as ideal subjects to learn how the evolution rate varies among different gene duplication pairs at both sequence level and network level.</p><p id="p-6">Here, we investigated the evolutionary rates of the different WGD pairs and found some variations in these rates, although within a small range. Correlating these rates with sequence, network, and fitness data, we found that gene expression and fitness correlate strongly with evolutionary rates of WGD duplicates. As essentiality and redundancy of genes interplay with expression and fitness effects, we set out to understand this interplay using WGD pairs. We first established rates of gain/loss of network interactions by using sequence divergence. We also developed a model of correlation between sequence divergence and network divergence, which captures the synchronized evolution at the sequence and network levels. Then, we used network local topologies (neighborhoods of WGD pairs) as proxies for functional similarity and divergence. On the basis of this connection between local topologies and functional similarities, we developed an expectation-maximization algorithm and learned the evolutionary fates of WGD pairs and correlated them with epistatic effects. Our results reveal the extent of conserved functionalization (CF), subfunctionalization (SF), and neofunctionalization (NF) that ensued after WGD. Furthermore, epistatis analyses correlated well with the inferences made.</p><p id="p-7">Our results demonstrate the power of WGD as “calibrated” data points to investigate network evolution and the use of networks and their topologies to shed light on evolution after gene duplication, and in particular, after WGD. We find gene pairs that arose due to WGD have similar properties to those of gene pairs that arose due to small-scale gene duplication events. This observation further generalizes our results from evolution after WGD to evolution after duplication.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>An EM algorithm for determining the fate of WGD gene pairs</h3><p id="p-8">From a network perspective, the fate of a WGD gene pair can be inferred from the shared neighborhoods of the pair. To achieve this task, we developed an expectation-maximization (EM) that is inspired by the work of <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Zeng and Hannenhalli (2013)</a>. The original method of <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Zeng and Hannenhalli (2013)</a> characterizes function by tissue-specific gene expression level, whereas we characterize function by normalized neighborhood sizes. The approach of <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Zeng and Hannenhalli (2013)</a> does not work here because they use sequence similarity of paralogs to construct a phylogenetic tree whose branch lengths serve as a surrogate of time since duplication. We, instead, target WGD pairs in which all the genes were duplicated at the same time.</p><p id="p-9">Our EM algorithm works as follows. Let the neighborhood of paralog genes <em>g</em><sub>1</sub> and <em>g</em><sub>2</sub> be both <em>N</em><sub>0</sub> right after duplication, and be <em>N</em>(<em>g</em><sub>1</sub>) and <em>N</em>(<em>g</em><sub>2</sub>) be the neighborhoods at present. Let the size for normalization be <em>ttl</em> = |<em>N</em>(<em>g</em><sub>1</sub>) ∪ <em>N</em>(<em>g</em><sub>2</sub>)| and define<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2049/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2049/embed/mml-math-1.gif"/></noscript></span></span>Under pure CF, we expect <em>a</em> = <em>b</em> = <em>x</em> = <em>sh</em> = 1; under pure SF, we expect <em>a</em> + <em>b</em> = x = 1, <em>sh</em> = 0; and, under pure NF, we expect <em>a</em> = <em>x</em> (or <em>b</em> = <em>x</em>) and <em>a</em> + <em>b</em> = 1 &gt; <em>x</em>, <em>sh</em> = 0. We further normalize the three values by their maximum value as follows:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2049/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2049/embed/mml-math-2.gif"/></noscript></span></span>The probabilistic model for classification can be captured as: (1) SF: <em>a</em> + <em>b</em> = 1; (2) CF: <em>a</em> + 1 = 2<em>x</em>; and, (3) NF: <em>x</em> = <em>a</em>, <em>x</em> ≤ 0.5. For any given <em>x</em>, <em>y</em>, and <em>z</em> values, it is classified by a plane for the feature points.</p><p id="p-10">Suppose that under CF model, rate of losing one of the two edges originated from duplication is <em>μ<sub>d</sub></em>, and rate of gaining a new edge is <em>μ<sub>a</sub></em>; under SF model, rate of losing one of the two edges originated from duplication is <em>μ<sub>D</sub></em>, and rate of gaining a new edge is <em>μ<sub>a</sub></em>; under NF model, rate of losing one of the two edges originated from duplication is <em>μ<sub>D</sub></em>, and rate of gaining a new edge is <em>μ<sub>A</sub></em> (assuming neofunctionalizaiton is accompanied by subfunctionalization [SF]). In general, <em>μ<sub>D</sub></em> &gt; <em>μ<sub>d</sub></em> and <em>μ<sub>A</sub></em> &gt; <em>μ<sub>a</sub></em>.</p><p id="p-11">Let <em>θ</em> be the set of parameters, including all the <em>μ</em> values listed previously. Let <em>Z</em>(<em>g</em><sub>1</sub>, <em>g</em><sub>2</sub>) ∈ {<em>CF</em>, <em>SF</em>, <em>NF</em>} be the fate for WGD gene pair <em>g</em><sub>1</sub> and <em>g</em><sub>2</sub>, and <em>sh</em>(<em>g</em><sub>1</sub>, <em>g</em><sub>2</sub>) be the observed normalized shared neighborhood size (|<em>N</em>(<em>g</em><sub>1</sub>) ∩ <em>N</em>(<em>g</em><sub>2</sub>)|/<em>ttl</em>). We then apply the standard EM framework as follows:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-12">Initialize the parameters <em>θ</em> to some random values.</p></li><li id="list-item-2"><p id="p-13">Compute the best value for <em>Z</em> given these parameter values. That is, according to the probabilistic model for classification, under current <em>θ</em> value, infer the most probable fate for each WGD gene pair.</p></li><li id="list-item-3"><p id="p-14">Use the computed values of <em>Z</em> to compute a better estimate for the parameters <em>θ</em>.</p></li><li id="list-item-4"><p id="p-15">Repeat steps 2 and 3 until converge.</p></li></ol><p id="p-16">To avoid local maxima, we repeated the process with several different starting values of <em>θ</em>.</p></div></div><div class="section results" id="sec-3"><h2 class="">Results</h2><p id="p-17">All results reported herein are based on WGD pairs of genes and PPI) data from <em>S. cerevisiae</em>. The PPI data were downloaded from the DIP database (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Xenarios <em>et al.</em> 2000</a>), which has high confidence value for links (interactions). To validate our results, we also used the low-throughput links and links supported by more than a single high-throughput experiment in the BIOGrid database (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Stark <em>et al.</em> 2006</a>). The sequence and gene family data were downloaded from <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Butler <em>et al.</em> (2009)</a>.</p><div id="sec-4" class="subsection"><h3>Sequence divergence of WGD pairs</h3><p id="p-18">As we set out to use a set of whole-genome duplication pairs, or WGD pairs for short, we first inspected the variability across WGD pairs in terms of sequence divergence, mutation rates, and other properties. Consider two sequences that have diverged for time <em>t</em>, and let <em>r</em> be the mutation rate per site. Further, assume that the observed normalized distance between the two sequences is <em>p</em> (that is, <em>p</em> is the proportion of sites at which the two sequences differ). Assuming equality of substitution rates among sites and equal amino acid frequencies, we have the relationship (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Nei and Kumar 2000</a>)<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2049/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2049/embed/mml-math-3.gif"/></noscript></span></span>For <em>S. cerevisiae</em> WGD pairs, <em>t</em> is estimated to be approximately 100 million years (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Wolfe and Shields 1997</a>). Given that we can compute <em>p</em> from the WGD pairs, we can compute the mutation rate <em>r</em> for each pair of WGD gene sequences as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2049/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2049/embed/mml-math-4.gif"/></noscript></span></span>Because <em>t</em> is the same for all WGD gene pairs, in this work we will compute <em>rt</em> instead:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2049/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2049/embed/mml-math-5.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>The distribution of <em>rt</em> values of WGD pairs is given in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(A) Distribution of rt of WGD pairs with normal curve fitting (mean = 0.3268, SD = 0.1685). (B) Distribution of proportion of sequence identity for WGD pairs and pairs of other paralogs. Because the duplication time of &#x201C;other paralogs&#x201D; pairs is unknown, we do not use rt here." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1988628563" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(A) Distribution of &lt;em&gt;rt&lt;/em&gt; of WGD pairs with normal curve fitting (mean = 0.3268, SD = 0.1685). (B) Distribution of proportion of sequence identity for WGD pairs and pairs of other paralogs. Because the duplication time of &#x201C;other paralogs&#x201D; pairs is unknown, we do not use &lt;em&gt;rt&lt;/em&gt; here.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2049/F1.medium.gif" width="440" height="186"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/3/11/2049/F1.medium.gif" width="440" height="186"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463608" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">(A) Distribution of <em>rt</em> of WGD pairs with normal curve fitting (mean = 0.3268, SD = 0.1685). (B) Distribution of proportion of sequence identity for WGD pairs and pairs of other paralogs. Because the duplication time of “other paralogs” pairs is unknown, we do not use <em>rt</em> here.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">As <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> shows, a normal distribution with mean 0.3268 and SD as 0.1685 gives a good fit for the data. Notice that a big portion of WGD pairs have <em>rt</em> values that are close to 0, which means a large portion of WGD pairs do not diverge much from each other. Also notice that the overall possible value for <em>rt</em> is within a relatively small range ([0, 0.8]), which means the mutation rate for different WGD pairs are not very different from each other.</p><p id="p-21">From <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>, we can see that the <em>rt</em> values of WGD pairs can be fitted to a normal curve except for the peak at <em>rt</em> = 0. Because <em>rt</em> here is computed based on the equation 1 − <em>P</em> = <em>e</em><sup>2</sup><em><sup>rt</sup></em> and 1 − <em>p</em> is the sequence identity, we plotted the distribution of sequence identity proportions in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a> for both WGD pairs and other paralogous pairs (pairs of paralogs that are the result of a small-scale duplication event). Although non-WGD paralogous pairs have different times of duplication, the overall trend shows that WGD pairs have much greater paralog sequence identity, which could mean that either the mutation rate <em>r</em> is smaller for WGD pairs than for non-WGD pairs, or that many of the individual small-scale duplication events are more recent than the WGD event.</p><p id="p-22">One caveat of observing high level of sequence identity for WGD pairs is that WGD pairs may have gone through a significant amount of interlocus gene conversions. At least 10% of WGD pairs in yeast have experienced gene conversion and the average time length of concerted evolution is about 58~75 million years (Z. Lin, unpublished data), which could potentially result in a small shift to the right in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>. It is important to note that different non-WGD paralogous pairs originated from duplication events at different times.</p></div><div id="sec-5" class="subsection"><h3>What influences the evolutionary rates of different WGD pairs?</h3><p id="p-23">As we stated previously, it seems that the mutation rates are not very different for the different WGD pairs. Still, variability exists in the rates, and the question is: what factors play a role in this variability? To answer this question, we correlated the divergence rates of WGD pairs with three metrics: the length of gene sequences, the number of gene copies in the family, and the degree of the gene in the PPI network. The results are shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="For the set of WGD pairs, the lengths of gene sequences, the number of copies within the families, and the degree of the genes, respectively, are shown against rt." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1988628563" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;For the set of WGD pairs, the lengths of gene sequences, the number of copies within the families, and the degree of the genes, respectively, are shown against &lt;em&gt;rt&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2049/F2.medium.gif" width="440" height="142"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/3/11/2049/F2.medium.gif" width="440" height="142"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463581" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">For the set of WGD pairs, the lengths of gene sequences, the number of copies within the families, and the degree of the genes, respectively, are shown against <em>rt</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">We calculated Pearson’s correlations for data in each of the three panels. The correlation between <em>rt</em> and the gene sequence length is 0.261, with <em>P</em> = 0.0002, which implies that WGD pairs of longer gene sequences diverge more than pairs with shorter gene sequences. This finding makes sense because <em>r</em> is the mutation rate per site, and longer gene sequences accumulate more mutations and result in greater degrees of divergence between the genes involved in a WGD pair. The correlation between <em>rt</em> and the copy number is −0.071, with <em>P</em> = 0.1382, which indicates almost no correlation between the two. The correlation between <em>rt</em> and the average degree of WGD pairs is −0.135, with <em>P</em> = 0.0005, which implies that WGD pairs with greater connectivity diverge slower at the sequence level. However, this might be a case of cause-effect: certain WGD pairs evolve slower, resulting in the loss of fewer neighbors, and thus greater connectivity. Furthermore, this negative correlation between divergence and connectivity is reasonable because an increase number of mutations, particularly those in regions involved in the interactions, would result in an increased (albeit not necessarily at the same rate) loss of interactions. This agrees with recent findings on how mutation at the genomic level, combined with neutral evolutionary forces, shape emergent properties at the network level (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Ruths and Nakhleh 2013</a>) and can explain correlations between network properties and gene duplicability (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Zhu <em>et al.</em> 2012</a>).</p><p id="p-26">Furthermore, we used the shared neighborhood size as a measure of gene divergence at the network level and conducted a series of similar analyses to understand whether there is a correlation between “network-level divergence” and those properties. For a given gene <em>g</em>, we denote by <em>N<sub>t</sub></em>(<em>g</em>) the set of all neighbors of gene <em>g</em> in some protein interaction network of interest at time <em>t</em> during evolution. Consider two paralogous genes, <em>g</em><sub>1</sub> and <em>g</em><sub>2</sub>, where <em>g</em><sub>2</sub> is duplicated from <em>g</em><sub>1</sub> at time 0. We denote by <em>sh<sub>t</sub></em>(<em>g</em><sub>1</sub>, <em>g</em><sub>2</sub>) = |<em>N<sub>t</sub></em>(<em>g</em><sub>1</sub>) ∩ <em>N<sub>t</sub></em>(<em>g</em><sub>2</sub>)| denote the size of the shared neighborhoods of <em>g</em><sub>1</sub> and <em>g</em><sub>2</sub> at time <em>t</em>. In this part, because we are considering pairs of extant WGD pairs, we drop the <em>t</em> in the subscript. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows the gene length, copy number, and degree properties of individual genes as they relate to the shared neighborhood sizes of their containing WGD pairs.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="For the set of WGD pairs, the lengths of gene sequences, the number of copies within the families, and the degree of the genes, respectively, are shown against sh(g1, g2)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1988628563" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;For the set of WGD pairs, the lengths of gene sequences, the number of copies within the families, and the degree of the genes, respectively, are shown against &lt;em&gt;sh&lt;/em&gt;(&lt;em&gt;g&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;, &lt;em&gt;g&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2049/F3.medium.gif" width="440" height="152"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/3/11/2049/F3.medium.gif" width="440" height="152"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463508" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">For the set of WGD pairs, the lengths of gene sequences, the number of copies within the families, and the degree of the genes, respectively, are shown against <em>sh</em>(<em>g</em><sub>1</sub>, <em>g</em><sub>2</sub>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">We calculated Pearson’s correlations for the data. The correlation between shared neighborhood size and the gene length is 0.106, with <em>P</em> = 0.0008, and the correlation between shared neighborhood size and the average degree of the two genes is 0.558, with <em>P</em> &lt; 2.2<em>e</em>−16. These results given the impression of a much stronger correlation between WGD pairs properties and their network divergence than with their sequence divergence. However, one thing to notice is that the shared neighborhood size is highly correlated to the node degrees. If we use shared neighborhood size as a measure of network divergence, then it is possible that all the observations of shared neighborhood size are simply artifacts of degrees in the PPI network. To test this hypothesis, we computed the normalized shared neighborhood size, which is computed as the shared neighborhood size divided by the number of neighbors of either of the genes in the pair. The correlation between normalized shared neighborhood size and the gene length is 0.028, with <em>P</em> = 0.3963, the correlation between normalized shared neighborhood size and the copy number is 0.046, with <em>P</em> = 0.1612, and the correlation between normalized shared neighborhood size and the average degree of the two genes is −0.099 with <em>P</em> = 0.002. In other words, when we normalize the shared neighborhood size, none of the former observed correlations remain significant.</p><p id="p-29">Further, we correlated divergence at the sequence level with gene expression and fitness levels. For gene expression levels, we used the data from <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Spellman <em>et al.</em> (1998)</a> and (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Tsankov <em>et al.</em> (2010)</a>. These data are obtained by different groups using different experimental methods, and we apply our analysis to both data sets to validate our results. For gene fitness levels, the data are obtained from <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Giaever <em>et al.</em> (2002)</a>, who use five different media under 31 different conditions. We used the normal conditions (condition 18 and 19 in <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Giaever <em>et al.</em>2002</a>) and computed the average fitness values in the five media. Plots of <em>rt</em> values <em>vs.</em> expression and fitness levels of WGD pairs are given in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="(Left) Expression levels and (right) fitness levels of single genes as a function of the rt values of WGD pairs. For a given WGD pair, the expression levels and fitness levels of both genes are plotted individually in the corresponding rt value for their containing pair." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1988628563" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;(Left) Expression levels and (right) fitness levels of single genes as a function of the &lt;em&gt;rt&lt;/em&gt; values of WGD pairs. For a given WGD pair, the expression levels and fitness levels of both genes are plotted individually in the corresponding &lt;em&gt;rt&lt;/em&gt; value for their containing pair.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2049/F4.medium.gif" width="440" height="162"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/3/11/2049/F4.medium.gif" width="440" height="162"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463485" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-30" class="first-child">(Left) Expression levels and (right) fitness levels of single genes as a function of the <em>rt</em> values of WGD pairs. For a given WGD pair, the expression levels and fitness levels of both genes are plotted individually in the corresponding <em>rt</em> value for their containing pair.</p><div class="sb-div caption-clear"></div></div></div><p id="p-31">The correlation between <em>rt</em> and expression levels is −0.3263, with <em>P</em> &lt; 2.2<em>e</em>−16, indicating that WGD pairs that diverge faster tend to have lower expression levels. The correlation between <em>rt</em> and the fitness levels is −0.285, with <em>P</em> = 7.16<em>e</em>−7, indicating that genes that diverge faster also tends to have lower fitness levels. These strong correlations might have to do with the fates of the duplicated genes, and how redundancy, or lack thereof, created by duplication interplays with fitness effects of the gene pairs. We set out to investigate this by first establishing a connection between WGD pairs evolution and the evolution of their respective interactions in a PPI network, and then learning the fates of duplicated genes from the network topology.</p></div><div id="sec-6" class="subsection"><h3>The rate of PPI evolution as a function of sequence divergence</h3><p id="p-32">Recall the definitions of <em>N<sub>t</sub></em>(<em>g</em>) and <em>sh<sub>t</sub></em>(<em>g</em><sub>1</sub>, <em>g</em><sub>2</sub>) given previously. Furthermore, we denote by <em>d<sub>t</sub></em>(<em>g</em><sub>1</sub>, <em>g</em><sub>2</sub>) the distance between the two sequences of <em>g</em><sub>1</sub> and <em>g</em><sub>2</sub> (in terms of the number of positions they differ at). It is reasonable to assume that <em>sh</em><sub>0</sub>(<em>g</em><sub>1</sub>, <em>g</em><sub>2</sub>) = <em>N</em><sub>0</sub>(<em>g</em><sub>1</sub>) = <em>N</em><sub>0</sub>(<em>g</em><sub>2</sub>) and that <em>d</em><sub>0</sub>(<em>g</em><sub>1</sub>, <em>g</em><sub>2</sub>) = 0. As time progresses, both the sequences of <em>g</em><sub>1</sub> and <em>g</em><sub>2</sub> as well as <em>N<sub>t</sub></em>(<em>g</em><sub>1</sub>) and <em>N<sub>t</sub></em>(<em>g</em><sub>2</sub>) begin to diverge, the former due to mutations at the sequence level and the latter due to gain/loss of interactions.</p><p id="p-33">Suppose after some time <em>T</em>, we have <em>d<sub>T</sub></em> = <em>Lp</em> positions, where <em>L</em> = <em>L</em>(<em>g</em><sub>1</sub>, <em>g</em><sub>2</sub>) is the length of the aligned portion between the two sequences, and <em>p</em> is the proportion of sequence difference at this length. We discard insertions/deletions as the rate of nucleotide substitution is estimated to be orders of magnitude greater than that of insertion and deletion (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Saitou 1994</a>). Let us assume that of the <em>d</em> differences at time <em>T</em>, a proportion of <em>μ</em><sub>ℓ</sub> result in the loss of new interactions, and a proportion of <em>μ<sub>a</sub></em> result in the gain of new interactions.* That is, <em>μ</em><sub>ℓ</sub> and <em>μ<sub>a</sub></em> can be thought of as the proportions of sequence substitutions that result in the loss and gain of interactions, respectively. Assuming that <em>μ</em><sub>ℓ</sub> and <em>μ<sub>a</sub></em> are very small (which is a reasonable assumption), and that in two duplicate genes, all positions in the sequences have identical mutation rates, we obtain<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2049/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2049/embed/mml-math-6.gif"/></noscript></span></span>where <em>a</em> =| <em>sh</em><sub>0</sub>(<em>g</em><sub>1</sub>, <em>g</em><sub>2</sub>)| is the initial number of shared neighbors. The rationale for this equation is as follows. Of <em>d</em> mutations, each of the two paralogous genes gains a new edge with rate <em>μ<sub>a</sub></em>, so that the expected number of newly gained edges is <em>d</em> ⋅ <em>μ<sub>a</sub></em>. For the shared neighbors, the gain of edges needs to happen for the same neighbor of both <em>g</em><sub>1</sub> and <em>g</em><sub>2</sub> or regain a lost edge such that it can contribute to <em>sh<sub>T</sub></em>(<em>g</em><sub>1</sub>, <em>g</em><sub>2</sub>).</p><p id="p-34">Replacing <em>d</em> with <em>Lp</em> in the aforementioned formula, we obtain <em>shT</em>(<em>g</em><sub>1</sub>, <em>g</em><sub>2</sub>) = <em>a</em>(1 − <em>μ</em><sub>ℓ</sub>)<em><sup>Lp</sup></em> + <em>Lp</em> ⋅ <em>μ<sub>a</sub></em>. When <em>μ</em><sub>ℓ</sub> is very small, we have (1 − <em>μ</em><sub>ℓ</sub>)<em><sup>L</sup></em> ∼1 − <em>Lμ</em><sub>ℓ</sub>. Thus, we obtain<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2049/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/3/11/2049/embed/mml-math-7.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>As we are interested in obtaining estimates of <em>μ</em><sub>ℓ</sub> and <em>μ<sub>a</sub></em> from WGD pairs, we fit the function in Equation (2) to data obtained from WGD pairs from <em>S. cerevisiae</em> that are the result of the WGD event that occurred in yeast approximately 100 million years ago. Because different WGD pairs with the same sequence divergence have different shared neighborhood sizes, we considered both the average and maximum shared neighborhood sizes for given sequence divergence values. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> shows the results with the function fitting.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The average (left) and maximum (right) shared neighborhood sizes of WGD pairs as functions of the divergence between the pair&#x2019;s sequences. The normalized sequence distance is d(g1, g2)/L(g1, g2). The red curves are the results of fitting data to Equation (2). Estimated L&#x3BC;&#x2113; 0.9261 for the average neighborhood size case and 0.9533 for the maximum neighborhood size case. Estimated L&#x3BC;a is about 0.0001 in both cases." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1988628563" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;The average (left) and maximum (right) shared neighborhood sizes of WGD pairs as functions of the divergence between the pair&#x2019;s sequences. The normalized sequence distance is &lt;em&gt;d&lt;/em&gt;(&lt;em&gt;g&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;, &lt;em&gt;g&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;)/&lt;em&gt;L&lt;/em&gt;(&lt;em&gt;g&lt;/em&gt;&lt;sub&gt;1&lt;/sub&gt;, &lt;em&gt;g&lt;/em&gt;&lt;sub&gt;2&lt;/sub&gt;). The red curves are the results of fitting data to Equation (2). Estimated &lt;em&gt;L&#x3BC;&lt;/em&gt;&lt;sub&gt;&#x2113;&lt;/sub&gt; 0.9261 for the average neighborhood size case and 0.9533 for the maximum neighborhood size case. Estimated &lt;em&gt;L&#x3BC;&lt;sub&gt;a&lt;/sub&gt;&lt;/em&gt; is about 0.0001 in both cases.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2049/F5.medium.gif" width="440" height="175"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/3/11/2049/F5.medium.gif" width="440" height="175"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463552" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">The average (left) and maximum (right) shared neighborhood sizes of WGD pairs as functions of the divergence between the pair’s sequences. The normalized sequence distance is <em>d</em>(<em>g</em><sub>1</sub>, <em>g</em><sub>2</sub>)/<em>L</em>(<em>g</em><sub>1</sub>, <em>g</em><sub>2</sub>). The red curves are the results of fitting data to Equation (2). Estimated <em>Lμ</em><sub>ℓ</sub> 0.9261 for the average neighborhood size case and 0.9533 for the maximum neighborhood size case. Estimated <em>Lμ<sub>a</sub></em> is about 0.0001 in both cases.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">In a recent study by <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Qian <em>et al.</em> (2011)</a>, the authors experimentally examined 87 potential interactions between <em>K. waltii</em> proteins, whose one-to-one orthologs in the related budding yeast <em>S. cerevisiae</em> were reported to interact. Their estimate of the evolutionary rate of protein interactions was (2.6 ± 1.6) × 10<sup>−10</sup> per PPI per year, which is three orders of magnitude lower than the rate of protein sequence evolution measured by the number of amino acid substitutions. In other words, our analysis here provides a similar results based on a different data set. It is interesting to combine these results with the recent findings of (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Teichmann and Babu 2004</a>) who showed that about 90% of interactions in transcription regulation networks of <em>Escherichia coli</em> and <em>S. cerevisiae</em> arose due to gene duplication.</p><p id="p-37">Although our results agree well with the results of <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Qian <em>et al.</em> (2011)</a>, the approaches taken are very different. <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Qian <em>et al.</em> (2011)</a>. examined PPI divergence after speciation, whereas we examined PPI divergence after WGD. In other words, <a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Qian <em>et al.</em> (2011)</a> examined PPIs between interacting pair (<em>A</em>, <em>B</em>) and their interacting orthologs, or interlogs, (<em>A</em>′, <em>B</em>′), whereas we examined PPIs between two pairs (<em>A</em>, <em>C</em>) and (<em>A</em>′, <em>C</em>) where <em>A</em> and <em>A</em>′ are paralogs. The fact that all pairs of paralogs we consider are the result of the WGD even in <em>S. cerevisiae</em> allows us to use the event as a calibration point and make use of the fact that all pairs have exactly the same age.</p><p id="p-38">It is important to note that the results in <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a> are based on data from the DIP database of PPI networks. This database records only high-confidence links and has a relatively high false-negative rate compared with a false positive rate. We repeated the same analysis by using data from the BIOGrid database (with only links that are supported either by low-throughput experiments or by more than a single high-throughput experiment). The trends we obtained are similar to those in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, with the other difference that the data points and fitted curves are shifted up slightly. The estimated <em>μ</em><sub>ℓ</sub> and <em>μ<sub>a</sub></em> values were very close to those estimated using the DIP database.</p></div><div id="sec-7" class="subsection"><h3>The fate of WGD gene pairs</h3><p id="p-39">After gene duplication, duplicates can have different functional fates, such as maintaining the same function as the ancestral single-copy gene, developing a new function, etc. Given our previous results regarding the use of shared neighborhoods of WGD pairs to estimate the rate of divergence, we here use the neighborhoods of WGD pairs as proxies of their functional fates. For CF, the two genes in a WGD pair maintain exactly the same set of neighbors; in SF, each gene in a WGD pair maintains a subset of original neighbors, whereas the union of their neighbors equals the original set. Finally, in NF, one gene in the WGD pair develops a new set of neighbors while losing all of the duplicated neighbors. According to this strategy, pure conserved functionalization would result in a normalized shared neighborhood size equal to 1, whereas pure subfunctionalization and neofunctionalization would both result in a normalized shared neighborhood size of 0. <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> illustrates these three categories.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Three fates of a duplicated gene from a network perspective." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1988628563" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Three fates of a duplicated gene from a network perspective.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2049/F6.medium.gif" width="440" height="376"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/3/11/2049/F6.medium.gif" width="440" height="376"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463491" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-40" class="first-child">Three fates of a duplicated gene from a network perspective.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">In <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>, we show the distribution of normalized shared neighborhood sizes of WGD pairs. As <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a> shows, only a very small portion of the WGD pairs actually maintain exactly the same set of neighbors. Approximately 40% of the pairs have totally exclusive neighbors, and most of the gene pairs (60%) share some neighbors while also maintaining some different neighbors. This agrees with the widely known fact that pure SF and NF are rare, and that a large fraction of gene duplicates undergo rapid SF followed by prolonged period of NF referred to as the sub-neo-functionalization model (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">H<span class="sc">e</span> and Zhang 2005</a>).</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of normalized shared neighborhood sizes of WGD pairs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1988628563" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of normalized shared neighborhood sizes of WGD pairs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2049/F7.medium.gif" width="440" height="394"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/3/11/2049/F7.medium.gif" width="440" height="394"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463551" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-42" class="first-child">Distribution of normalized shared neighborhood sizes of WGD pairs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">To estimate the actual proportion of gene pairs whose fate is CF, SF, or NF (also, sub-neo-functionalization), we developed an EM algorithm that was inspired by <a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Zeng and Hannenhalli (2013)</a> to estimate the fates from network data (see <em>Materials and Methods</em> for full details). Using this algorithm, we estimate that approximately 7−9% of WGD pairs underwent CF, approximately 18−21% WGD pairs underwent NF, and that the rest of WGD pairs (70−75%) underwent SF.</p><p id="p-44">To further explore how these estimated fates correlate with fitness data (as we discussed previously), we categorized gene fitness of WGD pairs by their inferred types. <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Segre <em>et al.</em> (2005)</a> studied the fitness and genetic interactions in yeast on a genome scale and grouped pairs of genes into one of the three categories according to epistasis analysis. Let <em>w</em><sub>1</sub> and <em>w</em><sub>2</sub> be the effect on fitness of single-knockout of genes <em>g</em><sub>1</sub> and <em>g</em><sub>2</sub>, respectively, and let <em>w</em><sub>12</sub> be the effect on fitness of double-knockout of both <em>g</em><sub>1</sub> and <em>g</em><sub>2</sub>. Let <em>e</em> = <em>w</em><sub>12</sub> − <em>w</em><sub>1</sub> ⋅ <em>w</em><sub>2</sub>. By inspecting the <em>e</em> values for the different WGD pairs, each pair can be categorized as “no epistasis” (<em>e</em> = 0), “aggravating” (<em>e</em> &lt; 0), or “buffering” (<em>e</em> &gt; 0). We obtained the knockout fitness data from <a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Segre <em>et al.</em> (2005)</a> and inspected the epistasis status of the three WGD pair groups (CF, NF, and SF).</p><p id="p-45">For all 550 WGD pairs, only 182 pairs have both PPI data for inferring duplication type based on our methodology and data from epistasis analysis. The values of <em>w</em><sub>12</sub> and <em>w</em><sub>1</sub> ⋅ <em>w</em><sub>2</sub> for WGD pairs in the three groups are shown in <a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a>.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fitness effects of double-knockouts of WGD pairs in the groups (A) CF, (B) NF, and (C) SF. The dashed line corresponds to no epistasis, whereas the regions above and below the line correspond to buffering and aggravating epistasis, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1988628563" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Fitness effects of double-knockouts of WGD pairs in the groups (A) CF, (B) NF, and (C) SF. The dashed line corresponds to no epistasis, whereas the regions above and below the line correspond to buffering and aggravating epistasis, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2049/F8.medium.gif" width="440" height="153"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/3/11/2049/F8.medium.gif" width="440" height="153"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463514" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-46" class="first-child">Fitness effects of double-knockouts of WGD pairs in the groups (A) CF, (B) NF, and (C) SF. The dashed line corresponds to no epistasis, whereas the regions above and below the line correspond to buffering and aggravating epistasis, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-47">For the SF group, 2 pairs have no epistasis, 45 pairs are buffering, and 77 pairs are aggravating. For the NF group, 0 pairs have no epistasis, 11 pairs are buffering, and 32 pairs are aggravating. For the CF group, 0 pairs have no epistasis, 3 pairs are buffering, and 12 pairs are aggravating. Overall, WGD gene pairs tend to have more of a buffering epistatic effect, and the trend is more obvious when the duplication pairs evolve with CF.</p><p id="p-48"><a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Dean <em>et al.</em> (2008)</a> pointed out that most duplicated genes are functionally redundant. For essential reactions, only 0.2% show negative epistasis. For nonessential reactions, 4% show negative epistasis. Our results show that WGD pairs have high proportion with negative epistasis, which means WGD genes are highly redundant. Also, the SF group has the lowest ratio of aggravating pairs while CF group has the highest ratio of aggravating pairs. This indicates that CF group is most functional redundant among the three groups, which makes sense given the conserved functionality. Furthermore, these result demonstrate the utility of using network structure for determining the evolutionary fate of gene duplicates.</p></div></div><div class="section discussion" id="sec-8"><h2 class="">Discussion</h2><p id="p-49">In this work, we took a network perspective on the evolution of WGD pairs and investigated WGD pairs in yeast with respect to the yeast’s PPI network. The calibrated time of all gene pairs in this data set makes it an ideal data set for understanding evolution of gene duplications. We correlated divergence of WGD duplicates at the sequence and network level. Further, we demonstrated strong correlations between WGD pair divergence and fitness. Finally, using the neighbors of WGD pairs as proxies for the functions of genes in these pairs, we developed a method to infer the evolutionary fate of WGD pairs and then correlated the categories of WGD pairs with different fates with fitness effects. Our results indicate that network connectivities can provide a powerful tool to investigate and understand the evolution of gene duplicates.</p><p id="p-50">Notice that the estimated <em>μ<sub>a</sub></em> is much smaller than <em>μ</em><sub>ℓ</sub>, which means that during evolution, the chance to add an edge for one or both gene in the duplicated pair is about three orders of magnitude smaller than deleting an edge. This agrees with the hypothesized DMC (duplication-mutation with complementarity) model (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Middendorf <em>et al.</em> 2005</a>) of network evolution regarding gene divergence after duplication. In other words, these types of analyses can help inform whether commonly used models of network evolution are plausible, as well as derive new ones.</p><p id="p-51">It is important to point out that network data does not come without error. Indeed, network data are very erroneous when compared, for example, with sequence data. Although we conducted our analyses independently with two sources of data (DIP and BioGrid), and despite the good agreement between the two, we still expect inaccuracies of network data to be present and affect the results. As technologies for deriving interaction data continue to improve, it would be interesting to apply these methods to more accurate network data.</p><p id="p-52">Another factor that could affect our results is gene conversion, because interlocus conversion events that occurred after WGD significantly affect the estimated sequence divergence and, consequently, the correlations between sequence and network divergence. It is estimated that only approximately 10% gene pairs underwent gene conversion, and it would be interesting to investigate how gene conversion comes into play between sequence and network divergence.</p><p id="p-53">Finally, a question naturally arises as to whether WGD pairs are a good representative of gene duplicate pairs in general. To investigate this question, we inspected four properties of WGD and non-WGD pairs: PPI degrees, lengths of gene sequences, expression levels, and fitness values. The results are shown in <a id="xref-fig-9-1" class="xref-fig" href="#F9">Figure 9</a>.</p><div id="F9" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F9.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The PPI degree, gene length, gene expression level, and fitness level of WGD pairs and non-WGD pairs are shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1988628563" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The PPI degree, gene length, gene expression level, and fitness level of WGD pairs and non-WGD pairs are shown.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 9" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/3/11/2049/F9.medium.gif" width="440" height="342"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 9" src="https://www.g3journal.org/content/ggg/3/11/2049/F9.medium.gif" width="440" height="342"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F9.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 9" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/3/11/2049/F9.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/463513" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 9</span> <p id="p-54" class="first-child">The PPI degree, gene length, gene expression level, and fitness level of WGD pairs and non-WGD pairs are shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-55">The figure clearly shows that with the exception of gene lengths, WGD and non-WGD pairs agree in terms of other properties. These results indicate that WGD pairs provide a good sample of gene duplicates in general. Given the knowledge about their duplication time, they are the ideal candidates of gene duplications to shed light on network evolution, and to translate network-based information from WGD pairs to general duplicate pairs. These results further highlight the significance of our findings on modeling network evolution and developing model-based methods for ancestral network reconstruction (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Navlakha and Kingsford 2011</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Zhu and Nakhleh 2012</a>).</p><p id="p-56">*Here, we are assuming that gain and loss of interactions are governed mainly by substitutions at the sequence level.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-57">This work was supported in part by National Science Foundation grant CCF-0622037, an Alfred P. Sloan Research Fellowship, and a Guggenheim Fellowship to L.N. The contents are solely the responsibility of the authors and do not necessarily represent the official views of the NSF, the Alfred P. Sloan Foundation, or the Guggenheim Foundation.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-58"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2013-09-07"><span class="received-label">Received </span>September 7, 2013.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2013-09-10"><span class="accepted-label">Accepted </span>September 10, 2013.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2013 Zhu <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.1" data-doi="10.1093/bioinformatics/18.11.1486"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bhan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galas</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dewey</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A duplication growth model of gene expression networks.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1486</span>–<span class="cit-lpage">1493</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DBhan%26rft.auinit1%253DA.%26rft.volume%253D18%26rft.issue%253D11%26rft.spage%253D1486%26rft.epage%253D1493%26rft.atitle%253DA%2Bduplication%2Bgrowth%2Bmodel%2Bof%2Bgene%2Bexpression%2Bnetworks%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F18.11.1486%26rft_id%253Dinfo%253Apmid%252F12424120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMTgvMTEvMTQ4NiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.2" data-doi="10.1038/nature08064"><div class="cit-metadata"><cite>Butler, G., M. D. Rasmussen, M. F. Lin, M. A. Santos, S. Sakthikumar, <em>et al</em>., <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolution of pathogenicity and sexual reproduction in eight Candida genomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">459</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DButler%26rft.auinit1%253DG.%26rft.volume%253D459%26rft.issue%253D7247%26rft.spage%253D657%26rft.epage%253D662%26rft.atitle%253DEvolution%2Bof%2Bpathogenicity%2Band%2Bsexual%2Breproduction%2Bin%2Beight%2BCandida%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08064%26rft_id%253Dinfo%253Apmid%252F19465905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19465905&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266608600034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.3" data-doi="10.1371/journal.pgen.1000113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petrov</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Pervasive and persistent redundancy among duplicated genes in yeast.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DDean%26rft.auinit1%253DE.%2BJ.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253De1000113%26rft.epage%253De1000113%26rft.atitle%253DPervasive%2Band%2Bpersistent%2Bredundancy%2Bamong%2Bduplicated%2Bgenes%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000113%26rft_id%253Dinfo%253Apmid%252F18604285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000113&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18604285&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2049.4"><div class="cit-metadata"><ol class="cit-ed-list"><li><span class="cit-ed"><span class="cit-name-surname">Dittmar</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-ed"><span class="cit-name-surname">Liberles</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite> (Editors), <span class="cit-pub-date">2010</span> <span class="cit-source">Evolution after Gene Duplication</span>. <span class="cit-publ-name">Wiley-Blackwell</span>, <span class="cit-publ-loc">Hoboken, NJ</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.5" data-doi="10.1038/nature00935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riles</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Functional profiling of the <em>Saccharomyces cerevisiae</em> genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>: <span class="cit-fpage">387</span>–<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.11.2049.6"><div class="cit-metadata unstructured">Gibson, T., and D. Goldberg, 2008 Reverse engineering the evolution of protein interaction networks. Proceedings of the Pacific Symposium on Biocomputing, pp. 190–202. The Big Island of Hawaii.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.7" data-doi="10.1534/genetics.104.037051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Rapid subfunctionalization accompanied by prolonged and substantial neofunctionalization in duplicate gene evolution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1157</span>–<span class="cit-lpage">1164</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.037051%26rft_id%253Dinfo%253Apmid%252F15654095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS8yLzExNTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.8" data-doi="10.1038/nrg2689"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kondrashov</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The evolution of gene duplications: classifying and distinguishing between models.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DInnan%26rft.auinit1%253DH.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D97%26rft.epage%253D108%26rft.atitle%253DThe%2Bevolution%2Bof%2Bgene%2Bduplications%253A%2Bclassifying%2Band%2Bdistinguishing%2Bbetween%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2689%26rft_id%253Dinfo%253Apmid%252F20051986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2689&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20051986&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273688200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2049.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jukes</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantor</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span>, pp. <span class="cit-fpage">21</span>–<span class="cit-lpage">132</span> in <span class="cit-source">Evolution of Protein Molecules</span>. <span class="cit-publ-name">Academic Press</span>, <span class="cit-publ-loc">San Diego</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.10" data-doi="10.1038/nature02424"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Proof and evolutionary analysis of ancient genome duplication in the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">428</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">624</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKellis%26rft.auinit1%253DM.%26rft.volume%253D428%26rft.issue%253D6983%26rft.spage%253D617%26rft.epage%253D624%26rft.atitle%253DProof%2Band%2Bevolutionary%2Banalysis%2Bof%2Bancient%2Bgenome%2Bduplication%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02424%26rft_id%253Dinfo%253Apmid%252F15004568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02424&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15004568&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220697200034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.11.2049.11"><div class="cit-metadata unstructured">Li, S., P. Choi, T. Wu, and L. Zhang, 2012 Reconstruction of network evolutionary history from extant network topology and duplication history. Proceedings of the 8th International Conference on Bioinformatics Research and Applications, pp. 165–176. Dallas, Texas.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.12" data-doi="10.1073/pnas.0409515102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Middendorf</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziv</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiggins</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Inferring network mechanisms: the <em>Drosophila melanogaster</em> protein interaction network.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">3192</span>–<span class="cit-lpage">3197</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0409515102%26rft_id%253Dinfo%253Apmid%252F15728374%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAyLzkvMzE5MiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.13" data-doi="10.1371/journal.pcbi.1001119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Navlakha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kingsford</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Network archaeology: uncovering ancient networks from present-day interactions.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e1001119</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DNavlakha%26rft.auinit1%253DS.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De1001119%26rft.epage%253De1001119%26rft.atitle%253DNetwork%2Barchaeology%253A%2Buncovering%2Bancient%2Bnetworks%2Bfrom%2Bpresent-day%2Binteractions.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1001119%26rft_id%253Dinfo%253Apmid%252F21533211%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1001119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21533211&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2049.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nei</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">Molecular Evolution and Phylogenetics</span>. <span class="cit-publ-name">Oxford University Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-book" id="cit-3.11.2049.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-source">Evolution by Gene Duplication</span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.16" data-doi="10.1038/ng1686"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pál</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Papp</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lercher</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Adaptive evolution of bacterial metabolic networks by horizontal gene transfer.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">1372</span>–<span class="cit-lpage">1375</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DP%25C3%25A1l%26rft.auinit1%253DC.%26rft.volume%253D37%26rft.issue%253D12%26rft.spage%253D1372%26rft.epage%253D1375%26rft.atitle%253DAdaptive%2Bevolution%2Bof%2Bbacterial%2Bmetabolic%2Bnetworks%2Bby%2Bhorizontal%2Bgene%2Btransfer.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1686%26rft_id%253Dinfo%253Apmid%252F16311593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1686&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16311593&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233657300019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.17" data-doi="10.1093/molbev/msp150"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paps</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baguna</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riutort</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Bilaterian phylogeny: a broad sampling of 13 nuclear genes provides a new Lophotrochozoa phylogeny and supports a paraphyletic basal A coelomorpha.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2397</span>–<span class="cit-lpage">2406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsp150%26rft_id%253Dinfo%253Apmid%252F19602542%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMC8yMzk3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.18" data-doi="10.1073/pnas.0706339104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pinney</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amoutzias</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rattray</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">D. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Reconstruction of ancestral protein interaction networks for the bZIP transcription factors.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">20449</span>–<span class="cit-lpage">20453</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0706339104%26rft_id%253Dinfo%253Apmid%252F18077348%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA0LzUxLzIwNDQ5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.19" data-doi="10.1073/pnas.1104695108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Qian</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Measuring the evolutionary rate of protein-protein interaction.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">8725</span>–<span class="cit-lpage">8730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1104695108%26rft_id%253Dinfo%253Apmid%252F21555556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA4LzIxLzg3MjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.20" data-doi="10.1371/journal.pcbi.0030230"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ratmann</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jørgensen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinkley</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stumpf</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richardson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Using likelihood-free inference to compare evolutionary dynamics of the protein networks of H. pylori and <em>P. falciparum</em>.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DRatmann%26rft.auinit1%253DO.%26rft.volume%253D3%26rft.issue%253D11%26rft.spage%253De230%26rft.epage%253De230%26rft.atitle%253DUsing%2Blikelihood-free%2Binference%2Bto%2Bcompare%2Bevolutionary%2Bdynamics%2Bof%2Bthe%2Bprotein%2Bnetworks%2Bof%2BH.%2Bpylori%2Band%2BP.%2Bfalciparum.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0030230%26rft_id%253Dinfo%253Apmid%252F18052538%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0030230&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18052538&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.21" data-doi="10.1073/pnas.1217630110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ruths</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakhleh</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Neutral forces acting on intragenomic variability shape the <em>Escherichia coli</em> regulatory network topology.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">7754</span>–<span class="cit-lpage">7759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1217630110%26rft_id%253Dinfo%253Apmid%252F23610404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzE5Lzc3NTQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saitou</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Evolutionary rates of insertion and deletion in noncoding nucleotide sequences of primates.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">504</span>–<span class="cit-lpage">512</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSaitou%26rft.auinit1%253DN%26rft.volume%253D11%26rft.issue%253D3%26rft.spage%253D504%26rft.epage%253D512%26rft.atitle%253DEvolutionary%2Brates%2Bof%2Binsertion%2Band%2Bdeletion%2Bin%2Bnoncoding%2Bnucleotide%2Bsequences%2Bof%2Bprimates%26rft_id%253Dinfo%253Apmid%252F8015443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjExLzMvNTA0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Segre</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DeLuna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kishony</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Modular epistasis in yeast metabolism.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">83</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D37%26rft.spage%253D77%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.24" data-doi="10.1091/mbc.9.12.3273"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spellman</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherlock</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iyer</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Comprehensive identification of cell cycle-regulated genes of the yeast <em>Saccharomyces cerevisiae</em> by microarray hybridization.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">3273</span>–<span class="cit-lpage">3297</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DSpellman%26rft.auinit1%253DP.%2BT.%26rft.volume%253D9%26rft.issue%253D12%26rft.spage%253D3273%26rft.epage%253D3297%26rft.atitle%253DComprehensive%2BIdentification%2Bof%2BCell%2BCycle-regulated%2BGenes%2Bof%2Bthe%2BYeast%2BSaccharomyces%2Bcerevisiae%2Bby%2BMicroarray%2BHybridization%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.9.12.3273%26rft_id%253Dinfo%253Apmid%252F9843569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjkvMTIvMzI3MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzMvMTEvMjA0OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.25" data-doi="10.1093/nar/gkj109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breitkreutz</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reguly</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boucher</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breitkreutz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">BioGRID: a general repository for interaction datasets.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">D535</span>–<span class="cit-lpage">D539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DStark%26rft.auinit1%253DC.%26rft.volume%253D34%26rft.issue%253Dsuppl_1%26rft.spage%253DD535%26rft.epage%253DD539%26rft.atitle%253DBioGRID%253A%2Ba%2Bgeneral%2Brepository%2Bfor%2Binteraction%2Bdatasets%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkj109%26rft_id%253Dinfo%253Apmid%252F16381927%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNC9zdXBwbF8xL0Q1MzUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.26" data-doi="10.1038/ng1340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teichmann</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Gene regulatory network growth by duplication.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">492</span>–<span class="cit-lpage">496</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTeichmann%26rft.auinit1%253DS.%2BA.%26rft.volume%253D36%26rft.issue%253D5%26rft.spage%253D492%26rft.epage%253D496%26rft.atitle%253DGene%2Bregulatory%2Bnetwork%2Bgrowth%2Bby%2Bduplication.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1340%26rft_id%253Dinfo%253Apmid%252F15107850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15107850&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221183000022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.27" data-doi="10.1371/journal.pbio.1000414"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsankov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Socha</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Regev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rando</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The role of nucleosome positioning in the evolution of gene regulation.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1000414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DTsankov%26rft.auinit1%253DA.%2BM.%26rft.volume%253D8%26rft.issue%253D7%26rft.spage%253De1000414%26rft.epage%253De1000414%26rft.atitle%253DThe%2Brole%2Bof%2Bnucleosome%2Bpositioning%2Bin%2Bthe%2Bevolution%2Bof%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.1000414%26rft_id%253Dinfo%253Apmid%252F20625544%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.1000414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20625544&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.28" data-doi="10.1073/pnas.0600061103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiuf</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brameier</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hagberg</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stumpf</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A likelihood approach to analysis of network data.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">7566</span>–<span class="cit-lpage">7570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0600061103%26rft_id%253Dinfo%253Apmid%252F16682633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTAzLzIwLzc1NjYiO3M6NDoiYXRvbSI7czoxOToiL2dnZy8zLzExLzIwNDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.29" data-doi="10.1038/42711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shields</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Molecular evidence for an ancient duplication of the entire yeast genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">387</span>: <span class="cit-fpage">708</span>–<span class="cit-lpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWolfe%26rft.auinit1%253DK.%2BH.%26rft.volume%253D387%26rft.issue%253D6634%26rft.spage%253D708%26rft.epage%253D713%26rft.atitle%253DMolecular%2Bevidence%2Bfor%2Ban%2Bancient%2Bduplication%2Bof%2Bthe%2Bentire%2Byeast%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F42711%26rft_id%253Dinfo%253Apmid%252F9192896%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/42711&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9192896&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XD86900055&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.30" data-doi="10.1093/nar/28.1.289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xenarios</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salwinski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baron</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcotte</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">DIP: the database of interacting proteins.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DXenarios%26rft.auinit1%253DI.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D289%26rft.epage%253D291%26rft.atitle%253DDIP%253A%2Bthe%2BDatabase%2Bof%2BInteracting%2BProteins%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.289%26rft_id%253Dinfo%253Apmid%252F10592249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjI4LzEvMjg5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvMy8xMS8yMDQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hannenhalli</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Inferring evolution of gene duplicates using probabilistic models and nonparametric belief propagation.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">S15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253DS15%26rft_id%253Dinfo%253Apmid%252F23368094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23368094&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.11.2049.32"><div class="cit-metadata unstructured">Zhang, S., H. Liu, X. Ning, and X. Zhang, 2006 A graph-theoretic method for mining functional modules in large sparse protein interaction networks. Proceedings of the Sixth IEEE International Conference on Data Mining-Workshops, pp. 130–135. Brussels, Belgium.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-confproc" id="cit-3.11.2049.33"><div class="cit-metadata unstructured">Zhu, Y., and L. Nakhleh, 2012 Reconstructing the evolution of molecular interaction networks under the DMC and link dynamics models, pp. 57–68 in <em>Workshop on Algorithms in Bioinformatics</em>, Springer, New York.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-3.11.2049.34" data-doi="10.1371/journal.pone.0044491"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Du</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakhleh</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gene duplicability-connectivity-complexity across organisms and a neutral evolutionary explanation.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e44491</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DZhu%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D9%26rft.spage%253De44491%26rft.epage%253De44491%26rft.atitle%253DGene%2Bduplicability-connectivity-complexity%2Bacross%2Borganisms%2Band%2Ba%2Bneutral%2Bevolutionary%2Bexplanation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0044491%26rft_id%253Dinfo%253Apmid%252F22984517%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0044491&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22984517&amp;link_type=MED&amp;atom=%2Fggg%2F3%2F11%2F2049.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/3/11/2049.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/3/11/2031" title="Construction of Reference Chromosome-Scale Pseudomolecules for Potato: Integrating the Potato Genome with Genetic and Physical Maps" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/3/11/2059" title="Molecular Phylogeography of a Human Autosomal Skin Color Locus Under Natural Selection" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/3/11"><span>Volume 3 Issue 11, November 2013</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/3/11.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/3/11.cover-source.jpg?itok=A7iZ1ztq" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 3 (11)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/3/11/2049" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-mIzrNJ_fjxM8YqZwYfiZv-GcgTr_VyRJL-514aR_2RY" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/6th-annual-recombiscb-conference-regulatory-systems-genomics" class="" data-icon-position="" data-hide-link-title="0">6th Annual RECOMB/ISCB Conference on Regulatory &amp; Systems Genomics</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/3/11/2049" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/3/11/2049" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/3/11/2049" class="active" data-icon-position="" data-hide-link-title="0">Evolution After Whole-Genome Duplication: A Network Perspective</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459567" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-V6bIuzNO0S1-VA4MNXMdwksWKc0LB54m7CjoxBEI6cw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/2049.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/3/11/2049" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Rh4a6MW2xe8hQx_mEw6lJ-8G-Pt5JOPyvolhw8PsuHc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/3/11/2049" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/3/11/2049.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459567" id="node459567--2" data-pisa="ggg;3/11/2049" data-pisa-master="ggg;g3.113.008458" data-apath="/ggg/3/11/2049.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evolution After Whole-Genome Duplication: A Network Perspective</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yun Zhu</span>, <span class="highwire-citation-author" data-delta="1">Zhenguo Lin</span> and <span class="highwire-citation-author" data-delta="2">Luay Nakhleh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2049-2057;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008458 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459567--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yun Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Computer Science, Rice University, Houston, Texas 77005</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yun%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYun%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zhenguo Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, Rice University, Houston, Texas 77005</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhenguo%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhenguo%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Luay Nakhleh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Computer Science, Rice University, Houston, Texas 77005</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, Rice University, Houston, Texas 77005</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luay%2BNakhleh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nakhleh%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuay%2BNakhleh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nakhleh@rice.edu" class="" data-icon-position="" data-hide-link-title="0">nakhleh@rice.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/3/11/2049" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/3/11/2049&amp;title=Evolution%20After%20Whole-Genome%20Duplication%3A%20A%20Network%20Perspective" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/3/11/2049&amp;title=Evolution%20After%20Whole-Genome%20Duplication%3A%20A%20Network%20Perspective" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/3/11/2049&amp;title=Evolution%20After%20Whole-Genome%20Duplication%3A%20A%20Network%20Perspective" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/3/11/2049&amp;text=Evolution%20After%20Whole-Genome%20Duplication%3A%20A%20Network%20Perspective" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/3/11/2049&amp;title=Evolution%20After%20Whole-Genome%20Duplication%3A%20A%20Network%20Perspective" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/3/11/2049&amp;t=Evolution%20After%20Whole-Genome%20Duplication%3A%20A%20Network%20Perspective" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/3/11/2049&amp;title=Evolution%20After%20Whole-Genome%20Duplication%3A%20A%20Network%20Perspective" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/3/11/2049&amp;title=Evolution%20After%20Whole-Genome%20Duplication%3A%20A%20Network%20Perspective" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459567/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459567" id="node459567--3" data-pisa="ggg;3/11/2049" data-pisa-master="ggg;g3.113.008458" data-apath="/ggg/3/11/2049.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Evolution After Whole-Genome Duplication: A Network Perspective</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yun Zhu</span>, <span class="highwire-citation-author" data-delta="1">Zhenguo Lin</span> and <span class="highwire-citation-author" data-delta="2">Luay Nakhleh</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">November 1, 2013 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 3 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 11 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2049-2057;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.113.008458 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459567--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yun Zhu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Computer Science, Rice University, Houston, Texas 77005</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yun%2BZhu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYun%2BZhu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Zhenguo Lin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, Rice University, Houston, Texas 77005</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhenguo%2BLin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lin%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhenguo%2BLin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Luay Nakhleh </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Computer Science, Rice University, Houston, Texas 77005</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Ecology and Evolutionary Biology, Rice University, Houston, Texas 77005</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luay%2BNakhleh%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nakhleh%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALuay%2BNakhleh%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nakhleh@rice.edu" class="" data-icon-position="" data-hide-link-title="0">nakhleh@rice.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2049&amp;rft.epage=2057&amp;rft.atitle=Evolution%20After%20Whole-Genome%20Duplication%3A%20A%20Network%20Perspective&amp;rft.volume=3&amp;rft.issue=11&amp;rft.date=2013-11-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Zhu%2C+Yun&amp;rft.au=Lin%2C+Zhenguo&amp;rft.au=Nakhleh%2C+Luay"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459567/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459567/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459567/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459567/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459567/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459567/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459567/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459567/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459567/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459567/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459567/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459567/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/3/11/2049" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-3" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/3/11/2049.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/3/11/2049.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/3/11/2049.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
